
<html lang="en"     class="pb-page"  data-request-id="6ec1a558-4953-48d3-a633-f4cb748dd4d4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-8;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.0c00193;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Biological Evaluation of Quinazolin-4-one-Based Hydroxamic Acids as Dual PI3K/HDAC Inhibitors" /></meta><meta name="dc.Creator" content="Ashish  Thakur" /></meta><meta name="dc.Creator" content="Gregory J.  Tawa" /></meta><meta name="dc.Creator" content="Mark J.  Henderson" /></meta><meta name="dc.Creator" content="Carina  Danchik" /></meta><meta name="dc.Creator" content="Suiyang  Liu" /></meta><meta name="dc.Creator" content="Pranav  Shah" /></meta><meta name="dc.Creator" content="Amy Q.  Wang" /></meta><meta name="dc.Creator" content="Garrett  Dunn" /></meta><meta name="dc.Creator" content="Md  Kabir" /></meta><meta name="dc.Creator" content="Elias C.  Padilha" /></meta><meta name="dc.Creator" content="Xin  Xu" /></meta><meta name="dc.Creator" content="Anton  Simeonov" /></meta><meta name="dc.Creator" content="Surender  Kharbanda" /></meta><meta name="dc.Creator" content="Richard  Stone" /></meta><meta name="dc.Creator" content="Gurmit  Grewal" /></meta><meta name="dc.Description" content="A series of quinazolin-4-one based hydroxamic acids was rationally designed and synthesized as novel dual PI3K/HDAC inhibitors by incorporating an HDAC pharmacophore into a PI3K inhibitor (Idelalis..." /></meta><meta name="Description" content="A series of quinazolin-4-one based hydroxamic acids was rationally designed and synthesized as novel dual PI3K/HDAC inhibitors by incorporating an HDAC pharmacophore into a PI3K inhibitor (Idelalis..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 26, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00193" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00193" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00193" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00193" /></link>
        
    
    

<title>Design, Synthesis, and Biological Evaluation of Quinazolin-4-one-Based Hydroxamic Acids as Dual PI3K/HDAC Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00193" /></meta><meta property="og:title" content="Design, Synthesis, and Biological Evaluation of Quinazolin-4-one-Based Hydroxamic Acids as Dual PI3K/HDAC Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0026.jpeg" /></meta><meta property="og:description" content="A series of quinazolin-4-one based hydroxamic acids was rationally designed and synthesized as novel dual PI3K/HDAC inhibitors by incorporating an HDAC pharmacophore into a PI3K inhibitor (Idelalisib) via an optimized linker. Several of these dual inhibitors were highly potent (IC50 &lt; 10 nM) and selective against PI3Kγ, δ and HDAC6 enzymes and exhibited good antiproliferative activity against multiple cancer cell lines. The lead compound 48c, induced necrosis in several mutant and FLT3-resistant AML cell lines and primary blasts from AML patients, while showing no cytotoxicity against normal PBMCs, NIH3T3, and HEK293 cells. Target engagement of PI3Kδ and HDAC6 by 48c was demonstrated in MV411 cells using the cellular thermal shift assay (CETSA). Compound 48c showed good pharmacokinetics properties in mice via intraperitoneal (ip) administration and provides a means to examine the biological effects of inhibiting these two important enzymes with a single molecule, either in vitro or in vivo." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00193"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00193">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00193&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00193&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00193&amp;href=/doi/10.1021/acs.jmedchem.0c00193" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 4256-4292</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00156" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00200" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Biological Evaluation of Quinazolin-4-one-Based Hydroxamic Acids as Dual PI3K/HDAC Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ashish Thakur</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ashish Thakur</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><span class="conrtib-corresp"><strong>*</strong>A.T.: phone, +1-301-827-7179; email, <a href="/cdn-cgi/l/email-protection#badbc9d2d3c9d294ced2dbd1cfc8fad4d3d294ddd5cc"><span class="__cf_email__" data-cfemail="fc9d8f94958f94d288949d97898ebc929594d29b938a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ashish++Thakur">Ashish Thakur</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8456-1088" title="Orcid link">http://orcid.org/0000-0002-8456-1088</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gregory J. Tawa</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gregory J. Tawa</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gregory+J.++Tawa">Gregory J. Tawa</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark J. Henderson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark J. Henderson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark+J.++Henderson">Mark J. Henderson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8590-7338" title="Orcid link">http://orcid.org/0000-0001-8590-7338</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carina Danchik</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carina Danchik</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carina++Danchik">Carina Danchik</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Suiyang Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Suiyang Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Suiyang++Liu">Suiyang Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pranav Shah</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pranav Shah</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pranav++Shah">Pranav Shah</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Amy Q. Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Amy Q. Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Amy+Q.++Wang">Amy Q. Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Garrett Dunn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Garrett Dunn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Garrett++Dunn">Garrett Dunn</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Md Kabir</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Md Kabir</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Md++Kabir">Md Kabir</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Elias C. Padilha</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Elias C. Padilha</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Elias+C.++Padilha">Elias C. Padilha</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Xu">Xin Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anton Simeonov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anton Simeonov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anton++Simeonov">Anton Simeonov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Surender Kharbanda</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Surender Kharbanda</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Surender++Kharbanda">Surender Kharbanda</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard Stone</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Stone</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dana-Farber Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Stone">Richard Stone</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Gurmit Grewal</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gurmit Grewal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">National Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</div></div><span class="conrtib-corresp"><strong>*</strong>G.G.; phone, +1-508-740-7465; email, <a href="/cdn-cgi/l/email-protection#ed8a989f808499c38a9f889a8c81ad8a808c8481c38e8280"><span class="__cf_email__" data-cfemail="1f786a6d72766b31786d7a687e735f78727e7673317c7072">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gurmit++Grewal">Gurmit Grewal</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00193&amp;href=/doi/10.1021%2Facs.jmedchem.0c00193" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 4256–4292</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 26, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>3 February 2020</li><li><span class="item_label"><b>Published</b> online</span>26 March 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 April 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00193" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00193</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D4256%26pageCount%3D37%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DAshish%2BThakur%252C%2BGregory%2BJ.%2BTawa%252C%2BMark%2BJ.%2BHenderson%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D8%26contentID%3Dacs.jmedchem.0c00193%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BEvaluation%2Bof%2BQuinazolin-4-one-Based%2BHydroxamic%2BAcids%2Bas%2BDual%2BPI3K%252FHDAC%2BInhibitors%26numPages%3D37%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D4292%26publicationDate%3DApril%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00193"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2357</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00193" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Biological Evaluation of Quinazolin-4-one-Based Hydroxamic Acids as Dual PI3K/HDAC Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ashish&quot;,&quot;last_name&quot;:&quot;Thakur&quot;},{&quot;first_name&quot;:&quot;Gregory&quot;,&quot;last_name&quot;:&quot;J. Tawa&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;J. Henderson&quot;},{&quot;first_name&quot;:&quot;Carina&quot;,&quot;last_name&quot;:&quot;Danchik&quot;},{&quot;first_name&quot;:&quot;Suiyang&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Pranav&quot;,&quot;last_name&quot;:&quot;Shah&quot;},{&quot;first_name&quot;:&quot;Amy&quot;,&quot;last_name&quot;:&quot;Q. Wang&quot;},{&quot;first_name&quot;:&quot;Garrett&quot;,&quot;last_name&quot;:&quot;Dunn&quot;},{&quot;first_name&quot;:&quot;Md&quot;,&quot;last_name&quot;:&quot;Kabir&quot;},{&quot;first_name&quot;:&quot;Elias&quot;,&quot;last_name&quot;:&quot;C. Padilha&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Anton&quot;,&quot;last_name&quot;:&quot;Simeonov&quot;},{&quot;first_name&quot;:&quot;Surender&quot;,&quot;last_name&quot;:&quot;Kharbanda&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Stone&quot;},{&quot;first_name&quot;:&quot;Gurmit&quot;,&quot;last_name&quot;:&quot;Grewal&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;26&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;4256-4292&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00193&quot;},&quot;abstract&quot;:&quot;A series of quinazolin-4-one based hydroxamic acids was rationally designed and synthesized as novel dual PI3K/HDAC inhibitors by incorporating an HDAC pharmacophore into a PI3K inhibitor (Idelalisib) via an optimized linker. Several of these dual inhibitors were highly potent (IC50 &lt; 10 nM) and selective against PI3Kγ, δ and HDAC6 enzymes and exhibited good antiproliferative activity against multiple cancer cell lines. The lead compound 48c, induced necrosis in several mutant and FLT3-resistant AML cell lines and primary blasts from AML patients, while showing no cytotoxicity against normal PBMCs, NIH3T3, and HEK293 cells. Target engagement of PI3Kδ and HDAC6 by 48c was demonstrated in MV411 cells using the cellular thermal shift assay (CETSA). Compound 48c showed good pharmacokinetics properties in mice via intraperitoneal (ip) administration and provides a means to examine the biological effects of inhibiting these two important enzymes with a single molecule, either in vitro or in vivo.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00193&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00193" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00193&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00193" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00193&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00193" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00193&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00193&amp;href=/doi/10.1021/acs.jmedchem.0c00193" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00193" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00193" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00193%26sid%3Dliteratum%253Aachs%26pmid%3D32212730%26genre%3Darticle%26aulast%3DThakur%26date%3D2020%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BEvaluation%2Bof%2BQuinazolin-4-one-Based%2BHydroxamic%2BAcids%2Bas%2BDual%2BPI3K%252FHDAC%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D8%26spage%3D4256%26epage%3D4292%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292178" title="Organic compounds">Organic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/jmcmar.2020.63.issue-8/20200423/jmcmar.2020.63.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">A series of quinazolin-4-one based hydroxamic acids was rationally designed and synthesized as novel dual PI3K/HDAC inhibitors by incorporating an HDAC pharmacophore into a PI3K inhibitor (Idelalisib) via an optimized linker. Several of these dual inhibitors were highly potent (IC<sub>50</sub> < 10 nM) and selective against PI3Kγ, δ and HDAC6 enzymes and exhibited good antiproliferative activity against multiple cancer cell lines. The lead compound <b>48c</b>, induced necrosis in several mutant and FLT3-resistant AML cell lines and primary blasts from AML patients, while showing no cytotoxicity against normal PBMCs, NIH3T3, and HEK293 cells. Target engagement of PI3Kδ and HDAC6 by <b>48c</b> was demonstrated in MV411 cells using the cellular thermal shift assay (CETSA). Compound <b>48c</b> showed good pharmacokinetics properties in mice via intraperitoneal (ip) administration and provides a means to examine the biological effects of inhibiting these two important enzymes with a single molecule, either in vitro or in vivo.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43240" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43240" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Most of the currently approved drugs based on the “single-target single drug” model are becoming less effective in the treatment of complex, heterogeneous, multigenic diseases, like cancer.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> While drug combination therapies are employed as an alternative approach to achieve efficacy, their benefits are often negated by adverse drug–drug interactions, unpredictable pharmacokinetic (PK) and safety profiles, and poor patient compliance.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> In recent years, there has been an increasing inclination toward discovering “multitarget drugs” to address the limited efficacy and resistance or toxicity associated with many single-target or combination-based therapies.<a onclick="showRef(event, 'ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8 ref9 ref10 ref11 ref12">(5−12)</a></div><div class="NLM_p">Class I phosphoinositide 3-kinases (PI3Ks) are lipid kinase enzymes that transduce signals from cell surface receptors (RTK, GPCR, etc.) to downstream effectors (AKT, mTOR, etc.), leading to a variety of cellular processes, including cell proliferation, survival, differentiation, metabolism, and angiogenesis.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Specifically, class I PI3Ks phosphorylate phosphatidylinositol (4,5)-bisphosphate (PI(4,5)P2 or PIP2) in vivo to form the secondary messenger phosphatidylinositol (3,4,5)-trisphosphate (PI(3,4,5)P3 or PIP3), which activates the pleckstrin homology (PH)-domain of protein kinases (Akt, BTK, etc.), leading to downstream signaling.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a> The four isoforms of Class I PI3Ks (PI3Kα, PI3Kβ, PI3Kγ, and PI3Kδ) exist as heterodimers, consisting of a catalytic p110 (α, β, δ, γ) subunit and a regulatory p85 (α, β, δ) or p101/p84 (γ) subunit.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> The PI3Kα and PI3Kβ isoforms are ubiquitously expressed, whereas PI3Kγ and PI3Kδ expression is limited to leukocytes.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Aberrant regulation of class I PI3Ks has been implicated in several cancer types, and their inhibition continues to be an active cancer therapeutic area with four recently approved PI3K inhibitor drugs (Idelalisib, Copanlisib, Duvelisib, and Alpelisib) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and several others in ongoing clinical trials.<a onclick="showRef(event, 'ref17 ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20">(17−20)</a> Despite tremendous progress in the discovery of PI3K inhibitors (PI3Ki) over the past decade, clinical trials with PI3Ki as a monotherapy have shown poor efficacy,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> prompting their evaluation in combination therapies and/or developing PI3K-based multitarget drugs.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0001.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. FDA Approved PI3K inhibitor drugs for cancer (CLL = chronic lymphocytic leukemia; FL = follicular lymphoma; SLL = small lymphocytic lymphoma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Histone deacetylases (HDACs) are an important class of epigenetic enzymes that regulate gene expression by removing acetyl groups from ε-amino lysine residues on histones.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> HDACs play an important role in regulating expression of various proteins, including tumor suppressors (p53, p21, etc.) and transcription factors (e.g., TFIIE, TCF, SF1, etc.).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Dysregulation of HDACs is involved in cancer initiation and proliferation.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> HDAC inhibition has emerged as a therapeutic approach for cancer and several inhibitors, including Vorinostat (SAHA), Romidepsin, Belinostat, Panobinostat, and Chidamide, have been approved in recent years (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Pan-HDAC inhibitors, such as Panobinostat and Vorinostat (SAHA), modulate the acetylation status of a wide range of protein targets leading to a therapeutic response; however, these molecules also have undesirable side-effects, including hematological, gastrointestinal, and cardiac toxicity.<a onclick="showRef(event, 'ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27">(25−27)</a> SAHA monotherapy was approved by the U.S. Food and Drug Administration (FDA) in 2006 for the treatment of cutaneous T-cell lymphoma (CTCL), however, it has been demonstrated to have little efficacy.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0002.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Selected approved HDAC inhibitor drugs for cancer. (B) Chemical structure of dual PI3K/HDAC inhibitor, Fimepinostat (CTCL = cutaneous T-cell lymphoma; PTCL = peripheral T-cell lymphoma; DLBCL = diffuse large B-cell lymphoma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although both PI3K and HDAC inhibitor drugs are limited by insufficient efficacy and resistance mutations, there is strong evidence that simultaneous inhibition of both PI3K and HDAC can synergistically inhibit tumor growth and address these limitations by improving efficacy, limiting resistance, and providing a better therapeutic window than single inhibitors.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28−30)</a> Selective inhibitors of specific isoforms of HDAC and PI3K potentially would have a better toxicity profile and therefore bigger therapeutic window.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In this context, several dual PI3K/HDAC inhibitors have been recently reported in literature,<a onclick="showRef(event, 'ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref34">(32−34)</a> with Curis Inc.’s Fimepinostat (CUDC-907) currently being evaluated in phase 2 clinical trials for treatment of diffuse large B-cell lymphoma (DLBCL) [<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B].<a onclick="showRef(event, 'ref19 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref19 ref35 ref36">(19,35,36)</a> Dual inhibitors, such as CUDC-907, may thus offer improved therapeutic benefit through synergistic effects in inhibiting cancer cell proliferation and their compensatory pathways for survival.<a onclick="showRef(event, 'ref19 ref35'); return false;" href="javascript:void(0);" class="ref ref19 ref35">(19,35)</a> However, CUDC-907 exhibits pan-HDAC and pan-PI3K inhibition, which might lead to toxicity and low tolerability.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> Thus, there remains an unmet need for new dual inhibitors having high potency and selectivity.</div><div class="NLM_p last">Herein we report the design, synthesis, and biological evaluation of potent and selective dual PI3K/HDAC inhibitors.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44656" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44656" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design of Dual Inhibitors</h3><div class="NLM_p">The rational design of our PI3K/HDAC dual inhibitors commenced by analyzing the crystal structures of Idelalisib (<b>1</b>) bound to PI3Kδ [PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0">4XE0</a>]<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A) and Vorinostat (<b>2</b>) bound to HDAC2 [PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>]<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), respectively. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, the purine moiety in Idelalisib binds to the hinge (Glu826, Val828) and the quinazolinone core extends up into a pocket formed by the G loop residues, Trp760 and Met752. The carbonyl and fluoro group in the quinazolinone core at fourth and fifth positions, respectively, extend into the solvent exposed area, and therefore, serve as desired locations to attach the HDAC pharmacophore. The phenyl ring on the nitrogen in the quinazolinone core is largely solvent exposed (on top) but makes hydrophobic contacts underneath to methyl groups on Thr833 and Met900. In the Vorinostat-HDAC2 structure (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), Zn<sup>2+</sup> is penta-coordinated (Asp181, Asp260, His183, and both oxygens on hydroxamic acid) and the hydroxamic acid group forms hydrogen bonds to two charge relay histidines (His145 and His146) and a nearby Tyr308. The Vorinostat aliphatic chain traverses a hydrophobic portion of the pocket (Leu144, Phe155). The phenyl group forms a hydrogen bond to the rim of the pocket at Asp104 via the amino nitrogen.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0003.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Chemical structure of the PI3Kδ inhibitor, Idelalisib (<b>1</b>). (B) Crystal structures of Idelalisib (<b>1</b>) [top view] bound to PI3Kδ (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0">4XE0</a>) with protein surface (blue) and without protein surface, respectively. (C) Chemical structure of the HDAC inhibitor, Vorinostat (<b>2</b>). (D)  Crystal structures of Vorinostat (<b>2</b>) bound to HDAC2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>) with protein surface (blue) and without protein surface, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In general, most of the HDAC inhibitors, including Vorinostat, share these three common structural components: a zinc binding group (e.g., hydroxamic acid), a linker occupying the hydrophobic tunnel in the active site, and a cap group that sits outside the hydrophobic tunnel and interacts with the HDAC surface residues (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> More importantly, the HDAC inhibitors can accommodate a wide variety of cap groups, which allows for replacement of the cap with a second target pharmacophore to design HDAC inhibitor based multitarget drugs.<a onclick="showRef(event, 'ref32 ref33 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref32 ref33 ref42 ref43 ref44 ref45 ref46 ref47 ref48 ref49 ref50">(32,33,42−50)</a> Thus, by assessing the binding features of Idelalisib and Vorinostat in PI3Kδ and HDAC2 enzymes, respectively, we designed quinazoline- and quinazolinone-based dual PI3K/HDAC inhibitors (<b>3</b> and <b>4</b>, <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) by incorporating the PI3Kδ inhibitor as the cap group of an HDAC pharmacophore and optimizing for the appropriate linker-hydroxamic acid combination at quinazoline’s C-4 position (<b>3</b>) or quinazolinone’s C-5 position (<b>4</b>) to achieve potencies against both the targets.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0004.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design strategy for dual PI3K/HDAC inhibitor types <b>3</b> and <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">The C-4 substituted quinazoline compounds (<b>10a</b>,<b>b</b> and <b>14a</b>,<b>b</b>) were synthesized from <i>tert</i>-butyl (<i>S</i>)-(1-(4-chloroquinazolin-2-yl)propyl)carbamate <b>5</b><a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> in 6–7 steps as outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The compound <b>5</b> was subjected to Sonogashira cross-coupling reactions with alkynes <b>6a</b> and <b>6b</b>, using a copper-free Pd/P<sup>t</sup>Bu<sub>3</sub>-catalytic system to afford <b>7a</b>,<b>b</b>. Reduction of alkyne group to alkane in <b>7a</b>,<b>b</b> using Pd/C hydrogenation, followed by boc-deprotection and nucleophilic aromatic substitution of 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine <b>8</b> with the resulting free amine, yielded compounds <b>9a</b>,<b>b</b>. Hydrolysis of the alkyl ester in <b>9a</b>,<b>b</b> to the respective carboxylic acid, followed by coupling with NH<sub>2</sub>OTHP (THP = tetrahydropyran) and THP-deprotection using trifluoroacetic acid formed the target compounds <b>10a</b>,<b>b</b>.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0014.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of C-4 Alkyl/Amino-alkyl Substituted Quinazolines (<b>10a</b>,<b>b</b> and <b>14a</b>,<b>b</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>6a</b>, <b>6b</b> (1.2 equiv), [PdCl(allyl)]<sub>2</sub> (0.05 equiv), P<sup><i>t</i></sup>Bu<sub>3</sub>HBF<sub>4</sub> (0.20 equiv), 1,4-dioxane, 16 h, rt; (b) (i) 10 wt % Pd/C, H<sub>2</sub> balloon, EtOAc, 20 h, rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (iii) <b>8</b> (2.0 equiv), triethyamine (4.0 equiv), EtOH, 1 h, MW, 100 °C; (c) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt; (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 20 h, rt (<i>For</i><b>9a</b><i>, only c(ii) and c(iii) were used</i>); (d) <b>11a</b>,<b>b</b> (2.0 equiv), triethyamine (4.0 equiv), EtOH, 1 h, 100 °C; (e) <b>8</b> (2.0 equiv), triethyamine (4.0 equiv), EtOH, 1 h, 100 °C.</p></p></figure><div class="NLM_p">The synthesis of C-4 amino-alkyl substituted compounds (<b>14a</b>,<b>b</b>) began by subjecting compound <b>5</b> to nucleophilic aromatic substitution with amines <b>11a</b>,<b>b</b> to yield compounds <b>12a</b>,<b>b</b>. Boc-deprotection in <b>12a</b>,<b>b</b>, followed by the installation of a purine kinase hinge binding moiety using nucleophilic aromatic substitution (S<sub>N</sub>Ar) on <b>8</b>, afforded compounds <b>13a</b>,<b>b</b>. Target compounds <b>14a</b>,<b>b</b> were synthesized from <b>13a</b>,<b>b</b> by converting alkyl ester to hydroxamic acid, in a fashion similar to that described above for compounds <b>10a</b>,<b>b</b>.</div><div class="NLM_p">The procedure for syntheses of C-5 alkyl/aryl substituted quinazolinone-based compounds is depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The quinazolinone compound <b>15a</b><a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> was subjected to Sonogashira cross-coupling reactions with alkynes <b>16a</b>–<b>c</b> to yield compounds <b>17a</b>–<b>c</b>. Hydrogenation of alkyne in <b>17a</b>–<b>c</b> followed by Boc-deprotection and nucleophilic aromatic substitution on purine <b>8</b> afforded compounds <b>18a</b>–<b>c</b>. Target compounds <b>19a</b>–<b>c</b> were derived from <b>18a</b>–<b>c</b> by either: (a) hydrolyzing the alkyl ester to carboxylic acid, coupling with NH<sub>2</sub>OTHP followed by deprotection of THP-protecting groups using TFA, or (b) converting the alkyl ester directly to hydroxamic acid using aq NH<sub>2</sub>OH/LiOH and subsequent removal of the THP-protecting group using TFA. The latter method also formed carboxylic acid as the minor side product due to ester hydrolysis (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Supporting Information</a>).</div><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0015.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of C-5 Alkyl/Aryl Substituted Quinazolinones (<b>19a</b>–<b>c</b> and <b>24a</b>–<b>c</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>16a</b>, <b>16b</b>, <b>16c</b> (1.2 equiv), [PdCl(allyl)]<sub>2</sub> (0.05 equiv), P<i><sup>t</sup></i>Bu<sub>3</sub>HBF<sub>4</sub> (0.20 equiv), 1,4-dioxane, 16 h, rt; (b) (i) 10 wt % Pd/C, H<sub>2</sub> balloon, EtOAc, 20 h, rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (iii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, MW, 100 °C; (c) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt, (<i>For</i><b>17a</b><i>, only c(ii) and c(iii)</i> were used); (d) (i) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (e) boronic acid/ester <b>20a</b>,<b>b</b> (1.2 equiv), [XPhos Pd(crotyl)Cl] (0.05 equiv), K<sub>3</sub>PO<sub>4</sub> (3.0 equiv), 1,4-dioxane/water, 1–2 h MW, 100 °C; (f) [Bpin]<sub>2</sub> (1.2 equiv), [Pd(dppf)Cl<sub>2</sub>] (0.05 equiv), KOAc (3.0 equiv) 1,4-dioxane, 16 h, 100 °C; (g) <b>20c</b> (0.85 equiv), [XPhos Pd(crotyl)Cl] (0.05 equiv), K<sub>3</sub>PO<sub>4</sub> (3.0 equiv), 1,4-dioxane/water, 10 h, 100 °C; (h) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, 100 °C.</p></p></figure><div class="NLM_p">For C-5 aryl substituted quinazolinone-based compounds, 5-bromo or 5-chloro-quinazolinone (<b>15a</b> or <b>15a′</b>) was either subjected to a Suzuki–Miyaura cross-coupling reaction with boronic acids <b>20a</b>,<b>b</b> or first converted to boronate ester <b>21</b> using Pd(dppf)Cl<sub>2</sub>/Bpin<sub>2</sub>, followed by Suzuki–Miyaura coupling with aryl bromide <b>20c</b>, to form compounds <b>22a</b>–<b>c</b>. These C-5 arylated molecules <b>22a</b>–<b>c</b> were converted to target compounds <b>24a</b>–<b>c</b> in a sequence of steps similar to that previously described for compounds <b>19a</b>–<b>c</b>.</div><div class="NLM_p">The synthesis of C-5 amino-alkyl/aryl substituted quinazolinones involved the Pd/XantPhos-catalyzed amination of 5-bromo-quinazolinone <b>15a</b>,<b>b</b> with amines <b>25a</b>–<b>h</b>, yielding compounds <b>26a</b>–<b>i</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). Subsequent attachment of purine <b>8</b> and conversion of alkyl esters to hydroxamic acids afforded the target compounds <b>28a</b>–<b>h</b>. For compound <b>28i</b>, the methyl ester in <b>27d</b> was converted to <i>N</i>-methyl hydroxamic acid via hydrolysis to carboxylic acid, followed by its coupling with <i>N</i>-methyl hydroxylamine using HATU/DIPEA.</div><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0016.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of C-5 Amino-alkyl/Aryl Substituted Quinazolinones (<b>28a</b>–<b>i</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>25a</b>–<b>h</b> (1.3 equiv), [Xantphos Palladacycle Gen. 4] (0.03 equiv), Cs<sub>2</sub>CO<sub>3</sub> (3.0 equiv), toluene or 1,4-dioxane, 16 h, rt; (b) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, MW, 100 °C; (c) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (d) (i) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (e) (i) LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt, (ii) <i>N</i>-methylhydroxylamine hydrochloride (1.5 equiv), HATU (1.5), DIPEA (2.5 equiv), DMF, rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt.</p></p></figure><div class="NLM_p">The procedure for synthesizing C-5 alkyl-amino substituted quinazolinones commenced from the Pd-catalyzed cyanation of C-5 bromide in <b>15a</b>,<b>b</b> and subsequent hydrogenation of the nitrile group to form compounds <b>30a</b>,<b>b</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). The free amine in <b>30a</b> was reacted with ethyl 2-bromothiazole-4-carboxylate, <b>31</b> to yield <b>32</b>, which was converted to final compound <b>33</b> using a similar sequence of steps as described in previous schemes. For target compounds <b>37</b>, <b>40a</b>,<b>b</b>, and <b>43</b>, compound <b>29a</b> was subjected to Boc-deprotection and nucleophilic aromatic substitution on purine <b>8</b> to afford <b>34</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Hydrogenation of the nitrile group in <b>34</b> to free amine was utilized in synthesizing a variety of linkers using (a) amide-coupling with <b>35</b>, (b) reductive amination with <b>38a</b>,<b>b</b>, and (c) substitution with <b>41</b>, to form compounds <b>36</b>, <b>39a</b>,<b>b</b>, and <b>42</b>. Subsequent conversion of alkyl esters to hydroxamic acids and removal of the THP protecting groups afforded the target compounds, <b>37</b>, <b>40a</b>,<b>b</b>, and <b>43</b>.</div><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0017.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of C-5 Alkyl-amino Substituted Quinazolinones (<b>33</b>, <b>37</b>, <b>40</b>, and <b>43</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub> (0.05 equiv), Zn(CN)<sub>2</sub> (1.2 equiv), DMF, 100 °C, 16 h; (b) Raney Ni, H<sub>2</sub>, MeOH, 20 h; (c) <b>31</b> (2.0 equiv) DIPEA (4.0 equiv), DMF, 180 °C, MW, 30 min; (d) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, MW, 100 °C; (e) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (f) <b>35</b> (1.5 equiv), HATU (1.2 equiv), DIPEA (3.0 equiv), DMF, 16 h, rt; (g) <b>38</b> (1 equiv), cat. AcOH, DCE, 2 h, then NaBH(OAC)<sub>3</sub> (3.0 equiv), 2 h; (h) <b>41</b> (2.0 equiv), triethylamine (4.0 equiv), EtOH, 100 °C, MW, 3 h.</p></p></figure><div class="NLM_p">C-5 alkyl-substituted quinazolinones with pyrimidine kinase hinge binding groups (<b>46a</b>–<b>d</b>) were readily synthesized from <b>17b</b>,<b>c</b> in four steps as shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The alkyne group in <b>17b</b>,<b>c</b> was hydrogenated using Raney-Ni followed by Boc-deprotection and S<sub>N</sub>Ar on chloropyrimidines <b>44a</b>–<b>c</b> to form compounds <b>45a</b>–<b>d</b>. Subsequent conversion of methyl esters in <b>45a</b>–<b>d</b> to hydroxamic acids using hydroxylamine afforded the target compounds <b>46a</b>–<b>d</b>.</div><figure id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0018.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of C-5 Substituted Quinazolinones with Pyrimidine Hinge Binding Groups (<b>46a</b>–<b>d</b> and <b>48a</b>–<b>o</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) 10 wt % Pd/C, H<sub>2</sub> balloon, EtOAc, 20 h, rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (iii) <b>44</b> (1.5–2.0 equiv), DIPEA (3.0–4.0 equiv), <i>n</i>-butanol, 2–3 h, MW, 130 °C, 38–88% (3 steps); (b) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt; 36–82%.</p></p></figure><div class="NLM_p">C-5 amino-alkyl substituted quinazolinones (<b>48a</b>–<b>o</b>) were synthesized in a similar fashion from compounds <b>26b</b>,<b>d</b>,<b>g</b>,<b>i</b> by attaching the pyrimidine kinase binding moieties using S<sub>N</sub>Ar reactions on <b>44a</b>,<b>b</b>,<b>d</b>–<b>l</b> and converting methyl esters in <b>47a</b>–<b>o</b> to hydroxamic acids.</div><div class="NLM_p">For synthesis of quinazolinones <b>51</b> and <b>54</b>, the nitrile group in compound <b>29b</b> was hydrogenated using Raney-Ni to generate free amine, which was either coupled to 4-(methoxycarbonyl)benzoic acid <b>35</b> using HATU/DIPEA or subjected to S<sub>N</sub>Ar reaction with ethyl 2-chloropyrimidine-5-carboxylate <b>41</b> to form compounds <b>49</b> and <b>52</b>, respectively (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>). Subsequent attachment of the pyrimidine kinase hinge binding group and transformation of alkyl esters to hydroxamic acids yielded the target compounds <b>51</b> and <b>54</b>.</div><figure id="sch6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0019.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of C-5 Alkyl-amino Substituted Quinazolinones (<b>51</b> and <b>54</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Raney Ni, H<sub>2</sub>, MeOH, 20 h; (b) <b>35</b> (1.5 equiv), HATU (1.2 equiv), DIPEA (3.0 equiv), DMF, 16 h; (c) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>44a</b> (1.5 equiv), DIPEA (3.0 equiv), <i>n</i>-butanol, 2–5 h, MW, 130–150 °C; (d) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt; (e) <b>41</b> (1.5 equiv), DIPEA (3.0 equiv), <i>n</i>-butanol, 2 h, MW, 130 °C.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> PI3K and HDAC: Enzyme Inhibition Activity and SAR</h3><div class="NLM_p">The dual inhibitors were screened against PI3K and HDAC enzymes at Reaction Biology Corporation, Malvern, PA, USA. Initially, we compared the enzyme inhibitory activities of 4-substituted quinazolines vs 5-substituted quinazolinones by screening against PI3Kδ and HDAC1–11 enzymes (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In general, 5-substituted quinazolinones were consistently found to be more potent than the 4-substituted quinazolines for PI3K and HDAC enzyme inhibition. As shown for a few examples in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, 5-substituted quinazolinones having 4- and 5-carbon alkylene linkers (<b>19a</b>, <b>19b</b>) and amino-propylene linker (<b>28a</b>), showed more potent PI3K/HDAC activities than their 4-substituted quinazoline counterparts (<b>10a</b>, <b>10b</b>, <b>14a</b>). Additionally, the 5-substituted compounds showed high potency and selectivity for HDAC6 inhibition, leading us to focus only on 5-substituted quinazolinones.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Enzyme Inhibitory Activities of 4-Substituted Quinazolines (<b>10a</b>,<b>b</b>, <b>14a</b>) vs 5-Substituted Quinazolinones (<b>19a</b>,<b>b</b>, <b>28a</b>)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0020.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="12" align="center">IC<sub>50</sub> (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="11" align="center">HDACs</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">PI3Kδ</th><th class="colsep0 rowsep0" align="center">1</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">3</th><th class="colsep0 rowsep0" align="center">4</th><th class="colsep0 rowsep0" align="center">5</th><th class="colsep0 rowsep0" align="center">6</th><th class="colsep0 rowsep0" align="center">7</th><th class="colsep0 rowsep0" align="center">8</th><th class="colsep0 rowsep0" align="center">9</th><th class="colsep0 rowsep0" align="center">10</th><th class="colsep0 rowsep0" align="center">11</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="center">49</td><td class="colsep0 rowsep0" align="center">5846</td><td class="colsep0 rowsep0" align="center">18190</td><td class="colsep0 rowsep0" align="center">7075</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">2169</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">1937</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">33730</td><td class="colsep0 rowsep0" align="center">23030</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19a</b></td><td class="colsep0 rowsep0" align="center"><5</td><td class="colsep0 rowsep0" align="center">429</td><td class="colsep0 rowsep0" align="center">1231</td><td class="colsep0 rowsep0" align="center">427</td><td class="colsep0 rowsep0" align="center">70910</td><td class="colsep0 rowsep0" align="center">23610</td><td class="colsep0 rowsep0" align="center">36</td><td class="colsep0 rowsep0" align="center">90810</td><td class="colsep0 rowsep0" align="center">385</td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center">2073</td><td class="colsep0 rowsep0" align="center">2089</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10b</b></td><td class="colsep0 rowsep0" align="center">68</td><td class="colsep0 rowsep0" align="center">5390</td><td class="colsep0 rowsep0" align="center">14100</td><td class="colsep0 rowsep0" align="center">10600</td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center">43700</td><td class="colsep0 rowsep0" align="center">39</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">353</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">16900</td><td class="colsep0 rowsep0" align="center">9819</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="center">0.2</td><td class="colsep0 rowsep0" align="center">388</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">11290</td><td class="colsep0 rowsep0" align="center">8106</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center">24400</td><td class="colsep0 rowsep0" align="center">253</td><td class="colsep0 rowsep0" align="center">34850</td><td class="colsep0 rowsep0" align="center">1502</td><td class="colsep0 rowsep0" align="center">2155</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14a</b></td><td class="colsep0 rowsep0" align="center">520</td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center">431</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">6194</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">74140</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28a</b></td><td class="colsep0 rowsep0" align="center">46</td><td class="colsep0 rowsep0" align="center">1575</td><td class="colsep0 rowsep0" align="center">3247</td><td class="colsep0 rowsep0" align="center">1031</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">9087</td><td class="colsep0 rowsep0" align="center">37</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">1758</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">3362</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PI-103</td><td class="colsep0 rowsep0" align="center">3</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TSA</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">9</td><td class="colsep0 rowsep0" align="center">25</td><td class="colsep0 rowsep0" align="center">14</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">3</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">371</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">61</td><td class="colsep0 rowsep0" align="center">3470</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TMP 269</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">281</td><td class="colsep0 rowsep0" align="center">333</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">108</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">49</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Compounds were tested in singlet 10-dose IC<sub>50</sub> mode with 3-fold serial dilution starting at 1 μM for PI3Kδ and 10 μM for HDAC1–11 enzymes. Empty cells indicate no inhibition or compound activity that could not be fit to an IC<sub>50</sub> curve. NT = Compound not tested against enzyme. PI-103 was used as a control compound for PI3Kδ, whereas trichostatin A (TSA) and TMP 269 were used as control compounds for HDAC enzymes.</p></div></div></div><div class="NLM_p">Next, we explored the linker region on 5-substituted quinazolinones by incorporating a variety of aryl, heteroaryl, and amino-aryl groups on the PI3K core. Initially we screened against PI3K isoforms (α, β, γ, δ) and HDAC isoforms (1, 2, 3, 6, 8, and 10) at a single dose (1 μM for PI3Ks and 10 μM for HDAC enzymes). As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, dual inhibitors with a variety of linkers were synthesized and tested to establish SAR. Dual inhibitors with aryl (<b>24a</b>) and heteroaryl linkers such as benzothiazole (<b>24c</b>) showed high potency for PI3Kδ and HDAC6, 8, whereas benzyl linker (<b>24b</b>) only showed high PI3Kδ activity. The use of amino-alkyl heterocycle linkers such as 3-(methylamino) pyridine (<b>28f</b>) and 2-(methylamino) thiazole (<b>33</b>) showed good inhibition for PI3K and HDAC6, but the corresponding alkyl-amino furan (<b>28h</b>) lost its potency against both enzymes. Linkers containing basic amines (<b>40a</b>,<b>b</b>) exhibited modest to high HDAC6 activity (80% for <b>40a</b>, 99% for <b>40b</b>) but were inactive against PI3Ks. Select dual inhibitors that showed high single-point potency (% inhibition) in both PI3Kδ and HDAC6 were subsequently tested in 10-dose IC<sub>50</sub> titration against PI3Ks (α, β, γ, δ) and HDAC 1–11 isozymes.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. PI3K/HDAC Enzyme Inhibition (%) Activities of 5-Substituted Quinazolinones with Purine as the Hinge Binding group<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0021.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">inhibition (%) at 1 μM</th><th class="rowsep1 colsep0" colspan="6" align="center">inhibition (%) at 10 μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">PI3K</th><th class="rowsep1 colsep0" colspan="6" align="center">HDAC</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">α</th><th class="colsep0 rowsep0" align="center">β</th><th class="colsep0 rowsep0" align="center">γ</th><th class="colsep0 rowsep0" align="center">δ</th><th class="colsep0 rowsep0" align="center">1</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">3</th><th class="colsep0 rowsep0" align="center">6</th><th class="colsep0 rowsep0" align="center">8</th><th class="colsep0 rowsep0" align="center">10</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24a</b></td><td class="colsep0 rowsep0" align="center">84.5 ± 0.3</td><td class="colsep0 rowsep0" align="center">44.0 ± 0.6</td><td class="colsep0 rowsep0" align="center">81.6 ± 0.2</td><td class="colsep0 rowsep0" align="center">94.7 ± 0.02</td><td class="colsep0 rowsep0" align="center">63.3 ± 3.2</td><td class="colsep0 rowsep0" align="center">52.3 ± 0.4</td><td class="colsep0 rowsep0" align="center">48.2 ± 0.9</td><td class="colsep0 rowsep0" align="center">93.3 ± 0.3</td><td class="colsep0 rowsep0" align="center">99.1 ± 0.1</td><td class="colsep0 rowsep0" align="center">54.0 ± 0.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24b</b></td><td class="colsep0 rowsep0" align="center">64.1 ± 1.0</td><td class="colsep0 rowsep0" align="center">37.5 ± 2.4</td><td class="colsep0 rowsep0" align="center">79.1 ± 0.02</td><td class="colsep0 rowsep0" align="center">95.8 ± 1.3</td><td class="colsep0 rowsep0" align="center">–0.9 ± 2.8</td><td class="colsep0 rowsep0" align="center">11.5 ± 0.1</td><td class="colsep0 rowsep0" align="center">–13.4 ± 0.3</td><td class="colsep0 rowsep0" align="center">70.7 ± 2.9</td><td class="colsep0 rowsep0" align="center">70.1 ± 0.6</td><td class="colsep0 rowsep0" align="center">20.0 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24c</b></td><td class="colsep0 rowsep0" align="center">84.7 ± 0.2</td><td class="colsep0 rowsep0" align="center">38.6 ± 1.4</td><td class="colsep0 rowsep0" align="center">88.0 ± 0.6</td><td class="colsep0 rowsep0" align="center">92.6 ± 0.6</td><td class="colsep0 rowsep0" align="center">50.0 ± 2.3</td><td class="colsep0 rowsep0" align="center">40.5 ± 1.7</td><td class="colsep0 rowsep0" align="center">20.9 ± 7.2</td><td class="colsep0 rowsep0" align="center">95.6 ± 0.4</td><td class="colsep0 rowsep0" align="center">98.2 ± 0.3</td><td class="colsep0 rowsep0" align="center">33.1 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28f</b></td><td class="colsep0 rowsep0" align="center">45.8 ± 2.7</td><td class="colsep0 rowsep0" align="center">4.4 ± 4.3</td><td class="colsep0 rowsep0" align="center">39.9 ± 5.0</td><td class="colsep0 rowsep0" align="center">88.3 ± 0.5</td><td class="colsep0 rowsep0" align="center">20.5 ± 0.3</td><td class="colsep0 rowsep0" align="center">16.3 ± 0.9</td><td class="colsep0 rowsep0" align="center">14.3 ± 1.6</td><td class="colsep0 rowsep0" align="center">90.5 ± 0.7</td><td class="colsep0 rowsep0" align="center">87.2 ± 0.1</td><td class="colsep0 rowsep0" align="center">–7.0 ± 1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28h</b></td><td class="colsep0 rowsep0" align="center">18.1 ± 0.4</td><td class="colsep0 rowsep0" align="center">3.8 ± 1.3</td><td class="colsep0 rowsep0" align="center">48.0 ± 1.3</td><td class="colsep0 rowsep0" align="center">61.2 ± 1.0</td><td class="colsep0 rowsep0" align="center">18.3 ± 6.7</td><td class="colsep0 rowsep0" align="center">28.0 ± 3.6</td><td class="colsep0 rowsep0" align="center">–6.3 ± 3.2</td><td class="colsep0 rowsep0" align="center">72.3 ± 0.03</td><td class="colsep0 rowsep0" align="center">68.4 ± 3.3</td><td class="colsep0 rowsep0" align="center">17.0 ± 2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="center">45.6 ± 1.0</td><td class="colsep0 rowsep0" align="center">6.8 ± 0.7</td><td class="colsep0 rowsep0" align="center">59.9 ± 0.4</td><td class="colsep0 rowsep0" align="center">95.2 ± 0.1</td><td class="colsep0 rowsep0" align="center">5.2 ± 3.9</td><td class="colsep0 rowsep0" align="center">4.1 ± 1.1</td><td class="colsep0 rowsep0" align="center">2.0 ± 3.5</td><td class="colsep0 rowsep0" align="center">89.1 ± 0.1</td><td class="colsep0 rowsep0" align="center">58.4 ± 0.8</td><td class="colsep0 rowsep0" align="center">–4.6 ± 4.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40a</b></td><td class="colsep0 rowsep0" align="center">1.8 ± 0.7</td><td class="colsep0 rowsep0" align="center">5.3 ± 0.3</td><td class="colsep0 rowsep0" align="center">–12.2 ± 0.7</td><td class="colsep0 rowsep0" align="center">56.0 ± 1.0</td><td class="colsep0 rowsep0" align="center">24.3 ± 3.3</td><td class="colsep0 rowsep0" align="center">30.4 ± 3.8</td><td class="colsep0 rowsep0" align="center">–4.3 ± 1.1</td><td class="colsep0 rowsep0" align="center">79.9 ± 0.7</td><td class="colsep0 rowsep0" align="center">54.5 ± 0.3</td><td class="colsep0 rowsep0" align="center">26.4 ± 0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40b</b></td><td class="colsep0 rowsep0" align="center">–6.8 ± 0.2</td><td class="colsep0 rowsep0" align="center">–6.0 ± 0.3</td><td class="colsep0 rowsep0" align="center">–9.1 ± 3.4</td><td class="colsep0 rowsep0" align="center">20.5 ± 0.7</td><td class="colsep0 rowsep0" align="center">65.6 ± 2.0</td><td class="colsep0 rowsep0" align="center">57.2 ± 5.7</td><td class="colsep0 rowsep0" align="center">19.1 ± 1.4</td><td class="colsep0 rowsep0" align="center">99.2 ± 0.7</td><td class="colsep0 rowsep0" align="center">93.5 ± 0.3</td><td class="colsep0 rowsep0" align="center">57.2 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PI-103 (IC<sub>50</sub>)</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="center">63</td><td class="colsep0 rowsep0" align="center">7</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TSA (IC<sub>50</sub>)</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">18</td><td class="colsep0 rowsep0" align="center">48</td><td class="colsep0 rowsep0" align="center">40</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">519</td><td class="colsep0 rowsep0" align="center">63</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Compounds were tested in single dose duplicate at 1 μM for PI3Ks and 10 μM for HDAC enzymes. NT = Compound not tested against enzyme. PI-103 was used as a control compound for PI3Kδ, whereas trichostatin A (TSA) was used as control compounds for HDAC enzymes.</p></div></div></div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a> shows the PI3K and HDAC enzyme potencies of the 5-substituted quinazolinone with purine hinge binding group. The dual inhibitor containing the pentyl group as linker (<b>19b</b>) exhibited high activity and selectivity for both PI3Kδ (IC<sub>50</sub> = 0.2 nM, >210-fold selectivity over PI3Kα,β,γ) and HDAC6 (IC<sub>50</sub> = 13 nM, >65 fold-selectivity over HDAC1–5,7–11). Substitution with a homobenzyl (<b>19c</b>) or phenyl linker (<b>24a</b>) showed high PI3Kδ inhibition but poor activity against HDAC6. Swapping the homobenzylic linker with benzylamino group (<b>28d</b>) restored HDAC6 activity and selectivity along with good PI3Kδ activity. Interestingly, substituting the benzylamino linker in <b>28d</b> with a 3-methylbenzylamino group in <b>28e</b> maintained the PI3Kδ activity but completely abolished the HDAC activity. A similar trend in PI3Kδ/HDAC6 potency was observed with 3-(aminomethyl)pyridyl linker in <b>28f</b>, where HDAC6 activity was lost, presumably due to hydrogen-bonding interaction of the pyridyl N atom with O–H group on hydroxamic acid, thereby impacting the chelation of hydroxamic acid with Zn in the HDAC6 enzyme (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). To further support this hydrogen bonding rationale, we designed and synthesized compound <b>28g</b> with a 2-(aminomethyl)pyridyl linker, where the possibility of H-bonding interaction with the pyridyl N atom and −OH group of hydroxamic acid was eliminated. Gratifyingly, the HDAC6 potency was restored in <b>28g</b> (IC<sub>50</sub> = 9 nM) when compared with <b>28f</b> (IC<sub>50</sub> = 169 nM) (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. PI3K/HDAC Enzyme Inhibitory Activities of 5-Substituted Quinazolinones with Purine Hinge Binding Group<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0022.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="15" align="center">IC<sub>50</sub> (nM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">PI3K</th><th class="rowsep1 colsep0" colspan="11" align="center">HDAC</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">α</th><th class="colsep0 rowsep0" align="center">β</th><th class="colsep0 rowsep0" align="center">γ</th><th class="colsep0 rowsep0" align="center">δ</th><th class="colsep0 rowsep0" align="center">1</th><th class="colsep0 rowsep0" align="center">2</th><th class="colsep0 rowsep0" align="center">3</th><th class="colsep0 rowsep0" align="center">4</th><th class="colsep0 rowsep0" align="center">5</th><th class="colsep0 rowsep0" align="center">6</th><th class="colsep0 rowsep0" align="center">7</th><th class="colsep0 rowsep0" align="center">8</th><th class="colsep0 rowsep0" align="center">9</th><th class="colsep0 rowsep0" align="center">10</th><th class="colsep0 rowsep0" align="center">11</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="center">122</td><td class="colsep0 rowsep0" align="center">286</td><td class="colsep0 rowsep0" align="center">43</td><td class="colsep0 rowsep0" align="center">0.2</td><td class="colsep0 rowsep0" align="center">1390</td><td class="colsep0 rowsep0" align="center">3030</td><td class="colsep0 rowsep0" align="center">2400</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">8060</td><td class="colsep0 rowsep0" align="center">13</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">856</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">1240</td><td class="colsep0 rowsep0" align="center">1840</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19c</b></td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">>10<sup>5</sup></td><td class="colsep0 rowsep0" align="center">889</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">3840</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">9000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24a</b></td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">0.7</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">362</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">71</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28d</b></td><td class="colsep0 rowsep0" align="center">254</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">103</td><td class="colsep0 rowsep0" align="center">47</td><td class="colsep0 rowsep0" align="center">3925</td><td class="colsep0 rowsep0" align="center">7173</td><td class="colsep0 rowsep0" align="center">6699</td><td class="colsep0 rowsep0" align="center">1751</td><td class="colsep0 rowsep0" align="center">2144</td><td class="colsep0 rowsep0" align="center">5</td><td class="colsep0 rowsep0" align="center">229</td><td class="colsep0 rowsep0" align="center">250</td><td class="colsep0 rowsep0" align="center">3335</td><td class="colsep0 rowsep0" align="center">14700</td><td class="colsep0 rowsep0" align="center">324</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28e</b></td><td class="colsep0 rowsep0" align="center">517</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">203</td><td class="colsep0 rowsep0" align="center">41</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">2699</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">6922</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">NT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28f</b></td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">43</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">169</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28g</b></td><td class="colsep0 rowsep0" align="center">392</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">135</td><td class="colsep0 rowsep0" align="center">26</td><td class="colsep0 rowsep0" align="center">6067</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">10990</td><td class="colsep0 rowsep0" align="center">381</td><td class="colsep0 rowsep0" align="center">439</td><td class="colsep0 rowsep0" align="center">9</td><td class="colsep0 rowsep0" align="center">104</td><td class="colsep0 rowsep0" align="center">204</td><td class="colsep0 rowsep0" align="center">922</td><td class="colsep0 rowsep0" align="center">9996</td><td class="colsep0 rowsep0" align="center">2255</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28i</b></td><td class="colsep0 rowsep0" align="center">387</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">162</td><td class="colsep0 rowsep0" align="center">81</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="center">266</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">234</td><td class="colsep0 rowsep0" align="center">0.6</td><td class="colsep0 rowsep0" align="center">8434</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">10</td><td class="colsep0 rowsep0" align="center">23120</td><td class="colsep0 rowsep0" align="center">138</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">19890</td><td class="colsep0 rowsep0" align="center">1778</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="center">422</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">995</td><td class="colsep0 rowsep0" align="center">49</td><td class="colsep0 rowsep0" align="center">67870</td><td class="colsep0 rowsep0" align="center">25480</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">1428</td><td class="colsep0 rowsep0" align="center">3827</td><td class="colsep0 rowsep0" align="center">6</td><td class="colsep0 rowsep0" align="center">60</td><td class="colsep0 rowsep0" align="center">320</td><td class="colsep0 rowsep0" align="center">5349</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PI-103</td><td class="colsep0 rowsep0" align="center">4</td><td class="colsep0 rowsep0" align="center">7</td><td class="colsep0 rowsep0" align="center">81</td><td class="colsep0 rowsep0" align="center">8</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TSA</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">10</td><td class="colsep0 rowsep0" align="center">34</td><td class="colsep0 rowsep0" align="center">19</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">2</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">445</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">45</td><td class="colsep0 rowsep0" align="center">4826</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TMP 269</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">244</td><td class="colsep0 rowsep0" align="center">278</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">62</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">19</td><td class="colsep0 rowsep0" align="center">NT</td><td class="colsep0 rowsep0" align="center">NT</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Compounds were tested in singlet 10-dose IC<sub>50</sub> mode with 3-fold serial dilution starting at 1 μM for PI3Kα, β, γ, δ, and 10 μM for HDAC1–11 enzymes. Empty cells indicate no inhibition or compound activity that could not be fit to an IC<sub>50</sub> curve. NT = Compound not tested against enzyme. PI-103 was used as a control compound for PI3Ks, whereas trichostatin A (TSA) and TMP 269 were used as control compounds for HDAC enzymes.</p></div></div></div><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Hydrogen bonding rationale for HDAC activity comparison in <b>28f</b> and <b>28g</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Substituting the hydroxamic acid in <b>28d</b> with <i>N</i>-methyl hydroxamic acid as zinc-binding group in <b>28i</b> led to a 2-fold drop in PI3Kδ potency and complete loss in activity against all HDACs tested. The dual inhibitor containing <i>N</i>-methylbenzamide (<b>37</b>) as the linker resulted in high activity and selectivity in both PI3Kδ and HDAC6. Compound <b>43</b> with a 2-(aminomethyl)-pyrimidinyl linker also showed good potency and selectivity against PI3Kδ and HDAC6 inhibition.</div><div class="NLM_p">Despite high activity and selectivity observed with several 5-substituted quinazolinones against PI3Kδ/HDAC6, most did not show expected antiproliferative/cell kill activities when tested against the NCI-60 panel (see section titled, “<a class="ref internalNav" href="#sec2_6" aria-label="NCI-60 Human Tumor Cell Lines Screen">NCI-60 Human Tumor Cell Lines Screen</a>”). We hypothesized that their poor cellular potency might be due to low permeability (predicted by PAMPA permeability)<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> generally observed with these dual inhibitors. <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> shows the PAMPA permeability values of selected 5-substituted quinazolinones with a purine hinge binding group.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. PAMPA Permeability Data of Selected 5-Substituted Quinazolinones with Purine Hinge Binding Group<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">PAMPA permeability<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="center"><1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24a</b></td><td class="colsep0 rowsep0" align="center">1.2 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24c</b></td><td class="colsep0 rowsep0" align="center">2.4 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28d</b></td><td class="colsep0 rowsep0" align="center">6.4 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28e</b></td><td class="colsep0 rowsep0" align="center">2.6 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="center">1.3 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="center"><1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40b</b></td><td class="colsep0 rowsep0" align="center"><1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>43</b></td><td class="colsep0 rowsep0" align="center"><1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Average of two runs ± standard deviation (SD). PAMPA permeability values were determined at pH 7.4 (10<sup>–6</sup> cm/s).</p></div><div class="footnote" id="t4fn2"><sup><sup>b</sup></sup><p class="last">No SD values provided if all PAMPA permeability values are less than 1.</p></div></div></div><div class="NLM_p">To improve upon the cellular potencies of 5-substituted quinazolinones, we investigated a new series of dual inhibitors by substituting the purine kinase hinge binding moiety with amino-pyrimidine groups.<a onclick="showRef(event, 'ref52 ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref52 ref54 ref55 ref56">(52,54−56)</a><a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> shows the PI3K and HDAC enzyme inhibition data for selected 5-substituted quinazolinones with amino-pyrimidine hinge binding groups. Compounds <b>46a</b>,<b>b</b> with 2,4-diamino-6-pyrimidine-5-carbonitrile and 2-amino-6-methylpyrimidine-5-carbonitrile hinge binding groups, respectively, showed excellent PI3Kδ activity (IC<sub>50</sub> = 0.2–0.7 nM), but a weaker HDAC6 potency (IC<sub>50</sub> = 28–69 nM), when compared with compound <b>19b</b> from the purine series (PI3Kδ IC<sub>50</sub> = 0.2 nM, HDAC6 IC<sub>50</sub> = 13 nM). Compound <b>46d</b> with a homobenzyl linker and 2,4-diamino-6-pyrimidine-5-carbonitrile hinge binding group was highly active against PI3Kδ (IC<sub>50</sub> = 0.2 nM) but showed a modest potency for HDAC6 (IC<sub>50</sub> = 77 nM). Dual inhibitors with amino-benzyl/amino-methylpyridyl linkers were explored with a variety of amino-pyrimidine kinase hinge binders to establish SAR. Replacement of the pentyl linker in <b>46b</b> with aminobutyl linker in <b>48a</b> with the same hinge binding group showed high potencies for PI3Kγ,δ but no activity against HDACs. Compound <b>48b</b> with 2,4-diamino-6-pyrimidine-5-carbonitrile hinge binder showed moderate potency for PI3Kγ,δ but high potency and selectivity for HDCA6. The PI3K potency was restored in <b>48c</b> using a 2-amino-6-methylpyrimidine-5-carbonitrile hinge binding group while maintaining good potency for HDAC6. Removing the nitrile group in <b>48c</b> gave compound <b>48d</b>, which was inactive against PI3K. Dual inhibitors with amino-pyrimidines lacking the 2-amino group in the hinge region (<b>48e</b>,<b>f</b>,<b>g</b>) showed high HDAC6 activity (IC<sub>50</sub> = 3–4 nM) but poor PI3Kδ potency (PI3Kδ IC<sub>50</sub> = 60 nM for <b>48e</b>, 133 nM for <b>48f</b>, 189 nM for <b>48g</b>). Similar detrimental effects on PI3K potencies were observed when the C-5 nitrile group on the aminopyrimidine was changed to Cl (<b>48h</b>, <b>48j</b>) and the C-2 amino group was changed to Me (<b>48i</b>). Replacement of the C-6 methyl/amino group in the kinase hinge to CF<sub>3</sub>, cyclopropyl, and CF<sub>2</sub>H, respectively, were also not good for PI3K activity (<b>48k</b>, <b>48l</b>, <b>48m</b>). Overall, only 2-amino-6-methylpyrimidine-5-carbonitrile and 2,4-diamino-6-pyrimidine-5-carbonitrile hinge binding groups were revealed as optimal hinge binders for achieving high potencies for both PI3K and HDAC enzymes. While HDAC6 activity and selectivity obtained by using these amino-pyrimidine hinge binders were similar to their purine hinge binder counterparts, less PI3K selectivity was observed with aminopyrimidine series, with almost equipotent activities against PI3Kγ and PI3Kδ in many examples. Swapping the ethyl group at R<sup>1</sup> with a methyl group was tolerated well to obtain compounds (<b>48n</b>, <b>48o</b>) with single-digit nM activities against PI3Kγ,δ and HDAC6. Compounds having 2,4-diamino-6-pyrimidine-5-carbonitrile as hinge binder with <i>N</i>-methylbenzamide and 2-(aminomethyl)-pyrimidine linkers (<b>51</b> and <b>54</b>) also showed high PI3K/HDAC6 activity. In general, dual inhibitors with aminopyrimidine hinge binders showed comparatively better PAMPA permeability (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>) than their purine hinge binding counterparts. Additionally, 5-substituted quinazolinones with amino-pyrimidine hinge binders showed potent cell activities when tested in the NCI-60 panel (see section <a class="ref internalNav" href="#sec6_3" aria-label="NCI-60 assays">NCI-60 assays</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Supporting Information</a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. PI3K/HDAC Enzyme Inhibitory Activities of 5-Substituted Quinazolinones with Amino-pyrimidine Hinge Binders<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0023.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0024.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0025.gif" alt="" id="GRAPHIC-d7e3352-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Compounds were tested in singlet 10-dose IC<sub>50</sub> mode with 3-fold serial dilution starting at 1 μM for PI3Kα, β, γ, δ, and 10 μM for HDAC1–11 enzymes. Empty cells indicate no inhibition or compound activity that could not be fit to an IC<sub>50</sub> curve. NT = Compound not tested against enzyme. PI-103 was used as a control compound for PI3Ks, whereas trichostatin A (TSA) and TMP 269 were used as control compounds for HDAC enzymes.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Selectivity of HDAC6 inhibition over other HDAC isoforms (1, 2, 3, 4, 5, 7, 8, 9, 10, and 11). Please see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Table S1</a> in Supporting Information for full IC<sub>50</sub> data against HDAC1–11.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. PAMPA Permeability Data of Selected 5-Substituted Quinazolinones with Amino-pyrimidine Hinge Binding Groups</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">PAMPA permeability<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a>,<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46b</b></td><td class="colsep0 rowsep0" align="char" char="±">25.3 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46c</b></td><td class="colsep0 rowsep0" align="char" char="±">22.7 ± 7.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48b</b></td><td class="colsep0 rowsep0" align="char" char="±">4.9 ± 0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48c</b></td><td class="colsep0 rowsep0" align="char" char="±">12.0 ± 0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48d</b></td><td class="colsep0 rowsep0" align="char" char="±">15.7 ± 5.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48f</b></td><td class="colsep0 rowsep0" align="char" char="±">50.5 ± 12.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48g</b></td><td class="colsep0 rowsep0" align="char" char="±">77.5 ± 11.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48h</b></td><td class="colsep0 rowsep0" align="char" char="±">30.2 ± 3.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48i</b></td><td class="colsep0 rowsep0" align="char" char="±">12.4 ± 1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48j</b></td><td class="colsep0 rowsep0" align="char" char="±">149.1 ± 2.1</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">Average of two runs ± standard deviation (SD). PAMPA permeability values were determined at pH 7.4 (10<sup>–6</sup> cm/s).</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">No SD values provided if all PAMPA permbeability values are less than 1.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Molecular Modeling: HDAC6 Binding and Selectivity</h3><div class="NLM_p">The dual inhibitor <b>19b</b> was docked into the solved crystal structure of hHDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.pdb) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The geometry of the protein around the <b>19b</b> hydroxamic acid headgroup mirrors that of Vorinostat (see <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The aliphatic linker chain, in similar fashion, traverses a hydrophobic region of the pocket. The cap group, which is composed of a quinazolinone ring system, on the other hand, forms a hydrogen bond to Ser568 via the quinazolinone carbonyl. This hydrogen bonding interaction on the rim of the pocket is distinctly different than that made by Vorinostat.</div><figure id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Compound <b>19b</b> bound to HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.pdb). (B) Variation in amino acid residues at positions 567 and 568 across different HDACs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> HDAC6 Selectivity</h4><div class="NLM_p">Using the procedure described in Methods, we identified Ser568 as the only residue on HDAC6 that could be responsible for the HDAC selectivity profile of <b>19b</b>. As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, Ser568 in HDAC6 forms a hydrogen bond with the quinazolinone carbonyl on <b>19b</b>. In the case of HDACs-1, 2, 3, and 8, Ser568 becomes Asp568 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). The negatively charged carboxyl group on Asp568 is a hydrogen bond acceptor and does not form a hydrogen bond with the quinazolinone carbonyl on <b>19b</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S1–S3</a>, Supporting Information). Rather, the Asp568 carboxyl repulses the quinazolinone carbonyl and swings away from the ligand. The nearby presence of Glu567 (HDAC1,2, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S1</a>, Supporting Information) and Asp567 (HDAC3, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S2</a>, Supporting Information) contributes to this process as these residues have negatively charged carboxyl side chains.</div><div class="NLM_p last">In the case of HDAC8, the residue adjacent to Asp568 is tyrosine, which contains a neutral phenyl side chain (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S3</a>, Supporting Information). As a result, the conformational shift in the carboxyl group on position 568 is less significant than for HDACs-1,2,3. Nevertheless, the H bond to the quinazolinone carbonyl is still broken. In the case of HDACs-4,5,7, and 9, threonine occupies position 568. Thr568 contains a hydrogen bond donor (hydroxyl), but the hydroxyl group does not interact with the quinazolinone carbonyl on the ligand. That is because the extra methyl group on the Thr568 side chain clashes with nearby Gly619 and prevents the OH group from obtaining a favorable hydrogen bonding position with respect to the quinazolinone carbonyl. In fact, it is easier for the Thr568 OH group to hydrogen bond with the carboxyl side chain of nearby Asp567 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S4</a>, Supporting Information). In the case of HDAC10, Ser568 becomes Ala568 resulting in an obvious loss of hydrogen bonding to the quinazolinone carbonyl on the ligand (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S5</a>, Supporting Information). In HDAC11, Ser568 becomes Asn568. Although asparagine has a hydrogen bond donating nitrogen group, its side chain is one carbon too long to adopt a reasonable orientation for hydrogen bonding to the quinazolinone carbonyl on the ligand (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S6</a>, Supporting Information). The <b>19b</b>-HDAC interaction energies relative to <b>19b</b>-HDAC6 were computed for each of the scenarios described above. These interaction energies are completely consistent with the idea that there is one less hydrogen bond between <b>19b</b> and HDACs-1, 2, 3, 4, 5, 7, 8, 9, 10,11 than there is between <b>19b</b> and HDAC6 hence, explaining the selectivity of <b>19b</b> to HDAC6.</div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Kinase Selectivity</h3><div class="NLM_p">The kinase selectivity of <b>48c</b> was investigated at 1 μM concentration against a panel of 412 kinases, including 24 atypical kinases and 17 lipid kinases at Reaction Biology Corporation, PA. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A, the dual inhibitor <b>48c</b> showed very high selectivity across the diverse kinase families, including atypical kinases, with less than 50% inhibition among all the kinases tested (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Table S2</a>, Supporting Information). In the case of the highly homologated lipid kinases (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B), compound <b>48c</b> showed between 74 and 83% inhibition for PI3Kα, β, and other PI3K mutants while maintaining the high selectivity for PI3Kγ and PI3Kδ (≥99% inhibition), as previously observed.</div><figure id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0007.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Kinase selectivity profile of compound <b>48c</b> (1 μM concentration) against 412 kinases. (B) Selectivity data (% Inhibition at 1 μM) of compound <b>48c</b> against lipid kinases. Kinase profiling was done at Reaction Biology, Malvern, PA, USA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> NCI-60 Human Tumor Cell Lines Screen</h3><div class="NLM_p">The dual inhibitors were tested against 60 human cancer cell lines at the National Cancer Institute (NCI-60 panel).<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The NCI-60 employs an initial singe-dose (10 μM compound concentration) growth inhibition screen against 60 cancer cell lines and selects only the most potent compounds for subsequent 5-point dose–response studies. As previously mentioned, most of the dual inhibitors (except <b>28d</b>) with the purine hinge binding group showed a poor growth inhibition profile (average growth inhibition (%) = 0–33%) across all the 60 cell lines tested and therefore were not evaluated in dose–response IC<sub>50</sub>. Compound <b>28d</b> with the amino-benzyl linker and purine hinge binder was the only dual inhibitor (from the purine series) selected for NCI-60–cell line dose–response and showed potent antiproliferative activities against multiple human cancer cell line types (<a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>). In contrast to the purine series, several dual inhibitors bearing amino-pyrimidine kinase hinge binding groups were evaluated for dose–response, with a few of them even showing better GI<sub>50</sub> than Idelalisib and SAHA.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Antiproliferative Activities of Selected Dual Inhibitors against Various Human Cancer Cell Lines at NCI-60<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="7" align="center" char=".">GI<sub>50</sub> (μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cancer type</th><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center" char="."><b>28d</b></th><th class="colsep0 rowsep0" align="center" char="."><b>46b</b></th><th class="colsep0 rowsep0" align="center" char="."><b>48b</b></th><th class="colsep0 rowsep0" align="center" char="."><b>48c</b></th><th class="colsep0 rowsep0" align="center" char="."><b>48o</b></th><th class="colsep0 rowsep0" align="center" char=".">Idelalisib</th><th class="colsep0 rowsep0" align="center" char=".">Vorinostat (SAHA)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">leukemia</td><td class="colsep0 rowsep0" align="left">CCRF-CEM</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.05</td><td class="colsep0 rowsep0" align="char" char=".">22.3</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SR</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">non-small cell lung cancer</td><td class="colsep0 rowsep0" align="left">HOP-62</td><td class="colsep0 rowsep0" align="char" char=".">11.8</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">2.4</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">>100.0</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HOP-92</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">14.1</td><td class="colsep0 rowsep0" align="char" char=".">2.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">colon cancer</td><td class="colsep0 rowsep0" align="left">HCT-2998</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">38.5</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">KM-12</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">4.0</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CNS cancer</td><td class="colsep0 rowsep0" align="left">SNB-75</td><td class="colsep0 rowsep0" align="char" char=".">3.0</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">U251</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">53.2</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">melanoma</td><td class="colsep0 rowsep0" align="left">LOX IMVI</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">33.5</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">M14</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">37.8</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ovarian cancer</td><td class="colsep0 rowsep0" align="left">IGROV1</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">OVCAR3</td><td class="colsep0 rowsep0" align="char" char=".">3.4</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">17.7</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">renal cancer</td><td class="colsep0 rowsep0" align="left">A498</td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">0.7</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">CAKI-1</td><td class="colsep0 rowsep0" align="char" char=".">14.6</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">20.5</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">RXF 393</td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="char" char=".">0.2</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="9" align="left">      </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">breast cancer</td><td class="colsep0 rowsep0" align="left">MDA-MB-231/ATCC</td><td class="colsep0 rowsep0" align="char" char=".">2.8</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">2.3</td><td class="colsep0 rowsep0" align="char" char=".">1.8</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">42.3</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">HS 578T</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">2.0</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td><td class="colsep0 rowsep0" align="char" char=".">6.0</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">T-47D</td><td class="colsep0 rowsep0" align="char" char=".">1.7</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">5.2</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Compounds were tested at NCI-60 panel in singlet 5-dose IC<sub>50</sub> mode with 10-fold serial dilution starting at a concentration of 100 μM.</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Dual Inhibitors Decrease Survival and Induce Cell Death in Diverse Types of AML Cells</h3><div class="NLM_p">Dual inhibitor <b>48c</b> was evaluated in cell viability assays against several myeloma and leukemia cell lines as shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A,B. Cell-viability was determined using the CellTiter-Glo (Promega) assay<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> or by the trypan blue exclusion assay.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Compound <b>48c</b> showed good inhibitory activity against FLT3-ITD mutant AML cell lines, MOLM-14 and MV411. Further studies were performed with <b>48c</b> in other types of AML cell lines. THP-1 is a spontaneously immortalized monocyte-like cell line derived from the peripheral blood of a childhood case of acute monocytic leukemia (M5 subtype).<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> HL-60 is a human promyelocytic leukemia cell line with a p53-null mutation.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Compound <b>48c</b> demonstrated antiproliferative activity against these cell lines with IC<sub>50</sub> values in the 2–5 μM range (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). Like MOLM-14 cells, MOLM-13 AML cells have one wild-type FLT3 and one FLT3-ITD allele.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Treatment of MOLM-13 cells with <b>48c</b> resulted in inhibition of growth that was concentration-dependent [<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B(2)]. As shown previously, loss of MOLM-13 cell survival was observed in response to treatment with the type II ATP competitive inhibitor HG-7-85-01 (HG) (data not shown).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Interestingly, MOLM-13 cells selected for resistance to HG were also sensitive to compound <b>48c</b> treatment [<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B(3)], indicating that targeting PI3Kδ/HDAC6 is effective against MOLM-13 cells resistant to FLT3 inhibitors. Dual inhibitors <b>48c</b>, <b>48n</b>, and <b>48o</b>, when screened alongside Idelalisib,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> SAHA<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and CUDC-907<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> (at their IC<sub>50</sub> range) against MV411 cell line displayed better potency than Idelalisib and SAHA but were less potent than the pan PI3K/HDAC inhibitor, CUDC-907, as expected (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>C). Importantly, dual inhibitor <b>48c</b> showed no cytotoxicity against normal human peripheral blood mononuclear cells (PBMCs) [<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>D], demonstrating its high selectivity in targeting leukemia cells over PBMCs. Additionally, dual inhibitors <b>48c</b>, <b>48n</b>, and <b>48o</b>, were nontoxic against NIH3T3 normal fibroblasts, whereas significant toxicity was observed with the pan-inhibitor CUDC-907, even at a low concentration of 50 nM (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>E). These studies clearly indicate that our isoform-selective dual inhibitors display potent antitumor activity with no toxicity in normal cells.</div><figure id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0008.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Cell viability assays were performed in duplicate with IC<sub>50</sub> values expressed as mean ± SD of two independent runs. (B) In vitro cell-kill activity of <b>48c</b>. (C) In vitro cell-kill activity of compounds <b>48n</b>, <b>48o</b>, <b>48c</b>, Idelalisib, SAHA, and CUDC-907 against MV411 cells. (D) PBMCs viability with <b>48c</b> treatment. (E) NIH3T3 cell viability upon treatment with compounds <b>48n</b>, <b>48o</b>, <b>48c</b>, Idelalisib, SAHA, and CUDC-907.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further define the activity of these dual inhibitors on AML cell lines, we determined the potential induction of cell death as measured by propidium iodide (PI) alone or PI/Annexin V staining followed by flow cytometry.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Dual inhibitors <b>48c</b>, <b>48n</b>, and <b>48o</b> induced cell death via necrosis in multiple AML cell lines as shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A–C. Both compounds <b>48n</b> and <b>48o</b> were able to induce >50% necrosis at 5 μM concentration in MOLM-14, MV4–11, and HL-60 tumor cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A,B). Similarly, dual inhibitor <b>48c</b> was highly active for inducing necrosis across several AML cell lines, including >60% necrosis in MOLM-13 FLT3 inhibitor-resistant cell line (MOLM-13 HG Res) (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>C). To determine whether primary AML cells also respond to compound <b>48c</b>, blasts from a patient with AML were grown in short-term culture. The AML primary cells were then treated with 5 μM compound <b>48c</b> for 72 h. The results demonstrate induction of substantial necrosis in these primary cells (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>D). Notably, dual inhibitors <b>48c</b>, <b>48n</b>, and <b>48o</b> displayed no significant necrotic activity against normal human HEK293 cells, whereas pan-inhibitors SAHA and CUDC-907, induced necrosis in these cells, again demonstrating the selective targeting of tumor cells over normal cells by our compounds.</div><figure id="fig9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0009.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Necrosis rate (%) of <b>48n</b> at 5 μM concentration. (B) Necrosis rate (%) of <b>48o</b> at 5 μM concentration. (C) Necrosis rate (%) of <b>48c</b> at 5 μM concentration. (D) Necrosis rate (%) of <b>48c</b> at 5 μM concentration against primary AML blasts. (E) Necrosis rate (%) of compounds <b>48n</b>, <b>48o</b>, <b>48c</b>, Idelalisib, SAHA, and CUDC-907 against HEK293 cells. Necrosis assays were performed in duplicate and % necrosis values are expressed as mean ± SD of two independent runs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Clonogenic Survival Assay</h3><div class="NLM_p">Encouraged by their growth inhibitory activities against multiple AML cell lines, dual inhibitors <b>48c</b> and <b>48o</b> were tested in an in vitro clonogenic assay for inhibition against MOLM-14 colony formation (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). MOLM-14 cells were seeded in 6-well plates for 24 h and then left untreated or treated with either 2.5 or 5 μM of <b>48c</b> and <b>48o</b>, respectively. After 48 h, the cells were washed and seeded in soft agar for the generation of colonies. After 10 days, colonies of >50 cells were counted. Plating efficiency of untreated MOLM-14 cells as colony formation is very high as multiple colonies (>50 cells) were seen. Gratifyingly, treatment of MOLM-14 cells with our dual inhibitors <b>48c</b> and <b>48o</b> resulted in significant decrease in clonogenic survival as shown below in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>.</div><figure id="fig10" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0010.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Inhibition of MOLM-14 colony formation by <b>48c</b> and <b>48o</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> PI3Kδ Inhibition Decreases Activation of Downstream Effectors</h3><div class="NLM_p">PI3K activates the AKT pathway. MOLM-14 cells were treated for 1, 6, and 24 h, with dual inhibitor <b>48c</b> (5.0 μM) and analyzed for p-AKT and total AKT levels by immunoblotting. In concert with the demonstration that dual inhibitors downregulate PI3K activation, treatment of MOLM-14 cells with compound <b>48c</b> was associated with complete inhibition of p-AKT with no change in the total AKT protein levels as shown in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>.</div><figure id="fig11" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0011.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Immunoblot analysis for PI3K inhibition in MOLM-14 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> CETSA</h3><div class="NLM_p">Next, we examined target engagement using the cellular thermal shift assay (CETSA).<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> Three lymphocytic cell lines were profiled by immunoblot for HDAC6 and PI3Kδ expression, and MV411 was selected for subsequent experiments based on detection of both targets (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">Figure S7</a>, Supporting Information). The CETSA melt profile (<i>T</i><sub>agg</sub>) of each protein was then measured (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>A,B), and 58 °C (HDAC6) and 52 °C (PI3Kδ) were selected for isothermal dose–response experiments. The control compounds showed the anticipated responses, with CAY10603 (HDAC6 inhibitor) stabilizing HDAC6 and Idelalisib (PI3Kδ inhibitor) stabilizing PI3Kδ. For the dual inhibitors, robust HDAC6 stabilization was observed for <b>48c</b>, with less stabilization observed for <b>46b</b> and HDAC6 only inhibitor <b>48d</b> (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>C). For PI3Kδ, PI3K inhibitor <b>45b</b>, and dual inhibitor, <b>48c</b> stabilized the protein, whereas <b>46b</b> and <b>48d</b> did not (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>D). These results indicate compound <b>48c</b> can engage both HDAC6 and PI3Kδ in the complex cellular environment.</div><figure id="fig12" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0012.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. CETSA analysis. (A) <i>T</i><sub>agg</sub> measurement for HDAC6 in MV411 cells, (B) <i>T</i><sub>agg</sub> measurement for PI3Kδ in MV411 cells, (C) melt profile for HDAC6, (D) melt profile for PI3Kδ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> In Vitro ADME Assays</h3><div class="NLM_p">Selected dual inhibitors were evaluated in multispecies (human, mouse, and rat) liver microsome/cytosol stability and Caco-2 permeability and efflux assays (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>). Compound <b>19b</b> showed good stability in human liver microsomes (HLM) but was poor in mouse (MLM) and rat liver microsomes (RLM). <b>19b</b> displayed low (<i>t</i><sub>1/2</sub> < 30 min) to moderate (30 < <i>t</i><sub>1/2</sub> < 60 min) metabolic stability in liver cytosol from the three species. Additionally, it was found to have low permeability in Caco-2 cell monolayer assay and an efflux ratio of 11, indicating the involvement of efflux transporters. Dual inhibitor <b>46b</b>, the pyrimidine hinge binding analogue of <b>19b</b>, showed similar stability in HLM and MLM assays as <b>19b</b> but was significantly better in RLM and cytosol assays. In contrast to compounds <b>19b</b> and <b>46b</b>, the dual inhibitors bearing amino-benzyl linker (<b>48b</b>, <b>48c</b>, <b>48n</b>, <b>48o</b>) displayed moderate to high stability in MLM and RLM assays but the HLM stability varied from low to high with compound <b>48n</b>, showing high stability across the three species tested. Additionally, these compounds show high cytosolic stability in human, mouse, and rat assays. The lead compound <b>48c</b> had low cell permeability (1.9 × 10<sup>–6</sup> cm/s), however, its efflux ratio of 3.5 was found to be on the lower side.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. In Vitro Metabolic Stability (Liver Microsome and Cytosol) and Caco-2 Permeability of Selected Dual PI3K/HDAC Inhibitors<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">microsomal stability (<i>t</i><sub>1/2</sub>, min)<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a>,<a class="ref internalNav" href="#t8fn4" aria-label="d">d</a></th><th class="rowsep1 colsep0" colspan="3" align="center">cytosolic stability (<i>t</i><sub>1/2</sub>, min)<a class="ref internalNav" href="#t8fn3" aria-label="c">c</a>,<a class="ref internalNav" href="#t8fn4" aria-label="d">d</a></th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char="±">HLM</th><th class="colsep0 rowsep0" align="center" char="±">MLM</th><th class="colsep0 rowsep0" align="center">RLM</th><th class="colsep0 rowsep0" align="center">HLC</th><th class="colsep0 rowsep0" align="center">MLC</th><th class="colsep0 rowsep0" align="center">RLC</th><th class="colsep0 rowsep0" align="center">Caco-2 efflux ratio (BA/AB)<a class="ref internalNav" href="#t8fn5" aria-label="e">e</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19b</b></td><td class="colsep0 rowsep0" align="char" char="±">93.1 ± 4.9</td><td class="colsep0 rowsep0" align="char" char="±">9.0 ± 0.03</td><td class="colsep0 rowsep0" align="center">19.1 ± 0.1</td><td class="colsep0 rowsep0" align="center">40.6 ± 7.5</td><td class="colsep0 rowsep0" align="center">27.7 ± 1.5</td><td class="colsep0 rowsep0" align="center">19.1 ± 0.1</td><td class="colsep0 rowsep0" align="center">9.5/0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>46b</b></td><td class="colsep0 rowsep0" align="char" char="±">91.5 ± 2.2</td><td class="colsep0 rowsep0" align="char" char="±">9.0 ± 0.4</td><td class="colsep0 rowsep0" align="center">49.8 ± 3.6</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48b</b></td><td class="colsep0 rowsep0" align="char" char="±">45.7 ± 15.1</td><td class="colsep0 rowsep0" align="char" char="±">60.4 ± 1.4</td><td class="colsep0 rowsep0" align="center">95.4 ± 4.5</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">52.1 ± 4.7</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">7.5/1.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48c</b></td><td class="colsep0 rowsep0" align="char" char="±">22.4 ± 2.2</td><td class="colsep0 rowsep0" align="char" char="±">51.7 ± 0.02</td><td class="colsep0 rowsep0" align="center">100.6 ± 14.6</td><td class="colsep0 rowsep0" align="center">116.5 ± 4.9</td><td class="colsep0 rowsep0" align="center">78.4 ± 13.5</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">6.5/1.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48n</b></td><td class="colsep0 rowsep0" align="char" char="±">95.1 ± 30.6</td><td class="colsep0 rowsep0" align="char" char="±">92.8 ± 17.3</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>48o</b></td><td class="colsep0 rowsep0" align="char" char="±">32.2 ± 2.7</td><td class="colsep0 rowsep0" align="char" char="±">74.4 ± 5.3</td><td class="colsep0 rowsep0" align="center">98.6 ± 12.5</td><td class="colsep0 rowsep0" align="center">105.7 ± 20.2</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">>120</td><td class="colsep0 rowsep0" align="center">ND</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">Average of two runs ± standard deviation (SD).</p></div><div class="footnote" id="t8fn2"><sup><sup>b</sup></sup><p class="last">Multispecies liver microsomal stability in human (HLM), mouse (MLM) and rat (RLM).</p></div><div class="footnote" id="t8fn3"><sup><sup>c</sup></sup><p class="last">Multispecies liver cytosolic stability in human (HLC), mouse (MLC) and rat (RLC).</p></div><div class="footnote" id="t8fn4"><sup><sup>d</sup></sup><p class="last">No SD values provided if all replicate half-life values exceeded 120 min.</p></div><div class="footnote" id="t8fn5"><sup><sup>e</sup></sup><p class="last">Caco-2 permeability [apical to basolateral (AB); basolateral to apical (BA); 10<sup>–6</sup> cm/s). ND = not determined.</p></div></div></div></div><div id="sec2_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Pharmacokinetics</h3><div class="NLM_p">Because of its potent PI3K/HDAC inhibitory and antitumor activity, compound <b>48c</b> was progressed to an in vivo pharmacokinetic (PK) study in female Balb/c mice (<a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">Table <a class="ref showTableEvent internalNav" data-ID="tbl9" href="#tbl9">9</a></a> and <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). The plasma PK of <b>48c</b> via intravenous (iv) route at 3 mg/kg dose revealed its high clearance at 57 mL/min/kg with terminal half-life (<i>t</i><sub>1/2</sub>) of 1 h (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). Unfortunately, po administration of <b>48c</b> at 10 mg/kg dose exhibited very low exposure with an oral bioavailability (<i>F</i>) of only 0.7% (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). When administered intraperitoneally (ip) at 50 and 150 mg/kg dose, respectively, compound <b>48c</b> displayed better PK properties with significantly improved in vivo exposure in terms of <i>C</i><sub>max</sub> and AUC when compared to po administration (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B). This preliminary PK assessment suggested that ip administration would be the preferred dosing route for a proof of concept (POC) study in an in vivo antitumor efficacy mouse model.</div><div class="NLM_table-wrap" id="tbl9"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 9. PK Profile of Compound <b>48c</b><a class="ref internalNav" href="#t9fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">dosing route</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">iv</th><th class="colsep0 rowsep0" align="center">po</th><th class="colsep0 rowsep0" align="center">ip</th><th class="colsep0 rowsep0" align="center">ip</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">dose (mg/kg)</td><td class="colsep0 rowsep0" align="left">3</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">150</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>t</i><sub>1/2</sub> (h)</td><td class="colsep0 rowsep0" align="left">1.01</td><td class="colsep0 rowsep0" align="left">1.4</td><td class="colsep0 rowsep0" align="left">2.7<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">1.7<a class="ref internalNav" href="#t9fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>T</i><sub>max</sub> (h)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.167</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>C</i><sub>max</sub> (ng/mL)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">18.4</td><td class="colsep0 rowsep0" align="left">1443</td><td class="colsep0 rowsep0" align="left">7783</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–<i>t</i></sub> (h·ng/mL)</td><td class="colsep0 rowsep0" align="left">869</td><td class="colsep0 rowsep0" align="left">20.1</td><td class="colsep0 rowsep0" align="left">8493</td><td class="colsep0 rowsep0" align="left">40100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AUC<sub>0–<i>t</i></sub>/<i>D</i> (h·mg/mL)</td><td class="colsep0 rowsep0" align="left">291</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">170</td><td class="colsep0 rowsep0" align="left">267</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Vd<sub>ss</sub> (L/kg)</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CL<sub>p</sub> (mL/min/kg)</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>F</i> (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">NC<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">NC<a class="ref internalNav" href="#t9fn3" aria-label="c">c</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t9fn1"><div class="footnote" id="t9fn1"><sup><sup>a</sup></sup><p class="last">Formulation: IV (3 mg/kg), EtOH/PEG-300/sterile water (10:60:30); PO (10 mg/kg), EtOH/PEG-300/sterile water (10:60:30); IP (50 mg/kg), DMSO/PEG-300/sterile water (10:40:50); IP (150 mg/kg), DMSO/PEG-300/”20% HP-β-CD in water (1:4)” (10:40:50), adjusted to pH 3.0 with 1 M HCl.</p></div><div class="footnote" id="t9fn2"><sup><sup>b</sup></sup><p class="last">The 24 h data point was excluded for terminal half-life (<i>t</i><sub>1/2</sub>) calculation.</p></div><div class="footnote" id="t9fn3"><sup><sup>c</sup></sup><p class="last">NC = not calculated.</p></div></div></div><figure id="fig13" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0013.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Plasma concentration vs time profiles for <b>48c</b> in female Balb/c mice via: (A) iv (3 mg/kg) and po (10 mg/kg) dosing, and (B) ip (50 mg/kg and 150 mg/kg) dosing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66338" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66338" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we have rationally designed and synthesized a series of quinazolin-4-one-based hydroxamic acids for the dual inhibition of PI3K and HDAC enzymes. Structure–activity relationship (SAR) exploration revealed several compounds as potent and selective PI3Kγ, δ/HDAC6 inhibitors with good cellular potencies against a panel of 60 different cancer cell lines at National Cancer Institute (NCI-60). Particularly, compound <b>48c</b> was identified as a potent dual inhibitor with high kinome selectivity and potent antiproliferative activity against various cancer cell lines, including leukemia, melanoma, renal, nonsmall cell lung cancer, central nervous system (CNS) cancer, and breast cancer. Dual inhibitor <b>48c</b> also showed good potency in inducing cell death via necrosis in multiple AML cell lines, including the FLT3-ITD mutant and FLT3-inhibitor resistant cell lines, and primary cells from AML patients. The dual PI3Kδ/HDAC6 activity of <b>48c</b> was further supported by the target engagement studies in the MV411 cell line using the cellular thermal shift assay (CETSA). Compound <b>48c</b> showed decent microsomal stability and in vivo PK profile and warrants further exploration in an antitumor efficacy mouse model. Further lead optimization efforts to achieve new potent dual inhibitors with favorable in vivo PK profiles are currently underway in our laboratory.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13128" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13128" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Chemistry</h3><div class="NLM_p">All air or moisture sensitive reactions were performed under positive pressure of nitrogen with oven-dried glassware. Chemical reagents and anhydrous solvents were obtained from commercial sources and used as is. Preparative purification was performed on a Waters semipreparative HPLC instrument. The column used was a Phenomenex Luna C18 (5 μm, 30 mm × 75 mm) at a flow rate of 45 mL/min. The mobile phase consisted of acetonitrile and water (each containing 0.1% trifluoroacetic acid). A gradient from 10% to 50% acetonitrile over 8 min was used during the purification. Fraction collection was triggered by UV detection (220 nm). Alternately, flash chromatography on silica gel was performed using forced flow (liquid) of the indicated solvent system on Biotage KP-Sil prepacked cartridges and using the Biotage SP-1 automated chromatography system. Analytical analysis for purity was determined by two different methods denoted as final QC methods 1 and 2. Method 1: Analysis was performed on an Agilent 1290 Infinity series HPLC instrument. UHPLC long gradient equivalent from 4% to 100% acetonitrile (0.05% trifluoroacetic acid) in water over 3 min run time of 4.5 min with a flow rate of 0.8 mL/min. A Phenomenex Luna C18 column (3 μm, 3 mm × 75 mm) was used at a temperature of 50 °C. Method 2: Analysis was performed on an Agilent 1260 with a 7 min gradient from 4% to 100% acetonitrile (containing 0.025% trifluoroacetic acid) in water (containing 0.05% trifluoroacetic acid) over 8 min run time at a flow rate of 1 mL/min. A Phenomenex Luna C18 column (3 μm, 3 mm × 75 mm) was used at a temperature of 50 °C. Purity determination was performed using an Agilent diode array detector for both method 1 and method 2. Mass determination was performed using an Agilent 6130 mass spectrometer with electrospray ionization in the positive mode. All assay compounds had >95% purity based on both analytical methods. <sup>1</sup>H NMR spectra were recorded on Varian 400 MHz spectrometers. All proton spectra are referenced relative to the deuterated solvent peak: 7.27 ppm for CDCl<sub>3</sub>, 2.50 ppm (center line signal) for DMSO-<i>d</i><sub>6</sub>. High resolution mass spectrometry results were recorded on Agilent 6210 time-of-flight LC/MS system.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> General Procedure for Sonogashira Cross-Coupling Reaction (GP1)</h4><div class="NLM_p last">Aryl bromide (1.0 equiv), allylpalladium(II) chloride dimer (0.05 equiv), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (0.20 equiv), and alkyne (1.2 equiv) (if solid at room temperature) were weighed and added to a MW vial equipped with a stir bar. The vial was covered with a rubber septum and placed under nitrogen atmosphere. In a separate scintillation vial, DABCO was weighed and dissolved in dry 1,4-dioxane (5 mL/mmol of aryl bromide). This DABCO solution and alkyne (if liquid at room temperature) were added to the MW vial via syringe, and the resulting mixture was bubbled with nitrogen for 5 min followed by stirring for 16 h at room temperature under nitrogen atmosphere. After 16 h, the crude reaction mixture was filtered through a short pad of Celite and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using forced flow of ethyl acetate/hexanes system on Biotage KP-Sil prepacked cartridges and using the Biotage SP-1 automated chromatography system to afford the cross-coupled product.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> General Procedure for Pd/C-Catalyzed Hydrogenation (GP2)</h4><div class="NLM_p last">The substituted alkyne (1.0 equiv) and 10 wt % Pd/C were added to a 20 mL scintillation vial fitted with a rubber septum. The reaction vial was evacuated followed by the addition of dry EtOAc (0.1M). The vacuum was removed, and the reaction vial was stirred for 20 h under an atmosphere of hydrogen using a balloon. After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite, concentrated in vacuo, and carried over to the next step without purification.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> General Procedure for Attachment of Purine Hinge Binding Group (GP3)</h4><div class="NLM_p last">The Boc-protected amine (1.0 equiv) was dissolved in DCM (0.1 M) in a vial, and trifluoroacetic acid (20 equiv) was added dropwise to it. The resulting mixture was stirred at room temperature for 3 h. After completion of reaction (by LC-MS), the reaction mixture was worked-up by either of the following two methods. Method A: The crude reaction was quenched with aqueous saturated NaHCO<sub>3</sub> solution and extracted three times with DCM. The combined organic layers were dried over anhydrous MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford the free amine. Method B: The crude reaction mixture was concentrated in vacuo, redissolved in 1–2 mL of DCM, and passed through preconditioned PL-HCO<sub>3</sub> MP SPE device (bicarbonate resin solid phase extraction) and washed with 2 mL of DCM. The filtrate was concentrated in vacuo to afford the free amine. The resulting free amine was dissolved in ethanol (0.4 M) in a microwave vial equipped with a stir bar followed by the addition of 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine (1.5–2.0 equiv) and triethylamine (3.0–4.0 equiv) to it. The vial was sealed and heated for 4 h at 100 °C in a microwave. After completion of reaction (by LC-MS), the reaction mixture was concentrated in vacuo and the remaining residue was purified by flash chromatography on silica gel using forced flow of indicated solvent system on Biotage KP-Sil prepacked cartridges and using the Biotage SP-1 automated chromatography system to afford the product.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> General Procedure for Conversion of Alkyl Ester to Hydroxamic Acid (GP4)</h4><div class="NLM_p last">The alkyl ester was converted to hydroxamic acid in the final product by either of the following two methods. Method A: The alkyl ester was dissolved in THF/water (0.1 M, 1:1 by vol) in a vial equipped with a stir bar, and LiOH·H<sub>2</sub>O (2.0 equiv) was added to it. The resulting mixture was stirred at room temperature for 10 h and concentrated in vacuo to afford the carboxylic acid. The crude carboxylic acid was dissolved in DMF (0.1 M) in a vial, and <i>O</i>-(tetrahydro-2<i>H</i>-pyran-2-yl)hydroxylamine [NH<sub>2</sub>OTHP] (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv), 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), and 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv) were added to it. The resulting mixture was stirred at room temperature for 16 h, concentrated in vacuo, and purified by flash chromatography on silica using forced flow of 0–10% MeOH/DCM system on Biotage KP-Sil prepacked cartridges and using the Biotage SP-1 automated chromatography system. The purified compound was dissolved in DCM (0.1 M), and TFA (20.0 equiv) was added to it. The resulting mixture was stirred for 20 h. After completion of reaction (by LC-MS), the reaction mixture was concentrated in vacuo and purified on preparative HPLC to afford the final compound. Method B: The alkyl ester was dissolved in MeOH (0.1M) or THF/MeOH (0.1 M, 1:1 by vol) in a MW vial equipped with a stir bar and LiOH·H<sub>2</sub>O (1.1 equiv), and 50% hydroxylamine in water solution (30.0 equiv) were added at to it 0 °C. The MW vial was sealed, and the resulting solution was stirred at 0 °C for 2 h, then allowed to warmup to room temperature overnight. After completion of reaction by LC-MS, the reaction mixture was concentrated in vacuo, dissolved in DCM/MeOH (0.1M, 1:1 by vol), and TFA (20.0 equiv) was added to it. The resulting mixture was stirred at room temperature for 20 h. After completion of reaction (by LC-MS), the reaction mixture was concentrated in vacuo and purified on preparative HPLC to afford the final compound.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Procedure for Synthesis of C4-Subsitiuted Quinazolines (<b>10a</b>,<b>b</b> and <b>14a</b>,<b>b</b>, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h4></div><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Sonogashira Cross-Coupling of Compound <b>5</b> with Alkynes <b>6a</b>,<b>b</b></h4><div id="sec4_1_5_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41"> (<i>S</i>)-<i>tert</i>-Butyl 5-(2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)quinazolin-4-yl)pent-4-ynoate (<b>7a</b>)</h5><div class="NLM_p last">The general procedure GP1 was used with (<i>S</i>)-<i>tert</i>-butyl (1-(4-chloroquinazolin-2-yl)propyl)carbamate <b>5</b> (180.0 mg, 0.56 mmol), allylpalladium(II) chloride dimer (10.1 mg, 0.028 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (16.0 mg, 0.06 mmol), DABCO (125.0 mg, 1.12 mmol), and <i>tert</i>-butyl pent-4-ynoate <b>6a</b> (104.0 mg, 0.67 mmol) in dry 1,4-dioxane (2.8 mL). The remaining residue was purified by flash chromatography on silica using 0–25% ethyl acetate/hexanes system to afford (<i>S</i>)-<i>tert</i>-butyl 5-(2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)quinazolin-4-yl)pent-4-ynoate <b>7a</b> (150.0 mg, 0.34 mmol) as a yellow oil in 61% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.85 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 440.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.29–8.24 (m, 1H), 7.99 (d, <i>J</i> = 8.5 Hz, 1H), 7.88 (ddd, <i>J</i> = 8.5, 6.9, 1.4 Hz, 1H), 7.62 (ddd, <i>J</i> = 8.3, 6.9, 1.2 Hz, 1H), 5.90 (d, <i>J</i> = 8.2 Hz, 1H), 5.02 (d, <i>J</i> = 7.3 Hz, 1H), 2.91 (t, <i>J</i> = 7.4 Hz, 2H), 2.69 (t, <i>J</i> = 7.1 Hz, 2H), 2.15–2.02 (m, 1H), 1.89 (dt, <i>J</i> = 14.0, 7.3 Hz, 1H), 1.49 (s, 9H), 1.47 (s, 9H) 0.90 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_5_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Methyl (<i>S</i>)-6-(2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)quinazolin-4-yl)hex-5-ynoate (<b>7b</b>)</h5><div class="NLM_p last">The general procedure GP1 was used with (<i>S</i>)-<i>tert</i>-butyl (1-(4-chloroquinazolin-2-yl)propyl)carbamate <b>5</b> (225.0 mg, 0.70 mmol), allylpalladium(II) chloride dimer (13.0 mg, 0.04 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (20.0 mg, 0.07 mmol), DABCO (157.0 mg, 1.4 mmol), and methyl hex-5-ynoate <b>6b</b> (115.0 mg, 0.91 mmol) in dry 1,4-dioxane (3.5 mL). The remaining residue was purified by flash chromatography on silica using 0–30% ethyl acetate/hexanes system to afford the product, methyl (<i>S</i>)-6-(2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)quinazolin-4-yl)hex-5-ynoate <b>7b</b> (163.6 mg, 0.398 mmol) as a yellow oil in 57% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.51 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 412.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.26 (ddd, <i>J</i> = 8.3, 1.5, 0.7 Hz, 1H), 8.03–7.97 (m, 1H), 7.90 (ddd, <i>J</i> = 8.4, 6.9, 1.4 Hz, 1H), 7.65 (ddd, <i>J</i> = 8.2, 6.8, 1.2 Hz, 1H), 5.91 (d, <i>J</i> = 8.2 Hz, 1H), 5.03 (d, <i>J</i> = 6.9 Hz, 1H), 3.72 (s, 3H), 2.74 (t, <i>J</i> = 7.1 Hz, 2H), 2.60 (t, <i>J</i> = 7.3 Hz, 2H), 2.10 (p, <i>J</i> = 7.2 Hz, 2H), 1.90 (dt, <i>J</i> = 13.6, 7.2 Hz, 2H), 1.46 (s, 9H), 0.91 (dd, <i>J</i> = 8.2, 7.0 Hz, 3H).</div></div><div id="sec4_1_5_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> (<i>S</i>)-5-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)quinazolin-4-yl)-<i>N</i>-hydroxypentanamide (<b>10a</b>)</h5><div class="NLM_p">Step 1. (i) <i>Alkyne hydrogenation</i>. The general procedure GP2 was used with compound <b>7a</b> (45.0 mg, 0.102 mmol) and 10 wt % Pd/C (4.5 mg). After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite, concentrated in vacuo, and carried over to the next step without purification. (ii) <i>Attachment of purine hinge binding group</i>. The general procedure GP3 was used on the hydrogenated product (formed above) with trifluoroacetic acid (233.0 mg, 2.04 mmol, 0.16 mL) in DCM (1.0 mL). The crude reaction was worked-up (method B) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine <b>8</b> (48.7 mg, 0.20 mmol) and triethylamine (41.3 mg, 0.41 mmol, 57 μL) in ethanol (0.3 mL). The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH (0.5% AcOH)/DCM to afford the product 5-(2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4-yl)pentanoic acid <b>9a</b> (27.7 mg, 0.057 mmol) in 56% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.92 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 490.3. Step 2. <i>Conversion of carboxylic acid to hydroxamic acid</i>. The general procedure GP4 (method A, without the LiOH hydrolysis step) was used with carboxylic acid, <b>9a</b> (16.3 mg, 0.033 mmol) to afford the final product, (<i>S</i>)-5-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4-yl)-<i>N</i>-hydroxypentanamide, TFA (<b>10a</b>) (7.5 mg, 0.014 mmol) in 55% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.40 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 421.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.50 (s, 1H), 8.40–8.28 (m, 3H), 8.03–7.94 (m, 2H), 7.76–7.67 (m, 1H), 3.32 (h, <i>J</i> = 7.9 Hz, 2H), 2.23–2.14 (m, 1H), 2.07 (dt, <i>J</i> = 19.8, 6.9 Hz, 3H), 1.85–1.76 (m, 2H), 1.76–1.66 (m, 2H), 1.62–1.57 (m, 1H), 0.93 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M+ H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>25</sub>N<sub>8</sub>O<sub>2</sub> 421.2095, found 421.2087.</div><div class="NLM_p last">Compound <b>9b</b> was synthesized using a similar procedure, as described above for <b>9a</b>.</div></div><div id="sec4_1_5_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> (<i>S</i>)-6-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)quinazolin-4-yl)-<i>N</i>-hydroxyhexanamide (<b>10b</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>9b</b> (70.4 mg, 0.14 mmol) to afford the final product, (<i>S</i>)-6-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4-yl)-<i>N</i>-hydroxyhexanamide, TFA (<b>10b</b>) (45.0 mg, 0.082 mmol) in 60% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.53 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 435.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.35 (s, 1H), 8.39 (d, <i>J</i> = 8.8 Hz, 2H), 8.32 (dt, <i>J</i> = 8.4, 1.1 Hz, 1H), 7.99–7.95 (m, 2H), 7.76–7.70 (m, 1H), 5.59 (s, 1H), 3.29 (h, <i>J</i> = 7.1 Hz, 2H), 2.20 (dd, <i>J</i> = 14.6, 7.7 Hz, 1H), 2.08 (dt, <i>J</i> = 13.6, 7.4 Hz, 1H), 1.93 (t, <i>J</i> = 7.3 Hz, 2H), 1.79 (p, <i>J</i> = 7.5 Hz, 2H), 1.53 (p, <i>J</i> = 7.3 Hz, 2H), 1.34 (p, <i>J</i> = 7.7 Hz, 2H), 0.94 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M+ H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>27</sub>N<sub>8</sub>O<sub>2</sub> 435.2251, found 435.2250.</div></div><div id="sec4_1_5_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> (<i>S</i>)-4-((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)quinazolin-4-yl)amino)-<i>N</i>-hydroxybutanamide (<b>14a</b>)</h5><div class="NLM_p">Step 1. <i>N-Alkylation.</i> Compound <b>5</b> (103.0 mg, 0.32 mmol) was dissolved in ethanol (0.7 mL) in a 5 mL microwave vial equipped with a stir bar, and ethyl 4-aminobutyrate hydrochloride (107.0 mg, 0.64 mmol) and triethylamine (130.0 mg, 1.28 mmol, 0.18 mL) were added to it. The vial was sealed and heated at 100 °C for 1 h in a microwave. After completion of reaction (by LC-MS), the reaction mixture was concentrated in vacuo and the remaining residue was purified on silica using 0–30% EtOAc/hexanes to afford (<i>S</i>)-ethyl 4-((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)quinazolin-4-yl)amino)butanoate <b>12a</b> (125.0 mg, 0.30 mmol) in 94% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.85 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 417.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.78 (d, <i>J</i> = 8.4 Hz, 1H), 7.75–7.63 (m, 2H), 7.40 (t, <i>J</i> = 7.6 Hz, 1H), 6.51 (s, 1H), 6.04 (d, <i>J</i> = 7.9 Hz, 1H), 4.75 (q, <i>J</i> = 6.6 Hz, 1H), 4.12 (q, <i>J</i> = 7.2 Hz, 2H), 3.71 (q, <i>J</i> = 6.2 Hz, 2H), 2.50 (t, <i>J</i> = 6.7 Hz, 2H), 2.07 (p, <i>J</i> = 6.8 Hz, 2H), 2.03–1.97 (m, 1H), 1.88 (dt, <i>J</i> = 13.8, 7.0 Hz, 1H), 1.47 (s, 9H), 1.22 (t, <i>J</i> = 7.1 Hz, 3H), 0.89 (t, <i>J</i> = 7.5 Hz, 3H). Step 2. <i>Attachment of purine hinge binding group.</i> The general procedure GP3 was used compound <b>12a</b> (120.0 mg, 0.29 mmol) and with trifluoroacetic acid (661.0 mg, 5.8 mmol, 0.44 mL) in DCM (3.0 mL). The crude Boc-deprotected amine was worked-up (method A) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine <b>8</b> (140.4 mg, 0.58 mmol) and triethylamine (118.0 mg, 1.16 mmol, 0.16 mL) in ethanol (0.7 mL). The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford ethyl 4-((2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4-yl)amino)butanoate <b>13a</b> (113.0 mg, 0.22 mmol) in 77% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.78 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 519.3. Step 3. <i>Conversion of alkyl ester to hydroxamic acid.</i> The general procedure GP4 (method A) was used with compound <b>13a</b> (100.0 mg, 0.193 mmol) to afford the final product, (<i>S</i>)-4-((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4-yl)amino)-<i>N</i>-hydroxybutanamide, TFA (<b>14a</b>) (39.0 mg, 0.073 mmol) in 38% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 2.70 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 422.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.56 (s, 1H), 8.38 (d, <i>J</i> = 8.1 Hz, 1H), 8.28 (d, <i>J</i> = 16.0 Hz, 2H), 8.05–7.96 (m, 1H), 7.88–7.81 (m, 1H), 7.74 (t, <i>J</i> = 7.7 Hz, 1H), 3.63 (ddd, <i>J</i> = 35.6, 13.3, 6.7 Hz, 3H), 2.20–2.02 (m, 4H), 2.02–1.88 (m, 1H), 1.78 (dq, <i>J</i> = 13.4, 6.7 Hz, 1H), 1.03 (t, <i>J</i> = 7.4 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M+ H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>24</sub>N<sub>9</sub>O<sub>2</sub> 422.2047, found 422.2055.</div><div class="NLM_p last">Compound <b>13b</b> was synthesized from <b>5</b> and <b>11b</b>, using the similar procedure as described for <b>13a</b>.</div></div><div id="sec4_1_5_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> (<i>S</i>)-6-((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)quinazolin-4-yl)amino)-<i>N</i>-hydroxyhexanamide (<b>14b</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>13b</b> (280.0 mg, 0.526 mmol) to afford the final product, (<i>S</i>)-6-((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4-yl)amino)-<i>N</i>-hydroxyhexanamide, TFA <b>14b</b> (112.5 mg, 0.200 mmol) in 38% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 2.77 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 450.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.09 (s, 1H), 8.38 (dd, <i>J</i> = 8.5, 1.2 Hz, 2H), 8.31 (s, 1H), 8.22 (s, 1H), 8.01 (ddd, <i>J</i> = 8.3, 7.1, 1.2 Hz, 1H), 7.85 (dd, <i>J</i> = 8.5, 1.1 Hz, 1H), 7.74 (ddd, <i>J</i> = 8.3, 7.2, 1.2 Hz, 1H), 5.24 (s, 1H), 3.69 (dq, <i>J</i> = 13.2, 6.6 Hz, 1H), 3.55 (dt, <i>J</i> = 13.1, 6.6 Hz, 1H), 2.14 (q, <i>J</i> = 7.2 Hz, 2H), 1.88 (t, <i>J</i> = 7.3 Hz, 2H), 1.56–1.41 (m, 2H), 1.41–1.34 (m, 2H), 1.17 (q, <i>J</i> = 7.7 Hz, 2H), 1.06 (t, <i>J</i> = 7.4 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>9</sub>O<sub>2</sub> 450.2360, found 450.2377.</div></div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Procedure for Synthesis of C5-Subsitiuted Quinazolinones (<b>19a</b>–<b>c</b> and <b>24a</b>–<b>c</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h4></div><div id="sec4_1_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> Sonogashira Cross-Coupling of Compound <b>15a</b> with Alkynes <b>16a</b>–<b>c</b></h4><div id="sec4_1_6_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> <i>tert</i>-Butyl (<i>S</i>)-5-(2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)pent-4-ynoate (<b>17a</b>)</h5><div class="NLM_p last">The general procedure GP1 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (150.0 mg, 0.33 mmol), allylpalladium(II) chloride dimer (5.9 mg, 0.02 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (9.5 mg, 0.03 mmol), <i>tert</i>-butyl pent-4-yn-oate <b>16a</b> (60.6 mg, 0.39 mmol), and DABCO (73.4 mg, 0.66 mmol) in 2.5 mL of dry 1,4-dioxane. The remaining residue was purified by flash chromatography on silica using 0–25% ethyl acetate/hexanes to afford <i>tert</i>-butyl (<i>S</i>)-5-(2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)pent-4-ynoate <b>17a</b> (130.0 mg, 0.245 mmol) as a yellow oil in 75% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.90 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 532.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.67–7.48 (m, 6H), 7.37 (d, <i>J</i> = 8.0 Hz, 1H), 7.32–7.28 (m, 1H), 5.55 (s, 1H), 4.38 (s, 1H), 2.74 (dd, <i>J</i> = 8.4, 6.7 Hz, 2H), 2.55 (dd, <i>J</i> = 8.3, 6.7 Hz, 2H), 1.47–1.40 (m, 18H), 0.76 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_6_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> Methyl (<i>S</i>)-6-(2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hex-5-ynoate (<b>17b</b>)</h5><div class="NLM_p last">The general procedure GP1 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (234.0 mg, 0.51 mmol), allylpalladium(II) chloride dimer (9.3 mg, 0.03 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (15.0 mg, 0.05 mmol), methyl hex-5-ynoate <b>16b</b> (77.0 mg, 0613 mmol), and DABCO (115.0 mg, 1.02 mmol) in 2.5 mL of dry 1,4-dioxane. The remaining residue was purified by flash chromatography on silica using 0–30% ethyl acetate/hexanes to afford methyl (<i>S</i>)-6-(2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hex-5-ynoate <b>17b</b> (162.0 mg, 0.322 mmol) as a yellow oil in 63% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.60 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 504.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.67–7.47 (m, 6H), 7.39–7.28 (m, 2H), 5.49 (d, <i>J</i> = 9.0 Hz, 1H), 4.38 (s, 1H), 3.65 (s, 3H), 2.53 (dt, <i>J</i> = 15.0, 7.2 Hz, 4H), 1.94 (p, <i>J</i> = 7.2 Hz, 2H), 1.79–1.66 (m, 1H), 1.57–1.45 (m, 2H), 1.43 (s, 9H), 0.75 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_6_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> Methyl (<i>S</i>)-4-((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethynyl)benzoate (<b>17c</b>)</h5><div class="NLM_p last">The general procedure GP1 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (183.0 mg, 0.40 mmol), allylpalladium(II) chloride dimer (7.2 mg, 0.02 mmol), tri-<i>tert</i>-butylphosphonium tetrafluoroborate (12.0 mg, 0.04 mmol), methyl 4-ethynylbenzoate <b>17c</b> (77.0 mg, 0.48 mmol), and DABCO (90.0 mg, 0.80 mmol) in 2.0 mL of dry 1,4-dioxane. The resulting mixture was stirred at room temperature for 16 h and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using 0–25% ethyl acetate/hexanes to afford methyl (<i>S</i>)-4-((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethynyl)benzoate <b>17c</b> (200.0 mg, 0.37 mmol) as a yellow solid in 93% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.78 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 538.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.00–7.95 (m, 2H), 7.74–7.69 (m, 3H), 7.66–7.49 (m, 5H), 7.40 (d, <i>J</i> = 7.9 Hz, 1H), 7.36–7.30 (m, 1H), 5.49 (d, <i>J</i> = 9.0 Hz, 1H), 4.40 (s, 1H), 3.91 (s, 3H), 1.75 (ddd, <i>J</i> = 13.9, 7.3, 4.6 Hz, 1H), 1.55–1.48 (m, 1H), 1.43 (s, 9H), 0.77 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_6_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> (<i>S</i>)-5-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxypentanamide (<b>19a</b>)</h5><div class="NLM_p">Step 1. (i) <i>Alkyne hydrogenation</i>. The general procedure GP2 was used with compound <b>17a</b> (102.0 mg, 0.19 mmol) and 10 wt % Pd/C (10.0 mg) in dry ethyl acetate (1.9 mL). After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite, concentrated in vacuo, and carried over to the next step without purification. (ii) <i>Attachment of purine hinge binding group</i>. The general procedure GP3 was used on the hydrogenated product (formed above) with trifluoroacetic acid (438.0 mg, 3.84 mmol, 0.29 mL) in DCM (1.9 mL). The crude Boc-deprotected amine was worked-up (method B) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine <b>8</b> (92.0 mg, 0.38 mmol) and triethylamine (78.0 mg, 0.77 mmol, 0.1 mL) in ethanol (0.5 mL). The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc (0.5% AcOH by vol)/hexanes to afford 5-(4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)pentanoic acid 5.2n (60.0 mg, 0.103 mmol) in 54% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.17 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 582.4. Step 2. <i>Conversion of carboxylic acid to hydroxamic acid</i>. The general procedure GP4 (method A, without the LiOH hydolysis step) was used with carboxylic acid, <b>18a</b> (17.0 mg, 0.03 mmol), to afford the final product, (<i>S</i>)-5-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxypentanamide, TFA <b>19a</b> (8.0 mg, 0.013 mmol) in 43% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.74 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 513.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.28 (s, 1H), 8.26 (s, 1H), 7.68 (t, <i>J</i> = 7.7 Hz, 1H), 7.56 (d, <i>J</i> = 7.3 Hz, 3H), 7.50 (d, <i>J</i> = 1.2 Hz, 1H), 7.48 (s, 1H), 7.28 (dd, <i>J</i> = 7.6, 1.2 Hz, 1H), 7.20 (s, 1H), 7.07 (s, 1H), 6.95 (s, 1H), 4.74 (s, 1H), 3.81 (s, 1H), 3.14 (s, 2H), 1.97–1.90 (m, 2H), 1.90–1.79 (m, 1H), 1.53–1.48 (m, 3H), 0.76 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>29</sub>N<sub>8</sub>O<sub>3</sub> 513.2357, found 513.2346.</div><div class="NLM_p last">Compounds <b>19b</b> and <b>19c</b> were synthesized using the similar procedure as described for <b>19a</b>.</div></div><div id="sec4_1_6_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> (<i>S</i>)-6-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide (<b>19b</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>18b</b> (60.0 mg, 0.098 mmol) to afford the final product (<i>S</i>)-6-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide, TFA <b>19b</b> (15.0 mg, 0.023 mmol) in 24% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.75 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 527.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.27 (s, 1H), 8.32 (s, 2H), 7.73–7.64 (m, 1H), 7.59–7.46 (m, 5H), 7.29 (dd, <i>J</i> = 7.5, 1.3 Hz, 1H), 4.76 (s, 1H), 3.13 (t, <i>J</i> = 7.7 Hz, 2H), 2.04–1.95 (m, 1H), 1.94–1.78 (m, 3H), 1.47 (p, <i>J</i> = 7.6 Hz, 4H), 1.27 (dt, <i>J</i> = 14.4, 7.6 Hz, 2H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>31</sub>N<sub>8</sub>O<sub>3</sub> 527.2514, found 527.2524.</div></div><div id="sec4_1_6_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> (<i>S</i>)-4-(2-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethyl)-<i>N</i>-hydroxybenzamide (<b>19c</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>18c</b> (12.0 mg, 0.019 mmol) to afford the final product (<i>S</i>)-4-(2-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethyl)-<i>N</i>-hydroxybenzamide, TFA <b>19c</b> (3.0 mg, 0.005 mmol) in 25% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.94 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 561.3. HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>29</sub>N<sub>8</sub>O<sub>3</sub> 561.2357, found 561.2357.</div></div><div id="sec4_1_6_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> (<i>S</i>)-4-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxybenzamide (<b>24a</b>)</h5><div class="NLM_p">Step 1. <i>Suzuki cross-coupling reaction</i>. ((<i>S</i>)-<i>tert</i>-butyl (1-(5-chloro-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a′</b> (261.0 mg, 0.63 mmol), chloro(crotyl)(2-dicyclohexylphosphino-2′,4′,6′-triisopropybiphenyl)palladium(II) [Pd-170] (21.0 mg, 0.03 mmol), and (4-(ethoxycarbonyl)phenyl)boronic acid <b>20a</b> (147.0 mg, 0.76 mmol) were suspended in 1,4-dioxane/water (2.0 mL, 4:1 by vol) in a MW vial equipped with a stir bar under N<sub>2</sub> atmosphere, and potassium phosphate (402.0 mg, 1.89 mmol) was added to it. The MW vial was sealed and heated at 100 °C for 1 h in a MW reactor. The reaction mixture was allowed to cool to RT, quenched with water, and then extracted 3 times with ethyl acetate. The combined organic fractions were dried over MgSO<sub>4</sub> and then concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using 0–35% ethyl acetate/hexanes to afford ethyl (<i>S</i>)-4-(2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)benzoate <b>22a</b> (320.0 mg, 0.61 mmol) as a colorless solid in 96% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.82 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 528.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.05–8.00 (m, 2H), 7.80–7.76 (m, 2H), 7.56–7.40 (m, 3H), 7.39–7.30 (m, 3H), 7.27 (t, <i>J</i> = 4.4 Hz, 1H), 7.22–7.16 (m, 1H), 5.57 (d, <i>J</i> = 9.1 Hz, 1H), 4.41 (d, <i>J</i> = 8.6 Hz, 1H), 4.36 (q, <i>J</i> = 7.1 Hz, 2H), 1.76 (ddd, <i>J</i> = 14.2, 7.4, 4.6 Hz, 1H), 1.62–1.53 (m, 1H), 1.45 (s, 9H), 0.78 (t, <i>J</i> = 7.4 Hz, 3H). Step 2. <i>Installation of purine hinge binding group</i>. The general procedure GP3 was used with <b>22a</b> (320.0 mg, 0.61 mmol) and trifluoroacetic acid (1.38 g, 12.13 mmol, 0.93 mL) in DCM (6.0 mL). The crude Boc-deprotected amine was worked-up (method A) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine (289.0 mg, 1.21 mmol) and triethylamine (245 mg, 2.42 mmol, 0.34 mL) in ethanol (1.2 mL). The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford ethyl 4-(4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)benzoate <b>23a</b> (348.0 mg, 0.55 mmol) in 91% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.56 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 630.3. Step 3. <i>Conversion of alkyl ester to hydroxamic acid</i>. The general procedure GP4 (method A) was used with compound <b>23a</b> (73.2 mg, 0.116 mmol) to afford the final product, (<i>S</i>)-4-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxybenzamide, TFA <b>24a</b> (53.0 mg, 0.082 mmol) in 71% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.56 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 533.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.18 (s, 1H), 8.69 (s, 1H), 8.42 (s, 2H), 7.84 (dd, <i>J</i> = 8.2, 7.4 Hz, 1H), 7.74–7.64 (m, 3H), 7.57–7.37 (m, 5H), 7.34–7.24 (m, 3H), 5.75 (s, 1H), 4.80 (s, 1H), 2.03 (ddd, <i>J</i> = 11.6, 7.4, 3.6 Hz, 1H), 1.87 (dt, <i>J</i> = 14.7, 7.6 Hz, 1H), 0.79 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>25</sub>N<sub>8</sub>O<sub>3</sub> 533.2044, found 533.2044.</div><div class="NLM_p last">Compound <b>24b</b> was synthesized using the similar procedure as described for <b>24a</b>.</div></div><div id="sec4_1_6_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> (<i>S</i>)-2-(4-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)phenyl)-<i>N</i>-hydroxyacetamide (<b>24b</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>23b</b> (33.4 mg, 0.052 mmol) to afford the final product, (<i>S</i>)-2-(4-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)phenyl)-<i>N</i>-hydroxyacetamide, TFA <b>24b</b> (12.7 mg, 0.019 mmol) in 36% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.66 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 546.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.64 (s, 1H), 8.67 (s, 1H), 8.42 (s, 2H), 7.81 (t, <i>J</i> = 7.8 Hz, 1H), 7.67 (dd, <i>J</i> = 8.1, 1.3 Hz, 1H), 7.57–7.39 (m, 5H), 7.25 (dd, <i>J</i> = 7.4, 1.3 Hz, 1H), 7.21–7.14 (m, 4H), 5.75 (s, 2H), 4.81 (s, 1H), 2.03 (dd, <i>J</i> = 12.1, 6.8 Hz, 1H), 1.86 (dt, <i>J</i> = 14.7, 7.7 Hz, 1H), 0.79 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>27</sub>N<sub>8</sub>O<sub>3</sub> 547.2201, found 547.2203.</div></div><div id="sec4_1_6_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Ethyl (<i>S</i>)-2-(2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)benzo[<i>d</i>]thiazole-6-carboxylate (<b>22c</b>)</h5><div class="NLM_p">To a mixture of ((<i>S</i>)-<i>tert</i>-butyl (1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (116.0 mg, 0.25 mmol) and 4,4,4′,4′,5,5,5′,5′-octamethyl-2,2′-bi(1,3,2-dioxaborolane) [BPin]<sub>2</sub> (77.0 mg, 0.30 mmol) in 1,4-dioxane (1.0 mL) in a sealed tube, Pd(dppf)Cl<sub>2</sub> (9.3 mg, 13.0 μmol) and potassium acetate (74.5 mg, 0.76 mmol) were added under N<sub>2</sub> bubbling through the solvent. The resulting mixture was stirred at 100 °C for 16 h. After completion of the reaction, the crude reaction mixture was filtered into a MW vial equipped with a stir bar, and ethyl 2-bromobenzo[<i>d</i>]thiazole-6-carboxylate <b>20c</b> (60.0 mg, 0.21 mmol) and 0.1 mL of water were added to it. [Pd-170] (10.6 mg, 7.1 μmol) and potassium phosphate (134.0 mg, 0.63 mmol) were added to this mixture under nitrogen atmosphere. The MW vial was sealed and heated at 100 °C for 10 h. The reaction mixture was allowed to cool to room temperature, quenched with water, and then extracted 3 times with ethyl acetate. The combined organic fractions were dried over MgSO<sub>4</sub> and then concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using 0–45% EtOAc/hexanes to afford the coupled product, ethyl (<i>S</i>)-2-(2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)benzo[<i>d</i>]thiazole-6-carboxylate (87.0 mg, 0.15 mmol) <b>22c</b> in 70% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.80 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 585.2. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.59 (d, <i>J</i> = 1.6 Hz, 1H), 8.14 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H), 8.05 (d, <i>J</i> = 8.6 Hz, 1H), 7.94 (d, <i>J</i> = 8.3 Hz, 1H), 7.90–7.82 (m, 1H), 7.63–7.40 (m, 4H), 7.36 (d, <i>J</i> = 8.0 Hz, 1H), 7.23 (d, <i>J</i> = 7.3 Hz, 1H), 5.50 (d, <i>J</i> = 9.1 Hz, 1H), 4.42 (q, <i>J</i> = 7.1 Hz, 3H), 1.75 (dt, <i>J</i> = 12.6, 6.9 Hz, 1H), 1.58–1.52 (m, 1H), 1.44 (m, 9H), 0.78 (t, <i>J</i> = 7.4 Hz, 3H).</div><div class="NLM_p last">Compound <b>23c</b> was synthesized from <b>22c</b> using the similar procedure as described for <b>23a</b>.</div></div><div id="sec4_1_6_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> (<i>S</i>)-2-(2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxybenzo[<i>d</i>]thiazole-6-carboxamide (<b>24c</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>23c</b> (16.4 mg, 0.024 mmol) to afford the final product, (<i>S</i>)-2-(2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxybenzo[<i>d</i>]thiazole-6-carboxamide, TFA <b>24c</b> (5.0 mg, 0.007 mmol) in 32% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.64 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 590.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.33 (s, 1H), 8.47 (d, <i>J</i> = 1.7 Hz, 1H), 8.27 (s, 2H), 8.02 (d, <i>J</i> = 8.5 Hz, 1H), 7.97–7.83 (m, 4H), 7.67–7.60 (m, 2H), 7.58–7.47 (m, 4H), 7.42 (s, 3H), 4.73 (s, 1H), 2.01 (s, 1H), 1.89 (s, 1H), 0.78 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>24</sub>N<sub>9</sub>O<sub>3</sub>S 590.1717, found 590.1708.</div></div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> General Procedure for Pd-Catalyzed Amination Reaction (GP5)</h4><div class="NLM_p last">Aryl bromide (1 equiv), methanesulfonato[9,9-dimethyl-4,5-bis(diphenylphosphino)xanthene](2′-methylamino-1,1′-biphenyl-2-yl)palladium(II) XantPhos Palladacycle (methanesulfonato[9,9-dimethyl-4,5-bis(diphenylphosphino)xanthene](2′-methylamino-1,1′-biphenyl-2-yl)palladium(II), Strem Chemicals Inc.) (0.025–0.05 equiv), and amine [if solid] (1.3 equiv) were weighed and added to a microwave vial equipped with a stir bar. The vial was covered with a rubber septum, evacuated, and then filled with nitrogen. Dry toluene or 1,4-dioxane (0.2 M) and amine [if oil at room temperature] (1.3 equiv) were added to the vial, followed by the addition of Cs<sub>2</sub>CO<sub>3</sub> (3.0 equiv) under nitrogen bubbling through the solvent. The microwave vial was sealed and heated at 110 °C for 20 h. After 20 h, the crude reaction mixture was filtered through a short pad of Celite and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using forced flow of ethyl acetate/hexanes system on Biotage KP-Sil prepacked cartridges and using the Biotage SP-1 automated chromatography system to afford the coupled product.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Procedure for Synthesis of C5-Subsitiuted Quinazolinones (<b>28a</b>–<b>h</b>, <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h4></div><div id="sec4_1_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> Pd-Catalyzed Amination of Compounds <b>15a</b> or<b>15b</b> with Amines <b>25a</b>–<b>g</b></h4><div id="sec4_1_8_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> Ethyl (<i>S</i>)-4-((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)butanoate (<b>26a</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (75.2 mg, 0.16 mmol), [XantPhos Palladacycle] (3.9 mg, 4.10 μmol), Cs<sub>2</sub>CO<sub>3</sub> (160.0 mg, 0.49 mmol), and 4-ethoxy-4-oxobutan-1-aminium chloride <b>25a</b> (35.8 mg, 0.21 mmol) were combined in dry toluene (0.8 mL). The resulting mixture was heated at 110 °C for 20 h and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using 0–30% ethyl acetate/hexanes to afford the coupled product, ethyl (<i>S</i>)-4-((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)butanoate <b>26a</b> (58.4 mg, 0.115 mmol) in 70% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.77 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 509.4. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.49 (s, 1H), 7.62 (d, <i>J</i> = 7.1 Hz, 1H), 7.55 (q, <i>J</i> = 8.0, 7.6 Hz, 3H), 7.40 (s, 1H), 6.93 (d, <i>J</i> = 27.4 Hz, 1H), 6.56 (d, <i>J</i> = 8.4 Hz, 1H), 5.57 (s, 1H), 4.39–4.32 (m, 1H), 4.12 (q, <i>J</i> = 7.1 Hz, 2H), 3.25 (q, <i>J</i> = 6.5 Hz, 2H), 2.41 (t, <i>J</i> = 7.3 Hz, 2H), 1.97 (p, <i>J</i> = 7.2 Hz, 2H), 1.73 (s, 2H), 1.43 (s, 9H), 1.24 (t, <i>J</i> = 7.1 Hz, 3H), 0.77 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_8_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> Methyl (<i>S</i>)-5-((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)pentanoate (<b>26b</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (218.4 mg, 0.48 mmol), [XantPhos Palladacycle] (13.7 mg, 14.0 μmol), Cs<sub>2</sub>CO<sub>3</sub> (466.0 mg, 1.43 mmol), and 5-methoxy-5-oxopentan-1-aminium chloride <b>25b</b> (96.0 mg, 0.57 mmol) in dry toluene (2.4 mL). The remaining residue was purified by flash chromatography on silica using 0–50% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-5-((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)pentanoate <b>26b</b> (153.0 mg, 0.30 mmol) in 63% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.75 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 509.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.42 (s, 1H), 7.57 (td, <i>J</i> = 18.2, 14.9, 7.7 Hz, 4H), 7.41 (s, 1H), 7.26 (s, 1H), 6.53 (d, <i>J</i> = 8.4 Hz, 1H), 4.35 (s, 1H), 3.66 (s, 3H), 3.20 (q, <i>J</i> = 6.2 Hz, 2H), 2.34 (t, <i>J</i> = 6.9 Hz, 2H), 1.79–1.64 (m, 6H), 1.43 (s, 9H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H).</div></div><div id="sec4_1_8_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> Methyl (<i>S</i>)-6-((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)hexanoate (<b>26c</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (121.2 mg, 0.26 mmol), [XantPhos Palladacycle] (6.3 mg, 6.61 μmol), Cs<sub>2</sub>CO<sub>3</sub> (258.0 mg, 0.79 mmol), and 6-methoxy-6-oxohexan-1-aminium chloride <b>25c</b> (62.4 mg, 0.34 mmol) in dry toluene (1.3 mL). The remaining residue was purified by flash chromatography on silica using 0–20% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-6-((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)hexanoate <b>26c</b> (126.0 mg, 0.24 mmol) in 91% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.83 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 523.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.44 (d, <i>J</i> = 5.3 Hz, 1H), 7.66–7.48 (m, 4H), 7.37 (d, <i>J</i> = 7.9 Hz, 1H), 6.84 (d, <i>J</i> = 7.8 Hz, 1H), 6.50 (d, <i>J</i> = 8.4 Hz, 1H), 4.40–4.30 (m, 1H), 3.65 (s, 3H), 3.17 (td, <i>J</i> = 6.9, 5.1 Hz, 2H), 2.31 (t, <i>J</i> = 7.5 Hz, 2H), 1.67 (ddt, <i>J</i> = 17.5, 15.2, 7.6 Hz, 6H), 1.58–1.45 (m, 2H), 1.43 (s, 9H), 0.76 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_8_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> Methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate (<b>26d</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (95.2 mg, 0.21 mmol), [XantPhos Palladacycle] (4.9 mg, 5.19 μmol), Cs<sub>2</sub>CO<sub>3</sub> (203.0 mg, 0.62 mmol), and methyl 4-(aminomethyl)benzoate hydrochloride <b>25d</b> (54.3 mg, 1.3 mmol) in dry toluene (1.0 mL). The remaining residue was purified by flash chromatography on silica using 0–30% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>26d</b> (100.0 mg, 0.184 mmol) as an off-white solid in 89% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.82 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 542.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.03 (t, <i>J</i> = 5.6 Hz, 1H), 8.02–7.95 (m, 2H), 7.57 (td, <i>J</i> = 17.2, 14.9, 8.1 Hz, 3H), 7.49–7.37 (m, 4H), 7.31–7.28 (m, 1H), 6.91 (d, <i>J</i> = 7.8 Hz, 1H), 6.40 (d, <i>J</i> = 8.4 Hz, 1H), 5.54 (s, 1H), 4.49 (d, <i>J</i> = 5.8 Hz, 2H), 4.43–4.33 (m, 1H), 3.90 (s, 3H), 1.79–1.70 (m, 1H), 1.64–1.56 (m, 1H), 1.43 (s, 9H), 0.77 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_8_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> Methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-2-methylbenzoate (<b>26e</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (170.0 mg, 0.411 mmol), [XantPhos Palladacycle] (20.0 mg, 0.02 mmol), Cs<sub>2</sub>CO<sub>3</sub> (401.0 mg, 1.23 mmol), and methyl 5-(aminomethyl)picolinate dihydrochloride <b>25e</b> (88.0 mg, 0.493 mmol) in dry toluene (2.0 mL). The remaining residue was purified by flash chromatography on silica using 0–50% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-2-methylbenzoate <b>26e</b> (175.0 mg, 0.31 mmol) in 88% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.88 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 557.3.</div></div><div id="sec4_1_8_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> Methyl (<i>S</i>)-5-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)picolinate (<b>26f</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (96.5 mg, 0.21 mmol), [XantPhos Palladacycle] (5.0 mg, 5.26 μmol), Cs<sub>2</sub>CO<sub>3</sub> (206.0 mg, 0.63 mmol), and methyl 5-(aminomethyl)picolinate dihydrochloride <b>25f</b> (54.3 mg, 1.3 mmol) in dry toluene (1.0 mL). The remaining residue was purified by flash chromatography on silica using 0–80% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-5-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl) picolinate <b>26f</b> (67.5 mg, 0.124 mmol) in 59% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.56 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 544.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.07 (s, 1H), 8.73 (d, <i>J</i> = 2.1 Hz, 1H), 8.08 (d, <i>J</i> = 8.0 Hz, 1H), 7.82 (dd, <i>J</i> = 8.1, 2.2 Hz, 1H), 7.67–7.51 (m, 3H), 7.47 (t, <i>J</i> = 8.1 Hz, 1H), 7.41 (d, <i>J</i> = 8.2 Hz, 1H), 7.29 (s, 1H), 6.97 (s, 1H), 6.38 (d, <i>J</i> = 8.4 Hz, 1H), 4.54 (d, <i>J</i> = 5.8 Hz, 2H), 4.38 (s, 1H), 4.01 (d, <i>J</i> = 1.2 Hz, 3H), 1.78–1.72 (m, 1H), 1.63–1.53 (m, 1H) 1.43 (s, 9H), 0.78 (t, <i>J</i> = 7.3 Hz, 3H).</div></div><div id="sec4_1_8_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> Methyl (<i>S</i>)-6-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)nicotinate (<b>26g</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (109.0 mg, 0.24 mmol), [XantPhos Palladacycle] (6.8 mg, 0.007 mmol), Cs<sub>2</sub>CO<sub>3</sub> (232.0 mg, 0.71 mmol), and methyl 5-(aminomethyl)picolinate dihydrochloride <b>25g</b> (58.0 mg, 0.29 mmol) in dry toluene (1.2 mL). The remaining residue was purified by flash chromatography on silica using 0–50% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-6-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)nicotinate <b>26g</b> (100.0 mg, 0.18 mmol) in 77% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.58 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 544.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 9.28–9.15 (m, 2H), 8.20 (dd, <i>J</i> = 8.2, 2.2 Hz, 1H), 7.67–7.50 (m, 3H), 7.44 (dt, <i>J</i> = 13.8, 9.0 Hz, 3H), 7.33–7.28 (m, 1H), 6.91 (d, <i>J</i> = 7.9 Hz, 1H), 6.37 (d, <i>J</i> = 8.3 Hz, 1H), 5.56–5.51 (m, 1H), 4.64 (d, <i>J</i> = 6.0 Hz, 2H), 4.39 (s, 1H), 3.94 (d, <i>J</i> = 1.6 Hz, 3H), 1.78–1.70 (m, 1H), 1.60–1.50 (m, 1H), 1.40 (s, 9H), 0.77 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_8_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> Methyl (<i>S</i>)-5-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)furan-2-carboxylate (<b>26h</b>)</h5><div class="NLM_p last">The general procedure GP5 was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>15a</b> (56.6 mg, 0.12 mmol) [XantPhos Palladacycle] (2.9 mg, 3.09 μmol), Cs<sub>2</sub>CO<sub>3</sub> (121.0 mg, 0.37 mmol), and methyl 5-(aminomethyl)furan-2-carboxylate hydrochloride <b>25h</b> (35.5 mg, 0.18 mmol) in dry toluene (0.6 mL). The remaining residue was purified by flash chromatography on silica using 0–55% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-5-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)furan-2-carboxylate <b>26h</b> (58.0 mg, 0.11 mmol) in 88% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.71 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 533.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.94 (t, <i>J</i> = 5.9 Hz, 1H), 7.66–7.48 (m, 4H), 7.38 (d, <i>J</i> = 7.9 Hz, 1H), 7.09 (d, <i>J</i> = 3.5 Hz, 1H), 6.94 (d, <i>J</i> = 7.9 Hz, 1H), 6.51 (d, <i>J</i> = 8.4 Hz, 1H), 6.34 (d, <i>J</i> = 3.3 Hz, 1H), 4.48 (d, <i>J</i> = 5.8 Hz, 2H), 4.39 (d, <i>J</i> = 11.2 Hz, 1H), 3.88 (s, 3H), 1.74 (ddd, <i>J</i> = 14.3, 7.4, 4.8 Hz, 1H), 1.55 (dt, <i>J</i> = 14.0, 7.1 Hz, 1H), 1.43 (s, 9H), 0.77 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_8_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> Methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate (<b>26i</b>)</h5><div class="NLM_p last">The procedure mentioned in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>(3) was used with <i>tert</i>-butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)ethyl)carbamate <b>15a</b> (300.0 mg, 0.675 mmol) [XantPhos Palladacycle] (19.0 mg, 0.02 mmol), Cs<sub>2</sub>CO<sub>3</sub> (660.0 mg, 2.03 mmol), and methyl 4-(aminomethyl)benzoate hydrochloride (177.0 mg, 0.88 mmol) in dry toluene (3.4 mL). The remaining residue was purified by flash chromatography on silica using 0–50% ethyl acetate/hexanes to afford the coupled product, methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>26i</b> (274.0 mg, 0.52 mmol) as an off-white solid in 77% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.53 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 529.3. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.98 (s, 1H), 8.02–7.96 (m, 2H), 7.69–7.37 (m, 8H), 7.30 (d, <i>J</i> = 7.3 Hz, 1H), 6.42 (d, <i>J</i> = 8.4 Hz, 1H), 4.52 (m, 1H), 4.49 (d, <i>J</i> = 5.9 Hz, 2H), 3.91 (d, <i>J</i> = 1.2 Hz, 3H), 1.43 (s, 9H), 1.32 (m, 3H).</div></div><div id="sec4_1_8_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> (<i>S</i>)-4-((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxybutanamide (<b>28a</b>)</h5><div class="NLM_p">Step 1. <i>Installation of purine hinge binding group</i>. The general procedure GP3 was used with <b>26a</b> (83.0 mg, 0.16 mmol) and trifluoroacetic acid (374.0 mg, 3.28 mmol, 0.25 mL) in DCM (1.6 mL). The crude Boc-deprotected amine was worked-up (method A) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine (78.0 mg, 0.33 mmol) and triethylamine (66.0 mg, 0.66 mmol, 91.0 μL) in ethanol (0.5 mL). The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford ethyl 4-((4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)amino)butanoate <b>27a</b> (53.0 mg, 0.09 mmol) in 53% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.46 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 611.4. Step 2. <i>Conversion of alkyl ester to hydroxamic acid</i>. The general procedure GP4 (method B) was used with compound <b>27a</b> (50.0 mg, 0.08 mmol) to afford the final product, (<i>S</i>)-4-((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxybutanamide, TFA <b>28a</b> (8.8 mg, 0.02 mmol) in 21% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.50 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 514.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.39 (s, 1H), 8.49 (s, 1H), 8.38 (s, 2H), 7.59–7.47 (m, 5H), 6.70 (d, <i>J</i> = 7.9 Hz, 1H), 6.57 (d, <i>J</i> = 8.5 Hz, 1H), 5.75 (s, 1H), 4.74 (s, 1H), 3.16 (d, <i>J</i> = 4.9 Hz, 3H), 2.08–1.92 (m, 3H), 1.79 (h, <i>J</i> = 7.3, 6.9 Hz, 3H), 0.76 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>26</sub>H<sub>28</sub>N<sub>9</sub>O<sub>3</sub> 514.2310, found 514.2285 [Note: In addition to compound <b>28a</b>, a side product originating from the hydrolysis of ethyl ester in <b>27a</b> to carboxylic acid was also isolated after TFA deprotection step, in 7% yield].</div><div class="NLM_p last">Compounds <b>26b</b>–<b>g</b> were converted to <b>27b</b>–<b>g</b> using the similar procedure, as described above for <b>27a</b>.</div></div><div id="sec4_1_8_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> (<i>S</i>)-5-((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxypentanamide (<b>28b</b>)</h5><div class="NLM_p last">The procedure mentioned in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> (method B) was used with compound <b>27b</b> (68.5 mg, 0.112 mmol) to afford the product, (<i>S</i>)-5-((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxypentanamide, TFA <b>28b</b> (42.0 mg, 0.065 mmol) in 61% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.69 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 528.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.33 (s, 1H), 8.71 (s, 1H), 8.44 (s, 2H), 7.52 (td, <i>J</i> = 13.5, 5.2 Hz, 5H), 6.70 (d, <i>J</i> = 7.9 Hz, 1H), 6.56 (d, <i>J</i> = 8.4 Hz, 1H), 4.76 (s, 1H), 3.14 (d, <i>J</i> = 5.7 Hz, 2H), 2.10–1.93 (m, 3H), 1.82 (dt, <i>J</i> = 15.1, 7.6 Hz, 1H), 1.56 (dt, <i>J</i> = 7.0, 3.5 Hz, 4H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H).</div></div><div id="sec4_1_8_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> (<i>S</i>)-6-((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxyhexanamide (<b>28c</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>27c</b> (135.0 mg, 0.216 mmol) to afford the final product, (<i>S</i>)-6-((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxyhexanamide, TFA <b>28c</b> (30.2 mg, 0.056 mmol) in 32% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.02 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 542.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.33–10.28 (m, 1H), 8.89 (s, 1H), 8.48 (s, 2H), 7.53 (dq, <i>J</i> = 13.3, 8.0, 4.8 Hz, 5H), 6.70 (d, <i>J</i> = 7.9 Hz, 1H), 6.56 (d, <i>J</i> = 8.5 Hz, 1H), 4.77 (s, 1H), 3.13 (t, <i>J</i> = 6.9 Hz, 2H), 2.01 (ddd, <i>J</i> = 14.2, 7.5, 4.5 Hz, 1H), 1.93 (t, <i>J</i> = 7.4 Hz, 2H), 1.83 (h, <i>J</i> = 7.4 Hz, 1H), 1.53 (dp, <i>J</i> = 22.8, 7.3 Hz, 4H), 1.37–1.25 (m, 2H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>32</sub>N<sub>9</sub>O<sub>3</sub> 542.2623, found 542.2647 [Note: In addition to compound <b>28c</b>, a side product originating from the hydrolysis of methyl ester in <b>27c</b> to carboxylic acid was also isolated after TFA deprotection step, in 14% yield].</div></div><div id="sec4_1_8_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> (<i>S</i>)-4-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>28d</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>27d</b> (78.1 mg, 0.121 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>28d</b> (35.5 mg, 0.106 mmol) in 50% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.88 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 562.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.15 (s, 1H), 8.97 (s, 1H), 8.77 (s, 1H), 8.45 (s, 2H), 7.73–7.66 (m, 2H), 7.58 (s, 2H), 7.55 (t, <i>J</i> = 1.6 Hz, 1H), 7.52–7.36 (m, 5H), 6.73 (d, <i>J</i> = 7.9 Hz, 1H), 6.48 (d, <i>J</i> = 8.4 Hz, 1H), 4.77 (s, 1H), 4.52–4.46 (m, 2H), 2.01 (ddd, <i>J</i> = 14.2, 7.6, 4.6 Hz, 1H), 1.83 (dt, <i>J</i> = 14.4, 7.5 Hz, 1H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>28</sub>N<sub>9</sub>O<sub>3</sub> 562.2310, found 562.2322.</div></div><div id="sec4_1_8_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> (<i>S</i>)-4-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxy-2-methylbenzamide (<b>28e</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>27e</b> (105.0 mg, 0.159 mmol) to afford the final product (<i>S</i>)-4-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxy-2-methylbenzamide, TFA <b>28e</b> (60.0 mg, 0.087 mmol) in 55% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.18 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.76 (s, 1H), 8.88 (s, 1H), 8.74 (s, 1H), 8.44 (s, 2H), 7.61–7.52 (m, 3H), 7.56–7.43 (m, 4H), 7.26–7.13 (m, 3H), 6.73 (d, <i>J</i> = 7.9 Hz, 1H), 6.51 (dd, <i>J</i> = 8.5, 1.0 Hz, 1H), 4.78 (s, 1H), 4.41 (s, 2H), 2.30 (s, 3H), 2.01 (ddd, <i>J</i> = 14.0, 7.4, 4.4 Hz, 1H), 1.83 (dt, <i>J</i> = 14.8, 8.0 Hz, 1H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>N<sub>9</sub>O<sub>3</sub>; 576.2466, found 576.2471 [Note: In addition to compound <b>28e</b>, a side product originating from the hydrolysis of methyl ester in <b>27e</b> to carboxylic acid was also isolated after TFA deprotection step, in 13% yield].</div></div><div id="sec4_1_8_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> (<i>S</i>)-5-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxypicolinamide (<b>28f</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>28f</b> (60.5 mg, 0.094 mmol) to afford the final product, (<i>S</i>)-5-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxypicolinamide, TFA <b>28f</b> (20.0 mg, 0.03 mmol) in 32% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.77 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 563.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 9.02 (s, 1H), 8.85 (s, 1H), 8.58 (d, <i>J</i> = 2.0 Hz, 1H), 8.47 (s, 2H), 7.96–7.83 (m, 2H), 7.66–7.40 (m, 6H), 6.75 (d, <i>J</i> = 7.9 Hz, 1H), 6.53 (d, <i>J</i> = 8.4 Hz, 1H), 4.78 (s, 1H), 4.58 (d, <i>J</i> = 3.4 Hz, 2H), 2.02 (ddd, <i>J</i> = 14.4, 7.5, 4.5 Hz, 1H), 1.82 (dt, <i>J</i> = 14.7, 7.7 Hz, 1H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>27</sub>N<sub>10</sub>O<sub>3</sub> 563.2262, found 563.2273.</div></div><div id="sec4_1_8_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> (<i>S</i>)-6-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxynicotinamide (<b>28g</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>28g</b> (79.1 mg, 0.122 mmol) to afford the final product, (<i>S</i>)-6-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxynicotinamide, TFA <b>28g</b> (30.0 mg, 0.044 mmol) in 36% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.54 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 563.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.32 (s, 1H), 9.22 (s, 1H), 8.96 (s, 1H), 8.85 (dd, <i>J</i> = 2.2, 0.8 Hz, 1H), 8.51 (s, 2H), 8.06 (dd, <i>J</i> = 8.1, 2.3 Hz, 1H), 7.63–7.42 (m, 6H), 6.76 (d, <i>J</i> = 7.9 Hz, 1H), 6.54–6.49 (m, 1H), 4.81 (s, 1H), 4.59 (s, 2H), 2.08–1.98 (m, 1H), 1.83 (dt, <i>J</i> = 14.7, 7.8 Hz, 1H), 0.79 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>27</sub>N<sub>10</sub>O<sub>3</sub> 563.2262, found 563.2283.</div></div><div id="sec4_1_8_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> (<i>S</i>)-5-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxyfuran-2-carboxamide (<b>28h</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>28h</b> (61.5 mg, 0.097 mmol) to afford the final product, (<i>S</i>)-5-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxyfuran-2-carboxamide, TFA <b>28h</b> (41.0 mg, 0.076 mmol) in 78% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.73 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 552.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (s, 2H), 8.83 (s, 3H), 8.46 (d, <i>J</i> = 2.9 Hz, 2H), 7.65–7.42 (m, 6H), 6.95 (d, <i>J</i> = 3.4 Hz, 1H), 6.77 (d, <i>J</i> = 7.9 Hz, 1H), 6.69 (d, <i>J</i> = 8.4 Hz, 1H), 6.45 (d, <i>J</i> = 3.4 Hz, 1H), 4.76 (dd, <i>J</i> = 10.5, 4.9 Hz, 1H), 4.48 (d, <i>J</i> = 3.7 Hz, 2H), 2.00 (ddd, <i>J</i> = 14.3, 7.4, 4.3 Hz, 1H), 1.82 (dt, <i>J</i> = 14.5, 7.6 Hz, 1H), 0.76 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>26</sub>N<sub>9</sub>O<sub>4</sub> 552.2102, found 552.2106.</div></div><div id="sec4_1_8_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> (<i>S</i>)-4-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxy-<i>N</i>-methylbenzamide (<b>28i</b>)</h5><div class="NLM_p last">Compound <b>27d</b> (40.6 mg, 0.063) was suspended in MeOH/water (1.2 mL, 1:1) in a vial equipped with a stir bar, and LiOH·H<sub>2</sub>O (5.3 mg, 0.13 mmol) was added to it. The resulting mixture was stirred at room temperature for 10 h and concentrated in vacuo. This crude product was dissolved in DMF (1.2 mL), and <i>N</i>-methylhydroxylamine hydrochloride (8.0 mg, 0.95 mmol), DIPEA (18.0 mg, 0.16 mmol), and 2-(3<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-3-yl)-1,1,3,3-tetramethylisouronium hexafluorophosphate(V) [HATU] (36.0 mg, 0.095 mmol) were added to it. The resulting mixture was stirred at room temperature for 16 h, concentrated in vacuo, and redissolved in DCM (1.2 mL) in a vial followed by dropwise addition of TFA (144.0 mg, 1.26 mmol, 0.10 mL) into it. The resulting mixture was stirred for 10 h, concentrated in vacuo, and purified on preparative HPLC to afford the final compound, (<i>S</i>)-4-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxy-<i>N</i>-methylbenzamide, TFA <b>28i</b> (13.0 mg, 0.019 mmol) in 30% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.15 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.95 (s, 1H), 8.96 (s, 1H), 8.54 (s, 1H), 8.39 (s, 1H), 7.57 (dt, <i>J</i> = 6.6, 1.9 Hz, 4H), 7.47 (dd, <i>J</i> = 10.6, 5.7 Hz, 3H), 7.40–7.32 (m, 2H), 6.73 (d, <i>J</i> = 7.8 Hz, 1H), 6.50 (dd, <i>J</i> = 8.5, 1.0 Hz, 1H), 4.77 (s, 1H), 4.48 (s, 2H), 3.22 (s, 3H), 1.99 (ddd, <i>J</i> = 11.7, 7.4, 3.7 Hz, 1H), 1.87–1.78 (m, 1H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>N<sub>9</sub>O<sub>3</sub> 576.2466, found 576.2494.</div></div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> Procedure for Synthesis of C5-Substituted Quinazolinones (<b>33</b>, <b>37</b>, <b>40a</b>,<b>b</b>, <b>47</b>, <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h4><div id="sec4_1_9_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> <i>tert</i>-Butyl (<i>S</i>)-(1-(5-Cyano-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate (<b>29a</b>)</h5><div class="NLM_p last"><i>tert</i>-Butyl (<i>S</i>)-(1-(5-bromo-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)ethyl)carbamate <b>15a</b> (181.0 mg, 0.40 mmol), zinc cyanide (58.0 mg, 0.49 mmol), and tetrakis(triphenylphosphine)Pd(0) (23.0 mg, 0.02 mmol) were suspended in dry DMF (2.0 mL) in a MW vial equipped with a stir bar under nitrogen atmosphere. The mixture was bubbled with N<sub>2</sub> gas for 2 min, sealed and heated at 100 °C for 16 h. After 16 h, the crude reaction mixture was filtered through a short pad of Celite and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica using 0–45% ethyl acetate/hexanes to afford <i>tert</i>-butyl (<i>S</i>)-(1-(5-cyano-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>29a</b> (154.0 mg, 0.38 mmol) as a colorless solid in 96% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.64 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 405.2. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.99–7.92 (m, 1H), 7.91–7.79 (m, 2H), 7.65–7.50 (m, 3H), 7.42 (d, <i>J</i> = 7.8 Hz, 1H), 7.30 (d, <i>J</i> = 7.1 Hz, 1H), 5.39 (d, <i>J</i> = 9.0 Hz, 1H), 4.46 (s, 1H), 1.75 (ddd, <i>J</i> = 12.2, 7.3, 4.6 Hz, 1H), 1.55–1.49 (m, 1H), 1.43 (s, 9H), 0.78 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_1_9_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> <i>tert</i>-Butyl (<i>S</i>)-(1-(5-(Aminomethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate (<b>30a</b>)</h5><div class="NLM_p">Compound <b>29a</b> (180.0 mg, 0.44 mmol) was dissolved in methanolic ammonia (2.2 mL, 7N in MeOH) in a 20 mL scintillation vial, and Raney Ni (20.0 mg (approx)) was added to it. The reaction vial was evacuated and then kept under hydrogen atmosphere using a balloon. The resulting suspension was stirred at room temperature for 20 h. After completion of reaction (by LC-MS), the crude reaction mixture was carefully filtered under nitrogen and concentrated in vacuo to afford <i>tert</i>-butyl (<i>S</i>)-(1-(5-(aminomethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-2-yl)propyl)carbamate <b>30a</b>. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.85 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 409.3, that was carried to the next step without purification.</div><div class="NLM_p last">Compound <b>30b</b> was synthesized from <b>15a′</b> using the similar procedure as described for <b>30a</b>.</div></div><div id="sec4_1_9_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> Ethyl (<i>S</i>)-2-(((2-(1-((<i>tert</i>-Butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)thiazole-4-carboxylate (<b>32</b>)</h5><div class="NLM_p last">Compound <b>30a</b> (102.0 mg, 0.25 mmol) was dissolved in dry DMF (0.6 mL) in a microwave vial, and ethyl 2-bromothiazole-4-carboxylate <b>31</b> (118.0 mg, 0.5 mmol) and <i>N</i>-ethyl-<i>N</i>-isopropylpropan-2-amine [DIPEA] (129.0 mg, 1.00 mmol) were added to it. The microwave vial was sealed and heated at 180 °C for 30 min in a microwave. After completion of the reaction, the reaction mixture was concentrated in vacuo and the remaining residue was purified using 0–5% MeOH/DCM to afford ethyl (<i>S</i>)-2-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)thiazole-4-carboxylate <b>32</b> (52.8 mg, 0.094 mmol) in 38% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.55 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 564.3.</div></div><div id="sec4_1_9_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> (<i>S</i>)-2-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)-<i>N</i>-hydroxythiazole-4-carboxamide (<b>33</b>)</h5><div class="NLM_p">The general procedure GP3 was used with compound <b>32a</b> (78.8 mg, 0.14 mmol) and trifluoroacetic acid (319.0 mg, 2.80 mmol, 0.21 mL) in DCM (1.4 mL). The crude Boc-deprotected amine was worked-up (method A) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine <b>8</b> (66.8 mg, 0.28 mmol) and triethylamine (56.7 mg, 0.56 mmol, 78.0 μL) in ethanol (0.3 mL). The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford ethyl 2-(((4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)methyl)amino)thiazole-4-carboxylate (68.1 mg, 0.102 mmol) in 73% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.35 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 666.3.</div><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>32a</b> (62.4 mg, 0.09 mmol) to afford the final product, (<i>S</i>)-2-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)-<i>N</i>-hydroxythiazole-4-carboxamide, TFA <b>33</b> (17.6 mg, 0.03 mmol) in 33% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.71 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 569.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (s, 1H), 8.50 (s, 1H), 8.37 (s, 2H), 8.05 (t, <i>J</i> = 6.2 Hz, 1H), 7.76 (t, <i>J</i> = 7.8 Hz, 1H), 7.65–7.55 (m, 5H), 7.52 (s, 2H), 7.17 (s, 1H), 5.03 (d, <i>J</i> = 6.0 Hz, 2H), 4.79 (s, 1H), 2.00 (d, <i>J</i> = 11.7 Hz, 1H), 1.90–1.82 (m, 1H), 0.78 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>25</sub>N<sub>10</sub>O<sub>3</sub>S 569.1826, found 569.1805.</div></div><div id="sec4_1_9_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> 4-Oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazoline-5-carbonitrile (<b>34</b>)</h5><div class="NLM_p last">The general procedure GP3 was used with compound <b>29a</b> (250.0 mg, 0.63 mmol) and trifluoroacetic acid (1.44 g, 12.7 mmol, 0.97 mL) in DCM (6.3 mL). The crude Boc-deprotected amine was worked-up (method B) and reacted with 6-chloro-9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purine <b>8</b> (300.0 mg, 1.26 mmol) and triethylamine (256.0 mg, 2.52 mmol, 0.35 mL) in ethanol (1.6 mL). The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford 4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazoline-5-carbonitrile <b>34</b> (258.4 mg, 0.51 mmol) in 81% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.18 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 507.3.</div></div><div id="sec4_1_9_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> Methyl 4-(((4-Oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)methyl)carbamoyl)benzoate (<b>36</b>)</h5><div class="NLM_p last">Compound <b>34</b> (136.7 mg, 0.27 mmol) was dissolved in ammonia (1.5 mL, 7N in MeOH) in a 20 mL scintillation vial, and Raney Ni (20.0 mg (approx)) was added to it. The reaction vial was evacuated and then kept under hydrogen atmosphere using a balloon. The resulting suspension was stirred at room temperature for 20 h. After completion of reaction (by LC-MS), the crude reaction mixture was carefully filtered under nitrogen, concentrated in vacuo to afford the hydrogenated product, 5-(aminomethyl)-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4(3<i>H</i>)-one [LC-MS (method 1): <i>t</i><sub>R</sub> = 2.67 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 511.3)]. This hydrogenated product was dissolved in DMF (1.3 mL) in a 20 mL scintillation vial, and 4-(methoxycarbonyl)benzoic acid <b>35</b> (68.0 mg, 0.41 mmol), DIPEA (106.0 mg, 0.82 mmol, 0.14 mL), and HATU (125.0 mg, 0.33 mmol) were added to it. The resulting mixture was stirred at room temperature for 16 h and concentrated in vacuo. The remaining residue was purified using 0–10% MeOH/DCM to afford methyl 4-(((4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)methyl)carbamoyl)benzoate <b>36</b> (160.0 mg, 0.24 mmol) in 87% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.32 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 673.3.</div></div><div id="sec4_1_9_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> (<i>S</i>)-<i>N</i>1-((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)-<i>N</i>4-hydroxyterephthalamide (<b>37</b>)</h5><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>36</b> (80.0 mg, 0.119 mmol) to afford the final product, (<i>S</i>)-<i>N</i>1-((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)-<i>N</i>4-hydroxyterephthalamide, TFA <b>37</b> (26.0 mg, 0.037 mmol) in 31% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.54 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 590.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.35 (s, 1H), 9.04 (t, <i>J</i> = 6.0 Hz, 1H), 8.56 (s, 1H), 8.39 (s, 2H), 7.98–7.94 (m, 2H), 7.86–7.82 (m, 2H), 7.76 (t, <i>J</i> = 7.9 Hz, 1H), 7.65–7.48 (m, 6H), 7.41 (d, <i>J</i> = 7.6 Hz, 1H), 5.02 (d, <i>J</i> = 5.8 Hz, 2H), 4.79 (s, 1H), 2.03 (ddd, <i>J</i> = 14.3, 7.5, 4.5 Hz, 1H), 1.89–1.82 (m, 1H), 0.78 (t, <i>J</i> = 7.3 Hz, 3H)). HRMS (ESI) <i>m</i>/<i>z</i>: [M+ H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>28</sub>N<sub>9</sub>O<sub>4</sub> 590.2259, found 590.2265.</div></div><div id="sec4_1_9_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> (<i>S</i>)-2-((((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)methyl)-<i>N</i>-hydroxythiazole-4-carboxamide (<b>40a</b>)</h5><div class="NLM_p last">5-(Aminomethyl)-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4(3<i>H</i>)-one (20.0 mg, 0.04 mmol) was dissolved in DCE (1 mL), and ethyl 2-formylthiazole-4-carboxylate <b>38a</b> (7.3 mg, 0.04 mmol) and a drop of acetic acid were added to it. The resulting mixture was stirred at room temperature for 2 h. After 2 h, sodium triacetoxy borohydride (24.9 mg, 0.12 mmol) was added and the reaction mixture was stirred at room temperature for 2 h. After completion of the reaction (by LC-MS), a saturated aqueous solution of sodium bicarbonate was added. The product was extracted three times with DCM. The combined organic layers were washed, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH (with 1% triethylamine)/DCM to afford ethyl 2-((((4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)methyl)amino)methyl)thiazole-4-carboxylate <b>39a</b> (17.4 mg, 0.026 mmol) in 65% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.96 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 680.3. The general procedure GP4 (method B) was used with compound <b>39a</b> (11.0 mg, 0.018 mmol) to afford the final product, (<i>S</i>)-2-((((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)methyl)-<i>N</i>-hydroxythiazole-4-carboxamide, TFA <b>40a</b> (5.0 mg, 0.007 mmol) in 39% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.14 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 583.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 10.08 (s, 1H), 9.22 (s, 2H), 8.30 (s, 1H), 8.21 (s, 2H), 7.87 (dd, <i>J</i> = 8.3, 7.3 Hz, 1H), 7.78–7.73 (m, 1H), 7.68 (dt, <i>J</i> = 6.6, 2.2 Hz, 1H), 7.62 (dd, <i>J</i> = 7.4, 1.3 Hz, 2H), 7.55 (d, <i>J</i> = 7.4 Hz, 2H), 4.76 (d, <i>J</i> = 15.8 Hz, 4H), 4.64 (s, 2H), 1.96–1.89 (m, 2H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + 2H]<sup>2+</sup> (M + <i>z</i>/<i>z</i>, <i>z</i> = 2) calcd for C<sub>28</sub>H<sub>28</sub>N<sub>10</sub>O<sub>3</sub>S 292.1028, found 292.1042.</div></div><div id="sec4_1_9_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> (<i>S</i>)-4-((((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>40b</b>)</h5><div class="NLM_p last">5-(Aminomethyl)-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4(3<i>H</i>)-one (44.1 mg, 0.09 mmol) was dissolved in DCE (2 mL), and ethyl 2-formylthiazole-4-carboxylate <b>38b</b> (14.1 mg, 0.09 mmol) and a drop of acetic acid were added to it. The resulting mixture was stirred at room temperature for 2 h. After 2 h of stirring, sodium triacetoxy borohydride (54.9 mg, 0.26 mmol) was added and the reaction mixture was stirred at room temperature for 2 h. After completion of the reaction (by LC-MS), a saturated aqueous solution of sodium bicarbonate was added. The product was extracted three times with DCM. The combined organic layers were washed, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH (with 1% triethylamine)/DCM to afford methyl 4-((((4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)methyl)amino)methyl)benzoate <b>39b</b> (53.3 mg, 0.08 mmol) in 94% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.98 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 659.3. The general procedure GP4 (method A) was used with compound <b>39b</b> (53.3 mg, 0.081 mmol) to afford the final product, (<i>S</i>)-4-((((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>40b</b> (33.5 mg, 0.049 mmol) in 41% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.09 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.27 (s, 1H), 8.99 (s, 2H), 8.26 (d, <i>J</i> = 5.8 Hz, 2H), 8.18 (s, 1H), 7.85 (dd, <i>J</i> = 8.3, 7.3 Hz, 1H), 7.81–7.71 (m, 3H), 7.67 (qt, <i>J</i> = 6.2, 3.5 Hz, 2H), 7.59–7.50 (m, 5H), 4.78 (s, 1H), 4.58 (dd, <i>J</i> = 11.5, 5.8 Hz, 2H), 4.29 (t, <i>J</i> = 5.8 Hz, 2H), 1.97–1.85 (m, 2H), 0.77 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + 2H]<sup>2+</sup> (M + <i>z</i>/<i>z</i>; <i>z</i> = 2) calcd for C<sub>31</sub>H<sub>31</sub>N<sub>9</sub>O<sub>3</sub> 288.6269, found 288.6270.</div></div><div id="sec4_1_9_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> (<i>S</i>)-2-(((2-(1-((9<i>H</i>-Purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)-<i>N</i>-hydroxypyrimidine-5-carboxamide (<b>43</b>)</h5><div class="NLM_p">5-(Aminomethyl)-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)quinazolin-4(3<i>H</i>)-one (160.0 mg, 0.31 mmol) was dissolved in ethanol (0.7 mL) in a microwave vial equipped with a sir bar, and ethyl 2-chloropyrimidine-5-carboxylate <b>41</b> (117.0 mg, 0.63 mmol) and triethylamine (127.0 mg, 1.25 mmol, 0.18 mL) were added to it. The microwave vial was sealed, and the resulting mixture was heated at 90 °C for 3 h in a microwave. After completion of the reaction, the reaction mixture was concentrated in vacuo, and the remaining residue was purified using 0–5% MeOH/DCM to afford ethyl 2-(((4-oxo-3-phenyl-2-((1<i>S</i>)-1-((9-(tetrahydro-2<i>H</i>-pyran-2-yl)-9<i>H</i>-purin-6-yl)amino)propyl)-3,4-dihydroquinazolin-5-yl)methyl)amino)pyrimidine-5-carboxylate <b>42</b> (170.0 mg, 0.26 mmol) in 82% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.41 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 661.3.</div><div class="NLM_p last">The general procedure GP4 (method A) was used with compound <b>42</b> (72.0 mg, 0.11 mmol) to afford the final product, (<i>S</i>)-2-(((2-(1-((9<i>H</i>-purin-6-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)-<i>N</i>-hydroxypyrimidine-5-carboxamide, TFA <b>43</b> (42.0 mg, 0.062 mmol) in 57% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.84 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 564.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.05 (s, 1H), 10.01 (d, <i>J</i> = 6.8 Hz, 1H), 8.73–8.53 (m, 4H), 8.17 (d, <i>J</i> = 6.6 Hz, 1H), 7.72 (t, <i>J</i> = 7.9 Hz, 1H), 7.66–7.45 (m, 6H), 7.37 (d, <i>J</i> = 7.6 Hz, 1H), 5.08 (d, <i>J</i> = 4.8 Hz, 2H), 4.77 (ddd, <i>J</i> = 10.7, 6.9, 3.8 Hz, 2H), 2.15–2.02 (m, 1H), 1.97–1.82 (m, 1H), 0.86 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>26</sub>N<sub>11</sub>O<sub>3</sub> 564.2215, found 564.2207.</div></div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> General Procedure for Attachment of Pyrimidine Hinge Binding Group (GP6)</h4><div class="NLM_p last">The Boc-protected amine (1.0 equiv) was dissolved in DCM (0.1 M) in a vial, and trifluoroacetic acid (20 equiv) was added dropwise to it. The resulting mixture was stirred at room temperature for 3 h. After completion of reaction (by LC-MS), the reaction mixture was worked-up by either of the following two methods. Method A: The crude reaction was quenched with aqueous saturated NaHCO<sub>3</sub> solution and extracted three times with DCM. The combined organic layers were dried over anhydrous MgSO<sub>4</sub>, filtered, and concentrated in vacuo to afford the free amine. Method B: The crude reaction mixture was concentrated in vacuo, redissolved in 1–2 mL of DCM, and passed through preconditioned PL-HCO<sub>3</sub> MP SPE device (bicarbonate resin solid phase extraction) and washed with 2 mL of DCM. The filtrate was concentrated in vacuo to afford the free amine. The free amine was dissolved in <i>n</i>-butanol (0.4 M) in a microwave vial equipped with a stir bar followed by the addition of substituted chloro-pyrimidine (1.5–2.0 equiv) and DIPEA (3.0–4.0 equiv) to it. The vial was sealed and heated for 1–5 h at 130 °C in a microwave. After completion of reaction (by LC-MS), the reaction mixture was concentrated in vacuo and the remaining residue was purified by flash chromatography on silica gel using forced flow of indicated solvent system on Biotage KP-Sil prepacked cartridges and using the Biotage SP-1 automated chromatography system to afford the coupled product.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> Procedure for Synthesis of C5-Substituted quinazolinones (<b>46a</b>–<b>d</b> and <b>48a</b>–<b>o</b>, <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>)</h4><div id="sec4_1_11_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> Methyl (<i>S</i>)-6-(2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hexanoate (<b>45a</b>)</h5><div class="NLM_p last">The general procedure GP2 was used with compound <b>17b</b> (96.0 mg, 0.19 mmol) and 10 wt % Pd/C (9.6 mg). After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite and concentrated in vacuo. The general procedure GP6 was used on the hydrogenated product (formed above) with trifluoroacetic acid (440.0 mg, 3.86 mmol, 0.30 mL) in DCM (2.0 mL). The crude reaction was worked-up (method B) and heated with 2,4-diamino-6-chloropyrimidine-5-carbonitrile <b>44a</b> (49.0 mg, 0.29 mmol) and DIPEA (75.0 mg, 0.58 mmol, 0.10 mL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–70% EtOAc/hexanes to afford methyl (<i>S</i>)-6-(2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hexanoate <b>45a</b> (66.0 mg, 0.12 mmol) in 63% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.13 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 541.3.</div></div><div id="sec4_1_11_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> Methyl (<i>S</i>)-6-(2-(1-((2-Amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hexanoate (<b>45b</b>)</h5><div class="NLM_p last">The general procedure GP2 was used with compound <b>17b</b> (141.0 mg, 0.28 mmol) and 10 wt % Pd/C (9.6 mg). After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite and concentrated in vacuo. The general procedure GP6 was used on the hydrogenated product (formed above) with trifluoroacetic acid (643.0 mg, 5.64 mmol, 0.43 mL) in DCM (2.8 mL). The crude reaction was worked-up (method B) and heated with 2-amino-4-chloro-6-methylpyrimidine-5-carbonitrile <b>44b</b> (71.0 mg, 0.42 mmol) and DIPEA (109.0 mg, 0.85 mmol, 0.15 mL) in <i>n</i>-butanol (0.7 mL) at 130 °C for 1 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–70% EtOAc/hexanes to afford methyl (<i>S</i>)-6-(2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hexanoate <b>45b</b> (76.0 mg, 0.14 mmol) in 50% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.19 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 540.3.</div></div><div id="sec4_1_11_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> Methyl (<i>S</i>)-6-(2-(1-((2-Amino-5-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hexanoate (<b>45c</b>)</h5><div class="NLM_p last">The general procedure GP2 was used with compound <b>17b</b> (96.0 mg, 0.19 mmol) and 10 wt % Pd/C (9.6 mg). After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite and concentrated in vacuo. The general procedure GP6 was used on the hydrogenated product (formed above) with trifluoroacetic acid (440.0 mg, 3.86 mmol, 0.30 mL) in DCM (2.0 mL). The crude reaction was worked-up (method B) and heated with 4-chloro-5-methylpyrimidin-2-amine <b>45c</b> (42.0 mg, 0.29 mmol) and DIPEA (75.0 mg, 0.58 mmol, 0.10 mL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–10% MeOH/DCM to afford the product methyl (<i>S</i>)-6-(2-(1-((2-amino-5-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)hexanoate <b>45c</b> (38.0 mg, 0.07 mmol) in 38% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.2 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 515.3.</div></div><div id="sec4_1_11_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> Methyl (<i>S</i>)-4-(2-(2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethyl)benzoate (<b>45d</b>)</h5><div class="NLM_p last">The general procedure GP2 was used with compound <b>17c</b> (81.2 mg, 0.15 mmol) and 10 wt % Pd/C (8.0 mg). After completion of reaction (determined by LC-MS), the crude reaction mixture was filtered using Celite and concentrated in vacuo. The general procedure GP6 was used on the hydrogenated product (formed above) with trifluoroacetic acid (333.0 mg, 2.92 mmol, 0.22 mL) in DCM (1.5 mL). The crude reaction was worked-up (method B) and heated with 2,4-diamino-6-chloropyrimidine-5-carbonitrile <b>44a</b> (50.0 mg, 0.29 mmol) and DIPEA (75.0 mg, 0.58 mmol, 0.1 mL) in <i>n</i>-butanol (0.4 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(2-(2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethyl)benzoate <b>45d</b> (64.0 mg, 0.11 mmol) in 76% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.23 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 575.3.</div></div><div id="sec4_1_11_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> (<i>S</i>)-6-(2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide (<b>46a</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>45a</b> (54.0 mg, 0.10 mmol) to afford the final product, (<i>S</i>)-6-(2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide, TFA <b>46a</b> (40.0 mg, 0.061 mmol) in 61% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.91 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 542.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.29 (s, 1H), 8.05 (s, 3H), 7.74 (t, <i>J</i> = 7.8 Hz, 2H), 7.57–7.42 (m, 7H), 7.33 (dd, <i>J</i> = 7.6, 1.3 Hz, 1H), 4.77 (td, <i>J</i> = 7.6, 4.9 Hz, 1H), 3.14 (ddq, <i>J</i> = 12.4, 8.0, 5.5, 4.2 Hz, 2H), 1.90 (t, <i>J</i> = 7.2 Hz, 3H), 1.82–1.71 (m, 1H), 1.49 (dp, <i>J</i> = 15.0, 7.4 Hz, 4H), 1.28 (q, <i>J</i> = 8.1 Hz, 2H), 0.71 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M+ H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>32</sub>N<sub>9</sub>O<sub>3</sub> 542.2623, found 542.2635.</div></div><div id="sec4_1_11_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> (<i>S</i>)-6-(2-(1-((2-Amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide (<b>46b</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>45b</b> (54.0 mg, 0.10 mmol) to afford the final product, (<i>S</i>)-6-(2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide, TFA <b>46b</b> (45.0 mg, 0.069 mmol) in 69% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.04 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 541.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.28 (s, 1H), 8.09 (s, 1H), 7.73 (t, <i>J</i> = 7.8 Hz, 1H), 7.59–7.45 (m, 6H), 7.45–7.36 (m, 2H), 7.32 (dd, <i>J</i> = 7.5, 1.3 Hz, 1H), 4.77 (td, <i>J</i> = 7.6, 4.8 Hz, 1H), 3.14 (t, <i>J</i> = 7.7 Hz, 2H), 2.54 (s, 1H), 2.34 (s, 3H), 2.00–1.86 (m, 3H), 1.79 (dp, <i>J</i> = 14.7, 7.4 Hz, 1H), 1.49 (dq, <i>J</i> = 15.1, 7.6 Hz, 4H), 1.28 (p, <i>J</i> = 7.7 Hz, 2H), 0.72 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>33</sub>N<sub>8</sub>O<sub>3</sub> 541.2670, found 541.2680.</div></div><div id="sec4_1_11_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> (<i>S</i>)-6-(2-(1-((2-Amino-5-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide (<b>46c</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>45c</b> (35.0 mg, 0.068 mmol) to afford the final product, (<i>S</i>)-6-(2-(1-((2-amino-5-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)-<i>N</i>-hydroxyhexanamide, TFA <b>46c</b> (30.0 mg, 0.048 mmol) in 70% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.06 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 516.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.72 (s, 1H), 10.29 (s, 1H), 8.09 (d, <i>J</i> = 7.7 Hz, 1H), 7.73 (t, <i>J</i> = 7.8 Hz, 1H), 7.57–7.50 (m, 4H), 7.42–7.31 (m, 5H), 4.85 (td, <i>J</i> = 7.9, 5.2 Hz, 1H), 3.14 (t, <i>J</i> = 7.7 Hz, 2H), 2.08–1.78 (m, 8H), 1.47 (dt, <i>J</i> = 14.9, 7.0 Hz, 4H), 1.27 (p, <i>J</i> = 7.5, 7.1 Hz, 2H), 0.76 (t, <i>J</i> = 7.3 Hz, 3H)). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>34</sub>N<sub>7</sub>O<sub>3</sub> 516.2718, found 516.2717.</div></div><div id="sec4_1_11_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> (<i>S</i>)-4-(2-(2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethyl)-<i>N</i>-hydroxybenzamide (<b>46d</b>)</h5><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>45d</b> (56.0 mg, 0.097 mmol) to afford the final product, (<i>S</i>)-4-(2-(2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)ethyl)-<i>N</i>-hydroxybenzamide, TFA <b>46d</b> (27.2 mg, 0.039 mmol) in 40% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.34 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.10 (s, 1H), 7.77–7.71 (m, 1H), 7.67–7.61 (m, 3H), 7.60–7.44 (m, 6H), 7.36–7.25 (m, 4H), 4.76 (q, <i>J</i> = 7.3 Hz, 1H), 2.91–2.82 (m, 2H), 1.94–1.85 (m, 1H), 1.75 (dt, <i>J</i> = 14.5, 7.4 Hz, 1H), 0.74–0.65 (m, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>N<sub>9</sub>O<sub>3</sub> 576.2466, found 576.2475 [Note: In addition to compound <b>46d</b>, a side product originating from the hydrolysis of methyl ester in <b>45d</b> to carboxylic acid was also isolated in 7% yield].</div></div><div id="sec4_1_11_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> (<i>S</i>)-5-((2-(1-((2-Amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxypentanamide (<b>48a</b>)</h5><div class="NLM_p">The general procedure GP6 was used with compound <b>26b</b> (98.0 mg, 0.193 mmol) and trifluoroacetic acid (439.0 mg, 3.85 mmol, 0.3 mL) in DCM (2.0 mL). The crude reaction was worked-up (method B) and heated with 2-amino-4-chloro-6-methylpyrimidine-5-carbonitrile <b>44b</b> (49.0 mg, 0.29 mmol) and DIPEA (75.0 mg, 0.56 mmol, 0.1 mL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-5-((2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)pentanoate <b>47a</b> (89.0 mg, 0.164 mmol) in 85% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.18 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 541.3.</div><div class="NLM_p last">The general procedure GP4 (method B) was used with compound <b>47a</b> (74.0 mg, 0.14 mmol) to afford the final product, (<i>S</i>)-5-((2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)-<i>N</i>-hydroxypentanamide, TFA <b>48a</b> (73.5 mg, 0.112 mmol) in 82% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.02 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 542.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.30 (s, 1H), 8.03 (s, 1H), 7.63–7.51 (m, 3H), 7.51–7.46 (m, 3H), 7.46–7.37 (m, 2H), 6.74 (dd, <i>J</i> = 7.9, 1.0 Hz, 1H), 6.59 (d, <i>J</i> = 8.5 Hz, 1H), 4.74 (td, <i>J</i> = 7.6, 4.8 Hz, 1H), 3.18 (d, <i>J</i> = 6.1 Hz, 2H), 2.54 (s, 1H), 2.35 (s, 3H), 2.02–1.85 (m, 3H), 1.76 (dp, <i>J</i> = 14.6, 7.4 Hz, 1H), 1.57 (q, <i>J</i> = 4.1 Hz, 4H), 0.70 (t, <i>J</i> = 7.3 Hz, 3H)). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>32</sub>N<sub>9</sub>O<sub>3</sub> 542.2623, found 542.2626 [Note: In addition to compound <b>48a</b>, a side product originating from the hydrolysis of methyl ester in <b>47a</b> to carboxylic acid was also isolated in 7% yield].</div></div><div id="sec4_1_11_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> (<i>S</i>)-4-(((2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48b</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (80.0 mg, 0.15 mmol) and trifluoroacetic acid (336.0 mg, 2.95 mmol, 0.23 mL) in DCM (1.7 mL). The crude reaction was worked-up (method B) and heated with 2,4-diamino-6-chloropyrimidine-5-carbonitrile <b>44a</b> (50.0 mg, 0.29 mmol) and DIPEA (76.0 mg, 0.59 mmol, 0.1 mL) in 2-propanol (0.7 mL) at 130 °C for 1 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford the product methyl (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47b</b> (70.0 mg, 0.10 mmol) in 69% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.18 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.3. The general procedure GP4 (method B) was used with compound <b>47b</b> (41.7 mg, 0.072 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48b</b> (21.2 mg, 0.031 mmol) in 42% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.98 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 577.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.12 (s, 1H), 8.96 (s, 1H), 7.70 (dd, <i>J</i> = 8.2, 1.7 Hz, 2H), 7.55 (t, <i>J</i> = 7.3 Hz, 2H), 7.52–7.43 (m, 5H), 7.40 (dd, <i>J</i> = 8.2, 1.7 Hz, 2H), 6.76 (d, <i>J</i> = 7.8 Hz, 1H), 6.51 (d, <i>J</i> = 8.4 Hz, 1H), 4.72 (q, <i>J</i> = 7.1 Hz, 1H), 4.50 (d, <i>J</i> = 4.4 Hz, 2H), 1.85 (q, <i>J</i> = 6.4, 5.8 Hz, 1H), 1.71 (dt, <i>J</i> = 14.5, 7.4 Hz, 1H), 0.72–0.64 (m, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>29</sub>N<sub>10</sub>O<sub>3</sub> 577.2419, found 577.2407.</div></div><div id="sec4_1_11_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> (<i>S</i>)-4-(((2-(1-((2-Amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48c</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (68.0 mg, 0.125 mmol) and trifluoroacetic acid (286.0 mg, 2.51 mmol, 0.19 mL) in DCM (1.2 mL). The crude reaction was worked-up (method B) and heated with 2-amino-4-chloro-6-methylpyrimidine-5-carbonitrile <b>44b</b> (32.0 mg, 0.19 mmol) and DIPEA (48.0 mg, 0.38 mmol, 65.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 1 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–70% EtOAc/hexanes to afford the product methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47c</b> (60.0 mg, 0.10 mmol) in 84% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.24 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 575.3. The general procedure GP4 (method B) was used with compound <b>47c</b> (54.0 mg, 0.094 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48c</b> (34.0 mg, 0.05 mmol) in 52% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.18 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.97 (s, 1H), 8.00 (s, 1H), 7.73–7.66 (m, 2H), 7.59–7.45 (m, 5H), 7.45–7.36 (m, 4H), 6.76 (dd, <i>J</i> = 7.9, 0.8 Hz, 1H), 6.51 (dd, <i>J</i> = 8.5, 0.9 Hz, 1H), 4.74 (td, <i>J</i> = 7.7, 4.7 Hz, 1H), 4.50 (s, 2H), 2.33 (s, 3H), 1.90 (dtd, <i>J</i> = 14.2, 7.2, 4.7 Hz, 1H), 1.82–1.71 (m, 1H), 0.70 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>N<sub>9</sub>O<sub>3</sub> 576.2466, found 576.2481 [Note: For PK studies, <b>48c</b> was converted to a free base].</div></div><div id="sec4_1_11_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> (<i>S</i>)-4-(((2-(1-((2-Amino-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48d</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (52.0 mg, 0.096 mmol) and trifluoroacetic acid (219.0 mg, 1.92 mmol, 0.15 mL) in DCM (1.0 mL). The crude reaction was worked-up (method B) and heated with 4-chloro-6-methylpyrimidin-2-amine <b>44d</b> (21.0 mg, 0.14 mmol) and DIPEA (37.0 mg, 0.29 mmol, 50.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–10% MeOH/DCM to afford methyl (<i>S</i>)-4-(((2-(1-((2-amino-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47d</b> (29.0 mg, 0.053 mmol) in 55% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.97 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 550.3. The general procedure GP4 (method B) was used with compound <b>47d</b> (35.0 mg, 0.064 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48d</b> (18.0 mg, 0.027 mmol) in 42% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.23 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 551.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.08 (s, 1H), 11.14 (s, 1H), 8.96 (d, <i>J</i> = 7.2 Hz, 2H), 7.72–7.68 (m, 2H), 7.60–7.45 (m, 6H), 7.42–7.37 (m, 3H), 6.73 (d, <i>J</i> = 7.8 Hz, 1H), 6.51 (d, <i>J</i> = 8.4 Hz, 1H), 5.99 (s, 1H), 4.51 (ddd, <i>J</i> = 11.5, 5.5, 2.7 Hz, 3H), 2.20 (s, 3H), 1.88 (ddd, <i>J</i> = 14.2, 7.4, 4.1 Hz, 1H), 1.64 (ddd, <i>J</i> = 13.9, 9.0, 7.0 Hz, 1H), 0.67 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>31</sub>N<sub>8</sub>O<sub>3</sub> 551.2514, found 551.2532 [Note: In addition to compound <b>48d</b>, a side product originating from the hydrolysis of methyl ester in <b>47d</b> to carboxylic acid was also isolated in 7% yield].</div></div><div id="sec4_1_11_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> (<i>S</i>)-4-(((2-(1-((6-Amino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48e</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (48.3 mg, 0.09 mmol) and trifluoroacetic acid (203.0 mg, 1.78 mmol, 0.14 mL) in DCM (1.0 mL). The crude reaction was worked-up (method B) and heated with 4-amino-6-chloropyrimidine-5-carbonitrile <b>44e</b> (27.5 mg, 0.18 mmol) and DIPEA (46.0 mg, 0.36 mmol, 62.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 1 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((6-amino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47e</b> (25.0 mg, 0.045 mmol) in 50% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.39 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 561.3. The general procedure GP4 (method B) was used with compound <b>47e</b> (72.0 mg, 0.13 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((6-amino-5-cyanopyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48e</b> (34.0 mg, 0.05 mmol) in 39% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.36 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 562.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.96 (s, 1H), 7.93 (s, 1H), 7.73–7.66 (m, 2H), 7.60–7.45 (m, 6H), 7.42–7.32 (m, 3H), 6.78–6.70 (m, 1H), 6.49 (d, <i>J</i> = 8.4 Hz, 1H), 4.65 (td, <i>J</i> = 7.9, 4.8 Hz, 1H), 4.49 (s, 2H), 2.54 (s, 1H), 1.86 (tq, <i>J</i> = 12.2, 7.4, 6.2 Hz, 1H), 1.80–1.72 (m, 1H), 0.70 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>28</sub>N<sub>9</sub>O<sub>3</sub> 562.2310, found 562.2330. [Note: In addition to compounds <b>48e</b>, a side product originating from the hydrolysis of methyl ester in <b>47e</b> to carboxylic acid was also isolated in 2% yield].</div></div><div id="sec4_1_11_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> (<i>S</i>)-4-(((2-(1-((6-Amino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48f</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (61.8 mg, 0.114 mmol) and trifluoroacetic acid (260.0 mg, 2.28 mmol, 0.17 mL) in DCM (1.2 mL). The crude reaction was worked-up (method B) and heated with 5,6-dichloropyrimidin-4-amine <b>44f</b> (37.0 mg, 0.23 mmol) and DIPEA (59.0 mg, 0.46 mmol, 79.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((6-amino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47f</b> (55.0 mg, 0.096 mmol) in 85% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.25 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 570.2. The general procedure GP4 (method B) was used with compound <b>47f</b> (55.0 mg, 0.096 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((6-amino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48f</b> (53.0 mg, 0.077 mmol) in 80% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.50 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 571.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.95 (s, 1H), 8.03 (s, 1H), 7.73–7.66 (m, 2H), 7.60–7.51 (m, 2H), 7.51–7.44 (m, 5H), 7.40 (d, <i>J</i> = 8.1 Hz, 3H), 6.75 (d, <i>J</i> = 7.8 Hz, 1H), 6.49 (d, <i>J</i> = 8.4 Hz, 1H), 4.63 (td, <i>J</i> = 8.1, 4.2 Hz, 1H), 4.49 (s, 2H), 1.93–1.75 (m, 2H), 0.70 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>28</sub>ClN<sub>8</sub>O<sub>3</sub> 571.1967, found 571.1972 [Note: In addition to compound <b>48f</b>, a side product originating from the hydrolysis of methyl ester in <b>47f</b> to carboxylic acid was also isolated in 9% yield].</div></div><div id="sec4_1_11_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> (<i>S</i>)-6-(((2-(1-((6-Amino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxynicotinamide (<b>48g</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26g</b> (66.0 mg, 0.12 mmol) and trifluoroacetic acid (277.0 mg, 2.45 mmol, 0.19 mL) in DCM (1.2 mL). The crude reaction was worked-up (method B) and heated with 5,6-dichloropyrimidin-4-amine <b>44f</b> (40.0 mg, 0.24 mmol) and DIPEA (63.0 mg, 0.48 mmol, 83.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-6-(((2-(1-((6-amino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)nicotinate <b>47g</b> (30.0 mg, 0.053 mmol) in 43% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.06 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 571.2. The general procedure GP4 (method B) was used with compound <b>47g</b> (30.0 mg, 0.053 mmol) to afford the final product, (<i>S</i>)-6-(((2-(1-((6-amino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxynicotinamide, TFA <b>48g</b> (13.0 mg, 0.019 mmol) in 36% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.89 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 572.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.31 (s, 1H), 9.21 (s, 1H), 8.84 (d, <i>J</i> = 2.1 Hz, 1H), 8.05 (dd, <i>J</i> = 8.1, 2.2 Hz, 1H), 7.89 (s, 1H), 7.60–7.42 (m, 6H), 6.93 (d, <i>J</i> = 22.9 Hz, 2H), 6.76 (d, <i>J</i> = 7.6 Hz, 1H), 6.50 (d, <i>J</i> = 8.5 Hz, 1H), 5.75 (s, 1H), 4.64–4.55 (m, 2H), 3.10 (qd, <i>J</i> = 7.3, 4.8 Hz, 1H), 1.90–1.72 (m, 2H), 0.70 (t, <i>J</i> = 7.2 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>28</sub>H<sub>27</sub>ClN<sub>9</sub>O<sub>3</sub> 572.1920, found 572.1936.</div></div><div id="sec4_1_11_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> (<i>S</i>)-4-(((2-(1-((2-Amino-5-chloro-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48f</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (72.0 mg, 0.133 mmol) and trifluoroacetic acid (303.0 mg, 2.65 mmol, 0.20 mL) in DCM (1.3 mL). The crude reaction was worked-up (method B) and heated with 4,5-dichloro-6-methylpyrimidin-2-amine <b>44g</b> (36.0 mg, 0.20 mmol) and DIPEA (52.0 mg, 0.40 mmol, 70.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford the product methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-chloro-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47h</b> (68.0 mg, 0.116 mmol) in 88% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.15 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 584.2. The general procedure GP4 (method B) was used with compound <b>47h</b> (68.0 mg, 0.116 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-5-chloro-6-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48h</b> (62.0 mg, 0.089 mmol) in 76% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.19 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 585.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.97 (t, <i>J</i> = 5.8 Hz, 1H), 8.39 (d, <i>J</i> = 7.4 Hz, 1H), 7.70 (d, <i>J</i> = 8.0 Hz, 2H), 7.61–7.46 (m, 4H), 7.46–7.36 (m, 5H), 6.78 (d, <i>J</i> = 7.8 Hz, 1H), 6.53 (d, <i>J</i> = 8.4 Hz, 1H), 4.78 (td, <i>J</i> = 7.7, 4.8 Hz, 1H), 4.51 (d, <i>J</i> = 4.8 Hz, 2H), 2.33 (s, 3H), 2.03–1.91 (m, 1H), 1.83 (dq, <i>J</i> = 14.4, 7.4 Hz, 1H), 0.72 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>30</sub>ClN<sub>8</sub>O<sub>3</sub> 585.2124, found 585.2151 [Note: In addition to compound <b>48h</b>, a side product originating from the hydrolysis of methyl ester in <b>47h</b> to carboxylic acid, was also isolated in 4% yield].</div></div><div id="sec4_1_11_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> (<i>S</i>)-4-(((2-(1-((6-Amino-5-cyano-2-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48i</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (86.0 mg, 0.158 mmol) and trifluoroacetic acid (361.0 mg, 3.17 mmol, 0.24 mL) in DCM (1.6 mL). The crude reaction was worked-up (method B) and heated with 4-amino-6-chloro-2-methylpyrimidine-5-carbonitrile <b>44h</b> (40.0 mg, 0.24 mmol) and DIPEA (61.0 mg, 0.47 mmol, 84.0 μL) in <i>n</i>-butanol (0.3 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((6-amino-5-cyano-2-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47i</b> (75.0 mg, 0.131 mmol) in 83% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.33 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 575.3. The general procedure GP4 (method B) was used with compound <b>47i</b> (73.0 mg, 0.13 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((6-amino-5-cyano-2-methylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48i</b> (56.0 mg, 0.08 mmol) in 64% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.17 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 576.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.96 (s, 1H), 7.69 (d, <i>J</i> = 8.1 Hz, 2H), 7.59–7.42 (m, 8H), 7.40 (d, <i>J</i> = 8.0 Hz, 2H), 6.76 (d, <i>J</i> = 7.9 Hz, 1H), 6.49 (d, <i>J</i> = 8.4 Hz, 1H), 4.78 (td, <i>J</i> = 8.3, 4.3 Hz, 1H), 4.50 (s, 2H), 2.17 (s, 3H), 1.92–1.75 (m, 2H), 0.68 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>30</sub>CN<sub>9</sub>O<sub>3</sub> 576.2466, found 576.2470 [Note: In addition to compound <b>48i</b>, a side product originating from the hydrolysis of methyl ester in <b>47i</b> to carboxylic acid, was also isolated in 3% yield].</div></div><div id="sec4_1_11_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> (<i>S</i>)-4-(((2-(1-((2,6-Diamino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48j</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (64.0 mg, 0.12 mmol) and trifluoroacetic acid (269.0 mg, 2.36 mmol, 0.18 mL) in DCM (1.2 mL). The crude reaction was worked-up (method B) and heated with 5,6-dichloropyrimidine-2,4-diamine <b>44i</b> (32.0 mg, 0.18 mmol) and DIPEA (46.0 mg, 0.35 mmol, 63.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 3 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47j</b> (32.0 mg, 0.06 mmol) in 46% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.78 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 585.2. The general procedure GP4 (method B) was used with compound <b>47j</b> (31.0 mg, 0.05 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-chloropyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48j</b> (15.0 mg, 0.02 mmol) in 40% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.29 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 586.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.96 (d, <i>J</i> = 6.4 Hz, 1H), 7.70 (d, <i>J</i> = 8.0 Hz, 2H), 7.61–7.46 (m, 7H), 7.46–7.36 (m, 3H), 7.27 (s, 1H), 6.77 (d, <i>J</i> = 7.9 Hz, 1H), 6.51 (d, <i>J</i> = 8.4 Hz, 1H), 4.69 (td, <i>J</i> = 7.7, 4.4 Hz, 1H), 4.50 (d, <i>J</i> = 4.7 Hz, 2H), 1.88 (ddd, <i>J</i> = 12.9, 7.4, 4.9 Hz, 1H), 1.76 (p, <i>J</i> = 7.4 Hz, 1H), 0.68 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>29</sub>CN<sub>9</sub>O<sub>3</sub> 586.2076, found 586.2087.</div></div><div id="sec4_1_11_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> (<i>S</i>)-4-(((2-(1-((2-Amino-5-cyano-6-(trifluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48k</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (85.0 mg, 0.16 mmol) and trifluoroacetic acid (361.0 mg, 3.17 mmol, 0.24 mL) in DCM (1.6 mL). The crude reaction was worked-up (method B) and heated with 2-amino-4-chloro-6-(trifluoromethyl)pyrimidine-5-carbonitrile <b>44j</b> (52.0 mg, 0.24 mmol) and DIPEA (61.0 mg, 0.47 mmol, 82.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-(trifluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47k</b> (94.0 mg, 0.15 mmol) in 95% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.71 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 629.3. The general procedure GP4 (method B) was used with compound <b>47k</b> (94.0 mg, 0.15 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-(trifluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48k</b> (45.0 mg, 0.06 mmol) in 41% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 5.39 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 630.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.97 (s, 1H), 7.83 (d, <i>J</i> = 7.3 Hz, 1H), 7.70 (d, <i>J</i> = 8.0 Hz, 2H), 7.55–7.45 (m, 3H), 7.40 (d, <i>J</i> = 7.4 Hz, 3H), 7.29 (s, 1H), 6.76 (d, <i>J</i> = 7.9 Hz, 1H), 6.51 (d, <i>J</i> = 8.4 Hz, 1H), 4.80 (td, <i>J</i> = 7.3, 4.8 Hz, 1H), 4.50 (s, 2H), 1.92 (dq, <i>J</i> = 13.3, 7.0 Hz, 1H), 1.76 (hept, <i>J</i> = 7.4 Hz, 1H), 0.72 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>27</sub>F<sub>3</sub>N<sub>9</sub>O<sub>3</sub> 630.2183, found 630.2205.</div></div><div id="sec4_1_11_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> (<i>S</i>)-4-(((2-(1-((2-Amino-5-cyano-6-cyclopropylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48l</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (92.0 mg, 0.17 mmol) and trifluoroacetic acid (387.0 mg, 3.4 mmol, 0.26 mL) in DCM (1.7 mL). The crude reaction was worked-up (method B) and heated with 2-amino-4-chloro-6-cyclopropylpyrimidine-5-carbonitrile <b>44k</b> (50.0 mg, 0.26 mmol) and DIPEA (66.0 mg, 0.51 mmol, 89.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-cyclopropylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47l</b> (40.0 mg, 0.07 mmol) in 39% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.51 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 601.3. The general procedure GP4 (method B) was used with compound <b>47l</b> (40.0 mg, 0.07 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-cyclopropylpyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48l</b> (28.0 mg, 0.04 mmol) in 59% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.74 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 602.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.18–11.06 (m, 1H), 8.96 (s, 1H), 8.01–7.90 (m, 1H), 7.72–7.66 (m, 2H), 7.58–7.44 (m, 6H), 7.40 (d, <i>J</i> = 8.1 Hz, 2H), 7.21 (d, <i>J</i> = 7.1 Hz, 1H), 6.75 (d, <i>J</i> = 7.9 Hz, 2H), 6.50 (d, <i>J</i> = 8.4 Hz, 1H), 4.66 (td, <i>J</i> = 7.7, 4.2 Hz, 1H), 4.50 (s, 2H), 2.05 (p, <i>J</i> = 6.3 Hz, 1H), 1.84 (ddd, <i>J</i> = 14.1, 7.3, 4.6 Hz, 1H), 1.72 (dt, <i>J</i> = 14.4, 7.2 Hz, 1H), 1.00 (d, <i>J</i> = 6.0 Hz, 4H), 0.66 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>33</sub>H<sub>32</sub>N<sub>9</sub>O<sub>3</sub> 602.2623, found 602.2617.</div></div><div id="sec4_1_11_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> (<i>S</i>)-4-(((2-(1-((2-Amino-5-cyano-6-(difluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48m</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26d</b> (50.5 mg, 0.114 mmol) and trifluoroacetic acid (260.0 mg, 2.28 mmol, 0.17 mL) in DCM (1.1 mL). The crude reaction was worked-up (method B) and heated with 5-bromo-4-chloro-6-(difluoromethyl)pyrimidin-2-amine <b>44l</b> (44.0 mg, 0.17 mmol) and DIPEA (44.0 mg, 0.34 mmol, 60.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-bromo-6-(difluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate (65.0 mg, 0.098 mmol) in 86% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.70 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 664.2. Methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-bromo-6-(difluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate (65.0 mg, 0.098 mmol) and zinc cyanide (14.0 mg, 0.12 mmol) were suepended in DMF (1.0 mL) under nitrogen atmosphere in a MW vial, and tris(dibenzylideneacetone)dipalladium(0) (9.0 mg, 9.8 μmol) and 1,1′-Bis(diphenylphosphino)ferrocene [dppf] (10.9 mg, 0.02 mmol) were added to it. The resulting mixture was bubbled with nitrogen for 5 min, followed by sealing the MW vial and heating at °C in microwave for 2 h. After completion of reaction, the crude reaction mixture was concentrated in vacuo and purified by silica gel chromatography using 0–60% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-(difluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47m</b> (50.0 mg, 0.08 mmol) in 84% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.42 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 611.2. The general procedure GP4 (method B) was used with compound <b>47m</b> (50.0 mg, 0.08 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-(difluoromethyl)pyrimidin-4-yl)amino)propyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48m</b> (18.0 mg, 0.0258 mmol) in 30% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 4.82 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> 612.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.13 (s, 1H), 8.97 (s, 1H), 7.69 (d, <i>J</i> = 7.9 Hz, 2H), 7.64 (d, <i>J</i> = 7.3 Hz, 1H), 7.55–7.38 (m, 6H), 7.08 (s, 1H), 6.81–6.73 (m, 1H), 6.66 (s, 1H), 6.54–6.48 (m, 1H), 4.75 (q, <i>J</i> = 6.7 Hz, 1H), 4.50 (s, 2H), 1.94–1.84 (m, 1H), 1.75 (dt, <i>J</i> = 14.1, 7.2 Hz, 1H), 0.70 (t, <i>J</i> = 7.3 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>31</sub>H<sub>28</sub>F<sub>2</sub>N<sub>9</sub>O<sub>3</sub> 612.2278, found 612.2262 [Note: In addition to compound <b>48m</b>, a side product originating from the hydrolysis of methyl ester in <b>47m</b> to carboxylic acid was also isolated in 4% yield].</div></div><div id="sec4_1_11_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> (<i>S</i>)-4-(((2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48n</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26i</b> (103.0 mg, 0.195 mmol) and trifluoroacetic acid (444.0 mg, 3.9 mmol, 0.29 mL) in DCM (2.0 mL). The crude reaction was worked-up (method B) and heated with 2,4-diamino-6-chloropyrimidine-5-carbonitrile <b>44a</b> (50.0 mg, 0.29 mmol) and DIPEA (76.0 mg, 0.59 mmol, 102.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47n</b> (88.0 mg, 0.16 mmol) in 80% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.11 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 562.2. The general procedure GP4 (method B) was used with compound <b>47n</b> (88.0 mg, 0.16 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48n</b> (38.0 mg, 0.06 mmol) in 64% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.90 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 563.2. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.14 (s, 1H), 8.97 (s, 1H), 8.09 (d, <i>J</i> = 6.5 Hz, 1H), 7.96 (s, 2H), 7.73–7.68 (m, 2H), 7.54–7.48 (m, 3H), 7.46–7.38 (m, 5H), 6.78 (dd, <i>J</i> = 7.8, 0.9 Hz, 1H), 6.52 (dd, <i>J</i> = 8.6, 1.1 Hz, 1H), 5.75 (s, 2H), 4.89 (p, <i>J</i> = 6.6 Hz, 1H), 4.51 (s, 2H), 1.34 (d, <i>J</i> = 6.6 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>27</sub>N<sub>10</sub>O<sub>3</sub> 563.2262, found 563.2257 [Note: In addition to compound <b>48n</b>, a side product originating from the hydrolysis of methyl ester in <b>47n</b> to carboxylic acid was also isolated in 5% yield].</div></div><div id="sec4_1_11_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> (<i>S</i>)-4-(((2-(1-((2-Amino-5-cyano-6-methylpyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide (<b>48o</b>)</h5><div class="NLM_p last">The general procedure GP6 was used with compound <b>26i</b> (134.0 mg, 0.25 mmol) and trifluoroacetic acid (579.0 mg, 5.1 mmol, 0.39 mL) in DCM (2.5 mL). The crude reaction was worked-up (method B) and heated with 2-amino-4-chloro-6-methylpyrimidine-5-carbonitrile <b>44b</b> (64.2 mg, 0.38 mmol) and DIPEA (98.0 mg, 0.76 mmol, 133.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–100% EtOAc/hexanes to afford methyl (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)benzoate <b>47o</b> (90.0 mg, 0.16 mmol) in 63% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.12 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 561.2. The general procedure GP4 (method B) was used with compound <b>47o</b> (84.0 mg, 0.15 mmol) to afford the final product, (<i>S</i>)-4-(((2-(1-((2-amino-5-cyano-6-methylpyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)amino)methyl)-<i>N</i>-hydroxybenzamide, TFA <b>48o</b> (53.0 mg, 0.08 mmol) in 52% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.99 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 562.1. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.14 (s, 1H), 8.98 (s, 1H), 8.37 (s, 1H), 7.70 (d, <i>J</i> = 7.9 Hz, 2H), 7.54–7.48 (m, 3H), 7.45–7.35 (m, 5H), 6.79 (d, <i>J</i> = 7.9 Hz, 1H), 6.52 (d, <i>J</i> = 8.4 Hz, 1H), 4.91 (p, <i>J</i> = 5.8, 4.9 Hz, 1H), 4.51 (s, 2H), 2.35 (s, 3H), 1.36 (d, <i>J</i> = 6.5 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>29</sub>H<sub>28</sub>N<sub>9</sub>O<sub>3</sub> 562.2310, found 562.2301 [Note: In addition to compound <b>480</b>, a side product originating from the hydrolysis of methyl ester in <b>47o</b> to carboxylic acid was also isolated in 3% yield].</div></div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> Procedure for Synthesis of C5-Substituted Quinazolinones (<b>51</b> and <b>54</b>, <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>)</h4><div id="sec4_1_12_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> (<i>S</i>)-<i>N</i>1-((2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)-N4-hydroxyterephthalamide (<b>51</b>)</h5><div class="NLM_p last">Compound <b>29b</b> (36.0 mg, 0.09 mmol) was dissolved in ammonia (1.0 mL, 7N in MeOH) in a 20 mL scintillation vial and Raney Ni (10.0 mg (approx.)) was added to it. The reaction vial was evacuated and then kept under hydrogen atmosphere using a balloon. The resulting suspension was stirred at room temperature for 20 h. After completion of reaction (by LC-MS), the crude reaction mixture was carefully filtered under nitrogen and concentrated in vacuo. This hydrogenated product was dissolved in DMF (1.0 mL) in a 20 mL scintillation vial, and 4-(methoxycarbonyl)benzoic acid <b>35</b> (18.2 mg, 0.10 mmol), DIPEA (24.0 mg, 0.18 mmol, 32.0 μL), and HATU (38.0 mg, 0.10 mmol) were added to it. The resulting mixture was stirred at room temperature for 16 h and concentrated in vacuo. The remaining residue was purified using 0–10% MeOH/DCM to afford methyl (<i>S</i>)-4-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)carbamoyl)benzoate <b>49</b> (49.0 mg, 0.088 mmol) in 96% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.45 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 557.1. The general procedure GP6 was used with compound <b>49</b> (49.0 mg, 0.09 mmol) and trifluoroacetic acid (201.0 mg, 1.76 mmol, 0.14 mL) in DCM (1.0 mL). The crude reaction was worked-up (method B) and heated with 2,4-diamino-6-chloropyrimidine-5-carbonitrile <b>44a</b> (22.3 mg, 0.13 mmol) and DIPEA (34.1 mg, 0.26 mmol, 46.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 5 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford methyl (<i>S</i>)-4-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)carbamoyl)benzoate <b>50</b> (40.0 mg, 0.07 mmol) in 77% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 2.85 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 590.0. The general procedure GP4 (method B) was used with compound <b>50</b> (40.0 mg, 0.07 mmol) to afford the final product, (<i>S</i>)-<i>N</i>1-((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)-<i>N</i>4-hydroxyterephthalamide, TFA <b>51</b> (11.0 mg, 0.016 mmol) in 23% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.55 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 591.3. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (s, 1H), 9.07 (t, <i>J</i> = 6.0 Hz, 1H), 8.01–7.94 (m, 2H), 7.89–7.76 (m, 4H), 7.64–7.53 (m, 4H), 7.53–7.40 (m, 5H), 5.03 (d, <i>J</i> = 5.9 Hz, 2H), 4.88 (t, <i>J</i> = 6.8 Hz, 1H), 1.35 (d, <i>J</i> = 6.7 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>30</sub>H<sub>27</sub>N<sub>10</sub>O<sub>4</sub> 591.2211, found 591.2210.</div></div><div id="sec4_1_12_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> (<i>S</i>)-2-(((2-(1-((2,6-Diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)-<i>N</i>-hydroxypyrimidine-5-carboxamide (<b>54</b>)</h5><div class="NLM_p last">Compound <b>29b</b> (125.0 mg, 0.32 mmol) was dissolved in ammonia (2.0 mL, 7N in MeOH) in a 20 mL scintillation vial, and Raney Ni (20.0 mg (approx) was added to it. The reaction vial was evacuated and then kept under hydrogen atmosphere using a balloon. The resulting suspension was stirred at room temperature for 20 h. After completion of reaction (by LC-MS), the crude reaction mixture was carefully filtered under nitrogen, concentrated in vacuo, and redissolved in <i>n</i>-BuOH (0.5 mL) in a MW vial. Ethyl 2-chloropyrimidine-5-carboxylate <b>41</b> (85.0 mg, 0.46 mmol) and DIPEA (118.0 mg, 0.91 mmol, 0.16 mL) were added to the MW vial, and the vial was sealed and heated at 130 °C for 1 h in a MW reactor. After completion of the reaction, the reaction mixture was concentrated in vacuo, and the remaining residue was purified using 0–100% EtOAc/hexanes to afford ethyl (<i>S</i>)-2-(((2-(1-((<i>tert</i>-butoxycarbonyl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)pyrimidine-5-carboxylate <b>52</b> (138.0 mg, 0.25 mmol) in 83% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.75 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 545.3. The general procedure GP6 was used with compound <b>52</b> (138.0 mg, 0.25 mmol) and trifluoroacetic acid (579.0 mg, 5.1 mmol, 0.39 mL) in DCM (2.5 mL). The crude reaction was worked-up (method B) and heated with 2,4-diamino-6-chloropyrimidine-5-carbonitrile <b>44a</b> (64.0 mg, 0.38 mmol) and DIPEA (98.0 mg, 0.76 mmol, 133.0 μL) in <i>n</i>-butanol (0.5 mL) at 130 °C for 2 h in a MW reactor. The remaining residue was purified by flash chromatography on silica gel using 0–5% MeOH/DCM to afford ethyl (<i>S</i>)-2-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)pyrimidine-5-carboxylate <b>53</b> (116.0 mg, 0.20 mmol) in 79% yield. LC–MS (method 1): <i>t</i><sub>R</sub> = 3.11 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 578.1. The general procedure GP4 (method B) was used with compound <b>53</b> (116.0 mg, 0.20 mmol) to afford the final product, (<i>S</i>)-2-(((2-(1-((2,6-diamino-5-cyanopyrimidin-4-yl)amino)ethyl)-4-oxo-3-phenyl-3,4-dihydroquinazolin-5-yl)methyl)amino)-<i>N</i>-hydroxypyrimidine-5-carboxamide, TFA <b>54</b> (68.0 mg, 0.10 mmol) in 50% yield. LC–MS (method 2): <i>t</i><sub>R</sub> = 3.38 min, <i>m</i>/<i>z</i> (M + H)<sup>+</sup> = 565.0. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.36 (s, 1H), 9.07 (t, <i>J</i> = 6.0 Hz, 1H), 8.01–7.94 (m, 2H), 7.89–7.76 (m, 4H), 7.64–7.53 (m, 4H), 7.53–7.40 (m, 5H), 5.03 (d, <i>J</i> = 5.9 Hz, 2H), 4.88 (t, <i>J</i> = 6.8 Hz, 1H), 2.54 (s, 2H), 1.35 (d, <i>J</i> = 6.7 Hz, 3H). HRMS (ESI) <i>m</i>/<i>z</i>: [M + H]<sup>+</sup> calcd for C<sub>27</sub>H<sub>25</sub>N<sub>12</sub>O<sub>3</sub> 565.2167, found 565.2164.</div></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> Molecular Modeling</h3><div class="NLM_p">All structure visualizations and computations were performed using molecular operating environment (MOE).<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> The images of Idelalisib bound to PI3Kδ (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B) and SAHA (Vorinostat) bound to HDAC-2 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>D) are based on the protein data bank entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>.pdb<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XE0">5XE0</a>.pdb<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> respectively. The structure of HDAC-6 bound to <b>19b</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) was derived from <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.pdb (Trichostatin A bound to HDAC-6).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> In this derivation, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.pdb was loaded into MOE, protein prep was run, and hydrogens were added. The structure was then minimized, using the MMFF94 potential,<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> to a local minimum associated with the crystal structure coordinates. Then Trichostatin A was truncated all the way to the hydroxamic acid headgroup. <b>19b</b> was grown out from the hydroxamic acid headgroup one atom at a time. If the placement of an atom resulted in a new torsion angle, the torsion was searched to find a local minimum before the next atom was added. Once <b>19b</b> was completely grown into the HDAC-6 binding pocket, the whole structure was minimized to 0.001 RMS.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i120" class="anchor-spacer"></div><h4 class="article-section__title" id="_i120"> HDAC-6 Selectivity Carboxamide Compounds</h4><div class="NLM_p last">The sequences of HDACs-1, 2, 3, 4, 5, 6, 7, 8, 9, 10, and 11 were downloaded from UNIPROT<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> and aligned. Using the structure that we modeled of <b>19b</b> bound to HDAC-6 we determined all HDAC-6 residues within 4.5 Å of <b>19b</b>. These residue positions were then marked on the HDAC-6 sequence, and their counterpart residues on the other aligned HDACs were determined. All marked HDAC-6 residue positions that were not conserved across the other kinases were identified. Then one at a time each nonconserved residue position in HDAC-6 was mutated to its value in each of the other HDACs, and the effect of the mutation on <b>19b</b> interactions with the protein was determined.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> Biological Assay Protocols</h3><div id="sec6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> PI3K Assay Protocol (HTRF Assay Platform)</h4><div class="NLM_p last">(1) <i>Assay Principle</i>. The PIP3 product is detected by displacement of biotin-PIP3 from an energy transfer complex consisting of europium-labeled anti-GST monoclonal antibody, a GST-tagged pleckstrin homology (PH) domain, biotinylated PIP3 and streptavidin–allophycocyanin (APC). Excitation of europium in the complex results in an energy transfer to the APC and a fluorescent emission at 665 nm. The PIP3 product formed by PI3-kinase(h) activity displaces biotin-PIP3 from the complex, resulting in a loss of energy transfer, and thus, a decrease in signal. <i>This is a 3-step reaction</i>: First, the kinase reaction with PIP2 substrate is carried out in the presence of ATP, and the reaction is quenched with stop solution, and then, finally detect by adding detection mixture followed by incubation. (2) <i>Reaction Conditions. Assay Buffer</i>: HEPES 50 mM (pH7.0), NaN3 0.02%, BSA 0.01%, orthovanadate 0.1 mM, 1% DMSO. <i>Detection Buffer</i>: HEPES 10 mM (pH7.0), BSA 0.02%, KF 0.16 M, EDTA 4 mM. <i>Substrate</i>: 10 μM PIP2 substrate (PI(4,5)P2). <i>ATP</i>: 10 μM ATP under standard conditions. <i>Control Inhibitor</i>: PI-103. (3) <i>Assay Procedure</i>. (A) Prepare substrate in freshly prepared reaction buffer. (B) Deliver kinase into the substrate solution and gently mix. (C) Deliver compounds in 100% DMSO into the kinase reaction mixture by Acoustic Technology (Echo550; nanoliter range), incubate for 10 min at room temperature. (D) Deliver ATP into the reaction mixture to initiate the reaction. (E) Incubate for 30 min at 30 °C. (F) Quench the reaction with stop solution. (G) Add detection mixture and incubate for overnight. H) Measure HTRF: Ex = 320 nm, ratio of Em = 615 nm and Em = 665 nm. (4) <i>Data Analysis</i>. The emission ratio is converted into μM PIP3 production based on PIP3 standard curves. The nonlinear regression to obtain the standard curve and IC<sub>50</sub> values are performed using Graphpad Prism software.</div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i123" class="anchor-spacer"></div><h4 class="article-section__title" id="_i123"> HDAC Fluorescent Activity Assay</h4><div class="NLM_p last"><i>Assay Description</i>: The HDAC fluorescent activity assay is based on the unique fluorogenic substrate and developer combination. This assay is a highly sensitive and validated. The assay procedure has two steps (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, Howitz, 2015 Drug Discovery Today: Technologies). First, the fluorogenic substrate, which comprises an acetylated lysine side chain, is incubated with a purified HDAC enzyme. Deacetylation of the substrate sensitizes the substrate so that, in the second step, treatment with the Developer produces a fluorophore. <i>Compound Handling</i>: Testing compounds were dissolved in 100% DMSO to specific concentration. The serial dilution was conducted by epMotion 5070 in DMSO. <i>Materials and Reagents</i>: HDAC reaction buffer: 50 mM Tris-HCl, pH 8.0, 137 mM NaCl, 2.7 mM KCl, and 1 mM MgCl2, add fresh 1 mg/mL BSA, 1% DMSO. <i>Substrate</i>: HDAC1,2,3,6,10: Fluorogenic peptide from p53 residues 379–382 (RHKK(Ac)AMC). HDAC4,5,7, 9 and 11: Fluorogenic HDAC class2a substrate (trifluoroacetyl lysine). HDAC 8: fluorogenic peptide from p53 residues 379–382 (RHK(Ac)K(Ac)AMC). <i>General Reaction Procedure</i>: standard IC<sub>50</sub> determination. <i>Deacetylation Step</i>: (A) Deliver 2X enzyme in wells of reaction plate except no enzyme control wells. Add buffer in No En wells. (B) Deliver compounds in 100% DMSO into the enzyme mixture by Acoustic technology (Echo550; nanoliter range). Spin down and preincubation. (C) Deliver 2× substrate mixture (fluorogenic HDAC substrate and co-factor if applicable) in all reaction wells to initiate the reaction. Spin and shake. (D) Incubate for 1–2 h at 30 °C with seal. <i>Development Step</i>: (E) Add developer with Trichostatin A (or TMP269) to stop the reaction and to generate fluorescent color. (F) Fluorescence was read (excitatory, 360; emission, 460) using the EnVision Multilabel Plate Reader (PerkinElmer). (G) Take end point reading for analysis after the development reaches plateau. <i>Data Analysis</i>: The percentages of enzyme activity (relative to DMSO controls) and IC<sub>50</sub> values were calculated using the GraphPad Prism 4 program based on a sigmoidal dose–response equation.</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i124" class="anchor-spacer"></div><h4 class="article-section__title" id="_i124"> NCI-60 Cellular Assays</h4><div class="NLM_p last">Assay Protocols and Data Analysis procedures are available at: <a href="https://dtp.cancer.gov/discovery_development/nci-60/methodology.htm" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/methodology.htm</a></div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> CETSA Protocol</h4><div class="NLM_p last">K-562 (chronic myelogenous leukemia), MV-4–11 (B-myelomonocytic leukemia) and THP-1 (acute monocytic leukemia) were cultured in RPMI 1640 (ThermoFisher, cat. no, 11835) supplemented with 10% FBS (Hyclone) and 0.5× penicillin/streptomycin (ThermoFisher). Then 2 ×10<sup>6</sup> cells were rinsed with PBS and resuspended in 200 μL of DPBS (+Ca, +Mg) + 1 g/L <span class="smallcaps smallerCapital">d</span>-glucose + 1× Halt protease inhibitor cocktail (ThermoFisher). Four freeze–thaw cycles (dry ice/EtOH, 37 °C water bath) were performed, and lysates were centrifuged at 17 000<i>g</i> for 15 min (4 °C). The soluble fraction was separated on a 4–12% Bis-Tris gel using MOPS running buffer and transferred to a nitrocellulose membrane. Primary antibodies were anti-HDAC6 (ab133493, 1:10 000), anti-HDAC6 (ab82557, 1:1000), and anti-PI3kD (ab109006, 1:1000). Secondary antibodies were HRP conjugated, and SuperSignal West Pico chemiluminescent substrate was used for detection. Membranes were imaged on a ChemiDoc imaging system (BioRad). CETSA melt curves were determined by collecting MV-4–11 cells in DPBS (+Ca, +Mg) supplemented 1 g/L <span class="smallcaps smallerCapital">d</span>-glucose, 1× protease inhibitors, and 0.5% DMSO at 15 × 10<sup>6</sup> cells/mL. Cell suspensions were distributed to 0.2 mL PCR tubes, and samples were heated for 3.5 min to various temperatures. After allowing samples to equilibrate to room temperature, samples were processed by freeze–thaw for immunoblot as outlined above. Isothermal dose–response CETSA was performed by collecting MV-4–11 cells in DPBS (+Ca, +Mg) supplemented 1 g/L <span class="smallcaps smallerCapital">d</span>-glucose, 0.5× protease inhibitors, 10 × 10<sup>6</sup> cells/mL. Then 1 μL of compound (at 100× final concentration; 1% DMSO final) was added to 99 μL of cell suspension and cells were incubated at 37 °C for 1 h. Samples were then heated for 3.5 min at either 52 °C (PI3Kδ) or 58 °C (HDAC6). Cells were lysed by freeze–thaw cycles and processed for immunoblot as outlined above. Densitometry analysis was performed using ImageQuant TL and a rolling ball radius 80 for background subtraction.</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> Cell Culture and Reagents</h4><div class="NLM_p last">Human MOLM-14 AML, Mv4–11, HL-60, THP-1, MOLM-13 wt, MOLM-13 FLT3 inhibitor resistant, and U-937 cells were maintained in RPMI1640 medium containing 10% heat-inactivated fetal bovine serum (FBS), 100 units/mL penecillin, 100 ug/mL streptomycin, and 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine. MOLM-13 cells were selected for resistance to FLT3 inhibitor HG-7-85-01 as described (Weisberg, 2011). Cells were treated with different dual inhibitor compounds (NCATS, NIH).</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> Cell Viability Assays</h4><div class="NLM_p">RPMI 8226 and U266B1 cells obtained from ATCC and were cultured in RPMI1640 (Gibco, cat. no. 11875093) supplemented with 10% FBS (HyClone), 50 U/mL penicillin, and 50 μg/mL streptomycin (Life Technologies). For cell viability assays, RPMI 8226 and U226B1 cells were seeded at 5 × 10<sup>4</sup> cells/mL (50 μL volume) into white, tissue culture-treated 384-well plates (Corning, cat. no. 3570BC). T63 nL of compound solutions in DMSO were immediately added using a Wako Pin-tool. Plates were covered with BreatheEasy seals (Diversified Biotech) and incubated at 37 °C, 5% CO<sub>2</sub> for 5 days. Then 25 μL of CellTiter-Glo (Promega) was then added to each well, and plates were incubated at RT for 10 min. Luminescence was measured on a ViewLux plate reader.</div><div class="NLM_p">Human MOLM-14, MOLM-13 wt, MOLM-13 FLT3 inhibitor resistant, MV4–11, HL-60, THP-, and U-937 cell lines were treated with different concentrations of dual inhibitors for the indicated times. Cell viability was determined by trypan blue exclusion. For studies with primary AML blasts, cells were obtained pretherapy from AML patients under an approved protocol (DFCI 01-206). Normal PBMCs were obtained from normal donors under consent. Cells were left untreated or treated with test compound for the indicated times. Cell death was determined by staining with propidium iodide (PI) alone or PI/Annexin V and then analyzed by flow cytometry.</div><div id="sec6_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i128" class="anchor-spacer"></div><h5 class="article-section__title" id="_i128"> Necrosis Assays</h5><div class="NLM_p last">For analysis of cell death by necrosis, cells were stained with propidium iodide (PI) alone or PI/Annexin V, and then analyzed by flow cytometry as described.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a></div></div><div id="sec6_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> Immunoblot Analysis</h5><div class="NLM_p last">Cells were harvested and rinsed with ice-cold phosphate buffered saline (PBS). Ice-cold lysis buffer [0.5 mL; 20 mM Tris (pH 7.5), 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, 2.5 mM sodium pyrophosphate, 1 mM β-glycerophosphate, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 1 mg/mL leupeptin, 1 mM PMSF] was added to 1 × 10<sup>7</sup> cells and sonicated on ice four times for 5 s each followed by microcentrifugation for 10 min at 4 °C. Soluble proteins were separated on gels and transferred to nitrocellulose filters. The lysates were immunoblotted with antiphospho AKT, total AKT, and b-actin antibodies (Cell Signaling Technology), Antigen–antibody complexes were visualized by enhanced chemiluminescence (GE Healthcare).</div></div></div></div><div id="sec7" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130"> Pharmacokinetic Analyses</h3><div id="sec7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i131" class="anchor-spacer"></div><h4 class="article-section__title" id="_i131"> Parallel Artificial Membrane Permeability Assay (PAMPA)</h4><div class="NLM_p last">Stirring double-sink PAMPA method (patented by pION Inc.) was employed to determine the permeability of compounds via PAMPA. The PAMPA lipid membrane consisted of an artificial membrane of a proprietary lipid mixture and dodecane (Pion Inc.), optimized to predict gastrointestinal tract (GIT) passive permeability. The lipid was immobilized on a plastic matrix of a 96-well “donor” filter plate placed below a 96-well “acceptor” plate. pH 7.4 solution was used in both donor and acceptor wells. The test articles, stocked in 10 mM DMSO solutions, were diluted to 0.05 mM in aqueous buffer (pH 7.4), and the concentration of DMSO was 0.5% in the final solution. During the 30 min permeation period at room temperature, the test samples in the donor compartment were stirred using the Gutbox technology (Pion Inc.) to reduce the aqueous boundary layer. The test article concentrations in the donor and acceptor compartments were measured using a UV plate reader (Nano Quant, Infinite 200 PRO, Tecan Inc., Männedorf, Switzerland). Permeability calculations were performed using Pion Inc. software and were expressed in units of 10<sup>–6</sup>cm/s. Compounds with low or weak UV signal we analyzed using high resolution LC/MS (Thermo QExactive). The three controls used were ranitidine (low permeability), dexamethasone (moderate permeability), and verapamil (high permeability). Method has been previously published.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a></div></div><div id="sec7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i132" class="anchor-spacer"></div><h4 class="article-section__title" id="_i132"> Multispecies Multitime Point Assays</h4><div class="NLM_p last"><i>Microsome Stability Assay</i>: Briefly, each reaction mixture (110 μL) consisted of a test article (1 μM), human, SD rat, or CD-1 mouse microsomal fractions (0.5 mg/mL), and NADPH regenerating system (1 μM) in phosphate buffer at pH 7.4. <i>Cytosol Stability Assay</i>: Briefly, each reaction mixture (110 μL) consisted of a test article (1 μM) and either human, SD rat, or CD-1 mouse cytosol fractions (2 mg/mL) in phosphate buffer (100 mM) at pH 7.4. Samples were incubated in 384-well plates at 37 °C for 0, 5, 10, 15, 30, and 60 min. Sample analysis and half-life calculations were performed using a previously described method.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> The three controls used were Buspirone (short half-life), Loperamide (moderate half-life), and Antipyrine (long half-life).</div></div><div id="sec7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i133" class="anchor-spacer"></div><h4 class="article-section__title" id="_i133"> Pharmacokinetics (PK) [NCATS]</h4><div class="NLM_p last">Adult female Balb/c mice (<i>n</i> = 3/sampling time point) were obtained from Charles River Laboratory (Frederick, MD). All experimental procedures were approved by the Animal Care and Use Committee (ACUC) of the NIH Division of Veterinary Resources (DVR). Pharmacokinetics (PK) was evaluated after a single dose of 3 mg/kg intravenous (IV) and 10 mg/kg oral gavage (PO) administration. Dosing solutions were freshly prepared on the day of administration in 70% PEG-300/30% H<sub>2</sub>O/10% ethanol. The blood samples (∼80 μL) were collected in K2EDTA tubes after drug administration based on the PK study protocols, and plasma (∼30 μL) was harvested after centrifugation at 3000 rpm for 10 min. All plasma samples were stored at −80 °C until analysis. Ultraperformance liquid chromatography–tandem mass spectrometry (UPLC-MS/MS) methods were developed to determine compounds’ concentrations in mouse plasma samples. Mass spectrometric analysis was performed on a Waters Xevo TQ-S triple quadrupole instrument using electrospray ionization in positive mode with the selected reaction monitoring. The separation of test compounds from endogenous components was performed on an Acquity BEH C18 column (50 mm × 2.1 mm, 1.7 μ) using a Waters Acquity UPLC system with 0.6 mL/min flow rate and gradient elution. The mobile phases were 0.1% formic acid in water and 0.1% formic acid in acetonitrile. The calibration standards and quality control samples were prepared in the blank mouse plasma. Aliquots of 10 μL plasma samples were mixed with 200 μL of internal standard in acetonitrile to precipitate proteins in a 96-well plate. Then 1 μL of supernatant was injected for the UPLC-MS/MS analysis. Data were analyzed using TargetLynx V4.1 (Waters Corp., Milford, MA). The pharmacokinetic parameters, e.g., plasma clearance (CL<sub>p</sub>) and volume of distribution at steady-state (Vd<sub>ss</sub>), were calculated using the noncompartmental approach (models 200 and 201) of the pharmacokinetic software Phoenix WinNonlin, version 8.0 (Certara, St. Louis, MO). The area under the plasma concentration versus time curve (AUC) was calculated using the linear trapezoidal method. The slope of the apparent terminal phase was estimated by log linear regression using at least 3 data points, and the terminal rate constant (λ) was derived from the slope. AUC<sub>0–∞</sub> was estimated as the sum of the AUC<sub>0–<i>t</i></sub> (where “<i>t</i>” is the time of the last measurable concentration) and <i>C</i><sub><i>t</i></sub>/λ. The apparent terminal half-life (<i>t</i><sub>1/2</sub>) was calculated as 0.693/λ.</div></div><div id="sec7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i134" class="anchor-spacer"></div><h4 class="article-section__title" id="_i134"> Pharmacokinetics (PK) [Pharmaron]</h4><div class="NLM_p last">Female Balb/c mice (sourced from Si Bei Fu Laboratory Animal Technology Co. Ltd.), approximately 6–8 weeks of age, and a weight of approximately 20–30 g, were dosed with <b>48c</b> at 50 and 150 mg/kg via IP injection. The formulations, [DMSO/PEG300/sterile water (10:40:50)] and (DMSO/PEG300/“20% HP-β-CD in water (1:4)” (10:40:50), adjusted pH 3–4 with 1 M HCl) was prepared on the day of dosing. Each cohort had three mice, and plasma was collected at pre, 5 min, 15 min, 30 min, 1 h, 2 h, 4 h, 8 h, 12 h, and 24 h post dose for IP. Approximately 0.025 mL of blood was collected by the dorsal metatarsal vein at each time point. All blood samples were transferred into plastic microcentrifuge tubes containing Heparin-Na as anticoagulant. Collection tubes with blood samples and anticoagulant were inverted several times for proper mixing of the tube contents and then placed on wet ice prior to centrifugation for plasma. Blood samples will be centrifuged at 4000<i>g</i> for 5 min at 4 °C to obtain plasma. Plasma samples will be stored in polypropylene tubes, quickly frozen in a freezer, and kept at −75 ± 15 °C until analyzed by LC-MS/MS. The following pharmacokinetic parameters were measured: <i>t</i><sub>1/2</sub>, <i>C</i><sub>max</sub>, <i>T</i><sub>max</sub>, and AUC<sub>0–<i>t</i></sub>.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i135"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00193" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32380" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32380" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00193?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00193</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">PI3K/HDAC enzyme inhibitory activities of selected 5-substituted quinazolinones; HPLC-MS chromatogram of compounds <b>19b</b>, <b>46b</b>, <b>48b</b>, <b>48c</b>, <b>48n</b>, and <b>48o</b>, respectively; figures showing compound <b>19b</b> bound to HDAC1, 2 (Glu567, Asp568), HDAC3 (Asp567, Asp568), HDAC8 (Tyr567, Asp568), HDAC4, 5, 7, 9 (Asp567, Thr568), HDAC10 (Asp567, Ala568), and HDAC11 (Pro567, Asn568), respectively; kinome scan of compound <b>48c</b>, and immunoblot analysis for PI3Kδ and HDAC6 expression (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular strings for all final compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf">jm0c00193_si_001.pdf (1.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_002.csv">jm0c00193_si_002.csv (4.25 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c00193" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ashish Thakur</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8456-1088" title="Orcid link">http://orcid.org/0000-0002-8456-1088</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3756445f5e445f19435f565c424577595e5f19505841"><span class="__cf_email__" data-cfemail="90f1e3f8f9e3f8bee4f8f1fbe5e2d0fef9f8bef7ffe6">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gurmit Grewal</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#dabdafa8b7b3aef4bda8bfadbbb69abdb7bbb3b6f4b9b5b7"><span class="__cf_email__" data-cfemail="2a4d5f5847435e044d584f5d4b466a4d474b434604494547">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gregory J. Tawa</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark J. Henderson</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8590-7338" title="Orcid link">http://orcid.org/0000-0001-8590-7338</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carina Danchik</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Suiyang Liu</span> - <span class="hlFld-Affiliation affiliation">Dana-Farber
Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pranav Shah</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Amy Q. Wang</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Garrett Dunn</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Md Kabir</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Elias C. Padilha</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Xu</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anton Simeonov</span> - <span class="hlFld-Affiliation affiliation">National
Center for Advancing Translational Sciences, National Institutes of Health, 9800 Medical Center Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Surender Kharbanda</span> - <span class="hlFld-Affiliation affiliation">Dana-Farber
Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Stone</span> - <span class="hlFld-Affiliation affiliation">Dana-Farber
Cancer Institute, 450 Brookline Avenue, Boston, Massachusetts 02215, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products, or organizations imply endorsement by the U.S. Government.<br /></br>The authors declare the following competing financial interest(s): Ashish Thakur, Gurmit Grewal, Gregory J. Tawa and Anton Simeonov are inventors of a patent application associated with this work.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e12977-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i137">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We gratefully acknowledge the funding from Intramural Research Program, National Center for Advancing Translational Sciences (NCATS), National Institutes of Health (NIH). We thank Paul Shinn and his team for the assistance with compound management and Christopher Leclair and his team for analytical chemistry and purification support. We also thank Reaction Biology Corporation for screening compounds in enzymatic assays and Pharmaron Inc. for conducting the pharmacokinetic studies.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">PI3K</td><td class="NLM_def"><p class="first last">phosphoinositide 3-kinase</p></td></tr><tr><td class="NLM_term">PIP2</td><td class="NLM_def"><p class="first last">phosphatidylinositol 4,5-diphosphate</p></td></tr><tr><td class="NLM_term">PIP3</td><td class="NLM_def"><p class="first last">phosphatidylinositol 3,4,5-triphosphate</p></td></tr><tr><td class="NLM_term">RTK</td><td class="NLM_def"><p class="first last">receptor tyrosine kinase</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">SAHA</td><td class="NLM_def"><p class="first last">suberanilohydroxamic acid</p></td></tr><tr><td class="NLM_term">Akt</td><td class="NLM_def"><p class="first last">protein kinase B</p></td></tr><tr><td class="NLM_term">mTOR</td><td class="NLM_def"><p class="first last">mammalian target of rapamycin</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute monocytic leukemia</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>, <i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">DCE</td><td class="NLM_def"><p class="first last">1,2-dichloroethane</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DABCO</td><td class="NLM_def"><p class="first last">(1,4-diazabicyclo[2.2.2]octane)</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>, <i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">Boc</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyloxycarbonyl</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last"><i>N</i>-[(dimethylamino)-1<i>H</i>-1,2,3-triazolo-[4,5-<i>b</i>]pyridin-1-ylmethylene]-<i>N</i>-methylmethanaminium hexafluorophosphate <i>N</i>-oxide</p></td></tr><tr><td class="NLM_term">PAMPA</td><td class="NLM_def"><p class="first last">parallel artificial membrane permeability assay</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i139">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62220" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62220" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 73 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span> <span> </span><span class="NLM_article-title">Why Is Cancer Drug Discovery So Difficult?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1038/nrd2155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrd2155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=17159925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=115-120&author=A.+Kambauthor=S.+Weeauthor=C.+Lengauer&title=Why+Is+Cancer+Drug+Discovery+So+Difficult%3F&doi=10.1038%2Fnrd2155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Why is cancer drug discovery so difficult?</span></div><div class="casAuthors">Kamb, Alexander; Wee, Susan; Lengauer, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-120</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Thirty-five years after the 'war on cancer' was declared, the discovery of anticancer drugs remains a highly challenging endeavor.  Here, we consider the factors responsible, such as tumor heterogeneity, and suggest strategies to improve the chances of short-term success in the development of novel anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsRgFvOR6O7Vg90H21EOLACvtfcHk0liwuSLin-zY2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGlsrw%253D&md5=0ee9e19541b8f3f0ed495b3de44b3ad3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd2155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2155%26sid%3Dliteratum%253Aachs%26aulast%3DKamb%26aufirst%3DA.%26aulast%3DWee%26aufirst%3DS.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DWhy%2520Is%2520Cancer%2520Drug%2520Discovery%2520So%2520Difficult%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D115%26epage%3D120%26doi%3D10.1038%2Fnrd2155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of Cancer: The Next Generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+Cancer%3A+The+Next+Generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0liwuSLin-zY2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520Cancer%253A%2520The%2520Next%2520Generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lgVZ0SKrYjUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">De Novo Design of Multitarget Ligands with an Iterative Fragment-Growing Strategy</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1021/ci500021v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500021v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjvVOnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1235-1241&author=E.+Shangauthor=Y.+Yuanauthor=X.+Chenauthor=Y.+Liuauthor=J.+Peiauthor=L.+Lai&title=De+Novo+Design+of+Multitarget+Ligands+with+an+Iterative+Fragment-Growing+Strategy&doi=10.1021%2Fci500021v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">De Novo Design of Multitarget Ligands with an Iterative Fragment-Growing Strategy</span></div><div class="casAuthors">Shang, Erchang; Yuan, Yaxia; Chen, Xinyi; Liu, Ying; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1235-1241</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of multitarget drugs has recently attracted much attention.  Most of the reported multitarget ligands have been serendipitous discoveries.  Although a few methods have been developed for rational multitarget drug discovery, there is a lack of elegant methods for de novo multitarget drug design and optimization, esp. for multiple targets with large differences in their binding sites.  In this paper, we report the first de novo multitarget ligand design method, with an iterative fragment-growing strategy.  Using this method, dual-target inhibitors for COX-2 and LTA4H were designed, with the most potent one inhibiting PGE2 and LTB4 prodn. in the human whole blood assay with IC50 values of 7.0 and 7.1 μM, resp.  Our strategy is generally applicable in rational and efficient multitarget drug design, esp. for the design of highly integrated inhibitors for proteins with dissimilar binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYeSpPS20iL7Vg90H21EOLACvtfcHk0lgVZ0SKrYjUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjvVOnsbY%253D&md5=964c8053de5457730dc369cf8f8ca6a7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fci500021v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500021v%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DE.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DDe%2520Novo%2520Design%2520of%2520Multitarget%2520Ligands%2520with%2520an%2520Iterative%2520Fragment-Growing%2520Strategy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1235%26epage%3D1241%26doi%3D10.1021%2Fci500021v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span></span> <span> </span><span class="NLM_article-title">Designed Multiple Ligands. An Emerging Drug Discovery Paradigm</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6523</span>– <span class="NLM_lpage">6543</span>, <span class="refDoi"> DOI: 10.1021/jm058225d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm058225d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6523-6543&author=R.+Morphyauthor=Z.+Rankovic&title=Designed+Multiple+Ligands.+An+Emerging+Drug+Discovery+Paradigm&doi=10.1021%2Fjm058225d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Designed Multiple Ligands.</span></div><div class="casAuthors">Morphy, Richard; Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6523-6543</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The review discusses main areas in designed multiple ligands (DMLs) discovery that include 5HT transporter-based DMLs for depression, dopamine D2-receptor based DMLs for schizophrenia, histamine H1 receptor-based DMLs for allergies, DMLs targeting the angiotensin system for hypertension, nonsteroidal anti-inflammatory drugs, and PPAR-based DMLs for metabolic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgouWcjsiKgbVg90H21EOLACvtfcHk0lgVZ0SKrYjUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL&md5=00e3bf5a521fc91996ce95bb379a0b5c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm058225d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm058225d%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DDesigned%2520Multiple%2520Ligands.%2520An%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6523%26epage%3D6543%26doi%3D10.1021%2Fjm058225d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Boyle, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meegan, M. J.</span></span> <span> </span><span class="NLM_article-title">Designed Multiple Ligands for Cancer Therapy</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4722</span>– <span class="NLM_lpage">4737</span>, <span class="refDoi"> DOI: 10.2174/092986711797535344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.2174%2F092986711797535344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21919848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyhsLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=4722-4737&author=N.+M.+O%E2%80%99Boyleauthor=M.+J.+Meegan&title=Designed+Multiple+Ligands+for+Cancer+Therapy&doi=10.2174%2F092986711797535344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Designed multiple ligands for cancer therapy</span></div><div class="casAuthors">O'Boyle, N. M.; Meegan, M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">4722-4737</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The concept of a single chem. entity with desirable activity at more than one biol. target is an attractive one.  Increasingly, multiple complex biochem. pathways are implicated in a variety of diseases including cancer.  Successful treatment of these conditions often depends on pharmaceutical intervention at multiple pathways, with a combination of different drugs.  Designed multiple ligands (DMLs) are drugs which act at multiple biomol. targets.  Numerous terms have been used to describe such ligands, including multiple-target directed ligands, heterodimers, promiscuous drugs and pan-agonists.  However, although there are many reported examples of multiple-targeting anti-cancer agents, no review of these has been presented to date.  A huge variety of biol. signalling-pathways, proteins and enzymes are currently targeted and implicated in the pathogenesis of cancer.  This review will provide an overview of reported designed multiple ligands for cancer and an exploration of the advantages and drawbacks of such compds.  The review also provides brief commentaries on the biol. processes and proteins that are currently targeted in cancer therapy and the potential for dual or triple targeting of these with designed multiple ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS2P_3Ows00bVg90H21EOLACvtfcHk0lgVZ0SKrYjUqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyhsLnF&md5=6d9f613ed668a18663fc2369a1041341</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F092986711797535344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711797535344%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DMeegan%26aufirst%3DM.%2BJ.%26atitle%3DDesigned%2520Multiple%2520Ligands%2520for%2520Cancer%2520Therapy%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D4722%26epage%3D4737%26doi%3D10.2174%2F092986711797535344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costantino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlocco, D.</span></span> <span> </span><span class="NLM_article-title">Designed Multiple Ligands: Basic Research vs Clinical Outcomes</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3353</span>– <span class="NLM_lpage">3387</span>, <span class="refDoi"> DOI: 10.2174/092986712801215883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.2174%2F092986712801215883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=22680630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOhtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=3353-3387&author=L.+Costantinoauthor=D.+Barlocco&title=Designed+Multiple+Ligands%3A+Basic+Research+vs+Clinical+Outcomes&doi=10.2174%2F092986712801215883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Designed multiple ligands: basic research vs clinical outcomes</span></div><div class="casAuthors">Costantino, L.; Barlocco, D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3353-3387</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The clin. treatment of multifactorial pathologies (e.g. cancer, Alzheimer's disease, psychiatric disorders), is still a major challenge.  Many researches have been published dealing with the design of multiple ligands, able to act at the same time towards several targets relevant for a given pathol.  In the present review, the clin. results about these compds. have been reported, together with the design strategy adopted, in order to allow a crit. evaluation of the outcomes of these efforts.  What is emerging is that several effective design strategies of multitarget compds. are available, and the choice among these appears to be dependent on the therapeutic area considered.  However, at present, besides multitarget drugs discovered during optimization efforts by means of phenotypic assays, drug coadministrations or fixed dose formulations appear to be more useful therapeutic options than designed multiple ligands; this scenario will change when systems biol. will be able to select crit. targets, i.e. nodal proteins that should be inhibited in order to obtain a therapeutic action; at this point, the design of multiple ligands will allow a renaissance of medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobv_P2UM6L_7Vg90H21EOLACvtfcHk0lh9IUi1kUNhpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOhtrrN&md5=82123bbafc10a19c65a0bc0c5be72546</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F092986712801215883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712801215883%26sid%3Dliteratum%253Aachs%26aulast%3DCostantino%26aufirst%3DL.%26aulast%3DBarlocco%26aufirst%3DD.%26atitle%3DDesigned%2520Multiple%2520Ligands%253A%2520Basic%2520Research%2520vs%2520Clinical%2520Outcomes%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D3353%26epage%3D3387%26doi%3D10.2174%2F092986712801215883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talevi, A.</span></span> <span> </span><span class="NLM_article-title">Multi-Target Pharmacology: Possibilities and Limitations of the ″Skeleton Key Approach″ from a Medicinal Chemist Perspective</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">205</span>, <span class="refDoi"> DOI: 10.3389/fphar.2015.00205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3389%2Ffphar.2015.00205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=26441661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1amurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=205&author=A.+Talevi&title=Multi-Target+Pharmacology%3A+Possibilities+and+Limitations+of+the+%E2%80%B3Skeleton+Key+Approach%E2%80%B3+from+a+Medicinal+Chemist+Perspective&doi=10.3389%2Ffphar.2015.00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target pharmacology: possibilities and limitations of the "skeleton key approach" from a medicinal chemist perspective</span></div><div class="casAuthors">Talevi, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">205/1-205/7</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Multi-target drugs have raised considerable interest in the last decade owing to their advantages in the treatment of complex diseases and health conditions linked to drug resistance issues.  Prospective drug repositioning to treat comorbid conditions is an addnl., overlooked application of multi-target ligands.  While medicinal chemists usually rely on some version of the lock and key paradigm to design novel therapeutics, modern pharmacol. recognizes that the mid- and long-term effects of a given drug on a biol. system may depend not only on the specific ligand-target recognition events but also on the influence of the repeated administration of a drug on the cell gene signature.  The design of multi-target agents usually imposes challenging restrictions on the topol. or flexibility of the candidate drugs, which are briefly discussed in the present article.  Finally, computational strategies to approach the identification of novel multi-target agents are overviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSdZokyMjNSLVg90H21EOLACvtfcHk0lh9IUi1kUNhpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1amurnM&md5=169718a4644d962da33806772a9a33e3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2015.00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2015.00205%26sid%3Dliteratum%253Aachs%26aulast%3DTalevi%26aufirst%3DA.%26atitle%3DMulti-Target%2520Pharmacology%253A%2520Possibilities%2520and%2520Limitations%2520of%2520the%2520%25E2%2580%25B3Skeleton%2520Key%2520Approach%25E2%2580%25B3%2520from%2520a%2520Medicinal%2520Chemist%2520Perspective%26jtitle%3DFront.%2520Pharmacol.%26date%3D2015%26volume%3D6%26spage%3D205%26doi%3D10.3389%2Ffphar.2015.00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, D. F.</span></span> <span> </span><span class="NLM_article-title">Designing Multi-Targeted Agents: An Emerging Anticancer Drug Discovery Paradigm</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.ejmech.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28494256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsV2rtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2017&pages=195-211&author=R.+G.+Fuauthor=Y.+Sunauthor=W.+B.+Shengauthor=D.+F.+Liao&title=Designing+Multi-Targeted+Agents%3A+An+Emerging+Anticancer+Drug+Discovery+Paradigm&doi=10.1016%2Fj.ejmech.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Designing multi-targeted agents: An emerging anticancer drug discovery paradigm</span></div><div class="casAuthors">Fu, Rong-geng; Sun, Yuan; Sheng, Wen-bing; Liao, Duan-fang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">195-211</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The dominant paradigm in drug discovery is to design ligands with max. selectivity to act on individual drug targets.  With the target-based approach, many new chem. entities have been discovered, developed, and further approved as drugs.  However, there are a large no. of complex diseases such as cancer that cannot be effectively treated or cured only with one medicine to modulate the biol. function of a single target.  As simultaneous intervention of two (or multiple) cancer progression relevant targets has shown improved therapeutic efficacy, the innovation of multi-targeted drugs has become a promising and prevailing research topic and numerous multi-targeted anticancer agents are currently at various developmental stages.  However, most multi-pharmacophore scaffolds are usually discovered by serendipity or screening, while rational design by combining existing pharmacophore scaffolds remains an enormous challenge.  In this review, four types of multi-pharmacophore modes are discussed, and the examples from literature will be used to introduce attractive lead compds. with the capability of simultaneously interfering with different enzyme or signaling pathway of cancer progression, which will reveal the trends and insights to help the design of the next generation multi-targeted anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGjXR-zscrArVg90H21EOLACvtfcHk0lh9IUi1kUNhpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsV2rtL8%253D&md5=511561bd2cd186d5dafa905b7257bf32</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DR.%2BG.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DSheng%26aufirst%3DW.%2BB.%26aulast%3DLiao%26aufirst%3DD.%2BF.%26atitle%3DDesigning%2520Multi-Targeted%2520Agents%253A%2520An%2520Emerging%2520Anticancer%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D136%26spage%3D195%26epage%3D211%26doi%3D10.1016%2Fj.ejmech.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastasio, T. J.</span></span> <span> </span><span class="NLM_article-title">Editorial: Computational and Experimental Approaches in Multi-Target Pharmacology</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">443</span>, <span class="refDoi"> DOI: 10.3389/fphar.2017.00443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3389%2Ffphar.2017.00443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28713280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC1cjotFaltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=443&author=T.+J.+Anastasio&title=Editorial%3A+Computational+and+Experimental+Approaches+in+Multi-Target+Pharmacology&doi=10.3389%2Ffphar.2017.00443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Editorial: Computational and Experimental Approaches in Multi-target Pharmacology</span></div><div class="casAuthors">Anastasio Thomas J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">443</span>
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTo-NN9xtLw775oR8RchjrrfW6udTcc2ebTO3N_S8Jaibntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjotFaltg%253D%253D&md5=fb9b52602bd095218f8cc00bacbfea4b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2017.00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2017.00443%26sid%3Dliteratum%253Aachs%26aulast%3DAnastasio%26aufirst%3DT.%2BJ.%26atitle%3DEditorial%253A%2520Computational%2520and%2520Experimental%2520Approaches%2520in%2520Multi-Target%2520Pharmacology%26jtitle%3DFront.%2520Pharmacol.%26date%3D2017%26volume%3D8%26spage%3D443%26doi%3D10.3389%2Ffphar.2017.00443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Proschak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology by Design: A Medicinal Chemist’s Perspective on Multitargeting Compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKms7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=420-444&author=E.+Proschakauthor=H.+Starkauthor=D.+Merk&title=Polypharmacology+by+Design%3A+A+Medicinal+Chemist%E2%80%99s+Perspective+on+Multitargeting+Compounds&doi=10.1021%2Facs.jmedchem.8b00760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology by Design: A Medicinal Chemist's Perspective on Multitargeting Compounds</span></div><div class="casAuthors">Proschak, Ewgenij; Stark, Holger; Merk, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">420-444</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multitargeting compds. comprising activity on more than a single biol. target have gained remarkable relevance in drug discovery owing to the complexity of multifactorial diseases such as cancer, inflammation, or the metabolic syndrome.  Polypharmacol. drug profiles can produce additive or synergistic effects while reducing side effects and significantly contribute to the high therapeutic success of indispensable drugs such as aspirin.  While their identification has long been the result of serendipity, medicinal chem. now tends to design polypharmacol.  Modern in vitro pharmacol. methods and chem. probes allow a systematic search for rational target combinations and recent innovations in computational technologies, crystallog., or fragment-based design equip multitarget compd. development with valuable tools.  In this Perspective, we analyze the relevance of multiple ligands in drug discovery and the versatile toolbox to design polypharmacol.  We conclude that despite some characteristic challenges remaining unresolved, designed polypharmacol. holds enormous potential to secure future therapeutic innovation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOEwJSxWw3vbVg90H21EOLACvtfcHk0lh4xPKGSSwzzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKms7fE&md5=6dc4b8e1c4777372396626140a302663</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00760%26sid%3Dliteratum%253Aachs%26aulast%3DProschak%26aufirst%3DE.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DMerk%26aufirst%3DD.%26atitle%3DPolypharmacology%2520by%2520Design%253A%2520A%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Perspective%2520on%2520Multitargeting%2520Compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D420%26epage%3D444%26doi%3D10.1021%2Facs.jmedchem.8b00760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span> <span> </span><span class="NLM_article-title">Harnessing Polypharmacology with Medicinal Chemistry</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVWjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=273-275&author=M.+L.+Bolognesi&title=Harnessing+Polypharmacology+with+Medicinal+Chemistry&doi=10.1021%2Facsmedchemlett.9b00039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing Polypharmacology with Medicinal Chemistry</span></div><div class="casAuthors">Bolognesi, Maria Laura</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">273-275</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polypharmacol. has expanded enormously over the last ten years, with several multitarget drugs (MTDs) already in the market.  This Viewpoint provides a basis for a discussion about the crit. need to develop MTDs in a more rationale and conscious way.  A checklist to maximize success in polypharmacol. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4yXxJ_uU72bVg90H21EOLACvtfcHk0lh4xPKGSSwzzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVWjt7g%253D&md5=28a8cc47ae2966a2ecd8a411adfb1ef5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00039%26sid%3Dliteratum%253Aachs%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26atitle%3DHarnessing%2520Polypharmacology%2520with%2520Medicinal%2520Chemistry%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D273%26epage%3D275%26doi%3D10.1021%2Facsmedchemlett.9b00039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Signalling through the Lipid Products of Phosphoinositide-3-OH Kinase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1038/42648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2F42648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=9192891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADyaK2sXktVWgt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=1997&pages=673-676&author=A.+Tokerauthor=L.+C.+Cantley&title=Signalling+through+the+Lipid+Products+of+Phosphoinositide-3-OH+Kinase&doi=10.1038%2F42648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Signalling through the lipid products of phosphoinositide-3-OH kinase</span></div><div class="casAuthors">Toker, Alex; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">6634</span>),
    <span class="NLM_cas:pages">673-676</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">A review, with 47 refs.  When a stimulatory agonist mol. binds at the exterior of the cell membrane, a second messenger transduces the signal to the interior of the cell.  Second messengers can be derived from phospholipids in the membrane by the action of the enzymes phospholipase C or phosphoinositide-3-OH kinase (PI(3)K).  PI(3)K is a key player in many cellular responses, including the movement of organelle membranes, shape alteration through rearrangement of cytoskeletal actin, transformation and chemotaxis.  But how PI(3)K mediates these responses is only now becoming clear.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbe14cub5uAbVg90H21EOLACvtfcHk0lh4xPKGSSwzzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktVWgt7c%253D&md5=9c0d3fdb3a9a10027050c09ed3def89a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2F42648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F42648%26sid%3Dliteratum%253Aachs%26aulast%3DToker%26aufirst%3DA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DSignalling%2520through%2520the%2520Lipid%2520Products%2520of%2520Phosphoinositide-3-OH%2520Kinase%26jtitle%3DNature%26date%3D1997%26volume%3D387%26spage%3D673%26epage%3D676%26doi%3D10.1038%2F42648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanhaesebroeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillermet-Guibert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graupera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilanges, B.</span></span> <span> </span><span class="NLM_article-title">The Emerging Mechanisms of Isoform-Specific PI3K Signalling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1038/nrm2882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrm2882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=20379207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVersrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=329-341&author=B.+Vanhaesebroeckauthor=J.+Guillermet-Guibertauthor=M.+Grauperaauthor=B.+Bilanges&title=The+Emerging+Mechanisms+of+Isoform-Specific+PI3K+Signalling&doi=10.1038%2Fnrm2882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging mechanisms of isoform-specific PI3K signaling</span></div><div class="casAuthors">Vanhaesebroeck, Bart; Guillermet-Guibert, Julie; Graupera, Mariona; Bilanges, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">329-341</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Phosphoinositide 3-kinases (PI3Ks) function early in intracellular signal transduction pathways and affect many biol. functions.  A further level of complexity derives from the existence of eight PI3K isoforms, which are divided into class I, class II and class III PI3Ks.  PI3K signaling has been implicated in metabolic control, immunity, angiogenesis and cardiovascular homeostasis, and is one of the most frequently deregulated pathways in cancer.  PI3K inhibitors have recently entered clin. trials in oncol.  A better understanding of how the different PI3K isoforms are regulated and control signaling could uncover their roles in pathol. and reveal in which disease contexts their blockade could be most beneficial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1WFE8yaCnkrVg90H21EOLACvtfcHk0ljgiPoxNnw5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVersrc%253D&md5=d8507e3901857b68bb0a48dc7fea2b5b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrm2882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2882%26sid%3Dliteratum%253Aachs%26aulast%3DVanhaesebroeck%26aufirst%3DB.%26aulast%3DGuillermet-Guibert%26aufirst%3DJ.%26aulast%3DGraupera%26aufirst%3DM.%26aulast%3DBilanges%26aufirst%3DB.%26atitle%3DThe%2520Emerging%2520Mechanisms%2520of%2520Isoform-Specific%2520PI3K%2520Signalling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2010%26volume%3D11%26spage%3D329%26epage%3D341%26doi%3D10.1038%2Fnrm2882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the Phosphoinositide 3-Kinase Pathway in Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">627</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd2926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrd2926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=19644473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Wgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=627-644&author=P.+Liuauthor=H.+Chengauthor=T.+M.+Robertsauthor=J.+J.+Zhao&title=Targeting+the+Phosphoinositide+3-Kinase+Pathway+in+Cancer&doi=10.1038%2Fnrd2926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the phosphoinositide 3-kinase pathway in cancer</span></div><div class="casAuthors">Liu, Pixu; Cheng, Hailing; Roberts, Thomas M.; Zhao, Jean J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">627-644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The phosphoinositide 3-kinase (PI3K) pathway is a key signal transduction system that links oncogenes and multiple receptor classes to many essential cellular functions, and is perhaps the most commonly activated signalling pathway in human cancer.  This pathway therefore presents both an opportunity and a challenge for cancer therapy.  Even as inhibitors that target PI3K isoforms and other major nodes in the pathway, including AKT and mammalian target of rapamycin (mTOR), reach clin. trials, major issues remain.  Here, we highlight recent progress that has been made in our understanding of the PI3K pathway and discuss the potential of and challenges for the development of therapeutic agents that target this pathway in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUk9T-ZoNDtLVg90H21EOLACvtfcHk0ljgiPoxNnw5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Wgtb4%253D&md5=fdc94abd15178a9dbe7c56dafbfb7787</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd2926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2926%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26atitle%3DTargeting%2520the%2520Phosphoinositide%25203-Kinase%2520Pathway%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D627%26epage%3D644%26doi%3D10.1038%2Fnrd2926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fruman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rommel, C.</span></span> <span> </span><span class="NLM_article-title">PI3K and Cancer: Lessons, Challenges and Opportunities</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1038/nrd4204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrd4204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=24481312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=140-156&author=D.+A.+Frumanauthor=C.+Rommel&title=PI3K+and+Cancer%3A+Lessons%2C+Challenges+and+Opportunities&doi=10.1038%2Fnrd4204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PI3K and cancer: lessons, challenges and opportunities</span></div><div class="casAuthors">Fruman, David A.; Rommel, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">140-156</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The central role of phosphoinositide 3-kinase (PI3K) activation in tumor cell biol. has prompted a sizeable effort to target PI3K and/or downstream kinases such as AKT and mammalian target of rapamycin (mTOR) in cancer.  However, emerging clin. data show limited single-agent activity of inhibitors targeting PI3K, AKT or mTOR at tolerated doses.  One exception is the response to PI3Kδ inhibitors in chronic lymphocytic leukemia, where a combination of cell-intrinsic and -extrinsic activities drive efficacy.  Here, we review key challenges and opportunities for the clin. development of inhibitors targeting the PI3K-AKT-mTOR pathway.  Through a greater focus on patient selection, increased understanding of immune modulation and strategic application of rational combinations, it should be possible to realize the potential of this promising class of targeted anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlD2rY54m9LbVg90H21EOLACvtfcHk0ljgiPoxNnw5lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SnsLc%253D&md5=c4f6b4b9f9fcacd6ef55eb17d3a4f425</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd4204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4204%26sid%3Dliteratum%253Aachs%26aulast%3DFruman%26aufirst%3DD.%2BA.%26aulast%3DRommel%26aufirst%3DC.%26atitle%3DPI3K%2520and%2520Cancer%253A%2520Lessons%252C%2520Challenges%2520and%2520Opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D140%26epage%3D156%26doi%3D10.1038%2Fnrd4204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stark, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriskantharajah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessel, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okkenhaug, K.</span></span> <span> </span><span class="NLM_article-title">PI3K Inhibitors in Inflammation, Autoimmunity and Cancer</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2015.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.coph.2015.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=26093105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWqs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=82-91&author=A.+K.+Starkauthor=S.+Sriskantharajahauthor=E.+M.+Hesselauthor=K.+Okkenhaug&title=PI3K+Inhibitors+in+Inflammation%2C+Autoimmunity+and+Cancer&doi=10.1016%2Fj.coph.2015.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PI3K inhibitors in inflammation, autoimmunity and cancer</span></div><div class="casAuthors">Stark, Anne-Katrien; Sriskantharajah, Srividya; Hessel, Edith M.; Okkenhaug, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-91</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The healthy immune system protects against infection and malignant transformation without causing significant damage to host tissues.  Immune dysregulation results in diverse pathologies including autoimmune disease, chronic inflammatory disorders, allergies as well as immune deficiencies and cancer.  Phosphoinositide 3-kinase (PI3K) signalling has been shown to be a key pathway in the regulation of the immune response and continues to be the focus of intense research.  In recent years we have gained detailed understanding of PI3K signalling, and saw the development of potent and highly selective small mol. inhibitors, of which several are currently in clin. trials for the treatment of immune-related disorders and cancer.  The role of PI3K signalling in the immune response has been the subject of detailed reviews; here we focus on relevant recent progress in pre-clin. and clin. development of PI3K inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNWPQZuO9fE7Vg90H21EOLACvtfcHk0lg-uIVTFAeucQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWqs7%252FM&md5=382eb159c8ec5e3a4bc14f71f54dd295</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DStark%26aufirst%3DA.%2BK.%26aulast%3DSriskantharajah%26aufirst%3DS.%26aulast%3DHessel%26aufirst%3DE.%2BM.%26aulast%3DOkkenhaug%26aufirst%3DK.%26atitle%3DPI3K%2520Inhibitors%2520in%2520Inflammation%252C%2520Autoimmunity%2520and%2520Cancer%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D82%26epage%3D91%26doi%3D10.1016%2Fj.coph.2015.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garces, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span> <span> </span><span class="NLM_article-title">Class 1 PI3K Clinical Candidates and Recent Inhibitor Design Strategies: A Medicinal Chemistry Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4815</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFOku7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4815-4850&author=A.+E.+Garcesauthor=M.+J.+Stocks&title=Class+1+PI3K+Clinical+Candidates+and+Recent+Inhibitor+Design+Strategies%3A+A+Medicinal+Chemistry+Perspective&doi=10.1021%2Facs.jmedchem.8b01492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Class 1 PI3K Clinical Candidates and Recent Inhibitor Design Strategies: A Medicinal Chemistry Perspective</span></div><div class="casAuthors">Garces, Aimie E.; Stocks, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4815-4850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol 3-kinases (PI3Ks) are a family of lipid kinases that phosphorylate the 3-OH of the inositol ring of phosphoinositides, and deregulation of this pathway has implications in many diseases.  The search for novel PI3K inhibitors has been at the forefront of academic and industrial medicinal chem. with over 600 medicinal chem.-based publications and patents appearing to date, leading to 38 clin. candidates and the launch of two drugs, idelalisib in 2014 and copanlisib in 2017.  This Perspective will discuss medicinal chem. design approaches to novel isoform-selective inhibitors through consideration of brief case histories of compds. that have progressed into clin. development or that have revealed new structural motifs in this highly competitive area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUu2mgndklHbVg90H21EOLACvtfcHk0lg-uIVTFAeucQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFOku7bK&md5=907395a39ed6fb683464ee92c6a2537f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01492%26sid%3Dliteratum%253Aachs%26aulast%3DGarces%26aufirst%3DA.%2BE.%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26atitle%3DClass%25201%2520PI3K%2520Clinical%2520Candidates%2520and%2520Recent%2520Inhibitor%2520Design%2520Strategies%253A%2520A%2520Medicinal%2520Chemistry%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4815%26epage%3D4850%26doi%3D10.1021%2Facs.jmedchem.8b01492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span> <span> </span><span class="NLM_article-title">Targeting PI3K in Cancer: Mechanisms and Advances in Clinical Trials</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">26</span>, <span class="refDoi"> DOI: 10.1186/s12943-019-0954-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1186%2Fs12943-019-0954-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30782187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BB3cflt1Kluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=26&author=J.+Yangauthor=J.+Nieauthor=X.+Maauthor=Y.+Weiauthor=Y.+Pengauthor=X.+Wei&title=Targeting+PI3K+in+Cancer%3A+Mechanisms+and+Advances+in+Clinical+Trials&doi=10.1186%2Fs12943-019-0954-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting PI3K in cancer: mechanisms and advances in clinical trials</span></div><div class="casAuthors">Yang Jing; Nie Ji; Ma Xuelei; Wei Yuquan; Peng Yong; Wei Xiawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phosphatidylinositol-3-kinase (PI3K)/AKT/mammalian target of rapamycin (mTOR) signaling is one of the most important intracellular pathways, which can be considered as a master regulator for cancer.  Enormous efforts have been dedicated to the development of drugs targeting PI3K signaling, many of which are currently employed in clinical trials evaluation, and it is becoming increasingly clear that PI3K inhibitors are effective in inhibiting tumor progression.  PI3K inhibitors are subdivided into dual PI3K/mTOR inhibitors, pan-PI3K inhibitors and isoform-specific inhibitors.  In this review, we performed a critical review to summarize the role of the PI3K pathway in tumor development, recent PI3K inhibitors development based on clinical trials, and the mechanisms of resistance to PI3K inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQGm3fFMcHFSMScOqQTSkOffW6udTcc2eb5CRAfrJvMgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cflt1Kluw%253D%253D&md5=4267e4df49544ebe11ab4255e69e5bbe</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1186%2Fs12943-019-0954-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-019-0954-x%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DNie%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DX.%26atitle%3DTargeting%2520PI3K%2520in%2520Cancer%253A%2520Mechanisms%2520and%2520Advances%2520in%2520Clinical%2520Trials%26jtitle%3DMol.%2520Cancer%26date%3D2019%26volume%3D18%26spage%3D26%26doi%3D10.1186%2Fs12943-019-0954-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Markham, A.</span></span> <span> </span><span class="NLM_article-title">Alpelisib: First Global Approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1249</span>– <span class="NLM_lpage">1253</span>, <span class="refDoi"> DOI: 10.1007/s40265-019-01161-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1007%2Fs40265-019-01161-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=31256368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCiur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=1249-1253&author=A.+Markham&title=Alpelisib%3A+First+Global+Approval&doi=10.1007%2Fs40265-019-01161-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Alpelisib: First Global Approval</span></div><div class="casAuthors">Markham, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1249-1253</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Alpelisib (Piqray)-an orally available phosphatidylinositol 3-kinase (PI3K) inhibitor with specific activity against PI3K alpha (PI3Kα)-is being developed by Novartis for the treatment of breast cancer.  Alpelisib has demonstrated efficacy in combination with fulvestrant as treatment for hormone receptor (HR)-pos., human epidermal growth factor receptor-2 (HER2)-neg. breast cancer in patients with a PIK3CA mutation and was recently approved for this indication in the USA.  This article summarizes the milestones in the development of alpelisib leading to this first approval.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX1sEhr0QGHrVg90H21EOLACvtfcHk0lg-uIVTFAeucQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCiur%252FN&md5=4d9e9d1e3742a9bce160ca2c7987ce6c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs40265-019-01161-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-019-01161-6%26sid%3Dliteratum%253Aachs%26aulast%3DMarkham%26aufirst%3DA.%26atitle%3DAlpelisib%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2019%26volume%3D79%26spage%3D1249%26epage%3D1253%26doi%3D10.1007%2Fs40265-019-01161-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pons-Tostivint, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillermet-Guibert, J.</span></span> <span> </span><span class="NLM_article-title">Targeting PI3K Signaling in Combination Cancer Therapy</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">454</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2017.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.trecan.2017.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28718419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVOksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=454-469&author=E.+Pons-Tostivintauthor=B.+Thibaultauthor=J.+Guillermet-Guibert&title=Targeting+PI3K+Signaling+in+Combination+Cancer+Therapy&doi=10.1016%2Fj.trecan.2017.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting PI3K Signaling in Combination Cancer Therapy</span></div><div class="casAuthors">Pons-Tostivint, Elvire; Thibault, Benoit; Guillermet-Guibert, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">454-469</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeting upstream phosphatidylinositol-3-kinases (PI3Ks) in the PI3K/Akt/mTOR pathway appears to be a promising therapy in solid cancers; however, first early clin. trials with PI3K inhibitors in monotherapy have been disappointing.  A massive array of preclin. and clin. trials are currently evaluating combinations of PI3K inhibitors in targeted therapies.  These combinations include co-treatments with drugs directed against other intra-/extracellular signaling mols., nuclear hormone receptors, DNA damage repair enzymes, and immune modulators.  We review the literature and pinpoint mechanisms of action in different genomic and organ contexts.  Combinatorial approaches are potentially superior to monotherapies and should become alternative clin. strategies to treat cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotS0vLAVeC87Vg90H21EOLACvtfcHk0ljcjMBqh_kyMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVOksQ%253D%253D&md5=3b905f4d00de4428b52078434ad11f12</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DPons-Tostivint%26aufirst%3DE.%26aulast%3DThibault%26aufirst%3DB.%26aulast%3DGuillermet-Guibert%26aufirst%3DJ.%26atitle%3DTargeting%2520PI3K%2520Signaling%2520in%2520Combination%2520Cancer%2520Therapy%26jtitle%3DTrends%2520Cancer%26date%3D2017%26volume%3D3%26spage%3D454%26epage%3D469%26doi%3D10.1016%2Fj.trecan.2017.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">West, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">New and Emerging HDAC Inhibitors for Cancer Treatment</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1172/JCI69738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1172%2FJCI69738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=24382387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFejtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=30-39&author=A.+C.+Westauthor=R.+W.+Johnstone&title=New+and+Emerging+HDAC+Inhibitors+for+Cancer+Treatment&doi=10.1172%2FJCI69738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">New and emerging HDAC inhibitors for cancer treatment</span></div><div class="casAuthors">West, Alison C.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-39</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  Epigenetic enzymes are often dysregulated in human tumors through mutation, altered expression, or inappropriate recruitment to certain loci.  The identification of these enzymes and their partner proteins has driven the rapid development of small-mol. inhibitors that target the cancer epigenome.  Herein, the authors discuss the influence of aberrantly regulated histone deacetylases (HDACs) in tumorigenesis.  The authors examine HDAC inhibitors (HDACis) targeting class I, II, and IV HDACs that are currently under development for use as anticancer agents following the FDA approval of two HDACis, vorinostat and romidepsin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELUU7P0B_grVg90H21EOLACvtfcHk0ljcjMBqh_kyMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFejtg%253D%253D&md5=ab59960223b33e89725f6539b031f531</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1172%2FJCI69738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI69738%26sid%3Dliteratum%253Aachs%26aulast%3DWest%26aufirst%3DA.%2BC.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DNew%2520and%2520Emerging%2520HDAC%2520Inhibitors%2520for%2520Cancer%2520Treatment%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2014%26volume%3D124%26spage%3D30%26epage%3D39%26doi%3D10.1172%2FJCI69738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Trend of Histone Deacetylase Inhibitors in Cancer Therapy: Isoform Selectivity or Multitargeted Strategy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1002/med.21320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1002%2Fmed.21320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=24782318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVClsb3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=63-84&author=L.+Zhangauthor=Y.+Hanauthor=Q.+Jiangauthor=C.+Wangauthor=X.+Chenauthor=X.+Liauthor=F.+Xuauthor=Y.+Jiangauthor=Q.+Wangauthor=W.+Xu&title=Trend+of+Histone+Deacetylase+Inhibitors+in+Cancer+Therapy%3A+Isoform+Selectivity+or+Multitargeted+Strategy&doi=10.1002%2Fmed.21320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Trend of Histone Deacetylase Inhibitors in Cancer Therapy: Isoform Selectivity or Multitargeted Strategy</span></div><div class="casAuthors">Zhang, Lei; Han, Yantao; Jiang, Qixiao; Wang, Chunbo; Chen, Xuehong; Li, Xiaoguang; Xu, Fuming; Jiang, Yuqi; Wang, Qiang; Xu, Wenfang</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-84</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Pharmacol. inhibition of histone deacetylases (HDACs) has been successfully applied in the treatment of a wide range of disorders, including Parkinson's disease, infection, cardiac diseases, inflammation, and esp. cancer.  HDAC inhibitors (HDACIs) have been proved to be effective antitumor agents by various stages of investigation.  At present, there are two opposite focuses of HDACI design in the cancer therapy, highly selective inhibitor strategy and dual- or multitargeted inhibitors.  The former method, which is supposed to elucidate the function of individual HDAC and provide candidate inhibitors with fewer side effects, has been widely accepted by the inhibitor developer.  The latter approach, though less practiced, has promising potential for the antitumor therapy based on HDACIs.  Effective HDACIs, some of which are in clinic anticancer research, have been developed by both methods.  In order to gain insight into HDACI design, the strategies and achievements of the two diverse methods are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm6HJUNw_9dLVg90H21EOLACvtfcHk0ljcjMBqh_kyMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVClsb3M&md5=791e7d6b057c92f206c8d0d75099e021</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fmed.21320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21320%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DTrend%2520of%2520Histone%2520Deacetylase%2520Inhibitors%2520in%2520Cancer%2520Therapy%253A%2520Isoform%2520Selectivity%2520or%2520Multitargeted%2520Strategy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D63%26epage%3D84%26doi%3D10.1002%2Fmed.21320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckschlager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiborova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrabeta, J.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitors as Anticancer Drugs</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1414</span>, <span class="refDoi"> DOI: 10.3390/ijms18071414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3390%2Fijms18071414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28671573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1414&author=T.+Eckschlagerauthor=J.+Plchauthor=M.+Stiborovaauthor=J.+Hrabeta&title=Histone+Deacetylase+Inhibitors+as+Anticancer+Drugs&doi=10.3390%2Fijms18071414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors as anticancer drugs</span></div><div class="casAuthors">Eckschlager, Tomas; Plch, Johana; Stiborova, Marie; Hrabeta, Jan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1414/1-1414/25</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes.  Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT).  HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response.  Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc.  HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy.  HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma.  Other HDAC inhibitors are in clin. trials for the treatment of hematol. and solid malignancies.  The results of such studies are promising but further larger studies are needed.  Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance.  Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclin. and clin. studies, including the combination with other therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNVhoLBAHnrVg90H21EOLACvtfcHk0lhBnMIcEISlyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P&md5=6018c7709a09548214bfe34bd91107a9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3390%2Fijms18071414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071414%26sid%3Dliteratum%253Aachs%26aulast%3DEckschlager%26aufirst%3DT.%26aulast%3DPlch%26aufirst%3DJ.%26aulast%3DStiborova%26aufirst%3DM.%26aulast%3DHrabeta%26aufirst%3DJ.%26atitle%3DHistone%2520Deacetylase%2520Inhibitors%2520as%2520Anticancer%2520Drugs%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1414%26doi%3D10.3390%2Fijms18071414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suraweera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Byrne, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, D. J.</span></span> <span> </span><span class="NLM_article-title">Combination Therapy with Histone Deacetylase Inhibitors (HDACi) for the Treatment of Cancer: Achieving the Full Therapeutic Potential of HDACi</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">92</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3389%2Ffonc.2018.00092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29651407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC1MjitlSksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=92&author=A.+Suraweeraauthor=K.+J.+O%E2%80%99Byrneauthor=D.+J.+Richard&title=Combination+Therapy+with+Histone+Deacetylase+Inhibitors+%28HDACi%29+for+the+Treatment+of+Cancer%3A+Achieving+the+Full+Therapeutic+Potential+of+HDACi&doi=10.3389%2Ffonc.2018.00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Combination Therapy With Histone Deacetylase Inhibitors (HDACi) for the Treatment of Cancer: Achieving the Full Therapeutic Potential of HDACi</span></div><div class="casAuthors">Suraweera Amila; O'Byrne Kenneth J; Richard Derek J; O'Byrne Kenneth J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">92</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Genetic and epigenetic changes in DNA are involved in cancer development and tumor progression.  Histone deacetylases (HDACs) are key regulators of gene expression that act as transcriptional repressors by removing acetyl groups from histones.  HDACs are dysregulated in many cancers, making them a therapeutic target for the treatment of cancer.  Histone deacetylase inhibitors (HDACi), a novel class of small-molecular therapeutics, are now approved by the Food and Drug Administration as anticancer agents.  While they have shown great promise, resistance to HDACi is often observed and furthermore, HDACi have shown limited success in treating solid tumors.  The combination of HDACi with standard chemotherapeutic drugs has demonstrated promising anticancer effects in both preclinical and clinical studies.  In this review, we summarize the research thus far on HDACi in combination therapy, with other anticancer agents and their translation into preclinical and clinical studies.  We additionally highlight the side effects associated with HDACi in cancer therapy and discuss potential biomarkers to either select or predict a patient's response to these agents, in order to limit the off-target toxicity associated with HDACi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTprF0Hdrz8rZZoeNcnHOQ9fW6udTcc2eaJ9KQmVWpNurntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjitlSksA%253D%253D&md5=2f0ace7215bc7519c23cad264f73693d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00092%26sid%3Dliteratum%253Aachs%26aulast%3DSuraweera%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Byrne%26aufirst%3DK.%2BJ.%26aulast%3DRichard%26aufirst%3DD.%2BJ.%26atitle%3DCombination%2520Therapy%2520with%2520Histone%2520Deacetylase%2520Inhibitors%2520%2528HDACi%2529%2520for%2520the%2520Treatment%2520of%2520Cancer%253A%2520Achieving%2520the%2520Full%2520Therapeutic%2520Potential%2520of%2520HDACi%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D92%26doi%3D10.3389%2Ffonc.2018.00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinoza-Delgado, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piekarz, R. L.</span></span> <span> </span><span class="NLM_article-title">Clinical Toxicities of Histone Deacetylase Inhibitors</span>. <i>Pharmaceuticals</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2751</span>– <span class="NLM_lpage">2767</span>, <span class="refDoi"> DOI: 10.3390/ph3092751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3390%2Fph3092751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27713375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSmtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=2751-2767&author=S.+Subramanianauthor=S.+E.+Batesauthor=J.+J.+Wrightauthor=I.+Espinoza-Delgadoauthor=R.+L.+Piekarz&title=Clinical+Toxicities+of+Histone+Deacetylase+Inhibitors&doi=10.3390%2Fph3092751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical toxicities of histone deacetylase inhibitors</span></div><div class="casAuthors">Subramanian, Srividya; Bates, Susan E.; Wright, John J.; Espinoza-Delgado, Igor; Piekarz, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceuticals</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2751-2767</span>CODEN:
                <span class="NLM_cas:coden">PHARH2</span>;
        ISSN:<span class="NLM_cas:issn">1424-8247</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  The HDAC inhibitors are a new family of antineoplastic agents.  Since the entry of these agents into our therapeutic armamentarium, there has been increasing interest in their use.  Although this family comprises chem. compds. from unrelated chem. classes that have different HDAC isoform specificities, they surprisingly have very similar toxicity profiles.  In contrast, the obsd. toxicity profile is somewhat different from that of traditional cytotoxic chemotherapeutic agents and from other epigenetic agents.  While some of the side effects may be familiar to the oncologist, others are less commonly seen.  As some patients remain on therapy for a prolonged period of time, the long-term sequelae need to be characterized.  In addn., since preclin. models suggest promising activity when used in combination with other antineoplastic agents, combination trials are being pursued.  It will thus be important to distinguish the relative toxicity attributed to these agents and be alert to the exacerbation of toxicities obsd. in single agent studies.  Notably, few of the agents in this class have completed phase 2 testing.  Consequently, more clin. experience is needed to det. the relative frequency of the obsd. side effects, and to identify and develop approaches to mitigate potential clin. sequelae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrriVU62aelgbVg90H21EOLACvtfcHk0lhBnMIcEISlyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSmtrvK&md5=d0937756ff0e5582c1e33901c913ff97</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fph3092751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph3092751%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DS.%26aulast%3DBates%26aufirst%3DS.%2BE.%26aulast%3DWright%26aufirst%3DJ.%2BJ.%26aulast%3DEspinoza-Delgado%26aufirst%3DI.%26aulast%3DPiekarz%26aufirst%3DR.%2BL.%26atitle%3DClinical%2520Toxicities%2520of%2520Histone%2520Deacetylase%2520Inhibitors%26jtitle%3DPharmaceuticals%26date%3D2010%26volume%3D3%26spage%3D2751%26epage%3D2767%26doi%3D10.3390%2Fph3092751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N. S.</span></span> <span> </span><span class="NLM_article-title">Panobinostat and Multiple Myeloma in 2018</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">516</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2017-0644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1634%2Ftheoncologist.2017-0644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29445026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC1MrhslOluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=516-517&author=A.+J.+Yeeauthor=N.+S.+Raje&title=Panobinostat+and+Multiple+Myeloma+in+2018&doi=10.1634%2Ftheoncologist.2017-0644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Panobinostat and Multiple Myeloma in 2018</span></div><div class="casAuthors">Yee Andrew J; Raje Noopur S; Yee Andrew J; Raje Noopur S</div><div class="citationInfo"><span class="NLM_cas:title">The oncologist</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">516-517</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShdJOozuyZkPtos8R3JUhXfW6udTcc2eYLjJPWN21s_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrhslOluw%253D%253D&md5=7202e92be5a6f4cc72d5317b35ec687a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2017-0644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2017-0644%26sid%3Dliteratum%253Aachs%26aulast%3DYee%26aufirst%3DA.%2BJ.%26aulast%3DRaje%26aufirst%3DN.%2BS.%26atitle%3DPanobinostat%2520and%2520Multiple%2520Myeloma%2520in%25202018%26jtitle%3DOncologist%26date%3D2018%26volume%3D23%26spage%3D516%26epage%3D517%26doi%3D10.1634%2Ftheoncologist.2017-0644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thurn, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munster, P. N.</span></span> <span> </span><span class="NLM_article-title">Rational Therapeutic Combinations with Histone Deacetylase Inhibitors for the Treatment of Cancer</span>. <i>Future Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.2217/fon.11.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.2217%2Ffon.11.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21345145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWlurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=263-283&author=K.+T.+Thurnauthor=S.+Thomasauthor=A.+Mooreauthor=P.+N.+Munster&title=Rational+Therapeutic+Combinations+with+Histone+Deacetylase+Inhibitors+for+the+Treatment+of+Cancer&doi=10.2217%2Ffon.11.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rational therapeutic combinations with histone deacetylase inhibitors for the treatment of cancer</span></div><div class="casAuthors">Thurn, K. Ted; Thomas, Scott; Moore, Amy; Munster, Pamela N.</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-283</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1479-6694</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) regulate the acetylation of a variety of histone and nonhistone proteins, controlling the transcription and regulation of genes involved in cell cycle control, proliferation, survival, DNA repair and differentiation.  Unsurprisingly, HDAC expression is frequently altered in hematol. and solid tumor malignancies.  Two HDAC inhibitors (vorinostat and romidepsin) have been approved by the US FDA for the treatment of cutaneous T-cell lymphoma.  As single agents, treatment with HDAC inhibitors has demonstrated limited clin. benefit for patients with solid tumors, prompting the investigation of novel treatment combinations with other cancer therapeutics.  In this article, the rationales and clin. progress of several combinations with HDAC inhibitors are presented, including DNA-damaging chemotherapeutic agents, radiotherapy, hormonal therapies, DNA methyltransferase inhibitors and various small-mol. inhibitors.  The future application of HDAC inhibitors as a treatment for cancer is discussed, examg. current hurdles to overcome before realizing the potential of this new approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn7EKQGlvu_LVg90H21EOLACvtfcHk0lhKEW-DKXq4BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWlurs%253D&md5=193fc07f4e19bf16fcd7ee202057ef7c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2217%2Ffon.11.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon.11.2%26sid%3Dliteratum%253Aachs%26aulast%3DThurn%26aufirst%3DK.%2BT.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DMoore%26aufirst%3DA.%26aulast%3DMunster%26aufirst%3DP.%2BN.%26atitle%3DRational%2520Therapeutic%2520Combinations%2520with%2520Histone%2520Deacetylase%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Cancer%26jtitle%3DFuture%2520Oncol.%26date%3D2011%26volume%3D7%26spage%3D263%26epage%3D283%26doi%3D10.2217%2Ffon.11.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lannutti, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almasan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsi, E. D.</span></span> <span> </span><span class="NLM_article-title">The Phosphatidylinositol 3-Kinases (PI3K) Inhibitor GS-1101 Synergistically Potentiates Histone Deacetylase Inhibitor-Induced Proliferation Inhibition and Apoptosis through the Inactivation of PI3K and Extracellular Signal-Regulated Kinase Pathways</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1111/bjh.12498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1111%2Fbjh.12498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=23889282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOgsr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2013&pages=72-80&author=J.+Bodoauthor=X.+Zhaoauthor=A.+Sharmaauthor=B.+T.+Hillauthor=C.+A.+Portellauthor=B.+J.+Lannuttiauthor=A.+Almasanauthor=E.+D.+Hsi&title=The+Phosphatidylinositol+3-Kinases+%28PI3K%29+Inhibitor+GS-1101+Synergistically+Potentiates+Histone+Deacetylase+Inhibitor-Induced+Proliferation+Inhibition+and+Apoptosis+through+the+Inactivation+of+PI3K+and+Extracellular+Signal-Regulated+Kinase+Pathways&doi=10.1111%2Fbjh.12498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The phosphatidylinositol 3-kinases (PI3K) inhibitor GS-1101 synergistically potentiates histone deacetylase inhibitor-induced proliferation inhibition and apoptosis through the inactivation of PI3K and extracellular signal-regulated kinase pathways</span></div><div class="casAuthors">Bodo, Juraj; Zhao, Xiaoxian; Sharma, Arishya; Hill, Brian T.; Portell, Craig A.; Lannutti, Brian J.; Almasan, Alexandru; Hsi, Eric D.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-80</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Previously, we showed that inhibition of the protein kinase C β (PKCβ)/AKT pathway augments engagement of the histone deacetylase inhibitor (HDI)-induced apoptosis in lymphoma cells.  In the present study, we investigated the cytotoxicity and mechanisms of cell death induced by the delta isoform-specific phosphatidylinositide 3-kinase (PI3K) inhibitor, GS-1101, in combination with the HDI, panobinostat (LBH589) and suberoylanilide hydroxamic acid (SAHA).  Lymphoma cell lines, primary non-Hodgkin Lymphoma (NHL) and chronic lymphocytic leukemia (CLL) cells were simultaneously treated with the HDI,LBH589 and GS-1101.  An interaction of the LBH589/GS-1101 combination was formally examd. by using various concns. of LBH589 and GS-1101.  Combined treatment resulted in a synergistic inhibition of proliferation and showed synergistic effect on apoptotic induction in all tested cell lines and primary NHL and CLL cells.  This study indicates that interference with PI3K signalling dramatically increases HDI-mediated apoptosis in malignant haematopoietic cells, possibly through both AKT-dependent or AKT- independent mechanisms.  Moreover, the increase in HDI-related apoptosis obsd. in PI3K inhibitor-treated cells appears to be related to the disruption of the extracellular signal-regulated kinase (ERK) signalling pathway.  This study provides a strong rational for testing the combination of PI3K inhibitors and HDI in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf_Pb47waY5bVg90H21EOLACvtfcHk0lhKEW-DKXq4BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOgsr7J&md5=ab27c7447ad089cc1d38f668dcc35760</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1111%2Fbjh.12498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.12498%26sid%3Dliteratum%253Aachs%26aulast%3DBodo%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DHill%26aufirst%3DB.%2BT.%26aulast%3DPortell%26aufirst%3DC.%2BA.%26aulast%3DLannutti%26aufirst%3DB.%2BJ.%26aulast%3DAlmasan%26aufirst%3DA.%26aulast%3DHsi%26aufirst%3DE.%2BD.%26atitle%3DThe%2520Phosphatidylinositol%25203-Kinases%2520%2528PI3K%2529%2520Inhibitor%2520GS-1101%2520Synergistically%2520Potentiates%2520Histone%2520Deacetylase%2520Inhibitor-Induced%2520Proliferation%2520Inhibition%2520and%2520Apoptosis%2520through%2520the%2520Inactivation%2520of%2520PI3K%2520and%2520Extracellular%2520Signal-Regulated%2520Kinase%2520Pathways%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2013%26volume%3D163%26spage%3D72%26epage%3D80%26doi%3D10.1111%2Fbjh.12498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garancher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esparza, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udaka, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seker-Cin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brabetz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogiso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kool, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wechsler-Reya, R. J.</span></span> <span> </span><span class="NLM_article-title">HDAC and PI3K Antagonists Cooperate to Inhibit Growth of MYC-Driven Medulloblastoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">323</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.ccell.2016.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=26977882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=311-323&author=Y.+Peiauthor=K.+W.+Liuauthor=J.+Wangauthor=A.+Garancherauthor=R.+Taoauthor=L.+A.+Esparzaauthor=D.+L.+Maierauthor=Y.+T.+Udakaauthor=N.+Muradauthor=S.+Morrissyauthor=H.+Seker-Cinauthor=S.+Brabetzauthor=L.+Qiauthor=M.+Kogisoauthor=S.+Schubertauthor=J.+M.+Olsonauthor=Y.+J.+Choauthor=X.+N.+Liauthor=J.+R.+Crawfordauthor=M.+L.+Levyauthor=M.+Koolauthor=S.+M.+Pfisterauthor=M.+D.+Taylorauthor=R.+J.+Wechsler-Reya&title=HDAC+and+PI3K+Antagonists+Cooperate+to+Inhibit+Growth+of+MYC-Driven+Medulloblastoma&doi=10.1016%2Fj.ccell.2016.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC and PI3K Antagonists Cooperate to Inhibit Growth of MYC-Driven Medulloblastoma</span></div><div class="casAuthors">Pei, Yanxin; Liu, Kun-Wei; Wang, Jun; Garancher, Alexandra; Tao, Ran; Esparza, Lourdes A.; Maier, Donna L.; Udaka, Yoko T.; Murad, Najiba; Morrissy, Sorana; Seker-Cin, Huriye; Brabetz, Sebastian; Qi, Lin; Kogiso, Mari; Schubert, Simone; Olson, James M.; Cho, Yoon-Jae; Li, Xiao-Nan; Crawford, John R.; Levy, Michael L.; Kool, Marcel; Pfister, Stefan M.; Taylor, Michael D.; Wechsler-Reya, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">311-323</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Medulloblastoma (MB) is a highly malignant pediatric brain tumor.  Despite aggressive therapy, many patients succumb to the disease, and survivors experience severe side effects from treatment.  MYC-driven MB has a particularly poor prognosis and would greatly benefit from more effective therapies.  We used an animal model of MYC-driven MB to screen for drugs that decrease viability of tumor cells.  Among the most effective compds. were histone deacetylase inhibitors (HDACIs).  HDACIs potently inhibit survival of MYC-driven MB cells in vitro, in part by inducing expression of the FOXO1 tumor suppressor gene.  HDACIs also synergize with phosphatidylinositol 3-kinase inhibitors to inhibit tumor growth in vivo.  These studies identify an effective combination therapy for the most aggressive form of MB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfHu8mC8vyrVg90H21EOLACvtfcHk0lh-8Y_5NMXwjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWitL0%253D&md5=c3e20bc1d1ef7af52c2a9ee935b8b7e5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGarancher%26aufirst%3DA.%26aulast%3DTao%26aufirst%3DR.%26aulast%3DEsparza%26aufirst%3DL.%2BA.%26aulast%3DMaier%26aufirst%3DD.%2BL.%26aulast%3DUdaka%26aufirst%3DY.%2BT.%26aulast%3DMurad%26aufirst%3DN.%26aulast%3DMorrissy%26aufirst%3DS.%26aulast%3DSeker-Cin%26aufirst%3DH.%26aulast%3DBrabetz%26aufirst%3DS.%26aulast%3DQi%26aufirst%3DL.%26aulast%3DKogiso%26aufirst%3DM.%26aulast%3DSchubert%26aufirst%3DS.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCho%26aufirst%3DY.%2BJ.%26aulast%3DLi%26aufirst%3DX.%2BN.%26aulast%3DCrawford%26aufirst%3DJ.%2BR.%26aulast%3DLevy%26aufirst%3DM.%2BL.%26aulast%3DKool%26aufirst%3DM.%26aulast%3DPfister%26aufirst%3DS.%2BM.%26aulast%3DTaylor%26aufirst%3DM.%2BD.%26aulast%3DWechsler-Reya%26aufirst%3DR.%2BJ.%26atitle%3DHDAC%2520and%2520PI3K%2520Antagonists%2520Cooperate%2520to%2520Inhibit%2520Growth%2520of%2520MYC-Driven%2520Medulloblastoma%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D311%26epage%3D323%26doi%3D10.1016%2Fj.ccell.2016.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernatchez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span> <span> </span><span class="NLM_article-title">Kinase and Histone Deacetylase Hybrid Inhibitors for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3171</span>– <span class="NLM_lpage">3183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00189</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00189" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKntbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3171-3183&author=Y.+Luanauthor=J.+Liauthor=J.+A.+Bernatchezauthor=R.+Li&title=Kinase+and+Histone+Deacetylase+Hybrid+Inhibitors+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.8b00189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase and Histone Deacetylase Hybrid Inhibitors for Cancer Therapy</span></div><div class="casAuthors">Luan, Yepeng; Li, Jerry; Bernatchez, Jean A.; Li, Rongshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3171-3183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs), encompassing at least 18 members, are promising targets for anticancer drug discovery and development.  To date, five histone deacetylase inhibitors (HDACis) have been approved for cancer treatment, and numerous others are undergoing clin. trials.  It has been well validated that an agent that can simultaneously and effectively inhibit two or more targets may offer greater therapeutic benefits over single-acting agents in preventing resistance to treatment and in potentiating synergistic effects.  A prime example of a bifunctional agent is the hybrid HDAC inhibitor.  In this perspective, the authors review the majority of reported kinase/HDAC hybrid inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvS3FgFwT7r7Vg90H21EOLACvtfcHk0lh-8Y_5NMXwjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKntbnF&md5=a75c9578b68044a0f702bbf901c3cdcf</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00189%26sid%3Dliteratum%253Aachs%26aulast%3DLuan%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DBernatchez%26aufirst%3DJ.%2BA.%26aulast%3DLi%26aufirst%3DR.%26atitle%3DKinase%2520and%2520Histone%2520Deacetylase%2520Hybrid%2520Inhibitors%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3171%26epage%3D3183%26doi%3D10.1021%2Facs.jmedchem.8b00189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Biological Evaluation of 2,3-Dihydroimidazo[1,2-C]Quinazoline Derivatives as Novel Phosphatidylinositol 3-Kinase and Histone Deacetylase Dual Inhibitors</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">52180</span>– <span class="NLM_lpage">52186</span>, <span class="refDoi"> DOI: 10.1039/C7RA08835C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1039%2FC7RA08835C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyqs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=52180-52186&author=Y.+Wuauthor=W.+Daiauthor=X.+Chenauthor=A.+Gengauthor=Y.+Chenauthor=T.+Luauthor=Y.+Zhu&title=Design%2C+Synthesis+and+Biological+Evaluation+of+2%2C3-Dihydroimidazo%5B1%2C2-C%5DQuinazoline+Derivatives+as+Novel+Phosphatidylinositol+3-Kinase+and+Histone+Deacetylase+Dual+Inhibitors&doi=10.1039%2FC7RA08835C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of 2,3-dihydroimidazo[1,2-c]quinazoline derivatives as novel phosphatidylinositol 3-kinase and histone deacetylase dual inhibitors</span></div><div class="casAuthors">Wu, Yichao; Dai, Weichen; Chen, Xin; Geng, Aixin; Chen, Yadong; Lu, Tao; Zhu, Yong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">82</span>),
    <span class="NLM_cas:pages">52180-52186</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors are known to induce multiple epigenetic modifications affecting signaling networks and act synergistically with phosphatidylinositol 3-kinase (PI3K) inhibitors for the treatment of cancer.  Herein we present a novel design approach for cancer drug development by incorporating HDAC inhibitory functionality into a PI3K inhibitor pharmacophore to construct dual-acting inhibitors.  The designed compds. were synthesized and showed inhibitory activities against PI3K and HDAC.  The representative dual PI3K/HDAC inhibitors, compds. 12a-j, showed potent antiproliferative activities against K562 and Hut78 in cellular assays.  This work may lay the foundation for developing novel dual PI3K/HDAC inhibitors as potential anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgJqax7GMFRrVg90H21EOLACvtfcHk0lh-8Y_5NMXwjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslyqs7zL&md5=895434b4f0bf7e7dc7504dffeaba0fe1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1039%2FC7RA08835C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7RA08835C%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGeng%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%25202%252C3-Dihydroimidazo%255B1%252C2-C%255DQuinazoline%2520Derivatives%2520as%2520Novel%2520Phosphatidylinositol%25203-Kinase%2520and%2520Histone%2520Deacetylase%2520Dual%2520Inhibitors%26jtitle%3DRSC%2520Adv.%26date%3D2017%26volume%3D7%26spage%3D52180%26epage%3D52186%26doi%3D10.1039%2FC7RA08835C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Preclinical Evaluation of Fused Pyrimidine-Based Hydroxamates for the Treatment of Hepatocellular Carcinoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1552</span>– <span class="NLM_lpage">1575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1552-1575&author=D.+Chenauthor=C.+K.+Sohauthor=W.+H.+Gohauthor=H.+Wang&title=Design%2C+Synthesis%2C+and+Preclinical+Evaluation+of+Fused+Pyrimidine-Based+Hydroxamates+for+the+Treatment+of+Hepatocellular+Carcinoma&doi=10.1021%2Facs.jmedchem.7b01465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Preclinical Evaluation of Fused Pyrimidine-Based Hydroxamates for the Treatment of Hepatocellular Carcinoma</span></div><div class="casAuthors">Chen, Dizhong; Soh, Chang Kai; Goh, Wei Huang; Wang, Haishan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1552-1575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Class I histone deacetylases (HDACs) are highly expressed and/or upregulated in hepatocellular carcinoma (HCC) and are assocd. with aggressiveness, spread, and increased mortality of HCC.  Activation of phosphatidylinositol 3-kinase-Akt-mammalian target of rapamycin (PI3K/Akt/mTOR) signaling pathway was involved in the development of HCC and acquired resistance to sorafenib.  A series of purine or 5H-pyrrolo[3,2-d]pyrimidine based hydroxamates were designed and developed as multitarget drugs to modulate both HDACs and the PI3K/Akt/mTOR pathway.  Among 39 cell lines screened, the mols. (e.g., I, II, and III) were the most selective against leukemia, lymphoma, and HCC cells; they also demonstrated target modulation in cancer cell lines and in mice bearing MV4-11 and HepG2 tumors.  Compd. II in particular showed significant single agent oral efficacy in hypervascular liver cancer models (e.g., HepG2, HuH-7, and Hep3B) and was well-tolerated.  These encouraging results, along with its favorable target profile and tissue distribution, warrant further development of II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOkc89pglhFLVg90H21EOLACvtfcHk0lgkFay1CdHQ9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsLk%253D&md5=7261ba423c033773f4b1b5cea80af725</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01465%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DSoh%26aufirst%3DC.%2BK.%26aulast%3DGoh%26aufirst%3DW.%2BH.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Preclinical%2520Evaluation%2520of%2520Fused%2520Pyrimidine-Based%2520Hydroxamates%2520for%2520the%2520Treatment%2520of%2520Hepatocellular%2520Carcinoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1552%26epage%3D1575%26doi%3D10.1021%2Facs.jmedchem.7b01465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6992</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00390</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00390" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6992-7014&author=K.+Zhangauthor=F.+Laiauthor=S.+Linauthor=M.+Jiauthor=J.+Zhangauthor=Y.+Zhangauthor=J.+Jinauthor=R.+Fuauthor=D.+Wuauthor=H.+Tianauthor=N.+Xueauthor=L.+Shengauthor=X.+Zouauthor=Y.+Liauthor=X.+Chenauthor=H.+Xu&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+4-Methyl+Quinazoline+Derivatives+as+Anticancer+Agents+Simultaneously+Targeting+Phosphoinositide+3-Kinases+and+Histone+Deacetylases&doi=10.1021%2Facs.jmedchem.9b00390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span></div><div class="casAuthors">Zhang, Kehui; Lai, Fangfang; Lin, Songwen; Ji, Ming; Zhang, Jingbo; Zhang, Yan; Jin, Jing; Fu, Rong; Wu, Deyu; Tian, Hua; Xue, Nina; Sheng, Li; Zou, Xiaowen; Li, Yan; Chen, Xiaoguang; Xu, Heng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6992-7014</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polypharmacol. is a promising paradigm in modern drug discovery.  Herein, we have discovered a series of novel PI3K and HDAC dual inhibitors in which the hydroxamic acid moiety as the zinc binding functional group was introduced to a quinazoline-based PI3K pharmacophore through an appropriate linker.  Systematic structure-activity relationship studies resulted in lead compds. 23 (shown in graphical abstr. and duplicated as I) and 36 (II) that simultaneously inhibited PI3K and HDAC with nanomolar potencies and demonstrated favorable antiproliferative activities.  Compds. 23 and 36 efficiently modulated the expression of p-AKT and Ac-H3, arrested the cell cycle, and induced apoptosis in HCT116 cancer cells.  Following pharmacokinetic studies, 23 was further evaluated in HCT116 and HGC-27 xenograft models to show significant in vivo anticancer efficacies with tumor growth inhibitions of 45.8% (po, 150 mg/kg) and 62.6% (i.p., 30 mg/kg), resp.  Overall, this work shows promise in discovering new anticancer therapeutics by the approach of simultaneously targeting PI3K and HDAC pathways with a single mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6xLiFaGhwk7Vg90H21EOLACvtfcHk0lgkFay1CdHQ9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D&md5=de4c9a630208e49eac58dcad630adcd6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00390%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DXue%26aufirst%3DN.%26aulast%3DSheng%26aufirst%3DL.%26aulast%3DZou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%25204-Methyl%2520Quinazoline%2520Derivatives%2520as%2520Anticancer%2520Agents%2520Simultaneously%2520Targeting%2520Phosphoinositide%25203-Kinases%2520and%2520Histone%2520Deacetylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6992%26epage%3D7014%26doi%3D10.1021%2Facs.jmedchem.9b00390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atoyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellarocca, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattaey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Dual HDAC and PI3K Inhibitor CUDC-907 Downregulates MYC and Suppresses Growth of MYC-Dependent Cancers</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1158%2F1535-7163.MCT-16-0390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27980108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVertro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=285-299&author=K.+Sunauthor=R.+Atoyanauthor=M.+A.+Borekauthor=S.+Dellaroccaauthor=M.+E.+Samsonauthor=A.+W.+Maauthor=G.+X.+Xuauthor=T.+Pattersonauthor=D.+P.+Tuckauthor=J.+L.+Vinerauthor=A.+Fattaeyauthor=J.+Wang&title=Dual+HDAC+and+PI3K+Inhibitor+CUDC-907+Downregulates+MYC+and+Suppresses+Growth+of+MYC-Dependent+Cancers&doi=10.1158%2F1535-7163.MCT-16-0390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Dual HDAC and PI3K Inhibitor CUDC-907 Downregulates MYC and Suppresses Growth of MYC-dependent Cancers</span></div><div class="casAuthors">Sun, Kaiming; Atoyan, Ruzanna; Borek, Mylissa A.; Dellarocca, Steven; Samson, Maria Elena S.; Ma, Anna W.; Xu, Guang-Xin; Patterson, Troy; Tuck, David P.; Viner, Jaye L.; Fattaey, Ali; Wang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-299</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Upregulation of MYC is a common driver event in human cancers, and some tumors depend on MYC to maintain transcriptional programs that promote cell growth and proliferation.  Preclin. studies have suggested that individually targeting upstream regulators of MYC, such as histone deacetylases (HDAC) and phosphoinositide 3-kinases (PI3K), can reduce MYC protein levels and suppress the growth of MYC-driven cancers.  Synergy between HDAC and PI3K inhibition in inducing cancer cell death has also been reported, but the involvement of MYC regulation is unclear.  In this study, we demonstrated that HDAC and PI3K inhibition synergistically downregulates MYC protein levels and induces apoptosis in "double-hit" (DH) diffuse large B-cell lymphoma (DLBCL) cells.  Furthermore, CUDC-907, a small-mol. dual-acting inhibitor of both class I and II HDACs and class I PI3Ks, effectively suppresses the growth and survival of MYC-altered or MYC-dependent cancer cells, such as DH DLBCL and BRD-NUT fusion-pos. NUT midline carcinoma (NMC) cells, and MYC protein downregulation is an early event induced by CUDC-907 treatment.  Consistently, the antitumor activity of CUDC-907 against multiple MYC-driven cancer types was also demonstrated in animal models, including DLBCL and NMC xenograft models, Myc transgenic tumor syngeneic models, and MYC-amplified solid tumor patient-derived xenograft (PDX) models.  Our findings suggest that dual function HDAC and PI3K inhibitor CUDC-907 is an effective agent targeting MYC and thus may be developed as potential therapy for MYC-dependent cancers.  Mol Cancer Ther; 16(2); 285-99. 2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo82AzF-jV397Vg90H21EOLACvtfcHk0lgkFay1CdHQ9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVertro%253D&md5=d4ffd5f39546fa076448dbfb36406183</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0390%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DK.%26aulast%3DAtoyan%26aufirst%3DR.%26aulast%3DBorek%26aufirst%3DM.%2BA.%26aulast%3DDellarocca%26aufirst%3DS.%26aulast%3DSamson%26aufirst%3DM.%2BE.%26aulast%3DMa%26aufirst%3DA.%2BW.%26aulast%3DXu%26aufirst%3DG.%2BX.%26aulast%3DPatterson%26aufirst%3DT.%26aulast%3DTuck%26aufirst%3DD.%2BP.%26aulast%3DViner%26aufirst%3DJ.%2BL.%26aulast%3DFattaey%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDual%2520HDAC%2520and%2520PI3K%2520Inhibitor%2520CUDC-907%2520Downregulates%2520MYC%2520and%2520Suppresses%2520Growth%2520of%2520MYC-Dependent%2520Cancers%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D285%26epage%3D299%26doi%3D10.1158%2F1535-7163.MCT-16-0390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sermer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendel, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span> <span> </span><span class="NLM_article-title">Emerging Epigenetic-Modulating Therapies in Lymphoma</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/s41571-019-0190-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fs41571-019-0190-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30837715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyjt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=494-507&author=D.+Sermerauthor=L.+Pasqualucciauthor=H.+G.+Wendelauthor=A.+Melnickauthor=A.+Younes&title=Emerging+Epigenetic-Modulating+Therapies+in+Lymphoma&doi=10.1038%2Fs41571-019-0190-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging epigenetic-modulating therapies in lymphoma</span></div><div class="casAuthors">Sermer, David; Pasqualucci, Laura; Wendel, Hans-Guido; Melnick, Ari; Younes, Anas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">494-507</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Despite considerable advances in the treatment of lymphoma, the prognosis of patients with relapsed and/or refractory disease continues to be poor; thus, a continued need exists for the development of novel approaches and therapies.  Epigenetic dysregulation might drive and/or promote tumorigenesis in various types of malignancies and is prevalent in both B cell and T cell lymphomas.  Over the past decade, a large no. of epigenetic-modifying agents have been developed and introduced into the clin. management of patients with haematol. malignancies.  In this Review, we provide a concise overview of the most promising epigenetic therapies for the treatment of lymphomas, including inhibitors of histone deacetylases (HDACs), DNA methyltransferases (DNMTs), enhancer of zeste homolog 2 (EZH2), bromodomain and extra-terminal domain proteins (BETs), protein arginine N-methyltransferases (PRMTs) and isocitrate dehydrogenases (IDHs), and highlight the most promising future directions of research in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLSVtVzNWVu7Vg90H21EOLACvtfcHk0lgHL3Znd4ygog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyjt77O&md5=5f7659b71bb5bc1abe3ebc43874b6bf6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fs41571-019-0190-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-019-0190-8%26sid%3Dliteratum%253Aachs%26aulast%3DSermer%26aufirst%3DD.%26aulast%3DPasqualucci%26aufirst%3DL.%26aulast%3DWendel%26aufirst%3DH.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DYounes%26aufirst%3DA.%26atitle%3DEmerging%2520Epigenetic-Modulating%2520Therapies%2520in%2520Lymphoma%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D494%26epage%3D507%26doi%3D10.1038%2Fs41571-019-0190-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdeja, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerecitano, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neelapu, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clancy, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, Y.</span></span> <span> </span><span class="NLM_article-title">Safety, Tolerability, and Preliminary Activity of CUDC-907, a First-in-Class, Oral, Dual Inhibitor of HDAC and PI3K, in Patients with Relapsed or Refractory Lymphoma or Multiple Myeloma: An Open-Label, Dose-Escalation, Phase 1 Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(15)00584-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2FS1470-2045%2815%2900584-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27049457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=622-631&author=A.+Younesauthor=J.+G.+Berdejaauthor=M.+R.+Patelauthor=I.+Flinnauthor=J.+F.+Gerecitanoauthor=S.+S.+Neelapuauthor=K.+R.+Kellyauthor=A.+R.+Copelandauthor=A.+Akinsauthor=M.+S.+Clancyauthor=L.+Gongauthor=J.+Wangauthor=A.+Maauthor=J.+L.+Vinerauthor=Y.+Oki&title=Safety%2C+Tolerability%2C+and+Preliminary+Activity+of+CUDC-907%2C+a+First-in-Class%2C+Oral%2C+Dual+Inhibitor+of+HDAC+and+PI3K%2C+in+Patients+with+Relapsed+or+Refractory+Lymphoma+or+Multiple+Myeloma%3A+An+Open-Label%2C+Dose-Escalation%2C+Phase+1+Trial&doi=10.1016%2FS1470-2045%2815%2900584-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Safety, tolerability, and preliminary activity of CUDC-907, a first-in-class, oral, dual inhibitor of HDAC and PI3K, in patients with relapsed or refractory lymphoma or multiple myeloma: an open-label, dose-escalation, phase 1 trial</span></div><div class="casAuthors">Younes, Anas; Berdeja, Jesus G.; Patel, Manish R.; Flinn, Ian; Gerecitano, John F.; Neelapu, Sattva S.; Kelly, Kevin R.; Copeland, Amanda R.; Akins, Amy; Clancy, Myles S.; Gong, Lucy; Wang, Jing; Ma, Anna; Viner, Jaye L.; Oki, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">622-631</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Treatment options for patients with relapsed or refractory lymphoma and multiple myeloma are limited.  CUDC-907 is an oral, first-in-class, small mol. that is designed to inhibit both histone deacetylase (HDAC) and PI3K enzymes, which are members of common oncogenic pathways in haematol. malignancies.  We aimed to assess overall safety and preliminary activity in this dose-escalation study of CUDC-907 monotherapy in patients with relapsed or refractory lymphoma and multiple myeloma.  This open-label, first-in-man, phase 1 trial recruited adult patients (aged ≥18 years) with lymphoma or multiple myeloma who were refractory to or had relapsed after two or more previous regimens, from four US cancer centers.  CUDC-907 was orally administered in a std. 3 + 3 dose-escalation design at four different dosing schedules, to which participants were sequentially assigned as follows: once daily, intermittently (twice or three times weekly; simultaneous enrolment), and daily for 5 days followed by a 2-day break (5/2), in 21-day cycles.  Dosing started at 30 mg for the once-daily schedule and 60 mg for other schedules, escalating in 30 mg increments.  Patients continued to receive CUDC-907 until disease progression or until other treatment discontinuation criteria were met.  The primary objective was to det. the max. tolerated dose (MTD) and recommended phase 2 dose, assessed in patients who received at least 66% of cycle 1 doses without modification and those who had a dose-limiting toxicity (DLT) in cycle 1 irresp. of dose modification.  We assessed safety in all patients who received at least one dose of study drug.  This ongoing trial is registered at ClinicalTrials.gov, no. NCT01742988.  Between Jan 23, 2013, and July 27, 2015, we enrolled 44 patients, of whom ten were sequentially assigned to CUDC-907 once-daily (MTD 60 mg), 12 to twice-weekly (MTD 150 mg), 15 to three-times-weekly (MTD 150 mg), and seven to the 5/2 dosing schedule (MTD 60 mg). 37 (84%) patients had discontinued study drug as a result of progressive disease or clin. signs of progressive disease at the data cutoff.  Four DLTs occurred in three of 40 DLT-evaluable patients (diarrhoea and hyperglycemia in one patient on 60 mg once daily, hyperglycemia in one patient on 150 mg twice weekly, and diarrhoea in one patient on 150 mg three times weekly); no DLTs were reported in patients on the 5/2 schedule.  Grade 3 or worse adverse events occurred in 19 (43%) of 44 patients, the most common of which were thrombocytopenia (in nine [20%] of 44 patients), neutropenia (three [7%]), and hyperglycemia (three [7%]). 11 (25%) of 44 patients had serious adverse events, three of which were regarded as treatment related (epistaxis and the DLTs of diarrhoea and hyperglycemia).  Adverse events led to dose redns. in six (14%) patients and treatment discontinuation in seven (16%).  Five (14%) of 37 response-evaluable patients achieved an objective response (two complete responses and three partial responses).  All five responses occurred in the subgroup of patients with diffuse large B-cell lymphoma (DLBCL; n=9), and three occurred in those with transformed follicular lymphoma DLBCL (n=5). 21 (57%) of 37 response-evaluable patients had stable disease, including those with DLBCL, Hodgkin's lymphoma, and multiple myeloma.  On the basis of these findings, we selected CUDC-907 60 mg on the 5/2 dosing schedule as the recommended phase 2 dose.  The safety and tolerability profile of CUDC-907 and the promising preliminary evidence of response support continued development of CUDC-907 at the 60 mg 5/2 dosing schedule, alone and in combination with other therapies.  A dose-expansion trial of this dose in patients with refractory and relapsed DLBCL in particular, is ongoing.  Curis, Inc, and the Leukemia and Lymphoma Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAFEt7vSuwM7Vg90H21EOLACvtfcHk0lgHL3Znd4ygog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOhs78%253D&md5=b759c77ac239b6ae0052d30c9bea3221</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2815%2900584-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252815%252900584-7%26sid%3Dliteratum%253Aachs%26aulast%3DYounes%26aufirst%3DA.%26aulast%3DBerdeja%26aufirst%3DJ.%2BG.%26aulast%3DPatel%26aufirst%3DM.%2BR.%26aulast%3DFlinn%26aufirst%3DI.%26aulast%3DGerecitano%26aufirst%3DJ.%2BF.%26aulast%3DNeelapu%26aufirst%3DS.%2BS.%26aulast%3DKelly%26aufirst%3DK.%2BR.%26aulast%3DCopeland%26aufirst%3DA.%2BR.%26aulast%3DAkins%26aufirst%3DA.%26aulast%3DClancy%26aufirst%3DM.%2BS.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DViner%26aufirst%3DJ.%2BL.%26aulast%3DOki%26aufirst%3DY.%26atitle%3DSafety%252C%2520Tolerability%252C%2520and%2520Preliminary%2520Activity%2520of%2520CUDC-907%252C%2520a%2520First-in-Class%252C%2520Oral%252C%2520Dual%2520Inhibitor%2520of%2520HDAC%2520and%2520PI3K%252C%2520in%2520Patients%2520with%2520Relapsed%2520or%2520Refractory%2520Lymphoma%2520or%2520Multiple%2520Myeloma%253A%2520An%2520Open-Label%252C%2520Dose-Escalation%252C%2520Phase%25201%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D622%26epage%3D631%26doi%3D10.1016%2FS1470-2045%2815%2900584-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, R. R.</span></span> <span> </span><span class="NLM_article-title">Safety and Tolerability of Histone Deacetylase (HDAC) Inhibitors in Oncology</span>. <i>Drug Saf.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1007/s40264-018-0773-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1007%2Fs40264-018-0773-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30649740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvFOqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2019&pages=235-245&author=R.+R.+Shah&title=Safety+and+Tolerability+of+Histone+Deacetylase+%28HDAC%29+Inhibitors+in+Oncology&doi=10.1007%2Fs40264-018-0773-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Safety and Tolerability of Histone Deacetylase (HDAC) Inhibitors in Oncology</span></div><div class="casAuthors">Shah, Rashmi R.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Safety</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-245</span>CODEN:
                <span class="NLM_cas:coden">DRSAEA</span>;
        ISSN:<span class="NLM_cas:issn">0114-5916</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) are expressed at increased levels in cells of various malignancies, and the use of HDAC inhibitors has improved outcomes in patients with haematol. malignancies (T-cell lymphomas and multiple myeloma).  However, they are not as effective in solid tumors.  Five agents are currently approved under various jurisdictions, namely belinostat, chidamide, panobinostat, romidepsin and vorinostat.  These agents are assocd. with a range of class-related and agent-specific serious and/or severe adverse effects, notably myelosuppression, diarrhoea and various cardiac effects.  Among the cardiac effects are ST-T segment abnormalities and QTc interval prolongation of the ECG, isolated cases of atrial fibrillation and, in rare instances, ventricular tachyarrhythmias.  In order to improve the safety profile of this class of drugs as well as their efficacy in indications already approved and to further widen their indications, a large no. of newer HDAC inhibitors with varying degrees of HDAC isoform selectivity have been synthesized and are currently under clin. development.  Preliminary evidence from early studies suggests that they may be effective in non-haematol. cancers as well when used in combination with other therapeutic modalities, but that they too appear to be assocd. with the above class-related adverse effects.  As the database accumulates, the safety, efficacy and risk/benefit of the newer agents and their indications will become clearer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_FzvurFGlNLVg90H21EOLACvtfcHk0lg0ocoQAkAM5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvFOqtrY%253D&md5=db23e29ac410c716393aae079abf18cd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs40264-018-0773-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40264-018-0773-9%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DR.%2BR.%26atitle%3DSafety%2520and%2520Tolerability%2520of%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Inhibitors%2520in%2520Oncology%26jtitle%3DDrug%2520Saf.%26date%3D2019%26volume%3D42%26spage%3D235%26epage%3D245%26doi%3D10.1007%2Fs40264-018-0773-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagpacan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papalia, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span> <span> </span><span class="NLM_article-title">Structural, Biochemical, and Biophysical Characterization of Idelalisib Binding to Phosphoinositide 3-Kinase Delta</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">8439</span>– <span class="NLM_lpage">8446</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.634683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1074%2Fjbc.M114.634683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=25631052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVSqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=8439-8446&author=J.+R.+Somozaauthor=D.+Koditekauthor=A.+G.+Villasenorauthor=N.+Novikovauthor=M.+H.+Wongauthor=A.+Liclicanauthor=W.+Xingauthor=L.+Lagpacanauthor=R.+Wangauthor=B.+E.+Schultzauthor=G.+A.+Papaliaauthor=D.+Samuelauthor=L.+Ladauthor=M.+E.+McGrath&title=Structural%2C+Biochemical%2C+and+Biophysical+Characterization+of+Idelalisib+Binding+to+Phosphoinositide+3-Kinase+Delta&doi=10.1074%2Fjbc.M114.634683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural, Biochemical, and Biophysical Characterization of Idelalisib Binding to Phosphoinositide 3-Kinase δ</span></div><div class="casAuthors">Somoza, John R.; Koditek, David; Villasenor, Armando G.; Novikov, Nikolai; Wong, Melanie H.; Liclican, Albert; Xing, Weimei; Lagpacan, Leanna; Wang, Ruth; Schultz, Brian E.; Papalia, Giuseppe A.; Samuel, Dharmaraj; Lad, Latesh; McGrath, Mary E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8439-8446</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Idelalisib (also known as GS-1101, CAL-101, IC489666, and Zydelig) is a PI3Kδ inhibitor that has recently been approved for the treatment of several hematol. malignancies.  Given its use in human diseases, we needed a clear picture of how idelalisib binds to and inhibits PI3Kδ.  Our data show that idelalisib is a potent and selective inhibitor of the kinase activity of PI3Kδ.  A kinetic characterization clearly demonstrated ATP-competitive inhibition, and several addnl. biochem. and biophys. assays showed that the compd. binds reversibly and noncovalently to the kinase.  A crystal structure of idelalisib bound to the p110δ subunit of PI3Kδ furthers our understanding of the binding interactions that confer the potency and selectivity of idelalisib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnXZo8_qK79rVg90H21EOLACvtfcHk0lg0ocoQAkAM5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVSqtrs%253D&md5=2969b7407f4edc367249215bbbf22a0e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.634683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.634683%26sid%3Dliteratum%253Aachs%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DNovikov%26aufirst%3DN.%26aulast%3DWong%26aufirst%3DM.%2BH.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DXing%26aufirst%3DW.%26aulast%3DLagpacan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DPapalia%26aufirst%3DG.%2BA.%26aulast%3DSamuel%26aufirst%3DD.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26atitle%3DStructural%252C%2520Biochemical%252C%2520and%2520Biophysical%2520Characterization%2520of%2520Idelalisib%2520Binding%2520to%2520Phosphoinositide%25203-Kinase%2520Delta%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D8439%26epage%3D8446%26doi%3D10.1074%2Fjbc.M114.634683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lauffer, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mintzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zilberleyb, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flicke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritscher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorowicz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunzner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrusan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koth, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupardus, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, P.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase (HDAC) Inhibitor Kinetic Rate Constants Correlate with Cellular Histone Acetylation but Not Transcription and Cell Viability</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">26926</span>– <span class="NLM_lpage">26943</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.490706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1074%2Fjbc.M113.490706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=23897821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClt7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=26926-26943&author=B.+E.+Laufferauthor=R.+Mintzerauthor=R.+Fongauthor=S.+Mukundauthor=C.+Tamauthor=I.+Zilberleybauthor=B.+Flickeauthor=A.+Ritscherauthor=G.+Fedorowiczauthor=R.+Valleroauthor=D.+F.+Ortwineauthor=J.+Gunznerauthor=Z.+Modrusanauthor=L.+Neumannauthor=C.+M.+Kothauthor=P.+J.+Lupardusauthor=J.+S.+Kaminkerauthor=C.+E.+Heiseauthor=P.+Steiner&title=Histone+Deacetylase+%28HDAC%29+Inhibitor+Kinetic+Rate+Constants+Correlate+with+Cellular+Histone+Acetylation+but+Not+Transcription+and+Cell+Viability&doi=10.1074%2Fjbc.M113.490706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase (HDAC) Inhibitor Kinetic Rate Constants Correlate with Cellular Histone Acetylation but Not Transcription and Cell Viability</span></div><div class="casAuthors">Lauffer, Benjamin E. L.; Mintzer, Robert; Fong, Rina; Mukund, Susmith; Tam, Christine; Zilberleyb, Inna; Flicke, Birgit; Ritscher, Allegra; Fedorowicz, Grazyna; Vallero, Roxanne; Ortwine, Daniel F.; Gunzner, Janet; Modrusan, Zora; Neumann, Lars; Koth, Christopher M.; Lupardus, Patrick J.; Kaminker, Joshua S.; Heise, Christopher E.; Steiner, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">26926-26943</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) are crit. in the control of gene expression, and dysregulation of their activity has been implicated in a broad range of diseases, including cancer, cardiovascular, and neurol. diseases.  HDAC inhibitors (HDACi) employing different zinc chelating functionalities such as hydroxamic acids and benzamides have shown promising results in cancer therapy.  Although it has also been suggested that HDACi with increased isoenzyme selectivity and potency may broaden their clin. utility and minimize side effects, the translation of this idea to the clinic remains to be investigated.  Moreover, a detailed understanding of how HDACi with different pharmacol. properties affect biol. functions in vitro and in vivo is still missing.  Here, we show that a panel of benzamide-contg. HDACi are slow tight-binding inhibitors with long residence times unlike the hydroxamate-contg. HDACi vorinostat and trichostatin-A.  Characterization of changes in H2BK5 and H4K14 acetylation following HDACi treatment in the neuroblastoma cell line SH-SY5Y revealed that the timing and magnitude of histone acetylation mirrored both the assocn. and dissocn. kinetic rates of the inhibitors.  In contrast, cell viability and microarray gene expression anal. indicated that cell death induction and changes in transcriptional regulation do not correlate with the dissocn. kinetic rates of the HDACi.  Therefore, our study suggests that detg. how the selective and kinetic inhibition properties of HDACi affect cell function will help to evaluate their therapeutic utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2goaOx8CDGbVg90H21EOLACvtfcHk0lg5odDB4wuZig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClt7jM&md5=af1074c79e8abd932313f4ae13333a98</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.490706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.490706%26sid%3Dliteratum%253Aachs%26aulast%3DLauffer%26aufirst%3DB.%2BE.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DMukund%26aufirst%3DS.%26aulast%3DTam%26aufirst%3DC.%26aulast%3DZilberleyb%26aufirst%3DI.%26aulast%3DFlicke%26aufirst%3DB.%26aulast%3DRitscher%26aufirst%3DA.%26aulast%3DFedorowicz%26aufirst%3DG.%26aulast%3DVallero%26aufirst%3DR.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DGunzner%26aufirst%3DJ.%26aulast%3DModrusan%26aufirst%3DZ.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DKoth%26aufirst%3DC.%2BM.%26aulast%3DLupardus%26aufirst%3DP.%2BJ.%26aulast%3DKaminker%26aufirst%3DJ.%2BS.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DSteiner%26aufirst%3DP.%26atitle%3DHistone%2520Deacetylase%2520%2528HDAC%2529%2520Inhibitor%2520Kinetic%2520Rate%2520Constants%2520Correlate%2520with%2520Cellular%2520Histone%2520Acetylation%2520but%2520Not%2520Transcription%2520and%2520Cell%2520Viability%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D26926%26epage%3D26943%26doi%3D10.1074%2Fjbc.M113.490706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span> <span> </span><span class="NLM_article-title">Zinc Binding Groups for Histone Deacetylase Inhibitors</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1080/14756366.2017.1417274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1080%2F14756366.2017.1417274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29616828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVGhtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=714-721&author=L.+Zhangauthor=J.+Zhangauthor=Q.+Jiangauthor=L.+Zhangauthor=W.+Song&title=Zinc+Binding+Groups+for+Histone+Deacetylase+Inhibitors&doi=10.1080%2F14756366.2017.1417274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Zinc binding groups for histone deacetylase inhibitors</span></div><div class="casAuthors">Zhang, Lei; Zhang, Jian; Jiang, Qixiao; Zhang, Li; Song, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">714-721</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Zinc binding groups (ZBGs) play a crucial role in targeting histone deacetylase inhibitors (HDACIs) to the active site of histone deacetylases (HDACs), thus detg. the potency of HDACIs.  Due to the high affinity to the zinc ion, hydroxamic acid is the most commonly used ZBG in the structure of HDACs.  An alternative ZBG is benzamide group, which features excellent inhibitory selectivity for class I HDACs.  Various ZBGs have been designed and tested to improve the activity and selectivity of HDACIs, and to overcome the pharmacokinetic limitations of current HDACIs.  Herein, different kinds of ZBGs are reviewed and their features have been discussed for further design of HDACIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXFywPdICZVrVg90H21EOLACvtfcHk0lg5odDB4wuZig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVGhtrc%253D&md5=771d801a575618430f1cdcd03044dbdc</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1080%2F14756366.2017.1417274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14756366.2017.1417274%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DW.%26atitle%3DZinc%2520Binding%2520Groups%2520for%2520Histone%2520Deacetylase%2520Inhibitors%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2018%26volume%3D33%26spage%3D714%26epage%3D721%26doi%3D10.1080%2F14756366.2017.1417274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6992</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00390</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00390" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6992-7014&author=K.+Zhangauthor=F.+Laiauthor=S.+Linauthor=M.+Jiauthor=J.+Zhangauthor=Y.+Zhangauthor=J.+Jinauthor=R.+Fuauthor=D.+Wuauthor=H.+Tianauthor=N.+Xueauthor=L.+Shengauthor=X.+Zouauthor=Y.+Liauthor=X.+Chenauthor=H.+Xu&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+4-Methyl+Quinazoline+Derivatives+as+Anticancer+Agents+Simultaneously+Targeting+Phosphoinositide+3-Kinases+and+Histone+Deacetylases&doi=10.1021%2Facs.jmedchem.9b00390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span></div><div class="casAuthors">Zhang, Kehui; Lai, Fangfang; Lin, Songwen; Ji, Ming; Zhang, Jingbo; Zhang, Yan; Jin, Jing; Fu, Rong; Wu, Deyu; Tian, Hua; Xue, Nina; Sheng, Li; Zou, Xiaowen; Li, Yan; Chen, Xiaoguang; Xu, Heng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6992-7014</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polypharmacol. is a promising paradigm in modern drug discovery.  Herein, we have discovered a series of novel PI3K and HDAC dual inhibitors in which the hydroxamic acid moiety as the zinc binding functional group was introduced to a quinazoline-based PI3K pharmacophore through an appropriate linker.  Systematic structure-activity relationship studies resulted in lead compds. 23 (shown in graphical abstr. and duplicated as I) and 36 (II) that simultaneously inhibited PI3K and HDAC with nanomolar potencies and demonstrated favorable antiproliferative activities.  Compds. 23 and 36 efficiently modulated the expression of p-AKT and Ac-H3, arrested the cell cycle, and induced apoptosis in HCT116 cancer cells.  Following pharmacokinetic studies, 23 was further evaluated in HCT116 and HGC-27 xenograft models to show significant in vivo anticancer efficacies with tumor growth inhibitions of 45.8% (po, 150 mg/kg) and 62.6% (i.p., 30 mg/kg), resp.  Overall, this work shows promise in discovering new anticancer therapeutics by the approach of simultaneously targeting PI3K and HDAC pathways with a single mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6xLiFaGhwk7Vg90H21EOLACvtfcHk0ljgGEf5HacruQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D&md5=de4c9a630208e49eac58dcad630adcd6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00390%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DXue%26aufirst%3DN.%26aulast%3DSheng%26aufirst%3DL.%26aulast%3DZou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%25204-Methyl%2520Quinazoline%2520Derivatives%2520as%2520Anticancer%2520Agents%2520Simultaneously%2520Targeting%2520Phosphoinositide%25203-Kinases%2520and%2520Histone%2520Deacetylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6992%26epage%3D7014%26doi%3D10.1021%2Facs.jmedchem.9b00390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustafa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong-Thi, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. X. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanujulu, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chng, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, J. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B. W.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Ligand Efficient Dual Inhibitors of Janus Kinase (JAK) and Histone Deacetylase (HDAC) Based on Ruxolitinib and Vorinostat</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8336</span>– <span class="NLM_lpage">8357</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00678</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00678" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8336-8357&author=L.+Yaoauthor=N.+Mustafaauthor=E.+C.+Tanauthor=A.+Poulsenauthor=P.+Singhauthor=M.+D.+Duong-Thiauthor=J.+X.+T.+Leeauthor=P.+M.+Ramanujuluauthor=W.+J.+Chngauthor=J.+J.+Y.+Yenauthor=S.+Ohlsonauthor=B.+W.+Dymock&title=Design+and+Synthesis+of+Ligand+Efficient+Dual+Inhibitors+of+Janus+Kinase+%28JAK%29+and+Histone+Deacetylase+%28HDAC%29+Based+on+Ruxolitinib+and+Vorinostat&doi=10.1021%2Facs.jmedchem.7b00678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Ligand Efficient Dual Inhibitors of Janus Kinase (JAK) and Histone Deacetylase (HDAC) Based on Ruxolitinib and Vorinostat</span></div><div class="casAuthors">Yao, Lianbin; Mustafa, Nurulhuda; Tan, Eng Chong; Poulsen, Anders; Singh, Prachi; Duong-Thi, Minh-Dao; Lee, Jeannie X. T.; Ramanujulu, Pondy Murugappan; Chng, Wee Joo; Yen, Jeffrey J. Y.; Ohlson, Sten; Dymock, Brian W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8336-8357</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple oncogenic pathways is a desirable goal in cancer therapy.  To achieve such an outcome with a single mol. would simplify treatment regimes.  Herein the core features of ruxolitinib, a marketed JAK1/2 inhibitor, have been merged with the HDAC inhibitor vorinostat, leading to new mols. that are bispecific targeted JAK/HDAC inhibitors.  A preferred pyrazole substituted pyrrolopyrimidine, I (R = H), inhibits JAK1 and HDACs 1, 2, 3, 6, and 10 with IC50 values of less than 20 nM, is <100 nM potent against JAK2 and HDAC11, and is selective for the JAK family against a panel of 97 kinases.  Broad cellular antiproliferative potency of I (R = H) is supported by demonstration of JAK-STAT and HDAC pathway blockade in hematol. cell lines.  Me analog I (R = Me) has an even more selective profile.  This study provides new leads for assessment of JAK and HDAC pathway dual inhibition achieved with a single mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWx3JXIW8YbVg90H21EOLACvtfcHk0ljgGEf5HacruQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtLzJ&md5=5ac471ee0cc413b08b926803790de7f9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00678%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DL.%26aulast%3DMustafa%26aufirst%3DN.%26aulast%3DTan%26aufirst%3DE.%2BC.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DDuong-Thi%26aufirst%3DM.%2BD.%26aulast%3DLee%26aufirst%3DJ.%2BX.%2BT.%26aulast%3DRamanujulu%26aufirst%3DP.%2BM.%26aulast%3DChng%26aufirst%3DW.%2BJ.%26aulast%3DYen%26aufirst%3DJ.%2BJ.%2BY.%26aulast%3DOhlson%26aufirst%3DS.%26aulast%3DDymock%26aufirst%3DB.%2BW.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Ligand%2520Efficient%2520Dual%2520Inhibitors%2520of%2520Janus%2520Kinase%2520%2528JAK%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Based%2520on%2520Ruxolitinib%2520and%2520Vorinostat%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8336%26epage%3D8357%26doi%3D10.1021%2Facs.jmedchem.7b00678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Janus Kinase 2 (JAK2) and Histone Deacetylase (HDAC) Dual Inhibitors as a Novel Strategy for the Combinational Treatment of Leukemia and Invasive Fungal Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6056</span>– <span class="NLM_lpage">6074</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Snu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6056-6074&author=Y.+Huangauthor=G.+Dongauthor=H.+Liauthor=N.+Liuauthor=W.+Zhangauthor=C.+Sheng&title=Discovery+of+Janus+Kinase+2+%28JAK2%29+and+Histone+Deacetylase+%28HDAC%29+Dual+Inhibitors+as+a+Novel+Strategy+for+the+Combinational+Treatment+of+Leukemia+and+Invasive+Fungal+Infections&doi=10.1021%2Facs.jmedchem.8b00393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Janus Kinase 2 (JAK2) and Histone Deacetylase (HDAC) Dual Inhibitors as a Novel Strategy for the Combinational Treatment of Leukemia and Invasive Fungal Infections</span></div><div class="casAuthors">Huang, Yahui; Dong, Guoqiang; Li, Huanqiu; Liu, Na; Zhang, Wannian; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6056-6074</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Clin., leukemia patients often suffer from the limited efficacy of chemotherapy and high risks of infection by invasive fungal pathogens.  Herein, a novel therapeutic strategy was developed in which a small mol. can simultaneously treat leukemia and invasive fungal infections (IFIs).  Novel Janus kinase 2 (JAK2) and histone deacetylase (HDAC) dual inhibitors were identified to possess potent anti-proliferative activity toward hematol. cell lines and excellent synergistic effects with fluconazole to treat resistant Candida albicans infections.  In particular, compd. I, a highly active and selective JAK2/HDAC6 dual inhibitor, showed excellent in vivo antitumor efficacy in several acute myeloid leukemia (AML) models and synergized with fluconazole for the treatment of resistant C. albicans infections.  This study highlights the therapeutic potential of JAK2/HDAC dual inhibitors in treating AML and IFIs and provides an efficient strategy for multitargeting drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3_ii7EQG1qLVg90H21EOLACvtfcHk0ljfuGUW5i2Zrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Snu7rN&md5=ba7cf74761120f0526ab520196e3dded</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00393%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520Janus%2520Kinase%25202%2520%2528JAK2%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Dual%2520Inhibitors%2520as%2520a%2520Novel%2520Strategy%2520for%2520the%2520Combinational%2520Treatment%2520of%2520Leukemia%2520and%2520Invasive%2520Fungal%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6056%26epage%3D6074%26doi%3D10.1021%2Facs.jmedchem.8b00393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krieger, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ressing, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkhardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of the First-in-Class Dual Histone Deacetylase-Proteasome Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10299</span>– <span class="NLM_lpage">10309</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01487</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01487" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10299-10309&author=S.+Bhatiaauthor=V.+Kriegerauthor=M.+Grollauthor=J.+D.+Oskoauthor=N.+Ressingauthor=H.+Ahlertauthor=A.+Borkhardtauthor=T.+Kurzauthor=D.+W.+Christiansonauthor=J.+Hauerauthor=F.+K.+Hansen&title=Discovery+of+the+First-in-Class+Dual+Histone+Deacetylase-Proteasome+Inhibitor&doi=10.1021%2Facs.jmedchem.8b01487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the First-in-Class Dual Histone Deacetylase-Proteasome Inhibitor</span></div><div class="casAuthors">Bhatia, Sanil; Krieger, Viktoria; Groll, Michael; Osko, Jeremy D.; Ressing, Nina; Ahlert, Heinz; Borkhardt, Arndt; Kurz, Thomas; Christianson, David W.; Hauer, Julia; Hansen, Finn K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10299-10309</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual- or multitarget drugs have emerged as a promising alternative to combination therapies.  Proteasome inhibitors (PIs) possess synergistic activity with histone deacetylase (HDAC) inhibitors due to the simultaneous blockage of the ubiquitin degrdn. and aggresome pathways.  Here, the authors present the design, synthesis, binding modes, and anticancer properties of RTS-V5 as the first-in-class dual HDAC-proteasome ligand.  The inhibition of both targets was confirmed by biochem. and cellular assays as well as x-ray crystal structures of the 20S proteasome and HDAC6 complexed with RTS-V5.  Cytotoxicity assays with leukemia and multiple myeloma cell lines as well as therapy refractory primary patient-derived leukemia cells demonstrated that RTS-V5 possesses potent and selective anticancer activity.  The results will thus guide the structure-based optimization of dual HDAC-proteasome inhibitors for the treatment of hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwJMqcDlzXA7Vg90H21EOLACvtfcHk0ljfuGUW5i2Zrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K&md5=7d47e664bd77ac1bc5ebd39ea6eed02a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01487%26sid%3Dliteratum%253Aachs%26aulast%3DBhatia%26aufirst%3DS.%26aulast%3DKrieger%26aufirst%3DV.%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DRessing%26aufirst%3DN.%26aulast%3DAhlert%26aufirst%3DH.%26aulast%3DBorkhardt%26aufirst%3DA.%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DHauer%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DF.%2BK.%26atitle%3DDiscovery%2520of%2520the%2520First-in-Class%2520Dual%2520Histone%2520Deacetylase-Proteasome%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10299%26epage%3D10309%26doi%3D10.1021%2Facs.jmedchem.8b01487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Simultaneously Inhibiting P53-Murine Double Minute 2 (MDM2) Interaction and Histone Deacetylases (HDACs): Discovery of Novel Multitargeting Antitumor Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7245</span>– <span class="NLM_lpage">7260</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00664</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00664" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlykur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7245-7260&author=S.+Heauthor=G.+Dongauthor=S.+Wuauthor=K.+Fangauthor=Z.+Miaoauthor=W.+Wangauthor=C.+Sheng&title=Small+Molecules+Simultaneously+Inhibiting+P53-Murine+Double+Minute+2+%28MDM2%29+Interaction+and+Histone+Deacetylases+%28HDACs%29%3A+Discovery+of+Novel+Multitargeting+Antitumor+Agents&doi=10.1021%2Facs.jmedchem.8b00664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Simultaneously Inhibiting p53-Murine Double Minute 2 (MDM2) Interaction and Histone Deacetylases (HDACs): Discovery of Novel Multitargeting Antitumor Agents</span></div><div class="casAuthors">He, Shipeng; Dong, Guoqiang; Wu, Shanchao; Fang, Kun; Miao, Zhenyuan; Wang, Wei; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7245-7260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P53-Murine double minute 2 (MDM2) interaction and histone deacetylases (HDACs) are important targets in antitumor drug development.  Inspired by the synergistic effects between MDM2 and HDACs, the first MDM2/HDACs dual inhibitors were identified, which showed excellent activities against both targets.  In particular, compd. 14d was proven to be a potent and orally active MDM2/HDAC dual inhibitor, whose antitumor mechanisms were validated in cancer cells.  Compd. 14d showed excellent in vivo antitumor potency in the A549 xenograft model, providing a promising lead compd. for the development of novel antitumor agents.  Also, this proof-of-concept study offers a novel and efficient strategy for multitargeting antitumor drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx94Uvm3pP8bVg90H21EOLACvtfcHk0ljfuGUW5i2Zrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlykur3L&md5=68871e41ecb786e135b01742e59eec85</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00664%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DK.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DSmall%2520Molecules%2520Simultaneously%2520Inhibiting%2520P53-Murine%2520Double%2520Minute%25202%2520%2528MDM2%2529%2520Interaction%2520and%2520Histone%2520Deacetylases%2520%2528HDACs%2529%253A%2520Discovery%2520of%2520Novel%2520Multitargeting%2520Antitumor%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7245%26epage%3D7260%26doi%3D10.1021%2Facs.jmedchem.8b00664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Pazopanib-Based HDAC and VEGFR Dual Inhibitors Targeting Cancer Epigenetics and Angiogenesis Simultaneously</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5304</span>– <span class="NLM_lpage">5322</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvVGrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5304-5322&author=J.+Zangauthor=X.+Liangauthor=Y.+Huangauthor=Y.+Jiaauthor=X.+Liauthor=W.+Xuauthor=C.+J.+Chouauthor=Y.+Zhang&title=Discovery+of+Novel+Pazopanib-Based+HDAC+and+VEGFR+Dual+Inhibitors+Targeting+Cancer+Epigenetics+and+Angiogenesis+Simultaneously&doi=10.1021%2Facs.jmedchem.8b00384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Pazopanib-Based HDAC and VEGFR Dual Inhibitors Targeting Cancer Epigenetics and Angiogenesis Simultaneously</span></div><div class="casAuthors">Zang, Jie; Liang, Xuewu; Huang, Yongxue; Jia, Yuping; Li, Xiaoyang; Xu, Wenfang; Chou, C. James; Zhang, Yingjie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5304-5322</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein a novel series of pazopanib hybrids as polypharmacol. antitumor agents were developed based on the crosstalk between histone deacetylases (HDACs) and vascular endothelial growth factor (VEGF) pathway.  Among them, one ortho-aminoanilide 6d and one hydroxamic acid 13f exhibited considerable total HDACs and VEGFR-2 inhibitory activities.  The HDAC inhibitory activities endowed 6d and 13f with potent antiproliferative activities, which was not obsd. in the approved VEGFR inhibitor pazopanib.  Compds. 6d and 13f possessed comparable HDAC isoform selectivity profiles to the clin. class I HDAC inhibitor MS-275 and the approved pan-HDAC inhibitor SAHA, resp. 6d and 13f also exhibited uncompromised multiple tyrosine kinases inhibitory activities relative to pazopanib.  The intracellular dual inhibition to HDAC and VEGFR of 6d and 13f was validated by Western blot anal.  In both HUVECs tube formation assay and rat thoracic aorta rings assay, 6d and 13f showed comparable antiangiogenic potencies to pazopanib.  What's more, 6d possessed desirable pharmacokinetic profiles with the oral bioavailability of 72% in SD rats and considerable in vivo antitumor efficacy in a human colorectal adenocarcinoma (HT-29) xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpmz6rwKiKSbVg90H21EOLACvtfcHk0ljFkxy2CXvbTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvVGrsL8%253D&md5=437da0a6cca1ec211219aef87a6ddbc5</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00384%26sid%3Dliteratum%253Aachs%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DChou%26aufirst%3DC.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Novel%2520Pazopanib-Based%2520HDAC%2520and%2520VEGFR%2520Dual%2520Inhibitors%2520Targeting%2520Cancer%2520Epigenetics%2520and%2520Angiogenesis%2520Simultaneously%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5304%26epage%3D5322%26doi%3D10.1021%2Facs.jmedchem.8b00384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of N1-(4-((7-Cyclopentyl-6-(Dimethylcarbamoyl)-7H-Pyrrolo[2,3-d]Pyrimidin-2-yl)Amino)Phenyl)- N8-Hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-Dependent Kinase 4/9 (CDK4/9) and Histone Deacetlyase1 (HDAC1) against Malignant Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3166</span>– <span class="NLM_lpage">3192</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3166-3192&author=Y.+Liauthor=X.+Luoauthor=Q.+Guoauthor=Y.+Nieauthor=T.+Wangauthor=C.+Zhangauthor=Z.+Huangauthor=X.+Wangauthor=Y.+Liuauthor=Y.+Chenauthor=J.+Zhengauthor=S.+Yangauthor=Y.+Fanauthor=R.+Xiang&title=Discovery+of+N1-%284-%28%287-Cyclopentyl-6-%28Dimethylcarbamoyl%29-7H-Pyrrolo%5B2%2C3-d%5DPyrimidin-2-yl%29Amino%29Phenyl%29-+N8-Hydroxyoctanediamide+as+a+Novel+Inhibitor+Targeting+Cyclin-Dependent+Kinase+4%2F9+%28CDK4%2F9%29+and+Histone+Deacetlyase1+%28HDAC1%29+against+Malignant+Cancer&doi=10.1021%2Facs.jmedchem.8b00209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-dependent Kinase 4/9 (CDK4/9) and Histone Deacetylase 1 (HDAC1) against Malignant Cancer</span></div><div class="casAuthors">Li, Yongtao; Luo, Xiaohe; Guo, Qingxiang; Nie, Yongwei; Wang, Tianqi; Zhang, Chao; Huang, Zhi; Wang, Xin; Liu, Yanhua; Chen, Yanan; Zheng, Jianyu; Yang, Shengyong; Fan, Yan; Xiang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3166-3192</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of highly potent, selective inhibitors targeting both CDK4/9 and HDAC1 have been designed and synthesized.  N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide (I) was discovered.  The lead compd. I with excellent CDK4/9 and HDAC1 inhibitory activity of IC50 = 8.8, 12, and 2.2 nM, resp., can effectively induce apoptosis of cancer cell lines.  The kinase profiling of compd. I showed excellent selectivity and specificity.  Compd. I induces G2/M arrest in high concn. and G0/G1 arrest in low concn. to prevent the proliferation and differentiation of cancer cells.  Mice bared-breast cancer treated with I showed significant antitumor efficacy.  The insight into mechanisms of I indicated that it could induce cancer cell death via cell apoptosis based on CDK4/9 and HDAC1 repression and phosphorylation of p53.  The data demonstrated the compd. I could be a promising drug candidate for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4B7BpJ-foDbVg90H21EOLACvtfcHk0ljFkxy2CXvbTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D&md5=b1ca389e67a3ccd3fe5c005cb547e369</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00209%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DNie%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520N1-%25284-%2528%25287-Cyclopentyl-6-%2528Dimethylcarbamoyl%2529-7H-Pyrrolo%255B2%252C3-d%255DPyrimidin-2-yl%2529Amino%2529Phenyl%2529-%2520N8-Hydroxyoctanediamide%2520as%2520a%2520Novel%2520Inhibitor%2520Targeting%2520Cyclin-Dependent%2520Kinase%25204%252F9%2520%2528CDK4%252F9%2529%2520and%2520Histone%2520Deacetlyase1%2520%2528HDAC1%2529%2520against%2520Malignant%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3166%26epage%3D3192%26doi%3D10.1021%2Facs.jmedchem.8b00209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1592</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1577-1592&author=Y.+Chenauthor=X.+Yuanauthor=W.+Zhangauthor=M.+Tangauthor=L.+Zhengauthor=F.+Wangauthor=W.+Yanauthor=S.+Yangauthor=Y.+Weiauthor=J.+Heauthor=L.+Chen&title=Discovery+of+Novel+Dual+Histone+Deacetylase+and+Mammalian+Target+of+Rapamycin+Target+Inhibitors+as+a+Promising+Strategy+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.8b01825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span></div><div class="casAuthors">Chen, Yong; Yuan, Xue; Zhang, Wanhua; Tang, Minghai; Zheng, Li; Wang, Fang; Yan, Wei; Yang, Shengyong; Wei, Yuquan; He, Jun; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1577-1592</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present study, a series of novel dual-target histone deacetylase (HDAC) and mammalian target of rapamycin (mTOR) inhibitors were designed and synthesized using pyrimidine-pyrazolyl pharmacophore to append HDAC recognition cap and hydroxamic acid as a zinc-binding motif.  Among them, I was the optimal lead compd. with potent inhibition activities against mTOR and HDAC1 with half-maximal inhibitory concn. of 1.2 and 0.19 nM, resp.  Western blot confirmed that I could upregulate acetylation of H3 and α-tubulin and downregulate mTOR-related downstream mediators.  I could also stimulate cell cycle arrest in G0/G1 phase and induce tumor cell apoptosis.  I showed comparable antitumor activity with the combination medication in MM1S xenograft model with a tumor growth inhibitory rate of 72.5%, without causing significant loss of body wt. and toxicity.  All of the results indicated that I could be a promising dual target inhibitor for treating hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ3AdehNv1ObVg90H21EOLACvtfcHk0lhwXdY8zgg0NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D&md5=7a2ff5020aedb0e3c302e65d993301e6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01825%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520Novel%2520Dual%2520Histone%2520Deacetylase%2520and%2520Mammalian%2520Target%2520of%2520Rapamycin%2520Target%2520Inhibitors%2520as%2520a%2520Promising%2520Strategy%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1577%26epage%3D1592%26doi%3D10.1021%2Facs.jmedchem.8b01825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Janus Kinase (JAK) and Histone Deacetylase (HDAC) Dual Inhibitors for the Treatment of Hematological Malignancies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3898</span>– <span class="NLM_lpage">3923</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVSls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3898-3923&author=X.+Liangauthor=J.+Zangauthor=X.+Liauthor=S.+Tangauthor=M.+Huangauthor=M.+Gengauthor=C.+J.+Chouauthor=C.+Liauthor=Y.+Caoauthor=W.+Xuauthor=H.+Liuauthor=Y.+Zhang&title=Discovery+of+Novel+Janus+Kinase+%28JAK%29+and+Histone+Deacetylase+%28HDAC%29+Dual+Inhibitors+for+the+Treatment+of+Hematological+Malignancies&doi=10.1021%2Facs.jmedchem.8b01597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Janus Kinase (JAK) and Histone Deacetylase (HDAC) Dual Inhibitors for the Treatment of Hematological Malignancies</span></div><div class="casAuthors">Liang, Xuewu; Zang, Jie; Li, Xiaoyang; Tang, Shuai; Huang, Min; Geng, Meiyu; Chou, C. James; Li, Chunpu; Cao, Yichun; Xu, Wenfang; Liu, Hong; Zhang, Yingjie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3898-3923</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concurrent inhibition of Janus kinase (JAK) and histone deacetylase (HDAC) could potentially improve the efficacy of the HDAC inhibitors in the treatment of cancers and resolve the problem of HDAC inhibitor resistance in some tumors.  Here, a novel series of pyrimidin-2-amino-pyrazol hydroxamate derivs. as JAK and HDAC dual inhibitors was designed, synthesized, and evaluated, among which I possessed potent and balanced activities against both JAK2 and HDAC6 with half-maximal inhibitory concn. at the nanomolar level. I exhibited improved antiproliferative and proapoptotic activities over SAHA and ruxolitinib in several hematol. cell lines.  Remarkably, I exhibited more potent antiproliferation effect than the combination of SAHA and ruxolitinib in HEL cells bearing JAK2V617F mutation.  Pharmacokinetic studies in mice showed that I possessed good bioavailability after i.p. administration.  Finally, I showed antitumor efficacy with no significant toxicity in a HEL xenograft model.  Collectively, the results confirm the therapeutic potential of JAK and HDAC dual inhibitors in hematol. malignancies and provide valuable leads for further structural optimization and antitumor mechanism study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOXlwxqcv7m7Vg90H21EOLACvtfcHk0lhwXdY8zgg0NQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVSls7o%253D&md5=356363f40f24ac2ef32194e23a29328e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01597%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DChou%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Novel%2520Janus%2520Kinase%2520%2528JAK%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Dual%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Hematological%2520Malignancies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3898%26epage%3D3923%26doi%3D10.1021%2Facs.jmedchem.8b01597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cakici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karabuga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulukanli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilic, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Asymmetric Catalytic Activity of (1S,1′S)-4,4′-Biquinazoline-Based Primary Amines</span>. <i>Tetrahedron: Asymmetry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1016/j.tetasy.2011.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.tetasy.2011.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1GntL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=300-308&author=M.+Cakiciauthor=M.+Catirauthor=S.+Karabugaauthor=S.+Ulukanliauthor=H.+Kilic&title=Synthesis+and+Asymmetric+Catalytic+Activity+of+%281S%2C1%E2%80%B2S%29-4%2C4%E2%80%B2-Biquinazoline-Based+Primary+Amines&doi=10.1016%2Fj.tetasy.2011.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and asymmetric catalytic activity of (1S,1'S)-4,4'-biquinazoline-based primary amines</span></div><div class="casAuthors">Cakici, Murat; Catir, Mustafa; Karabuga, Semistan; Ulukanli, Sabri; Kilic, Hamdullah</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron: Asymmetry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">300-308</span>CODEN:
                <span class="NLM_cas:coden">TASYE3</span>;
        ISSN:<span class="NLM_cas:issn">0957-4166</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of (1S,1'S)-4,4'-biquinazoline-based primary amines I (R = Me, i-Pr, t-Bu, Bn) was prepd. from natural amino acids via a six-step reaction sequence of protection and condensation followed by key synthetic steps including chlorination, nickel(0)-mediated homocoupling, and deprotection.  These novel amines were screened for the asym. ethylation of aryl aldehydes with di-Et zinc to yield alcs. with an (S)-configuration with enantiomeric excesses (ee) varying from 2% to 95%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9txIL-5n3qrVg90H21EOLACvtfcHk0ljfDbYEFqN1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1GntL8%253D&md5=f47113d732de61f07d9686c2dd5ebbd8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tetasy.2011.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetasy.2011.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DCakici%26aufirst%3DM.%26aulast%3DCatir%26aufirst%3DM.%26aulast%3DKarabuga%26aufirst%3DS.%26aulast%3DUlukanli%26aufirst%3DS.%26aulast%3DKilic%26aufirst%3DH.%26atitle%3DSynthesis%2520and%2520Asymmetric%2520Catalytic%2520Activity%2520of%2520%25281S%252C1%25E2%2580%25B2S%2529-4%252C4%25E2%2580%25B2-Biquinazoline-Based%2520Primary%2520Amines%26jtitle%3DTetrahedron%253A%2520Asymmetry%26date%3D2011%26volume%3D22%26spage%3D300%26epage%3D308%26doi%3D10.1016%2Fj.tetasy.2011.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Sar Study of 5-Alkynyl Substituted Quinazolin-4(3H)-ones as Phosphoinositide 3-Kinase Delta (PI3Kδ) Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1156</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.ejmech.2016.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27846451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVChsbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=1156-1171&author=M.+Weiauthor=X.+Zhangauthor=X.+Wangauthor=Z.+Songauthor=J.+Dingauthor=L.+H.+Mengauthor=A.+Zhang&title=Sar+Study+of+5-Alkynyl+Substituted+Quinazolin-4%283H%29-ones+as+Phosphoinositide+3-Kinase+Delta+%28PI3K%CE%B4%29+Inhibitors&doi=10.1016%2Fj.ejmech.2016.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">SAR study of 5-alkynyl substituted quinazolin-4(3H)-ones as phosphoinositide 3-kinase delta (PI3Kδ) inhibitors</span></div><div class="casAuthors">Wei, Manman; Zhang, Xi; Wang, Xiang; Song, Zilan; Ding, Jian; Meng, Ling-Hua; Zhang, Ao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1156-1171</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">PI3Kδ is a key component in the aberrant signaling transduction in B cell malignancy, therefore specific targeting PI3Kδ has become an attractive molecularly targeted therapy for chronic lymphocytic leukemia (CLL).  Herein, we describe the discovery and optimization of a series of 5-alkynyl substituted PI3Kδ inhibitors based on the first FDA-approved inhibitor idelalisib.  Compd. 8d bearing the 1-morpholinohex-5-yn-1-one moiety as the C5-substituent was identified to have high potency against PI3Kδ (3.82 nM) and SU-DHL-6 cells (7.60 nM), resp.  It was 154-fold selective over PI3Kα, 133-fold selective against PI3Kβ, and 24-fold selective against PI3Kγ.  Treatment of MOLT-4 and SU-DHL-6 cells with compd. 8d for 1 h resulted in redn. of phosphorylation of both Akt (S473) and its downstream S6k1 (T389) in a concn.-dependent manner.  Compd. 8d showed potent anti-proliferative activity as well against T lymphoblast MOLT-4, suggesting its potential activity in T-cell leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYWHSwSiMyQrVg90H21EOLACvtfcHk0ljfDbYEFqN1aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVChsbjJ&md5=809c50e6dc45e1e4871af760fe27cefc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DL.%2BH.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DSar%2520Study%2520of%25205-Alkynyl%2520Substituted%2520Quinazolin-4%25283H%2529-ones%2520as%2520Phosphoinositide%25203-Kinase%2520Delta%2520%2528PI3K%25CE%25B4%2529%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D125%26spage%3D1156%26epage%3D1171%26doi%3D10.1016%2Fj.ejmech.2016.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adveef, A.</span></span> <span> </span><span class="NLM_article-title">Permeability—PAMPA</span>. In  <i>Absorption and Drug Development</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">498</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=319-498&author=A.+Adveef&title=Absorption+and+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAdveef%26aufirst%3DA.%26atitle%3DPermeability%25E2%2580%2594PAMPA%26btitle%3DAbsorption%2520and%2520Drug%2520Development%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26spage%3D319%26epage%3D498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evarts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forseth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haran, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashishian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppenol, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepist, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perreault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of Orally Efficacious Phosphoinositide 3-Kinase Delta Inhibitors with Improved Metabolic Stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9228</span>– <span class="NLM_lpage">9242</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01169</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01169" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGrs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9228-9242&author=L.+Patelauthor=J.+Chandrasekharauthor=J.+Evartsauthor=K.+Forsethauthor=A.+C.+Haranauthor=C.+Ipauthor=A.+Kashishianauthor=M.+Kimauthor=D.+Koditekauthor=S.+Koppenolauthor=L.+Ladauthor=E.+I.+Lepistauthor=M.+E.+McGrathauthor=S.+Perreaultauthor=K.+D.+Puriauthor=A.+G.+Villasenorauthor=J.+R.+Somozaauthor=B.+H.+Steinerauthor=J.+Therrienauthor=J.+Treibergauthor=G.+Phillips&title=Discovery+of+Orally+Efficacious+Phosphoinositide+3-Kinase+Delta+Inhibitors+with+Improved+Metabolic+Stability&doi=10.1021%2Facs.jmedchem.6b01169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Efficacious Phosphoinositide 3-Kinase δ Inhibitors with Improved Metabolic Stability</span></div><div class="casAuthors">Patel, Leena; Chandrasekhar, Jayaraman; Evarts, Jerry; Forseth, Kristen; Haran, Aaron C.; Ip, Carmen; Kashishian, Adam; Kim, Musong; Koditek, David; Koppenol, Sandy; Lad, Latesh; Lepist, Eve-Irene; McGrath, Mary E.; Perreault, Stephane; Puri, Kamal D.; Villasenor, Armando G.; Somoza, John R.; Steiner, Bart H.; Therrien, Joseph; Treiberg, Jennifer; Phillips, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9228-9242</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant signaling of phosphoinositide-3-kinase delta (PI3K-delta) has been implicated in numerous pathologies including hematol. malignancies and rheumatoid arthritis.  Described in this manuscript is the discovery, optimization and in vivo evaluation of a novel series of pyridine-contg. PI3K-delta inhibitors.  This work led to the discovery of I, a highly selective inhibitor of PI3K-delta which displays an excellent pharmacokinetic profile and is efficacious in a rodent model of rheumatoid arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1LgrP9YV8frVg90H21EOLACvtfcHk0lgGoJKnbnyCSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGrs7rF&md5=6d9dff0ec5270aab250fcb4800d8635a</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01169%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DEvarts%26aufirst%3DJ.%26aulast%3DForseth%26aufirst%3DK.%26aulast%3DHaran%26aufirst%3DA.%2BC.%26aulast%3DIp%26aufirst%3DC.%26aulast%3DKashishian%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DKoppenol%26aufirst%3DS.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DLepist%26aufirst%3DE.%2BI.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26aulast%3DPerreault%26aufirst%3DS.%26aulast%3DPuri%26aufirst%3DK.%2BD.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DSteiner%26aufirst%3DB.%2BH.%26aulast%3DTherrien%26aufirst%3DJ.%26aulast%3DTreiberg%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520Orally%2520Efficacious%2520Phosphoinositide%25203-Kinase%2520Delta%2520Inhibitors%2520with%2520Improved%2520Metabolic%2520Stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9228%26epage%3D9242%26doi%3D10.1021%2Facs.jmedchem.6b01169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evarts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haran, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepist, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perreault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span> <span> </span><span class="NLM_article-title">2,4,6-Triaminopyrimidine as a Novel Hinge Binder in a Series of PI3Kδ Selective Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3532</span>– <span class="NLM_lpage">3548</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00213</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00213" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1CgtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3532-3548&author=L.+Patelauthor=J.+Chandrasekharauthor=J.+Evartsauthor=A.+C.+Haranauthor=C.+Ipauthor=J.+A.+Kaplanauthor=M.+Kimauthor=D.+Koditekauthor=L.+Ladauthor=E.+I.+Lepistauthor=M.+E.+McGrathauthor=N.+Novikovauthor=S.+Perreaultauthor=K.+D.+Puriauthor=J.+R.+Somozaauthor=B.+H.+Steinerauthor=K.+L.+Stevensauthor=J.+Therrienauthor=J.+Treibergauthor=A.+G.+Villasenorauthor=A.+Yeungauthor=G.+Phillips&title=2%2C4%2C6-Triaminopyrimidine+as+a+Novel+Hinge+Binder+in+a+Series+of+PI3K%CE%B4+Selective+Inhibitors&doi=10.1021%2Facs.jmedchem.6b00213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">2,4,6-Triaminopyrimidine as a Novel Hinge Binder in a Series of PI3Kδ Selective Inhibitors</span></div><div class="casAuthors">Patel, Leena; Chandrasekhar, Jayaraman; Evarts, Jerry; Haran, Aaron C.; Ip, Carmen; Kaplan, Joshua A.; Kim, Musong; Koditek, David; Lad, Latesh; Lepist, Eve-Irene; McGrath, Mary E.; Novikov, Nikolai; Perreault, Stephane; Puri, Kamal D.; Somoza, John R.; Steiner, Bart H.; Stevens, Kirk L.; Therrien, Joseph; Treiberg, Jennifer; Villasenor, Armando G.; Yeung, Arthur; Phillips, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3532-3548</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of phosphoinositide 3-kinase δ (PI3Kδ) is an appealing target for several hematol. malignancies and inflammatory diseases.  Herein, we describe the discovery and optimization of a series of propeller shaped PI3Kδ inhibitors comprising a novel triaminopyrimidine hinge binder.  Combinations of electronic and structural strategies were employed to mitigate aldehyde oxidase mediated metab.  This medicinal chem. effort culminated in the identification of 52, a potent and highly selective inhibitor of PI3Kδ that demonstrates efficacy in a rat model of arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpygPlHbWp_TLVg90H21EOLACvtfcHk0lgGoJKnbnyCSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1CgtL4%253D&md5=9e01fff097eccd127a9b9284d725d7e1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00213%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DEvarts%26aufirst%3DJ.%26aulast%3DHaran%26aufirst%3DA.%2BC.%26aulast%3DIp%26aufirst%3DC.%26aulast%3DKaplan%26aufirst%3DJ.%2BA.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DLepist%26aufirst%3DE.%2BI.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26aulast%3DNovikov%26aufirst%3DN.%26aulast%3DPerreault%26aufirst%3DS.%26aulast%3DPuri%26aufirst%3DK.%2BD.%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DSteiner%26aufirst%3DB.%2BH.%26aulast%3DStevens%26aufirst%3DK.%2BL.%26aulast%3DTherrien%26aufirst%3DJ.%26aulast%3DTreiberg%26aufirst%3DJ.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DYeung%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DG.%26atitle%3D2%252C4%252C6-Triaminopyrimidine%2520as%2520a%2520Novel%2520Hinge%2520Binder%2520in%2520a%2520Series%2520of%2520PI3K%25CE%25B4%2520Selective%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3532%26epage%3D3548%26doi%3D10.1021%2Facs.jmedchem.6b00213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perreault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evarts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashishian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepist, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yahiaoui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Phosphoinositide 3-Kinase (PI3K) Beta/Delta Inhibitor for the Treatment of Phosphatase and Tensin Homolog (PTEN) Deficient Tumors: Building PI3Kβ Potency in a PI3Kδ-Selective Template by Targeting Nonconserved Asp856</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1567</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01821</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01821" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVansro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1555-1567&author=S.+Perreaultauthor=J.+Chandrasekharauthor=Z.+H.+Cuiauthor=J.+Evartsauthor=J.+Haoauthor=J.+A.+Kaplanauthor=A.+Kashishianauthor=K.+S.+Keeganauthor=T.+Kenneyauthor=D.+Koditekauthor=L.+Ladauthor=E.+I.+Lepistauthor=M.+E.+McGrathauthor=L.+Patelauthor=B.+Phillipsauthor=J.+Therrienauthor=J.+Treibergauthor=A.+Yahiaouiauthor=G.+Phillips&title=Discovery+of+a+Phosphoinositide+3-Kinase+%28PI3K%29+Beta%2FDelta+Inhibitor+for+the+Treatment+of+Phosphatase+and+Tensin+Homolog+%28PTEN%29+Deficient+Tumors%3A+Building+PI3K%CE%B2+Potency+in+a+PI3K%CE%B4-Selective+Template+by+Targeting+Nonconserved+Asp856&doi=10.1021%2Facs.jmedchem.6b01821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Phosphoinositide 3-Kinase (PI3K) β/δ Inhibitor for the Treatment of Phosphatase and Tensin Homolog (PTEN) Deficient Tumors: Building PI3Kβ Potency in a PI3Kδ-Selective Template by Targeting Nonconserved Asp856</span></div><div class="casAuthors">Perreault, Stephane; Chandrasekhar, Jayaraman; Cui, Zhi-Hua; Evarts, Jerry; Hao, Jia; Kaplan, Joshua A.; Kashishian, Adam; Keegan, Kathleen S.; Kenney, Thomas; Koditek, David; Lad, Latesh; Lepist, Eve-Irene; McGrath, Mary E.; Patel, Leena; Phillips, Bart; Therrien, Joseph; Treiberg, Jennifer; Yahiaoui, Anella; Phillips, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1555-1567</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinase (PI3K) beta signaling is required to sustain cancer cell growth in which the tumor suppressor phosphatase and tensin homolog (PTEN) has been deactivated.  This manuscript describes the discovery, optimization, and in vivo evaluation of a novel series of PI3Kbeta/delta inhibitors in which PI3Kbeta potency was built in a PI3Kdelta-selective template.  This work led to the discovery of a highly selective PI3Kbeta/delta inhibitor displaying excellent pharmacokinetic profile and efficacy in a human PTEN-deficient LNCaP prostate carcinoma xenograft tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf82mTX9c-e7Vg90H21EOLACvtfcHk0lgZJofidfGW2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVansro%253D&md5=0ebc8ca6291d39f945926e11b34fced6</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01821%26sid%3Dliteratum%253Aachs%26aulast%3DPerreault%26aufirst%3DS.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DZ.%2BH.%26aulast%3DEvarts%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DJ.%26aulast%3DKaplan%26aufirst%3DJ.%2BA.%26aulast%3DKashishian%26aufirst%3DA.%26aulast%3DKeegan%26aufirst%3DK.%2BS.%26aulast%3DKenney%26aufirst%3DT.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DLepist%26aufirst%3DE.%2BI.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26aulast%3DPatel%26aufirst%3DL.%26aulast%3DPhillips%26aufirst%3DB.%26aulast%3DTherrien%26aufirst%3DJ.%26aulast%3DTreiberg%26aufirst%3DJ.%26aulast%3DYahiaoui%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520Phosphoinositide%25203-Kinase%2520%2528PI3K%2529%2520Beta%252FDelta%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Phosphatase%2520and%2520Tensin%2520Homolog%2520%2528PTEN%2529%2520Deficient%2520Tumors%253A%2520Building%2520PI3K%25CE%25B2%2520Potency%2520in%2520a%2520PI3K%25CE%25B4-Selective%2520Template%2520by%2520Targeting%2520Nonconserved%2520Asp856%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1555%26epage%3D1567%26doi%3D10.1021%2Facs.jmedchem.6b01821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase 6 Structure and Molecular Basis of Catalysis and Inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnchembio.2134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27454933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=741-747&author=Y.+Haiauthor=D.+W.+Christianson&title=Histone+Deacetylase+6+Structure+and+Molecular+Basis+of+Catalysis+and+Inhibition&doi=10.1038%2Fnchembio.2134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span></div><div class="casAuthors">Hai, Yang; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">741-747</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a crit. target for drug design because of its role in oncogenic transformation and cancer metastasis, and is unique among all histone deacetylases in that it contains tandem catalytic domains designated CD1 and CD2.  We now report the crystal structures of CD2 from Homo sapiens HDAC6 and of CD1 and CD2 from Danio rerio HDAC6.  We correlated these structures with activity measurements using 13 different substrates.  The catalytic activity of CD2 from both species exhibited broad substrate specificity, whereas that of CD1 was highly specific for substrates bearing C-terminal acetyllysine residues.  Crystal structures of substrate complexes yielded unprecedented snapshots of the catalytic mechanism.  Addnl., crystal structures of complexes with eight different inhibitors, including belinostat and panobinostat (currently used in cancer chemotherapy), the macrocyclic tetrapeptide HC toxin, and the HDAC6-specific inhibitor N-hydroxy-4-(2-((2-hydroxyethyl)(phenyl)amino)-2-oxoethyl)benzamide, revealed surprising new insight regarding changes in Zn2+ coordination and isoenzyme-specific inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCfziGBHBRLVg90H21EOLACvtfcHk0lgZJofidfGW2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ&md5=e167bc6f66e00923da4f2bced2b306b9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2134%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520Deacetylase%25206%2520Structure%2520and%2520Molecular%2520Basis%2520of%2520Catalysis%2520and%2520Inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D741%26epage%3D747%26doi%3D10.1038%2Fnchembio.2134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span> <i>NCI-60
Human Tumor Cell Lines Screen</i>; <span class="NLM_publisher-name">National Cancer
Institute</span>, <span class="NLM_year">2015</span>; <a href="https://dtp.cancer.gov/discovery_development/nci-60/" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/</a> (accessed 2019-10-10).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+NCI-60%0AHuman+Tumor+Cell+Lines+Screen%3B+National+Cancer%0AInstitute%2C+2015%3B+https%3A%2F%2Fdtp.cancer.gov%2Fdiscovery_development%2Fnci-60%2F+%28accessed+2019-10-10%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26jtitle%3DNCI-60%250AHuman%2520Tumor%2520Cell%2520Lines%2520Screen%26pub%3DNational%2520Cancer%250AInstitute%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span> <i>Celltiter-Glo Luminescent
Cell Viability Assay</i>; <span class="NLM_publisher-name">Promega</span>, <span class="NLM_year">2015</span>; <a href="https://www.promega.com/-/media/files/resources/protocols/technical-bulletins/0/celltiter-glo-luminescent-cell-viability-assay-protocol.pdf?la=en" class="extLink">https://www.promega.com/-/media/files/resources/protocols/technical-bulletins/0/celltiter-glo-luminescent-cell-viability-assay-protocol.pdf?la=en</a> (accessed 2019-10-15).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Celltiter-Glo+Luminescent%0ACell+Viability+Assay%3B+Promega%2C+2015%3B+https%3A%2F%2Fwww.promega.com%2F-%2Fmedia%2Ffiles%2Fresources%2Fprotocols%2Ftechnical-bulletins%2F0%2Fcelltiter-glo-luminescent-cell-viability-assay-protocol.pdf%3Fla%3Den+%28accessed+2019-10-15%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DCelltiter-Glo%2520Luminescent%250ACell%2520Viability%2520Assay%26pub%3DPromega%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sattler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span> <span> </span><span class="NLM_article-title">Reversible Resistance Induced by FLT3 Inhibition: A Novel Resistance Mechanism in Mutant FLT3-Expressing Cells</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e25351</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0025351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1371%2Fjournal.pone.0025351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21980431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25351&author=E.+Weisbergauthor=A.+Rayauthor=E.+Nelsonauthor=S.+Adamiaauthor=R.+Barrettauthor=M.+Sattlerauthor=C.+Zhangauthor=J.+F.+Daleyauthor=D.+Frankauthor=E.+Foxauthor=J.+D.+Griffin&title=Reversible+Resistance+Induced+by+FLT3+Inhibition%3A+A+Novel+Resistance+Mechanism+in+Mutant+FLT3-Expressing+Cells&doi=10.1371%2Fjournal.pone.0025351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible resistance induced by FLT3 inhibition: a novel resistance mechanism in mutant FLT3-expressing cells</span></div><div class="casAuthors">Weisberg, Ellen; Ray, Arghya; Nelson, Erik; Adamia, Sophia; Barrett, Rosemary; Sattler, Martin; Zhang, Chengsheng; Daley, John F.; Frank, David; Fox, Edward; Griffin, James D.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25351</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Objectives: Clin. responses achieved with FLT3 kinase inhibitors in acute myeloid leukemia (AML) are typically transient and partial.  Thus, there is a need for identification of mol. mechanisms of clin. resistance to these drugs.  In response, we characterized MOLM13 AML cell lines made resistant to two structurally-independent FLT3 inhibitors.  Methods: MOLM13 cells were made drug resistant via prolonged exposure to midostaurin and HG-7-85-01, resp.  Cell proliferation was detd. by Trypan blue exclusion.  Protein expression was assessed by immunoblotting, immunopptn., and flow cytometry.  Cycloheximide was used to det. protein half-life.  RT-PCR was performed to det. FLT3 mRNA levels, and FISH anal. was performed to det. FLT3 gene expression.  Results and Conclusions: We found that MOLM13 cells readily developed cross-resistance when exposed to either midostaurin or HG-7-85-01.  Resistance in both lines was assocd. with dramatically elevated levels of cell surface FLT3 and elevated levels of phosphor-MAPK, but not phospho-STAT5.  The increase in FLT3-ITD expression was at least in part due to reduced turnover of the receptor, with prolonged half-life.  Importantly, the drug-resistant phenotype could be rapidly reversed upon withdrawal of either inhibitor.  Consistent with this phenotype, no significant evidence of FLT3 gene amplification, kinase domain mutations, or elevated levels of mRNA was obsd., suggesting that protein turnover may be part of an auto-regulatory pathway initiated by FLT3 kinase activity.  Interestingly, FLT3 inhibitor resistance also correlated with resistance to cytosine arabinoside.  Over-expression of FLT3 protein in response to kinase inhibitors may be part of a novel mechanism that could contribute to clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWa-oQ_co7RLVg90H21EOLACvtfcHk0linXz7jHkKNOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCjsL7M&md5=bc8ce2e7906ac3bce393573ee7a6e685</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025351%26sid%3Dliteratum%253Aachs%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DNelson%26aufirst%3DE.%26aulast%3DAdamia%26aufirst%3DS.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DSattler%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDaley%26aufirst%3DJ.%2BF.%26aulast%3DFrank%26aufirst%3DD.%26aulast%3DFox%26aufirst%3DE.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26atitle%3DReversible%2520Resistance%2520Induced%2520by%2520FLT3%2520Inhibition%253A%2520A%2520Novel%2520Resistance%2520Mechanism%2520in%2520Mutant%2520FLT3-Expressing%2520Cells%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25351%26doi%3D10.1371%2Fjournal.pone.0025351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bosshart, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzelmann, M.</span></span> <span> </span><span class="NLM_article-title">THP-1 Cells as a Model for Human Monocytes</span>. <i>Ann. Transl Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">438</span>, <span class="refDoi"> DOI: 10.21037/atm.2016.08.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.21037%2Fatm.2016.08.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27942529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslOhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=438&author=H.+Bosshartauthor=M.+Heinzelmann&title=THP-1+Cells+as+a+Model+for+Human+Monocytes&doi=10.21037%2Fatm.2016.08.53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">THP-1 cells as a model for human monocytes</span></div><div class="casAuthors">Bosshart, Herbert; Heinzelmann, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Translational Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">438/1-438/4</span>CODEN:
                <span class="NLM_cas:coden">ATMNDX</span>;
        ISSN:<span class="NLM_cas:issn">2305-5847</span>.
    
            (<span class="NLM_cas:orgname">AME Publishing Co.</span>)
        </div><div class="casAbstract">In mammals, circulating monocytes provide effective protection against infection with gram-neg. bacteria.  Once challenged with lipolysaccharide (LPS), a macromol. structure of the cell wall of Gram-neg. bacteria, peripheral monocytes migrate to lymph nodes, differentiation into dendritic cells, and present antigen to CD4 and CD8 T cells with high efficacy, therby initiating adaptive immunity against gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEYECqXXAYJ7Vg90H21EOLACvtfcHk0linXz7jHkKNOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslOhsrc%253D&md5=794ec7557dd34eaa3185f2f33ea94261</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.21037%2Fatm.2016.08.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21037%252Fatm.2016.08.53%26sid%3Dliteratum%253Aachs%26aulast%3DBosshart%26aufirst%3DH.%26aulast%3DHeinzelmann%26aufirst%3DM.%26atitle%3DTHP-1%2520Cells%2520as%2520a%2520Model%2520for%2520Human%2520Monocytes%26jtitle%3DAnn.%2520Transl%2520Med.%26date%3D2016%26volume%3D4%26spage%3D438%26doi%3D10.21037%2Fatm.2016.08.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, R. P.</span>; <span class="NLM_string-name">Madhivanan, S.</span>; <span class="NLM_string-name">Sundararajan, R.</span>; <span class="NLM_string-name">Wan-Ying Lin, C.</span>; <span class="NLM_string-name">Sankaranarayanan, K.</span></span> <span> </span><span class="NLM_article-title">An in Vitro Study of Electroporation of Leukemia and Cervical Cancer Cells</span>. In  <i>Electroporation-Based Therapies for Cancer</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundararajan, R.</span></span>, Ed.; <span class="NLM_publisher-name">Woodhead Publishing</span>, <span class="NLM_publisher-loc">Cambridge UK</span>, <span class="NLM_year">2014</span>; Chapter 7, pp  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">183</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1533%2F9781908818294.161" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=161-183&author=R.+P.+Ramachandran&author=S.+Madhivanan&author=R.+Sundararajan&author=C.+Wan-Ying+Lin&author=K.+Sankaranarayananauthor=R.+Sundararajan&title=Electroporation-Based+Therapies+for+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1533%2F9781908818294.161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1533%252F9781908818294.161%26sid%3Dliteratum%253Aachs%26aulast%3DRamachandran%26aufirst%3DR.%2BP.%26atitle%3DAn%2520in%2520Vitro%2520Study%2520of%2520Electroporation%2520of%2520Leukemia%2520and%2520Cervical%2520Cancer%2520Cells%26btitle%3DElectroporation-Based%2520Therapies%2520for%2520Cancer%26aulast%3DSundararajan%26aufirst%3DR.%26pub%3DWoodhead%2520Publishing%26date%3D2014%26spage%3D161%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLeod, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uphoff, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishizaki, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orita, K.</span></span> <span> </span><span class="NLM_article-title">Two Acute Monocytic Leukemia (AML-M5a) Cell Lines (MOLM-13 and MOLM-14) with Interclonal Phenotypic Heterogeneity Showing MLL-AF9 Fusion Resulting from an Occult Chromosome Insertion, Ins(11;9)(Q23;P22p23)</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1477</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2400768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fsj.leu.2400768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=9305600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADyaK2sXnsFagsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=1469-1477&author=Y.+Matsuoauthor=R.+A.+MacLeodauthor=C.+C.+Uphoffauthor=H.+G.+Drexlerauthor=C.+Nishizakiauthor=Y.+Katayamaauthor=G.+Kimuraauthor=N.+Fujiiauthor=E.+Omotoauthor=M.+Haradaauthor=K.+Orita&title=Two+Acute+Monocytic+Leukemia+%28AML-M5a%29+Cell+Lines+%28MOLM-13+and+MOLM-14%29+with+Interclonal+Phenotypic+Heterogeneity+Showing+MLL-AF9+Fusion+Resulting+from+an+Occult+Chromosome+Insertion%2C+Ins%2811%3B9%29%28Q23%3BP22p23%29&doi=10.1038%2Fsj.leu.2400768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Two acute monocytic leukemia (AML-M5a) cell lines (MOLM-13 and MOLM-14) with interclonal phenotypic heterogeneity showing MLL-AF9 fusion resulting from an occult chromosome insertion, ins(11;9)(q23;p22p23)</span></div><div class="casAuthors">Matsuo, Y.; Macleod, R. A. F.; Uphoff, C. C.; Drexler, H. G.; Nishizaki, C.; Katayama, Y.; Kimura, G.; Fujii, N.; Omoto, E.; Harada, M.; Orita, K.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1469-1477</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">We describe two new human leukemia cell lines, MOLM-13 and MOLM-14, established from the peripheral blood of a patient at relapse of acute monocytic leukemia, FAB M5a, which had evolved from myelodysplastic syndrome (MDS).  Both cell lines express monocyte-specific esterase (MSE) and MLL-AF9 fusion mRNA.  Gene fusion is assocd. with a minute chromosomal insertion, ins(11;9)(q23;p22p23).  MOLM-13 and MOLM-14 are the first cell lines with, and represent the third reported case of, MLL gene rearrangement arising via chromosomal insertion.  Both cell lines carry trisomy 8 which was also present during the MDS phase, as well as the most frequent trisomies assocd. with t(9;11), ie, +6, +13, +19 variously present in different subclones.  Despite having these features in common, differences in antigen expression were noted between the two cell lines: that of MOLM-13 being CD34+, CD13-, CD14-, CD15+, CD33+; whereas MOLM-14 was CD4+, CD13+, CD14+, CD15+, CD33+.  Differentiation to macrophage-like morphol. could be induced in both cell lines after stimulation with INF-γ alone, or in combination with TNF-α, which treatment also induced or upregulated, expression of certain myelomonocyte-assocd. antigens, including CD13, CD14, CD15, CD64, CD65 and CD87.  Together, these data confirm that both cell lines are likely to be novel in vitro models for studying monocytic differentiation and leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYiVJrnDc7BbVg90H21EOLACvtfcHk0ljdy1pvicipig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsFagsrc%253D&md5=8bec3174ce582e7829f78e9bd8946154</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2400768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2400768%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuo%26aufirst%3DY.%26aulast%3DMacLeod%26aufirst%3DR.%2BA.%26aulast%3DUphoff%26aufirst%3DC.%2BC.%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26aulast%3DNishizaki%26aufirst%3DC.%26aulast%3DKatayama%26aufirst%3DY.%26aulast%3DKimura%26aufirst%3DG.%26aulast%3DFujii%26aufirst%3DN.%26aulast%3DOmoto%26aufirst%3DE.%26aulast%3DHarada%26aufirst%3DM.%26aulast%3DOrita%26aufirst%3DK.%26atitle%3DTwo%2520Acute%2520Monocytic%2520Leukemia%2520%2528AML-M5a%2529%2520Cell%2520Lines%2520%2528MOLM-13%2520and%2520MOLM-14%2529%2520with%2520Interclonal%2520Phenotypic%2520Heterogeneity%2520Showing%2520MLL-AF9%2520Fusion%2520Resulting%2520from%2520an%2520Occult%2520Chromosome%2520Insertion%252C%2520Ins%252811%253B9%2529%2528Q23%253BP22p23%2529%26jtitle%3DLeukemia%26date%3D1997%26volume%3D11%26spage%3D1469%26epage%3D1477%26doi%3D10.1038%2Fsj.leu.2400768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galinsky, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span> <span> </span><span class="NLM_article-title">Characterization of Selective and Potent PI3Kδ Inhibitor (PI3KDIN-015) for B-Cell Malignances</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">32641</span>– <span class="NLM_lpage">32651</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.8702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.18632%2Foncotarget.8702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27081697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC28bgtFOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=32641-32651&author=X.+Liuauthor=A.+Wangauthor=X.+Liangauthor=C.+Chenauthor=J.+Liuauthor=Z.+Zhaoauthor=H.+Wuauthor=Y.+Dengauthor=L.+Wangauthor=B.+Wangauthor=J.+Wuauthor=F.+Liuauthor=S.+M.+Fernandesauthor=S.+Adamiaauthor=R.+M.+Stoneauthor=I.+A.+Galinskyauthor=J.+R.+Brownauthor=J.+D.+Griffinauthor=S.+Zhangauthor=T.+Lohauthor=X.+Zhangauthor=W.+Wangauthor=E.+L.+Weisbergauthor=J.+Liuauthor=Q.+Liu&title=Characterization+of+Selective+and+Potent+PI3K%CE%B4+Inhibitor+%28PI3KDIN-015%29+for+B-Cell+Malignances&doi=10.18632%2Foncotarget.8702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of selective and potent PI3Kδ inhibitor (PI3KDIN- 015) for B-Cell malignances</span></div><div class="casAuthors">Liu Xiaochuan; Loh Teckpeng; Liu Xiaochuan; Wang Aoli; Liang Xiaofei; Chen Cheng; Liu Juanjuan; Zhao Zheng; Wu Hong; Deng Yuanxin; Wang Li; Wang Beilei; Wu Jiaxin; Liu Feiyang; Zhang Xin; Wang Wenchao; Liu Jing; Liu Qingsong; Wang Aoli; Liu Juanjuan; Wu Hong; Deng Yuanxin; Wu Jiaxin; Liu Feiyang; Liang Xiaofei; Chen Cheng; Zhao Zheng; Wang Li; Wang Beilei; Zhang Shanchun; Wang Wenchao; Liu Jing; Liu Qingsong; Fernandes Stacey M; Adamia Sophia; Stone Richard M; Galinsky Ilene A; Brown Jennifer R; Griffin James D; Weisberg Ellen L; Zhang Shanchun; Liu Qingsong</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">32641-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PI3Kδ is predominately expressed in leukocytes and has been found overexpressed in B-cell related malignances such as CLL and AML.  We have discovered a highly selective ATP competitive PI3Kd inhibitor PI3KD-IN-015, which exhibits a high selectivity among other PI3K isoforms in both biochemical assays and cellular assay, meanwhile did not inhibit most of other protein kinases in the kinome.  PI3KD-IN-015 demonstrates moderately anti-proliferation efficacies against a variety of B-cell related cancer cell lines through down-regulate the PI3K signaling significantly.  It induced both apoptosis and autophagy in B-cell malignant cell lines.  In addition, combination of autophagy inhibitor Bafilomycin could potentiate the moderate anti-proliferation effect of PI3KD-IN-015.  PI3KD-IN-015 shows anti-proliferation efficacy against CLL and AML patient primary cells.  Collectively, these results indicate that PI3KD-IN-015 may be useful drug candidate for further development of anti-B-cell related malignances therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBYhYh3CW1QWZyxfu64m4ffW6udTcc2eYUOt7hn79lbbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bgtFOgtw%253D%253D&md5=58b7ed0f416a5793f540405880c1c9f9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.8702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.8702%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DFernandes%26aufirst%3DS.%2BM.%26aulast%3DAdamia%26aufirst%3DS.%26aulast%3DStone%26aufirst%3DR.%2BM.%26aulast%3DGalinsky%26aufirst%3DI.%2BA.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLoh%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DCharacterization%2520of%2520Selective%2520and%2520Potent%2520PI3K%25CE%25B4%2520Inhibitor%2520%2528PI3KDIN-015%2529%2520for%2520B-Cell%2520Malignances%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D32641%26epage%3D32651%26doi%3D10.18632%2Foncotarget.8702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novotny-Diermayr, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangthongpitag, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sausgruber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greicius, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hentze, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethirajulu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. M.</span></span> <span> </span><span class="NLM_article-title">SB939, a Novel Potent and Orally Active Histone Deacetylase Inhibitor with High Tumor Exposure and Efficacy in Mouse Models of Colorectal Cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">642</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1158%2F1535-7163.MCT-09-0689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=20197387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFens7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=642-652&author=V.+Novotny-Diermayrauthor=K.+Sangthongpitagauthor=C.+Y.+Huauthor=X.+Wuauthor=N.+Sausgruberauthor=P.+Yeoauthor=G.+Greiciusauthor=S.+Petterssonauthor=A.+L.+Liangauthor=Y.+K.+Lohauthor=Z.+Bondayauthor=K.+C.+Gohauthor=H.+Hentzeauthor=S.+Hartauthor=H.+Wangauthor=K.+Ethirajuluauthor=J.+M.+Wood&title=SB939%2C+a+Novel+Potent+and+Orally+Active+Histone+Deacetylase+Inhibitor+with+High+Tumor+Exposure+and+Efficacy+in+Mouse+Models+of+Colorectal+Cancer&doi=10.1158%2F1535-7163.MCT-09-0689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">SB939, a Novel Potent and Orally Active Histone Deacetylase Inhibitor with High Tumor Exposure and Efficacy in Mouse Models of Colorectal Cancer</span></div><div class="casAuthors">Novotny-Diermayr, Veronica; Sangthongpitag, Kanda; Hu, Chang Yong; Wu, Xiaofeng; Sausgruber, Nina; Yeo, Pauline; Greicius, Gediminas; Pettersson, Sven; Liang, Ai Leng; Loh, Yung Kiang; Bonday, Zahid; Goh, Kee Chuan; Hentze, Hannes; Hart, Stefan; Wang, Haishan; Ethirajulu, Kantharaj; Wood, Jeanette Marjorie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">642-652</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although clin. responses in liq. tumors and certain lymphomas have been reported, the clin. efficacy of histone deacetylase inhibitors in solid tumors has been limited.  This may be in part due to the poor pharmacokinetic of these drugs, resulting in inadequate tumor concns. of the drug.  SB939 is a new hydroxamic acid based histone deacetylase inhibitor with improved physicochem., pharmaceutical, and pharmacokinetic properties.  In vitro, SB939 inhibits class I, II, and IV HDACs, with no effects on other zinc binding enzymes, and shows significant antiproliferative activity against a wide variety of tumor cell lines.  It has very favorable pharmacokinetic properties after oral dosing in mice, with >4-fold increased bioavailability and 3.3-fold increased half-life over suberoylanilide hydroxamic acid (SAHA).  In contrast to SAHA, SB939 accumulates in tumor tissue and induces a sustained inhibition of histone acetylation in tumor tissue.  These excellent pharmacokinetic properties translated into a dose-dependent antitumor efficacy in a xenograft model of human colorectal cancer (HCT-116), with a tumor growth inhibition of 94% vs. 48% for SAHA (both at max. tolerated dose), and was also effective when given in different intermittent schedules.  Furthermore, in APCmin mice, a genetic mouse model of early-stage colon cancer, SB939 inhibited adenoma formation, hemocult scores, and increased hematocrit values more effectively than 5-fluorouracil.  Emerging clin. data from phase I trials in cancer patients indicate that the pharmacokinetic and pharmacol. advantages of SB939 are translated to the clinic.  The efficacy of SB939 reported here in two very different models of colorectal cancer warrants further investigation in patients.  Mol Cancer Ther; 9(3); 642-52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw7LYpqzlxHbVg90H21EOLACvtfcHk0linuRPedN-WEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFens7Y%253D&md5=ffe0b917c40862c4ca0b414c8ef4c493</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0689%26sid%3Dliteratum%253Aachs%26aulast%3DNovotny-Diermayr%26aufirst%3DV.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DC.%2BY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSausgruber%26aufirst%3DN.%26aulast%3DYeo%26aufirst%3DP.%26aulast%3DGreicius%26aufirst%3DG.%26aulast%3DPettersson%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DA.%2BL.%26aulast%3DLoh%26aufirst%3DY.%2BK.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DGoh%26aufirst%3DK.%2BC.%26aulast%3DHentze%26aufirst%3DH.%26aulast%3DHart%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DEthirajulu%26aufirst%3DK.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26atitle%3DSB939%252C%2520a%2520Novel%2520Potent%2520and%2520Orally%2520Active%2520Histone%2520Deacetylase%2520Inhibitor%2520with%2520High%2520Tumor%2520Exposure%2520and%2520Efficacy%2520in%2520Mouse%2520Models%2520of%2520Colorectal%2520Cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D642%26epage%3D652%26doi%3D10.1158%2F1535-7163.MCT-09-0689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madlambayan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzinic, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taub, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Y.</span></span> <span> </span><span class="NLM_article-title">Antileukemic Activity and Mechanism of Action of the Novel PI3K and Histone Deacetylase Dual Inhibitor CUDC-907 in Acute Myeloid Leukemia</span>. <i>Haematologica</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">2225</span>– <span class="NLM_lpage">2240</span>, <span class="refDoi"> DOI: 10.3324/haematol.2018.201343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3324%2Fhaematol.2018.201343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30819918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFantb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2019&pages=2225-2240&author=X.+Liauthor=Y.+Suauthor=G.+Madlambayanauthor=H.+Edwardsauthor=L.+Polinauthor=J.+Kushnerauthor=S.+H.+Dzinicauthor=K.+Whiteauthor=J.+Maauthor=T.+Knightauthor=G.+Wangauthor=Y.+Wangauthor=J.+Yangauthor=J.+W.+Taubauthor=H.+Linauthor=Y.+Ge&title=Antileukemic+Activity+and+Mechanism+of+Action+of+the+Novel+PI3K+and+Histone+Deacetylase+Dual+Inhibitor+CUDC-907+in+Acute+Myeloid+Leukemia&doi=10.3324%2Fhaematol.2018.201343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Antileukemic activity and mechanism of action of the novel PI3K and histone deacetylase dual inhibitor CUDC-907 in acute myeloid leukemia</span></div><div class="casAuthors">Li, Xinyu; Su, Yongwei; Madlambayan, Gerard; Edwards, Holly; Polin, Lisa; Kushner, Juiwanna; Dzinic, Sijana H.; White, Kathryn; Ma, Jun; Knight, Tristan; Wang, Guan; Wang, Yue; Yang, Jay; Taub, Jeffrey W.; Lin, Hai; Ge, Yubin</div><div class="citationInfo"><span class="NLM_cas:title">Haematologica</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2225-2240</span>CODEN:
                <span class="NLM_cas:coden">HAEMAX</span>;
        ISSN:<span class="NLM_cas:issn">1592-8721</span>.
    
            (<span class="NLM_cas:orgname">Ferrata Storti Foundation</span>)
        </div><div class="casAbstract">I nduction therapy for patients with acute myeloid leukemia (AML) has remained largely unchanged for over 40 years, while overall survival rates remain unacceptably low, highlighting the need for new therapies.  The PI3K/Akt pathway is constitutively active in the majority of patients with AML.  Given that histone deacetylase inhibitors have been shown to synergize with PI3K inhibitors in preclin. AML models, we investigated the novel dual-acting PI3K and histone deacetylase inhibitor CUDC-907 in AML cells both in vitro and in vivo.  We demonstrated that CUDC-907 induces apoptosis in AML cell lines and primary AML samples and shows in vivo efficacy in an AML cell line-derived xenograft mouse model.  CUDC-907-induced apoptosis was partially dependent on Mcl-1, Bim, and c-Myc.  CUDC-907 induced DNA damage in AML cells while sparing normal hematopoietic cells.  Downregulation of CHK1, Wee1, and RRM1, and induction of DNA damage also contributed to CUDC-907-induced apoptosis of AML cells.  In addn., CUDC-907 treatment decreased leukemia progenitor cells in primary AML samples ex vivo, while also sparing normal hematopoietic progenitor cells.  These findings support the clin. development of CUDC-907 for the treatment of AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfDoYe90TwbVg90H21EOLACvtfcHk0liU0U9A5DQ5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFantb7I&md5=850a38d8ad0b426511353e6efa053985</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2018.201343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2018.201343%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DMadlambayan%26aufirst%3DG.%26aulast%3DEdwards%26aufirst%3DH.%26aulast%3DPolin%26aufirst%3DL.%26aulast%3DKushner%26aufirst%3DJ.%26aulast%3DDzinic%26aufirst%3DS.%2BH.%26aulast%3DWhite%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DKnight%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTaub%26aufirst%3DJ.%2BW.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DGe%26aufirst%3DY.%26atitle%3DAntileukemic%2520Activity%2520and%2520Mechanism%2520of%2520Action%2520of%2520the%2520Novel%2520PI3K%2520and%2520Histone%2520Deacetylase%2520Dual%2520Inhibitor%2520CUDC-907%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DHaematologica%26date%3D2019%26volume%3D104%26spage%3D2225%26epage%3D2240%26doi%3D10.3324%2Fhaematol.2018.201343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avigan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblatt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharbanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufe, D.</span></span> <span> </span><span class="NLM_article-title">MUC1-C Oncoprotein Suppresses Reactive Oxygen Species-Induced Terminal Differentiation of Acute Myelogenous Leukemia Cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">4863</span>– <span class="NLM_lpage">4870</span>, <span class="refDoi"> DOI: 10.1182/blood-2010-10-296632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1182%2Fblood-2010-10-296632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21422470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVWms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2011&pages=4863-4870&author=L.+Yinauthor=Z.+Wuauthor=D.+Aviganauthor=J.+Rosenblattauthor=R.+Stoneauthor=S.+Kharbandaauthor=D.+Kufe&title=MUC1-C+Oncoprotein+Suppresses+Reactive+Oxygen+Species-Induced+Terminal+Differentiation+of+Acute+Myelogenous+Leukemia+Cells&doi=10.1182%2Fblood-2010-10-296632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">MUC1-C oncoprotein suppresses reactive oxygen species-induced terminal differentiation of acute myelogenous leukemia cells</span></div><div class="casAuthors">Yin, Li; Wu, Zekui; Avigan, David; Rosenblatt, Jacalyn; Stone, Richard; Kharbanda, Surender; Kufe, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4863-4870</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) cells are characterized by unlimited self-renewal and an impaired capacity to undergo terminal differentiation.  The MUC1 oncoprotein is aberrantly expressed in AML cells; however, there has been no evidence for involvement of MUC1 in myeloid leukemogenesis.  Cell-penetrating peptide inhibitors of the MUC1-C subunit block its oligomerization and thereby oncogenic function.  The present results demonstrate that treatment of human MOLM-14 and MV4-11 AML cells with these inhibitors is assocd. with arrest of growth, induction of late apoptosis/necrosis, and loss of self-renewal capacity.  Similar results were obtained with primary blasts from patients with AML.  Inhibition of MUC1-C was assocd. with increases in reactive oxygen species (ROS) and depletion of glutathione.  Increases in ROS have been linked to induction of hematopoietic cell differentiation along the myeloid lineage.  In this regard, inhibition of MUC1-C was assocd. with induction of a terminally differentiated myeloid phenotype in AML cell lines and primary blasts by an ROS-dependent mechanism.  These findings indicate that MUC1-C function is of importance to AML cell self-renewal and that inhibition of MUC1-C represents a potential therapeutic approach to induce terminal differentiation of AML cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rwCDYhyZALVg90H21EOLACvtfcHk0liU0U9A5DQ5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVWms7Y%253D&md5=a69f20f0d3dfd3872398b6bccb223e0c</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1182%2Fblood-2010-10-296632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2010-10-296632%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DAvigan%26aufirst%3DD.%26aulast%3DRosenblatt%26aufirst%3DJ.%26aulast%3DStone%26aufirst%3DR.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DKufe%26aufirst%3DD.%26atitle%3DMUC1-C%2520Oncoprotein%2520Suppresses%2520Reactive%2520Oxygen%2520Species-Induced%2520Terminal%2520Differentiation%2520of%2520Acute%2520Myelogenous%2520Leukemia%2520Cells%26jtitle%3DBlood%26date%3D2011%26volume%3D117%26spage%3D4863%26epage%3D4870%26doi%3D10.1182%2Fblood-2010-10-296632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asawa, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallgren, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp-Thomas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span> <span> </span><span class="NLM_article-title">A Widely-Applicable High-Throughput Cellular Thermal Shift Assay (CETSA) Using Split Nano Luciferase</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">9472</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-27834-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fs41598-018-27834-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29930256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9472&author=N.+J.+Martinezauthor=R.+R.+Asawaauthor=M.+G.+Cyrauthor=A.+Zakharovauthor=D.+J.+Urbanauthor=J.+S.+Rothauthor=E.+Wallgrenauthor=C.+Klumpp-Thomasauthor=N.+P.+Coussensauthor=G.+Raiauthor=S.-M.+Yangauthor=M.+D.+Hallauthor=J.+J.+Maruganauthor=A.+Simeonovauthor=M.+J.+Henderson&title=A+Widely-Applicable+High-Throughput+Cellular+Thermal+Shift+Assay+%28CETSA%29+Using+Split+Nano+Luciferase&doi=10.1038%2Fs41598-018-27834-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A widely-applicable high-throughput cellular thermal shift assay (CETSA) using split Nano Luciferase</span></div><div class="casAuthors">Martinez Natalia J; Asawa Rosita R; Cyr Matthew G; Zakharov Alexey; Urban Daniel J; Roth Jacob S; Wallgren Eric; Klumpp-Thomas Carleen; Coussens Nathan P; Rai Ganesha; Yang Shyh-Ming; Hall Matthew D; Marugan Juan J; Simeonov Anton; Henderson Mark J</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9472</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Assessment of the interactions between a drug and its protein target in a physiologically relevant cellular environment constitutes a major challenge in the pre-clinical drug discovery space.  The Cellular Thermal Shift Assay (CETSA) enables such an assessment by quantifying the changes in the thermal stability of proteins upon ligand binding in intact cells.  Here, we present the development and validation of a homogeneous, standardized, target-independent, and high-throughput (384- and 1536-well formats) CETSA platform that uses a split Nano Luciferase approach (SplitLuc CETSA).  The broad applicability of the assay was demonstrated for diverse targets, and its performance was compared with independent biochemical and cell-based readouts using a set of well-characterized inhibitors.  Moreover, we investigated the utility of the platform as a primary assay for high-throughput screening.  The SplitLuc CETSA presented here enables target engagement studies for medium and high-throughput applications.  Additionally, it provides a rapid assay development and screening platform for targets where phenotypic or other cell-based assays are not readily available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD30bRGadycx0LsK7uFuY7fW6udTcc2ea8VG1hd2FDqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D&md5=d4d1c56ac6d2cb79fe69911293cab32c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-27834-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-27834-y%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26aulast%3DAsawa%26aufirst%3DR.%2BR.%26aulast%3DCyr%26aufirst%3DM.%2BG.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DRoth%26aufirst%3DJ.%2BS.%26aulast%3DWallgren%26aufirst%3DE.%26aulast%3DKlumpp-Thomas%26aufirst%3DC.%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26atitle%3DA%2520Widely-Applicable%2520High-Throughput%2520Cellular%2520Thermal%2520Shift%2520Assay%2520%2528CETSA%2529%2520Using%2520Split%2520Nano%2520Luciferase%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D9472%26doi%3D10.1038%2Fs41598-018-27834-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span> <i>Molecular Operating Environment (MOE)</i>, <span class="NLM_edition">2013.08</span>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span>, <span class="NLM_year">2013</span>; <a href="https://www.chemcomp.com/Products.htm" class="extLink">https://www.chemcomp.com/Products.htm</a> (accessed 2019-03-02).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+ULC%2C+2013%3B+https%3A%2F%2Fwww.chemcomp.com%2FProducts.htm+%28accessed+2019-03-02%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization, and Performance of MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<490::AID-JCC1>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=490-519&author=T.+A.+Halgren&title=Merck+Molecular+Force+Field.+I.+Basis%2C+Form%2C+Scope%2C+Parameterization%2C+and+Performance+of+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">490-519</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article introduces MMFF94, the initial published version of the Merck mol. force field (MMFF).  It describes the objectives set for MMFF, the form it takes, and the range of systems to which it applies.  This study also outlines the methodol. employed in parameterizing MMFF94 and summarizes its performance in reproducing computational and exptl. data.  Though similar to MM3 in some respects, MMFF94 differs in ways intended to facilitate application to condensed-phase processes in mol.-dynamics simulations.  Indeed, MMFF94 seeks to achieve MM3-like accuracy for small mols. in a combined "org./protein" force field that is equally applicable to proteins and other systems of biol. significance.  A second distinguishing feature is that the core protion of MMFF94 has primarily been derived from high-quality computational data-ca. 500 mol. structures optimized at the HF/6-31G* level, 475 structures optimized at the MP2/6-31G* level, 380 MP2/6-31* structures evaluated at a defined approxn. to the MP4SDQ/TZP level, and 1450 structures partly derived from MP2/6-31G* geometries and evaluated at the MP2/TZP level.  A third distinguishing feature is that MMFF94 has been parameterized for a wide variety of chem. systems of interest to org. and combinations of functional groups for which little, if any, useful exptl. data are available.  The methodol. used in parameterizing MMFF94 represents a fourth distinguishing feature.  Rather than using the common "functional group" approach, nearly all MMFF parameters have been detd. in a mutually consistent fashion from the full set of available computational data.  MMFF94 reproduces the computational data used in its parameterization very well.  In addn., MMFF94 reproduces exptl. bond lengths (0.014 Å root mean square [rms]), bond angles (1.2° rms), vibrational frequencies (61 cm-1 rms), conformational energies (0.38 kcal/mol rms), and rotational barriers (0.39 kcal/mol rms) very nearly as well as does MM3 for comparable systems.  MMFF94 also describes intermol. interactions in hydrogen-bonded systems in a way that closely parallels that given by the highly regarded OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAzCYfKnN83rVg90H21EOLACvtfcHk0ljE-0Vd14Y9fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D&md5=56fb073477b4f49d1dfbd786fc56a480</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C490%253A%253AAID-JCC1%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520Molecular%2520Force%2520Field.%2520I.%2520Basis%252C%2520Form%252C%2520Scope%252C%2520Parameterization%252C%2520and%2520Performance%2520of%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D490%26epage%3D519%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">The
UniProt Consortium</span> <span> </span><span class="NLM_article-title">Uniprot:
The Universal Protein Knowledgebase</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The%0AUniProt+Consortium&title=Uniprot%3A%0AThe+Universal+Protein+Knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniprot%253A%250AThe%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly Predictive and Interpretable Models for PAMPA Permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+Predictive+and+Interpretable+Models+for+PAMPA+Permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0lixiWEsuIUF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520Predictive%2520and%2520Interpretable%2520Models%2520for%2520PAMPA%2520Permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">An Automated High-Throughput Metabolic Stability Assay Using an Integrated High-Resolution Accurate Mass Method and Automated Data Analysis Software</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1653</span>, <span class="refDoi"> DOI: 10.1124/dmd.116.072017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1124%2Fdmd.116.072017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27417180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1653&author=P.+Shahauthor=E.+Kernsauthor=D.-T.+Nguyenauthor=R.+S.+Obachauthor=A.+Q.+Wangauthor=A.+Zakharovauthor=J.+McKewauthor=A.+Simeonovauthor=C.+E.+C.+A.+Hopauthor=X.+Xu&title=An+Automated+High-Throughput+Metabolic+Stability+Assay+Using+an+Integrated+High-Resolution+Accurate+Mass+Method+and+Automated+Data+Analysis+Software&doi=10.1124%2Fdmd.116.072017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">An automated high-throughput metabolic stability assay using an integrated high-resolution accurate mass method and automated data analysis software</span></div><div class="casAuthors">Shah, Pranav; Kerns, Edward; Nguyen, Dac-Trung; Scott, Obach R.; Wang, Amy Q.; Zakharov, Alexey; McKew, John; Simeonov, Anton; Hop, Cornelis E. C. A.; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1653-1661</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Advancement of in silico tools would be enabled by the availability of data for metabolic reaction rates and intrinsic clearance (CLint) of a diverse compd. structure data set by specific metabolic enzymes.  Our goal is to measure CLint for a large set of compds. with each major human cytochrome P 450 (P 450) isoenzyme.  To achieve our goal, it is of utmost importance to develop an automated, robust, sensitive, high-throughput metabolic stability assay that can efficiently handle a large vol. of compd. sets.  The substrate depletion method [in vitro half-life (t1/2) method] was chosen to det. CLint.  The assay (384-well format) consisted of three parts: (1) a robotic system for incubation and sample cleanup; (2) two different integrated, ultraperformance liq. chromatog./ mass spectrometry (UPLC/MS) platforms to det. the percent remaining of parent compd., and (3) an automated data anal. system.  The CYP3A4 assay was evaluated using two long t1/2 compds., carbamazepine and antipyrine (t1/2 > 30 min); one moderate t1/2 compd., ketoconazole (10 < t1/2 < 30 min); and two short t1/2 compds., loperamide and buspirone (t < 10 min).  Interday and intraday precision and accuracy of the assay were within acceptable range (∼12%) for the linear range obsd.  Using this assay, CYP3A4 CLint and t1/2 values for more than 3000 compds. were measured.  This high-throughput, automated, and robust assay allows for rapid metabolic stability screening of large compd. sets and enables advanced computational modeling for individual human P 450 isoenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos6dcRqLvacbVg90H21EOLACvtfcHk0lixiWEsuIUF1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK&md5=92b4f23376d1cffa70c6fdc209b5569c</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.072017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.072017%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DP.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DD.-T.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DMcKew%26aufirst%3DJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DAn%2520Automated%2520High-Throughput%2520Metabolic%2520Stability%2520Assay%2520Using%2520an%2520Integrated%2520High-Resolution%2520Accurate%2520Mass%2520Method%2520and%2520Automated%2520Data%2520Analysis%2520Software%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1653%26doi%3D10.1124%2Fdmd.116.072017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xiaopeng Peng, Jingxuan Chen, Ling Li, Zhiqiang Sun, Jin Liu, Yichang Ren, Junli Huang, <span class="NLM_string-name hlFld-ContribAuthor">Jianjun Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient Synthesis and Bioevaluation of Novel Dual Tubulin/Histone Deacetylase 3 Inhibitors as Potential Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8447-8473. <a href="https://doi.org/10.1021/acs.jmedchem.1c00413" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00413</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00413&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00413%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEfficient%252BSynthesis%252Band%252BBioevaluation%252Bof%252BNovel%252BDual%252BTubulin%25252FHistone%252BDeacetylase%252B3%252BInhibitors%252Bas%252BPotential%252BAnticancer%252BAgents%26aulast%3DPeng%26aufirst%3DXiaopeng%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08032021%26date%3D07062021%26volume%3D64%26issue%3D12%26spage%3D8447%26epage%3D8473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shushu Hao, Jiazhi Yang, Peng Liu, Jing Xu, Chenchen Yang, <span class="NLM_string-name hlFld-ContribAuthor">Feng Li</span>. </span><span class="cited-content_cbyCitation_article-title">Linear-Organic-Polymer-Supported Iridium Complex as a Recyclable Auto-Tandem Catalyst for the Synthesis of Quinazolinones via Selective Hydration/Acceptorless Dehydrogenative Coupling from o-Aminobenzonitriles. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2021,</strong> <em>23 </em>
                                    (7)
                                     , 2553-2558. <a href="https://doi.org/10.1021/acs.orglett.1c00475" title="DOI URL">https://doi.org/10.1021/acs.orglett.1c00475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.1c00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.1c00475%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DLinear-Organic-Polymer-Supported%252BIridium%252BComplex%252Bas%252Ba%252BRecyclable%252BAuto-Tandem%252BCatalyst%252Bfor%252Bthe%252BSynthesis%252Bof%252BQuinazolinones%252Bvia%252BSelective%252BHydration%25252FAcceptorless%252BDehydrogenative%252BCoupling%252Bfrom%252Bo-Aminobenzonitriles%26aulast%3DHao%26aufirst%3DShushu%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D08022021%26date%3D17032021%26volume%3D23%26issue%3D7%26spage%3D2553%26epage%3D2558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xin-Hui Zhang, Qin-Ma, Hui-Pan Wu, Mussa Yussuf Khamis, Yi-Han Li, Li-Ying Ma, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A Review of Progress in Histone Deacetylase 6 Inhibitors Research: Structural Specificity and Functional Diversity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1362-1391. <a href="https://doi.org/10.1021/acs.jmedchem.0c01782" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01782</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01782%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BReview%252Bof%252BProgress%252Bin%252BHistone%252BDeacetylase%252B6%252BInhibitors%252BResearch%25253A%252BStructural%252BSpecificity%252Band%252BFunctional%252BDiversity%26aulast%3DZhang%26aufirst%3DXin-Hui%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D12102020%26date%3D01022021%26volume%3D64%26issue%3D3%26spage%3D1362%26epage%3D1391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dorjbal  Dorjsuren</span>, <span class="hlFld-ContribAuthor ">Richard T.  Eastman</span>, <span class="hlFld-ContribAuthor ">Kathryn J.  Wicht</span>, <span class="hlFld-ContribAuthor ">Daniel  Jansen</span>, <span class="hlFld-ContribAuthor ">Daniel C.  Talley</span>, <span class="hlFld-ContribAuthor ">Benjamin A.  Sigmon</span>, <span class="hlFld-ContribAuthor ">Alexey V.  Zakharov</span>, <span class="hlFld-ContribAuthor ">Norma  Roncal</span>, <span class="hlFld-ContribAuthor ">Andrew T.  Girvin</span>, <span class="hlFld-ContribAuthor ">Yevgeniya  Antonova-Koch</span>, <span class="hlFld-ContribAuthor ">Paul M.  Will</span>, <span class="hlFld-ContribAuthor ">Pranav  Shah</span>, <span class="hlFld-ContribAuthor ">Hongmao  Sun</span>, <span class="hlFld-ContribAuthor ">Carleen  Klumpp-Thomas</span>, <span class="hlFld-ContribAuthor ">Sachel  Mok</span>, <span class="hlFld-ContribAuthor ">Tomas  Yeo</span>, <span class="hlFld-ContribAuthor ">Stephan  Meister</span>, <span class="hlFld-ContribAuthor ">Juan Jose  Marugan</span>, <span class="hlFld-ContribAuthor ">Leila S.  Ross</span>, <span class="hlFld-ContribAuthor ">Xin  Xu</span>, <span class="hlFld-ContribAuthor ">David J.  Maloney</span>, <span class="hlFld-ContribAuthor ">Ajit  Jadhav</span>, <span class="hlFld-ContribAuthor ">Bryan T.  Mott</span>, <span class="hlFld-ContribAuthor ">Richard J.  Sciotti</span>, <span class="hlFld-ContribAuthor ">Elizabeth A.  Winzeler</span>, <span class="hlFld-ContribAuthor ">Norman C.  Waters</span>, <span class="hlFld-ContribAuthor ">Robert F.  Campbell</span>, <span class="hlFld-ContribAuthor ">Wenwei  Huang</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>, <span class="hlFld-ContribAuthor ">David A.  Fidock</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoprotective antimalarials identified through quantitative high-throughput screening of Plasmodium blood and liver stage parasites. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2021,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-021-81486-z" title="DOI URL">https://doi.org/10.1038/s41598-021-81486-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-021-81486-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-021-81486-z%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DChemoprotective%252Bantimalarials%252Bidentified%252Bthrough%252Bquantitative%252Bhigh-throughput%252Bscreening%252Bof%252BPlasmodium%252Bblood%252Band%252Bliver%252Bstage%252Bparasites%26aulast%3DDorjsuren%26aufirst%3DDorjbal%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunal  Nepali</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in epigenetic cancer therapeutics: clinical advancement and emerging trends. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Science</span><span> <strong>2021,</strong> <em>28 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12929-021-00721-x" title="DOI URL">https://doi.org/10.1186/s12929-021-00721-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12929-021-00721-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12929-021-00721-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Science%26atitle%3DRecent%252Bdevelopments%252Bin%252Bepigenetic%252Bcancer%252Btherapeutics%25253A%252Bclinical%252Badvancement%252Band%252Bemerging%252Btrends%26aulast%3DNepali%26aufirst%3DKunal%26date%3D2021%26date%3D2021%26volume%3D28%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Heba S.A.  ElZahabi</span>, <span class="hlFld-ContribAuthor ">Mohamed S.  Nafie</span>, <span class="hlFld-ContribAuthor ">Dina  Osman</span>, <span class="hlFld-ContribAuthor ">Nehal H.  Elghazawy</span>, <span class="hlFld-ContribAuthor ">Dalia H.  Soliman</span>, <span class="hlFld-ContribAuthor ">Abdelghany Ali H.  EL-Helby</span>, <span class="hlFld-ContribAuthor ">Reem K.  Arafa</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and evaluation of new quinazolin-4-one derivatives as apoptotic enhancers and autophagy inhibitors with potent antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113609. <a href="https://doi.org/10.1016/j.ejmech.2021.113609" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113609</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113609&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113609%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252Bnew%252Bquinazolin-4-one%252Bderivatives%252Bas%252Bapoptotic%252Benhancers%252Band%252Bautophagy%252Binhibitors%252Bwith%252Bpotent%252Bantitumor%252Bactivity%26aulast%3DElZahabi%26aufirst%3DHeba%2BS.A.%26date%3D2021%26volume%3D222%26spage%3D113609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ma  Su</span>, <span class="hlFld-ContribAuthor ">Xingyu  Gong</span>, <span class="hlFld-ContribAuthor ">Feng  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">An update on the emerging approaches for histone deacetylase (HDAC) inhibitor drug discovery and future perspectives. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 745-761. <a href="https://doi.org/10.1080/17460441.2021.1877656" title="DOI URL">https://doi.org/10.1080/17460441.2021.1877656</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1877656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1877656%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAn%252Bupdate%252Bon%252Bthe%252Bemerging%252Bapproaches%252Bfor%252Bhistone%252Bdeacetylase%252B%252528HDAC%252529%252Binhibitor%252Bdrug%252Bdiscovery%252Band%252Bfuture%252Bperspectives%26aulast%3DSu%26aufirst%3DMa%26date%3D2021%26date%3D2021%26volume%3D16%26issue%3D7%26spage%3D745%26epage%3D761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arshdeep  Singh</span>, <span class="hlFld-ContribAuthor ">Ting-Yu  Chang</span>, <span class="hlFld-ContribAuthor ">Navdeep  Kaur</span>, <span class="hlFld-ContribAuthor ">Kai-Cheng  Hsu</span>, <span class="hlFld-ContribAuthor ">Yun  Yen</span>, <span class="hlFld-ContribAuthor ">Tony Eight  Lin</span>, <span class="hlFld-ContribAuthor ">Mei-Jung  Lai</span>, <span class="hlFld-ContribAuthor ">Sung-Bau  Lee</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">CAP rigidification of MS-275 and chidamide leads to enhanced antiproliferative effects mediated through HDAC1, 2 and tubulin polymerization inhibition. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>215 </em>, 113169. <a href="https://doi.org/10.1016/j.ejmech.2021.113169" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113169</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113169%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCAP%252Brigidification%252Bof%252BMS-275%252Band%252Bchidamide%252Bleads%252Bto%252Benhanced%252Bantiproliferative%252Beffects%252Bmediated%252Bthrough%252BHDAC1%25252C%252B2%252Band%252Btubulin%252Bpolymerization%252Binhibition%26aulast%3DSingh%26aufirst%3DArshdeep%26date%3D2021%26volume%3D215%26spage%3D113169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyu  Li</span>, <span class="hlFld-ContribAuthor ">Jennifer E.  Golden</span>. </span><span class="cited-content_cbyCitation_article-title">Construction of
              N
              ‐Boc‐2‐Alkylaminoquinazolin‐4(3
              H
              )‐Ones via a Three‐Component, One‐Pot Protocol Mediated by Copper(II) Chloride that Spares Enantiomeric Purity. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2021,</strong> <em>363 </em>
                                    (6)
                                     , 1638-1645. <a href="https://doi.org/10.1002/adsc.202001279" title="DOI URL">https://doi.org/10.1002/adsc.202001279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.202001279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.202001279%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DConstruction%252Bof%252BN%252B%2525E2%252580%252590Boc%2525E2%252580%2525902%2525E2%252580%252590Alkylaminoquinazolin%2525E2%252580%2525904%2525283%252BH%252B%252529%2525E2%252580%252590Ones%252Bvia%252Ba%252BThree%2525E2%252580%252590Component%25252C%252BOne%2525E2%252580%252590Pot%252BProtocol%252BMediated%252Bby%252BCopper%252528II%252529%252BChloride%252Bthat%252BSpares%252BEnantiomeric%252BPurity%26aulast%3DLi%26aufirst%3DXiaoyu%26date%3D2021%26date%3D2021%26volume%3D363%26issue%3D6%26spage%3D1638%26epage%3D1645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel A.  Rodrigues</span>, <span class="hlFld-ContribAuthor ">Pedro S. M.  Pinheiro</span>, <span class="hlFld-ContribAuthor ">Carlos A. M.  Fraga</span>. </span><span class="cited-content_cbyCitation_article-title">Multitarget Inhibition of Histone Deacetylase (HDAC) and Phosphatidylinositol‐3‐kinase (PI3K): Current and Future Prospects. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2021,</strong> <em>16 </em>
                                    (3)
                                     , 448-457. <a href="https://doi.org/10.1002/cmdc.202000643" title="DOI URL">https://doi.org/10.1002/cmdc.202000643</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000643%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DMultitarget%252BInhibition%252Bof%252BHistone%252BDeacetylase%252B%252528HDAC%252529%252Band%252BPhosphatidylinositol%2525E2%252580%2525903%2525E2%252580%252590kinase%252B%252528PI3K%252529%25253A%252BCurrent%252Band%252BFuture%252BProspects%26aulast%3DRodrigues%26aufirst%3DDaniel%2BA.%26date%3D2021%26date%3D2020%26volume%3D16%26issue%3D3%26spage%3D448%26epage%3D457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amr K.A.  Bass</span>, <span class="hlFld-ContribAuthor ">Mona S.  El-Zoghbi</span>, <span class="hlFld-ContribAuthor ">El-Shimaa M.  Nageeb</span>, <span class="hlFld-ContribAuthor ">Mamdouh F.A.  Mohamed</span>, <span class="hlFld-ContribAuthor ">Mohamed  Badr</span>, <span class="hlFld-ContribAuthor ">Gamal El-Din A.  Abuo-Rahma</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive review for anticancer hybridized multitargeting HDAC inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>209 </em>, 112904. <a href="https://doi.org/10.1016/j.ejmech.2020.112904" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112904</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112904%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComprehensive%252Breview%252Bfor%252Banticancer%252Bhybridized%252Bmultitargeting%252BHDAC%252Binhibitors%26aulast%3DBass%26aufirst%3DAmr%2BK.A.%26date%3D2021%26volume%3D209%26spage%3D112904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuan  Gao</span>, <span class="hlFld-ContribAuthor ">Hang  Zhang</span>, <span class="hlFld-ContribAuthor ">Frédéric  Lirussi</span>, <span class="hlFld-ContribAuthor ">Carmen  Garrido</span>, <span class="hlFld-ContribAuthor ">Xiang-Yang  Ye</span>, <span class="hlFld-ContribAuthor ">Tian  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Dual inhibitors of histone deacetylases and other cancer-related targets: A pharmacological perspective. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2020,</strong> <em>182 </em>, 114224. <a href="https://doi.org/10.1016/j.bcp.2020.114224" title="DOI URL">https://doi.org/10.1016/j.bcp.2020.114224</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2020.114224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2020.114224%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DDual%252Binhibitors%252Bof%252Bhistone%252Bdeacetylases%252Band%252Bother%252Bcancer-related%252Btargets%25253A%252BA%252Bpharmacological%252Bperspective%26aulast%3DGao%26aufirst%3DYuan%26date%3D2020%26volume%3D182%26spage%3D114224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaopeng  Peng</span>, <span class="hlFld-ContribAuthor ">Zhiqiang  Sun</span>, <span class="hlFld-ContribAuthor ">Peihua  Kuang</span>, <span class="hlFld-ContribAuthor ">Jianjun  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress on HDAC inhibitors with dual targeting capabilities for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>208 </em>, 112831. <a href="https://doi.org/10.1016/j.ejmech.2020.112831" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112831</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112831%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bprogress%252Bon%252BHDAC%252Binhibitors%252Bwith%252Bdual%252Btargeting%252Bcapabilities%252Bfor%252Bcancer%252Btreatment%26aulast%3DPeng%26aufirst%3DXiaopeng%26date%3D2020%26volume%3D208%26spage%3D112831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bernhard  Biersack</span>, <span class="hlFld-ContribAuthor ">Sibel  Polat</span>, <span class="hlFld-ContribAuthor ">Michael  Höpfner</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer properties of chimeric HDAC and kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Seminars in Cancer Biology</span><span> <strong>2020,</strong> <em>184 </em><a href="https://doi.org/10.1016/j.semcancer.2020.11.005" title="DOI URL">https://doi.org/10.1016/j.semcancer.2020.11.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.semcancer.2020.11.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.semcancer.2020.11.005%26sid%3Dliteratum%253Aachs%26jtitle%3DSeminars%2520in%2520Cancer%2520Biology%26atitle%3DAnticancer%252Bproperties%252Bof%252Bchimeric%252BHDAC%252Band%252Bkinase%252Binhibitors%26aulast%3DBiersack%26aufirst%3DBernhard%26date%3D2020%26volume%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yufeng  Xiao</span>, <span class="hlFld-ContribAuthor ">Xuan  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in small molecular modulators targeting histone deacetylase 6. </span><span class="cited-content_cbyCitation_journal-name">Future Drug Discovery</span><span> <strong>2020,</strong> <em>2 </em>
                                    (4)
                                     , FDD53. <a href="https://doi.org/10.4155/fdd-2020-0023" title="DOI URL">https://doi.org/10.4155/fdd-2020-0023</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fdd-2020-0023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffdd-2020-0023%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Drug%2520Discovery%26atitle%3DRecent%252Badvances%252Bin%252Bsmall%252Bmolecular%252Bmodulators%252Btargeting%252Bhistone%252Bdeacetylase%252B6%26aulast%3DXiao%26aufirst%3DYufeng%26date%3D2020%26volume%3D2%26issue%3D4%26spage%3DFDD53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0026.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0026.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0001.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. FDA Approved PI3K inhibitor drugs for cancer (CLL = chronic lymphocytic leukemia; FL = follicular lymphoma; SLL = small lymphocytic lymphoma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0002.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Selected approved HDAC inhibitor drugs for cancer. (B) Chemical structure of dual PI3K/HDAC inhibitor, Fimepinostat (CTCL = cutaneous T-cell lymphoma; PTCL = peripheral T-cell lymphoma; DLBCL = diffuse large B-cell lymphoma).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0003.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Chemical structure of the PI3Kδ inhibitor, Idelalisib (<b>1</b>). (B) Crystal structures of Idelalisib (<b>1</b>) [top view] bound to PI3Kδ (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0">4XE0</a>) with protein surface (blue) and without protein surface, respectively. (C) Chemical structure of the HDAC inhibitor, Vorinostat (<b>2</b>). (D)  Crystal structures of Vorinostat (<b>2</b>) bound to HDAC2 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ">4LXZ</a>) with protein surface (blue) and without protein surface, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0004.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design strategy for dual PI3K/HDAC inhibitor types <b>3</b> and <b>4</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0014.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of C-4 Alkyl/Amino-alkyl Substituted Quinazolines (<b>10a</b>,<b>b</b> and <b>14a</b>,<b>b</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>6a</b>, <b>6b</b> (1.2 equiv), [PdCl(allyl)]<sub>2</sub> (0.05 equiv), P<sup><i>t</i></sup>Bu<sub>3</sub>HBF<sub>4</sub> (0.20 equiv), 1,4-dioxane, 16 h, rt; (b) (i) 10 wt % Pd/C, H<sub>2</sub> balloon, EtOAc, 20 h, rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (iii) <b>8</b> (2.0 equiv), triethyamine (4.0 equiv), EtOH, 1 h, MW, 100 °C; (c) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt; (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 20 h, rt (<i>For</i><b>9a</b><i>, only c(ii) and c(iii) were used</i>); (d) <b>11a</b>,<b>b</b> (2.0 equiv), triethyamine (4.0 equiv), EtOH, 1 h, 100 °C; (e) <b>8</b> (2.0 equiv), triethyamine (4.0 equiv), EtOH, 1 h, 100 °C.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0015.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of C-5 Alkyl/Aryl Substituted Quinazolinones (<b>19a</b>–<b>c</b> and <b>24a</b>–<b>c</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>16a</b>, <b>16b</b>, <b>16c</b> (1.2 equiv), [PdCl(allyl)]<sub>2</sub> (0.05 equiv), P<i><sup>t</sup></i>Bu<sub>3</sub>HBF<sub>4</sub> (0.20 equiv), 1,4-dioxane, 16 h, rt; (b) (i) 10 wt % Pd/C, H<sub>2</sub> balloon, EtOAc, 20 h, rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (iii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, MW, 100 °C; (c) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt, (<i>For</i><b>17a</b><i>, only c(ii) and c(iii)</i> were used); (d) (i) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (e) boronic acid/ester <b>20a</b>,<b>b</b> (1.2 equiv), [XPhos Pd(crotyl)Cl] (0.05 equiv), K<sub>3</sub>PO<sub>4</sub> (3.0 equiv), 1,4-dioxane/water, 1–2 h MW, 100 °C; (f) [Bpin]<sub>2</sub> (1.2 equiv), [Pd(dppf)Cl<sub>2</sub>] (0.05 equiv), KOAc (3.0 equiv) 1,4-dioxane, 16 h, 100 °C; (g) <b>20c</b> (0.85 equiv), [XPhos Pd(crotyl)Cl] (0.05 equiv), K<sub>3</sub>PO<sub>4</sub> (3.0 equiv), 1,4-dioxane/water, 10 h, 100 °C; (h) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, 100 °C.</p></p></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0016.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of C-5 Amino-alkyl/Aryl Substituted Quinazolinones (<b>28a</b>–<b>i</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>25a</b>–<b>h</b> (1.3 equiv), [Xantphos Palladacycle Gen. 4] (0.03 equiv), Cs<sub>2</sub>CO<sub>3</sub> (3.0 equiv), toluene or 1,4-dioxane, 16 h, rt; (b) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, MW, 100 °C; (c) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (d) (i) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (e) (i) LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt, (ii) <i>N</i>-methylhydroxylamine hydrochloride (1.5 equiv), HATU (1.5), DIPEA (2.5 equiv), DMF, rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0017.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of C-5 Alkyl-amino Substituted Quinazolinones (<b>33</b>, <b>37</b>, <b>40</b>, and <b>43</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub> (0.05 equiv), Zn(CN)<sub>2</sub> (1.2 equiv), DMF, 100 °C, 16 h; (b) Raney Ni, H<sub>2</sub>, MeOH, 20 h; (c) <b>31</b> (2.0 equiv) DIPEA (4.0 equiv), DMF, 180 °C, MW, 30 min; (d) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>8</b> (1.5–2.5 equiv), triethyamine (3.0–5.0 equiv), EtOH, 4 h, MW, 100 °C; (e) (i) LiOH·H<sub>2</sub>O (2.0 equiv), MeOH/H<sub>2</sub>O, 10 h, rt, (ii) NH<sub>2</sub>OTHP (3.1 equiv), <i>N</i>-methyl morpholine (3.0 equiv) 3-(((ethylimino)methylene)amino)-<i>N</i>,<i>N</i>-dimethylpropan-1-amine hydrochloride [EDC·HCl] (1.4 equiv), 1<i>H</i>-[1,2,3]triazolo[4,5-<i>b</i>]pyridin-1-ol [HOAT] (1.2 equiv), DMF, 16 h rt, (iii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM/MeOH, 20 h, rt; (f) <b>35</b> (1.5 equiv), HATU (1.2 equiv), DIPEA (3.0 equiv), DMF, 16 h, rt; (g) <b>38</b> (1 equiv), cat. AcOH, DCE, 2 h, then NaBH(OAC)<sub>3</sub> (3.0 equiv), 2 h; (h) <b>41</b> (2.0 equiv), triethylamine (4.0 equiv), EtOH, 100 °C, MW, 3 h.</p></p></figure><figure data-id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0018.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of C-5 Substituted Quinazolinones with Pyrimidine Hinge Binding Groups (<b>46a</b>–<b>d</b> and <b>48a</b>–<b>o</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) 10 wt % Pd/C, H<sub>2</sub> balloon, EtOAc, 20 h, rt, (ii) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (iii) <b>44</b> (1.5–2.0 equiv), DIPEA (3.0–4.0 equiv), <i>n</i>-butanol, 2–3 h, MW, 130 °C, 38–88% (3 steps); (b) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt; 36–82%.</p></p></figure><figure data-id="sch6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0019.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of C-5 Alkyl-amino Substituted Quinazolinones (<b>51</b> and <b>54</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) Raney Ni, H<sub>2</sub>, MeOH, 20 h; (b) <b>35</b> (1.5 equiv), HATU (1.2 equiv), DIPEA (3.0 equiv), DMF, 16 h; (c) (i) CF<sub>3</sub>CO<sub>2</sub>H (20.0 equiv), DCM, 3 h, rt, (ii) <b>44a</b> (1.5 equiv), DIPEA (3.0 equiv), <i>n</i>-butanol, 2–5 h, MW, 130–150 °C; (d) aq 50 wt % NH<sub>2</sub>OH (30.0 equiv), LiOH·H<sub>2</sub>O (1.1 equiv), MeOH/H<sub>2</sub>O, 20 h, 0 °C to rt; (e) <b>41</b> (1.5 equiv), DIPEA (3.0 equiv), <i>n</i>-butanol, 2 h, MW, 130 °C.</p></p></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Hydrogen bonding rationale for HDAC activity comparison in <b>28f</b> and <b>28g</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Compound <b>19b</b> bound to HDAC6 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU">5EDU</a>.pdb). (B) Variation in amino acid residues at positions 567 and 568 across different HDACs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0007.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) Kinase selectivity profile of compound <b>48c</b> (1 μM concentration) against 412 kinases. (B) Selectivity data (% Inhibition at 1 μM) of compound <b>48c</b> against lipid kinases. Kinase profiling was done at Reaction Biology, Malvern, PA, USA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0008.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Cell viability assays were performed in duplicate with IC<sub>50</sub> values expressed as mean ± SD of two independent runs. (B) In vitro cell-kill activity of <b>48c</b>. (C) In vitro cell-kill activity of compounds <b>48n</b>, <b>48o</b>, <b>48c</b>, Idelalisib, SAHA, and CUDC-907 against MV411 cells. (D) PBMCs viability with <b>48c</b> treatment. (E) NIH3T3 cell viability upon treatment with compounds <b>48n</b>, <b>48o</b>, <b>48c</b>, Idelalisib, SAHA, and CUDC-907.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0009.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Necrosis rate (%) of <b>48n</b> at 5 μM concentration. (B) Necrosis rate (%) of <b>48o</b> at 5 μM concentration. (C) Necrosis rate (%) of <b>48c</b> at 5 μM concentration. (D) Necrosis rate (%) of <b>48c</b> at 5 μM concentration against primary AML blasts. (E) Necrosis rate (%) of compounds <b>48n</b>, <b>48o</b>, <b>48c</b>, Idelalisib, SAHA, and CUDC-907 against HEK293 cells. Necrosis assays were performed in duplicate and % necrosis values are expressed as mean ± SD of two independent runs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0010.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Inhibition of MOLM-14 colony formation by <b>48c</b> and <b>48o</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0011.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Immunoblot analysis for PI3K inhibition in MOLM-14 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0012.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. CETSA analysis. (A) <i>T</i><sub>agg</sub> measurement for HDAC6 in MV411 cells, (B) <i>T</i><sub>agg</sub> measurement for PI3Kδ in MV411 cells, (C) melt profile for HDAC6, (D) melt profile for PI3Kδ.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/medium/jm0c00193_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0013.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Plasma concentration vs time profiles for <b>48c</b> in female Balb/c mice via: (A) iv (3 mg/kg) and po (10 mg/kg) dosing, and (B) ip (50 mg/kg and 150 mg/kg) dosing.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-8/acs.jmedchem.0c00193/20200416/images/large/jm0c00193_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00193&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i139">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58584" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58584" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 73 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamb, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span> <span> </span><span class="NLM_article-title">Why Is Cancer Drug Discovery So Difficult?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">115</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1038/nrd2155</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrd2155" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=17159925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFGlsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=115-120&author=A.+Kambauthor=S.+Weeauthor=C.+Lengauer&title=Why+Is+Cancer+Drug+Discovery+So+Difficult%3F&doi=10.1038%2Fnrd2155"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Why is cancer drug discovery so difficult?</span></div><div class="casAuthors">Kamb, Alexander; Wee, Susan; Lengauer, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">115-120</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Thirty-five years after the 'war on cancer' was declared, the discovery of anticancer drugs remains a highly challenging endeavor.  Here, we consider the factors responsible, such as tumor heterogeneity, and suggest strategies to improve the chances of short-term success in the development of novel anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYsRgFvOR6O7Vg90H21EOLACvtfcHk0lg4cFgIuEAsYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFGlsrw%253D&md5=0ee9e19541b8f3f0ed495b3de44b3ad3</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd2155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2155%26sid%3Dliteratum%253Aachs%26aulast%3DKamb%26aufirst%3DA.%26aulast%3DWee%26aufirst%3DS.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DWhy%2520Is%2520Cancer%2520Drug%2520Discovery%2520So%2520Difficult%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D115%26epage%3D120%26doi%3D10.1038%2Fnrd2155" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of Cancer: The Next Generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+Cancer%3A+The+Next+Generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lg6DaZzyBB34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520Cancer%253A%2520The%2520Next%2520Generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anighoro, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology: Challenges and Opportunities in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">7874</span>– <span class="NLM_lpage">7887</span>, <span class="refDoi"> DOI: 10.1021/jm5006463</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5006463" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=7874-7887&author=A.+Anighoroauthor=J.+Bajorathauthor=G.+Rastelli&title=Polypharmacology%3A+Challenges+and+Opportunities+in+Drug+Discovery&doi=10.1021%2Fjm5006463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology: Challenges and Opportunities in Drug Discovery</span></div><div class="casAuthors">Anighoro, Andrew; Bajorath, Jurgen; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7874-7887</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  At present, the legendary magic bullet, i.e., a drug with high potency and selectivity toward a specific biol. target, shares the spotlight with an emerging and alternative polypharmacol. approach.  Polypharmacol. suggests that more effective drugs can be developed by specifically modulating multiple targets.  It is generally thought that complex diseases such as cancer and central nervous system diseases may require complex therapeutic approaches.  In this respect, a drug that "hits" multiple sensitive nodes belonging to a network of interacting targets offers the potential for higher efficacy and may limit drawbacks generally arising from the use of a single-target drug or a combination of multiple drugs.  In this review, we will compare advantages and disadvantages of multitarget vs. combination therapies, discuss potential drug promiscuity arising from off-target effects, comment on drug repurposing, and introduce approaches to the computational design of multitarget drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFOTyEu80AZrVg90H21EOLACvtfcHk0lg6DaZzyBB34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVShsL3I&md5=5ab36c961a84462d90bc25130d90a281</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm5006463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5006463%26sid%3Dliteratum%253Aachs%26aulast%3DAnighoro%26aufirst%3DA.%26aulast%3DBajorath%26aufirst%3DJ.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DPolypharmacology%253A%2520Challenges%2520and%2520Opportunities%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D7874%26epage%3D7887%26doi%3D10.1021%2Fjm5006463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">De Novo Design of Multitarget Ligands with an Iterative Fragment-Growing Strategy</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1235</span>– <span class="NLM_lpage">1241</span>, <span class="refDoi"> DOI: 10.1021/ci500021v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500021v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjvVOnsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=1235-1241&author=E.+Shangauthor=Y.+Yuanauthor=X.+Chenauthor=Y.+Liuauthor=J.+Peiauthor=L.+Lai&title=De+Novo+Design+of+Multitarget+Ligands+with+an+Iterative+Fragment-Growing+Strategy&doi=10.1021%2Fci500021v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">De Novo Design of Multitarget Ligands with an Iterative Fragment-Growing Strategy</span></div><div class="casAuthors">Shang, Erchang; Yuan, Yaxia; Chen, Xinyi; Liu, Ying; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1235-1241</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of multitarget drugs has recently attracted much attention.  Most of the reported multitarget ligands have been serendipitous discoveries.  Although a few methods have been developed for rational multitarget drug discovery, there is a lack of elegant methods for de novo multitarget drug design and optimization, esp. for multiple targets with large differences in their binding sites.  In this paper, we report the first de novo multitarget ligand design method, with an iterative fragment-growing strategy.  Using this method, dual-target inhibitors for COX-2 and LTA4H were designed, with the most potent one inhibiting PGE2 and LTB4 prodn. in the human whole blood assay with IC50 values of 7.0 and 7.1 μM, resp.  Our strategy is generally applicable in rational and efficient multitarget drug design, esp. for the design of highly integrated inhibitors for proteins with dissimilar binding pockets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYeSpPS20iL7Vg90H21EOLACvtfcHk0lg6DaZzyBB34g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjvVOnsbY%253D&md5=964c8053de5457730dc369cf8f8ca6a7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fci500021v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500021v%26sid%3Dliteratum%253Aachs%26aulast%3DShang%26aufirst%3DE.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DDe%2520Novo%2520Design%2520of%2520Multitarget%2520Ligands%2520with%2520an%2520Iterative%2520Fragment-Growing%2520Strategy%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D1235%26epage%3D1241%26doi%3D10.1021%2Fci500021v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morphy, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span></span> <span> </span><span class="NLM_article-title">Designed Multiple Ligands. An Emerging Drug Discovery Paradigm</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">6523</span>– <span class="NLM_lpage">6543</span>, <span class="refDoi"> DOI: 10.1021/jm058225d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm058225d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6523-6543&author=R.+Morphyauthor=Z.+Rankovic&title=Designed+Multiple+Ligands.+An+Emerging+Drug+Discovery+Paradigm&doi=10.1021%2Fjm058225d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Designed Multiple Ligands.</span></div><div class="casAuthors">Morphy, Richard; Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6523-6543</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The review discusses main areas in designed multiple ligands (DMLs) discovery that include 5HT transporter-based DMLs for depression, dopamine D2-receptor based DMLs for schizophrenia, histamine H1 receptor-based DMLs for allergies, DMLs targeting the angiotensin system for hypertension, nonsteroidal anti-inflammatory drugs, and PPAR-based DMLs for metabolic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgouWcjsiKgbVg90H21EOLACvtfcHk0litFWkbwrmCaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL&md5=00e3bf5a521fc91996ce95bb379a0b5c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fjm058225d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm058225d%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DDesigned%2520Multiple%2520Ligands.%2520An%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6523%26epage%3D6543%26doi%3D10.1021%2Fjm058225d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Boyle, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meegan, M. J.</span></span> <span> </span><span class="NLM_article-title">Designed Multiple Ligands for Cancer Therapy</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4722</span>– <span class="NLM_lpage">4737</span>, <span class="refDoi"> DOI: 10.2174/092986711797535344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.2174%2F092986711797535344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21919848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyhsLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=4722-4737&author=N.+M.+O%E2%80%99Boyleauthor=M.+J.+Meegan&title=Designed+Multiple+Ligands+for+Cancer+Therapy&doi=10.2174%2F092986711797535344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Designed multiple ligands for cancer therapy</span></div><div class="casAuthors">O'Boyle, N. M.; Meegan, M. J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">4722-4737</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The concept of a single chem. entity with desirable activity at more than one biol. target is an attractive one.  Increasingly, multiple complex biochem. pathways are implicated in a variety of diseases including cancer.  Successful treatment of these conditions often depends on pharmaceutical intervention at multiple pathways, with a combination of different drugs.  Designed multiple ligands (DMLs) are drugs which act at multiple biomol. targets.  Numerous terms have been used to describe such ligands, including multiple-target directed ligands, heterodimers, promiscuous drugs and pan-agonists.  However, although there are many reported examples of multiple-targeting anti-cancer agents, no review of these has been presented to date.  A huge variety of biol. signalling-pathways, proteins and enzymes are currently targeted and implicated in the pathogenesis of cancer.  This review will provide an overview of reported designed multiple ligands for cancer and an exploration of the advantages and drawbacks of such compds.  The review also provides brief commentaries on the biol. processes and proteins that are currently targeted in cancer therapy and the potential for dual or triple targeting of these with designed multiple ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS2P_3Ows00bVg90H21EOLACvtfcHk0litFWkbwrmCaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyhsLnF&md5=6d9f613ed668a18663fc2369a1041341</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F092986711797535344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711797535344%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%2BM.%26aulast%3DMeegan%26aufirst%3DM.%2BJ.%26atitle%3DDesigned%2520Multiple%2520Ligands%2520for%2520Cancer%2520Therapy%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2011%26volume%3D18%26spage%3D4722%26epage%3D4737%26doi%3D10.2174%2F092986711797535344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costantino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlocco, D.</span></span> <span> </span><span class="NLM_article-title">Designed Multiple Ligands: Basic Research vs Clinical Outcomes</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3353</span>– <span class="NLM_lpage">3387</span>, <span class="refDoi"> DOI: 10.2174/092986712801215883</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.2174%2F092986712801215883" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=22680630" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOhtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=3353-3387&author=L.+Costantinoauthor=D.+Barlocco&title=Designed+Multiple+Ligands%3A+Basic+Research+vs+Clinical+Outcomes&doi=10.2174%2F092986712801215883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Designed multiple ligands: basic research vs clinical outcomes</span></div><div class="casAuthors">Costantino, L.; Barlocco, D.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">3353-3387</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The clin. treatment of multifactorial pathologies (e.g. cancer, Alzheimer's disease, psychiatric disorders), is still a major challenge.  Many researches have been published dealing with the design of multiple ligands, able to act at the same time towards several targets relevant for a given pathol.  In the present review, the clin. results about these compds. have been reported, together with the design strategy adopted, in order to allow a crit. evaluation of the outcomes of these efforts.  What is emerging is that several effective design strategies of multitarget compds. are available, and the choice among these appears to be dependent on the therapeutic area considered.  However, at present, besides multitarget drugs discovered during optimization efforts by means of phenotypic assays, drug coadministrations or fixed dose formulations appear to be more useful therapeutic options than designed multiple ligands; this scenario will change when systems biol. will be able to select crit. targets, i.e. nodal proteins that should be inhibited in order to obtain a therapeutic action; at this point, the design of multiple ligands will allow a renaissance of medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobv_P2UM6L_7Vg90H21EOLACvtfcHk0litFWkbwrmCaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOhtrrN&md5=82123bbafc10a19c65a0bc0c5be72546</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F092986712801215883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712801215883%26sid%3Dliteratum%253Aachs%26aulast%3DCostantino%26aufirst%3DL.%26aulast%3DBarlocco%26aufirst%3DD.%26atitle%3DDesigned%2520Multiple%2520Ligands%253A%2520Basic%2520Research%2520vs%2520Clinical%2520Outcomes%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D3353%26epage%3D3387%26doi%3D10.2174%2F092986712801215883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Talevi, A.</span></span> <span> </span><span class="NLM_article-title">Multi-Target Pharmacology: Possibilities and Limitations of the ″Skeleton Key Approach″ from a Medicinal Chemist Perspective</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">205</span>, <span class="refDoi"> DOI: 10.3389/fphar.2015.00205</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3389%2Ffphar.2015.00205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=26441661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1amurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=205&author=A.+Talevi&title=Multi-Target+Pharmacology%3A+Possibilities+and+Limitations+of+the+%E2%80%B3Skeleton+Key+Approach%E2%80%B3+from+a+Medicinal+Chemist+Perspective&doi=10.3389%2Ffphar.2015.00205"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Multi-target pharmacology: possibilities and limitations of the "skeleton key approach" from a medicinal chemist perspective</span></div><div class="casAuthors">Talevi, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">205/1-205/7</span>CODEN:
                <span class="NLM_cas:coden">FPRHAU</span>;
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    
            (<span class="NLM_cas:orgname">Frontiers Media S.A.</span>)
        </div><div class="casAbstract">A review.  Multi-target drugs have raised considerable interest in the last decade owing to their advantages in the treatment of complex diseases and health conditions linked to drug resistance issues.  Prospective drug repositioning to treat comorbid conditions is an addnl., overlooked application of multi-target ligands.  While medicinal chemists usually rely on some version of the lock and key paradigm to design novel therapeutics, modern pharmacol. recognizes that the mid- and long-term effects of a given drug on a biol. system may depend not only on the specific ligand-target recognition events but also on the influence of the repeated administration of a drug on the cell gene signature.  The design of multi-target agents usually imposes challenging restrictions on the topol. or flexibility of the candidate drugs, which are briefly discussed in the present article.  Finally, computational strategies to approach the identification of novel multi-target agents are overviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSdZokyMjNSLVg90H21EOLACvtfcHk0litFWkbwrmCaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1amurnM&md5=169718a4644d962da33806772a9a33e3</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2015.00205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2015.00205%26sid%3Dliteratum%253Aachs%26aulast%3DTalevi%26aufirst%3DA.%26atitle%3DMulti-Target%2520Pharmacology%253A%2520Possibilities%2520and%2520Limitations%2520of%2520the%2520%25E2%2580%25B3Skeleton%2520Key%2520Approach%25E2%2580%25B3%2520from%2520a%2520Medicinal%2520Chemist%2520Perspective%26jtitle%3DFront.%2520Pharmacol.%26date%3D2015%26volume%3D6%26spage%3D205%26doi%3D10.3389%2Ffphar.2015.00205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, D. F.</span></span> <span> </span><span class="NLM_article-title">Designing Multi-Targeted Agents: An Emerging Anticancer Drug Discovery Paradigm</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.ejmech.2017.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28494256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsV2rtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2017&pages=195-211&author=R.+G.+Fuauthor=Y.+Sunauthor=W.+B.+Shengauthor=D.+F.+Liao&title=Designing+Multi-Targeted+Agents%3A+An+Emerging+Anticancer+Drug+Discovery+Paradigm&doi=10.1016%2Fj.ejmech.2017.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Designing multi-targeted agents: An emerging anticancer drug discovery paradigm</span></div><div class="casAuthors">Fu, Rong-geng; Sun, Yuan; Sheng, Wen-bing; Liao, Duan-fang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">195-211</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  The dominant paradigm in drug discovery is to design ligands with max. selectivity to act on individual drug targets.  With the target-based approach, many new chem. entities have been discovered, developed, and further approved as drugs.  However, there are a large no. of complex diseases such as cancer that cannot be effectively treated or cured only with one medicine to modulate the biol. function of a single target.  As simultaneous intervention of two (or multiple) cancer progression relevant targets has shown improved therapeutic efficacy, the innovation of multi-targeted drugs has become a promising and prevailing research topic and numerous multi-targeted anticancer agents are currently at various developmental stages.  However, most multi-pharmacophore scaffolds are usually discovered by serendipity or screening, while rational design by combining existing pharmacophore scaffolds remains an enormous challenge.  In this review, four types of multi-pharmacophore modes are discussed, and the examples from literature will be used to introduce attractive lead compds. with the capability of simultaneously interfering with different enzyme or signaling pathway of cancer progression, which will reveal the trends and insights to help the design of the next generation multi-targeted anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGjXR-zscrArVg90H21EOLACvtfcHk0liMK9maTYpVQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsV2rtL8%253D&md5=511561bd2cd186d5dafa905b7257bf32</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DR.%2BG.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DSheng%26aufirst%3DW.%2BB.%26aulast%3DLiao%26aufirst%3DD.%2BF.%26atitle%3DDesigning%2520Multi-Targeted%2520Agents%253A%2520An%2520Emerging%2520Anticancer%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D136%26spage%3D195%26epage%3D211%26doi%3D10.1016%2Fj.ejmech.2017.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastasio, T. J.</span></span> <span> </span><span class="NLM_article-title">Editorial: Computational and Experimental Approaches in Multi-Target Pharmacology</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">443</span>, <span class="refDoi"> DOI: 10.3389/fphar.2017.00443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3389%2Ffphar.2017.00443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28713280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC1cjotFaltg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=443&author=T.+J.+Anastasio&title=Editorial%3A+Computational+and+Experimental+Approaches+in+Multi-Target+Pharmacology&doi=10.3389%2Ffphar.2017.00443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Editorial: Computational and Experimental Approaches in Multi-target Pharmacology</span></div><div class="casAuthors">Anastasio Thomas J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">443</span>
        ISSN:<span class="NLM_cas:issn">1663-9812</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTo-NN9xtLw775oR8RchjrrfW6udTcc2ea8ala8p2Aoj7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cjotFaltg%253D%253D&md5=fb9b52602bd095218f8cc00bacbfea4b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2017.00443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2017.00443%26sid%3Dliteratum%253Aachs%26aulast%3DAnastasio%26aufirst%3DT.%2BJ.%26atitle%3DEditorial%253A%2520Computational%2520and%2520Experimental%2520Approaches%2520in%2520Multi-Target%2520Pharmacology%26jtitle%3DFront.%2520Pharmacol.%26date%3D2017%26volume%3D8%26spage%3D443%26doi%3D10.3389%2Ffphar.2017.00443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Proschak, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, D.</span></span> <span> </span><span class="NLM_article-title">Polypharmacology by Design: A Medicinal Chemist’s Perspective on Multitargeting Compounds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">444</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00760</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00760" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKms7fE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=420-444&author=E.+Proschakauthor=H.+Starkauthor=D.+Merk&title=Polypharmacology+by+Design%3A+A+Medicinal+Chemist%E2%80%99s+Perspective+on+Multitargeting+Compounds&doi=10.1021%2Facs.jmedchem.8b00760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Polypharmacology by Design: A Medicinal Chemist's Perspective on Multitargeting Compounds</span></div><div class="casAuthors">Proschak, Ewgenij; Stark, Holger; Merk, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">420-444</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multitargeting compds. comprising activity on more than a single biol. target have gained remarkable relevance in drug discovery owing to the complexity of multifactorial diseases such as cancer, inflammation, or the metabolic syndrome.  Polypharmacol. drug profiles can produce additive or synergistic effects while reducing side effects and significantly contribute to the high therapeutic success of indispensable drugs such as aspirin.  While their identification has long been the result of serendipity, medicinal chem. now tends to design polypharmacol.  Modern in vitro pharmacol. methods and chem. probes allow a systematic search for rational target combinations and recent innovations in computational technologies, crystallog., or fragment-based design equip multitarget compd. development with valuable tools.  In this Perspective, we analyze the relevance of multiple ligands in drug discovery and the versatile toolbox to design polypharmacol.  We conclude that despite some characteristic challenges remaining unresolved, designed polypharmacol. holds enormous potential to secure future therapeutic innovation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOEwJSxWw3vbVg90H21EOLACvtfcHk0liMK9maTYpVQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKms7fE&md5=6dc4b8e1c4777372396626140a302663</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00760%26sid%3Dliteratum%253Aachs%26aulast%3DProschak%26aufirst%3DE.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DMerk%26aufirst%3DD.%26atitle%3DPolypharmacology%2520by%2520Design%253A%2520A%2520Medicinal%2520Chemist%25E2%2580%2599s%2520Perspective%2520on%2520Multitargeting%2520Compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D420%26epage%3D444%26doi%3D10.1021%2Facs.jmedchem.8b00760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolognesi, M. L.</span></span> <span> </span><span class="NLM_article-title">Harnessing Polypharmacology with Medicinal Chemistry</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtVWjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=273-275&author=M.+L.+Bolognesi&title=Harnessing+Polypharmacology+with+Medicinal+Chemistry&doi=10.1021%2Facsmedchemlett.9b00039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing Polypharmacology with Medicinal Chemistry</span></div><div class="casAuthors">Bolognesi, Maria Laura</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">273-275</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polypharmacol. has expanded enormously over the last ten years, with several multitarget drugs (MTDs) already in the market.  This Viewpoint provides a basis for a discussion about the crit. need to develop MTDs in a more rationale and conscious way.  A checklist to maximize success in polypharmacol. is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4yXxJ_uU72bVg90H21EOLACvtfcHk0liMK9maTYpVQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtVWjt7g%253D&md5=28a8cc47ae2966a2ecd8a411adfb1ef5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00039%26sid%3Dliteratum%253Aachs%26aulast%3DBolognesi%26aufirst%3DM.%2BL.%26atitle%3DHarnessing%2520Polypharmacology%2520with%2520Medicinal%2520Chemistry%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D273%26epage%3D275%26doi%3D10.1021%2Facsmedchemlett.9b00039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Signalling through the Lipid Products of Phosphoinositide-3-OH Kinase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>387</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">676</span>, <span class="refDoi"> DOI: 10.1038/42648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2F42648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=9192891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADyaK2sXktVWgt7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=387&publication_year=1997&pages=673-676&author=A.+Tokerauthor=L.+C.+Cantley&title=Signalling+through+the+Lipid+Products+of+Phosphoinositide-3-OH+Kinase&doi=10.1038%2F42648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Signalling through the lipid products of phosphoinositide-3-OH kinase</span></div><div class="casAuthors">Toker, Alex; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">387</span>
        (<span class="NLM_cas:issue">6634</span>),
    <span class="NLM_cas:pages">673-676</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">A review, with 47 refs.  When a stimulatory agonist mol. binds at the exterior of the cell membrane, a second messenger transduces the signal to the interior of the cell.  Second messengers can be derived from phospholipids in the membrane by the action of the enzymes phospholipase C or phosphoinositide-3-OH kinase (PI(3)K).  PI(3)K is a key player in many cellular responses, including the movement of organelle membranes, shape alteration through rearrangement of cytoskeletal actin, transformation and chemotaxis.  But how PI(3)K mediates these responses is only now becoming clear.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbe14cub5uAbVg90H21EOLACvtfcHk0lgQunSjFqVFAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktVWgt7c%253D&md5=9c0d3fdb3a9a10027050c09ed3def89a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2F42648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F42648%26sid%3Dliteratum%253Aachs%26aulast%3DToker%26aufirst%3DA.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DSignalling%2520through%2520the%2520Lipid%2520Products%2520of%2520Phosphoinositide-3-OH%2520Kinase%26jtitle%3DNature%26date%3D1997%26volume%3D387%26spage%3D673%26epage%3D676%26doi%3D10.1038%2F42648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vanhaesebroeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillermet-Guibert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graupera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilanges, B.</span></span> <span> </span><span class="NLM_article-title">The Emerging Mechanisms of Isoform-Specific PI3K Signalling</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1038/nrm2882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrm2882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=20379207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVersrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=329-341&author=B.+Vanhaesebroeckauthor=J.+Guillermet-Guibertauthor=M.+Grauperaauthor=B.+Bilanges&title=The+Emerging+Mechanisms+of+Isoform-Specific+PI3K+Signalling&doi=10.1038%2Fnrm2882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging mechanisms of isoform-specific PI3K signaling</span></div><div class="casAuthors">Vanhaesebroeck, Bart; Guillermet-Guibert, Julie; Graupera, Mariona; Bilanges, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">329-341</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Phosphoinositide 3-kinases (PI3Ks) function early in intracellular signal transduction pathways and affect many biol. functions.  A further level of complexity derives from the existence of eight PI3K isoforms, which are divided into class I, class II and class III PI3Ks.  PI3K signaling has been implicated in metabolic control, immunity, angiogenesis and cardiovascular homeostasis, and is one of the most frequently deregulated pathways in cancer.  PI3K inhibitors have recently entered clin. trials in oncol.  A better understanding of how the different PI3K isoforms are regulated and control signaling could uncover their roles in pathol. and reveal in which disease contexts their blockade could be most beneficial.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1WFE8yaCnkrVg90H21EOLACvtfcHk0lgQunSjFqVFAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVersrc%253D&md5=d8507e3901857b68bb0a48dc7fea2b5b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fnrm2882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm2882%26sid%3Dliteratum%253Aachs%26aulast%3DVanhaesebroeck%26aufirst%3DB.%26aulast%3DGuillermet-Guibert%26aufirst%3DJ.%26aulast%3DGraupera%26aufirst%3DM.%26aulast%3DBilanges%26aufirst%3DB.%26atitle%3DThe%2520Emerging%2520Mechanisms%2520of%2520Isoform-Specific%2520PI3K%2520Signalling%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2010%26volume%3D11%26spage%3D329%26epage%3D341%26doi%3D10.1038%2Fnrm2882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span> <span> </span><span class="NLM_article-title">Targeting the Phosphoinositide 3-Kinase Pathway in Cancer</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">627</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd2926</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrd2926" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=19644473" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1Wgtb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=627-644&author=P.+Liuauthor=H.+Chengauthor=T.+M.+Robertsauthor=J.+J.+Zhao&title=Targeting+the+Phosphoinositide+3-Kinase+Pathway+in+Cancer&doi=10.1038%2Fnrd2926"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the phosphoinositide 3-kinase pathway in cancer</span></div><div class="casAuthors">Liu, Pixu; Cheng, Hailing; Roberts, Thomas M.; Zhao, Jean J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">627-644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The phosphoinositide 3-kinase (PI3K) pathway is a key signal transduction system that links oncogenes and multiple receptor classes to many essential cellular functions, and is perhaps the most commonly activated signalling pathway in human cancer.  This pathway therefore presents both an opportunity and a challenge for cancer therapy.  Even as inhibitors that target PI3K isoforms and other major nodes in the pathway, including AKT and mammalian target of rapamycin (mTOR), reach clin. trials, major issues remain.  Here, we highlight recent progress that has been made in our understanding of the PI3K pathway and discuss the potential of and challenges for the development of therapeutic agents that target this pathway in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUk9T-ZoNDtLVg90H21EOLACvtfcHk0lgQunSjFqVFAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1Wgtb4%253D&md5=fdc94abd15178a9dbe7c56dafbfb7787</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd2926&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2926%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26atitle%3DTargeting%2520the%2520Phosphoinositide%25203-Kinase%2520Pathway%2520in%2520Cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D627%26epage%3D644%26doi%3D10.1038%2Fnrd2926" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fruman, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rommel, C.</span></span> <span> </span><span class="NLM_article-title">PI3K and Cancer: Lessons, Challenges and Opportunities</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">140</span>– <span class="NLM_lpage">156</span>, <span class="refDoi"> DOI: 10.1038/nrd4204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnrd4204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=24481312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1SnsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=140-156&author=D.+A.+Frumanauthor=C.+Rommel&title=PI3K+and+Cancer%3A+Lessons%2C+Challenges+and+Opportunities&doi=10.1038%2Fnrd4204"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">PI3K and cancer: lessons, challenges and opportunities</span></div><div class="casAuthors">Fruman, David A.; Rommel, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">140-156</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The central role of phosphoinositide 3-kinase (PI3K) activation in tumor cell biol. has prompted a sizeable effort to target PI3K and/or downstream kinases such as AKT and mammalian target of rapamycin (mTOR) in cancer.  However, emerging clin. data show limited single-agent activity of inhibitors targeting PI3K, AKT or mTOR at tolerated doses.  One exception is the response to PI3Kδ inhibitors in chronic lymphocytic leukemia, where a combination of cell-intrinsic and -extrinsic activities drive efficacy.  Here, we review key challenges and opportunities for the clin. development of inhibitors targeting the PI3K-AKT-mTOR pathway.  Through a greater focus on patient selection, increased understanding of immune modulation and strategic application of rational combinations, it should be possible to realize the potential of this promising class of targeted anticancer agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlD2rY54m9LbVg90H21EOLACvtfcHk0lhBPr3XnWRwyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1SnsLc%253D&md5=c4f6b4b9f9fcacd6ef55eb17d3a4f425</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd4204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4204%26sid%3Dliteratum%253Aachs%26aulast%3DFruman%26aufirst%3DD.%2BA.%26aulast%3DRommel%26aufirst%3DC.%26atitle%3DPI3K%2520and%2520Cancer%253A%2520Lessons%252C%2520Challenges%2520and%2520Opportunities%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D140%26epage%3D156%26doi%3D10.1038%2Fnrd4204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stark, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriskantharajah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessel, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okkenhaug, K.</span></span> <span> </span><span class="NLM_article-title">PI3K Inhibitors in Inflammation, Autoimmunity and Cancer</span>. <i>Curr. Opin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1016/j.coph.2015.05.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.coph.2015.05.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=26093105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWqs7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=82-91&author=A.+K.+Starkauthor=S.+Sriskantharajahauthor=E.+M.+Hesselauthor=K.+Okkenhaug&title=PI3K+Inhibitors+in+Inflammation%2C+Autoimmunity+and+Cancer&doi=10.1016%2Fj.coph.2015.05.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">PI3K inhibitors in inflammation, autoimmunity and cancer</span></div><div class="casAuthors">Stark, Anne-Katrien; Sriskantharajah, Srividya; Hessel, Edith M.; Okkenhaug, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">82-91</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The healthy immune system protects against infection and malignant transformation without causing significant damage to host tissues.  Immune dysregulation results in diverse pathologies including autoimmune disease, chronic inflammatory disorders, allergies as well as immune deficiencies and cancer.  Phosphoinositide 3-kinase (PI3K) signalling has been shown to be a key pathway in the regulation of the immune response and continues to be the focus of intense research.  In recent years we have gained detailed understanding of PI3K signalling, and saw the development of potent and highly selective small mol. inhibitors, of which several are currently in clin. trials for the treatment of immune-related disorders and cancer.  The role of PI3K signalling in the immune response has been the subject of detailed reviews; here we focus on relevant recent progress in pre-clin. and clin. development of PI3K inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNWPQZuO9fE7Vg90H21EOLACvtfcHk0lhBPr3XnWRwyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWqs7%252FM&md5=382eb159c8ec5e3a4bc14f71f54dd295</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2015.05.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2015.05.017%26sid%3Dliteratum%253Aachs%26aulast%3DStark%26aufirst%3DA.%2BK.%26aulast%3DSriskantharajah%26aufirst%3DS.%26aulast%3DHessel%26aufirst%3DE.%2BM.%26aulast%3DOkkenhaug%26aufirst%3DK.%26atitle%3DPI3K%2520Inhibitors%2520in%2520Inflammation%252C%2520Autoimmunity%2520and%2520Cancer%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D23%26spage%3D82%26epage%3D91%26doi%3D10.1016%2Fj.coph.2015.05.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Garces, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span> <span> </span><span class="NLM_article-title">Class 1 PI3K Clinical Candidates and Recent Inhibitor Design Strategies: A Medicinal Chemistry Perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">4815</span>– <span class="NLM_lpage">4850</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01492</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01492" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFOku7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=4815-4850&author=A.+E.+Garcesauthor=M.+J.+Stocks&title=Class+1+PI3K+Clinical+Candidates+and+Recent+Inhibitor+Design+Strategies%3A+A+Medicinal+Chemistry+Perspective&doi=10.1021%2Facs.jmedchem.8b01492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Class 1 PI3K Clinical Candidates and Recent Inhibitor Design Strategies: A Medicinal Chemistry Perspective</span></div><div class="casAuthors">Garces, Aimie E.; Stocks, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4815-4850</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphatidylinositol 3-kinases (PI3Ks) are a family of lipid kinases that phosphorylate the 3-OH of the inositol ring of phosphoinositides, and deregulation of this pathway has implications in many diseases.  The search for novel PI3K inhibitors has been at the forefront of academic and industrial medicinal chem. with over 600 medicinal chem.-based publications and patents appearing to date, leading to 38 clin. candidates and the launch of two drugs, idelalisib in 2014 and copanlisib in 2017.  This Perspective will discuss medicinal chem. design approaches to novel isoform-selective inhibitors through consideration of brief case histories of compds. that have progressed into clin. development or that have revealed new structural motifs in this highly competitive area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUu2mgndklHbVg90H21EOLACvtfcHk0lhBPr3XnWRwyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFOku7bK&md5=907395a39ed6fb683464ee92c6a2537f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01492%26sid%3Dliteratum%253Aachs%26aulast%3DGarces%26aufirst%3DA.%2BE.%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26atitle%3DClass%25201%2520PI3K%2520Clinical%2520Candidates%2520and%2520Recent%2520Inhibitor%2520Design%2520Strategies%253A%2520A%2520Medicinal%2520Chemistry%2520Perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D4815%26epage%3D4850%26doi%3D10.1021%2Facs.jmedchem.8b01492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span> <span> </span><span class="NLM_article-title">Targeting PI3K in Cancer: Mechanisms and Advances in Clinical Trials</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">26</span>, <span class="refDoi"> DOI: 10.1186/s12943-019-0954-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1186%2Fs12943-019-0954-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30782187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BB3cflt1Kluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=26&author=J.+Yangauthor=J.+Nieauthor=X.+Maauthor=Y.+Weiauthor=Y.+Pengauthor=X.+Wei&title=Targeting+PI3K+in+Cancer%3A+Mechanisms+and+Advances+in+Clinical+Trials&doi=10.1186%2Fs12943-019-0954-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting PI3K in cancer: mechanisms and advances in clinical trials</span></div><div class="casAuthors">Yang Jing; Nie Ji; Ma Xuelei; Wei Yuquan; Peng Yong; Wei Xiawei</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">26</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phosphatidylinositol-3-kinase (PI3K)/AKT/mammalian target of rapamycin (mTOR) signaling is one of the most important intracellular pathways, which can be considered as a master regulator for cancer.  Enormous efforts have been dedicated to the development of drugs targeting PI3K signaling, many of which are currently employed in clinical trials evaluation, and it is becoming increasingly clear that PI3K inhibitors are effective in inhibiting tumor progression.  PI3K inhibitors are subdivided into dual PI3K/mTOR inhibitors, pan-PI3K inhibitors and isoform-specific inhibitors.  In this review, we performed a critical review to summarize the role of the PI3K pathway in tumor development, recent PI3K inhibitors development based on clinical trials, and the mechanisms of resistance to PI3K inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQGm3fFMcHFSMScOqQTSkOffW6udTcc2earFOF-hkRwrrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cflt1Kluw%253D%253D&md5=4267e4df49544ebe11ab4255e69e5bbe</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1186%2Fs12943-019-0954-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-019-0954-x%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DNie%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DX.%26atitle%3DTargeting%2520PI3K%2520in%2520Cancer%253A%2520Mechanisms%2520and%2520Advances%2520in%2520Clinical%2520Trials%26jtitle%3DMol.%2520Cancer%26date%3D2019%26volume%3D18%26spage%3D26%26doi%3D10.1186%2Fs12943-019-0954-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Markham, A.</span></span> <span> </span><span class="NLM_article-title">Alpelisib: First Global Approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1249</span>– <span class="NLM_lpage">1253</span>, <span class="refDoi"> DOI: 10.1007/s40265-019-01161-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1007%2Fs40265-019-01161-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=31256368" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlCiur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=1249-1253&author=A.+Markham&title=Alpelisib%3A+First+Global+Approval&doi=10.1007%2Fs40265-019-01161-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Alpelisib: First Global Approval</span></div><div class="casAuthors">Markham, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1249-1253</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Alpelisib (Piqray)-an orally available phosphatidylinositol 3-kinase (PI3K) inhibitor with specific activity against PI3K alpha (PI3Kα)-is being developed by Novartis for the treatment of breast cancer.  Alpelisib has demonstrated efficacy in combination with fulvestrant as treatment for hormone receptor (HR)-pos., human epidermal growth factor receptor-2 (HER2)-neg. breast cancer in patients with a PIK3CA mutation and was recently approved for this indication in the USA.  This article summarizes the milestones in the development of alpelisib leading to this first approval.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqX1sEhr0QGHrVg90H21EOLACvtfcHk0ljY3Tz36Ya5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlCiur%252FN&md5=4d9e9d1e3742a9bce160ca2c7987ce6c</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs40265-019-01161-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-019-01161-6%26sid%3Dliteratum%253Aachs%26aulast%3DMarkham%26aufirst%3DA.%26atitle%3DAlpelisib%253A%2520First%2520Global%2520Approval%26jtitle%3DDrugs%26date%3D2019%26volume%3D79%26spage%3D1249%26epage%3D1253%26doi%3D10.1007%2Fs40265-019-01161-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pons-Tostivint, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thibault, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillermet-Guibert, J.</span></span> <span> </span><span class="NLM_article-title">Targeting PI3K Signaling in Combination Cancer Therapy</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">454</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2017.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.trecan.2017.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28718419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXksVOksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=454-469&author=E.+Pons-Tostivintauthor=B.+Thibaultauthor=J.+Guillermet-Guibert&title=Targeting+PI3K+Signaling+in+Combination+Cancer+Therapy&doi=10.1016%2Fj.trecan.2017.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting PI3K Signaling in Combination Cancer Therapy</span></div><div class="casAuthors">Pons-Tostivint, Elvire; Thibault, Benoit; Guillermet-Guibert, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">454-469</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeting upstream phosphatidylinositol-3-kinases (PI3Ks) in the PI3K/Akt/mTOR pathway appears to be a promising therapy in solid cancers; however, first early clin. trials with PI3K inhibitors in monotherapy have been disappointing.  A massive array of preclin. and clin. trials are currently evaluating combinations of PI3K inhibitors in targeted therapies.  These combinations include co-treatments with drugs directed against other intra-/extracellular signaling mols., nuclear hormone receptors, DNA damage repair enzymes, and immune modulators.  We review the literature and pinpoint mechanisms of action in different genomic and organ contexts.  Combinatorial approaches are potentially superior to monotherapies and should become alternative clin. strategies to treat cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotS0vLAVeC87Vg90H21EOLACvtfcHk0ljY3Tz36Ya5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXksVOksQ%253D%253D&md5=3b905f4d00de4428b52078434ad11f12</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2017.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2017.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DPons-Tostivint%26aufirst%3DE.%26aulast%3DThibault%26aufirst%3DB.%26aulast%3DGuillermet-Guibert%26aufirst%3DJ.%26atitle%3DTargeting%2520PI3K%2520Signaling%2520in%2520Combination%2520Cancer%2520Therapy%26jtitle%3DTrends%2520Cancer%26date%3D2017%26volume%3D3%26spage%3D454%26epage%3D469%26doi%3D10.1016%2Fj.trecan.2017.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">West, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span> <span> </span><span class="NLM_article-title">New and Emerging HDAC Inhibitors for Cancer Treatment</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">30</span>– <span class="NLM_lpage">39</span>, <span class="refDoi"> DOI: 10.1172/JCI69738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1172%2FJCI69738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=24382387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntFejtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2014&pages=30-39&author=A.+C.+Westauthor=R.+W.+Johnstone&title=New+and+Emerging+HDAC+Inhibitors+for+Cancer+Treatment&doi=10.1172%2FJCI69738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">New and emerging HDAC inhibitors for cancer treatment</span></div><div class="casAuthors">West, Alison C.; Johnstone, Ricky W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">30-39</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">A review.  Epigenetic enzymes are often dysregulated in human tumors through mutation, altered expression, or inappropriate recruitment to certain loci.  The identification of these enzymes and their partner proteins has driven the rapid development of small-mol. inhibitors that target the cancer epigenome.  Herein, the authors discuss the influence of aberrantly regulated histone deacetylases (HDACs) in tumorigenesis.  The authors examine HDAC inhibitors (HDACis) targeting class I, II, and IV HDACs that are currently under development for use as anticancer agents following the FDA approval of two HDACis, vorinostat and romidepsin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpELUU7P0B_grVg90H21EOLACvtfcHk0ljY3Tz36Ya5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntFejtg%253D%253D&md5=ab59960223b33e89725f6539b031f531</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1172%2FJCI69738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI69738%26sid%3Dliteratum%253Aachs%26aulast%3DWest%26aufirst%3DA.%2BC.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26atitle%3DNew%2520and%2520Emerging%2520HDAC%2520Inhibitors%2520for%2520Cancer%2520Treatment%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2014%26volume%3D124%26spage%3D30%26epage%3D39%26doi%3D10.1172%2FJCI69738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span> <span> </span><span class="NLM_article-title">Trend of Histone Deacetylase Inhibitors in Cancer Therapy: Isoform Selectivity or Multitargeted Strategy</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">63</span>– <span class="NLM_lpage">84</span>, <span class="refDoi"> DOI: 10.1002/med.21320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1002%2Fmed.21320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=24782318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVClsb3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=63-84&author=L.+Zhangauthor=Y.+Hanauthor=Q.+Jiangauthor=C.+Wangauthor=X.+Chenauthor=X.+Liauthor=F.+Xuauthor=Y.+Jiangauthor=Q.+Wangauthor=W.+Xu&title=Trend+of+Histone+Deacetylase+Inhibitors+in+Cancer+Therapy%3A+Isoform+Selectivity+or+Multitargeted+Strategy&doi=10.1002%2Fmed.21320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Trend of Histone Deacetylase Inhibitors in Cancer Therapy: Isoform Selectivity or Multitargeted Strategy</span></div><div class="casAuthors">Zhang, Lei; Han, Yantao; Jiang, Qixiao; Wang, Chunbo; Chen, Xuehong; Li, Xiaoguang; Xu, Fuming; Jiang, Yuqi; Wang, Qiang; Xu, Wenfang</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">63-84</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Pharmacol. inhibition of histone deacetylases (HDACs) has been successfully applied in the treatment of a wide range of disorders, including Parkinson's disease, infection, cardiac diseases, inflammation, and esp. cancer.  HDAC inhibitors (HDACIs) have been proved to be effective antitumor agents by various stages of investigation.  At present, there are two opposite focuses of HDACI design in the cancer therapy, highly selective inhibitor strategy and dual- or multitargeted inhibitors.  The former method, which is supposed to elucidate the function of individual HDAC and provide candidate inhibitors with fewer side effects, has been widely accepted by the inhibitor developer.  The latter approach, though less practiced, has promising potential for the antitumor therapy based on HDACIs.  Effective HDACIs, some of which are in clinic anticancer research, have been developed by both methods.  In order to gain insight into HDACI design, the strategies and achievements of the two diverse methods are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm6HJUNw_9dLVg90H21EOLACvtfcHk0ljY3Tz36Ya5og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVClsb3M&md5=791e7d6b057c92f206c8d0d75099e021</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2Fmed.21320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21320%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DTrend%2520of%2520Histone%2520Deacetylase%2520Inhibitors%2520in%2520Cancer%2520Therapy%253A%2520Isoform%2520Selectivity%2520or%2520Multitargeted%2520Strategy%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2015%26volume%3D35%26spage%3D63%26epage%3D84%26doi%3D10.1002%2Fmed.21320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eckschlager, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plch, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stiborova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hrabeta, J.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase Inhibitors as Anticancer Drugs</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1414</span>, <span class="refDoi"> DOI: 10.3390/ijms18071414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3390%2Fijms18071414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28671573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=1414&author=T.+Eckschlagerauthor=J.+Plchauthor=M.+Stiborovaauthor=J.+Hrabeta&title=Histone+Deacetylase+Inhibitors+as+Anticancer+Drugs&doi=10.3390%2Fijms18071414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase inhibitors as anticancer drugs</span></div><div class="casAuthors">Eckschlager, Tomas; Plch, Johana; Stiborova, Marie; Hrabeta, Jan</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1414/1-1414/25</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Carcinogenesis cannot be explained only by genetic alterations, but also involves epigenetic processes.  Modification of histones by acetylation plays a key role in epigenetic regulation of gene expression and is controlled by the balance between histone deacetylases (HDAC) and histone acetyltransferases (HAT).  HDAC inhibitors induce cancer cell cycle arrest, differentiation and cell death, reduce angiogenesis and modulate immune response.  Mechanisms of anticancer effects of HDAC inhibitors are not uniform; they may be different and depend on the cancer type, HDAC inhibitors, doses, etc.  HDAC inhibitors seem to be promising anti-cancer drugs particularly in the combination with other anti-cancer drugs and/or radiotherapy.  HDAC inhibitors vorinostat, romidepsin and belinostat have been approved for some T-cell lymphoma and panobinostat for multiple myeloma.  Other HDAC inhibitors are in clin. trials for the treatment of hematol. and solid malignancies.  The results of such studies are promising but further larger studies are needed.  Because of the reversibility of epigenetic changes during cancer development, the potency of epigenetic therapies seems to be of great importance.  Here, we summarize the data on different classes of HDAC inhibitors, mechanisms of their actions and discuss novel results of preclin. and clin. studies, including the combination with other therapeutic modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPNVhoLBAHnrVg90H21EOLACvtfcHk0lhWNR9OPvad0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVGgt73P&md5=6018c7709a09548214bfe34bd91107a9</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.3390%2Fijms18071414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18071414%26sid%3Dliteratum%253Aachs%26aulast%3DEckschlager%26aufirst%3DT.%26aulast%3DPlch%26aufirst%3DJ.%26aulast%3DStiborova%26aufirst%3DM.%26aulast%3DHrabeta%26aufirst%3DJ.%26atitle%3DHistone%2520Deacetylase%2520Inhibitors%2520as%2520Anticancer%2520Drugs%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D1414%26doi%3D10.3390%2Fijms18071414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Suraweera, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Byrne, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richard, D. J.</span></span> <span> </span><span class="NLM_article-title">Combination Therapy with Histone Deacetylase Inhibitors (HDACi) for the Treatment of Cancer: Achieving the Full Therapeutic Potential of HDACi</span>. <i>Front. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">92</span>, <span class="refDoi"> DOI: 10.3389/fonc.2018.00092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3389%2Ffonc.2018.00092" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29651407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC1MjitlSksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=92&author=A.+Suraweeraauthor=K.+J.+O%E2%80%99Byrneauthor=D.+J.+Richard&title=Combination+Therapy+with+Histone+Deacetylase+Inhibitors+%28HDACi%29+for+the+Treatment+of+Cancer%3A+Achieving+the+Full+Therapeutic+Potential+of+HDACi&doi=10.3389%2Ffonc.2018.00092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Combination Therapy With Histone Deacetylase Inhibitors (HDACi) for the Treatment of Cancer: Achieving the Full Therapeutic Potential of HDACi</span></div><div class="casAuthors">Suraweera Amila; O'Byrne Kenneth J; Richard Derek J; O'Byrne Kenneth J</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">92</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Genetic and epigenetic changes in DNA are involved in cancer development and tumor progression.  Histone deacetylases (HDACs) are key regulators of gene expression that act as transcriptional repressors by removing acetyl groups from histones.  HDACs are dysregulated in many cancers, making them a therapeutic target for the treatment of cancer.  Histone deacetylase inhibitors (HDACi), a novel class of small-molecular therapeutics, are now approved by the Food and Drug Administration as anticancer agents.  While they have shown great promise, resistance to HDACi is often observed and furthermore, HDACi have shown limited success in treating solid tumors.  The combination of HDACi with standard chemotherapeutic drugs has demonstrated promising anticancer effects in both preclinical and clinical studies.  In this review, we summarize the research thus far on HDACi in combination therapy, with other anticancer agents and their translation into preclinical and clinical studies.  We additionally highlight the side effects associated with HDACi in cancer therapy and discuss potential biomarkers to either select or predict a patient's response to these agents, in order to limit the off-target toxicity associated with HDACi.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTprF0Hdrz8rZZoeNcnHOQ9fW6udTcc2ea1d8JkTPZXQ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MjitlSksA%253D%253D&md5=2f0ace7215bc7519c23cad264f73693d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2018.00092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2018.00092%26sid%3Dliteratum%253Aachs%26aulast%3DSuraweera%26aufirst%3DA.%26aulast%3DO%25E2%2580%2599Byrne%26aufirst%3DK.%2BJ.%26aulast%3DRichard%26aufirst%3DD.%2BJ.%26atitle%3DCombination%2520Therapy%2520with%2520Histone%2520Deacetylase%2520Inhibitors%2520%2528HDACi%2529%2520for%2520the%2520Treatment%2520of%2520Cancer%253A%2520Achieving%2520the%2520Full%2520Therapeutic%2520Potential%2520of%2520HDACi%26jtitle%3DFront.%2520Oncol.%26date%3D2018%26volume%3D8%26spage%3D92%26doi%3D10.3389%2Ffonc.2018.00092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bates, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espinoza-Delgado, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piekarz, R. L.</span></span> <span> </span><span class="NLM_article-title">Clinical Toxicities of Histone Deacetylase Inhibitors</span>. <i>Pharmaceuticals</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">2751</span>– <span class="NLM_lpage">2767</span>, <span class="refDoi"> DOI: 10.3390/ph3092751</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3390%2Fph3092751" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27713375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFSmtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2010&pages=2751-2767&author=S.+Subramanianauthor=S.+E.+Batesauthor=J.+J.+Wrightauthor=I.+Espinoza-Delgadoauthor=R.+L.+Piekarz&title=Clinical+Toxicities+of+Histone+Deacetylase+Inhibitors&doi=10.3390%2Fph3092751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical toxicities of histone deacetylase inhibitors</span></div><div class="casAuthors">Subramanian, Srividya; Bates, Susan E.; Wright, John J.; Espinoza-Delgado, Igor; Piekarz, Richard L.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceuticals</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">2751-2767</span>CODEN:
                <span class="NLM_cas:coden">PHARH2</span>;
        ISSN:<span class="NLM_cas:issn">1424-8247</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  The HDAC inhibitors are a new family of antineoplastic agents.  Since the entry of these agents into our therapeutic armamentarium, there has been increasing interest in their use.  Although this family comprises chem. compds. from unrelated chem. classes that have different HDAC isoform specificities, they surprisingly have very similar toxicity profiles.  In contrast, the obsd. toxicity profile is somewhat different from that of traditional cytotoxic chemotherapeutic agents and from other epigenetic agents.  While some of the side effects may be familiar to the oncologist, others are less commonly seen.  As some patients remain on therapy for a prolonged period of time, the long-term sequelae need to be characterized.  In addn., since preclin. models suggest promising activity when used in combination with other antineoplastic agents, combination trials are being pursued.  It will thus be important to distinguish the relative toxicity attributed to these agents and be alert to the exacerbation of toxicities obsd. in single agent studies.  Notably, few of the agents in this class have completed phase 2 testing.  Consequently, more clin. experience is needed to det. the relative frequency of the obsd. side effects, and to identify and develop approaches to mitigate potential clin. sequelae.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrriVU62aelgbVg90H21EOLACvtfcHk0lhWNR9OPvad0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFSmtrvK&md5=d0937756ff0e5582c1e33901c913ff97</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fph3092751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fph3092751%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DS.%26aulast%3DBates%26aufirst%3DS.%2BE.%26aulast%3DWright%26aufirst%3DJ.%2BJ.%26aulast%3DEspinoza-Delgado%26aufirst%3DI.%26aulast%3DPiekarz%26aufirst%3DR.%2BL.%26atitle%3DClinical%2520Toxicities%2520of%2520Histone%2520Deacetylase%2520Inhibitors%26jtitle%3DPharmaceuticals%26date%3D2010%26volume%3D3%26spage%3D2751%26epage%3D2767%26doi%3D10.3390%2Fph3092751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yee, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raje, N. S.</span></span> <span> </span><span class="NLM_article-title">Panobinostat and Multiple Myeloma in 2018</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">516</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2017-0644</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1634%2Ftheoncologist.2017-0644" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29445026" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC1MrhslOluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=516-517&author=A.+J.+Yeeauthor=N.+S.+Raje&title=Panobinostat+and+Multiple+Myeloma+in+2018&doi=10.1634%2Ftheoncologist.2017-0644"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Panobinostat and Multiple Myeloma in 2018</span></div><div class="casAuthors">Yee Andrew J; Raje Noopur S; Yee Andrew J; Raje Noopur S</div><div class="citationInfo"><span class="NLM_cas:title">The oncologist</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">516-517</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcShdJOozuyZkPtos8R3JUhXfW6udTcc2eYjpc3c9eYZG7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MrhslOluw%253D%253D&md5=7202e92be5a6f4cc72d5317b35ec687a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2017-0644&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2017-0644%26sid%3Dliteratum%253Aachs%26aulast%3DYee%26aufirst%3DA.%2BJ.%26aulast%3DRaje%26aufirst%3DN.%2BS.%26atitle%3DPanobinostat%2520and%2520Multiple%2520Myeloma%2520in%25202018%26jtitle%3DOncologist%26date%3D2018%26volume%3D23%26spage%3D516%26epage%3D517%26doi%3D10.1634%2Ftheoncologist.2017-0644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thurn, K. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munster, P. N.</span></span> <span> </span><span class="NLM_article-title">Rational Therapeutic Combinations with Histone Deacetylase Inhibitors for the Treatment of Cancer</span>. <i>Future Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">283</span>, <span class="refDoi"> DOI: 10.2217/fon.11.2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.2217%2Ffon.11.2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21345145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisVWlurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=263-283&author=K.+T.+Thurnauthor=S.+Thomasauthor=A.+Mooreauthor=P.+N.+Munster&title=Rational+Therapeutic+Combinations+with+Histone+Deacetylase+Inhibitors+for+the+Treatment+of+Cancer&doi=10.2217%2Ffon.11.2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rational therapeutic combinations with histone deacetylase inhibitors for the treatment of cancer</span></div><div class="casAuthors">Thurn, K. Ted; Thomas, Scott; Moore, Amy; Munster, Pamela N.</div><div class="citationInfo"><span class="NLM_cas:title">Future Oncology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-283</span>CODEN:
                <span class="NLM_cas:coden">FOUNBN</span>;
        ISSN:<span class="NLM_cas:issn">1479-6694</span>.
    
            (<span class="NLM_cas:orgname">Future Medicine Ltd.</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) regulate the acetylation of a variety of histone and nonhistone proteins, controlling the transcription and regulation of genes involved in cell cycle control, proliferation, survival, DNA repair and differentiation.  Unsurprisingly, HDAC expression is frequently altered in hematol. and solid tumor malignancies.  Two HDAC inhibitors (vorinostat and romidepsin) have been approved by the US FDA for the treatment of cutaneous T-cell lymphoma.  As single agents, treatment with HDAC inhibitors has demonstrated limited clin. benefit for patients with solid tumors, prompting the investigation of novel treatment combinations with other cancer therapeutics.  In this article, the rationales and clin. progress of several combinations with HDAC inhibitors are presented, including DNA-damaging chemotherapeutic agents, radiotherapy, hormonal therapies, DNA methyltransferase inhibitors and various small-mol. inhibitors.  The future application of HDAC inhibitors as a treatment for cancer is discussed, examg. current hurdles to overcome before realizing the potential of this new approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqn7EKQGlvu_LVg90H21EOLACvtfcHk0ljhBhXYxrE4jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisVWlurs%253D&md5=193fc07f4e19bf16fcd7ee202057ef7c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.2217%2Ffon.11.2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Ffon.11.2%26sid%3Dliteratum%253Aachs%26aulast%3DThurn%26aufirst%3DK.%2BT.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DMoore%26aufirst%3DA.%26aulast%3DMunster%26aufirst%3DP.%2BN.%26atitle%3DRational%2520Therapeutic%2520Combinations%2520with%2520Histone%2520Deacetylase%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Cancer%26jtitle%3DFuture%2520Oncol.%26date%3D2011%26volume%3D7%26spage%3D263%26epage%3D283%26doi%3D10.2217%2Ffon.11.2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bodo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portell, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lannutti, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almasan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsi, E. D.</span></span> <span> </span><span class="NLM_article-title">The Phosphatidylinositol 3-Kinases (PI3K) Inhibitor GS-1101 Synergistically Potentiates Histone Deacetylase Inhibitor-Induced Proliferation Inhibition and Apoptosis through the Inactivation of PI3K and Extracellular Signal-Regulated Kinase Pathways</span>. <i>Br. J. Haematol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>163</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.1111/bjh.12498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1111%2Fbjh.12498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=23889282" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVOgsr7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=163&publication_year=2013&pages=72-80&author=J.+Bodoauthor=X.+Zhaoauthor=A.+Sharmaauthor=B.+T.+Hillauthor=C.+A.+Portellauthor=B.+J.+Lannuttiauthor=A.+Almasanauthor=E.+D.+Hsi&title=The+Phosphatidylinositol+3-Kinases+%28PI3K%29+Inhibitor+GS-1101+Synergistically+Potentiates+Histone+Deacetylase+Inhibitor-Induced+Proliferation+Inhibition+and+Apoptosis+through+the+Inactivation+of+PI3K+and+Extracellular+Signal-Regulated+Kinase+Pathways&doi=10.1111%2Fbjh.12498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">The phosphatidylinositol 3-kinases (PI3K) inhibitor GS-1101 synergistically potentiates histone deacetylase inhibitor-induced proliferation inhibition and apoptosis through the inactivation of PI3K and extracellular signal-regulated kinase pathways</span></div><div class="casAuthors">Bodo, Juraj; Zhao, Xiaoxian; Sharma, Arishya; Hill, Brian T.; Portell, Craig A.; Lannutti, Brian J.; Almasan, Alexandru; Hsi, Eric D.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Haematology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">163</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-80</span>CODEN:
                <span class="NLM_cas:coden">BJHEAL</span>;
        ISSN:<span class="NLM_cas:issn">0007-1048</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Previously, we showed that inhibition of the protein kinase C β (PKCβ)/AKT pathway augments engagement of the histone deacetylase inhibitor (HDI)-induced apoptosis in lymphoma cells.  In the present study, we investigated the cytotoxicity and mechanisms of cell death induced by the delta isoform-specific phosphatidylinositide 3-kinase (PI3K) inhibitor, GS-1101, in combination with the HDI, panobinostat (LBH589) and suberoylanilide hydroxamic acid (SAHA).  Lymphoma cell lines, primary non-Hodgkin Lymphoma (NHL) and chronic lymphocytic leukemia (CLL) cells were simultaneously treated with the HDI,LBH589 and GS-1101.  An interaction of the LBH589/GS-1101 combination was formally examd. by using various concns. of LBH589 and GS-1101.  Combined treatment resulted in a synergistic inhibition of proliferation and showed synergistic effect on apoptotic induction in all tested cell lines and primary NHL and CLL cells.  This study indicates that interference with PI3K signalling dramatically increases HDI-mediated apoptosis in malignant haematopoietic cells, possibly through both AKT-dependent or AKT- independent mechanisms.  Moreover, the increase in HDI-related apoptosis obsd. in PI3K inhibitor-treated cells appears to be related to the disruption of the extracellular signal-regulated kinase (ERK) signalling pathway.  This study provides a strong rational for testing the combination of PI3K inhibitors and HDI in the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf_Pb47waY5bVg90H21EOLACvtfcHk0ljhBhXYxrE4jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVOgsr7J&md5=ab27c7447ad089cc1d38f668dcc35760</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1111%2Fbjh.12498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbjh.12498%26sid%3Dliteratum%253Aachs%26aulast%3DBodo%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DHill%26aufirst%3DB.%2BT.%26aulast%3DPortell%26aufirst%3DC.%2BA.%26aulast%3DLannutti%26aufirst%3DB.%2BJ.%26aulast%3DAlmasan%26aufirst%3DA.%26aulast%3DHsi%26aufirst%3DE.%2BD.%26atitle%3DThe%2520Phosphatidylinositol%25203-Kinases%2520%2528PI3K%2529%2520Inhibitor%2520GS-1101%2520Synergistically%2520Potentiates%2520Histone%2520Deacetylase%2520Inhibitor-Induced%2520Proliferation%2520Inhibition%2520and%2520Apoptosis%2520through%2520the%2520Inactivation%2520of%2520PI3K%2520and%2520Extracellular%2520Signal-Regulated%2520Kinase%2520Pathways%26jtitle%3DBr.%2520J.%2520Haematol.%26date%3D2013%26volume%3D163%26spage%3D72%26epage%3D80%26doi%3D10.1111%2Fbjh.12498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garancher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esparza, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udaka, Y. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seker-Cin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brabetz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogiso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kool, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wechsler-Reya, R. J.</span></span> <span> </span><span class="NLM_article-title">HDAC and PI3K Antagonists Cooperate to Inhibit Growth of MYC-Driven Medulloblastoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">311</span>– <span class="NLM_lpage">323</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.ccell.2016.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=26977882" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XksVWitL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=311-323&author=Y.+Peiauthor=K.+W.+Liuauthor=J.+Wangauthor=A.+Garancherauthor=R.+Taoauthor=L.+A.+Esparzaauthor=D.+L.+Maierauthor=Y.+T.+Udakaauthor=N.+Muradauthor=S.+Morrissyauthor=H.+Seker-Cinauthor=S.+Brabetzauthor=L.+Qiauthor=M.+Kogisoauthor=S.+Schubertauthor=J.+M.+Olsonauthor=Y.+J.+Choauthor=X.+N.+Liauthor=J.+R.+Crawfordauthor=M.+L.+Levyauthor=M.+Koolauthor=S.+M.+Pfisterauthor=M.+D.+Taylorauthor=R.+J.+Wechsler-Reya&title=HDAC+and+PI3K+Antagonists+Cooperate+to+Inhibit+Growth+of+MYC-Driven+Medulloblastoma&doi=10.1016%2Fj.ccell.2016.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">HDAC and PI3K Antagonists Cooperate to Inhibit Growth of MYC-Driven Medulloblastoma</span></div><div class="casAuthors">Pei, Yanxin; Liu, Kun-Wei; Wang, Jun; Garancher, Alexandra; Tao, Ran; Esparza, Lourdes A.; Maier, Donna L.; Udaka, Yoko T.; Murad, Najiba; Morrissy, Sorana; Seker-Cin, Huriye; Brabetz, Sebastian; Qi, Lin; Kogiso, Mari; Schubert, Simone; Olson, James M.; Cho, Yoon-Jae; Li, Xiao-Nan; Crawford, John R.; Levy, Michael L.; Kool, Marcel; Pfister, Stefan M.; Taylor, Michael D.; Wechsler-Reya, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">311-323</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Medulloblastoma (MB) is a highly malignant pediatric brain tumor.  Despite aggressive therapy, many patients succumb to the disease, and survivors experience severe side effects from treatment.  MYC-driven MB has a particularly poor prognosis and would greatly benefit from more effective therapies.  We used an animal model of MYC-driven MB to screen for drugs that decrease viability of tumor cells.  Among the most effective compds. were histone deacetylase inhibitors (HDACIs).  HDACIs potently inhibit survival of MYC-driven MB cells in vitro, in part by inducing expression of the FOXO1 tumor suppressor gene.  HDACIs also synergize with phosphatidylinositol 3-kinase inhibitors to inhibit tumor growth in vivo.  These studies identify an effective combination therapy for the most aggressive form of MB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfHu8mC8vyrVg90H21EOLACvtfcHk0lglyxEtmXrOBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XksVWitL0%253D&md5=c3e20bc1d1ef7af52c2a9ee935b8b7e5</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DGarancher%26aufirst%3DA.%26aulast%3DTao%26aufirst%3DR.%26aulast%3DEsparza%26aufirst%3DL.%2BA.%26aulast%3DMaier%26aufirst%3DD.%2BL.%26aulast%3DUdaka%26aufirst%3DY.%2BT.%26aulast%3DMurad%26aufirst%3DN.%26aulast%3DMorrissy%26aufirst%3DS.%26aulast%3DSeker-Cin%26aufirst%3DH.%26aulast%3DBrabetz%26aufirst%3DS.%26aulast%3DQi%26aufirst%3DL.%26aulast%3DKogiso%26aufirst%3DM.%26aulast%3DSchubert%26aufirst%3DS.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCho%26aufirst%3DY.%2BJ.%26aulast%3DLi%26aufirst%3DX.%2BN.%26aulast%3DCrawford%26aufirst%3DJ.%2BR.%26aulast%3DLevy%26aufirst%3DM.%2BL.%26aulast%3DKool%26aufirst%3DM.%26aulast%3DPfister%26aufirst%3DS.%2BM.%26aulast%3DTaylor%26aufirst%3DM.%2BD.%26aulast%3DWechsler-Reya%26aufirst%3DR.%2BJ.%26atitle%3DHDAC%2520and%2520PI3K%2520Antagonists%2520Cooperate%2520to%2520Inhibit%2520Growth%2520of%2520MYC-Driven%2520Medulloblastoma%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D29%26spage%3D311%26epage%3D323%26doi%3D10.1016%2Fj.ccell.2016.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernatchez, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span> <span> </span><span class="NLM_article-title">Kinase and Histone Deacetylase Hybrid Inhibitors for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3171</span>– <span class="NLM_lpage">3183</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00189</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00189" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitFKntbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3171-3183&author=Y.+Luanauthor=J.+Liauthor=J.+A.+Bernatchezauthor=R.+Li&title=Kinase+and+Histone+Deacetylase+Hybrid+Inhibitors+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.8b00189"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase and Histone Deacetylase Hybrid Inhibitors for Cancer Therapy</span></div><div class="casAuthors">Luan, Yepeng; Li, Jerry; Bernatchez, Jean A.; Li, Rongshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3171-3183</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs), encompassing at least 18 members, are promising targets for anticancer drug discovery and development.  To date, five histone deacetylase inhibitors (HDACis) have been approved for cancer treatment, and numerous others are undergoing clin. trials.  It has been well validated that an agent that can simultaneously and effectively inhibit two or more targets may offer greater therapeutic benefits over single-acting agents in preventing resistance to treatment and in potentiating synergistic effects.  A prime example of a bifunctional agent is the hybrid HDAC inhibitor.  In this perspective, the authors review the majority of reported kinase/HDAC hybrid inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvS3FgFwT7r7Vg90H21EOLACvtfcHk0lglyxEtmXrOBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitFKntbnF&md5=a75c9578b68044a0f702bbf901c3cdcf</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00189%26sid%3Dliteratum%253Aachs%26aulast%3DLuan%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DBernatchez%26aufirst%3DJ.%2BA.%26aulast%3DLi%26aufirst%3DR.%26atitle%3DKinase%2520and%2520Histone%2520Deacetylase%2520Hybrid%2520Inhibitors%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3171%26epage%3D3183%26doi%3D10.1021%2Facs.jmedchem.8b00189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Biological Evaluation of 2,3-Dihydroimidazo[1,2-C]Quinazoline Derivatives as Novel Phosphatidylinositol 3-Kinase and Histone Deacetylase Dual Inhibitors</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">52180</span>– <span class="NLM_lpage">52186</span>, <span class="refDoi"> DOI: 10.1039/C7RA08835C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1039%2FC7RA08835C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslyqs7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=52180-52186&author=Y.+Wuauthor=W.+Daiauthor=X.+Chenauthor=A.+Gengauthor=Y.+Chenauthor=T.+Luauthor=Y.+Zhu&title=Design%2C+Synthesis+and+Biological+Evaluation+of+2%2C3-Dihydroimidazo%5B1%2C2-C%5DQuinazoline+Derivatives+as+Novel+Phosphatidylinositol+3-Kinase+and+Histone+Deacetylase+Dual+Inhibitors&doi=10.1039%2FC7RA08835C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of 2,3-dihydroimidazo[1,2-c]quinazoline derivatives as novel phosphatidylinositol 3-kinase and histone deacetylase dual inhibitors</span></div><div class="casAuthors">Wu, Yichao; Dai, Weichen; Chen, Xin; Geng, Aixin; Chen, Yadong; Lu, Tao; Zhu, Yong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">82</span>),
    <span class="NLM_cas:pages">52180-52186</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors are known to induce multiple epigenetic modifications affecting signaling networks and act synergistically with phosphatidylinositol 3-kinase (PI3K) inhibitors for the treatment of cancer.  Herein we present a novel design approach for cancer drug development by incorporating HDAC inhibitory functionality into a PI3K inhibitor pharmacophore to construct dual-acting inhibitors.  The designed compds. were synthesized and showed inhibitory activities against PI3K and HDAC.  The representative dual PI3K/HDAC inhibitors, compds. 12a-j, showed potent antiproliferative activities against K562 and Hut78 in cellular assays.  This work may lay the foundation for developing novel dual PI3K/HDAC inhibitors as potential anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgJqax7GMFRrVg90H21EOLACvtfcHk0lglyxEtmXrOBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslyqs7zL&md5=895434b4f0bf7e7dc7504dffeaba0fe1</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1039%2FC7RA08835C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7RA08835C%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDai%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGeng%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Biological%2520Evaluation%2520of%25202%252C3-Dihydroimidazo%255B1%252C2-C%255DQuinazoline%2520Derivatives%2520as%2520Novel%2520Phosphatidylinositol%25203-Kinase%2520and%2520Histone%2520Deacetylase%2520Dual%2520Inhibitors%26jtitle%3DRSC%2520Adv.%26date%3D2017%26volume%3D7%26spage%3D52180%26epage%3D52186%26doi%3D10.1039%2FC7RA08835C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soh, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Preclinical Evaluation of Fused Pyrimidine-Based Hydroxamates for the Treatment of Hepatocellular Carcinoma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1552</span>– <span class="NLM_lpage">1575</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01465</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01465" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1552-1575&author=D.+Chenauthor=C.+K.+Sohauthor=W.+H.+Gohauthor=H.+Wang&title=Design%2C+Synthesis%2C+and+Preclinical+Evaluation+of+Fused+Pyrimidine-Based+Hydroxamates+for+the+Treatment+of+Hepatocellular+Carcinoma&doi=10.1021%2Facs.jmedchem.7b01465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Preclinical Evaluation of Fused Pyrimidine-Based Hydroxamates for the Treatment of Hepatocellular Carcinoma</span></div><div class="casAuthors">Chen, Dizhong; Soh, Chang Kai; Goh, Wei Huang; Wang, Haishan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1552-1575</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Class I histone deacetylases (HDACs) are highly expressed and/or upregulated in hepatocellular carcinoma (HCC) and are assocd. with aggressiveness, spread, and increased mortality of HCC.  Activation of phosphatidylinositol 3-kinase-Akt-mammalian target of rapamycin (PI3K/Akt/mTOR) signaling pathway was involved in the development of HCC and acquired resistance to sorafenib.  A series of purine or 5H-pyrrolo[3,2-d]pyrimidine based hydroxamates were designed and developed as multitarget drugs to modulate both HDACs and the PI3K/Akt/mTOR pathway.  Among 39 cell lines screened, the mols. (e.g., I, II, and III) were the most selective against leukemia, lymphoma, and HCC cells; they also demonstrated target modulation in cancer cell lines and in mice bearing MV4-11 and HepG2 tumors.  Compd. II in particular showed significant single agent oral efficacy in hypervascular liver cancer models (e.g., HepG2, HuH-7, and Hep3B) and was well-tolerated.  These encouraging results, along with its favorable target profile and tissue distribution, warrant further development of II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOkc89pglhFLVg90H21EOLACvtfcHk0ljSAbk7VyE2Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsLk%253D&md5=7261ba423c033773f4b1b5cea80af725</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01465%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DSoh%26aufirst%3DC.%2BK.%26aulast%3DGoh%26aufirst%3DW.%2BH.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Preclinical%2520Evaluation%2520of%2520Fused%2520Pyrimidine-Based%2520Hydroxamates%2520for%2520the%2520Treatment%2520of%2520Hepatocellular%2520Carcinoma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1552%26epage%3D1575%26doi%3D10.1021%2Facs.jmedchem.7b01465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6992</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00390</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00390" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6992-7014&author=K.+Zhangauthor=F.+Laiauthor=S.+Linauthor=M.+Jiauthor=J.+Zhangauthor=Y.+Zhangauthor=J.+Jinauthor=R.+Fuauthor=D.+Wuauthor=H.+Tianauthor=N.+Xueauthor=L.+Shengauthor=X.+Zouauthor=Y.+Liauthor=X.+Chenauthor=H.+Xu&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+4-Methyl+Quinazoline+Derivatives+as+Anticancer+Agents+Simultaneously+Targeting+Phosphoinositide+3-Kinases+and+Histone+Deacetylases&doi=10.1021%2Facs.jmedchem.9b00390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span></div><div class="casAuthors">Zhang, Kehui; Lai, Fangfang; Lin, Songwen; Ji, Ming; Zhang, Jingbo; Zhang, Yan; Jin, Jing; Fu, Rong; Wu, Deyu; Tian, Hua; Xue, Nina; Sheng, Li; Zou, Xiaowen; Li, Yan; Chen, Xiaoguang; Xu, Heng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6992-7014</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polypharmacol. is a promising paradigm in modern drug discovery.  Herein, we have discovered a series of novel PI3K and HDAC dual inhibitors in which the hydroxamic acid moiety as the zinc binding functional group was introduced to a quinazoline-based PI3K pharmacophore through an appropriate linker.  Systematic structure-activity relationship studies resulted in lead compds. 23 (shown in graphical abstr. and duplicated as I) and 36 (II) that simultaneously inhibited PI3K and HDAC with nanomolar potencies and demonstrated favorable antiproliferative activities.  Compds. 23 and 36 efficiently modulated the expression of p-AKT and Ac-H3, arrested the cell cycle, and induced apoptosis in HCT116 cancer cells.  Following pharmacokinetic studies, 23 was further evaluated in HCT116 and HGC-27 xenograft models to show significant in vivo anticancer efficacies with tumor growth inhibitions of 45.8% (po, 150 mg/kg) and 62.6% (i.p., 30 mg/kg), resp.  Overall, this work shows promise in discovering new anticancer therapeutics by the approach of simultaneously targeting PI3K and HDAC pathways with a single mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6xLiFaGhwk7Vg90H21EOLACvtfcHk0ljSAbk7VyE2Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D&md5=de4c9a630208e49eac58dcad630adcd6</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00390%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DXue%26aufirst%3DN.%26aulast%3DSheng%26aufirst%3DL.%26aulast%3DZou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%25204-Methyl%2520Quinazoline%2520Derivatives%2520as%2520Anticancer%2520Agents%2520Simultaneously%2520Targeting%2520Phosphoinositide%25203-Kinases%2520and%2520Histone%2520Deacetylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6992%26epage%3D7014%26doi%3D10.1021%2Facs.jmedchem.9b00390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atoyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellarocca, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samson, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, G. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patterson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fattaey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Dual HDAC and PI3K Inhibitor CUDC-907 Downregulates MYC and Suppresses Growth of MYC-Dependent Cancers</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-16-0390</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1158%2F1535-7163.MCT-16-0390" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27980108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVertro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=285-299&author=K.+Sunauthor=R.+Atoyanauthor=M.+A.+Borekauthor=S.+Dellaroccaauthor=M.+E.+Samsonauthor=A.+W.+Maauthor=G.+X.+Xuauthor=T.+Pattersonauthor=D.+P.+Tuckauthor=J.+L.+Vinerauthor=A.+Fattaeyauthor=J.+Wang&title=Dual+HDAC+and+PI3K+Inhibitor+CUDC-907+Downregulates+MYC+and+Suppresses+Growth+of+MYC-Dependent+Cancers&doi=10.1158%2F1535-7163.MCT-16-0390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Dual HDAC and PI3K Inhibitor CUDC-907 Downregulates MYC and Suppresses Growth of MYC-dependent Cancers</span></div><div class="casAuthors">Sun, Kaiming; Atoyan, Ruzanna; Borek, Mylissa A.; Dellarocca, Steven; Samson, Maria Elena S.; Ma, Anna W.; Xu, Guang-Xin; Patterson, Troy; Tuck, David P.; Viner, Jaye L.; Fattaey, Ali; Wang, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">285-299</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Upregulation of MYC is a common driver event in human cancers, and some tumors depend on MYC to maintain transcriptional programs that promote cell growth and proliferation.  Preclin. studies have suggested that individually targeting upstream regulators of MYC, such as histone deacetylases (HDAC) and phosphoinositide 3-kinases (PI3K), can reduce MYC protein levels and suppress the growth of MYC-driven cancers.  Synergy between HDAC and PI3K inhibition in inducing cancer cell death has also been reported, but the involvement of MYC regulation is unclear.  In this study, we demonstrated that HDAC and PI3K inhibition synergistically downregulates MYC protein levels and induces apoptosis in "double-hit" (DH) diffuse large B-cell lymphoma (DLBCL) cells.  Furthermore, CUDC-907, a small-mol. dual-acting inhibitor of both class I and II HDACs and class I PI3Ks, effectively suppresses the growth and survival of MYC-altered or MYC-dependent cancer cells, such as DH DLBCL and BRD-NUT fusion-pos. NUT midline carcinoma (NMC) cells, and MYC protein downregulation is an early event induced by CUDC-907 treatment.  Consistently, the antitumor activity of CUDC-907 against multiple MYC-driven cancer types was also demonstrated in animal models, including DLBCL and NMC xenograft models, Myc transgenic tumor syngeneic models, and MYC-amplified solid tumor patient-derived xenograft (PDX) models.  Our findings suggest that dual function HDAC and PI3K inhibitor CUDC-907 is an effective agent targeting MYC and thus may be developed as potential therapy for MYC-dependent cancers.  Mol Cancer Ther; 16(2); 285-99. 2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo82AzF-jV397Vg90H21EOLACvtfcHk0ljSAbk7VyE2Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVertro%253D&md5=d4ffd5f39546fa076448dbfb36406183</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-16-0390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-16-0390%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DK.%26aulast%3DAtoyan%26aufirst%3DR.%26aulast%3DBorek%26aufirst%3DM.%2BA.%26aulast%3DDellarocca%26aufirst%3DS.%26aulast%3DSamson%26aufirst%3DM.%2BE.%26aulast%3DMa%26aufirst%3DA.%2BW.%26aulast%3DXu%26aufirst%3DG.%2BX.%26aulast%3DPatterson%26aufirst%3DT.%26aulast%3DTuck%26aufirst%3DD.%2BP.%26aulast%3DViner%26aufirst%3DJ.%2BL.%26aulast%3DFattaey%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDual%2520HDAC%2520and%2520PI3K%2520Inhibitor%2520CUDC-907%2520Downregulates%2520MYC%2520and%2520Suppresses%2520Growth%2520of%2520MYC-Dependent%2520Cancers%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2017%26volume%3D16%26spage%3D285%26epage%3D299%26doi%3D10.1158%2F1535-7163.MCT-16-0390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sermer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqualucci, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wendel, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span> <span> </span><span class="NLM_article-title">Emerging Epigenetic-Modulating Therapies in Lymphoma</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">494</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/s41571-019-0190-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fs41571-019-0190-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30837715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyjt77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=494-507&author=D.+Sermerauthor=L.+Pasqualucciauthor=H.+G.+Wendelauthor=A.+Melnickauthor=A.+Younes&title=Emerging+Epigenetic-Modulating+Therapies+in+Lymphoma&doi=10.1038%2Fs41571-019-0190-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging epigenetic-modulating therapies in lymphoma</span></div><div class="casAuthors">Sermer, David; Pasqualucci, Laura; Wendel, Hans-Guido; Melnick, Ari; Younes, Anas</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">494-507</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Despite considerable advances in the treatment of lymphoma, the prognosis of patients with relapsed and/or refractory disease continues to be poor; thus, a continued need exists for the development of novel approaches and therapies.  Epigenetic dysregulation might drive and/or promote tumorigenesis in various types of malignancies and is prevalent in both B cell and T cell lymphomas.  Over the past decade, a large no. of epigenetic-modifying agents have been developed and introduced into the clin. management of patients with haematol. malignancies.  In this Review, we provide a concise overview of the most promising epigenetic therapies for the treatment of lymphomas, including inhibitors of histone deacetylases (HDACs), DNA methyltransferases (DNMTs), enhancer of zeste homolog 2 (EZH2), bromodomain and extra-terminal domain proteins (BETs), protein arginine N-methyltransferases (PRMTs) and isocitrate dehydrogenases (IDHs), and highlight the most promising future directions of research in this area.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLSVtVzNWVu7Vg90H21EOLACvtfcHk0lgA_6YGVG7Aaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyjt77O&md5=5f7659b71bb5bc1abe3ebc43874b6bf6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fs41571-019-0190-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-019-0190-8%26sid%3Dliteratum%253Aachs%26aulast%3DSermer%26aufirst%3DD.%26aulast%3DPasqualucci%26aufirst%3DL.%26aulast%3DWendel%26aufirst%3DH.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26aulast%3DYounes%26aufirst%3DA.%26atitle%3DEmerging%2520Epigenetic-Modulating%2520Therapies%2520in%2520Lymphoma%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D494%26epage%3D507%26doi%3D10.1038%2Fs41571-019-0190-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Younes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdeja, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flinn, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerecitano, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neelapu, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clancy, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viner, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oki, Y.</span></span> <span> </span><span class="NLM_article-title">Safety, Tolerability, and Preliminary Activity of CUDC-907, a First-in-Class, Oral, Dual Inhibitor of HDAC and PI3K, in Patients with Relapsed or Refractory Lymphoma or Multiple Myeloma: An Open-Label, Dose-Escalation, Phase 1 Trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">622</span>– <span class="NLM_lpage">631</span>, <span class="refDoi"> DOI: 10.1016/S1470-2045(15)00584-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2FS1470-2045%2815%2900584-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27049457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOhs78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=622-631&author=A.+Younesauthor=J.+G.+Berdejaauthor=M.+R.+Patelauthor=I.+Flinnauthor=J.+F.+Gerecitanoauthor=S.+S.+Neelapuauthor=K.+R.+Kellyauthor=A.+R.+Copelandauthor=A.+Akinsauthor=M.+S.+Clancyauthor=L.+Gongauthor=J.+Wangauthor=A.+Maauthor=J.+L.+Vinerauthor=Y.+Oki&title=Safety%2C+Tolerability%2C+and+Preliminary+Activity+of+CUDC-907%2C+a+First-in-Class%2C+Oral%2C+Dual+Inhibitor+of+HDAC+and+PI3K%2C+in+Patients+with+Relapsed+or+Refractory+Lymphoma+or+Multiple+Myeloma%3A+An+Open-Label%2C+Dose-Escalation%2C+Phase+1+Trial&doi=10.1016%2FS1470-2045%2815%2900584-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Safety, tolerability, and preliminary activity of CUDC-907, a first-in-class, oral, dual inhibitor of HDAC and PI3K, in patients with relapsed or refractory lymphoma or multiple myeloma: an open-label, dose-escalation, phase 1 trial</span></div><div class="casAuthors">Younes, Anas; Berdeja, Jesus G.; Patel, Manish R.; Flinn, Ian; Gerecitano, John F.; Neelapu, Sattva S.; Kelly, Kevin R.; Copeland, Amanda R.; Akins, Amy; Clancy, Myles S.; Gong, Lucy; Wang, Jing; Ma, Anna; Viner, Jaye L.; Oki, Yasuhiro</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">622-631</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Treatment options for patients with relapsed or refractory lymphoma and multiple myeloma are limited.  CUDC-907 is an oral, first-in-class, small mol. that is designed to inhibit both histone deacetylase (HDAC) and PI3K enzymes, which are members of common oncogenic pathways in haematol. malignancies.  We aimed to assess overall safety and preliminary activity in this dose-escalation study of CUDC-907 monotherapy in patients with relapsed or refractory lymphoma and multiple myeloma.  This open-label, first-in-man, phase 1 trial recruited adult patients (aged ≥18 years) with lymphoma or multiple myeloma who were refractory to or had relapsed after two or more previous regimens, from four US cancer centers.  CUDC-907 was orally administered in a std. 3 + 3 dose-escalation design at four different dosing schedules, to which participants were sequentially assigned as follows: once daily, intermittently (twice or three times weekly; simultaneous enrolment), and daily for 5 days followed by a 2-day break (5/2), in 21-day cycles.  Dosing started at 30 mg for the once-daily schedule and 60 mg for other schedules, escalating in 30 mg increments.  Patients continued to receive CUDC-907 until disease progression or until other treatment discontinuation criteria were met.  The primary objective was to det. the max. tolerated dose (MTD) and recommended phase 2 dose, assessed in patients who received at least 66% of cycle 1 doses without modification and those who had a dose-limiting toxicity (DLT) in cycle 1 irresp. of dose modification.  We assessed safety in all patients who received at least one dose of study drug.  This ongoing trial is registered at ClinicalTrials.gov, no. NCT01742988.  Between Jan 23, 2013, and July 27, 2015, we enrolled 44 patients, of whom ten were sequentially assigned to CUDC-907 once-daily (MTD 60 mg), 12 to twice-weekly (MTD 150 mg), 15 to three-times-weekly (MTD 150 mg), and seven to the 5/2 dosing schedule (MTD 60 mg). 37 (84%) patients had discontinued study drug as a result of progressive disease or clin. signs of progressive disease at the data cutoff.  Four DLTs occurred in three of 40 DLT-evaluable patients (diarrhoea and hyperglycemia in one patient on 60 mg once daily, hyperglycemia in one patient on 150 mg twice weekly, and diarrhoea in one patient on 150 mg three times weekly); no DLTs were reported in patients on the 5/2 schedule.  Grade 3 or worse adverse events occurred in 19 (43%) of 44 patients, the most common of which were thrombocytopenia (in nine [20%] of 44 patients), neutropenia (three [7%]), and hyperglycemia (three [7%]). 11 (25%) of 44 patients had serious adverse events, three of which were regarded as treatment related (epistaxis and the DLTs of diarrhoea and hyperglycemia).  Adverse events led to dose redns. in six (14%) patients and treatment discontinuation in seven (16%).  Five (14%) of 37 response-evaluable patients achieved an objective response (two complete responses and three partial responses).  All five responses occurred in the subgroup of patients with diffuse large B-cell lymphoma (DLBCL; n=9), and three occurred in those with transformed follicular lymphoma DLBCL (n=5). 21 (57%) of 37 response-evaluable patients had stable disease, including those with DLBCL, Hodgkin's lymphoma, and multiple myeloma.  On the basis of these findings, we selected CUDC-907 60 mg on the 5/2 dosing schedule as the recommended phase 2 dose.  The safety and tolerability profile of CUDC-907 and the promising preliminary evidence of response support continued development of CUDC-907 at the 60 mg 5/2 dosing schedule, alone and in combination with other therapies.  A dose-expansion trial of this dose in patients with refractory and relapsed DLBCL in particular, is ongoing.  Curis, Inc, and the Leukemia and Lymphoma Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAFEt7vSuwM7Vg90H21EOLACvtfcHk0lgA_6YGVG7Aaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOhs78%253D&md5=b759c77ac239b6ae0052d30c9bea3221</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2815%2900584-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252815%252900584-7%26sid%3Dliteratum%253Aachs%26aulast%3DYounes%26aufirst%3DA.%26aulast%3DBerdeja%26aufirst%3DJ.%2BG.%26aulast%3DPatel%26aufirst%3DM.%2BR.%26aulast%3DFlinn%26aufirst%3DI.%26aulast%3DGerecitano%26aufirst%3DJ.%2BF.%26aulast%3DNeelapu%26aufirst%3DS.%2BS.%26aulast%3DKelly%26aufirst%3DK.%2BR.%26aulast%3DCopeland%26aufirst%3DA.%2BR.%26aulast%3DAkins%26aufirst%3DA.%26aulast%3DClancy%26aufirst%3DM.%2BS.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DViner%26aufirst%3DJ.%2BL.%26aulast%3DOki%26aufirst%3DY.%26atitle%3DSafety%252C%2520Tolerability%252C%2520and%2520Preliminary%2520Activity%2520of%2520CUDC-907%252C%2520a%2520First-in-Class%252C%2520Oral%252C%2520Dual%2520Inhibitor%2520of%2520HDAC%2520and%2520PI3K%252C%2520in%2520Patients%2520with%2520Relapsed%2520or%2520Refractory%2520Lymphoma%2520or%2520Multiple%2520Myeloma%253A%2520An%2520Open-Label%252C%2520Dose-Escalation%252C%2520Phase%25201%2520Trial%26jtitle%3DLancet%2520Oncol.%26date%3D2016%26volume%3D17%26spage%3D622%26epage%3D631%26doi%3D10.1016%2FS1470-2045%2815%2900584-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, R. R.</span></span> <span> </span><span class="NLM_article-title">Safety and Tolerability of Histone Deacetylase (HDAC) Inhibitors in Oncology</span>. <i>Drug Saf.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1007/s40264-018-0773-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1007%2Fs40264-018-0773-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30649740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvFOqtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2019&pages=235-245&author=R.+R.+Shah&title=Safety+and+Tolerability+of+Histone+Deacetylase+%28HDAC%29+Inhibitors+in+Oncology&doi=10.1007%2Fs40264-018-0773-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Safety and Tolerability of Histone Deacetylase (HDAC) Inhibitors in Oncology</span></div><div class="casAuthors">Shah, Rashmi R.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Safety</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">235-245</span>CODEN:
                <span class="NLM_cas:coden">DRSAEA</span>;
        ISSN:<span class="NLM_cas:issn">0114-5916</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) are expressed at increased levels in cells of various malignancies, and the use of HDAC inhibitors has improved outcomes in patients with haematol. malignancies (T-cell lymphomas and multiple myeloma).  However, they are not as effective in solid tumors.  Five agents are currently approved under various jurisdictions, namely belinostat, chidamide, panobinostat, romidepsin and vorinostat.  These agents are assocd. with a range of class-related and agent-specific serious and/or severe adverse effects, notably myelosuppression, diarrhoea and various cardiac effects.  Among the cardiac effects are ST-T segment abnormalities and QTc interval prolongation of the ECG, isolated cases of atrial fibrillation and, in rare instances, ventricular tachyarrhythmias.  In order to improve the safety profile of this class of drugs as well as their efficacy in indications already approved and to further widen their indications, a large no. of newer HDAC inhibitors with varying degrees of HDAC isoform selectivity have been synthesized and are currently under clin. development.  Preliminary evidence from early studies suggests that they may be effective in non-haematol. cancers as well when used in combination with other therapeutic modalities, but that they too appear to be assocd. with the above class-related adverse effects.  As the database accumulates, the safety, efficacy and risk/benefit of the newer agents and their indications will become clearer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_FzvurFGlNLVg90H21EOLACvtfcHk0lgA_6YGVG7Aaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvFOqtrY%253D&md5=db23e29ac410c716393aae079abf18cd</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1007%2Fs40264-018-0773-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40264-018-0773-9%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DR.%2BR.%26atitle%3DSafety%2520and%2520Tolerability%2520of%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Inhibitors%2520in%2520Oncology%26jtitle%3DDrug%2520Saf.%26date%3D2019%26volume%3D42%26spage%3D235%26epage%3D245%26doi%3D10.1007%2Fs40264-018-0773-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liclican, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagpacan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papalia, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samuel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span> <span> </span><span class="NLM_article-title">Structural, Biochemical, and Biophysical Characterization of Idelalisib Binding to Phosphoinositide 3-Kinase Delta</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">8439</span>– <span class="NLM_lpage">8446</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.634683</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1074%2Fjbc.M114.634683" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=25631052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsVSqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=8439-8446&author=J.+R.+Somozaauthor=D.+Koditekauthor=A.+G.+Villasenorauthor=N.+Novikovauthor=M.+H.+Wongauthor=A.+Liclicanauthor=W.+Xingauthor=L.+Lagpacanauthor=R.+Wangauthor=B.+E.+Schultzauthor=G.+A.+Papaliaauthor=D.+Samuelauthor=L.+Ladauthor=M.+E.+McGrath&title=Structural%2C+Biochemical%2C+and+Biophysical+Characterization+of+Idelalisib+Binding+to+Phosphoinositide+3-Kinase+Delta&doi=10.1074%2Fjbc.M114.634683"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Structural, Biochemical, and Biophysical Characterization of Idelalisib Binding to Phosphoinositide 3-Kinase δ</span></div><div class="casAuthors">Somoza, John R.; Koditek, David; Villasenor, Armando G.; Novikov, Nikolai; Wong, Melanie H.; Liclican, Albert; Xing, Weimei; Lagpacan, Leanna; Wang, Ruth; Schultz, Brian E.; Papalia, Giuseppe A.; Samuel, Dharmaraj; Lad, Latesh; McGrath, Mary E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">8439-8446</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Idelalisib (also known as GS-1101, CAL-101, IC489666, and Zydelig) is a PI3Kδ inhibitor that has recently been approved for the treatment of several hematol. malignancies.  Given its use in human diseases, we needed a clear picture of how idelalisib binds to and inhibits PI3Kδ.  Our data show that idelalisib is a potent and selective inhibitor of the kinase activity of PI3Kδ.  A kinetic characterization clearly demonstrated ATP-competitive inhibition, and several addnl. biochem. and biophys. assays showed that the compd. binds reversibly and noncovalently to the kinase.  A crystal structure of idelalisib bound to the p110δ subunit of PI3Kδ furthers our understanding of the binding interactions that confer the potency and selectivity of idelalisib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnXZo8_qK79rVg90H21EOLACvtfcHk0lgTEKpC43P-YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsVSqtrs%253D&md5=2969b7407f4edc367249215bbbf22a0e</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.634683&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.634683%26sid%3Dliteratum%253Aachs%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DNovikov%26aufirst%3DN.%26aulast%3DWong%26aufirst%3DM.%2BH.%26aulast%3DLiclican%26aufirst%3DA.%26aulast%3DXing%26aufirst%3DW.%26aulast%3DLagpacan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DPapalia%26aufirst%3DG.%2BA.%26aulast%3DSamuel%26aufirst%3DD.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26atitle%3DStructural%252C%2520Biochemical%252C%2520and%2520Biophysical%2520Characterization%2520of%2520Idelalisib%2520Binding%2520to%2520Phosphoinositide%25203-Kinase%2520Delta%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D8439%26epage%3D8446%26doi%3D10.1074%2Fjbc.M114.634683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lauffer, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mintzer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tam, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zilberleyb, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flicke, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ritscher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorowicz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vallero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortwine, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunzner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modrusan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koth, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupardus, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaminker, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, P.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase (HDAC) Inhibitor Kinetic Rate Constants Correlate with Cellular Histone Acetylation but Not Transcription and Cell Viability</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>288</i></span>,  <span class="NLM_fpage">26926</span>– <span class="NLM_lpage">26943</span>, <span class="refDoi"> DOI: 10.1074/jbc.M113.490706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1074%2Fjbc.M113.490706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=23897821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVClt7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=26926-26943&author=B.+E.+Laufferauthor=R.+Mintzerauthor=R.+Fongauthor=S.+Mukundauthor=C.+Tamauthor=I.+Zilberleybauthor=B.+Flickeauthor=A.+Ritscherauthor=G.+Fedorowiczauthor=R.+Valleroauthor=D.+F.+Ortwineauthor=J.+Gunznerauthor=Z.+Modrusanauthor=L.+Neumannauthor=C.+M.+Kothauthor=P.+J.+Lupardusauthor=J.+S.+Kaminkerauthor=C.+E.+Heiseauthor=P.+Steiner&title=Histone+Deacetylase+%28HDAC%29+Inhibitor+Kinetic+Rate+Constants+Correlate+with+Cellular+Histone+Acetylation+but+Not+Transcription+and+Cell+Viability&doi=10.1074%2Fjbc.M113.490706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Deacetylase (HDAC) Inhibitor Kinetic Rate Constants Correlate with Cellular Histone Acetylation but Not Transcription and Cell Viability</span></div><div class="casAuthors">Lauffer, Benjamin E. L.; Mintzer, Robert; Fong, Rina; Mukund, Susmith; Tam, Christine; Zilberleyb, Inna; Flicke, Birgit; Ritscher, Allegra; Fedorowicz, Grazyna; Vallero, Roxanne; Ortwine, Daniel F.; Gunzner, Janet; Modrusan, Zora; Neumann, Lars; Koth, Christopher M.; Lupardus, Patrick J.; Kaminker, Joshua S.; Heise, Christopher E.; Steiner, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">26926-26943</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Histone deacetylases (HDACs) are crit. in the control of gene expression, and dysregulation of their activity has been implicated in a broad range of diseases, including cancer, cardiovascular, and neurol. diseases.  HDAC inhibitors (HDACi) employing different zinc chelating functionalities such as hydroxamic acids and benzamides have shown promising results in cancer therapy.  Although it has also been suggested that HDACi with increased isoenzyme selectivity and potency may broaden their clin. utility and minimize side effects, the translation of this idea to the clinic remains to be investigated.  Moreover, a detailed understanding of how HDACi with different pharmacol. properties affect biol. functions in vitro and in vivo is still missing.  Here, we show that a panel of benzamide-contg. HDACi are slow tight-binding inhibitors with long residence times unlike the hydroxamate-contg. HDACi vorinostat and trichostatin-A.  Characterization of changes in H2BK5 and H4K14 acetylation following HDACi treatment in the neuroblastoma cell line SH-SY5Y revealed that the timing and magnitude of histone acetylation mirrored both the assocn. and dissocn. kinetic rates of the inhibitors.  In contrast, cell viability and microarray gene expression anal. indicated that cell death induction and changes in transcriptional regulation do not correlate with the dissocn. kinetic rates of the HDACi.  Therefore, our study suggests that detg. how the selective and kinetic inhibition properties of HDACi affect cell function will help to evaluate their therapeutic utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2goaOx8CDGbVg90H21EOLACvtfcHk0lgTEKpC43P-YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVClt7jM&md5=af1074c79e8abd932313f4ae13333a98</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.490706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.490706%26sid%3Dliteratum%253Aachs%26aulast%3DLauffer%26aufirst%3DB.%2BE.%26aulast%3DMintzer%26aufirst%3DR.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DMukund%26aufirst%3DS.%26aulast%3DTam%26aufirst%3DC.%26aulast%3DZilberleyb%26aufirst%3DI.%26aulast%3DFlicke%26aufirst%3DB.%26aulast%3DRitscher%26aufirst%3DA.%26aulast%3DFedorowicz%26aufirst%3DG.%26aulast%3DVallero%26aufirst%3DR.%26aulast%3DOrtwine%26aufirst%3DD.%2BF.%26aulast%3DGunzner%26aufirst%3DJ.%26aulast%3DModrusan%26aufirst%3DZ.%26aulast%3DNeumann%26aufirst%3DL.%26aulast%3DKoth%26aufirst%3DC.%2BM.%26aulast%3DLupardus%26aufirst%3DP.%2BJ.%26aulast%3DKaminker%26aufirst%3DJ.%2BS.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DSteiner%26aufirst%3DP.%26atitle%3DHistone%2520Deacetylase%2520%2528HDAC%2529%2520Inhibitor%2520Kinetic%2520Rate%2520Constants%2520Correlate%2520with%2520Cellular%2520Histone%2520Acetylation%2520but%2520Not%2520Transcription%2520and%2520Cell%2520Viability%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D26926%26epage%3D26943%26doi%3D10.1074%2Fjbc.M113.490706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span> <span> </span><span class="NLM_article-title">Zinc Binding Groups for Histone Deacetylase Inhibitors</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">714</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1080/14756366.2017.1417274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1080%2F14756366.2017.1417274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29616828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXntVGhtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=714-721&author=L.+Zhangauthor=J.+Zhangauthor=Q.+Jiangauthor=L.+Zhangauthor=W.+Song&title=Zinc+Binding+Groups+for+Histone+Deacetylase+Inhibitors&doi=10.1080%2F14756366.2017.1417274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Zinc binding groups for histone deacetylase inhibitors</span></div><div class="casAuthors">Zhang, Lei; Zhang, Jian; Jiang, Qixiao; Zhang, Li; Song, Weiguo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">714-721</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Zinc binding groups (ZBGs) play a crucial role in targeting histone deacetylase inhibitors (HDACIs) to the active site of histone deacetylases (HDACs), thus detg. the potency of HDACIs.  Due to the high affinity to the zinc ion, hydroxamic acid is the most commonly used ZBG in the structure of HDACs.  An alternative ZBG is benzamide group, which features excellent inhibitory selectivity for class I HDACs.  Various ZBGs have been designed and tested to improve the activity and selectivity of HDACIs, and to overcome the pharmacokinetic limitations of current HDACIs.  Herein, different kinds of ZBGs are reviewed and their features have been discussed for further design of HDACIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXFywPdICZVrVg90H21EOLACvtfcHk0lgTEKpC43P-YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXntVGhtrc%253D&md5=771d801a575618430f1cdcd03044dbdc</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1080%2F14756366.2017.1417274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14756366.2017.1417274%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DW.%26atitle%3DZinc%2520Binding%2520Groups%2520for%2520Histone%2520Deacetylase%2520Inhibitors%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2018%26volume%3D33%26spage%3D714%26epage%3D721%26doi%3D10.1080%2F14756366.2017.1417274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">6992</span>– <span class="NLM_lpage">7014</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00390</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00390" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=6992-7014&author=K.+Zhangauthor=F.+Laiauthor=S.+Linauthor=M.+Jiauthor=J.+Zhangauthor=Y.+Zhangauthor=J.+Jinauthor=R.+Fuauthor=D.+Wuauthor=H.+Tianauthor=N.+Xueauthor=L.+Shengauthor=X.+Zouauthor=Y.+Liauthor=X.+Chenauthor=H.+Xu&title=Design%2C+Synthesis%2C+and+Biological+Evaluation+of+4-Methyl+Quinazoline+Derivatives+as+Anticancer+Agents+Simultaneously+Targeting+Phosphoinositide+3-Kinases+and+Histone+Deacetylases&doi=10.1021%2Facs.jmedchem.9b00390"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of 4-Methyl Quinazoline Derivatives as Anticancer Agents Simultaneously Targeting Phosphoinositide 3-Kinases and Histone Deacetylases</span></div><div class="casAuthors">Zhang, Kehui; Lai, Fangfang; Lin, Songwen; Ji, Ming; Zhang, Jingbo; Zhang, Yan; Jin, Jing; Fu, Rong; Wu, Deyu; Tian, Hua; Xue, Nina; Sheng, Li; Zou, Xiaowen; Li, Yan; Chen, Xiaoguang; Xu, Heng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6992-7014</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polypharmacol. is a promising paradigm in modern drug discovery.  Herein, we have discovered a series of novel PI3K and HDAC dual inhibitors in which the hydroxamic acid moiety as the zinc binding functional group was introduced to a quinazoline-based PI3K pharmacophore through an appropriate linker.  Systematic structure-activity relationship studies resulted in lead compds. 23 (shown in graphical abstr. and duplicated as I) and 36 (II) that simultaneously inhibited PI3K and HDAC with nanomolar potencies and demonstrated favorable antiproliferative activities.  Compds. 23 and 36 efficiently modulated the expression of p-AKT and Ac-H3, arrested the cell cycle, and induced apoptosis in HCT116 cancer cells.  Following pharmacokinetic studies, 23 was further evaluated in HCT116 and HGC-27 xenograft models to show significant in vivo anticancer efficacies with tumor growth inhibitions of 45.8% (po, 150 mg/kg) and 62.6% (i.p., 30 mg/kg), resp.  Overall, this work shows promise in discovering new anticancer therapeutics by the approach of simultaneously targeting PI3K and HDAC pathways with a single mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6xLiFaGhwk7Vg90H21EOLACvtfcHk0li75q52-kkkcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFamtLg%253D&md5=de4c9a630208e49eac58dcad630adcd6</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00390&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00390%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DF.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DTian%26aufirst%3DH.%26aulast%3DXue%26aufirst%3DN.%26aulast%3DSheng%26aufirst%3DL.%26aulast%3DZou%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DH.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Biological%2520Evaluation%2520of%25204-Methyl%2520Quinazoline%2520Derivatives%2520as%2520Anticancer%2520Agents%2520Simultaneously%2520Targeting%2520Phosphoinositide%25203-Kinases%2520and%2520Histone%2520Deacetylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D6992%26epage%3D7014%26doi%3D10.1021%2Facs.jmedchem.9b00390" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mustafa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong-Thi, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. X. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanujulu, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chng, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yen, J. J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dymock, B. W.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Ligand Efficient Dual Inhibitors of Janus Kinase (JAK) and Histone Deacetylase (HDAC) Based on Ruxolitinib and Vorinostat</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8336</span>– <span class="NLM_lpage">8357</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00678</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00678" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8336-8357&author=L.+Yaoauthor=N.+Mustafaauthor=E.+C.+Tanauthor=A.+Poulsenauthor=P.+Singhauthor=M.+D.+Duong-Thiauthor=J.+X.+T.+Leeauthor=P.+M.+Ramanujuluauthor=W.+J.+Chngauthor=J.+J.+Y.+Yenauthor=S.+Ohlsonauthor=B.+W.+Dymock&title=Design+and+Synthesis+of+Ligand+Efficient+Dual+Inhibitors+of+Janus+Kinase+%28JAK%29+and+Histone+Deacetylase+%28HDAC%29+Based+on+Ruxolitinib+and+Vorinostat&doi=10.1021%2Facs.jmedchem.7b00678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Ligand Efficient Dual Inhibitors of Janus Kinase (JAK) and Histone Deacetylase (HDAC) Based on Ruxolitinib and Vorinostat</span></div><div class="casAuthors">Yao, Lianbin; Mustafa, Nurulhuda; Tan, Eng Chong; Poulsen, Anders; Singh, Prachi; Duong-Thi, Minh-Dao; Lee, Jeannie X. T.; Ramanujulu, Pondy Murugappan; Chng, Wee Joo; Yen, Jeffrey J. Y.; Ohlson, Sten; Dymock, Brian W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8336-8357</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple oncogenic pathways is a desirable goal in cancer therapy.  To achieve such an outcome with a single mol. would simplify treatment regimes.  Herein the core features of ruxolitinib, a marketed JAK1/2 inhibitor, have been merged with the HDAC inhibitor vorinostat, leading to new mols. that are bispecific targeted JAK/HDAC inhibitors.  A preferred pyrazole substituted pyrrolopyrimidine, I (R = H), inhibits JAK1 and HDACs 1, 2, 3, 6, and 10 with IC50 values of less than 20 nM, is <100 nM potent against JAK2 and HDAC11, and is selective for the JAK family against a panel of 97 kinases.  Broad cellular antiproliferative potency of I (R = H) is supported by demonstration of JAK-STAT and HDAC pathway blockade in hematol. cell lines.  Me analog I (R = Me) has an even more selective profile.  This study provides new leads for assessment of JAK and HDAC pathway dual inhibition achieved with a single mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocWx3JXIW8YbVg90H21EOLACvtfcHk0li75q52-kkkcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOjtLzJ&md5=5ac471ee0cc413b08b926803790de7f9</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00678%26sid%3Dliteratum%253Aachs%26aulast%3DYao%26aufirst%3DL.%26aulast%3DMustafa%26aufirst%3DN.%26aulast%3DTan%26aufirst%3DE.%2BC.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DSingh%26aufirst%3DP.%26aulast%3DDuong-Thi%26aufirst%3DM.%2BD.%26aulast%3DLee%26aufirst%3DJ.%2BX.%2BT.%26aulast%3DRamanujulu%26aufirst%3DP.%2BM.%26aulast%3DChng%26aufirst%3DW.%2BJ.%26aulast%3DYen%26aufirst%3DJ.%2BJ.%2BY.%26aulast%3DOhlson%26aufirst%3DS.%26aulast%3DDymock%26aufirst%3DB.%2BW.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Ligand%2520Efficient%2520Dual%2520Inhibitors%2520of%2520Janus%2520Kinase%2520%2528JAK%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Based%2520on%2520Ruxolitinib%2520and%2520Vorinostat%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8336%26epage%3D8357%26doi%3D10.1021%2Facs.jmedchem.7b00678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of Janus Kinase 2 (JAK2) and Histone Deacetylase (HDAC) Dual Inhibitors as a Novel Strategy for the Combinational Treatment of Leukemia and Invasive Fungal Infections</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6056</span>– <span class="NLM_lpage">6074</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00393</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00393" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Snu7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6056-6074&author=Y.+Huangauthor=G.+Dongauthor=H.+Liauthor=N.+Liuauthor=W.+Zhangauthor=C.+Sheng&title=Discovery+of+Janus+Kinase+2+%28JAK2%29+and+Histone+Deacetylase+%28HDAC%29+Dual+Inhibitors+as+a+Novel+Strategy+for+the+Combinational+Treatment+of+Leukemia+and+Invasive+Fungal+Infections&doi=10.1021%2Facs.jmedchem.8b00393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Janus Kinase 2 (JAK2) and Histone Deacetylase (HDAC) Dual Inhibitors as a Novel Strategy for the Combinational Treatment of Leukemia and Invasive Fungal Infections</span></div><div class="casAuthors">Huang, Yahui; Dong, Guoqiang; Li, Huanqiu; Liu, Na; Zhang, Wannian; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6056-6074</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Clin., leukemia patients often suffer from the limited efficacy of chemotherapy and high risks of infection by invasive fungal pathogens.  Herein, a novel therapeutic strategy was developed in which a small mol. can simultaneously treat leukemia and invasive fungal infections (IFIs).  Novel Janus kinase 2 (JAK2) and histone deacetylase (HDAC) dual inhibitors were identified to possess potent anti-proliferative activity toward hematol. cell lines and excellent synergistic effects with fluconazole to treat resistant Candida albicans infections.  In particular, compd. I, a highly active and selective JAK2/HDAC6 dual inhibitor, showed excellent in vivo antitumor efficacy in several acute myeloid leukemia (AML) models and synergized with fluconazole for the treatment of resistant C. albicans infections.  This study highlights the therapeutic potential of JAK2/HDAC dual inhibitors in treating AML and IFIs and provides an efficient strategy for multitargeting drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3_ii7EQG1qLVg90H21EOLACvtfcHk0lhKJvZ90FzTqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Snu7rN&md5=ba7cf74761120f0526ab520196e3dded</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00393%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520Janus%2520Kinase%25202%2520%2528JAK2%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Dual%2520Inhibitors%2520as%2520a%2520Novel%2520Strategy%2520for%2520the%2520Combinational%2520Treatment%2520of%2520Leukemia%2520and%2520Invasive%2520Fungal%2520Infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6056%26epage%3D6074%26doi%3D10.1021%2Facs.jmedchem.8b00393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krieger, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groll, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osko, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ressing, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahlert, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borkhardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurz, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, F. K.</span></span> <span> </span><span class="NLM_article-title">Discovery of the First-in-Class Dual Histone Deacetylase-Proteasome Inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">10299</span>– <span class="NLM_lpage">10309</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01487</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01487" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=10299-10309&author=S.+Bhatiaauthor=V.+Kriegerauthor=M.+Grollauthor=J.+D.+Oskoauthor=N.+Ressingauthor=H.+Ahlertauthor=A.+Borkhardtauthor=T.+Kurzauthor=D.+W.+Christiansonauthor=J.+Hauerauthor=F.+K.+Hansen&title=Discovery+of+the+First-in-Class+Dual+Histone+Deacetylase-Proteasome+Inhibitor&doi=10.1021%2Facs.jmedchem.8b01487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of the First-in-Class Dual Histone Deacetylase-Proteasome Inhibitor</span></div><div class="casAuthors">Bhatia, Sanil; Krieger, Viktoria; Groll, Michael; Osko, Jeremy D.; Ressing, Nina; Ahlert, Heinz; Borkhardt, Arndt; Kurz, Thomas; Christianson, David W.; Hauer, Julia; Hansen, Finn K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10299-10309</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual- or multitarget drugs have emerged as a promising alternative to combination therapies.  Proteasome inhibitors (PIs) possess synergistic activity with histone deacetylase (HDAC) inhibitors due to the simultaneous blockage of the ubiquitin degrdn. and aggresome pathways.  Here, the authors present the design, synthesis, binding modes, and anticancer properties of RTS-V5 as the first-in-class dual HDAC-proteasome ligand.  The inhibition of both targets was confirmed by biochem. and cellular assays as well as x-ray crystal structures of the 20S proteasome and HDAC6 complexed with RTS-V5.  Cytotoxicity assays with leukemia and multiple myeloma cell lines as well as therapy refractory primary patient-derived leukemia cells demonstrated that RTS-V5 possesses potent and selective anticancer activity.  The results will thus guide the structure-based optimization of dual HDAC-proteasome inhibitors for the treatment of hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwJMqcDlzXA7Vg90H21EOLACvtfcHk0lhKJvZ90FzTqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVWrur7K&md5=7d47e664bd77ac1bc5ebd39ea6eed02a</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01487%26sid%3Dliteratum%253Aachs%26aulast%3DBhatia%26aufirst%3DS.%26aulast%3DKrieger%26aufirst%3DV.%26aulast%3DGroll%26aufirst%3DM.%26aulast%3DOsko%26aufirst%3DJ.%2BD.%26aulast%3DRessing%26aufirst%3DN.%26aulast%3DAhlert%26aufirst%3DH.%26aulast%3DBorkhardt%26aufirst%3DA.%26aulast%3DKurz%26aufirst%3DT.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26aulast%3DHauer%26aufirst%3DJ.%26aulast%3DHansen%26aufirst%3DF.%2BK.%26atitle%3DDiscovery%2520of%2520the%2520First-in-Class%2520Dual%2520Histone%2520Deacetylase-Proteasome%2520Inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D10299%26epage%3D10309%26doi%3D10.1021%2Facs.jmedchem.8b01487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Small Molecules Simultaneously Inhibiting P53-Murine Double Minute 2 (MDM2) Interaction and Histone Deacetylases (HDACs): Discovery of Novel Multitargeting Antitumor Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7245</span>– <span class="NLM_lpage">7260</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00664</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00664" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlykur3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7245-7260&author=S.+Heauthor=G.+Dongauthor=S.+Wuauthor=K.+Fangauthor=Z.+Miaoauthor=W.+Wangauthor=C.+Sheng&title=Small+Molecules+Simultaneously+Inhibiting+P53-Murine+Double+Minute+2+%28MDM2%29+Interaction+and+Histone+Deacetylases+%28HDACs%29%3A+Discovery+of+Novel+Multitargeting+Antitumor+Agents&doi=10.1021%2Facs.jmedchem.8b00664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Simultaneously Inhibiting p53-Murine Double Minute 2 (MDM2) Interaction and Histone Deacetylases (HDACs): Discovery of Novel Multitargeting Antitumor Agents</span></div><div class="casAuthors">He, Shipeng; Dong, Guoqiang; Wu, Shanchao; Fang, Kun; Miao, Zhenyuan; Wang, Wei; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7245-7260</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P53-Murine double minute 2 (MDM2) interaction and histone deacetylases (HDACs) are important targets in antitumor drug development.  Inspired by the synergistic effects between MDM2 and HDACs, the first MDM2/HDACs dual inhibitors were identified, which showed excellent activities against both targets.  In particular, compd. 14d was proven to be a potent and orally active MDM2/HDAC dual inhibitor, whose antitumor mechanisms were validated in cancer cells.  Compd. 14d showed excellent in vivo antitumor potency in the A549 xenograft model, providing a promising lead compd. for the development of novel antitumor agents.  Also, this proof-of-concept study offers a novel and efficient strategy for multitargeting antitumor drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx94Uvm3pP8bVg90H21EOLACvtfcHk0lhKJvZ90FzTqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlykur3L&md5=68871e41ecb786e135b01742e59eec85</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00664%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DFang%26aufirst%3DK.%26aulast%3DMiao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DSmall%2520Molecules%2520Simultaneously%2520Inhibiting%2520P53-Murine%2520Double%2520Minute%25202%2520%2528MDM2%2529%2520Interaction%2520and%2520Histone%2520Deacetylases%2520%2528HDACs%2529%253A%2520Discovery%2520of%2520Novel%2520Multitargeting%2520Antitumor%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7245%26epage%3D7260%26doi%3D10.1021%2Facs.jmedchem.8b00664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Pazopanib-Based HDAC and VEGFR Dual Inhibitors Targeting Cancer Epigenetics and Angiogenesis Simultaneously</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">5304</span>– <span class="NLM_lpage">5322</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00384</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00384" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXpvVGrsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=5304-5322&author=J.+Zangauthor=X.+Liangauthor=Y.+Huangauthor=Y.+Jiaauthor=X.+Liauthor=W.+Xuauthor=C.+J.+Chouauthor=Y.+Zhang&title=Discovery+of+Novel+Pazopanib-Based+HDAC+and+VEGFR+Dual+Inhibitors+Targeting+Cancer+Epigenetics+and+Angiogenesis+Simultaneously&doi=10.1021%2Facs.jmedchem.8b00384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Pazopanib-Based HDAC and VEGFR Dual Inhibitors Targeting Cancer Epigenetics and Angiogenesis Simultaneously</span></div><div class="casAuthors">Zang, Jie; Liang, Xuewu; Huang, Yongxue; Jia, Yuping; Li, Xiaoyang; Xu, Wenfang; Chou, C. James; Zhang, Yingjie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5304-5322</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Herein a novel series of pazopanib hybrids as polypharmacol. antitumor agents were developed based on the crosstalk between histone deacetylases (HDACs) and vascular endothelial growth factor (VEGF) pathway.  Among them, one ortho-aminoanilide 6d and one hydroxamic acid 13f exhibited considerable total HDACs and VEGFR-2 inhibitory activities.  The HDAC inhibitory activities endowed 6d and 13f with potent antiproliferative activities, which was not obsd. in the approved VEGFR inhibitor pazopanib.  Compds. 6d and 13f possessed comparable HDAC isoform selectivity profiles to the clin. class I HDAC inhibitor MS-275 and the approved pan-HDAC inhibitor SAHA, resp. 6d and 13f also exhibited uncompromised multiple tyrosine kinases inhibitory activities relative to pazopanib.  The intracellular dual inhibition to HDAC and VEGFR of 6d and 13f was validated by Western blot anal.  In both HUVECs tube formation assay and rat thoracic aorta rings assay, 6d and 13f showed comparable antiangiogenic potencies to pazopanib.  What's more, 6d possessed desirable pharmacokinetic profiles with the oral bioavailability of 72% in SD rats and considerable in vivo antitumor efficacy in a human colorectal adenocarcinoma (HT-29) xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpmz6rwKiKSbVg90H21EOLACvtfcHk0lghI5sh3V8e2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXpvVGrsL8%253D&md5=437da0a6cca1ec211219aef87a6ddbc5</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00384%26sid%3Dliteratum%253Aachs%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DChou%26aufirst%3DC.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Novel%2520Pazopanib-Based%2520HDAC%2520and%2520VEGFR%2520Dual%2520Inhibitors%2520Targeting%2520Cancer%2520Epigenetics%2520and%2520Angiogenesis%2520Simultaneously%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D5304%26epage%3D5322%26doi%3D10.1021%2Facs.jmedchem.8b00384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, R.</span></span> <span> </span><span class="NLM_article-title">Discovery of N1-(4-((7-Cyclopentyl-6-(Dimethylcarbamoyl)-7H-Pyrrolo[2,3-d]Pyrimidin-2-yl)Amino)Phenyl)- N8-Hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-Dependent Kinase 4/9 (CDK4/9) and Histone Deacetlyase1 (HDAC1) against Malignant Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">3166</span>– <span class="NLM_lpage">3192</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00209</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00209" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=3166-3192&author=Y.+Liauthor=X.+Luoauthor=Q.+Guoauthor=Y.+Nieauthor=T.+Wangauthor=C.+Zhangauthor=Z.+Huangauthor=X.+Wangauthor=Y.+Liuauthor=Y.+Chenauthor=J.+Zhengauthor=S.+Yangauthor=Y.+Fanauthor=R.+Xiang&title=Discovery+of+N1-%284-%28%287-Cyclopentyl-6-%28Dimethylcarbamoyl%29-7H-Pyrrolo%5B2%2C3-d%5DPyrimidin-2-yl%29Amino%29Phenyl%29-+N8-Hydroxyoctanediamide+as+a+Novel+Inhibitor+Targeting+Cyclin-Dependent+Kinase+4%2F9+%28CDK4%2F9%29+and+Histone+Deacetlyase1+%28HDAC1%29+against+Malignant+Cancer&doi=10.1021%2Facs.jmedchem.8b00209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide as a Novel Inhibitor Targeting Cyclin-dependent Kinase 4/9 (CDK4/9) and Histone Deacetylase 1 (HDAC1) against Malignant Cancer</span></div><div class="casAuthors">Li, Yongtao; Luo, Xiaohe; Guo, Qingxiang; Nie, Yongwei; Wang, Tianqi; Zhang, Chao; Huang, Zhi; Wang, Xin; Liu, Yanhua; Chen, Yanan; Zheng, Jianyu; Yang, Shengyong; Fan, Yan; Xiang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3166-3192</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of highly potent, selective inhibitors targeting both CDK4/9 and HDAC1 have been designed and synthesized.  N1-(4-((7-Cyclopentyl-6-(dimethylcarbamoyl)-7H-pyrrolo[2,3-d]pyrimidin-2-yl)amino)phenyl)-N8-hydroxyoctanediamide (I) was discovered.  The lead compd. I with excellent CDK4/9 and HDAC1 inhibitory activity of IC50 = 8.8, 12, and 2.2 nM, resp., can effectively induce apoptosis of cancer cell lines.  The kinase profiling of compd. I showed excellent selectivity and specificity.  Compd. I induces G2/M arrest in high concn. and G0/G1 arrest in low concn. to prevent the proliferation and differentiation of cancer cells.  Mice bared-breast cancer treated with I showed significant antitumor efficacy.  The insight into mechanisms of I indicated that it could induce cancer cell death via cell apoptosis based on CDK4/9 and HDAC1 repression and phosphorylation of p53.  The data demonstrated the compd. I could be a promising drug candidate for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4B7BpJ-foDbVg90H21EOLACvtfcHk0lghI5sh3V8e2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktFWgs7c%253D&md5=b1ca389e67a3ccd3fe5c005cb547e369</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00209%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DGuo%26aufirst%3DQ.%26aulast%3DNie%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DXiang%26aufirst%3DR.%26atitle%3DDiscovery%2520of%2520N1-%25284-%2528%25287-Cyclopentyl-6-%2528Dimethylcarbamoyl%2529-7H-Pyrrolo%255B2%252C3-d%255DPyrimidin-2-yl%2529Amino%2529Phenyl%2529-%2520N8-Hydroxyoctanediamide%2520as%2520a%2520Novel%2520Inhibitor%2520Targeting%2520Cyclin-Dependent%2520Kinase%25204%252F9%2520%2528CDK4%252F9%2529%2520and%2520Histone%2520Deacetlyase1%2520%2528HDAC1%2529%2520against%2520Malignant%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D3166%26epage%3D3192%26doi%3D10.1021%2Facs.jmedchem.8b00209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1577</span>– <span class="NLM_lpage">1592</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01825</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01825" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1577-1592&author=Y.+Chenauthor=X.+Yuanauthor=W.+Zhangauthor=M.+Tangauthor=L.+Zhengauthor=F.+Wangauthor=W.+Yanauthor=S.+Yangauthor=Y.+Weiauthor=J.+Heauthor=L.+Chen&title=Discovery+of+Novel+Dual+Histone+Deacetylase+and+Mammalian+Target+of+Rapamycin+Target+Inhibitors+as+a+Promising+Strategy+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.8b01825"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Dual Histone Deacetylase and Mammalian Target of Rapamycin Target Inhibitors as a Promising Strategy for Cancer Therapy</span></div><div class="casAuthors">Chen, Yong; Yuan, Xue; Zhang, Wanhua; Tang, Minghai; Zheng, Li; Wang, Fang; Yan, Wei; Yang, Shengyong; Wei, Yuquan; He, Jun; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1577-1592</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the present study, a series of novel dual-target histone deacetylase (HDAC) and mammalian target of rapamycin (mTOR) inhibitors were designed and synthesized using pyrimidine-pyrazolyl pharmacophore to append HDAC recognition cap and hydroxamic acid as a zinc-binding motif.  Among them, I was the optimal lead compd. with potent inhibition activities against mTOR and HDAC1 with half-maximal inhibitory concn. of 1.2 and 0.19 nM, resp.  Western blot confirmed that I could upregulate acetylation of H3 and α-tubulin and downregulate mTOR-related downstream mediators.  I could also stimulate cell cycle arrest in G0/G1 phase and induce tumor cell apoptosis.  I showed comparable antitumor activity with the combination medication in MM1S xenograft model with a tumor growth inhibitory rate of 72.5%, without causing significant loss of body wt. and toxicity.  All of the results indicated that I could be a promising dual target inhibitor for treating hematol. malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ3AdehNv1ObVg90H21EOLACvtfcHk0liMFmvVfl8l5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXovFeguw%253D%253D&md5=7a2ff5020aedb0e3c302e65d993301e6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01825%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DTang%26aufirst%3DM.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYan%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520Novel%2520Dual%2520Histone%2520Deacetylase%2520and%2520Mammalian%2520Target%2520of%2520Rapamycin%2520Target%2520Inhibitors%2520as%2520a%2520Promising%2520Strategy%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1577%26epage%3D1592%26doi%3D10.1021%2Facs.jmedchem.8b01825" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chou, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of Novel Janus Kinase (JAK) and Histone Deacetylase (HDAC) Dual Inhibitors for the Treatment of Hematological Malignancies</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">3898</span>– <span class="NLM_lpage">3923</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1MXlvVSls7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=3898-3923&author=X.+Liangauthor=J.+Zangauthor=X.+Liauthor=S.+Tangauthor=M.+Huangauthor=M.+Gengauthor=C.+J.+Chouauthor=C.+Liauthor=Y.+Caoauthor=W.+Xuauthor=H.+Liuauthor=Y.+Zhang&title=Discovery+of+Novel+Janus+Kinase+%28JAK%29+and+Histone+Deacetylase+%28HDAC%29+Dual+Inhibitors+for+the+Treatment+of+Hematological+Malignancies&doi=10.1021%2Facs.jmedchem.8b01597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Janus Kinase (JAK) and Histone Deacetylase (HDAC) Dual Inhibitors for the Treatment of Hematological Malignancies</span></div><div class="casAuthors">Liang, Xuewu; Zang, Jie; Li, Xiaoyang; Tang, Shuai; Huang, Min; Geng, Meiyu; Chou, C. James; Li, Chunpu; Cao, Yichun; Xu, Wenfang; Liu, Hong; Zhang, Yingjie</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3898-3923</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concurrent inhibition of Janus kinase (JAK) and histone deacetylase (HDAC) could potentially improve the efficacy of the HDAC inhibitors in the treatment of cancers and resolve the problem of HDAC inhibitor resistance in some tumors.  Here, a novel series of pyrimidin-2-amino-pyrazol hydroxamate derivs. as JAK and HDAC dual inhibitors was designed, synthesized, and evaluated, among which I possessed potent and balanced activities against both JAK2 and HDAC6 with half-maximal inhibitory concn. at the nanomolar level. I exhibited improved antiproliferative and proapoptotic activities over SAHA and ruxolitinib in several hematol. cell lines.  Remarkably, I exhibited more potent antiproliferation effect than the combination of SAHA and ruxolitinib in HEL cells bearing JAK2V617F mutation.  Pharmacokinetic studies in mice showed that I possessed good bioavailability after i.p. administration.  Finally, I showed antitumor efficacy with no significant toxicity in a HEL xenograft model.  Collectively, the results confirm the therapeutic potential of JAK and HDAC dual inhibitors in hematol. malignancies and provide valuable leads for further structural optimization and antitumor mechanism study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOXlwxqcv7m7Vg90H21EOLACvtfcHk0liMFmvVfl8l5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXlvVSls7o%253D&md5=356363f40f24ac2ef32194e23a29328e</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01597%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DZang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DChou%26aufirst%3DC.%2BJ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520Novel%2520Janus%2520Kinase%2520%2528JAK%2529%2520and%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Dual%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Hematological%2520Malignancies%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D3898%26epage%3D3923%26doi%3D10.1021%2Facs.jmedchem.8b01597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cakici, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karabuga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulukanli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilic, H.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Asymmetric Catalytic Activity of (1S,1′S)-4,4′-Biquinazoline-Based Primary Amines</span>. <i>Tetrahedron: Asymmetry</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1016/j.tetasy.2011.01.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.tetasy.2011.01.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1GntL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2011&pages=300-308&author=M.+Cakiciauthor=M.+Catirauthor=S.+Karabugaauthor=S.+Ulukanliauthor=H.+Kilic&title=Synthesis+and+Asymmetric+Catalytic+Activity+of+%281S%2C1%E2%80%B2S%29-4%2C4%E2%80%B2-Biquinazoline-Based+Primary+Amines&doi=10.1016%2Fj.tetasy.2011.01.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and asymmetric catalytic activity of (1S,1'S)-4,4'-biquinazoline-based primary amines</span></div><div class="casAuthors">Cakici, Murat; Catir, Mustafa; Karabuga, Semistan; Ulukanli, Sabri; Kilic, Hamdullah</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron: Asymmetry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">300-308</span>CODEN:
                <span class="NLM_cas:coden">TASYE3</span>;
        ISSN:<span class="NLM_cas:issn">0957-4166</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of (1S,1'S)-4,4'-biquinazoline-based primary amines I (R = Me, i-Pr, t-Bu, Bn) was prepd. from natural amino acids via a six-step reaction sequence of protection and condensation followed by key synthetic steps including chlorination, nickel(0)-mediated homocoupling, and deprotection.  These novel amines were screened for the asym. ethylation of aryl aldehydes with di-Et zinc to yield alcs. with an (S)-configuration with enantiomeric excesses (ee) varying from 2% to 95%.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9txIL-5n3qrVg90H21EOLACvtfcHk0liMFmvVfl8l5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1GntL8%253D&md5=f47113d732de61f07d9686c2dd5ebbd8</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.tetasy.2011.01.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetasy.2011.01.009%26sid%3Dliteratum%253Aachs%26aulast%3DCakici%26aufirst%3DM.%26aulast%3DCatir%26aufirst%3DM.%26aulast%3DKarabuga%26aufirst%3DS.%26aulast%3DUlukanli%26aufirst%3DS.%26aulast%3DKilic%26aufirst%3DH.%26atitle%3DSynthesis%2520and%2520Asymmetric%2520Catalytic%2520Activity%2520of%2520%25281S%252C1%25E2%2580%25B2S%2529-4%252C4%25E2%2580%25B2-Biquinazoline-Based%2520Primary%2520Amines%26jtitle%3DTetrahedron%253A%2520Asymmetry%26date%3D2011%26volume%3D22%26spage%3D300%26epage%3D308%26doi%3D10.1016%2Fj.tetasy.2011.01.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, A.</span></span> <span> </span><span class="NLM_article-title">Sar Study of 5-Alkynyl Substituted Quinazolin-4(3H)-ones as Phosphoinositide 3-Kinase Delta (PI3Kδ) Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">1156</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.ejmech.2016.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27846451" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVChsbjJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2017&pages=1156-1171&author=M.+Weiauthor=X.+Zhangauthor=X.+Wangauthor=Z.+Songauthor=J.+Dingauthor=L.+H.+Mengauthor=A.+Zhang&title=Sar+Study+of+5-Alkynyl+Substituted+Quinazolin-4%283H%29-ones+as+Phosphoinositide+3-Kinase+Delta+%28PI3K%CE%B4%29+Inhibitors&doi=10.1016%2Fj.ejmech.2016.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">SAR study of 5-alkynyl substituted quinazolin-4(3H)-ones as phosphoinositide 3-kinase delta (PI3Kδ) inhibitors</span></div><div class="casAuthors">Wei, Manman; Zhang, Xi; Wang, Xiang; Song, Zilan; Ding, Jian; Meng, Ling-Hua; Zhang, Ao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1156-1171</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">PI3Kδ is a key component in the aberrant signaling transduction in B cell malignancy, therefore specific targeting PI3Kδ has become an attractive molecularly targeted therapy for chronic lymphocytic leukemia (CLL).  Herein, we describe the discovery and optimization of a series of 5-alkynyl substituted PI3Kδ inhibitors based on the first FDA-approved inhibitor idelalisib.  Compd. 8d bearing the 1-morpholinohex-5-yn-1-one moiety as the C5-substituent was identified to have high potency against PI3Kδ (3.82 nM) and SU-DHL-6 cells (7.60 nM), resp.  It was 154-fold selective over PI3Kα, 133-fold selective against PI3Kβ, and 24-fold selective against PI3Kγ.  Treatment of MOLT-4 and SU-DHL-6 cells with compd. 8d for 1 h resulted in redn. of phosphorylation of both Akt (S473) and its downstream S6k1 (T389) in a concn.-dependent manner.  Compd. 8d showed potent anti-proliferative activity as well against T lymphoblast MOLT-4, suggesting its potential activity in T-cell leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYWHSwSiMyQrVg90H21EOLACvtfcHk0liC5lExcGQR2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVChsbjJ&md5=809c50e6dc45e1e4871af760fe27cefc</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DMeng%26aufirst%3DL.%2BH.%26aulast%3DZhang%26aufirst%3DA.%26atitle%3DSar%2520Study%2520of%25205-Alkynyl%2520Substituted%2520Quinazolin-4%25283H%2529-ones%2520as%2520Phosphoinositide%25203-Kinase%2520Delta%2520%2528PI3K%25CE%25B4%2529%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D125%26spage%3D1156%26epage%3D1171%26doi%3D10.1016%2Fj.ejmech.2016.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adveef, A.</span></span> <span> </span><span class="NLM_article-title">Permeability—PAMPA</span>. In  <i>Absorption and Drug Development</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">319</span>– <span class="NLM_lpage">498</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=319-498&author=A.+Adveef&title=Absorption+and+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DAdveef%26aufirst%3DA.%26atitle%3DPermeability%25E2%2580%2594PAMPA%26btitle%3DAbsorption%2520and%2520Drug%2520Development%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012%26spage%3D319%26epage%3D498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evarts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forseth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haran, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashishian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koppenol, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepist, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perreault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of Orally Efficacious Phosphoinositide 3-Kinase Delta Inhibitors with Improved Metabolic Stability</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9228</span>– <span class="NLM_lpage">9242</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01169</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01169" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFGrs7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9228-9242&author=L.+Patelauthor=J.+Chandrasekharauthor=J.+Evartsauthor=K.+Forsethauthor=A.+C.+Haranauthor=C.+Ipauthor=A.+Kashishianauthor=M.+Kimauthor=D.+Koditekauthor=S.+Koppenolauthor=L.+Ladauthor=E.+I.+Lepistauthor=M.+E.+McGrathauthor=S.+Perreaultauthor=K.+D.+Puriauthor=A.+G.+Villasenorauthor=J.+R.+Somozaauthor=B.+H.+Steinerauthor=J.+Therrienauthor=J.+Treibergauthor=G.+Phillips&title=Discovery+of+Orally+Efficacious+Phosphoinositide+3-Kinase+Delta+Inhibitors+with+Improved+Metabolic+Stability&doi=10.1021%2Facs.jmedchem.6b01169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Efficacious Phosphoinositide 3-Kinase δ Inhibitors with Improved Metabolic Stability</span></div><div class="casAuthors">Patel, Leena; Chandrasekhar, Jayaraman; Evarts, Jerry; Forseth, Kristen; Haran, Aaron C.; Ip, Carmen; Kashishian, Adam; Kim, Musong; Koditek, David; Koppenol, Sandy; Lad, Latesh; Lepist, Eve-Irene; McGrath, Mary E.; Perreault, Stephane; Puri, Kamal D.; Villasenor, Armando G.; Somoza, John R.; Steiner, Bart H.; Therrien, Joseph; Treiberg, Jennifer; Phillips, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9228-9242</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant signaling of phosphoinositide-3-kinase delta (PI3K-delta) has been implicated in numerous pathologies including hematol. malignancies and rheumatoid arthritis.  Described in this manuscript is the discovery, optimization and in vivo evaluation of a novel series of pyridine-contg. PI3K-delta inhibitors.  This work led to the discovery of I, a highly selective inhibitor of PI3K-delta which displays an excellent pharmacokinetic profile and is efficacious in a rodent model of rheumatoid arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1LgrP9YV8frVg90H21EOLACvtfcHk0liC5lExcGQR2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFGrs7rF&md5=6d9dff0ec5270aab250fcb4800d8635a</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01169%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DEvarts%26aufirst%3DJ.%26aulast%3DForseth%26aufirst%3DK.%26aulast%3DHaran%26aufirst%3DA.%2BC.%26aulast%3DIp%26aufirst%3DC.%26aulast%3DKashishian%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DKoppenol%26aufirst%3DS.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DLepist%26aufirst%3DE.%2BI.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26aulast%3DPerreault%26aufirst%3DS.%26aulast%3DPuri%26aufirst%3DK.%2BD.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DSteiner%26aufirst%3DB.%2BH.%26aulast%3DTherrien%26aufirst%3DJ.%26aulast%3DTreiberg%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520Orally%2520Efficacious%2520Phosphoinositide%25203-Kinase%2520Delta%2520Inhibitors%2520with%2520Improved%2520Metabolic%2520Stability%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9228%26epage%3D9242%26doi%3D10.1021%2Facs.jmedchem.6b01169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Patel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evarts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haran, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ip, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepist, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novikov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perreault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puri, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somoza, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villasenor, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span> <span> </span><span class="NLM_article-title">2,4,6-Triaminopyrimidine as a Novel Hinge Binder in a Series of PI3Kδ Selective Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3532</span>– <span class="NLM_lpage">3548</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00213</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00213" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28Xkt1CgtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=3532-3548&author=L.+Patelauthor=J.+Chandrasekharauthor=J.+Evartsauthor=A.+C.+Haranauthor=C.+Ipauthor=J.+A.+Kaplanauthor=M.+Kimauthor=D.+Koditekauthor=L.+Ladauthor=E.+I.+Lepistauthor=M.+E.+McGrathauthor=N.+Novikovauthor=S.+Perreaultauthor=K.+D.+Puriauthor=J.+R.+Somozaauthor=B.+H.+Steinerauthor=K.+L.+Stevensauthor=J.+Therrienauthor=J.+Treibergauthor=A.+G.+Villasenorauthor=A.+Yeungauthor=G.+Phillips&title=2%2C4%2C6-Triaminopyrimidine+as+a+Novel+Hinge+Binder+in+a+Series+of+PI3K%CE%B4+Selective+Inhibitors&doi=10.1021%2Facs.jmedchem.6b00213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">2,4,6-Triaminopyrimidine as a Novel Hinge Binder in a Series of PI3Kδ Selective Inhibitors</span></div><div class="casAuthors">Patel, Leena; Chandrasekhar, Jayaraman; Evarts, Jerry; Haran, Aaron C.; Ip, Carmen; Kaplan, Joshua A.; Kim, Musong; Koditek, David; Lad, Latesh; Lepist, Eve-Irene; McGrath, Mary E.; Novikov, Nikolai; Perreault, Stephane; Puri, Kamal D.; Somoza, John R.; Steiner, Bart H.; Stevens, Kirk L.; Therrien, Joseph; Treiberg, Jennifer; Villasenor, Armando G.; Yeung, Arthur; Phillips, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3532-3548</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of phosphoinositide 3-kinase δ (PI3Kδ) is an appealing target for several hematol. malignancies and inflammatory diseases.  Herein, we describe the discovery and optimization of a series of propeller shaped PI3Kδ inhibitors comprising a novel triaminopyrimidine hinge binder.  Combinations of electronic and structural strategies were employed to mitigate aldehyde oxidase mediated metab.  This medicinal chem. effort culminated in the identification of 52, a potent and highly selective inhibitor of PI3Kδ that demonstrates efficacy in a rat model of arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpygPlHbWp_TLVg90H21EOLACvtfcHk0lhBcg74VtAMyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xkt1CgtL4%253D&md5=9e01fff097eccd127a9b9284d725d7e1</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00213%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DEvarts%26aufirst%3DJ.%26aulast%3DHaran%26aufirst%3DA.%2BC.%26aulast%3DIp%26aufirst%3DC.%26aulast%3DKaplan%26aufirst%3DJ.%2BA.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DLepist%26aufirst%3DE.%2BI.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26aulast%3DNovikov%26aufirst%3DN.%26aulast%3DPerreault%26aufirst%3DS.%26aulast%3DPuri%26aufirst%3DK.%2BD.%26aulast%3DSomoza%26aufirst%3DJ.%2BR.%26aulast%3DSteiner%26aufirst%3DB.%2BH.%26aulast%3DStevens%26aufirst%3DK.%2BL.%26aulast%3DTherrien%26aufirst%3DJ.%26aulast%3DTreiberg%26aufirst%3DJ.%26aulast%3DVillasenor%26aufirst%3DA.%2BG.%26aulast%3DYeung%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DG.%26atitle%3D2%252C4%252C6-Triaminopyrimidine%2520as%2520a%2520Novel%2520Hinge%2520Binder%2520in%2520a%2520Series%2520of%2520PI3K%25CE%25B4%2520Selective%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D3532%26epage%3D3548%26doi%3D10.1021%2Facs.jmedchem.6b00213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perreault, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evarts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashishian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kenney, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koditek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lad, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepist, E. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGrath, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therrien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiberg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yahiaoui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Phosphoinositide 3-Kinase (PI3K) Beta/Delta Inhibitor for the Treatment of Phosphatase and Tensin Homolog (PTEN) Deficient Tumors: Building PI3Kβ Potency in a PI3Kδ-Selective Template by Targeting Nonconserved Asp856</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1567</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01821</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01821" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVansro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1555-1567&author=S.+Perreaultauthor=J.+Chandrasekharauthor=Z.+H.+Cuiauthor=J.+Evartsauthor=J.+Haoauthor=J.+A.+Kaplanauthor=A.+Kashishianauthor=K.+S.+Keeganauthor=T.+Kenneyauthor=D.+Koditekauthor=L.+Ladauthor=E.+I.+Lepistauthor=M.+E.+McGrathauthor=L.+Patelauthor=B.+Phillipsauthor=J.+Therrienauthor=J.+Treibergauthor=A.+Yahiaouiauthor=G.+Phillips&title=Discovery+of+a+Phosphoinositide+3-Kinase+%28PI3K%29+Beta%2FDelta+Inhibitor+for+the+Treatment+of+Phosphatase+and+Tensin+Homolog+%28PTEN%29+Deficient+Tumors%3A+Building+PI3K%CE%B2+Potency+in+a+PI3K%CE%B4-Selective+Template+by+Targeting+Nonconserved+Asp856&doi=10.1021%2Facs.jmedchem.6b01821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Phosphoinositide 3-Kinase (PI3K) β/δ Inhibitor for the Treatment of Phosphatase and Tensin Homolog (PTEN) Deficient Tumors: Building PI3Kβ Potency in a PI3Kδ-Selective Template by Targeting Nonconserved Asp856</span></div><div class="casAuthors">Perreault, Stephane; Chandrasekhar, Jayaraman; Cui, Zhi-Hua; Evarts, Jerry; Hao, Jia; Kaplan, Joshua A.; Kashishian, Adam; Keegan, Kathleen S.; Kenney, Thomas; Koditek, David; Lad, Latesh; Lepist, Eve-Irene; McGrath, Mary E.; Patel, Leena; Phillips, Bart; Therrien, Joseph; Treiberg, Jennifer; Yahiaoui, Anella; Phillips, Gary</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1555-1567</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Phosphoinositide 3-kinase (PI3K) beta signaling is required to sustain cancer cell growth in which the tumor suppressor phosphatase and tensin homolog (PTEN) has been deactivated.  This manuscript describes the discovery, optimization, and in vivo evaluation of a novel series of PI3Kbeta/delta inhibitors in which PI3Kbeta potency was built in a PI3Kdelta-selective template.  This work led to the discovery of a highly selective PI3Kbeta/delta inhibitor displaying excellent pharmacokinetic profile and efficacy in a human PTEN-deficient LNCaP prostate carcinoma xenograft tumor model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpf82mTX9c-e7Vg90H21EOLACvtfcHk0li7yG7DqODSzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVansro%253D&md5=0ebc8ca6291d39f945926e11b34fced6</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01821%26sid%3Dliteratum%253Aachs%26aulast%3DPerreault%26aufirst%3DS.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DZ.%2BH.%26aulast%3DEvarts%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DJ.%26aulast%3DKaplan%26aufirst%3DJ.%2BA.%26aulast%3DKashishian%26aufirst%3DA.%26aulast%3DKeegan%26aufirst%3DK.%2BS.%26aulast%3DKenney%26aufirst%3DT.%26aulast%3DKoditek%26aufirst%3DD.%26aulast%3DLad%26aufirst%3DL.%26aulast%3DLepist%26aufirst%3DE.%2BI.%26aulast%3DMcGrath%26aufirst%3DM.%2BE.%26aulast%3DPatel%26aufirst%3DL.%26aulast%3DPhillips%26aufirst%3DB.%26aulast%3DTherrien%26aufirst%3DJ.%26aulast%3DTreiberg%26aufirst%3DJ.%26aulast%3DYahiaoui%26aufirst%3DA.%26aulast%3DPhillips%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520a%2520Phosphoinositide%25203-Kinase%2520%2528PI3K%2529%2520Beta%252FDelta%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Phosphatase%2520and%2520Tensin%2520Homolog%2520%2528PTEN%2529%2520Deficient%2520Tumors%253A%2520Building%2520PI3K%25CE%25B2%2520Potency%2520in%2520a%2520PI3K%25CE%25B4-Selective%2520Template%2520by%2520Targeting%2520Nonconserved%2520Asp856%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1555%26epage%3D1567%26doi%3D10.1021%2Facs.jmedchem.6b01821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christianson, D. W.</span></span> <span> </span><span class="NLM_article-title">Histone Deacetylase 6 Structure and Molecular Basis of Catalysis and Inhibition</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">741</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fnchembio.2134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27454933" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=741-747&author=Y.+Haiauthor=D.+W.+Christianson&title=Histone+Deacetylase+6+Structure+and+Molecular+Basis+of+Catalysis+and+Inhibition&doi=10.1038%2Fnchembio.2134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Histone deacetylase 6 structure and molecular basis of catalysis and inhibition</span></div><div class="casAuthors">Hai, Yang; Christianson, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">741-747</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone deacetylase 6 (HDAC6) is a crit. target for drug design because of its role in oncogenic transformation and cancer metastasis, and is unique among all histone deacetylases in that it contains tandem catalytic domains designated CD1 and CD2.  We now report the crystal structures of CD2 from Homo sapiens HDAC6 and of CD1 and CD2 from Danio rerio HDAC6.  We correlated these structures with activity measurements using 13 different substrates.  The catalytic activity of CD2 from both species exhibited broad substrate specificity, whereas that of CD1 was highly specific for substrates bearing C-terminal acetyllysine residues.  Crystal structures of substrate complexes yielded unprecedented snapshots of the catalytic mechanism.  Addnl., crystal structures of complexes with eight different inhibitors, including belinostat and panobinostat (currently used in cancer chemotherapy), the macrocyclic tetrapeptide HC toxin, and the HDAC6-specific inhibitor N-hydroxy-4-(2-((2-hydroxyethyl)(phenyl)amino)-2-oxoethyl)benzamide, revealed surprising new insight regarding changes in Zn2+ coordination and isoenzyme-specific inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqCfziGBHBRLVg90H21EOLACvtfcHk0lgp0pvKPk9fIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1Kmu7rJ&md5=e167bc6f66e00923da4f2bced2b306b9</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2134%26sid%3Dliteratum%253Aachs%26aulast%3DHai%26aufirst%3DY.%26aulast%3DChristianson%26aufirst%3DD.%2BW.%26atitle%3DHistone%2520Deacetylase%25206%2520Structure%2520and%2520Molecular%2520Basis%2520of%2520Catalysis%2520and%2520Inhibition%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D741%26epage%3D747%26doi%3D10.1038%2Fnchembio.2134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span> <i>NCI-60
Human Tumor Cell Lines Screen</i>; <span class="NLM_publisher-name">National Cancer
Institute</span>, <span class="NLM_year">2015</span>; <a href="https://dtp.cancer.gov/discovery_development/nci-60/" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/</a> (accessed 2019-10-10).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+NCI-60%0AHuman+Tumor+Cell+Lines+Screen%3B+National+Cancer%0AInstitute%2C+2015%3B+https%3A%2F%2Fdtp.cancer.gov%2Fdiscovery_development%2Fnci-60%2F+%28accessed+2019-10-10%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dgov%26sid%3Dliteratum%253Aachs%26jtitle%3DNCI-60%250AHuman%2520Tumor%2520Cell%2520Lines%2520Screen%26pub%3DNational%2520Cancer%250AInstitute%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span> <i>Celltiter-Glo Luminescent
Cell Viability Assay</i>; <span class="NLM_publisher-name">Promega</span>, <span class="NLM_year">2015</span>; <a href="https://www.promega.com/-/media/files/resources/protocols/technical-bulletins/0/celltiter-glo-luminescent-cell-viability-assay-protocol.pdf?la=en" class="extLink">https://www.promega.com/-/media/files/resources/protocols/technical-bulletins/0/celltiter-glo-luminescent-cell-viability-assay-protocol.pdf?la=en</a> (accessed 2019-10-15).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Celltiter-Glo+Luminescent%0ACell+Viability+Assay%3B+Promega%2C+2015%3B+https%3A%2F%2Fwww.promega.com%2F-%2Fmedia%2Ffiles%2Fresources%2Fprotocols%2Ftechnical-bulletins%2F0%2Fcelltiter-glo-luminescent-cell-viability-assay-protocol.pdf%3Fla%3Den+%28accessed+2019-10-15%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26jtitle%3DCelltiter-Glo%2520Luminescent%250ACell%2520Viability%2520Assay%26pub%3DPromega%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sattler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span> <span> </span><span class="NLM_article-title">Reversible Resistance Induced by FLT3 Inhibition: A Novel Resistance Mechanism in Mutant FLT3-Expressing Cells</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e25351</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0025351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1371%2Fjournal.pone.0025351" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21980431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e25351&author=E.+Weisbergauthor=A.+Rayauthor=E.+Nelsonauthor=S.+Adamiaauthor=R.+Barrettauthor=M.+Sattlerauthor=C.+Zhangauthor=J.+F.+Daleyauthor=D.+Frankauthor=E.+Foxauthor=J.+D.+Griffin&title=Reversible+Resistance+Induced+by+FLT3+Inhibition%3A+A+Novel+Resistance+Mechanism+in+Mutant+FLT3-Expressing+Cells&doi=10.1371%2Fjournal.pone.0025351"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible resistance induced by FLT3 inhibition: a novel resistance mechanism in mutant FLT3-expressing cells</span></div><div class="casAuthors">Weisberg, Ellen; Ray, Arghya; Nelson, Erik; Adamia, Sophia; Barrett, Rosemary; Sattler, Martin; Zhang, Chengsheng; Daley, John F.; Frank, David; Fox, Edward; Griffin, James D.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e25351</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Objectives: Clin. responses achieved with FLT3 kinase inhibitors in acute myeloid leukemia (AML) are typically transient and partial.  Thus, there is a need for identification of mol. mechanisms of clin. resistance to these drugs.  In response, we characterized MOLM13 AML cell lines made resistant to two structurally-independent FLT3 inhibitors.  Methods: MOLM13 cells were made drug resistant via prolonged exposure to midostaurin and HG-7-85-01, resp.  Cell proliferation was detd. by Trypan blue exclusion.  Protein expression was assessed by immunoblotting, immunopptn., and flow cytometry.  Cycloheximide was used to det. protein half-life.  RT-PCR was performed to det. FLT3 mRNA levels, and FISH anal. was performed to det. FLT3 gene expression.  Results and Conclusions: We found that MOLM13 cells readily developed cross-resistance when exposed to either midostaurin or HG-7-85-01.  Resistance in both lines was assocd. with dramatically elevated levels of cell surface FLT3 and elevated levels of phosphor-MAPK, but not phospho-STAT5.  The increase in FLT3-ITD expression was at least in part due to reduced turnover of the receptor, with prolonged half-life.  Importantly, the drug-resistant phenotype could be rapidly reversed upon withdrawal of either inhibitor.  Consistent with this phenotype, no significant evidence of FLT3 gene amplification, kinase domain mutations, or elevated levels of mRNA was obsd., suggesting that protein turnover may be part of an auto-regulatory pathway initiated by FLT3 kinase activity.  Interestingly, FLT3 inhibitor resistance also correlated with resistance to cytosine arabinoside.  Over-expression of FLT3 protein in response to kinase inhibitors may be part of a novel mechanism that could contribute to clin. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWa-oQ_co7RLVg90H21EOLACvtfcHk0lgp0pvKPk9fIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCjsL7M&md5=bc8ce2e7906ac3bce393573ee7a6e685</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0025351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0025351%26sid%3Dliteratum%253Aachs%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DRay%26aufirst%3DA.%26aulast%3DNelson%26aufirst%3DE.%26aulast%3DAdamia%26aufirst%3DS.%26aulast%3DBarrett%26aufirst%3DR.%26aulast%3DSattler%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DDaley%26aufirst%3DJ.%2BF.%26aulast%3DFrank%26aufirst%3DD.%26aulast%3DFox%26aufirst%3DE.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26atitle%3DReversible%2520Resistance%2520Induced%2520by%2520FLT3%2520Inhibition%253A%2520A%2520Novel%2520Resistance%2520Mechanism%2520in%2520Mutant%2520FLT3-Expressing%2520Cells%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De25351%26doi%3D10.1371%2Fjournal.pone.0025351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bosshart, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzelmann, M.</span></span> <span> </span><span class="NLM_article-title">THP-1 Cells as a Model for Human Monocytes</span>. <i>Ann. Transl Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">438</span>, <span class="refDoi"> DOI: 10.21037/atm.2016.08.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.21037%2Fatm.2016.08.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27942529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslOhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=438&author=H.+Bosshartauthor=M.+Heinzelmann&title=THP-1+Cells+as+a+Model+for+Human+Monocytes&doi=10.21037%2Fatm.2016.08.53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">THP-1 cells as a model for human monocytes</span></div><div class="casAuthors">Bosshart, Herbert; Heinzelmann, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Translational Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">438/1-438/4</span>CODEN:
                <span class="NLM_cas:coden">ATMNDX</span>;
        ISSN:<span class="NLM_cas:issn">2305-5847</span>.
    
            (<span class="NLM_cas:orgname">AME Publishing Co.</span>)
        </div><div class="casAbstract">In mammals, circulating monocytes provide effective protection against infection with gram-neg. bacteria.  Once challenged with lipolysaccharide (LPS), a macromol. structure of the cell wall of Gram-neg. bacteria, peripheral monocytes migrate to lymph nodes, differentiation into dendritic cells, and present antigen to CD4 and CD8 T cells with high efficacy, therby initiating adaptive immunity against gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEYECqXXAYJ7Vg90H21EOLACvtfcHk0lggO05CewCvaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslOhsrc%253D&md5=794ec7557dd34eaa3185f2f33ea94261</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.21037%2Fatm.2016.08.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21037%252Fatm.2016.08.53%26sid%3Dliteratum%253Aachs%26aulast%3DBosshart%26aufirst%3DH.%26aulast%3DHeinzelmann%26aufirst%3DM.%26atitle%3DTHP-1%2520Cells%2520as%2520a%2520Model%2520for%2520Human%2520Monocytes%26jtitle%3DAnn.%2520Transl%2520Med.%26date%3D2016%26volume%3D4%26spage%3D438%26doi%3D10.21037%2Fatm.2016.08.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramachandran, R. P.</span>; <span class="NLM_string-name">Madhivanan, S.</span>; <span class="NLM_string-name">Sundararajan, R.</span>; <span class="NLM_string-name">Wan-Ying Lin, C.</span>; <span class="NLM_string-name">Sankaranarayanan, K.</span></span> <span> </span><span class="NLM_article-title">An in Vitro Study of Electroporation of Leukemia and Cervical Cancer Cells</span>. In  <i>Electroporation-Based Therapies for Cancer</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundararajan, R.</span></span>, Ed.; <span class="NLM_publisher-name">Woodhead Publishing</span>, <span class="NLM_publisher-loc">Cambridge UK</span>, <span class="NLM_year">2014</span>; Chapter 7, pp  <span class="NLM_fpage">161</span>– <span class="NLM_lpage">183</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1533%2F9781908818294.161" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=161-183&author=R.+P.+Ramachandran&author=S.+Madhivanan&author=R.+Sundararajan&author=C.+Wan-Ying+Lin&author=K.+Sankaranarayananauthor=R.+Sundararajan&title=Electroporation-Based+Therapies+for+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1533%2F9781908818294.161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1533%252F9781908818294.161%26sid%3Dliteratum%253Aachs%26aulast%3DRamachandran%26aufirst%3DR.%2BP.%26atitle%3DAn%2520in%2520Vitro%2520Study%2520of%2520Electroporation%2520of%2520Leukemia%2520and%2520Cervical%2520Cancer%2520Cells%26btitle%3DElectroporation-Based%2520Therapies%2520for%2520Cancer%26aulast%3DSundararajan%26aufirst%3DR.%26pub%3DWoodhead%2520Publishing%26date%3D2014%26spage%3D161%26epage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matsuo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLeod, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uphoff, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishizaki, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katayama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujii, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orita, K.</span></span> <span> </span><span class="NLM_article-title">Two Acute Monocytic Leukemia (AML-M5a) Cell Lines (MOLM-13 and MOLM-14) with Interclonal Phenotypic Heterogeneity Showing MLL-AF9 Fusion Resulting from an Occult Chromosome Insertion, Ins(11;9)(Q23;P22p23)</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1477</span>, <span class="refDoi"> DOI: 10.1038/sj.leu.2400768</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fsj.leu.2400768" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=9305600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADyaK2sXnsFagsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1997&pages=1469-1477&author=Y.+Matsuoauthor=R.+A.+MacLeodauthor=C.+C.+Uphoffauthor=H.+G.+Drexlerauthor=C.+Nishizakiauthor=Y.+Katayamaauthor=G.+Kimuraauthor=N.+Fujiiauthor=E.+Omotoauthor=M.+Haradaauthor=K.+Orita&title=Two+Acute+Monocytic+Leukemia+%28AML-M5a%29+Cell+Lines+%28MOLM-13+and+MOLM-14%29+with+Interclonal+Phenotypic+Heterogeneity+Showing+MLL-AF9+Fusion+Resulting+from+an+Occult+Chromosome+Insertion%2C+Ins%2811%3B9%29%28Q23%3BP22p23%29&doi=10.1038%2Fsj.leu.2400768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Two acute monocytic leukemia (AML-M5a) cell lines (MOLM-13 and MOLM-14) with interclonal phenotypic heterogeneity showing MLL-AF9 fusion resulting from an occult chromosome insertion, ins(11;9)(q23;p22p23)</span></div><div class="casAuthors">Matsuo, Y.; Macleod, R. A. F.; Uphoff, C. C.; Drexler, H. G.; Nishizaki, C.; Katayama, Y.; Kimura, G.; Fujii, N.; Omoto, E.; Harada, M.; Orita, K.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1469-1477</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Stockton Press</span>)
        </div><div class="casAbstract">We describe two new human leukemia cell lines, MOLM-13 and MOLM-14, established from the peripheral blood of a patient at relapse of acute monocytic leukemia, FAB M5a, which had evolved from myelodysplastic syndrome (MDS).  Both cell lines express monocyte-specific esterase (MSE) and MLL-AF9 fusion mRNA.  Gene fusion is assocd. with a minute chromosomal insertion, ins(11;9)(q23;p22p23).  MOLM-13 and MOLM-14 are the first cell lines with, and represent the third reported case of, MLL gene rearrangement arising via chromosomal insertion.  Both cell lines carry trisomy 8 which was also present during the MDS phase, as well as the most frequent trisomies assocd. with t(9;11), ie, +6, +13, +19 variously present in different subclones.  Despite having these features in common, differences in antigen expression were noted between the two cell lines: that of MOLM-13 being CD34+, CD13-, CD14-, CD15+, CD33+; whereas MOLM-14 was CD4+, CD13+, CD14+, CD15+, CD33+.  Differentiation to macrophage-like morphol. could be induced in both cell lines after stimulation with INF-γ alone, or in combination with TNF-α, which treatment also induced or upregulated, expression of certain myelomonocyte-assocd. antigens, including CD13, CD14, CD15, CD64, CD65 and CD87.  Together, these data confirm that both cell lines are likely to be novel in vitro models for studying monocytic differentiation and leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYiVJrnDc7BbVg90H21EOLACvtfcHk0lggO05CewCvaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXnsFagsrc%253D&md5=8bec3174ce582e7829f78e9bd8946154</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2400768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2400768%26sid%3Dliteratum%253Aachs%26aulast%3DMatsuo%26aufirst%3DY.%26aulast%3DMacLeod%26aufirst%3DR.%2BA.%26aulast%3DUphoff%26aufirst%3DC.%2BC.%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26aulast%3DNishizaki%26aufirst%3DC.%26aulast%3DKatayama%26aufirst%3DY.%26aulast%3DKimura%26aufirst%3DG.%26aulast%3DFujii%26aufirst%3DN.%26aulast%3DOmoto%26aufirst%3DE.%26aulast%3DHarada%26aufirst%3DM.%26aulast%3DOrita%26aufirst%3DK.%26atitle%3DTwo%2520Acute%2520Monocytic%2520Leukemia%2520%2528AML-M5a%2529%2520Cell%2520Lines%2520%2528MOLM-13%2520and%2520MOLM-14%2529%2520with%2520Interclonal%2520Phenotypic%2520Heterogeneity%2520Showing%2520MLL-AF9%2520Fusion%2520Resulting%2520from%2520an%2520Occult%2520Chromosome%2520Insertion%252C%2520Ins%252811%253B9%2529%2528Q23%253BP22p23%2529%26jtitle%3DLeukemia%26date%3D1997%26volume%3D11%26spage%3D1469%26epage%3D1477%26doi%3D10.1038%2Fsj.leu.2400768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galinsky, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Q.</span></span> <span> </span><span class="NLM_article-title">Characterization of Selective and Potent PI3Kδ Inhibitor (PI3KDIN-015) for B-Cell Malignances</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">32641</span>– <span class="NLM_lpage">32651</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.8702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.18632%2Foncotarget.8702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27081697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BC28bgtFOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=32641-32651&author=X.+Liuauthor=A.+Wangauthor=X.+Liangauthor=C.+Chenauthor=J.+Liuauthor=Z.+Zhaoauthor=H.+Wuauthor=Y.+Dengauthor=L.+Wangauthor=B.+Wangauthor=J.+Wuauthor=F.+Liuauthor=S.+M.+Fernandesauthor=S.+Adamiaauthor=R.+M.+Stoneauthor=I.+A.+Galinskyauthor=J.+R.+Brownauthor=J.+D.+Griffinauthor=S.+Zhangauthor=T.+Lohauthor=X.+Zhangauthor=W.+Wangauthor=E.+L.+Weisbergauthor=J.+Liuauthor=Q.+Liu&title=Characterization+of+Selective+and+Potent+PI3K%CE%B4+Inhibitor+%28PI3KDIN-015%29+for+B-Cell+Malignances&doi=10.18632%2Foncotarget.8702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of selective and potent PI3Kδ inhibitor (PI3KDIN- 015) for B-Cell malignances</span></div><div class="casAuthors">Liu Xiaochuan; Loh Teckpeng; Liu Xiaochuan; Wang Aoli; Liang Xiaofei; Chen Cheng; Liu Juanjuan; Zhao Zheng; Wu Hong; Deng Yuanxin; Wang Li; Wang Beilei; Wu Jiaxin; Liu Feiyang; Zhang Xin; Wang Wenchao; Liu Jing; Liu Qingsong; Wang Aoli; Liu Juanjuan; Wu Hong; Deng Yuanxin; Wu Jiaxin; Liu Feiyang; Liang Xiaofei; Chen Cheng; Zhao Zheng; Wang Li; Wang Beilei; Zhang Shanchun; Wang Wenchao; Liu Jing; Liu Qingsong; Fernandes Stacey M; Adamia Sophia; Stone Richard M; Galinsky Ilene A; Brown Jennifer R; Griffin James D; Weisberg Ellen L; Zhang Shanchun; Liu Qingsong</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">32641-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">PI3Kδ is predominately expressed in leukocytes and has been found overexpressed in B-cell related malignances such as CLL and AML.  We have discovered a highly selective ATP competitive PI3Kd inhibitor PI3KD-IN-015, which exhibits a high selectivity among other PI3K isoforms in both biochemical assays and cellular assay, meanwhile did not inhibit most of other protein kinases in the kinome.  PI3KD-IN-015 demonstrates moderately anti-proliferation efficacies against a variety of B-cell related cancer cell lines through down-regulate the PI3K signaling significantly.  It induced both apoptosis and autophagy in B-cell malignant cell lines.  In addition, combination of autophagy inhibitor Bafilomycin could potentiate the moderate anti-proliferation effect of PI3KD-IN-015.  PI3KD-IN-015 shows anti-proliferation efficacy against CLL and AML patient primary cells.  Collectively, these results indicate that PI3KD-IN-015 may be useful drug candidate for further development of anti-B-cell related malignances therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSBYhYh3CW1QWZyxfu64m4ffW6udTcc2eYZa1Dqe61Smbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bgtFOgtw%253D%253D&md5=58b7ed0f416a5793f540405880c1c9f9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.8702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.8702%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DLiang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDeng%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DFernandes%26aufirst%3DS.%2BM.%26aulast%3DAdamia%26aufirst%3DS.%26aulast%3DStone%26aufirst%3DR.%2BM.%26aulast%3DGalinsky%26aufirst%3DI.%2BA.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLoh%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26atitle%3DCharacterization%2520of%2520Selective%2520and%2520Potent%2520PI3K%25CE%25B4%2520Inhibitor%2520%2528PI3KDIN-015%2529%2520for%2520B-Cell%2520Malignances%26jtitle%3DOncotarget%26date%3D2016%26volume%3D7%26spage%3D32641%26epage%3D32651%26doi%3D10.18632%2Foncotarget.8702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novotny-Diermayr, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangthongpitag, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sausgruber, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greicius, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loh, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonday, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hentze, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethirajulu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. M.</span></span> <span> </span><span class="NLM_article-title">SB939, a Novel Potent and Orally Active Histone Deacetylase Inhibitor with High Tumor Exposure and Efficacy in Mouse Models of Colorectal Cancer</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">642</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-09-0689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1158%2F1535-7163.MCT-09-0689" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=20197387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3cXivFens7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=642-652&author=V.+Novotny-Diermayrauthor=K.+Sangthongpitagauthor=C.+Y.+Huauthor=X.+Wuauthor=N.+Sausgruberauthor=P.+Yeoauthor=G.+Greiciusauthor=S.+Petterssonauthor=A.+L.+Liangauthor=Y.+K.+Lohauthor=Z.+Bondayauthor=K.+C.+Gohauthor=H.+Hentzeauthor=S.+Hartauthor=H.+Wangauthor=K.+Ethirajuluauthor=J.+M.+Wood&title=SB939%2C+a+Novel+Potent+and+Orally+Active+Histone+Deacetylase+Inhibitor+with+High+Tumor+Exposure+and+Efficacy+in+Mouse+Models+of+Colorectal+Cancer&doi=10.1158%2F1535-7163.MCT-09-0689"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">SB939, a Novel Potent and Orally Active Histone Deacetylase Inhibitor with High Tumor Exposure and Efficacy in Mouse Models of Colorectal Cancer</span></div><div class="casAuthors">Novotny-Diermayr, Veronica; Sangthongpitag, Kanda; Hu, Chang Yong; Wu, Xiaofeng; Sausgruber, Nina; Yeo, Pauline; Greicius, Gediminas; Pettersson, Sven; Liang, Ai Leng; Loh, Yung Kiang; Bonday, Zahid; Goh, Kee Chuan; Hentze, Hannes; Hart, Stefan; Wang, Haishan; Ethirajulu, Kantharaj; Wood, Jeanette Marjorie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">642-652</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Although clin. responses in liq. tumors and certain lymphomas have been reported, the clin. efficacy of histone deacetylase inhibitors in solid tumors has been limited.  This may be in part due to the poor pharmacokinetic of these drugs, resulting in inadequate tumor concns. of the drug.  SB939 is a new hydroxamic acid based histone deacetylase inhibitor with improved physicochem., pharmaceutical, and pharmacokinetic properties.  In vitro, SB939 inhibits class I, II, and IV HDACs, with no effects on other zinc binding enzymes, and shows significant antiproliferative activity against a wide variety of tumor cell lines.  It has very favorable pharmacokinetic properties after oral dosing in mice, with >4-fold increased bioavailability and 3.3-fold increased half-life over suberoylanilide hydroxamic acid (SAHA).  In contrast to SAHA, SB939 accumulates in tumor tissue and induces a sustained inhibition of histone acetylation in tumor tissue.  These excellent pharmacokinetic properties translated into a dose-dependent antitumor efficacy in a xenograft model of human colorectal cancer (HCT-116), with a tumor growth inhibition of 94% vs. 48% for SAHA (both at max. tolerated dose), and was also effective when given in different intermittent schedules.  Furthermore, in APCmin mice, a genetic mouse model of early-stage colon cancer, SB939 inhibited adenoma formation, hemocult scores, and increased hematocrit values more effectively than 5-fluorouracil.  Emerging clin. data from phase I trials in cancer patients indicate that the pharmacokinetic and pharmacol. advantages of SB939 are translated to the clinic.  The efficacy of SB939 reported here in two very different models of colorectal cancer warrants further investigation in patients.  Mol Cancer Ther; 9(3); 642-52.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrw7LYpqzlxHbVg90H21EOLACvtfcHk0liPDOd08hTX5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXivFens7Y%253D&md5=ffe0b917c40862c4ca0b414c8ef4c493</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-09-0689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-09-0689%26sid%3Dliteratum%253Aachs%26aulast%3DNovotny-Diermayr%26aufirst%3DV.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DHu%26aufirst%3DC.%2BY.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DSausgruber%26aufirst%3DN.%26aulast%3DYeo%26aufirst%3DP.%26aulast%3DGreicius%26aufirst%3DG.%26aulast%3DPettersson%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DA.%2BL.%26aulast%3DLoh%26aufirst%3DY.%2BK.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DGoh%26aufirst%3DK.%2BC.%26aulast%3DHentze%26aufirst%3DH.%26aulast%3DHart%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DEthirajulu%26aufirst%3DK.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26atitle%3DSB939%252C%2520a%2520Novel%2520Potent%2520and%2520Orally%2520Active%2520Histone%2520Deacetylase%2520Inhibitor%2520with%2520High%2520Tumor%2520Exposure%2520and%2520Efficacy%2520in%2520Mouse%2520Models%2520of%2520Colorectal%2520Cancer%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2010%26volume%3D9%26spage%3D642%26epage%3D652%26doi%3D10.1158%2F1535-7163.MCT-09-0689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madlambayan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kushner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dzinic, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taub, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Y.</span></span> <span> </span><span class="NLM_article-title">Antileukemic Activity and Mechanism of Action of the Novel PI3K and Histone Deacetylase Dual Inhibitor CUDC-907 in Acute Myeloid Leukemia</span>. <i>Haematologica</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">2225</span>– <span class="NLM_lpage">2240</span>, <span class="refDoi"> DOI: 10.3324/haematol.2018.201343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.3324%2Fhaematol.2018.201343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=30819918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFantb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2019&pages=2225-2240&author=X.+Liauthor=Y.+Suauthor=G.+Madlambayanauthor=H.+Edwardsauthor=L.+Polinauthor=J.+Kushnerauthor=S.+H.+Dzinicauthor=K.+Whiteauthor=J.+Maauthor=T.+Knightauthor=G.+Wangauthor=Y.+Wangauthor=J.+Yangauthor=J.+W.+Taubauthor=H.+Linauthor=Y.+Ge&title=Antileukemic+Activity+and+Mechanism+of+Action+of+the+Novel+PI3K+and+Histone+Deacetylase+Dual+Inhibitor+CUDC-907+in+Acute+Myeloid+Leukemia&doi=10.3324%2Fhaematol.2018.201343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Antileukemic activity and mechanism of action of the novel PI3K and histone deacetylase dual inhibitor CUDC-907 in acute myeloid leukemia</span></div><div class="casAuthors">Li, Xinyu; Su, Yongwei; Madlambayan, Gerard; Edwards, Holly; Polin, Lisa; Kushner, Juiwanna; Dzinic, Sijana H.; White, Kathryn; Ma, Jun; Knight, Tristan; Wang, Guan; Wang, Yue; Yang, Jay; Taub, Jeffrey W.; Lin, Hai; Ge, Yubin</div><div class="citationInfo"><span class="NLM_cas:title">Haematologica</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2225-2240</span>CODEN:
                <span class="NLM_cas:coden">HAEMAX</span>;
        ISSN:<span class="NLM_cas:issn">1592-8721</span>.
    
            (<span class="NLM_cas:orgname">Ferrata Storti Foundation</span>)
        </div><div class="casAbstract">I nduction therapy for patients with acute myeloid leukemia (AML) has remained largely unchanged for over 40 years, while overall survival rates remain unacceptably low, highlighting the need for new therapies.  The PI3K/Akt pathway is constitutively active in the majority of patients with AML.  Given that histone deacetylase inhibitors have been shown to synergize with PI3K inhibitors in preclin. AML models, we investigated the novel dual-acting PI3K and histone deacetylase inhibitor CUDC-907 in AML cells both in vitro and in vivo.  We demonstrated that CUDC-907 induces apoptosis in AML cell lines and primary AML samples and shows in vivo efficacy in an AML cell line-derived xenograft mouse model.  CUDC-907-induced apoptosis was partially dependent on Mcl-1, Bim, and c-Myc.  CUDC-907 induced DNA damage in AML cells while sparing normal hematopoietic cells.  Downregulation of CHK1, Wee1, and RRM1, and induction of DNA damage also contributed to CUDC-907-induced apoptosis of AML cells.  In addn., CUDC-907 treatment decreased leukemia progenitor cells in primary AML samples ex vivo, while also sparing normal hematopoietic progenitor cells.  These findings support the clin. development of CUDC-907 for the treatment of AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTfDoYe90TwbVg90H21EOLACvtfcHk0ljVkuc3jS0R_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFantb7I&md5=850a38d8ad0b426511353e6efa053985</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2018.201343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2018.201343%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DMadlambayan%26aufirst%3DG.%26aulast%3DEdwards%26aufirst%3DH.%26aulast%3DPolin%26aufirst%3DL.%26aulast%3DKushner%26aufirst%3DJ.%26aulast%3DDzinic%26aufirst%3DS.%2BH.%26aulast%3DWhite%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DKnight%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DTaub%26aufirst%3DJ.%2BW.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DGe%26aufirst%3DY.%26atitle%3DAntileukemic%2520Activity%2520and%2520Mechanism%2520of%2520Action%2520of%2520the%2520Novel%2520PI3K%2520and%2520Histone%2520Deacetylase%2520Dual%2520Inhibitor%2520CUDC-907%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DHaematologica%26date%3D2019%26volume%3D104%26spage%3D2225%26epage%3D2240%26doi%3D10.3324%2Fhaematol.2018.201343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avigan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenblatt, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stone, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kharbanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kufe, D.</span></span> <span> </span><span class="NLM_article-title">MUC1-C Oncoprotein Suppresses Reactive Oxygen Species-Induced Terminal Differentiation of Acute Myelogenous Leukemia Cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">4863</span>– <span class="NLM_lpage">4870</span>, <span class="refDoi"> DOI: 10.1182/blood-2010-10-296632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1182%2Fblood-2010-10-296632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=21422470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmsVWms7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2011&pages=4863-4870&author=L.+Yinauthor=Z.+Wuauthor=D.+Aviganauthor=J.+Rosenblattauthor=R.+Stoneauthor=S.+Kharbandaauthor=D.+Kufe&title=MUC1-C+Oncoprotein+Suppresses+Reactive+Oxygen+Species-Induced+Terminal+Differentiation+of+Acute+Myelogenous+Leukemia+Cells&doi=10.1182%2Fblood-2010-10-296632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">MUC1-C oncoprotein suppresses reactive oxygen species-induced terminal differentiation of acute myelogenous leukemia cells</span></div><div class="casAuthors">Yin, Li; Wu, Zekui; Avigan, David; Rosenblatt, Jacalyn; Stone, Richard; Kharbanda, Surender; Kufe, Donald</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4863-4870</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Acute myeloid leukemia (AML) cells are characterized by unlimited self-renewal and an impaired capacity to undergo terminal differentiation.  The MUC1 oncoprotein is aberrantly expressed in AML cells; however, there has been no evidence for involvement of MUC1 in myeloid leukemogenesis.  Cell-penetrating peptide inhibitors of the MUC1-C subunit block its oligomerization and thereby oncogenic function.  The present results demonstrate that treatment of human MOLM-14 and MV4-11 AML cells with these inhibitors is assocd. with arrest of growth, induction of late apoptosis/necrosis, and loss of self-renewal capacity.  Similar results were obtained with primary blasts from patients with AML.  Inhibition of MUC1-C was assocd. with increases in reactive oxygen species (ROS) and depletion of glutathione.  Increases in ROS have been linked to induction of hematopoietic cell differentiation along the myeloid lineage.  In this regard, inhibition of MUC1-C was assocd. with induction of a terminally differentiated myeloid phenotype in AML cell lines and primary blasts by an ROS-dependent mechanism.  These findings indicate that MUC1-C function is of importance to AML cell self-renewal and that inhibition of MUC1-C represents a potential therapeutic approach to induce terminal differentiation of AML cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1rwCDYhyZALVg90H21EOLACvtfcHk0ljVkuc3jS0R_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmsVWms7Y%253D&md5=a69f20f0d3dfd3872398b6bccb223e0c</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1182%2Fblood-2010-10-296632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2010-10-296632%26sid%3Dliteratum%253Aachs%26aulast%3DYin%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DAvigan%26aufirst%3DD.%26aulast%3DRosenblatt%26aufirst%3DJ.%26aulast%3DStone%26aufirst%3DR.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DKufe%26aufirst%3DD.%26atitle%3DMUC1-C%2520Oncoprotein%2520Suppresses%2520Reactive%2520Oxygen%2520Species-Induced%2520Terminal%2520Differentiation%2520of%2520Acute%2520Myelogenous%2520Leukemia%2520Cells%26jtitle%3DBlood%26date%3D2011%26volume%3D117%26spage%3D4863%26epage%3D4870%26doi%3D10.1182%2Fblood-2010-10-296632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asawa, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cyr, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urban, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallgren, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klumpp-Thomas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coussens, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marugan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henderson, M. J.</span></span> <span> </span><span class="NLM_article-title">A Widely-Applicable High-Throughput Cellular Thermal Shift Assay (CETSA) Using Split Nano Luciferase</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">9472</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-27834-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1038%2Fs41598-018-27834-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=29930256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=9472&author=N.+J.+Martinezauthor=R.+R.+Asawaauthor=M.+G.+Cyrauthor=A.+Zakharovauthor=D.+J.+Urbanauthor=J.+S.+Rothauthor=E.+Wallgrenauthor=C.+Klumpp-Thomasauthor=N.+P.+Coussensauthor=G.+Raiauthor=S.-M.+Yangauthor=M.+D.+Hallauthor=J.+J.+Maruganauthor=A.+Simeonovauthor=M.+J.+Henderson&title=A+Widely-Applicable+High-Throughput+Cellular+Thermal+Shift+Assay+%28CETSA%29+Using+Split+Nano+Luciferase&doi=10.1038%2Fs41598-018-27834-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">A widely-applicable high-throughput cellular thermal shift assay (CETSA) using split Nano Luciferase</span></div><div class="casAuthors">Martinez Natalia J; Asawa Rosita R; Cyr Matthew G; Zakharov Alexey; Urban Daniel J; Roth Jacob S; Wallgren Eric; Klumpp-Thomas Carleen; Coussens Nathan P; Rai Ganesha; Yang Shyh-Ming; Hall Matthew D; Marugan Juan J; Simeonov Anton; Henderson Mark J</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">9472</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Assessment of the interactions between a drug and its protein target in a physiologically relevant cellular environment constitutes a major challenge in the pre-clinical drug discovery space.  The Cellular Thermal Shift Assay (CETSA) enables such an assessment by quantifying the changes in the thermal stability of proteins upon ligand binding in intact cells.  Here, we present the development and validation of a homogeneous, standardized, target-independent, and high-throughput (384- and 1536-well formats) CETSA platform that uses a split Nano Luciferase approach (SplitLuc CETSA).  The broad applicability of the assay was demonstrated for diverse targets, and its performance was compared with independent biochemical and cell-based readouts using a set of well-characterized inhibitors.  Moreover, we investigated the utility of the platform as a primary assay for high-throughput screening.  The SplitLuc CETSA presented here enables target engagement studies for medium and high-throughput applications.  Additionally, it provides a rapid assay development and screening platform for targets where phenotypic or other cell-based assays are not readily available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD30bRGadycx0LsK7uFuY7fW6udTcc2eYfSdAACU3kW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c%252FgsFyktQ%253D%253D&md5=d4d1c56ac6d2cb79fe69911293cab32c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-27834-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-27834-y%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez%26aufirst%3DN.%2BJ.%26aulast%3DAsawa%26aufirst%3DR.%2BR.%26aulast%3DCyr%26aufirst%3DM.%2BG.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DUrban%26aufirst%3DD.%2BJ.%26aulast%3DRoth%26aufirst%3DJ.%2BS.%26aulast%3DWallgren%26aufirst%3DE.%26aulast%3DKlumpp-Thomas%26aufirst%3DC.%26aulast%3DCoussens%26aufirst%3DN.%2BP.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DYang%26aufirst%3DS.-M.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DMarugan%26aufirst%3DJ.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHenderson%26aufirst%3DM.%2BJ.%26atitle%3DA%2520Widely-Applicable%2520High-Throughput%2520Cellular%2520Thermal%2520Shift%2520Assay%2520%2528CETSA%2529%2520Using%2520Split%2520Nano%2520Luciferase%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D9472%26doi%3D10.1038%2Fs41598-018-27834-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span> <i>Molecular Operating Environment (MOE)</i>, <span class="NLM_edition">2013.08</span>; <span class="NLM_publisher-name">Chemical Computing Group ULC</span>, <span class="NLM_year">2013</span>; <a href="https://www.chemcomp.com/Products.htm" class="extLink">https://www.chemcomp.com/Products.htm</a> (accessed 2019-03-02).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Molecular+Operating+Environment+%28MOE%29%2C+2013.08%3B+Chemical+Computing+Group+ULC%2C+2013%3B+https%3A%2F%2Fwww.chemcomp.com%2FProducts.htm+%28accessed+2019-03-02%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Operating%2520Environment%2520%2528MOE%2529%26pub%3DChemical%2520Computing%2520Group%2520ULC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span> <span> </span><span class="NLM_article-title">Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization, and Performance of MMFF94</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">490</span>– <span class="NLM_lpage">519</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1096-987X(199604)17:5/6<490::AID-JCC1>3.0.CO;2-P</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1996&pages=490-519&author=T.+A.+Halgren&title=Merck+Molecular+Force+Field.+I.+Basis%2C+Form%2C+Scope%2C+Parameterization%2C+and+Performance+of+MMFF94&doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Merck molecular force field. I. Basis, form, scope, parameterization, and performance of MMFF94</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5 & 6</span>),
    <span class="NLM_cas:pages">490-519</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">This article introduces MMFF94, the initial published version of the Merck mol. force field (MMFF).  It describes the objectives set for MMFF, the form it takes, and the range of systems to which it applies.  This study also outlines the methodol. employed in parameterizing MMFF94 and summarizes its performance in reproducing computational and exptl. data.  Though similar to MM3 in some respects, MMFF94 differs in ways intended to facilitate application to condensed-phase processes in mol.-dynamics simulations.  Indeed, MMFF94 seeks to achieve MM3-like accuracy for small mols. in a combined "org./protein" force field that is equally applicable to proteins and other systems of biol. significance.  A second distinguishing feature is that the core protion of MMFF94 has primarily been derived from high-quality computational data-ca. 500 mol. structures optimized at the HF/6-31G* level, 475 structures optimized at the MP2/6-31G* level, 380 MP2/6-31* structures evaluated at a defined approxn. to the MP4SDQ/TZP level, and 1450 structures partly derived from MP2/6-31G* geometries and evaluated at the MP2/TZP level.  A third distinguishing feature is that MMFF94 has been parameterized for a wide variety of chem. systems of interest to org. and combinations of functional groups for which little, if any, useful exptl. data are available.  The methodol. used in parameterizing MMFF94 represents a fourth distinguishing feature.  Rather than using the common "functional group" approach, nearly all MMFF parameters have been detd. in a mutually consistent fashion from the full set of available computational data.  MMFF94 reproduces the computational data used in its parameterization very well.  In addn., MMFF94 reproduces exptl. bond lengths (0.014 Å root mean square [rms]), bond angles (1.2° rms), vibrational frequencies (61 cm-1 rms), conformational energies (0.38 kcal/mol rms), and rotational barriers (0.39 kcal/mol rms) very nearly as well as does MM3 for comparable systems.  MMFF94 also describes intermol. interactions in hydrogen-bonded systems in a way that closely parallels that given by the highly regarded OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAzCYfKnN83rVg90H21EOLACvtfcHk0lgl1WXT64p6HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhvVGmsbk%253D&md5=56fb073477b4f49d1dfbd786fc56a480</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291096-987X%2528199604%252917%253A5%252F6%253C490%253A%253AAID-JCC1%253E3.0.CO%253B2-P%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DMerck%2520Molecular%2520Force%2520Field.%2520I.%2520Basis%252C%2520Form%252C%2520Scope%252C%2520Parameterization%252C%2520and%2520Performance%2520of%2520MMFF94%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1996%26volume%3D17%26spage%3D490%26epage%3D519%26doi%3D10.1002%2F%28SICI%291096-987X%28199604%2917%3A5%2F6%3C490%3A%3AAID-JCC1%3E3.0.CO%3B2-P" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">The
UniProt Consortium</span> <span> </span><span class="NLM_article-title">Uniprot:
The Universal Protein Knowledgebase</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The%0AUniProt+Consortium&title=Uniprot%3A%0AThe+Universal+Protein+Knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniprot%253A%250AThe%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jadhav, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">Highly Predictive and Interpretable Models for PAMPA Permeability</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1266</span>– <span class="NLM_lpage">1276</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.12.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1016%2Fj.bmc.2016.12.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=28082071" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1266-1276&author=H.+Sunauthor=K.+Nguyenauthor=E.+Kernsauthor=Z.+Yanauthor=K.+R.+Yuauthor=P.+Shahauthor=A.+Jadhavauthor=X.+Xu&title=Highly+Predictive+and+Interpretable+Models+for+PAMPA+Permeability&doi=10.1016%2Fj.bmc.2016.12.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Highly predictive and interpretable models for PAMPA permeability</span></div><div class="casAuthors">Sun, Hongmao; Nguyen, Kimloan; Kerns, Edward; Yan, Zhengyin; Yu, Kyeong Ri; Shah, Pranav; Jadhav, Ajit; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1266-1276</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cell membrane permeability is an important determinant for oral absorption and bioavailability of a drug mol.  An in silico model predicting drug permeability is described, which is built based on a large permeability dataset of 7488 compd. entries or 5435 structurally unique mols. measured by the same lab using parallel artificial membrane permeability assay (PAMPA).  On the basis of customized mol. descriptors, the support vector regression (SVR) model trained with 4071 compds. with quant. data is able to predict the remaining 1364 compds. with the qual. data with an area under the curve of receiver operating characteristic (AUC-ROC) of 0.90.  The support vector classification (SVC) model trained with half of the whole dataset comprised of both the quant. and the qual. data produced accurate predictions to the remaining data with the AUC-ROC of 0.88.  The results suggest that the developed SVR model is highly predictive and provides medicinal chemists a useful in silico tool to facilitate design and synthesis of novel compds. with optimal drug-like properties, and thus accelerate the lead optimization in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-y8FhTLLqbrVg90H21EOLACvtfcHk0lgl1WXT64p6HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXntlOhsA%253D%253D&md5=2f6b60dfcd168b5bf837405ca52457b2</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.12.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.12.049%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DYu%26aufirst%3DK.%2BR.%26aulast%3DShah%26aufirst%3DP.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DHighly%2520Predictive%2520and%2520Interpretable%2520Models%2520for%2520PAMPA%2520Permeability%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D1266%26epage%3D1276%26doi%3D10.1016%2Fj.bmc.2016.12.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakharov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKew, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hop, C. E. C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span> <span> </span><span class="NLM_article-title">An Automated High-Throughput Metabolic Stability Assay Using an Integrated High-Resolution Accurate Mass Method and Automated Data Analysis Software</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">1653</span>, <span class="refDoi"> DOI: 10.1124/dmd.116.072017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=10.1124%2Fdmd.116.072017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=27417180" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1653&author=P.+Shahauthor=E.+Kernsauthor=D.-T.+Nguyenauthor=R.+S.+Obachauthor=A.+Q.+Wangauthor=A.+Zakharovauthor=J.+McKewauthor=A.+Simeonovauthor=C.+E.+C.+A.+Hopauthor=X.+Xu&title=An+Automated+High-Throughput+Metabolic+Stability+Assay+Using+an+Integrated+High-Resolution+Accurate+Mass+Method+and+Automated+Data+Analysis+Software&doi=10.1124%2Fdmd.116.072017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">An automated high-throughput metabolic stability assay using an integrated high-resolution accurate mass method and automated data analysis software</span></div><div class="casAuthors">Shah, Pranav; Kerns, Edward; Nguyen, Dac-Trung; Scott, Obach R.; Wang, Amy Q.; Zakharov, Alexey; McKew, John; Simeonov, Anton; Hop, Cornelis E. C. A.; Xu, Xin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism & Disposition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1653-1661</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">1521-009X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Advancement of in silico tools would be enabled by the availability of data for metabolic reaction rates and intrinsic clearance (CLint) of a diverse compd. structure data set by specific metabolic enzymes.  Our goal is to measure CLint for a large set of compds. with each major human cytochrome P 450 (P 450) isoenzyme.  To achieve our goal, it is of utmost importance to develop an automated, robust, sensitive, high-throughput metabolic stability assay that can efficiently handle a large vol. of compd. sets.  The substrate depletion method [in vitro half-life (t1/2) method] was chosen to det. CLint.  The assay (384-well format) consisted of three parts: (1) a robotic system for incubation and sample cleanup; (2) two different integrated, ultraperformance liq. chromatog./ mass spectrometry (UPLC/MS) platforms to det. the percent remaining of parent compd., and (3) an automated data anal. system.  The CYP3A4 assay was evaluated using two long t1/2 compds., carbamazepine and antipyrine (t1/2 > 30 min); one moderate t1/2 compd., ketoconazole (10 < t1/2 < 30 min); and two short t1/2 compds., loperamide and buspirone (t < 10 min).  Interday and intraday precision and accuracy of the assay were within acceptable range (∼12%) for the linear range obsd.  Using this assay, CYP3A4 CLint and t1/2 values for more than 3000 compds. were measured.  This high-throughput, automated, and robust assay allows for rapid metabolic stability screening of large compd. sets and enables advanced computational modeling for individual human P 450 isoenzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGos6dcRqLvacbVg90H21EOLACvtfcHk0lizmQNerbtZYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVyrtbrK&md5=92b4f23376d1cffa70c6fdc209b5569c</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1124%2Fdmd.116.072017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fdmd.116.072017%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DP.%26aulast%3DKerns%26aufirst%3DE.%26aulast%3DNguyen%26aufirst%3DD.-T.%26aulast%3DObach%26aufirst%3DR.%2BS.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DZakharov%26aufirst%3DA.%26aulast%3DMcKew%26aufirst%3DJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHop%26aufirst%3DC.%2BE.%2BC.%2BA.%26aulast%3DXu%26aufirst%3DX.%26atitle%3DAn%2520Automated%2520High-Throughput%2520Metabolic%2520Stability%2520Assay%2520Using%2520an%2520Integrated%2520High-Resolution%2520Accurate%2520Mass%2520Method%2520and%2520Automated%2520Data%2520Analysis%2520Software%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2016%26volume%3D44%26spage%3D1653%26doi%3D10.1124%2Fdmd.116.072017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4XE0','PDB','4XE0'); return false;">PDB: 4XE0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4LXZ','PDB','4LXZ'); return false;">PDB: 4LXZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EDU','PDB','5EDU'); return false;">PDB: 5EDU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XE0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5XE0','PDB','5XE0'); return false;">PDB: 5XE0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i135"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00193">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_36746"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00193?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00193</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">PI3K/HDAC enzyme inhibitory activities of selected 5-substituted quinazolinones; HPLC-MS chromatogram of compounds <b>19b</b>, <b>46b</b>, <b>48b</b>, <b>48c</b>, <b>48n</b>, and <b>48o</b>, respectively; figures showing compound <b>19b</b> bound to HDAC1, 2 (Glu567, Asp568), HDAC3 (Asp567, Asp568), HDAC8 (Tyr567, Asp568), HDAC4, 5, 7, 9 (Asp567, Thr568), HDAC10 (Asp567, Ala568), and HDAC11 (Pro567, Asn568), respectively; kinome scan of compound <b>48c</b>, and immunoblot analysis for PI3Kδ and HDAC6 expression (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular strings for all final compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_001.pdf">jm0c00193_si_001.pdf (1.42 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00193/suppl_file/jm0c00193_si_002.csv">jm0c00193_si_002.csv (4.25 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00193&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-8%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00193%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00193" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d263be4c3d60","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
