
<html lang="en"     class="pb-page"  data-request-id="c7e86cc3-eaf5-4e45-b621-6bb2679b96e8"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2020.63.issue-5;article:article:10.1021/acs.jmedchem.9b01526"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib" /></meta><meta name="dc.Creator" content="Greta  Bagnolini" /></meta><meta name="dc.Creator" content="Domenico  Milano" /></meta><meta name="dc.Creator" content="Marcella  Manerba" /></meta><meta name="dc.Creator" content="Fabrizio  Schipani" /></meta><meta name="dc.Creator" content="Jose Antonio  Ortega" /></meta><meta name="dc.Creator" content="Dario  Gioia" /></meta><meta name="dc.Creator" content="Federico  Falchi" /></meta><meta name="dc.Creator" content="Andrea  Balboni" /></meta><meta name="dc.Creator" content="Fulvia  Farabegoli" /></meta><meta name="dc.Creator" content="Francesca  De Franco" /></meta><meta name="dc.Creator" content="Janet  Robertson" /></meta><meta name="dc.Creator" content="Roberto  Pellicciari" /></meta><meta name="dc.Creator" content="Isabella  Pallavicini" /></meta><meta name="dc.Creator" content="Sebastiano  Peri" /></meta><meta name="dc.Creator" content="Saverio  Minucci" /></meta><meta name="dc.Creator" content="Stefania  Girotto" /></meta><meta name="dc.Creator" content="Giuseppina  Di Stefano" /></meta><meta name="dc.Creator" content="Marinella  Roberti" /></meta><meta name="dc.Creator" content="Andrea  Cavalli" /></meta><meta name="dc.Description" content="Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates. One example is the use of PARP inhibitors (PARPi) in oncology patients with BRCA mutations. Here, we..." /></meta><meta name="Description" content="Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates. One example is the use of PARP inhibitors (PARPi) in oncology patients with BRCA mutations. Here, we..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 10, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01526" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01526" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01526" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01526" /></link>
        
    
    

<title>Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01526" /></meta><meta property="og:title" content="Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0019.jpeg" /></meta><meta property="og:description" content="Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates. One example is the use of PARP inhibitors (PARPi) in oncology patients with BRCA mutations. Here, we exploit a new paradigm based on the possibility of triggering synthetic lethality using only small organic molecules (dubbed “fully small-molecule-induced synthetic lethality”). We exploited this paradigm to target pancreatic cancer, one of the major unmet needs in oncology. We discovered a dihydroquinolone pyrazoline-based molecule (35d) that disrupts the RAD51-BRCA2 protein–protein interaction, thus mimicking the effect of BRCA2 mutation. 35d inhibits the homologous recombination in a human pancreatic adenocarcinoma cell line. In addition, it synergizes with olaparib (a PARPi) to trigger synthetic lethality. This strategy aims to widen the use of PARPi in BRCA-competent and olaparib-resistant cancers, making fully small-molecule-induced synthetic lethality an innovative approach toward unmet oncological needs." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01526"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01526">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01526&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01526&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01526&amp;href=/doi/10.1021/acs.jmedchem.9b01526" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2588-2619</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01572" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01681" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Greta Bagnolini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Greta Bagnolini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Greta++Bagnolini">Greta Bagnolini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Domenico Milano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Domenico Milano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Domenico++Milano">Domenico Milano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marcella Manerba</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcella Manerba</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcella++Manerba">Marcella Manerba</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fabrizio Schipani</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabrizio Schipani</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabrizio++Schipani">Fabrizio Schipani</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jose Antonio Ortega</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jose Antonio Ortega</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jose+Antonio++Ortega">Jose Antonio Ortega</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dario Gioia</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dario Gioia</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dario++Gioia">Dario Gioia</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Federico Falchi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Federico Falchi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Federico++Falchi">Federico Falchi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andrea Balboni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Balboni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Balboni">Andrea Balboni</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fulvia Farabegoli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fulvia Farabegoli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fulvia++Farabegoli">Fulvia Farabegoli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Francesca De Franco</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Francesca De Franco</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">TES Pharma S.r.l., Via Palmiro Togliatti 22bis, I-06073 Corciano, Perugia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Francesca++De+Franco">Francesca De Franco</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Janet Robertson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janet Robertson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">TES Pharma S.r.l., Via Palmiro Togliatti 22bis, I-06073 Corciano, Perugia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janet++Robertson">Janet Robertson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roberto Pellicciari</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roberto Pellicciari</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">TES Pharma S.r.l., Via Palmiro Togliatti 22bis, I-06073 Corciano, Perugia, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roberto++Pellicciari">Roberto Pellicciari</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Isabella Pallavicini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Isabella Pallavicini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Experimental Oncology at the IEO, European Institute of Oncology IRCCS, IFOM-IEO Campus, Via Adamello 16, 20100 Milan, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Isabella++Pallavicini">Isabella Pallavicini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sebastiano Peri</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sebastiano Peri</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Experimental Oncology at the IEO, European Institute of Oncology IRCCS, IFOM-IEO Campus, Via Adamello 16, 20100 Milan, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sebastiano++Peri">Sebastiano Peri</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Saverio Minucci</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Saverio Minucci</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Experimental Oncology at the IEO, European Institute of Oncology IRCCS, IFOM-IEO Campus, Via Adamello 16, 20100 Milan, Italy</div><div class="loa-info-affiliations-info">Department of Biosciences, University of Milan, Via Celoria 26, 20100 Milan, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Saverio++Minucci">Saverio Minucci</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stefania Girotto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefania Girotto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefania++Girotto">Stefania Girotto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Giuseppina Di Stefano</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Giuseppina Di Stefano</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Experimental, Diagnostic and Specialty Medicine, University of Bologna, Via S. Giacomo 14, 40126 Bologna, Italy</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppina++Di+Stefano">Giuseppina Di Stefano</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marinella Roberti</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marinella Roberti</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>M.R.: phone, +39 051 2099738; email, <a href="/cdn-cgi/l/email-protection#5a373b2833343f36363b742835383f282e331a2f3433383574332e"><span class="__cf_email__" data-cfemail="1974786b70777c757578376b767b7c6b6d70596c77707b7637706d">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marinella++Roberti">Marinella Roberti</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9807-2886" title="Orcid link">http://orcid.org/0000-0001-9807-2886</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Andrea Cavalli</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andrea Cavalli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Computational and Chemical Biology, Istituto Italiano di Tecnologia, Via Morego 30, 16163 Genoa, Italy</div><div class="loa-info-affiliations-info">Department of Pharmacy and Biotechnology, University of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</div></div><span class="conrtib-corresp"><strong>*</strong>A.C.: phone, +39 010 71781530; email, <a href="/cdn-cgi/l/email-protection#54353a302631357a3735223538383d143d3d207a3d20"><span class="__cf_email__" data-cfemail="02636c667067632c616374636e6e6b426b6b762c6b76">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Cavalli">Andrea Cavalli</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6370-1176" title="Orcid link">http://orcid.org/0000-0002-6370-1176</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01526&amp;href=/doi/10.1021%2Facs.jmedchem.9b01526" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2588–2619</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 10, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 September 2019</li><li><span class="item_label"><b>Published</b> online</span>10 February 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 March 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01526" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01526</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="https://acsopenscience.org/open-access/licensing-options/"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by"><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_cc-1612971093887.svg" alt="CC: Creative Commons" /></img><span><strong>CC: </strong>Creative Commons</span></a><a href="https://creativecommons.org/licenses/by/4.0/"><img src="/pb-assets/images/licensesLogos/OA_Icons_by-1612971093257.svg" alt="BY: Credit must be given to the creator" /></img><span><strong>BY: </strong>Credit must be given to the creator</span></a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01526"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2243</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01526" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Synthetic Lethality in Pancreatic Cancer: Discovery of a New RAD51-BRCA2 Small Molecule Disruptor That Inhibits Homologous Recombination and Synergizes with Olaparib&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Greta&quot;,&quot;last_name&quot;:&quot;Bagnolini&quot;},{&quot;first_name&quot;:&quot;Domenico&quot;,&quot;last_name&quot;:&quot;Milano&quot;},{&quot;first_name&quot;:&quot;Marcella&quot;,&quot;last_name&quot;:&quot;Manerba&quot;},{&quot;first_name&quot;:&quot;Fabrizio&quot;,&quot;last_name&quot;:&quot;Schipani&quot;},{&quot;first_name&quot;:&quot;Jose&quot;,&quot;last_name&quot;:&quot;Antonio Ortega&quot;},{&quot;first_name&quot;:&quot;Dario&quot;,&quot;last_name&quot;:&quot;Gioia&quot;},{&quot;first_name&quot;:&quot;Federico&quot;,&quot;last_name&quot;:&quot;Falchi&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Balboni&quot;},{&quot;first_name&quot;:&quot;Fulvia&quot;,&quot;last_name&quot;:&quot;Farabegoli&quot;},{&quot;first_name&quot;:&quot;Francesca&quot;,&quot;last_name&quot;:&quot;De Franco&quot;},{&quot;first_name&quot;:&quot;Janet&quot;,&quot;last_name&quot;:&quot;Robertson&quot;},{&quot;first_name&quot;:&quot;Roberto&quot;,&quot;last_name&quot;:&quot;Pellicciari&quot;},{&quot;first_name&quot;:&quot;Isabella&quot;,&quot;last_name&quot;:&quot;Pallavicini&quot;},{&quot;first_name&quot;:&quot;Sebastiano&quot;,&quot;last_name&quot;:&quot;Peri&quot;},{&quot;first_name&quot;:&quot;Saverio&quot;,&quot;last_name&quot;:&quot;Minucci&quot;},{&quot;first_name&quot;:&quot;Stefania&quot;,&quot;last_name&quot;:&quot;Girotto&quot;},{&quot;first_name&quot;:&quot;Giuseppina&quot;,&quot;last_name&quot;:&quot;Di Stefano&quot;},{&quot;first_name&quot;:&quot;Marinella&quot;,&quot;last_name&quot;:&quot;Roberti&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Cavalli&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;2588-2619&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01526&quot;},&quot;abstract&quot;:&quot;Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates. One example is the use of PARP inhibitors (PARPi) in oncology patients with BRCA mutations. Here, we exploit a new paradigm based on the possibility of triggering synthetic lethality using only small organic molecules (dubbed “fully small-molecule-induced synthetic lethality”). We exploited this paradigm to target pancreatic cancer, one of the major unmet needs in oncology. We discovered a dihydroquinolone pyrazoline-based molecule (35d) that disrupts the RAD51-BRCA2 protein–protein interaction, thus mimicking the effect of BRCA2 mutation. 35d inhibits the homologous recombination in a human pancreatic adenocarcinoma cell line. In addition, it synergizes with olaparib (a PARPi) to trigger synthetic lethality. This strategy aims to widen the use of PARPi in BRCA-competent and olaparib-resistant cancers, making fully small-molecule-induced synthetic lethality an innovative approach toward unmet oncological needs.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01526&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01526" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01526&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01526" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01526&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01526" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01526&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01526&amp;href=/doi/10.1021/acs.jmedchem.9b01526" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01526" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01526" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01526%26sid%3Dliteratum%253Aachs%26pmid%3D32037829%26genre%3Darticle%26aulast%3DBagnolini%26date%3D2020%26atitle%3DSynthetic%2BLethality%2Bin%2BPancreatic%2BCancer%253A%2BDiscovery%2Bof%2Ba%2BNew%2BRAD51-BRCA2%2BSmall%2BMolecule%2BDisruptor%2BThat%2BInhibits%2BHomologous%2BRecombination%2Band%2BSynergizes%2Bwith%2BOlaparib%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D5%26spage%3D2588%26epage%3D2619%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (4)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/jmcmar.2020.63.issue-5/20200312/jmcmar.2020.63.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Synthetic lethality is an innovative framework for discovering novel anticancer drug candidates. One example is the use of PARP inhibitors (PARPi) in oncology patients with <i>BRCA</i> mutations. Here, we exploit a new paradigm based on the possibility of triggering synthetic lethality using only small organic molecules (dubbed “fully small-molecule-induced synthetic lethality”). We exploited this paradigm to target pancreatic cancer, one of the major unmet needs in oncology. We discovered a dihydroquinolone pyrazoline-based molecule (<b>35d</b>) that disrupts the RAD51-BRCA2 protein–protein interaction, thus mimicking the effect of <i>BRCA2</i> mutation. <b>35d</b> inhibits the homologous recombination in a human pancreatic adenocarcinoma cell line. In addition, it synergizes with olaparib (a PARPi) to trigger synthetic lethality. This strategy aims to widen the use of PARPi in <i>BRCA</i>-competent and olaparib-resistant cancers, making fully small-molecule-induced synthetic lethality an innovative approach toward unmet oncological needs.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/5"><issue-title>Women in Medicinal Chemistry</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04184" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04184" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Synthetic lethality is a new opportunity for discovering new anticancer molecules for personalized targeted therapies. The concept derives from genetic studies in model organisms.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> Two genes are synthetically lethal if the perturbation of either gene alone has no effect on cell viability, but the simultaneous impairment of both genes results in cell death. In principle, small organic molecules can target the synthetically lethal partner of an altered gene in cancer cells but not in normal cells. This creates opportunities to selectively kill cancer cells while sparing normal cells.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10 ref11 ref12">(6−12)</a></div><div class="NLM_p">The DNA repair and DNA damage response (DDR) pathways are suitable for the application of synthetic lethality as a novel anticancer therapeutic strategy.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> Genome instability is a hallmark of cancer.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> DNA damage occurs constantly in cells due to the continuous exposition to endogenous and exogenous stressors. Consequently, cells have evolved a complex coordinated DDR, which orchestrates a network of cellular processes to repair DNA damage and preserve genome integrity. DDR thus prevents the transmission of altered genetic material to daughter cells and acts as a tumor-suppressive barrier. Defects in DDR are associated with the accumulation of oncogenic mutations and genome instability, and they contribute to cancer initiation and progression. However, cancer cells with defects in one DDR pathway can become reliant on other pathways for their survival. Targeting these other DDR pathways can potentially cause selective cancer cell death through synthetic lethality. The classic example is the clinical application of poly (ADP-ribose)polymerase (PARP) inhibitors in oncology patients with <i>BRCA1/2</i> mutations. PARP is crucial for repairing DNA single-strand breaks (SSBs), whereas BRCA1/2 are important for repairing DNA double-strand breaks (DSBs) by homologous recombination (HR). The simultaneous impairment of both repair mechanisms results in cell-cycle arrest and apoptosis of cancer cells through synthetic lethality. In 2014, olaparib was the first PARP inhibitor (PARPi) approved to treat advanced ovarian cancer associated with defective <i>BRCA</i> genes.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In 2018, olaparib was approved to treat metastatic breast tumors associated with germline <i>BRCA</i> mutations.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In 2019, olaparib gained the FDA approval as first-line maintenance treatment of germline BRCA-mutated metastatic pancreatic cancer. It appears to be a new treatment option for this disease, which is one of the major unmet needs in oncology.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">One of BRCA2’s key mechanisms in DDR is to recruit RAD51, an evolutionarily conserved recombinase, at the site of DSBs where it performs DSB repair through HR.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Additionally, the expression of RAD51 and the rate of RAD51-mediated HR are both elevated in a wide variety of cancers (e.g., breast, pancreatic).<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Moreover, the cellular amount of RAD51 is positively correlated with resistance to radiotherapies or chemotherapies that induce DNA damage.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a></div><div class="NLM_p">The RAD51-BRCA2 interaction is mediated by eight well-conserved motifs, known as BRC repeats.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27">(24−27)</a> The X-ray crystallographic structure of the fourth BRC repeat (BRC4) is available in complex with the catalytic domain of RAD51, making the RAD51-BRCA2 interaction suitable for the structure-based design of small molecule inhibitors of protein–protein interactions (PPIs). Indeed, Lee et al. recently identified a small molecule inhibitor of RAD51-BRCA2 for potential cancer treatment.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a></div><div class="NLM_p">In this context, we recently proposed a new anticancer drug discovery concept, dubbed “fully small-molecules-induced synthetic lethality”.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> This concept combines RAD51-BRCA2 disruptors with olaparib to simultaneously impair two DNA repair pathways, thus mimicking the synthetic lethality described above. We carried out a successful virtual screening campaign at the FxxA pocket (i.e., zone I), one of the two RAD51 pockets responsible for BRC4 binding. This allowed us to discover a series of triazole-based compounds. Compounds <b>1</b> and <b>2</b> were selected as initial hit candidates (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). In line with our hypothesis, <b>2</b> increased the sensitivity to olaparib in pancreatic cancer cells (BxPC-3) with fully functional <i>BRCA2</i>. Notably, this synergistic effect was not observed in Capan-1, pancreas adenocarcinoma cells that lack <i>BRCA2</i>.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Furthermore, to discover more effective compounds, we conducted a chemical modification campaign around the triazole moiety. We also improved the biological screening cascade with experiments to characterize how the new compounds disrupt the RAD51-BRCA2 interaction and inhibit DSB repair. We obtained compound <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) with an improved profile relative to the initial hits (according to a biochemical ELISA assay) and a clear mechanism of action, allowing synergy with olaparib in cancer cells BxPC3, where olaparib is normally inactive. However, with <b>3</b>, we could not fully reproduce the paradigm of synthetic lethality.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> This could be due to its low-level potency, which did not cause HR inhibition greater than 40%. Additionally, the inherent resistance to apoptosis of BxPC3 cells, which bear a mutant p53, could have further prevented apoptosis. Therefore, further biological experiments and new classes of RAD51-BRCA2 are needed to confirm this paradigm and to assess its potential as an innovative anticancer strategy.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of the previously identified triazoles <b>1</b>–<b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To this end, we report here on the identification of a new class of RAD51-BRCA2 disruptors based on the dihydroquinolone pyrazoline moiety (<b>4d</b>–<b>57d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). We attempt to depict general structure–activity relationships (SARs) of this new class of compounds as RAD51-BRCA2 inhibitors and outline the biological profile of the most promising derivative <b>35d</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures and EC<sub>50</sub> of Compounds <b>4d</b>–<b>57d</b> on ELISA Assay<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0016.gif" alt="" id="gr16" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0017.gif" alt="" id="gr17" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0018.gif" alt="" id="GRAPHIC-d7e605-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Footnotes: *All points were tested in triplicate with error bars indicating the standard deviation. **NA: not active.</p></div></div><div></div></div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34571" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34571" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Hit Identification and Optimization</h3><div class="NLM_p">Targeting protein–protein interactions (PPIs) is an attractive strategy for designing innovative drugs for complex diseases such as cancer. Indeed, the first PPI inhibitors for cancer are now in clinical development.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a></div><div class="NLM_p">In this context, to identify RAD51-BRCA2 disruptors, we used the available X-ray crystallographic structure of the fourth BRC repeat (BRC4) in complex with the catalytic domain of RAD51.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> BRC4 binds RAD51 in two different hydrophobic pockets (zone I and zone II, respectively). One pocket (named zone I) can lodge BRC4’s FxxA motif (residues 1524–1527 of BRCA2) and is critical for RAD51 multimerization. The other pocket (named zone II) can lodge the BRC4’s LFDE motif (residues 1545–1548 of BRCA2) far from the oligomerization interface (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref24 ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref24 ref33 ref34 ref35">(24,33−35)</a> Recently, we ran a successful virtual screening campaign based on high-throughput docking at the FxxA pocket to identify the first RAD51-BRCA2 disruptors.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> To increase the chemical diversity and identify a novel class of RAD51-BRCA2 disruptors, we performed a second virtual screening campaign targeting the LFDE binding pocket (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_004.pdf" class="ext-link">Supporting Information</a>). This binding pocket is more evolutionarily conserved than the FxxA. Furthermore, mutation at the LFDE causes cellular lethality and failure of RAD51 assembly in nuclear foci at the site of DNA breaks in vivo. This further suggests this pocket as a critical site for RAD51’s mechanism of action.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> To the best of our knowledge, no inhibitor that binds the LFDE binding pocket has been reported so far in the literature. This may open up new possibilities for combining molecules targeting zone I and zone II toward a more in depth understanding of the mechanism of inhibition of RAD51-BRCA2 interaction.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) RAD51-BRCA2 BRC repeat complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N0W">1N0W</a>). RAD51 is represented as a surface, BRC4 as a cartoon. The two hot spots of the interaction between the proteins (Phe1524 and Phe1546) are highlighted in sticks. (B) Zone II magnification showing the interacting residues of BRC4 (yellow) and RAD51 (white).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Here, 42 small molecules were selected, purchased, and tested for their inhibitory activity using a competitive biochemical ELISA assay, as previously described by Rajendra et al.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Among the tested compounds, the commercially available dihydroquinolone pyrazoline derivative <b>4d</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>) was the best candidate in terms of EC<sub>50</sub> and chemical tractability. Its activity was confirmed by retesting the newly synthesized compound <b>4d</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Indeed, the dihydroquinolone pyrazoline moiety is a core structure of compounds with different biological targets.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a> The binding mode to RAD51 of both enantiomers of <b>4d</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), as obtained by induced-fit docking simulations, displays some points of interaction similar to those of the crystallographic BRC4-RAD51 complex. Specifically, the docking model suggests that (i) the fluorophenyl ring in position 5 of the pyrazoline lies (similar to the Phe1546 of BRC4) in a hydrophobic pocket outlined by the side chains of Leu204, Tyr205, Met251, Leu255, and Phe259 of RAD51 and (ii) the carboxyl group of the pyrazoline side chain forms an ionic interaction with the Arg250 (or Arg247) of RAD51, as does the side chain Glu1548 of BRC4. In addition, the model suggests that the carbonyl and the nitrogen of the dihydroquinolone moiety, together with the carbonyl group of the pyrazoline side chain, establish hydrogen bonds with Arg254 and Glu258. Notably, both enantiomers show the same global pattern of interactions.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Both enantiomers of compound <b>4d</b> docked into the LFDE binding site (site II) of RAD51 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N0W">1N0W</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overview of the optimization strategy of <b>4d</b> for SAR exploration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To improve the RAD51-BRCA2 inhibitory activity of <b>4d</b>, we conducted a chemical modification campaign around the dihydroquinolone pyrazoline core. We synthesized a chemical library that contained a variety of aromatic substitutions (red and blue regions) in combination with modifications of the acyl chain moiety (green region) (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). All compounds were synthesized and tested as racemic mixtures, after verifying that the enantiomers of the hit compound <b>4d</b> showed the same biochemical activity and the same binding mode (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, details in the <a class="ref internalNav" href="#sec2_3" aria-label="Biological Evaluation">Biological Evaluation</a> section). First, a series of different acyl chains (namely, acetyl, propionyl, 3-aminopropionyl, 4-amino-4-oxobutanoyl, 4-methoxy-4-oxobutanoyl, 3-(methylsulfonamido)propanoyl) was introduced on the pyrazoline nitrogen (<b>5d</b>–<b>10d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Next, the aromatic ring A was modified by replacing the fluorine atom with different substituents, including chlorine and bromine atoms and methoxy, <i>tert</i>-butyl, and trifluoromethyl groups, leaving the succinate acyl chain unchanged (<b>11d</b>–<b>16d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In addition, the aromatic ring A was replaced by different substituted biphenyl or heterocycle groups in order to probe its role (<b>17d</b>–<b>32d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). The ring A was also modified in combination with the propionyl (<b>33d</b>–<b>35d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) or acetyl substitution on the pyrazoline nitrogen (<b>36d</b>–<b>51d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Regarding the dihydroquinolone core, the chlorine atom in the C-ring was removed, leaving the acyl chain unchanged and introducing some different substituents in the phenyl ring A (<b>52d</b>–<b>57d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chemistry</h3><div class="NLM_p">The desired dihydroquinolone pyrazoline derivatives <b>4d</b>–<b>6d</b> and <b>11d</b>–<b>57d</b> were achieved, taking advantage of a common synthetic strategy previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The commercially available 2-aminobenzophenones <b>58</b> and <b>59</b> treated with ethyl acetoacetate afforded the corresponding methyl ketones <b>60a</b> and <b>61a</b>, which underwent base-catalyzed condensation with the appropriate aryl aldehydes <b>62</b>–<b>85</b> yielding the α,β-unsaturated aryl ketones <b>86b</b>–<b>114b</b>. In turn, <b>86b</b>–<b>114b</b> were treated with hydrazine monohydrate to yield the pyrazoline derivatives <b>86c</b>–<b>114c</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The pyrazoline amines <b>86c</b>–<b>114c</b> functionalized with succinic anhydride <b>115</b>, acetic anhydride <b>116</b>, or propionic acid <b>117</b> afforded the corresponding desired dihydroquinolone pyrazoline derivatives <b>4d</b>–<b>6d</b> and <b>11d</b>–<b>57d</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). <b>7d</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>) was obtained by coupling <b>86c</b> with the commercially available 3-((<i>tert</i>-butoxycarbonyl)amino)propanoic acid <b>118</b> to achieve the Boc-aminopyrazoline derivative <b>119</b>. In turn, <b>119</b> was Boc-deprotected to afford the desired <b>7d</b>. <b>8d</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>) was obtained by HATU-mediated coupling of <b>4d</b> with ammonium chloride. <b>9d</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>) was afforded by coupling <b>86c</b> with the commercially available 4-methoxy-4-oxobutanoic acid <b>120</b>. The synthesis of <b>10d</b> (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>) began with the commercially available methyl 3-aminopropanoate <b>121</b>, which was treated with methanesulfonyl chloride <b>122</b> to afford the corresponding methyl 3-(methylsulfonamido)propanoate <b>123</b>, which underwent basic hydrolysis to give 3-(methylsulfonamido)propanoic acid <b>124</b>. The coupling reaction of <b>86c</b> with <b>124</b> afforded the desired <b>10d</b>.</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0010.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Dihydroquinolone Pyrazoline Intermediates <b>86c</b>–<b>114c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl acetoacetate, DMF, 120 °C μWave or reflux, <b>60a</b> quantitative, <b>61a</b> 53%; (b) KOH, EtOH/H<sub>2</sub>O 4:3 v/v (0.05 M), 0 °C to rt, 53%, quantitative; (c) hydrazine monohydrate, EtOH, 110 °C μWave, 45 min, 64%, quantitative.</p></p></figure><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0011.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Final Dihydroquinolone Pyrazolines <b>4d</b>–<b>57d</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0012.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>7d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HOBt, EDCI, DCM, overnight, yield 25%; (b) HCl 4 M in dioxane, rt, 15 min; (c) NaOH 0.5 M in EtOAc, rt, 15 min, yield 50%.</p></p></figure><figure id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0013.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>8d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU (1.5 equiv), EDC (1.5 equiv), DCM, DMF, ammonium chloride (5.0 equiv), DIPEA (4.0 equiv), rt, 26 h, yield 46%.</p></p></figure><figure id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0014.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>9d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HOBt (1.1 equiv), EDC (1.1 equiv), TEA (2.2. equiv) DCM, rt, 16 h, yield 42%.</p></p></figure><figure id="sch6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0015.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>10d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TEA (5.0 equiv), anhydrous DCM, methanesulfonyl chloride <b>122</b> (2.0 equiv), rt, 48 h, yield 82%; (b) MeOH/THF (1:1 v/v), 2 M LiOH, rt, 16 h, yield 82%; (c) HOBt (1.1 equiv), EDC (1.1 equiv), TEA (2.2 equiv), DCM, rt, 16 h, yield 15%.</p></p></figure><div class="NLM_p last">The aldehydes <b>67</b>, <b>70</b>, <b>74</b>–<b>84</b>, not commercially available, were prepared following standard procedures as reported in <a href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_004.pdf" class="ext-link">Supporting Information (Schemes S1–S9)</a>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Biological Evaluation</h3><div class="NLM_p">To investigate the mechanism of action of the new dihydroquinolone pyrazoline derivatives, different biological assays were performed. As a primary screening, the ability of compounds <b>5d</b>–<b>57d</b> to inhibit RAD51-BRCA2 interaction was investigated with a competitive biochemical ELISA assay against the parent compound <b>4d</b> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). This assay is effective in evaluating the ability of new molecules to compete with BRC4 to bind to RAD51.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Replacing the acyl chain of the pyrazoline nitrogen yielded compounds (<b>5d</b>–<b>10d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) that had reduced inhibitory activity (<b>5d</b>, EC<sub>50</sub> = 50 ± 10 μM; <b>6d</b>, EC<sub>50</sub> = 34 ± 3 μM) or were completely inactive (<b>7d</b>–<b>10d</b>). These data indicate that no improvement in RAD51-BRCA2 inhibitory activity was achieved with this subset of compounds relative to the parent <b>4d</b> (EC<sub>50</sub> = 16 ± 4 μM). Replacing the fluorine atom on ring A with bromine led to <b>12d</b> (EC<sub>50</sub> = 16 ± 2 μM), which shows the same activity as the initial hit <b>4d</b>. The activity was affected when the fluorine was replaced with chlorine (<b>11d</b>, EC<sub>50</sub> = 59 ± 8 μM) and trifluoromethyl groups (<b>16d</b>, EC<sub>50</sub> = 29 ± 2 μM). The replacement of fluorine with electron-donating groups yielded the inactive compounds (<b>13d</b>–<b>15d</b>). For the subset of dihydroquinolone pyrazolines, in which the aromatic ring A was replaced by different substituted biphenyl or heterocycle groups (<b>17d</b>–<b>32d</b>, <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>), the <i>N</i>-methylindole derivative <b>22d</b> showed a fairly good potency with an EC<sub>50</sub> = 2 ± 0.5 μM, 8 times higher than that of the parent <b>4d</b> (EC<sub>50</sub> = 16 ± 4 μM). <b>18d</b>–<b>21d</b>, <b>23d</b>, <b>25d</b>, <b>30d</b>, and <b>31d</b> were active at micromolar range, all very similar to the initial hit. A drop in activity was observed with compound <b>32d</b>, while derivatives <b>24d</b>, <b>26d</b>–<b>29d</b> were inactive. The replacement of the aromatic ring A in combination with the substitution of the pyrazoline nitrogen with either propionyl or acetyl chain yielded compounds <b>33d</b>–<b>51d</b>. The 1-<i>N</i>-acetyl-5-(1-<i>N</i>-propyl)indazolylpyrazoline <b>49d</b> showed the best activity of the series with EC<sub>50</sub> = 0.95 ± 0.05 μM, while <b>33d</b>–<b>36d</b>, <b>39d</b>, <b>45d</b> showed an activity very similar to that of the initial hit. A drop in potency was observed with compounds <b>37d</b>, <b>42d</b>–<b>44d</b>, <b>46d</b>–<b>47d</b>, and <b>51d</b>, while <b>38d</b>, <b>40d</b>–<b>41d</b>, <b>48d</b>, and <b>50d</b> were inactive. Finally, removing the chlorine atom on the dihydroquinolone core led to the completely inactive compounds <b>52d</b>–<b>57d</b>, suggesting an active role for the halogen. The enantiomers (<b>4d-I</b> and <b>4d-II</b>) of the racemic hit compound <b>4d</b>, separated via reverse phase chiral chromatography, showed a very similar inhibitory activity (<b>4d-I</b>, EC<sub>50</sub> = 4 ± 0.5 μM; <b>4d-II</b>, 10 ± 1 μM) to that of the parent <b>4d</b>, suggesting no stereochemical preference of these compounds for the hypothesized molecular target RAD51 (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). As expected for PPI disruptors, the SARs of the new series of dihydroquinolone pyrazoline were rather complex to rationalize, with many cliffs and spikes that were difficult to understand. Nonetheless, the SAR campaign allowed us to identify several compounds with interesting EC<sub>50</sub> values ranging from 0.95 to 20 μM. Accordingly, <b>4d</b>, <b>12d</b>, <b>18d</b>, <b>20d</b>–<b>23d</b>, <b>30d</b>, <b>31d</b>, <b>33d</b>–<b>36d</b>, <b>39d</b>, <b>45d</b>, and <b>49d</b> were submitted to cell-based study (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Preliminary Biological Screening of Compounds Showing EC<sub>50</sub> on ELISA Assay Ranging from 0.95 to 20 μM</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">EC<sub>50</sub> ELISA (μM)</th><th class="colsep0 rowsep0" align="center">preliminary biological screening</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4d</b></td><td class="colsep0 rowsep0" align="left">16 ± 4</td><td class="colsep0 rowsep0" align="left">HR inhibition = 10%, 40 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="left">16 ± 2</td><td class="colsep0 rowsep0" align="left">HR inhibition = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18d</b></td><td class="colsep0 rowsep0" align="left">13 ± 1</td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20d</b></td><td class="colsep0 rowsep0" align="left">20 ± 2</td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21d</b></td><td class="colsep0 rowsep0" align="left">20 ± 4</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22d</b></td><td class="colsep0 rowsep0" align="left">2 ± 0.5</td><td class="colsep0 rowsep0" align="left">HR inhibition = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = not present at 5 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23d</b></td><td class="colsep0 rowsep0" align="left">10 ± 2</td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30d</b></td><td class="colsep0 rowsep0" align="left">15 ± 3</td><td class="colsep0 rowsep0" align="left">NE<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31d</b></td><td class="colsep0 rowsep0" align="left">17 ± 4</td><td class="colsep0 rowsep0" align="left">HR inhibition = NDD<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = present at 20 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33d</b></td><td class="colsep0 rowsep0" align="left">8 ± 2</td><td class="colsep0 rowsep0" align="left">HR inhibition = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34d</b></td><td class="colsep0 rowsep0" align="left">19 ± 1</td><td class="colsep0 rowsep0" align="left">HR inhibition = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = NE<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35d</b></td><td class="colsep0 rowsep0" align="left">19 ± 1</td><td class="colsep0 rowsep0" align="left">HR inhibition = 54%, 20 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = present at 15 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36d</b></td><td class="colsep0 rowsep0" align="left">10 ± 0.7</td><td class="colsep0 rowsep0" align="left">HR inhibition = 24%, 10 μM</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39d</b></td><td class="colsep0 rowsep0" align="left">15 ± 4</td><td class="colsep0 rowsep0" align="left">HR inhibition = NDD<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>45d</b></td><td class="colsep0 rowsep0" align="left">18 ± 1</td><td class="colsep0 rowsep0" align="left">HR inhibition = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>49d</b></td><td class="colsep0 rowsep0" align="left">0.95 ± 0.05</td><td class="colsep0 rowsep0" align="left">HR inhibition = not present</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">olaparib association = not present</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">NE, not evaluable.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">NDD, not dose-dependent.</p></div></div></div><div class="NLM_p">Our working hypothesis is that compounds disrupting RAD51-BRCA2 interaction should affect HR repair and increase the efficacy of PARPi in treating breast, ovarian, and pancreatic cancer. For its clinical relevance, pancreatic adenocarcinoma was selected as our final model for cell-based experiments, and BxPC3 cells were selected for a straightforward comparison with the previously reported triazole derivatives.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> BxPC3 is derived from a human adenocarcinoma that expresses fully functional BRCA2.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> As shown in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and according to the rationale of our hypothesis, our preliminary screening consisted of verifying the efficacy of compounds in inhibiting cell HR and/or in increasing the antiproliferative effect of olaparib. For each compound, both parameters were verified using only one or two doses, in the range of the EC<sub>50</sub> obtained with the ELISA test. HR activity was assessed by evaluating the recombination rate between two transfected plasmids, using a commercially available assay. This preliminary investigation allowed us to rapidly exclude molecules showing (i) low or no activity (<b>4d</b>, <b>12d</b>, <b>18d</b>, <b>20d</b>, <b>22d</b>, <b>23d</b>, <b>33d</b>, <b>45d</b>, <b>49d</b>), (ii) poor solubility in cell culture media (<b>21d</b>, <b>30d</b>, <b>34d</b>), (iii) discrepancy between the data obtained in the two different screening procedures such as HR inhibition not confirmed by increased olaparib efficacy in the viability assay or, on the contrary, increased olaparib efficacy without HR inhibition (<b>31d</b>, <b>36d</b>), (iv) a non-dose-dependent effect (<b>39d</b>). As for the emissucinic acid-containing compounds (<b>4d</b>, <b>12d</b>, <b>18d</b>, <b>20d</b>–<b>23d</b>, <b>30d</b>, <b>31d</b>), one of the reasons for their lower potency in cells might be the general poor permeability, likely related to the ionizable acid moiety. The data reported in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> show that <b>35d</b> was the most promising compound in the HR activity test. We then characterized <b>35d</b> in additional biophysical and cell-based experiments.</div><div class="NLM_p">To further assess the physical interaction between <b>35d</b> and RAD51, a microscale thermophoresis (MST) assay was performed on the recombinant human RAD51 (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). The binding assay allowed us to determine the dissociation constant (<i>K</i><sub>d</sub>) for the RAD51-<b>35d</b> interaction and binding. The final binding curve (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) shows that <b>35d</b> binds to RAD51 with a <i>K</i><sub>d</sub> value of 11 ± 6 μM. This is in agreement with the ELISA assay, supporting the initial hypothesis that <b>35d</b> could act as a RAD51-BRCA2 disruptor.</div><figure id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0005.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MST analysis of His-hRAD51-<b>35d</b> binding. Titration curve of (RED-tris-NTA 2nd Generation)-His-hRAD51 (80 nM) with increasing concentrations of <b>35d</b>. Sigmoidal fitting curve was obtained using the Affinity Analysis software of NanoTemper Technologies. MST data are the average of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To further characterize the effect of <b>35d</b> on HR, the compound was administered at different doses to BxPC3 cells for 5 h simultaneously with plasmid transfection; as shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, it produced a statistically significant dose–response effect in reducing cell HR. The compound was tested up to 40 μM (an upper limit in terms of solubility), and we estimated the dose causing a 50% inhibition of HR by applying the polynomial regression to the collected data. The EC<sub>50</sub> was 18.4 μM.</div><figure id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0006.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Effect on HR caused by <b>35d</b> administered to BxPC3 cells during plasmid transfection (5 h). HR was evaluated by real-time PCR, as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Data were statistically analyzed using the column statistics of Prism 5 software, which applies the inferences analysis and the one-sample <i>t</i> test. The observed inhibitory effect was significantly different from 0 (the level of untreated cultures) for all tested doses, with <i>p</i> < 0.05. (B, C) Immunofluorescence detection of RAD51 in BxPC-3 nuclei after a treatment with 50 μM cisplatin (Cpl) given separately or in combination with 20 μM <b>35d</b>. Experimental details are reported in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. (B) Representative pictures showing DAPI-stained cell nuclei and the corresponding immune-labeling of RAD51 localization. In untreated cells (CTR), RAD51 labeling is clearly evident in cytoplasm and does not appear in cell nuclei. In the pictures of Cpl-exposed cells, nuclear localization of the protein is clearly evident in 3 out of the 6 shown nuclei. A higher magnification detail was included for this sample. (C) The bar graph shows the percentage of RAD51-labeled nuclei counted by two independent observers who analyzed the treated cultures. Data were statistically evaluated by applying the one-way ANOVA, which indicated a significantly increased nuclear RAD51 labeling caused by Cpl (<i>p</i> < 0.05) and no statistically significant difference between cells treated with <b>35d</b> and those exposed to <b>35d</b> + Cpl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">An additional evidence of compromised HR was obtained by assessing the localization of RAD51 in BxPC-3 nuclei after DNA damage. The results of this experiment are reported in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B,C. To obtain massive DNA damage, BxPC-3 cultures were exposed for 1 h to 50 μM cisplatin. The immunohistochemical staining of RAD51 shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B revealed evident nuclear foci in cisplatin-treated cells, which appeared significantly reduced when the drug was administered in association with 20 μM <b>35d</b>. Furthermore, the percentage of RAD51-labeled nuclei measured in cultures exposed to 20 μM <b>35d</b> was superimposable to that observed in cells exposed to the association of cisplatin with 20 μM <b>35d</b> (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C).</div><div class="NLM_p">These results were in good agreement with the ELISA and MST outcomes, ultimately pointing to <b>35d</b> as a novel RAD51-BRCA2 disruptor with a clear capability to interfere with HR.</div><div class="NLM_p">The sustained inhibition of HR in cells should result in increased DNA damage, ultimately leading to mutations and chromosome aberrations; these effects are expected to be further amplified by PARP inhibition. The extent of DNA damage produced in cells treated for 48 h with 20 μM <b>35d</b>, administered singularly or in association with 10 μM olaparib, was studied by evidencing nuclear γ-H2AX foci by immunofluorescence. The experiment was performed on both BxPC-3 and Capan-1 cultures. Capan-1 cells are derived from a human pancreatic adenocarcinoma (very similar to BxPC3cells) and are BRCA2-defective.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> As a consequence, they do not operate RAD51-BRCA2-dependent HR. The olaparib dose was selected on the basis of previously obtained results with the same cell cultures.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Results are reported in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A,B.</div><figure id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0007.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A, B) Evaluation of DNA damage through immune detection of nuclear γ-H2AX foci in BxPC-3 and Capan-1 cells exposed for 48 h to olaparib (10 μM) or <b>35d</b> (20 μM), given alone or in combination. (A) Representative pictures showing DAPI-stained cell nuclei and the corresponding immune-labeling of γ-H2AX. In BxPC-3 cells, coadministration of <b>35d</b> and olaparib produced increased γ-H2AX labeling. A higher magnification detail was included for this sample. As expected, Capan-1 cells showed a constitutive γ-H2AX labeling that was highly increased by olaparib but was unaffected by <b>35d</b> coadministration. (B) The bar graph shows the percentage of γ-H2AX -labeled nuclei counted by two independent observers who analyzed the treated cultures. Data obtained in bxPC-3 cells were statistically evaluated by applying the one-way ANOVA, which indicated a statistically significant difference between the cultures treated with olaparib and those exposed to olaparib + <b>35d</b>. (C, D) Evaluation of micronuclei generation in BxPC3 cells treated (72 h) with <b>35d</b> and olaparib, given alone or in combination. (C) Representative pictures showing DAPI-stained cell nuclei. White asterisks indicate the presence of micronuclei. (D) The percentage of cells bearing micronuclei was estimated by two independent observers, by analyzing 100–250 cells for each treatment sample. The obtained results were statistically analyzed by applying the one-way ANOVA, which indicated a <i>p</i> value of <0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The microscope pictures shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A showed increased evidence of γ-H2AX labeling in nuclei of BxPC-3 cells exposed to the compounds’ association, compared to the labeling observed in cultures treated with olaparib. In Capan-1 cultures, the constitutive γ-H2AX labeling in nuclei appeared more evident than that observed in BxPC-3 cells.</div><div class="NLM_p">Moreover, in agreement with data showing higher PARPi sensitivity for cells lacking functional HR,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> these cultures showed increased nuclear labeling when exposed to olaparib. Notably, this labeling was not further enhanced by <b>35d</b> coadministration. The percentage of γ-H2AX-labeled nuclei measured in all treated cultures is shown in the bar graphs of <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B.</div><div class="NLM_p">The sustained and increased DNA damage produced in BxPC-3 nuclei generates chromosomal aberrations which can be visualized through the presence of small DNA-staining bodies outside the main nucleus (micronuclei). The microscope pictures of <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C show the appearance of this feature in BxPC3 cells exposed for 72 h to 20 μM <b>35d</b>, administered alone or in combination with 10 μM olaparib. The percentage of cells bearing micronuclei is reported in the graph of <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D. Notably, cells bearing micronuclei were markedly more frequent in cultures treated with the <b>35d</b>/olaparib combination.</div><div class="NLM_p">Taken together, the results reported in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a> significantly support the requested mechanism of action for <b>35d</b>. Therefore, we conducted further experiments to test whether the <b>35d</b>/olaparib combination would induce synthetic lethality.</div><div class="NLM_p">We simultaneously evaluated cell viability and cell death, measured at 72 h in BxPC3 cells exposed to <b>35d</b> alone or in combination with 10 μM olaparib (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The statistical analysis (see legend of <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>) compared the results for cultures treated with different doses of <b>35d</b> or the <b>35d</b>/olaparib combination (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A, upper panel). When applied to the data of the cell viability experiment, this analysis indicated a statistically significant difference produced by <b>35d</b> in combination with olaparib in all the treated BxPC3 cultures, with <i>p</i> values ranging from 0.01 (10 μM <b>35d</b>) to 0.0001 (15 and 20 μM <b>35d</b>). When we applied the same analysis to the data of the cell death experiment, we found no statistically significant increase in cell death for olaparib when coadministered with 10–15 μM <b>35d</b>. Notably, in cultures exposed to olaparib + 20 μM <b>35d</b> (the concentration producing the highest inhibition of HR), the evidence for cell death was markedly increased and statistically significant, with <i>p</i> < 0.0001 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A, lower panel).</div><figure id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0008.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) BxPC3 cell viability and death measured after 72 h exposure to <b>35d</b> and 10 μM olaparib, given alone or in combination. Data were analyzed by two-way ANOVA using the two treatments (<b>35d</b> and olaparib) as variables. In the cell viability experiment, Bonferroni post-test indicated a statistically significant difference produced by olaparib coadministration in all BxPC3 cultures treated with the different <b>35d</b> doses, with <i>p</i> values ranging from 0.01 (10 μM <b>35d</b>) to 0.0001 (15 and 20 μM <b>35d</b>). The same analysis was applied to the data from the cell death experiment and indicated that no statistically significant increase in cell death was produced by olaparib when coadministered with 10–15 μM <b>35d</b>. In cultures exposed to olaparib + 20 μM <b>35d</b> the evidence for cell death was markedly increased and statistically significant, with <i>p</i> < 0.0001. (B) After the 72 h treatment, BxPC3 cells were stained with vital dyes. As shown in the microscope pictures, the only culture displaying sharp evidence of cell death was that exposed to the combination of olaparib/20 μM <b>35d</b>, as demonstrated by PI nuclear staining.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When cell death is a consequence of progressive DNA damage accumulation induced by simultaneous PARP and HR inhibition, we would expect it to emerge gradually over time. To confirm and better characterize the lethality observed in cultures treated with the <b>35d</b>/olaparib combination, we therefore considered it inappropriate to conduct a simple evaluation of the commonly used markers (e.g., caspase activation), since these can show very transient changes. Instead, we observed cell morphology and reaction to vital dyes after the 72 h treatment, when the experiment of <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A indicated a significant level of cell death.</div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B shows microscope pictures of BxPC3 cells stained with mixed DAPI and PI. The simultaneous use of these two dyes can demonstrate cell death and indicate the death pattern.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> DAPI is cell-permeable and shows nuclear morphology; healthy cells appear to display normal nuclear morphology in the absence of PI staining, since this dye is not cell-permeable. Cells undergoing apoptosis display nuclear condensation, which is indicated by increased DAPI staining. PI staining indicates compromised membrane integrity, which characterizes necrotic cells and late-apoptotic cells maintained in culture.</div><div class="NLM_p">The microscope pictures show that untreated BxPC3 cells display only a moderate DAPI staining of their nucleus. Nuclear DAPI staining is slightly increased in cells treated with <b>35d</b> alone but is strikingly bright only in cells treated with the <b>35d</b>/olaparib combination. Furthermore, PI staining appeared only in these cultures, confirming the manifestation of synthetic lethality. The simultaneous marked staining of these cells with both dyes could indicate an apoptotic phenomenon followed by compromised membrane integrity because of cell persistence in culture.</div><div class="NLM_p">As expected, the sustained and increased DNA damage observed in BxPC3 cells treated with the <b>35d</b>/olaparib combination (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>) reproduced the desired mechanism of synthetic lethality (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). These results are also relevant given the mutated p53 status of BxPC3 cells, which should make them more resistant to mechanisms that induce cell death.</div><div class="NLM_p">Finally, the antiproliferative effect of the <b>35d</b>/olaparib combination was also studied on the HR-defective Capan-1 culture and on a non-neoplastic cell line derived from human kidney (HK-2). Moreover, to evaluate the antineoplastic potency of the <b>35d</b>/olaparib association, we also calculated the combination index (CI) of the two compounds, using the procedure previously described<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</div><figure id="fig9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0009.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Antiproliferative effect caused by <b>35d</b> and its association with olaparib, measured in Capan-1 cells (which do not operate RAD51-BRCA2-dependent HR) and in normal immortalized human renal cells (HK-2). The same procedures described for BxPC3 cells were used here. (B) Combination indexes of the olaparib/<b>35d</b> association measured in the three used cell lines, calculated according to the method previously described.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> For BxPC-3 cells, the data reported in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A (cell proliferation) were used. Data were analyzed using the column statistics and the one-sample <i>t</i> test of Prism 5 software. For BxPC-3 cells, this test showed a statistically significant difference from 0.8 for the combination of olaparib and 20 μΜ <b>35d</b>. Values of <0.8 indicate synergism between the two compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">According to this method, CI < 0.8 indicates synergism while a result ranging from 0.8 to 1.2 indicates additive effects. This evaluation (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>) was performed for all the three studied cell cultures; for BxPC-3 cells, the data reported in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A (cell viability experiment) were used for the calculation of CI. Interestingly, in these cells the potency of the compound combination increased in parallel with the <b>35d</b> dose, reaching a statistically significant difference from 0.8 at the 20 μM concentration. The effects observed when 20 μM <b>35d</b> is combined with 10 μM olaparib may thus indicate a synergism between the two compounds.</div><div class="NLM_p">In HK-2 cultures, no dose of <b>35d</b> appeared to significantly increase the antiproliferative power of olaparib (two-way ANOVA). In Capan-1 cells, the same statistical evaluation showed significantly increased antiproliferative effects for <b>35d/</b>olaparib. However, in these cells, the CI did not show increasing potency of the compound combination with dose escalation. Moreover, at all tested doses, the value of CI was not significantly different from 1, which is the level measured in BxPC3 cells exposed to 10 μM <b>35d</b>, a dose that did not relevantly affect (≈10%) HR (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). This result further supports the idea that the overadditive effects in the <b>35d</b>/olaparib combination arise from and are strictly related to HR inhibition.</div><div class="NLM_p last">Overall, these data supported our working hypothesis that combining a RAD51-BRCA2 small molecule disruptor with olaparib could be a valuable strategy for inducing synthetic lethality in cancer cell lines with fully functional <i>BRCA</i> genes and homologous recombination. This includes pancreatic cancer, which is a major unmet need in oncology. We believe that this paradigm could be used to discover innovative anticancer therapies based on other lethal gene pairs using a similar medicinal chemistry strategy.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62455" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62455" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Continuing our research line, we described a series of dihydroquinolone pyrazoline derivatives as a new class of RAD51-BRCA2 disruptors. Compound <b>4d</b> was identified as a promising hit, and subsequent SAR efforts yielded <b>35d</b> with the desired biological profile. As expected, <b>35d</b> bound to its target (RAD51) and inhibited the protein–protein interaction between RAD51 and BRCA2. Importantly, it synergized and reproduced the paradigm of synthetic lethality in combination with olaparib in pancreatic cancer cells (BxPC3). These effects were strictly related to the extent of HR inhibition in a dose-dependent trend. This is the most promising achievement of the current investigation and supports our working hypothesis that one can trigger synthetic lethality using only small organic molecules. Interestingly, the observed synthetic lethality was triggered by tackling two biochemically different mechanisms: enzyme inhibition (PARP) and protein–protein disruption (RAD51-BRCA2). This highlights how complex and diverse mechanisms of action can synergistically contribute to the same physiological and, in turn, pharmacological activity. We note, however, that <b>35d</b>’s low solubility may affect its metabolic and pharmacokinetic profile (DM/PK), preventing it from being studied further in in vivo cancer models. Structural tuning is therefore required (and currently ongoing) to discover more drug-like dihydroquinolone pyrazoline derivatives.</div><div class="NLM_p last">In conclusion, we have further shown that synthetic lethality may be a suitable framework for discovering innovative anticancer therapies. We are confident that this novel concept will open up several new avenues based on other lethal gene pairs to meet the medical needs in oncology.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15785" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15785" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry. General Chemical Methods</h3><div class="NLM_p last">Solvents and reagents were obtained from commercial suppliers and used without further purification. If required, solvents were distilled prior to use. Automated column chromatography purifications were conducted using a Teledyne ISCO apparatus (CombiFlash Rf) with prepacked silica gel columns of different sizes (from 4 to 120 g). Mixtures of increasing polarity of cyclohexane and ethyl acetate or dichloromethane and methanol/ethanol were used as eluents. Preparative TLCs were performed using Macherey-Nagel precoated 0.05 mm TLC plates (SIL G-50 UV254). Microwave heating was performed using Explorer-48 positions instrument (CEM). NMR experiments were run on a Bruker Avance III 400 MHz spectrometer (400.13 MHz for <sup>1</sup>H and 100.62 MHz for <sup>13</sup>C), equipped with a BBI probe and Z-gradients, or on a Bruker FT NMR Avance III 600-MHz spectrometer (600.130 MHz for <sup>1</sup>H and 150.903 MHz for <sup>13</sup>C) equipped with a 5 mm CryoProbe QCI quadruple resonance, a shielded Z-gradient coil, and the automatic sample changer SampleJet NMR system. Spectra were acquired at 300 K, using deuterated dimethylsulfoxide (DMSO-<i>d</i><sub>6</sub>) or deuterated chloroform (CDCl<sub>3</sub>) as solvents. Chemical shifts for <sup>1</sup>H and <sup>13</sup>C spectra were recorded in parts per million using the residual nondeuterated solvent as the internal standard (for CDCl<sub>3</sub>, <sup>1</sup>H 7.26 ppm, <sup>13</sup>C 77.16 ppm; for DMSO-<i>d</i><sub>6</sub>, <sup>1</sup>H 2.50 ppm, <sup>13</sup>C 39.52 ppm). UPLC–MS analyses were run on a Waters ACQUITY UPLC/MS system consisting of an SQD (single quadrupole detector) mass spectrometer equipped with an electrospray ionization interface and a photodiode array detector. The PDA range was 210–400 nm. The analyses were performed on either an ACQUITY UPLC HSS T3 C<sub>18</sub> column (50 × 2.1 mm i.d., particle size 1.8 μm) with a VanGuard HSS T3 C<sub>18</sub> precolumn (5 mm × 2.1 mm i.d., particle size 1.8 μm) (log <i>D</i> < 1) or an ACQUITY UPLC BEH C18 column (50 mm × 2.1 mm i.d., particle size 1.7 μm) with a VanGuard BEH C<sub>18</sub> precolumn (5 mm × 2.1 mm i.d., particle size 1.7 μm) (log <i>D</i> > 1). The mobile phase was 10 mM NH<sub>4</sub>OAc in H<sub>2</sub>O at pH 5 adjusted with AcOH (A) and 10 mM NH<sub>4</sub>OAc in MeCN–H<sub>2</sub>O (95:5) at pH 5 (B). Electrospray ionization in positive and negative mode was applied in the mass scan range 100–500 Da. Methods and gradients used were the following: <i>Polar method.</i> Column: Waters ACQUITY UPLC HSS T3 C<sub>18</sub>, 1.8 μm, 50 mm × 2.1 mm i.d. Precolumn: VanGuard HSS T3 C<sub>18</sub>, 1.8 μm, 5 mm × 2.1 mm i.d. Linear gradient: 0–0.2 min, 0% B; 0.2–2.7 min, 0–50% B; 2.7–2.8 min, 50–100% B; 2.8–3.0 min, 100% B. Flow rate: 0.5 mL/min. <i>Generic method.</i> Column: Waters ACQUITY UPLC BEH C<sub>18</sub>, 1.7 μm, 50 mm × 2.1 mm i.d. Linear gradient: 0–0.2 min, 5% B; 0.2–2.7 min, 5–95% B; 2.7–2.8 min, 95–100% B; 2.8–3.0 min, 100% B. Flow rate: 0.5 mL/min. <i>Apolar method.</i> Column: Waters ACQUITY UPLC BEH C<sub>18</sub>, 1.7 μm, 50 mm × 2.1 mm i.d. Precolumn: VanGuard BEH C<sub>18</sub>, 1.7 μm, 5 mm × 2.1 mm i.d. Gradient: 0–0.2 min, 50% B; 0.2–2.7 min, 50–100% B; 2.7–3.0 min, 100% B. Flow rate: 0.5 mL/min. Compounds were named using the naming algorithm developed by CambridgeSoft Corporation and used in ChemBioDraw Ultra 15.0. All final compounds displayed ≥95% purity as determined by UPLC/MS analysis. All final synthesized compounds were checked for PAINS compliance.</div></div><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure A for the Synthesis of Quinolin-2(1<i>H</i>)-one Acrolyl Intermediates</h3><div class="NLM_p last">This procedure has been applied to the preparation of <b>86b</b>–<b>114b</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In a round-bottomed flask, the appropriate quinolin-2(1<i>H</i>)-one (<b>60a</b>–<b>61a</b>, 1.00 equiv) and potassium hydroxide (25.00 equiv) were stirred in EtOH/H<sub>2</sub>O (4:3 v/v, 0.05 M) at 0 °C for 45 min prior to the addition of an appropriately substituted aryl aldehyde (<b>62</b>–<b>85</b>, 1.00 equiv). The reaction mixture was stirred overnight as it gradually reached room temperature. The reaction was quenched by slow addition of acetic acid (25.00 equiv). The crude was extracted with DCM/H<sub>2</sub>O (3 × 50 mL), the organic layer was then dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed under reduced pressure. The desired compound was obtained after purification over silica gel unless otherwise noted.</div></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure B for Synthesis of Pyrazol-3-yl-quinolin-2(1<i>H</i>)-one Intermediates (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h3><div class="NLM_p last">This procedure has been applied to the preparation of <b>86c</b>–<b>114c</b>. In a microwaveable vessel, the appropriate quinolin-2(1<i>H</i>)-one acrolyl intermediate (<b>86b</b>–<b>114b</b>, 1.00 equiv) was dissolved in EtOH absolute (0.2 M), and hydrazine monohydrate (2.00 equiv) was added. The mixture was microwaved with stirring for 45 min at 110 °C (200 W). The EtOH was removed under reduced pressure. Crude was purified over silica gel, unless otherwise noted, to afford desired compounds.</div></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedures C<sub>1</sub> to Obtain the Final Desired Dihydroquinolone Pyrazoline Derivatives (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h3><div class="NLM_p last">This procedure has been applied to synthesize <b>4d</b>, <b>11d</b>–<b>32d</b>, <b>52d</b>–<b>57d</b>. In a microwaveable vessel, the appropriate pyrazol-3-ylquinolin-2(1<i>H</i>)-one intermediate (<b>86c</b>−<b>90c</b>, <b>92c</b>−<b>93c</b>, <b>95c</b>−<b>103c</b>, <b>108c</b>, 1.00 equiv) was dissolved in anhydrous THF (0.5 M). The succinic anhydride <b>115</b> (2.00 equiv) was added. The solution was microwaved (200 W) with stirring for 45 min at the appropriate temperature. The THF was removed under reduced pressure, and the organic layers were dissolved in DCM, washed with HCl<sub>aq</sub> pH 2 (3 × 30 mL), and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude was purified over silica gel.</div></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Procedures C<sub>2</sub> to Obtain the Final Desired Dihydroquinolone Pyrazoline Derivatives (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h3><div class="NLM_p last">This procedure has been applied to synthesize <b>5d</b>, <b>36d</b>–<b>51d</b>. In a microwaveable vessel, the appropriate pyrazol-3-ylquinolin-2(1<i>H</i>)-one intermediate (<b>86c</b>−<b>90c</b>, <b>92c</b>−<b>93c</b>, <b>95c</b>−<b>103c</b>, <b>108c</b>, 1.00 equiv) was dissolved in anhydrous THF (0.5 M). The acetic anhydride <b>116</b> (2.00 equiv) was added. The solution was microwaved (200 W) with stirring for 45 min at the appropriate temperature. The THF was removed under reduced pressure, and the organic layers were dissolved in DCM, washed with HCl<sub>aq</sub> pH 2 (3 × 30 mL), and dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was removed under reduced pressure, and the crude was purified over silica gel.</div></div><div id="sec4_1_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> General Procedures C<sub>3</sub> to Obtain the Final Dihydroquinolone Pyrazoline Derivatives (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>)</h3><div class="NLM_p last">This procedure has been applied to synthesize <b>6d</b>, <b>33d</b>–<b>35d</b>. In a round-bottomed flask, propionic acid <b>117</b> (1.80 equiv), HOBt (1.80 equiv), and EDCI (1.80 equiv) were stirred in DCM (0.50 M) at room temperature for 1 h. Then a solution of the pyrazol-3-ylquinolin-2(1<i>H</i>)-one intermediate <b>86c</b>–<b>89c</b> (1.00 equiv) and Et<sub>3</sub>N (2.50 equiv) in DCM (0.50 M) was added. The reaction was stirred at room temperature for 3 h. The organic layer was washed with NaHCO<sub>3(aq)</sub> 1 M (1 × 50 mL), citric acid 10% (1 × 50 mL), and H<sub>2</sub>O (1 × 50 mL), dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed under reduced pressure. The desired compound was obtained after purification over silica gel unless otherwise noted.</div></div><div id="sec4_1_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>4d</b>)</h3><div class="NLM_p last">6-Chloro-3-(5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one <b>86c</b> (270 mg, 0.65 mmol) and succinic anhydride <b>115</b> (130 mg, 1.30 mmol) were microwaved (120 °C, 200 W) according to general procedure C<sub>1</sub>. The crude was purified by flash column chromatography (SiO<sub>2</sub> gold 24 g; 0–50% EtOH/DCM) to afford the desired <b>4d</b> (208 mg, 62% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.02 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.37 (m, 5H), 7.28 (dt, <i>J</i> = 6.8, 2.0 Hz, 1H), 7.04 (t, <i>J</i> = 8.9 Hz, 2H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.88–6.76 (m, 2H), 5.32 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 3.73 (dd, <i>J</i> = 18.5, 12.0 Hz, 1H), 2.79 (dd, <i>J</i> = 18.4, 4.5 Hz, 1H), 2.48–2.40 (m, 2H), 2.32–2.26 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.95, 169.09, 160.56, 152.85, 150.38, 137.78, 135.00, 131.68, 129.88, 128.97, 128.88, 128.81, 127.99, 127.90, 126.57, 126.51, 125.07, 121.14, 118.09, 115.67, 115.46, 95.62, 58.69, 45.66, 29.04, 28.69. In agreement with that previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>t</i><sub>R</sub> = 2.04 min (generic method). ESI-MS for C<sub>28</sub>H<sub>21</sub>ClFN<sub>3</sub>O<sub>4</sub>: calculated 517.1, found <i>m</i>/<i>z</i> 518.4, 520.4 [M + H] <sup>+</sup>; 516.4, 518.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 98%.</div></div><div id="sec4_1_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 3-(1-Acetyl-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>5d</b>)</h3><div class="NLM_p last">6-Chloro-3-(5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one <b>86c</b> (200 mg, 0.48 mmol) and acetic anhydride <b>116</b> (91 μL, 0.96 mmol) were microwaved (120 °C, 200 W) according to general procedure C<sub>2</sub>. The crude was purified by direct phase flash column chromatography (SiO<sub>2</sub> gold 24 g, 0–20% MeOH/DCM) to afford the desired <b>5d</b> (207 mg, yield 47%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.61–7.50 (m, 3H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 7.41 (dt, <i>J</i> = 6.6, 1.9 Hz, 1H), 7.29 (dt, <i>J</i> = 6.4, 1.9 Hz, 1H), 7.06 (t, <i>J</i> = 8.9 Hz, 2H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.84 (dd, <i>J</i> = 8.6, 5.6 Hz, 2H), 5.32 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 3.73 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.84 (dd, <i>J</i> = 18.5, 4.5 Hz, 1H), 1.88 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.53, 162.80, 160.57, 160.39, 152.67, 150.35, 138.87, 138.84, 137.79, 135.12, 131.64, 129.84, 129.04, 128.96, 128.80, 128.76, 128.00, 127.93, 126.53, 126.49, 125.02, 121.11, 118.09, 115.67, 115.45, 58.49, 45.71, 21.85. <i>t</i><sub>R</sub> = 2.38 min (generic method). ESI-MS for C<sub>26</sub>H<sub>19</sub>ClFN<sub>3</sub>O<sub>2</sub>: calculated 459.1, found <i>m</i>/<i>z</i> 460.1, 462.1 [M + H]<sup>+</sup>; 458.1, 460.1 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 6-Chloro-3-(5-(4-fluorophenyl)-1-propionyl-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>6d</b>)</h3><div class="NLM_p last">Compound <b>6d</b> was synthesized via general procedure C<sub>3</sub> using <b>86c</b> (250 mg, 0.60 mmol) and propionic acid <b>117</b> (80.8 μL, 1.08 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–1% MeOH/DCM) to afford <b>6d</b> (60 mg, yield 21%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.64 (dd, <i>J</i> = 2.0, 8.0 Hz, 1H), 7.58–7.50 (m, 3H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.41–7.39 (m, 1H), 7.28 (d, <i>J</i> = 5.6 Hz, 1H), 7.06 (t, <i>J</i> = 8.8 Hz, 2H), 6.93 (d, <i>J</i> = 2.0 Hz, 1H), 6.84 (dd, <i>J</i> = 4.0, 8.0, Hz, 2H), 5.32 (dd, <i>J</i> = 8.0, 12.0 Hz, 1H), 3.73 (dd, <i>J</i> = 12.0, 16.0 Hz, 1H), 2.81 (dd, <i>J</i> = 4.0, 16.0 Hz, 1H), 2.30–2.16 (m, 2H), 0.83 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 206.89, 170.93, 162.79, 160.57, 160.38, 152.63, 150.30, 139.00, 138.98, 137.75, 135.12, 131.63, 129.79, 128.97, 128.95, 128.86, 128.80, 127.96, 127.88, 126.53, 126.47, 125.10, 121.11, 118.07, 115.67, 115.46, 58.54, 45.54, 29.44, 9.27. ESI-MS for C<sub>27</sub>H<sub>21</sub>ClFN<sub>3</sub>O<sub>2</sub>: calculated 473.1, found <i>m</i>/<i>z</i> 474.2, 476.2 [M + H]<sup>+</sup>; 472.3, 474.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 3-(1-(3-Aminopropanoyl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>7d</b>) (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>)</h3><div class="NLM_p last">In round-bottom flask <b>119</b> (200 mg, 0.34 mmol) was treated with 4 M HCl in dioxane (4 mL) and stirred at rt for 15 min. The solvent was removed under reduced pressure. In a round-bottom flask the solid residue (110 mg, 0.21 mmol) was treated with NaOH 0.5 M (420 μL) in EtOAc (5 mL) stirring at rt for 30 min. The mixture was then diluted with further EtOAc and washed twice with H<sub>2</sub>O. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. Purification was performed by direct phase flash chromatography (0–10% MeOH/DCM, 0–0.1% NH<sub>4</sub>OH). Yield 50 mg, 50%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.63 (dd, <i>J</i> = 12.0, 4.0 Hz, 1H), 7.57–7.50 (m, 3H), 7.54 (d, <i>J</i> = 4.8 Hz, 1H), 7.42–7.40 (m, 1H), 7.26 (d, <i>J</i> = 7.2 Hz, 1H), 7.03 (t, <i>J</i> = 8.8 Hz, 2H), 6.92 (d, <i>J</i> = 2.0 Hz, 1H), 6.81 (dd, <i>J</i> = 2.0, 8.0 Hz, 2H), 5.32 (dd, <i>J</i> = 2.0, 12.0 Hz, 1H), 3.72 (dd, <i>J</i> = 12.0, 20.0 Hz, 1H), 2.76 (dd, <i>J</i> = 4.0, 16.0 Hz, 1H), 2.57 (t, <i>J</i> = 6.6 Hz, 2H), 2.42–2.28 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.42, 162.77, 160.56, 160.36, 152.76, 150.29, 138.96, 138.93, 137.78, 135.02, 131.63, 129.86, 128.99, 128.92, 128.89, 128.80, 127.95, 127.86, 126.52, 126.45, 125.09, 121.11, 118.10, 115.66, 115.45, 110.00, 58.53, 45.61, 38.06, 38.04. ESI-MS for C<sub>27</sub>H<sub>22</sub>ClFN<sub>4</sub>O<sub>2</sub>: calculated 488.1, found <i>m</i>/<i>z</i> 489.4, 491.4 [M + H]<sup>+</sup>, 487.3, 489.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanamide (<b>8d</b>) (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>)</h3><div class="NLM_p last">In a round bottomed flask 4-(3-(6-chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic acid <b>4d</b> (258 mg, 0.50 mmol) was dissolved with 3.2 mL of anhydrous DCM, then HATU (285 mg, 0.75 mmol), EDC (144 mg, 0.75 mmol), and 1.8 mL of anhydrous DMF were added. The mixture was stirred at rt for 10 min. Ammonium chloride (144 mg 2.50 mmol) and soon after DIPEA (348 μL, 2.00 mmol) were added. The reaction mixture was thus stirred at rt for 26 h. Purification was performed by direct phase chromatography (SiO<sub>2</sub> gold 24 g, 2.5–50% EtOH/DCM). Yield 120 mg, 46%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.38 (m, 5H), 7.28 (d, <i>J</i> = 7.4 Hz, 1H), 7.19 (s, 1H), 7.04 (dd, <i>J</i> = 9.9, 7.7 Hz, 2H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 6.84–6.78 (m, 2H), 6.68 (s, 1H), 5.31 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 3.71 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.76 (dd, <i>J</i> = 18.4, 4.5 Hz, 1H), 2.54–2.45 (m, 2H), 2.15 (t, <i>J</i> = 7.4 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.51, 169.56, 160.65, 160.38, 152.73, 150.28, 138.90, 137.94, 135.00, 131.61, 129.94, 129.94, 129.02, 128.99, 128.87, 128.79, 127.99, 127.90, 126.48, 125.10, 121.14, 118.19, 115.66, 115.45, 58.66, 45.64, 29.82, 29.38. <i>t</i><sub>R</sub> = 2.09 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>ClFN<sub>4</sub>O<sub>3</sub>: calculated 516.1, found <i>m</i>/<i>z</i> 517.4, 519.3 [M + H]<sup>+</sup>; 515.4, 517.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Methyl 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoate (<b>9d</b>) (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>)</h3><div class="NLM_p last">In a round-bottomed flask, commercially available 4-methoxy-4-oxobutanoic acid <b>120</b> (153 mg, 1.16 mmol), 6-chloro-3-(5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one <b>86c</b> (486 mg, 1.16 mmol), HOBT (173 mg, 1.28 mmol) were dissolved in DCM. Then TEA (355.8 μL, 2.55 mmol) was added, followed by EDCI (245 mg, 1.28 mmol) suspended in DCM. The mixture was stirred overnight at rt. The solvent was removed under vacuum, the residue was dissolved in ethyl acetate and washed with H<sub>2</sub>O, NaHCO<sub>3</sub> sat. solution and 5% citric acid. The organic phase was dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated to dryness. The title compound was obtained after purification by direct phase flash column chromatography (0–30% EtOAc/DCM). Yield 256 mg, 42%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 7.60 (dd, <i>J</i> = 2.2, 9.0 Hz, 1H), 7.53–7.48 (m, 3 H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.37 (dd, <i>J</i> = 2.2, 4.6 Hz, 1H), 7.23 (d, <i>J</i> = 6.8 Hz, 1H), 7.01 (t, <i>J</i> = 8.8 Hz, 2H), 6.89 (d, <i>J</i> = 2.0, 1H), 6.77 (dd, <i>J</i> = 5.4, 8.6 Hz, 2H), 5.28 (dd, <i>J</i> = 12.0, 4.0 Hz, 1H), 3.50 (s, 3H), 3.67 (dd, <i>J</i> = 12.0, 18.0 Hz, 1H), 2.75 (dd, <i>J</i> = 4.0, 18.0 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) 172.9, 168.8, 160.0, 153.1, 150.1, 139.0, 135.0, 131.9, 129.8, 128.9, 127.9, 126.5, 125.0, 120.9, 118.0, 115.6, 115.4, 58.7, 21.7, 45.0, 28.9, 28.4. UPLC–MS purity (UV at 215 nm) 98%.</div></div><div id="sec4_1_1_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-(3-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-3-oxopropyl)methanesulfonamide (<b>10d</b>) (<a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>)</h3><div class="NLM_p last">In a round-bottom flask, 6-chloro-3-(5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one <b>86c</b> (500 mg, 1.20 mmol), 3-(methylsulfonamido)propanoic acid <b>124</b> (200 mg, 1.20 mmol), HOBT (178 mg, 1.32 mmol) were stirred in DCM prior to the addition of triethylamine (367.7 mL, 2.64 mmol) and EDCI (253 mg, 1.32 mmol) at 0 °C. The reaction was stirred overnight while the mixture gradually reached rt. The solvent was removed under vacuum, and the residue was dissolved in DCM. The organic layer was washed with H<sub>2</sub>O, NaHCO<sub>3</sub> sat. solution, and 5% citric acid, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and evaporated to dryness. The title compound was obtained after purification over direct phase flash column chromatography (0–60% EtOAc/DCM). Yield 100 mg, 15%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 7.60 (dd, <i>J</i> = 2.0, 8.0 Hz, 1H), 7.55–7.49 (m, 3H), 7.41 (d, <i>J</i> = 8.0 Hz, 1H), 7.38 (d, <i>J</i> = 6.0 Hz, 1H), 7.24 (d, J = 6.0 Hz, 1H), 7.00 (t, <i>J</i> = 8.0 Hz, 2H), 6.89 (d, <i>J</i> = 2.0 Hz, 2H), 6.77 (dd, <i>J</i> = 4.0, 8.0 Hz, 2H), 5.29 (dd, <i>J</i> = 4.0, 12.0 Hz, 1H), 3.69 (dd, <i>J</i> = 12.0, 20.0 Hz, 1H), 3.03–2.98 (m, 2H), 2.80 (s, 3H), 2.72 (dd, <i>J</i> = 4.0, 20.0 Hz, 1H), 2.51–2.47 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.01, 162.81, 160.49, 160.40, 153.25, 150.39, 138.71, 137.78, 134.86, 131.68, 129.94, 129.10, 128.94, 128.91, 128.83, 128.02, 127.94, 126.57, 125.01, 121.07, 118.09, 115.69, 115.48, 58.63, 45.67, 38.61, 34.73. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-chlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>11d</b>)</h3><div class="NLM_p last">Compound <b>11d</b> was synthesized via general procedure C<sub>1</sub> using <b>87c</b> (253 mg, 0.58 mmol) with succinic anhydride <b>115</b> (116 mg, 1.16 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–60% EtOH/DCM) to afford <b>11d</b> (231 mg, yield 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.03 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59–7.48 (m, 3H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.41 (dt, <i>J</i> = 6.3, 2.0 Hz, 1H), 7.31–7.23 (m, 3H), 6.93 (d, <i>J</i> = 2.4 Hz, 1H), 6.83–6.76 (m, 2H), 5.32 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.74 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.78 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.48–2.41 (m, 2H), 2.29 (t, <i>J</i> = 7.1 Hz, 2H). In agreement with that previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>t</i><sub>R</sub> = 2.16 min (generic method). ESI-MS for C<sub>28</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>4</sub>: calculated 533.1, found <i>m</i>/<i>z</i> 534.4, 536.4, 538.3 [M + H]<sup>+</sup>; 532.4, 534.4, 536.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 4-(5-(4-Bromophenyl)-3-(6-chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>12d</b>)</h3><div class="NLM_p last">Compound <b>12d</b> was synthesized via general procedure C<sub>1</sub> using <b>88c</b> (330 mg, 0.69 mmol) with succinic anhydride <b>115</b> (138 mg, 1.38 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–50% EtOH/DCM) to afford <b>12d</b> (275 mg, yield 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.03 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.59–7.47 (m, 3H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.40 (d, <i>J</i> = 8.4 Hz, 2H), 7.27 (dd, <i>J</i> = 6.5, 1.8 Hz, 1H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 6.73 (d, <i>J</i> = 8.5 Hz, 2H), 5.30 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.74 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.77 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.48–2.36 (m, 2H), 2.28 (t, <i>J</i> = 7.1 Hz, 2H). In agreement with that previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>t</i><sub>R</sub> = 2.18 min (generic method). ESI-MS for C<sub>28</sub>H<sub>21</sub>BrClN<sub>3</sub>O<sub>4</sub>: calculated 577.0, found <i>m</i>/<i>z</i> 578.2, 580.2, 582.3 [M + H]<sup>+</sup>; 576.1, 578.1, 580.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-methoxyphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>13d</b>)</h3><div class="NLM_p last">Compound <b>13d</b> was synthesized via general procedure C<sub>1</sub> using <b>89c</b> (320 mg, 0.74 mmol) with succinic anhydride <b>115</b> (148 mg, 1.48 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–15% EtOH/DCM) to afford <b>13d</b> (339 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.51 (m, 3H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.36–7.31 (m, 1H), 7.27–7.21 (m, 1H), 7.02 (dd, <i>J</i> = 8.9, 5.9 Hz, 3H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 6.80 (d, <i>J</i> = 8.7 Hz, 2H), 4.52 (td, <i>J</i> = 10.4, 2.9 Hz, 1H), 3.73 (s, 3H), 3.18 (dd, <i>J</i> = 16.4, 10.9 Hz, 1H), 2.58–2.52 (m, 5H). In agreement with that previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>t</i><sub>R</sub> = 1.99 min (generic method). ESI-MS for C<sub>29</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>5</sub>: calculated 529.1, found <i>m</i>/<i>z</i> 530.5, 532.4 [M + H]<sup>+</sup>; 528.4, 530.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 4-(5-(4-(<i>tert</i>-Butyl)phenyl)-3-(6-chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>14d</b>)</h3><div class="NLM_p last">Compound <b>14d</b> was synthesized via general procedure C<sub>1</sub> using <b>90c</b> (124 mg, 0.27 mmol) with succinic anhydride <b>115</b> (54 mg, 0.54 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>14d</b> (121 mg, yield 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.27 (d, <i>J</i> = 78.0 Hz, 1H), 12.08 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.59–7.47 (m, 3H), 7.47–7.43 (m, 1H), 7.40 (dd, <i>J</i> = 4.6, 2.1 Hz, 1H), 7.28 (d, <i>J</i> = 7.5 Hz, 1H), 7.23 (d, <i>J</i> = 8.3 Hz, 2H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 6.73 (d, <i>J</i> = 8.3 Hz, 2H), 5.26 (dd, <i>J</i> = 11.9, 4.5 Hz, 1H), 3.69 (dt, <i>J</i> = 24.9, 12.5 Hz, 1H), 2.85 (ddd, <i>J</i> = 23.0, 18.2, 4.3 Hz, 1H), 2.47 (dd, <i>J</i> = 6.7, 4.0 Hz, 2H), 2.36–2.22 (m, 2H), 1.27 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.52, 168.56, 160.09, 152.41, 149.87, 149.19, 139.24, 137.30, 134.59, 131.15, 129.30, 128.57, 128.47, 128.33, 126.06, 126.03, 125.13, 125.09, 124.67, 120.69, 117.60, 58.62, 45.26, 34.12, 31.11, 28.57, 28.21, 26.32. <i>t</i><sub>R</sub> = 2.39 min (generic method). ESI-MS for C<sub>32</sub>H<sub>30</sub>ClN<sub>3</sub>O<sub>4</sub>: calculated 555.2, found <i>m</i>/<i>z</i> 556.5, 558.4 [M + H]<sup>+</sup>; 554.4, 556.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 97%.</div></div><div id="sec4_1_1_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(3-fluoro-4-methoxyphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>15d</b>)</h3><div class="NLM_p last">Compound <b>15d</b> was synthesized via general procedure C<sub>1</sub> using <b>91c</b> (184 mg, 0.41 mmol) with succinic anhydride <b>115</b> (82 mg, 0.82 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>15d</b> (115 mg, yield 51%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.06 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59–7.43 (m, 4H), 7.43–7.38 (m, 1H), 7.29 (dd, <i>J</i> = 7.1, 1.9 Hz, 1H), 7.01 (t, <i>J</i> = 8.8 Hz, 1H), 6.94 (d, <i>J</i> = 2.4 Hz, 1H), 6.69–6.61 (m, 2H), 5.27 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 3.82 (s, 3H), 3.70 (dd, <i>J</i> = 18.5, 12.0 Hz, 1H), 2.82 (dd, <i>J</i> = 18.5, 4.5 Hz, 1H), 2.46 (dd, <i>J</i> = 14.3, 7.0 Hz, 2H), 2.29 (t, <i>J</i> = 7.2 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.96, 169.09, 160.59, 152.89, 150.37, 146.62, 137.79, 135.43, 135.05, 131.68, 129.76, 129.01, 128.94, 128.84, 128.74, 126.57, 125.06, 123.56, 122.25, 121.13, 118.10, 114.22, 113.68, 113.50, 58.52, 56.48, 45.52, 29.02, 28.67. <i>t</i><sub>R</sub> = 2.00 min (generic method). ESI-MS for C<sub>29</sub>H<sub>23</sub>ClFN<sub>3</sub>O<sub>5</sub>: calculated 547.1, found <i>m</i>/<i>z</i> 548.3, 550.3 [M + H]<sup>+</sup>; 546.3, 548.2 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-(trifluoromethyl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>16d</b>)</h3><div class="NLM_p last">Compound <b>16d</b> was synthesized via general procedure C<sub>1</sub> using <b>92c</b> (309 mg, 0.66 mmol) with succinic anhydride <b>115</b> (132 mg, 1.32 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g, 0–10% EtOH/DCM) to afford <b>16d</b> (266 mg, yield 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.05 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.39 (m, 7H), 7.31–7.19 (m, 1H), 7.00 (d, <i>J</i> = 8.1 Hz, 2H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 5.42 (dd, <i>J</i> = 12.1, 4.7 Hz, 1H), 3.78 (dd, <i>J</i> = 18.5, 12.2 Hz, 1H), 2.80 (dd, <i>J</i> = 18.5, 4.7 Hz, 1H), 2.59–2.38 (m, 2H), 2.36–2.26 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.45, 168.78, 160.05, 152.42, 150.01, 146.65, 137.33, 134.55, 131.24, 129.42, 128.54, 128.44, 128.38, 128.35, 127.85, 127.53, 127.22, 126.20, 126.12, 126.04, 125.54, 125.39, 125.35, 124.45, 122.83, 120.63, 117.63, 58.51, 45.05, 28.49, 28.18, 26.32. In agreement with that previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>t</i><sub>R</sub> = 2.22 min (generic method). ESI-MS for C<sub>29</sub>H<sub>21</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>4</sub>: calculated 567.1, found <i>m</i>/<i>z</i> 568.5, 570.4, [M + H]<sup>+</sup>; 566.4, 568.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(furan-2-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>17d</b>)</h3><div class="NLM_p last">Compound <b>17d</b> was synthesized via general procedure C<sub>1</sub> using <b>93c</b> (201 mg, 0.51 mmol) with succinic anhydride <b>115</b> (102 mg, 1.02 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>17d</b> (226 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.06 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.54–7.43 (m, 5H), 7.32 (t, <i>J</i> = 6.4 Hz, 2H), 6.96 (d, <i>J</i> = 2.3 Hz, 1H), 6.33 (dd, <i>J</i> = 3.1, 1.9 Hz, 1H), 5.99 (d, <i>J</i> = 3.2 Hz, 1H), 5.39 (dd, <i>J</i> = 11.8, 4.5 Hz, 1H), 3.58 (dd, <i>J</i> = 18.2, 11.9 Hz, 1H), 3.17 (dd, <i>J</i> = 18.2, 4.6 Hz, 1H), 2.41–2.18 (m, 4H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.58, 173.49, 168.82, 160.10, 152.84, 152.49, 150.04, 142.05, 137.37, 134.77, 131.19, 128.81, 128.78, 128.29, 128.26, 128.24, 126.12, 126.07, 124.33, 120.63, 117.60, 110.33, 106.27, 56.01, 52.62, 41.72, 28.84, 28.52, 28.15. In agreement with that previously reported by Acker et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a><i>t</i><sub>R</sub> = 1.90 min (generic method). ESI-MS for C<sub>26</sub>H<sub>20</sub>ClN<sub>3</sub>O<sub>5</sub>: calculated 489.1, found <i>m</i>/<i>z</i> 490.4, 492.4 [M + H]<sup>+</sup>; 488.4, 490.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4′-fluoro-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>18d</b>)</h3><div class="NLM_p last">Compound <b>18d</b> was synthesized via general procedure C<sub>1</sub> using <b>94c</b> (171 mg, 0.35 mmol) with succinic anhydride <b>115</b> (70 mg, 0.70 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>18d</b> (167 mg, yield 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.08 (s, 1H), 7.71–7.61 (m, 3H), 7.55 (ddt, <i>J</i> = 10.6, 9.5, 4.0 Hz, 3H), 7.50–7.40 (m, 4H), 7.33–7.25 (m, 3H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.87 (d, <i>J</i> = 8.3 Hz, 2H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.77 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.84 (dd, <i>J</i> = 18.4, 4.7 Hz, 1H), 2.57–2.44 (m, 2H), 2.30 (t, <i>J</i> = 6.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.63, 173.50, 168.63, 163.05, 160.62, 160.11, 152.46, 149.94, 141.37, 137.96, 137.31, 136.34, 136.31, 134.60, 131.19, 129.41, 128.64, 128.56, 128.47, 128.33, 126.73, 126.08, 124.64, 120.69, 117.62, 115.78, 115.57, 58.66, 45.22, 28.96, 28.60, 28.23. <i>t</i><sub>R</sub> = 2.36 min (generic method). ESI-MS for C<sub>34</sub>H<sub>25</sub>ClFN<sub>3</sub>O<sub>4</sub>: calculated 593.1, found <i>m</i>/<i>z</i> 594.1, 596.1 [M + H]<sup>+</sup>, 592.2, 594.2 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4′-chloro-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>19d</b>)</h3><div class="NLM_p last">Compound <b>19d</b> was synthesized via general procedure C<sub>1</sub> using <b>95c</b> (227 mg, 0.44 mmol) with succinic anhydride <b>115</b> (88 mg, 0.88 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>19d</b> (205 mg, yield 77%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.04 (s, 1H), 7.67 (d, <i>J</i> = 8.6 Hz, 2H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59–7.48 (m, 7H), 7.47–7.39 (m, 2H), 7.29 (dd, <i>J</i> = 6.5, 2.0 Hz, 1H), 6.94 (d, <i>J</i> = 2.4 Hz, 1H), 6.88 (d, <i>J</i> = 8.3 Hz, 2H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.77 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.84 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.50 (m, 2H), 2.29 (t, <i>J</i> = 6.8 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.58, 173.49, 168.64, 160.10, 152.44, 149.94, 141.77, 138.63, 137.62, 137.31, 134.60, 132.26, 131.18, 129.40, 128.84, 128.52, 128.45, 128.37, 128.32, 126.72, 126.14, 126.10, 126.05, 124.62, 120.68, 117.61, 58.66, 45.21, 28.60, 28.23. <i>t</i><sub>R</sub> = 2.51 min (generic method). ESI-MS for C<sub>34</sub>H<sub>25</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>4</sub>: calculated 609.1, found <i>m</i>/<i>z</i> 610.0, 612.0, 614.2 [M + H]<sup>+</sup>; 608.1, 610.1, 612.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 4-[3-[4-(4-4-(5-(4′-Bromo-[1,1′-biphenyl]-4-yl)-3-(6-chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>20d</b>)</h3><div class="NLM_p last">Compound <b>20d</b> was synthesized via general procedure C<sub>1</sub> using <b>96c</b> (214 mg, 0.38 mmol) with succinic anhydride <b>115</b> (76 mg, 0.76 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>20d</b> (197 mg, yield 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.08 (s, 1H), 7.68–7.38 (m, 12H), 7.30–7.26 (m, 1H), 6.94 (d, <i>J</i> = 2.4 Hz, 1H), 6.89 (d, <i>J</i> = 8.4 Hz, 2H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.78 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.84 (dd, <i>J</i> = 18.5, 4.7 Hz, 1H), 2.58–2.43 (m, <i>J</i> = 3.7 Hz, 2H), 2.30 (t, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.50, 168.66, 160.11, 152.44, 149.95, 141.81, 139.00, 137.67, 137.31, 134.60, 131.76, 131.18, 129.40, 128.71, 128.52, 128.45, 128.33, 126.68, 126.16, 126.11, 126.05, 124.62, 120.84, 120.68, 117.61, 58.67, 45.20, 28.61, 28.24. <i>t</i><sub>R</sub> = 2.52 min (generic method). ESI-MS for C<sub>34</sub>H<sub>25</sub>BrClN<sub>3</sub>O<sub>4</sub>: calculated 653.1, found <i>m</i>/<i>z</i> 653.9, 655.9, 657.9 [M + H]<sup>+</sup>; 652.0, 654.0, 655.8 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4′-methoxy-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>21d</b>)</h3><div class="NLM_p last">Compound <b>21d</b> was synthesized via general procedure C<sub>1</sub> using <b>97c</b> (545 mg, 1.10 mmol) with succinic anhydride <b>115</b> (220 mg, 2.20 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g, 0–10% EtOH/DCM) to afford <b>21d</b> (359 mg, yield 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.03 (s, 1H), 7.67–7.50 (m, 6H), 7.44 (dt, <i>J</i> = 7.7, 5.9 Hz, 4H), 7.29 (dd, <i>J</i> = 6.7, 2.1 Hz, 1H), 7.02 (d, <i>J</i> = 8.8 Hz, 2H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.84 (d, <i>J</i> = 8.3 Hz, 2H), 5.33 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.79 (s, 4H), 2.84 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 2.30 (t, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.51, 168.62, 160.12, 158.85, 152.44, 149.92, 140.63, 138.67, 137.31, 134.60, 132.23, 131.17, 129.40, 128.54, 128.44, 128.32, 127.68, 126.23, 126.10, 126.00, 124.66, 120.69, 117.61, 114.34, 58.70, 55.14, 45.24, 28.62, 28.25. <i>t</i><sub>R</sub> = 2.30 min (generic method). ESI-MS for C<sub>35</sub>H<sub>28</sub>ClN<sub>3</sub>O<sub>5</sub>: calculated 605.2, found <i>m</i>/<i>z</i> 606.0, 608.0 [M + H]<sup>+</sup>; 604.0, 606.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(1-methyl-1<i>H</i>-indol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>22d</b>)</h3><div class="NLM_p last">Compound <b>22d</b> was synthesized via general procedure C<sub>1</sub> using <b>98c</b> (153 mg, 0.34 mmol) with succinic anhydride <b>115</b> (68 mg, 0.68 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>22d</b> (150 mg, yield 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.18 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.60–7.55 (m, 2H), 7.55–7.49 (m, 1H), 7.44 (dd, <i>J</i> = 7.7, 4.3 Hz, 2H), 7.31–7.20 (m, 3H), 6.98 (d, <i>J</i> = 1.7 Hz, 1H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 6.61 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 6.34 (dd, <i>J</i> = 3.0, 0.8 Hz, 1H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.84–3.69 (m, 4H), 2.85 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 2.53–2.45 (m, 2H), 2.26 (t, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.72, 173.52, 168.43, 160.17, 152.27, 149.80, 137.29, 135.60, 134.66, 133.04, 131.11, 129.90, 129.47, 128.52, 128.43, 128.25, 127.73, 126.06, 124.84, 120.75, 118.98, 117.58, 117.22, 109.64, 100.28, 59.51, 45.76, 32.46, 29.17, 28.70, 28.27. <i>t</i><sub>R</sub> = 2.09 min (generic method). ESI-MS for C<sub>31</sub>H<sub>25</sub>ClN<sub>4</sub>O<sub>4</sub>: calculated 552.2, found <i>m</i>/<i>z</i> 553.1, 555.2 [M + H]<sup>+</sup>; 551.2, 553.2 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(1-ethyl-1<i>H</i>-indol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>23d</b>)</h3><div class="NLM_p last">Compound <b>23d</b> was synthesized via general procedure C<sub>1</sub> using <b>99c</b> (153 mg, 0.33 mmol) with succinic anhydride <b>115</b> (66 mg, 0.66 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>23d</b> (92 mg, yield 49%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.02 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.58 (dd, <i>J</i> = 5.5, 3.3 Hz, 2H), 7.52 (ddd, <i>J</i> = 8.9, 5.2, 3.1 Hz, 1H), 7.44 (dd, <i>J</i> = 8.7, 4.4 Hz, 2H), 7.35 (d, <i>J</i> = 3.1 Hz, 1H), 7.29 (dd, <i>J</i> = 8.5, 2.2 Hz, 2H), 6.95 (dd, <i>J</i> = 15.5, 2.0 Hz, 2H), 6.60 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 6.34 (d, <i>J</i> = 3.1 Hz, 1H), 5.34 (dd, <i>J</i> = 12.0, 4.7 Hz, 1H), 4.17 (q, <i>J</i> = 7.2 Hz, 2H), 3.77 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.86 (dd, <i>J</i> = 18.4, 4.7 Hz, 1H), 2.48–2.42 (m, 2H), 2.26 (t, <i>J</i> = 6.9 Hz, 2H), 1.34 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.53, 168.45, 160.17, 152.28, 149.80, 137.29, 134.66, 134.56, 133.01, 131.11, 129.45, 128.53, 128.42, 128.28, 127.89, 126.05, 124.84, 120.74, 118.93, 117.58, 117.32, 109.62, 100.49, 59.51, 45.75, 40.26, 28.72, 28.30, 15.53. <i>t</i><sub>R</sub> = 2.20 min (generic method). ESI-MS for C<sub>32</sub>H<sub>27</sub>ClN<sub>4</sub>O<sub>4</sub>: calculated 566.2, found <i>m</i>/<i>z</i> 567.2, 569.2 [M + H]<sup>+</sup>; 565.3, 567.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(1-ethyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>24d</b>)</h3><div class="NLM_p last">Compound <b>24d</b> was synthesized via general procedure C<sub>1</sub> using <b>100c</b> (196 mg, 0.42 mmol) with succinic anhydride <b>115</b> (84 mg, 0.84 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>24d</b> (177 mg, yield 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 12.03 (s, 1H), 7.98 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59 (dd, <i>J</i> = 5.4, 3.5 Hz, 2H), 7.52 (dd, <i>J</i> = 10.0, 6.6 Hz, 2H), 7.48–7.43 (m, 2H), 7.29 (d, <i>J</i> = 7.5 Hz, 1H), 7.14 (d, <i>J</i> = 1.5 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.83 (dd, <i>J</i> = 8.7, 1.6 Hz, 1H), 5.42 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.42 (q, <i>J</i> = 7.2 Hz, 2H), 3.80 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.85 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.54–2.48 (m, 2H), 2.28 (t, <i>J</i> = 6.9 Hz, 2H), 1.38 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.99, 169.05, 160.62, 152.86, 150.36, 138.51, 137.79, 135.07, 134.85, 132.81, 131.65, 130.00, 129.09, 128.97, 128.93, 128.78, 126.57, 126.54, 125.22, 124.57, 123.78, 121.19, 118.09, 117.79, 110.22, 59.57, 45.96, 43.53, 29.13, 28.72, 15.43. <i>t</i><sub>R</sub> = 1.95 min (generic method). ESI-MS for C<sub>31</sub>H<sub>26</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 567.2, found <i>m</i>/<i>z</i> 568.5, 570.4 [M + H]<sup>+</sup>, 566.4, 568.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(2-ethyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>25d</b>)</h3><div class="NLM_p last">Compound <b>25d</b> was synthesized via general procedure C<sub>1</sub> using <b>101c</b> (198 mg, 0.42 mmol) with succinic anhydride <b>115</b> (84 mg, 0.84 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>25d</b> (120 mg, yield 50%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.04 (s, 1H), 8.29 (s, 1H), 7.66–7.55 (m, 3H), 7.51 (td, <i>J</i> = 6.6, 5.5, 3.3 Hz, 1H), 7.44 (dd, <i>J</i> = 10.0, 6.8 Hz, 3H), 7.28 (d, <i>J</i> = 7.5 Hz, 1H), 7.07 (d, <i>J</i> = 1.6 Hz, 1H), 6.93 (d, <i>J</i> = 2.4 Hz, 1H), 6.61 (dd, <i>J</i> = 8.9, 1.7 Hz, 1H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.43 (q, <i>J</i> = 7.3 Hz, 2H), 3.77 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.82 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 2.54–2.44 (m, 2H), 2.28 (t, <i>J</i> = 6.9 Hz, 2H), 1.49 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.50, 168.55, 160.12, 152.37, 149.84, 147.32, 137.29, 134.57, 134.39, 131.15, 129.53, 128.57, 128.49, 128.42, 128.28, 126.07, 126.04, 124.75, 123.57, 123.00, 120.94, 120.72, 117.60, 117.32, 116.68, 59.31, 47.68, 45.24, 28.67, 28.26, 15.84. <i>t</i><sub>R</sub> = 1.85 min (generic method). ESI-MS for C<sub>31</sub>H<sub>26</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 567.2, found <i>m</i>/<i>z</i> 568.4, 570.4 [M + H]<sup>+</sup>, 566.5, 568.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(1-propyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>26d</b>)</h3><div class="NLM_p last">Compound <b>26d</b> was synthesized via general procedure C<sub>1</sub> using <b>102c</b> (176 mg, 0.36 mmol) with succinic anhydride <b>115</b> (72 mg, 0.72 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–25% EtOH/DCM) to afford <b>26d</b> (178 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 12.05 (s, 1H), 7.97 (d, <i>J</i> = 0.9 Hz, 1H), 7.62 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.57 (qd, <i>J</i> = 4.0, 1.0 Hz, 2H), 7.53–7.47 (m, 2H), 7.47–7.41 (m, 2H), 7.30–7.25 (m, 1H), 7.16–7.11 (m, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.84 (dd, <i>J</i> = 8.8, 1.6 Hz, 1H), 5.42 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.33 (t, <i>J</i> = 6.9 Hz, 2H), 3.78 (dd, <i>J</i> = 18.5, 12.0 Hz, 1H), 2.85 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.54–2.48 (m, 2H), 2.28 (t, <i>J</i> = 6.9 Hz, 2H), 1.83 (h, <i>J</i> = 7.2 Hz, 2H), 0.83 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.53, 168.61, 160.15, 152.41, 149.89, 138.59, 137.31, 134.60, 134.31, 132.32, 131.17, 129.50, 128.60, 128.49, 128.43, 128.32, 126.11, 126.06, 124.73, 124.13, 123.19, 120.71, 117.62, 117.28, 109.80, 59.09, 49.64, 45.49, 28.67, 28.26, 22.83, 11.17. <i>t</i><sub>R</sub> = 2.06 min (generic method). ESI-MS for C<sub>32</sub>H<sub>28</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 581.2, found <i>m</i>/<i>z</i> 582.1, 584.1 [M + H]<sup>+</sup>, 580.4, 582.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(2-propyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>27d</b>)</h3><div class="NLM_p last">Compound <b>27d</b> was synthesized via general procedure C<sub>1</sub> using <b>103c</b> (173 mg, 0.36 mmol) with succinic anhydride <b>115</b> (72 mg, 0.72 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>27d</b> (162 mg, yield 78%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 2H), 8.28 (d, <i>J</i> = 0.9 Hz, 1H), 7.63–7.54 (m, 3H), 7.50 (td, <i>J</i> = 6.6, 5.5, 3.3 Hz, 1H), 7.47–7.40 (m, 3H), 7.30–7.24 (m, 1H), 7.08 (t, <i>J</i> = 1.2 Hz, 1H), 6.93 (d, <i>J</i> = 2.4 Hz, 1H), 6.62 (dd, <i>J</i> = 8.9, 1.7 Hz, 1H), 5.36 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.35 (t, <i>J</i> = 6.9 Hz, 2H), 3.77 (dd, <i>J</i> = 18.5, 12.0 Hz, 1H), 2.83 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.54–2.50 (m, 2H), 2.29 (t, <i>J</i> = 6.9 Hz, 2H), 1.96–1.87 (m, 2H), 0.83 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.57, 168.64, 160.16, 152.44, 149.89, 147.41, 137.31, 134.59, 134.40, 131.16, 129.56, 128.60, 128.52, 128.43, 128.31, 126.13, 126.07, 124.77, 123.75, 123.60, 120.86, 120.74, 117.62, 117.39, 116.71, 59.36, 54.27, 45.27, 28.73, 28.32, 23.44, 10.91. <i>t</i><sub>R</sub> = 2.00 min (generic method). ESI-MS for C<sub>32</sub>H<sub>28</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 581.2, found <i>m</i>/<i>z</i> 582.2, 584.3 [M + H]<sup>+</sup>, 580.3, 582.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(1-cyclohexyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>28d</b>)</h3><div class="NLM_p last">Compound <b>28d</b> was synthesized via general procedure C<sub>1</sub> using <b>104c</b> (164 mg, 0.31 mmol) with succinic anhydride <b>115</b> (62 mg, 0.62 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–12% EtOH/DCM) to afford <b>28d</b> (169 mg, yield 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.07 (s, 1H), 7.96 (s, 1H), 7.65–7.49 (m, 5H), 7.48–7.41 (m, 2H), 7.28 (dd, <i>J</i> = 7.5, 1.8 Hz, 1H), 7.11 (d, <i>J</i> = 1.6 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.84 (dd, <i>J</i> = 8.8, 1.6 Hz, 1H), 5.41 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.54 (tt, <i>J</i> = 10.0, 5.5 Hz, 1H), 3.80 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.85 (dd, <i>J</i> = 18.5, 4.7 Hz, 1H), 2.53–2.46 (m, 2H), 2.28 (t, <i>J</i> = 6.9 Hz, 2H), 2.01–1.81 (m, 6H), 1.70 (dt, <i>J</i> = 12.8, 3.4 Hz, 1H), 1.58–1.43 (m, 2H), 1.26 (qt, <i>J</i> = 12.7, 3.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.74, 173.51, 168.59, 160.16, 152.36, 149.90, 137.70, 137.31, 134.64, 134.41, 132.08, 131.16, 129.51, 128.59, 128.48, 128.45, 128.30, 126.12, 126.07, 124.73, 123.92, 123.18, 120.71, 117.61, 117.24, 109.77, 59.12, 56.63, 45.50, 32.34, 28.67, 28.26, 25.08. <i>t</i><sub>R</sub> = 2.33 min (generic method). ESI-MS for C<sub>35</sub>H<sub>32</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 621.2, found <i>m</i>/<i>z</i> 622.3, 624.2 [M + H]<sup>+</sup>, 620.3, 622.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(2-cyclohexyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>29d</b>)</h3><div class="NLM_p last">Compound <b>29d</b> was synthesized via general procedure C<sub>1</sub> using <b>105c</b> (100 mg, 0.19 mmol) with succinic anhydride <b>115</b> (38 mg, 0.38 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>29d</b> (113 mg, yield 95%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.10 (s, 1H), 8.29 (d, <i>J</i> = 1.0 Hz, 1H), 7.65–7.55 (m, 3H), 7.53–7.48 (m, 1H), 7.44 (t, <i>J</i> = 8.7 Hz, 3H), 7.30–7.24 (m, 1H), 7.10–7.05 (m, 1H), 6.93 (d, <i>J</i> = 2.4 Hz, 1H), 6.60 (dd, <i>J</i> = 9.0, 1.7 Hz, 1H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.44 (tt, <i>J</i> = 11.2, 3.8 Hz, 1H), 3.77 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.82 (dd, <i>J</i> = 18.4, 4.7 Hz, 1H), 2.55–2.46 (m, 2H), 2.28 (t, <i>J</i> = 6.9 Hz, 2H), 2.14–2.02 (m, 2H), 1.95–1.81 (m, 4H), 1.70 (dd, <i>J</i> = 12.6, 3.7 Hz, 1H), 1.45 (qt, <i>J</i> = 12.5, 3.4 Hz, 2H), 1.25 (tdd, <i>J</i> = 15.6, 11.7, 7.8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.72, 173.52, 168.56, 160.14, 152.38, 149.87, 146.88, 137.30, 134.58, 134.35, 131.15, 129.52, 128.59, 128.51, 128.44, 128.30, 126.09, 126.05, 124.76, 123.49, 121.57, 120.73, 120.62, 117.61, 117.47, 116.76, 61.63, 59.36, 45.25, 33.29, 29.13, 28.69, 28.28, 24.92, 24.89. <i>t</i><sub>R</sub> = 2.20 min (generic method). ESI-MS for C<sub>35</sub>H<sub>32</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 621.2, found <i>m</i>/<i>z</i> 622.2, 624.2 [M + H]<sup>+</sup>, 620.3, 622.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-(1-ethyl-1<i>H</i>-indazol-5-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>30d</b>)</h3><div class="NLM_p last">Compound <b>30d</b> was synthesized via general procedure C<sub>1</sub> using <b>106c</b> (170 mg, 0.31 mmol) with succinic anhydride <b>115</b> (62 mg, 0.62 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>30d</b> (163 mg, yield 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.07 (s, 1H), 8.10 (d, <i>J</i> = 0.8 Hz, 1H), 7.99 (d, <i>J</i> = 1.6 Hz, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 7.67 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.51 (m, 5H), 7.48–7.40 (m, 2H), 7.30 (dd, <i>J</i> = 5.9, 1.8 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.89 (d, <i>J</i> = 8.3 Hz, 2H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.46 (q, <i>J</i> = 7.2 Hz, 2H), 3.78 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.86 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.59–2.44 (m, 2H), 2.31 (t, <i>J</i> = 6.8 Hz, 2H), 1.41 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.53, 168.65, 160.13, 152.47, 149.95, 140.77, 139.50, 138.23, 137.32, 134.63, 132.92, 132.54, 131.18, 129.43, 128.55, 128.47, 128.34, 126.90, 126.11, 126.07, 125.52, 124.67, 124.25, 120.70, 118.46, 117.62, 110.02, 58.72, 45.26, 43.15, 28.64, 28.26, 14.94. <i>t</i><sub>R</sub> = 2.24 min (generic method). ESI-MS for C<sub>37</sub>H<sub>30</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 643.2, found <i>m</i>/<i>z</i> 644.1, 646.1 [M + H]<sup>+</sup>, 642.2, 644.1 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-(2-ethyl-2<i>H</i>-indazol-5-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>31d</b>)</h3><div class="NLM_p last">Compound <b>31d</b> was synthesized via general procedure C<sub>1</sub> using <b>107c</b> (117 mg, 0.21 mmol) with succinic anhydride <b>115</b> (42 mg, 0.42 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–20% EtOH/DCM) to afford <b>31d</b> (66 mg, yield 49%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 12.15 (s, 1H), 8.42 (d, <i>J</i> = 0.9 Hz, 1H), 7.92 (t, <i>J</i> = 1.3 Hz, 1H), 7.68 (d, <i>J</i> = 8.9 Hz, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.50 (m, 6H), 7.48–7.41 (m, 2H), 7.30 (dd, <i>J</i> = 6.1, 1.9 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.87 (d, <i>J</i> = 8.4 Hz, 2H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.47 (q, <i>J</i> = 7.3 Hz, 2H), 3.78 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.86 (dd, <i>J</i> = 18.5, 4.7 Hz, 1H), 2.57–2.43 (m, 2H), 2.30 (t, <i>J</i> = 6.9 Hz, 2H), 1.52 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.63, 173.52, 168.64, 160.13, 152.47, 149.95, 147.41, 140.72, 137.33, 132.68, 128.56, 128.47, 128.35, 126.74, 126.11, 126.06, 126.02, 125.23, 124.68, 123.68, 121.91, 120.71, 117.89, 117.63, 117.45, 58.72, 47.80, 28.93, 28.63, 28.26, 15.76. <i>t</i><sub>R</sub> = 2.10 min (generic method). ESI-MS for C<sub>37</sub>H<sub>30</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 643.2, found <i>m</i>/<i>z</i> 644.2, 646.2 [M + H]<sup>+</sup>, 642.3, 644.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 4-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-(1-propyl-1<i>H</i>-pyrazol-4-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>32d</b>)</h3><div class="NLM_p last">Compound <b>32d</b> was synthesized via general procedure C<sub>1</sub> using <b>108c</b> (155 mg, 0.30 mmol) with succinic anhydride <b>115</b> (60 mg, 0.60 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–30% EtOH/DCM) to afford <b>32d</b> (189 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.10 (s, 1H), 8.14 (s, 1H), 7.84 (s, 1H), 7.62 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59–7.50 (m, 3H), 7.48–7.36 (m, 4H), 7.31–7.25 (m, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.77 (d, <i>J</i> = 8.0 Hz, 2H), 5.28 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.07 (t, <i>J</i> = 7.0 Hz, 2H), 3.75 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.82 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.48–2.41 (m, 2H), 2.28 (t, <i>J</i> = 6.9 Hz, 2H), 1.96–1.87 (m, 2H), 0.85 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.50, 168.56, 160.12, 152.39, 149.89, 139.75, 137.30, 135.88, 134.62, 131.56, 131.16, 129.42, 128.53, 128.44, 128.29, 126.88, 126.09, 125.98, 124.96, 124.68, 121.24, 120.70, 117.60, 58.75, 53.00, 45.20, 28.63, 28.26, 23.17, 10.93. <i>t</i><sub>R</sub> = 2.09 min (generic method). ESI-MS for C<sub>34</sub>H<sub>30</sub>ClN<sub>5</sub>O<sub>4</sub>: calculated 607.2, found <i>m</i>/<i>z</i> 608.0, 610.0 [M + H]<sup>+</sup>, 606.1, 608.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 6-Chloro-3-(5-(4-chlorophenyl)-1-propionyl-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>33d</b>)</h3><div class="NLM_p last">Compound <b>33d</b> was synthesized via general procedure C<sub>3</sub> using <b>87c</b> (200 mg, 0.46 mmol) and propionic acid <b>117</b> (60.5 μL, 0.83 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–2% MeOH/DCM) to afford <b>33d</b> (76 mg, yield 34%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.63 (dd, <i>J</i> = 9.0, 2.4 Hz, 1H), 7.54–7.49 (m, 3H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 7.41–7.39 (m, 1H), 7.30–7.27 (m, 3H), 6.92 (d, <i>J</i> = 2.4 Hz, 1H), 6.80 (d, <i>J</i> = 8.8 Hz, 2H), 5.31 (dd, <i>J</i> = 12.2, 4.6 Hz, 1H), 3.74 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.80 (dd, <i>J</i> = 18.4, 4.0 Hz, 1H), 2.31–2.14 (m, 2H), 0.82 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 206.88, 170.96, 160.55, 152.65, 150.33, 141.76, 137.75, 135.13, 132.03, 131.64, 129.80, 128.95, 128.87, 128.82, 128.79, 127.81, 126.55, 126.48, 125.04, 121.10, 118.07, 110.00, 58.61, 45.45, 31.11, 27.06, 9.26. ESI-MS for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub>: calculated 489.1, found <i>m</i>/<i>z</i> 490.2, 492.1, 494.1 [M + H]<sup>+</sup>; 488.3, 490.3, 492.1 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 98%.</div></div><div id="sec4_1_1_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 3-(5-(4-Bromophenyl)-1-propionyl-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>34d</b>)</h3><div class="NLM_p last">Compound <b>34d</b> was synthesized via general procedure C<sub>3</sub> using <b>88c</b> (150 mg, 0.31 mmol) and propionic acid <b>117</b> (41.8 μL, 0.56 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–0.5% MeOH/DCM) to afford <b>34d</b> (60 mg, yield 36%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.41 (s, 1H), 7.64 (dd, <i>J</i> = 12.8, 1.6 Hz, 1H), 7.55–7.50 (m, 3H), 7.46–7.41 (m, 4H), 7.28 (d, <i>J</i> = 6.8 Hz, 1H), 6.93 (d, <i>J</i> = 2.0 Hz, 1H), 6.75 (d, <i>J</i> = 8.0 Hz, 2H), 5.30 (dd, <i>J</i> = 12.4, 4.0 Hz, 1H), 3.75 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.80 (dd, <i>J</i> = 18.2, 4.6 Hz, 1H), 2.31–2.14 (m, 2H), 0.83 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 206.78, 171.02, 160.54, 152.60, 150.33, 142.18, 137.77, 135.16, 131.72, 131.61, 129.79, 128.97, 128.92, 128.84, 128.78, 128.18, 126.56, 126.49, 125.03, 121.13, 120.51, 118.06, 58.73, 45.40, 31.08, 27.05, 9.23. ESI-MS for C<sub>27</sub>H<sub>21</sub>BrClN<sub>3</sub>O<sub>2</sub>: calculated 533.0, found <i>m</i>/<i>z</i> 534.2, 536.1, 538.1 [M + H]<sup>+</sup>; 532.2, 534.1, 536.2 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 97%.</div></div><div id="sec4_1_1_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 6-Chloro-3-(5-(4-methoxyphenyl)-1-propionyl-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>35d</b>)</h3><div class="NLM_p last">Compound <b>35d</b> was synthesized via general procedure C<sub>3</sub> using <b>89c</b> (150 mg, 0.36 mmol) and propionic acid <b>117</b> (48.5 μL, 0.65 mmol). The crude was purified by direct phase flash column chromatography (SiO<sub>2</sub> gold 24 g; 0–2% MeOH/DCM), to afford <b>35d</b> (100 mg, yield 59%). <sup>1</sup>H NMR (400 MHz, DMSO<i>-d</i><sub><i>6</i></sub>) δ 12.36 (s, 1H), 7.64 (dd, <i>J</i> = 8.6, 2.4 Hz, 1H), 7.56–7.52 (m, 3H), 7.45 (d, <i>J</i> = 8.4 Hz, 1H), 7.40–7.38 (m, 1H), 7.29–7.27 (m, 1H), 6.92 (d, <i>J</i> = 2.0 Hz, 1H), 6.75 (dd, <i>J</i> = 15.2, 9.2 Hz, 4H), 5.24 (dd, <i>J</i> = 12.0, 4.8 Hz, 1H), 3.72 (s, 3H), 3.70 (dd, <i>J</i> = 16.0, 12.0 Hz, 1H), 2.81 (dd, <i>J</i> = 20.0, 4.4 Hz, 1H), 2.29–2.12 (m, 2H), 0.82 (t, <i>J</i> = 8.0 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO<i>-d</i><sub><i>6</i></sub>) δ 170.80, 160.60, 158.67, 152.56, 150.23, 137.74, 135.16, 134.94, 131.58, 129.76, 129.03, 128.89, 128.84, 128.77, 127.20, 126.51, 126.47, 125.22, 121.15, 118.05, 114.15, 58.73, 55.49, 45.59, 27.12, 9.31. ESI-MS for C<sub>28</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>3</sub>: calculated 485.1, found <i>m</i>/<i>z</i> 486.4, 488.4 [M + H]+; 484.3, 486.3 [M – H]-. UPLC–MS purity (UV at 215 nm) 98%.</div></div><div id="sec4_1_1_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> 3-(1-Acetyl-5-(4-chlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>36d</b>)</h3><div class="NLM_p last">Compound <b>36d</b> was synthesized via general procedure C<sub>2</sub> using <b>87c</b> (210 mg, 0.48 mmol) with acetic anhydride <b>116</b> (888 μL, 0.96 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–1.5% MeOH/DCM) to afford <b>36d</b> (200 mg, yield 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.41 (s, 1H), 7.64 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.58–7.53 (m, 3H), 7.44 (d, <i>J</i> = 8.0 Hz, 1H), 7.41–7.39 (m, 1H), 7.30–7.28 (m, 3H), 6.93 (d, <i>J</i> = 0.8 Hz, 1H), 6.80 (d, <i>J</i> = 7.6 Hz, 2H), 5.32 (dd, <i>J</i> = 12.0, 8.0 Hz, 1H), 3.74 (dd, <i>J</i> = 20.0, 12.0 Hz, 1H), 2.82 (dd, <i>J</i> = 16.0, 4.0 Hz, 1H), 1.87 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.56, 160.54, 152.67, 150.39, 141.60, 137.75, 135.11, 132.05, 131.65, 129.86, 129.01, 128.96, 128.81, 128.75, 127.83, 126.55, 126.49, 124.96, 121.10, 118.06, 58.56, 45.62, 21.82. ESI-MS for C<sub>26</sub>H<sub>19</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub>: calculated 475.1, found <i>m</i>/<i>z</i> 476.3, 478.4, 480.4 [M + H]<sup>+</sup>; 474.3, 476.3, 478.1 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> 3-(1-Acetyl-5-(4-bromophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>37d</b>)</h3><div class="NLM_p last">Compound <b>37d</b> was synthesized via general procedure C<sub>2</sub> using <b>88c</b> (100 mg, 0.21 mmol) with acetic anhydride <b>116</b> (40 μL, 0.42 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 12 g, 0–2% MeOH/DCM) to afford <b>37d</b> (80 mg, yield 73%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 7.64 (dd, <i>J</i> = 8.0, 4.0 Hz, 1H), 7.57–7.53 (m, 3H), 7.45–7.39 (m, 4H), 7.28 (dd, <i>J</i> = 6.0, 1.6 Hz, 1H), 6.93 (d, <i>J</i> = 2.0 Hz, 1H), 6.75 (d, <i>J</i> = 8.8 Hz, 2H), 5.30 (dd, <i>J</i> = 12.0, 4.4 Hz, 1H), 3.74 (dd, <i>J</i> = 18.5, 12.0 Hz, 1H), 2.82 (dd, <i>J</i> = 4.0, 20.0 Hz, 1H), 1.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.58, 160.54, 152.64, 150.39, 142.02, 137.78, 135.15, 131.72, 131.63, 129.83, 129.03, 128.94, 128.78, 128.74, 128.20, 126.56, 126.50, 124.96, 121.13, 120.54, 118.06, 58.67, 45.58, 21.78. ESI-MS for C<sub>26</sub>H<sub>19</sub>BrClN<sub>3</sub>O<sub>2</sub>: calculated 519.0, found <i>m</i>/<i>z</i> 520.1, 522.1, 524.1 [M + H]<sup>+</sup>; 518.1, 520.1, 522.2 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 3-(1-Acetyl-5-(4-methoxyphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>38d</b>)</h3><div class="NLM_p last">Compound <b>38d</b> was synthesized via general procedure C<sub>2</sub> using <b>89c</b> (65 mg, 0.15 mmol) with acetic anhydride <b>116</b> (28 μL, 0.30 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 12 g, 0–1.5% MeOH/DCM) to afford <b>38d</b> (30 mg, yield 43%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 7.55–7.52 (m, 3H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.40–7.39 (m, 1H), 7.29 (d, <i>J</i> = 6.8 Hz, 1H), 6.93 (d, <i>J</i> = 2.0 Hz, 1H), 6.75 (dd, <i>J</i> = 8.8, 16.0 Hz, 4H), 5.24 (dd, <i>J</i> = 12.0, 4.0 Hz, 1H), 3.72 (s, 3H), 3.70–3.65 (m,1H), 2.83 (dd, <i>J</i> = 4.4, 18.4 Hz, 2H), 1.84 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.42, 160.60, 158.70, 152.63, 150.29, 137.75, 135.14, 134.79, 131.60, 129.80, 129.09, 128.92, 128.78, 128.75, 127.24, 126.52, 126.49, 125.13, 121.14, 118.07, 114.15, 58.67, 55.49, 45.74, 21.90. ESI-MS for C<sub>27</sub>H<sub>22</sub>ClN<sub>3</sub>O<sub>3</sub>: calculated 471.1, found <i>m</i>/<i>z</i> 472.2, 474.2, [M + H]<sup>+</sup>; 470.3, 472.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 3-(1-Acetyl-5-(4-(<i>tert</i>-butyl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>39d</b>)</h3><div class="NLM_p last">Compound <b>39d</b> was synthesized via general procedure C<sub>2</sub> using <b>90c</b> (141 mg, 0.31 mmol) with acetic anhydride <b>116</b> (59 μL, 0.62 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–5% EtOH/DCM) to afford <b>39d</b> (105 mg, yield 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 7.63 (dt, <i>J</i> = 9.8, 4.9 Hz, 1H), 7.60–7.48 (m, 3H), 7.44 (t, <i>J</i> = 9.5 Hz, 1H), 7.41–7.36 (m, 1H), 7.30 (d, <i>J</i> = 7.1 Hz, 1H), 7.24 (d, <i>J</i> = 8.3 Hz, 2H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.75 (d, <i>J</i> = 8.3 Hz, 2H), 5.32–5.17 (m, 1H), 3.69 (dt, <i>J</i> = 28.5, 14.3 Hz, 1H), 2.86 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.86 (s, 3H), 1.27 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.01, 160.12, 152.20, 149.87, 149.25, 139.30, 137.30, 134.72, 131.14, 129.30, 128.66, 128.44, 128.31, 128.29, 126.07, 126.04, 125.16, 125.13, 124.65, 120.69, 117.60, 58.45, 45.30, 34.12, 31.10, 21.42. <i>t</i><sub>R</sub> = 1.83 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>28</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 497.2, found <i>m</i>/<i>z</i> 498.5, 500.5 [M + H]<sup>+</sup>; 494.4, 496.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> 3-(1-Acetyl-5-(4-(trifluoromethyl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>40d</b>)</h3><div class="NLM_p last">Compound <b>40d</b> was synthesized via general procedure C<sub>2</sub> using <b>92c</b> (326 mg, 0.70 mmol) with acetic anhydride <b>116</b> (132 μL, 1.40 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–5% EtOH/DCM) to afford <b>40d</b> (268 mg, yield 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 7.67–7.49 (m, 6H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 7.44–7.39 (m, 1H), 7.32–7.25 (m, 1H), 7.03 (d, <i>J</i> = 8.0 Hz, 2H), 6.95 (d, <i>J</i> = 2.4 Hz, 1H), 5.42 (dd, <i>J</i> = 12.1, 4.7 Hz, 1H), 3.79 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.87 (dd, <i>J</i> = 18.5, 4.8 Hz, 1H), 1.89 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.21, 160.08, 152.27, 150.01, 146.70, 137.34, 134.69, 131.23, 129.39, 128.54, 128.51, 128.33, 127.90, 127.58, 126.24, 126.11, 126.05, 125.52, 125.43, 125.39, 125.35, 124.41, 122.82, 120.62, 117.63, 58.36, 45.14, 21.30. <i>t</i><sub>R</sub> = 2.55 min (generic method). ESI-MS for C<sub>27</sub>H<sub>19</sub>ClF<sub>3</sub>N<sub>3</sub>O<sub>2</sub>: calculated 509.1, found <i>m</i>/<i>z</i> 510.5, 513.5 [M + H]<sup>+</sup>; 508.4, 510.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> 3-(1-Acetyl-5-(furan-2-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>41d</b>)</h3><div class="NLM_p last">Compound <b>41d</b> was synthesized via general procedure C<sub>2</sub> using <b>93c</b> (202 mg, 0.52 mmol) with acetic anhydride <b>116</b> (98 μL, 1.04 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–5% EtOH/DCM) to afford <b>41d</b> (162 mg, yield 73%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.52–7.42 (m, 5H), 7.35–7.29 (m, 2H), 6.96 (d, <i>J</i> = 2.3 Hz, 1H), 6.33 (dd, <i>J</i> = 3.3, 1.9 Hz, 1H), 6.03 (d, <i>J</i> = 3.2 Hz, 1H), 5.38 (dd, <i>J</i> = 11.9, 4.6 Hz, 1H), 3.57 (dd, <i>J</i> = 18.2, 11.9 Hz, 1H), 3.20 (dd, <i>J</i> = 18.2, 4.7 Hz, 1H), 1.75 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.26, 160.12, 152.98, 152.28, 150.04, 142.02, 137.37, 134.92, 131.18, 128.87, 128.80, 128.25, 128.19, 126.12, 126.07, 124.31, 120.62, 117.59, 110.33, 106.33, 52.35, 41.84, 21.33. <i>t</i><sub>R</sub> = 0.95 min (apolar method). ESI-MS for C<sub>24</sub>H<sub>18</sub>ClN<sub>3</sub>O<sub>3</sub>: calculated 431.1, found <i>m</i>/<i>z</i> 432.4, 434.4 [M + H]<sup>+</sup>; 430.4, 432.5 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> 3-(1-Acetyl-5-(4′-chloro-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>42d</b>)</h3><div class="NLM_p last">Compound <b>42d</b> was synthesized via general procedure C<sub>2</sub> using <b>95c</b> (224 mg, 0.44 mmol) with acetic anhydride <b>116</b> (83 μL, 0.88 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>42d</b> (245 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 7.71–7.48 (m, 10H), 7.48–7.39 (m, 2H), 7.29 (dt, <i>J</i> = 5.2, 2.0 Hz, 1H), 6.94 (d, <i>J</i> = 2.4 Hz, 1H), 6.90 (d, <i>J</i> = 8.3 Hz, 2H), 5.35 (dd, <i>J</i> = 12.1, 4.6 Hz, 1H), 3.78 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.88 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 1.88 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.09, 160.12, 152.23, 149.93, 141.80, 138.63, 137.66, 137.30, 134.73, 132.26, 131.16, 129.39, 128.82, 128.59, 128.49, 128.40, 128.37, 128.29, 126.74, 126.15, 126.10, 126.05, 124.60, 120.67, 117.60, 58.48, 45.29, 21.41. <i>t</i><sub>R</sub> = 2.00 min (apolar method). ESI-MS for C<sub>32</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub>: calculated 551.1, found <i>m</i>/<i>z</i> 552.0, 554.0, 556.1 [M + H]<sup>+</sup>; 550.1, 552.1, 554.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> 3-(1-Acetyl-5-(4′-bromo-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>43d</b>)</h3><div class="NLM_p last">Compound <b>43d</b> was synthesized via general procedure C<sub>2</sub> using <b>96c</b> (203 mg, 0.36 mmol) with acetic anhydride <b>116</b> (68 μL, 0.72 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>43d</b> (175 mg, yield 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 7.66–7.36 (m, 12H), 7.32–7.24 (m, 1H), 6.98–6.85 (m, 3H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.78 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.90 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.88 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.10, 160.12, 152.20, 149.92, 141.81, 138.97, 137.70, 137.30, 134.73, 131.71, 131.12, 129.36, 128.67, 128.58, 128.46, 128.36, 128.27, 126.67, 126.17, 126.12, 126.04, 124.57, 120.84, 120.65, 117.58, 79.15, 58.51, 45.28, 21.39. <i>t</i><sub>R</sub> = 2.52 min (apolar method). ESI-MS for C<sub>32</sub>H<sub>23</sub>BrClN<sub>3</sub>O<sub>2</sub>: calculated 595.1, found <i>m</i>/<i>z</i> 595.9, 597.9, 600.0 [M + H]<sup>+</sup>; 594.0, 596.0, 597.9 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> 3-(1-Acetyl-5-(4′-methoxy-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>44d</b>)</h3><div class="NLM_p last">Compound <b>44d</b> was synthesized via general procedure C<sub>2</sub> using <b>97c</b> (179 mg, 0.35 mmol) with acetic anhydride <b>116</b> (66 μL, 0.70 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–8% EtOH/DCM) to afford <b>44d</b> (158 mg, yield 83%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.53 (m, 5H), 7.45 (d, <i>J</i> = 8.5 Hz, 3H), 7.43–7.39 (m, 1H), 7.31–7.27 (m, 1H), 7.03–6.99 (m, 2H), 6.94 (d, <i>J</i> = 2.4 Hz, 1H), 6.86 (d, <i>J</i> = 8.3 Hz, 2H), 5.33 (dd, <i>J</i> = 12.1, 4.6 Hz, 1H), 3.79 (s, 4H), 2.88 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 1.88 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.09, 160.17, 158.87, 152.28, 149.95, 140.70, 138.74, 137.33, 134.75, 132.24, 131.20, 129.43, 128.64, 128.53, 128.43, 128.33, 127.72, 126.28, 126.12, 126.08, 126.04, 124.66, 120.71, 117.64, 114.35, 58.53, 55.16, 45.34, 21.47. <i>t</i><sub>R</sub> = 1.65 min (apolar method). ESI-MS for C<sub>33</sub>H<sub>26</sub>ClN<sub>3</sub>O<sub>3</sub>: calculated 547.2, found <i>m</i>/<i>z</i> 548.0, 550.0 [M + H]<sup>+</sup>; 546.0, 548.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> 3-(1-Acetyl-5-(1-methyl-1<i>H</i>-indol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>45d</b>)</h3><div class="NLM_p last">Compound <b>45d</b> was synthesized via general procedure C<sub>2</sub> using <b>98c</b> (127 mg, 0.28 mmol) with acetic anhydride <b>116</b> (53 μL, 0.56 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>45d</b> (107 mg, yield 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 7.64–7.56 (m, 3H), 7.53 (ddd, <i>J</i> = 8.9, 5.6, 3.3 Hz, 1H), 7.47–7.40 (m, 2H), 7.31–7.23 (m, 3H), 6.99 (d, <i>J</i> = 1.7 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.65 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 6.35 (dd, <i>J</i> = 3.1, 0.8 Hz, 1H), 5.36 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 3.75 (s, 4H), 2.90 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.86 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.93, 160.20, 152.06, 149.80, 137.30, 135.61, 134.79, 133.10, 131.09, 129.92, 129.46, 128.59, 128.47, 128.37, 128.23, 127.74, 126.06, 124.81, 120.74, 119.02, 117.58, 117.16, 109.63, 100.26, 59.32, 45.86, 32.45, 21.54. <i>t</i><sub>R</sub> = 2.41 min (generic method). ESI-MS for C<sub>29</sub>H<sub>23</sub>ClN<sub>4</sub>O<sub>2</sub>: calculated 494.1, found <i>m</i>/<i>z</i> 495.2, 497.2 [M + H]<sup>+</sup>; 493.3, 495.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> 3-(1-Acetyl-5-(1-ethyl-1<i>H</i>-indol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>46d</b>)</h3><div class="NLM_p last">Compound <b>46d</b> was synthesized via general procedure C<sub>2</sub> using <b>99c</b> (168 mg, 0.36 mmol) with acetic anhydride <b>116</b> (68 μL, 0.72 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>46d</b> (128 mg, yield 70%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 7.64–7.50 (m, 4H), 7.47–7.40 (m, 2H), 7.35 (d, <i>J</i> = 3.1 Hz, 1H), 7.32–7.27 (m, 2H), 6.97 (dd, <i>J</i> = 22.0, 2.0 Hz, 2H), 6.64 (dd, <i>J</i> = 8.5, 1.7 Hz, 1H), 6.35 (dd, <i>J</i> = 3.1, 0.8 Hz, 1H), 5.35 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.16 (q, <i>J</i> = 7.2 Hz, 2H), 3.78 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.91 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.86 (s, 3H), 1.33 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.95, 160.20, 152.07, 149.81, 137.29, 134.80, 134.57, 133.07, 131.09, 129.43, 128.61, 128.47, 128.36, 128.31, 128.24, 127.90, 126.07, 124.82, 120.74, 118.97, 117.58, 117.29, 109.62, 100.48, 59.33, 45.85, 40.28, 21.55, 21.18, 15.52. <i>t</i><sub>R</sub> = 2.51 min (generic method). ESI-MS for C<sub>30</sub>H<sub>25</sub>ClN<sub>4</sub>O<sub>2</sub>: calculated 508.2, found <i>m</i>/<i>z</i> 509.3, 511.2 [M + H]<sup>+</sup>; 507.3, 509.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 3-(1-Acetyl-5-(1-ethyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>47d</b>)</h3><div class="NLM_p last">Compound <b>47d</b> was synthesized via general procedure C<sub>2</sub> using <b>100c</b> (131 mg, 0.28 mmol) with acetic anhydride <b>116</b> (53 μL, 0.56 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 12 g, 0–5% EtOH/DCM) to afford <b>47d</b> (118 mg, yield 82%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 7.98 (d, <i>J</i> = 0.9 Hz, 1H), 7.66–7.57 (m, 3H), 7.57–7.49 (m, 2H), 7.48–7.40 (m, 2H), 7.29 (ddt, <i>J</i> = 7.6, 2.0, 1.0 Hz, 1H), 7.15 (t, <i>J</i> = 1.1 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.84 (dd, <i>J</i> = 8.7, 1.6 Hz, 1H), 5.41 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 4.41 (q, <i>J</i> = 7.2 Hz, 2H), 3.79 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.88 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.88 (s, 3H), 1.37 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.05, 160.18, 152.21, 149.88, 138.03, 137.33, 134.72, 134.44, 132.33, 131.16, 129.51, 128.57, 128.41, 128.29, 126.08, 124.71, 124.11, 123.28, 120.72, 117.63, 117.30, 109.78, 58.90, 45.57, 43.06, 21.50, 14.95. <i>t</i><sub>R</sub> = 2.29 min (generic method). ESI-MS for C<sub>29</sub>H<sub>24</sub>ClN<sub>5</sub>O<sub>2</sub>: calculated 509.2, found <i>m</i>/<i>z</i> 510.2, 512.2 [M + H]<sup>+</sup>, 508.3, 510.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> 3-(1-Acetyl-5-(2-ethyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>48d</b>)</h3><div class="NLM_p last">Compound <b>48d</b> was synthesized via general procedure C<sub>2</sub> using <b>101c</b> (80 mg, 0.17 mmol) with acetic anhydride <b>116</b> (32 μL, 0.34 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>48d</b> (67 mg, yield 77%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 8.31 (d, <i>J</i> = 0.9 Hz, 1H), 7.67–7.50 (m, 4H), 7.48–7.42 (m, 3H), 7.36–7.26 (m, 1H), 7.13–7.05 (m, 1H), 6.94 (d, <i>J</i> = 2.4 Hz, 1H), 6.65 (dd, <i>J</i> = 9.0, 1.7 Hz, 1H), 5.36 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.44 (q, <i>J</i> = 7.3 Hz, 2H), 3.78 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.87 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.89 (s, 3H), 1.50 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.04, 160.18, 152.20, 149.83, 147.32, 137.35, 134.71, 134.46, 131.13, 129.52, 128.57, 128.54, 128.37, 128.27, 126.04, 124.73, 123.59, 123.01, 120.93, 120.72, 117.63, 117.34, 116.65, 59.13, 47.69, 45.35, 21.52, 15.84. <i>t</i><sub>R</sub> = 2.14 min (generic method). ESI-MS for C<sub>29</sub>H<sub>24</sub>ClN<sub>5</sub>O<sub>2</sub>: calculated 509.2, found <i>m</i>/<i>z</i> 510.2, 512.2 [M + H]<sup>+</sup>, 508.2, 510.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> 3-(1-Acetyl-5-(1-propyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>49d</b>)</h3><div class="NLM_p last">Compound <b>49d</b> was synthesized via general procedure C<sub>2</sub> using <b>102c</b> (188 mg, 0.39 mmol) with acetic anhydride <b>116</b> (74 μL, 0.78 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>49d</b> (184 mg, yield 90%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 7.99 (d, <i>J</i> = 0.9 Hz, 1H), 7.65–7.57 (m, 3H), 7.56–7.50 (m, 2H), 7.47–7.41 (m, 2H), 7.36–7.25 (m, 1H), 7.17–7.09 (m, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.85 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 5.41 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 4.33 (t, <i>J</i> = 6.9 Hz, 2H), 3.78 (dd, <i>J</i> = 18.5, 12.0 Hz, 1H), 2.89 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 1.90–1.77 (m, 5H), 0.82 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.06, 160.14, 152.20, 149.85, 147.37, 137.29, 134.69, 134.43, 131.13, 129.52, 128.53, 128.35, 128.26, 126.07, 124.73, 123.73, 123.58, 120.81, 120.71, 117.59, 117.37, 116.65, 59.13, 54.23, 45.34, 23.40, 21.51, 10.86. <i>t</i><sub>R</sub> = 1.21 min (generic method). ESI-MS for C<sub>30</sub>H<sub>26</sub>ClN<sub>5</sub>O<sub>2</sub>: calculated 523.2, found <i>m</i>/<i>z</i> 524.1, 526.2 [M + H]<sup>+</sup>, 522.3, 524.2 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 94%.</div></div><div id="sec4_1_1_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> 3-(1-Acetyl-5-(2-propyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>50d</b>)</h3><div class="NLM_p last">Compound <b>50d</b> was synthesized via general procedure C<sub>2</sub> using <b>103c</b> (132 mg, 0.27 mmol) with acetic anhydride <b>116</b> (51 μL, 0.54 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–10% EtOH/DCM) to afford <b>50d</b> (120 mg, yield 85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 8.29 (d, <i>J</i> = 1.0 Hz, 1H), 7.65–7.56 (m, 3H), 7.56–7.49 (m, 1H), 7.48–7.40 (m, 3H), 7.32–7.26 (m, 1H), 7.08 (t, <i>J</i> = 1.2 Hz, 1H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 6.64 (dd, <i>J</i> = 8.9, 1.7 Hz, 1H), 5.35 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 4.36 (t, <i>J</i> = 6.9 Hz, 2H), 3.77 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.87 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.97–1.83 (m, 5H), 0.83 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.07, 160.15, 152.21, 149.85, 147.38, 137.31, 134.69, 134.43, 131.13, 129.52, 128.57, 128.53, 128.36, 128.26, 126.07, 126.04, 124.74, 123.74, 123.58, 120.81, 120.72, 117.61, 117.38, 116.65, 59.13, 54.24, 45.35, 23.41, 21.52, 10.87. <i>t</i><sub>R</sub> = 1.01 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>26</sub>ClN<sub>5</sub>O<sub>2</sub>: calculated 523.2, found <i>m</i>/<i>z</i> 524.2, 526.2 [M + H]<sup>+</sup>, 522.3, 524.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> 3-(1-Acetyl-5-(4-(1-propyl-1<i>H</i>-pyrazol-4-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>51d</b>)</h3><div class="NLM_p last">Compound <b>51d</b> was synthesized via general procedure C<sub>2</sub> using <b>108c</b> (146 mg, 0.31 mmol) with acetic anhydride <b>116</b> (59 μL, 0.62 mmol) (165 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 0–8% EtOH/DCM) to afford <b>51d</b> (138 mg, yield 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 8.13 (s, 1H), 7.83 (d, <i>J</i> = 0.7 Hz, 1H), 7.62 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60–7.52 (m, 3H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.43–7.38 (m, 3H), 7.30 (dt, <i>J</i> = 6.2, 1.7 Hz, 1H), 6.94 (d, <i>J</i> = 2.3 Hz, 1H), 6.79 (d, <i>J</i> = 8.3 Hz, 2H), 5.28 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 4.07 (t, <i>J</i> = 7.0 Hz, 2H), 3.75 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.86 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 1.91–1.75 (m, 5H), 0.85 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.01, 160.14, 152.19, 149.89, 139.79, 137.30, 135.89, 134.74, 131.59, 131.15, 129.41, 128.60, 128.49, 128.38, 128.26, 126.88, 126.08, 126.00, 124.96, 124.66, 121.22, 120.69, 117.59, 58.57, 52.99, 45.27, 23.17, 21.42, 10.92. <i>t</i><sub>R</sub> = 1.22 min (apolar method). ESI-MS for C<sub>32</sub>H<sub>28</sub>ClN<sub>5</sub>O<sub>2</sub>: calculated 549.2, found <i>m</i>/<i>z</i> 550.0, 552.0 [M + H]<sup>+</sup>, 548.1, 550.0 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) 99%.</div></div><div id="sec4_1_1_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> 4-(5-(4-Fluorophenyl)-3-(2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>52d</b>)</h3><div class="NLM_p last">Compound <b>52d</b> was synthesized via general procedure C<sub>1</sub> using <b>109c</b> (150 mg, 0.39 mmol) with succinic anhydride <b>115</b> (78 mg, 0.78 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 20–50% EtOH/DCM) to afford <b>52d</b> (114 mg, yield 59%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.22 (s, 1H), 12.01 (s, 1H), 7.63–7.34 (m, 6H), 7.25 (dt, <i>J</i> = 6.8, 1.9 Hz, 1H), 7.14 (ddd, <i>J</i> = 8.2, 7.0, 1.2 Hz, 1H), 7.07–6.98 (m, 3H), 6.86–6.78 (m, 2H), 5.31 (dd, <i>J</i> = 12.0, 4.5 Hz, 1H), 3.72 (dd, <i>J</i> = 18.4, 12.0 Hz, 1H), 2.78 (dd, <i>J</i> = 18.4, 4.6 Hz, 1H), 2.47 (m, 2H), 2.32–2.25 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.48, 168.56, 160.25, 152.77, 151.05, 141.11, 138.53, 138.42, 135.16, 131.27, 129.38, 128.53, 128.22, 128.20, 128.13, 127.54, 127.45, 127.38, 123.30, 122.25, 119.35, 115.51, 115.16, 114.95, 58.16, 45.27, 28.58, 28.23. <i>t</i><sub>R</sub> = 1.90 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>FN<sub>3</sub>O<sub>4</sub>: calculated 483.2, found <i>m</i>/<i>z</i> 484.5 [M + H]<sup>+</sup>, 482.5 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> 4-(5-(4-Bromophenyl)-3-(2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydropyrazol-1-yl)-4-oxobutanoic Acid (<b>53d</b>)</h3><div class="NLM_p last">Compound <b>53d</b> was synthesized via general procedure C<sub>1</sub> using <b>110c</b> (41 mg, 0.09 mmol) with succinic anhydride <b>115</b> (18 mg, 0.18 mmol) (120 °C, 200 W). Title compound <b>53d</b> was obtained after the acidic workup (46 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.22 (s, 1H), 12.01 (s, 1H), 7.61–7.45 (m, 4H), 7.45–7.36 (m, 4H), 7.25 (dt, <i>J</i> = 6.8, 1.9 Hz, 1H), 7.14 (ddd, <i>J</i> = 8.3, 7.0, 1.2 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 6.74 (d, <i>J</i> = 8.4 Hz, 2H), 5.29 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.74 (dd, <i>J</i> = 18.4, 12.1 Hz, 1H), 2.77 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.50 (m, 2H), 2.28 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.46, 168.60, 160.25, 152.78, 151.09, 141.60, 138.54, 135.17, 131.29, 131.23, 129.41, 128.51, 128.21, 128.14, 127.75, 127.39, 123.23, 122.27, 121.42, 120.03, 119.35, 115.53, 58.30, 45.12, 28.55, 28.22. <i>t</i><sub>R</sub> = 2.03 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>BrN<sub>3</sub>O<sub>4</sub>: calculated 543.1, found <i>m</i>/<i>z</i> 544.5, 546.5 [M + H]<sup>+</sup>; 542.5, 544.4. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> 4-(5-(3-Bromophenyl)-3-(2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>54d</b>)</h3><div class="NLM_p last">Compound <b>54d</b> was synthesized via general procedure C<sub>1</sub> using <b>111c</b> (121 mg, 0.27 mmol) with succinic anhydride <b>115</b> (54 mg, 0.54 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 20–90% EtOH/DCM) to afford <b>54d</b> (11 mg, yield 7%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.24 (s, 1H), 12.13 (s, 1H), 7.61–7.34 (m, 7H), 7.26 (dt, <i>J</i> = 7.8, 1.8 Hz, 1H), 7.23–7.11 (m, 3H), 7.06 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 6.76 (dt, <i>J</i> = 7.8, 1.3 Hz, 1H), 5.32 (dd, <i>J</i> = 12.1, 4.8 Hz, 1H), 3.74 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.84 (dd, <i>J</i> = 18.5, 4.8 Hz, 1H), 2.51 (m, 2H), 2.33–2.26 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 174.08, 173.94, 169.19, 160.76, 153.36, 151.60, 145.30, 139.04, 135.72, 131.79, 131.18, 130.45, 129.75, 128.92, 128.71, 128.56, 127.90, 124.94, 123.65, 122.75, 122.09, 119.80, 116.01, 58.80, 45.64, 29.39, 29.01, 28.67. <i>t</i><sub>R</sub> = 2.00 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>BrN<sub>3</sub>O<sub>4</sub>: calculated 543.1, found <i>m</i>/<i>z</i> 544.4, 546.4 [M + H]<sup>+</sup>; 542.4, 544.4 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> 4-(5-(4-Chlorophenyl)-3-(2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>55d</b>)</h3><div class="NLM_p last">Compound <b>55d</b> was synthesized via general procedure C<sub>1</sub> using <b>112c</b> (121 mg, 0.27 mmol) with succinic anhydride <b>115</b> (54 mg, 054 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 4–20% EtOH/DCM) to afford <b>55d</b> (130 mg, yield 67%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.23 (s, 1H), 12.05 (s, 1H), 7.61–7.46 (m, 4H), 7.43 (dd, <i>J</i> = 8.3, 1.1 Hz, 1H), 7.39 (dt, <i>J</i> = 6.0, 2.0 Hz, 1H), 7.30–7.23 (m, 3H), 7.15 (ddd, <i>J</i> = 8.3, 7.0, 1.2 Hz, 1H), 7.05 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 6.81 (d, <i>J</i> = 8.5 Hz, 2H), 5.31 (dd, <i>J</i> = 12.0, 4.6 Hz, 1H), 3.75 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.78 (dd, <i>J</i> = 18.5, 4.6 Hz, 1H), 2.58–2.44 (m, 2H), 2.30 (t, <i>J</i> = 7.1 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.95, 169.08, 160.73, 153.27, 151.57, 141.65, 139.02, 135.65, 132.01, 131.77, 129.89, 129.00, 128.79, 128.69, 128.62, 127.88, 123.73, 122.74, 119.83, 116.01, 58.72, 45.66, 29.04, 28.70. <i>t</i><sub>R</sub> = 1.97 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>ClN<sub>3</sub>O<sub>4</sub>: calculated 499.1, found <i>m</i>/<i>z</i> 500.4, 502.4 [M + H]<sup>+</sup>; 498.4, 500.3 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> 4-(5-(4-Methoxyphenyl)-3-(2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-4-oxobutanoic Acid (<b>56d</b>)</h3><div class="NLM_p last">Compound <b>56d</b> was synthesized via general procedure C<sub>1</sub> using <b>113c</b> (120 mg, 0.30 mmol) with succinic anhydride <b>115</b> (60 mg, 0.60 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 5–35% EtOH/DCM) to afford <b>56d</b> (99 mg, yield 67%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21 (s, 1H), 12.00 (s, 1H), 7.61–7.34 (m, 6H), 7.29–7.23 (m, 1H), 7.14 (ddd, <i>J</i> = 8.2, 7.0, 1.2 Hz, 1H), 7.04 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 6.79–6.66 (m, 4H), 5.23 (dd, <i>J</i> = 11.9, 4.5 Hz, 1H), 3.72 (s, 4H), 2.79 (dd, <i>J</i> = 18.4, 4.5 Hz, 1H), 2.46 (dd, <i>J</i> = 6.8, 2.4 Hz, 2H), 2.27 (t, <i>J</i> = 6.9 Hz, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.99, 168.91, 160.78, 158.68, 153.22, 151.47, 139.00, 135.70, 134.85, 131.71, 129.84, 129.08, 128.66, 128.60, 127.87, 127.27, 123.91, 122.71, 119.88, 115.99, 114.16, 58.85, 55.51, 45.80, 29.11, 28.74. <i>t</i><sub>R</sub> = 1.83 min (generic method). ESI-MS for C<sub>29</sub>H<sub>25</sub>N<sub>3</sub>O<sub>5</sub>: calculated 495.1, found <i>m</i>/<i>z</i> 496.5 [M + H]<sup>+</sup>; 494.5 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> 4-Oxo-4-(3-(2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-(trifluoromethyl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)butanoic Acid (<b>57d</b>)</h3><div class="NLM_p last">Compound <b>57d</b> was synthesized via general procedure C<sub>1</sub> using <b>114c</b> (189 mg, 0.45 mmol) with succinic anhydride <b>115</b> (90 mg, 0.90 mmol) (120 °C, 200 W). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g, 2–50% EtOH/DCM) to afford <b>57d</b> (173 mg, yield 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.23 (s, 1H), 12.02 (s, 1H), 7.64–7.36 (m, 8H), 7.25 (d, <i>J</i> = 7.3 Hz, 1H), 7.15 (ddd, <i>J</i> = 8.3, 7.1, 1.2 Hz, 1H), 7.08–6.98 (m, 3H), 5.42 (dd, <i>J</i> = 12.1, 4.7 Hz, 1H), 3.79 (dd, <i>J</i> = 18.5, 12.1 Hz, 1H), 2.81 (dd, <i>J</i> = 18.5, 4.7 Hz, 1H), 2.59–2.48 (m, 2H), 2.36–2.26 (m, 2H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.94, 169.21, 160.71, 153.30, 151.65, 147.20, 139.03, 135.67, 131.80, 129.88, 128.96, 128.70, 128.65, 128.29, 127.88, 126.70, 126.02, 125.84, 125.80, 123.63, 123.31, 122.76, 119.81, 116.02, 58.95, 45.63, 28.99, 28.68. <i>t</i><sub>R</sub> = 2.06 min (generic method). ESI-MS for C<sub>29</sub>H<sub>22</sub>F<sub>3</sub>N<sub>3</sub>O<sub>4</sub>: calculated 533.2, found <i>m</i>/<i>z</i> 534.5 [M + H]<sup>+</sup>; 532.5 [M – H]<sup>−</sup>. UPLC–MS purity (UV at 215 nm) >99.5%.</div></div><div id="sec4_1_1_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> 3-Acetyl-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>60a</b>) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h3><div class="NLM_p last">2-Amino-5-chlorophenyl)phenylmethanone <b>58</b> (1.460 g, 6.3 mmol) and ethyl acetoacetate (2.4 mL, 18.9 mmol) were dissolved in DMF (1.00 M) in an appropriately sized screw-capped pressure tube. The mixture was heated at 153 °C and stirred for 19 h. The solvent was removed under reduced pressure to afford <b>60a (</b>1.875 g, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59–7.48 (m, 3H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 7.39–7.28 (m, 2H), 6.95 (d, <i>J</i> = 2.3 Hz, 1H), 2.22 (s, 3H). <i>t</i><sub>R</sub> = 2.06 min (generic method). ESI-MS for C<sub>17</sub>H<sub>12</sub>ClNO<sub>2</sub>: calculated 297.1, found <i>m</i>/<i>z</i> 298.3, 300.3 [M + H]<sup>+</sup>; 296.3, 298.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> 3-Acetyl-4-phenylquinolin-2(1<i>H</i>)-one (<b>61a</b>) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)</h3><div class="NLM_p last">(2-Aminophenyl)phenylmethanone <b>59</b> (2.00 g, 10.1 mmol) and ethyl acetoacetate (1.9 mL, 15.2 mmol) were dissolved in DMF (1.00 M) in an appropriately sized microwaveable vessel and microwaved at 120 °C (200 W) for 1.5 h. The solvent was concentrated under reduced pressure, and the residue was diluted with DCM and washed (3 × 100 mL) with H<sub>2</sub>O. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered and the solvent removed under reduced pressure. <b>61a</b> was obtained after precipitation from EtOAc (1.44 g, 53% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.22 (s, 1H), 7.56 (ddd, <i>J</i> = 8.4, 7.1, 1.5 Hz, 1H), 7.54–7.47 (m, 3H), 7.41 (dd, <i>J</i> = 8.4, 1.2 Hz, 1H), 7.34–7.28 (m, 2H), 7.14 (ddd, <i>J</i> = 8.3, 7.1, 1.2 Hz, 1H), 7.05 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 2.21 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 201.60, 159.26, 146.96, 138.45, 134.19, 133.29, 131.20, 128.71, 128.64, 128.46, 127.03, 122.33, 118.89, 115.64, 31.47. <i>t</i><sub>R</sub> = 1.86 min (generic method). ESI-MS for C<sub>17</sub>H<sub>13</sub>NO<sub>2</sub>: calculated 263.1, found <i>m</i>/<i>z</i> 264.5 [M + H]<sup>+</sup>; 262.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> (<i>E</i>)-6-Chloro-3-(3-(4-fluorophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>86b</b>)</h3><div class="NLM_p last">3-Acetyl-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one <b>60a</b> (2.517 g, 8.45 mmol) and 4-fluorobenzaldehyde <b>62</b> (906 μL, 8.45 mmol) were allowed to react overnight according to general procedure A. Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–20% EtOH/DCM) to afford <b>86b</b> (2.548 g, 75% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 7.81–7.70 (m, 2H), 7.66 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.54–7.41 (m, 5H), 7.36–7.30 (m, 2H), 7.28–7.19 (m, 2H), 6.98 (d, <i>J</i> = 2.4 Hz, 1H), 6.73 (d, <i>J</i> = 16.4 Hz, 1H). <i>t</i><sub>R</sub> = 2.42 min (generic method). ESI-MS for C<sub>24</sub>H<sub>15</sub>ClFNO<sub>2</sub>: calculated 403.1, found <i>m</i>/<i>z</i> 404.1, 406.1 [M + H]<sup>+</sup>; 402.1, 404.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> 6-Chloro-3-(5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>86c</b>)</h3><div class="NLM_p last">(<i>E</i>)-6-Chloro-3-(3-(4-fluorophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one <b>86b</b> (317 mg, 0.78 mmol) and hydrazine monohydrate (76 μL, 1.56 mmol) were allowed to react according to general procedure B. Crude compound was purified by flash column chromatography (SiO<sub>2</sub> gold 40 g; 0–20% EtOH/DCM) to afford <b>86c</b> (303 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.50 (m, 2H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.33 (dt, <i>J</i> = 6.9, 1.5 Hz, 1H), 7.23 (dt, <i>J</i> = 6.8, 2.0 Hz, 1H), 7.14 (d, <i>J</i> = 3.2 Hz, 1H), 7.13–7.02 (m, 4H), 6.89 (d, <i>J</i> = 2.3 Hz, 1H), 4.59 (td, <i>J</i> = 10.9, 10.2, 3.1 Hz, 1H), 3.23 (dd, <i>J</i> = 16.5, 11.0 Hz, 1H), 2.59–2.52 (m, 1H). <i>t</i><sub>R</sub> = 2.31 min (generic method). ESI-MS for C<sub>24</sub>H<sub>17</sub>ClFN<sub>3</sub>O: calculated 417.1, found <i>m</i>/<i>z</i> 418.4, 420.4 [M + H]<sup>+</sup>; 416.4, 418.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> (<i>E</i>)-6-Chloro-3-(3-(4-chlorophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>87b</b>)</h3><div class="NLM_p last">Compound <b>87b</b> was synthesized via general procedure A using <b>60a</b> (304 mg, 1.00 mmol) and 4-chlorobenzaldehyde <b>63</b> (140 mg, 1.00 mmol). Title compound <b>87b</b> was obtained by precipitation and filtration from the reaction crude (429 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.70 (d, <i>J</i> = 8.6 Hz, 2H), 7.63 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.55–7.37 (m, 7H), 7.32 (dd, <i>J</i> = 7.6, 1.9 Hz, 2H), 6.96 (d, <i>J</i> = 2.4 Hz, 1H), 6.78 (d, <i>J</i> = 16.4 Hz, 1H). <i>t</i><sub>R</sub> = 2.55 min (generic method). ESI-MS for C<sub>24</sub>H<sub>15</sub>Cl<sub>2</sub>NO<sub>2</sub>: calculated 419.0, found <i>m</i>/<i>z</i> 420.4, 422.4, 424.4 [M + H]<sup>+</sup>; 418.5, 420.4, 422.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> 6-Chloro-3-(5-(4-chlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>87c</b>)</h3><div class="NLM_p last">Compound <b>87c</b> was synthesized via general procedure B using <b>87b</b> (429 mg, 1.00 mmol) with hydrazine hydrate (97 μL, 2.00 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>87c</b> (301 mg, yield 68%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.58 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.50 (tt, <i>J</i> = 7.5, 2.9 Hz, 3H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.34 (dt, <i>J</i> = 7.0, 1.7 Hz, 1H), 7.30 (d, <i>J</i> = 8.4 Hz, 2H), 7.23 (dt, <i>J</i> = 7.3, 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 3.2 Hz, 1H), 7.09 (d, <i>J</i> = 8.4 Hz, 2H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.59 (td, <i>J</i> = 11.6, 9.3, 3.1 Hz, 1H), 3.25 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.58–2.53 (m, 1H). <i>t</i><sub>R</sub> = 2.45 min (generic method). ESI-MS for C<sub>24</sub>H<sub>17</sub>Cl<sub>2</sub>N<sub>3</sub>O: calculated 433.1, found <i>m</i>/<i>z</i> 434.4, 436.4, 438.5 [M + H]<sup>+</sup>; 432.4, 434.4, 436.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> (<i>E</i>)-3-(3-(4-Bromophenyl)acryloyl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>88b</b>)</h3><div class="NLM_p last">Compound <b>88b</b> was synthesized via general procedure A using <b>60a</b> (328 mg, 1.10 mmol) and 4-bromobenzaldehyde <b>64</b> (204 mg, 1.10 mmol). Title compound <b>88b</b> was obtained by precipitation and filtration from the reaction crude (511 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.67–7.47 (m, 6H), 7.46–7.35 (m, 4H), 7.30 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 6.93 (d, <i>J</i> = 2.3 Hz, 1H), 6.80 (d, <i>J</i> = 16.4 Hz, 1H). <i>t</i><sub>R</sub> = 2.59 min (generic method). ESI-MS for C<sub>24</sub>H<sub>15</sub>BrClNO<sub>2</sub>: calculated 463.0, found <i>m</i>/<i>z</i> 464.3, 466.2, 468.2 [M + H]<sup>+</sup>; 462.2, 464.2, 466.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> 3-(5-(4-Bromophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>88c</b>)</h3><div class="NLM_p last">Compound <b>88c</b> was synthesized via general procedure B using <b>88b</b> (510 mg, 1.10 mmol) with hydrazine hydrate (110 μL, 2.20 mmol). Purification was performed by direct phase flash chromatography (0–8% EtOH/DCM) to afford <b>88c</b> (349 mg, yield 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.58 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.56–7.45 (m, 3H), 7.42 (dd, <i>J</i> = 8.6, 5.5 Hz, 3H), 7.34 (dt, <i>J</i> = 7.2, 1.6 Hz, 1H), 7.23 (dt, <i>J</i> = 7.2, 2.0 Hz, 1H), 7.18 (d, <i>J</i> = 3.2 Hz, 1H), 7.07–6.99 (m, 2H), 6.89 (d, <i>J</i> = 2.3 Hz, 1H), 4.58 (td, <i>J</i> = 11.6, 9.3, 3.2 Hz, 1H), 3.25 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.57–2.53 (m, 1H). <i>t</i><sub>R</sub> = 2.50 min (generic method). ESI-MS for C<sub>24</sub>H<sub>17</sub>BrClN<sub>3</sub>O: calculated 477.0, found <i>m</i>/<i>z</i> 478.2, 480.2, 482.2 [M + H]<sup>+</sup>; 476.3, 478.2, 480.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_68" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> (<i>E</i>)-6-Chloro-3-(3-(4-methoxyphenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>89b</b>)</h3><div class="NLM_p last">3-Acetyl-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one <b>60a</b> (611 mg, 2.00 mmol) and <i>p</i>-anisaldehyde <b>65</b> (243 μL, 2.00 mmol) were allowed to react overnight according to general procedure A. Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–5% EtOH/DCM), affording the desired <b>89b</b> (867 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.31 (s, 1H), 7.68–7.58 (m, 3H), 7.49–7.38 (m, 5H), 7.34–7.29 (m, 2H), 6.99–6.89 (m, 3H), 6.60 (d, <i>J</i> = 16.3 Hz, 1H), 3.78 (s, 3H). <i>t</i><sub>R</sub> = 1.19 min (apolar method). ESI-MS for C<sub>25</sub>H<sub>18</sub>ClNO<sub>3</sub>: calculated 415.1, found <i>m</i>/<i>z</i> 416.4, 418.4 [M + H]<sup>+</sup>; 414.4, 416.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_69" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> 6-Chloro-3-(5-(4-methoxyphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>89c</b>)</h3><div class="NLM_p last">(<i>E</i>)-6-Chloro-3-(3-(4-methoxyphenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one <b>89b</b> (780 mg, 1.9 mmol) and hydrazine monohydrate (185 μL, 3.8 mmol) were allowed to react according to general procedure B. Crude compound was purified by flash column chromatography (SiO<sub>2</sub> gold 40 g; 0–10% EtOH/DCM) to afford <b>89c</b> (611 mg, yield 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.53–7.47 (m, 3H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.34 (dd, <i>J</i> = 10.3, 3.8 Hz, 1H), 7.27–7.19 (m, 1H), 7.02 (dd, <i>J</i> = 8.9, 5.9 Hz, 3H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 6.80 (d, <i>J</i> = 8.7 Hz, 2H), 4.52 (td, <i>J</i> = 10.4, 2.9 Hz, 1H), 3.73 (s, 3H), 3.18 (dd, <i>J</i> = 16.4, 10.9 Hz, 1H), 2.58–2.53 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.84, 160.17, 152.10, 149.77, 149.57, 137.28, 134.76, 131.10, 129.96, 129.28, 128.71, 128.42, 128.30, 128.28, 126.41, 126.04, 124.80, 120.72, 117.58, 112.23, 58.37, 45.22, 40.19, 21.49. <i>t</i><sub>R</sub> = 2.24 min (generic method). ESI-MS for C<sub>25</sub>H<sub>20</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 429.1, found <i>m</i>/<i>z</i> 430.4, 432.4 [M + H]<sup>+</sup>; 428.4, 430.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_70" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> (<i>E</i>)-3-(3-(4-(<i>tert</i>-Butyl)phenyl)acryloyl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>90b</b>)</h3><div class="NLM_p last">Compound <b>90b</b> was synthesized via general procedure A using <b>60a</b> (543 mg, 1.80 mmol) and 4-<i>tert</i>-butylbenzaldeyde <b>66</b> (301 μL, 1.80 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–10% EtOH/DCM) to afford <b>90b</b> (441 mg, yield 55%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.58 (d, <i>J</i> = 8.5 Hz, 2H), 7.50–7.37 (m, 7H), 7.32 (dd, <i>J</i> = 7.7, 1.9 Hz, 2H), 6.97 (d, <i>J</i> = 2.4 Hz, 1H), 6.69 (d, <i>J</i> = 16.4 Hz, 1H), 1.26 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.82, 159.40, 153.81, 146.61, 146.19, 137.50, 133.60, 132.56, 131.52, 130.89, 128.86, 128.80, 128.50, 128.43, 126.64, 126.05, 125.71, 125.56, 120.56, 117.68, 79.15, 34.62, 30.97, 30.82. Rt 2.18 min (generic method). ESI-MS for C28H24ClNO2: calculated 441.1, found <i>m</i>/<i>z</i> 442.4, 444.4 [M + H]+; 440.4, 442.4 [M – H]-.</div></div><div id="sec4_1_1_71" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> 3-[5-(4-<i>tert</i>-Butylphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl]-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>90c</b>)</h3><div class="NLM_p last">Compound <b>90c</b> was synthesized via general procedure B using <b>90b</b> (369 mg, 0.83 mmol) with hydrazine hydrate (81 μL, 1.66 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–90% EtOAc/CHX) to afford <b>90c</b> (351 mg, yield 93%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.56 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.49 (dtd, <i>J</i> = 9.6, 7.2, 5.3 Hz, 3H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.33 (dt, <i>J</i> = 7.2, 1.7 Hz, 1H), 7.25 (d, <i>J</i> = 8.3 Hz, 3H), 7.06 (d, <i>J</i> = 3.0 Hz, 1H), 7.02 (d, <i>J</i> = 8.4 Hz, 2H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.54 (t, <i>J</i> = 10.4 Hz, 1H), 3.19 (dd, <i>J</i> = 16.4, 10.9 Hz, 1H), 2.58 (dd, <i>J</i> = 16.4, 9.6 Hz, 1H), 1.26 (s, 9H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.54, 149.21, 148.30, 145.49, 140.31, 136.97, 135.21, 130.32, 129.19, 128.72, 128.26, 128.21, 128.14, 127.79, 127.18, 126.88, 126.33, 126.18, 125.75, 125.62, 124.92, 120.92, 117.33, 79.16, 62.69, 44.34, 34.09, 31.13, 31.05, 30.74. <i>t</i><sub>R</sub> = 1.82 min (generic method). ESI-MS for C<sub>28</sub>H<sub>26</sub>ClN<sub>3</sub>O: calculated 455.2, found <i>m</i>/<i>z</i> 456.5, 458.4, [M + H]<sup>+</sup>; 454.5 456.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_72" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> (<i>E</i>)-6-Chloro-3-(3-(3-fluoro-4-methoxyphenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>91b</b>)</h3><div class="NLM_p last">Compound <b>91b</b> was synthesized via general procedure A using <b>60a</b> (473 mg, 1.60 mmol) and 3-fluoro-4-methoxybenzaldehyde <b>67</b> (247 mg, 1.60 mmol). Title compound <b>91b</b> was obtained after precipitation and filtration from the reaction crude (547 mg, yield 79%). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.70–7.62 (m, 2H), 7.52–7.40 (m, 6H), 7.35–7.30 (m, 2H), 7.18 (t, <i>J</i> = 8.7 Hz, 1H), 6.97 (d, <i>J</i> = 2.4 Hz, 1H), 6.69 (d, <i>J</i> = 16.3 Hz, 1H), 3.88 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 194.19, 159.89, 153.13, 150.69, 149.82, 149.71, 147.06, 145.83, 138.00, 134.10, 133.01, 131.35, 129.34, 129.27, 128.90, 127.92, 127.85, 127.10, 126.87, 126.52, 126.02, 121.07, 118.17, 115.70, 115.52, 114.26, 56.62. <i>t</i><sub>R</sub> = 2.35 min (generic method). ESI-MS for C<sub>25</sub>H<sub>17</sub>ClFNO<sub>3</sub>: calculated 433.1, found <i>m</i>/<i>z</i> 434.4, 436.4 [M + H]<sup>+</sup>; 432.3, 434.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_73" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> 6-Chloro-3-(5-(3-fluoro-4-methoxyphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>91c</b>)</h3><div class="NLM_p last">Compound <b>91c</b> was synthesized via general procedure B using <b>91b</b> (203 mg, 0.50 mmol) with hydrazine hydrate (49 μL, 1.00 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>91c</b> (213 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.63–7.39 (m, 5H), 7.35–7.29 (m, 1H), 7.23 (dt, <i>J</i> = 7.0, 2.0 Hz, 1H), 7.12 (d, <i>J</i> = 3.1 Hz, 1H), 7.03 (t, <i>J</i> = 8.6 Hz, 1H), 6.97–6.85 (m, 3H), 4.55 (ddd, <i>J</i> = 11.5, 9.0, 2.9 Hz, 1H), 3.81 (s, 3H), 3.19 (dd, <i>J</i> = 16.5, 11.0 Hz, 1H), 2.60–2.53 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.01, 152.92, 150.50, 148.84, 146.52, 145.91, 137.45, 137.04, 135.64, 130.85, 129.61, 129.14, 128.70, 127.20, 126.24, 126.10, 123.00, 121.36, 117.82, 114.42, 114.23, 114.02, 62.31, 56.48, 44.83. <i>t</i><sub>R</sub> = 2.27 min (generic method). ESI-MS for C<sub>25</sub>H<sub>19</sub>ClFN<sub>3</sub>O<sub>2</sub>: calculated 447.1, found <i>m</i>/<i>z</i> 448.4, 450.3 [M + H]<sup>+</sup>; 446.3, 448.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_74" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> (<i>E</i>)-6-Chloro-4-phenyl-3-(3-(4-(trifluoromethyl)phenyl)acryloyl)quinolin-2(1<i>H</i>)-one (<b>92b</b>)</h3><div class="NLM_p last">Compound <b>92b</b> was synthesized via general procedure A using <b>60a</b> (760 mg, 2.50 mmol) and 4-(trifluoromethyl)benzaldehyde <b>68</b> (341 μL, 2.50 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–5% EtOH/DCM) to afford <b>92b</b> (820 mg, yield 71%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.38 (s, 1H), 7.89 (d, <i>J</i> = 8.2 Hz, 2H), 7.74 (d, <i>J</i> = 8.4 Hz, 2H), 7.67 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.62–7.57 (m, 1H), 7.53–7.40 (m, 4H), 7.36–7.31 (m, 2H), 6.99 (d, <i>J</i> = 2.3 Hz, 1H), 6.91 (d, <i>J</i> = 16.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.90, 159.39, 147.09, 144.03, 138.29, 137.58, 133.52, 132.23, 131.05, 130.34, 130.07, 130.02, 129.59, 129.19, 128.88, 128.48, 126.12, 125.68, 125.65, 125.62, 125.30, 122.59, 120.55, 117.74. <i>t</i><sub>R</sub> = 1.65 min (apolar method). ESI-MS for C<sub>25</sub>H<sub>15</sub>ClF<sub>3</sub>NO<sub>2</sub>: calculated 453.1, found <i>m</i>/<i>z</i> 454.4, 456.4 [M + H]<sup>+</sup>; 452.4, 454.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_75" class="NLM_sec NLM_sec_level_2"><div id="ac_i92" class="anchor-spacer"></div><h3 class="article-section__title" id="_i92"> 6-Chloro-4-phenyl-3-(5-(4-(trifluoromethyl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)quinolin-2(1<i>H</i>)-one (<b>92c</b>)</h3><div class="NLM_p last">Compound <b>92c</b> was synthesized via general procedure B using <b>92b</b> (752 mg, 1.70 mmol) with hydrazine hydrate (165 μL, 3.40 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–5% EtOH/DCM) to afford <b>92c</b> (696 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21 (s, 1H), 7.63–7.56 (m, 3H), 7.56–7.44 (m, 3H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.32 (m, 1H), 7.29 (d, <i>J</i> = 8.5 Hz, 3H), 7.25–7.20 (m, 1H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.75–4.64 (m, 1H), 3.39–3.25 (m, 1H), 2.63–2.54 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.49, 148.49, 148.37, 145.45, 136.99, 135.13, 130.43, 129.25, 128.63, 128.29, 128.23, 128.20, 127.68, 127.22, 126.58, 125.79, 125.64, 125.11, 125.08, 122.95, 120.88, 117.37, 62.16, 44.50. <i>t</i><sub>R</sub> = 2.52 min (generic method). ESI-MS for C<sub>25</sub>H<sub>17</sub>ClF<sub>3</sub>N<sub>3</sub>O: calculated 467.1, found <i>m</i>/<i>z</i> 468.4, 469.5 [M + H]<sup>+</sup>, 466.4, 468.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_76" class="NLM_sec NLM_sec_level_2"><div id="ac_i93" class="anchor-spacer"></div><h3 class="article-section__title" id="_i93"> (<i>E</i>)-6-Chloro-3-(3-(furan-2-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>93b</b>)</h3><div class="NLM_p last">Compound <b>93b</b> was synthesized via general procedure A using <b>60a</b> (503 mg, 1.70 mmol) and furan-2-carbaldehyde <b>69</b> (147 μL, 1.70 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>93b</b> (471 mg, yield 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.34 (s, 1H), 7.85 (d, <i>J</i> = 1.7 Hz, 1H), 7.64 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.51–7.40 (m, 4H), 7.35 (s, 1H), 7.33–7.26 (m, 2H), 6.96 (dd, <i>J</i> = 6.9, 2.9 Hz, 2H), 6.63 (dd, <i>J</i> = 3.5, 1.8 Hz, 1H), 6.37 (d, <i>J</i> = 16.1 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.04, 159.41, 150.27, 146.93, 146.60, 137.52, 133.55, 132.45, 132.30, 130.98, 128.84, 128.46, 126.08, 125.58, 123.96, 120.51, 117.69, 117.67, 113.14, 40.19, 39.98. <i>t</i><sub>R</sub> = 2.24 min (generic method). ESI-MS for C<sub>22</sub>H<sub>14</sub>ClNO<sub>3</sub>: calculated 375.1, found <i>m</i>/<i>z</i> 376.3, 378.3 [M + H]<sup>+</sup>; 374.3, 376.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_77" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> 6-Chloro-3-(5-(furan-2-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>93c</b>)</h3><div class="NLM_p last">Compound <b>93c</b> was synthesized via general procedure B using <b>93b</b> (442 mg, 1.20 mmol) with hydrazine hydrate (117 μL, 2.40 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–7.5% EtOH/DCM) to afford <b>93c</b> (422 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21 (s, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.54–7.51 (m, 1H), 7.50–7.38 (m, 4H), 7.31–7.23 (m, 2H), 7.07 (d, <i>J</i> = 2.8 Hz, 1H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 6.34 (dd, <i>J</i> = 3.1, 1.9 Hz, 1H), 6.17 (d, <i>J</i> = 3.2 Hz, 1H), 4.61–4.53 (m, 1H), 3.04 (dd, <i>J</i> = 16.2, 10.6 Hz, 1H), 2.87 (dd, <i>J</i> = 16.2, 9.2 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.55, 155.04, 148.48, 146.34, 142.11, 137.05, 135.23, 130.50, 128.95, 128.90, 128.34, 128.26, 128.18, 126.54, 125.83, 125.74, 120.91, 117.42, 110.33, 105.94, 40.63, 40.06. <i>t</i><sub>R</sub> = 2.14 min (generic method). ESI-MS for C<sub>22</sub>H<sub>16</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 389.1, found <i>m</i>/<i>z</i> 390.4, 392.3 [M + H]<sup>+</sup>; 388.4, 390.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_78" class="NLM_sec NLM_sec_level_2"><div id="ac_i95" class="anchor-spacer"></div><h3 class="article-section__title" id="_i95"> (<i>E</i>)-6-Chloro-3-(3-(4′-fluoro-[1,1′-biphenyl]-4-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>94b</b>)</h3><div class="NLM_p last">Compound <b>94b</b> was synthesized via general procedure A using <b>60a</b> (214 mg, 0.72 mmol) and 4′-fluoro-[1,1′-biphenyl]-4-carbaldehyde <b>70</b> (144 mg, 0.72 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–7% EtOH/CHCl<sub>3</sub>) to afford <b>94b</b> (305 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 7.82–7.62 (m, 7H), 7.57–7.40 (m, 5H), 7.36–7.26 (m, 4H), 6.98 (d, <i>J</i> = 2.3 Hz, 1H), 6.80 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.83, 163.40, 160.96, 159.45, 146.77, 145.64, 141.15, 137.55, 135.57, 135.54, 133.63, 133.34, 132.54, 131.52, 131.42, 130.93, 129.33, 128.90, 128.84, 128.80, 128.72, 128.46, 127.29, 126.98, 126.10, 125.60, 120.59, 117.71, 115.91, 115.70. <i>t</i><sub>R</sub> = 1.90 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>19</sub>ClFNO<sub>2</sub>: calculated 479.1, found <i>m</i>/<i>z</i> 480.2, 482.2 [M + H]<sup>+</sup>, 478.2, 480.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_79" class="NLM_sec NLM_sec_level_2"><div id="ac_i96" class="anchor-spacer"></div><h3 class="article-section__title" id="_i96"> 6-Chloro-3-(5-(4′-fluoro-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>94c</b>)</h3><div class="NLM_p last">Compound <b>94c</b> was synthesized via general procedure B using <b>94b</b> (273 mg, 0.57 mmol) with hydrazine hydrate (55 μL, 1.14 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>94c</b> (188 mg, yield 67%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (s, 1H), 7.71–7.64 (m, 2H), 7.60–7.46 (m, 6H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.32 (m, 1H), 7.31–7.21 (m, 3H), 7.19–7.13 (m, 3H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.67–4.57 (m, 1H), 3.26 (dd, <i>J</i> = 16.6, 11.1 Hz, 1H), 2.60 (dd, <i>J</i> = 16.4, 9.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.98, 160.55, 160.52, 148.35, 145.48, 142.66, 137.78, 136.96, 136.44, 136.41, 135.19, 130.33, 129.22, 128.69, 128.51, 128.43, 128.25, 128.17, 127.06, 126.78, 126.48, 125.75, 125.62, 120.91, 117.32, 115.73, 115.52, 62.49, 44.43. <i>t</i><sub>R</sub> = 1.72 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>21</sub>ClFN<sub>3</sub>O: calculated 493.1, found <i>m</i>/<i>z</i> 494.2, 496.1 [M + H]<sup>+</sup>, 492.2, 494.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_80" class="NLM_sec NLM_sec_level_2"><div id="ac_i97" class="anchor-spacer"></div><h3 class="article-section__title" id="_i97"> (<i>E</i>)-6-Chloro-3-(3-(4′-chloro-[1,1′-biphenyl]-4-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>95b</b>)</h3><div class="NLM_p last">Compound <b>95b</b> was synthesized via general procedure A using <b>60a</b> (339 mg, 1.10 mmol) and 4′-chloro-[1,1′-biphenyl]-4-carbaldehyde <b>71</b> (238 mg, 1.10 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>95b</b> (567 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 7.78–7.68 (m, 6H), 7.65 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.57–7.48 (m, 3H), 7.48–7.40 (m, 4H), 7.34 (dd, <i>J</i> = 7.8, 1.8 Hz, 2H), 6.99 (d, <i>J</i> = 2.3 Hz, 1H), 6.81 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.80, 159.41, 146.75, 145.51, 140.78, 137.87, 137.53, 133.70, 133.61, 132.90, 132.51, 130.92, 129.34, 128.93, 128.88, 128.82, 128.45, 127.42, 126.98, 126.07, 125.58, 120.57, 117.69, 79.15. <i>t</i><sub>R</sub> = 2.16 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>19</sub>Cl<sub>2</sub>NO<sub>2</sub>: calculated 495.1, found <i>m</i>/<i>z</i> 496.0, 497.9, 499.9, [M + H]<sup>+</sup>; 494.1, 496.0, 498.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_81" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> 6-Chloro-3-(5-(4′-chloro-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>95c</b>)</h3><div class="NLM_p last">Compound <b>95c</b> was synthesized via general procedure B using <b>95b</b> (558 mg, 1.10 mmol) with hydrazine hydrate (107 μL, 2.20 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–15% EtOH/DCM) to afford <b>95c</b> (501 mg, yield 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.69–7.63 (m, 2H), 7.59–7.46 (m, 8H), 7.41 (d, <i>J</i> = 8.7 Hz, 1H), 7.37–7.31 (m, 1H), 7.26–7.20 (m, 1H), 7.17 (d, <i>J</i> = 8.3 Hz, 2H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.63 (ddd, <i>J</i> = 11.3, 9.5, 3.0 Hz, 1H), 3.27 (dd, <i>J</i> = 16.4, 11.1 Hz, 1H), 2.60 (dd, <i>J</i> = 16.4, 9.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.53, 148.37, 145.49, 143.11, 138.74, 137.45, 136.97, 135.20, 132.16, 130.35, 129.23, 128.82, 128.69, 128.28, 128.18, 127.14, 126.77, 126.49, 125.77, 125.63, 120.91, 117.34, 62.49, 44.45. <i>t</i><sub>R</sub> = 1.99 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O: calculated 509.1, found <i>m</i>/<i>z</i> 510.0, 511.9, 514.0 [M + H]<sup>+</sup>; 508.0, 510.0, 511.9 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_82" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> (<i>E</i>)-3-(3-(4′-Bromo-[1,1′-biphenyl]-4-yl)acryloyl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>96b</b>)</h3><div class="NLM_p last">Compound <b>96b</b> was synthesized via general procedure A using <b>60a</b> (337 mg, 1.10 mmol) and 4′-bromo-[1,1′-biphenyl]-4-carbaldehyde <b>72</b> (287 mg, 1.10 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>96b</b> (576 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 7.80–7.68 (m, 4H), 7.68–7.62 (m, 5H), 7.57–7.39 (m, 5H), 7.34 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 6.99 (d, <i>J</i> = 2.3 Hz, 1H), 6.81 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.81, 159.41, 146.76, 145.52, 140.83, 138.24, 137.53, 133.74, 133.60, 132.51, 131.85, 130.92, 129.36, 128.88, 128.82, 128.77, 128.44, 127.43, 126.94, 126.07, 125.58, 121.53, 120.57, 117.69, 79.15. <i>t</i><sub>R</sub> = 2.20 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>19</sub>BrClNO<sub>2</sub>: calculated 539.0, found <i>m</i>/<i>z</i> 539.9, 541.9, 543.9 [M + H]<sup>+</sup>; 538.0, 540.0, 542.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_83" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> 3-(5-(4′-Bromo-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-6-chloro-4-phenylquinolin-2(1<i>H</i>)-one (<b>96c</b>)</h3><div class="NLM_p last">Compound <b>96c</b> was synthesized via general procedure B using <b>96b</b> (570 mg, 1.00 mmol) with hydrazine hydrate (97 μL, 2.00 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>96c</b> (472 mg, yield 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.68–7.46 (m, 10H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.33 (m, 1H), 7.26–7.21 (m, 1H), 7.19–7.15 (m, 3H), 6.90 (d, <i>J</i> = 2.4 Hz, 1H), 4.63 (ddd, <i>J</i> = 11.3, 9.5, 3.1 Hz, 1H), 3.27 (dd, <i>J</i> = 16.4, 11.0 Hz, 1H), 2.60 (dd, <i>J</i> = 16.4, 9.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.53, 148.37, 145.48, 143.17, 139.11, 137.48, 136.97, 135.20, 131.74, 130.36, 129.23, 128.69, 128.63, 128.27, 128.18, 127.16, 126.77, 126.45, 125.76, 125.63, 120.91, 120.72, 117.34, 79.15, 62.49, 44.45, 40.15, 39.99, 39.94, 39.73, 39.52, 39.31, 39.10, 38.89. <i>t</i><sub>R</sub> = 2.05 min (apolar method). ESI-MS for C<sub>30</sub>H<sub>21</sub>BrClN<sub>3</sub>O: calculated 553.0, found <i>m</i>/<i>z</i> 553.9, 555.9, 557.9 [M + H]<sup>+</sup>, 551.9, 554.0, 556.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_84" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> (<i>E</i>)-6-Chloro-3-(3-(4′-methoxy-[1,1′-biphenyl]-4-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>97b</b>)</h3><div class="NLM_p last">Compound <b>97b</b> was synthesized via general procedure A using <b>60a</b> (430 mg, 1.40 mmol) and 4′-methoxy-[1,1′-biphenyl]-4-carbaldehyde <b>73</b> (297 mg, 1.40 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>97b</b> (528 mg, yield 74%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 7.75–7.62 (m, 6H), 7.55–7.40 (m, 6H), 7.33 (ddt, <i>J</i> = 6.7, 3.2, 1.8 Hz, 2H), 7.05–7.00 (m, 2H), 6.98 (d, <i>J</i> = 2.4 Hz, 1H), 6.77 (d, <i>J</i> = 16.4 Hz, 1H), 3.79 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.75, 159.42, 159.38, 159.07, 146.68, 145.85, 141.92, 137.52, 133.63, 132.58, 131.32, 131.13, 130.90, 129.30, 128.88, 128.81, 128.69, 128.65, 128.43, 127.84, 126.86, 126.37, 126.22, 126.06, 125.69, 125.57, 120.58, 117.69, 114.43, 55.19, 31.34. <i>t</i><sub>R</sub> = 1.81 min (apolar method). ESI-MS for C<sub>31</sub>H<sub>22</sub>ClNO<sub>3</sub>: calculated 491.1, found <i>m</i>/<i>z</i> 492.0, 494.0 [M + H]<sup>+</sup>; 490.0, 492.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_85" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> 6-Chloro-3-(5-(4′-methoxy-[1,1′-biphenyl]-4-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>97c</b>)</h3><div class="NLM_p last">Compound <b>97c</b> was synthesized via general procedure B using <b>97b</b> (522 mg, 1.10 mmol) with hydrazine hydrate (107 μL, 2.20 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–10% EtOH/DCM) to afford <b>97c</b> (412 mg, yield 77%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.57 (dd, <i>J</i> = 8.9, 2.4 Hz, 3H), 7.54–7.44 (m, 5H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.31 (m, 1H), 7.24 (td, <i>J</i> = 4.2, 3.7, 1.8 Hz, 1H), 7.16–7.11 (m, 3H), 7.01 (d, <i>J</i> = 8.8 Hz, 2H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.60 (ddd, <i>J</i> = 11.0, 9.5, 3.1 Hz, 1H), 3.79 (s, 3H), 3.25 (dd, <i>J</i> = 16.4, 11.0 Hz, 1H), 2.60 (dd, <i>J</i> = 16.4, 9.5 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.54, 158.79, 148.34, 145.49, 141.88, 138.51, 136.97, 135.21, 132.34, 130.35, 129.24, 128.71, 128.27, 128.18, 127.60, 126.99, 126.82, 126.01, 125.75, 125.63, 120.93, 117.33, 114.32, 62.58, 55.14, 44.44. <i>t</i><sub>R</sub> = 2.60 min (generic method). ESI-MS for C<sub>31</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 505.2, found <i>m</i>/<i>z</i> 506.0, 508.0 [M + H]<sup>+</sup>; 504.1, 506.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_86" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> (<i>E</i>)-6-Chloro-3-(3-(1-methyl-1<i>H</i>-indol-5-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>98b</b>)</h3><div class="NLM_p last">Compound <b>98b</b> was synthesized via general procedure A using <b>60a</b> (545 mg, 1.80 mmol) and 1-methyl-1<i>H</i>-indole-5-carbaldehyde <b>74</b> (287 mg, 1.80 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>98b</b> (427 mg, yield 53%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.31 (s, 1H), 7.86 (d, <i>J</i> = 1.5 Hz, 1H), 7.65 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.56 (d, <i>J</i> = 16.3 Hz, 1H), 7.51–7.30 (m, 9H), 6.97 (d, <i>J</i> = 2.3 Hz, 1H), 6.65 (d, <i>J</i> = 16.3 Hz, 1H), 6.44 (dd, <i>J</i> = 3.2, 0.7 Hz, 1H), 3.79 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.46, 159.48, 148.78, 146.31, 137.77, 137.46, 133.77, 132.92, 131.03, 130.76, 128.89, 128.72, 128.37, 128.15, 125.99, 125.53, 125.36, 124.31, 123.22, 121.09, 120.65, 117.63, 110.33, 101.50, 32.61. <i>t</i><sub>R</sub> = 2.45 min (generic method). ESI-MS for C<sub>27</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>2</sub>: calculated 438.1, found <i>m</i>/<i>z</i> 439.2, 441.1 [M + H]<sup>+</sup>; 437.3, 439.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_87" class="NLM_sec NLM_sec_level_2"><div id="ac_i104" class="anchor-spacer"></div><h3 class="article-section__title" id="_i104"> 6-Chloro-3-(5-(1-methyl-1<i>H</i>-indol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>98c</b>)</h3><div class="NLM_p last">Compound <b>98c</b> was synthesized via general procedure B using <b>98b</b> (370 mg, 0.84 mmol) with hydrazine hydrate (82 μL, 1.68 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>98c</b> (325 mg, yield 85%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.57–7.46 (m, 4H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.35 (dt, <i>J</i> = 7.2, 1.7 Hz, 1H), 7.31–7.21 (m, 4H), 7.02 (s, 1H), 6.91 (td, <i>J</i> = 3.9, 1.7 Hz, 2H), 6.34 (dd, <i>J</i> = 3.0, 0.7 Hz, 1H), 4.66 (t, <i>J</i> = 10.5 Hz, 1H), 3.75 (s, 3H), 3.22 (dd, <i>J</i> = 16.4, 11.0 Hz, 1H), 2.63 (dd, <i>J</i> = 16.4, 9.9 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.60, 148.21, 145.51, 136.97, 135.70, 135.30, 133.87, 130.28, 129.72, 129.24, 128.74, 128.27, 128.17, 127.79, 127.03, 125.76, 125.62, 120.96, 120.01, 118.09, 117.32, 109.44, 100.21, 63.57, 44.92, 32.45. <i>t</i><sub>R</sub> = 2.38 min (generic method). ESI-MS for C<sub>27</sub>H<sub>21</sub>ClN<sub>4</sub>O: calculated 452.1, found <i>m</i>/<i>z</i> 453.2, 455.2 [M + H]<sup>+</sup>; 451.3, 453.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_88" class="NLM_sec NLM_sec_level_2"><div id="ac_i105" class="anchor-spacer"></div><h3 class="article-section__title" id="_i105"> (<i>E</i>)-6-Chloro-3-(3-(1-ethyl-1<i>H</i>-indol-5-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>99b</b>)</h3><div class="NLM_p last">Compound <b>99b</b> was synthesized via general procedure A using <b>60a</b> (402 mg, 1.30 mmol) and 1-ethyl-1<i>H</i>-indole-5-carbaldehyde <b>75</b> (225 mg, 1.30 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 40 g; 0–5% EtOH/DCM) to afford <b>99b</b> (395 mg, yield 65%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.31 (s, 1H), 7.85 (d, <i>J</i> = 1.4 Hz, 1H), 7.65 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.56 (d, <i>J</i> = 16.3 Hz, 1H), 7.50–7.45 (m, 3H), 7.45–7.38 (m, 4H), 7.33 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 6.97 (d, <i>J</i> = 2.3 Hz, 1H), 6.65 (d, <i>J</i> = 16.2 Hz, 1H), 6.45 (d, <i>J</i> = 3.1 Hz, 1H), 4.20 (q, <i>J</i> = 7.2 Hz, 2H), 1.34 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.47, 159.48, 148.79, 146.30, 137.46, 136.80, 133.77, 132.92, 130.76, 129.42, 128.98, 128.89, 128.72, 128.66, 128.37, 128.30, 125.99, 125.53, 125.35, 124.27, 123.35, 121.02, 120.65, 117.63, 110.32, 101.75, 79.15, 15.44. <i>t</i><sub>R</sub> = 2.56 min (generic method). ESI-MS for C<sub>28</sub>H<sub>21</sub>ClN<sub>2</sub>O<sub>2</sub>: calculated 452.1, found <i>m</i>/<i>z</i> 453.2, 455.2 [M + H]<sup>+</sup>; 451.3, 453.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_89" class="NLM_sec NLM_sec_level_2"><div id="ac_i106" class="anchor-spacer"></div><h3 class="article-section__title" id="_i106"> 6-Chloro-3-(5-(1-ethyl-1<i>H</i>-indol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>99c</b>)</h3><div class="NLM_p last">Compound <b>99c</b> was synthesized via general procedure B using <b>99b</b> (336 mg, 0.79 mmol) with hydrazine hydrate (77 μL, 1.58 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>99c</b> (322 mg, yield 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (s, 1H), 7.60–7.48 (m, 4H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.30 (m, 3H), 7.25 (dt, <i>J</i> = 6.6, 2.1 Hz, 2H), 7.00 (d, <i>J</i> = 3.2 Hz, 1H), 6.89 (dd, <i>J</i> = 9.7, 2.0 Hz, 2H), 6.34 (dd, <i>J</i> = 3.2, 0.8 Hz, 1H), 4.64 (td, <i>J</i> = 10.4, 3.2 Hz, 1H), 4.17 (q, <i>J</i> = 7.2 Hz, 2H), 3.21 (dd, <i>J</i> = 16.4, 11.0 Hz, 1H), 2.63 (dd, <i>J</i> = 16.4, 9.9 Hz, 1H), 1.33 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.59, 148.22, 145.47, 136.97, 135.29, 134.66, 133.82, 130.30, 129.23, 128.75, 128.27, 128.18, 128.12, 127.94, 127.04, 125.74, 125.63, 120.97, 119.94, 118.22, 117.33, 109.48, 100.42, 63.56, 56.02, 44.87, 18.54, 15.49. <i>t</i><sub>R</sub> = 2.51 min (generic method). ESI-MS for C<sub>28</sub>H<sub>23</sub>ClN<sub>4</sub>O: calculated 466.2, found <i>m</i>/<i>z</i> 467.2, 469.2 [M + H]<sup>+</sup>; 465.3, 467.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_90" class="NLM_sec NLM_sec_level_2"><div id="ac_i107" class="anchor-spacer"></div><h3 class="article-section__title" id="_i107"> (<i>E</i>)-6-Chloro-3-(3-(1-ethyl-1<i>H</i>-indazol-5-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>100b</b>)</h3><div class="NLM_p last">Compound <b>100b</b> was synthesized via general procedure A using <b>60a</b> (292 mg, 0.98 mmol) and 1-ethyl-1<i>H</i>-indazole-5-carbaldehyde <b>76</b> (171 mg, 0.98 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>100b</b> (275 mg, yield 61%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.09 (s, 1H), 8.07 (d, <i>J</i> = 1.4 Hz, 1H), 7.72 (dd, <i>J</i> = 8.9, 1.6 Hz, 1H), 7.69–7.57 (m, 3H), 7.48 (d, <i>J</i> = 8.8 Hz, 1H), 7.46–7.36 (m, 3H), 7.34 (dd, <i>J</i> = 7.8, 1.8 Hz, 2H), 6.98 (d, <i>J</i> = 2.3 Hz, 1H), 6.74 (d, <i>J</i> = 16.3 Hz, 1H), 4.43 (q, <i>J</i> = 7.2 Hz, 2H), 1.38 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.64, 159.45, 147.35, 146.51, 139.48, 137.50, 133.76, 133.70, 132.72, 130.84, 128.89, 128.76, 128.40, 126.97, 126.03, 125.64, 125.55, 125.18, 123.97, 123.80, 120.63, 117.66, 110.26, 43.20, 14.85. <i>t</i><sub>R</sub> = 2.35 min (generic method). ESI-MS for C<sub>27</sub>H<sub>20</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 453.1, found <i>m</i>/<i>z</i> 454.4, 456.4 [M + H]<sup>+</sup>, 452.4, 454.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_91" class="NLM_sec NLM_sec_level_2"><div id="ac_i108" class="anchor-spacer"></div><h3 class="article-section__title" id="_i108"> 6-Chloro-3-(5-(1-ethyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>100c</b>)</h3><div class="NLM_p last">Compound <b>100c</b> was synthesized via general procedure B using <b>100b</b> (260 mg, 0.57 mmol) with hydrazine hydrate (55 μL, 1.14 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>100c</b> (228 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21 (s, 1H), 7.97 (d, <i>J</i> = 0.9 Hz, 1H), 7.59–7.46 (m, 5H), 7.42 (d, <i>J</i> = 8.8 Hz, 2H), 7.38–7.34 (m, 1H), 7.27–7.21 (m, 1H), 7.17–7.08 (m, 2H), 6.91 (d, <i>J</i> = 2.4 Hz, 1H), 4.71 (ddd, <i>J</i> = 11.3, 9.3, 2.5 Hz, 1H), 4.41 (q, <i>J</i> = 7.2 Hz, 2H), 3.27 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.62 (dd, <i>J</i> = 16.5, 9.4 Hz, 1H), 1.38 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.06, 148.81, 145.93, 138.63, 137.46, 136.04, 135.71, 134.85, 132.74, 130.82, 129.75, 129.17, 128.76, 128.70, 127.36, 126.93, 126.25, 126.11, 125.62, 124.49, 123.84, 121.42, 118.50, 117.82, 110.00, 109.89, 63.43, 45.22, 43.52, 15.40. <i>t</i><sub>R</sub> = 2.19 min (generic method). ESI-MS for C<sub>27</sub>H<sub>22</sub>ClN<sub>5</sub>O: calculated 467.1, found <i>m</i>/<i>z</i> 468.5, 470.5 [M + H]<sup>+</sup>, 466.5, 468.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_92" class="NLM_sec NLM_sec_level_2"><div id="ac_i109" class="anchor-spacer"></div><h3 class="article-section__title" id="_i109"> (<i>E</i>)-6-Chloro-3-(3-(2-ethyl-2<i>H</i>-indazol-5-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>101b</b>)</h3><div class="NLM_p last">Compound <b>101b</b> was synthesized via general procedure A using <b>60a</b> (307 mg, 1.00 mmol) and 2-ethyl-2<i>H</i>-indazole-5-carbaldehyde <b>77</b> (174 mg, 1.00 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>101b</b> (386 mg, yield 82%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.48 (s, 1H), 8.02 (s, 1H), 7.65 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.61–7.51 (m, 3H), 7.51–7.38 (m, 4H), 7.34 (dd, <i>J</i> = 7.7, 1.9 Hz, 2H), 6.98 (d, <i>J</i> = 2.3 Hz, 1H), 6.68 (d, <i>J</i> = 16.3 Hz, 1H), 4.44 (q, <i>J</i> = 7.3 Hz, 2H), 1.49 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.61, 159.46, 148.47, 147.59, 146.48, 137.49, 133.71, 132.78, 130.83, 128.88, 128.76, 128.40, 127.38, 126.02, 125.55, 125.30, 125.16, 123.56, 121.42, 120.63, 117.65, 117.56, 47.87, 15.57. <i>t</i><sub>R</sub> = 2.22 min (generic method). ESI-MS for C<sub>27</sub>H<sub>20</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 453.1, found <i>m</i>/<i>z</i> 454.4, 456.4 [M + H]<sup>+</sup>, 452.4, 454.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_93" class="NLM_sec NLM_sec_level_2"><div id="ac_i110" class="anchor-spacer"></div><h3 class="article-section__title" id="_i110"> 6-Chloro-3-(5-(2-ethyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>101c</b>)</h3><div class="NLM_p last">Compound <b>101c</b> was synthesized via general procedure B using <b>101b</b> (302 mg, 0.60 mmol) with hydrazine hydrate (58 μL, 1.20 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>101c</b> (236 mg, yield 76%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 8.27 (d, <i>J</i> = 0.9 Hz, 1H), 7.59–7.38 (m, 6H), 7.35 (td, <i>J</i> = 3.1, 1.5 Hz, 2H), 7.28–7.20 (m, 1H), 7.08 (d, <i>J</i> = 3.2 Hz, 1H), 6.98–6.86 (m, 2H), 4.65 (td, <i>J</i> = 10.3, 3.1 Hz, 1H), 4.42 (q, <i>J</i> = 7.3 Hz, 2H), 3.24 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.60 (dd, <i>J</i> = 16.5, 9.6 Hz, 1H), 1.48 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.56, 148.29, 147.50, 145.53, 136.97, 135.54, 135.22, 130.32, 129.28, 128.68, 128.27, 128.20, 126.91, 125.76, 125.62, 124.73, 122.79, 121.02, 120.94, 117.33, 117.28, 117.07, 63.27, 47.64, 44.41, 15.77. <i>t</i><sub>R</sub> = 2.06 min (generic method). ESI-MS for C<sub>27</sub>H<sub>22</sub>ClN<sub>5</sub>O: calculated 467.1, found <i>m</i>/<i>z</i> 468.5, 470.4 [M + H]<sup>+</sup>, 466.4, 468.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_94" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> (<i>E</i>)-6-Chloro-4-phenyl-3-(3-(1-propyl-1<i>H</i>-indazol-5-yl)acryloyl)quinolin-2(1<i>H</i>)-one (<b>102b</b>)</h3><div class="NLM_p last">Compound <b>102b</b> was synthesized via general procedure A using <b>60a</b> (402 mg, 1.35 mmol) and 1-propyl-1<i>H</i>-indazole-5-carbaldehyde <b>78</b> (254 mg, 1.35 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–7% EtOH/DCM) to afford <b>102b</b> (401 mg, yield 64%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.10 (d, <i>J</i> = 0.8 Hz, 1H), 8.07 (d, <i>J</i> = 1.4 Hz, 1H), 7.72 (dd, <i>J</i> = 9.0, 1.6 Hz, 1H), 7.70–7.57 (m, 3H), 7.48 (d, <i>J</i> = 8.8 Hz, 1H), 7.44–7.38 (m, 3H), 7.34 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 6.98 (d, <i>J</i> = 2.4 Hz, 1H), 6.74 (d, <i>J</i> = 16.3 Hz, 1H), 4.36 (t, <i>J</i> = 6.9 Hz, 2H), 1.82 (h, <i>J</i> = 7.2 Hz, 2H), 0.80 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.66, 159.46, 147.36, 146.52, 140.12, 137.50, 133.80, 133.70, 132.74, 130.85, 128.89, 128.77, 128.41, 126.94, 126.03, 125.65, 125.55, 125.20, 123.98, 123.65, 120.64, 117.67, 110.36, 49.71, 22.82, 11.04. <i>t</i><sub>R</sub> = 2.51 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 467.1, found <i>m</i>/<i>z</i> 468.2, 470.2 [M + H]<sup>+</sup>, 466.3, 468.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_95" class="NLM_sec NLM_sec_level_2"><div id="ac_i112" class="anchor-spacer"></div><h3 class="article-section__title" id="_i112"> 6-Chloro-4-phenyl-3-(5-(1-propyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)quinolin-2(1<i>H</i>)-one (<b>102c</b>)</h3><div class="NLM_p last">Compound <b>102c</b> was synthesized via general procedure B using <b>102b</b> (395 mg, 0.84 mmol) with hydrazine hydrate (82 μL, 1.68 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>102c</b> (381 mg, yield 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 8.00–7.91 (m, 1H), 7.59–7.45 (m, 6H), 7.42 (d, <i>J</i> = 8.7 Hz, 2H), 7.38–7.31 (m, 1H), 7.24 (dd, <i>J</i> = 6.8, 1.9 Hz, 1H), 7.12 (dd, <i>J</i> = 8.7, 1.6 Hz, 1H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.70 (t, <i>J</i> = 10.3 Hz, 1H), 4.33 (t, <i>J</i> = 6.9 Hz, 2H), 3.25 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.62 (dd, <i>J</i> = 16.5, 9.5 Hz, 1H), 1.88–1.77 (m, 2H), 0.81 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.54, 148.29, 145.42, 138.73, 136.96, 135.44, 135.21, 132.22, 130.31, 129.23, 128.68, 128.25, 128.22, 128.18, 126.87, 125.74, 125.60, 125.13, 123.18, 120.92, 117.98, 117.32, 109.56, 62.94, 49.59, 44.66, 22.81, 11.09. <i>t</i><sub>R</sub> = 2.34 min (generic method). ESI-MS for C<sub>28</sub>H<sub>24</sub>ClN<sub>5</sub>O: calculated 481.2, found <i>m</i>/<i>z</i> 482.1, 484.2 [M + H]<sup>+</sup>, 480.2, 482.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_96" class="NLM_sec NLM_sec_level_2"><div id="ac_i113" class="anchor-spacer"></div><h3 class="article-section__title" id="_i113"> (<i>E</i>)-6-Chloro-4-phenyl-3-(3-(2-propyl-2<i>H</i>-indazol-5-yl)acryloyl)quinolin-2(1<i>H</i>)-one (<b>103b</b>)</h3><div class="NLM_p last">Compound <b>103b</b> was synthesized via general procedure A using <b>60a</b> (292 mg, 0.98 mmol) and 2-propyl-2<i>H</i>-indazole-5-carbaldehyde <b>79</b> (184 mg, 0.98 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–7% EtOH/DCM) to afford <b>103b</b> (372 mg, yield 81%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.47 (s, 1H), 8.04–8.00 (m, 1H), 7.65 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.60–7.51 (m, 3H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.44–7.38 (m, 3H), 7.36–7.32 (m, 2H), 6.97 (d, <i>J</i> = 2.3 Hz, 1H), 6.68 (d, <i>J</i> = 16.2 Hz, 1H), 4.36 (t, <i>J</i> = 6.9 Hz, 2H), 1.96–1.87 (m, 2H), 0.83 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.63, 159.46, 148.52, 147.59, 146.47, 137.50, 133.71, 132.79, 130.83, 128.88, 128.77, 128.41, 127.39, 126.02, 125.87, 125.55, 125.31, 123.56, 121.32, 120.63, 117.66, 117.59, 54.36, 23.22, 10.84. <i>t</i><sub>R</sub> = 2.37 min (generic method). ESI-MS for C<sub>28</sub>H<sub>22</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 467.1, found <i>m</i>/<i>z</i> 468.2, 470.2 [M + H]<sup>+</sup>, 466.3, 468.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_97" class="NLM_sec NLM_sec_level_2"><div id="ac_i114" class="anchor-spacer"></div><h3 class="article-section__title" id="_i114"> 6-Chloro-4-phenyl-3-(5-(2-propyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)quinolin-2(1<i>H</i>)-one (<b>103c</b>)</h3><div class="NLM_p last">Compound <b>103c</b> was synthesized via general procedure B using <b>103b</b> (342 mg, 0.73 mmol) with hydrazine hydrate (71 μL, 1.46 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>103c</b> (316 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 8.27 (d, <i>J</i> = 0.9 Hz, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.55–7.44 (m, 4H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.33 (m, 2H), 7.24 (dt, <i>J</i> = 6.6, 1.9 Hz, 1H), 7.08 (d, <i>J</i> = 3.2 Hz, 1H), 6.92 (dd, <i>J</i> = 8.9, 1.6 Hz, 1H), 6.90 (d, <i>J</i> = 2.4 Hz, 1H), 4.70–4.59 (m, 1H), 4.35 (t, <i>J</i> = 6.9 Hz, 2H), 3.24 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.60 (dd, <i>J</i> = 16.4, 9.6 Hz, 1H), 1.96–1.87 (m, 2H), 0.82 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.58, 148.31, 147.56, 145.50, 136.98, 135.55, 135.23, 130.37, 129.31, 128.70, 128.30, 128.28, 128.23, 126.93, 125.76, 125.64, 124.75, 123.59, 120.95, 120.89, 117.36, 117.31, 117.11, 63.26, 54.20, 44.40, 23.40, 10.89. <i>t</i><sub>R</sub> = 2.23 min (generic method). ESI-MS for C<sub>28</sub>H<sub>24</sub>ClN<sub>5</sub>O: calculated 481.2, found <i>m</i>/<i>z</i> 482.2, 484.2 [M + H]<sup>+</sup>, 480.3, 482.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_98" class="NLM_sec NLM_sec_level_2"><div id="ac_i115" class="anchor-spacer"></div><h3 class="article-section__title" id="_i115"> (<i>E</i>)-6-Chloro-3-(3-(1-cyclohexyl-1<i>H</i>-indazol-5-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>104b</b>)</h3><div class="NLM_p last">Compound <b>104b</b> was synthesized via general procedure A using <b>60a</b> (134 mg, 0.46 mmol) and 1-cyclohexyl-1<i>H</i>-indazole-5-carbaldehyde <b>80</b> (105 mg, 0.46 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>104b</b> (193 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.08 (s, 1H), 8.06 (d, <i>J</i> = 1.4 Hz, 1H), 7.72–7.69 (m, 2H), 7.65 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.60 (d, <i>J</i> = 16.3 Hz, 1H), 7.48 (d, <i>J</i> = 8.8 Hz, 1H), 7.45–7.37 (m, 3H), 7.33 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 6.98 (d, <i>J</i> = 2.4 Hz, 1H), 6.73 (d, <i>J</i> = 16.3 Hz, 1H), 4.60 (tt, <i>J</i> = 9.6, 4.8 Hz, 1H), 1.98–1.80 (m, 6H), 1.70 (d, <i>J</i> = 13.0 Hz, 1H), 1.48 (td, <i>J</i> = 12.4, 11.1, 4.6 Hz, 2H), 1.25 (dtd, <i>J</i> = 12.9, 9.7, 3.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.66, 159.47, 147.44, 146.53, 139.15, 137.51, 133.72, 133.57, 132.73, 130.86, 128.90, 128.79, 128.42, 127.00, 126.05, 125.59, 125.56, 125.00, 123.98, 123.66, 120.65, 117.68, 110.37, 56.77, 32.26, 24.98. <i>t</i><sub>R</sub> = 1.94 min (apolar method). ESI-MS for C<sub>31</sub>H<sub>26</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 507.2, found <i>m</i>/<i>z</i> 508.2, 510.2 [M + H]<sup>+</sup>, 506.3, 508.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_99" class="NLM_sec NLM_sec_level_2"><div id="ac_i116" class="anchor-spacer"></div><h3 class="article-section__title" id="_i116"> 6-Chloro-3-(5-(1-cyclohexyl-1<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>104c</b>)</h3><div class="NLM_p last">Compound <b>104c</b> was synthesized via general procedure B using <b>104b</b> (187 mg, 0.37 mmol) with hydrazine hydrate (36 μL, 0.74 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>104c</b> (172 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 7.95 (d, <i>J</i> = 0.8 Hz, 1H), 7.62–7.55 (m, 2H), 7.50 (dddd, <i>J</i> = 13.9, 7.1, 5.1, 4.0 Hz, 3H), 7.44–7.39 (m, 2H), 7.38–7.32 (m, 1H), 7.28–7.20 (m, 1H), 7.15–7.08 (m, 2H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.69 (ddd, <i>J</i> = 11.0, 9.5, 3.2 Hz, 1H), 4.55 (tt, <i>J</i> = 10.0, 5.3 Hz, 1H), 3.25 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.61 (dd, <i>J</i> = 16.5, 9.5 Hz, 1H), 1.88 (ddd, <i>J</i> = 20.4, 10.2, 3.5 Hz, 6H), 1.71 (dt, <i>J</i> = 13.0, 3.2 Hz, 1H), 1.51 (qd, <i>J</i> = 12.1, 11.7, 5.9 Hz, 2H), 1.26 (tdd, <i>J</i> = 12.8, 9.5, 4.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.56, 148.32, 145.45, 137.82, 136.98, 135.55, 135.24, 132.00, 130.35, 129.24, 128.70, 128.29, 128.26, 128.21, 126.89, 125.76, 125.63, 124.92, 123.21, 120.93, 118.01, 117.35, 109.60, 62.97, 56.62, 44.74, 32.30, 25.06. <i>t</i><sub>R</sub> = 2.65 min (generic method). ESI-MS for C<sub>31</sub>H<sub>28</sub>ClN<sub>5</sub>O: calculated 521.2, found <i>m</i>/<i>z</i> 522.3, 524.3 [M + H]<sup>+</sup>, 520.3, 522.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_100" class="NLM_sec NLM_sec_level_2"><div id="ac_i117" class="anchor-spacer"></div><h3 class="article-section__title" id="_i117"> (<i>E</i>)-6-Chloro-3-(3-(2-cyclohexyl-2<i>H</i>-indazol-5-yl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>105b</b>)</h3><div class="NLM_p last">Compound <b>105b</b> was synthesized via general procedure A using <b>60a</b> (89 mg, 0.30 mmol) and 2-cyclohexyl-2<i>H</i>-indazole-5-carbaldehyde <b>81</b> (68 mg, 0.30 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 12 g; 0–5% EtOH/DCM) to afford <b>105b</b> (136 mg, yield 90%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.49 (s, 1H), 8.00 (d, <i>J</i> = 1.5 Hz, 1H), 7.65 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.59–7.52 (m, 3H), 7.47 (d, <i>J</i> = 8.8 Hz, 1H), 7.45–7.38 (m, 3H), 7.35–7.30 (m, 2H), 6.97 (d, <i>J</i> = 2.3 Hz, 1H), 6.66 (d, <i>J</i> = 16.3 Hz, 1H), 4.45 (tt, <i>J</i> = 11.3, 3.8 Hz, 1H), 2.09 (dd, <i>J</i> = 13.0, 4.0 Hz, 2H), 1.86 (ddt, <i>J</i> = 14.4, 9.8, 5.6 Hz, 4H), 1.76–1.63 (m, 1H), 1.51–1.37 (m, 2H), 1.25 (qt, <i>J</i> = 13.1, 3.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.63, 159.47, 148.04, 147.67, 146.47, 137.49, 133.72, 132.78, 130.84, 128.89, 128.78, 128.41, 127.34, 126.03, 125.55, 125.42, 125.24, 123.78, 123.45, 121.13, 120.65, 117.70, 61.84, 33.12, 24.85, 24.82. <i>t</i><sub>R</sub> = 2.60 min (generic method). ESI-MS for C<sub>31</sub>H<sub>26</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 507.2, found <i>m</i>/<i>z</i> 508.3, 510.2 [M + H]<sup>+</sup>, 506.3, 508.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_101" class="NLM_sec NLM_sec_level_2"><div id="ac_i118" class="anchor-spacer"></div><h3 class="article-section__title" id="_i118"> 6-Chloro-3-(5-(2-cyclohexyl-2<i>H</i>-indazol-5-yl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>105c</b>)</h3><div class="NLM_p last">Compound <b>105c</b> was synthesized via general procedure B using <b>105b</b> (130 mg, 0.26 mmol) with hydrazine hydrate (25 μL, 0.52 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>105c</b> (117 mg, yield 86%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.19 (s, 1H), 8.28 (d, <i>J</i> = 0.9 Hz, 1H), 7.59–7.44 (m, 5H), 7.41 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.31 (m, 2H), 7.23 (dt, <i>J</i> = 6.5, 2.0 Hz, 1H), 7.07 (d, <i>J</i> = 3.2 Hz, 1H), 6.94–6.87 (m, 2H), 4.64 (td, <i>J</i> = 9.9, 3.1 Hz, 1H), 4.43 (tt, <i>J</i> = 11.3, 3.8 Hz, 1H), 3.23 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.59 (dd, <i>J</i> = 16.4, 9.6 Hz, 1H), 2.15–2.03 (m, 2H), 1.90–1.80 (m, 4H), 1.69 (dt, <i>J</i> = 12.5, 3.3 Hz, 1H), 1.44 (qt, <i>J</i> = 13.0, 3.5 Hz, 2H), 1.35–1.17 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.58, 148.32, 147.05, 145.53, 136.99, 135.52, 135.24, 130.37, 129.30, 128.70, 128.31, 128.29, 128.24, 126.93, 125.77, 125.65, 124.65, 121.38, 120.96, 120.70, 117.37, 117.34, 117.22, 63.29, 61.58, 44.46, 33.30, 24.93, 24.90. <i>t</i><sub>R</sub> = 2.47 min (generic method). ESI-MS for C<sub>31</sub>H<sub>28</sub>ClN<sub>5</sub>O: calculated 521.2, found <i>m</i>/<i>z</i> 522.2, 524.2 [M + H]<sup>+</sup>, 520.2, 522.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_102" class="NLM_sec NLM_sec_level_2"><div id="ac_i119" class="anchor-spacer"></div><h3 class="article-section__title" id="_i119"> (<i>E</i>)-6-Chloro-3-(3-(4-(1-ethyl-1<i>H</i>-indazol-5-yl)phenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>106b</b>)</h3><div class="NLM_p last">Compound <b>106b</b> was synthesized via general procedure A using <b>60a</b> (137 mg, 0.46 mmol) and 4-(1-ethyl-1<i>H</i>-indazol-5-yl)benzaldehyde <b>82</b> (115 mg, 0.46 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–2% EtOH/DCM) to afford <b>106b</b> (244 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 8.10 (d, <i>J</i> = 7.3 Hz, 2H), 7.74 (s, 5H), 7.65 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.59–7.39 (m, 6H), 7.34 (dd, <i>J</i> = 7.8, 1.8 Hz, 2H), 6.99 (d, <i>J</i> = 2.4 Hz, 1H), 6.80 (d, <i>J</i> = 16.3 Hz, 1H), 4.46 (q, <i>J</i> = 7.2 Hz, 2H), 1.40 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.80, 159.46, 146.73, 145.88, 142.84, 138.45, 137.54, 133.65, 133.16, 132.74, 132.59, 131.71, 130.93, 129.35, 128.91, 128.47, 127.09, 126.97, 126.09, 125.60, 125.35, 124.25, 120.61, 118.93, 117.71, 110.15, 43.17, 14.92. <i>t</i><sub>R</sub> = 2.64 min (generic method). ESI-MS for C<sub>33</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 529.2, found <i>m</i>/<i>z</i> 530.2, 532.2 [M + H]<sup>+</sup>, 528.3, 530.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_103" class="NLM_sec NLM_sec_level_2"><div id="ac_i120" class="anchor-spacer"></div><h3 class="article-section__title" id="_i120"> 6-Chloro-3-(5-(4-(1-ethyl-1<i>H</i>-indazol-5-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>106c</b>)</h3><div class="NLM_p last">Compound <b>106c</b> was synthesized via general procedure B using <b>106b</b> (233 mg, 0.44 mmol) with hydrazine hydrate (43 μL, 0.88 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–8% EtOH/DCM) to afford <b>106c</b> (181 mg, yield 75%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.20 (s, 1H), 8.10 (d, <i>J</i> = 0.9 Hz, 1H), 7.98 (t, <i>J</i> = 1.2 Hz, 1H), 7.74 (d, <i>J</i> = 8.8 Hz, 1H), 7.67 (dd, <i>J</i> = 8.8, 1.7 Hz, 1H), 7.60–7.47 (m, 6H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.36 (dt, <i>J</i> = 6.4, 1.8 Hz, 1H), 7.28–7.24 (m, 1H), 7.20–7.14 (m, 3H), 6.90 (d, <i>J</i> = 2.4 Hz, 1H), 4.63 (td, <i>J</i> = 10.9, 10.3, 3.2 Hz, 1H), 4.46 (q, <i>J</i> = 7.2 Hz, 2H), 3.29–3.20 (m, 1H), 2.62 (dd, <i>J</i> = 16.4, 9.4 Hz, 1H), 1.41 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.55, 148.36, 145.50, 142.03, 139.35, 138.19, 136.98, 135.22, 132.90, 132.66, 130.37, 129.25, 128.73, 128.29, 128.20, 127.06, 126.83, 126.69, 125.76, 125.64, 125.48, 124.24, 120.94, 118.34, 117.35, 110.00, 62.58, 44.46, 43.14, 14.93. <i>t</i><sub>R</sub> = 1.49 min (apolar method). ESI-MS for C<sub>33</sub>H<sub>26</sub>ClN<sub>5</sub>O: calculated 543.2, found <i>m</i>/<i>z</i> 544.1, 546.1 [M + H]<sup>+</sup>, 542.2, 544.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_104" class="NLM_sec NLM_sec_level_2"><div id="ac_i121" class="anchor-spacer"></div><h3 class="article-section__title" id="_i121"> (<i>E</i>)-6-Chloro-3-(3-(4-(2-ethyl-2<i>H</i>-indazol-5-yl)phenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>107b</b>)</h3><div class="NLM_p last">Compound <b>107b</b> was synthesized via general procedure A using <b>60a</b> (92 mg, 0.31 mmol) and 4-(2-ethyl-2<i>H</i>-indazol-5-yl)benzaldehyde <b>83</b> (78 mg, 0.31 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 12 g; 0–10% EtOH/DCM) to afford <b>107b</b> (150 mg, yield 91%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 1H), 8.44 (d, <i>J</i> = 0.8 Hz, 1H), 8.04 (t, <i>J</i> = 1.3 Hz, 1H), 7.73 (s, 4H), 7.70–7.63 (m, 2H), 7.59 (dd, <i>J</i> = 9.1, 1.8 Hz, 1H), 7.57–7.42 (m, 5H), 7.34 (dd, <i>J</i> = 7.8, 1.8 Hz, 2H), 6.99 (d, <i>J</i> = 2.3 Hz, 1H), 6.79 (d, <i>J</i> = 16.4 Hz, 1H), 4.47 (q, <i>J</i> = 7.3 Hz, 2H), 1.52 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.79, 159.45, 147.56, 146.70, 145.88, 143.10, 137.54, 133.65, 132.67, 132.59, 131.84, 130.91, 129.31, 128.90, 128.83, 128.45, 126.92, 126.07, 125.58, 124.89, 124.04, 121.88, 120.60, 118.56, 117.70, 117.56, 47.83, 15.72. <i>t</i><sub>R</sub> = 2.48 min (generic method). ESI-MS for C<sub>33</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 529.2, found <i>m</i>/<i>z</i> 530.1, 532.1 [M + H]<sup>+</sup>, 528.2, 530.2 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_105" class="NLM_sec NLM_sec_level_2"><div id="ac_i122" class="anchor-spacer"></div><h3 class="article-section__title" id="_i122"> 6-Chloro-3-(5-(4-(2-ethyl-2<i>H</i>-indazol-5-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>107c</b>)</h3><div class="NLM_p last">Compound <b>107c</b> was synthesized via general procedure B using <b>107b</b> (141 mg, 0.27 mmol) with hydrazine hydrate (26 μL, 0.54 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–15% EtOH/DCM) to afford <b>107c</b> (123 mg, yield 83%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.41 (d, <i>J</i> = 0.9 Hz, 1H), 7.92–7.88 (m, 1H), 7.67 (d, <i>J</i> = 9.0 Hz, 1H), 7.59–7.48 (m, 7H), 7.42 (d, <i>J</i> = 8.8 Hz, 1H), 7.37–7.33 (m, 1H), 7.25 (dq, <i>J</i> = 4.9, 1.9 Hz, 1H), 7.16 (dd, <i>J</i> = 9.3, 2.7 Hz, 3H), 6.90 (d, <i>J</i> = 2.4 Hz, 1H), 4.62 (ddd, <i>J</i> = 10.9, 9.4, 3.1 Hz, 1H), 4.47 (q, <i>J</i> = 7.3 Hz, 2H), 3.27 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.62 (dd, <i>J</i> = 16.4, 9.5 Hz, 1H), 1.52 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.67, 148.34, 147.41, 145.61, 141.99, 139.64, 137.15, 135.26, 132.81, 130.34, 129.28, 128.75, 128.31, 128.21, 127.04, 126.81, 126.55, 125.72, 125.64, 125.23, 123.65, 121.93, 120.96, 117.77, 117.44, 62.61, 47.80, 44.49, 15.76. <i>t</i><sub>R</sub> = 2.35 min (generic method). ESI-MS for C<sub>33</sub>H<sub>26</sub>ClN<sub>5</sub>O: calculated 543.2, found <i>m</i>/<i>z</i> 544.2, 546.2 [M + H]<sup>+</sup>, 542.3, 544.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_106" class="NLM_sec NLM_sec_level_2"><div id="ac_i123" class="anchor-spacer"></div><h3 class="article-section__title" id="_i123"> (<i>E</i>)-6-Chloro-4-phenyl-3-(3-(4-(1-propyl-1<i>H</i>-pyrazol-4-yl)phenyl)acryloyl)quinolin-2(1<i>H</i>)-one (<b>108b</b>)</h3><div class="NLM_p last">Compound <b>108b</b> was synthesized via general procedure A using <b>60a</b> (324 mg, 1.10 mmol) and 4-(1-propyl-1<i>H</i>-pyrazol-4-yl)benzaldehyde <b>84</b> (236 mg, 1.10 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–8% EtOH/DCM) to afford <b>108b</b> (387 mg, yield 72%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.33 (s, 1H), 8.26 (d, <i>J</i> = 0.8 Hz, 1H), 7.94 (d, <i>J</i> = 0.8 Hz, 1H), 7.68–7.55 (m, 5H), 7.50–7.38 (m, 5H), 7.33 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 6.98 (d, <i>J</i> = 2.3 Hz, 1H), 6.73 (d, <i>J</i> = 16.4 Hz, 1H), 4.06 (t, <i>J</i> = 7.0 Hz, 2H), 1.96–1.87 (m, 2H), 0.84 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 193.72, 159.43, 146.62, 146.14, 137.50, 136.32, 135.30, 133.65, 132.61, 131.69, 130.87, 129.40, 128.88, 128.79, 128.42, 127.62, 126.26, 126.05, 125.56, 125.07, 120.85, 120.59, 117.67, 53.07, 23.07, 10.90. <i>t</i><sub>R</sub> = 2.46 min (generic method). ESI-MS for C<sub>30</sub>H<sub>24</sub>ClN<sub>3</sub>O<sub>2</sub>: calculated 493.2, found <i>m</i>/<i>z</i> 494.0, 496.0 [M + H]<sup>+</sup>, 492.0, 494.0 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_107" class="NLM_sec NLM_sec_level_2"><div id="ac_i124" class="anchor-spacer"></div><h3 class="article-section__title" id="_i124"> 6-Chloro-4-phenyl-3-(5-(4-(1-propyl-1<i>H</i>-pyrazol-4-yl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)quinolin-2(1<i>H</i>)-one (<b>108c</b>)</h3><div class="NLM_p last">Compound <b>108c</b> was synthesized via general procedure B using <b>108b</b> (358 mg, 0.72 mmol) with hydrazine hydrate (70 μL, 1.44 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–10% EtOH/DCM) to afford <b>108c</b> (326 mg, yield 89%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (s, 1H), 8.13 (d, <i>J</i> = 0.8 Hz, 1H), 7.83 (d, <i>J</i> = 0.8 Hz, 1H), 7.57 (dd, <i>J</i> = 8.8, 2.4 Hz, 1H), 7.51 (tdd, <i>J</i> = 6.6, 3.8, 1.8 Hz, 3H), 7.42 (dd, <i>J</i> = 8.5, 6.2 Hz, 3H), 7.34 (dt, <i>J</i> = 6.8, 1.7 Hz, 1H), 7.26–7.22 (m, 1H), 7.09 (d, <i>J</i> = 3.2 Hz, 1H), 7.08–7.03 (m, 2H), 6.90 (d, <i>J</i> = 2.3 Hz, 1H), 4.61–4.49 (m, 1H), 4.06 (t, <i>J</i> = 7.0 Hz, 2H), 3.22 (dd, <i>J</i> = 16.4, 11.0 Hz, 1H), 2.56 (dd, <i>J</i> = 16.4, 9.6 Hz, 1H), 1.96–1.87 (m, 2H), 0.85 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.53, 148.31, 145.48, 140.99, 136.97, 135.81, 135.21, 131.39, 130.34, 129.22, 128.71, 128.27, 128.19, 126.92, 126.85, 126.79, 125.75, 125.62, 124.76, 121.34, 120.92, 117.33, 62.66, 52.98, 44.43, 23.16, 10.94. <i>t</i><sub>R</sub> = 2.34 min (generic method). ESI-MS for C<sub>30</sub>H<sub>26</sub>ClN<sub>5</sub>O: calculated 507.2, found <i>m</i>/<i>z</i> 508.0, 510.1 [M + H]<sup>+</sup>, 506.1, 508.1 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_108" class="NLM_sec NLM_sec_level_2"><div id="ac_i125" class="anchor-spacer"></div><h3 class="article-section__title" id="_i125"> (<i>E</i>)-3-(3-(4-Fluorophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>109b</b>)</h3><div class="NLM_p last">Compound <b>109b</b> was synthesized via general procedure A using <b>61a</b> (265 mg, 1.00 mmol) and 4-fluorobenzaldehyde <b>62</b> (107 μL, 1.00 mmol). Title compound <b>109b</b> was obtained after precipitation and filtration from the reaction crude (345 mg, yield 93%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.30 (s, 1H), 7.73 (d, <i>J</i> = 7.2 Hz, 2H), 7.65–7.00 (m, 12H), 6.73 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 194.31, 159.77, 147.76, 144.37, 134.36, 131.36, 130.92, 130.92, 128.92, 128.46, 128.23, 127.52, 126.86, 126.86, 124.54, 122.05, 119.21, 116.00, 115.92, 115.90, 115.88, 115.79. <i>t</i><sub>R</sub> = 2.25 min (generic method). ESI-MS for C<sub>24</sub>H<sub>16</sub>FNO<sub>2</sub>: calculated 369.1, found <i>m</i>/<i>z</i> 370.5 [M + H]<sup>+</sup>; 368.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_109" class="NLM_sec NLM_sec_level_2"><div id="ac_i126" class="anchor-spacer"></div><h3 class="article-section__title" id="_i126"> 3-(5-(4-Fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>109c</b>)</h3><div class="NLM_p last">Compound <b>109c</b> was synthesized via general procedure B using <b>109b</b> (300 mg, 0.81 mmol) with hydrazine hydrate (79 μL, 1.62 mmol). Title compound <b>109c</b> was obtained after precipitation from DCM (292 mg, yield 94%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.11 (s, 1H), 7.54–7.44 (m, 4H), 7.40 (dd, <i>J</i> = 8.3, 1.2 Hz, 1H), 7.32 (dt, <i>J</i> = 7.4, 1.7 Hz, 1H), 7.21 (dt, <i>J</i> = 6.8, 1.9 Hz, 1H), 7.14–7.02 (m, 6H), 6.99 (dd, <i>J</i> = 8.1, 1.4 Hz, 1H), 4.58 (td, <i>J</i> = 10.9, 3.1 Hz, 1H), 3.23 (dd, <i>J</i> = 16.4, 11.0 Hz, 1H), 2.56–2.52 (m, 1H). <i>t</i><sub>R</sub> = 2.13 min (generic method). ESI-MS for C<sub>24</sub>H<sub>18</sub>FN<sub>3</sub>O: calculated 383.1, found <i>m</i>/<i>z</i> 384.5 [M + H]<sup>+</sup>, 382.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_110" class="NLM_sec NLM_sec_level_2"><div id="ac_i127" class="anchor-spacer"></div><h3 class="article-section__title" id="_i127"> (<i>E</i>)-3-(3-(4-Bromophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>110b</b>)</h3><div class="NLM_p last">Compound <b>110b</b> was synthesized via general procedure A using <b>61a</b> (265 mg, 1.00 mmol) and 4-bromobenzaldehyde <b>64</b> (185 mg, 1.00 mmol). Title compound <b>110b</b> was obtained after precipitation and filtration from the reaction crude (396 mg, yield 92%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.67–7.50 (m, 5H), 7.45–7.36 (m, 5H), 7.29 (dd, <i>J</i> = 7.7, 1.8 Hz, 2H), 7.15–7.02 (m, 2H), 6.79 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 210.70, 133.70, 131.89, 131.23, 130.49, 129.00, 128.43, 128.25, 126.84, 124.03, 121.73, 119.29, 116.57, 99.54. <i>t</i><sub>R</sub> = 2.43 min (generic method). ESI-MS for C<sub>24</sub>H<sub>16</sub>BrNO<sub>2</sub>: calculated 429.0, found <i>m</i>/<i>z</i> 430.4, 432.4 [M + H]<sup>+</sup>; 428.4, 430.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_111" class="NLM_sec NLM_sec_level_2"><div id="ac_i128" class="anchor-spacer"></div><h3 class="article-section__title" id="_i128"> 3-(5-(4-Bromophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>110c</b>)</h3><div class="NLM_p last">Compound <b>110c</b> was synthesized via general procedure B using <b>110b</b> (300 mg, 0.70 mmol) with hydrazine hydrate (68 μL, 1.40 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0.2–3% EtOH/DCM) to afford <b>110c</b> (213 mg, yield 69%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.04 (s, 1H), 7.58–7.36 (m, 7H), 7.31 (dt, <i>J</i> = 7.3, 1.6 Hz, 1H), 7.20 (dt, <i>J</i> = 7.3, 2.0 Hz, 1H), 7.13–6.95 (m, 5H), 4.56 (ddd, <i>J</i> = 11.0, 9.3, 3.3 Hz, 1H), 3.30–3.14 (m, 1H), 2.57–2.51 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.71, 149.63, 145.96, 143.13, 138.24, 135.77, 131.05, 130.51, 129.28, 128.73, 128.69, 128.10, 128.06, 127.90, 126.94, 125.39, 121.92, 119.82, 119.57, 115.31, 62.01, 44.62. <i>t</i><sub>R</sub> = 2.31 min (generic method). ESI-MS for C<sub>24</sub>H<sub>18</sub>BrN<sub>3</sub>O: calculated 443.1, found <i>m</i>/<i>z</i> 444.4, 446.5 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_1_112" class="NLM_sec NLM_sec_level_2"><div id="ac_i129" class="anchor-spacer"></div><h3 class="article-section__title" id="_i129"> (<i>E</i>)-3-(3-(3-Bromophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>111b</b>)</h3><div class="NLM_p last">Compound <b>111b</b> was synthesized via general procedure A using <b>61a</b> (265 mg, 1.00 mmol) and 3-bromobenzaldehyde <b>85</b> (117 μL, 1.00 mmol). Title compound <b>111b</b> was obtained after precipitation and filtration from the reaction crude (380 mg, yield 88%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (s, 1H), 7.93 (s, 1H), 7.69 (d, <i>J</i> = 7.8 Hz, 1H), 7.59 (ddt, <i>J</i> = 7.2, 3.9, 1.6 Hz, 2H), 7.51–7.37 (m, 5H), 7.37–7.27 (m, 3H), 7.16 (td, <i>J</i> = 7.6, 7.1, 1.2 Hz, 1H), 7.08 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 6.84 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 194.29, 159.58, 148.05, 143.93, 138.80, 136.83, 134.24, 133.07, 131.37, 131.26, 131.06, 130.87, 128.92, 128.85, 128.52, 128.26, 127.14, 126.93, 122.24, 122.19, 119.23, 115.66. <i>t</i><sub>R</sub> = 2.40 min (generic method). ESI-MS for C<sub>24</sub>H<sub>16</sub>BrNO<sub>2</sub>: calculated 429.0, found <i>m</i>/<i>z</i> 430.4, 432.4 [M + H]<sup>+</sup>; 428.3, 430.3 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_113" class="NLM_sec NLM_sec_level_2"><div id="ac_i130" class="anchor-spacer"></div><h3 class="article-section__title" id="_i130"> 3-(5-(3-Bromophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>111c</b>)</h3><div class="NLM_p last">Compound <b>111c</b> was synthesized via general procedure B using <b>111b</b> (300 mg, 0.70 mmol) with hydrazine hydrate (68 μL, 1.40 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 1–5% EtOH/DCM) to afford <b>111c</b> (256 mg, yield 84%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.06 (s, 1H), 7.56–7.36 (m, 7H), 7.36–7.29 (m, 1H), 7.26–7.18 (m, 2H), 7.18–7.06 (m, 3H), 7.01 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 4.59 (ddd, <i>J</i> = 11.1, 9.3, 3.4 Hz, 1H), 3.24 (dd, <i>J</i> = 16.6, 11.2 Hz, 1H), 2.56 (dd, <i>J</i> = 16.6, 9.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.67, 149.59, 146.52, 145.91, 138.21, 135.73, 130.47, 130.39, 129.66, 129.25, 129.10, 128.64, 128.02, 128.00, 127.87, 126.89, 125.61, 125.30, 121.87, 121.52, 119.47, 115.27, 61.94, 44.57. <i>t</i><sub>R</sub> = 2.30 min (generic method). ESI-MS for C<sub>24</sub>H<sub>18</sub>BrN<sub>3</sub>O: calculated 443.0, found <i>m</i>/<i>z</i> 444.5, 446.4 [M + H]<sup>+</sup>; 442.6, 444.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_114" class="NLM_sec NLM_sec_level_2"><div id="ac_i131" class="anchor-spacer"></div><h3 class="article-section__title" id="_i131"> (<i>E</i>)-3-(3-(4-Chlorophenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>112b</b>)</h3><div class="NLM_p last">Compound <b>112b</b> was synthesized via general procedure A using <b>61a</b> (169 mg, 0.64 mmol) and 4-chlorobenzaldehyde <b>63</b> (90 mg, 0.64 mmol). Title compound <b>112b</b> was obtained after precipitation and filtration from the reaction crude (246 mg, quantitative yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.18 (s, 1H), 7.71 (d, <i>J</i> = 8.3 Hz, 2H), 7.59 (td, <i>J</i> = 7.6, 7.0, 1.5 Hz, 1H), 7.52–7.37 (m, 7H), 7.34–7.29 (m, 2H), 7.17 (t, <i>J</i> = 7.6 Hz, 1H), 7.12–7.05 (m, 1H), 6.79 (d, <i>J</i> = 16.4 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 194.66, 160.05, 148.50, 144.66, 139.25, 135.67, 134.74, 133.75, 131.78, 131.55, 130.77, 129.39, 129.00, 128.73, 128.65, 127.42, 122.69, 119.69, 116.13. <i>t</i><sub>R</sub> = 2.37 min (generic method). ESI-MS for C<sub>24</sub>H<sub>16</sub>ClNO<sub>2</sub>: calculated 385.1, found <i>m</i>/<i>z</i> 386.4, 388.4 [M + H]<sup>+</sup>; 384.4, 386.4 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_115" class="NLM_sec NLM_sec_level_2"><div id="ac_i132" class="anchor-spacer"></div><h3 class="article-section__title" id="_i132"> 3-(5-(4-Chlorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>112c</b>)</h3><div class="NLM_p last">Compound <b>112c</b> was synthesized via general procedure B using <b>112b</b> (225 mg, 0.58 mmol) with hydrazine hydrate (56 μL, 1.16 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 1–10% EtOH/DCM) to afford <b>112c</b> (185 mg, yield 79%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.05 (s, 1H), 7.56–7.43 (m, 4H), 7.40 (dd, <i>J</i> = 8.3, 1.2 Hz, 1H), 7.35–7.26 (m, 3H), 7.21 (dt, <i>J</i> = 7.3, 2.0 Hz, 1H), 7.13–7.06 (m, 4H), 7.00 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 4.58 (ddd, <i>J</i> = 10.9, 9.2, 3.2 Hz, 1H), 3.25 (dd, <i>J</i> = 16.5, 11.1 Hz, 1H), 2.58–2.52 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.18, 150.09, 146.43, 143.16, 138.71, 136.24, 131.79, 130.97, 129.76, 129.16, 128.82, 128.60, 128.56, 128.53, 128.36, 127.41, 125.88, 122.39, 120.04, 115.78, 62.43, 45.12. <i>t</i><sub>R</sub> = 2.26 min (generic method). ESI-MS for C<sub>24</sub>H<sub>18</sub>ClN<sub>3</sub>O: calculated 399.1, found <i>m</i>/<i>z</i> 400.4, 402.4 [M + H]<sup>+</sup>; 398.4, 400.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_116" class="NLM_sec NLM_sec_level_2"><div id="ac_i133" class="anchor-spacer"></div><h3 class="article-section__title" id="_i133"> (<i>E</i>)-3-(3-(4-Methoxyphenyl)acryloyl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>113b</b>)</h3><div class="NLM_p last">Compound <b>113b</b> was synthesized via general procedure A using <b>61a</b> (265 mg, 1.00 mmol) and <i>p</i>-anisaldehyde <b>65</b> (122 μL, 1.00 mmol). Title compound <b>113b</b> was obtained after precipitation and filtration from the reaction crude (304 mg, yield 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.16 (s, 1H), 7.66–7.51 (m, 3H), 7.47–7.34 (m, 5H), 7.32–7.27 (m, 2H), 7.14 (t, <i>J</i> = 7.7 Hz, 1H), 7.06 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 6.93 (d, <i>J</i> = 8.8 Hz, 2H), 6.60 (d, <i>J</i> = 16.3 Hz, 1H), 3.78 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 194.07, 161.37, 159.72, 147.51, 145.75, 138.92, 134.40, 131.59, 130.85, 130.48, 128.91, 128.41, 128.19, 126.83, 125.41, 122.04, 119.23, 115.76, 114.36, 55.33. <i>t</i><sub>R</sub> = 2.19 min (generic method). ESI-MS for C<sub>25</sub>H<sub>19</sub>NO<sub>3</sub>: calculated 381.1, found <i>m</i>/<i>z</i> 382.5 [M + H]<sup>+</sup>, 380.5 [M – H]<sup>+</sup>.</div></div><div id="sec4_1_1_117" class="NLM_sec NLM_sec_level_2"><div id="ac_i134" class="anchor-spacer"></div><h3 class="article-section__title" id="_i134"> 3-(5-(4-Methoxyphenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)-4-phenylquinolin-2(1<i>H</i>)-one (<b>113c</b>)</h3><div class="NLM_p last">Compound <b>113c</b> was synthesized via general procedure B using <b>113b</b> (300 mg, 0.79 mmol) with hydrazine hydrate (77 μL, 1.58 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0–5% EtOH/DCM) to afford <b>113c</b> (212 mg, yield 69%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.02 (s, 1H), 7.54–7.43 (m, 4H), 7.39 (dd, <i>J</i> = 8.3, 1.2 Hz, 1H), 7.31 (dq, <i>J</i> = 7.6, 1.4 Hz, 1H), 7.21 (ddd, <i>J</i> = 5.3, 4.0, 2.0 Hz, 1H), 7.09 (ddd, <i>J</i> = 8.2, 7.0, 1.2 Hz, 1H), 7.00 (dt, <i>J</i> = 8.2, 2.2 Hz, 3H), 6.82–6.76 (m, 2H), 4.51 (t, <i>J</i> = 10.3 Hz, 1H), 3.72 (s, 3H), 3.16 (dd, <i>J</i> = 16.4, 10.9 Hz, 1H), 2.57–2.51 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.73, 158.22, 149.48, 146.05, 138.21, 135.81, 135.45, 130.43, 129.25, 128.74, 128.08, 128.05, 127.86, 127.59, 126.91, 125.62, 121.88, 119.59, 115.28, 113.57, 62.36, 55.02, 44.63. <i>t</i><sub>R</sub> = 2.08 min (generic method). ESI-MS for C<sub>25</sub>H<sub>21</sub>N<sub>3</sub>O<sub>2</sub>: calculated 395.1, found <i>m</i>/<i>z</i> 396.5 [M + H]<sup>+</sup>; 394.6 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_118" class="NLM_sec NLM_sec_level_2"><div id="ac_i135" class="anchor-spacer"></div><h3 class="article-section__title" id="_i135"> (<i>E</i>)-4-Phenyl-3-(3-(4-(trifluoromethyl)phenyl)acryloyl)quinolin-2(1<i>H</i>)-one (<b>114b</b>)</h3><div class="NLM_p last">Compound <b>114b</b> was synthesized via general procedure A using <b>61a</b> (301 mg, 1.10 mmol) and 4-trifluoromethylbenzaldehyde <b>68</b> (150 μL, 1.10 mmol). Title compound <b>114b</b> was obtained after precipitation and filtration from the reaction crude (414 mg, yield 87%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.23 (s, 1H), 7.81 (dd, <i>J</i> = 62.7, 7.9 Hz, 4H), 7.67–7.24 (m, 8H), 7.25–7.05 (m, 2H), 6.91 (d, <i>J</i> = 16.2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 194.74, 160.08, 148.71, 143.97, 139.35, 138.85, 134.71, 131.63, 130.31, 129.64, 129.40, 129.04, 128.76, 127.45, 126.14, 122.71, 119.67, 116.20. <i>t</i><sub>R</sub> = 2.41 min (generic method). ESI-MS for C<sub>25</sub>H<sub>16</sub>F<sub>3</sub>NO<sub>2</sub>: calculated 419.1, found <i>m</i>/<i>z</i> 420.5 [M + H]<sup>+</sup>; 418.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_119" class="NLM_sec NLM_sec_level_2"><div id="ac_i136" class="anchor-spacer"></div><h3 class="article-section__title" id="_i136"> 4-Phenyl-3-(5-(4-(trifluoromethyl)phenyl)-4,5-dihydro-1<i>H</i>-pyrazol-3-yl)quinolin-2(1<i>H</i>)-one (<b>114c</b>)</h3><div class="NLM_p last">Compound <b>114c</b> was synthesized via general procedure B using <b>114b</b> (256 mg, 0.60 mmol) with hydrazine hydrate (58 μL, 1.20 mmol). Purification was performed by direct phase flash chromatography (SiO<sub>2</sub> gold 24 g; 0.8–6% EtOH/DCM) to afford <b>114c</b> (214 mg, yield 80%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.05 (s, 1H), 7.60 (d, <i>J</i> = 8.1 Hz, 2H), 7.55–7.37 (m, 5H), 7.32 (dd, <i>J</i> = 11.5, 7.6 Hz, 3H), 7.24–7.17 (m, 2H), 7.10 (ddd, <i>J</i> = 8.2, 7.0, 1.2 Hz, 1H), 7.00 (dd, <i>J</i> = 8.2, 1.4 Hz, 1H), 4.69 (ddd, <i>J</i> = 11.6, 9.1, 2.8 Hz, 1H), 3.33 (m, 1H), 2.63–2.52 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.17, 150.18, 148.95, 146.44, 138.72, 136.22, 131.00, 129.76, 129.22, 129.13, 128.78, 128.57, 128.51, 128.37, 128.02, 127.72, 127.42, 126.14, 125.77, 125.62, 125.57, 125.53, 122.40, 120.02, 115.79, 62.59, 45.15. <i>t</i><sub>R</sub> = 2.30 min (generic method). ESI-MS for C<sub>25</sub>H<sub>18</sub>F<sub>3</sub>N<sub>3</sub>O: calculated 433.1, found <i>m</i>/<i>z</i> 434.4 [M + H]<sup>+</sup>; 432.5 [M – H]<sup>−</sup>.</div></div><div id="sec4_1_1_120" class="NLM_sec NLM_sec_level_2"><div id="ac_i137" class="anchor-spacer"></div><h3 class="article-section__title" id="_i137"> <i>tert</i>-Butyl (3-(3-(6-Chloro-2-oxo-4-phenyl-1,2-dihydroquinolin-3-yl)-5-(4-fluorophenyl)-4,5-dihydro-1<i>H</i>-pyrazol-1-yl)-3-oxopropyl)carbamate (<b>119</b>)</h3><div class="NLM_p last">In round-bottom flask, the commercially available 3-((<i>tert</i>-butoxycarbonyl)amino)propanoic acid <b>118</b> (182 mg, 0.96 mmol), HOBt (156 mg, 1.16 mmol), and EDCI (222 mg, 1.16 mmol) were stirred in DCM (15 mL) at rt for 1 h. Then a solution of the <b>86c</b> (400 mg, 0.96 mmol) and Et<sub>3</sub>N (296 μL, 2.12 mmol) in DCM (5 mL) was added. The mixture was stirred at rt overnight. The solvent was removed under reduced pressure, the residue redissolved with EtOAc and then washed with H<sub>2</sub>O, 1 M NaHCO<sub>3</sub>, and finally 10% citric acid. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated to dryness. Purification was performed by direct phase flash chromatography (0–30% EtOAc/DCM). Yield 72 mg, 25%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.40 (s, 1H), 7.65 (dd, <i>J</i> = 4.0, 8.0 Hz, 1H), 7.59–7.50 (m, 4H), 7.46 (d, <i>J</i> = 8.0 Hz, 1H), 7.43–7.42 (m, 1H), 7.28 (d, <i>J</i> = 8.0 Hz, 1H), 7.04 (t, <i>J</i> = 8.0 Hz, 2H), 6.93 (d, <i>J</i> = 2.0 Hz, 1H), 6.80 (dd, <i>J</i> = 4.0, 8.0 Hz, 2H), 6.66 (t, <i>J</i> = 8.0 Hz, 1H), 5.32 (dd, <i>J</i> = 4.0, 12.0 Hz, 1H), 3.73 (dd, <i>J</i> = 12.0, 16.0 Hz, 1H), 2.99 (dd, <i>J</i> = 8.0, 12.0 Hz, 2H), 2.76 (dd, <i>J</i> = 4.0, 20.0 Hz, 1H) 2.41 (q, <i>J</i> = 8.0 Hz, 2H), 1.38 (s, 9H).</div></div><div id="sec4_1_1_121" class="NLM_sec NLM_sec_level_2"><div id="ac_i138" class="anchor-spacer"></div><h3 class="article-section__title" id="_i138"> Methyl 3-(Methylsulfonamido)propanoate (<b>123</b>)</h3><div class="NLM_p last">In a dried round-bottom flask, commercially available methyl 3-aminopropanoate <b>121</b> (745 mg, 5.34 mmol) and triethylamine (3.7 mL, 26.68 mmol) were stirred in anhydrous DCM (5.6 mL) prior to the addition of methanesulfonyl chloride <b>122</b> (1.65 mL, 10.67 mmol). After stirring at rt for 2 days, the reaction was quenched with NaHCO<sub>3</sub> sat. solution and extracted three times with CHCl<sub>3</sub>. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and evaporated to dryness. The title compound was obtained after purification over direct phase flash column chromatography (0–40% EtOAc/petroleum ether). Yield 791 mg, 82%. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>-<i>d</i>) δ 4.94 (s, 1H), 3.72 (s, 3H), m (3.42–3.37, 2H), 2.97 (s, 3H), 2.64 (t, <i>J</i> = 4 Hz, 2H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>-<i>d</i>) δ 172.63, 52.18, 40.60, 38.89, 34.58.</div></div><div id="sec4_1_1_122" class="NLM_sec NLM_sec_level_2"><div id="ac_i139" class="anchor-spacer"></div><h3 class="article-section__title" id="_i139"> 3-(Methylsulfonamido)propanoic Acid (<b>124</b>)</h3><div class="NLM_p last">In round-bottom flask, methyl 3-(methylsulfonamido)propanoate <b>123</b> (791 mg, 4.36 mmol) was stirred in MeOH/THF (1:1 v/v) prior to the addition of 2 M LiOH (2.5 mL). The reaction was stirred at rt overnight. The reaction mixture was acidified with 1 M HCl<sub>aq</sub> (pH 2) and extracted three times with ethyl acetate. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered and evaporated to dryness to obtain the title compound. Yield 600 mg, 82%. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.29 (s, 1H), 7.02 (t, <i>J</i> = 4.0 Hz, 1H), 7.15 (dd, <i>J</i> = 7.0, 12.6, 2H), 2.43 (t, <i>J</i> = 7.2 Hz, 2H).</div></div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i140" class="anchor-spacer"></div><h3 class="article-section__title" id="_i140"> Biology. ELISA Assay</h3><div class="NLM_p last">Competitive ELISA screening assay using biotinylated BRC4 peptide to disrupt the BRC4–RAD51 interaction was performed by modifying the method described by Rajendra et al.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> BRC4-biotinylated peptide (N-term biotin-KEPTLLGFHTASGKKVKIAKESLDKVKNLFDEKEQ from Life Technologies) was used to coat 384-well plates (Nunc). After washing with PBS containing 0.05% Tween-20 (PBST) and blocking with the solution BSA 1% /PBST, overnight hybridization with human RAD51 protein (NP_002866 Creative Biomart, NY) was performed. Test compounds were added in dose response from 0.01 to 100 μM in triplicate with constant DMSO 1%. Antibody raised against RAD51 (Millipore) and HRP-secondary antibody staining to develop the 3,3′,5,5′-tetramethylbenzidine signal (Sigma) quenched with 1 M HCl was used as the assay readout. Colorimetric measure was read on a Victor5 (PerkinElmer) plate reader. BRC4 and Rad51 were included in the assay as positive control. Results were analyzed by using GraphPad software.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i141" class="anchor-spacer"></div><h3 class="article-section__title" id="_i141"> Protocol for the Expression and Purification of His-hRAD51</h3><div class="NLM_p last">hRAD51 was expressed in <i>E. coli</i> Rosetta2(DE3)pLysS cells. A saturated overnight culture of Rosetta2(DE3)pLysS/pET15b-His-hRAD51 was diluted (1:1000) into a fresh TB-5052 autoinduction medium containing ampicillin (100 μg/mL). The flasks were shaken at 200 rpm at 20 °C for 72 h. The pellet was subsequently resuspended in an appropriate volume of buffer A (20 mM Tris-HCl (pH 8.00), 500 mM NaCl, 10 mM imidazole, 5 mM DTT, 10% (v/v) glycerol) supplemented with protease inhibitor cocktail (SIGMAFAST protease inhibitor cocktail tablets, EDTA-50 free). The cell suspension was lysed on ice trough sonication (24 rounds of 30 in.; amplitude 85%; Tip MS72; Bandelin Sonoplus HD2070 sonicator). The disrupted cell suspension was centrifuged for 1 h at 13 000 rpm. The supernatant fraction was filtered with a 0.45 μm (MiniSart syringe filter 0.45 μm) membrane to remove residual particulates before chromatography. The supernatant was applied onto a His-Trap column (His-TrapTM FF 5 mL, GE Healthcare) equilibrated with buffer A. A wash step was performed using 10% of buffer B (20 mM Tris-HCl (pH 8.00), 500 mM NaCl, 500 mM imidazole, 10% (v/v) glycerol). The protein was then eluted with a linear gradient from 10% to 100% of buffer B over 10 column volumes. Fractions (0.5 mL) were collected and analyzed by SDS–PAGE. Collected fractions corresponding to the recombinant protein were dialyzed overnight at 4 °C against buffer C (50 mM Tris-HCl (pH 8.00), 200 mM KCl, 0.25 mM EDTA, 2 mM DTT, 10% (v/v) glycerol). Dialyzed protein was loaded onto an anion exchange column (ResQ, GE Healthcare) equilibrated in buffer C. The elution was performed with a linear gradient of buffer B (50 mM Tris-HCl (pH 8.00), 1 M KCl, 0.25 mM EDTA, 2 mM DTT, 10% (v/v) glycerol). Fractions (0.5 mL) were collected and analyzed by SDS–PAGE. Fractions containing His-hRAD51 were pooled and dialyzed against the storage buffer (20 mM Hepes (pH 8.00), 250 mM KCl, 0.1 mM EDTA, 2 mM DTT, 10% (v/v) glycerol). The protein yield was determined from the optical absorption at 280 nm (extinction coefficient 14 900 M<sup>–1</sup> cm<sup>–1</sup>) of the final sample.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i142" class="anchor-spacer"></div><h3 class="article-section__title" id="_i142"> Microscale Thermophoresis</h3><div class="NLM_p last">The recombinant protein hRAD51 was labeled with the Monolith His-Tag labeling kit RED-tris-NTA 2nd Generation kit (NanoTemper Technologies). MST measurements were simultaneously performed on 16 capillaries containing a constant concentration (25 nM) of labeled RED-tris-NTA 2nd Generation His-hRAD51 protein and 16 different concentrations of <b>35d</b> in order to determine a concentration-dependent MST binding curve. The highest <b>35d</b> concentration tested was 40 μM. Measurements were carried out in MST buffer (20 mM Hepes (pH 8.00), 250 mM KCl, 0.1 mM EDTA, 5% (v/v) glycerol, 5% DMSO).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i143" class="anchor-spacer"></div><h3 class="article-section__title" id="_i143"> Cell Culture and Treatments</h3><div class="NLM_p">BxPC-3 and Capan-1 cells were grown in RPMI 1640 supplemented with 10% FBS, 100 U/mL penicillin/streptomycin, 2 mM glutamine. All media and supplements were from Sigma-Aldrich. Non-neoplastic, immortalized cells from human kidney (HK-2, ATCC CRL2190) were grown in DMEM:F12 medium containing 40 ng/mL dexamethasone and supplemented as described above.</div><div class="NLM_p last">All cultures were routinely tested for Mycoplasma contamination. Treatments (olaparib and BRCA2-RAD51 disruptors) were administered in culture medium supplemented with 0.6% DMSO. The same amount of DMSO was added to the control, untreated cultures.</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i144" class="anchor-spacer"></div><h3 class="article-section__title" id="_i144"> Homologous Recombination Assay</h3><div class="NLM_p">Homologous recombination (HR) was assessed by using a commercially available assay (Norgen). This assay is based on cell transfection with two plasmids that, upon cell entry, recombine. The efficiency of HR can be assessed by real-time PCR, using primer mixtures included in the assay kit. Different primer mixtures allow one to discriminate between the original plasmid backbones and their recombination product.</div><div class="NLM_p last">BxPC3 cells (2 × 10<sup>5</sup> per well) were seeded in a 24-well plate and allowed to adhere overnight. Co-transfection with the two plasmids was performed in Lipofectamine 2000 (Invitrogen) according to the manufacturer’s instructions. During transfection (5 h), cells were exposed to different doses of RAD51-BRCA2 disruptors, dissolved in RPMI in the presence of 0.6% DMSO. After washing with PBS, cells were harvested, and DNA was isolated using Illustra Tissue and Cell Genomic Prep Mini Spin kit (GE Healthcare). Sample concentration was measured using an ONDA Nano Genius photometer. The efficiency of HR was assessed by real-time PCR, using 25 ng of template, the primer mixtures included in the assay kit and following the protocol indicated by the manufacturer. Data analysis was based on the ΔΔCt method: [recombination product/backbone plasmids]<sup>treated</sup> versus [recombination product/backbone plasmids]<sup>control</sup>.</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i145" class="anchor-spacer"></div><h3 class="article-section__title" id="_i145"> Immunofluorescence</h3><div class="NLM_p">Immunofluorescence was used for studying RAD51 nuclear translocation and for evaluating DNA damage through the detection of γH2AX nuclear foci. To visualize RAD51 in cell nuclei, BxPC-3 cells were seeded on glass coverslips placed in a 6-well culture plate (2 × 10<sup>5</sup> cells/well) and allowed to adhere overnight. Cultures were then preincubated with 20 μM <b>35d</b> for 1 h and subsequently exposed to 50 μM cisplatin for an additional 1.5 h. Medium was removed, and cells were maintained in the presence of 20 μM <b>35d</b> for 4 h. After this time, cultures growing on coverslips were fixed in PBS containing 1% formalin for 20 min, permeabilized in 70% ethanol, air-dried, and washed twice with PBS. Samples were incubated in 10% bovine serum albumin (BSA) in PBS for 30 min at 37 °C and subsequently exposed to an anti-RAD51 mouse monoclonal antibody (Santa Cruz Biotechnology, 1:1000 in 5% BSA/PBS) overnight at 4 °C. After washing, coverslips were incubated with an anti-mouse FITC-conjugated secondary antibody (1:1000 in 1% BSA/PBS) for 1 h at 37 °C, washed, air-dried, and mounted with a solution of DAPI (2 μg/mL) and DABCO.</div><div class="NLM_p last">To evaluate DNA damage through γH2AX nuclear foci, BXPC3 and Capan1 cells were seeded on glass coverslips in 6-well tissue culture plate (2 × 10<sup>5</sup> cells/well) and allowed to adhere overnight. After 48 h treatment with olaparib (10 μM) or <b>35d</b> (20 μM) given alone or in combination, cultures growing on coverslips were fixed and treated as described above. For this experiment, the used antibodies were a rabbit polyclonal anti-γH2AX (Abcam, 1:1000 in 5% BSA/PBS) and a secondary anti-rabbit rhodamine-labeled (Novus Biologicals, 1:1000 in 1% BSA/PBS). For both experiments, images were acquired using a Nikon fluorescent microscope equipped with filters for FITC, TRITC, and DAPI. The percentage of cells bearing nuclear foci was estimated by two independent observers, by analyzing 100–250 cells for each treatment sample.</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i146" class="anchor-spacer"></div><h3 class="article-section__title" id="_i146"> Cell Viability Assay</h3><div class="NLM_p last">Cell viability was assessed with the CellTiter-Glo luminescent cell viability assay from Promega. For this experiment, 1 × 10<sup>4</sup> cells in 200 μL of culture medium were seeded into each well of a 96-multiwell white body plate and allowed to adhere overnight. After 72 h incubation in the presence of olaparib (10 μM) and the RAD51-BRCA2 disruptors alone or in combination, the plate was allowed to equilibrate at room temperature for 30 min and the CellTiter-Glo reactive was directly added to each well. The plate was kept on a shaker for 10 min to induce cell lysis, and its luminescence was measured with a Fluoroskan Ascent FL reader (Labsystems).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i147" class="anchor-spacer"></div><h3 class="article-section__title" id="_i147"> Cytotoxicity Assay</h3><div class="NLM_p last">Cell death was assessed by applying the CellTox Green cytotoxicity assay (Promega). Briefly, BxPC-3 (1 × 10<sup>4</sup>/well) were plated in 96-well plates and treated for 72 h with olaparib (10 μM) and the BRCA2-RAD51 disruptors administered alone or in combination. At the end of treatment, the CellTox dye was added to cell cultures and the green fluorescence signal, which is produced by the binding interaction with dead cell DNA, was measured following the manufacturer’s instructions.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i148" class="anchor-spacer"></div><h3 class="article-section__title" id="_i148"> Assessment of Cell Death with Vital Dyes</h3><div class="NLM_p last">BxPC-3 cells were grown on coverslips placed in a 6-multiwell plate (5 × 10<sup>5</sup> cells/well). After a 72 h treatment with olaparib (10 μM) or <b>35d</b> (20 μM) given alone or in combination, wells were rapidly washed with PBS and filled with 500 μL of a PBS solution containing DAPI (4.6 μg/mL) and PI (50 μg/mL). After a 10 min incubation at room temperature under light-shielded condition, they were washed with PBS, fixed (10 min) with 10% neutral buffered formalin solution, and washed again to eliminate the fixative. Coverslips were then applied on glass slides using two drops of mounting media (DABCO). Images were acquired on a Nikon fluorescent microscope equipped with filters for DAPI and PI.</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i149" class="anchor-spacer"></div><h3 class="article-section__title" id="_i149"> Micronuclei Visualization</h3><div class="NLM_p last">For micronuclei visualization, cells were grown on coverslips placed in a 6-multiwell plate (5 × 10<sup>5</sup> cells/well). After a 72 h treatment with olaparib (10 μM) or <b>35d</b> (20 μM) given alone or in combination, wells were rapidly washed with PBS and then fixed for 10 min with cold methanol. Coverslips were then air-dried and mounted on glass slides using a solution of DAPI (5 μg/mL)/DABCO. Images were acquired on a Nikon fluorescent microscope equipped with filters for DAPI. A cell was considered to contain micronuclei if the following criteria were met: (i) one or more round fluorescent bodies were present in the cytoplasm which did not touch the main nucleus; (ii) they were <<sup>1</sup>/<sub>3</sub> of the main nucleus diameter; (iii) they were nonrefractile, to exclude foreign bodies. The percentage of cells bearing micronuclei was estimated by two independent observers, by analyzing 100–250 cells for each treatment sample.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i150"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01526" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78350" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78350" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.9b01526?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01526</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Structure representation of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_002.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Structure representation of <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Computational methods; separation and ELISA assay results of enantiomers <b>4d-I</b> and <b>4d-II</b>; experimental procedure and characterization data of aldehydes <b>67</b>, <b>70</b>, <b>74</b>–<b>84</b>; <sup>1</sup>H and <sup>13</sup>C NMR spectra of final compounds <b>4d</b>–<b>10d</b>, <b>14d</b>–<b>15d</b>, <b>18d</b>–<b>57d</b>; HPLC–MS analysis of selected final compounds <b>4d</b>–<b>10d</b>, <b>14d</b>–<b>15d</b>, <b>18</b>–<b>25d</b>, <b>35d</b>, <b>49d</b>, <b>57d</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_004.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_001.csv">jm9b01526_si_001.csv (4.88 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_002.pdb">jm9b01526_si_002.pdb (271.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_003.pdb">jm9b01526_si_003.pdb (271.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_004.pdf">jm9b01526_si_004.pdf (10.94 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b01526" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marinella Roberti</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9807-2886" title="Orcid link">http://orcid.org/0000-0001-9807-2886</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5934382b30373c353538772b363b3c2b2d30192c37303b3677302d"><span class="__cf_email__" data-cfemail="2b464a5942454e47474a055944494e595f426b5e4542494405425f">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Cavalli</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6370-1176" title="Orcid link">http://orcid.org/0000-0002-6370-1176</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3859565c4a5d59165b594e595454517851514c16514c"><span class="__cf_email__" data-cfemail="4a2b242e382f2b64292b3c2b2626230a23233e64233e">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Greta Bagnolini</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Domenico Milano</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcella Manerba</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabrizio Schipani</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jose Antonio Ortega</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dario Gioia</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Federico Falchi</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span>; 
    <span>Present Address:
                        F.F.: Molecular Horizon srl, Via Montelino 30, 06084 Bettona (PG), Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Balboni</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fulvia Farabegoli</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy and Biotechnology, University
of Bologna, Via Belmeloro 6, 40126 Bologna, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Francesca De Franco</span> - <span class="hlFld-Affiliation affiliation">TES
Pharma S.r.l., Via Palmiro Togliatti 22bis, I-06073 Corciano, Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janet Robertson</span> - <span class="hlFld-Affiliation affiliation">TES
Pharma S.r.l., Via Palmiro Togliatti 22bis, I-06073 Corciano, Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberto Pellicciari</span> - <span class="hlFld-Affiliation affiliation">TES
Pharma S.r.l., Via Palmiro Togliatti 22bis, I-06073 Corciano, Perugia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Isabella Pallavicini</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology at the IEO, European
Institute of Oncology IRCCS, IFOM-IEO Campus, Via Adamello 16, 20100 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastiano Peri</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology at the IEO, European
Institute of Oncology IRCCS, IFOM-IEO Campus, Via Adamello 16, 20100 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Saverio Minucci</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental Oncology at the IEO, European
Institute of Oncology IRCCS, IFOM-IEO Campus, Via Adamello 16, 20100 Milan, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biosciences, University of Milan, Via Celoria 26, 20100 Milan, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefania Girotto</span> - <span class="hlFld-Affiliation affiliation">Computational
and Chemical Biology, Istituto Italiano
di Tecnologia, Via Morego 30, 16163 Genoa, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppina Di Stefano</span> - <span class="hlFld-Affiliation affiliation">Department
of Experimental, Diagnostic and Specialty Medicine, University of Bologna, Via S. Giacomo 14, 40126 Bologna, Italy</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>G.B., D.M., and M.M. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): One patent application protecting the class of compounds disclosed in this article has been filed by the following authors: Greta Bagnolini, Domenico Milano, Marcella Manerba, Jose Antonio Ortega, Francesca De Franco, Roberto Pellicciari, Saverio Minucci, Giuseppina Di Stefano, Marinella Roberti, and Andrea Cavalli.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i153">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04482" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04482" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Associazione Italiana per la Ricerca sul Cancro AIRC (Progetto IG 2018, id 21386), the Italian Institute of Technology (IIT), and the University of Bologna. We thank Prof. Maurizio Recanatini for fruitful discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">Boc<sub>2</sub>O</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butyl dicarbonate</p></td></tr><tr><td class="NLM_term">CHX</td><td class="NLM_def"><p class="first last">cyclohexane</p></td></tr><tr><td class="NLM_term">DCM</td><td class="NLM_def"><p class="first last">dichloromethane</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethylsulfoxide</p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last"><i>N</i>-(3-dimethylaminopropyl)-<i>N</i>′-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">ESI</td><td class="NLM_def"><p class="first last">electrospray ionization</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry</p></td></tr><tr><td class="NLM_term">μWave</td><td class="NLM_def"><p class="first last">microwave</p></td></tr><tr><td class="NLM_term">MeI</td><td class="NLM_def"><p class="first last">iodomethane</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TLC</td><td class="NLM_def"><p class="first last">thin layer chromatography</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">DAPI</td><td class="NLM_def"><p class="first last">4′,6-diamidino-2-phenylindole</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">propidium iodide</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i155">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51658" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51658" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, C. B.</span></span> <span> </span><span class="NLM_article-title">The origin and variations in sexual and sex limited characters</span>. <i>Am. Nat.</i> <span class="NLM_year" style="font-weight: bold;">1922</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1086/279847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1086%2F279847" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1922&pages=51-63&author=C.+B.+Bridges&title=The+origin+and+variations+in+sexual+and+sex+limited+characters&doi=10.1086%2F279847"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1086%2F279847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F279847%26sid%3Dliteratum%253Aachs%26aulast%3DBridges%26aufirst%3DC.%2BB.%26atitle%3DThe%2520origin%2520and%2520variations%2520in%2520sexual%2520and%2520sex%2520limited%2520characters%26jtitle%3DAm.%2520Nat.%26date%3D1922%26volume%3D56%26spage%3D51%26epage%3D63%26doi%3D10.1086%2F279847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobzhansky, T.</span></span> <span> </span><span class="NLM_article-title">Genetics of natural populations. Xiii. Recombination and variability in populations of Drosophila pseudoobscura</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">1946</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">290</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=20985721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1946&pages=269-290&author=T.+Dobzhansky&title=Genetics+of+natural+populations.+Xiii.+Recombination+and+variability+in+populations+of+Drosophila+pseudoobscura"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genetics of natural populations; recombination and variability in populations of Drosophila pseudoobscura</span></div><div class="casAuthors">DOBZHANSKY T</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">1946</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">269-90</span>
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXehIoFNdovudfST00W0BcfW6udTcc2eZ3YT5V2rkoZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D&md5=51f55835f5a3053a235bdea89458573c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDobzhansky%26aufirst%3DT.%26atitle%3DGenetics%2520of%2520natural%2520populations.%2520Xiii.%2520Recombination%2520and%2520variability%2520in%2520populations%2520of%2520Drosophila%2520pseudoobscura%26jtitle%3DGenetics%26date%3D1946%26volume%3D31%26spage%3D269%26epage%3D290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, J. C.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality and semi-lethality among functionally related mutants of Drosophila melanfgaster</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">1968</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">44</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fgenetics%2F59.1.37" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=5683639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A280%3ADyaF1M%252FhtFyrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1968&pages=37-44&author=J.+C.+Lucchesi&title=Synthetic+lethality+and+semi-lethality+among+functionally+related+mutants+of+Drosophila+melanfgaster"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality and semi-lethality among functionally related mutants of Drosophila melanfgaster</span></div><div class="casAuthors">Lucchesi J C</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">1968</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-44</span>
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXM96_03HVQCO2kE7krGOBfW6udTcc2eYa-rr158sOs7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF1M%252FhtFyrtg%253D%253D&md5=0a74c995f844634c47a35e6a5b39b90a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fgenetics%2F59.1.37&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fgenetics%252F59.1.37%26sid%3Dliteratum%253Aachs%26aulast%3DLucchesi%26aufirst%3DJ.%2BC.%26atitle%3DSynthetic%2520lethality%2520and%2520semi-lethality%2520among%2520functionally%2520related%2520mutants%2520of%2520Drosophila%2520melanfgaster%26jtitle%3DGenetics%26date%3D1968%26volume%3D59%26spage%3D37%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schekman, R.</span></span> <span> </span><span class="NLM_article-title">Distinct sets of SEC genes govern transport vesicle formation and fusion early in the secretory pathway</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(90)90483-U</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2F0092-8674%2890%2990483-U" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=2188733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK3cXktFyrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1990&pages=723-733&author=C.+A.+Kaiserauthor=R.+Schekman&title=Distinct+sets+of+SEC+genes+govern+transport+vesicle+formation+and+fusion+early+in+the+secretory+pathway&doi=10.1016%2F0092-8674%2890%2990483-U"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct sets of SEC genes govern transport vesicle formation and fusion early in the secretory pathway</span></div><div class="casAuthors">Kaiser, Chris A.; Schekman, Randy</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">723-33</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">A vesicular intermediate in protein transport from the endoplasmic reticulum is detected in a subset of temp.-sensitive mutants blocked early in the yeast secretory pathway.  By electron microscopy three of the mutants, sec18, sec17, and sec22, accumulate 50 nm vesicles at the nonpermissive temp.  Vesicle accumulation is blocked by the mutations sec12, sec13, sec16, and sec23 as shown by anal. of double-mutant strains.  Thus the early SEC genes can be divided into vesicle forming and vesicle fusion functions.  Synthetic lethal interactions between sec mutations define two groups of SEC genes, corresponding to the groups involved in vesicle formation or fusion.  Mutations in two of the genes involved in vesicle fusion, SEC17 and SEC18, are lethal in combination, and five of six possible pairwise combinations of mutations in genes required for vesicle formation, SEC12, SEC13, SEC16, and SEC23, are lethal.  These interactions suggest cooperation between different SEC gene products in vesicle budding and vesicle fusion processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriiTAfitWtmrVg90H21EOLACvtfcHk0lhz12wcZk3png"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktFyrsLY%253D&md5=e4c09689d2eaf9acfe470d244bf4b39c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2890%2990483-U&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252890%252990483-U%26sid%3Dliteratum%253Aachs%26aulast%3DKaiser%26aufirst%3DC.%2BA.%26aulast%3DSchekman%26aufirst%3DR.%26atitle%3DDistinct%2520sets%2520of%2520SEC%2520genes%2520govern%2520transport%2520vesicle%2520formation%2520and%2520fusion%2520early%2520in%2520the%2520secretory%2520pathway%26jtitle%3DCell%26date%3D1990%26volume%3D61%26spage%3D723%26epage%3D733%26doi%3D10.1016%2F0092-8674%2890%2990483-U" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botstein, D.</span></span> <span> </span><span class="NLM_article-title">A group of interacting yeast DNA replication genes</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">958</span>– <span class="NLM_lpage">969</span>, <span class="refDoi"> DOI: 10.1101/gad.5.6.958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1101%2Fgad.5.6.958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=2044962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK38XhsVGju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1991&pages=958-969&author=K.+M.+Hennessyauthor=A.+Leeauthor=E.+Chenauthor=D.+Botstein&title=A+group+of+interacting+yeast+DNA+replication+genes&doi=10.1101%2Fgad.5.6.958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A group of interacting yeast DNA replication genes</span></div><div class="casAuthors">Hennessy, Kevin M.; Lee, Angela; Chen, Ellson; Botstein, David</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">958-69</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    </div><div class="casAbstract">Mutations in the cell-division-cycle genes CDC46 and CDC47 were originally isolated as suppressors of mutations in 2 other cell-division-cycle genes (CDC45 and CDC54).  Several combinations of mutations were found in these genes that result in allele-specific suppression and synthetic lethality, confirming that this set of genes forms a group of genetically interacting components.  Here, it is shown that the other genes, like CDC46, are all involved in an early step of DNA replication, possibly initiation of DNA synthesis.  Mutants defective in each of the 4 genes exhibit high rates of mitotic chromosome loss and recombination.  The mutants appear also to accumulate chromosome damage that can be detected by a novel chromosome electrophoresis assay.  Conditional mutants in this group, under fully nonpermissive conditions, show cell-cycle arrest at the beginning of DNA synthesis; under less stringent conditions, some arrest later, in S-phase.  The DNA sequence of the CDC46 gene indicates that the protein is a member of a new family of genes apparently required for DNA initiation, with family members now identified in Saccharomyces cerevisiae, Schizosaccharomyces pombe, and mouse cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmvD7dqS8PXbVg90H21EOLACvtfcHk0ljARKjKN1aBNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsVGju7o%253D&md5=a2dae99c0886b0f4b3ce744b46c43253</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.5.6.958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.5.6.958%26sid%3Dliteratum%253Aachs%26aulast%3DHennessy%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DBotstein%26aufirst%3DD.%26atitle%3DA%2520group%2520of%2520interacting%2520yeast%2520DNA%2520replication%2520genes%26jtitle%3DGenes%2520Dev.%26date%3D1991%26volume%3D5%26spage%3D958%26epage%3D969%26doi%3D10.1101%2Fgad.5.6.958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The concept of synthetic lethality in the context of anticancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nrc1691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnrc1691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=16110319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsleks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=689-698&author=W.+G.+Kaelin&title=The+concept+of+synthetic+lethality+in+the+context+of+anticancer+therapy&doi=10.1038%2Fnrc1691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Concept of Synthetic Lethality in the Context of Anticancer Therapy</span></div><div class="casAuthors">Kaelin, William G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">689-698</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Two genes are synthetic lethal if mutation of either alone is compatible with viability but mutation of both leads to death.  So, targeting a gene that is synthetic lethal to a cancer-relevant mutation should kill only cancer cells and spare normal cells.  Synthetic lethality therefore provides a conceptual framework for the development of cancer-specific cytotoxic agents.  This paradigm has not been exploited in the past because there were no robust methods for systematically identifying synthetic lethal genes.  This is changing as a result of the increased availability of chem. and genetic tools for perturbing gene function in somatic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf4V-KQ6I5ILVg90H21EOLACvtfcHk0ljkBNS6DBJwvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsleks7c%253D&md5=4d5e38282cd8a3d2c3451d56ca88ac8b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc1691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1691%26sid%3Dliteratum%253Aachs%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DThe%2520concept%2520of%2520synthetic%2520lethality%2520in%2520the%2520context%2520of%2520anticancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2005%26volume%3D5%26spage%3D689%26epage%3D698%26doi%3D10.1038%2Fnrc1691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dar, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, R.</span></span> <span> </span><span class="NLM_article-title">Synthetic Lethality: From research to precision cancer nanomedicine</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.2174/1568009617666170630141931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.2174%2F1568009617666170630141931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=28669337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvFejtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=337-346&author=A.+Guptaauthor=A.+Ahmadauthor=A.+I.+Darauthor=R.+Khan&title=Synthetic+Lethality%3A+From+research+to+precision+cancer+nanomedicine&doi=10.2174%2F1568009617666170630141931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethality: From Research to Precision Cancer Nanomedicine</span></div><div class="casAuthors">Gupta, Anuradha; Ahmad, Anas; Dar, Aqib Iqbal; Khan, Rehan</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-346</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer is an evolutionary disease with multiple genetic alterations, accumulated due to chromosomal instability and/or aneuploidy and it sometimes acquires drug-resistant phenotype also.  Whole genome sequencing and mutational anal. helped in understanding the differences among persons for predisposition of a disease and its treatment non-responsiveness.  Thus, mol. targeted therapies came into existence.  Among them, the concept of synthetic lethality have enthralled great attention as it is a pragmatic approach towards exploiting cancer cell specific mutations to specifically kill cancer cells without affecting normal cells and thus enhancing anti-cancer drug therapeutic index.  Thus, this approach helped in discovering new therapeutic mols. for development of precision medicine.  Nanotechnol. helped in delivering these mols. to the target site in an effective concn. thus reducing off target effects of drugs, dose and dosage frequency drugs.  Researchers have tried to deliver siRNA targeting synthetic lethal partner for target cancer cell killing by incorporating it in nanoparticles and it has shown efficacy by preventing tumor progression.  This review summarizes the brief introduction of synthetic lethality, and synthetic lethal gene interactions, with a major focus on its therapeutic anticancer potential with the application of nanotechnol. for development of personalized medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNT60k7B0777Vg90H21EOLACvtfcHk0ljkBNS6DBJwvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvFejtbc%253D&md5=a2385841eda520b634f1f6ab2eaa885a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1568009617666170630141931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009617666170630141931%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DA.%26aulast%3DAhmad%26aufirst%3DA.%26aulast%3DDar%26aufirst%3DA.%2BI.%26aulast%3DKhan%26aufirst%3DR.%26atitle%3DSynthetic%2520Lethality%253A%2520From%2520research%2520to%2520precision%2520cancer%2520nanomedicine%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2018%26volume%3D18%26spage%3D337%26epage%3D346%26doi%3D10.2174%2F1568009617666170630141931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccia, A. J.</span></span> <span> </span><span class="NLM_article-title">Harnessing synthetic lethal interactions in anticancer drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1038/nrd3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnrd3374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=21532565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVGltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=351-364&author=D.+A.+Chanauthor=A.+J.+Giaccia&title=Harnessing+synthetic+lethal+interactions+in+anticancer+drug+discovery&doi=10.1038%2Fnrd3374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing synthetic lethal interactions in anticancer drug discovery</span></div><div class="casAuthors">Chan, Denise A.; Giaccia, Amato J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">351-364</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Unique features of tumors that can be exploited by targeted therapies are a key focus of current cancer research.  One such approach is known as synthetic lethality screening, which involves searching for genetic interactions of two mutations whereby the presence of either mutation alone has no effect on cell viability but the combination of the two mutations results in cell death.  The presence of one of these mutations in cancer cells but not in normal cells can therefore create opportunities to selectively kill cancer cells by mimicking the effect of the second genetic mutation with targeted therapy.  Here, we summarize strategies that can be used to identify synthetic lethal interactions for anticancer drug discovery, describe examples of such interactions that are currently being investigated in preclin. and clin. studies of targeted anticancer therapies, and discuss the challenges of realizing the full potential of such therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrndLGWxThAjrVg90H21EOLACvtfcHk0ljLO5eGAEaEqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVGltrY%253D&md5=816a9dc23817f6bf0597b4a026fd2aac</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3374%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DD.%2BA.%26aulast%3DGiaccia%26aufirst%3DA.%2BJ.%26atitle%3DHarnessing%2520synthetic%2520lethal%2520interactions%2520in%2520anticancer%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D351%26epage%3D364%26doi%3D10.1038%2Fnrd3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beijersbergen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessels, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernards, R.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality in cancer therapeutics</span>. <i>Annu. Rev. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1146/annurev-cancerbio-042016-073434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1146%2Fannurev-cancerbio-042016-073434" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2017&pages=141-161&author=R.+Beijersbergenauthor=L.+Wesselsauthor=R.+Bernards&title=Synthetic+lethality+in+cancer+therapeutics&doi=10.1146%2Fannurev-cancerbio-042016-073434"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cancerbio-042016-073434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cancerbio-042016-073434%26sid%3Dliteratum%253Aachs%26aulast%3DBeijersbergen%26aufirst%3DR.%26aulast%3DWessels%26aufirst%3DL.%26aulast%3DBernards%26aufirst%3DR.%26atitle%3DSynthetic%2520lethality%2520in%2520cancer%2520therapeutics%26jtitle%3DAnnu.%2520Rev.%2520Cancer%2520Biol.%26date%3D2017%26volume%3D1%26spage%3D141%26epage%3D161%26doi%3D10.1146%2Fannurev-cancerbio-042016-073434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span> <span> </span><span class="NLM_article-title">Targetin epigenetics using synthetic lethality in precision medicine</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3381</span>– <span class="NLM_lpage">3392</span>, <span class="refDoi"> DOI: 10.1007/s00018-018-2866-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1007%2Fs00018-018-2866-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30003270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlSqt7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2018&pages=3381-3392&author=E.+Chen&title=Targetin+epigenetics+using+synthetic+lethality+in+precision+medicine&doi=10.1007%2Fs00018-018-2866-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetics using synthetic lethality in precision medicine</span></div><div class="casAuthors">Chen, Ee Sin</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3381-3392</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Technol. breakthroughs in genomics have had a significant impact on clin. therapy for human diseases, allowing us to use patient genetic differences to guide medical care.  The "synthetic lethal approach" leverages on cancer-specific genetic rewiring to deliver a therapeutic regimen that preferentially targets malignant cells while sparing normal cells.  The utility of this system is evident in several recent studies, particularly in poor prognosis cancers with loss-of-function mutations that become "treatable" when two otherwise discrete and unrelated genes are targeted simultaneously.  This review focuses on the chemotherapeutic targeting of epigenetic alterations in cancer cells and consolidates a network that outlines the interplay between epigenetic and genetic regulators in DNA damage repair.  This network consists of numerous synergistically acting relationships that are druggable, even in recalcitrant triple-neg. breast cancer.  This collective knowledge points to the dawn of a new era of personalized medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcAwplWbsPg7Vg90H21EOLACvtfcHk0ljLO5eGAEaEqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlSqt7%252FK&md5=f56cdc453006c19cfffd0598b4ce6a65</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00018-018-2866-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-018-2866-0%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DE.%26atitle%3DTargetin%2520epigenetics%2520using%2520synthetic%2520lethality%2520in%2520precision%2520medicine%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2018%26volume%3D75%26spage%3D3381%26epage%3D3392%26doi%3D10.1007%2Fs00018-018-2866-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethal therapies for cancer: what’s next after PARP inhibitors?</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0055-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fs41571-018-0055-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=29955114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yks7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=564-576&author=A.+Ashworthauthor=C.+J.+Lord&title=Synthetic+lethal+therapies+for+cancer%3A+what%E2%80%99s+next+after+PARP+inhibitors%3F&doi=10.1038%2Fs41571-018-0055-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethal therapies for cancer: what's next after PARP inhibitors?</span></div><div class="casAuthors">Ashworth, Alan; Lord, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">564-576</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The genetic concept of synthetic lethality has now been validated clin. through the demonstrated efficacy of poly(ADP-ribose) polymerase (PARP) inhibitors for the treatment of cancers in individuals with germline loss-of-function mutations in either BRCA1 or BRCA2.  Three different PARP inhibitors have now been approved for the treatment of patients with BRCA-mutant ovarian cancer and one for those with BRCA-mutant breast cancer; these agents have also shown promising results in patients with BRCA-mutant prostate cancer.  Here, it describe a no. of other synthetic lethal interactions that have been discovered in cancer.  It discuss some of the underlying principles that might increase the likelihood of clin. efficacy and how new computational and exptl. approaches are now facilitating the discovery and validation of synthetic lethal interactions.  Finally, it make suggestions on possible future directions and challenges facing researchers in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjc1AuMwtxGbVg90H21EOLACvtfcHk0lhqL7xFyro07w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yks7zN&md5=77597e5d235648558bf27b141b1db9cd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0055-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0055-6%26sid%3Dliteratum%253Aachs%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26atitle%3DSynthetic%2520lethal%2520therapies%2520for%2520cancer%253A%2520what%25E2%2580%2599s%2520next%2520after%2520PARP%2520inhibitors%253F%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D15%26spage%3D564%26epage%3D576%26doi%3D10.1038%2Fs41571-018-0055-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parameswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundapur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vizeacoumar, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freywald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uppalapati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vizeacoumar, F. J.</span></span> <span> </span><span class="NLM_article-title">A road map to personalizing targeted cancer therapies using synthetic lethality</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2018.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.trecan.2018.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30616753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=11-29&author=S.+Parameswaranauthor=D.+Kundapurauthor=F.+S.+Vizeacoumarauthor=A.+Freywaldauthor=M.+Uppalapatiauthor=F.+J.+Vizeacoumar&title=A+road+map+to+personalizing+targeted+cancer+therapies+using+synthetic+lethality&doi=10.1016%2Fj.trecan.2018.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Road Map to Personalizing Targeted Cancer Therapies Using Synthetic Lethality</span></div><div class="casAuthors">Parameswaran, Sreejit; Kundapur, Deeksha; Vizeacoumar, Frederick S.; Freywald, Andrew; Uppalapati, Maruti; Vizeacoumar, Franco J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-29</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeted therapies rely on the genetic and epigenetic status of the tumor cells and are seen as the most promising approach to treat cancer today.  However, current targeted therapies focus on directly inhibiting those mols. that are altered in tumor cells.  Unfortunately, targeting these mols., even with specific inhibitors, is challenging as tumor cells rewire their genetic circuitry to eliminate genetic dependency on these targets.  Here, we describe how synthetic lethality approaches can be used to identify genetic dependencies and develop personalized targeted therapies.  We also discuss strategies to specifically target these genetic dependencies, using small mol. and biol. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvte78cztNurVg90H21EOLACvtfcHk0lhqL7xFyro07w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKluw%253D%253D&md5=e6e4d907a4d565756650d1762bfd4bb8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2018.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2018.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DParameswaran%26aufirst%3DS.%26aulast%3DKundapur%26aufirst%3DD.%26aulast%3DVizeacoumar%26aufirst%3DF.%2BS.%26aulast%3DFreywald%26aufirst%3DA.%26aulast%3DUppalapati%26aufirst%3DM.%26aulast%3DVizeacoumar%26aufirst%3DF.%2BJ.%26atitle%3DA%2520road%2520map%2520to%2520personalizing%2520targeted%2520cancer%2520therapies%2520using%2520synthetic%2520lethality%26jtitle%3DTrends%2520Cancer%26date%3D2019%26volume%3D5%26spage%3D11%26epage%3D29%26doi%3D10.1016%2Fj.trecan.2018.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nandi, S.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality in DNA repair network: a novel avenue in targeted cancer therapy and combination therapeutics</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">929</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1002/iub.1696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2Fiub.1696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=29171189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKht7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2017&pages=929-937&author=S.+Bhattacharjeeauthor=S.+Nandi&title=Synthetic+lethality+in+DNA+repair+network%3A+a+novel+avenue+in+targeted+cancer+therapy+and+combination+therapeutics&doi=10.1002%2Fiub.1696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality in DNA repair network: A novel avenue in targeted cancer therapy and combination therapeutics</span></div><div class="casAuthors">Bhattacharjee, Sonali; Nandi, Saikat</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">929-937</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Synthetic lethality refers to a lethal phenotype that results from the simultaneous disruptions of two genes, while the disruption of either gene alone is viable.  Many DNA double strand break repair (DSBR) genes have synthetic lethal relationships with oncogenes and tumor suppressor genes, which can be exploited for targeted cancer therapy, an approach referred to as combination therapy.  DNA double-strand breaks (DSBs) are one of the most toxic lesions to a cell and can be repaired by non-homologous end joining (NHEJ) or homologous recombination (HR).  HR and NHEJ genes are particularly attractive targets for cancer therapy because these genes have altered expression patterns in cancer cells when compared with normal cells and these genetic abnormalities can be targeted for selectively killing cancer cells.  Here, we review recent advances in the development of small mol. inhibitors against HR and NHEJ genes to induce synthetic lethality and address the future directions and clin. relevance of this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB457gpMFejrVg90H21EOLACvtfcHk0lhqL7xFyro07w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKht7vK&md5=07083a1ed6b8e033c21e0c61e8fab4b2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fiub.1696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.1696%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharjee%26aufirst%3DS.%26aulast%3DNandi%26aufirst%3DS.%26atitle%3DSynthetic%2520lethality%2520in%2520DNA%2520repair%2520network%253A%2520a%2520novel%2520avenue%2520in%2520targeted%2520cancer%2520therapy%2520and%2520combination%2520therapeutics%26jtitle%3DIUBMB%2520Life%26date%3D2017%26volume%3D69%26spage%3D929%26epage%3D937%26doi%3D10.1002%2Fiub.1696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minchom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aversa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, J.</span></span> <span> </span><span class="NLM_article-title">Dancing with the DNA damage response: next-generation anti-cancer therapeutic strategies</span>. <i>Ther. Adv. Med. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1177/1758835918786658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1177%2F1758835918786658" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=1-18&author=A.+Minchomauthor=C.+Aversaauthor=J.+Lopez&title=Dancing+with+the+DNA+damage+response%3A+next-generation+anti-cancer+therapeutic+strategies&doi=10.1177%2F1758835918786658"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1177%2F1758835918786658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1758835918786658%26sid%3Dliteratum%253Aachs%26aulast%3DMinchom%26aufirst%3DA.%26aulast%3DAversa%26aufirst%3DC.%26aulast%3DLopez%26aufirst%3DJ.%26atitle%3DDancing%2520with%2520the%2520DNA%2520damage%2520response%253A%2520next-generation%2520anti-cancer%2520therapeutic%2520strategies%26jtitle%3DTher.%2520Adv.%2520Med.%2520Oncol.%26date%3D2018%26volume%3D10%26spage%3D1%26epage%3D18%26doi%3D10.1177%2F1758835918786658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pilie, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, T. A.</span></span> <span> </span><span class="NLM_article-title">State-of-the-art strategies for targeting the DNA damage response in cancer</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0114-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fs41571-018-0114-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30356138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSjtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=81-104&author=P.+G.+Pilieauthor=C.+Tangauthor=G.+B.+Millsauthor=T.+A.+Yap&title=State-of-the-art+strategies+for+targeting+the+DNA+damage+response+in+cancer&doi=10.1038%2Fs41571-018-0114-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">State-of-the-art strategies for targeting the DNA damage response in cancer</span></div><div class="casAuthors">Pilie, Patrick G.; Tang, Chad; Mills, Gordon B.; Yap, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-104</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Genomic instability is a key hallmark of cancer that arises owing to defects in the DNA damage response (DDR) and/or increased replication stress.  These alterations promote the clonal evolution of cancer cells via the accumulation of driver aberrations, including gene copy-no. changes, rearrangements and mutations; however, these same defects also create vulnerabilities that are relatively specific to cancer cells, which could potentially be exploited to increase the therapeutic index of anticancer treatments and thereby improve patient outcomes.  The discovery that BRCA-mutant cancer cells are exquisitely sensitive to inhibition of poly(ADP-ribose) polymerase has ushered in a new era of research on biomarker-driven synthetic lethal treatment strategies for different cancers.  The therapeutic landscape of antitumor agents targeting the DDR has rapidly expanded to include inhibitors of other key mediators of DNA repair and replication, such as ATM, ATR, CHK1 and CHK2, DNA-PK and WEE1.  Efforts to optimize these therapies are ongoing across a range of cancers, involving the development of predictive biomarker assays of responsiveness (beyond BRCA mutations), assessment of the mechanisms underlying intrinsic and acquired resistance, and evaluation of rational, tolerable combinations with std.-of-care treatments (such as chemotherapeutics and radiation), novel molecularly targeted agents and immune-checkpoint inhibitors.  In this Review, we discuss the current status of anticancer therapies targeting the DDR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpgsKJRZycYLVg90H21EOLACvtfcHk0ljkhs-NLz94_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSjtb%252FI&md5=f82199b7e32e11aabcad7cb664774e63</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0114-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0114-z%26sid%3Dliteratum%253Aachs%26aulast%3DPilie%26aufirst%3DP.%2BG.%26aulast%3DTang%26aufirst%3DC.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DYap%26aufirst%3DT.%2BA.%26atitle%3DState-of-the-art%2520strategies%2520for%2520targeting%2520the%2520DNA%2520damage%2520response%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D81%26epage%3D104%26doi%3D10.1038%2Fs41571-018-0114-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0ljkhs-NLz94_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deeks, E. D.</span></span> <span> </span><span class="NLM_article-title">Olaparib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1007/s40265-015-0345-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1007%2Fs40265-015-0345-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=25616434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVejtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=231-240&author=E.+D.+Deeks&title=Olaparib%3A+first+global+approval&doi=10.1007%2Fs40265-015-0345-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Olaparib: First Global Approval</span></div><div class="casAuthors">Deeks, Emma D.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-240</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Olaparib (Lynparza) is an oral, small mol., poly (ADP-ribose) polymerase inhibitor being developed by AstraZeneca for the treatment of solid tumors.  The primary indication that olaparib is being developed for is BRCA mutation-pos. ovarian cancer.  A capsule formulation of the drug has received approval for use in this setting in the EU and USA, and a tablet formulation is in global phase III trials (including in the USA, EU, Australia, Brazil, Canada, China, Israel, Japan, Russia and South Korea).  In addn., phase III trials in breast, gastric and pancreatic cancer are underway/planned, and phase I/II investigation is being conducted in other malignancies, including prostate cancer, non-small cell lung cancer, Ewing's sarcoma and advanced cancer.  This article summarizes the milestones in the development of olaparib leading to this first approval for ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4agVSDkOKqbVg90H21EOLACvtfcHk0ljkhs-NLz94_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVejtbg%253D&md5=cd68d932b6532a87c0768590afd81058</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs40265-015-0345-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-015-0345-6%26sid%3Dliteratum%253Aachs%26aulast%3DDeeks%26aufirst%3DE.%2BD.%26atitle%3DOlaparib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2015%26volume%3D75%26spage%3D231%26epage%3D240%26doi%3D10.1007%2Fs40265-015-0345-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><a href="http://www.fda.gov/Drugs/InformationOnDrugs/ApprovedDrugs/ucm592357.htm" class="extLink">www.fda.gov/Drugs/InformationOnDrugs/ApprovedDrugs/ucm592357.htm</a> (accessed Sep 15, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=www.fda.gov%2FDrugs%2FInformationOnDrugs%2FApprovedDrugs%2Fucm592357.htm+%28accessed+Sep+15%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cutsem, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarulla, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochhauser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinacher-Schick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortora, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algul, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGuinness, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlienger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locker, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, H. L.</span></span> <span> </span><span class="NLM_article-title">Maintenance olaparib for germline BRCA-mutated metastatic pancreatic cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1903387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1056%2FNEJMoa1903387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=31157963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnsFCqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2019&pages=317-327&author=T.+Golanauthor=P.+Hammelauthor=M.+Reniauthor=E.+Van+Cutsemauthor=T.+Macarullaauthor=M.+J.+Hallauthor=J.+Parkauthor=D.+Hochhauserauthor=D.+Arnoldauthor=D.+Ohauthor=A.+Reinacher-Schickauthor=G.+Tortoraauthor=H.+Algulauthor=E.+M.+O%E2%80%99Reillyauthor=D.+McGuinnessauthor=K.+Y.+Cuiauthor=K.+Schliengerauthor=G.+Y.+Lockerauthor=H.+L.+Kindler&title=Maintenance+olaparib+for+germline+BRCA-mutated+metastatic+pancreatic+cancer&doi=10.1056%2FNEJMoa1903387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Maintenance olaparib for germline BRCA-mutated metastatic pancreatic cancer</span></div><div class="casAuthors">Golan, Talia; Hammel, Pascal; Reni, Michele; Van Cutsem, Eric; Macarulla, Teresa; Hall, Michael J.; Park, Joon-Oh; Hochhauser, Daniel; Arnold, Dirk; Oh, Do-Youn; Reinacher-Schick, Anke; Tortora, Giampaolo; Algul, Hana; O'Reilly, Eileen M.; McGuinness, David; Cui, Karen Y.; Schlienger, Katia; Locker, Gershon Y.; Kindler, Hedy L.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">317-327</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background Patients with a germline BRCA1 or BRCA2 mutation make up a small subgroup of those with metastatic pancreatic cancer.  The poly(ADP-ribose) polymerase (PARP) inhibitor olaparib has had antitumor activity in this population. methods We conducted a randomized, double-blind, placebo-controlled, phase 3 trial to evaluate the efficacy of olaparib as maintenance therapy in patients who had a germline BRCA1 or BRCA2 mutation and metastatic pancreatic cancer and disease that had not progressed during first-line platinum-based chemotherapy.  Patients were randomly assigned, in a 3:2 ratio, to receive maintenance olaparib tablets (300 mg twice daily) or placebo.  The primary end point was progression-free survival, which was assessed by blinded independent central review. results Of the 3315 patients who underwent screening, 154 underwent randomization and were assigned to a trial intervention (92 to receive olaparib and 62 to receive placebo).  The median progression-free survival was significantly longer in the olaparib group than in the placebo group (7.4 mo vs. 3.8 mo; hazard ratio for disease progression or death, 0.53; 95% confidence interval [CI], 0.35 to 0.82; P = 0.004).  An interim anal. of overall survival, at a data maturity of 46%, showed no difference between the olaparib and placebo groups (median, 18.9 mo vs. 18.1 mo; hazard ratio for death, 0.91; 95% CI, 0.56 to 1.46; P = 0.68).  There was no significant between-group difference in health-related quality of life, as indicated by the overall change from baseline in the global quality-of-life score (on a 100-point scale, with higher scores indicating better quality of life) based on the European Organization for Research and Treatment of Cancer Quality of Life Questionnaire (between-group difference, -2.47 points; 95% CI, -7.27 to 2.33).  The incidence of grade 3 or higher adverse events was 40% in the olaparib group and 23% in the placebo group (between-group difference, 16 percentage points; 95% CI, -0.02 to 31); 5% and 2% of the patients, resp., discontinued the trial intervention because of an adverse event. conclusions Among patients with a germline BRCA mutation and metastatic pancreatic cancer, progression-free survival was longer with maintenance olaparib than with placebo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_mABSEPws9rVg90H21EOLACvtfcHk0lgdJH6s4X-yqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnsFCqtbo%253D&md5=06afa1aada240931f620ecf6f974b2e2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1903387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1903387%26sid%3Dliteratum%253Aachs%26aulast%3DGolan%26aufirst%3DT.%26aulast%3DHammel%26aufirst%3DP.%26aulast%3DReni%26aufirst%3DM.%26aulast%3DVan%2BCutsem%26aufirst%3DE.%26aulast%3DMacarulla%26aufirst%3DT.%26aulast%3DHall%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DHochhauser%26aufirst%3DD.%26aulast%3DArnold%26aufirst%3DD.%26aulast%3DOh%26aufirst%3DD.%26aulast%3DReinacher-Schick%26aufirst%3DA.%26aulast%3DTortora%26aufirst%3DG.%26aulast%3DAlgul%26aufirst%3DH.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DE.%2BM.%26aulast%3DMcGuinness%26aufirst%3DD.%26aulast%3DCui%26aufirst%3DK.%2BY.%26aulast%3DSchlienger%26aufirst%3DK.%26aulast%3DLocker%26aufirst%3DG.%2BY.%26aulast%3DKindler%26aufirst%3DH.%2BL.%26atitle%3DMaintenance%2520olaparib%2520for%2520germline%2520BRCA-mutated%2520metastatic%2520pancreatic%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2019%26volume%3D381%26spage%3D317%26epage%3D327%26doi%3D10.1056%2FNEJMoa1903387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIlwraith, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span> <span> </span><span class="NLM_article-title">Role of BRCA2 in control of the RAD51 recombination and DNA repair protein</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)00175-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2FS1097-2765%2801%2900175-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=11239456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=273-282&author=A.+A.+Daviesauthor=J.+Y.+Massonauthor=M.+J.+McIlwraithauthor=A.+Z.+Stasiakauthor=A.+Stasiakauthor=A.+R.+Venkitaramanauthor=S.+C.+West&title=Role+of+BRCA2+in+control+of+the+RAD51+recombination+and+DNA+repair+protein&doi=10.1016%2FS1097-2765%2801%2900175-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Role of BRCA2 in control of the RAD51 recombination and DNA repair protein</span></div><div class="casAuthors">Davies, Adelina A.; Masson, Jean-Yves; McIlwraith, Michael J.; Stasiak, Alicja Z.; Stasiak, Andrzej; Venkitaraman, Ashok R.; West, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-282</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Individuals carrying BRCA2 mutations are predisposed to breast and ovarian cancers.  Here, we show that BRCA2 plays a dual role in regulating the actions of RAD51, a protein essential for homologous recombination and DNA repair.  First, interactions between RAD51 and the BRC3 or BRC4 regions of BRCA2 block nucleoprotein filament formation by RAD51.  Alterations to the BRC3 region that mimic cancer-assocd. BRCA2 mutations fail to exhibit this effect.  Second, transport of RAD51 to the nucleus is defective in cells carrying a cancer-assocd. BRCA2 truncation.  Thus, BRCA2 regulates both the intracellular localization and DNA binding ability of RAD51.  Loss of these controls following BRCA2 inactivation may be a key event leading to genomic instability and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6admGLM9NrLVg90H21EOLACvtfcHk0lgdJH6s4X-yqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kitrc%253D&md5=9295ae3b5e51e58beeed4f9adb6e20e2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900175-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900175-7%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DA.%2BA.%26aulast%3DMasson%26aufirst%3DJ.%2BY.%26aulast%3DMcIlwraith%26aufirst%3DM.%2BJ.%26aulast%3DStasiak%26aufirst%3DA.%2BZ.%26aulast%3DStasiak%26aufirst%3DA.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DWest%26aufirst%3DS.%2BC.%26atitle%3DRole%2520of%2520BRCA2%2520in%2520control%2520of%2520the%2520RAD51%2520recombination%2520and%2520DNA%2520repair%2520protein%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D7%26spage%3D273%26epage%3D282%26doi%3D10.1016%2FS1097-2765%2801%2900175-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, H. L.</span></span> <span> </span><span class="NLM_article-title">The consequences of Rad51 overexpression for normal and tumor cells</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2007.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.dnarep.2007.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=18243065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1Sgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=686-693&author=H.+L.+Klein&title=The+consequences+of+Rad51+overexpression+for+normal+and+tumor+cells&doi=10.1016%2Fj.dnarep.2007.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The consequences of Rad51 overexpression for normal and tumor cells</span></div><div class="casAuthors">Klein, Hannah L.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">686-693</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The Rad51 recombinase is an essential factor for homologous recombination and the repair of DNA double strand breaks, binding transiently to both single stranded and double stranded DNA during the recombination reaction.  The use of a homologous recombination mechanism to repair DNA damage is controlled at several levels, including the binding of Rad51 to single stranded DNA to form the Rad51 nucleofilament, which is controlled through the action of DNA helicases that can counteract nucleofilament formation.  Overexpression of Rad51 in different organisms and cell types has a wide assortment of consequences, ranging from increased homologous recombination and increased resistance to DNA damaging agents to disruption of the cell cycle and apoptotic cell death.  Rad51 expression is increased in p53-neg. cells, and since p53 is often mutated in tumor cells, there is a tendency for Rad51 to be overexpressed in tumor cells, leading to increased resistance to DNA damage and drugs used in chemotherapies.  As cells with increased Rad51 levels are more resistant to DNA damage, there is a selection for tumor cells to have higher Rad51 levels.  While increased Rad51 can provide drug resistance, it also leads to increased genomic instability and may contribute to carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsGGiiU8otVLVg90H21EOLACvtfcHk0lgdJH6s4X-yqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1Sgsbw%253D&md5=824aee339ab1c58745af64778d24378a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2007.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2007.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DH.%2BL.%26atitle%3DThe%2520consequences%2520of%2520Rad51%2520overexpression%2520for%2520normal%2520and%2520tumor%2520cells%26jtitle%3DDNA%2520Repair%26date%3D2008%26volume%3D7%26spage%3D686%26epage%3D693%26doi%3D10.1016%2Fj.dnarep.2007.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henning, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzbecher, H. W.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination and cell cycle checkpoints: Rad51 in tumour progression and therapy resistance</span>. <i>Toxicology</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1016/S0300-483X(03)00291-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2FS0300-483X%2803%2900291-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=14599770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1KhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2003&pages=91-109&author=W.+Henningauthor=H.+W.+Sturzbecher&title=Homologous+recombination+and+cell+cycle+checkpoints%3A+Rad51+in+tumour+progression+and+therapy+resistance&doi=10.1016%2FS0300-483X%2803%2900291-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination and cell cycle checkpoints: Rad51 in tumor progression and therapy resistance</span></div><div class="casAuthors">Henning, Wilhelm; Sturzbecher, Horst-Werner</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">91-109</span>CODEN:
                <span class="NLM_cas:coden">TXCYAC</span>;
        ISSN:<span class="NLM_cas:issn">0300-483X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  We provide an overview of the functional interrelationship between genes and proteins related to DNA repair by homologous recombination and cell cycle regulation in relation to the progression and therapy resistance of human tumors.  To ensure the high-fidelity transmission of genetic information from one generation to the next, cells have evolved mechanisms to monitor genome integrity.  Upon DNA damage, cells initiate complex response pathways including cell cycle arrest, activation of genes and gene products involved in DNA repair, and under some circumstances, the triggering of programmed cell death.  Deregulation of this coordinated response leads to genetic instability and is fundamental to the etiol. of human cancer.  Homologous recombination involved in DNA repair is induced by environmental damage as well as misreplication during the normal cell cycle.  However, when not regulated properly, it can result in the loss of heterozygosity or genetic rearrangements, central to the process of carcinogenesis.  The central step of homologous recombination is the DNA strand exchange reaction catalyzed by the eukaryotic Rad51 protein.  Here, we describe the recent progress in our understanding of how Rad51 is involved in the signaling and repair of DNA damage and how tumor suppressors, such as p53, ATM, BRCA1, BRCA2, BLM and FANCD2 are linked to Rad51-dependent pathways.  An increased knowledge of the role of Rad51 in DNA repair by homologous recombination and its effects on cell cycle progression, tumor development and tumor resistance may provide opportunities for identifying improved diagnostic markers and developing more effective treatments for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDyHSTW9TAOrVg90H21EOLACvtfcHk0li2NMiIQAiVfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1KhsLc%253D&md5=b2436c287b5ee3537ea7837f4707f04f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0300-483X%2803%2900291-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0300-483X%252803%252900291-9%26sid%3Dliteratum%253Aachs%26aulast%3DHenning%26aufirst%3DW.%26aulast%3DSturzbecher%26aufirst%3DH.%2BW.%26atitle%3DHomologous%2520recombination%2520and%2520cell%2520cycle%2520checkpoints%253A%2520Rad51%2520in%2520tumour%2520progression%2520and%2520therapy%2520resistance%26jtitle%3DToxicology%26date%3D2003%26volume%3D193%26spage%3D91%26epage%3D109%26doi%3D10.1016%2FS0300-483X%2803%2900291-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yanagisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuyama, J.</span></span> <span> </span><span class="NLM_article-title">Increased expression of human DNA repair genes, XRCC1, XRCC3 and RAD51, in radioresistant human KB carcinoma cell line N10</span>. <i>Oral Oncol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1016/S1368-8375(98)00045-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2FS1368-8375%2898%2900045-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9930366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK1MXitVSrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1998&pages=524-528&author=T.+Yanagisawaauthor=M.+Uradeauthor=Y.+Yamamotoauthor=J.+Furuyama&title=Increased+expression+of+human+DNA+repair+genes%2C+XRCC1%2C+XRCC3+and+RAD51%2C+in+radioresistant+human+KB+carcinoma+cell+line+N10&doi=10.1016%2FS1368-8375%2898%2900045-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Increased expression of human DNA repair genes, XRCC1, XRCC3 and RAD51, in radioresistant human KB carcinoma cell line N10</span></div><div class="casAuthors">Yanagisawa, T.; Urade, M.; Yamamoto, Y.; Furuyama, J.</div><div class="citationInfo"><span class="NLM_cas:title">Oral Oncology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">EJCCER</span>;
        ISSN:<span class="NLM_cas:issn">0964-1955</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The radioresistant N10 and parental KB cell lines were examd. for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot anal. using five kinds of DNA probes (XRCC1, XRCC3, XRCC5, RAD51, RAD52).  In the unirradiated condition, N10 cells showed higher expression of XRCC1, XRCC3 and RAD51 mRNA than did KB cells.  The X-irradn. induced a time-dependent increase in the mRNA levels of XRCC3 and RAD51 in both cell lines with a max. at 2 h postirradn.  The XRCC1 mRNA in N10 was maintained at the same level even after irradn., whereas that in KB was decreased after irradn.  There was no difference in the expression of XRCC5 and RAD52 mRNA between N10 and KB cells in both unirradiated and irradiated conditions.  From these findings, it was suggested that XRCC1, XRCC3 and RAD51 contribute to the radioresistance in cell line N10.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSOn6VPpumMLVg90H21EOLACvtfcHk0lhtzHpl_k6yxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitVSrsw%253D%253D&md5=0446ed646f85ef5372de148ab1123a2c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS1368-8375%2898%2900045-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1368-8375%252898%252900045-1%26sid%3Dliteratum%253Aachs%26aulast%3DYanagisawa%26aufirst%3DT.%26aulast%3DUrade%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DY.%26aulast%3DFuruyama%26aufirst%3DJ.%26atitle%3DIncreased%2520expression%2520of%2520human%2520DNA%2520repair%2520genes%252C%2520XRCC1%252C%2520XRCC3%2520and%2520RAD51%252C%2520in%2520radioresistant%2520human%2520KB%2520carcinoma%2520cell%2520line%2520N10%26jtitle%3DOral%2520Oncol.%26date%3D1998%26volume%3D34%26spage%3D524%26epage%3D528%26doi%3D10.1016%2FS1368-8375%2898%2900045-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span> <span> </span><span class="NLM_article-title">Insights into DNA recombination from the structure of a RAD51-BRCA2 complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>420</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/nature01230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnature01230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=12442171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD38XovVektLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=420&publication_year=2002&pages=287-293&author=L.+Pellegriniauthor=D.+S.+Yuauthor=T.+Loauthor=S.+Anandauthor=M.+Leeauthor=T.+M.+Blundellauthor=A.+R.+Venkitaraman&title=Insights+into+DNA+recombination+from+the+structure+of+a+RAD51-BRCA2+complex&doi=10.1038%2Fnature01230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into DNA recombination from the structure of a RAD51-BRCA2 complex</span></div><div class="casAuthors">Pellegrini, Luca; Yu, David S.; Lo, Thomas; Anand, Shubha; Lee, MiYoung; Blundell, Tom L.; Venkitaraman, Ashok R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">420</span>
        (<span class="NLM_cas:issue">6913</span>),
    <span class="NLM_cas:pages">287-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The breast cancer susceptibility protein BRCA2 controls the function of RAD51, a recombinase enzyme, in pathways for DNA repair by homologous recombination.  We report here the structure of a complex between an evolutionarily conserved sequence in BRCA2 (the BRC repeat) and the RecA-homol. domain of RAD51.  The BRC repeat mimics a motif in RAD51 that serves as an interface for oligomerization between individual RAD51 monomers, thus enabling BRCA2 to control the assembly of the RAD51 nucleoprotein filament, which is essential for strand-pairing reactions during DNA recombination.  The RAD51 oligomerization motif is highly conserved among RecA-like recombinases, highlighting a common evolutionary origin for the mechanism of nucleoprotein filament formation, mirrored in the BRC repeat.  Cancer-assocd. mutations that affect the BRC repeat disrupt its predicted interaction with RAD51, yielding structural insight into mechanisms for cancer susceptibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo44KTdJmBb5rVg90H21EOLACvtfcHk0lhtzHpl_k6yxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovVektLY%253D&md5=5dc084f8efb75cf95bc31d4f5b1797e3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature01230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01230%26sid%3Dliteratum%253Aachs%26aulast%3DPellegrini%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DD.%2BS.%26aulast%3DLo%26aufirst%3DT.%26aulast%3DAnand%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DBlundell%26aufirst%3DT.%2BM.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26atitle%3DInsights%2520into%2520DNA%2520recombination%2520from%2520the%2520structure%2520of%2520a%2520RAD51-BRCA2%2520complex%26jtitle%3DNature%26date%3D2002%26volume%3D420%26spage%3D287%26epage%3D293%26doi%3D10.1038%2Fnature01230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span> <span> </span><span class="NLM_article-title">Internal repeats in the BRCA2 protein sequence</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1038/ng0596-22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fng0596-22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=8673099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK28XksVWgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1996&pages=22-23&author=P.+Borkauthor=N.+Blombergauthor=M.+Nilges&title=Internal+repeats+in+the+BRCA2+protein+sequence&doi=10.1038%2Fng0596-22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Internal repeats in the BRCA2 protein sequence</span></div><div class="casAuthors">Bork, Peer; Blomberg, Niklas; Nilges, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-23</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">Eight repetitive units were found in the breast cancer susceptibility gene BRCA2 protein sequence.  The 8 internal repeats (BRC repeats) are located in the central portion of the protein and cover nearly a third of the protein; all repeats are encoded by the large exon 11.  The length of a minimal BRC domain should not exceed 80 amino acids, and at least 4 secondary structure elements are present in a single repeat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxD_rMsWQKAbVg90H21EOLACvtfcHk0lhtzHpl_k6yxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XksVWgtbw%253D&md5=4bc5b248f589f7bb9a1c6018d45ecc17</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fng0596-22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng0596-22%26sid%3Dliteratum%253Aachs%26aulast%3DBork%26aufirst%3DP.%26aulast%3DBlomberg%26aufirst%3DN.%26aulast%3DNilges%26aufirst%3DM.%26atitle%3DInternal%2520repeats%2520in%2520the%2520BRCA2%2520protein%2520sequence%26jtitle%3DNat.%2520Genet.%26date%3D1996%26volume%3D13%26spage%3D22%26epage%3D23%26doi%3D10.1038%2Fng0596-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ormonde, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavtigian, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, P. L.</span></span> <span> </span><span class="NLM_article-title">RAD51 interacts with the evolutionarily conserved BRC motifs in the human breast cancer susceptibility gene brca2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">31941</span>– <span class="NLM_lpage">31944</span>, <span class="refDoi"> DOI: 10.1074/jbc.272.51.31941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1074%2Fjbc.272.51.31941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9405383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK1cXis1CmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=31941-31944&author=A.+K.+Wongauthor=R.+Peroauthor=P.+A.+Ormondeauthor=S.+V.+Tavtigianauthor=P.+L.+Bartel&title=RAD51+interacts+with+the+evolutionarily+conserved+BRC+motifs+in+the+human+breast+cancer+susceptibility+gene+brca2&doi=10.1074%2Fjbc.272.51.31941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">RAD51 interacts with the evolutionarily conserved BRC motifs in the human breast cancer susceptibility gene brca2</span></div><div class="casAuthors">Wong, Alexander K. C.; Pero, Ralph; Ormonde, Patricia A.; Tavtigian, Sean V.; Bartel, Paul L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">31941-31944</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Recent work has shown that the murine BRCA2 tumor suppressor protein interacts with the murine RAD51 protein.  This interaction suggests that BRCA2 participates in DNA repair.  Residues 3196-3232 of the murine BRCA2 protein were shown to be involved in this interaction.  Here, we report the detailed mapping of addnl. domains that are involved in interactions between the human homologs of these two proteins.  Through yeast two-hybrid and biochem. assays, we demonstrate that the RAD51 protein interacts specifically with the eight evolutionarily conserved BRC motifs encoded in exon 11 of brca2 and with a similar motif found in a Caenorhabditis elegans hypothetical protein.  Deletion anal. demonstrates that residues 98-339 of human RAD51 interact with the 59-residue minimal region that is conserved in all BRC motifs.  These data suggest that the BRC repeats function to bind RAD51.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_ZSZn6x5YvrVg90H21EOLACvtfcHk0lhtzHpl_k6yxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1CmtA%253D%253D&md5=5fef058fdd28151371930a354e30af86</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.51.31941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.51.31941%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DA.%2BK.%26aulast%3DPero%26aufirst%3DR.%26aulast%3DOrmonde%26aufirst%3DP.%2BA.%26aulast%3DTavtigian%26aufirst%3DS.%2BV.%26aulast%3DBartel%26aufirst%3DP.%2BL.%26atitle%3DRAD51%2520interacts%2520with%2520the%2520evolutionarily%2520conserved%2520BRC%2520motifs%2520in%2520the%2520human%2520breast%2520cancer%2520susceptibility%2520gene%2520brca2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D31941%26epage%3D31944%26doi%3D10.1074%2Fjbc.272.51.31941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bignell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micklem, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooster, R.</span></span> <span> </span><span class="NLM_article-title">The BRC repeats are conserved in mammalian BRCA2 proteins</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1093/hmg/6.1.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fhmg%2F6.1.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9002670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK2sXlsleqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=53-58&author=G.+Bignellauthor=G.+Micklemauthor=M.+R.+Strattonauthor=A.+Ashworthauthor=R.+Wooster&title=The+BRC+repeats+are+conserved+in+mammalian+BRCA2+proteins&doi=10.1093%2Fhmg%2F6.1.53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The BRC repeats are conserved in mammalian BRCA2 proteins</span></div><div class="casAuthors">Bignell, Graham; Micklem, Gos; Stratton, Michael R.; Ashworth, Alan; Wooster, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-58</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The breast cancer susceptibility gene BRCA2 encodes a protein of 3418 amino acids which does not exhibit substantial sequence similarity to any other protein in the public databases.  A dot matrix comparison of BRCA2 with itself revealed an eight times repeated motif in the segment of the protein encoded by exon 11.  As a preliminary test of the hypothesis that these motifs are functionally significant, we have sequenced exon 11 of BRCA2 in six mammals.  An alignment of the predicted protein sequences shows that, overall, the motifs have been conserved while much of the intervening sequences has diverged.  These data support the notion that the BRC motifs are important in BRCA2 function.  There is, however, considerable interspecies variation within certain motif units, raising the possibility of redundancy and that not all of the repeats are required for the normal function of BRCA2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIqJmZz4B52LVg90H21EOLACvtfcHk0ljky-5RZPZN2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlsleqsw%253D%253D&md5=6fdd1ddf4fcc0430f32e8c2b6c8ab9ca</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F6.1.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F6.1.53%26sid%3Dliteratum%253Aachs%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DMicklem%26aufirst%3DG.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DWooster%26aufirst%3DR.%26atitle%3DThe%2520BRC%2520repeats%2520are%2520conserved%2520in%2520mammalian%2520BRCA2%2520proteins%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D1997%26volume%3D6%26spage%3D53%26epage%3D58%26doi%3D10.1093%2Fhmg%2F6.1.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldblatt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span> <span> </span><span class="NLM_article-title">A novel small molecule RAD51 inactivator overcomes imatinib-resistance in chronic myeloid leukaemia</span>. <i>EMBO Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1002/emmm.201201760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2Femmm.201201760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=23341130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsFKhtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=353-365&author=J.+Zhuauthor=L.+Zhouauthor=G.+Wuauthor=H.+Konigauthor=X.+Linauthor=G.+Liauthor=X.+Qiuauthor=C.+Chenauthor=C.+Huauthor=E.+Goldblattauthor=R.+Bhatiaauthor=A.+R.+Chamberlinauthor=P.+Chenauthor=W.+Lee&title=A+novel+small+molecule+RAD51+inactivator+overcomes+imatinib-resistance+in+chronic+myeloid+leukaemia&doi=10.1002%2Femmm.201201760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A novel small molecule RAD51 inactivator overcomes imatinib-resistance in chronic myeloid leukaemia</span></div><div class="casAuthors">Zhu, Jiewen; Zhou, Longen; Wu, Guikai; Konig, Heiko; Lin, Xiaoqin; Li, Guideng; Qiu, Xiao-Long; Chen, Chi-Fen; Hu, Chun-Mei; Goldblatt, Erin; Bhatia, Ravi; Chamberlin, A. Richard; Chen, Phang-Lang; Lee, Wen-Hwa</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-365</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">RAD51 recombinase activity plays a crit. role for cancer cell proliferation and survival, and often contributes to drug-resistance.  Abnormally elevated RAD51 function and hyperactive homologous recombination (HR) rates have been found in a panel of cancers, including breast cancer and chronic myeloid leukemia (CML).  Directly targeting RAD51 and attenuating the deregulated RAD51 activity has therefore been proposed as an alternative and supplementary strategy for cancer treatment.  Here we show that a newly identified small mol., IBR2, disrupts RAD51 multimerization, accelerates proteasome-mediated RAD51 protein degrdn., reduces ionizing radiation-induced RAD51 foci formation, impairs HR, inhibits cancer cell growth and induces apoptosis.  In a murine imatinib-resistant CML model bearing the T315I Bcr-abl mutation, IBR2, but not imatinib, significantly prolonged animal survival.  Moreover, IBR2 effectively inhibits the proliferation of CD34+ progenitor cells from CML patients resistant to known BCR-ABL inhibitors.  Therefore, small mol. inhibitors of RAD51 may suggest a novel class of broad-spectrum therapeutics for difficult-to-treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwQGeSwZ19nrVg90H21EOLACvtfcHk0ljky-5RZPZN2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsFKhtr8%253D&md5=e956911425043bd5e2f18ddc0c10eec7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Femmm.201201760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Femmm.201201760%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DKonig%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DGoldblatt%26aufirst%3DE.%26aulast%3DBhatia%26aufirst%3DR.%26aulast%3DChamberlin%26aufirst%3DA.%2BR.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DW.%26atitle%3DA%2520novel%2520small%2520molecule%2520RAD51%2520inactivator%2520overcomes%2520imatinib-resistance%2520in%2520chronic%2520myeloid%2520leukaemia%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2013%26volume%3D5%26spage%3D353%26epage%3D365%26doi%3D10.1002%2Femmm.201201760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span>; <span class="NLM_string-name">Chen, P. L.</span>; <span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Composition and Methods for the Disruption of BRCA2-RAD51 Interaction</span>. PCT Int. Appl. <span class="NLM_patent">WO 2006/044933</span>, <span class="NLM_year">2006</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=W.+H.+Lee&author=P.+L.+Chen&author=J.+Zhu&title=Composition+and+Methods+for+the+Disruption+of+BRCA2-RAD51+Interaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%2BH.%26atitle%3DComposition%2520and%2520Methods%2520for%2520the%2520Disruption%2520of%2520BRCA2-RAD51%2520Interaction%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Ianni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manerba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farabegoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallavicini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Stefano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality triggered by combining olaparib with BRCA2-Rad51 disruptors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2491</span>– <span class="NLM_lpage">2497</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00707</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00707" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlymt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2491-2497&author=F.+Falchiauthor=E.+Giacominiauthor=T.+Masiniauthor=N.+Boutardauthor=L.+Di+Ianniauthor=M.+Manerbaauthor=F.+Farabegoliauthor=L.+Rossiniauthor=J.+Robertsonauthor=S.+Minucciauthor=I.+Pallaviciniauthor=G.+Di+Stefanoauthor=M.+Robertiauthor=R.+Pellicciariauthor=A.+Cavalli&title=Synthetic+lethality+triggered+by+combining+olaparib+with+BRCA2-Rad51+disruptors&doi=10.1021%2Facschembio.7b00707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethality Triggered by Combining Olaparib with BRCA2-Rad51 Disruptors</span></div><div class="casAuthors">Falchi, Federico; Giacomini, Elisa; Masini, Tiziana; Boutard, Nicolas; Di Ianni, Lorenza; Manerba, Marcella; Farabegoli, Fulvia; Rossini, Lara; Robertson, Janet; Minucci, Saverio; Pallavicini, Isabella; Di Stefano, Giuseppina; Roberti, Marinella; Pellicciari, Roberto; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2491-2497</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In BRCA2-defective cells, poly(ADP [ADP]-ribose) polymerase inhibitors can trigger synthetic lethality, as two independent DNA-repairing mechanisms are simultaneously impaired.  Here, we have pharmacol. induced synthetic lethality, which was triggered by combining two different small org. mols.  When administered with a BRCA2-Rad51 disruptor in nonmutant cells, Olaparib showed anticancer activity comparable to that shown when administered alone in BRCA2-defective cells.  This strategy could represent an innovative approach to anticancer drug discovery and could be extended to other synthetic lethality pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh9-yqLe986rVg90H21EOLACvtfcHk0ljTjZtOAxS-AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlymt7fF&md5=f2196fb93d0e86d8aa4f090e0434afc4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00707%26sid%3Dliteratum%253Aachs%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DGiacomini%26aufirst%3DE.%26aulast%3DMasini%26aufirst%3DT.%26aulast%3DBoutard%26aufirst%3DN.%26aulast%3DDi%2BIanni%26aufirst%3DL.%26aulast%3DManerba%26aufirst%3DM.%26aulast%3DFarabegoli%26aufirst%3DF.%26aulast%3DRossini%26aufirst%3DL.%26aulast%3DRobertson%26aufirst%3DJ.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DDi%2BStefano%26aufirst%3DG.%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSynthetic%2520lethality%2520triggered%2520by%2520combining%2520olaparib%2520with%2520BRCA2-Rad51%2520disruptors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2491%26epage%3D2497%26doi%3D10.1021%2Facschembio.7b00707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagnolini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balboni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manerba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farabegoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Franco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallavicini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Stefano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Rad51/BRCA2 disruptors inhibit homologous recombination and synergize with olaparib in pancreatic cancer cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.ejmech.2019.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30660828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVOku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2019&pages=80-92&author=M.+Robertiauthor=F.+Schipaniauthor=G.+Bagnoliniauthor=D.+Milanoauthor=E.+Giacominiauthor=F.+Falchiauthor=A.+Balboniauthor=M.+Manerbaauthor=F.+Farabegoliauthor=F.+De+Francoauthor=J.+Robertsonauthor=S.+Minucciauthor=I.+Pallaviciniauthor=G.+Di+Stefanoauthor=S.+Girottoauthor=R.+Pellicciariauthor=A.+Cavalli&title=Rad51%2FBRCA2+disruptors+inhibit+homologous+recombination+and+synergize+with+olaparib+in+pancreatic+cancer+cells&doi=10.1016%2Fj.ejmech.2019.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rad51/BRCA2 disruptors inhibit homologous recombination and synergize with olaparib in pancreatic cancer cells</span></div><div class="casAuthors">Roberti, Marinella; Schipani, Fabrizio; Bagnolini, Greta; Milano, Domenico; Giacomini, Elisa; Falchi, Federico; Balboni, Andrea; Manerba, Marcella; Farabegoli, Fulvia; De Franco, Francesca; Robertson, Janet; Minucci, Saverio; Pallavicini, Isabella; Di Stefano, Giuseppina; Girotto, Stefania; Pellicciari, Roberto; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-92</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Olaparib is a PARP inhibitor (PARPi).  For patients bearing BRCA1 or BRCA2 mutations, olaparib is approved to treat ovarian cancer and in clin. trials to treat breast and pancreatic cancers.  In BRCA2-defective patients, PARPi inhibits DNA single-strand break repair, while BRCA2 mutations hamper double-strand break repair.  Recently, the authors identified a series of triazole derivs. that mimic BRCA2 mutations by disrupting the Rad51-BRCA2 interaction and thus double-strand break repair.  Here, the authors have computationally designed, synthesized, and tested over 40 novel derivs.  Addnl., they designed and conducted novel biol. assays to characterize how they disrupt the Rad51-BRCA2 interaction and inhibit double-strand break repair.  These compds., synergized with olaparib to target pancreatic cancer cells with functional BRCA2.  Compd. I significantly increased olaparib's effect on the viability of BxPC-3 cells.  This supports the idea that small org. mols. can mimic genetic mutations to improve the profile of anticancer drugs for precision medicine.  Moreover, this paradigm could be exploited in other genetic pathways to discover innovative anticancer targets and drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO64ucZDf8MLVg90H21EOLACvtfcHk0ljTjZtOAxS-AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVOku7k%253D&md5=3090401197babb650036d569a0195a7a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DSchipani%26aufirst%3DF.%26aulast%3DBagnolini%26aufirst%3DG.%26aulast%3DMilano%26aufirst%3DD.%26aulast%3DGiacomini%26aufirst%3DE.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DBalboni%26aufirst%3DA.%26aulast%3DManerba%26aufirst%3DM.%26aulast%3DFarabegoli%26aufirst%3DF.%26aulast%3DDe%2BFranco%26aufirst%3DF.%26aulast%3DRobertson%26aufirst%3DJ.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DDi%2BStefano%26aufirst%3DG.%26aulast%3DGirotto%26aufirst%3DS.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRad51%252FBRCA2%2520disruptors%2520inhibit%2520homologous%2520recombination%2520and%2520synergize%2520with%2520olaparib%2520in%2520pancreatic%2520cancer%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D165%26spage%3D80%26epage%3D92%26doi%3D10.1016%2Fj.ejmech.2019.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein-protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0ljTjZtOAxS-AA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein-protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajendra, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span> <span> </span><span class="NLM_article-title">Two modules in the BRC repeats of BRCA2 mediate structural and functional interactions with the RAD51 recombinase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fnar%2Fgkp873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=19875419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2nuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=82-96&author=E.+Rajendraauthor=A.+R.+Venkitaraman&title=Two+modules+in+the+BRC+repeats+of+BRCA2+mediate+structural+and+functional+interactions+with+the+RAD51+recombinase&doi=10.1093%2Fnar%2Fgkp873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Two modules in the BRC repeats of BRCA2 mediate structural and functional interactions with the RAD51 recombinase</span></div><div class="casAuthors">Rajendra, Eeson; Venkitaraman, Ashok R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-96</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The breast and ovarian cancer suppressor protein BRCA2 controls the RAD51 recombinase in reactions that lead to homologous DNA recombination (HDR).  BRCA2 binds RAD51 via eight conserved BRC repeat motifs of approx. 35 amino acids, each with a varying capacity to bind RAD51.  BRC repeats both promote and inhibit RAD51 assembly on different DNA substrates to regulate HDR, but the structural basis for these functions is unclear.  Here, we demarcate two tetrameric clusters of hydrophobic residues in the BRC repeats, interacting with distinct pockets in RAD51, and show that the co-location of both modules within a single BRC repeat is necessary for BRC-RAD51 binding and function.  The two modules comprise the sequence FxxA, known to inhibit RAD51 assembly by blocking the oligomerization interface, and a previously unrecognized tetramer with the consensus sequence LFDE, which binds to a RAD51 pocket distinct from this interface.  The LFDE motif is essential in BRC repeats for modes of RAD51 binding both permissive and inhibitory to RAD51 oligomerization.  Targeted insertion of point mutations in RAD51 that disrupt the LFDE-binding pocket impair its assembly at DNA damage sites in living cells.  Our findings suggest a model for the modular architecture of BRC repeats that provides fresh insight into the mechanisms regulating homologous DNA recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl7J1JIDpuErVg90H21EOLACvtfcHk0liAgiVc2G04NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2nuw%253D%253D&md5=8fec23291c3037866be93b27f8a6f685</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp873%26sid%3Dliteratum%253Aachs%26aulast%3DRajendra%26aufirst%3DE.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26atitle%3DTwo%2520modules%2520in%2520the%2520BRC%2520repeats%2520of%2520BRCA2%2520mediate%2520structural%2520and%2520functional%2520interactions%2520with%2520the%2520RAD51%2520recombinase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D82%26epage%3D96%26doi%3D10.1093%2Fnar%2Fgkp873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomme, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renodon-Corniere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleury, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumizaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of filament formation of human Rad51 protein by a small peptide derived from the BRC-motif of the BRCA2 protein</span>. <i>Genes Cells</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2443.2008.01180.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1111%2Fj.1365-2443.2008.01180.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=18429819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1SgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=471-481&author=J.+Nommeauthor=Y.+Takizawaauthor=S.+F.+Martinezauthor=A.+Renodon-Corniereauthor=F.+Fleuryauthor=P.+Weigelauthor=K.+Yamamotoauthor=H.+Kurumizakaauthor=M.+Takahashi&title=Inhibition+of+filament+formation+of+human+Rad51+protein+by+a+small+peptide+derived+from+the+BRC-motif+of+the+BRCA2+protein&doi=10.1111%2Fj.1365-2443.2008.01180.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of filament formation of human Rad51 protein by a small peptide derived from the BRC-motif of the BRCA2 protein</span></div><div class="casAuthors">Nomme, Julian; Takizawa, Yoshimasa; Martinez, Susan F.; Renodon-Corniere, Axelle; Fleury, Fabrice; Weigel, Pierre; Yamamoto, Ken-ichi; Kurumizaka, Hitoshi; Takahashi, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-481</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Inc.</span>)
        </div><div class="casAbstract">Human Rad51 is a key element of recombinational DNA repair and is related to the resistance of cancer cells to chemo- and radiotherapies.  The protein is thus a potential target of anti-cancer treatment.  The crystallog. anal. shows that the BRC-motif of the BRCA2 tumor suppressor is in contact with the subunit-subunit interface of Rad51 and could thus prevent filament formation of Rad51.  However, biochem. anal. indicates that a BRC-motif peptide of 69 amino acids preferentially binds to the N-terminal part of Rad51.  We show exptl. that a short peptide of 28 amino acids derived from the BRC4 motif binds to the subunit-subunit interface and dissocs. its filament, both in the presence and absence of DNA, certainly by binding to dissocd. monomers.  The inhibition is efficient and specific for Rad51: the peptide does not even interact with Rad51 homologs or prevent their interaction with DNA.  Neither the N-terminal nor the C-terminal half of the peptide interacts with human Rad51, indicating that both parts are involved in the interaction, as expected from the crystal structure.  These results suggest the possibility of developing inhibitors of human Rad51 based on this peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4OeZl8N7HpLVg90H21EOLACvtfcHk0liAgiVc2G04NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1SgtLc%253D&md5=9a0525500b096f871c6ad7ffcf9d9a3e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2443.2008.01180.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2443.2008.01180.x%26sid%3Dliteratum%253Aachs%26aulast%3DNomme%26aufirst%3DJ.%26aulast%3DTakizawa%26aufirst%3DY.%26aulast%3DMartinez%26aufirst%3DS.%2BF.%26aulast%3DRenodon-Corniere%26aufirst%3DA.%26aulast%3DFleury%26aufirst%3DF.%26aulast%3DWeigel%26aufirst%3DP.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DKurumizaka%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520filament%2520formation%2520of%2520human%2520Rad51%2520protein%2520by%2520a%2520small%2520peptide%2520derived%2520from%2520the%2520BRC-motif%2520of%2520the%2520BRCA2%2520protein%26jtitle%3DGenes%2520Cells%26date%3D2008%26volume%3D13%26spage%3D471%26epage%3D481%26doi%3D10.1111%2Fj.1365-2443.2008.01180.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomme, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renodon-Corniere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asanomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span> <span> </span><span class="NLM_article-title">Design of potent inhibitors of human RAD51 recombinase based on BRC motifs of BRCA2 protein: modeling and experimental validation of a chimera peptide</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5782</span>– <span class="NLM_lpage">5791</span>, <span class="refDoi"> DOI: 10.1021/jm1002974</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1002974" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ersro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5782-5791&author=J.+Nommeauthor=A.+Renodon-Corniereauthor=Y.+Asanomiauthor=K.+Sakaguchiauthor=A.+Z.+Stasiakauthor=A.+Stasiakauthor=B.+Nordenauthor=V.+Tranauthor=M.+Takahashi&title=Design+of+potent+inhibitors+of+human+RAD51+recombinase+based+on+BRC+motifs+of+BRCA2+protein%3A+modeling+and+experimental+validation+of+a+chimera+peptide&doi=10.1021%2Fjm1002974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Potent Inhibitors of Human RAD51 Recombinase Based on BRC Motifs of BRCA2 Protein: Modeling and Experimental Validation of a Chimera Peptide</span></div><div class="casAuthors">Nomme, Julian; Renodon-Corniere, Axelle; Asanomi, Yuya; Sakaguchi, Kazuyasu; Stasiak, Alicja Z.; Stasiak, Andrzej; Norden, Bengt; Tran, Vinh; Takahashi, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5782-5791</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have previously shown that a 28-amino acid peptide derived from the BRC4 motif of BRCA2 tumor suppressor inhibits selectively human RAD51 recombinase (HsRad51).  With the aim of designing better inhibitors for cancer treatment, we combined an in silico docking approach with in vitro biochem. testing to construct a highly efficient chimera peptide from eight existing human BRC motifs.  We built a mol. model of all BRC motifs complexed with HsRad51 based on the crystal structure of the BRC4 motif-HsRad51 complex, computed the interaction energy of each residue in each BRC motif, and selected the best amino acid residue at each binding position.  This anal. enabled us to propose four amino acid substitutions in the BRC4 motif.  Three of these increased the inhibitory effect in vitro, and this effect was found to be additive.  We thus obtained a peptide that is about 10 times more efficient in inhibiting HsRad51-ssDNA complex formation than the original peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBdxPy1TGL2rVg90H21EOLACvtfcHk0ljI4TOmN0xxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ersro%253D&md5=3b6a3ee9962a5dc4966e5dad9653835d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm1002974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1002974%26sid%3Dliteratum%253Aachs%26aulast%3DNomme%26aufirst%3DJ.%26aulast%3DRenodon-Corniere%26aufirst%3DA.%26aulast%3DAsanomi%26aufirst%3DY.%26aulast%3DSakaguchi%26aufirst%3DK.%26aulast%3DStasiak%26aufirst%3DA.%2BZ.%26aulast%3DStasiak%26aufirst%3DA.%26aulast%3DNorden%26aufirst%3DB.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DTakahashi%26aufirst%3DM.%26atitle%3DDesign%2520of%2520potent%2520inhibitors%2520of%2520human%2520RAD51%2520recombinase%2520based%2520on%2520BRC%2520motifs%2520of%2520BRCA2%2520protein%253A%2520modeling%2520and%2520experimental%2520validation%2520of%2520a%2520chimera%2520peptide%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5782%26epage%3D5791%26doi%3D10.1021%2Fjm1002974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acker, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slabber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacsa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynelis, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">liotta, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships and pharmacophore model of a noncompetitive pyrazoline containing class of GluN2C/GluN2D selective antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6434</span>– <span class="NLM_lpage">6456</span>, <span class="refDoi"> DOI: 10.1021/jm400652r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400652r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Slsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6434-6456&author=T.+M.+Ackerauthor=A.+Khatriauthor=K.+M.+Vanceauthor=C.+Slabberauthor=J.+Bacsaauthor=J.+P.+Snyderauthor=S.+F.+Traynelisauthor=D.+C.+liotta&title=Structure-activity+relationships+and+pharmacophore+model+of+a+noncompetitive+pyrazoline+containing+class+of+GluN2C%2FGluN2D+selective+antagonists&doi=10.1021%2Fjm400652r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships and Pharmacophore Model of a Noncompetitive Pyrazoline Containing Class of GluN2C/GluN2D Selective Antagonists</span></div><div class="casAuthors">Acker, Timothy M.; Khatri, Alpa; Vance, Katie M.; Slabber, Cathryn; Bacsa, John; Snyder, James P.; Traynelis, Stephen F.; Liotta, Dennis C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6434-6456</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we describe the synthesis and structure-activity relationship for a class of pyrazoline-contg. dihydroquinolone neg. allosteric modulators (I) of the NMDA receptor that show strong subunit selectivity for GluN2C- and GluN2D-contg. receptors over GluN2A- and GluN2B-contg. receptors.  Several members of this class inhibit NMDA receptor responses in the nanomolar range and are more than 50-fold selective over GluN1/GluN2A and GluN1/GluN2B NMDA receptors, as well as AMPA, kainate, GABA, glycine, nicotinic, serotonin, and purinergic receptors.  Anal. of the purified enantiomers of one of the more potent and selective compds. (II) shows that the S-enantiomer is both more potent and more selective than the R-enantiomer.  The S-enantiomer had an IC50 of 0.17-0.22 μM at GluN2D- and GluN2C-contg. receptors, resp., and showed over 70-fold selectivity over other NMDA receptor subunits.  The subunit selectivity of this class of compds. should be useful in defining the role of GluN2C- and GluN2D-contg. receptors in specific brain circuits in both physiol. and pathophysiol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowVI7nRMtmybVg90H21EOLACvtfcHk0ljI4TOmN0xxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Slsb3J&md5=0b0c940da1bf0c21012f0457aa8335a5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm400652r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400652r%26sid%3Dliteratum%253Aachs%26aulast%3DAcker%26aufirst%3DT.%2BM.%26aulast%3DKhatri%26aufirst%3DA.%26aulast%3DVance%26aufirst%3DK.%2BM.%26aulast%3DSlabber%26aufirst%3DC.%26aulast%3DBacsa%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DTraynelis%26aufirst%3DS.%2BF.%26aulast%3Dliotta%26aufirst%3DD.%2BC.%26atitle%3DStructure-activity%2520relationships%2520and%2520pharmacophore%2520model%2520of%2520a%2520noncompetitive%2520pyrazoline%2520containing%2520class%2520of%2520GluN2C%252FGluN2D%2520selective%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6434%26epage%3D6456%26doi%3D10.1021%2Fjm400652r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uvarani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandraprakash, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umamaheswari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, P. S.</span></span> <span> </span><span class="NLM_article-title">An Expedient Approach for the Synthesis of Bioactive Pyrazole, Isoxazole and Benzodiazepine-Substituted Quinolin-2(1H)-one Derivatives</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1082</span>– <span class="NLM_lpage">1092</span>, <span class="refDoi"> DOI: 10.1002/jhet.2192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2Fjhet.2192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gls7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2015&pages=1082-1092&author=M.+Sankaranauthor=C.+Uvaraniauthor=K.+Chandraprakashauthor=M.+Umamaheswariauthor=P.+S.+Mohan&title=An+Expedient+Approach+for+the+Synthesis+of+Bioactive+Pyrazole%2C+Isoxazole+and+Benzodiazepine-Substituted+Quinolin-2%281H%29-one+Derivatives&doi=10.1002%2Fjhet.2192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">An Expedient Approach for the Synthesis of Bioactive Pyrazole, Isoxazole and Benzodiazepine-Substituted Quinolin-2(1H)-one Derivatives</span></div><div class="casAuthors">Sankaran, Mathan; Uvarani, Chokkalingam; Chandraprakash, Kumarasamy; Umamaheswari, Mani; Mohan, Palathurai Subramaniam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1082-1092</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A sequence of functionalized pyrazole, isoxazole and benzodiazepine-substituted quinolone derivs. were synthesized in good yield starting from readily available starting precursors.  These approaches lead to the synthesis of hitherto unknown compds. with a varied substitution pattern by both conventional and microwave-assisted method.  A good no. of analogs were evaluated for their in vitro cytotoxicity against human cervical and colon cancer cell lines by MTT protocol.  Almost all the selected compds. showed remarkable cytotoxic activities.  Among them, pyrazole-substituted quinolone derivs. I [R = Ph; R1 = H, Cl; R2 = Ph, 4-MeOC6H4; R3 = Ph, 4-FC6H4] emerged as the most promising scaffolds.  These scaffolds would be used for further mol. level studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwrxpMoc_4lLVg90H21EOLACvtfcHk0ljI4TOmN0xxSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gls7vL&md5=2cee1203f9021dcf7352616553145917</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjhet.2192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.2192%26sid%3Dliteratum%253Aachs%26aulast%3DSankaran%26aufirst%3DM.%26aulast%3DUvarani%26aufirst%3DC.%26aulast%3DChandraprakash%26aufirst%3DK.%26aulast%3DUmamaheswari%26aufirst%3DM.%26aulast%3DMohan%26aufirst%3DP.%2BS.%26atitle%3DAn%2520Expedient%2520Approach%2520for%2520the%2520Synthesis%2520of%2520Bioactive%2520Pyrazole%252C%2520Isoxazole%2520and%2520Benzodiazepine-Substituted%2520Quinolin-2%25281H%2529-one%2520Derivatives%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2015%26volume%3D52%26spage%3D1082%26epage%3D1092%26doi%3D10.1002%2Fjhet.2192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, J. T.</span></span> <span> </span><span class="NLM_article-title">Double-strand break repair deficiency and radiation sensitivity in BRCA2 mutant cancer cells</span>. <i>J. Natl. Cancer. Inst.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">985</span>, <span class="refDoi"> DOI: 10.1093/jnci/90.13.978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fjnci%2F90.13.978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9665145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK1cXksVSnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1998&pages=978-985&author=D.+W.+Abbottauthor=M.+L.+Freemanauthor=J.+T.+Holt&title=Double-strand+break+repair+deficiency+and+radiation+sensitivity+in+BRCA2+mutant+cancer+cells&doi=10.1093%2Fjnci%2F90.13.978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Double-strand break repair deficiency and radiation sensitivity in BRCA2 mutant cancer cells</span></div><div class="casAuthors">Abbott, Derek W.; Freeman, Michael L.; Holt, Jeffrey T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">978-985</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The protein product of the BRCA2 gene mediates repair of double-strand breaks in DNA.  Because a no. of cancer therapies exert cytotoxic effects via the initiation of double-strand breaks, cancers comprised of cells carrying BRCA2 gene mutations may be more amenable to treatment with agents that cause such breaks.  We identified a human pancreatic adenocarcinoma cell line lacking one copy of the BRCA2 gene and contg. a mutation (6174delT) in the remaining copy.  In vitro and in vivo expts. were conducted with this cell line and with other carcinoma cell lines matched for similar genetic mutations, similar differentiation status, and/or similar carcinoma type to examine double-strand break repair, sensitivity to drugs that induce double-strand breaks, and radiation sensitivity.  BRCA2-defective cells were unable to repair the double-strand DNA breaks induced by ionizing radiation.  These cells were also markedly sensitive to mitoxantrone, amsacrine, and etoposide (drugs that induce double-strand breaks and to ionizing radiation).  Introduction of antisense BRCA2 deoxyribonucleotides into cells possessing normal BRCA2 function led to increased sensitivity to mitoxantrone.  Tumors formed by injection of BRCA2-defective cells into nude mice were highly sensitive (>90% tumor size redn.) to both ionizing radiation and mitoxantrone when compared with tumors exhibiting normal BRCA2 function.  Histol. anal. of irradiated BRCA2-defective tumors showed a large degree of necrosis compared with that obsd. for control tumors possessing normal BRCA2 function.  BRCA2-defective cancer cells are highly sensitive to agents that cause double-strand breaks in DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBRI6YDqpNubVg90H21EOLACvtfcHk0lj3C1ce00e8iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksVSnsbg%253D&md5=b26b75011711ff951d10f6398e7dc31e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F90.13.978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F90.13.978%26sid%3Dliteratum%253Aachs%26aulast%3DAbbott%26aufirst%3DD.%2BW.%26aulast%3DFreeman%26aufirst%3DM.%2BL.%26aulast%3DHolt%26aufirst%3DJ.%2BT.%26atitle%3DDouble-strand%2520break%2520repair%2520deficiency%2520and%2520radiation%2520sensitivity%2520in%2520BRCA2%2520mutant%2520cancer%2520cells%26jtitle%3DJ.%2520Natl.%2520Cancer.%2520Inst.%26date%3D1998%26volume%3D90%26spage%3D978%26epage%3D985%26doi%3D10.1093%2Fjnci%2F90.13.978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnellmann, R. G.</span></span> <span> </span><span class="NLM_article-title">Measurement of cell death in mammalian cells</span>. <i>Curr. Protoc. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">12.8.1</span>– <span class="NLM_lpage">12.8.22</span>, <span class="refDoi"> DOI: 10.1002/0471141755.ph1208s25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2F0471141755.ph1208s25" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=12.8.1-12.8.22&author=B.+S.+Cummingsauthor=R.+G.+Schnellmann&title=Measurement+of+cell+death+in+mammalian+cells&doi=10.1002%2F0471141755.ph1208s25"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2F0471141755.ph1208s25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471141755.ph1208s25%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DB.%2BS.%26aulast%3DSchnellmann%26aufirst%3DR.%2BG.%26atitle%3DMeasurement%2520of%2520cell%2520death%2520in%2520mammalian%2520cells%26jtitle%3DCurr.%2520Protoc.%2520Pharmacol.%26date%3D2004%26volume%3D25%26spage%3D12.8.1%26epage%3D12.8.22%26doi%3D10.1002%2F0471141755.ph1208s25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Saikat Chakraborty, Kartik Dutta, Pooja Gupta, Anubrata Das, Amit Das, Sunil Kumar Ghosh, <span class="NLM_string-name hlFld-ContribAuthor">Birija Sankar Patro</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RECQL5 Functions, by a Small Molecule, Selectively Kills Breast Cancer in Vitro and in Vivo. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1524-1544. <a href="https://doi.org/10.1021/acs.jmedchem.0c01692" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01692</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01692&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01692%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BRECQL5%252BFunctions%25252C%252Bby%252Ba%252BSmall%252BMolecule%25252C%252BSelectively%252BKills%252BBreast%252BCancer%252Bin%252BVitro%252Band%252Bin%252BVivo%26aulast%3DChakraborty%26aufirst%3DSaikat%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D28092020%26date%3D02022021%26volume%3D64%26issue%3D3%26spage%3D1524%26epage%3D1544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Samuel H. Myers, Jose Antonio Ortega, <span class="NLM_string-name hlFld-ContribAuthor">Andrea Cavalli</span>. </span><span class="cited-content_cbyCitation_article-title">Synthetic Lethality through the Lens of Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14151-14183. <a href="https://doi.org/10.1021/acs.jmedchem.0c00766" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00766</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00766&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00766%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthetic%252BLethality%252Bthrough%252Bthe%252BLens%252Bof%252BMedicinal%252BChemistry%26aulast%3DMyers%26aufirst%3DSamuel%2BH.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D02112020%26volume%3D63%26issue%3D23%26spage%3D14151%26epage%3D14183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samer  Alkassis</span>, <span class="hlFld-ContribAuthor ">Omid  Yazdanpanah</span>, <span class="hlFld-ContribAuthor ">Philip Agop  Philip</span>. </span><span class="cited-content_cbyCitation_article-title">BRCA mutations in pancreatic cancer and progress in their targeting. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2021,</strong> <em>9 </em>, 1-11. <a href="https://doi.org/10.1080/14728222.2021.1957462" title="DOI URL">https://doi.org/10.1080/14728222.2021.1957462</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14728222.2021.1957462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14728222.2021.1957462%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DBRCA%252Bmutations%252Bin%252Bpancreatic%252Bcancer%252Band%252Bprogress%252Bin%252Btheir%252Btargeting%26aulast%3DAlkassis%26aufirst%3DSamer%26date%3D2021%26date%3D2021%26volume%3D9%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Singarajanahalli Mundarinti  Krishna Reddy</span>, <span class="hlFld-ContribAuthor ">Subramaniyan  Prasanna Kumari</span>, <span class="hlFld-ContribAuthor ">Subramaniapillai  Selva Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium catalysed hydrolysis-free arylation of aliphatic nitriles for the synthesis of 4-arylquinolin-2-one/pyrazolone derivatives. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2021,</strong> <em>7 </em>, 153296. <a href="https://doi.org/10.1016/j.tetlet.2021.153296" title="DOI URL">https://doi.org/10.1016/j.tetlet.2021.153296</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2021.153296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2021.153296%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DPalladium%252Bcatalysed%252Bhydrolysis-free%252Barylation%252Bof%252Baliphatic%252Bnitriles%252Bfor%252Bthe%252Bsynthesis%252Bof%252B4-arylquinolin-2-one%25252Fpyrazolone%252Bderivatives%26aulast%3DKrishna%2BReddy%26aufirst%3DSingarajanahalli%2BMundarinti%26date%3D2021%26volume%3D7%26spage%3D153296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Irina S.  Shkundina</span>, <span class="hlFld-ContribAuthor ">Alexander A.  Gall</span>, <span class="hlFld-ContribAuthor ">Alexej  Dick</span>, <span class="hlFld-ContribAuthor ">Simon  Cocklin</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Mazin</span>. </span><span class="cited-content_cbyCitation_article-title">New RAD51 Inhibitors to Target Homologous Recombination in Human Cells. </span><span class="cited-content_cbyCitation_journal-name">Genes</span><span> <strong>2021,</strong> <em>12 </em>
                                    (6)
                                     , 920. <a href="https://doi.org/10.3390/genes12060920" title="DOI URL">https://doi.org/10.3390/genes12060920</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/genes12060920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fgenes12060920%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%26atitle%3DNew%252BRAD51%252BInhibitors%252Bto%252BTarget%252BHomologous%252BRecombination%252Bin%252BHuman%252BCells%26aulast%3DShkundina%26aufirst%3DIrina%2BS.%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D6%26spage%3D920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duncan E.  Scott</span>, <span class="hlFld-ContribAuthor ">Nicola J.  Francis-Newton</span>, <span class="hlFld-ContribAuthor ">May E.  Marsh</span>, <span class="hlFld-ContribAuthor ">Anthony G.  Coyne</span>, <span class="hlFld-ContribAuthor ">Gerhard  Fischer</span>, <span class="hlFld-ContribAuthor ">Tommaso  Moschetti</span>, <span class="hlFld-ContribAuthor ">Andrew R.  Bayly</span>, <span class="hlFld-ContribAuthor ">Timothy D.  Sharpe</span>, <span class="hlFld-ContribAuthor ">Kalina T.  Haas</span>, <span class="hlFld-ContribAuthor ">Lorraine  Barber</span>, <span class="hlFld-ContribAuthor ">Chiara R.  Valenzano</span>, <span class="hlFld-ContribAuthor ">Rajavel  Srinivasan</span>, <span class="hlFld-ContribAuthor ">David J.  Huggins</span>, <span class="hlFld-ContribAuthor ">Miyoung  Lee</span>, <span class="hlFld-ContribAuthor ">Amy  Emery</span>, <span class="hlFld-ContribAuthor ">Bryn  Hardwick</span>, <span class="hlFld-ContribAuthor ">Matthias  Ehebauer</span>, <span class="hlFld-ContribAuthor ">Claudio  Dagostin</span>, <span class="hlFld-ContribAuthor ">Alessandro  Esposito</span>, <span class="hlFld-ContribAuthor ">Luca  Pellegrini</span>, <span class="hlFld-ContribAuthor ">Trevor  Perrior</span>, <span class="hlFld-ContribAuthor ">Grahame  McKenzie</span>, <span class="hlFld-ContribAuthor ">Tom L.  Blundell</span>, <span class="hlFld-ContribAuthor ">Marko  Hyvönen</span>, <span class="hlFld-ContribAuthor ">John  Skidmore</span>, <span class="hlFld-ContribAuthor ">Ashok R.  Venkitaraman</span>, <span class="hlFld-ContribAuthor ">Chris  Abell</span>. </span><span class="cited-content_cbyCitation_article-title">A small-molecule inhibitor of the BRCA2-RAD51 interaction modulates RAD51 assembly and potentiates DNA damage-induced cell death. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (6)
                                     , 835-847.e5. <a href="https://doi.org/10.1016/j.chembiol.2021.02.006" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.02.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.02.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DA%252Bsmall-molecule%252Binhibitor%252Bof%252Bthe%252BBRCA2-RAD51%252Binteraction%252Bmodulates%252BRAD51%252Bassembly%252Band%252Bpotentiates%252BDNA%252Bdamage-induced%252Bcell%252Bdeath%26aulast%3DScott%26aufirst%3DDuncan%2BE.%26date%3D2021%26volume%3D28%26issue%3D6%26spage%3D835%26epage%3D847.e5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Balboni</span>, <span class="hlFld-ContribAuthor ">Marzia  Govoni</span>, <span class="hlFld-ContribAuthor ">Valentina  Rossi</span>, <span class="hlFld-ContribAuthor ">Marinella  Roberti</span>, <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>, <span class="hlFld-ContribAuthor ">Giuseppina  Di Stefano</span>, <span class="hlFld-ContribAuthor ">Marcella  Manerba</span>. </span><span class="cited-content_cbyCitation_article-title">Lactate dehydrogenase inhibition affects homologous recombination repair independently of cell metabolic asset; implications for anticancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - General Subjects</span><span> <strong>2021,</strong> <em>1865 </em>
                                    (1)
                                     , 129760. <a href="https://doi.org/10.1016/j.bbagen.2020.129760" title="DOI URL">https://doi.org/10.1016/j.bbagen.2020.129760</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagen.2020.129760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagen.2020.129760%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520General%2520Subjects%26atitle%3DLactate%252Bdehydrogenase%252Binhibition%252Baffects%252Bhomologous%252Brecombination%252Brepair%252Bindependently%252Bof%252Bcell%252Bmetabolic%252Basset%25253B%252Bimplications%252Bfor%252Banticancer%252Btreatment%26aulast%3DBalboni%26aufirst%3DAndrea%26date%3D2021%26volume%3D1865%26issue%3D1%26spage%3D129760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kerry Silva  McPherson</span>, <span class="hlFld-ContribAuthor ">Dmitry M.  Korzhnev</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein–protein interactions in the DNA damage response pathways for cancer chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2021,</strong> <em>461 </em><a href="https://doi.org/10.1039/D1CB00101A" title="DOI URL">https://doi.org/10.1039/D1CB00101A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1CB00101A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1CB00101A%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DTargeting%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Bin%252Bthe%252BDNA%252Bdamage%252Bresponse%252Bpathways%252Bfor%252Bcancer%252Bchemotherapy%26aulast%3DMcPherson%26aufirst%3DKerry%2BSilva%26date%3D2021%26date%3D2021%26volume%3D461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leydi M.  Moreno</span>, <span class="hlFld-ContribAuthor ">Jairo  Quiroga</span>, <span class="hlFld-ContribAuthor ">Rodrigo  Abonia</span>, <span class="hlFld-ContribAuthor ">Antonino  Lauria</span>, <span class="hlFld-ContribAuthor ">Annamaria  Martorana</span>, <span class="hlFld-ContribAuthor ">Henry  Insuasty</span>, <span class="hlFld-ContribAuthor ">Braulio  Insuasty</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, biological evaluation, and
              in silico
              studies of novel chalcone- and pyrazoline-based 1,3,5-triazines as potential anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (56)
                                     , 34114-34129. <a href="https://doi.org/10.1039/D0RA06799G" title="DOI URL">https://doi.org/10.1039/D0RA06799G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA06799G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA06799G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSynthesis%25252C%252Bbiological%252Bevaluation%25252C%252Band%252Bin%252Bsilico%252Bstudies%252Bof%252Bnovel%252Bchalcone-%252Band%252Bpyrazoline-based%252B1%25252C3%25252C5-triazines%252Bas%252Bpotential%252Banticancer%252Bagents%26aulast%3DMoreno%26aufirst%3DLeydi%2BM.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D56%26spage%3D34114%26epage%3D34129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dimitris  Matiadis</span>, <span class="hlFld-ContribAuthor ">Marina  Sagnou</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrazoline Hybrids as Promising Anticancer Agents: An Up-to-Date Overview. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (15)
                                     , 5507. <a href="https://doi.org/10.3390/ijms21155507" title="DOI URL">https://doi.org/10.3390/ijms21155507</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21155507&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21155507%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DPyrazoline%252BHybrids%252Bas%252BPromising%252BAnticancer%252BAgents%25253A%252BAn%252BUp-to-Date%252BOverview%26aulast%3DMatiadis%26aufirst%3DDimitris%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D15%26spage%3D5507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Camille  Huart</span>, <span class="hlFld-ContribAuthor ">Jia-Wei  Chen</span>, <span class="hlFld-ContribAuthor ">Benjamin  Le Calvé</span>, <span class="hlFld-ContribAuthor ">Carine  Michiels</span>, <span class="hlFld-ContribAuthor ">Anne-Catherine  Wéra</span>. </span><span class="cited-content_cbyCitation_article-title">Could Protons and Carbon Ions Be the Silver Bullets Against Pancreatic Cancer?. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (13)
                                     , 4767. <a href="https://doi.org/10.3390/ijms21134767" title="DOI URL">https://doi.org/10.3390/ijms21134767</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21134767&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21134767%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DCould%252BProtons%252Band%252BCarbon%252BIons%252BBe%252Bthe%252BSilver%252BBullets%252BAgainst%252BPancreatic%252BCancer%25253F%26aulast%3DHuart%26aufirst%3DCamille%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D13%26spage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0019.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of the previously identified triazoles <b>1</b>–<b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) RAD51-BRCA2 BRC repeat complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N0W">1N0W</a>). RAD51 is represented as a surface, BRC4 as a cartoon. The two hot spots of the interaction between the proteins (Phe1524 and Phe1546) are highlighted in sticks. (B) Zone II magnification showing the interacting residues of BRC4 (yellow) and RAD51 (white).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Both enantiomers of compound <b>4d</b> docked into the LFDE binding site (site II) of RAD51 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N0W">1N0W</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Overview of the optimization strategy of <b>4d</b> for SAR exploration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0010.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Dihydroquinolone Pyrazoline Intermediates <b>86c</b>–<b>114c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) ethyl acetoacetate, DMF, 120 °C μWave or reflux, <b>60a</b> quantitative, <b>61a</b> 53%; (b) KOH, EtOH/H<sub>2</sub>O 4:3 v/v (0.05 M), 0 °C to rt, 53%, quantitative; (c) hydrazine monohydrate, EtOH, 110 °C μWave, 45 min, 64%, quantitative.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0011.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Final Dihydroquinolone Pyrazolines <b>4d</b>–<b>57d</b></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0012.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compound <b>7d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HOBt, EDCI, DCM, overnight, yield 25%; (b) HCl 4 M in dioxane, rt, 15 min; (c) NaOH 0.5 M in EtOAc, rt, 15 min, yield 50%.</p></p></figure><figure data-id="sch4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0013.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compound <b>8d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HATU (1.5 equiv), EDC (1.5 equiv), DCM, DMF, ammonium chloride (5.0 equiv), DIPEA (4.0 equiv), rt, 26 h, yield 46%.</p></p></figure><figure data-id="sch5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0014.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compound <b>9d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) HOBt (1.1 equiv), EDC (1.1 equiv), TEA (2.2. equiv) DCM, rt, 16 h, yield 42%.</p></p></figure><figure data-id="sch6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0015.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>10d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TEA (5.0 equiv), anhydrous DCM, methanesulfonyl chloride <b>122</b> (2.0 equiv), rt, 48 h, yield 82%; (b) MeOH/THF (1:1 v/v), 2 M LiOH, rt, 16 h, yield 82%; (c) HOBt (1.1 equiv), EDC (1.1 equiv), TEA (2.2 equiv), DCM, rt, 16 h, yield 15%.</p></p></figure><figure data-id="fig5" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0005.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MST analysis of His-hRAD51-<b>35d</b> binding. Titration curve of (RED-tris-NTA 2nd Generation)-His-hRAD51 (80 nM) with increasing concentrations of <b>35d</b>. Sigmoidal fitting curve was obtained using the Affinity Analysis software of NanoTemper Technologies. MST data are the average of three replicates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0006.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (A) Effect on HR caused by <b>35d</b> administered to BxPC3 cells during plasmid transfection (5 h). HR was evaluated by real-time PCR, as described in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Data were statistically analyzed using the column statistics of Prism 5 software, which applies the inferences analysis and the one-sample <i>t</i> test. The observed inhibitory effect was significantly different from 0 (the level of untreated cultures) for all tested doses, with <i>p</i> < 0.05. (B, C) Immunofluorescence detection of RAD51 in BxPC-3 nuclei after a treatment with 50 μM cisplatin (Cpl) given separately or in combination with 20 μM <b>35d</b>. Experimental details are reported in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. (B) Representative pictures showing DAPI-stained cell nuclei and the corresponding immune-labeling of RAD51 localization. In untreated cells (CTR), RAD51 labeling is clearly evident in cytoplasm and does not appear in cell nuclei. In the pictures of Cpl-exposed cells, nuclear localization of the protein is clearly evident in 3 out of the 6 shown nuclei. A higher magnification detail was included for this sample. (C) The bar graph shows the percentage of RAD51-labeled nuclei counted by two independent observers who analyzed the treated cultures. Data were statistically evaluated by applying the one-way ANOVA, which indicated a significantly increased nuclear RAD51 labeling caused by Cpl (<i>p</i> < 0.05) and no statistically significant difference between cells treated with <b>35d</b> and those exposed to <b>35d</b> + Cpl.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0007.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A, B) Evaluation of DNA damage through immune detection of nuclear γ-H2AX foci in BxPC-3 and Capan-1 cells exposed for 48 h to olaparib (10 μM) or <b>35d</b> (20 μM), given alone or in combination. (A) Representative pictures showing DAPI-stained cell nuclei and the corresponding immune-labeling of γ-H2AX. In BxPC-3 cells, coadministration of <b>35d</b> and olaparib produced increased γ-H2AX labeling. A higher magnification detail was included for this sample. As expected, Capan-1 cells showed a constitutive γ-H2AX labeling that was highly increased by olaparib but was unaffected by <b>35d</b> coadministration. (B) The bar graph shows the percentage of γ-H2AX -labeled nuclei counted by two independent observers who analyzed the treated cultures. Data obtained in bxPC-3 cells were statistically evaluated by applying the one-way ANOVA, which indicated a statistically significant difference between the cultures treated with olaparib and those exposed to olaparib + <b>35d</b>. (C, D) Evaluation of micronuclei generation in BxPC3 cells treated (72 h) with <b>35d</b> and olaparib, given alone or in combination. (C) Representative pictures showing DAPI-stained cell nuclei. White asterisks indicate the presence of micronuclei. (D) The percentage of cells bearing micronuclei was estimated by two independent observers, by analyzing 100–250 cells for each treatment sample. The obtained results were statistically analyzed by applying the one-way ANOVA, which indicated a <i>p</i> value of <0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0008.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) BxPC3 cell viability and death measured after 72 h exposure to <b>35d</b> and 10 μM olaparib, given alone or in combination. Data were analyzed by two-way ANOVA using the two treatments (<b>35d</b> and olaparib) as variables. In the cell viability experiment, Bonferroni post-test indicated a statistically significant difference produced by olaparib coadministration in all BxPC3 cultures treated with the different <b>35d</b> doses, with <i>p</i> values ranging from 0.01 (10 μM <b>35d</b>) to 0.0001 (15 and 20 μM <b>35d</b>). The same analysis was applied to the data from the cell death experiment and indicated that no statistically significant increase in cell death was produced by olaparib when coadministered with 10–15 μM <b>35d</b>. In cultures exposed to olaparib + 20 μM <b>35d</b> the evidence for cell death was markedly increased and statistically significant, with <i>p</i> < 0.0001. (B) After the 72 h treatment, BxPC3 cells were stained with vital dyes. As shown in the microscope pictures, the only culture displaying sharp evidence of cell death was that exposed to the combination of olaparib/20 μM <b>35d</b>, as demonstrated by PI nuclear staining.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/medium/jm9b01526_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0009.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Antiproliferative effect caused by <b>35d</b> and its association with olaparib, measured in Capan-1 cells (which do not operate RAD51-BRCA2-dependent HR) and in normal immortalized human renal cells (HK-2). The same procedures described for BxPC3 cells were used here. (B) Combination indexes of the olaparib/<b>35d</b> association measured in the three used cell lines, calculated according to the method previously described.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> For BxPC-3 cells, the data reported in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A (cell proliferation) were used. Data were analyzed using the column statistics and the one-sample <i>t</i> test of Prism 5 software. For BxPC-3 cells, this test showed a statistically significant difference from 0.8 for the combination of olaparib and 20 μΜ <b>35d</b>. Values of <0.8 indicate synergism between the two compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-5/acs.jmedchem.9b01526/20210220/images/large/jm9b01526_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01526&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i155">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05576" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05576" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, C. B.</span></span> <span> </span><span class="NLM_article-title">The origin and variations in sexual and sex limited characters</span>. <i>Am. Nat.</i> <span class="NLM_year" style="font-weight: bold;">1922</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1086/279847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1086%2F279847" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=1922&pages=51-63&author=C.+B.+Bridges&title=The+origin+and+variations+in+sexual+and+sex+limited+characters&doi=10.1086%2F279847"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1086%2F279847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1086%252F279847%26sid%3Dliteratum%253Aachs%26aulast%3DBridges%26aufirst%3DC.%2BB.%26atitle%3DThe%2520origin%2520and%2520variations%2520in%2520sexual%2520and%2520sex%2520limited%2520characters%26jtitle%3DAm.%2520Nat.%26date%3D1922%26volume%3D56%26spage%3D51%26epage%3D63%26doi%3D10.1086%2F279847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dobzhansky, T.</span></span> <span> </span><span class="NLM_article-title">Genetics of natural populations. Xiii. Recombination and variability in populations of Drosophila pseudoobscura</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">1946</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">290</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=20985721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1946&pages=269-290&author=T.+Dobzhansky&title=Genetics+of+natural+populations.+Xiii.+Recombination+and+variability+in+populations+of+Drosophila+pseudoobscura"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Genetics of natural populations; recombination and variability in populations of Drosophila pseudoobscura</span></div><div class="casAuthors">DOBZHANSKY T</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">1946</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">269-90</span>
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQXehIoFNdovudfST00W0BcfW6udTcc2eYcpFa4b5LciLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaH28%252FgtlyhtA%253D%253D&md5=51f55835f5a3053a235bdea89458573c</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDobzhansky%26aufirst%3DT.%26atitle%3DGenetics%2520of%2520natural%2520populations.%2520Xiii.%2520Recombination%2520and%2520variability%2520in%2520populations%2520of%2520Drosophila%2520pseudoobscura%26jtitle%3DGenetics%26date%3D1946%26volume%3D31%26spage%3D269%26epage%3D290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lucchesi, J. C.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality and semi-lethality among functionally related mutants of Drosophila melanfgaster</span>. <i>Genetics</i> <span class="NLM_year" style="font-weight: bold;">1968</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">44</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fgenetics%2F59.1.37" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=5683639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A280%3ADyaF1M%252FhtFyrtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=1968&pages=37-44&author=J.+C.+Lucchesi&title=Synthetic+lethality+and+semi-lethality+among+functionally+related+mutants+of+Drosophila+melanfgaster"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality and semi-lethality among functionally related mutants of Drosophila melanfgaster</span></div><div class="casAuthors">Lucchesi J C</div><div class="citationInfo"><span class="NLM_cas:title">Genetics</span>
        (<span class="NLM_cas:date">1968</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">37-44</span>
        ISSN:<span class="NLM_cas:issn">0016-6731</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTXM96_03HVQCO2kE7krGOBfW6udTcc2eYcpFa4b5LciLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF1M%252FhtFyrtg%253D%253D&md5=0a74c995f844634c47a35e6a5b39b90a</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fgenetics%2F59.1.37&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fgenetics%252F59.1.37%26sid%3Dliteratum%253Aachs%26aulast%3DLucchesi%26aufirst%3DJ.%2BC.%26atitle%3DSynthetic%2520lethality%2520and%2520semi-lethality%2520among%2520functionally%2520related%2520mutants%2520of%2520Drosophila%2520melanfgaster%26jtitle%3DGenetics%26date%3D1968%26volume%3D59%26spage%3D37%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schekman, R.</span></span> <span> </span><span class="NLM_article-title">Distinct sets of SEC genes govern transport vesicle formation and fusion early in the secretory pathway</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">733</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(90)90483-U</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2F0092-8674%2890%2990483-U" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=2188733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK3cXktFyrsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=1990&pages=723-733&author=C.+A.+Kaiserauthor=R.+Schekman&title=Distinct+sets+of+SEC+genes+govern+transport+vesicle+formation+and+fusion+early+in+the+secretory+pathway&doi=10.1016%2F0092-8674%2890%2990483-U"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct sets of SEC genes govern transport vesicle formation and fusion early in the secretory pathway</span></div><div class="casAuthors">Kaiser, Chris A.; Schekman, Randy</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">723-33</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">A vesicular intermediate in protein transport from the endoplasmic reticulum is detected in a subset of temp.-sensitive mutants blocked early in the yeast secretory pathway.  By electron microscopy three of the mutants, sec18, sec17, and sec22, accumulate 50 nm vesicles at the nonpermissive temp.  Vesicle accumulation is blocked by the mutations sec12, sec13, sec16, and sec23 as shown by anal. of double-mutant strains.  Thus the early SEC genes can be divided into vesicle forming and vesicle fusion functions.  Synthetic lethal interactions between sec mutations define two groups of SEC genes, corresponding to the groups involved in vesicle formation or fusion.  Mutations in two of the genes involved in vesicle fusion, SEC17 and SEC18, are lethal in combination, and five of six possible pairwise combinations of mutations in genes required for vesicle formation, SEC12, SEC13, SEC16, and SEC23, are lethal.  These interactions suggest cooperation between different SEC gene products in vesicle budding and vesicle fusion processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriiTAfitWtmrVg90H21EOLACvtfcHk0lgsc1TlCbwv6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktFyrsLY%253D&md5=e4c09689d2eaf9acfe470d244bf4b39c</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2890%2990483-U&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252890%252990483-U%26sid%3Dliteratum%253Aachs%26aulast%3DKaiser%26aufirst%3DC.%2BA.%26aulast%3DSchekman%26aufirst%3DR.%26atitle%3DDistinct%2520sets%2520of%2520SEC%2520genes%2520govern%2520transport%2520vesicle%2520formation%2520and%2520fusion%2520early%2520in%2520the%2520secretory%2520pathway%26jtitle%3DCell%26date%3D1990%26volume%3D61%26spage%3D723%26epage%3D733%26doi%3D10.1016%2F0092-8674%2890%2990483-U" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hennessy, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Botstein, D.</span></span> <span> </span><span class="NLM_article-title">A group of interacting yeast DNA replication genes</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">958</span>– <span class="NLM_lpage">969</span>, <span class="refDoi"> DOI: 10.1101/gad.5.6.958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1101%2Fgad.5.6.958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=2044962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK38XhsVGju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1991&pages=958-969&author=K.+M.+Hennessyauthor=A.+Leeauthor=E.+Chenauthor=D.+Botstein&title=A+group+of+interacting+yeast+DNA+replication+genes&doi=10.1101%2Fgad.5.6.958"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A group of interacting yeast DNA replication genes</span></div><div class="casAuthors">Hennessy, Kevin M.; Lee, Angela; Chen, Ellson; Botstein, David</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">958-69</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    </div><div class="casAbstract">Mutations in the cell-division-cycle genes CDC46 and CDC47 were originally isolated as suppressors of mutations in 2 other cell-division-cycle genes (CDC45 and CDC54).  Several combinations of mutations were found in these genes that result in allele-specific suppression and synthetic lethality, confirming that this set of genes forms a group of genetically interacting components.  Here, it is shown that the other genes, like CDC46, are all involved in an early step of DNA replication, possibly initiation of DNA synthesis.  Mutants defective in each of the 4 genes exhibit high rates of mitotic chromosome loss and recombination.  The mutants appear also to accumulate chromosome damage that can be detected by a novel chromosome electrophoresis assay.  Conditional mutants in this group, under fully nonpermissive conditions, show cell-cycle arrest at the beginning of DNA synthesis; under less stringent conditions, some arrest later, in S-phase.  The DNA sequence of the CDC46 gene indicates that the protein is a member of a new family of genes apparently required for DNA initiation, with family members now identified in Saccharomyces cerevisiae, Schizosaccharomyces pombe, and mouse cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmvD7dqS8PXbVg90H21EOLACvtfcHk0lgsc1TlCbwv6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsVGju7o%253D&md5=a2dae99c0886b0f4b3ce744b46c43253</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1101%2Fgad.5.6.958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.5.6.958%26sid%3Dliteratum%253Aachs%26aulast%3DHennessy%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DE.%26aulast%3DBotstein%26aufirst%3DD.%26atitle%3DA%2520group%2520of%2520interacting%2520yeast%2520DNA%2520replication%2520genes%26jtitle%3DGenes%2520Dev.%26date%3D1991%26volume%3D5%26spage%3D958%26epage%3D969%26doi%3D10.1101%2Fgad.5.6.958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaelin, W. G.,  Jr.</span></span> <span> </span><span class="NLM_article-title">The concept of synthetic lethality in the context of anticancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">689</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nrc1691</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnrc1691" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=16110319" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpsleks7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=689-698&author=W.+G.+Kaelin&title=The+concept+of+synthetic+lethality+in+the+context+of+anticancer+therapy&doi=10.1038%2Fnrc1691"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The Concept of Synthetic Lethality in the Context of Anticancer Therapy</span></div><div class="casAuthors">Kaelin, William G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">689-698</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review with refs.  Two genes are synthetic lethal if mutation of either alone is compatible with viability but mutation of both leads to death.  So, targeting a gene that is synthetic lethal to a cancer-relevant mutation should kill only cancer cells and spare normal cells.  Synthetic lethality therefore provides a conceptual framework for the development of cancer-specific cytotoxic agents.  This paradigm has not been exploited in the past because there were no robust methods for systematically identifying synthetic lethal genes.  This is changing as a result of the increased availability of chem. and genetic tools for perturbing gene function in somatic cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrf4V-KQ6I5ILVg90H21EOLACvtfcHk0lgsc1TlCbwv6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpsleks7c%253D&md5=4d5e38282cd8a3d2c3451d56ca88ac8b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc1691&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1691%26sid%3Dliteratum%253Aachs%26aulast%3DKaelin%26aufirst%3DW.%2BG.%26atitle%3DThe%2520concept%2520of%2520synthetic%2520lethality%2520in%2520the%2520context%2520of%2520anticancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2005%26volume%3D5%26spage%3D689%26epage%3D698%26doi%3D10.1038%2Fnrc1691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dar, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, R.</span></span> <span> </span><span class="NLM_article-title">Synthetic Lethality: From research to precision cancer nanomedicine</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">346</span>, <span class="refDoi"> DOI: 10.2174/1568009617666170630141931</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.2174%2F1568009617666170630141931" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=28669337" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvFejtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=337-346&author=A.+Guptaauthor=A.+Ahmadauthor=A.+I.+Darauthor=R.+Khan&title=Synthetic+Lethality%3A+From+research+to+precision+cancer+nanomedicine&doi=10.2174%2F1568009617666170630141931"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethality: From Research to Precision Cancer Nanomedicine</span></div><div class="casAuthors">Gupta, Anuradha; Ahmad, Anas; Dar, Aqib Iqbal; Khan, Rehan</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">337-346</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Cancer is an evolutionary disease with multiple genetic alterations, accumulated due to chromosomal instability and/or aneuploidy and it sometimes acquires drug-resistant phenotype also.  Whole genome sequencing and mutational anal. helped in understanding the differences among persons for predisposition of a disease and its treatment non-responsiveness.  Thus, mol. targeted therapies came into existence.  Among them, the concept of synthetic lethality have enthralled great attention as it is a pragmatic approach towards exploiting cancer cell specific mutations to specifically kill cancer cells without affecting normal cells and thus enhancing anti-cancer drug therapeutic index.  Thus, this approach helped in discovering new therapeutic mols. for development of precision medicine.  Nanotechnol. helped in delivering these mols. to the target site in an effective concn. thus reducing off target effects of drugs, dose and dosage frequency drugs.  Researchers have tried to deliver siRNA targeting synthetic lethal partner for target cancer cell killing by incorporating it in nanoparticles and it has shown efficacy by preventing tumor progression.  This review summarizes the brief introduction of synthetic lethality, and synthetic lethal gene interactions, with a major focus on its therapeutic anticancer potential with the application of nanotechnol. for development of personalized medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNT60k7B0777Vg90H21EOLACvtfcHk0lh1_tO0z1bFGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvFejtbc%253D&md5=a2385841eda520b634f1f6ab2eaa885a</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.2174%2F1568009617666170630141931&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009617666170630141931%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DA.%26aulast%3DAhmad%26aufirst%3DA.%26aulast%3DDar%26aufirst%3DA.%2BI.%26aulast%3DKhan%26aufirst%3DR.%26atitle%3DSynthetic%2520Lethality%253A%2520From%2520research%2520to%2520precision%2520cancer%2520nanomedicine%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2018%26volume%3D18%26spage%3D337%26epage%3D346%26doi%3D10.2174%2F1568009617666170630141931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giaccia, A. J.</span></span> <span> </span><span class="NLM_article-title">Harnessing synthetic lethal interactions in anticancer drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">351</span>– <span class="NLM_lpage">364</span>, <span class="refDoi"> DOI: 10.1038/nrd3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnrd3374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=21532565" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVGltrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=351-364&author=D.+A.+Chanauthor=A.+J.+Giaccia&title=Harnessing+synthetic+lethal+interactions+in+anticancer+drug+discovery&doi=10.1038%2Fnrd3374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing synthetic lethal interactions in anticancer drug discovery</span></div><div class="casAuthors">Chan, Denise A.; Giaccia, Amato J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">351-364</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Unique features of tumors that can be exploited by targeted therapies are a key focus of current cancer research.  One such approach is known as synthetic lethality screening, which involves searching for genetic interactions of two mutations whereby the presence of either mutation alone has no effect on cell viability but the combination of the two mutations results in cell death.  The presence of one of these mutations in cancer cells but not in normal cells can therefore create opportunities to selectively kill cancer cells by mimicking the effect of the second genetic mutation with targeted therapy.  Here, we summarize strategies that can be used to identify synthetic lethal interactions for anticancer drug discovery, describe examples of such interactions that are currently being investigated in preclin. and clin. studies of targeted anticancer therapies, and discuss the challenges of realizing the full potential of such therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrndLGWxThAjrVg90H21EOLACvtfcHk0lh1_tO0z1bFGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVGltrY%253D&md5=816a9dc23817f6bf0597b4a026fd2aac</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrd3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3374%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DD.%2BA.%26aulast%3DGiaccia%26aufirst%3DA.%2BJ.%26atitle%3DHarnessing%2520synthetic%2520lethal%2520interactions%2520in%2520anticancer%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D351%26epage%3D364%26doi%3D10.1038%2Fnrd3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beijersbergen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessels, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernards, R.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality in cancer therapeutics</span>. <i>Annu. Rev. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1146/annurev-cancerbio-042016-073434</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1146%2Fannurev-cancerbio-042016-073434" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2017&pages=141-161&author=R.+Beijersbergenauthor=L.+Wesselsauthor=R.+Bernards&title=Synthetic+lethality+in+cancer+therapeutics&doi=10.1146%2Fannurev-cancerbio-042016-073434"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1146%2Fannurev-cancerbio-042016-073434&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-cancerbio-042016-073434%26sid%3Dliteratum%253Aachs%26aulast%3DBeijersbergen%26aufirst%3DR.%26aulast%3DWessels%26aufirst%3DL.%26aulast%3DBernards%26aufirst%3DR.%26atitle%3DSynthetic%2520lethality%2520in%2520cancer%2520therapeutics%26jtitle%3DAnnu.%2520Rev.%2520Cancer%2520Biol.%26date%3D2017%26volume%3D1%26spage%3D141%26epage%3D161%26doi%3D10.1146%2Fannurev-cancerbio-042016-073434" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E.</span></span> <span> </span><span class="NLM_article-title">Targetin epigenetics using synthetic lethality in precision medicine</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3381</span>– <span class="NLM_lpage">3392</span>, <span class="refDoi"> DOI: 10.1007/s00018-018-2866-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1007%2Fs00018-018-2866-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30003270" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlSqt7%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2018&pages=3381-3392&author=E.+Chen&title=Targetin+epigenetics+using+synthetic+lethality+in+precision+medicine&doi=10.1007%2Fs00018-018-2866-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetics using synthetic lethality in precision medicine</span></div><div class="casAuthors">Chen, Ee Sin</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">3381-3392</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Basel</span>)
        </div><div class="casAbstract">Technol. breakthroughs in genomics have had a significant impact on clin. therapy for human diseases, allowing us to use patient genetic differences to guide medical care.  The "synthetic lethal approach" leverages on cancer-specific genetic rewiring to deliver a therapeutic regimen that preferentially targets malignant cells while sparing normal cells.  The utility of this system is evident in several recent studies, particularly in poor prognosis cancers with loss-of-function mutations that become "treatable" when two otherwise discrete and unrelated genes are targeted simultaneously.  This review focuses on the chemotherapeutic targeting of epigenetic alterations in cancer cells and consolidates a network that outlines the interplay between epigenetic and genetic regulators in DNA damage repair.  This network consists of numerous synergistically acting relationships that are druggable, even in recalcitrant triple-neg. breast cancer.  This collective knowledge points to the dawn of a new era of personalized medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrcAwplWbsPg7Vg90H21EOLACvtfcHk0ljh67u649-OTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlSqt7%252FK&md5=f56cdc453006c19cfffd0598b4ce6a65</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs00018-018-2866-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-018-2866-0%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DE.%26atitle%3DTargetin%2520epigenetics%2520using%2520synthetic%2520lethality%2520in%2520precision%2520medicine%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2018%26volume%3D75%26spage%3D3381%26epage%3D3392%26doi%3D10.1007%2Fs00018-018-2866-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethal therapies for cancer: what’s next after PARP inhibitors?</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">576</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0055-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fs41571-018-0055-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=29955114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yks7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=564-576&author=A.+Ashworthauthor=C.+J.+Lord&title=Synthetic+lethal+therapies+for+cancer%3A+what%E2%80%99s+next+after+PARP+inhibitors%3F&doi=10.1038%2Fs41571-018-0055-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethal therapies for cancer: what's next after PARP inhibitors?</span></div><div class="casAuthors">Ashworth, Alan; Lord, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">564-576</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  The genetic concept of synthetic lethality has now been validated clin. through the demonstrated efficacy of poly(ADP-ribose) polymerase (PARP) inhibitors for the treatment of cancers in individuals with germline loss-of-function mutations in either BRCA1 or BRCA2.  Three different PARP inhibitors have now been approved for the treatment of patients with BRCA-mutant ovarian cancer and one for those with BRCA-mutant breast cancer; these agents have also shown promising results in patients with BRCA-mutant prostate cancer.  Here, it describe a no. of other synthetic lethal interactions that have been discovered in cancer.  It discuss some of the underlying principles that might increase the likelihood of clin. efficacy and how new computational and exptl. approaches are now facilitating the discovery and validation of synthetic lethal interactions.  Finally, it make suggestions on possible future directions and challenges facing researchers in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjc1AuMwtxGbVg90H21EOLACvtfcHk0ljh67u649-OTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yks7zN&md5=77597e5d235648558bf27b141b1db9cd</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0055-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0055-6%26sid%3Dliteratum%253Aachs%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26atitle%3DSynthetic%2520lethal%2520therapies%2520for%2520cancer%253A%2520what%25E2%2580%2599s%2520next%2520after%2520PARP%2520inhibitors%253F%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D15%26spage%3D564%26epage%3D576%26doi%3D10.1038%2Fs41571-018-0055-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parameswaran, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundapur, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vizeacoumar, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freywald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uppalapati, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vizeacoumar, F. J.</span></span> <span> </span><span class="NLM_article-title">A road map to personalizing targeted cancer therapies using synthetic lethality</span>. <i>Trends Cancer</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">29</span>, <span class="refDoi"> DOI: 10.1016/j.trecan.2018.11.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.trecan.2018.11.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30616753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1MXkvVKluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2019&pages=11-29&author=S.+Parameswaranauthor=D.+Kundapurauthor=F.+S.+Vizeacoumarauthor=A.+Freywaldauthor=M.+Uppalapatiauthor=F.+J.+Vizeacoumar&title=A+road+map+to+personalizing+targeted+cancer+therapies+using+synthetic+lethality&doi=10.1016%2Fj.trecan.2018.11.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">A Road Map to Personalizing Targeted Cancer Therapies Using Synthetic Lethality</span></div><div class="casAuthors">Parameswaran, Sreejit; Kundapur, Deeksha; Vizeacoumar, Frederick S.; Freywald, Andrew; Uppalapati, Maruti; Vizeacoumar, Franco J.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cancer</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-29</span>CODEN:
                <span class="NLM_cas:coden">TCRAB5</span>;
        ISSN:<span class="NLM_cas:issn">2405-8033</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Targeted therapies rely on the genetic and epigenetic status of the tumor cells and are seen as the most promising approach to treat cancer today.  However, current targeted therapies focus on directly inhibiting those mols. that are altered in tumor cells.  Unfortunately, targeting these mols., even with specific inhibitors, is challenging as tumor cells rewire their genetic circuitry to eliminate genetic dependency on these targets.  Here, we describe how synthetic lethality approaches can be used to identify genetic dependencies and develop personalized targeted therapies.  We also discuss strategies to specifically target these genetic dependencies, using small mol. and biol. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvte78cztNurVg90H21EOLACvtfcHk0ljh67u649-OTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXkvVKluw%253D%253D&md5=e6e4d907a4d565756650d1762bfd4bb8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.trecan.2018.11.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.trecan.2018.11.001%26sid%3Dliteratum%253Aachs%26aulast%3DParameswaran%26aufirst%3DS.%26aulast%3DKundapur%26aufirst%3DD.%26aulast%3DVizeacoumar%26aufirst%3DF.%2BS.%26aulast%3DFreywald%26aufirst%3DA.%26aulast%3DUppalapati%26aufirst%3DM.%26aulast%3DVizeacoumar%26aufirst%3DF.%2BJ.%26atitle%3DA%2520road%2520map%2520to%2520personalizing%2520targeted%2520cancer%2520therapies%2520using%2520synthetic%2520lethality%26jtitle%3DTrends%2520Cancer%26date%3D2019%26volume%3D5%26spage%3D11%26epage%3D29%26doi%3D10.1016%2Fj.trecan.2018.11.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nandi, S.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality in DNA repair network: a novel avenue in targeted cancer therapy and combination therapeutics</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">929</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.1002/iub.1696</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2Fiub.1696" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=29171189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKht7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2017&pages=929-937&author=S.+Bhattacharjeeauthor=S.+Nandi&title=Synthetic+lethality+in+DNA+repair+network%3A+a+novel+avenue+in+targeted+cancer+therapy+and+combination+therapeutics&doi=10.1002%2Fiub.1696"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic lethality in DNA repair network: A novel avenue in targeted cancer therapy and combination therapeutics</span></div><div class="casAuthors">Bhattacharjee, Sonali; Nandi, Saikat</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">929-937</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Synthetic lethality refers to a lethal phenotype that results from the simultaneous disruptions of two genes, while the disruption of either gene alone is viable.  Many DNA double strand break repair (DSBR) genes have synthetic lethal relationships with oncogenes and tumor suppressor genes, which can be exploited for targeted cancer therapy, an approach referred to as combination therapy.  DNA double-strand breaks (DSBs) are one of the most toxic lesions to a cell and can be repaired by non-homologous end joining (NHEJ) or homologous recombination (HR).  HR and NHEJ genes are particularly attractive targets for cancer therapy because these genes have altered expression patterns in cancer cells when compared with normal cells and these genetic abnormalities can be targeted for selectively killing cancer cells.  Here, we review recent advances in the development of small mol. inhibitors against HR and NHEJ genes to induce synthetic lethality and address the future directions and clin. relevance of this approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB457gpMFejrVg90H21EOLACvtfcHk0ljsGY1_-x60WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKht7vK&md5=07083a1ed6b8e033c21e0c61e8fab4b2</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1002%2Fiub.1696&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fiub.1696%26sid%3Dliteratum%253Aachs%26aulast%3DBhattacharjee%26aufirst%3DS.%26aulast%3DNandi%26aufirst%3DS.%26atitle%3DSynthetic%2520lethality%2520in%2520DNA%2520repair%2520network%253A%2520a%2520novel%2520avenue%2520in%2520targeted%2520cancer%2520therapy%2520and%2520combination%2520therapeutics%26jtitle%3DIUBMB%2520Life%26date%3D2017%26volume%3D69%26spage%3D929%26epage%3D937%26doi%3D10.1002%2Fiub.1696" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minchom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aversa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, J.</span></span> <span> </span><span class="NLM_article-title">Dancing with the DNA damage response: next-generation anti-cancer therapeutic strategies</span>. <i>Ther. Adv. Med. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1177/1758835918786658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1177%2F1758835918786658" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=1-18&author=A.+Minchomauthor=C.+Aversaauthor=J.+Lopez&title=Dancing+with+the+DNA+damage+response%3A+next-generation+anti-cancer+therapeutic+strategies&doi=10.1177%2F1758835918786658"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1177%2F1758835918786658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1758835918786658%26sid%3Dliteratum%253Aachs%26aulast%3DMinchom%26aufirst%3DA.%26aulast%3DAversa%26aufirst%3DC.%26aulast%3DLopez%26aufirst%3DJ.%26atitle%3DDancing%2520with%2520the%2520DNA%2520damage%2520response%253A%2520next-generation%2520anti-cancer%2520therapeutic%2520strategies%26jtitle%3DTher.%2520Adv.%2520Med.%2520Oncol.%26date%3D2018%26volume%3D10%26spage%3D1%26epage%3D18%26doi%3D10.1177%2F1758835918786658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pilie, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, T. A.</span></span> <span> </span><span class="NLM_article-title">State-of-the-art strategies for targeting the DNA damage response in cancer</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">81</span>– <span class="NLM_lpage">104</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0114-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fs41571-018-0114-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30356138" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVSjtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=81-104&author=P.+G.+Pilieauthor=C.+Tangauthor=G.+B.+Millsauthor=T.+A.+Yap&title=State-of-the-art+strategies+for+targeting+the+DNA+damage+response+in+cancer&doi=10.1038%2Fs41571-018-0114-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">State-of-the-art strategies for targeting the DNA damage response in cancer</span></div><div class="casAuthors">Pilie, Patrick G.; Tang, Chad; Mills, Gordon B.; Yap, Timothy A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-104</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Genomic instability is a key hallmark of cancer that arises owing to defects in the DNA damage response (DDR) and/or increased replication stress.  These alterations promote the clonal evolution of cancer cells via the accumulation of driver aberrations, including gene copy-no. changes, rearrangements and mutations; however, these same defects also create vulnerabilities that are relatively specific to cancer cells, which could potentially be exploited to increase the therapeutic index of anticancer treatments and thereby improve patient outcomes.  The discovery that BRCA-mutant cancer cells are exquisitely sensitive to inhibition of poly(ADP-ribose) polymerase has ushered in a new era of research on biomarker-driven synthetic lethal treatment strategies for different cancers.  The therapeutic landscape of antitumor agents targeting the DDR has rapidly expanded to include inhibitors of other key mediators of DNA repair and replication, such as ATM, ATR, CHK1 and CHK2, DNA-PK and WEE1.  Efforts to optimize these therapies are ongoing across a range of cancers, involving the development of predictive biomarker assays of responsiveness (beyond BRCA mutations), assessment of the mechanisms underlying intrinsic and acquired resistance, and evaluation of rational, tolerable combinations with std.-of-care treatments (such as chemotherapeutics and radiation), novel molecularly targeted agents and immune-checkpoint inhibitors.  In this Review, we discuss the current status of anticancer therapies targeting the DDR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpgsKJRZycYLVg90H21EOLACvtfcHk0ljsGY1_-x60WQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVSjtb%252FI&md5=f82199b7e32e11aabcad7cb664774e63</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0114-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0114-z%26sid%3Dliteratum%253Aachs%26aulast%3DPilie%26aufirst%3DP.%2BG.%26aulast%3DTang%26aufirst%3DC.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DYap%26aufirst%3DT.%2BA.%26atitle%3DState-of-the-art%2520strategies%2520for%2520targeting%2520the%2520DNA%2520damage%2520response%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2019%26volume%3D16%26spage%3D81%26epage%3D104%26doi%3D10.1038%2Fs41571-018-0114-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lgPvtzZB764QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Deeks, E. D.</span></span> <span> </span><span class="NLM_article-title">Olaparib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">231</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1007/s40265-015-0345-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1007%2Fs40265-015-0345-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=25616434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVejtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=231-240&author=E.+D.+Deeks&title=Olaparib%3A+first+global+approval&doi=10.1007%2Fs40265-015-0345-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Olaparib: First Global Approval</span></div><div class="casAuthors">Deeks, Emma D.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">231-240</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Olaparib (Lynparza) is an oral, small mol., poly (ADP-ribose) polymerase inhibitor being developed by AstraZeneca for the treatment of solid tumors.  The primary indication that olaparib is being developed for is BRCA mutation-pos. ovarian cancer.  A capsule formulation of the drug has received approval for use in this setting in the EU and USA, and a tablet formulation is in global phase III trials (including in the USA, EU, Australia, Brazil, Canada, China, Israel, Japan, Russia and South Korea).  In addn., phase III trials in breast, gastric and pancreatic cancer are underway/planned, and phase I/II investigation is being conducted in other malignancies, including prostate cancer, non-small cell lung cancer, Ewing's sarcoma and advanced cancer.  This article summarizes the milestones in the development of olaparib leading to this first approval for ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4agVSDkOKqbVg90H21EOLACvtfcHk0lgPvtzZB764QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVejtbg%253D&md5=cd68d932b6532a87c0768590afd81058</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1007%2Fs40265-015-0345-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-015-0345-6%26sid%3Dliteratum%253Aachs%26aulast%3DDeeks%26aufirst%3DE.%2BD.%26atitle%3DOlaparib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2015%26volume%3D75%26spage%3D231%26epage%3D240%26doi%3D10.1007%2Fs40265-015-0345-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><a href="http://www.fda.gov/Drugs/InformationOnDrugs/ApprovedDrugs/ucm592357.htm" class="extLink">www.fda.gov/Drugs/InformationOnDrugs/ApprovedDrugs/ucm592357.htm</a> (accessed Sep 15, <span class="NLM_year">2019</span>).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=www.fda.gov%2FDrugs%2FInformationOnDrugs%2FApprovedDrugs%2Fucm592357.htm+%28accessed+Sep+15%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26date%3D2019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Cutsem, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macarulla, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochhauser, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnold, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinacher-Schick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tortora, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Algul, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Reilly, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGuinness, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlienger, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locker, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kindler, H. L.</span></span> <span> </span><span class="NLM_article-title">Maintenance olaparib for germline BRCA-mutated metastatic pancreatic cancer</span>. <i>N. Engl. J. Med.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1056/NEJMoa1903387</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1056%2FNEJMoa1903387" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=31157963" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnsFCqtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2019&pages=317-327&author=T.+Golanauthor=P.+Hammelauthor=M.+Reniauthor=E.+Van+Cutsemauthor=T.+Macarullaauthor=M.+J.+Hallauthor=J.+Parkauthor=D.+Hochhauserauthor=D.+Arnoldauthor=D.+Ohauthor=A.+Reinacher-Schickauthor=G.+Tortoraauthor=H.+Algulauthor=E.+M.+O%E2%80%99Reillyauthor=D.+McGuinnessauthor=K.+Y.+Cuiauthor=K.+Schliengerauthor=G.+Y.+Lockerauthor=H.+L.+Kindler&title=Maintenance+olaparib+for+germline+BRCA-mutated+metastatic+pancreatic+cancer&doi=10.1056%2FNEJMoa1903387"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Maintenance olaparib for germline BRCA-mutated metastatic pancreatic cancer</span></div><div class="casAuthors">Golan, Talia; Hammel, Pascal; Reni, Michele; Van Cutsem, Eric; Macarulla, Teresa; Hall, Michael J.; Park, Joon-Oh; Hochhauser, Daniel; Arnold, Dirk; Oh, Do-Youn; Reinacher-Schick, Anke; Tortora, Giampaolo; Algul, Hana; O'Reilly, Eileen M.; McGuinness, David; Cui, Karen Y.; Schlienger, Katia; Locker, Gershon Y.; Kindler, Hedy L.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">317-327</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">1533-4406</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background Patients with a germline BRCA1 or BRCA2 mutation make up a small subgroup of those with metastatic pancreatic cancer.  The poly(ADP-ribose) polymerase (PARP) inhibitor olaparib has had antitumor activity in this population. methods We conducted a randomized, double-blind, placebo-controlled, phase 3 trial to evaluate the efficacy of olaparib as maintenance therapy in patients who had a germline BRCA1 or BRCA2 mutation and metastatic pancreatic cancer and disease that had not progressed during first-line platinum-based chemotherapy.  Patients were randomly assigned, in a 3:2 ratio, to receive maintenance olaparib tablets (300 mg twice daily) or placebo.  The primary end point was progression-free survival, which was assessed by blinded independent central review. results Of the 3315 patients who underwent screening, 154 underwent randomization and were assigned to a trial intervention (92 to receive olaparib and 62 to receive placebo).  The median progression-free survival was significantly longer in the olaparib group than in the placebo group (7.4 mo vs. 3.8 mo; hazard ratio for disease progression or death, 0.53; 95% confidence interval [CI], 0.35 to 0.82; P = 0.004).  An interim anal. of overall survival, at a data maturity of 46%, showed no difference between the olaparib and placebo groups (median, 18.9 mo vs. 18.1 mo; hazard ratio for death, 0.91; 95% CI, 0.56 to 1.46; P = 0.68).  There was no significant between-group difference in health-related quality of life, as indicated by the overall change from baseline in the global quality-of-life score (on a 100-point scale, with higher scores indicating better quality of life) based on the European Organization for Research and Treatment of Cancer Quality of Life Questionnaire (between-group difference, -2.47 points; 95% CI, -7.27 to 2.33).  The incidence of grade 3 or higher adverse events was 40% in the olaparib group and 23% in the placebo group (between-group difference, 16 percentage points; 95% CI, -0.02 to 31); 5% and 2% of the patients, resp., discontinued the trial intervention because of an adverse event. conclusions Among patients with a germline BRCA mutation and metastatic pancreatic cancer, progression-free survival was longer with maintenance olaparib than with placebo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_mABSEPws9rVg90H21EOLACvtfcHk0liFbnYulg-cHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnsFCqtbo%253D&md5=06afa1aada240931f620ecf6f974b2e2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1903387&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1903387%26sid%3Dliteratum%253Aachs%26aulast%3DGolan%26aufirst%3DT.%26aulast%3DHammel%26aufirst%3DP.%26aulast%3DReni%26aufirst%3DM.%26aulast%3DVan%2BCutsem%26aufirst%3DE.%26aulast%3DMacarulla%26aufirst%3DT.%26aulast%3DHall%26aufirst%3DM.%2BJ.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DHochhauser%26aufirst%3DD.%26aulast%3DArnold%26aufirst%3DD.%26aulast%3DOh%26aufirst%3DD.%26aulast%3DReinacher-Schick%26aufirst%3DA.%26aulast%3DTortora%26aufirst%3DG.%26aulast%3DAlgul%26aufirst%3DH.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DE.%2BM.%26aulast%3DMcGuinness%26aufirst%3DD.%26aulast%3DCui%26aufirst%3DK.%2BY.%26aulast%3DSchlienger%26aufirst%3DK.%26aulast%3DLocker%26aufirst%3DG.%2BY.%26aulast%3DKindler%26aufirst%3DH.%2BL.%26atitle%3DMaintenance%2520olaparib%2520for%2520germline%2520BRCA-mutated%2520metastatic%2520pancreatic%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2019%26volume%3D381%26spage%3D317%26epage%3D327%26doi%3D10.1056%2FNEJMoa1903387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIlwraith, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">West, S. C.</span></span> <span> </span><span class="NLM_article-title">Role of BRCA2 in control of the RAD51 recombination and DNA repair protein</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">273</span>– <span class="NLM_lpage">282</span>, <span class="refDoi"> DOI: 10.1016/S1097-2765(01)00175-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2FS1097-2765%2801%2900175-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=11239456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD3MXis1Kitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=273-282&author=A.+A.+Daviesauthor=J.+Y.+Massonauthor=M.+J.+McIlwraithauthor=A.+Z.+Stasiakauthor=A.+Stasiakauthor=A.+R.+Venkitaramanauthor=S.+C.+West&title=Role+of+BRCA2+in+control+of+the+RAD51+recombination+and+DNA+repair+protein&doi=10.1016%2FS1097-2765%2801%2900175-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Role of BRCA2 in control of the RAD51 recombination and DNA repair protein</span></div><div class="casAuthors">Davies, Adelina A.; Masson, Jean-Yves; McIlwraith, Michael J.; Stasiak, Alicja Z.; Stasiak, Andrzej; Venkitaraman, Ashok R.; West, Stephen C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-282</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Individuals carrying BRCA2 mutations are predisposed to breast and ovarian cancers.  Here, we show that BRCA2 plays a dual role in regulating the actions of RAD51, a protein essential for homologous recombination and DNA repair.  First, interactions between RAD51 and the BRC3 or BRC4 regions of BRCA2 block nucleoprotein filament formation by RAD51.  Alterations to the BRC3 region that mimic cancer-assocd. BRCA2 mutations fail to exhibit this effect.  Second, transport of RAD51 to the nucleus is defective in cells carrying a cancer-assocd. BRCA2 truncation.  Thus, BRCA2 regulates both the intracellular localization and DNA binding ability of RAD51.  Loss of these controls following BRCA2 inactivation may be a key event leading to genomic instability and tumorigenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6admGLM9NrLVg90H21EOLACvtfcHk0liFbnYulg-cHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXis1Kitrc%253D&md5=9295ae3b5e51e58beeed4f9adb6e20e2</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900175-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900175-7%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DA.%2BA.%26aulast%3DMasson%26aufirst%3DJ.%2BY.%26aulast%3DMcIlwraith%26aufirst%3DM.%2BJ.%26aulast%3DStasiak%26aufirst%3DA.%2BZ.%26aulast%3DStasiak%26aufirst%3DA.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DWest%26aufirst%3DS.%2BC.%26atitle%3DRole%2520of%2520BRCA2%2520in%2520control%2520of%2520the%2520RAD51%2520recombination%2520and%2520DNA%2520repair%2520protein%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D7%26spage%3D273%26epage%3D282%26doi%3D10.1016%2FS1097-2765%2801%2900175-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, H. L.</span></span> <span> </span><span class="NLM_article-title">The consequences of Rad51 overexpression for normal and tumor cells</span>. <i>DNA Repair</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1016/j.dnarep.2007.12.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.dnarep.2007.12.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=18243065" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD1cXls1Sgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=686-693&author=H.+L.+Klein&title=The+consequences+of+Rad51+overexpression+for+normal+and+tumor+cells&doi=10.1016%2Fj.dnarep.2007.12.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">The consequences of Rad51 overexpression for normal and tumor cells</span></div><div class="casAuthors">Klein, Hannah L.</div><div class="citationInfo"><span class="NLM_cas:title">DNA Repair</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">686-693</span>CODEN:
                <span class="NLM_cas:coden">DRNEAR</span>;
        ISSN:<span class="NLM_cas:issn">1568-7864</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The Rad51 recombinase is an essential factor for homologous recombination and the repair of DNA double strand breaks, binding transiently to both single stranded and double stranded DNA during the recombination reaction.  The use of a homologous recombination mechanism to repair DNA damage is controlled at several levels, including the binding of Rad51 to single stranded DNA to form the Rad51 nucleofilament, which is controlled through the action of DNA helicases that can counteract nucleofilament formation.  Overexpression of Rad51 in different organisms and cell types has a wide assortment of consequences, ranging from increased homologous recombination and increased resistance to DNA damaging agents to disruption of the cell cycle and apoptotic cell death.  Rad51 expression is increased in p53-neg. cells, and since p53 is often mutated in tumor cells, there is a tendency for Rad51 to be overexpressed in tumor cells, leading to increased resistance to DNA damage and drugs used in chemotherapies.  As cells with increased Rad51 levels are more resistant to DNA damage, there is a selection for tumor cells to have higher Rad51 levels.  While increased Rad51 can provide drug resistance, it also leads to increased genomic instability and may contribute to carcinogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsGGiiU8otVLVg90H21EOLACvtfcHk0lhSl_Mxgl3KFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXls1Sgsbw%253D&md5=824aee339ab1c58745af64778d24378a</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.dnarep.2007.12.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dnarep.2007.12.008%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DH.%2BL.%26atitle%3DThe%2520consequences%2520of%2520Rad51%2520overexpression%2520for%2520normal%2520and%2520tumor%2520cells%26jtitle%3DDNA%2520Repair%26date%3D2008%26volume%3D7%26spage%3D686%26epage%3D693%26doi%3D10.1016%2Fj.dnarep.2007.12.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henning, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sturzbecher, H. W.</span></span> <span> </span><span class="NLM_article-title">Homologous recombination and cell cycle checkpoints: Rad51 in tumour progression and therapy resistance</span>. <i>Toxicology</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1016/S0300-483X(03)00291-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2FS0300-483X%2803%2900291-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=14599770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD3sXos1KhsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2003&pages=91-109&author=W.+Henningauthor=H.+W.+Sturzbecher&title=Homologous+recombination+and+cell+cycle+checkpoints%3A+Rad51+in+tumour+progression+and+therapy+resistance&doi=10.1016%2FS0300-483X%2803%2900291-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Homologous recombination and cell cycle checkpoints: Rad51 in tumor progression and therapy resistance</span></div><div class="casAuthors">Henning, Wilhelm; Sturzbecher, Horst-Werner</div><div class="citationInfo"><span class="NLM_cas:title">Toxicology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">91-109</span>CODEN:
                <span class="NLM_cas:coden">TXCYAC</span>;
        ISSN:<span class="NLM_cas:issn">0300-483X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  We provide an overview of the functional interrelationship between genes and proteins related to DNA repair by homologous recombination and cell cycle regulation in relation to the progression and therapy resistance of human tumors.  To ensure the high-fidelity transmission of genetic information from one generation to the next, cells have evolved mechanisms to monitor genome integrity.  Upon DNA damage, cells initiate complex response pathways including cell cycle arrest, activation of genes and gene products involved in DNA repair, and under some circumstances, the triggering of programmed cell death.  Deregulation of this coordinated response leads to genetic instability and is fundamental to the etiol. of human cancer.  Homologous recombination involved in DNA repair is induced by environmental damage as well as misreplication during the normal cell cycle.  However, when not regulated properly, it can result in the loss of heterozygosity or genetic rearrangements, central to the process of carcinogenesis.  The central step of homologous recombination is the DNA strand exchange reaction catalyzed by the eukaryotic Rad51 protein.  Here, we describe the recent progress in our understanding of how Rad51 is involved in the signaling and repair of DNA damage and how tumor suppressors, such as p53, ATM, BRCA1, BRCA2, BLM and FANCD2 are linked to Rad51-dependent pathways.  An increased knowledge of the role of Rad51 in DNA repair by homologous recombination and its effects on cell cycle progression, tumor development and tumor resistance may provide opportunities for identifying improved diagnostic markers and developing more effective treatments for cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDyHSTW9TAOrVg90H21EOLACvtfcHk0lhSl_Mxgl3KFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXos1KhsLc%253D&md5=b2436c287b5ee3537ea7837f4707f04f</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2FS0300-483X%2803%2900291-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0300-483X%252803%252900291-9%26sid%3Dliteratum%253Aachs%26aulast%3DHenning%26aufirst%3DW.%26aulast%3DSturzbecher%26aufirst%3DH.%2BW.%26atitle%3DHomologous%2520recombination%2520and%2520cell%2520cycle%2520checkpoints%253A%2520Rad51%2520in%2520tumour%2520progression%2520and%2520therapy%2520resistance%26jtitle%3DToxicology%26date%3D2003%26volume%3D193%26spage%3D91%26epage%3D109%26doi%3D10.1016%2FS0300-483X%2803%2900291-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yanagisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urade, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furuyama, J.</span></span> <span> </span><span class="NLM_article-title">Increased expression of human DNA repair genes, XRCC1, XRCC3 and RAD51, in radioresistant human KB carcinoma cell line N10</span>. <i>Oral Oncol.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">524</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1016/S1368-8375(98)00045-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2FS1368-8375%2898%2900045-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9930366" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK1MXitVSrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1998&pages=524-528&author=T.+Yanagisawaauthor=M.+Uradeauthor=Y.+Yamamotoauthor=J.+Furuyama&title=Increased+expression+of+human+DNA+repair+genes%2C+XRCC1%2C+XRCC3+and+RAD51%2C+in+radioresistant+human+KB+carcinoma+cell+line+N10&doi=10.1016%2FS1368-8375%2898%2900045-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Increased expression of human DNA repair genes, XRCC1, XRCC3 and RAD51, in radioresistant human KB carcinoma cell line N10</span></div><div class="casAuthors">Yanagisawa, T.; Urade, M.; Yamamoto, Y.; Furuyama, J.</div><div class="citationInfo"><span class="NLM_cas:title">Oral Oncology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">524-528</span>CODEN:
                <span class="NLM_cas:coden">EJCCER</span>;
        ISSN:<span class="NLM_cas:issn">0964-1955</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The radioresistant N10 and parental KB cell lines were examd. for the expression of human DNA repair genes which were related to the repair of radiation-induced DNA damage by northern blot anal. using five kinds of DNA probes (XRCC1, XRCC3, XRCC5, RAD51, RAD52).  In the unirradiated condition, N10 cells showed higher expression of XRCC1, XRCC3 and RAD51 mRNA than did KB cells.  The X-irradn. induced a time-dependent increase in the mRNA levels of XRCC3 and RAD51 in both cell lines with a max. at 2 h postirradn.  The XRCC1 mRNA in N10 was maintained at the same level even after irradn., whereas that in KB was decreased after irradn.  There was no difference in the expression of XRCC5 and RAD52 mRNA between N10 and KB cells in both unirradiated and irradiated conditions.  From these findings, it was suggested that XRCC1, XRCC3 and RAD51 contribute to the radioresistance in cell line N10.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSOn6VPpumMLVg90H21EOLACvtfcHk0lhwO1YCunOwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXitVSrsw%253D%253D&md5=0446ed646f85ef5372de148ab1123a2c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2FS1368-8375%2898%2900045-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1368-8375%252898%252900045-1%26sid%3Dliteratum%253Aachs%26aulast%3DYanagisawa%26aufirst%3DT.%26aulast%3DUrade%26aufirst%3DM.%26aulast%3DYamamoto%26aufirst%3DY.%26aulast%3DFuruyama%26aufirst%3DJ.%26atitle%3DIncreased%2520expression%2520of%2520human%2520DNA%2520repair%2520genes%252C%2520XRCC1%252C%2520XRCC3%2520and%2520RAD51%252C%2520in%2520radioresistant%2520human%2520KB%2520carcinoma%2520cell%2520line%2520N10%26jtitle%3DOral%2520Oncol.%26date%3D1998%26volume%3D34%26spage%3D524%26epage%3D528%26doi%3D10.1016%2FS1368-8375%2898%2900045-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blundell, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span> <span> </span><span class="NLM_article-title">Insights into DNA recombination from the structure of a RAD51-BRCA2 complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>420</i></span>,  <span class="NLM_fpage">287</span>– <span class="NLM_lpage">293</span>, <span class="refDoi"> DOI: 10.1038/nature01230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnature01230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=12442171" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD38XovVektLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=420&publication_year=2002&pages=287-293&author=L.+Pellegriniauthor=D.+S.+Yuauthor=T.+Loauthor=S.+Anandauthor=M.+Leeauthor=T.+M.+Blundellauthor=A.+R.+Venkitaraman&title=Insights+into+DNA+recombination+from+the+structure+of+a+RAD51-BRCA2+complex&doi=10.1038%2Fnature01230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into DNA recombination from the structure of a RAD51-BRCA2 complex</span></div><div class="casAuthors">Pellegrini, Luca; Yu, David S.; Lo, Thomas; Anand, Shubha; Lee, MiYoung; Blundell, Tom L.; Venkitaraman, Ashok R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">420</span>
        (<span class="NLM_cas:issue">6913</span>),
    <span class="NLM_cas:pages">287-293</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The breast cancer susceptibility protein BRCA2 controls the function of RAD51, a recombinase enzyme, in pathways for DNA repair by homologous recombination.  We report here the structure of a complex between an evolutionarily conserved sequence in BRCA2 (the BRC repeat) and the RecA-homol. domain of RAD51.  The BRC repeat mimics a motif in RAD51 that serves as an interface for oligomerization between individual RAD51 monomers, thus enabling BRCA2 to control the assembly of the RAD51 nucleoprotein filament, which is essential for strand-pairing reactions during DNA recombination.  The RAD51 oligomerization motif is highly conserved among RecA-like recombinases, highlighting a common evolutionary origin for the mechanism of nucleoprotein filament formation, mirrored in the BRC repeat.  Cancer-assocd. mutations that affect the BRC repeat disrupt its predicted interaction with RAD51, yielding structural insight into mechanisms for cancer susceptibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo44KTdJmBb5rVg90H21EOLACvtfcHk0lhwO1YCunOwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XovVektLY%253D&md5=5dc084f8efb75cf95bc31d4f5b1797e3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnature01230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01230%26sid%3Dliteratum%253Aachs%26aulast%3DPellegrini%26aufirst%3DL.%26aulast%3DYu%26aufirst%3DD.%2BS.%26aulast%3DLo%26aufirst%3DT.%26aulast%3DAnand%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%26aulast%3DBlundell%26aufirst%3DT.%2BM.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26atitle%3DInsights%2520into%2520DNA%2520recombination%2520from%2520the%2520structure%2520of%2520a%2520RAD51-BRCA2%2520complex%26jtitle%3DNature%26date%3D2002%26volume%3D420%26spage%3D287%26epage%3D293%26doi%3D10.1038%2Fnature01230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bork, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilges, M.</span></span> <span> </span><span class="NLM_article-title">Internal repeats in the BRCA2 protein sequence</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1038/ng0596-22</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fng0596-22" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=8673099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK28XksVWgtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1996&pages=22-23&author=P.+Borkauthor=N.+Blombergauthor=M.+Nilges&title=Internal+repeats+in+the+BRCA2+protein+sequence&doi=10.1038%2Fng0596-22"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Internal repeats in the BRCA2 protein sequence</span></div><div class="casAuthors">Bork, Peer; Blomberg, Niklas; Nilges, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-23</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">Eight repetitive units were found in the breast cancer susceptibility gene BRCA2 protein sequence.  The 8 internal repeats (BRC repeats) are located in the central portion of the protein and cover nearly a third of the protein; all repeats are encoded by the large exon 11.  The length of a minimal BRC domain should not exceed 80 amino acids, and at least 4 secondary structure elements are present in a single repeat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxD_rMsWQKAbVg90H21EOLACvtfcHk0lhwO1YCunOwvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XksVWgtbw%253D&md5=4bc5b248f589f7bb9a1c6018d45ecc17</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fng0596-22&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng0596-22%26sid%3Dliteratum%253Aachs%26aulast%3DBork%26aufirst%3DP.%26aulast%3DBlomberg%26aufirst%3DN.%26aulast%3DNilges%26aufirst%3DM.%26atitle%3DInternal%2520repeats%2520in%2520the%2520BRCA2%2520protein%2520sequence%26jtitle%3DNat.%2520Genet.%26date%3D1996%26volume%3D13%26spage%3D22%26epage%3D23%26doi%3D10.1038%2Fng0596-22" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wong, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pero, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ormonde, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavtigian, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartel, P. L.</span></span> <span> </span><span class="NLM_article-title">RAD51 interacts with the evolutionarily conserved BRC motifs in the human breast cancer susceptibility gene brca2</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>272</i></span>,  <span class="NLM_fpage">31941</span>– <span class="NLM_lpage">31944</span>, <span class="refDoi"> DOI: 10.1074/jbc.272.51.31941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1074%2Fjbc.272.51.31941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9405383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK1cXis1CmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=31941-31944&author=A.+K.+Wongauthor=R.+Peroauthor=P.+A.+Ormondeauthor=S.+V.+Tavtigianauthor=P.+L.+Bartel&title=RAD51+interacts+with+the+evolutionarily+conserved+BRC+motifs+in+the+human+breast+cancer+susceptibility+gene+brca2&doi=10.1074%2Fjbc.272.51.31941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">RAD51 interacts with the evolutionarily conserved BRC motifs in the human breast cancer susceptibility gene brca2</span></div><div class="casAuthors">Wong, Alexander K. C.; Pero, Ralph; Ormonde, Patricia A.; Tavtigian, Sean V.; Bartel, Paul L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">31941-31944</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Recent work has shown that the murine BRCA2 tumor suppressor protein interacts with the murine RAD51 protein.  This interaction suggests that BRCA2 participates in DNA repair.  Residues 3196-3232 of the murine BRCA2 protein were shown to be involved in this interaction.  Here, we report the detailed mapping of addnl. domains that are involved in interactions between the human homologs of these two proteins.  Through yeast two-hybrid and biochem. assays, we demonstrate that the RAD51 protein interacts specifically with the eight evolutionarily conserved BRC motifs encoded in exon 11 of brca2 and with a similar motif found in a Caenorhabditis elegans hypothetical protein.  Deletion anal. demonstrates that residues 98-339 of human RAD51 interact with the 59-residue minimal region that is conserved in all BRC motifs.  These data suggest that the BRC repeats function to bind RAD51.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_ZSZn6x5YvrVg90H21EOLACvtfcHk0lhRb-uZq3NGXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXis1CmtA%253D%253D&md5=5fef058fdd28151371930a354e30af86</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.51.31941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.51.31941%26sid%3Dliteratum%253Aachs%26aulast%3DWong%26aufirst%3DA.%2BK.%26aulast%3DPero%26aufirst%3DR.%26aulast%3DOrmonde%26aufirst%3DP.%2BA.%26aulast%3DTavtigian%26aufirst%3DS.%2BV.%26aulast%3DBartel%26aufirst%3DP.%2BL.%26atitle%3DRAD51%2520interacts%2520with%2520the%2520evolutionarily%2520conserved%2520BRC%2520motifs%2520in%2520the%2520human%2520breast%2520cancer%2520susceptibility%2520gene%2520brca2%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D31941%26epage%3D31944%26doi%3D10.1074%2Fjbc.272.51.31941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bignell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micklem, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stratton, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wooster, R.</span></span> <span> </span><span class="NLM_article-title">The BRC repeats are conserved in mammalian BRCA2 proteins</span>. <i>Hum. Mol. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1093/hmg/6.1.53</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fhmg%2F6.1.53" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9002670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK2sXlsleqsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1997&pages=53-58&author=G.+Bignellauthor=G.+Micklemauthor=M.+R.+Strattonauthor=A.+Ashworthauthor=R.+Wooster&title=The+BRC+repeats+are+conserved+in+mammalian+BRCA2+proteins&doi=10.1093%2Fhmg%2F6.1.53"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The BRC repeats are conserved in mammalian BRCA2 proteins</span></div><div class="casAuthors">Bignell, Graham; Micklem, Gos; Stratton, Michael R.; Ashworth, Alan; Wooster, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Human Molecular Genetics</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-58</span>CODEN:
                <span class="NLM_cas:coden">HMGEE5</span>;
        ISSN:<span class="NLM_cas:issn">0964-6906</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The breast cancer susceptibility gene BRCA2 encodes a protein of 3418 amino acids which does not exhibit substantial sequence similarity to any other protein in the public databases.  A dot matrix comparison of BRCA2 with itself revealed an eight times repeated motif in the segment of the protein encoded by exon 11.  As a preliminary test of the hypothesis that these motifs are functionally significant, we have sequenced exon 11 of BRCA2 in six mammals.  An alignment of the predicted protein sequences shows that, overall, the motifs have been conserved while much of the intervening sequences has diverged.  These data support the notion that the BRC motifs are important in BRCA2 function.  There is, however, considerable interspecies variation within certain motif units, raising the possibility of redundancy and that not all of the repeats are required for the normal function of BRCA2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIqJmZz4B52LVg90H21EOLACvtfcHk0lhRb-uZq3NGXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXlsleqsw%253D%253D&md5=6fdd1ddf4fcc0430f32e8c2b6c8ab9ca</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fhmg%2F6.1.53&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhmg%252F6.1.53%26sid%3Dliteratum%253Aachs%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DMicklem%26aufirst%3DG.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DWooster%26aufirst%3DR.%26atitle%3DThe%2520BRC%2520repeats%2520are%2520conserved%2520in%2520mammalian%2520BRCA2%2520proteins%26jtitle%3DHum.%2520Mol.%2520Genet.%26date%3D1997%26volume%3D6%26spage%3D53%26epage%3D58%26doi%3D10.1093%2Fhmg%2F6.1.53" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konig, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldblatt, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span> <span> </span><span class="NLM_article-title">A novel small molecule RAD51 inactivator overcomes imatinib-resistance in chronic myeloid leukaemia</span>. <i>EMBO Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">353</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1002/emmm.201201760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2Femmm.201201760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=23341130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjsFKhtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=353-365&author=J.+Zhuauthor=L.+Zhouauthor=G.+Wuauthor=H.+Konigauthor=X.+Linauthor=G.+Liauthor=X.+Qiuauthor=C.+Chenauthor=C.+Huauthor=E.+Goldblattauthor=R.+Bhatiaauthor=A.+R.+Chamberlinauthor=P.+Chenauthor=W.+Lee&title=A+novel+small+molecule+RAD51+inactivator+overcomes+imatinib-resistance+in+chronic+myeloid+leukaemia&doi=10.1002%2Femmm.201201760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">A novel small molecule RAD51 inactivator overcomes imatinib-resistance in chronic myeloid leukaemia</span></div><div class="casAuthors">Zhu, Jiewen; Zhou, Longen; Wu, Guikai; Konig, Heiko; Lin, Xiaoqin; Li, Guideng; Qiu, Xiao-Long; Chen, Chi-Fen; Hu, Chun-Mei; Goldblatt, Erin; Bhatia, Ravi; Chamberlin, A. Richard; Chen, Phang-Lang; Lee, Wen-Hwa</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">353-365</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">RAD51 recombinase activity plays a crit. role for cancer cell proliferation and survival, and often contributes to drug-resistance.  Abnormally elevated RAD51 function and hyperactive homologous recombination (HR) rates have been found in a panel of cancers, including breast cancer and chronic myeloid leukemia (CML).  Directly targeting RAD51 and attenuating the deregulated RAD51 activity has therefore been proposed as an alternative and supplementary strategy for cancer treatment.  Here we show that a newly identified small mol., IBR2, disrupts RAD51 multimerization, accelerates proteasome-mediated RAD51 protein degrdn., reduces ionizing radiation-induced RAD51 foci formation, impairs HR, inhibits cancer cell growth and induces apoptosis.  In a murine imatinib-resistant CML model bearing the T315I Bcr-abl mutation, IBR2, but not imatinib, significantly prolonged animal survival.  Moreover, IBR2 effectively inhibits the proliferation of CD34+ progenitor cells from CML patients resistant to known BCR-ABL inhibitors.  Therefore, small mol. inhibitors of RAD51 may suggest a novel class of broad-spectrum therapeutics for difficult-to-treat cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwQGeSwZ19nrVg90H21EOLACvtfcHk0ljkbOd3OSow1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjsFKhtr8%253D&md5=e956911425043bd5e2f18ddc0c10eec7</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Femmm.201201760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Femmm.201201760%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DKonig%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DC.%26aulast%3DGoldblatt%26aufirst%3DE.%26aulast%3DBhatia%26aufirst%3DR.%26aulast%3DChamberlin%26aufirst%3DA.%2BR.%26aulast%3DChen%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DW.%26atitle%3DA%2520novel%2520small%2520molecule%2520RAD51%2520inactivator%2520overcomes%2520imatinib-resistance%2520in%2520chronic%2520myeloid%2520leukaemia%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2013%26volume%3D5%26spage%3D353%26epage%3D365%26doi%3D10.1002%2Femmm.201201760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W. H.</span>; <span class="NLM_string-name">Chen, P. L.</span>; <span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Composition and Methods for the Disruption of BRCA2-RAD51 Interaction</span>. PCT Int. Appl. <span class="NLM_patent">WO 2006/044933</span>, <span class="NLM_year">2006</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&author=W.+H.+Lee&author=P.+L.+Chen&author=J.+Zhu&title=Composition+and+Methods+for+the+Disruption+of+BRCA2-RAD51+Interaction"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DW.%2BH.%26atitle%3DComposition%2520and%2520Methods%2520for%2520the%2520Disruption%2520of%2520BRCA2-RAD51%2520Interaction%26date%3D2006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masini, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutard, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Ianni, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manerba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farabegoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallavicini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Stefano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethality triggered by combining olaparib with BRCA2-Rad51 disruptors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2491</span>– <span class="NLM_lpage">2497</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00707</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00707" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlymt7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2491-2497&author=F.+Falchiauthor=E.+Giacominiauthor=T.+Masiniauthor=N.+Boutardauthor=L.+Di+Ianniauthor=M.+Manerbaauthor=F.+Farabegoliauthor=L.+Rossiniauthor=J.+Robertsonauthor=S.+Minucciauthor=I.+Pallaviciniauthor=G.+Di+Stefanoauthor=M.+Robertiauthor=R.+Pellicciariauthor=A.+Cavalli&title=Synthetic+lethality+triggered+by+combining+olaparib+with+BRCA2-Rad51+disruptors&doi=10.1021%2Facschembio.7b00707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethality Triggered by Combining Olaparib with BRCA2-Rad51 Disruptors</span></div><div class="casAuthors">Falchi, Federico; Giacomini, Elisa; Masini, Tiziana; Boutard, Nicolas; Di Ianni, Lorenza; Manerba, Marcella; Farabegoli, Fulvia; Rossini, Lara; Robertson, Janet; Minucci, Saverio; Pallavicini, Isabella; Di Stefano, Giuseppina; Roberti, Marinella; Pellicciari, Roberto; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2491-2497</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In BRCA2-defective cells, poly(ADP [ADP]-ribose) polymerase inhibitors can trigger synthetic lethality, as two independent DNA-repairing mechanisms are simultaneously impaired.  Here, we have pharmacol. induced synthetic lethality, which was triggered by combining two different small org. mols.  When administered with a BRCA2-Rad51 disruptor in nonmutant cells, Olaparib showed anticancer activity comparable to that shown when administered alone in BRCA2-defective cells.  This strategy could represent an innovative approach to anticancer drug discovery and could be extended to other synthetic lethality pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh9-yqLe986rVg90H21EOLACvtfcHk0ljkbOd3OSow1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlymt7fF&md5=f2196fb93d0e86d8aa4f090e0434afc4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00707%26sid%3Dliteratum%253Aachs%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DGiacomini%26aufirst%3DE.%26aulast%3DMasini%26aufirst%3DT.%26aulast%3DBoutard%26aufirst%3DN.%26aulast%3DDi%2BIanni%26aufirst%3DL.%26aulast%3DManerba%26aufirst%3DM.%26aulast%3DFarabegoli%26aufirst%3DF.%26aulast%3DRossini%26aufirst%3DL.%26aulast%3DRobertson%26aufirst%3DJ.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DDi%2BStefano%26aufirst%3DG.%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DSynthetic%2520lethality%2520triggered%2520by%2520combining%2520olaparib%2520with%2520BRCA2-Rad51%2520disruptors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2491%26epage%3D2497%26doi%3D10.1021%2Facschembio.7b00707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roberti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bagnolini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milano, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacomini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falchi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balboni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manerba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farabegoli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Franco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallavicini, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Stefano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girotto, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Rad51/BRCA2 disruptors inhibit homologous recombination and synergize with olaparib in pancreatic cancer cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1016%2Fj.ejmech.2019.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=30660828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVOku7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2019&pages=80-92&author=M.+Robertiauthor=F.+Schipaniauthor=G.+Bagnoliniauthor=D.+Milanoauthor=E.+Giacominiauthor=F.+Falchiauthor=A.+Balboniauthor=M.+Manerbaauthor=F.+Farabegoliauthor=F.+De+Francoauthor=J.+Robertsonauthor=S.+Minucciauthor=I.+Pallaviciniauthor=G.+Di+Stefanoauthor=S.+Girottoauthor=R.+Pellicciariauthor=A.+Cavalli&title=Rad51%2FBRCA2+disruptors+inhibit+homologous+recombination+and+synergize+with+olaparib+in+pancreatic+cancer+cells&doi=10.1016%2Fj.ejmech.2019.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Rad51/BRCA2 disruptors inhibit homologous recombination and synergize with olaparib in pancreatic cancer cells</span></div><div class="casAuthors">Roberti, Marinella; Schipani, Fabrizio; Bagnolini, Greta; Milano, Domenico; Giacomini, Elisa; Falchi, Federico; Balboni, Andrea; Manerba, Marcella; Farabegoli, Fulvia; De Franco, Francesca; Robertson, Janet; Minucci, Saverio; Pallavicini, Isabella; Di Stefano, Giuseppina; Girotto, Stefania; Pellicciari, Roberto; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-92</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Olaparib is a PARP inhibitor (PARPi).  For patients bearing BRCA1 or BRCA2 mutations, olaparib is approved to treat ovarian cancer and in clin. trials to treat breast and pancreatic cancers.  In BRCA2-defective patients, PARPi inhibits DNA single-strand break repair, while BRCA2 mutations hamper double-strand break repair.  Recently, the authors identified a series of triazole derivs. that mimic BRCA2 mutations by disrupting the Rad51-BRCA2 interaction and thus double-strand break repair.  Here, the authors have computationally designed, synthesized, and tested over 40 novel derivs.  Addnl., they designed and conducted novel biol. assays to characterize how they disrupt the Rad51-BRCA2 interaction and inhibit double-strand break repair.  These compds., synergized with olaparib to target pancreatic cancer cells with functional BRCA2.  Compd. I significantly increased olaparib's effect on the viability of BxPC-3 cells.  This supports the idea that small org. mols. can mimic genetic mutations to improve the profile of anticancer drugs for precision medicine.  Moreover, this paradigm could be exploited in other genetic pathways to discover innovative anticancer targets and drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrO64ucZDf8MLVg90H21EOLACvtfcHk0ljR14NmpnWBqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVOku7k%253D&md5=3090401197babb650036d569a0195a7a</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DRoberti%26aufirst%3DM.%26aulast%3DSchipani%26aufirst%3DF.%26aulast%3DBagnolini%26aufirst%3DG.%26aulast%3DMilano%26aufirst%3DD.%26aulast%3DGiacomini%26aufirst%3DE.%26aulast%3DFalchi%26aufirst%3DF.%26aulast%3DBalboni%26aufirst%3DA.%26aulast%3DManerba%26aufirst%3DM.%26aulast%3DFarabegoli%26aufirst%3DF.%26aulast%3DDe%2BFranco%26aufirst%3DF.%26aulast%3DRobertson%26aufirst%3DJ.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DPallavicini%26aufirst%3DI.%26aulast%3DDi%2BStefano%26aufirst%3DG.%26aulast%3DGirotto%26aufirst%3DS.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRad51%252FBRCA2%2520disruptors%2520inhibit%2520homologous%2520recombination%2520and%2520synergize%2520with%2520olaparib%2520in%2520pancreatic%2520cancer%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D165%26spage%3D80%26epage%3D92%26doi%3D10.1016%2Fj.ejmech.2019.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+molecules%2C+big+targets%3A+drug+discovery+faces+the+protein-protein+interaction+challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0ljR14NmpnWBqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520molecules%252C%2520big%2520targets%253A%2520drug%2520discovery%2520faces%2520the%2520protein-protein%2520interaction%2520challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rajendra, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span> <span> </span><span class="NLM_article-title">Two modules in the BRC repeats of BRCA2 mediate structural and functional interactions with the RAD51 recombinase</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">82</span>– <span class="NLM_lpage">96</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp873</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fnar%2Fgkp873" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=19875419" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtF2nuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=82-96&author=E.+Rajendraauthor=A.+R.+Venkitaraman&title=Two+modules+in+the+BRC+repeats+of+BRCA2+mediate+structural+and+functional+interactions+with+the+RAD51+recombinase&doi=10.1093%2Fnar%2Fgkp873"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Two modules in the BRC repeats of BRCA2 mediate structural and functional interactions with the RAD51 recombinase</span></div><div class="casAuthors">Rajendra, Eeson; Venkitaraman, Ashok R.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-96</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The breast and ovarian cancer suppressor protein BRCA2 controls the RAD51 recombinase in reactions that lead to homologous DNA recombination (HDR).  BRCA2 binds RAD51 via eight conserved BRC repeat motifs of approx. 35 amino acids, each with a varying capacity to bind RAD51.  BRC repeats both promote and inhibit RAD51 assembly on different DNA substrates to regulate HDR, but the structural basis for these functions is unclear.  Here, we demarcate two tetrameric clusters of hydrophobic residues in the BRC repeats, interacting with distinct pockets in RAD51, and show that the co-location of both modules within a single BRC repeat is necessary for BRC-RAD51 binding and function.  The two modules comprise the sequence FxxA, known to inhibit RAD51 assembly by blocking the oligomerization interface, and a previously unrecognized tetramer with the consensus sequence LFDE, which binds to a RAD51 pocket distinct from this interface.  The LFDE motif is essential in BRC repeats for modes of RAD51 binding both permissive and inhibitory to RAD51 oligomerization.  Targeted insertion of point mutations in RAD51 that disrupt the LFDE-binding pocket impair its assembly at DNA damage sites in living cells.  Our findings suggest a model for the modular architecture of BRC repeats that provides fresh insight into the mechanisms regulating homologous DNA recombination.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrl7J1JIDpuErVg90H21EOLACvtfcHk0lgfwJYcKKWz4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtF2nuw%253D%253D&md5=8fec23291c3037866be93b27f8a6f685</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp873%26sid%3Dliteratum%253Aachs%26aulast%3DRajendra%26aufirst%3DE.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26atitle%3DTwo%2520modules%2520in%2520the%2520BRC%2520repeats%2520of%2520BRCA2%2520mediate%2520structural%2520and%2520functional%2520interactions%2520with%2520the%2520RAD51%2520recombinase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3D82%26epage%3D96%26doi%3D10.1093%2Fnar%2Fgkp873" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomme, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takizawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renodon-Corniere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleury, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurumizaka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of filament formation of human Rad51 protein by a small peptide derived from the BRC-motif of the BRCA2 protein</span>. <i>Genes Cells</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1111/j.1365-2443.2008.01180.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1111%2Fj.1365-2443.2008.01180.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=18429819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmt1SgtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=471-481&author=J.+Nommeauthor=Y.+Takizawaauthor=S.+F.+Martinezauthor=A.+Renodon-Corniereauthor=F.+Fleuryauthor=P.+Weigelauthor=K.+Yamamotoauthor=H.+Kurumizakaauthor=M.+Takahashi&title=Inhibition+of+filament+formation+of+human+Rad51+protein+by+a+small+peptide+derived+from+the+BRC-motif+of+the+BRCA2+protein&doi=10.1111%2Fj.1365-2443.2008.01180.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of filament formation of human Rad51 protein by a small peptide derived from the BRC-motif of the BRCA2 protein</span></div><div class="casAuthors">Nomme, Julian; Takizawa, Yoshimasa; Martinez, Susan F.; Renodon-Corniere, Axelle; Fleury, Fabrice; Weigel, Pierre; Yamamoto, Ken-ichi; Kurumizaka, Hitoshi; Takahashi, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Genes to Cells</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-481</span>CODEN:
                <span class="NLM_cas:coden">GECEFL</span>;
        ISSN:<span class="NLM_cas:issn">1356-9597</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Inc.</span>)
        </div><div class="casAbstract">Human Rad51 is a key element of recombinational DNA repair and is related to the resistance of cancer cells to chemo- and radiotherapies.  The protein is thus a potential target of anti-cancer treatment.  The crystallog. anal. shows that the BRC-motif of the BRCA2 tumor suppressor is in contact with the subunit-subunit interface of Rad51 and could thus prevent filament formation of Rad51.  However, biochem. anal. indicates that a BRC-motif peptide of 69 amino acids preferentially binds to the N-terminal part of Rad51.  We show exptl. that a short peptide of 28 amino acids derived from the BRC4 motif binds to the subunit-subunit interface and dissocs. its filament, both in the presence and absence of DNA, certainly by binding to dissocd. monomers.  The inhibition is efficient and specific for Rad51: the peptide does not even interact with Rad51 homologs or prevent their interaction with DNA.  Neither the N-terminal nor the C-terminal half of the peptide interacts with human Rad51, indicating that both parts are involved in the interaction, as expected from the crystal structure.  These results suggest the possibility of developing inhibitors of human Rad51 based on this peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4OeZl8N7HpLVg90H21EOLACvtfcHk0lgfwJYcKKWz4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXmt1SgtLc%253D&md5=9a0525500b096f871c6ad7ffcf9d9a3e</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2443.2008.01180.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2443.2008.01180.x%26sid%3Dliteratum%253Aachs%26aulast%3DNomme%26aufirst%3DJ.%26aulast%3DTakizawa%26aufirst%3DY.%26aulast%3DMartinez%26aufirst%3DS.%2BF.%26aulast%3DRenodon-Corniere%26aufirst%3DA.%26aulast%3DFleury%26aufirst%3DF.%26aulast%3DWeigel%26aufirst%3DP.%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DKurumizaka%26aufirst%3DH.%26aulast%3DTakahashi%26aufirst%3DM.%26atitle%3DInhibition%2520of%2520filament%2520formation%2520of%2520human%2520Rad51%2520protein%2520by%2520a%2520small%2520peptide%2520derived%2520from%2520the%2520BRC-motif%2520of%2520the%2520BRCA2%2520protein%26jtitle%3DGenes%2520Cells%26date%3D2008%26volume%3D13%26spage%3D471%26epage%3D481%26doi%3D10.1111%2Fj.1365-2443.2008.01180.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomme, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renodon-Corniere, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asanomi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stasiak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span> <span> </span><span class="NLM_article-title">Design of potent inhibitors of human RAD51 recombinase based on BRC motifs of BRCA2 protein: modeling and experimental validation of a chimera peptide</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">5782</span>– <span class="NLM_lpage">5791</span>, <span class="refDoi"> DOI: 10.1021/jm1002974</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1002974" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3cXos1ersro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5782-5791&author=J.+Nommeauthor=A.+Renodon-Corniereauthor=Y.+Asanomiauthor=K.+Sakaguchiauthor=A.+Z.+Stasiakauthor=A.+Stasiakauthor=B.+Nordenauthor=V.+Tranauthor=M.+Takahashi&title=Design+of+potent+inhibitors+of+human+RAD51+recombinase+based+on+BRC+motifs+of+BRCA2+protein%3A+modeling+and+experimental+validation+of+a+chimera+peptide&doi=10.1021%2Fjm1002974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Potent Inhibitors of Human RAD51 Recombinase Based on BRC Motifs of BRCA2 Protein: Modeling and Experimental Validation of a Chimera Peptide</span></div><div class="casAuthors">Nomme, Julian; Renodon-Corniere, Axelle; Asanomi, Yuya; Sakaguchi, Kazuyasu; Stasiak, Alicja Z.; Stasiak, Andrzej; Norden, Bengt; Tran, Vinh; Takahashi, Masayuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5782-5791</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We have previously shown that a 28-amino acid peptide derived from the BRC4 motif of BRCA2 tumor suppressor inhibits selectively human RAD51 recombinase (HsRad51).  With the aim of designing better inhibitors for cancer treatment, we combined an in silico docking approach with in vitro biochem. testing to construct a highly efficient chimera peptide from eight existing human BRC motifs.  We built a mol. model of all BRC motifs complexed with HsRad51 based on the crystal structure of the BRC4 motif-HsRad51 complex, computed the interaction energy of each residue in each BRC motif, and selected the best amino acid residue at each binding position.  This anal. enabled us to propose four amino acid substitutions in the BRC4 motif.  Three of these increased the inhibitory effect in vitro, and this effect was found to be additive.  We thus obtained a peptide that is about 10 times more efficient in inhibiting HsRad51-ssDNA complex formation than the original peptide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBdxPy1TGL2rVg90H21EOLACvtfcHk0lhGh_9-0ocdjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXos1ersro%253D&md5=3b6a3ee9962a5dc4966e5dad9653835d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm1002974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1002974%26sid%3Dliteratum%253Aachs%26aulast%3DNomme%26aufirst%3DJ.%26aulast%3DRenodon-Corniere%26aufirst%3DA.%26aulast%3DAsanomi%26aufirst%3DY.%26aulast%3DSakaguchi%26aufirst%3DK.%26aulast%3DStasiak%26aufirst%3DA.%2BZ.%26aulast%3DStasiak%26aufirst%3DA.%26aulast%3DNorden%26aufirst%3DB.%26aulast%3DTran%26aufirst%3DV.%26aulast%3DTakahashi%26aufirst%3DM.%26atitle%3DDesign%2520of%2520potent%2520inhibitors%2520of%2520human%2520RAD51%2520recombinase%2520based%2520on%2520BRC%2520motifs%2520of%2520BRCA2%2520protein%253A%2520modeling%2520and%2520experimental%2520validation%2520of%2520a%2520chimera%2520peptide%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5782%26epage%3D5791%26doi%3D10.1021%2Fjm1002974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Acker, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vance, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slabber, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bacsa, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traynelis, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">liotta, D. C.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships and pharmacophore model of a noncompetitive pyrazoline containing class of GluN2C/GluN2D selective antagonists</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6434</span>– <span class="NLM_lpage">6456</span>, <span class="refDoi"> DOI: 10.1021/jm400652r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400652r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Slsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6434-6456&author=T.+M.+Ackerauthor=A.+Khatriauthor=K.+M.+Vanceauthor=C.+Slabberauthor=J.+Bacsaauthor=J.+P.+Snyderauthor=S.+F.+Traynelisauthor=D.+C.+liotta&title=Structure-activity+relationships+and+pharmacophore+model+of+a+noncompetitive+pyrazoline+containing+class+of+GluN2C%2FGluN2D+selective+antagonists&doi=10.1021%2Fjm400652r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Activity Relationships and Pharmacophore Model of a Noncompetitive Pyrazoline Containing Class of GluN2C/GluN2D Selective Antagonists</span></div><div class="casAuthors">Acker, Timothy M.; Khatri, Alpa; Vance, Katie M.; Slabber, Cathryn; Bacsa, John; Snyder, James P.; Traynelis, Stephen F.; Liotta, Dennis C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6434-6456</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here we describe the synthesis and structure-activity relationship for a class of pyrazoline-contg. dihydroquinolone neg. allosteric modulators (I) of the NMDA receptor that show strong subunit selectivity for GluN2C- and GluN2D-contg. receptors over GluN2A- and GluN2B-contg. receptors.  Several members of this class inhibit NMDA receptor responses in the nanomolar range and are more than 50-fold selective over GluN1/GluN2A and GluN1/GluN2B NMDA receptors, as well as AMPA, kainate, GABA, glycine, nicotinic, serotonin, and purinergic receptors.  Anal. of the purified enantiomers of one of the more potent and selective compds. (II) shows that the S-enantiomer is both more potent and more selective than the R-enantiomer.  The S-enantiomer had an IC50 of 0.17-0.22 μM at GluN2D- and GluN2C-contg. receptors, resp., and showed over 70-fold selectivity over other NMDA receptor subunits.  The subunit selectivity of this class of compds. should be useful in defining the role of GluN2C- and GluN2D-contg. receptors in specific brain circuits in both physiol. and pathophysiol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowVI7nRMtmybVg90H21EOLACvtfcHk0lhGh_9-0ocdjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Slsb3J&md5=0b0c940da1bf0c21012f0457aa8335a5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm400652r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400652r%26sid%3Dliteratum%253Aachs%26aulast%3DAcker%26aufirst%3DT.%2BM.%26aulast%3DKhatri%26aufirst%3DA.%26aulast%3DVance%26aufirst%3DK.%2BM.%26aulast%3DSlabber%26aufirst%3DC.%26aulast%3DBacsa%26aufirst%3DJ.%26aulast%3DSnyder%26aufirst%3DJ.%2BP.%26aulast%3DTraynelis%26aufirst%3DS.%2BF.%26aulast%3Dliotta%26aufirst%3DD.%2BC.%26atitle%3DStructure-activity%2520relationships%2520and%2520pharmacophore%2520model%2520of%2520a%2520noncompetitive%2520pyrazoline%2520containing%2520class%2520of%2520GluN2C%252FGluN2D%2520selective%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6434%26epage%3D6456%26doi%3D10.1021%2Fjm400652r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sankaran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uvarani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandraprakash, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Umamaheswari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, P. S.</span></span> <span> </span><span class="NLM_article-title">An Expedient Approach for the Synthesis of Bioactive Pyrazole, Isoxazole and Benzodiazepine-Substituted Quinolin-2(1H)-one Derivatives</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1082</span>– <span class="NLM_lpage">1092</span>, <span class="refDoi"> DOI: 10.1002/jhet.2192</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2Fjhet.2192" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Gls7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2015&pages=1082-1092&author=M.+Sankaranauthor=C.+Uvaraniauthor=K.+Chandraprakashauthor=M.+Umamaheswariauthor=P.+S.+Mohan&title=An+Expedient+Approach+for+the+Synthesis+of+Bioactive+Pyrazole%2C+Isoxazole+and+Benzodiazepine-Substituted+Quinolin-2%281H%29-one+Derivatives&doi=10.1002%2Fjhet.2192"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">An Expedient Approach for the Synthesis of Bioactive Pyrazole, Isoxazole and Benzodiazepine-Substituted Quinolin-2(1H)-one Derivatives</span></div><div class="casAuthors">Sankaran, Mathan; Uvarani, Chokkalingam; Chandraprakash, Kumarasamy; Umamaheswari, Mani; Mohan, Palathurai Subramaniam</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1082-1092</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A sequence of functionalized pyrazole, isoxazole and benzodiazepine-substituted quinolone derivs. were synthesized in good yield starting from readily available starting precursors.  These approaches lead to the synthesis of hitherto unknown compds. with a varied substitution pattern by both conventional and microwave-assisted method.  A good no. of analogs were evaluated for their in vitro cytotoxicity against human cervical and colon cancer cell lines by MTT protocol.  Almost all the selected compds. showed remarkable cytotoxic activities.  Among them, pyrazole-substituted quinolone derivs. I [R = Ph; R1 = H, Cl; R2 = Ph, 4-MeOC6H4; R3 = Ph, 4-FC6H4] emerged as the most promising scaffolds.  These scaffolds would be used for further mol. level studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwrxpMoc_4lLVg90H21EOLACvtfcHk0lgiQ_dVxTyNaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Gls7vL&md5=2cee1203f9021dcf7352616553145917</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjhet.2192&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.2192%26sid%3Dliteratum%253Aachs%26aulast%3DSankaran%26aufirst%3DM.%26aulast%3DUvarani%26aufirst%3DC.%26aulast%3DChandraprakash%26aufirst%3DK.%26aulast%3DUmamaheswari%26aufirst%3DM.%26aulast%3DMohan%26aufirst%3DP.%2BS.%26atitle%3DAn%2520Expedient%2520Approach%2520for%2520the%2520Synthesis%2520of%2520Bioactive%2520Pyrazole%252C%2520Isoxazole%2520and%2520Benzodiazepine-Substituted%2520Quinolin-2%25281H%2529-one%2520Derivatives%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2015%26volume%3D52%26spage%3D1082%26epage%3D1092%26doi%3D10.1002%2Fjhet.2192" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abbott, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holt, J. T.</span></span> <span> </span><span class="NLM_article-title">Double-strand break repair deficiency and radiation sensitivity in BRCA2 mutant cancer cells</span>. <i>J. Natl. Cancer. Inst.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">978</span>– <span class="NLM_lpage">985</span>, <span class="refDoi"> DOI: 10.1093/jnci/90.13.978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1093%2Fjnci%2F90.13.978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=9665145" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=1%3ACAS%3A528%3ADyaK1cXksVSnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1998&pages=978-985&author=D.+W.+Abbottauthor=M.+L.+Freemanauthor=J.+T.+Holt&title=Double-strand+break+repair+deficiency+and+radiation+sensitivity+in+BRCA2+mutant+cancer+cells&doi=10.1093%2Fjnci%2F90.13.978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Double-strand break repair deficiency and radiation sensitivity in BRCA2 mutant cancer cells</span></div><div class="casAuthors">Abbott, Derek W.; Freeman, Michael L.; Holt, Jeffrey T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">978-985</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The protein product of the BRCA2 gene mediates repair of double-strand breaks in DNA.  Because a no. of cancer therapies exert cytotoxic effects via the initiation of double-strand breaks, cancers comprised of cells carrying BRCA2 gene mutations may be more amenable to treatment with agents that cause such breaks.  We identified a human pancreatic adenocarcinoma cell line lacking one copy of the BRCA2 gene and contg. a mutation (6174delT) in the remaining copy.  In vitro and in vivo expts. were conducted with this cell line and with other carcinoma cell lines matched for similar genetic mutations, similar differentiation status, and/or similar carcinoma type to examine double-strand break repair, sensitivity to drugs that induce double-strand breaks, and radiation sensitivity.  BRCA2-defective cells were unable to repair the double-strand DNA breaks induced by ionizing radiation.  These cells were also markedly sensitive to mitoxantrone, amsacrine, and etoposide (drugs that induce double-strand breaks and to ionizing radiation).  Introduction of antisense BRCA2 deoxyribonucleotides into cells possessing normal BRCA2 function led to increased sensitivity to mitoxantrone.  Tumors formed by injection of BRCA2-defective cells into nude mice were highly sensitive (>90% tumor size redn.) to both ionizing radiation and mitoxantrone when compared with tumors exhibiting normal BRCA2 function.  Histol. anal. of irradiated BRCA2-defective tumors showed a large degree of necrosis compared with that obsd. for control tumors possessing normal BRCA2 function.  BRCA2-defective cancer cells are highly sensitive to agents that cause double-strand breaks in DNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBRI6YDqpNubVg90H21EOLACvtfcHk0lgiQ_dVxTyNaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXksVSnsbg%253D&md5=b26b75011711ff951d10f6398e7dc31e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F90.13.978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F90.13.978%26sid%3Dliteratum%253Aachs%26aulast%3DAbbott%26aufirst%3DD.%2BW.%26aulast%3DFreeman%26aufirst%3DM.%2BL.%26aulast%3DHolt%26aufirst%3DJ.%2BT.%26atitle%3DDouble-strand%2520break%2520repair%2520deficiency%2520and%2520radiation%2520sensitivity%2520in%2520BRCA2%2520mutant%2520cancer%2520cells%26jtitle%3DJ.%2520Natl.%2520Cancer.%2520Inst.%26date%3D1998%26volume%3D90%26spage%3D978%26epage%3D985%26doi%3D10.1093%2Fjnci%2F90.13.978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnellmann, R. G.</span></span> <span> </span><span class="NLM_article-title">Measurement of cell death in mammalian cells</span>. <i>Curr. Protoc. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">12.8.1</span>– <span class="NLM_lpage">12.8.22</span>, <span class="refDoi"> DOI: 10.1002/0471141755.ph1208s25</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;key=10.1002%2F0471141755.ph1208s25" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=12.8.1-12.8.22&author=B.+S.+Cummingsauthor=R.+G.+Schnellmann&title=Measurement+of+cell+death+in+mammalian+cells&doi=10.1002%2F0471141755.ph1208s25"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1002%2F0471141755.ph1208s25&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471141755.ph1208s25%26sid%3Dliteratum%253Aachs%26aulast%3DCummings%26aufirst%3DB.%2BS.%26aulast%3DSchnellmann%26aufirst%3DR.%2BG.%26atitle%3DMeasurement%2520of%2520cell%2520death%2520in%2520mammalian%2520cells%26jtitle%3DCurr.%2520Protoc.%2520Pharmacol.%26date%3D2004%26volume%3D25%26spage%3D12.8.1%26epage%3D12.8.22%26doi%3D10.1002%2F0471141755.ph1208s25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N0W" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N0W','PDB','1N0W'); return false;">PDB: 1N0W</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i150"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01526">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_13158"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.9b01526?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01526</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_001.csv">CSV</a>)</p></li><li><p class="inline">Structure representation of <a id="rightTab-" class=" internalNav" href="#fig3">Figure <a id="rightTab-" class=" internalNav" href="#fig3">3</a></a>A (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_002.pdb">PDB</a>)</p></li><li><p class="inline">Structure representation of <a id="rightTab-" class=" internalNav" href="#fig3">Figure <a id="rightTab-" class=" internalNav" href="#fig3">3</a></a>B (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_003.pdb">PDB</a>)</p></li><li><p class="inline">Computational methods; separation and ELISA assay results of enantiomers <b>4d-I</b> and <b>4d-II</b>; experimental procedure and characterization data of aldehydes <b>67</b>, <b>70</b>, <b>74</b>–<b>84</b>; <sup>1</sup>H and <sup>13</sup>C NMR spectra of final compounds <b>4d</b>–<b>10d</b>, <b>14d</b>–<b>15d</b>, <b>18d</b>–<b>57d</b>; HPLC–MS analysis of selected final compounds <b>4d</b>–<b>10d</b>, <b>14d</b>–<b>15d</b>, <b>18</b>–<b>25d</b>, <b>35d</b>, <b>49d</b>, <b>57d</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_004.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_001.csv">jm9b01526_si_001.csv (4.88 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_002.pdb">jm9b01526_si_002.pdb (271.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_003.pdb">jm9b01526_si_003.pdb (271.69 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01526/suppl_file/jm9b01526_si_004.pdf">jm9b01526_si_004.pdf (10.94 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01526&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-5%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01526" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01526" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d2d23e163d48","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
