
<html lang="en"     class="pb-page"  data-request-id="6185d906-ceae-4af5-a78f-95cde09dbcac"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.0c02036;issue:issue:10.1021/jmcmar.2021.64.issue-9;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Modulation of the Passive Permeability of Semipeptidic Macrocycles: N- and C-Methylations Fine-Tune Conformation and Properties" /></meta><meta name="dc.Creator" content="Christian  Comeau" /></meta><meta name="dc.Creator" content="Benjamin  Ries" /></meta><meta name="dc.Creator" content="Thomas  Stadelmann" /></meta><meta name="dc.Creator" content="Jacob  Tremblay" /></meta><meta name="dc.Creator" content="Sylvain  Poulet" /></meta><meta name="dc.Creator" content="Ulrike  Fröhlich" /></meta><meta name="dc.Creator" content="Jérôme  Côté" /></meta><meta name="dc.Creator" content="Pierre-Luc  Boudreault" /></meta><meta name="dc.Creator" content="Rabeb Mouna  Derbali" /></meta><meta name="dc.Creator" content="Philippe  Sarret" /></meta><meta name="dc.Creator" content="Michel  Grandbois" /></meta><meta name="dc.Creator" content="Grégoire  Leclair" /></meta><meta name="dc.Creator" content="Sereina  Riniker" /></meta><meta name="dc.Creator" content="Éric  Marsault" /></meta><meta name="dc.Description" content="Incorporating small modifications to peptidic macrocycles can have a major influence on their properties. For instance, N-methylation has been shown to impact permeability. A better understanding o..." /></meta><meta name="Description" content="Incorporating small modifications to peptidic macrocycles can have a major influence on their properties. For instance, N-methylation has been shown to impact permeability. A better understanding o..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 22, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02036" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 The Authors. Published by American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02036" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02036" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02036" /></link>
        
    
    

<title>Modulation of the Passive Permeability of Semipeptidic Macrocycles: N- and C-Methylations Fine-Tune Conformation and Properties | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02036" /></meta><meta property="og:title" content="Modulation of the Passive Permeability of Semipeptidic Macrocycles: N- and C-Methylations Fine-Tune Conformation and Properties" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0014.jpeg" /></meta><meta property="og:description" content="Incorporating small modifications to peptidic macrocycles can have a major influence on their properties. For instance, N-methylation has been shown to impact permeability. A better understanding of the relationship between permeability and structure is of key importance as peptidic drugs are often associated with unfavorable pharmacokinetic profiles. Starting from a semipeptidic macrocycle backbone composed of a tripeptide tethered head-to-tail with an alkyl linker, we investigated two small changes: peptide-to-peptoid substitution and various methyl placements on the nonpeptidic linker. Implementing these changes in parallel, we created a collection of 36 compounds. Their permeability was then assessed in parallel artificial membrane permeability assay (PAMPA) and Caco-2 assays. Our results show a systematic improvement in permeability associated with one peptoid position in the cycle, while the influence of methyl substitution varies on a case-by-case basis. Using a combination of molecular dynamics simulations and NMR measurements, we offer hypotheses to explain such behavior." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02036"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02036">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02036&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02036&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02036&amp;href=/doi/10.1021/acs.jmedchem.0c02036" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 5365-5383</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01941" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01545" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Modulation of the Passive Permeability of Semipeptidic Macrocycles: N- and C-Methylations Fine-Tune Conformation and Properties</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Christian Comeau</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Comeau</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Comeau">Christian Comeau</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Benjamin Ries</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin Ries</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Applied Biosciences, ETH Zürich, Vladimir-Prelog-Weg 2, 8093 Zürich, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Ries">Benjamin Ries</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0945-8304" title="Orcid link">http://orcid.org/0000-0002-0945-8304</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Stadelmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Stadelmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Applied Biosciences, ETH Zürich, Vladimir-Prelog-Weg 2, 8093 Zürich, Switzerland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Stadelmann">Thomas Stadelmann</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2919-0939" title="Orcid link">http://orcid.org/0000-0003-2919-0939</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jacob Tremblay</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jacob Tremblay</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jacob++Tremblay">Jacob Tremblay</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sylvain Poulet</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sylvain Poulet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sylvain++Poulet">Sylvain Poulet</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ulrike Fröhlich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ulrike Fröhlich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ulrike++Fr%C3%B6hlich">Ulrike Fröhlich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jérôme Côté</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jérôme Côté</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%A9r%C3%B4me++C%C3%B4t%C3%A9">Jérôme Côté</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Pierre-Luc Boudreault</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pierre-Luc Boudreault</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pierre-Luc++Boudreault">Pierre-Luc Boudreault</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Rabeb Mouna Derbali</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rabeb Mouna Derbali</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculté de pharmacie, Université de Montréal, C.P. 6128, succursale Centre-ville, Montréal, Quebec, Canada H3C 3J7</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rabeb+Mouna++Derbali">Rabeb Mouna Derbali</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philippe Sarret</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philippe Sarret</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#9fcff7f6f3f6efeffab1ccfeededfaebb1dfcaccf7faedfdedf0f0f4fab1fcfe"><span class="__cf_email__" data-cfemail="64340c0d080d1414014a3705161601104a2431370c011606160b0b0f014a0705">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philippe++Sarret">Philippe Sarret</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7627-701X" title="Orcid link">http://orcid.org/0000-0002-7627-701X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michel Grandbois</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michel Grandbois</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michel++Grandbois">Michel Grandbois</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Grégoire Leclair</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Grégoire Leclair</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculté de pharmacie, Université de Montréal, C.P. 6128, succursale Centre-ville, Montréal, Quebec, Canada H3C 3J7</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gr%C3%A9goire++Leclair">Grégoire Leclair</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sereina Riniker</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sereina Riniker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Applied Biosciences, ETH Zürich, Vladimir-Prelog-Weg 2, 8093 Zürich, Switzerland</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#ec9f9e85828587899eac89988496c28f84"><span class="__cf_email__" data-cfemail="3241405b5c5b5957407257465a481c515a">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sereina++Riniker">Sereina Riniker</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1893-4031" title="Orcid link">http://orcid.org/0000-0003-1893-4031</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Éric Marsault</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Éric Marsault</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Département de Pharmacologie−Physiologie, Institut de Pharmacologie de Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=%C3%89ric++Marsault">Éric Marsault</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5305-8762" title="Orcid link">http://orcid.org/0000-0002-5305-8762</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02036&amp;href=/doi/10.1021%2Facs.jmedchem.0c02036" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 5365–5383</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 22, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>23 November 2020</li><li><span class="item_label"><b>Published</b> online</span>22 March 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 May 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02036" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02036</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 The Authors. Published by American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5365%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DChristian%2BComeau%252C%2BBenjamin%2BRies%252C%2BThomas%2BStadelmann%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D9%26contentID%3Dacs.jmedchem.0c02036%26title%3DModulation%2Bof%2Bthe%2BPassive%2BPermeability%2Bof%2BSemipeptidic%2BMacrocycles%253A%2BN-%2Band%2BC-Methylations%2BFine-Tune%2BConformation%2Band%2BProperties%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5383%26publicationDate%3DMay%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02036"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1272</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">1</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02036" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Modulation of the Passive Permeability of Semipeptidic Macrocycles: N- and C-Methylations Fine-Tune Conformation and Properties&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Comeau&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Ries&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Stadelmann&quot;},{&quot;first_name&quot;:&quot;Jacob&quot;,&quot;last_name&quot;:&quot;Tremblay&quot;},{&quot;first_name&quot;:&quot;Sylvain&quot;,&quot;last_name&quot;:&quot;Poulet&quot;},{&quot;first_name&quot;:&quot;Ulrike&quot;,&quot;last_name&quot;:&quot;Fröhlich&quot;},{&quot;first_name&quot;:&quot;Jérôme&quot;,&quot;last_name&quot;:&quot;Côté&quot;},{&quot;first_name&quot;:&quot;Pierre-Luc&quot;,&quot;last_name&quot;:&quot;Boudreault&quot;},{&quot;first_name&quot;:&quot;Rabeb&quot;,&quot;last_name&quot;:&quot;Mouna Derbali&quot;},{&quot;first_name&quot;:&quot;Philippe&quot;,&quot;last_name&quot;:&quot;Sarret&quot;},{&quot;first_name&quot;:&quot;Michel&quot;,&quot;last_name&quot;:&quot;Grandbois&quot;},{&quot;first_name&quot;:&quot;Grégoire&quot;,&quot;last_name&quot;:&quot;Leclair&quot;},{&quot;first_name&quot;:&quot;Sereina&quot;,&quot;last_name&quot;:&quot;Riniker&quot;},{&quot;first_name&quot;:&quot;Éric&quot;,&quot;last_name&quot;:&quot;Marsault&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;5365-5383&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02036&quot;},&quot;abstract&quot;:&quot;Incorporating small modifications to peptidic macrocycles can have a major influence on their properties. For instance, N-methylation has been shown to impact permeability. A better understanding of the relationship between permeability and structure is of key importance as peptidic drugs are often associated with unfavorable pharmacokinetic profiles. Starting from a semipeptidic macrocycle backbone composed of a tripeptide tethered head-to-tail with an alkyl linker, we investigated two small changes: peptide-to-peptoid substitution and various methyl placements on the nonpeptidic linker. Implementing these changes in parallel, we created a collection of 36 compounds. Their permeability was then assessed in parallel artificial membrane permeability assay (PAMPA) and Caco-2 assays. Our results show a systematic improvement in permeability associated with one peptoid position in the cycle, while the influence of methyl substitution varies on a case-by-case basis. Using a combination of molecular dynamics simula&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02036&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02036" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02036&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02036" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02036&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02036" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02036&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02036&amp;href=/doi/10.1021/acs.jmedchem.0c02036" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02036" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02036" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02036%26sid%3Dliteratum%253Aachs%26pmid%3D33750117%26genre%3Darticle%26aulast%3DComeau%26date%3D2021%26atitle%3DModulation%2Bof%2Bthe%2BPassive%2BPermeability%2Bof%2BSemipeptidic%2BMacrocycles%253A%2BN-%2Band%2BC-Methylations%2BFine-Tune%2BConformation%2Band%2BProperties%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D9%26spage%3D5365%26epage%3D5383%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291503" title="Organic polymers">Organic polymers</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=292504" title="Permeability">Permeability</a>,</li><li><a href="/action/doSearch?ConceptID=291163" title="Macrocycles">Macrocycles</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/jmcmar.2021.64.issue-9/20210513/jmcmar.2021.64.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Incorporating small modifications to peptidic macrocycles can have a major influence on their properties. For instance, N-methylation has been shown to impact permeability. A better understanding of the relationship between permeability and structure is of key importance as peptidic drugs are often associated with unfavorable pharmacokinetic profiles. Starting from a semipeptidic macrocycle backbone composed of a tripeptide tethered head-to-tail with an alkyl linker, we investigated two small changes: peptide-to-peptoid substitution and various methyl placements on the nonpeptidic linker. Implementing these changes in parallel, we created a collection of 36 compounds. Their permeability was then assessed in parallel artificial membrane permeability assay (PAMPA) and Caco-2 assays. Our results show a systematic improvement in permeability associated with one peptoid position in the cycle, while the influence of methyl substitution varies on a case-by-case basis. Using a combination of molecular dynamics simulations and NMR measurements, we offer hypotheses to explain such behavior.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61876" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61876" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Macrocycles have recently gathered increasing levels of interest in medicinal chemistry.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1−6)</a> Their unique combination of conformationally constrained structure and high level of structural information allows for the design of large, organized structures suitable to interact with extended and featureless binding sites such as those found in protein–protein interactions.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10">(7−10)</a> Most Food and Drug Administration (FDA)-approved macrocyclic drugs belong to natural products (e.g., erythromycin, tacrolimus) or peptides (e.g., sandostatin, eptifibatin).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Peptidic or semipeptidic scaffolds bridge the gap between small molecules and biologics, allying synthetic ease and broad choice of natural and non-natural amino acids required for rapid and thorough pharmacophoric exploration. The main challenge with peptides resides in their physicochemical and pharmacokinetics-absorption, distribution, metabolism, and excretion (PK-ADME) properties. While cyclic peptides are typically more stable to proteases compared to their linear counterparts, their high polarity often translates into low bioavailability.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Nonetheless, some cyclic peptides cross cell membranes.<a onclick="showRef(event, 'ref12 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref12 ref14 ref15">(12,14,15)</a> Developing tools and knowledge to optimize and better predict their structure–permeability relationship is therefore a requirement for the field. Such quest found inspiration in studies of the natural cycloundecapeptide cyclosporine A, which is administered orally. One prominent structural feature of this natural macrocycle is its high number of N-methylated residues (7 out of 11) and its dynamic structural adaptation to its environment (also known as (aka) chameleonic properties).<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> The effect of N-methylation on permeability of cyclic hexa- and heptapeptides has been systematically investigated since the number and position of N-methylations may be beneficial or detrimental for permeability.<a onclick="showRef(event, 'ref15 ref19 ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref15 ref19 ref20 ref21 ref22 ref23">(15,19−23)</a> Less explored are the N-alkylated glycines—aka peptoids—in which side chain has been moved from the α carbon to the amide nitrogen.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Similarly to N-methylation, this modification removes one H-bond donor, yet it also removes one stereogenic center and induces glycine-like secondary structures. The peptoid amide also facilitates cis–trans isomerization compared to the corresponding N-methylation.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Synthetically, the inclusion of peptoids is also compatible with solid-phase protocols and allows for an almost unlimited variety of side chains, where virtually any primary amine can be used.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> More recently, the impact of the dynamics of macrocycles in response to their environment, which can range from polar in water, nonhomogeneous in the presence of its target, to lipophilic in the membrane, has been appreciated.<a onclick="showRef(event, 'ref17 ref18 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref27 ref28">(17,18,27,28)</a></div><div class="NLM_p">A powerful tool to modulate the properties of peptidic macrocycles is the inclusion of a nonpeptidic tether unit.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29−31)</a> This tether can serve multiple purposes: in the context of a target interacting with a specific sequence, various tethers can be screened without modifying the peptide recognition sequence, while providing a simple handle for modulating affinity and PK properties. Small modifications in size, shape, or functional groups on the tether can dramatically influence on this kind of constrained system.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Additionally, a tether may facilitate macrocyclization, which can be challenging synthetically.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The relationship between structure and permeability is known to be elusive for this class of compounds, with small structural modifications often yielding permeability cliffs.<a onclick="showRef(event, 'ref14 ref19 ref21 ref23 ref31 ref33 ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref14 ref19 ref21 ref23 ref31 ref33 ref34 ref35 ref36">(14,19,21,23,31,33−36)</a></div><div class="NLM_p">To support our efforts in this direction<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and pinpoint the effects of conformational modulation on permeability, we synthesized a library of closely related compounds based on chemotype <b>A</b> composed of a tripeptide tethered head-to-tail with a nonpeptidic linker (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Two classes of modifications were implemented on chemotype <b>A</b>: single peptoid replacement (<b>B</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) or regio- and stereocontrolled linker C-methylation (<b>C</b>, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). All of the possible combinations of these variations were generated, providing a total of 4 × 9 = 36 compounds with identical molecular weights (except for non-methylated tether derivatives), nearly identical sequence and identical ring sizes, leaving as little room as possible for confounding factors. The passive permeability of the resulting macrocycles was measured in the parallel artificial membrane permeability assay (PAMPA) and their cellular permeability in the Caco-2 assay.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> We then selected two pairs of diastereomers that differ only by their stereochemistry of the tether methyl group yet either differ greatly in passive permeability or not, and performed molecular dynamics (MD) simulations coupled with solution NMR to rationalize the origin of these differences.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Two classes of modifications implemented on model compound (<b>A)</b>: Nala, Nleu, and Nphe peptoids (<b>B</b> showing Nleu) and regio/stereocontrolled C-methylation (<b>C</b> showing 2<i>R</i> methylation).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84383" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84383" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design</h3><div class="NLM_p last">The library was designed on a common sequence (Ala-Leu-Phe-Tether), with a single peptide-to-peptoid substitution and a single stereocontrolled methyl substitution on the tether (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). This created a two-dimensional array of analogs, which were named according to both the peptide and the tether substitution parameter. The symbol ⌀ on the peptide side denotes fully peptidic sequence, and on the tether side, it denotes no Me substitution. For example, compound <b>A</b> is named ⌀-⌀, <b>B</b> is termed Nleu-⌀, and compound <b>C</b> is termed ⌀-2<i>R</i>.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">Tethers were synthesized as summarized in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. For methyl substitution in position 2 (compound <b>7<i>S</i></b>), 5-bromovaleric acid <b>1</b> was used as the starting point. Azide substitution provided acid <b>2</b>, which was attached to Evans’ oxazolidinone chiral auxiliary to provide precursor <b>3</b><i><b>S</b></i>. The latter underwent methylation to generate <b>4</b><i><b>S</b></i>, which was followed by hydrolysis to generate azidoacid <b>5</b><i><b>S</b></i> as described by Orwig et al.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Azide reduction generated amino acid <b>6</b><i><b>S</b></i>, which was protected as a Boc carbamate to generate tether <b>7</b><i><b>S</b></i>. Its enantiomer <b>7</b><i><b>R</b></i> was obtained using the other enantiomer of the chiral auxiliary. To generate 3-substituted tethers, (<i>R</i>)-citronellol <b>8</b><i><b>R</b></i> was first oxidized to carboxylic acid <b>9</b><i><b>R</b></i>, then protected as a methyl ester <b>10</b><i><b>R</b></i>. Its double bond was then cleaved oxidatively with ozone followed by oxidative workup, yielding acid <b>11</b><i><b>R</b></i>, which was used as a substrate for Curtius rearrangement to generate Boc-protected aminoester <b>12</b><i><b>R</b></i>, which delivered Boc-protected tether <b>13</b><i><b>R</b></i> after ester hydrolysis. Its enantiomer was generated identically from (<i>S</i>)-citronellol. On the other hand, performing the Curtius rearrangement<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> from acid <b>9</b><i><b>R</b></i> generated Boc-protected amine <b>14</b><i><b>R</b></i>, which was oxidatively cleaved to Boc-protected tether <b>15</b><i><b>R</b></i>. Synthesis of 5-substituted tethers started from Boc-alanine <b>16</b><i><b>R</b></i> with homologation using the Arndt–Eistert reaction,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> followed by esterification to generate ester <b>18</b><i><b>R</b></i>, which was then reduced to the corresponding aldehyde <b>19</b><i><b>R</b></i>. The latter underwent a Horner–Wadsworth–Emmons olefination to generate unsaturated ester <b>20</b><i><b>R</b></i>. Saponification followed by double-bond hydrogenation delivered tether <b>22</b><i><b>R</b></i>. Its enantiomer was obtained from Boc-(<i>D</i>) alanine.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0012.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Tethers<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NaN<sub>3</sub>, dimethyl sulfoxide (DMSO), quant.; (ii) <i>t</i>-BuCOCl, Et<sub>3</sub>N, tetrahydrofuran (THF), 0 °C; (iii) <i>n</i>-BuLi, (<i>S</i>)-4-benzyl-2-oxazolidinone, THF, −78 °C, 81%. (b) Potassium bis(trimethylsilyl)amide (KHMDS), MeI, THF, −78 °C, 89%. (c) (i) H<sub>2</sub>O<sub>2</sub>, LiOH, H<sub>2</sub>O/THF, 98%, diastereomeric excess (de): 75%; (ii) Pd/C, H<sub>2</sub>, MeOH/AcOH; (iii) (Boc)<sub>2</sub>O, NaHCO<sub>3</sub>, dioxane/H<sub>2</sub>O, 39%. (d) (i) Pyridinium dichromate (PDC), dimethylformamide (DMF), 72%; (ii) CH<sub>2</sub>N<sub>2</sub>, Et<sub>2</sub>O, 95%. (e) RuCl<sub>3</sub>, NaIO<sub>4</sub>, CCl<sub>4</sub>/acetonitrile (ACN)/H<sub>2</sub>O, 70%. (f) (i) diphenylphosphoryl azide (DPPA), Et<sub>3</sub>N, <i>t</i>-BuOH, reflux, 56%; (ii) LiOH, MeOH/H<sub>2</sub>O, 80%. (g) Boc<sub>2</sub>O, NaN<sub>3</sub>, Zn(OTf)<sub>2</sub>, Bu<sub>4</sub>NBr, <i>t</i>-BuOH, THF, 33%. (h) (i) O<sub>3</sub>, MeOH, −78 °C; (ii) H<sub>2</sub>O<sub>2</sub>, NaOH, H<sub>2</sub>O, 60%. (i) (i) isobutylchloroformate, Et<sub>3</sub>N, THF, −10 °C; (ii) CH<sub>2</sub>N<sub>2</sub>, Et<sub>2</sub>O, 52%; (iii) AgOBn, Et<sub>3</sub>N, MeOH, 70%. (j) (i) DiBAl-H, dichloromethane (DCM), −78 °C; (ii) triethyl phosphonoacetate, NaH, THF, 0 °C, 72%. (k) (i) NaOH, H<sub>2</sub>O/MeOH, 96%; (ii) H<sub>2</sub>, Pd/C, MeOH, 99%.</p></p></figure><div class="NLM_p">With the nine tethers in hand, 36 linear precursors were assembled on solid support using standard Fmoc solid-phase peptide synthesis coupled with the native tripeptide and one peptoid variation for each amino acid position (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Formation of the three peptoids required different methodologies: loading Nphe on resin <b>30</b> was accomplished first by loading bromoacetic acid followed by addition of benzylamine. For Nleu coupling <b>28</b>, a similar two-step methodology was used with the difference that bromoacetic acid was coupled by first activating it with <i>N</i>,<i>N</i>′-diisopropylcarbodiimide (DIC). Finally, the Nala peptoid <b>26</b> was coupled directly as it is commercially available in its Fmoc-protected form. The linear peptides were then cleaved and macrocyclized in solution as indicated in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Final macrocycles were purified using MS-triggered preparative high-performance liquid chromatography (HPLC) (see the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>).</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0013.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Solid-Phase Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (exemplified for macrocycle Ø-Ø): (a) (i) Fmoc-Phe-OH, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), DCM, 16 h; (ii) piperidine, DMF, 2 × 5 min; (b) (i) Fmoc-Leu-OH, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU), DIPEA, DMF, 3 h; (ii) piperidine, DMF, 2 × 5 min; (c) (i) bromoacetic acid, DIPEA, DCM 45 min; (ii) benzylamine, DMF, 45 min; (d) (i) bromoacetic acid, DIC, DMF, 45 min; (ii) isobutylamine, DMF, 45 min (e) (i) Fmoc-Sar-OH, HATU, DIPEA, DMF, 3 h; (ii) piperidine, DMF, 2 × 5 min; (f) (i) 30% 1,1,1,3,3,3-hexafluoro-2-propanol (HFIP)/DCM, 2 × 30 min; (ii) (3-(diethoxyphosphoryloxy)-1,2,3-benzotriazin-4(3<i>H</i>)-one) (DEPBT), DIPEA, DMF, 16–72 h.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Permeability Results</h3><div class="NLM_p">The passive and cellular permeabilities of the resulting library of 36 macrocycles were determined using the PAMPA and the Caco-2 monolayer assays, respectively. The latter was performed in both directions (apical-to-basolateral and basolateral-to-apical) so as to obtain a BA/AB value as a measure of efflux. The results of those experiments are summarized below (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>) and available in detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Supporting Information</a> (SI) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Tables S16–S18</a>).</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Permeability results in the form of heatmaps. For heatmaps 1–3, the values are expressed as −log(<i>P</i><sub>app</sub>), so lower values mean higher permeability (in order of increasing permeability: blue, white, red, and black). Heatmap 4 shows the BA/AB ratio, which represents a measure of efflux.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (Top) PAMPA results with standard deviation, ranked; (middle) statistical analysis of the PAMPA data shows a significant (<i>p</i> < 0.0001; unpaired <i>t</i>-test) effect for the Nleu vs other macrocycles; and (bottom) a good correlation between PAMPA and Caco-2<sub>A→B</sub> (<i>R</i><sup>2</sup> = 0.91).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">From the PAMPA and Caco-2<sub>A→B</sub> results, it appears that: (1) the presence of a leucine peptoid (Nleu) generally increases both passive and cellular permeability significantly; (2) some epimers possess remarkably different passive permeability values; for instance, Nleu-5R and Nleu-5S differ in passive permeability by 2 orders of magnitude yet are structurally distinct only by the orientation of the linker methyl group; and (3) compared to the reference macrocycle ⌀-⌀, some modifications are detrimental to both passive and cellular permeabilities, while others are favorable.</div><div class="NLM_p">From the basolateral to apical (Caco-2<sub>B→A</sub>) results, we calculated the BA/AB ratio as a measure of efflux.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Values range from 10 to 120, which leaves no doubt that this class of compounds is generally subject to efflux. It is, however, interesting to observe that the Nleu peptoid series also stands out in the Caco-2 assay in both directions, suggesting that passive permeability is the main differentiating factor for this series of compounds, not efflux. There is in fact a good correlation (<i>R</i><sup>2</sup> = 0.91) between PAMPA and Caco-2<sub>A→B</sub> permeabilities (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). This is very different from a series of similar macrocycles published recently by our group, which bear a secondary amine instead of the third amide bond, and which showed very small PAMPA permeability distribution yet very important, SAR-able variations in Caco-2 permeability.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">To explain the high permeability observed with compounds containing Nleu, two hypotheses can be invoked: (A) the Nleu peptoid is intrinsically advantageous compared to leucine; or (B) position 2 in the ring benefits most from peptoid substitution. To answer this question, we inverted the positions of Ala and Leu in the sequence and tried both substitutions (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). PAMPA permeability supports hypothesis B: the permeability-enhancing effect observed in the Nleu series is due to the N-alkylation at position 2.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inverting peptoid positions to better understand whether higher permeabilities observed with Nleu are due to its nature or to its position within the macrocycle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">There are a few differences between an amino acid and its peptoid. Namely, the peptoid has (1) no H-bond donor; (2) a slightly stronger H-bond acceptor due to the higher substitution on the nitrogen atom; (3) a lower cis/trans transition barrier; (4) higher backbone flexibility due to the absence of a chiral side chain on the α carbon; and (5) no chirality. We designed additional compounds based on the original Ala-Leu-Phe-tether scaffold to untangle some of those effects and pinpoint the most permeability-enhancing modification introduced by this peptoid substitution (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The 2S linker was chosen for those additional compounds because this series shows the most statistically significant differences among themselves in PAMPA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Four additional compounds to better elucidate the reason why a peptoid in this position has a beneficial effect on passive permeability. All leucine derivatives are in position 2. Numbers in parentheses indicate a significant difference (<i>p</i> < 0.05; one-way analysis of variance (ANOVA)/Tukey).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Two compounds appear more permeable than Nleu-2S: NMeLeu-2S and (<span class="smallcaps smallerCapital">d</span>)NMeLeu-2S, suggesting that substituting the amide proton is beneficial. Indeed, both (<span class="smallcaps smallerCapital">d</span>)Leu-2S and αMe-Leu-2S, which still have this amide proton, display a lower permeability than Nleu-2S. From the point of view of identifying why Nleu was found to be so successful, we can also rule out the orientation of the side chain: the three most permeable compounds are (<span class="smallcaps smallerCapital">d</span>)NMeLeu-2S, NMeLeu-2S, and Nleu-2S and thus all orientation are represented. Two other factors could explain why (<span class="smallcaps smallerCapital">d</span>)NMeLeu-2S and NMeLeu-2S show higher permeability than Nleu-2S (other than the slight increase in lipophilicity associated with an additional methyl group, an effect shared with the poorly permeable αMe-Leu-2S): they possess lower backbone flexibility, losing the glycine-like effect of peptoids, and they would be expected to have a smaller cis population as the methyl is less bulky than the isobutyl chain.<a onclick="showRef(event, 'ref25 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref25 ref43 ref44">(25,43,44)</a> However, these effects do not seem to be as important as the loss of the H-bond donor (compare αMe-Leu-2S with Nleu-2S), and do not hold comparison with ⌀-2S, which also has reduced backbone flexibility and lower expected cis population. In summary, the strongest factor appears to be the loss of the H-bond donor. The increased permeability observed with the Leu-to-Nleu substitution therefore appears to be caused by a change in the intramolecular H-bonding pattern. While it is likely that this pattern also explains why the peptoid substitution was successful at this position and not for Nala and Nphe, we did not investigate this sequence-dependent point further.</div><div class="NLM_p">As for the influence of the methyl group, no obvious trend could be observed. Yet, there are some significant differences between pairs of epimers. Also, the sole exception to the increase in permeability with Nleu was found in compound Nleu-5S, while its epimer Nleu-5R is the most permeable of the series. We decided to investigate this central observation by analyzing the associated conformations in greater detail using a combination of structural data (NMR measurements) and computer-simulated trajectories (molecular dynamics simulations).</div><div class="NLM_p">A “permeability cliff” such as that observed between Nleu-5R and Nleu-5S (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) is also observed, to a lesser extent, between some other pairs of epimers (e.g., Nala-4R vs 4S with −log(<i>P</i><sub>e</sub>) = 7.63 and 6.63, Nphe-2R vs 2S with 7.16 and 6.31, and Nphe-3R vs 3S with 7.49 and 6.49). To obtain a better understanding of the underlying conformational changes, extensive MD simulations of the Nleu-5R and Nleu-5S macrocycles (the most distant epimers in terms of permeability) were performed in both polar and apolar environments (i.e., water and chloroform). The starting conformations used for simulations showed similar distributions in terms of hydrogen bonds (H-bonds) and backbone torsional angles (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Tables S7 and S8</a> in the SI). For each molecule, approximately 50% of the starting structures had a <i>trans</i>-peptoid bond and 50% had a <i>cis</i>-peptoid bond. As a control, we used a second pair of peptides (Nleu-2S and Nleu-2R) with the same structural change but similar PAMPA permeability (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S16</a> in the SI).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selected cyclic peptides studied with experimental NMR analysis and molecular dynamics (MD) simulations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The cumulative 25 μs simulation data for each peptide and solvent were clustered separately based on the backbone dihedrals and the polar atom distances. The resulting clusters could be structurally classified depending on the conformation of the peptoid bond (i.e., cis or trans; see <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Tables S9 and S10</a> in the SI). The cis–trans isomerization represents a very slow process in the simulations, which occurred only rarely (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S11</a> in the SI). Due to the low number of transitions, the process could not be modeled robustly. Therefore, the clusters with the <i>cis</i>- and <i>trans</i>-peptoid bond are analyzed separately in the following.</div><div class="NLM_p">The NMR experiments in chloroform-<i>d</i> showed that the four compounds adopt at least two different conformations in solution. The major conformer was identified with all amides in trans conformation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S1</a> in the SI). It was not possible to assign the minor conformers due to signal overlap and low intensity. In the case of Nleu-5R and Nleu-5S, a third conformer could be identified based on exchange spectroscopy (EXSY) cross-peaks in the nuclear Overhauser enhancement spectroscopy (NOESY) spectrum, which is barely detectable in the <sup>1</sup>H spectrum. The corresponding conformer ratios are listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. The results from the MD simulations are compared to the NMR data of the major conformer (i.e., <sup>3</sup><i>J</i><sub>HN–Hα</sub> coupling constants and nuclear Overhauser effect (NOE)-derived distances, given in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Tables S2–S6</a> in the SI) to validate the simulation results.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Ratios of Conformer Population Observed in NMR Spectra (CDCl<sub>3</sub>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">ratio</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-2R</td><td class="colsep0 rowsep0" align="left">100:8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-2S</td><td class="colsep0 rowsep0" align="left">100:3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-5R</td><td class="colsep0 rowsep0" align="left">100:4:0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-5S</td><td class="colsep0 rowsep0" align="left">100:16:1</td></tr></tbody></table></div></div><div class="NLM_p">The clusters with all amides in trans conformation are in good agreement with the <sup>3</sup><i>J</i><sub>HN–Hα</sub> coupling constants (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), whereas the clusters containing the <i>cis</i>-peptoid bond deviate significantly from the experimental values. For Nleu-2R, the <sup>3</sup><i>J</i><sub>HN–Hα</sub> coupling analysis is missing as we could not determine the <sup>3</sup><i>J</i><sub>HN–Hα</sub> couplings reliably due to line broadening in the spectrum. The NOE upper distance bounds are also generally reproduced in these clusters (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Figures S5–S9</a> in the SI). Based on these findings, we focus the analysis in the following on those clusters, which have a reasonable agreement with the NMR data (i.e., clusters 1 and 4 for Nleu-5R, clusters 1, 5, and 6 for Nleu-5S, cluster 1 for Nleu-2R, and clusters 1 and 2 for Nleu-2S).</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Root-mean-square deviation (RMSD, in hertz) between <sup>3</sup><i>J</i><sub>HN–Hα</sub> coupling constants in chloroform from NMR measurements and from MD simulations. Clusters with the peptoid bond in trans conformation are shown in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A necessary condition for good membrane permeability is the adoption of conformations that shield polar groups optimally from the apolar environment.<a onclick="showRef(event, 'ref45 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47">(45−47)</a> Therefore, we first analyzed the hydrogen-bonding patterns in the clusters in chloroform. For the peptides in this study, a maximum number of two H-bonds can be formed in a conformation due to ring strain. As can be seen in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>, the percentage of sampled conformations with two H-bonds differs significantly between Nleu-5R (30%) and Nleu-5S (7%). At the same time, the percentage of conformations without a H-bond is increased for Nleu-5S (25%) compared to Nleu-5R (8%). For the other pair, Nleu-2R and Nleu-2S, the percentages are more similar and in between those of Nleu-5R and Nleu-5S.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Percentage of Sampled Conformations with Zero, One, or Two Hydrogen Bonds in Chloroform<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">number of hydrogen bonds</th><th class="colsep0 rowsep0" align="center" char=".">0</th><th class="colsep0 rowsep0" align="center" char=".">1</th><th class="colsep0 rowsep0" align="center" char=".">2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-5R (%)</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">63</td><td class="colsep0 rowsep0" align="char" char=".">30</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-5S (%)</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-2R (%)</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-2S (%)</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="char" char=".">13</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Analysis was restricted to the clusters with the <i>trans</i>-peptoid bond.</p></div></div></div><div class="NLM_p">For a given molecule in an apolar environment, having access to conformations in which polar groups are shielded—such as by H-bonding—should be energetically favorable. To assess this effect, we extracted the potential energy of the peptides (i.e., intramolecular and peptide-solvent contributions) from the trajectories. The normality of each potential-energy distribution was confirmed by the Shapiro–Wilk test<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S12</a> in the SI). The Fisher <i>t</i>-test<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> was employed to determine if the means of the distributions differ statistically significantly (<i>p</i> < 0.05). This was found to be the case for each pair of distributions (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S13</a> in the SI). On average, the potential energy of Nleu-5R is 9 kJ/mol lower (i.e., more favorable) in chloroform compared to Nleu-5S, whereas the difference in the average potential energy between Nleu-2R and Nleu-2S is 6 kJ/mol. In many studies in the literature, it was found that the three-dimensional (3D) polar surface area (3D-PSA) is a good measure for the degree of polar shielding in conformations.<a onclick="showRef(event, 'ref31 ref45 ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref31 ref45 ref50 ref51">(31,45,50,51)</a> However, for the present set of four peptides, no correlation was observed between the 3D-PSA and the potential energy (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Figure S10</a> in the SI). The ring strain in the relatively small backbone cycle of the peptides affects the geometry of the intramolecular H-bonds, which is likely not reflected appropriately in the 3D-PSA calculation. In summary, the ranking Nleu-5R < Nleu-2S < Nleu-2R < Nleu-5S, which was found in terms of both hydrogen-bonding patterns and potential energies, matches well with the experimental permeability data.</div><div class="NLM_p">The findings described above indicate that the change in stereochemistry of the methyl group in position 5 between Nleu-5R and Nleu-5S leads to different conformational behavior. A detailed analysis of the H-bonds showed that only Nleu-5S forms a H-bond between Ala-O and the tether-NH with an occurrence of 24% in chloroform (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This H-bond across the ring of Nleu-5S prevents the formation of other H-bonds (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B). Such a conformation with a single H-bond is likely less favorable (compared to one with more H-bonds) in chloroform because less polar groups are shielded. In the dominant conformation of Nleu-5R, on the other hand, two H-bonds can be formed across the ring (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A).</div><figure id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Snapshots of Nleu-5R (A) and Nleu-5S (B) from MD simulations in chloroform. Hydrogen bonds are shown with their percentage of the absolute occurrence in chloroform in the <i>trans</i>-peptoid clusters. Pictures were generated with PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Hydrogen Bond Occurrence in Percentage for the Sampled Conformations in Chloroform<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">H-bond</th><th class="colsep0 rowsep0" align="center" char=".">Nleu-2R (%)</th><th class="colsep0 rowsep0" align="center" char=".">Nleu-2S (%)</th><th class="colsep0 rowsep0" align="center" char=".">Nleu-5R (%)</th><th class="colsep0 rowsep0" align="center" char=".">Nleu-5S (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Nleu-O tether-NH</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">28</td><td class="colsep0 rowsep0" align="char" char=".">33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ala-O tether-NH</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char=".">24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Phe-O Ala-NH</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char="."><1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ala-O Phe-NH</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">17</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Analysis was restricted to the clusters with the <i>trans</i>-peptoid bond.</p></div></div></div><div class="NLM_p">Next, we analyzed the torsional-angle distributions in the backbone ring of the peptides. The change in stereochemistry of the methyl group at position 5 leads to a shift in the torsional-angle distributions of the tether units for Nleu-5S compared to Nleu-5R (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A). This shift results in a bent conformation of the ring (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B), which allows only one H-bond to form between Ala-O and tether-NH (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). There is also a shift in the backbone torsional-angle distributions between Nleu-2R and Nleu-2S, however, to a much smaller extent (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Figure S11</a> in the SI).</div><figure id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0010.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Torsional-angle distributions of the tether in Nleu-5R (blue) and Nleu-5S (orange) in chloroform. The analysis was restricted to the clusters with the <i>trans</i>-peptoid bond. (B) Torsional angles of the tether (shown in cyan and orange) corresponding to the peaks of the distributions. Pictures were generated with PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The change in the stereocenter also affects the χ<sub>1</sub>-angle of the phenylalanine residue as the tether conformation hinders the rotation around this torsion due to a steric clash with the carbonyl group that is facing out of the backbone ring (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0011.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Torsional-angle distributions of the χ<sub>1</sub> torsional angle of the phenylalanine residue in Nleu-5R (blue) and Nleu-5S (orange) in chloroform. Analysis was restricted to the clusters with the <i>trans</i>-peptoid bond. (B) χ<sub>1</sub> torsional angle of the phenylalanine residue (shown in purple) corresponding to the peaks of the distributions. The backbone carbonyl interferes with the rotation around this torsion is highlighted with a red circle. Pictures were generated with PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The results for the simulations in water are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">SI</a> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Tables S10–S15</a>). The analysis of the hydrogen-bonding patterns in water showed that Nleu-5R has a higher percentage (about 10%) of conformers with no H-bonds compared to Nleu-2R, Nleu-2S, and Nleu-5S (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S14</a>). In general, however, no major differences between the peptides could be observed in water.</div><div class="NLM_p last">The findings, taken together, suggest that the permeability cliff observed between Nleu-5R and Nleu-5S is related to their propensity for conformations with a maximized number of intramolecular H-bonds in the apolar environment. Their ability to adopt such conformations is in turn affected by the stereochemistry of the methyl group at position 5 in the tether as it determines the preferred torsional angles of the tether.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">A total of 42 macrocycles were synthesized and their permeability assessed in the PAMPA and Caco-2 assays. The combination of these data, NMR measurements, and molecular dynamics simulations allows us to draw some conclusions that are hopefully applicable to other systems. The systematic higher permeability of macrocycles bearing an Nleu peptoid is striking and well above statistical significance. Our experiments suggest this effect is due to the removal of this specific H-bond donor, thus working similarly to the more widely used N-methylation strategy. This systematic effect shows that “masking” H-bond donors should be considered early in the design of cyclic peptides. Possibly, it is a matter of finding the right one(s), i.e., those that allow for the most favorable H-bonding patterns in the rest of the macrocycle.</div><div class="NLM_p">The methyl position on the tether had little effect in most cases, with a few notable exceptions. Nala-2S has the lowest passive permeability, while its epimer is average. Conversely, Nleu-5R is the most permeable compound from our initial library, while its counterpart Nleu-5S is the exception among the Nleu compounds for its low permeability. A detailed analysis of torsion angles points once more at intramolecular H-bonds. Nleu-5R and Nleu-5S have different intramolecular H-bonding patterns. It seems likely that the 2 and 5 positions have the highest potential to introduce significant conformational changes due to their proximity to H-bond partners (the tether’s carbonyl and nitrogen, respectively). These positions might also have more impact due to the flexible nature of the tether we used, as they are close to the sp<sup>2</sup>-like amides. It is also noteworthy that a simple inversion of stereochemistry was shown to exert long-distance influence, modifying the phenylalanine’s rotation.</div><div class="NLM_p last">Altogether, this study sheds light on the relationship between structure and permeability in this class of compounds. The two seemingly very different substitutions we explored were both found to affect permeability through a change in the intramolecular H-bonding pattern.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70054" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70054" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> PAMPA<a onclick="showRef(event, 'ref53 ref54'); return false;" href="javascript:void(0);" class="ref ref53 ref54">(53,54)</a></h3><div class="NLM_p last">Passive permeability assays were run in triplicate on 96-well hydrophobic poly(vinylidene difluoride) (PVDF) plates (Millipore, 0.45 μM, 200 μL) and 300 μL receptor plates (Millipore). Assays were carried out using a 10 μL membrane (2% lecithin in dodecane) with a 100 μM solution in a phosphate buffer (pH = 6.4) and shaken for 17 h (25 °C, 50 rpm).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Caco-2 Assay<a onclick="showRef(event, 'ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56">(54−56)</a></h3><div class="NLM_p last">Assays were run in triplicate on polycarbonate Transwell plates (Corning) with hydrophilic filters (0.45 μM, Millipore). Caco-2 cells were passaged >72 times and grown on the filter for 22 days, at which point the transepithelial electrical resistance (TEER) was over 300 Ω cm<sup>2</sup>. Compounds were tested in an initial 10 μM solution using an Hank’s balanced salt solution (HBSS) buffer (<i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES)/NaOH, pH = 7.4) and a sample was taken after 60 and 120 min at 37 °C. Propranolol and atenolol were used as control compounds for high and low permeabilities, respectively. The assays were performed in both apical-to-basolateral and basolateral-to-apical directions to better assess the effect of efflux.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Simulation</h3><div class="NLM_p">In the computational studies, two pairs of structurally similar cyclic peptides were selected, i.e., Nleu-5R/Nleu-5S and Nleu-2R/Nleu-2S. The first pair presents a “permeability cliff”, i.e., the two peptides show a large difference in the passive permeability in the PAMPA assay (Nleu-5R: 5.39; Nleu-5S: 7.21), despite a high structural similarity. In contrast, the second pair is similar in both structure and permeability (Nleu-2R: 6.14; Nleu-2S: 5.80). For each of these four peptides, 250 starting coordinates were generated using the macrocycle variant of the OMEGA conformer generator from OpenEye.<a onclick="showRef(event, 'ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref59">(57−59)</a> Conformers were energy-minimized for maximum 2000 steps with the steepest descent<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> approach using the GROMOS software package<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> with the GROMOS 54A7 force field.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Each minimized starting conformation was solvated in a cubic box of simple-point-charge (SPC) water<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> (on average, 4172 solvent molecules) or chloroform<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> (on average, 980 solvent molecules). For each system, a molecular dynamics (MD) simulation of 101 ns length was performed under isothermal–isobaric (NPT) conditions with the leap-frog integration algorithm<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and a time step of 2 fs. The first 1 ns was discarded as equilibration. Bond lengths were constrained with SHAKE<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and a tolerance of 10<sup>–4</sup> nm. Nonbonded interactions were calculated using a twin-range scheme with a short-range cutoff of 0.8 nm and a long-range cutoff of 1.4 nm. The electrostatic nonbonded contributions beyond the long-range cutoff were calculated with the reaction-field<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> approach, setting the dielectric permittivity to 61<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> for water, and to 4.8<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> for chloroform. The temperature was kept constant at 300 K using the weak coupling scheme<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> and a coupling time of 0.1 ps<sup>–1</sup>. The pressure was kept at 1.031 bar (1 atm) with the same type of algorithm, a coupling time of 0.5 ps<sup>–1</sup>, and an isothermal compressibility of 0.001654 bar<sup>–1</sup> for chloroform and 0.0004575 bar<sup>–1</sup> for water. Translational motion of the center of mass of the simulation box was removed every 2 ps. Energies and coordinates were written every 5 ps.</div><div class="NLM_p last">Trajectory analysis was performed with PyEmma<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> and MDTraj.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> The selection of features for the clustering consisted of the distances between all pairs of polar atoms and the backbone torsional angles, which resulted in 57 features. This selection was reduced to three to five dimensions (depending on the peptide) with TICA<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> using a cumulative variance of 0.9 as criterium and a TICA correlation lag time of 50 ps. Based on these TICs, the frames were clustered with a common nearest neighbor (CNN) algorithm<a onclick="showRef(event, 'ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref73 ref74">(73,74)</a> using a cutoff of 0.2 and a similarity of 20. Comparison of selected clusters with NMR experiments was performed with the GROMOS++ package of programs.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> The coefficients for the Karplus curve were taken from Vögeli et al.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> Analysis of hydrogen bonds and torsional angles was performed with MDTraj. The 3D polar surface area (3D-PSA) was calculated with our implementation<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> of the workflow in ref <a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> using PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> Statistical analysis of all results was carried out using the Python packages pandas, NumPy and SciPy.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> NMR Measurements</h3><div class="NLM_p">The selected peptides Nleu-5R, Nleu-5S, Nleu-2R, and Nleu-2S were characterized by NMR in chloroform-<i>d</i> (Cambridge Isotope Laboratories). The following spectra were recorded if not stated otherwise: <sup>1</sup>H NMR, total correlation spectroscopy (TOCSY), double-quantum filtered correlation spectroscopy (COSY), multiplicity edited <sup>13</sup>C heteronuclear single quantum coherence (HSQC), <sup>13</sup>C heteronuclear multiple bond correlation (HMBC), and NOESY. All spectra were measured at 25 °C on a Bruker Avance III HD 600 MHz spectrometer equipped with a N<sub>2</sub>-cooled Prodigy triple resonance probe. <sup>13</sup>C HSQC and TOCSY spectra were recorded with sensitivity enhancement. TOCSY was run with an 80 ms DIPSI2 isotropic mixing time. The mixing time for the NOESY experiments was set to 300 ms if not stated otherwise. For compound Nleu-5R, an EASY-ROESY<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> spectrum with 100 ms mixing time was recorded instead of a NOESY. For all spectra, the time domain in both dimensions was extended to twice its size by zero filling, apodized with a cos<sup>2</sup> function, and the baseline of the resulting spectra was corrected with a polynomial of fifth order or using the Whittacker smoother algorithm.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> Processing was done with Bruker TopSpin version 4.0 (Bruker Biospin AG) and MestReNova 12.0 (Mestrelab Research). Resonance assignment and volume integration of the ROESY cross-peaks were performed with SPARKY 3.115.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> The assignments are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S1</a> in the SI.</div><div class="NLM_p"><sup>3</sup><i>J</i><sub>HN–Hα</sub> coupling constants for compounds Nleu-5R, Nleu-5S, Nleu-2R, and Nleu-2S were extracted directly from the <sup>1</sup>H spectrum with MestReNova and are summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Table S2</a> in the SI.</div><div class="NLM_p">Volumes were extracted from NOESY and ROESY spectra by integration of the cross-peaks with a Gaussian function (<a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>).<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_m001.gif" alt="" id="_i18" /></img><span class="labelSpan">(1)</span></span></div><div class="NLM_p last"><i>V</i><sub>1,2</sub> is the volume of the cross-peak between proton 1 and 2, <i>a</i> and <i>b</i> are fitting parameters, and <i>r</i><sub>1,2</sub> is the corresponding interatomic distance. A two-point calibration was done with the averaged interatomic distance (e.g., on both sides of the diagonal) between the diastereotopic protons NLeu Hα<sub>1</sub>–Hα<sub>2</sub> (1.8 Å) and the distance between Hα and Hβ* in the alanine residue (2.65 Å).<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> In the second case, the volume was previously divided by 3 to account for the three protons in the methyl group. Cross-peaks integrated on both sides of the diagonal were averaged, and error bonds of ±20% were added to the calculated distance. Since the GROMOS++ programs can do multiplicity correction and averaging over indistinguishable protons automatically, the reported data do not account for that. The volumes and the corresponding distances can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">Tables S3–S6</a> in the SI.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Materials and Equipment Used in Synthesis</h3><div class="NLM_p">All solvents and chemicals were used as purchased without further purification. The progress of all reactions was monitored on Silicycle silica gel plates using either ethyl acetate/<i>n</i>-hexane or dichloromethane/methanol. Column chromatography was performed with SILIFLASH P60 silica (40–63 μM, 230–400 mesh, 60 Å).</div><div class="NLM_p">NMR data were recorded on a Bruker Ascent 400. Chemical shifts are given in parts per million (ppm) (δ relative to the residual solvent peaks for both <sup>1</sup>H and <sup>13</sup>C). Chloroform (7.27 ppm), DMSO (2.50 ppm), and MeOH (3.31 ppm) were used as the internal standards for protons and for carbons (77.0, 39.5, and 49.2 ppm, respectively).</div><div class="NLM_p">High-resolution mass spectrometry (HRMS) was performed with a maXis (electrospray ionization quadrupole time-of-flight (ESI-QTOF)).</div><div class="NLM_p">Final products were purified on a Waters preparative HPLC system (Waters Sample Manager 2767, Binary Gradient Module 2545, SQ Detector 2) with an XSelect Peptide CSH C18 OBD Prep Column (100 × 19 mm<sup>2</sup>, 5 μM spherical particle size) with a flow of 20 mL/min on a 15 min gradient of varying proportions of acetonitrile in water containing 0.1% formic acid (in both solvents).</div><div class="NLM_p last">Purity analysis of the final macrocycles was performed with UPLC/MS Acquity H-Class using a BEH C18 column (50 × 2.1 mm<sup>2</sup>, 1.7 μM spherical particle size) with a flow of 0.8 mL/min on a 2.5 min gradient from 5 to 95% acetonitrile in water containing 0.1% formic acid (in both solvents). All chromatography equipment was purchased at Waters (Canada). The purity was determined from the UV spectrum, by the ratio of the area under the curve (AUC) of the expected compound over the sum of the AUC of all peaks that did not also appear on blanks. All final macrocycles were obtained with a purity of >95%.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Loading and Capping Procedure <b>23</b></h3><div class="NLM_p last">To Wang resin (200 mg, nominal loading: 1.5 mmol/g) was added anhydrous THF (2 mL), and the mixture was left for 30 min. Meanwhile, a solution of Fmoc-Phe-OH (3 equiv) was prepared in anh. THF (2 mL). To this solution was added PPh<sub>3</sub> (3 equiv) and diisopropyl azodicarboxylate (DIAD) (3 equiv). The THF from the resin solution was removed by filtration, and the amino acid solution was added. The mixture was left on an orbital shaker overnight. The resin was then filtered and washed in the following sequence: 3× DMF, 3× DMC, 3× <i>i</i>PrOH, 3× DCM, 3× <i>i</i>PrOH, 3× DCM. A freshly prepared solution of DCM/Ac<sub>2</sub>O/DIPEA (15:2:1, 2 mL) was then added, and the mixture was left on an orbital shaker for 30 min, after which it was washed using the same sequence as above.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Loading Procedure for Nphe <b>30</b></h3><div class="NLM_p last">To a 2-chlorotrityl chloride resin (200 mg, nominal loading: 1.2 mmol/g) was added anhydrous DCM (2 mL), and the mixture was left for 30 min. Meanwhile, a solution of bromoacetic acid (4 equiv) was prepared in anh. DCM (2 mL). To this solution was added DIPEA (8 equiv) The DCM from the resin solution was removed by filtration, and the bromoacetic acid solution was added. The mixture was left on an orbital shaker for 1 h. The resin was then filtered and washed six times with DMF. A solution of benzylamine (20 equiv) in DMF (2 mL) was added, and the mixture was left on an orbital shaker for 1 h. The resin was then filtered and washed six times with DMF.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Deprotection Procedure</h3><div class="NLM_p last">To the resin (200 mg) was added a solution of 20% piperidine in DMF (2 mL), and the mixture was left on an orbital shaker for 20 min. The resin was then washed twice with DMF, and a solution of 20% piperidine in DMF (2 mL) was added once more and then left on the orbital shaker for another 20 min. The resin was then washed six times with DMF.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Coupling Procedure for Amino Acids and Nala <b>24</b>, <b>26</b></h3><div class="NLM_p last">A solution of amino acid (3 equiv) and HATU (2.5 equiv) in DMF (2 mL) was prepared, then DIPEA (5 equiv) was added. The resulting yellow solution was added to the deprotected resin and left on the orbital shaker for 3 h. The resin was then filtered and washed in the following sequence: 3× DMF, 3× DCM, 3× <i>i</i>PrOH, 3× DCM, 3× <i>i</i>PrOH, 3× DCM.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Coupling Procedure for Nleu <b>28</b></h3><div class="NLM_p last">A solution of bromoacetic acid (6.8 equiv) and DIC (8 equiv) in DMF (2 mL) was added to the deprotected resin and left on the orbital shaker for 1 h. The resin was then filtered and washed six times with DMF. A solution of isobutylamine (20 equiv) in DMF (2 mL) was then added to the resin and left on the orbital shaker for 1 h. The resin was then filtered and washed six times with DMF.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cleavage and Deprotection Procedure</h3><div class="NLM_p last">A 50% solution of trifluoroacetic acid (TFA) in DCM was added to the Wang resin (30% HFIP in DCM for 2-Cl chlorotrityl resin), and the mixture was left on the orbital shaker for 1 h. The solution was then filtered and concentrated under reduced pressure, adding DCM when dry (3×) to remove most of the TFA (or HFIP).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Macrocyclization Procedure</h3><div class="NLM_p last">The linear peptide was dissolved in DMF (0.025 mol/L), DEPBT was added (1.1 equiv), followed by DIPEA (3 equiv). The resulting yellow solution was left to stir until completion, typically around 72 h. It was then filtered on a carbonate scavenging cartridge and concentrated under reduced pressure. The crude product was purified on a preparative HPLC-MS.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Synthesis of Tether <b>7</b><i><b>S</b></i></h3><div class="NLM_p last">The <b>7<i>R</i></b> linker was synthesized using the same procedures.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 5-Azidovaleric Acid <b>2</b></h3><div class="NLM_p last">5-Bromovaleric acid <b>1</b> (5.00 g, 27.6 mmol, 1 equiv) was dissolved in DMSO (275 mL, 0.1 M), sodium azide (7.18 g, 110 mmol, 4 equiv) was added, and the mixture was stirred overnight. Upon completion, 1 M HCl was added (300 mL) and the solution was extracted with ethyl acetate (3 × 300 mL). The organic phases were combined and washed with a 1:1 mixture of brine and 1 M HCl (6 × 300 mL), dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain an orange oil (4.05 g, quantitative) that was used directly for the next step. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 11.42 (1H, s), 3.31 (2H, t, <i>J</i> = 6.6 Hz), 2.40 (2H, t, <i>J</i> = 7.2 Hz), 1.77–1.61 (4H, m); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 178.0, 51.0, 33.3, 28.2, 21.8; HRMS [M – H]<sup>−</sup> calcd for C<sub>5</sub>H<sub>9</sub>N<sub>3</sub>O<sub>2</sub>: 142.0611, found: 142.0621.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 1-[(4<i>S</i>)-4-Benzyl-2-oxo-1,3-oxazolidin-3-yl]-5-azido-1-pentanone <b>3<i>S</i></b></h3><div class="NLM_p last">5-Azidovaleric acid <b>2</b> (3.95 g, 27.6 mmol, 1 equiv) was dissolved in THF (92 mL, 0.3 M), and the solution was cooled to −78 °C. Triethylamine (4.62 mL, 33.1 mmol, 1.2 equiv) was added, followed by trimethylacetyl chloride (3.74 mL, 30.4 mmol, 1.1 equiv), and the solution was left to stir and warm up to 0 °C for 1 h. In a separate flask, (<i>S</i>)-4-benzyl-2-oxazolidinone (5.87 g, 33.1 mmol, 1.2 equiv) was dissolved in THF (39 mL, 0.7 M) and cooled to −78 °C. To this solution was added a solution of <i>n</i>-butyllithium in hexanes (33.1 mmol, 1.2 equiv) dropwise. The solution containing the azide was cooled back to −78 °C, and the oxazolidinone solution was added to it by a canula. It was left to stir and warm up to 0 °C for 2 h. When completed, the solution was poured into sat. NH<sub>4</sub>Cl (300 mL) and extracted with EtOAc (3 × 300 mL). The organic fractions were merged, washed with brine (1 × 300 mL), dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain an orange oil. This crude product was purified by flash chromatography (7:3 hexanes/ethyl acetate), yielding a colorless oil (6.06 g, 73%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.40–7.25 (3H, m), 7.24–7.17 (2H, m), 4.74–4.63 (1H, m), 4.27–4.16 (3H, m), 3.39–3.25 (3H, m), 3.10–2.87 (2H, m); 1.87–1.64 (4H, m) <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 172.3, 153.4, 135.2, 129.4, 128.9, 127.3, 66.2, 55.1, 51.1, 37.9, 34.9, 28.2, 21.3; HRMS [M + Na]<sup>+</sup> calcd for C<sub>15</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>: 325.1271, found: 325.1278.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (2<i>S</i>)-1-[(4<i>S</i>)-4-Benzyl-2-oxo-1,3-oxazolidin-3-yl]-5-azido-2-methyl-1-pentanone <b>4<i>S</i></b></h3><div class="NLM_p last">1-[(4<i>S</i>)-4-Benzyl-2-oxo-1,3-oxazolidin-3-yl]-5-azido-1-pentanone <b>3</b><i><b>S</b></i> (6.06 g, 20.0 mmol, 1 equiv) was dissolved in THF (10 mL, 2 M) and cooled to −78 °C. In a separate vessel, a 0.5 M solution of potassium bis(trimethylsilyl)amide in toluene (56 mL, 28.1 mmol, 1.4 equiv) was dissolved in cold (−78 °C) THF (134 mL, 0.15 M). The solution containing the oxazolidinone was then added by a canula to the KHMDS solution and left to stir for 30 min. Methyl iodide (3.74 mL, 60 mmol, 3 equiv) was added, left to stir, and warmed up to 0 °C for 2 h. The reaction mixture was then poured into a saturated solution of NH<sub>4</sub>Cl, and the product was extracted with ethyl acetate (3 × 500 mL). The organic fractions were merged, washed with brine (1 × 500 mL), dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a crude yellow oil (6.74 g, quant., de = 69%), which was used directly for the next step. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 7.40–7.25 (3H, m), 7.25–7.15 (2H, m), 4.75–4.65 (1H, m), 4.28–4.15 (2H, m), 3.82–3.67 (1H, m), 3.39–3.20 (3H, m), 2.78 (1H, dd, <i>J</i> = 13.1, 9.6 Hz), 1.92–1.77 (1H, m), 1.73–1.43 (3H, m), 1.26 (3H, d, <i>J</i> = 6.8 Hz) <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 176.6, 153.0, 135.1, 129.4, 128.9, 127.4, 66.1, 55.2, 51.3, 37.9, 37.4, 30.3, 26.6, 17.5; HRMS [M + Na]<sup>+</sup> calcd for C<sub>16</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>: 339.1428, found: 339.1430.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (2<i>S</i>)-5-Azido-2-methylvaleric Acid <b>5</b><i><b>S</b></i></h3><div class="NLM_p last">(2<i>S</i>)-1-[(4<i>S</i>)-4-Benzyl-2-oxo-1,3-oxazolidin-3-yl]-5-azido-2-methyl-1-pentanone <b>4</b><i><b>S</b></i> (1.51 g, 4.77 mmol, 1 equiv) was dissolved in THF/H<sub>2</sub>O (3:1, 100 mL, 0.05 M), and the solution was cooled to 0 °C. Hydrogen peroxide (2.70 mL, 23.8 mmol, 5 equiv) was added, followed by lithium hydroxide (228 mg, 9.52 mmol, 2 equiv) and the solution was left to stir until completion (approximately 20 min). A saturated solution of sodium sulfite was added (5 mL), and the solution was concentrated under reduced pressure to remove most of the THF. A saturated solution of sodium bicarbonate (100 mL) was added, and the product was washed with dichloromethane (3 × 100 mL). The aqueous phase was acidified to pH = 1 using 1 M HCl and extracted with ethyl acetate (3 × 100 mL). The ethyl acetate fractions were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a colorless oil (730 mg, 98%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 10.61 (1H, s), 3.30 (2H, t, <i>J</i> = 6.6 Hz), 2.57–2.44 (1H, m), 1.83–1.48 (4H, m), 1.22 (3H, d, <i>J</i> = 6.9 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 182.2, 51.2, 38.9, 30.5, 26.5, 16.9; HRMS [M – H]<sup>−</sup> calcd for C<sub>6</sub>H<sub>11</sub>N<sub>3</sub>O<sub>2</sub>: 156.0768, found: 156.0772.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (2<i>S</i>)-5-Amino-2-methylvaleric Acid <b>6</b><i><b>S</b></i></h3><div class="NLM_p last">(2<i>S</i>)-5-Azido-2-methylvaleric acid <b>5</b><i><b>S</b></i> (3.04 g, 19.3 mmol, 1 equiv) was dissolved in methanol (40 mL, 0.5 M) with 5% (v/v, 2 mL) acetic acid, palladium on carbon (10 wt %) was added (310 mg), and the solution was put under 300 psi of hydrogen and shaken for 16 h. The Pd/C was removed by filtration over a Celite pad, and the solution was concentrated under reduced pressure to obtain a colorless oil in quantitative yield. This crude product was used directly in the next step.</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (2<i>S</i>)-2-Methyl-5-(<i>tert</i>-butoxycarbonylamino)valeric Acid <b>7</b><i><b>S</b></i></h3><div class="NLM_p last">(2<i>S</i>)-5-Amino-2-methylvaleric acid <b>6</b><i><b>S</b></i> (2.54 g, 19.4 mmol, 1 equiv) was dissolved in a 2:1 mixture of dioxane and water (60 mL, 0.3 M) and sodium bicarbonate was added until pH reached 8. Di-<i>tert</i>-butyl dicarbonate (4.65 g, 21.3 mmol, 1.1 equiv) was added, and the solution was stirred for 16 h. Upon completion, the dioxane was removed under reduced pressure, 1 M HCl was added (100 mL), and the product was extracted with ethyl acetate (3 × 100 mL). The organic fractions were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a colorless oil (3.50 g, 78%). The product was isolated by flash chromatography (DCM/MeOH, 95:5) and coevaporated multiple times with toluene to yield a colorless oil (1.73 g, 39%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 5.75 (0.2H, br s), 4.60 (0.8H, br s), 3.13 (2H, br t), 2.55–2.43 (1H, m), 1.77–1.63 (1H, m), 1.59–1.49 (3H, m), 1.45 (9H, s), 1.20 (3H, d, <i>J</i> = 7.0 Hz); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 181.8, 156.0, 79.3, 40.3, 39.0, 30.6, 28.4, 27.7, 16.9; HRMS [M + Na]<sup>+</sup> calcd for C<sub>11</sub>H<sub>21</sub>NO<sub>4</sub>: 254.1363, found: 254.1368.</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Synthesis of Tether <b>13<i>R</i></b></h3><div class="NLM_p last">The <b>13<i>S</i></b> linker was synthesized using the same procedures.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (3<i>R</i>)-3,7-Dimethyl-6-octenoic Acid <b>9</b><i><b>R</b></i></h3><div class="NLM_p last">(3<i>R</i>)-3,7-Dimethyl-6-octen-1-ol <b>8</b><i><b>R</b></i> (5.00 g, 32.0 mmol, 1 equiv) was dissolved in DMF (100 mL, 0.3 M), and pyridinium dichromate (60.2 g, 160 mmol, 5 equiv) was added. The solution was stirred for 16 h. Upon completion, diethyl ether (300 mL) was added, and the mixture was washed with half-sat. NH<sub>4</sub>Cl (6 × 300 mL) and brine (1 × 300 mL). The organic fractions were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a colorless oil. The product was isolated by flash chromatography (hexanes/EtOAc, 17:3) to yield a colorless oil (3.93 g, 72%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 11.50 (1H, br s), 5.15–5.05 (1H, m), 2.38 (1H, dd, <i>J</i> = 15.4, 5.9 Hz), 2.16 (1H, dd, <i>J</i> = 15.4, 8.0 Hz), 2.07–1.92 (3H, m), 1.69 (3H, d, <i>J</i> = 1.0 Hz), 1.61 (3H, s), 1.45–1.34 (1H, m), 1.31–1.21 (1H, m), 0.99 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 179.4, 131.7, 124.1, 41.5, 36.7, 29.8, 25.7, 25.4, 19.6, 17.6; HRMS [M – H]<sup>−</sup> calcd for C<sub>10</sub>H<sub>18</sub>O<sub>2</sub>: 169.1223, found: 169.1233.</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Methyl (3<i>R</i>)-3,7-Dimethyl-6-octenoate <b>10</b><i><b>R</b></i></h3><div class="NLM_p last">(3<i>R</i>)-3,7-Dimethyl-6-octenoic acid <b>9</b><i><b>R</b></i> (2.51 g, 14.7 mmol, 1 equiv) was dissolved in diethyl ether (50 mL, 0.3 M). A solution of diazomethane (73.7 mmol, 5 equiv) in diethyl ether (200 mL, 0.37 M) was added, and the reaction was stirred for 2 h. Upon completion, drops of acetic acid were added until the solution became colorless. The solvent was concentrated by reduced pressure to yield a slightly yellow oil (2.60 g, 61%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 5.14–5.04 (1H, m), 3.67 (3H, s), 2.33 (1H, dd, <i>J</i> = 14.8, 5.9 Hz), 2.12 (1H, dd, <i>J</i> = 15.2, 8.1 Hz), 2.06–1.87 (3H, m), 1.69 (3H, d, <i>J</i> = 0.9 Hz), 1.61 (3H, s), 1.43–1.16 (3H, m), 0.95 (3H, 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 173.7, 131.5, 124.2, 51.3, 41.6, 36.7, 30.0, 25.7, 25.4, 19.6, 17.6; HRMS [M + Na]<sup>+</sup> calcd for C<sub>11</sub>H<sub>20</sub>O<sub>2</sub>: 207.1356, found: 207.1358.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> (4<i>R</i>)-5-Methoxycarbonyl-4-methylvaleric Acid <b>11</b><i><b>R</b></i></h3><div class="NLM_p last">Methyl (3<i>R</i>)-3,7-dimethyl-6-octenoate <b>10</b><i><b>R</b></i> (2.60 g, 14.1 mmol, 1 equiv) was dissolved in a 2:2:3 mixture of carbon tetrachloride (40 mL), acetonitrile (40 mL), and water (60 mL). Sodium periodate (12.4 g, 57.9 mmol, 4.1 equiv) was added, followed by ruthenium(III) chloride (65 mg, 0.31 mmol, 2.2 mol %). The resulting heterogeneous solution was stirred vigorously for 2 h or until completion. 1 M HCl was added, and the product was extracted with dichloromethane (3 × 200 mL). The organic fractions were combined and concentrated under reduced pressure, then resolubilized in ethyl acetate and extracted with dil. NaHCO<sub>3</sub> (3 × 200 mL). The aqueous phase was acidified to pH 1 using 1 M HCl and extracted with ethyl acetate (5 × 200 mL). The organic fractions from the last extraction were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a colorless oil (1.65 g, 67%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.67 (3H, s), 2.46–2.26 (3H, m), 2.23–2.12 (1H, m), 2.07–1.93 (1H, m), 1.77–1.64 (1H, m), 1.61–1.47 (1H, m), 0.96 (3H, d, <i>J</i> = 6.7 Hz); <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>): δ 179.0, 173.2, 51.5, 41.2, 31.5, 31.2, 29.8, 19.3; HRMS [M + Na]<sup>+</sup> calcd for C<sub>8</sub>H<sub>14</sub>O<sub>4</sub>: 197.0784, found: 197.0786.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Methyl (3<i>R</i>)-3-Methyl-5-(<i>tert</i>-butoxycarbonylamino)valerate <b>12</b><i><b>R</b></i></h3><div class="NLM_p last">(4<i>R</i>)-5-Methoxycarbonyl-4-methylvaleric acid <b>11</b><i><b>R</b></i> (2.30 g, 13.2 mmol, 1 equiv) was dissolved in <i>tert-</i>butanol (130 mL, 0.1 M), then diphenylphosphoryl azide (3.13 mL, 14.5 mmol, 1.1 equiv) was added, followed by triethylamine (2.02 mL, 14.5 mmol, 1.1 equiv), and the mixture was refluxed for 16 h. Upon completion, the solvent was concentrated under reduced pressure and dissolved in ethyl acetate (100 mL). It was washed with half-sat. NaHCO<sub>3</sub> (2 × 100 mL), water (2 × 100 mL), and brine (1 × 100 mL). The organic fractions were combined, filtered through a pad of Celite, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain an orange oil (2.36 g, 73%), which was used without further purification. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 4.66 (1H, br s), 3.61 (3H, s), 3.18–2.95 (2H, m), 2.27 (1H, dd, <i>J</i> = 15.0, 6.3 Hz), 2.11 (1H, dd, <i>J</i> = 15.1, 7.7 Hz), 2.03–1.90 (1H, m), 1.52–1.41 (1H, m), 1.38 (9H, s), 1.35–1.28 (1H, m), 0.91 (3H, d, <i>J</i> = 6.7 Hz); HRMS [M + Na]<sup>+</sup> calcd for C<sub>12</sub>H<sub>23</sub>NO<sub>4</sub>: 268.1519, found: 268.1518.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (3<i>R</i>)-3-Methyl-5-(<i>tert</i>-butoxycarbonylamino)valeric Acid <b>13</b><i><b>R</b></i></h3><div class="NLM_p last">Methyl (3<i>R</i>)-3-methyl-5-(<i>tert</i>-butoxycarbonylamino)valerate <b>12</b><i><b>R</b></i> (571 mg, 2.33 mmol, 1 equiv) was dissolved in methanol (12 mL, 0.2 M), 1 M NaOH was added (12 mL, 12 mmol, 5 equiv), and the reaction was left to stir for 3 h. Upon completion, 1 M HCl was added to pH 2 and the product was extracted with dichloromethane (3 × 50 mL). The organic fractions were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a colorless oil (541 mg, quant.). The product was isolated by flash chromatography (DCM/MeOH, 19:1) to afford a colorless oil (484 mg, 90%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 10.64 (1H, br s), 6.05 (0.3H, s), 4.67 (0.7H, s), 3.12 (2H, s), 2.43–2.26 (1H, m), 2.18 (1H, dd, <i>J</i> = 15.3, 7.5 Hz), 2.10–1.90 (1H, m), 1.65–1.48 (1H, m), 1.43 (9H, s), 0.98 (3H, 7.4 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 178.2, 156.1, 79.3, 41.3, 38.3, 36.5, 28.3, 27.5, 19.5; HRMS [M – H]<sup>−</sup> calcd for C<sub>11</sub>H<sub>21</sub>NO<sub>4</sub>: 230.1387, found: 230.1388.</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Synthesis of Tether <b>15</b><i><b>R</b></i></h3><div class="NLM_p last">The <b>15<i>S</i></b> linker was synthesized using the same procedures.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (2<i>R</i>)-2,6-Dimethyl-5-heptenylamino 2,2-Dimethylpropionate <b>14</b><i><b>R</b></i></h3><div class="NLM_p last">(3<i>R</i>)-3,7-Dimethyl-6-octenoic acid <b>9</b><i><b>R</b></i> (2.03 g, 11.9 mmol, 1 equiv) was dissolved in anhydrous THF (120 mL, 0.1 M), and the following reagents were added, in order: sodium azide (2.72 g, 3.5 equiv), zinc trifluoromethanesulfonate (143 mg, 0.394 mmol, 3.3 mol %), tetrabutylammonium bromide (577 mg, 1.79 mmol, 15 mol %), <i>tert-</i>butanol (571 μL, 5.97 mmol, 0.5 equiv), and di-<i>tert</i>-butyl dicarbonate (2.87 g, 13.13 mmol, 1.1 equiv) The mixture was left to stir for 16 h. Upon completion, a 10% w/v sodium nitrite solution was added (240 mL), followed by 120 mL of ethyl acetate and the mixture was left to stir for 10 min. The two phases were separated, and the remaining aqueous phase was extracted with further portions of ethyl acetate (3 × 120 mL). The organic phases were combined; washed with NH<sub>4</sub>Cl (2 × 250 mL), NaHCO<sub>3</sub> (2 × 250 mL), and brine (1 × 250 mL); dried using MgSO<sub>4</sub>; filtered; and concentrated under reduced pressure to obtain a yellow oil (2.76 g, 96%) The product was isolated by flash chromatography (hexanes/EtOAc, 9:1) to afford a colorless oil (1.30 g, 45%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 5.13–5.04 (1H, m), 4.55 (1H, s), 3.15–2.80 (2H, m), 2.09–1.90 (2H, m), 1.68 (3H, d, <i>J</i> = 0.9 Hz); 1.66–1.54 (1H, m), 1.60 (3H, s), 1.45 (9H, s), 1.42–1.31 (1H, m), 1.20–1.07 (1H, m), 0.90 (3H, d, <i>J</i> = 6.8 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 156.1, 131.5, 124.4, 79.0, 46.5, 34.3, 33.2, 28.4, 25.7, 25.3, 17.6, 17.4; HRMS [M + Na]<sup>+</sup> calcd for C<sub>14</sub>H<sub>27</sub>NO<sub>2</sub>: 264.1934, found: 264.1939.</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (4<i>R</i>)-4-Methyl-5-(<i>tert</i>-butoxycarbonylamino)valeric Acid <b>15</b><i><b>R</b></i></h3><div class="NLM_p last">(2<i>R</i>)-2,6-Dimethyl-5-heptenylamino 2,2-dimethylpropionate <b>14</b><i><b>R</b></i> (2.40 g, 9.96 mmol, 1 equiv) was dissolved in methanol (50 mL, 0.2 M) and chilled to −78 °C. Ozone was bubbled through the solution for 45 min followed by oxygen for 30 min. The flask was removed from the cooling bath, and 30% hydrogen peroxide was added (5.64 mL, 49.7 mmol, 5 equiv), followed by 1 M sodium hydroxide (20 mL, 19.9 mmol, 2 equiv), and it was left to stir for 16 h. Upon completion, the flask was chilled to 0 °C and a saturated solution of sodium sulfite (5 mL) was added dropwise to quench excess peroxides. The solution was acidified to pH 2 using 1 M HCl and extracted with dichloromethane (3 × 50 mL). The organic phases were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a slightly yellow oil (2.33 g, quant.), and the product was isolated by flash chromatography (hexanes/EtOAc/formic acid, 98:1:1–95:4:1) and then coevaporated five times with toluene to remove traces of formic acid to afford a slightly yellow oil (1.29 g, 56%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 12.50–8.10 (1H, br s), 5.95 (0.3H, s), 4.67 (0.7H, s), 3.15–2.92 (2H, m), 2.51–2.27 (2H, m), 1.82–1.58 (2H, m), 1.45 (9H, s), 0.92 (3H, d, <i>J</i> = 6.7 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): 178.8, 156.2, 79.3, 46.0, 33.2, 31.5, 28.8, 28.4, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>11</sub>H<sub>21</sub>NO<sub>4</sub>: 254.1363, found: 254.1368.</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Synthesis of Tether <b>22<i>R</i></b></h3><div class="NLM_p last">The <b>22<i>S</i></b> linker was synthesized using the same procedures.</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (1<i>S</i>)-3-Diazo-1-methyl-2-oxopropylamino 2,2-Dimethylpropionate <b>17</b><i><b>R</b></i></h3><div class="NLM_p last">Boc-<span class="smallcaps smallerCapital">l</span>-Ala-OH <b>16</b><i><b>R</b></i> (1.00 g, 5.29 mmol, 1 equiv) was dissolved in anhydrous THF (21 mL, 0.25 M), and the solution was cooled to −15 °C. Isobutylchloroformate (564 μL, 5.81 mmol, 1.1 equiv) was added, followed by triethylamine (420 μL, 5.81 mmol, 1.1 equiv) After 15 min of stirring, it was removed from the cooling bath and left to warm up to room temperature. The resulting white mixture was filtered on a pad of Celite to obtain a colorless solution. To this solution was slowly added a freshly prepared solution of diazomethane in ether (30 mL, 0.88 M, 5 equiv), and the resulting yellow solution was left to stir for 2.5 h. Upon completion, the solution was quenched with acetic acid until colorless. It was then washed twice with sat. NaHCO<sub>3</sub> (2 × 60 mL), sat. NH<sub>4</sub>Cl (2 × 60 mL), and brine (1 × 60 mL). The organic phase was dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a yellow oil (>100%). This crude product was purified by flash chromatography (4:1 hexanes/ethyl acetate) to obtain a yellow crystalline solid (718 mg, 64%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 5.44 (1H, s), 5.12 (1H, s), 4.23 (1H, s), 1.45 (9H, s), 1.33 (3H, d, <i>J</i> = 7.0 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 194.4, 155.1, 80.0, 53.4, 28.3, 18.5; HRMS [M + Na]<sup>+</sup> calcd for C<sub>9</sub>H<sub>15</sub>N<sub>3</sub>O<sub>3</sub>: 236.1006, found: 236.1010.</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Methyl (3<i>S</i>)-3-(<i>tert</i>-Butoxycarbonylamino)butyrate <b>18</b><i><b>R</b></i></h3><div class="NLM_p last">(1<i>S</i>)-3-Diazo-1-methyl-2-oxopropylamino 2,2-dimethylpropionate <b>17</b><i><b>R</b></i> (718 mg, 3.37 mmol, 1.0 equiv) was dissolved in anhydrous methanol (34 mL, 0.1 M). To this was added dropwise a solution of silver benzoate (78 mg, 0.34 mmol, 0.1 equiv) in triethylamine (3.4 mL, 0.1 M with respect to silver benzoate). After about 5 min, the solution turned black and was left to stir overnight. Upon completion, the solution was filtered on Celite and concentrated under reduced pressure to yield an orange oil, which was solubilized in ethyl acetate (30 mL); washed with water (1 × 30 mL), 0.1 M HCl (1 × 30 mL), and brine (1 × 30 mL); dried using MgSO<sub>4</sub>; filtered; and concentrated under reduced pressure to obtain a black oil. This crude product was purified by flash chromatography (3:1 hexanes/ethyl acetate) to obtain a yellow solid (628 mg, 86%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 4.91 (1H, s), 4.03 (1H, m), 3.68 (3H, s), 2.50 (2H, m), 1.43 (9H, s), 1.21 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 172.0, 155.0, 79.3, 51.6, 43.4, 40.6, 28.4, 20.4; HRMS [M + Na]<sup>+</sup> calcd for C<sub>10</sub>H<sub>19</sub>NO<sub>4</sub>: 240.1206, found: 240.1212.</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> (1<i>S</i>)-2-Formyl-1-methylethylamino 2,2-Dimethylpropionate <b>19</b><i><b>R</b></i></h3><div class="NLM_p last">Methyl (3<i>S</i>)-3-(<i>tert</i>-butoxycarbonylamino)butyrate <b>18</b><i><b>R</b></i> (1.53 g, 7.05 mmol, 1 equiv) was dissolved in anhydrous dichloromethane (14 mL, 0.5 M) and chilled to −78 °C. To this was added a 1 M diisobutylaluminium hydride solution (12.0 mL, 12.0 mmol, 1.7 equiv) dropwise. Upon completion, the reaction mixture was transferred into a saturated solution of potassium sodium tartrate (50 mL) and the mixture was left to stir for 1 h. The two phases were separated, and the aqueous phase was extracted with dichloromethane (3 × 50 mL). The organic phases were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a yellow oil. The product was isolated by flash chromatography (DCM/MeOH, 99:1) to afford a slightly yellow oil (1.05 g, 86%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 9.76 (1H, t, <i>J</i> = 1.9 Hz), 4.69 (1H, s), 4.23–3.98 (1H, m), 2.70–2.47 (2H, m), 1.43 (9H, s), 1.24 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 155.1, 79.6, 50.6, 42.3, 28.3, 21.0, aldehyde signal not observed; HRMS [M + Na]<sup>+</sup> calcd for C<sub>11</sub>H<sub>21</sub>NO<sub>4</sub>: 210.1101, found: 210.1104.</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Ethyl (5<i>S</i>,<i>E</i>)-5-(<i>tert</i>-Butoxycarbonylamino)-2-hexenoate <b>20</b><i><b>R</b></i></h3><div class="NLM_p last">Triethyl 2-phosphonoacetate <b>19</b><i><b>R</b></i> (1.41 mL, 7.05 mmol, 1.1 equiv) was dissolved in anhydrous THF (8 mL, 0.8 M), sodium hydride (200 mg, 8.33 mmol, 1.3 equiv) was added, and the mixture was left to stir for 1 h. In a separate vessel, (1<i>S</i>)-2-formyl-1-methylethylamino 2,2-dimethylpropionate (1.11 g, 6.41 mmol, 1 equiv) was dissolved in anhydrous THF (3.2 mL, 2 M). This solution was then added dropwise to the first solution and left to stir for 1 h. Upon completion, water (10 mL) was added and the product was extracted with dichloromethane (5 × 20 mL). The organic phases were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a slightly yellow oil. This crude product was purified by flash chromatography (hexanes/ethyl acetate, 17:3) to afford a colorless oil (1.11 g, 68%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 6.90 (1H, dt, <i>J</i> = 15.6, 7.5 Hz), 5.86 (1H, dt, <i>J</i> = 15.6, 1.3 Hz), 4.41 (1H, s), 4.18 (2H, q, <i>J</i> = 7.1 Hz), 3.81 (1H, s), 2.40–2.30 (2H, m), 1.43 (9H, s), 1.28 (3H, t, <i>J</i> = 7.1 Hz), 1.14 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 166.2, 155.1, 144.6, 124.0, 79.3, 60.3, 45.6, 39.5, 28.3, 20.6, 14.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>13</sub>H<sub>23</sub>NO<sub>4</sub>: 280.1519, found: 280.1525.</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> (5<i>S</i>,<i>E</i>)-5-(<i>tert</i>-Butoxycarbonylamino)-2-hexenoic Acid <b>21</b><i><b>R</b></i></h3><div class="NLM_p last">Ethyl (5<i>S</i>,<i>E</i>)-5-(<i>tert</i>-butoxycarbonylamino)-2-hexenoate <b>20</b><i><b>R</b></i> (989 mg, 3.84 mmol, 1 equiv) was dissolved in a 1:1 mixture of water and methanol (40 mL, 0.1 M), sodium hydroxide (768 mg, 19.2 mmol, 5 equiv) was added, and the solution was stirred for 2 h. Upon completion, the methanol was largely removed under reduced pressure and 1 M HCl (20 mL) was added to pH 2. The product was extracted with dichloromethane (3 × 40 mL). The organic phases were combined, dried using MgSO<sub>4</sub>, filtered, and concentrated under reduced pressure to obtain a white solid (807 mg, 92%) that was used without further purification. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 9.77 (1H, br s), 7.01 (1H, dt, <i>J</i> = 15.6, 7.5 Hz), 5.87 (1H, dt, <i>J</i> = 15.6, 1.3 Hz), 4.48 (1H, s), 3.84 (1H, s), 2.45–2.30 (2H, m), 1.44 (9H, s), 1.16 (3H, d, <i>J</i> = 6.6 Hz); HRMS [M + Na]<sup>+</sup> calcd for C<sub>11</sub>H<sub>19</sub>NO<sub>4</sub>: 252.1206, found: 252.1212.</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> (5<i>S</i>)-5-(<i>tert</i>-Butoxycarbonylamino)hexanoic Acid <b>22</b><i><b>R</b></i></h3><div class="NLM_p last">(5<i>S</i>,<i>E</i>)-5-(<i>tert</i>-Butoxycarbonylamino)-2-hexenoic acid <b>21</b><i><b>R</b></i> (807 mg, 3.52 mmol, 1 equiv) was dissolved in methanol (35 mL, 0.1 M), palladium on carbon (10 wt %) was added (15 mg), and the solution was put under hydrogen atmosphere and shaken for 16 h. Upon completion, the reaction mixture was filtered through a pad of Celite and concentrated under reduced pressure to afford a colorless oil (796 mg, 98%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 5.56 (0.3H, s), 4.39 (0.7H, s), 3.66 (1H, s), 2.37 (2H, td, <i>J</i> = 7.4, 2.0 Hz), 1.81–1.57 (3H, m), 1.50–1.41 (3H, m), 1.45 (9H, s), 1.13 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 178.5, 155.4, 79.2, 53.4, 46.1, 36.5, 33.6, 28.4, 21.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>11</sub>H<sub>21</sub>NO<sub>4</sub>: 254.1363, found: 254.1369.</div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> (3<i>S</i>,6<i>S</i>,9<i>S</i>)-3-Benzyl-6-isobutyl-9-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-Ø)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.32–7.16 (5H, m), 4.46 (1H, dd, <i>J</i> = 11.5, 4.2 Hz), 4.04–3.89 (2H, m), 3.53–3.38 (2H, m), 3.07 (1H, dd, <i>J</i> = 14.4, 11.3 Hz), 3.02–2.93 (1H, m), 2.30–2.20 (1H, m), 2.12–2.03 (1H, m), 1.79–1.66 (3H, m), 1.66–1.58 (3H, m), 1.58–1.49 (3H, m), 1.47 (3H, d, <i>J</i> = 7.2 Hz), 1.40–1.30 (1H, m), 0.87 (3H, d, <i>J</i> = 6.3 Hz), 0.82 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, MeOD): δ 176.6, 176.5, 174.5, 173.5, 139.6, 130.3, 129.6, 127.7, 56.8, 55.5, 52.3, 40.8, 39.7, 37.7, 36.1, 28.1, 26.1, 23.3, 22.9, 22.1, 16.5; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 453.2472, found: 453.2473.</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> (6<i>S</i>,9<i>S</i>)-4-Benzyl-6-isobutyl-9-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-Ø)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.33–7.15 (5H, m), 4.79–4.72 (1H, q, <i>J</i> = 7 Hz), 4.50–4.42 (1H, m), 4.28 (1H, d, <i>J</i> = 14.2 Hz), 3.59–3.52 (1H, m), 3.28–3.12 (3H, m), 2.94–2.85 (1H, dd, <i>J</i> = 13.6, 10.1 Hz), 2.29–2.19 (1H, m), 2.11–2.03 (1H, m), 1.99–1.87 (1H, m), 1.83–1.70 (1H, m), 1.67–1.54 (2H, m), 1.53–1.41 (2H, m), 1.32 (3H, d, <i>J</i> = 7 Hz), 0.98 (3H, d, <i>J</i> = 6.7 Hz), 0.92 (3H, d, <i>J</i> = 6.8 Hz), 0.85–0.77 (1H, m); <sup>13</sup>C NMR (400 MHz, MeOD): δ 174.8, 172.6, 170.8, 169.1, 138.6, 129.1, 128.9, 128.2, 126.3, 126.2, 56.1, 55.2, 51.3, 44.3, 37.6, 36.2, 34.1, 27.4, 27.1, 25.5, 23.0, 20.0, 19.9, 19.8, 19.7, 17.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 453.2472, found: 453.2465.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> (3<i>S</i>,9<i>S</i>)-3-Benzyl-7-isobutyl-9-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-Ø)</h3><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, MeOD): δ 7.36–7.19 (5H, m), 4.74–4.66 (1H, m), 4.44–4.35 (1H, m), 4.07–3.97 (1H, m), 3.66–3.57 (1H, d, <i>J</i> = 15.0 Hz), 3.56–3.39 (2H, m), 3.16–3.01 (1H, m), 3.01–2.88 (2H, m), 2.83–2.70 (1H, m), 2.25–2.12 (1H, m), 2.05–1.35 (5H, m), 1.35–1.15 (3H, m), 0.89 (3H, d, <i>J</i> = 6.6 Hz), 0.83 (3H, d, <i>J</i> = 6.6 Hz); <sup>13</sup>C NMR (75 MHz, MeOD): δ 177.2, 174.7, 174.2, 173.7, 139.2, 130.4, 130.2, 129.7, 129.6, 127.8, 56.0, 55.0, 54.7, 40.9, 40.0, 39.6, 38.9, 35.7, 32.1, 27.9, 25.9, 23.4, 22.9, 22.7, 22.5, 21.6; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 453.2472, found: 453.2470.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> (3<i>S</i>,6<i>S</i>)-3-Benzyl-6-isobutyl-10-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-Ø)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.15 (1H, d, <i>J</i> = 6.3 Hz), 7.97 (1H, d, <i>J</i> = 8.5 Hz), 7.43–7.35 (2H, m), 7.33–7.26 (1H, m), 7.22–7.15 (2H, m), 6.97 (1H, t, <i>J</i> = 5.3 Hz), 5.00 (1H, d, <i>J</i> = 17.1 Hz), 4.54–4.45 (1H, m), 4.45–4.34 (2H, m), 4.10–4.01 (1H, m), 3.23 (1H, d, <i>J</i> = 15.7 Hz), 3.20–2.98 (2H, m), 2.28–2.19 (1H, m), 2.03–1.88 (1H, m), 1.69–1.54 (1H, m), 1.54–1.42 (4H, m), 1.42–1.30 (2H, m), 1.22 (3H, d, <i>J</i> = 7.3 Hz), 0.69 (3H, d, <i>J</i> = 6.3 Hz), 0.67 (3H, d, <i>J</i> = 6.2 Hz); <sup>13</sup>C NMR (100 MHz,): δ 172.1, 167.5, 137.4, 128.8, 127.4, 126.6, 51.6, 51.0, 50.4, 47.0, 40.7, 40.4, 37.6, 34.2, 27.4, 23.9, 22.9, 22.4, 21.9, 17.0; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 453.2472, found: 453.2468.</div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,12<i>R</i>)-3-Benzyl-6-isobutyl-9,12-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-2R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.70 (1H, d, <i>J</i> = 7.5 Hz), 8.24 (1H, s), 8.02 (1H, s), 7.32–7.13 (5H, m), 6.72 (1H, d, <i>J</i> = 6.7 Hz), 6.59 (1H, d, <i>J</i> = 7.9 Hz), 4.41–4.31 (1H, m), 4.96–4.84 (2H, m), 2.95–2.72 (2H, m), 2.23–2.11 (1H, m), 1.69–1.56 (1H, m), 1.54–1.43 (2H, m), 1.40 (3H, d, <i>J</i> = 7.0 Hz), 1.37–1.10 (5H, m), 1.00 (3H, d, <i>J</i> = 6.8 Hz), 0.81 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 176.0, 173.8, 171.6, 170.7, 138.5, 128.9, 128.1, 126.1, 53.9, 49.9, 40.8, 37.7, 36.7, 30.1, 25.9, 24.2, 22.6, 21.4, 18.3, 16.8; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2638.</div></div><div id="sec4_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,12<i>S</i>)-3-Benzyl-6-isobutyl-9,12-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.13 (1H, d, <i>J</i> = 8.0 Hz, H<sub>N</sub> Phe), 7.91 (1H, d, <i>J</i> = 7.9 Hz, H<sub>N</sub> Ala), 7.81 (1H, d, <i>J</i> = 8.4 Hz, H<sub>N</sub> Leu), 7.50–7.43 (1H, m, H<sub>N</sub> Linker), 7.30–7.13 (5H, m), 4.23–4.10 (2H, m, H<sub>α</sub> Phe and H<sub>α</sub> Ala), 4.03 (1H, td, <i>J</i> = 8.4, 6.3 Hz, H<sub>α</sub> Leu), 3.27–3.18 (1H, m, H<sub>5</sub> Linker), 3.17–3.07 (1H, m, H<sub>β</sub> Phe), 2.90 (1H, dd, <i>J</i> = 13.8, 9.1 Hz, H<sub>β</sub> Phe), 2.39–2.29 (1H, m, H<sub>2</sub> Linker), 1.49–1.38 (3H, m, H<sub>β</sub> Leu and H<sub>4</sub> Linker), 1.37–1.28 (4H, m, H<sub>β</sub> Leu, H<sub>γ</sub> Leu and H<sub>3</sub> Linker), 1.23 (3H, d, <i>J</i> = 7.3 Hz, H<sub>β</sub> Ala), 0.95 (3H, d, <i>J</i> = 6.8 Hz, H<sub>Me</sub> Linker), 0.81 (3H, d, <i>J</i> = 6.3 Hz, H<sub>δ</sub> Leu), 0.78 (3H, d, <i>J</i> = 6.2 Hz, H<sub>δ</sub> Leu); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.59 (1H, br s, H<sub>N</sub> Phe), 7.34–7.16 (5H, m), 6.57 (1H, s, H<sub>N</sub> Leu), 6.39 (2H, s, H<sub>N</sub> Ala and H<sub>N</sub> Linker), 4.79–4.57 (1H, m, H<sub>α</sub> Phe), 4.22–4.08 (1H, m, H<sub>α</sub> Ala), 3.90 (1H, br s, H<sub>α</sub> Leu), 3.67–3.52 (1H, m, H<sub>5</sub> Linker), 3.46 (1H, dd, <i>J</i> = 14.0, 5.0 Hz, H<sub>β</sub> Phe), 3.04–2.94 (1H, m, H<sub>β</sub> Phe), 2.93–2.86 (1H, br s, H<sub>5</sub> Linker), 2.48–2.36 (1H, m, H<sub>2</sub> Linker), 1.57–1.51 (3H, m, H<sub>β</sub> Leu and H<sub>3</sub> Linker), 1.49 (3H, d, <i>J</i> = 7.0 Hz, H<sub>β</sub> Ala), 1.45–1.35 (1H, m, H<sub>γ</sub> Leu), 1.32–1.21 (1H, m), 1.13 (3H, d, <i>J</i> = 4.6 Hz, H<sub>Me</sub> Linker), 0.86 (3H, d, <i>J</i> = 6.5 Hz, H<sub>δ</sub> Leu), 0.80 (3H, d, <i>J</i> = 6.5 Hz, H<sub>δ</sub> Leu); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.3, 172.7, 171.1, 170.3, 138.3, 129.0, 128.1, 126.2, 55.1, 52.9, 49.5, 41.1, 38.1, 37.7, 36.2, 30.6, 25.0, 24.2, 22.4, 22.2, 17.7, 15.3; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2640.</div></div><div id="sec4_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> (6<i>S</i>,9<i>S</i>,12<i>R</i>)-4-Benzyl-6-isobutyl-9,12-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-2R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.37 (1H, d, <i>J</i> = 8.0 Hz), 8.27 (1H, d, <i>J</i> = 5.3 Hz), 7.43–7.34 (2H, m), 7.34–7.26 (1H, m), 7.22–7.13 (2H, m), 7.03–6.95 (1H, m), 5.07 (1H, d, <i>J</i> = 17.1 Hz), 4.46–4.28 (2H, m), 4.28–4.19 (1H, m), 4.05–3.94 (1H, m), 3.16 (1H, d, <i>J</i> = 16.2 Hz), 3.04–2.82 (2H, m), 2.20–2.10 (1H, m), 2.06–1.95 (1H, m), 1.82–1.68 (1H, m), 1.68–1.54 (1H, m), 1.54–1.37 (3H, m), 1.22 (3H, d, <i>J</i> = 7.2 Hz), 0.87 (3H, d, <i>J</i> = 6.7 Hz), 0.73 (3H, d, <i>J</i> = 2.7 Hz), 0.72 (3H, d, <i>J</i> = 2.6 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.2, 173.1, 171.5, 167.6, 137.4, 128.8, 127.4, 126.7, 51.3, 50.7, 50.4, 47.8, 44.1, 40.2, 31.7, 31.4, 29.5, 24.0, 22.9, 21.9, 18.1, 16.6; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2624.</div></div><div id="sec4_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> (6<i>S</i>,9<i>S</i>,12<i>S</i>)-4-Benzyl-6-isobutyl-9,12-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.77 (1H, d, <i>J</i> = 6.2 Hz, H<sub>N</sub> Leu), 8.06 (1H, d, <i>J</i> = 8.7 Hz, H<sub>N</sub> Ala), 7.46–7.12 (5H, m), 6.89 (1H, s, H<sub>N</sub> Linker), 5.32 (1H, br s), 5.18 (1H, d, <i>J</i> = 15 Hz), 4.68–4.57 (1H, m, H<sub>α</sub> Phe), 4.56–4.47 (1H, m, H<sub>α</sub> Ala), 4.40–4.21 (1H, m, H<sub>α</sub> Leu), 3.77 (1H, d, <i>J</i> = 15.0 Hz, H<sub>β</sub> Phe), 3.21–3.09 (1H, m, H<sub>5</sub> Linker), 3.00–2.90 (1H, m, H<sub>5</sub> Linker), 2.69–2.68 (m, 1H), 2.48–2.38 (1H, m, H<sub>2</sub> Linker), 2.35–2.30 (1H, dt, 3.7, 1.8 Hz), 1.70–1.58 (1H, m, H<sub>4</sub> Linker), 1.58–1.48 (1H, m, H<sub>β</sub> Leu), 1.48–1.39 (1H, m, H<sub>γ</sub> Leu), 1.33–1.21 (3H, m, H<sub>3</sub> Linker), 1.17 (3H, d, <i>J</i> = 7.0 Hz, H<sub>β</sub> Ala), 1.14–1.07 (1H, m, H<sub>4</sub> Linker), 0.93 (3H, d, <i>J</i> = 6.7 Hz, H<sub>Me</sub> Linker), 0.91–0.83 (2H, m), 0.66 (3H, d, <i>J</i> = 6.3 Hz, H<sub>δ</sub> Leu), 0.41 (3H, d, <i>J</i> = 4.3 Hz, H<sub>δ</sub> Leu); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.41–7.17 (5H, m), 6.98 (1H, br s, H<sub>N</sub> Leu), 6.57–6.47 (1H, m, H<sub>N</sub> Linker), 5.87 (1H, d, <i>J</i> = 7.7 Hz, H<sub>N</sub> Ala), 5.34 (1H, d, <i>J</i> = 17.1 Hz, H<sub>β</sub> Phe), 4.81–4.73 (1H, m, H<sub>α</sub> Leu), 4.70 (1H, d, <i>J</i> = 14.9 Hz, H<sub>α</sub> Nphe), 4.57–4.54 (1H, m, H<sub>α</sub> Ala), 4.50 (1H, d, <i>J</i> = 17.1 Hz, H<sub>β</sub> Nphe), 3.51–3.41 (1H, m, H<sub>5</sub> Linker), 3.34 (1H, d, <i>J</i> = 14.9 Hz, H<sub>α</sub> Nphe), 3.08–2.95 (1H, m, H<sub>5</sub> Linker), 2.27–2.12 (1H, m, H<sub>2</sub> Linker), 1.64–1.57 (2H, m, H<sub>3</sub> Linker), 1.54–1.40 (3H, m, H<sub>4</sub> Linker and H<sub>γ</sub> Leu), 1.36 (3H, d, <i>J</i> = 6.8 Hz, H<sub>β</sub> Ala), 1.15 (3H, d, <i>J</i> = 6.7 Hz, H<sub>Me</sub> Linker), 0.76 (3H, d, <i>J</i> = 6.7 Hz, H<sub>δ</sub> Leu), 0.59 (3H, d, <i>J</i> = 6.5 Hz, H<sub>δ</sub> Leu); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.9, 168.0, 138.0, 128.8, 128.5, 127.6, 127.3, 126.5, 52.5, 38.6, 37.5, 31.6, 25.8, 23.7, 23.1, 22.9, 22.0, 20.9, 19.7, 18.1, 17.2, 15.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2622.</div></div><div id="sec4_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> (3<i>S</i>,9<i>S</i>,12<i>R</i>)-3-Benzyl-7-isobutyl-9,12-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-2R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.79 (1H, d, <i>J</i> = 9.2 Hz), 8.41 (1H, s), 8.31 (1H, d, <i>J</i> = 8.5 Hz), 8.21 (1H, d, <i>J</i> = 1.9 Hz), 7.78–7.68 (1H, d, <i>J</i> = 7.9 Hz), 7.48–7.39 (1H, m), 7.32–7.14 (10H, m), 4.69–4.55 (2H, m), 4.48–4.38 (1H, m), 4.32–4.23 (2H, m), 4.20–4.09 (1H, d, <i>J</i> = 18.3 Hz), 3.20–3.11 (2H, m), 3.10–3.01 (2H, m), 3.00–2.81 (2H, m), 2.80–2.69 (2H, m), 2.27–2.03 (3H, m), 1.95–1.82 (1H, m), 1.79–1.52 (4H, m), 1.47–1.28 (3H, m), 1.21 (3H, d, <i>J</i> = 7.0 Hz), 1.17 (3H, d, <i>J</i> = 6.7 Hz), 1.08 (3H, d, <i>J</i> = 7.2 Hz), 0.93 (3H, d, <i>J</i> = 6.8 Hz), 0.89 (3H, d, <i>J</i> = 6.7 Hz), 0.82 (3H, d, <i>J</i> = 6.7 Hz), 0.75 (3H, d, <i>J</i> = 6.7 Hz), 0.68 (3H, d, <i>J</i> = 6.7 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 176.2, 175.6, 174.02, 170.4, 169.1, 167.9, 138.7, 138.3, 129.2, 129.0, 128.2, 128.1, 126.2, 55.5, 55.0, 54.9, 53.8, 50.7, 44.6, 42.0, 38.4, 36.2, 31.6, 30.8, 27.2, 26.5, 25.8, 20.1, 19.9, 19.7, 17.2, 16.7, 16.4; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2625.</div></div><div id="sec4_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> (3<i>S</i>,9<i>S</i>,12<i>S</i>)-3-Benzyl-7-isobutyl-9,12-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.36 (1H, d, <i>J</i> = 8.2 Hz, H<sub>N</sub> Phe), 7.99 (1H, d, <i>J</i> = 8.0 Hz, H<sub>N</sub> Ala), 7.46–7.36 (1H, m, H<sub>N</sub> Linker), 7.30–7.15 (5H, m), 4.72–4.65 (1H, m, H<sub>α</sub> Ala), 4.25–4.19 (1H, m, H<sub>α</sub> Phe), 4.16 (1H, d, <i>J</i> = 14.0 Hz, H<sub>α</sub> Nleu), 3.48–3.39 (1H, m, H<sub>β</sub> Nleu), 3.29–3.24 (1H, m, H<sub>α</sub> Nleu), 3.19–3.11 (1H, m, H<sub>β</sub> Phe), 3.11–3.08 (1H, m, H<sub>β</sub> Nleu), 3.07–2.96 (2H, m, H<sub>5</sub> Linker), 2.75 (1H, dd, <i>J</i> = 14.3, 10.9 Hz, H<sub>β</sub> Phe), 2.34–2.31 (1H, m, H<sub>2</sub> Linker), 1.93–1.85 (1H, m, H<sub>γ</sub> Nleu), 1.71–1.61 (2H, m, H<sub>4</sub> Linker), 1.25–1.22 (2H, m, H<sub>3</sub> Linker), 1.20 (3H, d, <i>J</i> = 6.8 Hz, H<sub>β</sub> Ala), 0.91 (3H, d, <i>J</i> = 6.7 Hz, H<sub>Me</sub> Linker), 0.90 (3H, d, <i>J</i> = 6.3 Hz, H<sub>δ</sub> Nleu), 0.84 (3H, d, <i>J</i> = 6.7 Hz, H<sub>α</sub> Nleu); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.34–7.19 (5H, m), 7.25 (1H, d, <i>J</i> = 7.2 Hz, H<sub>N</sub> Phe), 6.39 (1H, d, <i>J</i> = 8.9 Hz, H<sub>N</sub> Ala), 5.70 (1H, br s, H<sub>N</sub> Linker), 5.05–4.95 (1H, m, H<sub>α</sub> Ala), 4.67 (1H, dd, <i>J</i> = 13.5, 0.9 Hz, H<sub>α</sub> Nleu), 4.50 (1H, dt, <i>J</i> = 9.2, 7.5 Hz, H<sub>α</sub> Phe), 3.61–3.49 (1H, m, H<sub>5</sub> Linker), 3.44 (1H, dd, <i>J</i> = 14.4, 8.0 Hz, H<sub>β</sub> Nleu), 3.15 (1H, d, <i>J</i> = 13.7 Hz, H<sub>α</sub> Nleu), 3.09–3.05 (2H, m, H<sub>β</sub> Phe), 3.03–2.94 (1H, m, H<sub>β</sub> Nleu), 2.84–2.75 (1H, m, H<sub>5</sub> Linker), 2.47–2.38 (1H, m, H<sub>2</sub> Linker), 1.96–1.85 (1H, m, H<sub>γ</sub> Nleu), 1.59–1.51 (2H, m, H<sub>4</sub> Linker), 1.35 (3H, d, <i>J</i> = 6.7 Hz, H<sub>β</sub> Ala), 1.33–1.25 (3H, m, H<sub>3</sub> Linker), 1.08 (3H, d, <i>J</i> = 6.7 Hz, H<sub>Me</sub> Linker), 0.95 (3H, d, <i>J</i> = 6.7 Hz, H<sub>δ</sub> Nleu), 0.92 (3H, d, <i>J</i> = 6.8 Hz, H<sub>δ</sub> Nleu); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.4, 175.0, 171.0, 169.4, 138.7, 129.1, 128.9, 128.3, 128.2, 126.3, 56.3, 55.5, 51.4, 44.2, 37.8, 36.2, 32.0, 27.5, 26.0, 20.0, 19.9, 19.8, 17.4, 17.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2626.</div></div><div id="sec4_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (3<i>S</i>,6<i>S</i>,12<i>R</i>)-3-Benzyl-6-isobutyl-10-methyl-12-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-2R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.84 (1H, d, <i>J</i> = 5.5 Hz), 7.37 (1H, d, <i>J</i> = 9.4 Hz), 7.32–7.13 (5H, m), 6.36 (1H, d, <i>J</i> = 9.4 Hz), 4.46–4.35 (1H, m), 4.28 (1H, d, <i>J</i> = 15.0 Hz), 3.83–3.73 (1H, m), 3.60 (1H, d, <i>J</i> = 15.0 Hz), 3.21 (3H, s), 2.95–2.86 (1H, m), 2.86–2.79 (1H, m), 2.71–2.62 (1H, m), 1.82–1.68 (1H, m), 1.65–1.53 (1H, m), 1.48–1.36 (1H, m), 1.32–1.05 (4H, m), 0.97 (3H, d, <i>J</i> = 6.7 Hz), 0.80 (3H, d, <i>J</i> = 6.5 Hz), 0.74 (3H, 6.5 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 178.1, 172.8, 171.9, 171.1, 138.8, 129.2, 128.7, 126.7, 54.1, 53.5, 40.6, 40.4, 40.2, 39.7, 39.5, 39.3, 39.2, 25.0, 24.5, 23.3, 21.3, 18.8; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2626.</div></div><div id="sec4_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> (3<i>S</i>,6<i>S</i>,12<i>S</i>)-3-Benzyl-6-isobutyl-10-methyl-12-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.29 (1H, d, <i>J</i> = 4.3 Hz, H<sub>N</sub> Leu), 8.06 (1H, d, <i>J</i> = 8.7 Hz, H<sub>N</sub> Phe), 7.32–7.11 (5H, m), 6.87 (1H, dd, <i>J</i> = 6.7, 2.2 Hz, H<sub>N</sub> Linker), 4.49 (1H, d, <i>J</i> = 17.8 Hz, H<sub>α</sub> Nala), 4.34 (1H, ddd, <i>J</i> = 11.2, 8.8, 4.1 Hz, H<sub>α</sub> Phe), 3.81 (1H, d, <i>J</i> = 17.7 Hz, H<sub>α</sub> Nala), 3.82–3.75 (1H, m, H<sub>α</sub> Leu), 3.29–3.23 (2H, m, H<sub>β</sub> Phe and H<sub>5</sub> Linker), 2.84 (1H, dd, <i>J</i> = 14.1, 11.4 Hz), 2.83–2.78 (1H, m, H<sub>5</sub> Linker), 2.72 (3H, s, H<sub>Me</sub> Linker), 2.69–2.65 (1H, m, H<sub>2</sub> Linker), 1.62–1.50 (1H, m, H<sub>3</sub> Linker), 1.47–1.38 (2H, m, H<sub>4</sub> Linker), 1.36–1.22 (2H, m, H<sub>β</sub> Leu and H<sub>γ</sub> Leu), 1.21–1.06 (2H, m, H<sub>β</sub> Leu and H<sub>3</sub> Linker), 0.96 (3H, d, <i>J</i> = 6.7 Hz, H<sub>Me</sub> Linker), 0.79 (3H, d, <i>J</i> = 6.2 Hz, H<sub>δ</sub> Leu), 0.71 (3H, d, <i>J</i> = 6.3 Hz, H<sub>δ</sub> Leu); <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.42 (1H, br s, H<sub>N</sub> Phe), 7.35–7.13 (1H, br s, N<sub>H</sub> Linker and N<sub>H</sub> Leu), 4.65 (1H, br s, H<sub>α</sub> Phe), 4.13–3.98 (1H, m), 3.86 (1H, br s, H<sub>α</sub> Leu), 3.62–3.48 (2H, m, H<sub>5</sub> Linker), 3.47–3.38 (1H, m, H<sub>β</sub> Phe), 3.18 (3H, s, H<sub>β</sub> Nala), 3.13–3.04 (2H, m, H<sub>5</sub> Linker and H<sub>β</sub> Phe), 2.93 (2H, s), 2.89–2.78 (1H, m, H<sub>2</sub> Linker), 1.80–1.66 (2H, br s, H<sub>3</sub> Linker), 1.54–1.45 (1H, br s, H<sub>γ</sub> Leu), 1.16 (3H, d, <i>J</i> = 6.7 Hz, H<sub>Me</sub> Linker), 0.95–0.78 (6H, m, H<sub>δ</sub> Leu); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.9, 171.8, 170.8, 169.9, 138.6, 128.9, 128.0, 126.0, 54.0, 53.9, 51.2, 35.9, 34.8, 34.4, 30.3, 23.8, 22.2, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2632.</div></div><div id="sec4_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,13<i>R</i>)-3-Benzyl-6-isobutyl-9,13-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-3R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.22–8.06 (3H, m), 7.54 (1H, t, <i>J</i> = 5.3 Hz), 7.29–7.11 (5H,m), 4.20–4.12 (1H, m), 3.95–3.86 (1H, m), 3.28–3.18 (1H, m), 3.13 (1H, dd, <i>J</i> = 13.8, 5.2 Hz), 2.99–2.88 (2H, m), 2.14–2.04 (1H, m), 1.92–1.83 (1H, m), 1.74–1.61 (1H, m), 1.53–1.23 (5H, m), 1.21 (3H, d, <i>J</i> = 7.3 Hz), 0.89 (3H, d, <i>J</i> = 6.8 Hz), 0.82–0.73 (6H, m); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.9, 172.3, 171.2, 170.4, 138.5, 129.0, 128.1, 126.1, 55.3, 53.1, 49.9, 42.4, 40.4, 40.0, 36.4, 36.0, 35.6, 28.5, 24.3, 22.5, 22.2, 20.9, 17.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2639.</div></div><div id="sec4_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,13<i>S</i>)-3-Benzyl-6-isobutyl-9,13-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-3S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.52 (1H, d, <i>J</i> = 7 Hz), 8.17–8.05 (1H, m), 8.05–7.93 (1H, m), 7.30–7.13 (5H, m), 6.90–6.80 (1H, m), 4.39–4.30 (1H, m), 4.08–3.91 (2H, m), 3.27 (1H, dd, <i>J</i> = 13.9, 4.5 Hz), 2.87–2.75 (2H, m), 2.21 (1H, d, <i>J</i> = 10.8 Hz), 1.89–1.63 (3H, m), 1.49–1.38 (1H, m), 1.38–1.34 (1H, m), 1.32 (3H, d, <i>J</i> = 7.2 Hz), 1.26–1.15 (1H, m), 1.15–1.03 (1H, m), 0.93 (3H, d, <i>J</i> = 6.2 Hz), 0.80 (3H, d, <i>J</i> = 6.5 Hz), 0.73 (3H, d, <i>J</i> = 6.3 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.2, 172.1, 171.5, 170.2, 138.4, 128.9, 128.1, 126.1, 53.9, 53.2, 50.0, 43.4, 36.5, 35.8, 34.8, 27.4, 24.2, 22.5, 21.7, 21.0, 17.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2644.</div></div><div id="sec4_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (6<i>S</i>,9<i>S</i>,13<i>R</i>)-4-Benzyl-6-isobutyl-9,13-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-3R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.24 (1H, d, <i>J</i> = 6.2 Hz), 7.65 (1H, d, <i>J</i> = 9.2 Hz) 7.42–7.34 (2H, m), 7.33–7.26 (1H, m), 7.23–7.12 (2H, m), 7.01–6.91 (1H, m), 6.55 (3H, br s), 4.99 (1H, d, <i>J</i> = 17.3 Hz), 4.80–4.70 (1H, m), 4.46 (1H, d, <i>J</i> = 17.1 Hz), 4.34 (1H, d, <i>J</i> = 16.1 Hz), 4.06–3.96 (1H, m), 3.30–3.19 (1H, m), 3.13–3.02 (1H, m), 2.23 (1H, d, <i>J</i> = 12.5 Hz), 1.85–1.75 (2H, m), 1.74–1.62 (1H, m), 1.51–1.40 (1H, m), 1.40–1.15 (2H, m), 1.23 (3H, d, <i>J</i> = 7.2 Hz), 0.93 (3H, d, <i>J</i> = 6.5 Hz), 0.65 (3H, d, <i>J</i> = 6.5 Hz), 0.60 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.8, 172.6, 172.5, 167.3, 137.5, 128.8, 127.4, 126.4, 51.7, 51.5, 50.6, 45.9, 42.3, 40.9, 40.4, 36.1, 34.7, 28.0, 23.9, 23.9, 22.9, 21.8, 21.6, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2622.</div></div><div id="sec4_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> (6<i>S</i>,9<i>S</i>,13<i>S</i>)-4-Benzyl-6-isobutyl-9,13-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-3S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.69 (1H, s), 8.10 (1H, s), 7.46–7.06 (5H, m), 6.87 (1H, s), 6.66 (1H, s), 5.34 (1H, s), 4.68–4.44 (1H, m), 4.44–4.14 (2H, m), 2.93 (2H, s), 2.14–1.64 (3H, m), 1.63–1.35 (3H, m), 1.21–1.00 (3H, m), 0.90 (3H, d, <i>J</i> = 6.8 Hz), 0.65 (2H, s), 0.36 (2H, s); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.1, 173.5, 172.0, 168.0, 138.0, 128.7, 127.3, 126.4, 51.9, 47.5, 42.1, 38.6, 35.8, 33.6, 29.6, 29.3, 24.2, 23.7, 23.0, 22.1, 20.8, 18.4, 17.5; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2621.</div></div><div id="sec4_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> (3<i>S</i>,9<i>S</i>,13<i>R</i>)-3-Benzyl-7-isobutyl-9,13-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-3R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.23 (1H, d, <i>J</i> = 8.5 Hz), 8.05 (1H, d, <i>J</i> = 7.3 Hz), 7.51–7.41 (1H, m), 7.30–7.12 (6H, m), 4.64–4.52 (1H, m), 4.32–4.22 (1H, m), 4.18 (1H, d, <i>J</i> = 14.0 Hz), 3.22 (1H, d, <i>J</i> = 14.0 Hz), 3.20–3.11 (2H, m), 3.10–2.97 (2H, m), 2.76 (1H, dd, <i>J</i> = 14.5, 10.7 Hz), 2.10 (1H, dd, <i>J</i> = 12.2, 9.6 Hz), 1.93–1.73 (3H, m), 1.63–1.50 (1H, m), 1.19 (3H, d, <i>J</i> = 6.8 Hz), 1.14–1.02 (1H, m), 0.92–0.85 (6H, m), 0.82 (3H, d, <i>J</i> = 6.7 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.8, 172.0, 171.0, 169.0, 138.6, 128.9, 128.24, 128.19, 126.2, 56.1, 55.1, 51.3, 44.6, 42.3, 36.3, 35.9, 34.1, 29.0, 27.3, 20.6, 19.8, 19.7, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2626.</div></div><div id="sec4_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> (3<i>S</i>,9<i>S</i>,13<i>S</i>)-3-Benzyl-7-isobutyl-9,13-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-3S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.36–8.25 (1H, m), 8.21–8.14 (1H, m), 8.08–7.96 (1H, m), 7.66–7.57 (1H, m), 7.36–7.29 (1H, m), 7.29–7.12 (6H, m), 4.75–4.57 (1H, m), 4.46–4.30 (1H, m), 4.27–4.16 (1H, m), 3.65–3.53 (1H, m), 3.28–3.17 (3H, m), 3.17–3.05 (2H, m), 3.03–2.86 (2H, m), 2.82–2.69 (1H, m), 2.39–2.27 (1H, m), 2.16–2.02 (1H, m), 2.00–1.55 (5H, m), 1.31–1.15 (2H, m), 1.16 (3H, dd, <i>J</i> = 18.9, 6.8 Hz), 0.97–0.87 (6H, m), 0.71 (3H, dd, <i>J</i> = 26.0, 6.7 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.1, 172.9, 172.4, 172.1, 170.7, 170.5, 169.9, 168.5, 139.2, 129.5, 129.4, 128.6, 128.5, 126.7, 126.6, 56.4, 55.8, 55.3, 53.4, 51.8, 51.6, 44.8, 44.6, 43.0, 42.2, 40.6, 40.4, 40.2, 39.9, 39.7, 39.5, 39.3, 37.7, 37.1, 36.7, 35.7, 35.1, 30.0, 29.1, 27.7, 26.2, 22.1, 21.7, 20.6, 20.30, 20.25, 20.2, 18.1, 17.9; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2630.</div></div><div id="sec4_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> (3<i>S</i>,6<i>S</i>,13<i>R</i>)-3-Benzyl-6-isobutyl-10-methyl-13-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-3R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.44 (1H, s), 8.32–8.15 (1H, m), 7.61 (1H, s), 7.31–7.06 (5H, m), 4.51–4.35 (2H, m), 3.97–3.86 (1H, m), 3.81 (1H, d, <i>J</i> = 18.1 Hz), 3.22–3.11 (1H, m), 3.11–2.93 (2H, m), 2.75 (3H, s), 2.39 (1H, d, <i>J</i> = 12.5 Hz), 1.92–1.71 (2H, m), 1.63–1.44 (1H, m), 1.41–1.19 (3H, m), 1.19–1.06 (1H, m), 0.80 (3H, d, <i>J</i> = 6.3 Hz), 0.77 (3H, d, <i>J</i> = 6.0 Hz), 0.71 (3H, 6.3 Hz); HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2626.</div></div><div id="sec4_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> (3<i>S</i>,6<i>S</i>,13<i>S</i>)-3-Benzyl-6-isobutyl-10-methyl-13-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-3S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.85 (1H, d, <i>J</i> = 5.5 Hz), 8.51–8.35 (1H, m), 7.31–7.11 (9H, m), 6.50–6.40 (1H, m), 4.50–4.18 (3H, m), 4.00–3.89 (1H, m), 3.82–3.73 (1H, m), 3.60 (1H, d, <i>J</i> = 15.0 Hz), 3.19 (3H, s), 2.75–2.66 (3H, m), 1.97–1.90 (1H, m), 1.84–1.66 (2H, m), 1.63–1.51 (1H, m), 1.45–1.33 (1H, m), 1.31–1.25 (1H, m), 1.22–1.16 (3H, m), 1.16–1.04 (1H, m), 0.98 (3H, d, <i>J</i> = 6.8 Hz), 0.83–0.70 (13H, m), 0.67 (2H, d, <i>J</i> = 6.3 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.1, 172.2, 171.4, 170.5, 138.2, 133.7, 129.0, 128.7, 128.2, 128.0, 126.2, 53.5, 53.1, 53.0, 40.4, 38.5, 37.2, 36.7, 34.8, 34.1, 28.2, 24.0, 23.7, 22.8, 22.4, 22.1, 21.2, 20.9, 19.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2631.</div></div><div id="sec4_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,14<i>R</i>)-3-Benzyl-6-isobutyl-9,14-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-4R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.28–8.12 (3H, m), 7.70–7.60 (1H, m), 7.29–7.13 (5H, m), 4.36–4.24 (1H, m), 4.08–3.96 (2H, m), 3.21–3.08 (2H, m), 2.88 (1H, dd, <i>J</i> = 14.0, 9.1 Hz), 2.70–2.59 (1H, m), 2.13–2.00 (2H, m), 1.77–1.62 (1H, m), 1.51–1.36 (2H, m), 1.34–1.13 (3H, m), 1.21 (3H, d, <i>J</i> = 7.2 Hz), 0.86–0.77 (6H, m), 0.75 (3H, 6.2 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.7, 172.9, 171.2, 170.1, 138.2, 129.0, 128.1, 126.2, 55.0, 54.9, 53.1, 50.3, 44.6, 40.8, 36.6, 33.2, 32.8, 29.8, 24.3, 22.5, 22.0, 18.0, 17.2; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2641.</div></div><div id="sec4_57" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,14<i>S</i>)-3-Benzyl-6-isobutyl-9,14-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-4S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.48 (1H, d, <i>J</i> = 6.7 Hz), 8.18 (1H, s), 8.05 (1H, s), 7.30–7.12 (7H, m), 4.41–4.30 (1H, m), 4.07–3.90 (2H, m), 3.27–3.17 (1H, m), 3.14–3.01 (1H, m), 2.86–2.68 (2H, m), 2.26–2.16 (1H, m), 1.75–1.63 (1H, m), 1.48–1.37 (1H, m), 1.31 (3H, d, <i>J</i> = 7.2 Hz), 1.27–1.17 (1H, m), 1.13–1.01 (1H, m), 0.85 (3H, d, <i>J</i> = 6.7 Hz), 0.79 (3H, d, <i>J</i> = 6.3 Hz), 0.73 (3H, d, <i>J</i> = 6.3 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.1, 173.0, 171.4, 170.3, 138.3, 128.9, 128.1, 126.1, 54.2, 53.3, 50.0, 44.1, 40.2, 36.8, 32.2, 31.8, 29.6, 24.2, 22.5, 21.8, 17.6, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2613.</div></div><div id="sec4_58" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> (6<i>S</i>,9<i>S</i>,14<i>R</i>)-4-Benzyl-6-isobutyl-9,14-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-4R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.98 (1H, d, <i>J</i> = 8.5 Hz), 7.63 (1H, d, <i>J</i> = 8.0 Hz), 7.45–7.12 (7H, m), 6.86–6.75 (1H, m), 4.97 (1H, d, <i>J</i> = 17.1 Hz), 4.68–4.57 (1H, m), 4.55–4.36 (2H, m), 4.26–4.17 (1H, m), 3.18–3.07 (1H, m), 3.01–2.79 (2H, m), 2.22–2.10 (1H, m), 1.64–1.36 (6H, m), 1.33–1.10 (9H, m), 1.03–0.92 (1H, m), 0.90–0.75 (2H, m), 0.68 (3H, d, <i>J</i> = 6.5 Hz), 0.63 (3H, d, <i>J</i> = 6.3 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 175.0, 172.8, 172.5, 167.6, 137.3, 128.8, 127.5, 126.5, 51.8, 51.4, 49.9, 41.3, 40.8, 37.9, 32.1, 26.6, 23.9, 23.0, 21.7, 17.5, 16.8; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2623.</div></div><div id="sec4_59" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> (6<i>S</i>,9<i>S</i>,14<i>S</i>)-4-Benzyl-6-isobutyl-9,14-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-4S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.20 (1H, d, <i>J</i> = 6.2 Hz), 7.86 (1H, d, <i>J</i> = 8.7 Hz), 7.48–7.34 (2H, m), 7.34–7.24 (1H, m), 7.24–7.13 (2H, m), 7.12–7.01 (1H, m), 4.93 (1H, d, <i>J</i> = 17.1 Hz), 4.74–4.57 (1H, m), 4.51–4.32 (2H, m), 4.10–3.97 (1H, m), 3.27 (1H, d, <i>J</i> = 15.7 Hz), 3.12–2.98 (1H, m), 2.95–2.79 (1H, m), 2.37–2.21 (1H, m), 2.06–1.78 (2H, m), 1.65–1.26 (5H, m), 1.22 (3H, d, <i>J</i> = 7.3 Hz), 0.83 (3H, d, <i>J</i> = 6.7 Hz), 0.75–0.55 (6H, m); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.3, 172.8, 172.5, 167.5, 137.4, 128.8, 127.4, 126.5, 51.5, 51.0, 50.4, 46.4, 43.7, 40.7, 31.4, 30.9, 29.2, 24.0, 22.8, 22.0, 17.8, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2621.</div></div><div id="sec4_60" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> (3<i>S</i>,9<i>S</i>,14<i>R</i>)-3-Benzyl-7-isobutyl-9,14-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-4R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.12–7.99 (2H, m), 7.44–7.34 (1H, m), 7.31–7.10 (6H, m), 4.53–4.43 (1H, m), 4.34–4.24 (1H, m), 4.18 (1H, d, <i>J</i> = 14.2 Hz), 3.24–3.12 (2H, m), 3.11–2.99 (1H, m), 2.99–2.76 (3H, m), 2.19–1.94 (2H, m), 1.93–1.72 (4H, m), 1.49–1.30 (2H, m), 1.19 (3H, d, <i>J</i> = 6.7 Hz), 1.12 (1H, d, <i>J</i> = 6.3 Hz), 1.09–1.00 (1H, m), 0.94–0.83 (7H, m), 0.81 (3H, d, <i>J</i> = 6.5 Hz), 0.77–0.64 (2H, m); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.8, 173.3, 170.6, 168.5, 138.6, 129.1, 128.9, 128.1, 126.1, 56.0, 54.9, 54.7, 51.5, 45.2, 45.0, 36.3, 33.0, 31.2, 30.7, 27.4, 19.9, 19.79, 19.75, 18.7, 16.6; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2620.</div></div><div id="sec4_61" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> (3<i>S</i>,9<i>S</i>,14<i>S</i>)-3-Benzyl-7-isobutyl-9,14-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-4S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): <i>δ</i> 8.39 (1H, dd, <i>J</i> = 70.7, 6.42 Hz), 8.00 (1H, dd, <i>J</i> = 46.4, 8.4 Hz), 7.87 (1H, dt, <i>J</i> = 84.4, 5.5 Hz), 7.31–7.11 (5H, m), 4.68 (1H, dt, <i>J</i> = 51.1, 7.2 Hz), 4.25 (1H, dd, <i>J</i> = 120.0, 18 Hz), 4.35 (1H, m), 3.19–2.72 (6H, m), 2.07 (1H, dm, 181.8 Hz), 2.06–1.21 (6H, m), 1.14 (3H, dd, <i>J</i> = 28.1, 6.6 Hz), 0.90–0.75 (6H, m), 0.68 (3H, dd, <i>J</i> = 11.0, 6.7 Hz); HRMS [M + Na]+ calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2622.</div></div><div id="sec4_62" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> (3<i>S</i>,6<i>S</i>,14<i>R</i>)-3-Benzyl-6-isobutyl-10-methyl-14-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-4R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.07 (1H, d, <i>J</i> = 6.5 Hz), 7.90 (1H, d, <i>J</i> = 8.9 Hz), 7.53–7.45 (1H, m), 7.27–7.10 (5H, m), 4.47–4.30 (2H, m), 3.97–3.89 (1H, m), 3.86 (1H, d, <i>J</i> = 18.1 Hz), 3.28–3.18 (1H, m), 3.15 (1H, s), 3.11–3.03 (1H, m), 2.75 (3H, s), 2.35–2.22 (1H, m), 2.13–2.03 (1H, m), 1.67–1.55 (1H, m), 1.55–1.43 (1H, m), 1.43–1.35 (1H, m), 1.35–1.25 (1H, m), 1.20–1.07 (2H, m), 0.88–0.78 (6H, m), 0.77–0.69 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.4, 171.8, 170.7, 168.9, 138.0, 129.1, 128.8, 128.2, 128.0, 126.1, 54.1, 53.4, 52.1, 45.2, 40.1, 37.2, 34.7, 32.6, 32.1, 30.7, 24.1, 22.4, 21.9, 18.7; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2628.</div></div><div id="sec4_63" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> (3<i>S</i>,6<i>S</i>,14<i>S</i>)-3-Benzyl-6-isobutyl-10-methyl-14-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-4S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (1H, d, <i>J</i> = 5.5 Hz), 8.21 (1H, m), 7.98 (1H, d, <i>J</i> = 8.5 Hz), 7.14–6.88 (11H, m), 6.33 (1H, d, <i>J</i> = 9.1 Hz), 4.25–3.99 (3H, m), 3.71–3.61 (1H, m), 3.60–3.51 (1H, m), 3.37 (1H, d, <i>J</i> = 15.0 Hz), 3.23–3.14 (2H, m), 3.04–2.92 (2H, m), 2.97 (3H, s), 2.87–2.73 (1H, m), 2.56 (2H, d, <i>J</i> = 13.3 Hz), 2.33 (1H, d, <i>J</i> = 11.8 Hz), 2.27 (2H, s), 2.21–2.07 (1H, m), 1.98–1.86 (1H, m), 1.80–1.70 (1H, m), 1.53–1.21 (3H, m), 1.21–0.81 (6H, m), 0.73 (2H, d, <i>J</i> = 6.7 Hz), 0.66 (2H, d, <i>J</i> = 6.7 Hz), 0.56 (5H, 6.3 Hz), 0.53–0.43 (5H, m); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 174.9, 172.8, 172.1, 171.8, 171.4, 170.35, 170.31, 169.6, 138.4, 138.2, 129.05, 128.98, 128.7, 128.2, 128.00, 127.96, 126.2, 126.1, 53.9, 52.9, 52.1, 45.6, 44.5, 39.8, 39.04, 39.02, 37.2, 36.1, 34.4, 32.9, 30.7, 29.8, 29.5, 29.1, 24.0, 23.9, 22.8, 22.2, 20.9, 18.5, 17.9; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2627.</div></div><div id="sec4_64" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,15<i>R</i>)-3-Benzyl-6-isobutyl-9,15-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-5R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.33–7.14 (5H, m), 4.56 (1H, dd, <i>J</i> = 11.4, 3.9 Hz), 4.10–4.02 (1H, m), 3.89–3.76 (2H, m), 3.47 (1H, dd, <i>J</i> = 14.7, 3.9 Hz), 2.98 (1H, dd, <i>J</i> = 15.4, 11.7 Hz), 2.22 (1H, dt, <i>J</i> = 13.3, 3.9 Hz), 2.05 (1H, td, <i>J</i> = 13.0, 3.4 Hz), 1.75–1.61 (2H, m), 1.53 (3H, d, <i>J</i> = 7.2 Hz), 1.52–1.38 (3H, m), 1.38–1.27 (2H, m), 1.10 (3H, d, <i>J</i> = 6.5 Hz), 0.87 (3H, d, <i>J</i> = 6.5 Hz), 0.81 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, MeOD): δ 176.5, 176.4, 174.7, 172.9, 139.6, 130.2, 129.6, 127.7, 56.1, 55.7, 52.4, 46.8, 40.8, 38.2, 36.2, 35.5, 26.1, 23.3, 22.9, 22.0, 21.9, 16.3; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2640.</div></div><div id="sec4_65" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,15<i>S</i>)-3-Benzyl-6-isobutyl-9,15-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Ø-5S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.31–7.16 (5H, m), 4.14–4.02 (2H, m), 3.79–3.68 (1H, m), 3.58–3.52 (1H, m), 3.49–3.33 (2H, m), 2.20–2.04 (2H, m), 1.84–1.56 (5H, m), 1.55–1.40 (2H, m), 1.33 (3H, d, <i>J</i> = 7.3 Hz), 1.16 (3H, d, <i>J</i> = 6.7 Hz), 0.91 (3H, d, <i>J</i> = 6.7 Hz), 0.87 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, MeOD): δ 175.6, 175.4, 172.3, 171.3, 138.8, 128.7, 128.0, 126.1, 90.0 58.0, 53.4, 50.3, 45.9, 37.5, 34.5, 34.2, 33.4, 24.7, 22.1, 21.6, 20.4, 20.0, 15.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2630.</div></div><div id="sec4_66" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> (6<i>S</i>,9<i>S</i>,15<i>R</i>)-4-Benzyl-6-isobutyl-9,15-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-5R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.49–7.24 (5H, m), 4.99–4.90 (2H, m), 4.67 (1H, d, <i>J</i> = 16.7 Hz), 4.41 (1H, d, <i>J</i> = 16.6 Hz), 4.24 (1H, q, <i>J</i> = 7.3 Hz), 3.79–3.69 (1H, m), 3.60 (1H, d, <i>J</i> = 16.4 Hz), 2.45–2.36 (1H, m), 2.04–1.94 (1H, m), 1.85–1.73 (1H, m), 1.64–1.44 (5H, m), 1.38 (3H, d, <i>J</i> = 7.3 Hz), 1.31–1.21 (1H, m), 0.99 (3H, d, <i>J</i> = 6.5 Hz), 0.78 (3H, d, <i>J</i> = 6.5 Hz), 0.72 (3H, d, 6.5 Hz); <sup>13</sup>C NMR (100 MHz, MeOD): δ 177.2, 176.0, 175.6, 138.1, 130.5, 129.3, 128.4, 54.4, 53.7, 52.3, 49.3, 47.3, 42.7, 36.0, 35.2, 26.0, 24.4, 23.7, 22.3, 22.1, 17.1; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2625.</div></div><div id="sec4_67" class="NLM_sec NLM_sec_level_2"><div id="ac_i81" class="anchor-spacer"></div><h3 class="article-section__title" id="_i81"> (6<i>S</i>,9<i>S</i>,15<i>S</i>)-4-Benzyl-6-isobutyl-9,15-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nphe-5S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.45–7.35 (2H, m), 7.35–7.28 (1H, m), 7.28–7.18 (2H, m), 5.27 (1H, d, <i>J</i> = 17.4 Hz), 4.69 (1H, dd, <i>J</i> = 16.5, 1.2 Hz), 4.42 (1H, d, <i>J</i> = 17.4 Hz), 4.21 (1H, q, <i>J</i> = 7.2 Hz), 4.00 (1H, dd, <i>J</i> = 8.6, 5.7 Hz), 3.78–3.67 (1H, m), 3.25 (1H, d, <i>J</i> = 16.6 Hz), 2.21–2.11 (2H, m), 1.81–1.49 (7H, m), 1.35 (3H, d, <i>J</i> = 7.2 Hz), 1.14 (3H, d, <i>J</i> = 6.5 Hz), 0.88–0.79 (6H, m); <sup>13</sup>C NMR (100 MHz, MeOD): δ 177.4, 176.1, 173.8, 170.7, 138.1, 130.3, 129.0, 127.8, 53.3, 52.3, 52.0, 51.5, 47.5, 41.8, 36.1, 34.9, 25.8, 24.4, 23.6, 22.3, 21.7, 16.9; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2630.</div></div><div id="sec4_68" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> (3<i>S</i>,9<i>S</i>,15<i>R</i>)-3-Benzyl-7-isobutyl-9,15-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-5R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06–7.94 (1H, m), 7.83–7.68 (1H, m), 7.68–7.50 (2H, m), 7.30–7.11 (5H, m), 6.96 (1H, s), 4.79–4.62 (1H, m), 4.61–4.51 (1H, m), 4.49–4.35 (1H, m), 3.84–3.67 (1H, m), 3.60–3.44 (1H, m), 3.21–3.03 (2H, m), 3.01–2.88 (1H, m),2.87–2.74 (1H, m), 2.34–2.17 (1H, m), 2.14–1.92 (1H, m), 1.90–1.71 (2H, m), 1.58–1.30 (3H, m), 1.31–1.20 (1H, m), 1.16 (2H, d, <i>J</i> = 6.4 Hz), 1.06 (2H, d, <i>J</i> = 6.6 Hz), 0.91 (2H, d, <i>J</i> = 6.6 Hz), 0.88–0.83 (2H, m), 0.83–0.78 (2H, d, <i>J</i> = 6.6 Hz), 0.74–0.64 (3H, m); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.2, 172.1, 172.0, 171.3, 170.2, 169.6, 168.5, 168.1, 138.3, 137.7, 129.1, 128.14, 128.10, 126.3, 126.2, 55.5, 54.5, 54.2, 54.1, 52.9, 50.1, 43.4, 43.1, 42.8, 40.4, 37.4, 35.7, 35.3, 34.6, 34.0, 33.2, 27.1, 25.5, 22.6, 21.6, 21.4, 20.1, 20.04, 19.96, 19.8, 19.4, 17.9, 17.5; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2623.</div></div><div id="sec4_69" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> (3<i>S</i>,9<i>S</i>,15<i>S</i>)-3-Benzyl-7-isobutyl-9,15-dimethyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nleu-5S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.32–7.15 (5H, m), 4.57 (1H, q, <i>J</i> = 7.2 Hz), 4.44 (1H, dd, <i>J</i> = 11.7, 3.6 Hz), 4.07 (1H, d, <i>J</i> = 14.7 Hz), 3.84–3.71 (1H, m), 3.65–3.54 (1H, m), 3.42 (1H, dd, <i>J</i> = 14.4, 3.6 Hz), 3.24–3.15 (1H, m), 2.94 (1H, dd, <i>J</i> = 14.2, 11.6 Hz), 2.15–2.04 (2H, m), 1.94–1.80 (2H, m), 1.75–1.51 (2H, m), 1.45–1.35 (1H, m), 1.32 (3H, d, <i>J</i> = 7.2 Hz), 1.15 (3H, d, <i>J</i> = 6.5 Hz), 0.97 (3H, d, <i>J</i> = 6.5 Hz), 0.89 (3H, d, <i>J</i> = 6.7 Hz); <sup>13</sup>C NMR (100 MHz, MeOD): δ 177.8, 177.2, 173.2, 171.7, 139.8, 130.1, 129.6, 127.7, 59.2, 57.1, 53.9, 47.8, 47.6, 37.2, 36.0, 35.6, 29.7, 24.8, 22.9, 20.61, 20.57, 16.6; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2621.</div></div><div id="sec4_70" class="NLM_sec NLM_sec_level_2"><div id="ac_i84" class="anchor-spacer"></div><h3 class="article-section__title" id="_i84"> (3<i>S</i>,6<i>S</i>,15<i>R</i>)-3-Benzyl-6-isobutyl-10-methyl-15-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-5R)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, MeOD): δ 7.33–7.15 (5H, m), 4.57 (1H, dd, <i>J</i> = 11.8, 3.9 Hz), 4.10–4.02 (1H, m), 3.89–3.76 (2H, m), 3.47 (1H, dd, <i>J</i> = 14.4, 3.9 Hz), 2.98 (1H, dd, <i>J</i> = 15.3, 11.7 Hz), 2.22 (1H, dt, <i>J</i> = 13.3, 3.8 Hz), 2.05 (1H, td, <i>J</i> = 13.3, 3.4 Hz), 1.74–1.61 (2H, m), 1.58–1.41 (3H, m), 1.53 (3H, d, <i>J</i> = 7.2 Hz), 1.41–1.28 (2H, m), 1.10 (3H, d, <i>J</i> = 6.5 Hz), 0.87 (3H, d, <i>J</i> = 6.3 Hz), 0.82 (3H, d, <i>J</i> = 6.5 Hz); <sup>13</sup>C NMR (100 MHz, MeOD): δ 177.3, 174.8, 174.4, 173.0, 139.4, 130.4, 130.2, 129.7, 129.6, 127.8, 55.9, 55.0, 54.7, 47.2, 40.8, 39.9, 38.7, 35.7, 31.9, 25.9, 23.4, 23.3, 22.0, 21.6; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2628.</div></div><div id="sec4_71" class="NLM_sec NLM_sec_level_2"><div id="ac_i85" class="anchor-spacer"></div><h3 class="article-section__title" id="_i85"> (3<i>S</i>,6<i>S</i>,15<i>S</i>)-3-Benzyl-6-isobutyl-10-methyl-15-methyl-1,4,7,10-tetraza-2,5,8,11-cyclopentadecanetetrone (Nala-5S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, MeOD): δ 4.34–4.14 (5H, m), 4.70–4.60 (1H, m), 4.39–4.23 (1H, m), 4.14–3.90 (1H, m), 3.55 (1H, d, <i>J</i> = 15.0 Hz), 3.25–3.13 (3H, m), 2.99–2.86 (1H, dd, <i>J</i> = 13.5, 10.6 Hz), 2.64–1.75 (4H, m), 1.74–1.25 (6H, m), 1.25–1.12 (3H, m), 0.92–0.74 (6H, m); <sup>13</sup>C NMR (75 MHz, MeOD): δ 175.2, 171.9, 137.0, 128.8, 128.1, 126.4, 55.6, 53.5, 52.8, 46.1, 39.5, 38.6, 37.3, 34.4, 32.9, 31.1, 24.4, 24.3, 22.0, 21.6, 19.9, 17.1, 17.0; HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2629.</div></div><div id="sec4_72" class="NLM_sec NLM_sec_level_2"><div id="ac_i86" class="anchor-spacer"></div><h3 class="article-section__title" id="_i86"> (3<i>S</i>,6<i>S</i>)-3-Benzyl-10-isobutyl-6-methyl-1,4,7,10-tetraazacyclopentadecane-2,5,8,11-tetraone (<b>32</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.89 (0.6H, d, <i>J</i> = 5.3 Hz, H<sub>N</sub> Ala), 8.77 (0.4H, d, <i>J</i> = 5.6 Hz, H<sub>N</sub> Ala), 8.48 (0.4H, br s), 8.42 (0.4H, d, <i>J</i> = 9.1 Hz, H<sub>N</sub> Phe), 7.45–7.40 (0.4H, m, H<sub>N</sub> Linker), 7.37 (0.6H, d, <i>J</i> = 9.6 Hz, H<sub>N</sub> Phe), 7.31–7.12 (5H, m), 6.47 (0.6H, d, <i>J</i> = 8.5 Hz, H<sub>N</sub> Linker), 6.30 (0.2H, br s), 4.45–4.28 (1H, m, H<sub>α</sub> Phe), 4.16 (1H, d, <i>J</i> = 14.7 Hz, H<sub>α</sub> Nleu), 3.93–3.76 (1H, m, H<sub>α</sub> Ala), 3.62 (1H, d, <i>J</i> = 14.7 Hz, H<sub>α</sub> Nleu), 3.66–3.58 (2H, m), 3.25–3.17 (1H, m, H<sub>5</sub> Linker), 3.16–3.08 (1H, m, H<sub>β</sub> Phe), 2.99–2.90 (1H, ddd, <i>J</i> = 9.9, 6.8, 3.2 Hz, H<sub>β</sub> Phe), 2.87–2.75 (1H, m, H<sub>5</sub> Linker), 2.74–2.65 (1H, m), 2.41–2.29 (0.4H, m), 2.04–1.95 (0.6H, dt, <i>J</i> = 14.3, 3.4 Hz), 1.89–1.69 (1H, m, H<sub>γ</sub> Nleu), 1.64–1.34 (2H, m, H<sub>4</sub> Linker), 1.16–1.04 (1H, m), 1.00 (1H, d, <i>J</i> = 7.3 Hz, H<sub>β</sub> Ala), 0.92 (1.5H, d, <i>J</i> = 6.7 Hz, H<sub>δ</sub> Leu), 0.87 (1.5H, d, <i>J</i> = 6.7 Hz, H<sub>δ</sub> Leu), 0.82 (1.5H, d, <i>J</i> = 6.7 Hz, H<sub>δ</sub> Leu), 0.75 (1.5H, d, <i>J</i> = 6.7 Hz, H<sub>δ</sub> Leu). HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 453.2472, found: 453.2472.</div></div><div id="sec4_73" class="NLM_sec NLM_sec_level_2"><div id="ac_i87" class="anchor-spacer"></div><h3 class="article-section__title" id="_i87"> (3<i>S</i>,9<i>S</i>)-3-Benzyl-9-isobutyl-7-methyl-1,4,7,10-tetraazacyclopentadecane-2,5,8,11-tetraone (<b>33</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.32 (1H, d, <i>J</i> = 8.4 Hz, H<sub>N</sub> Phe), 7.92 (1H, d, <i>J</i> = 8.4 Hz, H<sub>N</sub> Leu), 7.37 (1H, t, <i>J</i> = 5.6 Hz, H<sub>N</sub> Linker), 7.31–7.14 (5H, m), 4.75–4.62 (1H, m, H<sub>α</sub> Leu), 4.29 (1H, d, <i>J</i> = 14.2 Hz, H<sub>α</sub> Nala), 4.27–4.21 (1H, m, H<sub>α</sub> Phe), 3.21 (1H, d, <i>J</i> = 14.0 Hz, H<sub>α</sub> Nala), 3.17–3.10 (1H, m, H<sub>β</sub> Phe), 3.13 (3H, s, H<sub>β</sub> Nala), 3.04 (1H, q, <i>J</i> = 5.8 Hz, H<sub>5</sub> Linker), 2.73 (1H, dd, <i>J</i> = 14.0, 10.3 Hz), 2.22 (1H, ddd, <i>J</i> = 12.6, 9.7, 4.7 Hz, H<sub>2</sub> Linker), 1.95–1.87 (1H, m, H<sub>2</sub> Linker), 1.68–1.54 (2H, m, H<sub>4</sub> Linker and H<sub>γ</sub> Leu), 1.53–1.32 (4H, m, H<sub>β</sub> Leu), 1.26–1.13 (1H, m, H<sub>4</sub> Linker), 0.89 (6H, dd, <i>J</i> = 6.5, 5.5 Hz, H<sub>δ</sub> Leu). HRMS [M + Na]<sup>+</sup> calcd for C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>: 453.2472, found: 453.2470.</div></div><div id="sec4_74" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,12<i>S</i>)-3-Benzyl-6-isobutyl-7,9,12-trimethyl-1,4,7,10-tetraazacyclopentadecane-2,5,8,11-tetraone (NMeLeu-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.06 (dd, <i>J</i> = 32.5, 7.7 Hz, 1H), 7.70 (dd, <i>J</i> = 17.3, 8.5 Hz, 1H), 7.56 (dd, <i>J</i> = 7.9, 3.5 Hz, 1H), 7.30–7.15 (m, 5H), 4.66–4.52 (m, 2H), 4.11–4.04 (m, 1H), 3.23–3.07 (m, 3H), 2.88 (d, <i>J</i> = 10.2 Hz, 2H), 2.28 (s, 2H), 2.11 (s, 1H), 1.84–1.69 (m, 1H), 1.63–1.53 (m, 1H), 1.46–1.31 (m, 2H) 1.28–1.19 (m, 2H), 1.14 (dd, <i>J</i> = 14.8, 6.8 Hz, 3H), 0.98 (d, <i>J</i> = 6.1 Hz, 1H), 0.93 (d, <i>J</i> = 6.7 Hz, 3H), 0.86 (d, <i>J</i> = 6.4 Hz, 3H), 0.83 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 175.4, 171.7, 170.6, 169.4, 137.8, 129.0, 128.2, 126.4, 58.6, 54.3, 44.4, 37.4, 36.5, 36.2, 35.6, 30.0, 28.5, 24.3, 23.1, 21.8, 18.0, 15.8. HRMS [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>38</sub>N<sub>4</sub>O<sub>4</sub>: 481.2785, found: 481.2797.</div></div><div id="sec4_75" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> (3<i>S</i>,6<i>S</i>,9<i>S</i>,12<i>S</i>)-3-Benzyl-6-isobutyl-6,9,12-trimethyl-1,4,7,10-tetraazacyclopentadecane-2,5,8,11-tetraone (αMe-Leu-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.52 (s, 1H), 8.34 (d, <i>J</i> = 5.2 Hz, 1H), 7.31–7.14 (m, 6H), 6.87 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 4.03 (dd, <i>J</i> = 7.2, 5.3 Hz, 1H), 3.82–3.76 (m, 1H), 3.41–3.37 (m, 1H), 3.29–3.24 (m, 2H), 2.79 (dd, <i>J</i> = 13.6, 2.7 Hz, 1H), 2.32–2.23 (m, 1H), 1.96 (d, <i>J</i> = 6.8 Hz, 1H), 1.66–1.42 (m, 4H), 1.32 (d, <i>J</i> = 8.8 Hz, 1H), 1.17 (d, <i>J</i> = 7.1 Hz, 3H), 1.13–1.07 (m, 1H), 0.96 (s, 3H), 0.88 (d, <i>J</i> = 6.7 Hz, 3H), 0.84 (d, <i>J</i> = 6.6 Hz, 3H), 0.79 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 176.6, 174.5, 174.3, 170.3, 139.5, 129.1, 128.0, 126.0, 59.1, 57.7, 49.4, 38.0, 33.5, 30.2, 25.2, 24.8, 23.8, 23.5, 23.3, 17.7, 15.9. HRMS [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>38</sub>N<sub>4</sub>O<sub>4</sub>: 481.2785, found: 481.2790.</div></div><div id="sec4_76" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> (3<i>S</i>,6<i>R</i>,9<i>S</i>,12<i>S</i>)-3-Benzyl-6-isobutyl-7,9,12-trimethyl-1,4,7,10-tetraazacyclopentadecane-2,5,8,11-tetraone ((<span class="smallcaps smallerCapital">d</span>)NMeLeu-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.02 (d, <i>J</i> = 9.1 Hz, 1H), 7.83 (d, <i>J</i> = 8.3 Hz, 1H), 7.63 (t, <i>J</i> = 5.8 Hz, 1H), 7.33–7.11 (m, 5H), 4.76 (dd, <i>J</i> = 8.3, 6.8 Hz, 2H), 4.43 (td, <i>J</i> = 9.6, 5.1 Hz, 1H), 3.25–3.15 (m, 1H), 3.04 (dd, <i>J</i> = 13.7, 5.1 Hz, 1H), 2.94–2.86 (m, 1H), 2.83 (s, 3H), 2.72 (dd, <i>J</i> = 13.7, 10.0 Hz, 1H), 2.35 (q, <i>J</i> = 6.5 Hz, 1H), 1.46 (ddd, <i>J</i> = 13.4, 8.4, 6.7 Hz, 2H), 1.37–1.19 (m, 3H), 1.15 (d, <i>J</i> = 6.7 Hz, 3H), 1.13–1.06 (m, 1H), 1.03–0.97 (m, 1H), 0.92 (d, <i>J</i> = 6.6 Hz, 3H), 0.82 (d, <i>J</i> = 6.5 Hz, 3H), 0.72 (d, <i>J</i> = 6.5 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 175.0, 173.6, 171.1, 169.9, 138.2, 129.0, 128.0, 126.1, 54.4, 54.2, 44.2, 37.4, 37.0, 36.1, 31.4, 29.9, 26.1, 24.1, 22.6, 22.3, 17.5, 15.9. HRMS [M + Na]<sup>+</sup> calcd for C<sub>25</sub>H<sub>38</sub>N<sub>4</sub>O<sub>4</sub>: 481.2785, found: 481.2793.</div></div><div id="sec4_77" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> (3<i>S</i>,6<i>R</i>,9<i>S</i>,12<i>S</i>)-3-Benzyl-6-isobutyl-9,12-dimethyl-1,4,7,10-tetraazacyclopentadecane-2,5,8,11-tetraone ((<span class="smallcaps smallerCapital">d</span>)Leu-2S)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.63–8.42 (m, 2H), 8.01 (dd, <i>J</i> = 21.2, 7.6 Hz, 1H), 7.73 (d, <i>J</i> = 7.7 Hz, 1H), 7.27–7.14 (m, 6H), 4.36–4.31 (m, 1H), 4.28–4.22 (m, 1H), 4.18–4.12 (m, 1H), 4.08–4.01 (m, 1H), 3.92–3.83 (m, 1H), 3.17 (ddd, <i>J</i> = 13.8, 3.8, 1.8 Hz, 1H), 2.76–2.65 (m, 2H), 2.37–2.29 (m, 1H), 2.15 (d, <i>J</i> = 3.8 Hz, 1H), 1.61 (s, 1H), 1.30–1.20 (m, 4H), 1.15 (d, <i>J</i> = 7.1 Hz, 2H), 1.05–0.97 (m, 3H), 0.77 (d, <i>J</i> = 6.4 Hz, 3H), 0.68 (t, <i>J</i> = 6.2 Hz, 3H). <sup>13</sup>C NMR (101 MHz, DMSO): δ 175.6, 174.5, 171.7, 170.8, 129.0, 128.0, 127.9, 126.0, 48.2, 38.1, 26.1, 23.9, 23.8, 22.9, 22.7, 22.2, 21.9, 17.4, 16.4, 16.0. HRMS [M + Na]<sup>+</sup> calcd for C<sub>24</sub>H<sub>36</sub>N<sub>4</sub>O<sub>4</sub>: 467.2629, found: 467.2630.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02036" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02036?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02036</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Detailed NMR analysis of Nleu-2R, Nleu-2S, Nleu-5R, and Nleu-5S; analysis of MD simulations; permeability data; and NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES, molecular formula strings, and all permeability data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_001.csv">jm0c02036_si_001.csv (7.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf">jm0c02036_si_002.pdf (30.96 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02036" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45558" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45558" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philippe Sarret</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7627-701X" title="Orcid link">http://orcid.org/0000-0002-7627-701X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ecbc848580859c9c89c2bf8d9e9e8998c2acb9bf84899e8e9e83838789c28f8d"><span class="__cf_email__" data-cfemail="33635b5a5f5a4343561d6052414156471d7366605b564151415c5c58561d5052">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sereina Riniker</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Applied Biosciences, ETH
Zürich, Vladimir-Prelog-Weg 2, 8093 Zürich, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1893-4031" title="Orcid link">http://orcid.org/0000-0003-1893-4031</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7a0908131413111f083a1f0e1200541912"><span class="__cf_email__" data-cfemail="cab9b8a3a4a3a1afb88aafbea2b0e4a9a2">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Comeau</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Ries</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Applied Biosciences, ETH
Zürich, Vladimir-Prelog-Weg 2, 8093 Zürich, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0945-8304" title="Orcid link">http://orcid.org/0000-0002-0945-8304</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Stadelmann</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Applied Biosciences, ETH
Zürich, Vladimir-Prelog-Weg 2, 8093 Zürich, Switzerland</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2919-0939" title="Orcid link">http://orcid.org/0000-0003-2919-0939</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jacob Tremblay</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sylvain Poulet</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ulrike Fröhlich</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jérôme Côté</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pierre-Luc Boudreault</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rabeb Mouna Derbali</span> - <span class="hlFld-Affiliation affiliation">Faculté
de pharmacie, Université de Montréal, C.P. 6128, succursale Centre-ville, Montréal, Quebec, Canada H3C 3J7</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michel Grandbois</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Grégoire Leclair</span> - <span class="hlFld-Affiliation affiliation">Faculté
de pharmacie, Université de Montréal, C.P. 6128, succursale Centre-ville, Montréal, Quebec, Canada H3C 3J7</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Éric Marsault</span> - <span class="hlFld-Affiliation affiliation">Département
de Pharmacologie−Physiologie, Institut de Pharmacologie de
Sherbrooke, Université de Sherbrooke, 3001, 12e av nord, Sherbrooke, Quebec, Canada J1H5N4</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-5305-8762" title="Orcid link">http://orcid.org/0000-0002-5305-8762</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>C.C. and B.R. contributed equally to this work.</p></li><li><span class="author-information-subsection-header">Funding</span><p>Financial support from the Natural Sciences and Engineering Research Council of Canada, the Canada Foundation for Innovation, the Centre des Neurosciences de Sherbrooke, the Swiss National Science Foundation (grant no. 20002-178762), and ETH Zurich (ETH-34 17-2) is gratefully acknowledged.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6336-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i94">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">M. M. Gregor Weiss and Shuzhe Wang are acknowledged for helpful discussions.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ACN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">DEPBT</td><td class="NLM_def"><p class="first last">(3-(diethoxyphosphoryloxy)-1,2,3-benzotriazin-4(3H)-one)</p></td></tr><tr><td class="NLM_term">DIBAL</td><td class="NLM_def"><p class="first last">diisobutylaluminium hydride</p></td></tr><tr><td class="NLM_term">DIC</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>′-diisopropylcarbodiimide</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last">N,N-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DPPA</td><td class="NLM_def"><p class="first last">diphenylphosphoryl azide</p></td></tr><tr><td class="NLM_term">EXSY</td><td class="NLM_def"><p class="first last">exchange spectroscopy</p></td></tr><tr><td class="NLM_term">KHMDS</td><td class="NLM_def"><p class="first last">potassium bis(trimethylsilyl)amide</p></td></tr><tr><td class="NLM_term">Nala</td><td class="NLM_def"><p class="first last">alanine peptoid (N-methylglycine)</p></td></tr><tr><td class="NLM_term">Nleu</td><td class="NLM_def"><p class="first last">leucine peptoid (N-isobutylglycine)</p></td></tr><tr><td class="NLM_term">Nphe</td><td class="NLM_def"><p class="first last">phenylalanine peptoid (N-benzylglycine)</p></td></tr><tr><td class="NLM_term">PDC</td><td class="NLM_def"><p class="first last">pyridinium dichromate</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20881" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20881" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 81 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Exploration of Macrocycles for Drug Discovery--an Underexploited Structural Class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+Exploration+of+Macrocycles+for+Drug+Discovery%2D%2Dan+Underexploited+Structural+Class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0ljbSCPdGOZI3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Exploration%2520of%2520Macrocycles%2520for%2520Drug%2520Discovery--an%2520Underexploited%2520Structural%2520Class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0ljbSCPdGOZI3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Macrocycles as Protein–Protein Interaction Inhibitors</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">1109</span>– <span class="NLM_lpage">1125</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1042%2FBCJ20160619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28298556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=1109-1125&author=P.+G.+Doughertyauthor=Z.+Qianauthor=D.+Pei&title=Macrocycles+as+Protein%E2%80%93Protein+Interaction+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles as protein-protein interaction inhibitors</span></div><div class="casAuthors">Dougherty, Patrick G.; Qian, Ziqing; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1109-1125</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Macrocyclic compds. such as cyclic peptides have emerged as a new and exciting class of drug candidates for inhibition of intracellular protein-protein interactions, which are challenging targets for conventional drug modalities (i.e. small mols. and proteins).  Over the past decade, several complementary technologies have been developed to synthesize macrocycle libraries and screen them for binding to therapeutically relevant targets.  Two different approaches have also been explored to increase the membrane permeability of cyclic peptides.  In this review, we discuss these methods and their applications in the discovery of macrocyclic compds. against protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFYrVReYebVg90H21EOLACvtfcHk0ljbSCPdGOZI3Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D&md5=abd5d7af8383fbe5b553d9501c27ce9c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160619%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DMacrocycles%2520as%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2017%26spage%3D1109%26epage%3D1125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+Are+Great+Cycles%3A+Applications%2C+Opportunities%2C+and+Challenges+of+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0ljc1HF1F2c8aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520Are%2520Great%2520Cycles%253A%2520Applications%252C%2520Opportunities%252C%2520and%2520Challenges%2520of%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdalla, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaw, L. J.</span></span> <span> </span><span class="NLM_article-title">Natural Cyclic Peptides as an Attractive Modality for Therapeutics: A Mini Review</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="NLM_elocation-id">2080</span> <span class="refDoi"> DOI: 10.3390/molecules23082080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.3390%2Fmolecules23082080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+A.+Abdallaauthor=L.+J.+McGaw&title=Natural+Cyclic+Peptides+as+an+Attractive+Modality+for+Therapeutics%3A+A+Mini+Review&doi=10.3390%2Fmolecules23082080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Natural cyclic peptides as an attractive modality for therapeutics: a mini review</span></div><div class="casAuthors">Abdalla, Muna Ali; McGaw, Lyndy J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2080/1-2080/19</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Peptides are important biomols. which facilitate the understanding of complex biol. processes, which in turn could be serendipitous biol. targets for future drugs.  They are classified as a unique therapeutic niche and will play an important role as fascinating agents in the pharmaceutical landscape.  Until now, more than 40 cyclic peptide drugs are currently in the market, and approx. one new cyclopeptide drug enters the market annually on av.  Interestingly, the majority of clin. approved cyclic peptides are derived from natural sources, such as peptide antibiotics and human peptide hormones.  In this report, the importance of cyclic peptides is discussed, and their role in drug discovery as interesting therapeutic biomols. will be highlighted.  Recently isolated naturally occurring cyclic peptides from microorganisms, sponges, and other sources with a wide range of pharmacol. properties are reviewed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopsXVTy7L4WrVg90H21EOLACvtfcHk0ljc1HF1F2c8aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ&md5=4b71e28d331f6131595ac933b6a2f71a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23082080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23082080%26sid%3Dliteratum%253Aachs%26aulast%3DAbdalla%26aufirst%3DM.%2BA.%26aulast%3DMcGaw%26aufirst%3DL.%2BJ.%26atitle%3DNatural%2520Cyclic%2520Peptides%2520as%2520an%2520Attractive%2520Modality%2520for%2520Therapeutics%253A%2520A%2520Mini%2520Review%26jtitle%3DMolecules%26date%3D2018%26doi%3D10.3390%2Fmolecules23082080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span>; <span class="NLM_string-name">Peterson, M. L.</span></span>, Eds.;  <i>Practical Medicinal Chemistry with Macrocycles</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2F9781119092599" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=E.+Marsault&author=M.+L.+Peterson&title=Practical+Medicinal+Chemistry+with+Macrocycles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2F9781119092599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781119092599%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26btitle%3DPractical%2520Medicinal%2520Chemistry%2520with%2520Macrocycles%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahadur, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, P.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Interaction and Quaternary Structure</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1017/S0033583508004708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1017%2FS0033583508004708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=18812015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFKhurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=133-180&author=J.+Janinauthor=R.+P.+Bahadurauthor=P.+Chakrabarti&title=Protein-Protein+Interaction+and+Quaternary+Structure&doi=10.1017%2FS0033583508004708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interaction and quaternary structure</span></div><div class="casAuthors">Janin, Joel; Bahadur, Ranjit P.; Chakrabarti, Pinak</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-180</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Protein-protein recognition plays an essential role in structure and function.  Specific non-covalent interactions stabilize the structure of macromol. assemblies, exemplified in this review by oligomeric proteins and the capsids of icosahedral viruses.  They also allow proteins to form complexes that have a very wide range of stability and lifetimes and are involved in all cellular processes.  We present some of the structure-based computational methods that have been developed to characterize the quaternary structure of oligomeric proteins and other mol. assemblies and analyze the properties of the interfaces between the subunits.  We compare the size, the chem. and amino acid compns. and the at. packing of the subunit interfaces of protein-protein complexes, oligomeric proteins, viral capsids and protein-nucleic acid complexes.  These biol. significant interfaces are generally close-packed, whereas the non-specific interfaces between mols. in protein crystals are loosely packed, an observation that gives a structural basis to specific recognition.  A distinction is made within each interface between a core that contains buried atoms and a solvent accessible rim.  The core and the rim differ in their amino acid compn. and their conservation in evolution, and the distinction helps correlating the structural data with the results of site-directed mutagenesis and in vitro studies of self-assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraAce2lMynI7Vg90H21EOLACvtfcHk0li0xX8WNeocHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFKhurnN&md5=9598e158c9e4240f31a629dc9267406e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1017%2FS0033583508004708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583508004708%26sid%3Dliteratum%253Aachs%26aulast%3DJanin%26aufirst%3DJ.%26aulast%3DBahadur%26aufirst%3DR.%2BP.%26aulast%3DChakrabarti%26aufirst%3DP.%26atitle%3DProtein-Protein%2520Interaction%2520and%2520Quaternary%2520Structure%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2008%26volume%3D41%26spage%3D133%26epage%3D180%26doi%3D10.1017%2FS0033583508004708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chène, P.</span></span> <span> </span><span class="NLM_article-title">Drugs Targeting Protein-Protein Interactions</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">400</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fcmdc.200600004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=16892375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD28XjslehtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=400-411&author=P.+Ch%C3%A8ne&title=Drugs+Targeting+Protein-Protein+Interactions&doi=10.1002%2Fcmdc.200600004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs targeting protein-protein interactions</span></div><div class="casAuthors">Chene, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Most biol. processes involve permanent and nonpermanent interactions between different proteins, and many protein complexes play a key role in various human diseases.  Therefore, mols. that prevent the formation of these protein complexes could be valuable new therapeutic agents to treat these diseases.  Protein interfaces have not evolved to bind low-mol.-wt. mols., as is the case with enzyme catalytic sites.  It is therefore difficult to identify small compds. that inhibit protein-protein interactions.  However, there is considerable diversity in the structure of protein interfaces, some of which may be more attractive than others for medicinal chem.  One of the main challenges in drug discovery is to identify these interfaces and to exploit their properties to make marketable drugs.  Here, the properties of protein interfaces are discussed in light of their use as drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIvsBJWwqDxrVg90H21EOLACvtfcHk0li0xX8WNeocHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjslehtbY%253D&md5=7c7951d7caa7f03351493c6b3bdcd517</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600004%26sid%3Dliteratum%253Aachs%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26atitle%3DDrugs%2520Targeting%2520Protein-Protein%2520Interactions%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D400%26epage%3D411%26doi%3D10.1002%2Fcmdc.200600004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecules, Big Targets: Drug Discovery Faces the Protein-Protein Interaction Challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+Molecules%2C+Big+Targets%3A+Drug+Discovery+Faces+the+Protein-Protein+Interaction+Challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0li0xX8WNeocHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520Molecules%252C%2520Big%2520Targets%253A%2520Drug%2520Discovery%2520Faces%2520the%2520Protein-Protein%2520Interaction%2520Challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Modell, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blosser, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Systematic Targeting of Protein-Protein Interactions Approaches to Targeting Protein-Protein Interactions</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">702</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.tips.2016.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=27267699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVagtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=702-713&author=A.+E.+Modellauthor=S.+L.+Blosserauthor=P.+S.+Arora&title=Systematic+Targeting+of+Protein-Protein+Interactions+Approaches+to+Targeting+Protein-Protein+Interactions&doi=10.1016%2Fj.tips.2016.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Targeting of Protein-Protein Interactions</span></div><div class="casAuthors">Modell, Ashley E.; Blosser, Sarah L.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">702-713</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, protein-protein interactions (PPIs) have gone from being neglected as 'undruggable' to being considered attractive targets for the development of therapeutics.  Recent advances in computational anal., fragment-based screening, and mol. design have revealed promising strategies to address the basic mol. recognition challenge: how to target large protein surfaces with specificity.  Several systematic and complementary workflows have been developed to yield successful inhibitors of PPIs.  Here we review the major contemporary approaches utilized for the discovery of inhibitors and focus on a structure-based workflow, from the selection of a biol. target to design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDD_smUe5_ErVg90H21EOLACvtfcHk0lgKp4MFZzo_Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVagtLo%253D&md5=8b74817b8538b81a3ab0d9b4536fc28e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DModell%26aufirst%3DA.%2BE.%26aulast%3DBlosser%26aufirst%3DS.%2BL.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DSystematic%2520Targeting%2520of%2520Protein-Protein%2520Interactions%2520Approaches%2520to%2520Targeting%2520Protein-Protein%2520Interactions%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2016%26volume%3D37%26spage%3D702%26epage%3D713%26doi%3D10.1016%2Fj.tips.2016.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lgKp4MFZzo_Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Natural Products Chart the Frontier of Oral Bioavailability in the Pursuit of Undruggable Targets</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.cbpa.2017.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28570865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=141-147&author=M.+R.+Naylorauthor=A.+T.+Bockusauthor=M.+J.+Blancoauthor=R.+S.+Lokey&title=Cyclic+Peptide+Natural+Products+Chart+the+Frontier+of+Oral+Bioavailability+in+the+Pursuit+of+Undruggable+Targets&doi=10.1016%2Fj.cbpa.2017.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide natural products chart the frontier of oral bioavailability in the pursuit of undruggable targets</span></div><div class="casAuthors">Naylor, Matthew R.; Bockus, Andrew T.; Blanco, Maria-Jesus; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-147</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As interest in protein-protein interactions and other previously-undruggable targets increases, medicinal chemists are returning to natural products for design inspiration toward mols. that transcend the paradigm of small mol. drugs.  These compds., esp. peptides, often have poor ADME properties and thus require a more nuanced understanding of structure-property relationships to achieve desirable oral bioavailability.  Although there have been few clin. successes in this chem. space to date, recent work has identified opportunities to introduce favorable physicochem. properties to peptidic macrocycles that maintain activity and oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanPrD1D-ePrVg90H21EOLACvtfcHk0lgKp4MFZzo_Mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D&md5=175671ba338608979d38d340866df885</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DNaylor%26aufirst%3DM.%2BR.%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DBlanco%26aufirst%3DM.%2BJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCyclic%2520Peptide%2520Natural%2520Products%2520Chart%2520the%2520Frontier%2520of%2520Oral%2520Bioavailability%2520in%2520the%2520Pursuit%2520of%2520Undruggable%2520Targets%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D141%26epage%3D147%26doi%3D10.1016%2Fj.cbpa.2017.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fosgerau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics: Current Status and Future Directions</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.drudis.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=25450771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=122-128&author=K.+Fosgerauauthor=T.+Hoffmann&title=Peptide+Therapeutics%3A+Current+Status+and+Future+Directions&doi=10.1016%2Fj.drudis.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics: current status and future directions</span></div><div class="casAuthors">Fosgerau, Keld; Hoffmann, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized for being highly selective and efficacious and, at the same time, relatively safe and well tolerated.  Consequently, there is an increased interest in peptides in pharmaceutical research and development (R&D), and approx. 140 peptide therapeutics are currently being evaluated in clin. trials.  Given that the low-hanging fruits in the form of obvious peptide targets have already been picked, it has now become necessary to explore new routes beyond traditional peptide design.  Examples of such approaches are multifunctional and cell penetrating peptides, as well as peptide drug conjugates.  Here, we discuss the current status, strengths, and weaknesses of peptides as medicines and the emerging new opportunities in peptide drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmhpp8GnJ6JLVg90H21EOLACvtfcHk0liGSUc0ggP_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM&md5=f6b74aaa4e3fdb317b091b0450f3618a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DFosgerau%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26atitle%3DPeptide%2520Therapeutics%253A%2520Current%2520Status%2520and%2520Future%2520Directions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.drudis.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colless, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamernig, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design and Synthesis of an Orally Bioavailable Peptide Guided by NMR Amide Temperature Coefficients</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">17504</span>– <span class="NLM_lpage">17509</span>, <span class="refDoi"> DOI: 10.1073/pnas.1417611111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1073%2Fpnas.1417611111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=25416591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=17504-17509&author=C.+K.+Wangauthor=S.+E.+Northfieldauthor=B.+Collessauthor=S.+Chaousisauthor=I.+Hamernigauthor=R.-J.+Lohmanauthor=D.+S.+Nielsenauthor=C.+I.+Schroederauthor=S.+Lirasauthor=D.+A.+Priceauthor=D.+P.+Fairlieauthor=D.+J.+Craik&title=Rational+Design+and+Synthesis+of+an+Orally+Bioavailable+Peptide+Guided+by+NMR+Amide+Temperature+Coefficients&doi=10.1073%2Fpnas.1417611111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</span></div><div class="casAuthors">Wang, Conan K.; Northfield, Susan E.; Colless, Barbara; Chaousis, Stephanie; Hamernig, Ingrid; Lohman, Rink-Jan; Nielsen, Daniel S.; Schroeder, Christina I.; Liras, Spiros; Price, David A.; Fairlie, David P.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17504-17509</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Enhancing the oral bioavailability of peptide drug leads is a major challenge in drug design.  As such, methods to address this challenge are highly sought after by the pharmaceutical industry.  Here, we propose a strategy to identify appropriate amides for N-methylation using temp. coeffs. measured by NMR to identify exposed amides in cyclic peptides.  N-methylation effectively caps these amides, modifying the overall solvation properties of the peptides and making them more membrane permeable.  The approach for identifying sites for N-methylation is a rapid alternative to the elucidation of 3D structures of peptide drug leads, which has been a commonly used structure-guided approach in the past.  Five leucine-rich peptide scaffolds are reported with selectively designed N-methylated derivs.  In vitro membrane permeability was assessed by parallel artificial membrane permeability assay and Caco-2 assay.  The most promising N-methylated peptide was then tested in vivo.  Here we report a novel peptide (15), which displayed an oral bioavailability of 33% in a rat model, thus validating the design approach.  We show that this approach can also be used to explain the notable increase in oral bioavailability of a somatostatin analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1pwUl6qUw7Vg90H21EOLACvtfcHk0liGSUc0ggP_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE&md5=02803e516bf514e3d941769b7bbf3607</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417611111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417611111%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DColless%26aufirst%3DB.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DHamernig%26aufirst%3DI.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DRational%2520Design%2520and%2520Synthesis%2520of%2520an%2520Orally%2520Bioavailable%2520Peptide%2520Guided%2520by%2520NMR%2520Amide%2520Temperature%2520Coefficients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D17504%26epage%3D17509%26doi%3D10.1073%2Fpnas.1417611111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmonds, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Improving on Nature: Making a Cyclic Heptapeptide Orally Bioavailable</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12059</span>– <span class="NLM_lpage">12063</span>, <span class="refDoi"> DOI: 10.1002/anie.201405364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fanie.201405364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFCisrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=12059-12063&author=D.+S.+Nielsenauthor=H.+N.+Hoangauthor=R.+J.+Lohmanauthor=T.+A.+Hillauthor=A.+J.+Luckeauthor=D.+J.+Craikauthor=D.+J.+Edmondsauthor=D.+A.+Griffithauthor=C.+J.+Rotterauthor=R.+B.+Ruggeriauthor=D.+A.+Priceauthor=S.+Lirasauthor=D.+P.+Fairlie&title=Improving+on+Nature%3A+Making+a+Cyclic+Heptapeptide+Orally+Bioavailable&doi=10.1002%2Fanie.201405364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improving on Nature: Making a Cyclic Heptapeptide Orally Bioavailable</span></div><div class="casAuthors">Nielsen, Daniel S.; Hoang, Huy N.; Lohman, Rink-Jan; Hill, Timothy A.; Lucke, Andrew J.; Craik, David J.; Edmonds, David J.; Griffith, David A.; Rotter, Charles J.; Ruggeri, Roger B.; Price, David A.; Liras, Spiros; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">12059-12063</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The use of peptides in medicine is limited by low membrane permeability, metabolic instability, high clearance, and negligible oral bioavailability.  The prediction of oral bioavailability of drugs relies on physicochem. properties that favor passive permeability and oxidative metabolic stability, but these may not be useful for peptides.  Here we investigate effects of heterocyclic constraints, intramol. hydrogen bonds, and side chains on the oral bioavailability of cyclic heptapeptides.  NMR-derived structures, amide H-D exchange rates, and temp.-dependent chem. shifts showed that the combination of rigidification, stronger hydrogen bonds, and solvent shielding by branched side chains enhances the oral bioavailability of cyclic heptapeptides in rats without the need for N-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqefcEjv3-Cb7Vg90H21EOLACvtfcHk0liGSUc0ggP_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFCisrvJ&md5=18d41a670c2ff65b42fe037ac896ab53</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fanie.201405364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201405364%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DLohman%26aufirst%3DR.%2BJ.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DEdmonds%26aufirst%3DD.%2BJ.%26aulast%3DGriffith%26aufirst%3DD.%2BA.%26aulast%3DRotter%26aufirst%3DC.%2BJ.%26aulast%3DRuggeri%26aufirst%3DR.%2BB.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DImproving%2520on%2520Nature%253A%2520Making%2520a%2520Cyclic%2520Heptapeptide%2520Orally%2520Bioavailable%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D12059%26epage%3D12063%26doi%3D10.1002%2Fanie.201405364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Quantifying the Chameleonic Properties of Macrocycles and Other High-Molecular-Weight Drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">712</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.drudis.2016.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=26891978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=712&author=A.+Whittyauthor=M.+Zhongauthor=L.+Viarengoauthor=D.+Beglovauthor=D.+R.+Hallauthor=S.+Vajda&title=Quantifying+the+Chameleonic+Properties+of+Macrocycles+and+Other+High-Molecular-Weight+Drugs&doi=10.1016%2Fj.drudis.2016.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span></div><div class="casAuthors">Whitty, Adrian; Zhong, Mengqi; Viarengo, Lauren; Beglov, Dmitri; Hall, David R.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">712-717</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Key to the pharmaceutical utility of certain macrocyclic drugs is a 'chameleonic' ability to change their conformation to expose polar groups in aq. soln., but bury them when traversing lipid membranes.  Based on anal. of the structures of 20 macrocyclic compds. that are approved oral drugs, we propose that good soly. requires a topol. polar surface area (TPSA, in Å2) of ≥0.2 × mol. wt. (MW).  Meanwhile, good passive membrane permeability requires a mol. (i.e., 3D) PSA in nonpolar environments of ≤140 Å2.  We show that one or other of these limits is almost invariably violated for compds. with MW > 600 Da, suggesting that some degree of chameleonic behavior is required for most high MW oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlN7rCqts5wLVg90H21EOLACvtfcHk0liIHORcYIt2CA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D&md5=024476298f7b72efa3f0a9303bed5900</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DViarengo%26aufirst%3DL.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DQuantifying%2520the%2520Chameleonic%2520Properties%2520of%2520Macrocycles%2520and%2520Other%2520High-Molecular-Weight%2520Drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26spage%3D712%26doi%3D10.1016%2Fj.drudis.2016.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieske, L. H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdélyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Solution Conformations Explain the Chameleonic Behaviour of Macrocyclic Drugs</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5231</span>– <span class="NLM_lpage">5244</span>, <span class="refDoi"> DOI: 10.1002/chem.201905599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fchem.201905599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=32027758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsVWmsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=5231-5244&author=E.+Daneliusauthor=V.+Poongavanamauthor=S.+Peintnerauthor=L.+H.+E.+Wieskeauthor=M.+Erd%C3%A9lyiauthor=J.+Kihlberg&title=Solution+Conformations+Explain+the+Chameleonic+Behaviour+of+Macrocyclic+Drugs&doi=10.1002%2Fchem.201905599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Solution conformations explain the chameleonic behaviour of macrocyclic drugs</span></div><div class="casAuthors">Danelius, Emma; Poongavanam, Vasanthanathan; Peintner, Stefan; Wieske, Lianne H. E.; Erdelyi, Mate; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5231-5244</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">It has been hypothesised that drugs in the chem. space "beyond the rule of 5" (bRo5) must behave as mol. chameleons to combine otherwise conflicting properties, including aq. soly., cell permeability and target binding.  Evidence for this has, however, been limited to the cyclic peptide cyclosporine A.  Herein, we show that the non-peptidic and macrocyclic drugs roxithromycin, telithromycin and spiramycin behave as mol. chameleons, with rifampicin showing a less pronounced behavior.  In particular roxithromycin, telithromycin and spiramycin display a marked, yet limited flexibility and populate significantly less polar and more compact conformational ensembles in an apolar than in a polar environment.  In addn. to balancing of membrane permeability and aq. soly., this flexibility also allows binding to targets that vary in structure between species.  The drugs' passive cell permeability correlates to their 3D polar surface area and corroborate two theor. models for permeability, developed for cyclic peptides.  We conclude that mol. chameleonicity should be incorporated in the design of orally administered drugs in the bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZJsCKr3Rq-rVg90H21EOLACvtfcHk0lhKfES2h3A-YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsVWmsLo%253D&md5=353528feef6aa1130f8a16204a9f40ce</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fchem.201905599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201905599%26sid%3Dliteratum%253Aachs%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DWieske%26aufirst%3DL.%2BH.%2BE.%26aulast%3DErd%25C3%25A9lyi%26aufirst%3DM.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DSolution%2520Conformations%2520Explain%2520the%2520Chameleonic%2520Behaviour%2520of%2520Macrocyclic%2520Drugs%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2020%26volume%3D26%26spage%3D5231%26epage%3D5244%26doi%3D10.1002%2Fchem.201905599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mühlbauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Interconversion Rates between Conformational States as Rationale for the Membrane Permeability of Cyclosporines</span>. <i>ChemPhysChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3309</span>– <span class="NLM_lpage">3314</span>, <span class="refDoi"> DOI: 10.1002/cphc.201700995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fcphc.201700995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28921848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslCmur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=3309-3314&author=J.+Witekauthor=M.+M%C3%BChlbauerauthor=B.+G.+Kellerauthor=M.+Blatterauthor=A.+Meissnerauthor=T.+Wagnerauthor=S.+Riniker&title=Interconversion+Rates+between+Conformational+States+as+Rationale+for+the+Membrane+Permeability+of+Cyclosporines&doi=10.1002%2Fcphc.201700995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Interconversion rates between conformational states as rationale for the membrane permeability of cyclosporins</span></div><div class="casAuthors">Witek, Jagna; Muehlbauer, Max; Keller, Bettina G.; Blatter, Markus; Meissner, Axel; Wagner, Trixie; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">ChemPhysChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3309-3314</span>CODEN:
                <span class="NLM_cas:coden">CPCHFT</span>;
        ISSN:<span class="NLM_cas:issn">1439-4235</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic peptides have regained interest as potential inhibitors of challenging targets but have often a low bioavailability.  The natural product, cyclosporin A (CsA) is the textbook exception.  Despite its size and polar backbone, it is able to passively cross membranes.  This ability is hypothesized to be due to a conformational change from the low-energy conformation in water to a "congruent" conformation that is populated both in water and inside the membrane.  Here, we used a combination of NMR measurements and kinetic models based on mol. dynamics simulations to rationalize the difference in the membrane permeability of cyclosporin E (CsE) and CsA.  The structure of CsE differed only in a backbone methylation, but its membrane permeability was one order of magnitude lower.  The most striking difference was found in the interconversion rates between the conformational states favored in water and in CHCl3, which are up to one order of magnitude slower for CsE compared to CsA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLejkF8vE_e7Vg90H21EOLACvtfcHk0lhKfES2h3A-YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslCmur%252FL&md5=f80741cec728a4852aeb411e64e5c784</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fcphc.201700995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcphc.201700995%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DM%25C3%25BChlbauer%26aufirst%3DM.%26aulast%3DKeller%26aufirst%3DB.%2BG.%26aulast%3DBlatter%26aufirst%3DM.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DInterconversion%2520Rates%2520between%2520Conformational%2520States%2520as%2520Rationale%2520for%2520the%2520Membrane%2520Permeability%2520of%2520Cyclosporines%26jtitle%3DChemPhysChem%26date%3D2017%26volume%3D18%26spage%3D3309%26epage%3D3314%26doi%3D10.1002%2Fcphc.201700995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Räder, A. F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmüller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Improving Oral Bioavailability of Cyclic Peptides by N-Methylation</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2766</span>– <span class="NLM_lpage">2773</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.08.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.bmc.2017.08.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28886995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC1cbmt1Wgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2766-2773&author=A.+F.+B.+R%C3%A4derauthor=F.+Reichartauthor=M.+Weinm%C3%BCllerauthor=H.+Kessler&title=Improving+Oral+Bioavailability+of+Cyclic+Peptides+by+N-Methylation&doi=10.1016%2Fj.bmc.2017.08.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Improving oral bioavailability of cyclic peptides by N-methylation</span></div><div class="casAuthors">Rader Andreas F B; Reichart Florian; Weinmuller Michael; Kessler Horst</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2766-2773</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The renaissance of peptides in pharmaceutical industry results from their importance in many biological functions.  However, low metabolic stability and the lack of oral availability of most peptides is a certain limitation.  Whereas metabolic instability may be often overcome by development of small cyclic peptides containing d-amino acids, the very low oral availability of most peptides is a serious limitation for some medicinal applications.  The situation is complicated because a twofold optimization - biological activity and oral availability - is required to overcome this problem.  Moreover, most simple "rules" for achieving oral availability are not general and are applicable only to limited cases.  Many structural modifications for increasing biological activities and metabolic stabilities of cyclic peptides have been described, of which N-alkylation is probably the most common.  This mini-review focuses on the effects of N-methylation of cyclic peptides in strategies to optimize bioavailabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHNydRSBPhBmHOrQ6qeX-mfW6udTcc2eYZY1vqkiH0K7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbmt1Wgsw%253D%253D&md5=b9e5f76b838ace1fcddd555d5dd3fed2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.08.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.08.031%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25A4der%26aufirst%3DA.%2BF.%2BB.%26aulast%3DReichart%26aufirst%3DF.%26aulast%3DWeinm%25C3%25BCller%26aufirst%3DM.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DImproving%2520Oral%2520Bioavailability%2520of%2520Cyclic%2520Peptides%2520by%2520N-Methylation%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2766%26epage%3D2773%26doi%3D10.1016%2Fj.bmc.2017.08.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzelman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rand, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelev, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">On-Resin N-Methylation of Cyclic Peptides for Discovery of Orally Bioavailable Scaffolds</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">810</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1038/nchembio.664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnchembio.664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=21946276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=810-817&author=T.+R.+Whiteauthor=C.+M.+Renzelmanauthor=A.+C.+Randauthor=T.+Rezaiauthor=C.+M.+McEwenauthor=V.+M.+Gelevauthor=R.+A.+Turnerauthor=R.+G.+Liningtonauthor=S.+S.+F.+Leungauthor=A.+S.+Kalgutkarauthor=J.+N.+Baumanauthor=Y.+Zhangauthor=S.+Lirasauthor=D.+A.+Priceauthor=A.+M.+Mathiowetzauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=On-Resin+N-Methylation+of+Cyclic+Peptides+for+Discovery+of+Orally+Bioavailable+Scaffolds&doi=10.1038%2Fnchembio.664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</span></div><div class="casAuthors">White, Tina R.; Renzelman, Chad M.; Rand, Arthur C.; Rezai, Taha; McEwen, Cayla M.; Gelev, Vladimir M.; Turner, Rushia A.; Linington, Roger G.; Leung, Siegfried S. F.; Kalgutkar, Amit S.; Bauman, Jonathan N.; Zhang, Yizhong; Liras, Spiros; Price, David A.; Mathiowetz, Alan M.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">810-817</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Backbone N-methylation is common among peptide natural products and has a substantial impact on both the phys. properties and the conformational states of cyclic peptides.  However, the specific impact of N-methylation on passive membrane diffusion in cyclic peptides has not been investigated systematically.  Here we report a method for the selective, on-resin N-methylation of cyclic peptides to generate compds. with drug-like membrane permeability and oral bioavailability.  The selectivity and degree of N-methylation of the cyclic peptide was dependent on backbone stereochem., suggesting that conformation dictates the regiochem. of the N-methylation reaction.  The permeabilities of the N-Me variants were corroborated by computational studies on a 1024-member virtual library of N-Me cyclic peptides.  One of the most permeable compds., a cyclic hexapeptide (mol. mass = 755 Da) with three N-Me groups, showed an oral bioavailability of 28% in rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreigsRodBtOLVg90H21EOLACvtfcHk0lj-cM2a0v9TFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL&md5=6f9f01f4ab523bb46cc0198cc8f1c34e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.664%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DT.%2BR.%26aulast%3DRenzelman%26aufirst%3DC.%2BM.%26aulast%3DRand%26aufirst%3DA.%2BC.%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DMcEwen%26aufirst%3DC.%2BM.%26aulast%3DGelev%26aufirst%3DV.%2BM.%26aulast%3DTurner%26aufirst%3DR.%2BA.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DBauman%26aufirst%3DJ.%2BN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DOn-Resin%2520N-Methylation%2520of%2520Cyclic%2520Peptides%2520for%2520Discovery%2520of%2520Orally%2520Bioavailable%2520Scaffolds%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D810%26epage%3D817%26doi%3D10.1038%2Fnchembio.664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran, M. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">12125</span>– <span class="NLM_lpage">12133</span>, <span class="refDoi"> DOI: 10.1021/ja303200d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303200d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=12125-12133&author=J.+G.+Beckauthor=J.+Chatterjeeauthor=B.+Lauferauthor=M.+U.+Kiranauthor=A.+O.+Frankauthor=S.+Neubauerauthor=O.+Ovadiaauthor=S.+Greenbergauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Intestinal+Permeability+of+Cyclic+Peptides%3A+Common+Key+Backbone+Motifs+Identified&doi=10.1021%2Fja303200d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span></div><div class="casAuthors">Beck, Johannes G.; Chatterjee, Jayanta; Laufer, Burkhardt; Kiran, Marelli Udaya; Frank, Andreas O.; Neubauer, Stefanie; Ovadia, Oded; Greenberg, Sarit; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">12125-12133</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Insufficient oral bioavailability is considered as a key limitation for the widespread development of peptides as therapeutics.  While the oral bioavailability of small org. compds. is often estd. from simple rules, similar rules do not apply to peptides, and even the high oral bioavailability that is described for a small no. of peptides is not well understood.  Here we present two highly Caco-2 permeable template structures based on a library of 54 cyclo(-d-Ala-Ala5-) peptides with different N-methylation patterns.  The first all-trans template structure possesses two β-turns of type II along Ala6-d-Ala1 and Ala3-Ala4 and is only found for one peptide with two N-Me groups at d-Ala1 and Ala6 [NMe(1,6)].  The second single-cis template possesses a characteristic cis peptide bond preceding Ala5, which results in type VI β-turn geometry along Ala4-Ala5.  Although the second template structure is found in seven peptides carrying N-Me groups on Ala5, high Caco-2 permeability is only found for a subgroup of two of them [NMe(1,5) and NMe(1,2,4,5)], suggesting that N-methylation of d-Ala1 is a prerequisite for high permeability of the second template structure.  The structural similarity of the second template structure with the orally bioavailable somatostatin analog cyclo(-Pro-Phe-NMe-d-Trp-NMe-Lys-Thr-NMe-Phe-), and the striking resemblance with both β-turns of the orally bioavailable peptide cyclosporine A, suggests that the introduction of bioactive sequences on the highly Caco-2 permeable templates may result in potent orally bioavailable drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4AC8OVfXJMbVg90H21EOLACvtfcHk0lj-cM2a0v9TFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D&md5=ecf4dc5aa62a8ebb7c5cc99e8009f528</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja303200d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303200d%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DJ.%2BG.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DLaufer%26aufirst%3DB.%26aulast%3DKiran%26aufirst%3DM.%2BU.%26aulast%3DFrank%26aufirst%3DA.%2BO.%26aulast%3DNeubauer%26aufirst%3DS.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DGreenberg%26aufirst%3DS.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DIntestinal%2520Permeability%2520of%2520Cyclic%2520Peptides%253A%2520Common%2520Key%2520Backbone%2520Motifs%2520Identified%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D12125%26epage%3D12133%26doi%3D10.1021%2Fja303200d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langenegger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jelinek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Improving Oral Bioavailability of Peptides by Multiple N-Methylation: Somatostatin Analogues</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2599</span>, <span class="refDoi"> DOI: 10.1002/anie.200705797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fanie.200705797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2595-2599&author=E.+Bironauthor=J.+Chatterjeeauthor=O.+Ovadiaauthor=D.+Langeneggerauthor=J.+Brueggenauthor=D.+Hoyerauthor=H.+A.+Schmidauthor=R.+Jelinekauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Improving+Oral+Bioavailability+of+Peptides+by+Multiple+N-Methylation%3A+Somatostatin+Analogues&doi=10.1002%2Fanie.200705797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Improving oral bioavailability of peptides by multiple N-methylation: somatostatin analogues</span></div><div class="casAuthors">Biron, Eric; Chatterjee, Jayanta; Ovadia, Oded; Langenegger, Daniel; Brueggen, Joseph; Hoyer, Daniel; Schmid, Herbert A.; Jelnick, Raz; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2595-2599</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A complete library of the N-methylated somatostatin cyclopeptidic analog Veber-Hirschmann peptide cyclo(-PFwKTF-) has been prepd. with the aim of improving its bioavailability.  Several analogs from the library were found to bind to the somatostatin receptor in the nanomolar range and one of them shows a significant oral bioavailability of 10%.  Conformational anal. shows that N-methylation is allowed at specific positions without affecting the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64OMbjkAtErVg90H21EOLACvtfcHk0liilp8IYbijbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D&md5=42c312786da82941dde6671b1130b61b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.200705797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200705797%26sid%3Dliteratum%253Aachs%26aulast%3DBiron%26aufirst%3DE.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DLangenegger%26aufirst%3DD.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DSchmid%26aufirst%3DH.%2BA.%26aulast%3DJelinek%26aufirst%3DR.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DImproving%2520Oral%2520Bioavailability%2520of%2520Peptides%2520by%2520Multiple%2520N-Methylation%253A%2520Somatostatin%2520Analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D2595%26epage%3D2599%26doi%3D10.1002%2Fanie.200705797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzelman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rand, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcewen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelev, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amit, S.</span></span> <span> </span><span class="NLM_article-title">On-Resin N-Methylation of Cyclic Peptides for Discovery of Orally Bioavailable Scaffolds</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">810</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1038/nchembio.664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnchembio.664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=21946276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=810-817&author=T.+R.+Whiteauthor=C.+M.+Renzelmanauthor=A.+C.+Randauthor=T.+Rezaiauthor=C.+M.+Mcewenauthor=V.+M.+Gelevauthor=R.+A.+Turnerauthor=R.+G.+Liningtonauthor=S.+S.+F.+Leungauthor=S.+Amit&title=On-Resin+N-Methylation+of+Cyclic+Peptides+for+Discovery+of+Orally+Bioavailable+Scaffolds&doi=10.1038%2Fnchembio.664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</span></div><div class="casAuthors">White, Tina R.; Renzelman, Chad M.; Rand, Arthur C.; Rezai, Taha; McEwen, Cayla M.; Gelev, Vladimir M.; Turner, Rushia A.; Linington, Roger G.; Leung, Siegfried S. F.; Kalgutkar, Amit S.; Bauman, Jonathan N.; Zhang, Yizhong; Liras, Spiros; Price, David A.; Mathiowetz, Alan M.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">810-817</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Backbone N-methylation is common among peptide natural products and has a substantial impact on both the phys. properties and the conformational states of cyclic peptides.  However, the specific impact of N-methylation on passive membrane diffusion in cyclic peptides has not been investigated systematically.  Here we report a method for the selective, on-resin N-methylation of cyclic peptides to generate compds. with drug-like membrane permeability and oral bioavailability.  The selectivity and degree of N-methylation of the cyclic peptide was dependent on backbone stereochem., suggesting that conformation dictates the regiochem. of the N-methylation reaction.  The permeabilities of the N-Me variants were corroborated by computational studies on a 1024-member virtual library of N-Me cyclic peptides.  One of the most permeable compds., a cyclic hexapeptide (mol. mass = 755 Da) with three N-Me groups, showed an oral bioavailability of 28% in rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreigsRodBtOLVg90H21EOLACvtfcHk0liilp8IYbijbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL&md5=6f9f01f4ab523bb46cc0198cc8f1c34e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.664%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DT.%2BR.%26aulast%3DRenzelman%26aufirst%3DC.%2BM.%26aulast%3DRand%26aufirst%3DA.%2BC.%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DMcewen%26aufirst%3DC.%2BM.%26aulast%3DGelev%26aufirst%3DV.%2BM.%26aulast%3DTurner%26aufirst%3DR.%2BA.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DAmit%26aufirst%3DS.%26atitle%3DOn-Resin%2520N-Methylation%2520of%2520Cyclic%2520Peptides%2520for%2520Discovery%2520of%2520Orally%2520Bioavailable%2520Scaffolds%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D810%26epage%3D817%26doi%3D10.1038%2Fnchembio.664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkeley, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2928</span>– <span class="NLM_lpage">2931</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=2928-2931&author=J.+Schwochertauthor=R.+Turnerauthor=M.+Thangauthor=R.+F.+Berkeleyauthor=A.+R.+Ponkeyauthor=K.+M.+Rodriguezauthor=S.+S.+F.+Leungauthor=B.+Khunteauthor=G.+Goetzauthor=C.+Limberakisauthor=A.+S.+Kalgutkarauthor=H.+Engauthor=M.+J.+Shapiroauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Peptide+to+Peptoid+Substitutions+Increase+Cell+Permeability+in+Cyclic+Hexapeptides&doi=10.1021%2Facs.orglett.5b01162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span></div><div class="casAuthors">Schwochert, Joshua; Turner, Rushia; Thang, Melissa; Berkeley, Ray F.; Ponkey, Alexandra R.; Rodriguez, Kelsie M.; Leung, Siegfried S. F.; Khunte, Bhagyashree; Goetz, Gilles; Limberakis, Chris; Kalgutkar, Amit S.; Eng, Heather; Shapiro, Michael J.; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2928-2931</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of peptide-to-peptoid substitutions on the passive membrane permeability of an N-methylated cyclic hexapeptide is examd.  In general, substitutions maintained permeability but increased conformational heterogeneity.  Diversification with nonproteinogenic side chains increased permeability up to 3-fold.  Addnl., the conformational impact of peptoid substitutions within a β-turn are explored.  Based on these results, the strategic incorporation of peptoid residues into cyclic peptides can maintain or improve cell permeability, while increasing access to diverse side-chain functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUG1_hYdVn7Vg90H21EOLACvtfcHk0lgw4iK-sWKzFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D&md5=156985175ddcafd34a67e81713934040</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01162%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DThang%26aufirst%3DM.%26aulast%3DBerkeley%26aufirst%3DR.%2BF.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DRodriguez%26aufirst%3DK.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520to%2520Peptoid%2520Substitutions%2520Increase%2520Cell%2520Permeability%2520in%2520Cyclic%2520Hexapeptides%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D2928%26epage%3D2931%26doi%3D10.1021%2Facs.orglett.5b01162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sui, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabenstein, D. L.</span></span> <span> </span><span class="NLM_article-title">Kinetics and Equilibria of Cis/Trans Isomerization of Backbone Amide Bonds in Peptoids</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">12042</span>– <span class="NLM_lpage">12048</span>, <span class="refDoi"> DOI: 10.1021/ja0740925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0740925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFOhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=12042-12048&author=Q.+Suiauthor=D.+Borchardtauthor=D.+L.+Rabenstein&title=Kinetics+and+Equilibria+of+Cis%2FTrans+Isomerization+of+Backbone+Amide+Bonds+in+Peptoids&doi=10.1021%2Fja0740925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics and Equilibria of Cis/Trans Isomerization of Backbone Amide Bonds in Peptoids</span></div><div class="casAuthors">Sui, Qiang; Borchardt, Dan; Rabenstein, Dallas L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12042-12048</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. activities of N-substituted glycine oligomers (peptoids) have been the subject of extensive research.  As compared to peptides, both the cis and trans conformations of the backbone amide bonds of peptoids can be significantly populated.  Thus, peptoids are mixts. of configurational isomers, with the no. of isomers increasing by a factor of 2 with each addnl. monomer residue.  Here, the authors report the results of a study of the kinetics and equil. of cis/trans isomerization of the amide bonds of N-acetylated peptoid monomers, dipeptoids, and tripeptoids by NMR spectroscopy.  Resonance intensities indicate the cis conformation of the backbone amide bonds of the peptoids studied is more populated than is generally the case for the analogous secondary amide bond to proline residues in acyclic peptides.  Rate consts. were measured by inversion-magnetization transfer techniques over a range of temps., and activation parameters were derived from the temp. dependence of the rate consts.  The rate of cis/trans isomerization by rotation around the amide bonds in the peptoids studied is generally slower than that around amide bonds to proline residues and takes place on the NMR inversion-magnetization transfer time scale only by rotation around the amide bond to the C-terminal peptoid residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYolNa9NHLi7Vg90H21EOLACvtfcHk0lgw4iK-sWKzFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFOhs70%253D&md5=f9c41f1342aa801b453a37fbf215acab</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja0740925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0740925%26sid%3Dliteratum%253Aachs%26aulast%3DSui%26aufirst%3DQ.%26aulast%3DBorchardt%26aufirst%3DD.%26aulast%3DRabenstein%26aufirst%3DD.%2BL.%26atitle%3DKinetics%2520and%2520Equilibria%2520of%2520Cis%252FTrans%2520Isomerization%2520of%2520Backbone%2520Amide%2520Bonds%2520in%2520Peptoids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D12042%26epage%3D12048%26doi%3D10.1021%2Fja0740925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Culf, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, R. J.</span></span> <span> </span><span class="NLM_article-title">Solid-Phase Synthesis of N-Substituted Glycine Oligomers (α-Peptoids) and Derivatives</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5282</span>– <span class="NLM_lpage">5335</span>, <span class="refDoi"> DOI: 10.3390/molecules15085282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.3390%2Fmolecules15085282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=20714299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVagt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=5282-5335&author=A.+S.+Culfauthor=R.+J.+Ouellette&title=Solid-Phase+Synthesis+of+N-Substituted+Glycine+Oligomers+%28%CE%B1-Peptoids%29+and+Derivatives&doi=10.3390%2Fmolecules15085282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-phase synthesis of N-substituted glycine oligomers (α-peptoids) and derivatives</span></div><div class="casAuthors">Culf, Adrian S.; Ouellette, Rodney J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5282-5335</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  Peptoids (N-substituted polyglycines and extended peptoids with variant backbone amino-acid monomer units) are oligomeric synthetic polymers that are becoming a valuable mol. tool in the biosciences.  The exploration of peptoids in drug design and evaluation of their secondary structures are of particular interest.  Major advantages of peptoids as research and pharmaceutical tools include the ease and economy of synthesis, highly variable backbone and side-chain chem. possibilities.  In addn., peptoids have been shown to be highly active in biol. systems while resistant to proteolytic decay.  This review with 227 refs. discusses the work done by R. N. Zuckermann et al. from 1992 to 2010 on solid-phase synthesis and biochem. applications of peptoids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv9ZaCWTLgMrVg90H21EOLACvtfcHk0lgw4iK-sWKzFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVagt7fL&md5=907128401ccbbfc77d78e64042b76ac1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fmolecules15085282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules15085282%26sid%3Dliteratum%253Aachs%26aulast%3DCulf%26aufirst%3DA.%2BS.%26aulast%3DOuellette%26aufirst%3DR.%2BJ.%26atitle%3DSolid-Phase%2520Synthesis%2520of%2520N-Substituted%2520Glycine%2520Oligomers%2520%2528%25CE%25B1-Peptoids%2529%2520and%2520Derivatives%26jtitle%3DMolecules%26date%3D2010%26volume%3D15%26spage%3D5282%26epage%3D5335%26doi%3D10.3390%2Fmolecules15085282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Toward the Elucidation of the Mechanism for Passive Membrane Permeability of Cyclic Peptides</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">639</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.4155%2Ffmc-2018-0429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=30920310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFals78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=637-639&author=S.+Riniker&title=Toward+the+Elucidation+of+the+Mechanism+for+Passive+Membrane+Permeability+of+Cyclic+Peptides&doi=10.4155%2Ffmc-2018-0429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the elucidation of the mechanism for passive membrane permeability of cyclic peptides</span></div><div class="casAuthors">Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">637-639</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb9bmRbGqnybVg90H21EOLACvtfcHk0ljZOerS4Rnacg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFals78%253D&md5=36f67a8e4c65f04cc8caec625ba85152</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0429%26sid%3Dliteratum%253Aachs%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DToward%2520the%2520Elucidation%2520of%2520the%2520Mechanism%2520for%2520Passive%2520Membrane%2520Permeability%2520of%2520Cyclic%2520Peptides%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D637%26epage%3D639%26doi%3D10.4155%2Ffmc-2018-0429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dounas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouché, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Rationalization of the Membrane Permeability Differences in a Series of Analogue Cyclic Decapeptides</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">294</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=294-308&author=J.+Witekauthor=S.+Wangauthor=B.+Schroederauthor=R.+Lingwoodauthor=A.+Dounasauthor=H.+J.+Rothauthor=M.+Fouch%C3%A9author=M.+Blatterauthor=O.+Lemkeauthor=B.+Kellerauthor=S.+Riniker&title=Rationalization+of+the+Membrane+Permeability+Differences+in+a+Series+of+Analogue+Cyclic+Decapeptides&doi=10.1021%2Facs.jcim.8b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rationalization of the Membrane Permeability Differences in a Series of Analogue Cyclic Decapeptides</span></div><div class="casAuthors">Witek, Jagna; Wang, Shuzhe; Schroeder, Benjamin; Lingwood, Robin; Dounas, Andreas; Roth, Hans-Jorg; Fouche, Marianne; Blatter, Markus; Lemke, Oliver; Keller, Bettina; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">294-308</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclization and selected backbone N-methylations are found to be often necessary but not sufficient conditions for peptidic drugs to have a good bioavailability.  Thus, the design of cyclic peptides with good passive membrane permeability and good soly. remains a challenge.  The backbone scaffold of a recently published series of cyclic decapeptides with six selected backbone N-methylations was designed to favor the adoption of a closed conformation with β-turns and four transannular hydrogen bonds.  Although this conformation was indeed adopted by the peptides as detd. by NMR measurements, substantial differences in the membrane permeability were obsd.  In this work, we aim to rationalize the impact of discrete side chain modifications on membrane permeability for six of these cyclic decapeptides.  The thermodn. and kinetic properties were investigated using mol. dynamics simulations and Markov state modeling in water and chloroform.  The study highlights the influence that side-chain modifications can have on the backbone conformation.  Peptides with a D-proline in the β-turns were more likely to adopt, even in water, the closed conformation with transannular hydrogen bonds, which facilitates transition through the membrane.  The population of the closed conformation in water was found to correlate pos. with PAMPA log Pe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGnkfImrV0ObVg90H21EOLACvtfcHk0ljZOerS4Rnacg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitbzO&md5=9d1d3e3133734870652778bd67c92694</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00485%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DB.%26aulast%3DLingwood%26aufirst%3DR.%26aulast%3DDounas%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DH.%2BJ.%26aulast%3DFouch%25C3%25A9%26aufirst%3DM.%26aulast%3DBlatter%26aufirst%3DM.%26aulast%3DLemke%26aufirst%3DO.%26aulast%3DKeller%26aufirst%3DB.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DRationalization%2520of%2520the%2520Membrane%2520Permeability%2520Differences%2520in%2520a%2520Series%2520of%2520Analogue%2520Cyclic%2520Decapeptides%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D294%26epage%3D308%26doi%3D10.1021%2Facs.jcim.8b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benakli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaubien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Déziel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, G.</span></span> <span> </span><span class="NLM_article-title">Potent Macrocyclic Antagonists to the Motilin Receptor Presenting Novel Unnatural Amino Acids</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4187</span>– <span class="NLM_lpage">4190</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.05.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.bmcl.2007.05.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=17533127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsVKntrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4187-4190&author=E.+Marsaultauthor=K.+Benakliauthor=S.+Beaubienauthor=C.+Saint-Louisauthor=R.+D%C3%A9zielauthor=G.+Fraser&title=Potent+Macrocyclic+Antagonists+to+the+Motilin+Receptor+Presenting+Novel+Unnatural+Amino+Acids&doi=10.1016%2Fj.bmcl.2007.05.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Potent macrocyclic antagonists to the motilin receptor presenting novel unnatural amino acids</span></div><div class="casAuthors">Marsault, Eric; Benakli, Kamel; Beaubien, Sylvie; Saint-Louis, Carl; Deziel, Robert; Fraser, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4187-4190</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Novel, potent small mol. motilin receptor antagonists are described.  These peptidomimetic macrocycles, e.g., I (X = OH, Cl, CF3; R = CH2NH2, NH2, NHPr-i, NHCONH2, 2-pyrimidinylamino, pyrrolidino), are composed of a tripeptide cyclized backbone-to-backbone with a nonpeptidic tether and bear new unnatural amino acids contg. basic side chains.  I (X = 3-CF3, R = pyrrolidino) was the most potent motilin receptor antagonist (Ki = 18 nM, IC50 = 2 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Hm7_Tmr1A7Vg90H21EOLACvtfcHk0ljZOerS4Rnacg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsVKntrw%253D&md5=8e2ca54b6e6a94e227abdfe883dc2d64</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.05.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.05.043%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DBenakli%26aufirst%3DK.%26aulast%3DBeaubien%26aufirst%3DS.%26aulast%3DSaint-Louis%26aufirst%3DC.%26aulast%3DD%25C3%25A9ziel%26aufirst%3DR.%26aulast%3DFraser%26aufirst%3DG.%26atitle%3DPotent%2520Macrocyclic%2520Antagonists%2520to%2520the%2520Motilin%2520Receptor%2520Presenting%2520Novel%2520Unnatural%2520Amino%2520Acids%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4187%26epage%3D4190%26doi%3D10.1016%2Fj.bmcl.2007.05.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoveyda, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vézina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benakli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaubien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinault, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaseshan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foucher, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beauchemin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhérer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, G. L.</span></span> <span> </span><span class="NLM_article-title">Optimization of the Potency and Pharmacokinetic Properties of a Macrocyclic Ghrelin Receptor Agonist (Part I): Development of Ulimorelin (TZP-101) from Hit to Clinic</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8305</span>– <span class="NLM_lpage">8320</span>, <span class="refDoi"> DOI: 10.1021/jm2007062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC38%252Fms1antg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8305-8320&author=H.+R.+Hoveydaauthor=E.+Marsaultauthor=R.+Gagnonauthor=A.+P.+Mathieuauthor=M.+V%C3%A9zinaauthor=A.+Landryauthor=Z.+Wangauthor=K.+Benakliauthor=S.+Beaubienauthor=C.+Saint-Louisauthor=M.+Brassardauthor=J.+F.+Pinaultauthor=L.+Ouelletauthor=S.+Bhatauthor=M.+Ramaseshanauthor=X.+Pengauthor=L.+Foucherauthor=S.+Beaucheminauthor=P.+Bh%C3%A9rerauthor=D.+F.+Veberauthor=M.+L.+Petersonauthor=G.+L.+Fraser&title=Optimization+of+the+Potency+and+Pharmacokinetic+Properties+of+a+Macrocyclic+Ghrelin+Receptor+Agonist+%28Part+I%29%3A+Development+of+Ulimorelin+%28TZP-101%29+from+Hit+to+Clinic&doi=10.1021%2Fjm2007062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the potency and pharmacokinetic properties of a macrocyclic ghrelin receptor agonist (Part I): Development of ulimorelin (TZP-101) from hit to clinic</span></div><div class="casAuthors">Hoveyda Hamid R; Marsault Eric; Gagnon Rene; Mathieu Axel P; Vezina Martin; Landry Annick; Wang Zhigang; Benakli Kamel; Beaubien Sylvie; Saint-Louis Carl; Brassard Martin; Pinault Jean-Francois; Ouellet Luc; Bhat Shridhar; Ramaseshan Mahesh; Peng Xiaowen; Foucher Laurence; Beauchemin Sophie; Bherer Patrick; Veber Daniel F; Peterson Mark L; Fraser Graeme L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8305-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">High-throughput screening of Tranzyme Pharma's proprietary macrocycle library using the aequorin Ca2+-bioluminescence assay against the human ghrelin receptor (GRLN) led to the discovery of novel agonists against this G-protein coupled receptor.  Early hits such as 1 (Ki=86 nM, EC50=134 nM) though potent in vitro displayed poor pharmacokinetic properties that required optimization.  While such macrocycles are not fully rule-of-five compliant, principally due to their molecular weight and clogP, optimization of their pharmacokinetic properties proved feasible largely through conformational rigidification.  Extensive SAR led to the identification of 2 (Ki=16 nM, EC50=29 nM), also known as ulimorelin or TZP-101, which has progressed to phase III human clinical trials for the treatment of postoperative ileus.  X-ray structure and detailed NMR studies indicated a rigid peptidomimetic portion in 2 that is best defined as a nonideal type-I' β-turn.  Compound 2 is 24% orally bioavailable in both rats and monkeys.  Despite its potency, in vitro and in gastric emptying studies, 2 did not induce growth hormone (GH) release in rats, thus demarcating the GH versus GI pharmacology of GRLN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREoes2wBiC73y-c-E1v-fmfW6udTcc2eY1w-JgrW-uorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252Fms1antg%253D%253D&md5=bdc3b14ed821c61c4fecea50c60f0b07</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm2007062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007062%26sid%3Dliteratum%253Aachs%26aulast%3DHoveyda%26aufirst%3DH.%2BR.%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DGagnon%26aufirst%3DR.%26aulast%3DMathieu%26aufirst%3DA.%2BP.%26aulast%3DV%25C3%25A9zina%26aufirst%3DM.%26aulast%3DLandry%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DBenakli%26aufirst%3DK.%26aulast%3DBeaubien%26aufirst%3DS.%26aulast%3DSaint-Louis%26aufirst%3DC.%26aulast%3DBrassard%26aufirst%3DM.%26aulast%3DPinault%26aufirst%3DJ.%2BF.%26aulast%3DOuellet%26aufirst%3DL.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DRamaseshan%26aufirst%3DM.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DFoucher%26aufirst%3DL.%26aulast%3DBeauchemin%26aufirst%3DS.%26aulast%3DBh%25C3%25A9rer%26aufirst%3DP.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26aulast%3DFraser%26aufirst%3DG.%2BL.%26atitle%3DOptimization%2520of%2520the%2520Potency%2520and%2520Pharmacokinetic%2520Properties%2520of%2520a%2520Macrocyclic%2520Ghrelin%2520Receptor%2520Agonist%2520%2528Part%2520I%2529%253A%2520Development%2520of%2520Ulimorelin%2520%2528TZP-101%2529%2520from%2520Hit%2520to%2520Clinic%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8305%26epage%3D8320%26doi%3D10.1021%2Fjm2007062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Roux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaise, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comeau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giarrusso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collin, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koelling, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göller, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seep, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tshitenge, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittendorf, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, É.</span></span> <span> </span><span class="NLM_article-title">Structure-Permeability Relationship of Semi-Peptidic Macrocycles – Understanding and Optimizing Passive Permeability and Efflux Ratio</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">6774</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=6774&author=A.+Le%0ARouxauthor=E.+Blaiseauthor=P.-L.+Boudreaultauthor=C.+Comeauauthor=A.+Doucetauthor=M.+Giarrussoauthor=M.-P.+Collinauthor=T.+Neubauerauthor=F.+Koellingauthor=A.+H.+G%C3%B6llerauthor=L.+Seepauthor=D.+T.+Tshitengeauthor=M.+Wittwerauthor=M.+Kullmannauthor=A.+Hillischauthor=J.+Mittendorfauthor=%C3%89.+Marsault&title=Structure-Permeability+Relationship+of+Semi-Peptidic+Macrocycles+%E2%80%93+Understanding+and+Optimizing+Passive+Permeability+and+Efflux+Ratio&doi=10.1021%2Facs.jmedchem.0c00013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Permeability Relationship of Semipeptidic Macrocycles-Understanding and Optimizing Passive Permeability and Efflux Ratio</span></div><div class="casAuthors">Le Roux, Antoine; Blaise, Emilie; Boudreault, Pierre-Luc; Comeau, Christian; Doucet, Annie; Giarrusso, Marilena; Collin, Marie-Pierre; Neubauer, Thomas; Kolling, Florian; Goller, Andreas H.; Seep, Lea; Tshitenge, Dieudonne T.; Wittwer, Matthias; Kullmann, Maximilian; Hillisch, Alexander; Mittendorf, Joachim; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6774-6783</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herein report the first thorough anal. of the structure-permeability relationship of semipeptidic macrocycles.  In total, 47 macrocycles were synthesized using a hybrid solid-phase/soln. strategy, and then their passive and cellular permeability was assessed using the parallel artificial membrane permeability assay (PAMPA) and Caco-2 assay, resp.  The results indicate that semipeptidic macrocycles generally possess high passive permeability based on the PAMPA, yet their cellular permeability is governed by efflux, as reported in the Caco-2 assay.  Structural variations led to tractable structure-permeability and structure-efflux relationships, wherein the linker length, stereoinversion, N-methylation, and peptoids site-specifically impact the permeability and efflux.  Extensive NMR, mol. dynamics, and ensemble-based three-dimensional polar surface area (3D-PSA) studies showed that ensemble-based 3D-PSA is a good predictor of passive permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqCamxP_rXRLVg90H21EOLACvtfcHk0ljXIkcmxVk9vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F&md5=2794db50565ab0efbc123d0ec541ff32</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00013%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BRoux%26aufirst%3DA.%26aulast%3DBlaise%26aufirst%3DE.%26aulast%3DBoudreault%26aufirst%3DP.-L.%26aulast%3DComeau%26aufirst%3DC.%26aulast%3DDoucet%26aufirst%3DA.%26aulast%3DGiarrusso%26aufirst%3DM.%26aulast%3DCollin%26aufirst%3DM.-P.%26aulast%3DNeubauer%26aufirst%3DT.%26aulast%3DKoelling%26aufirst%3DF.%26aulast%3DG%25C3%25B6ller%26aufirst%3DA.%2BH.%26aulast%3DSeep%26aufirst%3DL.%26aulast%3DTshitenge%26aufirst%3DD.%2BT.%26aulast%3DWittwer%26aufirst%3DM.%26aulast%3DKullmann%26aufirst%3DM.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DMittendorf%26aufirst%3DJ.%26aulast%3DMarsault%26aufirst%3D%25C3%2589.%26atitle%3DStructure-Permeability%2520Relationship%2520of%2520Semi-Peptidic%2520Macrocycles%2520%25E2%2580%2593%2520Understanding%2520and%2520Optimizing%2520Passive%2520Permeability%2520and%2520Efflux%2520Ratio%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26spage%3D6774%26doi%3D10.1021%2Facs.jmedchem.0c00013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Appavoo, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Conformational Control of Macrocycles by Remote Structural Modification</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">9724</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00742</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00742" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFKisbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=9724&author=S.+D.+Appavooauthor=S.+Huhauthor=D.+B.+Diazauthor=A.+K.+Yudin&title=Conformational+Control+of+Macrocycles+by+Remote+Structural+Modification&doi=10.1021%2Facs.chemrev.8b00742"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Control of Macrocycles by Remote Structural Modification</span></div><div class="casAuthors">Appavoo, Solomon D.; Huh, Sungjoon; Diaz, Diego B.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9724-9752</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The conformational anal. of macrocycles is a complex and challenging problem.  There are many factors that contribute to this complexity.  These include a large no. of degrees of freedom, noncovalent transannular interactions such as hydrogen bonds and hydrophobic interactions, and a range of steric interactions, along with ring strain effects.  To a greater extent than within acyclic mols., such interactions within macrocycles are coupled such that changing one dihedral angle can significantly affect other dihedral angles, further complicating the situation.  However, this coupling of bond rotations and transannular interactions enables the transmission of three-dimensional information from one side of a macrocycle to the other.  Making relatively small structural modifications to a macrocycle can result in local conformational changes that propagate along the ring to affect distal structural features.  The factors that control how such changes can propagate are poorly understood and it is difficult to predict which modifications will result in significant conformational reorganizations of remote regions of a macrocycle.  This review discusses examples where small structural modifications to macrocyclic scaffolds change the conformational preferences of structurally remote regions of the ring.  We will highlight evidence provided for conformational changes triggered by remote substituents and explanations about how these changes might occur in an effort to further understand the factors that control such phenomena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraP983oaKMB7Vg90H21EOLACvtfcHk0ljXIkcmxVk9vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFKisbvF&md5=07fd201af7e700eb02ddd73c0dc33a57</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00742%26sid%3Dliteratum%253Aachs%26aulast%3DAppavoo%26aufirst%3DS.%2BD.%26aulast%3DHuh%26aufirst%3DS.%26aulast%3DDiaz%26aufirst%3DD.%2BB.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DConformational%2520Control%2520of%2520Macrocycles%2520by%2520Remote%2520Structural%2520Modification%26jtitle%3DChem.%2520Rev.%26date%3D2019%26spage%3D9724%26doi%3D10.1021%2Facs.chemrev.8b00742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sok, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Going Out on a Limb: Delineating the Effects of β-Branching, N-Methylation, and Side Chain Size on the Passive Permeability, Solubility, and Flexibility of Sanguinamide A Analogues</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7409</span>– <span class="NLM_lpage">7418</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7409-7418&author=A.+T.+Bockusauthor=J.+A.+Schwochertauthor=C.+R.+Pyeauthor=C.+E.+Townsendauthor=V.+Sokauthor=M.+A.+Bednarekauthor=R.+S.+Lokey&title=Going+Out+on+a+Limb%3A+Delineating+the+Effects+of+%CE%B2-Branching%2C+N-Methylation%2C+and+Side+Chain+Size+on+the+Passive+Permeability%2C+Solubility%2C+and+Flexibility+of+Sanguinamide+A+Analogues&doi=10.1021%2Facs.jmedchem.5b00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Going Out on a Limb: Delineating The Effects of β-Branching, N-Methylation, and Side Chain Size on the Passive Permeability, Solubility, and Flexibility of Sanguinamide A Analogues</span></div><div class="casAuthors">Bockus, Andrew T.; Schwochert, Joshua A.; Pye, Cameron R.; Townsend, Chad E.; Sok, Vong; Bednarek, Maria A.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7409-7418</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is well established that intramol. hydrogen bonding and N-methylation play important roles in the passive permeability of cyclic peptides, but other structural features have been explored less intensively.  Recent studies on the oral bioavailability of the cyclic heptapeptide sanguinamide A have raised the question of whether steric occlusion of polar groups via β-branching is an effective, yet untapped, tool in cyclic peptide permeability optimization.  We report the structures of 17 sanguinamide A analogs designed to test the relative contributions of β-branching, N-methylation, and side chain size to passive membrane permeability and aq. soly.  We demonstrate that β-branching has little effect on permeability compared to the effects of aliph. carbon count and N-methylation of exposed NH groups.  We highlight a new N-methylated analog of sanguinamide A with a Leu substitution at position 2 that exhibits solvent-dependent flexibility and improved permeability over that of the natural product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH9vWd5V2ShLVg90H21EOLACvtfcHk0ljXIkcmxVk9vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7zM&md5=618d20dfdb5543cd67c9f427a780652e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00919%26sid%3Dliteratum%253Aachs%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DSchwochert%26aufirst%3DJ.%2BA.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DTownsend%26aufirst%3DC.%2BE.%26aulast%3DSok%26aufirst%3DV.%26aulast%3DBednarek%26aufirst%3DM.%2BA.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DGoing%2520Out%2520on%2520a%2520Limb%253A%2520Delineating%2520the%2520Effects%2520of%2520%25CE%25B2-Branching%252C%2520N-Methylation%252C%2520and%2520Side%2520Chain%2520Size%2520on%2520the%2520Passive%2520Permeability%252C%2520Solubility%252C%2520and%2520Flexibility%2520of%2520Sanguinamide%2520A%2520Analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7409%26epage%3D7418%26doi%3D10.1021%2Facs.jmedchem.5b00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1021/ja508766b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja508766b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=715-721&author=W.+M.+Hewittauthor=S.+S.+F.+Leungauthor=C.+R.+Pyeauthor=A.+R.+Ponkeyauthor=M.+Bednarekauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Cell-Permeable+Cyclic+Peptides+from+Synthetic+Libraries+Inspired+by+Natural+Products&doi=10.1021%2Fja508766b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span></div><div class="casAuthors">Hewitt, William M.; Leung, Siegfried S. F.; Pye, Cameron R.; Ponkey, Alexandra R.; Bednarek, Maria; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">715-721</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug design efforts are turning to a new generation of therapeutic targets, such as protein-protein interactions (PPIs), that had previously been considered "undruggable" by typical small mols.  There is an emerging view that accessing these targets will require mols. that are larger and more complex than typical small mol. drugs.  Here, the authors present a methodol. for the discovery of geometrically diverse, membrane permeable cyclic peptide scaffolds based on the synthesis and permeability screening of a combinatorial library, followed by deconvolution of membrane-permeable scaffolds to identify cyclic peptides with good to excellent passive cell permeabilities.  The authors use a combination of exptl. and computational approaches to investigate structure-permeability relationships in one of these scaffolds, and uncover structural and conformational factors that govern passive membrane diffusion in a related set of cyclic peptide diastereomers.  Further, the authors investigate the dependency of permeability on side-chain identity of one of these scaffolds through single-point diversifications to show the adaptability of these scaffolds toward development of permeability-biased libraries suitable for bioactivity screens.  Overall, the results demonstrate that many novel, cell permeable scaffolds exist beyond those found in extant natural products, and that such scaffolds can be rapidly identified using a combination of synthesis and deconvolution which can, in principle, be applied to any type of macrocyclic template.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wbqbKaUG9rVg90H21EOLACvtfcHk0lhtChPNU_l0lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF&md5=0d112bf0da119b6a50b55997f64ac214</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja508766b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja508766b%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DBednarek%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCell-Permeable%2520Cyclic%2520Peptides%2520from%2520Synthetic%2520Libraries%2520Inspired%2520by%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D715%26epage%3D721%26doi%3D10.1021%2Fja508766b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Testing the Conformational Hypothesis of Passive Membrane Permeability Using Synthetic Cyclic Peptide Diastereomers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">2510</span>– <span class="NLM_lpage">2511</span>, <span class="refDoi"> DOI: 10.1021/ja0563455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0563455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=2510-2511&author=T.+Rezaiauthor=B.+Yuauthor=G.+L.+Millhauserauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Testing+the+Conformational+Hypothesis+of+Passive+Membrane+Permeability+Using+Synthetic+Cyclic+Peptide+Diastereomers&doi=10.1021%2Fja0563455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Testing the conformational hypothesis of passive membrane permeability using synthetic cyclic peptide diastereomers</span></div><div class="casAuthors">Rezai, Taha; Yu, Bin; Millhauser, Glenn L.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2510-2511</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Little is known about the effect of conformation on passive membrane diffusion rates in small mols.  Evidence suggests that intramol. hydrogen bonding may play a role by reducing the energetic cost of desolvating hydrogen bond donors, esp. amide N-H groups.  We set out to test this hypothesis by investigating the passive membrane diffusion characteristics of a series of cyclic peptide diastereomers based on the sequence cyclo[Leu-Leu-Leu-Leu-Pro-Tyr].  We identified two cyclic hexapeptide diastereomers based on this sequence, whose membrane diffusion rates differed by nearly two log units.  Results of soln. NMR studies and hydrogen/deuterium (H/D) exchange expts. showed that membrane diffusion rates correlated with the degree of intramol. hydrogen bonding and H/D exchange rates.  The most permeable diastereomer, cyclo[D-Leu-D-Leu-Leu-D-Leu-Pro-Tyr], exhibited a passive membrane diffusion rate comparable to that of the orally available drug cyclosporine A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqooDObTy1BXrVg90H21EOLACvtfcHk0lhtChPNU_l0lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D&md5=52e5cac6648efb70e743f294e3f8c4ec</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja0563455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0563455%26sid%3Dliteratum%253Aachs%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DTesting%2520the%2520Conformational%2520Hypothesis%2520of%2520Passive%2520Membrane%2520Permeability%2520Using%2520Synthetic%2520Cyclic%2520Peptide%2520Diastereomers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D2510%26epage%3D2511%26doi%3D10.1021%2Fja0563455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrchan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muncipinto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norinder, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Structural and Conformational Determinants of Macrocycle Cell Permeability</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">1065</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnchembio.2203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=27748751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yktb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1065&author=B.+Overauthor=P.+Matssonauthor=C.+Tyrchanauthor=P.+Arturssonauthor=B.+C.+Doakauthor=M.+A.+Foleyauthor=C.+Hilgendorfauthor=S.+E.+Johnstonauthor=M.+D.+Leeauthor=R.+J.+Lewisauthor=P.+McCarrenauthor=G.+Muncipintoauthor=U.+Norinderauthor=M.+W.+D.+Perryauthor=J.+R.+Duvallauthor=J.+Kihlberg&title=Structural+and+Conformational+Determinants+of+Macrocycle+Cell+Permeability&doi=10.1038%2Fnchembio.2203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and conformational determinants of macrocycle cell permeability</span></div><div class="casAuthors">Over, Bjoern; Matsson, Paer; Tyrchan, Christian; Artursson, Per; Doak, Bradley C.; Foley, Michael A.; Hilgendorf, Constanze; Johnston, Stephen E.; Lee, Maurice D., IV; Lewis, Richard J.; McCarren, Patrick; Muncipinto, Giovanni; Norinder, Ulf; Perry, Matthew W. D.; Duvall, Jeremy R.; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1065-1074</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macrocycles are of increasing interest as chem. probes and drugs for intractable targets like protein-protein interactions, but the determinants of their cell permeability and oral absorption are poorly understood.  To enable rational design of cell-permeable macrocycles, we generated an extensive data set under consistent exptl. conditions for more than 200 non-peptidic, de novo-designed macrocycles from the Broad Institute's diversity-oriented screening collection.  This revealed how specific functional groups, substituents and mol. properties impact cell permeability.  Anal. of energy-minimized structures for stereo- and regioisomeric sets provided fundamental insight into how dynamic, intramol. interactions in the 3D conformations of macrocycles may be linked to physicochem. properties and permeability.  Combined use of quant. structure-permeability modeling and the procedure for conformational anal. now, for the first time, provides chemists with a rational approach to design cell-permeable non-peptidic macrocycles with potential for oral absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-fvl5GK0lVrVg90H21EOLACvtfcHk0lg2YapOrUZiEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yktb3K&md5=a7ff616d7e7e561b96c46a2767325ff6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2203%26sid%3Dliteratum%253Aachs%26aulast%3DOver%26aufirst%3DB.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DTyrchan%26aufirst%3DC.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DHilgendorf%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DS.%2BE.%26aulast%3DLee%26aufirst%3DM.%2BD.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DMcCarren%26aufirst%3DP.%26aulast%3DMuncipinto%26aufirst%3DG.%26aulast%3DNorinder%26aufirst%3DU.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DStructural%2520and%2520Conformational%2520Determinants%2520of%2520Macrocycle%2520Cell%2520Permeability%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26spage%3D1065%26doi%3D10.1038%2Fnchembio.2203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span>; <span class="NLM_string-name">Kerns, E. H.</span></span> <i>Drug-Like Properties</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Di&author=E.+H.+Kerns&title=Drug-Like+Properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26btitle%3DDrug-Like%2520Properties%26pub%3DElsevier%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balazs, A. Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span> <span> </span><span class="NLM_article-title">Free Ligand 1D NMR Conformational Signatures to Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9418</span>– <span class="NLM_lpage">9437</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00716</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00716" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVygtbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9418-9437&author=A.+Y.+S.+Balazsauthor=R.+J.+Carbajoauthor=N.+L.+Daviesauthor=Y.+Dongauthor=A.+W.+Hirdauthor=J.+W.+Johannesauthor=M.+L.+Lambauthor=W.+McCoullauthor=P.+Rauboauthor=G.+R.+Robbauthor=M.+J.+Packerauthor=E.+Chiarparin&title=Free+Ligand+1D+NMR+Conformational+Signatures+to+Enhance+Structure+Based+Drug+Design+of+a+Mcl-1+Inhibitor+%28AZD5991%29+and+Other+Synthetic+Macrocycles&doi=10.1021%2Facs.jmedchem.9b00716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles</span></div><div class="casAuthors">Balazs, Amber Y. S.; Carbajo, Rodrigo J.; Davies, Nichola L.; Dong, Yu; Hird, Alexander W.; Johannes, Jeffrey W.; Lamb, Michelle L.; McCoull, William; Raubo, Piotr; Robb, Graeme R.; Packer, Martin J.; Chiarparin, Elisabetta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9418-9437</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The three-dimensional conformations adopted by a free ligand in soln. impact bioactivity and physicochem. properties.  Soln. 1D NMR spectra inherently contain information on ligand conformational flexibility and three-dimensional shape, as well as on the propensity of the free ligand to fully pre-organize into the bioactive conformation.  Herein we discuss some key learnings, distd. from our experience developing potent and selective synthetic macrocyclic inhibitors, including Mcl-1 clin. candidate AZD5991.  Case studies have been selected from recent oncol. research projects, demonstrating how 1D NMR conformational signatures can complement X-ray protein-ligand structural information to guide medicinal chem. optimization.  Learning to ext. free ligand conformational information from routinely available 1D NMR signatures has proven to be fast enough to guide medicinal chem. decisions within design cycles for compd. optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4XmqKE0MA7bVg90H21EOLACvtfcHk0lg2YapOrUZiEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVygtbnK&md5=95088de9c61c16e8a2366d1d91ab385e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00716%26sid%3Dliteratum%253Aachs%26aulast%3DBalazs%26aufirst%3DA.%2BY.%2BS.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DHird%26aufirst%3DA.%2BW.%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DChiarparin%26aufirst%3DE.%26atitle%3DFree%2520Ligand%25201D%2520NMR%2520Conformational%2520Signatures%2520to%2520Enhance%2520Structure%2520Based%2520Drug%2520Design%2520of%2520a%2520Mcl-1%2520Inhibitor%2520%2528AZD5991%2529%2520and%2520Other%2520Synthetic%2520Macrocycles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9418%26epage%3D9437%26doi%3D10.1021%2Facs.jmedchem.9b00716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orwig, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dix, T. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Cα Methylated Carboxylic Acids: Isosteres of Arginine and Lysine for Use as N-Terminal Capping Residues in Polypeptides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">7007</span>– <span class="NLM_lpage">7009</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2005.08.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.tetlet.2005.08.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvVSnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=7007-7009&author=K.+S.+Orwigauthor=T.+A.+Dix&title=Synthesis+of+C%CE%B1+Methylated+Carboxylic+Acids%3A+Isosteres+of+Arginine+and+Lysine+for+Use+as+N-Terminal+Capping+Residues+in+Polypeptides&doi=10.1016%2Fj.tetlet.2005.08.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Cα methylated carboxylic acids: isosteres of arginine and lysine for use as N-terminal capping residues in polypeptides</span></div><div class="casAuthors">Orwig, Kevin S.; Dix, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">7007-7009</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Replacement of the N-terminal α-amine with the isosteric Me functionality in bioactive peptides can influence various pharmacokinetic parameters, including hydrophobicity and stability.  Cα methylated amino acid analogs are thus of great interest to expand the repertoire of nonnatural synthons available as N-terminal capping' residues for peptide-based drug design.  Several novel amino acid analogs stereoselectively modified in the Cα position with a Me group in place of the α-amine were prepd. using 4-benzyl-2-oxazolidinone as chiral auxiliary and via substitution of bromine with azide in ω-bromo acids, following by cyclization methylation and azide redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD-Qv1Yu0lGrVg90H21EOLACvtfcHk0lg2YapOrUZiEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvVSnu7s%253D&md5=09fedb7b3b0b7e3387649a8c8ee0b04b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2005.08.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2005.08.056%26sid%3Dliteratum%253Aachs%26aulast%3DOrwig%26aufirst%3DK.%2BS.%26aulast%3DDix%26aufirst%3DT.%2BA.%26atitle%3DSynthesis%2520of%2520C%25CE%25B1%2520Methylated%2520Carboxylic%2520Acids%253A%2520Isosteres%2520of%2520Arginine%2520and%2520Lysine%2520for%2520Use%2520as%2520N-Terminal%2520Capping%2520Residues%2520in%2520Polypeptides%26jtitle%3DTetrahedron%2520Lett.%26date%3D2005%26volume%3D46%26spage%3D7007%26epage%3D7009%26doi%3D10.1016%2Fj.tetlet.2005.08.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leogane, O.</span></span> <span> </span><span class="NLM_article-title">Boc-Protected Amines via a Mild and Efficient One-Pot Curtius Rearrangement</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">4107</span>– <span class="NLM_lpage">4110</span>, <span class="refDoi"> DOI: 10.1021/ol051428b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol051428b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2MXosVWlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=4107-4110&author=H.+Lebelauthor=O.+Leogane&title=Boc-Protected+Amines+via+a+Mild+and+Efficient+One-Pot+Curtius+Rearrangement&doi=10.1021%2Fol051428b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Boc-Protected Amines via a Mild and Efficient One-Pot Curtius Rearrangement</span></div><div class="casAuthors">Lebel, Helene; Leogane, Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4107-4110</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The reaction of carboxylic acids RCO2H (R = Me2CHCH2, PhCHEt, 1-adamantyl, etc.) with di-tert-Bu dicarbonate and sodium azide allowed the formation of acyl azide intermediates, which undergo a Curtius rearrangement in the presence of tetrabutylammonium bromide and zinc(II) triflate.  The in situ trapping of the isocyanates obtained led to the desired tert-Bu carbamates RNHCO2CMe3 in high yields at low temp.  These reaction conditions are compatible with a variety of substrates, including malonate derivs., which provide access to protected amino acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphgJudT_mqRLVg90H21EOLACvtfcHk0lioMwgITrI22Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXosVWlsLo%253D&md5=eb5133b9768e1e9338c5b4927f7af816</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fol051428b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol051428b%26sid%3Dliteratum%253Aachs%26aulast%3DLebel%26aufirst%3DH.%26aulast%3DLeogane%26aufirst%3DO.%26atitle%3DBoc-Protected%2520Amines%2520via%2520a%2520Mild%2520and%2520Efficient%2520One-Pot%2520Curtius%2520Rearrangement%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D4107%26epage%3D4110%26doi%3D10.1021%2Fol051428b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKervey, M. A.</span></span> <span> </span><span class="NLM_article-title">Organic Synthesis with α-Diazocarbonyl Compounds</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1091</span>– <span class="NLM_lpage">1160</span>, <span class="refDoi"> DOI: 10.1021/cr00028a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00028a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK2cXktVyhsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1994&pages=1091-1160&author=T.+Yeauthor=M.+A.+McKervey&title=Organic+Synthesis+with+%CE%B1-Diazocarbonyl+Compounds&doi=10.1021%2Fcr00028a010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Organic Synthesis with α-Diazo Carbonyl Compounds</span></div><div class="casAuthors">Ye, Tao; McKervey, M. Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1091-160</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">Review with 786 ref.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqFAjV4JS6YbVg90H21EOLACvtfcHk0lioMwgITrI22Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVyhsbY%253D&md5=ddb663d7c11140798e6aae631887e2b9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fcr00028a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00028a010%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DT.%26aulast%3DMcKervey%26aufirst%3DM.%2BA.%26atitle%3DOrganic%2520Synthesis%2520with%2520%25CE%25B1-Diazocarbonyl%2520Compounds%26jtitle%3DChem.%2520Rev.%26date%3D1994%26volume%3D94%26spage%3D1091%26epage%3D1160%26doi%3D10.1021%2Fcr00028a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petusky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jupp, P.</span></span> <span> </span><span class="NLM_article-title">Combined Application of Parallel Artificial Membrane Permeability Assay and Caco-2 Permeability Assays in Drug Discovery</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1440</span>– <span class="NLM_lpage">1453</span>, <span class="refDoi"> DOI: 10.1002/jps.20075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fjps.20075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=15124203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks12iu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=1440-1453&author=E.+H.+Kernsauthor=L.+Diauthor=S.+Petuskyauthor=M.+Farrisauthor=R.+Leyauthor=P.+Jupp&title=Combined+Application+of+Parallel+Artificial+Membrane+Permeability+Assay+and+Caco-2+Permeability+Assays+in+Drug+Discovery&doi=10.1002%2Fjps.20075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Combined application of parallel artificial membrane permeability assay and Caco-2 permeability assays in drug discovery</span></div><div class="casAuthors">Kerns, Edward H.; Di, Li; Petusky, Susan; Farris, Michele; Ley, Rob; Jupp, Phil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1440-1453</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Data from permeability profiling using the parallel artificial membrane permeability assay (PAMPA) and cell monolayer (Caco-2 and MDR1-MDCKII) methods were compared for two published compd. sets and one inhouse set.  A majority of compds. in each set correlated (R2 = 0.76-0.92), indicating the predominance of passive diffusion in the permeation of these compds.  Compds. that did not correlate grouped into two subsets.  One subset had higher PAMPA permeability than cell monolayer permeability and consisted of compds. that are subject to secretory mechanisms: efflux or reduced passive diffusion of bases under Caco-2 when run under a pH gradient.  The other subset had higher cell monolayer permeability than PAMPA permeability and consisted of compds. that are subject to absorptive mechanisms: paracellular, active transport, or increased passive diffusion of acids under Caco-2 when run under a pH gradient.  Given the characteristics of the two methods, these studies suggest how PAMPA and Caco-2 can be synergistically applied for efficient and rapid investigation of permeation mechanisms in drug discovery.  During early discovery, all compds. can be rapidly screened using PAMPA at low pH and neutral pH to assess passive diffusion permeability to indicate potential for gastrointestinal and cell assay permeation.  During intermediate discovery, selected compds. can be addnl. assayed by apical-to-basolateral Caco-2, which, in combination with PAMPA data, indicates susceptibility to addnl. permeation mechanisms (secretory and absorptive).  During mid-to-late discovery, selected candidates can be examd. in detail via multiple directional Caco-2 expts. and with transporter inhibitors for complete characterization of permeation mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG1M-v5UG2RLVg90H21EOLACvtfcHk0lioMwgITrI22Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks12iu7w%253D&md5=428596e93f30763e94ced9341d23a02e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjps.20075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.20075%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DPetusky%26aufirst%3DS.%26aulast%3DFarris%26aufirst%3DM.%26aulast%3DLey%26aufirst%3DR.%26aulast%3DJupp%26aufirst%3DP.%26atitle%3DCombined%2520Application%2520of%2520Parallel%2520Artificial%2520Membrane%2520Permeability%2520Assay%2520and%2520Caco-2%2520Permeability%2520Assays%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D1440%26epage%3D1453%26doi%3D10.1002%2Fjps.20075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel-Andreasen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fristrup, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">Cis-Trans Amide Bond Rotamers in β-Peptoids and Peptoids: Evaluation of Stereoelectronic Effects in Backbone and Side Chains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">2835</span>– <span class="NLM_lpage">2844</span>, <span class="refDoi"> DOI: 10.1021/ja312532x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja312532x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtleltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=2835-2844&author=J.+S.+Laursenauthor=J.+Engel-Andreasenauthor=P.+Fristrupauthor=P.+Harrisauthor=C.+A.+Olsen&title=Cis-Trans+Amide+Bond+Rotamers+in+%CE%B2-Peptoids+and+Peptoids%3A+Evaluation+of+Stereoelectronic+Effects+in+Backbone+and+Side+Chains&doi=10.1021%2Fja312532x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Cis-Trans Amide Bond Rotamers in β-Peptoids and Peptoids: Evaluation of Stereoelectronic Effects in Backbone and Side Chains</span></div><div class="casAuthors">Laursen, Jonas S.; Engel-Andreasen, Jens; Fristrup, Peter; Harris, Pernille; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2835-2844</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nonnatural peptide analogs have significant potential for the development of new materials and pharmacol. active ligands.  One such architecture, the β-peptoids (N-alkyl-β-alanines), found use in a variety of biol. active compds. but was sparsely studied with respect to folding propensity.  Thus, the authors here report a study of the effect of structural variations on the cis-trans amide bond rotamer equil. in a selection of monomer model systems.  In addn. to various side chain effects, which correlated well with previous studies of α-peptoids, the authors present the synthesis and study of cis-trans isomerism in the first examples of peptoids and β-peptoids contg. thioamide bonds as well as trifluoroacetylated peptoids and β-peptoids.  These systems revealed an increase in the preference for cis-amides as compared to their parent compds. and thus provide novel strategies for affecting the folding of peptoid constructs.  By using NMR spectroscopy, x-ray crystallog. anal., and d. functional theory calcns., the authors present evidence for the presence of thioamide-arom. interactions through Csp2 -H···Samide hydrogen bonding, which stabilize certain peptoid conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqXidNft09rVg90H21EOLACvtfcHk0ljcA2LMdTNakQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtleltL0%253D&md5=5af65208996e55a516a95461a1c728db</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja312532x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja312532x%26sid%3Dliteratum%253Aachs%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DEngel-Andreasen%26aufirst%3DJ.%26aulast%3DFristrup%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DCis-Trans%2520Amide%2520Bond%2520Rotamers%2520in%2520%25CE%25B2-Peptoids%2520and%2520Peptoids%253A%2520Evaluation%2520of%2520Stereoelectronic%2520Effects%2520in%2520Backbone%2520and%2520Side%2520Chains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D2835%26epage%3D2844%26doi%3D10.1021%2Fja312532x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel-Andreasen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wich, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">Effects of Thionation and Fluorination on Cis-Trans Isomerization in Tertiary Amides: An Investigation of N-Alkylglycine (Peptoid) Rotamers</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">5415</span>– <span class="NLM_lpage">5427</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.5b00048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.5b00048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjslelsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2015&pages=5415-5427&author=J.+Engel-Andreasenauthor=K.+Wichauthor=J.+S.+Laursenauthor=P.+Harrisauthor=C.+A.+Olsen&title=Effects+of+Thionation+and+Fluorination+on+Cis-Trans+Isomerization+in+Tertiary+Amides%3A+An+Investigation+of+N-Alkylglycine+%28Peptoid%29+Rotamers&doi=10.1021%2Facs.joc.5b00048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of thionation and fluorination on cis-trans isomerization in tertiary amides: An investigation of N-alkylglycine (peptoid) rotamers</span></div><div class="casAuthors">Engel-Andreasen, Jens; Wich, Kathrine; Laursen, Jonas S.; Harris, Pernille; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5415-5427</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptoids constitute a class of peptidomimetics with potential as protease resistant, biol. active ligands.  To harness the full potential of such compds., however, detailed predictive insight into their propensity to adopt well-defined secondary structures is highly desirable.  In this work we present an investigation of the effects of thioamides and/or fluorides in peptoid monomer model systems using chem. synthesis, NMR spectroscopy, and X-ray crystallog.  We find that the steric environment surrounding the tertiary amide bonds is the key promoter of conformational preference, and X-ray crystallog. interrogation of our model systems did not suggest the presence of stabilizing n → π* interactions unless the carbonyls were altered electronically by α-halogenation or thioamide formation.  In addn. to the function as an investigative tool, these two types of modification may thus be utilized as stabilizers of secondary structure in future oligomer designs, such as the cis-amide-based polypeptoid helixes that resemble the polyproline type-I helix.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF7WiuezU4sLVg90H21EOLACvtfcHk0ljcA2LMdTNakQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjslelsrg%253D&md5=28cddf791106a3f97213164b89bddc2f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.5b00048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.5b00048%26sid%3Dliteratum%253Aachs%26aulast%3DEngel-Andreasen%26aufirst%3DJ.%26aulast%3DWich%26aufirst%3DK.%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DEffects%2520of%2520Thionation%2520and%2520Fluorination%2520on%2520Cis-Trans%2520Isomerization%2520in%2520Tertiary%2520Amides%253A%2520An%2520Investigation%2520of%2520N-Alkylglycine%2520%2528Peptoid%2529%2520Rotamers%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D5415%26epage%3D5427%26doi%3D10.1021%2Facs.joc.5b00048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebastiano, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backlund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Impact of Dynamically Exposed Polarity on Permeability and Solubility of Chameleonic Drugs Beyond the Rule of 5</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">4189</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=4189&author=M.+R.+Sebastianoauthor=B.+C.+Doakauthor=M.+Backlundauthor=V.+Poongavanamauthor=B.+Overauthor=G.+Ermondiauthor=G.+Caronauthor=J.+Kihlberg&title=Impact+of+Dynamically+Exposed+Polarity+on+Permeability+and+Solubility+of+Chameleonic+Drugs+Beyond+the+Rule+of+5&doi=10.1021%2Facs.jmedchem.8b00347"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00347%26sid%3Dliteratum%253Aachs%26aulast%3DSebastiano%26aufirst%3DM.%2BR.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DBacklund%26aufirst%3DM.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DImpact%2520of%2520Dynamically%2520Exposed%2520Polarity%2520on%2520Permeability%2520and%2520Solubility%2520of%2520Chameleonic%2520Drugs%2520Beyond%2520the%2520Rule%2520of%25205%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26spage%3D4189%26doi%3D10.1021%2Facs.jmedchem.8b00347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakenhut, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitlock, G. A.</span></span> <span> </span><span class="NLM_article-title">Intramolecular Hydrogen Bonding to Improve Membrane Permeability and Absorption in beyond Rule of Five Chemical Space</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1039/c1md00093d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1039%2Fc1md00093d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=669-674&author=A.+Alexauthor=D.+S.+Millanauthor=M.+Perezauthor=F.+Wakenhutauthor=G.+A.+Whitlock&title=Intramolecular+Hydrogen+Bonding+to+Improve+Membrane+Permeability+and+Absorption+in+beyond+Rule+of+Five+Chemical+Space&doi=10.1039%2Fc1md00093d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Intramolecular hydrogen bonding to improve membrane permeability and absorption in beyond rule of five chemical space</span></div><div class="casAuthors">Alex, Alexander; Millan, David S.; Perez, Manuel; Wakenhut, Florian; Whitlock, Gavin A.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">669-674</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Utilizing 'beyond rule of five' chem. space is becoming increasingly important in drug design, but is usually at odds with good oral absorption.  The formation of intramol. hydrogen bonds in drug mols. is hypothesised to shield polarity facilitating improved membrane permeability and intestinal absorption.  NMR based evidence for intramol. hydrogen bonding in several 'beyond rule of five' oral drugs is described.  Furthermore, the propensity for these drugs to form intramol. hydrogen bonds could be predicted for through modeling the lowest energy conformation in the gas phase.  The modulation of apparent lipophilicity through intramol. hydrogen bonding in these mols. is supported by intrinsic cell permeability and intestinal absorption data in rat and human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp93l2KJMdB2bVg90H21EOLACvtfcHk0lgz9nxrYioOGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D&md5=3effbdd29832ac7b57eabbc26c66217b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2Fc1md00093d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00093d%26sid%3Dliteratum%253Aachs%26aulast%3DAlex%26aufirst%3DA.%26aulast%3DMillan%26aufirst%3DD.%2BS.%26aulast%3DPerez%26aufirst%3DM.%26aulast%3DWakenhut%26aufirst%3DF.%26aulast%3DWhitlock%26aufirst%3DG.%2BA.%26atitle%3DIntramolecular%2520Hydrogen%2520Bonding%2520to%2520Improve%2520Membrane%2520Permeability%2520and%2520Absorption%2520in%2520beyond%2520Rule%2520of%2520Five%2520Chemical%2520Space%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D669%26epage%3D674%26doi%3D10.1039%2Fc1md00093d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindhagen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjö, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiesser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Toward the Design of Molecular Chameleons: Flexible Shielding of an Amide Bond Enhances Macrocycle Cell Permeability</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5737</span>– <span class="NLM_lpage">5742</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.8b02447</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.8b02447" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yksr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=5737-5742&author=M.+Tyagiauthor=V.+Poongavanamauthor=M.+Lindhagenauthor=A.+Pettersenauthor=P.+Sj%C3%B6author=S.+Schiesserauthor=J.+Kihlberg&title=Toward+the+Design+of+Molecular+Chameleons%3A+Flexible+Shielding+of+an+Amide+Bond+Enhances+Macrocycle+Cell+Permeability&doi=10.1021%2Facs.orglett.8b02447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Design of Molecular Chameleons: Flexible Shielding of an Amide Bond Enhances Macrocycle Cell Permeability</span></div><div class="casAuthors">Tyagi, Mohit; Poongavanam, Vasanthanathan; Lindhagen, Marika; Pettersen, Anna; Sjoe, Peter; Schiesser, Stefan; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5737-5742</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of macrocycles inspired by natural products were synthesized to investigate how side-chains may shield amide bonds and influence cell permeability.  NMR spectroscopy and x-ray crystallog. revealed that the Ph group of phenylalanine, but not the side-chains of homologous or aliph. amino acids, shields the adjacent amide bond through an intramol. NH-π interaction.  This resulted in increased cell permeability, suggesting that NH-π interactions may be used in the design of mol. chameleons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1E2xKB5kDZrVg90H21EOLACvtfcHk0lgz9nxrYioOGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yksr7O&md5=24182bbd5b612d25ca9fd9cdbb3f9bc4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.8b02447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.8b02447%26sid%3Dliteratum%253Aachs%26aulast%3DTyagi%26aufirst%3DM.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DLindhagen%26aufirst%3DM.%26aulast%3DPettersen%26aufirst%3DA.%26aulast%3DSj%25C3%25B6%26aufirst%3DP.%26aulast%3DSchiesser%26aufirst%3DS.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DToward%2520the%2520Design%2520of%2520Molecular%2520Chameleons%253A%2520Flexible%2520Shielding%2520of%2520an%2520Amide%2520Bond%2520Enhances%2520Macrocycle%2520Cell%2520Permeability%26jtitle%3DOrg.%2520Lett.%26date%3D2018%26volume%3D20%26spage%3D5737%26epage%3D5742%26doi%3D10.1021%2Facs.orglett.8b02447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, M. B.</span></span> <span> </span><span class="NLM_article-title">Biometrika Trust An Analysis of Variance Test for Normality (Complete Samples) Published by: Oxford University Press on Behalf of Biometrika Trust Stable</span>. <i>Biometrika</i> <span class="NLM_year" style="font-weight: bold;">1965</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">611</span>, <span class="refDoi"> DOI: 10.1093/biomet/52.3-4.591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1093%2Fbiomet%2F52.3-4.591" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1965&pages=591-611&author=S.+S.+Shapiroauthor=M.+B.+Wilk&title=Biometrika+Trust+An+Analysis+of+Variance+Test+for+Normality+%28Complete+Samples%29+Published+by%3A+Oxford+University+Press+on+Behalf+of+Biometrika+Trust+Stable&doi=10.1093%2Fbiomet%2F52.3-4.591"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fbiomet%2F52.3-4.591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbiomet%252F52.3-4.591%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DS.%2BS.%26aulast%3DWilk%26aufirst%3DM.%2BB.%26atitle%3DBiometrika%2520Trust%2520An%2520Analysis%2520of%2520Variance%2520Test%2520for%2520Normality%2520%2528Complete%2520Samples%2529%2520Published%2520by%253A%2520Oxford%2520University%2520Press%2520on%2520Behalf%2520of%2520Biometrika%2520Trust%2520Stable%26jtitle%3DBiometrika%26date%3D1965%26volume%3D52%26spage%3D591%26epage%3D611%26doi%3D10.1093%2Fbiomet%2F52.3-4.591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. L.</span></span> <span> </span><span class="NLM_article-title">Breakthroughs in Statistics, Volume III</span>. <i>Technometrics</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">165</span>, <span class="refDoi"> DOI: 10.1080/00401706.1998.10485220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1080%2F00401706.1998.10485220" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1998&pages=165&author=S.+Kotzauthor=N.+L.+Johnson&title=Breakthroughs+in+Statistics%2C+Volume+III&doi=10.1080%2F00401706.1998.10485220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1080%2F00401706.1998.10485220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00401706.1998.10485220%26sid%3Dliteratum%253Aachs%26aulast%3DKotz%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DN.%2BL.%26atitle%3DBreakthroughs%2520in%2520Statistics%252C%2520Volume%2520III%26jtitle%3DTechnometrics%26date%3D1998%26volume%3D40%26spage%3D165%26doi%3D10.1080%2F00401706.1998.10485220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vorherr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desrayaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, M.</span></span> <span> </span><span class="NLM_article-title">Modulation of Oral Bioavailability and Metabolism for Closely Related Cyclic Hexapeptides</span>. <i>Int. J. Pept. Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1007/s10989-017-9590-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1007%2Fs10989-017-9590-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=29527142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlt1Srsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=35-48&author=T.+Vorherrauthor=I.+Lewisauthor=J.+Berghausenauthor=S.+Desrayaudauthor=M.+Schaefer&title=Modulation+of+Oral+Bioavailability+and+Metabolism+for+Closely+Related+Cyclic+Hexapeptides&doi=10.1007%2Fs10989-017-9590-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of Oral Bioavailability and Metabolism for Closely Related Cyclic Hexapeptides</span></div><div class="casAuthors">Vorherr, Thomas; Lewis, Ian; Berghausen, Joerg; Desrayaud, Sandrine; Schaefer, Michael</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide Research and Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-48</span>CODEN:
                <span class="NLM_cas:coden">IJPRFC</span>;
        ISSN:<span class="NLM_cas:issn">1573-3149</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: Recently, a variety of studies concerned with the permeability and oral bioavailability of cyclic peptides have been reported.  In particular, strategies aiming at modifying peptides to maintain or to enhance soly. while enabling permeability constitute a significant challenge, but are of high interest to ensure a smooth drug discovery process.  Current methodologies include N-methylation, matching of hydrogen bonding acceptors and donors across the macrocycle, and addnl. masking of polarity.  In this study, we investigate further the pivotal effects of shielding on permeability and studied the metab. of the corresponding peptides in more detail by comparing peptide concns. in the portal vs. the jugular vein in rats.  Interestingly, minor changes in one particular side chain impacts both permeability and liver metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZnrDMY7yZHLVg90H21EOLACvtfcHk0lgHC1d5mzYrww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlt1Srsrw%253D&md5=2faed8ef9cf1de91b6a5fd4c73c44c5d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2Fs10989-017-9590-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10989-017-9590-8%26sid%3Dliteratum%253Aachs%26aulast%3DVorherr%26aufirst%3DT.%26aulast%3DLewis%26aufirst%3DI.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDesrayaud%26aufirst%3DS.%26aulast%3DSchaefer%26aufirst%3DM.%26atitle%3DModulation%2520of%2520Oral%2520Bioavailability%2520and%2520Metabolism%2520for%2520Closely%2520Related%2520Cyclic%2520Hexapeptides%26jtitle%3DInt.%2520J.%2520Pept.%2520Res.%2520Ther.%26date%3D2018%26volume%3D24%26spage%3D35%26epage%3D48%26doi%3D10.1007%2Fs10989-017-9590-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peraro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Emerging Methods and Design Principles for Cell-Penetrant Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">11868</span>– <span class="NLM_lpage">11881</span>, <span class="refDoi"> DOI: 10.1002/anie.201801361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fanie.201801361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeqtrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=11868-11881&author=L.+Peraroauthor=J.+A.+Kritzer&title=Emerging+Methods+and+Design+Principles+for+Cell-Penetrant+Peptides&doi=10.1002%2Fanie.201801361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Methods and Design Principles for Cell-Penetrant Peptides</span></div><div class="casAuthors">Peraro, Leila; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">11868-11881</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review on the diverse chem. approaches that have yielded cell-penetrant peptides, and the three distinct strategies identified: masking backbone amides, guanidinium group patterning, and amphipathic patterning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGQbUZr5CdxbVg90H21EOLACvtfcHk0lgHC1d5mzYrww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeqtrjK&md5=f627595f28e69841d51bbe535388a579</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fanie.201801361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201801361%26sid%3Dliteratum%253Aachs%26aulast%3DPeraro%26aufirst%3DL.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DEmerging%2520Methods%2520and%2520Design%2520Principles%2520for%2520Cell-Penetrant%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D11868%26epage%3D11881%26doi%3D10.1002%2Fanie.201801361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger, L.</span></span> <i>The PyMOL Molecular Graphics System</i>, version 1.8; <span class="NLM_publisher-name">Schrödinger, LLC</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Schr%C3%B6dinger&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lgHC1d5mzYrww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avdeef, A.</span></span> <i>Absorption and Drug Development</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2F9781118286067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=A.+Avdeef&title=Absorption+and+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F9781118286067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118286067%26sid%3Dliteratum%253Aachs%26aulast%3DAvdeef%26aufirst%3DA.%26btitle%3DAbsorption%2520and%2520Drug%2520Development%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raub, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Human Colon Carcinoma Cell Line (Caco-2) as a Model System for Intestinal Epithelial Permeability</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1016/S0016-5085(89)80072-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2FS0016-5085%2889%2980072-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=2914637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1989&pages=736-749&author=I.+J.+Hidalgoauthor=T.+J.+Raubauthor=R.+T.+Borchardt&title=Characterization+of+the+Human+Colon+Carcinoma+Cell+Line+%28Caco-2%29+as+a+Model+System+for+Intestinal+Epithelial+Permeability&doi=10.1016%2FS0016-5085%2889%2980072-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the human colon carcinoma cell line (Caco-2) as a model system for intestinal epithelial permeability</span></div><div class="casAuthors">Hidalgo I J; Raub T J; Borchardt R T</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">736-49</span>
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    </div><div class="casAbstract">Caco-2 cells develop morphologic characteristics of normal enterocytes when grown on plastic dishes or nitrocellulose filters.  The purpose of this study was to determine whether Caco-2 cells undergo similar differentiation when grown on Transwell polycarbonate membranes, and to study the suitability of Caco-2 monolayers as an intestinal epithelial transport model system.  Transepithelial electrical resistance values after confluence were 173.5 omega.cm2 and remained unchanged through day 17.  Permeabilities to the water-soluble fluid-phase markers that do not permeate the membrane, Lucifer yellow CH, [14C]inulin, [14C]polyethylene glycol, and [3H] dextran were less than 0.25% of the administered amount per hour after day 10.  Qualitative evaluation of uptake and permeability to horseradish peroxidase confirmed the similarity in uptake and barrier properties between this cell system and the small intestinal epithelial layer.  We conclude that Caco-2 cells grown on collagen-coated polycarbonate membranes should represent a valuable transport model system for the small intestinal epithelium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOn97989rzEIqdXsLcGTEUfW6udTcc2ebbHh1VvSI2w7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D&md5=59ffccc1dd4dccb01400c05df95ac1ff</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2FS0016-5085%2889%2980072-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0016-5085%252889%252980072-1%26sid%3Dliteratum%253Aachs%26aulast%3DHidalgo%26aufirst%3DI.%2BJ.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DCharacterization%2520of%2520the%2520Human%2520Colon%2520Carcinoma%2520Cell%2520Line%2520%2528Caco-2%2529%2520as%2520a%2520Model%2520System%2520for%2520Intestinal%2520Epithelial%2520Permeability%26jtitle%3DGastroenterology%26date%3D1989%26volume%3D96%26spage%3D736%26epage%3D749%26doi%3D10.1016%2FS0016-5085%2889%2980072-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubatsch, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ragnarsson, E. G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span> <span> </span><span class="NLM_article-title">Determination of Drug Permeability and Prediction of Drug Absorption in Caco-2 Monolayers</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2111</span>– <span class="NLM_lpage">2119</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnprot.2007.303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=17853866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2111-2119&author=I.+Hubatschauthor=E.+G.+E.+Ragnarssonauthor=P.+Artursson&title=Determination+of+Drug+Permeability+and+Prediction+of+Drug+Absorption+in+Caco-2+Monolayers&doi=10.1038%2Fnprot.2007.303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of drug permeability and prediction of drug absorption in Caco-2 monolayers</span></div><div class="casAuthors">Hubatsch, Ina; Ragnarsson, Eva G. E.; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2111-2119</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Permeability coeffs. across monolayers of the human colon carcinoma cell line Caco-2, cultured on permeable supports, are commonly used to predict the absorption of orally administered drugs and other xenobiotics.  This protocol describes our method for the cultivation, characterization and detn. of permeability coeffs. of xenobiotics (which are, typically, drug-like compds.) in the Caco-2 model.  A few modifications that have been introduced over the years are incorporated in the protocol.  The method can be used to trace the permeability of a test compd. in two directions, from the apical to the basolateral side or vice versa, and both passive and active transport processes can be studied.  The permeability assay can be completed within one working day, provided that the Caco-2 monolayers have been cultured and differentiated on the permeable supports 3 wk in advance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX7zGXziU_VLVg90H21EOLACvtfcHk0lg31gLyP8s4wA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb%252FN&md5=85e2f95420919ee61c2d691c305bc1ef</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.303%26sid%3Dliteratum%253Aachs%26aulast%3DHubatsch%26aufirst%3DI.%26aulast%3DRagnarsson%26aufirst%3DE.%2BG.%2BE.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DDetermination%2520of%2520Drug%2520Permeability%2520and%2520Prediction%2520of%2520Drug%2520Absorption%2520in%2520Caco-2%2520Monolayers%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2111%26epage%3D2119%26doi%3D10.1038%2Fnprot.2007.303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Learning from the Data Set and the Analysis of Failures</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2919</span>– <span class="NLM_lpage">2936</span>, <span class="refDoi"> DOI: 10.1021/ci300314k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300314k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2919-2936&author=P.+C.+D.+Hawkinsauthor=A.+Nicholls&title=Conformer+Generation+with+OMEGA%3A+Learning+from+the+Data+Set+and+the+Analysis+of+Failures&doi=10.1021%2Fci300314k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Learning from the Data Set and the Analysis of Failures</span></div><div class="casAuthors">Hawkins, Paul C. D.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2919-2936</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently published a high quality validation set for testing conformer generators, consisting of structures from both the PDB and the CSD and tested the performance of the authors' conformer generator, OMEGA, on these sets.  In the present publication, we focus on understanding the suitability of those data sets for validation and identifying and learning from OMEGA's failures.  We compare, for the first time we are aware of, the coverage of the applicable property spaces between the validation data sets we used and the parent compd. sets to det. if our data sets adequately sample these property spaces.  We also introduce the concept of torsion fingerprinting and compare this method of dissimilation to the more traditional graph-centric diversification methods we used in our previous publication.  To improve our ability to programmatically identify cases where the crystallog. conformation is not well reproduced computationally, we introduce a new metric to compare conformations, RMSTanimoto.  This new metric is used alongside those from our previous publication to efficiently identify reprodn. failures.  We find RMSTanimoto to be particularly effective in identifying failures for the smallest mols. in our data sets.  Anal. of the nature of these failures, particularly those for the CSD, sheds further light on the issue of strain in crystallog. structures.  Some of the residual failure cases not resolved by simple changes in OMEGA's defaults present significant challenges to conformer generation engines like OMEGA and are a source of new avenues to further improve their performance, while others illustrate the pitfalls of validating against crystallog. ligand conformations, particularly those from the PDB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTlHYy7GV9RbVg90H21EOLACvtfcHk0lieyH-4Jiok3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN&md5=852c44d8587480e69709cefeee9206ac</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci300314k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300314k%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Learning%2520from%2520the%2520Data%2520Set%2520and%2520the%2520Analysis%2520of%2520Failures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2919%26epage%3D2936%26doi%3D10.1021%2Fci300314k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+Generation+with+OMEGA%3A+Algorithm+and+Validation+Using+High+Quality+Structures+from+the+Protein+Databank+and+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0lieyH-4Jiok3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520Validation%2520Using%2520High%2520Quality%2520Structures%2520from%2520the%2520Protein%2520Databank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdelyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11742</span>– <span class="NLM_lpage">11757</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01379</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=11742-11757&author=V.+Poongavanamauthor=E.+Daneliusauthor=S.+Peintnerauthor=L.+Alcarazauthor=G.+Caronauthor=M.+D.+Cummingsauthor=S.+Wlodekauthor=M.+Erdelyiauthor=P.+C.+D.+Hawkinsauthor=G.+Ermondiauthor=J.+Kihlberg&title=Conformational+Sampling+of+Macrocyclic+Drugs+in+Different+Environments%3A+Can+We+Find+the+Relevant+Conformations%3F&doi=10.1021%2Facsomega.8b01379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Danelius, Emma; Peintner, Stefan; Alcaraz, Lilian; Caron, Giulia; Cummings, Maxwell D.; Wlodek, Stanislaw; Erdelyi, Mate; Hawkins, Paul C. D.; Ermondi, Giuseppe; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11742-11757</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility is a major determinant of the properties of macrocycles and other drugs in beyond rule of 5 (bRo5) space.  Prediction of conformations is essential for design of drugs in this space and the authors have evaluated three tools for conformational sampling of a set of ten bRo5 drugs and clin. candidates in polar and apolar environments.  The distance-geometry based OMEGA was found to yield ensembles spanning larger structure and property spaces than the two MD based methods, MOE-LowMode MD (MOE) and MacroModel-Macrocycle Conformational Sampling (MC).  Both MC and OMEGA, but not MOE, generated different ensembles for polar and apolar environments.  All three conformational search methods generated conformers similar to the crystal structure conformers for nine of the ten compds., with OMEGA performing somewhat better than MOE and MC.  MOE and OMEGA found all six conformers of roxithromycin that were identified by NMR in aq. solns., while only OMEGA sampled the three conformers obsd. in chloroform.  The authors suggest that characterization of conformers using mol. descriptors, e.g. the radius of gyration (Rgyr) and polar surface area (PSA), is preferred to energy- or RMSD-based methods for selection of biol. relevant conformers in drug discovery in bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoetOG-yDeciLVg90H21EOLACvtfcHk0lieyH-4Jiok3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI&md5=a8722551b161e907daf55146f22b6d57</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DErdelyi%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DConformational%2520Sampling%2520of%2520Macrocyclic%2520Drugs%2520in%2520Different%2520Environments%253A%2520Can%2520We%2520Find%2520the%2520Relevant%2520Conformations%253F%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D11742%26epage%3D11757%26doi%3D10.1021%2Facsomega.8b01379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruder, S.</span></span> <span> </span><span class="NLM_article-title">An Overview of Gradient Descent Optimization Algorithms</span>. <span class="NLM_year">2016</span>, arXiv:1609.04747. arXiv.org e-Print archive. <a href="https://arxiv.org/abs/1609.04747" class="extLink">https://arxiv.org/abs/1609.04747</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=S.+Ruder&title=An+Overview+of+Gradient+Descent+Optimization+Algorithms"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRuder%26aufirst%3DS.%26atitle%3DAn%2520Overview%2520of%2520Gradient%2520Descent%2520Optimization%2520Algorithms%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Architecture, Implementation and Parallelisation of the GROMOS Software for Biomolecular Simulation</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">890</span>– <span class="NLM_lpage">903</span>, <span class="refDoi"> DOI: 10.1016/j.cpc.2011.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.cpc.2011.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2012&pages=890-903&author=N.+Schmidauthor=C.+D.+Christauthor=M.+Christenauthor=A.+P.+Eichenbergerauthor=W.+F.+Van+Gunsteren&title=Architecture%2C+Implementation+and+Parallelisation+of+the+GROMOS+Software+for+Biomolecular+Simulation&doi=10.1016%2Fj.cpc.2011.12.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Architecture, implementation and parallelisation of the GROMOS software for biomolecular simulation</span></div><div class="casAuthors">Schmid, Nathan; Christ, Clara D.; Christen, Markus; Eichenberger, Andreas P.; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">890-903</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this work, the design of the latest version of the GROMOS software for biomol. simulation, GROMOS11, is discussed.  Detailed organization and class descriptions of the MD++ simulation program and the GROMOS++ anal. package are given.  It is shown how the code was documented, how it can be easily modified and extended, how debugging of it is carried out.  Addnl. efficiency and parallelization concepts are presented and benchmarked.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3_FLsjZIvWLVg90H21EOLACvtfcHk0ljH9sTWitCdUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKiurc%253D&md5=9b2d6aeebff579549e24c044e9ef8ef4</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2011.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2011.12.014%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DChrist%26aufirst%3DC.%2BD.%26aulast%3DChristen%26aufirst%3DM.%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DArchitecture%252C%2520Implementation%2520and%2520Parallelisation%2520of%2520the%2520GROMOS%2520Software%2520for%2520Biomolecular%2520Simulation%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2012%26volume%3D183%26spage%3D890%26epage%3D903%26doi%3D10.1016%2Fj.cpc.2011.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choutko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Definition and Testing of the GROMOS Force-Field Versions 54A7 and 54B7</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1007/s00249-011-0700-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1007%2Fs00249-011-0700-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=21533652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=843-856&author=N.+Schmidauthor=A.+P.+Eichenbergerauthor=A.+Choutkoauthor=S.+Rinikerauthor=M.+Wingerauthor=A.+E.+Markauthor=W.+F.+Van+Gunsteren&title=Definition+and+Testing+of+the+GROMOS+Force-Field+Versions+54A7+and+54B7&doi=10.1007%2Fs00249-011-0700-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Definition and testing of the GROMOS force-field versions 54A7 and 54B7</span></div><div class="casAuthors">Schmid, Nathan; Eichenberger, Andreas P.; Choutko, Alexandra; Riniker, Sereina; Winger, Moritz; Mark, Alan E.; Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">843-856</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">New parameter sets of the GROMOS biomol. force field, 54A7 and 54B7, are introduced.  These parameter sets summarize some previously published force field modifications: The 53A6 helical propensities are cor. through new φ/ψ torsional angle terms and a modification of the N-H, C=O repulsion, a new atom type for a charged -CH3 in the choline moiety is added, the Na+ and Cl- ions are modified to reproduce the free energy of hydration, and addnl. improper torsional angle types for free energy calcns. involving a chirality change are introduced.  The new helical propensity modification is tested using the benchmark proteins hen egg-white lysozyme, fox1 RNA binding domain, chorismate mutase and the GCN4-p1 peptide.  The stability of the proteins is improved in comparison with the 53A6 force field, and good agreement with a range of primary exptl. data is obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3tjHc1auDa7Vg90H21EOLACvtfcHk0ljH9sTWitCdUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D&md5=a1cd133f1e663da949324060e00c950d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs00249-011-0700-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-011-0700-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DChoutko%26aufirst%3DA.%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWinger%26aufirst%3DM.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DDefinition%2520and%2520Testing%2520of%2520the%2520GROMOS%2520Force-Field%2520Versions%252054A7%2520and%252054B7%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2011%26volume%3D40%26spage%3D843%26epage%3D856%26doi%3D10.1007%2Fs00249-011-0700-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span>; <span class="NLM_string-name">Postma, J. P.</span>; <span class="NLM_string-name">van Gunsteren, W. F.</span>; <span class="NLM_string-name">Hermans, J.</span></span> <span> </span><span class="NLM_article-title">Interaction Models for Water in Relation to Protein Hydration</span>. In  <i>Intermolecular Forces</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pullman, B.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">1981</span>; Vol.  <span class="NLM_volume">14</span>, pp  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">342</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1007%2F978-94-015-7658-1_21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1981&pages=331-342&author=H.+J.+C.+Berendsen&author=J.+P.+Postma&author=W.+F.+van+Gunsteren&author=J.+Hermansauthor=B.+Pullman&title=Intermolecular+Forces"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2F978-94-015-7658-1_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-015-7658-1_21%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DInteraction%2520Models%2520for%2520Water%2520in%2520Relation%2520to%2520Protein%2520Hydration%26btitle%3DIntermolecular%2520Forces%26aulast%3DPullman%26aufirst%3DB.%26pub%3DSpringer%26date%3D1981%26volume%3D14%26spage%3D331%26epage%3D342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tironi, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">A Molecular Dynamics Simulation Study of Chloroform</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1080/00268979400101331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1080%2F00268979400101331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK2cXntVSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1994&pages=381-403&author=I.+G.+Tironiauthor=W.+F.+Van+Gunsteren&title=A+Molecular+Dynamics+Simulation+Study+of+Chloroform&doi=10.1080%2F00268979400101331"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular dynamics simulation study of chloroform</span></div><div class="casAuthors">Tironi, Ilario G.; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">381-403</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis</span>)
        </div><div class="casAbstract">Three different chloroform models have been investigated using mol. dynamics computer simulation.  The thermodn., structural and dynamic properties of the various models were investigated in detail.  In particular the potential energies, diffusion coeffs. and rotational correlation times obtained for each model are compared with expt.  It is found that the theory of rotational Brownian motion fails in describing the rotational diffusion of chloroform.  The force field of Dietz and Heinzinger was found to give good overall agreement with expt.  An extended investigation of this chloroform model has been performed.  Values are reported for the isothermal compressibility, the thermal expansion coeff. and the const. vol. heat capacity.  The values agree well with expt.  The static and frequency dependent dielec. permittivity were computed from a 1.2 ns simulation conducted under reaction field boundary conditions.  Considering the fact that the model is rigid with fixed partial charges, the static dielec. const. and Debye relaxation time compare well with expt.  From the same simulation the shear viscosity was computed using the off-diagonal elements of the pressure tensor, both via an Einstein type relation and via a Green-Kubo equation.  The calcd. viscosities show good agreement with exptl. values.  The excess Helmholtz energy is calcd. using the thermodn. integration technique and simulations of 50 and 80 ps.  The value obtained for the excess Helmholtz energy matches the theor. value within a few per cent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom3066SYtKJrVg90H21EOLACvtfcHk0lhq59IvfTBchw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXntVSnsbc%253D&md5=1a1218133613b92f48c7d8d658d177a8</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1080%2F00268979400101331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268979400101331%26sid%3Dliteratum%253Aachs%26aulast%3DTironi%26aufirst%3DI.%2BG.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DA%2520Molecular%2520Dynamics%2520Simulation%2520Study%2520of%2520Chloroform%26jtitle%3DMol.%2520Phys.%26date%3D1994%26volume%3D83%26spage%3D381%26epage%3D403%26doi%3D10.1080%2F00268979400101331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">A Leap-Frog Algorithm for Stochastic Dynamics</span>. <i>Mol. Simul.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1080/08927028808080941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1080%2F08927028808080941" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1988&pages=173-185&author=W.+F.+Van%0AGunsterenauthor=H.+J.+C.+Berendsen&title=A+Leap-Frog+Algorithm+for+Stochastic+Dynamics&doi=10.1080%2F08927028808080941"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1080%2F08927028808080941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F08927028808080941%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DA%2520Leap-Frog%2520Algorithm%2520for%2520Stochastic%2520Dynamics%26jtitle%3DMol.%2520Simul.%26date%3D1988%26volume%3D1%26spage%3D173%26epage%3D185%26doi%3D10.1080%2F08927028808080941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical Integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.+P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+Integration+of+the+Cartesian+Equations+of+Motion+of+a+System+with+Constraints%3A+Molecular+Dynamics+of+n-Alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0lhq59IvfTBchw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.%2BP.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520Integration%2520of%2520the%2520Cartesian%2520Equations%2520of%2520Motion%2520of%2520a%2520System%2520with%2520Constraints%253A%2520Molecular%2520Dynamics%2520of%2520n-Alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tironi, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperb, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">A Generalized Reaction Field Method for Molecular Dynamics Simulations</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">5451</span>– <span class="NLM_lpage">5459</span>, <span class="refDoi"> DOI: 10.1063/1.469273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.469273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK2MXkslOmsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1995&pages=5451-5459&author=I.+G.+Tironiauthor=R.+Sperbauthor=P.+E.+Smithauthor=W.+F.+van+Gunsteren&title=A+Generalized+Reaction+Field+Method+for+Molecular+Dynamics+Simulations&doi=10.1063%2F1.469273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A generalized reaction field method for molecular dynamics simulations</span></div><div class="casAuthors">Tironi, Ilario G.; Sperb, Rene; Smith, Paul E.; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5451-9</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Mol.-dynamics simulations of ionic systems require the inclusion of long-range electrostatic forces.  We propose an expression for the long-range electrostatic forces based on an anal. soln. of the Poisson-Boltzmann equation outside a spherical cutoff, which can easily be implemented in mol. simulation programs.  An anal. soln. of the linearized Poisson-Boltzmann (PB) equation valid in a spherical region is obtained.  From this general soln. special expressions are derived for evaluating the electrostatic potential and its deriv. at the origin of the sphere.  These expressions have been implemented for mol.-dynamics (MD) simulations, such that the surface of the cutoff sphere around a charged particle is identified with the spherical boundary of the Poisson-Boltzmann problem.  The anal. soln. of the Poisson-Boltzmann equation is valid for the cutoff sphere and can be used for calcg. the reaction field forces on the central charge, assuming a uniform continuum of given ionic strength beyond the cutoff.  MD simulations are preformed for a periodic system consisting of 2127 SPC water mols. with 40 NaCl ions (1 M).  We compared the structural and dynamic results obtained from MD simulations in which the long-range electrostatic interactions are treated differently; using a cutoff radius, using a cutoff radius and a Poisson-Boltzmann generalized reaction field force, and using the Ewald summation.  Application of the Poisson-Boltzmann generalized reaction field gives a dramatic improvement of the structure of the soln. compared to a simple cutoff treatment, at no extra computational cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3IHTuIOoegbVg90H21EOLACvtfcHk0lhq59IvfTBchw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkslOmsLg%253D&md5=b20169702e5929b6e5967d7cbc2fdb01</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1063%2F1.469273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.469273%26sid%3Dliteratum%253Aachs%26aulast%3DTironi%26aufirst%3DI.%2BG.%26aulast%3DSperb%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DP.%2BE.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DA%2520Generalized%2520Reaction%2520Field%2520Method%2520for%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D102%26spage%3D5451%26epage%3D5459%26doi%3D10.1063%2F1.469273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinz, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hünenberger, P. H.</span></span> <span> </span><span class="NLM_article-title">Comparison of Four Methods to Compute the Dielectric Permittivity of Liquids from Molecular Dynamics Simulations</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1136</span>, <span class="refDoi"> DOI: 10.1063/1.1379764</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.1379764" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFCmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2001&pages=1125-1136&author=T.+N.+Heinzauthor=W.+F.+Van+Gunsterenauthor=P.+H.+H%C3%BCnenberger&title=Comparison+of+Four+Methods+to+Compute+the+Dielectric+Permittivity+of+Liquids+from+Molecular+Dynamics+Simulations&doi=10.1063%2F1.1379764"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of four methods to compute the dielectric permittivity of liquids from molecular dynamics simulations</span></div><div class="casAuthors">Heinz, Tim N.; van Gunsteren, Wilfred F.; Hunenberger, Philippe H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1125-1136</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Four methods to compute the dielec. permittivity ε of a liq. from mol. simulations are compared in the context of the simple point charge (SPC) water model.  In the 1st method (unrestrained method), ε is evaluated from the fluctuations of the box dipole moment M, monitored during a single equil. simulation.  In the three other methods, ε is evaluated from the probability distribution p(M) of the dipole moment norm.  This distribution is itself evaluated in three different ways: (i) from multiple simulations involving a M-dependent biasing potential (umbrella-sampling method), (ii) from multiple simulations involving a constrained dipole moment norm (M-constraint method), or (iii) from fitting of incomplete p(M) ests. to a Maxwell distribution (fitting method).  The four methods converge to an identical est. of ε=61 ± 1 for SPC water (256 mols., reaction-field electrostatics).  The convergence properties, advantages, and drawbacks of the different methods are analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXa1Mr7u_k2bVg90H21EOLACvtfcHk0lj1s-p0keRreg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFCmurg%253D&md5=626fcf439551260b8e297b818d9e3319</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1063%2F1.1379764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1379764%26sid%3Dliteratum%253Aachs%26aulast%3DHeinz%26aufirst%3DT.%2BN.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%2BH.%26atitle%3DComparison%2520of%2520Four%2520Methods%2520to%2520Compute%2520the%2520Dielectric%2520Permittivity%2520of%2520Liquids%2520from%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2001%26volume%3D115%26spage%3D1125%26epage%3D1136%26doi%3D10.1063%2F1.1379764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics with Coupling to an External Bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van+Gunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+Dynamics+with+Coupling+to+an+External+Bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lj1s-p0keRreg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520Dynamics%2520with%2520Coupling%2520to%2520an%2520External%2520Bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scherer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trendelkamp-Schroer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Hernández, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plattner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehmeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noé, F.</span></span> <span> </span><span class="NLM_article-title">PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">5525</span>– <span class="NLM_lpage">5542</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00743</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00743" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Gksr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=5525-5542&author=M.+K.+Schererauthor=B.+Trendelkamp-Schroerauthor=F.+Paulauthor=G.+P%C3%A9rez-Hern%C3%A1ndezauthor=M.+Hoffmannauthor=N.+Plattnerauthor=C.+Wehmeyerauthor=J.-H.+Prinzauthor=F.+No%C3%A9&title=PyEMMA+2%3A+A+Software+Package+for+Estimation%2C+Validation%2C+and+Analysis+of+Markov+Models&doi=10.1021%2Facs.jctc.5b00743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models</span></div><div class="casAuthors">Scherer, Martin K.; Trendelkamp-Schroer, Benjamin; Paul, Fabian; Perez-Hernandez, Guillermo; Hoffmann, Moritz; Plattner, Nuria; Wehmeyer, Christoph; Prinz, Jan-Hendrik; Noe, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5525-5542</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Markov (state) models (MSMs) and related models of mol. kinetics have recently received a surge of interest as they can systematically reconcile simulation data from either a few long or many short simulations and allow us to analyze the essential metastable structures, thermodn., and kinetics of the mol. system under investigation.  However, the estn., validation, and anal. of such models is far from trivial and involves sophisticated and often numerically sensitive methods.  In this work, we present the open-source Python package PyEMMA (pyemma.org) that provides accurate and efficient algorithms for kinetic model construction.  PyEMMA can read all common mol. dynamics data formats, helps in the selection of input features, provides easy access to dimension redn. algorithms such as principal component anal. (PCA) and time-lagged independent component anal. (TICA) and clustering algorithms such as k-means, and contains estimators for MSMs, hidden Markov models, and several other models.  Systematic model validation and error calcn. methods are provided.  PyEMMA offers a wealth of anal. functions such that the user can conveniently compute mol. observables of interest.  We have derived a systematic and accurate way to coarse-grain MSMs to few states and to illustrate the structures of the metastable states of the system.  Plotting functions to produce a manuscript-ready presentation of the results are available.  In this work, we demonstrate the features of the software and show new methodol. concepts and results produced by PyEMMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY6T9WAZpy17Vg90H21EOLACvtfcHk0lj1s-p0keRreg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Gksr7N&md5=dc04b91b37acb7c0bfae1bc0475dcd93</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00743%26sid%3Dliteratum%253Aachs%26aulast%3DScherer%26aufirst%3DM.%2BK.%26aulast%3DTrendelkamp-Schroer%26aufirst%3DB.%26aulast%3DPaul%26aufirst%3DF.%26aulast%3DP%25C3%25A9rez-Hern%25C3%25A1ndez%26aufirst%3DG.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DPlattner%26aufirst%3DN.%26aulast%3DWehmeyer%26aufirst%3DC.%26aulast%3DPrinz%26aufirst%3DJ.-H.%26aulast%3DNo%25C3%25A9%26aufirst%3DF.%26atitle%3DPyEMMA%25202%253A%2520A%2520Software%2520Package%2520for%2520Estimation%252C%2520Validation%252C%2520and%2520Analysis%2520of%2520Markov%2520Models%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D5525%26epage%3D5542%26doi%3D10.1021%2Facs.jctc.5b00743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGibbon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beauchamp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrigan, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swails, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwantes, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1528</span>– <span class="NLM_lpage">1532</span>, <span class="refDoi"> DOI: 10.1016/j.bpj.2015.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.bpj.2015.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=26488642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKhu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2015&pages=1528-1532&author=R.+T.+McGibbonauthor=K.+A.+Beauchampauthor=M.+P.+Harriganauthor=C.+Kleinauthor=J.+M.+Swailsauthor=C.+X.+Hern%C3%A1ndezauthor=C.+R.+Schwantesauthor=L.+P.+Wangauthor=T.+J.+Laneauthor=V.+S.+Pande&title=MDTraj%3A+A+Modern+Open+Library+for+the+Analysis+of+Molecular+Dynamics+Trajectories&doi=10.1016%2Fj.bpj.2015.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</span></div><div class="casAuthors">McGibbon, Robert T.; Beauchamp, Kyle A.; Harrigan, Matthew P.; Klein, Christoph; Swails, Jason M.; Hernandez, Carlos X.; Schwantes, Christian R.; Wang, Lee-Ping; Lane, Thomas J.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1528-1532</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">As mol. dynamics (MD) simulations continue to evolve into powerful computational tools for studying complex biomol. systems, the necessity of flexible and easy-to-use software tools for the anal. of these simulations is growing.  We have developed MDTraj, a modern, lightwt., and fast software package for analyzing MD simulations.  MDTraj reads and writes trajectory data in a wide variety of commonly used formats.  It provides a large no. of trajectory anal. capabilities including minimal root-mean-square-deviation calcns., secondary structure assignment, and the extn. of common order parameters.  The package has a strong focus on interoperability with the wider scientific Python ecosystem, bridging the gap between MD data and the rapidly growing collection of industry-std. statistical anal. and visualization tools in Python.  MDTraj is a powerful and user-friendly software package that simplifies the anal. of MD data and connects these datasets with the modern interactive data science software ecosystem in Python.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU5T-L7qmT8LVg90H21EOLACvtfcHk0li2bei3cOo7cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKhu7%252FI&md5=9cbd2138ce200e7d63fda8ae756df43f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2015.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2015.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DMcGibbon%26aufirst%3DR.%2BT.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DHarrigan%26aufirst%3DM.%2BP.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHern%25C3%25A1ndez%26aufirst%3DC.%2BX.%26aulast%3DSchwantes%26aufirst%3DC.%2BR.%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DLane%26aufirst%3DT.%2BJ.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMDTraj%253A%2520A%2520Modern%2520Open%2520Library%2520for%2520the%2520Analysis%2520of%2520Molecular%2520Dynamics%2520Trajectories%26jtitle%3DBiophys.%2520J.%26date%3D2015%26volume%3D109%26spage%3D1528%26epage%3D1532%26doi%3D10.1016%2Fj.bpj.2015.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molgedey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, H. G.</span></span> <span> </span><span class="NLM_article-title">Separation of a Mixture of Independent Signals Using Time Delayed Correlations</span>. <i>Phys. Rev. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3634</span>– <span class="NLM_lpage">3637</span>, <span class="refDoi"> DOI: 10.1103/PhysRevLett.72.3634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1103%2FPhysRevLett.72.3634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10056251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC2sfpsFKisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1994&pages=3634-3637&author=L.+Molgedeyauthor=H.+G.+Schuster&title=Separation+of+a+Mixture+of+Independent+Signals+Using+Time+Delayed+Correlations&doi=10.1103%2FPhysRevLett.72.3634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Separation of a mixture of independent signals using time delayed correlations</span></div><div class="casAuthors">Molgedey; Schuster</div><div class="citationInfo"><span class="NLM_cas:title">Physical review letters</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3634-3637</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKlcmWVlGfwsScOqQTSkOffW6udTcc2ebaswOf0arnQ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sfpsFKisQ%253D%253D&md5=fc21b44ff5e05410e236cad768bcee81</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.72.3634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.72.3634%26sid%3Dliteratum%253Aachs%26aulast%3DMolgedey%26aufirst%3DL.%26aulast%3DSchuster%26aufirst%3DH.%2BG.%26atitle%3DSeparation%2520of%2520a%2520Mixture%2520of%2520Independent%2520Signals%2520Using%2520Time%2520Delayed%2520Correlations%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1994%26volume%3D72%26spage%3D3634%26epage%3D3637%26doi%3D10.1103%2FPhysRevLett.72.3634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daura, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Comparing Geometric and Kinetic Cluster Algorithms for Molecular Simulation Data</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>, <span class="NLM_elocation-id">074110</span> <span class="refDoi"> DOI: 10.1063/1.3301140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.3301140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=20170218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1OqurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&author=B.+Kellerauthor=X.+Dauraauthor=W.+F.+Van+Gunsteren&title=Comparing+Geometric+and+Kinetic+Cluster+Algorithms+for+Molecular+Simulation+Data&doi=10.1063%2F1.3301140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing geometric and kinetic cluster algorithms for molecular simulation data</span></div><div class="casAuthors">Keller, Bettina; Daura, Xavier; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">074110/1-074110/16</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The identification of metastable states of a mol. plays an important role in the interpretation of mol. simulation data because the free-energy surface, the relative populations in this landscape, and ultimately also the dynamics of the mol. under study can be described in terms of these states.  We compare the results of three different geometric cluster algorithms (neighbor algorithm, K-medoids algorithm, and common-nearest-neighbor algorithm) among each other and to the results of a kinetic cluster algorithm.  First, we demonstrate the characteristics of each of the geometric cluster algorithms using five two-dimensional data sets.  Second, we analyze the mol. dynamics data of a β-heptapeptide in methanol - a mol. that exhibits a distinct folded state, a structurally diverse unfolded state, and a fast folding/unfolding equil. - using both geometric and kinetic cluster algorithms.  We find that geometric clustering strongly depends on the algorithm used and that the d. based common-nearest-neighbor algorithm is the most robust of the three geometric cluster algorithms with respect to variations in the input parameters and the distance metric.  When comparing the geometric cluster results to the metastable states of the β-heptapeptide as identified by kinetic clustering, we find that in most cases the folded state is identified correctly but the overlap of geometric clusters with further metastable states is often at best approx.  (c) 2010 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKj0e-QRDQDrVg90H21EOLACvtfcHk0ljMDLfxmWdLXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1OqurY%253D&md5=753b03fa1abb3a218e177e8cfd026964</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1063%2F1.3301140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3301140%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DB.%26aulast%3DDaura%26aufirst%3DX.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DComparing%2520Geometric%2520and%2520Kinetic%2520Cluster%2520Algorithms%2520for%2520Molecular%2520Simulation%2520Data%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D132%26doi%3D10.1063%2F1.3301140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiß, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ries, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Volume-Scaled Common Nearest Neighbor Clustering Algorithm with Free-Energy Hierarchy</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>154</i></span>,  <span class="NLM_fpage">084106</span>, <span class="refDoi"> DOI: 10.1063/5.0025797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F5.0025797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=33639726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltFOjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2021&pages=084106&author=R.+G.+Wei%C3%9Fauthor=B.+Riesauthor=S.+Wangauthor=S.+Riniker&title=Volume-Scaled+Common+Nearest+Neighbor+Clustering+Algorithm+with+Free-Energy+Hierarchy&doi=10.1063%2F5.0025797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Volume-scaled common nearest neighbor clustering algorithm with free-energy hierarchy</span></div><div class="casAuthors">Weiss, R. Gregor; Ries, Benjamin; Wang, Shuzhe; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084106</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The combination of Markov state modeling (MSM) and mol. dynamics (MD) simulations has been shown in recent years to be a valuable approach to unravel the slow processes of mol. systems with increasing complexity.  While the algorithms for intermediate steps in the MSM workflow such as featurization and dimensionality redn. have been specifically adapted to MD datasets, conventional clustering methods are generally applied to the discretization step.  This work adds to recent efforts to develop specialized d.-based clustering algorithms for the Boltzmann-weighted data from MD simulations.  We introduce the vol.-scaled common nearest neighbor (vs-CNN) clustering that is an adapted version of the common nearest neighbor (CNN) algorithm.  A major advantage of the proposed algorithm is that the introduced d.-based criterion directly links to a free-energy notion via Boltzmann inversion.  Such a free-energy perspective allows a straightforward hierarchical scheme to identify conformational clusters at different levels of a generally rugged free-energy landscape of complex mol. systems.  (c) 2021 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwvedFcqDbYrVg90H21EOLACvtfcHk0ljMDLfxmWdLXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltFOjurs%253D&md5=41655db8575a58aa74693d4bc3cf7495</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1063%2F5.0025797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F5.0025797%26sid%3Dliteratum%253Aachs%26aulast%3DWei%25C3%259F%26aufirst%3DR.%2BG.%26aulast%3DRies%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DVolume-Scaled%2520Common%2520Nearest%2520Neighbor%2520Clustering%2520Algorithm%2520with%2520Free-Energy%2520Hierarchy%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2021%26volume%3D154%26spage%3D084106%26doi%3D10.1063%2F5.0025797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolenc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerke, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horta, B. A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oostenbrink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">GROMOS++ Software for the Analysis of Biomolecular Simulation Trajectories</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3379</span>– <span class="NLM_lpage">3390</span>, <span class="refDoi"> DOI: 10.1021/ct2003622</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct2003622" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOit77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=3379-3390&author=A.+P.+Eichenbergerauthor=J.+R.+Allisonauthor=J.+Dolencauthor=D.+P.+Geerkeauthor=B.+A.+C.+Hortaauthor=K.+Meierauthor=C.+Oostenbrinkauthor=N.+Schmidauthor=D.+Steinerauthor=D.+Wangauthor=W.+F.+Van+Gunsteren&title=GROMOS%2B%2B+Software+for+the+Analysis+of+Biomolecular+Simulation+Trajectories&doi=10.1021%2Fct2003622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">GROMOS++ Software for the Analysis of Biomolecular Simulation Trajectories</span></div><div class="casAuthors">Eichenberger, Andreas P.; Allison, Jane R.; Dolenc, Jozica; Geerke, Daan P.; Horta, Bruno A. C.; Meier, Katharina; Oostenbrink, Chris; Schmid, Nathan; Steiner, Denise; Wang, Dongqi; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3379-3390</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  GROMOS++ is a set of C++ programs for pre- and postprocessing of mol. dynamics simulation trajectories and as such is part of the GROningen Mol. Simulation software for (bio)mol. simulation.  It contains more than 70 programs that can be used to prep. data for the prodn. of mol. simulation trajectories and to analyze these.  These programs are reviewed and the various structural, dynamic, and thermodn. quantities that can be analyzed using time series, correlation functions, and distributions are described together with tech. aspects of their implementation in GROMOS.  A few examples of the use of GROMOS++ for the anal. of MD trajectories are given.  A full list of all GROMOS++ programs, together with an indication of their capabilities, is given in the Appendix.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoODOTei0lrjbVg90H21EOLACvtfcHk0lgP7UTEE23d8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOit77O&md5=b87a1d1b93445e9fe5a67ddbd3f1197f</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fct2003622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct2003622%26sid%3Dliteratum%253Aachs%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DAllison%26aufirst%3DJ.%2BR.%26aulast%3DDolenc%26aufirst%3DJ.%26aulast%3DGeerke%26aufirst%3DD.%2BP.%26aulast%3DHorta%26aufirst%3DB.%2BA.%2BC.%26aulast%3DMeier%26aufirst%3DK.%26aulast%3DOostenbrink%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DSteiner%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DGROMOS%252B%252B%2520Software%2520for%2520the%2520Analysis%2520of%2520Biomolecular%2520Simulation%2520Trajectories%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D3379%26epage%3D3390%26doi%3D10.1021%2Fct2003622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vögeli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Güntert, P.</span></span> <span> </span><span class="NLM_article-title">Compiled Data Set of Exact NOE Distance Limits, Residual Dipolar Couplings and Scalar Couplings for the Protein GB3</span>. <i>Data Brief</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.dib.2015.08.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.dib.2015.08.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=26504890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC28zlvFKhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=99-106&author=B.+V%C3%B6geliauthor=S.+Olssonauthor=R.+Riekauthor=P.+G%C3%BCntert&title=Compiled+Data+Set+of+Exact+NOE+Distance+Limits%2C+Residual+Dipolar+Couplings+and+Scalar+Couplings+for+the+Protein+GB3&doi=10.1016%2Fj.dib.2015.08.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Compiled data set of exact NOE distance limits, residual dipolar couplings and scalar couplings for the protein GB3</span></div><div class="casAuthors">Vogeli Beat; Riek Roland; Olsson Simon; Guntert Peter</div><div class="citationInfo"><span class="NLM_cas:title">Data in brief</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-106</span>
        ISSN:<span class="NLM_cas:issn">2352-3409</span>.
    </div><div class="casAbstract">We compiled an NMR data set consisting of exact nuclear Overhauser enhancement (eNOE) distance limits, residual dipolar couplings (RDCs) and scalar (J) couplings for GB3, which forms one of the largest and most diverse data set for structural characterization of a protein to date.  All data have small experimental errors, which are carefully estimated.  We use the data in the research article Vogeli et al., 2015, Complementarity and congruence between exact NOEs and traditional NMR probes for spatial decoding of protein dynamics, J.  Struct.  Biol., 191, 3, 306-317, doi:10.1016/j.jsb.2015.07.008 [1] for cross-validation in multiple-state structural ensemble calculation.  We advocate this set to be an ideal test case for molecular dynamics simulations and structure calculations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQGhCQAQpdjhJ5E29YNGvQfW6udTcc2ebKbl9_3Ij1R7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zlvFKhuw%253D%253D&md5=feecb999e7043e445f1439d8f752c38e</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.dib.2015.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dib.2015.08.020%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25B6geli%26aufirst%3DB.%26aulast%3DOlsson%26aufirst%3DS.%26aulast%3DRiek%26aufirst%3DR.%26aulast%3DG%25C3%25BCntert%26aufirst%3DP.%26atitle%3DCompiled%2520Data%2520Set%2520of%2520Exact%2520NOE%2520Distance%2520Limits%252C%2520Residual%2520Dipolar%2520Couplings%2520and%2520Scalar%2520Couplings%2520for%2520the%2520Protein%2520GB3%26jtitle%3DData%2520Brief%26date%3D2015%26volume%3D5%26spage%3D99%26epage%3D106%26doi%3D10.1016%2Fj.dib.2015.08.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Virtanen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gommers, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliphant, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haberland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cournapeau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burovski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weckesser, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Walt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millman, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayorov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, A. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polat, İ.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderPlas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laxalde, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perktold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimrman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriksen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quintero, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archibald, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedregosa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Mulbregt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijaykumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietro</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilboll, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloeckner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scopatz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häggström, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasechnik, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivetti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieser, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenders, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Probst, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silterra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönberger, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miranda Cardoso, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, J. L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez-Iglesias, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuczynski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tritz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newville, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kümmerer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolingbroke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tartre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowaczyk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shebanov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlyk, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodtkorb, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGibbon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldbauer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tygier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">More, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pudlik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pingel, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halchenko, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Baeza, Y.</span></span> <span> </span><span class="NLM_article-title">SciPy 1.0: Fundamental Algorithms for Scientific Computing in Python</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1038/s41592-019-0686-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fs41592-019-0686-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=32015543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3cXislCjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&pages=261-272&author=P.+Virtanenauthor=R.+Gommersauthor=T.+E.+Oliphantauthor=M.+Haberlandauthor=T.+Reddyauthor=D.+Cournapeauauthor=E.+Burovskiauthor=P.+Petersonauthor=W.+Weckesserauthor=J.+Brightauthor=S.+J.+van+der+Waltauthor=M.+Brettauthor=J.+Wilsonauthor=K.+J.+Millmanauthor=N.+Mayorovauthor=A.+R.+J.+Nelsonauthor=E.+Jonesauthor=R.+Kernauthor=E.+Larsonauthor=C.+J.+Careyauthor=%C4%B0.+Polatauthor=Y.+Fengauthor=E.+W.+Mooreauthor=J.+VanderPlasauthor=D.+Laxaldeauthor=J.+Perktoldauthor=R.+Cimrmanauthor=I.+Henriksenauthor=E.+A.+Quinteroauthor=C.+R.+Harrisauthor=A.+M.+Archibaldauthor=A.+H.+Ribeiroauthor=F.+Pedregosaauthor=P.+van+Mulbregtauthor=A.+Vijaykumarauthor=A.+Bardelliauthor=+Pietroauthor=A.+Rothbergauthor=A.+Hilbollauthor=A.+Kloecknerauthor=A.+Scopatzauthor=A.+Leeauthor=A.+Rokemauthor=C.+N.+Woodsauthor=C.+Fultonauthor=C.+Massonauthor=C.+H%C3%A4ggstr%C3%B6mauthor=C.+Fitzgeraldauthor=D.+A.+Nicholsonauthor=D.+R.+Hagenauthor=D.+V.+Pasechnikauthor=E.+Olivettiauthor=E.+Martinauthor=E.+Wieserauthor=F.+Silvaauthor=F.+Lendersauthor=F.+Wilhelmauthor=G.+Youngauthor=G.+A.+Priceauthor=G.+L.+Ingoldauthor=G.+E.+Allenauthor=G.+R.+Leeauthor=H.+Audrenauthor=I.+Probstauthor=J.+P.+Dietrichauthor=J.+Silterraauthor=J.+T.+Webberauthor=J.+Slaviauthor=J.+Nothmanauthor=J.+Buchnerauthor=J.+Kulickauthor=J.+L.+Sch%C3%B6nbergerauthor=J.+V.+de+Miranda+Cardosoauthor=J.+Reimerauthor=J.+Harringtonauthor=J.+L.+C.+Rodr%C3%ADguezauthor=J.+Nunez-Iglesiasauthor=J.+Kuczynskiauthor=K.+Tritzauthor=M.+Thomaauthor=M.+Newvilleauthor=M.+K%C3%BCmmererauthor=M.+Bolingbrokeauthor=M.+Tartreauthor=M.+Pakauthor=N.+J.+Smithauthor=N.+Nowaczykauthor=N.+Shebanovauthor=O.+Pavlykauthor=P.+A.+Brodtkorbauthor=P.+Leeauthor=R.+T.+McGibbonauthor=R.+Feldbauerauthor=S.+Lewisauthor=S.+Tygierauthor=S.+Sievertauthor=S.+Vignaauthor=S.+Petersonauthor=S.+Moreauthor=T.+Pudlikauthor=T.+Oshimaauthor=T.+J.+Pingelauthor=T.+P.+Robitailleauthor=T.+Spuraauthor=T.+R.+Jonesauthor=T.+Ceraauthor=T.+Leslieauthor=T.+Zitoauthor=T.+Kraussauthor=U.+Upadhyayauthor=Y.+O.+Halchenkoauthor=Y.+V%C3%A1zquez-Baeza&title=SciPy+1.0%3A+Fundamental+Algorithms+for+Scientific+Computing+in+Python&doi=10.1038%2Fs41592-019-0686-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">SciPy 1.0: fundamental algorithms for scientific computing in Python</span></div><div class="casAuthors">Virtanen, Pauli; Gommers, Ralf; Oliphant, Travis E.; Haberland, Matt; Reddy, Tyler; Cournapeau, David; Burovski, Evgeni; Peterson, Pearu; Weckesser, Warren; Bright, Jonathan; van der Walt, Stefan J.; Brett, Matthew; Wilson, Joshua; Millman, K. Jarrod; Mayorov, Nikolay; Nelson, Andrew R. J.; Jones, Eric; Kern, Robert; Larson, Eric; Carey, C. J.; Polat, Ilhan; Feng, Yu; Moore, Eric W.; Vander Plas, Jake; Laxalde, Denis; Perktold, Josef; Cimrman, Robert; Henriksen, Ian; Quintero, E. A.; Harris, Charles R.; Archibald, Anne M.; Ribeiro, Antonio H.; Pedregosa, Fabian; van Mulbregt, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-272</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: SciPy is an open-source scientific computing library for the Python programming language.  Since its initial release in 2001, SciPy has become a de facto std. for leveraging scientific algorithms in Python, with over 600 unique code contributors, thousands of dependent packages, over 100,000 dependent repositories and millions of downloads per yr.  In this work, we provide an overview of the capabilities and development practices of SciPy 1.0 and highlight some recent tech. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanlbhUgKQQ7Vg90H21EOLACvtfcHk0lgeTDzXZvHywA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXislCjuro%253D&md5=f007632188adeb57a43469157898e0a8</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fs41592-019-0686-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41592-019-0686-2%26sid%3Dliteratum%253Aachs%26aulast%3DVirtanen%26aufirst%3DP.%26aulast%3DGommers%26aufirst%3DR.%26aulast%3DOliphant%26aufirst%3DT.%2BE.%26aulast%3DHaberland%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DT.%26aulast%3DCournapeau%26aufirst%3DD.%26aulast%3DBurovski%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DP.%26aulast%3DWeckesser%26aufirst%3DW.%26aulast%3DBright%26aufirst%3DJ.%26aulast%3Dvan%2Bder%2BWalt%26aufirst%3DS.%2BJ.%26aulast%3DBrett%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DMillman%26aufirst%3DK.%2BJ.%26aulast%3DMayorov%26aufirst%3DN.%26aulast%3DNelson%26aufirst%3DA.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DKern%26aufirst%3DR.%26aulast%3DLarson%26aufirst%3DE.%26aulast%3DCarey%26aufirst%3DC.%2BJ.%26aulast%3DPolat%26aufirst%3D%25C4%25B0.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DMoore%26aufirst%3DE.%2BW.%26aulast%3DVanderPlas%26aufirst%3DJ.%26aulast%3DLaxalde%26aufirst%3DD.%26aulast%3DPerktold%26aufirst%3DJ.%26aulast%3DCimrman%26aufirst%3DR.%26aulast%3DHenriksen%26aufirst%3DI.%26aulast%3DQuintero%26aufirst%3DE.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BR.%26aulast%3DArchibald%26aufirst%3DA.%2BM.%26aulast%3DRibeiro%26aufirst%3DA.%2BH.%26aulast%3DPedregosa%26aufirst%3DF.%26aulast%3Dvan%2BMulbregt%26aufirst%3DP.%26aulast%3DVijaykumar%26aufirst%3DA.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DPietro%26aulast%3DRothberg%26aufirst%3DA.%26aulast%3DHilboll%26aufirst%3DA.%26aulast%3DKloeckner%26aufirst%3DA.%26aulast%3DScopatz%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DRokem%26aufirst%3DA.%26aulast%3DWoods%26aufirst%3DC.%2BN.%26aulast%3DFulton%26aufirst%3DC.%26aulast%3DMasson%26aufirst%3DC.%26aulast%3DH%25C3%25A4ggstr%25C3%25B6m%26aufirst%3DC.%26aulast%3DFitzgerald%26aufirst%3DC.%26aulast%3DNicholson%26aufirst%3DD.%2BA.%26aulast%3DHagen%26aufirst%3DD.%2BR.%26aulast%3DPasechnik%26aufirst%3DD.%2BV.%26aulast%3DOlivetti%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DWieser%26aufirst%3DE.%26aulast%3DSilva%26aufirst%3DF.%26aulast%3DLenders%26aufirst%3DF.%26aulast%3DWilhelm%26aufirst%3DF.%26aulast%3DYoung%26aufirst%3DG.%26aulast%3DPrice%26aufirst%3DG.%2BA.%26aulast%3DIngold%26aufirst%3DG.%2BL.%26aulast%3DAllen%26aufirst%3DG.%2BE.%26aulast%3DLee%26aufirst%3DG.%2BR.%26aulast%3DAudren%26aufirst%3DH.%26aulast%3DProbst%26aufirst%3DI.%26aulast%3DDietrich%26aufirst%3DJ.%2BP.%26aulast%3DSilterra%26aufirst%3DJ.%26aulast%3DWebber%26aufirst%3DJ.%2BT.%26aulast%3DSlavi%26aufirst%3DJ.%26aulast%3DNothman%26aufirst%3DJ.%26aulast%3DBuchner%26aufirst%3DJ.%26aulast%3DKulick%26aufirst%3DJ.%26aulast%3DSch%25C3%25B6nberger%26aufirst%3DJ.%2BL.%26aulast%3Dde%2BMiranda%2BCardoso%26aufirst%3DJ.%2BV.%26aulast%3DReimer%26aufirst%3DJ.%26aulast%3DHarrington%26aufirst%3DJ.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DJ.%2BL.%2BC.%26aulast%3DNunez-Iglesias%26aufirst%3DJ.%26aulast%3DKuczynski%26aufirst%3DJ.%26aulast%3DTritz%26aufirst%3DK.%26aulast%3DThoma%26aufirst%3DM.%26aulast%3DNewville%26aufirst%3DM.%26aulast%3DK%25C3%25BCmmerer%26aufirst%3DM.%26aulast%3DBolingbroke%26aufirst%3DM.%26aulast%3DTartre%26aufirst%3DM.%26aulast%3DPak%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DN.%2BJ.%26aulast%3DNowaczyk%26aufirst%3DN.%26aulast%3DShebanov%26aufirst%3DN.%26aulast%3DPavlyk%26aufirst%3DO.%26aulast%3DBrodtkorb%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DMcGibbon%26aufirst%3DR.%2BT.%26aulast%3DFeldbauer%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DS.%26aulast%3DTygier%26aufirst%3DS.%26aulast%3DSievert%26aufirst%3DS.%26aulast%3DVigna%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DS.%26aulast%3DMore%26aufirst%3DS.%26aulast%3DPudlik%26aufirst%3DT.%26aulast%3DOshima%26aufirst%3DT.%26aulast%3DPingel%26aufirst%3DT.%2BJ.%26aulast%3DRobitaille%26aufirst%3DT.%2BP.%26aulast%3DSpura%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DCera%26aufirst%3DT.%26aulast%3DLeslie%26aufirst%3DT.%26aulast%3DZito%26aufirst%3DT.%26aulast%3DKrauss%26aufirst%3DT.%26aulast%3DUpadhyay%26aufirst%3DU.%26aulast%3DHalchenko%26aufirst%3DY.%2BO.%26aulast%3DV%25C3%25A1zquez-Baeza%26aufirst%3DY.%26atitle%3DSciPy%25201.0%253A%2520Fundamental%2520Algorithms%2520for%2520Scientific%2520Computing%2520in%2520Python%26jtitle%3DNat.%2520Methods%26date%3D2020%26volume%3D17%26spage%3D261%26epage%3D272%26doi%3D10.1038%2Fs41592-019-0686-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiele, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleucher, J.</span></span> <span> </span><span class="NLM_article-title">EASY ROESY: Reliable Cross-Peak Integration in Adiabatic Symmetrized ROESY</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1002/chem.200802027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fchem.200802027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=19065697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=585-588&author=C.+M.+Thieleauthor=K.+Petzoldauthor=J.+Schleucher&title=EASY+ROESY%3A+Reliable+Cross-Peak+Integration+in+Adiabatic+Symmetrized+ROESY&doi=10.1002%2Fchem.200802027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">EASY ROESY: reliable cross-peak integration in adiabatic symmetrized ROESY</span></div><div class="casAuthors">Thiele, Christina M.; Petzold, Katja; Schleucher, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">585-588</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Ests. of intramol. distances are essential for structure detn.  For medium-sized mols., ROESY NMR spectroscopy is the method of choice for obtaining distances.  However, the integration of ROESY cross-peaks is problematic, owing to the offset dependence of the integrals and/or TOCSY artifacts.  We here present EASY ROESY (Efficient Adiabatic SYmmetried ROESY), which yields reliable intramol. distances without a sample-specific setup.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ogpufwLVLLVg90H21EOLACvtfcHk0ljOu2CzsqbI1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGjsLg%253D&md5=0434726d1d60d71a8fe6907b22a6dd0c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2Fchem.200802027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200802027%26sid%3Dliteratum%253Aachs%26aulast%3DThiele%26aufirst%3DC.%2BM.%26aulast%3DPetzold%26aufirst%3DK.%26aulast%3DSchleucher%26aufirst%3DJ.%26atitle%3DEASY%2520ROESY%253A%2520Reliable%2520Cross-Peak%2520Integration%2520in%2520Adiabatic%2520Symmetrized%2520ROESY%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2009%26volume%3D15%26spage%3D585%26epage%3D588%26doi%3D10.1002%2Fchem.200802027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, P. H. C.</span></span> <span> </span><span class="NLM_article-title">A Perfect Smoother</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3631</span>– <span class="NLM_lpage">3636</span>, <span class="refDoi"> DOI: 10.1021/ac034173t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac034173t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFGqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=3631-3636&author=P.+H.+C.+Eilers&title=A+Perfect+Smoother&doi=10.1021%2Fac034173t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">A Perfect Smoother</span></div><div class="casAuthors">Eilers, Paul H. C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3631-3636</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The A. Savitzky and M. Golay (1964) filter has several disadvantages.  A very attractive alternative is a smoother based on penalized least squares, extending ideas presented by E. T. Whittaker (1923).  This smoother is extremely fast, gives continuous control over smoothness, interpolates automatically, and allows fast leave-one-out cross-validation.  It can be programmed in a few lines of Matlab code.  Theory, implementation, and applications are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3M72rA6bntLVg90H21EOLACvtfcHk0ljOu2CzsqbI1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFGqsLw%253D&md5=eba4aed2ebbf3e48bd80049d8e0ce165</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fac034173t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac034173t%26sid%3Dliteratum%253Aachs%26aulast%3DEilers%26aufirst%3DP.%2BH.%2BC.%26atitle%3DA%2520Perfect%2520Smoother%26jtitle%3DAnal.%2520Chem.%26date%3D2003%26volume%3D75%26spage%3D3631%26epage%3D3636%26doi%3D10.1021%2Fac034173t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kneller, T. D.</span>; <span class="NLM_string-name">Goddard, D. G.</span></span> <i>SPARKY3</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=T.+D.+Kneller&author=D.+G.+Goddard&title=SPARKY3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DKneller%26aufirst%3DT.%2BD.%26jtitle%3DSPARKY3%26pub%3DUniversity%2520of%2520California%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, D.</span></span> <span> </span><span class="NLM_article-title">Nuclear Overhauser Effect</span>.  <i>Encyclopedia of Magnetic Resonance</i>; <span class="NLM_publisher-name">John Wiley & Sons, Ltd.</span>, <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2F9780470034590.emrstm0350.pub2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=1-16&author=D.+Neuhaus&title=Encyclopedia+of+Magnetic+Resonance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1002%2F9780470034590.emrstm0350.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470034590.emrstm0350.pub2%26sid%3Dliteratum%253Aachs%26aulast%3DNeuhaus%26aufirst%3DD.%26atitle%3DNuclear%2520Overhauser%2520Effect%26btitle%3DEncyclopedia%2520of%2520Magnetic%2520Resonance%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Ltd%26date%3D2011%26spage%3D1%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 1 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Philippe Sarret</span>. </span><span class="cited-content_cbyCitation_article-title">Éric Marsault (1971–2021): A Legacy through the Prism of Relationship Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (9)
                                     , 5221-5224. <a href="https://doi.org/10.1021/acs.jmedchem.1c00481" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00481</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00481&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00481%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3D%2525C3%252589ric%252BMarsault%252B%2525281971%2525E2%252580%2525932021%252529%25253A%252BA%252BLegacy%252Bthrough%252Bthe%252BPrism%252Bof%252BRelationship%252BChemistry%26aulast%3DSarret%26aufirst%3DPhilippe%26date%3D2021%26date%3D2021%26date%3D24032021%26volume%3D64%26issue%3D9%26spage%3D5221%26epage%3D5224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Two classes of modifications implemented on model compound (<b>A)</b>: Nala, Nleu, and Nphe peptoids (<b>B</b> showing Nleu) and regio/stereocontrolled C-methylation (<b>C</b> showing 2<i>R</i> methylation).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0012.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of the Tethers<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) (i) NaN<sub>3</sub>, dimethyl sulfoxide (DMSO), quant.; (ii) <i>t</i>-BuCOCl, Et<sub>3</sub>N, tetrahydrofuran (THF), 0 °C; (iii) <i>n</i>-BuLi, (<i>S</i>)-4-benzyl-2-oxazolidinone, THF, −78 °C, 81%. (b) Potassium bis(trimethylsilyl)amide (KHMDS), MeI, THF, −78 °C, 89%. (c) (i) H<sub>2</sub>O<sub>2</sub>, LiOH, H<sub>2</sub>O/THF, 98%, diastereomeric excess (de): 75%; (ii) Pd/C, H<sub>2</sub>, MeOH/AcOH; (iii) (Boc)<sub>2</sub>O, NaHCO<sub>3</sub>, dioxane/H<sub>2</sub>O, 39%. (d) (i) Pyridinium dichromate (PDC), dimethylformamide (DMF), 72%; (ii) CH<sub>2</sub>N<sub>2</sub>, Et<sub>2</sub>O, 95%. (e) RuCl<sub>3</sub>, NaIO<sub>4</sub>, CCl<sub>4</sub>/acetonitrile (ACN)/H<sub>2</sub>O, 70%. (f) (i) diphenylphosphoryl azide (DPPA), Et<sub>3</sub>N, <i>t</i>-BuOH, reflux, 56%; (ii) LiOH, MeOH/H<sub>2</sub>O, 80%. (g) Boc<sub>2</sub>O, NaN<sub>3</sub>, Zn(OTf)<sub>2</sub>, Bu<sub>4</sub>NBr, <i>t</i>-BuOH, THF, 33%. (h) (i) O<sub>3</sub>, MeOH, −78 °C; (ii) H<sub>2</sub>O<sub>2</sub>, NaOH, H<sub>2</sub>O, 60%. (i) (i) isobutylchloroformate, Et<sub>3</sub>N, THF, −10 °C; (ii) CH<sub>2</sub>N<sub>2</sub>, Et<sub>2</sub>O, 52%; (iii) AgOBn, Et<sub>3</sub>N, MeOH, 70%. (j) (i) DiBAl-H, dichloromethane (DCM), −78 °C; (ii) triethyl phosphonoacetate, NaH, THF, 0 °C, 72%. (k) (i) NaOH, H<sub>2</sub>O/MeOH, 96%; (ii) H<sub>2</sub>, Pd/C, MeOH, 99%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0013.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Solid-Phase Synthesis<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions (exemplified for macrocycle Ø-Ø): (a) (i) Fmoc-Phe-OH, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), DCM, 16 h; (ii) piperidine, DMF, 2 × 5 min; (b) (i) Fmoc-Leu-OH, 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-<i>b</i>]pyridinium 3-oxide hexafluorophosphate (HATU), DIPEA, DMF, 3 h; (ii) piperidine, DMF, 2 × 5 min; (c) (i) bromoacetic acid, DIPEA, DCM 45 min; (ii) benzylamine, DMF, 45 min; (d) (i) bromoacetic acid, DIC, DMF, 45 min; (ii) isobutylamine, DMF, 45 min (e) (i) Fmoc-Sar-OH, HATU, DIPEA, DMF, 3 h; (ii) piperidine, DMF, 2 × 5 min; (f) (i) 30% 1,1,1,3,3,3-hexafluoro-2-propanol (HFIP)/DCM, 2 × 30 min; (ii) (3-(diethoxyphosphoryloxy)-1,2,3-benzotriazin-4(3<i>H</i>)-one) (DEPBT), DIPEA, DMF, 16–72 h.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Permeability results in the form of heatmaps. For heatmaps 1–3, the values are expressed as −log(<i>P</i><sub>app</sub>), so lower values mean higher permeability (in order of increasing permeability: blue, white, red, and black). Heatmap 4 shows the BA/AB ratio, which represents a measure of efflux.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (Top) PAMPA results with standard deviation, ranked; (middle) statistical analysis of the PAMPA data shows a significant (<i>p</i> < 0.0001; unpaired <i>t</i>-test) effect for the Nleu vs other macrocycles; and (bottom) a good correlation between PAMPA and Caco-2<sub>A→B</sub> (<i>R</i><sup>2</sup> = 0.91).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inverting peptoid positions to better understand whether higher permeabilities observed with Nleu are due to its nature or to its position within the macrocycle.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Four additional compounds to better elucidate the reason why a peptoid in this position has a beneficial effect on passive permeability. All leucine derivatives are in position 2. Numbers in parentheses indicate a significant difference (<i>p</i> < 0.05; one-way analysis of variance (ANOVA)/Tukey).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Selected cyclic peptides studied with experimental NMR analysis and molecular dynamics (MD) simulations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Root-mean-square deviation (RMSD, in hertz) between <sup>3</sup><i>J</i><sub>HN–Hα</sub> coupling constants in chloroform from NMR measurements and from MD simulations. Clusters with the peptoid bond in trans conformation are shown in green.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Snapshots of Nleu-5R (A) and Nleu-5S (B) from MD simulations in chloroform. Hydrogen bonds are shown with their percentage of the absolute occurrence in chloroform in the <i>trans</i>-peptoid clusters. Pictures were generated with PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0010.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. (A) Torsional-angle distributions of the tether in Nleu-5R (blue) and Nleu-5S (orange) in chloroform. The analysis was restricted to the clusters with the <i>trans</i>-peptoid bond. (B) Torsional angles of the tether (shown in cyan and orange) corresponding to the peaks of the distributions. Pictures were generated with PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The change in the stereocenter also affects the χ<sub>1</sub>-angle of the phenylalanine residue as the tether conformation hinders the rotation around this torsion due to a steric clash with the carbonyl group that is facing out of the backbone ring (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/medium/jm0c02036_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0011.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Torsional-angle distributions of the χ<sub>1</sub> torsional angle of the phenylalanine residue in Nleu-5R (blue) and Nleu-5S (orange) in chloroform. Analysis was restricted to the clusters with the <i>trans</i>-peptoid bond. (B) χ<sub>1</sub> torsional angle of the phenylalanine residue (shown in purple) corresponding to the peaks of the distributions. The backbone carbonyl interferes with the rotation around this torsion is highlighted with a red circle. Pictures were generated with PyMol.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-9/acs.jmedchem.0c02036/20210506/images/large/jm0c02036_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02036&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i97">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 81 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hale, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Exploration of Macrocycles for Drug Discovery--an Underexploited Structural Class</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">608</span>– <span class="NLM_lpage">624</span>, <span class="refDoi"> DOI: 10.1038/nrd2590</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnrd2590" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=18591981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=608-624&author=E.+M.+Driggersauthor=S.+P.+Haleauthor=J.+Leeauthor=N.+K.+Terrett&title=The+Exploration+of+Macrocycles+for+Drug+Discovery%2D%2Dan+Underexploited+Structural+Class&doi=10.1038%2Fnrd2590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The exploration of macrocycles for drug discovery - an underexploited structural class</span></div><div class="casAuthors">Driggers, Edward M.; Hale, Stephen P.; Lee, Jinbo; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">608-624</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Natural products comprised of a macrocycle ring structure have proven their therapeutic applications as antibiotics, immunosuppressants as well as anticancer agents.  Despite this, macrocyclic compds. remain under-explored.  Terrett and colleagues review the properties and features of current macrocycle drugs, emphasizing the vast potential of synthetic macrocycles in drug discovery.  Macrocyclic natural products have evolved to fulfil numerous biochem. functions, and their profound pharmacol. properties have led to their development as drugs.  A macrocycle provides diverse functionality and stereochem. complexity in a conformationally pre-organized ring structure.  This can result in high affinity and selectivity for protein targets, while preserving sufficient bioavailability to reach intracellular locations.  Despite these valuable characteristics, and the proven success of more than 100 marketed macrocycle drugs derived from natural products, this structural class has been poorly explored within drug discovery.  This is in part due to concerns about synthetic intractability and non-drug-like properties.  This Review describes the growing body of data in favor of macrocyclic therapeutics, and demonstrates that this class of compds. can be both fully drug-like in its properties and readily prepd. owing to recent advances in synthetic medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDtW8LpE4ru7Vg90H21EOLACvtfcHk0lhgqG0UdWtOdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnvFShsbc%253D&md5=16e091818053e5b163b81078a8d03094</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd2590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2590%26sid%3Dliteratum%253Aachs%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DHale%26aufirst%3DS.%2BP.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Exploration%2520of%2520Macrocycles%2520for%2520Drug%2520Discovery--an%2520Underexploited%2520Structural%2520Class%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D608%26epage%3D624%26doi%3D10.1038%2Fnrd2590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, I.</span></span> <span> </span><span class="NLM_article-title">Macrocycles in New Drug Discovery</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1409</span>– <span class="NLM_lpage">1438</span>, <span class="refDoi"> DOI: 10.4155/fmc.12.93</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.4155%2Ffmc.12.93" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=22857532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=1409-1438&author=J.+Mallinsonauthor=I.+Collins&title=Macrocycles+in+New+Drug+Discovery&doi=10.4155%2Ffmc.12.93"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles in new drug discovery</span></div><div class="casAuthors">Mallinson, Jamie; Collins, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1409-1438</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  The use of drug-like macrocycles is emerging as an exciting area of medicinal chem., with several recent examples highlighting the favorable changes in biol. and physicochem. properties that macrocyclization can afford.  Natural product macrocycles and their synthetic derivs. have long been clin. useful and attention is now being focused on the wider use of macrocyclic scaffolds in medicinal chem. in the search for new drugs for increasingly challenging targets.  With the increasing awareness of concepts of drug-likeness and the dangers of mol. obesity', functionalized macrocyclic scaffolds could provide a way to generate ligand-efficient mols. with enhanced properties.  In this review we will sep. discuss the effects of macrocyclization upon potency, selectivity and physicochem. properties, concg. on recent case histories in oncol. drug discovery.  Addnl., we will highlight selected advances in the synthesis of macrocycles and provide an outlook on the future use of macrocyclic scaffolds in medicinal chem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSrzcbKW7Tc7Vg90H21EOLACvtfcHk0lgClTMNhzrY2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFCht7rO&md5=63ad275af92ac89ae6b36c5071b578f8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.93&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.93%26sid%3Dliteratum%253Aachs%26aulast%3DMallinson%26aufirst%3DJ.%26aulast%3DCollins%26aufirst%3DI.%26atitle%3DMacrocycles%2520in%2520New%2520Drug%2520Discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2012%26volume%3D4%26spage%3D1409%26epage%3D1438%26doi%3D10.4155%2Ffmc.12.93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dougherty, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span> <span> </span><span class="NLM_article-title">Macrocycles as Protein–Protein Interaction Inhibitors</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_fpage">1109</span>– <span class="NLM_lpage">1125</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1042%2FBCJ20160619" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28298556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=1109-1125&author=P.+G.+Doughertyauthor=Z.+Qianauthor=D.+Pei&title=Macrocycles+as+Protein%E2%80%93Protein+Interaction+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles as protein-protein interaction inhibitors</span></div><div class="casAuthors">Dougherty, Patrick G.; Qian, Ziqing; Pei, Dehua</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">474</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1109-1125</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Macrocyclic compds. such as cyclic peptides have emerged as a new and exciting class of drug candidates for inhibition of intracellular protein-protein interactions, which are challenging targets for conventional drug modalities (i.e. small mols. and proteins).  Over the past decade, several complementary technologies have been developed to synthesize macrocycle libraries and screen them for binding to therapeutically relevant targets.  Two different approaches have also been explored to increase the membrane permeability of cyclic peptides.  In this review, we discuss these methods and their applications in the discovery of macrocyclic compds. against protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFYrVReYebVg90H21EOLACvtfcHk0lgClTMNhzrY2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVygt7g%253D&md5=abd5d7af8383fbe5b553d9501c27ce9c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1042%2FBCJ20160619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBCJ20160619%26sid%3Dliteratum%253Aachs%26aulast%3DDougherty%26aufirst%3DP.%2BG.%26aulast%3DQian%26aufirst%3DZ.%26aulast%3DPei%26aufirst%3DD.%26atitle%3DMacrocycles%2520as%2520Protein%25E2%2580%2593Protein%2520Interaction%2520Inhibitors%26jtitle%3DBiochem.%2520J.%26date%3D2017%26spage%3D1109%26epage%3D1125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span> <span> </span><span class="NLM_article-title">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">2004</span>, <span class="refDoi"> DOI: 10.1021/jm1012374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1012374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1961-2004&author=E.+Marsaultauthor=M.+L.+Peterson&title=Macrocycles+Are+Great+Cycles%3A+Applications%2C+Opportunities%2C+and+Challenges+of+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1021%2Fjm1012374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocycles Are Great Cycles: Applications, Opportunities, and Challenges of Synthetic Macrocycles in Drug Discovery</span></div><div class="casAuthors">Marsault, Eric; Peterson, Mark L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1961-2004</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles occupy a unique segment of chem. space.  In the past decade, their chem. diversity expanded significantly, supported by advances in bioinformatics and synthetic methodol.  As a consequence, this structural type has now been successfully tested on most biol. target classes.  The goal of this article is to put into perspective the current applications, opportunities, and challenges assocd. with synthetic macrocycles in drug discovery.  Accordingly, the first part of this article is dedicated to the drug discovery aspects of macrocycles and highlights salient features of their medicinal chem.  This section is organized by target class, a choice aimed at providing the reader an appreciation of the structural diversity generated for each class.  To give the reader an appreciation of the tools available to construct macrocyclic scaffolds, the site and method of the pivotal macrocyclization step are indicated in the figures.  Readers are referred to the source articles for further details.  In the second part, the technologies and synthetic approaches that already have demonstrated utility or possess a high potential for macrocycle-based drug discovery are discussed.  Finally, a perspective on the future of synthetic macrocycles in medicinal chem. is offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrygQLE6JraHrVg90H21EOLACvtfcHk0lgClTMNhzrY2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXivVSjtbo%253D&md5=97c1120896e90a85790a9ac1274259f3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm1012374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1012374%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26atitle%3DMacrocycles%2520Are%2520Great%2520Cycles%253A%2520Applications%252C%2520Opportunities%252C%2520and%2520Challenges%2520of%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1961%26epage%3D2004%26doi%3D10.1021%2Fjm1012374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abdalla, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGaw, L. J.</span></span> <span> </span><span class="NLM_article-title">Natural Cyclic Peptides as an Attractive Modality for Therapeutics: A Mini Review</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="NLM_elocation-id">2080</span> <span class="refDoi"> DOI: 10.3390/molecules23082080</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.3390%2Fmolecules23082080" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+A.+Abdallaauthor=L.+J.+McGaw&title=Natural+Cyclic+Peptides+as+an+Attractive+Modality+for+Therapeutics%3A+A+Mini+Review&doi=10.3390%2Fmolecules23082080"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Natural cyclic peptides as an attractive modality for therapeutics: a mini review</span></div><div class="casAuthors">Abdalla, Muna Ali; McGaw, Lyndy J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2080/1-2080/19</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Peptides are important biomols. which facilitate the understanding of complex biol. processes, which in turn could be serendipitous biol. targets for future drugs.  They are classified as a unique therapeutic niche and will play an important role as fascinating agents in the pharmaceutical landscape.  Until now, more than 40 cyclic peptide drugs are currently in the market, and approx. one new cyclopeptide drug enters the market annually on av.  Interestingly, the majority of clin. approved cyclic peptides are derived from natural sources, such as peptide antibiotics and human peptide hormones.  In this report, the importance of cyclic peptides is discussed, and their role in drug discovery as interesting therapeutic biomols. will be highlighted.  Recently isolated naturally occurring cyclic peptides from microorganisms, sponges, and other sources with a wide range of pharmacol. properties are reviewed herein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopsXVTy7L4WrVg90H21EOLACvtfcHk0lj48v2pTTZW_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVaktrvJ&md5=4b71e28d331f6131595ac933b6a2f71a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23082080&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23082080%26sid%3Dliteratum%253Aachs%26aulast%3DAbdalla%26aufirst%3DM.%2BA.%26aulast%3DMcGaw%26aufirst%3DL.%2BJ.%26atitle%3DNatural%2520Cyclic%2520Peptides%2520as%2520an%2520Attractive%2520Modality%2520for%2520Therapeutics%253A%2520A%2520Mini%2520Review%26jtitle%3DMolecules%26date%3D2018%26doi%3D10.3390%2Fmolecules23082080" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span>; <span class="NLM_string-name">Peterson, M. L.</span></span>, Eds.;  <i>Practical Medicinal Chemistry with Macrocycles</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2F9781119092599" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=E.+Marsault&author=M.+L.+Peterson&title=Practical+Medicinal+Chemistry+with+Macrocycles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2F9781119092599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781119092599%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26btitle%3DPractical%2520Medicinal%2520Chemistry%2520with%2520Macrocycles%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahadur, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakrabarti, P.</span></span> <span> </span><span class="NLM_article-title">Protein-Protein Interaction and Quaternary Structure</span>. <i>Q. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">133</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1017/S0033583508004708</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1017%2FS0033583508004708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=18812015" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFKhurnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=133-180&author=J.+Janinauthor=R.+P.+Bahadurauthor=P.+Chakrabarti&title=Protein-Protein+Interaction+and+Quaternary+Structure&doi=10.1017%2FS0033583508004708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interaction and quaternary structure</span></div><div class="casAuthors">Janin, Joel; Bahadur, Ranjit P.; Chakrabarti, Pinak</div><div class="citationInfo"><span class="NLM_cas:title">Quarterly Reviews of Biophysics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">133-180</span>CODEN:
                <span class="NLM_cas:coden">QURBAW</span>;
        ISSN:<span class="NLM_cas:issn">0033-5835</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review.  Protein-protein recognition plays an essential role in structure and function.  Specific non-covalent interactions stabilize the structure of macromol. assemblies, exemplified in this review by oligomeric proteins and the capsids of icosahedral viruses.  They also allow proteins to form complexes that have a very wide range of stability and lifetimes and are involved in all cellular processes.  We present some of the structure-based computational methods that have been developed to characterize the quaternary structure of oligomeric proteins and other mol. assemblies and analyze the properties of the interfaces between the subunits.  We compare the size, the chem. and amino acid compns. and the at. packing of the subunit interfaces of protein-protein complexes, oligomeric proteins, viral capsids and protein-nucleic acid complexes.  These biol. significant interfaces are generally close-packed, whereas the non-specific interfaces between mols. in protein crystals are loosely packed, an observation that gives a structural basis to specific recognition.  A distinction is made within each interface between a core that contains buried atoms and a solvent accessible rim.  The core and the rim differ in their amino acid compn. and their conservation in evolution, and the distinction helps correlating the structural data with the results of site-directed mutagenesis and in vitro studies of self-assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraAce2lMynI7Vg90H21EOLACvtfcHk0lj48v2pTTZW_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFKhurnN&md5=9598e158c9e4240f31a629dc9267406e</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1017%2FS0033583508004708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1017%252FS0033583508004708%26sid%3Dliteratum%253Aachs%26aulast%3DJanin%26aufirst%3DJ.%26aulast%3DBahadur%26aufirst%3DR.%2BP.%26aulast%3DChakrabarti%26aufirst%3DP.%26atitle%3DProtein-Protein%2520Interaction%2520and%2520Quaternary%2520Structure%26jtitle%3DQ.%2520Rev.%2520Biophys.%26date%3D2008%26volume%3D41%26spage%3D133%26epage%3D180%26doi%3D10.1017%2FS0033583508004708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chène, P.</span></span> <span> </span><span class="NLM_article-title">Drugs Targeting Protein-Protein Interactions</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">400</span>– <span class="NLM_lpage">411</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200600004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fcmdc.200600004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=16892375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD28XjslehtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=400-411&author=P.+Ch%C3%A8ne&title=Drugs+Targeting+Protein-Protein+Interactions&doi=10.1002%2Fcmdc.200600004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs targeting protein-protein interactions</span></div><div class="casAuthors">Chene, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Most biol. processes involve permanent and nonpermanent interactions between different proteins, and many protein complexes play a key role in various human diseases.  Therefore, mols. that prevent the formation of these protein complexes could be valuable new therapeutic agents to treat these diseases.  Protein interfaces have not evolved to bind low-mol.-wt. mols., as is the case with enzyme catalytic sites.  It is therefore difficult to identify small compds. that inhibit protein-protein interactions.  However, there is considerable diversity in the structure of protein interfaces, some of which may be more attractive than others for medicinal chem.  One of the main challenges in drug discovery is to identify these interfaces and to exploit their properties to make marketable drugs.  Here, the properties of protein interfaces are discussed in light of their use as drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIvsBJWwqDxrVg90H21EOLACvtfcHk0lj48v2pTTZW_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjslehtbY%253D&md5=7c7951d7caa7f03351493c6b3bdcd517</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600004%26sid%3Dliteratum%253Aachs%26aulast%3DCh%25C3%25A8ne%26aufirst%3DP.%26atitle%3DDrugs%2520Targeting%2520Protein-Protein%2520Interactions%26jtitle%3DChemMedChem%26date%3D2006%26volume%3D1%26spage%3D400%26epage%3D411%26doi%3D10.1002%2Fcmdc.200600004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecules, Big Targets: Drug Discovery Faces the Protein-Protein Interaction Challenge</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">533</span>– <span class="NLM_lpage">550</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.29</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnrd.2016.29" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=27050677" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2016&pages=533-550&author=D.+E.+Scottauthor=A.+R.+Baylyauthor=C.+Abellauthor=J.+Skidmore&title=Small+Molecules%2C+Big+Targets%3A+Drug+Discovery+Faces+the+Protein-Protein+Interaction+Challenge&doi=10.1038%2Fnrd.2016.29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecules, big targets: drug discovery faces the protein-protein interaction challenge</span></div><div class="casAuthors">Scott, Duncan E.; Bayly, Andrew R.; Abell, Chris; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">533-550</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein-protein interactions (PPIs) are of pivotal importance in the regulation of biol. systems and are consequently implicated in the development of disease states.  Recent work has begun to show that, with the right tools, certain classes of PPI can yield to the efforts of medicinal chemists to develop inhibitors, and the first PPI inhibitors have reached clin. development.  In this Review, we describe the research leading to these breakthroughs and highlight the existence of groups of structurally related PPIs within the PPI target class.  For each of these groups, we use examples of successful discovery efforts to illustrate the research strategies that have proved most useful.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK-aGzHa9mP7Vg90H21EOLACvtfcHk0ljCgXu2e8HRjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XlsFOgur4%253D&md5=9f0953567ae9338b8b36aa78b8160655</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.29&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.29%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DBayly%26aufirst%3DA.%2BR.%26aulast%3DAbell%26aufirst%3DC.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSmall%2520Molecules%252C%2520Big%2520Targets%253A%2520Drug%2520Discovery%2520Faces%2520the%2520Protein-Protein%2520Interaction%2520Challenge%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2016%26volume%3D15%26spage%3D533%26epage%3D550%26doi%3D10.1038%2Fnrd.2016.29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Modell, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blosser, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Systematic Targeting of Protein-Protein Interactions Approaches to Targeting Protein-Protein Interactions</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">702</span>– <span class="NLM_lpage">713</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2016.05.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.tips.2016.05.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=27267699" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28XovVagtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=702-713&author=A.+E.+Modellauthor=S.+L.+Blosserauthor=P.+S.+Arora&title=Systematic+Targeting+of+Protein-Protein+Interactions+Approaches+to+Targeting+Protein-Protein+Interactions&doi=10.1016%2Fj.tips.2016.05.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Targeting of Protein-Protein Interactions</span></div><div class="casAuthors">Modell, Ashley E.; Blosser, Sarah L.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">702-713</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Over the past decade, protein-protein interactions (PPIs) have gone from being neglected as 'undruggable' to being considered attractive targets for the development of therapeutics.  Recent advances in computational anal., fragment-based screening, and mol. design have revealed promising strategies to address the basic mol. recognition challenge: how to target large protein surfaces with specificity.  Several systematic and complementary workflows have been developed to yield successful inhibitors of PPIs.  Here we review the major contemporary approaches utilized for the discovery of inhibitors and focus on a structure-based workflow, from the selection of a biol. target to design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDD_smUe5_ErVg90H21EOLACvtfcHk0ljCgXu2e8HRjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XovVagtLo%253D&md5=8b74817b8538b81a3ab0d9b4536fc28e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2016.05.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2016.05.008%26sid%3Dliteratum%253Aachs%26aulast%3DModell%26aufirst%3DA.%2BE.%26aulast%3DBlosser%26aufirst%3DS.%2BL.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DSystematic%2520Targeting%2520of%2520Protein-Protein%2520Interactions%2520Approaches%2520to%2520Targeting%2520Protein-Protein%2520Interactions%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2016%26volume%3D37%26spage%3D702%26epage%3D713%26doi%3D10.1016%2Fj.tips.2016.05.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0li77FCjd2qjgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naylor, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blanco, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cyclic Peptide Natural Products Chart the Frontier of Oral Bioavailability in the Pursuit of Undruggable Targets</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.cbpa.2017.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28570865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=141-147&author=M.+R.+Naylorauthor=A.+T.+Bockusauthor=M.+J.+Blancoauthor=R.+S.+Lokey&title=Cyclic+Peptide+Natural+Products+Chart+the+Frontier+of+Oral+Bioavailability+in+the+Pursuit+of+Undruggable+Targets&doi=10.1016%2Fj.cbpa.2017.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclic peptide natural products chart the frontier of oral bioavailability in the pursuit of undruggable targets</span></div><div class="casAuthors">Naylor, Matthew R.; Bockus, Andrew T.; Blanco, Maria-Jesus; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">141-147</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">As interest in protein-protein interactions and other previously-undruggable targets increases, medicinal chemists are returning to natural products for design inspiration toward mols. that transcend the paradigm of small mol. drugs.  These compds., esp. peptides, often have poor ADME properties and thus require a more nuanced understanding of structure-property relationships to achieve desirable oral bioavailability.  Although there have been few clin. successes in this chem. space to date, recent work has identified opportunities to introduce favorable physicochem. properties to peptidic macrocycles that maintain activity and oral bioavailability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanPrD1D-ePrVg90H21EOLACvtfcHk0li77FCjd2qjgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovVSgsLw%253D&md5=175671ba338608979d38d340866df885</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DNaylor%26aufirst%3DM.%2BR.%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DBlanco%26aufirst%3DM.%2BJ.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCyclic%2520Peptide%2520Natural%2520Products%2520Chart%2520the%2520Frontier%2520of%2520Oral%2520Bioavailability%2520in%2520the%2520Pursuit%2520of%2520Undruggable%2520Targets%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D141%26epage%3D147%26doi%3D10.1016%2Fj.cbpa.2017.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fosgerau, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics: Current Status and Future Directions</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">122</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.drudis.2014.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=25450771" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=122-128&author=K.+Fosgerauauthor=T.+Hoffmann&title=Peptide+Therapeutics%3A+Current+Status+and+Future+Directions&doi=10.1016%2Fj.drudis.2014.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics: current status and future directions</span></div><div class="casAuthors">Fosgerau, Keld; Hoffmann, Torsten</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">122-128</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized for being highly selective and efficacious and, at the same time, relatively safe and well tolerated.  Consequently, there is an increased interest in peptides in pharmaceutical research and development (R&D), and approx. 140 peptide therapeutics are currently being evaluated in clin. trials.  Given that the low-hanging fruits in the form of obvious peptide targets have already been picked, it has now become necessary to explore new routes beyond traditional peptide design.  Examples of such approaches are multifunctional and cell penetrating peptides, as well as peptide drug conjugates.  Here, we discuss the current status, strengths, and weaknesses of peptides as medicines and the emerging new opportunities in peptide drug design and development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmhpp8GnJ6JLVg90H21EOLACvtfcHk0li77FCjd2qjgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2ksLnM&md5=f6b74aaa4e3fdb317b091b0450f3618a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DFosgerau%26aufirst%3DK.%26aulast%3DHoffmann%26aufirst%3DT.%26atitle%3DPeptide%2520Therapeutics%253A%2520Current%2520Status%2520and%2520Future%2520Directions%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D122%26epage%3D128%26doi%3D10.1016%2Fj.drudis.2014.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Northfield, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colless, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaousis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamernig, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design and Synthesis of an Orally Bioavailable Peptide Guided by NMR Amide Temperature Coefficients</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">17504</span>– <span class="NLM_lpage">17509</span>, <span class="refDoi"> DOI: 10.1073/pnas.1417611111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1073%2Fpnas.1417611111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=25416591" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=17504-17509&author=C.+K.+Wangauthor=S.+E.+Northfieldauthor=B.+Collessauthor=S.+Chaousisauthor=I.+Hamernigauthor=R.-J.+Lohmanauthor=D.+S.+Nielsenauthor=C.+I.+Schroederauthor=S.+Lirasauthor=D.+A.+Priceauthor=D.+P.+Fairlieauthor=D.+J.+Craik&title=Rational+Design+and+Synthesis+of+an+Orally+Bioavailable+Peptide+Guided+by+NMR+Amide+Temperature+Coefficients&doi=10.1073%2Fpnas.1417611111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational design and synthesis of an orally bioavailable peptide guided by NMR amide temperature coefficients</span></div><div class="casAuthors">Wang, Conan K.; Northfield, Susan E.; Colless, Barbara; Chaousis, Stephanie; Hamernig, Ingrid; Lohman, Rink-Jan; Nielsen, Daniel S.; Schroeder, Christina I.; Liras, Spiros; Price, David A.; Fairlie, David P.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17504-17509</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Enhancing the oral bioavailability of peptide drug leads is a major challenge in drug design.  As such, methods to address this challenge are highly sought after by the pharmaceutical industry.  Here, we propose a strategy to identify appropriate amides for N-methylation using temp. coeffs. measured by NMR to identify exposed amides in cyclic peptides.  N-methylation effectively caps these amides, modifying the overall solvation properties of the peptides and making them more membrane permeable.  The approach for identifying sites for N-methylation is a rapid alternative to the elucidation of 3D structures of peptide drug leads, which has been a commonly used structure-guided approach in the past.  Five leucine-rich peptide scaffolds are reported with selectively designed N-methylated derivs.  In vitro membrane permeability was assessed by parallel artificial membrane permeability assay and Caco-2 assay.  The most promising N-methylated peptide was then tested in vivo.  Here we report a novel peptide (15), which displayed an oral bioavailability of 33% in a rat model, thus validating the design approach.  We show that this approach can also be used to explain the notable increase in oral bioavailability of a somatostatin analog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ1pwUl6qUw7Vg90H21EOLACvtfcHk0lgQ4heFbV0Ixw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKmu7vE&md5=02803e516bf514e3d941769b7bbf3607</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417611111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417611111%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DC.%2BK.%26aulast%3DNorthfield%26aufirst%3DS.%2BE.%26aulast%3DColless%26aufirst%3DB.%26aulast%3DChaousis%26aufirst%3DS.%26aulast%3DHamernig%26aufirst%3DI.%26aulast%3DLohman%26aufirst%3DR.-J.%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DSchroeder%26aufirst%3DC.%2BI.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DRational%2520Design%2520and%2520Synthesis%2520of%2520an%2520Orally%2520Bioavailable%2520Peptide%2520Guided%2520by%2520NMR%2520Amide%2520Temperature%2520Coefficients%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D17504%26epage%3D17509%26doi%3D10.1073%2Fpnas.1417611111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoang, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohman, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucke, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edmonds, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffith, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rotter, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fairlie, D. P.</span></span> <span> </span><span class="NLM_article-title">Improving on Nature: Making a Cyclic Heptapeptide Orally Bioavailable</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">12059</span>– <span class="NLM_lpage">12063</span>, <span class="refDoi"> DOI: 10.1002/anie.201405364</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fanie.201405364" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFCisrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=12059-12063&author=D.+S.+Nielsenauthor=H.+N.+Hoangauthor=R.+J.+Lohmanauthor=T.+A.+Hillauthor=A.+J.+Luckeauthor=D.+J.+Craikauthor=D.+J.+Edmondsauthor=D.+A.+Griffithauthor=C.+J.+Rotterauthor=R.+B.+Ruggeriauthor=D.+A.+Priceauthor=S.+Lirasauthor=D.+P.+Fairlie&title=Improving+on+Nature%3A+Making+a+Cyclic+Heptapeptide+Orally+Bioavailable&doi=10.1002%2Fanie.201405364"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Improving on Nature: Making a Cyclic Heptapeptide Orally Bioavailable</span></div><div class="casAuthors">Nielsen, Daniel S.; Hoang, Huy N.; Lohman, Rink-Jan; Hill, Timothy A.; Lucke, Andrew J.; Craik, David J.; Edmonds, David J.; Griffith, David A.; Rotter, Charles J.; Ruggeri, Roger B.; Price, David A.; Liras, Spiros; Fairlie, David P.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">12059-12063</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The use of peptides in medicine is limited by low membrane permeability, metabolic instability, high clearance, and negligible oral bioavailability.  The prediction of oral bioavailability of drugs relies on physicochem. properties that favor passive permeability and oxidative metabolic stability, but these may not be useful for peptides.  Here we investigate effects of heterocyclic constraints, intramol. hydrogen bonds, and side chains on the oral bioavailability of cyclic heptapeptides.  NMR-derived structures, amide H-D exchange rates, and temp.-dependent chem. shifts showed that the combination of rigidification, stronger hydrogen bonds, and solvent shielding by branched side chains enhances the oral bioavailability of cyclic heptapeptides in rats without the need for N-methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqefcEjv3-Cb7Vg90H21EOLACvtfcHk0lgQ4heFbV0Ixw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFCisrvJ&md5=18d41a670c2ff65b42fe037ac896ab53</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1002%2Fanie.201405364&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201405364%26sid%3Dliteratum%253Aachs%26aulast%3DNielsen%26aufirst%3DD.%2BS.%26aulast%3DHoang%26aufirst%3DH.%2BN.%26aulast%3DLohman%26aufirst%3DR.%2BJ.%26aulast%3DHill%26aufirst%3DT.%2BA.%26aulast%3DLucke%26aufirst%3DA.%2BJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DEdmonds%26aufirst%3DD.%2BJ.%26aulast%3DGriffith%26aufirst%3DD.%2BA.%26aulast%3DRotter%26aufirst%3DC.%2BJ.%26aulast%3DRuggeri%26aufirst%3DR.%2BB.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DFairlie%26aufirst%3DD.%2BP.%26atitle%3DImproving%2520on%2520Nature%253A%2520Making%2520a%2520Cyclic%2520Heptapeptide%2520Orally%2520Bioavailable%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D12059%26epage%3D12063%26doi%3D10.1002%2Fanie.201405364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Quantifying the Chameleonic Properties of Macrocycles and Other High-Molecular-Weight Drugs</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">712</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.drudis.2016.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=26891978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=712&author=A.+Whittyauthor=M.+Zhongauthor=L.+Viarengoauthor=D.+Beglovauthor=D.+R.+Hallauthor=S.+Vajda&title=Quantifying+the+Chameleonic+Properties+of+Macrocycles+and+Other+High-Molecular-Weight+Drugs&doi=10.1016%2Fj.drudis.2016.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying the chameleonic properties of macrocycles and other high-molecular-weight drugs</span></div><div class="casAuthors">Whitty, Adrian; Zhong, Mengqi; Viarengo, Lauren; Beglov, Dmitri; Hall, David R.; Vajda, Sandor</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">712-717</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Key to the pharmaceutical utility of certain macrocyclic drugs is a 'chameleonic' ability to change their conformation to expose polar groups in aq. soln., but bury them when traversing lipid membranes.  Based on anal. of the structures of 20 macrocyclic compds. that are approved oral drugs, we propose that good soly. requires a topol. polar surface area (TPSA, in Å2) of ≥0.2 × mol. wt. (MW).  Meanwhile, good passive membrane permeability requires a mol. (i.e., 3D) PSA in nonpolar environments of ≤140 Å2.  We show that one or other of these limits is almost invariably violated for compds. with MW > 600 Da, suggesting that some degree of chameleonic behavior is required for most high MW oral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlN7rCqts5wLVg90H21EOLACvtfcHk0ljF_D8SQCvV3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtVKrurs%253D&md5=024476298f7b72efa3f0a9303bed5900</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DM.%26aulast%3DViarengo%26aufirst%3DL.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DQuantifying%2520the%2520Chameleonic%2520Properties%2520of%2520Macrocycles%2520and%2520Other%2520High-Molecular-Weight%2520Drugs%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26spage%3D712%26doi%3D10.1016%2Fj.drudis.2016.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieske, L. H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdélyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Solution Conformations Explain the Chameleonic Behaviour of Macrocyclic Drugs</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">5231</span>– <span class="NLM_lpage">5244</span>, <span class="refDoi"> DOI: 10.1002/chem.201905599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fchem.201905599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=32027758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsVWmsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2020&pages=5231-5244&author=E.+Daneliusauthor=V.+Poongavanamauthor=S.+Peintnerauthor=L.+H.+E.+Wieskeauthor=M.+Erd%C3%A9lyiauthor=J.+Kihlberg&title=Solution+Conformations+Explain+the+Chameleonic+Behaviour+of+Macrocyclic+Drugs&doi=10.1002%2Fchem.201905599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Solution conformations explain the chameleonic behaviour of macrocyclic drugs</span></div><div class="casAuthors">Danelius, Emma; Poongavanam, Vasanthanathan; Peintner, Stefan; Wieske, Lianne H. E.; Erdelyi, Mate; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5231-5244</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">It has been hypothesised that drugs in the chem. space "beyond the rule of 5" (bRo5) must behave as mol. chameleons to combine otherwise conflicting properties, including aq. soly., cell permeability and target binding.  Evidence for this has, however, been limited to the cyclic peptide cyclosporine A.  Herein, we show that the non-peptidic and macrocyclic drugs roxithromycin, telithromycin and spiramycin behave as mol. chameleons, with rifampicin showing a less pronounced behavior.  In particular roxithromycin, telithromycin and spiramycin display a marked, yet limited flexibility and populate significantly less polar and more compact conformational ensembles in an apolar than in a polar environment.  In addn. to balancing of membrane permeability and aq. soly., this flexibility also allows binding to targets that vary in structure between species.  The drugs' passive cell permeability correlates to their 3D polar surface area and corroborate two theor. models for permeability, developed for cyclic peptides.  We conclude that mol. chameleonicity should be incorporated in the design of orally administered drugs in the bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZJsCKr3Rq-rVg90H21EOLACvtfcHk0ljF_D8SQCvV3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsVWmsLo%253D&md5=353528feef6aa1130f8a16204a9f40ce</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fchem.201905599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201905599%26sid%3Dliteratum%253Aachs%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DWieske%26aufirst%3DL.%2BH.%2BE.%26aulast%3DErd%25C3%25A9lyi%26aufirst%3DM.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DSolution%2520Conformations%2520Explain%2520the%2520Chameleonic%2520Behaviour%2520of%2520Macrocyclic%2520Drugs%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2020%26volume%3D26%26spage%3D5231%26epage%3D5244%26doi%3D10.1002%2Fchem.201905599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mühlbauer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Interconversion Rates between Conformational States as Rationale for the Membrane Permeability of Cyclosporines</span>. <i>ChemPhysChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3309</span>– <span class="NLM_lpage">3314</span>, <span class="refDoi"> DOI: 10.1002/cphc.201700995</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fcphc.201700995" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28921848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslCmur%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=3309-3314&author=J.+Witekauthor=M.+M%C3%BChlbauerauthor=B.+G.+Kellerauthor=M.+Blatterauthor=A.+Meissnerauthor=T.+Wagnerauthor=S.+Riniker&title=Interconversion+Rates+between+Conformational+States+as+Rationale+for+the+Membrane+Permeability+of+Cyclosporines&doi=10.1002%2Fcphc.201700995"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Interconversion rates between conformational states as rationale for the membrane permeability of cyclosporins</span></div><div class="casAuthors">Witek, Jagna; Muehlbauer, Max; Keller, Bettina G.; Blatter, Markus; Meissner, Axel; Wagner, Trixie; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">ChemPhysChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3309-3314</span>CODEN:
                <span class="NLM_cas:coden">CPCHFT</span>;
        ISSN:<span class="NLM_cas:issn">1439-4235</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Cyclic peptides have regained interest as potential inhibitors of challenging targets but have often a low bioavailability.  The natural product, cyclosporin A (CsA) is the textbook exception.  Despite its size and polar backbone, it is able to passively cross membranes.  This ability is hypothesized to be due to a conformational change from the low-energy conformation in water to a "congruent" conformation that is populated both in water and inside the membrane.  Here, we used a combination of NMR measurements and kinetic models based on mol. dynamics simulations to rationalize the difference in the membrane permeability of cyclosporin E (CsE) and CsA.  The structure of CsE differed only in a backbone methylation, but its membrane permeability was one order of magnitude lower.  The most striking difference was found in the interconversion rates between the conformational states favored in water and in CHCl3, which are up to one order of magnitude slower for CsE compared to CsA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLejkF8vE_e7Vg90H21EOLACvtfcHk0ljF_D8SQCvV3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslCmur%252FL&md5=f80741cec728a4852aeb411e64e5c784</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fcphc.201700995&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcphc.201700995%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DM%25C3%25BChlbauer%26aufirst%3DM.%26aulast%3DKeller%26aufirst%3DB.%2BG.%26aulast%3DBlatter%26aufirst%3DM.%26aulast%3DMeissner%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DT.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DInterconversion%2520Rates%2520between%2520Conformational%2520States%2520as%2520Rationale%2520for%2520the%2520Membrane%2520Permeability%2520of%2520Cyclosporines%26jtitle%3DChemPhysChem%26date%3D2017%26volume%3D18%26spage%3D3309%26epage%3D3314%26doi%3D10.1002%2Fcphc.201700995" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Räder, A. F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmüller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Improving Oral Bioavailability of Cyclic Peptides by N-Methylation</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2766</span>– <span class="NLM_lpage">2773</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.08.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.bmc.2017.08.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=28886995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC1cbmt1Wgsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2766-2773&author=A.+F.+B.+R%C3%A4derauthor=F.+Reichartauthor=M.+Weinm%C3%BCllerauthor=H.+Kessler&title=Improving+Oral+Bioavailability+of+Cyclic+Peptides+by+N-Methylation&doi=10.1016%2Fj.bmc.2017.08.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Improving oral bioavailability of cyclic peptides by N-methylation</span></div><div class="casAuthors">Rader Andreas F B; Reichart Florian; Weinmuller Michael; Kessler Horst</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & medicinal chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2766-2773</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The renaissance of peptides in pharmaceutical industry results from their importance in many biological functions.  However, low metabolic stability and the lack of oral availability of most peptides is a certain limitation.  Whereas metabolic instability may be often overcome by development of small cyclic peptides containing d-amino acids, the very low oral availability of most peptides is a serious limitation for some medicinal applications.  The situation is complicated because a twofold optimization - biological activity and oral availability - is required to overcome this problem.  Moreover, most simple "rules" for achieving oral availability are not general and are applicable only to limited cases.  Many structural modifications for increasing biological activities and metabolic stabilities of cyclic peptides have been described, of which N-alkylation is probably the most common.  This mini-review focuses on the effects of N-methylation of cyclic peptides in strategies to optimize bioavailabilities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRHNydRSBPhBmHOrQ6qeX-mfW6udTcc2eYN_0cu8_htnLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbmt1Wgsw%253D%253D&md5=b9e5f76b838ace1fcddd555d5dd3fed2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.08.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.08.031%26sid%3Dliteratum%253Aachs%26aulast%3DR%25C3%25A4der%26aufirst%3DA.%2BF.%2BB.%26aulast%3DReichart%26aufirst%3DF.%26aulast%3DWeinm%25C3%25BCller%26aufirst%3DM.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DImproving%2520Oral%2520Bioavailability%2520of%2520Cyclic%2520Peptides%2520by%2520N-Methylation%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2766%26epage%3D2773%26doi%3D10.1016%2Fj.bmc.2017.08.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzelman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rand, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEwen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelev, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauman, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">On-Resin N-Methylation of Cyclic Peptides for Discovery of Orally Bioavailable Scaffolds</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">810</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1038/nchembio.664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnchembio.664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=21946276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=810-817&author=T.+R.+Whiteauthor=C.+M.+Renzelmanauthor=A.+C.+Randauthor=T.+Rezaiauthor=C.+M.+McEwenauthor=V.+M.+Gelevauthor=R.+A.+Turnerauthor=R.+G.+Liningtonauthor=S.+S.+F.+Leungauthor=A.+S.+Kalgutkarauthor=J.+N.+Baumanauthor=Y.+Zhangauthor=S.+Lirasauthor=D.+A.+Priceauthor=A.+M.+Mathiowetzauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=On-Resin+N-Methylation+of+Cyclic+Peptides+for+Discovery+of+Orally+Bioavailable+Scaffolds&doi=10.1038%2Fnchembio.664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</span></div><div class="casAuthors">White, Tina R.; Renzelman, Chad M.; Rand, Arthur C.; Rezai, Taha; McEwen, Cayla M.; Gelev, Vladimir M.; Turner, Rushia A.; Linington, Roger G.; Leung, Siegfried S. F.; Kalgutkar, Amit S.; Bauman, Jonathan N.; Zhang, Yizhong; Liras, Spiros; Price, David A.; Mathiowetz, Alan M.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">810-817</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Backbone N-methylation is common among peptide natural products and has a substantial impact on both the phys. properties and the conformational states of cyclic peptides.  However, the specific impact of N-methylation on passive membrane diffusion in cyclic peptides has not been investigated systematically.  Here we report a method for the selective, on-resin N-methylation of cyclic peptides to generate compds. with drug-like membrane permeability and oral bioavailability.  The selectivity and degree of N-methylation of the cyclic peptide was dependent on backbone stereochem., suggesting that conformation dictates the regiochem. of the N-methylation reaction.  The permeabilities of the N-Me variants were corroborated by computational studies on a 1024-member virtual library of N-Me cyclic peptides.  One of the most permeable compds., a cyclic hexapeptide (mol. mass = 755 Da) with three N-Me groups, showed an oral bioavailability of 28% in rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreigsRodBtOLVg90H21EOLACvtfcHk0ljJ0STYW4pb1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL&md5=6f9f01f4ab523bb46cc0198cc8f1c34e</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.664%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DT.%2BR.%26aulast%3DRenzelman%26aufirst%3DC.%2BM.%26aulast%3DRand%26aufirst%3DA.%2BC.%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DMcEwen%26aufirst%3DC.%2BM.%26aulast%3DGelev%26aufirst%3DV.%2BM.%26aulast%3DTurner%26aufirst%3DR.%2BA.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DBauman%26aufirst%3DJ.%2BN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DOn-Resin%2520N-Methylation%2520of%2520Cyclic%2520Peptides%2520for%2520Discovery%2520of%2520Orally%2520Bioavailable%2520Scaffolds%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D810%26epage%3D817%26doi%3D10.1038%2Fnchembio.664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran, M. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberg, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">12125</span>– <span class="NLM_lpage">12133</span>, <span class="refDoi"> DOI: 10.1021/ja303200d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303200d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=12125-12133&author=J.+G.+Beckauthor=J.+Chatterjeeauthor=B.+Lauferauthor=M.+U.+Kiranauthor=A.+O.+Frankauthor=S.+Neubauerauthor=O.+Ovadiaauthor=S.+Greenbergauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Intestinal+Permeability+of+Cyclic+Peptides%3A+Common+Key+Backbone+Motifs+Identified&doi=10.1021%2Fja303200d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Intestinal Permeability of Cyclic Peptides: Common Key Backbone Motifs Identified</span></div><div class="casAuthors">Beck, Johannes G.; Chatterjee, Jayanta; Laufer, Burkhardt; Kiran, Marelli Udaya; Frank, Andreas O.; Neubauer, Stefanie; Ovadia, Oded; Greenberg, Sarit; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">12125-12133</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Insufficient oral bioavailability is considered as a key limitation for the widespread development of peptides as therapeutics.  While the oral bioavailability of small org. compds. is often estd. from simple rules, similar rules do not apply to peptides, and even the high oral bioavailability that is described for a small no. of peptides is not well understood.  Here we present two highly Caco-2 permeable template structures based on a library of 54 cyclo(-d-Ala-Ala5-) peptides with different N-methylation patterns.  The first all-trans template structure possesses two β-turns of type II along Ala6-d-Ala1 and Ala3-Ala4 and is only found for one peptide with two N-Me groups at d-Ala1 and Ala6 [NMe(1,6)].  The second single-cis template possesses a characteristic cis peptide bond preceding Ala5, which results in type VI β-turn geometry along Ala4-Ala5.  Although the second template structure is found in seven peptides carrying N-Me groups on Ala5, high Caco-2 permeability is only found for a subgroup of two of them [NMe(1,5) and NMe(1,2,4,5)], suggesting that N-methylation of d-Ala1 is a prerequisite for high permeability of the second template structure.  The structural similarity of the second template structure with the orally bioavailable somatostatin analog cyclo(-Pro-Phe-NMe-d-Trp-NMe-Lys-Thr-NMe-Phe-), and the striking resemblance with both β-turns of the orally bioavailable peptide cyclosporine A, suggests that the introduction of bioactive sequences on the highly Caco-2 permeable templates may result in potent orally bioavailable drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4AC8OVfXJMbVg90H21EOLACvtfcHk0lhgdfANZQl6Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1Cisbw%253D&md5=ecf4dc5aa62a8ebb7c5cc99e8009f528</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja303200d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303200d%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DJ.%2BG.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DLaufer%26aufirst%3DB.%26aulast%3DKiran%26aufirst%3DM.%2BU.%26aulast%3DFrank%26aufirst%3DA.%2BO.%26aulast%3DNeubauer%26aufirst%3DS.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DGreenberg%26aufirst%3DS.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DIntestinal%2520Permeability%2520of%2520Cyclic%2520Peptides%253A%2520Common%2520Key%2520Backbone%2520Motifs%2520Identified%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D12125%26epage%3D12133%26doi%3D10.1021%2Fja303200d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Biron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatterjee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovadia, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langenegger, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoyer, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jelinek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Improving Oral Bioavailability of Peptides by Multiple N-Methylation: Somatostatin Analogues</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">2595</span>– <span class="NLM_lpage">2599</span>, <span class="refDoi"> DOI: 10.1002/anie.200705797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fanie.200705797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2008&pages=2595-2599&author=E.+Bironauthor=J.+Chatterjeeauthor=O.+Ovadiaauthor=D.+Langeneggerauthor=J.+Brueggenauthor=D.+Hoyerauthor=H.+A.+Schmidauthor=R.+Jelinekauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Improving+Oral+Bioavailability+of+Peptides+by+Multiple+N-Methylation%3A+Somatostatin+Analogues&doi=10.1002%2Fanie.200705797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Improving oral bioavailability of peptides by multiple N-methylation: somatostatin analogues</span></div><div class="casAuthors">Biron, Eric; Chatterjee, Jayanta; Ovadia, Oded; Langenegger, Daniel; Brueggen, Joseph; Hoyer, Daniel; Schmid, Herbert A.; Jelnick, Raz; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2595-2599</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A complete library of the N-methylated somatostatin cyclopeptidic analog Veber-Hirschmann peptide cyclo(-PFwKTF-) has been prepd. with the aim of improving its bioavailability.  Several analogs from the library were found to bind to the somatostatin receptor in the nanomolar range and one of them shows a significant oral bioavailability of 10%.  Conformational anal. shows that N-methylation is allowed at specific positions without affecting the bioactive conformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo64OMbjkAtErVg90H21EOLACvtfcHk0lhgdfANZQl6Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFartLw%253D&md5=42c312786da82941dde6671b1130b61b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fanie.200705797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.200705797%26sid%3Dliteratum%253Aachs%26aulast%3DBiron%26aufirst%3DE.%26aulast%3DChatterjee%26aufirst%3DJ.%26aulast%3DOvadia%26aufirst%3DO.%26aulast%3DLangenegger%26aufirst%3DD.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DHoyer%26aufirst%3DD.%26aulast%3DSchmid%26aufirst%3DH.%2BA.%26aulast%3DJelinek%26aufirst%3DR.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DImproving%2520Oral%2520Bioavailability%2520of%2520Peptides%2520by%2520Multiple%2520N-Methylation%253A%2520Somatostatin%2520Analogues%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2008%26volume%3D47%26spage%3D2595%26epage%3D2599%26doi%3D10.1002%2Fanie.200705797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">White, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renzelman, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rand, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcewen, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelev, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linington, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amit, S.</span></span> <span> </span><span class="NLM_article-title">On-Resin N-Methylation of Cyclic Peptides for Discovery of Orally Bioavailable Scaffolds</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">810</span>– <span class="NLM_lpage">817</span>, <span class="refDoi"> DOI: 10.1038/nchembio.664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnchembio.664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=21946276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=810-817&author=T.+R.+Whiteauthor=C.+M.+Renzelmanauthor=A.+C.+Randauthor=T.+Rezaiauthor=C.+M.+Mcewenauthor=V.+M.+Gelevauthor=R.+A.+Turnerauthor=R.+G.+Liningtonauthor=S.+S.+F.+Leungauthor=S.+Amit&title=On-Resin+N-Methylation+of+Cyclic+Peptides+for+Discovery+of+Orally+Bioavailable+Scaffolds&doi=10.1038%2Fnchembio.664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">On-resin N-methylation of cyclic peptides for discovery of orally bioavailable scaffolds</span></div><div class="casAuthors">White, Tina R.; Renzelman, Chad M.; Rand, Arthur C.; Rezai, Taha; McEwen, Cayla M.; Gelev, Vladimir M.; Turner, Rushia A.; Linington, Roger G.; Leung, Siegfried S. F.; Kalgutkar, Amit S.; Bauman, Jonathan N.; Zhang, Yizhong; Liras, Spiros; Price, David A.; Mathiowetz, Alan M.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">810-817</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Backbone N-methylation is common among peptide natural products and has a substantial impact on both the phys. properties and the conformational states of cyclic peptides.  However, the specific impact of N-methylation on passive membrane diffusion in cyclic peptides has not been investigated systematically.  Here we report a method for the selective, on-resin N-methylation of cyclic peptides to generate compds. with drug-like membrane permeability and oral bioavailability.  The selectivity and degree of N-methylation of the cyclic peptide was dependent on backbone stereochem., suggesting that conformation dictates the regiochem. of the N-methylation reaction.  The permeabilities of the N-Me variants were corroborated by computational studies on a 1024-member virtual library of N-Me cyclic peptides.  One of the most permeable compds., a cyclic hexapeptide (mol. mass = 755 Da) with three N-Me groups, showed an oral bioavailability of 28% in rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreigsRodBtOLVg90H21EOLACvtfcHk0lhgdfANZQl6Lw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1anurjL&md5=6f9f01f4ab523bb46cc0198cc8f1c34e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.664%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DT.%2BR.%26aulast%3DRenzelman%26aufirst%3DC.%2BM.%26aulast%3DRand%26aufirst%3DA.%2BC.%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DMcewen%26aufirst%3DC.%2BM.%26aulast%3DGelev%26aufirst%3DV.%2BM.%26aulast%3DTurner%26aufirst%3DR.%2BA.%26aulast%3DLinington%26aufirst%3DR.%2BG.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DAmit%26aufirst%3DS.%26atitle%3DOn-Resin%2520N-Methylation%2520of%2520Cyclic%2520Peptides%2520for%2520Discovery%2520of%2520Orally%2520Bioavailable%2520Scaffolds%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D810%26epage%3D817%26doi%3D10.1038%2Fnchembio.664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkeley, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2928</span>– <span class="NLM_lpage">2931</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=2928-2931&author=J.+Schwochertauthor=R.+Turnerauthor=M.+Thangauthor=R.+F.+Berkeleyauthor=A.+R.+Ponkeyauthor=K.+M.+Rodriguezauthor=S.+S.+F.+Leungauthor=B.+Khunteauthor=G.+Goetzauthor=C.+Limberakisauthor=A.+S.+Kalgutkarauthor=H.+Engauthor=M.+J.+Shapiroauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Peptide+to+Peptoid+Substitutions+Increase+Cell+Permeability+in+Cyclic+Hexapeptides&doi=10.1021%2Facs.orglett.5b01162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span></div><div class="casAuthors">Schwochert, Joshua; Turner, Rushia; Thang, Melissa; Berkeley, Ray F.; Ponkey, Alexandra R.; Rodriguez, Kelsie M.; Leung, Siegfried S. F.; Khunte, Bhagyashree; Goetz, Gilles; Limberakis, Chris; Kalgutkar, Amit S.; Eng, Heather; Shapiro, Michael J.; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2928-2931</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of peptide-to-peptoid substitutions on the passive membrane permeability of an N-methylated cyclic hexapeptide is examd.  In general, substitutions maintained permeability but increased conformational heterogeneity.  Diversification with nonproteinogenic side chains increased permeability up to 3-fold.  Addnl., the conformational impact of peptoid substitutions within a β-turn are explored.  Based on these results, the strategic incorporation of peptoid residues into cyclic peptides can maintain or improve cell permeability, while increasing access to diverse side-chain functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUG1_hYdVn7Vg90H21EOLACvtfcHk0lhatXMr661rqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D&md5=156985175ddcafd34a67e81713934040</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01162%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DThang%26aufirst%3DM.%26aulast%3DBerkeley%26aufirst%3DR.%2BF.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DRodriguez%26aufirst%3DK.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520to%2520Peptoid%2520Substitutions%2520Increase%2520Cell%2520Permeability%2520in%2520Cyclic%2520Hexapeptides%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D2928%26epage%3D2931%26doi%3D10.1021%2Facs.orglett.5b01162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sui, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabenstein, D. L.</span></span> <span> </span><span class="NLM_article-title">Kinetics and Equilibria of Cis/Trans Isomerization of Backbone Amide Bonds in Peptoids</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">12042</span>– <span class="NLM_lpage">12048</span>, <span class="refDoi"> DOI: 10.1021/ja0740925</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0740925" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvFOhs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2007&pages=12042-12048&author=Q.+Suiauthor=D.+Borchardtauthor=D.+L.+Rabenstein&title=Kinetics+and+Equilibria+of+Cis%2FTrans+Isomerization+of+Backbone+Amide+Bonds+in+Peptoids&doi=10.1021%2Fja0740925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetics and Equilibria of Cis/Trans Isomerization of Backbone Amide Bonds in Peptoids</span></div><div class="casAuthors">Sui, Qiang; Borchardt, Dan; Rabenstein, Dallas L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12042-12048</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The biol. activities of N-substituted glycine oligomers (peptoids) have been the subject of extensive research.  As compared to peptides, both the cis and trans conformations of the backbone amide bonds of peptoids can be significantly populated.  Thus, peptoids are mixts. of configurational isomers, with the no. of isomers increasing by a factor of 2 with each addnl. monomer residue.  Here, the authors report the results of a study of the kinetics and equil. of cis/trans isomerization of the amide bonds of N-acetylated peptoid monomers, dipeptoids, and tripeptoids by NMR spectroscopy.  Resonance intensities indicate the cis conformation of the backbone amide bonds of the peptoids studied is more populated than is generally the case for the analogous secondary amide bond to proline residues in acyclic peptides.  Rate consts. were measured by inversion-magnetization transfer techniques over a range of temps., and activation parameters were derived from the temp. dependence of the rate consts.  The rate of cis/trans isomerization by rotation around the amide bonds in the peptoids studied is generally slower than that around amide bonds to proline residues and takes place on the NMR inversion-magnetization transfer time scale only by rotation around the amide bond to the C-terminal peptoid residue.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYolNa9NHLi7Vg90H21EOLACvtfcHk0lhatXMr661rqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvFOhs70%253D&md5=f9c41f1342aa801b453a37fbf215acab</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1021%2Fja0740925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0740925%26sid%3Dliteratum%253Aachs%26aulast%3DSui%26aufirst%3DQ.%26aulast%3DBorchardt%26aufirst%3DD.%26aulast%3DRabenstein%26aufirst%3DD.%2BL.%26atitle%3DKinetics%2520and%2520Equilibria%2520of%2520Cis%252FTrans%2520Isomerization%2520of%2520Backbone%2520Amide%2520Bonds%2520in%2520Peptoids%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2007%26volume%3D129%26spage%3D12042%26epage%3D12048%26doi%3D10.1021%2Fja0740925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Culf, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, R. J.</span></span> <span> </span><span class="NLM_article-title">Solid-Phase Synthesis of N-Substituted Glycine Oligomers (α-Peptoids) and Derivatives</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5282</span>– <span class="NLM_lpage">5335</span>, <span class="refDoi"> DOI: 10.3390/molecules15085282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.3390%2Fmolecules15085282" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=20714299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVagt7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2010&pages=5282-5335&author=A.+S.+Culfauthor=R.+J.+Ouellette&title=Solid-Phase+Synthesis+of+N-Substituted+Glycine+Oligomers+%28%CE%B1-Peptoids%29+and+Derivatives&doi=10.3390%2Fmolecules15085282"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Solid-phase synthesis of N-substituted glycine oligomers (α-peptoids) and derivatives</span></div><div class="casAuthors">Culf, Adrian S.; Ouellette, Rodney J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">5282-5335</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">Molecular Diversity Preservation International</span>)
        </div><div class="casAbstract">A review.  Peptoids (N-substituted polyglycines and extended peptoids with variant backbone amino-acid monomer units) are oligomeric synthetic polymers that are becoming a valuable mol. tool in the biosciences.  The exploration of peptoids in drug design and evaluation of their secondary structures are of particular interest.  Major advantages of peptoids as research and pharmaceutical tools include the ease and economy of synthesis, highly variable backbone and side-chain chem. possibilities.  In addn., peptoids have been shown to be highly active in biol. systems while resistant to proteolytic decay.  This review with 227 refs. discusses the work done by R. N. Zuckermann et al. from 1992 to 2010 on solid-phase synthesis and biochem. applications of peptoids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv9ZaCWTLgMrVg90H21EOLACvtfcHk0lindEJOL_JreA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVagt7fL&md5=907128401ccbbfc77d78e64042b76ac1</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3390%2Fmolecules15085282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules15085282%26sid%3Dliteratum%253Aachs%26aulast%3DCulf%26aufirst%3DA.%2BS.%26aulast%3DOuellette%26aufirst%3DR.%2BJ.%26atitle%3DSolid-Phase%2520Synthesis%2520of%2520N-Substituted%2520Glycine%2520Oligomers%2520%2528%25CE%25B1-Peptoids%2529%2520and%2520Derivatives%26jtitle%3DMolecules%26date%3D2010%26volume%3D15%26spage%3D5282%26epage%3D5335%26doi%3D10.3390%2Fmolecules15085282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Toward the Elucidation of the Mechanism for Passive Membrane Permeability of Cyclic Peptides</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">637</span>– <span class="NLM_lpage">639</span>, <span class="refDoi"> DOI: 10.4155/fmc-2018-0429</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.4155%2Ffmc-2018-0429" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=30920310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1MXpvFals78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=637-639&author=S.+Riniker&title=Toward+the+Elucidation+of+the+Mechanism+for+Passive+Membrane+Permeability+of+Cyclic+Peptides&doi=10.4155%2Ffmc-2018-0429"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the elucidation of the mechanism for passive membrane permeability of cyclic peptides</span></div><div class="casAuthors">Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">637-639</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpb9bmRbGqnybVg90H21EOLACvtfcHk0lindEJOL_JreA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXpvFals78%253D&md5=36f67a8e4c65f04cc8caec625ba85152</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.4155%2Ffmc-2018-0429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc-2018-0429%26sid%3Dliteratum%253Aachs%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DToward%2520the%2520Elucidation%2520of%2520the%2520Mechanism%2520for%2520Passive%2520Membrane%2520Permeability%2520of%2520Cyclic%2520Peptides%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2019%26volume%3D11%26spage%3D637%26epage%3D639%26doi%3D10.4155%2Ffmc-2018-0429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schroeder, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dounas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fouché, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemke, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Rationalization of the Membrane Permeability Differences in a Series of Analogue Cyclic Decapeptides</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">294</span>– <span class="NLM_lpage">308</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00485</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00485" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=294-308&author=J.+Witekauthor=S.+Wangauthor=B.+Schroederauthor=R.+Lingwoodauthor=A.+Dounasauthor=H.+J.+Rothauthor=M.+Fouch%C3%A9author=M.+Blatterauthor=O.+Lemkeauthor=B.+Kellerauthor=S.+Riniker&title=Rationalization+of+the+Membrane+Permeability+Differences+in+a+Series+of+Analogue+Cyclic+Decapeptides&doi=10.1021%2Facs.jcim.8b00485"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rationalization of the Membrane Permeability Differences in a Series of Analogue Cyclic Decapeptides</span></div><div class="casAuthors">Witek, Jagna; Wang, Shuzhe; Schroeder, Benjamin; Lingwood, Robin; Dounas, Andreas; Roth, Hans-Jorg; Fouche, Marianne; Blatter, Markus; Lemke, Oliver; Keller, Bettina; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">294-308</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclization and selected backbone N-methylations are found to be often necessary but not sufficient conditions for peptidic drugs to have a good bioavailability.  Thus, the design of cyclic peptides with good passive membrane permeability and good soly. remains a challenge.  The backbone scaffold of a recently published series of cyclic decapeptides with six selected backbone N-methylations was designed to favor the adoption of a closed conformation with β-turns and four transannular hydrogen bonds.  Although this conformation was indeed adopted by the peptides as detd. by NMR measurements, substantial differences in the membrane permeability were obsd.  In this work, we aim to rationalize the impact of discrete side chain modifications on membrane permeability for six of these cyclic decapeptides.  The thermodn. and kinetic properties were investigated using mol. dynamics simulations and Markov state modeling in water and chloroform.  The study highlights the influence that side-chain modifications can have on the backbone conformation.  Peptides with a D-proline in the β-turns were more likely to adopt, even in water, the closed conformation with transannular hydrogen bonds, which facilitates transition through the membrane.  The population of the closed conformation in water was found to correlate pos. with PAMPA log Pe.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGnkfImrV0ObVg90H21EOLACvtfcHk0lindEJOL_JreA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitbzO&md5=9d1d3e3133734870652778bd67c92694</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00485&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00485%26sid%3Dliteratum%253Aachs%26aulast%3DWitek%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSchroeder%26aufirst%3DB.%26aulast%3DLingwood%26aufirst%3DR.%26aulast%3DDounas%26aufirst%3DA.%26aulast%3DRoth%26aufirst%3DH.%2BJ.%26aulast%3DFouch%25C3%25A9%26aufirst%3DM.%26aulast%3DBlatter%26aufirst%3DM.%26aulast%3DLemke%26aufirst%3DO.%26aulast%3DKeller%26aufirst%3DB.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DRationalization%2520of%2520the%2520Membrane%2520Permeability%2520Differences%2520in%2520a%2520Series%2520of%2520Analogue%2520Cyclic%2520Decapeptides%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D294%26epage%3D308%26doi%3D10.1021%2Facs.jcim.8b00485" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benakli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaubien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Déziel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, G.</span></span> <span> </span><span class="NLM_article-title">Potent Macrocyclic Antagonists to the Motilin Receptor Presenting Novel Unnatural Amino Acids</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4187</span>– <span class="NLM_lpage">4190</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.05.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.bmcl.2007.05.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=17533127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsVKntrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2007&pages=4187-4190&author=E.+Marsaultauthor=K.+Benakliauthor=S.+Beaubienauthor=C.+Saint-Louisauthor=R.+D%C3%A9zielauthor=G.+Fraser&title=Potent+Macrocyclic+Antagonists+to+the+Motilin+Receptor+Presenting+Novel+Unnatural+Amino+Acids&doi=10.1016%2Fj.bmcl.2007.05.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Potent macrocyclic antagonists to the motilin receptor presenting novel unnatural amino acids</span></div><div class="casAuthors">Marsault, Eric; Benakli, Kamel; Beaubien, Sylvie; Saint-Louis, Carl; Deziel, Robert; Fraser, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">4187-4190</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Novel, potent small mol. motilin receptor antagonists are described.  These peptidomimetic macrocycles, e.g., I (X = OH, Cl, CF3; R = CH2NH2, NH2, NHPr-i, NHCONH2, 2-pyrimidinylamino, pyrrolidino), are composed of a tripeptide cyclized backbone-to-backbone with a nonpeptidic tether and bear new unnatural amino acids contg. basic side chains.  I (X = 3-CF3, R = pyrrolidino) was the most potent motilin receptor antagonist (Ki = 18 nM, IC50 = 2 nM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Hm7_Tmr1A7Vg90H21EOLACvtfcHk0lh2krq7U59eBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsVKntrw%253D&md5=8e2ca54b6e6a94e227abdfe883dc2d64</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.05.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.05.043%26sid%3Dliteratum%253Aachs%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DBenakli%26aufirst%3DK.%26aulast%3DBeaubien%26aufirst%3DS.%26aulast%3DSaint-Louis%26aufirst%3DC.%26aulast%3DD%25C3%25A9ziel%26aufirst%3DR.%26aulast%3DFraser%26aufirst%3DG.%26atitle%3DPotent%2520Macrocyclic%2520Antagonists%2520to%2520the%2520Motilin%2520Receptor%2520Presenting%2520Novel%2520Unnatural%2520Amino%2520Acids%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2007%26volume%3D17%26spage%3D4187%26epage%3D4190%26doi%3D10.1016%2Fj.bmcl.2007.05.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoveyda, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vézina, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landry, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benakli, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beaubien, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saint-Louis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinault, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellet, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaseshan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foucher, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beauchemin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhérer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veber, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, G. L.</span></span> <span> </span><span class="NLM_article-title">Optimization of the Potency and Pharmacokinetic Properties of a Macrocyclic Ghrelin Receptor Agonist (Part I): Development of Ulimorelin (TZP-101) from Hit to Clinic</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8305</span>– <span class="NLM_lpage">8320</span>, <span class="refDoi"> DOI: 10.1021/jm2007062</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2007062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC38%252Fms1antg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8305-8320&author=H.+R.+Hoveydaauthor=E.+Marsaultauthor=R.+Gagnonauthor=A.+P.+Mathieuauthor=M.+V%C3%A9zinaauthor=A.+Landryauthor=Z.+Wangauthor=K.+Benakliauthor=S.+Beaubienauthor=C.+Saint-Louisauthor=M.+Brassardauthor=J.+F.+Pinaultauthor=L.+Ouelletauthor=S.+Bhatauthor=M.+Ramaseshanauthor=X.+Pengauthor=L.+Foucherauthor=S.+Beaucheminauthor=P.+Bh%C3%A9rerauthor=D.+F.+Veberauthor=M.+L.+Petersonauthor=G.+L.+Fraser&title=Optimization+of+the+Potency+and+Pharmacokinetic+Properties+of+a+Macrocyclic+Ghrelin+Receptor+Agonist+%28Part+I%29%3A+Development+of+Ulimorelin+%28TZP-101%29+from+Hit+to+Clinic&doi=10.1021%2Fjm2007062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of the potency and pharmacokinetic properties of a macrocyclic ghrelin receptor agonist (Part I): Development of ulimorelin (TZP-101) from hit to clinic</span></div><div class="casAuthors">Hoveyda Hamid R; Marsault Eric; Gagnon Rene; Mathieu Axel P; Vezina Martin; Landry Annick; Wang Zhigang; Benakli Kamel; Beaubien Sylvie; Saint-Louis Carl; Brassard Martin; Pinault Jean-Francois; Ouellet Luc; Bhat Shridhar; Ramaseshan Mahesh; Peng Xiaowen; Foucher Laurence; Beauchemin Sophie; Bherer Patrick; Veber Daniel F; Peterson Mark L; Fraser Graeme L</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8305-20</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">High-throughput screening of Tranzyme Pharma's proprietary macrocycle library using the aequorin Ca2+-bioluminescence assay against the human ghrelin receptor (GRLN) led to the discovery of novel agonists against this G-protein coupled receptor.  Early hits such as 1 (Ki=86 nM, EC50=134 nM) though potent in vitro displayed poor pharmacokinetic properties that required optimization.  While such macrocycles are not fully rule-of-five compliant, principally due to their molecular weight and clogP, optimization of their pharmacokinetic properties proved feasible largely through conformational rigidification.  Extensive SAR led to the identification of 2 (Ki=16 nM, EC50=29 nM), also known as ulimorelin or TZP-101, which has progressed to phase III human clinical trials for the treatment of postoperative ileus.  X-ray structure and detailed NMR studies indicated a rigid peptidomimetic portion in 2 that is best defined as a nonideal type-I' β-turn.  Compound 2 is 24% orally bioavailable in both rats and monkeys.  Despite its potency, in vitro and in gastric emptying studies, 2 did not induce growth hormone (GH) release in rats, thus demarcating the GH versus GI pharmacology of GRLN.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREoes2wBiC73y-c-E1v-fmfW6udTcc2eYidQphkA89drntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252Fms1antg%253D%253D&md5=bdc3b14ed821c61c4fecea50c60f0b07</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm2007062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2007062%26sid%3Dliteratum%253Aachs%26aulast%3DHoveyda%26aufirst%3DH.%2BR.%26aulast%3DMarsault%26aufirst%3DE.%26aulast%3DGagnon%26aufirst%3DR.%26aulast%3DMathieu%26aufirst%3DA.%2BP.%26aulast%3DV%25C3%25A9zina%26aufirst%3DM.%26aulast%3DLandry%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DBenakli%26aufirst%3DK.%26aulast%3DBeaubien%26aufirst%3DS.%26aulast%3DSaint-Louis%26aufirst%3DC.%26aulast%3DBrassard%26aufirst%3DM.%26aulast%3DPinault%26aufirst%3DJ.%2BF.%26aulast%3DOuellet%26aufirst%3DL.%26aulast%3DBhat%26aufirst%3DS.%26aulast%3DRamaseshan%26aufirst%3DM.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DFoucher%26aufirst%3DL.%26aulast%3DBeauchemin%26aufirst%3DS.%26aulast%3DBh%25C3%25A9rer%26aufirst%3DP.%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DPeterson%26aufirst%3DM.%2BL.%26aulast%3DFraser%26aufirst%3DG.%2BL.%26atitle%3DOptimization%2520of%2520the%2520Potency%2520and%2520Pharmacokinetic%2520Properties%2520of%2520a%2520Macrocyclic%2520Ghrelin%2520Receptor%2520Agonist%2520%2528Part%2520I%2529%253A%2520Development%2520of%2520Ulimorelin%2520%2528TZP-101%2529%2520from%2520Hit%2520to%2520Clinic%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8305%26epage%3D8320%26doi%3D10.1021%2Fjm2007062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Le
Roux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaise, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boudreault, P.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comeau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucet, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giarrusso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collin, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neubauer, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koelling, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Göller, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seep, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tshitenge, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittwer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kullmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hillisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mittendorf, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsault, É.</span></span> <span> </span><span class="NLM_article-title">Structure-Permeability Relationship of Semi-Peptidic Macrocycles – Understanding and Optimizing Passive Permeability and Efflux Ratio</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_fpage">6774</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&pages=6774&author=A.+Le%0ARouxauthor=E.+Blaiseauthor=P.-L.+Boudreaultauthor=C.+Comeauauthor=A.+Doucetauthor=M.+Giarrussoauthor=M.-P.+Collinauthor=T.+Neubauerauthor=F.+Koellingauthor=A.+H.+G%C3%B6llerauthor=L.+Seepauthor=D.+T.+Tshitengeauthor=M.+Wittwerauthor=M.+Kullmannauthor=A.+Hillischauthor=J.+Mittendorfauthor=%C3%89.+Marsault&title=Structure-Permeability+Relationship+of+Semi-Peptidic+Macrocycles+%E2%80%93+Understanding+and+Optimizing+Passive+Permeability+and+Efflux+Ratio&doi=10.1021%2Facs.jmedchem.0c00013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Permeability Relationship of Semipeptidic Macrocycles-Understanding and Optimizing Passive Permeability and Efflux Ratio</span></div><div class="casAuthors">Le Roux, Antoine; Blaise, Emilie; Boudreault, Pierre-Luc; Comeau, Christian; Doucet, Annie; Giarrusso, Marilena; Collin, Marie-Pierre; Neubauer, Thomas; Kolling, Florian; Goller, Andreas H.; Seep, Lea; Tshitenge, Dieudonne T.; Wittwer, Matthias; Kullmann, Maximilian; Hillisch, Alexander; Mittendorf, Joachim; Marsault, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6774-6783</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herein report the first thorough anal. of the structure-permeability relationship of semipeptidic macrocycles.  In total, 47 macrocycles were synthesized using a hybrid solid-phase/soln. strategy, and then their passive and cellular permeability was assessed using the parallel artificial membrane permeability assay (PAMPA) and Caco-2 assay, resp.  The results indicate that semipeptidic macrocycles generally possess high passive permeability based on the PAMPA, yet their cellular permeability is governed by efflux, as reported in the Caco-2 assay.  Structural variations led to tractable structure-permeability and structure-efflux relationships, wherein the linker length, stereoinversion, N-methylation, and peptoids site-specifically impact the permeability and efflux.  Extensive NMR, mol. dynamics, and ensemble-based three-dimensional polar surface area (3D-PSA) studies showed that ensemble-based 3D-PSA is a good predictor of passive permeability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqCamxP_rXRLVg90H21EOLACvtfcHk0lh2krq7U59eBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSht73F&md5=2794db50565ab0efbc123d0ec541ff32</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00013%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BRoux%26aufirst%3DA.%26aulast%3DBlaise%26aufirst%3DE.%26aulast%3DBoudreault%26aufirst%3DP.-L.%26aulast%3DComeau%26aufirst%3DC.%26aulast%3DDoucet%26aufirst%3DA.%26aulast%3DGiarrusso%26aufirst%3DM.%26aulast%3DCollin%26aufirst%3DM.-P.%26aulast%3DNeubauer%26aufirst%3DT.%26aulast%3DKoelling%26aufirst%3DF.%26aulast%3DG%25C3%25B6ller%26aufirst%3DA.%2BH.%26aulast%3DSeep%26aufirst%3DL.%26aulast%3DTshitenge%26aufirst%3DD.%2BT.%26aulast%3DWittwer%26aufirst%3DM.%26aulast%3DKullmann%26aufirst%3DM.%26aulast%3DHillisch%26aufirst%3DA.%26aulast%3DMittendorf%26aufirst%3DJ.%26aulast%3DMarsault%26aufirst%3D%25C3%2589.%26atitle%3DStructure-Permeability%2520Relationship%2520of%2520Semi-Peptidic%2520Macrocycles%2520%25E2%2580%2593%2520Understanding%2520and%2520Optimizing%2520Passive%2520Permeability%2520and%2520Efflux%2520Ratio%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26spage%3D6774%26doi%3D10.1021%2Facs.jmedchem.0c00013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Appavoo, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yudin, A. K.</span></span> <span> </span><span class="NLM_article-title">Conformational Control of Macrocycles by Remote Structural Modification</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">9724</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.8b00742</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.8b00742" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFKisbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=9724&author=S.+D.+Appavooauthor=S.+Huhauthor=D.+B.+Diazauthor=A.+K.+Yudin&title=Conformational+Control+of+Macrocycles+by+Remote+Structural+Modification&doi=10.1021%2Facs.chemrev.8b00742"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Control of Macrocycles by Remote Structural Modification</span></div><div class="casAuthors">Appavoo, Solomon D.; Huh, Sungjoon; Diaz, Diego B.; Yudin, Andrei K.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9724-9752</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The conformational anal. of macrocycles is a complex and challenging problem.  There are many factors that contribute to this complexity.  These include a large no. of degrees of freedom, noncovalent transannular interactions such as hydrogen bonds and hydrophobic interactions, and a range of steric interactions, along with ring strain effects.  To a greater extent than within acyclic mols., such interactions within macrocycles are coupled such that changing one dihedral angle can significantly affect other dihedral angles, further complicating the situation.  However, this coupling of bond rotations and transannular interactions enables the transmission of three-dimensional information from one side of a macrocycle to the other.  Making relatively small structural modifications to a macrocycle can result in local conformational changes that propagate along the ring to affect distal structural features.  The factors that control how such changes can propagate are poorly understood and it is difficult to predict which modifications will result in significant conformational reorganizations of remote regions of a macrocycle.  This review discusses examples where small structural modifications to macrocyclic scaffolds change the conformational preferences of structurally remote regions of the ring.  We will highlight evidence provided for conformational changes triggered by remote substituents and explanations about how these changes might occur in an effort to further understand the factors that control such phenomena.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraP983oaKMB7Vg90H21EOLACvtfcHk0lgtpCkQXLfwvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFKisbvF&md5=07fd201af7e700eb02ddd73c0dc33a57</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.8b00742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.8b00742%26sid%3Dliteratum%253Aachs%26aulast%3DAppavoo%26aufirst%3DS.%2BD.%26aulast%3DHuh%26aufirst%3DS.%26aulast%3DDiaz%26aufirst%3DD.%2BB.%26aulast%3DYudin%26aufirst%3DA.%2BK.%26atitle%3DConformational%2520Control%2520of%2520Macrocycles%2520by%2520Remote%2520Structural%2520Modification%26jtitle%3DChem.%2520Rev.%26date%3D2019%26spage%3D9724%26doi%3D10.1021%2Facs.chemrev.8b00742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sok, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Going Out on a Limb: Delineating the Effects of β-Branching, N-Methylation, and Side Chain Size on the Passive Permeability, Solubility, and Flexibility of Sanguinamide A Analogues</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7409</span>– <span class="NLM_lpage">7418</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00919</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00919" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7zM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7409-7418&author=A.+T.+Bockusauthor=J.+A.+Schwochertauthor=C.+R.+Pyeauthor=C.+E.+Townsendauthor=V.+Sokauthor=M.+A.+Bednarekauthor=R.+S.+Lokey&title=Going+Out+on+a+Limb%3A+Delineating+the+Effects+of+%CE%B2-Branching%2C+N-Methylation%2C+and+Side+Chain+Size+on+the+Passive+Permeability%2C+Solubility%2C+and+Flexibility+of+Sanguinamide+A+Analogues&doi=10.1021%2Facs.jmedchem.5b00919"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Going Out on a Limb: Delineating The Effects of β-Branching, N-Methylation, and Side Chain Size on the Passive Permeability, Solubility, and Flexibility of Sanguinamide A Analogues</span></div><div class="casAuthors">Bockus, Andrew T.; Schwochert, Joshua A.; Pye, Cameron R.; Townsend, Chad E.; Sok, Vong; Bednarek, Maria A.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7409-7418</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">It is well established that intramol. hydrogen bonding and N-methylation play important roles in the passive permeability of cyclic peptides, but other structural features have been explored less intensively.  Recent studies on the oral bioavailability of the cyclic heptapeptide sanguinamide A have raised the question of whether steric occlusion of polar groups via β-branching is an effective, yet untapped, tool in cyclic peptide permeability optimization.  We report the structures of 17 sanguinamide A analogs designed to test the relative contributions of β-branching, N-methylation, and side chain size to passive membrane permeability and aq. soly.  We demonstrate that β-branching has little effect on permeability compared to the effects of aliph. carbon count and N-methylation of exposed NH groups.  We highlight a new N-methylated analog of sanguinamide A with a Leu substitution at position 2 that exhibits solvent-dependent flexibility and improved permeability over that of the natural product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH9vWd5V2ShLVg90H21EOLACvtfcHk0lgtpCkQXLfwvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtl2qu7zM&md5=618d20dfdb5543cd67c9f427a780652e</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00919%26sid%3Dliteratum%253Aachs%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DSchwochert%26aufirst%3DJ.%2BA.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DTownsend%26aufirst%3DC.%2BE.%26aulast%3DSok%26aufirst%3DV.%26aulast%3DBednarek%26aufirst%3DM.%2BA.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DGoing%2520Out%2520on%2520a%2520Limb%253A%2520Delineating%2520the%2520Effects%2520of%2520%25CE%25B2-Branching%252C%2520N-Methylation%252C%2520and%2520Side%2520Chain%2520Size%2520on%2520the%2520Passive%2520Permeability%252C%2520Solubility%252C%2520and%2520Flexibility%2520of%2520Sanguinamide%2520A%2520Analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7409%26epage%3D7418%26doi%3D10.1021%2Facs.jmedchem.5b00919" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1021/ja508766b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja508766b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=715-721&author=W.+M.+Hewittauthor=S.+S.+F.+Leungauthor=C.+R.+Pyeauthor=A.+R.+Ponkeyauthor=M.+Bednarekauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Cell-Permeable+Cyclic+Peptides+from+Synthetic+Libraries+Inspired+by+Natural+Products&doi=10.1021%2Fja508766b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span></div><div class="casAuthors">Hewitt, William M.; Leung, Siegfried S. F.; Pye, Cameron R.; Ponkey, Alexandra R.; Bednarek, Maria; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">715-721</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug design efforts are turning to a new generation of therapeutic targets, such as protein-protein interactions (PPIs), that had previously been considered "undruggable" by typical small mols.  There is an emerging view that accessing these targets will require mols. that are larger and more complex than typical small mol. drugs.  Here, the authors present a methodol. for the discovery of geometrically diverse, membrane permeable cyclic peptide scaffolds based on the synthesis and permeability screening of a combinatorial library, followed by deconvolution of membrane-permeable scaffolds to identify cyclic peptides with good to excellent passive cell permeabilities.  The authors use a combination of exptl. and computational approaches to investigate structure-permeability relationships in one of these scaffolds, and uncover structural and conformational factors that govern passive membrane diffusion in a related set of cyclic peptide diastereomers.  Further, the authors investigate the dependency of permeability on side-chain identity of one of these scaffolds through single-point diversifications to show the adaptability of these scaffolds toward development of permeability-biased libraries suitable for bioactivity screens.  Overall, the results demonstrate that many novel, cell permeable scaffolds exist beyond those found in extant natural products, and that such scaffolds can be rapidly identified using a combination of synthesis and deconvolution which can, in principle, be applied to any type of macrocyclic template.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wbqbKaUG9rVg90H21EOLACvtfcHk0lhTPjHQPUbcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF&md5=0d112bf0da119b6a50b55997f64ac214</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fja508766b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja508766b%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DBednarek%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCell-Permeable%2520Cyclic%2520Peptides%2520from%2520Synthetic%2520Libraries%2520Inspired%2520by%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D715%26epage%3D721%26doi%3D10.1021%2Fja508766b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rezai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millhauser, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Testing the Conformational Hypothesis of Passive Membrane Permeability Using Synthetic Cyclic Peptide Diastereomers</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">2510</span>– <span class="NLM_lpage">2511</span>, <span class="refDoi"> DOI: 10.1021/ja0563455</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0563455" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2006&pages=2510-2511&author=T.+Rezaiauthor=B.+Yuauthor=G.+L.+Millhauserauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Testing+the+Conformational+Hypothesis+of+Passive+Membrane+Permeability+Using+Synthetic+Cyclic+Peptide+Diastereomers&doi=10.1021%2Fja0563455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Testing the conformational hypothesis of passive membrane permeability using synthetic cyclic peptide diastereomers</span></div><div class="casAuthors">Rezai, Taha; Yu, Bin; Millhauser, Glenn L.; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2510-2511</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Little is known about the effect of conformation on passive membrane diffusion rates in small mols.  Evidence suggests that intramol. hydrogen bonding may play a role by reducing the energetic cost of desolvating hydrogen bond donors, esp. amide N-H groups.  We set out to test this hypothesis by investigating the passive membrane diffusion characteristics of a series of cyclic peptide diastereomers based on the sequence cyclo[Leu-Leu-Leu-Leu-Pro-Tyr].  We identified two cyclic hexapeptide diastereomers based on this sequence, whose membrane diffusion rates differed by nearly two log units.  Results of soln. NMR studies and hydrogen/deuterium (H/D) exchange expts. showed that membrane diffusion rates correlated with the degree of intramol. hydrogen bonding and H/D exchange rates.  The most permeable diastereomer, cyclo[D-Leu-D-Leu-Leu-D-Leu-Pro-Tyr], exhibited a passive membrane diffusion rate comparable to that of the orally available drug cyclosporine A.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqooDObTy1BXrVg90H21EOLACvtfcHk0lhTPjHQPUbcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtVGqu7k%253D&md5=52e5cac6648efb70e743f294e3f8c4ec</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fja0563455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0563455%26sid%3Dliteratum%253Aachs%26aulast%3DRezai%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DMillhauser%26aufirst%3DG.%2BL.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DTesting%2520the%2520Conformational%2520Hypothesis%2520of%2520Passive%2520Membrane%2520Permeability%2520Using%2520Synthetic%2520Cyclic%2520Peptide%2520Diastereomers%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2006%26volume%3D128%26spage%3D2510%26epage%3D2511%26doi%3D10.1021%2Fja0563455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyrchan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgendorf, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarren, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muncipinto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norinder, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Structural and Conformational Determinants of Macrocycle Cell Permeability</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_fpage">1065</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2203</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnchembio.2203" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=27748751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yktb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&pages=1065&author=B.+Overauthor=P.+Matssonauthor=C.+Tyrchanauthor=P.+Arturssonauthor=B.+C.+Doakauthor=M.+A.+Foleyauthor=C.+Hilgendorfauthor=S.+E.+Johnstonauthor=M.+D.+Leeauthor=R.+J.+Lewisauthor=P.+McCarrenauthor=G.+Muncipintoauthor=U.+Norinderauthor=M.+W.+D.+Perryauthor=J.+R.+Duvallauthor=J.+Kihlberg&title=Structural+and+Conformational+Determinants+of+Macrocycle+Cell+Permeability&doi=10.1038%2Fnchembio.2203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and conformational determinants of macrocycle cell permeability</span></div><div class="casAuthors">Over, Bjoern; Matsson, Paer; Tyrchan, Christian; Artursson, Per; Doak, Bradley C.; Foley, Michael A.; Hilgendorf, Constanze; Johnston, Stephen E.; Lee, Maurice D., IV; Lewis, Richard J.; McCarren, Patrick; Muncipinto, Giovanni; Norinder, Ulf; Perry, Matthew W. D.; Duvall, Jeremy R.; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1065-1074</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Macrocycles are of increasing interest as chem. probes and drugs for intractable targets like protein-protein interactions, but the determinants of their cell permeability and oral absorption are poorly understood.  To enable rational design of cell-permeable macrocycles, we generated an extensive data set under consistent exptl. conditions for more than 200 non-peptidic, de novo-designed macrocycles from the Broad Institute's diversity-oriented screening collection.  This revealed how specific functional groups, substituents and mol. properties impact cell permeability.  Anal. of energy-minimized structures for stereo- and regioisomeric sets provided fundamental insight into how dynamic, intramol. interactions in the 3D conformations of macrocycles may be linked to physicochem. properties and permeability.  Combined use of quant. structure-permeability modeling and the procedure for conformational anal. now, for the first time, provides chemists with a rational approach to design cell-permeable non-peptidic macrocycles with potential for oral absorption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-fvl5GK0lVrVg90H21EOLACvtfcHk0lhTPjHQPUbcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yktb3K&md5=a7ff616d7e7e561b96c46a2767325ff6</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2203%26sid%3Dliteratum%253Aachs%26aulast%3DOver%26aufirst%3DB.%26aulast%3DMatsson%26aufirst%3DP.%26aulast%3DTyrchan%26aufirst%3DC.%26aulast%3DArtursson%26aufirst%3DP.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DHilgendorf%26aufirst%3DC.%26aulast%3DJohnston%26aufirst%3DS.%2BE.%26aulast%3DLee%26aufirst%3DM.%2BD.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DMcCarren%26aufirst%3DP.%26aulast%3DMuncipinto%26aufirst%3DG.%26aulast%3DNorinder%26aufirst%3DU.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DStructural%2520and%2520Conformational%2520Determinants%2520of%2520Macrocycle%2520Cell%2520Permeability%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26spage%3D1065%26doi%3D10.1038%2Fnchembio.2203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span>; <span class="NLM_string-name">Kerns, E. H.</span></span> <i>Drug-Like Properties</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Elsevier</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=L.+Di&author=E.+H.+Kerns&title=Drug-Like+Properties"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26btitle%3DDrug-Like%2520Properties%26pub%3DElsevier%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balazs, A. Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbajo, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hird, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannes, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoull, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiarparin, E.</span></span> <span> </span><span class="NLM_article-title">Free Ligand 1D NMR Conformational Signatures to Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9418</span>– <span class="NLM_lpage">9437</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00716</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00716" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVygtbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9418-9437&author=A.+Y.+S.+Balazsauthor=R.+J.+Carbajoauthor=N.+L.+Daviesauthor=Y.+Dongauthor=A.+W.+Hirdauthor=J.+W.+Johannesauthor=M.+L.+Lambauthor=W.+McCoullauthor=P.+Rauboauthor=G.+R.+Robbauthor=M.+J.+Packerauthor=E.+Chiarparin&title=Free+Ligand+1D+NMR+Conformational+Signatures+to+Enhance+Structure+Based+Drug+Design+of+a+Mcl-1+Inhibitor+%28AZD5991%29+and+Other+Synthetic+Macrocycles&doi=10.1021%2Facs.jmedchem.9b00716"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Free Ligand 1D NMR Conformational Signatures To Enhance Structure Based Drug Design of a Mcl-1 Inhibitor (AZD5991) and Other Synthetic Macrocycles</span></div><div class="casAuthors">Balazs, Amber Y. S.; Carbajo, Rodrigo J.; Davies, Nichola L.; Dong, Yu; Hird, Alexander W.; Johannes, Jeffrey W.; Lamb, Michelle L.; McCoull, William; Raubo, Piotr; Robb, Graeme R.; Packer, Martin J.; Chiarparin, Elisabetta</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9418-9437</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The three-dimensional conformations adopted by a free ligand in soln. impact bioactivity and physicochem. properties.  Soln. 1D NMR spectra inherently contain information on ligand conformational flexibility and three-dimensional shape, as well as on the propensity of the free ligand to fully pre-organize into the bioactive conformation.  Herein we discuss some key learnings, distd. from our experience developing potent and selective synthetic macrocyclic inhibitors, including Mcl-1 clin. candidate AZD5991.  Case studies have been selected from recent oncol. research projects, demonstrating how 1D NMR conformational signatures can complement X-ray protein-ligand structural information to guide medicinal chem. optimization.  Learning to ext. free ligand conformational information from routinely available 1D NMR signatures has proven to be fast enough to guide medicinal chem. decisions within design cycles for compd. optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4XmqKE0MA7bVg90H21EOLACvtfcHk0lgSsQ4Qkv1TDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVygtbnK&md5=95088de9c61c16e8a2366d1d91ab385e</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00716&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00716%26sid%3Dliteratum%253Aachs%26aulast%3DBalazs%26aufirst%3DA.%2BY.%2BS.%26aulast%3DCarbajo%26aufirst%3DR.%2BJ.%26aulast%3DDavies%26aufirst%3DN.%2BL.%26aulast%3DDong%26aufirst%3DY.%26aulast%3DHird%26aufirst%3DA.%2BW.%26aulast%3DJohannes%26aufirst%3DJ.%2BW.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DMcCoull%26aufirst%3DW.%26aulast%3DRaubo%26aufirst%3DP.%26aulast%3DRobb%26aufirst%3DG.%2BR.%26aulast%3DPacker%26aufirst%3DM.%2BJ.%26aulast%3DChiarparin%26aufirst%3DE.%26atitle%3DFree%2520Ligand%25201D%2520NMR%2520Conformational%2520Signatures%2520to%2520Enhance%2520Structure%2520Based%2520Drug%2520Design%2520of%2520a%2520Mcl-1%2520Inhibitor%2520%2528AZD5991%2529%2520and%2520Other%2520Synthetic%2520Macrocycles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9418%26epage%3D9437%26doi%3D10.1021%2Facs.jmedchem.9b00716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Orwig, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dix, T. A.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Cα Methylated Carboxylic Acids: Isosteres of Arginine and Lysine for Use as N-Terminal Capping Residues in Polypeptides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">7007</span>– <span class="NLM_lpage">7009</span>, <span class="refDoi"> DOI: 10.1016/j.tetlet.2005.08.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.tetlet.2005.08.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvVSnu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2005&pages=7007-7009&author=K.+S.+Orwigauthor=T.+A.+Dix&title=Synthesis+of+C%CE%B1+Methylated+Carboxylic+Acids%3A+Isosteres+of+Arginine+and+Lysine+for+Use+as+N-Terminal+Capping+Residues+in+Polypeptides&doi=10.1016%2Fj.tetlet.2005.08.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Cα methylated carboxylic acids: isosteres of arginine and lysine for use as N-terminal capping residues in polypeptides</span></div><div class="casAuthors">Orwig, Kevin S.; Dix, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">7007-7009</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Replacement of the N-terminal α-amine with the isosteric Me functionality in bioactive peptides can influence various pharmacokinetic parameters, including hydrophobicity and stability.  Cα methylated amino acid analogs are thus of great interest to expand the repertoire of nonnatural synthons available as N-terminal capping' residues for peptide-based drug design.  Several novel amino acid analogs stereoselectively modified in the Cα position with a Me group in place of the α-amine were prepd. using 4-benzyl-2-oxazolidinone as chiral auxiliary and via substitution of bromine with azide in ω-bromo acids, following by cyclization methylation and azide redn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD-Qv1Yu0lGrVg90H21EOLACvtfcHk0lgSsQ4Qkv1TDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvVSnu7s%253D&md5=09fedb7b3b0b7e3387649a8c8ee0b04b</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2005.08.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2005.08.056%26sid%3Dliteratum%253Aachs%26aulast%3DOrwig%26aufirst%3DK.%2BS.%26aulast%3DDix%26aufirst%3DT.%2BA.%26atitle%3DSynthesis%2520of%2520C%25CE%25B1%2520Methylated%2520Carboxylic%2520Acids%253A%2520Isosteres%2520of%2520Arginine%2520and%2520Lysine%2520for%2520Use%2520as%2520N-Terminal%2520Capping%2520Residues%2520in%2520Polypeptides%26jtitle%3DTetrahedron%2520Lett.%26date%3D2005%26volume%3D46%26spage%3D7007%26epage%3D7009%26doi%3D10.1016%2Fj.tetlet.2005.08.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leogane, O.</span></span> <span> </span><span class="NLM_article-title">Boc-Protected Amines via a Mild and Efficient One-Pot Curtius Rearrangement</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">4107</span>– <span class="NLM_lpage">4110</span>, <span class="refDoi"> DOI: 10.1021/ol051428b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol051428b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2MXosVWlsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=4107-4110&author=H.+Lebelauthor=O.+Leogane&title=Boc-Protected+Amines+via+a+Mild+and+Efficient+One-Pot+Curtius+Rearrangement&doi=10.1021%2Fol051428b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Boc-Protected Amines via a Mild and Efficient One-Pot Curtius Rearrangement</span></div><div class="casAuthors">Lebel, Helene; Leogane, Olivier</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4107-4110</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The reaction of carboxylic acids RCO2H (R = Me2CHCH2, PhCHEt, 1-adamantyl, etc.) with di-tert-Bu dicarbonate and sodium azide allowed the formation of acyl azide intermediates, which undergo a Curtius rearrangement in the presence of tetrabutylammonium bromide and zinc(II) triflate.  The in situ trapping of the isocyanates obtained led to the desired tert-Bu carbamates RNHCO2CMe3 in high yields at low temp.  These reaction conditions are compatible with a variety of substrates, including malonate derivs., which provide access to protected amino acids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphgJudT_mqRLVg90H21EOLACvtfcHk0lgYXfUjZcol5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXosVWlsLo%253D&md5=eb5133b9768e1e9338c5b4927f7af816</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fol051428b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol051428b%26sid%3Dliteratum%253Aachs%26aulast%3DLebel%26aufirst%3DH.%26aulast%3DLeogane%26aufirst%3DO.%26atitle%3DBoc-Protected%2520Amines%2520via%2520a%2520Mild%2520and%2520Efficient%2520One-Pot%2520Curtius%2520Rearrangement%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D4107%26epage%3D4110%26doi%3D10.1021%2Fol051428b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ye, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKervey, M. A.</span></span> <span> </span><span class="NLM_article-title">Organic Synthesis with α-Diazocarbonyl Compounds</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">1091</span>– <span class="NLM_lpage">1160</span>, <span class="refDoi"> DOI: 10.1021/cr00028a010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00028a010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK2cXktVyhsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=1994&pages=1091-1160&author=T.+Yeauthor=M.+A.+McKervey&title=Organic+Synthesis+with+%CE%B1-Diazocarbonyl+Compounds&doi=10.1021%2Fcr00028a010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Organic Synthesis with α-Diazo Carbonyl Compounds</span></div><div class="casAuthors">Ye, Tao; McKervey, M. Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1091-160</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">Review with 786 ref.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqFAjV4JS6YbVg90H21EOLACvtfcHk0lgYXfUjZcol5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXktVyhsbY%253D&md5=ddb663d7c11140798e6aae631887e2b9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fcr00028a010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00028a010%26sid%3Dliteratum%253Aachs%26aulast%3DYe%26aufirst%3DT.%26aulast%3DMcKervey%26aufirst%3DM.%2BA.%26atitle%3DOrganic%2520Synthesis%2520with%2520%25CE%25B1-Diazocarbonyl%2520Compounds%26jtitle%3DChem.%2520Rev.%26date%3D1994%26volume%3D94%26spage%3D1091%26epage%3D1160%26doi%3D10.1021%2Fcr00028a010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kerns, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petusky, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farris, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jupp, P.</span></span> <span> </span><span class="NLM_article-title">Combined Application of Parallel Artificial Membrane Permeability Assay and Caco-2 Permeability Assays in Drug Discovery</span>. <i>J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">1440</span>– <span class="NLM_lpage">1453</span>, <span class="refDoi"> DOI: 10.1002/jps.20075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fjps.20075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=15124203" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2cXks12iu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=1440-1453&author=E.+H.+Kernsauthor=L.+Diauthor=S.+Petuskyauthor=M.+Farrisauthor=R.+Leyauthor=P.+Jupp&title=Combined+Application+of+Parallel+Artificial+Membrane+Permeability+Assay+and+Caco-2+Permeability+Assays+in+Drug+Discovery&doi=10.1002%2Fjps.20075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Combined application of parallel artificial membrane permeability assay and Caco-2 permeability assays in drug discovery</span></div><div class="casAuthors">Kerns, Edward H.; Di, Li; Petusky, Susan; Farris, Michele; Ley, Rob; Jupp, Phil</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1440-1453</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Data from permeability profiling using the parallel artificial membrane permeability assay (PAMPA) and cell monolayer (Caco-2 and MDR1-MDCKII) methods were compared for two published compd. sets and one inhouse set.  A majority of compds. in each set correlated (R2 = 0.76-0.92), indicating the predominance of passive diffusion in the permeation of these compds.  Compds. that did not correlate grouped into two subsets.  One subset had higher PAMPA permeability than cell monolayer permeability and consisted of compds. that are subject to secretory mechanisms: efflux or reduced passive diffusion of bases under Caco-2 when run under a pH gradient.  The other subset had higher cell monolayer permeability than PAMPA permeability and consisted of compds. that are subject to absorptive mechanisms: paracellular, active transport, or increased passive diffusion of acids under Caco-2 when run under a pH gradient.  Given the characteristics of the two methods, these studies suggest how PAMPA and Caco-2 can be synergistically applied for efficient and rapid investigation of permeation mechanisms in drug discovery.  During early discovery, all compds. can be rapidly screened using PAMPA at low pH and neutral pH to assess passive diffusion permeability to indicate potential for gastrointestinal and cell assay permeation.  During intermediate discovery, selected compds. can be addnl. assayed by apical-to-basolateral Caco-2, which, in combination with PAMPA data, indicates susceptibility to addnl. permeation mechanisms (secretory and absorptive).  During mid-to-late discovery, selected candidates can be examd. in detail via multiple directional Caco-2 expts. and with transporter inhibitors for complete characterization of permeation mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG1M-v5UG2RLVg90H21EOLACvtfcHk0lgYXfUjZcol5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXks12iu7w%253D&md5=428596e93f30763e94ced9341d23a02e</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fjps.20075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.20075%26sid%3Dliteratum%253Aachs%26aulast%3DKerns%26aufirst%3DE.%2BH.%26aulast%3DDi%26aufirst%3DL.%26aulast%3DPetusky%26aufirst%3DS.%26aulast%3DFarris%26aufirst%3DM.%26aulast%3DLey%26aufirst%3DR.%26aulast%3DJupp%26aufirst%3DP.%26atitle%3DCombined%2520Application%2520of%2520Parallel%2520Artificial%2520Membrane%2520Permeability%2520Assay%2520and%2520Caco-2%2520Permeability%2520Assays%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D1440%26epage%3D1453%26doi%3D10.1002%2Fjps.20075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel-Andreasen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fristrup, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">Cis-Trans Amide Bond Rotamers in β-Peptoids and Peptoids: Evaluation of Stereoelectronic Effects in Backbone and Side Chains</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">2835</span>– <span class="NLM_lpage">2844</span>, <span class="refDoi"> DOI: 10.1021/ja312532x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja312532x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtleltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=2835-2844&author=J.+S.+Laursenauthor=J.+Engel-Andreasenauthor=P.+Fristrupauthor=P.+Harrisauthor=C.+A.+Olsen&title=Cis-Trans+Amide+Bond+Rotamers+in+%CE%B2-Peptoids+and+Peptoids%3A+Evaluation+of+Stereoelectronic+Effects+in+Backbone+and+Side+Chains&doi=10.1021%2Fja312532x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Cis-Trans Amide Bond Rotamers in β-Peptoids and Peptoids: Evaluation of Stereoelectronic Effects in Backbone and Side Chains</span></div><div class="casAuthors">Laursen, Jonas S.; Engel-Andreasen, Jens; Fristrup, Peter; Harris, Pernille; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2835-2844</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nonnatural peptide analogs have significant potential for the development of new materials and pharmacol. active ligands.  One such architecture, the β-peptoids (N-alkyl-β-alanines), found use in a variety of biol. active compds. but was sparsely studied with respect to folding propensity.  Thus, the authors here report a study of the effect of structural variations on the cis-trans amide bond rotamer equil. in a selection of monomer model systems.  In addn. to various side chain effects, which correlated well with previous studies of α-peptoids, the authors present the synthesis and study of cis-trans isomerism in the first examples of peptoids and β-peptoids contg. thioamide bonds as well as trifluoroacetylated peptoids and β-peptoids.  These systems revealed an increase in the preference for cis-amides as compared to their parent compds. and thus provide novel strategies for affecting the folding of peptoid constructs.  By using NMR spectroscopy, x-ray crystallog. anal., and d. functional theory calcns., the authors present evidence for the presence of thioamide-arom. interactions through Csp2 -H···Samide hydrogen bonding, which stabilize certain peptoid conformations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqXidNft09rVg90H21EOLACvtfcHk0lgd5mLHB7ZQ1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtleltL0%253D&md5=5af65208996e55a516a95461a1c728db</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fja312532x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja312532x%26sid%3Dliteratum%253Aachs%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DEngel-Andreasen%26aufirst%3DJ.%26aulast%3DFristrup%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DCis-Trans%2520Amide%2520Bond%2520Rotamers%2520in%2520%25CE%25B2-Peptoids%2520and%2520Peptoids%253A%2520Evaluation%2520of%2520Stereoelectronic%2520Effects%2520in%2520Backbone%2520and%2520Side%2520Chains%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D2835%26epage%3D2844%26doi%3D10.1021%2Fja312532x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel-Andreasen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wich, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laursen, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, C. A.</span></span> <span> </span><span class="NLM_article-title">Effects of Thionation and Fluorination on Cis-Trans Isomerization in Tertiary Amides: An Investigation of N-Alkylglycine (Peptoid) Rotamers</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">5415</span>– <span class="NLM_lpage">5427</span>, <span class="refDoi"> DOI: 10.1021/acs.joc.5b00048</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.joc.5b00048" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjslelsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2015&pages=5415-5427&author=J.+Engel-Andreasenauthor=K.+Wichauthor=J.+S.+Laursenauthor=P.+Harrisauthor=C.+A.+Olsen&title=Effects+of+Thionation+and+Fluorination+on+Cis-Trans+Isomerization+in+Tertiary+Amides%3A+An+Investigation+of+N-Alkylglycine+%28Peptoid%29+Rotamers&doi=10.1021%2Facs.joc.5b00048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of thionation and fluorination on cis-trans isomerization in tertiary amides: An investigation of N-alkylglycine (peptoid) rotamers</span></div><div class="casAuthors">Engel-Andreasen, Jens; Wich, Kathrine; Laursen, Jonas S.; Harris, Pernille; Olsen, Christian A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5415-5427</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptoids constitute a class of peptidomimetics with potential as protease resistant, biol. active ligands.  To harness the full potential of such compds., however, detailed predictive insight into their propensity to adopt well-defined secondary structures is highly desirable.  In this work we present an investigation of the effects of thioamides and/or fluorides in peptoid monomer model systems using chem. synthesis, NMR spectroscopy, and X-ray crystallog.  We find that the steric environment surrounding the tertiary amide bonds is the key promoter of conformational preference, and X-ray crystallog. interrogation of our model systems did not suggest the presence of stabilizing n → π* interactions unless the carbonyls were altered electronically by α-halogenation or thioamide formation.  In addn. to the function as an investigative tool, these two types of modification may thus be utilized as stabilizers of secondary structure in future oligomer designs, such as the cis-amide-based polypeptoid helixes that resemble the polyproline type-I helix.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF7WiuezU4sLVg90H21EOLACvtfcHk0lgd5mLHB7ZQ1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjslelsrg%253D&md5=28cddf791106a3f97213164b89bddc2f</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.joc.5b00048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.joc.5b00048%26sid%3Dliteratum%253Aachs%26aulast%3DEngel-Andreasen%26aufirst%3DJ.%26aulast%3DWich%26aufirst%3DK.%26aulast%3DLaursen%26aufirst%3DJ.%2BS.%26aulast%3DHarris%26aufirst%3DP.%26aulast%3DOlsen%26aufirst%3DC.%2BA.%26atitle%3DEffects%2520of%2520Thionation%2520and%2520Fluorination%2520on%2520Cis-Trans%2520Isomerization%2520in%2520Tertiary%2520Amides%253A%2520An%2520Investigation%2520of%2520N-Alkylglycine%2520%2528Peptoid%2529%2520Rotamers%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2015%26volume%3D80%26spage%3D5415%26epage%3D5427%26doi%3D10.1021%2Facs.joc.5b00048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebastiano, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backlund, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Over, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Impact of Dynamically Exposed Polarity on Permeability and Solubility of Chameleonic Drugs Beyond the Rule of 5</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">4189</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00347</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00347" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=4189&author=M.+R.+Sebastianoauthor=B.+C.+Doakauthor=M.+Backlundauthor=V.+Poongavanamauthor=B.+Overauthor=G.+Ermondiauthor=G.+Caronauthor=J.+Kihlberg&title=Impact+of+Dynamically+Exposed+Polarity+on+Permeability+and+Solubility+of+Chameleonic+Drugs+Beyond+the+Rule+of+5&doi=10.1021%2Facs.jmedchem.8b00347"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00347%26sid%3Dliteratum%253Aachs%26aulast%3DSebastiano%26aufirst%3DM.%2BR.%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DBacklund%26aufirst%3DM.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DOver%26aufirst%3DB.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DImpact%2520of%2520Dynamically%2520Exposed%2520Polarity%2520on%2520Permeability%2520and%2520Solubility%2520of%2520Chameleonic%2520Drugs%2520Beyond%2520the%2520Rule%2520of%25205%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26spage%3D4189%26doi%3D10.1021%2Facs.jmedchem.8b00347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Alex, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millan, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakenhut, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitlock, G. A.</span></span> <span> </span><span class="NLM_article-title">Intramolecular Hydrogen Bonding to Improve Membrane Permeability and Absorption in beyond Rule of Five Chemical Space</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">669</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1039/c1md00093d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1039%2Fc1md00093d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=669-674&author=A.+Alexauthor=D.+S.+Millanauthor=M.+Perezauthor=F.+Wakenhutauthor=G.+A.+Whitlock&title=Intramolecular+Hydrogen+Bonding+to+Improve+Membrane+Permeability+and+Absorption+in+beyond+Rule+of+Five+Chemical+Space&doi=10.1039%2Fc1md00093d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Intramolecular hydrogen bonding to improve membrane permeability and absorption in beyond rule of five chemical space</span></div><div class="casAuthors">Alex, Alexander; Millan, David S.; Perez, Manuel; Wakenhut, Florian; Whitlock, Gavin A.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">669-674</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Utilizing 'beyond rule of five' chem. space is becoming increasingly important in drug design, but is usually at odds with good oral absorption.  The formation of intramol. hydrogen bonds in drug mols. is hypothesised to shield polarity facilitating improved membrane permeability and intestinal absorption.  NMR based evidence for intramol. hydrogen bonding in several 'beyond rule of five' oral drugs is described.  Furthermore, the propensity for these drugs to form intramol. hydrogen bonds could be predicted for through modeling the lowest energy conformation in the gas phase.  The modulation of apparent lipophilicity through intramol. hydrogen bonding in these mols. is supported by intrinsic cell permeability and intestinal absorption data in rat and human.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp93l2KJMdB2bVg90H21EOLACvtfcHk0lgd5mLHB7ZQ1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotl2gt7o%253D&md5=3effbdd29832ac7b57eabbc26c66217b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1039%2Fc1md00093d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1md00093d%26sid%3Dliteratum%253Aachs%26aulast%3DAlex%26aufirst%3DA.%26aulast%3DMillan%26aufirst%3DD.%2BS.%26aulast%3DPerez%26aufirst%3DM.%26aulast%3DWakenhut%26aufirst%3DF.%26aulast%3DWhitlock%26aufirst%3DG.%2BA.%26atitle%3DIntramolecular%2520Hydrogen%2520Bonding%2520to%2520Improve%2520Membrane%2520Permeability%2520and%2520Absorption%2520in%2520beyond%2520Rule%2520of%2520Five%2520Chemical%2520Space%26jtitle%3DMedChemComm%26date%3D2011%26volume%3D2%26spage%3D669%26epage%3D674%26doi%3D10.1039%2Fc1md00093d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tyagi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindhagen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjö, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiesser, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Toward the Design of Molecular Chameleons: Flexible Shielding of an Amide Bond Enhances Macrocycle Cell Permeability</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">5737</span>– <span class="NLM_lpage">5742</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.8b02447</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.8b02447" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1yksr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2018&pages=5737-5742&author=M.+Tyagiauthor=V.+Poongavanamauthor=M.+Lindhagenauthor=A.+Pettersenauthor=P.+Sj%C3%B6author=S.+Schiesserauthor=J.+Kihlberg&title=Toward+the+Design+of+Molecular+Chameleons%3A+Flexible+Shielding+of+an+Amide+Bond+Enhances+Macrocycle+Cell+Permeability&doi=10.1021%2Facs.orglett.8b02447"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the Design of Molecular Chameleons: Flexible Shielding of an Amide Bond Enhances Macrocycle Cell Permeability</span></div><div class="casAuthors">Tyagi, Mohit; Poongavanam, Vasanthanathan; Lindhagen, Marika; Pettersen, Anna; Sjoe, Peter; Schiesser, Stefan; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5737-5742</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of macrocycles inspired by natural products were synthesized to investigate how side-chains may shield amide bonds and influence cell permeability.  NMR spectroscopy and x-ray crystallog. revealed that the Ph group of phenylalanine, but not the side-chains of homologous or aliph. amino acids, shields the adjacent amide bond through an intramol. NH-π interaction.  This resulted in increased cell permeability, suggesting that NH-π interactions may be used in the design of mol. chameleons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1E2xKB5kDZrVg90H21EOLACvtfcHk0lhdtoa2e1_Tcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1yksr7O&md5=24182bbd5b612d25ca9fd9cdbb3f9bc4</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.8b02447&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.8b02447%26sid%3Dliteratum%253Aachs%26aulast%3DTyagi%26aufirst%3DM.%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DLindhagen%26aufirst%3DM.%26aulast%3DPettersen%26aufirst%3DA.%26aulast%3DSj%25C3%25B6%26aufirst%3DP.%26aulast%3DSchiesser%26aufirst%3DS.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DToward%2520the%2520Design%2520of%2520Molecular%2520Chameleons%253A%2520Flexible%2520Shielding%2520of%2520an%2520Amide%2520Bond%2520Enhances%2520Macrocycle%2520Cell%2520Permeability%26jtitle%3DOrg.%2520Lett.%26date%3D2018%26volume%3D20%26spage%3D5737%26epage%3D5742%26doi%3D10.1021%2Facs.orglett.8b02447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilk, M. B.</span></span> <span> </span><span class="NLM_article-title">Biometrika Trust An Analysis of Variance Test for Normality (Complete Samples) Published by: Oxford University Press on Behalf of Biometrika Trust Stable</span>. <i>Biometrika</i> <span class="NLM_year" style="font-weight: bold;">1965</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">591</span>– <span class="NLM_lpage">611</span>, <span class="refDoi"> DOI: 10.1093/biomet/52.3-4.591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1093%2Fbiomet%2F52.3-4.591" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=1965&pages=591-611&author=S.+S.+Shapiroauthor=M.+B.+Wilk&title=Biometrika+Trust+An+Analysis+of+Variance+Test+for+Normality+%28Complete+Samples%29+Published+by%3A+Oxford+University+Press+on+Behalf+of+Biometrika+Trust+Stable&doi=10.1093%2Fbiomet%2F52.3-4.591"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1093%2Fbiomet%2F52.3-4.591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbiomet%252F52.3-4.591%26sid%3Dliteratum%253Aachs%26aulast%3DShapiro%26aufirst%3DS.%2BS.%26aulast%3DWilk%26aufirst%3DM.%2BB.%26atitle%3DBiometrika%2520Trust%2520An%2520Analysis%2520of%2520Variance%2520Test%2520for%2520Normality%2520%2528Complete%2520Samples%2529%2520Published%2520by%253A%2520Oxford%2520University%2520Press%2520on%2520Behalf%2520of%2520Biometrika%2520Trust%2520Stable%26jtitle%3DBiometrika%26date%3D1965%26volume%3D52%26spage%3D591%26epage%3D611%26doi%3D10.1093%2Fbiomet%2F52.3-4.591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kotz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, N. L.</span></span> <span> </span><span class="NLM_article-title">Breakthroughs in Statistics, Volume III</span>. <i>Technometrics</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">165</span>, <span class="refDoi"> DOI: 10.1080/00401706.1998.10485220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1080%2F00401706.1998.10485220" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=1998&pages=165&author=S.+Kotzauthor=N.+L.+Johnson&title=Breakthroughs+in+Statistics%2C+Volume+III&doi=10.1080%2F00401706.1998.10485220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1080%2F00401706.1998.10485220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00401706.1998.10485220%26sid%3Dliteratum%253Aachs%26aulast%3DKotz%26aufirst%3DS.%26aulast%3DJohnson%26aufirst%3DN.%2BL.%26atitle%3DBreakthroughs%2520in%2520Statistics%252C%2520Volume%2520III%26jtitle%3DTechnometrics%26date%3D1998%26volume%3D40%26spage%3D165%26doi%3D10.1080%2F00401706.1998.10485220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vorherr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berghausen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desrayaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaefer, M.</span></span> <span> </span><span class="NLM_article-title">Modulation of Oral Bioavailability and Metabolism for Closely Related Cyclic Hexapeptides</span>. <i>Int. J. Pept. Res. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1007/s10989-017-9590-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1007%2Fs10989-017-9590-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=29527142" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlt1Srsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=35-48&author=T.+Vorherrauthor=I.+Lewisauthor=J.+Berghausenauthor=S.+Desrayaudauthor=M.+Schaefer&title=Modulation+of+Oral+Bioavailability+and+Metabolism+for+Closely+Related+Cyclic+Hexapeptides&doi=10.1007%2Fs10989-017-9590-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Modulation of Oral Bioavailability and Metabolism for Closely Related Cyclic Hexapeptides</span></div><div class="casAuthors">Vorherr, Thomas; Lewis, Ian; Berghausen, Joerg; Desrayaud, Sandrine; Schaefer, Michael</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Peptide Research and Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">35-48</span>CODEN:
                <span class="NLM_cas:coden">IJPRFC</span>;
        ISSN:<span class="NLM_cas:issn">1573-3149</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Abstr.: Recently, a variety of studies concerned with the permeability and oral bioavailability of cyclic peptides have been reported.  In particular, strategies aiming at modifying peptides to maintain or to enhance soly. while enabling permeability constitute a significant challenge, but are of high interest to ensure a smooth drug discovery process.  Current methodologies include N-methylation, matching of hydrogen bonding acceptors and donors across the macrocycle, and addnl. masking of polarity.  In this study, we investigate further the pivotal effects of shielding on permeability and studied the metab. of the corresponding peptides in more detail by comparing peptide concns. in the portal vs. the jugular vein in rats.  Interestingly, minor changes in one particular side chain impacts both permeability and liver metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZnrDMY7yZHLVg90H21EOLACvtfcHk0lgZtmjO-oT5mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlt1Srsrw%253D&md5=2faed8ef9cf1de91b6a5fd4c73c44c5d</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1007%2Fs10989-017-9590-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10989-017-9590-8%26sid%3Dliteratum%253Aachs%26aulast%3DVorherr%26aufirst%3DT.%26aulast%3DLewis%26aufirst%3DI.%26aulast%3DBerghausen%26aufirst%3DJ.%26aulast%3DDesrayaud%26aufirst%3DS.%26aulast%3DSchaefer%26aufirst%3DM.%26atitle%3DModulation%2520of%2520Oral%2520Bioavailability%2520and%2520Metabolism%2520for%2520Closely%2520Related%2520Cyclic%2520Hexapeptides%26jtitle%3DInt.%2520J.%2520Pept.%2520Res.%2520Ther.%26date%3D2018%26volume%3D24%26spage%3D35%26epage%3D48%26doi%3D10.1007%2Fs10989-017-9590-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peraro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Emerging Methods and Design Principles for Cell-Penetrant Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">11868</span>– <span class="NLM_lpage">11881</span>, <span class="refDoi"> DOI: 10.1002/anie.201801361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fanie.201801361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeqtrjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=11868-11881&author=L.+Peraroauthor=J.+A.+Kritzer&title=Emerging+Methods+and+Design+Principles+for+Cell-Penetrant+Peptides&doi=10.1002%2Fanie.201801361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Methods and Design Principles for Cell-Penetrant Peptides</span></div><div class="casAuthors">Peraro, Leila; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">11868-11881</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review on the diverse chem. approaches that have yielded cell-penetrant peptides, and the three distinct strategies identified: masking backbone amides, guanidinium group patterning, and amphipathic patterning.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGQbUZr5CdxbVg90H21EOLACvtfcHk0lgZtmjO-oT5mg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeqtrjK&md5=f627595f28e69841d51bbe535388a579</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fanie.201801361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201801361%26sid%3Dliteratum%253Aachs%26aulast%3DPeraro%26aufirst%3DL.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DEmerging%2520Methods%2520and%2520Design%2520Principles%2520for%2520Cell-Penetrant%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D11868%26epage%3D11881%26doi%3D10.1002%2Fanie.201801361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schrödinger, L.</span></span> <i>The PyMOL Molecular Graphics System</i>, version 1.8; <span class="NLM_publisher-name">Schrödinger, LLC</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Schr%C3%B6dinger&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DSchr%25C3%25B6dinger%26aufirst%3DL.%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kansy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senner, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gubernator, K.</span></span> <span> </span><span class="NLM_article-title">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1010</span>, <span class="refDoi"> DOI: 10.1021/jm970530e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm970530e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=1998&pages=1007-1010&author=M.+Kansyauthor=F.+Sennerauthor=K.+Gubernator&title=Physicochemical+High+Throughput+Screening%3A+Parallel+Artificial+Membrane+Permeation+Assay+in+the+Description+of+Passive+Absorption+Processes&doi=10.1021%2Fjm970530e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Physicochemical High Throughput Screening: Parallel Artificial Membrane Permeation Assay in the Description of Passive Absorption Processes</span></div><div class="casAuthors">Kansy, Manfred; Senner, Frank; Gubernator, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1007-1010</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The majority of pharmaceutical discovery projects aim for an orally available form of a new therapeutic principle.  One obstacle for per oral treatment is intestinal absorption.  A simple predictive exptl. method to assess the potential of intestinal absorption is therefore highly welcome.  The authors describe here the PAMPA (Parallel Artificial Membrane Permeation Assay) method, a simple, robust, high throughput screen (HTS) which has been shown to be predictive for passive diffusion through membranes and thus for oral absorption.  In addn. PAMPA can deliver information in parallel on the lipophilicity, the ionization state and the soly. of a compd.  Therefore PAMPA complements HTS, combinatorial chem. and parallel synthesis efforts in preclin. pharmaceutical research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcwCGodCr1rVg90H21EOLACvtfcHk0lhtER8shv0JXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhs12kt70%253D&md5=85d8ab409313b583de52348eb4d895da</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm970530e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm970530e%26sid%3Dliteratum%253Aachs%26aulast%3DKansy%26aufirst%3DM.%26aulast%3DSenner%26aufirst%3DF.%26aulast%3DGubernator%26aufirst%3DK.%26atitle%3DPhysicochemical%2520High%2520Throughput%2520Screening%253A%2520Parallel%2520Artificial%2520Membrane%2520Permeation%2520Assay%2520in%2520the%2520Description%2520of%2520Passive%2520Absorption%2520Processes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1998%26volume%3D41%26spage%3D1007%26epage%3D1010%26doi%3D10.1021%2Fjm970530e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avdeef, A.</span></span> <i>Absorption and Drug Development</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2012</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2F9781118286067" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&author=A.+Avdeef&title=Absorption+and+Drug+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F9781118286067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118286067%26sid%3Dliteratum%253Aachs%26aulast%3DAvdeef%26aufirst%3DA.%26btitle%3DAbsorption%2520and%2520Drug%2520Development%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hidalgo, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raub, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, R. T.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Human Colon Carcinoma Cell Line (Caco-2) as a Model System for Intestinal Epithelial Permeability</span>. <i>Gastroenterology</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>96</i></span>,  <span class="NLM_fpage">736</span>– <span class="NLM_lpage">749</span>, <span class="refDoi"> DOI: 10.1016/S0016-5085(89)80072-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2FS0016-5085%2889%2980072-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=2914637" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1989&pages=736-749&author=I.+J.+Hidalgoauthor=T.+J.+Raubauthor=R.+T.+Borchardt&title=Characterization+of+the+Human+Colon+Carcinoma+Cell+Line+%28Caco-2%29+as+a+Model+System+for+Intestinal+Epithelial+Permeability&doi=10.1016%2FS0016-5085%2889%2980072-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the human colon carcinoma cell line (Caco-2) as a model system for intestinal epithelial permeability</span></div><div class="casAuthors">Hidalgo I J; Raub T J; Borchardt R T</div><div class="citationInfo"><span class="NLM_cas:title">Gastroenterology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">736-49</span>
        ISSN:<span class="NLM_cas:issn">0016-5085</span>.
    </div><div class="casAbstract">Caco-2 cells develop morphologic characteristics of normal enterocytes when grown on plastic dishes or nitrocellulose filters.  The purpose of this study was to determine whether Caco-2 cells undergo similar differentiation when grown on Transwell polycarbonate membranes, and to study the suitability of Caco-2 monolayers as an intestinal epithelial transport model system.  Transepithelial electrical resistance values after confluence were 173.5 omega.cm2 and remained unchanged through day 17.  Permeabilities to the water-soluble fluid-phase markers that do not permeate the membrane, Lucifer yellow CH, [14C]inulin, [14C]polyethylene glycol, and [3H] dextran were less than 0.25% of the administered amount per hour after day 10.  Qualitative evaluation of uptake and permeability to horseradish peroxidase confirmed the similarity in uptake and barrier properties between this cell system and the small intestinal epithelial layer.  We conclude that Caco-2 cells grown on collagen-coated polycarbonate membranes should represent a valuable transport model system for the small intestinal epithelium.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOn97989rzEIqdXsLcGTEUfW6udTcc2eaPysjIyialdLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaL1M7itFeqsg%253D%253D&md5=59ffccc1dd4dccb01400c05df95ac1ff</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2FS0016-5085%2889%2980072-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0016-5085%252889%252980072-1%26sid%3Dliteratum%253Aachs%26aulast%3DHidalgo%26aufirst%3DI.%2BJ.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26atitle%3DCharacterization%2520of%2520the%2520Human%2520Colon%2520Carcinoma%2520Cell%2520Line%2520%2528Caco-2%2529%2520as%2520a%2520Model%2520System%2520for%2520Intestinal%2520Epithelial%2520Permeability%26jtitle%3DGastroenterology%26date%3D1989%26volume%3D96%26spage%3D736%26epage%3D749%26doi%3D10.1016%2FS0016-5085%2889%2980072-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hubatsch, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ragnarsson, E. G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artursson, P.</span></span> <span> </span><span class="NLM_article-title">Determination of Drug Permeability and Prediction of Drug Absorption in Caco-2 Monolayers</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">2111</span>– <span class="NLM_lpage">2119</span>, <span class="refDoi"> DOI: 10.1038/nprot.2007.303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fnprot.2007.303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=17853866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=2111-2119&author=I.+Hubatschauthor=E.+G.+E.+Ragnarssonauthor=P.+Artursson&title=Determination+of+Drug+Permeability+and+Prediction+of+Drug+Absorption+in+Caco-2+Monolayers&doi=10.1038%2Fnprot.2007.303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Determination of drug permeability and prediction of drug absorption in Caco-2 monolayers</span></div><div class="casAuthors">Hubatsch, Ina; Ragnarsson, Eva G. E.; Artursson, Per</div><div class="citationInfo"><span class="NLM_cas:title">Nature Protocols</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2111-2119</span>CODEN:
                <span class="NLM_cas:coden">NPARDW</span>;
        ISSN:<span class="NLM_cas:issn">1750-2799</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Permeability coeffs. across monolayers of the human colon carcinoma cell line Caco-2, cultured on permeable supports, are commonly used to predict the absorption of orally administered drugs and other xenobiotics.  This protocol describes our method for the cultivation, characterization and detn. of permeability coeffs. of xenobiotics (which are, typically, drug-like compds.) in the Caco-2 model.  A few modifications that have been introduced over the years are incorporated in the protocol.  The method can be used to trace the permeability of a test compd. in two directions, from the apical to the basolateral side or vice versa, and both passive and active transport processes can be studied.  The permeability assay can be completed within one working day, provided that the Caco-2 monolayers have been cultured and differentiated on the permeable supports 3 wk in advance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpX7zGXziU_VLVg90H21EOLACvtfcHk0lhdtcs3Z6B4LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFagsb%252FN&md5=85e2f95420919ee61c2d691c305bc1ef</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2007.303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2007.303%26sid%3Dliteratum%253Aachs%26aulast%3DHubatsch%26aufirst%3DI.%26aulast%3DRagnarsson%26aufirst%3DE.%2BG.%2BE.%26aulast%3DArtursson%26aufirst%3DP.%26atitle%3DDetermination%2520of%2520Drug%2520Permeability%2520and%2520Prediction%2520of%2520Drug%2520Absorption%2520in%2520Caco-2%2520Monolayers%26jtitle%3DNat.%2520Protoc.%26date%3D2007%26volume%3D2%26spage%3D2111%26epage%3D2119%26doi%3D10.1038%2Fnprot.2007.303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, A.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Learning from the Data Set and the Analysis of Failures</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">2919</span>– <span class="NLM_lpage">2936</span>, <span class="refDoi"> DOI: 10.1021/ci300314k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300314k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2919-2936&author=P.+C.+D.+Hawkinsauthor=A.+Nicholls&title=Conformer+Generation+with+OMEGA%3A+Learning+from+the+Data+Set+and+the+Analysis+of+Failures&doi=10.1021%2Fci300314k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Learning from the Data Set and the Analysis of Failures</span></div><div class="casAuthors">Hawkins, Paul C. D.; Nicholls, Anthony</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2919-2936</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We recently published a high quality validation set for testing conformer generators, consisting of structures from both the PDB and the CSD and tested the performance of the authors' conformer generator, OMEGA, on these sets.  In the present publication, we focus on understanding the suitability of those data sets for validation and identifying and learning from OMEGA's failures.  We compare, for the first time we are aware of, the coverage of the applicable property spaces between the validation data sets we used and the parent compd. sets to det. if our data sets adequately sample these property spaces.  We also introduce the concept of torsion fingerprinting and compare this method of dissimilation to the more traditional graph-centric diversification methods we used in our previous publication.  To improve our ability to programmatically identify cases where the crystallog. conformation is not well reproduced computationally, we introduce a new metric to compare conformations, RMSTanimoto.  This new metric is used alongside those from our previous publication to efficiently identify reprodn. failures.  We find RMSTanimoto to be particularly effective in identifying failures for the smallest mols. in our data sets.  Anal. of the nature of these failures, particularly those for the CSD, sheds further light on the issue of strain in crystallog. structures.  Some of the residual failure cases not resolved by simple changes in OMEGA's defaults present significant challenges to conformer generation engines like OMEGA and are a source of new avenues to further improve their performance, while others illustrate the pitfalls of validating against crystallog. ligand conformations, particularly those from the PDB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTlHYy7GV9RbVg90H21EOLACvtfcHk0lhdtcs3Z6B4LQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFekurzN&md5=852c44d8587480e69709cefeee9206ac</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci300314k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300314k%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DNicholls%26aufirst%3DA.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Learning%2520from%2520the%2520Data%2520Set%2520and%2520the%2520Analysis%2520of%2520Failures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D2919%26epage%3D2936%26doi%3D10.1021%2Fci300314k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skillman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahl, M. T.</span></span> <span> </span><span class="NLM_article-title">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">584</span>, <span class="refDoi"> DOI: 10.1021/ci100031x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100031x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=572-584&author=P.+C.+D.+Hawkinsauthor=A.+G.+Skillmanauthor=G.+L.+Warrenauthor=B.+A.+Ellingsonauthor=M.+T.+Stahl&title=Conformer+Generation+with+OMEGA%3A+Algorithm+and+Validation+Using+High+Quality+Structures+from+the+Protein+Databank+and+Cambridge+Structural+Database&doi=10.1021%2Fci100031x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Conformer Generation with OMEGA: Algorithm and Validation Using High Quality Structures from the Protein Databank and Cambridge Structural Database</span></div><div class="casAuthors">Hawkins, Paul C. D.; Skillman, A. Geoffrey; Warren, Gregory L.; Ellingson, Benjamin A.; Stahl, Matthew T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">572-584</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Here, we present the algorithm and validation for OMEGA, a systematic, knowledge-based conformer generator.  The algorithm consists of three phases: assembly of an initial 3D structure from a library of fragments; exhaustive enumeration of all rotatable torsions using values drawn from a knowledge-based list of angles, thereby generating a large set of conformations; and sampling of this set by geometric and energy criteria.  Validation of conformer generators like OMEGA has often been undertaken by comparing computed conformer sets to exptl. mol. conformations from crystallog., usually from the Protein Databank (PDB).  Such an approach is fraught with difficulty due to the systematic problems with small mol. structures in the PDB.  Methods are presented to identify a diverse set of small mol. structures from cocomplexes in the PDB that has maximal reliability.  A challenging set of 197 high quality, carefully selected ligand structures from well-solved models was obtained using these methods.  This set will provide a sound basis for comparison and validation of conformer generators in the future.  Validation results from this set are compared to the results using structures of a set of druglike mols. extd. from the Cambridge Structural Database (CSD).  OMEGA is found to perform very well in reproducing the crystallog. conformations from both these data sets using two complementary metrics of success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqq1SOwPH1NzrVg90H21EOLACvtfcHk0liRVVyYGlmsEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtlaisrY%253D&md5=fb87ecc9c51eddef63b41fffcd9babee</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci100031x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100031x%26sid%3Dliteratum%253Aachs%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DSkillman%26aufirst%3DA.%2BG.%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DEllingson%26aufirst%3DB.%2BA.%26aulast%3DStahl%26aufirst%3DM.%2BT.%26atitle%3DConformer%2520Generation%2520with%2520OMEGA%253A%2520Algorithm%2520and%2520Validation%2520Using%2520High%2520Quality%2520Structures%2520from%2520the%2520Protein%2520Databank%2520and%2520Cambridge%2520Structural%2520Database%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D572%26epage%3D584%26doi%3D10.1021%2Fci100031x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poongavanam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danelius, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peintner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alcaraz, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wlodek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erdelyi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, P. C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ermondi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">11742</span>– <span class="NLM_lpage">11757</span>, <span class="refDoi"> DOI: 10.1021/acsomega.8b01379</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.8b01379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2018&pages=11742-11757&author=V.+Poongavanamauthor=E.+Daneliusauthor=S.+Peintnerauthor=L.+Alcarazauthor=G.+Caronauthor=M.+D.+Cummingsauthor=S.+Wlodekauthor=M.+Erdelyiauthor=P.+C.+D.+Hawkinsauthor=G.+Ermondiauthor=J.+Kihlberg&title=Conformational+Sampling+of+Macrocyclic+Drugs+in+Different+Environments%3A+Can+We+Find+the+Relevant+Conformations%3F&doi=10.1021%2Facsomega.8b01379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational Sampling of Macrocyclic Drugs in Different Environments: Can We Find the Relevant Conformations?</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Danelius, Emma; Peintner, Stefan; Alcaraz, Lilian; Caron, Giulia; Cummings, Maxwell D.; Wlodek, Stanislaw; Erdelyi, Mate; Hawkins, Paul C. D.; Ermondi, Giuseppe; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">11742-11757</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational flexibility is a major determinant of the properties of macrocycles and other drugs in beyond rule of 5 (bRo5) space.  Prediction of conformations is essential for design of drugs in this space and the authors have evaluated three tools for conformational sampling of a set of ten bRo5 drugs and clin. candidates in polar and apolar environments.  The distance-geometry based OMEGA was found to yield ensembles spanning larger structure and property spaces than the two MD based methods, MOE-LowMode MD (MOE) and MacroModel-Macrocycle Conformational Sampling (MC).  Both MC and OMEGA, but not MOE, generated different ensembles for polar and apolar environments.  All three conformational search methods generated conformers similar to the crystal structure conformers for nine of the ten compds., with OMEGA performing somewhat better than MOE and MC.  MOE and OMEGA found all six conformers of roxithromycin that were identified by NMR in aq. solns., while only OMEGA sampled the three conformers obsd. in chloroform.  The authors suggest that characterization of conformers using mol. descriptors, e.g. the radius of gyration (Rgyr) and polar surface area (PSA), is preferred to energy- or RMSD-based methods for selection of biol. relevant conformers in drug discovery in bRo5 space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoetOG-yDeciLVg90H21EOLACvtfcHk0liRVVyYGlmsEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslKhtrnI&md5=a8722551b161e907daf55146f22b6d57</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facsomega.8b01379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.8b01379%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DDanelius%26aufirst%3DE.%26aulast%3DPeintner%26aufirst%3DS.%26aulast%3DAlcaraz%26aufirst%3DL.%26aulast%3DCaron%26aufirst%3DG.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DWlodek%26aufirst%3DS.%26aulast%3DErdelyi%26aufirst%3DM.%26aulast%3DHawkins%26aufirst%3DP.%2BC.%2BD.%26aulast%3DErmondi%26aufirst%3DG.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DConformational%2520Sampling%2520of%2520Macrocyclic%2520Drugs%2520in%2520Different%2520Environments%253A%2520Can%2520We%2520Find%2520the%2520Relevant%2520Conformations%253F%26jtitle%3DACS%2520Omega%26date%3D2018%26volume%3D3%26spage%3D11742%26epage%3D11757%26doi%3D10.1021%2Facsomega.8b01379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ruder, S.</span></span> <span> </span><span class="NLM_article-title">An Overview of Gradient Descent Optimization Algorithms</span>. <span class="NLM_year">2016</span>, arXiv:1609.04747. arXiv.org e-Print archive. <a href="https://arxiv.org/abs/1609.04747" class="extLink">https://arxiv.org/abs/1609.04747</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=S.+Ruder&title=An+Overview+of+Gradient+Descent+Optimization+Algorithms"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRuder%26aufirst%3DS.%26atitle%3DAn%2520Overview%2520of%2520Gradient%2520Descent%2520Optimization%2520Algorithms%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christ, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Architecture, Implementation and Parallelisation of the GROMOS Software for Biomolecular Simulation</span>. <i>Comput. Phys. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>183</i></span>,  <span class="NLM_fpage">890</span>– <span class="NLM_lpage">903</span>, <span class="refDoi"> DOI: 10.1016/j.cpc.2011.12.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.cpc.2011.12.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFKiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=183&publication_year=2012&pages=890-903&author=N.+Schmidauthor=C.+D.+Christauthor=M.+Christenauthor=A.+P.+Eichenbergerauthor=W.+F.+Van+Gunsteren&title=Architecture%2C+Implementation+and+Parallelisation+of+the+GROMOS+Software+for+Biomolecular+Simulation&doi=10.1016%2Fj.cpc.2011.12.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Architecture, implementation and parallelisation of the GROMOS software for biomolecular simulation</span></div><div class="casAuthors">Schmid, Nathan; Christ, Clara D.; Christen, Markus; Eichenberger, Andreas P.; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Computer Physics Communications</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">183</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">890-903</span>CODEN:
                <span class="NLM_cas:coden">CPHCBZ</span>;
        ISSN:<span class="NLM_cas:issn">0010-4655</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">In this work, the design of the latest version of the GROMOS software for biomol. simulation, GROMOS11, is discussed.  Detailed organization and class descriptions of the MD++ simulation program and the GROMOS++ anal. package are given.  It is shown how the code was documented, how it can be easily modified and extended, how debugging of it is carried out.  Addnl. efficiency and parallelization concepts are presented and benchmarked.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3_FLsjZIvWLVg90H21EOLACvtfcHk0ljYYDBBYzA7dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFKiurc%253D&md5=9b2d6aeebff579549e24c044e9ef8ef4</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.cpc.2011.12.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cpc.2011.12.014%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DChrist%26aufirst%3DC.%2BD.%26aulast%3DChristen%26aufirst%3DM.%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DArchitecture%252C%2520Implementation%2520and%2520Parallelisation%2520of%2520the%2520GROMOS%2520Software%2520for%2520Biomolecular%2520Simulation%26jtitle%3DComput.%2520Phys.%2520Commun.%26date%3D2012%26volume%3D183%26spage%3D890%26epage%3D903%26doi%3D10.1016%2Fj.cpc.2011.12.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choutko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mark, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Definition and Testing of the GROMOS Force-Field Versions 54A7 and 54B7</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">856</span>, <span class="refDoi"> DOI: 10.1007/s00249-011-0700-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1007%2Fs00249-011-0700-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=21533652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=843-856&author=N.+Schmidauthor=A.+P.+Eichenbergerauthor=A.+Choutkoauthor=S.+Rinikerauthor=M.+Wingerauthor=A.+E.+Markauthor=W.+F.+Van+Gunsteren&title=Definition+and+Testing+of+the+GROMOS+Force-Field+Versions+54A7+and+54B7&doi=10.1007%2Fs00249-011-0700-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Definition and testing of the GROMOS force-field versions 54A7 and 54B7</span></div><div class="casAuthors">Schmid, Nathan; Eichenberger, Andreas P.; Choutko, Alexandra; Riniker, Sereina; Winger, Moritz; Mark, Alan E.; Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">843-856</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">New parameter sets of the GROMOS biomol. force field, 54A7 and 54B7, are introduced.  These parameter sets summarize some previously published force field modifications: The 53A6 helical propensities are cor. through new φ/ψ torsional angle terms and a modification of the N-H, C=O repulsion, a new atom type for a charged -CH3 in the choline moiety is added, the Na+ and Cl- ions are modified to reproduce the free energy of hydration, and addnl. improper torsional angle types for free energy calcns. involving a chirality change are introduced.  The new helical propensity modification is tested using the benchmark proteins hen egg-white lysozyme, fox1 RNA binding domain, chorismate mutase and the GCN4-p1 peptide.  The stability of the proteins is improved in comparison with the 53A6 force field, and good agreement with a range of primary exptl. data is obtained.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3tjHc1auDa7Vg90H21EOLACvtfcHk0ljYYDBBYzA7dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVCgs74%253D&md5=a1cd133f1e663da949324060e00c950d</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1007%2Fs00249-011-0700-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00249-011-0700-9%26sid%3Dliteratum%253Aachs%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DChoutko%26aufirst%3DA.%26aulast%3DRiniker%26aufirst%3DS.%26aulast%3DWinger%26aufirst%3DM.%26aulast%3DMark%26aufirst%3DA.%2BE.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DDefinition%2520and%2520Testing%2520of%2520the%2520GROMOS%2520Force-Field%2520Versions%252054A7%2520and%252054B7%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D2011%26volume%3D40%26spage%3D843%26epage%3D856%26doi%3D10.1007%2Fs00249-011-0700-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span>; <span class="NLM_string-name">Postma, J. P.</span>; <span class="NLM_string-name">van Gunsteren, W. F.</span>; <span class="NLM_string-name">Hermans, J.</span></span> <span> </span><span class="NLM_article-title">Interaction Models for Water in Relation to Protein Hydration</span>. In  <i>Intermolecular Forces</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pullman, B.</span></span>, Ed.; <span class="NLM_publisher-name">Springer</span>, <span class="NLM_year">1981</span>; Vol.  <span class="NLM_volume">14</span>, pp  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">342</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1007%2F978-94-015-7658-1_21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1981&pages=331-342&author=H.+J.+C.+Berendsen&author=J.+P.+Postma&author=W.+F.+van+Gunsteren&author=J.+Hermansauthor=B.+Pullman&title=Intermolecular+Forces"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1007%2F978-94-015-7658-1_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-94-015-7658-1_21%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DInteraction%2520Models%2520for%2520Water%2520in%2520Relation%2520to%2520Protein%2520Hydration%26btitle%3DIntermolecular%2520Forces%26aulast%3DPullman%26aufirst%3DB.%26pub%3DSpringer%26date%3D1981%26volume%3D14%26spage%3D331%26epage%3D342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tironi, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">A Molecular Dynamics Simulation Study of Chloroform</span>. <i>Mol. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">403</span>, <span class="refDoi"> DOI: 10.1080/00268979400101331</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1080%2F00268979400101331" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK2cXntVSnsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1994&pages=381-403&author=I.+G.+Tironiauthor=W.+F.+Van+Gunsteren&title=A+Molecular+Dynamics+Simulation+Study+of+Chloroform&doi=10.1080%2F00268979400101331"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">A molecular dynamics simulation study of chloroform</span></div><div class="casAuthors">Tironi, Ilario G.; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Physics</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">381-403</span>CODEN:
                <span class="NLM_cas:coden">MOPHAM</span>;
        ISSN:<span class="NLM_cas:issn">0026-8976</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis</span>)
        </div><div class="casAbstract">Three different chloroform models have been investigated using mol. dynamics computer simulation.  The thermodn., structural and dynamic properties of the various models were investigated in detail.  In particular the potential energies, diffusion coeffs. and rotational correlation times obtained for each model are compared with expt.  It is found that the theory of rotational Brownian motion fails in describing the rotational diffusion of chloroform.  The force field of Dietz and Heinzinger was found to give good overall agreement with expt.  An extended investigation of this chloroform model has been performed.  Values are reported for the isothermal compressibility, the thermal expansion coeff. and the const. vol. heat capacity.  The values agree well with expt.  The static and frequency dependent dielec. permittivity were computed from a 1.2 ns simulation conducted under reaction field boundary conditions.  Considering the fact that the model is rigid with fixed partial charges, the static dielec. const. and Debye relaxation time compare well with expt.  From the same simulation the shear viscosity was computed using the off-diagonal elements of the pressure tensor, both via an Einstein type relation and via a Green-Kubo equation.  The calcd. viscosities show good agreement with exptl. values.  The excess Helmholtz energy is calcd. using the thermodn. integration technique and simulations of 50 and 80 ps.  The value obtained for the excess Helmholtz energy matches the theor. value within a few per cent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom3066SYtKJrVg90H21EOLACvtfcHk0li9KcmOYjPD9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXntVSnsbc%253D&md5=1a1218133613b92f48c7d8d658d177a8</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1080%2F00268979400101331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00268979400101331%26sid%3Dliteratum%253Aachs%26aulast%3DTironi%26aufirst%3DI.%2BG.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DA%2520Molecular%2520Dynamics%2520Simulation%2520Study%2520of%2520Chloroform%26jtitle%3DMol.%2520Phys.%26date%3D1994%26volume%3D83%26spage%3D381%26epage%3D403%26doi%3D10.1080%2F00268979400101331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">A Leap-Frog Algorithm for Stochastic Dynamics</span>. <i>Mol. Simul.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">185</span>, <span class="refDoi"> DOI: 10.1080/08927028808080941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1080%2F08927028808080941" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1988&pages=173-185&author=W.+F.+Van%0AGunsterenauthor=H.+J.+C.+Berendsen&title=A+Leap-Frog+Algorithm+for+Stochastic+Dynamics&doi=10.1080%2F08927028808080941"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1080%2F08927028808080941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F08927028808080941%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DA%2520Leap-Frog%2520Algorithm%2520for%2520Stochastic%2520Dynamics%26jtitle%3DMol.%2520Simul.%26date%3D1988%26volume%3D1%26spage%3D173%26epage%3D185%26doi%3D10.1080%2F08927028808080941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryckaert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccotti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span> <span> </span><span class="NLM_article-title">Numerical Integration of the Cartesian Equations of Motion of a System with Constraints: Molecular Dynamics of n-Alkanes</span>. <i>J. Comput. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1016/0021-9991(77)90098-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2F0021-9991%2877%2990098-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1977&pages=327-341&author=J.+P.+Ryckaertauthor=G.+Ciccottiauthor=H.+J.+C.+Berendsen&title=Numerical+Integration+of+the+Cartesian+Equations+of+Motion+of+a+System+with+Constraints%3A+Molecular+Dynamics+of+n-Alkanes&doi=10.1016%2F0021-9991%2877%2990098-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Numerical integration of the Cartesian equations of motion of a system with constraints:  molecular dynamics of n-alkanes</span></div><div class="casAuthors">Ryckaert, Jean Paul; Ciccotti, Giovanni; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Physics</span>
        (<span class="NLM_cas:date">1977</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">327-41</span>CODEN:
                <span class="NLM_cas:coden">JCTPAH</span>;
        ISSN:<span class="NLM_cas:issn">0021-9991</span>.
    </div><div class="casAbstract">A numerical algorithm integrating the 3N Cartesian equation of motion of a system of N points subject to holonomic constraints is applied to mol. dynamics simulation of a liq. of 64 butane mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPlhG1U3Uw7Vg90H21EOLACvtfcHk0li5zDINmo0tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE2sXktVGhsL4%253D&md5=b4aecddfde149117813a5ea4f5353ce2</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2F0021-9991%2877%2990098-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0021-9991%252877%252990098-5%26sid%3Dliteratum%253Aachs%26aulast%3DRyckaert%26aufirst%3DJ.%2BP.%26aulast%3DCiccotti%26aufirst%3DG.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DNumerical%2520Integration%2520of%2520the%2520Cartesian%2520Equations%2520of%2520Motion%2520of%2520a%2520System%2520with%2520Constraints%253A%2520Molecular%2520Dynamics%2520of%2520n-Alkanes%26jtitle%3DJ.%2520Comput.%2520Phys.%26date%3D1977%26volume%3D23%26spage%3D327%26epage%3D341%26doi%3D10.1016%2F0021-9991%2877%2990098-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tironi, I. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sperb, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">A Generalized Reaction Field Method for Molecular Dynamics Simulations</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">5451</span>– <span class="NLM_lpage">5459</span>, <span class="refDoi"> DOI: 10.1063/1.469273</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.469273" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaK2MXkslOmsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=1995&pages=5451-5459&author=I.+G.+Tironiauthor=R.+Sperbauthor=P.+E.+Smithauthor=W.+F.+van+Gunsteren&title=A+Generalized+Reaction+Field+Method+for+Molecular+Dynamics+Simulations&doi=10.1063%2F1.469273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A generalized reaction field method for molecular dynamics simulations</span></div><div class="casAuthors">Tironi, Ilario G.; Sperb, Rene; Smith, Paul E.; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">5451-9</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Mol.-dynamics simulations of ionic systems require the inclusion of long-range electrostatic forces.  We propose an expression for the long-range electrostatic forces based on an anal. soln. of the Poisson-Boltzmann equation outside a spherical cutoff, which can easily be implemented in mol. simulation programs.  An anal. soln. of the linearized Poisson-Boltzmann (PB) equation valid in a spherical region is obtained.  From this general soln. special expressions are derived for evaluating the electrostatic potential and its deriv. at the origin of the sphere.  These expressions have been implemented for mol.-dynamics (MD) simulations, such that the surface of the cutoff sphere around a charged particle is identified with the spherical boundary of the Poisson-Boltzmann problem.  The anal. soln. of the Poisson-Boltzmann equation is valid for the cutoff sphere and can be used for calcg. the reaction field forces on the central charge, assuming a uniform continuum of given ionic strength beyond the cutoff.  MD simulations are preformed for a periodic system consisting of 2127 SPC water mols. with 40 NaCl ions (1 M).  We compared the structural and dynamic results obtained from MD simulations in which the long-range electrostatic interactions are treated differently; using a cutoff radius, using a cutoff radius and a Poisson-Boltzmann generalized reaction field force, and using the Ewald summation.  Application of the Poisson-Boltzmann generalized reaction field gives a dramatic improvement of the structure of the soln. compared to a simple cutoff treatment, at no extra computational cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3IHTuIOoegbVg90H21EOLACvtfcHk0li5zDINmo0tNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXkslOmsLg%253D&md5=b20169702e5929b6e5967d7cbc2fdb01</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1063%2F1.469273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.469273%26sid%3Dliteratum%253Aachs%26aulast%3DTironi%26aufirst%3DI.%2BG.%26aulast%3DSperb%26aufirst%3DR.%26aulast%3DSmith%26aufirst%3DP.%2BE.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DA%2520Generalized%2520Reaction%2520Field%2520Method%2520for%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1995%26volume%3D102%26spage%3D5451%26epage%3D5459%26doi%3D10.1063%2F1.469273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinz, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hünenberger, P. H.</span></span> <span> </span><span class="NLM_article-title">Comparison of Four Methods to Compute the Dielectric Permittivity of Liquids from Molecular Dynamics Simulations</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>115</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1136</span>, <span class="refDoi"> DOI: 10.1063/1.1379764</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.1379764" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFCmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=115&publication_year=2001&pages=1125-1136&author=T.+N.+Heinzauthor=W.+F.+Van+Gunsterenauthor=P.+H.+H%C3%BCnenberger&title=Comparison+of+Four+Methods+to+Compute+the+Dielectric+Permittivity+of+Liquids+from+Molecular+Dynamics+Simulations&doi=10.1063%2F1.1379764"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of four methods to compute the dielectric permittivity of liquids from molecular dynamics simulations</span></div><div class="casAuthors">Heinz, Tim N.; van Gunsteren, Wilfred F.; Hunenberger, Philippe H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">115</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1125-1136</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">Four methods to compute the dielec. permittivity ε of a liq. from mol. simulations are compared in the context of the simple point charge (SPC) water model.  In the 1st method (unrestrained method), ε is evaluated from the fluctuations of the box dipole moment M, monitored during a single equil. simulation.  In the three other methods, ε is evaluated from the probability distribution p(M) of the dipole moment norm.  This distribution is itself evaluated in three different ways: (i) from multiple simulations involving a M-dependent biasing potential (umbrella-sampling method), (ii) from multiple simulations involving a constrained dipole moment norm (M-constraint method), or (iii) from fitting of incomplete p(M) ests. to a Maxwell distribution (fitting method).  The four methods converge to an identical est. of ε=61 ± 1 for SPC water (256 mols., reaction-field electrostatics).  The convergence properties, advantages, and drawbacks of the different methods are analyzed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXa1Mr7u_k2bVg90H21EOLACvtfcHk0lh9mPq-yXR-7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXltFCmurg%253D&md5=626fcf439551260b8e297b818d9e3319</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1063%2F1.1379764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1379764%26sid%3Dliteratum%253Aachs%26aulast%3DHeinz%26aufirst%3DT.%2BN.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DH%25C3%25BCnenberger%26aufirst%3DP.%2BH.%26atitle%3DComparison%2520of%2520Four%2520Methods%2520to%2520Compute%2520the%2520Dielectric%2520Permittivity%2520of%2520Liquids%2520from%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2001%26volume%3D115%26spage%3D1125%26epage%3D1136%26doi%3D10.1063%2F1.1379764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berendsen, H. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Postma, J. P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gunsteren, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiNola, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haak, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics with Coupling to an External Bath</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>81</i></span>,  <span class="NLM_fpage">3684</span>– <span class="NLM_lpage">3690</span>, <span class="refDoi"> DOI: 10.1063/1.448118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.448118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=81&publication_year=1984&pages=3684-3690&author=H.+J.+C.+Berendsenauthor=J.+P.+M.+Postmaauthor=W.+F.+van+Gunsterenauthor=A.+DiNolaauthor=J.+R.+Haak&title=Molecular+Dynamics+with+Coupling+to+an+External+Bath&doi=10.1063%2F1.448118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics with coupling to an external bath</span></div><div class="casAuthors">Berendsen, H. J. C.; Postma, J. P. M.; Van Gunsteren, W. F.; DiNola, A.; Haak, J. R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">81</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3684-90</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">In mol. dynamics (MD) simulations, the need often arises to maintain such parameters as temp. or pressure rather than energy and vol., or to impose gradients for studying transport properties in nonequil. MD.  A method is described to realize coupling to an external bath with const. temp. or pressure with adjustable time consts. for the coupling.  The method is easily extendable to other variables and to gradients, and can be applied also to polyat. mols. involving internal constraints.  The influence of coupling time consts. on dynamical variables is evaluated.  A leap-frog algorithm is presented for the general case involving constraints with coupling to both a const. temp. and a const. pressure bath.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoecrpUZWkh37Vg90H21EOLACvtfcHk0lh9mPq-yXR-7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXmtlGksbY%253D&md5=5510dc00297d63b91ee3a7a4ae5aacb1</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1063%2F1.448118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.448118%26sid%3Dliteratum%253Aachs%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26aulast%3DPostma%26aufirst%3DJ.%2BP.%2BM.%26aulast%3Dvan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DDiNola%26aufirst%3DA.%26aulast%3DHaak%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520Dynamics%2520with%2520Coupling%2520to%2520an%2520External%2520Bath%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1984%26volume%3D81%26spage%3D3684%26epage%3D3690%26doi%3D10.1063%2F1.448118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scherer, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trendelkamp-Schroer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pérez-Hernández, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plattner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wehmeyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinz, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noé, F.</span></span> <span> </span><span class="NLM_article-title">PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">5525</span>– <span class="NLM_lpage">5542</span>, <span class="refDoi"> DOI: 10.1021/acs.jctc.5b00743</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00743" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Gksr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=5525-5542&author=M.+K.+Schererauthor=B.+Trendelkamp-Schroerauthor=F.+Paulauthor=G.+P%C3%A9rez-Hern%C3%A1ndezauthor=M.+Hoffmannauthor=N.+Plattnerauthor=C.+Wehmeyerauthor=J.-H.+Prinzauthor=F.+No%C3%A9&title=PyEMMA+2%3A+A+Software+Package+for+Estimation%2C+Validation%2C+and+Analysis+of+Markov+Models&doi=10.1021%2Facs.jctc.5b00743"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">PyEMMA 2: A Software Package for Estimation, Validation, and Analysis of Markov Models</span></div><div class="casAuthors">Scherer, Martin K.; Trendelkamp-Schroer, Benjamin; Paul, Fabian; Perez-Hernandez, Guillermo; Hoffmann, Moritz; Plattner, Nuria; Wehmeyer, Christoph; Prinz, Jan-Hendrik; Noe, Frank</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5525-5542</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Markov (state) models (MSMs) and related models of mol. kinetics have recently received a surge of interest as they can systematically reconcile simulation data from either a few long or many short simulations and allow us to analyze the essential metastable structures, thermodn., and kinetics of the mol. system under investigation.  However, the estn., validation, and anal. of such models is far from trivial and involves sophisticated and often numerically sensitive methods.  In this work, we present the open-source Python package PyEMMA (pyemma.org) that provides accurate and efficient algorithms for kinetic model construction.  PyEMMA can read all common mol. dynamics data formats, helps in the selection of input features, provides easy access to dimension redn. algorithms such as principal component anal. (PCA) and time-lagged independent component anal. (TICA) and clustering algorithms such as k-means, and contains estimators for MSMs, hidden Markov models, and several other models.  Systematic model validation and error calcn. methods are provided.  PyEMMA offers a wealth of anal. functions such that the user can conveniently compute mol. observables of interest.  We have derived a systematic and accurate way to coarse-grain MSMs to few states and to illustrate the structures of the metastable states of the system.  Plotting functions to produce a manuscript-ready presentation of the results are available.  In this work, we demonstrate the features of the software and show new methodol. concepts and results produced by PyEMMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY6T9WAZpy17Vg90H21EOLACvtfcHk0lh9mPq-yXR-7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Gksr7N&md5=dc04b91b37acb7c0bfae1bc0475dcd93</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00743&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00743%26sid%3Dliteratum%253Aachs%26aulast%3DScherer%26aufirst%3DM.%2BK.%26aulast%3DTrendelkamp-Schroer%26aufirst%3DB.%26aulast%3DPaul%26aufirst%3DF.%26aulast%3DP%25C3%25A9rez-Hern%25C3%25A1ndez%26aufirst%3DG.%26aulast%3DHoffmann%26aufirst%3DM.%26aulast%3DPlattner%26aufirst%3DN.%26aulast%3DWehmeyer%26aufirst%3DC.%26aulast%3DPrinz%26aufirst%3DJ.-H.%26aulast%3DNo%25C3%25A9%26aufirst%3DF.%26atitle%3DPyEMMA%25202%253A%2520A%2520Software%2520Package%2520for%2520Estimation%252C%2520Validation%252C%2520and%2520Analysis%2520of%2520Markov%2520Models%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2015%26volume%3D11%26spage%3D5525%26epage%3D5542%26doi%3D10.1021%2Facs.jctc.5b00743" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGibbon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beauchamp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrigan, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swails, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernández, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwantes, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">1528</span>– <span class="NLM_lpage">1532</span>, <span class="refDoi"> DOI: 10.1016/j.bpj.2015.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.bpj.2015.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=26488642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKhu7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2015&pages=1528-1532&author=R.+T.+McGibbonauthor=K.+A.+Beauchampauthor=M.+P.+Harriganauthor=C.+Kleinauthor=J.+M.+Swailsauthor=C.+X.+Hern%C3%A1ndezauthor=C.+R.+Schwantesauthor=L.+P.+Wangauthor=T.+J.+Laneauthor=V.+S.+Pande&title=MDTraj%3A+A+Modern+Open+Library+for+the+Analysis+of+Molecular+Dynamics+Trajectories&doi=10.1016%2Fj.bpj.2015.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">MDTraj: A Modern Open Library for the Analysis of Molecular Dynamics Trajectories</span></div><div class="casAuthors">McGibbon, Robert T.; Beauchamp, Kyle A.; Harrigan, Matthew P.; Klein, Christoph; Swails, Jason M.; Hernandez, Carlos X.; Schwantes, Christian R.; Wang, Lee-Ping; Lane, Thomas J.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1528-1532</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">As mol. dynamics (MD) simulations continue to evolve into powerful computational tools for studying complex biomol. systems, the necessity of flexible and easy-to-use software tools for the anal. of these simulations is growing.  We have developed MDTraj, a modern, lightwt., and fast software package for analyzing MD simulations.  MDTraj reads and writes trajectory data in a wide variety of commonly used formats.  It provides a large no. of trajectory anal. capabilities including minimal root-mean-square-deviation calcns., secondary structure assignment, and the extn. of common order parameters.  The package has a strong focus on interoperability with the wider scientific Python ecosystem, bridging the gap between MD data and the rapidly growing collection of industry-std. statistical anal. and visualization tools in Python.  MDTraj is a powerful and user-friendly software package that simplifies the anal. of MD data and connects these datasets with the modern interactive data science software ecosystem in Python.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU5T-L7qmT8LVg90H21EOLACvtfcHk0ljvd9xGSqhGgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKhu7%252FI&md5=9cbd2138ce200e7d63fda8ae756df43f</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.bpj.2015.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bpj.2015.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DMcGibbon%26aufirst%3DR.%2BT.%26aulast%3DBeauchamp%26aufirst%3DK.%2BA.%26aulast%3DHarrigan%26aufirst%3DM.%2BP.%26aulast%3DKlein%26aufirst%3DC.%26aulast%3DSwails%26aufirst%3DJ.%2BM.%26aulast%3DHern%25C3%25A1ndez%26aufirst%3DC.%2BX.%26aulast%3DSchwantes%26aufirst%3DC.%2BR.%26aulast%3DWang%26aufirst%3DL.%2BP.%26aulast%3DLane%26aufirst%3DT.%2BJ.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMDTraj%253A%2520A%2520Modern%2520Open%2520Library%2520for%2520the%2520Analysis%2520of%2520Molecular%2520Dynamics%2520Trajectories%26jtitle%3DBiophys.%2520J.%26date%3D2015%26volume%3D109%26spage%3D1528%26epage%3D1532%26doi%3D10.1016%2Fj.bpj.2015.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Molgedey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, H. G.</span></span> <span> </span><span class="NLM_article-title">Separation of a Mixture of Independent Signals Using Time Delayed Correlations</span>. <i>Phys. Rev. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">3634</span>– <span class="NLM_lpage">3637</span>, <span class="refDoi"> DOI: 10.1103/PhysRevLett.72.3634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1103%2FPhysRevLett.72.3634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10056251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC2sfpsFKisQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=1994&pages=3634-3637&author=L.+Molgedeyauthor=H.+G.+Schuster&title=Separation+of+a+Mixture+of+Independent+Signals+Using+Time+Delayed+Correlations&doi=10.1103%2FPhysRevLett.72.3634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Separation of a mixture of independent signals using time delayed correlations</span></div><div class="casAuthors">Molgedey; Schuster</div><div class="citationInfo"><span class="NLM_cas:title">Physical review letters</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">3634-3637</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKlcmWVlGfwsScOqQTSkOffW6udTcc2eYM5Gjpsd8ep7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sfpsFKisQ%253D%253D&md5=fc21b44ff5e05410e236cad768bcee81</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.72.3634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.72.3634%26sid%3Dliteratum%253Aachs%26aulast%3DMolgedey%26aufirst%3DL.%26aulast%3DSchuster%26aufirst%3DH.%2BG.%26atitle%3DSeparation%2520of%2520a%2520Mixture%2520of%2520Independent%2520Signals%2520Using%2520Time%2520Delayed%2520Correlations%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1994%26volume%3D72%26spage%3D3634%26epage%3D3637%26doi%3D10.1103%2FPhysRevLett.72.3634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keller, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daura, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">Comparing Geometric and Kinetic Cluster Algorithms for Molecular Simulation Data</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>132</i></span>, <span class="NLM_elocation-id">074110</span> <span class="refDoi"> DOI: 10.1063/1.3301140</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F1.3301140" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=20170218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1OqurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&author=B.+Kellerauthor=X.+Dauraauthor=W.+F.+Van+Gunsteren&title=Comparing+Geometric+and+Kinetic+Cluster+Algorithms+for+Molecular+Simulation+Data&doi=10.1063%2F1.3301140"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Comparing geometric and kinetic cluster algorithms for molecular simulation data</span></div><div class="casAuthors">Keller, Bettina; Daura, Xavier; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">074110/1-074110/16</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The identification of metastable states of a mol. plays an important role in the interpretation of mol. simulation data because the free-energy surface, the relative populations in this landscape, and ultimately also the dynamics of the mol. under study can be described in terms of these states.  We compare the results of three different geometric cluster algorithms (neighbor algorithm, K-medoids algorithm, and common-nearest-neighbor algorithm) among each other and to the results of a kinetic cluster algorithm.  First, we demonstrate the characteristics of each of the geometric cluster algorithms using five two-dimensional data sets.  Second, we analyze the mol. dynamics data of a β-heptapeptide in methanol - a mol. that exhibits a distinct folded state, a structurally diverse unfolded state, and a fast folding/unfolding equil. - using both geometric and kinetic cluster algorithms.  We find that geometric clustering strongly depends on the algorithm used and that the d. based common-nearest-neighbor algorithm is the most robust of the three geometric cluster algorithms with respect to variations in the input parameters and the distance metric.  When comparing the geometric cluster results to the metastable states of the β-heptapeptide as identified by kinetic clustering, we find that in most cases the folded state is identified correctly but the overlap of geometric clusters with further metastable states is often at best approx.  (c) 2010 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKj0e-QRDQDrVg90H21EOLACvtfcHk0ljQrVFH97WkiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1OqurY%253D&md5=753b03fa1abb3a218e177e8cfd026964</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1063%2F1.3301140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.3301140%26sid%3Dliteratum%253Aachs%26aulast%3DKeller%26aufirst%3DB.%26aulast%3DDaura%26aufirst%3DX.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DComparing%2520Geometric%2520and%2520Kinetic%2520Cluster%2520Algorithms%2520for%2520Molecular%2520Simulation%2520Data%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2010%26volume%3D132%26doi%3D10.1063%2F1.3301140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weiß, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ries, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riniker, S.</span></span> <span> </span><span class="NLM_article-title">Volume-Scaled Common Nearest Neighbor Clustering Algorithm with Free-Energy Hierarchy</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>154</i></span>,  <span class="NLM_fpage">084106</span>, <span class="refDoi"> DOI: 10.1063/5.0025797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1063%2F5.0025797" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=33639726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3MXltFOjurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=154&publication_year=2021&pages=084106&author=R.+G.+Wei%C3%9Fauthor=B.+Riesauthor=S.+Wangauthor=S.+Riniker&title=Volume-Scaled+Common+Nearest+Neighbor+Clustering+Algorithm+with+Free-Energy+Hierarchy&doi=10.1063%2F5.0025797"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Volume-scaled common nearest neighbor clustering algorithm with free-energy hierarchy</span></div><div class="casAuthors">Weiss, R. Gregor; Ries, Benjamin; Wang, Shuzhe; Riniker, Sereina</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">154</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">084106</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">The combination of Markov state modeling (MSM) and mol. dynamics (MD) simulations has been shown in recent years to be a valuable approach to unravel the slow processes of mol. systems with increasing complexity.  While the algorithms for intermediate steps in the MSM workflow such as featurization and dimensionality redn. have been specifically adapted to MD datasets, conventional clustering methods are generally applied to the discretization step.  This work adds to recent efforts to develop specialized d.-based clustering algorithms for the Boltzmann-weighted data from MD simulations.  We introduce the vol.-scaled common nearest neighbor (vs-CNN) clustering that is an adapted version of the common nearest neighbor (CNN) algorithm.  A major advantage of the proposed algorithm is that the introduced d.-based criterion directly links to a free-energy notion via Boltzmann inversion.  Such a free-energy perspective allows a straightforward hierarchical scheme to identify conformational clusters at different levels of a generally rugged free-energy landscape of complex mol. systems.  (c) 2021 American Institute of Physics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwvedFcqDbYrVg90H21EOLACvtfcHk0ljQrVFH97WkiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXltFOjurs%253D&md5=41655db8575a58aa74693d4bc3cf7495</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1063%2F5.0025797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F5.0025797%26sid%3Dliteratum%253Aachs%26aulast%3DWei%25C3%259F%26aufirst%3DR.%2BG.%26aulast%3DRies%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DRiniker%26aufirst%3DS.%26atitle%3DVolume-Scaled%2520Common%2520Nearest%2520Neighbor%2520Clustering%2520Algorithm%2520with%2520Free-Energy%2520Hierarchy%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2021%26volume%3D154%26spage%3D084106%26doi%3D10.1063%2F5.0025797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eichenberger, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allison, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolenc, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geerke, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horta, B. A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oostenbrink, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gunsteren, W. F.</span></span> <span> </span><span class="NLM_article-title">GROMOS++ Software for the Analysis of Biomolecular Simulation Trajectories</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">3379</span>– <span class="NLM_lpage">3390</span>, <span class="refDoi"> DOI: 10.1021/ct2003622</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct2003622" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFOit77O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=3379-3390&author=A.+P.+Eichenbergerauthor=J.+R.+Allisonauthor=J.+Dolencauthor=D.+P.+Geerkeauthor=B.+A.+C.+Hortaauthor=K.+Meierauthor=C.+Oostenbrinkauthor=N.+Schmidauthor=D.+Steinerauthor=D.+Wangauthor=W.+F.+Van+Gunsteren&title=GROMOS%2B%2B+Software+for+the+Analysis+of+Biomolecular+Simulation+Trajectories&doi=10.1021%2Fct2003622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">GROMOS++ Software for the Analysis of Biomolecular Simulation Trajectories</span></div><div class="casAuthors">Eichenberger, Andreas P.; Allison, Jane R.; Dolenc, Jozica; Geerke, Daan P.; Horta, Bruno A. C.; Meier, Katharina; Oostenbrink, Chris; Schmid, Nathan; Steiner, Denise; Wang, Dongqi; van Gunsteren, Wilfred F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3379-3390</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  GROMOS++ is a set of C++ programs for pre- and postprocessing of mol. dynamics simulation trajectories and as such is part of the GROningen Mol. Simulation software for (bio)mol. simulation.  It contains more than 70 programs that can be used to prep. data for the prodn. of mol. simulation trajectories and to analyze these.  These programs are reviewed and the various structural, dynamic, and thermodn. quantities that can be analyzed using time series, correlation functions, and distributions are described together with tech. aspects of their implementation in GROMOS.  A few examples of the use of GROMOS++ for the anal. of MD trajectories are given.  A full list of all GROMOS++ programs, together with an indication of their capabilities, is given in the Appendix.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoODOTei0lrjbVg90H21EOLACvtfcHk0ljQrVFH97WkiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFOit77O&md5=b87a1d1b93445e9fe5a67ddbd3f1197f</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fct2003622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct2003622%26sid%3Dliteratum%253Aachs%26aulast%3DEichenberger%26aufirst%3DA.%2BP.%26aulast%3DAllison%26aufirst%3DJ.%2BR.%26aulast%3DDolenc%26aufirst%3DJ.%26aulast%3DGeerke%26aufirst%3DD.%2BP.%26aulast%3DHorta%26aufirst%3DB.%2BA.%2BC.%26aulast%3DMeier%26aufirst%3DK.%26aulast%3DOostenbrink%26aufirst%3DC.%26aulast%3DSchmid%26aufirst%3DN.%26aulast%3DSteiner%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26atitle%3DGROMOS%252B%252B%2520Software%2520for%2520the%2520Analysis%2520of%2520Biomolecular%2520Simulation%2520Trajectories%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2011%26volume%3D7%26spage%3D3379%26epage%3D3390%26doi%3D10.1021%2Fct2003622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vögeli, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Güntert, P.</span></span> <span> </span><span class="NLM_article-title">Compiled Data Set of Exact NOE Distance Limits, Residual Dipolar Couplings and Scalar Couplings for the Protein GB3</span>. <i>Data Brief</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">99</span>– <span class="NLM_lpage">106</span>, <span class="refDoi"> DOI: 10.1016/j.dib.2015.08.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1016%2Fj.dib.2015.08.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=26504890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A280%3ADC%252BC28zlvFKhuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=99-106&author=B.+V%C3%B6geliauthor=S.+Olssonauthor=R.+Riekauthor=P.+G%C3%BCntert&title=Compiled+Data+Set+of+Exact+NOE+Distance+Limits%2C+Residual+Dipolar+Couplings+and+Scalar+Couplings+for+the+Protein+GB3&doi=10.1016%2Fj.dib.2015.08.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Compiled data set of exact NOE distance limits, residual dipolar couplings and scalar couplings for the protein GB3</span></div><div class="casAuthors">Vogeli Beat; Riek Roland; Olsson Simon; Guntert Peter</div><div class="citationInfo"><span class="NLM_cas:title">Data in brief</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">99-106</span>
        ISSN:<span class="NLM_cas:issn">2352-3409</span>.
    </div><div class="casAbstract">We compiled an NMR data set consisting of exact nuclear Overhauser enhancement (eNOE) distance limits, residual dipolar couplings (RDCs) and scalar (J) couplings for GB3, which forms one of the largest and most diverse data set for structural characterization of a protein to date.  All data have small experimental errors, which are carefully estimated.  We use the data in the research article Vogeli et al., 2015, Complementarity and congruence between exact NOEs and traditional NMR probes for spatial decoding of protein dynamics, J.  Struct.  Biol., 191, 3, 306-317, doi:10.1016/j.jsb.2015.07.008 [1] for cross-validation in multiple-state structural ensemble calculation.  We advocate this set to be an ideal test case for molecular dynamics simulations and structure calculations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQGhCQAQpdjhJ5E29YNGvQfW6udTcc2ea8HaihM3XTBrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zlvFKhuw%253D%253D&md5=feecb999e7043e445f1439d8f752c38e</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1016%2Fj.dib.2015.08.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.dib.2015.08.020%26sid%3Dliteratum%253Aachs%26aulast%3DV%25C3%25B6geli%26aufirst%3DB.%26aulast%3DOlsson%26aufirst%3DS.%26aulast%3DRiek%26aufirst%3DR.%26aulast%3DG%25C3%25BCntert%26aufirst%3DP.%26atitle%3DCompiled%2520Data%2520Set%2520of%2520Exact%2520NOE%2520Distance%2520Limits%252C%2520Residual%2520Dipolar%2520Couplings%2520and%2520Scalar%2520Couplings%2520for%2520the%2520Protein%2520GB3%26jtitle%3DData%2520Brief%26date%3D2015%26volume%3D5%26spage%3D99%26epage%3D106%26doi%3D10.1016%2Fj.dib.2015.08.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Virtanen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gommers, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliphant, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haberland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cournapeau, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burovski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weckesser, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bright, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Walt, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millman, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayorov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, A. R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kern, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carey, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polat, İ.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">VanderPlas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laxalde, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perktold, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimrman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henriksen, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quintero, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Archibald, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedregosa, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Mulbregt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijaykumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietro</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothberg, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilboll, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kloeckner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scopatz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokem, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulton, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Häggström, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholson, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hagen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasechnik, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olivetti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wieser, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenders, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Audren, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Probst, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silterra, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Webber, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönberger, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Miranda Cardoso, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez, J. L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez-Iglesias, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuczynski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tritz, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thoma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newville, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kümmerer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolingbroke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tartre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pak, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowaczyk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shebanov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavlyk, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodtkorb, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGibbon, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldbauer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tygier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sievert, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">More, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pudlik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oshima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pingel, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robitaille, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauss, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upadhyay, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halchenko, Y. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vázquez-Baeza, Y.</span></span> <span> </span><span class="NLM_article-title">SciPy 1.0: Fundamental Algorithms for Scientific Computing in Python</span>. <i>Nat. Methods</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1038/s41592-019-0686-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1038%2Fs41592-019-0686-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=32015543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BB3cXislCjuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&pages=261-272&author=P.+Virtanenauthor=R.+Gommersauthor=T.+E.+Oliphantauthor=M.+Haberlandauthor=T.+Reddyauthor=D.+Cournapeauauthor=E.+Burovskiauthor=P.+Petersonauthor=W.+Weckesserauthor=J.+Brightauthor=S.+J.+van+der+Waltauthor=M.+Brettauthor=J.+Wilsonauthor=K.+J.+Millmanauthor=N.+Mayorovauthor=A.+R.+J.+Nelsonauthor=E.+Jonesauthor=R.+Kernauthor=E.+Larsonauthor=C.+J.+Careyauthor=%C4%B0.+Polatauthor=Y.+Fengauthor=E.+W.+Mooreauthor=J.+VanderPlasauthor=D.+Laxaldeauthor=J.+Perktoldauthor=R.+Cimrmanauthor=I.+Henriksenauthor=E.+A.+Quinteroauthor=C.+R.+Harrisauthor=A.+M.+Archibaldauthor=A.+H.+Ribeiroauthor=F.+Pedregosaauthor=P.+van+Mulbregtauthor=A.+Vijaykumarauthor=A.+Bardelliauthor=+Pietroauthor=A.+Rothbergauthor=A.+Hilbollauthor=A.+Kloecknerauthor=A.+Scopatzauthor=A.+Leeauthor=A.+Rokemauthor=C.+N.+Woodsauthor=C.+Fultonauthor=C.+Massonauthor=C.+H%C3%A4ggstr%C3%B6mauthor=C.+Fitzgeraldauthor=D.+A.+Nicholsonauthor=D.+R.+Hagenauthor=D.+V.+Pasechnikauthor=E.+Olivettiauthor=E.+Martinauthor=E.+Wieserauthor=F.+Silvaauthor=F.+Lendersauthor=F.+Wilhelmauthor=G.+Youngauthor=G.+A.+Priceauthor=G.+L.+Ingoldauthor=G.+E.+Allenauthor=G.+R.+Leeauthor=H.+Audrenauthor=I.+Probstauthor=J.+P.+Dietrichauthor=J.+Silterraauthor=J.+T.+Webberauthor=J.+Slaviauthor=J.+Nothmanauthor=J.+Buchnerauthor=J.+Kulickauthor=J.+L.+Sch%C3%B6nbergerauthor=J.+V.+de+Miranda+Cardosoauthor=J.+Reimerauthor=J.+Harringtonauthor=J.+L.+C.+Rodr%C3%ADguezauthor=J.+Nunez-Iglesiasauthor=J.+Kuczynskiauthor=K.+Tritzauthor=M.+Thomaauthor=M.+Newvilleauthor=M.+K%C3%BCmmererauthor=M.+Bolingbrokeauthor=M.+Tartreauthor=M.+Pakauthor=N.+J.+Smithauthor=N.+Nowaczykauthor=N.+Shebanovauthor=O.+Pavlykauthor=P.+A.+Brodtkorbauthor=P.+Leeauthor=R.+T.+McGibbonauthor=R.+Feldbauerauthor=S.+Lewisauthor=S.+Tygierauthor=S.+Sievertauthor=S.+Vignaauthor=S.+Petersonauthor=S.+Moreauthor=T.+Pudlikauthor=T.+Oshimaauthor=T.+J.+Pingelauthor=T.+P.+Robitailleauthor=T.+Spuraauthor=T.+R.+Jonesauthor=T.+Ceraauthor=T.+Leslieauthor=T.+Zitoauthor=T.+Kraussauthor=U.+Upadhyayauthor=Y.+O.+Halchenkoauthor=Y.+V%C3%A1zquez-Baeza&title=SciPy+1.0%3A+Fundamental+Algorithms+for+Scientific+Computing+in+Python&doi=10.1038%2Fs41592-019-0686-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">SciPy 1.0: fundamental algorithms for scientific computing in Python</span></div><div class="casAuthors">Virtanen, Pauli; Gommers, Ralf; Oliphant, Travis E.; Haberland, Matt; Reddy, Tyler; Cournapeau, David; Burovski, Evgeni; Peterson, Pearu; Weckesser, Warren; Bright, Jonathan; van der Walt, Stefan J.; Brett, Matthew; Wilson, Joshua; Millman, K. Jarrod; Mayorov, Nikolay; Nelson, Andrew R. J.; Jones, Eric; Kern, Robert; Larson, Eric; Carey, C. J.; Polat, Ilhan; Feng, Yu; Moore, Eric W.; Vander Plas, Jake; Laxalde, Denis; Perktold, Josef; Cimrman, Robert; Henriksen, Ian; Quintero, E. A.; Harris, Charles R.; Archibald, Anne M.; Ribeiro, Antonio H.; Pedregosa, Fabian; van Mulbregt, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Methods</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">261-272</span>CODEN:
                <span class="NLM_cas:coden">NMAEA3</span>;
        ISSN:<span class="NLM_cas:issn">1548-7091</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: SciPy is an open-source scientific computing library for the Python programming language.  Since its initial release in 2001, SciPy has become a de facto std. for leveraging scientific algorithms in Python, with over 600 unique code contributors, thousands of dependent packages, over 100,000 dependent repositories and millions of downloads per yr.  In this work, we provide an overview of the capabilities and development practices of SciPy 1.0 and highlight some recent tech. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoanlbhUgKQQ7Vg90H21EOLACvtfcHk0ljk4tvQ_fXy6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXislCjuro%253D&md5=f007632188adeb57a43469157898e0a8</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fs41592-019-0686-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41592-019-0686-2%26sid%3Dliteratum%253Aachs%26aulast%3DVirtanen%26aufirst%3DP.%26aulast%3DGommers%26aufirst%3DR.%26aulast%3DOliphant%26aufirst%3DT.%2BE.%26aulast%3DHaberland%26aufirst%3DM.%26aulast%3DReddy%26aufirst%3DT.%26aulast%3DCournapeau%26aufirst%3DD.%26aulast%3DBurovski%26aufirst%3DE.%26aulast%3DPeterson%26aufirst%3DP.%26aulast%3DWeckesser%26aufirst%3DW.%26aulast%3DBright%26aufirst%3DJ.%26aulast%3Dvan%2Bder%2BWalt%26aufirst%3DS.%2BJ.%26aulast%3DBrett%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DJ.%26aulast%3DMillman%26aufirst%3DK.%2BJ.%26aulast%3DMayorov%26aufirst%3DN.%26aulast%3DNelson%26aufirst%3DA.%2BR.%2BJ.%26aulast%3DJones%26aufirst%3DE.%26aulast%3DKern%26aufirst%3DR.%26aulast%3DLarson%26aufirst%3DE.%26aulast%3DCarey%26aufirst%3DC.%2BJ.%26aulast%3DPolat%26aufirst%3D%25C4%25B0.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DMoore%26aufirst%3DE.%2BW.%26aulast%3DVanderPlas%26aufirst%3DJ.%26aulast%3DLaxalde%26aufirst%3DD.%26aulast%3DPerktold%26aufirst%3DJ.%26aulast%3DCimrman%26aufirst%3DR.%26aulast%3DHenriksen%26aufirst%3DI.%26aulast%3DQuintero%26aufirst%3DE.%2BA.%26aulast%3DHarris%26aufirst%3DC.%2BR.%26aulast%3DArchibald%26aufirst%3DA.%2BM.%26aulast%3DRibeiro%26aufirst%3DA.%2BH.%26aulast%3DPedregosa%26aufirst%3DF.%26aulast%3Dvan%2BMulbregt%26aufirst%3DP.%26aulast%3DVijaykumar%26aufirst%3DA.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DPietro%26aulast%3DRothberg%26aufirst%3DA.%26aulast%3DHilboll%26aufirst%3DA.%26aulast%3DKloeckner%26aufirst%3DA.%26aulast%3DScopatz%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DRokem%26aufirst%3DA.%26aulast%3DWoods%26aufirst%3DC.%2BN.%26aulast%3DFulton%26aufirst%3DC.%26aulast%3DMasson%26aufirst%3DC.%26aulast%3DH%25C3%25A4ggstr%25C3%25B6m%26aufirst%3DC.%26aulast%3DFitzgerald%26aufirst%3DC.%26aulast%3DNicholson%26aufirst%3DD.%2BA.%26aulast%3DHagen%26aufirst%3DD.%2BR.%26aulast%3DPasechnik%26aufirst%3DD.%2BV.%26aulast%3DOlivetti%26aufirst%3DE.%26aulast%3DMartin%26aufirst%3DE.%26aulast%3DWieser%26aufirst%3DE.%26aulast%3DSilva%26aufirst%3DF.%26aulast%3DLenders%26aufirst%3DF.%26aulast%3DWilhelm%26aufirst%3DF.%26aulast%3DYoung%26aufirst%3DG.%26aulast%3DPrice%26aufirst%3DG.%2BA.%26aulast%3DIngold%26aufirst%3DG.%2BL.%26aulast%3DAllen%26aufirst%3DG.%2BE.%26aulast%3DLee%26aufirst%3DG.%2BR.%26aulast%3DAudren%26aufirst%3DH.%26aulast%3DProbst%26aufirst%3DI.%26aulast%3DDietrich%26aufirst%3DJ.%2BP.%26aulast%3DSilterra%26aufirst%3DJ.%26aulast%3DWebber%26aufirst%3DJ.%2BT.%26aulast%3DSlavi%26aufirst%3DJ.%26aulast%3DNothman%26aufirst%3DJ.%26aulast%3DBuchner%26aufirst%3DJ.%26aulast%3DKulick%26aufirst%3DJ.%26aulast%3DSch%25C3%25B6nberger%26aufirst%3DJ.%2BL.%26aulast%3Dde%2BMiranda%2BCardoso%26aufirst%3DJ.%2BV.%26aulast%3DReimer%26aufirst%3DJ.%26aulast%3DHarrington%26aufirst%3DJ.%26aulast%3DRodr%25C3%25ADguez%26aufirst%3DJ.%2BL.%2BC.%26aulast%3DNunez-Iglesias%26aufirst%3DJ.%26aulast%3DKuczynski%26aufirst%3DJ.%26aulast%3DTritz%26aufirst%3DK.%26aulast%3DThoma%26aufirst%3DM.%26aulast%3DNewville%26aufirst%3DM.%26aulast%3DK%25C3%25BCmmerer%26aufirst%3DM.%26aulast%3DBolingbroke%26aufirst%3DM.%26aulast%3DTartre%26aufirst%3DM.%26aulast%3DPak%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DN.%2BJ.%26aulast%3DNowaczyk%26aufirst%3DN.%26aulast%3DShebanov%26aufirst%3DN.%26aulast%3DPavlyk%26aufirst%3DO.%26aulast%3DBrodtkorb%26aufirst%3DP.%2BA.%26aulast%3DLee%26aufirst%3DP.%26aulast%3DMcGibbon%26aufirst%3DR.%2BT.%26aulast%3DFeldbauer%26aufirst%3DR.%26aulast%3DLewis%26aufirst%3DS.%26aulast%3DTygier%26aufirst%3DS.%26aulast%3DSievert%26aufirst%3DS.%26aulast%3DVigna%26aufirst%3DS.%26aulast%3DPeterson%26aufirst%3DS.%26aulast%3DMore%26aufirst%3DS.%26aulast%3DPudlik%26aufirst%3DT.%26aulast%3DOshima%26aufirst%3DT.%26aulast%3DPingel%26aufirst%3DT.%2BJ.%26aulast%3DRobitaille%26aufirst%3DT.%2BP.%26aulast%3DSpura%26aufirst%3DT.%26aulast%3DJones%26aufirst%3DT.%2BR.%26aulast%3DCera%26aufirst%3DT.%26aulast%3DLeslie%26aufirst%3DT.%26aulast%3DZito%26aufirst%3DT.%26aulast%3DKrauss%26aufirst%3DT.%26aulast%3DUpadhyay%26aufirst%3DU.%26aulast%3DHalchenko%26aufirst%3DY.%2BO.%26aulast%3DV%25C3%25A1zquez-Baeza%26aufirst%3DY.%26atitle%3DSciPy%25201.0%253A%2520Fundamental%2520Algorithms%2520for%2520Scientific%2520Computing%2520in%2520Python%26jtitle%3DNat.%2520Methods%26date%3D2020%26volume%3D17%26spage%3D261%26epage%3D272%26doi%3D10.1038%2Fs41592-019-0686-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thiele, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleucher, J.</span></span> <span> </span><span class="NLM_article-title">EASY ROESY: Reliable Cross-Peak Integration in Adiabatic Symmetrized ROESY</span>. <i>Chem. – Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">588</span>, <span class="refDoi"> DOI: 10.1002/chem.200802027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2Fchem.200802027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=19065697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGjsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=585-588&author=C.+M.+Thieleauthor=K.+Petzoldauthor=J.+Schleucher&title=EASY+ROESY%3A+Reliable+Cross-Peak+Integration+in+Adiabatic+Symmetrized+ROESY&doi=10.1002%2Fchem.200802027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">EASY ROESY: reliable cross-peak integration in adiabatic symmetrized ROESY</span></div><div class="casAuthors">Thiele, Christina M.; Petzold, Katja; Schleucher, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">585-588</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Ests. of intramol. distances are essential for structure detn.  For medium-sized mols., ROESY NMR spectroscopy is the method of choice for obtaining distances.  However, the integration of ROESY cross-peaks is problematic, owing to the offset dependence of the integrals and/or TOCSY artifacts.  We here present EASY ROESY (Efficient Adiabatic SYmmetried ROESY), which yields reliable intramol. distances without a sample-specific setup.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7ogpufwLVLLVg90H21EOLACvtfcHk0ljk4tvQ_fXy6Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGjsLg%253D&md5=0434726d1d60d71a8fe6907b22a6dd0c</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2Fchem.200802027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200802027%26sid%3Dliteratum%253Aachs%26aulast%3DThiele%26aufirst%3DC.%2BM.%26aulast%3DPetzold%26aufirst%3DK.%26aulast%3DSchleucher%26aufirst%3DJ.%26atitle%3DEASY%2520ROESY%253A%2520Reliable%2520Cross-Peak%2520Integration%2520in%2520Adiabatic%2520Symmetrized%2520ROESY%26jtitle%3DChem.%2520%25E2%2580%2593%2520Eur.%2520J.%26date%3D2009%26volume%3D15%26spage%3D585%26epage%3D588%26doi%3D10.1002%2Fchem.200802027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, P. H. C.</span></span> <span> </span><span class="NLM_article-title">A Perfect Smoother</span>. <i>Anal. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">3631</span>– <span class="NLM_lpage">3636</span>, <span class="refDoi"> DOI: 10.1021/ac034173t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ac034173t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=1%3ACAS%3A528%3ADC%252BD3sXktFGqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2003&pages=3631-3636&author=P.+H.+C.+Eilers&title=A+Perfect+Smoother&doi=10.1021%2Fac034173t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">A Perfect Smoother</span></div><div class="casAuthors">Eilers, Paul H. C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3631-3636</span>CODEN:
                <span class="NLM_cas:coden">ANCHAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2700</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The A. Savitzky and M. Golay (1964) filter has several disadvantages.  A very attractive alternative is a smoother based on penalized least squares, extending ideas presented by E. T. Whittaker (1923).  This smoother is extremely fast, gives continuous control over smoothness, interpolates automatically, and allows fast leave-one-out cross-validation.  It can be programmed in a few lines of Matlab code.  Theory, implementation, and applications are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3M72rA6bntLVg90H21EOLACvtfcHk0ljMb_Koqyjukg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXktFGqsLw%253D&md5=eba4aed2ebbf3e48bd80049d8e0ce165</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1021%2Fac034173t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fac034173t%26sid%3Dliteratum%253Aachs%26aulast%3DEilers%26aufirst%3DP.%2BH.%2BC.%26atitle%3DA%2520Perfect%2520Smoother%26jtitle%3DAnal.%2520Chem.%26date%3D2003%26volume%3D75%26spage%3D3631%26epage%3D3636%26doi%3D10.1021%2Fac034173t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kneller, T. D.</span>; <span class="NLM_string-name">Goddard, D. G.</span></span> <i>SPARKY3</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=T.+D.+Kneller&author=D.+G.+Goddard&title=SPARKY3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DKneller%26aufirst%3DT.%2BD.%26jtitle%3DSPARKY3%26pub%3DUniversity%2520of%2520California%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neuhaus, D.</span></span> <span> </span><span class="NLM_article-title">Nuclear Overhauser Effect</span>.  <i>Encyclopedia of Magnetic Resonance</i>; <span class="NLM_publisher-name">John Wiley & Sons, Ltd.</span>, <span class="NLM_year">2011</span>; pp  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">16</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;key=10.1002%2F9780470034590.emrstm0350.pub2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=1-16&author=D.+Neuhaus&title=Encyclopedia+of+Magnetic+Resonance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1002%2F9780470034590.emrstm0350.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470034590.emrstm0350.pub2%26sid%3Dliteratum%253Aachs%26aulast%3DNeuhaus%26aufirst%3DD.%26atitle%3DNuclear%2520Overhauser%2520Effect%26btitle%3DEncyclopedia%2520of%2520Magnetic%2520Resonance%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Ltd%26date%3D2011%26spage%3D1%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i92"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02036">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_55145"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02036?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02036</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Detailed NMR analysis of Nleu-2R, Nleu-2S, Nleu-5R, and Nleu-5S; analysis of MD simulations; permeability data; and NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf">PDF</a>)</p></li><li><p class="inline">SMILES, molecular formula strings, and all permeability data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_001.csv">jm0c02036_si_001.csv (7.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02036/suppl_file/jm0c02036_si_002.pdf">jm0c02036_si_002.pdf (30.96 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02036&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02036%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-9%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02036" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b93bba423cb8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
