
<html lang="en"     class="pb-page"  data-request-id="f6257c78-98a5-437f-b514-ca97a884e984"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2016.59.issue-2;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;article:article:10.1021/acs.jmedchem.5b01369"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Small Molecule Inhibitors of Ca2+-S100B Reveal Two Protein Conformations" /></meta><meta name="dc.Creator" content="Michael C.  Cavalier" /></meta><meta name="dc.Creator" content="Mohd. Imran  Ansari" /></meta><meta name="dc.Creator" content="Adam D.  Pierce" /></meta><meta name="dc.Creator" content="Paul T.  Wilder" /></meta><meta name="dc.Creator" content="Laura E.  McKnight" /></meta><meta name="dc.Creator" content="E. Prabhu  Raman" /></meta><meta name="dc.Creator" content="David B.  Neau" /></meta><meta name="dc.Creator" content="Padmavani  Bezawada" /></meta><meta name="dc.Creator" content="Milad J.  Alasady" /></meta><meta name="dc.Creator" content="Thomas H.  Charpentier" /></meta><meta name="dc.Creator" content="Kristen M.  Varney" /></meta><meta name="dc.Creator" content="Eric A.  Toth" /></meta><meta name="dc.Creator" content="Alexander D.  MacKerell, Jr." /></meta><meta name="dc.Creator" content="Andrew  Coop" /></meta><meta name="dc.Creator" content="David J.  Weber" /></meta><meta name="dc.Description" content="The drug pentamidine inhibits calcium-dependent complex formation with p53 (CaS100B·p53) in malignant melanoma (MM) and restores p53 tumor suppressor activity in vivo. However, off-target effects a..." /></meta><meta name="Description" content="The drug pentamidine inhibits calcium-dependent complex formation with p53 (CaS100B·p53) in malignant melanoma (MM) and restores p53 tumor suppressor activity in vivo. However, off-target effects a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 13, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01369" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01369" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01369" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01369" /></link>
        
    
    

<title>Small Molecule Inhibitors of Ca2+-S100B Reveal Two Protein Conformations | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01369" /></meta><meta property="og:title" content="Small Molecule Inhibitors of Ca2+-S100B Reveal Two Protein Conformations" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0007.jpeg" /></meta><meta property="og:description" content="The drug pentamidine inhibits calcium-dependent complex formation with p53 (CaS100B·p53) in malignant melanoma (MM) and restores p53 tumor suppressor activity in vivo. However, off-target effects associated with this drug were problematic in MM patients. Structure–activity relationship (SAR) studies were therefore completed here with 23 pentamidine analogues, and X-ray structures of CaS100B·inhibitor complexes revealed that the C-terminus of S100B adopts two different conformations, with location of Phe87 and Phe88 being the distinguishing feature and termed the “FF-gate”. For symmetric pentamidine analogues (CaS100B·5a, CaS100B·6b) a channel between sites 1 and 2 on S100B was occluded by residue Phe88, but for an asymmetric pentamidine analogue (CaS100B·17), this same channel was open. The CaS100B·17 structure illustrates, for the first time, a pentamidine analog capable of binding the “open” form of the “FF-gate” and provides a means to block all three “hot spots” on CaS100B, which will impact next generation CaS100B·p53 inhibitor design." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01369"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01369">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01369&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01369&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01369&amp;href=/doi/10.1021/acs.jmedchem.5b01369" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 2</span><span class="cit-fg-pageRange">, 592-608</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/2" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01153" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01372" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Small Molecule Inhibitors of Ca<sup>2+</sup>-S100B Reveal Two Protein Conformations</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+C.++Cavalier">Michael C. Cavalier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mohd.+Imran++Ansari">Mohd. Imran Ansari</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Adam+D.++Pierce">Adam D. Pierce</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Paul+T.++Wilder">Paul T. Wilder</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Laura+E.++McKnight">Laura E. McKnight</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=E.+Prabhu++Raman">E. Prabhu Raman</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+B.++Neau">David B. Neau</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Padmavani++Bezawada">Padmavani Bezawada</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Milad+J.++Alasady">Milad J. Alasady</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+H.++Charpentier">Thomas H. Charpentier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kristen+M.++Varney">Kristen M. Varney</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Eric+A.++Toth">Eric A. Toth</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alexander+D.++MacKerell%2C++Jr.">Alexander D. MacKerell, Jr.</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrew++Coop">Andrew Coop</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Weber">David J. Weber</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics (CBT), University of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Marlene and Stewart Greenebaum Cancer Center, University of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Computer Aided Drug Design Center, School of Pharmacy, University of Maryland, Baltimore, Maryland 21201, United States</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Institute for Bioscience and Biotechnology Research, 9600 Gudelsky Drive, Rockville, Maryland 20850, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">NE-CAT, Argonne Illinois 60439, United States</span></div><div class="corresp-info"><strong>*</strong>Address: Department of Biochemistry and Molecular Biology, University of Maryland School of Medicine, Biomedical Research Facility Room 439, 108 N. Greene Street, Baltimore, MD 21201. Phone: (410) 706-4354. Fax: (410) 706-0438. E-mail: <a href="/cdn-cgi/l/email-protection#a2c6d5c7c0c7d0e2d1cdcf8cd7cfc3d0dbcec3ccc68cc7c6d7"><span class="__cf_email__" data-cfemail="670310020502152714080a49120a06151e0b06090349020312">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01369&amp;href=/doi/10.1021%2Facs.jmedchem.5b01369" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 2</span><span class="cit-pageRange">, 592–608</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 4, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 September 2015</li><li><span class="item_label"><b>Published</b> online</span>13 January 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 28 January 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01369" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01369</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D592%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMichael%2BC.%2BCavalier%252C%2BMohd.%2BImran%2BAnsari%252C%2BAdam%2BD.%2BPierce%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D2%26contentID%3Dacs.jmedchem.5b01369%26title%3DSmall%2BMolecule%2BInhibitors%2Bof%2BCa2%252B-S100B%2BReveal%2BTwo%2BProtein%2BConformations%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D608%26publicationDate%3DJanuary%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01369"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1481</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01369" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Small Molecule Inhibitors of Ca2+-S100B Reveal Two Protein Conformations&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;C. Cavalier&quot;},{&quot;first_name&quot;:&quot;Mohd.&quot;,&quot;last_name&quot;:&quot;Imran Ansari&quot;},{&quot;first_name&quot;:&quot;Adam&quot;,&quot;last_name&quot;:&quot;D. Pierce&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;T. Wilder&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;E. McKnight&quot;},{&quot;first_name&quot;:&quot;E.&quot;,&quot;last_name&quot;:&quot;Prabhu Raman&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;B. Neau&quot;},{&quot;first_name&quot;:&quot;Padmavani&quot;,&quot;last_name&quot;:&quot;Bezawada&quot;},{&quot;first_name&quot;:&quot;Milad&quot;,&quot;last_name&quot;:&quot;J. Alasady&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;H. Charpentier&quot;},{&quot;first_name&quot;:&quot;Kristen&quot;,&quot;last_name&quot;:&quot;M. Varney&quot;},{&quot;first_name&quot;:&quot;Eric&quot;,&quot;last_name&quot;:&quot;A. Toth&quot;},{&quot;first_name&quot;:&quot;Alexander&quot;,&quot;last_name&quot;:&quot;D. MacKerell&quot;},{&quot;first_name&quot;:&quot;Andrew&quot;,&quot;last_name&quot;:&quot;Coop&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Weber&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;13&quot;,&quot;issue&quot;:&quot;2&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;592-608&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01369&quot;},&quot;abstract&quot;:&quot;The drug pentamidine inhibits calcium-dependent complex formation with p53 (CaS100B·p53) in malignant melanoma (MM) and restores p53 tumor suppressor activity in vivo. However, off-target effects associated with this drug were problematic in MM patients. Structure–activity relationship (SAR) studies were therefore completed here with 23 pentamidine analogues, and X-ray structures of CaS100B·inhibitor complexes revealed that the C-terminus of S100B adopts two different conformations, with location of Phe87 and Phe88 being the distinguishing feature and termed the “FF-gate”. For symmetric pentamidine analogues (CaS100B·5a, CaS100B·6b) a channel between sites 1 and 2 on S100B was occluded by residue Phe88, but for an asymmetric pentamidine analogue (CaS100B·17), this same channel was open. The CaS100B·17 structure illustrates, for the first time, a pentamidine analog capable of binding the “open” form of the “FF-gate” and provides a means to block all three “hot spots” on CaS100B, which will impact next generati&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01369&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01369" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01369&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01369" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01369&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01369" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01369&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01369&amp;href=/doi/10.1021/acs.jmedchem.5b01369" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01369" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01369" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (9 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01369%26sid%3Dliteratum%253Aachs%26pmid%3D26727270%26genre%3Darticle%26aulast%3DCavalier%26date%3D2016%26atitle%3DSmall%2BMolecule%2BInhibitors%2Bof%2BCa2%252B-S100B%2BReveal%2BTwo%2BProtein%2BConformations%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D2%26spage%3D592%26epage%3D608%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291763" title="Noncovalent interactions">Noncovalent interactions</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/2" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/jmcmar.2016.59.issue-2/20160128/jmcmar.2016.59.issue-2.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The drug pentamidine inhibits calcium-dependent complex formation with p53 (<sup>Ca</sup>S100B<i>·</i>p53) in malignant melanoma (MM) and restores p53 tumor suppressor activity in vivo. However, off-target effects associated with this drug were problematic in MM patients. Structure–activity relationship (SAR) studies were therefore completed here with 23 pentamidine analogues, and X-ray structures of <sup>Ca</sup>S100B<i>·</i>inhibitor complexes revealed that the C-terminus of S100B adopts two different conformations, with location of Phe87 and Phe88 being the distinguishing feature and termed the “FF-gate”. For symmetric pentamidine analogues (<sup>Ca</sup>S100B·<b>5a</b>, <sup>Ca</sup>S100B·<b>6b</b>) a channel between sites 1 and 2 on S100B was occluded by residue Phe88, but for an asymmetric pentamidine analogue (<sup>Ca</sup>S100B<i>·</i><b>17</b>), this same channel was open. The <sup>Ca</sup>S100B<i>·</i><b>17</b> structure illustrates, for the first time, a pentamidine analog capable of binding the “open” form of the “FF-gate” and provides a means to block all three “hot spots” on <sup>Ca</sup>S100B, which will impact next generation <sup>Ca</sup>S100B<i>·</i>p53 inhibitor design.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_76620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_76620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Long-time survival (>3 years) remains extremely poor for most malignant melanoma (MM) patients (>70% mortality), and side effects from treatments are often severe.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Prognosis is even more problematic after the onset of metastasis and/or when drug-resistant mechanisms arise.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> While advances in immune checkpoint inhibitors can have remarkable outcomes for some MM patients, these therapies have limited or small effects in others (50%), so trials with immune checkpoint inhibitors are often done in drug combination trials including with chemotherapy regimens, radiation, and/or other targeted approaches.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> One alternative target in MM is a calcium-binding protein, S100B, which is an important biomarker for this and other cancers.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> S100B is elevated in >90% of MM patients, and its protein level correlates directly with poor survival (<1 year) and relapse, and it is especially predictive as a biomarker when used in combination with other diagnostic indicators.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9-11)</a> For example, MM vaccines are only effective in a small number of patients (5–10%) and they nearly all have low levels of S100B.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> While this correlation is not fully understood, mechanistic studies show that lowering S100B levels via siRNA or via small molecule inhibitors restores p53 levels and its tumor suppressor activities including UV-activated apoptosis pathways found in normal melanocytes.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14-18)</a> Studies suggest that S100B blocks p53 oligomerization, promotes p53 phosphorylation in its C-terminus, and contributes to p53 degradation in concert with the E3 ubiquitin ligase, hdm2.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Elevated S100B and overexpression of the receptor for advanced glycation end products (RAGE) confer a metastatic phenotype,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> so learning how S100B affects tumor progression and metastasis is ongoing including the development of novel S100B inhibitors (termed SBiXs) to ablate S100B-dependent effects in cancer.</div><div class="NLM_p">One of the first compounds found to inhibit S100B was the FDA-approved drug pentamidine.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> In these studies, pentamidine, <b>4b (</b>SBi1), binds two pockets in the p53 binding site on Ca<sup>2+</sup>-S100B (termed sites 2 and 3, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), and it was repurposed for use in a phase II clinical trial for treatment of relapsed and/or refractory melanoma in patients with detectable levels of S100B protein (<a href="http://www.clinicaltrials.gov" class="extLink">www.clinicaltrials.gov</a>, identifier NCT00729807). While p53 levels were often restored in these patients after treatment, the toxicity profile for pentamidine was not ideal, likely in part since <b>4b</b> binds to several cellular targets, including DNA.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20-22)</a> For these reasons, structure–activity relationship (SAR) studies are ongoing to improve the affinity and selectivity of pentamidine analogues for binding Ca<sup>2+</sup>-S100B. Thus far, modifying the chemical scaffold of pentamidine first yielded heptamidine, <b>4a</b> (SBi4211), which occupies the same two persistent binding sites as pentamidine (i.e., sites 2 and 3), only it does so with a single molecule bound to the protein versus two pentamidines/monomer.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> However, improvements to this drug scaffold are still needed to increase its affinity and a provide a meaningful therapeutic effect.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Therefore, in the present study, structure–activity relationships (SARs) of novel pentamidine/heptamidine derivatives were explored.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0009.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overall structure of S100B: (A) homodimer of S100B (blue tubes) with Ca<sup>2+</sup> bound (green spheres); (B) surface rendering of S100B with hot spots 1 (magenta), 2 (yellow), and 3 (gray) highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">The SBiX compounds synthesized here bind sites 2 and 3, as found for pentamidine and heptamidine; however, few such analogues interact with site 1 (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) or showed potential for efficient extension into site 1. This is due to conformational exchange in this region involving Phe87 and Phe88, which serve as a gate to access a narrow channel linking sites 1 and 2 (termed the “FF-gate”). In one case, structural results with compound <b>17</b> (SBi29), indicate that the site 2/3 interactions in the S100B<i>·</i><b>17</b> complex are made in a unique way, as compared to the other pentamidine analogues. Importantly, the FF-gate is “open” in the S100B<i>·</i><b>17</b> complex such that it can accommodate modifications that would possibly extend into site 1 from site 2 efficiently while retaining an amidine moiety like those in the pentamidine diamidine scaffold. Importantly, the gained understanding of the FF-gate now permits a more rational design of novel S100B inhibitor(s) that can simultaneously occupy sites 1 and 2 and perhaps all three target-binding pockets in the <sup>Ca</sup>S100B<i>·</i>p53 complex (i.e., sites 1, 2, and 3). Although chemical scaffolds occupying site 1 have been successfully identified,<a onclick="showRef(event, 'ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref24 ref25">(24, 25)</a> combining moieties or tethering with pentamidine derivatives toward sites 2 and 3 has not yet been achieved. Therefore, the results described here represent a significant advance toward developing such SBiX molecules that achieves a threshold required for a useful therapeutic window.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Ligand Design Strategy</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72232" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72232" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The development of small molecule inhibitors that block the <sup>Ca</sup>S100B<i>·</i>p53 interaction requires inhibition of large protein–protein interfaces (PPIs) that were once thought to be difficult, if not impossible, to block. It is now recognized that the interfaces of most PPIs have smaller amino acid domains, termed “hot spots” or “persistent sites”, that contribute more energetically to the PPIs than the surrounding protein surface.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Successful strategies to target small molecule PPI inhibitors were developed, and as a result, several PPIs have been successfully inhibited, including those involving bcl-6, bcl-2, p53, ERK, tubulin, and several SH2 domains, to name but a few.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> For PPIs involving S100B, two inhibitors, pentamidine (<b>4b</b>) and heptamidine (<b>4a</b>), were discovered to block the <sup>Ca</sup>S100B<i>·</i>p53 complex and occupy persistent binding sites 2 and 3 simultaneously within Ca<sup>2+</sup>-S100B<a onclick="showRef(event, 'ref23 ref29'); return false;" href="javascript:void(0);" class="ref ref23 ref29">(23, 29)</a> (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). Efforts were made here to find the most efficient means to extend the heptamidine scaffold toward site 1 and create compounds capable of binding all three persistent hot spots in S100B simultaneously (i.e., sites 1, 2, and 3).</div><div class="NLM_p">To facilitate this design strategy, the site identification by ligand competitive saturation (SILCS) approach was applied to map the functional group requirements of the three-targeted sites. From the SILCS calculations, 3D FragMaps are obtained that indicate the different functional group requirements of the protein, information that can be used to identify moieties of a compound that are contributing to binding as well as to be used to indicate how modifications of a compound can be used to improve affinity. During the design stage, what is termed a “tier 1” SILCS approach was used. This approach includes aromatic maps based on benzene, aliphatic maps based on propane, and hydrogen bond donors and acceptors maps based on water hydrogen atoms and oxygen atoms, respectively. Shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a> are the SILCS FragMaps overlaid on the crystal structure of the <sup>Ca</sup>S100B·SBi132 complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GK1">3GK1</a>) along with the crystallographic orientation of heptamidine (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FQO">4FQO</a>). The approximate locations of the three hot spots are labeled. The occupancy of sites 2 and 3 by heptamidine is evident, with aromatic and aliphatic FragMaps in the vicinity of the aromatic rings and aliphatic linker of heptamidine indicating the contribution of those moieties to ligand binding. The aliphatic and aromatic FragMaps cover a relatively wide range indicating that variations in the linker length between the aromatic rings could be tolerated. Notably, only small hydrogen bond donor FragMaps are directly adjacent to either amidine moiety, indicating that the positive charges may not be essential for binding. In addition, in regions beyond the amidine moieties in sites 1 and 3 are hydrogen bond donor FragMaps that are adjacent to acidic residues (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B), suggesting that extending those moieties with additional positively charged groups with linkers of varying lengths may lead to improved binding. Therefore, functionalities were added to the amidine moieties of both heptamidine (<b>4a)</b> and pentamidine (<b>4b</b>)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> in an attempt to improve affinity. In addition, the extended aromatic/aliphatic FragMaps around the linker region in the P2 site, supported by other <sup>Ca</sup>S100B<i>·</i>SBiX X-ray structures<a onclick="showRef(event, 'ref23 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref23 ref29 ref30">(23, 29, 30)</a> indicated that increased diversity of the linker could facilitate binding.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0010.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. SILCS FragMaps, showing crystal orientation of heptamidine (licorice, atom-type coloring) overlaid on the SILCS FragMaps calculated using the SBi132·S100B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GK1">3GK1</a>) crystal structure. (A) FragMaps are shown for aliphatic (green), aromatic (purple), and hydrogen bond donors (blue) with the protein show in surface representation. The approximate locations of the site 1, 2, and 3 pockets are indicated. (B) Protein in cartoon representation with acidic residues in red licorice along with the hydrogen bond donors (blue). Arrow heads indicate hydrogen bond acceptor FragMaps that are adjacent to acidic residue side chains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">On the basis of these design criteria and synthetic considerations, a number of novel analogs were developed. Compounds of series <b>5</b>, <b>6</b>, and <b>7</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) include modification of the amidine moiety, yielding neutral species in aliphatic rings ranging in size from five to seven non-hydrogen atoms that may have favorable interactions as evidenced by the aliphatic FragMaps in the vicinity of the amidine moieties and the adjacent aromatic rings. Alternatively, the amidine moieties were substituted with phenylamine and amide in <b>10</b> and <b>11</b>. In addition, in the <b>5</b>, <b>6</b>, and <b>7</b> series the linker length between the aromatic rings was varied from <i>n</i> = 3, as in pentamidine, to <i>n</i> = 6, which is one methylene group longer than heptamidine. To take advantage of the donor FragMaps in the vicinity of acid groups (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B), compounds <b>8</b> and <b>9</b> were designed. These include a primary amino group terminating aliphatic linkers of five and seven methylene groups in <b>8</b> and <b>9</b>, respectively, with different linker lengths between the aromatic rings such that <b>8</b> represents a shorter compound as compared to the two <b>9</b> analogs. To address the presence of the aliphatic and aromatic FragMaps in the vicinity of the linker in heptamidine (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A), compounds <b>16a</b> and <b>b</b> were designed in which a linker of the same length as that in heptamidine is present that contains a tertiary amine to which a benzyl or propyl moiety is attached. An additional compound, <b>17</b>, was identified later in the project based on the developed SAR model, as described below.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemical Synthesis</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Symmetric molecules flanked by diamidine moieties were synthesized using short, convergent syntheses strategies as shown in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch4" aria-label="4">4</a>. For example, pentamidine derivatives <b>4a</b>–<b>c</b> were synthesized by the Pinner reaction as described (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) starting from 4-cyanophenol.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0003.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Diamidines <b>4a</b>–<b>c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CsCO<sub>3</sub>, acetone, reflux, 14 h; (b) methanol/dioxane (1:1), HCl gas; (c) ethanolic ammonia, ammonium chloride, 80 °C, 32 h.</p></p></figure><div class="NLM_p">Cyclic amidines (<b>5a</b>–<b>d</b>, <b>6a</b>–<b>d</b>, and <b>7a</b>–<b>d</b>) were synthesized according to literature with some modification (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Specifically, treatment of dibenzimidate with diaminoethane, diaminopropane, and diaminobutane at room temperature in DMF accomplished cyclic amidine in satisfactory yield.<a onclick="showRef(event, 'ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref31 ref32 ref33">(31-33)</a> Increasing the chain length of the diamine results in open chain products (<b>8</b> and <b>9a</b>, <b>9b</b>), which were useful for testing aliphatic chain derivatives of varying lengths (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Cyclic and Fatty Amine Substituted Diamidine Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1,2-diaminoethane, DMF, rt; (b) 1,3-diaminopropane, DMF, rt; (c) 1,4-diaminobutane, DMF, rt; (d) 1,2-diaminopentane, DMF, rt; (e) 1,2-diaminoheptane, DMF, rt.</p></p></figure><div class="NLM_p">For further synthesis of N-derived diamidine compound <b>10</b>, compound <b>4a</b> was coupled with phenyl isocyanate to furnish final diamidine compound in good yield. Compound <b>11</b> was obtained through the carbomylation of compound <b>4a</b> with <i>N</i>-succinamidyl-<i>N</i>-methyl carbamate at 40 °C for 2 h (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of <i>N</i>-Acyl Substituted Diamidine Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) phenyl isocyanate, DBU, DMF, 40 °C; (b) <i>N</i>-succinamidyl-<i>N</i>-methyl carbamate, DBU, DMF, 40 °C.</p></p></figure><div class="NLM_p">Compounds <b>16a</b>,<b>b</b> were synthesized to further diversify the structure–activity relationship series, N-ligated. For these compounds, 1,3-dibromopropane was reacted with benzylamine and propylamine to obtain compounds <b>13a</b>,<b>b</b>, which then reacted with 4-hydroxybenzonitrile to furnish the intermediates <b>14a</b>,<b>b</b>. After that, the well-known Pinner reaction was used to prepare the target compounds <b>16a</b>,<b>b</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a></div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0006.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Preparation of Analogs with Central Amine Substitutions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) for <b>13a</b>, benzylamine, <b>13b</b>, propylamine, K<sub>2</sub>CO<sub>3</sub>, acetone; (b) <b>1</b>, CsCO<sub>3</sub>, acetone; (c) methanol, HCl gas; (d) ethanolic ammonia, ammonium chloride.</p></p></figure><div class="NLM_p last">One additional commercial compound, <b>17</b>, was included in the study. Commercially available compound <b>17</b> maintains many of the advantageous functionalities of the pentamidine scaffold but is a more rigid scaffold because it has fewer rotatable bonds.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18764" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18764" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Characterization of Cyclic Diamidine Analogs Bound to Ca<sup>2+</sup>-S100B (<sup>Ca</sup>S100B)</h3><div class="NLM_p">Twelve compounds prepared in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> retained the linker scaffold as found for compounds in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>; however, the diamidine moiety for these 12 molecules was modified to form a five-membered ring (compounds <b>5a</b>–<b>d</b>), a six-membered ring (compounds <b>6a</b>–<b>d</b>), or a seven-membered ring (compounds <b>7a</b>–<b>d</b>; <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). These compounds (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) were designed to retain hydrogen bond donor capacity and to include aliphatic hydrocarbon regions on the compounds that could potentially provide hydrophobic interactions with residues Leu44, Ile80, and Ala83 of Ca<sup>2+</sup>-S100B at or nearby the lower portion of site 1 as indicated by the aliphatic FragMaps adjacent to the amidine moiety (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><div class="NLM_p">By use of an established fluorescent polarization competition assay (FPCA),<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> compounds <b>5a</b>–<b>d</b>, <b>6a</b>–<b>d</b>, and <b>7a</b>–<b>d</b> were tested to determine if they were able to compete with a probe, TAMRA-TRTK, which binds site 1 in Ca<sup>2+</sup>-S100B (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Of the 12 compounds in this series of cyclic diamidine analogues, they all interacted with Ca<sup>2+</sup>-S100B as measured using NMR; however, none were able to compete for binding with the TAMRA-TRTK probe at low concentrations in the FPCA (i.e., <1 mM). In cellular assays (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), EC<sub>50</sub> values for these compounds were all in the micromolar range (EC<sub>50</sub>, <b>5a</b>–<b>d</b>, ∼20–60 μM; <b>6a</b>,<b>b</b>, ∼200–600 μM; <b>6c</b>–<b>d</b>, ∼25–45 μM; <b>7a</b>,<b>b</b>,<b>d</b>, ∼5–20 μM; <b>7c</b>, ∼250–300 μM; <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>), but the EC<sub>50</sub> values for the compounds in all four of these series were all (except <b>7a</b>,<b>d</b>) higher than those observed for comparable studies with pentamidine (EC<sub>50</sub>, <b>4b</b>, 17 ± 2 μM; <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) and none showed improvement over that observed for heptamidine, which also demonstrated an S100B specific affect (<sup>High S100B</sup>EC<sub>50</sub>, <b>4b</b>, 11 ± 2 μM; <sup>Low S100B</sup>EC<sub>50</sub>, <b>4b</b>, 26 ± 4 μM; <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). While the NMR spectra of Ca<sup>2+</sup>-S100B upon binding to <b>5a</b>, <b>6a</b>, <b>6c</b>, and <b>6d</b> showed similar chemical shift perturbations as pentamidine and heptamidine (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>), the binding of the other compounds in this series (<b>5b</b>, <b>5c</b>, <b>5d</b>, <b>6b</b>, <b>7a</b>–<b>d</b>) caused significant broadening to the NMR spectra due to an intermediate exchange binding regime and/or because of protein aggregation. The NMR and FPCA results provided indication that the cyclic diamidine moieties in <b>5a</b>–<b>d</b>, <b>6a</b>–<b>d</b>, or <b>7a</b>–<b>d</b> did not interact with site 1 of Ca<sup>2+</sup>-S100B. Nonetheless, X-ray crystallography experiments were initiated and structure determinations were completed for Ca<sup>2+</sup>-S100B complexes with compounds <b>5a</b> (SBi4225) and <b>6b</b> (SBi4214) to further explore this possibility.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Tabulation of Binding Assay Results<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">FPCA IC<sub>50</sub> (μM)</th><th class="rowsep1 colsep0" colspan="3" align="center">calcd <i>K</i><sub>D</sub> (μM)<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">internal ID SBiX</th><th class="colsep0 rowsep0" align="center">HSQC<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" colspan="3" align="center">mean ± SD</th><th class="colsep0 rowsep0" colspan="3" align="center">mean ± SD</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="left">4225</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">2222.1 </td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">213.5</td><td class="colsep0 rowsep0" align="left">202.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">19.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">4236</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">51.1</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">2.3</td><td class="colsep0 rowsep0" align="left">4.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left">4232</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">6.7</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9b</b></td><td class="colsep0 rowsep0" align="left">4230</td><td class="colsep0 rowsep0" align="left">A</td><td class="colsep0 rowsep0" align="left">18.3</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">4212</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">981.7</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">31.1</td><td class="colsep0 rowsep0" align="left">89.5</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">2.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">66.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">2.0</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last"><b>4a</b><i>K</i><sub>D</sub> (1D NMR<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a>): 6.9 ± 0.9 μM. <sup>Cells</sup>EC<sub>50</sub> = 26 ± 4 μM. <b>4b</b><i>K</i><sub>D</sub> (ITC<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a>): 50 ± 5 μM. Fluorescence:<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> 44 ± 4 μM. 1D NMR:<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> 0.13 ± 0.01 μM. <sup>Cells</sup>EC<sub>50</sub> = 26 ± 4 μM; <b>4c</b>.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">A = signal lost likely due to formation of soluble aggregates. + = binding.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">The <i>K</i><sub>D</sub> was calculated from the FPCA IC<sub>50</sub> using the Nikolovska–Coleska equation.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Tabulation of Cellular Assay Results</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">high S100B, EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="rowsep1 colsep0" colspan="3" align="center">low S100B, EC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">prep compd</th><th class="colsep0 rowsep0" align="center">internal ID SBiX</th><th class="colsep0 rowsep0" colspan="3" align="center">mean ± SD</th><th class="colsep0 rowsep0" colspan="3" align="center">mean ± SD</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4a</b></td><td class="colsep0 rowsep0" align="left">4211</td><td class="colsep0 rowsep0" align="left">11.5</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">2.4</td><td class="colsep0 rowsep0" align="left">25.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">4.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4b</b></td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">16.3</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">1.7</td><td class="colsep0 rowsep0" align="left">16.7</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">2.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4c</b></td><td class="colsep0 rowsep0" align="left">4210</td><td class="colsep0 rowsep0" align="left">245.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">15.5</td><td class="colsep0 rowsep0" align="left">234.0</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">7.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="left">4225</td><td class="colsep0 rowsep0" align="left">32.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">1.5</td><td class="colsep0 rowsep0" align="left">21.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">1.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">4224</td><td class="colsep0 rowsep0" align="left">56.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">7.0</td><td class="colsep0 rowsep0" align="left">38.9</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">6.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5c</b></td><td class="colsep0 rowsep0" align="left">4226</td><td class="colsep0 rowsep0" align="left">52.0</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">4.5</td><td class="colsep0 rowsep0" align="left">34.5</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">4.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5d</b></td><td class="colsep0 rowsep0" align="left">4221</td><td class="colsep0 rowsep0" align="left">21.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">4.4</td><td class="colsep0 rowsep0" align="left">24.3</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">5.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">4213</td><td class="colsep0 rowsep0" align="left">612.5</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">232.9</td><td class="colsep0 rowsep0" align="left">288.9</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">106.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="left">4214</td><td class="colsep0 rowsep0" align="left">203.1</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">147.7</td><td class="colsep0 rowsep0" align="left">283.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">221.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">4218</td><td class="colsep0 rowsep0" align="left">44.1</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">35.0</td><td class="colsep0 rowsep0" align="left">42.7</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">43.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="left">4217</td><td class="colsep0 rowsep0" align="left">24.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">8.9</td><td class="colsep0 rowsep0" align="left">37.9</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">32.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7a</b></td><td class="colsep0 rowsep0" align="left">4227</td><td class="colsep0 rowsep0" align="left">7.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">1.6</td><td class="colsep0 rowsep0" align="left">9.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">2.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7b</b></td><td class="colsep0 rowsep0" align="left">4223</td><td class="colsep0 rowsep0" align="left">20.0</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">3.0</td><td class="colsep0 rowsep0" align="left">19.0</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7c</b></td><td class="colsep0 rowsep0" align="left">4228</td><td class="colsep0 rowsep0" align="left">264.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">107.3</td><td class="colsep0 rowsep0" align="left">301.2</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">139.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7d</b></td><td class="colsep0 rowsep0" align="left">4222</td><td class="colsep0 rowsep0" align="left">5.4</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.9</td><td class="colsep0 rowsep0" align="left">4.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">4236</td><td class="colsep0 rowsep0" align="left">37.1</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">3.6</td><td class="colsep0 rowsep0" align="left">33.3</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9a</b></td><td class="colsep0 rowsep0" align="left">4232</td><td class="colsep0 rowsep0" align="left">24.3</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">6.3</td><td class="colsep0 rowsep0" align="left">25.4</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">6.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9b</b></td><td class="colsep0 rowsep0" align="left">4230</td><td class="colsep0 rowsep0" align="left">46.3</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">5.6</td><td class="colsep0 rowsep0" align="left">45.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">6.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10</b></td><td class="colsep0 rowsep0" align="left">4235</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">0.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">4212</td><td class="colsep0 rowsep0" align="left">130.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">18.1</td><td class="colsep0 rowsep0" align="left">146.1</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">17.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16a</b></td><td class="colsep0 rowsep0" align="left">4239</td><td class="colsep0 rowsep0" align="left">51.8</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">6.0</td><td class="colsep0 rowsep0" align="left">41.0</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">7.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16b</b></td><td class="colsep0 rowsep0" align="left">4238</td><td class="colsep0 rowsep0" align="left">863.6</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">103.2</td><td class="colsep0 rowsep0" align="left">500.1</td><td class="colsep0 rowsep0" align="center">±</td><td class="colsep0 rowsep0" align="left">128.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">29</td><td class="colsep0 rowsep0" align="left"><1.0</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><1.0</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">The SYBR Green cell proliferation assay was done using WM115 melanoma cells transfected with shRNA<sup>scrambled</sup> (i.e., high S100B) or shRNA<sup>S100B</sup> (low S100B).</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Chemical Shift Perturbations for Residues in <sup>Ca</sup>S100B upon the Addition of Inhibitors<a class="ref internalNav" href="#t3fn1" aria-label="α">α</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">helix 1</th><th class="colsep0 rowsep0" align="center">loop 2 (hinge)</th><th class="colsep0 rowsep0" align="center">helix 3</th><th class="colsep0 rowsep0" align="center">helix 4</th><th class="colsep0 rowsep0" align="center">C-terminal loop</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4b</b><a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></td><td class="colsep0 rowsep0" align="left">A9 ILL F14 H15</td><td class="colsep0 rowsep0" align="left">S41 L44 E45 E46</td><td class="colsep0 rowsep0" align="left">V52 T59</td><td class="colsep0 rowsep0" align="left">M74 V77 A83</td><td class="colsep0 rowsep0" align="left">E89 H90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>11</b></td><td class="colsep0 rowsep0" align="left">K5 A9</td><td class="colsep0 rowsep0" align="left">L40 K48</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">A75 V77 S78 T81 T82 A83 C84 H85</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">S1</td><td class="colsep0 rowsep0" align="left">H42 E45</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">S78 T81 T82 C84 H85</td><td class="colsep0 rowsep0" align="left">E86 F88 E89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="left">S1</td><td class="colsep0 rowsep0" align="left">H42</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">S78 T81 T82 C84 H85</td><td class="colsep0 rowsep0" align="left">E86 F88 E89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16b</b></td><td class="colsep0 rowsep0" align="left">S1 ILL F14</td><td class="colsep0 rowsep0" align="left">H42 E45</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">S78 T81 T82</td><td class="colsep0 rowsep0" align="left">E89 H90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">TRTK<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">L40 H42 L44 E45 I47</td><td class="colsep0 rowsep0" align="left">V52 K55 V56 T59</td><td class="colsep0 rowsep0" align="left">M79 V80 A83 C84</td><td class="colsep0 rowsep0" align="left">F87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">p53<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">L44</td><td class="colsep0 rowsep0" align="left">V52 V56</td><td class="colsep0 rowsep0" align="left">M79 V80</td><td class="colsep0 rowsep0" align="left">F87</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>α</sup><p class="last">Mapping of the binding site on <sup>Ca</sup>S100B using chemical shift perturbations (Δ<sup>1</sup>H + Δ<sup>15</sup>N) for residues upon addition of inhibitors.</p></div></div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> Global Conformation</h4><div class="NLM_p">X-ray structures of Ca<sup>2+</sup>-S100B bound to either <b>5a</b> or <b>6b</b> consist of four helixes (helix 1, E2-G19; helix 2, K28-L40; helix 3, E49-D61; helix 4, D69-F86) and are assembled as an active symmetric homodimer (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>a), as found previously in other structures of S100B. Both the typical and noncanonical EF-hand calcium binding domains in each subunit were found to coordinate a Ca<sup>2+</sup> ion in the two structures.<a onclick="showRef(event, 'ref29 ref30 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref37 ref38">(29, 30, 37, 38)</a> A comparison with the structure of Ca<sup>2+</sup>-S100B in the absence of compound (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MHO">1MHO</a><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>) shows a strong conservation of the global fold (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Table S1</a>) with the only significant deviation observed at C-terminal residues His85-Glu91 involving, most notably, residues Phe87 and Phe88 (<a class="ref internalNav" href="#fig3" aria-label="Figures 3">Figures 3</a>D and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>D). This segment of the polypeptide consistently conforms to accommodate the binding of the SBiX molecules, as similarly found for the binding of pentamidine and heptamidine.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0011.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystallographic structure of the S100B·<b>5a</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN">5DKN</a>). (A) |<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>| electron density omit map (blue mesh) of <b>5a</b> (ball and stick) contoured at 2.5σ levels. (B) <b>5a</b> within sites 2 (yellow highlight) and 3 (gray highlight) of the S100B homodimer (surface in blue). (C) Modeled orientation of <b>5a</b> overlaid on the previously elucidated compound-bound structures of pentamidine and heptamidine. (D) The specific interactions of <b>5a</b> within dimeric S100B (blue ribbons) are rendered with residues (sticks) within 4 Å of the <b>5a</b> highlighted in yellow (site 2) or gray (site 3). <b>5a</b> is situated within a hydrophobic pocket with atoms within hydrogen bond distance at the backbone carbonyl of His85 and the side chain of Cys84.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0012.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystallographic structure of the S100B·<b>6b</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>). (A) |<i>F</i><sub>o</sub>|− |<i>F</i><sub>c</sub>| electron density omit map (blue mesh) of <b>6b</b> (ball and stick) contoured at 2.5σ levels. (B) <b>6b</b> within sites 2 (yellow highlight) and 3 (gray highlight) of the S100B homodimer (surface in blue). (C) Modeled orientation of <b>6b</b> overlaid on the previously elucidated compound-bound structures of pentamidine and heptamidine. (D) The specific interactions of <b>6b</b> with dimeric S100B (blue ribbons) are rendered with residues (sticks) within 4 Å of the <b>6b</b> highlighted in yellow (site 2) or gray (site 3). <b>6b</b> is situated within a hydrophobic pocket with atoms within hydrogen bond distance at the backbone carbonyl of His85 and the side chain of Cys84.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> <sup>Ca</sup>S100B·<b>5a</b></h4><div class="NLM_p">The X-ray structure of <b>5a</b> bound, calcium loaded S100B (<sup>Ca</sup>S100B·<b>5a</b>) was solved at 1.53 Å resolution (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). The asymmetric unit contained one S100B monomer with the second monomer forming the biologically active dimeric form of S100B contained within the crystal symmetry. Residues Met0 to His90 were modeled. The monomer in the <sup>Ca</sup>S100B·<b>5a</b> structure bound two calcium ions and a <b>5a</b> molecule. In the final refined model (<i>R</i><sub>crys</sub>/<i>R</i><sub>free</sub> of 0.195/0.216), all residues were in the favored (100%) region of the Ramachandran plot with no outliers (see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Statistics of Reflection Data and Structure Refinements<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">ligand</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>5a</b></th><th class="colsep0 rowsep0" align="center"><b>6b</b></th><th class="colsep0 rowsep0" align="center"><b>17</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>41212</td><td class="colsep0 rowsep0" align="left"><i>P</i>41212</td><td class="colsep0 rowsep0" align="left"><i>P</i>212121</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell dimension (Å)</td><td class="colsep0 rowsep0" align="left">64.0, 64.0, 47.6</td><td class="colsep0 rowsep0" align="left">63.5, 63.5, 48.3</td><td class="colsep0 rowsep0" align="left">45.7, 47.4, 90.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell angle (deg)</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td><td class="colsep0 rowsep0" align="left">90.0, 90.0, 90.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">47.56–1.53</td><td class="colsep0 rowsep0" align="left">38.43–1.59</td><td class="colsep0 rowsep0" align="left">33.30–1.74</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections observed</td><td class="colsep0 rowsep0" align="left">74 444 (10 810)</td><td class="colsep0 rowsep0" align="left">97 324 (4162)</td><td class="colsep0 rowsep0" align="left">82 661 (3111)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">15 494 (2197)</td><td class="colsep0 rowsep0" align="left">13 794 (663)</td><td class="colsep0 rowsep0" align="left">20 971 (951)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections <i>R</i><sub>free</sub> set</td><td class="colsep0 rowsep0" align="left">1546</td><td class="colsep0 rowsep0" align="left">1376</td><td class="colsep0 rowsep0" align="left">1994</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.8 (99.6)</td><td class="colsep0 rowsep0" align="left">99.9 (99.3)</td><td class="colsep0 rowsep0" align="left">98.7 (83.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">4.8 (4.9)</td><td class="colsep0 rowsep0" align="left">7.1 (6.3)</td><td class="colsep0 rowsep0" align="left">3.9 (3.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">⟨<i>I</i>/σ⟩</td><td class="colsep0 rowsep0" align="left">26.1 (3.8)</td><td class="colsep0 rowsep0" align="left">23.3 (1.5)</td><td class="colsep0 rowsep0" align="left">11.8 (1.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC<sub>1/2</sub> (%)</td><td class="colsep0 rowsep0" align="left">99.9 (90.5)</td><td class="colsep0 rowsep0" align="left">99.9 (51.7)</td><td class="colsep0 rowsep0" align="left">99.7 (68.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>sym</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.048 (0.347)</td><td class="colsep0 rowsep0" align="left">0.046 (1.28)</td><td class="colsep0 rowsep0" align="left">0.055 (0.693)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>crys</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.195</td><td class="colsep0 rowsep0" align="left">0.207</td><td class="colsep0 rowsep0" align="left">0.209</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub><a class="ref internalNav" href="#t4fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">0.216</td><td class="colsep0 rowsep0" align="left">0.230</td><td class="colsep0 rowsep0" align="left">0.247</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of amino acids</td><td class="colsep0 rowsep0" align="left">91</td><td class="colsep0 rowsep0" align="left">91</td><td class="colsep0 rowsep0" align="left">179</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of protein atoms</td><td class="colsep0 rowsep0" align="left">747</td><td class="colsep0 rowsep0" align="left">743</td><td class="colsep0 rowsep0" align="left">1,457</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of heteroatoms</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">33</td><td class="colsep0 rowsep0" align="left">33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. of waters</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">108</td><td class="colsep0 rowsep0" align="left">172</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a> bond length (Å)</td><td class="colsep0 rowsep0" align="left">0.012</td><td class="colsep0 rowsep0" align="left">0.006</td><td class="colsep0 rowsep0" align="left">0.013</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">angle (deg)</td><td class="colsep0 rowsep0" align="left">0.937</td><td class="colsep0 rowsep0" align="left">0.971</td><td class="colsep0 rowsep0" align="left">1.217</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">mean <i>B</i> factor</td><td class="colsep0 rowsep0" align="left">18.59</td><td class="colsep0 rowsep0" align="left">31.69</td><td class="colsep0 rowsep0" align="left">30.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein atom (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">17.18</td><td class="colsep0 rowsep0" align="left">30.32</td><td class="colsep0 rowsep0" align="left">29.13</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">heteroatom (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">21.54</td><td class="colsep0 rowsep0" align="left">38.29</td><td class="colsep0 rowsep0" align="left">31.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water atom (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">25.92</td><td class="colsep0 rowsep0" align="left">39.05</td><td class="colsep0 rowsep0" align="left">39.03</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran outlier (%)</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td><td class="colsep0 rowsep0" align="left">0.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran favored (%)</td><td class="colsep0 rowsep0" align="left">100.0</td><td class="colsep0 rowsep0" align="left">100.0</td><td class="colsep0 rowsep0" align="left">100.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">The numbers in parentheses represent values from the highest resolution shell: <sup>Ca</sup>S100B·<b>5a</b> (1.61–1.53 Å), <sup>Ca</sup>S100B·<b>6b</b> (1.62–1.59 Å), and <sup>Ca</sup>S100B·<b>17</b> (1.77–1.59 Å).</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last"><i>R</i><sub>sym</sub> = ∑<sub><i>h</i></sub>(∑ <sub><i>j</i></sub>|<i>I</i><sub><i>hj</i></sub> – ⟨<i>I</i><sub><i>h</i></sub>⟩|/∑<i>I</i><sub><i>h</i>,<i>j</i></sub>), where <i>h</i> is set of Miller indices, <i>j</i> is set of observations of reflection <i>h</i>, and ⟨<i>I</i><sub><i>h</i></sub>⟩ is the mean intensity. <i>R</i><sub>crys</sub> = ∑<sub><i>h</i></sub>||<i>F</i><sub>o,<i>h</i></sub>| – |<i>F</i><sub>c,<i>h</i></sub>||/∑<sub><i>h</i></sub>|<i>F</i><sub>o,<i>h</i></sub>|. <i>R</i><sub>free</sub> was calculated using a percentage of the complete data set excluded from refinement.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">The rmsd values are deviation from ideal values.</p></div></div></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <sup>Ca</sup>S100B·<b>6b</b></h4><div class="NLM_p last">The X-ray structure of <b>6b</b> bound, calcium loaded S100B (<sup>Ca</sup>S100B·<b>6b</b>) was solved at 1.59 Å resolution (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a). The asymmetric unit contained one S100B monomer with the second monomer forming the biologically active dimeric form of S100B contained within the crystal symmetry. Residues Met0 to His90 were modeled. The monomer in the <sup>Ca</sup>S100B·<b>6b</b> structure bound two calcium ions and a <b>6b</b> molecule. In the final refined model (<i>R</i><sub>crys</sub>/<i>R</i><sub>free</sub> of 0.207/0.230), all residues were in the favored (100%) region of the Ramachandran plot with no outliers (see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> Structural Comparisons of <sup>Ca</sup>S100B·<b>5a</b>, <sup>Ca</sup>S100B·<b>6b</b>, <sup>Ca</sup>S100B·Pentamidine, and <sup>Ca</sup>S100B·Heptamidine</h4><div class="NLM_p last">As both <b>6b</b> and <b>5a</b> are derivative compounds based on the previously published SBiX molecules pentamidine (<b>4b</b>) and heptamidine (<b>4a</b>), they share similar binding sites and interactions as their predecessors<a onclick="showRef(event, 'ref23 ref29'); return false;" href="javascript:void(0);" class="ref ref23 ref29">(23, 29)</a> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B,C and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B,C). These derivatives bind within close proximity of several residues of S100B in a pocket formed by helix 4, loop 2, and helix 1′ (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D and <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>D). The diazacyclohexene and the imidazoline groups of <b>6b</b> and <b>5a</b>, respectively, are able to participate in hydrogen bonds with the backbone carbonyl His85 (helix 4) while participating in stacking interactions with Phe88 and Phe87 (helix 4). Such interactions were previously described in pentamidine and heptamidine structures. Similar to both pentamidine and heptamidine, <b>5a</b> and <b>6b</b> occupy both binding sites 2 and 3. However, like heptamidine and unlike pentamidine, the derivatives occupy both sites with a single molecule. When compared to the preceding compounds, the larger substituent groups of <b>5a</b> and <b>6b</b> within site 2 do not extend into site 1, consistent with the FPCA results above. Rather, in both cases, the ring starts to protrude into solvent with few if any interactions with the protein. On the other hand, within site 3 the larger ring displaces the benzyl moiety 0.9–1.2 Å from the center in both cases (i.e., for <b>5a</b> and <b>6b</b>) while the second benzyl moieties occupy the same hydrophobic site formed by the side chains of Leu40, Phe43, and Phe87. However, neither derivative compound makes the hydrogen bond with the backbone carbonyl of Phe43 as observed with heptamidine.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Characterization of Noncyclic Aliphatic Amine Substituted Diamidine Analogs</h3><div class="NLM_p">Three additional compounds (<b>8</b>, <b>9a</b>, <b>9b</b>) prepared in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> retained the linker scaffold as in the <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> compounds; however, the diamidine moiety for these three molecules was modified to include long-chain aliphatic primary amine moieties. It was thought that these compounds would also retain a hydrogen bond donor as found in the original diamidines (in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) but extend into sites 1 and 3 to occupy regions indicated by donor SILCS FragMaps adjacent to acidic residues (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). In addition, the linkers may provide hydrophobic interactions with residues of Ca<sup>2+</sup>-S100B at or nearby the lower portion of site 1, also termed here as a channel connecting sites 1 and sites 2 (i.e., with residues Phe43, Leu44, Ile80, and Ala83) as indicated by the SILCS FragMaps (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><div class="NLM_p">The in vitro fluorescence polarization competition assay (FPCA) was completed with these three compounds, and they were all (<b>8</b>, <b>9a</b>, <b>9b</b>) found to compete with the site 1 probe, so IC<sub>50</sub> values were measured and the dissociation constants determined to be in the low micromolar range in all cases (<5 μM; see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). However, their EC<sub>50</sub> values in cells were significantly higher than their <i>K</i><sub>D</sub> values (EC<sub>50</sub>(<b>8</b>) ≈ 35–40 μM; EC<sub>50</sub>(<b>9a</b>,<b>b</b>) ≈ 25–50 μM), so off-target affects are likely in each of these cases. Likewise, all three of these compounds interacted with Ca<sup>2+</sup>-S100B as measured using NMR, and in the case of compound <b>8</b>, it showed similar chemical shift perturbations as pentamidine and heptamidine, as well as numerous additional perturbations. The other two compounds (<b>9a</b>, <b>9b</b>) caused significant broadening to the NMR spectra due to an intermediate exchange and/or because of protein aggregation. In these cases, the NMR and FPCA results provided indication that the long-chain primary amine moiety did indeed interact with site 1 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Nonetheless, X-ray crystallography experiments were initiated and structure determinations were attempted for Ca<sup>2+</sup>-S100B complexes with compounds <b>8</b>, <b>9a</b>, and <b>9b</b> to further explore this possibility.</div><div class="NLM_p last">Cocrystals of <b>8</b>, <b>9a</b>, and <b>9b</b> were obtained from conditions similar to that of <b>6b</b> and <b>5a</b>. Although an examination of electron density maps could confirm the presence of small-molecule ligands occupying the predicted binding sites, this subfamily of compounds maintained low occupancy despite various attempts at improvement. Among these compounds, the S100B·<b>9a</b> crystal diffraction data provided the best ligand electron density, and the atoms of benzamidine-like chemical groups could be accurately modeled. However, the acyl chains terminated with amino groups could not be tracked in the electron density with the same confidence. Therefore, in silico methods were used to predict the positions of atoms with weak and/or missing electron density (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>). Both AutoDock and MC-SILCS sampling similarly place the linker alkyl chain. The location of one of the terminal alkyl chains predicted by AutoDock places the amino group such that it hydrogen-bonds with Glu86 and His85. The location of the second amino group does not allow hydrogen bonding with the protein. The only favorable interactions would be with the hydrophobic environment provided by the side chains of Leu44, Ala83, and Phe88. MC-SILCS, on the other hand, places the first amino group in the vicinity of Glu2 (of the other S100B chain), and the second group close to Glu46, forming hydrophilic interactions in both cases. These locations are associated with the positive donor SILCS FragMap adjacent to these residues leading to favorable placement of the basic group (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>). The MC-SILCS docking also point to the diversity of conformations populated by the terminal groups. The additional hydrogen bonding predicted by AutoDock and/or MC-SILCS would explain the increased affinity of this subfamily for S100B as measured by FPCA. The diversity of orientations found by the two methods is also consistent with the alkyl tails not being resolved in the crystal structure. The small differences in affinity between the amino group containing compounds are likely due to the varying lengths of linkers and associated positions of the amino groups, which would likely impact the hydrogen bond network between the ligands and the protein. Importantly, the SILCS modeling nicely explains why these molecules compete with TRTK12, since an interaction at Glu46 would compete with the interactions between TRTK12 and S100B as seen in the cocrystal structure.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Characterization of <i>N</i>-Acyl Substituted Diamidine Analogs</h3><div class="NLM_p last">Two compounds (<b>10</b>, <b>11</b>) were prepared as shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> that retained the heptamidine linker scaffold; however, the diamidine moieties for these molecules were designed to retain two hydrogen bond donors as found in heptamidine but also to have some aliphatic character either via a phenyl ring (compound <b>10</b>) or with a methyl group (compound <b>11</b>). Both of these compounds had an affect in the cellular assays with compound <b>10</b> having contributions from general toxicity effects (EC<sub>50</sub>(<b>10</b>) < 1 μM; EC<sub>50</sub>(<b>11</b>) ≈ 130–150 μM). The in vitro fluorescence polarization competition assay (FPCA) was completed with these compounds and neither was able to compete with TAMRA-TRTK, indicating that they do not interact with site 1 despite their ability to bind Ca<sup>2+</sup>-S100B as determined by NMR (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Figures S2–S5</a>). <b>11</b> showed a significant number of chemical shift perturbations that mimicked those found for pentamidine and heptamidine (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Figure S6</a>). <b>10</b> did not perturb chemical shifts at the concentrations tested. Although X-ray crystallography experiments were initiated, crystallization of Ca<sup>2+</sup>-S100B complexes with compounds <b>10</b> and <b>11</b> was not successful. While the atomic level detail of compounds <b>10</b> and <b>11</b> bound to Ca<sup>2+</sup>-S100B remains somewhat elusive, the NMR and FPCA data indicate that neither of these compounds extend into site 1.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Characterization of Diamidine Analogs with a Central Substituted Amine</h3><div class="NLM_p last">Two compounds (<b>16a</b>, <b>16b</b>) were prepared as shown in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> that retained the symmetric diamidine moieties; however, the heptamidine linker was designed to include a substituted secondary amine. Such a substitution was designed to complement a hydrophobic pocket that is indicated in the SILCS FragMaps and observed in the heptamidine X-ray crystal structure.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> For these complexes, the in vitro fluorescence polarization competition assay (FPCA) was completed, and as expected, neither was able to compete with TAMRA-TRTK. Unexpectedly, the compounds bound weakly and the chemical shift perturbations were unlike those observed for pentamidine and heptamidine, suggesting an alternative-binding mode. The EC<sub>50</sub> values for these compounds were measured in cellular assays and found to affect cell death quite differently, with <b>16b</b> having little if any effect on cellular functions (EC<sub>50</sub>(<b>16a</b>) ≈ 40–50 μM; EC<sub>50</sub>(<b>16b</b>) > 500 μM).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Characterization of an Asymmetric Diamidine Molecule</h3><div class="NLM_p">In all of the structural studies completed thus far with symmetric diamidines, it was becoming clear that the position of Phe88 in the pocket between sites 1 and 2 was a common feature. Such conformation of the protein target necessitated the synthesis of very large molecules to circumvent this occlusion and to compete with a site 1 binding peptide (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>). This requires a compound that eliminates the symmetric nature of the pentamidine/heptamidine scaffold such that only site 3 interactions would occur. In addition, it was of interest to determine whether it was possible to have the channel between sites 1 and 2 open while occupying sites 2 and 3 with an inhibitor. Using chemical similarity searching with the program MOE (Chemical Computing Group), we searched an in house database of commercially available molecules for those that contain a single aromatic amidine moiety as present in heptamidine. One commercial molecule, compound <b>17</b>, fit these criteria and was tested for its ability to bind to Ca<sup>2+</sup>-S100B. Remarkably, <b>17</b> was found to bind Ca<sup>2+</sup>-S100B via NMR, has a binding affinity in the low micromolar range (<i>K</i><sub>D</sub> = 6.0 ± 0.2 μM; <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), and had a low EC<sub>50</sub> in cellular assays (EC<sub>50</sub> < 1 μM; <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) likely due to contributions from general toxicity affects. Therefore, compound <b>17</b> is not itself an end point for a pharmaceutical, but it turned out to provide very important information for structure-based drug design. Specifically, <b>17</b> was found to inhibit TAMRA-TRTK binding into site 1 in the FPCA, despite being designed to interact with sites 2 and 3. Notably, this is the first small molecule with a pentamidine-like scaffold (MW < 500 Da) shown to block the site 1 binding peptide TAMRA-TRTK. X-ray crystal determination of <sup>Ca</sup>S100B·<b>17</b> was thus undertaken to identify the binding site of <b>17</b>.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <sup>Ca</sup>S100B·<b>17</b></h4><div class="NLM_p">The X-ray structure of <b>17</b> bound, calcium loaded S100B (<sup>Ca</sup>S100B·<b>17</b>) was solved at 1.74 Å resolution (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A). The asymmetric unit contained two S100B monomers. Residues Met0 to Phe88 were modeled in monomer A, while residues Met0 to Glu89 were modeled in monomer B. The monomers in the <sup>Ca</sup>S100B<i>·</i><b>17</b> structure bound two calcium ions. Only one molecule <b>17</b> was modeled, as the symmetrical binding cleft formed by the dimer remained vacant. In the final refined model (<i>R</i><sub>crys</sub>/<i>R</i><sub>free</sub> of 0.209/0.247), all residues were in the favored (100%) region of the Ramachandran plot with no outliers (see <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). The occupation of the binding cleft by <b>17</b> in sites 2 and 3 was discovered to indeed be the case, but it required an entirely different conformation of helix 4 versus that observed in the structures of the other pentamidine/heptamidine scaffold compounds. In fact, <b>17</b> binds a more native-like conformation (unliganded) of Ca<sup>2+</sup>-S100B while other pentamidine derivatives require a breaking of the last turn of helix 4 coinciding with a repositioning of Phe87 and Phe88 in stacking position with the cyclic moieties (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). Although the global positioning of <b>17</b> in site 2 and 3 remains similar (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>B,C), there are other major differing interactions with the protein. The carbamimidoylphenoxyl group of <b>17</b>, which would be expected to hydrogen-bond with the backbone carbonyl of His85, as occurs with pentamidine, lies in the hydrophobic pocket formed by Val8, Phe88, and Met0, where it makes bidentate hydrogen bonds with the side chain carbonyl of Asp12 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). In addition, the indolecarboximidamide of <b>17</b> is in the hydrophobic pocket formed by Ala83, Phe43, and Phe87 and makes hydrogen bonds with not only the backbone carbonyl of Phe43 but also the backbone carbonyl His42 and the side chain carbonyl of Glu45. Notably, the binding orientation of <b>17</b> is such that the FF-gate residue, Phe88, assumes an open conformation that allows access between sites 1 and 2. Thus, when comparing all of the inhibitor complex structures so far available for Ca<sup>2+</sup>-S100B, there are two major conformations observed for the position of the C-terminal residues. One is with Phe88 within the small channel connecting sites 1 and 2 (i.e., the “closed” FF-gate), and the other is with this small channel open and Phe88 outside this narrow channel (i.e., the “open” FF-gate) and part of helix 4 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0001.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystallographic structure of the S100B·<b>17</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR">5DKR</a>). (A) |<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>| electron density omit map (blue mesh) of <b>17</b> (ball and stick) contoured at 2.5σ levels. (B) <b>17</b> within sites 2 (yellow highlight) and 3 (gray highlight) of the S100B homodimer (surface in blue). (C) Modeled orientation of <b>17</b> overlaid on the previously elucidated compound-bound structures of pentamidine and heptamidine. (D) The specific interactions of <b>17</b> with dimeric S100B (blue ribbons) are rendered with residues (sticks) within 4 Å of the <b>17</b> highlighted in yellow (site 2) or gray (site 3). Compound <b>17</b> is situated within a hydrophobic pocket with atoms within hydrogen bond distance at the backbone carbonyls of His42 and Phe43, and the side chains of Aps12 and Glu45.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72433" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72433" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Three binding sites or “hot spots” on Ca<sup>2+</sup>-bound S100B were recognized in structural studies with several different <sup>Ca</sup>S100B<i>·</i>peptide and <sup>Ca</sup>S100B<i>·</i>SBiX complexes.<a onclick="showRef(event, 'ref38 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref38 ref40 ref41">(38, 40, 41)</a> Pentamidine, which binds to sites 2 and 3, exhibits enough potency to be used in clinical trials for malignant melanoma. However, its low efficacy, poor specificity, and toxicity issues limit its therapeutic potential and needed to be addressed. The development of heptamidine was a reasonable improvement as indicated by increased efficacy in cellular assays, with the efficacy shown to be dependent on S100B levels (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). In the evaluation of heptamidine, a more stable knockdown of S100B was developed and used, and selectivity in killing melanoma cells with high S100B over those with low S100B was clearly observed.</div><div class="NLM_p">In this study, we attempted to extend the heptamidine scaffold from binding sites 2 and 3 toward site 1. The addition of aliphatic amine groups indeed showed significant competition with the site 1 and can be used as a molecular probe of the S100B binding pocket. Also, the results here indicate that the optimal length of linker region between the aromatic moieties and from the amidine to amine is seven methylene groups. From the available crystallographic structure of S100B in complex with heptamidine,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> the importance of hydrogen bonding made with the backbone carbonyl of Phe43 may be assumed. Although higher affinity binders were observed <i>in vitro</i>, it did not improve the efficacy in cellular assays when compared to many of the other diamidine derivatives (see <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>); this result may be due to decreased cellular absorption or unfavorable solubility.</div><div class="NLM_p">Further analysis of the crystal structures of S100B in complex with symmetric pentamidine analogs revealed a consistent reorganization of the C-terminal helix from the native state toward the closing of the “FF-gate”. This includes heptamidine even though such a reorganization of the C-terminal helix was not reported in the original coordinates deposited in the PDB. A reexamination of the deposited raw data for <sup>Ca</sup>S100B-heptamidine (structure factors) and newly calculated maps, in light of these new structures, showed that the heptamidine-bound state of S100B does indeed exhibit the same reorganization of the C-terminal helix associated with a “closed” FF-gate, with Phe88 blocking the cavity connecting sites 1 and 2, as found here for structures of <sup>Ca</sup>S100B·<b>5a</b> and <sup>Ca</sup>S100B·<b>6b</b>. Thus, the protein conformational change necessary for the binding of symmetric pentamidine analogs leads to closing of the FF-gate, which pushes the inhibitors away from site 1 and toward bulk solvent. As the occupation of site 1 (the p53-peptide binding site) is predicted to be important for successful inhibition of the targeted PPI, the dependence on this conformation change for binding has proven to be problematic. The inability of pentamidine/heptamidine and many of the analogs to compete effectively with the peptide probe of site 1 may, at least partially, be explained by the need for the FF-gate to be in the “open” conformation with a more native-like orientation of Phe87 and Phe88. Relaxation dispersion curves of S100B in the absence and presence of peptides or small molecule inhibitors indicate that the C-terminus of Ca<sup>2+</sup>-S100B exists in at least two slowly exchanging conformations in the millisecond time regime in solution, consistent with the FF-gating mechanism proposed here<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>). Notably, by breaking the symmetry of heptamidine and its other analogs, we identified a compound, <b>17</b>, that binds to the “open” FF-gate while maintaining several desirable features of the pentamidine scaffold. Unfortunately, the killing efficacy of <b>17</b> far exceeds in binding affinity and shows no S100B-dependent killing in S100B-knockdown cell lines. Thus, compound <b>17</b> does not resolve the toxicity problems of pentamidine-like compounds, but its structure bound to <sup>Ca</sup>S100B represents an important advance in S100B inhibitor design, perhaps as a linker, since it enables the design of inhibitors that can take advantage of all three hot spots in <sup>Ca</sup>S100B for the first time (i.e., sites 1, 2, and 3; <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).</div><figure id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0002.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Small molecule inhibitors of Ca<sup><b>2+</b></sup>-S100B reveal two protein conformations. (A) Overlaid crystallographic structures (tubes) of S100B in uninhibited (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MHO">1MHO</a>, blue) and pentamidine (and analog, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>) inhibited conformations. A substantial conformational shift is observed in the C-terminal loop region. (B) A close-up rendering of the C-terminal loop region reveals that Phe87 and Phe88 (sticks) are the major contributors to the conformational shift. (C) Surface rendering showing that a bound pentamidine analog (ball and stick, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>) has difficulty accessing site 1 (magenta) from site 2 (yellow) because of the obstruction caused by intruding phenylalanine side chains (green). (D) Other non-pentamidine-like inhibitors (ball and stick, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PE0">4PE0</a>) have ready access to site 1 from site 2 when the phenylalanine side chains are in a native conformation.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">Experimental Procedures</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">All reactions were performed under an atmosphere of nitrogen, and all solvents were removed on a rotary evaporator under reduced pressure. TLC was performed on plates coated with silica gel GHLF 0.25 mm plates (60 F254) (Analtech). Mass spectra were obtained on a ThermoFinnigan LCQ Classic. <sup>1</sup>H and <sup>13</sup>C NMR spectra were obtained using a 500 MHz Varian NMR spectrometer. Melting points were determined in open capillary tubes using a Mel-Temp melting point apparatus; melting points are uncorrected. Combustion analysis was performed by Atlantic Microlab, Inc. (Norcross, GA). Synthesis follows that of Hisashi et al. (<a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a>–<a class="ref internalNav" href="#sch4" aria-label="4">4</a>) Purity of all of the compounds synthesized and/or used in this study were determined to be >95% by HPLC analyses (see <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Supporting Information Figures S7–S29</a>).</div><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Heptamidine (<b>4a</b>), Pentamidine (<b>4b</b>), Hexamidine (<b>4c</b>)</h3><div class="NLM_p last">Compounds <b>4a</b>–<b>c</b> were synthesized by reported procedure<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> as shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>.</div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 1,7-Bis(4-(4, 5-dihydro-1<i>H</i>-imidazol-2-yl)phenoxy)heptane (<b>5a</b>)</h3><div class="NLM_p"><b>3a</b> (0.1 g, 0.25 mmol) and 1,2-diaminoethane (0.075 g, 1.25 mmol) were dissolved in DMF (2 mL) and stirred at room temperature overnight. DMF was removed under vacuum (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The crude reaction mixture was purified on neutral alumina column (6:94 methanol/chloroform) to yield the pure product <b>5a</b> as a white solid. Yield 61%; Mp (250–252 °C). <sup><b>1</b></sup><b>H NMR</b> (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.98 (d, <i>J</i> = 8.8 Hz, 4H), 7.11 (d, <i>J</i> = 8.8 Hz, 4H), 4.07 (t, <i>J</i> = 6.2 Hz, 4H), 3.85 (m, 8H), 1.72–1.74 (m, 4H), 1.41 (m, 6H); <sup><b>13</b></sup><b>C NMR</b> (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.8, 162.6, 130.3, 116.0, 114.7, 67.9, 45.3, 28.3, 25.3; MS (ESI) <i>m</i>/<i>z</i>: 421 [M + H]</div><div class="NLM_p last">Compounds <b>5b</b>–<b>d</b> were similarly synthesized as per the above procedure (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 1,5-Bis(4-(4,5-dihydro-1<i>H</i>-imidazol-2-yl)phenoxy)pentane (<b>5b</b>)</h3><div class="NLM_p last">White solid. Yield 61%. Mp (250–252 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.01 (d, <i>J</i> = 8.8 Hz, 4H), 7.13 (d, <i>J</i> = 8.4 Hz, 4H), 4.11 (t, <i>J</i> = 6.4 Hz, 4H), 3.83 (m, 8H), 1.77–1.82 (m, 4H), 1.57–1.59 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.3, 162.2, 130.9, 115.3, 68.4, 45.4, 28.5, 22.4. MS (ESI) <i>m</i>/<i>z</i>: 393 [M + H].</div></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 1,6-Bis(4-(4,5-dihydro-1<i>H</i>-imidazol-2-yl)phenoxy)hexane (<b>5c</b>)</h3><div class="NLM_p last">Off white solid. Yield 63%. Mp (221–223 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.00 (d, <i>J</i> = 8.4 Hz, 4H), 7.12 (d, <i>J</i> = 8.4 Hz, 4H), 4.09 (t, <i>J</i> = 6.2 Hz, 4H), 3.85 (m, 8H), 1.76 (m, 4H), 1.48 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.3, 163.3, 130.9, 115.8, 115.3, 68.4, 45.4, 28.8, 25.5. MS (ESI) <i>m</i>/<i>z</i>: 407 [M + H].</div></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 1,8-Bis(4-(4, 5-dihydro-1<i>H</i>-imidazol-2-yl)phenoxy)octane (<b>5d</b>)</h3><div class="NLM_p last">Off white solid. Yield 61%. Mp (198–200 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.01 (d, <i>J</i> = 8.4 Hz, 4H), 7.14 (d, <i>J</i> = 8.4 Hz, 4H), 4.08 (t, <i>J</i> = 6.4 Hz, 4H), 3.91 (m, 8H), 1.71–1.75 (m, 4H), 1.35–1.42 (m, 8H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.9, 163.1, 130.6, 114.9, 68.1, 44.6, 28.6, 28.3, 25.3. MS (ESI) <i>m</i>/<i>z</i>: 435 [M + H].</div></div><div id="sec6_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 1,7-Bis(4-(1,4,5,6-tetrahydropyrimidin-2-yl)phenoxy)heptane (<b>6a</b>)</h3><div class="NLM_p"><b>3a</b> (0.1 g, 0.25 mmol) and 1,3-diaminopropane (0.092 g, 1.25 mmol) were dissolved in DMF (2 mL) and stirred at room temperature overnight. DMF was removed under vacuum (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The crude reaction mixture was purified on neutral alumina column (6:94 methanol/chloroform) to yield the pure product <b>6a</b> as a white solid. Yield 65%. Mp (183–185 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.72 (d, <i>J</i> = 8.4 Hz, 4H), 7.12 (d, <i>J</i> = 8.4 Hz, 2H), 4.06 (t, <i>J</i> = 6.4 Hz, 4H), 3.45–3.47 (m, 8H), 1.94 (m, 4H), 1.72–1.74 (m, 4H), 1.41 (m, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 158.8, 129.8, 120.4, 115.1, 68.4, 28.8, 25.8, 18.3. MS (ESI) <i>m</i>/<i>z</i>: 449 [M + H].</div><div class="NLM_p last">Compounds <b>6b</b>–<b>d</b> were similarly synthesized as per the above procedure (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div></div><div id="sec6_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 1,5-Bis(4-(1,4,5,6-Tetrahydropyrimidin-2-yl)phenoxy)pentane (<b>6b</b>)</h3><div class="NLM_p last">White solid. Yield 62%. Mp (99–101 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.83 (d, <i>J</i> = 8.8 Hz, 4H), 7.23 (d, <i>J</i> = 8.4 Hz, 4H), 4.19 (t, <i>J</i> = 6.2 Hz, 4H), 3.55 (m, 8H), 1.92–2.05 (m, 4H), 1.87–1.92 (m, 4H), 1.67–1.9 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 158.8, 129.8, 120.5, 115.1, 68.3, 28.6, 22.5, 18.3. MS (ESI) <i>m</i>/<i>z</i>: 421 [M + H].</div></div><div id="sec6_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 1,6-Bis(4-(1,4,5,6-Tetrahydropyrimidin-2-yl)phenoxy)hexane (<b>6c</b>)</h3><div class="NLM_p last"><b>3c</b> (0.1 g, 0.26 mmol) and 1,3-diaminopropane (0.096 g, 1.3 mmol) were reacted using the same procedure as the one described for the synthesis of <b>6a</b>, providing <b>6c</b> as white solid. Yield 61%. Mp (274–276 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.72 (d, <i>J</i> = 8.4 Hz, 4H), 7.13 (d, <i>J</i> = 9.2 Hz, 4H), 4.08 (t, <i>J</i> = 6.2 Hz, 4H), 3.44–3.48 (m, 8H), 1.94–1.97 (m, 4H), 1.76 (m, 4H), 1.49 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 158.8, 129.8, 120.5, 115.1, 68.4, 28.8, 25.6, 18.3. MS (ESI) <i>m</i>/<i>z</i>: 435 [M + H].</div></div><div id="sec6_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 1,8-Bis(4-(1,4,5,6-tetrahydropyrimidin-2-yl)phenoxy)octane (<b>6d</b>)</h3><div class="NLM_p last">White solid. Yield 64%. Mp (260–262 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.73 (d, <i>J</i> = 8.8 Hz, 4H), 7.13 (d, <i>J</i> = 8.8 Hz, 4H), 4.06 (t, <i>J</i> = 6.6 Hz, 4H), 3.34–3.47 (m, 8H), 1.94–1.96 (m, 4H), 1.72–1.75 (m, 4H), 1.36–1.42 (m, 8H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 158.8, 129.8, 115.1, 68.4, 29.1, 28.9, 25.8, 18.3. MS (ESI) <i>m</i>/<i>z</i>: 463 [M + H].</div></div><div id="sec6_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> 1,7-Bis(4-(4,5,6,7-tetrahydro-1<i>H</i>-1,3-diazepin-2-yl)phenoxy)heptane (<b>7a</b>)</h3><div class="NLM_p"><b>3a</b> (0.1 g, 0.25 mmol) and 1,4-diaminobutane (0.11 g, 1.25 mmol) were dissolved in DMF (2 mL) and stirred at room temperature for overnight. DMF was removed under vacuum (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The crude reaction mixture was purified on neutral alumina column (6:94 methanol/chloroform) to yield the pure product <b>7a</b> as a white solid. Yield 60%. Mp (169–171 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.71 (m, 4H), 7.1 (m, 4H), 4.06 (m, 4H), 3.62 (m, 8H), 1.92 (m, 8H), 1.73 (m, 4H), 1.42 (m, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.5, 163.3, 131.5, 121.6, 114.9, 68.4, 28.8, 25.9, 25.7. MS (ESI) <i>m</i>/<i>z</i>: 477 [M + H]</div><div class="NLM_p last">Compounds <b>7b</b>–<b>d</b> were similarly synthesized as per the above procedure (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div></div><div id="sec6_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> 1,5-Bis(4-(4,5,6,7-tetrahydro-1<i>H</i>-1,3-diazepin-2-yl)phenoxy)pentane (<b>7b</b>)</h3><div class="NLM_p last">Off white solid. Yield 59%. Mp (153–155 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.74 (d, <i>J</i> = 6.00 Hz, 4H), 7.10 (d, <i>J</i> = 6.00 Hz, 4H), 4.10 (m, 4H), 3.63 (m, 8H), 1.92 (m, 8H), 1.80 (m, 4H), 1.57 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.4, 163.2, 131.5, 121.6, 115.0, 68.4, 43.9, 28.5, 25.9, 22.4. MS (ESI) <i>m</i>/<i>z</i>: 449 [M + H].</div></div><div id="sec6_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> 1,6-Bis(4-(4,5,6,7-tetrahydro-1<i>H</i>-1,3-diazepin-2-yl)phenoxy)hexane (<b>7c</b>)</h3><div class="NLM_p last">White solid. Yield 57%. Mp (174–176 °C). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.57–7.71 (m, 4H), 7.09–7.21 (m, 4H), 4.06–4.21 (m, 4H), 3.56–3.69 (m, 4H), 3.12–3.22 (m, 4H), 1.81–2.01 (m, 4H), 1.62–1.79 (m, 6H), 1.51–1.59 (m, 2H), 1.40–1.51 (m, 4H). MS (ESI) <i>m</i>/<i>z</i>: 463 [M + H],</div></div><div id="sec6_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> 1,8-Bis(4-(4,5,6,7-tetrahydro-1<i>H</i>-1,3-diazepin-2-yl)phenoxy)octane (<b>7d</b>)</h3><div class="NLM_p last"><b>3d</b> (0.1 g, 0.24 mmol) and 1,4-diaminobutane (0.106 g, 1.21 mmol) were reacted using the same procedure as the one described for the synthesis of <b>7a</b>, providing <b>7d</b> as off white solid. Yield 62%. Mp (223–225 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.69 (d, <i>J</i> = 7.6 Hz, 4H), 7.11 (d, <i>J</i> = 8.8 Hz, 4H), 4.05–4.08 (m, 4H), 3.63 (m, 8H), 1.93 (m, 8H), 1.72–1.74 (m, 4H), 1.36–1.42 (m, 8H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 164.5, 163.3, 131.5, 130.8, 121.6, 115.5, 114.9, 68.4, 43.9, 29.0, 28.8, 25.9, 25.7. MS (ESI) <i>m</i>/<i>z</i>: 491 [M + H]</div></div><div id="sec6_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 4,4′-(Pentane-1,5-diylbis(oxy))bis(<i>N</i>-(5-aminopentyl)benzimidamide) (<b>8</b>)</h3><div class="NLM_p last"><b>3b</b> (0.1 g, 0.25 mmol) and 1,2-diaminopentane (0.128 g, 1.25 mmol) were dissolved in DMF (2 mL) and stirred at room temperature overnight. DMF was removed under vacuum, and the crude reaction mixture was washed with chloroform (5 mL each). Residue was extracted with water and chloroform, water layer was separated and evaporated to obtain solid residue (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). It was dissolved in methanol (1 mL), and product was precipitated by the addition of diethyl ether (5 mL). Solvent was decanted and dried in vacuum to obtain pure product <b>8a</b> as a white solid. Yield 51%. Mp (109–111 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.80 (d, <i>J</i> = 7.2 Hz, 4H), 7.09 (d, <i>J</i> = 7.6 Hz, 4H), 4.08 (m, 4H), 3.41 (m, 4H), 2.73 (m, 4H), 1.40–1.79 (18). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 162.2, 130.6, 120.9, 114.9, 68.3, 42.7, 28.6, 27.3, 27.2, 23.4, 22.4. MS (ESI) <i>m</i>/<i>z</i>: 511 [M + H].</div></div><div id="sec6_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 4,4′-(Heptane-1,7-diylbis(oxy))bis(<i>N</i>-(7-aminoheptyl)benzimidamide) (<b>9a</b>)</h3><div class="NLM_p last">Procedure is similar as described for <b>8</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). White solid. Yield 47%. Mp (98–101 °C). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 7.89 (d, <i>J</i> = 8 Hz, 4H), 7.30 (d, <i>J</i> = 8.4 Hz, 4H), 4.28 (m, 4H), 3.3 (m, 4H), 2.99 (m, 2H), 1.64–2.03 (m, 32H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 162.1, 130.5, 121.1, 114.9, 79.6, 68.4, 42.8, 28.8, 28.0, 27.2, 25.8, 23.5. MS (ESI) <i>m</i>/<i>z</i>: 595 [M + H].</div></div><div id="sec6_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> 4,4′-(Octane-1,8-diylbis(oxy))bis(<i>N</i>-(7-aminoheptyl)benzimidamide) (<b>9b</b>)</h3><div class="NLM_p last">Solid. Yield 42%. Mp (111–113 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.79 (d, <i>J</i> = 8 Hz, 4H), 7.08 (d, <i>J</i> = 7.6 Hz, 4H), 4.04 (m, 4H), 2.68 (m, 4H), 1.53–1.70 (m, 12H), 1.25–1.39 (m, 24H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.9, 162.1, 130.5, 120.8, 114.9, 68.4, 42.8, 40.3, 29.0, 28.8, 28.5, 28.4, 27.7, 26.4, 26.2, 26.1, 25.7. MS (ESI) <i>m</i>/<i>z</i>: 609 [M + H].</div></div><div id="sec6_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> 4,4′-(Heptane-1,7-diylbis(oxy))bis(<i>N</i>-(phenylcarbamoyl)benzimidamide) (<b>10</b>)</h3><div class="NLM_p last"><b>4a</b> (0.1 g, 0.27 mmol) and DBU (0.16 mL, 1.1 mmol) were dissolved in DMF (2 mL) and stirred at 40 °C. A solution of phenyl isocyanate (0.06 mL, 0.54 mmol) in DMF (0.5 mL) was added to the above reaction mixture and was stirred at 40 °C for 2 h. DMF was removed under reduced pressure, chloroform (20 mL) was added to the residue and washed with water (10 mL) (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). The organic layer was separated, concentrated, and purified on neutral alumina column (4:96 methanol/chloroform) to yield the pure product <b>11</b> as a white solid. Yield 68%. Mp (136–138 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.01 (d, <i>J</i> = 8.4 Hz, 4H), 7.65 (d, <i>J</i> = 8.0 Hz, 4H), 7.24 (t,<i>J</i> = 7.8 Hz, 4H) 7.03 (d, <i>J</i> = 9.6 Hz, 4H), 6.91–6.95 (m, 2H), 4.03–4.06 (m, 4H), 1.74 (m, 4H) 1.43 (m, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.9, 130.4, 129.1, 119.0, 114.7, 68.3, 28.9, 25.8. MS (ESI) <i>m</i>/<i>z</i>: 607 [M + H].</div></div><div id="sec6_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> 4,4′-(Heptane-1,7-diylbis(oxy))bis(<i>N</i>-(methylcarbamoyl)benzimidamide) (<b>11</b>)</h3><div class="NLM_p last"><b>4a</b> (0.1 g, 0.27 mmol) and DBU (0.20 mL, 1.35 mmol) were dissolved in DMF (2 mL) and stirred at 40 °C. A solution of <i>N</i>-succinamidyl-<i>N</i>-methyl carbamate (0.093 g, 0.54 mmol) in DMF (0.5 mL) was added to the above reaction mixture and was stirred at 40 °C for 2 h (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). DMF was removed under reduced pressure, and chloroform (20 mL) was added to the residue and washed with water (10 mL). The organic layer was separated, chloroform was evaporated and purified on neutral alumina column (5:95 methanol/chloroform) to yield the pure product <b>13a</b> as a white solid. Yield 61%. Mp (147–149 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.90 (d, <i>J</i> = 8.4 Hz, 4H), 6.97 (d, <i>J</i> = 8.4 Hz, 4H), 4.02 (t, <i>J</i> = 6.4 Hz, 4H), 2.60 (s, 6H), 1.71–1.74 (m, 4H), 1.41 (m, 6H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.8, 161.5, 129.3, 127.5, 114.2, 68.0, 28.9, 26.6, 25.8. MS (ESI) <i>m</i>/<i>z</i>: 483 [M + H].</div></div><div id="sec6_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> <i>N</i>-Benzyl-3-bromo-<i>N</i>-(3-bromopropyl)propan-1-amine (<b>13a</b>)</h3><div class="NLM_p last">Potassium carbonate (3.22 g/23.4 mmol/2.5 equiv) and 1,3-dibromopropane (<b>12</b>) (4.32 g/21.5 mmol/2.3 equiv) were added to benzylamine (1.0 mL/9.3 mmol/1 equiv) in acetone (50 mL). The reaction mixture was refluxed for 48 h at 60 °C and then cooled to room temperature. Acetone was removed under reduced pressure to give a solid residue. Water (75 mL) was added to the residue and extracted with dichloromethane (25 mL × 3). The organic layer was separated and dried with sodium sulfate, filtered, and evaporated to give pure solid <b>13a</b> (2.15 g, 69%) (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.14–7.54 (m, 5H), 3.29–3.61 (m, 6H), 2.21–2.46 (m, 4H), 1.92–2.14 (m, 4H). MS (ESI) <i>m</i>/<i>z</i>: 350 [M + H].</div></div><div id="sec6_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> 3-Bromo-<i>N</i>-(3-bromopropyl)-<i>N</i>-propylpropan-1-amine (<b>13b</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 3.53–3.41 (m, 4H), 2.59–3.46 (m, 4H), 2.21–2.29 (m, 2H), 1.91–1.99 (m, 4H), 1.40–1.51 (m, 2H), 0.91–0.99 (m, 3H). MS (ESI) <i>m</i>/<i>z</i>: 302 [M + H].</div></div><div id="sec6_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 4,4′-(((Benzylazanediyl)bis(propane-3,1-diyl))bis(oxy))dibenzonitrile (<b>14a</b>)</h3><div class="NLM_p last">Cesium carbonate (10.9 g/33.6 mmol/4 equiv) and <b>15a</b> (1.4 g/4.2 mmol/0.5 equiv) were added to <b>1</b> (1 g/8.4 mmol/1 equiv) in acetone (50 mL). The reaction mixture was refluxed for 48 h at 60 °C and then cooled to room temperature. Acetone was removed under reduced pressure to give a solid residue (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). Water (75 mL) was added to the residue and extracted with dichloromethane (25 mL × 3). The organic layer was separated and washed with 10% sodium hydroxide in water (10 mL × 3) and washed with water until neutral to pH paper. The organic layer was dried with sodium sulfate, filtered, and evaporated to give pure solid <b>14a</b>. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.40–7.62 (m, 4H), 7.717–7.39 (M, 5H), 6.79–6.87 (m, 4H), 3.82–4.06 (m, 4H), 3.61 (s, 2H), 2.59–2.68 (m, 4H), 1.89–1.99 (m, 4H). MS (ESI) <i>m</i>/<i>z</i>: 425 [M + H].</div></div><div id="sec6_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> 4,4′-(((Propylazanediyl)bis(propane-3,1-diyl))bis(oxy))dibenzonitrile (<b>14b</b>)</h3><div class="NLM_p last"><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 7.53–7.61 (m, 4H), 6.80–6.92 (m, 4H), 4.09–4.19 (m, 4H), 2.42–2.53 (m, 4H), 1.82–1.99 (m, 4H), 1.39–1.51 (m, 2H), 0.92–0.98 (m, 3H). MS (ESI) <i>m</i>/<i>z</i>: 378 [M + H].</div></div><div id="sec6_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> 4,4′-(((Benzylazanediyl)bis(propane-3,1-diyl))bis(oxy))dibenzimidamide (<b>16a</b>)</h3><div class="NLM_p">1.0 g of <b>14a</b> (2.3 mmol) in anhydrous dioxane (25 mL) and dry methanol (20 mL) was saturated with hydrochloric acid gas at 0–5 °C under nitrogen atmosphere. The reaction mixture was stirred at room temperature for 36 h, after which the solvent was reduced by 50%. 30 mL of anhydrous ether was added and the resulting precipitate was filtered, washed with anhydrous ether (10 mL × 5), and dried under vacuum to yield <b>15a</b> (0.83 g, 73% yield). Dry solid <b>15a</b> (0.8 g, 1.6 mmol) was dissolved in dry ethanol (50 mL) and 2 M ethanolic ammonia (50 mL), to which ammonium chloride (0.025 g, 0.47 mmol) was added. This mixture was refluxed at 80 °C for 32 h under nitrogen atmosphere, after which it was cooled and concentrated (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>). The solid obtained was filtered and washed with cold ethanol (5 mL × 3) and ether (10 mL × 3) to give diamidine product <b>16a</b> as brown solid. Yield 75%. Mp (183–185 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.82–7.84 (m, 4H), 7.18–7.25 (m, 5H), 7.01–7.3 (m, 4H), 4.04–4.05 (m, 4H), 3.56 (s, 2H), 2.55 (m, 4H), 1.88 (m, 4H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.0, 163.4, 130.5, 129.7, 128.9, 128.4, 127.1, 119.5, 119.3, 115.0, 66.4, 58.4, 49.7, 26.5. MS (ESI) <i>m</i>/<i>z</i>: 460 [M + H].</div><div class="NLM_p last">Compounds <b>16b</b> was similarly synthesized as per the above procedure (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>).</div></div><div id="sec6_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 4,4′-(((Propylazanediyl)bis(propane-3,1-diyl))bis(oxy))dibenzimidamide (<b>16b</b>)</h3><div class="NLM_p last">Off white solid. Yield 67%. Mp (177–179 °C). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88 (d, <i>J</i> = 8.4 Hz, 4H), 7.16 (d, <i>J</i> = 8.8 Hz, 4H), 4.21 (m, 4H), 3.25 (m, 4H), 3.07 (m, 2H), 2.24 (m, 4H), 1.73–1.75 (m, 2H), 0.92 (t, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.0, 163.0, 130.6, 120.0, 115.2, 65.9, 23.2, 16.8, 11.3. MS (ESI) <i>m</i>/<i>z</i>: 412 [M + H].</div></div><div id="sec6_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 2-(4-(4-Carbamimidoylphenoxy)phenyl)-1<i>H</i>-indole-6-carboximidamide (<b>17</b>)</h3><div class="NLM_p last"><b>17</b> was purchased from Sigma-Aldrich (St. Louis, MO, catalog no. A8675). A purity of >95% was confirmed HPLC and by HPLC–MS.</div></div><div id="sec6_25_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Cellular Assays</h3><div id="sec6_25_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Cell Lines and Cell Culture</h4><div class="NLM_p last">WM115 (American Type Tissue Collection (ATCC)) malignant melanoma cells were cultured in minimum essential medium (Invitrogen) supplemented with 10% heat-inactivated fetal bovine serum (FBS). Cells were maintained in a 37 °C incubator with 5% CO<sub>2.</sub> The WM115 cells have the activating BRAF V600E mutation and wild type NRAS.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> As shown by Western blot analysis, the WM115 cells have elevated levels of endogenous S100B protein.</div></div><div id="sec6_25_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Growth Inhibition Assay</h4><div class="NLM_p last">A collection of pentamidine analogs were tested for their ability to inhibit the growth of WM115 human melanoma cells expressing high levels of S100B (WM115<sup>shSCRAMBLED</sup>) and low levels of S100B (WM115<sup>shS100B</sup>)<a onclick="showRef(event, 'ref30 ref43'); return false;" href="javascript:void(0);" class="ref ref30 ref43">(30, 43)</a> using a modification of the high-throughput screening assay performed by Bachman et al.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Using a Biomek FX laboratory automation workstation (Beckman-Coulter) equipped with a 96-channel pipetting head, an amount of 20 μL of MEM (Invitrogen) supplemented with 10% fetal bovine serum was added to each well of a 384-well tissue culture plate (Corning) containing enough cells such that growing uninhibited they reach 80% confluence in ∼5 days. The cells are propagated in the presence of 0.5 mg/mL puromycin to maintain the vector expressing S100B siRNA to knockdown (KD) the protein or the same vector containing a scrambled control siRNA. After 24 h of growth at 37 °C in 5% CO<sub>2</sub> humidity controlled incubator the cells were treated with 20 μL of compound in the same culture media, while in the control cultures, only an equivalent amount of DMSO with the compound is added. Titrations of the compounds were performed in triplicate using the Biomek NXP laboratory automation workstation (Beckman-Coulter) equipped with an 8-channel pipetting head before being added to the cell plates. After four additional days of incubation, the cells were lysed by the adding 20 μL of lysis buffer consisting of 1.8% Igepal with 1:10 000 dilution of SYBR Green I (Invitrogen). The covered plates were incubated for another 24 h at 37 °C in a 5% CO<sub>2</sub> humidity controlled incubator. The fluorescence intensity was then read through the bottom of the plate using a POLARstar Optima fluorescent plate reader (BMG) with 485 nm excitation and 520 nm emission filters. The SYBR-green fluorescence is used to measure total DNA that in turn correlates with cell number as previously described.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> The EC<sub>50</sub> values of the compounds were determined using serial dilutions and done in quadruplicate, as above, with no more than 1.0% DMSO or 1000 μM compound.</div></div></div><div id="sec6_25_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Protein Purification</h3><div class="NLM_p last"><sup>15</sup>N-labeled bovine or rat S100B was expressed and purified (>99%) with methods similar to those previously.<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46, 47)</a> The concentrations of S100B stock solutions were determined using the Bio-Rad protein assay (Bio-Rad Inc., Hercules, CA) using bovine serum albumin as a standard. The S100B was concentrated to 10–15 mM using Amicon Ultra centrifugal filter units with a 10 kDa MWCO and stored at a concentration of ∼10 mM in 0.25 mM Tris, pH 7.2, with 0.25 mM DTT at −20 °C until use.</div></div><div id="sec6_25_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Protein NMR Spectrometry</h3><div class="NLM_p last">The Ca<sup>2+</sup>-loaded S100B·SBiXs HSQC samples contained 0.1 mM S100B rat subunit, 0.125 mM SBiX, 0.34 mM NaN<sub>3</sub>, 15 mM NaCl, 5% DMSO-<i>d</i><sub>6</sub>, 10 mM CaCl<sub>2</sub>, 10% D<sub>2</sub>O, 0.2% TPEN, and 10 mM HEPES, adjusted to pH 7.2 with HCl. Heteronuclear single-quantum coherence (HSQC) NMR data were collected at 37 °C with a Bruker Avance 800 US2 (800.27 MHz for protons) instrument equipped with pulsed-field gradients, four frequency channels, and a triple-resonance, <i>z</i>-axis gradient cryogenic probe. Data were processed with NMRPipe, and proton chemical shifts were reported with respect to the H<sub>2</sub>O or HDO signal taken as 4.608 ppm relative to external TSP (0.0 ppm). The <sup>15</sup>N chemical shifts were indirectly referenced as previously described using the following ratio of the zero-point frequency: 0.101 329 05 for <sup>15</sup>N to <sup>1</sup>H. HSQC spectra were visually inspected for chemical shift perturbations (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>, <a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">Figures S2–S6</a>).</div></div><div id="sec6_25_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Fluorescence Polarization Competition Assay</h3><div class="NLM_p last">The ability of the compounds to compete with a peptide probe was evaluated using an adaptation of Wilder et al., 2010 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Briefly, changes in fluorescence polarization were measured upon competition with the TAMRA-labeled version of the peptide TRTK12, which is derived from the CapZ protein residues 265–276 (TRTKIDWNKILS), for Ca<sup>2+</sup>-loaded S100B. The S100B TAMRA-TRTK FPCA was performed in 384-well black polypropylene microplate with a final volume of 20 μL containing 10 nM TAMRA-TRTK, 1 μM S100B rat, 0.01% Triton X-100, 0.5% DMSO, 50 mM HEPEs, pH 7.2, 100 mM KCl, 15 mM NaCl, 10 mM CaCl<sub>2</sub>. Titrations of each compound were done beginning from the highest concentration possible. The fluorescence polarization was measured at room temperature in a BMG PHERAstar Plus multimode microplate equipped with dual detection PMTs with an excitation of 544 nm and emission of 590 nm.</div></div><div id="sec6_25_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> X-ray Crystallography</h3><div id="sec6_25_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Crystallization</h4><div class="NLM_p last">Crystallization experiments were conducted using vapor diffusion methods. <sup>Ca</sup>S100B·<b>5a</b> crystals were grown in hanging drops consisting of 3.0:3.0 μL protein solution (40 mg/mL S100B bovine; 10 mM cacodylate, pH 7.2; 7.5 mM CaCl<sub>2</sub>; and 4 mM <b>5a</b> prepared in DMSO) and mother liquor (10% Peg 3,350; 0.1 M cacodylate, pH 6.4; 7.5 mM CaCl<sub>2</sub>) with no additional volume of mother liquor left in the crystallization chamber. <sup>Ca</sup>S100B·<b>6b</b> crystals were grown in sitting drops consisting of 0.75:0.75 μL protein solution (20 mg/mL S100B bovine; 10 mM cacodylate, pH 7.2; 7.5 mM CaCl<sub>2</sub>; and 2 mM <b>6b</b> prepared in DMSO) and mother liquor (25% Peg 3,350; 0.1 M Bis-Tris, pH 7.5; 7.5 mM CaCl2; 5% glycerol). <sup>Ca</sup>S100B·<b>6b</b> crystals were grown with mother liquor left in the crystallization chamber. <sup>Ca</sup>S100B·<b>17</b> crystals were grown in sitting drops consisting of 0.75:0.75 μL protein solution (20 mg/mL S100B bovine; 10 mM cacodylate, pH 7.2; 7.5 mM CaCl<sub>2</sub>; and 2 mM <b>10</b> prepared in DMSO) and mother liquor (40% 2-methyl-2,4-pentanediol; 0.1 M Hepes, pH 7.0; and 7.5 mM CaCl<sub>2</sub>). The crystals were grown over a period of 1–14 days at a temperature of 295 K. Crystals were not cryoprotected before being flash-cooled in liquid nitrogen.</div></div><div id="sec6_25_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Data Collection and Processing</h4><div class="NLM_p">Diffraction data for <sup>Ca</sup>S100B·<b>5a</b> crystals were collected Northeastern Collaborative Access Team (NE-CAT) beamline at the Advanced Photon Source (Argonne National Laboratory). Data were recorded at 100 K on an ADSC Q315 (315 mm × 315 mm) detector and processed by NE-CAT’s RAPD automated processing (<a href="https://rapd.nec.aps.anl.gov/rapd" class="extLink">https://rapd.nec.aps.anl.gov/rapd</a>), which uses XDS<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> for integration and scaling. A 1.53 Å data set was collected at a wavelength of 0.979 20 Å while oscillating the crystal 1.0° each frame. The space group was determined to be <i>P</i>41212.</div><div class="NLM_p last">Diffraction data for <sup>Ca</sup>S100B·<b>6b</b> and <sup>Ca</sup>S100B·<b>17</b> crystals were collected remotely<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48, 49)</a> at the Stanford Synchrotron Radiation Lightsource (SSRL) beamline 7-1. Data were recorded at 100 K on an ADSC Q315 (315 mm × 315 mm) detector with collection strategies generated by BLU-ICE.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Data sets were processed and integrated by AUTOXDS.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> A 1.59 Å data set was collected at a wavelength of 1.1271 Å while oscillating the crystal 0.74° or 0.75°, respectively, each frame. The space groups were determined to be <i>P</i>41212 and <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, respectively. Diffraction data statistics for both models are summarized in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>.</div></div><div id="sec6_25_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Structure Determination and Refinement</h4><div class="NLM_p">To determine the structure of S100B in complex with SBiXs, molecular replacement (MR) was performed. A search model from a previously elucidated S100B structure (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MHO">1MHO</a><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a>) was generated by removing coordinates for ligands and water. Molecular replacement was carried out using the AUTOMR<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52, 53)</a> function of the PHENIX<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> software suite. The models were finished by manual building within COOT.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> The models were refined with the PHENIX.REFINE<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> function of the PHENIX<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> software suite. Ligands and waters were incorporated into the models by referring to the |<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>| omit maps. The structure refinement statistics for both models are summarized in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>.</div><div class="NLM_p last">Coordinates and structure factors have been deposited in the Protein Data Bank with the following accession numbers in parentheses: <sup>Ca</sup>S100B·<b>5a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN">5DKN</a>), <sup>Ca</sup>S100B·<b>6b</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>), <sup>Ca</sup>S100B·<b>17</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR">5DKR</a>).</div></div></div><div id="sec6_25_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> In Silico Methods</h3><div class="NLM_p">Ligand design was performed using the site identification by ligand competitive saturation (SILCS) approach.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57, 58)</a> For ligand design the tier 1 SILCS approach was applied in which the protein is immersed in an aqueous solution containing 1 M benzene and propane. SILCS simulations, initiated from the SBi132·S100B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GK1">3GK1</a>) crystal structures, were performed as previously described yielding probability distributions for aromatic groups based on benzene carbons, aliphatic groups based on propane carbons, hydrogen bond donors based on water hydrogen waters and hydrogen bond acceptors based on water oxygens.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> The probability distributions in the presence of the protein are normalized with respect to probability distributions in the absence of the protein; such final distributions, termed FragMaps, take into account both functional group and protein desolvation as well as protein flexibility and protein–functional group interactions. FragMaps may be visualized as 3D contour maps that indicate the regions around the protein favorable for the different classes of functional groups (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).</div><div class="NLM_p last">The prediction of the location of the missing aminoalkyl tails of <b>9a</b> was accomplished by applying two independent methods. In both approaches the initiator Met0 residue was retained, since this residue is present in the S100B cocrystal data with <b>9a</b>. In the first method, molecular docking was performed using the AutoDock Vina program.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Standard parameters were applied to obtain 20 docked orientations. A consensus-binding mode was obtained based on favorable AutoDock score and the overlap of the partially resolved part of the ligand with the crystal data. The second approach involved the more recent tier 2 SILCS approach.<a onclick="showRef(event, 'ref57 ref60'); return false;" href="javascript:void(0);" class="ref ref57 ref60">(57, 60)</a> Tier 2 SILCS simulations were performed on the protein conformation obtained from the crystal data with compound <b>9a</b> following the removal of the partially resolved ligand. The 10 simulation trajectories that spanned a cumulative sampling time of 400 ns were used to calculate the protein functional group affinity FragMaps. In the tier 2 approach a wider range of solute molecules are used in the aqueous solution including benzene, propane, methanol, formamide, acetaldehyde, imidazole, acetate, and methylammonium at ∼0.25 M. Notable is the inclusion of methylammonium allowing the calculation of positive FragMaps that are directly relevant for docking of the positively charged amines of <b>9a</b>. Docking of <b>9a</b> into the “field” of the FragMaps used the validated MC-SILCS approach that involves Monte Carlo (MC) simulation based sampling. MC moves included translational, rotational, and dihedral sampling with Metropolis criteria determined using the ligand grid free energies obtained from the FragMaps.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Intramolecular energies of the ligand during MC sampling were calculated using the CHARMM general force field.<a onclick="showRef(event, 'ref61 ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref61 ref62 ref63">(61-63)</a> The initial conformation of the ligand was seeded with the resolved atoms from the crystal data. The MC-SILCS docking revealed a consensus-binding mode, which differed in the location of the aminoalkyl tails from the consensus AutoDock conformation.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01369">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42335" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42335" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01369" class="ext-link">10.1021/acs.jmedchem.5b01369</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional details of crystal model RMSD, in silico data, 2D <sup>1</sup>H–<sup>15</sup>N S100B HSQC NMR spectra, purities derived by HPLC, 1D <sup>1</sup>H and <sup>13</sup>C compound NMR spectra, and SMILES (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf">jm5b01369_si_001.pdf (3.2 MB)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><sup>Ca</sup>S100B·<b>5a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN">5DKN</a>), <sup>Ca</sup>S100B·<b>6b</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>), <sup>Ca</sup>S100B·<b>17</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR">5DKR</a>).</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b01369" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88082" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88082" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Weber</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span>; 
    <span class="hlFld-Affiliation affiliation">Marlene
and Stewart Greenebaum Cancer Center, University
of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a9cddecccbccdbe9dac6c487dcc4c8dbd0c5c8c7cd87cccddc"><span class="__cf_email__" data-cfemail="ceaab9abacabbc8ebda1a3e0bba3afbcb7a2afa0aae0abaabb">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael C. Cavalier</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohd. Imran Ansari</span> - <span class="hlFld-Affiliation affiliation">Computer
Aided Drug Design Center, School of Pharmacy, University of Maryland, Baltimore, Maryland 21201, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adam D. Pierce</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul T. Wilder</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span>; 
    <span class="hlFld-Affiliation affiliation">Marlene
and Stewart Greenebaum Cancer Center, University
of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura E. McKnight</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">E. Prabhu Raman</span> - <span class="hlFld-Affiliation affiliation">Computer
Aided Drug Design Center, School of Pharmacy, University of Maryland, Baltimore, Maryland 21201, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David B. Neau</span> - <span class="hlFld-Affiliation affiliation">NE-CAT, Argonne Illinois 60439, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Padmavani Bezawada</span> - <span class="hlFld-Affiliation affiliation">Computer
Aided Drug Design Center, School of Pharmacy, University of Maryland, Baltimore, Maryland 21201, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Milad J. Alasady</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas H. Charpentier</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kristen M. Varney</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eric A. Toth</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Molecular Biology, Center for Biomolecular Therapeutics
(CBT), University of Maryland School of
Medicine, Baltimore, Maryland 21201, United
States</span>; 
    <span class="hlFld-Affiliation affiliation">Marlene
and Stewart Greenebaum Cancer Center, University
of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
for Bioscience and Biotechnology Research, 9600 Gudelsky Drive, Rockville, Maryland 20850, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alexander D. MacKerell</span> - <span class="hlFld-Affiliation affiliation">Computer
Aided Drug Design Center, School of Pharmacy, University of Maryland, Baltimore, Maryland 21201, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Marlene
and Stewart Greenebaum Cancer Center, University
of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrew Coop</span> - <span class="hlFld-Affiliation affiliation">Computer
Aided Drug Design Center, School of Pharmacy, University of Maryland, Baltimore, Maryland 21201, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Marlene
and Stewart Greenebaum Cancer Center, University
of Maryland School of Medicine, Baltimore, Maryland 21201, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Alexander D. MacKerell, Jr., is co-founder and CSO of Silcsbio LLC.<br /></br></div></li></ul></div><div class="ack" id="ACK-d93e5249-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i63">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67650" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67650" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by NIH Grants GM58888 and CA107331 to D.J.W. and Grant GM070855 to A.D.M., Jr. The training of M.C.C. is supported by NIH T32 Grant CA154274. The Advanced Photon Source (APS) was used in this research, and it is supported, in part, by a grant from the National Institute of General Medical Sciences (Grant P41 GM103403) in the National Institutes of Health and by a U.S. Department of Energy (DOE) contract (Grant DE-AC02-06CH11357). In the DOE, the APS is part of the Office of Science Users Facility operated by the U.S. Office of Science at the Argonne National Laboratory. The U.S. Department of Energy, Office of Science, and Office of Basic Energy Sciences supports Stanford Synchrotron Radiation Light (SSRL) synchrotron source as part of the SLAC National Accelerator Laboratory (Grant DE-AC02-76SF00515). Additional support is provided to the SSRL Structural Molecular Biology Program by the DOE Office of Biological and Environmental Research and by the National Institutes of Health, National Institute of General Medical Sciences (including Grant P41GM103393). The NMR spectrometers used in these studies were purchased, in part, with funds from shared instrumentation grants from the National Institutes of Health [Grants S10 RR10441, S10 RR15741, S10 RR16812, and S10 RR23447 (D.J.W.)] and from the National Science Foundation (Grant DBI 1005795 to D.J.W.). The authors also thank Jace W. Jones and the University of Maryland School of Pharmacy Mass Spectrometry Centre for their help with HPLC-MS.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i64" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i64"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i65" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i65"> Abbreviations Used</h2><tr><td class="NLM_term">MM</td><td class="NLM_def"><p class="first last">malignant melanoma</p></td></tr><tr><td class="NLM_term">SBiX</td><td class="NLM_def"><p class="first last">S100B inhibitor with X being a number</p></td></tr><tr><td class="NLM_term"><sup>Ca</sup>S100B</td><td class="NLM_def"><p class="first last">calcium-bound S100B</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">SILCS</td><td class="NLM_def"><p class="first last">site identification by ligand competitive saturation</p></td></tr><tr><td class="NLM_term">FPCA</td><td class="NLM_def"><p class="first last">fluorescence polarization competition assay</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73186" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73186" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 63 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Gogas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abali, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ascierto, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demidov, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pehamberger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schachter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggermont, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauschild, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espinosa, E.</span><span> </span><span class="NLM_article-title">Who Benefits Most From Adjuvant Interferon Treatment for Melanoma?</span> <span class="citation_source-journal">Am. J. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">54</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1097/MJT.0b013e31829e883d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1097%2FMJT.0b013e31829e883d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24176884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC2c7hsV2iuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=54-60&author=H.+Gogasauthor=H.+Abaliauthor=P.+A.+Asciertoauthor=L.+Demidovauthor=H.+Pehambergerauthor=C.+Robertauthor=J.+Schachterauthor=A.+M.+Eggermontauthor=A.+Hauschildauthor=E.+Espinosa&title=Who+Benefits+Most+From+Adjuvant+Interferon+Treatment+for+Melanoma%3F&doi=10.1097%2FMJT.0b013e31829e883d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Who benefits most from adjuvant interferon treatment for melanoma?</span></div><div class="casAuthors">Gogas Helen; Abali Huseyin; Ascierto Paolo A; Demidov Lev; Pehamberger Hubert; Robert Caroline; Schachter Jacob; Eggermont Alexander M M; Hauschild Axel; Espinosa Enrique</div><div class="citationInfo"><span class="NLM_cas:title">American journal of therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Metastatic melanoma has a poor prognosis; the median survival for patients with stage IV melanoma ranges from 8 to 18 months after diagnosis.  Interferon-α provides significant improvement in disease-free survival at the cost of poor tolerability.  Identifying patients who benefit the most may improve the cost:benefit ratio.  In addition, no data exist for the role of adjuvant therapy in noncutaneous melanoma.  Molecular profiles may help to identify patients who benefit the most from adjuvant interferon therapy.  In this review, the American Joint Commission on Cancer 2009 staging criteria and emerging biomarker data to guide adjuvant treatment decisions will be discussed.  Several criteria to guide selection of patients are discussed in detail.  These include Breslow thickness, number of positive lymph nodes, whether or not the primary lesion has ulcerated, immunologic markers, and cytokine profiles.  Substantial progress has been made in deciding which patients benefit from interferon-α adjuvant therapy.  Interferon-α is the only agent currently approved for the adjuvant treatment of this deadly disease, despite its side effect profile.  More effective drugs with better tolerability are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-C19TuUAlt2Zyxfu64m4ffW6udTcc2eYjDKMGod0_MLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7hsV2iuw%253D%253D&md5=36ae877eaa1be5340ed4296d5f2a0e37</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1097%2FMJT.0b013e31829e883d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMJT.0b013e31829e883d%26sid%3Dliteratum%253Aachs%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DAbali%26aufirst%3DH.%26aulast%3DAscierto%26aufirst%3DP.%2BA.%26aulast%3DDemidov%26aufirst%3DL.%26aulast%3DPehamberger%26aufirst%3DH.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DSchachter%26aufirst%3DJ.%26aulast%3DEggermont%26aufirst%3DA.%2BM.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DEspinosa%26aufirst%3DE.%26atitle%3DWho%2520Benefits%2520Most%2520From%2520Adjuvant%2520Interferon%2520Treatment%2520for%2520Melanoma%253F%26jtitle%3DAm.%2520J.%2520Ther.%26date%3D2015%26volume%3D22%26spage%3D54%26epage%3D60%26doi%3D10.1097%2FMJT.0b013e31829e883d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Jang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkins, M. B.</span><span> </span><span class="NLM_article-title">Which drug, and when, for patients with BRAF-mutant melanoma?</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">e60</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1016/S1470-2045(12)70539-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2FS1470-2045%2812%2970539-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=e60-69&author=S.+Jangauthor=M.+B.+Atkins&title=Which+drug%2C+and+when%2C+for+patients+with+BRAF-mutant+melanoma%3F&doi=10.1016%2FS1470-2045%2812%2970539-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970539-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970539-9%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DS.%26aulast%3DAtkins%26aufirst%3DM.%2BB.%26atitle%3DWhich%2520drug%252C%2520and%2520when%252C%2520for%2520patients%2520with%2520BRAF-mutant%2520melanoma%253F%26jtitle%3DLancet%2520Oncol.%26date%3D2013%26volume%3D14%26spage%3De60%26epage%3D69%26doi%3D10.1016%2FS1470-2045%2812%2970539-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Topalian, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodi, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahmer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gettinger, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDermott, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powderly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvajal, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkins, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leming, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spigel, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonia, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drake, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardoll, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharfman, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anders, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taube, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMiller, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korman, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jure-Kunkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollia, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigginton, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sznol, M.</span><span> </span><span class="NLM_article-title">Safety, activity, and immune correlates of anti-PD-1 antibody in cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">366</span><span class="NLM_x">, </span> <span class="NLM_fpage">2443</span><span class="NLM_x">–</span> <span class="NLM_lpage">2454</span><span class="refDoi"> DOI: 10.1056/NEJMoa1200690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1056%2FNEJMoa1200690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=22658127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2rs7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2012&pages=2443-2454&author=S.+L.+Topalianauthor=F.+S.+Hodiauthor=J.+R.+Brahmerauthor=S.+N.+Gettingerauthor=D.+C.+Smithauthor=D.+F.+McDermottauthor=J.+D.+Powderlyauthor=R.+D.+Carvajalauthor=J.+A.+Sosmanauthor=M.+B.+Atkinsauthor=P.+D.+Lemingauthor=D.+R.+Spigelauthor=S.+J.+Antoniaauthor=L.+Hornauthor=C.+G.+Drakeauthor=D.+M.+Pardollauthor=L.+Chenauthor=W.+H.+Sharfmanauthor=R.+A.+Andersauthor=J.+M.+Taubeauthor=T.+L.+McMillerauthor=H.+Xuauthor=A.+J.+Kormanauthor=M.+Jure-Kunkelauthor=S.+Agrawalauthor=D.+McDonaldauthor=G.+D.+Kolliaauthor=A.+Guptaauthor=J.+M.+Wiggintonauthor=M.+Sznol&title=Safety%2C+activity%2C+and+immune+correlates+of+anti-PD-1+antibody+in+cancer&doi=10.1056%2FNEJMoa1200690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Safety, activity, and immune correlates of anti-PD-1 antibody in cancer</span></div><div class="casAuthors">Topalian, Suzanne L.; Hodi, F. Stephen; Brahmer, Julie R.; Gettinger, Scott N.; Smith, David C.; McDermott, David F.; Powderly, John D.; Carvajal, Richard D.; Sosman, Jeffrey A.; Atkins, Michael B.; Leming, Philip D.; Spigel, David R.; Antonia, Scott J.; Horn, Leora; Drake, Charles G.; Pardoll, Drew M.; Chen, Lieping; Sharfman, William H.; Anders, Robert A.; Taube, Janis M.; McMiller, Tracee L.; Xu, Haiying; Korman, Alan J.; Jure-Kunkel, Maria; Agrawal, Shruti; McDonald, Daniel; Kollia, Georgia D.; Gupta, Ashok; Wigginton, Jon M.; Sznol, Mario</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">366</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">2443-2454</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Blockade of programmed death 1 (PD-1), an inhibitory receptor expressed by T cells, can overcome immune resistance.  We assessed the antitumor activity and safety of BMS-936558, an antibody that specifically blocks PD-1.  Methods: We enrolled patients with advanced melanoma, non-small-cell lung cancer, castration-resistant prostate cancer, or renal-cell or colorectal cancer to receive anti-PD-1 antibody at a dose of 0.1 to 10.0 mg per kg of body wt. every 2 wk.  Response was assessed after each 8-wk treatment cycle.  Patients received up to 12 cycles until disease progression or a complete response occurred.  Results: A total of 296 patients received treatment through Feb. 24, 2012.  Grade 3 or 4 drug-related adverse events occurred in 14% of patients; there were three deaths from pulmonary toxicity.  No max. tolerated dose was defined.  Adverse events consistent with immune-related causes were obsd.  Among 236 patients in whom response could be evaluated, objective responses (complete or partial responses) were obsd. in those with non-small-cell lung cancer, melanoma, or renal-cell cancer.  Cumulative response rates (all doses) were 18% among patients with non-small-cell lung cancer (14 of 76 patients), 28% among patients with melanoma (26 of 94 patients), and 27% among patients with renal-cell cancer (9 of 33 patients).  Responses were durable; 20 of 31 responses lasted 1 yr or more in patients with 1 yr or more of follow-up.  To assess the role of intratumoral PD-1 ligand (PD-L1) expression in the modulation of the PD-1-PD-L1 pathway, immunohistochem. anal. was performed on pretreatment tumor specimens obtained from 42 patients.  Of 17 patients with PD-L1-neg. tumors, none had an objective response; 9 of 25 patients (36%) with PD-L1-pos. tumors had an objective response (P = 0.006).  Conclusions: Anti-PD-1 antibody produced objective responses in approx. one in four to one in five patients with non-small-cell lung cancer, melanoma, or renal-cell cancer; the adverse-event profile does not appear to preclude its use.  Preliminary data suggest a relationship between PD-L1 expression on tumor cells and objective response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLOPvHybmlvbVg90H21EOLACvtfcHk0li5L-LWkQS6-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2rs7fN&md5=7c13f228ef92b6089982fc878d3a394b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1200690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1200690%26sid%3Dliteratum%253Aachs%26aulast%3DTopalian%26aufirst%3DS.%2BL.%26aulast%3DHodi%26aufirst%3DF.%2BS.%26aulast%3DBrahmer%26aufirst%3DJ.%2BR.%26aulast%3DGettinger%26aufirst%3DS.%2BN.%26aulast%3DSmith%26aufirst%3DD.%2BC.%26aulast%3DMcDermott%26aufirst%3DD.%2BF.%26aulast%3DPowderly%26aufirst%3DJ.%2BD.%26aulast%3DCarvajal%26aufirst%3DR.%2BD.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DAtkins%26aufirst%3DM.%2BB.%26aulast%3DLeming%26aufirst%3DP.%2BD.%26aulast%3DSpigel%26aufirst%3DD.%2BR.%26aulast%3DAntonia%26aufirst%3DS.%2BJ.%26aulast%3DHorn%26aufirst%3DL.%26aulast%3DDrake%26aufirst%3DC.%2BG.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSharfman%26aufirst%3DW.%2BH.%26aulast%3DAnders%26aufirst%3DR.%2BA.%26aulast%3DTaube%26aufirst%3DJ.%2BM.%26aulast%3DMcMiller%26aufirst%3DT.%2BL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DKorman%26aufirst%3DA.%2BJ.%26aulast%3DJure-Kunkel%26aufirst%3DM.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DMcDonald%26aufirst%3DD.%26aulast%3DKollia%26aufirst%3DG.%2BD.%26aulast%3DGupta%26aufirst%3DA.%26aulast%3DWigginton%26aufirst%3DJ.%2BM.%26aulast%3DSznol%26aufirst%3DM.%26atitle%3DSafety%252C%2520activity%252C%2520and%2520immune%2520correlates%2520of%2520anti-PD-1%2520antibody%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D366%26spage%3D2443%26epage%3D2454%26doi%3D10.1056%2FNEJMoa1200690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Marcus, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crocker, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelton, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maynard, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delman, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, M.</span><span> </span><span class="NLM_article-title">Prognostic Factors for Overall Survival After Radiosurgery for Brain Metastases From Melanoma</span> <span class="citation_source-journal">Am. J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span><span class="refDoi"> DOI: 10.1097/COC.0b013e318280d7be</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1097%2FCOC.0b013e318280d7be" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23428955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC3svgt1ehsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2014&pages=580-584&author=D.+M.+Marcusauthor=M.+Loweauthor=M.+K.+Khanauthor=D.+H.+Lawsonauthor=I.+R.+Crockerauthor=J.+W.+Sheltonauthor=A.+Meltonauthor=N.+Maynardauthor=K.+A.+Delmanauthor=G.+W.+Carlsonauthor=M.+Rizzo&title=Prognostic+Factors+for+Overall+Survival+After+Radiosurgery+for+Brain+Metastases+From+Melanoma&doi=10.1097%2FCOC.0b013e318280d7be"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic factors for overall survival after radiosurgery for brain metastases from melanoma</span></div><div class="casAuthors">Marcus David M; Lowe Michael; Khan Mohammad K; Lawson David H; Crocker Ian R; Shelton Joseph W; Melton Alisa; Maynard Necia; Delman Keith A; Carlson Grant W; Rizzo Monica</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">580-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Brain metastases (BM) cause significant morbidity and mortality in patients with melanoma.  We aimed to identify prognostic factors for overall survival (OS) in patients undergoing stereotactic radiosurgery (SRS) for BM from melanoma.  METHODS:  We identified 135 patients treated with SRS at Emory University between 1998 and 2010 for BM from melanoma.  We recorded patient age, number and size of all BM, Karnofsky Performance Status (KPS), presence of extracranial metastases, serum lactate dehydrogenase (LDH), use of whole-brain radiation therapy (WBRT), use of temozolomide, and surgical resection of BM.  We used the Kaplan-Meier method to calculate OS, and we compared time-to-event data with the log-rank test.  We performed Cox multivariate analysis to identify factors independently associated with OS.  RESULTS:  Median OS for all patients was 6.9 months.  Patients with KPS ≥ 90, 70 to 80, and <70 had median OS of 10.4, 6.1, and 4.5 months, respectively (P=0.02).  Patients with LDH<240 had median OS of 7.8 months versus 3.5 months for LDH ≥ 240 (P=0.01).  Patients receiving WBRT had median OS of 7.3 months versus 6.5 months for patients not receiving WBRT (P=0.05).  KPS and LDH (but not WBRT) were significantly associated with OS on multivariate analysis.  CONCLUSIONS:  In addition to previously identified prognostic factors for OS in patients with BM from melanoma, serum LDH is independently associated with OS.  If this finding is confirmed in a prospective manner, the serum LDH level should be included in future prognostic algorithms for patients with melanoma and BM who are to receive SRS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvRw4O17JYEy86Js-BCLiifW6udTcc2eYr0uA2PhJ5H7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svgt1ehsg%253D%253D&md5=223c0630464123049e9164ff79e4b1e8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1097%2FCOC.0b013e318280d7be&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCOC.0b013e318280d7be%26sid%3Dliteratum%253Aachs%26aulast%3DMarcus%26aufirst%3DD.%2BM.%26aulast%3DLowe%26aufirst%3DM.%26aulast%3DKhan%26aufirst%3DM.%2BK.%26aulast%3DLawson%26aufirst%3DD.%2BH.%26aulast%3DCrocker%26aufirst%3DI.%2BR.%26aulast%3DShelton%26aufirst%3DJ.%2BW.%26aulast%3DMelton%26aufirst%3DA.%26aulast%3DMaynard%26aufirst%3DN.%26aulast%3DDelman%26aufirst%3DK.%2BA.%26aulast%3DCarlson%26aufirst%3DG.%2BW.%26aulast%3DRizzo%26aufirst%3DM.%26atitle%3DPrognostic%2520Factors%2520for%2520Overall%2520Survival%2520After%2520Radiosurgery%2520for%2520Brain%2520Metastases%2520From%2520Melanoma%26jtitle%3DAm.%2520J.%2520Clin.%2520Oncol.%26date%3D2014%26volume%3D37%26spage%3D580%26epage%3D584%26doi%3D10.1097%2FCOC.0b013e318280d7be" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Espinosa, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grob, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dummer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutkowski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gogas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kefford, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggermont, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin Algarra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauschild, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schadendorf, D.</span><span> </span><span class="NLM_article-title">Treatment Algorithms in Stage IV Melanoma</span> <span class="citation_source-journal">Am. J. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1097/MJT.0b013e31829e885c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1097%2FMJT.0b013e31829e885c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24413374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC2czksFWjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=61-67&author=E.+Espinosaauthor=J.+J.+Grobauthor=R.+Dummerauthor=P.+Rutkowskiauthor=C.+Robertauthor=H.+Gogasauthor=R.+Keffordauthor=A.+M.+Eggermontauthor=S.+Martin+Algarraauthor=A.+Hauschildauthor=D.+Schadendorf&title=Treatment+Algorithms+in+Stage+IV+Melanoma&doi=10.1097%2FMJT.0b013e31829e885c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment algorithms in stage IV melanoma</span></div><div class="casAuthors">Espinosa Enrique; Grob Jean-Jacques; Dummer Reinhard; Rutkowski Piotr; Robert Caroline; Gogas Helen; Kefford Richard; Eggermont Alexander M M; Martin Algarra Salvador; Hauschild Axel; Schadendorf Dirk</div><div class="citationInfo"><span class="NLM_cas:title">American journal of therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The molecular classification of melanoma and the advent of new drugs are changing the paradigm of therapy for advanced melanoma.  A review of the recent key studies was performed, followed by a discussion in an expert forum.  The aim of this review was to generate a therapeutic algorithm for stage IV melanoma.  Tumor genotyping for BRAF and/or KIT should be performed before selection of therapy.  For most BRAF-mutated melanoma patients and particularly those with a high tumor load, vemurafenib or other BRAF inhibitors such as dabrafenib are the treatment of choice.  KIT inhibitors can be effective in KIT-mutant tumors, especially in those patients with mutations at exons 11 and 13.  Ipilimumab is a good option for patients with nontargetable or nondetected mutations and those who progress under therapy with vemurafenib or a KIT inhibitor.  There is still a role for conventional chemotherapy either as first-line treatment in BRAF wild-type patients or as salvage therapy in second or third line, or after other treatment modalities.  Participation in clinical trials is strongly encouraged, either in first or in subsequent lines.  New therapeutic options for advanced melanoma are guided by tumor genotyping.  The current therapeutic algorithm includes kinase inhibitors, anti-CTLA4 therapy, immunotherapy, and chemotherapy, depending on the tumor genotype and response to previous treatments.  Participation in clinical trials should always be encouraged because the treatment goal is long-term survival and potential cure in a subset of patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3HXRikISteBFwe7cAcrfPfW6udTcc2eZ14hJPbuXn-rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czksFWjuw%253D%253D&md5=8bd49605b84ed3c186be1944c75661c3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1097%2FMJT.0b013e31829e885c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMJT.0b013e31829e885c%26sid%3Dliteratum%253Aachs%26aulast%3DEspinosa%26aufirst%3DE.%26aulast%3DGrob%26aufirst%3DJ.%2BJ.%26aulast%3DDummer%26aufirst%3DR.%26aulast%3DRutkowski%26aufirst%3DP.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DKefford%26aufirst%3DR.%26aulast%3DEggermont%26aufirst%3DA.%2BM.%26aulast%3DMartin%2BAlgarra%26aufirst%3DS.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DSchadendorf%26aufirst%3DD.%26atitle%3DTreatment%2520Algorithms%2520in%2520Stage%2520IV%2520Melanoma%26jtitle%3DAm.%2520J.%2520Ther.%26date%3D2015%26volume%3D22%26spage%3D61%26epage%3D67%26doi%3D10.1097%2FMJT.0b013e31829e885c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Lee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahasranaman, S.</span><span> </span><span class="NLM_article-title">Immune checkpoint inhibitors: An introduction to the next generation cancer immunotherapy</span> <span class="citation_source-journal">J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1002/jcph.591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fjcph.591" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Leeauthor=M.+Guptaauthor=S.+Sahasranaman&title=Immune+checkpoint+inhibitors%3A+An+introduction+to+the+next+generation+cancer+immunotherapy&doi=10.1002%2Fjcph.591"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fjcph.591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcph.591%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DL.%26aulast%3DGupta%26aufirst%3DM.%26aulast%3DSahasranaman%26aufirst%3DS.%26atitle%3DImmune%2520checkpoint%2520inhibitors%253A%2520An%2520introduction%2520to%2520the%2520next%2520generation%2520cancer%2520immunotherapy%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2015%26doi%3D10.1002%2Fjcph.591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Mahoney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rennert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, G. J.</span><span> </span><span class="NLM_article-title">Combination cancer immunotherapy and new immunomodulatory targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span><span class="refDoi"> DOI: 10.1038/nrd4591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1038%2Fnrd4591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=26228759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KiurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=561-584&author=K.+M.+Mahoneyauthor=P.+D.+Rennertauthor=G.+J.+Freeman&title=Combination+cancer+immunotherapy+and+new+immunomodulatory+targets&doi=10.1038%2Fnrd4591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Combination cancer immunotherapy and new immunomodulatory targets</span></div><div class="casAuthors">Mahoney, Kathleen M.; Rennert, Paul D.; Freeman, Gordon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">561-584</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting immune checkpoints such as programmed cell death protein 1 (PD1), programmed cell death 1 ligand 1 (PDL1) and cytotoxic T lymphocyte antigen 4 (CTLA4) has achieved noteworthy benefit in multiple cancers by blocking immunoinhibitory signals and enabling patients to produce an effective antitumor response.  Inhibitors of CTLA4, PD1 or PDL1 administered as single agents have resulted in durable tumor regression in some patients, and combinations of PD1 and CTLA4 inhibitors may enhance antitumor benefit.  Numerous addnl. immunomodulatory pathways as well as inhibitory factors expressed or secreted by myeloid and stromal cells in the tumor microenvironment are potential targets for synergizing with immune checkpoint blockade.  Given the breadth of potential targets in the immune system, crit. questions to address include which combinations should move forward in development and which patients will benefit from these treatments.  This Review discusses the leading drug targets that are expressed on tumor cells and in the tumor microenvironment that allow enhancement of the antitumor immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwi7ChUpfjArVg90H21EOLACvtfcHk0lhwDc6PUC1brw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KiurrJ&md5=b0b598b2a7c62794b4f03c4cea196a97</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd4591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4591%26sid%3Dliteratum%253Aachs%26aulast%3DMahoney%26aufirst%3DK.%2BM.%26aulast%3DRennert%26aufirst%3DP.%2BD.%26aulast%3DFreeman%26aufirst%3DG.%2BJ.%26atitle%3DCombination%2520cancer%2520immunotherapy%2520and%2520new%2520immunomodulatory%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D561%26epage%3D584%26doi%3D10.1038%2Fnrd4591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Bresnick, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, D. B.</span><span> </span><span class="NLM_article-title">S100 proteins in cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span><span class="refDoi"> DOI: 10.1038/nrc3893</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1038%2Fnrc3893" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=25614008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=96-109&author=A.+R.+Bresnickauthor=D.+J.+Weberauthor=D.+B.+Zimmer&title=S100+proteins+in+cancer&doi=10.1038%2Fnrc3893"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">S100 proteins in cancer</span></div><div class="casAuthors">Bresnick, Anne R.; Weber, David J.; Zimmer, Danna B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">96-109</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In humans, the S100 protein family is composed of 21 members that exhibit a high degree of structural similarity, but are not functionally interchangeable.  This family of proteins modulates cellular responses by functioning both as intracellular Ca2+ sensors and as extracellular factors.  Dysregulated expression of multiple members of the S100 family is a common feature of human cancers, with each type of cancer showing a unique S100 protein profile or signature.  Emerging in vivo evidence indicates that the biol. of most S100 proteins is complex and multifactorial, and that these proteins actively contribute to tumorigenic processes such as cell proliferation, metastasis, angiogenesis and immune evasion.  Drug discovery efforts have identified leads for inhibiting several S100 family members, and two of the identified inhibitors have progressed to clin. trials in patients with cancer.  This Review highlights new findings regarding the role of S100 family members in cancer diagnosis and treatment, the contribution of S100 signalling to tumor biol., and the discovery and development of S100 inhibitors for treating cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTg0j8SyoeNrVg90H21EOLACvtfcHk0lhwDc6PUC1brw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrg%253D&md5=a78954b14527eea6fef2fd94ec51c6d1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc3893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3893%26sid%3Dliteratum%253Aachs%26aulast%3DBresnick%26aufirst%3DA.%2BR.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26aulast%3DZimmer%26aufirst%3DD.%2BB.%26atitle%3DS100%2520proteins%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D96%26epage%3D109%26doi%3D10.1038%2Fnrc3893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">El Hajj, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Journe, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laios, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sales, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galibert, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Kempen, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spatz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badran, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsimont, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awada, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghanem, G.</span><span> </span><span class="NLM_article-title">Tyrosinase-related protein 1 mRNA expression in lymph node metastases predicts overall survival in high-risk melanoma patients</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">1641</span><span class="NLM_x">–</span> <span class="NLM_lpage">1647</span><span class="refDoi"> DOI: 10.1038/bjc.2013.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1038%2Fbjc.2013.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23519055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVOru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2013&pages=1641-1647&author=P.+El+Hajjauthor=F.+Journeauthor=M.+Wiedigauthor=I.+Laiosauthor=F.+Salesauthor=M.+D.+Galibertauthor=L.+C.+Van+Kempenauthor=A.+Spatzauthor=B.+Badranauthor=D.+Larsimontauthor=A.+Awadaauthor=G.+Ghanem&title=Tyrosinase-related+protein+1+mRNA+expression+in+lymph+node+metastases+predicts+overall+survival+in+high-risk+melanoma+patients&doi=10.1038%2Fbjc.2013.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosinase-related protein 1 mRNA expression in lymph node metastases predicts overall survival in high-risk melanoma patients</span></div><div class="casAuthors">El Hajj, P.; Journe, F.; Wiedig, M.; Laios, I.; Sales, F.; Galibert, M.-D.; Van Kempen, L. C.; Spatz, A.; Badran, B.; Larsimont, D.; Awada, A.; Ghanem, G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1641-1647</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: Clin. outcome of high-risk melanoma patients is not reliably predicted from histopathol. analyses of primary tumors and is often adjusted during disease progression.  Our study aimed at extending our previous findings in skin metastases to evaluate the prognostic value of tyrosinase-related protein 1 (TYRP1) in lymph node metastases of stages III and IV melanoma patients.  Methods: TYRP1 mRNA expression in 104 lymph node metastases was quantified by real-time PCR and normalized to S100 calcium-binding protein B (S100B) mRNA expression to correct for tumor load.  TYRP1/S100B ratios were calcd. and median was used as cutoff value.  TYRP1/S100B mRNA values were correlated to clin. follow-up and histopathol. characteristics of the primary lesion.  Results: A high TYRP1/S100B mRNA ratio significantly correlated with reduced disease-free (DFS) and overall survival (OS; Cox regression anal., P=0.005 and 0.01, resp.), increased Breslow thickness (Spearman's rho test, P<0.001) and the presence of ulceration (Mann-Whitney test, P=0.02) of the primaries.  Moreover, high TYRP1/S100B was of better prognostic value (lower P-value) for OS than Breslow thickness and ulceration.  Finally, it was well conserved during disease progression with respect to high/low TYRP1 groups.  Conclusion: High TYRP1/S100B mRNA expression in lymph node metastases from melanoma patients is assocd. with unfavorable clin. outcome.  Its evaluation in lymph node metastases may refine initial prognosis for metastatic patients, may define prognosis for those with unknown or non-evaluable primary lesions and may allow different management of the two groups of patients.  British Journal of Cancer (2013) 108, 1641-1647; doi:10.1038/bjc.2013.115 www.bjcancer.com; published online 21 March 2013.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-Am-0FV-6N7Vg90H21EOLACvtfcHk0lgRCh2CA2J7BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVOru7s%253D&md5=29bd07ddea92f9a72cc4055e4a223b07</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2013.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2013.115%26sid%3Dliteratum%253Aachs%26aulast%3DEl%2BHajj%26aufirst%3DP.%26aulast%3DJourne%26aufirst%3DF.%26aulast%3DWiedig%26aufirst%3DM.%26aulast%3DLaios%26aufirst%3DI.%26aulast%3DSales%26aufirst%3DF.%26aulast%3DGalibert%26aufirst%3DM.%2BD.%26aulast%3DVan%2BKempen%26aufirst%3DL.%2BC.%26aulast%3DSpatz%26aufirst%3DA.%26aulast%3DBadran%26aufirst%3DB.%26aulast%3DLarsimont%26aufirst%3DD.%26aulast%3DAwada%26aufirst%3DA.%26aulast%3DGhanem%26aufirst%3DG.%26atitle%3DTyrosinase-related%2520protein%25201%2520mRNA%2520expression%2520in%2520lymph%2520node%2520metastases%2520predicts%2520overall%2520survival%2520in%2520high-risk%2520melanoma%2520patients%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2013%26volume%3D108%26spage%3D1641%26epage%3D1647%26doi%3D10.1038%2Fbjc.2013.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Henry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabre, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guiraud, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastide, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro-Peray, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavabre-Bertrand, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meunier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoebner, P. E.</span><span> </span><span class="NLM_article-title">Clinical use of p-proteasome in discriminating metastatic melanoma patients: comparative study with LDH, MIA and S100B protein</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1002/ijc.27991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fijc.27991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23238767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFaksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2013&pages=142-148&author=L.+Henryauthor=C.+Fabreauthor=I.+Guiraudauthor=S.+Bastideauthor=P.+Fabbro-Perayauthor=J.+Martinezauthor=T.+Lavabre-Bertrandauthor=L.+Meunierauthor=P.+E.+Stoebner&title=Clinical+use+of+p-proteasome+in+discriminating+metastatic+melanoma+patients%3A+comparative+study+with+LDH%2C+MIA+and+S100B+protein&doi=10.1002%2Fijc.27991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical use of p-proteasome in discriminating metastatic melanoma patients: Comparative study with LDH, MIA and S100B protein</span></div><div class="casAuthors">Henry, Laurent; Fabre, Cecile; Guiraud, Isabelle; Bastide, Sophie; Fabbro-Peray, Pascale; Martinez, Jean; Lavabre-Bertrand, Thierry; Meunier, Laurent; Stoebner, Pierre-Emmanuel</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">142-148</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Plasmatic proteasome (p-proteasome) has recently been described as a new marker for metastatic melanoma.  The objective of this study was to compare the diagnostic and prognostic values of p-proteasome with three other melanoma serol. markers: S100B protein, melanoma inhibitory activity protein (MIA) and lactate dehydrogenase (LDH) in the plasma of 121 stage I-IV melanoma patients.  Lab. analyses were performed by standardized ELISA (p-proteasome, MIA), immunoluminometric assay (S100B) and colorimetry (LDH).  We found that all markers were relevant for discriminating metastatic from nonmetastatic patients but p-proteasome displayed the highest diagnostic accuracy.  P-proteasome and S100B were the most sensitive (58.1%) and p-proteasome and MIA the most specific (98.7 and 100%) in detecting metastatic disease.  P-proteasome and S100B had the highest area under receiver operating characteristics curve, 0.811 (95% CI: 0.725-0.897) and 0.822 (95% CI: 0.738-0.906), resp.  These two markers were the best in detecting patients with lymph node metastases.  S100B, MIA and LDH diagnostic accuracy was increased when these markers were combined with p-proteasome.  As shown with univariate anal., shorter progression-free and overall survival rates were significantly assocd. with elevated plasma levels of each markers.  The multivariate Cox regression anal. identified p-proteasome as the only independent predictor of a poorer progression-free survival (p = 0.030).  In conclusion, this comparative study established that p-proteasome quantification in combination with other melanoma biomarkers is an attractive approach for the biol. follow-up of melanoma patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYSvRRY7hWyLVg90H21EOLACvtfcHk0lgRCh2CA2J7BQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFaksQ%253D%253D&md5=707afce35f8d0a631156812a485264e9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fijc.27991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27991%26sid%3Dliteratum%253Aachs%26aulast%3DHenry%26aufirst%3DL.%26aulast%3DFabre%26aufirst%3DC.%26aulast%3DGuiraud%26aufirst%3DI.%26aulast%3DBastide%26aufirst%3DS.%26aulast%3DFabbro-Peray%26aufirst%3DP.%26aulast%3DMartinez%26aufirst%3DJ.%26aulast%3DLavabre-Bertrand%26aufirst%3DT.%26aulast%3DMeunier%26aufirst%3DL.%26aulast%3DStoebner%26aufirst%3DP.%2BE.%26atitle%3DClinical%2520use%2520of%2520p-proteasome%2520in%2520discriminating%2520metastatic%2520melanoma%2520patients%253A%2520comparative%2520study%2520with%2520LDH%252C%2520MIA%2520and%2520S100B%2520protein%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2013%26volume%3D133%26spage%3D142%26epage%3D148%26doi%3D10.1002%2Fijc.27991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Wieder, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tekin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum-Krumme, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klode, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altenbernd, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bockisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagarajah, J.</span><span> </span><span class="NLM_article-title">18FDG-PET to assess recurrence and long term survival in patients with malignant melanoma</span> <span class="citation_source-journal">Nuklearmedizin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">198</span><span class="NLM_x">–</span> <span class="NLM_lpage">203</span><span class="refDoi"> DOI: 10.3413/Nukmed-0584-13-05</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.3413%2FNukmed-0584-13-05" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23969722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC3sbitVSlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=198-203&author=H.+A.+Wiederauthor=G.+Tekinauthor=S.+Rosenbaum-Krummeauthor=J.+Klodeauthor=J.+Altenberndauthor=A.+Bockischauthor=J.+Nagarajah&title=18FDG-PET+to+assess+recurrence+and+long+term+survival+in+patients+with+malignant+melanoma&doi=10.3413%2FNukmed-0584-13-05"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">18FDG-PET to assess recurrence and long term survival in patients with malignant melanoma</span></div><div class="casAuthors">Wieder H A; Tekin G; Rosenbaum-Krumme S; Klode J; Altenbernd J; Bockisch A; Nagarajah J</div><div class="citationInfo"><span class="NLM_cas:title">Nuklearmedizin. Nuclear medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">198-203</span>
        ISSN:<span class="NLM_cas:issn">0029-5566</span>.
    </div><div class="casAbstract">AIM:  To assess the diagnostic and prognostic value of FDG-PET/CT in the follow-up of malignant melanoma in comparison to the serum protein S100B.  PATIENTS AND METHODS:  A total of ninety patients with either low-risk or high-risk malignant melanoma, respectively, were included in this study.  The follow-up of the patients was pursuant with the guidelines of the German Dermatological Association.  The diagnostic accuracy and diagnostic power were determined for PET/CT and for the serum protein S100B.  RESULTS:  In 28 of the 90 patients PET/CT was positive in the follow up, 47 patients had an elevated Serum S100B level.  Sensitivity, specificity, PPV and NPV of PET/CT for the total groups of patients were 87%, 93%, 87% and 93%.  The corresponding values for the serum protein S100B were 65%, 52%, 43% and 74%, respectively.  PET/CT positive patients showed a significantly (p < 0.001) higher risk of melanoma associated death compared to patients with PET/CT negative findings.  No statistical significance could be found in the 5 year survival rate between the S100B positive and S100B negative patients.  CONCLUSION:  PET/CT is suitable to confirm or exclude recurrences and can be used to assess the prognosis in melanoma patients.  The diagnostic accuracy and the prognostic power is much higher compared to the serum protein S100B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhZsd1Ka8S_a91pGlMMoT5fW6udTcc2ebKDzOoB9Zqfbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbitVSlsw%253D%253D&md5=db1d11c67d035946cde7e7e53a4fe813</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3413%2FNukmed-0584-13-05&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3413%252FNukmed-0584-13-05%26sid%3Dliteratum%253Aachs%26aulast%3DWieder%26aufirst%3DH.%2BA.%26aulast%3DTekin%26aufirst%3DG.%26aulast%3DRosenbaum-Krumme%26aufirst%3DS.%26aulast%3DKlode%26aufirst%3DJ.%26aulast%3DAltenbernd%26aufirst%3DJ.%26aulast%3DBockisch%26aufirst%3DA.%26aulast%3DNagarajah%26aufirst%3DJ.%26atitle%3D18FDG-PET%2520to%2520assess%2520recurrence%2520and%2520long%2520term%2520survival%2520in%2520patients%2520with%2520malignant%2520melanoma%26jtitle%3DNuklearmedizin%26date%3D2013%26volume%3D52%26spage%3D198%26epage%3D203%26doi%3D10.3413%2FNukmed-0584-13-05" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Schiltz, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dillman, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korse, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cubellis, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Gast, G. C.</span><span> </span><span class="NLM_article-title">Lack of elevation of serum S100B in patients with metastatic melanoma as a predictor of outcome after induction with an autologous vaccine of proliferating tumor cells and dendritic cells</span> <span class="citation_source-journal">Cancer Biother.Radiopharm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1089/cbr.2007.0432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1089%2Fcbr.2007.0432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=18454690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsV2msbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2008&pages=214-221&author=P.+M.+Schiltzauthor=R.+O.+Dillmanauthor=C.+M.+Korseauthor=J.+M.+Cubellisauthor=G.+J.+Leeauthor=G.+C.+De+Gast&title=Lack+of+elevation+of+serum+S100B+in+patients+with+metastatic+melanoma+as+a+predictor+of+outcome+after+induction+with+an+autologous+vaccine+of+proliferating+tumor+cells+and+dendritic+cells&doi=10.1089%2Fcbr.2007.0432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Lack of Elevation of Serum S100B in Patients with Metastatic Melanoma as a Predictor of Outcome After Induction with an Autologous Vaccine of Proliferating Tumor Cells and Dendritic Cells</span></div><div class="casAuthors">Schiltz, Patric M.; Dillman, Robert O.; Korse, Catharina M.; Cubellis, James M.; Lee, Gregory J.; De Gast, Gijsbert C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biotherapy & Radiopharmaceuticals</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">214-221</span>CODEN:
                <span class="NLM_cas:coden">CBRAFJ</span>;
        ISSN:<span class="NLM_cas:issn">1084-9785</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Serum levels of S100B and/or lactate dehydrogenase (LDH) are putative tumor markers in melanoma.  Early changes in such markers may correlate with a pos. immune response to vaccine therapy.  In patients with metastatic melanoma, S100B and LDH serum levels were measured at baseline, and 1 wk after 3 weekly s.c. injections of investigational, patient-specific vaccines consisting of autologous dendritic cells loaded with antigens from irradiated proliferating autologous tumor cells, and suspended in granulocyte macrophage colony-stimulating factor.  There was a poor correlation between S100B and LDH levels at baseline (p = 0.324).  Fourteen (14) patients with measurable disease had higher S100B (p = 0.0456) and LDH (p = 0.0013) levels than 31 patients who lacked measurable disease at that time.  Fourteen (14) deceased patients (median survival, 13 mo) had a mean baseline S100B of 0.62 μg/L (95% confidence interval [CI] 0.00-1.66) and LDH of 815 U/L (95% CI 222-1408); 31 surviving patients (median follow-up, 35.4 mo) had mean S100B of 0.07 μg/L (95% CI 0.00-0.23; p = 0.006) and LDH of 442 U/L (95% CI 296-588; p = 0.002).  Elevated baseline levels of LDH and S100B were each predictive of inferior progression-free survival (PFS) and overall survival (OS) (both p <0.0001), but S100B was a better predictor for PFS than LDH.  Changes in LDH between baseline and week 4 were not predictive of survival, but an increase in S100B predicted for inferior OS ( p = 0.039).  Both LDH and S100B are predictive tumor markers in patients with metastatic melanoma.  This is the first study to examine the changes in serum levels of LDH and S100B in response to an autologous tumor-cell vaccine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob3Ob4jCpV37Vg90H21EOLACvtfcHk0lgTYI3H8mf2Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsV2msbk%253D&md5=be20069dce53f3e1bb68d8aa6ed141b5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1089%2Fcbr.2007.0432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fcbr.2007.0432%26sid%3Dliteratum%253Aachs%26aulast%3DSchiltz%26aufirst%3DP.%2BM.%26aulast%3DDillman%26aufirst%3DR.%2BO.%26aulast%3DKorse%26aufirst%3DC.%2BM.%26aulast%3DCubellis%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DG.%2BJ.%26aulast%3DDe%2BGast%26aufirst%3DG.%2BC.%26atitle%3DLack%2520of%2520elevation%2520of%2520serum%2520S100B%2520in%2520patients%2520with%2520metastatic%2520melanoma%2520as%2520a%2520predictor%2520of%2520outcome%2520after%2520induction%2520with%2520an%2520autologous%2520vaccine%2520of%2520proliferating%2520tumor%2520cells%2520and%2520dendritic%2520cells%26jtitle%3DCancer%2520Biother.Radiopharm.%26date%3D2008%26volume%3D23%26spage%3D214%26epage%3D221%26doi%3D10.1089%2Fcbr.2007.0432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">O’Rourke, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanagan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">See, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slater, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purdie, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellem, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, C. W.</span><span> </span><span class="NLM_article-title">Durable complete clinical responses in a phase I/II trial using an autologous melanoma cell/dendritic cell vaccine</span> <span class="citation_source-journal">Cancer Immunol. Immunother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">387</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1007/s00262-003-0375-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1007%2Fs00262-003-0375-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=12682787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BD3s3hslSrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=387-395&author=M.+G.+O%E2%80%99Rourkeauthor=M.+Johnsonauthor=C.+Lanaganauthor=J.+Seeauthor=J.+Yangauthor=J.+R.+Bellauthor=G.+J.+Slaterauthor=B.+M.+Kerrauthor=B.+Croweauthor=D.+M.+Purdieauthor=S.+L.+Elliottauthor=K.+A.+Ellemauthor=C.+W.+Schmidt&title=Durable+complete+clinical+responses+in+a+phase+I%2FII+trial+using+an+autologous+melanoma+cell%2Fdendritic+cell+vaccine&doi=10.1007%2Fs00262-003-0375-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Durable complete clinical responses in a phase I/II trial using an autologous melanoma cell/dendritic cell vaccine</span></div><div class="casAuthors">O'Rourke Michael G E; Johnson Maree; Lanagan Catherine; See Janet; Yang Jie; Bell John R; Slater Greg J; Kerr Beverley M; Crowe Beth; Purdie David M; Elliott Suzanne L; Ellem Kay A O; Schmidt Christopher W</div><div class="citationInfo"><span class="NLM_cas:title">Cancer immunology, immunotherapy : CII</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">387-95</span>
        ISSN:<span class="NLM_cas:issn">0340-7004</span>.
    </div><div class="casAbstract">Advanced metastatic melanoma is incurable by standard treatments, but occasionally responds to immunotherapy.  Recent trials using dendritic cells (DC) as a cellular adjuvant have concentrated on defined peptides as the source of antigens, and rely on foreign proteins as a source of help to generate a cell-mediated immune response.  This approach limits patient accrual, because currently defined, non-mutated epitopes are restricted by a small number of human leucocyte antigens.  It also fails to take advantage of mutated epitopes peculiar to the patient's own tumour, and of CD4+ T lymphocytes as potential effectors of anti-tumour immunity.  We therefore sought to determine whether a fully autologous DC vaccine is feasible, and of therapeutic benefit.  Patients with American Joint Cancer Committee stage IV melanoma were treated with a fully autologous immunotherapy consisting of monocyte-derived DC, matured after culture with irradiated tumour cells.  Of 19 patients enrolled into the trial, sufficient tumour was available to make treatments for 17.  Of these, 12 received a complete priming phase of six cycles of either 0.9x10(6) or 5x10(6) DC/intradermal injection, at 2-weekly intervals.  Where possible, treatment continued with the lower dose at 6-weekly intervals.  The remaining five patients could not complete priming, due to progressive disease.  Three of the 12 patients who completed priming have durable complete responses (average duration 35 months+), three had partial responses, and the remaining six had progressive disease (WHO criteria).  Disease regression was not correlated with dose or with the development of delayed type hypersensitivity responses to intradermal challenge with irradiated, autologous tumour.  However, plasma S-100B levels prior to the commencement of treatment correlated with objective clinical response ( P=0.05) and survival (log rank P<0.001).  The treatment had minimal side-effects and was well tolerated by all patients.  Mature, monocyte-derived DC preparations exposed to appropriate tumour antigen sources can be reliably produced for patients with advanced metastatic melanoma, and in a subset of those patients with lower volume disease their repeated administration results in durable complete responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTtEdEdYQFKPwvupmecNw_lfW6udTcc2ebKDzOoB9Zqfbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3hslSrtA%253D%253D&md5=5df19e634f7c8d9229b1d8f9afdf5135</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00262-003-0375-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00262-003-0375-x%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DM.%2BG.%26aulast%3DJohnson%26aufirst%3DM.%26aulast%3DLanagan%26aufirst%3DC.%26aulast%3DSee%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DJ.%2BR.%26aulast%3DSlater%26aufirst%3DG.%2BJ.%26aulast%3DKerr%26aufirst%3DB.%2BM.%26aulast%3DCrowe%26aufirst%3DB.%26aulast%3DPurdie%26aufirst%3DD.%2BM.%26aulast%3DElliott%26aufirst%3DS.%2BL.%26aulast%3DEllem%26aufirst%3DK.%2BA.%26aulast%3DSchmidt%26aufirst%3DC.%2BW.%26atitle%3DDurable%2520complete%2520clinical%2520responses%2520in%2520a%2520phase%2520I%252FII%2520trial%2520using%2520an%2520autologous%2520melanoma%2520cell%252Fdendritic%2520cell%2520vaccine%26jtitle%3DCancer%2520Immunol.%2520Immunother.%26date%3D2003%26volume%3D52%26spage%3D387%26epage%3D395%26doi%3D10.1007%2Fs00262-003-0375-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">The calcium-binding protein S100B down-regulates p53 and apoptosis in malignant melanoma</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">27487</span><span class="NLM_x">–</span> <span class="NLM_lpage">27498</span><span class="refDoi"> DOI: 10.1074/jbc.M110.155382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1074%2Fjbc.M110.155382" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=27487-27498&author=J.+Linauthor=Q.+Yangauthor=P.+T.+Wilderauthor=F.+Carrierauthor=D.+J.+Weber&title=The+calcium-binding+protein+S100B+down-regulates+p53+and+apoptosis+in+malignant+melanoma&doi=10.1074%2Fjbc.M110.155382"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.155382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.155382%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520calcium-binding%2520protein%2520S100B%2520down-regulates%2520p53%2520and%2520apoptosis%2520in%2520malignant%2520melanoma%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D27487%26epage%3D27498%26doi%3D10.1074%2Fjbc.M110.155382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markowitz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Inhibiting S100B restores p53 levels in primary malignant melanoma cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">34071</span><span class="NLM_x">–</span> <span class="NLM_lpage">34077</span><span class="refDoi"> DOI: 10.1074/jbc.M405419200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1074%2Fjbc.M405419200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=15178678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtFOhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=34071-34077&author=J.+Linauthor=Q.+Yangauthor=Z.+Yanauthor=J.+Markowitzauthor=P.+T.+Wilderauthor=F.+Carrierauthor=D.+J.+Weber&title=Inhibiting+S100B+restores+p53+levels+in+primary+malignant+melanoma+cancer+cells&doi=10.1074%2Fjbc.M405419200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting S100B restores p53 levels in primary malignant melanoma cancer cells</span></div><div class="casAuthors">Lin, Jing; Yang, Qingyuan; Yan, Zhe; Markowitz, Joseph; Wilder, Paul T.; Carrier, France; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">34071-34077</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">S100 calcium-binding proteins such as S100B are elevated in primary malignant melanoma and are used as markers for this and numerous other cancers.  Wild-type p53 protein levels are relatively low in these cancer cells (i.e., when compared with cells without S100B) but are elevated when RNA antisense to S100B is introduced.  This result implicates S100B in the down-regulation of p53 and is consistent with the large decreases in p53 protein levels obsd. previously in transient co-transfections of p53 and S100B.  Down-regulation of p53 in primary malignant melanoma cells is likely the result of a direct interaction with S100B, which was obsd. by co-immunopptn. expts.  Furthermore, p53 binds regions of the S100B promoter, one of which matches the 20-nucleotide p53-binding consensus DNA sequence perfectly.  Therefore, when p53 levels increase, it contributes to its own demise by up-regulating the transcription of S100B as part of a neg. feedback loop.  This is analogous to what is found for another protein that down-regulates p53, namely hdm2 (human double mutant 2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTg-HaV94udrVg90H21EOLACvtfcHk0lhM910UfR91MA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtFOhsLk%253D&md5=40834f8704a05a5ee85e4d949231c0c2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M405419200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M405419200%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DMarkowitz%26aufirst%3DJ.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DInhibiting%2520S100B%2520restores%2520p53%2520levels%2520in%2520primary%2520malignant%2520melanoma%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D34071%26epage%3D34077%26doi%3D10.1074%2Fjbc.M405419200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Markowitz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gitti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldisseri, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Udan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Identification and characterization of small molecule inhibitors of the calcium-dependent S100B-p53 tumor suppressor interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5085</span><span class="NLM_x">–</span> <span class="NLM_lpage">5093</span><span class="refDoi"> DOI: 10.1021/jm0497038</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0497038" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5085-5093&author=J.+Markowitzauthor=I.+Chenauthor=R.+Gittiauthor=D.+M.+Baldisseriauthor=Y.+Panauthor=R.+Udanauthor=F.+Carrierauthor=A.+D.+MacKerellauthor=D.+J.+Weber&title=Identification+and+characterization+of+small+molecule+inhibitors+of+the+calcium-dependent+S100B-p53+tumor+suppressor+interaction&doi=10.1021%2Fjm0497038"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm0497038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0497038%26sid%3Dliteratum%253Aachs%26aulast%3DMarkowitz%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DGitti%26aufirst%3DR.%26aulast%3DBaldisseri%26aufirst%3DD.%2BM.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DUdan%26aufirst%3DR.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520calcium-dependent%2520S100B-p53%2520tumor%2520suppressor%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5085%26epage%3D5093%26doi%3D10.1021%2Fjm0497038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKnight, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">The evolution of S100B inhibitors for the treatment of malignant melanoma</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span><span class="refDoi"> DOI: 10.4155/fmc.12.191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.4155%2Ffmc.12.191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23256816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVKgsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=97-109&author=K.+G.+Hartmanauthor=L.+E.+McKnightauthor=M.+A.+Lirianoauthor=D.+J.+Weber&title=The+evolution+of+S100B+inhibitors+for+the+treatment+of+malignant+melanoma&doi=10.4155%2Ffmc.12.191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of S100B inhibitors for the treatment of malignant melanoma</span></div><div class="casAuthors">Hartman, Kira G.; McKnight, Laura E.; Liriano, Melissa A.; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-109</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Malignant melanoma continues to be an extremely fatal cancer due to a lack of viable treatment options for patients.  The calcium-binding protein S100B has long been used as a clin. biomarker, aiding in malignant melanoma staging and patient prognosis.  However, the discovery of p53 as a S100B target and the consequent impact on cell apoptosis redirected research efforts towards the development of inhibitors of this S100B-p53 interaction.  Several approaches, including computer-aided drug design, fluorescence polarization competition assays, NMR, x-ray crystallog. and cell-based screens have been performed to identify compds. that block the S100B-p53 assocn., reactivate p53 transcriptional activities and induce cancer cell death.  Eight promising compds., including pentamidine, are presented in this review and the potential for future modifications is discussed.  Synthesis of compd. derivs. will likely exhibit increased S100B affinity and mimic important S100B-target dynamic properties that will result in high specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7sdVwSO2M2rVg90H21EOLACvtfcHk0ljWub_JaKS1ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVKgsLfN&md5=96d214505432e590eb1903f51810b35f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.191%26sid%3Dliteratum%253Aachs%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DMcKnight%26aufirst%3DL.%2BE.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520evolution%2520of%2520S100B%2520inhibitors%2520for%2520the%2520treatment%2520of%2520malignant%2520melanoma%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D97%26epage%3D109%26doi%3D10.4155%2Ffmc.12.191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Hartman, K. G.; Wilder, P. T.; Varney, K.; MacKerell, A. D.,  Jr.; Coop, A.; Zimmer, D.; Lapidus, R. G.; Weber, D. J.</span><span> </span><span class="NLM_article-title">Inhibiting S100B in Malignant Melanoma</span>. In  <span class="citation_source-book">Melanoma—From Early Detection to Treatment</span>; <span class="NLM_contrib-group">Duc, H. T.</span>, Ed.; <span class="NLM_publisher-name">InTech</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">649</span><span class="NLM_x">–</span> <span class="NLM_lpage">667</span>, DOI: <span class="refDoi"> DOI: 10.5772/55176</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.5772%2F55176" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=649-667&author=K.+G.+Hartman&author=P.+T.+Wilder&author=K.+Varney&author=A.+D.+MacKerell&author=A.+Coop&author=D.+Zimmer&author=R.+G.+Lapidus&author=D.+J.+Weberauthor=H.+T.+Duc&title=Melanoma%E2%80%94From+Early+Detection+to+Treatment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.5772%2F55176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.5772%252F55176%26sid%3Dliteratum%253Aachs%26aulast%3DHartman%26aufirst%3DK.%2BG.%26atitle%3DInhibiting%2520S100B%2520in%2520Malignant%2520Melanoma%26btitle%3DMelanoma%25E2%2580%2594From%2520Early%2520Detection%2520to%2520Treatment%26aulast%3DDuc%26aufirst%3DH.%2BT.%26pub%3DInTech%26date%3D2013%26spage%3D649%26epage%3D667%26doi%3D10.5772%2F55176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Meghnani, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetter, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leclerc, E.</span><span> </span><span class="NLM_article-title">RAGE overexpression confers a metastatic phenotype to the WM115 human primary melanoma cell line</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Basis Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1842</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1027</span><span class="refDoi"> DOI: 10.1016/j.bbadis.2014.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.bbadis.2014.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24613454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFyisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1842&publication_year=2014&pages=1017-1027&author=V.+Meghnaniauthor=S.+W.+Vetterauthor=E.+Leclerc&title=RAGE+overexpression+confers+a+metastatic+phenotype+to+the+WM115+human+primary+melanoma+cell+line&doi=10.1016%2Fj.bbadis.2014.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">RAGE overexpression confers a metastatic phenotype to the WM115 human primary melanoma cell line</span></div><div class="casAuthors">Meghnani, Varsha; Vetter, Stefan W.; Leclerc, Estelle</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Basis of Disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1842</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1017-1027</span>CODEN:
                <span class="NLM_cas:coden">BBADEX</span>;
        ISSN:<span class="NLM_cas:issn">0925-4439</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">The formation of melanoma metastases from primary tumor cells is a complex phenomenon that involves the regulation of multiple genes.  We have previously shown that the receptor for advanced glycation end products (RAGE) was up-regulated in late metastatic stages of melanoma patient samples and we hypothesized that up-regulation of RAGE in cells forming a primary melanoma tumor could contribute to the metastatic switch of these cells.  To test our hypothesis, we overexpressed RAGE in the WM115 human melanoma cell line that was established from a primary melanoma tumor of a patient.  We show here that overexpression of RAGE in these cells is assocd. with mesenchymal-like morphologies of the cells.  These cells demonstrate higher migration abilities and reduced proliferation properties, suggesting that the cells have switched to a metastatic phenotype.  At the mol. level, we show that RAGE overexpression is assocd. with the up-regulation of the RAGE ligand S100B and the down-regulation of p53, ERK1/2, cyclin E and NF-κB.  Our study supports a role of RAGE in the metastatic switch of melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmJa067cU0o7Vg90H21EOLACvtfcHk0ljWub_JaKS1ig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFyisLk%253D&md5=ffcf426cd8cf7d0ebd06171c57b6eed4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bbadis.2014.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbadis.2014.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DMeghnani%26aufirst%3DV.%26aulast%3DVetter%26aufirst%3DS.%2BW.%26aulast%3DLeclerc%26aufirst%3DE.%26atitle%3DRAGE%2520overexpression%2520confers%2520a%2520metastatic%2520phenotype%2520to%2520the%2520WM115%2520human%2520primary%2520melanoma%2520cell%2520line%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Basis%2520Dis.%26date%3D2014%26volume%3D1842%26spage%3D1017%26epage%3D1027%26doi%3D10.1016%2Fj.bbadis.2014.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Briceland, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailie, G. R.</span><span> </span><span class="NLM_article-title">Pentamidine-associated nephrotoxicity and hyperkalemia in patients with AIDS</span> <span class="citation_source-journal">Ann. Pharmacother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1171</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1177/106002809102501102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1177%2F106002809102501102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1991&pages=1171-1174&author=L.+L.+Bricelandauthor=G.+R.+Bailie&title=Pentamidine-associated+nephrotoxicity+and+hyperkalemia+in+patients+with+AIDS&doi=10.1177%2F106002809102501102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F106002809102501102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F106002809102501102%26sid%3Dliteratum%253Aachs%26aulast%3DBriceland%26aufirst%3DL.%2BL.%26aulast%3DBailie%26aufirst%3DG.%2BR.%26atitle%3DPentamidine-associated%2520nephrotoxicity%2520and%2520hyperkalemia%2520in%2520patients%2520with%2520AIDS%26jtitle%3DAnn.%2520Pharmacother.%26date%3D1991%26volume%3D25%26spage%3D1171%26epage%3D1174%26doi%3D10.1177%2F106002809102501102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Edwards, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Crystal structure of a pentamidine-oligonucleotide complex: implications for DNA-binding properties</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">7104</span><span class="NLM_x">–</span> <span class="NLM_lpage">7109</span><span class="refDoi"> DOI: 10.1021/bi00146a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00146a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADyaK38XkvVaqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1992&pages=7104-7109&author=K.+J.+Edwardsauthor=T.+C.+Jenkinsauthor=S.+Neidle&title=Crystal+structure+of+a+pentamidine-oligonucleotide+complex%3A+implications+for+DNA-binding+properties&doi=10.1021%2Fbi00146a011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a pentamidine-oligonucleotide complex:  implications for DNA-binding properties</span></div><div class="casAuthors">Edwards, Karen J.; Jenkins, Terence C.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">7104-9</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The crystal structure of the complex formed between the dodecanucleotide d(CGCGAATTCGCG)2 and the drug pentamidine, which is active against the Pneumocystis carinii pathogen in AIDS patients, was detd. to a resoln. of 2.1 Å and an R-factor of 19.4%.  The drug was in the 5'-AATT minor groove region of the duplex, with the amidinium groups H-bonded to adenine N3 atoms in an interstrand manner.  The drug mol. adopts an extended conformation, and the immediate binding site spans 4 base pairs.  Structural details of the drug-DNA interactions are discussed, and comparison is made with the dodecamer complex of the structurally similar berenil ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhj8RQpE1GobVg90H21EOLACvtfcHk0lgKqA7Fh61ZNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XkvVaqtr4%253D&md5=c702833ed79000720539571e08baa68e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fbi00146a011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00146a011%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DK.%2BJ.%26aulast%3DJenkins%26aufirst%3DT.%2BC.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520a%2520pentamidine-oligonucleotide%2520complex%253A%2520implications%2520for%2520DNA-binding%2520properties%26jtitle%3DBiochemistry%26date%3D1992%26volume%3D31%26spage%3D7104%26epage%3D7109%26doi%3D10.1021%2Fbi00146a011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Pathak, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhawan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindner, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borden, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farver, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, T.</span><span> </span><span class="NLM_article-title">Pentamidine is an inhibitor of PRL phosphatases with anticancer activity</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1264</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=12516958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmtFalsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=1255-1264&author=M.+K.+Pathakauthor=D.+Dhawanauthor=D.+J.+Lindnerauthor=E.+C.+Bordenauthor=C.+Farverauthor=T.+Yi&title=Pentamidine+is+an+inhibitor+of+PRL+phosphatases+with+anticancer+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pentamidine is an inhibitor of PRL phosphatases with anticancer activity</span></div><div class="casAuthors">Pathak, Manas K.; Dhawan, Deepika; Lindner, Daniel J.; Borden, Ernest C.; Farver, Carol; Yi, Taolin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1255-1264</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Thr PRL family oncogenic phosphatases are attractive targets for developing inhibitors as anticancer therapeutics given their potentially pathogenic role in human malignancies.  Herein we demonstrate that pentamidine, an anti-protozoa drug with an unknown mechanism of action, is an inhibitor of PRLs with anticancer potential.  Pentamidine at its therapeutic doses inhibited recombinant PRL phosphatases in vitro and inactivated ectopically expressed PRLs in NIH3T3 transfectants with an effective duration more than 24 h after a pulse cell treatment.  The drug had in vitro growth-inhibitory activity against human cancer cell lines that express the endogenous PRLs.  Pentamidine at a tolerable dose markedly inhibited the growth of WM9 human melanoma tumors in nude mice coincident with the induction of tumor cell necrosis and is capable of inactivating ectopically expressed PRL-2 in the cancer cells.  These observations suggest the potential of pentamidine in anticancer therapies and may provide a basis for developing novel PTPase-targeted therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkt1FZKVxVYLVg90H21EOLACvtfcHk0lgKqA7Fh61ZNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmtFalsg%253D%253D&md5=e695157767544c6a55e5c6b7a4966d4c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPathak%26aufirst%3DM.%2BK.%26aulast%3DDhawan%26aufirst%3DD.%26aulast%3DLindner%26aufirst%3DD.%2BJ.%26aulast%3DBorden%26aufirst%3DE.%2BC.%26aulast%3DFarver%26aufirst%3DC.%26aulast%3DYi%26aufirst%3DT.%26atitle%3DPentamidine%2520is%2520an%2520inhibitor%2520of%2520PRL%2520phosphatases%2520with%2520anticancer%2520activity%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2002%26volume%3D1%26spage%3D1255%26epage%3D1264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">McKnight, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bezawada, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kudrimoti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godoy-Ruiz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Structure-Based Discovery of a Novel Pentamidine-Related Inhibitor of the Calcium-Binding Protein S100B</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">975</span><span class="NLM_x">–</span> <span class="NLM_lpage">979</span><span class="refDoi"> DOI: 10.1021/ml300166s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300166s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=975-979&author=L.+E.+McKnightauthor=E.+P.+Ramanauthor=P.+Bezawadaauthor=S.+Kudrimotiauthor=P.+T.+Wilderauthor=K.+G.+Hartmanauthor=R.+Godoy-Ruizauthor=E.+A.+Tothauthor=A.+Coopauthor=A.+D.+Mackerellauthor=D.+J.+Weber&title=Structure-Based+Discovery+of+a+Novel+Pentamidine-Related+Inhibitor+of+the+Calcium-Binding+Protein+S100B&doi=10.1021%2Fml300166s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml300166s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300166s%26sid%3Dliteratum%253Aachs%26aulast%3DMcKnight%26aufirst%3DL.%2BE.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DBezawada%26aufirst%3DP.%26aulast%3DKudrimoti%26aufirst%3DS.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DGodoy-Ruiz%26aufirst%3DR.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DStructure-Based%2520Discovery%2520of%2520a%2520Novel%2520Pentamidine-Related%2520Inhibitor%2520of%2520the%2520Calcium-Binding%2520Protein%2520S100B%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D975%26epage%3D979%26doi%3D10.1021%2Fml300166s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Agamennone, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cesari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lalli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turlizzi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Conte, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turano, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padova, A.</span><span> </span><span class="NLM_article-title">Fragmenting the S100B-p53 Interaction: Combined Virtual/Biophysical Screening Approaches to Identify Ligands</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">428</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1002/cmdc.200900393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fcmdc.200900393" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=428-435&author=M.+Agamennoneauthor=L.+Cesariauthor=D.+Lalliauthor=E.+Turlizziauthor=R.+Del+Conteauthor=P.+Turanoauthor=S.+Manganiauthor=A.+Padova&title=Fragmenting+the+S100B-p53+Interaction%3A+Combined+Virtual%2FBiophysical+Screening+Approaches+to+Identify+Ligands&doi=10.1002%2Fcmdc.200900393"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900393%26sid%3Dliteratum%253Aachs%26aulast%3DAgamennone%26aufirst%3DM.%26aulast%3DCesari%26aufirst%3DL.%26aulast%3DLalli%26aufirst%3DD.%26aulast%3DTurlizzi%26aufirst%3DE.%26aulast%3DDel%2BConte%26aufirst%3DR.%26aulast%3DTurano%26aufirst%3DP.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DPadova%26aufirst%3DA.%26atitle%3DFragmenting%2520the%2520S100B-p53%2520Interaction%253A%2520Combined%2520Virtual%252FBiophysical%2520Screening%2520Approaches%2520to%2520Identify%2520Ligands%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D428%26epage%3D435%26doi%3D10.1002%2Fcmdc.200900393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Yoshimura, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyafusa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, K.</span><span> </span><span class="NLM_article-title">Identification of small-molecule inhibitors of the human S100B-p53 interaction and evaluation of their activity in human melanoma cells</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1109</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.bmc.2012.12.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.bmc.2012.12.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1109-1115&author=C.+Yoshimuraauthor=T.+Miyafusaauthor=K.+Tsumoto&title=Identification+of+small-molecule+inhibitors+of+the+human+S100B-p53+interaction+and+evaluation+of+their+activity+in+human+melanoma+cells&doi=10.1016%2Fj.bmc.2012.12.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.12.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.12.042%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DC.%26aulast%3DMiyafusa%26aufirst%3DT.%26aulast%3DTsumoto%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520small-molecule%2520inhibitors%2520of%2520the%2520human%2520S100B-p53%2520interaction%2520and%2520evaluation%2520of%2520their%2520activity%2520in%2520human%2520melanoma%2520cells%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D1109%26epage%3D1115%26doi%3D10.1016%2Fj.bmc.2012.12.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bogan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, K. S.</span><span> </span><span class="NLM_article-title">Anatomy of hot spots in protein interfaces</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1006/jmbi.1998.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1006%2Fjmbi.1998.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=9653027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1-9&author=A.+A.+Boganauthor=K.+S.+Thorn&title=Anatomy+of+hot+spots+in+protein+interfaces&doi=10.1006%2Fjmbi.1998.1843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomy of hot spots in protein interfaces</span></div><div class="casAuthors">Bogan A A; Thorn K S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Binding of one protein to another is involved in nearly all biological functions, yet the principles governing the interaction of proteins are not fully understood.  To analyze the contributions of individual amino acid residues in protein-protein binding we have compiled a database of 2325 alanine mutants for which the change in free energy of binding upon mutation to alanine has been measured (available at http://motorhead. ucsf.edu/thorn/hotspot).  Our analysis shows that at the level of side-chains there is little correlation between buried surface area and free energy of binding.  We find that the free energy of binding is not evenly distributed across interfaces; instead, there are hot spots of binding energy made up of a small subset of residues in the dimer interface.  These hot spots are enriched in tryptophan, tyrosine and arginine, and are surrounded by energetically less important residues that most likely serve to occlude bulk solvent from the hot spot.  Occlusion of solvent is found to be a necessary condition for highly energetic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZMrH3BevWP3ELvajU9pjPfW6udTcc2eZ809EJZtpV-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D&md5=e3ef3c857d1fe070deff25dc54d09587</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.1843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.1843%26sid%3Dliteratum%253Aachs%26aulast%3DBogan%26aufirst%3DA.%2BA.%26aulast%3DThorn%26aufirst%3DK.%2BS.%26atitle%3DAnatomy%2520of%2520hot%2520spots%2520in%2520protein%2520interfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D280%26spage%3D1%26epage%3D9%26doi%3D10.1006%2Fjmbi.1998.1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Cerchietti, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghetu, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da Silva, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fares, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">400</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.ccr.2009.12.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=20385364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=400-411&author=L.+C.+Cerchiettiauthor=A.+F.+Ghetuauthor=X.+Zhuauthor=G.+F.+Da+Silvaauthor=S.+Zhongauthor=M.+Matthewsauthor=K.+L.+Buntingauthor=J.+M.+Poloauthor=C.+Faresauthor=C.+H.+Arrowsmithauthor=S.+N.+Yangauthor=M.+Garciaauthor=A.+Coopauthor=A.+D.+Mackerellauthor=G.+G.+Priveauthor=A.+Melnick&title=A+small-molecule+inhibitor+of+BCL6+kills+DLBCL+cells+in+vitro+and+in+vivo&doi=10.1016%2Fj.ccr.2009.12.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Ghetu, Alexandru F.; Zhu, Xiao; Da Silva, Gustavo F.; Zhong, Shijun; Matthews, Marilyn; Bunting, Karen L.; Polo, Jose M.; Fares, Christophe; Arrowsmith, Cheryl H.; Yang, Shao Ning; Garcia, Monica; Coop, Andrew; MacKerell, Alexander D., Jr.; Prive, Gilbert G.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most frequently involved oncogene in diffuse large B cell lymphoma (DLBCL).  We combined computer-aided drug design with functional assays to identify low-mol.-wt. compds. that bind to the corepressor binding groove of the BCL6 BTB domain.  One such compd. disrupted BCL6/corepressor complexes in vitro and in vivo, and was obsd. by X-ray crystallog. and NMR to bind the crit. site within the BTB groove.  This compd. could induce expression of BCL6 target genes and kill BCL6-pos. DLBCL cell lines.  In xenotransplantation expts., the compd. was nontoxic and potently suppressed DLBCL tumors in vivo.  The compd. also killed primary DLBCLs from human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdcScjNPjhiLVg90H21EOLACvtfcHk0lizxcP9DKeWAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D&md5=64f30a813c26394b69da6de95991f8dd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.050%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DM.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DFares%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520small-molecule%2520inhibitor%2520of%2520BCL6%2520kills%2520DLBCL%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D400%26epage%3D411%26doi%3D10.1016%2Fj.ccr.2009.12.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Samadani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brophy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priyakumar, U. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St John, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, P.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of ERK-mediated immediate early gene expression and proliferation of melanoma cells expressing mutated BRaf</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">438</span><span class="refDoi"> DOI: 10.1042/BJ20131571</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1042%2FBJ20131571" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=25695333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslCgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=425-438&author=R.+Samadaniauthor=J.+Zhangauthor=A.+Brophyauthor=T.+Oashiauthor=U.+D.+Priyakumarauthor=E.+P.+Ramanauthor=F.+J.+St+Johnauthor=K.+Y.+Jungauthor=S.+Fletcherauthor=E.+Pozharskiauthor=A.+D.+MacKerellauthor=P.+Shapiro&title=Small-molecule+inhibitors+of+ERK-mediated+immediate+early+gene+expression+and+proliferation+of+melanoma+cells+expressing+mutated+BRaf&doi=10.1042%2FBJ20131571"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of ERK-mediated immediate early gene expression and proliferation of melanoma cells expressing mutated BRaf</span></div><div class="casAuthors">Samadani, Ramin; Zhang, Jun; Brophy, Amanda; Oashi, Taiji; Priyakumar, U. Deva; Raman, E. Prabhu; St. John, Franz J.; Jung, Kwan-Young; Fletcher, Steven; Pozharski, Edwin; MacKerell, Alexander D.; Shapiro, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">425-438</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Constitutive activation of the extracellular-signal-regulated kinases 1 and 2 (ERK1/2) are central to regulating the proliferation and survival of many cancer cells.  The current inhibitors of ERK1/2 target ATP binding or the catalytic site and are therefore limited in their utility for elucidating the complex biol. roles of ERK1/2 through its phosphorylation and regulation of over 100 substrate proteins.  To overcome this limitation, a combination of computational and exptl. methods was used to identify low-mol.-mass inhibitors that are intended to target ERK1/2 substrate-docking domains and selectively interfere with ERK1/2 regulation of substrate proteins.  In the present study, we report the identification and characterization of compds. with a thienyl benzenesulfonate scaffold that were designed to inhibit ERK1/2 substrates contg. an F-site or DEF (docking site for ERK, FXF) motif.  Exptl. evidence shows the compds. inhibit the expression of F-site contg. immediate early genes (IEGs) of the Fos family, including c-Fos and Fra1, and transcriptional regulation of the activator protein-1 (AP-1) complex.  Moreover, this class of compds. selectively induces apoptosis in melanoma cells contg. mutated BRaf and constitutively active ERK1/2 signalling, including melanoma cells that are inherently resistant to clin. relevant kinase inhibitors.  These findings represent the identification and initial characterization of a novel class of compds. that inhibit ERK1/2 signalling functions and their potential utility for elucidating ERK1/2 and other signalling events that control the growth and survival of cancer cells contg. elevated ERK1/2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy0bGU3k6ObbVg90H21EOLACvtfcHk0li3h-nGGOmqgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslCgsrg%253D&md5=7a04a08b6b0a015ac4b673ab714f7e3e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBJ20131571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20131571%26sid%3Dliteratum%253Aachs%26aulast%3DSamadani%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBrophy%26aufirst%3DA.%26aulast%3DOashi%26aufirst%3DT.%26aulast%3DPriyakumar%26aufirst%3DU.%2BD.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DSt%2BJohn%26aufirst%3DF.%2BJ.%26aulast%3DJung%26aufirst%3DK.%2BY.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DShapiro%26aufirst%3DP.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520ERK-mediated%2520immediate%2520early%2520gene%2520expression%2520and%2520proliferation%2520of%2520melanoma%2520cells%2520expressing%2520mutated%2520BRaf%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D425%26epage%3D438%26doi%3D10.1042%2FBJ20131571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Divalent metal ion complexes of S100B in the absence and presence of pentamidine</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.06.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.jmb.2008.06.047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2008&pages=56-73&author=T.+H.+Charpentierauthor=P.+T.+Wilderauthor=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=E.+Pozharskiauthor=A.+D.+MacKerellauthor=A.+Coopauthor=E.+A.+Tothauthor=D.+J.+Weber&title=Divalent+metal+ion+complexes+of+S100B+in+the+absence+and+presence+of+pentamidine&doi=10.1016%2Fj.jmb.2008.06.047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.06.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.06.047%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DDivalent%2520metal%2520ion%2520complexes%2520of%2520S100B%2520in%2520the%2520absence%2520and%2520presence%2520of%2520pentamidine%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D382%26spage%3D56%26epage%3D73%26doi%3D10.1016%2Fj.jmb.2008.06.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Cavalier, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alasady, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neau, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Covalent small molecule inhibitors of Ca(2+)-bound S100B</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6628</span><span class="NLM_x">–</span> <span class="NLM_lpage">6640</span><span class="refDoi"> DOI: 10.1021/bi5005552</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi5005552" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Ciu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6628-6640&author=M.+C.+Cavalierauthor=A.+D.+Pierceauthor=P.+T.+Wilderauthor=M.+J.+Alasadyauthor=K.+G.+Hartmanauthor=D.+B.+Neauauthor=T.+L.+Foleyauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=E.+A.+Tothauthor=D.+J.+Weber&title=Covalent+small+molecule+inhibitors+of+Ca%282%2B%29-bound+S100B&doi=10.1021%2Fbi5005552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Small Molecule Inhibitors of Ca2+-Bound S100B</span></div><div class="casAuthors">Cavalier, Michael C.; Pierce, Adam D.; Wilder, Paul T.; Alasady, Milad J.; Hartman, Kira G.; Neau, David B.; Foley, Timothy L.; Jadhav, Ajit; Maloney, David J.; Simeonov, Anton; Toth, Eric A.; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">6628-6640</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Elevated levels of the tumor marker S100B are obsd. in malignant melanoma, and this EF-hand-contg. protein was shown to directly bind wild-type (wt) p53 in a Ca2+-dependent manner, dissoc. the p53 tetramer, and inhibit its tumor suppression functions.  Likewise, inhibiting S100B with small interfering RNA (siRNAS100B) is sufficient to restore wild-type p53 levels and its downstream gene products and induce the arrest of cell growth and UV-dependent apoptosis in malignant melanoma.  Therefore, it is a goal to develop S100B inhibitors (SBiXs) that inhibit the S100B-p53 complex and restore active p53 in this deadly cancer.  Using a structure-activity relationship by NMR approach (SAR by NMR), three persistent binding pockets are found on S100B, termed sites 1-3.  While inhibitors that simultaneously bind sites 2 and 3 are in place, no mols. that simultaneously bind all three persistent sites are available.  For this purpose, Cys84 was used in this study as a potential means to bridge sites 1 and 2 because it is located in a small crevice between these two deeper pockets on the protein.  Using a fluorescence polarization competition assay, several Cys84-modified S100B complexes were identified and examd. further.  For five such SBiX-S100B complexes, crystallog. structures confirmed their covalent binding to Cys84 near site 2 and thus present straightforward chem. biol. strategies for bridging sites 1 and 3.  Importantly, one such compd., SC1982, showed an S100B-dependent death response in assays with WM115 malignant melanoma cells, so it will be particularly useful for the design of SBiX mols. with improved affinity and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHMFpfVuslNLVg90H21EOLACvtfcHk0li3h-nGGOmqgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Ciu7zO&md5=bd0423ee89babc4ee4e8e40282f0bd10</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fbi5005552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi5005552%26sid%3Dliteratum%253Aachs%26aulast%3DCavalier%26aufirst%3DM.%2BC.%26aulast%3DPierce%26aufirst%3DA.%2BD.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DAlasady%26aufirst%3DM.%2BJ.%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DNeau%26aufirst%3DD.%2BB.%26aulast%3DFoley%26aufirst%3DT.%2BL.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DCovalent%2520small%2520molecule%2520inhibitors%2520of%2520Ca%25282%252B%2529-bound%2520S100B%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D6628%26epage%3D6640%26doi%3D10.1021%2Fbi5005552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Spychala, J.</span><span> </span><span class="NLM_article-title">4-(Cyclic Amidino)phenols - Preparation and Use In A Diamidine Synthesis</span> <span class="citation_source-journal">Synth. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1083</span><span class="NLM_x">–</span> <span class="NLM_lpage">1094</span><span class="refDoi"> DOI: 10.1080/00397910008087126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1080%2F00397910008087126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVOrur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2000&pages=1083-1094&author=J.+Spychala&title=4-%28Cyclic+Amidino%29phenols+-+Preparation+and+Use+In+A+Diamidine+Synthesis&doi=10.1080%2F00397910008087126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">4-(Cyclic-amidino)phenols. Preparation and use in a diamidine synthesis</span></div><div class="casAuthors">Spychala, Jaroslaw</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1083-1094</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">The Pinner synthesis was applied to the prepn. of phenols 4-substituted with cyclic amidines in high yields from readily available 4-HOC6H4C(OMe):NH.HCl and aliph. diamines.  An alternative attempt was made to convert 4-HOC6H4CSNH2 to 4-(1,4,5,6-tetrahydro-5-hydroxy-2-pyrimidinyl)phenol.  A procedure for the prepn. of 3,6-bis(4-hydroxyphenyl)-1,2,4,5-tetrazine is reported.  The syntheses of 2,4-bis[4-(4,5-dihydro-1H-imidazol-2-yl)phenoxy]pyrimidine and 2-chloro-4-[4-(4,5-dihydro-1H-imidazol-2-yl)phenoxy]pyrimidine exemplify the use of the title compds. as synthetic intermediates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFhiWY8pPdKLVg90H21EOLACvtfcHk0lhxNaUn3Njw0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVOrur8%253D&md5=4e57399d36f3950250f02d1504c55d43</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1080%2F00397910008087126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00397910008087126%26sid%3Dliteratum%253Aachs%26aulast%3DSpychala%26aufirst%3DJ.%26atitle%3D4-%2528Cyclic%2520Amidino%2529phenols%2520-%2520Preparation%2520and%2520Use%2520In%2520A%2520Diamidine%2520Synthesis%26jtitle%3DSynth.%2520Commun.%26date%3D2000%26volume%3D30%26spage%3D1083%26epage%3D1094%26doi%3D10.1080%2F00397910008087126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Pigini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brasili, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cassinelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giardina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gulini, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quaglia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melchiorre, C.</span><span> </span><span class="NLM_article-title">Synthesis and Alpha-Blocking Activity of Some Analogs of Idazoxan</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1016/0223-5234(87)90263-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2F0223-5234%2887%2990263-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1987&pages=273-276&author=M.+Piginiauthor=L.+Brasiliauthor=A.+Cassinelliauthor=D.+Giardinaauthor=U.+Guliniauthor=W.+Quagliaauthor=C.+Melchiorre&title=Synthesis+and+Alpha-Blocking+Activity+of+Some+Analogs+of+Idazoxan&doi=10.1016%2F0223-5234%2887%2990263-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2F0223-5234%2887%2990263-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0223-5234%252887%252990263-7%26sid%3Dliteratum%253Aachs%26aulast%3DPigini%26aufirst%3DM.%26aulast%3DBrasili%26aufirst%3DL.%26aulast%3DCassinelli%26aufirst%3DA.%26aulast%3DGiardina%26aufirst%3DD.%26aulast%3DGulini%26aufirst%3DU.%26aulast%3DQuaglia%26aufirst%3DW.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Alpha-Blocking%2520Activity%2520of%2520Some%2520Analogs%2520of%2520Idazoxan%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D1987%26volume%3D22%26spage%3D273%26epage%3D276%26doi%3D10.1016%2F0223-5234%2887%2990263-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Manley, P. J.; Nanda, K. K.; Trotter, B. W.</span><span> </span><span class="NLM_article-title">Imidazoisoindole neuropeptide s receptor antagonists</span>. WO 2010056567 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=P.+J.+Manley&author=K.+K.+Nanda&author=B.+W.+Trotter&title=Imidazoisoindole+neuropeptide+s+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DManley%26aufirst%3DP.%2BJ.%26atitle%3DImidazoisoindole%2520neuropeptide%2520s%2520receptor%2520antagonists%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Perner, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDomenico, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koenig, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gomtsyan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayburt, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drizin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, G. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jinkerson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keddy, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lukin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honore, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikusa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetter, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. George, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvis, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faltynek, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. H.</span><span> </span><span class="NLM_article-title">In vitro structure-activity relationship and in vivo characterization of 1-(Aryl)-3-(4-(amino)benzyl)urea transient receptor potential vanilloid 1 antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">3651</span><span class="NLM_x">–</span> <span class="NLM_lpage">3660</span><span class="refDoi"> DOI: 10.1021/jm070276i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070276i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=3651-3660&author=R.+J.+Pernerauthor=S.+DiDomenicoauthor=J.+R.+Koenigauthor=A.+Gomtsyanauthor=E.+K.+Bayburtauthor=R.+G.+Schmidtauthor=I.+Drizinauthor=G.+Z.+Zhengauthor=S.+C.+Turnerauthor=T.+Jinkersonauthor=B.+S.+Brownauthor=R.+G.+Keddyauthor=K.+Lukinauthor=H.+A.+McDonaldauthor=P.+Honoreauthor=J.+Mikusaauthor=K.+C.+Marshauthor=J.+M.+Wetterauthor=K.+St.+Georgeauthor=M.+F.+Jarvisauthor=C.+R.+Faltynekauthor=C.+H.+Lee&title=In+vitro+structure-activity+relationship+and+in+vivo+characterization+of+1-%28Aryl%29-3-%284-%28amino%29benzyl%29urea+transient+receptor+potential+vanilloid+1+antagonists&doi=10.1021%2Fjm070276i"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm070276i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070276i%26sid%3Dliteratum%253Aachs%26aulast%3DPerner%26aufirst%3DR.%2BJ.%26aulast%3DDiDomenico%26aufirst%3DS.%26aulast%3DKoenig%26aufirst%3DJ.%2BR.%26aulast%3DGomtsyan%26aufirst%3DA.%26aulast%3DBayburt%26aufirst%3DE.%2BK.%26aulast%3DSchmidt%26aufirst%3DR.%2BG.%26aulast%3DDrizin%26aufirst%3DI.%26aulast%3DZheng%26aufirst%3DG.%2BZ.%26aulast%3DTurner%26aufirst%3DS.%2BC.%26aulast%3DJinkerson%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DB.%2BS.%26aulast%3DKeddy%26aufirst%3DR.%2BG.%26aulast%3DLukin%26aufirst%3DK.%26aulast%3DMcDonald%26aufirst%3DH.%2BA.%26aulast%3DHonore%26aufirst%3DP.%26aulast%3DMikusa%26aufirst%3DJ.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DWetter%26aufirst%3DJ.%2BM.%26aulast%3DSt.%2BGeorge%26aufirst%3DK.%26aulast%3DJarvis%26aufirst%3DM.%2BF.%26aulast%3DFaltynek%26aufirst%3DC.%2BR.%26aulast%3DLee%26aufirst%3DC.%2BH.%26atitle%3DIn%2520vitro%2520structure-activity%2520relationship%2520and%2520in%2520vivo%2520characterization%2520of%25201-%2528Aryl%2529-3-%25284-%2528amino%2529benzyl%2529urea%2520transient%2520receptor%2520potential%2520vanilloid%25201%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D3651%26epage%3D3660%26doi%3D10.1021%2Fjm070276i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Koyanagi, T.; Tsukuda, S.; Nakamura, Y.; Tsunematsu, K.; Shimizu, M.</span>; <span class="NLM_contrib-group">Iwamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, M.</span><span> </span><span class="NLM_article-title">Plant disease control agent containing arylamidine derivative or salt thereof</span>. WO 2013062024,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=T.+Koyanagi&author=S.+Tsukuda&author=Y.+Nakamura&author=K.+Tsunematsu&author=M.+Shimizuauthor=T.+Iwamotoauthor=Y.+Kawaiauthor=M.+Ohno&title=Plant+disease+control+agent+containing+arylamidine+derivative+or+salt+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKoyanagi%26aufirst%3DT.%26aulast%3DIwamoto%26aufirst%3DT.%26aulast%3DKawai%26aufirst%3DY.%26aulast%3DOhno%26aufirst%3DM.%26atitle%3DPlant%2520disease%2520control%2520agent%2520containing%2520arylamidine%2520derivative%2520or%2520salt%2520thereof%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">de Jong, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engbersen, J. F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huskens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhoudt, D. N.</span><span> </span><span class="NLM_article-title">Cyclodextrin dimers as receptor molecules for steroid sensors</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">4034</span><span class="NLM_x">–</span> <span class="NLM_lpage">4040</span><span class="refDoi"> DOI: 10.1002/1521-3765(20001103)6:21<4034::AID-CHEM4034>3.0.CO;2-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=11126966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotlajsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=4034-4040&author=M.+R.+de+Jongauthor=J.+F.+J.+Engbersenauthor=J.+Huskensauthor=D.+N.+Reinhoudt&title=Cyclodextrin+dimers+as+receptor+molecules+for+steroid+sensors&doi=10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclodextrin dimers as receptor molecules for steroid sensors</span></div><div class="casAuthors">De Jong, Menno R.; Engbersen, Johan F. J.; Huskens, Jurriaan; Reinhoudt, David N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry--A European Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4034-4040</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The dansyl-modified dipropanolamine bridged β-cyclodextrin dimer complexes strongly with the steroidal bile salts.  Relative to native β-cyclodextrin, the binding of cholate and deoxycholate salts is esp. enhanced.  These steroids bind exclusively in a 1:1 fashion.  For other bile salts (chenodeoxycholate, ursodeoxycholate, and lithocholate) both 1:1 and 1:2 complexes were obsd. with stabilities similar to those of native β-cyclodextrin.  This indicates that only one cavity was used, with a small contribution from the 2nd.  The difference is attributed to the absence of a 12-hydroxy group in the 2nd group of steroids.  Comparison with a dimer that lacks the dansyl moiety shows that this group esp. hinders the cooperative binding of cholate and deoxycholate.  The smaller interference in the binding of the other steroids indicates that self-inclusion of the dansyl moiety hardly occurs.  This weak self-inclusion is supported by fluorescence studies.  The dansyl fluorescence of dimer is less blue-shifted than that of other known dansyl-appended cyclodextrin derivs.; this is indicative of a more polar micro-environment.  Addn. of guests causes a change in fluorescence intensity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolyZHcivP5b7Vg90H21EOLACvtfcHk0lhKgEyGFBfaHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotlajsrs%253D&md5=7ceed8c31a40403d84e7475c436ae90d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3765%252820001103%25296%253A21%253C4034%253A%253AAID-CHEM4034%253E3.0.CO%253B2-3%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BJong%26aufirst%3DM.%2BR.%26aulast%3DEngbersen%26aufirst%3DJ.%2BF.%2BJ.%26aulast%3DHuskens%26aufirst%3DJ.%26aulast%3DReinhoudt%26aufirst%3DD.%2BN.%26atitle%3DCyclodextrin%2520dimers%2520as%2520receptor%2520molecules%2520for%2520steroid%2520sensors%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2000%26volume%3D6%26spage%3D4034%26epage%3D4040%26doi%3D10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">The effects of CapZ peptide (TRTK-12) binding to S100B-Ca2+ as examined by NMR and X-ray crystallography</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">396</span><span class="NLM_x">, </span> <span class="NLM_fpage">1227</span><span class="NLM_x">–</span> <span class="NLM_lpage">1243</span><span class="refDoi"> DOI: 10.1016/j.jmb.2009.12.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.jmb.2009.12.057" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=2010&pages=1227-1243&author=T.+H.+Charpentierauthor=L.+E.+Thompsonauthor=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=P.+T.+Wilderauthor=E.+Pozharskiauthor=E.+A.+Tothauthor=D.+J.+Weber&title=The+effects+of+CapZ+peptide+%28TRTK-12%29+binding+to+S100B-Ca2%2B+as+examined+by+NMR+and+X-ray+crystallography&doi=10.1016%2Fj.jmb.2009.12.057"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.12.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.12.057%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DThompson%26aufirst%3DL.%2BE.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520effects%2520of%2520CapZ%2520peptide%2520%2528TRTK-12%2529%2520binding%2520to%2520S100B-Ca2%252B%2520as%2520examined%2520by%2520NMR%2520and%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D396%26spage%3D1227%26epage%3D1243%26doi%3D10.1016%2Fj.jmb.2009.12.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gianni, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">In vitro screening and structural characterization of inhibitors of the S100B-p53 interaction</span> <span class="citation_source-journal">Int. J. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2010</span><span class="NLM_x">, </span> <span class="NLM_fpage">109</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.2147/IJHTS.S8210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.2147%2FIJHTS.S8210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC2srmt1OmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&pages=109-126&author=P.+T.+Wilderauthor=T.+H.+Charpentierauthor=M.+A.+Lirianoauthor=K.+Gianniauthor=K.+M.+Varneyauthor=E.+Pozharskiauthor=A.+Coopauthor=E.+A.+Tothauthor=A.+D.+Mackerellauthor=D.+J.+Weber&title=In+vitro+screening+and+structural+characterization+of+inhibitors+of+the+S100B-p53+interaction&doi=10.2147%2FIJHTS.S8210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro screening and structural characterization of inhibitors of the S100B-p53 interaction</span></div><div class="casAuthors">Wilder Paul T; Charpentier Thomas H; Liriano Melissa A; Gianni Kira; Varney Kristen M; Pozharski Edwin; Coop Andrew; Toth Eric A; Mackerell Alex D; Weber David J</div><div class="citationInfo"><span class="NLM_cas:title">International journal of high throughput screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2010</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-126</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">S100B is highly over-expressed in many cancers, including malignant melanoma.  In such cancers, S100B binds wild-type p53 in a calcium-dependent manner, sequestering it, and promoting its degradation, resulting in the loss of p53-dependent tumor suppression activities.  Therefore, S100B inhibitors may be able to restore wild-type p53 levels in certain cancers and provide a useful therapeutic strategy.  In this regard, an automated and sensitive fluorescence polarization competition assay (FPCA) was developed and optimized to screen rapidly for lead compounds that bind Ca(2+)-loaded S100B and inhibit S100B target complex formation.  A screen of 2000 compounds led to the identification of 26 putative S100B low molecular weight inhibitors.  The binding of these small molecules to S100B was confirmed by nuclear magnetic resonance spectroscopy, and additional structural information was provided by x-ray crystal structures of several compounds in complexes with S100B.  Notably, many of the identified inhibitors function by chemically modifying Cys84 in protein.  These results validate the use of high-throughput FPCA to facilitate the identification of compounds that inhibit S100B.  These lead compounds will be the subject of future optimization studies with the ultimate goal of developing a drug with therapeutic activity for the treatment of malignant melanoma and/or other cancers with elevated S100B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdL3gvK7FnJNHxxy173TfNfW6udTcc2eaXrBFVfTezQ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmt1OmtQ%253D%253D&md5=d6384f2ba3954e6520c8c452bc6ce311</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2147%2FIJHTS.S8210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FIJHTS.S8210%26sid%3Dliteratum%253Aachs%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DGianni%26aufirst%3DK.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DIn%2520vitro%2520screening%2520and%2520structural%2520characterization%2520of%2520inhibitors%2520of%2520the%2520S100B-p53%2520interaction%26jtitle%3DInt.%2520J.%2520High%2520Throughput%2520Screening%26date%3D2010%26volume%3D2010%26spage%3D109%26epage%3D126%26doi%3D10.2147%2FIJHTS.S8210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Matsumura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiba, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kai, Y.</span><span> </span><span class="NLM_article-title">A novel mode of target recognition suggested by the 2.0 A structure of holo S100B from bovine brain</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1016/S0969-2126(98)00024-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2FS0969-2126%2898%2900024-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=233-241&author=H.+Matsumuraauthor=T.+Shibaauthor=T.+Inoueauthor=S.+Haradaauthor=Y.+Kai&title=A+novel+mode+of+target+recognition+suggested+by+the+2.0+A+structure+of+holo+S100B+from+bovine+brain&doi=10.1016%2FS0969-2126%2898%2900024-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900024-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900024-0%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumura%26aufirst%3DH.%26aulast%3DShiba%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHarada%26aufirst%3DS.%26aulast%3DKai%26aufirst%3DY.%26atitle%3DA%2520novel%2520mode%2520of%2520target%2520recognition%2520suggested%2520by%2520the%25202.0%2520A%2520structure%2520of%2520holo%2520S100B%2520from%2520bovine%2520brain%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D233%26epage%3D241%26doi%3D10.1016%2FS0969-2126%2898%2900024-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Small molecules bound to unique sites in the target protein binding cleft of calcium-bound S100B as characterized by nuclear magnetic resonance and X-ray crystallography</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6202</span><span class="NLM_x">–</span> <span class="NLM_lpage">6212</span><span class="refDoi"> DOI: 10.1021/bi9005754</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9005754" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Clsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=6202-6212&author=T.+H.+Charpentierauthor=P.+T.+Wilderauthor=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=S.+Zhongauthor=A.+Coopauthor=E.+Pozharskiauthor=A.+D.+MacKerellauthor=E.+A.+Tothauthor=D.+J.+Weber&title=Small+molecules+bound+to+unique+sites+in+the+target+protein+binding+cleft+of+calcium-bound+S100B+as+characterized+by+nuclear+magnetic+resonance+and+X-ray+crystallography&doi=10.1021%2Fbi9005754"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Bound to Unique Sites in the Target Protein Binding Cleft of Calcium-Bound S100B As Characterized by Nuclear Magnetic Resonance and X-ray Crystallography</span></div><div class="casAuthors">Charpentier, Thomas H.; Wilder, Paul T.; Liriano, Melissa A.; Varney, Kristen M.; Zhong, Shijun; Coop, Andrew; Pozharski, Edwin; MacKerell, Alexander D.; Toth, Eric A.; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">6202-6212</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural studies are part of a rational drug design program aimed at inhibiting the S100B-p53 interaction and restoring wild-type p53 function in malignant melanoma.  To this end, structures of three compds. (SBi132, SBi1279, and SBi523) bound to Ca2+-S100B were detd. by X-ray crystallog. at 2.10 Å (Rfree = 0.257), 1.98 Å (Rfree = 0.281), and 1.90 Å (Rfree = 0.228) resoln., resp.  Upon comparison, SBi132, SBi279, and SBi523 were found to bind in distinct locations and orientations within the hydrophobic target binding pocket of Ca2+-S100B with minimal structural changes obsd. for the protein upon complex formation with each compd.  Specifically, SBi132 binds nearby residues in loop 2 (His-42, Phe-43, and Leu-44) and helix 4 (Phe-76, Met-79, Ile-80, Ala-83, Cys-84, Phe-87, and Phe-88), whereas SBi523 interacts with a sep. site defined by residues within loop 2 (Ser-41, His-42, Phe-43, Leu-44, Glu-45, and Glu-46) and one residue on helix 4 (Phe-87).  The SBi279 binding site on Ca2+-S100B overlaps the SBi132 and SBi523 sites and contacts residues in both loop 2 (Ser-41, His-42, Phe-43, Leu-44, and Glu-45) and helix 4 (Ile-80, Ala-83, Cys-84, Phe-87, and Phe-88).  NMR data, including satn. transfer difference (STD) and 15N backbone and 13C side chain chem. shift perturbations, were consistent with the X-ray crystal structures and demonstrated the relevance of all three small mol.-S100B complexes in soln.  The discovery that SBi132, SBi279, and SBi523 bind to proximal sites on Ca2+-S100B could be useful for the development of a new class of mol.(s) that interacts with one or more of these binding sites simultaneously, thereby yielding novel tight binding inhibitors specific for blocking protein-protein interactions involving S100B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdiyYUBektHLVg90H21EOLACvtfcHk0ljIvs4OYTlyrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Clsr4%253D&md5=77ccf09a15aae75bdcde6dfabb996370</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fbi9005754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9005754%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DSmall%2520molecules%2520bound%2520to%2520unique%2520sites%2520in%2520the%2520target%2520protein%2520binding%2520cleft%2520of%2520calcium-bound%2520S100B%2520as%2520characterized%2520by%2520nuclear%2520magnetic%2520resonance%2520and%2520X-ray%2520crystallography%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D6202%26epage%3D6212%26doi%3D10.1021%2Fbi9005754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bair, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppenheim, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adhya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Recognition of the tumor suppressor protein p53 and other protein targets by the calcium-binding protein S100B</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1763</span><span class="NLM_x">, </span> <span class="NLM_fpage">1284</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2006.08.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.bbamcr.2006.08.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=17010455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CnsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1763&publication_year=2006&pages=1284-1297&author=P.+T.+Wilderauthor=J.+Linauthor=C.+L.+Bairauthor=T.+H.+Charpentierauthor=D.+Yangauthor=M.+Lirianoauthor=K.+M.+Varneyauthor=A.+Leeauthor=A.+B.+Oppenheimauthor=S.+Adhyaauthor=F.+Carrierauthor=D.+J.+Weber&title=Recognition+of+the+tumor+suppressor+protein+p53+and+other+protein+targets+by+the+calcium-binding+protein+S100B&doi=10.1016%2Fj.bbamcr.2006.08.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of the tumor suppressor protein p53 and other protein targets by the calcium-binding protein S100B</span></div><div class="casAuthors">Wilder, Paul T.; Lin, Jing; Bair, Catherine L.; Charpentier, Thomas H.; Yang, Dong; Liriano, Melissa; Varney, Kristen M.; Lee, Andrew; Oppenheim, Amos B.; Adhya, Sankar; Carrier, France; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1763</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1284-1297</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">S100B is an EF-hand contg. calcium-binding protein of the S100 protein family that exerts its biol. effect by binding and affecting various target proteins.  A consensus sequence for S100B target proteins was published as (K/R)(L/I)xWxxIL and matches a region in the actin-capping protein CapZ (V. V. Ivanenkov, G. A. Jamieson, Jr., E. Gruenstein, R. V. Dimlich, Characterization of S-100b binding epitopes. Identification of a novel target, the actin capping protein, CapZ, J. Biol. Chem. 270 (1995) 14651-14658).  Several addnl. S100B targets are known including p53, a nuclear Dbf2-related (NDR) kinase, the RAGE receptor, neuromodulin, protein kinase C, and others.  Examg. the binding sites of such targets and new protein sequence searches provided addnl. potential target proteins for S100B including Hdm2 and Hdm4, which were both found to bind S100B in a calcium-dependent manner.  The interaction between S100B and the Hdm2 and/or the Hdm4 proteins may be important physiol. in light of evidence that like Hdm2, S100B also contributes to lowering protein levels of the tumor suppressor protein, p53.  For the S100B-p53 interaction, it was found that phosphorylation of specific serine and/or threonine residues reduces the affinity of the S100B-p53 interaction by as much as an order of magnitude, and is important for protecting p53 from S100B-dependent down-regulation, a scenario that is similar to what is found for the Hdm2-p53 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrQ0k0TbFI5bVg90H21EOLACvtfcHk0ljIvs4OYTlyrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1CnsLbJ&md5=0354920ca745638506d75a1fd54dc057</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2006.08.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2006.08.024%26sid%3Dliteratum%253Aachs%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DBair%26aufirst%3DC.%2BL.%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DLiriano%26aufirst%3DM.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DOppenheim%26aufirst%3DA.%2BB.%26aulast%3DAdhya%26aufirst%3DS.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DRecognition%2520of%2520the%2520tumor%2520suppressor%2520protein%2520p53%2520and%2520other%2520protein%2520targets%2520by%2520the%2520calcium-binding%2520protein%2520S100B%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2006%26volume%3D1763%26spage%3D1284%26epage%3D1297%26doi%3D10.1016%2Fj.bbamcr.2006.08.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishima, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Target Binding to S100B Reduces Dynamic Properties and Increases Ca(2+)-Binding Affinity for Wild Type and EF-Hand Mutant Proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">423</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1016/j.jmb.2012.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.jmb.2012.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=22824086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Skur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2012&pages=365-385&author=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=N.+T.+Wrightauthor=C.+L.+Hoffmanauthor=E.+A.+Tothauthor=R.+Ishimaauthor=D.+J.+Weber&title=Target+Binding+to+S100B+Reduces+Dynamic+Properties+and+Increases+Ca%282%2B%29-Binding+Affinity+for+Wild+Type+and+EF-Hand+Mutant+Proteins&doi=10.1016%2Fj.jmb.2012.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Target Binding to S100B Reduces Dynamic Properties and Increases Ca2+-Binding Affinity for Wild Type and EF-Hand Mutant Proteins</span></div><div class="casAuthors">Liriano, Melissa A.; Varney, Kristen M.; Wright, Nathan T.; Hoffman, Cassandra L.; Toth, Eric A.; Ishima, Rieko; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">365-385</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mutations in the second EF-hand (D61N, D63N, D65N, and E72A) of S100B were used to study its Ca2+ binding and dynamic properties in the absence and presence of a bound target, TRTK-12.  With D63NS100B as an exception (D63NKD = 50 ± 9 μM), Ca2+ binding to EF2-hand mutants were reduced by more than 8-fold in the absence of TRTK-12 (D61NKD = 412 ± 67 μM, D65NKD = 968 ± 171 μM, and E72AKD = 471 ± 133 μM), when compared to wild-type protein (WTKD = 56 ± 9 μM).  For the TRTK-12 complexes, the Ca2+-binding affinity to wild type (WT+TRTKKD = 12 ± 10 μM) and the EF2 mutants was increased by 5- to 14-fold vs. in the absence of target (D61N+TRTKKD = 29 ± 1.2 μM, D63N+TRTKKD = 10 ± 2.2 μM, D65N+TRTKKD = 73 ± 4.4 μM, and E72A+TRTKKD = 18 ± 3.7 μM).  In addn., Rex, as measured using relaxation dispersion for side-chain 15N resonances of Asn63 (D63NS100B), was reduced upon TRTK-12 binding when measured by NMR.  Likewise, backbone motions on multiple timescales (picoseconds to milliseconds) throughout wild type, D61NS100B, D63NS100B, and D65NS100B were lowered upon binding TRTK-12.  However, the X-ray structures of Ca2+-bound (2.0 Å) and TRTK-bound (1.2 Å) D63NS100B showed no change in Ca2+ coordination; thus, these and analogous structural data for the wild-type protein could not be used to explain how target binding increased Ca2+-binding affinity in soln.  Therefore, a model for how S100B-TRTK-12 complex formation increases Ca2+ binding is discussed, which considers changes in protein dynamics upon binding the target TRTK-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrx_vsGCby77Vg90H21EOLACvtfcHk0ljIvs4OYTlyrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Skur3E&md5=42fc8bae1a6b41f2c67d5bea4355474b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DWright%26aufirst%3DN.%2BT.%26aulast%3DHoffman%26aufirst%3DC.%2BL.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DTarget%2520Binding%2520to%2520S100B%2520Reduces%2520Dynamic%2520Properties%2520and%2520Increases%2520Ca%25282%252B%2529-Binding%2520Affinity%2520for%2520Wild%2520Type%2520and%2520EF-Hand%2520Mutant%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D423%26spage%3D365%26epage%3D385%26doi%3D10.1016%2Fj.jmb.2012.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vitolo, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Complex Formation between S100B Protein and the p90 Ribosomal S6 Kinase (RSK) in Malignant Melanoma Is Calcium-dependent and Inhibits Extracellular Signal-regulated Kinase (ERK)-mediated Phosphorylation of RSK</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">12886</span><span class="NLM_x">–</span> <span class="NLM_lpage">12895</span><span class="refDoi"> DOI: 10.1074/jbc.M114.561613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1074%2Fjbc.M114.561613" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=12886-12895&author=K.+G.+Hartmanauthor=M.+I.+Vitoloauthor=A.+D.+Pierceauthor=J.+M.+Foxauthor=P.+Shapiroauthor=S.+S.+Martinauthor=P.+T.+Wilderauthor=D.+J.+Weber&title=Complex+Formation+between+S100B+Protein+and+the+p90+Ribosomal+S6+Kinase+%28RSK%29+in+Malignant+Melanoma+Is+Calcium-dependent+and+Inhibits+Extracellular+Signal-regulated+Kinase+%28ERK%29-mediated+Phosphorylation+of+RSK&doi=10.1074%2Fjbc.M114.561613"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.561613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.561613%26sid%3Dliteratum%253Aachs%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DVitolo%26aufirst%3DM.%2BI.%26aulast%3DPierce%26aufirst%3DA.%2BD.%26aulast%3DFox%26aufirst%3DJ.%2BM.%26aulast%3DShapiro%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BS.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DComplex%2520Formation%2520between%2520S100B%2520Protein%2520and%2520the%2520p90%2520Ribosomal%2520S6%2520Kinase%2520%2528RSK%2529%2520in%2520Malignant%2520Melanoma%2520Is%2520Calcium-dependent%2520and%2520Inhibits%2520Extracellular%2520Signal-regulated%2520Kinase%2520%2528ERK%2529-mediated%2520Phosphorylation%2520of%2520RSK%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D12886%26epage%3D12895%26doi%3D10.1074%2Fjbc.M114.561613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Bachman, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sager, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheong, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span> </span><span class="NLM_article-title">Identification of compounds that inhibit growth of 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine-resistant cancer cells</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1030</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-05-0038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1158%2F1535-7163.MCT-05-0038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=15956261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1026-1030&author=K.+E.+Bachmanauthor=J.+Sagerauthor=I.+Cheongauthor=M.+Cattoauthor=A.+Bardelliauthor=B.+H.+Parkauthor=B.+Vogelsteinauthor=A.+Carottiauthor=K.+W.+Kinzlerauthor=C.+Lengauer&title=Identification+of+compounds+that+inhibit+growth+of+2-amino-1-methyl-6-phenylimidazo%284%2C5-b%29pyridine-resistant+cancer+cells&doi=10.1158%2F1535-7163.MCT-05-0038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of compounds that inhibit growth of 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine-resistant cancer cells</span></div><div class="casAuthors">Bachman, Kurtis E.; Sager, Jason; Cheong, Ian; Catto, Marco; Bardelli, Alberto; Park, Ben Ho; Vogelstein, Bert; Carotti, Angelo; Kinzler, Kenneth W.; Lengauer, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1026-1030</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The dietary carcinogen 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine (PhIP) is a heterocyclic amine and is a common byproduct of cooked meat and fish.  Although most cells undergo apoptosis when exposed to this mutagen, subsets develop resistance.  Rather than die, these resistant cells persist and accumulate mutations, thereby driving tumorigenesis of exposed organs within the gastrointestinal tract.  By applying a high-throughput cell-based screen of 32,000 small mols., we have identified a family of compds. that specifically inhibit the growth of PhIP-resistant cancer cells.  These compds. may prove useful for the treatment or prevention of gastrointestinal tumors arising after exposure to PhIP and related carcinogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4mDWR1eMdSrVg90H21EOLACvtfcHk0lhmOGxdwbyahA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLc%253D&md5=96a2eb4d56adf22774222dd648576c66</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-05-0038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-05-0038%26sid%3Dliteratum%253Aachs%26aulast%3DBachman%26aufirst%3DK.%2BE.%26aulast%3DSager%26aufirst%3DJ.%26aulast%3DCheong%26aufirst%3DI.%26aulast%3DCatto%26aufirst%3DM.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520compounds%2520that%2520inhibit%2520growth%2520of%25202-amino-1-methyl-6-phenylimidazo%25284%252C5-b%2529pyridine-resistant%2520cancer%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2005%26volume%3D4%26spage%3D1026%26epage%3D1030%26doi%3D10.1158%2F1535-7163.MCT-05-0038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Myers, M. A.</span><span> </span><span class="NLM_article-title">Direct measurement of cell numbers in microtitre plate cultures using the fluorescent dye SYBR green I</span> <span class="citation_source-journal">J. Immunol. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">212</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1016/S0022-1759(98)00011-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2FS0022-1759%2898%2900011-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=9671157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADyaK1cXjtVKlsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=212&publication_year=1998&pages=99-103&author=M.+A.+Myers&title=Direct+measurement+of+cell+numbers+in+microtitre+plate+cultures+using+the+fluorescent+dye+SYBR+green+I&doi=10.1016%2FS0022-1759%2898%2900011-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of cell numbers in microtitre plate cultures using the fluorescent dye SYBR green I</span></div><div class="casAuthors">Myers, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunological Methods</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">JIMMBG</span>;
        ISSN:<span class="NLM_cas:issn">0022-1759</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The quantitation of cell nos. is essential for many exptl. procedures.  This study describes the use of the fluorescent DNA binding dye, SYBR green I, to quantitate accurately cell nos. in the wells of microtitre plates by detn. of DNA content.  The assay involves a single-step procedure and is sensitive to as few as 1000 cells/well.  The advantages of SYBR green I are the low level of background fluorescence in the absence of DNA and the similarities of its absorption and emission spectra to those of fluorescein, which makes it possible to use a wide range of equipment for detection of the amt. of bound dye.  A particular application of the method is the normalization of data generated by enzyme-linked immunosorbent assays on whole cells.  The procedure is applicable to both fixed and unfixed cells, although for fixed cells permeabilisation provides for greater sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJmFe4xjlzzbVg90H21EOLACvtfcHk0lhmOGxdwbyahA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtVKlsbk%253D&md5=4802e4136dc659cbb2120d00a382d423</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS0022-1759%2898%2900011-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-1759%252898%252900011-8%26sid%3Dliteratum%253Aachs%26aulast%3DMyers%26aufirst%3DM.%2BA.%26atitle%3DDirect%2520measurement%2520of%2520cell%2520numbers%2520in%2520microtitre%2520plate%2520cultures%2520using%2520the%2520fluorescent%2520dye%2520SYBR%2520green%2520I%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1998%26volume%3D212%26spage%3D99%26epage%3D103%26doi%3D10.1016%2FS0022-1759%2898%2900011-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Drohat, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amburgey, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abildgaard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starich, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldisseri, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Solution structure of rat apo-S100B(beta beta) as determined by NMR spectroscopy</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">11577</span><span class="NLM_x">–</span> <span class="NLM_lpage">11588</span><span class="refDoi"> DOI: 10.1021/bi9612226</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9612226" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=11577-11588&author=A.+C.+Drohatauthor=J.+C.+Amburgeyauthor=F.+Abildgaardauthor=M.+R.+Starichauthor=D.+Baldisseriauthor=D.+J.+Weber&title=Solution+structure+of+rat+apo-S100B%28beta+beta%29+as+determined+by+NMR+spectroscopy&doi=10.1021%2Fbi9612226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fbi9612226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9612226%26sid%3Dliteratum%253Aachs%26aulast%3DDrohat%26aufirst%3DA.%2BC.%26aulast%3DAmburgey%26aufirst%3DJ.%2BC.%26aulast%3DAbildgaard%26aufirst%3DF.%26aulast%3DStarich%26aufirst%3DM.%2BR.%26aulast%3DBaldisseri%26aufirst%3DD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DSolution%2520structure%2520of%2520rat%2520apo-S100B%2528beta%2520beta%2529%2520as%2520determined%2520by%2520NMR%2520spectroscopy%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26spage%3D11577%26epage%3D11588%26doi%3D10.1021%2Fbi9612226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Amburgey, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abildgaard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starich, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilt, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">1H, 13C and 15N NMR assignments and solution secondary structure of rat Apo-S100 beta</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1007/BF00211781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1007%2FBF00211781" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=171-179&author=J.+C.+Amburgeyauthor=F.+Abildgaardauthor=M.+R.+Starichauthor=S.+Shahauthor=D.+C.+Hiltauthor=D.+J.+Weber&title=1H%2C+13C+and+15N+NMR+assignments+and+solution+secondary+structure+of+rat+Apo-S100+beta&doi=10.1007%2FBF00211781"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2FBF00211781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00211781%26sid%3Dliteratum%253Aachs%26aulast%3DAmburgey%26aufirst%3DJ.%2BC.%26aulast%3DAbildgaard%26aufirst%3DF.%26aulast%3DStarich%26aufirst%3DM.%2BR.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DHilt%26aufirst%3DD.%2BC.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3D1H%252C%252013C%2520and%252015N%2520NMR%2520assignments%2520and%2520solution%2520secondary%2520structure%2520of%2520rat%2520Apo-S100%2520beta%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D171%26epage%3D179%26doi%3D10.1007%2FBF00211781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span> </span><span class="NLM_article-title">An automated system to mount cryo-cooled protein crystals on a synchrotron beamline, using compact sample cassettes and a small-scale robot</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">726</span><span class="refDoi"> DOI: 10.1107/S0021889802016709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0021889802016709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24899734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD38Xos1emtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2002&pages=720-726&author=A.+E.+Cohenauthor=P.+J.+Ellisauthor=M.+D.+Millerauthor=A.+M.+Deaconauthor=R.+P.+Phizackerley&title=An+automated+system+to+mount+cryo-cooled+protein+crystals+on+a+synchrotron+beamline%2C+using+compact+sample+cassettes+and+a+small-scale+robot&doi=10.1107%2FS0021889802016709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An automated system to mount cryo-cooled protein crystals on a synchrotron beamline, using compact sample cassettes and a small-scale robot</span></div><div class="casAuthors">Cohen, Aina E.; Ellis, Paul J.; Miller, Mitchell D.; Deacon, Ashley M.; Phizackerley, R. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">720-726</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">An automated system for mounting and dismounting pre-frozen crystals has been implemented at the Stanford Synchrotron Radiation Lab. (SSRL).  It is based on a small industrial robot and compact cylindrical cassettes, each holding up to 96 crystals mounted on Hampton Research sample pins.  For easy shipping and storage, the cassette fits inside several popular dry-shippers and long-term storage Dewars.  A dispensing Dewar holds up to three cassettes in liq. nitrogen adjacent to the beamline goniometer.  The robot uses a permanent magnet tool to ext. samples from, and insert samples into a cassette, and a cryo-tong tool to transfer them to and from the beamline goniometer.  The system is simple, with few moving parts, reliable in operation and convenient to use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoreUoI2UBMKbVg90H21EOLACvtfcHk0lik5tttYTWdhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xos1emtL8%253D&md5=5dae59f9642fa186e2cd42bce3f76ee2</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0021889802016709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889802016709%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26atitle%3DAn%2520automated%2520system%2520to%2520mount%2520cryo-cooled%2520protein%2520crystals%2520on%2520a%2520synchrotron%2520beamline%252C%2520using%2520compact%2520sample%2520cassettes%2520and%2520a%2520small-scale%2520robot%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2002%26volume%3D35%26spage%3D720%26epage%3D726%26doi%3D10.1107%2FS0021889802016709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Soltis, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eriksson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollenbeck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathews, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moorhead, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van dem Bedem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsyba, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, G.</span><span> </span><span class="NLM_article-title">New paradigm for macromolecular crystallography experiments at SSRL: automated crystal screening and remote data collection</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">1210</span><span class="NLM_x">–</span> <span class="NLM_lpage">1221</span><span class="refDoi"> DOI: 10.1107/S0907444908030564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444908030564" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=1210-1221&author=S.+M.+Soltisauthor=A.+E.+Cohenauthor=A.+Deaconauthor=T.+Erikssonauthor=A.+Gonzalezauthor=S.+McPhillipsauthor=H.+Chuiauthor=P.+Duntenauthor=M.+Hollenbeckauthor=I.+Mathewsauthor=M.+Millerauthor=P.+Moorheadauthor=R.+P.+Phizackerleyauthor=C.+Smithauthor=J.+Songauthor=H.+van+dem+Bedemauthor=P.+Ellisauthor=P.+Kuhnauthor=T.+McPhillipsauthor=N.+Sauterauthor=K.+Sharpauthor=I.+Tsybaauthor=G.+Wolf&title=New+paradigm+for+macromolecular+crystallography+experiments+at+SSRL%3A+automated+crystal+screening+and+remote+data+collection&doi=10.1107%2FS0907444908030564"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444908030564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444908030564%26sid%3Dliteratum%253Aachs%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%26aulast%3DEriksson%26aufirst%3DT.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DMcPhillips%26aufirst%3DS.%26aulast%3DChui%26aufirst%3DH.%26aulast%3DDunten%26aufirst%3DP.%26aulast%3DHollenbeck%26aufirst%3DM.%26aulast%3DMathews%26aufirst%3DI.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DMoorhead%26aufirst%3DP.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSmith%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3Dvan%2Bdem%2BBedem%26aufirst%3DH.%26aulast%3DEllis%26aufirst%3DP.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DMcPhillips%26aufirst%3DT.%26aulast%3DSauter%26aufirst%3DN.%26aulast%3DSharp%26aufirst%3DK.%26aulast%3DTsyba%26aufirst%3DI.%26aulast%3DWolf%26aufirst%3DG.%26atitle%3DNew%2520paradigm%2520for%2520macromolecular%2520crystallography%2520experiments%2520at%2520SSRL%253A%2520automated%2520crystal%2520screening%2520and%2520remote%2520data%2520collection%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D1210%26epage%3D1221%26doi%3D10.1107%2FS0907444908030564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">McPhillips, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soltis, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span> <span class="citation_source-journal">J. Synchrotron Radiat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0lizFGpLsjzVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Gonzalez, A.; Tsai, Y.</span> <span class="citation_source-book">AUTOXDS</span>; <span class="NLM_publisher-name">Stanford University</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span><a href="http://smb.slac.stanford.edu/facilities/software/xds/#autoxds_script" class="extLink">http://smb.slac.stanford.edu/facilities/software/xds/#autoxds_script</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=A.+Gonzalez&author=Y.+Tsai&title=AUTOXDS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DA.%26btitle%3DAUTOXDS%26pub%3DStanford%2520University%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0lizFGpLsjzVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lizFGpLsjzVSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0li6LvJCgof0wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0li6LvJCgof0wQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Afonine, P. V.; Grosse-Kunstleve, R. W.; Adams, P. D.</span><span> </span><span class="NLM_article-title">The Phenix Refinement Framework</span>.  <span class="citation_source-journal">CCP4 Newsl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span>, Contribution 8.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2005&author=P.+V.+Afonine&author=R.+W.+Grosse-Kunstleve&author=P.+D.+Adams&title=The+Phenix+Refinement+Framework"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26atitle%3DThe%2520Phenix%2520Refinement%2520Framework%26jtitle%3DCCP4%2520Newsl.%26date%3D2005%26volume%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Computational fragment-based binding site identification by ligand competitive saturation</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000435</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1371%2Fjournal.pcbi.1000435" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19593374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BD1MvnvVGmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000435&author=O.+Guvenchauthor=A.+D.+MacKerell&title=Computational+fragment-based+binding+site+identification+by+ligand+competitive+saturation&doi=10.1371%2Fjournal.pcbi.1000435"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Computational fragment-based binding site identification by ligand competitive saturation</span></div><div class="casAuthors">Guvench Olgun; MacKerell Alexander D Jr</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1000435</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery using NMR and x-ray crystallographic methods has proven utility but also non-trivial time, materials, and labor costs.  Current computational fragment-based approaches circumvent these issues but suffer from limited representations of protein flexibility and solvation effects, leading to difficulties with rigorous ranking of fragment affinities.  To overcome these limitations we describe an explicit solvent all-atom molecular dynamics methodology (SILCS: Site Identification by Ligand Competitive Saturation) that uses small aliphatic and aromatic molecules plus water molecules to map the affinity pattern of a protein for hydrophobic groups, aromatic groups, hydrogen bond donors, and hydrogen bond acceptors.  By simultaneously incorporating ligands representative of all these functionalities, the method is an in silico free energy-based competition assay that generates three-dimensional probability maps of fragment binding (FragMaps) indicating favorable fragment:protein interactions.  Applied to the two-fold symmetric oncoprotein BCL-6, the SILCS method yields two-fold symmetric FragMaps that recapitulate the crystallographic binding modes of the SMRT and BCOR peptides.  These FragMaps account both for important sequence and structure differences in the C-terminal halves of the two peptides and also the high mobility of the BCL-6 His116 sidechain in the peptide-binding groove.  Such SILCS FragMaps can be used to qualitatively inform the design of small-molecule inhibitors or as scoring grids for high-throughput in silico docking that incorporate both an atomic-level description of solvation and protein flexibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWmTORSXQt9tNTrvPZP1IpfW6udTcc2ebhziNYWoP_YLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvnvVGmsw%253D%253D&md5=2367585f0d41c2221a812ae552a2217a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000435%26sid%3Dliteratum%253Aachs%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DComputational%2520fragment-based%2520binding%2520site%2520identification%2520by%2520ligand%2520competitive%2520saturation%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000435%26doi%3D10.1371%2Fjournal.pcbi.1000435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.</span><span> </span><span class="NLM_article-title">Reproducing crystal binding modes of ligand functional groups using Site-Identification by Ligand Competitive Saturation (SILCS) simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">877</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1021/ci100462t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100462t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFegtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=877-896&author=E.+P.+Ramanauthor=W.+Yuauthor=O.+Guvenchauthor=A.+D.+Mackerell&title=Reproducing+crystal+binding+modes+of+ligand+functional+groups+using+Site-Identification+by+Ligand+Competitive+Saturation+%28SILCS%29+simulations&doi=10.1021%2Fci100462t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Reproducing Crystal Binding Modes of Ligand Functional Groups Using Site-Identification by Ligand Competitive Saturation (SILCS) Simulations</span></div><div class="casAuthors">Raman, E. Prabhu; Yu, Wenbo; Guvench, Olgun; MacKerell, Alexander D. Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">877-896</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The applicability of a computational method, Site Identification by Ligand Competitive Satn. (SILCS), to identify regions on a protein surface with which different types of functional groups on low-mol. wt. inhibitors interact is demonstrated.  The method involves mol. dynamics (MD) simulations of a protein in an aq. soln. of chem. diverse small mols. from which probability distributions of fragments types, termed FragMaps, are obtained.  In the present application, SILCS simulations are performed with an aq. soln. of 1 M benzene and propane to map the affinity pattern of the protein for arom. and aliph. functional groups.  In addn., water hydrogen and oxygen atoms serve as probes for hydrogen-bond donor and acceptor affinity, resp.  The method is tested using a set of 7 proteins for which crystal structures of complexes with several high affinity inhibitors are known.  Good agreement is obtained between FragMaps and the positions of chem. similar functional groups in inhibitors as obsd. in the x-ray crystallog. structures.  Quant. capabilities of the SILCS approach are demonstrated by converting FragMaps to free energies, termed Grid Free Energies (GFE), and showing correlation between the GFE values and exptl. binding affinities.  For proteins for which ligand decoy sets are available, GFE values are shown to typically score the crystal conformation and conformations similar to it more favorable than decoys.  Addnl., SILCS is tested for its ability to capture the subtle differences in ligand affinity across homologous proteins, information which may be of utility toward specificity-guided drug design.  Taken together, our results show that SILCS can recapitulate the known location of functional groups of bound inhibitors for a no. of proteins, suggesting that the method may be of utility for rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBfyWq2aa2prVg90H21EOLACvtfcHk0lgauvtym7pqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFegtr8%253D&md5=7a80452191971ec6b08086e8bce11e6f</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci100462t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100462t%26sid%3Dliteratum%253Aachs%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DReproducing%2520crystal%2520binding%2520modes%2520of%2520ligand%2520functional%2520groups%2520using%2520Site-Identification%2520by%2520Ligand%2520Competitive%2520Saturation%2520%2528SILCS%2529%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D877%26epage%3D896%26doi%3D10.1021%2Fci100462t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lgauvtym7pqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lakkaraju, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Inclusion of multiple fragment types in the site identification by ligand competitive saturation (SILCS) approach</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3384</span><span class="NLM_x">–</span> <span class="NLM_lpage">3398</span><span class="refDoi"> DOI: 10.1021/ci4005628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4005628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyqt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=3384-3398&author=E.+P.+Ramanauthor=W.+Yuauthor=S.+K.+Lakkarajuauthor=A.+D.+MacKerell&title=Inclusion+of+multiple+fragment+types+in+the+site+identification+by+ligand+competitive+saturation+%28SILCS%29+approach&doi=10.1021%2Fci4005628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Inclusion of Multiple Fragment Types in the Site Identification by Ligand Competitive Saturation (SILCS) Approach</span></div><div class="casAuthors">Raman, E. Prabhu; Yu, Wenbo; Lakkaraju, Sirish K.; MacKerell, Alexander D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3384-3398</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The site identification by ligand competitive satn. (SILCS) method identifies the location and approx. affinities of small mol. fragments on a target macromol. surface by performing mol. dynamics (MD) simulations of the target in an aq. soln. of small mols. representative of different chem. functional groups.  In this study, we introduce a set of small mols. to map potential interactions made by neutral hydrogen bond donors and acceptors and charged donor and acceptor fragments in addn. to nonpolar fragments.  The affinity pattern is obtained in the form of discretized probability or, equivalently, free energy maps, called FragMaps, which can be visualized with the target surface.  We performed SILCS simulations for four proteins for which structural and thermodn. data is available for multiple diverse ligands.  Good overlap is shown between high affinity regions identified by the FragMaps and the crystallog. positions of ligand functional groups with similar chem. functionality, thus demonstrating the validity of the qual. information obtained from the simulations.  To test the ability of FragMaps in providing quant. predictions, we calc. the previously introduced ligand grid free energy (LGFE) metric and observe its correspondence with exptl. measured binding affinity.  LGFE is computed for different conformational ensembles and improvement in prediction is shown with increasing ligand conformational sampling.  Ensemble generation includes a Monte Carlo sampling approach that uses the GFE FragMaps directly as the energy function.  The results show that some but not all exptl. trends are predicted and warrant improvements in the scoring methodol.  In addn., the potential utility of atom-based free energy contributions to the LGFE scores and the use of multiple ligands in SILCS to identify displaceable water mols. during ligand design are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCXtdZ9K_e2bVg90H21EOLACvtfcHk0lgauvtym7pqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyqt7vK&md5=6eb2518f4edbd88ff49868c91d367d73</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci4005628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4005628%26sid%3Dliteratum%253Aachs%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLakkaraju%26aufirst%3DS.%2BK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DInclusion%2520of%2520multiple%2520fragment%2520types%2520in%2520the%2520site%2520identification%2520by%2520ligand%2520competitive%2520saturation%2520%2528SILCS%2529%2520approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D3384%26epage%3D3398%26doi%3D10.1021%2Fci4005628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) I: bond perception and atom typing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3144</span><span class="NLM_x">–</span> <span class="NLM_lpage">3154</span><span class="refDoi"> DOI: 10.1021/ci300363c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300363c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3144-3154&author=K.+Vanommeslaegheauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+I%3A+bond+perception+and+atom+typing&doi=10.1021%2Fci300363c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing</span></div><div class="casAuthors">Vanommeslaeghe, K.; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3144-3154</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug-like mols. alone or interacting with biol. systems.  In simulations involving biol. macromols., the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and charges is required.  In the present article, which is part I of a series of two, we present the algorithms for bond perception and atom typing for the CHARMM General Force Field (CGenFF).  The CGenFF atom typer first assocs. attributes to the atoms and bonds in a mol., such as valence, bond order, and ring membership among others.  Of note are a no. of features that are specifically required for CGenFF.  This information is then used by the atom typing routine to assign CGenFF atom types based on a programmable decision tree.  This allows for straight-forward implementation of CGenFF's complicated atom typing rules and for equally straight-forward updating of the atom typing scheme as the force field grows.  The presented atom typer was validated by assigning correct atom types on 477 model compds. including in the training set as well as 126 test-set mols. that were constructed to specifically verify its different components.  The program may be utilized via an online implementation at https://www.paramchem.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvkdOrvB3NELVg90H21EOLACvtfcHk0liCd85Qvc2uAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL&md5=c6679293f4a2501f2bcadf2020ca1473</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci300363c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300363c%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520I%253A%2520bond%2520perception%2520and%2520atom%2520typing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3144%26epage%3D3154%26doi%3D10.1021%2Fci300363c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3155</span><span class="NLM_x">–</span> <span class="NLM_lpage">3168</span><span class="refDoi"> DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3155-3168&author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&doi=10.1021%2Fci3003649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0liCd85Qvc2uAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3155%26epage%3D3168%26doi%3D10.1021%2Fci3003649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatcher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darian, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobyov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+Mackerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0liCd85Qvc2uAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Srinivasaraghavan Kannan, Pietro G. A. Aronica, Yaw Sing Tan, <span class="NLM_string-name hlFld-ContribAuthor">Chandra S. Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibiting S100B(ββ) for Activating Wild-Type p53: Design of Stapled Peptides. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2019,</strong> <em>4 </em>
                                    (3)
                                     , 5335-5344. <a href="https://doi.org/10.1021/acsomega.9b00097" title="DOI URL">https://doi.org/10.1021/acsomega.9b00097</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b00097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b00097%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DInhibiting%252BS100B%252528%2525CE%2525B2%2525CE%2525B2%252529%252Bfor%252BActivating%252BWild-Type%252Bp53%25253A%252BDesign%252Bof%252BStapled%252BPeptides%26aulast%3DKannan%26aufirst%3DSrinivasaraghavan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D11012019%26date%3D04032019%26volume%3D4%26issue%3D3%26spage%3D5335%26epage%3D5344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yohan  Malki</span>, <span class="hlFld-ContribAuthor ">Jean  Martinez</span>, <span class="hlFld-ContribAuthor ">Nicolas  Masurier</span>. </span><span class="cited-content_cbyCitation_article-title">1,3‐Diazepine: A privileged scaffold in medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2021,</strong> <em>41 </em>
                                    (4)
                                     , 2247-2315. <a href="https://doi.org/10.1002/med.21795" title="DOI URL">https://doi.org/10.1002/med.21795</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21795%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3D1%25252C3%2525E2%252580%252590Diazepine%25253A%252BA%252Bprivileged%252Bscaffold%252Bin%252Bmedicinal%252Bchemistry%26aulast%3DMalki%26aufirst%3DYohan%26date%3D2021%26date%3D2021%26volume%3D41%26issue%3D4%26spage%3D2247%26epage%3D2315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Srinivasaraghavan  Kannan</span>, <span class="hlFld-ContribAuthor ">Pietro G. A.  Aronica</span>, <span class="hlFld-ContribAuthor ">Thanh Binh  Nguyen</span>, <span class="hlFld-ContribAuthor ">Jianguo  Li</span>, <span class="hlFld-ContribAuthor ">Chandra S.  Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Design of Macrocyclic Binders of S100B(ββ): Novel Peptide Theranostics. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (3)
                                     , 721. <a href="https://doi.org/10.3390/molecules26030721" title="DOI URL">https://doi.org/10.3390/molecules26030721</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26030721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26030721%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DComputational%252BDesign%252Bof%252BMacrocyclic%252BBinders%252Bof%252BS100B%252528%2525CE%2525B2%2525CE%2525B2%252529%25253A%252BNovel%252BPeptide%252BTheranostics%26aulast%3DKannan%26aufirst%3DSrinivasaraghavan%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D3%26spage%3D721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brianna D.  Young</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yu</span>, <span class="hlFld-ContribAuthor ">Darex J. Vera  Rodríguez</span>, <span class="hlFld-ContribAuthor ">Kristen M.  Varney</span>, <span class="hlFld-ContribAuthor ">Alexander D.  MacKerell</span>, <span class="hlFld-ContribAuthor ">David J.  Weber</span>. </span><span class="cited-content_cbyCitation_article-title">Specificity of Molecular Fragments Binding to S100B versus S100A1 as Identified by NMR and Site Identification by Ligand Competitive Saturation (SILCS). </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (2)
                                     , 381. <a href="https://doi.org/10.3390/molecules26020381" title="DOI URL">https://doi.org/10.3390/molecules26020381</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26020381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26020381%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSpecificity%252Bof%252BMolecular%252BFragments%252BBinding%252Bto%252BS100B%252Bversus%252BS100A1%252Bas%252BIdentified%252Bby%252BNMR%252Band%252BSite%252BIdentification%252Bby%252BLigand%252BCompetitive%252BSaturation%252B%252528SILCS%252529%26aulast%3DYoung%26aufirst%3DBrianna%2BD.%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D2%26spage%3D381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke-Jia  Wu</span>, <span class="hlFld-ContribAuthor ">Wanhe  Wang</span>, <span class="hlFld-ContribAuthor ">Hui-Min David  Wang</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Interfering with S100B–effector protein interactions for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (9)
                                     , 1754-1761. <a href="https://doi.org/10.1016/j.drudis.2020.07.010" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.07.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.07.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.07.010%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DInterfering%252Bwith%252BS100B%2525E2%252580%252593effector%252Bprotein%252Binteractions%252Bfor%252Bcancer%252Btherapy%26aulast%3DWu%26aufirst%3DKe-Jia%26date%3D2020%26volume%3D25%26issue%3D9%26spage%3D1754%26epage%3D1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jacques  Baudier</span>, <span class="hlFld-ContribAuthor ">Jean Christophe  Deloulme</span>, <span class="hlFld-ContribAuthor ">Gary S.  Shaw</span>. </span><span class="cited-content_cbyCitation_article-title">The Zn
              2+
              and Ca
              2+
              ‐binding S100B and S100A1 proteins: beyond the myths. </span><span class="cited-content_cbyCitation_journal-name">Biological Reviews</span><span> <strong>2020,</strong> <em>95 </em>
                                    (3)
                                     , 738-758. <a href="https://doi.org/10.1111/brv.12585" title="DOI URL">https://doi.org/10.1111/brv.12585</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/brv.12585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fbrv.12585%26sid%3Dliteratum%253Aachs%26jtitle%3DBiological%2520Reviews%26atitle%3DThe%252BZn%252B2%25252B%252Band%252BCa%252B2%25252B%252B%2525E2%252580%252590binding%252BS100B%252Band%252BS100A1%252Bproteins%25253A%252Bbeyond%252Bthe%252Bmyths%26aulast%3DBaudier%26aufirst%3DJacques%26date%3D2020%26date%3D2020%26volume%3D95%26issue%3D3%26spage%3D738%26epage%3D758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Patricia de Aguiar  Amaral</span>, <span class="hlFld-ContribAuthor ">Delphine  Autheman</span>, <span class="hlFld-ContribAuthor ">Guilherme Dias  de Melo</span>, <span class="hlFld-ContribAuthor ">Nicolas  Gouault</span>, <span class="hlFld-ContribAuthor ">Jean-François  Cupif</span>, <span class="hlFld-ContribAuthor ">Sophie  Goyard</span>, <span class="hlFld-ContribAuthor ">Patricia  Dutra</span>, <span class="hlFld-ContribAuthor ">Nicolas  Coatnoan</span>, <span class="hlFld-ContribAuthor ">Alain  Cosson</span>, <span class="hlFld-ContribAuthor ">Damien  Monet</span>, <span class="hlFld-ContribAuthor ">Frederick  Saul</span>, <span class="hlFld-ContribAuthor ">Ahmed  Haouz</span>, <span class="hlFld-ContribAuthor ">Philippe  Uriac</span>, <span class="hlFld-ContribAuthor ">Arnaud  Blondel</span>, <span class="hlFld-ContribAuthor ">Paola  Minoprio</span>, . </span><span class="cited-content_cbyCitation_article-title">Designed mono- and di-covalent inhibitors trap modeled functional motions for Trypanosoma cruzi proline racemase in crystallography. </span><span class="cited-content_cbyCitation_journal-name">PLOS Neglected Tropical Diseases</span><span> <strong>2018,</strong> <em>12 </em>
                                    (10)
                                     , e0006853. <a href="https://doi.org/10.1371/journal.pntd.0006853" title="DOI URL">https://doi.org/10.1371/journal.pntd.0006853</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pntd.0006853&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pntd.0006853%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Neglected%2520Tropical%2520Diseases%26atitle%3DDesigned%252Bmono-%252Band%252Bdi-covalent%252Binhibitors%252Btrap%252Bmodeled%252Bfunctional%252Bmotions%252Bfor%252BTrypanosoma%252Bcruzi%252Bproline%252Bracemase%252Bin%252Bcrystallography%26aulast%3DAmaral%26aufirst%3DPatricia%2Bde%2BAguiar%26date%3D2018%26date%3D2018%26volume%3D12%26issue%3D10%26spage%3De0006853" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zephan  Melville</span>, <span class="hlFld-ContribAuthor ">Ehson  Aligholizadeh</span>, <span class="hlFld-ContribAuthor ">Laura E.  McKnight</span>, <span class="hlFld-ContribAuthor ">Dylan J.  Weber</span>, <span class="hlFld-ContribAuthor ">Edwin  Pozharski</span>, <span class="hlFld-ContribAuthor ">David J.  Weber</span>. </span><span class="cited-content_cbyCitation_article-title">X-ray crystal structure of human calcium-bound S100A1. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section F Structural Biology Communications</span><span> <strong>2017,</strong> <em>73 </em>
                                    (4)
                                     , 215-221. <a href="https://doi.org/10.1107/S2053230X17003983" title="DOI URL">https://doi.org/10.1107/S2053230X17003983</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2053230X17003983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2053230X17003983%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520F%2520Structural%2520Biology%2520Communications%26atitle%3DX-ray%252Bcrystal%252Bstructure%252Bof%252Bhuman%252Bcalcium-bound%252BS100A1%26aulast%3DMelville%26aufirst%3DZephan%26date%3D2017%26date%3D2017%26volume%3D73%26issue%3D4%26spage%3D215%26epage%3D221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olgun  Guvench</span>. </span><span class="cited-content_cbyCitation_article-title">Computational functional group mapping for drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2016,</strong> <em>21 </em>
                                    (12)
                                     , 1928-1931. <a href="https://doi.org/10.1016/j.drudis.2016.06.030" title="DOI URL">https://doi.org/10.1016/j.drudis.2016.06.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2016.06.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2016.06.030%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DComputational%252Bfunctional%252Bgroup%252Bmapping%252Bfor%252Bdrug%252Bdiscovery%26aulast%3DGuvench%26aufirst%3DOlgun%26date%3D2016%26volume%3D21%26issue%3D12%26spage%3D1928%26epage%3D1931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael C.  Cavalier</span>, <span class="hlFld-ContribAuthor ">Zephan  Melville</span>, <span class="hlFld-ContribAuthor ">Ehson  Aligholizadeh</span>, <span class="hlFld-ContribAuthor ">E. Prabhu  Raman</span>, <span class="hlFld-ContribAuthor ">Wenbo  Yu</span>, <span class="hlFld-ContribAuthor ">Lei  Fang</span>, <span class="hlFld-ContribAuthor ">Milad  Alasady</span>, <span class="hlFld-ContribAuthor ">Adam D.  Pierce</span>, <span class="hlFld-ContribAuthor ">Paul T.  Wilder</span>, <span class="hlFld-ContribAuthor ">Alexander D.  MacKerell</span>, <span class="hlFld-ContribAuthor ">David J.  Weber</span>. </span><span class="cited-content_cbyCitation_article-title">Novel protein–inhibitor interactions in site 3 of Ca
              2+
              -bound S100B as discovered by X-ray crystallography. </span><span class="cited-content_cbyCitation_journal-name">Acta Crystallographica Section D Structural Biology</span><span> <strong>2016,</strong> <em>72 </em>
                                    (6)
                                     , 753-760. <a href="https://doi.org/10.1107/S2059798316005532" title="DOI URL">https://doi.org/10.1107/S2059798316005532</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1107/S2059798316005532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1107%2FS2059798316005532%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Crystallographica%2520Section%2520D%2520Structural%2520Biology%26atitle%3DNovel%252Bprotein%2525E2%252580%252593inhibitor%252Binteractions%252Bin%252Bsite%252B3%252Bof%252BCa%252B2%25252B%252B-bound%252BS100B%252Bas%252Bdiscovered%252Bby%252BX-ray%252Bcrystallography%26aulast%3DCavalier%26aufirst%3DMichael%2BC.%26date%3D2016%26date%3D2016%26volume%3D72%26issue%3D6%26spage%3D753%26epage%3D760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0009.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overall structure of S100B: (A) homodimer of S100B (blue tubes) with Ca<sup>2+</sup> bound (green spheres); (B) surface rendering of S100B with hot spots 1 (magenta), 2 (yellow), and 3 (gray) highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0010.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. SILCS FragMaps, showing crystal orientation of heptamidine (licorice, atom-type coloring) overlaid on the SILCS FragMaps calculated using the SBi132·S100B (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GK1">3GK1</a>) crystal structure. (A) FragMaps are shown for aliphatic (green), aromatic (purple), and hydrogen bond donors (blue) with the protein show in surface representation. The approximate locations of the site 1, 2, and 3 pockets are indicated. (B) Protein in cartoon representation with acidic residues in red licorice along with the hydrogen bond donors (blue). Arrow heads indicate hydrogen bond acceptor FragMaps that are adjacent to acidic residue side chains.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0003.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Preparation of Diamidines <b>4a</b>–<b>c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) CsCO<sub>3</sub>, acetone, reflux, 14 h; (b) methanol/dioxane (1:1), HCl gas; (c) ethanolic ammonia, ammonium chloride, 80 °C, 32 h.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Preparation of Cyclic and Fatty Amine Substituted Diamidine Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1,2-diaminoethane, DMF, rt; (b) 1,3-diaminopropane, DMF, rt; (c) 1,4-diaminobutane, DMF, rt; (d) 1,2-diaminopentane, DMF, rt; (e) 1,2-diaminoheptane, DMF, rt.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Preparation of <i>N</i>-Acyl Substituted Diamidine Analogs<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) phenyl isocyanate, DBU, DMF, 40 °C; (b) <i>N</i>-succinamidyl-<i>N</i>-methyl carbamate, DBU, DMF, 40 °C.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0006.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Preparation of Analogs with Central Amine Substitutions<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) for <b>13a</b>, benzylamine, <b>13b</b>, propylamine, K<sub>2</sub>CO<sub>3</sub>, acetone; (b) <b>1</b>, CsCO<sub>3</sub>, acetone; (c) methanol, HCl gas; (d) ethanolic ammonia, ammonium chloride.</p></p></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0011.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystallographic structure of the S100B·<b>5a</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN">5DKN</a>). (A) |<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>| electron density omit map (blue mesh) of <b>5a</b> (ball and stick) contoured at 2.5σ levels. (B) <b>5a</b> within sites 2 (yellow highlight) and 3 (gray highlight) of the S100B homodimer (surface in blue). (C) Modeled orientation of <b>5a</b> overlaid on the previously elucidated compound-bound structures of pentamidine and heptamidine. (D) The specific interactions of <b>5a</b> within dimeric S100B (blue ribbons) are rendered with residues (sticks) within 4 Å of the <b>5a</b> highlighted in yellow (site 2) or gray (site 3). <b>5a</b> is situated within a hydrophobic pocket with atoms within hydrogen bond distance at the backbone carbonyl of His85 and the side chain of Cys84.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0012.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystallographic structure of the S100B·<b>6b</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>). (A) |<i>F</i><sub>o</sub>|− |<i>F</i><sub>c</sub>| electron density omit map (blue mesh) of <b>6b</b> (ball and stick) contoured at 2.5σ levels. (B) <b>6b</b> within sites 2 (yellow highlight) and 3 (gray highlight) of the S100B homodimer (surface in blue). (C) Modeled orientation of <b>6b</b> overlaid on the previously elucidated compound-bound structures of pentamidine and heptamidine. (D) The specific interactions of <b>6b</b> with dimeric S100B (blue ribbons) are rendered with residues (sticks) within 4 Å of the <b>6b</b> highlighted in yellow (site 2) or gray (site 3). <b>6b</b> is situated within a hydrophobic pocket with atoms within hydrogen bond distance at the backbone carbonyl of His85 and the side chain of Cys84.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0001.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Crystallographic structure of the S100B·<b>17</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR">5DKR</a>). (A) |<i>F</i><sub>o</sub>| – |<i>F</i><sub>c</sub>| electron density omit map (blue mesh) of <b>17</b> (ball and stick) contoured at 2.5σ levels. (B) <b>17</b> within sites 2 (yellow highlight) and 3 (gray highlight) of the S100B homodimer (surface in blue). (C) Modeled orientation of <b>17</b> overlaid on the previously elucidated compound-bound structures of pentamidine and heptamidine. (D) The specific interactions of <b>17</b> with dimeric S100B (blue ribbons) are rendered with residues (sticks) within 4 Å of the <b>17</b> highlighted in yellow (site 2) or gray (site 3). Compound <b>17</b> is situated within a hydrophobic pocket with atoms within hydrogen bond distance at the backbone carbonyls of His42 and Phe43, and the side chains of Aps12 and Glu45.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/medium/jm-2015-013694_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0002.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Small molecule inhibitors of Ca<sup><b>2+</b></sup>-S100B reveal two protein conformations. (A) Overlaid crystallographic structures (tubes) of S100B in uninhibited (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MHO">1MHO</a>, blue) and pentamidine (and analog, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>) inhibited conformations. A substantial conformational shift is observed in the C-terminal loop region. (B) A close-up rendering of the C-terminal loop region reveals that Phe87 and Phe88 (sticks) are the major contributors to the conformational shift. (C) Surface rendering showing that a bound pentamidine analog (ball and stick, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>) has difficulty accessing site 1 (magenta) from site 2 (yellow) because of the obstruction caused by intruding phenylalanine side chains (green). (D) Other non-pentamidine-like inhibitors (ball and stick, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PE0">4PE0</a>) have ready access to site 1 from site 2 when the phenylalanine side chains are in a native conformation.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-2/acs.jmedchem.5b01369/20160122/images/large/jm-2015-013694_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01369&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i66">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20538" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20538" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 63 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Gogas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abali, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ascierto, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demidov, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pehamberger, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schachter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggermont, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauschild, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espinosa, E.</span><span> </span><span class="NLM_article-title">Who Benefits Most From Adjuvant Interferon Treatment for Melanoma?</span> <span class="citation_source-journal">Am. J. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">54</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1097/MJT.0b013e31829e883d</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1097%2FMJT.0b013e31829e883d" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24176884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC2c7hsV2iuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=54-60&author=H.+Gogasauthor=H.+Abaliauthor=P.+A.+Asciertoauthor=L.+Demidovauthor=H.+Pehambergerauthor=C.+Robertauthor=J.+Schachterauthor=A.+M.+Eggermontauthor=A.+Hauschildauthor=E.+Espinosa&title=Who+Benefits+Most+From+Adjuvant+Interferon+Treatment+for+Melanoma%3F&doi=10.1097%2FMJT.0b013e31829e883d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Who benefits most from adjuvant interferon treatment for melanoma?</span></div><div class="casAuthors">Gogas Helen; Abali Huseyin; Ascierto Paolo A; Demidov Lev; Pehamberger Hubert; Robert Caroline; Schachter Jacob; Eggermont Alexander M M; Hauschild Axel; Espinosa Enrique</div><div class="citationInfo"><span class="NLM_cas:title">American journal of therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-60</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Metastatic melanoma has a poor prognosis; the median survival for patients with stage IV melanoma ranges from 8 to 18 months after diagnosis.  Interferon-α provides significant improvement in disease-free survival at the cost of poor tolerability.  Identifying patients who benefit the most may improve the cost:benefit ratio.  In addition, no data exist for the role of adjuvant therapy in noncutaneous melanoma.  Molecular profiles may help to identify patients who benefit the most from adjuvant interferon therapy.  In this review, the American Joint Commission on Cancer 2009 staging criteria and emerging biomarker data to guide adjuvant treatment decisions will be discussed.  Several criteria to guide selection of patients are discussed in detail.  These include Breslow thickness, number of positive lymph nodes, whether or not the primary lesion has ulcerated, immunologic markers, and cytokine profiles.  Substantial progress has been made in deciding which patients benefit from interferon-α adjuvant therapy.  Interferon-α is the only agent currently approved for the adjuvant treatment of this deadly disease, despite its side effect profile.  More effective drugs with better tolerability are needed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ-C19TuUAlt2Zyxfu64m4ffW6udTcc2eYqp6BtBT1zy7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7hsV2iuw%253D%253D&md5=36ae877eaa1be5340ed4296d5f2a0e37</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1097%2FMJT.0b013e31829e883d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMJT.0b013e31829e883d%26sid%3Dliteratum%253Aachs%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DAbali%26aufirst%3DH.%26aulast%3DAscierto%26aufirst%3DP.%2BA.%26aulast%3DDemidov%26aufirst%3DL.%26aulast%3DPehamberger%26aufirst%3DH.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DSchachter%26aufirst%3DJ.%26aulast%3DEggermont%26aufirst%3DA.%2BM.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DEspinosa%26aufirst%3DE.%26atitle%3DWho%2520Benefits%2520Most%2520From%2520Adjuvant%2520Interferon%2520Treatment%2520for%2520Melanoma%253F%26jtitle%3DAm.%2520J.%2520Ther.%26date%3D2015%26volume%3D22%26spage%3D54%26epage%3D60%26doi%3D10.1097%2FMJT.0b013e31829e883d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Jang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkins, M. B.</span><span> </span><span class="NLM_article-title">Which drug, and when, for patients with BRAF-mutant melanoma?</span> <span class="citation_source-journal">Lancet Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">e60</span><span class="NLM_x">–</span> <span class="NLM_lpage">69</span><span class="refDoi"> DOI: 10.1016/S1470-2045(12)70539-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2FS1470-2045%2812%2970539-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=e60-69&author=S.+Jangauthor=M.+B.+Atkins&title=Which+drug%2C+and+when%2C+for+patients+with+BRAF-mutant+melanoma%3F&doi=10.1016%2FS1470-2045%2812%2970539-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1470-2045%2812%2970539-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1470-2045%252812%252970539-9%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DS.%26aulast%3DAtkins%26aufirst%3DM.%2BB.%26atitle%3DWhich%2520drug%252C%2520and%2520when%252C%2520for%2520patients%2520with%2520BRAF-mutant%2520melanoma%253F%26jtitle%3DLancet%2520Oncol.%26date%3D2013%26volume%3D14%26spage%3De60%26epage%3D69%26doi%3D10.1016%2FS1470-2045%2812%2970539-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Topalian, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodi, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahmer, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gettinger, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDermott, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powderly, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carvajal, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sosman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atkins, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leming, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spigel, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Antonia, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horn, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drake, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardoll, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharfman, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anders, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taube, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMiller, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korman, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jure-Kunkel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agrawal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollia, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigginton, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sznol, M.</span><span> </span><span class="NLM_article-title">Safety, activity, and immune correlates of anti-PD-1 antibody in cancer</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">366</span><span class="NLM_x">, </span> <span class="NLM_fpage">2443</span><span class="NLM_x">–</span> <span class="NLM_lpage">2454</span><span class="refDoi"> DOI: 10.1056/NEJMoa1200690</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1056%2FNEJMoa1200690" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=22658127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2rs7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2012&pages=2443-2454&author=S.+L.+Topalianauthor=F.+S.+Hodiauthor=J.+R.+Brahmerauthor=S.+N.+Gettingerauthor=D.+C.+Smithauthor=D.+F.+McDermottauthor=J.+D.+Powderlyauthor=R.+D.+Carvajalauthor=J.+A.+Sosmanauthor=M.+B.+Atkinsauthor=P.+D.+Lemingauthor=D.+R.+Spigelauthor=S.+J.+Antoniaauthor=L.+Hornauthor=C.+G.+Drakeauthor=D.+M.+Pardollauthor=L.+Chenauthor=W.+H.+Sharfmanauthor=R.+A.+Andersauthor=J.+M.+Taubeauthor=T.+L.+McMillerauthor=H.+Xuauthor=A.+J.+Kormanauthor=M.+Jure-Kunkelauthor=S.+Agrawalauthor=D.+McDonaldauthor=G.+D.+Kolliaauthor=A.+Guptaauthor=J.+M.+Wiggintonauthor=M.+Sznol&title=Safety%2C+activity%2C+and+immune+correlates+of+anti-PD-1+antibody+in+cancer&doi=10.1056%2FNEJMoa1200690"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Safety, activity, and immune correlates of anti-PD-1 antibody in cancer</span></div><div class="casAuthors">Topalian, Suzanne L.; Hodi, F. Stephen; Brahmer, Julie R.; Gettinger, Scott N.; Smith, David C.; McDermott, David F.; Powderly, John D.; Carvajal, Richard D.; Sosman, Jeffrey A.; Atkins, Michael B.; Leming, Philip D.; Spigel, David R.; Antonia, Scott J.; Horn, Leora; Drake, Charles G.; Pardoll, Drew M.; Chen, Lieping; Sharfman, William H.; Anders, Robert A.; Taube, Janis M.; McMiller, Tracee L.; Xu, Haiying; Korman, Alan J.; Jure-Kunkel, Maria; Agrawal, Shruti; McDonald, Daniel; Kollia, Georgia D.; Gupta, Ashok; Wigginton, Jon M.; Sznol, Mario</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">366</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">2443-2454</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">Background: Blockade of programmed death 1 (PD-1), an inhibitory receptor expressed by T cells, can overcome immune resistance.  We assessed the antitumor activity and safety of BMS-936558, an antibody that specifically blocks PD-1.  Methods: We enrolled patients with advanced melanoma, non-small-cell lung cancer, castration-resistant prostate cancer, or renal-cell or colorectal cancer to receive anti-PD-1 antibody at a dose of 0.1 to 10.0 mg per kg of body wt. every 2 wk.  Response was assessed after each 8-wk treatment cycle.  Patients received up to 12 cycles until disease progression or a complete response occurred.  Results: A total of 296 patients received treatment through Feb. 24, 2012.  Grade 3 or 4 drug-related adverse events occurred in 14% of patients; there were three deaths from pulmonary toxicity.  No max. tolerated dose was defined.  Adverse events consistent with immune-related causes were obsd.  Among 236 patients in whom response could be evaluated, objective responses (complete or partial responses) were obsd. in those with non-small-cell lung cancer, melanoma, or renal-cell cancer.  Cumulative response rates (all doses) were 18% among patients with non-small-cell lung cancer (14 of 76 patients), 28% among patients with melanoma (26 of 94 patients), and 27% among patients with renal-cell cancer (9 of 33 patients).  Responses were durable; 20 of 31 responses lasted 1 yr or more in patients with 1 yr or more of follow-up.  To assess the role of intratumoral PD-1 ligand (PD-L1) expression in the modulation of the PD-1-PD-L1 pathway, immunohistochem. anal. was performed on pretreatment tumor specimens obtained from 42 patients.  Of 17 patients with PD-L1-neg. tumors, none had an objective response; 9 of 25 patients (36%) with PD-L1-pos. tumors had an objective response (P = 0.006).  Conclusions: Anti-PD-1 antibody produced objective responses in approx. one in four to one in five patients with non-small-cell lung cancer, melanoma, or renal-cell cancer; the adverse-event profile does not appear to preclude its use.  Preliminary data suggest a relationship between PD-L1 expression on tumor cells and objective response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLOPvHybmlvbVg90H21EOLACvtfcHk0liigD_4AAjk1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2rs7fN&md5=7c13f228ef92b6089982fc878d3a394b</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa1200690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa1200690%26sid%3Dliteratum%253Aachs%26aulast%3DTopalian%26aufirst%3DS.%2BL.%26aulast%3DHodi%26aufirst%3DF.%2BS.%26aulast%3DBrahmer%26aufirst%3DJ.%2BR.%26aulast%3DGettinger%26aufirst%3DS.%2BN.%26aulast%3DSmith%26aufirst%3DD.%2BC.%26aulast%3DMcDermott%26aufirst%3DD.%2BF.%26aulast%3DPowderly%26aufirst%3DJ.%2BD.%26aulast%3DCarvajal%26aufirst%3DR.%2BD.%26aulast%3DSosman%26aufirst%3DJ.%2BA.%26aulast%3DAtkins%26aufirst%3DM.%2BB.%26aulast%3DLeming%26aufirst%3DP.%2BD.%26aulast%3DSpigel%26aufirst%3DD.%2BR.%26aulast%3DAntonia%26aufirst%3DS.%2BJ.%26aulast%3DHorn%26aufirst%3DL.%26aulast%3DDrake%26aufirst%3DC.%2BG.%26aulast%3DPardoll%26aufirst%3DD.%2BM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSharfman%26aufirst%3DW.%2BH.%26aulast%3DAnders%26aufirst%3DR.%2BA.%26aulast%3DTaube%26aufirst%3DJ.%2BM.%26aulast%3DMcMiller%26aufirst%3DT.%2BL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DKorman%26aufirst%3DA.%2BJ.%26aulast%3DJure-Kunkel%26aufirst%3DM.%26aulast%3DAgrawal%26aufirst%3DS.%26aulast%3DMcDonald%26aufirst%3DD.%26aulast%3DKollia%26aufirst%3DG.%2BD.%26aulast%3DGupta%26aufirst%3DA.%26aulast%3DWigginton%26aufirst%3DJ.%2BM.%26aulast%3DSznol%26aufirst%3DM.%26atitle%3DSafety%252C%2520activity%252C%2520and%2520immune%2520correlates%2520of%2520anti-PD-1%2520antibody%2520in%2520cancer%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2012%26volume%3D366%26spage%3D2443%26epage%3D2454%26doi%3D10.1056%2FNEJMoa1200690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Marcus, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crocker, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelton, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maynard, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delman, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlson, G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, M.</span><span> </span><span class="NLM_article-title">Prognostic Factors for Overall Survival After Radiosurgery for Brain Metastases From Melanoma</span> <span class="citation_source-journal">Am. J. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">580</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span><span class="refDoi"> DOI: 10.1097/COC.0b013e318280d7be</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1097%2FCOC.0b013e318280d7be" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23428955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC3svgt1ehsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2014&pages=580-584&author=D.+M.+Marcusauthor=M.+Loweauthor=M.+K.+Khanauthor=D.+H.+Lawsonauthor=I.+R.+Crockerauthor=J.+W.+Sheltonauthor=A.+Meltonauthor=N.+Maynardauthor=K.+A.+Delmanauthor=G.+W.+Carlsonauthor=M.+Rizzo&title=Prognostic+Factors+for+Overall+Survival+After+Radiosurgery+for+Brain+Metastases+From+Melanoma&doi=10.1097%2FCOC.0b013e318280d7be"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic factors for overall survival after radiosurgery for brain metastases from melanoma</span></div><div class="casAuthors">Marcus David M; Lowe Michael; Khan Mohammad K; Lawson David H; Crocker Ian R; Shelton Joseph W; Melton Alisa; Maynard Necia; Delman Keith A; Carlson Grant W; Rizzo Monica</div><div class="citationInfo"><span class="NLM_cas:title">American journal of clinical oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">580-4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">OBJECTIVES:  Brain metastases (BM) cause significant morbidity and mortality in patients with melanoma.  We aimed to identify prognostic factors for overall survival (OS) in patients undergoing stereotactic radiosurgery (SRS) for BM from melanoma.  METHODS:  We identified 135 patients treated with SRS at Emory University between 1998 and 2010 for BM from melanoma.  We recorded patient age, number and size of all BM, Karnofsky Performance Status (KPS), presence of extracranial metastases, serum lactate dehydrogenase (LDH), use of whole-brain radiation therapy (WBRT), use of temozolomide, and surgical resection of BM.  We used the Kaplan-Meier method to calculate OS, and we compared time-to-event data with the log-rank test.  We performed Cox multivariate analysis to identify factors independently associated with OS.  RESULTS:  Median OS for all patients was 6.9 months.  Patients with KPS ≥ 90, 70 to 80, and <70 had median OS of 10.4, 6.1, and 4.5 months, respectively (P=0.02).  Patients with LDH<240 had median OS of 7.8 months versus 3.5 months for LDH ≥ 240 (P=0.01).  Patients receiving WBRT had median OS of 7.3 months versus 6.5 months for patients not receiving WBRT (P=0.05).  KPS and LDH (but not WBRT) were significantly associated with OS on multivariate analysis.  CONCLUSIONS:  In addition to previously identified prognostic factors for OS in patients with BM from melanoma, serum LDH is independently associated with OS.  If this finding is confirmed in a prospective manner, the serum LDH level should be included in future prognostic algorithms for patients with melanoma and BM who are to receive SRS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSvRw4O17JYEy86Js-BCLiifW6udTcc2eapEY9z4DUTubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3svgt1ehsg%253D%253D&md5=223c0630464123049e9164ff79e4b1e8</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1097%2FCOC.0b013e318280d7be&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCOC.0b013e318280d7be%26sid%3Dliteratum%253Aachs%26aulast%3DMarcus%26aufirst%3DD.%2BM.%26aulast%3DLowe%26aufirst%3DM.%26aulast%3DKhan%26aufirst%3DM.%2BK.%26aulast%3DLawson%26aufirst%3DD.%2BH.%26aulast%3DCrocker%26aufirst%3DI.%2BR.%26aulast%3DShelton%26aufirst%3DJ.%2BW.%26aulast%3DMelton%26aufirst%3DA.%26aulast%3DMaynard%26aufirst%3DN.%26aulast%3DDelman%26aufirst%3DK.%2BA.%26aulast%3DCarlson%26aufirst%3DG.%2BW.%26aulast%3DRizzo%26aufirst%3DM.%26atitle%3DPrognostic%2520Factors%2520for%2520Overall%2520Survival%2520After%2520Radiosurgery%2520for%2520Brain%2520Metastases%2520From%2520Melanoma%26jtitle%3DAm.%2520J.%2520Clin.%2520Oncol.%26date%3D2014%26volume%3D37%26spage%3D580%26epage%3D584%26doi%3D10.1097%2FCOC.0b013e318280d7be" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Espinosa, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grob, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dummer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rutkowski, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gogas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kefford, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggermont, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin Algarra, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauschild, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schadendorf, D.</span><span> </span><span class="NLM_article-title">Treatment Algorithms in Stage IV Melanoma</span> <span class="citation_source-journal">Am. J. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">61</span><span class="NLM_x">–</span> <span class="NLM_lpage">67</span><span class="refDoi"> DOI: 10.1097/MJT.0b013e31829e885c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1097%2FMJT.0b013e31829e885c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24413374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC2czksFWjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=61-67&author=E.+Espinosaauthor=J.+J.+Grobauthor=R.+Dummerauthor=P.+Rutkowskiauthor=C.+Robertauthor=H.+Gogasauthor=R.+Keffordauthor=A.+M.+Eggermontauthor=S.+Martin+Algarraauthor=A.+Hauschildauthor=D.+Schadendorf&title=Treatment+Algorithms+in+Stage+IV+Melanoma&doi=10.1097%2FMJT.0b013e31829e885c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Treatment algorithms in stage IV melanoma</span></div><div class="casAuthors">Espinosa Enrique; Grob Jean-Jacques; Dummer Reinhard; Rutkowski Piotr; Robert Caroline; Gogas Helen; Kefford Richard; Eggermont Alexander M M; Martin Algarra Salvador; Hauschild Axel; Schadendorf Dirk</div><div class="citationInfo"><span class="NLM_cas:title">American journal of therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">61-7</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The molecular classification of melanoma and the advent of new drugs are changing the paradigm of therapy for advanced melanoma.  A review of the recent key studies was performed, followed by a discussion in an expert forum.  The aim of this review was to generate a therapeutic algorithm for stage IV melanoma.  Tumor genotyping for BRAF and/or KIT should be performed before selection of therapy.  For most BRAF-mutated melanoma patients and particularly those with a high tumor load, vemurafenib or other BRAF inhibitors such as dabrafenib are the treatment of choice.  KIT inhibitors can be effective in KIT-mutant tumors, especially in those patients with mutations at exons 11 and 13.  Ipilimumab is a good option for patients with nontargetable or nondetected mutations and those who progress under therapy with vemurafenib or a KIT inhibitor.  There is still a role for conventional chemotherapy either as first-line treatment in BRAF wild-type patients or as salvage therapy in second or third line, or after other treatment modalities.  Participation in clinical trials is strongly encouraged, either in first or in subsequent lines.  New therapeutic options for advanced melanoma are guided by tumor genotyping.  The current therapeutic algorithm includes kinase inhibitors, anti-CTLA4 therapy, immunotherapy, and chemotherapy, depending on the tumor genotype and response to previous treatments.  Participation in clinical trials should always be encouraged because the treatment goal is long-term survival and potential cure in a subset of patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3HXRikISteBFwe7cAcrfPfW6udTcc2eYhpUX6SpbaB7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czksFWjuw%253D%253D&md5=8bd49605b84ed3c186be1944c75661c3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1097%2FMJT.0b013e31829e885c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMJT.0b013e31829e885c%26sid%3Dliteratum%253Aachs%26aulast%3DEspinosa%26aufirst%3DE.%26aulast%3DGrob%26aufirst%3DJ.%2BJ.%26aulast%3DDummer%26aufirst%3DR.%26aulast%3DRutkowski%26aufirst%3DP.%26aulast%3DRobert%26aufirst%3DC.%26aulast%3DGogas%26aufirst%3DH.%26aulast%3DKefford%26aufirst%3DR.%26aulast%3DEggermont%26aufirst%3DA.%2BM.%26aulast%3DMartin%2BAlgarra%26aufirst%3DS.%26aulast%3DHauschild%26aufirst%3DA.%26aulast%3DSchadendorf%26aufirst%3DD.%26atitle%3DTreatment%2520Algorithms%2520in%2520Stage%2520IV%2520Melanoma%26jtitle%3DAm.%2520J.%2520Ther.%26date%3D2015%26volume%3D22%26spage%3D61%26epage%3D67%26doi%3D10.1097%2FMJT.0b013e31829e885c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Lee, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sahasranaman, S.</span><span> </span><span class="NLM_article-title">Immune checkpoint inhibitors: An introduction to the next generation cancer immunotherapy</span> <span class="citation_source-journal">J. Clin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span><span class="NLM_elocation-id"></span><span class="refDoi"> DOI: 10.1002/jcph.591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fjcph.591" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=L.+Leeauthor=M.+Guptaauthor=S.+Sahasranaman&title=Immune+checkpoint+inhibitors%3A+An+introduction+to+the+next+generation+cancer+immunotherapy&doi=10.1002%2Fjcph.591"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fjcph.591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcph.591%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DL.%26aulast%3DGupta%26aufirst%3DM.%26aulast%3DSahasranaman%26aufirst%3DS.%26atitle%3DImmune%2520checkpoint%2520inhibitors%253A%2520An%2520introduction%2520to%2520the%2520next%2520generation%2520cancer%2520immunotherapy%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2015%26doi%3D10.1002%2Fjcph.591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Mahoney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rennert, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, G. J.</span><span> </span><span class="NLM_article-title">Combination cancer immunotherapy and new immunomodulatory targets</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">584</span><span class="refDoi"> DOI: 10.1038/nrd4591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1038%2Fnrd4591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=26228759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1KiurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=561-584&author=K.+M.+Mahoneyauthor=P.+D.+Rennertauthor=G.+J.+Freeman&title=Combination+cancer+immunotherapy+and+new+immunomodulatory+targets&doi=10.1038%2Fnrd4591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Combination cancer immunotherapy and new immunomodulatory targets</span></div><div class="casAuthors">Mahoney, Kathleen M.; Rennert, Paul D.; Freeman, Gordon J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">561-584</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeting immune checkpoints such as programmed cell death protein 1 (PD1), programmed cell death 1 ligand 1 (PDL1) and cytotoxic T lymphocyte antigen 4 (CTLA4) has achieved noteworthy benefit in multiple cancers by blocking immunoinhibitory signals and enabling patients to produce an effective antitumor response.  Inhibitors of CTLA4, PD1 or PDL1 administered as single agents have resulted in durable tumor regression in some patients, and combinations of PD1 and CTLA4 inhibitors may enhance antitumor benefit.  Numerous addnl. immunomodulatory pathways as well as inhibitory factors expressed or secreted by myeloid and stromal cells in the tumor microenvironment are potential targets for synergizing with immune checkpoint blockade.  Given the breadth of potential targets in the immune system, crit. questions to address include which combinations should move forward in development and which patients will benefit from these treatments.  This Review discusses the leading drug targets that are expressed on tumor cells and in the tumor microenvironment that allow enhancement of the antitumor immune response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwi7ChUpfjArVg90H21EOLACvtfcHk0lhqQPNbx0qqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1KiurrJ&md5=b0b598b2a7c62794b4f03c4cea196a97</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd4591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4591%26sid%3Dliteratum%253Aachs%26aulast%3DMahoney%26aufirst%3DK.%2BM.%26aulast%3DRennert%26aufirst%3DP.%2BD.%26aulast%3DFreeman%26aufirst%3DG.%2BJ.%26atitle%3DCombination%2520cancer%2520immunotherapy%2520and%2520new%2520immunomodulatory%2520targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2015%26volume%3D14%26spage%3D561%26epage%3D584%26doi%3D10.1038%2Fnrd4591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Bresnick, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, D. B.</span><span> </span><span class="NLM_article-title">S100 proteins in cancer</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">96</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span><span class="refDoi"> DOI: 10.1038/nrc3893</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1038%2Fnrc3893" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=25614008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVartrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=96-109&author=A.+R.+Bresnickauthor=D.+J.+Weberauthor=D.+B.+Zimmer&title=S100+proteins+in+cancer&doi=10.1038%2Fnrc3893"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">S100 proteins in cancer</span></div><div class="casAuthors">Bresnick, Anne R.; Weber, David J.; Zimmer, Danna B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">96-109</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In humans, the S100 protein family is composed of 21 members that exhibit a high degree of structural similarity, but are not functionally interchangeable.  This family of proteins modulates cellular responses by functioning both as intracellular Ca2+ sensors and as extracellular factors.  Dysregulated expression of multiple members of the S100 family is a common feature of human cancers, with each type of cancer showing a unique S100 protein profile or signature.  Emerging in vivo evidence indicates that the biol. of most S100 proteins is complex and multifactorial, and that these proteins actively contribute to tumorigenic processes such as cell proliferation, metastasis, angiogenesis and immune evasion.  Drug discovery efforts have identified leads for inhibiting several S100 family members, and two of the identified inhibitors have progressed to clin. trials in patients with cancer.  This Review highlights new findings regarding the role of S100 family members in cancer diagnosis and treatment, the contribution of S100 signalling to tumor biol., and the discovery and development of S100 inhibitors for treating cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTg0j8SyoeNrVg90H21EOLACvtfcHk0lhqQPNbx0qqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVartrg%253D&md5=a78954b14527eea6fef2fd94ec51c6d1</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrc3893&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3893%26sid%3Dliteratum%253Aachs%26aulast%3DBresnick%26aufirst%3DA.%2BR.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26aulast%3DZimmer%26aufirst%3DD.%2BB.%26atitle%3DS100%2520proteins%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2015%26volume%3D15%26spage%3D96%26epage%3D109%26doi%3D10.1038%2Fnrc3893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">El Hajj, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Journe, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiedig, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laios, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sales, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galibert, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Kempen, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spatz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Badran, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsimont, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awada, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghanem, G.</span><span> </span><span class="NLM_article-title">Tyrosinase-related protein 1 mRNA expression in lymph node metastases predicts overall survival in high-risk melanoma patients</span> <span class="citation_source-journal">Br. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">1641</span><span class="NLM_x">–</span> <span class="NLM_lpage">1647</span><span class="refDoi"> DOI: 10.1038/bjc.2013.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1038%2Fbjc.2013.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23519055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVOru7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2013&pages=1641-1647&author=P.+El+Hajjauthor=F.+Journeauthor=M.+Wiedigauthor=I.+Laiosauthor=F.+Salesauthor=M.+D.+Galibertauthor=L.+C.+Van+Kempenauthor=A.+Spatzauthor=B.+Badranauthor=D.+Larsimontauthor=A.+Awadaauthor=G.+Ghanem&title=Tyrosinase-related+protein+1+mRNA+expression+in+lymph+node+metastases+predicts+overall+survival+in+high-risk+melanoma+patients&doi=10.1038%2Fbjc.2013.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosinase-related protein 1 mRNA expression in lymph node metastases predicts overall survival in high-risk melanoma patients</span></div><div class="casAuthors">El Hajj, P.; Journe, F.; Wiedig, M.; Laios, I.; Sales, F.; Galibert, M.-D.; Van Kempen, L. C.; Spatz, A.; Badran, B.; Larsimont, D.; Awada, A.; Ghanem, G.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1641-1647</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Background: Clin. outcome of high-risk melanoma patients is not reliably predicted from histopathol. analyses of primary tumors and is often adjusted during disease progression.  Our study aimed at extending our previous findings in skin metastases to evaluate the prognostic value of tyrosinase-related protein 1 (TYRP1) in lymph node metastases of stages III and IV melanoma patients.  Methods: TYRP1 mRNA expression in 104 lymph node metastases was quantified by real-time PCR and normalized to S100 calcium-binding protein B (S100B) mRNA expression to correct for tumor load.  TYRP1/S100B ratios were calcd. and median was used as cutoff value.  TYRP1/S100B mRNA values were correlated to clin. follow-up and histopathol. characteristics of the primary lesion.  Results: A high TYRP1/S100B mRNA ratio significantly correlated with reduced disease-free (DFS) and overall survival (OS; Cox regression anal., P=0.005 and 0.01, resp.), increased Breslow thickness (Spearman's rho test, P<0.001) and the presence of ulceration (Mann-Whitney test, P=0.02) of the primaries.  Moreover, high TYRP1/S100B was of better prognostic value (lower P-value) for OS than Breslow thickness and ulceration.  Finally, it was well conserved during disease progression with respect to high/low TYRP1 groups.  Conclusion: High TYRP1/S100B mRNA expression in lymph node metastases from melanoma patients is assocd. with unfavorable clin. outcome.  Its evaluation in lymph node metastases may refine initial prognosis for metastatic patients, may define prognosis for those with unknown or non-evaluable primary lesions and may allow different management of the two groups of patients.  British Journal of Cancer (2013) 108, 1641-1647; doi:10.1038/bjc.2013.115 www.bjcancer.com; published online 21 March 2013.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-Am-0FV-6N7Vg90H21EOLACvtfcHk0lhqQPNbx0qqrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVOru7s%253D&md5=29bd07ddea92f9a72cc4055e4a223b07</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2013.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2013.115%26sid%3Dliteratum%253Aachs%26aulast%3DEl%2BHajj%26aufirst%3DP.%26aulast%3DJourne%26aufirst%3DF.%26aulast%3DWiedig%26aufirst%3DM.%26aulast%3DLaios%26aufirst%3DI.%26aulast%3DSales%26aufirst%3DF.%26aulast%3DGalibert%26aufirst%3DM.%2BD.%26aulast%3DVan%2BKempen%26aufirst%3DL.%2BC.%26aulast%3DSpatz%26aufirst%3DA.%26aulast%3DBadran%26aufirst%3DB.%26aulast%3DLarsimont%26aufirst%3DD.%26aulast%3DAwada%26aufirst%3DA.%26aulast%3DGhanem%26aufirst%3DG.%26atitle%3DTyrosinase-related%2520protein%25201%2520mRNA%2520expression%2520in%2520lymph%2520node%2520metastases%2520predicts%2520overall%2520survival%2520in%2520high-risk%2520melanoma%2520patients%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2013%26volume%3D108%26spage%3D1641%26epage%3D1647%26doi%3D10.1038%2Fbjc.2013.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Henry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabre, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guiraud, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bastide, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro-Peray, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavabre-Bertrand, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meunier, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoebner, P. E.</span><span> </span><span class="NLM_article-title">Clinical use of p-proteasome in discriminating metastatic melanoma patients: comparative study with LDH, MIA and S100B protein</span> <span class="citation_source-journal">Int. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">142</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span><span class="refDoi"> DOI: 10.1002/ijc.27991</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fijc.27991" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23238767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntFaksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2013&pages=142-148&author=L.+Henryauthor=C.+Fabreauthor=I.+Guiraudauthor=S.+Bastideauthor=P.+Fabbro-Perayauthor=J.+Martinezauthor=T.+Lavabre-Bertrandauthor=L.+Meunierauthor=P.+E.+Stoebner&title=Clinical+use+of+p-proteasome+in+discriminating+metastatic+melanoma+patients%3A+comparative+study+with+LDH%2C+MIA+and+S100B+protein&doi=10.1002%2Fijc.27991"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Clinical use of p-proteasome in discriminating metastatic melanoma patients: Comparative study with LDH, MIA and S100B protein</span></div><div class="casAuthors">Henry, Laurent; Fabre, Cecile; Guiraud, Isabelle; Bastide, Sophie; Fabbro-Peray, Pascale; Martinez, Jean; Lavabre-Bertrand, Thierry; Meunier, Laurent; Stoebner, Pierre-Emmanuel</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">142-148</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Plasmatic proteasome (p-proteasome) has recently been described as a new marker for metastatic melanoma.  The objective of this study was to compare the diagnostic and prognostic values of p-proteasome with three other melanoma serol. markers: S100B protein, melanoma inhibitory activity protein (MIA) and lactate dehydrogenase (LDH) in the plasma of 121 stage I-IV melanoma patients.  Lab. analyses were performed by standardized ELISA (p-proteasome, MIA), immunoluminometric assay (S100B) and colorimetry (LDH).  We found that all markers were relevant for discriminating metastatic from nonmetastatic patients but p-proteasome displayed the highest diagnostic accuracy.  P-proteasome and S100B were the most sensitive (58.1%) and p-proteasome and MIA the most specific (98.7 and 100%) in detecting metastatic disease.  P-proteasome and S100B had the highest area under receiver operating characteristics curve, 0.811 (95% CI: 0.725-0.897) and 0.822 (95% CI: 0.738-0.906), resp.  These two markers were the best in detecting patients with lymph node metastases.  S100B, MIA and LDH diagnostic accuracy was increased when these markers were combined with p-proteasome.  As shown with univariate anal., shorter progression-free and overall survival rates were significantly assocd. with elevated plasma levels of each markers.  The multivariate Cox regression anal. identified p-proteasome as the only independent predictor of a poorer progression-free survival (p = 0.030).  In conclusion, this comparative study established that p-proteasome quantification in combination with other melanoma biomarkers is an attractive approach for the biol. follow-up of melanoma patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYSvRRY7hWyLVg90H21EOLACvtfcHk0liKiqhzXV2rkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntFaksQ%253D%253D&md5=707afce35f8d0a631156812a485264e9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1002%2Fijc.27991&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.27991%26sid%3Dliteratum%253Aachs%26aulast%3DHenry%26aufirst%3DL.%26aulast%3DFabre%26aufirst%3DC.%26aulast%3DGuiraud%26aufirst%3DI.%26aulast%3DBastide%26aufirst%3DS.%26aulast%3DFabbro-Peray%26aufirst%3DP.%26aulast%3DMartinez%26aufirst%3DJ.%26aulast%3DLavabre-Bertrand%26aufirst%3DT.%26aulast%3DMeunier%26aufirst%3DL.%26aulast%3DStoebner%26aufirst%3DP.%2BE.%26atitle%3DClinical%2520use%2520of%2520p-proteasome%2520in%2520discriminating%2520metastatic%2520melanoma%2520patients%253A%2520comparative%2520study%2520with%2520LDH%252C%2520MIA%2520and%2520S100B%2520protein%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2013%26volume%3D133%26spage%3D142%26epage%3D148%26doi%3D10.1002%2Fijc.27991" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Wieder, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tekin, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenbaum-Krumme, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klode, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altenbernd, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bockisch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagarajah, J.</span><span> </span><span class="NLM_article-title">18FDG-PET to assess recurrence and long term survival in patients with malignant melanoma</span> <span class="citation_source-journal">Nuklearmedizin</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">198</span><span class="NLM_x">–</span> <span class="NLM_lpage">203</span><span class="refDoi"> DOI: 10.3413/Nukmed-0584-13-05</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.3413%2FNukmed-0584-13-05" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23969722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC3sbitVSlsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2013&pages=198-203&author=H.+A.+Wiederauthor=G.+Tekinauthor=S.+Rosenbaum-Krummeauthor=J.+Klodeauthor=J.+Altenberndauthor=A.+Bockischauthor=J.+Nagarajah&title=18FDG-PET+to+assess+recurrence+and+long+term+survival+in+patients+with+malignant+melanoma&doi=10.3413%2FNukmed-0584-13-05"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">18FDG-PET to assess recurrence and long term survival in patients with malignant melanoma</span></div><div class="casAuthors">Wieder H A; Tekin G; Rosenbaum-Krumme S; Klode J; Altenbernd J; Bockisch A; Nagarajah J</div><div class="citationInfo"><span class="NLM_cas:title">Nuklearmedizin. Nuclear medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">198-203</span>
        ISSN:<span class="NLM_cas:issn">0029-5566</span>.
    </div><div class="casAbstract">AIM:  To assess the diagnostic and prognostic value of FDG-PET/CT in the follow-up of malignant melanoma in comparison to the serum protein S100B.  PATIENTS AND METHODS:  A total of ninety patients with either low-risk or high-risk malignant melanoma, respectively, were included in this study.  The follow-up of the patients was pursuant with the guidelines of the German Dermatological Association.  The diagnostic accuracy and diagnostic power were determined for PET/CT and for the serum protein S100B.  RESULTS:  In 28 of the 90 patients PET/CT was positive in the follow up, 47 patients had an elevated Serum S100B level.  Sensitivity, specificity, PPV and NPV of PET/CT for the total groups of patients were 87%, 93%, 87% and 93%.  The corresponding values for the serum protein S100B were 65%, 52%, 43% and 74%, respectively.  PET/CT positive patients showed a significantly (p < 0.001) higher risk of melanoma associated death compared to patients with PET/CT negative findings.  No statistical significance could be found in the 5 year survival rate between the S100B positive and S100B negative patients.  CONCLUSION:  PET/CT is suitable to confirm or exclude recurrences and can be used to assess the prognosis in melanoma patients.  The diagnostic accuracy and the prognostic power is much higher compared to the serum protein S100B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhZsd1Ka8S_a91pGlMMoT5fW6udTcc2eYasJV2YOUUZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3sbitVSlsw%253D%253D&md5=db1d11c67d035946cde7e7e53a4fe813</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3413%2FNukmed-0584-13-05&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3413%252FNukmed-0584-13-05%26sid%3Dliteratum%253Aachs%26aulast%3DWieder%26aufirst%3DH.%2BA.%26aulast%3DTekin%26aufirst%3DG.%26aulast%3DRosenbaum-Krumme%26aufirst%3DS.%26aulast%3DKlode%26aufirst%3DJ.%26aulast%3DAltenbernd%26aufirst%3DJ.%26aulast%3DBockisch%26aufirst%3DA.%26aulast%3DNagarajah%26aufirst%3DJ.%26atitle%3D18FDG-PET%2520to%2520assess%2520recurrence%2520and%2520long%2520term%2520survival%2520in%2520patients%2520with%2520malignant%2520melanoma%26jtitle%3DNuklearmedizin%26date%3D2013%26volume%3D52%26spage%3D198%26epage%3D203%26doi%3D10.3413%2FNukmed-0584-13-05" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Schiltz, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dillman, R. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korse, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cubellis, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Gast, G. C.</span><span> </span><span class="NLM_article-title">Lack of elevation of serum S100B in patients with metastatic melanoma as a predictor of outcome after induction with an autologous vaccine of proliferating tumor cells and dendritic cells</span> <span class="citation_source-journal">Cancer Biother.Radiopharm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1089/cbr.2007.0432</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1089%2Fcbr.2007.0432" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=18454690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlsV2msbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2008&pages=214-221&author=P.+M.+Schiltzauthor=R.+O.+Dillmanauthor=C.+M.+Korseauthor=J.+M.+Cubellisauthor=G.+J.+Leeauthor=G.+C.+De+Gast&title=Lack+of+elevation+of+serum+S100B+in+patients+with+metastatic+melanoma+as+a+predictor+of+outcome+after+induction+with+an+autologous+vaccine+of+proliferating+tumor+cells+and+dendritic+cells&doi=10.1089%2Fcbr.2007.0432"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Lack of Elevation of Serum S100B in Patients with Metastatic Melanoma as a Predictor of Outcome After Induction with an Autologous Vaccine of Proliferating Tumor Cells and Dendritic Cells</span></div><div class="casAuthors">Schiltz, Patric M.; Dillman, Robert O.; Korse, Catharina M.; Cubellis, James M.; Lee, Gregory J.; De Gast, Gijsbert C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Biotherapy & Radiopharmaceuticals</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">214-221</span>CODEN:
                <span class="NLM_cas:coden">CBRAFJ</span>;
        ISSN:<span class="NLM_cas:issn">1084-9785</span>.
    
            (<span class="NLM_cas:orgname">Mary Ann Liebert, Inc.</span>)
        </div><div class="casAbstract">Serum levels of S100B and/or lactate dehydrogenase (LDH) are putative tumor markers in melanoma.  Early changes in such markers may correlate with a pos. immune response to vaccine therapy.  In patients with metastatic melanoma, S100B and LDH serum levels were measured at baseline, and 1 wk after 3 weekly s.c. injections of investigational, patient-specific vaccines consisting of autologous dendritic cells loaded with antigens from irradiated proliferating autologous tumor cells, and suspended in granulocyte macrophage colony-stimulating factor.  There was a poor correlation between S100B and LDH levels at baseline (p = 0.324).  Fourteen (14) patients with measurable disease had higher S100B (p = 0.0456) and LDH (p = 0.0013) levels than 31 patients who lacked measurable disease at that time.  Fourteen (14) deceased patients (median survival, 13 mo) had a mean baseline S100B of 0.62 μg/L (95% confidence interval [CI] 0.00-1.66) and LDH of 815 U/L (95% CI 222-1408); 31 surviving patients (median follow-up, 35.4 mo) had mean S100B of 0.07 μg/L (95% CI 0.00-0.23; p = 0.006) and LDH of 442 U/L (95% CI 296-588; p = 0.002).  Elevated baseline levels of LDH and S100B were each predictive of inferior progression-free survival (PFS) and overall survival (OS) (both p <0.0001), but S100B was a better predictor for PFS than LDH.  Changes in LDH between baseline and week 4 were not predictive of survival, but an increase in S100B predicted for inferior OS ( p = 0.039).  Both LDH and S100B are predictive tumor markers in patients with metastatic melanoma.  This is the first study to examine the changes in serum levels of LDH and S100B in response to an autologous tumor-cell vaccine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob3Ob4jCpV37Vg90H21EOLACvtfcHk0liKiqhzXV2rkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlsV2msbk%253D&md5=be20069dce53f3e1bb68d8aa6ed141b5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1089%2Fcbr.2007.0432&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1089%252Fcbr.2007.0432%26sid%3Dliteratum%253Aachs%26aulast%3DSchiltz%26aufirst%3DP.%2BM.%26aulast%3DDillman%26aufirst%3DR.%2BO.%26aulast%3DKorse%26aufirst%3DC.%2BM.%26aulast%3DCubellis%26aufirst%3DJ.%2BM.%26aulast%3DLee%26aufirst%3DG.%2BJ.%26aulast%3DDe%2BGast%26aufirst%3DG.%2BC.%26atitle%3DLack%2520of%2520elevation%2520of%2520serum%2520S100B%2520in%2520patients%2520with%2520metastatic%2520melanoma%2520as%2520a%2520predictor%2520of%2520outcome%2520after%2520induction%2520with%2520an%2520autologous%2520vaccine%2520of%2520proliferating%2520tumor%2520cells%2520and%2520dendritic%2520cells%26jtitle%3DCancer%2520Biother.Radiopharm.%26date%3D2008%26volume%3D23%26spage%3D214%26epage%3D221%26doi%3D10.1089%2Fcbr.2007.0432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">O’Rourke, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanagan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">See, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slater, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crowe, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Purdie, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elliott, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellem, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, C. W.</span><span> </span><span class="NLM_article-title">Durable complete clinical responses in a phase I/II trial using an autologous melanoma cell/dendritic cell vaccine</span> <span class="citation_source-journal">Cancer Immunol. Immunother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">387</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1007/s00262-003-0375-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1007%2Fs00262-003-0375-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=12682787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BD3s3hslSrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=387-395&author=M.+G.+O%E2%80%99Rourkeauthor=M.+Johnsonauthor=C.+Lanaganauthor=J.+Seeauthor=J.+Yangauthor=J.+R.+Bellauthor=G.+J.+Slaterauthor=B.+M.+Kerrauthor=B.+Croweauthor=D.+M.+Purdieauthor=S.+L.+Elliottauthor=K.+A.+Ellemauthor=C.+W.+Schmidt&title=Durable+complete+clinical+responses+in+a+phase+I%2FII+trial+using+an+autologous+melanoma+cell%2Fdendritic+cell+vaccine&doi=10.1007%2Fs00262-003-0375-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Durable complete clinical responses in a phase I/II trial using an autologous melanoma cell/dendritic cell vaccine</span></div><div class="casAuthors">O'Rourke Michael G E; Johnson Maree; Lanagan Catherine; See Janet; Yang Jie; Bell John R; Slater Greg J; Kerr Beverley M; Crowe Beth; Purdie David M; Elliott Suzanne L; Ellem Kay A O; Schmidt Christopher W</div><div class="citationInfo"><span class="NLM_cas:title">Cancer immunology, immunotherapy : CII</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">387-95</span>
        ISSN:<span class="NLM_cas:issn">0340-7004</span>.
    </div><div class="casAbstract">Advanced metastatic melanoma is incurable by standard treatments, but occasionally responds to immunotherapy.  Recent trials using dendritic cells (DC) as a cellular adjuvant have concentrated on defined peptides as the source of antigens, and rely on foreign proteins as a source of help to generate a cell-mediated immune response.  This approach limits patient accrual, because currently defined, non-mutated epitopes are restricted by a small number of human leucocyte antigens.  It also fails to take advantage of mutated epitopes peculiar to the patient's own tumour, and of CD4+ T lymphocytes as potential effectors of anti-tumour immunity.  We therefore sought to determine whether a fully autologous DC vaccine is feasible, and of therapeutic benefit.  Patients with American Joint Cancer Committee stage IV melanoma were treated with a fully autologous immunotherapy consisting of monocyte-derived DC, matured after culture with irradiated tumour cells.  Of 19 patients enrolled into the trial, sufficient tumour was available to make treatments for 17.  Of these, 12 received a complete priming phase of six cycles of either 0.9x10(6) or 5x10(6) DC/intradermal injection, at 2-weekly intervals.  Where possible, treatment continued with the lower dose at 6-weekly intervals.  The remaining five patients could not complete priming, due to progressive disease.  Three of the 12 patients who completed priming have durable complete responses (average duration 35 months+), three had partial responses, and the remaining six had progressive disease (WHO criteria).  Disease regression was not correlated with dose or with the development of delayed type hypersensitivity responses to intradermal challenge with irradiated, autologous tumour.  However, plasma S-100B levels prior to the commencement of treatment correlated with objective clinical response ( P=0.05) and survival (log rank P<0.001).  The treatment had minimal side-effects and was well tolerated by all patients.  Mature, monocyte-derived DC preparations exposed to appropriate tumour antigen sources can be reliably produced for patients with advanced metastatic melanoma, and in a subset of those patients with lower volume disease their repeated administration results in durable complete responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTtEdEdYQFKPwvupmecNw_lfW6udTcc2eYdP99G2xuGorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3s3hslSrtA%253D%253D&md5=5df19e634f7c8d9229b1d8f9afdf5135</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1007%2Fs00262-003-0375-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00262-003-0375-x%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DM.%2BG.%26aulast%3DJohnson%26aufirst%3DM.%26aulast%3DLanagan%26aufirst%3DC.%26aulast%3DSee%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DJ.%2BR.%26aulast%3DSlater%26aufirst%3DG.%2BJ.%26aulast%3DKerr%26aufirst%3DB.%2BM.%26aulast%3DCrowe%26aufirst%3DB.%26aulast%3DPurdie%26aufirst%3DD.%2BM.%26aulast%3DElliott%26aufirst%3DS.%2BL.%26aulast%3DEllem%26aufirst%3DK.%2BA.%26aulast%3DSchmidt%26aufirst%3DC.%2BW.%26atitle%3DDurable%2520complete%2520clinical%2520responses%2520in%2520a%2520phase%2520I%252FII%2520trial%2520using%2520an%2520autologous%2520melanoma%2520cell%252Fdendritic%2520cell%2520vaccine%26jtitle%3DCancer%2520Immunol.%2520Immunother.%26date%3D2003%26volume%3D52%26spage%3D387%26epage%3D395%26doi%3D10.1007%2Fs00262-003-0375-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">The calcium-binding protein S100B down-regulates p53 and apoptosis in malignant melanoma</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">27487</span><span class="NLM_x">–</span> <span class="NLM_lpage">27498</span><span class="refDoi"> DOI: 10.1074/jbc.M110.155382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1074%2Fjbc.M110.155382" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=27487-27498&author=J.+Linauthor=Q.+Yangauthor=P.+T.+Wilderauthor=F.+Carrierauthor=D.+J.+Weber&title=The+calcium-binding+protein+S100B+down-regulates+p53+and+apoptosis+in+malignant+melanoma&doi=10.1074%2Fjbc.M110.155382"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.155382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.155382%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520calcium-binding%2520protein%2520S100B%2520down-regulates%2520p53%2520and%2520apoptosis%2520in%2520malignant%2520melanoma%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D27487%26epage%3D27498%26doi%3D10.1074%2Fjbc.M110.155382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markowitz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Inhibiting S100B restores p53 levels in primary malignant melanoma cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">34071</span><span class="NLM_x">–</span> <span class="NLM_lpage">34077</span><span class="refDoi"> DOI: 10.1074/jbc.M405419200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1074%2Fjbc.M405419200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=15178678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtFOhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=34071-34077&author=J.+Linauthor=Q.+Yangauthor=Z.+Yanauthor=J.+Markowitzauthor=P.+T.+Wilderauthor=F.+Carrierauthor=D.+J.+Weber&title=Inhibiting+S100B+restores+p53+levels+in+primary+malignant+melanoma+cancer+cells&doi=10.1074%2Fjbc.M405419200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting S100B restores p53 levels in primary malignant melanoma cancer cells</span></div><div class="casAuthors">Lin, Jing; Yang, Qingyuan; Yan, Zhe; Markowitz, Joseph; Wilder, Paul T.; Carrier, France; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">32</span>),
    <span class="NLM_cas:pages">34071-34077</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">S100 calcium-binding proteins such as S100B are elevated in primary malignant melanoma and are used as markers for this and numerous other cancers.  Wild-type p53 protein levels are relatively low in these cancer cells (i.e., when compared with cells without S100B) but are elevated when RNA antisense to S100B is introduced.  This result implicates S100B in the down-regulation of p53 and is consistent with the large decreases in p53 protein levels obsd. previously in transient co-transfections of p53 and S100B.  Down-regulation of p53 in primary malignant melanoma cells is likely the result of a direct interaction with S100B, which was obsd. by co-immunopptn. expts.  Furthermore, p53 binds regions of the S100B promoter, one of which matches the 20-nucleotide p53-binding consensus DNA sequence perfectly.  Therefore, when p53 levels increase, it contributes to its own demise by up-regulating the transcription of S100B as part of a neg. feedback loop.  This is analogous to what is found for another protein that down-regulates p53, namely hdm2 (human double mutant 2).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTg-HaV94udrVg90H21EOLACvtfcHk0ljmReg5QV3ACw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtFOhsLk%253D&md5=40834f8704a05a5ee85e4d949231c0c2</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M405419200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M405419200%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DQ.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DMarkowitz%26aufirst%3DJ.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DInhibiting%2520S100B%2520restores%2520p53%2520levels%2520in%2520primary%2520malignant%2520melanoma%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D34071%26epage%3D34077%26doi%3D10.1074%2Fjbc.M405419200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Markowitz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gitti, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldisseri, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Udan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Identification and characterization of small molecule inhibitors of the calcium-dependent S100B-p53 tumor suppressor interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">5085</span><span class="NLM_x">–</span> <span class="NLM_lpage">5093</span><span class="refDoi"> DOI: 10.1021/jm0497038</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0497038" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5085-5093&author=J.+Markowitzauthor=I.+Chenauthor=R.+Gittiauthor=D.+M.+Baldisseriauthor=Y.+Panauthor=R.+Udanauthor=F.+Carrierauthor=A.+D.+MacKerellauthor=D.+J.+Weber&title=Identification+and+characterization+of+small+molecule+inhibitors+of+the+calcium-dependent+S100B-p53+tumor+suppressor+interaction&doi=10.1021%2Fjm0497038"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm0497038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0497038%26sid%3Dliteratum%253Aachs%26aulast%3DMarkowitz%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DI.%26aulast%3DGitti%26aufirst%3DR.%26aulast%3DBaldisseri%26aufirst%3DD.%2BM.%26aulast%3DPan%26aufirst%3DY.%26aulast%3DUdan%26aufirst%3DR.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520small%2520molecule%2520inhibitors%2520of%2520the%2520calcium-dependent%2520S100B-p53%2520tumor%2520suppressor%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5085%26epage%3D5093%26doi%3D10.1021%2Fjm0497038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKnight, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">The evolution of S100B inhibitors for the treatment of malignant melanoma</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">109</span><span class="refDoi"> DOI: 10.4155/fmc.12.191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.4155%2Ffmc.12.191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=23256816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVKgsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=97-109&author=K.+G.+Hartmanauthor=L.+E.+McKnightauthor=M.+A.+Lirianoauthor=D.+J.+Weber&title=The+evolution+of+S100B+inhibitors+for+the+treatment+of+malignant+melanoma&doi=10.4155%2Ffmc.12.191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">The evolution of S100B inhibitors for the treatment of malignant melanoma</span></div><div class="casAuthors">Hartman, Kira G.; McKnight, Laura E.; Liriano, Melissa A.; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-109</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Malignant melanoma continues to be an extremely fatal cancer due to a lack of viable treatment options for patients.  The calcium-binding protein S100B has long been used as a clin. biomarker, aiding in malignant melanoma staging and patient prognosis.  However, the discovery of p53 as a S100B target and the consequent impact on cell apoptosis redirected research efforts towards the development of inhibitors of this S100B-p53 interaction.  Several approaches, including computer-aided drug design, fluorescence polarization competition assays, NMR, x-ray crystallog. and cell-based screens have been performed to identify compds. that block the S100B-p53 assocn., reactivate p53 transcriptional activities and induce cancer cell death.  Eight promising compds., including pentamidine, are presented in this review and the potential for future modifications is discussed.  Synthesis of compd. derivs. will likely exhibit increased S100B affinity and mimic important S100B-target dynamic properties that will result in high specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7sdVwSO2M2rVg90H21EOLACvtfcHk0lgbVG25bvFWjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVKgsLfN&md5=96d214505432e590eb1903f51810b35f</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.4155%2Ffmc.12.191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.12.191%26sid%3Dliteratum%253Aachs%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DMcKnight%26aufirst%3DL.%2BE.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520evolution%2520of%2520S100B%2520inhibitors%2520for%2520the%2520treatment%2520of%2520malignant%2520melanoma%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D97%26epage%3D109%26doi%3D10.4155%2Ffmc.12.191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Hartman, K. G.; Wilder, P. T.; Varney, K.; MacKerell, A. D.,  Jr.; Coop, A.; Zimmer, D.; Lapidus, R. G.; Weber, D. J.</span><span> </span><span class="NLM_article-title">Inhibiting S100B in Malignant Melanoma</span>. In  <span class="citation_source-book">Melanoma—From Early Detection to Treatment</span>; <span class="NLM_contrib-group">Duc, H. T.</span>, Ed.; <span class="NLM_publisher-name">InTech</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">649</span><span class="NLM_x">–</span> <span class="NLM_lpage">667</span>, DOI: <span class="refDoi"> DOI: 10.5772/55176</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.5772%2F55176" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=649-667&author=K.+G.+Hartman&author=P.+T.+Wilder&author=K.+Varney&author=A.+D.+MacKerell&author=A.+Coop&author=D.+Zimmer&author=R.+G.+Lapidus&author=D.+J.+Weberauthor=H.+T.+Duc&title=Melanoma%E2%80%94From+Early+Detection+to+Treatment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.5772%2F55176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.5772%252F55176%26sid%3Dliteratum%253Aachs%26aulast%3DHartman%26aufirst%3DK.%2BG.%26atitle%3DInhibiting%2520S100B%2520in%2520Malignant%2520Melanoma%26btitle%3DMelanoma%25E2%2580%2594From%2520Early%2520Detection%2520to%2520Treatment%26aulast%3DDuc%26aufirst%3DH.%2BT.%26pub%3DInTech%26date%3D2013%26spage%3D649%26epage%3D667%26doi%3D10.5772%2F55176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Meghnani, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vetter, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leclerc, E.</span><span> </span><span class="NLM_article-title">RAGE overexpression confers a metastatic phenotype to the WM115 human primary melanoma cell line</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Basis Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">1842</span><span class="NLM_x">, </span> <span class="NLM_fpage">1017</span><span class="NLM_x">–</span> <span class="NLM_lpage">1027</span><span class="refDoi"> DOI: 10.1016/j.bbadis.2014.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.bbadis.2014.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24613454" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotFyisLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1842&publication_year=2014&pages=1017-1027&author=V.+Meghnaniauthor=S.+W.+Vetterauthor=E.+Leclerc&title=RAGE+overexpression+confers+a+metastatic+phenotype+to+the+WM115+human+primary+melanoma+cell+line&doi=10.1016%2Fj.bbadis.2014.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">RAGE overexpression confers a metastatic phenotype to the WM115 human primary melanoma cell line</span></div><div class="casAuthors">Meghnani, Varsha; Vetter, Stefan W.; Leclerc, Estelle</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Basis of Disease</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">1842</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1017-1027</span>CODEN:
                <span class="NLM_cas:coden">BBADEX</span>;
        ISSN:<span class="NLM_cas:issn">0925-4439</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">The formation of melanoma metastases from primary tumor cells is a complex phenomenon that involves the regulation of multiple genes.  We have previously shown that the receptor for advanced glycation end products (RAGE) was up-regulated in late metastatic stages of melanoma patient samples and we hypothesized that up-regulation of RAGE in cells forming a primary melanoma tumor could contribute to the metastatic switch of these cells.  To test our hypothesis, we overexpressed RAGE in the WM115 human melanoma cell line that was established from a primary melanoma tumor of a patient.  We show here that overexpression of RAGE in these cells is assocd. with mesenchymal-like morphologies of the cells.  These cells demonstrate higher migration abilities and reduced proliferation properties, suggesting that the cells have switched to a metastatic phenotype.  At the mol. level, we show that RAGE overexpression is assocd. with the up-regulation of the RAGE ligand S100B and the down-regulation of p53, ERK1/2, cyclin E and NF-κB.  Our study supports a role of RAGE in the metastatic switch of melanoma cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmJa067cU0o7Vg90H21EOLACvtfcHk0lh9WUaBhnXsTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotFyisLk%253D&md5=ffcf426cd8cf7d0ebd06171c57b6eed4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.bbadis.2014.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbadis.2014.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DMeghnani%26aufirst%3DV.%26aulast%3DVetter%26aufirst%3DS.%2BW.%26aulast%3DLeclerc%26aufirst%3DE.%26atitle%3DRAGE%2520overexpression%2520confers%2520a%2520metastatic%2520phenotype%2520to%2520the%2520WM115%2520human%2520primary%2520melanoma%2520cell%2520line%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Basis%2520Dis.%26date%3D2014%26volume%3D1842%26spage%3D1017%26epage%3D1027%26doi%3D10.1016%2Fj.bbadis.2014.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Briceland, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bailie, G. R.</span><span> </span><span class="NLM_article-title">Pentamidine-associated nephrotoxicity and hyperkalemia in patients with AIDS</span> <span class="citation_source-journal">Ann. Pharmacother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1991</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1171</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1177/106002809102501102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1177%2F106002809102501102" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1991&pages=1171-1174&author=L.+L.+Bricelandauthor=G.+R.+Bailie&title=Pentamidine-associated+nephrotoxicity+and+hyperkalemia+in+patients+with+AIDS&doi=10.1177%2F106002809102501102"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F106002809102501102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F106002809102501102%26sid%3Dliteratum%253Aachs%26aulast%3DBriceland%26aufirst%3DL.%2BL.%26aulast%3DBailie%26aufirst%3DG.%2BR.%26atitle%3DPentamidine-associated%2520nephrotoxicity%2520and%2520hyperkalemia%2520in%2520patients%2520with%2520AIDS%26jtitle%3DAnn.%2520Pharmacother.%26date%3D1991%26volume%3D25%26spage%3D1171%26epage%3D1174%26doi%3D10.1177%2F106002809102501102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Edwards, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Crystal structure of a pentamidine-oligonucleotide complex: implications for DNA-binding properties</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">7104</span><span class="NLM_x">–</span> <span class="NLM_lpage">7109</span><span class="refDoi"> DOI: 10.1021/bi00146a011</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00146a011" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADyaK38XkvVaqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1992&pages=7104-7109&author=K.+J.+Edwardsauthor=T.+C.+Jenkinsauthor=S.+Neidle&title=Crystal+structure+of+a+pentamidine-oligonucleotide+complex%3A+implications+for+DNA-binding+properties&doi=10.1021%2Fbi00146a011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of a pentamidine-oligonucleotide complex:  implications for DNA-binding properties</span></div><div class="casAuthors">Edwards, Karen J.; Jenkins, Terence C.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">7104-9</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">The crystal structure of the complex formed between the dodecanucleotide d(CGCGAATTCGCG)2 and the drug pentamidine, which is active against the Pneumocystis carinii pathogen in AIDS patients, was detd. to a resoln. of 2.1 Å and an R-factor of 19.4%.  The drug was in the 5'-AATT minor groove region of the duplex, with the amidinium groups H-bonded to adenine N3 atoms in an interstrand manner.  The drug mol. adopts an extended conformation, and the immediate binding site spans 4 base pairs.  Structural details of the drug-DNA interactions are discussed, and comparison is made with the dodecamer complex of the structurally similar berenil ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhj8RQpE1GobVg90H21EOLACvtfcHk0lh9WUaBhnXsTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XkvVaqtr4%253D&md5=c702833ed79000720539571e08baa68e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fbi00146a011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00146a011%26sid%3Dliteratum%253Aachs%26aulast%3DEdwards%26aufirst%3DK.%2BJ.%26aulast%3DJenkins%26aufirst%3DT.%2BC.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520a%2520pentamidine-oligonucleotide%2520complex%253A%2520implications%2520for%2520DNA-binding%2520properties%26jtitle%3DBiochemistry%26date%3D1992%26volume%3D31%26spage%3D7104%26epage%3D7109%26doi%3D10.1021%2Fbi00146a011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Pathak, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhawan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindner, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borden, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farver, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, T.</span><span> </span><span class="NLM_article-title">Pentamidine is an inhibitor of PRL phosphatases with anticancer activity</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">1255</span><span class="NLM_x">–</span> <span class="NLM_lpage">1264</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=12516958" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmtFalsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2002&pages=1255-1264&author=M.+K.+Pathakauthor=D.+Dhawanauthor=D.+J.+Lindnerauthor=E.+C.+Bordenauthor=C.+Farverauthor=T.+Yi&title=Pentamidine+is+an+inhibitor+of+PRL+phosphatases+with+anticancer+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pentamidine is an inhibitor of PRL phosphatases with anticancer activity</span></div><div class="casAuthors">Pathak, Manas K.; Dhawan, Deepika; Lindner, Daniel J.; Borden, Ernest C.; Farver, Carol; Yi, Taolin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1255-1264</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Thr PRL family oncogenic phosphatases are attractive targets for developing inhibitors as anticancer therapeutics given their potentially pathogenic role in human malignancies.  Herein we demonstrate that pentamidine, an anti-protozoa drug with an unknown mechanism of action, is an inhibitor of PRLs with anticancer potential.  Pentamidine at its therapeutic doses inhibited recombinant PRL phosphatases in vitro and inactivated ectopically expressed PRLs in NIH3T3 transfectants with an effective duration more than 24 h after a pulse cell treatment.  The drug had in vitro growth-inhibitory activity against human cancer cell lines that express the endogenous PRLs.  Pentamidine at a tolerable dose markedly inhibited the growth of WM9 human melanoma tumors in nude mice coincident with the induction of tumor cell necrosis and is capable of inactivating ectopically expressed PRL-2 in the cancer cells.  These observations suggest the potential of pentamidine in anticancer therapies and may provide a basis for developing novel PTPase-targeted therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkt1FZKVxVYLVg90H21EOLACvtfcHk0lh9WUaBhnXsTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmtFalsg%253D%253D&md5=e695157767544c6a55e5c6b7a4966d4c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPathak%26aufirst%3DM.%2BK.%26aulast%3DDhawan%26aufirst%3DD.%26aulast%3DLindner%26aufirst%3DD.%2BJ.%26aulast%3DBorden%26aufirst%3DE.%2BC.%26aulast%3DFarver%26aufirst%3DC.%26aulast%3DYi%26aufirst%3DT.%26atitle%3DPentamidine%2520is%2520an%2520inhibitor%2520of%2520PRL%2520phosphatases%2520with%2520anticancer%2520activity%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2002%26volume%3D1%26spage%3D1255%26epage%3D1264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">McKnight, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bezawada, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kudrimoti, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godoy-Ruiz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Structure-Based Discovery of a Novel Pentamidine-Related Inhibitor of the Calcium-Binding Protein S100B</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">975</span><span class="NLM_x">–</span> <span class="NLM_lpage">979</span><span class="refDoi"> DOI: 10.1021/ml300166s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300166s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=975-979&author=L.+E.+McKnightauthor=E.+P.+Ramanauthor=P.+Bezawadaauthor=S.+Kudrimotiauthor=P.+T.+Wilderauthor=K.+G.+Hartmanauthor=R.+Godoy-Ruizauthor=E.+A.+Tothauthor=A.+Coopauthor=A.+D.+Mackerellauthor=D.+J.+Weber&title=Structure-Based+Discovery+of+a+Novel+Pentamidine-Related+Inhibitor+of+the+Calcium-Binding+Protein+S100B&doi=10.1021%2Fml300166s"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fml300166s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300166s%26sid%3Dliteratum%253Aachs%26aulast%3DMcKnight%26aufirst%3DL.%2BE.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DBezawada%26aufirst%3DP.%26aulast%3DKudrimoti%26aufirst%3DS.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DGodoy-Ruiz%26aufirst%3DR.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DStructure-Based%2520Discovery%2520of%2520a%2520Novel%2520Pentamidine-Related%2520Inhibitor%2520of%2520the%2520Calcium-Binding%2520Protein%2520S100B%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D975%26epage%3D979%26doi%3D10.1021%2Fml300166s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Agamennone, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cesari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lalli, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turlizzi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Conte, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turano, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mangani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padova, A.</span><span> </span><span class="NLM_article-title">Fragmenting the S100B-p53 Interaction: Combined Virtual/Biophysical Screening Approaches to Identify Ligands</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">428</span><span class="NLM_x">–</span> <span class="NLM_lpage">435</span><span class="refDoi"> DOI: 10.1002/cmdc.200900393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fcmdc.200900393" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=428-435&author=M.+Agamennoneauthor=L.+Cesariauthor=D.+Lalliauthor=E.+Turlizziauthor=R.+Del+Conteauthor=P.+Turanoauthor=S.+Manganiauthor=A.+Padova&title=Fragmenting+the+S100B-p53+Interaction%3A+Combined+Virtual%2FBiophysical+Screening+Approaches+to+Identify+Ligands&doi=10.1002%2Fcmdc.200900393"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900393%26sid%3Dliteratum%253Aachs%26aulast%3DAgamennone%26aufirst%3DM.%26aulast%3DCesari%26aufirst%3DL.%26aulast%3DLalli%26aufirst%3DD.%26aulast%3DTurlizzi%26aufirst%3DE.%26aulast%3DDel%2BConte%26aufirst%3DR.%26aulast%3DTurano%26aufirst%3DP.%26aulast%3DMangani%26aufirst%3DS.%26aulast%3DPadova%26aufirst%3DA.%26atitle%3DFragmenting%2520the%2520S100B-p53%2520Interaction%253A%2520Combined%2520Virtual%252FBiophysical%2520Screening%2520Approaches%2520to%2520Identify%2520Ligands%26jtitle%3DChemMedChem%26date%3D2010%26volume%3D5%26spage%3D428%26epage%3D435%26doi%3D10.1002%2Fcmdc.200900393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Yoshimura, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyafusa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, K.</span><span> </span><span class="NLM_article-title">Identification of small-molecule inhibitors of the human S100B-p53 interaction and evaluation of their activity in human melanoma cells</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">1109</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.bmc.2012.12.042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.bmc.2012.12.042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1109-1115&author=C.+Yoshimuraauthor=T.+Miyafusaauthor=K.+Tsumoto&title=Identification+of+small-molecule+inhibitors+of+the+human+S100B-p53+interaction+and+evaluation+of+their+activity+in+human+melanoma+cells&doi=10.1016%2Fj.bmc.2012.12.042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.12.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.12.042%26sid%3Dliteratum%253Aachs%26aulast%3DYoshimura%26aufirst%3DC.%26aulast%3DMiyafusa%26aufirst%3DT.%26aulast%3DTsumoto%26aufirst%3DK.%26atitle%3DIdentification%2520of%2520small-molecule%2520inhibitors%2520of%2520the%2520human%2520S100B-p53%2520interaction%2520and%2520evaluation%2520of%2520their%2520activity%2520in%2520human%2520melanoma%2520cells%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D1109%26epage%3D1115%26doi%3D10.1016%2Fj.bmc.2012.12.042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bogan, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thorn, K. S.</span><span> </span><span class="NLM_article-title">Anatomy of hot spots in protein interfaces</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1006/jmbi.1998.1843</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1006%2Fjmbi.1998.1843" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=9653027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=1998&pages=1-9&author=A.+A.+Boganauthor=K.+S.+Thorn&title=Anatomy+of+hot+spots+in+protein+interfaces&doi=10.1006%2Fjmbi.1998.1843"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Anatomy of hot spots in protein interfaces</span></div><div class="casAuthors">Bogan A A; Thorn K S</div><div class="citationInfo"><span class="NLM_cas:title">Journal of molecular biology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-9</span>
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">Binding of one protein to another is involved in nearly all biological functions, yet the principles governing the interaction of proteins are not fully understood.  To analyze the contributions of individual amino acid residues in protein-protein binding we have compiled a database of 2325 alanine mutants for which the change in free energy of binding upon mutation to alanine has been measured (available at http://motorhead. ucsf.edu/thorn/hotspot).  Our analysis shows that at the level of side-chains there is little correlation between buried surface area and free energy of binding.  We find that the free energy of binding is not evenly distributed across interfaces; instead, there are hot spots of binding energy made up of a small subset of residues in the dimer interface.  These hot spots are enriched in tryptophan, tyrosine and arginine, and are surrounded by energetically less important residues that most likely serve to occlude bulk solvent from the hot spot.  Occlusion of solvent is found to be a necessary condition for highly energetic interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZMrH3BevWP3ELvajU9pjPfW6udTcc2eaooijvvLfNR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK1czhslSgtg%253D%253D&md5=e3ef3c857d1fe070deff25dc54d09587</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1998.1843&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1998.1843%26sid%3Dliteratum%253Aachs%26aulast%3DBogan%26aufirst%3DA.%2BA.%26aulast%3DThorn%26aufirst%3DK.%2BS.%26atitle%3DAnatomy%2520of%2520hot%2520spots%2520in%2520protein%2520interfaces%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1998%26volume%3D280%26spage%3D1%26epage%3D9%26doi%3D10.1006%2Fjmbi.1998.1843" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Cerchietti, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghetu, A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Da Silva, G. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunting, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polo, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fares, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prive, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melnick, A.</span><span> </span><span class="NLM_article-title">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">400</span><span class="NLM_x">–</span> <span class="NLM_lpage">411</span><span class="refDoi"> DOI: 10.1016/j.ccr.2009.12.050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.ccr.2009.12.050" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=20385364" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2010&pages=400-411&author=L.+C.+Cerchiettiauthor=A.+F.+Ghetuauthor=X.+Zhuauthor=G.+F.+Da+Silvaauthor=S.+Zhongauthor=M.+Matthewsauthor=K.+L.+Buntingauthor=J.+M.+Poloauthor=C.+Faresauthor=C.+H.+Arrowsmithauthor=S.+N.+Yangauthor=M.+Garciaauthor=A.+Coopauthor=A.+D.+Mackerellauthor=G.+G.+Priveauthor=A.+Melnick&title=A+small-molecule+inhibitor+of+BCL6+kills+DLBCL+cells+in+vitro+and+in+vivo&doi=10.1016%2Fj.ccr.2009.12.050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule inhibitor of BCL6 kills DLBCL cells in vitro and in vivo</span></div><div class="casAuthors">Cerchietti, Leandro C.; Ghetu, Alexandru F.; Zhu, Xiao; Da Silva, Gustavo F.; Zhong, Shijun; Matthews, Marilyn; Bunting, Karen L.; Polo, Jose M.; Fares, Christophe; Arrowsmith, Cheryl H.; Yang, Shao Ning; Garcia, Monica; Coop, Andrew; MacKerell, Alexander D., Jr.; Prive, Gilbert G.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">400-411</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The BCL6 transcriptional repressor is the most frequently involved oncogene in diffuse large B cell lymphoma (DLBCL).  We combined computer-aided drug design with functional assays to identify low-mol.-wt. compds. that bind to the corepressor binding groove of the BCL6 BTB domain.  One such compd. disrupted BCL6/corepressor complexes in vitro and in vivo, and was obsd. by X-ray crystallog. and NMR to bind the crit. site within the BTB groove.  This compd. could induce expression of BCL6 target genes and kill BCL6-pos. DLBCL cell lines.  In xenotransplantation expts., the compd. was nontoxic and potently suppressed DLBCL tumors in vivo.  The compd. also killed primary DLBCLs from human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdcScjNPjhiLVg90H21EOLACvtfcHk0lib_xjStXuG5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2msb8%253D&md5=64f30a813c26394b69da6de95991f8dd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.12.050%26sid%3Dliteratum%253Aachs%26aulast%3DCerchietti%26aufirst%3DL.%2BC.%26aulast%3DGhetu%26aufirst%3DA.%2BF.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DDa%2BSilva%26aufirst%3DG.%2BF.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DMatthews%26aufirst%3DM.%26aulast%3DBunting%26aufirst%3DK.%2BL.%26aulast%3DPolo%26aufirst%3DJ.%2BM.%26aulast%3DFares%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DYang%26aufirst%3DS.%2BN.%26aulast%3DGarcia%26aufirst%3DM.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DPrive%26aufirst%3DG.%2BG.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DA%2520small-molecule%2520inhibitor%2520of%2520BCL6%2520kills%2520DLBCL%2520cells%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D17%26spage%3D400%26epage%3D411%26doi%3D10.1016%2Fj.ccr.2009.12.050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Samadani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brophy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Priyakumar, U. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St John, F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, K. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, P.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of ERK-mediated immediate early gene expression and proliferation of melanoma cells expressing mutated BRaf</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">467</span><span class="NLM_x">, </span> <span class="NLM_fpage">425</span><span class="NLM_x">–</span> <span class="NLM_lpage">438</span><span class="refDoi"> DOI: 10.1042/BJ20131571</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1042%2FBJ20131571" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=25695333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmslCgsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=467&publication_year=2015&pages=425-438&author=R.+Samadaniauthor=J.+Zhangauthor=A.+Brophyauthor=T.+Oashiauthor=U.+D.+Priyakumarauthor=E.+P.+Ramanauthor=F.+J.+St+Johnauthor=K.+Y.+Jungauthor=S.+Fletcherauthor=E.+Pozharskiauthor=A.+D.+MacKerellauthor=P.+Shapiro&title=Small-molecule+inhibitors+of+ERK-mediated+immediate+early+gene+expression+and+proliferation+of+melanoma+cells+expressing+mutated+BRaf&doi=10.1042%2FBJ20131571"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibitors of ERK-mediated immediate early gene expression and proliferation of melanoma cells expressing mutated BRaf</span></div><div class="casAuthors">Samadani, Ramin; Zhang, Jun; Brophy, Amanda; Oashi, Taiji; Priyakumar, U. Deva; Raman, E. Prabhu; St. John, Franz J.; Jung, Kwan-Young; Fletcher, Steven; Pozharski, Edwin; MacKerell, Alexander D.; Shapiro, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">467</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">425-438</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">Constitutive activation of the extracellular-signal-regulated kinases 1 and 2 (ERK1/2) are central to regulating the proliferation and survival of many cancer cells.  The current inhibitors of ERK1/2 target ATP binding or the catalytic site and are therefore limited in their utility for elucidating the complex biol. roles of ERK1/2 through its phosphorylation and regulation of over 100 substrate proteins.  To overcome this limitation, a combination of computational and exptl. methods was used to identify low-mol.-mass inhibitors that are intended to target ERK1/2 substrate-docking domains and selectively interfere with ERK1/2 regulation of substrate proteins.  In the present study, we report the identification and characterization of compds. with a thienyl benzenesulfonate scaffold that were designed to inhibit ERK1/2 substrates contg. an F-site or DEF (docking site for ERK, FXF) motif.  Exptl. evidence shows the compds. inhibit the expression of F-site contg. immediate early genes (IEGs) of the Fos family, including c-Fos and Fra1, and transcriptional regulation of the activator protein-1 (AP-1) complex.  Moreover, this class of compds. selectively induces apoptosis in melanoma cells contg. mutated BRaf and constitutively active ERK1/2 signalling, including melanoma cells that are inherently resistant to clin. relevant kinase inhibitors.  These findings represent the identification and initial characterization of a novel class of compds. that inhibit ERK1/2 signalling functions and their potential utility for elucidating ERK1/2 and other signalling events that control the growth and survival of cancer cells contg. elevated ERK1/2 activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy0bGU3k6ObbVg90H21EOLACvtfcHk0lib_xjStXuG5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmslCgsrg%253D&md5=7a04a08b6b0a015ac4b673ab714f7e3e</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1042%2FBJ20131571&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20131571%26sid%3Dliteratum%253Aachs%26aulast%3DSamadani%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBrophy%26aufirst%3DA.%26aulast%3DOashi%26aufirst%3DT.%26aulast%3DPriyakumar%26aufirst%3DU.%2BD.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DSt%2BJohn%26aufirst%3DF.%2BJ.%26aulast%3DJung%26aufirst%3DK.%2BY.%26aulast%3DFletcher%26aufirst%3DS.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DShapiro%26aufirst%3DP.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520ERK-mediated%2520immediate%2520early%2520gene%2520expression%2520and%2520proliferation%2520of%2520melanoma%2520cells%2520expressing%2520mutated%2520BRaf%26jtitle%3DBiochem.%2520J.%26date%3D2015%26volume%3D467%26spage%3D425%26epage%3D438%26doi%3D10.1042%2FBJ20131571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Divalent metal ion complexes of S100B in the absence and presence of pentamidine</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">382</span><span class="NLM_x">, </span> <span class="NLM_fpage">56</span><span class="NLM_x">–</span> <span class="NLM_lpage">73</span><span class="refDoi"> DOI: 10.1016/j.jmb.2008.06.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.jmb.2008.06.047" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=382&publication_year=2008&pages=56-73&author=T.+H.+Charpentierauthor=P.+T.+Wilderauthor=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=E.+Pozharskiauthor=A.+D.+MacKerellauthor=A.+Coopauthor=E.+A.+Tothauthor=D.+J.+Weber&title=Divalent+metal+ion+complexes+of+S100B+in+the+absence+and+presence+of+pentamidine&doi=10.1016%2Fj.jmb.2008.06.047"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2008.06.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2008.06.047%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DDivalent%2520metal%2520ion%2520complexes%2520of%2520S100B%2520in%2520the%2520absence%2520and%2520presence%2520of%2520pentamidine%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2008%26volume%3D382%26spage%3D56%26epage%3D73%26doi%3D10.1016%2Fj.jmb.2008.06.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Cavalier, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alasady, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neau, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foley, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jadhav, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maloney, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeonov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Covalent small molecule inhibitors of Ca(2+)-bound S100B</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">6628</span><span class="NLM_x">–</span> <span class="NLM_lpage">6640</span><span class="refDoi"> DOI: 10.1021/bi5005552</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi5005552" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1Ciu7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=6628-6640&author=M.+C.+Cavalierauthor=A.+D.+Pierceauthor=P.+T.+Wilderauthor=M.+J.+Alasadyauthor=K.+G.+Hartmanauthor=D.+B.+Neauauthor=T.+L.+Foleyauthor=A.+Jadhavauthor=D.+J.+Maloneyauthor=A.+Simeonovauthor=E.+A.+Tothauthor=D.+J.+Weber&title=Covalent+small+molecule+inhibitors+of+Ca%282%2B%29-bound+S100B&doi=10.1021%2Fbi5005552"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent Small Molecule Inhibitors of Ca2+-Bound S100B</span></div><div class="casAuthors">Cavalier, Michael C.; Pierce, Adam D.; Wilder, Paul T.; Alasady, Milad J.; Hartman, Kira G.; Neau, David B.; Foley, Timothy L.; Jadhav, Ajit; Maloney, David J.; Simeonov, Anton; Toth, Eric A.; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">6628-6640</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Elevated levels of the tumor marker S100B are obsd. in malignant melanoma, and this EF-hand-contg. protein was shown to directly bind wild-type (wt) p53 in a Ca2+-dependent manner, dissoc. the p53 tetramer, and inhibit its tumor suppression functions.  Likewise, inhibiting S100B with small interfering RNA (siRNAS100B) is sufficient to restore wild-type p53 levels and its downstream gene products and induce the arrest of cell growth and UV-dependent apoptosis in malignant melanoma.  Therefore, it is a goal to develop S100B inhibitors (SBiXs) that inhibit the S100B-p53 complex and restore active p53 in this deadly cancer.  Using a structure-activity relationship by NMR approach (SAR by NMR), three persistent binding pockets are found on S100B, termed sites 1-3.  While inhibitors that simultaneously bind sites 2 and 3 are in place, no mols. that simultaneously bind all three persistent sites are available.  For this purpose, Cys84 was used in this study as a potential means to bridge sites 1 and 2 because it is located in a small crevice between these two deeper pockets on the protein.  Using a fluorescence polarization competition assay, several Cys84-modified S100B complexes were identified and examd. further.  For five such SBiX-S100B complexes, crystallog. structures confirmed their covalent binding to Cys84 near site 2 and thus present straightforward chem. biol. strategies for bridging sites 1 and 3.  Importantly, one such compd., SC1982, showed an S100B-dependent death response in assays with WM115 malignant melanoma cells, so it will be particularly useful for the design of SBiX mols. with improved affinity and specificity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHMFpfVuslNLVg90H21EOLACvtfcHk0lgi4-tde7ykEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1Ciu7zO&md5=bd0423ee89babc4ee4e8e40282f0bd10</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fbi5005552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi5005552%26sid%3Dliteratum%253Aachs%26aulast%3DCavalier%26aufirst%3DM.%2BC.%26aulast%3DPierce%26aufirst%3DA.%2BD.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DAlasady%26aufirst%3DM.%2BJ.%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DNeau%26aufirst%3DD.%2BB.%26aulast%3DFoley%26aufirst%3DT.%2BL.%26aulast%3DJadhav%26aufirst%3DA.%26aulast%3DMaloney%26aufirst%3DD.%2BJ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DCovalent%2520small%2520molecule%2520inhibitors%2520of%2520Ca%25282%252B%2529-bound%2520S100B%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D6628%26epage%3D6640%26doi%3D10.1021%2Fbi5005552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Spychala, J.</span><span> </span><span class="NLM_article-title">4-(Cyclic Amidino)phenols - Preparation and Use In A Diamidine Synthesis</span> <span class="citation_source-journal">Synth. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">1083</span><span class="NLM_x">–</span> <span class="NLM_lpage">1094</span><span class="refDoi"> DOI: 10.1080/00397910008087126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1080%2F00397910008087126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXitVOrur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2000&pages=1083-1094&author=J.+Spychala&title=4-%28Cyclic+Amidino%29phenols+-+Preparation+and+Use+In+A+Diamidine+Synthesis&doi=10.1080%2F00397910008087126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">4-(Cyclic-amidino)phenols. Preparation and use in a diamidine synthesis</span></div><div class="casAuthors">Spychala, Jaroslaw</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1083-1094</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">The Pinner synthesis was applied to the prepn. of phenols 4-substituted with cyclic amidines in high yields from readily available 4-HOC6H4C(OMe):NH.HCl and aliph. diamines.  An alternative attempt was made to convert 4-HOC6H4CSNH2 to 4-(1,4,5,6-tetrahydro-5-hydroxy-2-pyrimidinyl)phenol.  A procedure for the prepn. of 3,6-bis(4-hydroxyphenyl)-1,2,4,5-tetrazine is reported.  The syntheses of 2,4-bis[4-(4,5-dihydro-1H-imidazol-2-yl)phenoxy]pyrimidine and 2-chloro-4-[4-(4,5-dihydro-1H-imidazol-2-yl)phenoxy]pyrimidine exemplify the use of the title compds. as synthetic intermediates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFhiWY8pPdKLVg90H21EOLACvtfcHk0lgi4-tde7ykEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXitVOrur8%253D&md5=4e57399d36f3950250f02d1504c55d43</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1080%2F00397910008087126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F00397910008087126%26sid%3Dliteratum%253Aachs%26aulast%3DSpychala%26aufirst%3DJ.%26atitle%3D4-%2528Cyclic%2520Amidino%2529phenols%2520-%2520Preparation%2520and%2520Use%2520In%2520A%2520Diamidine%2520Synthesis%26jtitle%3DSynth.%2520Commun.%26date%3D2000%26volume%3D30%26spage%3D1083%26epage%3D1094%26doi%3D10.1080%2F00397910008087126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Pigini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brasili, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cassinelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giardina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gulini, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quaglia, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melchiorre, C.</span><span> </span><span class="NLM_article-title">Synthesis and Alpha-Blocking Activity of Some Analogs of Idazoxan</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1987</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">273</span><span class="NLM_x">–</span> <span class="NLM_lpage">276</span><span class="refDoi"> DOI: 10.1016/0223-5234(87)90263-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2F0223-5234%2887%2990263-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1987&pages=273-276&author=M.+Piginiauthor=L.+Brasiliauthor=A.+Cassinelliauthor=D.+Giardinaauthor=U.+Guliniauthor=W.+Quagliaauthor=C.+Melchiorre&title=Synthesis+and+Alpha-Blocking+Activity+of+Some+Analogs+of+Idazoxan&doi=10.1016%2F0223-5234%2887%2990263-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2F0223-5234%2887%2990263-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0223-5234%252887%252990263-7%26sid%3Dliteratum%253Aachs%26aulast%3DPigini%26aufirst%3DM.%26aulast%3DBrasili%26aufirst%3DL.%26aulast%3DCassinelli%26aufirst%3DA.%26aulast%3DGiardina%26aufirst%3DD.%26aulast%3DGulini%26aufirst%3DU.%26aulast%3DQuaglia%26aufirst%3DW.%26aulast%3DMelchiorre%26aufirst%3DC.%26atitle%3DSynthesis%2520and%2520Alpha-Blocking%2520Activity%2520of%2520Some%2520Analogs%2520of%2520Idazoxan%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D1987%26volume%3D22%26spage%3D273%26epage%3D276%26doi%3D10.1016%2F0223-5234%2887%2990263-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Manley, P. J.; Nanda, K. K.; Trotter, B. W.</span><span> </span><span class="NLM_article-title">Imidazoisoindole neuropeptide s receptor antagonists</span>. WO 2010056567 A1,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=P.+J.+Manley&author=K.+K.+Nanda&author=B.+W.+Trotter&title=Imidazoisoindole+neuropeptide+s+receptor+antagonists"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DManley%26aufirst%3DP.%2BJ.%26atitle%3DImidazoisoindole%2520neuropeptide%2520s%2520receptor%2520antagonists%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Perner, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDomenico, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koenig, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gomtsyan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bayburt, E. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drizin, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, G. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jinkerson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keddy, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lukin, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonald, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honore, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikusa, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsh, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetter, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. George, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvis, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faltynek, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. H.</span><span> </span><span class="NLM_article-title">In vitro structure-activity relationship and in vivo characterization of 1-(Aryl)-3-(4-(amino)benzyl)urea transient receptor potential vanilloid 1 antagonists</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">3651</span><span class="NLM_x">–</span> <span class="NLM_lpage">3660</span><span class="refDoi"> DOI: 10.1021/jm070276i</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070276i" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=3651-3660&author=R.+J.+Pernerauthor=S.+DiDomenicoauthor=J.+R.+Koenigauthor=A.+Gomtsyanauthor=E.+K.+Bayburtauthor=R.+G.+Schmidtauthor=I.+Drizinauthor=G.+Z.+Zhengauthor=S.+C.+Turnerauthor=T.+Jinkersonauthor=B.+S.+Brownauthor=R.+G.+Keddyauthor=K.+Lukinauthor=H.+A.+McDonaldauthor=P.+Honoreauthor=J.+Mikusaauthor=K.+C.+Marshauthor=J.+M.+Wetterauthor=K.+St.+Georgeauthor=M.+F.+Jarvisauthor=C.+R.+Faltynekauthor=C.+H.+Lee&title=In+vitro+structure-activity+relationship+and+in+vivo+characterization+of+1-%28Aryl%29-3-%284-%28amino%29benzyl%29urea+transient+receptor+potential+vanilloid+1+antagonists&doi=10.1021%2Fjm070276i"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm070276i&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070276i%26sid%3Dliteratum%253Aachs%26aulast%3DPerner%26aufirst%3DR.%2BJ.%26aulast%3DDiDomenico%26aufirst%3DS.%26aulast%3DKoenig%26aufirst%3DJ.%2BR.%26aulast%3DGomtsyan%26aufirst%3DA.%26aulast%3DBayburt%26aufirst%3DE.%2BK.%26aulast%3DSchmidt%26aufirst%3DR.%2BG.%26aulast%3DDrizin%26aufirst%3DI.%26aulast%3DZheng%26aufirst%3DG.%2BZ.%26aulast%3DTurner%26aufirst%3DS.%2BC.%26aulast%3DJinkerson%26aufirst%3DT.%26aulast%3DBrown%26aufirst%3DB.%2BS.%26aulast%3DKeddy%26aufirst%3DR.%2BG.%26aulast%3DLukin%26aufirst%3DK.%26aulast%3DMcDonald%26aufirst%3DH.%2BA.%26aulast%3DHonore%26aufirst%3DP.%26aulast%3DMikusa%26aufirst%3DJ.%26aulast%3DMarsh%26aufirst%3DK.%2BC.%26aulast%3DWetter%26aufirst%3DJ.%2BM.%26aulast%3DSt.%2BGeorge%26aufirst%3DK.%26aulast%3DJarvis%26aufirst%3DM.%2BF.%26aulast%3DFaltynek%26aufirst%3DC.%2BR.%26aulast%3DLee%26aufirst%3DC.%2BH.%26atitle%3DIn%2520vitro%2520structure-activity%2520relationship%2520and%2520in%2520vivo%2520characterization%2520of%25201-%2528Aryl%2529-3-%25284-%2528amino%2529benzyl%2529urea%2520transient%2520receptor%2520potential%2520vanilloid%25201%2520antagonists%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D3651%26epage%3D3660%26doi%3D10.1021%2Fjm070276i" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Koyanagi, T.; Tsukuda, S.; Nakamura, Y.; Tsunematsu, K.; Shimizu, M.</span>; <span class="NLM_contrib-group">Iwamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, M.</span><span> </span><span class="NLM_article-title">Plant disease control agent containing arylamidine derivative or salt thereof</span>. WO 2013062024,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=T.+Koyanagi&author=S.+Tsukuda&author=Y.+Nakamura&author=K.+Tsunematsu&author=M.+Shimizuauthor=T.+Iwamotoauthor=Y.+Kawaiauthor=M.+Ohno&title=Plant+disease+control+agent+containing+arylamidine+derivative+or+salt+thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKoyanagi%26aufirst%3DT.%26aulast%3DIwamoto%26aufirst%3DT.%26aulast%3DKawai%26aufirst%3DY.%26aulast%3DOhno%26aufirst%3DM.%26atitle%3DPlant%2520disease%2520control%2520agent%2520containing%2520arylamidine%2520derivative%2520or%2520salt%2520thereof%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">de Jong, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engbersen, J. F. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huskens, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhoudt, D. N.</span><span> </span><span class="NLM_article-title">Cyclodextrin dimers as receptor molecules for steroid sensors</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">4034</span><span class="NLM_x">–</span> <span class="NLM_lpage">4040</span><span class="refDoi"> DOI: 10.1002/1521-3765(20001103)6:21<4034::AID-CHEM4034>3.0.CO;2-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=11126966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD3cXotlajsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2000&pages=4034-4040&author=M.+R.+de+Jongauthor=J.+F.+J.+Engbersenauthor=J.+Huskensauthor=D.+N.+Reinhoudt&title=Cyclodextrin+dimers+as+receptor+molecules+for+steroid+sensors&doi=10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclodextrin dimers as receptor molecules for steroid sensors</span></div><div class="casAuthors">De Jong, Menno R.; Engbersen, Johan F. J.; Huskens, Jurriaan; Reinhoudt, David N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry--A European Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4034-4040</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The dansyl-modified dipropanolamine bridged β-cyclodextrin dimer complexes strongly with the steroidal bile salts.  Relative to native β-cyclodextrin, the binding of cholate and deoxycholate salts is esp. enhanced.  These steroids bind exclusively in a 1:1 fashion.  For other bile salts (chenodeoxycholate, ursodeoxycholate, and lithocholate) both 1:1 and 1:2 complexes were obsd. with stabilities similar to those of native β-cyclodextrin.  This indicates that only one cavity was used, with a small contribution from the 2nd.  The difference is attributed to the absence of a 12-hydroxy group in the 2nd group of steroids.  Comparison with a dimer that lacks the dansyl moiety shows that this group esp. hinders the cooperative binding of cholate and deoxycholate.  The smaller interference in the binding of the other steroids indicates that self-inclusion of the dansyl moiety hardly occurs.  This weak self-inclusion is supported by fluorescence studies.  The dansyl fluorescence of dimer is less blue-shifted than that of other known dansyl-appended cyclodextrin derivs.; this is indicative of a more polar micro-environment.  Addn. of guests causes a change in fluorescence intensity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolyZHcivP5b7Vg90H21EOLACvtfcHk0lgK3dqWScuFvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXotlajsrs%253D&md5=7ceed8c31a40403d84e7475c436ae90d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1521-3765%252820001103%25296%253A21%253C4034%253A%253AAID-CHEM4034%253E3.0.CO%253B2-3%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BJong%26aufirst%3DM.%2BR.%26aulast%3DEngbersen%26aufirst%3DJ.%2BF.%2BJ.%26aulast%3DHuskens%26aufirst%3DJ.%26aulast%3DReinhoudt%26aufirst%3DD.%2BN.%26atitle%3DCyclodextrin%2520dimers%2520as%2520receptor%2520molecules%2520for%2520steroid%2520sensors%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2000%26volume%3D6%26spage%3D4034%26epage%3D4040%26doi%3D10.1002%2F1521-3765%2820001103%296%3A21%3C4034%3A%3AAID-CHEM4034%3E3.0.CO%3B2-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">The effects of CapZ peptide (TRTK-12) binding to S100B-Ca2+ as examined by NMR and X-ray crystallography</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">396</span><span class="NLM_x">, </span> <span class="NLM_fpage">1227</span><span class="NLM_x">–</span> <span class="NLM_lpage">1243</span><span class="refDoi"> DOI: 10.1016/j.jmb.2009.12.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.jmb.2009.12.057" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=396&publication_year=2010&pages=1227-1243&author=T.+H.+Charpentierauthor=L.+E.+Thompsonauthor=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=P.+T.+Wilderauthor=E.+Pozharskiauthor=E.+A.+Tothauthor=D.+J.+Weber&title=The+effects+of+CapZ+peptide+%28TRTK-12%29+binding+to+S100B-Ca2%2B+as+examined+by+NMR+and+X-ray+crystallography&doi=10.1016%2Fj.jmb.2009.12.057"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2009.12.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2009.12.057%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DThompson%26aufirst%3DL.%2BE.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DThe%2520effects%2520of%2520CapZ%2520peptide%2520%2528TRTK-12%2529%2520binding%2520to%2520S100B-Ca2%252B%2520as%2520examined%2520by%2520NMR%2520and%2520X-ray%2520crystallography%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D396%26spage%3D1227%26epage%3D1243%26doi%3D10.1016%2Fj.jmb.2009.12.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gianni, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">In vitro screening and structural characterization of inhibitors of the S100B-p53 interaction</span> <span class="citation_source-journal">Int. J. High Throughput Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2010</span><span class="NLM_x">, </span> <span class="NLM_fpage">109</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.2147/IJHTS.S8210</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.2147%2FIJHTS.S8210" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BC2srmt1OmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2010&publication_year=2010&pages=109-126&author=P.+T.+Wilderauthor=T.+H.+Charpentierauthor=M.+A.+Lirianoauthor=K.+Gianniauthor=K.+M.+Varneyauthor=E.+Pozharskiauthor=A.+Coopauthor=E.+A.+Tothauthor=A.+D.+Mackerellauthor=D.+J.+Weber&title=In+vitro+screening+and+structural+characterization+of+inhibitors+of+the+S100B-p53+interaction&doi=10.2147%2FIJHTS.S8210"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">In vitro screening and structural characterization of inhibitors of the S100B-p53 interaction</span></div><div class="casAuthors">Wilder Paul T; Charpentier Thomas H; Liriano Melissa A; Gianni Kira; Varney Kristen M; Pozharski Edwin; Coop Andrew; Toth Eric A; Mackerell Alex D; Weber David J</div><div class="citationInfo"><span class="NLM_cas:title">International journal of high throughput screening</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2010</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-126</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">S100B is highly over-expressed in many cancers, including malignant melanoma.  In such cancers, S100B binds wild-type p53 in a calcium-dependent manner, sequestering it, and promoting its degradation, resulting in the loss of p53-dependent tumor suppression activities.  Therefore, S100B inhibitors may be able to restore wild-type p53 levels in certain cancers and provide a useful therapeutic strategy.  In this regard, an automated and sensitive fluorescence polarization competition assay (FPCA) was developed and optimized to screen rapidly for lead compounds that bind Ca(2+)-loaded S100B and inhibit S100B target complex formation.  A screen of 2000 compounds led to the identification of 26 putative S100B low molecular weight inhibitors.  The binding of these small molecules to S100B was confirmed by nuclear magnetic resonance spectroscopy, and additional structural information was provided by x-ray crystal structures of several compounds in complexes with S100B.  Notably, many of the identified inhibitors function by chemically modifying Cys84 in protein.  These results validate the use of high-throughput FPCA to facilitate the identification of compounds that inhibit S100B.  These lead compounds will be the subject of future optimization studies with the ultimate goal of developing a drug with therapeutic activity for the treatment of malignant melanoma and/or other cancers with elevated S100B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTdL3gvK7FnJNHxxy173TfNfW6udTcc2eZDgviQ49KUCLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srmt1OmtQ%253D%253D&md5=d6384f2ba3954e6520c8c452bc6ce311</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.2147%2FIJHTS.S8210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2147%252FIJHTS.S8210%26sid%3Dliteratum%253Aachs%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DGianni%26aufirst%3DK.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DIn%2520vitro%2520screening%2520and%2520structural%2520characterization%2520of%2520inhibitors%2520of%2520the%2520S100B-p53%2520interaction%26jtitle%3DInt.%2520J.%2520High%2520Throughput%2520Screening%26date%3D2010%26volume%3D2010%26spage%3D109%26epage%3D126%26doi%3D10.2147%2FIJHTS.S8210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Matsumura, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiba, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inoue, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kai, Y.</span><span> </span><span class="NLM_article-title">A novel mode of target recognition suggested by the 2.0 A structure of holo S100B from bovine brain</span> <span class="citation_source-journal">Structure</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">233</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1016/S0969-2126(98)00024-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2FS0969-2126%2898%2900024-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=233-241&author=H.+Matsumuraauthor=T.+Shibaauthor=T.+Inoueauthor=S.+Haradaauthor=Y.+Kai&title=A+novel+mode+of+target+recognition+suggested+by+the+2.0+A+structure+of+holo+S100B+from+bovine+brain&doi=10.1016%2FS0969-2126%2898%2900024-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2FS0969-2126%2898%2900024-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0969-2126%252898%252900024-0%26sid%3Dliteratum%253Aachs%26aulast%3DMatsumura%26aufirst%3DH.%26aulast%3DShiba%26aufirst%3DT.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHarada%26aufirst%3DS.%26aulast%3DKai%26aufirst%3DY.%26atitle%3DA%2520novel%2520mode%2520of%2520target%2520recognition%2520suggested%2520by%2520the%25202.0%2520A%2520structure%2520of%2520holo%2520S100B%2520from%2520bovine%2520brain%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D233%26epage%3D241%26doi%3D10.1016%2FS0969-2126%2898%2900024-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coop, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pozharski, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Small molecules bound to unique sites in the target protein binding cleft of calcium-bound S100B as characterized by nuclear magnetic resonance and X-ray crystallography</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6202</span><span class="NLM_x">–</span> <span class="NLM_lpage">6212</span><span class="refDoi"> DOI: 10.1021/bi9005754</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9005754" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1Clsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2009&pages=6202-6212&author=T.+H.+Charpentierauthor=P.+T.+Wilderauthor=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=S.+Zhongauthor=A.+Coopauthor=E.+Pozharskiauthor=A.+D.+MacKerellauthor=E.+A.+Tothauthor=D.+J.+Weber&title=Small+molecules+bound+to+unique+sites+in+the+target+protein+binding+cleft+of+calcium-bound+S100B+as+characterized+by+nuclear+magnetic+resonance+and+X-ray+crystallography&doi=10.1021%2Fbi9005754"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules Bound to Unique Sites in the Target Protein Binding Cleft of Calcium-Bound S100B As Characterized by Nuclear Magnetic Resonance and X-ray Crystallography</span></div><div class="casAuthors">Charpentier, Thomas H.; Wilder, Paul T.; Liriano, Melissa A.; Varney, Kristen M.; Zhong, Shijun; Coop, Andrew; Pozharski, Edwin; MacKerell, Alexander D.; Toth, Eric A.; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">6202-6212</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structural studies are part of a rational drug design program aimed at inhibiting the S100B-p53 interaction and restoring wild-type p53 function in malignant melanoma.  To this end, structures of three compds. (SBi132, SBi1279, and SBi523) bound to Ca2+-S100B were detd. by X-ray crystallog. at 2.10 Å (Rfree = 0.257), 1.98 Å (Rfree = 0.281), and 1.90 Å (Rfree = 0.228) resoln., resp.  Upon comparison, SBi132, SBi279, and SBi523 were found to bind in distinct locations and orientations within the hydrophobic target binding pocket of Ca2+-S100B with minimal structural changes obsd. for the protein upon complex formation with each compd.  Specifically, SBi132 binds nearby residues in loop 2 (His-42, Phe-43, and Leu-44) and helix 4 (Phe-76, Met-79, Ile-80, Ala-83, Cys-84, Phe-87, and Phe-88), whereas SBi523 interacts with a sep. site defined by residues within loop 2 (Ser-41, His-42, Phe-43, Leu-44, Glu-45, and Glu-46) and one residue on helix 4 (Phe-87).  The SBi279 binding site on Ca2+-S100B overlaps the SBi132 and SBi523 sites and contacts residues in both loop 2 (Ser-41, His-42, Phe-43, Leu-44, and Glu-45) and helix 4 (Ile-80, Ala-83, Cys-84, Phe-87, and Phe-88).  NMR data, including satn. transfer difference (STD) and 15N backbone and 13C side chain chem. shift perturbations, were consistent with the X-ray crystal structures and demonstrated the relevance of all three small mol.-S100B complexes in soln.  The discovery that SBi132, SBi279, and SBi523 bind to proximal sites on Ca2+-S100B could be useful for the development of a new class of mol.(s) that interacts with one or more of these binding sites simultaneously, thereby yielding novel tight binding inhibitors specific for blocking protein-protein interactions involving S100B.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdiyYUBektHLVg90H21EOLACvtfcHk0lhgdEQhPx1kPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1Clsr4%253D&md5=77ccf09a15aae75bdcde6dfabb996370</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fbi9005754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9005754%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DCoop%26aufirst%3DA.%26aulast%3DPozharski%26aufirst%3DE.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DSmall%2520molecules%2520bound%2520to%2520unique%2520sites%2520in%2520the%2520target%2520protein%2520binding%2520cleft%2520of%2520calcium-bound%2520S100B%2520as%2520characterized%2520by%2520nuclear%2520magnetic%2520resonance%2520and%2520X-ray%2520crystallography%26jtitle%3DBiochemistry%26date%3D2009%26volume%3D48%26spage%3D6202%26epage%3D6212%26doi%3D10.1021%2Fbi9005754" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bair, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charpentier, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liriano, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppenheim, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adhya, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carrier, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Recognition of the tumor suppressor protein p53 and other protein targets by the calcium-binding protein S100B</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1763</span><span class="NLM_x">, </span> <span class="NLM_fpage">1284</span><span class="NLM_x">–</span> <span class="NLM_lpage">1297</span><span class="refDoi"> DOI: 10.1016/j.bbamcr.2006.08.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.bbamcr.2006.08.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=17010455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CnsLbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1763&publication_year=2006&pages=1284-1297&author=P.+T.+Wilderauthor=J.+Linauthor=C.+L.+Bairauthor=T.+H.+Charpentierauthor=D.+Yangauthor=M.+Lirianoauthor=K.+M.+Varneyauthor=A.+Leeauthor=A.+B.+Oppenheimauthor=S.+Adhyaauthor=F.+Carrierauthor=D.+J.+Weber&title=Recognition+of+the+tumor+suppressor+protein+p53+and+other+protein+targets+by+the+calcium-binding+protein+S100B&doi=10.1016%2Fj.bbamcr.2006.08.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Recognition of the tumor suppressor protein p53 and other protein targets by the calcium-binding protein S100B</span></div><div class="casAuthors">Wilder, Paul T.; Lin, Jing; Bair, Catherine L.; Charpentier, Thomas H.; Yang, Dong; Liriano, Melissa; Varney, Kristen M.; Lee, Andrew; Oppenheim, Amos B.; Adhya, Sankar; Carrier, France; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular Cell Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1763</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1284-1297</span>CODEN:
                <span class="NLM_cas:coden">BBAMCO</span>;
        ISSN:<span class="NLM_cas:issn">0167-4889</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">S100B is an EF-hand contg. calcium-binding protein of the S100 protein family that exerts its biol. effect by binding and affecting various target proteins.  A consensus sequence for S100B target proteins was published as (K/R)(L/I)xWxxIL and matches a region in the actin-capping protein CapZ (V. V. Ivanenkov, G. A. Jamieson, Jr., E. Gruenstein, R. V. Dimlich, Characterization of S-100b binding epitopes. Identification of a novel target, the actin capping protein, CapZ, J. Biol. Chem. 270 (1995) 14651-14658).  Several addnl. S100B targets are known including p53, a nuclear Dbf2-related (NDR) kinase, the RAGE receptor, neuromodulin, protein kinase C, and others.  Examg. the binding sites of such targets and new protein sequence searches provided addnl. potential target proteins for S100B including Hdm2 and Hdm4, which were both found to bind S100B in a calcium-dependent manner.  The interaction between S100B and the Hdm2 and/or the Hdm4 proteins may be important physiol. in light of evidence that like Hdm2, S100B also contributes to lowering protein levels of the tumor suppressor protein, p53.  For the S100B-p53 interaction, it was found that phosphorylation of specific serine and/or threonine residues reduces the affinity of the S100B-p53 interaction by as much as an order of magnitude, and is important for protecting p53 from S100B-dependent down-regulation, a scenario that is similar to what is found for the Hdm2-p53 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrQ0k0TbFI5bVg90H21EOLACvtfcHk0lhgdEQhPx1kPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1CnsLbJ&md5=0354920ca745638506d75a1fd54dc057</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bbamcr.2006.08.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbamcr.2006.08.024%26sid%3Dliteratum%253Aachs%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DBair%26aufirst%3DC.%2BL.%26aulast%3DCharpentier%26aufirst%3DT.%2BH.%26aulast%3DYang%26aufirst%3DD.%26aulast%3DLiriano%26aufirst%3DM.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DLee%26aufirst%3DA.%26aulast%3DOppenheim%26aufirst%3DA.%2BB.%26aulast%3DAdhya%26aufirst%3DS.%26aulast%3DCarrier%26aufirst%3DF.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DRecognition%2520of%2520the%2520tumor%2520suppressor%2520protein%2520p53%2520and%2520other%2520protein%2520targets%2520by%2520the%2520calcium-binding%2520protein%2520S100B%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Res.%26date%3D2006%26volume%3D1763%26spage%3D1284%26epage%3D1297%26doi%3D10.1016%2Fj.bbamcr.2006.08.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Liriano, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varney, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toth, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishima, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Target Binding to S100B Reduces Dynamic Properties and Increases Ca(2+)-Binding Affinity for Wild Type and EF-Hand Mutant Proteins</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">423</span><span class="NLM_x">, </span> <span class="NLM_fpage">365</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1016/j.jmb.2012.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2Fj.jmb.2012.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=22824086" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Skur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=423&publication_year=2012&pages=365-385&author=M.+A.+Lirianoauthor=K.+M.+Varneyauthor=N.+T.+Wrightauthor=C.+L.+Hoffmanauthor=E.+A.+Tothauthor=R.+Ishimaauthor=D.+J.+Weber&title=Target+Binding+to+S100B+Reduces+Dynamic+Properties+and+Increases+Ca%282%2B%29-Binding+Affinity+for+Wild+Type+and+EF-Hand+Mutant+Proteins&doi=10.1016%2Fj.jmb.2012.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Target Binding to S100B Reduces Dynamic Properties and Increases Ca2+-Binding Affinity for Wild Type and EF-Hand Mutant Proteins</span></div><div class="casAuthors">Liriano, Melissa A.; Varney, Kristen M.; Wright, Nathan T.; Hoffman, Cassandra L.; Toth, Eric A.; Ishima, Rieko; Weber, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">423</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">365-385</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Mutations in the second EF-hand (D61N, D63N, D65N, and E72A) of S100B were used to study its Ca2+ binding and dynamic properties in the absence and presence of a bound target, TRTK-12.  With D63NS100B as an exception (D63NKD = 50 ± 9 μM), Ca2+ binding to EF2-hand mutants were reduced by more than 8-fold in the absence of TRTK-12 (D61NKD = 412 ± 67 μM, D65NKD = 968 ± 171 μM, and E72AKD = 471 ± 133 μM), when compared to wild-type protein (WTKD = 56 ± 9 μM).  For the TRTK-12 complexes, the Ca2+-binding affinity to wild type (WT+TRTKKD = 12 ± 10 μM) and the EF2 mutants was increased by 5- to 14-fold vs. in the absence of target (D61N+TRTKKD = 29 ± 1.2 μM, D63N+TRTKKD = 10 ± 2.2 μM, D65N+TRTKKD = 73 ± 4.4 μM, and E72A+TRTKKD = 18 ± 3.7 μM).  In addn., Rex, as measured using relaxation dispersion for side-chain 15N resonances of Asn63 (D63NS100B), was reduced upon TRTK-12 binding when measured by NMR.  Likewise, backbone motions on multiple timescales (picoseconds to milliseconds) throughout wild type, D61NS100B, D63NS100B, and D65NS100B were lowered upon binding TRTK-12.  However, the X-ray structures of Ca2+-bound (2.0 Å) and TRTK-bound (1.2 Å) D63NS100B showed no change in Ca2+ coordination; thus, these and analogous structural data for the wild-type protein could not be used to explain how target binding increased Ca2+-binding affinity in soln.  Therefore, a model for how S100B-TRTK-12 complex formation increases Ca2+ binding is discussed, which considers changes in protein dynamics upon binding the target TRTK-12.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrx_vsGCby77Vg90H21EOLACvtfcHk0lhxpLnEVDyY9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1Skur3E&md5=42fc8bae1a6b41f2c67d5bea4355474b</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2012.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2012.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DLiriano%26aufirst%3DM.%2BA.%26aulast%3DVarney%26aufirst%3DK.%2BM.%26aulast%3DWright%26aufirst%3DN.%2BT.%26aulast%3DHoffman%26aufirst%3DC.%2BL.%26aulast%3DToth%26aufirst%3DE.%2BA.%26aulast%3DIshima%26aufirst%3DR.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DTarget%2520Binding%2520to%2520S100B%2520Reduces%2520Dynamic%2520Properties%2520and%2520Increases%2520Ca%25282%252B%2529-Binding%2520Affinity%2520for%2520Wild%2520Type%2520and%2520EF-Hand%2520Mutant%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D423%26spage%3D365%26epage%3D385%26doi%3D10.1016%2Fj.jmb.2012.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Hartman, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vitolo, M. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilder, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Complex Formation between S100B Protein and the p90 Ribosomal S6 Kinase (RSK) in Malignant Melanoma Is Calcium-dependent and Inhibits Extracellular Signal-regulated Kinase (ERK)-mediated Phosphorylation of RSK</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">12886</span><span class="NLM_x">–</span> <span class="NLM_lpage">12895</span><span class="refDoi"> DOI: 10.1074/jbc.M114.561613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1074%2Fjbc.M114.561613" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=12886-12895&author=K.+G.+Hartmanauthor=M.+I.+Vitoloauthor=A.+D.+Pierceauthor=J.+M.+Foxauthor=P.+Shapiroauthor=S.+S.+Martinauthor=P.+T.+Wilderauthor=D.+J.+Weber&title=Complex+Formation+between+S100B+Protein+and+the+p90+Ribosomal+S6+Kinase+%28RSK%29+in+Malignant+Melanoma+Is+Calcium-dependent+and+Inhibits+Extracellular+Signal-regulated+Kinase+%28ERK%29-mediated+Phosphorylation+of+RSK&doi=10.1074%2Fjbc.M114.561613"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.561613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.561613%26sid%3Dliteratum%253Aachs%26aulast%3DHartman%26aufirst%3DK.%2BG.%26aulast%3DVitolo%26aufirst%3DM.%2BI.%26aulast%3DPierce%26aufirst%3DA.%2BD.%26aulast%3DFox%26aufirst%3DJ.%2BM.%26aulast%3DShapiro%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BS.%26aulast%3DWilder%26aufirst%3DP.%2BT.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DComplex%2520Formation%2520between%2520S100B%2520Protein%2520and%2520the%2520p90%2520Ribosomal%2520S6%2520Kinase%2520%2528RSK%2529%2520in%2520Malignant%2520Melanoma%2520Is%2520Calcium-dependent%2520and%2520Inhibits%2520Extracellular%2520Signal-regulated%2520Kinase%2520%2528ERK%2529-mediated%2520Phosphorylation%2520of%2520RSK%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D12886%26epage%3D12895%26doi%3D10.1074%2Fjbc.M114.561613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Bachman, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sager, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheong, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catto, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardelli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span> </span><span class="NLM_article-title">Identification of compounds that inhibit growth of 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine-resistant cancer cells</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1030</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-05-0038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1158%2F1535-7163.MCT-05-0038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=15956261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=1026-1030&author=K.+E.+Bachmanauthor=J.+Sagerauthor=I.+Cheongauthor=M.+Cattoauthor=A.+Bardelliauthor=B.+H.+Parkauthor=B.+Vogelsteinauthor=A.+Carottiauthor=K.+W.+Kinzlerauthor=C.+Lengauer&title=Identification+of+compounds+that+inhibit+growth+of+2-amino-1-methyl-6-phenylimidazo%284%2C5-b%29pyridine-resistant+cancer+cells&doi=10.1158%2F1535-7163.MCT-05-0038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of compounds that inhibit growth of 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine-resistant cancer cells</span></div><div class="casAuthors">Bachman, Kurtis E.; Sager, Jason; Cheong, Ian; Catto, Marco; Bardelli, Alberto; Park, Ben Ho; Vogelstein, Bert; Carotti, Angelo; Kinzler, Kenneth W.; Lengauer, Christoph</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1026-1030</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The dietary carcinogen 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine (PhIP) is a heterocyclic amine and is a common byproduct of cooked meat and fish.  Although most cells undergo apoptosis when exposed to this mutagen, subsets develop resistance.  Rather than die, these resistant cells persist and accumulate mutations, thereby driving tumorigenesis of exposed organs within the gastrointestinal tract.  By applying a high-throughput cell-based screen of 32,000 small mols., we have identified a family of compds. that specifically inhibit the growth of PhIP-resistant cancer cells.  These compds. may prove useful for the treatment or prevention of gastrointestinal tumors arising after exposure to PhIP and related carcinogens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4mDWR1eMdSrVg90H21EOLACvtfcHk0lhxpLnEVDyY9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltFWrsLc%253D&md5=96a2eb4d56adf22774222dd648576c66</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-05-0038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-05-0038%26sid%3Dliteratum%253Aachs%26aulast%3DBachman%26aufirst%3DK.%2BE.%26aulast%3DSager%26aufirst%3DJ.%26aulast%3DCheong%26aufirst%3DI.%26aulast%3DCatto%26aufirst%3DM.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520compounds%2520that%2520inhibit%2520growth%2520of%25202-amino-1-methyl-6-phenylimidazo%25284%252C5-b%2529pyridine-resistant%2520cancer%2520cells%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2005%26volume%3D4%26spage%3D1026%26epage%3D1030%26doi%3D10.1158%2F1535-7163.MCT-05-0038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Myers, M. A.</span><span> </span><span class="NLM_article-title">Direct measurement of cell numbers in microtitre plate cultures using the fluorescent dye SYBR green I</span> <span class="citation_source-journal">J. Immunol. Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">212</span><span class="NLM_x">, </span> <span class="NLM_fpage">99</span><span class="NLM_x">–</span> <span class="NLM_lpage">103</span><span class="refDoi"> DOI: 10.1016/S0022-1759(98)00011-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1016%2FS0022-1759%2898%2900011-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=9671157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADyaK1cXjtVKlsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=212&publication_year=1998&pages=99-103&author=M.+A.+Myers&title=Direct+measurement+of+cell+numbers+in+microtitre+plate+cultures+using+the+fluorescent+dye+SYBR+green+I&doi=10.1016%2FS0022-1759%2898%2900011-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Direct measurement of cell numbers in microtitre plate cultures using the fluorescent dye SYBR green I</span></div><div class="casAuthors">Myers, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Immunological Methods</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">212</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">99-103</span>CODEN:
                <span class="NLM_cas:coden">JIMMBG</span>;
        ISSN:<span class="NLM_cas:issn">0022-1759</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">The quantitation of cell nos. is essential for many exptl. procedures.  This study describes the use of the fluorescent DNA binding dye, SYBR green I, to quantitate accurately cell nos. in the wells of microtitre plates by detn. of DNA content.  The assay involves a single-step procedure and is sensitive to as few as 1000 cells/well.  The advantages of SYBR green I are the low level of background fluorescence in the absence of DNA and the similarities of its absorption and emission spectra to those of fluorescein, which makes it possible to use a wide range of equipment for detection of the amt. of bound dye.  A particular application of the method is the normalization of data generated by enzyme-linked immunosorbent assays on whole cells.  The procedure is applicable to both fixed and unfixed cells, although for fixed cells permeabilisation provides for greater sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJmFe4xjlzzbVg90H21EOLACvtfcHk0lhz6LSZpQnneg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXjtVKlsbk%253D&md5=4802e4136dc659cbb2120d00a382d423</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1016%2FS0022-1759%2898%2900011-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-1759%252898%252900011-8%26sid%3Dliteratum%253Aachs%26aulast%3DMyers%26aufirst%3DM.%2BA.%26atitle%3DDirect%2520measurement%2520of%2520cell%2520numbers%2520in%2520microtitre%2520plate%2520cultures%2520using%2520the%2520fluorescent%2520dye%2520SYBR%2520green%2520I%26jtitle%3DJ.%2520Immunol.%2520Methods%26date%3D1998%26volume%3D212%26spage%3D99%26epage%3D103%26doi%3D10.1016%2FS0022-1759%2898%2900011-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Drohat, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amburgey, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abildgaard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starich, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldisseri, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">Solution structure of rat apo-S100B(beta beta) as determined by NMR spectroscopy</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">11577</span><span class="NLM_x">–</span> <span class="NLM_lpage">11588</span><span class="refDoi"> DOI: 10.1021/bi9612226</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9612226" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=1996&pages=11577-11588&author=A.+C.+Drohatauthor=J.+C.+Amburgeyauthor=F.+Abildgaardauthor=M.+R.+Starichauthor=D.+Baldisseriauthor=D.+J.+Weber&title=Solution+structure+of+rat+apo-S100B%28beta+beta%29+as+determined+by+NMR+spectroscopy&doi=10.1021%2Fbi9612226"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Fbi9612226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9612226%26sid%3Dliteratum%253Aachs%26aulast%3DDrohat%26aufirst%3DA.%2BC.%26aulast%3DAmburgey%26aufirst%3DJ.%2BC.%26aulast%3DAbildgaard%26aufirst%3DF.%26aulast%3DStarich%26aufirst%3DM.%2BR.%26aulast%3DBaldisseri%26aufirst%3DD.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3DSolution%2520structure%2520of%2520rat%2520apo-S100B%2528beta%2520beta%2529%2520as%2520determined%2520by%2520NMR%2520spectroscopy%26jtitle%3DBiochemistry%26date%3D1996%26volume%3D35%26spage%3D11577%26epage%3D11588%26doi%3D10.1021%2Fbi9612226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Amburgey, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abildgaard, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Starich, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hilt, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, D. J.</span><span> </span><span class="NLM_article-title">1H, 13C and 15N NMR assignments and solution secondary structure of rat Apo-S100 beta</span> <span class="citation_source-journal">J. Biomol. NMR</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">171</span><span class="NLM_x">–</span> <span class="NLM_lpage">179</span><span class="refDoi"> DOI: 10.1007/BF00211781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1007%2FBF00211781" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1995&pages=171-179&author=J.+C.+Amburgeyauthor=F.+Abildgaardauthor=M.+R.+Starichauthor=S.+Shahauthor=D.+C.+Hiltauthor=D.+J.+Weber&title=1H%2C+13C+and+15N+NMR+assignments+and+solution+secondary+structure+of+rat+Apo-S100+beta&doi=10.1007%2FBF00211781"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1007%2FBF00211781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00211781%26sid%3Dliteratum%253Aachs%26aulast%3DAmburgey%26aufirst%3DJ.%2BC.%26aulast%3DAbildgaard%26aufirst%3DF.%26aulast%3DStarich%26aufirst%3DM.%2BR.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DHilt%26aufirst%3DD.%2BC.%26aulast%3DWeber%26aufirst%3DD.%2BJ.%26atitle%3D1H%252C%252013C%2520and%252015N%2520NMR%2520assignments%2520and%2520solution%2520secondary%2520structure%2520of%2520rat%2520Apo-S100%2520beta%26jtitle%3DJ.%2520Biomol.%2520NMR%26date%3D1995%26volume%3D6%26spage%3D171%26epage%3D179%26doi%3D10.1007%2FBF00211781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span> </span><span class="NLM_article-title">An automated system to mount cryo-cooled protein crystals on a synchrotron beamline, using compact sample cassettes and a small-scale robot</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">720</span><span class="NLM_x">–</span> <span class="NLM_lpage">726</span><span class="refDoi"> DOI: 10.1107/S0021889802016709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0021889802016709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=24899734" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD38Xos1emtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2002&pages=720-726&author=A.+E.+Cohenauthor=P.+J.+Ellisauthor=M.+D.+Millerauthor=A.+M.+Deaconauthor=R.+P.+Phizackerley&title=An+automated+system+to+mount+cryo-cooled+protein+crystals+on+a+synchrotron+beamline%2C+using+compact+sample+cassettes+and+a+small-scale+robot&doi=10.1107%2FS0021889802016709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">An automated system to mount cryo-cooled protein crystals on a synchrotron beamline, using compact sample cassettes and a small-scale robot</span></div><div class="casAuthors">Cohen, Aina E.; Ellis, Paul J.; Miller, Mitchell D.; Deacon, Ashley M.; Phizackerley, R. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">720-726</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">An automated system for mounting and dismounting pre-frozen crystals has been implemented at the Stanford Synchrotron Radiation Lab. (SSRL).  It is based on a small industrial robot and compact cylindrical cassettes, each holding up to 96 crystals mounted on Hampton Research sample pins.  For easy shipping and storage, the cassette fits inside several popular dry-shippers and long-term storage Dewars.  A dispensing Dewar holds up to three cassettes in liq. nitrogen adjacent to the beamline goniometer.  The robot uses a permanent magnet tool to ext. samples from, and insert samples into a cassette, and a cryo-tong tool to transfer them to and from the beamline goniometer.  The system is simple, with few moving parts, reliable in operation and convenient to use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoreUoI2UBMKbVg90H21EOLACvtfcHk0lhz6LSZpQnneg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xos1emtL8%253D&md5=5dae59f9642fa186e2cd42bce3f76ee2</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0021889802016709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889802016709%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26atitle%3DAn%2520automated%2520system%2520to%2520mount%2520cryo-cooled%2520protein%2520crystals%2520on%2520a%2520synchrotron%2520beamline%252C%2520using%2520compact%2520sample%2520cassettes%2520and%2520a%2520small-scale%2520robot%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2002%26volume%3D35%26spage%3D720%26epage%3D726%26doi%3D10.1107%2FS0021889802016709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Soltis, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eriksson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chui, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunten, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollenbeck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathews, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moorhead, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van dem Bedem, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharp, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsyba, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, G.</span><span> </span><span class="NLM_article-title">New paradigm for macromolecular crystallography experiments at SSRL: automated crystal screening and remote data collection</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">64</span><span class="NLM_x">, </span> <span class="NLM_fpage">1210</span><span class="NLM_x">–</span> <span class="NLM_lpage">1221</span><span class="refDoi"> DOI: 10.1107/S0907444908030564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444908030564" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2008&pages=1210-1221&author=S.+M.+Soltisauthor=A.+E.+Cohenauthor=A.+Deaconauthor=T.+Erikssonauthor=A.+Gonzalezauthor=S.+McPhillipsauthor=H.+Chuiauthor=P.+Duntenauthor=M.+Hollenbeckauthor=I.+Mathewsauthor=M.+Millerauthor=P.+Moorheadauthor=R.+P.+Phizackerleyauthor=C.+Smithauthor=J.+Songauthor=H.+van+dem+Bedemauthor=P.+Ellisauthor=P.+Kuhnauthor=T.+McPhillipsauthor=N.+Sauterauthor=K.+Sharpauthor=I.+Tsybaauthor=G.+Wolf&title=New+paradigm+for+macromolecular+crystallography+experiments+at+SSRL%3A+automated+crystal+screening+and+remote+data+collection&doi=10.1107%2FS0907444908030564"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS0907444908030564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444908030564%26sid%3Dliteratum%253Aachs%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%26aulast%3DEriksson%26aufirst%3DT.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DMcPhillips%26aufirst%3DS.%26aulast%3DChui%26aufirst%3DH.%26aulast%3DDunten%26aufirst%3DP.%26aulast%3DHollenbeck%26aufirst%3DM.%26aulast%3DMathews%26aufirst%3DI.%26aulast%3DMiller%26aufirst%3DM.%26aulast%3DMoorhead%26aufirst%3DP.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSmith%26aufirst%3DC.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3Dvan%2Bdem%2BBedem%26aufirst%3DH.%26aulast%3DEllis%26aufirst%3DP.%26aulast%3DKuhn%26aufirst%3DP.%26aulast%3DMcPhillips%26aufirst%3DT.%26aulast%3DSauter%26aufirst%3DN.%26aulast%3DSharp%26aufirst%3DK.%26aulast%3DTsyba%26aufirst%3DI.%26aulast%3DWolf%26aufirst%3DG.%26atitle%3DNew%2520paradigm%2520for%2520macromolecular%2520crystallography%2520experiments%2520at%2520SSRL%253A%2520automated%2520crystal%2520screening%2520and%2520remote%2520data%2520collection%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2008%26volume%3D64%26spage%3D1210%26epage%3D1221%26doi%3D10.1107%2FS0907444908030564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">McPhillips, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McPhillips, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deacon, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garman, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauter, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phizackerley, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soltis, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, P.</span><span> </span><span class="NLM_article-title">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span> <span class="citation_source-journal">J. Synchrotron Radiat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span><span class="refDoi"> DOI: 10.1107/S0909049502015170</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0909049502015170" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=12409628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2002&pages=401-406&author=T.+M.+McPhillipsauthor=S.+E.+McPhillipsauthor=H.+J.+Chiuauthor=A.+E.+Cohenauthor=A.+M.+Deaconauthor=P.+J.+Ellisauthor=E.+Garmanauthor=A.+Gonzalezauthor=N.+K.+Sauterauthor=R.+P.+Phizackerleyauthor=S.+M.+Soltisauthor=P.+Kuhn&title=Blu-Ice+and+the+Distributed+Control+System%3A+software+for+data+acquisition+and+instrument+control+at+macromolecular+crystallography+beamlines&doi=10.1107%2FS0909049502015170"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Blu-Ice and the Distributed Control System: software for data acquisition and instrument control at macromolecular crystallography beamlines</span></div><div class="casAuthors">McPhillips, Timothy M.; McPhillips, Scott E.; Chiu, Hsiu Ju; Cohen, Aina E.; Deacon, Ashley M.; Ellis, Paul J.; Garman, Elspeth; Gonzalez, Ana; Sauter, Nicholas K.; Phizackerley, R. Paul; Soltis, S. Michael; Kuhn, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">401-406</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">The Blu-Ice and Distributed Control System (DCS) software packages were developed to provide unified control over the disparate hardware resources available at a macromol. crystallog. beamline.  Blu-Ice is a user interface that provides scientific experimenters and beamline support staff with intuitive graphical tools for collecting diffraction data and configuring beamlines for expts.  Blu-Ice communicates with the hardware at a beamline via DCS, an instrument-control and data-acquisition package designed to integrate hardware resources in a highly heterogeneous networked computing environment.  Together, Blu-Ice and DCS provide a flexible platform for increasing the ease of use, the level of automation and the remote accessibility of beamlines.  Blu-Ice and DCS are currently installed on four Stanford Synchrotron Radiation Lab. crystallog. beamlines and are being implemented at sister light sources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHVsQUIFJPBLVg90H21EOLACvtfcHk0ljcpYepKe5qnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xotleluro%253D&md5=c77228bd50890463f3e43e981a2e238a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0909049502015170&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049502015170%26sid%3Dliteratum%253Aachs%26aulast%3DMcPhillips%26aufirst%3DT.%2BM.%26aulast%3DMcPhillips%26aufirst%3DS.%2BE.%26aulast%3DChiu%26aufirst%3DH.%2BJ.%26aulast%3DCohen%26aufirst%3DA.%2BE.%26aulast%3DDeacon%26aufirst%3DA.%2BM.%26aulast%3DEllis%26aufirst%3DP.%2BJ.%26aulast%3DGarman%26aufirst%3DE.%26aulast%3DGonzalez%26aufirst%3DA.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DPhizackerley%26aufirst%3DR.%2BP.%26aulast%3DSoltis%26aufirst%3DS.%2BM.%26aulast%3DKuhn%26aufirst%3DP.%26atitle%3DBlu-Ice%2520and%2520the%2520Distributed%2520Control%2520System%253A%2520software%2520for%2520data%2520acquisition%2520and%2520instrument%2520control%2520at%2520macromolecular%2520crystallography%2520beamlines%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2002%26volume%3D9%26spage%3D401%26epage%3D406%26doi%3D10.1107%2FS0909049502015170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Gonzalez, A.; Tsai, Y.</span> <span class="citation_source-book">AUTOXDS</span>; <span class="NLM_publisher-name">Stanford University</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span><a href="http://smb.slac.stanford.edu/facilities/software/xds/#autoxds_script" class="extLink">http://smb.slac.stanford.edu/facilities/software/xds/#autoxds_script</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=A.+Gonzalez&author=Y.+Tsai&title=AUTOXDS"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DA.%26btitle%3DAUTOXDS%26pub%3DStanford%2520University%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span> </span><span class="NLM_article-title">Solving structures of protein complexes by molecular replacement with Phaser</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">32</span><span class="NLM_x">–</span> <span class="NLM_lpage">41</span><span class="refDoi"> DOI: 10.1107/S0907444906045975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444906045975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=17164524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=32-41&author=A.+J.+McCoy&title=Solving+structures+of+protein+complexes+by+molecular+replacement+with+Phaser&doi=10.1107%2FS0907444906045975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Solving structures of protein complexes by molecular replacement with Phaser</span></div><div class="casAuthors">McCoy, Airlie J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">32-41</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Mol. replacement (MR) generally becomes more difficult as the no. of components in the asym. unit requiring sep. MR models (i.e. the dimensionality of the search) increases.  When the proportion of the total scattering contributed by each search component is small, the signal in the search for each component in isolation is weak or non-existent.  Maximum-likelihood MR functions enable complex asym. units to be built up from individual components with a 'tree search with pruning' approach.  This method, as implemented in the automated search procedure of the program Phaser, has been very successful in solving many previously intractable MR problems.  However, there are a no. of cases in which the automated search procedure of Phaser is suboptimal or encounters difficulties.  These include cases where there are a large no. of copies of the same component in the asym. unit or where the components of the asym. unit have greatly varying B factors.  Two case studies are presented to illustrate how Phaser can be used to best advantage in the std. 'automated MR' mode and two case studies are used to show how to modify the automated search strategy for problematic cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAvs9W8hQ3l7Vg90H21EOLACvtfcHk0ljcpYepKe5qnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSqu77F&md5=323a5297bdf622ebaf398a87c39dc794</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1107%2FS0907444906045975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906045975%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26atitle%3DSolving%2520structures%2520of%2520protein%2520complexes%2520by%2520molecular%2520replacement%2520with%2520Phaser%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26spage%3D32%26epage%3D41%26doi%3D10.1107%2FS0907444906045975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Mccoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser crystallographic software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+Mccoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljelRmB-6vtcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMccoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkoczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+comprehensive+Python-based+system+for+macromolecular+structure+solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljelRmB-6vtcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520comprehensive%2520Python-based%2520system%2520for%2520macromolecular%2520structure%2520solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0ljelRmB-6vtcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Afonine, P. V.; Grosse-Kunstleve, R. W.; Adams, P. D.</span><span> </span><span class="NLM_article-title">The Phenix Refinement Framework</span>.  <span class="citation_source-journal">CCP4 Newsl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span>, Contribution 8.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2005&author=P.+V.+Afonine&author=R.+W.+Grosse-Kunstleve&author=P.+D.+Adams&title=The+Phenix+Refinement+Framework"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26atitle%3DThe%2520Phenix%2520Refinement%2520Framework%26jtitle%3DCCP4%2520Newsl.%26date%3D2005%26volume%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Computational fragment-based binding site identification by ligand competitive saturation</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1000435</span><span class="refDoi"> DOI: 10.1371/journal.pcbi.1000435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1371%2Fjournal.pcbi.1000435" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19593374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A280%3ADC%252BD1MvnvVGmsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=e1000435&author=O.+Guvenchauthor=A.+D.+MacKerell&title=Computational+fragment-based+binding+site+identification+by+ligand+competitive+saturation&doi=10.1371%2Fjournal.pcbi.1000435"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Computational fragment-based binding site identification by ligand competitive saturation</span></div><div class="casAuthors">Guvench Olgun; MacKerell Alexander D Jr</div><div class="citationInfo"><span class="NLM_cas:title">PLoS computational biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1000435</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery using NMR and x-ray crystallographic methods has proven utility but also non-trivial time, materials, and labor costs.  Current computational fragment-based approaches circumvent these issues but suffer from limited representations of protein flexibility and solvation effects, leading to difficulties with rigorous ranking of fragment affinities.  To overcome these limitations we describe an explicit solvent all-atom molecular dynamics methodology (SILCS: Site Identification by Ligand Competitive Saturation) that uses small aliphatic and aromatic molecules plus water molecules to map the affinity pattern of a protein for hydrophobic groups, aromatic groups, hydrogen bond donors, and hydrogen bond acceptors.  By simultaneously incorporating ligands representative of all these functionalities, the method is an in silico free energy-based competition assay that generates three-dimensional probability maps of fragment binding (FragMaps) indicating favorable fragment:protein interactions.  Applied to the two-fold symmetric oncoprotein BCL-6, the SILCS method yields two-fold symmetric FragMaps that recapitulate the crystallographic binding modes of the SMRT and BCOR peptides.  These FragMaps account both for important sequence and structure differences in the C-terminal halves of the two peptides and also the high mobility of the BCL-6 His116 sidechain in the peptide-binding groove.  Such SILCS FragMaps can be used to qualitatively inform the design of small-molecule inhibitors or as scoring grids for high-throughput in silico docking that incorporate both an atomic-level description of solvation and protein flexibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWmTORSXQt9tNTrvPZP1IpfW6udTcc2eY0gnKWUjewJbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvnvVGmsw%253D%253D&md5=2367585f0d41c2221a812ae552a2217a</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1000435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1000435%26sid%3Dliteratum%253Aachs%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DComputational%2520fragment-based%2520binding%2520site%2520identification%2520by%2520ligand%2520competitive%2520saturation%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3De1000435%26doi%3D10.1371%2Fjournal.pcbi.1000435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.</span><span> </span><span class="NLM_article-title">Reproducing crystal binding modes of ligand functional groups using Site-Identification by Ligand Competitive Saturation (SILCS) simulations</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">877</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1021/ci100462t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100462t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFegtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=877-896&author=E.+P.+Ramanauthor=W.+Yuauthor=O.+Guvenchauthor=A.+D.+Mackerell&title=Reproducing+crystal+binding+modes+of+ligand+functional+groups+using+Site-Identification+by+Ligand+Competitive+Saturation+%28SILCS%29+simulations&doi=10.1021%2Fci100462t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Reproducing Crystal Binding Modes of Ligand Functional Groups Using Site-Identification by Ligand Competitive Saturation (SILCS) Simulations</span></div><div class="casAuthors">Raman, E. Prabhu; Yu, Wenbo; Guvench, Olgun; MacKerell, Alexander D. Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">877-896</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The applicability of a computational method, Site Identification by Ligand Competitive Satn. (SILCS), to identify regions on a protein surface with which different types of functional groups on low-mol. wt. inhibitors interact is demonstrated.  The method involves mol. dynamics (MD) simulations of a protein in an aq. soln. of chem. diverse small mols. from which probability distributions of fragments types, termed FragMaps, are obtained.  In the present application, SILCS simulations are performed with an aq. soln. of 1 M benzene and propane to map the affinity pattern of the protein for arom. and aliph. functional groups.  In addn., water hydrogen and oxygen atoms serve as probes for hydrogen-bond donor and acceptor affinity, resp.  The method is tested using a set of 7 proteins for which crystal structures of complexes with several high affinity inhibitors are known.  Good agreement is obtained between FragMaps and the positions of chem. similar functional groups in inhibitors as obsd. in the x-ray crystallog. structures.  Quant. capabilities of the SILCS approach are demonstrated by converting FragMaps to free energies, termed Grid Free Energies (GFE), and showing correlation between the GFE values and exptl. binding affinities.  For proteins for which ligand decoy sets are available, GFE values are shown to typically score the crystal conformation and conformations similar to it more favorable than decoys.  Addnl., SILCS is tested for its ability to capture the subtle differences in ligand affinity across homologous proteins, information which may be of utility toward specificity-guided drug design.  Taken together, our results show that SILCS can recapitulate the known location of functional groups of bound inhibitors for a no. of proteins, suggesting that the method may be of utility for rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBfyWq2aa2prVg90H21EOLACvtfcHk0ljTWzW9Ulck1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFegtr8%253D&md5=7a80452191971ec6b08086e8bce11e6f</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fci100462t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100462t%26sid%3Dliteratum%253Aachs%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DReproducing%2520crystal%2520binding%2520modes%2520of%2520ligand%2520functional%2520groups%2520using%2520Site-Identification%2520by%2520Ligand%2520Competitive%2520Saturation%2520%2528SILCS%2529%2520simulations%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D877%26epage%3D896%26doi%3D10.1021%2Fci100462t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">461</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0ljTWzW9Ulck1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lakkaraju, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Inclusion of multiple fragment types in the site identification by ligand competitive saturation (SILCS) approach</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">3384</span><span class="NLM_x">–</span> <span class="NLM_lpage">3398</span><span class="refDoi"> DOI: 10.1021/ci4005628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci4005628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslyqt7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=3384-3398&author=E.+P.+Ramanauthor=W.+Yuauthor=S.+K.+Lakkarajuauthor=A.+D.+MacKerell&title=Inclusion+of+multiple+fragment+types+in+the+site+identification+by+ligand+competitive+saturation+%28SILCS%29+approach&doi=10.1021%2Fci4005628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Inclusion of Multiple Fragment Types in the Site Identification by Ligand Competitive Saturation (SILCS) Approach</span></div><div class="casAuthors">Raman, E. Prabhu; Yu, Wenbo; Lakkaraju, Sirish K.; MacKerell, Alexander D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3384-3398</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The site identification by ligand competitive satn. (SILCS) method identifies the location and approx. affinities of small mol. fragments on a target macromol. surface by performing mol. dynamics (MD) simulations of the target in an aq. soln. of small mols. representative of different chem. functional groups.  In this study, we introduce a set of small mols. to map potential interactions made by neutral hydrogen bond donors and acceptors and charged donor and acceptor fragments in addn. to nonpolar fragments.  The affinity pattern is obtained in the form of discretized probability or, equivalently, free energy maps, called FragMaps, which can be visualized with the target surface.  We performed SILCS simulations for four proteins for which structural and thermodn. data is available for multiple diverse ligands.  Good overlap is shown between high affinity regions identified by the FragMaps and the crystallog. positions of ligand functional groups with similar chem. functionality, thus demonstrating the validity of the qual. information obtained from the simulations.  To test the ability of FragMaps in providing quant. predictions, we calc. the previously introduced ligand grid free energy (LGFE) metric and observe its correspondence with exptl. measured binding affinity.  LGFE is computed for different conformational ensembles and improvement in prediction is shown with increasing ligand conformational sampling.  Ensemble generation includes a Monte Carlo sampling approach that uses the GFE FragMaps directly as the energy function.  The results show that some but not all exptl. trends are predicted and warrant improvements in the scoring methodol.  In addn., the potential utility of atom-based free energy contributions to the LGFE scores and the use of multiple ligands in SILCS to identify displaceable water mols. during ligand design are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCXtdZ9K_e2bVg90H21EOLACvtfcHk0ljA4467ZxP14Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslyqt7vK&md5=6eb2518f4edbd88ff49868c91d367d73</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fci4005628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci4005628%26sid%3Dliteratum%253Aachs%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLakkaraju%26aufirst%3DS.%2BK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DInclusion%2520of%2520multiple%2520fragment%2520types%2520in%2520the%2520site%2520identification%2520by%2520ligand%2520competitive%2520saturation%2520%2528SILCS%2529%2520approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D3384%26epage%3D3398%26doi%3D10.1021%2Fci4005628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) I: bond perception and atom typing</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3144</span><span class="NLM_x">–</span> <span class="NLM_lpage">3154</span><span class="refDoi"> DOI: 10.1021/ci300363c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300363c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3144-3154&author=K.+Vanommeslaegheauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+I%3A+bond+perception+and+atom+typing&doi=10.1021%2Fci300363c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) I: Bond Perception and Atom Typing</span></div><div class="casAuthors">Vanommeslaeghe, K.; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3144-3154</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug-like mols. alone or interacting with biol. systems.  In simulations involving biol. macromols., the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and charges is required.  In the present article, which is part I of a series of two, we present the algorithms for bond perception and atom typing for the CHARMM General Force Field (CGenFF).  The CGenFF atom typer first assocs. attributes to the atoms and bonds in a mol., such as valence, bond order, and ring membership among others.  Of note are a no. of features that are specifically required for CGenFF.  This information is then used by the atom typing routine to assign CGenFF atom types based on a programmable decision tree.  This allows for straight-forward implementation of CGenFF's complicated atom typing rules and for equally straight-forward updating of the atom typing scheme as the force field grows.  The presented atom typer was validated by assigning correct atom types on 477 model compds. including in the training set as well as 126 test-set mols. that were constructed to specifically verify its different components.  The program may be utilized via an online implementation at https://www.paramchem.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvkdOrvB3NELVg90H21EOLACvtfcHk0ljA4467ZxP14Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fL&md5=c6679293f4a2501f2bcadf2020ca1473</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fci300363c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300363c%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520I%253A%2520bond%2520perception%2520and%2520atom%2520typing%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3144%26epage%3D3154%26doi%3D10.1021%2Fci300363c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">Automation of the CHARMM General Force Field (CGenFF) II: assignment of bonded parameters and partial atomic charges</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3155</span><span class="NLM_x">–</span> <span class="NLM_lpage">3168</span><span class="refDoi"> DOI: 10.1021/ci3003649</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003649" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3155-3168&author=K.+Vanommeslaegheauthor=E.+P.+Ramanauthor=A.+D.+MacKerell&title=Automation+of+the+CHARMM+General+Force+Field+%28CGenFF%29+II%3A+assignment+of+bonded+parameters+and+partial+atomic+charges&doi=10.1021%2Fci3003649"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Automation of the CHARMM General Force Field (CGenFF) II: Assignment of Bonded Parameters and Partial Atomic Charges</span></div><div class="casAuthors">Vanommeslaeghe, K.; Raman, E. Prabhu; MacKerell, A. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3155-3168</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. mechanics force fields are widely used in computer-aided drug design for the study of drug candidates interacting with biol. systems.  In these simulations, the biol. part is typically represented by a specialized biomol. force field, while the drug is represented by a matching general (org.) force field.  In order to apply these general force fields to an arbitrary drug-like mol., functionality for assignment of atom types, parameters, and partial at. charges is required.  In the present article, algorithms for the assignment of parameters and charges for the CHARMM General Force Field (CGenFF) are presented.  These algorithms rely on the existing parameters and charges that were detd. as part of the parametrization of the force field.  Bonded parameters are assigned based on the similarity between the atom types that define said parameters, while charges are detd. using an extended bond-charge increment scheme.  Charge increments were optimized to reproduce the charges on model compds. that were part of the parametrization of the force field.  Case studies are presented to clarify the functioning of the algorithms and the significance of their output data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0zgm66CLdn7Vg90H21EOLACvtfcHk0ljA4467ZxP14Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Gns7fF&md5=e676ad1f42cb1e98dd353d4d285e8d13</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci3003649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003649%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DRaman%26aufirst%3DE.%2BP.%26aulast%3DMacKerell%26aufirst%3DA.%2BD.%26atitle%3DAutomation%2520of%2520the%2520CHARMM%2520General%2520Force%2520Field%2520%2528CGenFF%2529%2520II%253A%2520assignment%2520of%2520bonded%2520parameters%2520and%2520partial%2520atomic%2520charges%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3155%26epage%3D3168%26doi%3D10.1021%2Fci3003649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Vanommeslaeghe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatcher, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Acharya, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kundu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darian, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guvench, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopes, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vorobyov, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackerell, A. D.,  Jr.</span><span> </span><span class="NLM_article-title">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">671</span><span class="NLM_x">–</span> <span class="NLM_lpage">690</span><span class="refDoi"> DOI: 10.1002/jcc.21367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=10.1002%2Fjcc.21367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=19575467" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=671-690&author=K.+Vanommeslaegheauthor=E.+Hatcherauthor=C.+Acharyaauthor=S.+Kunduauthor=S.+Zhongauthor=J.+Shimauthor=E.+Darianauthor=O.+Guvenchauthor=P.+Lopesauthor=I.+Vorobyovauthor=A.+D.+Mackerell&title=CHARMM+general+force+field%3A+A+force+field+for+drug-like+molecules+compatible+with+the+CHARMM+all-atom+additive+biological+force+fields&doi=10.1002%2Fjcc.21367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">CHARMM general force field: A force field for drug-like molecules compatible with the CHARMM all-atom additive biological force fields</span></div><div class="casAuthors">Vanommeslaeghe, K.; Hatcher, E.; Acharya, C.; Kundu, S.; Zhong, S.; Shim, J.; Darian, E.; Guvench, O.; Lopes, P.; Vorobyov, I.; Mackerell, A. D., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">671-690</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The widely used CHARMM additive all-atom force field includes parameters for proteins, nucleic acids, lipids, and carbohydrates.  In the present article, an extension of the CHARMM force field to drug-like mols. is presented.  The resulting CHARMM General Force Field (CGenFF) covers a wide range of chem. groups present in biomols. and drug-like mols., including a large no. of heterocyclic scaffolds.  The parametrization philosophy behind the force field focuses on quality at the expense of transferability, with the implementation concg. on an extensible force field.  Statistics related to the quality of the parametrization with a focus on exptl. validation are presented.  Addnl., the parametrization procedure, described fully in the present article in the context of the model systems, pyrrolidine, and 3-phenoxymethyl-pyrrolidine will allow users to readily extend the force field to chem. groups that are not explicitly covered in the force field as well as add functional groups to and link together mols. already available in the force field.  CGenFF thus makes it possible to perform "all-CHARMM" simulations on drug-target interactions thereby extending the utility of CHARMM force fields to medicinally relevant systems. © 2009 Wiley Periodicals, Inc.J Comput Chem, 2010.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCuh0CLdmxN7Vg90H21EOLACvtfcHk0lifoZazwGygUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlentbc%253D&md5=26212e0e4f73bded0c89d4b411cd3833</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21367%26sid%3Dliteratum%253Aachs%26aulast%3DVanommeslaeghe%26aufirst%3DK.%26aulast%3DHatcher%26aufirst%3DE.%26aulast%3DAcharya%26aufirst%3DC.%26aulast%3DKundu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DS.%26aulast%3DShim%26aufirst%3DJ.%26aulast%3DDarian%26aufirst%3DE.%26aulast%3DGuvench%26aufirst%3DO.%26aulast%3DLopes%26aufirst%3DP.%26aulast%3DVorobyov%26aufirst%3DI.%26aulast%3DMackerell%26aufirst%3DA.%2BD.%26atitle%3DCHARMM%2520general%2520force%2520field%253A%2520A%2520force%2520field%2520for%2520drug-like%2520molecules%2520compatible%2520with%2520the%2520CHARMM%2520all-atom%2520additive%2520biological%2520force%2520fields%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D671%26epage%3D690%26doi%3D10.1002%2Fjcc.21367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GK1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GK1','PDB','3GK1'); return false;">PDB: 3GK1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FQO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FQO','PDB','4FQO'); return false;">PDB: 4FQO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MHO" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MHO','PDB','1MHO'); return false;">PDB: 1MHO</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN','PDB','5DKN'); return false;">PDB: 5DKN</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ','PDB','5DKQ'); return false;">PDB: 5DKQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR','PDB','5DKR'); return false;">PDB: 5DKR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PE0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4PE0','PDB','4PE0'); return false;">PDB: 4PE0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i61"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01369">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_23542"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01369">10.1021/acs.jmedchem.5b01369</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional details of crystal model RMSD, in silico data, 2D <sup>1</sup>H–<sup>15</sup>N S100B HSQC NMR spectra, purities derived by HPLC, 1D <sup>1</sup>H and <sup>13</sup>C compound NMR spectra, and SMILES (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01369/suppl_file/jm5b01369_si_001.pdf">jm5b01369_si_001.pdf (3.2 MB)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last"><sup>Ca</sup>S100B·<b>5a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKN">5DKN</a>), <sup>Ca</sup>S100B·<b>6b</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKQ">5DKQ</a>), <sup>Ca</sup>S100B·<b>17</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DKR">5DKR</a>).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01369&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-2%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01369" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01369" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                9MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799a0ddae7d3c52","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
