
<html lang="en"     class="pb-page"  data-request-id="67631dfc-589a-484b-bb10-d2493baac0cd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2017.60.issue-8;pageGroup:string:Publication Pages;article:article:10.1021/acs.jmedchem.6b01904;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Novel 1,4-Diacylpiperazines as Selective and Cell-Active eIF4A3 Inhibitors" /></meta><meta name="dc.Creator" content="Masahiro  Ito" /></meta><meta name="dc.Creator" content="Toshio  Tanaka" /></meta><meta name="dc.Creator" content="Douglas R.  Cary" /></meta><meta name="dc.Creator" content="Misa  Iwatani-Yoshihara" /></meta><meta name="dc.Creator" content="Yusuke  Kamada" /></meta><meta name="dc.Creator" content="Tomohiro  Kawamoto" /></meta><meta name="dc.Creator" content="Samuel  Aparicio" /></meta><meta name="dc.Creator" content="Atsushi  Nakanishi" /></meta><meta name="dc.Creator" content="Yasuhiro  Imaeda" /></meta><meta name="dc.Description" content="Eukaryotic initiation factor 4A3 (eIF4A3), a member of the DEAD-box RNA helicase family, is one of the core components of the exon junction complex (EJC). The EJC is known to be involved in a varie..." /></meta><meta name="Description" content="Eukaryotic initiation factor 4A3 (eIF4A3), a member of the DEAD-box RNA helicase family, is one of the core components of the exon junction complex (EJC). The EJC is known to be involved in a varie..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 11, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01904" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01904" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01904" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01904" /></link>
        
    
    

<title>Discovery of Novel 1,4-Diacylpiperazines as Selective and Cell-Active eIF4A3 Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01904" /></meta><meta property="og:title" content="Discovery of Novel 1,4-Diacylpiperazines as Selective and Cell-Active eIF4A3 Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0017.jpeg" /></meta><meta property="og:description" content="Eukaryotic initiation factor 4A3 (eIF4A3), a member of the DEAD-box RNA helicase family, is one of the core components of the exon junction complex (EJC). The EJC is known to be involved in a variety of RNA metabolic processes typified by nonsense-mediated RNA decay (NMD). In order to identify molecular probes to investigate the functions and therapeutic relevance of eIF4A3, a search for selective eIF4A3 inhibitors was conducted. Through the chemical optimization of 1,4-diacylpiperazine derivatives identified via high-throughput screening (HTS), we discovered the first reported selective eIF4A3 inhibitor 53a exhibiting cellular NMD inhibitory activity. A surface plasmon resonance (SPR) biosensing assay ascertained the direct binding of 53a and its analog 52a to eIF4A3 and revealed that the binding occurs at a non-ATP binding site. Compounds 52a and 53a represent novel molecular probes for further study of eIF4A3, the EJC, and NMD." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01904"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01904">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01904&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01904&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01904&amp;href=/doi/10.1021/acs.jmedchem.6b01904" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 8</span><span class="cit-fg-pageRange">, 3335-3351</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/8" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01892" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00083" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Novel 1,4-Diacylpiperazines as Selective and Cell-Active eIF4A3 Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masahiro++Ito">Masahiro Ito</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5965-535X" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Toshio++Tanaka">Toshio Tanaka</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Douglas+R.++Cary">Douglas R. Cary</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-3940-6078" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Misa++Iwatani-Yoshihara">Misa Iwatani-Yoshihara</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yusuke++Kamada">Yusuke Kamada</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-3701-2075" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tomohiro++Kawamoto">Tomohiro Kawamoto</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Samuel++Aparicio">Samuel Aparicio</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Atsushi++Nakanishi">Atsushi Nakanishi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yasuhiro++Imaeda">Yasuhiro Imaeda</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Pharmaceutical Research Division, Takeda Pharmaceutical Company Limited, 26-1, Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Molecular Oncology, BC Cancer Agency, 675 W. 10th Avenue, Vancouver, BC V5Z 1L3, Canada</span></div><div class="corresp-info"><strong>*</strong>M.I.: phone, +81-466-32-1196; e-mail, <a href="/cdn-cgi/l/email-protection#ed808c9e8c85849f82c3849982ad998c8688898cc38e8280"><span class="__cf_email__" data-cfemail="c0ada1b3a1a8a9b2afeea9b4af80b4a1aba5a4a1eea3afad">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>Y.I.: phone, +81-466-32-4112; e-mail, <a href="/cdn-cgi/l/email-protection#136a7260667b7a617c3d7a7e72767772536772787677723d707c7e"><span class="__cf_email__" data-cfemail="a1d8c0d2d4c9c8d3ce8fc8ccc0c4c5c0e1d5c0cac4c5c08fc2cecc">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01904&amp;href=/doi/10.1021%2Facs.jmedchem.6b01904" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 8</span><span class="cit-pageRange">, 3335–3351</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 30, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 January 2017</li><li><span class="item_label"><b>Published</b> online</span>11 April 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 April 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01904" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01904</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3335%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMasahiro%2BIto%252C%2BToshio%2BTanaka%252C%2BDouglas%2BR.%2BCary%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D8%26contentID%3Dacs.jmedchem.6b01904%26title%3DDiscovery%2Bof%2BNovel%2B1%252C4-Diacylpiperazines%2Bas%2BSelective%2Band%2BCell-Active%2BeIF4A3%2BInhibitors%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3351%26publicationDate%3DApril%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01904"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1905</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">13</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01904" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Novel 1,4-Diacylpiperazines as Selective and Cell-Active eIF4A3 Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Masahiro&quot;,&quot;last_name&quot;:&quot;Ito&quot;},{&quot;first_name&quot;:&quot;Toshio&quot;,&quot;last_name&quot;:&quot;Tanaka&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;R. Cary&quot;},{&quot;first_name&quot;:&quot;Misa&quot;,&quot;last_name&quot;:&quot;Iwatani-Yoshihara&quot;},{&quot;first_name&quot;:&quot;Yusuke&quot;,&quot;last_name&quot;:&quot;Kamada&quot;},{&quot;first_name&quot;:&quot;Tomohiro&quot;,&quot;last_name&quot;:&quot;Kawamoto&quot;},{&quot;first_name&quot;:&quot;Samuel&quot;,&quot;last_name&quot;:&quot;Aparicio&quot;},{&quot;first_name&quot;:&quot;Atsushi&quot;,&quot;last_name&quot;:&quot;Nakanishi&quot;},{&quot;first_name&quot;:&quot;Yasuhiro&quot;,&quot;last_name&quot;:&quot;Imaeda&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;11&quot;,&quot;issue&quot;:&quot;8&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;3335-3351&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01904&quot;},&quot;abstract&quot;:&quot;Eukaryotic initiation factor 4A3 (eIF4A3), a member of the DEAD-box RNA helicase family, is one of the core components of the exon junction complex (EJC). The EJC is known to be involved in a variety of RNA metabolic processes typified by nonsense-mediated RNA decay (NMD). In order to identify molecular probes to investigate the functions and therapeutic relevance of eIF4A3, a search for selective eIF4A3 inhibitors was conducted. Through the chemical optimization of 1,4-diacylpiperazine derivatives identified via high-throughput screening (HTS), we discovered the first reported selective eIF4A3 inhibitor 53a exhibiting cellular NMD inhibitory activity. A surface plasmon resonance (SPR) biosensing assay ascertained the direct binding of 53a and its analog 52a to eIF4A3 and revealed that the binding occurs at a non-ATP binding site. Compounds 52a and 53a represent novel molecular probes for further study of eIF4A3, the EJC, and NMD.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01904&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01904" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01904&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01904" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01904&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01904" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01904&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01904&amp;href=/doi/10.1021/acs.jmedchem.6b01904" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01904" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01904" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (1 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01904%26sid%3Dliteratum%253Aachs%26pmid%3D28358513%26genre%3Darticle%26aulast%3DIto%26date%3D2017%26atitle%3DDiscovery%2Bof%2BNovel%2B1%252C4-Diacylpiperazines%2Bas%2BSelective%2Band%2BCell-Active%2BeIF4A3%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D8%26spage%3D3335%26epage%3D3351%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292119" title="Surface plasmon resonance">Surface plasmon resonance</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/8" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/jmcmar.2017.60.issue-8/20170427/jmcmar.2017.60.issue-8.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_null" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0017.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Eukaryotic initiation factor 4A3 (eIF4A3), a member of the DEAD-box RNA helicase family, is one of the core components of the exon junction complex (EJC). The EJC is known to be involved in a variety of RNA metabolic processes typified by nonsense-mediated RNA decay (NMD). In order to identify molecular probes to investigate the functions and therapeutic relevance of eIF4A3, a search for selective eIF4A3 inhibitors was conducted. Through the chemical optimization of 1,4-diacylpiperazine derivatives identified via high-throughput screening (HTS), we discovered the first reported selective eIF4A3 inhibitor <b>53a</b> exhibiting cellular NMD inhibitory activity. A surface plasmon resonance (SPR) biosensing assay ascertained the direct binding of <b>53a</b> and its analog <b>52a</b> to eIF4A3 and revealed that the binding occurs at a non-ATP binding site. Compounds <b>52a</b> and <b>53a</b> represent novel molecular probes for further study of eIF4A3, the EJC, and NMD.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27507" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27507" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Eukaryotic initiation factor 4A3 (eIF4A3) is a member of the DEAD-box RNA helicase family.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1-4)</a> DEAD-box RNA helicases play essential roles in RNA metabolism<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> (e.g., pre-mRNA splicing, RNA transport, translation, and RNA decay) and generally function as part of large multiprotein complexes. DEAD-box proteins possess ATPase activity that is stimulated by RNA, and the ATPase cycle (e.g., ATP binding, hydrolysis, and release of products) changes the affinity of DEAD-box proteins for RNA.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> These ATP-driven changes in RNA affinity can be used to unwind the short RNA duplexes, to promote rearrangements of complex and structured RNAs, to remodel RNA-protein complexes, or to form a long-lived, highly stable complex on RNA (RNA clamping). eIF4A3 is known to be a core component of the exon junction complex (EJC) and has been suggested as an ATP-dependent RNA clamp that can serve as a nucleation center to recruit other EJC components.<a onclick="showRef(event, 'ref1 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref6">(1, 6)</a> The EJC is involved in splicing, transport, translation, and nonsense-mediated RNA decay (NMD).<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11 ref12 ref13 ref14">(7-14)</a> Among these functions, the most widely studied is NMD, which is the translation-dependent surveillance mechanism that recognizes mRNAs containing premature termination codons (PTC) to prevent the accumulation of truncated proteins.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Since nonsense mutations often cause hereditary and sporadic diseases due to a deficiency of essential proteins, a combination of an NMD inhibitor and a PTC read-through therapy has the potential to treat a variety of genetic diseases via the restoration of full-length proteins.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> There also are several reports implying the possible application of NMD inhibitors to cancer therapy (e.g., induction of cancer immunity<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> or sensitization of cancer cells to a chemotherapeutic agent).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">Although little is known about the detailed regulatory mechanisms of eIF4A3 involving the EJC, NMD, and other functions, it is known that small interfering RNA (siRNA)-mediated knockdown of eIF4A3 leads to a defect in NMD.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8-10)</a> Currently, several NMD suppressors with a variety of mechanisms are known;<a onclick="showRef(event, 'ref20 ref21 ref22 ref23 ref24 ref25'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23 ref24 ref25">(20-25)</a> however, there are no such compounds that act via the selective inhibition of eIF4A3. It is expected that selective eIF4A3 inhibitors with NMD suppression activity will expedite our understanding of the NMD process and the function of the EJC in RNA metabolism. A marine natural product, hippuristanol (<b>1</b>),<a onclick="showRef(event, 'ref26 ref27 ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28 ref29">(26-29)</a> has been reported as a pan-eIF4A inhibitor (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>), but NMD inhibitory activity of <b>1</b> has not been reported. We recently reported a novel eIF4A3 selective inhibitor (<b>2</b>) with an ATP-competitive mechanism.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> However, <b>2</b> was inactive at 30 μM in a cellular assay for the detection of NMD suppression because of low permeability caused by the indispensable carboxylic acid moiety. Since such ATP-competitive inhibitors showed a narrow structure–activity relationship (SAR) and did not appear amenable to further chemical optimization, our attention shifted to other chemical series. Through high-throughput screening (HTS), a class of 1,4-diacylpiperazine derivatives (e.g., <b>3</b>) was identified as a promising chemotype. In this report, we focus on the discovery of 1,4-diacylpiperazine derivatives <b>52a</b> and <b>53a</b>, which bind to a non-ATP binding site of eIF4A3 and demonstrate cellular NMD inhibitory activity.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of eIF4A3 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21673" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21673" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Unless otherwise specified, the compounds described herein are all in the racemic form. The synthesis of 1-acyl-2-phenylpiperazines <b>7a</b>–<b>d</b> and <b>8a</b>,<b>b</b> (enantiomers of <b>7b</b>), which were employed as common intermediates in this work, is depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The 2-phenylpiperazine compounds <b>4a</b> and <b>4c</b>,<b>d</b> were selectively protected<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> at the 4-position by a Boc group and then treated with 4-chlorobenzoyl chloride or 4-bromobenzoyl chloride, followed by removal of the Boc group to give 1-acyl-2-phenylpiperazines (<b>7a</b>–<b>d</b>). Chiral separation of the 4-bromobenzoyl derivative <b>7b</b> afforded <b>8a</b> as the (<i>S</i>)-enantiomer and <b>8b</b> as the (<i>R</i>)-enantiomer. The absolute stereochemistry of these compounds was determined by X-ray crystal structure analysis using the TsOH salt of <b>8a</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf" class="ext-link">Supporting Information</a>).</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>7a</b>–<b>d</b> and <b>8a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0–15 °C, 66–83%; (b) Boc<sub>2</sub>O, THF, toluene, 0 °C to rt, 70%; (c) 4-chlorobenzoyl chloride or 4-bromobenzoyl chloride, Et<sub>3</sub>N or DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, 0–15 °C, 78–95%; (d) 4 M HCl in EtOAc, 0 °C or rt, 66% to quant; (e) (i) aq NaHCO<sub>3</sub>, rt, 97%; (ii) chiral HPLC, 41–42%.</p></p></figure><div class="NLM_p">Attachment of varied acyl moieties to <b>7a</b>–<b>d</b> and <b>8a</b>,<b>b</b> was conducted as shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Condensation of 1-acyl-2-phenylpiperazines <b>7a</b>–<b>d</b> and <b>8a</b>,<b>b</b> with the corresponding acids or esters <b>9</b>–<b>21</b> and <b>24</b>–<b>28</b> under a variety of conditions afforded the 1,4-diacylpiperazine derivatives <b>3</b>, <b>29</b>–<b>48</b>, <b>52a</b>,<b>b</b>, and <b>53a</b>,<b>b</b>. Meanwhile, compounds <b>49</b>–<b>51</b> were synthesized using the following two-step procedures. Amidation of the intermediate <b>7a</b> with acids <b>22</b> and subsequent C–H activation reaction<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> at the 2-positon of oxazole (condition h) furnished <b>49</b>, whereas condensation of <b>7a</b> with acid <b>23</b> followed by Suzuki coupling (condition i) afforded <b>50</b> and <b>51</b>.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>3</b>, <b>29</b>–<b>51</b>, <b>52a</b>,<b>b</b>, <b>53a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>9</b>–<b>10</b>, <b>12</b>, <b>14</b>, <b>15</b>, <b>18</b>, <b>20</b>, or <b>21</b>, HATU, DIPEA, DMF, rt, 24–61%; (b) <b>11</b>, <b>13</b>, <b>21</b>, or <b>25</b>–<b>26</b>, EDC·HCl, HOBt, DIPEA, DMF, rt, 34–79%; (c) (i) <b>16</b> or <b>19</b>, LiOH·H<sub>2</sub>O, THF, MeOH, H<sub>2</sub>O, rt; (ii) HATU, DIPEA, DMF, rt, 17–71%; (d) (i) <b>17</b>, TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt; (ii) HATU, Et<sub>3</sub>N, DMF, rt, 53%; (e) (i) aq NaHCO<sub>3</sub>, rt; (ii) <b>24</b> or <b>27</b>, (Me<sub>3</sub>Al)<sub>2</sub>·DABCO,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> THF, microwave, 130 °C, 47–78%; (f) (i) aq NaHCO<sub>3</sub>, rt; (ii) <b>28</b>, Me<sub>3</sub>Al, THF, microwave, 110 °C, 55%; (g) <b>13</b> or <b>26</b>, EDC·HCl, HOBt, DMF, rt, 50–81%; (h) (i) <b>22</b>, EDC·HCl, HOBt, DMF, rt, 25%; (ii) 2-bromobenzonitrile, Pd(OAc)<sub>2</sub>, CuI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, microwave, 150 °C, 16%; (i) (i) <b>23</b>, HATU, DIPEA, DMF, 0 °C, 95%; (ii) 3-cyanophenylboronic acid or 4-cyanophenylboronic acid, Na<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DME, H<sub>2</sub>O, reflux, 30–49%.</p></p></figure><div class="NLM_p">Compounds <b>55</b>–<b>60</b> were synthesized as illustrated in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Selective amidation of piperazine <b>7a</b> at the 4-position of the piperazine with carboxylic acid <b>21</b> yielded intermediate <b>54</b>, followed by reduction of the amide carbonyl group and subsequent acylation at the 4-position of the piperazine to afford amide <b>55</b>. Meanwhile, alkylation at the 4-position on the piperazine of <b>54</b> provided compound <b>56</b>. Acylation of <b>54</b> with the corresponding acid chlorides or 4-bromobenzoic acid afforded compounds <b>57</b>–<b>60</b>.</div><figure id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>55</b>–<b>60</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>21</b>, HATU, DIPEA, DMF, rt, 85%; (b) (i) BH<sub>3</sub>·Me<sub>2</sub>S, THF, 0 °C to rt, 31%; (ii) 4-chlorobenzoyl chloride, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 56%; (c) 4-chlorobenzyl bromide, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 19%; (d) corresponding acyl chloride (3-chlorobenzoyl chloride for <b>57</b>, 2-chlorobenzoyl chloride for <b>58</b>, and benzoyl chloride for <b>59</b>), Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 41–46%; (e) 4-bromobenzoic acid, EDC, HOBt, DIPEA, THF, rt, 96%.</p></p></figure><div class="NLM_p">The synthesis of compounds <b>63</b>, <b>65</b>, and <b>68</b> was conducted as depicted in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. Acylation of commercially available piperazinone <b>61</b> with 4-chlorobenzoyl chloride at the 4-position of piperazinone followed by alkylation with <b>62</b> provided compound <b>63</b>. Protection of the 4-position of 2-cyclohexylpiperazine <b>64</b> with a Boc group, acylation with 4-chlorobenzoyl chloride, removal of the Boc group, and condensation with <b>21</b> provided compound <b>65</b>. Treatment of α-bromoester <b>66</b><a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> with 1,3-propanediamine provided the homopiperazinone, followed by the reduction of the amide carbonyl group and selective protection at the 4-position of the homopiperazine by a Boc group to afford intermediate <b>67</b>. Subsequently, the acylation of <b>67</b> with 4-chlorobenzoyl chloride followed by the removal of the Boc group under acidic conditions and finally amide coupling with carboxylic acid <b>21</b> yielded compound <b>68</b>.</div><figure id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0009.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>63</b>, <b>65</b>, and <b>68</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) 4-chlorobenzoyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt; (ii) <b>62</b>, NaH, DMF, 0 °C, 12% for 2 steps; (b) (i) Boc<sub>2</sub>O, Et<sub>3</sub>N, THF, 0 °C to rt, 73%; (ii) 4-chlorobenzoyl chloride, DIPEA, THF, 0 °C, 99%; (iii) 4 M HCl in EtOAc, rt, 91%; (iv) <b>21</b>, EDC·HCl, HOBt, DIPEA, DMF, 0 °C to rt, 24%; (c) (i) 1,3-propanediamine, MeONa, MeOH, rt to 80 °C, 23%; (ii) BH<sub>3</sub>·Me<sub>2</sub>S, THF, 0–80 °C; (iii) Boc<sub>2</sub>O, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 72% for 2 steps; (d) (i) 4-chlorobenzoyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 93%; (ii) 4 M HCl in dioxane, rt; (iii) <b>21</b>, HATU, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 34% for 2 steps.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i8">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72526" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72526" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Our design strategy based on compound <b>3</b> is illustrated in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>. First, we investigated the SARs of the following key substructures: the 1-acyl group (A), the 4-acyl group (B), the 3-phenyl moiety (C), and the 1,4-diacylpiperazine ring (D). Then, combinations of the most promising functional groups were examined to further enhance biological activity.</div><figure id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design strategy around compound <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Biological evaluation was conducted using the ADP-Glo assay system (Promega) in which inhibition of the RNA-dependent ATPase activity of eIF4A3 was detected, and the results were expressed as IC<sub>50</sub> values. At first, replacement of the indole ring of compound <b>3</b> with other fused heterocycles was examined (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). As a result, pyrazolopyridine <b>30</b> showed similar potency as <b>3</b>, whereas the structurally similar indazole derivative <b>29</b> showed significantly diminished activity. These results suggested that subtle structural differences between the indole and indazole rings have an effect on binding to eIF4A3. Next, we designed and synthesized a variety of biaryl amide derivatives inspired by compound <b>30</b>. Phenylpyrazole <b>31</b> was equipotent to <b>30</b>, while regioisomer <b>32</b> showed deteriorated activity. 2-Phenyloxazole-5-yl derivative <b>34</b> turned out to be the most potent among the oxazole series <b>33</b>–<b>35</b>. In addition to that, 5-methylpyrazole <b>36</b> exhibited the same level of inhibitory activity as <b>31</b>. Considering these observations, the nitrogen atom at the 3-position of oxazole <b>34</b> or the 2-position of pyrazoles <b>31</b> and <b>36</b> was judged to be indispensable for inhibitory activity. This implies the existence of a hydrogen bonding interaction between the nitrogen atom in these compounds and the eIF4A3 protein.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR around the 1-Acyl Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0010.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0011.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div><div></div></div><div class="NLM_p">Modifications to the diacylpiperazine scaffold of compound <b>34</b> are depicted in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>. All major modifications, including removal of a carbonyl group (<b>55</b> and <b>56</b>), relocation of a carbonyl group (<b>63</b>), and ring expansion to homopiperazine (<b>68</b>), resulted in substantially diminished inhibitory activity. These results indicate the importance of both carbonyl groups in the 1,4-diacylpiperazine scaffold and the central ring size. We have two main hypotheses regarding the function of the 1,4-diacylpiperazine moiety for the expression of eIF4A3 inhibitory activity: either the carbonyl groups are necessary to form key hydrogen bonds with eIF4A3 or the 1,4-diacylpiperazine acts as a structural scaffold to direct the 1- and 4-aromatic rings along appropriate vectors toward hydrophobic pockets on the protein.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. SAR around the 1,4-Diacylpiperazine Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0012.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="rowsep0" align="center">compd</th><th class="rowsep0" align="center">eIF4A3 ATPase IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>55</b></td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>56</b></td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>63</b></td><td class="rowsep0" align="left">>100</td></tr><tr valign="top"><td align="left"><b>68</b></td><td align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div></div><div class="NLM_p">Next, the SARs of substituents at the 3-position on the piperazine of compound <b>34</b> were investigated (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). The 3-chloro (<b>37</b>) and 2-chloro (<b>38</b>) derivatives showed decreased activity, and substitution with an aliphatic cyclohexyl moiety (<b>65</b>) was not tolerated. These results indicate that the 4-chlorophenyl group at the piperazine 3-position is essential for potency. Then, modifications around the 4-acyl moiety were conducted. Similar to the SAR of substituents at the 3-postion on the piperazine (<b>37</b> and <b>38</b>), the 3-chlorobenzoyl (<b>57</b>) and 2-chlorobenzoyl (<b>58</b>) derivatives exhibited reduced activity. The unsubstituted phenyl derivative <b>59</b> showed slightly decreased activity, while the 4-bromo derivative <b>60</b> demonstrated slightly increased potency compared with the 4-chloro derivative <b>34</b>. These observations imply the importance of a 4-chloro or 4-bromo group. These halogen atoms might fill in a small hydrophobic cavity of the eIF4A3 protein or might be involved in a halogen bonding.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. SAR around the 3-Position of the Piperazine and 4-Acyl Moiety</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0013.gif" alt="" id="fx4" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="rowsep0" align="center">compd</th><th class="rowsep0" align="center">R<sup>1</sup></th><th class="rowsep0" align="center">R<sup>2</sup></th><th class="rowsep0" align="center">eIF4A3 ATPase IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>37</b></td><td class="rowsep0" align="left">3-chlorophenyl</td><td class="rowsep0" align="left">4-chlorophenyl</td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>38</b></td><td class="rowsep0" align="left">2-chlorophenyl</td><td class="rowsep0" align="left">4-chlorophenyl</td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>65</b></td><td class="rowsep0" align="left"><i>c</i>-hexyl</td><td class="rowsep0" align="left">4-chlorophenyl</td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>57</b></td><td class="rowsep0" align="left">4-chlorophenyl</td><td class="rowsep0" align="left">3-chlorophenyl</td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>58</b></td><td class="rowsep0" align="left">4-chlorophenyl</td><td class="rowsep0" align="left">2-chlorophenyl</td><td class="rowsep0" align="left">>100</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>59</b></td><td class="rowsep0" align="left">4-chlorophenyl</td><td class="rowsep0" align="left">phenyl</td><td class="rowsep0" align="left">11 (7.1–16)</td></tr><tr valign="top"><td align="left"><b>60</b></td><td align="left">4-chlorophenyl</td><td align="left">4-bromophenyl</td><td align="left">0.66 (0.38–1.1)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div></div><div class="NLM_p">In consideration to the basic SAR information presented above, we conducted further modifications of the 1-acyl moiety on compound <b>30</b> (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). To enhance potency through additional interactions with eIF4A3, the introduction of a bromine atom on the pyrazolopyridine ring of <b>30</b> was examined first. The 7-bromo derivative <b>39</b> was found to be equipotent to <b>30</b>, while the 6-bromo (<b>40</b>) and 5-bromo (<b>41</b>) derivatives showed slightly increased potency. Contrary to the other isomers, only the 4-bromo derivative <b>42</b> exhibited decreased potency. These results indicate the existence of sufficient space to accept a bromo group around the 5- to 7-positions of the pyrazolopyridine ring. Replacement of the 4-chlorobenzoyl moiety of <b>40</b> with a 4-bromobenzoyl moiety (<b>43</b>) tended to improve potency slightly. Interestingly, further investigation of substituents at the 6-position revealed that the 6-methoxy derivative <b>44</b> showed a similar level of potency as the 6-bromo derivative <b>43</b>, whereas the 6-cyano derivative <b>45</b> displayed less potent activity. Further modifications of the biaryl series <b>34</b> and <b>36</b> were also executed. Introduction of a cyano group into each position of the phenyl ring of <b>34</b> revealed that the 3-cyano derivative <b>50</b> is the most potent among them. Further optimization produced equipotent derivatives such as 4-bromobenzoyl derivatives <b>46</b> and <b>47</b>. Replacement of the oxazole ring in <b>46</b> with a 5-methylpyrazole also led to the identification of similarly potent compound <b>48</b>.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. SAR of the Pyrazolo[1,5-<i>a</i>]pyridine, the 2-Aryl-1,3-oxazole, and 1-Aryl-5-methyl-1<i>H</i>-pyrazole Derivatives</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0014.gif" alt="" id="fx5" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0015.gif" alt="" id="fx6" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div><div></div></div><div class="NLM_p">Compounds <b>43</b>–<b>44</b> and <b>46</b>–<b>48</b> were tested in a reporter gene assay to evaluate NMD suppression (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Intriguingly, all the compounds exhibited NMD inhibition at 3 μM. As far as we know, this is the first example of an NMD inhibitor operating through a mechanism of eIF4A3 inhibition. Of the most potent compounds from the reporter assay, compounds <b>43</b> and <b>48</b> were selected for chiral separation and further evaluation.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. NMD Reporter Assay Results for Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="rowsep0" align="center">compd</th><th class="rowsep0" align="center">eIF4A3 ATPase IC<sub>50</sub> (μM)</th><th class="rowsep0" align="center">NMD reporter assay fold change of firefly to renilla luciferase activity against DMSO control, 10 μM, 6 h (SD)<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>43</b></td><td class="rowsep0" align="left">0.42</td><td class="rowsep0" align="left">1.9 (0.20)</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>44</b></td><td class="rowsep0" align="left">0.92</td><td class="rowsep0" align="left">1.7 (0.06)</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>46</b></td><td class="rowsep0" align="left">0.36</td><td class="rowsep0" align="left">1.8 (0.03)</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>47</b></td><td class="rowsep0" align="left">0.39</td><td class="rowsep0" align="left">1.8 (0.06)</td></tr><tr valign="top"><td align="left"><b>48</b></td><td align="left">0.58</td><td align="left">2.3 (0.12)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">HEK293T cells transfected with the NMD reporter gene were treated with the indicated compounds at 10 μM for 6 h. An increase in the ratio of firefly to renilla luciferase activity, indicating NMD inhibition, was detected and normalized with DMSO treated samples. <i>n</i> = 2, SD values shown in parentheses.</p></div></div></div><div class="NLM_p">The ATPase inhibitory activities of the (<i>S</i>)- and (<i>R</i>)-enantiomers of compounds <b>43</b> and <b>48</b> are shown in <a class="ref internalNav" href="#tbl6" aria-label="Table 6">Table 6</a>. The (<i>S</i>)-isomers <b>52a</b> and <b>53a</b> turned out to be more potent (eutomers), and they showed >100-fold more potent inhibitory activity than the corresponding distomers <b>52b</b> and <b>53b</b>. The dramatic difference in activity between enantiomers was noteworthy, and this observation suggests that the stereocenter on the piperazine ring is clearly recognized by the eIF4A3 protein. In addition, <b>53a</b> exhibited excellent selectivity over other helicases and showed significant NMD inhibition at 10 and 3 μM in the reporter assay, while its distomer <b>53b</b> did not show any inhibitory activity at the same concentrations (<a class="ref internalNav" href="#tbl7" aria-label="Table 7">Table 7</a>, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). These results imply that the NMD inhibition of <b>53a</b> is correlated with eIF4A3 ATPase inhibitory activity and supports the validity of <b>53a</b> as a molecular probe for studying eIF4A3 function.</div><figure id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. NMD inhibition by <b>53a</b> and <b>53b</b> in the reporter gene assay. HEK293T cells transfected with the NMD reporter gene were treated with the indicated compounds at 10 μM or 3 μM for 6 h. An increase in the ratio of firefly to renilla luciferase activity, indicating NMD inhibition, was detected and normalized with DMSO treated samples. The <i>y</i>-axis in the bar graph represents the fold increase in the ratio of firefly to renilla luciferase activity. <i>n</i> = 2, and error bars represent the SD values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. ATPase Inhibitory Activity of Enantiomerically Pure Compounds against eIF4A3</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0016.gif" alt="" id="fx7" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="rowsep0" align="center">compd</th><th class="rowsep0" align="center">stereochemistry</th><th class="rowsep0" align="center">eIF4A3 ATPase IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>52a</b></td><td class="rowsep0" align="left"><i>S</i></td><td class="rowsep0" align="left">0.20 (0.16–0.25)</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>52b</b></td><td class="rowsep0" align="left"><i>R</i></td><td class="rowsep0" align="left">60 (21–177)</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>53a</b></td><td class="rowsep0" align="left"><i>S</i></td><td class="rowsep0" align="left">0.26 (0.18–0.38)</td></tr><tr valign="top"><td align="left"><b>53b</b></td><td align="left"><i>R</i></td><td align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup>a</sup><p class="last"><i>n</i> = 2, 95% confidence intervals shown in parentheses.</p></div></div></div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Selectivity over Other Helicases</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="rowsep0" align="center"> </th><th class="rowsep1" colspan="4" align="center">ATPase IC<sub>50</sub> (μM)</th></tr><tr valign="top" class="rowsep0"><th class="rowsep0" align="center">compd</th><th class="rowsep0" align="center">eIF4A1<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></th><th class="rowsep0" align="center">eIF4A2<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="rowsep0" align="center">DHX29<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th><th class="rowsep0" align="center">BRR2<a class="ref internalNav" href="#t7fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top"><td align="left"><b>53a</b></td><td align="left">>100</td><td align="left">>100</td><td align="left">>100</td><td align="left">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup>a</sup><p class="last"><i>n</i> = 2.</p></div><div class="footnote" id="t7fn2"><sup>b</sup><p class="last"><i>n</i> = 4.</p></div></div></div><div class="NLM_p">Binding affinity and kinetics of compounds (<b>52a</b>,<b>b</b> and <b>53a</b>) were analyzed using a surface plasmon resonance (SPR) biosensing assay (<a class="ref internalNav" href="#tbl8" aria-label="Table 8">Table 8</a>). The SPR experiments ascertained the direct binding of <b>53a</b> and its analog <b>52a</b> to eIF4A3. Furthermore, both <b>52a</b> and <b>53a</b> showed double-digit nanomolar binding affinity, and the <i>K</i><sub>D</sub> values of these compounds appeared to correlate with ATPase inhibitory activity. It is noteworthy that the kinetic profiles of <b>52a</b> and <b>53a</b> are distinct from the previously reported ATP-competitive inhibitor <b>2</b>, and they showed a slower <i>k</i><sub>off</sub> and longer dissociation <i>t</i><sub>1/2</sub>. The identification of two types of eIF4A3 inhibitors enabled us to study the binding modes of both types, using them in a complementary manner. As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, the absence or presence of compound <b>52a</b> at a saturated concentration did not affect the SPR binding signals of <b>2</b> against eIF4A3. In other words, the data suggest the formation of a ternary complex composed of <b>52a</b>, <b>2</b>, and eIF4A3 and that <b>52a</b> binds to a non-ATP binding site of eIF4A3. Next, the known pan-eIF4A inhibitor <b>1</b> was used for similar SPR experiments. The binding signals of <b>1</b> against eIF4A3 were detected in a dose dependent manner in the absence of <b>52a</b> (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D–F). Contrary to that, in the presence of <b>52a</b>, binding signals of <b>1</b> were clearly weakened (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A–C). Hence, the observations imply two possibilities: <b>52a</b> might bind to the same binding site as <b>1</b> or bind to other non-ATP binding site and induce a conformational change to prevent the binding of <b>1</b>. These results regarding the binding affinity, kinetics, and mode analysis of <b>52a</b> and <b>53a</b> also demonstrate the validity of these eIF4A3 inhibitors as excellent molecular probes.</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SPR binding studies of <b>2</b> against eIF4A3 in the presence or absence of <b>52a</b>. The binding signals of compound <b>2</b> against immobilized eIF4A3 were detected in the presence of <b>52a</b> at a saturated concentration (A–C) or absence of <b>52a</b> (D–F). Parts A and D represent the SPR response to varying concentrations of <b>2</b>. <i>y</i> axis is SPR response (RU), and <i>x</i> axis is concentration of <b>2</b>. Parts B and E represent the SPR sensorgrams. <i>y</i> axis is SPR response, and <i>x</i> axis is time. Parts C and F represent the schematics of the experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0005.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SPR binding studies of <b>1</b> against eIF4A3 in the presence or absence of <b>52a</b>. The binding signals of compound <b>1</b> against immobilized eIF4A3 were detected in the presence of <b>52a</b> at a saturated concentration (A–C) or absence of <b>52a</b> (D–F). Parts A and D represent the SPR response to varying concentrations of <b>1</b>. <i>y</i> axis is SPR response (RU), and <i>x</i> axis is concentration of <b>1</b>. Parts B and E represent the SPR sensorgrams. The <i>y</i> axis is the SPR response, and the <i>x</i> axis is time. Parts C and F represent the schematics of the experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. Binding Affinity and Kinetics of <b>52a</b>,<b>b</b> and <b>53a</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="rowsep0"><th class="rowsep0" align="center">compd</th><th class="rowsep0" align="center">eIF4A3 ATPase IC<sub>50</sub> (μM)</th><th class="rowsep0" align="center">eIF4A3 SPR <i>K</i><sub>D</sub> (μM)<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></th><th class="rowsep0" align="center"><i>k</i><sub>on</sub> (M<sup>–1</sup> s<sup>–1</sup>), <i>k</i><sub>off</sub> (s<sup>–1</sup>)<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></th><th class="rowsep0" align="center"><i>t</i><sub>1/2</sub> (s)<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>52a</b></td><td class="rowsep0" align="left">0.20</td><td class="rowsep0" align="left">0.057</td><td class="rowsep0" align="left">2.8 × 10<sup>4</sup>, 1.6 × 10<sup>–3</sup></td><td class="rowsep0" align="left">438.2</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>52b</b></td><td class="rowsep0" align="left">60</td><td class="rowsep0" align="left">16<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></td><td class="rowsep0" align="left">1.5 × 10<sup>3</sup>, 2.4 × 10<sup>–2</sup><a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></td><td class="rowsep0" align="left">28.6<a class="ref internalNav" href="#t8fn2" aria-label="b">b</a></td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>53a</b></td><td class="rowsep0" align="left">0.26</td><td class="rowsep0" align="left">0.043</td><td class="rowsep0" align="left">1.6 × 10<sup>4</sup>, 6.9 × 10<sup>–4</sup></td><td class="rowsep0" align="left">1006.9</td></tr><tr valign="top" class="rowsep0"><td class="rowsep0" align="left"><b>2</b><a class="ref internalNav" href="#t8fn3" aria-label="c">c</a></td><td class="rowsep0" align="left">0.97</td><td class="rowsep0" align="left">0.20</td><td class="rowsep0" align="left">4.0 × 10<sup>5</sup>, 7.8 × 10<sup>–2</sup></td><td class="rowsep0" align="left">8.9</td></tr><tr valign="top"><td align="left"><b>1</b><a class="ref internalNav" href="#t8fn3" aria-label="c">c</a></td><td align="left">40</td><td align="left">50</td><td align="left">1.1 × 10<sup>3</sup>, 5.7 × 10<sup>–2</sup></td><td align="left">12.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup>a</sup><p class="last">Sensorgrams of each compound are in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf" class="ext-link">Supporting Information Figure 1</a>.</p></div><div class="footnote" id="t8fn2"><sup>b</sup><p class="last">Approximate values due to compound precipitation.</p></div><div class="footnote" id="t8fn3"><sup>c</sup><p class="last">Reference <a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">30</a>.</p></div></div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45330" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45330" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">As a result of chemical modifications of various 1,4-diacylpiperazine derivatives, we discovered the first selective eIF4A3 inhibitor <b>53a</b> exhibiting cellular NMD inhibitory activity. Compound <b>53a</b> displayed high selectivity in favor of eIF4A3 and not for the eIF4A1 and eIF4A2 protein subtypes of the eIF4A family or other helicases. The clear difference in eIF4A3 inhibitory activity between <b>53a</b> and its distomer <b>53b</b> revealed the importance of stereochemistry at the 3-position of the piperazine ring for eIF4A3 inhibition. The lack of significant NMD inhibition by <b>53b</b> also supports the supposition that NMD inhibition of <b>53a</b> is mediated by eIF4A3 inhibition (and demonstrates the utility of this compound as a good negative control). The SPR biosensing assay provided evidence of the direct binding of <b>52a</b> and <b>53a</b> with eIF4A3 and their distinct kinetic profiles from the ATP-competitive inhibitor <b>2</b>. Furthermore, the SPR experiments also revealed that <b>52a</b> binds to a non-ATP binding site of eIF4A3. Among our eIF4A3 inhibitors, compounds <b>52a</b><a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and <b>53a</b> could be invaluable molecular probes to study the functions of eIF4A3 and the EJC, as well as for more detailed mechanistic studies of NMD. In addition, these compounds may ultimately help lead to the development of new treatments in the areas of genetic disease or cancer. Further biological investigations using our selective eIF4A3 inhibitors will be published soon.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_56346" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_56346" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Proton nuclear magnetic resonance (<sup>1</sup>H NMR) spectra were recorded on Bruker AVANCE-300 (300 MHz) and Bruker AVANCE-400 (400 MHz) instruments in CDCl<sub>3</sub> or DMSO-<i>d</i><sub>6</sub> solution. Chemical shifts are given in parts per million (ppm) with tetramethylsilane as an internal standard. Abbreviations are used as follows: s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet, dd = doublet of doublets, dt = doublet of triplets, brs = broad singlet. Coupling constants (<i>J</i> values) are given in hertz (Hz). Elemental analyses were carried out by the Elemental Microanalysis Team in the Medicinal Chemistry Research Laboratories, Sumika Chemical Analysis Service, or Toray Research Center and were within 0.4% of the theoretical values. Low-resolution mass spectra (MS) were acquired using an Agilent LC/MS system (Agilent1200SL/Agilent6130MS, Agilent1200SL/Agilent1956MS, or Agilent1200SL/Agilent6110MS) or Shimadzu UFLC/MS (Prominence UFLC high pressure gradient system/LCMS-2020) operating in an electron spray ionization mode (ESI+). The column used was an L-column 2 ODS (3.0 mm × 50 mm i.d., 3 μm, CERI) with a temperature of 40 °C and a flow rate of 1.2 or 1.5 mL/min. Condition 1: Mobile phases A and B under acidic conditions were 0.05% TFA in water and 0.05% TFA in MeCN, respectively. The ratio of mobile phase B was increased linearly from 5% to 90% over 0.9 min, 90% over the next 1.1 min. Condition 2: Mobile phases A and B under a neutral condition were a mixture of 5 mmol/L AcONH<sub>4</sub> and MeCN (9/1, v/v) and a mixture of 5 mmol/L AcONH<sub>4</sub> and MeCN (1/9, v/v), respectively. The ratio of mobile phase B was increased linearly from 5% to 90% over 0.9 min, 90% over the next 1.1 min. Specific rotation values were carried out by Sumika Chemical Analysis Service and observed in MeOH (1 w/v %).</div><div class="NLM_p">The purities of all the compounds tested in biological systems were assessed as being >95% using elemental analysis or analytical HPLC. Purity data were collected by HPLC with NQAD (nanoquality analyte detector) or Corona CAD (charged aerosol detector). The column was an L-column 2 ODS (30 mm × 2.1 mm i.d., CERI) or a Capcell Pak C18AQ (50 mm × 3.0 mm i.d., Shiseido) with a temperature of 50 °C and a flow rate of 0.5 mL/min. Mobile phases A and B under a neutral conditions were a mixture of 50 mmol/L AcONH<sub>4</sub>, water, and MeCN (1/8/1, v/v/v) and a mixture of 50 mmol/L AcONH<sub>4</sub> and MeCN (1/9, v/v), respectively. The ratio of mobile phase B was increased linearly from 5% to 95% over 3 min, 95% over the next 1 min. Reaction progress was determined by thin layer chromatography (TLC) analysis on Merck Kieselgel 60 F254 plates or Fuji Silysia NH plates. Chromatographic purification was carried out on silica gel columns (Merck Kieselgel 60, 70–230 mesh or 230–400 mesh, Merck; Chromatorex NH-DM 1020, 100–200 mesh, Fuji Silysia Chemical; Inject column and Universal column, YAMAZEN, <a href="http://yamazenusa.com/products/columns/" class="extLink">http://yamazenusa.com/products/columns/</a>; or Purif-Pack Si or NH, Shoko Scientific, <a href="http://shoko-sc.co.jp/english2/" class="extLink">http://shoko-sc.co.jp/english2/</a>). Preparative TLC was carried out on Merck Kieselgel 60 PLC plates. Preparative HPLC was acquired using a Gilson preparative HPLC system with UV detector (220 and 254 nm). The column used was a Symmetrix ODS-R (150 mm × 30 mm i.d., 5 μm, Boston Analytics), Synergi C18 (150 mm × 30 mm i.d., 4 μm, Phenomenex), Xbridge C18 (150 mm × 20 mm i.d., 5 μm, Waters), or Gemini C18 (150 mm × 25 mm i.d., 10 μm, Phenomenex), and a flow rate of 25 mL/min. Condition 1: Mobile phases A and B under acidic conditions were 0.225% formic acid in water and MeCN, respectively. Condition 2: Mobile phases A and B under acidic conditions were 1% TFA in water and MeCN, respectively. Condition 3: Mobile phases A and B under basic conditions were 0.05% aqueous ammonia solution and MeCN, respectively. The ratio of mobile phase B was increased linearly between 8 and 15 min. All commercially available solvents and reagents were used without further purification. Yields were not optimized.</div><div class="NLM_p">Compound <b>1</b> (hippuristanol) was purchased from Centaurus Biopharma (fee-for-service).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(1<i>H</i>-indol-3-yl)methanone (<b>3</b>)</h3><div class="NLM_p last">HATU (153 mg, 0.40 mmol) was added to a solution of <b>7a</b> (100 mg, 0.27 mmol), <b>9</b> (43 mg, 0.27 mmol), and DIPEA (70 mg, 0.54 mmol) in DMF (3 mL). After stirring at room temperature for 18 h, the reaction mixture was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 53/47 to 23/77) to give <b>3</b> (35 mg, 27%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.39–3.43 (m, 2H), 3.72–3.74 (m, 1H), 3.77–3.78 (m, 1H), 3.85–3.87 (m, 1H), 4.66–4.70 (m, 1H), 5.49 (s, 1H), 7.05–7.08 (m, 1H), 7.12–7.15 (m, 1H), 7.34 (s, 4H), 7.41–7.44 (m, 1H), 7.56 (s, 5H), 7.58–7.59 (m, 1H), 11.29 (s, 1H). MS <i>m</i>/<i>z</i> 478 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>26</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub>·1.25H<sub>2</sub>O: C, 62.34; H, 4.73; N, 8.39. Found: C, 62.27; H, 4.48; N, 8.38.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> <i>tert</i>-Butyl 3-(4-Chlorophenyl)piperazine-1-carboxylate (<b>5a</b>)</h3><div class="NLM_p last">To a solution of <b>4a</b> (100 mg, 0.51 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) were added Boc<sub>2</sub>O (111 mg, 0.51 mmol) and Et<sub>3</sub>N (51 mg, 0.51 mmol) at 0 °C. After being stirred at 15 °C for 2 h, the reaction mixture was diluted with water, extracted with CH<sub>2</sub>Cl<sub>2</sub>, and concentrated. The residue was purified by column chromatography on silica gel (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 50/1 to 30/1) to give <b>5a</b> (100 mg, 66%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.45–1.48 (m, 9H), 2.68 (brs, 1H), 2.77–2.99 (m, 2H), 3.06 (d, <i>J</i> = 8.1 Hz, 1H), 3.68 (dd, <i>J</i> = 10.5, 3.0 Hz, 1H), 4.04 (brs, 2H), 7.27–7.38 (m, 4H). 1H was not observed. MS <i>m</i>/<i>z</i> 297 (M + H)<sup>+</sup>.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>tert</i>-Butyl 3-(3-Chlorophenyl)piperazine-1-carboxylate (<b>5c</b>)</h3><div class="NLM_p last">To a solution of <b>4c</b> (0.94 g, 4.78 mmol) in toluene (12 mL) and THF (2 mL) was added Boc<sub>2</sub>O (1.11 mL, 4.78 mmol) dropwise at 0 °C. After stirring at room temperature for 3 h, the reaction mixture was concentrated. The residue was purified by silica gel column chromatography (hexane/EtOAc = 50/1 to 7/3) to give <b>5c</b> (0.99 g, 70%) as a yellow oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.47 (s, 9H), 2.71 (brs, 1H), 2.80–2.99 (m, 2H), 3.07 (d, <i>J</i> = 9.5 Hz, 1H), 3.69 (dd, <i>J</i> = 10.4, 2.9 Hz, 1H), 4.04 (brs, 2H), 7.19–7.33 (m, 3H), 7.44 (s, 1H). 1H was not observed. MS <i>m</i>/<i>z</i> 297 (M + H)<sup>+</sup>.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>tert</i>-Butyl 3-(2-Chlorophenyl)piperazine-1-carboxylate (<b>5d</b>)</h3><div class="NLM_p last">Compound <b>5d</b> was prepared from compound <b>4d</b> in a similar manner to that described for the synthesis of compound <b>5a</b> and obtained in 83% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.48 (s, 9H), 2.64 (dd, <i>J</i> = 12.6, 10.1 Hz, 1H), 2.84–3.00 (m, 2H), 3.02–3.14 (m, 1H), 3.95–4.09 (m, 1H), 4.14 (dd, <i>J</i> = 10.1, 2.8 Hz, 1H), 4.21 (d, <i>J</i> = 12.1 Hz, 1H), 7.16–7.31 (m, 3H), 7.33–7.41 (m, 1H), 7.62 (d, <i>J</i> = 7.3 Hz, 1H). MS <i>m</i>/<i>z</i> 297 (M + H)<sup>+</sup>.</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> <i>tert</i>-Butyl 4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazine-1-carboxylate (<b>6a</b>)</h3><div class="NLM_p last">To a solution of <b>5a</b> (6.30 g, 21.0 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (50 mL) were added 4-chlorobenzoyl chloride (3.90 g, 22.0 mmol) and Et<sub>3</sub>N (4.20 g, 42.0 mmol) at 0 °C. After stirring at 15 °C for 2 h, the reaction mixture was diluted with water, extracted with CH<sub>2</sub>Cl<sub>2</sub>, and concentrated. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 50/1) to give <b>6a</b> (8.65 g, 95%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.45 (brs, 9H), 3.03 (brs, 2H), 3.29 (brs, 1H), 3.78–4.20 (m, 2H), 4.64 (brs, 1H), 5.35–6.10 (m, 1H), 7.29–7.46 (m, 8H). MS <i>m</i>/<i>z</i> 457 (M + Na)<sup>+</sup>.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> <i>tert</i>-Butyl 4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazine-1-carboxylate (<b>6b</b>)</h3><div class="NLM_p last">Compound <b>6b</b> was prepared from compounds <b>5a</b> and 4-bromobenzoyl chloride in a similar manner to that described for the synthesis of compound <b>6a</b> and obtained in 87% yield as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 1.33 (s, 9H), 2.77–3.18 (m, 2H), 3.43–3.97 (m, 2H), 4.12–5.08 (m, 2H), 5.68 (brs, 1H), 7.08–7.53 (m, 6H), 7.67 (s, 2H). MS <i>m</i>/<i>z</i> 423 (M – (<i>t</i>-Bu) + H)<sup>+</sup>.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> <i>tert</i>-Butyl 4-(4-Chlorobenzoyl)-3-(3-chlorophenyl)piperazine-1-carboxylate (<b>6c</b>)</h3><div class="NLM_p last">Compound <b>6c</b> was prepared from compound <b>5c</b> in a similar manner to that described for the synthesis of compound <b>6a</b> and obtained in 78% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.49 (brs, 9H), 2.75–3.18 (m, 2H), 3.30 (d, <i>J</i> = 10.4 Hz, 1H), 3.79–4.14 (m, 1H), 4.67 (d, <i>J</i> = 14.0 Hz, 1H), 7.21–7.33 (m, 3H), 7.36–7.47 (m, 5H). 2H were not observed. MS <i>m</i>/<i>z</i> 379 (M – (<i>t</i>-Bu) + H)<sup>+</sup>.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>tert</i>-Butyl 4-(4-Chlorobenzoyl)-3-(2-chlorophenyl)piperazine-1-carboxylate (<b>6d</b>)</h3><div class="NLM_p last">To a solution of <b>5d</b> (1.00 g, 3.37 mmol) and DIPEA (871 mg, 6.74 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (30 mL) was added 4-chlorobenzoyl chloride (708 mg, 4.04 mmol) dropwise at 0 °C. The mixture was stirred at room temperature for 4 h. The reaction mixture was concentrated. The residue was purified by silica gel column chromatography (petroleum ether/EtOAc = 30/1 to 3/1) to give <b>6d</b> (1.40 g, 95%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.17–1.50 (m, 9H), 3.17 (brs, 1H), 3.42–4.37 (m, 5H), 5.68 (s, 1H), 7.18–7.28 (m, 3H), 7.29–7.46 (m, 5H). MS <i>m</i>/<i>z</i> 379 (M – (<i>t</i>-Bu) + H)<sup>+</sup>.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> (4-Chlorophenyl)-(2-(4-chlorophenyl)piperazin-1-yl)methanone Hydrochloride (<b>7a</b>)</h3><div class="NLM_p last">A mixture of <b>6a</b> (1.26 g, 2.89 mmol) and 4 M hydrogen chloride EtOAc solution (15 mL, 60.0 mmol) was stirred at 0 °C for 3 h. Then the mixture was concentrated to give <b>7a</b> (1.06 g, 99%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.98–3.28 (m, 3H), 3.52 (dd, <i>J</i> = 13.7, 4.6 Hz, 1H), 3.96 (d, <i>J</i> = 13.2 Hz, 2H), 5.60 (brs, 1H), 7.35–7.47 (m, 2H), 7.47–7.62 (m, 6H), 8.26–9.92 (m, 2H). MS <i>m</i>/<i>z</i> 335 (M + H)<sup>+</sup>.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> (4-Bromophenyl)-(2-(4-chlorophenyl)piperazin-1-yl)methanone Hydrochloride (<b>7b</b>)</h3><div class="NLM_p last">A mixture of <b>6b</b> (7.00 g, 14.6 mmol) and 4 M hydrogen chloride EtOAc solution (20 mL, 80.0 mmol) was stirred at room temperature for 2 h. Then the mixture was concentrated to give <b>7b</b> (4.00 g, 66%) as a yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.03–3.13 (m, 1H), 3.14–3.28 (m, 3H), 3.51–3.54 (m, 1H), 3.94–3.98 (m, 1H), 5.60 (s, 1H), 7.37–7.46 (m, 2H), 7.47–7.55 (m, 4H), 7.69 (d, <i>J</i> = 8.0 Hz, 2H), 8.60 (s, 1H), 9.71 (s, 1H). MS <i>m</i>/<i>z</i> 379 (M + H)<sup>+</sup>.</div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> (4-Chlorophenyl)-(2-(3-chlorophenyl)piperazin-1-yl)methanone Hydrochloride (<b>7c</b>)</h3><div class="NLM_p last">Compound <b>7c</b> was prepared from compound <b>6c</b> in a similar manner to that described for the synthesis of compound <b>7b</b> and obtained quantitatively as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.01–3.36 (m, 3H), 3.53 (brs, 1H), 4.00 (d, <i>J</i> = 13.7 Hz, 2H), 5.62 (brs, 1H), 7.33 (d, <i>J</i> = 7.4 Hz, 1H), 7.39–7.52 (m, 3H), 7.57 (s, 4H), 8.60 (brs, 1H), 9.72 (d, <i>J</i> = 8.4 Hz, 1H). MS <i>m</i>/<i>z</i> 335 (M + H)<sup>+</sup>.</div></div><div id="sec5_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> (4-Chlorophenyl)-(2-(2-chlorophenyl)piperazin-1-yl)methanone Hydrochloride (<b>7d</b>)</h3><div class="NLM_p last">Compound <b>7d</b> was prepared from compound <b>6d</b> in a similar manner to that described for the synthesis of compound <b>7b</b> and obtained in 84% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.27 (d, <i>J</i> = 5.7 Hz, 2H), 3.38–3.50 (m, 1H), 3.51–3.63 (m, 1H), 3.83–4.09 (m, 2H), 5.64 (dd, <i>J</i> = 8.5, 5.8 Hz, 1H), 7.31–7.44 (m, 2H), 7.44–7.55 (m, 5H), 7.55–7.63 (m, 1H), 9.41 (brs, 2H). MS <i>m</i>/<i>z</i> 335 (M + H)<sup>+</sup>.</div></div><div id="sec5_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (4-Bromophenyl)((2<i>S</i>)-2-(4-chlorophenyl)piperazin-1-yl)methanone (<b>8a</b>) and (4-Bromophenyl)((2<i>R</i>)-2-(4-chlorophenyl)piperazin-1-yl)methanone (<b>8b</b>)</h3><div class="NLM_p">To a solution of <b>6b</b> (3.40 g, 7.09 mmol) in THF (30 mL) was added 4 M hydrogen chloride EtOAc solution (53.1 mL, 213 mmol) at 0 °C. The resulting mixture was allowed to warm to room temperature and stirred at room temperature for 1 h. The reaction mixture was basified with saturated aqueous NaHCO<sub>3</sub> solution at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 9/1 to 0/10) to give (4-bromophenyl)-(2-(4-chlorophenyl)piperazin-1-yl)methanone (free form of <b>7b</b>, 2.60 g, 97%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.69–3.01 (m, 2H), 3.04–3.30 (m, 2H), 3.62 (d, <i>J</i> = 13.0 Hz, 1H), 4.96–6.01 (m, 1H), 7.27–7.47 (m, 6H), 7.54 (d, <i>J</i> = 8.2 Hz, 2H). 2H were not observed. MS <i>m</i>/<i>z</i> 379 (M + H)<sup>+</sup>.</div><div class="NLM_p">The racemic mixture (2.60 g) was separated by HPLC (YMC K-prep system, CHIRALPAK AD (Daicel) 500 mm × 50 mm i.d., hexane/<i>i</i>-PrOH/Et<sub>2</sub>NH = 400/600/1 (v/v/v), flow rate of 60 mL/min) to give <b>8a</b> (1.13 g, 42%, 99.9% ee, retention time 14.2 min) as a colorless oil and <b>8b</b> (1.10 g, 41%, 99.8% ee, retention time 24.0 min) as a colorless oil. <b>8a</b>: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.70–2.99 (m, 2H), 3.01–3.28 (m, 2H), 3.62 (d, <i>J</i> = 12.6 Hz, 1H), 4.61–6.25 (m, 1H), 7.27–7.45 (m, 6H), 7.54 (d, <i>J</i> = 8.1 Hz, 2H). 2H were not observed. MS <i>m</i>/<i>z</i> 379 (M + H)<sup>+</sup>. <b>8b</b>: <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.73–3.01 (m, 2H), 3.02–3.33 (m, 2H), 3.62 (d, <i>J</i> = 12.6 Hz, 1H), 5.01–6.06 (m, 1H), 7.27–7.46 (m, 6H), 7.54 (d, <i>J</i> = 8.1 Hz, 2H). 2H were not observed. MS <i>m</i>/<i>z</i> 379 (M + H)<sup>+</sup>.</div><div class="NLM_p last">Analytical HPLC conditions for determination of the % ee values of <b>8a</b> and <b>8b</b>: Waters system, CHIRALPAK AD (Daicel) 250 mm × 4.6 mm i.d., hexane/<i>i</i>-PrOH/Et<sub>2</sub>NH = 400/600/1 (v/v/v), flow rate of 0.5 mL/min. Absolute stereochemistry was determined as a (2<i>S</i>)-enantiomer by X-ray crystal structure analysis of TsOH salt of <b>8a</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf" class="ext-link">Supporting Information</a>).</div></div><div id="sec5_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(1<i>H</i>-indazol-3-yl)methanone (<b>29</b>)</h3><div class="NLM_p last">Compound <b>29</b> was prepared from compounds <b>7a</b> and <b>10</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 39% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.39–3.56 (m, 2H), 3.82–4.02 (m, 2H), 4.37–4.52 (m, 1H), 4.99–5.10 (m, 1H), 5.54 (s, 1H,), 7.15–7.24 (m, 1H), 7.26–7.44 (m, 5H), 7.50 (s, 4H), 7.57–7.60 (m, 1H), 7.88 (d, <i>J</i> = 8.1 Hz, 1H), 13.12–13.45 (s, 1H). MS <i>m</i>/<i>z</i> 479 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·H<sub>2</sub>O: C, 60.37; H, 4.46; N, 11.26. Found: C, 60.11; H, 4.32; N, 11.14.</div></div><div id="sec5_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(pyrazolo[1,5-<i>a</i>]pyridin-3-yl)methanone (<b>30</b>)</h3><div class="NLM_p last">A mixture of <b>7a</b> (100 mg, 0.30 mmol), <b>11</b> (53 mg, 0.33 mmol), EDC·HCl (69 mg, 0.36 mmol), HOBt (55 mg, 0.36 mmol), and DIPEA (0.16 mL, 0.89 mmol) in DMF (3 mL) was stirred at room temperature overnight. The reaction mixture was poured into water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 95/5 to 20/80) to give <b>30</b> (112 mg, 78%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.23–3.46 (m, 2H), 3.52–4.51 (m, 3H), 4.98 (d, <i>J</i> = 14.1 Hz, 1H), 5.36–6.06 (m, 1H), 6.89–6.98 (m, 1H), 7.27–7.47 (m, 9H), 7.86 (d, <i>J</i> = 9.0 Hz, 1H), 7.92 (s, 1H), 8.49 (d, <i>J</i> = 6.9 Hz, 1H). MS <i>m</i>/<i>z</i> 479 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>: C, 62.64; H, 4.21; N, 11.69. Found: C, 62.34; H, 4.20; N, 11.48.</div></div><div id="sec5_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(1-phenyl-1<i>H</i>-pyrazol-4-yl)methanone (<b>31</b>)</h3><div class="NLM_p last">Compound <b>31</b> was prepared from compounds <b>7a</b> and <b>18</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 50% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.42–3.51 (m, 2H), 3.73–3.79 (m, 1H), 3.79–3.94 (m, 1H), 4.05–4.12 (m, 1H), 4.60–4.66 (m, 1H), 5.50 (brs, 1H), 7.35–7.40 (m, 5H), 7.50–7.55 (m, 6H), 7.81–7.88 (m, 3H), 8.59 (s, 1H). MS <i>m</i>/<i>z</i> 505 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·0.3H<sub>2</sub>O: C, 63.49; H, 4.46; N, 10.97. Found: C, 63.38; H, 4.43; N, 10.95.</div></div><div id="sec5_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(1-phenyl-1<i>H</i>-pyrazol-3-yl)methanone (<b>32</b>)</h3><div class="NLM_p last">A solution of <b>19</b> (45 mg, 0.21 mmol) and lithium hydroxide hydrate (26 mg, 0.62 mmol) in THF (1 mL), MeOH (1 mL), and water (1 mL) was stirred at 40 °C for 1 h. The reaction mixture was concentrated, acidified to pH 3 with 1 M hydrochloric acid, and extracted with EtOAc. The separated organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The residue was dissolved in DMF (2 mL), then <b>7a</b> (77 mg, 0.21 mmol), DIPEA (81 mg, 0.62 mmol), and HATU (119 mg, 0.31 mmol) were added. After being stirred at room temperature for 1.5 h, the reaction mixture was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 37/63 to 16/84) to give <b>32</b> (75 mg, 71%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.32–3.53 (m, 2H), 3.77–4.04 (m, 2H), 4.34–4.53 (m, 1H), 4.94–5.06 (m, 1H), 5.54 (s, 1H), 6.75 (d, <i>J</i> = 2.1 Hz, 1H), 7.37 (s, 5H), 7.50 (s, 6H), 7.75–7.85 (m, 2H), 8.42 (s, 1H). MS <i>m</i>/<i>z</i> 505 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>22</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·0.5H<sub>2</sub>O: C, 63.04; H, 4.51; N, 10.89. Found: C, 63.00; H, 4.42; N, 10.74.</div></div><div id="sec5_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-4-yl)methanone (<b>33</b>)</h3><div class="NLM_p last">Compound <b>33</b> was prepared from compounds <b>7a</b> and <b>20</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 24% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.27–3.41 (m, 2H), 3.70–3.81 (m, 1H), 3.84–3.96 (m, 1H), 4.22–4.37 (m, 1H), 4.84–4.97 (m, 1H), 5.41–5.51 (m, 1H), 7.25–7.35 (m, 4H), 7.41 (s, 4H), 7.47 (m, 3H), 7.86–7.95 (m, 2H), 8.38 (s, 1H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.2H<sub>2</sub>O: C, 66.52; H, 4.86; N, 8.31. Found: C, 66.47; H, 4.90; N, 8.40.</div></div><div id="sec5_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>34</b>)</h3><div class="NLM_p last">Compound <b>34</b> was prepared from compounds <b>7a</b> and <b>21</b> in a similar manner to that described for the synthesis of compound <b>30</b> and obtained in 79% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.21–3.84 (m, 4H), 4.35–4.49 (m, 1H), 4.93–5.22 (m, 1H), 5.50–6.08 (m, 1H), 7.29–7.56 (m, 11H), 7.69 (s, 1H), 8.00 (dd, <i>J</i> = 7.8, 1.6 Hz, 2H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.1H<sub>2</sub>O: C, 63.81; H, 4.20; N, 8.27. Found: C, 63.71; H, 4.09; N, 8.23.</div></div><div id="sec5_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(5-phenyl-1,3-oxazol-2-yl)methanone (<b>35</b>)</h3><div class="NLM_p last">Compound <b>7a</b> (170 mg, 0.46 mmol) was basified with saturated aqueous NaHCO<sub>3</sub> solution and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated. The residue was azeotroped with toluene and dissolved in THF (3 mL). To the solution were added <b>24</b> (99 mg, 0.46 mmol) and (Me<sub>3</sub>Al)<sub>2</sub>·DABCO<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> (94 mg, 0.37 mmol) at room temperature. The resulting mixture was heated at 130 °C for 1 h under microwave irradiation. The reaction mixture was diluted with THF and added Na<sub>2</sub>SO<sub>4</sub> decahydrate (147 mg, 0.46 mmol) at room temperature. After stirring at room temperature overnight, the insoluble material was removed by filtration and washed with THF. The filtrate was concentrated, and the residue was purified by silica gel column chromatography (hexane/EtOAc = 95/5 to 30/70) to give <b>35</b> (180 mg, 78%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.10–3.52 (m, 3H), 4.01 (d, <i>J</i> = 10.9 Hz, 1H), 4.50 (d, <i>J</i> = 10.9 Hz, 1H), 5.29 (t, <i>J</i> = 15.0 Hz, 1H), 5.77 (d, <i>J</i> = 15.0 Hz, 1H), 7.29–7.54 (m, 12H), 7.74 (d, <i>J</i> = 6.8 Hz, 2H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>: C, 64.04; H, 4.18; N, 8.30. Found: C, 64.08; H, 4.26; N, 8.22.</div></div><div id="sec5_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(5-methyl-1-phenyl-1<i>H</i>-pyrazol-4-yl)methanone (<b>36</b>)</h3><div class="NLM_p last">Compound <b>36</b> was prepared from compounds <b>7a</b> and <b>25</b> in a similar manner to that described for the synthesis of compound <b>30</b> and obtained in 34% yield as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 2.31 (s, 3H), 3.28–4.22 (m, 5H), 5.09–5.89 (m, 2H), 7.33–7.50 (m, 14H). MS <i>m</i>/<i>z</i> 519 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>: C, 64.74; H, 4.66; N, 10.79. Found: C, 64.52; H, 4.75; N, 10.83.</div></div><div id="sec5_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> (4-(4-Chlorobenzoyl)-3-(3-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>37</b>)</h3><div class="NLM_p last">Compound <b>37</b> was prepared from compounds <b>7c</b> and <b>21</b> in a similar manner to that described for the synthesis of compound <b>30</b> and obtained in 51% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.37–3.88 (m, 3H), 4.17 (brs, 1H), 4.76 (d, <i>J</i> = 13.2 Hz, 1H), 5.72 (brs, 1H), 7.34 (brs, 4H), 7.57 (d, <i>J</i> = 6.1 Hz, 7H), 7.82 (brs, 1H), 8.02 (dd, <i>J</i> = 7.5, 2.1 Hz, 2H). 1H was not observed. MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.2H<sub>2</sub>O: C, 63.59; H, 4.23; N, 8.24. Found: C, 63.46; H, 4.11; N, 8.17.</div></div><div id="sec5_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> (4-(4-Chlorobenzoyl)-3-(2-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>38</b>)</h3><div class="NLM_p last">Compound <b>38</b> was prepared from compound <b>7d</b> and <b>21</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 31% yield as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.43–4.88 (m, 6H), 5.31–6.30 (m, 1H), 7.18–7.41 (m, 7H), 7.42–7.57 (m, 5H), 8.03 (brs, 2H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.2H<sub>2</sub>O: C, 63.59; H, 4.23; N, 8.24. Found: C, 63.46; H, 4.11; N, 8.17.</div></div><div id="sec5_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> (7-Bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)(4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)methanone (<b>39</b>)</h3><div class="NLM_p last">Compound <b>39</b> was prepared from compounds <b>7a</b> and <b>12</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 43% yield as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.40–3.51 (m, 2H), 3.80–4.11 (m, 3H), 4.60–4.66 (m, 1H), 5.49 (brs, 1H), 7.28–7.33 (m, 5H), 7.43–7.52 (m, 5H), 7.80 (d, <i>J</i> = 8.8 Hz, 1H), 8.25 (s, 1H). MS <i>m</i>/<i>z</i> 557 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>BrCl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·0.4H<sub>2</sub>O: C, 53.10; H, 3.53; N, 9.91. Found: C, 53.16; H, 3.47; N, 9.96.</div></div><div id="sec5_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> (6-Bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)(4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)methanone (<b>40</b>)</h3><div class="NLM_p last">Compound <b>40</b> was prepared from compounds <b>7a</b> and <b>13</b> in a similar manner to that described for the synthesis of compound <b>30</b> and obtained in 68% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.37–3.46 (m, 2H), 3.76 (brs, 2H), 4.04–4.22 (m, 1H), 4.68 (d, <i>J</i> = 14.3 Hz, 1H), 5.63 (brs, 1H), 7.32 (brs, 4H), 7.42–7.62 (m, 5H), 7.71 (brs, 1H), 8.21 (s, 1H), 9.19 (s, 1H). MS <i>m</i>/<i>z</i> 557 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>BrCl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·0.75C<sub>4</sub>H<sub>8</sub>O<sub>2</sub>: C, 53.87; H, 4.04; N, 8.97. Found: C, 54.03; H, 4.11; N, 9.05. Purity 98.6% (HPLC).</div></div><div id="sec5_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> (5-Bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)(4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)methanone (<b>41</b>)</h3><div class="NLM_p last">Compound <b>41</b> was prepared from compounds <b>7a</b> and <b>14</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 55% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.40–3.54 (m, 2H), 3.84 (dd, <i>J</i> = 15.0, 6.0 Hz, 1H), 3.92–4.10 (m, 2H), 4.60 (dd, <i>J</i> = 15.0, 3.0 Hz, 1H), 5.47 (brs, 1H), 7.20 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 7.32 (s, 4H), 7.48 (s, 4H), 7.95 (d, <i>J</i> = 3.0 Hz, 1H), 8.15 (s, 1H), 8.64 (d, <i>J</i> = 6.0 Hz, 1H). MS <i>m</i>/<i>z</i> 557 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>BrCl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·0.7H<sub>2</sub>O: C, 52.60; H, 3.60; N, 9.81. Found: C, 52.52; H, 3.44; N, 9.96.</div></div><div id="sec5_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> (4-Bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)(4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)methanone (<b>42</b>)</h3><div class="NLM_p last">Compound <b>42</b> was prepared from compounds <b>7a</b> and <b>15</b> in a similar manner to that described for the synthesis of compound <b>3</b> and obtained in 61% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.21–3.34 (m, 2H), 3.58–3.64 (m, 1H), 3.86–3.89 (m, 2H), 4.60 (brs, 1H), 5.49 (brs, 1H), 6.89 (t, <i>J</i> = 6.0 Hz, 1H), 7.35–7.40 (m, 4H), 7.48 (s, 4H), 7.58 (d, <i>J</i> = 9.0 Hz, 1H), 7.91 (s, 1H), 8.71 (d, <i>J</i> = 6.0 Hz, 1H). MS <i>m</i>/<i>z</i> 557 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>BrCl<sub>2</sub>N<sub>4</sub>O<sub>2</sub>·H<sub>2</sub>O: C, 52.11; H, 3.67; N, 9.72. Found: C, 52.24; H, 3.51; N, 9.86.</div></div><div id="sec5_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> (4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(6-bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)methanone (<b>43</b>)</h3><div class="NLM_p last">Compound <b>43</b> was prepared from compounds <b>7b</b> and <b>13</b> in a similar manner to that described for the synthesis of compound <b>30</b> and obtained in 76% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.43 (brs, 2H), 3.75 (brs, 2H), 4.03 (q, <i>J</i> = 7.2 Hz, 1H), 4.68 (d, <i>J</i> = 12.0 Hz, 1H), 5.67 (brs, 1H), 7.20–7.50 (m, 6H), 7.55 (dd, <i>J</i> = 9.4, 1.3 Hz, 1H), 7.68 (d, <i>J</i> = 7.6 Hz, 3H), 8.21 (s, 1H), 9.19 (s, 1H). MS <i>m</i>/<i>z</i> 601 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>Br<sub>2</sub>ClN<sub>4</sub>O<sub>2</sub>·0.2C<sub>6</sub>H<sub>14</sub>: C, 50.76; H, 3.54; N, 9.04. Found: C, 50.78; H, 3.71; N, 8.90. Purity 97.7% (HPLC).</div></div><div id="sec5_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> (4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(6-methoxypyrazolo[1,5-<i>a</i>]pyridin-3-yl)methanone (<b>44</b>)</h3><div class="NLM_p last">To a solution of <b>16</b> (80 mg, 0.39 mmol) in MeOH (1 mL), THF (1 mL), and water (1 mL) was added lithium hydroxide hydrate (98 mg, 2.33 mmol) at room temperature. After stirring at 50 °C for 2 h, the reaction mixture was concentrated to give lithium 6-methoxypyrazolo[1,5-<i>a</i>]pyridine-3-carboxylate (70 mg, 91%) as a white solid. A solution of lithium 6-methoxypyrazolo[1,5-<i>a</i>]pyridine-3-carboxylate (70 mg, 0.35 mmol), <b>7b</b> (148 mg, 0.35 mmol), HATU (148 mg, 0.39 mmol), and DIPEA (183 mg, 1.41 mmol) in DMF (2 mL) was stirred at room temperature for 2 h. The reaction mixture was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 53/47 to 23/77) to give <b>44</b> (33 mg, 17%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.42–3.48 (m, 2H), 3.78–3.94 (m, 5H), 4.07–4.11 (m, 1H), 4.57–4.63 (m, 1H), 5.46 (s, 1H), 7.21–7.31 (m, 1H), 7.38 (s, 4H), 7.41 (d, <i>J</i> = 8.4 Hz, 2H), 7.62 (d, <i>J</i> = 8.4 Hz, 2H), 7.71 (d, <i>J</i> = 9.6 Hz, 1H), 8.00 (s, 1H), 8.37 (d, <i>J</i> = 1.5 Hz, 1H). MS <i>m</i>/<i>z</i> 553 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>26</sub>H<sub>22</sub>BrClN<sub>4</sub>O<sub>3</sub>·1.5H<sub>2</sub>O: C, 53.76; H, 4.34; N, 9.65. Found: C, 53.48; H, 3.98; N, 9.51.</div></div><div id="sec5_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> 3-((4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)pyrazolo[1,5-<i>a</i>]pyridine-6-carbonitrile (<b>45</b>)</h3><div class="NLM_p last">To a solution of <b>17</b> (80 mg, 0.33 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was added TFA (3 mL). The resulting mixture was stirred at room temperature for 1 h and then concentrated. To a solution of the residue, <b>7b</b> (94 mg, 0.23 mmol), and Et<sub>3</sub>N (83 mg, 0.82 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) was added HATU (86 mg, 0.23 mmol). The resulting mixture was stirred at room temperature for 5 h and concentrated. The residue was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 40/60 to 10/90) to give <b>45</b> (60 mg, 53%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.38–3.55 (m, 2H), 3.83 (dd, <i>J</i> = 14.0, 4.4 Hz, 1H), 3.94 (d, <i>J</i> = 10.0 Hz, 1H), 4.08 (d, <i>J</i> = 11.6 Hz, 1H), 4.62 (d, <i>J</i> = 14.0 Hz, 1H), 5.48 (s, 1H), 7.31 (s, 4H), 7.42 (d, <i>J</i> = 8.4 Hz, 2H), 7.58 (d, <i>J</i> = 9.2 Hz, 1H), 7.65 (d, <i>J</i> = 8.4 Hz, 2H), 7.83 (d, <i>J</i> = 9.2 Hz, 1H), 8.36 (s, 1H), 9.54 (s, 1H,). MS <i>m</i>/<i>z</i> 548 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>26</sub>H<sub>19</sub>BrClN<sub>5</sub>O<sub>2</sub>·0.3H<sub>2</sub>O: C, 56.35; H, 3.56; N, 12.64. Found: C, 56.44; H, 3.52; N, 12.45.</div></div><div id="sec5_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> 3-(5-((4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-1,3-oxazol-2-yl)benzonitrile (<b>46</b>)</h3><div class="NLM_p last">Compound <b>46</b> was prepared from compounds <b>7b</b> and <b>27</b> in a similar manner to that described for the synthesis of compound <b>35</b> and obtained in 47% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.00–3.80 (m, 3H), 3.97–4.56 (m, 2H), 4.85–5.25 (m, 1H), 5.50–6.05 (m, 1H), 7.27–7.42 (m, 6H), 7.55–7.69 (m, 4H), 7.74–7.83 (m, 1H), 8.20–8.32 (m, 2H). MS <i>m</i>/<i>z</i> 575 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>20</sub>BrClN<sub>4</sub>O<sub>3</sub>: C, 58.40; H, 3.50; N, 9.73. Found: C, 58.41; H, 3.73; N, 9.69.</div></div><div id="sec5_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> 2-(5-((4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-1,3-oxazol-2-yl)isonicotinonitrile (<b>47</b>)</h3><div class="NLM_p last">To a solution of <b>6b</b> (3.40 g, 7.09 mmol) in THF (30 mL) was added 4 M hydrogen chloride EtOAc solution (53.1 mL, 213 mmol) at 0 °C. The mixture was allowed to warm to room temperature and stirred at room temperature for 1 h. The mixture was basified with saturated aqueous NaHCO<sub>3</sub> solution at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 9/1 to 0/100) to give (4-bromophenyl)-(2-(4-chlorophenyl)piperazin-1-yl)methanone (free form of <b>7b</b>, 2.60 g, 97%) as a colorless oil. A mixture of (4-bromophenyl)-(2-(4-chlorophenyl)piperazin-1-yl)methanone (312 mg, 0.82 mmol), <b>28</b> (200 mg, 0.82 mmol), and 1 M Me<sub>3</sub>Al in CH<sub>2</sub>Cl<sub>2</sub> (1.64 mL, 1.64 mmol) in THF (10 mL) was stirred at 110 °C under microwave irradiation for 1 h. The reaction mixture was diluted with water and extracted with EtOAc. The separated organic layer was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 52/48 to 22/78) to give <b>47</b> (260 mg, 55%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.44–3.62 (m, 2H), 3.86–4.03 (m, 2H), 4.08–4.16 (m, 1H), 4.66 (dd, <i>J</i> = 14.8, 3.9 Hz, 1H), 5.53 (brs, 1H), 7.33–7.45 (m, 6H), 7.61–7.67 (m, 2H), 7.82 (s, 1H), 7.97 (dd, <i>J</i> = 4.8, 1.5 Hz, 1H), 8.30 (s, 1H), 9.03 (dd, <i>J</i> = 4.8, 0.9 Hz, 1H). MS <i>m</i>/<i>z</i> 576 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>19</sub>BrClN<sub>5</sub>O<sub>3</sub>·0.7H<sub>2</sub>O: C, 55.02; H, 3.49; N, 11.88. Found: C, 54.94; H, 3.45; N, 12.04.</div></div><div id="sec5_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 3-(4-((4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-5-methyl-1<i>H</i>-pyrazol-1-yl)benzonitrile (<b>48</b>)</h3><div class="NLM_p last">Compound <b>48</b> was prepared from compounds <b>7b</b> and <b>26</b> in a similar manner to that described for the synthesis of compound <b>30</b> and obtained in 64% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.20 (brs, 3H), 2.95–3.41 (m, 2H), 3.82–4.13 (m, 2H), 4.90 (brs, 1H), 5.73 (brs, 1H), 7.19–7.55 (m, 6H), 7.62–7.82 (m, 4H), 7.87–7.99 (m, 2H), 8.07 (s, 1H). 1H was not observed. MS <i>m</i>/<i>z</i> 588 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>29</sub>H<sub>23</sub>BrClN<sub>5</sub>O<sub>2</sub>·0.1H<sub>2</sub>O: C, 58.97; H, 3.96; N, 11.86. Found: C, 59.16; H, 4.33; H, N, 11.74.</div></div><div id="sec5_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> 2-(5-((4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-1,3-oxazol-2-yl)benzonitrile (<b>49</b>)</h3><div class="NLM_p">A mixture of <b>7a</b> (70 mg, 0.19 mmol), <b>22</b> (29.8 mg, 0.26 mmol), EDC·HCl (50.5 mg, 0.26 mmol), and HOBt (35.6 mg, 0.26 mmol) in DMF (3 mL) was stirred at room temperature overnight. The reaction mixture was poured into water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 98/2 to 50/50) to give (4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(1,3-oxazol-5-yl)methanone (20 mg, 25%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.54 (brs, 2H), 4.02 (brs, 1H), 4.70 (brs, 1H), 5.62 (brs, 1H), 7.40 (brs, 4H), 7.54 (brs, 4H), 7.67 (brs, 1H), 8.54 (s, 1H). 2H were not observed. MS <i>m</i>/<i>z</i> 430 (M + H)<sup>+</sup>.</div><div class="NLM_p last">A mixture of (4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(1,3-oxazol-5-yl)methanone (100 mg, 0.232 mmol), 2-boromobenzonitrile (64 mg, 0.348 mmol), Pd(OAc)<sub>2</sub> (3 mg, 12 μmol), Cs<sub>2</sub>CO<sub>3</sub> (151 mg, 0.465 mmol), and CuI (44 mg, 0.232 mmol) in DMF (2 mL) was bubbled with N<sub>2</sub> for 2 min, then heated at 150 °C for 5 min under microwave irradiation. The reaction mixture was poured into water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative HPLC (condition 2, 1% TFA in water/MeCN = 90/10 to 20/80) to give <b>49</b> (20 mg, 16%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.44–3.64 (m, 2H), 3.85–4.00 (m, 2H), 4.20–4.25 (d, <i>J</i> = 16.8 Hz, 1H), 4.68–4.74 (dd, <i>J</i> = 18.8, 4.0 Hz, 1H), 5.53 (brs, 1H), 7.38 (s, 4H), 7.50 (s, 4H), 7.73–7.79 (m, 1H), 7.86–7.91 (m, 2H), 7.99–8.01 (m, 1H), 8.17–8.19 (m, 1H). MS <i>m</i>/<i>z</i> 531 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub>·0.3H<sub>2</sub>O: C, 62.65; H, 3.87; N, 10.44. Found: C, 62.57; H, 3.90; N, 10.46.</div></div><div id="sec5_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> 3-(5-((4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-1,3-oxazol-2-yl)benzonitrile (<b>50</b>)</h3><div class="NLM_p">To a solution of <b>23</b> (354 mg, 1.48 mmol) in DMF (10 mL) were added HATU (844 mg, 2.22 mmol) and DIPEA (0.773 mL, 4.44 mmol) at 0 °C. The resulting mixture was stirred at 0 °C for 5 min, and then <b>7a</b> (550 mg, 1.48 mmol) was added to the mixture at 0 °C. After stirring at 0 °C for 30 min, the reaction mixture was poured into water at room temperature and extracted with EtOAc. The organic layer was washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated. The residue was purified by silica gel column chromatography (hexane/EtOAc = 95/5 to 30/70) to give (4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-iodo-1,3-oxazol-5-yl)methanone (785 mg, 95%) as a pale yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 2.90–3.86 (m, 3H), 4.26 (brs, 1H), 5.00 (brs, 1H), 5.39–6.03 (m, 1H), 7.27–7.45 (m, 8H), 7.51 (s, 1H). 1H was not observed. MS <i>m</i>/<i>z</i> 556 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a solution of (4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-iodo-1,3-oxazol-5-yl)methanone (140 mg, 0.252 mmol) and 3-cyanophenylboronic acid (74 mg, 0.503 mmol) in DME (3 mL) and H<sub>2</sub>O (1 mL) were added Pd(PPh<sub>3</sub>)<sub>4</sub> (29 mg, 25 μmol) and Na<sub>2</sub>CO<sub>3</sub> (53 mg, 0.503 mmol) under N<sub>2</sub>. The resulting mixture was refluxed for 12 h. The reaction mixture was concentrated, and the residue was purified by preparative TLC (EtOAc/petroleum ether =1/1) to give <b>50</b> (40 mg, 30%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.45–3.60 (m, 2H), 3.87–4.02 (m, 2H), 4.16–4.20 (m, 1H), 4.68–4.73 (dd, <i>J</i> = 14.0, 3.2 Hz, 1H), 5.53 (brs, 1H), 7.39 (s, 4H), 7.52 (s, 4H), 7.78–7.80 (m, 2H), 8.01 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 8.30 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 8.34 (t, <i>J</i> = 1.2 Hz, 1H). MS <i>m</i>/<i>z</i> 531 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub>·0.4H<sub>2</sub>O: C, 62.44; H, 3.89; N, 10.40. Found: C, 62.40; H, 3.92; N, 10.50.</div></div><div id="sec5_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> 4-(5-((4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-1,3-oxazol-2-yl)benzonitrile (<b>51</b>)</h3><div class="NLM_p last">Compound <b>51</b> was prepared from (4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-iodo-1,3-oxazol-5-yl)methanone in a similar manner to that described for the synthesis of compound <b>50</b> and obtained in 49% yield as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.11–3.81 (m, 3H), 3.96–4.50 (m, 2H), 4.77–5.18 (m, 1H), 5.39–6.02 (m, 1H), 7.28–7.46 (m, 8H), 7.67 (s, 1H), 7.77 (d, <i>J</i> = 8.2 Hz, 2H), 8.11 (d, <i>J</i> = 8.2 Hz, 2H). MS <i>m</i>/<i>z</i> 531 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub>·0.5H<sub>2</sub>O: C, 62.23; H, 3.92; N, 10.37. Found: C, 62.15; H, 3.94; N, 10.42.</div></div><div id="sec5_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> ((3<i>S</i>)-4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(6-bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)methanone (<b>52a</b>)</h3><div class="NLM_p last">A solution of <b>8a</b> (200 mg, 0.53 mmol), <b>13</b> (165 mg, 0.68 mmol), EDC·HCl (131 mg, 0.68 mmol), and HOBt (93 mg, 0.68 mmol) in DMF (5 mL) was stirred at room temperature overnight. The reaction mixture was poured into water and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 8/2 to 2/8) to give <b>52a</b> (232 mg, 73%, >99.9% ee, retention time 28.6 min) as colorless needles. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 °C) δ 3.35–3.51 (m, 2H), 3.80 (dd, <i>J</i> = 14.1, 4.5 Hz, 1H), 3.91 (d, <i>J</i> = 11.0 Hz, 1H), 4.03–4.12 (m, 1H), 4.60 (dd, <i>J</i> = 14.1, 2.6 Hz, 1H), 5.45 (brs, 1H), 7.26–7.33 (m, 4H), 7.37–7.42 (m, 2H), 7.49 (dd, <i>J</i> = 9.3, 1.7 Hz, 1H), 7.60–7.65 (m, 2H), 7.71 (d, <i>J</i> = 10.0 Hz, 1H), 8.12 (s, 1H), 9.01–9.03 (m, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 °C) δ 41.7, 44.7, 46.2, 54.4, 106.3, 107.4, 119.2, 123.1, 128.4 (2C), 128.4 (2C), 128.9 (2C), 129.26, 129.32, 131.5 (2C), 132.0, 135.0, 137.8, 139.0, 141.7, 163.2, 169.4. MS <i>m</i>/<i>z</i> 601 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>Br<sub>2</sub>ClN<sub>4</sub>O<sub>2</sub>·0.25C<sub>6</sub>H<sub>14</sub>O: C, 50.66; H, 3.61; N, 8.92. Found: C, 50.54; H, 3.52; N, 8.95. Purity 100% (HPLC). [α]<sub>D</sub><sup>25</sup> = +4.7°.</div></div><div id="sec5_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> ((3<i>R</i>)-4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(6-bromopyrazolo[1,5-<i>a</i>]pyridin-3-yl)methanone (<b>52b</b>)</h3><div class="NLM_p">Compound <b>52b</b> was prepared from compounds <b>13</b> and <b>8b</b> in a similar manner to that described for the synthesis of compound <b>52a</b> and obtained in 50% yield (>99.9% ee, retention time 33.9 min) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 3.36–3.55 (m, 2H), 3.63–3.90 (m, 2H), 4.07 (brs, 1H), 4.68 (d, <i>J</i> = 13.5 Hz, 1H), 5.66 (brs, 1H), 7.33 (brs, 4H), 7.45 (brs, 2H), 7.55 (d, <i>J</i> = 9.4 Hz, 1H), 7.68 (d, <i>J</i> = 7.3 Hz, 3H), 8.21 (s, 1H), 9.20 (s, 1H). MS <i>m</i>/<i>z</i> 601 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>25</sub>H<sub>19</sub>Br<sub>2</sub>ClN<sub>4</sub>O<sub>2</sub>·0.25C<sub>6</sub>H<sub>14</sub>O: C, 50.66; H, 3.61; N, 8.92. Found: C, 50.57; H, 3.66; N, 8.70. Purity 100% (HPLC). [α]<sub>D</sub><sup>25</sup> = −4.9°.</div><div class="NLM_p last">Analytical HPLC condition for determination of the % ee values of <b>52a</b> and <b>52b</b>: Waters system, CHIRALPAK IC-3 (Daicel) 250 mm × 4.6 mm i.d., EtOH 100%, flow rate of 0.5 mL/min.</div></div><div id="sec5_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> 3-(4-(((3<i>S</i>)-4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-5-methyl-1<i>H</i>-pyrazol-1-yl)benzonitrile (<b>53a</b>)</h3><div class="NLM_p last">Compound <b>53a</b> was prepared from compounds <b>26</b> and <b>8a</b> in a similar manner to that described for the synthesis of compound <b>52a</b> and obtained in 81% yield (>99.0% ee, retention time 14.3 min) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 °C) δ 2.21 (s, 3H), 3.24–3.40 (m, 2H), 3.68 (dd, <i>J</i> = 14.1, 3.8 Hz, 1H), 3.90 (d, <i>J</i> = 9.3 Hz, 1H), 3.98 (d, <i>J</i> = 10.3 Hz, 1H), 4.64 (d, <i>J</i> = 13.4 Hz, 1H), 5.48 (brs, 1H), 7.28–7.33 (m, 2H), 7.35–7.44 (m, 4H), 7.61–7.65 (m, 3H), 7.73 (t, <i>J</i> = 8.0 Hz, 1H), 7.83–7.90 (m, 2H), 7.97 (t, <i>J</i> = 1.6 Hz, 1H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 90 °C) δ 11.0, 41.5, 44.5, 46.0, 54.1, 112.6, 116.0, 117.7, 123.2, 128.1, 128.48 (2C), 128.49 (2C), 128.9 (2C), 129.6, 130.7, 131.6 (2C), 131.8, 132.1, 134.9, 137.5, 139.2, 139.6, 140.5, 164.0, 169.4. MS <i>m</i>/<i>z</i> 588 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>29</sub>H<sub>23</sub>BrClN<sub>5</sub>O<sub>2</sub>·0.25C<sub>6</sub>H<sub>14</sub>O·0.1H<sub>2</sub>O: C, 59.45; H, 4.37; N, 11.36. Found: C, 59.37; H, 4.34; N, 11.25. Purity 98.6% (HPLC). [α]<sub>D</sub><sup>25</sup> = +11.2°.</div></div><div id="sec5_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> 3-(4-(((3<i>R</i>)-4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)carbonyl)-5-methyl-1<i>H</i>-pyrazol-1-yl)benzonitrile (<b>53b</b>)</h3><div class="NLM_p">Compound <b>53b</b> was prepared from compounds <b>26</b> and <b>8b</b> in a similar manner to that described for the synthesis of compound <b>52a</b> and obtained in 74% yield (>99.0% ee, retention time 17.7 min) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.20 (brs, 3H), 3.38–3.71 (m, 2H), 3.94 (brs, 2H), 4.94 (brs, 1H), 5.78 (brs, 1H), 7.19–7.54 (m, 6H), 7.62–7.83 (m, 4H), 7.93 (dd, <i>J</i> = 14.2, 7.8 Hz, 2H), 8.08 (s, 1H). 1H was not observed. MS <i>m</i>/<i>z</i> 588 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>29</sub>H<sub>23</sub>BrClN<sub>5</sub>O<sub>2</sub>·0.25C<sub>6</sub>H<sub>14</sub>O·0.1H<sub>2</sub>O: C, 59.45; H, 4.37; N, 11.36. Found: C, 59.43; H, 4.28; N, 11.34. Purity 98.1% (HPLC). [α]<sub>D</sub><sup>25</sup> = −11.3°.</div><div class="NLM_p last">Analytical HPLC conditions for determination of the % ee values of <b>53a</b> and <b>53b</b>: Waters system, CHIRALPAK AS-3 (Daicel) 250 mm × 4.6 mm i.d., EtOH/hexane = 60/40, flow rate of 0.5 mL/min.</div></div><div id="sec5_41" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> (3-(4-Chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>54</b>)</h3><div class="NLM_p last">To a solution of <b>4a</b> (1.00 g, 5.08 mmol) in DMF (10 mL) were added HATU (2.81 g, 7.39 mmol), DIPEA (1.79 g, 13.9 mmol), and <b>21</b> (874 mg, 4.62 mmol). The resulting mixture was stirred at room temperature for 12 h. The reaction mixture was diluted with EtOAc, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10/1) to give <b>54</b> (1.45 g, 85%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.77–2.17 (m, 1H), 2.62–3.54 (m, 4H), 3.85 (dd, <i>J</i> = 10.6, 2.9 Hz, 1H), 4.56 (brs, 2H), 7.29–7.59 (m, 8H), 7.68 (s, 1H), 8.06 (d, <i>J</i> = 7.2 Hz, 1H). MS <i>m</i>/<i>z</i> 368 (M + H)<sup>+</sup>.</div></div><div id="sec5_42" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> (4-Chlorophenyl)(2-(4-chlorophenyl)-4-((2-phenyl-1,3-oxazol-5-yl)methyl)piperazin-1-yl)methanone (<b>55</b>)</h3><div class="NLM_p">To a solution of <b>54</b> (200 mg, 0.54 mmol) in THF (10 mL) was added borane dimethylsulfide complex (10 M in THF, 54 μL, 0.54 mmol) in one portion at 0 °C under N<sub>2</sub>. After being stirred at 0 °C for 5 min and at room temperature for 16 h, the reaction mixture was cooled to 0 °C and quenched with MeOH (15 mL). The resulting mixture was stirred at 0 °C for 0.5 h and then acidified with 4 M hydrochloric acid to pH 3–4. After stirring at 80 °C for 0.5 h, the reaction mixture was neutralized with saturated aqueous Na<sub>2</sub>CO<sub>3</sub> solution and extracted with CH<sub>2</sub>Cl<sub>2</sub>. The separated organic phase was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10/1) to give 5-((3-(4-chlorophenyl)piperazin-1-yl)methyl)-2-phenyloxazole (60 mg, 31%) as a yellow gum. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 2.23 (t, <i>J</i> = 10.8 Hz, 1H), 2.40 (t, <i>J</i> = 9.8 Hz, 1H), 2.94 (d, <i>J</i> = 10.8 Hz, 2H), 3.04–3.14 (m, 2H), 3.72 (s, 2H), 3.93 (d, <i>J</i> = 9.6 Hz, 1H), 7.04 (s, 1H), 7.27–7.30 (m, 2H), 7.35–7.36 (m, 2H), 7.45 (brs, 3H), 8.04 (d, <i>J</i> = 4.0 Hz, 2H). 1H was not observed. MS <i>m</i>/<i>z</i> 354 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a mixture of 5-((3-(4-chlorophenyl)piperazin-1-yl)methyl)-2-phenyloxazole (45 mg, 0.13 mmol) and 4-chlorobenzoyl chloride (33 mg, 0.19 mmol) in MeCN (2 mL) was added K<sub>2</sub>CO<sub>3</sub> (53 mg, 0.38 mmol) in one portion at room temperature. The resulting mixture was stirred at room temperature for 16 h. The reaction mixture was filtered, and the filtrate was concentrated. The residue was purified by preparative HPLC (condition 3, 0.05% aqueous ammonia solution/MeCN = 43/57 to 13/87) to give <b>55</b> (35 mg, 56%) as a yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.30–2.38 (m, 1H), 2.62 (dd, <i>J</i> = 12.3, 4.2 Hz, 1H), 2.89–2.93 (m, 1H), 3.08–3.17 (m, 1H), 3.45 (d, <i>J</i> = 12.3 Hz, 1H), 3.70–3.84 (m, 3H), 5.46 (s, 1H), 7.14 (s, 1H), 7.32–7.34 (m, 2H), 7.40–7.53 (m, 9H), 7.94–7.97 (m, 2H). MS <i>m</i>/<i>z</i> 492 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub>·0.25H<sub>2</sub>O: C, 65.26; H, 4.77; N, 8.46. Found: C, 65.30; H, 4.82; N, 8.54.</div></div><div id="sec5_43" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> (4-(4-Chlorobenzyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>56</b>)</h3><div class="NLM_p last">To a mixture of <b>54</b> (200 mg, 0.54 mmol) and 4-chlorobenzyl bromide (112 mg, 0.54 mmol) in MeCN (3 mL) was added K<sub>2</sub>CO<sub>3</sub> (225 mg, 1.63 mmol) in one portion at room temperature. The resulting mixture was stirred at 80 °C for 16 h. The reaction mixture was filtered, and the filtrate was concentrated. The residue was purified by preparative TLC (petroleum ether/EtOAc = 5/1) and preparative HPLC (condition 3, 0.05% aqueous ammonia solution/MeCN = 32/68 to 2/98) to give <b>56</b> (52 mg, 19%) as a pale yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 2.25–2.34 (m, 1H), 2.91–2.97 (m, 1H), 3.08 (d, <i>J</i> = 14.1 Hz, 1H), 3.18 (t, <i>J</i> = 11.7 Hz, 1H), 3.31 (t, <i>J</i> = 11.3 Hz, 1H), 3.52 (dd, <i>J</i> = 10.5, 3.0 Hz, 1H), 3.63 (d, <i>J</i> = 13.8 Hz, 1H), 4.32 (t, <i>J</i> = 15.9 Hz, 2H), 7.31 (q, <i>J</i> = 4.6 Hz, 4H), 7.43–7.46 (m, 2H), 7.56–7.58 (m, 5H), 7.76 (s, 1H), 7.98–8.02 (m, 2H). MS <i>m</i>/<i>z</i> 492 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>2</sub>: C, 65.86; H, 4.71; N, 8.53. Found: C, 65.69; H, 4.78; N, 8.60.</div></div><div id="sec5_44" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> (4-(3-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>57</b>)</h3><div class="NLM_p last">To a solution of <b>54</b> (100 mg, 0.27 mmol) and Et<sub>3</sub>N (55 mg, 0.54 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5 mL) was added 3-chlorobenzoyl chloride (95 mg, 0.54 mmol) dropwise at 0 °C. The resulting mixture was stirred at room temperature for 12 h. The reaction mixture was quenched with water and extracted with EtOAc. The separated organic phase was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 45/55 to 15/85) to give <b>57</b> (63 mg, 46%) as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.50 (m, 2H), 3.88–3.98 (m, 2H), 4.02 (d, <i>J</i> = 12.3 Hz, 1H), 4.64 (dd, <i>J</i> = 12.3, 4.2 Hz, 1H), 5.52 (brs, 1H), 7.39 (m, 11H), 7.72 (s, 1H), 7.99 (dd, <i>J</i> = 7.8, 2.7 Hz, 2H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.3H<sub>2</sub>O: C, 63.36; H, 4.25; N, 8.21. Found: C, 63.37; H, 4.23; N, 8.35.</div></div><div id="sec5_45" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> (4-(2-Chlorobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>58</b>)</h3><div class="NLM_p last">Compound <b>58</b> was prepared from compounds <b>54</b> and 2-chlorobenzoyl chloride in a similar manner to that described for the synthesis of compound <b>57</b> and obtained in 41% yield as a pale yellow solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.43 (m, 3H), 3.84 (d, <i>J</i> = 14.4 Hz, 1H), 4.15 (d, <i>J</i> = 14.1 Hz, 1H), 4.67 (d, <i>J</i> = 14.1 Hz, 1H), 5.84 (brs, 1H), 7.39 (m, 11H), 7.73 (s, 1H), 7.98 (dd, <i>J</i> = 7.8, 2.7 Hz, 2H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.3H<sub>2</sub>O: C, 63.36; H, 4.25; N, 8.21. Found: C, 63.16; H, 4.05; N, 8.08.</div></div><div id="sec5_46" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> (4-Benzoyl-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>59</b>)</h3><div class="NLM_p last">Compound <b>59</b> was prepared from compounds <b>54</b> and benzoyl chloride in a similar manner to that described for the synthesis of compound <b>57</b> and obtained in 45% yield as a white solid. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 3.38–3.63 (m, 2H), 3.87 (dd, <i>J</i> = 14.1, 4.5 Hz, 1H), 4.02 (d, <i>J</i> = 12.3 Hz, 1H), 4.17 (d, <i>J</i> = 12.3 Hz, 1H), 4.69 (dd, <i>J</i> = 14.1, 4.5 Hz, 1H), 5.56 (brs, 1H), 7.38 (s, 4H), 7.46 (s, 5H), 7.51–7.63 (m, 3H), 7.71 (s, 1H), 8.00 (dd, <i>J</i> = 7.2, 3.0 Hz, 2H). MS <i>m</i>/<i>z</i> 472 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>22</sub>ClN<sub>3</sub>O<sub>3</sub>·0.3H<sub>2</sub>O: C, 67.94; H, 4.77; N, 8.80. Found: C, 67.99; H, 4.73; N, 8.60.</div></div><div id="sec5_47" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> (4-(4-Bromobenzoyl)-3-(4-chlorophenyl)piperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>60</b>)</h3><div class="NLM_p last">A solution of 4-bromobenzoic acid (48 mg, 0.24 mmol), <b>54</b> (80 mg, 0.22 mmol), DIPEA (0.11 mL, 0.65 mmol), HOBt (50 mg, 0.33 mmol), and EDC (0.06 mL, 0.33 mmol) in THF (3 mL) was stirred at room temperature overnight. The reaction mixture was poured into water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over MgSO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 20/1 to 3/7) to give <b>60</b> (115 mg, 96%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 3.11–3.77 (m, 3H), 4.42 (d, <i>J</i> = 10.8 Hz, 1H), 4.85–5.23 (m, 1H), 5.25–6.15 (m, 1H), 7.28–7.41 (m, 6H), 7.43–7.55 (m, 3H), 7.59 (d, <i>J</i> = 8.3 Hz, 2H), 8.00 (dd, <i>J</i> = 7.8, 1.6 Hz, 1H), 8.00 (dd, <i>J</i> = 7.8, 1.6 Hz, 2H). 1H was not observed. MS <i>m</i>/<i>z</i> 550 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>BrClN<sub>3</sub>O<sub>3</sub>: C, 58.87; H, 3.84; N, 7.63. Found: C, 58.73; H, 3.99; N, 7.73.</div></div><div id="sec5_48" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> 4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)-1-((2-phenyl-1,3-oxazol-5-yl)methyl)piperazin-2-one (<b>63</b>)</h3><div class="NLM_p last">To a solution of <b>61</b> (2.00 g, 6.65 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) was added a solution of 4-chlorobenzoyl chloride (1.28 g, 7.31 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) at 0 °C. The mixture was stirred at room temperature for 16 h. To the reaction mixture was added saturated aqueous NaHCO<sub>3</sub>. The mixture was concentrated and extracted with EtOAc. The separated organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by preparative TLC (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 20/1) to give 4-(4-chlorobenzoyl)-3-(4-chlorophenyl)piperazin-2-one (2.00 g) as a crude product. To a solution of the crude (240 mg) in DMF (5 mL) at 0 °C was added NaH (60% in oil, 27 mg, 0.68 mmol). The mixture was stirred at 0 °C for 0.5 h. Then to the mixture was added a solution of <b>62</b> (130 mg, 0.67 mmol) in DMF (5 mL) at 0 °C. After stirring at 0 °C for 0.5 h, the reaction mixture was poured into water and extracted with EtOAc. The separated organic layer was dried over Na<sub>2</sub>SO<sub>4</sub> and concentrated. The residue was purified by preparative HPLC (condition 3, 0.05% aqueous ammonia solution/MeCN = 53/47 to 23/77) to give <b>63</b> (49 mg, 12% for 2 steps) as a white solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.31–3.54 (m, 2H), 3.56–3.78 (m, 1H), 4.57 (d, <i>J</i> = 15.6 Hz, 1H), 4.84–5.20 (m, 1H), 5.51 (brs, 1H), 6.46 (brs, 1H), 7.16 (s, 1H), 7.30–7.55 (m, 11H), 8.01 (brs, 2H). MS <i>m</i>/<i>z</i> 506 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.25H<sub>2</sub>O: C, 63.48; H, 4.24; N, 8.22. Found: C, 63.58; H, 4.22; N, 8.26.</div></div><div id="sec5_49" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> (4-(4-Chlorobenzoyl)-3-cyclohexylpiperazin-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>65</b>)</h3><div class="NLM_p">To a solution of 2-cyclohexylpiperazine <b>64</b> (250 mg, 1.49 mmol) in THF (10 mL) were added Et<sub>3</sub>N (0.62 mL, 4.46 mmol) and Boc<sub>2</sub>O (0.34 mL, 1.48 mmol) at 0 °C. After stirring at 0 °C to room temperature overnight, the reaction mixture was poured into water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 95/5 to 20/80) to give <i>tert</i>-butyl 3-cyclohexylpiperazine-1-carboxylate (290 mg, 73%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 0.88–1.11 (m, 2H), 1.11–1.35 (m, 4H), 1.46 (s, 9H), 1.62–1.88 (m, 5H), 2.27–2.38 (m, 1H), 2.41–2.62 (m, 1H), 2.62–2.89 (m, 2H), 2.97 (d, <i>J</i> = 11.4 Hz, 1H), 3.80–4.30 (m, 2H). 1H was not observed. MS <i>m</i>/<i>z</i> 269 (M + H)<sup>+</sup>.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl 3-cyclohexylpiperazine-1-carboxylate (100 mg, 0.37 mmol) in THF (3 mL) were added DIPEA (0.20 mL, 1.14 mmol) and 4-chlorobenzoyl chloride (0.071 mL, 0.559 mmol) at 0 °C. After stirring at 0 °C for 1 h, the reaction mixture was poured into water at room temperature and extracted with EtOAc. The organic layer was separated, washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The residue was purified by NH-silica gel column chromatography (hexane/EtOAc = 95/5 to 20/80) to give <i>tert</i>-butyl 4-(4-chlorobenzoyl)-3-cyclohexylpiperazine-1-carboxylate (150 mg, 99%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 0.94–1.33 (m, 5H), 1.47 (s, 9H), 1.60–2.10 (m, 6H), 2.56–2.98 (m, 2H), 3.05–3.56 (m, 2H), 3.98–4.55 (m, 3H), 7.30 (d, <i>J</i> = 8.1 Hz, 2H), 7.39 (d, <i>J</i> = 8.1 Hz, 2H). MS <i>m</i>/<i>z</i> 351 (M – (<i>t</i>-Bu) + H)<sup>+</sup>.</div><div class="NLM_p">To a solution of <i>tert</i>-butyl 4-(4-chlorobenzoyl)-3-cyclohexylpiperazine-1-carboxylate (260 mg, 0.64 mmol) in EtOAc (10 mL) was added 4 M hydrogen chloride EtOAc solution (3.2 mL, 12.8 mmol) at room temperature. After stirring at room temperature for 5 h, the reaction mixture was concentrated to give (4-chlorophenyl)-(2-cyclohexylpiperazin-1-yl)methanone hydrochloride (200 mg, 91%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 0.75–1.37 (m, 5H), 1.45–1.86 (m, 5H), 2.03–2.20 (m, 1H), 2.85–3.29 (m, 4H), 3.37–3.66 (m, 2H), 4.27–4.63 (m, 1H), 7.46 (d, <i>J</i> = 8.1 Hz, 2H), 7.53 (d, <i>J</i> = 8.1 Hz, 2H), 9.29 (brs, 2H). MS <i>m</i>/<i>z</i> 307 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a solution of (4-chlorophenyl)-(2-cyclohexylpiperazin-1-yl)methanone hydrochloride (60 mg, 0.17 mmol), <b>21</b> (40 mg, 0.21 mmol), HOBt (28 mg, 0.21 mmol), and DIPEA (113 mg, 0.87 mmol) in DMF (3 mL) was added EDC·HCl (40 mg, 0.21 mmol) at 0 °C. After stirring at 0 °C to room temperature for 12 h, the reaction mixture was poured into water at room temperature and extracted with EtOAc. The combined organic layer was concentrated, and the residue was purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 40/60 to 10/90) to give <b>65</b> (20 mg, 24%) as a light gray powder. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 0.52–0.99 (m, 1H), 1.26–1.36 (m, 4H), 1.61–1.78 (m, 6H), 3.04 (s, 1H), 3.31–3.54 (m, 2H), 3.66 (d, <i>J</i> = 10.8 Hz, 1H), 4.48–4.66 (m, 3H), 7.35–7.37 (m, 2H), 7.43–7.45 (m, 2H), 7.51–7.54 (m, 3H), 7.75 (s, 1H), 8.07 (d, <i>J</i> = 5.6 Hz, 2H). MS <i>m</i>/<i>z</i> 478 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>27</sub>H<sub>28</sub>ClN<sub>3</sub>O<sub>3</sub>·0.75H<sub>2</sub>O: C, 65.98; H, 6.05; N, 8.55. Found: C, 65.84; H, 5.81; N, 8.41.</div></div><div id="sec5_50" class="NLM_sec NLM_sec_level_2"><div id="ac_i68" class="anchor-spacer"></div><h3 class="article-section__title" id="_i68"> <i>tert</i>-Butyl 3-(4-Chlorophenyl)-1,4-diazepane-1-carboxylate (<b>67</b>)</h3><div class="NLM_p">To a mixture of propane-1,3-diamine (2.25 g, 30.4 mmol) in MeOH (50 mL) was added <b>66</b> (4.00 g, 15.2 mmol) portionwise at 0 °C. The mixture was stirred at room temperature for 1 h. Then to the mixture was added NaOMe (2.05 g, 38.0 mmol). After stirring at 80 °C for 16 h, the reaction mixture was concentrated. The residue was purified by silica gel column chromatography (CH<sub>2</sub>Cl<sub>2</sub>/MeOH = 10/1) to give 3-(4-chlorophenyl)-1,4-diazepan-2-one (800 mg, 23%) as a light yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.72–1.84 (m, 3H), 3.02–3.09 (m, 1H), 3.33–3.47 (m, 3H), 4.57 (s, 1H), 6.08 (s, 1H), 7.31–7.37 (m, 4H). MS <i>m</i>/<i>z</i> 225 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a mixture of 3-(4-chlorophenyl)-1,4-diazepan-2-one (800 mg, 3.56 mmol) in THF (30 mL) at 0 °C was added borane dimethylsulfide complex (10 M, 5.00 mL, 50.0 mmol) dropwise. The mixture was stirred at 80 °C for 16 h. To the mixture was added MeOH (20 mL) dropwise, and the mixture was stirred at room temperature for 1 h. Then the mixture was poured into 4 M hydrochloric acid (30 mL). After stirring at 80 °C for 1 h, the mixture was concentrated. To a solution of the residue (752 mg) and Et<sub>3</sub>N (1.08 g, 10.7 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) was added Boc<sub>2</sub>O (545 mg, 2.5 mmol) dropwise at 0 °C. After stirring at room temperature for 16 h, the reaction mixture was concentrated. The residue was purified by silica gel column chromatography on silica gel (petroleum ether/ethyl acetate = 10/1) to give <b>67</b> (800 mg, 72% for 2 steps) as a light yellow solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.51 (s, 9H), 1.77–2.16 (m, 3H), 2.67–2.98 (m, 2H), 3.13–3.33 (m, 2H), 3.80–3.92 (m, 2H), 7.29–7.40 (m, 4H). 1H was not observed. MS <i>m</i>/<i>z</i> 311 (M + H)<sup>+</sup>.</div></div><div id="sec5_51" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> (4-(4-Chlorobenzoyl)-3-(4-chlorophenyl)-1,4-diazepan-1-yl)(2-phenyl-1,3-oxazol-5-yl)methanone (<b>68</b>)</h3><div class="NLM_p">To a mixture of <b>67</b> (351 mg, 1.13 mmol) and Et<sub>3</sub>N (342 mg, 3.38 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) was added 4-chlorobenzoyl chloride (218 mg, 1.24 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL) dropwise at room temperature. After stirring at room temperature for 3 h, the reaction mixture was concentrated, and the residue was purified by silica gel column chromatography (petroleum ether/ethyl acetate = 1/1) to give <i>tert</i>-butyl 4-(4-chlorobenzoyl)-3-(4-chlorophenyl)-1,4-diazepane-1-carboxylate (472 mg, 93%) as a light yellow solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 1.29 (s, 3H), 1.48 (s, 6H), 1.60–1.69 (m, 1H), 2.07–2.11 (m, 1H), 2.66–3.07 (m, 3H), 4.04–4.10 (m, 1H), 4.26–4.31 (m, 1H), 4.59–4.65 (m, 1H), 4.90–5.23 (m, 1H), 6.82–7.07 (m, 4H), 7.08–7.18 (m, 2H) 7.24–7.33 (m, 2H). MS <i>m</i>/<i>z</i> 449 (M + H)<sup>+</sup>.</div><div class="NLM_p last">To a mixture of <i>tert</i>-butyl 4-(4-chlorobenzoyl)-3-(4-chlorophenyl)-1,4-diazepane-1-carboxylate (520 mg, 1.16 mmol) in 1,4-dioxane (10 mL) was added 4 M hydrogen chloride in dioxane (10.0 mL, 40.0 mmol). After stirring at room temperature for 5 h, the reaction mixture was concentrated to give (4-chlorophenyl)-[2-(4-chlorophenyl)-1,4-diazepan-1-yl]methanone hydrochloride (320 mg) as a crude product. To a mixture of the crude (77 mg), <b>21</b> (38 mg, 0.20 mmol), and Et<sub>3</sub>N (61 mg, 0.60 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL) was added HATU (84 mg, 0.22 mmol) at room temperature. After stirring at room temperature for 16 h, the reaction mixture was concentrated and purified by preparative HPLC (condition 1, 0.225% aqueous formic acid solution/MeCN = 58/42 to 12/88) to give <b>68</b> (36 mg, 34% for 2 steps) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, <i>T</i> = 80 °C) δ 1.63–2.25 (m, 2H), 3.17–3.93 (m, 3H), 4.03–4.90 (m, 3H), 4.91–6.12 (m, 1H), 6.71–7.72 (m, 11H), 7.76–8.42 (m, 3H). MS <i>m</i>/<i>z</i> 520 (M + H)<sup>+</sup>. Anal. Calcd for C<sub>28</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>3</sub>O<sub>3</sub>·0.75H<sub>2</sub>O: C, 62.99; H, 4.63; N, 7.87. Found: C, 62.98; H, 4.52; N, 7.93.</div></div><div id="sec5_52" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Preparation of Enzymes</h3><div class="NLM_p last">The human recombinant proteins, full-length eIF4A3, MLN51 (residues 137–283), full-length eIF4A1, full-length eIF4B, and eIF4G (residues 712–1451) were expressed in <i>Escherichia coli</i> BL21(DE3) as fusion proteins with 6 × histidine (His)-small ubiquitin-like modifier (SUMO) or His tags followed by a tobacco etch virus (TEV) protease cleavage site at the N-terminus and purification by Ni-NTA superflow affinity column (QIAGEN) and Superdex 200 gel-filtration column (GE Healthcare). The His-SUMO or His tags were cleaved with SUMO protease or TEV protease. Human recombinant full-length DHX29 and full-length BRR2 were expressed in Sf-9 insect cells as fusion proteins with FLAG-His or His tags at the N terminus, respectively, using the BaculoDirect C-term Baculovirus expression system (Thermo Fisher Scientific). FLAG-His-DHX29 and His-BRR2 were purified by Ni-NTA superflow affinity column and Superdex 200 gel-filtration column. Protein concentrations were determined using a BCA protein assay kit (Thermo Fisher Scientific) with bovine serum albumin as a standard. Recombinant human His-eIF4A2 protein was purchased from GeneCopoeia.</div></div><div id="sec5_53" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> RNA-Dependent ATPase Assay</h3><div class="NLM_p last">The RNA-dependent ATPase assay was performed using the ADP-Glo assay system (Promega). Single-stranded RNA poly(U) was purchased from MP Biomedicals. The assay buffer comprised 20 mM Tris-HCl (pH7.5), 2.5 mM MgCl<sub>2</sub>, 100 mM KCl, 1 mM dithiothreitol (DTT), and 0.01% (v/v) Tween20. To enhance ATPase activity for eIF4A, the equivalent molar concentration of MLN51 for 150 nM eIF4A3 or eIF4B and eIF4G for 100 nM eIF4A1 or eIF4A2 were added. Regarding the ATPase assays for DHX29 or BRR2, the optimal concentrations were 6.4 nM and 6.25 nM, respectively. Concentrations of ATP or RNA were set at the <i>K</i><sub>m</sub> value of each substrate for each enzyme as follows: 35 μM ATP and 1.5 μg/mL poly(U) for eIF4A1 and eIF4A3; 20 μM ATP and 3.0 μg/mL poly(U) for eIF4A2; 30 μM ATP and 1.8 μg/mL poly(U) for DHX29; and 20 μM ATP and 2.5 μg/mL poly(U) for BRR2. After the addition of the substrates and test compounds, the ATPase reactions were started by the addition of the enzymes. They were incubated at room temperature for 30 min for eIF4A3, DHX29, and BRR2 or 40 min for eIF4A1 and eIF4A2. The enzymatic reactions were terminated by ADP-Glo reagent, and then ADP-Glo detection reagent was added to detect ADP. Luminescent signals were measured using an EnVision 2102 multilabel plate reader (PerkinElmer). We defined the luminescent signals of the reaction without enzyme as 100% inhibitory activity and those of the complete reaction mixture as 0% inhibitory activity. Curve fittings and calculations of IC<sub>50</sub> values were performed using the program XLfit version 5 (ID Business Solutions) with the maximum and minimum of the curve constrained to 100 and 0, respectively.</div></div><div id="sec5_54" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Reporter Plasmids</h3><div class="NLM_p last">The reporter gene plasmid was constructed as described in the literature.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Briefly, the pGL4.12 SV40-Luc2CP-BGG vector was amplified by polymerase chain reaction (PCR) from pGL4.13 (Promega), and the β-globin genomic sequence from Exon II to the poly(A) signal was amplified by PCR from a human genomic library (Clontech Laboratories). Both PCR products were inserted into pGL4.12 (Promega). A pRL-TK vector as an internal control was purchased from Promega.</div></div><div id="sec5_55" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> NMD Reporter Assay</h3><div class="NLM_p last">HEK293T cells were transfected with the pGL4.12 SV40-Luc2CP-BGG vector and a pRL-TK vector as an internal control using Lipofectamine LTX (Thermo Fisher Scientific). The compounds were treated for 6 h after transfection of the two vectors for 24 h. Relative firefly to renilla luciferase activity was determined using the dual luciferase kit (Promega). Luminescent signals were detected by EnVision 2102 multilabel plate reader (PerkinElmer).</div></div><div id="sec5_56" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Surface Plasmon Resonance (SPR) Biosensing Assay</h3><div class="NLM_p last">SPR biosensing experiments were performed on BiacoreS200 and BiacoreT100 instruments equipped with CM7 sensor chips (GE Healthcare). Immobilization of eIF4A3 onto the CM7 sensor surface was performed following the standard amine coupling procedure according to the manufacturer’s instructions. HBS-P+ (10 mM Tris-HCl, pH 7.4, 150 mM NaCl, 0.05% Surfactant P20) supplemented with 1 mM DTT was used as the running buffer for immobilization. Typical immobilization levels ranged from 5000 to 10 000 RU. All binding experiments were performed in 20 mM Tris-HCl, pH 8.0, 150 mM NaCl, 0.2 mM tris(2-carboxyethyl)phosphine (TCEP), 0.02% Surfactant P20 (v/v), and 1–2% DMSO (v/v) at 25 °C. Compound solutions at several concentrations (6–12 points) were injected for 60–120 s at a flow rate of 50 μL/min, and the dissociation was followed for up to 120–1200 s depending on the compound’s off-rate. Competition experiments were performed in the presence of blocking compound <b>52a</b> at a saturated concentration, diluted in the running buffer.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Data processing and analyzing were performed using BiacoreS200 and BiacoreT100 evaluation software (GE Healthcare). Sensorgrams were double-referenced prior to global fitting the concentration series to a 1:1 binding model. Dissociative half-lives <i>t</i><sub>1/2</sub> were calculated from the dissociation rate constants <i>k</i><sub>off</sub> (<i>t</i><sub>1/2</sub> = ln 2/<i>k</i><sub>off</sub>).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i75"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01904">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38003" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38003" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01904" class="ext-link">10.1021/acs.jmedchem.6b01904</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings and some data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">SPR sensorgrams; X-ray crystal structure analysis of <b>8a</b>; synthetic procedure for intermediates <b>17</b>, <b>23</b>, <b>26</b>–<b>28</b>, and <b>62</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_001.csv">jm6b01904_si_001.csv (3.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf">jm6b01904_si_002.pdf (530.5 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01904" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01776" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01776" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masahiro Ito</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5965-535X" title="Orcid link">http://orcid.org/0000-0001-5965-535X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#05686476646d6c776a2b6c716a4571646e6061642b666a68"><span class="__cf_email__" data-cfemail="dfb2beacbeb7b6adb0f1b6abb09fabbeb4babbbef1bcb0b2">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasuhiro Imaeda</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0e776f7d7b66677c612067636f6b6a6f4e7a6f656b6a6f206d6163"><span class="__cf_email__" data-cfemail="770e1604021f1e0518591e1a161213163703161c1213165914181a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Toshio Tanaka</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas R. Cary</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3940-6078" title="Orcid link">http://orcid.org/0000-0003-3940-6078</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Misa Iwatani-Yoshihara</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yusuke Kamada</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3701-2075" title="Orcid link">http://orcid.org/0000-0003-3701-2075</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tomohiro Kawamoto</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samuel Aparicio</span> - <span class="hlFld-Affiliation affiliation">Department
of Molecular Oncology, BC Cancer Agency, 675 W. 10th Avenue, Vancouver, BC V5Z 1L3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Atsushi Nakanishi</span> - <span class="hlFld-Affiliation affiliation">Pharmaceutical
Research Division, Takeda Pharmaceutical
Company Limited, 26-1,
Muraoka-Higashi 2-Chome, Fujisawa, Kanagawa 251-8555, Japan</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6104-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33425" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33425" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Yasumi Kumagai and Motoo Iida for their assistance with NMR spectroscopic experiments and analysis, Mitsuyoshi Nishitani for acquiring and determining X-ray crystal structures, and Yumi Imai and Haruyuki Nishida for supervising of structure determination. We also thank Natsumi Fujii, Chie Kushibe, and Katsuhiko Miwa for HPLC purification and analysis of compounds, Ryosuke Arai, Kouichi Iwanaga, and Izumi Nomura for conducting high-throughput synthesis. We thank Jyun-ichi Sakamoto for support of SPR analysis, Takashi Ito for preparation of proteins, and Shinji Moriyama, Shin-ichi Matsumoto, Shigeru Kondo, and Akihiko Naotsuka for conducting high-throughput screening. We express our appreciation to Nao Morishita, Satoshi Endo, Daisuke Nakata, Shinsuke Araki, Masato Yugami, Ryo Mizojiri, and Hironobu Maezaki for helpful discussions and advice. We thank Ryosuke Tokunoh, Tomohiko Suzaki, Yuuko Hitomi, Atsuo Baba, Hiroshi Nara, and Nobuo Cho for their cooperation in helping us execute this work. We thank Hiromichi Kimura, Daisuke Morishita, and Satoshi Kitazawa for the pioneering spirit they demonstrated in initiating the eIF4A3 project.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i77" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i77"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i78" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i78"> Abbreviations Used</h2><tr><td class="NLM_term">Boc<sub>2</sub>O</td><td class="NLM_def"><p class="first last">di-<i>tert</i>-butyl dicarbonate</p></td></tr><tr><td class="NLM_term">BRR2</td><td class="NLM_def"><p class="first last">bad response to refrigeration 2 (an ATP-dependent RNA helicase)</p></td></tr><tr><td class="NLM_term">CDCl<sub>3</sub></td><td class="NLM_def"><p class="first last">deuterated chloroform</p></td></tr><tr><td class="NLM_term">DBU</td><td class="NLM_def"><p class="first last">1,8-diazabicyclo[5.4.0]undec-7-ene</p></td></tr><tr><td class="NLM_term">DHX29</td><td class="NLM_def"><p class="first last">DEAH-box helicase 29 (an ATP-dependent RNA helicase)</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>′-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DME</td><td class="NLM_def"><p class="first last">1,2-dimethoxyethane</p></td></tr><tr><td class="NLM_term">DMSO-<i>d</i><sub>6</sub></td><td class="NLM_def"><p class="first last">dimethyl sulfoxide-<i>d</i><sub>6</sub></p></td></tr><tr><td class="NLM_term">EDC</td><td class="NLM_def"><p class="first last">1-(3-(dimethylamino)propyl)-3-ethylcarbodiimide</p></td></tr><tr><td class="NLM_term">EDC·HCl</td><td class="NLM_def"><p class="first last">1-(3-(dimethylamino)propyl)-3-ethylcarbodiimide hydrochloride</p></td></tr><tr><td class="NLM_term">Et<sub>2</sub>NH</td><td class="NLM_def"><p class="first last">diethylamine</p></td></tr><tr><td class="NLM_term">Et<sub>3</sub>N</td><td class="NLM_def"><p class="first last">triethylamine</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last"><i>O</i>-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">HOBt</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">Me<sub>3</sub>Al</td><td class="NLM_def"><p class="first last">trimethylaluminum</p></td></tr><tr><td class="NLM_term">(Me<sub>3</sub>Al)<sub>2</sub>·DABCO</td><td class="NLM_def"><p class="first last">bis(trimethylaluminum) 1,4-diazabicyclo[2.2.2]octane adduct</p></td></tr><tr><td class="NLM_term">MeCN</td><td class="NLM_def"><p class="first last">acetonitrile</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MTBE</td><td class="NLM_def"><p class="first last">methyl <i>tert</i>-butyl ether</p></td></tr><tr><td class="NLM_term">NaOMe</td><td class="NLM_def"><p class="first last">sodium methoxide</p></td></tr><tr><td class="NLM_term">Pd(OAc)<sub>2</sub></td><td class="NLM_def"><p class="first last">palladium(II) acetate</p></td></tr><tr><td class="NLM_term">Pd(PPh<sub>3</sub>)<sub>4</sub></td><td class="NLM_def"><p class="first last">tetrakis(triphenylphosphine)palladium(0)</p></td></tr><tr><td class="NLM_term"><i>i</i>-PrOH</td><td class="NLM_def"><p class="first last">2-propanol</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationships</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr><tr><td class="NLM_term">THF</td><td class="NLM_def"><p class="first last">tetrahydrofuran</p></td></tr><tr><td class="NLM_term">TsOH</td><td class="NLM_def"><p class="first last"><i>p</i>-toluenesulfonic acid</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14533" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14533" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 39 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Linder, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jankowsky, E.</span><span> </span><span class="NLM_article-title">From unwinding to clamping — the DEAD box RNA helicase family</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">516</span><span class="refDoi"> DOI: 10.1038/nrm3154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnrm3154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=21779027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpt1aqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=505-516&author=P.+Linderauthor=E.+Jankowsky&title=From+unwinding+to+clamping+%E2%80%94+the+DEAD+box+RNA+helicase+family&doi=10.1038%2Fnrm3154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">From unwinding to clamping - the DEAD box RNA helicase family</span></div><div class="casAuthors">Linder, Patrick; Jankowsky, Eckhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">505-516</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  RNA helicases of the DEAD box family are present in all eukaryotic cells and in many bacteria and archaea.  These highly conserved enzymes are required for RNA metab. from transcription to degrdn. and are therefore important players in gene expression.  DEAD box proteins use ATP to unwind short duplex RNA in an unusual fashion and remodel RNA-protein complexes, but they can also function as ATP-dependent RNA clamps to provide nucleation centers that establish larger RNA-protein complexes.  Structural, mechanistic, and mol. biol. studies have started to reveal how these conserved proteins can perform such diverse functions and how accessory proteins have a central role in their regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7W4tiGRbeKLVg90H21EOLACvtfcHk0ljM1AfE7u4tRQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpt1aqtL4%253D&md5=66620c984f8c51c587f9988e774c1f21</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm3154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3154%26sid%3Dliteratum%253Aachs%26aulast%3DLinder%26aufirst%3DP.%26aulast%3DJankowsky%26aufirst%3DE.%26atitle%3DFrom%2520unwinding%2520to%2520clamping%2520%25E2%2580%2594%2520the%2520DEAD%2520box%2520RNA%2520helicase%2520family%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2011%26volume%3D12%26spage%3D505%26epage%3D516%26doi%3D10.1038%2Fnrm3154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Tanner, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linder, P.</span><span> </span><span class="NLM_article-title">DExD/H box RNA helicases: from generic motors to specific dissociation functions</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">251</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1016/S1097-2765(01)00329-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2FS1097-2765%2801%2900329-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=11545728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=251-262&author=N.+K.+Tannerauthor=P.+Linder&title=DExD%2FH+box+RNA+helicases%3A+from+generic+motors+to+specific+dissociation+functions&doi=10.1016%2FS1097-2765%2801%2900329-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">DExD/H box RNA helicases: from generic motors to specific dissociation functions</span></div><div class="casAuthors">Tanner, N. Kyle; Linder, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">251-262</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with refs.  RNA helicases of the DEAD box and related DExD/H proteins form a very large superfamily of proteins conserved from bacteria and viruses to humans.  They have seven to eight conserved motifs, the characteristics of which are used to subgroup members into individual families.  They are assocd. with all processes involving RNA mols., including transcription, editing, splicing, ribosome biogenesis, RNA export, translation, RNA turnover, and organelle gene expression.  Anal. of the three-dimensional structures obtained through the crystn. of viral and cellular RNA helicases reveals a strong structural homol. to DNA helicases.  In this review, we discuss our current understanding of RNA helicases and their biol. function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFLDauWWZ4c7Vg90H21EOLACvtfcHk0lhDfRDEISw62A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbw%253D&md5=90a403e4bd3c295f5198dd1bcc601ab5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900329-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900329-X%26sid%3Dliteratum%253Aachs%26aulast%3DTanner%26aufirst%3DN.%2BK.%26aulast%3DLinder%26aufirst%3DP.%26atitle%3DDExD%252FH%2520box%2520RNA%2520helicases%253A%2520from%2520generic%2520motors%2520to%2520specific%2520dissociation%2520functions%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D8%26spage%3D251%26epage%3D262%26doi%3D10.1016%2FS1097-2765%2801%2900329-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Jankowsky, E.</span><span> </span><span class="NLM_article-title">RNA Helicases at work: binding and rearranging</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.tibs.2010.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.tibs.2010.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=20813532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=19-29&author=E.+Jankowsky&title=RNA+Helicases+at+work%3A+binding+and+rearranging&doi=10.1016%2Fj.tibs.2010.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">RNA helicases at work: binding and rearranging</span></div><div class="casAuthors">Jankowsky, Eckhard</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-29</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RNA helicases are ubiquitous, highly conserved enzymes that participate in nearly all aspects of RNA metab.  These proteins bind or remodel RNA or RNA-protein complexes in an ATP-dependent fashion.  How RNA helicases phys. perform their cellular tasks has been a longstanding question, but in recent years, intriguing models have started to link structure, mechanism, and biol. function for some RNA helicases.  Here, the author outlines the current view on major structural and mechanistic themes of RNA helicase function, and on emerging phys. models for the cellular roles of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-XYT4X_o2lLVg90H21EOLACvtfcHk0lhDfRDEISw62A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKgtg%253D%253D&md5=e7ed17c0d999efea7833a1548e0f80fa</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DJankowsky%26aufirst%3DE.%26atitle%3DRNA%2520Helicases%2520at%2520work%253A%2520binding%2520and%2520rearranging%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D19%26epage%3D29%26doi%3D10.1016%2Fj.tibs.2010.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Jarmoskaite, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R.</span><span> </span><span class="NLM_article-title">RNA helicase proteins as chaperones and remodelers</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1146/annurev-biochem-060713-035546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1146%2Fannurev-biochem-060713-035546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=24635478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOhtrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=697-725&author=I.+Jarmoskaiteauthor=R.+Russell&title=RNA+helicase+proteins+as+chaperones+and+remodelers&doi=10.1146%2Fannurev-biochem-060713-035546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">RNA helicase proteins as chaperones and remodelers</span></div><div class="casAuthors">Jarmoskaite, Inga; Russell, Rick</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">697-725</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Superfamily 2 helicase proteins are ubiquitous in RNA biol. and have an extraordinarily broad set of functional roles.  Central among these roles are the promotion of rearrangements of structured RNAs and the remodeling of ribonucleoprotein complexes (RNPs), allowing formation of native RNA structure or progression through a functional cycle of structures.  Although all superfamily 2 helicases share a conserved helicase core, they are divided evolutionarily into several families, and it is principally proteins from three families, the DEAD-box, DEAH/RHA, and Ski2-like families, that function to manipulate structured RNAs and RNPs.  Strikingly, there are emerging differences in the mechanisms of these proteins, both between families and within the largest family (DEAD-box), and these differences appear to be tuned to their RNA or RNP substrates and their specific roles.  This review outlines basic mechanistic features of the three families and surveys individual proteins and the current understanding of their biol. substrates and mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0VcunkVppbbVg90H21EOLACvtfcHk0lhDfRDEISw62A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOhtrnL&md5=6031909d6b1762b3d7955d5ce0af7b58</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060713-035546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060713-035546%26sid%3Dliteratum%253Aachs%26aulast%3DJarmoskaite%26aufirst%3DI.%26aulast%3DRussell%26aufirst%3DR.%26atitle%3DRNA%2520helicase%2520proteins%2520as%2520chaperones%2520and%2520remodelers%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2014%26volume%3D83%26spage%3D697%26epage%3D725%26doi%3D10.1146%2Fannurev-biochem-060713-035546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Rocak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linder, P.</span><span> </span><span class="NLM_article-title">DEAD-box proteins: the driving forces behind RNA metabolism</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">232</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1038/nrm1335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnrm1335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=14991003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1CltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=232-241&author=S.+Rocakauthor=P.+Linder&title=DEAD-box+proteins%3A+the+driving+forces+behind+RNA+metabolism&doi=10.1038%2Fnrm1335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">DEAD-box proteins: the driving forces behind RNA metabolism</span></div><div class="casAuthors">Rocak, Sanda; Linder, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">232-241</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  RNA helicases from the DEAD-box family are found in almost all organisms and play important roles in RNA metab.  They are assocd. with many processes ranging from RNA synthesis to RNA degrdn.  DEAD-box proteins use the energy from ATP hydrolysis to rearrange intermol. or intramol. RNA structures, or dissoc. RNA-protein complexes.  Such dynamic rearrangements are fundamental for many, if not all, steps in the life of an RNA mol.  Recent biochem., genetic, and structural data shed light on how these proteins power the metab. of RNA within a cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxJcMxFGudxrVg90H21EOLACvtfcHk0li-HOBCY64KWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1CltL8%253D&md5=85203c8f3ef20f4896086036382bce61</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrm1335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1335%26sid%3Dliteratum%253Aachs%26aulast%3DRocak%26aufirst%3DS.%26aulast%3DLinder%26aufirst%3DP.%26atitle%3DDEAD-box%2520proteins%253A%2520the%2520driving%2520forces%2520behind%2520RNA%2520metabolism%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2004%26volume%3D5%26spage%3D232%26epage%3D241%26doi%3D10.1038%2Fnrm1335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Saulière, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murigneux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquenet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Tonquèze, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audic, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paillard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crollius, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Hir, H.</span><span> </span><span class="NLM_article-title">CLIP-seq of eIF4AIII reveals transcriptome-wide mapping of the human exon junction complex</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1131</span><span class="refDoi"> DOI: 10.1038/nsmb.2420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnsmb.2420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=23085716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFeqsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1124-1131&author=J.+Sauli%C3%A8reauthor=V.+Murigneuxauthor=Z.+Wangauthor=E.+Marquenetauthor=I.+Barbosaauthor=O.+Le+Tonqu%C3%A8zeauthor=Y.+Audicauthor=L.+Paillardauthor=H.+R.+Crolliusauthor=H.+Le+Hir&title=CLIP-seq+of+eIF4AIII+reveals+transcriptome-wide+mapping+of+the+human+exon+junction+complex&doi=10.1038%2Fnsmb.2420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">CLIP-seq of eIF4AIII reveals transcriptome-wide mapping of the human exon junction complex</span></div><div class="casAuthors">Sauliere, Jerome; Murigneux, Valentine; Wang, Zhen; Marquenet, Emelie; Barbosa, Isabelle; Le Tonqueze, Olivier; Audic, Yann; Paillard, Luc; Crollius, Hugues Roest; Le Hir, Herve</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1124-1131</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The exon junction complex (EJC) is a central effector of the fate of mRNAs, linking nuclear processing to mRNA transport, translation and surveillance.  However, little is known about its transcriptome-wide targets.  We used crosslinking and immunopptn. methods coupled to high-throughput sequencing (CLIP-seq) in human cells to identify the binding sites of the DEAD-box helicase eIF4AIII, an EJC core component.  CLIP reads form peaks that are located mainly in spliced mRNAs.  Most expressed exons harbor peaks either in the canonical EJC region, located ∼24 nucleotides upstream of exonic junctions, or in other noncanonical regions.  Notably, both of these types of peaks are preferentially assocd. with unstructured and purine-rich sequences contg. the motif GAAGA, which is a potential binding site for EJC-assocd. factors.  Therefore, EJC positions vary spatially and quant. between exons.  This transcriptome-wide mapping of human eIF4AIII reveals unanticipated aspects of the EJC and broadens its potential impact on post-transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmnvsbuadcFbVg90H21EOLACvtfcHk0li-HOBCY64KWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFeqsrjE&md5=6483cf0baffbe1af6f26c339a22f9ba9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2420%26sid%3Dliteratum%253Aachs%26aulast%3DSauli%25C3%25A8re%26aufirst%3DJ.%26aulast%3DMurigneux%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMarquenet%26aufirst%3DE.%26aulast%3DBarbosa%26aufirst%3DI.%26aulast%3DLe%2BTonqu%25C3%25A8ze%26aufirst%3DO.%26aulast%3DAudic%26aufirst%3DY.%26aulast%3DPaillard%26aufirst%3DL.%26aulast%3DCrollius%26aufirst%3DH.%2BR.%26aulast%3DLe%2BHir%26aufirst%3DH.%26atitle%3DCLIP-seq%2520of%2520eIF4AIII%2520reveals%2520transcriptome-wide%2520mapping%2520of%2520the%2520human%2520exon%2520junction%2520complex%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1124%26epage%3D1131%26doi%3D10.1038%2Fnsmb.2420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Le Hir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, G. R.</span><span> </span><span class="NLM_article-title">Structural insights into the exon junction complex</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1016/j.sbi.2007.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.sbi.2007.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=18164611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVGitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=112-119&author=H.+Le+Hirauthor=G.+R.+Andersen&title=Structural+insights+into+the+exon+junction+complex&doi=10.1016%2Fj.sbi.2007.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the exon junction complex</span></div><div class="casAuthors">Le Hir, Herve; Andersen, Gregers Rom</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-119</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In higher eukaryotes, the exon junction complex is loaded onto spliced mRNAs at a precise position upstream of exon junctions, where it remains during nuclear export and cytoplasmic localization until it is removed during the first translation round.  The exon junction core complex consists of four proteins that form a dynamic binding platform for a variety of peripheral factors involved in mRNA metab.  In the complex, mRNA binding is mediated by the DEAD-box protein eIF4AIII, and inhibition of its ATPase activity forms the mechanistic basis for the long-term stability of the complex.  Recent crystal structures of the exon junction complex and eIF4AIII have provided the structural framework for investigating the function of the eIF4AIII ATPase and for localization of surface patches involved in binding peripheral factors.  Addnl., by comparison with the structure of a second DEAD-box protein also bound to RNA and ATP, general principles for the ATPase and unwinding/mRNP remodeling activities for this important group of enzymes can be proposed on the basis of at. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV9L-Nb_8PHLVg90H21EOLACvtfcHk0li-HOBCY64KWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVGitrs%253D&md5=efeb06df610d55dc6b155f92a234dbbe</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2007.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2007.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BHir%26aufirst%3DH.%26aulast%3DAndersen%26aufirst%3DG.%2BR.%26atitle%3DStructural%2520insights%2520into%2520the%2520exon%2520junction%2520complex%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D18%26spage%3D112%26epage%3D119%26doi%3D10.1016%2Fj.sbi.2007.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Palacios, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatfield, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St Johnston, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izaurralde, E.</span><span> </span><span class="NLM_article-title">An eIF4AIII-containing complex required for mRNA localization and nonsense-mediated mRNA decay</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">427</span><span class="NLM_x">, </span> <span class="NLM_fpage">753</span><span class="NLM_x">–</span> <span class="NLM_lpage">757</span><span class="refDoi"> DOI: 10.1038/nature02351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnature02351" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2004&pages=753-757&author=I.+M.+Palaciosauthor=D.+Gatfieldauthor=D.+St+Johnstonauthor=E.+Izaurralde&title=An+eIF4AIII-containing+complex+required+for+mRNA+localization+and+nonsense-mediated+mRNA+decay&doi=10.1038%2Fnature02351"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature02351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature02351%26sid%3Dliteratum%253Aachs%26aulast%3DPalacios%26aufirst%3DI.%2BM.%26aulast%3DGatfield%26aufirst%3DD.%26aulast%3DSt%2BJohnston%26aufirst%3DD.%26aulast%3DIzaurralde%26aufirst%3DE.%26atitle%3DAn%2520eIF4AIII-containing%2520complex%2520required%2520for%2520mRNA%2520localization%2520and%2520nonsense-mediated%2520mRNA%2520decay%26jtitle%3DNature%26date%3D2004%26volume%3D427%26spage%3D753%26epage%3D757%26doi%3D10.1038%2Fnature02351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ferraiuolo, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ler, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa-Mattioli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonenberg, N.</span><span> </span><span class="NLM_article-title">A nuclear translation-like factor eIF4AIII is recruited to the mRNA during splicing and functions in nonsense-mediated decay</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">4118</span><span class="NLM_x">–</span> <span class="NLM_lpage">4123</span><span class="refDoi"> DOI: 10.1073/pnas.0400933101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1073%2Fpnas.0400933101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4118-4123&author=M.+A.+Ferraiuoloauthor=C.-S.+Leeauthor=L.+W.+Lerauthor=J.+L.+Hsuauthor=M.+Costa-Mattioliauthor=M.-J.+Luoauthor=R.+Reedauthor=N.+Sonenberg&title=A+nuclear+translation-like+factor+eIF4AIII+is+recruited+to+the+mRNA+during+splicing+and+functions+in+nonsense-mediated+decay&doi=10.1073%2Fpnas.0400933101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0400933101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0400933101%26sid%3Dliteratum%253Aachs%26aulast%3DFerraiuolo%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DC.-S.%26aulast%3DLer%26aufirst%3DL.%2BW.%26aulast%3DHsu%26aufirst%3DJ.%2BL.%26aulast%3DCosta-Mattioli%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.-J.%26aulast%3DReed%26aufirst%3DR.%26aulast%3DSonenberg%26aufirst%3DN.%26atitle%3DA%2520nuclear%2520translation-like%2520factor%2520eIF4AIII%2520is%2520recruited%2520to%2520the%2520mRNA%2520during%2520splicing%2520and%2520functions%2520in%2520nonsense-mediated%2520decay%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D4118%26epage%3D4123%26doi%3D10.1073%2Fpnas.0400933101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Shibuya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tange, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonenberg, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, M. J.</span><span> </span><span class="NLM_article-title">eIF4AIII binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">346</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span><span class="refDoi"> DOI: 10.1038/nsmb750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnsmb750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=15034551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFCiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=346-351&author=T.+Shibuyaauthor=T.+O.+Tangeauthor=N.+Sonenbergauthor=M.+J.+Moore&title=eIF4AIII+binds+spliced+mRNA+in+the+exon+junction+complex+and+is+essential+for+nonsense-mediated+decay&doi=10.1038%2Fnsmb750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">eIF4AIII binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay</span></div><div class="casAuthors">Shibuya, Toshiharu; Tange, Thomas O.; Sonenberg, Nahum; Moore, Melissa J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">346-351</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The exon junction complex (EJC), a set of proteins deposited on mRNAs as a consequence of pre-mRNA splicing, is a key effector of downstream mRNA metab.  We have identified eIF4AIII, a member of the eukaryotic translation initiation factor 4A family of RNA helicases (also known as DExH/D box proteins), as a novel EJC core component.  Crosslinking and antibody inhibition studies suggest that eIF4AIII constitutes at least part of the platform anchoring other EJC components to spliced mRNAs.  A nucleocytoplasmic shuttling protein, eIF4AIII assocs. in vitro and in vivo with two other EJC core factors, Y14 and Magoh.  In mammalian cells, eIF4AIII is essential for nonsense-mediated mRNA decay (NMD).  Finally, a model is proposed by which eIF4AIII represents a new functional class of DExH/D box proteins that act as RNA clamps or 'place holders' for the sequence-independent attachment of addnl. factors to RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotzxv4GWKKmrVg90H21EOLACvtfcHk0lhYx0OmBrV1xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFCiurw%253D&md5=08c7904741747af05644798f28159f7f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb750%26sid%3Dliteratum%253Aachs%26aulast%3DShibuya%26aufirst%3DT.%26aulast%3DTange%26aufirst%3DT.%2BO.%26aulast%3DSonenberg%26aufirst%3DN.%26aulast%3DMoore%26aufirst%3DM.%2BJ.%26atitle%3DeIF4AIII%2520binds%2520spliced%2520mRNA%2520in%2520the%2520exon%2520junction%2520complex%2520and%2520is%2520essential%2520for%2520nonsense-mediated%2520decay%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D346%26epage%3D351%26doi%3D10.1038%2Fnsmb750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Le Hir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Séraphin, B.</span><span> </span><span class="NLM_article-title">EJCs at the heart of translational control</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span><span class="refDoi"> DOI: 10.1016/j.cell.2008.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.cell.2008.04.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2008&pages=213-216&author=H.+Le+Hirauthor=B.+S%C3%A9raphin&title=EJCs+at+the+heart+of+translational+control&doi=10.1016%2Fj.cell.2008.04.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BHir%26aufirst%3DH.%26aulast%3DS%25C3%25A9raphin%26aufirst%3DB.%26atitle%3DEJCs%2520at%2520the%2520heart%2520of%2520translational%2520control%26jtitle%3DCell%26date%3D2008%26volume%3D133%26spage%3D213%26epage%3D216%26doi%3D10.1016%2Fj.cell.2008.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurt, E.</span><span> </span><span class="NLM_article-title">A conserved mRNA export machinery coupled to pre-mRNA splicing</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">523</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00627-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2FS0092-8674%2802%2900627-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2002&pages=523-531&author=R.+Reedauthor=E.+Hurt&title=A+conserved+mRNA+export+machinery+coupled+to+pre-mRNA+splicing&doi=10.1016%2FS0092-8674%2802%2900627-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900627-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900627-X%26sid%3Dliteratum%253Aachs%26aulast%3DReed%26aufirst%3DR.%26aulast%3DHurt%26aufirst%3DE.%26atitle%3DA%2520conserved%2520mRNA%2520export%2520machinery%2520coupled%2520to%2520pre-mRNA%2520splicing%26jtitle%3DCell%26date%3D2002%26volume%3D108%26spage%3D523%26epage%3D531%26doi%3D10.1016%2FS0092-8674%2802%2900627-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Michelle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cloutier, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toutant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkreta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thibault, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durand, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garneau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gendron, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapointe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Hir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinck, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elela, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabot, B.</span><span> </span><span class="NLM_article-title">Proteins associated with the exon junction complex also control the alternative splicing of apoptotic regulators</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">954</span><span class="NLM_x">–</span> <span class="NLM_lpage">967</span><span class="refDoi"> DOI: 10.1128/MCB.06130-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1128%2FMCB.06130-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=22203037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtV2ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=954-967&author=L.+Michelleauthor=A.+Cloutierauthor=J.+Toutantauthor=L.+Shkretaauthor=P.+Thibaultauthor=M.+Durandauthor=D.+Garneauauthor=D.+Gendronauthor=E.+Lapointeauthor=S.+Coutureauthor=H.+Le+Hirauthor=R.+Klinckauthor=S.+A.+Elelaauthor=P.+Prinosauthor=B.+Chabot&title=Proteins+associated+with+the+exon+junction+complex+also+control+the+alternative+splicing+of+apoptotic+regulators&doi=10.1128%2FMCB.06130-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Proteins associated with the exon junction complex also control the alternative splicing of apoptotic regulators</span></div><div class="casAuthors">Michelle, Laetitia; Cloutier, Alexandre; Toutant, Johanne; Shkreta, Lulzim; Thibault, Philippe; Durand, Mathieu; Garneau, Daniel; Gendron, Daniel; Lapointe, Elvy; Couture, Sonia; Le Hir, Herve; Klinck, Roscoe; Abou Elela, Sherif; Prinos, Panagiotis; Chabot, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">954-967</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Several apoptotic regulators, including Bcl-x, are alternatively spliced to produce isoforms with opposite functions.  We have used an RNA interference strategy to map the regulatory landscape controlling the expression of the Bcl-x splice variants in human cells.  Depleting proteins known as core (Y14 and eIF4A3) or auxiliary (RNPS1, Acinus, and SAP18) components of the exon junction complex (EJC) improved the prodn. of the proapoptotic Bcl-xS splice variant.  This effect was not seen when we depleted EJC proteins that typically participate in mRNA export (UAP56, Aly/Ref, and TAP) or that assoc. with the EJC to enforce nonsense-mediated RNA decay (MNL51, Upf1, Upf2, and Upf3b).  Core and auxiliary EJC components modulated Bcl-x splicing through different cis-acting elements, further suggesting that this activity is distinct from the established EJC function.  In support of a direct role in splicing control, recombinant eIF4A3, Y14, and Magoh proteins assocd. preferentially with the endogenous Bcl-x pre-mRNA, interacted with a model Bcl-x pre-mRNA in early splicing complexes, and specifically shifted Bcl-x alternative splicing in nuclear exts.  Finally, the depletion of Y14, eIF4A3, RNPS1, SAP18, and Acinus also encouraged the prodn. of other proapoptotic splice variants, suggesting that EJC-assocd. components are important regulators of apoptosis acting at the alternative splicing level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQrodxO9g_gLVg90H21EOLACvtfcHk0lhYx0OmBrV1xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtV2ju7k%253D&md5=612d5fe051d6a9b43d5e20a583b719dd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.06130-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06130-11%26sid%3Dliteratum%253Aachs%26aulast%3DMichelle%26aufirst%3DL.%26aulast%3DCloutier%26aufirst%3DA.%26aulast%3DToutant%26aufirst%3DJ.%26aulast%3DShkreta%26aufirst%3DL.%26aulast%3DThibault%26aufirst%3DP.%26aulast%3DDurand%26aufirst%3DM.%26aulast%3DGarneau%26aufirst%3DD.%26aulast%3DGendron%26aufirst%3DD.%26aulast%3DLapointe%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DS.%26aulast%3DLe%2BHir%26aufirst%3DH.%26aulast%3DKlinck%26aufirst%3DR.%26aulast%3DElela%26aufirst%3DS.%2BA.%26aulast%3DPrinos%26aufirst%3DP.%26aulast%3DChabot%26aufirst%3DB.%26atitle%3DProteins%2520associated%2520with%2520the%2520exon%2520junction%2520complex%2520also%2520control%2520the%2520alternative%2520splicing%2520of%2520apoptotic%2520regulators%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D954%26epage%3D967%26doi%3D10.1128%2FMCB.06130-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murigneux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Hir, H.</span><span> </span><span class="NLM_article-title">Transcriptome-wide modulation of splicing by the exon junction complex</span> <span class="citation_source-journal">Genome Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">551</span><span class="refDoi"> DOI: 10.1186/s13059-014-0551-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1186%2Fs13059-014-0551-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=551&author=Z.+Wangauthor=V.+Murigneuxauthor=H.+Le+Hir&title=Transcriptome-wide+modulation+of+splicing+by+the+exon+junction+complex&doi=10.1186%2Fs13059-014-0551-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1186%2Fs13059-014-0551-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13059-014-0551-7%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMurigneux%26aufirst%3DV.%26aulast%3DLe%2BHir%26aufirst%3DH.%26atitle%3DTranscriptome-wide%2520modulation%2520of%2520splicing%2520by%2520the%2520exon%2520junction%2520complex%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D551%26doi%3D10.1186%2Fs13059-014-0551-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Baker, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, R.</span><span> </span><span class="NLM_article-title">Nonsense-mediated mRNA decay: terminating erroneous gene expression</span> <span class="citation_source-journal">Curr. Opin. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1016/j.ceb.2004.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.ceb.2004.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=15145354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktVCqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2004&pages=293-299&author=K.+E.+Bakerauthor=R.+Parker&title=Nonsense-mediated+mRNA+decay%3A+terminating+erroneous+gene+expression&doi=10.1016%2Fj.ceb.2004.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Nonsense-mediated mRNA decay: terminating erroneous gene expression</span></div><div class="casAuthors">Baker, Kristian E.; Parker, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">293-299</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Nonsense-mediated mRNA decay is a surveillance pathway that reduces errors in gene expression by eliminating aberrant mRNAs that encode incomplete polypeptides.  Recent expts. suggest a working model whereby premature and normal translation termination events are distinct as a consequence of the spatial relationship between the termination codon and mRNA binding proteins, a relationship partially established by nuclear pre-mRNA processing.  Aberrant termination then leads to both translational repression and an increased susceptibility of the mRNA to multiple RNases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0ycN1Dri1eLVg90H21EOLACvtfcHk0lghwASvmeBIKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktVCqsbY%253D&md5=ac182d78f4a5500120a98f0707455196</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2004.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2004.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DK.%2BE.%26aulast%3DParker%26aufirst%3DR.%26atitle%3DNonsense-mediated%2520mRNA%2520decay%253A%2520terminating%2520erroneous%2520gene%2520expression%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2004%26volume%3D16%26spage%3D293%26epage%3D299%26doi%3D10.1016%2Fj.ceb.2004.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Keeling, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedwell, D. M.</span><span> </span><span class="NLM_article-title">Suppression of nonsense mutations as a therapeutic approach to treat genetic diseases</span> <span class="citation_source-journal">Wiley Interdiscip. Rev. RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">852</span><span class="refDoi"> DOI: 10.1002/wrna.95</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1002%2Fwrna.95" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=21976286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyrsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=837-852&author=K.+M.+Keelingauthor=D.+M.+Bedwell&title=Suppression+of+nonsense+mutations+as+a+therapeutic+approach+to+treat+genetic+diseases&doi=10.1002%2Fwrna.95"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of nonsense mutations as a therapeutic approach to treat genetic diseases</span></div><div class="casAuthors">Keeling, Kim M.; Bedwell, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">837-852</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Suppression therapy is a treatment strategy for genetic diseases caused by nonsense mutations.  This therapeutic approach utilizes pharmacol. agents that suppress translation termination at in-frame premature termination codons (PTCs) to restore translation of a full-length, functional polypeptide.  The efficiency of various classes of compds. to suppress PTCs in mammalian cells is discussed along with the current limitations of this therapy.  We also elaborate on approaches to improve the efficiency of suppression that include methods to enhance the effectiveness of current suppression drugs and the design or discovery of new, more effective suppression agents.  Finally, we discuss the role of nonsense-mediated mRNA decay (NMD) in limiting the effectiveness of suppression therapy, and describe tactics that may allow the efficiency of NMD to be modulated in order to enhance suppression therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBNm5QrXfib7Vg90H21EOLACvtfcHk0lghwASvmeBIKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyrsrjO&md5=c51c5d5bde9daac8d9d7adaa018107e1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fwrna.95&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.95%26sid%3Dliteratum%253Aachs%26aulast%3DKeeling%26aufirst%3DK.%2BM.%26aulast%3DBedwell%26aufirst%3DD.%2BM.%26atitle%3DSuppression%2520of%2520nonsense%2520mutations%2520as%2520a%2520therapeutic%2520approach%2520to%2520treat%2520genetic%2520diseases%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520RNA%26date%3D2011%26volume%3D2%26spage%3D837%26epage%3D852%26doi%3D10.1002%2Fwrna.95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigoryan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breda, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivella, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, L. B.</span><span> </span><span class="NLM_article-title">Identification and characterization of small molecules that inhibit nonsense mediated RNA decay and suppress nonsense p53 mutations</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">3104</span><span class="NLM_x">–</span> <span class="NLM_lpage">3113</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-2235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1158%2F0008-5472.CAN-13-2235" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=3104-3113&author=L.+Martinauthor=A.+Grigoryanauthor=D.+Wangauthor=J.+Wangauthor=L.+Bredaauthor=S.+Rivellaauthor=T.+Cardozoauthor=L.+B.+Gardner&title=Identification+and+characterization+of+small+molecules+that+inhibit+nonsense+mediated+RNA+decay+and+suppress+nonsense+p53+mutations&doi=10.1158%2F0008-5472.CAN-13-2235"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-2235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-2235%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DGrigoryan%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBreda%26aufirst%3DL.%26aulast%3DRivella%26aufirst%3DS.%26aulast%3DCardozo%26aufirst%3DT.%26aulast%3DGardner%26aufirst%3DL.%2BB.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520small%2520molecules%2520that%2520inhibit%2520nonsense%2520mediated%2520RNA%2520decay%2520and%2520suppress%2520nonsense%2520p53%2520mutations%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D3104%26epage%3D3113%26doi%3D10.1158%2F0008-5472.CAN-13-2235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Pastor, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolonias, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giangrande, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilboa, E.</span><span> </span><span class="NLM_article-title">Induction of tumour immunity by targeted inhibition of nonsense-mediated mRNA decay</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">465</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span><span class="refDoi"> DOI: 10.1038/nature08999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnature08999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=20463739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFOrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2010&pages=227-230&author=F.+Pastorauthor=D.+Koloniasauthor=P.+H.+Giangrandeauthor=E.+Gilboa&title=Induction+of+tumour+immunity+by+targeted+inhibition+of+nonsense-mediated+mRNA+decay&doi=10.1038%2Fnature08999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of tumour immunity by targeted inhibition of nonsense-mediated mRNA decay</span></div><div class="casAuthors">Pastor, Fernando; Kolonias, Despina; Giangrande, Paloma H.; Gilboa, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">7295</span>),
    <span class="NLM_cas:pages">227-230</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The main reason why tumors are not controlled by the immune system is that, unlike pathogens, they do not express potent tumor rejection antigens (TRAs).  Tumor vaccination aims at stimulating a systemic immune response targeted to, mostly weak, antigens expressed in the disseminated tumor lesions.  Main challenges in developing effective vaccination protocols are the identification of potent and broadly expressed TRAs and effective adjuvants to stimulate a robust and durable immune response.  Here the authors describe an alternative approach in which the expression of new, and thereby potent, antigens are induced in tumor cells by inhibiting nonsense-mediated mRNA decay (NMD).  Small interfering RNA (siRNA)-mediated inhibition of NMD in tumor cells led to the expression of new antigenic determinants and their immune-mediated rejection.  In s.c. and metastatic tumor models, tumor-targeted delivery of NMD factor-specific siRNAs conjugated to oligonucleotide aptamer ligands led to significant inhibition of tumor growth that was superior to that of vaccination with granulocyte-macrophage colony-stimulating factor (GM-CSF)-expressing irradiated tumor cells, and could be further enhanced by co-stimulation.  Tumor-targeted NMD inhibition forms the basis of a simple, broadly useful, and clin. feasible approach to enhance the antigenicity of disseminated tumors leading to their immune recognition and rejection.  The cell-free chem. synthesized oligonucleotide backbone of aptamer-siRNAs reduces the risk of immunogenicity and enhances the feasibility of generating reagents suitable for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdL4APQthUHbVg90H21EOLACvtfcHk0lghwASvmeBIKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFOrsbk%253D&md5=57a1b9b7c4e1e90fcc50f30a158ffa22</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature08999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08999%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DF.%26aulast%3DKolonias%26aufirst%3DD.%26aulast%3DGiangrande%26aufirst%3DP.%2BH.%26aulast%3DGilboa%26aufirst%3DE.%26atitle%3DInduction%2520of%2520tumour%2520immunity%2520by%2520targeted%2520inhibition%2520of%2520nonsense-mediated%2520mRNA%2520decay%26jtitle%3DNature%26date%3D2010%26volume%3D465%26spage%3D227%26epage%3D230%26doi%3D10.1038%2Fnature08999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Popp, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maquat, L. E.</span><span> </span><span class="NLM_article-title">Attenuation of nonsense-mediated mRNA decay facilitates the response to chemotherapeutics</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">6632</span><span class="NLM_x">–</span> <span class="NLM_lpage">6632</span><span class="refDoi"> DOI: 10.1038/ncomms7632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fncomms7632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=25808464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2itLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6632-6632&author=M.+W.+Poppauthor=L.+E.+Maquat&title=Attenuation+of+nonsense-mediated+mRNA+decay+facilitates+the+response+to+chemotherapeutics&doi=10.1038%2Fncomms7632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuation of nonsense-mediated mRNA decay facilitates the response to chemotherapeutics</span></div><div class="casAuthors">Popp, Maximilian W.; Maquat, Lynne E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6632</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nonsense-mediated mRNA decay (NMD) limits the prodn. of aberrant mRNAs contg. a premature termination codon and also controls the levels of endogenous transcripts.  Here we show that when human cells are treated with clin. used chemotherapeutic compds., NMD activity declines partly as a result of the proteolytic prodn. of a dominant-interfering form of the key NMD factor UPF1.  Prodn. of cleaved UPF1 functions to upregulate genes involved in the response to apoptotic stresses.  The biol. consequence is the promotion of cell death.  Combined exposure of cells to a small-mol. inhibitor of NMD, NMDI-1, and the chemotherapeutic doxorubicin leads to enhanced cell death, while inhibiting UPF1 cleavage protects cells from doxorubicin challenge.  We propose a model to explain why the expression levels of genes producing mRNAs of diverse structure that encode proteins of diverse function are under the purview of NMD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKtCDG1_OzHbVg90H21EOLACvtfcHk0lhd_rzgM3VaKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2itLnJ&md5=05e84ebc26b77de5e7c1da1b9e9ac2f2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fncomms7632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7632%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DM.%2BW.%26aulast%3DMaquat%26aufirst%3DL.%2BE.%26atitle%3DAttenuation%2520of%2520nonsense-mediated%2520mRNA%2520decay%2520facilitates%2520the%2520response%2520to%2520chemotherapeutics%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6632%26epage%3D6632%26doi%3D10.1038%2Fncomms7632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Hug, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caceres, J. F.</span><span> </span><span class="NLM_article-title">Mechanism and regulation of the nonsense-mediated decay pathway</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1483</span><span class="NLM_x">–</span> <span class="NLM_lpage">1495</span><span class="refDoi"> DOI: 10.1093/nar/gkw010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1093%2Fnar%2Fgkw010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1483-1495&author=N.+Hugauthor=D.+Longmanauthor=J.+F.+Caceres&title=Mechanism+and+regulation+of+the+nonsense-mediated+decay+pathway&doi=10.1093%2Fnar%2Fgkw010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw010%26sid%3Dliteratum%253Aachs%26aulast%3DHug%26aufirst%3DN.%26aulast%3DLongman%26aufirst%3DD.%26aulast%3DCaceres%26aufirst%3DJ.%2BF.%26atitle%3DMechanism%2520and%2520regulation%2520of%2520the%2520nonsense-mediated%2520decay%2520pathway%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D1483%26epage%3D1495%26doi%3D10.1093%2Fnar%2Fgkw010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Kurosaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maquat, L. E.</span><span> </span><span class="NLM_article-title">Nonsense-mediated mRNA decay in humans at a glance</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span><span class="refDoi"> DOI: 10.1242/jcs.181008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1242%2Fjcs.181008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=26787741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpt1Omt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=461-467&author=T.+Kurosakiauthor=L.+E.+Maquat&title=Nonsense-mediated+mRNA+decay+in+humans+at+a+glance&doi=10.1242%2Fjcs.181008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Nonsense-mediated mRNA decay in humans at a glance</span></div><div class="casAuthors">Kurosaki, Tatsuaki; Maquat, Lynne E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">461-467</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Nonsense-mediated mRNA decay (NMD) is an mRNA quality-control mechanism that typifies all eukaryotes examd. to date.  NMD surveys newly synthesized mRNAs and degrades those that harbor a premature termination codon (PTC), thereby preventing the prodn. of truncated proteins that could result in disease in humans.  This is evident from dominantly inherited diseases that are due to PTC-contg. mRNAs that escape NMD.  Although many cellular NMD targets derive from mistakes made during, for example, pre-mRNA splicing and, possibly, transcription initiation, NMD also targets ∼10% of normal physiol. mRNAs so as to promote an appropriate cellular response to changing environmental milieus, including those that induce apoptosis, maturation or differentiation, Over the past ∼35 years, a central goal in the NMD field has been to understand how cells discriminate mRNAs that are targeted by NMD from those that are not.  In this Cell Science at a Glance and the accompanying poster, we review progress made towards this goal, focusing on human studies and the role of the key NMD factor up-frameshift protein 1 (UPF1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofrNkqevLoPLVg90H21EOLACvtfcHk0lhd_rzgM3VaKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpt1Omt7g%253D&md5=0f76cab950a742708107974616a88a4e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1242%2Fjcs.181008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.181008%26sid%3Dliteratum%253Aachs%26aulast%3DKurosaki%26aufirst%3DT.%26aulast%3DMaquat%26aufirst%3DL.%2BE.%26atitle%3DNonsense-mediated%2520mRNA%2520decay%2520in%2520humans%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D461%26epage%3D467%26doi%3D10.1242%2Fjcs.181008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Bhuvanagiri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Putzker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leicht, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krijgsveld, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnwald, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jovanovic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sieber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hentze, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulozik, A. E.</span><span> </span><span class="NLM_article-title">5-azacytidine inhibits nonsense-mediated decay in a MYC-dependent fashion</span> <span class="citation_source-journal">EMBO Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1593</span><span class="NLM_x">–</span> <span class="NLM_lpage">1609</span><span class="refDoi"> DOI: 10.15252/emmm.201404461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.15252%2Femmm.201404461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=25319547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVamur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1593-1609&author=M.+Bhuvanagiriauthor=J.+Lewisauthor=K.+Putzkerauthor=J.+P.+Beckerauthor=S.+Leichtauthor=J.+Krijgsveldauthor=R.+Batraauthor=B.+Turnwaldauthor=B.+Jovanovicauthor=C.+Hauerauthor=J.+Sieberauthor=M.+W.+Hentzeauthor=A.+E.+Kulozik&title=5-azacytidine+inhibits+nonsense-mediated+decay+in+a+MYC-dependent+fashion&doi=10.15252%2Femmm.201404461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">5-azacytidine inhibits nonsense-mediated decay in a MYC-dependent fashion</span></div><div class="casAuthors">Bhuvanagiri, Madhuri; Lewis, Joe; Putzker, Kerstin; Becker, Jonas P.; Leicht, Stefan; Krijgsveld, Jeroen; Batra, Richa; Turnwald, Brad; Jovanovic, Bogdan; Hauer, Christian; Sieber, Jana; Hentze, Matthias W.; Kulozik, Andreas E.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1593-1609</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Nonsense-mediated RNA decay (NMD) is an RNA-based quality control mechanism that eliminates transcripts bearing premature translation termination codons (PTC).  Approx., one-third of all inherited disorders and some forms of cancer are caused by nonsense or frame shift mutations that introduce PTCs, and NMD can modulate the clin. phenotype of these diseases.  5-Azacytidine is an analog of the naturally occurring pyrimidine nucleoside cytidine, which is approved for the treatment of myelodysplastic syndrome and myeloid leukemia.  Here, we reveal that 5-azacytidine inhibits NMD in a dose-dependent fashion specifically upregulating the expression of both PTC-contg. mutant and cellular NMD targets.  Moreover, this activity of 5-azacytidine depends on the induction of MYC expression, thus providing a link between the effect of this drug and one of the key cellular pathways that are known to affect NMD activity.  Furthermore, the effective concn. of 5-azacytidine in cells corresponds to drug levels used in patients, qualifying 5-azacytidine as a candidate drug that could potentially be repurposed for the treatment of Mendelian and acquired genetic diseases that are caused by PTC mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruolEF6XIUibVg90H21EOLACvtfcHk0lhd_rzgM3VaKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVamur%252FJ&md5=1af5afdc70b4289ea84c87509ae9998c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.15252%2Femmm.201404461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Femmm.201404461%26sid%3Dliteratum%253Aachs%26aulast%3DBhuvanagiri%26aufirst%3DM.%26aulast%3DLewis%26aufirst%3DJ.%26aulast%3DPutzker%26aufirst%3DK.%26aulast%3DBecker%26aufirst%3DJ.%2BP.%26aulast%3DLeicht%26aufirst%3DS.%26aulast%3DKrijgsveld%26aufirst%3DJ.%26aulast%3DBatra%26aufirst%3DR.%26aulast%3DTurnwald%26aufirst%3DB.%26aulast%3DJovanovic%26aufirst%3DB.%26aulast%3DHauer%26aufirst%3DC.%26aulast%3DSieber%26aufirst%3DJ.%26aulast%3DHentze%26aufirst%3DM.%2BW.%26aulast%3DKulozik%26aufirst%3DA.%2BE.%26atitle%3D5-azacytidine%2520inhibits%2520nonsense-mediated%2520decay%2520in%2520a%2520MYC-dependent%2520fashion%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2014%26volume%3D6%26spage%3D1593%26epage%3D1609%26doi%3D10.15252%2Femmm.201404461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Dang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Low, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. O.</span><span> </span><span class="NLM_article-title">Inhibition of nonsense-mediated mRNA decay by the natural product pateamine A through eukaryotic initiation factor 4AIII</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">23613</span><span class="NLM_x">–</span> <span class="NLM_lpage">23621</span><span class="refDoi"> DOI: 10.1074/jbc.M109.009985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1074%2Fjbc.M109.009985" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23613-23621&author=Y.+Dangauthor=W.+K.+Lowauthor=J.+Xuauthor=N.+H.+Gehringauthor=H.+C.+Dietzauthor=D.+Romoauthor=J.+O.+Liu&title=Inhibition+of+nonsense-mediated+mRNA+decay+by+the+natural+product+pateamine+A+through+eukaryotic+initiation+factor+4AIII&doi=10.1074%2Fjbc.M109.009985"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.009985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.009985%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DY.%26aulast%3DLow%26aufirst%3DW.%2BK.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DGehring%26aufirst%3DN.%2BH.%26aulast%3DDietz%26aufirst%3DH.%2BC.%26aulast%3DRomo%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26atitle%3DInhibition%2520of%2520nonsense-mediated%2520mRNA%2520decay%2520by%2520the%2520natural%2520product%2520pateamine%2520A%2520through%2520eukaryotic%2520initiation%2520factor%25204AIII%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D23613%26epage%3D23621%26doi%3D10.1074%2Fjbc.M109.009985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Durand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cougot, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahuteau-Betzer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, C.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grierson, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tazi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, F.</span><span> </span><span class="NLM_article-title">Inhibition of nonsense-mediated mRNA decay (NMD) by a new chemical molecule reveals the dynamic of NMD factors in P-bodies</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">1145</span><span class="NLM_x">–</span> <span class="NLM_lpage">1160</span><span class="refDoi"> DOI: 10.1083/jcb.200611086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1083%2Fjcb.200611086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2007&pages=1145-1160&author=S.+Durandauthor=N.+Cougotauthor=F.+Mahuteau-Betzerauthor=C.-H.+Nguyenauthor=D.+S.+Griersonauthor=E.+Bertrandauthor=J.+Taziauthor=F.+Lejeune&title=Inhibition+of+nonsense-mediated+mRNA+decay+%28NMD%29+by+a+new+chemical+molecule+reveals+the+dynamic+of+NMD+factors+in+P-bodies&doi=10.1083%2Fjcb.200611086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200611086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200611086%26sid%3Dliteratum%253Aachs%26aulast%3DDurand%26aufirst%3DS.%26aulast%3DCougot%26aufirst%3DN.%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26aulast%3DNguyen%26aufirst%3DC.-H.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26aulast%3DBertrand%26aufirst%3DE.%26aulast%3DTazi%26aufirst%3DJ.%26aulast%3DLejeune%26aufirst%3DF.%26atitle%3DInhibition%2520of%2520nonsense-mediated%2520mRNA%2520decay%2520%2528NMD%2529%2520by%2520a%2520new%2520chemical%2520molecule%2520reveals%2520the%2520dynamic%2520of%2520NMD%2520factors%2520in%2520P-bodies%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2007%26volume%3D178%26spage%3D1145%26epage%3D1160%26doi%3D10.1083%2Fjcb.200611086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Feng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, R.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Triggs-Raine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, J.</span><span> </span><span class="NLM_article-title">Increase of a group of PTC<sup>+</sup> transcripts by curcumin through inhibition of the NMD pathway</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1849</span><span class="NLM_x">, </span> <span class="NLM_fpage">1104</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2015.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.bbagrm.2015.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=25934542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvVSjs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1849&publication_year=2015&pages=1104-1115&author=D.+Fengauthor=R.-C.+Suauthor=L.+Zouauthor=B.+Triggs-Raineauthor=S.+Huangauthor=J.+Xie&title=Increase+of+a+group+of+PTC%2B+transcripts+by+curcumin+through+inhibition+of+the+NMD+pathway&doi=10.1016%2Fj.bbagrm.2015.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Increase of a group of PTC+ transcripts by curcumin through inhibition of the NMD pathway</span></div><div class="casAuthors">Feng, Dairong; Su, Ruey-Chyi; Zou, Liping; Triggs-Raine, Barbara; Huang, Shangzhi; Xie, Jiuyong</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1849</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1104-1115</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nonsense-mediated mRNA decay (NMD), an mRNA surveillance mechanism, eliminates premature termination codon-contg. (PTC+) transcripts.  For instance, it maintains the homeostasis of splicing factors and degrades aberrant transcripts of human genetic disease genes.  Here we examine the inhibitory effect on the NMD pathway and consequent increase of PTC+ transcripts by the dietary compd. curcumin.  We have found that several PTC+ transcripts including that of serine/arginine-rich splicing factor 1 (SRSF1) were specifically increased in cells by curcumin.  We also obsd. a similar curcumin effect on the PTC+ mutant transcript from a Tay-Sachs-causing HEXA allele or from a beta-globin reporter gene.  The curcumin effect was accompanied by significantly reduced expression of the NMD factors UPF1, 2, 3A and 3B.  Consistently, in chromatin immunopptn. assays, curcumin specifically reduced the occupancy of acetyl-histone H3 and RNA polymerase II at the promoter region (- 376 to - 247 nt) of human UPF1, in a time- and dosage-dependent way.  Importantly, knocking down UPF1 abolished or substantially reduced the difference of PTC+ transcript levels between control and curcumin-treated cells.  The disrupted curcumin effect was efficiently rescued by expression of exogenous Myc-UPF1 in the knockdown cells.  Together, our data demonstrate that a group of PTC+ transcripts are stabilized by a dietary compd. curcumin through the inhibition of UPF factor expression and the NMD pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreaYtNoV7jJ7Vg90H21EOLACvtfcHk0ljKb3OHzHBIYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvVSjs7w%253D&md5=2b3492d4150198be313e62c662b5b483</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2015.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2015.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DSu%26aufirst%3DR.-C.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DTriggs-Raine%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DJ.%26atitle%3DIncrease%2520of%2520a%2520group%2520of%2520PTC%252B%2520transcripts%2520by%2520curcumin%2520through%2520inhibition%2520of%2520the%2520NMD%2520pathway%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2015%26volume%3D1849%26spage%3D1104%26epage%3D1115%26doi%3D10.1016%2Fj.bbagrm.2015.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Bordeleau, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oberer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindqvist, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chard, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belsham, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span> </span><span class="NLM_article-title">Functional characterization of IRESes by an inhibitor of the RNA helicase eIF4A</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nchembio776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnchembio776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=16532013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFOnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=213-220&author=M.+E.+Bordeleauauthor=A.+Moriauthor=M.+Obererauthor=L.+Lindqvistauthor=L.+S.+Chardauthor=T.+Higaauthor=G.+J.+Belshamauthor=G.+Wagnerauthor=J.+Tanakaauthor=J.+Pelletier&title=Functional+characterization+of+IRESes+by+an+inhibitor+of+the+RNA+helicase+eIF4A&doi=10.1038%2Fnchembio776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of IRESes by an inhibitor of the RNA helicase eIF4A</span></div><div class="casAuthors">Bordeleau, Marie-Eve; Mori, Ayaka; Oberer, Monika; Lindqvist, Lisa; Chard, Louisa S.; Higa, Tatsuo; Belsham, Graham J.; Wagner, Gerhard; Tanaka, Junichi; Pelletier, Jerry</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">213-220</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA helicases are mol. motors that are involved in virtually all aspects of RNA metab.  Eukaryotic initiation factor (eIF) 4A is the prototypical member of the DEAD-box family of RNA helicases.  It is thought to use energy from ATP hydrolysis to unwind mRNA structure and, in conjunction with other translation factors, it preps. mRNA templates for ribosome recruitment during translation initiation.  In screening marine exts. for new eukaryotic translation initiation inhibitors, we identified the natural product hippuristanol.  We show here that this compd. is a selective and potent inhibitor of eIF4A RNA-binding activity that can be used to distinguish between eIF4A-dependent and -independent modes of translation initiation in vitro and in vivo.  We also show that poliovirus replication is delayed when infected cells are exposed to hippuristanol.  Our study demonstrates the feasibility of selectively targeting members of the DEAD-box helicase family with small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphVpPHxoy7Y7Vg90H21EOLACvtfcHk0lhwxfyPdqYMag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFOnsbg%253D&md5=cbeecb6be4f6ab28c47b4c0542ba4d3d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio776%26sid%3Dliteratum%253Aachs%26aulast%3DBordeleau%26aufirst%3DM.%2BE.%26aulast%3DMori%26aufirst%3DA.%26aulast%3DOberer%26aufirst%3DM.%26aulast%3DLindqvist%26aufirst%3DL.%26aulast%3DChard%26aufirst%3DL.%2BS.%26aulast%3DHiga%26aufirst%3DT.%26aulast%3DBelsham%26aufirst%3DG.%2BJ.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DTanaka%26aufirst%3DJ.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DFunctional%2520characterization%2520of%2520IRESes%2520by%2520an%2520inhibitor%2520of%2520the%2520RNA%2520helicase%2520eIF4A%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D213%26epage%3D220%26doi%3D10.1038%2Fnchembio776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Lindqvist, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oberer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reibarkh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cencic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bordeleau, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marintchev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagotto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span> </span><span class="NLM_article-title">Selective pharmacological targeting of a DEAD box RNA helicase</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1583</span><span class="refDoi"> DOI: 10.1371/journal.pone.0001583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1371%2Fjournal.pone.0001583" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=18270573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A280%3ADC%252BD1c7htlagtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e1583&author=L.+Lindqvistauthor=M.+Obererauthor=M.+Reibarkhauthor=R.+Cencicauthor=M.+E.+Bordeleauauthor=E.+Vogtauthor=A.+Marintchevauthor=J.+Tanakaauthor=F.+Fagottoauthor=M.+Altmannauthor=G.+Wagnerauthor=J.+Pelletier&title=Selective+pharmacological+targeting+of+a+DEAD+box+RNA+helicase&doi=10.1371%2Fjournal.pone.0001583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Selective pharmacological targeting of a DEAD box RNA helicase</span></div><div class="casAuthors">Lindqvist Lisa; Oberer Monika; Reibarkh Mikhail; Cencic Regina; Bordeleau Marie-Eve; Vogt Emily; Marintchev Assen; Tanaka Junichi; Fagotto Francois; Altmann Michael; Wagner Gerhard; Pelletier Jerry</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1583</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">RNA helicases represent a large family of proteins implicated in many biological processes including ribosome biogenesis, splicing, translation and mRNA degradation.  However, these proteins have little substrate specificity, making inhibition of selected helicases a challenging problem.  The prototypical DEAD box RNA helicase, eIF4A, works in conjunction with other translation factors to prepare mRNA templates for ribosome recruitment during translation initiation.  Herein, we provide insight into the selectivity of a small molecule inhibitor of eIF4A, hippuristanol.  This coral-derived natural product binds to amino acids adjacent to, and overlapping with, two conserved motifs present in the carboxy-terminal domain of eIF4A.  Mutagenesis of amino acids within this region allowed us to alter the hippuristanol-sensitivity of eIF4A and undertake structure/function studies.  Our results provide an understanding into how selective targeting of RNA helicases for pharmacological intervention can be achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKpFB5Tdsa_tXdcedVHT5tfW6udTcc2ea69_LuDJNFarntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7htlagtg%253D%253D&md5=b29e717681032adca5889d26e11e480d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0001583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0001583%26sid%3Dliteratum%253Aachs%26aulast%3DLindqvist%26aufirst%3DL.%26aulast%3DOberer%26aufirst%3DM.%26aulast%3DReibarkh%26aufirst%3DM.%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DBordeleau%26aufirst%3DM.%2BE.%26aulast%3DVogt%26aufirst%3DE.%26aulast%3DMarintchev%26aufirst%3DA.%26aulast%3DTanaka%26aufirst%3DJ.%26aulast%3DFagotto%26aufirst%3DF.%26aulast%3DAltmann%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DSelective%2520pharmacological%2520targeting%2520of%2520a%2520DEAD%2520box%2520RNA%2520helicase%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De1583%26doi%3D10.1371%2Fjournal.pone.0001583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Higa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukitani, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikuchi, H.</span><span> </span><span class="NLM_article-title">Hippuristanols, cytotoxic polyoxygenated steroids from the gorgonian isis hippuris</span> <span class="citation_source-journal">Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1647</span><span class="NLM_x">–</span> <span class="NLM_lpage">1650</span><span class="refDoi"> DOI: 10.1246/cl.1981.1647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1246%2Fcl.1981.1647" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1981&pages=1647-1650&author=T.+Higaauthor=J.+Tanakaauthor=Y.+Tsukitaniauthor=H.+Kikuchi&title=Hippuristanols%2C+cytotoxic+polyoxygenated+steroids+from+the+gorgonian+isis+hippuris&doi=10.1246%2Fcl.1981.1647"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1246%2Fcl.1981.1647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1246%252Fcl.1981.1647%26sid%3Dliteratum%253Aachs%26aulast%3DHiga%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DJ.%26aulast%3DTsukitani%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DH.%26atitle%3DHippuristanols%252C%2520cytotoxic%2520polyoxygenated%2520steroids%2520from%2520the%2520gorgonian%2520isis%2520hippuris%26jtitle%3DChem.%2520Lett.%26date%3D1981%26volume%3D10%26spage%3D1647%26epage%3D1650%26doi%3D10.1246%2Fcl.1981.1647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Cencic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span> </span><span class="NLM_article-title">Hippuristanol - A potent steroid inhibitor of eukaryotic initiation factor 4A</span> <span class="citation_source-journal">Translation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1137381</span><span class="refDoi"> DOI: 10.1080/21690731.2015.1137381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1080%2F21690731.2015.1137381" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=e1137381&author=R.+Cencicauthor=J.+Pelletier&title=Hippuristanol+-+A+potent+steroid+inhibitor+of+eukaryotic+initiation+factor+4A&doi=10.1080%2F21690731.2015.1137381"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1080%2F21690731.2015.1137381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21690731.2015.1137381%26sid%3Dliteratum%253Aachs%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DHippuristanol%2520-%2520A%2520potent%2520steroid%2520inhibitor%2520of%2520eukaryotic%2520initiation%2520factor%25204A%26jtitle%3DTranslation%26date%3D2016%26volume%3D4%26spage%3De1137381%26doi%3D10.1080%2F21690731.2015.1137381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogabe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span> </span><span class="NLM_article-title">Discovery of selective ATP-competitive eIF4A3 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2200</span><span class="NLM_x">–</span> <span class="NLM_lpage">2209</span><span class="refDoi"> DOI: 10.1016/j.bmc.2017.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.bmc.2017.02.035" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2200-2209&author=M.+Itoauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Sogabeauthor=S.+Nakaoauthor=T.+Tanakaauthor=T.+Kawamotoauthor=S.+Aparicioauthor=A.+Nakanishiauthor=Y.+Imaeda&title=Discovery+of+selective+ATP-competitive+eIF4A3+inhibitors&doi=10.1016%2Fj.bmc.2017.02.035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DM.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DNakao%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DNakanishi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520selective%2520ATP-competitive%2520eIF4A3%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2200%26epage%3D2209%26doi%3D10.1016%2Fj.bmc.2017.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Sánchez-Céspedes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martínez-Aguado, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vega-Holm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serna-Gallego, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrugal-Lorenzo, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pachón, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iglesias-Guerra, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vega-Pérez, J. M.</span><span> </span><span class="NLM_article-title">New 4-acyl-1-phenylaminocarbonyl-2-phenylpiperazine derivatives as potential inhibitors of adenovirus infection. synthesis, biological evaluation, and structure–activity relationships</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5432</span><span class="NLM_x">–</span> <span class="NLM_lpage">5448</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00300</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00300" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5432-5448&author=J.+S%C3%A1nchez-C%C3%A9spedesauthor=P.+Mart%C3%ADnez-Aguadoauthor=M.+Vega-Holmauthor=A.+Serna-Gallegoauthor=J.+I.+Candelaauthor=J.+A.+Marrugal-Lorenzoauthor=J.+Pach%C3%B3nauthor=F.+Iglesias-Guerraauthor=J.+M.+Vega-P%C3%A9rez&title=New+4-acyl-1-phenylaminocarbonyl-2-phenylpiperazine+derivatives+as+potential+inhibitors+of+adenovirus+infection.+synthesis%2C+biological+evaluation%2C+and+structure%E2%80%93activity+relationships&doi=10.1021%2Facs.jmedchem.6b00300"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00300%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-C%25C3%25A9spedes%26aufirst%3DJ.%26aulast%3DMart%25C3%25ADnez-Aguado%26aufirst%3DP.%26aulast%3DVega-Holm%26aufirst%3DM.%26aulast%3DSerna-Gallego%26aufirst%3DA.%26aulast%3DCandela%26aufirst%3DJ.%2BI.%26aulast%3DMarrugal-Lorenzo%26aufirst%3DJ.%2BA.%26aulast%3DPach%25C3%25B3n%26aufirst%3DJ.%26aulast%3DIglesias-Guerra%26aufirst%3DF.%26aulast%3DVega-P%25C3%25A9rez%26aufirst%3DJ.%2BM.%26atitle%3DNew%25204-acyl-1-phenylaminocarbonyl-2-phenylpiperazine%2520derivatives%2520as%2520potential%2520inhibitors%2520of%2520adenovirus%2520infection.%2520synthesis%252C%2520biological%2520evaluation%252C%2520and%2520structure%25E2%2580%2593activity%2520relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5432%26epage%3D5448%26doi%3D10.1021%2Facs.jmedchem.6b00300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Glynn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, S.</span><span> </span><span class="NLM_article-title">Microwave acceleration in DABAL-Me<sub>3</sub>-mediated amide formation</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5687</span><span class="NLM_x">–</span> <span class="NLM_lpage">5688</span><span class="refDoi"> DOI: 10.1016/j.tetlet.2008.07.090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.tetlet.2008.07.090" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2008&pages=5687-5688&author=D.+Glynnauthor=D.+Bernierauthor=S.+Woodward&title=Microwave+acceleration+in+DABAL-Me3-mediated+amide+formation&doi=10.1016%2Fj.tetlet.2008.07.090"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2008.07.090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2008.07.090%26sid%3Dliteratum%253Aachs%26aulast%3DGlynn%26aufirst%3DD.%26aulast%3DBernier%26aufirst%3DD.%26aulast%3DWoodward%26aufirst%3DS.%26atitle%3DMicrowave%2520acceleration%2520in%2520DABAL-Me3-mediated%2520amide%2520formation%26jtitle%3DTetrahedron%2520Lett.%26date%3D2008%26volume%3D49%26spage%3D5687%26epage%3D5688%26doi%3D10.1016%2Fj.tetlet.2008.07.090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Besselievre, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahuteau-Betzer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grierson, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piguel, S.</span><span> </span><span class="NLM_article-title">Ligandless microwave-assisted Pd/Cu-catalyzed direct arylation of oxazoles</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">3278</span><span class="NLM_x">–</span> <span class="NLM_lpage">3280</span><span class="refDoi"> DOI: 10.1021/jo7027135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo7027135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVWht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=3278-3280&author=F.+Besselievreauthor=F.+Mahuteau-Betzerauthor=D.+S.+Griersonauthor=S.+Piguel&title=Ligandless+microwave-assisted+Pd%2FCu-catalyzed+direct+arylation+of+oxazoles&doi=10.1021%2Fjo7027135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Ligandless microwave-assisted Pd/Cu-catalyzed direct arylation of oxazoles</span></div><div class="casAuthors">Besselievre, Francois; Mahuteau-Betzer, Florence; Grierson, David S.; Piguel, Sandrine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3278-3280</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient microwave-assisted palladium/copper co-mediated direct arylation of oxazoles with aryl bromides under ligandless conditions has been developed.  The method is functional group tolerant and provides rapid access to medicinally relevant compds., e.g. I (R = 2-MeC6H4, 1-naphthyl, 3-NCC6H4, etc.), in good yields.  Coupled to the van Leusen oxazole ring synthesis, this methodol. is illustrated by an expedient two-step synthesis of the four 2,5-diaryloxazole alkaloids, texamine, texaline, balsoxin, and O-Me-halfordinol from com. available starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEHwRO9EHnbVg90H21EOLACvtfcHk0liHhgxZnqxUPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVWht7g%253D&md5=7beb810c4a3d6ac4f4013736651f0804</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjo7027135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo7027135%26sid%3Dliteratum%253Aachs%26aulast%3DBesselievre%26aufirst%3DF.%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26aulast%3DPiguel%26aufirst%3DS.%26atitle%3DLigandless%2520microwave-assisted%2520Pd%252FCu-catalyzed%2520direct%2520arylation%2520of%2520oxazoles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D3278%26epage%3D3280%26doi%3D10.1021%2Fjo7027135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Sweeney, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavassoli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, J. F.</span><span> </span><span class="NLM_article-title">[2,3]-Sigmatropic rearrangements of didehydropiperidinium ylids</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">10113</span><span class="NLM_x">–</span> <span class="NLM_lpage">10126</span><span class="refDoi"> DOI: 10.1016/S0040-4020(02)01397-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2FS0040-4020%2802%2901397-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=10113-10126&author=J.+B.+Sweeneyauthor=A.+Tavassoliauthor=N.+B.+Carterauthor=J.+F.+Hayes&title=%5B2%2C3%5D-Sigmatropic+rearrangements+of+didehydropiperidinium+ylids&doi=10.1016%2FS0040-4020%2802%2901397-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2802%2901397-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252802%252901397-2%26sid%3Dliteratum%253Aachs%26aulast%3DSweeney%26aufirst%3DJ.%2BB.%26aulast%3DTavassoli%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DN.%2BB.%26aulast%3DHayes%26aufirst%3DJ.%2BF.%26atitle%3D%255B2%252C3%255D-Sigmatropic%2520rearrangements%2520of%2520didehydropiperidinium%2520ylids%26jtitle%3DTetrahedron%26date%3D2002%26volume%3D58%26spage%3D10113%26epage%3D10126%26doi%3D10.1016%2FS0040-4020%2802%2901397-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Wilcken, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joerger, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeckler, F. M.</span><span> </span><span class="NLM_article-title">Principles and applications of halogen bonding in medicinal chemistry and chemical biology</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1363</span><span class="NLM_x">–</span> <span class="NLM_lpage">1388</span><span class="refDoi"> DOI: 10.1021/jm3012068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3012068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1363-1388&author=R.+Wilckenauthor=M.+O.+Zimmermannauthor=A.+Langeauthor=A.+C.+Joergerauthor=F.+M.+Boeckler&title=Principles+and+applications+of+halogen+bonding+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Fjm3012068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Wilcken, Rainer; Zimmermann, Markus O.; Lange, Andreas; Joerger, Andreas C.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1363-1388</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Halogen bonding has been known in material science for decades, but until recently, halogen bonds in protein-ligand interactions were largely the result of serendipitous discovery rather than rational design.  In this Perspective, we provide insights into the phenomenon of halogen bonding, with special focus on its role in drug discovery.  We summarize the theor. background defining its strength and directionality, provide a systematic anal. of its occurrence and interaction geometries in protein-ligand complexes, and give recent examples where halogen bonding has been successfully harnessed for lead identification and optimization.  In light of these data, we discuss the potential and limitations of exploiting halogen bonds for mol. recognition and rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjholHfNZ9kbVg90H21EOLACvtfcHk0ljymwr0Q6cf1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ&md5=81ba024e4427fac3e7af0a4cd330909c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm3012068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3012068%26sid%3Dliteratum%253Aachs%26aulast%3DWilcken%26aufirst%3DR.%26aulast%3DZimmermann%26aufirst%3DM.%2BO.%26aulast%3DLange%26aufirst%3DA.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPrinciples%2520and%2520applications%2520of%2520halogen%2520bonding%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1363%26epage%3D1388%26doi%3D10.1021%2Fjm3012068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Yamashita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izumi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashima, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saari, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katsuhata, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwamatsu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hachiya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurata, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, S.</span><span> </span><span class="NLM_article-title">SMG-8 and SMG-9, two novel subunits of the SMG-1 complex, regulate remodeling of the mRNA surveillance complex during nonsense-mediated mRNA decay</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1105</span><span class="refDoi"> DOI: 10.1101/gad.1767209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1101%2Fgad.1767209" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=1091-1105&author=A.+Yamashitaauthor=N.+Izumiauthor=I.+Kashimaauthor=T.+Ohnishiauthor=B.+Saariauthor=Y.+Katsuhataauthor=R.+Muramatsuauthor=T.+Moritaauthor=A.+Iwamatsuauthor=T.+Hachiyaauthor=R.+Kurataauthor=H.+Hiranoauthor=P.+Andersonauthor=S.+Ohno&title=SMG-8+and+SMG-9%2C+two+novel+subunits+of+the+SMG-1+complex%2C+regulate+remodeling+of+the+mRNA+surveillance+complex+during+nonsense-mediated+mRNA+decay&doi=10.1101%2Fgad.1767209"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1101%2Fgad.1767209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1767209%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DIzumi%26aufirst%3DN.%26aulast%3DKashima%26aufirst%3DI.%26aulast%3DOhnishi%26aufirst%3DT.%26aulast%3DSaari%26aufirst%3DB.%26aulast%3DKatsuhata%26aufirst%3DY.%26aulast%3DMuramatsu%26aufirst%3DR.%26aulast%3DMorita%26aufirst%3DT.%26aulast%3DIwamatsu%26aufirst%3DA.%26aulast%3DHachiya%26aufirst%3DT.%26aulast%3DKurata%26aufirst%3DR.%26aulast%3DHirano%26aufirst%3DH.%26aulast%3DAnderson%26aufirst%3DP.%26aulast%3DOhno%26aufirst%3DS.%26atitle%3DSMG-8%2520and%2520SMG-9%252C%2520two%2520novel%2520subunits%2520of%2520the%2520SMG-1%2520complex%252C%2520regulate%2520remodeling%2520of%2520the%2520mRNA%2520surveillance%2520complex%2520during%2520nonsense-mediated%2520mRNA%2520decay%26jtitle%3DGenes%2520Dev.%26date%3D2009%26volume%3D23%26spage%3D1091%26epage%3D1105%26doi%3D10.1101%2Fgad.1767209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Iwatani, M.; Ito, M.; Ishibashi, Y.; Oki, H.; Sakamoto, J.; Tanaka, T.; Morishita, D.; Ito, T.; Kimura, H.; Imaeda, Y.; Aparicio, S.; Nakanishi, A.; Kawamoto, T.</span> Unpublished data. Detailed profiles of <b>52a</b> including selectivity and NMD inhibitory activity will be published separately.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Iwatani%2C+M.%3B+Ito%2C+M.%3B+Ishibashi%2C+Y.%3B+Oki%2C+H.%3B+Sakamoto%2C+J.%3B+Tanaka%2C+T.%3B+Morishita%2C+D.%3B+Ito%2C+T.%3B+Kimura%2C+H.%3B+Imaeda%2C+Y.%3B+Aparicio%2C+S.%3B+Nakanishi%2C+A.%3B+Kawamoto%2C+T.+Unpublished+data.+Detailed+profiles+of+52a+including+selectivity+and+NMD+inhibitory+activity+will+be+published+separately."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DIwatani%26aufirst%3DM." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span> </span><span class="NLM_article-title">Expeditious synthesis of hippuristanol and congeners with potent antiproliferative activities.J</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">10356</span><span class="NLM_x">–</span> <span class="NLM_lpage">10359</span><span class="refDoi"> DOI: 10.1002/chem.200901732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1002%2Fchem.200901732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=19746484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kis7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=10356-10359&author=W.+Liauthor=Y.+Dangauthor=J.+O.+Liuauthor=B.+Yu&title=Expeditious+synthesis+of+hippuristanol+and+congeners+with+potent+antiproliferative+activities.J&doi=10.1002%2Fchem.200901732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Expeditious Synthesis of Hippuristanol and Congeners with Potent Antiproliferative Activities</span></div><div class="casAuthors">Li, Wei; Dang, Yongjun; Liu, Jun O.; Yu, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">10356-10359, S10356/1-S10356/74</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Hippuristanol and its congeners with modified E and F rings are prepd.  The effect of hippuristanol and its analogs on HeLa cell proliferation and in vitro translation was detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBRZBR8sT50rVg90H21EOLACvtfcHk0ljymwr0Q6cf1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kis7fI&md5=4462fbcf3275780e6233cb8d0ed2f814</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fchem.200901732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200901732%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DExpeditious%2520synthesis%2520of%2520hippuristanol%2520and%2520congeners%2520with%2520potent%2520antiproliferative%2520activities.J%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2009%26volume%3D15%26spage%3D10356%26epage%3D10359%26doi%3D10.1002%2Fchem.200901732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Navratilova, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathias, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span> </span><span class="NLM_article-title">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1177/1087057111422746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1177%2F1087057111422746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=22068706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=183-193&author=I.+Navratilovaauthor=G.+Macdonaldauthor=C.+Robinsonauthor=S.+Hughesauthor=J.+Mathiasauthor=C.+Phillipsauthor=A.+Cook&title=Biosensor-based+approach+to+the+identification+of+protein+kinase+ligands+with+dual-site+modes+of+action&doi=10.1177%2F1087057111422746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span></div><div class="casAuthors">Navratilova, Iva; MacDonald, Graeme; Robinson, Colin; Hughes, Samantha; Mathias, John; Phillips, Chris; Cook, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-193</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors have used a surface plasmon resonance (SPR)-based biosensor approach to identify and characterize compds. with a unique binding mode to protein kinases.  Biacore was used to characterize hits from an enzymic high-throughput screen of the Tec family tyrosine kinase, IL2-inducible T cell kinase (ITK).  Complex binding kinetics was obsd. for some compds., which led to identification of compds. that bound simultaneously at both the ATP binding site and a second, allosteric site on ITK.  The presence of the second binding site was confirmed by X-ray crystallog.  The second site is located in the N-terminal lobe of the protein kinase catalytic domain, adjacent to but distinct from the ATP site.  To enable rapid optimization of binding properties, a competition-based Biacore assay has been developed to successfully identify second site noncompetitive binders that have been confirmed by X-ray crystallog. studies.  The authors have found that SPR technol. is a key method for rapid identification of compds. with dual-site modes of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_O18si5LWPLVg90H21EOLACvtfcHk0lg6e_Lc0o5HVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D&md5=8860bb955a27b7ba6b167725224c9e60</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1177%2F1087057111422746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422746%26sid%3Dliteratum%253Aachs%26aulast%3DNavratilova%26aufirst%3DI.%26aulast%3DMacdonald%26aufirst%3DG.%26aulast%3DRobinson%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%26aulast%3DMathias%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DA.%26atitle%3DBiosensor-based%2520approach%2520to%2520the%2520identification%2520of%2520protein%2520kinase%2520ligands%2520with%2520dual-site%2520modes%2520of%2520action%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D183%26epage%3D193%26doi%3D10.1177%2F1087057111422746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 13 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Regina Cencic, Sai Kiran Naineni, Lauren Pugsley, Patrick Senéchal, Ananya Sahni, <span class="NLM_string-name hlFld-ContribAuthor">Jerry Pelletier</span>. </span><span class="cited-content_cbyCitation_article-title">CRISPR-Based Screen Links an Inhibitor of Nonsense-Mediated Decay to eIF4A3 Target Engagement. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (6)
                                     , 1621-1629. <a href="https://doi.org/10.1021/acschembio.0c00253" title="DOI URL">https://doi.org/10.1021/acschembio.0c00253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00253%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DCRISPR-Based%252BScreen%252BLinks%252Ban%252BInhibitor%252Bof%252BNonsense-Mediated%252BDecay%252Bto%252BeIF4A3%252BTarget%252BEngagement%26aulast%3DCencic%26aufirst%3DRegina%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02042020%26date%3D13052020%26date%3D28052020%26date%3D13052020%26volume%3D15%26issue%3D6%26spage%3D1621%26epage%3D1629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Niels Borlinghaus, Sebastian Gergel, <span class="NLM_string-name hlFld-ContribAuthor">Bettina M. Nestl</span>. </span><span class="cited-content_cbyCitation_article-title">Biocatalytic Access to Piperazines from Diamines and Dicarbonyls. </span><span class="cited-content_cbyCitation_journal-name">ACS Catalysis</span><span> <strong>2018,</strong> <em>8 </em>
                                    (4)
                                     , 3727-3732. <a href="https://doi.org/10.1021/acscatal.8b00291" title="DOI URL">https://doi.org/10.1021/acscatal.8b00291</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscatal.8b00291&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscatal.8b00291%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Catalysis%26atitle%3DBiocatalytic%252BAccess%252Bto%252BPiperazines%252Bfrom%252BDiamines%252Band%252BDicarbonyls%26aulast%3DBorlinghaus%26aufirst%3DNiels%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D23012018%26date%3D20032018%26date%3D28032018%26date%3D26032018%26volume%3D8%26issue%3D4%26spage%3D3727%26epage%3D3732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryo  Mizojiri</span>, <span class="hlFld-ContribAuthor ">Daisuke  Nakata</span>, <span class="hlFld-ContribAuthor ">Yoshihiko  Satoh</span>, <span class="hlFld-ContribAuthor ">Daisuke  Morishita</span>, <span class="hlFld-ContribAuthor ">Sachio  Shibata</span>, <span class="hlFld-ContribAuthor ">Misa  Iwatani-Yoshihara</span>, <span class="hlFld-ContribAuthor ">Yohei  Kosugi</span>, <span class="hlFld-ContribAuthor ">Mai  Kosaka</span>, <span class="hlFld-ContribAuthor ">Junpei  Takeda</span>, <span class="hlFld-ContribAuthor ">Shigekazu  Sasaki</span>, <span class="hlFld-ContribAuthor ">Kazuaki  Takami</span>, <span class="hlFld-ContribAuthor ">Koichiro  Fukuda</span>, <span class="hlFld-ContribAuthor ">Masahiro  Kamaura</span>, <span class="hlFld-ContribAuthor ">Shinobu  Sasaki</span>, <span class="hlFld-ContribAuthor ">Ryosuke  Arai</span>, <span class="hlFld-ContribAuthor ">Douglas R.  Cary</span>, and <span class="hlFld-ContribAuthor ">Yasuhiro  Imaeda</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel 5-(Piperazine-1-carbonyl)pyridin-2(1H)-one Derivatives as Orally eIF4A3-Selective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (10)
                                     , 1077-1082. <a href="https://doi.org/10.1021/acsmedchemlett.7b00283" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.7b00283</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.7b00283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.7b00283%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252BNovel%252B5-%252528Piperazine-1-carbonyl%252529pyridin-2%2525281H%252529-one%252BDerivatives%252Bas%252BOrally%252BeIF4A3-Selective%252BInhibitors%26aulast%3DMizojiri%26aufirst%3DRyo%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D13072017%26date%3D08092017%26date%3D14092017%26date%3D12102017%26date%3D08092017%26volume%3D8%26issue%3D10%26spage%3D1077%26epage%3D1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Misa  Iwatani-Yoshihara</span>, <span class="hlFld-ContribAuthor ">Masahiro  Ito</span>, <span class="hlFld-ContribAuthor ">Yoshihiro  Ishibashi</span>, <span class="hlFld-ContribAuthor ">Hideyuki  Oki</span>, <span class="hlFld-ContribAuthor ">Toshio  Tanaka</span>, <span class="hlFld-ContribAuthor ">Daisuke  Morishita</span>, <span class="hlFld-ContribAuthor ">Takashi  Ito</span>, <span class="hlFld-ContribAuthor ">Hiromichi  Kimura</span>, <span class="hlFld-ContribAuthor ">Yasuhiro  Imaeda</span>, <span class="hlFld-ContribAuthor ">Samuel  Aparicio</span>, <span class="hlFld-ContribAuthor ">Atsushi  Nakanishi</span>, and <span class="hlFld-ContribAuthor ">Tomohiro  Kawamoto</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery and Characterization of a Eukaryotic Initiation Factor 4A-3-Selective Inhibitor That Suppresses Nonsense-Mediated mRNA Decay. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2017,</strong> <em>12 </em>
                                    (7)
                                     , 1760-1768. <a href="https://doi.org/10.1021/acschembio.7b00041" title="DOI URL">https://doi.org/10.1021/acschembio.7b00041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.7b00041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.7b00041%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DDiscovery%252Band%252BCharacterization%252Bof%252Ba%252BEukaryotic%252BInitiation%252BFactor%252B4A-3-Selective%252BInhibitor%252BThat%252BSuppresses%252BNonsense-Mediated%252BmRNA%252BDecay%26aulast%3DIwatani-Yoshihara%26aufirst%3DMisa%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D17012017%26date%3D25042017%26date%3D10052017%26date%3D21072017%26date%3D25042017%26volume%3D12%26issue%3D7%26spage%3D1760%26epage%3D1768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junshuang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yang  Jiang</span>, <span class="hlFld-ContribAuthor ">Lian  Chen</span>, <span class="hlFld-ContribAuthor ">Yue  Ma</span>, <span class="hlFld-ContribAuthor ">Haiying  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinpeng  Zhou</span>, <span class="hlFld-ContribAuthor ">Hao  Li</span>, <span class="hlFld-ContribAuthor ">Zhitao  Jing</span>. </span><span class="cited-content_cbyCitation_article-title">The EIF4A3/CASC2/RORA Feedback Loop Regulates the Aggressive Phenotype in Glioblastomas. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.3389/fonc.2021.699933" title="DOI URL">https://doi.org/10.3389/fonc.2021.699933</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2021.699933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2021.699933%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DThe%252BEIF4A3%25252FCASC2%25252FRORA%252BFeedback%252BLoop%252BRegulates%252Bthe%252BAggressive%252BPhenotype%252Bin%252BGlioblastomas%26aulast%3DZhao%26aufirst%3DJunshuang%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Despoina  Sakellariou</span>, <span class="hlFld-ContribAuthor ">Matteo  Tiberti</span>, <span class="hlFld-ContribAuthor ">Thomas H.  Kleiber</span>, <span class="hlFld-ContribAuthor ">Lorea  Blazquez</span>, <span class="hlFld-ContribAuthor ">Aida Rodríguez  López</span>, <span class="hlFld-ContribAuthor ">Marie Holm  Abildgaard</span>, <span class="hlFld-ContribAuthor ">Michal  Lubas</span>, <span class="hlFld-ContribAuthor ">Jiri  Bartek</span>, <span class="hlFld-ContribAuthor ">Elena  Papaleo</span>, <span class="hlFld-ContribAuthor ">Lisa B.  Frankel</span>. </span><span class="cited-content_cbyCitation_article-title">eIF4A3 regulates the TFEB-mediated transcriptional response via GSK3B to control autophagy. </span><span class="cited-content_cbyCitation_journal-name">Cell Death & Differentiation</span><span> <strong>2021,</strong> <em>20 </em><a href="https://doi.org/10.1038/s41418-021-00822-y" title="DOI URL">https://doi.org/10.1038/s41418-021-00822-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41418-021-00822-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41418-021-00822-y%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Death%2520%2526%2520Differentiation%26atitle%3DeIF4A3%252Bregulates%252Bthe%252BTFEB-mediated%252Btranscriptional%252Bresponse%252Bvia%252BGSK3B%252Bto%252Bcontrol%252Bautophagy%26aulast%3DSakellariou%26aufirst%3DDespoina%26date%3D2021%26date%3D2021%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marcel Mirel  Popa</span>, <span class="hlFld-ContribAuthor ">Sergiu  Shova</span>, <span class="hlFld-ContribAuthor ">Madalina  Hrubaru</span>, <span class="hlFld-ContribAuthor ">Loredana  Barbu</span>, <span class="hlFld-ContribAuthor ">Constantin  Draghici</span>, <span class="hlFld-ContribAuthor ">Florea  Dumitrascu</span>, <span class="hlFld-ContribAuthor ">Denisa E.  Dumitrescu</span>. </span><span class="cited-content_cbyCitation_article-title">Introducing chirality in halogenated 3-arylsydnones and their corresponding 1-arylpyrazoles obtained by 1,3-dipolar cycloaddition. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (27)
                                     , 15656-15664. <a href="https://doi.org/10.1039/D0RA02368J" title="DOI URL">https://doi.org/10.1039/D0RA02368J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA02368J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA02368J%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DIntroducing%252Bchirality%252Bin%252Bhalogenated%252B3-arylsydnones%252Band%252Btheir%252Bcorresponding%252B1-arylpyrazoles%252Bobtained%252Bby%252B1%25252C3-dipolar%252Bcycloaddition%26aulast%3DPopa%26aufirst%3DMarcel%2BMirel%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D27%26spage%3D15656%26epage%3D15664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shoichi  Nakao</span>, <span class="hlFld-ContribAuthor ">Masahiro  Nogami</span>, <span class="hlFld-ContribAuthor ">Misa  Iwatani</span>, <span class="hlFld-ContribAuthor ">Toshihiro  Imaeda</span>, <span class="hlFld-ContribAuthor ">Masahiro  Ito</span>, <span class="hlFld-ContribAuthor ">Toshio  Tanaka</span>, <span class="hlFld-ContribAuthor ">Michiko  Tawada</span>, <span class="hlFld-ContribAuthor ">Satoshi  Endo</span>, <span class="hlFld-ContribAuthor ">Douglas R.  Cary</span>, <span class="hlFld-ContribAuthor ">Momoko  Ohori</span>, <span class="hlFld-ContribAuthor ">Yasuhiro  Imaeda</span>, <span class="hlFld-ContribAuthor ">Tomohiro  Kawamoto</span>, <span class="hlFld-ContribAuthor ">Samuel  Aparicio</span>, <span class="hlFld-ContribAuthor ">Atsushi  Nakanishi</span>, <span class="hlFld-ContribAuthor ">Shinsuke  Araki</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a selective DDX3X inhibitor with newly developed quantitative high-throughput RNA helicase assays. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2020,</strong> <em>523 </em>
                                    (3)
                                     , 795-801. <a href="https://doi.org/10.1016/j.bbrc.2019.12.094" title="DOI URL">https://doi.org/10.1016/j.bbrc.2019.12.094</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2019.12.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2019.12.094%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DIdentification%252Bof%252Ba%252Bselective%252BDDX3X%252Binhibitor%252Bwith%252Bnewly%252Bdeveloped%252Bquantitative%252Bhigh-throughput%252BRNA%252Bhelicase%252Bassays%26aulast%3DNakao%26aufirst%3DShoichi%26date%3D2020%26volume%3D523%26issue%3D3%26spage%3D795%26epage%3D801" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alborz  Mazloomian</span>, <span class="hlFld-ContribAuthor ">Shinsuke  Araki</span>, <span class="hlFld-ContribAuthor ">Momoko  Ohori</span>, <span class="hlFld-ContribAuthor ">Amal M.  El-Naggar</span>, <span class="hlFld-ContribAuthor ">Damian  Yap</span>, <span class="hlFld-ContribAuthor ">Ali  Bashashati</span>, <span class="hlFld-ContribAuthor ">Shoichi  Nakao</span>, <span class="hlFld-ContribAuthor ">Poul H.  Sorensen</span>, <span class="hlFld-ContribAuthor ">Atsushi  Nakanishi</span>, <span class="hlFld-ContribAuthor ">Sohrab  Shah</span>, <span class="hlFld-ContribAuthor ">Samuel  Aparicio</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological systems analysis defines EIF4A3 functions in cell-cycle and RNA stress granule formation. </span><span class="cited-content_cbyCitation_journal-name">Communications Biology</span><span> <strong>2019,</strong> <em>2 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42003-019-0391-9" title="DOI URL">https://doi.org/10.1038/s42003-019-0391-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42003-019-0391-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42003-019-0391-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Biology%26atitle%3DPharmacological%252Bsystems%252Banalysis%252Bdefines%252BEIF4A3%252Bfunctions%252Bin%252Bcell-cycle%252Band%252BRNA%252Bstress%252Bgranule%252Bformation%26aulast%3DMazloomian%26aufirst%3DAlborz%26date%3D2019%26date%3D2019%26volume%3D2%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krister J  Barkovich</span>, <span class="hlFld-ContribAuthor ">Megan K  Moore</span>, <span class="hlFld-ContribAuthor ">Qi  Hu</span>, <span class="hlFld-ContribAuthor ">Kevan M  Shokat</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical genetic inhibition of DEAD-box proteins using covalent complementarity. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2018,</strong> <em>46 </em>
                                    (17)
                                     , 8689-8699. <a href="https://doi.org/10.1093/nar/gky706" title="DOI URL">https://doi.org/10.1093/nar/gky706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gky706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgky706%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DChemical%252Bgenetic%252Binhibition%252Bof%252BDEAD-box%252Bproteins%252Busing%252Bcovalent%252Bcomplementarity%26aulast%3DBarkovich%26aufirst%3DKrister%2BJ%26date%3D2018%26date%3D2018%26volume%3D46%26issue%3D17%26spage%3D8689%26epage%3D8699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jennifer  Chu</span>, <span class="hlFld-ContribAuthor ">Jerry  Pelletier</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical and CRISPR/Cas9 Tools for Functional Characterization of RNA Helicases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 221-245. <a href="https://doi.org/10.1007/978-981-10-8372-3_14" title="DOI URL">https://doi.org/10.1007/978-981-10-8372-3_14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-10-8372-3_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-10-8372-3_14%26sid%3Dliteratum%253Aachs%26atitle%3DChemical%252Band%252BCRISPR%25252FCas9%252BTools%252Bfor%252BFunctional%252BCharacterization%252Bof%252BRNA%252BHelicases%26aulast%3DChu%26aufirst%3DJennifer%26date%3D2018%26date%3D2018%26spage%3D221%26epage%3D245%26pub%3DSpringer%2520Singapore%26atitle%3DApplied%252BRNA%252BBioscience%26aulast%3DMasuda%26aufirst%3DSeiji%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Masahiro  Ito</span>, <span class="hlFld-ContribAuthor ">Misa  Iwatani</span>, <span class="hlFld-ContribAuthor ">Takeshi  Yamamoto</span>, <span class="hlFld-ContribAuthor ">Toshio  Tanaka</span>, <span class="hlFld-ContribAuthor ">Tomohiro  Kawamoto</span>, <span class="hlFld-ContribAuthor ">Daisuke  Morishita</span>, <span class="hlFld-ContribAuthor ">Atsushi  Nakanishi</span>, <span class="hlFld-ContribAuthor ">Hironobu  Maezaki</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of spiro[indole-3,2′-pyrrolidin]-2(1H)-one based inhibitors targeting Brr2, a core component of the U5 snRNP. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2017,</strong> <em>25 </em>
                                    (17)
                                     , 4753-4767. <a href="https://doi.org/10.1016/j.bmc.2017.07.017" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.07.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.07.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.07.017%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bspiro%25255Bindole-3%25252C2%2525E2%252580%2525B2-pyrrolidin%25255D-2%2525281H%252529-one%252Bbased%252Binhibitors%252Btargeting%252BBrr2%25252C%252Ba%252Bcore%252Bcomponent%252Bof%252Bthe%252BU5%252BsnRNP%26aulast%3DIto%26aufirst%3DMasahiro%26date%3D2017%26volume%3D25%26issue%3D17%26spage%3D4753%26epage%3D4767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_null" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0017.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of eIF4A3 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of <b>7a</b>–<b>d</b> and <b>8a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Boc<sub>2</sub>O, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0–15 °C, 66–83%; (b) Boc<sub>2</sub>O, THF, toluene, 0 °C to rt, 70%; (c) 4-chlorobenzoyl chloride or 4-bromobenzoyl chloride, Et<sub>3</sub>N or DIPEA, CH<sub>2</sub>Cl<sub>2</sub>, 0–15 °C, 78–95%; (d) 4 M HCl in EtOAc, 0 °C or rt, 66% to quant; (e) (i) aq NaHCO<sub>3</sub>, rt, 97%; (ii) chiral HPLC, 41–42%.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <b>3</b>, <b>29</b>–<b>51</b>, <b>52a</b>,<b>b</b>, <b>53a</b>,<b>b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>9</b>–<b>10</b>, <b>12</b>, <b>14</b>, <b>15</b>, <b>18</b>, <b>20</b>, or <b>21</b>, HATU, DIPEA, DMF, rt, 24–61%; (b) <b>11</b>, <b>13</b>, <b>21</b>, or <b>25</b>–<b>26</b>, EDC·HCl, HOBt, DIPEA, DMF, rt, 34–79%; (c) (i) <b>16</b> or <b>19</b>, LiOH·H<sub>2</sub>O, THF, MeOH, H<sub>2</sub>O, rt; (ii) HATU, DIPEA, DMF, rt, 17–71%; (d) (i) <b>17</b>, TFA, CH<sub>2</sub>Cl<sub>2</sub>, rt; (ii) HATU, Et<sub>3</sub>N, DMF, rt, 53%; (e) (i) aq NaHCO<sub>3</sub>, rt; (ii) <b>24</b> or <b>27</b>, (Me<sub>3</sub>Al)<sub>2</sub>·DABCO,<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> THF, microwave, 130 °C, 47–78%; (f) (i) aq NaHCO<sub>3</sub>, rt; (ii) <b>28</b>, Me<sub>3</sub>Al, THF, microwave, 110 °C, 55%; (g) <b>13</b> or <b>26</b>, EDC·HCl, HOBt, DMF, rt, 50–81%; (h) (i) <b>22</b>, EDC·HCl, HOBt, DMF, rt, 25%; (ii) 2-bromobenzonitrile, Pd(OAc)<sub>2</sub>, CuI, Cs<sub>2</sub>CO<sub>3</sub>, DMF, microwave, 150 °C, 16%; (i) (i) <b>23</b>, HATU, DIPEA, DMF, 0 °C, 95%; (ii) 3-cyanophenylboronic acid or 4-cyanophenylboronic acid, Na<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, DME, H<sub>2</sub>O, reflux, 30–49%.</p></p></figure><figure data-id="sch3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <b>55</b>–<b>60</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) <b>21</b>, HATU, DIPEA, DMF, rt, 85%; (b) (i) BH<sub>3</sub>·Me<sub>2</sub>S, THF, 0 °C to rt, 31%; (ii) 4-chlorobenzoyl chloride, K<sub>2</sub>CO<sub>3</sub>, MeCN, rt, 56%; (c) 4-chlorobenzyl bromide, K<sub>2</sub>CO<sub>3</sub>, MeCN, 80 °C, 19%; (d) corresponding acyl chloride (3-chlorobenzoyl chloride for <b>57</b>, 2-chlorobenzoyl chloride for <b>58</b>, and benzoyl chloride for <b>59</b>), Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 41–46%; (e) 4-bromobenzoic acid, EDC, HOBt, DIPEA, THF, rt, 96%.</p></p></figure><figure data-id="sch4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0009.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of <b>63</b>, <b>65</b>, and <b>68</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) 4-chlorobenzoyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt; (ii) <b>62</b>, NaH, DMF, 0 °C, 12% for 2 steps; (b) (i) Boc<sub>2</sub>O, Et<sub>3</sub>N, THF, 0 °C to rt, 73%; (ii) 4-chlorobenzoyl chloride, DIPEA, THF, 0 °C, 99%; (iii) 4 M HCl in EtOAc, rt, 91%; (iv) <b>21</b>, EDC·HCl, HOBt, DIPEA, DMF, 0 °C to rt, 24%; (c) (i) 1,3-propanediamine, MeONa, MeOH, rt to 80 °C, 23%; (ii) BH<sub>3</sub>·Me<sub>2</sub>S, THF, 0–80 °C; (iii) Boc<sub>2</sub>O, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, 0 °C to rt, 72% for 2 steps; (d) (i) 4-chlorobenzoyl chloride, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 93%; (ii) 4 M HCl in dioxane, rt; (iii) <b>21</b>, HATU, Et<sub>3</sub>N, CH<sub>2</sub>Cl<sub>2</sub>, rt, 34% for 2 steps.</p></p></figure><figure data-id="fig2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design strategy around compound <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. NMD inhibition by <b>53a</b> and <b>53b</b> in the reporter gene assay. HEK293T cells transfected with the NMD reporter gene were treated with the indicated compounds at 10 μM or 3 μM for 6 h. An increase in the ratio of firefly to renilla luciferase activity, indicating NMD inhibition, was detected and normalized with DMSO treated samples. The <i>y</i>-axis in the bar graph represents the fold increase in the ratio of firefly to renilla luciferase activity. <i>n</i> = 2, and error bars represent the SD values.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. SPR binding studies of <b>2</b> against eIF4A3 in the presence or absence of <b>52a</b>. The binding signals of compound <b>2</b> against immobilized eIF4A3 were detected in the presence of <b>52a</b> at a saturated concentration (A–C) or absence of <b>52a</b> (D–F). Parts A and D represent the SPR response to varying concentrations of <b>2</b>. <i>y</i> axis is SPR response (RU), and <i>x</i> axis is concentration of <b>2</b>. Parts B and E represent the SPR sensorgrams. <i>y</i> axis is SPR response, and <i>x</i> axis is time. Parts C and F represent the schematics of the experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/medium/jm-2016-01904p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0005.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. SPR binding studies of <b>1</b> against eIF4A3 in the presence or absence of <b>52a</b>. The binding signals of compound <b>1</b> against immobilized eIF4A3 were detected in the presence of <b>52a</b> at a saturated concentration (A–C) or absence of <b>52a</b> (D–F). Parts A and D represent the SPR response to varying concentrations of <b>1</b>. <i>y</i> axis is SPR response (RU), and <i>x</i> axis is concentration of <b>1</b>. Parts B and E represent the SPR sensorgrams. The <i>y</i> axis is the SPR response, and the <i>x</i> axis is time. Parts C and F represent the schematics of the experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-8/acs.jmedchem.6b01904/20170421/images/large/jm-2016-01904p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01904&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82023" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82023" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 39 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Linder, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jankowsky, E.</span><span> </span><span class="NLM_article-title">From unwinding to clamping — the DEAD box RNA helicase family</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">516</span><span class="refDoi"> DOI: 10.1038/nrm3154</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnrm3154" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=21779027" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpt1aqtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2011&pages=505-516&author=P.+Linderauthor=E.+Jankowsky&title=From+unwinding+to+clamping+%E2%80%94+the+DEAD+box+RNA+helicase+family&doi=10.1038%2Fnrm3154"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">From unwinding to clamping - the DEAD box RNA helicase family</span></div><div class="casAuthors">Linder, Patrick; Jankowsky, Eckhard</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">505-516</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  RNA helicases of the DEAD box family are present in all eukaryotic cells and in many bacteria and archaea.  These highly conserved enzymes are required for RNA metab. from transcription to degrdn. and are therefore important players in gene expression.  DEAD box proteins use ATP to unwind short duplex RNA in an unusual fashion and remodel RNA-protein complexes, but they can also function as ATP-dependent RNA clamps to provide nucleation centers that establish larger RNA-protein complexes.  Structural, mechanistic, and mol. biol. studies have started to reveal how these conserved proteins can perform such diverse functions and how accessory proteins have a central role in their regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7W4tiGRbeKLVg90H21EOLACvtfcHk0lh2-RpGV15fkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpt1aqtL4%253D&md5=66620c984f8c51c587f9988e774c1f21</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm3154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3154%26sid%3Dliteratum%253Aachs%26aulast%3DLinder%26aufirst%3DP.%26aulast%3DJankowsky%26aufirst%3DE.%26atitle%3DFrom%2520unwinding%2520to%2520clamping%2520%25E2%2580%2594%2520the%2520DEAD%2520box%2520RNA%2520helicase%2520family%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2011%26volume%3D12%26spage%3D505%26epage%3D516%26doi%3D10.1038%2Fnrm3154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Tanner, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linder, P.</span><span> </span><span class="NLM_article-title">DExD/H box RNA helicases: from generic motors to specific dissociation functions</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">251</span><span class="NLM_x">–</span> <span class="NLM_lpage">262</span><span class="refDoi"> DOI: 10.1016/S1097-2765(01)00329-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2FS1097-2765%2801%2900329-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=11545728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2001&pages=251-262&author=N.+K.+Tannerauthor=P.+Linder&title=DExD%2FH+box+RNA+helicases%3A+from+generic+motors+to+specific+dissociation+functions&doi=10.1016%2FS1097-2765%2801%2900329-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">DExD/H box RNA helicases: from generic motors to specific dissociation functions</span></div><div class="casAuthors">Tanner, N. Kyle; Linder, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">251-262</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review with refs.  RNA helicases of the DEAD box and related DExD/H proteins form a very large superfamily of proteins conserved from bacteria and viruses to humans.  They have seven to eight conserved motifs, the characteristics of which are used to subgroup members into individual families.  They are assocd. with all processes involving RNA mols., including transcription, editing, splicing, ribosome biogenesis, RNA export, translation, RNA turnover, and organelle gene expression.  Anal. of the three-dimensional structures obtained through the crystn. of viral and cellular RNA helicases reveals a strong structural homol. to DNA helicases.  In this review, we discuss our current understanding of RNA helicases and their biol. function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFLDauWWZ4c7Vg90H21EOLACvtfcHk0lh2-RpGV15fkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXmvFWgsbw%253D&md5=90a403e4bd3c295f5198dd1bcc601ab5</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2801%2900329-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252801%252900329-X%26sid%3Dliteratum%253Aachs%26aulast%3DTanner%26aufirst%3DN.%2BK.%26aulast%3DLinder%26aufirst%3DP.%26atitle%3DDExD%252FH%2520box%2520RNA%2520helicases%253A%2520from%2520generic%2520motors%2520to%2520specific%2520dissociation%2520functions%26jtitle%3DMol.%2520Cell%26date%3D2001%26volume%3D8%26spage%3D251%26epage%3D262%26doi%3D10.1016%2FS1097-2765%2801%2900329-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Jankowsky, E.</span><span> </span><span class="NLM_article-title">RNA Helicases at work: binding and rearranging</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">19</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.tibs.2010.07.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.tibs.2010.07.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=20813532" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKgtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=19-29&author=E.+Jankowsky&title=RNA+Helicases+at+work%3A+binding+and+rearranging&doi=10.1016%2Fj.tibs.2010.07.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">RNA helicases at work: binding and rearranging</span></div><div class="casAuthors">Jankowsky, Eckhard</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-29</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  RNA helicases are ubiquitous, highly conserved enzymes that participate in nearly all aspects of RNA metab.  These proteins bind or remodel RNA or RNA-protein complexes in an ATP-dependent fashion.  How RNA helicases phys. perform their cellular tasks has been a longstanding question, but in recent years, intriguing models have started to link structure, mechanism, and biol. function for some RNA helicases.  Here, the author outlines the current view on major structural and mechanistic themes of RNA helicase function, and on emerging phys. models for the cellular roles of these enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-XYT4X_o2lLVg90H21EOLACvtfcHk0ljzYUq-fCV67g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKgtg%253D%253D&md5=e7ed17c0d999efea7833a1548e0f80fa</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.07.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.07.008%26sid%3Dliteratum%253Aachs%26aulast%3DJankowsky%26aufirst%3DE.%26atitle%3DRNA%2520Helicases%2520at%2520work%253A%2520binding%2520and%2520rearranging%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D19%26epage%3D29%26doi%3D10.1016%2Fj.tibs.2010.07.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Jarmoskaite, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, R.</span><span> </span><span class="NLM_article-title">RNA helicase proteins as chaperones and remodelers</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">83</span><span class="NLM_x">, </span> <span class="NLM_fpage">697</span><span class="NLM_x">–</span> <span class="NLM_lpage">725</span><span class="refDoi"> DOI: 10.1146/annurev-biochem-060713-035546</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1146%2Fannurev-biochem-060713-035546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=24635478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFOhtrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2014&pages=697-725&author=I.+Jarmoskaiteauthor=R.+Russell&title=RNA+helicase+proteins+as+chaperones+and+remodelers&doi=10.1146%2Fannurev-biochem-060713-035546"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">RNA helicase proteins as chaperones and remodelers</span></div><div class="casAuthors">Jarmoskaite, Inga; Russell, Rick</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">697-725</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Superfamily 2 helicase proteins are ubiquitous in RNA biol. and have an extraordinarily broad set of functional roles.  Central among these roles are the promotion of rearrangements of structured RNAs and the remodeling of ribonucleoprotein complexes (RNPs), allowing formation of native RNA structure or progression through a functional cycle of structures.  Although all superfamily 2 helicases share a conserved helicase core, they are divided evolutionarily into several families, and it is principally proteins from three families, the DEAD-box, DEAH/RHA, and Ski2-like families, that function to manipulate structured RNAs and RNPs.  Strikingly, there are emerging differences in the mechanisms of these proteins, both between families and within the largest family (DEAD-box), and these differences appear to be tuned to their RNA or RNP substrates and their specific roles.  This review outlines basic mechanistic features of the three families and surveys individual proteins and the current understanding of their biol. substrates and mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0VcunkVppbbVg90H21EOLACvtfcHk0ljzYUq-fCV67g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFOhtrnL&md5=6031909d6b1762b3d7955d5ce0af7b58</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060713-035546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060713-035546%26sid%3Dliteratum%253Aachs%26aulast%3DJarmoskaite%26aufirst%3DI.%26aulast%3DRussell%26aufirst%3DR.%26atitle%3DRNA%2520helicase%2520proteins%2520as%2520chaperones%2520and%2520remodelers%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2014%26volume%3D83%26spage%3D697%26epage%3D725%26doi%3D10.1146%2Fannurev-biochem-060713-035546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Rocak, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Linder, P.</span><span> </span><span class="NLM_article-title">DEAD-box proteins: the driving forces behind RNA metabolism</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">232</span><span class="NLM_x">–</span> <span class="NLM_lpage">241</span><span class="refDoi"> DOI: 10.1038/nrm1335</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnrm1335" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=14991003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhs1CltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2004&pages=232-241&author=S.+Rocakauthor=P.+Linder&title=DEAD-box+proteins%3A+the+driving+forces+behind+RNA+metabolism&doi=10.1038%2Fnrm1335"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">DEAD-box proteins: the driving forces behind RNA metabolism</span></div><div class="casAuthors">Rocak, Sanda; Linder, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">232-241</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  RNA helicases from the DEAD-box family are found in almost all organisms and play important roles in RNA metab.  They are assocd. with many processes ranging from RNA synthesis to RNA degrdn.  DEAD-box proteins use the energy from ATP hydrolysis to rearrange intermol. or intramol. RNA structures, or dissoc. RNA-protein complexes.  Such dynamic rearrangements are fundamental for many, if not all, steps in the life of an RNA mol.  Recent biochem., genetic, and structural data shed light on how these proteins power the metab. of RNA within a cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxJcMxFGudxrVg90H21EOLACvtfcHk0ljV8otse8DAeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhs1CltL8%253D&md5=85203c8f3ef20f4896086036382bce61</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrm1335&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1335%26sid%3Dliteratum%253Aachs%26aulast%3DRocak%26aufirst%3DS.%26aulast%3DLinder%26aufirst%3DP.%26atitle%3DDEAD-box%2520proteins%253A%2520the%2520driving%2520forces%2520behind%2520RNA%2520metabolism%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2004%26volume%3D5%26spage%3D232%26epage%3D241%26doi%3D10.1038%2Fnrm1335" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Saulière, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murigneux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquenet, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbosa, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Tonquèze, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audic, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paillard, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crollius, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Hir, H.</span><span> </span><span class="NLM_article-title">CLIP-seq of eIF4AIII reveals transcriptome-wide mapping of the human exon junction complex</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1131</span><span class="refDoi"> DOI: 10.1038/nsmb.2420</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnsmb.2420" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=23085716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFeqsrjE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1124-1131&author=J.+Sauli%C3%A8reauthor=V.+Murigneuxauthor=Z.+Wangauthor=E.+Marquenetauthor=I.+Barbosaauthor=O.+Le+Tonqu%C3%A8zeauthor=Y.+Audicauthor=L.+Paillardauthor=H.+R.+Crolliusauthor=H.+Le+Hir&title=CLIP-seq+of+eIF4AIII+reveals+transcriptome-wide+mapping+of+the+human+exon+junction+complex&doi=10.1038%2Fnsmb.2420"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">CLIP-seq of eIF4AIII reveals transcriptome-wide mapping of the human exon junction complex</span></div><div class="casAuthors">Sauliere, Jerome; Murigneux, Valentine; Wang, Zhen; Marquenet, Emelie; Barbosa, Isabelle; Le Tonqueze, Olivier; Audic, Yann; Paillard, Luc; Crollius, Hugues Roest; Le Hir, Herve</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1124-1131</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The exon junction complex (EJC) is a central effector of the fate of mRNAs, linking nuclear processing to mRNA transport, translation and surveillance.  However, little is known about its transcriptome-wide targets.  We used crosslinking and immunopptn. methods coupled to high-throughput sequencing (CLIP-seq) in human cells to identify the binding sites of the DEAD-box helicase eIF4AIII, an EJC core component.  CLIP reads form peaks that are located mainly in spliced mRNAs.  Most expressed exons harbor peaks either in the canonical EJC region, located ∼24 nucleotides upstream of exonic junctions, or in other noncanonical regions.  Notably, both of these types of peaks are preferentially assocd. with unstructured and purine-rich sequences contg. the motif GAAGA, which is a potential binding site for EJC-assocd. factors.  Therefore, EJC positions vary spatially and quant. between exons.  This transcriptome-wide mapping of human eIF4AIII reveals unanticipated aspects of the EJC and broadens its potential impact on post-transcriptional regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmnvsbuadcFbVg90H21EOLACvtfcHk0ljV8otse8DAeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFeqsrjE&md5=6483cf0baffbe1af6f26c339a22f9ba9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.2420&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.2420%26sid%3Dliteratum%253Aachs%26aulast%3DSauli%25C3%25A8re%26aufirst%3DJ.%26aulast%3DMurigneux%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMarquenet%26aufirst%3DE.%26aulast%3DBarbosa%26aufirst%3DI.%26aulast%3DLe%2BTonqu%25C3%25A8ze%26aufirst%3DO.%26aulast%3DAudic%26aufirst%3DY.%26aulast%3DPaillard%26aufirst%3DL.%26aulast%3DCrollius%26aufirst%3DH.%2BR.%26aulast%3DLe%2BHir%26aufirst%3DH.%26atitle%3DCLIP-seq%2520of%2520eIF4AIII%2520reveals%2520transcriptome-wide%2520mapping%2520of%2520the%2520human%2520exon%2520junction%2520complex%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1124%26epage%3D1131%26doi%3D10.1038%2Fnsmb.2420" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Le Hir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andersen, G. R.</span><span> </span><span class="NLM_article-title">Structural insights into the exon junction complex</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">112</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1016/j.sbi.2007.11.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.sbi.2007.11.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=18164611" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVGitrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=112-119&author=H.+Le+Hirauthor=G.+R.+Andersen&title=Structural+insights+into+the+exon+junction+complex&doi=10.1016%2Fj.sbi.2007.11.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the exon junction complex</span></div><div class="casAuthors">Le Hir, Herve; Andersen, Gregers Rom</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-119</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In higher eukaryotes, the exon junction complex is loaded onto spliced mRNAs at a precise position upstream of exon junctions, where it remains during nuclear export and cytoplasmic localization until it is removed during the first translation round.  The exon junction core complex consists of four proteins that form a dynamic binding platform for a variety of peripheral factors involved in mRNA metab.  In the complex, mRNA binding is mediated by the DEAD-box protein eIF4AIII, and inhibition of its ATPase activity forms the mechanistic basis for the long-term stability of the complex.  Recent crystal structures of the exon junction complex and eIF4AIII have provided the structural framework for investigating the function of the eIF4AIII ATPase and for localization of surface patches involved in binding peripheral factors.  Addnl., by comparison with the structure of a second DEAD-box protein also bound to RNA and ATP, general principles for the ATPase and unwinding/mRNP remodeling activities for this important group of enzymes can be proposed on the basis of at. structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoV9L-Nb_8PHLVg90H21EOLACvtfcHk0ljV8otse8DAeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVGitrs%253D&md5=efeb06df610d55dc6b155f92a234dbbe</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2007.11.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2007.11.002%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BHir%26aufirst%3DH.%26aulast%3DAndersen%26aufirst%3DG.%2BR.%26atitle%3DStructural%2520insights%2520into%2520the%2520exon%2520junction%2520complex%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D18%26spage%3D112%26epage%3D119%26doi%3D10.1016%2Fj.sbi.2007.11.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Palacios, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatfield, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St Johnston, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izaurralde, E.</span><span> </span><span class="NLM_article-title">An eIF4AIII-containing complex required for mRNA localization and nonsense-mediated mRNA decay</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">427</span><span class="NLM_x">, </span> <span class="NLM_fpage">753</span><span class="NLM_x">–</span> <span class="NLM_lpage">757</span><span class="refDoi"> DOI: 10.1038/nature02351</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnature02351" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=427&publication_year=2004&pages=753-757&author=I.+M.+Palaciosauthor=D.+Gatfieldauthor=D.+St+Johnstonauthor=E.+Izaurralde&title=An+eIF4AIII-containing+complex+required+for+mRNA+localization+and+nonsense-mediated+mRNA+decay&doi=10.1038%2Fnature02351"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnature02351&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature02351%26sid%3Dliteratum%253Aachs%26aulast%3DPalacios%26aufirst%3DI.%2BM.%26aulast%3DGatfield%26aufirst%3DD.%26aulast%3DSt%2BJohnston%26aufirst%3DD.%26aulast%3DIzaurralde%26aufirst%3DE.%26atitle%3DAn%2520eIF4AIII-containing%2520complex%2520required%2520for%2520mRNA%2520localization%2520and%2520nonsense-mediated%2520mRNA%2520decay%26jtitle%3DNature%26date%3D2004%26volume%3D427%26spage%3D753%26epage%3D757%26doi%3D10.1038%2Fnature02351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ferraiuolo, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ler, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa-Mattioli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonenberg, N.</span><span> </span><span class="NLM_article-title">A nuclear translation-like factor eIF4AIII is recruited to the mRNA during splicing and functions in nonsense-mediated decay</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">4118</span><span class="NLM_x">–</span> <span class="NLM_lpage">4123</span><span class="refDoi"> DOI: 10.1073/pnas.0400933101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1073%2Fpnas.0400933101" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2004&pages=4118-4123&author=M.+A.+Ferraiuoloauthor=C.-S.+Leeauthor=L.+W.+Lerauthor=J.+L.+Hsuauthor=M.+Costa-Mattioliauthor=M.-J.+Luoauthor=R.+Reedauthor=N.+Sonenberg&title=A+nuclear+translation-like+factor+eIF4AIII+is+recruited+to+the+mRNA+during+splicing+and+functions+in+nonsense-mediated+decay&doi=10.1073%2Fpnas.0400933101"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0400933101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0400933101%26sid%3Dliteratum%253Aachs%26aulast%3DFerraiuolo%26aufirst%3DM.%2BA.%26aulast%3DLee%26aufirst%3DC.-S.%26aulast%3DLer%26aufirst%3DL.%2BW.%26aulast%3DHsu%26aufirst%3DJ.%2BL.%26aulast%3DCosta-Mattioli%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DM.-J.%26aulast%3DReed%26aufirst%3DR.%26aulast%3DSonenberg%26aufirst%3DN.%26atitle%3DA%2520nuclear%2520translation-like%2520factor%2520eIF4AIII%2520is%2520recruited%2520to%2520the%2520mRNA%2520during%2520splicing%2520and%2520functions%2520in%2520nonsense-mediated%2520decay%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2004%26volume%3D101%26spage%3D4118%26epage%3D4123%26doi%3D10.1073%2Fpnas.0400933101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Shibuya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tange, T. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonenberg, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, M. J.</span><span> </span><span class="NLM_article-title">eIF4AIII binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay</span> <span class="citation_source-journal">Nat. Struct. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">346</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span><span class="refDoi"> DOI: 10.1038/nsmb750</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnsmb750" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=15034551" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFCiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=346-351&author=T.+Shibuyaauthor=T.+O.+Tangeauthor=N.+Sonenbergauthor=M.+J.+Moore&title=eIF4AIII+binds+spliced+mRNA+in+the+exon+junction+complex+and+is+essential+for+nonsense-mediated+decay&doi=10.1038%2Fnsmb750"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">eIF4AIII binds spliced mRNA in the exon junction complex and is essential for nonsense-mediated decay</span></div><div class="casAuthors">Shibuya, Toshiharu; Tange, Thomas O.; Sonenberg, Nahum; Moore, Melissa J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">346-351</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The exon junction complex (EJC), a set of proteins deposited on mRNAs as a consequence of pre-mRNA splicing, is a key effector of downstream mRNA metab.  We have identified eIF4AIII, a member of the eukaryotic translation initiation factor 4A family of RNA helicases (also known as DExH/D box proteins), as a novel EJC core component.  Crosslinking and antibody inhibition studies suggest that eIF4AIII constitutes at least part of the platform anchoring other EJC components to spliced mRNAs.  A nucleocytoplasmic shuttling protein, eIF4AIII assocs. in vitro and in vivo with two other EJC core factors, Y14 and Magoh.  In mammalian cells, eIF4AIII is essential for nonsense-mediated mRNA decay (NMD).  Finally, a model is proposed by which eIF4AIII represents a new functional class of DExH/D box proteins that act as RNA clamps or 'place holders' for the sequence-independent attachment of addnl. factors to RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotzxv4GWKKmrVg90H21EOLACvtfcHk0lgLOdjcSbQJ4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXisFCiurw%253D&md5=08c7904741747af05644798f28159f7f</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnsmb750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb750%26sid%3Dliteratum%253Aachs%26aulast%3DShibuya%26aufirst%3DT.%26aulast%3DTange%26aufirst%3DT.%2BO.%26aulast%3DSonenberg%26aufirst%3DN.%26aulast%3DMoore%26aufirst%3DM.%2BJ.%26atitle%3DeIF4AIII%2520binds%2520spliced%2520mRNA%2520in%2520the%2520exon%2520junction%2520complex%2520and%2520is%2520essential%2520for%2520nonsense-mediated%2520decay%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D346%26epage%3D351%26doi%3D10.1038%2Fnsmb750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Le Hir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Séraphin, B.</span><span> </span><span class="NLM_article-title">EJCs at the heart of translational control</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">133</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">216</span><span class="refDoi"> DOI: 10.1016/j.cell.2008.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.cell.2008.04.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2008&pages=213-216&author=H.+Le+Hirauthor=B.+S%C3%A9raphin&title=EJCs+at+the+heart+of+translational+control&doi=10.1016%2Fj.cell.2008.04.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2008.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2008.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DLe%2BHir%26aufirst%3DH.%26aulast%3DS%25C3%25A9raphin%26aufirst%3DB.%26atitle%3DEJCs%2520at%2520the%2520heart%2520of%2520translational%2520control%26jtitle%3DCell%26date%3D2008%26volume%3D133%26spage%3D213%26epage%3D216%26doi%3D10.1016%2Fj.cell.2008.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Reed, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurt, E.</span><span> </span><span class="NLM_article-title">A conserved mRNA export machinery coupled to pre-mRNA splicing</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">523</span><span class="NLM_x">–</span> <span class="NLM_lpage">531</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00627-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2FS0092-8674%2802%2900627-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2002&pages=523-531&author=R.+Reedauthor=E.+Hurt&title=A+conserved+mRNA+export+machinery+coupled+to+pre-mRNA+splicing&doi=10.1016%2FS0092-8674%2802%2900627-X"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900627-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900627-X%26sid%3Dliteratum%253Aachs%26aulast%3DReed%26aufirst%3DR.%26aulast%3DHurt%26aufirst%3DE.%26atitle%3DA%2520conserved%2520mRNA%2520export%2520machinery%2520coupled%2520to%2520pre-mRNA%2520splicing%26jtitle%3DCell%26date%3D2002%26volume%3D108%26spage%3D523%26epage%3D531%26doi%3D10.1016%2FS0092-8674%2802%2900627-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Michelle, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cloutier, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toutant, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkreta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thibault, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Durand, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garneau, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gendron, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapointe, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Hir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinck, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elela, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prinos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chabot, B.</span><span> </span><span class="NLM_article-title">Proteins associated with the exon junction complex also control the alternative splicing of apoptotic regulators</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">954</span><span class="NLM_x">–</span> <span class="NLM_lpage">967</span><span class="refDoi"> DOI: 10.1128/MCB.06130-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1128%2FMCB.06130-11" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=22203037" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtV2ju7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2012&pages=954-967&author=L.+Michelleauthor=A.+Cloutierauthor=J.+Toutantauthor=L.+Shkretaauthor=P.+Thibaultauthor=M.+Durandauthor=D.+Garneauauthor=D.+Gendronauthor=E.+Lapointeauthor=S.+Coutureauthor=H.+Le+Hirauthor=R.+Klinckauthor=S.+A.+Elelaauthor=P.+Prinosauthor=B.+Chabot&title=Proteins+associated+with+the+exon+junction+complex+also+control+the+alternative+splicing+of+apoptotic+regulators&doi=10.1128%2FMCB.06130-11"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Proteins associated with the exon junction complex also control the alternative splicing of apoptotic regulators</span></div><div class="casAuthors">Michelle, Laetitia; Cloutier, Alexandre; Toutant, Johanne; Shkreta, Lulzim; Thibault, Philippe; Durand, Mathieu; Garneau, Daniel; Gendron, Daniel; Lapointe, Elvy; Couture, Sonia; Le Hir, Herve; Klinck, Roscoe; Abou Elela, Sherif; Prinos, Panagiotis; Chabot, Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">954-967</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Several apoptotic regulators, including Bcl-x, are alternatively spliced to produce isoforms with opposite functions.  We have used an RNA interference strategy to map the regulatory landscape controlling the expression of the Bcl-x splice variants in human cells.  Depleting proteins known as core (Y14 and eIF4A3) or auxiliary (RNPS1, Acinus, and SAP18) components of the exon junction complex (EJC) improved the prodn. of the proapoptotic Bcl-xS splice variant.  This effect was not seen when we depleted EJC proteins that typically participate in mRNA export (UAP56, Aly/Ref, and TAP) or that assoc. with the EJC to enforce nonsense-mediated RNA decay (MNL51, Upf1, Upf2, and Upf3b).  Core and auxiliary EJC components modulated Bcl-x splicing through different cis-acting elements, further suggesting that this activity is distinct from the established EJC function.  In support of a direct role in splicing control, recombinant eIF4A3, Y14, and Magoh proteins assocd. preferentially with the endogenous Bcl-x pre-mRNA, interacted with a model Bcl-x pre-mRNA in early splicing complexes, and specifically shifted Bcl-x alternative splicing in nuclear exts.  Finally, the depletion of Y14, eIF4A3, RNPS1, SAP18, and Acinus also encouraged the prodn. of other proapoptotic splice variants, suggesting that EJC-assocd. components are important regulators of apoptosis acting at the alternative splicing level.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQrodxO9g_gLVg90H21EOLACvtfcHk0liGRjvD-nvteg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtV2ju7k%253D&md5=612d5fe051d6a9b43d5e20a583b719dd</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.06130-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.06130-11%26sid%3Dliteratum%253Aachs%26aulast%3DMichelle%26aufirst%3DL.%26aulast%3DCloutier%26aufirst%3DA.%26aulast%3DToutant%26aufirst%3DJ.%26aulast%3DShkreta%26aufirst%3DL.%26aulast%3DThibault%26aufirst%3DP.%26aulast%3DDurand%26aufirst%3DM.%26aulast%3DGarneau%26aufirst%3DD.%26aulast%3DGendron%26aufirst%3DD.%26aulast%3DLapointe%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DS.%26aulast%3DLe%2BHir%26aufirst%3DH.%26aulast%3DKlinck%26aufirst%3DR.%26aulast%3DElela%26aufirst%3DS.%2BA.%26aulast%3DPrinos%26aufirst%3DP.%26aulast%3DChabot%26aufirst%3DB.%26atitle%3DProteins%2520associated%2520with%2520the%2520exon%2520junction%2520complex%2520also%2520control%2520the%2520alternative%2520splicing%2520of%2520apoptotic%2520regulators%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2012%26volume%3D32%26spage%3D954%26epage%3D967%26doi%3D10.1128%2FMCB.06130-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murigneux, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le Hir, H.</span><span> </span><span class="NLM_article-title">Transcriptome-wide modulation of splicing by the exon junction complex</span> <span class="citation_source-journal">Genome Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">551</span><span class="refDoi"> DOI: 10.1186/s13059-014-0551-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1186%2Fs13059-014-0551-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=551&author=Z.+Wangauthor=V.+Murigneuxauthor=H.+Le+Hir&title=Transcriptome-wide+modulation+of+splicing+by+the+exon+junction+complex&doi=10.1186%2Fs13059-014-0551-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1186%2Fs13059-014-0551-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13059-014-0551-7%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMurigneux%26aufirst%3DV.%26aulast%3DLe%2BHir%26aufirst%3DH.%26atitle%3DTranscriptome-wide%2520modulation%2520of%2520splicing%2520by%2520the%2520exon%2520junction%2520complex%26jtitle%3DGenome%2520Biol.%26date%3D2014%26volume%3D15%26spage%3D551%26doi%3D10.1186%2Fs13059-014-0551-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Baker, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, R.</span><span> </span><span class="NLM_article-title">Nonsense-mediated mRNA decay: terminating erroneous gene expression</span> <span class="citation_source-journal">Curr. Opin. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">293</span><span class="NLM_x">–</span> <span class="NLM_lpage">299</span><span class="refDoi"> DOI: 10.1016/j.ceb.2004.03.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.ceb.2004.03.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=15145354" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD2cXktVCqsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2004&pages=293-299&author=K.+E.+Bakerauthor=R.+Parker&title=Nonsense-mediated+mRNA+decay%3A+terminating+erroneous+gene+expression&doi=10.1016%2Fj.ceb.2004.03.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Nonsense-mediated mRNA decay: terminating erroneous gene expression</span></div><div class="casAuthors">Baker, Kristian E.; Parker, Roy</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Cell Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">293-299</span>CODEN:
                <span class="NLM_cas:coden">COCBE3</span>;
        ISSN:<span class="NLM_cas:issn">0955-0674</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review.  Nonsense-mediated mRNA decay is a surveillance pathway that reduces errors in gene expression by eliminating aberrant mRNAs that encode incomplete polypeptides.  Recent expts. suggest a working model whereby premature and normal translation termination events are distinct as a consequence of the spatial relationship between the termination codon and mRNA binding proteins, a relationship partially established by nuclear pre-mRNA processing.  Aberrant termination then leads to both translational repression and an increased susceptibility of the mRNA to multiple RNases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0ycN1Dri1eLVg90H21EOLACvtfcHk0liv9HORM6O4iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXktVCqsbY%253D&md5=ac182d78f4a5500120a98f0707455196</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ceb.2004.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ceb.2004.03.003%26sid%3Dliteratum%253Aachs%26aulast%3DBaker%26aufirst%3DK.%2BE.%26aulast%3DParker%26aufirst%3DR.%26atitle%3DNonsense-mediated%2520mRNA%2520decay%253A%2520terminating%2520erroneous%2520gene%2520expression%26jtitle%3DCurr.%2520Opin.%2520Cell%2520Biol.%26date%3D2004%26volume%3D16%26spage%3D293%26epage%3D299%26doi%3D10.1016%2Fj.ceb.2004.03.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Keeling, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedwell, D. M.</span><span> </span><span class="NLM_article-title">Suppression of nonsense mutations as a therapeutic approach to treat genetic diseases</span> <span class="citation_source-journal">Wiley Interdiscip. Rev. RNA</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">837</span><span class="NLM_x">–</span> <span class="NLM_lpage">852</span><span class="refDoi"> DOI: 10.1002/wrna.95</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1002%2Fwrna.95" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=21976286" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyrsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=837-852&author=K.+M.+Keelingauthor=D.+M.+Bedwell&title=Suppression+of+nonsense+mutations+as+a+therapeutic+approach+to+treat+genetic+diseases&doi=10.1002%2Fwrna.95"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of nonsense mutations as a therapeutic approach to treat genetic diseases</span></div><div class="casAuthors">Keeling, Kim M.; Bedwell, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Wiley Interdisciplinary Reviews: RNA</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">837-852</span>CODEN:
                <span class="NLM_cas:coden">WIRRCU</span>;
        ISSN:<span class="NLM_cas:issn">1757-7012</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Suppression therapy is a treatment strategy for genetic diseases caused by nonsense mutations.  This therapeutic approach utilizes pharmacol. agents that suppress translation termination at in-frame premature termination codons (PTCs) to restore translation of a full-length, functional polypeptide.  The efficiency of various classes of compds. to suppress PTCs in mammalian cells is discussed along with the current limitations of this therapy.  We also elaborate on approaches to improve the efficiency of suppression that include methods to enhance the effectiveness of current suppression drugs and the design or discovery of new, more effective suppression agents.  Finally, we discuss the role of nonsense-mediated mRNA decay (NMD) in limiting the effectiveness of suppression therapy, and describe tactics that may allow the efficiency of NMD to be modulated in order to enhance suppression therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBNm5QrXfib7Vg90H21EOLACvtfcHk0liv9HORM6O4iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyrsrjO&md5=c51c5d5bde9daac8d9d7adaa018107e1</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fwrna.95&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwrna.95%26sid%3Dliteratum%253Aachs%26aulast%3DKeeling%26aufirst%3DK.%2BM.%26aulast%3DBedwell%26aufirst%3DD.%2BM.%26atitle%3DSuppression%2520of%2520nonsense%2520mutations%2520as%2520a%2520therapeutic%2520approach%2520to%2520treat%2520genetic%2520diseases%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%2520RNA%26date%3D2011%26volume%3D2%26spage%3D837%26epage%3D852%26doi%3D10.1002%2Fwrna.95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Martin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grigoryan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breda, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rivella, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, L. B.</span><span> </span><span class="NLM_article-title">Identification and characterization of small molecules that inhibit nonsense mediated RNA decay and suppress nonsense p53 mutations</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">3104</span><span class="NLM_x">–</span> <span class="NLM_lpage">3113</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-2235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1158%2F0008-5472.CAN-13-2235" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=3104-3113&author=L.+Martinauthor=A.+Grigoryanauthor=D.+Wangauthor=J.+Wangauthor=L.+Bredaauthor=S.+Rivellaauthor=T.+Cardozoauthor=L.+B.+Gardner&title=Identification+and+characterization+of+small+molecules+that+inhibit+nonsense+mediated+RNA+decay+and+suppress+nonsense+p53+mutations&doi=10.1158%2F0008-5472.CAN-13-2235"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-2235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-2235%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DL.%26aulast%3DGrigoryan%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DBreda%26aufirst%3DL.%26aulast%3DRivella%26aufirst%3DS.%26aulast%3DCardozo%26aufirst%3DT.%26aulast%3DGardner%26aufirst%3DL.%2BB.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520small%2520molecules%2520that%2520inhibit%2520nonsense%2520mediated%2520RNA%2520decay%2520and%2520suppress%2520nonsense%2520p53%2520mutations%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D3104%26epage%3D3113%26doi%3D10.1158%2F0008-5472.CAN-13-2235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Pastor, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolonias, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giangrande, P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilboa, E.</span><span> </span><span class="NLM_article-title">Induction of tumour immunity by targeted inhibition of nonsense-mediated mRNA decay</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">465</span><span class="NLM_x">, </span> <span class="NLM_fpage">227</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span><span class="refDoi"> DOI: 10.1038/nature08999</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnature08999" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=20463739" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlvFOrsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=465&publication_year=2010&pages=227-230&author=F.+Pastorauthor=D.+Koloniasauthor=P.+H.+Giangrandeauthor=E.+Gilboa&title=Induction+of+tumour+immunity+by+targeted+inhibition+of+nonsense-mediated+mRNA+decay&doi=10.1038%2Fnature08999"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Induction of tumour immunity by targeted inhibition of nonsense-mediated mRNA decay</span></div><div class="casAuthors">Pastor, Fernando; Kolonias, Despina; Giangrande, Paloma H.; Gilboa, Eli</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">465</span>
        (<span class="NLM_cas:issue">7295</span>),
    <span class="NLM_cas:pages">227-230</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The main reason why tumors are not controlled by the immune system is that, unlike pathogens, they do not express potent tumor rejection antigens (TRAs).  Tumor vaccination aims at stimulating a systemic immune response targeted to, mostly weak, antigens expressed in the disseminated tumor lesions.  Main challenges in developing effective vaccination protocols are the identification of potent and broadly expressed TRAs and effective adjuvants to stimulate a robust and durable immune response.  Here the authors describe an alternative approach in which the expression of new, and thereby potent, antigens are induced in tumor cells by inhibiting nonsense-mediated mRNA decay (NMD).  Small interfering RNA (siRNA)-mediated inhibition of NMD in tumor cells led to the expression of new antigenic determinants and their immune-mediated rejection.  In s.c. and metastatic tumor models, tumor-targeted delivery of NMD factor-specific siRNAs conjugated to oligonucleotide aptamer ligands led to significant inhibition of tumor growth that was superior to that of vaccination with granulocyte-macrophage colony-stimulating factor (GM-CSF)-expressing irradiated tumor cells, and could be further enhanced by co-stimulation.  Tumor-targeted NMD inhibition forms the basis of a simple, broadly useful, and clin. feasible approach to enhance the antigenicity of disseminated tumors leading to their immune recognition and rejection.  The cell-free chem. synthesized oligonucleotide backbone of aptamer-siRNAs reduces the risk of immunogenicity and enhances the feasibility of generating reagents suitable for clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdL4APQthUHbVg90H21EOLACvtfcHk0lipDAAczlbBhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlvFOrsbk%253D&md5=57a1b9b7c4e1e90fcc50f30a158ffa22</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnature08999&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08999%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DF.%26aulast%3DKolonias%26aufirst%3DD.%26aulast%3DGiangrande%26aufirst%3DP.%2BH.%26aulast%3DGilboa%26aufirst%3DE.%26atitle%3DInduction%2520of%2520tumour%2520immunity%2520by%2520targeted%2520inhibition%2520of%2520nonsense-mediated%2520mRNA%2520decay%26jtitle%3DNature%26date%3D2010%26volume%3D465%26spage%3D227%26epage%3D230%26doi%3D10.1038%2Fnature08999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Popp, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maquat, L. E.</span><span> </span><span class="NLM_article-title">Attenuation of nonsense-mediated mRNA decay facilitates the response to chemotherapeutics</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">6632</span><span class="NLM_x">–</span> <span class="NLM_lpage">6632</span><span class="refDoi"> DOI: 10.1038/ncomms7632</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fncomms7632" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=25808464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2itLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6632-6632&author=M.+W.+Poppauthor=L.+E.+Maquat&title=Attenuation+of+nonsense-mediated+mRNA+decay+facilitates+the+response+to+chemotherapeutics&doi=10.1038%2Fncomms7632"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Attenuation of nonsense-mediated mRNA decay facilitates the response to chemotherapeutics</span></div><div class="casAuthors">Popp, Maximilian W.; Maquat, Lynne E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6632</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nonsense-mediated mRNA decay (NMD) limits the prodn. of aberrant mRNAs contg. a premature termination codon and also controls the levels of endogenous transcripts.  Here we show that when human cells are treated with clin. used chemotherapeutic compds., NMD activity declines partly as a result of the proteolytic prodn. of a dominant-interfering form of the key NMD factor UPF1.  Prodn. of cleaved UPF1 functions to upregulate genes involved in the response to apoptotic stresses.  The biol. consequence is the promotion of cell death.  Combined exposure of cells to a small-mol. inhibitor of NMD, NMDI-1, and the chemotherapeutic doxorubicin leads to enhanced cell death, while inhibiting UPF1 cleavage protects cells from doxorubicin challenge.  We propose a model to explain why the expression levels of genes producing mRNAs of diverse structure that encode proteins of diverse function are under the purview of NMD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKtCDG1_OzHbVg90H21EOLACvtfcHk0lipDAAczlbBhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2itLnJ&md5=05e84ebc26b77de5e7c1da1b9e9ac2f2</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fncomms7632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7632%26sid%3Dliteratum%253Aachs%26aulast%3DPopp%26aufirst%3DM.%2BW.%26aulast%3DMaquat%26aufirst%3DL.%2BE.%26atitle%3DAttenuation%2520of%2520nonsense-mediated%2520mRNA%2520decay%2520facilitates%2520the%2520response%2520to%2520chemotherapeutics%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6632%26epage%3D6632%26doi%3D10.1038%2Fncomms7632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Hug, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Longman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caceres, J. F.</span><span> </span><span class="NLM_article-title">Mechanism and regulation of the nonsense-mediated decay pathway</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1483</span><span class="NLM_x">–</span> <span class="NLM_lpage">1495</span><span class="refDoi"> DOI: 10.1093/nar/gkw010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1093%2Fnar%2Fgkw010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=1483-1495&author=N.+Hugauthor=D.+Longmanauthor=J.+F.+Caceres&title=Mechanism+and+regulation+of+the+nonsense-mediated+decay+pathway&doi=10.1093%2Fnar%2Fgkw010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw010%26sid%3Dliteratum%253Aachs%26aulast%3DHug%26aufirst%3DN.%26aulast%3DLongman%26aufirst%3DD.%26aulast%3DCaceres%26aufirst%3DJ.%2BF.%26atitle%3DMechanism%2520and%2520regulation%2520of%2520the%2520nonsense-mediated%2520decay%2520pathway%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D1483%26epage%3D1495%26doi%3D10.1093%2Fnar%2Fgkw010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Kurosaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maquat, L. E.</span><span> </span><span class="NLM_article-title">Nonsense-mediated mRNA decay in humans at a glance</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">129</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span><span class="refDoi"> DOI: 10.1242/jcs.181008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1242%2Fjcs.181008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=26787741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC28Xpt1Omt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2016&pages=461-467&author=T.+Kurosakiauthor=L.+E.+Maquat&title=Nonsense-mediated+mRNA+decay+in+humans+at+a+glance&doi=10.1242%2Fjcs.181008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Nonsense-mediated mRNA decay in humans at a glance</span></div><div class="casAuthors">Kurosaki, Tatsuaki; Maquat, Lynne E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">461-467</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Nonsense-mediated mRNA decay (NMD) is an mRNA quality-control mechanism that typifies all eukaryotes examd. to date.  NMD surveys newly synthesized mRNAs and degrades those that harbor a premature termination codon (PTC), thereby preventing the prodn. of truncated proteins that could result in disease in humans.  This is evident from dominantly inherited diseases that are due to PTC-contg. mRNAs that escape NMD.  Although many cellular NMD targets derive from mistakes made during, for example, pre-mRNA splicing and, possibly, transcription initiation, NMD also targets ∼10% of normal physiol. mRNAs so as to promote an appropriate cellular response to changing environmental milieus, including those that induce apoptosis, maturation or differentiation, Over the past ∼35 years, a central goal in the NMD field has been to understand how cells discriminate mRNAs that are targeted by NMD from those that are not.  In this Cell Science at a Glance and the accompanying poster, we review progress made towards this goal, focusing on human studies and the role of the key NMD factor up-frameshift protein 1 (UPF1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofrNkqevLoPLVg90H21EOLACvtfcHk0lh-kPvVUSNOnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xpt1Omt7g%253D&md5=0f76cab950a742708107974616a88a4e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1242%2Fjcs.181008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.181008%26sid%3Dliteratum%253Aachs%26aulast%3DKurosaki%26aufirst%3DT.%26aulast%3DMaquat%26aufirst%3DL.%2BE.%26atitle%3DNonsense-mediated%2520mRNA%2520decay%2520in%2520humans%2520at%2520a%2520glance%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2016%26volume%3D129%26spage%3D461%26epage%3D467%26doi%3D10.1242%2Fjcs.181008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Bhuvanagiri, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Putzker, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Becker, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leicht, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krijgsveld, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Batra, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turnwald, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jovanovic, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sieber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hentze, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulozik, A. E.</span><span> </span><span class="NLM_article-title">5-azacytidine inhibits nonsense-mediated decay in a MYC-dependent fashion</span> <span class="citation_source-journal">EMBO Mol. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1593</span><span class="NLM_x">–</span> <span class="NLM_lpage">1609</span><span class="refDoi"> DOI: 10.15252/emmm.201404461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.15252%2Femmm.201404461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=25319547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVamur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=1593-1609&author=M.+Bhuvanagiriauthor=J.+Lewisauthor=K.+Putzkerauthor=J.+P.+Beckerauthor=S.+Leichtauthor=J.+Krijgsveldauthor=R.+Batraauthor=B.+Turnwaldauthor=B.+Jovanovicauthor=C.+Hauerauthor=J.+Sieberauthor=M.+W.+Hentzeauthor=A.+E.+Kulozik&title=5-azacytidine+inhibits+nonsense-mediated+decay+in+a+MYC-dependent+fashion&doi=10.15252%2Femmm.201404461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">5-azacytidine inhibits nonsense-mediated decay in a MYC-dependent fashion</span></div><div class="casAuthors">Bhuvanagiri, Madhuri; Lewis, Joe; Putzker, Kerstin; Becker, Jonas P.; Leicht, Stefan; Krijgsveld, Jeroen; Batra, Richa; Turnwald, Brad; Jovanovic, Bogdan; Hauer, Christian; Sieber, Jana; Hentze, Matthias W.; Kulozik, Andreas E.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Molecular Medicine</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1593-1609</span>CODEN:
                <span class="NLM_cas:coden">EMMMAM</span>;
        ISSN:<span class="NLM_cas:issn">1757-4684</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Nonsense-mediated RNA decay (NMD) is an RNA-based quality control mechanism that eliminates transcripts bearing premature translation termination codons (PTC).  Approx., one-third of all inherited disorders and some forms of cancer are caused by nonsense or frame shift mutations that introduce PTCs, and NMD can modulate the clin. phenotype of these diseases.  5-Azacytidine is an analog of the naturally occurring pyrimidine nucleoside cytidine, which is approved for the treatment of myelodysplastic syndrome and myeloid leukemia.  Here, we reveal that 5-azacytidine inhibits NMD in a dose-dependent fashion specifically upregulating the expression of both PTC-contg. mutant and cellular NMD targets.  Moreover, this activity of 5-azacytidine depends on the induction of MYC expression, thus providing a link between the effect of this drug and one of the key cellular pathways that are known to affect NMD activity.  Furthermore, the effective concn. of 5-azacytidine in cells corresponds to drug levels used in patients, qualifying 5-azacytidine as a candidate drug that could potentially be repurposed for the treatment of Mendelian and acquired genetic diseases that are caused by PTC mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruolEF6XIUibVg90H21EOLACvtfcHk0lh-kPvVUSNOnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVamur%252FJ&md5=1af5afdc70b4289ea84c87509ae9998c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.15252%2Femmm.201404461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.15252%252Femmm.201404461%26sid%3Dliteratum%253Aachs%26aulast%3DBhuvanagiri%26aufirst%3DM.%26aulast%3DLewis%26aufirst%3DJ.%26aulast%3DPutzker%26aufirst%3DK.%26aulast%3DBecker%26aufirst%3DJ.%2BP.%26aulast%3DLeicht%26aufirst%3DS.%26aulast%3DKrijgsveld%26aufirst%3DJ.%26aulast%3DBatra%26aufirst%3DR.%26aulast%3DTurnwald%26aufirst%3DB.%26aulast%3DJovanovic%26aufirst%3DB.%26aulast%3DHauer%26aufirst%3DC.%26aulast%3DSieber%26aufirst%3DJ.%26aulast%3DHentze%26aufirst%3DM.%2BW.%26aulast%3DKulozik%26aufirst%3DA.%2BE.%26atitle%3D5-azacytidine%2520inhibits%2520nonsense-mediated%2520decay%2520in%2520a%2520MYC-dependent%2520fashion%26jtitle%3DEMBO%2520Mol.%2520Med.%26date%3D2014%26volume%3D6%26spage%3D1593%26epage%3D1609%26doi%3D10.15252%2Femmm.201404461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Dang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Low, W. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehring, N. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, H. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. O.</span><span> </span><span class="NLM_article-title">Inhibition of nonsense-mediated mRNA decay by the natural product pateamine A through eukaryotic initiation factor 4AIII</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">23613</span><span class="NLM_x">–</span> <span class="NLM_lpage">23621</span><span class="refDoi"> DOI: 10.1074/jbc.M109.009985</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1074%2Fjbc.M109.009985" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23613-23621&author=Y.+Dangauthor=W.+K.+Lowauthor=J.+Xuauthor=N.+H.+Gehringauthor=H.+C.+Dietzauthor=D.+Romoauthor=J.+O.+Liu&title=Inhibition+of+nonsense-mediated+mRNA+decay+by+the+natural+product+pateamine+A+through+eukaryotic+initiation+factor+4AIII&doi=10.1074%2Fjbc.M109.009985"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.009985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.009985%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DY.%26aulast%3DLow%26aufirst%3DW.%2BK.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DGehring%26aufirst%3DN.%2BH.%26aulast%3DDietz%26aufirst%3DH.%2BC.%26aulast%3DRomo%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26atitle%3DInhibition%2520of%2520nonsense-mediated%2520mRNA%2520decay%2520by%2520the%2520natural%2520product%2520pateamine%2520A%2520through%2520eukaryotic%2520initiation%2520factor%25204AIII%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D23613%26epage%3D23621%26doi%3D10.1074%2Fjbc.M109.009985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Durand, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cougot, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahuteau-Betzer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, C.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grierson, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bertrand, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tazi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, F.</span><span> </span><span class="NLM_article-title">Inhibition of nonsense-mediated mRNA decay (NMD) by a new chemical molecule reveals the dynamic of NMD factors in P-bodies</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">1145</span><span class="NLM_x">–</span> <span class="NLM_lpage">1160</span><span class="refDoi"> DOI: 10.1083/jcb.200611086</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1083%2Fjcb.200611086" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2007&pages=1145-1160&author=S.+Durandauthor=N.+Cougotauthor=F.+Mahuteau-Betzerauthor=C.-H.+Nguyenauthor=D.+S.+Griersonauthor=E.+Bertrandauthor=J.+Taziauthor=F.+Lejeune&title=Inhibition+of+nonsense-mediated+mRNA+decay+%28NMD%29+by+a+new+chemical+molecule+reveals+the+dynamic+of+NMD+factors+in+P-bodies&doi=10.1083%2Fjcb.200611086"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200611086&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200611086%26sid%3Dliteratum%253Aachs%26aulast%3DDurand%26aufirst%3DS.%26aulast%3DCougot%26aufirst%3DN.%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26aulast%3DNguyen%26aufirst%3DC.-H.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26aulast%3DBertrand%26aufirst%3DE.%26aulast%3DTazi%26aufirst%3DJ.%26aulast%3DLejeune%26aufirst%3DF.%26atitle%3DInhibition%2520of%2520nonsense-mediated%2520mRNA%2520decay%2520%2528NMD%2529%2520by%2520a%2520new%2520chemical%2520molecule%2520reveals%2520the%2520dynamic%2520of%2520NMD%2520factors%2520in%2520P-bodies%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2007%26volume%3D178%26spage%3D1145%26epage%3D1160%26doi%3D10.1083%2Fjcb.200611086" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Feng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, R.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Triggs-Raine, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, J.</span><span> </span><span class="NLM_article-title">Increase of a group of PTC<sup>+</sup> transcripts by curcumin through inhibition of the NMD pathway</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Gene Regul. Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">1849</span><span class="NLM_x">, </span> <span class="NLM_fpage">1104</span><span class="NLM_x">–</span> <span class="NLM_lpage">1115</span><span class="refDoi"> DOI: 10.1016/j.bbagrm.2015.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.bbagrm.2015.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=25934542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvVSjs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1849&publication_year=2015&pages=1104-1115&author=D.+Fengauthor=R.-C.+Suauthor=L.+Zouauthor=B.+Triggs-Raineauthor=S.+Huangauthor=J.+Xie&title=Increase+of+a+group+of+PTC%2B+transcripts+by+curcumin+through+inhibition+of+the+NMD+pathway&doi=10.1016%2Fj.bbagrm.2015.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Increase of a group of PTC+ transcripts by curcumin through inhibition of the NMD pathway</span></div><div class="casAuthors">Feng, Dairong; Su, Ruey-Chyi; Zou, Liping; Triggs-Raine, Barbara; Huang, Shangzhi; Xie, Jiuyong</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Gene Regulatory Mechanisms</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1849</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1104-1115</span>CODEN:
                <span class="NLM_cas:coden">BBAGC6</span>;
        ISSN:<span class="NLM_cas:issn">1874-9399</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Nonsense-mediated mRNA decay (NMD), an mRNA surveillance mechanism, eliminates premature termination codon-contg. (PTC+) transcripts.  For instance, it maintains the homeostasis of splicing factors and degrades aberrant transcripts of human genetic disease genes.  Here we examine the inhibitory effect on the NMD pathway and consequent increase of PTC+ transcripts by the dietary compd. curcumin.  We have found that several PTC+ transcripts including that of serine/arginine-rich splicing factor 1 (SRSF1) were specifically increased in cells by curcumin.  We also obsd. a similar curcumin effect on the PTC+ mutant transcript from a Tay-Sachs-causing HEXA allele or from a beta-globin reporter gene.  The curcumin effect was accompanied by significantly reduced expression of the NMD factors UPF1, 2, 3A and 3B.  Consistently, in chromatin immunopptn. assays, curcumin specifically reduced the occupancy of acetyl-histone H3 and RNA polymerase II at the promoter region (- 376 to - 247 nt) of human UPF1, in a time- and dosage-dependent way.  Importantly, knocking down UPF1 abolished or substantially reduced the difference of PTC+ transcript levels between control and curcumin-treated cells.  The disrupted curcumin effect was efficiently rescued by expression of exogenous Myc-UPF1 in the knockdown cells.  Together, our data demonstrate that a group of PTC+ transcripts are stabilized by a dietary compd. curcumin through the inhibition of UPF factor expression and the NMD pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreaYtNoV7jJ7Vg90H21EOLACvtfcHk0lixEBuON_VsBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvVSjs7w%253D&md5=2b3492d4150198be313e62c662b5b483</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bbagrm.2015.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbagrm.2015.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DFeng%26aufirst%3DD.%26aulast%3DSu%26aufirst%3DR.-C.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DTriggs-Raine%26aufirst%3DB.%26aulast%3DHuang%26aufirst%3DS.%26aulast%3DXie%26aufirst%3DJ.%26atitle%3DIncrease%2520of%2520a%2520group%2520of%2520PTC%252B%2520transcripts%2520by%2520curcumin%2520through%2520inhibition%2520of%2520the%2520NMD%2520pathway%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Gene%2520Regul.%2520Mech.%26date%3D2015%26volume%3D1849%26spage%3D1104%26epage%3D1115%26doi%3D10.1016%2Fj.bbagrm.2015.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Bordeleau, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mori, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oberer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindqvist, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chard, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belsham, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span> </span><span class="NLM_article-title">Functional characterization of IRESes by an inhibitor of the RNA helicase eIF4A</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">220</span><span class="refDoi"> DOI: 10.1038/nchembio776</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1038%2Fnchembio776" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=16532013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD28XisFOnsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2006&pages=213-220&author=M.+E.+Bordeleauauthor=A.+Moriauthor=M.+Obererauthor=L.+Lindqvistauthor=L.+S.+Chardauthor=T.+Higaauthor=G.+J.+Belshamauthor=G.+Wagnerauthor=J.+Tanakaauthor=J.+Pelletier&title=Functional+characterization+of+IRESes+by+an+inhibitor+of+the+RNA+helicase+eIF4A&doi=10.1038%2Fnchembio776"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Functional characterization of IRESes by an inhibitor of the RNA helicase eIF4A</span></div><div class="casAuthors">Bordeleau, Marie-Eve; Mori, Ayaka; Oberer, Monika; Lindqvist, Lisa; Chard, Louisa S.; Higa, Tatsuo; Belsham, Graham J.; Wagner, Gerhard; Tanaka, Junichi; Pelletier, Jerry</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">213-220</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA helicases are mol. motors that are involved in virtually all aspects of RNA metab.  Eukaryotic initiation factor (eIF) 4A is the prototypical member of the DEAD-box family of RNA helicases.  It is thought to use energy from ATP hydrolysis to unwind mRNA structure and, in conjunction with other translation factors, it preps. mRNA templates for ribosome recruitment during translation initiation.  In screening marine exts. for new eukaryotic translation initiation inhibitors, we identified the natural product hippuristanol.  We show here that this compd. is a selective and potent inhibitor of eIF4A RNA-binding activity that can be used to distinguish between eIF4A-dependent and -independent modes of translation initiation in vitro and in vivo.  We also show that poliovirus replication is delayed when infected cells are exposed to hippuristanol.  Our study demonstrates the feasibility of selectively targeting members of the DEAD-box helicase family with small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphVpPHxoy7Y7Vg90H21EOLACvtfcHk0lgpMqUDgnXmiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisFOnsbg%253D&md5=cbeecb6be4f6ab28c47b4c0542ba4d3d</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio776&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio776%26sid%3Dliteratum%253Aachs%26aulast%3DBordeleau%26aufirst%3DM.%2BE.%26aulast%3DMori%26aufirst%3DA.%26aulast%3DOberer%26aufirst%3DM.%26aulast%3DLindqvist%26aufirst%3DL.%26aulast%3DChard%26aufirst%3DL.%2BS.%26aulast%3DHiga%26aufirst%3DT.%26aulast%3DBelsham%26aufirst%3DG.%2BJ.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DTanaka%26aufirst%3DJ.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DFunctional%2520characterization%2520of%2520IRESes%2520by%2520an%2520inhibitor%2520of%2520the%2520RNA%2520helicase%2520eIF4A%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2006%26volume%3D2%26spage%3D213%26epage%3D220%26doi%3D10.1038%2Fnchembio776" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Lindqvist, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oberer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reibarkh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cencic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bordeleau, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogt, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marintchev, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fagotto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span> </span><span class="NLM_article-title">Selective pharmacological targeting of a DEAD box RNA helicase</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1583</span><span class="refDoi"> DOI: 10.1371/journal.pone.0001583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1371%2Fjournal.pone.0001583" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=18270573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A280%3ADC%252BD1c7htlagtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2008&pages=e1583&author=L.+Lindqvistauthor=M.+Obererauthor=M.+Reibarkhauthor=R.+Cencicauthor=M.+E.+Bordeleauauthor=E.+Vogtauthor=A.+Marintchevauthor=J.+Tanakaauthor=F.+Fagottoauthor=M.+Altmannauthor=G.+Wagnerauthor=J.+Pelletier&title=Selective+pharmacological+targeting+of+a+DEAD+box+RNA+helicase&doi=10.1371%2Fjournal.pone.0001583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Selective pharmacological targeting of a DEAD box RNA helicase</span></div><div class="casAuthors">Lindqvist Lisa; Oberer Monika; Reibarkh Mikhail; Cencic Regina; Bordeleau Marie-Eve; Vogt Emily; Marintchev Assen; Tanaka Junichi; Fagotto Francois; Altmann Michael; Wagner Gerhard; Pelletier Jerry</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1583</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">RNA helicases represent a large family of proteins implicated in many biological processes including ribosome biogenesis, splicing, translation and mRNA degradation.  However, these proteins have little substrate specificity, making inhibition of selected helicases a challenging problem.  The prototypical DEAD box RNA helicase, eIF4A, works in conjunction with other translation factors to prepare mRNA templates for ribosome recruitment during translation initiation.  Herein, we provide insight into the selectivity of a small molecule inhibitor of eIF4A, hippuristanol.  This coral-derived natural product binds to amino acids adjacent to, and overlapping with, two conserved motifs present in the carboxy-terminal domain of eIF4A.  Mutagenesis of amino acids within this region allowed us to alter the hippuristanol-sensitivity of eIF4A and undertake structure/function studies.  Our results provide an understanding into how selective targeting of RNA helicases for pharmacological intervention can be achieved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQKpFB5Tdsa_tXdcedVHT5tfW6udTcc2eaC1skp7PSLgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c7htlagtg%253D%253D&md5=b29e717681032adca5889d26e11e480d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0001583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0001583%26sid%3Dliteratum%253Aachs%26aulast%3DLindqvist%26aufirst%3DL.%26aulast%3DOberer%26aufirst%3DM.%26aulast%3DReibarkh%26aufirst%3DM.%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DBordeleau%26aufirst%3DM.%2BE.%26aulast%3DVogt%26aufirst%3DE.%26aulast%3DMarintchev%26aufirst%3DA.%26aulast%3DTanaka%26aufirst%3DJ.%26aulast%3DFagotto%26aufirst%3DF.%26aulast%3DAltmann%26aufirst%3DM.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DSelective%2520pharmacological%2520targeting%2520of%2520a%2520DEAD%2520box%2520RNA%2520helicase%26jtitle%3DPLoS%2520One%26date%3D2008%26volume%3D3%26spage%3De1583%26doi%3D10.1371%2Fjournal.pone.0001583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Higa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsukitani, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kikuchi, H.</span><span> </span><span class="NLM_article-title">Hippuristanols, cytotoxic polyoxygenated steroids from the gorgonian isis hippuris</span> <span class="citation_source-journal">Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1981</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1647</span><span class="NLM_x">–</span> <span class="NLM_lpage">1650</span><span class="refDoi"> DOI: 10.1246/cl.1981.1647</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1246%2Fcl.1981.1647" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1981&pages=1647-1650&author=T.+Higaauthor=J.+Tanakaauthor=Y.+Tsukitaniauthor=H.+Kikuchi&title=Hippuristanols%2C+cytotoxic+polyoxygenated+steroids+from+the+gorgonian+isis+hippuris&doi=10.1246%2Fcl.1981.1647"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1246%2Fcl.1981.1647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1246%252Fcl.1981.1647%26sid%3Dliteratum%253Aachs%26aulast%3DHiga%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DJ.%26aulast%3DTsukitani%26aufirst%3DY.%26aulast%3DKikuchi%26aufirst%3DH.%26atitle%3DHippuristanols%252C%2520cytotoxic%2520polyoxygenated%2520steroids%2520from%2520the%2520gorgonian%2520isis%2520hippuris%26jtitle%3DChem.%2520Lett.%26date%3D1981%26volume%3D10%26spage%3D1647%26epage%3D1650%26doi%3D10.1246%2Fcl.1981.1647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Cencic, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pelletier, J.</span><span> </span><span class="NLM_article-title">Hippuristanol - A potent steroid inhibitor of eukaryotic initiation factor 4A</span> <span class="citation_source-journal">Translation</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1137381</span><span class="refDoi"> DOI: 10.1080/21690731.2015.1137381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1080%2F21690731.2015.1137381" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2016&pages=e1137381&author=R.+Cencicauthor=J.+Pelletier&title=Hippuristanol+-+A+potent+steroid+inhibitor+of+eukaryotic+initiation+factor+4A&doi=10.1080%2F21690731.2015.1137381"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1080%2F21690731.2015.1137381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F21690731.2015.1137381%26sid%3Dliteratum%253Aachs%26aulast%3DCencic%26aufirst%3DR.%26aulast%3DPelletier%26aufirst%3DJ.%26atitle%3DHippuristanol%2520-%2520A%2520potent%2520steroid%2520inhibitor%2520of%2520eukaryotic%2520initiation%2520factor%25204A%26jtitle%3DTranslation%26date%3D2016%26volume%3D4%26spage%3De1137381%26doi%3D10.1080%2F21690731.2015.1137381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Ito, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwatani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kamada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sogabe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamoto, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakanishi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imaeda, Y.</span><span> </span><span class="NLM_article-title">Discovery of selective ATP-competitive eIF4A3 inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">2200</span><span class="NLM_x">–</span> <span class="NLM_lpage">2209</span><span class="refDoi"> DOI: 10.1016/j.bmc.2017.02.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.bmc.2017.02.035" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=2200-2209&author=M.+Itoauthor=M.+Iwataniauthor=Y.+Kamadaauthor=S.+Sogabeauthor=S.+Nakaoauthor=T.+Tanakaauthor=T.+Kawamotoauthor=S.+Aparicioauthor=A.+Nakanishiauthor=Y.+Imaeda&title=Discovery+of+selective+ATP-competitive+eIF4A3+inhibitors&doi=10.1016%2Fj.bmc.2017.02.035"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.02.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.02.035%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DM.%26aulast%3DIwatani%26aufirst%3DM.%26aulast%3DKamada%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DNakao%26aufirst%3DS.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKawamoto%26aufirst%3DT.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DNakanishi%26aufirst%3DA.%26aulast%3DImaeda%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520selective%2520ATP-competitive%2520eIF4A3%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D2200%26epage%3D2209%26doi%3D10.1016%2Fj.bmc.2017.02.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Sánchez-Céspedes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martínez-Aguado, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vega-Holm, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serna-Gallego, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candela, J. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marrugal-Lorenzo, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pachón, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iglesias-Guerra, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vega-Pérez, J. M.</span><span> </span><span class="NLM_article-title">New 4-acyl-1-phenylaminocarbonyl-2-phenylpiperazine derivatives as potential inhibitors of adenovirus infection. synthesis, biological evaluation, and structure–activity relationships</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">5432</span><span class="NLM_x">–</span> <span class="NLM_lpage">5448</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00300</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00300" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5432-5448&author=J.+S%C3%A1nchez-C%C3%A9spedesauthor=P.+Mart%C3%ADnez-Aguadoauthor=M.+Vega-Holmauthor=A.+Serna-Gallegoauthor=J.+I.+Candelaauthor=J.+A.+Marrugal-Lorenzoauthor=J.+Pach%C3%B3nauthor=F.+Iglesias-Guerraauthor=J.+M.+Vega-P%C3%A9rez&title=New+4-acyl-1-phenylaminocarbonyl-2-phenylpiperazine+derivatives+as+potential+inhibitors+of+adenovirus+infection.+synthesis%2C+biological+evaluation%2C+and+structure%E2%80%93activity+relationships&doi=10.1021%2Facs.jmedchem.6b00300"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00300&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00300%26sid%3Dliteratum%253Aachs%26aulast%3DS%25C3%25A1nchez-C%25C3%25A9spedes%26aufirst%3DJ.%26aulast%3DMart%25C3%25ADnez-Aguado%26aufirst%3DP.%26aulast%3DVega-Holm%26aufirst%3DM.%26aulast%3DSerna-Gallego%26aufirst%3DA.%26aulast%3DCandela%26aufirst%3DJ.%2BI.%26aulast%3DMarrugal-Lorenzo%26aufirst%3DJ.%2BA.%26aulast%3DPach%25C3%25B3n%26aufirst%3DJ.%26aulast%3DIglesias-Guerra%26aufirst%3DF.%26aulast%3DVega-P%25C3%25A9rez%26aufirst%3DJ.%2BM.%26atitle%3DNew%25204-acyl-1-phenylaminocarbonyl-2-phenylpiperazine%2520derivatives%2520as%2520potential%2520inhibitors%2520of%2520adenovirus%2520infection.%2520synthesis%252C%2520biological%2520evaluation%252C%2520and%2520structure%25E2%2580%2593activity%2520relationships%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5432%26epage%3D5448%26doi%3D10.1021%2Facs.jmedchem.6b00300" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Glynn, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernier, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, S.</span><span> </span><span class="NLM_article-title">Microwave acceleration in DABAL-Me<sub>3</sub>-mediated amide formation</span> <span class="citation_source-journal">Tetrahedron Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">5687</span><span class="NLM_x">–</span> <span class="NLM_lpage">5688</span><span class="refDoi"> DOI: 10.1016/j.tetlet.2008.07.090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2Fj.tetlet.2008.07.090" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2008&pages=5687-5688&author=D.+Glynnauthor=D.+Bernierauthor=S.+Woodward&title=Microwave+acceleration+in+DABAL-Me3-mediated+amide+formation&doi=10.1016%2Fj.tetlet.2008.07.090"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.tetlet.2008.07.090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tetlet.2008.07.090%26sid%3Dliteratum%253Aachs%26aulast%3DGlynn%26aufirst%3DD.%26aulast%3DBernier%26aufirst%3DD.%26aulast%3DWoodward%26aufirst%3DS.%26atitle%3DMicrowave%2520acceleration%2520in%2520DABAL-Me3-mediated%2520amide%2520formation%26jtitle%3DTetrahedron%2520Lett.%26date%3D2008%26volume%3D49%26spage%3D5687%26epage%3D5688%26doi%3D10.1016%2Fj.tetlet.2008.07.090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Besselievre, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahuteau-Betzer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grierson, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piguel, S.</span><span> </span><span class="NLM_article-title">Ligandless microwave-assisted Pd/Cu-catalyzed direct arylation of oxazoles</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">3278</span><span class="NLM_x">–</span> <span class="NLM_lpage">3280</span><span class="refDoi"> DOI: 10.1021/jo7027135</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo7027135" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD1cXjsVWht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2008&pages=3278-3280&author=F.+Besselievreauthor=F.+Mahuteau-Betzerauthor=D.+S.+Griersonauthor=S.+Piguel&title=Ligandless+microwave-assisted+Pd%2FCu-catalyzed+direct+arylation+of+oxazoles&doi=10.1021%2Fjo7027135"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Ligandless microwave-assisted Pd/Cu-catalyzed direct arylation of oxazoles</span></div><div class="casAuthors">Besselievre, Francois; Mahuteau-Betzer, Florence; Grierson, David S.; Piguel, Sandrine</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3278-3280</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An efficient microwave-assisted palladium/copper co-mediated direct arylation of oxazoles with aryl bromides under ligandless conditions has been developed.  The method is functional group tolerant and provides rapid access to medicinally relevant compds., e.g. I (R = 2-MeC6H4, 1-naphthyl, 3-NCC6H4, etc.), in good yields.  Coupled to the van Leusen oxazole ring synthesis, this methodol. is illustrated by an expedient two-step synthesis of the four 2,5-diaryloxazole alkaloids, texamine, texaline, balsoxin, and O-Me-halfordinol from com. available starting materials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEHwRO9EHnbVg90H21EOLACvtfcHk0lhGBUMVeTByjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXjsVWht7g%253D&md5=7beb810c4a3d6ac4f4013736651f0804</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjo7027135&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo7027135%26sid%3Dliteratum%253Aachs%26aulast%3DBesselievre%26aufirst%3DF.%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26aulast%3DGrierson%26aufirst%3DD.%2BS.%26aulast%3DPiguel%26aufirst%3DS.%26atitle%3DLigandless%2520microwave-assisted%2520Pd%252FCu-catalyzed%2520direct%2520arylation%2520of%2520oxazoles%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2008%26volume%3D73%26spage%3D3278%26epage%3D3280%26doi%3D10.1021%2Fjo7027135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Sweeney, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tavassoli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, N. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayes, J. F.</span><span> </span><span class="NLM_article-title">[2,3]-Sigmatropic rearrangements of didehydropiperidinium ylids</span> <span class="citation_source-journal">Tetrahedron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">10113</span><span class="NLM_x">–</span> <span class="NLM_lpage">10126</span><span class="refDoi"> DOI: 10.1016/S0040-4020(02)01397-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1016%2FS0040-4020%2802%2901397-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=10113-10126&author=J.+B.+Sweeneyauthor=A.+Tavassoliauthor=N.+B.+Carterauthor=J.+F.+Hayes&title=%5B2%2C3%5D-Sigmatropic+rearrangements+of+didehydropiperidinium+ylids&doi=10.1016%2FS0040-4020%2802%2901397-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2FS0040-4020%2802%2901397-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4020%252802%252901397-2%26sid%3Dliteratum%253Aachs%26aulast%3DSweeney%26aufirst%3DJ.%2BB.%26aulast%3DTavassoli%26aufirst%3DA.%26aulast%3DCarter%26aufirst%3DN.%2BB.%26aulast%3DHayes%26aufirst%3DJ.%2BF.%26atitle%3D%255B2%252C3%255D-Sigmatropic%2520rearrangements%2520of%2520didehydropiperidinium%2520ylids%26jtitle%3DTetrahedron%26date%3D2002%26volume%3D58%26spage%3D10113%26epage%3D10126%26doi%3D10.1016%2FS0040-4020%2802%2901397-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Wilcken, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmermann, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lange, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joerger, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boeckler, F. M.</span><span> </span><span class="NLM_article-title">Principles and applications of halogen bonding in medicinal chemistry and chemical biology</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1363</span><span class="NLM_x">–</span> <span class="NLM_lpage">1388</span><span class="refDoi"> DOI: 10.1021/jm3012068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3012068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1363-1388&author=R.+Wilckenauthor=M.+O.+Zimmermannauthor=A.+Langeauthor=A.+C.+Joergerauthor=F.+M.+Boeckler&title=Principles+and+applications+of+halogen+bonding+in+medicinal+chemistry+and+chemical+biology&doi=10.1021%2Fjm3012068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Principles and Applications of Halogen Bonding in Medicinal Chemistry and Chemical Biology</span></div><div class="casAuthors">Wilcken, Rainer; Zimmermann, Markus O.; Lange, Andreas; Joerger, Andreas C.; Boeckler, Frank M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1363-1388</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Halogen bonding has been known in material science for decades, but until recently, halogen bonds in protein-ligand interactions were largely the result of serendipitous discovery rather than rational design.  In this Perspective, we provide insights into the phenomenon of halogen bonding, with special focus on its role in drug discovery.  We summarize the theor. background defining its strength and directionality, provide a systematic anal. of its occurrence and interaction geometries in protein-ligand complexes, and give recent examples where halogen bonding has been successfully harnessed for lead identification and optimization.  In light of these data, we discuss the potential and limitations of exploiting halogen bonds for mol. recognition and rational drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjholHfNZ9kbVg90H21EOLACvtfcHk0lif8TALGxfvBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1GrurfJ&md5=81ba024e4427fac3e7af0a4cd330909c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm3012068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3012068%26sid%3Dliteratum%253Aachs%26aulast%3DWilcken%26aufirst%3DR.%26aulast%3DZimmermann%26aufirst%3DM.%2BO.%26aulast%3DLange%26aufirst%3DA.%26aulast%3DJoerger%26aufirst%3DA.%2BC.%26aulast%3DBoeckler%26aufirst%3DF.%2BM.%26atitle%3DPrinciples%2520and%2520applications%2520of%2520halogen%2520bonding%2520in%2520medicinal%2520chemistry%2520and%2520chemical%2520biology%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1363%26epage%3D1388%26doi%3D10.1021%2Fjm3012068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Yamashita, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Izumi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kashima, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saari, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katsuhata, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muramatsu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morita, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwamatsu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hachiya, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurata, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anderson, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, S.</span><span> </span><span class="NLM_article-title">SMG-8 and SMG-9, two novel subunits of the SMG-1 complex, regulate remodeling of the mRNA surveillance complex during nonsense-mediated mRNA decay</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1105</span><span class="refDoi"> DOI: 10.1101/gad.1767209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1101%2Fgad.1767209" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=1091-1105&author=A.+Yamashitaauthor=N.+Izumiauthor=I.+Kashimaauthor=T.+Ohnishiauthor=B.+Saariauthor=Y.+Katsuhataauthor=R.+Muramatsuauthor=T.+Moritaauthor=A.+Iwamatsuauthor=T.+Hachiyaauthor=R.+Kurataauthor=H.+Hiranoauthor=P.+Andersonauthor=S.+Ohno&title=SMG-8+and+SMG-9%2C+two+novel+subunits+of+the+SMG-1+complex%2C+regulate+remodeling+of+the+mRNA+surveillance+complex+during+nonsense-mediated+mRNA+decay&doi=10.1101%2Fgad.1767209"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1101%2Fgad.1767209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1767209%26sid%3Dliteratum%253Aachs%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DIzumi%26aufirst%3DN.%26aulast%3DKashima%26aufirst%3DI.%26aulast%3DOhnishi%26aufirst%3DT.%26aulast%3DSaari%26aufirst%3DB.%26aulast%3DKatsuhata%26aufirst%3DY.%26aulast%3DMuramatsu%26aufirst%3DR.%26aulast%3DMorita%26aufirst%3DT.%26aulast%3DIwamatsu%26aufirst%3DA.%26aulast%3DHachiya%26aufirst%3DT.%26aulast%3DKurata%26aufirst%3DR.%26aulast%3DHirano%26aufirst%3DH.%26aulast%3DAnderson%26aufirst%3DP.%26aulast%3DOhno%26aufirst%3DS.%26atitle%3DSMG-8%2520and%2520SMG-9%252C%2520two%2520novel%2520subunits%2520of%2520the%2520SMG-1%2520complex%252C%2520regulate%2520remodeling%2520of%2520the%2520mRNA%2520surveillance%2520complex%2520during%2520nonsense-mediated%2520mRNA%2520decay%26jtitle%3DGenes%2520Dev.%26date%3D2009%26volume%3D23%26spage%3D1091%26epage%3D1105%26doi%3D10.1101%2Fgad.1767209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Iwatani, M.; Ito, M.; Ishibashi, Y.; Oki, H.; Sakamoto, J.; Tanaka, T.; Morishita, D.; Ito, T.; Kimura, H.; Imaeda, Y.; Aparicio, S.; Nakanishi, A.; Kawamoto, T.</span> Unpublished data. Detailed profiles of <b>52a</b> including selectivity and NMD inhibitory activity will be published separately.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Iwatani%2C+M.%3B+Ito%2C+M.%3B+Ishibashi%2C+Y.%3B+Oki%2C+H.%3B+Sakamoto%2C+J.%3B+Tanaka%2C+T.%3B+Morishita%2C+D.%3B+Ito%2C+T.%3B+Kimura%2C+H.%3B+Imaeda%2C+Y.%3B+Aparicio%2C+S.%3B+Nakanishi%2C+A.%3B+Kawamoto%2C+T.+Unpublished+data.+Detailed+profiles+of+52a+including+selectivity+and+NMD+inhibitory+activity+will+be+published+separately."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DIwatani%26aufirst%3DM." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Li, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span> </span><span class="NLM_article-title">Expeditious synthesis of hippuristanol and congeners with potent antiproliferative activities.J</span> <span class="citation_source-journal">Chem. - Eur. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">10356</span><span class="NLM_x">–</span> <span class="NLM_lpage">10359</span><span class="refDoi"> DOI: 10.1002/chem.200901732</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1002%2Fchem.200901732" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=19746484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1Kis7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=10356-10359&author=W.+Liauthor=Y.+Dangauthor=J.+O.+Liuauthor=B.+Yu&title=Expeditious+synthesis+of+hippuristanol+and+congeners+with+potent+antiproliferative+activities.J&doi=10.1002%2Fchem.200901732"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Expeditious Synthesis of Hippuristanol and Congeners with Potent Antiproliferative Activities</span></div><div class="casAuthors">Li, Wei; Dang, Yongjun; Liu, Jun O.; Yu, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">10356-10359, S10356/1-S10356/74</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Hippuristanol and its congeners with modified E and F rings are prepd.  The effect of hippuristanol and its analogs on HeLa cell proliferation and in vitro translation was detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBRZBR8sT50rVg90H21EOLACvtfcHk0lif8TALGxfvBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1Kis7fI&md5=4462fbcf3275780e6233cb8d0ed2f814</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fchem.200901732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.200901732%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.%26aulast%3DDang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26aulast%3DYu%26aufirst%3DB.%26atitle%3DExpeditious%2520synthesis%2520of%2520hippuristanol%2520and%2520congeners%2520with%2520potent%2520antiproliferative%2520activities.J%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2009%26volume%3D15%26spage%3D10356%26epage%3D10359%26doi%3D10.1002%2Fchem.200901732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Navratilova, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macdonald, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robinson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathias, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phillips, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A.</span><span> </span><span class="NLM_article-title">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">183</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.1177/1087057111422746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=10.1177%2F1087057111422746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=22068706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=183-193&author=I.+Navratilovaauthor=G.+Macdonaldauthor=C.+Robinsonauthor=S.+Hughesauthor=J.+Mathiasauthor=C.+Phillipsauthor=A.+Cook&title=Biosensor-based+approach+to+the+identification+of+protein+kinase+ligands+with+dual-site+modes+of+action&doi=10.1177%2F1087057111422746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-based approach to the identification of protein kinase ligands with dual-site modes of action</span></div><div class="casAuthors">Navratilova, Iva; MacDonald, Graeme; Robinson, Colin; Hughes, Samantha; Mathias, John; Phillips, Chris; Cook, Andrew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">183-193</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">The authors have used a surface plasmon resonance (SPR)-based biosensor approach to identify and characterize compds. with a unique binding mode to protein kinases.  Biacore was used to characterize hits from an enzymic high-throughput screen of the Tec family tyrosine kinase, IL2-inducible T cell kinase (ITK).  Complex binding kinetics was obsd. for some compds., which led to identification of compds. that bound simultaneously at both the ATP binding site and a second, allosteric site on ITK.  The presence of the second binding site was confirmed by X-ray crystallog.  The second site is located in the N-terminal lobe of the protein kinase catalytic domain, adjacent to but distinct from the ATP site.  To enable rapid optimization of binding properties, a competition-based Biacore assay has been developed to successfully identify second site noncompetitive binders that have been confirmed by X-ray crystallog. studies.  The authors have found that SPR technol. is a key method for rapid identification of compds. with dual-site modes of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_O18si5LWPLVg90H21EOLACvtfcHk0li_g3HApWVKeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVKltL8%253D&md5=8860bb955a27b7ba6b167725224c9e60</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1177%2F1087057111422746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057111422746%26sid%3Dliteratum%253Aachs%26aulast%3DNavratilova%26aufirst%3DI.%26aulast%3DMacdonald%26aufirst%3DG.%26aulast%3DRobinson%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%26aulast%3DMathias%26aufirst%3DJ.%26aulast%3DPhillips%26aufirst%3DC.%26aulast%3DCook%26aufirst%3DA.%26atitle%3DBiosensor-based%2520approach%2520to%2520the%2520identification%2520of%2520protein%2520kinase%2520ligands%2520with%2520dual-site%2520modes%2520of%2520action%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2012%26volume%3D17%26spage%3D183%26epage%3D193%26doi%3D10.1177%2F1087057111422746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i75"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01904">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_75471"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01904">10.1021/acs.jmedchem.6b01904</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings and some data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_001.csv">CSV</a>)</p></li><li><p class="inline">SPR sensorgrams; X-ray crystal structure analysis of <b>8a</b>; synthetic procedure for intermediates <b>17</b>, <b>23</b>, <b>26</b>–<b>28</b>, and <b>62</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_001.csv">jm6b01904_si_001.csv (3.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01904/suppl_file/jm6b01904_si_002.pdf">jm6b01904_si_002.pdf (530.5 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01904&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-8%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01904%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01904" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                1MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799b1cdbf933d30","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
