
<html lang="en"     class="pb-page"  data-request-id="ba87402c-3348-4303-a8e4-139e6ce24665"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b00020;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2019.62.issue-9"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase" /></meta><meta name="dc.Creator" content="Roberto&#xA;A.  Crespo" /></meta><meta name="dc.Creator" content="Qun  Dang" /></meta><meta name="dc.Creator" content="Nian E.  Zhou" /></meta><meta name="dc.Creator" content="Liam M.  Guthrie" /></meta><meta name="dc.Creator" content="Thomas C.  Snavely" /></meta><meta name="dc.Creator" content="Wen  Dong" /></meta><meta name="dc.Creator" content="Kimberly A.  Loesch" /></meta><meta name="dc.Creator" content="Takao  Suzuki" /></meta><meta name="dc.Creator" content="Lanying  You" /></meta><meta name="dc.Creator" content="Wei  Wang" /></meta><meta name="dc.Creator" content="Theresa  O’Malley" /></meta><meta name="dc.Creator" content="Tanya  Parish" /></meta><meta name="dc.Creator" content="David B.  Olsen" /></meta><meta name="dc.Creator" content="James C.  Sacchettini" /></meta><meta name="dc.Description" content="Mycobacterium tuberculosis adenosine kinase (MtbAdoK) is an essential enzyme of Mtb and forms part of the purine salvage pathway within mycobacteria. Evidence suggests that the purine salvage pathw..." /></meta><meta name="Description" content="Mycobacterium tuberculosis adenosine kinase (MtbAdoK) is an essential enzyme of Mtb and forms part of the purine salvage pathway within mycobacteria. Evidence suggests that the purine salvage pathw..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 19, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00020" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00020" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00020" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00020" /></link>
        
    
    

<title>Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00020" /></meta><meta property="og:title" content="Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0010.jpeg" /></meta><meta property="og:description" content="Mycobacterium tuberculosis adenosine kinase (MtbAdoK) is an essential enzyme of Mtb and forms part of the purine salvage pathway within mycobacteria. Evidence suggests that the purine salvage pathway might play a crucial role in Mtb survival and persistence during its latent phase of infection. In these studies, we adopted a structural approach to the discovery, structure-guided design, and synthesis of a series of adenosine analogues that displayed inhibition constants ranging from 5 to 120 nM against the enzyme. Two of these compounds exhibited low micromolar activity against Mtb with half maximal effective inhibitory concentrations of 1.7 and 4.0 μM. Our selectivity and preliminary pharmacokinetic studies showed that the compounds possess a higher degree of specificity against MtbAdoK when compared with the human counterpart and are well tolerated in rodents, respectively. Finally, crystallographic studies showed the molecular basis of inhibition, potency, and selectivity and revealed the presence of a potentially therapeutically relevant cavity unique to the MtbAdoK homodimer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00020"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00020">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00020&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00020&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00020&amp;href=/doi/10.1021/acs.jmedchem.9b00020" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 9</span><span class="cit-fg-pageRange">, 4483-4499</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/9" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01986" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00056" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of <i>Mycobacterium tuberculosis</i> Adenosine Kinase</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Roberto A. Crespo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roberto A. Crespo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Texas A&M University, College Station, Texas 77843, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roberto%0AA.++Crespo">Roberto A. Crespo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qun Dang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qun Dang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Sharp Dohme Corporation, West Point Pennsylvania 19486, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qun++Dang">Qun Dang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nian E. Zhou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nian E. Zhou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Texas A&M University, College Station, Texas 77843, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nian+E.++Zhou">Nian E. Zhou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Liam M. Guthrie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liam M. Guthrie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">College of Medicine, Texas A&M University Health Science Center, Bryan, Texas 77807, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liam+M.++Guthrie">Liam M. Guthrie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas C. Snavely</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas C. Snavely</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Texas A&M University, College Station, Texas 77843, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+C.++Snavely">Thomas C. Snavely</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wen Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wen Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Texas A&M University, College Station, Texas 77843, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wen++Dong">Wen Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kimberly A. Loesch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kimberly A. Loesch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Texas A&M University, College Station, Texas 77843, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kimberly+A.++Loesch">Kimberly A. Loesch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Takao Suzuki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Takao Suzuki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Takao++Suzuki">Takao Suzuki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lanying You</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lanying You</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lanying++You">Lanying You</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">WuXi AppTec, 288 Fute Zhong Road, Shanghai 200131, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Wang">Wei Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Theresa O’Malley</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Theresa O’Malley</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">TB Discovery Research, Infectious Disease Research Institute, 1616 Eastlake Avenue E, Seattle, Washington 98102, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Theresa++O%E2%80%99Malley">Theresa O’Malley</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tanya Parish</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tanya Parish</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">TB Discovery Research, Infectious Disease Research Institute, 1616 Eastlake Avenue E, Seattle, Washington 98102, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tanya++Parish">Tanya Parish</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7507-0423" title="Orcid link">http://orcid.org/0000-0001-7507-0423</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David B. Olsen</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David B. Olsen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Sharp Dohme Corporation, West Point Pennsylvania 19486, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#d5b1b4a3bcb18abab9a6b0bb95b8b0a7b6befbb6bab8"><span class="__cf_email__" data-cfemail="583c392e313c0737342b3d3618353d2a3b33763b3735">[email protected]</span></a>. Phone: 215-652-5250 (D.B.O.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+B.++Olsen">David B. Olsen</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">James C. Sacchettini</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">James C. Sacchettini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Biochemistry and Biophysics, Texas A&M University, College Station, Texas 77843, United States</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#cbb8aaa8a8a3aebfbf8bbfaaa6bee5aeafbe"><span class="__cf_email__" data-cfemail="c3b0a2a0a0aba6b7b783b7a2aeb6eda6a7b6">[email protected]</span></a>. Phone: (979) 845-8548 (J.C.S.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=James+C.++Sacchettini">James C. Sacchettini</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5767-2367" title="Orcid link">http://orcid.org/0000-0001-5767-2367</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00020&amp;href=/doi/10.1021%2Facs.jmedchem.9b00020" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 9</span><span class="cit-pageRange">, 4483–4499</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 19, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>4 January 2019</li><li><span class="item_label"><b>Published</b> online</span>19 April 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 May 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00020" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00020</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice/index.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a><span class="article_header-cc-by text">with CC-BY<a href="/page/policy/authorchoice_ccby_termsofuse.html" title="License">license</a></span></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00020"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2692</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">6</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00020" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Drug Design of 6-Substituted Adenosine Analogues as Potent Inhibitors of Mycobacterium tuberculosis Adenosine Kinase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Roberto\nA.&quot;,&quot;last_name&quot;:&quot;Crespo&quot;},{&quot;first_name&quot;:&quot;Qun&quot;,&quot;last_name&quot;:&quot;Dang&quot;},{&quot;first_name&quot;:&quot;Nian&quot;,&quot;last_name&quot;:&quot;E. Zhou&quot;},{&quot;first_name&quot;:&quot;Liam&quot;,&quot;last_name&quot;:&quot;M. Guthrie&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;C. Snavely&quot;},{&quot;first_name&quot;:&quot;Wen&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Kimberly&quot;,&quot;last_name&quot;:&quot;A. Loesch&quot;},{&quot;first_name&quot;:&quot;Takao&quot;,&quot;last_name&quot;:&quot;Suzuki&quot;},{&quot;first_name&quot;:&quot;Lanying&quot;,&quot;last_name&quot;:&quot;You&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Theresa&quot;,&quot;last_name&quot;:&quot;O’Malley&quot;},{&quot;first_name&quot;:&quot;Tanya&quot;,&quot;last_name&quot;:&quot;Parish&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;B. Olsen&quot;},{&quot;first_name&quot;:&quot;James&quot;,&quot;last_name&quot;:&quot;C. Sacchettini&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;9&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;4483-4499&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00020&quot;},&quot;abstract&quot;:&quot;Mycobacterium tuberculosis adenosine kinase (MtbAdoK) is an essential enzyme of Mtb and forms part of the purine salvage pathway within mycobacteria. Evidence suggests that the purine salvage pathway might play a crucial role in Mtb survival and persistence during its latent phase of infection. In these studies, we adopted a structural approach to the discovery, structure-guided design, and synthesis of a series of adenosine analogues that displayed inhibition constants ranging from 5 to 120 nM against the enzyme. Two of these compounds exhibited low micromolar activity against Mtb with half maximal effective inhibitory concentrations of 1.7 and 4.0 μM. Our selectivity and preliminary pharmacokinetic studies showed that the compounds possess a higher degree of specificity against MtbAdoK when compared with the human counterpart and are well tolerated in rodents, respectively. Finally, crystallographic studies showed the molecular basis of inhibition, potency, and selectivity and revealed the presence of a pot&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00020&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00020" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00020&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00020" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00020&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00020" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00020&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00020&amp;href=/doi/10.1021/acs.jmedchem.9b00020" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00020" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00020" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00020%26sid%3Dliteratum%253Aachs%26genre%3Darticle%26aulast%3DCrespo%26date%3D2019%26atitle%3DStructure-Guided%2BDrug%2BDesign%2Bof%2B6-Substituted%2BAdenosine%2BAnalogues%2Bas%2BPotent%2BInhibitors%2Bof%2BMycobacterium%2Btuberculosis%2BAdenosine%2BKinase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D9%26spage%3D4483%26epage%3D4499%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291308" title="Nucleic acids">Nucleic acids</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292541" title="Cavities">Cavities</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/9" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/jmcmar.2019.62.issue-9/20190509/jmcmar.2019.62.issue-9.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText"><i>Mycobacterium tuberculosis</i> adenosine kinase (MtbAdoK) is an essential enzyme of Mtb and forms part of the purine salvage pathway within mycobacteria. Evidence suggests that the purine salvage pathway might play a crucial role in Mtb survival and persistence during its latent phase of infection. In these studies, we adopted a structural approach to the discovery, structure-guided design, and synthesis of a series of adenosine analogues that displayed inhibition constants ranging from 5 to 120 nM against the enzyme. Two of these compounds exhibited low micromolar activity against Mtb with half maximal effective inhibitory concentrations of 1.7 and 4.0 μM. Our selectivity and preliminary pharmacokinetic studies showed that the compounds possess a higher degree of specificity against MtbAdoK when compared with the human counterpart and are well tolerated in rodents, respectively. Finally, crystallographic studies showed the molecular basis of inhibition, potency, and selectivity and revealed the presence of a potentially therapeutically relevant cavity unique to the MtbAdoK homodimer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43656" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43656" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><i>Mycobacterium tuberculosis</i> (Mtb), the bacterium that causes pulmonary tuberculosis (TB), represents one of the major leading causes of death worldwide by a single infectious agent. One-third of the world’s population is thought to harbor latent TB, and around 5–10% of these infected individuals are expected to develop the active disease sometime during their lifetime.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The rapid emergence of multidrug resistant and extensively drug-resistant (XDR) TB demands the development of novel chemotherapeutic agents with novel molecular targets.</div><div class="NLM_p">The purine salvage pathway is a druggable pathway within mycobacteria. In this pathway, preformed nucleobases from the product of nucleic acid breakdown are converted to their corresponding purine nucleotides by the purine salvage enzymes. Although the de novo and purine salvage pathways have not been extensively studied in Mtb, it is known that Mtb possesses all of the enzymes required for both pathways. It is currently unknown if there is restrictive regulation of the two pathways. However, by switching to the salvage pathway Mtb can bypass several chemically demanding steps.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> This has led to the hypothesis that the salvage pathway might be the most likely source of nucleotides within the hostile and nutrient-deprived microenvironment encountered by Mtb during its latent phase of infection.<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a></div><div class="NLM_p"><i>M. tuberculosis</i> adenosine kinase (MtbAdoK) performs a critical step in the purine salvage pathway within mycobacteria. The enzyme catalyzes the conversion of adenosine to adenosine monophosphate in a Mg<sup>2+</sup> and adenosine 5′-triphosphate (ATP)-dependent manner.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The crystal structure of MtbAdoK has been previously solved at high resolution with the substrate (adenosine), substrate analogue 2-fluoroadenosine, ATP analogue AMP–PCP and without substrate (apo) at resolutions of 1.90, 1.93, 1.90, and 1.50 Å, respectively.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">The crystallographic data showed that the apo and the AMP–PCP structures adopted the opened conformation of the protein where the active site is solvent exposed. In contrast, the substrate or substrate analogue complexes revealed that the lid domain of MtbAdoK undergoes a 30° movement upon substrate binding.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> This conformational change effectively brings lid domain residues Asp12, Phe116, and Phe102 in close contacts with adenosine, thereby completing the active site (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S1a</a>).<a onclick="showRef(event, 'ref5 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref8 ref9">(5,7−9)</a> As previously described, stabilization of the ribose and adenine rings are primarily mediated by π-stacking interactions with residues Phe102 and Phe116, respectively. Hydrogen bonding networks with the adenine moiety were described to occur with residues Gln172, Gln173, Ser8, and Ser36′ while the ribose forms extensive hydrogen bonding interactions with residues Gln172, Asp12, Gly48, Asn52, and catalytic base Asp257 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S1b</a>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">MtbAdoK is significantly different when compared with eukaryotic adenosine kinases.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Although the human adenosine kinase (hAdoK) and MtbAdoK are composed of a small-lid like domain and large domain, hAdoK shares less than 20% sequence identity with MtbAdoK. The hAdoK structure has been previously solved at 1.4 Å with substrate bound. The structure was initially solved with two molecules of adenosine embedded within the enzyme, one in the active site and another in cofactor (ATP) site.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> A critical difference between MtbAdoK and its eukaryotic counterparts is that MtbAdoK is a functional homodimer.<a onclick="showRef(event, 'ref5 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref8">(5,8)</a> In addition, prior to its discovery and characterization by Long et al. in 2003, the <i>adoK</i> gene was thought to be unique to eukaryotic organisms and was annotated as a general carbohydrate kinase (<i>cbhK</i>) in Mtb.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> On the basis of its amino acid sequence, MtbAdoK is more closely related to members of the phosphofructokinase B family of sugars kinases, which includes ribokinase, a homodimer and the prototypical member of this family.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">MtbAdoK is essential for the survival of the bacilli when utilizing cholesterol as a carbon source and in infected mouse models.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Even though the precise role of the purine salvage pathway under these conditions is currently unknown, biochemical and structural characterization suggests that MtbAdoK might represent a new class of bacterial adenosine kinases that differs from its eukaryotic counterparts by its unique quaternary structure and regulatory mechanisms.<a onclick="showRef(event, 'ref4 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref4 ref12 ref13">(4,12,13)</a> Furthermore, our preliminary efforts to find novel drugs and drug targets identified MtbAdoK as the target of the nucleoside analogue iodotubercidin. Mtb H37Rv-resistant mutants raised against iodotubercidin identified three nonsynonymous mutations, that is, D251N, Q172P, and G262S; all of which displayed minimum inhibitory concentrations (MICs) > 100 μM when compared with WT (18.0 ± 10.0 μM). The essentiality, differences between hAdoK and MtbAdoK and on-target activity of the nucleoside analogue described above make MtbAdoK an attractive and novel drug target.</div><div class="NLM_p">To date, most reported inhibitors of MtbAdoK have been designed as substrate surrogates to elicit the production of toxic metabolites.<a onclick="showRef(event, 'ref14 ref15 ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16 ref17 ref18">(14−18)</a> For example, Parker et al. showed that the H37Ra Mtb strain was able to uptake the adenosine analogue 2-methyl-adenosine and that MtbAdoK was responsible for its activation into the toxic metabolite 2-methyl-AMP.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Despite having some advantages, AMP or ATP toxic metabolites could inadvertently lead to undesirable and cross-species off-target effects exacerbating the identification and diminishing the value of a potential drug target.</div><div class="NLM_p last">In this work, we adopted a structure-guided approach to the design of very potent and safe adenosine analogues. To gain insights into the chemical space surrounding the active site, we solved the crystal structures of MtbAdoK complexed to several adenosine analogues at resolutions between 1.70 and 2.23 Å. This initial approach laid the foundation for the structure-guided design and synthesis of several very potent 6-substituted adenosine analogues as selective inhibitors of MtbAdoK. Several of the synthesized compounds displayed low micromolar anti-Mtb activity in a whole-cell assay, which supports the concept that inhibition of MtbAdoK can be used as an approach to treat TB. Finally, these compounds exhibited a higher degree of specificity against MtbAdoK when compared with the human counterpart, and a 6-substituted adenosine analogue from our series was well tolerated in Swiss Webster mice at high levels of exposure while demonstrating adequate pharmacokinetic properties.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45086" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45086" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Structural analysis of the MtbAdoK–adenosine (<b>1</b>) complex showed a high potential for the structure-guided drug design (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Of particular interest are positions N6 and N7 of the adenine ring and position O5′ of the ribose moiety; all of which have been the focal point of several chemistry-based structural–activity relationship (SAR) studies.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15">(12−15)</a> These studies have highlighted that substitutions at the aforementioned positions lead to inhibitory activity against MtbAdoK and hAdoK. The chemistry-based efforts have also suggested that there is a higher degree of selectivity to MtbAdoK when substitutions are located at the N6-position when compared with the N7-analogues. However, the structural and molecular basis for the reported selectivity has remained unexplained. The 5′-position offers a more practical approach to inhibition given that the critical hydroxyl group is required for catalysis.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Positions of interest for structure-guided drug design and tool compounds utilized in crystallography studies. (a) Adenosine bound to the active site of MtbAdoK (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PKM">2PKM</a>). (b) Tool compounds used to explore the chemical space surrounding the positions highlighted in (a).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To guide our drug discovery efforts and to investigate the molecular basis of selectivity and inhibition, we solved the crystal structure complexes of several known AdoK inhibitors. Iodotubercidin (<b>2</b>) and sangivamycin (<b>3</b>) were selected to investigate the conformational flexibility and chemical properties surrounding the binding pocket near the N7-position of the adenosine scaffold. Additionally, 6-methylmercaptopurine riboside (<b>4</b>) and 5′-aminoadenosine (<b>5</b>) cocrystal structures were solved to explore the 6- and 5′-positions of the adenosine scaffold, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b).</div><div class="NLM_p">Our structural studies commenced with the cocrystallization of the N7-substituted analogues. Iodotubercidin possesses an iodine group at position 7 and is a semisynthetic derivative of the natural compound tubercidin, whereas sangivamycin is a natural compound derived from <i>Streptomyces rimosus</i> and contains an amide group at position 7.<a onclick="showRef(event, 'ref12 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref12 ref19 ref20">(12,19,20)</a> Compound <b>2</b> has been noted to be a general kinase inhibitor, displaying an inhibitory activity against MtbAdoK, hAdoK, MAP kinases, and Ser/Thr kinases while compound <b>3</b> has been previously observed to be an inhibitor of protein kinase C, MtbAdoK, and hAdoK.<a onclick="showRef(event, 'ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref19 ref20 ref21">(19−21)</a></div><div class="NLM_p">Cocrystallization of MtbAdoK with compounds <b>2</b> and <b>3</b> was produced via vapor diffusion, and the structures were solved using the molecular replacement (MR) method utilizing the previously reported adenosine-bound structure.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The crystals of both complexes were determined to be in the <i>P</i>4<sub>1</sub> crystal space group with two molecules in the asymmetric unit (ASU). The MtbAdoK-<b>2</b> complex was refined to 2.10 Å resolution and displayed a closed conformation of the enzyme with a backbone rmsd of 1.14 Å amongst all Cα atoms when superimposed to the MtbAdoK–adenosine structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S1 and S2</a>). The MtbAdoK-<b>3</b> complex refined to 2.10 Å resolution displayed a closed conformation of the enzyme with a backbone rmsd of 1.15 Å amongst all Cα atoms when superimposed to the MtbAdoK–adenosine structure (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S3 and S4</a>). As in the case of the hAdoK–adenosine structure, both crystal structures showed positive electron density for the inhibitor bound to the active site in a similar position to adenosine and electron density in the cofactor (ATP) binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S2a,b</a>). However, only compound <b>2</b> displayed full electron density in the ATP binding site while compound <b>3</b> displayed partial electron density for the adenine ring only. Overall, the crystal structures of compounds <b>2</b> and <b>3</b> complexed to MtbAdoK displayed many similarities with the previously reported MtbAdoK–adenosine complex. The inhibitors were bound in the same conformation and locations as we observed in the adenosine-bound structure. Residues Phe116 and Phe102 were involved in π-stacking interactions with the adenine and ribose rings, respectively. The ribose moiety was found to be stabilized by hydrogen bonding with several residues. Asp12 oxygen OD1 H-bonds with the O2′ and O3′ of ribose, OD2 of catalytic base Asp257 H-bonds with O5′, ND2 of Asn52 H-bonds with O3′, and backbone N of Gly48 H-bonds O2′; all of which are likely to contribute significantly to the binding affinity (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S1, S3, and S5</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure complexes of compounds <b>2–5</b>. (a) <b>2</b> bound to the active site of MtbAdoK. (b) <b>3</b> bound to the active site of MtbAdoK. (c,d) N7-substituents of compounds <b>2–3</b> are buried in an active site pocket composed of residues from chain A and chain B. (e) Chimney-like cavity observed above position N6 formed by residues from chain A and chain B. (f) <b>4</b> bound to the active site of MtbAdoK. (g) Methylmercapto group of <b>4</b> is accommodated in a compound-induced pocket formed by the movement of Arg176 (red). (h) Chimney-like cavity observed in the cocrystal structures of <b>2</b> and <b>3</b> is blocked by the conformational change of Arg176 (red). (i) <b>5</b> bound to the active site of MtbAdoK. In all cases, chain A residues are colored by heteroatom and chain B residues are color magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The most notable differences between the cocrystal structures of adenosine, <b>2</b>, and <b>3</b> were observed in the interactions formed by residues Gln172, Gln173, and Ser36′. In the adenosine-bound structure, Gln172 NE2 H-bonds with O5′ (2.62 Å) and O4′ (3.26 Å) while backbone O of Gln172 H-bonds with N6 (3.26 Å). Although interactions with Gln172 were observed in the MtbAdoK-<b>3</b> complex (NE2–O4′ distance of 3.23 Å and NE2–O5′ distance of 2.83 Å), no interactions formed with Gln172 were observed in the MtbAdoK-<b>2</b> complex. A similar scenario was seen for Gln173. In the MtbAdoK-<b>1</b> structure, NE2 of Gln173 H-bonds with N1 (2.99 Å) while OE1 of Gln173 H-bonds with N6 (2.96 Å) of the adenine scaffold. On the other hand, as observed in the MtbAdoK-<b>3</b> and MtbAdoK-<b>2</b> binary complexes, Gln173 only formed H-bonds with N6 with distances of 3.25 Å (OE1–N6) and 3.29 Å (OE1–N6), respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S1, S3, and S5</a>).</div><div class="NLM_p">The crystal structures of compounds <b>2</b> and <b>3</b> showed that the iodine group of <b>2</b> and the amide group of <b>3</b> was oriented toward Ser36′. One unique aspect of MtbAdoK, being a homodimer, is that residues from one subunit presumably complete the active of the other. It has been shown that Ser36 from chain B completes the active site of chain A and vice versa by hydrogen bonding with positions N6 (OG–N6 distance of 3.43 Å) and N7 (OG–N7 distance of 2.65 Å) of the adenine ring.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Ser36 forms part of the lid-domain of MtbAdoK, and as reported before, the dimerization interface of MtbAdoK occurs via extensive van der Waals (vdW) contacts formed by the lid-domain of each chain.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> The observed orientation of both N7-substituted adenosine analogues (<b>2–3</b>) suggests that the N7-position is obstructed by chain B. That is, chain B represents an imposing physical barrier at this position, thus limiting drug discovery efforts to the active site of the enzyme (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c,d). The MtbAdoK-<b>3</b> structure showed that the −NH<sub>2</sub> of the amide group of <b>3</b> formed a H-bond with OG of Ser36′ (3.00 Å, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Table S3</a>). In contrast, H-bond interactions mediated by Ser36′ with compound <b>2</b> were not observed in the MtbAdoK-<b>2</b> complex. Instead, the interaction is of vdW nature with OG–I distance of 3.07 Å (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Table S1</a>). Finally, the N7-substituents of <b>2</b> and <b>3</b> were observed to be buried in a pocket formed by several conserved active site residues of MtbAdoK including Phe102 (Cys123 in hAdoK) and Phe116 (Phe170 in hAdoK).</div><div class="NLM_p">Overall, structural analysis of the N7-analogues (<b>2–3</b>) suggested that bulky substitutions at the N7-position are likely to be favorably accommodated in the active site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c,d). Our structural findings go well in accordance with previously reported chemistry-based SAR efforts focused on N7-substituted adenosine analogues, where Snášel et al. reported that several N7-anthracene or N7-phenanthrene derivatives displayed high potency against MtbAdoK.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The same group synthesized several N7-chain-extended bulky groups that displayed low potency against the enzyme more than likely due to occlusion by chain B, validating our structural assessment.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> The bulky iodine group of <b>2</b> is observed to be completely buried in a pocket formed by residues Phe116, Phe102, Gln172, Gln173, Ala114, Ser115, Leu38′ and Ser36′ leading to several polar and vdW interactions (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c). In contrast, the amide group of <b>3</b> occupies a smaller portion of the pocket (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>d). We also observed that the amide group of <b>3</b> participates in many of the conserved H-bond interactions (Gln172 with O5′ and O4′ and Gln173 with N6) when compared with the MtbAdoK-<b>1</b> complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S3 and S5</a>). In contrast, the bulky iodine group of <b>2</b> sterically hinders many of the conserved interactions described in the MtbAdoK-<b>1</b> complex, including all of the H-bonds formed by Gln172 and Ser36′ (Gln172 with O5′, O4′ and N6 and Ser36′ with N6 and N7) and the H-bond between Gln173 and N1 (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S1 and S5</a>). Our enzymatic assay also showed that compound <b>2</b> is more potent than compound <b>3</b> displaying 50% inhibitory concentrations (IC<sub>50</sub>) of 1.2 and 16.5 μM, respectively, further validating our structural assessment. Finally, the crystal structure complexes of the N7 analogues revealed the presence of a “chimney-like” cavity situated above position N6 of the adenine scaffold and that is formed by residues from both chains of the MtbAdoK homodimer, thereby representing a unique structural feature of MtbAdoK (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>e).</div><div class="NLM_p">Next, we decided to explore the chemical space surrounding the binding pocket near the N6-position by cocrystallizing MtbAdoK with 6-methylmercaptopurine riboside. Compound <b>4</b> is a derivative of mercaptopurine, a drug used to treat acute lymphatic leukemia and possess a methylmercapto group at the N6-position of the adenosine scaffold. The binary complex of MtbAdoK-<b>4</b> was refined to 2.0 Å resolution with two molecules in the ASU in the P4<sub>1</sub> crystal space group (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>f, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S6 and S7</a>). Like the cocrystals structures of <b>2</b> and <b>3</b>, the MtbAdoK-<b>4</b> binary complex displayed a closed conformation of the enzyme with a backbone rmsd of 1.46 Å amongst all Cα atoms when compared with the MtbAdoK-<b>1</b> complex. Positive electron density was also observed for a molecule of <b>4</b> bound to the ATP site. As in the case of the MtbAdoK-<b>3</b> structure, we were only able to model the adenine moiety due to the absence of electron density of the ribose (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S3</a>).</div><div class="NLM_p">The MtbAdoK-<b>4</b> structure was very similar to the cocrystal structures of compounds <b>2</b>–<b>3</b> and MtbAdoK–adenosine complex. The aforementioned binding interactions formed by residues Phe102, Phe116, Asp12, Asp257, Ser8, Asn52, Gly48, and Val49 were maintained. The most notable differences were found with residues Gln173 and Ser36′. Foremost, the critical H-bonds formed by NE2 and OE1 of Gln173 with N1 and N6 of compound <b>4</b> were not observed. Also, the H-bonds formed by OG of Ser36′ with N6 and N7 of compound <b>4</b> were not observed in the cocrystal structure. The lack of interactions with residues Ser36′ and Gln173 can be attributed to steric shielding by the methylmercapto group of <b>4</b>. Detailed inspection of the binary complex revealed that the methylmercapto group of <b>4</b> was oriented toward a pocket formed by residues Gln173, Gln172, Met121, Phe116, Arg176, Leu38′, and Ser36′ (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>g). Arg176 was observed to be oriented toward the solvent in the MtbAdoK–adenosine and crystals structures of compounds <b>2</b> and <b>3</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c,d). However, the MtbAdoK-<b>4</b> structure revealed that Arg176 had well-defined electron density for the side chain oriented toward the methylmercapto group of <b>4</b>. The reorientation of Arg176 led to the formation of a new compound-induced pocket between residues Phe116, Met121, Gln172, Gln173, Arg176, Leu38′, and Ser36′, consequently leading to extensive vdW and polar interactions between the residues mentioned above and methylmercapto group (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>g). Remarkably, the compound-induced movement of Arg176 closed the “chimney-like” cavity we saw in the cocrystal structures of compounds <b>2</b> and <b>3</b>, indicating that residues surrounding the N6-position are flexible (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>c–e,g,h). The orientation and proximity (4.4 Å) of the cationic CZ of Arg176 with respect to the sulfur atom of the methylmercapto group suggests the possibility of cation–polar interactions. Finally, it should be noted that the methylmercapto group seemed to occupy a small portion of the pocket, indicating that bulky groups might be good candidates to further improve contacts at the position 6.</div><div class="NLM_p">Just like compound <b>2</b>, compound <b>4</b> displayed good potency (IC<sub>50</sub> of 2.3 μM) in our enzymatic assay. The similar potencies can be attributed to the comparable vdW and polar contacts formed by the substituents within their respective pockets. Although these two inhibitors have substitutions at different positions within the adenine ring, their mode of interaction with the protein is similar. That is, both substitutions prevented H-bonds with positions N1, N6, and N7 of the adenine moiety. Our structure-guided SAR studies revealed that the iodine group of <b>2</b> sterically hinders the interactions with the adenine atoms mentioned above and that are formed by residues Ser36′, Gln172, and Gln173. In contrast, the methylmercapto group of <b>4</b> not only sterically blocked the aforementioned interactions but also was accommodated in a new compound-induced pocket that is comprised in part by residues from both chains of the MtbAdoK homodimer (Gln173, Gln172, Met121, Phe116, Arg176, Leu38′, and Ser36′). It should be noted that compound <b>2</b> has been reported to be a very potent inhibitor of hAdoK (<i>K</i><sub>ihAdoK</sub> ≈ 30 nM and <i>K</i><sub>iMtbAdoK</sub> ≈ 210 nM).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The different degrees of potencies can be attributed to the way compound <b>2</b> interacts with the enzymes. In our MtbAdoK-<b>2</b> complex, the compound is shown to be partially solvent exposed because of the presence the “chimney-like” cavity. In contrast, the previously solved hAdoK-<b>2</b> complex showed that the compound is completely buried within the hAdoK monomer because of a unique latch-like region of hAdoK comprised by residues 23–57 of the enzyme’s lid-domain (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S4</a>).<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Also, the lack of selectivity of compound <b>2</b> can be attributed to the fact that the compound is buried within several conserved residues located in the active site. Overall, the N7 versus N6 substituents, exemplified by compounds <b>2–3</b> and <b>4</b>, seemed to be accommodated in mutually exclusive pockets. The N7 pocket is formed by several conserved active site residues while the N6 pocket is compound-induced and formed by unique MtbAdoK residues Gln172, Gln173, Ser36′, Leu38′, and flexible Arg176.</div><div class="NLM_p">Finally, to explore the potential for modifying the 5′-position and to evaluate if the ATP channel can be utilized as a viable route for drug design, we solved the crystal structure of 5′-aminoadenosine bound to MtbAdoK (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S5</a>). As in the case of the compounds described above, compound <b>5</b> has been previously shown to possess inhibitory activity against AdoKs.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The MtbAdoK-<b>5</b> cocrystal was obtained in the same condition as the reported adenosine-bound structure and was refined to 1.95 Å resolution in the <i>P</i>3<sub>1</sub>21 crystal space group with one molecule in the ASU. The binary complex displayed a closed conformation of the enzyme with a backbone rmsd of 0.28 Å amongst all Cα backbone atoms when compared with the MtbAdoK–adenosine complex, and the inhibitor was bound in a very similar fashion as adenosine (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>i, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S8 and S9</a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S6</a>). Unlike, the cocrystal structures of compounds <b>2–4</b>; the MtbAdoK-<b>5</b> structure had no electron density for a molecule of <b>5</b> bound to the ATP site. The interactions observed between <b>5</b> and MtbAdoK were remarkably similar to those previously described in the adenosine-bound complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S8 and S5</a>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Indeed, the main difference was the interaction formed by the 5′-NH<sub>2</sub>, displaying a H-bond with OD2 of Asp257 (2.61 Å). Here, enzymatic inhibition (IC<sub>50</sub> of 8.0 μM) is attributed to the absence of the hydroxyl group that is required for catalysis.</div><div class="NLM_p">On the basis of the preliminary structure-guided SAR efforts with compounds <b>2–5</b>, we screened a focused library of ∼140 nucleoside analogues with substitutions at N6, N7, and/or 5′ positions. The very potent compound identified in this screen (compound <b>6</b>, IC<sub>50</sub> of 196.5 ± 20.3 nM) had a bulky thiophene group at position 6, validating the pharmacological relevance of N6-substituted adenosine analogues (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). To investigate the structural basis of the ∼10× increase in potency when compared with compounds <b>2</b> and <b>4</b> and to advance our structure-guided drug discovery efforts, we crystallized the MtbAdoK-<b>6</b> binary complex. The cocrystal structure of compound <b>6</b> was solved by MR and refined to a resolution of 2.10 Å in the <i>P</i>2<sub>1</sub>2<sub>1</sub>2<sub>1</sub> crystal space group with two molecules in the ASU (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S10 and S11</a>). As observed in the structures described above, the MtbAdoK-<b>6</b> complex displayed a closed conformation with a backbone rmsd of 2.18 Å amongst all Cα atoms when compared with the MtbAdoK–adenosine complex. Unlike the structures of compounds <b>2–5</b>, the MtbAdoK-<b>6</b> complex revealed that the compound binds in a different orientation with respect to adenosine (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c) and that the bulky substituent prevented full lid domain closure by leaving a gap of 2.94 Å when compared with the fully closed MtbAdoK–adenosine complex. As in the case of the crystal structure complexes of compounds <b>2–4,</b> the MtbAdoK-<b>6</b> structure showed electron density in the active site and partial electron density in the ATP site (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S7</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of the MtbAdoK-<b>6</b> complex. (a) Chemical structure of compound <b>6</b>. (b) <b>6</b> bound to the active site of MtbAdoK. (c) Superimposition of the cocrystal structures of compound <b>6</b> (gray) and adenosine (blue). (d) Surface representation of residues forming the pocket where the thiophene group gets accommodated. Chain B is colored magenta, Arg176 is colored red, and chain A is colored by heteroatom. (e) Arg176 adopts a different conformation in the cocrystal structure of <b>4</b> (blue) when compared with the cocrystal structure of <b>6</b> (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The cocrystal structure of compound <b>6</b> showed that the interactions formed by residues Phe116, Phe102, Ser8, Asp12, Asn52, Asp257, Val49, and Gly48 were preserved. The significant differences were found once more with residues Gln172, Gln173, and Ser36′. In fact, none of the H-bond interactions formed between Gln172 (with O4′, O5′ and N6), Gln173 (with N1 and N6), and Ser36′ (with N6 and N7) were observed (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Table S10</a>). Instead, these residues were involved in weaker vdW interactions with the adenine ring and ribose rings. The thiophene moiety formed unique contacts. This group was observed to be completely buried within the newly identified and compound-induced pocket that is formed by residues Phe102, Phe116, Gln172, Gln173, Arg176, Ser36′, Leu38′ and Phe37′ (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d). Closer inspection of the pocket showed that Arg176 reorients once more to accommodate the bulkier thiophene group, confirming the plasticity of this residue (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>e). Furthermore, the orientation of the aromatic thiophene ring with respect to Phe116 suggests the possibility of parallel-displaced π-staking interactions, whereas the orientation of cationic CZ of Arg176 with respect to the sulfur atom of the thiophene group (CZ–S distance of 3.97 Å) suggests the possibility of T-shaped stacking interactions. Overall, the extensive vdW interactions within the pocket coupled to the stacking interactions are the most likely molecular basis for the significant increase in potency of <b>6</b> when compared with compounds <b>2–4</b>. Finally, the orientation of Arg176, when compared with the MtbAdoK-<b>4</b> structure, suggested that the “chimney-like” cavity we observed in the crystal structure complexes described above might be accessible if larger substitutions are utilized to force Arg176 to give access into the cavity.</div><div class="NLM_p">To further characterize compound <b>6</b>, we determined the antimycobacterial and cytotoxic profile of the compound. The antimycobacterial assay with compound <b>6</b> showed a half maximal effective concentration (EC<sub>50</sub>) of 25.6 ± 2.4 μM with a 50% cytotoxic concentration (CC<sub>50</sub>) > 100 μM against human dermal fibroblasts (HDF) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figures S8 and S9</a>). Indicating that, in general, 6-substituted adenosine analogues might show selectivity to MtbAdoK over the human counterpart.</div><div class="NLM_p">Overall, the structure-based SAR efforts showed that the N6, N7, and 5′ positions are indeed conducive to inhibition of the MtbAdoK enzyme through different molecular and structural means. The 5′-position provides a mechanistic approach for inhibition by taking advantage of the critical hydroxyl group required for catalysis and provides a platform to design inhibitors that might take advantage of the ATP channel. On the other hand, the crystal structures of the N7- and N6-substituted adenosine analogues showed that the substitutions sterically hindered conserved H-bonds formed by residues Gln172, Gln173, and Ser36′ with key positions N1, N6, and N7 of the adenine ring; all of which have been noted to be critical for substrate binding and recognition for human, rabbit, and <i>Toxoplasma gondii</i> adenosine kinase.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The crystal structures of the N7-substituted analogues (<b>2–3</b>) revealed the presence of a “chimney-like” cavity that extends above the N6-position, and that is formed by the MtbAdoK homodimer. However, the MtbAdoK-<b>4</b> complex showed that the cavity is closed by a conformational movement of Arg176 to interact with the N6-substituent of <b>4</b> thereby forming a new and compound-induced pocket. Identification of the very potent compound <b>6</b> revealed that the bulky aromatic thiophene group is not only completely buried within the newly identified pocket but also suggested that bulkier substitutions could open back up the “chimney-like” cavity.</div><div class="NLM_p">Despite offering inhibitory activity against MtbAdoK, the N7-substituents were observed to be buried in a pocket that is formed by several conserved active site residues and that the substituent itself is physically blocked by the dimerization event of MtbAdoK. Consequently, limiting medicinal chemistry efforts to the conserved active site. In contrast, the N6-position offers a better route for structure-guided drug design by providing the possibility of exploiting many contacts formed by the unique oligomerization state of MtbAdoK. In addition, it is possible that the presence of this cavity is the structural basis behind the reported higher degree of specificity conferred by N6-substituted analogues when compared with hAdoK.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Taken together, we hypothesized that chain extension via bulky substitutions at the N6-position might trigger a conformational change of Arg176, thereby opening the “chimney-like” cavity, thus providing a unique route to increase potency and selectivity.</div><div class="NLM_p">We next set out to systematically synthesize a series of 6-substituted adenosine analogues. Synthesis of the novel adenosine analogues (<b>7–18</b>) is shown in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Starting from the known 6-chloro-9<i>H</i>-purine <b>A0</b> or <b>A1</b> and TBS protected <b>A2</b>, 4-biphenyl and 4-Br-phenylpiperazinyl moieties were introduced by substitution in EtOH to afford <b>B1</b> and <b>B2</b> that was converted to variety of 4-arylphenyl(<i>R</i>)piperazine by using different arylboronic acids followed by removal of acetonide with/without the TBS group which gave the triol adenosines, <b>7</b>, <b>9</b>, <b>10</b>, <b>12</b>, <b>13</b>, <b>14</b>, <b>16</b>, and <b>18</b>, respectively.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23,24)</a> The cyclopentane <b>15</b> was directly synthesized from (1<i>R</i>,2<i>S</i>,3<i>R</i>,5<i>R</i>)-3-(6-chloro-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)cyclopentane-1,2-diol <b>A3</b> without alcohol protection.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The C–C bond at 6-position of purine was formed by Suzuki–Miyaura or Sonogashira cross-coupling to lead to alkyne analogues <b>11</b> and <b>17</b>. To the tri-acetyl protected 7-Cl-imidazopyridine <b>A4</b>, C–N coupling in the presence of second generation RuPhos precatalyst proceeded smoothly and global deprotection of acetyl groups by ammonium hydroxide gave the triol <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> and chemistry section of the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Adenosine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TBSCl, imidazole, DCM, 15 °C, 17 h, 99%; (b) 1-phenylpiperazine or 1-([1,1′-biphenyl]-4-yl)piperazine or 1-(4-bromophenyl)piperazine, DIEA, EtOH, 70–80 °C, 17 h, 13–90%; (c) ArB(OH)<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, XPhos Pd G2, THF/H<sub>2</sub>O, 70 °C, 17 h, 49–87%; (d) TFA, THF/H<sub>2</sub>O, 15–25 °C, 2–17 h, 13–67%; (e) 4,4,5,5-tetramethyl-2-(4-(phenylethynyl)phenyl)-1,3,2-dioxaborolane, K<sub>3</sub>PO<sub>4</sub>, XPhos Pd G2, THF, 70 °C, 12 h, 95% or 4-ethynyl-1,1′-biphenyl, Cs<sub>2</sub>CO<sub>3</sub>, CuI, XPhos Pd G2, CH<sub>3</sub>CN, 90 °C, 17 h, 27%; (f) 1-([1,1′-biphenyl]-4-yl)piperazine, Cs<sub>2</sub>CO<sub>3</sub>, 1-([1,1′-biphenyl]-4-yl)piperazine, RuPhos Pd G2, <i>tert</i>-amyl-OH, 100 °C, 17 h, 50%; and (g) NH<sub>4</sub>OH, MeOH, 15 °C, 24 h, 21%.</p></p></figure><div class="NLM_p">All synthesized adenosine analogues were characterized for inhibitory activity against MtbAdoK and hAdoK. The compounds were also tested against Mtb and HDF cells to profile their antimycobacterial and cytotoxic properties, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We began our efforts with compound <b>7</b>. This compound showed excellent potency and selectivity against MtbAdoK when compared with hAdoK. We hypothesized that the piperazine group might lead to favorable contacts with flexible residue Arg176 while the benzene ring could extend toward the unique cavity, thereby conferring selectivity.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR Data for Synthesized Adenosine Analogues with Substitutions at Position 6<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0009.gif" alt="" id="GRAPHIC-d7e1291-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">ID</th><th class="colsep0 rowsep0" align="center">R or R<sup>6</sup></th><th class="colsep0 rowsep0" align="center">Y</th><th class="colsep0 rowsep0" align="center">Z</th><th class="colsep0 rowsep0" align="center" char="±">MtbAdoK <i>K</i><sub>i</sub> (nM)</th><th class="colsep0 rowsep0" align="center">hAdoK <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">Mtb EC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">HDF CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">7</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">120.2 ± 0.9</td><td class="colsep0 rowsep0" align="left">5.1 ± 0.5</td><td class="colsep0 rowsep0" align="left">≥65.0</td><td class="colsep0 rowsep0" align="left">30.9 ± 4.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="char" char="±">16.3 ± 0.1</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">9</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">18.5 ± 0.06</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">18.9 ± 0.5</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">≥30.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">11</td><td class="colsep0 rowsep0" align="left">R<sup>6</sup></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">19.9 ± 0.03</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">≥50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">12</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">21.3 ± 0.2</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">13</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">23.0 ± 1.1</td><td class="colsep0 rowsep0" align="left">≥15.1</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">25.5 ± 2.8</td><td class="colsep0 rowsep0" align="left">0.41 ± 0.07</td><td class="colsep0 rowsep0" align="left">≥25.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">27.5 ± 3.5</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">16</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">32.6 ± 0.3</td><td class="colsep0 rowsep0" align="left">1.6 ± 0.1</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">17</td><td class="colsep0 rowsep0" align="left">R<sup>6</sup></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">48.0 ± 0.6</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">1.7 ± 0.02</td><td class="colsep0 rowsep0" align="left">3.5 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">18</td><td class="colsep0 rowsep0" align="left">R</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="char" char="±">5.3 ± 0.07</td><td class="colsep0 rowsep0" align="left">>50.0</td><td class="colsep0 rowsep0" align="left">4.0 ± 0.2</td><td class="colsep0 rowsep0" align="left">>50.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The error is reported as ± SD of 2 experiments.</p></div></div></div><div class="NLM_p">To investigate the structural basis of inhibition and selectivity of <b>7</b>, we cocrystallized the compound with MtbAdoK. The crystal structure of the MtbAdoK-<b>7</b> binary complex was solved by MR in the <i>P</i>4<sub>1</sub> crystal space group with two molecules in the ASU and refined to a resolution of 1.70 Å (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a–c). Like the structures described above, the MtbAdoK-<b>7</b> complex displayed a closed conformation of the enzyme with a backbone rmsd of 1.52 Å amongst all Cα atoms when compared with the MtbAdoK–adenosine structure. Unlike the structures described above, the MtbAdoK-<b>7</b> structure had no electron density for a molecule bound to the ATP site (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S10</a>). Overall, the interactions seen between the adenine ring and ribose scaffold were very similar to those observed in the MtbAdoK–adenosine complex (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S12 and S13</a>). Interestingly, the most significant differences were found once more with residues Gln172, Gln173, and Ser36′. Here, Gln173 and Ser36′ were not observed to make any of the critical H-bonds with positions N1, N6, and N7 while Gln172 retains the H-bonds with O4′ of ribose (NE2–O4′ distance of 3.03 Å) and O5′ (NE2–O5′ distance of 3.29 Å) but lacks the H-bond with N6. As in the case of the MtbAdoK-<b>6</b> structure, superimposition of the crystal structure complexes of MtbAdoK–adenosine and MtbAdoK-<b>7</b> showed that the bulky substitution of <b>7</b> causes the compound to bind in a different orientation with respect to adenosine with notable differences in critical lid domain residues Asp12, Phe102, and Phe116 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). Here, the lid domain gap with respect to the MtbAdoK-<b>1</b> complex was an average of 2.45 Å. We also observed that the phenylpiperazine forces Arg176 to open the “chimney-like” cavity by reorienting the residue back to the solvent. The crystal structure also showed that the piperazine group redirects the distal benzene group within the enzyme’s active site groove and that there was sufficient space to incorporate additional groups at the distal benzene ring to fully occupy the cavity (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structure of the MtbAdoK-7 complex. (a) 7 bound to the active site to MtbAdoK. Chain A is colored gray and chain B is colored magenta. (b) The bulky substituent of 7 (gray) forces the compound to bind in a different orientation with respect to adenosine (blue; PDB ID 2PKM). (c) The bulky substituent forces Arg176 to open the cavity while the distal benzene ring is reoriented back into the active site groove that is formed by residues of chain A (heteroatom) and chain B (magenta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As noted above, another position of interest was the 5′-position of the ribose scaffold. The preliminary structure-based SAR with compound <b>5</b> suggested that bisubstrate-like inhibitors might be able to take advantage of the ATP channel. To investigate if the ATP channel is a viable route for drug design, we synthesized several 5′-substituted adenosine analogues (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S11</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Table S14</a>). Despite having sufficient space to accommodate large groups, bulky phenyl-substituents at the 5′-position resulted in inhibitors with negligible inhibitory activity. Indeed, only one of the synthesized compounds displayed some measurable inhibitory activity against the enzyme with ∼53% inhibition at 12.5 μM of the compound. The lack of inhibitory activity of these analogues may be due to suboptimal angles adopted by the bulky substitution.</div><div class="NLM_p">The MtbAdoK-<b>7</b> complex suggested that further extension at the distal benzene group could be accommodated in the cavity and that rigid substituents at the 6-position might orient the substituent within the cavity. On the basis of these observations, we synthesized compounds <b>8–17</b>. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, all compounds displayed high potency against MtbAdoK with <i>K</i><sub>i</sub> values ranging from ∼16 to 48 nM. Overall, the SAR results are in accordance previous observations where it was noted that 6-substituted adenosine analogues displayed a higher degree of specificity to MtbAdoK when compared with hAdoK (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref12 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref14">(12,14)</a> The apparent lack of correlation between enzymatic inhibition and intracellular activity might be due to several reasons such as a lack of permeability, inefficient import, or rapid export. For example, it is known that Mtb possesses over 30 ATP-binding cassettes (ABC) transporters which are involved in the uptake and export of many molecules including nucleosides.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> In addition, Mtb possesses several classes of drug efflux pumps including ABC transporters, the small multidrug resistance superfamily, the major facilitator superfamily, and the resistance nodulation cell division superfamily; all of which have been attributed to efflux of many classes of antimycobacterial drugs.<a onclick="showRef(event, 'ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31 ref32">(29−32)</a> It is currently unknown how Mtb is able to uptake, discriminate, import, and export this series of compounds. Further experiments such as resistant mutant isolation, knockdown or overexpression strains, and systematic interrogation of potential transporters and efflux pumps will be needed to investigate the intracellular fate of the series.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Summary of SAR results for compounds <b>8–17</b>. Compound <b>7</b> bound to the active site of MtbAdoK. Based on the MtbAdoK-<b>7</b> complex, compounds <b>8–17</b> where synthesized. Chain A is colored white, and chain B is colored magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compounds <b>11</b> and <b>17</b> were designed to test different orientations of the 6-substituent within the cavity. These compounds possess a rigid triple bond instead of the flexible piperazine group. Because compound <b>17</b> displayed antimycobacterial activity, we decided to determine the cocrystal structure of MtbAdoK with compound <b>17</b>. The crystal structure of the MtbAdoK-<b>17</b> complex was solved by MR in the <i>P</i>4<sub>1</sub> crystal space group with two molecules in the ASU and refined to a resolution of 2.23 Å (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a–c and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S15 and S16</a>). The binary complex displayed a closed conformation of the enzyme with a backbone rmsd of 1.40 Å amongst all Cα atoms when compared with the MtbAdoK–adenosine structure. Just like the MtbAdoK-<b>7</b> structure, the MtbAdoK-<b>17</b> binary complex had no electron density for another molecule of <b>17</b> bound to the ATP site. Superimposition of the crystal structure complexes of compounds <b>17</b> and adenosine also showed that the bulky substituent of <b>17</b> causes the compound to bind in a different orientation with respect to adenosine, leaving lid domain gap of 1.81 Å when compared with the MtbAdoK–adenosine complex (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of the MtbAdoK-<b>17</b> complex. (a) <b>17</b> bound to the active site of MtbAdoK. (b) The large bulky substitution gets accommodated in the “chimney-like” cavity. (c) Compound <b>17</b> (gray) binds in a different orientation with respect to adenosine (blue). (d) Superimposition of crystal structure complexes of <b>17</b> (gray) and <b>7</b> (blue). Chain B residues forming the distal part of the “chimney-like” cavity colored magenta while chain A residues are colored by heteroatom.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The MtbAdoK-<b>17</b> binary complex showed that the compound binds to the active site of the enzyme forming similar contacts with the protein as observed with the adenosine-bound complex (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>a, <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Tables S15 and S5</a>). Structural differences were found again with residues Gln172, Gln173, and Ser36′. Only Gln173 was observed to form a weak H-bond with endocyclic N1 of the adenine ring (NE2–N1 distance of 3.21 Å) while residues Ser36′ and Gln172 were not observed to make any of the critical H-bond interactions described above. The major difference between the MtbAdoK-<b>7</b> and MtbAdoK-<b>17</b> structures is the orientation that the 6-substituent adopts as it goes into the “chimney-like” cavity. The cocrystal structure of the MtbAdoK-<b>7</b> complex suggests that the piperazine-containing adenosine analogues might reorient to the active site groove that is composed of hydrophobic residues from both chains including Ala175.A, Leu38.B, Leu35.B, Phe37.B, and polar residues Ser36.B and Arg176.A. In contrast, the rigidity imparted by the triple bond of compound <b>17</b> limits the substitution to adopt a bond angle of 180° with respect to the 6-position. Consequently, one side of the distal benzene ring of <b>17</b> was seen to be oriented toward backbone and main chain atoms of Phe37.B while the other side was observed to be partially solvent exposed (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>d and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">S12</a>).</div><div class="NLM_p">Finally, on the basis of the chemistry and structure-guided SAR efforts with compounds <b>7–17</b>, we synthesized compound <b>18</b>. This compound is an analogue of compounds <b>8</b> and <b>15</b>. The major difference is that <b>18</b> contains endocyclic atoms N1 and O4′ which we observed in the structures described above that they were involved in key H-bond interactions with residues Gln172 and Gln173, and the SAR was suggested to be relevant in potency. Indeed, compound <b>18</b> was our most potent derivative, exhibiting a <i>K</i><sub>i</sub> of ∼5 nM against MtbAdoK, no cytotoxicity, no activity against hAdoK, and an EC<sub>50</sub> of ∼4.0 μM against Mtb. Overall, the observed differences in binding modalities of reorientation within the enzyme versus solvent exposed could explain the overall higher potency of the piperazine-substituted analogues. Furthermore, the predominantly hydrophobic character of the “chimney-like” cavity goes in accordance with the SAR studies where hydrophobic substitutions displayed a higher degree of potency when compared with the polar substituted analogues (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_p">To further characterize our most potent derivative (<b>18</b>), we performed steady-state kinetic experiments and in vivo pharmacokinetics and tolerability studies. The kinetics experiments showed that increasing the concentration of adenosine in the presence of a fixed inhibitor concentration led to an increase of the <i>K</i><sub>m</sub>, consistent with a competitive mode of inhibition (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Figure S13a,b</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Table S17</a>). The <i>K</i><sub>m</sub> value for adenosine determined with our enzymatic assay was 1.71 ± 0.02 μM, well in agreement with previously reported values of 0.8–3.4 μM.<a onclick="showRef(event, 'ref4 ref33'); return false;" href="javascript:void(0);" class="ref ref4 ref33">(4,33)</a> To determine in vivo pharmacokinetic parameters, we administered compound <b>18</b> to female Swiss Webster mice via oral gavage as a single dose at 10 mg·kg<sup>–1</sup> and quantified plasma concentrations at 0, 0.25, 0.5, 1, 2, 4, 6, 8, and 24 h by liquid chromatography–mass spectrometry (LC/MS, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>a).<a onclick="showRef(event, 'ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref34 ref35">(34,35)</a> Compound <b>18</b> dosed at 10 mg·kg<sup>–1</sup> achieved a <i>C</i><sub>max</sub> of 0.13 ± 0.02 μg·mL<sup>–1</sup> at approximately 4 h with an AUC<sub>0–24h</sub> of 1.80 μg·h·mL<sup>–1</sup>. At this dose level, compound <b>18</b> was well tolerated.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In vivo tolerability studies were also performed by administering compound <b>18</b> to cohorts of female Swiss Webster mice (<i>n</i> = 3) at 50, 100, and 200 mg·kg<sup>–1</sup> via oral gavage once daily for 3 days. We evaluated mice twice daily for signs of acute toxicity and observed no adverse effects or weight loss (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Table S18</a>). Blood collected at 0, 1, 4, and 24 h after the first administered dose was used to determine PK for compound <b>18</b> at these three higher dose levels (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Data points collected from the 10 mg·kg<sup>–1</sup> study was included for comparison. Compound <b>18</b> achieved a maximum plasma concentration for all dose levels (50, 100, and 200 mg·kg<sup>–1</sup>) at approximately 4 h with <i>C</i><sub>max</sub> of 0.54 ± 0.22 μg·mL<sup>–1</sup> (1.1 ± 0.4 μM), 0.92 ± 0.09 μg·mL<sup>–1</sup> (1.9 ± 0.2 μM), and 1.58 ± 0.38 μg·mL<sup>–1</sup> (3.2 ± 0.8 μM), respectively (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>b,c). Calculated exposure for these higher dose levels improved overall exposure to 9.8, 15.3, and 23.2 μg·h·mL<sup>–1</sup>. On the basis of our preliminary pharmacokinetic data, compound <b>18</b> is very well tolerated during acute exposure. Most notably, at the highest concentration tested, <b>18</b> falls within the range of EC<sub>50</sub> value we determined in our in vitro assay (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Preliminary pharmacokinetics of compound <b>18</b>. (a) Oral PK study conducted at 10 mg·kg<sup>–1</sup> utilizing 9 Swiss Webster Female mice with three animal bleeds per time point. Maximum plasma concentration of 0.13 ± 0.02 μg·mL<sup>–1</sup> observed at 4 h following gavage. The AUC<sub>0–24h</sub> of 1.8 μg·h·mL<sup>–1</sup> was calculated using the linear trapezoid rule method. (b) Snapshot PK studies conducted from plasma samples taken from tolerability mice at 50, 100, and 200 mg·kg<sup>–1</sup> following first initial dose. 10 mg·kg<sup>–1</sup> data included for comparison. (c) <i>C</i><sub>max</sub>, <i>t</i><sub>max</sub>, and AUC of compound <b>18</b> at four dose levels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87153" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87153" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">These studies represent a systematic structure-guided approach to the rational design of very potent MtbAdoK inhibitors with low micromolar activity against Mtb. Furthermore, our studies offer a structural explanation behind the specificity of previously reported N6-substituted adenosine analogues.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15">(12−15)</a> That is, substitutions at the 6-position of the adenine scaffold might be accommodated in a compound-induced pocket that is formed within the unique oligomerization state of MtbAdoK when compared with hAdoK and other eukaryotic adenosine kinases. Unlike the crystal structures of compounds <b>2–6</b>, the crystal structures of <b>7</b> and <b>17</b> did not display any electron density for a compound bound to the ATP site suggesting that there might be a size threshold conferring specificity to the active site versus ATP site. Therefore, future strategies to design bisubstrate-like inhibitors should focus on small substituents at the N6-position with bulky groups at the 5′-position.</div><div class="NLM_p last">The crystal structures of the bulky 6-substituted adenosine analogues suggested that the reorientation conferred by the piperazine group might lead to more favorable contacts with the enzyme, thereby leading to a higher degree of potency. The predominantly hydrophobic character of the cavity goes in accordance with the SAR studies demonstrating that hydrophobic substitutions were preferred at the 6-position. Moreover, the lid domain gaps and different orientation of compounds <b>6</b>, <b>7</b>, and <b>17</b> with respect to the MtbAdoK–adenosine structure suggest that bulky substitutions at position 6 might exert its inhibitory effects by sterically preventing full lid domain closure. Our preliminary in vivo pharmacokinetics and tolerability studies showed that our most potent derivative is very well tolerated and that MIC values are indeed achievable. However, future studies regarding SAR derivatization will be needed to optimize pharmacokinetic parameters. Overall, we showed that the unique “chimney-like” cavity formed by the unique oligomerization state of MtbAdoK could be utilized as a canvas for future medicinal chemistry efforts to further improve the in vitro and in vivo potency and selectivity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07333" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07333" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> General Synthetic Chemistry Methods</h3><div class="NLM_p">Solvents, reagents, and intermediates that are commercially available were used as received. Reagents and intermediates that were not commercially available were prepared in the manner as described below. Compounds <b>1</b>, <b>3</b>, and <b>4</b> were purchased at Sigma, with specific purities of >99.0, >98, and >99%, respectively. Compound <b>2</b> was obtained from ChemScene with specific purity of >95%, and compounds <b>5</b> and <b>6</b> are part of Merck’s nucleoside collection library. <sup>1</sup>H NMR spectra were measured on either a Varian VNMR System 400 or Bruker AVANCE 400 spectrometer at 400 MHz and are reported as ppm downfield from Me<sub>4</sub>Si with number of protons, multiplicities, and coupling constants in hertz indicated parenthetically. Where LC/MS data are presented, analyses were performed using an Agilent 6110A MSD or an Applied Shimadzu 2020MSD. The parent ion is given. Unless otherwise stated, all compounds possess a purity ≥ 95% as determined by LC/MS. Purification was conducted with reversed-phase HPLC (Waters Xbridge Prep OBD C18 (150 × 30 mm, 5 μm), YMC-Actus Pro C18 (150 × 30 mm, 5 μm), and Agela ASB (150 × 25 mm, 5 μm) column) or silica gel column chromatography (Liang Chen Gui Yuan 200–300 mesh) or PTLC (Liang Chen Gui Yuan, acryloid cement; 0.5 × 200 × 200 mm). For the specific chemical synthesis of reaction intermediates, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(7-(4-([1-Phenyl]-4-yl)piperazin-1-yl)-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-3-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (<b>7</b>)</h4><div class="NLM_p last">To a suspension of 1-phenylpiperazine (210 mg, 1.3 mmol) in anhydrous EtOH (6 mL) were added DIEA (0.3 mL, 1.7 mmol) and 6-chloro-purine riboside <b>A0</b> (287 mg, 1 mmol) at room temperature, and it was stirred at 70 °C for 17 h after cooling to room temperature. The resulting solid was filtered and washed with dichloromethane (DCM, 6 mL × 2) and dried under reduced pressure to afford (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(7-(4-([1-phenyl]-4-yl)piperazin-1-yl)-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-3-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (<b>7</b>), (288 mg, 70% yield) as a white solid; LC/MS purity of 97%; Scheme A0 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. MS (ESI) <i>m</i>/<i>z</i>: 412.9 [M + H]<sup>+</sup>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.43 (s, 1H), 8.27 (s, 1H), 7.21–7.26 (m, 2H), 6.99–7.01 (m, 2H), 6.81 (m, 1H), 5.93 (d, <i>J</i> = 6.0 Hz, 1H), 5.42 (d, <i>J</i> = 6.0 Hz, 1H), 5.27 (m, 1H), 5.14 (d, <i>J</i> = 4.8 Hz, 1H), 4.58 (m, 1H), 4.38 (br s, 4H), 4.16 (m, 1H), 3.97 (m, 1H), 3.65–3.70 (m, 1H), 3.55–3.60 (m, 1H), 3.20–3.35 (m, 4H).</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(7-(4-([1,1′-Biphenyl]-4-yl)piperazin-1-yl)-3<i>H</i>-imidazo[4,5-b]pyridin-3-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (<b>8</b>)</h4><div class="NLM_p last">To a solution of (2<i>R</i>,3<i>R</i>,4<i>R</i>,5<i>R</i>)-2-(7-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-3-yl)-5-(acetoxymethyl)tetrahydrofuran-3,4-diyl diacetate <b>B4</b> (130 mg, 0.082 mmol (38.8% purity)) in MeOH (3 mL) was added ammonium hydroxide (1.5 mL) dropwise at 15 °C, and it was stirred at 15 °C for 24 h. The mixture was concentrated and purified by reversed-phase HPLC [Waters Xbridge Prep OBD C18 (150 × 30 mm, 5 μm) column, elution of acetonitrile/water (0.05% ammonia hydroxide)] to afford (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(7-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-3-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol <b>8</b> (8.81 mg, 21.3% yield) as a white solid; LC/MS purity of 97%; Scheme B <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.40 (s, 1H), 7.99 (d, <i>J</i> = 5.6 Hz, 1H), 7.62–7.56 (m, 4H), 7.43–7.39 (m, 2H), 7.29–7.27 (m, 1H), 7.10 (d, <i>J</i> = 8.8 Hz, 2H), 6.66 (d, <i>J</i> = 6.0 Hz, 1H), 5.69–5.67 (m, 1H), 5.42 (d, <i>J</i> = 6.0 Hz, 1H), 5.17 (d, <i>J</i> = 4.4 Hz, 1H), 4.66–4.65 (m, 1H), 4.15 (s, 1H), 4.01 (s, 4H), 3.98 (d, <i>J</i> = 2.4 Hz, 1H), 3.66–3.65 (m, 1H), 3.56–3.55 (m, 1H), 3.37 (s, 4H). MS (ESI) <i>m</i>/<i>z</i>: 488.1.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-(6-(4-(4-(6-methoxypyridin-3-yl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>9</b>)</h4><div class="NLM_p last">The mixture of 9-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(((tert-butyldimethylsilyl)oxy)methyl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)-6-(4-(4-(6-methoxypyridin-3-yl)phenyl)piperazin-1-yl)-9<i>H</i>-purine <b>B3-1</b> (80 mg, 0.119 mmol) in THF (1.0 mL) and H<sub>2</sub>O (0.2 mL) was added TFA (0.5 mL, 6.49 mmol), and the mixture was stirred at 20 °C for 17 h, concentrated, and purified by reversed-phase HPLC [Waters Xbridge Prep OBD C18 (150 × 30 mm, 5 μm) column, elution of acetonitrile/water (0.05% ammonia hydroxide)] to give (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(hydroxymethyl)-5-(6-(4-(4-(6-methoxypyridin-3-yl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol <b>9</b> (26.4 mg, 42.7% yield) as a yellow solid; LC/MS purity of 99%; Scheme C <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.45–8.42 (m, 2H), 8.28 (s, 1H), 7.96–7.93 (m, 1H), 7.54 (d, <i>J</i> = 8.4 Hz, 2H), 7.09 (d, <i>J</i> = 8.4 Hz, 2H), 6.86 (d, <i>J</i> = 8.4 Hz, 1H), 5.93 (d, <i>J</i> = 5.6 Hz, 1H), 5.48 (d, <i>J</i> = 5.6 Hz, 1H), 5.33 (t, <i>J</i> = 6.0 Hz, 1H), 5.21 (d, <i>J</i> = 4.4 Hz, 1H), 4.59–4.58 (m, 1H), 4.40 (br s, 4H), 4.15 (d, <i>J</i> = 3.2 Hz, 1H), 3.96 (s, 1H), 3.87 (s, 3H), 3.69–3.66 (m, 1H), 3.57–3.56 (m, 1H), 3.35 (s, 4H). MS (ESI) <i>m</i>/<i>z</i>: 520.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-(6-(4-(4′-(2-hydroxypropan-2-yl)-[1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>10</b>)</h4><div class="NLM_p last">Yellow solid. Following the similar procedure shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Scheme C</a> with different boronic acids (RB(OH)<sub>2</sub>), compound <b>10</b> was synthesized; LC/MS purity of 99%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.46 (s, 1H), 8.29 (s, 1H), 7.55–7.48 (m, 6H), 7.09 (d, <i>J</i> = 8.8 Hz, 2H), 5.94 (d, <i>J</i> = 5.6 Hz, 1H), 5.48 (d, <i>J</i> = 5.6 Hz, 1H), 5.33 (t, <i>J</i> = 5.6 Hz, 1H), 5.21 (d, <i>J</i> = 5.2 Hz, 1H), 5.01 (s, 1H), 4.60–4.58 (m, 1H), 4.40 (br s, 4H), 4.16–4.15 (m, 1H), 3.97–3.96 (m, 1H), 3.68–3.67 (m, 1H), 3.58–3.56 (m, 1H), 3.31 (s, 4H), 1.44 (s, 6H). MS (ESI) <i>m</i>/<i>z</i>: 547.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i13" class="anchor-spacer"></div><h4 class="article-section__title" id="_i13"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-(6-(4-(phenylethynyl)phenyl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>11</b>)</h4><div class="NLM_p last">To a solution of 9-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)-6-(4-(phenylethynyl)phenyl)-9<i>H</i>-purine <b>C1-1</b> (103 mg, 0.18 mmol) in THF (1.5 mL) and H<sub>2</sub>O (0.2 mL) was added TFA (2.5 mL), and it was stirred at 20 °C for 17 h. The mixture was diluted with dimethylformamide (DMF, 2 mL), and it was concentrated under reduced pressure. The residue was purified by reversed-phase HPLC [TFA condition: Agela ASB (150 × 25 mm, 5 μm) column, elution of acetonitrile/water (0.1% TFA)] and then lyophilized to give (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(hydroxymethyl)-5-(6-(4-(phenylethynyl)phenyl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol <b>11</b> (17.48 mg, 23.3% yield) as a yellow solid; LC/MS purity of 99%; Scheme D <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 9.04 (s, 1H), 8.97 (s, 1H), 8.92 (d, <i>J</i> = 8.4 Hz, 2H), 7.80 (d, <i>J</i> = 8.4 Hz, 2H), 7.62–7.61 (m, 2H), 7.47–7.46 (m, 3H), 6.11 (d, <i>J</i> = 5.6 Hz, 1H), 4.67–4.65 (m, 1H), 4.23–4.21 (m, 1H), 4.01 (d, <i>J</i> = 3.6 Hz, 1H), 3.71–3.70 (m, 1H), 3.62–3.59 (m, 1H). MS (ESI) <i>m</i>/<i>z</i>: 429.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-(6-(4-(4′-(trifluoromethyl)-[1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>12</b>)</h4><div class="NLM_p last">Yellow solid. Following the similar procedure shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Scheme C</a> with different boronic acids (RB(OH)<sub>2</sub>), compound <b>12</b> was synthesized; LC/MS purity of 99%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.44 (s, 1H), 8.28 (s, 1H), 7.82 (d, <i>J</i> = 8.4 Hz, 2H), 7.72 (d, <i>J</i> = 8.4 Hz, 2H), 7.64 (d, <i>J</i> = 8.4 Hz, 2H), 7.11 (d, <i>J</i> = 8.4 Hz, 2H), 5.92 (d, <i>J</i> = 6.0 Hz, 1H), 4.58–4.55 (m, 1H), 4.38 (br s, 4H), 4.15 (t, <i>J</i> = 3.2 Hz, 1H), 3.95 (d, <i>J</i> = 3.2 Hz, 1H), 3.65–3.64 (m, 1H), 3.55–3.53 (m, 1H), 3.37 (s, 4H). MS (ESI) <i>m</i>/<i>z</i>: 557.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-(4-(4-(1-(Difluoromethyl)-1<i>H</i>-pyrazol-4-yl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (<b>13</b>)</h4><div class="NLM_p last">Yellow solid. Following the similar procedure shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Scheme C</a> with different boronic acids (RB(OH)<sub>2</sub>), compound <b>13</b> was synthesized; LC/MS purity of 97%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.58 (s, 1H), 8.46 (s, 1H), 8.29 (s, 1H), 8.19 (s, 1H), 7.80 (t, <i>J</i> = 59.2 Hz, 1H), 7.57 (d, <i>J</i> = 8.4 Hz, 2H), 7.05 (d, <i>J</i> = 8.8 Hz, 2H), 5.93 (d, <i>J</i> = 5.6 Hz, 1H), 4.59–4.57 (m, 1H), 4.39 (br s, 4H), 4.15 (t, <i>J</i> = 3.2 Hz, 1H), 3.96 (s, 1H), 3.67–3.66 (m, 1H), 3.57–3.54 (m, 1H), 3.32 (s, 4H). MS (ESI) <i>m</i>/<i>z</i>: 529.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-(6-(4-(3′-(morpholinomethyl)-[1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>14</b>)</h4><div class="NLM_p last">Yellow solid. Following the similar procedure shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Scheme C</a> with different boronic acids (RB(OH)<sub>2</sub>), compound <b>14</b> was synthesized; LC/MS purity of 95%. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.34 (d, <i>J</i> = 12.4 Hz, 2H), 7.77–7.75 (m, 2H), 7.64–7.61 (m, 2H), 7.54–7.52 (m, 1H), 7.43–7.42 (m, 1H), 7.16 (d, <i>J</i> = 8.8 Hz, 2H), 6.01 (d, <i>J</i> = 5.6 Hz, 1H), 4.67–4.65 (m, 1H), 4.54 (br s, 4H), 4.42 (s, 2H), 4.33–4.32 (m, 1H), 4.20–4.19 (m, 1H), 4.19–3.98 (m, 2H), 3.89–3.77 (m, 4H), 3.46–3.40 (m, 6H), 3.31–3.20 (2H). MS (ESI) <i>m</i>/<i>z</i>: 588.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> (1<i>R</i>,2<i>S</i>,3<i>R</i>,5<i>R</i>)-3-(6-(4-([1,1′-Biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)cyclopentane-1,2-diol (<b>15</b>)</h4><div class="NLM_p last">The mixture of 1-([1,1′-biphenyl]-4-yl)piperazine (56.3 mg, 0.236 mmol), (1<i>R</i>,2<i>S</i>,3<i>R</i>,5<i>R</i>)-3-(6-chloro-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)cyclopentane-1,2-diol <b>A3</b> (100 mg, 0.197 mmol), and DIEA (0.103 mL, 0.590 mmol) in EtOH (5 mL) was stirred at 70 °C for 17 h. The mixture was cooled and concentrated, purified by reversed-phase HPLC [Waters Xbridge Prep OBD C18 (150 × 30 mm, 5 μm) column, elution of acetonitrile/water (0.05% ammonia hydroxide)] to give (1<i>R</i>,2<i>S</i>,3<i>R</i>,5<i>R</i>)-3-(6-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)cyclopentane-1,2-diol <b>15</b> (12.26 mg, 12.77% yield) as a white solid; LC/MS purity of 99%; Scheme F <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.29 (s, 1H), 8.27 (s, 1H), 7.62–7.55 (m, 4H), 7.42–7.39 (m, 2H), 7.29–7.27 (m, 1H), 7.09 (d, <i>J</i> = 8.8 Hz, 2H), 4.93 (d, <i>J</i> = 6.8 Hz, 1H), 4.75–4.66 (m, 3H), 4.40–4.15 (m, 5H), 3.84–3.83 (m, 1H), 3.51–3.46 (m, 2H), 3.35 (br s, 4H), 2.25–2.22 (m, 1H), 2.06–2.01 (m, 1H), 1.75–1.70 (m, 1H). MS (ESI) <i>m</i>/<i>z</i>: 487.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(Hydroxymethyl)-5-(6-(4-(4-(6-(trifluoromethyl)pyridin-3-yl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>16</b>)</h4><div class="NLM_p last">Yellow solid. Following the similar procedure shown in <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Scheme C</a> with different boronic acids (RB(OH)<sub>2</sub>), compound <b>16</b> was synthesized; LC/MS purity of 97%. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.94 (s, 1H), 8.38 (d, <i>J</i> = 16.4 Hz, 2H), 8.23 (d, <i>J</i> = 7.6 Hz, 1H), 7.83 (d, <i>J</i> = 8.0 Hz, 1H), 7.69 (d, <i>J</i> = 8.8 Hz, 2H), 7.20 (d, <i>J</i> = 8.8 Hz, 2H), 6.01 (d, <i>J</i> = 5.6 Hz, 1H), 4.66–4.65 (m, 1H), 4.56 (br s, 4H), 4.33–4.32 (m, 1H), 4.20–4.19 (m, 1H), 3.90–3.89 (m, 1H), 3.82–3.81 (m, 1H), 3.53–3.50 (m, 5H). MS (ESI) <i>m</i>/<i>z</i>: 558.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-([1,1′-Biphenyl]-4-ylethynyl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (<b>17</b>)</h4><div class="NLM_p last">Following similar procedure for <b>11</b> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Scheme D</a>, compound <b>17</b> was synthesized. The reaction of 6-([1,1′-biphenyl]-4-ylethynyl)-9-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(((<i>tert</i>-butyldimethylsilyl)oxy)methyl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)-9<i>H</i>-purine <b>C1-2</b> (36 mg, 0.062 mmol) gave (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-([1,1′-biphenyl]-4-ylethynyl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl) tetrahydrofuran-3,4-diol <b>17</b> (9.86 mg, 37.2% yield) as a yellow solid; LC/MS purity of 99%. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.96 (d, <i>J</i> = 8.4 Hz, 2H), 7.86–7.75 (m, 6H), 7.53–7.50 (m, 2H), 7.42–7.41 (m, 1H), 6.07 (d, <i>J</i> = 5.2 Hz, 1H), 5.61 (d, <i>J</i> = 4.8 Hz, 1H), 5.30 (s, 1H), 5.13 (s, 1H), 4.63 (s, 1H), 4.21 (s, 1H), 4.00 (d, <i>J</i> = 3.6 Hz, 1H), 3.73–3.70 (m, 1H), 3.61–3.58 (m, 1H). MS (ESI) <i>m</i>/<i>z</i>: 429.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-(4-([1,1′-Biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (<b>18</b>)</h4><div class="NLM_p last">To a solution of ((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-(6-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methanol <b>B1</b> (3.1 g, 5.50 mmol) in THF (8 mL) and water (1.5 mL) was added TFA (30 mL) dropwise, and it was stirred at 15 °C for 2 h. After NH<sub>4</sub>OH was added to adjust pH to 7–8, water (70 mL) was added and the mixture was filtered. The resulting solid was dried under reduced pressure, and the residue was purified by silica gel column chromatography (DCM: MeOH = 100:1 to 30:1) to give (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-(hydroxymethyl)tetrahydrofuran-3,4-diol (1.327 g, 48.2% yield) as a white solid; LC/MS purity of 98%; Scheme A <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.46 (s, 1H), 8.29 (s, 1H), 7.62–7.55 (m, 4H), 7.43–7.39 (m, 2H), 7.29–7.27 (m, 1H), 7.09 (d, <i>J</i> = 8.4 Hz, 2H), 5.94 (d, <i>J</i> = 6.0 Hz, 1H), 5.48 (d, <i>J</i> = 6.0 Hz, 1H), 5.35–5.32 (m, 1H), 5.20 (d, <i>J</i> = 4.8 Hz, 1H), 4.59 (d, <i>J</i> = 5.2 Hz, 1H), 4.40 (br s, 4H), 4.16 (d, <i>J</i> = 3.6 Hz, 1H), 3.97 (d, <i>J</i> = 2.8 Hz, 1H), 3.70–3.67 (m, 1H), 3.58–3.56 (m, 1H), 3.35 (s, 4H). MS (ESI) <i>m</i>/<i>z</i>: 489.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>N</i>-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-Dihydroxy-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-2-yl)methyl)-4-methylbenzenesulfonamide (<b>19</b>)</h4><div class="NLM_p last">To a solution of <i>N</i>-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-2,2-dimethyl-6-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-4-methylbenzenesulfonamide <b>D3</b> (72 mg, 0.090 mmol) in THF (1 mL) and H<sub>2</sub>O (0.1 mL) was added TFA (1 mL, 12.98 mmol). The mixture was stirred at 15 °C for 17 h and then DMF was added (2 mL), and the mixture was purified by reversed-phase HPLC [TFA condition: Agela ASB (150 × 25 mm, 5 μm) column, elution of acetonitrile/water (0.1% TFA)] and then lyophilized to give <i>N</i>-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-dihydroxy-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-2-yl)methyl)-4-methylbenzenesulfonamide (31.74 mg, 38.5% yield) as a white solid; LC/MS purity of 99%; Scheme S1 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.38 (s, 1H), 8.27–8.21 (m, 2H), 7.65 (d, <i>J</i> = 8.8 Hz, 2H), 7.52 (d, <i>J</i> = 8.4 Hz, 2H), 7.35 (d, <i>J</i> = 8.4 Hz, 2H), 7.11 (d, <i>J</i> = 8.4 Hz, 2H), 5.85 (d, <i>J</i> = 5.6 Hz, 1H), 4.76 (t, <i>J</i> = 5.6 Hz, 2H), 4.36 (br s, 4H), 4.07–3.96 (m, 2H), 3.47 (br s, 4H), 3.07–2.97 (m, 1H), 2.34 (s, 3H). MS (ESI) <i>m</i>/<i>z</i>: 634.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(((4-Nitrophenyl)amino)methyl)-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>20</b>)</h4><div class="NLM_p last">The solution of <i>N</i>-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-2,2-dimethyl-6-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-4-nitroaniline <b>D4</b> (50 mg, 0.078 mmol) and TFA (0.5 mL, 6.49 mmol) in THF (1.0 mL) and H<sub>2</sub>O (0.2 mL) was stirred at 20 °C for 17 h, and then, it was concentrated and purified by reversed-phase HPLC [TFA condition: Agela ASB (150 × 25 mm, 5 μm) column, elution of acetonitrile/water (0.1% TFA)] and then lyophilized to give (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(((4-nitrophenyl)amino)methyl)-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol <b>20</b> (24.24 mg, 51.5% yield) as a yellow solid; LC/MS purity of 99%; Scheme S2 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.29 (s, 1H), 8.10 (s, 1H), 7.83 (d, <i>J</i> = 9.2 Hz, 2H), 7.50 (d, <i>J</i> = 8.8 Hz, 2H), 7.09 (d, <i>J</i> = 8.8 Hz, 2H), 6.57 (d, <i>J</i> = 9.2 Hz, 2H), 5.96 (d, <i>J</i> = 4.0 Hz, 1H), 4.80–4.74 (m, 1H), 4.51–4.32 (m, 5H), 4.28–4.21 (m, 1H), 3.71–3.62 (m, 1H), 3.56–3.40 (m, 5H). MS (ESI) <i>m</i>/<i>z</i>: 601.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i>-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-Dihydroxy-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-2-yl)methyl)benzamide (<b>21</b>)</h4><div class="NLM_p last">To a solution of <i>N</i>-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-2,2-dimethyl-6-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)benzamide <b>D2</b> (68 mg, 0.090 mmol) in THF (1.0 mL) and H<sub>2</sub>O (0.1 mL) was added TFA (1.0 mL, 12.98 mmol). The mixture was stirred at 15 °C for 17 h, and then, it was concentrated under reduced pressure, purified by reversed-phase HPLC (TFA condition: Agela ASB [150 × 25 mm, 5 μm) column, elution of acetonitrile/water (0.1% TFA)] and then lyophilized to give <i>N</i>-(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-dihydroxy-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-2-yl)methyl)benzamide <b>21</b> (25.7 mg, 46.4% yield) as a white solid; LC/MS purity of 95%; Scheme S3 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.68 (s, 1H), 8.47 (s, 1H), 8.22 (s, 1H), 7.84 (d, <i>J</i> = 7.6 Hz, 2H), 7.55–7.45 (m, 5H), 7.13 (d, <i>J</i> = 8.4 Hz, 2H), 5.91 (d, <i>J</i> = 5.6 Hz, 1H), 4.76 (t, <i>J</i> = 5.6 Hz, 2H), 4.38 (br s, 4H), 4.19–4.10 (m, 2H), 3.65–3.45 (m, 5H). MS (ESI) <i>m</i>/<i>z</i>: 584.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-(4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-Dihydroxy-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-2-yl)methyl)amino)phenyl)methanesulfonamide (<b>22</b>)</h4><div class="NLM_p last">The mixture of <i>N</i>-(4-((((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-2,2-dimethyl-6-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)amino)phenyl)methanesulfonamide <b>D7</b> (50 mg, 0.036 mmol) and TFA (0.5 mL, 6.49 mmol) in THF (1.0 mL) and H<sub>2</sub>O (0.2 mL) was stirred at 20 °C for 17 h. The mixture was concentrated and purified by reversed-phase HPLC [TFA condition: Agela ASB (150 × 25 mm, 5 μm) column, elution of acetonitrile/water (0.1% TFA)] and then lyophilized to give <i>N</i>-(4-((((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-3,4-dihydroxy-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-2-yl)methyl)amino)phenyl)methanesulfonamide <b>21</b> (15.03 mg, 61.2% yield) as a yellow solid; LC/MS purity of 96%; Scheme S4 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.30 (d, <i>J</i> = 9.2 Hz, 2H), 7.52 (d, <i>J</i> = 7.2 Hz, 2H), 7.28–7.08 (m, 6H), 6.08 (br s, 1H), 4.58–4.26 (m, 6H), 3.89 (br s, 1H), 3.76–3.49 (m, 5H), 2.97 (s, 3H). MS (ESI) <i>m</i>/<i>z</i>: 649.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-(4-([1,1′-Biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-((benzyloxy)methyl)tetrahydrofuran-3,4-diol (<b>23</b>)</h4><div class="NLM_p last">The solution of 6-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-9-((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-6-((benzyloxy)methyl)-2,2-dimethyltetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)-9<i>H</i>-purine <b>B5</b> (700 mg, 1.131 mmol) and con. HCl (1 mL, 4.00 mmol) in MeOH (5 mL) was stirred at 25 °C for 1 h. Then, the mixture was neutralized with ammonium hydroxide (5 mL), concentrated, purified by reversed-phase HPLC [neutral condition: Waters Xtimate C18 (150 × 25 mm, 5 μm) column, elution of acetonitrile/water (10 mM NH<sub>4</sub>HCO<sub>3</sub>)] to give(2<i>R</i>,3<i>R</i>,4<i>S</i>,5<i>R</i>)-2-(6-(4-([1,1′-biphenyl]-4-yl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)-5-((benzyloxy)methyl)tetrahydrofuran-3,4-diol <b>23</b> (600 mg, 88% yield) as a white solid; LC/MS purity of 96%; Scheme S5 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>, 400 MHz): δ 8.37 (s, 1H), 8.29 (s, 1H), 7.41–7.31 (m, 8H), 7.09 (d, <i>J</i> = 8.4 Hz, 2H), 5.97 (d, <i>J</i> = 6.0 Hz, 1H), 5.55 (d, <i>J</i> = 5.6 Hz, 1H), 5.31 (d, <i>J</i> = 5.2 Hz, 1H), 4.58–4.54 (m, 3H), 4.40 (br s, 4H), 4.22–4.07 (m, 2H), 3.74–3.67 (m, 1H), 3.31 (s, 4H). MS (ESI) <i>m</i>/<i>z</i>: 579.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(((4-Fluorophenyl)amino)methyl)-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol (<b>24</b>)</h4><div class="NLM_p last">The mixture of <i>N</i>-(((3a<i>R</i>,4<i>R</i>,6<i>R</i>,6a<i>R</i>)-2,2-dimethyl-6-(6-(4-(4- (trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuro[3,4-<i>d</i>][1,3]dioxol-4-yl)methyl)-4-fluoroaniline <b>D9</b> (200 mg, 0.228 mmol) and con. HCl (0.5 mL, 2.0 mmol) in MeOH (5 mL) was stirred at 25 °C for 1 h. Then, the mixture was neutralized with ammonium hydroxide (5 mL), concentrated, purified by reversed-phase HPLC (basic condition: Waters Xbridge Prep OBD C18 (150 × 30 mm, 5 μm) column, elution of acetonitrile/water (0.05% ammonia hydroxide)) to give (2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-2-(((4-fluorophenyl)amino)methyl)-5-(6-(4-(4-(trifluoromethyl)phenyl)piperazin-1-yl)-9<i>H</i>-purin-9-yl)tetrahydrofuran-3,4-diol <b>23</b> (43.09 mg, 32.6% yield) as a white solid; LC/MS purity of 99%; Scheme S6 <a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">Supporting Information</a>. <sup>1</sup>H NMR (CD<sub>3</sub>OD, 400 MHz): δ 8.31 (s, 1H), 8.14 (s, 1H), 7.50 (d, <i>J</i> = 8.8 Hz, 2H), 7.10 (d, <i>J</i> = 8.8 Hz, 2H), 6.89–6.79 (m, 2H), 6.73–6.62 (m, 2H), 5.97 (d, <i>J</i> = 5.2 Hz, 1H), 4.81 (t, <i>J</i> = 5.6 Hz, 1H), 4.49–4.26 (m, 6H), 3.47–3.41 (m, 6H). MS (ESI) <i>m</i>/<i>z</i>: 574.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Mtb Resistant Mutant Isolation</h3><div class="NLM_p last">Isolation of Mtb H37Rv-resistant mutants to iodotubercidin on a solid medium was performed as described by Ioerger et al.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> Three mutants were subjected to whole genome sequencing. Single nucleotide polymorphisms were identified in Mtb <i>adoK</i> and confirmed by sequencing from genomic DNA.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Resistant mutants were confirmed on the solid medium by confirming the growth on plates containing 5× MIC and in liquid medium using the Microplate Alamar Blue Assay (MABA).<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cloning, Expression, and Purification of Recombinant MtbAdoK and hAdoK</h3><div class="NLM_p">The WT MtbAdoK gene (Rv2202c) was amplified by the polymerase chain reaction from total genomic DNA of <i>M. tuberculosis</i> H37Rv. The following oligonucleotides were used in the polymerase chain reaction 5′-GGAATTCCATATGGTGACGATCGCGGTAACC-3′ and 5′-CTTAAGCTTCTAGGCCAGCAC-3′, respectively. The amplified DNA fragment was digested with NdeI and HindIII restriction enzymes (New England Biolabs) and subcloned into the corresponding restriction sites of the pET28b vector containing an <i>N</i>-terminal Tobacco Etch Virus cleavable His-tag.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> hAdoK gene was amplified from clone HsCD00042641 (DNASU plasmid repository) using the following synthetic oligonucleotides in the polymerase chain reaction: 5′-GGAATTCCATATGATGACGTCAGTCAGAGAAAATATTC-3′ and 5′-CCCAAGCTTCTAGTGGAAGTCTGGC-3′. The amplified fragment was subsequently cloned into the same pET28b using the same procedures described above. In all cases, gene fidelity was confirmed by DNA sequencing, and sequenced plasmids were used to transform <i>Escherichia coli</i> BL21 (DE3) cells for protein expression. For protein expression, cell cultures were grown in an LB medium at 37.0 °C. Cells were induced with 0.5 mM isopropyl β-<span class="smallcaps smallerCapital">d</span>-1-thiogalactopyranoside (IPTG) when the cell density reached <i>A</i><sub>600</sub> ≈ 0.6–1.0. Cell cultures were incubated for 18 h at 18.0 °C before harvesting.</div><div class="NLM_p last">Harvested cells were lysed using a French press, and the lysate was centrifuged at 17 000 rpm for 1 h. Recombinant MtbAdoK and hAdoK were purified by using a HisTrap HP Nickel column (GE Healthcare). Purification buffers A and B contained, 50.0 mM HEPES, pH 7.5, 500.0 mM NaCl, 500.0 mM imidazole (buffer B only), and 5.0% glycerol. For crystallization studies, MtbAdoK was dialyzed in 20.0 mM HEPES, pH 7.5, 50.0 mM NaCl, 2.0 mM DTT, and 5.0% glycerol. For enzymatic assays, the proteins were dialyzed in 50.0 mM HEPES, 50.0 mM NaCl, 100.0 mM KCL, 4.0 mM DTT, and 20.0% glycerol. Finally, the proteins were aliquoted and stored in −80.0 °C for subsequent crystallization and enzymatic assays.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Enzymatic Assay, IC<sub>50</sub> Determination, Steady-State Kinetics, and <i>K</i><sub>i</sub> Determination</h3><div class="NLM_p">Compounds were tested against MtbAdoK using the pyruvate kinase–lactate dehydrogenase coupled assay system in a Cary100 UV–Vis spectrophotometer by monitoring the decrease in absorbance of NADH at 340 nm.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The reaction was started by the addition of 60 nM of enzyme into a final volume of 200 μL master mix containing 50.0 mM HEPES pH 7.5, 50.0 mM KCl, 6.0 mM MgCl<sub>2</sub> (4.0 mM MgCl<sub>2</sub> for hAdoK), 3.0 mM ATP (2.0 mM ATP for hAdoK), 200.0 μM NADH, 1.0 mM phosphoenolpyruvate, 1.0 mM DTT, 12.0 U/mL pyruvate kinase, 12.0 U/mL lactate dehydrogenase, and 15.0 μM adenosine. IC<sub>50</sub> values for each compound were determined by varying the concentration of the inhibitor at fixed concentrations of the enzyme and by fitting the dose–response data into the four-parameter logistic curve (<a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>) model of GraphPad prism 7.02, as follows<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_m001.gif" alt="" id="_i30" /></img><span class="labelSpan">(1)</span></span>where <i>I</i> is the logarithm of inhibitor concentration, <i>H</i> is the Hill slope and <i>Y</i>, <i>Y</i><sub>max</sub>, and <i>Y</i><sub>min</sub> are the specific activity, maximum specific activity, and minimum specific activity, respectively. Kinetic assays for MtbAdoK were performed essentially as described above with the following exceptions: the master mix contained 60.0 nM MtbAdoK, and the reaction was started by the addition of varying concentrations of adenosine in the presence of constant concentrations of compound (0.0, 20.0, and 40.0 nM). Kinetic data were obtained by fitting the initial velocity data into GraphPad Prism 7.02 nonlinear regression function of Michaelis–Menten model (<a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>), as follows<span class="NLM_disp-formula" id="eq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_m002.gif" alt="" id="_i31" /></img><span class="labelSpan">(2)</span></span>where <i>V</i><sub>o</sub> is the initial velocity, <i>Y</i><sub>max</sub> is the maximum specific activity, <i>S</i> is the substrate concentration, and <i>K</i><sub>m</sub> is the Michaelis–Menten constant.</div><div class="NLM_p">Specific activity values for kinetics and IC<sub>50</sub> measurements were determined using (<a class="ref internalNav" href="#eq3" aria-label="eq 3">eq 3</a>)<span class="NLM_disp-formula" id="eq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_m003.gif" alt="" id="_i32" /></img><span class="labelSpan">(3)</span></span>where <i>a</i> is the change in absorbance over time, ε<sub>NADH</sub> is the millimolar extinction coefficient of NADH, <i>b</i> is the pathlength, <i>d</i> is the dilution factor of the enzyme in the assay, and <i>c</i> is the concentration of enzyme stock used for the assay.</div><div class="NLM_p">Finally, the inhibition constant (<i>K</i><sub>i</sub>) was determined by using the Cheng–Prusoff relationship for competitive inhibition (<a class="ref internalNav" href="#eq4" aria-label="eq 4">eq 4</a>).<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a><span class="NLM_disp-formula" id="eq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_m004.gif" alt="" id="_i33" /></img><span class="labelSpan">(4)</span></span></div><div class="NLM_p last">In all cases, the compounds were serially diluted in 100.0% DMSO and added to the respective enzymatic reaction to a final concentration of 2.5% DMSO.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Crystallization, Data Collection, and Crystal Structure Determination</h3><div class="NLM_p last">Purified MtbAdoK was concentrated to 18 mg/mL before crystallization trials. All cocrystals were obtained by the vapor diffusion method. MtbAdoK-<b>2</b>, <b>3</b>, <b>4</b>, and <b>7</b> cocrystals were achieved by mixing 2.0 μL of protein solution, preincubated with 4.0 mM of the compound for 1.0 h at 25.0 °C, with 1.0 μL of 100.0 mM HEPES pH 7.5, 2.0 M ammonium sulfate and 2.0% PEG 400. MtbAdoK-<b>5</b> cocrystals were obtained by mixing 2.0 μL of protein solution, preincubated with 4.0 mM of <b>5</b> for 1.0 h at 25.0 °C, with 1.0 μL of 100.0 mM HEPES pH 7.5 and 1.2 M sodium citrate tribasic dihydrate. MtbAdoK-<b>6</b> cocrystals were obtained by mixing 2.0 μL of protein solution, preincubated with 4.0 mM of the compound for 1.0 h at 25.0 °C, with 1.0 μL of 5.0 M sodium formate. Finally, MtbAdoK-<b>17</b> cocrystals were obtained by mixing 2.0 μL of protein solution, preincubated with 2.0 mM <b>17</b> for 1 h at 25 °C, with 1.0 μL of 100 mM Bis–Tris pH 6.5, 2.0 M ammonium sulfate and 2.0% PEG 400. In all cases, crystals were grown at 20.0 °C. Prior to data collection, crystals were cryoprotected with Paratone (Hampton research) and flash frozen in liquid nitrogen. X-ray diffraction data were collected at Argonne’s National Lab Advanced Photon Source beamlines 19ID and 23ID. Diffraction data were indexed, scaled, and integrated using HKL2000.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Initial phases were obtained by MR in MOLREP using the high-resolution structure of the MtbAdoK-<b>1</b> complex with PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PKM">2PKM</a>.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Refinement was performed in PHENIX followed by iterative runs of inspection and manual modification using coot.<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> Ligand model and restraint files were created in ELBOW from the PHENIX suite and fitted into the electron density using COOT. Images and figures were rendered using Molsoft ICM, Chimera and PyMOL.<a onclick="showRef(event, 'ref47 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49">(47−49)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Antimycobacterial Assay</h3><div class="NLM_p last">Antitubercular testing and EC<sub>50</sub> determination was performed using the MABA assay in a 96-well format as previously described.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> Starter culture of Mtb mc<sup>2</sup> 7000 was grown in 7H9 media supplemented with OADC (Middlebrook), 0.5% dextrose, 0.085% NaCl, 0.05% Tyloxapol (Sigma), 0.25 μg/mL malachite green (Sigma), and 25 μg/mL pantothenate. Once cells reached an optical density of OD<sub>600</sub> ≈ 1.5, cells were diluted to an OD<sub>600</sub> of 0.01 in the same media composition without OADC. Compounds were serially diluted in 100.0% DMSO and added to the cells to a final concentration of DMSO of 2.5%. Plates were incubated for 6 days before staining with resazurin (Sigma). After staining, plates were incubated for 2 additional days for developing. Finally, developed plates were read using a POLARstar Omega spectrophotometer (BMG Labtech) and by monitoring the fluorescence emission of resazurin (excitation = 570.0 nm, emission = 585.0 nm). In all cases, rifampicin was used as a negative control. The anti-Tb activity was determined as a percent control, and EC<sub>50</sub> values for each compound were determined by utilizing <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Human Dermal Fibroblast Cytotoxicity Assay</h3><div class="NLM_p last">HDFs were purchased from ATCC (Manassas, VA). HDF cells were cultured in DMEM (Lonza) media supplemented with 10.0% fetal bovine serum (Lonza) and penicillin/streptomycin (Lonza). For the cytotoxicity assay, compound stocks were serially diluted in phosphate-buffered saline plus 10.0% DMSO. On the day of assay, HDF cells were trypsinized, counted, and resuspended at a concentration of 64 000 cells/mL in media. Cells were plated, overlaid with the compound serial dilutions, and incubated at 37.0 °C. After 48.0 h, resazurin dye was added, and the assay plates cultured for another 24.0 h. On the next day, the absorbance of the resazurin was measured on a microplate reader to assess cell death. Cytotoxicity was determined as a percent control, and CC<sub>50</sub> values for each compound were determined by utilizing <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Pharmacokinetic and Tolerability Studies</h3><div class="NLM_p">***All animal experiments described in this manuscript followed protocols approved by the Texas A&M University Institutional Animal Care and Use Committee under Animal Use Protocol “Preliminary in vivo PK/Tox on newly developed drug compounds” #2017-0368.***</div><div class="NLM_p">Eight-week old female ND4 Swiss Webster mice (∼28 g each) (Envigo, Indianapolis, IN) were used for PK and combination tolerability-snapshot PK studies. Compound <b>18</b> was formulated in 5% DMSO, 95% canola oil and administered as a single 100 μL oral gavage at 10 mg·kg<sup>–1</sup> for PK and as a 200 μL oral gavage at 50, 100, or 200 mg·kg<sup>–1</sup> QD × 3 days for tolerability-snapshot PK. Mice were anesthetized using isoflurane and 50 μL collected at 0, 0.25, 0.5, 1, 2, and 4 h survival bleed times for traditional PK or 0, 1, 4, and 24 h for tolerability-snapshot PK studies after the first administered dose. Around 200 μL was collected at 6, 8, and 24 h terminal bleed times for the traditional PK. Survival bleeds were drawn from the facial vein and terminal bleeds via cardiac puncture following CO<sub>2</sub> asphyxiation.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Three mice were used per time point.</div><div class="NLM_p last">Blood samples were centrifuged (3000<i>g</i>, 15 min) for plasma separation. Aliquots of plasma (10 μL) were extracted with two 500 μL equivalents of methanol containing 0.1% formic acid to ensure maximum recovery of compound <b>18</b> from plasma. The supernatant was collected from precipitated protein pellet following centrifugation (3000<i>g</i>, 5 min) and evaporated to dryness using a temperature-controlled Eppendorf Vacufuge at 30 °C for 4 h. The dry samples were reconstituted with 100 μL of methanol (1:10 original analyte dilution factor) spiked with 250 ng·mL<sup>–1</sup> verapamil (Sigma-Aldrich) as an internal standard and subjected to LC/MS analysis on a Bruker Daltronics micrOTOF-Q II mass spectrometer coupled with an Agilent 1200 Infinity series HPLC with temperature controlled autosampler (24 °C) and photodiode array detector. Standard solutions of compound <b>18</b> ranging from 1000 to 7.8125 ng·mL<sup>–1</sup> spiked into mouse control plasma was also analyzed to generate a calibration curve corrected for matrix effects. A linear regression generated from calibration samples was used to quantify samples.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The LOD of the LC/MS method was approximately 2 ng·mL<sup>–1</sup>. A 4.6 × 100 mm Kinetex 2.6 μm EVO C18 100 Å column at a flow rate of 0.5 mL·min<sup>–1</sup> was used in the analysis. The mobile phase consisted of water containing 0.1% formic acid as solvent A and acetonitrile with 0.1% formic acid as solvent B. The gradient conditions were maintained as follows: 90% A, 10% B to 100% B in 8 min; 100% held for 4 min as a washing step; 100% B back to 90% A in 2 min; 90% A, 10% B held for 3 min to reset and equilibrate the column. Each analytical run was automatically calibrated using a secondary line injection of 10 μL of sodium acetate external standard at 13 min. The injection volume of the analyte was 10 μL, and MS was operated in positive mode with electrospray ionization at source. Mass spectra were monitored in a range of 50–1000<i>m</i>/<i>z</i>. Data were processed using DataAnalysis 4.1 and QuantAnalysis (Bruker Daltronics).</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i38"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00020" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63226" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63226" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00020" class="ext-link">10.1021/acs.jmedchem.9b00020</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Schemes and reagents for chemical synthesis of intermediates and supporting figures and tables (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf">jm9b00020_si_001.pdf (1.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_002.csv">jm9b00020_si_002.csv (2.39 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Authors will release the atomic coordinates and experimental data upon article publication: MtbAdoK-<b>2</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C67">6C67</a>, MtbAdoK-<b>3</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9N">6C9N</a>, MtbAdoK-<b>4</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9P">6C9P</a>, MtbAdoK-<b>5</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9Q">6C9Q</a>, MtbAdoK-<b>6</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9R">6C9R</a>, MtbAdoK-<b>7</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9V">6C9V</a>, MtbAdoK-<b>17</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9S">6C9S</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00020" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David B. Olsen</span> - <span class="hlFld-Affiliation affiliation">Merck
Sharp Dohme Corporation, West Point Pennsylvania 19486, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7c181d0a15182313100f19123c11190e1f17521f1311"><span class="__cf_email__" data-cfemail="dcb8bdaab5b883b3b0afb9b29cb1b9aebfb7f2bfb3b1">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">James C. Sacchettini</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Texas A&M
University, College
Station, Texas 77843, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5767-2367" title="Orcid link">http://orcid.org/0000-0001-5767-2367</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3a495b5959525f4e4e7a4e5b574f145f5e4f"><span class="__cf_email__" data-cfemail="02716361616a6776764276636f772c676677">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberto
A. Crespo</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Texas A&M
University, College
Station, Texas 77843, United States</span>; 
    <span>Present Address:
                        GSK, Swedeland Road, King of Prussia, PA</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qun Dang</span> - <span class="hlFld-Affiliation affiliation">Merck
Sharp Dohme Corporation, West Point Pennsylvania 19486, United States</span>; 
    <span>Present Address:
                        Lilly China R&D Center, Eli Lilly and Company, Shanghai China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nian E. Zhou</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Texas A&M
University, College
Station, Texas 77843, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liam M. Guthrie</span> - <span class="hlFld-Affiliation affiliation">College
of Medicine, Texas A&M University Health
Science Center, Bryan, Texas 77807, United
States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas C. Snavely</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Texas A&M
University, College
Station, Texas 77843, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wen Dong</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Texas A&M
University, College
Station, Texas 77843, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kimberly A. Loesch</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry and Biophysics, Texas A&M
University, College
Station, Texas 77843, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Takao Suzuki</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong Road, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lanying You</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong Road, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Wang</span> - <span class="hlFld-Affiliation affiliation">WuXi
AppTec, 288 Fute Zhong Road, Shanghai 200131, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Theresa O’Malley</span> - <span class="hlFld-Affiliation affiliation">TB
Discovery Research, Infectious Disease Research
Institute, 1616 Eastlake Avenue E, Seattle, Washington 98102, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tanya Parish</span> - <span class="hlFld-Affiliation affiliation">TB
Discovery Research, Infectious Disease Research
Institute, 1616 Eastlake Avenue E, Seattle, Washington 98102, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7507-0423" title="Orcid link">http://orcid.org/0000-0001-7507-0423</a></span></div></li></ul></li><li></li><li><span class="author-information-subsection-header">Funding</span><p>This work was supported by grants from the Bill and Melinda Gates Foundation (OPP1024055), Welch Foundation (A-0015), NIAID-NIH (TB Structural Genomics grant P01A1095208) to J.C.S. and the Bill and Melinda Gates Foundation (OPP1024038) to T.P.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81652" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81652" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors would like to acknowledge the Argonne National Lab staff at beamlines 19-ID and 23-ID, the TB Drug Accelerator Program (TBDA), the Bill and Melinda Gates Foundation and the Welch Foundation for their financial and/or intellectual support.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Mtb</td><td class="NLM_def"><p class="first last"><i>Mycobacterium tuberculosis</i></p></td></tr><tr><td class="NLM_term">MtbAdoK</td><td class="NLM_def"><p class="first last"><i>Mycobacterium tuberculosis</i> adenosine kinase</p></td></tr><tr><td class="NLM_term">hAdoK</td><td class="NLM_def"><p class="first last">human adenosine kinase</p></td></tr><tr><td class="NLM_term">vdW</td><td class="NLM_def"><p class="first last">van der Waals</p></td></tr><tr><td class="NLM_term">HDF</td><td class="NLM_def"><p class="first last">human dermal fibroblasts</p></td></tr><tr><td class="NLM_term">MR</td><td class="NLM_def"><p class="first last">molecular replacement</p></td></tr><tr><td class="NLM_term">TBSCl</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyldimethylsilyl chloride</p></td></tr><tr><td class="NLM_term">EtOH</td><td class="NLM_def"><p class="first last">ethanol</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">ArB(OH)<sub>2</sub></td><td class="NLM_def"><p class="first last">aryl boronic acids</p></td></tr><tr><td class="NLM_term">XPhos Pd G2</td><td class="NLM_def"><p class="first last">chloro(2-dicyclohexylphosphino-2′,4′,6′-triisopropyl-1,1′-biphenyl)[2-(2′-amino-1,1′-biphenyl)]palladium(II)</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">RuPhos</td><td class="NLM_def"><p class="first last">2-dicyclohexylphosphino-2′,6′-diisopropoxybiphenyl</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the curve</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term"><i>C</i><sub>max</sub></td><td class="NLM_def"><p class="first last">maximum concentration</p></td></tr><tr><td class="NLM_term"><i>t</i><sub>max</sub></td><td class="NLM_def"><p class="first last">time at maximum concentration.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36136" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36136" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">WHO</span>.  <i>Global Tuberculosis Report</i>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=WHO&title=Global+Tuberculosis+Report"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DGlobal%2520Tuberculosis%2520Report%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ducati, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, D. S.</span></span> <span> </span><span class="NLM_article-title">Purine Salvage Pathway in <i>Mycobacterium tuberculosis</i></span>. <i>Curr. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1275</span>, <span class="refDoi"> DOI: 10.2174/092986711795029627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.2174%2F092986711795029627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=21366536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOku78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1258-1275&author=R.+G.+Ducatiauthor=A.+Bredaauthor=L.+A.+Bassoauthor=D.+S.+Santos&title=Purine+Salvage+Pathway+in+Mycobacterium+tuberculosis&doi=10.2174%2F092986711795029627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Purine salvage pathway in Mycobacterium tuberculosis</span></div><div class="casAuthors">Ducati, R. G.; Breda, A.; Basso, L. A.; Santos, D. S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1258-1275</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Millions of deaths worldwide are caused by the etiol. agent of tuberculosis, Mycobacterium tuberculosis.  The increasing prevalence of this disease, the emergence of drug-resistant strains, and the devastating effect of human immunodeficiency virus coinfection have led to an urgent need for the development of new and more efficient antimycobacterial drugs.  The modern approach to the development of new chem. compds. against complex diseases, esp. the neglected endemic ones, such as tuberculosis, is based on the use of defined mol. targets.  Among the advantages, this approach allows (i) the search and identification of lead compds. with defined mol. mechanisms against a specific target (e.g. enzymes from defined pathways), (ii) the anal. of a great no. of compds. with a favorable cost/benefit ratio, and (iii) the development of compds. with selective toxicity.  The present review describes the enzymes of the purine salvage pathway in M. tuberculosis as attractive targets for the development of new antimycobacterial agents.  Enzyme kinetics and structural data were included to provide a thorough knowledge on which to base the search for compds. with biol. activity.  The authors have focused on the mycobacterial homologues of this pathway as potential targets for the development of new antitubercular agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBhKqUvM--d7Vg90H21EOLACvtfcHk0ljUqG1QmImVBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOku78%253D&md5=6da82ca0d8c848913ee251c6c66ed7d6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F092986711795029627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711795029627%26sid%3Dliteratum%253Aachs%26aulast%3DDucati%26aufirst%3DR.%2BG.%26aulast%3DBreda%26aufirst%3DA.%26aulast%3DBasso%26aufirst%3DL.%2BA.%26aulast%3DSantos%26aufirst%3DD.%2BS.%26atitle%3DPurine%2520Salvage%2520Pathway%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DCurr.%2520Med.%2520Chem%26date%3D2011%26volume%3D18%26spage%3D1258%26epage%3D1275%26doi%3D10.2174%2F092986711795029627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villela, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Quitian, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducati, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, L. A.</span></span> <span> </span><span class="NLM_article-title">Pyrimidine Salvage Pathway in <i>Mycobacterium tuberculosis</i></span>. <i>Curr. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1286</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.2174/092986711795029555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.2174%2F092986711795029555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=21366534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1286-1298&author=A.+D.+Villelaauthor=Z.+A.+Sanchez-Quitianauthor=R.+G.+Ducatiauthor=D.+S.+Santosauthor=L.+A.+Basso&title=Pyrimidine+Salvage+Pathway+in+Mycobacterium+tuberculosis&doi=10.2174%2F092986711795029555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrimidine salvage pathway in Mycobacterium tuberculosis</span></div><div class="casAuthors">Villela, A. D.; Sanchez-Quitian, Z. A.; Ducati, R. G.; Santos, D. S.; Basso, L. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1286-1298</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The causative agent of tuberculosis (TB), Mycobacterium tuberculosis, infects one-third of the world population.  TB remains the leading cause of mortality due to a single bacterial pathogen.  The worldwide increase in incidence of M. tuberculosis has been attributed to the high proliferation rates of multi and extensively drug-resistant strains, and to co-infection with the human immunodeficiency virus.  There is thus a continuous requirement for studies on mycobacterial metab. to identify promising targets for the development of new agents to combat TB.  Singular characteristics of this pathogen, such as functional and structural features of enzymes involved in fundamental metabolic pathways, can be evaluated to identify possible targets for drug development.  Enzymes involved in the pyrimidine salvage pathway might be attractive targets for rational drug design against TB, since this pathway is vital for all bacterial cells, and is composed of enzymes considerably different from those present in humans.  Moreover, the enzymes of the pyrimidine salvage pathway might have an important role in the mycobacterial latent state, since M. tuberculosis has to recycle bases and/or nucleosides to survive in the hostile environment imposed by the host.  The present review describes the enzymes of M. tuberculosis pyrimidine salvage pathway as attractive targets for the development of new antimycobacterial agents.  Enzyme functional and structural data have been included to provide a broader knowledge on which to base the search for compds. with selective biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmR80dsbdfTrVg90H21EOLACvtfcHk0lj8RGDjBOFuvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOku70%253D&md5=00549ba222d7e08ce8cfd5a474c6cd86</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F092986711795029555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711795029555%26sid%3Dliteratum%253Aachs%26aulast%3DVillela%26aufirst%3DA.%2BD.%26aulast%3DSanchez-Quitian%26aufirst%3DZ.%2BA.%26aulast%3DDucati%26aufirst%3DR.%2BG.%26aulast%3DSantos%26aufirst%3DD.%2BS.%26aulast%3DBasso%26aufirst%3DL.%2BA.%26atitle%3DPyrimidine%2520Salvage%2520Pathway%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DCurr.%2520Med.%2520Chem%26date%3D2011%26volume%3D18%26spage%3D1286%26epage%3D1298%26doi%3D10.2174%2F092986711795029555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escuyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of a Unique Adenosine Kinase from <i>Mycobacterium tuberculosis</i></span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">6548</span>– <span class="NLM_lpage">6555</span>, <span class="refDoi"> DOI: 10.1128/jb.185.22.6548-6555.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1128%2Fjb.185.22.6548-6555.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=14594827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFKht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=2003&pages=6548-6555&author=M.+C.+Longauthor=V.+Escuyerauthor=W.+B.+Parker&title=Identification+and+Characterization+of+a+Unique+Adenosine+Kinase+from+Mycobacterium+tuberculosis&doi=10.1128%2Fjb.185.22.6548-6555.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of a unique adenosine kinase from Mycobacterium tuberculosis</span></div><div class="casAuthors">Long, Mary C.; Escuyer, Vincent; Parker, William B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6548-6555</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Adenosine kinase (AK) is a purine salvage enzyme that catalyzes the phosphorylation of adenosine to AMP.  In Mycobacterium tuberculosis, AK can also catalyze the phosphorylation of the adenosine analog 2-methyladenosine (methyl-Ado), the first step in the metab. of this compd. to an active form.  Purifn. of AK from M. tuberculosis yielded a 35-kDa protein that existed as a dimer in its native form.  Adenosine (Ado) was preferred as a substrate at least 30-fold (Km = 0.8±0.08 μM) over other natural nucleosides, and substrate inhibition was obsd. when Ado concns. exceeded 5 μM.  M. tuberculosis and human AKs exhibited different affinities for methyl-Ado, with Km values of 79 and 960 μM, resp., indicating that differences exist between the substrate binding sites of these enzymes.  ATP was a good phosphate donor (Km = 1100±140 μM); however, the activity levels obsd. with dGTP and GTP were 4.7 and 2.5 times the levels obsd. with ATP, resp.  M. tuberculosis AK activity was dependent on Mg2+, and activity was stimulated by potassium, as reflected by a decrease in the Km and an increase in Vmax for both Ado and methyl-Ado.  The N-terminal amino acid sequence of the purified enzyme revealed complete identity with Rv2202c, a protein currently classified as a hypothetical sugar kinase.  When an AK-deficient strain of M. tuberculosis (SRICK1) was transformed with this gene, it exhibited a 5,000-fold increase in AK activity compared to exts. from the original mutants.  These results verified that the protein that we identified as AK was coded for by Rv2202c.  AK is not commonly found in bacteria, and to the best of our knowledge, M. tuberculosis AK is the first bacterial AK to be characterized.  The enzyme shows greater sequence homol. with ribokinase and fructokinase than it does with other AKs.  The multiple differences that exist between M. tuberculosis and human AKs may provide the mol. basis for the development of nucleoside analog compds. with selective activity against M. tuberculosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLmpzZ1gIxfbVg90H21EOLACvtfcHk0lj8RGDjBOFuvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFKht7c%253D&md5=fd763397fd814d011383ed3b45f08043</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2Fjb.185.22.6548-6555.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.185.22.6548-6555.2003%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DEscuyer%26aufirst%3DV.%26aulast%3DParker%26aufirst%3DW.%2BB.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520a%2520Unique%2520Adenosine%2520Kinase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DJ.%2520Bacteriol.%26date%3D2003%26volume%3D185%26spage%3D6548%26epage%3D6555%26doi%3D10.1128%2Fjb.185.22.6548-6555.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palaninathan, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">High Resolution Crystal Structures ofMycobacterium tuberculosisAdenosine Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">27334</span>– <span class="NLM_lpage">27342</span>, <span class="refDoi"> DOI: 10.1074/jbc.m703290200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1074%2Fjbc.m703290200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=27334-27342&author=M.+C.+M.+Reddyauthor=S.+K.+Palaninathanauthor=N.+D.+Shettyauthor=J.+L.+Owenauthor=M.+D.+Watsonauthor=J.+C.+Sacchettini&title=High+Resolution+Crystal+Structures+ofMycobacterium+tuberculosisAdenosine+Kinase&doi=10.1074%2Fjbc.m703290200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m703290200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m703290200%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BC.%2BM.%26aulast%3DPalaninathan%26aufirst%3DS.%2BK.%26aulast%3DShetty%26aufirst%3DN.%2BD.%26aulast%3DOwen%26aufirst%3DJ.%2BL.%26aulast%3DWatson%26aufirst%3DM.%2BD.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DHigh%2520Resolution%2520Crystal%2520Structures%2520ofMycobacterium%2520tuberculosisAdenosine%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D27334%26epage%3D27342%26doi%3D10.1074%2Fjbc.m703290200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R. S.</span></span> <span> </span><span class="NLM_article-title">Adenosine kinase and ribokinase - the RK family of proteins</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2896</span>, <span class="refDoi"> DOI: 10.1007/s00018-008-8123-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1007%2Fs00018-008-8123-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=18560757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCqsr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2008&pages=2875-2896&author=J.+Parkauthor=R.+S.+Gupta&title=Adenosine+kinase+and+ribokinase+-+the+RK+family+of+proteins&doi=10.1007%2Fs00018-008-8123-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine kinase and ribokinase - the RK family of proteins</span></div><div class="casAuthors">Park, J.; Gupta, R. S.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2875-2896</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Ribokinase (RK) and adenosine kinase (AK) catalyze the phosphorylation of ribose and adenosine to ribose-5-phosphate and AMP, resp.  Belonging to the RK family of proteins, these enzymes share a no. of unique structural and functional elements.  Extensive work has been carried out on many aspects of these enzymes in recent years, and we summarize the wealth of information currently available on them.  The topics covered include descriptions of the primary and three-dimensional structures of AK and RK, their phylogenetic relationships, biochem. aspects of these enzymes including their reaction mechanisms and ionic requirements, and also work on certain inhibitors of these enzymes.  The cellular metab. and transport of ribose and adenosine are also briefly discussed, as well as the beneficial effects of ribose and adenosine in physiol. and how these effects can be harnessed for pharmacol. purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEqMyvD78pxbVg90H21EOLACvtfcHk0lj8RGDjBOFuvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCqsr3M&md5=c26f7345c2beb6741ee7a0850c86a4c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-8123-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-8123-1%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DGupta%26aufirst%3DR.%2BS.%26atitle%3DAdenosine%2520kinase%2520and%2520ribokinase%2520-%2520the%2520RK%2520family%2520of%2520proteins%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2008%26volume%3D65%26spage%3D2875%26epage%3D2896%26doi%3D10.1007%2Fs00018-008-8123-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ealick, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roos, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, R. G.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Toxoplasma gondii Adenosine Kinase Reveal a Novel Catalytic Mechanism and Prodrug Binding</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">875</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1006/jmbi.2000.3753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1006%2Fjmbi.2000.3753" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2000&pages=875-893&author=M.+A.+Schumacherauthor=D.+M.+Scottauthor=I.+I.+Mathewsauthor=S.+E.+Ealickauthor=D.+S.+Roosauthor=B.+Ullmanauthor=R.+G.+Brennan&title=Crystal+Structures+of+Toxoplasma+gondii+Adenosine+Kinase+Reveal+a+Novel+Catalytic+Mechanism+and+Prodrug+Binding&doi=10.1006%2Fjmbi.2000.3753"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.3753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.3753%26sid%3Dliteratum%253Aachs%26aulast%3DSchumacher%26aufirst%3DM.%2BA.%26aulast%3DScott%26aufirst%3DD.%2BM.%26aulast%3DMathews%26aufirst%3DI.%2BI.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26aulast%3DRoos%26aufirst%3DD.%2BS.%26aulast%3DUllman%26aufirst%3DB.%26aulast%3DBrennan%26aufirst%3DR.%2BG.%26atitle%3DCrystal%2520Structures%2520of%2520Toxoplasma%2520gondii%2520Adenosine%2520Kinase%2520Reveal%2520a%2520Novel%2520Catalytic%2520Mechanism%2520and%2520Prodrug%2520Binding%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D298%26spage%3D875%26epage%3D893%26doi%3D10.1006%2Fjmbi.2000.3753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ealick, S. E.</span></span> <span> </span><span class="NLM_article-title">Structure of Human Adenosine Kinase at 1.5 Å Resolution†,‡</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">15607</span>– <span class="NLM_lpage">15620</span>, <span class="refDoi"> DOI: 10.1021/bi9815445</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9815445" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK1cXmsleitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=15607-15620&author=I.+I.+Mathewsauthor=M.+D.+Erionauthor=S.+E.+Ealick&title=Structure+of+Human+Adenosine+Kinase+at+1.5+%C3%85+Resolution%E2%80%A0%2C%E2%80%A1&doi=10.1021%2Fbi9815445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human adenosine kinase at 1.5 Å resolution</span></div><div class="casAuthors">Mathews, Irimpan I.; Erion, Mark D.; Ealick, Steven E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">15607-15620</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adenosine kinase (I) is a key enzyme in the regulation of extracellular adenosine and intracellular adenylate levels.  Inhibitors of I elevate adenosine to levels that activate nearby adenosine receptors and produce a wide variety of therapeutically beneficial activities.  Accordingly, I is a promising target for new analgesic, neuroprotective, and cardioprotective agents.  Here, the authors detd. the structure of human I by x-ray crystallog. using MAD phasing techniques and refined the structure to 1.5 Å resoln.  The I structure consisted of 1 large α/β domain with 9 β-strands, 8 α-helixes, and 1 small α/β-domain with 5 β-strands and 2 α-helixes.  The active site was formed along the edge of the β-sheet in the large domain, whereas the small domain acted as a lid to cover the upper face of the active site.  The overall structure was similar to the recently reported structure of ribokinase from Escherichia coli.  The structure of ribokinase was detd. at 1.8 Å resoln. and represents the 1st structure of a new family of carbohydrate kinases.  Two mols. of adenosine were present in the I crystal structure with 1 adenosine mol. located in a site that matches the ribose site in ribokinase and probably represents the substrate-binding site.  The 2nd adenosine site overlapped the ADP site in ribokinase and probably represents the ATP site.  A Mg2+-binding site was obsd. in a trough between the 2 adenosine sites.  The structure of the active site was consistent with the obsd. substrate specificity.  The active-site model suggests that Asp-300 is an important catalytic residue involved in the deprotonation of the 5'-OH group during phosphate transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruB9B1HTkEJrVg90H21EOLACvtfcHk0lhZEI3_5f_12A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsleitbw%253D&md5=7aaad5a207883daea181c39e811e9af4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi9815445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9815445%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DI.%2BI.%26aulast%3DErion%26aufirst%3DM.%2BD.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DStructure%2520of%2520Human%2520Adenosine%2520Kinase%2520at%25201.5%2520%25C3%2585%2520Resolution%25E2%2580%25A0%252C%25E2%2580%25A1%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D15607%26epage%3D15620%26doi%3D10.1021%2Fbi9815445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romanello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachega, J. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassago, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandão-Neto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garratt, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, H. D. M.</span></span> <span> </span><span class="NLM_article-title">Adenosine kinase fromSchistosoma mansoni: structural basis for the differential incorporation of nucleoside analogues</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1107/s0907444912044800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444912044800" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=126-136&author=L.+Romanelloauthor=J.+F.+R.+Bachegaauthor=A.+Cassagoauthor=J.+Brand%C3%A3o-Netoauthor=R.+DeMarcoauthor=R.+C.+Garrattauthor=H.+D.+M.+Pereira&title=Adenosine+kinase+fromSchistosoma+mansoni%3A+structural+basis+for+the+differential+incorporation+of+nucleoside+analogues&doi=10.1107%2Fs0907444912044800"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1107%2Fs0907444912044800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444912044800%26sid%3Dliteratum%253Aachs%26aulast%3DRomanello%26aufirst%3DL.%26aulast%3DBachega%26aufirst%3DJ.%2BF.%2BR.%26aulast%3DCassago%26aufirst%3DA.%26aulast%3DBrand%25C3%25A3o-Neto%26aufirst%3DJ.%26aulast%3DDeMarco%26aufirst%3DR.%26aulast%3DGarratt%26aufirst%3DR.%2BC.%26aulast%3DPereira%26aufirst%3DH.%2BD.%2BM.%26atitle%3DAdenosine%2520kinase%2520fromSchistosoma%2520mansoni%253A%2520structural%2520basis%2520for%2520the%2520differential%2520incorporation%2520of%2520nucleoside%2520analogues%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D126%26epage%3D136%26doi%3D10.1107%2Fs0907444912044800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawronski, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeJesus, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akerley, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassetti, C. M.</span></span> <span> </span><span class="NLM_article-title">High-Resolution Phenotypic Profiling Defines Genes Essential for Mycobacterial Growth and Cholesterol Catabolism</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e1002251</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1002251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1371%2Fjournal.ppat.1002251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=21980284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCmsbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=J.+E.+Griffinauthor=J.+D.+Gawronskiauthor=M.+A.+DeJesusauthor=T.+R.+Ioergerauthor=B.+J.+Akerleyauthor=C.+M.+Sassetti&title=High-Resolution+Phenotypic+Profiling+Defines+Genes+Essential+for+Mycobacterial+Growth+and+Cholesterol+Catabolism&doi=10.1371%2Fjournal.ppat.1002251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution phenotypic profiling defines genes essential for mycobacterial growth and cholesterol catabolism</span></div><div class="casAuthors">Griffin, Jennifer E.; Gawronski, Jeffrey D.; DeJesus, Michael A.; Ioerger, Thomas R.; Akerley, Brian J.; Sassetti, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1002251</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The pathways that comprise cellular metab. are highly interconnected, and alterations in individual enzymes can have far-reaching effects.  As a result, global profiling methods that measure gene expression are of limited value in predicting how the loss of an individual function will affect the cell.  In this work, we employed a new method of global phenotypic profiling to directly define the genes required for the growth of Mycobacterium tuberculosis.  A combination of high-d. mutagenesis and deep-sequencing was used to characterize the compn. of complex mutant libraries exposed to different conditions.  This allowed the unambiguous identification of the genes that are essential for Mtb to grow in vitro, and proved to be a significant improvement over previous approaches.  To further explore functions that are required for persistence in the host, we defined the pathways necessary for the utilization of cholesterol, a crit. carbon source during infection.  Few of the genes we identified had previously been implicated in this adaptation by transcriptional profiling, and only a fraction were encoded in the chromosomal region known to encode sterol catabolic functions.  These genes comprise an unexpectedly large percentage of those previously shown to be required for bacterial growth in mouse tissue.  Thus, this single nutritional change accounts for a significant fraction of the adaptation to the host.  This work provides the most comprehensive genetic characterization of a sterol catabolic pathway to date, suggests putative roles for uncharacterized virulence genes, and precisely maps genes encoding potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZFTpTTOIkA7Vg90H21EOLACvtfcHk0lj3U-mtgvFMwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCmsbfN&md5=a97c2b9759b7a8bc9a304f5684701f13</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002251%26sid%3Dliteratum%253Aachs%26aulast%3DGriffin%26aufirst%3DJ.%2BE.%26aulast%3DGawronski%26aufirst%3DJ.%2BD.%26aulast%3DDeJesus%26aufirst%3DM.%2BA.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DAkerley%26aufirst%3DB.%2BJ.%26aulast%3DSassetti%26aufirst%3DC.%2BM.%26atitle%3DHigh-Resolution%2520Phenotypic%2520Profiling%2520Defines%2520Genes%2520Essential%2520for%2520Mycobacterial%2520Growth%2520and%2520Cholesterol%2520Catabolism%26jtitle%3DPLoS%2520Pathog.%26date%3D2011%26volume%3D7%26doi%3D10.1371%2Fjournal.ppat.1002251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothchild, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dartois, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galaviz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttenhower, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, E. J.</span></span> <span> </span><span class="NLM_article-title">Tryptophan Biosynthesis Protects Mycobacteria from CD4 T-Cell-Mediated Killing</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1296</span>– <span class="NLM_lpage">1308</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.10.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.cell.2013.10.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=24315099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFylurzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=1296-1308&author=Y.+J.+Zhangauthor=M.+C.+Reddyauthor=T.+R.+Ioergerauthor=A.+C.+Rothchildauthor=V.+Dartoisauthor=B.+M.+Schusterauthor=A.+Traunerauthor=D.+Wallisauthor=S.+Galavizauthor=C.+Huttenhowerauthor=J.+C.+Sacchettiniauthor=S.+M.+Beharauthor=E.+J.+Rubin&title=Tryptophan+Biosynthesis+Protects+Mycobacteria+from+CD4+T-Cell-Mediated+Killing&doi=10.1016%2Fj.cell.2013.10.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Tryptophan biosynthesis protects mycobacteria from CD4 T-cell-mediated killing</span></div><div class="casAuthors">Zhang, Yanjia J.; Reddy, Manchi C.; Ioerger, Thomas R.; Rothchild, Alissa C.; Dartois, Veronique; Schuster, Brian M.; Trauner, Andrej; Wallis, Deeann; Galaviz, Stacy; Huttenhower, Curtis; Sacchettini, James C.; Behar, Samuel M.; Rubin, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1296-1308</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bacteria that cause disease rely on their ability to counteract and overcome host defenses.  Here, we present a genome-scale study of Mycobacterium tuberculosis (Mtb) that uncovers the bacterial determinants of surviving host immunity, sets of genes we term "counteractomes.".  Through this anal., we found that CD4 T cells attempt to contain Mtb growth by starving it of tryptophan - a mechanism that successfully limits infections by Chlamydia and Leishmania, natural tryptophan auxotrophs.  Mtb, however, can synthesize tryptophan under stress conditions, and thus, starvation fails as an Mtb-killing mechanism.  We then identify a small-mol. inhibitor of Mtb tryptophan synthesis, which converts Mtb into a tryptophan auxotroph and restores the efficacy of a failed host defense.  Together, our findings demonstrate that the Mtb immune counteractomes serve as probes of host immunity, uncovering immune-mediated stresses that can be leveraged for therapeutic discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkCUFDhESMVrVg90H21EOLACvtfcHk0lj3U-mtgvFMwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFylurzM&md5=6947753d2524b4f7b352ddc94249317f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.10.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.10.045%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DReddy%26aufirst%3DM.%2BC.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DRothchild%26aufirst%3DA.%2BC.%26aulast%3DDartois%26aufirst%3DV.%26aulast%3DSchuster%26aufirst%3DB.%2BM.%26aulast%3DTrauner%26aufirst%3DA.%26aulast%3DWallis%26aufirst%3DD.%26aulast%3DGalaviz%26aufirst%3DS.%26aulast%3DHuttenhower%26aufirst%3DC.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DBehar%26aufirst%3DS.%2BM.%26aulast%3DRubin%26aufirst%3DE.%2BJ.%26atitle%3DTryptophan%2520Biosynthesis%2520Protects%2520Mycobacteria%2520from%2520CD4%2520T-Cell-Mediated%2520Killing%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D1296%26epage%3D1308%26doi%3D10.1016%2Fj.cell.2013.10.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship for nucleoside analogs as inhibitors or substrates of adenosine kinase from Mycobacterium tuberculosis</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1682</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2006.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.bcp.2006.03.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=1671-1682&author=M.+C.+Longauthor=W.+B.+Parker&title=Structure-activity+relationship+for+nucleoside+analogs+as+inhibitors+or+substrates+of+adenosine+kinase+from+Mycobacterium+tuberculosis&doi=10.1016%2Fj.bcp.2006.03.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2006.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2006.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DParker%26aufirst%3DW.%2BB.%26atitle%3DStructure-activity%2520relationship%2520for%2520nucleoside%2520analogs%2520as%2520inhibitors%2520or%2520substrates%2520of%2520adenosine%2520kinase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2006%26volume%3D71%26spage%3D1671%26epage%3D1682%26doi%3D10.1016%2Fj.bcp.2006.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaddix, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moukha-Chafiq, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship for adenosine kinase from Mycobacterium tuberculosis</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">1588</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2008.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.bcp.2008.01.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2008&pages=1588-1600&author=M.+C.+Longauthor=S.+C.+Shaddixauthor=O.+Moukha-Chafiqauthor=J.+A.+Maddryauthor=L.+Nagyauthor=W.+B.+Parker&title=Structure-activity+relationship+for+adenosine+kinase+from+Mycobacterium+tuberculosis&doi=10.1016%2Fj.bcp.2008.01.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2008.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2008.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DMoukha-Chafiq%26aufirst%3DO.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26aulast%3DNagy%26aufirst%3DL.%26aulast%3DParker%26aufirst%3DW.%2BB.%26atitle%3DStructure-activity%2520relationship%2520for%2520adenosine%2520kinase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2008%26volume%3D75%26spage%3D1588%26epage%3D1600%26doi%3D10.1016%2Fj.bcp.2008.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malnuit, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavětínská, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nauš, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Džubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdúch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolaříková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snášel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocek, M.</span></span> <span> </span><span class="NLM_article-title">2-Substituted 6-(Het)aryl-7-deazapurine Ribonucleosides: Synthesis, Inhibition of Adenosine Kinases, and Antimycobacterial Activity</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1079</span>– <span class="NLM_lpage">1093</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fcmdc.201500081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=25882678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsF2hu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1079-1093&author=V.+Malnuitauthor=L.+P.+Slav%C4%9Bt%C3%ADnsk%C3%A1author=P.+Nau%C5%A1author=P.+D%C5%BEub%C3%A1kauthor=M.+Hajd%C3%BAchauthor=J.+Stola%C5%99%C3%ADkov%C3%A1author=J.+Sn%C3%A1%C5%A1elauthor=I.+Pichov%C3%A1author=M.+Hocek&title=2-Substituted+6-%28Het%29aryl-7-deazapurine+Ribonucleosides%3A+Synthesis%2C+Inhibition+of+Adenosine+Kinases%2C+and+Antimycobacterial+Activity&doi=10.1002%2Fcmdc.201500081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">2-Substituted 6-(Het)aryl-7-deazapurine Ribonucleosides: Synthesis, Inhibition of Adenosine Kinases, and Antimycobacterial Activity</span></div><div class="casAuthors">Malnuit, Vincent; Slavetinska, Lenka Postova; Naus, Petr; Dzubak, Petr; Hajduch, Marian; Stolarikova, Jirina; Snasel, Jan; Pichova, Iva; Hocek, Michal</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1079-1093</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of 6-(hetero)aryl- or 6-methyl-7-deazapurine ribonucleosides, e.g. I, bearing a substituent at position 2 (Cl, F, NH2, or CH3) were prepd. by cross-coupling reactions at position 6 and functional group transformations at position 2.  Cytostatic, antiviral, and antimicrobial activity assays were performed.  The title compds. were obsd. to be potent and selective inhibitors of Mycobacterium tuberculosis adenosine kinase (ADK), but not human ADK; moreover, they were found to be non-cytotoxic.  The antimycobacterial activities against M. tuberculosis, however, were only moderate.  The reason for this could be due to either poor uptake through the cell wall or to parallel biosynthesis of adenosine monophosphate by the salvage pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxwXAKv9LqdbVg90H21EOLACvtfcHk0lh-HqR29r_U4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsF2hu70%253D&md5=1899f8370ad29dcfef9fee5a0de895dd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500081%26sid%3Dliteratum%253Aachs%26aulast%3DMalnuit%26aufirst%3DV.%26aulast%3DSlav%25C4%259Bt%25C3%25ADnsk%25C3%25A1%26aufirst%3DL.%2BP.%26aulast%3DNau%25C5%25A1%26aufirst%3DP.%26aulast%3DD%25C5%25BEub%25C3%25A1k%26aufirst%3DP.%26aulast%3DHajd%25C3%25BAch%26aufirst%3DM.%26aulast%3DStola%25C5%2599%25C3%25ADkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DSn%25C3%25A1%25C5%25A1el%26aufirst%3DJ.%26aulast%3DPichov%25C3%25A1%26aufirst%3DI.%26aulast%3DHocek%26aufirst%3DM.%26atitle%3D2-Substituted%25206-%2528Het%2529aryl-7-deazapurine%2520Ribonucleosides%253A%2520Synthesis%252C%2520Inhibition%2520of%2520Adenosine%2520Kinases%252C%2520and%2520Antimycobacterial%2520Activity%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1079%26epage%3D1093%26doi%3D10.1002%2Fcmdc.201500081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snášel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nauš, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dostál, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hnízda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourderioux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dušek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvořáková, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolaříková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zábranská, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konečný, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Džubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votruba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdúch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáčová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veverka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichová, I.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Inhibition of Mycobacterial and Human Adenosine Kinase by 7-Substituted 7-(Het)aryl-7-deazaadenine Ribonucleosides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8268</span>– <span class="NLM_lpage">8279</span>, <span class="refDoi"> DOI: 10.1021/jm500497v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500497v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1eju77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8268-8279&author=J.+Sn%C3%A1%C5%A1elauthor=P.+Nau%C5%A1author=J.+Dost%C3%A1lauthor=A.+Hn%C3%ADzdaauthor=J.+Fanfrl%C3%ADkauthor=J.+Bryndaauthor=A.+Bourderiouxauthor=M.+Du%C5%A1ekauthor=H.+Dvo%C5%99%C3%A1kov%C3%A1author=J.+Stola%C5%99%C3%ADkov%C3%A1author=H.+Z%C3%A1bransk%C3%A1author=R.+Pohlauthor=P.+Kone%C4%8Dn%C3%BDauthor=P.+D%C5%BEub%C3%A1kauthor=I.+Votrubaauthor=M.+Hajd%C3%BAchauthor=P.+%C5%98ez%C3%A1%C4%8Dov%C3%A1author=V.+Veverkaauthor=M.+Hocekauthor=I.+Pichov%C3%A1&title=Structural+Basis+for+Inhibition+of+Mycobacterial+and+Human+Adenosine+Kinase+by+7-Substituted+7-%28Het%29aryl-7-deazaadenine+Ribonucleosides&doi=10.1021%2Fjm500497v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of mycobacterial and human adenosine kinase by 7-substituted 7-(Het)aryl-7-deazaadenine ribonucleosides</span></div><div class="casAuthors">Snasel, Jan; Naus, Petr; Dostal, Jiri; Hnizda, Ales; Fanfrlik, Jindrich; Brynda, Jiri; Bourderioux, Aurelie; Dusek, Michal; Dvorakova, Hana; Stolarikova, Jirina; Zabranska, Helena; Pohl, Radek; Konecny, Petr; Dzubak, Petr; Votruba, Ivan; Hajduch, Marian; Rezacova, Pavlina; Veverka, Vaclav; Hocek, Michal; Pichova, Iva</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8268-8279</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adenosine kinase (ADK) from Mycobacterium tuberculosis (Mtb) was selected as a target for design of antimycobacterial nucleosides.  Screening of 7-(het)aryl-7-deazaadenine ribonucleosides with Mtb and human (h) ADKs and testing with wild-type and drug-resistant Mtb strains identified specific inhibitors of Mtb ADK with micromolar antimycobacterial activity and low cytotoxicity.  X-ray structures of complexes of Mtb and hADKs with 7-ethynyl-7-deazaadenosine showed differences in inhibitor interactions in the adenosine binding sites.  1D 1H STD NMR expts. revealed that these inhibitors are readily accommodated into the ATP and adenosine binding sites of Mtb ADK, whereas they bind preferentially into the adenosine site of hADK.  Occupation of the Mtb ADK ATP site with inhibitors and formation of catalytically less competent semiopen conformation of MtbADK after inhibitor binding in the adenosine site explain the lack of phosphorylation of 7-substituted-7-deazaadenosines.  Semiempirical quantum mech. anal. confirmed different affinity of nucleosides for the Mtb ADK adenosine and ATP sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGok5jNNyqDisbVg90H21EOLACvtfcHk0lh-HqR29r_U4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1eju77P&md5=0cd96e59d560dde8953e639db9dfa428</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm500497v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500497v%26sid%3Dliteratum%253Aachs%26aulast%3DSn%25C3%25A1%25C5%25A1el%26aufirst%3DJ.%26aulast%3DNau%25C5%25A1%26aufirst%3DP.%26aulast%3DDost%25C3%25A1l%26aufirst%3DJ.%26aulast%3DHn%25C3%25ADzda%26aufirst%3DA.%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DBourderioux%26aufirst%3DA.%26aulast%3DDu%25C5%25A1ek%26aufirst%3DM.%26aulast%3DDvo%25C5%2599%25C3%25A1kov%25C3%25A1%26aufirst%3DH.%26aulast%3DStola%25C5%2599%25C3%25ADkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DZ%25C3%25A1bransk%25C3%25A1%26aufirst%3DH.%26aulast%3DPohl%26aufirst%3DR.%26aulast%3DKone%25C4%258Dn%25C3%25BD%26aufirst%3DP.%26aulast%3DD%25C5%25BEub%25C3%25A1k%26aufirst%3DP.%26aulast%3DVotruba%26aufirst%3DI.%26aulast%3DHajd%25C3%25BAch%26aufirst%3DM.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258Dov%25C3%25A1%26aufirst%3DP.%26aulast%3DVeverka%26aufirst%3DV.%26aulast%3DHocek%26aufirst%3DM.%26aulast%3DPichov%25C3%25A1%26aufirst%3DI.%26atitle%3DStructural%2520Basis%2520for%2520Inhibition%2520of%2520Mycobacterial%2520and%2520Human%2520Adenosine%2520Kinase%2520by%25207-Substituted%25207-%2528Het%2529aryl-7-deazaadenine%2520Ribonucleosides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8268%26epage%3D8279%26doi%3D10.1021%2Fjm500497v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W.
B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaddix, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span> <span> </span><span class="NLM_article-title">Metabolism of 2-methyladenosine in <i>Mycobacterium tuberculosis</i></span>. <i>Tuberculosis</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.tube.2004.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.tube.2004.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=15207808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A280%3ADC%252BD2czgs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2004&pages=327-336&author=W.%0AB.+Parkerauthor=E.+W.+Barrowauthor=P.+W.+Allanauthor=S.+C.+Shaddixauthor=M.+C.+Longauthor=W.+W.+Barrowauthor=N.+Bansalauthor=J.+A.+Maddry&title=Metabolism+of+2-methyladenosine+in+Mycobacterium+tuberculosis&doi=10.1016%2Fj.tube.2004.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of 2-methyladenosine in Mycobacterium tuberculosis</span></div><div class="casAuthors">Parker William B; Barrow Esther W; Allan Paula W; Shaddix Sue C; Long Mary C; Barrow William W; Bansal Namita; Maddry Joseph A</div><div class="citationInfo"><span class="NLM_cas:title">Tuberculosis (Edinburgh, Scotland)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">327-36</span>
        ISSN:<span class="NLM_cas:issn">1472-9792</span>.
    </div><div class="casAbstract">2-Methyladenosine (methyl-Ado) has selective activity against Mycobacterium tuberculosis (M. tuberculosis).  In an effort to better understand its mechanism of action, we have characterized its metabolism in M. tuberculosis cells.  The primary intracellular metabolite of methyl-Ado was 2-methyl-adenylate (methyl-AMP).  Very little of the methyl-AMP was metabolized further.  A M. tuberculosis strain that was resistant to methyl-Ado did not express adenosine kinase and did not convert methyl-Ado to methyl-AMP in intact cells.  In contrast to these results, the primary intracellular metabolite of adenosine in M. tuberculosis cells was ATP, which was readily incorporated into RNA.  The rate of metabolism of methyl-Ado to methyl-AMP was similar to the rate of metabolism of adenosine to ATP.  Treatment of M. tuberculosis with methyl-Ado did not affect intracellular ATP levels.  Methyl-Ado and Ado were also cleaved to 2-methyladenine and adenine, respectively, which accumulated in the medium outside the cells.  These studies suggested that methyl-AMP was the active metabolite responsible for the cytotoxicity of this agent.  Furthermore, because methyl-Ado was poorly metabolized in human cells, these studies indicated that the selective activity of methyl-Ado was due to its selective activation by M. tuberculosis.  These studies have identified two enzyme reactions (Ado kinase and Ado cleavage) in M. tuberculosis that could be exploited for the rational design of new and selective anti-M. tuberculosis agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4k01LUFWZ4BfKb6ELK8t0fW6udTcc2eb4jaRzrgaC4Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czgs1Kiuw%253D%253D&md5=810bb7d4bc54e42ec5dec653c0ebcdf0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tube.2004.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tube.2004.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DAllan%26aufirst%3DP.%2BW.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26aulast%3DBansal%26aufirst%3DN.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26atitle%3DMetabolism%2520of%25202-methyladenosine%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DTuberculosis%26date%3D2004%26volume%3D84%26spage%3D327%26epage%3D336%26doi%3D10.1016%2Fj.tube.2004.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suling, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, N.</span></span> <span> </span><span class="NLM_article-title">The Metabolism of 2-methyladenosine in <i>Mycobacterium smegmatis</i></span>. <i>Microbiology</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1099/00221287-148-1-289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1099%2F00221287-148-1-289" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2002&pages=289-295&author=C.-K.+Chenauthor=W.+W.+Barrowauthor=W.+B.+Parkerauthor=E.+W.+Barrowauthor=J.+A.+Maddryauthor=W.+J.+Sulingauthor=P.+W.+Allanauthor=N.+Bansal&title=The+Metabolism+of+2-methyladenosine+in+Mycobacterium+smegmatis&doi=10.1099%2F00221287-148-1-289"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1099%2F00221287-148-1-289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F00221287-148-1-289%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.-K.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26aulast%3DSuling%26aufirst%3DW.%2BJ.%26aulast%3DAllan%26aufirst%3DP.%2BW.%26aulast%3DBansal%26aufirst%3DN.%26atitle%3DThe%2520Metabolism%2520of%25202-methyladenosine%2520in%2520Mycobacterium%2520smegmatis%26jtitle%3DMicrobiology%26date%3D2002%26volume%3D148%26spage%3D289%26epage%3D295%26doi%3D10.1099%2F00221287-148-1-289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suling, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span> <span> </span><span class="NLM_article-title">Antimycobacterial Activity of 2-methyl-adenosine</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1093/jac/dkg444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1093%2Fjac%2Fdkg444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=14563890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVektbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=801-808&author=E.+W.+Barrowauthor=L.+Westbrookauthor=N.+Bansalauthor=W.+J.+Sulingauthor=J.+A.+Maddryauthor=W.+B.+Parkerauthor=W.+W.+Barrow&title=Antimycobacterial+Activity+of+2-methyl-adenosine&doi=10.1093%2Fjac%2Fdkg444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Antimycobacterial activity of 2-methyl-adenosine</span></div><div class="casAuthors">Barrow, Esther W.; Westbrook, Louise; Bansal, Namita; Suling, William J.; Maddry, Joseph A.; Parker, William B.; Barrow, William W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">801-808</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The aims of this study were to assess the in vitro activity of 2-methyl-adenosine (I) against Mycobacterium tuberculosis, its intracellular efficacy, its effectiveness against M. tuberculosis in a persistent state model, and its potential mechanism of action.  In vitro activity was detd. by means of a colorimetric microdilution broth assay.  Intracellular activity was assessed with a Mono Mac 6 human monocytic cell line.  A hypoxic shift-down model was used to evaluate the effect of I on M. tuberculosis in a persistent state.  Mechanism-of-action studies were conducted by examg. the effect of I on the uptake of appropriate radiolabeled precursors into resp. mycobacterial macromol. components.  Studies confirmed the in vitro activity of I against M. tuberculosis and demonstrated intracellular efficacy against M. tuberculosis within macrophages.  I was able to significantly affect the viability of M. tuberculosis in a hypoxic shift-down model previously described to simulate the persistent state that results during tuberculosis.  Mechanism-of-action studies revealed that the immediate inhibitory effects of I were assocd. with protein and DNA synthesis and not RNA synthesis.  Results indicate that I, or similar derivs., might be effective against M. tuberculosis infections during latency.  This information should be helpful in understanding purine metab. of M. tuberculosis and also the metabolic activity of this important human pathogen in the persistent state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BOhd5Bqo4LVg90H21EOLACvtfcHk0liDx5WlcJOUzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVektbY%253D&md5=c033912d43527e5524320f0f8c27edd0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkg444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkg444%26sid%3Dliteratum%253Aachs%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DWestbrook%26aufirst%3DL.%26aulast%3DBansal%26aufirst%3DN.%26aulast%3DSuling%26aufirst%3DW.%2BJ.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26atitle%3DAntimycobacterial%2520Activity%2520of%25202-methyl-adenosine%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2003%26volume%3D52%26spage%3D801%26epage%3D808%26doi%3D10.1093%2Fjac%2Fdkg444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalmans, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Werve, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollen, M.</span></span> <span> </span><span class="NLM_article-title">Identification of the Glycogenic Compound 5-iodotubercidin as a General Protein Kinase Inhibitor</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>299</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1042/bj2990123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1042%2Fbj2990123" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=1994&pages=123-128&author=D.+Massillonauthor=W.+Stalmansauthor=G.+van+de+Werveauthor=M.+Bollen&title=Identification+of+the+Glycogenic+Compound+5-iodotubercidin+as+a+General+Protein+Kinase+Inhibitor&doi=10.1042%2Fbj2990123"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1042%2Fbj2990123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj2990123%26sid%3Dliteratum%253Aachs%26aulast%3DMassillon%26aufirst%3DD.%26aulast%3DStalmans%26aufirst%3DW.%26aulast%3Dvan%2Bde%2BWerve%26aufirst%3DG.%26aulast%3DBollen%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520the%2520Glycogenic%2520Compound%25205-iodotubercidin%2520as%2520a%2520General%2520Protein%2520Kinase%2520Inhibitor%26jtitle%3DBiochem.%2520J.%26date%3D1994%26volume%3D299%26spage%3D123%26epage%3D128%26doi%3D10.1042%2Fbj2990123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ugarkar, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DaRe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopcho, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schanzer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesner, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">Adenosine Kinase Inhibitors. 1. Synthesis, Enzyme Inhibition, and Antiseizure Activity of 5-Iodotubercidin Analogues</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2883</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm000024g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000024g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3cXksFGltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=2883-2893&author=B.+G.+Ugarkarauthor=J.+M.+DaReauthor=J.+J.+Kopchoauthor=C.+E.+Browneauthor=J.+M.+Schanzerauthor=J.+B.+Wiesnerauthor=M.+D.+Erion&title=Adenosine+Kinase+Inhibitors.+1.+Synthesis%2C+Enzyme+Inhibition%2C+and+Antiseizure+Activity+of+5-Iodotubercidin+Analogues&doi=10.1021%2Fjm000024g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine Kinase Inhibitors. 1. Synthesis, Enzyme Inhibition, and Anti-seizure Activity of 5-Iodotubercidin Analogues</span></div><div class="casAuthors">Ugarkar, Bheemarao G.; DaRe, Jay M.; Kopcho, Joseph J.; Browne, Clinton E. ,III; Schanzer, Juergen M.; Wiesner, James B.; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2883-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adenosine receptor agonists produce a wide variety of therapeutically useful pharmacologies.  However, to date they have failed to undergo successful clin. development due to dose-limiting side effects.  Adenosine kinase inhibitors (AKIs) represent an alternative strategy, since AKIs may raise local adenosine levels in a more site- and event-specific manner and thereby elicit the desired pharmacol. with a greater therapeutic window.  Starting with 5-iodotubercidin (IC50 = 0.026 μM) and 5'-amino-5'-deoxyadenosine (IC50 = 0.17 μM) as lead inhibitors of the isolated human AK, a variety of pyrrolo[2,3-d]pyrimidine nucleoside analogs were designed and prepd. by coupling 5-substituted-4-chloropyrrolo[2,3-d]pyrimidine bases with ribose analogs using the sodium salt-mediated glycosylation procedure.  5'-Amino-5'-deoxy analogs of 5-bromo- and 5-iodotubercidins were found to be the most potent AKIs reported to date (IC50s < 0.001 μM).  Several potent AKIs were shown to exhibit anticonvulsant activity in the rat maximal elec. shock (MES) induced seizure assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxKyAjFyc77Vg90H21EOLACvtfcHk0liDx5WlcJOUzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXksFGltbo%253D&md5=c7bcb2eeec305751b42af1a5abd44a99</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm000024g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000024g%26sid%3Dliteratum%253Aachs%26aulast%3DUgarkar%26aufirst%3DB.%2BG.%26aulast%3DDaRe%26aufirst%3DJ.%2BM.%26aulast%3DKopcho%26aufirst%3DJ.%2BJ.%26aulast%3DBrowne%26aufirst%3DC.%2BE.%26aulast%3DSchanzer%26aufirst%3DJ.%2BM.%26aulast%3DWiesner%26aufirst%3DJ.%2BB.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DAdenosine%2520Kinase%2520Inhibitors.%25201.%2520Synthesis%252C%2520Enzyme%2520Inhibition%252C%2520and%2520Antiseizure%2520Activity%2520of%25205-Iodotubercidin%2520Analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D2883%26epage%3D2893%26doi%3D10.1021%2Fjm000024g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loomis, C.
R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, R. M.</span></span> <span> </span><span class="NLM_article-title">Sangivamycin, a Nucleoside Analogue, is a Potent Inhibitor of Protein Kinase C</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>263</i></span>,  <span class="NLM_fpage">1682</span>– <span class="NLM_lpage">1692</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=1682-1692&author=C.%0AR.+Loomisauthor=R.+M.+Bell&title=Sangivamycin%2C+a+Nucleoside+Analogue%2C+is+a+Potent+Inhibitor+of+Protein+Kinase+C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLoomis%26aufirst%3DC.%2BR.%26aulast%3DBell%26aufirst%3DR.%2BM.%26atitle%3DSangivamycin%252C%2520a%2520Nucleoside%2520Analogue%252C%2520is%2520a%2520Potent%2520Inhibitor%2520of%2520Protein%2520Kinase%2520C%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D1682%26epage%3D1692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muchmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowart, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomtsyan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulenko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagwat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowaluk, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. L.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Human Adenosine Kinase Inhibitor Complexes Reveal Two Distinct Binding Modes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6726</span>– <span class="NLM_lpage">6731</span>, <span class="refDoi"> DOI: 10.1021/jm060189a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060189a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6726-6731&author=S.+W.+Muchmoreauthor=R.+A.+Smithauthor=A.+O.+Stewartauthor=M.+D.+Cowartauthor=A.+Gomtsyanauthor=M.+A.+Matulenkoauthor=H.+Yuauthor=J.+M.+Severinauthor=S.+S.+Bhagwatauthor=C.-H.+Leeauthor=E.+A.+Kowalukauthor=M.+F.+Jarvisauthor=C.+L.+Jakob&title=Crystal+Structures+of+Human+Adenosine+Kinase+Inhibitor+Complexes+Reveal+Two+Distinct+Binding+Modes&doi=10.1021%2Fjm060189a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm060189a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060189a%26sid%3Dliteratum%253Aachs%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DStewart%26aufirst%3DA.%2BO.%26aulast%3DCowart%26aufirst%3DM.%2BD.%26aulast%3DGomtsyan%26aufirst%3DA.%26aulast%3DMatulenko%26aufirst%3DM.%2BA.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DSeverin%26aufirst%3DJ.%2BM.%26aulast%3DBhagwat%26aufirst%3DS.%2BS.%26aulast%3DLee%26aufirst%3DC.-H.%26aulast%3DKowaluk%26aufirst%3DE.%2BA.%26aulast%3DJarvis%26aufirst%3DM.%2BF.%26aulast%3DJakob%26aufirst%3DC.%2BL.%26atitle%3DCrystal%2520Structures%2520of%2520Human%2520Adenosine%2520Kinase%2520Inhibitor%2520Complexes%2520Reveal%2520Two%2520Distinct%2520Binding%2520Modes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6726%26epage%3D6731%26doi%3D10.1021%2Fjm060189a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zemlicka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, T.</span></span> <span> </span><span class="NLM_article-title">O6-(4-Nitrophenyl)inosine and -Guanosine as Chromogenic Substrates for Adenosine Deaminase</span>. <i>Nucleosides Nucleotides</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1080/07328319608002410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1080%2F07328319608002410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK28XhtV2ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1996&pages=619-629&author=J.+Zemlickaauthor=T.+Endo&title=O6-%284-Nitrophenyl%29inosine+and+-Guanosine+as+Chromogenic+Substrates+for+Adenosine+Deaminase&doi=10.1080%2F07328319608002410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">O6-(4-nitrophenyl)inosine and -guanosine as chromogenic substrates for adenosine deaminase</span></div><div class="casAuthors">Zemlicka, Jiri; Endo, Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides & Nucleotides</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">619-29</span>CODEN:
                <span class="NLM_cas:coden">NUNUD5</span>;
        ISSN:<span class="NLM_cas:issn">0732-8311</span>.
    
            (<span class="NLM_cas:orgname">Dekker</span>)
        </div><div class="casAbstract">O6-(4-nitrophenyl)inosine (I), O6-(4-nitrophenyl)guanosine (II) and O6-(4-methylumbelliferonyl)inosine (III) were obtained by reaction of 6-chloro-9-(β-D-ribofuranosyl)purine or 2-amino-6-chloro-9-(β-D-ribofuranosyl)purine with sodium salts of 4-nitrophenol or 4-methylumbelliferone in N,N-dimethylformamide.  Similarly, 6-chloro-9-(β-D-2,3-isopropylideneribofuranosyl)purine was transformed to 2',3'-O-isopropylidene-O6-(4-nitrophenyl)inosine.  Compds. I and II are substrates for adenosine deaminase releasing 4-nitrophenol which is readily detected visually or spectrophotometrically.  Rate and extent of hydrolysis of I are significantly increase in the presence of purine nucleoside phosphorylase but xanthine oxidase has no influence.  A potential fluorogenic analog III is not a substrate for adenosine deaminase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq35yVXYd5rhLVg90H21EOLACvtfcHk0lj5Xjizovn0GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtV2ju7Y%253D&md5=7fd2b7f97f80bd9b2b50522119baa6bd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F07328319608002410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07328319608002410%26sid%3Dliteratum%253Aachs%26aulast%3DZemlicka%26aufirst%3DJ.%26aulast%3DEndo%26aufirst%3DT.%26atitle%3DO6-%25284-Nitrophenyl%2529inosine%2520and%2520-Guanosine%2520as%2520Chromogenic%2520Substrates%2520for%2520Adenosine%2520Deaminase%26jtitle%3DNucleosides%2520Nucleotides%26date%3D1996%26volume%3D15%26spage%3D619%26epage%3D629%26doi%3D10.1080%2F07328319608002410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluskey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moni, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, R. J.</span></span> <span> </span><span class="NLM_article-title">Diimidazo[1,2-c:4′,5′-e]pyrimidines: N6-N1 conformationally restricted adenosines</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(98)00101-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fs0960-894x%2898%2900101-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=695-698&author=D.+Campauthor=Y.+Liauthor=A.+McCluskeyauthor=R.+W.+Moniauthor=R.+J.+Quinn&title=Diimidazo%5B1%2C2-c%3A4%E2%80%B2%2C5%E2%80%B2-e%5Dpyrimidines%3A+N6-N1+conformationally+restricted+adenosines&doi=10.1016%2Fs0960-894x%2898%2900101-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fs0960-894x%2898%2900101-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0960-894x%252898%252900101-2%26sid%3Dliteratum%253Aachs%26aulast%3DCamp%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMcCluskey%26aufirst%3DA.%26aulast%3DMoni%26aufirst%3DR.%2BW.%26aulast%3DQuinn%26aufirst%3DR.%2BJ.%26atitle%3DDiimidazo%255B1%252C2-c%253A4%25E2%2580%25B2%252C5%25E2%2580%25B2-e%255Dpyrimidines%253A%2520N6-N1%2520conformationally%2520restricted%2520adenosines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1998%26volume%3D8%26spage%3D695%26epage%3D698%26doi%3D10.1016%2Fs0960-894x%2898%2900101-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katagiri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuhashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokufuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takebayashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, C.</span></span> <span> </span><span class="NLM_article-title">A Highly Efficient Synthesis of the Antiviral Agent (+)-cyclaradine Involving the Regioselective Cleavage of Epoxide by Neighboring Participation</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">1964</span>, <span class="refDoi"> DOI: 10.1016/s0040-4039(97)00234-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fs0040-4039%2897%2900234-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1997&pages=1961-1964&author=N.+Katagiriauthor=Y.+Matsuhashiauthor=H.+Kokufudaauthor=M.+Takebayashiauthor=C.+Kaneko&title=A+Highly+Efficient+Synthesis+of+the+Antiviral+Agent+%28%2B%29-cyclaradine+Involving+the+Regioselective+Cleavage+of+Epoxide+by+Neighboring+Participation&doi=10.1016%2Fs0040-4039%2897%2900234-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4039%2897%2900234-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4039%252897%252900234-7%26sid%3Dliteratum%253Aachs%26aulast%3DKatagiri%26aufirst%3DN.%26aulast%3DMatsuhashi%26aufirst%3DY.%26aulast%3DKokufuda%26aufirst%3DH.%26aulast%3DTakebayashi%26aufirst%3DM.%26aulast%3DKaneko%26aufirst%3DC.%26atitle%3DA%2520Highly%2520Efficient%2520Synthesis%2520of%2520the%2520Antiviral%2520Agent%2520%2528%252B%2529-cyclaradine%2520Involving%2520the%2520Regioselective%2520Cleavage%2520of%2520Epoxide%2520by%2520Neighboring%2520Participation%26jtitle%3DTetrahedron%2520Lett.%26date%3D1997%26volume%3D38%26spage%3D1961%26epage%3D1964%26doi%3D10.1016%2Fs0040-4039%2897%2900234-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuno, Y.</span></span> <span> </span><span class="NLM_article-title">A Novel Synthesis of 1-Deazaadenosine</span>. <i>Heterocycles</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.3987/s-1982-01-0305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.3987%2Fs-1982-01-0305" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1982&pages=305-309&author=T.+Itohauthor=T.+Sugawaraauthor=Y.+Mizuno&title=A+Novel+Synthesis+of+1-Deazaadenosine&doi=10.3987%2Fs-1982-01-0305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3987%2Fs-1982-01-0305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3987%252Fs-1982-01-0305%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DSugawara%26aufirst%3DT.%26aulast%3DMizuno%26aufirst%3DY.%26atitle%3DA%2520Novel%2520Synthesis%2520of%25201-Deazaadenosine%26jtitle%3DHeterocycles%26date%3D1982%26volume%3D17%26spage%3D305%26epage%3D309%26doi%3D10.3987%2Fs-1982-01-0305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilkens, S.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of ABC transporters</span>. <i>F1000Prime Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.12703/p7-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.12703%2Fp7-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=25750732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A280%3ADC%252BC2MnhvFWhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=14&author=S.+Wilkens&title=Structure+and+Mechanism+of+ABC+transporters&doi=10.12703%2Fp7-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of ABC transporters</span></div><div class="casAuthors">Wilkens Stephan</div><div class="citationInfo"><span class="NLM_cas:title">F1000prime reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14</span>
        ISSN:<span class="NLM_cas:issn">2051-7599</span>.
    </div><div class="casAbstract">All living organisms depend on primary and secondary membrane transport for the supply of external nutrients and removal or sequestration of unwanted (toxic) compounds.  Due to the chemical diversity of cellular molecules, it comes as no surprise that a significant part of the proteome is dedicated to the active transport of cargo across the plasma membrane or the membranes of subcellular organelles.  Transport against a chemical gradient can be driven by, for example, the free energy change associated with ATP hydrolysis (primary transport), or facilitated by the potential energy of the chemical gradient of another molecule (secondary transport).  Primary transporters include the rotary motor ATPases (F-, A-, and V-ATPases), P-type ATPases and a large family of integral membrane proteins referred to as "ABC" (ATP binding cassette) transporters.  ABC transporters are widespread in all forms of life and are characterized by two nucleotide-binding domains (NBD) and two transmembrane domains (TMDs).  ATP hydrolysis on the NBD drives conformational changes in the TMD, resulting in alternating access from inside and outside of the cell for unidirectional transport across the lipid bilayer.  Common to all ABC transporters is a signature sequence or motif, LSGGQ, that is involved in nucleotide binding.  Both importing and exporting ABC transporters are found in bacteria, whereas the majority of eukaryotic family members function in the direction of export.  Recent progress with the X-ray crystal structure determination of a variety of bacterial and eukaryotic ABC transporters has helped to advance our understanding of the ATP hydrolysis-driven transport mechanism but has also illustrated the large structural and functional diversity within the family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTnMdLvCnsfMK__Jf7-eMvJfW6udTcc2ebcxuxHcCeMprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnhvFWhsA%253D%253D&md5=c33f6447d0f32735fae9b6e48f5e18e9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.12703%2Fp7-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12703%252Fp7-14%26sid%3Dliteratum%253Aachs%26aulast%3DWilkens%26aufirst%3DS.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520ABC%2520transporters%26jtitle%3DF1000Prime%2520Rep.%26date%3D2015%26volume%3D7%26spage%3D14%26doi%3D10.12703%2Fp7-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braibant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilot, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Content, J.</span></span> <span> </span><span class="NLM_article-title">The ATP Binding Cassette (ABC) Transport Systems of <i>Mycobacterium tuberculosis</i></span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2000.tb00550.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1111%2Fj.1574-6976.2000.tb00550.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10978546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmt1Sjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2000&pages=449-467&author=M.+Braibantauthor=P.+Gilotauthor=J.+Content&title=The+ATP+Binding+Cassette+%28ABC%29+Transport+Systems+of+Mycobacterium+tuberculosis&doi=10.1111%2Fj.1574-6976.2000.tb00550.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The ATP binding cassette (ABC) transport systems of Mycobacterium tuberculosis</span></div><div class="casAuthors">Braibant, M.; Gilot, P.; Content, J.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-467</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 114 refs.  We have undertaken the inventory and assembly of the typical subunits of the ABC transporters encoded by the complete genome of Mycobacterium tuberculosis.  These subunits, i.e. the nucleotide binding domains (NBDs), the membrane-spanning domains (MSDs) and the substrate binding proteins (SBPs), were identified on the basis of their characteristic stretches of amino acids and/or conserved structure.  A total of 45 NBDs present in 38 proteins, of 47 MSDs present in 44 proteins and of 15 SBPs were found to be encoded by M. tuberculosis.  Anal. of transcriptional clusters and searches of homol. between the identified subunits of the transporters and proteins characterized in other organisms allowed the reconstitution of at least 26 complete (including at least one NBD and one MSD) and 11 incomplete ABC transporters.  Sixteen of them were unambiguously classified as importers whereas 21 were presumed to be exporters.  By searches of homol. with already known transporters from other organisms, potential substrates (peptides, macrolides, carbohydrates, multiple drugs, antibiotics, iron, anions) could be attributed to 30 of the ABC transporters identified in M. tuberculosis.  The ABC transporters have been further classified in nine different sub-families according to a tree obtained from the clustering of their NBDs.  Contrary to Escherichia coli and similarly to Bacillus subtilis, there is an equal representation of extruders and importers.  Many exporters were found to be potentially implicated in the transport of drugs, probably contributing to the resistance of M. tuberculosis to many antibiotics.  Interestingly, a transporter (absent in E. coli and in B. subtilis) potentially implicated in the export of a factor required for the bacterial attachment to the eukaryotic host cells was also identified.  In comparison to E. coli and B. subtilis, there is an under-representation of the importers (with the exception of the phosphate importers) in M. tuberculosis.  This may reflect the capacity of this bacterium to synthesize many essential compds. and to grow in the presence of few external nutrients.  The genes encoding the ABC transporters occupy about 2.5% of the genome of M. tuberculosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg1LOuNresYrVg90H21EOLACvtfcHk0liuzlb076b0Jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmt1Sjtr0%253D&md5=7ba9ad7b0615bf971cf8d402c10cc1ea</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2000.tb00550.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2000.tb00550.x%26sid%3Dliteratum%253Aachs%26aulast%3DBraibant%26aufirst%3DM.%26aulast%3DGilot%26aufirst%3DP.%26aulast%3DContent%26aufirst%3DJ.%26atitle%3DThe%2520ATP%2520Binding%2520Cassette%2520%2528ABC%2529%2520Transport%2520Systems%2520of%2520Mycobacterium%2520tuberculosis%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2000%26volume%3D24%26spage%3D449%26epage%3D467%26doi%3D10.1111%2Fj.1574-6976.2000.tb00550.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sandhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhter, Y.</span></span> <span> </span><span class="NLM_article-title">The Drug Binding Sites and Transport Mechanism of the RND Pumps from <i>Mycobacterium tuberculosis</i>: Insights from Molecular Dynamics Simulations</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>592</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2016.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.abb.2016.01.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=592&publication_year=2016&pages=38-49&author=P.+Sandhuauthor=Y.+Akhter&title=The+Drug+Binding+Sites+and+Transport+Mechanism+of+the+RND+Pumps+from+Mycobacterium+tuberculosis%3A+Insights+from+Molecular+Dynamics+Simulations&doi=10.1016%2Fj.abb.2016.01.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2016.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2016.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DSandhu%26aufirst%3DP.%26aulast%3DAkhter%26aufirst%3DY.%26atitle%3DThe%2520Drug%2520Binding%2520Sites%2520and%2520Transport%2520Mechanism%2520of%2520the%2520RND%2520Pumps%2520from%2520Mycobacterium%2520tuberculosis%253A%2520Insights%2520from%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2016%26volume%3D592%26spage%3D38%26epage%3D49%26doi%3D10.1016%2Fj.abb.2016.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Rossi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aínsa, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccardi, G.</span></span> <span> </span><span class="NLM_article-title">Role of Mycobacterial Efflux Transporters in Drug Resistance: an Unresolved Question</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2005.00002.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1111%2Fj.1574-6976.2005.00002.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=16438679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2006&pages=36-52&author=E.+De%0ARossiauthor=J.+A.+A%C3%ADnsaauthor=G.+Riccardi&title=Role+of+Mycobacterial+Efflux+Transporters+in+Drug+Resistance%3A+an+Unresolved+Question&doi=10.1111%2Fj.1574-6976.2005.00002.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Role of mycobacterial efflux transporters in drug resistance: an unresolved question</span></div><div class="casAuthors">De Rossi, Edda; Ainsa, Jose A.; Riccardi, Giovanna</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-52</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Two mechanisms are thought to be involved in the natural drug resistance of mycobacteria: the mycobacterial cell wall permeability barrier and active multidrug efflux pumps.  Genes encoding drug efflux transporters have been isolated from several mycobacterial species.  These proteins transport tetracycline, fluoroquinolones, aminoglycosides and other compds.  Recent reports have suggested that efflux pumps may also be involved in transporting isoniazid, one of the main drugs used to treat tuberculosis.  This review highlights recent advances in our understanding of efflux-mediated drug resistance in mycobacteria, including the distribution of efflux systems in these organisms, their substrate profiles and their contribution to drug resistance.  The balance between the drug transport into the cell and drug efflux is not yet clearly understood, and further studies are required in mycobacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpwjdJiKjbO7Vg90H21EOLACvtfcHk0lgJJXqKgivvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D&md5=41af6cedab317d501c489400a6979920</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2005.00002.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2005.00002.x%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BRossi%26aufirst%3DE.%26aulast%3DA%25C3%25ADnsa%26aufirst%3DJ.%2BA.%26aulast%3DRiccardi%26aufirst%3DG.%26atitle%3DRole%2520of%2520Mycobacterial%2520Efflux%2520Transporters%2520in%2520Drug%2520Resistance%253A%2520an%2520Unresolved%2520Question%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2006%26volume%3D30%26spage%3D36%26epage%3D52%26doi%3D10.1111%2Fj.1574-6976.2005.00002.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balganesh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinesh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruppath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, U.</span></span> <span> </span><span class="NLM_article-title">Efflux Pumps of Mycobacterium tuberculosis Play a Significant Role in Antituberculosis Activity of Potential Drug Candidates</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2643</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1128/aac.06003-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1128%2Faac.06003-11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=2643-2651&author=M.+Balganeshauthor=N.+Dineshauthor=S.+Sharmaauthor=S.+Kuruppathauthor=A.+V.+Nairauthor=U.+Sharma&title=Efflux+Pumps+of+Mycobacterium+tuberculosis+Play+a+Significant+Role+in+Antituberculosis+Activity+of+Potential+Drug+Candidates&doi=10.1128%2Faac.06003-11"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1128%2Faac.06003-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.06003-11%26sid%3Dliteratum%253Aachs%26aulast%3DBalganesh%26aufirst%3DM.%26aulast%3DDinesh%26aufirst%3DN.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKuruppath%26aufirst%3DS.%26aulast%3DNair%26aufirst%3DA.%2BV.%26aulast%3DSharma%26aufirst%3DU.%26atitle%3DEfflux%2520Pumps%2520of%2520Mycobacterium%2520tuberculosis%2520Play%2520a%2520Significant%2520Role%2520in%2520Antituberculosis%2520Activity%2520of%2520Potential%2520Drug%2520Candidates%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D2643%26epage%3D2651%26doi%3D10.1128%2Faac.06003-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pule, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Helden, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Victor, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louw, G. E.</span></span> <span> </span><span class="NLM_article-title">Efflux Pump Inhibitors: Targeting Mycobacterial Efflux Systems to Enhance TB Therapy</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1093/jac/dkv316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1093%2Fjac%2Fdkv316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=26472768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC28XotVeqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2016&pages=17-26&author=C.+M.+Puleauthor=S.+L.+Sampsonauthor=R.+M.+Warrenauthor=P.+A.+Blackauthor=P.+D.+van+Heldenauthor=T.+C.+Victorauthor=G.+E.+Louw&title=Efflux+Pump+Inhibitors%3A+Targeting+Mycobacterial+Efflux+Systems+to+Enhance+TB+Therapy&doi=10.1093%2Fjac%2Fdkv316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Efflux pump inhibitors: targeting mycobacterial efflux systems to enhance TB therapy</span></div><div class="casAuthors">Pule, Caroline M.; Sampson, Samantha L.; Warren, Robin M.; Black, Philippa A.; van Helden, Paul D.; Victor, Tommie C.; Louw, Gail E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-26</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The emergence of drug resistance continues to plague TB control, with a global increase in the prevalence of MDR-TB.  This acts as a gateway to XDR-TB and thus emphasizes the urgency for drug development and optimal treatment options.  Bedaquiline is the first new anti-TB drug approved by the FDA in 40 years and has been shown to be an effective treatment option for MDR Mycobacterium tuberculosis infection.  Bedaquiline has also recently been included in clin. trials for new regimens with the aim of improving and shortening treatment periods.  Alarmingly, efflux-mediated bedaquiline resistance, as well as efflux-mediated cross-resistance to clofazimine, has been identified in treatment failures.  This mechanism of resistance results in efflux of a variety of anti-TB drugs from the bacterial cell, thereby decreasing the intracellular drug concn.  In doing so, the bacillus is able to render the antibiotic treatment ineffective.  Recent studies have explored strategies to reverse the resistance phenotype conferred by efflux pump activation.  It was obsd. that the addn. of efflux pump inhibitors partially restored drug susceptibility in vitro and in vivo.  This has significant clin. implications, esp. in MDR-TB management where treatment options are extremely limited.  This review aims to highlight the current efflux pump inhibitors effective against M. tuberculosis, the effect of efflux pump inhibitors on mycobacterial growth and the clin. promise of treatment with efflux pump inhibitors and std. anti-TB therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6OyOfnhS48rVg90H21EOLACvtfcHk0lgJJXqKgivvRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotVeqsrs%253D&md5=c72ef921e040c1d9787cec02639bcd7d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkv316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkv316%26sid%3Dliteratum%253Aachs%26aulast%3DPule%26aufirst%3DC.%2BM.%26aulast%3DSampson%26aufirst%3DS.%2BL.%26aulast%3DWarren%26aufirst%3DR.%2BM.%26aulast%3DBlack%26aufirst%3DP.%2BA.%26aulast%3Dvan%2BHelden%26aufirst%3DP.%2BD.%26aulast%3DVictor%26aufirst%3DT.%2BC.%26aulast%3DLouw%26aufirst%3DG.%2BE.%26atitle%3DEfflux%2520Pump%2520Inhibitors%253A%2520Targeting%2520Mycobacterial%2520Efflux%2520Systems%2520to%2520Enhance%2520TB%2520Therapy%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2016%26volume%3D71%26spage%3D17%26epage%3D26%26doi%3D10.1093%2Fjac%2Fdkv316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R. S.</span></span> <span> </span><span class="NLM_article-title">Mycobacterial Adenosine Kinase is not a Typical Adenosine Kinase</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>583</i></span>,  <span class="NLM_fpage">2231</span>– <span class="NLM_lpage">2236</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2009.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.febslet.2009.06.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2009&pages=2231-2236&author=J.+Parkauthor=B.+Singhauthor=R.+S.+Gupta&title=Mycobacterial+Adenosine+Kinase+is+not+a+Typical+Adenosine+Kinase&doi=10.1016%2Fj.febslet.2009.06.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2009.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2009.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DGupta%26aufirst%3DR.%2BS.%26atitle%3DMycobacterial%2520Adenosine%2520Kinase%2520is%2520not%2520a%2520Typical%2520Adenosine%2520Kinase%26jtitle%3DFEBS%2520Lett.%26date%3D2009%26volume%3D583%26spage%3D2231%26epage%3D2236%26doi%3D10.1016%2Fj.febslet.2009.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shashank, R.</span></span> <span> </span><span class="NLM_article-title">Application of LC-MS in Supporting PK/PD Studies During Drug Discovery and Development</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">2514</span>– <span class="NLM_lpage">2526</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=2514-2526&author=T.+R.+Ahmedauthor=R.+Shashank&title=Application+of+LC-MS+in+Supporting+PK%2FPD+Studies+During+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAhmed%26aufirst%3DT.%2BR.%26aulast%3DShashank%26aufirst%3DR.%26atitle%3DApplication%2520of%2520LC-MS%2520in%2520Supporting%2520PK%252FPD%2520Studies%2520During%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DPharm.%2520Res.%26date%3D2012%26volume%3D5%26spage%3D2514%26epage%3D2526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hull, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabemampianina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorstenbosch, C. V. D.</span></span> <span> </span><span class="NLM_article-title">A Good Practice Guide to the Administration of Substances and Removal of Blood, Including Routes and Volumes</span>. <i>J. Appl. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1002/jat.727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fjat.727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=11180276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1Kmurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=15-23&author=K.-H.+Diehlauthor=R.+Hullauthor=D.+Mortonauthor=R.+Pfisterauthor=Y.+Rabemampianinaauthor=D.+Smithauthor=J.-M.+Vidalauthor=C.+V.+D.+Vorstenbosch&title=A+Good+Practice+Guide+to+the+Administration+of+Substances+and+Removal+of+Blood%2C+Including+Routes+and+Volumes&doi=10.1002%2Fjat.727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A good practice guide to the administration of substances and removal of blood, including routes and volumes</span></div><div class="casAuthors">Diehl, Karl-Heinz; Hull, Robin; Morton, David; Pfister, Rudolf; Rabemampianina, Yvon; Smith, David; Vidal, Jean-Marc; Van de Vorstenbosch, Cor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Toxicology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-23</span>CODEN:
                <span class="NLM_cas:coden">JJATDK</span>;
        ISSN:<span class="NLM_cas:issn">0260-437X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">This article is the result of an initiative between the European Federation of Pharmaceutical Industries Assocns. (EFPIA) and the European Center for the Validation of Alternative Methods (ECVAM).  Its objectives are to provide the researcher in the safety evaluation lab. with an up-to-date, easy-to-use set of data sheets to aid in the study design process while at the same time affording max. welfare considerations to the exptl. animals.  Although this article is targeted at researchers in the European Pharmaceutical Industry, it is considered that the principles underpinning the data sets and refinement proposals are equally applicable to all those who use these techniques on animals in their research, whether in research institutes, universities or other sectors of industry.  The implications of this article may lead to discussion with regulators, such as those responsible for pharmacopoeial testing.  There are numerous publications dealing with the administration of test substances and the removal of blood samples, and many labs. also have their own "inhouse" guidelines that have been developed by custom and practice over many years.  Within European Union Directive 86/609EEC we have an obligation to refine expts. to cause the min. amt. of stress.  We hope that this article will provide background data useful to those responsible for protocol design and review.  This guide is based on peer-reviewed publications whenever possible, but where this is not possible we have used "inhouse" data and the experience of those on the working party (as well as helpful comments submitted by the industry) for a final opinion.  The guide also addresses the continuing need to refine the techniques assocd. with the administration of substances and the withdrawal of blood, and suggests ways of doing so.  Data-sharing between labs. should be encouraged to avoid duplication of animal work, as well as sharing practical skills concerning animal welfare and scientific problems caused by "overdosing" in some way or another.  The recommendations in this guide refer to the "normal" animal, and special consideration is needed, for instance, during pregnancy and lactation.  Interpretation of studies may be confounded when large vols. are administered or excessive sampling employed, particularly if anesthetics are used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpirIZeMEFS_7Vg90H21EOLACvtfcHk0lhe-lKXpvIdBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1Kmurw%253D&md5=a1dca38550737c5ab02ae49f2c8eed34</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fjat.727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjat.727%26sid%3Dliteratum%253Aachs%26aulast%3DDiehl%26aufirst%3DK.-H.%26aulast%3DHull%26aufirst%3DR.%26aulast%3DMorton%26aufirst%3DD.%26aulast%3DPfister%26aufirst%3DR.%26aulast%3DRabemampianina%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DVidal%26aufirst%3DJ.-M.%26aulast%3DVorstenbosch%26aufirst%3DC.%2BV.%2BD.%26atitle%3DA%2520Good%2520Practice%2520Guide%2520to%2520the%2520Administration%2520of%2520Substances%2520and%2520Removal%2520of%2520Blood%252C%2520Including%2520Routes%2520and%2520Volumes%26jtitle%3DJ.%2520Appl.%2520Toxicol.%26date%3D2001%26volume%3D21%26spage%3D15%26epage%3D23%26doi%3D10.1002%2Fjat.727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isbell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span> <span> </span><span class="NLM_article-title">Snapshot PK: a Rapid Rodent <i>in vivo</i> Preclinical Screening Approach</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2007.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.drudis.2007.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=18405850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A280%3ADC%252BD1c3kslSkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=360-367&author=B.+Liuauthor=J.+Changauthor=W.+P.+Gordonauthor=J.+Isbellauthor=Y.+Zhouauthor=T.+Tuntland&title=Snapshot+PK%3A+a+Rapid+Rodent+in+vivo+Preclinical+Screening+Approach&doi=10.1016%2Fj.drudis.2007.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Snapshot PK: a rapid rodent in vivo preclinical screening approach</span></div><div class="casAuthors">Liu Bo; Chang Jonathan; Gordon William P; Isbell John; Zhou Yingyao; Tuntland Tove</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">360-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Described in this article are strategies implemented to increase the throughput of in vivo rodent pharmacokinetic (PK) studies using the snapshot PK study design and automated methods for compound submission, sample processing, data analysis and reporting.  Applying snapshot PK studies to categorize the oral exposure of >1300 discovery compounds as low, moderate or high resulted in an attrition rate of 86%.  The follow up full PK studies on the remaining compounds found that 98% of the compounds were predicted in the correct (69%) or adjacent (29%) oral exposure category by the snapshot PK studies.  These results demonstrate that the snapshot PK screen in rodents can serve as an effective and efficient in vivo tool in the compound selection process in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR7lxjN22Wtb4xhnOQkIHOxfW6udTcc2eZM0jttHOIqsLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3kslSkuw%253D%253D&md5=2f0ca34fc2b06e13fc85fb28be221ffe</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGordon%26aufirst%3DW.%2BP.%26aulast%3DIsbell%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTuntland%26aufirst%3DT.%26atitle%3DSnapshot%2520PK%253A%2520a%2520Rapid%2520Rodent%2520in%2520vivo%2520Preclinical%2520Screening%2520Approach%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2008%26volume%3D13%26spage%3D360%26epage%3D367%26doi%3D10.1016%2Fj.drudis.2007.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinn, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohaideen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boshoff, H. I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizrahi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassetti, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, C. E.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">Identification of New Drug Targets and Resistance Mechanisms in <i>Mycobacterium tuberculosis</i></span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e75245</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0075245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1371%2Fjournal.pone.0075245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=24086479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFeku7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=T.+R.+Ioergerauthor=T.+O%E2%80%99Malleyauthor=R.+Liaoauthor=K.+M.+Guinnauthor=M.+J.+Hickeyauthor=N.+Mohaideenauthor=K.+C.+Murphyauthor=H.+I.+M.+Boshoffauthor=V.+Mizrahiauthor=E.+J.+Rubinauthor=C.+M.+Sassettiauthor=C.+E.+Barryauthor=D.+R.+Shermanauthor=T.+Parishauthor=J.+C.+Sacchettini&title=Identification+of+New+Drug+Targets+and+Resistance+Mechanisms+in+Mycobacterium+tuberculosis&doi=10.1371%2Fjournal.pone.0075245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of new drug targets and resistance mechanisms in Mycobacterium tuberculosis</span></div><div class="casAuthors">Ioerger, Thomas R.; O'Malley, Theresa; Liao, Reiling; Guinn, Kristine M.; Hickey, Mark J.; Mohaideen, Nilofar; Murphy, Kenan C.; Boshoff, Helena I. M.; Mizrahi, Valerie; Rubin, Eric J.; Sassetti, Christopher M.; Barry, Clifton E., III; Sherman, David R.; Parish, Tanya; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e75245</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Identification of new drug targets is vital for the advancement of drug discovery against Mycobacterium tuberculosis, esp. given the increase of resistance worldwide to first- and second-line drugs.  Because traditional target-based screening has largely proven unsuccessful for antibiotic discovery, we have developed a scalable platform for target identification in M. tuberculosis that is based on whole-cell screening, coupled with whole-genome sequencing of resistant mutants and recombineering to confirm.  The method yields targets paired with whole-cell active compds., which can serve as novel scaffolds for drug development, mol. tools for validation, and/or as ligands for co-crystn.  It may also reveal other information about mechanisms of action, such as activation or efflux.  Using this method, we identified resistance-linked genes for eight compds. with anti-tubercular activity.  Four of the genes have previously been shown to be essential: AspS, aspartyl-tRNA synthetase, Pks13, a polyketide synthase involved in mycolic acid biosynthesis, MmpL3, a membrane transporter, and EccB3, a component of the ESX-3 type VII secretion system.  AspS and Pks13 represent novel targets in protein translation and cell-wall biosynthesis.  Both MmpL3 and EccB3 are involved in membrane transport.  Pks13, AspS, and EccB3 represent novel candidates not targeted by existing TB drugs, and the availability of whole-cell active inhibitors greatly increases their potential for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoen2dg2HIM0LVg90H21EOLACvtfcHk0lgzVldMiAMPqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFeku7rM&md5=2df0db796fc57f875aa12d4ac5f09984</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0075245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0075245%26sid%3Dliteratum%253Aachs%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DR.%26aulast%3DGuinn%26aufirst%3DK.%2BM.%26aulast%3DHickey%26aufirst%3DM.%2BJ.%26aulast%3DMohaideen%26aufirst%3DN.%26aulast%3DMurphy%26aufirst%3DK.%2BC.%26aulast%3DBoshoff%26aufirst%3DH.%2BI.%2BM.%26aulast%3DMizrahi%26aufirst%3DV.%26aulast%3DRubin%26aufirst%3DE.%2BJ.%26aulast%3DSassetti%26aufirst%3DC.%2BM.%26aulast%3DBarry%26aufirst%3DC.%2BE.%26aulast%3DSherman%26aufirst%3DD.%2BR.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DIdentification%2520of%2520New%2520Drug%2520Targets%2520and%2520Resistance%2520Mechanisms%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0075245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesula, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobos, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortune, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizrahi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">Variation among Genome Sequences of H37Rv Strains of <i>Mycobacterium tuberculosis</i> from Multiple Laboratories</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">3645</span>– <span class="NLM_lpage">3653</span>, <span class="refDoi"> DOI: 10.1128/jb.00166-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1128%2Fjb.00166-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=20472797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCktLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2010&pages=3645-3653&author=T.+R.+Ioergerauthor=Y.+Fengauthor=K.+Ganesulaauthor=X.+Chenauthor=K.+M.+Dobosauthor=S.+Fortuneauthor=W.+R.+Jacobsauthor=V.+Mizrahiauthor=T.+Parishauthor=E.+Rubinauthor=C.+Sassettiauthor=J.+C.+Sacchettini&title=Variation+among+Genome+Sequences+of+H37Rv+Strains+of+Mycobacterium+tuberculosis+from+Multiple+Laboratories&doi=10.1128%2Fjb.00166-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Variation among genome sequences of H37Rv strains of Mycobacterium tuberculosis from multiple laboratories</span></div><div class="casAuthors">Ioerger, Thomas R.; Feng, Yicheng; Ganesula, Krishna; Chen, Xiaohua; Dobos, Karen M.; Fortune, Sarah; Jacobs, William R., Jr.; Mizrahi, Valerie; Parish, Tanya; Rubin, Eric; Sassetti, Chris; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3645-3653</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The publication of the complete genome sequence for Mycobacterium tuberculosis H37Rv in 1998 has had a great impact on the research community.  Nonetheless, it is suspected that genetic differences have arisen in stocks of H37Rv that are maintained in different labs.  In order to assess the consistency of the genome sequences among H37Rv strains in use and the extent to which they have diverged from the original strain sequenced, we carried out whole-genome sequencing on 6 strains of H37Rv from different labs.  Polymorphisms at 73 sites were obsd., which were shared among the lab strains, though 72 of these were also shared with H37Ra and are likely to be due to sequencing errors in the original H37Rv ref. sequence.  An updated H37Rv genome sequence should be valuable to the tuberculosis research community as well as the broader microbial research community.  In addn., several polymorphisms unique to individual strains and several shared polymorphisms were identified and shown to be consistent with the known provenance of these strains.  Aside from nucleotide substitutions and insertion/deletions, multiple IS6110 transposition events were obsd., supporting the theory that they play a significant role in plasticity of the M. tuberculosis genome.  This genome-wide catalog of genetic differences can help explain any phenotypic differences that might be found, including a frameshift mutation in the mycocerosic acid synthase gene which causes two of the strains to be deficient in biosynthesis of the surface glycolipid phthiocerol dimycocerosate (PDIM).  The resequencing of these 6 lab strains represents a fortuitous "in vitro evolution" expt. that demonstrates how the M. tuberculosis genome continues to evolve even in a controlled environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD6qz-dULL8rVg90H21EOLACvtfcHk0lgzVldMiAMPqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCktLfE&md5=19086a827c0d86ac6b734fcc9dcb033b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2Fjb.00166-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.00166-10%26sid%3Dliteratum%253Aachs%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGanesula%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDobos%26aufirst%3DK.%2BM.%26aulast%3DFortune%26aufirst%3DS.%26aulast%3DJacobs%26aufirst%3DW.%2BR.%26aulast%3DMizrahi%26aufirst%3DV.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DRubin%26aufirst%3DE.%26aulast%3DSassetti%26aufirst%3DC.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DVariation%2520among%2520Genome%2520Sequences%2520of%2520H37Rv%2520Strains%2520of%2520Mycobacterium%2520tuberculosis%2520from%2520Multiple%2520Laboratories%26jtitle%3DJ.%2520Bacteriol.%26date%3D2010%26volume%3D192%26spage%3D3645%26epage%3D3653%26doi%3D10.1128%2Fjb.00166-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S.</span>; <span class="NLM_string-name">Lee, H.
S.</span>; <span class="NLM_string-name">Franzblau, S.</span></span> <span> </span><span class="NLM_article-title">Microplate Alamar Blue Assay (MABA) and Low Oxygen Recovery Assay (LORA) for <i>Mycobacterium tuberculosis</i></span>. In  <i>Mycobacteria Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, T.</span>, <span class="NLM_string-name">Roberts, D. M.</span></span>, Eds.; <span class="NLM_publisher-name">Springer New York</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">292</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1007%2F978-1-4939-2450-9_17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=281-292&author=S.+Cho&author=H.%0AS.+Lee&author=S.+Franzblauauthor=T.+Parish&author=D.+M.+Roberts&title=Mycobacteria+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2450-9_17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-2450-9_17%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DS.%26atitle%3DMicroplate%2520Alamar%2520Blue%2520Assay%2520%2528MABA%2529%2520and%2520Low%2520Oxygen%2520Recovery%2520Assay%2520%2528LORA%2529%2520for%2520Mycobacterium%2520tuberculosis%26btitle%3DMycobacteria%2520Protocols%26aulast%3DParish%26aufirst%3DT.%26pub%3DSpringer%2520New%2520York%26date%3D2015%26spage%3D281%26epage%3D292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blondin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serina, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmuller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barzu, O.</span></span> <span> </span><span class="NLM_article-title">Improved Spectrophotometric Assay of Nucleoside Monophosphate Kinase Activity Using the Pyruvate Kinase/Lactate Dehydrogenase Coupling System</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>220</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1006/abio.1994.1326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1006%2Fabio.1994.1326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=7978251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK2cXlslenu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=220&publication_year=1994&pages=219-221&author=C.+Blondinauthor=L.+Serinaauthor=L.+Wiesmullerauthor=A.+M.+Gillesauthor=O.+Barzu&title=Improved+Spectrophotometric+Assay+of+Nucleoside+Monophosphate+Kinase+Activity+Using+the+Pyruvate+Kinase%2FLactate+Dehydrogenase+Coupling+System&doi=10.1006%2Fabio.1994.1326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Improved spectrophotometric assay of nucleoside monophosphate kinase activity using the pyruvate kinase/lactate dehydrogenase coupling system</span></div><div class="casAuthors">Blondin, Catherine; Serina, Lidia; Wiesmueller, Lisa; Gilles, Anne-Marie; Barzu, Octavian</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">220</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-21</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">The title assay couples the formation of nucleoside diphosphates to the reaction catalyzed by pyruvate kinase and lactate dehydrogenase in the presence of phosphenolpyruvate and NADH as indicator.  The reactivity of various nucleoside diphosphates toward pyruvate kinase varies within 1-3 order of magnitude, and, hence, the coupling system may become rate limiting.  As a consequence, nucleoside mnophosphate kinase activity might by underestimate by a factor of 1 or 2.  However, in the presence of pure nucleoside diphosphate kinase, the 2:1 ration of NADH consumed/nucleoside monophosphate added is easily attained both for CMP and dCMP.  As nucleoside diphosphate kinase does not interfere with the activity of nucleoside monophosphate kinase activity in the presence of pyruvate kinase and nucleoside diphosphate kinase as coupling enzymes, this kinase was added to the assay mixt.  The reliability of the modified spectrophotometric assay in crude or pure preps. of enzyme was related to the rate of phosphorylation of the various nucleoside diphosphate products of the coupling system.  This rate was significantly increased by addn. of pure nucleoside diphosphate kinase, which might reduce the consumption of pyruvate kinase.  The overall cost of this improved procedure remained unchanged.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVctLrLvffA7Vg90H21EOLACvtfcHk0lgzVldMiAMPqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlslenu7w%253D&md5=fe31ee6e5636de62bb3f8218de8864d6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fabio.1994.1326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1994.1326%26sid%3Dliteratum%253Aachs%26aulast%3DBlondin%26aufirst%3DC.%26aulast%3DSerina%26aufirst%3DL.%26aulast%3DWiesmuller%26aufirst%3DL.%26aulast%3DGilles%26aufirst%3DA.%2BM.%26aulast%3DBarzu%26aufirst%3DO.%26atitle%3DImproved%2520Spectrophotometric%2520Assay%2520of%2520Nucleoside%2520Monophosphate%2520Kinase%2520Activity%2520Using%2520the%2520Pyruvate%2520Kinase%252FLactate%2520Dehydrogenase%2520Coupling%2520System%26jtitle%3DAnal.%2520Biochem.%26date%3D1994%26volume%3D220%26spage%3D219%26epage%3D221%26doi%3D10.1006%2Fabio.1994.1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cer, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudunuri, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeda, F. J.</span></span> <span> </span><span class="NLM_article-title">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">W441</span>– <span class="NLM_lpage">W445</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1093%2Fnar%2Fgkp253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=19395593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=W441-W445&author=R.+Z.+Cerauthor=U.+Mudunuriauthor=R.+Stephensauthor=F.+J.+Lebeda&title=IC50-to-Ki%3A+a+web-based+tool+for+converting+IC50+to+Ki+values+for+inhibitors+of+enzyme+activity+and+ligand+binding&doi=10.1093%2Fnar%2Fgkp253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span></div><div class="casAuthors">Cer, R. Z.; Mudunuri, U.; Stephens, R.; Lebeda, F. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W441-W445</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A new web-server tool ests. Ki values from exptl. detd. IC50 values for inhibitors of enzymes and of binding reactions between macromols. (e.g., proteins, nucleic acids) and ligands.  This converter was developed to enable end-users to help gauge the quality of the underlying assumptions used in these calcns. which depend on the type of mechanism of inhibitor action and the concns. of the interacting mol. species.  Addnl. calcns. are performed for nonclassical, tightly bound inhibitors of enzyme-substrate or of macromol.-ligand systems in which free, rather than total concns. of the reacting species are required.  Required user-defined input values include the total enzyme (or another target mol.) and substrate (or ligand) concns., the Km of the enzyme-substrate (or the Kd of the target-ligand) reaction, and the IC50 value.  Assumptions and caveats for these calcns. are discussed along with examples taken from the literature.  The host database for this converter contains kinetic consts. and other data for inhibitors of the proteolytic clostridial neurotoxins (http://botdb.abcc.ncifcrf.gov/toxin/kiConverter.jsp).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9gPXTwGaXwLVg90H21EOLACvtfcHk0lguopqeknwoIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D&md5=6b0ffe268d709ad8f80da6df29a9565d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp253%26sid%3Dliteratum%253Aachs%26aulast%3DCer%26aufirst%3DR.%2BZ.%26aulast%3DMudunuri%26aufirst%3DU.%26aulast%3DStephens%26aufirst%3DR.%26aulast%3DLebeda%26aufirst%3DF.%2BJ.%26atitle%3DIC50-to-Ki%253A%2520a%2520web-based%2520tool%2520for%2520converting%2520IC50%2520to%2520Ki%2520values%2520for%2520inhibitors%2520of%2520enzyme%2520activity%2520and%2520ligand%2520binding%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3DW441%26epage%3DW445%26doi%3D10.1093%2Fnar%2Fgkp253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the inhibition constant (KI) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. In  <i>Methods in Enzymology</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C. W.,  Jr.</span>, <span class="NLM_string-name">Sweet, R. M.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>: Macromolecular Crystallography, Part A, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.+W.+Carter&author=R.+M.+Sweet&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DCarter%26aufirst%3DC.%2BW.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Collaborative
Computational
Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4
Suite: Programs for Protein Crystallography</span>.  <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume">50</span> (),  <span class="NLM_fpage">760</span>.<span class="refDoi"> DOI: 10.1107/s0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760&author=Collaborative%0AComputational%0AProject%2C+Number+4&title=The+CCP4%0ASuite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2Fs0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2Fs0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%250ASuite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26doi%3D10.1107%2Fs0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: Building New Software for Automated Crystallographic Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/s0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+Building+New+Software+for+Automated+Crystallographic+Structure+Determination&doi=10.1107%2Fs0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lguopqeknwoIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2Fs0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520Building%2520New%2520Software%2520for%2520Automated%2520Crystallographic%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2Fs0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhydLrqtgJOOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera?A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%3FA+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lhydLrqtgJOOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%253FA%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span> <i>PyMOL The PyMOL
Molecular Graphics System</i>, version
1.8; <span class="NLM_publisher-name">Schrödinger,
LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PyMOL+The+PyMOL%0AMolecular+Graphics+System%2C+version%0A1.8%3B+Schr%C3%B6dinger%2C%0ALLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPyMOL%2520The%2520PyMOL%250AMolecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM?A new method for protein modeling and design: Applications to docking and structure prediction from the distorted native conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%3FA+new+method+for+protein+modeling+and+design%3A+Applications+to+docking+and+structure+prediction+from+the+distorted+native+conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lhydLrqtgJOOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%253FA%2520new%2520method%2520for%2520protein%2520modeling%2520and%2520design%253A%2520Applications%2520to%2520docking%2520and%2520structure%2520prediction%2520from%2520the%2520distorted%2520native%2520conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franzblau, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzig, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madico, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degnan, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilman, R. H.</span></span> <span> </span><span class="NLM_article-title">Rapid, Low-Technology MIC Determination with Clinical <i>Mycobacterium tuberculosis</i> Isolates by Using the Microplate Alamar Blue Assay</span>. <i>J. Clin. Microbiol</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">366</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=9466742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK1cXnsFCqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1998&pages=362-366&author=S.+G.+Franzblauauthor=R.+S.+Witzigauthor=J.+C.+McLaughlinauthor=P.+Torresauthor=G.+Madicoauthor=A.+Hernandezauthor=M.+T.+Degnanauthor=M.+B.+Cookauthor=V.+K.+Quenzerauthor=R.+M.+Fergusonauthor=R.+H.+Gilman&title=Rapid%2C+Low-Technology+MIC+Determination+with+Clinical+Mycobacterium+tuberculosis+Isolates+by+Using+the+Microplate+Alamar+Blue+Assay"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid, low-technology MIC determination with clinical Mycobacterium tuberculosis isolates by using the microplate Alamar Blue assay</span></div><div class="casAuthors">Franzblau, Scott G.; Witzig, Richard S.; Mclaughlin, James C.; Torres, Patricia; Madico, Guillermo; Hernandez, Antonio; Degnan, Michelle T.; Cook, Mary B.; Quenzer, Virginia K.; Ferguson, Robert M.; Gilman, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Microbiology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">362-366</span>CODEN:
                <span class="NLM_cas:coden">JCMIDW</span>;
        ISSN:<span class="NLM_cas:issn">0095-1137</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A colorimetric, microplate-based Alamar Blue assay (MABA) method was used to det. the MICs of isoniazid (INH), rifampin, streptomycin (SM), and ethambutol (EMB) for 34 Peruvian Mycobacterium tuberculosis isolates (including both pansensitive and multidrug-resistant strains) and the H37Rv strain by using bacterial suspensions prepd. directly from solid media.  Results for all isolates were available within 8 days.  Discordant results were obsd. on initial tests for 3 of 16 INH-susceptible isolates, 5 of 31 EMB-susceptible isolates, and 2 of 4 SM-resistant isolates (by the BACTEC 460 system).  The overall agreements between the MICs obtained by MABA and the results obtained with the BACTEC 460 system were 87.9% for initial results and 93.6% after retesting 12 of 17 samples with discrepant results.  Interpretation of MABA endpoints improved with tech. experience.  The MABA is a simple, rapid, low-cost, appropriate technol. which does not require expensive instrumentation and which makes use of a nontoxic, temp.-stable reagent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrofNKIH7Wtx7Vg90H21EOLACvtfcHk0lic3Z7uD2RPSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnsFCqug%253D%253D&md5=a3889d3ace22326dd2f7fffc8dd858c4</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFranzblau%26aufirst%3DS.%2BG.%26aulast%3DWitzig%26aufirst%3DR.%2BS.%26aulast%3DMcLaughlin%26aufirst%3DJ.%2BC.%26aulast%3DTorres%26aufirst%3DP.%26aulast%3DMadico%26aufirst%3DG.%26aulast%3DHernandez%26aufirst%3DA.%26aulast%3DDegnan%26aufirst%3DM.%2BT.%26aulast%3DCook%26aufirst%3DM.%2BB.%26aulast%3DQuenzer%26aufirst%3DV.%2BK.%26aulast%3DFerguson%26aufirst%3DR.%2BM.%26aulast%3DGilman%26aufirst%3DR.%2BH.%26atitle%3DRapid%252C%2520Low-Technology%2520MIC%2520Determination%2520with%2520Clinical%2520Mycobacterium%2520tuberculosis%2520Isolates%2520by%2520Using%2520the%2520Microplate%2520Alamar%2520Blue%2520Assay%26jtitle%3DJ.%2520Clin.%2520Microbiol%26date%3D1998%26volume%3D36%26spage%3D362%26epage%3D366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krieger, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freundlich, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawandi, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawandi, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huss, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavandera, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Discovery of Phenyl-diketo Acids as Potent Inhibitors of M. tuberculosis Malate Synthase</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1556</span>– <span class="NLM_lpage">1567</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.chembiol.2012.09.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=23261599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1556-1567&author=I.+V.+Kriegerauthor=J.+S.+Freundlichauthor=V.+B.+Gawandiauthor=J.+P.+Robertsauthor=V.+B.+Gawandiauthor=Q.+Sunauthor=J.+L.+Owenauthor=M.+T.+Fraileauthor=S.+I.+Hussauthor=J.-L.+Lavanderaauthor=T.+R.+Ioergerauthor=J.+C.+Sacchettini&title=Structure-Guided+Discovery+of+Phenyl-diketo+Acids+as+Potent+Inhibitors+of+M.+tuberculosis+Malate+Synthase&doi=10.1016%2Fj.chembiol.2012.09.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Discovery of Phenyl-diketo Acids as Potent Inhibitors of M. tuberculosis Malate Synthase</span></div><div class="casAuthors">Krieger, Inna V.; Freundlich, Joel S.; Gawandi, Vijay B.; Roberts, Justin P.; Gawandi, Vidyadhar B.; Sun, Qingan; Owen, Joshua L.; Fraile, Maria T.; Huss, Sofia I.; Lavandera, Jose-Luis; Ioerger, Thomas R.; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1556-1567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The glyoxylate shunt plays an important role in fatty acid metab. and has been shown to be crit. to survival of several pathogens involved in chronic infections.  For Mycobacterium tuberculosis (Mtb), a strain with a defective glyoxylate shunt was previously shown to be unable to establish infection in a mouse model.  We report the development of phenyl-diketo acid (PDKA) inhibitors of malate synthase (GlcB), one of two glyoxylate shunt enzymes, using structure-based methods.  PDKA inhibitors were active against Mtb grown on acetate, and overexpression of GlcB ameliorated this inhibition.  Crystal structures of complexes of GlcB with PDKA inhibitors guided optimization of potency.  A selected PDKA compd. demonstrated efficacy in a mouse model of tuberculosis.  The discovery of these PDKA derivs. provides chem. validation of GlcB as an attractive target for tuberculosis therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppNZdVaSD2jrVg90H21EOLACvtfcHk0lic3Z7uD2RPSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymur%252FN&md5=26fe3172f027bc08f49e7cbe63db4775</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DKrieger%26aufirst%3DI.%2BV.%26aulast%3DFreundlich%26aufirst%3DJ.%2BS.%26aulast%3DGawandi%26aufirst%3DV.%2BB.%26aulast%3DRoberts%26aufirst%3DJ.%2BP.%26aulast%3DGawandi%26aufirst%3DV.%2BB.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DOwen%26aufirst%3DJ.%2BL.%26aulast%3DFraile%26aufirst%3DM.%2BT.%26aulast%3DHuss%26aufirst%3DS.%2BI.%26aulast%3DLavandera%26aufirst%3DJ.-L.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DStructure-Guided%2520Discovery%2520of%2520Phenyl-diketo%2520Acids%2520as%2520Potent%2520Inhibitors%2520of%2520M.%2520tuberculosis%2520Malate%2520Synthase%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1556%26epage%3D1567%26doi%3D10.1016%2Fj.chembiol.2012.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 6 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhouling Xie, Xiaoxiao Yang, Yajun Duan, Jihong Han, <span class="NLM_string-name hlFld-ContribAuthor">Chenzhong Liao</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Kinase Inhibitors for the Treatment of Nononcologic Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (3)
                                     , 1283-1345. <a href="https://doi.org/10.1021/acs.jmedchem.0c01511" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01511</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01511%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BKinase%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BNononcologic%252BDiseases%26aulast%3DXie%26aufirst%3DZhouling%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29082020%26date%3D22012021%26volume%3D64%26issue%3D3%26spage%3D1283%26epage%3D1345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Srihari Konduri, Dodda Bhargavi, Jyothi Prashanth, Vagolu Siva Krishna, Dharmarajan Sriram, <span class="NLM_string-name hlFld-ContribAuthor">Koya Prabhakara Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Design and Synthesis of “Chloropicolinate Amides and Urea Derivatives” as Novel Inhibitors for Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (2)
                                     , 1657-1667. <a href="https://doi.org/10.1021/acsomega.0c05690" title="DOI URL">https://doi.org/10.1021/acsomega.0c05690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c05690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c05690%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDesign%252Band%252BSynthesis%252Bof%252B%2525E2%252580%25259CChloropicolinate%252BAmides%252Band%252BUrea%252BDerivatives%2525E2%252580%25259D%252Bas%252BNovel%252BInhibitors%252Bfor%252BMycobacterium%252Btuberculosis%26aulast%3DKonduri%26aufirst%3DSrihari%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D23112020%26date%3D28122020%26date%3D07012021%26volume%3D6%26issue%3D2%26spage%3D1657%26epage%3D1667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Peng-Fei Zhao, An Liu, Ming-Guang Wei, <span class="NLM_string-name hlFld-ContribAuthor">Zai-Qun Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Construction of 3D Antioxidants with Nucleosides as the Core: Inhibition of DNA Oxidation. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2019,</strong> <em>84 </em>
                                    (24)
                                     , 15854-15864. <a href="https://doi.org/10.1021/acs.joc.9b02104" title="DOI URL">https://doi.org/10.1021/acs.joc.9b02104</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b02104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b02104%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DConstruction%252Bof%252B3D%252BAntioxidants%252Bwith%252BNucleosides%252Bas%252Bthe%252BCore%25253A%252BInhibition%252Bof%252BDNA%252BOxidation%26aulast%3DZhao%26aufirst%3DPeng-Fei%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D07082019%26date%3D05122019%26volume%3D84%26issue%3D24%26spage%3D15854%26epage%3D15864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Srihari  Konduri</span>, <span class="hlFld-ContribAuthor ">Jyothi  Prashanth</span>, <span class="hlFld-ContribAuthor ">Vagolu Siva  Krishna</span>, <span class="hlFld-ContribAuthor ">Dharmarajan  Sriram</span>, <span class="hlFld-ContribAuthor ">J.N.  Behera</span>, <span class="hlFld-ContribAuthor ">Dionicio  Siegel</span>, <span class="hlFld-ContribAuthor ">Koya Prabhakara  Rao</span>. </span><span class="cited-content_cbyCitation_article-title">Design and synthesis of purine connected piperazine derivatives as novel inhibitors of Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (22)
                                     , 127512. <a href="https://doi.org/10.1016/j.bmcl.2020.127512" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127512</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127512&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127512%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%252Band%252Bsynthesis%252Bof%252Bpurine%252Bconnected%252Bpiperazine%252Bderivatives%252Bas%252Bnovel%252Binhibitors%252Bof%252BMycobacterium%252Btuberculosis%26aulast%3DKonduri%26aufirst%3DSrihari%26date%3D2020%26volume%3D30%26issue%3D22%26spage%3D127512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zuzana  Konyariková</span>, <span class="hlFld-ContribAuthor ">Karin  Savková</span>, <span class="hlFld-ContribAuthor ">Stanislav  Kozmon</span>, <span class="hlFld-ContribAuthor ">Katarína  Mikušová</span>. </span><span class="cited-content_cbyCitation_article-title">Biosynthesis of Galactan in Mycobacterium tuberculosis as a Viable TB Drug Target?. </span><span class="cited-content_cbyCitation_journal-name">Antibiotics</span><span> <strong>2020,</strong> <em>9 </em>
                                    (1)
                                     , 20. <a href="https://doi.org/10.3390/antibiotics9010020" title="DOI URL">https://doi.org/10.3390/antibiotics9010020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/antibiotics9010020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fantibiotics9010020%26sid%3Dliteratum%253Aachs%26jtitle%3DAntibiotics%26atitle%3DBiosynthesis%252Bof%252BGalactan%252Bin%252BMycobacterium%252Btuberculosis%252Bas%252Ba%252BViable%252BTB%252BDrug%252BTarget%25253F%26aulast%3DKonyarikov%25C3%25A1%26aufirst%3DZuzana%26date%3D2020%26date%3D2020%26volume%3D9%26issue%3D1%26spage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kai  Song</span>, <span class="hlFld-ContribAuthor ">Yu  Li</span>, <span class="hlFld-ContribAuthor ">Huawei  He</span>, <span class="hlFld-ContribAuthor ">Lina  Liu</span>, <span class="hlFld-ContribAuthor ">Ping  Zhao</span>, <span class="hlFld-ContribAuthor ">Qingyou  Xia</span>, <span class="hlFld-ContribAuthor ">Yejing  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">A Novel Adenosine Kinase from Bombyx mori: Enzymatic Activity, Structure, and Biological Function. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (15)
                                     , 3732. <a href="https://doi.org/10.3390/ijms20153732" title="DOI URL">https://doi.org/10.3390/ijms20153732</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20153732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20153732%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DA%252BNovel%252BAdenosine%252BKinase%252Bfrom%252BBombyx%252Bmori%25253A%252BEnzymatic%252BActivity%25252C%252BStructure%25252C%252Band%252BBiological%252BFunction%26aulast%3DSong%26aufirst%3DKai%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D15%26spage%3D3732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Positions of interest for structure-guided drug design and tool compounds utilized in crystallography studies. (a) Adenosine bound to the active site of MtbAdoK (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PKM">2PKM</a>). (b) Tool compounds used to explore the chemical space surrounding the positions highlighted in (a).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure complexes of compounds <b>2–5</b>. (a) <b>2</b> bound to the active site of MtbAdoK. (b) <b>3</b> bound to the active site of MtbAdoK. (c,d) N7-substituents of compounds <b>2–3</b> are buried in an active site pocket composed of residues from chain A and chain B. (e) Chimney-like cavity observed above position N6 formed by residues from chain A and chain B. (f) <b>4</b> bound to the active site of MtbAdoK. (g) Methylmercapto group of <b>4</b> is accommodated in a compound-induced pocket formed by the movement of Arg176 (red). (h) Chimney-like cavity observed in the cocrystal structures of <b>2</b> and <b>3</b> is blocked by the conformational change of Arg176 (red). (i) <b>5</b> bound to the active site of MtbAdoK. In all cases, chain A residues are colored by heteroatom and chain B residues are color magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structure of the MtbAdoK-<b>6</b> complex. (a) Chemical structure of compound <b>6</b>. (b) <b>6</b> bound to the active site of MtbAdoK. (c) Superimposition of the cocrystal structures of compound <b>6</b> (gray) and adenosine (blue). (d) Surface representation of residues forming the pocket where the thiophene group gets accommodated. Chain B is colored magenta, Arg176 is colored red, and chain A is colored by heteroatom. (e) Arg176 adopts a different conformation in the cocrystal structure of <b>4</b> (blue) when compared with the cocrystal structure of <b>6</b> (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Adenosine Analogues<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) TBSCl, imidazole, DCM, 15 °C, 17 h, 99%; (b) 1-phenylpiperazine or 1-([1,1′-biphenyl]-4-yl)piperazine or 1-(4-bromophenyl)piperazine, DIEA, EtOH, 70–80 °C, 17 h, 13–90%; (c) ArB(OH)<sub>2</sub>, K<sub>3</sub>PO<sub>4</sub>, XPhos Pd G2, THF/H<sub>2</sub>O, 70 °C, 17 h, 49–87%; (d) TFA, THF/H<sub>2</sub>O, 15–25 °C, 2–17 h, 13–67%; (e) 4,4,5,5-tetramethyl-2-(4-(phenylethynyl)phenyl)-1,3,2-dioxaborolane, K<sub>3</sub>PO<sub>4</sub>, XPhos Pd G2, THF, 70 °C, 12 h, 95% or 4-ethynyl-1,1′-biphenyl, Cs<sub>2</sub>CO<sub>3</sub>, CuI, XPhos Pd G2, CH<sub>3</sub>CN, 90 °C, 17 h, 27%; (f) 1-([1,1′-biphenyl]-4-yl)piperazine, Cs<sub>2</sub>CO<sub>3</sub>, 1-([1,1′-biphenyl]-4-yl)piperazine, RuPhos Pd G2, <i>tert</i>-amyl-OH, 100 °C, 17 h, 50%; and (g) NH<sub>4</sub>OH, MeOH, 15 °C, 24 h, 21%.</p></p></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structure of the MtbAdoK-7 complex. (a) 7 bound to the active site to MtbAdoK. Chain A is colored gray and chain B is colored magenta. (b) The bulky substituent of 7 (gray) forces the compound to bind in a different orientation with respect to adenosine (blue; PDB ID 2PKM). (c) The bulky substituent forces Arg176 to open the cavity while the distal benzene ring is reoriented back into the active site groove that is formed by residues of chain A (heteroatom) and chain B (magenta).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Summary of SAR results for compounds <b>8–17</b>. Compound <b>7</b> bound to the active site of MtbAdoK. Based on the MtbAdoK-<b>7</b> complex, compounds <b>8–17</b> where synthesized. Chain A is colored white, and chain B is colored magenta.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Crystal structure of the MtbAdoK-<b>17</b> complex. (a) <b>17</b> bound to the active site of MtbAdoK. (b) The large bulky substitution gets accommodated in the “chimney-like” cavity. (c) Compound <b>17</b> (gray) binds in a different orientation with respect to adenosine (blue). (d) Superimposition of crystal structure complexes of <b>17</b> (gray) and <b>7</b> (blue). Chain B residues forming the distal part of the “chimney-like” cavity colored magenta while chain A residues are colored by heteroatom.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/medium/jm-2019-00020r_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Preliminary pharmacokinetics of compound <b>18</b>. (a) Oral PK study conducted at 10 mg·kg<sup>–1</sup> utilizing 9 Swiss Webster Female mice with three animal bleeds per time point. Maximum plasma concentration of 0.13 ± 0.02 μg·mL<sup>–1</sup> observed at 4 h following gavage. The AUC<sub>0–24h</sub> of 1.8 μg·h·mL<sup>–1</sup> was calculated using the linear trapezoid rule method. (b) Snapshot PK studies conducted from plasma samples taken from tolerability mice at 50, 100, and 200 mg·kg<sup>–1</sup> following first initial dose. 10 mg·kg<sup>–1</sup> data included for comparison. (c) <i>C</i><sub>max</sub>, <i>t</i><sub>max</sub>, and AUC of compound <b>18</b> at four dose levels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-9/acs.jmedchem.9b00020/20190502/images/large/jm-2019-00020r_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00020&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i44">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15816" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15816" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">WHO</span>.  <i>Global Tuberculosis Report</i>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=WHO&title=Global+Tuberculosis+Report"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DGlobal%2520Tuberculosis%2520Report%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ducati, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, D. S.</span></span> <span> </span><span class="NLM_article-title">Purine Salvage Pathway in <i>Mycobacterium tuberculosis</i></span>. <i>Curr. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1258</span>– <span class="NLM_lpage">1275</span>, <span class="refDoi"> DOI: 10.2174/092986711795029627</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.2174%2F092986711795029627" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=21366536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOku78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1258-1275&author=R.+G.+Ducatiauthor=A.+Bredaauthor=L.+A.+Bassoauthor=D.+S.+Santos&title=Purine+Salvage+Pathway+in+Mycobacterium+tuberculosis&doi=10.2174%2F092986711795029627"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Purine salvage pathway in Mycobacterium tuberculosis</span></div><div class="casAuthors">Ducati, R. G.; Breda, A.; Basso, L. A.; Santos, D. S.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1258-1275</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Millions of deaths worldwide are caused by the etiol. agent of tuberculosis, Mycobacterium tuberculosis.  The increasing prevalence of this disease, the emergence of drug-resistant strains, and the devastating effect of human immunodeficiency virus coinfection have led to an urgent need for the development of new and more efficient antimycobacterial drugs.  The modern approach to the development of new chem. compds. against complex diseases, esp. the neglected endemic ones, such as tuberculosis, is based on the use of defined mol. targets.  Among the advantages, this approach allows (i) the search and identification of lead compds. with defined mol. mechanisms against a specific target (e.g. enzymes from defined pathways), (ii) the anal. of a great no. of compds. with a favorable cost/benefit ratio, and (iii) the development of compds. with selective toxicity.  The present review describes the enzymes of the purine salvage pathway in M. tuberculosis as attractive targets for the development of new antimycobacterial agents.  Enzyme kinetics and structural data were included to provide a thorough knowledge on which to base the search for compds. with biol. activity.  The authors have focused on the mycobacterial homologues of this pathway as potential targets for the development of new antitubercular agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBhKqUvM--d7Vg90H21EOLACvtfcHk0ljrk7vXARNb7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOku78%253D&md5=6da82ca0d8c848913ee251c6c66ed7d6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.2174%2F092986711795029627&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711795029627%26sid%3Dliteratum%253Aachs%26aulast%3DDucati%26aufirst%3DR.%2BG.%26aulast%3DBreda%26aufirst%3DA.%26aulast%3DBasso%26aufirst%3DL.%2BA.%26aulast%3DSantos%26aufirst%3DD.%2BS.%26atitle%3DPurine%2520Salvage%2520Pathway%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DCurr.%2520Med.%2520Chem%26date%3D2011%26volume%3D18%26spage%3D1258%26epage%3D1275%26doi%3D10.2174%2F092986711795029627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villela, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Quitian, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducati, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, L. A.</span></span> <span> </span><span class="NLM_article-title">Pyrimidine Salvage Pathway in <i>Mycobacterium tuberculosis</i></span>. <i>Curr. Med. Chem</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1286</span>– <span class="NLM_lpage">1298</span>, <span class="refDoi"> DOI: 10.2174/092986711795029555</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.2174%2F092986711795029555" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=21366534" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVOku70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1286-1298&author=A.+D.+Villelaauthor=Z.+A.+Sanchez-Quitianauthor=R.+G.+Ducatiauthor=D.+S.+Santosauthor=L.+A.+Basso&title=Pyrimidine+Salvage+Pathway+in+Mycobacterium+tuberculosis&doi=10.2174%2F092986711795029555"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrimidine salvage pathway in Mycobacterium tuberculosis</span></div><div class="casAuthors">Villela, A. D.; Sanchez-Quitian, Z. A.; Ducati, R. G.; Santos, D. S.; Basso, L. A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1286-1298</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  The causative agent of tuberculosis (TB), Mycobacterium tuberculosis, infects one-third of the world population.  TB remains the leading cause of mortality due to a single bacterial pathogen.  The worldwide increase in incidence of M. tuberculosis has been attributed to the high proliferation rates of multi and extensively drug-resistant strains, and to co-infection with the human immunodeficiency virus.  There is thus a continuous requirement for studies on mycobacterial metab. to identify promising targets for the development of new agents to combat TB.  Singular characteristics of this pathogen, such as functional and structural features of enzymes involved in fundamental metabolic pathways, can be evaluated to identify possible targets for drug development.  Enzymes involved in the pyrimidine salvage pathway might be attractive targets for rational drug design against TB, since this pathway is vital for all bacterial cells, and is composed of enzymes considerably different from those present in humans.  Moreover, the enzymes of the pyrimidine salvage pathway might have an important role in the mycobacterial latent state, since M. tuberculosis has to recycle bases and/or nucleosides to survive in the hostile environment imposed by the host.  The present review describes the enzymes of M. tuberculosis pyrimidine salvage pathway as attractive targets for the development of new antimycobacterial agents.  Enzyme functional and structural data have been included to provide a broader knowledge on which to base the search for compds. with selective biol. activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmR80dsbdfTrVg90H21EOLACvtfcHk0lj27z4TlVbYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVOku70%253D&md5=00549ba222d7e08ce8cfd5a474c6cd86</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F092986711795029555&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986711795029555%26sid%3Dliteratum%253Aachs%26aulast%3DVillela%26aufirst%3DA.%2BD.%26aulast%3DSanchez-Quitian%26aufirst%3DZ.%2BA.%26aulast%3DDucati%26aufirst%3DR.%2BG.%26aulast%3DSantos%26aufirst%3DD.%2BS.%26aulast%3DBasso%26aufirst%3DL.%2BA.%26atitle%3DPyrimidine%2520Salvage%2520Pathway%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DCurr.%2520Med.%2520Chem%26date%3D2011%26volume%3D18%26spage%3D1286%26epage%3D1298%26doi%3D10.2174%2F092986711795029555" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escuyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of a Unique Adenosine Kinase from <i>Mycobacterium tuberculosis</i></span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>185</i></span>,  <span class="NLM_fpage">6548</span>– <span class="NLM_lpage">6555</span>, <span class="refDoi"> DOI: 10.1128/jb.185.22.6548-6555.2003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1128%2Fjb.185.22.6548-6555.2003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=14594827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFKht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=2003&pages=6548-6555&author=M.+C.+Longauthor=V.+Escuyerauthor=W.+B.+Parker&title=Identification+and+Characterization+of+a+Unique+Adenosine+Kinase+from+Mycobacterium+tuberculosis&doi=10.1128%2Fjb.185.22.6548-6555.2003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of a unique adenosine kinase from Mycobacterium tuberculosis</span></div><div class="casAuthors">Long, Mary C.; Escuyer, Vincent; Parker, William B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">185</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6548-6555</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Adenosine kinase (AK) is a purine salvage enzyme that catalyzes the phosphorylation of adenosine to AMP.  In Mycobacterium tuberculosis, AK can also catalyze the phosphorylation of the adenosine analog 2-methyladenosine (methyl-Ado), the first step in the metab. of this compd. to an active form.  Purifn. of AK from M. tuberculosis yielded a 35-kDa protein that existed as a dimer in its native form.  Adenosine (Ado) was preferred as a substrate at least 30-fold (Km = 0.8±0.08 μM) over other natural nucleosides, and substrate inhibition was obsd. when Ado concns. exceeded 5 μM.  M. tuberculosis and human AKs exhibited different affinities for methyl-Ado, with Km values of 79 and 960 μM, resp., indicating that differences exist between the substrate binding sites of these enzymes.  ATP was a good phosphate donor (Km = 1100±140 μM); however, the activity levels obsd. with dGTP and GTP were 4.7 and 2.5 times the levels obsd. with ATP, resp.  M. tuberculosis AK activity was dependent on Mg2+, and activity was stimulated by potassium, as reflected by a decrease in the Km and an increase in Vmax for both Ado and methyl-Ado.  The N-terminal amino acid sequence of the purified enzyme revealed complete identity with Rv2202c, a protein currently classified as a hypothetical sugar kinase.  When an AK-deficient strain of M. tuberculosis (SRICK1) was transformed with this gene, it exhibited a 5,000-fold increase in AK activity compared to exts. from the original mutants.  These results verified that the protein that we identified as AK was coded for by Rv2202c.  AK is not commonly found in bacteria, and to the best of our knowledge, M. tuberculosis AK is the first bacterial AK to be characterized.  The enzyme shows greater sequence homol. with ribokinase and fructokinase than it does with other AKs.  The multiple differences that exist between M. tuberculosis and human AKs may provide the mol. basis for the development of nucleoside analog compds. with selective activity against M. tuberculosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLmpzZ1gIxfbVg90H21EOLACvtfcHk0lj27z4TlVbYZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFKht7c%253D&md5=fd763397fd814d011383ed3b45f08043</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2Fjb.185.22.6548-6555.2003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.185.22.6548-6555.2003%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DEscuyer%26aufirst%3DV.%26aulast%3DParker%26aufirst%3DW.%2BB.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520a%2520Unique%2520Adenosine%2520Kinase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DJ.%2520Bacteriol.%26date%3D2003%26volume%3D185%26spage%3D6548%26epage%3D6555%26doi%3D10.1128%2Fjb.185.22.6548-6555.2003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palaninathan, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shetty, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">High Resolution Crystal Structures ofMycobacterium tuberculosisAdenosine Kinase</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">27334</span>– <span class="NLM_lpage">27342</span>, <span class="refDoi"> DOI: 10.1074/jbc.m703290200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1074%2Fjbc.m703290200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=27334-27342&author=M.+C.+M.+Reddyauthor=S.+K.+Palaninathanauthor=N.+D.+Shettyauthor=J.+L.+Owenauthor=M.+D.+Watsonauthor=J.+C.+Sacchettini&title=High+Resolution+Crystal+Structures+ofMycobacterium+tuberculosisAdenosine+Kinase&doi=10.1074%2Fjbc.m703290200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m703290200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m703290200%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DM.%2BC.%2BM.%26aulast%3DPalaninathan%26aufirst%3DS.%2BK.%26aulast%3DShetty%26aufirst%3DN.%2BD.%26aulast%3DOwen%26aufirst%3DJ.%2BL.%26aulast%3DWatson%26aufirst%3DM.%2BD.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DHigh%2520Resolution%2520Crystal%2520Structures%2520ofMycobacterium%2520tuberculosisAdenosine%2520Kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D27334%26epage%3D27342%26doi%3D10.1074%2Fjbc.m703290200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R. S.</span></span> <span> </span><span class="NLM_article-title">Adenosine kinase and ribokinase - the RK family of proteins</span>. <i>Cell. Mol. Life Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">2875</span>– <span class="NLM_lpage">2896</span>, <span class="refDoi"> DOI: 10.1007/s00018-008-8123-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1007%2Fs00018-008-8123-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=18560757" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtFCqsr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2008&pages=2875-2896&author=J.+Parkauthor=R.+S.+Gupta&title=Adenosine+kinase+and+ribokinase+-+the+RK+family+of+proteins&doi=10.1007%2Fs00018-008-8123-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine kinase and ribokinase - the RK family of proteins</span></div><div class="casAuthors">Park, J.; Gupta, R. S.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2875-2896</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Ribokinase (RK) and adenosine kinase (AK) catalyze the phosphorylation of ribose and adenosine to ribose-5-phosphate and AMP, resp.  Belonging to the RK family of proteins, these enzymes share a no. of unique structural and functional elements.  Extensive work has been carried out on many aspects of these enzymes in recent years, and we summarize the wealth of information currently available on them.  The topics covered include descriptions of the primary and three-dimensional structures of AK and RK, their phylogenetic relationships, biochem. aspects of these enzymes including their reaction mechanisms and ionic requirements, and also work on certain inhibitors of these enzymes.  The cellular metab. and transport of ribose and adenosine are also briefly discussed, as well as the beneficial effects of ribose and adenosine in physiol. and how these effects can be harnessed for pharmacol. purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEqMyvD78pxbVg90H21EOLACvtfcHk0ljxnMbPioU9Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtFCqsr3M&md5=c26f7345c2beb6741ee7a0850c86a4c2</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2Fs00018-008-8123-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-008-8123-1%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DGupta%26aufirst%3DR.%2BS.%26atitle%3DAdenosine%2520kinase%2520and%2520ribokinase%2520-%2520the%2520RK%2520family%2520of%2520proteins%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2008%26volume%3D65%26spage%3D2875%26epage%3D2896%26doi%3D10.1007%2Fs00018-008-8123-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ealick, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roos, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, R. G.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Toxoplasma gondii Adenosine Kinase Reveal a Novel Catalytic Mechanism and Prodrug Binding</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">875</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1006/jmbi.2000.3753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1006%2Fjmbi.2000.3753" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2000&pages=875-893&author=M.+A.+Schumacherauthor=D.+M.+Scottauthor=I.+I.+Mathewsauthor=S.+E.+Ealickauthor=D.+S.+Roosauthor=B.+Ullmanauthor=R.+G.+Brennan&title=Crystal+Structures+of+Toxoplasma+gondii+Adenosine+Kinase+Reveal+a+Novel+Catalytic+Mechanism+and+Prodrug+Binding&doi=10.1006%2Fjmbi.2000.3753"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.2000.3753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.2000.3753%26sid%3Dliteratum%253Aachs%26aulast%3DSchumacher%26aufirst%3DM.%2BA.%26aulast%3DScott%26aufirst%3DD.%2BM.%26aulast%3DMathews%26aufirst%3DI.%2BI.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26aulast%3DRoos%26aufirst%3DD.%2BS.%26aulast%3DUllman%26aufirst%3DB.%26aulast%3DBrennan%26aufirst%3DR.%2BG.%26atitle%3DCrystal%2520Structures%2520of%2520Toxoplasma%2520gondii%2520Adenosine%2520Kinase%2520Reveal%2520a%2520Novel%2520Catalytic%2520Mechanism%2520and%2520Prodrug%2520Binding%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2000%26volume%3D298%26spage%3D875%26epage%3D893%26doi%3D10.1006%2Fjmbi.2000.3753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ealick, S. E.</span></span> <span> </span><span class="NLM_article-title">Structure of Human Adenosine Kinase at 1.5 Å Resolution†,‡</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">15607</span>– <span class="NLM_lpage">15620</span>, <span class="refDoi"> DOI: 10.1021/bi9815445</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi9815445" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK1cXmsleitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=15607-15620&author=I.+I.+Mathewsauthor=M.+D.+Erionauthor=S.+E.+Ealick&title=Structure+of+Human+Adenosine+Kinase+at+1.5+%C3%85+Resolution%E2%80%A0%2C%E2%80%A1&doi=10.1021%2Fbi9815445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of human adenosine kinase at 1.5 Å resolution</span></div><div class="casAuthors">Mathews, Irimpan I.; Erion, Mark D.; Ealick, Steven E.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">15607-15620</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adenosine kinase (I) is a key enzyme in the regulation of extracellular adenosine and intracellular adenylate levels.  Inhibitors of I elevate adenosine to levels that activate nearby adenosine receptors and produce a wide variety of therapeutically beneficial activities.  Accordingly, I is a promising target for new analgesic, neuroprotective, and cardioprotective agents.  Here, the authors detd. the structure of human I by x-ray crystallog. using MAD phasing techniques and refined the structure to 1.5 Å resoln.  The I structure consisted of 1 large α/β domain with 9 β-strands, 8 α-helixes, and 1 small α/β-domain with 5 β-strands and 2 α-helixes.  The active site was formed along the edge of the β-sheet in the large domain, whereas the small domain acted as a lid to cover the upper face of the active site.  The overall structure was similar to the recently reported structure of ribokinase from Escherichia coli.  The structure of ribokinase was detd. at 1.8 Å resoln. and represents the 1st structure of a new family of carbohydrate kinases.  Two mols. of adenosine were present in the I crystal structure with 1 adenosine mol. located in a site that matches the ribose site in ribokinase and probably represents the substrate-binding site.  The 2nd adenosine site overlapped the ADP site in ribokinase and probably represents the ATP site.  A Mg2+-binding site was obsd. in a trough between the 2 adenosine sites.  The structure of the active site was consistent with the obsd. substrate specificity.  The active-site model suggests that Asp-300 is an important catalytic residue involved in the deprotonation of the 5'-OH group during phosphate transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruB9B1HTkEJrVg90H21EOLACvtfcHk0ljxnMbPioU9Vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXmsleitbw%253D&md5=7aaad5a207883daea181c39e811e9af4</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fbi9815445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi9815445%26sid%3Dliteratum%253Aachs%26aulast%3DMathews%26aufirst%3DI.%2BI.%26aulast%3DErion%26aufirst%3DM.%2BD.%26aulast%3DEalick%26aufirst%3DS.%2BE.%26atitle%3DStructure%2520of%2520Human%2520Adenosine%2520Kinase%2520at%25201.5%2520%25C3%2585%2520Resolution%25E2%2580%25A0%252C%25E2%2580%25A1%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D15607%26epage%3D15620%26doi%3D10.1021%2Fbi9815445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Romanello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bachega, J. F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassago, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandão-Neto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarco, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garratt, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, H. D. M.</span></span> <span> </span><span class="NLM_article-title">Adenosine kinase fromSchistosoma mansoni: structural basis for the differential incorporation of nucleoside analogues</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">126</span>– <span class="NLM_lpage">136</span>, <span class="refDoi"> DOI: 10.1107/s0907444912044800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444912044800" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=126-136&author=L.+Romanelloauthor=J.+F.+R.+Bachegaauthor=A.+Cassagoauthor=J.+Brand%C3%A3o-Netoauthor=R.+DeMarcoauthor=R.+C.+Garrattauthor=H.+D.+M.+Pereira&title=Adenosine+kinase+fromSchistosoma+mansoni%3A+structural+basis+for+the+differential+incorporation+of+nucleoside+analogues&doi=10.1107%2Fs0907444912044800"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1107%2Fs0907444912044800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444912044800%26sid%3Dliteratum%253Aachs%26aulast%3DRomanello%26aufirst%3DL.%26aulast%3DBachega%26aufirst%3DJ.%2BF.%2BR.%26aulast%3DCassago%26aufirst%3DA.%26aulast%3DBrand%25C3%25A3o-Neto%26aufirst%3DJ.%26aulast%3DDeMarco%26aufirst%3DR.%26aulast%3DGarratt%26aufirst%3DR.%2BC.%26aulast%3DPereira%26aufirst%3DH.%2BD.%2BM.%26atitle%3DAdenosine%2520kinase%2520fromSchistosoma%2520mansoni%253A%2520structural%2520basis%2520for%2520the%2520differential%2520incorporation%2520of%2520nucleoside%2520analogues%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D126%26epage%3D136%26doi%3D10.1107%2Fs0907444912044800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawronski, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeJesus, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akerley, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassetti, C. M.</span></span> <span> </span><span class="NLM_article-title">High-Resolution Phenotypic Profiling Defines Genes Essential for Mycobacterial Growth and Cholesterol Catabolism</span>. <i>PLoS Pathog.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e1002251</span> <span class="refDoi"> DOI: 10.1371/journal.ppat.1002251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1371%2Fjournal.ppat.1002251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=21980284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCmsbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&author=J.+E.+Griffinauthor=J.+D.+Gawronskiauthor=M.+A.+DeJesusauthor=T.+R.+Ioergerauthor=B.+J.+Akerleyauthor=C.+M.+Sassetti&title=High-Resolution+Phenotypic+Profiling+Defines+Genes+Essential+for+Mycobacterial+Growth+and+Cholesterol+Catabolism&doi=10.1371%2Fjournal.ppat.1002251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution phenotypic profiling defines genes essential for mycobacterial growth and cholesterol catabolism</span></div><div class="casAuthors">Griffin, Jennifer E.; Gawronski, Jeffrey D.; DeJesus, Michael A.; Ioerger, Thomas R.; Akerley, Brian J.; Sassetti, Christopher M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Pathogens</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1002251</span>CODEN:
                <span class="NLM_cas:coden">PPLACN</span>;
        ISSN:<span class="NLM_cas:issn">1553-7374</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The pathways that comprise cellular metab. are highly interconnected, and alterations in individual enzymes can have far-reaching effects.  As a result, global profiling methods that measure gene expression are of limited value in predicting how the loss of an individual function will affect the cell.  In this work, we employed a new method of global phenotypic profiling to directly define the genes required for the growth of Mycobacterium tuberculosis.  A combination of high-d. mutagenesis and deep-sequencing was used to characterize the compn. of complex mutant libraries exposed to different conditions.  This allowed the unambiguous identification of the genes that are essential for Mtb to grow in vitro, and proved to be a significant improvement over previous approaches.  To further explore functions that are required for persistence in the host, we defined the pathways necessary for the utilization of cholesterol, a crit. carbon source during infection.  Few of the genes we identified had previously been implicated in this adaptation by transcriptional profiling, and only a fraction were encoded in the chromosomal region known to encode sterol catabolic functions.  These genes comprise an unexpectedly large percentage of those previously shown to be required for bacterial growth in mouse tissue.  Thus, this single nutritional change accounts for a significant fraction of the adaptation to the host.  This work provides the most comprehensive genetic characterization of a sterol catabolic pathway to date, suggests putative roles for uncharacterized virulence genes, and precisely maps genes encoding potential drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZFTpTTOIkA7Vg90H21EOLACvtfcHk0lglSuqZnM4pzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCmsbfN&md5=a97c2b9759b7a8bc9a304f5684701f13</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1371%2Fjournal.ppat.1002251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.ppat.1002251%26sid%3Dliteratum%253Aachs%26aulast%3DGriffin%26aufirst%3DJ.%2BE.%26aulast%3DGawronski%26aufirst%3DJ.%2BD.%26aulast%3DDeJesus%26aufirst%3DM.%2BA.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DAkerley%26aufirst%3DB.%2BJ.%26aulast%3DSassetti%26aufirst%3DC.%2BM.%26atitle%3DHigh-Resolution%2520Phenotypic%2520Profiling%2520Defines%2520Genes%2520Essential%2520for%2520Mycobacterial%2520Growth%2520and%2520Cholesterol%2520Catabolism%26jtitle%3DPLoS%2520Pathog.%26date%3D2011%26volume%3D7%26doi%3D10.1371%2Fjournal.ppat.1002251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rothchild, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dartois, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuster, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trauner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wallis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galaviz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huttenhower, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, E. J.</span></span> <span> </span><span class="NLM_article-title">Tryptophan Biosynthesis Protects Mycobacteria from CD4 T-Cell-Mediated Killing</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">1296</span>– <span class="NLM_lpage">1308</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.10.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.cell.2013.10.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=24315099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFylurzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2013&pages=1296-1308&author=Y.+J.+Zhangauthor=M.+C.+Reddyauthor=T.+R.+Ioergerauthor=A.+C.+Rothchildauthor=V.+Dartoisauthor=B.+M.+Schusterauthor=A.+Traunerauthor=D.+Wallisauthor=S.+Galavizauthor=C.+Huttenhowerauthor=J.+C.+Sacchettiniauthor=S.+M.+Beharauthor=E.+J.+Rubin&title=Tryptophan+Biosynthesis+Protects+Mycobacteria+from+CD4+T-Cell-Mediated+Killing&doi=10.1016%2Fj.cell.2013.10.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Tryptophan biosynthesis protects mycobacteria from CD4 T-cell-mediated killing</span></div><div class="casAuthors">Zhang, Yanjia J.; Reddy, Manchi C.; Ioerger, Thomas R.; Rothchild, Alissa C.; Dartois, Veronique; Schuster, Brian M.; Trauner, Andrej; Wallis, Deeann; Galaviz, Stacy; Huttenhower, Curtis; Sacchettini, James C.; Behar, Samuel M.; Rubin, Eric J.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1296-1308</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bacteria that cause disease rely on their ability to counteract and overcome host defenses.  Here, we present a genome-scale study of Mycobacterium tuberculosis (Mtb) that uncovers the bacterial determinants of surviving host immunity, sets of genes we term "counteractomes.".  Through this anal., we found that CD4 T cells attempt to contain Mtb growth by starving it of tryptophan - a mechanism that successfully limits infections by Chlamydia and Leishmania, natural tryptophan auxotrophs.  Mtb, however, can synthesize tryptophan under stress conditions, and thus, starvation fails as an Mtb-killing mechanism.  We then identify a small-mol. inhibitor of Mtb tryptophan synthesis, which converts Mtb into a tryptophan auxotroph and restores the efficacy of a failed host defense.  Together, our findings demonstrate that the Mtb immune counteractomes serve as probes of host immunity, uncovering immune-mediated stresses that can be leveraged for therapeutic discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkCUFDhESMVrVg90H21EOLACvtfcHk0lglSuqZnM4pzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFylurzM&md5=6947753d2524b4f7b352ddc94249317f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.10.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.10.045%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%2BJ.%26aulast%3DReddy%26aufirst%3DM.%2BC.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DRothchild%26aufirst%3DA.%2BC.%26aulast%3DDartois%26aufirst%3DV.%26aulast%3DSchuster%26aufirst%3DB.%2BM.%26aulast%3DTrauner%26aufirst%3DA.%26aulast%3DWallis%26aufirst%3DD.%26aulast%3DGalaviz%26aufirst%3DS.%26aulast%3DHuttenhower%26aufirst%3DC.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DBehar%26aufirst%3DS.%2BM.%26aulast%3DRubin%26aufirst%3DE.%2BJ.%26atitle%3DTryptophan%2520Biosynthesis%2520Protects%2520Mycobacteria%2520from%2520CD4%2520T-Cell-Mediated%2520Killing%26jtitle%3DCell%26date%3D2013%26volume%3D155%26spage%3D1296%26epage%3D1308%26doi%3D10.1016%2Fj.cell.2013.10.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship for nucleoside analogs as inhibitors or substrates of adenosine kinase from Mycobacterium tuberculosis</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1671</span>– <span class="NLM_lpage">1682</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2006.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.bcp.2006.03.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2006&pages=1671-1682&author=M.+C.+Longauthor=W.+B.+Parker&title=Structure-activity+relationship+for+nucleoside+analogs+as+inhibitors+or+substrates+of+adenosine+kinase+from+Mycobacterium+tuberculosis&doi=10.1016%2Fj.bcp.2006.03.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2006.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2006.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DParker%26aufirst%3DW.%2BB.%26atitle%3DStructure-activity%2520relationship%2520for%2520nucleoside%2520analogs%2520as%2520inhibitors%2520or%2520substrates%2520of%2520adenosine%2520kinase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2006%26volume%3D71%26spage%3D1671%26epage%3D1682%26doi%3D10.1016%2Fj.bcp.2006.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaddix, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moukha-Chafiq, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationship for adenosine kinase from Mycobacterium tuberculosis</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">1588</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2008.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.bcp.2008.01.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2008&pages=1588-1600&author=M.+C.+Longauthor=S.+C.+Shaddixauthor=O.+Moukha-Chafiqauthor=J.+A.+Maddryauthor=L.+Nagyauthor=W.+B.+Parker&title=Structure-activity+relationship+for+adenosine+kinase+from+Mycobacterium+tuberculosis&doi=10.1016%2Fj.bcp.2008.01.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2008.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2008.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DMoukha-Chafiq%26aufirst%3DO.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26aulast%3DNagy%26aufirst%3DL.%26aulast%3DParker%26aufirst%3DW.%2BB.%26atitle%3DStructure-activity%2520relationship%2520for%2520adenosine%2520kinase%2520from%2520Mycobacterium%2520tuberculosis%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2008%26volume%3D75%26spage%3D1588%26epage%3D1600%26doi%3D10.1016%2Fj.bcp.2008.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malnuit, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slavětínská, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nauš, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Džubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdúch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolaříková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snášel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichová, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocek, M.</span></span> <span> </span><span class="NLM_article-title">2-Substituted 6-(Het)aryl-7-deazapurine Ribonucleosides: Synthesis, Inhibition of Adenosine Kinases, and Antimycobacterial Activity</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1079</span>– <span class="NLM_lpage">1093</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fcmdc.201500081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=25882678" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsF2hu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1079-1093&author=V.+Malnuitauthor=L.+P.+Slav%C4%9Bt%C3%ADnsk%C3%A1author=P.+Nau%C5%A1author=P.+D%C5%BEub%C3%A1kauthor=M.+Hajd%C3%BAchauthor=J.+Stola%C5%99%C3%ADkov%C3%A1author=J.+Sn%C3%A1%C5%A1elauthor=I.+Pichov%C3%A1author=M.+Hocek&title=2-Substituted+6-%28Het%29aryl-7-deazapurine+Ribonucleosides%3A+Synthesis%2C+Inhibition+of+Adenosine+Kinases%2C+and+Antimycobacterial+Activity&doi=10.1002%2Fcmdc.201500081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">2-Substituted 6-(Het)aryl-7-deazapurine Ribonucleosides: Synthesis, Inhibition of Adenosine Kinases, and Antimycobacterial Activity</span></div><div class="casAuthors">Malnuit, Vincent; Slavetinska, Lenka Postova; Naus, Petr; Dzubak, Petr; Hajduch, Marian; Stolarikova, Jirina; Snasel, Jan; Pichova, Iva; Hocek, Michal</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1079-1093</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A series of 6-(hetero)aryl- or 6-methyl-7-deazapurine ribonucleosides, e.g. I, bearing a substituent at position 2 (Cl, F, NH2, or CH3) were prepd. by cross-coupling reactions at position 6 and functional group transformations at position 2.  Cytostatic, antiviral, and antimicrobial activity assays were performed.  The title compds. were obsd. to be potent and selective inhibitors of Mycobacterium tuberculosis adenosine kinase (ADK), but not human ADK; moreover, they were found to be non-cytotoxic.  The antimycobacterial activities against M. tuberculosis, however, were only moderate.  The reason for this could be due to either poor uptake through the cell wall or to parallel biosynthesis of adenosine monophosphate by the salvage pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxwXAKv9LqdbVg90H21EOLACvtfcHk0liG3WXxtCklgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsF2hu70%253D&md5=1899f8370ad29dcfef9fee5a0de895dd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500081%26sid%3Dliteratum%253Aachs%26aulast%3DMalnuit%26aufirst%3DV.%26aulast%3DSlav%25C4%259Bt%25C3%25ADnsk%25C3%25A1%26aufirst%3DL.%2BP.%26aulast%3DNau%25C5%25A1%26aufirst%3DP.%26aulast%3DD%25C5%25BEub%25C3%25A1k%26aufirst%3DP.%26aulast%3DHajd%25C3%25BAch%26aufirst%3DM.%26aulast%3DStola%25C5%2599%25C3%25ADkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DSn%25C3%25A1%25C5%25A1el%26aufirst%3DJ.%26aulast%3DPichov%25C3%25A1%26aufirst%3DI.%26aulast%3DHocek%26aufirst%3DM.%26atitle%3D2-Substituted%25206-%2528Het%2529aryl-7-deazapurine%2520Ribonucleosides%253A%2520Synthesis%252C%2520Inhibition%2520of%2520Adenosine%2520Kinases%252C%2520and%2520Antimycobacterial%2520Activity%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1079%26epage%3D1093%26doi%3D10.1002%2Fcmdc.201500081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Snášel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nauš, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dostál, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hnízda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanfrlík, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brynda, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourderioux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dušek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvořáková, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stolaříková, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zábranská, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pohl, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konečný, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Džubák, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votruba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajdúch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Řezáčová, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veverka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichová, I.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Inhibition of Mycobacterial and Human Adenosine Kinase by 7-Substituted 7-(Het)aryl-7-deazaadenine Ribonucleosides</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">8268</span>– <span class="NLM_lpage">8279</span>, <span class="refDoi"> DOI: 10.1021/jm500497v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500497v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1eju77P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=8268-8279&author=J.+Sn%C3%A1%C5%A1elauthor=P.+Nau%C5%A1author=J.+Dost%C3%A1lauthor=A.+Hn%C3%ADzdaauthor=J.+Fanfrl%C3%ADkauthor=J.+Bryndaauthor=A.+Bourderiouxauthor=M.+Du%C5%A1ekauthor=H.+Dvo%C5%99%C3%A1kov%C3%A1author=J.+Stola%C5%99%C3%ADkov%C3%A1author=H.+Z%C3%A1bransk%C3%A1author=R.+Pohlauthor=P.+Kone%C4%8Dn%C3%BDauthor=P.+D%C5%BEub%C3%A1kauthor=I.+Votrubaauthor=M.+Hajd%C3%BAchauthor=P.+%C5%98ez%C3%A1%C4%8Dov%C3%A1author=V.+Veverkaauthor=M.+Hocekauthor=I.+Pichov%C3%A1&title=Structural+Basis+for+Inhibition+of+Mycobacterial+and+Human+Adenosine+Kinase+by+7-Substituted+7-%28Het%29aryl-7-deazaadenine+Ribonucleosides&doi=10.1021%2Fjm500497v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for inhibition of mycobacterial and human adenosine kinase by 7-substituted 7-(Het)aryl-7-deazaadenine ribonucleosides</span></div><div class="casAuthors">Snasel, Jan; Naus, Petr; Dostal, Jiri; Hnizda, Ales; Fanfrlik, Jindrich; Brynda, Jiri; Bourderioux, Aurelie; Dusek, Michal; Dvorakova, Hana; Stolarikova, Jirina; Zabranska, Helena; Pohl, Radek; Konecny, Petr; Dzubak, Petr; Votruba, Ivan; Hajduch, Marian; Rezacova, Pavlina; Veverka, Vaclav; Hocek, Michal; Pichova, Iva</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8268-8279</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adenosine kinase (ADK) from Mycobacterium tuberculosis (Mtb) was selected as a target for design of antimycobacterial nucleosides.  Screening of 7-(het)aryl-7-deazaadenine ribonucleosides with Mtb and human (h) ADKs and testing with wild-type and drug-resistant Mtb strains identified specific inhibitors of Mtb ADK with micromolar antimycobacterial activity and low cytotoxicity.  X-ray structures of complexes of Mtb and hADKs with 7-ethynyl-7-deazaadenosine showed differences in inhibitor interactions in the adenosine binding sites.  1D 1H STD NMR expts. revealed that these inhibitors are readily accommodated into the ATP and adenosine binding sites of Mtb ADK, whereas they bind preferentially into the adenosine site of hADK.  Occupation of the Mtb ADK ATP site with inhibitors and formation of catalytically less competent semiopen conformation of MtbADK after inhibitor binding in the adenosine site explain the lack of phosphorylation of 7-substituted-7-deazaadenosines.  Semiempirical quantum mech. anal. confirmed different affinity of nucleosides for the Mtb ADK adenosine and ATP sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGok5jNNyqDisbVg90H21EOLACvtfcHk0liG3WXxtCklgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1eju77P&md5=0cd96e59d560dde8953e639db9dfa428</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fjm500497v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500497v%26sid%3Dliteratum%253Aachs%26aulast%3DSn%25C3%25A1%25C5%25A1el%26aufirst%3DJ.%26aulast%3DNau%25C5%25A1%26aufirst%3DP.%26aulast%3DDost%25C3%25A1l%26aufirst%3DJ.%26aulast%3DHn%25C3%25ADzda%26aufirst%3DA.%26aulast%3DFanfrl%25C3%25ADk%26aufirst%3DJ.%26aulast%3DBrynda%26aufirst%3DJ.%26aulast%3DBourderioux%26aufirst%3DA.%26aulast%3DDu%25C5%25A1ek%26aufirst%3DM.%26aulast%3DDvo%25C5%2599%25C3%25A1kov%25C3%25A1%26aufirst%3DH.%26aulast%3DStola%25C5%2599%25C3%25ADkov%25C3%25A1%26aufirst%3DJ.%26aulast%3DZ%25C3%25A1bransk%25C3%25A1%26aufirst%3DH.%26aulast%3DPohl%26aufirst%3DR.%26aulast%3DKone%25C4%258Dn%25C3%25BD%26aufirst%3DP.%26aulast%3DD%25C5%25BEub%25C3%25A1k%26aufirst%3DP.%26aulast%3DVotruba%26aufirst%3DI.%26aulast%3DHajd%25C3%25BAch%26aufirst%3DM.%26aulast%3D%25C5%2598ez%25C3%25A1%25C4%258Dov%25C3%25A1%26aufirst%3DP.%26aulast%3DVeverka%26aufirst%3DV.%26aulast%3DHocek%26aufirst%3DM.%26aulast%3DPichov%25C3%25A1%26aufirst%3DI.%26atitle%3DStructural%2520Basis%2520for%2520Inhibition%2520of%2520Mycobacterial%2520and%2520Human%2520Adenosine%2520Kinase%2520by%25207-Substituted%25207-%2528Het%2529aryl-7-deazaadenine%2520Ribonucleosides%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D8268%26epage%3D8279%26doi%3D10.1021%2Fjm500497v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W.
B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaddix, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span> <span> </span><span class="NLM_article-title">Metabolism of 2-methyladenosine in <i>Mycobacterium tuberculosis</i></span>. <i>Tuberculosis</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.tube.2004.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.tube.2004.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=15207808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A280%3ADC%252BD2czgs1Kiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2004&pages=327-336&author=W.%0AB.+Parkerauthor=E.+W.+Barrowauthor=P.+W.+Allanauthor=S.+C.+Shaddixauthor=M.+C.+Longauthor=W.+W.+Barrowauthor=N.+Bansalauthor=J.+A.+Maddry&title=Metabolism+of+2-methyladenosine+in+Mycobacterium+tuberculosis&doi=10.1016%2Fj.tube.2004.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Metabolism of 2-methyladenosine in Mycobacterium tuberculosis</span></div><div class="casAuthors">Parker William B; Barrow Esther W; Allan Paula W; Shaddix Sue C; Long Mary C; Barrow William W; Bansal Namita; Maddry Joseph A</div><div class="citationInfo"><span class="NLM_cas:title">Tuberculosis (Edinburgh, Scotland)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">327-36</span>
        ISSN:<span class="NLM_cas:issn">1472-9792</span>.
    </div><div class="casAbstract">2-Methyladenosine (methyl-Ado) has selective activity against Mycobacterium tuberculosis (M. tuberculosis).  In an effort to better understand its mechanism of action, we have characterized its metabolism in M. tuberculosis cells.  The primary intracellular metabolite of methyl-Ado was 2-methyl-adenylate (methyl-AMP).  Very little of the methyl-AMP was metabolized further.  A M. tuberculosis strain that was resistant to methyl-Ado did not express adenosine kinase and did not convert methyl-Ado to methyl-AMP in intact cells.  In contrast to these results, the primary intracellular metabolite of adenosine in M. tuberculosis cells was ATP, which was readily incorporated into RNA.  The rate of metabolism of methyl-Ado to methyl-AMP was similar to the rate of metabolism of adenosine to ATP.  Treatment of M. tuberculosis with methyl-Ado did not affect intracellular ATP levels.  Methyl-Ado and Ado were also cleaved to 2-methyladenine and adenine, respectively, which accumulated in the medium outside the cells.  These studies suggested that methyl-AMP was the active metabolite responsible for the cytotoxicity of this agent.  Furthermore, because methyl-Ado was poorly metabolized in human cells, these studies indicated that the selective activity of methyl-Ado was due to its selective activation by M. tuberculosis.  These studies have identified two enzyme reactions (Ado kinase and Ado cleavage) in M. tuberculosis that could be exploited for the rational design of new and selective anti-M. tuberculosis agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ4k01LUFWZ4BfKb6ELK8t0fW6udTcc2ebD32IJaC0oSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2czgs1Kiuw%253D%253D&md5=810bb7d4bc54e42ec5dec653c0ebcdf0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1016%2Fj.tube.2004.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tube.2004.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DAllan%26aufirst%3DP.%2BW.%26aulast%3DShaddix%26aufirst%3DS.%2BC.%26aulast%3DLong%26aufirst%3DM.%2BC.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26aulast%3DBansal%26aufirst%3DN.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26atitle%3DMetabolism%2520of%25202-methyladenosine%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DTuberculosis%26date%3D2004%26volume%3D84%26spage%3D327%26epage%3D336%26doi%3D10.1016%2Fj.tube.2004.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suling, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allan, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, N.</span></span> <span> </span><span class="NLM_article-title">The Metabolism of 2-methyladenosine in <i>Mycobacterium smegmatis</i></span>. <i>Microbiology</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1099/00221287-148-1-289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1099%2F00221287-148-1-289" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2002&pages=289-295&author=C.-K.+Chenauthor=W.+W.+Barrowauthor=W.+B.+Parkerauthor=E.+W.+Barrowauthor=J.+A.+Maddryauthor=W.+J.+Sulingauthor=P.+W.+Allanauthor=N.+Bansal&title=The+Metabolism+of+2-methyladenosine+in+Mycobacterium+smegmatis&doi=10.1099%2F00221287-148-1-289"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1099%2F00221287-148-1-289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1099%252F00221287-148-1-289%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DC.-K.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26aulast%3DSuling%26aufirst%3DW.%2BJ.%26aulast%3DAllan%26aufirst%3DP.%2BW.%26aulast%3DBansal%26aufirst%3DN.%26atitle%3DThe%2520Metabolism%2520of%25202-methyladenosine%2520in%2520Mycobacterium%2520smegmatis%26jtitle%3DMicrobiology%26date%3D2002%26volume%3D148%26spage%3D289%26epage%3D295%26doi%3D10.1099%2F00221287-148-1-289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, E. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westbrook, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bansal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suling, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maddry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrow, W. W.</span></span> <span> </span><span class="NLM_article-title">Antimycobacterial Activity of 2-methyl-adenosine</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">801</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1093/jac/dkg444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1093%2Fjac%2Fdkg444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=14563890" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVektbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=801-808&author=E.+W.+Barrowauthor=L.+Westbrookauthor=N.+Bansalauthor=W.+J.+Sulingauthor=J.+A.+Maddryauthor=W.+B.+Parkerauthor=W.+W.+Barrow&title=Antimycobacterial+Activity+of+2-methyl-adenosine&doi=10.1093%2Fjac%2Fdkg444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Antimycobacterial activity of 2-methyl-adenosine</span></div><div class="casAuthors">Barrow, Esther W.; Westbrook, Louise; Bansal, Namita; Suling, William J.; Maddry, Joseph A.; Parker, William B.; Barrow, William W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">801-808</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The aims of this study were to assess the in vitro activity of 2-methyl-adenosine (I) against Mycobacterium tuberculosis, its intracellular efficacy, its effectiveness against M. tuberculosis in a persistent state model, and its potential mechanism of action.  In vitro activity was detd. by means of a colorimetric microdilution broth assay.  Intracellular activity was assessed with a Mono Mac 6 human monocytic cell line.  A hypoxic shift-down model was used to evaluate the effect of I on M. tuberculosis in a persistent state.  Mechanism-of-action studies were conducted by examg. the effect of I on the uptake of appropriate radiolabeled precursors into resp. mycobacterial macromol. components.  Studies confirmed the in vitro activity of I against M. tuberculosis and demonstrated intracellular efficacy against M. tuberculosis within macrophages.  I was able to significantly affect the viability of M. tuberculosis in a hypoxic shift-down model previously described to simulate the persistent state that results during tuberculosis.  Mechanism-of-action studies revealed that the immediate inhibitory effects of I were assocd. with protein and DNA synthesis and not RNA synthesis.  Results indicate that I, or similar derivs., might be effective against M. tuberculosis infections during latency.  This information should be helpful in understanding purine metab. of M. tuberculosis and also the metabolic activity of this important human pathogen in the persistent state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BOhd5Bqo4LVg90H21EOLACvtfcHk0li2Rpn-X9PwJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVektbY%253D&md5=c033912d43527e5524320f0f8c27edd0</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkg444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkg444%26sid%3Dliteratum%253Aachs%26aulast%3DBarrow%26aufirst%3DE.%2BW.%26aulast%3DWestbrook%26aufirst%3DL.%26aulast%3DBansal%26aufirst%3DN.%26aulast%3DSuling%26aufirst%3DW.%2BJ.%26aulast%3DMaddry%26aufirst%3DJ.%2BA.%26aulast%3DParker%26aufirst%3DW.%2BB.%26aulast%3DBarrow%26aufirst%3DW.%2BW.%26atitle%3DAntimycobacterial%2520Activity%2520of%25202-methyl-adenosine%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2003%26volume%3D52%26spage%3D801%26epage%3D808%26doi%3D10.1093%2Fjac%2Fdkg444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Massillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stalmans, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van de Werve, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bollen, M.</span></span> <span> </span><span class="NLM_article-title">Identification of the Glycogenic Compound 5-iodotubercidin as a General Protein Kinase Inhibitor</span>. <i>Biochem. J.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>299</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">128</span>, <span class="refDoi"> DOI: 10.1042/bj2990123</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1042%2Fbj2990123" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=1994&pages=123-128&author=D.+Massillonauthor=W.+Stalmansauthor=G.+van+de+Werveauthor=M.+Bollen&title=Identification+of+the+Glycogenic+Compound+5-iodotubercidin+as+a+General+Protein+Kinase+Inhibitor&doi=10.1042%2Fbj2990123"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1042%2Fbj2990123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj2990123%26sid%3Dliteratum%253Aachs%26aulast%3DMassillon%26aufirst%3DD.%26aulast%3DStalmans%26aufirst%3DW.%26aulast%3Dvan%2Bde%2BWerve%26aufirst%3DG.%26aulast%3DBollen%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520the%2520Glycogenic%2520Compound%25205-iodotubercidin%2520as%2520a%2520General%2520Protein%2520Kinase%2520Inhibitor%26jtitle%3DBiochem.%2520J.%26date%3D1994%26volume%3D299%26spage%3D123%26epage%3D128%26doi%3D10.1042%2Fbj2990123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ugarkar, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DaRe, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kopcho, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schanzer, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesner, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erion, M. D.</span></span> <span> </span><span class="NLM_article-title">Adenosine Kinase Inhibitors. 1. Synthesis, Enzyme Inhibition, and Antiseizure Activity of 5-Iodotubercidin Analogues</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">2883</span>– <span class="NLM_lpage">2893</span>, <span class="refDoi"> DOI: 10.1021/jm000024g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm000024g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3cXksFGltbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=2883-2893&author=B.+G.+Ugarkarauthor=J.+M.+DaReauthor=J.+J.+Kopchoauthor=C.+E.+Browneauthor=J.+M.+Schanzerauthor=J.+B.+Wiesnerauthor=M.+D.+Erion&title=Adenosine+Kinase+Inhibitors.+1.+Synthesis%2C+Enzyme+Inhibition%2C+and+Antiseizure+Activity+of+5-Iodotubercidin+Analogues&doi=10.1021%2Fjm000024g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine Kinase Inhibitors. 1. Synthesis, Enzyme Inhibition, and Anti-seizure Activity of 5-Iodotubercidin Analogues</span></div><div class="casAuthors">Ugarkar, Bheemarao G.; DaRe, Jay M.; Kopcho, Joseph J.; Browne, Clinton E. ,III; Schanzer, Juergen M.; Wiesner, James B.; Erion, Mark D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2883-2893</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Adenosine receptor agonists produce a wide variety of therapeutically useful pharmacologies.  However, to date they have failed to undergo successful clin. development due to dose-limiting side effects.  Adenosine kinase inhibitors (AKIs) represent an alternative strategy, since AKIs may raise local adenosine levels in a more site- and event-specific manner and thereby elicit the desired pharmacol. with a greater therapeutic window.  Starting with 5-iodotubercidin (IC50 = 0.026 μM) and 5'-amino-5'-deoxyadenosine (IC50 = 0.17 μM) as lead inhibitors of the isolated human AK, a variety of pyrrolo[2,3-d]pyrimidine nucleoside analogs were designed and prepd. by coupling 5-substituted-4-chloropyrrolo[2,3-d]pyrimidine bases with ribose analogs using the sodium salt-mediated glycosylation procedure.  5'-Amino-5'-deoxy analogs of 5-bromo- and 5-iodotubercidins were found to be the most potent AKIs reported to date (IC50s < 0.001 μM).  Several potent AKIs were shown to exhibit anticonvulsant activity in the rat maximal elec. shock (MES) induced seizure assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxKyAjFyc77Vg90H21EOLACvtfcHk0li2Rpn-X9PwJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXksFGltbo%253D&md5=c7bcb2eeec305751b42af1a5abd44a99</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm000024g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm000024g%26sid%3Dliteratum%253Aachs%26aulast%3DUgarkar%26aufirst%3DB.%2BG.%26aulast%3DDaRe%26aufirst%3DJ.%2BM.%26aulast%3DKopcho%26aufirst%3DJ.%2BJ.%26aulast%3DBrowne%26aufirst%3DC.%2BE.%26aulast%3DSchanzer%26aufirst%3DJ.%2BM.%26aulast%3DWiesner%26aufirst%3DJ.%2BB.%26aulast%3DErion%26aufirst%3DM.%2BD.%26atitle%3DAdenosine%2520Kinase%2520Inhibitors.%25201.%2520Synthesis%252C%2520Enzyme%2520Inhibition%252C%2520and%2520Antiseizure%2520Activity%2520of%25205-Iodotubercidin%2520Analogues%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D2883%26epage%3D2893%26doi%3D10.1021%2Fjm000024g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loomis, C.
R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, R. M.</span></span> <span> </span><span class="NLM_article-title">Sangivamycin, a Nucleoside Analogue, is a Potent Inhibitor of Protein Kinase C</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1988</span>,  <span class="NLM_volume"><i>263</i></span>,  <span class="NLM_fpage">1682</span>– <span class="NLM_lpage">1692</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1988&pages=1682-1692&author=C.%0AR.+Loomisauthor=R.+M.+Bell&title=Sangivamycin%2C+a+Nucleoside+Analogue%2C+is+a+Potent+Inhibitor+of+Protein+Kinase+C"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLoomis%26aufirst%3DC.%2BR.%26aulast%3DBell%26aufirst%3DR.%2BM.%26atitle%3DSangivamycin%252C%2520a%2520Nucleoside%2520Analogue%252C%2520is%2520a%2520Potent%2520Inhibitor%2520of%2520Protein%2520Kinase%2520C%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1988%26volume%3D263%26spage%3D1682%26epage%3D1692" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muchmore, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowart, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomtsyan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matulenko, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Severin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagwat, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowaluk, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jarvis, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakob, C. L.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of Human Adenosine Kinase Inhibitor Complexes Reveal Two Distinct Binding Modes</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">6726</span>– <span class="NLM_lpage">6731</span>, <span class="refDoi"> DOI: 10.1021/jm060189a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm060189a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6726-6731&author=S.+W.+Muchmoreauthor=R.+A.+Smithauthor=A.+O.+Stewartauthor=M.+D.+Cowartauthor=A.+Gomtsyanauthor=M.+A.+Matulenkoauthor=H.+Yuauthor=J.+M.+Severinauthor=S.+S.+Bhagwatauthor=C.-H.+Leeauthor=E.+A.+Kowalukauthor=M.+F.+Jarvisauthor=C.+L.+Jakob&title=Crystal+Structures+of+Human+Adenosine+Kinase+Inhibitor+Complexes+Reveal+Two+Distinct+Binding+Modes&doi=10.1021%2Fjm060189a"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Fjm060189a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060189a%26sid%3Dliteratum%253Aachs%26aulast%3DMuchmore%26aufirst%3DS.%2BW.%26aulast%3DSmith%26aufirst%3DR.%2BA.%26aulast%3DStewart%26aufirst%3DA.%2BO.%26aulast%3DCowart%26aufirst%3DM.%2BD.%26aulast%3DGomtsyan%26aufirst%3DA.%26aulast%3DMatulenko%26aufirst%3DM.%2BA.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DSeverin%26aufirst%3DJ.%2BM.%26aulast%3DBhagwat%26aufirst%3DS.%2BS.%26aulast%3DLee%26aufirst%3DC.-H.%26aulast%3DKowaluk%26aufirst%3DE.%2BA.%26aulast%3DJarvis%26aufirst%3DM.%2BF.%26aulast%3DJakob%26aufirst%3DC.%2BL.%26atitle%3DCrystal%2520Structures%2520of%2520Human%2520Adenosine%2520Kinase%2520Inhibitor%2520Complexes%2520Reveal%2520Two%2520Distinct%2520Binding%2520Modes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6726%26epage%3D6731%26doi%3D10.1021%2Fjm060189a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zemlicka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endo, T.</span></span> <span> </span><span class="NLM_article-title">O6-(4-Nitrophenyl)inosine and -Guanosine as Chromogenic Substrates for Adenosine Deaminase</span>. <i>Nucleosides Nucleotides</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">619</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1080/07328319608002410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1080%2F07328319608002410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK28XhtV2ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1996&pages=619-629&author=J.+Zemlickaauthor=T.+Endo&title=O6-%284-Nitrophenyl%29inosine+and+-Guanosine+as+Chromogenic+Substrates+for+Adenosine+Deaminase&doi=10.1080%2F07328319608002410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">O6-(4-nitrophenyl)inosine and -guanosine as chromogenic substrates for adenosine deaminase</span></div><div class="casAuthors">Zemlicka, Jiri; Endo, Takeshi</div><div class="citationInfo"><span class="NLM_cas:title">Nucleosides & Nucleotides</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">619-29</span>CODEN:
                <span class="NLM_cas:coden">NUNUD5</span>;
        ISSN:<span class="NLM_cas:issn">0732-8311</span>.
    
            (<span class="NLM_cas:orgname">Dekker</span>)
        </div><div class="casAbstract">O6-(4-nitrophenyl)inosine (I), O6-(4-nitrophenyl)guanosine (II) and O6-(4-methylumbelliferonyl)inosine (III) were obtained by reaction of 6-chloro-9-(β-D-ribofuranosyl)purine or 2-amino-6-chloro-9-(β-D-ribofuranosyl)purine with sodium salts of 4-nitrophenol or 4-methylumbelliferone in N,N-dimethylformamide.  Similarly, 6-chloro-9-(β-D-2,3-isopropylideneribofuranosyl)purine was transformed to 2',3'-O-isopropylidene-O6-(4-nitrophenyl)inosine.  Compds. I and II are substrates for adenosine deaminase releasing 4-nitrophenol which is readily detected visually or spectrophotometrically.  Rate and extent of hydrolysis of I are significantly increase in the presence of purine nucleoside phosphorylase but xanthine oxidase has no influence.  A potential fluorogenic analog III is not a substrate for adenosine deaminase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq35yVXYd5rhLVg90H21EOLACvtfcHk0lj8VvoDq-g3lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtV2ju7Y%253D&md5=7fd2b7f97f80bd9b2b50522119baa6bd</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1080%2F07328319608002410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F07328319608002410%26sid%3Dliteratum%253Aachs%26aulast%3DZemlicka%26aufirst%3DJ.%26aulast%3DEndo%26aufirst%3DT.%26atitle%3DO6-%25284-Nitrophenyl%2529inosine%2520and%2520-Guanosine%2520as%2520Chromogenic%2520Substrates%2520for%2520Adenosine%2520Deaminase%26jtitle%3DNucleosides%2520Nucleotides%26date%3D1996%26volume%3D15%26spage%3D619%26epage%3D629%26doi%3D10.1080%2F07328319608002410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camp, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCluskey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moni, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, R. J.</span></span> <span> </span><span class="NLM_article-title">Diimidazo[1,2-c:4′,5′-e]pyrimidines: N6-N1 conformationally restricted adenosines</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">695</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1016/s0960-894x(98)00101-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fs0960-894x%2898%2900101-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1998&pages=695-698&author=D.+Campauthor=Y.+Liauthor=A.+McCluskeyauthor=R.+W.+Moniauthor=R.+J.+Quinn&title=Diimidazo%5B1%2C2-c%3A4%E2%80%B2%2C5%E2%80%B2-e%5Dpyrimidines%3A+N6-N1+conformationally+restricted+adenosines&doi=10.1016%2Fs0960-894x%2898%2900101-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fs0960-894x%2898%2900101-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0960-894x%252898%252900101-2%26sid%3Dliteratum%253Aachs%26aulast%3DCamp%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DMcCluskey%26aufirst%3DA.%26aulast%3DMoni%26aufirst%3DR.%2BW.%26aulast%3DQuinn%26aufirst%3DR.%2BJ.%26atitle%3DDiimidazo%255B1%252C2-c%253A4%25E2%2580%25B2%252C5%25E2%2580%25B2-e%255Dpyrimidines%253A%2520N6-N1%2520conformationally%2520restricted%2520adenosines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1998%26volume%3D8%26spage%3D695%26epage%3D698%26doi%3D10.1016%2Fs0960-894x%2898%2900101-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katagiri, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuhashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokufuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takebayashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, C.</span></span> <span> </span><span class="NLM_article-title">A Highly Efficient Synthesis of the Antiviral Agent (+)-cyclaradine Involving the Regioselective Cleavage of Epoxide by Neighboring Participation</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1961</span>– <span class="NLM_lpage">1964</span>, <span class="refDoi"> DOI: 10.1016/s0040-4039(97)00234-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fs0040-4039%2897%2900234-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1997&pages=1961-1964&author=N.+Katagiriauthor=Y.+Matsuhashiauthor=H.+Kokufudaauthor=M.+Takebayashiauthor=C.+Kaneko&title=A+Highly+Efficient+Synthesis+of+the+Antiviral+Agent+%28%2B%29-cyclaradine+Involving+the+Regioselective+Cleavage+of+Epoxide+by+Neighboring+Participation&doi=10.1016%2Fs0040-4039%2897%2900234-7"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4039%2897%2900234-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4039%252897%252900234-7%26sid%3Dliteratum%253Aachs%26aulast%3DKatagiri%26aufirst%3DN.%26aulast%3DMatsuhashi%26aufirst%3DY.%26aulast%3DKokufuda%26aufirst%3DH.%26aulast%3DTakebayashi%26aufirst%3DM.%26aulast%3DKaneko%26aufirst%3DC.%26atitle%3DA%2520Highly%2520Efficient%2520Synthesis%2520of%2520the%2520Antiviral%2520Agent%2520%2528%252B%2529-cyclaradine%2520Involving%2520the%2520Regioselective%2520Cleavage%2520of%2520Epoxide%2520by%2520Neighboring%2520Participation%26jtitle%3DTetrahedron%2520Lett.%26date%3D1997%26volume%3D38%26spage%3D1961%26epage%3D1964%26doi%3D10.1016%2Fs0040-4039%2897%2900234-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugawara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuno, Y.</span></span> <span> </span><span class="NLM_article-title">A Novel Synthesis of 1-Deazaadenosine</span>. <i>Heterocycles</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.3987/s-1982-01-0305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.3987%2Fs-1982-01-0305" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1982&pages=305-309&author=T.+Itohauthor=T.+Sugawaraauthor=Y.+Mizuno&title=A+Novel+Synthesis+of+1-Deazaadenosine&doi=10.3987%2Fs-1982-01-0305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.3987%2Fs-1982-01-0305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3987%252Fs-1982-01-0305%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DT.%26aulast%3DSugawara%26aufirst%3DT.%26aulast%3DMizuno%26aufirst%3DY.%26atitle%3DA%2520Novel%2520Synthesis%2520of%25201-Deazaadenosine%26jtitle%3DHeterocycles%26date%3D1982%26volume%3D17%26spage%3D305%26epage%3D309%26doi%3D10.3987%2Fs-1982-01-0305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilkens, S.</span></span> <span> </span><span class="NLM_article-title">Structure and Mechanism of ABC transporters</span>. <i>F1000Prime Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">14</span>, <span class="refDoi"> DOI: 10.12703/p7-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.12703%2Fp7-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=25750732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A280%3ADC%252BC2MnhvFWhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=14&author=S.+Wilkens&title=Structure+and+Mechanism+of+ABC+transporters&doi=10.12703%2Fp7-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and mechanism of ABC transporters</span></div><div class="casAuthors">Wilkens Stephan</div><div class="citationInfo"><span class="NLM_cas:title">F1000prime reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14</span>
        ISSN:<span class="NLM_cas:issn">2051-7599</span>.
    </div><div class="casAbstract">All living organisms depend on primary and secondary membrane transport for the supply of external nutrients and removal or sequestration of unwanted (toxic) compounds.  Due to the chemical diversity of cellular molecules, it comes as no surprise that a significant part of the proteome is dedicated to the active transport of cargo across the plasma membrane or the membranes of subcellular organelles.  Transport against a chemical gradient can be driven by, for example, the free energy change associated with ATP hydrolysis (primary transport), or facilitated by the potential energy of the chemical gradient of another molecule (secondary transport).  Primary transporters include the rotary motor ATPases (F-, A-, and V-ATPases), P-type ATPases and a large family of integral membrane proteins referred to as "ABC" (ATP binding cassette) transporters.  ABC transporters are widespread in all forms of life and are characterized by two nucleotide-binding domains (NBD) and two transmembrane domains (TMDs).  ATP hydrolysis on the NBD drives conformational changes in the TMD, resulting in alternating access from inside and outside of the cell for unidirectional transport across the lipid bilayer.  Common to all ABC transporters is a signature sequence or motif, LSGGQ, that is involved in nucleotide binding.  Both importing and exporting ABC transporters are found in bacteria, whereas the majority of eukaryotic family members function in the direction of export.  Recent progress with the X-ray crystal structure determination of a variety of bacterial and eukaryotic ABC transporters has helped to advance our understanding of the ATP hydrolysis-driven transport mechanism but has also illustrated the large structural and functional diversity within the family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTnMdLvCnsfMK__Jf7-eMvJfW6udTcc2ebj8e-QKStebLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MnhvFWhsA%253D%253D&md5=c33f6447d0f32735fae9b6e48f5e18e9</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.12703%2Fp7-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12703%252Fp7-14%26sid%3Dliteratum%253Aachs%26aulast%3DWilkens%26aufirst%3DS.%26atitle%3DStructure%2520and%2520Mechanism%2520of%2520ABC%2520transporters%26jtitle%3DF1000Prime%2520Rep.%26date%3D2015%26volume%3D7%26spage%3D14%26doi%3D10.12703%2Fp7-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Braibant, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilot, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Content, J.</span></span> <span> </span><span class="NLM_article-title">The ATP Binding Cassette (ABC) Transport Systems of <i>Mycobacterium tuberculosis</i></span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">449</span>– <span class="NLM_lpage">467</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2000.tb00550.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1111%2Fj.1574-6976.2000.tb00550.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10978546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3cXmt1Sjtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2000&pages=449-467&author=M.+Braibantauthor=P.+Gilotauthor=J.+Content&title=The+ATP+Binding+Cassette+%28ABC%29+Transport+Systems+of+Mycobacterium+tuberculosis&doi=10.1111%2Fj.1574-6976.2000.tb00550.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">The ATP binding cassette (ABC) transport systems of Mycobacterium tuberculosis</span></div><div class="casAuthors">Braibant, M.; Gilot, P.; Content, J.</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">449-467</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">A review with 114 refs.  We have undertaken the inventory and assembly of the typical subunits of the ABC transporters encoded by the complete genome of Mycobacterium tuberculosis.  These subunits, i.e. the nucleotide binding domains (NBDs), the membrane-spanning domains (MSDs) and the substrate binding proteins (SBPs), were identified on the basis of their characteristic stretches of amino acids and/or conserved structure.  A total of 45 NBDs present in 38 proteins, of 47 MSDs present in 44 proteins and of 15 SBPs were found to be encoded by M. tuberculosis.  Anal. of transcriptional clusters and searches of homol. between the identified subunits of the transporters and proteins characterized in other organisms allowed the reconstitution of at least 26 complete (including at least one NBD and one MSD) and 11 incomplete ABC transporters.  Sixteen of them were unambiguously classified as importers whereas 21 were presumed to be exporters.  By searches of homol. with already known transporters from other organisms, potential substrates (peptides, macrolides, carbohydrates, multiple drugs, antibiotics, iron, anions) could be attributed to 30 of the ABC transporters identified in M. tuberculosis.  The ABC transporters have been further classified in nine different sub-families according to a tree obtained from the clustering of their NBDs.  Contrary to Escherichia coli and similarly to Bacillus subtilis, there is an equal representation of extruders and importers.  Many exporters were found to be potentially implicated in the transport of drugs, probably contributing to the resistance of M. tuberculosis to many antibiotics.  Interestingly, a transporter (absent in E. coli and in B. subtilis) potentially implicated in the export of a factor required for the bacterial attachment to the eukaryotic host cells was also identified.  In comparison to E. coli and B. subtilis, there is an under-representation of the importers (with the exception of the phosphate importers) in M. tuberculosis.  This may reflect the capacity of this bacterium to synthesize many essential compds. and to grow in the presence of few external nutrients.  The genes encoding the ABC transporters occupy about 2.5% of the genome of M. tuberculosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrg1LOuNresYrVg90H21EOLACvtfcHk0lhJl3oQq5iLjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXmt1Sjtr0%253D&md5=7ba9ad7b0615bf971cf8d402c10cc1ea</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2000.tb00550.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2000.tb00550.x%26sid%3Dliteratum%253Aachs%26aulast%3DBraibant%26aufirst%3DM.%26aulast%3DGilot%26aufirst%3DP.%26aulast%3DContent%26aufirst%3DJ.%26atitle%3DThe%2520ATP%2520Binding%2520Cassette%2520%2528ABC%2529%2520Transport%2520Systems%2520of%2520Mycobacterium%2520tuberculosis%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2000%26volume%3D24%26spage%3D449%26epage%3D467%26doi%3D10.1111%2Fj.1574-6976.2000.tb00550.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sandhu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akhter, Y.</span></span> <span> </span><span class="NLM_article-title">The Drug Binding Sites and Transport Mechanism of the RND Pumps from <i>Mycobacterium tuberculosis</i>: Insights from Molecular Dynamics Simulations</span>. <i>Arch. Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>592</i></span>,  <span class="NLM_fpage">38</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.abb.2016.01.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.abb.2016.01.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=592&publication_year=2016&pages=38-49&author=P.+Sandhuauthor=Y.+Akhter&title=The+Drug+Binding+Sites+and+Transport+Mechanism+of+the+RND+Pumps+from+Mycobacterium+tuberculosis%3A+Insights+from+Molecular+Dynamics+Simulations&doi=10.1016%2Fj.abb.2016.01.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.abb.2016.01.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.abb.2016.01.007%26sid%3Dliteratum%253Aachs%26aulast%3DSandhu%26aufirst%3DP.%26aulast%3DAkhter%26aufirst%3DY.%26atitle%3DThe%2520Drug%2520Binding%2520Sites%2520and%2520Transport%2520Mechanism%2520of%2520the%2520RND%2520Pumps%2520from%2520Mycobacterium%2520tuberculosis%253A%2520Insights%2520from%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DArch.%2520Biochem.%2520Biophys.%26date%3D2016%26volume%3D592%26spage%3D38%26epage%3D49%26doi%3D10.1016%2Fj.abb.2016.01.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Rossi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aínsa, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riccardi, G.</span></span> <span> </span><span class="NLM_article-title">Role of Mycobacterial Efflux Transporters in Drug Resistance: an Unresolved Question</span>. <i>FEMS Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1111/j.1574-6976.2005.00002.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1111%2Fj.1574-6976.2005.00002.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=16438679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2006&pages=36-52&author=E.+De%0ARossiauthor=J.+A.+A%C3%ADnsaauthor=G.+Riccardi&title=Role+of+Mycobacterial+Efflux+Transporters+in+Drug+Resistance%3A+an+Unresolved+Question&doi=10.1111%2Fj.1574-6976.2005.00002.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Role of mycobacterial efflux transporters in drug resistance: an unresolved question</span></div><div class="casAuthors">De Rossi, Edda; Ainsa, Jose A.; Riccardi, Giovanna</div><div class="citationInfo"><span class="NLM_cas:title">FEMS Microbiology Reviews</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-52</span>CODEN:
                <span class="NLM_cas:coden">FMREE4</span>;
        ISSN:<span class="NLM_cas:issn">0168-6445</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Two mechanisms are thought to be involved in the natural drug resistance of mycobacteria: the mycobacterial cell wall permeability barrier and active multidrug efflux pumps.  Genes encoding drug efflux transporters have been isolated from several mycobacterial species.  These proteins transport tetracycline, fluoroquinolones, aminoglycosides and other compds.  Recent reports have suggested that efflux pumps may also be involved in transporting isoniazid, one of the main drugs used to treat tuberculosis.  This review highlights recent advances in our understanding of efflux-mediated drug resistance in mycobacteria, including the distribution of efflux systems in these organisms, their substrate profiles and their contribution to drug resistance.  The balance between the drug transport into the cell and drug efflux is not yet clearly understood, and further studies are required in mycobacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpwjdJiKjbO7Vg90H21EOLACvtfcHk0lhJl3oQq5iLjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvFKrsbw%253D&md5=41af6cedab317d501c489400a6979920</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1111%2Fj.1574-6976.2005.00002.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1574-6976.2005.00002.x%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BRossi%26aufirst%3DE.%26aulast%3DA%25C3%25ADnsa%26aufirst%3DJ.%2BA.%26aulast%3DRiccardi%26aufirst%3DG.%26atitle%3DRole%2520of%2520Mycobacterial%2520Efflux%2520Transporters%2520in%2520Drug%2520Resistance%253A%2520an%2520Unresolved%2520Question%26jtitle%3DFEMS%2520Microbiol.%2520Rev.%26date%3D2006%26volume%3D30%26spage%3D36%26epage%3D52%26doi%3D10.1111%2Fj.1574-6976.2005.00002.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Balganesh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinesh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuruppath, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nair, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, U.</span></span> <span> </span><span class="NLM_article-title">Efflux Pumps of Mycobacterium tuberculosis Play a Significant Role in Antituberculosis Activity of Potential Drug Candidates</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">2643</span>– <span class="NLM_lpage">2651</span>, <span class="refDoi"> DOI: 10.1128/aac.06003-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1128%2Faac.06003-11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=2643-2651&author=M.+Balganeshauthor=N.+Dineshauthor=S.+Sharmaauthor=S.+Kuruppathauthor=A.+V.+Nairauthor=U.+Sharma&title=Efflux+Pumps+of+Mycobacterium+tuberculosis+Play+a+Significant+Role+in+Antituberculosis+Activity+of+Potential+Drug+Candidates&doi=10.1128%2Faac.06003-11"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1128%2Faac.06003-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Faac.06003-11%26sid%3Dliteratum%253Aachs%26aulast%3DBalganesh%26aufirst%3DM.%26aulast%3DDinesh%26aufirst%3DN.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DKuruppath%26aufirst%3DS.%26aulast%3DNair%26aufirst%3DA.%2BV.%26aulast%3DSharma%26aufirst%3DU.%26atitle%3DEfflux%2520Pumps%2520of%2520Mycobacterium%2520tuberculosis%2520Play%2520a%2520Significant%2520Role%2520in%2520Antituberculosis%2520Activity%2520of%2520Potential%2520Drug%2520Candidates%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D2643%26epage%3D2651%26doi%3D10.1128%2Faac.06003-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pule, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sampson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Helden, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Victor, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louw, G. E.</span></span> <span> </span><span class="NLM_article-title">Efflux Pump Inhibitors: Targeting Mycobacterial Efflux Systems to Enhance TB Therapy</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1093/jac/dkv316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1093%2Fjac%2Fdkv316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=26472768" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC28XotVeqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2016&pages=17-26&author=C.+M.+Puleauthor=S.+L.+Sampsonauthor=R.+M.+Warrenauthor=P.+A.+Blackauthor=P.+D.+van+Heldenauthor=T.+C.+Victorauthor=G.+E.+Louw&title=Efflux+Pump+Inhibitors%3A+Targeting+Mycobacterial+Efflux+Systems+to+Enhance+TB+Therapy&doi=10.1093%2Fjac%2Fdkv316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Efflux pump inhibitors: targeting mycobacterial efflux systems to enhance TB therapy</span></div><div class="casAuthors">Pule, Caroline M.; Sampson, Samantha L.; Warren, Robin M.; Black, Philippa A.; van Helden, Paul D.; Victor, Tommie C.; Louw, Gail E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-26</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The emergence of drug resistance continues to plague TB control, with a global increase in the prevalence of MDR-TB.  This acts as a gateway to XDR-TB and thus emphasizes the urgency for drug development and optimal treatment options.  Bedaquiline is the first new anti-TB drug approved by the FDA in 40 years and has been shown to be an effective treatment option for MDR Mycobacterium tuberculosis infection.  Bedaquiline has also recently been included in clin. trials for new regimens with the aim of improving and shortening treatment periods.  Alarmingly, efflux-mediated bedaquiline resistance, as well as efflux-mediated cross-resistance to clofazimine, has been identified in treatment failures.  This mechanism of resistance results in efflux of a variety of anti-TB drugs from the bacterial cell, thereby decreasing the intracellular drug concn.  In doing so, the bacillus is able to render the antibiotic treatment ineffective.  Recent studies have explored strategies to reverse the resistance phenotype conferred by efflux pump activation.  It was obsd. that the addn. of efflux pump inhibitors partially restored drug susceptibility in vitro and in vivo.  This has significant clin. implications, esp. in MDR-TB management where treatment options are extremely limited.  This review aims to highlight the current efflux pump inhibitors effective against M. tuberculosis, the effect of efflux pump inhibitors on mycobacterial growth and the clin. promise of treatment with efflux pump inhibitors and std. anti-TB therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6OyOfnhS48rVg90H21EOLACvtfcHk0lgC0A02TZwKcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XotVeqsrs%253D&md5=c72ef921e040c1d9787cec02639bcd7d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkv316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkv316%26sid%3Dliteratum%253Aachs%26aulast%3DPule%26aufirst%3DC.%2BM.%26aulast%3DSampson%26aufirst%3DS.%2BL.%26aulast%3DWarren%26aufirst%3DR.%2BM.%26aulast%3DBlack%26aufirst%3DP.%2BA.%26aulast%3Dvan%2BHelden%26aufirst%3DP.%2BD.%26aulast%3DVictor%26aufirst%3DT.%2BC.%26aulast%3DLouw%26aufirst%3DG.%2BE.%26atitle%3DEfflux%2520Pump%2520Inhibitors%253A%2520Targeting%2520Mycobacterial%2520Efflux%2520Systems%2520to%2520Enhance%2520TB%2520Therapy%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2016%26volume%3D71%26spage%3D17%26epage%3D26%26doi%3D10.1093%2Fjac%2Fdkv316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, R. S.</span></span> <span> </span><span class="NLM_article-title">Mycobacterial Adenosine Kinase is not a Typical Adenosine Kinase</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>583</i></span>,  <span class="NLM_fpage">2231</span>– <span class="NLM_lpage">2236</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2009.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.febslet.2009.06.002" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=583&publication_year=2009&pages=2231-2236&author=J.+Parkauthor=B.+Singhauthor=R.+S.+Gupta&title=Mycobacterial+Adenosine+Kinase+is+not+a+Typical+Adenosine+Kinase&doi=10.1016%2Fj.febslet.2009.06.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2009.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2009.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DSingh%26aufirst%3DB.%26aulast%3DGupta%26aufirst%3DR.%2BS.%26atitle%3DMycobacterial%2520Adenosine%2520Kinase%2520is%2520not%2520a%2520Typical%2520Adenosine%2520Kinase%26jtitle%3DFEBS%2520Lett.%26date%3D2009%26volume%3D583%26spage%3D2231%26epage%3D2236%26doi%3D10.1016%2Fj.febslet.2009.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shashank, R.</span></span> <span> </span><span class="NLM_article-title">Application of LC-MS in Supporting PK/PD Studies During Drug Discovery and Development</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">2514</span>– <span class="NLM_lpage">2526</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=2514-2526&author=T.+R.+Ahmedauthor=R.+Shashank&title=Application+of+LC-MS+in+Supporting+PK%2FPD+Studies+During+Drug+Discovery+and+Development"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAhmed%26aufirst%3DT.%2BR.%26aulast%3DShashank%26aufirst%3DR.%26atitle%3DApplication%2520of%2520LC-MS%2520in%2520Supporting%2520PK%252FPD%2520Studies%2520During%2520Drug%2520Discovery%2520and%2520Development%26jtitle%3DPharm.%2520Res.%26date%3D2012%26volume%3D5%26spage%3D2514%26epage%3D2526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hull, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morton, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfister, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabemampianina, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vidal, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vorstenbosch, C. V. D.</span></span> <span> </span><span class="NLM_article-title">A Good Practice Guide to the Administration of Substances and Removal of Blood, Including Routes and Volumes</span>. <i>J. Appl. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">15</span>– <span class="NLM_lpage">23</span>, <span class="refDoi"> DOI: 10.1002/jat.727</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fjat.727" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=11180276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD3MXht1Kmurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=15-23&author=K.-H.+Diehlauthor=R.+Hullauthor=D.+Mortonauthor=R.+Pfisterauthor=Y.+Rabemampianinaauthor=D.+Smithauthor=J.-M.+Vidalauthor=C.+V.+D.+Vorstenbosch&title=A+Good+Practice+Guide+to+the+Administration+of+Substances+and+Removal+of+Blood%2C+Including+Routes+and+Volumes&doi=10.1002%2Fjat.727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">A good practice guide to the administration of substances and removal of blood, including routes and volumes</span></div><div class="casAuthors">Diehl, Karl-Heinz; Hull, Robin; Morton, David; Pfister, Rudolf; Rabemampianina, Yvon; Smith, David; Vidal, Jean-Marc; Van de Vorstenbosch, Cor</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Toxicology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">15-23</span>CODEN:
                <span class="NLM_cas:coden">JJATDK</span>;
        ISSN:<span class="NLM_cas:issn">0260-437X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">This article is the result of an initiative between the European Federation of Pharmaceutical Industries Assocns. (EFPIA) and the European Center for the Validation of Alternative Methods (ECVAM).  Its objectives are to provide the researcher in the safety evaluation lab. with an up-to-date, easy-to-use set of data sheets to aid in the study design process while at the same time affording max. welfare considerations to the exptl. animals.  Although this article is targeted at researchers in the European Pharmaceutical Industry, it is considered that the principles underpinning the data sets and refinement proposals are equally applicable to all those who use these techniques on animals in their research, whether in research institutes, universities or other sectors of industry.  The implications of this article may lead to discussion with regulators, such as those responsible for pharmacopoeial testing.  There are numerous publications dealing with the administration of test substances and the removal of blood samples, and many labs. also have their own "inhouse" guidelines that have been developed by custom and practice over many years.  Within European Union Directive 86/609EEC we have an obligation to refine expts. to cause the min. amt. of stress.  We hope that this article will provide background data useful to those responsible for protocol design and review.  This guide is based on peer-reviewed publications whenever possible, but where this is not possible we have used "inhouse" data and the experience of those on the working party (as well as helpful comments submitted by the industry) for a final opinion.  The guide also addresses the continuing need to refine the techniques assocd. with the administration of substances and the withdrawal of blood, and suggests ways of doing so.  Data-sharing between labs. should be encouraged to avoid duplication of animal work, as well as sharing practical skills concerning animal welfare and scientific problems caused by "overdosing" in some way or another.  The recommendations in this guide refer to the "normal" animal, and special consideration is needed, for instance, during pregnancy and lactation.  Interpretation of studies may be confounded when large vols. are administered or excessive sampling employed, particularly if anesthetics are used.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpirIZeMEFS_7Vg90H21EOLACvtfcHk0ljXnBFqNK8nTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXht1Kmurw%253D&md5=a1dca38550737c5ab02ae49f2c8eed34</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2Fjat.727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjat.727%26sid%3Dliteratum%253Aachs%26aulast%3DDiehl%26aufirst%3DK.-H.%26aulast%3DHull%26aufirst%3DR.%26aulast%3DMorton%26aufirst%3DD.%26aulast%3DPfister%26aufirst%3DR.%26aulast%3DRabemampianina%26aufirst%3DY.%26aulast%3DSmith%26aufirst%3DD.%26aulast%3DVidal%26aufirst%3DJ.-M.%26aulast%3DVorstenbosch%26aufirst%3DC.%2BV.%2BD.%26atitle%3DA%2520Good%2520Practice%2520Guide%2520to%2520the%2520Administration%2520of%2520Substances%2520and%2520Removal%2520of%2520Blood%252C%2520Including%2520Routes%2520and%2520Volumes%26jtitle%3DJ.%2520Appl.%2520Toxicol.%26date%3D2001%26volume%3D21%26spage%3D15%26epage%3D23%26doi%3D10.1002%2Fjat.727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isbell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span> <span> </span><span class="NLM_article-title">Snapshot PK: a Rapid Rodent <i>in vivo</i> Preclinical Screening Approach</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">360</span>– <span class="NLM_lpage">367</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2007.10.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.drudis.2007.10.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=18405850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A280%3ADC%252BD1c3kslSkuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=360-367&author=B.+Liuauthor=J.+Changauthor=W.+P.+Gordonauthor=J.+Isbellauthor=Y.+Zhouauthor=T.+Tuntland&title=Snapshot+PK%3A+a+Rapid+Rodent+in+vivo+Preclinical+Screening+Approach&doi=10.1016%2Fj.drudis.2007.10.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Snapshot PK: a rapid rodent in vivo preclinical screening approach</span></div><div class="casAuthors">Liu Bo; Chang Jonathan; Gordon William P; Isbell John; Zhou Yingyao; Tuntland Tove</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">360-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">Described in this article are strategies implemented to increase the throughput of in vivo rodent pharmacokinetic (PK) studies using the snapshot PK study design and automated methods for compound submission, sample processing, data analysis and reporting.  Applying snapshot PK studies to categorize the oral exposure of >1300 discovery compounds as low, moderate or high resulted in an attrition rate of 86%.  The follow up full PK studies on the remaining compounds found that 98% of the compounds were predicted in the correct (69%) or adjacent (29%) oral exposure category by the snapshot PK studies.  These results demonstrate that the snapshot PK screen in rodents can serve as an effective and efficient in vivo tool in the compound selection process in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR7lxjN22Wtb4xhnOQkIHOxfW6udTcc2ebwTJ0PWyJf6bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1c3kslSkuw%253D%253D&md5=2f0ca34fc2b06e13fc85fb28be221ffe</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2007.10.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2007.10.014%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DChang%26aufirst%3DJ.%26aulast%3DGordon%26aufirst%3DW.%2BP.%26aulast%3DIsbell%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DTuntland%26aufirst%3DT.%26atitle%3DSnapshot%2520PK%253A%2520a%2520Rapid%2520Rodent%2520in%2520vivo%2520Preclinical%2520Screening%2520Approach%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2008%26volume%3D13%26spage%3D360%26epage%3D367%26doi%3D10.1016%2Fj.drudis.2007.10.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Malley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guinn, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickey, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohaideen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boshoff, H. I. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizrahi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassetti, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, C. E.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">Identification of New Drug Targets and Resistance Mechanisms in <i>Mycobacterium tuberculosis</i></span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e75245</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0075245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1371%2Fjournal.pone.0075245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=24086479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFeku7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=T.+R.+Ioergerauthor=T.+O%E2%80%99Malleyauthor=R.+Liaoauthor=K.+M.+Guinnauthor=M.+J.+Hickeyauthor=N.+Mohaideenauthor=K.+C.+Murphyauthor=H.+I.+M.+Boshoffauthor=V.+Mizrahiauthor=E.+J.+Rubinauthor=C.+M.+Sassettiauthor=C.+E.+Barryauthor=D.+R.+Shermanauthor=T.+Parishauthor=J.+C.+Sacchettini&title=Identification+of+New+Drug+Targets+and+Resistance+Mechanisms+in+Mycobacterium+tuberculosis&doi=10.1371%2Fjournal.pone.0075245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of new drug targets and resistance mechanisms in Mycobacterium tuberculosis</span></div><div class="casAuthors">Ioerger, Thomas R.; O'Malley, Theresa; Liao, Reiling; Guinn, Kristine M.; Hickey, Mark J.; Mohaideen, Nilofar; Murphy, Kenan C.; Boshoff, Helena I. M.; Mizrahi, Valerie; Rubin, Eric J.; Sassetti, Christopher M.; Barry, Clifton E., III; Sherman, David R.; Parish, Tanya; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e75245</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Identification of new drug targets is vital for the advancement of drug discovery against Mycobacterium tuberculosis, esp. given the increase of resistance worldwide to first- and second-line drugs.  Because traditional target-based screening has largely proven unsuccessful for antibiotic discovery, we have developed a scalable platform for target identification in M. tuberculosis that is based on whole-cell screening, coupled with whole-genome sequencing of resistant mutants and recombineering to confirm.  The method yields targets paired with whole-cell active compds., which can serve as novel scaffolds for drug development, mol. tools for validation, and/or as ligands for co-crystn.  It may also reveal other information about mechanisms of action, such as activation or efflux.  Using this method, we identified resistance-linked genes for eight compds. with anti-tubercular activity.  Four of the genes have previously been shown to be essential: AspS, aspartyl-tRNA synthetase, Pks13, a polyketide synthase involved in mycolic acid biosynthesis, MmpL3, a membrane transporter, and EccB3, a component of the ESX-3 type VII secretion system.  AspS and Pks13 represent novel targets in protein translation and cell-wall biosynthesis.  Both MmpL3 and EccB3 are involved in membrane transport.  Pks13, AspS, and EccB3 represent novel candidates not targeted by existing TB drugs, and the availability of whole-cell active inhibitors greatly increases their potential for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoen2dg2HIM0LVg90H21EOLACvtfcHk0liC251t0b-n8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFeku7rM&md5=2df0db796fc57f875aa12d4ac5f09984</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0075245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0075245%26sid%3Dliteratum%253Aachs%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DO%25E2%2580%2599Malley%26aufirst%3DT.%26aulast%3DLiao%26aufirst%3DR.%26aulast%3DGuinn%26aufirst%3DK.%2BM.%26aulast%3DHickey%26aufirst%3DM.%2BJ.%26aulast%3DMohaideen%26aufirst%3DN.%26aulast%3DMurphy%26aufirst%3DK.%2BC.%26aulast%3DBoshoff%26aufirst%3DH.%2BI.%2BM.%26aulast%3DMizrahi%26aufirst%3DV.%26aulast%3DRubin%26aufirst%3DE.%2BJ.%26aulast%3DSassetti%26aufirst%3DC.%2BM.%26aulast%3DBarry%26aufirst%3DC.%2BE.%26aulast%3DSherman%26aufirst%3DD.%2BR.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DIdentification%2520of%2520New%2520Drug%2520Targets%2520and%2520Resistance%2520Mechanisms%2520in%2520Mycobacterium%2520tuberculosis%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0075245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganesula, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobos, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortune, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobs, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizrahi, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">Variation among Genome Sequences of H37Rv Strains of <i>Mycobacterium tuberculosis</i> from Multiple Laboratories</span>. <i>J. Bacteriol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>192</i></span>,  <span class="NLM_fpage">3645</span>– <span class="NLM_lpage">3653</span>, <span class="refDoi"> DOI: 10.1128/jb.00166-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1128%2Fjb.00166-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=20472797" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVCktLfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=192&publication_year=2010&pages=3645-3653&author=T.+R.+Ioergerauthor=Y.+Fengauthor=K.+Ganesulaauthor=X.+Chenauthor=K.+M.+Dobosauthor=S.+Fortuneauthor=W.+R.+Jacobsauthor=V.+Mizrahiauthor=T.+Parishauthor=E.+Rubinauthor=C.+Sassettiauthor=J.+C.+Sacchettini&title=Variation+among+Genome+Sequences+of+H37Rv+Strains+of+Mycobacterium+tuberculosis+from+Multiple+Laboratories&doi=10.1128%2Fjb.00166-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Variation among genome sequences of H37Rv strains of Mycobacterium tuberculosis from multiple laboratories</span></div><div class="casAuthors">Ioerger, Thomas R.; Feng, Yicheng; Ganesula, Krishna; Chen, Xiaohua; Dobos, Karen M.; Fortune, Sarah; Jacobs, William R., Jr.; Mizrahi, Valerie; Parish, Tanya; Rubin, Eric; Sassetti, Chris; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bacteriology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">192</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3645-3653</span>CODEN:
                <span class="NLM_cas:coden">JOBAAY</span>;
        ISSN:<span class="NLM_cas:issn">0021-9193</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The publication of the complete genome sequence for Mycobacterium tuberculosis H37Rv in 1998 has had a great impact on the research community.  Nonetheless, it is suspected that genetic differences have arisen in stocks of H37Rv that are maintained in different labs.  In order to assess the consistency of the genome sequences among H37Rv strains in use and the extent to which they have diverged from the original strain sequenced, we carried out whole-genome sequencing on 6 strains of H37Rv from different labs.  Polymorphisms at 73 sites were obsd., which were shared among the lab strains, though 72 of these were also shared with H37Ra and are likely to be due to sequencing errors in the original H37Rv ref. sequence.  An updated H37Rv genome sequence should be valuable to the tuberculosis research community as well as the broader microbial research community.  In addn., several polymorphisms unique to individual strains and several shared polymorphisms were identified and shown to be consistent with the known provenance of these strains.  Aside from nucleotide substitutions and insertion/deletions, multiple IS6110 transposition events were obsd., supporting the theory that they play a significant role in plasticity of the M. tuberculosis genome.  This genome-wide catalog of genetic differences can help explain any phenotypic differences that might be found, including a frameshift mutation in the mycocerosic acid synthase gene which causes two of the strains to be deficient in biosynthesis of the surface glycolipid phthiocerol dimycocerosate (PDIM).  The resequencing of these 6 lab strains represents a fortuitous "in vitro evolution" expt. that demonstrates how the M. tuberculosis genome continues to evolve even in a controlled environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD6qz-dULL8rVg90H21EOLACvtfcHk0liC251t0b-n8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVCktLfE&md5=19086a827c0d86ac6b734fcc9dcb033b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1128%2Fjb.00166-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fjb.00166-10%26sid%3Dliteratum%253Aachs%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DGanesula%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDobos%26aufirst%3DK.%2BM.%26aulast%3DFortune%26aufirst%3DS.%26aulast%3DJacobs%26aufirst%3DW.%2BR.%26aulast%3DMizrahi%26aufirst%3DV.%26aulast%3DParish%26aufirst%3DT.%26aulast%3DRubin%26aufirst%3DE.%26aulast%3DSassetti%26aufirst%3DC.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DVariation%2520among%2520Genome%2520Sequences%2520of%2520H37Rv%2520Strains%2520of%2520Mycobacterium%2520tuberculosis%2520from%2520Multiple%2520Laboratories%26jtitle%3DJ.%2520Bacteriol.%26date%3D2010%26volume%3D192%26spage%3D3645%26epage%3D3653%26doi%3D10.1128%2Fjb.00166-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S.</span>; <span class="NLM_string-name">Lee, H.
S.</span>; <span class="NLM_string-name">Franzblau, S.</span></span> <span> </span><span class="NLM_article-title">Microplate Alamar Blue Assay (MABA) and Low Oxygen Recovery Assay (LORA) for <i>Mycobacterium tuberculosis</i></span>. In  <i>Mycobacteria Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parish, T.</span>, <span class="NLM_string-name">Roberts, D. M.</span></span>, Eds.; <span class="NLM_publisher-name">Springer New York</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">281</span>– <span class="NLM_lpage">292</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1007%2F978-1-4939-2450-9_17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=281-292&author=S.+Cho&author=H.%0AS.+Lee&author=S.+Franzblauauthor=T.+Parish&author=D.+M.+Roberts&title=Mycobacteria+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-2450-9_17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-2450-9_17%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DS.%26atitle%3DMicroplate%2520Alamar%2520Blue%2520Assay%2520%2528MABA%2529%2520and%2520Low%2520Oxygen%2520Recovery%2520Assay%2520%2528LORA%2529%2520for%2520Mycobacterium%2520tuberculosis%26btitle%3DMycobacteria%2520Protocols%26aulast%3DParish%26aufirst%3DT.%26pub%3DSpringer%2520New%2520York%26date%3D2015%26spage%3D281%26epage%3D292" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blondin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serina, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmuller, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barzu, O.</span></span> <span> </span><span class="NLM_article-title">Improved Spectrophotometric Assay of Nucleoside Monophosphate Kinase Activity Using the Pyruvate Kinase/Lactate Dehydrogenase Coupling System</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>220</i></span>,  <span class="NLM_fpage">219</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1006/abio.1994.1326</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1006%2Fabio.1994.1326" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=7978251" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK2cXlslenu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=220&publication_year=1994&pages=219-221&author=C.+Blondinauthor=L.+Serinaauthor=L.+Wiesmullerauthor=A.+M.+Gillesauthor=O.+Barzu&title=Improved+Spectrophotometric+Assay+of+Nucleoside+Monophosphate+Kinase+Activity+Using+the+Pyruvate+Kinase%2FLactate+Dehydrogenase+Coupling+System&doi=10.1006%2Fabio.1994.1326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Improved spectrophotometric assay of nucleoside monophosphate kinase activity using the pyruvate kinase/lactate dehydrogenase coupling system</span></div><div class="casAuthors">Blondin, Catherine; Serina, Lidia; Wiesmueller, Lisa; Gilles, Anne-Marie; Barzu, Octavian</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">220</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">219-21</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">The title assay couples the formation of nucleoside diphosphates to the reaction catalyzed by pyruvate kinase and lactate dehydrogenase in the presence of phosphenolpyruvate and NADH as indicator.  The reactivity of various nucleoside diphosphates toward pyruvate kinase varies within 1-3 order of magnitude, and, hence, the coupling system may become rate limiting.  As a consequence, nucleoside mnophosphate kinase activity might by underestimate by a factor of 1 or 2.  However, in the presence of pure nucleoside diphosphate kinase, the 2:1 ration of NADH consumed/nucleoside monophosphate added is easily attained both for CMP and dCMP.  As nucleoside diphosphate kinase does not interfere with the activity of nucleoside monophosphate kinase activity in the presence of pyruvate kinase and nucleoside diphosphate kinase as coupling enzymes, this kinase was added to the assay mixt.  The reliability of the modified spectrophotometric assay in crude or pure preps. of enzyme was related to the rate of phosphorylation of the various nucleoside diphosphate products of the coupling system.  This rate was significantly increased by addn. of pure nucleoside diphosphate kinase, which might reduce the consumption of pyruvate kinase.  The overall cost of this improved procedure remained unchanged.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVctLrLvffA7Vg90H21EOLACvtfcHk0liC251t0b-n8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXlslenu7w%253D&md5=fe31ee6e5636de62bb3f8218de8864d6</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1006%2Fabio.1994.1326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1994.1326%26sid%3Dliteratum%253Aachs%26aulast%3DBlondin%26aufirst%3DC.%26aulast%3DSerina%26aufirst%3DL.%26aulast%3DWiesmuller%26aufirst%3DL.%26aulast%3DGilles%26aufirst%3DA.%2BM.%26aulast%3DBarzu%26aufirst%3DO.%26atitle%3DImproved%2520Spectrophotometric%2520Assay%2520of%2520Nucleoside%2520Monophosphate%2520Kinase%2520Activity%2520Using%2520the%2520Pyruvate%2520Kinase%252FLactate%2520Dehydrogenase%2520Coupling%2520System%26jtitle%3DAnal.%2520Biochem.%26date%3D1994%26volume%3D220%26spage%3D219%26epage%3D221%26doi%3D10.1006%2Fabio.1994.1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cer, R. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mudunuri, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebeda, F. J.</span></span> <span> </span><span class="NLM_article-title">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">W441</span>– <span class="NLM_lpage">W445</span>, <span class="refDoi"> DOI: 10.1093/nar/gkp253</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1093%2Fnar%2Fgkp253" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=19395593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2009&pages=W441-W445&author=R.+Z.+Cerauthor=U.+Mudunuriauthor=R.+Stephensauthor=F.+J.+Lebeda&title=IC50-to-Ki%3A+a+web-based+tool+for+converting+IC50+to+Ki+values+for+inhibitors+of+enzyme+activity+and+ligand+binding&doi=10.1093%2Fnar%2Fgkp253"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">IC50-to-Ki: a web-based tool for converting IC50 to Ki values for inhibitors of enzyme activity and ligand binding</span></div><div class="casAuthors">Cer, R. Z.; Mudunuri, U.; Stephens, R.; Lebeda, F. J.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">Web Server</span>),
    <span class="NLM_cas:pages">W441-W445</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A new web-server tool ests. Ki values from exptl. detd. IC50 values for inhibitors of enzymes and of binding reactions between macromols. (e.g., proteins, nucleic acids) and ligands.  This converter was developed to enable end-users to help gauge the quality of the underlying assumptions used in these calcns. which depend on the type of mechanism of inhibitor action and the concns. of the interacting mol. species.  Addnl. calcns. are performed for nonclassical, tightly bound inhibitors of enzyme-substrate or of macromol.-ligand systems in which free, rather than total concns. of the reacting species are required.  Required user-defined input values include the total enzyme (or another target mol.) and substrate (or ligand) concns., the Km of the enzyme-substrate (or the Kd of the target-ligand) reaction, and the IC50 value.  Assumptions and caveats for these calcns. are discussed along with examples taken from the literature.  The host database for this converter contains kinetic consts. and other data for inhibitors of the proteolytic clostridial neurotoxins (http://botdb.abcc.ncifcrf.gov/toxin/kiConverter.jsp).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9gPXTwGaXwLVg90H21EOLACvtfcHk0lhMKI_5wmm_iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosFSku7g%253D&md5=6b0ffe268d709ad8f80da6df29a9565d</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp253%26sid%3Dliteratum%253Aachs%26aulast%3DCer%26aufirst%3DR.%2BZ.%26aulast%3DMudunuri%26aufirst%3DU.%26aulast%3DStephens%26aufirst%3DR.%26aulast%3DLebeda%26aufirst%3DF.%2BJ.%26atitle%3DIC50-to-Ki%253A%2520a%2520web-based%2520tool%2520for%2520converting%2520IC50%2520to%2520Ki%2520values%2520for%2520inhibitors%2520of%2520enzyme%2520activity%2520and%2520ligand%2520binding%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2009%26volume%3D37%26spage%3DW441%26epage%3DW445%26doi%3D10.1093%2Fnar%2Fgkp253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yung-Chi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prusoff, W. H.</span></span> <span> </span><span class="NLM_article-title">Relationship between the inhibition constant (KI) and the concentration of inhibitor which causes 50 per cent inhibition (I50) of an enzymatic reaction</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">1973</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">3099</span>– <span class="NLM_lpage">3108</span>, <span class="refDoi"> DOI: 10.1016/0006-2952(73)90196-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2F0006-2952%2873%2990196-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=4202581" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=1973&pages=3099-3108&author=C.+Yung-Chiauthor=W.+H.+Prusoff&title=Relationship+between+the+inhibition+constant+%28KI%29+and+the+concentration+of+inhibitor+which+causes+50+per+cent+inhibition+%28I50%29+of+an+enzymatic+reaction&doi=10.1016%2F0006-2952%2873%2990196-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2F0006-2952%2873%2990196-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0006-2952%252873%252990196-2%26sid%3Dliteratum%253Aachs%26aulast%3DYung-Chi%26aufirst%3DC.%26aulast%3DPrusoff%26aufirst%3DW.%2BH.%26atitle%3DRelationship%2520between%2520the%2520inhibition%2520constant%2520%2528KI%2529%2520and%2520the%2520concentration%2520of%2520inhibitor%2520which%2520causes%252050%2520per%2520cent%2520inhibition%2520%2528I50%2529%2520of%2520an%2520enzymatic%2520reaction%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D1973%26volume%3D22%26spage%3D3099%26epage%3D3108%26doi%3D10.1016%2F0006-2952%2873%2990196-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span>; <span class="NLM_string-name">Minor, W.</span></span> <span> </span><span class="NLM_article-title">Processing of X-ray Diffraction Data Collected in Oscillation Mode</span>. In  <i>Methods in Enzymology</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C. W.,  Jr.</span>, <span class="NLM_string-name">Sweet, R. M.</span></span>, Eds.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">1997</span>; Vol.  <span class="NLM_volume">276</span>: Macromolecular Crystallography, Part A, pp  <span class="NLM_fpage">307</span>– <span class="NLM_lpage">326</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2FS0076-6879%2897%2976066-X" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1997&pages=307-326&author=Z.+Otwinowski&author=W.+Minorauthor=C.+W.+Carter&author=R.+M.+Sweet&title=Methods+in+Enzymology"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2FS0076-6879%2897%2976066-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252FS0076-6879%252897%252976066-X%26sid%3Dliteratum%253Aachs%26aulast%3DOtwinowski%26aufirst%3DZ.%26atitle%3DProcessing%2520of%2520X-ray%2520Diffraction%2520Data%2520Collected%2520in%2520Oscillation%2520Mode%26btitle%3DMethods%2520in%2520Enzymology%26aulast%3DCarter%26aufirst%3DC.%2BW.%26pub%3DAcademic%2520Press%26date%3D1997%26volume%3D276%26spage%3D307%26epage%3D326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Collaborative
Computational
Project, Number 4</span> <span> </span><span class="NLM_article-title">The CCP4
Suite: Programs for Protein Crystallography</span>.  <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume">50</span> (),  <span class="NLM_fpage">760</span>.<span class="refDoi"> DOI: 10.1107/s0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760&author=Collaborative%0AComputational%0AProject%2C+Number+4&title=The+CCP4%0ASuite%3A+Programs+for+Protein+Crystallography&doi=10.1107%2Fs0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2Fs0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%250ASuite%253A%2520Programs%2520for%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26doi%3D10.1107%2Fs0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span> <span> </span><span class="NLM_article-title">PHENIX: Building New Software for Automated Crystallographic Structure Determination</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1948</span>– <span class="NLM_lpage">1954</span>, <span class="refDoi"> DOI: 10.1107/s0907444902016657</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444902016657" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=12393927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2002&pages=1948-1954&author=P.+D.+Adamsauthor=R.+W.+Grosse-Kunstleveauthor=L.-W.+Hungauthor=T.+R.+Ioergerauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+J.+Readauthor=J.+C.+Sacchettiniauthor=N.+K.+Sauterauthor=T.+C.+Terwilliger&title=PHENIX%3A+Building+New+Software+for+Automated+Crystallographic+Structure+Determination&doi=10.1107%2Fs0907444902016657"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: building new software for automated crystallographic structure determination</span></div><div class="casAuthors">Adams, Paul D.; Grosse-Kunstleve, Ralf W.; Hung, Li Wei; Ioerger, Thomas R.; McCoy, Airlie J.; Moriarty, Nigel W.; Read, Randy J.; Sacchettini, James C.; Sauter, Nicholas K.; Terwilliger, Thomas C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">D58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1948-1954</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Munksgaard</span>)
        </div><div class="casAbstract">Structural genomics seeks to rapidly expand the no. of protein structures to ext. the max. amt. of information from genomic sequence databases.  The advent of several large-scale projects worldwide leads to many new challenges in the field of crystallog. macromol. structure detn.  A novel software package, called PHENIX (Python-based Hierarchical Environment for Integrated Xtallog.), is therefore being developed.  This new software will provide the necessary algorithms to proceed from reduced intensity data to a refined mol. model and to facilitate structure soln. for both the novice and expert crystallographer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOtuUfHXPmCbVg90H21EOLACvtfcHk0lhMKI_5wmm_iQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotVCksbc%253D&md5=9a3b66e1f7d064183f7a9ce108cb4e5e</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2Fs0907444902016657&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444902016657%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26atitle%3DPHENIX%253A%2520Building%2520New%2520Software%2520for%2520Automated%2520Crystallographic%2520Structure%2520Determination%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2002%26volume%3D58%26spage%3D1948%26epage%3D1954%26doi%3D10.1107%2Fs0907444902016657" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and development ofCoot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/s0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1107%2Fs0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+development+ofCoot&doi=10.1107%2Fs0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhT10Urq3vPAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2Fs0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252Fs0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520development%2520ofCoot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2Fs0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pettersen, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goddard, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couch, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenblatt, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrin, T. E.</span></span> <span> </span><span class="NLM_article-title">UCSF Chimera?A visualization system for exploratory research and analysis</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1605</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+Chimera%3FA+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lhT10Urq3vPAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520Chimera%253FA%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span> <i>PyMOL The PyMOL
Molecular Graphics System</i>, version
1.8; <span class="NLM_publisher-name">Schrödinger,
LLC</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PyMOL+The+PyMOL%0AMolecular+Graphics+System%2C+version%0A1.8%3B+Schr%C3%B6dinger%2C%0ALLC."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DPyMOL%2520The%2520PyMOL%250AMolecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%250ALLC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abagyan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totrov, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuznetsov, D.</span></span> <span> </span><span class="NLM_article-title">ICM?A new method for protein modeling and design: Applications to docking and structure prediction from the distorted native conformation</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">488</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1002/jcc.540150503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1002%2Fjcc.540150503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=1994&pages=488-506&author=R.+Abagyanauthor=M.+Totrovauthor=D.+Kuznetsov&title=ICM%3FA+new+method+for+protein+modeling+and+design%3A+Applications+to+docking+and+structure+prediction+from+the+distorted+native+conformation&doi=10.1002%2Fjcc.540150503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">ICM - a new method for protein modeling and design: applications to docking and structure prediction from the distorted native conformation</span></div><div class="casAuthors">Abagyan, Ruben; Totrov, Maxim; Kuznetsov, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">488-506</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    </div><div class="casAbstract">An efficient methodol., further referred to as internal coordinate modeling (ICM), for versatile modeling operations and global energy optimization on arbitrary fixed multimol. systems is described.  It is aimed at protein structure prediction, homol. modeling, mol. docking, NMR structure detn., and protein design.  The method uses and further develops a previously introduced approach to model biomol. structures in which bond lengths, bond angles, and torsion angles are considered as independent variables, any subset of them being fixed.  Here the authors simplify and generalize the basic description of the system, introduce the variable dihedral phase angle, and allow arbitrary connections of the mols. and conventional definition of the torsion angles.  Algorithms for calcn. of energy derivs. with respect to internal variables in the topol. tree of the system and for rapid evaluation of accessible surface are presented.  Multidimensional variables restraints are proposed to represent the statistical information about the torsion angle distributions in proteins.  To incorporate complex energy terms as solvation energy and electrostatics into a structure prediction procedure, a "double-energy" Monte Carlo minimization procedure in which these terms are omitted during the minimization state of the random step and included for the comparison with the previous conformation in a Markov chain is proposed and justified.  The ICM method is applied successfully to a mol. docking problem.  The procedure finds the correct parallel arrangement of 2 rigid helixes from a leucine zipper domain as the lowest-energy conformation (0.5 Å root mean square, RMS, deviation from the native structure) starting from completely random configuration.  Structures with antiparallel helixes or helixes staggered by one helix turn had energies higher by about 7 or 9 kcal/mol, resp.  Soft docking was also attempted.  A docking procedure allowing side-chain flexibility also converged to the parallel configuration starting from the helixes optimized individually.  To justify an internal coordinate approach to the structure prediction as opposed to a Cartesian one, energy hypersurfaces around the native structure of the squash seeds trypsin inhibitor were studied.  Torsion angle minimization from the optimal conformation randomly distorted up to the RMS deviation of 2.2 Å or angular RMS deviation of 10° restored the native conformation in most cases.  In contrast, Cartesian coordinate minimization did not reach the min. from deviations as small as 0.3 Å or 2°.  The authors conclude that the most promising detailed approach to the protein-folding problem would consist of some coarse global sampling strategy combined with the local energy minimization in the torsion coordinate space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6kscl2--38LVg90H21EOLACvtfcHk0lgXzQldnKDhLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXhtVWisA%253D%253D&md5=5f1a1836429860ded9012dab112c9c47</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fjcc.540150503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.540150503%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26aulast%3DTotrov%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DD.%26atitle%3DICM%253FA%2520new%2520method%2520for%2520protein%2520modeling%2520and%2520design%253A%2520Applications%2520to%2520docking%2520and%2520structure%2520prediction%2520from%2520the%2520distorted%2520native%2520conformation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1994%26volume%3D15%26spage%3D488%26epage%3D506%26doi%3D10.1002%2Fjcc.540150503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Franzblau, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzig, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madico, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Degnan, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cook, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quenzer, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilman, R. H.</span></span> <span> </span><span class="NLM_article-title">Rapid, Low-Technology MIC Determination with Clinical <i>Mycobacterium tuberculosis</i> Isolates by Using the Microplate Alamar Blue Assay</span>. <i>J. Clin. Microbiol</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">362</span>– <span class="NLM_lpage">366</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=9466742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADyaK1cXnsFCqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1998&pages=362-366&author=S.+G.+Franzblauauthor=R.+S.+Witzigauthor=J.+C.+McLaughlinauthor=P.+Torresauthor=G.+Madicoauthor=A.+Hernandezauthor=M.+T.+Degnanauthor=M.+B.+Cookauthor=V.+K.+Quenzerauthor=R.+M.+Fergusonauthor=R.+H.+Gilman&title=Rapid%2C+Low-Technology+MIC+Determination+with+Clinical+Mycobacterium+tuberculosis+Isolates+by+Using+the+Microplate+Alamar+Blue+Assay"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid, low-technology MIC determination with clinical Mycobacterium tuberculosis isolates by using the microplate Alamar Blue assay</span></div><div class="casAuthors">Franzblau, Scott G.; Witzig, Richard S.; Mclaughlin, James C.; Torres, Patricia; Madico, Guillermo; Hernandez, Antonio; Degnan, Michelle T.; Cook, Mary B.; Quenzer, Virginia K.; Ferguson, Robert M.; Gilman, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Microbiology</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">362-366</span>CODEN:
                <span class="NLM_cas:coden">JCMIDW</span>;
        ISSN:<span class="NLM_cas:issn">0095-1137</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">A colorimetric, microplate-based Alamar Blue assay (MABA) method was used to det. the MICs of isoniazid (INH), rifampin, streptomycin (SM), and ethambutol (EMB) for 34 Peruvian Mycobacterium tuberculosis isolates (including both pansensitive and multidrug-resistant strains) and the H37Rv strain by using bacterial suspensions prepd. directly from solid media.  Results for all isolates were available within 8 days.  Discordant results were obsd. on initial tests for 3 of 16 INH-susceptible isolates, 5 of 31 EMB-susceptible isolates, and 2 of 4 SM-resistant isolates (by the BACTEC 460 system).  The overall agreements between the MICs obtained by MABA and the results obtained with the BACTEC 460 system were 87.9% for initial results and 93.6% after retesting 12 of 17 samples with discrepant results.  Interpretation of MABA endpoints improved with tech. experience.  The MABA is a simple, rapid, low-cost, appropriate technol. which does not require expensive instrumentation and which makes use of a nontoxic, temp.-stable reagent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrofNKIH7Wtx7Vg90H21EOLACvtfcHk0lgXzQldnKDhLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnsFCqug%253D%253D&md5=a3889d3ace22326dd2f7fffc8dd858c4</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFranzblau%26aufirst%3DS.%2BG.%26aulast%3DWitzig%26aufirst%3DR.%2BS.%26aulast%3DMcLaughlin%26aufirst%3DJ.%2BC.%26aulast%3DTorres%26aufirst%3DP.%26aulast%3DMadico%26aufirst%3DG.%26aulast%3DHernandez%26aufirst%3DA.%26aulast%3DDegnan%26aufirst%3DM.%2BT.%26aulast%3DCook%26aufirst%3DM.%2BB.%26aulast%3DQuenzer%26aufirst%3DV.%2BK.%26aulast%3DFerguson%26aufirst%3DR.%2BM.%26aulast%3DGilman%26aufirst%3DR.%2BH.%26atitle%3DRapid%252C%2520Low-Technology%2520MIC%2520Determination%2520with%2520Clinical%2520Mycobacterium%2520tuberculosis%2520Isolates%2520by%2520Using%2520the%2520Microplate%2520Alamar%2520Blue%2520Assay%26jtitle%3DJ.%2520Clin.%2520Microbiol%26date%3D1998%26volume%3D36%26spage%3D362%26epage%3D366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krieger, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freundlich, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawandi, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gawandi, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraile, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huss, S. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lavandera, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ioerger, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sacchettini, J. C.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Discovery of Phenyl-diketo Acids as Potent Inhibitors of M. tuberculosis Malate Synthase</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1556</span>– <span class="NLM_lpage">1567</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2012.09.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=10.1016%2Fj.chembiol.2012.09.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=23261599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymur%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=1556-1567&author=I.+V.+Kriegerauthor=J.+S.+Freundlichauthor=V.+B.+Gawandiauthor=J.+P.+Robertsauthor=V.+B.+Gawandiauthor=Q.+Sunauthor=J.+L.+Owenauthor=M.+T.+Fraileauthor=S.+I.+Hussauthor=J.-L.+Lavanderaauthor=T.+R.+Ioergerauthor=J.+C.+Sacchettini&title=Structure-Guided+Discovery+of+Phenyl-diketo+Acids+as+Potent+Inhibitors+of+M.+tuberculosis+Malate+Synthase&doi=10.1016%2Fj.chembiol.2012.09.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Discovery of Phenyl-diketo Acids as Potent Inhibitors of M. tuberculosis Malate Synthase</span></div><div class="casAuthors">Krieger, Inna V.; Freundlich, Joel S.; Gawandi, Vijay B.; Roberts, Justin P.; Gawandi, Vidyadhar B.; Sun, Qingan; Owen, Joshua L.; Fraile, Maria T.; Huss, Sofia I.; Lavandera, Jose-Luis; Ioerger, Thomas R.; Sacchettini, James C.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1556-1567</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The glyoxylate shunt plays an important role in fatty acid metab. and has been shown to be crit. to survival of several pathogens involved in chronic infections.  For Mycobacterium tuberculosis (Mtb), a strain with a defective glyoxylate shunt was previously shown to be unable to establish infection in a mouse model.  We report the development of phenyl-diketo acid (PDKA) inhibitors of malate synthase (GlcB), one of two glyoxylate shunt enzymes, using structure-based methods.  PDKA inhibitors were active against Mtb grown on acetate, and overexpression of GlcB ameliorated this inhibition.  Crystal structures of complexes of GlcB with PDKA inhibitors guided optimization of potency.  A selected PDKA compd. demonstrated efficacy in a mouse model of tuberculosis.  The discovery of these PDKA derivs. provides chem. validation of GlcB as an attractive target for tuberculosis therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppNZdVaSD2jrVg90H21EOLACvtfcHk0lgXzQldnKDhLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymur%252FN&md5=26fe3172f027bc08f49e7cbe63db4775</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.09.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.09.018%26sid%3Dliteratum%253Aachs%26aulast%3DKrieger%26aufirst%3DI.%2BV.%26aulast%3DFreundlich%26aufirst%3DJ.%2BS.%26aulast%3DGawandi%26aufirst%3DV.%2BB.%26aulast%3DRoberts%26aufirst%3DJ.%2BP.%26aulast%3DGawandi%26aufirst%3DV.%2BB.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DOwen%26aufirst%3DJ.%2BL.%26aulast%3DFraile%26aufirst%3DM.%2BT.%26aulast%3DHuss%26aufirst%3DS.%2BI.%26aulast%3DLavandera%26aufirst%3DJ.-L.%26aulast%3DIoerger%26aufirst%3DT.%2BR.%26aulast%3DSacchettini%26aufirst%3DJ.%2BC.%26atitle%3DStructure-Guided%2520Discovery%2520of%2520Phenyl-diketo%2520Acids%2520as%2520Potent%2520Inhibitors%2520of%2520M.%2520tuberculosis%2520Malate%2520Synthase%26jtitle%3DChem.%2520Biol.%26date%3D2012%26volume%3D19%26spage%3D1556%26epage%3D1567%26doi%3D10.1016%2Fj.chembiol.2012.09.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PKM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2PKM','PDB','2PKM'); return false;">PDB: 2PKM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C67" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C67','PDB','6C67'); return false;">PDB: 6C67</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9N','PDB','6C9N'); return false;">PDB: 6C9N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9P','PDB','6C9P'); return false;">PDB: 6C9P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9Q" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9Q','PDB','6C9Q'); return false;">PDB: 6C9Q</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9R" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9R','PDB','6C9R'); return false;">PDB: 6C9R</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9V" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9V','PDB','6C9V'); return false;">PDB: 6C9V</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9S" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9S','PDB','6C9S'); return false;">PDB: 6C9S</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i38"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00020">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_00804"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00020">10.1021/acs.jmedchem.9b00020</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Schemes and reagents for chemical synthesis of intermediates and supporting figures and tables (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_001.pdf">jm9b00020_si_001.pdf (1.6 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00020/suppl_file/jm9b00020_si_002.csv">jm9b00020_si_002.csv (2.39 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Authors will release the atomic coordinates and experimental data upon article publication: MtbAdoK-<b>2</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C67">6C67</a>, MtbAdoK-<b>3</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9N">6C9N</a>, MtbAdoK-<b>4</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9P">6C9P</a>, MtbAdoK-<b>5</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9Q">6C9Q</a>, MtbAdoK-<b>6</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9R">6C9R</a>, MtbAdoK-<b>7</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9V">6C9V</a>, MtbAdoK-<b>17</b> complex-PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6C9S">6C9S</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00020&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00020%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-9" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00020" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679926dc2ae03c22","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
