
<html lang="en"     class="pb-page"  data-request-id="e9c6197b-9535-4070-b064-ded4a6cb8a90"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b00656;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2019.62.issue-17;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery and Optimization of 2-Arylquinazolin-4-ones into a Potent and Selective Tankyrase Inhibitor Modulating Wnt Pathway Activity" /></meta><meta name="dc.Creator" content="Hans-Peter  Buchstaller" /></meta><meta name="dc.Creator" content="Uwe  Anlauf" /></meta><meta name="dc.Creator" content="Dieter  Dorsch" /></meta><meta name="dc.Creator" content="Daniel  Kuhn" /></meta><meta name="dc.Creator" content="Martin  Lehmann" /></meta><meta name="dc.Creator" content="Birgitta  Leuthner" /></meta><meta name="dc.Creator" content="Djordje  Musil" /></meta><meta name="dc.Creator" content="Daniela  Radtki" /></meta><meta name="dc.Creator" content="Claudio  Ritzert" /></meta><meta name="dc.Creator" content="Felix  Rohdich" /></meta><meta name="dc.Creator" content="Richard  Schneider" /></meta><meta name="dc.Creator" content="Christina  Esdar" /></meta><meta name="dc.Description" content="Tankyrases 1 and 2 (TNKS1/2) are promising pharmacological targets that recently gained interest for anticancer therapy in Wnt pathway dependent tumors. 2-Aryl-quinazolinones were identified and op..." /></meta><meta name="Description" content="Tankyrases 1 and 2 (TNKS1/2) are promising pharmacological targets that recently gained interest for anticancer therapy in Wnt pathway dependent tumors. 2-Aryl-quinazolinones were identified and op..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 5, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00656" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00656" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00656" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00656" /></link>
        
    
    

<title>Discovery and Optimization of 2-Arylquinazolin-4-ones into a Potent and Selective Tankyrase Inhibitor Modulating Wnt Pathway Activity | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00656" /></meta><meta property="og:title" content="Discovery and Optimization of 2-Arylquinazolin-4-ones into a Potent and Selective Tankyrase Inhibitor Modulating Wnt Pathway Activity" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0011.jpeg" /></meta><meta property="og:description" content="Tankyrases 1 and 2 (TNKS1/2) are promising pharmacological targets that recently gained interest for anticancer therapy in Wnt pathway dependent tumors. 2-Aryl-quinazolinones were identified and optimized into potent tankyrase inhibitors through SAR exploration around the quinazolinone core and the 4′-position of the phenyl residue. These efforts were supported by analysis of TNKS X-ray and WaterMap structures and resulted in compound 5k, a potent, selective tankyrase inhibitor with favorable pharmacokinetic properties. The X-ray structure of 5k in complex with TNKS1 was solved and confirmed the design hypothesis. Modulation of Wnt pathway activity was demonstrated with this compound in a colorectal xenograft model in vivo." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00656"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00656">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00656&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00656&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00656&amp;href=/doi/10.1021/acs.jmedchem.9b00656" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 7897-7909</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00653" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00686" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Optimization of 2-Arylquinazolin-4-ones into a Potent and Selective Tankyrase Inhibitor Modulating Wnt Pathway Activity</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Hans-Peter Buchstaller</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hans-Peter Buchstaller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +49 6151-72-4092. E-mail<a href="/cdn-cgi/l/email-protection#b48e94dcd5dac799c4d1c0d1c69ad6c1d7dcc7c0d5d8d8d1c6f4d9d1c6d7dfd3c6dbc1c49ad7dbd9">: <span class="__cf_email__" data-cfemail="c0a8a1aeb3edb0a5b4a5b2eea2b5a3a8b3b4a1acaca5b280ada5b2a3aba7b2afb5b0eea3afad">[email protected]</span></a></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hans-Peter++Buchstaller">Hans-Peter Buchstaller</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0577-8319" title="Orcid link">http://orcid.org/0000-0002-0577-8319</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Uwe Anlauf</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Uwe Anlauf</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Uwe++Anlauf">Uwe Anlauf</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dieter Dorsch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dieter Dorsch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dieter++Dorsch">Dieter Dorsch</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel Kuhn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Kuhn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Kuhn">Daniel Kuhn</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Lehmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Lehmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Lehmann">Martin Lehmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Birgitta Leuthner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Birgitta Leuthner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Birgitta++Leuthner">Birgitta Leuthner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Djordje Musil</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Djordje Musil</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Djordje++Musil">Djordje Musil</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniela Radtki</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniela Radtki</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniela++Radtki">Daniela Radtki</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Claudio Ritzert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Claudio Ritzert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Claudio++Ritzert">Claudio Ritzert</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Felix Rohdich</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Felix Rohdich</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Felix++Rohdich">Felix Rohdich</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Richard Schneider</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Richard Schneider</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Schneider">Richard Schneider</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Christina Esdar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christina Esdar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christina++Esdar">Christina Esdar</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00656&amp;href=/doi/10.1021%2Facs.jmedchem.9b00656" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 7897–7909</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">August 5, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 April 2019</li><li><span class="item_label"><b>Published</b> online</span>5 August 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 September 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00656" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00656</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7897%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHans-Peter%2BBuchstaller%252C%2BUwe%2BAnlauf%252C%2BDieter%2BDorsch%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D17%26contentID%3Dacs.jmedchem.9b00656%26title%3DDiscovery%2Band%2BOptimization%2Bof%2B2-Arylquinazolin-4-ones%2Binto%2Ba%2BPotent%2Band%2BSelective%2BTankyrase%2BInhibitor%2BModulating%2BWnt%2BPathway%2BActivity%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7909%26publicationDate%3DSeptember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00656"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1329</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00656" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Optimization of 2-Arylquinazolin-4-ones into a Potent and Selective Tankyrase Inhibitor Modulating Wnt Pathway Activity&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Hans-Peter&quot;,&quot;last_name&quot;:&quot;Buchstaller&quot;},{&quot;first_name&quot;:&quot;Uwe&quot;,&quot;last_name&quot;:&quot;Anlauf&quot;},{&quot;first_name&quot;:&quot;Dieter&quot;,&quot;last_name&quot;:&quot;Dorsch&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Kuhn&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Lehmann&quot;},{&quot;first_name&quot;:&quot;Birgitta&quot;,&quot;last_name&quot;:&quot;Leuthner&quot;},{&quot;first_name&quot;:&quot;Djordje&quot;,&quot;last_name&quot;:&quot;Musil&quot;},{&quot;first_name&quot;:&quot;Daniela&quot;,&quot;last_name&quot;:&quot;Radtki&quot;},{&quot;first_name&quot;:&quot;Claudio&quot;,&quot;last_name&quot;:&quot;Ritzert&quot;},{&quot;first_name&quot;:&quot;Felix&quot;,&quot;last_name&quot;:&quot;Rohdich&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Schneider&quot;},{&quot;first_name&quot;:&quot;Christina&quot;,&quot;last_name&quot;:&quot;Esdar&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7897-7909&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00656&quot;},&quot;abstract&quot;:&quot;Tankyrases 1 and 2 (TNKS1/2) are promising pharmacological targets that recently gained interest for anticancer therapy in Wnt pathway dependent tumors. 2-Aryl-quinazolinones were identified and optimized into potent tankyrase inhibitors through SAR exploration around the quinazolinone core and the 4′-position of the phenyl residue. These efforts were supported by analysis of TNKS X-ray and WaterMap structures and resulted in compound 5k, a potent, selective tankyrase inhibitor with favorable pharmacokinetic properties. The X-ray structure of 5k in complex with TNKS1 was solved and confirmed the design hypothesis. Modulation of Wnt pathway activity was demonstrated with this compound in a colorectal xenograft model in vivo.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00656&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00656" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00656&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00656" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00656&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00656" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00656&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00656&amp;href=/doi/10.1021/acs.jmedchem.9b00656" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00656" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00656" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00656%26sid%3Dliteratum%253Aachs%26pmid%3D31381853%26genre%3Darticle%26aulast%3DBuchstaller%26date%3D2019%26atitle%3DDiscovery%2Band%2BOptimization%2Bof%2B2-Arylquinazolin-4-ones%2Binto%2Ba%2BPotent%2Band%2BSelective%2BTankyrase%2BInhibitor%2BModulating%2BWnt%2BPathway%2BActivity%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D17%26spage%3D7897%26epage%3D7909%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/jmcmar.2019.62.issue-17/20190912/jmcmar.2019.62.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Tankyrases 1 and 2 (TNKS1/2) are promising pharmacological targets that recently gained interest for anticancer therapy in Wnt pathway dependent tumors. 2-Aryl-quinazolinones were identified and optimized into potent tankyrase inhibitors through SAR exploration around the quinazolinone core and the 4′-position of the phenyl residue. These efforts were supported by analysis of TNKS X-ray and WaterMap structures and resulted in compound <b>5k</b>, a potent, selective tankyrase inhibitor with favorable pharmacokinetic properties. The X-ray structure of <b>5k</b> in complex with TNKS1 was solved and confirmed the design hypothesis. Modulation of Wnt pathway activity was demonstrated with this compound in a colorectal xenograft model <i>in vivo</i>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65018" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65018" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The two highly homologous human tankyrase isoforms, TNKS1 and TNKS2, are members of the poly(ADP-ribose) polymerase (PARP) family comprising 17 proteins that share a common catalytic PARP domain.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> These PARP proteins cleave NAD<sup>+</sup> into ADP-ribose and nicotinamide and transfer the ADP-ribose units onto their substrates, resulting in a post-translational modification referred to as PARsylation.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Recently, tankyrases were proposed as potential drug targets in cancer therapy.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Mechanistically, tankyrases mark axin proteins, the concentration-limiting components of the β-catenin destruction complex, for degradation by PARsylation and subsequent ubiquitination through the E3 ligase RNF146.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> Inhibition of tankyrase antagonizes the Wnt signal transduction pathway by stabilizing axin proteins and promoting β-catenin degradation.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> Therefore, inhibition of tankyrase activity appears to be a promising strategy for the treatment of cancer subtypes that depend on high Wnt pathway activity, like colorectal cancer.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a></div><div class="NLM_p">A diverse set of tankyrase inhibitors with a variety of different scaffolds has previously been reported, which can be classified into three different subgroups based on their binding mode.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Among these, the 2-phenyl-3,4-dihydroquinazolin-4-one scaffold was early in focus. The TNKS2 crystal structure of XAV939, which was a milestone in the discovery of tankyrase inhibitors, served as a basis for structure-based design efforts.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Obvious structural similarities with flavones were exploited by others for the development of a series of para-substituted 2-phenyl-3,4-dihydroquinazolin-4-ones, and comprehensive structural studies revealed several key interactions with the protein.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> This scaffold forms a hydrogen bond from the lactam NH of the pyrimidine ring to the carbonyl group of Gly1032, while the lactam carbonyl forms hydrogen bonds with the main chain amide of Gly1032 as well as with the hydroxy group of Ser1068. Additionally, the quinazolinone forms a π–π stacking interaction with Tyr1071, while the phenyl substituent fits well into a hydrophobic pocket lined together with Tyr1050, Tyr1060, His1031, and Pro1034. Interestingly, substituents at the para position of the distal phenyl ring yield a smaller hydrophobic tunnel, flanked by Phe1035 and Ile1075. While this interaction was intensively explored with a variety of residues, structure–activity relationships for the substitution pattern at the quinazolinone core have only been studied sparingly. For instance, it was described that derivatives with a methyl group at either the 6- or 7-position showed weak inhibitory activity against TNKS2.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> However, investigation of the impact of core substitution on tankyrase inhibition predominantly focused on the 8-position, because the applied model suggested that substitution would not be tolerated at the 5-, 6-, and 7-positions of the quinazolin-4-one and that only small groups at the 8-position may be acceptable. These studies revealed that methyl at this position appears to be optimum for potency.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> 2-Phenyl-3,4-dihydroquinazolin-4-one derivatives are the most studied classes of compounds as PARP inhibitors. Hence, for studies on the SAR of this chemotype on tankyrase inhibition determination of the selectivity over PARP1 was essential in order to identify structural features able to drive TNKS selectivity.<a onclick="showRef(event, 'ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref16">(14−16)</a></div><div class="NLM_p last">Herein, the identification of the 2-phenyl-quinazolin-4-one scaffold via biochemical HTS approach and the subsequent optimization of this chemotype into a potent, selective tankyrase inhibitor with favorable physicochemical profile and pharmacokinetic properties modulating Wnt pathway activity in a colorectal xenograft model is reported.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34254" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34254" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Identification and <i>in Vitro</i> Characterization of HTS Hits</h3><div class="NLM_p">The initial quinazoline hits were identified as active compounds against TNKS1 among other nicotinamide pocket binders in a high-throughput biochemical screen. The homogeneous TR-FRET assay monitored the inhibition of the physiological auto-PARsylation activity of tankyrase. In this HTS assay, 250 nM tankyrase was applied, thereby limiting the bottom IC<sub>50</sub> determination level to 100 nM. Thus, for further compound optimization, confirmation, and IC<sub>50</sub> determination, orthogonal more sensitive activity ELISA assays (TNKS1 and TNKS2) based on the same auto-PARsylation activity were developed. Selectivity of the compounds was determined in a PARP-1 assay, which was selected as a representative member of the PARP family. Cellular potency of tankyrase inhibitors was assessed by detection of axin2 accumulation using a quantitative antibody-based ELISA-like read-out. Since axin2 is marked for degradation upon PARsylation by tankyrase, the increase of axin2 protein upon inhibition of tankyrase can be used as surrogate for tankyrase activity in DLD1 cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> This cell line expresses a truncated version of the tumor suppressor APC, thereby triggering constitutive activation of the canonical Wnt pathway, and has already been used previously as valuable model by others to study the effects of tankyrase inhibitors on activity of canonical Wnt signaling.<a onclick="showRef(event, 'ref10 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref10 ref17 ref18">(10,17,18)</a></div><div class="NLM_p">Initial quinazolines were selected from the hit set of the original HTS due to their fragment-like structures and favorable efficiency indices (e.g., <b>1b</b>, LE = 0.52; LLE = 4.51 based on clogD). Properties of representative examples (<b>1a</b>–<b>c</b>) of the hit series are depicted in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Properties (Potency, Solubility, and Metabolic Stability) of Selected HTS Hits from Quinazolin-4-one Series</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0007.gif" alt="" id="gr1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Not determined.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Reference <a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a>.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">h, human microsomes; m, mouse microsomes.</p></div></div><div></div></div><div class="NLM_p last">Compounds <b>1a</b>–<b>c</b> showed significant TNKS1 and TNKS2 inhibition but were less potent (∼10-fold) in our hands compared to reported data.<a onclick="showRef(event, 'ref16 ref19'); return false;" href="javascript:void(0);" class="ref ref16 ref19">(16,19)</a> These hits were further evaluated and revealed to be inactive in the cellular mechanistic assay, poorly soluble, and metabolically unstable shown by <i>in vitro</i> microsomal assays. Moreover, the most potent tankyrase inhibitor <b>1b</b> was modestly selective (4-fold selective over PARP1). As such, efforts were concentrated on improving TNKS potency and PARP selectivity, while in parallel optimizing drug-like properties in order to deliver a compound for mechanistic evaluation by <i>in vivo</i> pharmacokinetic and pharmacodynamic (PK/PD) studies.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Chemistry: Part 1</h3><div class="NLM_p">The target quinazolin-4-ones were assembled by a classical method, which allowed the quick evaluation of SAR patterns and rapid optimization of the initial hits. This key condensation reaction of anthranilamides with substituted benzaldehydes in the presence of sodium disulfite at elevated temperature was widely tolerant of various functionality and provided quinazoline derivative <b>3b</b> intermediates <b>4a</b>–<b>m</b> in moderate to excellent yields (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Only reaction of 2-amino-5-fluorobenzamide with methyl 6-formylpyridine-3-carboxylate leading to <b>6</b> with a pyridyl substituent instead of the phenyl ring at position 2 afforded this intermediate in low yield. Subsequent functional group manipulation allowed the facile evaluation of SAR at the para position of the distal phenyl ring. For instance, the methyl ester function of <b>4a</b>–<b>m</b> and <b>6</b> could be transformed into a 1-hydroxy-1-methyl-ethyl moiety via a Grignard reaction leading to the final compounds <b>5a</b>–<b>m</b> and <b>7</b>. We observed a high variability of the yield of this reaction, which, however, did not correlate with the substitution pattern at the quinazolinone core. Preparation of two close analogs (<b>5n</b>, <b>5p</b>) of compound <b>5d</b> was accomplished by functionalization of the hydroxy group with excess ethane-1,2-diol or ethylene glycol monomethyl ether, respectively, under acidic conditions. Compound <b>5m</b> was derivatized with ethane-1,2-diol under the same conditions resulting in derivative <b>5o</b>.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0005.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-<i>t</i>-BuPhCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 3 h; (b) 4-COOMePhCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 2.5–24 h; (c) MeMgCl, CeCl<sub>3</sub>, THF; (d) HO(CH<sub>2</sub>)<sub>2</sub>OH, TsOH·H<sub>2</sub>O or HO(CH<sub>2</sub>)<sub>2</sub>OCH<sub>3</sub>, TsOH·H<sub>2</sub>O; (e) 4-COOMePyCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 24 h.</p></p></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structure–Activity Relationship: Part 1</h3><div class="NLM_p">The aim of the SAR study was to optimize tankyrase potency and selectivity over PARP1 and to improve physicochemical and pharmacokinetic properties in order to deliver a suitable compound for <i>in vivo</i> studies. Initial structural investigations concentrated on the 4′-position of the phenyl residue. The focus was predominantly directed toward determining the importance of the substitution pattern at the quinazolin-4-one core (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Exploration of the SAR commenced with derivative <b>3a</b>, which was demonstrated to be a potent tankyrase inhibitor with good selectivity over PARP-1 by others.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> According to the published crystal structure, the lipophilic <i>tert</i>-butyl group in the para position of the phenyl residue does not make additional interactions with the binding site but fits very well into the small hydrophobic tunnel, driving potency. The reported activity and selectivity (PARP1 35% inhibition at 10 μM) of this compound was confirmed in our biochemical assays, but more importantly, <b>3a</b> was the first compound of this series with micromolar activity in the cellular axin2 assay (EC<sub>50</sub> = 1.5 μM). Introduction of a fluoro substituent at the 6-position of the quinazolinone (<b>3b</b>) significantly improved biochemical and cellular activity, and the excellent selectivity over PARP1 is retained (>1000-fold). However, the poor metabolic stability of <b>3b</b> and the low solubility of both <i>tert</i>-butyl derivatives limited further evaluation in <i>in vivo</i> experiments. Further efforts focused on the identification of similarly potent TNKS inhibitors with an improved physicochemical profile. To this end, a few substituents at the 4-position of the distal phenyl residue were explored. Various substituents in this position were already known, and besides the <i>tert</i>-butyl and other lipophilic groups also more hydrophilic moieties were evaluated. Crystal structures of these hydrophilic substituted compounds revealed water molecules in this pocket.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> To analyze the thermodynamic properties of water molecules in the TNKS1 binding site, WaterMap calculations<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> were performed. Analysis of TNKS structures in complex with known nicotinamide pocket binders like XAV939 (TNKS1, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UH4">3UH4</a>) or compound <b>3a</b> (TNKS2, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AKU">5AKU</a>) revealed that the terminal group at the phenyl ring (e.g., CF<sub>3</sub> or <i>tert</i>-butyl) replaces up to three unstable water sites. In both cases, these unstable water sites were replaced by hydrophobic groups and thereby contributed positively to binding affinity by liberating water molecules into the solvent. A closer analysis of the hydrogen bond network of the WaterMap water clusters surrounding the <i>tert</i>-butyl group suggested that introduction of a hydroxyl group could potentially be beneficial by making additional contacts with water sites in this region (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Properties (Potency, Solubility, and Metabolic Stability) of 2-Phenyl-3,4-dihydroquinazolin-4-ones</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0008.gif" alt="" id="gr4" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="/" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">IC<sub>50</sub> (nM)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="/"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">R<sup>1</sup></th><th class="colsep0 rowsep0" align="center">R<sup>2</sup></th><th class="colsep0 rowsep0" align="center">R<sup>3</sup></th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center" char=".">TNKS1</th><th class="colsep0 rowsep0" align="center" char=".">TNKS2</th><th class="colsep0 rowsep0" align="center" char=".">PARP1</th><th class="colsep0 rowsep0" align="center" char=".">EC<sub>50</sub> (nM) DLD1 axin2</th><th class="colsep0 rowsep0" align="center" char="."><i>S</i> (μg/mL)</th><th class="colsep0 rowsep0" align="center" char="/">Cl<sub>int</sub> h/m<a class="ref internalNav" href="#t2fn3" aria-label="c">c</a> (μL/min/mg protein)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">7.9</td><td class="colsep0 rowsep0" align="char" char=".">8.7</td><td class="colsep0 rowsep0" align="char" char=".">>10000</td><td class="colsep0 rowsep0" align="char" char=".">1500</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="/">19/<10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3b</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">3.3</td><td class="colsep0 rowsep0" align="char" char=".">3.6</td><td class="colsep0 rowsep0" align="char" char=".">3900</td><td class="colsep0 rowsep0" align="char" char=".">310</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="/">70/>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">910</td><td class="colsep0 rowsep0" align="char" char=".">1300</td><td class="colsep0 rowsep0" align="char" char=".">38</td><td class="colsep0 rowsep0" align="char" char="/">15/46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">830</td><td class="colsep0 rowsep0" align="char" char=".">590</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char="/"><10/11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5c</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">50</td><td class="colsep0 rowsep0" align="char" char=".">4300</td><td class="colsep0 rowsep0" align="char" char=".">2000</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="/"><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5d</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">12.5</td><td class="colsep0 rowsep0" align="char" char=".">10.5</td><td class="colsep0 rowsep0" align="char" char=".">349</td><td class="colsep0 rowsep0" align="char" char=".">614</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char="/"><10/25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5e</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="char" char=".">1400</td><td class="colsep0 rowsep0" align="char" char=".">6300</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="/"><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5f</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">30</td><td class="colsep0 rowsep0" align="char" char=".">1100</td><td class="colsep0 rowsep0" align="char" char=".">620</td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char="/"><10/26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5g</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">410</td><td class="colsep0 rowsep0" align="char" char=".">1100</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="/">23/49</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5h</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">630</td><td class="colsep0 rowsep0" align="char" char=".">2100</td><td class="colsep0 rowsep0" align="char" char=".">153</td><td class="colsep0 rowsep0" align="char" char="/">12/45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5i</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">1.4</td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="/"><10/81</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5j</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">7.5</td><td class="colsep0 rowsep0" align="char" char=".">512</td><td class="colsep0 rowsep0" align="char" char=".">456</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char="/"><10/23</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5k</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">7.0</td><td class="colsep0 rowsep0" align="char" char=".">9.2</td><td class="colsep0 rowsep0" align="char" char=".">782</td><td class="colsep0 rowsep0" align="char" char=".">319</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char="/"><10/22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5l</b></td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">1300</td><td class="colsep0 rowsep0" align="char" char=".">610</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char="/"><10/<10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5m</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">0.8</td><td class="colsep0 rowsep0" align="char" char=".">0.5</td><td class="colsep0 rowsep0" align="char" char=".">60</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char="/"><10/180</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5n</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">2300</td><td class="colsep0 rowsep0" align="char" char=".">360</td><td class="colsep0 rowsep0" align="char" char=".">23</td><td class="colsep0 rowsep0" align="char" char="/"><10/32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5o</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">O(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char=".">1.5</td><td class="colsep0 rowsep0" align="char" char=".">790</td><td class="colsep0 rowsep0" align="char" char=".">79</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="/">37/180</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5p</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O(CH<sub>2</sub>)<sub>2</sub>OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">24</td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char=".">5700</td><td class="colsep0 rowsep0" align="char" char=".">760</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char="/">41/126</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="char" char=".">1000</td><td class="colsep0 rowsep0" align="char" char=".">700</td><td class="colsep0 rowsep0" align="char" char=".">2500</td><td class="colsep0 rowsep0" align="char" char=".">12000</td><td class="colsep0 rowsep0" align="char" char="."><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="/">15/<10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">All values are based at least on duplicates.</p></div><div class="footnote" id="t2fn2"><sup><sup>b</sup></sup><p class="last">Not determined.</p></div><div class="footnote" id="t2fn3"><sup><sup>c</sup></sup><p class="last">h, human microsomes; m, mouse microsomes.</p></div></div></div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. WaterMap analysis of TNKS2 in complex with <b>3a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AKU">5AKU</a>). Color scheme according to predicted Δ<i>G</i> gain when water site is displaced: red (unstable waters) with predicted Δ<i>G</i> gain >2 kcal/mol, stable waters in green with Δ<i>G</i> penalty <0 kcal/mol, yellow in between. Tyr1150, Gly1053, Ile1157, and Tyr1060 are omitted for clarity. Displayed are all water sites that have an overlap factor greater than 0.5 with <b>3a</b>. There are three water sites in close proximity to the <i>tert</i>-butyl group of <b>3a</b> (distance of 0.4, 0.6, and 1.2 Å of the respective methyl groups) that are replaced by the inhibitor. Two additional water sites from the second shell are shown interacting with one water site (shown by dashed lines). Design idea has been that this interaction can be mimicked by introduction of a hydroxy substituent instead of a methyl group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">This hypothesis was evaluated by exchanging one of the methyl groups from the <i>tert</i>-butyl residue with a hydroxy group. The resulting 1-hydroxy-1-methyl-ethyl residue retains the size of a <i>tert</i>-butyl group but has a significantly decreased lipophilicity (decrease of clogD from 3.4 (<b>3a</b>) to 2.6 (<b>5a</b>)). This minor structural change was the key design feature and sparked further optimization of this series. Matched pair analysis of <b>3a</b>/<b>5a</b> and <b>3b</b>/<b>5d</b>, respectively, revealed that biochemical and cellular potency of the more hydrophilic derivatives (<b>5a</b>, <b>5d</b>) was slightly reduced or even retained. More importantly, solubility and metabolic stability liabilities of previous molecules were overcome. These hydroxy derivatives possessed acceptable solubility and metabolic stability profiles. Consequently, compound <b>5d</b> represented a suitable compound for further characterization. Interestingly, the introduction of this hydrophilic moiety in <b>5a</b> and <b>5d</b> significantly enhanced binding to PARP1 (IC<sub>50</sub> < 1 μM), leading to lower overall selectivity as compared to the <i>tert</i>-butyl substituted derivatives. Subsequent evaluation of quinazolinone scaffold SAR was conducted with the terminal phenyl ring fixed with a 1-hydroxy-1-methyl-1-ethyl moiety. WaterMap analysis indicated that adding small residues in the 5-, 6-, and 8-position might enhance activity by displacing three unstable water molecules (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Initial SAR revealed that biochemical potency of the 5-fluoro derivative <b>5b</b> was similar to the unsubstituted derivative <b>5a</b>, but cellular activity was improved (∼2-fold) into the sub-micromolar range. The 5-chloro derivative <b>5c</b> was still a potent inhibitor of both enzymes with double-digit nanomolar IC<sub>50</sub> values, but the cellular potency was diminished. The impact of substitution of this position on selectivity was ambiguous because it was increased in the order F< H < Cl. The gain of cellular potency by introduction of a fluoro substituent at position 6 of the quinazolinone core was comparable to position 5. Compound <b>5d</b>, which was already discussed before, had a cellular EC<sub>50</sub> of 614 nM and was slightly less potent than the <i>tert</i>-butyl derivative <b>3b</b> and 2-fold more potent than the unsubstituted derivative <b>5a</b>. Introduction of a chlorine at this position (<b>5e</b>) caused a 20-fold loss of cellular potency. Comparable to the effect at position 5, the 6-fluoro substituted derivative (<b>5d</b>) was the least selective (28-fold over PARP1) within the series of investigated substituents. The <i>in vitro</i> profile of compound <b>5f</b> bearing a fluoro substituent at the 8-position was largely identical to the 5-fluoro derivative <b>5b</b>. At this position, a small decrease of potency by replacement of fluorine with chlorine (<b>5g</b>) was observed. However, the impact of halogen type on selectivity was reversed compared to position 5 with the best selectivity observed for the fluoro compound <b>5f</b> (∼80-fold). Thus, it can be summarized that a chloro substituent was tolerated at all three investigated sites resulting in still potent tankryase inhibitors, but a fluoro substituent was always superior especially with respect to cellular activity. The introduction of electron-donating groups like methyl or methoxy in the 8-position resulted in converse effects. Potency of the methoxy substituted derivative <b>5h</b> decreased compared to the unsubstituted derivative <b>5a</b>, while the methyl derivative <b>5i</b> was one of the most potent tankyrase inhibitors within this structural class. Although larger substituents can be accommodated in the 8-position, WaterMap analysis could not rationalize different activity for methyl and methoxy derivatives. Compound <b>5i</b> inhibited TNKS1 and TNKS2 in the single-digit nanomolar range (IC<sub>50</sub> = 1.2 and 1.4 nM, respectively) and showed a cellular EC<sub>50</sub> of 31 nM. Unfortunately, increase of lipophilicity (clogD 2.8) by this methyl group deteriorated solubility and stability in the mouse microsomal assay and made this compound overall less attractive. Monosubstitution at position 8 had a minor impact on isoform selectivity. Compounds with larger residues like chloro (<b>5g</b>) or methoxy (<b>5h</b>) at this position were 2–3-fold less selective than derivatives with smaller substituents like methyl or fluoro. Investigation of the impact of different substitution patterns on activity and properties by combination of the most interesting residues at each investigated position was the target of the preparation of the next set of compounds. 5,6-Difluoro derivative <b>5j</b> was more potent biochemically than the corresponding monofluoro derivatives <b>5b</b> and <b>5d</b>. The EC<sub>50</sub> of the induction of cellular axin2 by compound <b>5j</b> was determined to be 456 nM, which was in the same range as the determined value for derivative <b>5d</b> (EC<sub>50</sub> = 614 nM). Furthermore, <b>5j</b> was reasonably soluble and metabolically stable and was therefore another inhibitor with an attractive <i>in vitro</i> profile. The same was true for derivative <b>5k</b> with a 6,8-difluoro substitution pattern at the quinazolinone core. This compound was a single-digit nanomolar inhibitor of both tankyrases, was potent in the cellular assay (EC<sub>50</sub> = 319 nM), was metabolically stable, and was even slightly better soluble than <b>5j</b>. The third variant, the 5,8-difluoro substitution pattern, was less attractive because compound <b>5l</b> was inferior in all respects compared to the previous derivatives <b>5j</b> and <b>5k</b>. A tankyrase inhibitor with outstanding biochemical potency was the 6-fluoro-8-methyl derivative <b>5m</b>. According to our knowledge, this compound with even sub-nanomolar biochemical potency (TNKS1 IC<sub>50</sub> = 0.8 and TNKS2 IC<sub>50</sub> = 0.5 nM) is one of the most potent tankyrase inhibitors reported to date that solely binds to the nicotinamide pocket. Cellular potency (EC<sub>50</sub> = 37 nM) is also remarkable, but like <b>5i</b>, this compound was not further pursued due to its unfavorable physicochemical profile. The same was true for compound <b>5o</b>, a close analog of <b>5m</b> with a hydroxy-ethoxy residue, which was prepared as described for <b>5n</b> via acid-catalyzed condensation. Although <b>5o</b> was a potent inhibitor (TNKS1 IC<sub>50</sub> = 1.6; TNKS2 IC<sub>50</sub> = 1.5 nM; cellular EC<sub>50</sub> = 79 nM), increased lipophilicity resulted in decreased metabolic stability in microsomal assays. Substitution of the core at all three positions at once was also tolerated, and the resulting activities and properties resembled the profiles of the disubstituted derivatives <b>5j</b> and <b>5k</b> (data not shown). Additional evaluation of the WaterMap suggested that a larger hydrophilic moiety could replace additional water molecules in the binding cavity, which might increase potency. Moreover, the <i>in silico</i> prediction of a similar logD value for compound <b>5d</b> and, for example, <b>5n</b>, a close analogue functionalized with a hydroxy-ethyl group, sparked our interest in these derivatives. As such, two derivatives of compound <b>5d</b> were prepared by introduction of a hydroxy-ethoxy residue (<b>5n</b>) or a methoxy-ethoxy residue (<b>5p</b>) via acid-catalyzed condensation reaction. Both derivatizations delivered very potent tankyrase inhibitors with low double-digit nanomolar IC<sub>50</sub> values for both enzymes, but potency was not increased compared to <b>5d</b>. The above-mentioned water sites are relatively solvent exposed and apparently did not contribute as much as predicted to the overall binding affinity. It is interesting to note that the affinity to PARP1 decreased resulting in higher selectivity (∼230-fold). While the overall profile of <b>5p</b> bearing a methoxy-ethyl residue was inferior compared to <b>5d</b>, activity (TNKS1 IC<sub>50</sub> = 10 nM, TNKS2 IC<sub>50</sub> = 13 nM; EC<sub>50</sub> = 360 nM) and compound properties of <b>5n</b> were also very attractive. Biochemical and cellular data of derivative <b>7</b> were in the micromolar range and selectivity over PARP1 was least within the discussed set of compounds, which clearly demonstrated that the distal phenyl substituent could not be replaced by a 2-pyridyl residue.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Chemistry and SAR: Part 2</h3><div class="NLM_p">As the SAR of the quinazolinone scaffold has been largely mapped out, subsequent efforts were focused on improving the solubility of the current lead molecules through the introduction of basic side chains.</div><div class="NLM_p">Fortunately, the condensation reaction was also applicable to benzaldehydes bearing cyclic amines. The methyl-substituted piperazine derivatives <b>8a</b> and <b>8b</b> were accessible in moderate yields in this manner (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). In the case of the piperidine derivatives the Boc-protected building block was used. After acidic deprotection of the condensation products compounds <b>8c</b> and <b>8d</b> were obtained, which in turn were methylated to afford compounds <b>8e</b> and <b>8f</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). The potency of these derivatives demonstrated that large cyclic amines could be accommodated at the 4′-position (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>), but all matched pairs with 1-hydroxy-1-methyl-ethyl derivatives were less potent in biochemical and cellular assays. In addition, intrinsic solubility of the piperazine derivatives <b>8a</b> and <b>8b</b> and the methylpiperidine derivative <b>8f</b> was limited, which reduced their attractiveness. Slight solubility improvements compared to the 1-hydroxy-1-methyl-ethyl derivatives were only observed when the target compounds were isolated as salts, for example, <b>8d</b> as hydrochloride. Not surprisingly, however, introduction of basic motifs triggered inhibition of hERG. This unwanted activity was observed for <b>8d</b> (<i>K</i><sub>i</sub> = 6.8 μM), and therefore this compound was not considered for further investigation.</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-(4-methyl-piperazin-1-yl)-benzaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 2.5–24 h; (b) (i) 4-(4-formyl-phenyl)-piperidine-1-carboxylic acid <i>tert</i>-butyl ester, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 160 °C, 2 h; (ii) HCl (4 M in dioxane), CH<sub>2</sub>Cl<sub>2</sub>, rt, overnight; (c) CH<sub>2</sub>O, HCOOH, 80 °C, overnight.</p></p></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Properties (Potency, Solubility, and Metabolic Stability) of 2-Phenyl-3,4-dihydroquinazolin-4-ones with Basic Side Chains</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0009.gif" alt="" id="gr7" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0010.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">All values are based at least on duplicates.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">h, human; m, mouse.</p></div><div class="footnote" id="t3fn3"><sup><sup>c</sup></sup><p class="last">Hydrochloride.</p></div><div class="footnote" id="t3fn4"><sup><sup>d</sup></sup><p class="last">Trifluoroacetate.</p></div></div><div></div></div><div class="NLM_p last">The SAR of the quinazolinone scaffold has largely been explored and yielded molecules like <b>5d</b>, <b>5j</b>, and <b>5k</b> with promising potency and drug like properties (solubility and metabolic stability). From this set of optimized compounds with rather equivalent profiles, <b>5k</b> was selected and further characterized.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity of Compound <b>5k</b></h3><div class="NLM_p">To assess the target specificity of <b>5k</b>, biochemical inhibition of a broad panel of human PARP enzymes covering 70% of the entire family was determined. IC<sub>50</sub> values for TNKS1 (10 nM) and TNKS2 (7 nM), as well as for PARP1 (710 nM), were highly consistent with results from internal ELISA assays. Overall, only single-digit micromolar potency was observed for PARP2 and PARP3, but none of the other tested PARP enzymes were inhibited by <b>5k</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>), which indicated excellent selectivity of <b>5k</b> for TNKS1 and TNKS2 within the PARP family.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Selectivity of Compound <b>5k</b> against Members of the PARP Family<a class="ref internalNav" href="#tbl4-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center" char=".">TNKS1</th><th class="colsep0 rowsep0" align="center" char=".">TNKS2</th><th class="colsep0 rowsep0" align="center" char=".">PARP1</th><th class="colsep0 rowsep0" align="center" char=".">PARP2</th><th class="colsep0 rowsep0" align="center" char=".">PARP3</th><th class="colsep0 rowsep0" align="center" char=".">PARP6</th><th class="colsep0 rowsep0" align="center" char=".">PARP7</th><th class="colsep0 rowsep0" align="center" char=".">PARP8</th><th class="colsep0 rowsep0" align="center" char=".">PARP10</th><th class="colsep0 rowsep0" align="center" char=".">PARP11</th><th class="colsep0 rowsep0" align="center" char=".">PARP12</th><th class="colsep0 rowsep0" align="center" char=".">PARP15</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="char" char=".">0.01</td><td class="colsep0 rowsep0" align="char" char=".">0.007</td><td class="colsep0 rowsep0" align="char" char=".">0.71</td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char=".">1.9</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td><td class="colsep0 rowsep0" align="char" char=".">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl4-fn1"><div class="footnote" id="tbl4-fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values in μM.</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Crystal Structure of TNKS1 with Compound <b>5k</b></h3><div class="NLM_p">We were able to solve the X-ray structure of <b>5k</b> in complex with TNKS1 to a resolution of 1.2 Å (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a>), and indeed, the binding mode of <b>5k</b> was as expected (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The quinazolinone scaffold of this disubstituted derivative superimposes quite nicely with the unsubstituted core of <b>3a</b> (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AKU">5AKU</a> structure) and makes two hydrogen bonds with Gly1185 amide backbone and the side chain of Ser1221. Both fluoro substituents are well accommodated in the pocket and contribute to the increased potency by making additional van der Waals interactions. The fluoro atom in the 6-position addresses Lys1220 and Glu1291 and causes a shift of both residues enabling the observed contacts. The second fluoro substituent at position 8 interacts with Tyr1203 and Tyr1213 accompanied by a slight movement of both amino acids. Interestingly, the terminal 1-hydroxy-1-methyl-ethyl residue is rotated by approximately 60° compared to the <i>tert</i>-butyl group of <b>3a</b>, which allows the hydroxyl group to interact with the predicted water sites (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Indeed, these hydrogen bonds are observed in the X-ray structure of <b>5k</b> (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Thus, the determined binding mode confirmed the WaterMap design hypothesis that introduction of a polar hydroxyl group is able to address these water molecules and to maintain binding affinity.</div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0002.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode of <b>5k</b> in complex with TNKS1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a>). For clarity, amino acids 1201–1215 are not shown. The quinazolinone core makes three hydrogen bonds to Gly1185 and Ser1221, the terminal hydroxyl group exhibits two hydrogen bonds with water molecules (distances in Å).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> <i>In Vitro</i> and <i>in Vivo</i> Characterization of Compound <b>5k</b></h3><div class="NLM_p">To further confirm the mode of action of <b>5k</b>, cellular levels of tankyrase were analyzed, since it is well described that inhibition of tankyrase-mediated auto-PARsylation results in an accumulation of tankyrase protein.<a onclick="showRef(event, 'ref10 ref22'); return false;" href="javascript:void(0);" class="ref ref10 ref22">(10,22)</a> To this end, DLD1 cells were treated with different concentrations of <b>5k</b> for 24 h, which led to a dose-dependent increase of tankyrase protein abundance with an EC<sub>50</sub> of 320 nM. This is in the same potency range as the value for axin2 increase (EC<sub>50</sub> = 319 nM) although the absolute increase was higher for tankyrase protein (∼8-fold) compared to axin2 (∼3-fold) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). The favorable <i>in vitro</i> metabolic stability of <b>5k</b> observed in mouse as well as in human liver microsomes was further confirmed by stability testing in hepatocytes demonstrating high stability against phase I and II metabolic reactions (Cl<sub>int</sub> values of 15.2 and 1.25 μL/min/10<sup>6</sup> cells in mouse and human hepatocytes, respectively). Caco-2 permeability of <b>5k</b> was high (<i>P</i><sub>app</sub> a → b 40 × 10<sup>–6</sup> cm/s), and no efflux (efflux ratio of 0.9) was observed in this assay. Subsequently, <b>5k</b> was subjected to an <i>in vivo</i> PK study in mice and showed a favorable PK profile, that is, a low clearance of 0.41 L/h/kg and a moderate half-life of 1.4 h. Oral administration of 30 mg/kg of <b>5k</b> in mice resulted in a high bioavailability and suggested that free plasma level (fraction unbound, 8.6% free) exceeded the EC<sub>50</sub> value (319 nM) in the mechanistic cellular assay for a period of ≥20 h (data not shown), which emphasized the suitability of <b>5k</b> as a chemical tool for pharmacological <i>in vivo</i> evaluation.</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0003.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cellular mechanistic study of <b>5k</b> in DLD1 model. Cells were treated for 24 h with increasing concentrations of <b>5k</b> and protein abundance for tankyrase (TNKS1/TNKS2) and axin2 was determined in cell lysates using an antibody-based ELISA-like read-out.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> PK/PD Analysis of Compound <b>5k</b></h3><div class="NLM_p">To address the relationship between the pharmacokinetic (PK, concentration vs time) and pharmacodynamic (PD, effect vs time) effects of <b>5k</b>, we aimed to investigate modulation of Wnt pathway activity in the human colorectal cancer xenograft model DLD1. To this end, DLD1 tumor bearing mice were treated once with three different doses (10, 30, and 90 mg/kg) of <b>5k</b>. After 2, 4, 8, and 12 h, drug concentrations were determined in plasma and pharmacodynamic biomarkers were assessed in tumor samples.</div><div class="NLM_p">Exposure of <b>5k</b> increased in direct proportion to the dose (10, 30, and 90 mg/kg), and free plasma levels were above the cellular EC<sub>50</sub> of 319 nM for at least 12 h for the three tested doses (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). Inhibition of tankyrase activity was confirmed <i>ex vivo</i> by analyses of tankyrase and axin2 protein abundance in tumor lysates. A time dependent increase of both tankyrase (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b) and axin2 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c) protein levels with maximum effects after 12 h was already observed for the lowest dose of 10 mg/kg. Furthermore, we could show that <i>in vivo</i> inhibition of tankyrase translated into reduced canonical Wnt pathway activity of APC deficient colorectal cancer cells. Compared to control tumors, abundance of active nuclear β-catenin was significantly decreased in a time- and dose-dependent manner in tumors treated with <b>5k</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d). While <i>in vivo</i> PD effects have been reported for tankyrase inhibitors with different binding modes,<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> this is the first study that demonstrated concentration dependent <i>in vivo</i> target modulation of tankyrase activity with a nicotinamide pocket binder of the quinazolinone class in a human xenograft tumor model by quantitative assessment of pharmacodynamic biomarkers of the Wnt signaling pathway on protein level.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0004.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <i>In vivo</i> PK/PD study with <b>5k</b> in DLD1 xenograft model. Single treatment of DLD1 tumor bearing mice with 10, 30, and 90 mg/kg of <b>5k</b> for 2, 4, 8, and 12 h, respectively, using 5 mice per group. (a) Plasma concentration time profiles of <b>5k</b> are expressed as free concentrations calculated based on free fraction in mouse serum (<i>f</i><sub>ub</sub> 8.6%). <i>Ex vivo</i> lysates were analyzed for protein level of (b) tankyrase (TNKS1/TNKS2), (c) axin2, and (d) active nuclear β-catenin and were compared to untreated control tumor lysates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08641" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08641" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The identification and optimization of 2-aryl-quinazolin-4-ones into potent tankyrase inhibitors was achieved by exploration of the SAR around the quinazolinone core and 4′-position of the distal phenyl residue. While introduction of a 1-hydroxy-1-methyl-ethyl residue in the 4′-position was a key modification for the improvement of solubility and metabolic stability profiles of the compounds, biochemical as well as cellular potency was significantly increased by variation of core substitution. The lead compound <b>5k</b> identified through these efforts exhibited enhanced activity in biochemical and cellular assays, favorable selectivity over related PARP proteins, and acceptable pharmacokinetics in mice. Further, <b>5k</b> was evaluated <i>in vivo</i> in assays showing target modulation of tankyrase and subsequent reduction of Wnt pathway activity in a human xenograft tumor model. Overall, these findings provide the basis for the rationale design of further tankyrase inhibitors from different structural classes toward preclinical drug-like candidates for the treatment of patients with Wnt pathway driven tumors.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61360" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61360" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Chemical Synthesis</h3><div class="NLM_p">All solvents and reagents were obtained from commercial suppliers and were used without further purification. HPLC/MS spectra of the products were recorded on an Agilent 1100 HPLC system (1100 high pressure gradient pump, 1100 diode array detector, wavelength = 220 nm) interfaced to an Agilent 1100 mass spectrometer detector (positive mode). The following conditions were used: (A) column, Chromolith SpeedROD RP-18e, 50 × 4.6 mm<sup>2</sup>; gradient A/B = 96:4 to 0:100 in 3.3 min; flow rate, 2.40 mL/min; eluent A, water + 0.05% formic acid; eluent B, acetonitrile + 0.04% formic acid; (B) column, Chromolith SpeedROD RP-18e, 50 × 4.6 mm<sup>2</sup>; gradient A/B = 96:4 to 0:100 in 2.8 min; flow rate, 2.0 mL/min; eluent A, water + 0.05% formic acid; eluent B, acetonitrile + 0.04% formic acid; (C) column, Chromolith PerformanceROD RP-18e, 100 × 3 mm<sup>2</sup>; gradient A/B = 99:1 to 0:100 in 3.5 min; flow rate, 2.0 mL/min; eluent A, water + 0.05% formic acid; eluent B, acetonitrile + 0.04% formic acid; (D) column, Chromolith PerformanceROD RP-18e, 100 × 3 mm<sup>2</sup>; gradient A/B = 99:1 to 0:100 in 1.8 min; flow rate, 2.0 mL/min; eluent A, water + 0.05% formic acid; eluent B, acetonitrile + 0.04% formic acid. Proton magnetic resonance spectra (<sup>1</sup>H NMR) were recorded on a Bruker Avance II 400 (400 MHz, 1H) spectrometer or a Bruker DRX 500 spectrometer (500 MHz, 1H) using DMSO-<i>d</i><sub>6</sub> as solvent. Chemical shifts (δ) are reported in parts per million (ppm) relative to DMSO as internal reference. High resolution mass spectra were recorded on a Waters Micromass GCT Premier spectrometer (EI, 70 eV; accuracy <3 mDa; mass range 25–800 amu). Flash-chromatography was performed on an Isco CombiFlash Companion system using prepacked reversed-phase flash columns and mixtures of water/acetonitrile containing 0.1% HCOOH. The combined fractions were concentrated in vacuo; the aqueous residue was rendered basic and extracted with ethyl acetate. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. Preparative HPLC was performed on an Agilent system (1200 series) using a Chromolith prep RP-18e 100–25 column and mixtures of water/acetonitrile containing 0.1% HCOOH. The combined fractions were concentrated in vacuo; the aqueous residue was rendered basic and extracted with ethyl acetate. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. Final compounds were found to be >95% pure by HPLC and HPLC/MS analysis. Compound <b>3a</b> was purchased from a commercial supplier.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> 2-(4-<i>tert</i>-Butyl-phenyl)-6-fluoro-3H-quinazolin-4-one (<b>3b</b>)</h4><div class="NLM_p last">A suspension of 2-amino-5-fluorobenzamide <b>2b</b> (154.1 mg, 1.00 mmol), 4-<i>tert</i>-butyl-benzaldehyde (162.2 mg, 1.00 mmol), and sodium disulfite (194.0 mg, 1.02 mmol) in <i>N</i>,<i>N</i>-dimethylacetamide (13.0 mL) was heated to 150 °C and stirred at this temperature for 3 h. The reaction mixture was cooled to room temperature and poured into ice water. The resulting precipitate was collected by filtration, washed with water, and dried under vacuum to afford the title compound (276 mg, 93% yield) as a light gray solid. HPLC-MS <i>m</i>/<i>z</i>: 297.1 [M + H]. <i>R</i><sub>t</sub> (C): 3.10 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.59 (s, 1H), 8.15–8.09 (m, 2H), 7.81 (ddd, <i>J</i> = 8.9, 6.4, 4.0 Hz, 2H), 7.72 (td, <i>J</i> = 8.7, 3.0 Hz, 1H), 7.60–7.54 (m, 2H), 1.33 (s, 9H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>N<sub>2</sub>OF 296.1325; found 296.1326.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 4-(4-Oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4a</b>)</h4><div class="NLM_p">A suspension of 2-aminobenzamide (0.68 g; 5.00 mmol), methyl 4-formylbenzoate (0.82 g; 5.00 mmol), and sodium disulfite (0.95 g; 5.00 mmol) in <i>N</i>,<i>N</i>-dimethylacetamide (10.0 mL) was heated at 150 °C for 2.5 h, cooled to room temperature, and poured into ice water. The resulting precipitate was collected by filtration, washed with water, and dried under vacuum to afford compound <b>4a</b> as a solid (0.96 g; 68%). LC-MS <i>m</i>/<i>z</i>: 281.1 [M + H]. <i>R</i><sub>t</sub> (A): 1.95 min.</div><div class="NLM_p last">Intermediates <b>4b</b>–<b>4m</b> were obtained according to the procedure described for <b>4a</b>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-(5-Fluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4b</b>)</h4><div class="NLM_p last">Yield: 0.54 g (56%). LC-MS <i>m</i>/<i>z</i>: 299.1 [M + H]. <i>R</i><sub>t</sub> (B): 1.98 min.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-(5-Chloro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4c</b>)</h4><div class="NLM_p last">Yield: 0.66 g (83%). LC-MS <i>m</i>/<i>z</i>: 315.0 [M + H]. <i>R</i><sub>t</sub> (C): 2.62 min.</div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-(6-Fluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4d</b>)</h4><div class="NLM_p last">Yield: 2.80 g (90%). LC-MS <i>m</i>/<i>z</i>: 299.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.51 min.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 4-(6-Chloro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4e</b>)</h4><div class="NLM_p last">Yield: 1.35 g (84%). LC-MS <i>m</i>/<i>z</i>: 315.0 [M + H]. <i>R</i><sub>t</sub> (C): 2.73 min.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 4-(8-Fluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4f</b>)</h4><div class="NLM_p last">Yield: 1.19 g (88%). LC-MS <i>m</i>/<i>z</i>: 299.1 [M + H]. <i>R</i><sub>t</sub> (A): 2.05 min.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 4-(8-Chloro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4g</b>)</h4><div class="NLM_p last">Yield: 0.92 g (57%). LC-MS <i>m</i>/<i>z</i>: 315.0 [M + H]. <i>R</i><sub>t</sub> (C): 2.75 min.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 4-(8-Methoxy-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4h</b>)</h4><div class="NLM_p last">Yield: 0.73 g (52%). LC-MS <i>m</i>/<i>z</i>: 311.1 [M + H]. <i>R</i><sub>t</sub> (A): 1.84 min.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 4-(8-Methyl-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4i</b>)</h4><div class="NLM_p last">Yield: 1.47 g (75%). LC-MS <i>m</i>/<i>z</i>: 295.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.80 min.</div></div><div id="sec4_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-(5,6-Difluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4j</b>)</h4><div class="NLM_p last">Yield: 0.95 g (98%). LC-MS <i>m</i>/<i>z</i>: 317.0 [M + H]. <i>R</i><sub>t</sub> (C): 2.52 min.</div></div><div id="sec4_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-(6,8-Difluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4k</b>)</h4><div class="NLM_p last">Yield: 9.14 g (90%). LC-MS <i>m</i>/<i>z</i>: 317.0 [M + H]. <i>R</i><sub>t</sub> (D): 1.91 min.</div></div><div id="sec4_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-(5,8-Difluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4l</b>)</h4><div class="NLM_p last">Yield: 0.40 g (35%). LC-MS <i>m</i>/<i>z</i>: 317.0 [M + H]. <i>R</i><sub>t</sub> (D): 1.79 min.</div></div><div id="sec4_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-(6-Fluoro-8-methyl-4-oxo-3,4-dihydro-quinazolin-2-yl)-benzoic Acid Methyl Ester (<b>4m</b>)</h4><div class="NLM_p last">Yield: 0.42 g (77%). LC-MS <i>m</i>/<i>z</i>: 313.1 [M + H]. <i>R</i><sub>t</sub> (D): 2.03 min.</div></div><div id="sec4_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 2-[4-(1-Hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5a</b>)</h4><div class="NLM_p">Compound <b>4a</b> (955.5 g; 3.41 mmol) was suspended in THF (13.7 mL) and treated with cerium(III) chloride (882.1 mg; 3.58 mmol), and the mixture was stirred at ambient temperature for 4 h. Methylmagnesium chloride (20% solution in THF; 4.95 mL; 13.64 mmol) was added dropwise, and the reaction mixture was stirred for 1 h at ambient temperature. The reaction mixture was diluted with THF and quenched with 10% aqueous citric acid solution. The organic phase was separated, washed with brine, dried over sodium sulfate, filtered, and evaporated to dryness. The residue was purified by flash column chromatography to yield the title compound as colorless solid (310 mg; 32% yield). HPLC-MS <i>m</i>/<i>z</i>: 281.1 [M + H]. <i>R</i><sub>t</sub> (A): 1.71 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.47 (s, 1H), 8.18–8.10 (m, 3H), 7.83 (ddd, <i>J</i> = 8.6, 7.1, 1.6 Hz, 1H), 7.73 (ddd, <i>J</i> = 8.2, 1.2, 0.6 Hz, 1H), 7.67–7.59 (m, 2H), 7.51 (ddd, <i>J</i> = 8.1, 7.0, 1.2 Hz, 1H), 5.16 (s, 1H), 1.47 (s, 6H); HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>16</sub>N<sub>2</sub>O<sub>2</sub> 280.1212; found 280.1213.</div><div class="NLM_p last">Compounds <b>5b</b>–<b>5m</b> were obtained according to the procedure described for <b>5a</b>.</div></div><div id="sec4_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 5-Fluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5b</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 322 mg (60%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 299.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.11 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.45 (s, 1H), 8.16–8.10 (m, 2H), 7.79 (td, <i>J</i> = 8.2, 5.7 Hz, 1H), 7.67–7.60 (m, 2H), 7.54 (d, <i>J</i> = 8.0 Hz, 1H), 7.23 (ddd, <i>J</i> = 11.0, 8.2, 0.9 Hz, 1H), 5.15 (s, 1H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>F 298.1118; found 298.1116.</div></div><div id="sec4_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 5-Chloro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5c</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 482 mg (73%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 315.0 [M + H], <i>R</i><sub>t</sub> (C): 2.33 min. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.45 (s, 1H), 8.16–8.10 (m, 2H), 7.73 (t, <i>J</i> = 7.9 Hz, 1H), 7.69–7.60 (m, 3H), 7.49 (dd, <i>J</i> = 7.7, 1.3 Hz, 1H), 5.15 (s, 1H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>Cl 314.0822; found 314.0821.</div></div><div id="sec4_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-Fluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5d</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 1.09 g (71%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 299.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.22 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.57 (s, 1H), 8.15–8.10 (m, 2H), 7.85–7.78 (m, 2H), 7.71 (td, <i>J</i> = 8.7, 3.0 Hz, 1H), 7.65–7.61 (m, 2H), 5.14 (s, 1H), 1.47 (s, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.59, 159.83 (d, <i>J</i> = 245.3 Hz), 154.09, 151.75, 145.68, 130.18, 130.26–130.09 (m), 127.24, 124.74, 122.93 (d, <i>J</i> = 24.1 Hz), 122.06 (d, <i>J</i> = 8.2 Hz), 110.41 (d, <i>J</i> = 23.3 Hz), 70.67, 31.68. HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>F 298.1118; found 298.1115.</div></div><div id="sec4_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 6-Chloro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5e</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 853 mg (63%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 315.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.44 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.64 (s, 1H), 8.12 (d, <i>J</i> = 8.6 Hz, 2H), 8.09 (d, <i>J</i> = 2.4 Hz, 1H), 7.85 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H), 7.76 (d, <i>J</i> = 8.7 Hz, 1H), 7.63 (d, <i>J</i> = 8.6 Hz, 2H), 5.19 (s, 1H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>Cl 314.0822; found 314.0827.</div></div><div id="sec4_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 8-Fluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5f</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 165 mg (14%) beige solid. HPLC/MS <i>m</i>/<i>z</i>: 299.1 [M + H]. <i>R</i><sub>t</sub> (A): 1.81 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.63 (s, 1H), 8.17–8.11 (m, 2H), 7.96 (d, <i>J</i> = 8.0 Hz, 1H), 7.70 (ddd, <i>J</i> = 10.7, 8.0, 1.3 Hz, 1H), 7.67–7.61 (m, 2H), 7.49 (td, <i>J</i> = 8.0, 4.8 Hz, 1H), 5.17 (s, 1H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>F 298.1118; found 298.1120.</div></div><div id="sec4_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 8-Chloro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5g</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 802 mg (87%) beige solid. HPLC/MS <i>m</i>/<i>z</i>: 315.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.50 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.70 (s, 1H), 8.18 (d, <i>J</i> = 8.6 Hz, 2H), 8.11 (dd, <i>J</i> = 7.9, 1.4 Hz, 1H), 7.98 (dd, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.65 (d, <i>J</i> = 8.6 Hz, 2H), 7.47 (t, <i>J</i> = 7.8 Hz, 1H), 5.17 (s, 1H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>2</sub>O<sub>2</sub>Cl 314.0822; found 314.0820.</div></div><div id="sec4_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-[4-(1-Hydroxy-1-methyl-ethyl)-phenyl]-8-methoxy-3<i>H</i>-quinazolin-4-one (<b>5h</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 115 mg (16%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 311.1 [M + H]. <i>R</i><sub>t</sub> (A): 1.64 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.47 (s, 1H), 8.17–8.10 (m, 2H), 7.72 (dd, <i>J</i> = 7.8, 1.5 Hz, 1H), 7.67–7.61 (m, 2H), 7.45 (t, <i>J</i> = 7.9 Hz, 1H), 7.39 (dd, <i>J</i> = 8.1 Hz, 1.4, 1H), 5.16 (s, 1H), 3.96 (s, 3H), 1.48 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>O<sub>3</sub> 310.1317; found 310.1323.</div></div><div id="sec4_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-[4-(1-Hydroxy-1-methyl-ethyl)-phenyl]-8-methyl-3<i>H</i>-quinazolin-4-one (<b>5i</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 275 mg (47%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 295.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.52 min. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.45 (s, 1H), 8.19–8.14 (m, 2H), 8.01–7.97 (m, 1H), 7.71–7.67 (m, 1H), 7.63 (d, <i>J</i> = 8.5 Hz, 2H), 7.38 (t, <i>J</i> = 7.6 Hz, 1H), 5.14 (s, 1H), 2.62 (s, 3H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>N<sub>2</sub>O<sub>2</sub> 294.1368; found 294.1375.</div></div><div id="sec4_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 5,6-Difluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5j</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 530 mg (56%) beige solid. HPLC/MS <i>m</i>/<i>z</i>: 317.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.28 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.58 (s, 1H), 8.15–8.08 (m, 2H), 7.95–7.85 (m, 1H), 7.68–7.54 (m, 3H), 5.16 (s, 1H), 1.46 (s, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 159.32, 154.37, 152.54, 148.12 (dd, <i>J</i> = 68.9, 13.0 Hz), 146.37, 146.09 (dd, <i>J</i> = 50.4, 11.7 Hz), 129.71, 127.33, 124.75, 124.02–123.82 (m), 123.41 (d, <i>J</i> = 19.1 Hz), 111.61, 70.67, 31.67. HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub>F<sub>2</sub> 316.1023; found 316.1031.</div></div><div id="sec4_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 6,8-Difluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5k</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 3.83 g (71%) pale-yellow solid. HPLC/MS <i>m</i>/<i>z</i>: 317.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.37 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.76 (s, 1H), 8.12 (d, <i>J</i> = 8.5 Hz, 2H), 7.83 (ddd, <i>J</i> = 10.4, 9.1, 2.9 Hz, 1H), 7.71–7.67 (m, 1H), 7.64 (d, <i>J</i> = 8.5 Hz, 2H), 5.17 (s, 1H), 1.47 (s, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.73 (t, <i>J</i> = 3.6 Hz), 159.00 (dd, <i>J</i> = 205.8, 11.8 Hz), 156.99 (dd, <i>J</i> = 216.5, 11.8 Hz), 154.40, 152.35–152.29 (m), 135.43 (dd, <i>J</i> = 11.5, 2.3 Hz), 129.95, 127.40, 124.79, 123.26 (dd, <i>J</i> = 9.6, 2.2 Hz), 109.85 (dd, <i>J</i> = 28.2, 23.2 Hz), 106.54 (dd, <i>J</i> = 23.2, 4.3 Hz), 70.68, 31.67. HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub>F<sub>2</sub> 316.1023; found 316.1017.</div></div><div id="sec4_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 5,8-Difluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>5l</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 293 mg (74%) pale-yellow solid. HPLC/MS <i>m</i>/<i>z</i>: 317.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.68 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.64 (s, 1H), 8.17–8.10 (m, 2H), 7.70 (ddd, <i>J</i> = 10.0, 9.0, 4.3 Hz, 1H), 7.67–7.62 (m, 2H), 7.24 (ddd, <i>J</i> = 10.4, 9.0, 3.7 Hz, 1H), 5.16 (s, 1H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>2</sub>O<sub>2</sub>F<sub>2</sub> 316.1023; found 316.1019.</div></div><div id="sec4_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 6-Fluoro-2-[4-(1-hydroxy-1-methyl-ethyl)-phenyl]-8-methyl-3H-quinazolin-4-one (<b>5m</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 135 mg (27%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 313.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.89 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.58 (s, 1H), 8.18–8.12 (m, 2H), 7.68–7.59 (m, 4H), 5.15 (s, 1H), 2.64 (s, 3H), 1.47 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>N<sub>2</sub>O<sub>2</sub>F 312.1274; found 312.1281.</div></div><div id="sec4_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 6-Fluoro-2-{4-[1-(2-hydroxy-ethoxy)-1-methyl-ethyl]-phenyl}-3<i>H</i>-quinazolin-4-one (<b>5n</b>)</h4><div class="NLM_p last">To a suspension of <b>5d</b> (149.0 mg, 0.50 mmol) in ethane-1,2-diol (2 mL), toluene-4-sulfonic acid monohydrate (114.0 mg, 0.60 mmol) was added, and the reaction mixture was stirred for 3 h at ambient temperature. The suspension was then heated to 80 °C, and the resulting clear solution was stirred at this temperature for 5 h. The reaction mixture was cooled to room temperature, and water was added. The resulting precipitate was filtered off and washed with water. The residue was purified by chromatography (silica gel column; eluent cyclohexane/ethyl acetate) to afford the title compound (82 mg, 48% yield) as colorless crystals. HPLC/MS <i>m</i>/<i>z</i>: 343.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.28 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 8.15 (d, <i>J</i> = 8.6 Hz, 2H), 7.82 (m, 2H), 7.72 (td, <i>J</i> = 8.7, 3.0 Hz, 1H), 7.61 (d, <i>J</i> = 8.6 Hz, 2H), 4.58 (t, <i>J</i> = 5.7 Hz, 1H), 3.50 (q, <i>J</i> = 5.6 Hz, 2H), 3.19 (t, <i>J</i> = 5.6 Hz, 2H), 1.51 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>N<sub>2</sub>O<sub>3</sub>F 342.1380; found 342.1391.</div></div><div id="sec4_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 6-Fluoro-2-{4-[1-(2-hydroxy-ethoxy)-1-methyl-ethyl]-phenyl}-8-methyl-3<i>H</i>-quinazolin-4-one (<b>5o</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5n</b> using compound <b>5m</b>. Yield: 7 mg (6%) beige solid. HPLC/MS <i>m</i>/<i>z</i>: 357.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.90 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 8.21 (d, <i>J</i> = 8.5 Hz, 1H), 7.73–7.44 (m, 4H), 4.56 (t, <i>J</i> = 5.7 Hz, 1H), 3.52 (q, <i>J</i> = 5.6 Hz, 2H), 3.21 (t, <i>J</i> = 5.6 Hz, 2H), 2.65 (s, 3H), 1.52 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>2</sub>O<sub>3</sub>F 356.1536; found 356.1537.</div></div><div id="sec4_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 6-Fluoro-2-{4-[1-(2-methoxy-ethoxy)-1-methyl-ethyl]-phenyl}-3<i>H</i>-quinazolin-4-one (<b>5p</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5n</b> using ethylene glycol monomethyl ether. Yield: 64 mg (35%) colorless solid. HPLC/MS <i>m</i>/<i>z</i>: 357.2 [M + H]. <i>R</i><sub>t</sub> (C): 2.61 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.60 (s, 1H), 8.15 (m, 2H), 7.82 (m, 2H), 7.72 (td, <i>J</i> = 8.7, 3.0 Hz, 1H), 7.59 (m, 2H), 3.45 (dd, <i>J</i> = 5.7, 4.2 Hz, 2H), 3.29 (dd, <i>J</i> = 5.7, 4.2 Hz, 2H), 3.26 (s, 3H), 1.51 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>2</sub>O<sub>3</sub>F 356.1536; found 356.1544.</div></div><div id="sec4_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 6-(6-Fluoro-4-oxo-3,4-dihydro-quinazolin-2-yl)-nicotinic Acid Methyl Ester (<b>6</b>)</h4><div class="NLM_p last">Intermediate <b>6</b> was obtained according to the procedure described for <b>4a</b>. Yield: 0.30 g (17%). LC-MS <i>m</i>/<i>z</i>: 300.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.58 min.</div></div><div id="sec4_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 6-Fluoro-2-[5-(1-hydroxy-1-methyl-ethyl)-pyridin-2-yl]-3<i>H</i>-quinazolin-4-one (<b>7</b>)</h4><div class="NLM_p last">Preparation as described for compound <b>5a</b>. Yield: 177 mg (59%) yellow solid. HPLC/MS <i>m</i>/<i>z</i>: 300.1 [M + H]. <i>R</i><sub>t</sub> (C): 2.30 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.96 (s, 1H), 8.86 (d, <i>J</i> = 1.6, 1H), 8.38 (d, <i>J</i> = 8.3 Hz, 1H), 8.12 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.87 (m, 2H), 7.76 (td, <i>J</i> = 8.7, 3.1 Hz, 1H), 5.42 (s, 1H), 1.52 (s, 6H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>16</sub>H<sub>14</sub>N<sub>3</sub>O<sub>2</sub>F 299.1070; found 299.1058.</div></div><div id="sec4_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 6-Fluoro-2-[4-(4-methyl-piperazin-1-yl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>8a</b>)</h4><div class="NLM_p last">A solution of 2-amino-5-fluoro-benzamide (77.1 mg; 0.50 mmol), 4-(4-methyl-piperazin-1-yl)-benzaldehyde (102.1 mg; 0.50 mmol), and sodium disulfite (0.07 mL; 0.51 mmol) in <i>N</i>,<i>N</i>-dimethylacetamide (1 mL) was heated to 150 °C, stirred at this temperature for 3.5 h, cooled to ambient temperature, and stirred overnight. The reaction mixture was quenched with water. The resulting precipitate was collected by filtration, washed with water, triturated with hot <i>tert</i>-butyl methyl ether, filtered by suction, washed with <i>tert</i>-butyl methyl ether, and dried in vacuo to afford 117 mg (69%) as a brown solid. HPLC/MS <i>m</i>/<i>z</i>: 339.1 [M + H]. <i>R</i><sub>t</sub> (C): 1.67 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 8.10 (d, <i>J</i> = 9.1 Hz, 2H), 7.76 (m, 2H), 7.67 (td, <i>J</i> = 8.7, 3.0 Hz, 1H), 7.04 (d, <i>J</i> = 9.1 Hz, 2H), 3.32 (rn, 4H), 2.47 (m, 4H), 2.25 (s, 3H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>N<sub>4</sub>OF 338.1543; found 338.1542.</div></div><div id="sec4_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 6-Fluoro-8-methyl-2-[4-(4-methyl-piperazin-1-yl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>8b</b>)</h4><div class="NLM_p last">A solution of 2-amino-5-fluoro-3-methyl-benzamide (84.1 mg, 0.50 mmol), 4-(4-methyl-piperazin-1-yl)-benzaldehyde (123.0 mg, 0.60 mmol), and sodium disulfite (97.0 mg, 0.51 mmol) in <i>N</i>-methyl-pyrrolidone (1 mL) is heated to 150 °C, stirred at this temperature for 16 h, cooled to ambient temperature, and poured into ice water. The resulting precipitate is collected by filtration, washed with water, and dried. The crude product is purified by chromatography (silica gel column, eluent methanol/dichloromethane). Yield: 70 mg (40%) brown solid. HPLC/MS <i>m</i>/<i>z</i>: 353.2 [M + H]. <i>R</i><sub>t</sub> (D): 1.47 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.34 (s, 1H), 8.69–7.90 (m, 2H), 7.86–7.26 (m, 2H), 7.28–6.73 (m, 2H), 3.33–3.27 (m, 4H), 2.61 (s, 3H), 2.45 (t, <i>J</i> = 5.1 Hz, 4H), 2.23 (s, 3H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>4</sub>OF 352.1699; found 352.1701.</div></div><div id="sec4_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 6,8-Difluoro-2-(4-piperidin-4-yl-phenyl)-3<i>H</i>-quinazolin-4-one Hydrochloride (<b>8c</b>)</h4><div class="NLM_p last">Preparation of the Boc-protected intermediate was performed according to the procedure described for <b>8b</b>. The obtained precipitate after trituration with methanol and filtration by suction was slurried in a solution of HCI in dioxane (4 N, 2 mL) and dichloromethane (5 mL), and the mixture was stirred for 16 h at room temperature. The reaction mixture was evaporated, and the residue was purified by preparative HPLC. Yield: 242 mg (55%) pale-brown solid. HPLC/MS <i>m</i>/<i>z</i>: 342.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.44 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, CF<sub>3</sub>COOD) δ 8.35–8.17 (m, 2H), 7.72 (ddd, <i>J</i> = 8.3, 2.9, 1.4 Hz, 1H), 7.61 (ddd, <i>J</i> = 10.2, 8.9, 2.9 Hz, 1H), 7.52–7.31 (m, 2H), 3.61–3.38 (m, 2H), 3.08 (td, <i>J</i> = 12.8, 3.2 Hz, 2H), 3.00 (tt, <i>J</i> = 11.8, 3.9 Hz, 1H), 2.14–2.01 (m, 2H), 2.01–1.86 (m, 2H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>3</sub>OF<sub>2</sub> 341.1340; found 341.1337.</div></div><div id="sec4_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 6-Fluoro-8-methyl-2-(4-piperidin-4-yl-phenyl)-3<i>H</i>-quinazolin-4-one Hydrochloride (<b>8d</b>)</h4><div class="NLM_p last">Preparation as described for <b>8c</b>. Yield: 846 mg (75%) off-white solid. HPLC/MS <i>m</i>/<i>z</i>: 338.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.53 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, CF<sub>3</sub>COOD) δ 8.34–8.20 (m, 2H), 7.70 (dd, <i>J</i> = 8.5, 3.1 Hz, 1H), 7.52 (ddd, <i>J</i> = 9.3, 3.1, 1.0 Hz, 1H), 7.49–7.36 (m, 2H), 3.47 (dt, <i>J</i> = 12.7, 2.5 Hz, 2H), 3.09 (td, <i>J</i> = 12.8, 3.5 Hz, 2H), 3.00 (tt, <i>J</i> = 11.8, 3.9 Hz, 1H), 2.68 (s, 3H), 2.11–1.85 (m, 4H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>3</sub>OF 337.1590; found 337.1599.</div></div><div id="sec4_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 6,8-Difluoro-2-[4-(1-methyl-piperidin-4-yl)-phenyl]-3<i>H</i>-quinazolin-4-one Trifluoroacetate (<b>8e</b>)</h4><div class="NLM_p last">To a solution of 6,8-difluoro-2-(4-piperidin-4-yl-phenyl)-3<i>H</i>-quinazolin-4-one hydrochloride (238 mg, 0.63 mmol) in formic acid (2.0 mL), formaldehyde (37% aqueous solution, 160 μL, 1.27 mmol) was added. The mixture was heated to 80 °C and stirred at this temperature for 18 h. The reaction mixture was evaporated, and the residue was purified by preparative HPLC. Yield: 28 mg (9%) colorless powder. HPLC/MS <i>m</i>/<i>z</i>: 356.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.43 min. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.80 (s, 1H), 8.23–8.14 (m, 2H), 7.85 (ddd, <i>J</i> = 10.3, 9.0, 2.9 Hz, 1H), 7.71 (ddd, <i>J</i> = 8.3, 2.9, 1.2 Hz, 1H), 7.46 (d, <i>J</i> = 8.0 Hz, 2H), 3.60–3.46 (m, 2H), 3.10 (t, <i>J</i> = 12.6 Hz, 2H), 2.98–2.88 (m, 1H), 2.84 (s, 3H), 2.12–2.02 (m, 2H), 2.00–1.80 (m, 2H). HRMS (EI): <i>m</i>/<i>z</i> [M]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>3</sub>OF<sub>2</sub> 355.1496; found 355.1499.</div></div><div id="sec4_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 6-Fluoro-8-methyl-2-[4-(1-methyl-piperidin-4-yl)-phenyl]-3<i>H</i>-quinazolin-4-one (<b>8f</b>)</h4><div class="NLM_p last">Preparation as described for <b>8e</b>. Yield: 118 mg (67%) off-white solid. HPLC/MS <i>m</i>/<i>z</i>: 352.1 [M + H]. <i>R</i><sub>t</sub> (D): 1.52 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.55 (s, 1H), 8.24 (d, <i>J</i> = 8.0 Hz, 2H), 7.53 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 7.40 (dd, <i>J</i> = 9.6, 3.0 Hz, 1H), 7.33 (d, <i>J</i> = 8.4 Hz, 2H), 2.88 (dt, <i>J</i> = 11.7, 3.1 Hz, 2H), 2.60 (s, 3H), 2.20 (s, 3H), 1.97 (td, <i>J</i> = 11.5, 2.9 Hz, 2H), 1.83–1.61 (m, 4H). HRMS (EI): <i>m</i>/<i>z</i> [M]+ calcd for C<sub>21</sub>H<sub>22</sub>N<sub>3</sub>OF 351.1747; found 351.1749.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> TR-FRET Assay of TNKS1 and TNKS2 (Auto-PARsylation Assay)</h3><div class="NLM_p">The auto-PARsylation assay was performed as homogeneous HTRF (Cisbio, France) assay in Greiner low volume nb 384-well microtiter plates and was used for high throughput screen.</div><div class="NLM_p last">Truncated human recombinant GST-TEV-TNKS1 (1023–1327 aa) and GST-TEV-TNKS2 (873–1166 aa) were expressed in Sf9 insect cells with the baculovirus expression system. TNKS2 was purified by affinity purification. TNKS1 was purified by affinity purification, concentration, and gel filtration. GST-tagged TNKS1 (1023–1327 aa, 250 nM) or GST-tagged TNKS2 (873–1166 aa, 250 nM) and 5 μM bio-NAD (Biolog, Germany) as cosubstrate were incubated in a total volume of 5 μL (50 mM HEPES, 4 mM MgCl<sub>2</sub>, 0.05% Pluronic F-68, 1.4 mM DTT, 0.5% DMSO, pH 7.7) in the absence or presence of the test compound (10 dilution concentrations) for 90 min at 30 °C. The reaction was stopped by the addition of 1 μL of 50 mM EDTA solution. Two microliters of the detection solution (1.6 μM SA-Xlent, 7.4 nM Anti-GST-K (Cisbio, France) in 50 mM HEPES, 800 mM KF, 0.1% BSA, 20 mM EDTA, 0.1% CHAPS, pH 7.0) was added. After 1 h incubation at room temperature, the TR-FRET was measured with an Envision reader (PerkinElmer LAS Germany GmbH) (exc 340 nm; em 615 nm/665 nm). The auto-PARsylation activity was detectable directly via the increase in TR-FRET signal. As full value, the inhibitor-free reaction was used. The pharmacological zero value used was XAV-939 (Tocris) 5 μM fc. The inhibitory values (IC<sub>50</sub>) were determined using Condosseo from GeneData.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> PARP-1 Biochemical Activity Assay (Auto-PARsylation Assay)</h3><div class="NLM_p last">The auto-PARsylation assay for PARP activity was performed in the same assay format as for tankyrase. His-tagged Parp-1 (35 nM, human, recombinant, Enzo Life Sciences GmbH, Germany) and a mixture of 125 nM bio-NAD (Biolog, Germany) and 800 nM NAD as cosubstrate were incubated in a total volume of 6 μL (100 mM Tris/HCl, 4 mM MgCl<sub>2</sub>, 0.01% IGEPAL CA630, 1 mM DTT, 0.5% DMSO, pH 8, 13 ng/μL activated DNA (BPS Bioscience, US)) in the absence or presence of the test compound (10 dilution concentrations) for 150 min at 23 °C. The reaction was stopped by the addition of 4 μL of the stop/detection solution (70 nM SA-Xlent, 2.5 nM Anti-His-K (Eu-labeled anti-His, Cisbio, France) in 50 mM HEPES, 400 mM KF, 0.1% BSA, 20 mM EDTA, pH 7.0). After 1 h incubation at room temperature, the TR-FRET was measured with an Envision reader (PerkinElmer LAS Germany GmbH). As full value, the inhibitor-free reaction was used. The pharmacological zero value used was olaparib (LClabs, US) 1 μM fc. The inhibitory values (IC<sub>50</sub>) were determined using Condosseo from GeneData.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Activity ELISA of TNKS1 and TNKS2 (Auto-PARsylation Assay)<a onclick="showRef(event, 'ref23 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref25 ref26">(23,25,26)</a></h3><div class="NLM_p last">The activity ELISA was performed in 384-well glutathione coated microtiter plates (Express capture glutathione coated plate, Biocat, Germany), with expected binding capacity of 0.5–1 nM. Buffer pre-equilibrated plates were incubated with 50 μL of 20 ng/well GST-tagged TNKS1 (1023–1327 aa, prepared in-house, 6.4 nM) or GST-tagged TNKS2 (873–1166 aa, prepared in-house, 6.4 nM) in assay buffer (50 mM HEPES, 4 mM MgCl<sub>2</sub>, 0.05% Pluronic F-68, 2 mM DTT, pH 7.7) overnight at 4 °C. The plates were washed 3 times with PBS-T. Wells were blocked by incubation at room temperature for 20 min with 50 μL of blocking buffer (PBS, 0.05% Tween-20, 0.5% BSA). After a washing step with PBS-T the enzymatic reaction was performed in 50 μL reaction solution (50 mM HEPES, 4 mM MgCl<sub>2</sub>, 0.05% Pluronic F-68, 1.4 mM DTT, 0.5% DMSO, pH 7.7) with 10 μM bio-NAD (Biolog, Germany) as cosubstrate in the absence or presence of the test compound (10 dilution concentrations) for 1 h at 30 °C. The reaction was stopped by 3 times washing with PBS-T. For the detection, 50 μL of 20 ng/μL streptavidin–HRP conjugate (MoBiTec, Germany) in PBS/0.05%Tween-20/0.01%BSA was added, and the plates were incubated for 30 min at room temperature. After a wash step, 50 μL of SuperSignal ELISA Femto Maximum sensitivity substrate solution (ThermoFisherScientific) was added. Following a 1 min incubation at room temperature, luminescence signals were measured with an Envision reader (PerkinElmer LAS Germany GmbH) at 700 nm. As full value, the inhibitor-free reaction was used. The pharmacological zero value used was XAV-939 (Tocris) 5 μM (fc). The level of the luminescence signal correlated directly with the amount of auto-PARsylated tankyrase and therefore with activity of tankyrase. The inhibitory values (IC<sub>50</sub>) were determined in Condosseo from GeneData. S/B values were in the range of 10–15, and <i>z</i>′ was above 0.65 based on >1000 tested compounds.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> PARP Family Biochemical Assays</h3><div class="NLM_p last"><i>In vitro</i> enzymatic assays to determine the effect of compounds on activity of recombinant human PARP enzyme family members were performed at BPS Bioscience Inc., San Diego, using chemiluminescent assay kits according to their protocols (cat. nos. 80501, 80502, 80503, 80504, 80505, 80506, 80507, 80508, 80510, and 80511).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Axin2 and Tankyrase Cellular Assay</h3><div class="NLM_p last">Luminex-based bead assays were employed to assess the abundance of tankyrase (TNKS1/TNKS2) and axin2 proteins in cell lysates as described.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> DLD1 cells (ATCC CCL-221) were seeded with 4000 cells per well in 96-well cell culture plates. After 1 day, cells were treated with a serial dilution of test compounds with a final concentration of 0.3% DMSO for 24 h at 37 °C and 5% CO<sub>2</sub> in technical triplicates. For analyses of axin2 level, cells were lysed with 90 μL of NP40 cell lysis buffer (20 mM Tris/HCl, pH 8.0, 150 mM NaCl, 1% (v/v) NP40, 10% (v/v) glycerol) per well, and axin2 protein was isolated from cell lysates using a monoclonal axin2 antibody (R&D Systems no. MAB6078) bound to fluorescent carboxybeads. A polyclonal anti-axin2 antibody (Cell Signaling no. 2151) and an appropriate PE-fluorescent secondary antibody was used for detection. A RIPA cell lysis buffer (90 μL per well; 10 mM Tris HCl, 150 mM sodium chloride, 1% NP40, 1% Triton X-100, 0.4% sodium deoxycholate, 2 mM EDTA, 0.3% SDS) was applied for detection of tankyrase protein level. A monoclonal anti-tankyrase antibody (Thermo Scientific, MA1-41011) bound to fluorescent carboxybeads was used for isolation of tankyrase from cell lysates and a polyclonal anti-tankyrase antibody (Santa Cruz, sc-8337) and an appropriate PE-fluorescent secondary antibody was used for detection. The amount of isolated tankyrase and axin2 proteins was determined with a Luminex 200 device (Luminex Corporation) according to the manufacturer’s instruction by counting 100 events per sample. Half maximal effective concentrations (EC<sub>50</sub> values) were determined using the program Symyx Assay Explorer.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Crystallization and Structure Determination</h3><div class="NLM_p">Human TNKS1 (1104–1314) was expressed and purified as described in the literature.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Human TNKS1 (10.7 mg/mL) in 25 mM HEPES, 200 mM NaCl, 5% glycerol, and 14 mM β-mercaptoethanol, pH 8.0, was mixed in a 1:1 (v/v) ratio with the reservoir solution (0.1 M Bis-Tris, pH 6.5, 25% w/v PEG 3350). Crystals were obtained using the sitting drop vapor diffusion method by equilibrating against 100 μL of the reservoir solution for 1 week at 277 K. The crystals were harvested directly from the crystallization solution and cryo-cooled in liquid nitrogen. X-ray diffraction data were collected on X06SA-PX beamline at Swiss Light Source (SLS) synchrotron radiation source using Dectris Eiger 16M detector.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> The collected diffraction images were processed with XDS package.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The structure was solved by molecular replacement with the program PHASER from the CCP4 program suite<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> using TNKS1 structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RF5">2RF5</a>) as a search model. Crystallographic refinement was performed with Buster 2.11.7.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Model building was done in Coot.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The final model was obtained after several cycles of rebuilding in Coot with subsequent runs of Buster. The coordinates of the cocrystal structures of human TNKS1 in complex with <b>5k</b> have been deposited with RCSB Protein Data Bank under the accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a> (see <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Data Collection and Refinement Statistics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">Data Collection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Space group</td><td class="colsep0 rowsep0" align="left"><i>C</i>222<sub>1</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">77.94, 94.64, 62.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">α, β, γ (deg)</td><td class="colsep0 rowsep0" align="left">90.00, 90.00, 90.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">43.29–1.20 (1.27–1.20)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. observations</td><td class="colsep0 rowsep0" align="left">438565 (56508)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflns</td><td class="colsep0 rowsep0" align="left">70543 (10961)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">6.2 (5.2)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">97.5 (93.5)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub> (%)</td><td class="colsep0 rowsep0" align="left">4.4 (99.8)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CC<sub>1/2</sub> (%)</td><td class="colsep0 rowsep0" align="left">99.8 (58.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ(<i>I</i>)</td><td class="colsep0 rowsep0" align="left">10.80 (0.82)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution (Å)</td><td class="colsep0 rowsep0" align="left">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub> (%)</td><td class="colsep0 rowsep0" align="left">18.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>free</sub> (%)</td><td class="colsep0 rowsep0" align="left">21.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">number of atoms</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein</td><td class="colsep0 rowsep0" align="left">1682</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand</td><td class="colsep0 rowsep0" align="left">28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">solvent</td><td class="colsep0 rowsep0" align="left">336</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>B</i> factor all atoms (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">22.59</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>B</i> factor ligand (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">15.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rms deviations</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (deg)</td><td class="colsep0 rowsep0" align="left">1.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran statistics</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored region (%)</td><td class="colsep0 rowsep0" align="left">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">allowed region (%)</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">outliers (%)</td><td class="colsep0 rowsep0" align="left">0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PDB code</td><td class="colsep0 rowsep0" align="left"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a></td></tr></tbody></table></div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i64" class="anchor-spacer"></div><h3 class="article-section__title" id="_i64"> DLD-1 Xenograft Model</h3><div class="NLM_p last">Animal procedures were performed in accordance with the German Laws for Animal Welfare and were reviewed and approved by the Hessian Government Body (Regierungspräsidium) in Darmstadt (License\395). Animals were acclimatized a week prior to use, and general health status was monitored during the experiments. Female immunodeficient athymic nude mice (Envigo) 5–7 weeks old received subcutaneously implanted tumor cell suspensions of DLD-1 (5 million cells in 100 μL of PBS per animal) in the flank. Tumors were grew until the tumor volume ranged from 300 to 800 mm<sup>3</sup> (calculated <i>L</i> × <i>W</i> × <i>W</i>/2 from digital calliper measurements). Mice with appropriate tumor volume were stratified into control and treatment groups (<i>n</i> = 5/group). Administrations of vehicle or compound (formulated in 0.5% Methocel/0.25% Tween20/water) were orally given by gavage, and plasma and tumors were collected at the respective time points (2 h for vehicle-treated and 2, 4, 8, or 12 h for compound-treated) for subsequent PD analysis. Mice were euthanized, blood was collected via armpit bleeding and processed for plasma, and tumors were excised and frozen at −80 °C.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> PD Analyses</h3><div class="NLM_p last">Tumor samples were homogenized using a Precellys tissue homogenizer by applying 1000 μL of ice-cold lysis buffer to 100 mg of tumor tissue. Homogenates were mixed by vortexing and then cleared by centrifugation for 20 min at 4 °C. Lysates were shock-frozen in liquid nitrogen and stored at −80 °C. Protein concentrations were determined using the BCA method. Abundance of tankyrase and axin2 were determined in 50 μg of total protein per sample as technical triplicates applying the Luminex-based bead assays as described above. For detection of nuclear, transcriptionally active, “free” β-catenin level, a Luminex based protein–protein interaction using GST-ECT as capture molecule and 10 μg of total protein per sample was employed as described.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> Determination of Thermodynamic Solubility</h3><div class="NLM_p last">Thermodynamic solubility of the compounds was measured as described in the literature.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00656" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08123" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08123" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00656" class="ext-link">10.1021/acs.jmedchem.9b00656</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Assay data of the best compounds with SD/SEM for the values, assay curve of biochemical assay IC<sub>50</sub> determination for <b>5k</b>, ligand <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map at 2.5σ cutoff, protocol for determination of microsomal stability (intrinsic clearance), protocol for the determination of Kv11.1 (hERG) ion channel activity, and procedure for Caco-2 <i>P</i><sub>app</sub> determination (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_001.pdf">jm9b00656_si_001.pdf (382.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_002.csv">jm9b00656_si_002.csv (1.43 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors are deposited at the Protein Data Bank with code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a> (<b>5k</b>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00656" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38011" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38011" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hans-Peter Buchstaller</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0577-8319" title="Orcid link">http://orcid.org/0000-0002-0577-8319</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#b18b91d9d0dfc29cc1d4c5d4c39fd3c4d2d9c2c5d0ddddd4c3f1dcd4c3d2dad6c3dec4c19fd2dedc">: <span class="__cf_email__" data-cfemail="523a333c217f22372637207c3027313a2126333e3e3720123f3720313935203d27227c313d3f">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Uwe Anlauf</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dieter Dorsch</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Kuhn</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Lehmann</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Birgitta Leuthner</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Djordje Musil</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniela Radtki</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claudio Ritzert</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Felix Rohdich</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Schneider</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christina Esdar</span> - <span class="hlFld-Affiliation affiliation">Merck Healthcare KGaA, Global Research & Development, Frankfurter Strasse 250, 64293 Darmstadt, Germany</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71448" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71448" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Jutta Welge, Thorsten Herd, Ulrike Bauer, and Dagmar Wolf for excellent experimental support. We acknowledge the work of Cornelia Sommersdorf and Oliver Poetz (NMI, Reutlingen) for analyses of β-catenin expression in <i>ex vivo</i> lysates. The authors are grateful to Bayard Huck for helpful suggestions and proofreading the manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">APC</td><td class="NLM_def"><p class="first last">adenomatous polyposis coli</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancer</p></td></tr><tr><td class="NLM_term"><i>f</i><sub>ub</sub></td><td class="NLM_def"><p class="first last">fraction unbound</p></td></tr><tr><td class="NLM_term">GST</td><td class="NLM_def"><p class="first last">glutathione S transferase</p></td></tr><tr><td class="NLM_term">NAD</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">PARP</td><td class="NLM_def"><p class="first last">poly(ADP-ribose) polymerase</p></td></tr><tr><td class="NLM_term">RNF146</td><td class="NLM_def"><p class="first last">ring finger protein 146</p></td></tr><tr><td class="NLM_term">TNKS</td><td class="NLM_def"><p class="first last">tankyrase</p></td></tr><tr><td class="NLM_term">TR-FRET</td><td class="NLM_def"><p class="first last">time-resolved fluorescence resonance energy transfer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i71">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30434" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30434" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 35 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riffell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Tankyrase-targeted therapeutics: expanding opportunities in the PARP family</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1038/nrd3868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnrd3868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=23197039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCntbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=923-936&author=J.+L.+Riffellauthor=C.+J.+Lordauthor=A.+Ashworth&title=Tankyrase-targeted+therapeutics%3A+expanding+opportunities+in+the+PARP+family&doi=10.1038%2Fnrd3868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrase-targeted therapeutics: expanding opportunities in the PARP family</span></div><div class="casAuthors">Riffell, Jenna L.; Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">923-936</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The poly(ADP-ribose) polymerase (PARP) protein superfamily has wide-ranging roles in cellular processes such as DNA repair and WNT signalling.  Efforts to pharmacol. target PARP enzymes have largely focused on PARP1 and the closely related PARP2, but recent work highlighting the role of another family member, tankyrase 1 (TANK1; also known as PARP5A and ARTD5), in the control of WNT signalling has fuelled interest in the development of addnl. inhibitors to target this enzyme class.  Tankyrase function is also implicated in other processes such as the regulation of telomere length, lung fibrogenesis and myelination, suggesting that tankyrase inhibitors could have broad clin. utility.  Here, we discuss the biol. of tankyrases and the discovery of tankyrase-specific inhibitors.  We also consider the challenges that lie ahead for the clin. development of PARP family inhibitors in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6R0zBQ42UELVg90H21EOLACvtfcHk0lg_FsWa4M1jhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCntbvI&md5=510c523982433e4c22f06f8d1679e6f6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3868%26sid%3Dliteratum%253Aachs%26aulast%3DRiffell%26aufirst%3DJ.%2BL.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DTankyrase-targeted%2520therapeutics%253A%2520expanding%2520opportunities%2520in%2520the%2520PARP%2520family%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D923%26epage%3D936%26doi%3D10.1038%2Fnrd3868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauss, S.</span></span> <span> </span><span class="NLM_article-title">Tankyrases as drug targets</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">3576</span>– <span class="NLM_lpage">3593</span>, <span class="refDoi"> DOI: 10.1111/febs.12320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1111%2Ffebs.12320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=23648170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A280%3ADC%252BC3snhtFOqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=3576-3593&author=L.+Lehti%C3%B6author=N.+Chiauthor=S.+Krauss&title=Tankyrases+as+drug+targets&doi=10.1111%2Ffebs.12320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrases as drug targets</span></div><div class="casAuthors">Lehtio Lari; Chi Nai-Wen; Krauss Stefan</div><div class="citationInfo"><span class="NLM_cas:title">The FEBS journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3576-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Tankyrase 1 and tankyrase 2 are poly(ADP-ribosyl)ases that are distinguishable from other members of the enzyme family by the structural features of the catalytic domain, and the presence of a sterile α-motif multimerization domain and an ankyrin repeat protein-interaction domain.  Tankyrases are implicated in a multitude of cellular functions, including telomere homeostasis, mitotic spindle formation, vesicle transport linked to glucose metabolism, Wnt-β-catenin signaling, and viral replication.  In these processes, tankyrases interact with target proteins, catalyze poly(ADP-ribosyl)ation, and regulate protein interactions and stability.  The proposed roles of tankyrases in disease-relevant cellular processes have made them attractive drug targets.  Recently, several inhibitors have been identified.  The selectivity and potency of these small molecules can be rationalized by how they fit within the NAD(+)-binding groove of the catalytic domain.  Some molecules bind to the nicotinamide subsite, such as generic diphtheria toxin-like ADP-ribosyltransferase inhibitors, whereas others bind to a distinct adenosine subsite that diverges from other diphtheria toxin-like ADP-ribosyltransferases and confers specificity.  A highly potent dual-site inhibitor is also available.  Within the last few years, tankyrase inhibitors have proved to be useful chemical probes and potential lead compounds, especially for specific cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxqKZ7T5loVg20zIbnCCI3fW6udTcc2eaBI9oGJa85xbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snhtFOqug%253D%253D&md5=9643c0f2d7ee877400bffcdc8ad44c85</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12320%26sid%3Dliteratum%253Aachs%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26aulast%3DChi%26aufirst%3DN.%26aulast%3DKrauss%26aufirst%3DS.%26atitle%3DTankyrases%2520as%2520drug%2520targets%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D3576%26epage%3D3593%26doi%3D10.1111%2Ffebs.12320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haikarainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span> <span> </span><span class="NLM_article-title">Tankyrases: Structure, function and therapeutic implications in cancer</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">6472</span>– <span class="NLM_lpage">6488</span>, <span class="refDoi"> DOI: 10.2174/1381612820666140630101525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.2174%2F1381612820666140630101525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=24975604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgtLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=6472-6488&author=T.+Haikarainenauthor=S.+Kraussauthor=L.+Lehti%C3%B6&title=Tankyrases%3A+Structure%2C+function+and+therapeutic+implications+in+cancer&doi=10.2174%2F1381612820666140630101525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrases: Structure, Function and Therapeutic Implications in Cancer</span></div><div class="casAuthors">Haikarainen, Teemu; Krauss, Stefan; Lehtio, Lari</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">6472-6488</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Several cellular signaling pathways are regulated by ADP-ribosylation, a posttranslational modification catalyzed by members of the ARTD superfamily.  Tankyrases are distinguishable from the rest of this family by their unique domain organization, notably the sterile alpha motif responsible for oligomerization and ankyrin repeats mediating protein-protein interactions.  Tankyrases are involved in various cellular functions, such as telomere homeostasis, Wnt/β -catenin signaling, glucose metab., and cell cycle progression.  In these processes, Tankyrases regulate the interactions and stability of target proteins by poly (ADP-ribosyl)ation.  Modified proteins are subsequently recognized by the E3 ubiquitin ligase RNF146, poly-ubiquitinated and predominantly guided to 26S proteasomal degrdn.  Several small mol. inhibitors have been described for Tankyrases; they compete with the co-substrate NAD+ for binding to the ARTD catalytic domain.  The recent, highly potent and selective inhibitors possess several properties of lead compds. and can be used for proof-of-concept studies in cancer and other Tankyrase linked diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotYUovVf-x6LVg90H21EOLACvtfcHk0ljDjCEJImBXLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgtLfN&md5=ced27d2889d245696043a8cfa891166c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F1381612820666140630101525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612820666140630101525%26sid%3Dliteratum%253Aachs%26aulast%3DHaikarainen%26aufirst%3DT.%26aulast%3DKrauss%26aufirst%3DS.%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26atitle%3DTankyrases%253A%2520Structure%252C%2520function%2520and%2520therapeutic%2520implications%2520in%2520cancer%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2014%26volume%3D20%26spage%3D6472%26epage%3D6488%26doi%3D10.2174%2F1381612820666140630101525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riyaz, Sd.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahim, A.</span></span> <span> </span><span class="NLM_article-title">Tankyrase inhibitors as therapeutic targets for cancer</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1967</span>– <span class="NLM_lpage">1976</span>, <span class="refDoi"> DOI: 10.2174/1568026614666140929115831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.2174%2F1568026614666140929115831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=25262803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1967-1976&author=A.+Kamalauthor=Sd.+Riyazauthor=A.+K.+Srivastavaauthor=A.+Rahim&title=Tankyrase+inhibitors+as+therapeutic+targets+for+cancer&doi=10.2174%2F1568026614666140929115831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrase Inhibitors as Therapeutic Targets for Cancer</span></div><div class="casAuthors">Kamal, Ahmed; Riyaz, S.; Srivastava, Ajay Kumar; Rahim, Abdul</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1967-1976</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Tankyrase 1 and 2 belonging to the family of poly(ADP-ribosyl)ases play an important role in PARsylation by utilizing NAD+ as a substrate in order to generate ADP-ribose polymers.  Tankyrases are involved in a no. of cellular functions, that includes telomere homeostasis, mitotic spindle formation, vesicle transport linked to glucose metab., Wnt/β-catenin signalling, and viral replication.  These roles of tankyrases in disease-relevant cellular processes have made them attractive drug targets.  Recently, several inhibitors have been identified as potential clin. leads.  The current review covers the progress, mechanism and binding modes of recently known Tankyrase inhibitors and discusses the rational approaches that were used to identify the tankyrase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruFM_1i4ZP1LVg90H21EOLACvtfcHk0lhfnKxiGuWyNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7vP&md5=41cdf01744936fa538ef8e7307180feb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F1568026614666140929115831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026614666140929115831%26sid%3Dliteratum%253Aachs%26aulast%3DKamal%26aufirst%3DA.%26aulast%3DRiyaz%26aufirst%3DSd.%26aulast%3DSrivastava%26aufirst%3DA.%2BK.%26aulast%3DRahim%26aufirst%3DA.%26atitle%3DTankyrase%2520inhibitors%2520as%2520therapeutic%2520targets%2520for%2520cancer%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2014%26volume%3D14%26spage%3D1967%26epage%3D1976%26doi%3D10.2174%2F1568026614666140929115831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amé, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spenlehauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">The PARP superfamily</span>. <i>BioEssays</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">882</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1002/bies.20085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1002%2Fbies.20085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=15273990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2004&pages=882-893&author=J.-C.+Am%C3%A9author=C.+Spenlehauerauthor=G.+de+Murcia&title=The+PARP+superfamily&doi=10.1002%2Fbies.20085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The PARP superfamily</span></div><div class="casAuthors">Ame, Jean-Christophe; Spenlehauer, Catherine; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">BioEssays</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">882-893</span>CODEN:
                <span class="NLM_cas:coden">BIOEEJ</span>;
        ISSN:<span class="NLM_cas:issn">0265-9247</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribosyl)ation is an immediate DNA-damage-dependent post-translational modification of histones and other nuclear proteins that contributes to the survival of injured proliferating cells.  Poly(ADP-ribose) polymerases (PARPs) now constitute a large family of 18 proteins, encoded by different genes and displaying a conserved catalytic domain in which PARP-1 (113 kDa), the founding member, and PARP-2 (62 kDa) are so far the sole enzymes whose catalytic activity has been shown to be immediately stimulated by DNA strand breaks.  A large repertoire of sequences encoding novel PARPs now extends considerably the field of poly(ADP-ribosyl)ation reactions to various aspects of the cell biol. including cell proliferation and cell death.  Some of these new members interact with each other, share common partners and common subcellular localizations suggesting possible fine tuning in the regulation of this post-translational modification of proteins.  This review summarizes the present knowledge of this emerging superfamily, which might ultimately improve pharmacol. strategies to enhance both antitumor efficacy and the treatment of a no. of inflammatory and neurodegenerative disorders.  A provisional nomenclature is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkP1P6SYh67Vg90H21EOLACvtfcHk0lhRo6K9efMh8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D&md5=892151993f54e3c1a03019001a365fbf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fbies.20085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbies.20085%26sid%3Dliteratum%253Aachs%26aulast%3DAm%25C3%25A9%26aufirst%3DJ.-C.%26aulast%3DSpenlehauer%26aufirst%3DC.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DThe%2520PARP%2520superfamily%26jtitle%3DBioEssays%26date%3D2004%26volume%3D26%26spage%3D882%26epage%3D893%26doi%3D10.1002%2Fbies.20085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantzer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ame, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose): Novel functions for an old molecule</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nrm1963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnrm1963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=16829982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=517-528&issue=7&author=V.+Schreiberauthor=F.+Dantzerauthor=J.+C.+Ameauthor=G.+de+Murcia&title=Poly%28ADP-ribose%29%3A+Novel+functions+for+an+old+molecule&doi=10.1038%2Fnrm1963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose): novel functions for an old molecule</span></div><div class="casAuthors">Schreiber, Valerie; Dantzer, Francoise; Ame, Jean-Christophe; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-528</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The addn. to proteins of the neg. charged polymer of ADP-ribose, which is synthesized by poly(ADP-ribose) polymerase (PARPs) from NAD, is a unique post-translational modification.  It regulates not only cell survival and cell-death programs, but also an increasing no. of other biol. functions with which novel members of the PARP family have been assocd.  These functions include transcriptional regulation, telomere cohesion and mitotic spindle formation during cell division, intracellular trafficking, and energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsI4_BKfjdrVg90H21EOLACvtfcHk0lhRo6K9efMh8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D&md5=676e83755491500f3bf2137f348dfa1b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1963%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DV.%26aulast%3DDantzer%26aufirst%3DF.%26aulast%3DAme%26aufirst%3DJ.%2BC.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DPoly%2528ADP-ribose%2529%253A%2520Novel%2520functions%2520for%2520an%2520old%2520molecule%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26issue%3D7%26spage%3D517%26epage%3D528%26doi%3D10.1038%2Fnrm1963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liscio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asciutti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchiarulo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camaioni, E.</span></span> <span> </span><span class="NLM_article-title">Targeting Wnt-driven cancers: Discovery of novel tankyrase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">522</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.ejmech.2017.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=29107427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslentbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2017&pages=506-522&author=M.+Ferriauthor=P.+Liscioauthor=A.+Carottiauthor=S.+Asciuttiauthor=R.+Sardellaauthor=A.+Macchiaruloauthor=E.+Camaioni&title=Targeting+Wnt-driven+cancers%3A+Discovery+of+novel+tankyrase+inhibitors&doi=10.1016%2Fj.ejmech.2017.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Wnt-driven cancers: Discovery of novel tankyrase inhibitors</span></div><div class="casAuthors">Ferri, Martina; Liscio, Paride; Carotti, Andrea; Asciutti, Stefania; Sardella, Roccaldo; Macchiarulo, Antonio; Camaioni, Emidio</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">506-522</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Recent years have seen substantially heightened interest in the discovery of tankyrase inhibitors (TNKSi) as new promising anticancer agents.  In this framework, the aim of this review article is focused on the description of potent TNKSi also endowed with disruptor activity toward the Wnt/β-catenin signaling pathway.  Beginning with an overview of the most characterized TNKSi deriving from several drug design approaches and classifying them on the basis of the mol. interactions with the target, the authors discuss only those ones acting against Wnt cancer cell lines.  In addn., comprehensive structure property relationships (SPR) emerging from the hit evolution processes and preclin. results are provided.  The authors then review the most promising TNKSi hitherto reported in literature, acting in vivo models of Wnt-driven cancers.  Some out-looks on current issues and future directions in this field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAs1ycCxVwl7Vg90H21EOLACvtfcHk0ljC9TJerbWmyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslentbbJ&md5=9b4d843e6bb7ccdea56f9e9c8b70d18c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DFerri%26aufirst%3DM.%26aulast%3DLiscio%26aufirst%3DP.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DAsciutti%26aufirst%3DS.%26aulast%3DSardella%26aufirst%3DR.%26aulast%3DMacchiarulo%26aufirst%3DA.%26aulast%3DCamaioni%26aufirst%3DE.%26atitle%3DTargeting%2520Wnt-driven%2520cancers%253A%2520Discovery%2520of%2520novel%2520tankyrase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D142%26spage%3D506%26epage%3D522%26doi%3D10.1016%2Fj.ejmech.2017.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, F.</span></span> <span> </span><span class="NLM_article-title">RNF146 is a poly(ADP-ribose)-directed E3 ligase that regulates axin degradation and Wnt signaling</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1038/ncb2222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fncb2222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=21478859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsFGmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=623-629&author=Y.+Zhangauthor=S.+Liuauthor=C.+Mickaninauthor=Y.+Fengauthor=O.+Charlatauthor=G.+A.+Michaudauthor=M.+Schirleauthor=X.+Shiauthor=M.+Hildauthor=A.+Bauerauthor=V.+E.+Myerauthor=P.+M.+Finanauthor=J.+A.+Porterauthor=S.+M.+Huangauthor=F.+Cong&title=RNF146+is+a+poly%28ADP-ribose%29-directed+E3+ligase+that+regulates+axin+degradation+and+Wnt+signaling&doi=10.1038%2Fncb2222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RNF146 is a poly(ADP-ribose)-directed E3 ligase that regulates axin degradation and Wnt signalling</span></div><div class="casAuthors">Zhang, Yue; Liu, Shanming; Mickanin, Craig; Feng, Yan; Charlat, Olga; Michaud, Gregory A.; Schirle, Markus; Shi, Xiaoying; Hild, Marc; Bauer, Andreas; Myer, Vic E.; Finan, Peter M.; Porter, Jeffery A.; Huang, Shih-Min A.; Cong, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">623-629</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Wnt/β-catenin signalling pathway plays essential roles in embryonic development and adult tissue homeostasis, and deregulation of this pathway has been linked to cancer.  Axin is a concn.-limiting component of the β-catenin destruction complex, and its stability is regulated by tankyrase.  However, the mol. mechanism by which tankyrase-dependent poly(ADP-ribosyl)ation (PARsylation) is coupled to ubiquitylation and degrdn. of axin remains undefined.  Here, we identify RNF146, a RING-domain E3 ubiquitin ligase, as a pos. regulator of Wnt signalling.  RNF146 promotes Wnt signalling by mediating tankyrase-dependent degrdn. of axin.  Mechanistically, RNF146 directly interacts with poly(ADP-ribose) through its WWE domain, and promotes degrdn. of PARsylated proteins.  Using proteomics approaches, we have identified BLZF1 and CASC3 as further substrates targeted by tankyrase and RNF146 for degrdn.  Thus, identification of RNF146 as a PARsylation-directed E3 ligase establishes a mol. paradigm that links tankyrase-dependent PARsylation to ubiquitylation.  RNF146-dependent protein degrdn. may emerge as a major mechanism by which tankyrase exerts its function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgvUCNwxGZLrVg90H21EOLACvtfcHk0lj0XhnqLBzAiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsFGmurk%253D&md5=0029be867466c74516f1a7fbab62369d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fncb2222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2222%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DCharlat%26aufirst%3DO.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DFinan%26aufirst%3DP.%2BM.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DCong%26aufirst%3DF.%26atitle%3DRNF146%2520is%2520a%2520poly%2528ADP-ribose%2529-directed%2520E3%2520ligase%2520that%2520regulates%2520axin%2520degradation%2520and%2520Wnt%2520signaling%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2011%26volume%3D13%26spage%3D623%26epage%3D629%26doi%3D10.1038%2Fncb2222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callow, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bheddah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hongo, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin ligase RNF146 regulates tankyrase and axin to promote Wnt signaling</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e22595</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0022595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1371%2Fjournal.pone.0022595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=21799911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKgt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=M.+G.+Callowauthor=H.+Tranauthor=L.+Phuauthor=T.+Lauauthor=J.+Leeauthor=W.+N.+Sandovalauthor=P.+S.+Liuauthor=S.+Bheddahauthor=J.+Taoauthor=J.+R.+Lillauthor=J.-A.+Hongoauthor=D.+Davisauthor=D.+S.+Kirkpatrickauthor=P.+Polakisauthor=M.+Costa&title=Ubiquitin+ligase+RNF146+regulates+tankyrase+and+axin+to+promote+Wnt+signaling&doi=10.1371%2Fjournal.pone.0022595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin ligase RNF146 regulates tankyrase and Axin to promote Wnt signaling</span></div><div class="casAuthors">Callow, Marinella G.; Tran, Hoanh; Phu, Lilian; Lau, Ted; Lee, James; Sandoval, Wendy N.; Liu, Peter S.; Bheddah, Sheila; Tao, Janet; Lill, Jennie R.; Hongo, Jo-Anne; Davis, David; Kirkpatrick, Donald S.; Polakis, Paul; Costa, Mike</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e22595</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Canonical Wnt signaling is controlled intracellularly by the level of β-catenin protein, which is dependent on Axin scaffolding of a complex that phosphorylates β-catenin to target it for ubiquitylation and proteasomal degrdn.  This function of Axin is counteracted through relocalization of Axin protein to the Wnt receptor complex to allow for ligand-activated Wnt signaling.  AXIN1 and AXIN2 protein levels are regulated by tankyrase-mediated poly(ADP-ribosyl)ation (PARsylation), which destabilizes Axin and promotes signaling.  Mechanistically, how tankyrase limits Axin protein accumulation, and how tankyrase levels and activity are regulated for this function, are currently under investigation.  By RNAi screening, we identified the RNF146 RING-type ubiquitin E3 ligase as a pos. regulator of Wnt signaling that operates with tankyrase to maintain low steady-state levels of Axin proteins.  RNF146 also destabilizes tankyrases TNKS1 and TNKS2 proteins and, in a reciprocal relationship, tankyrase activity reduces RNF146 protein levels.  We show that RNF146, tankyrase, and Axin form a protein complex, and that RNF146 mediates ubiquitylation of all 3 proteins to target them for proteasomal degrdn.  RNF146 is a cytoplasmic protein that also prevents tankyrase protein aggregation at a centrosomal location.  Tankyrase auto-PARsylation and PARsylation of Axin is known to lead to proteasome-mediated degrdn. of these proteins, and we demonstrate that, through ubiquitylation, RNF146 mediates this process to regulate Wnt signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBK_DIvtiDN7Vg90H21EOLACvtfcHk0lj0XhnqLBzAiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKgt7vO&md5=df9dc5d0f0fc70e5d9c81783a34ee10c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0022595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0022595%26sid%3Dliteratum%253Aachs%26aulast%3DCallow%26aufirst%3DM.%2BG.%26aulast%3DTran%26aufirst%3DH.%26aulast%3DPhu%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSandoval%26aufirst%3DW.%2BN.%26aulast%3DLiu%26aufirst%3DP.%2BS.%26aulast%3DBheddah%26aufirst%3DS.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DLill%26aufirst%3DJ.%2BR.%26aulast%3DHongo%26aufirst%3DJ.-A.%26aulast%3DDavis%26aufirst%3DD.%26aulast%3DKirkpatrick%26aufirst%3DD.%2BS.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DCosta%26aufirst%3DM.%26atitle%3DUbiquitin%2520ligase%2520RNF146%2520regulates%2520tankyrase%2520and%2520axin%2520to%2520promote%2520Wnt%2520signaling%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0022595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishina, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiellette, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiessner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghidelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouwmeester, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, F.</span></span> <span> </span><span class="NLM_article-title">Tankyrase inhibition stabilizes axin and antagonizes Wnt signaling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1038/nature08356</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnature08356" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=19759537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFentbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=614-620&author=S.-M.+A.+Huangauthor=Y.+M.+Mishinaauthor=S.+Liuauthor=A.+Cheungauthor=F.+Stegmeierauthor=G.+A.+Michaudauthor=O.+Charlatauthor=E.+Wielletteauthor=Y.+Zhangauthor=S.+Wiessnerauthor=M.+Hildauthor=X.+Shiauthor=C.+J.+Wilsonauthor=C.+Mickaninauthor=V.+Myerauthor=A.+Fazalauthor=R.+Tomlinsonauthor=F.+Serlucaauthor=W.+Shaoauthor=H.+Chengauthor=M.+Shultzauthor=C.+Rauauthor=M.+Schirleauthor=J.+Schleglauthor=S.+Ghidelliauthor=S.+Fawellauthor=C.+Luauthor=D.+Curtisauthor=M.+W.+Kirschnerauthor=C.+Lengauerauthor=P.+M.+Finanauthor=J.+A.+Tallaricoauthor=T.+Bouwmeesterauthor=J.+A.+Porterauthor=A.+Bauerauthor=F.+Cong&title=Tankyrase+inhibition+stabilizes+axin+and+antagonizes+Wnt+signaling&doi=10.1038%2Fnature08356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrase inhibition stabilizes axin and antagonizes Wnt signalling</span></div><div class="casAuthors">Huang, Shih-Min A.; Mishina, Yuji M.; Liu, Shanming; Cheung, Atwood; Stegmeier, Frank; Michaud, Gregory A.; Charlat, Olga; Wiellette, Elizabeth; Zhang, Yue; Wiessner, Stephanie; Hild, Marc; Shi, Xiaoying; Wilson, Christopher J.; Mickanin, Craig; Myer, Vic; Fazal, Aleem; Tomlinson, Ronald; Serluca, Fabrizio; Shao, Wenlin; Cheng, Hong; Shultz, Michael; Rau, Christina; Schirle, Markus; Schlegl, Judith; Ghidelli, Sonja; Fawell, Stephen; Lu, Chris; Curtis, Daniel; Kirschner, Marc W.; Lengauer, Christoph; Finan, Peter M.; Tallarico, John A.; Bouwmeester, Tewis; Porter, Jeffery A.; Bauer, Andreas; Cong, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7264</span>),
    <span class="NLM_cas:pages">614-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The stability of the Wnt pathway transcription factor β-catenin is tightly regulated by the multi-subunit destruction complex.  Deregulated Wnt pathway activity has been implicated in many cancers, making this pathway an attractive target for anticancer therapies.  However, the development of targeted Wnt pathway inhibitors has been hampered by the limited no. of pathway components that are amenable to small mol. inhibition.  Here, we used a chem. genetic screen to identify a small mol., XAV939, which selectively inhibits β-catenin-mediated transcription.  XAV939 stimulates β-catenin degrdn. by stabilizing axin, the concn.-limiting component of the destruction complex.  Using a quant. chem. proteomic approach, we discovered that XAV939 stabilizes axin by inhibiting the poly-ADP-ribosylating enzymes tankyrase 1 and tankyrase 2.  Both tankyrase isoforms interact with a highly conserved domain of axin and stimulate its degrdn. through the ubiquitin-proteasome pathway.  Thus, our study provides new mechanistic insights into the regulation of axin protein homeostasis and presents new avenues for targeted Wnt pathway therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom3c1M4OoOarVg90H21EOLACvtfcHk0lgqhqGsltwfSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFentbzP&md5=6a76dfa7ed2a57b29e0c1ee367bd652c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature08356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.-M.%2BA.%26aulast%3DMishina%26aufirst%3DY.%2BM.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DCharlat%26aufirst%3DO.%26aulast%3DWiellette%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWiessner%26aufirst%3DS.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DTomlinson%26aufirst%3DR.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DShultz%26aufirst%3DM.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DGhidelli%26aufirst%3DS.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DFinan%26aufirst%3DP.%2BM.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DCong%26aufirst%3DF.%26atitle%3DTankyrase%2520inhibition%2520stabilizes%2520axin%2520and%2520antagonizes%2520Wnt%2520signaling%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D614%26epage%3D620%26doi%3D10.1038%2Fnature08356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mariotti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guettler, S.</span></span> <span> </span><span class="NLM_article-title">Regulation of Wnt/β-catenin signalling by tankyrase-dependent poly(ADP-ribosyl)ation and scaffolding</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">4611</span>– <span class="NLM_lpage">4636</span>, <span class="refDoi"> DOI: 10.1111/bph.14038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1111%2Fbph.14038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=28910490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslGqs7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=4611-4636&author=L.+Mariottiauthor=K.+Pollockauthor=S.+Guettler&title=Regulation+of+Wnt%2F%CE%B2-catenin+signalling+by+tankyrase-dependent+poly%28ADP-ribosyl%29ation+and+scaffolding&doi=10.1111%2Fbph.14038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of Wnt/β-catenin signalling by tankyrase-dependent poly(ADP-ribosyl)ation and scaffolding</span></div><div class="casAuthors">Mariotti, Laura; Pollock, Katie; Guettler, Sebastian</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4611-4636</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Wnt/β-catenin signalling pathway is pivotal for stem cell function and the control of cellular differentiation, both during embryonic development and tissue homeostasis in adults.  Its activity is carefully controlled through the concerted interactions of concn.-limited pathway components and a wide range of post-translational modifications, including phosphorylation, ubiquitylation, sumoylation, poly(ADP-ribosyl)ation (PARylation) and acetylation.  Regulation of Wnt/β-catenin signalling by PARylation was discovered relatively recently.  The PARP tankyrase PARylates AXIN1/2, an essential central scaffolding protein in the β-catenin destruction complex, and targets it for degrdn., thereby fine-tuning the responsiveness of cells to the Wnt signal.  The past few years have not only seen much progress in our understanding of the mol. mechanisms by which PARylation controls the pathway but also witnessed the successful development of tankyrase inhibitors as tool compds. and promising agents for the therapy of Wnt-dependent dysfunctions, including colorectal cancer.  Recent work has hinted at more complex roles of tankyrase in Wnt/β-catenin signalling as well as challenges and opportunities in the development of tankyrase inhibitors.  Here we review some of the latest advances in our understanding of tankyrase function in the pathway and efforts to modulate tankyrase activity to re-tune Wnt/β-catenin signalling in colorectal cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbg-X7McNWebVg90H21EOLACvtfcHk0li1qLueylEgSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslGqs7bE&md5=4ba71faa4fc2064e57390dae08b88c0f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1111%2Fbph.14038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.14038%26sid%3Dliteratum%253Aachs%26aulast%3DMariotti%26aufirst%3DL.%26aulast%3DPollock%26aufirst%3DK.%26aulast%3DGuettler%26aufirst%3DS.%26atitle%3DRegulation%2520of%2520Wnt%252F%25CE%25B2-catenin%2520signalling%2520by%2520tankyrase-dependent%2520poly%2528ADP-ribosyl%2529ation%2520and%2520scaffolding%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2017%26volume%3D174%26spage%3D4611%26epage%3D4636%26doi%3D10.1111%2Fbph.14038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">The
Cancer Genome Atlas Network.</span> <span> </span><span class="NLM_article-title">Comprehensive molecular characterization of human colon and rectal
cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>487</i></span>,  <span class="NLM_fpage">330</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nature11252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnature11252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22810696" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=330-337&author=The%0ACancer+Genome+Atlas+Network.&title=Comprehensive+molecular+characterization+of+human+colon+and+rectal%0Acancer&doi=10.1038%2Fnature11252"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature11252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11252%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520molecular%2520characterization%2520of%2520human%2520colon%2520and%2520rectal%250Acancer%26jtitle%3DNature%26date%3D2012%26volume%3D487%26spage%3D330%26epage%3D337%26doi%3D10.1038%2Fnature11252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vyas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, P.</span></span> <span> </span><span class="NLM_article-title">New PARP targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1038/nrc3748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnrc3748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=24898058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlKqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=502-509&author=S.+Vyasauthor=P.+Chang&title=New+PARP+targets+for+cancer+therapy&doi=10.1038%2Fnrc3748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">New PARP targets for cancer therapy</span></div><div class="casAuthors">Vyas, Sejal; Chang, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">502-509</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) modify target proteins post-translationally with poly(ADP-ribose) (PAR) or mono(ADP-ribose) (MAR) using NAD+ as substrate.  The best-studied PARPs generate PAR modifications and include PARP1 and the tankyrase PARP5A, both of which are targets for cancer therapy with inhibitors in either clin. trials or preclin. development.  There are 15 addnl. PARPs, most of which modify proteins with MAR, and their biol. is less well understood.  Recent data identify potentially cancer-relevant functions for these PARPs, which indicates that we need to understand more about these PARPs to effectively target them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouf6CtI5GhsrVg90H21EOLACvtfcHk0lg_NSTDf9qHJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlKqsb0%253D&md5=24a3b64d2d0518e13c9540ee012e29ca</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrc3748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3748%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DP.%26atitle%3DNew%2520PARP%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D502%26epage%3D509%26doi%3D10.1038%2Fnrc3748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nathubhai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Threadgill, M. D.</span></span> <span> </span><span class="NLM_article-title">Design and discovery of 2-arylquinazolin-4-ones as potent and selective inhibitors of tankyrases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1173</span>– <span class="NLM_lpage">1177</span>, <span class="refDoi"> DOI: 10.1021/ml400260b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400260b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aht7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1173-1177&author=A.+Nathubhaiauthor=P.+J.+Woodauthor=M.+D.+Lloydauthor=A.+S.+Thompsonauthor=M.+D.+Threadgill&title=Design+and+discovery+of+2-arylquinazolin-4-ones+as+potent+and+selective+inhibitors+of+tankyrases&doi=10.1021%2Fml400260b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Discovery of 2 Arylquinazolin-4-ones as Potent and Selective Inhibitors of Tankyrases</span></div><div class="casAuthors">Nathubhai, Amit; Wood, Pauline J.; Lloyd, Matthew D.; Thompson, Andrew S.; Threadgill, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1173-1177</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrases (TNKSs) are poly-(ADP-ribose)-polymerases (PARPs) that are overexpressed in several clin. cancers.  They regulate elongation of telomeres, regulate the Wnt system, and are essential for the function of the mitotic spindle.  A set of 2-arylquinazolin-4-ones has been designed and identified as potent and selective TNKS inhibitors, some being more potent and selective than the lead inhibitor XAV939, with IC50 = 3 nM vs.  TNKS-2.  Me was preferred at the 8-position and modest bulk at the 4-position of the 2-Ph group; electronic effects and H-bonding were irrelevant, but charge in the 4'-substituent must be avoided.  Mol. modeling facilitated initial design of the compds. and rationalization of the SAR of binding into the nicotinamide-binding site of the target enzymes.  These compds. have potential for further development into anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxfmX1o7LIsrVg90H21EOLACvtfcHk0lg_NSTDf9qHJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aht7nM&md5=23eb9a910b7952c76392bb1ae3b1185b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fml400260b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400260b%26sid%3Dliteratum%253Aachs%26aulast%3DNathubhai%26aufirst%3DA.%26aulast%3DWood%26aufirst%3DP.%2BJ.%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26aulast%3DThompson%26aufirst%3DA.%2BS.%26aulast%3DThreadgill%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520and%2520discovery%2520of%25202-arylquinazolin-4-ones%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520tankyrases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D1173%26epage%3D1177%26doi%3D10.1021%2Fml400260b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nathubhai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haikarainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Descalzo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Threadgill, M. D.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of 2-arylquinazolin-4-ones as highly selective and potent inhibitors of the Tankyrases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.ejmech.2016.04.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=27163581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslarsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=316-327&author=A.+Nathubhaiauthor=T.+Haikarainenauthor=P.+C.+Haywardauthor=S.+Mu%C3%B1oz-Descalzoauthor=A.+S.+Thompsonauthor=M.+D.+Lloydauthor=L.+Lehti%C3%B6author=M.+D.+Threadgill&title=Structure-activity+relationships+of+2-arylquinazolin-4-ones+as+highly+selective+and+potent+inhibitors+of+the+Tankyrases&doi=10.1016%2Fj.ejmech.2016.04.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of 2-arylquinazolin-4-ones as highly selective and potent inhibitors of the tankyrases</span></div><div class="casAuthors">Nathubhai, Amit; Haikarainen, Teemu; Hayward, Penelope C.; Munoz-Descalzo, Silvia; Thompson, Andrew S.; Lloyd, Matthew D.; Lehtio, Lari; Threadgill, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-327</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Tankyrases (TNKSs), members of the PARP (Poly(ADP-ribose)polymerases) superfamily of enzymes, have gained interest as therapeutic drug targets, esp. as they are involved in the regulation of Wnt signalling.  A series of 2-arylquinazolin-4-ones with varying substituents at the 8-position was synthesized.  An 8-Me group (compared to 8-H, 8-OMe, 8-OH), together with a 4'-hydrophobic or electron-withdrawing group, provided the most potency and selectivity towards TNKSs.  Co-crystal structures of selected compds. with TNKS-2 revealed that the protein around the 8-position is more hydrophobic in TNKS-2 compared to PARP-1/2, rationalising the selectivity.  The NAD+-binding site contains a hydrophobic cavity which accommodates the 2-aryl group; in TNKS-2, this has a tunnel to the exterior but the cavity is closed in PARP-1. 8-Methyl-2-(4-trifluoromethylphenyl)quinazolin-4-one(I) was identified as a potent and selective inhibitor of TNKSs and Wnt signalling.  This compd. and analogs could serve as mol. probes to study proliferative signalling and for development of inhibitors of TNKSs as drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYiaLuEeSrjbVg90H21EOLACvtfcHk0ljWbhH_F2GWtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslarsbs%253D&md5=c4bb8bad634199c8027e1ddf64c56f5e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.041%26sid%3Dliteratum%253Aachs%26aulast%3DNathubhai%26aufirst%3DA.%26aulast%3DHaikarainen%26aufirst%3DT.%26aulast%3DHayward%26aufirst%3DP.%2BC.%26aulast%3DMu%25C3%25B1oz-Descalzo%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DA.%2BS.%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26aulast%3DThreadgill%26aufirst%3DM.%2BD.%26atitle%3DStructure-activity%2520relationships%2520of%25202-arylquinazolin-4-ones%2520as%2520highly%2520selective%2520and%2520potent%2520inhibitors%2520of%2520the%2520Tankyrases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D316%26epage%3D327%26doi%3D10.1016%2Fj.ejmech.2016.04.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haikarainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkannagari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obaji, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pihlajaniemi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span> <span> </span><span class="NLM_article-title">para-Substituted 2-phenyl-3,4-dihydroquinazolin-4-ones as potent and selective Tankyrase inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1978</span>– <span class="NLM_lpage">1985</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1002%2Fcmdc.201300337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=24130191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1WmtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1978-1985&author=T.+Haikarainenauthor=J.+Koivunenauthor=M.+Narwalauthor=H.+Venkannagariauthor=E.+Obajiauthor=P.+Joensuuauthor=T.+Pihlajaniemiauthor=L.+Lehti%C3%B6&title=para-Substituted+2-phenyl-3%2C4-dihydroquinazolin-4-ones+as+potent+and+selective+Tankyrase+inhibitors&doi=10.1002%2Fcmdc.201300337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">para-Substituted 2-Phenyl-3,4-dihydroquinazolin-4-ones As Potent and Selective Tankyrase Inhibitors</span></div><div class="casAuthors">Haikarainen, Teemu; Koivunen, Jarkko; Narwal, Mohit; Venkannagari, Harikanth; Obaji, Ezeogo; Joensuu, Paeivi; Pihlajaniemi, Taina; Lehtioe, Lari</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1978-1985</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Human tankyrases are attractive drug targets, esp. for the treatment of cancer.  We identified a set of highly potent tankyrase inhibitors based on a 2-phenyl-3,4-dihydroquinazolin-4-one scaffold.  Substitutions at the para position of the scaffold's Ph group were evaluated as a strategy to increase potency and improve selectivity.  The best compds. displayed single-digit nanomolar potencies, and profiling against several human diphtheria-toxin-like ADP-ribosyltransferases revealed that a subset of these compds. are highly selective tankyrase inhibitors.  The compds. also effectively inhibit Wnt signaling in HEK293 cells.  The binding mode of all inhibitors was studied by protein X-ray crystallog.  This allowed us to establish a structural basis for the development of highly potent and selective tankyrase inhibitors based on the 2-phenyl-3,4-dihydroquinazolin-4-one scaffold and outline a rational approach to the modification of other inhibitor scaffolds that bind to the nicotinamide site of the catalytic domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFPGLvCfYMc7Vg90H21EOLACvtfcHk0li_MLIDGsUzug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1WmtbzK&md5=917b45b12c3170462210e8d260a1dd4c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300337%26sid%3Dliteratum%253Aachs%26aulast%3DHaikarainen%26aufirst%3DT.%26aulast%3DKoivunen%26aufirst%3DJ.%26aulast%3DNarwal%26aufirst%3DM.%26aulast%3DVenkannagari%26aufirst%3DH.%26aulast%3DObaji%26aufirst%3DE.%26aulast%3DJoensuu%26aufirst%3DP.%26aulast%3DPihlajaniemi%26aufirst%3DT.%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26atitle%3Dpara-Substituted%25202-phenyl-3%252C4-dihydroquinazolin-4-ones%2520as%2520potent%2520and%2520selective%2520Tankyrase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1978%26epage%3D1985%26doi%3D10.1002%2Fcmdc.201300337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman-Perez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teffera, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egge, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emkey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullady, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acquaviva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newcomb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strathdee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turci, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMauro, E. F.</span></span> <span> </span><span class="NLM_article-title">Development of novel dual Binders as potent, selective, and orally bioavailable tankyrase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10003</span>– <span class="NLM_lpage">10015</span>, <span class="refDoi"> DOI: 10.1021/jm401317z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401317z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=10003-10015&author=Z.+Huaauthor=H.+Bregmanauthor=J.+L.+Buchananauthor=N.+Chakkaauthor=A.+Guzman-Perezauthor=H.+Gunaydinauthor=X.+Huangauthor=Y.+Guauthor=V.+Berryauthor=J.+Liuauthor=Y.+Tefferaauthor=L.+Huangauthor=B.+Eggeauthor=R.+Emkeyauthor=E.+L.+Mulladyauthor=S.+Schneiderauthor=P.+S.+Andrewsauthor=L.+Acquavivaauthor=J.+Doveyauthor=A.+Mishraauthor=J.+Newcombauthor=D.+Saffranauthor=R.+Serafinoauthor=C.+A.+Strathdeeauthor=S.+M.+Turciauthor=M.+Stantonauthor=C.+Wilsonauthor=E.+F.+DiMauro&title=Development+of+novel+dual+Binders+as+potent%2C+selective%2C+and+orally+bioavailable+tankyrase+inhibitors&doi=10.1021%2Fjm401317z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Novel Dual Binders as Potent, Selective, and Orally Bioavailable Tankyrase Inhibitors</span></div><div class="casAuthors">Hua, Zihao; Bregman, Howard; Buchanan, John L.; Chakka, Nagasree; Guzman-Perez, Angel; Gunaydin, Hakan; Huang, Xin; Gu, Yan; Berry, Virginia; Liu, Jingzhou; Teffera, Yohannes; Huang, Liyue; Egge, Bryan; Emkey, Renee; Mullady, Erin L.; Schneider, Steve; Andrews, Paul S.; Acquaviva, Lisa; Dovey, Jennifer; Mishra, Ankita; Newcomb, John; Saffran, Douglas; Serafino, Randy; Strathdee, Craig A.; Turci, Susan M.; Stanton, Mary; Wilson, Cindy; DiMauro, Erin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10003-10015</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrases (TNKS1 and TNKS2) are proteins in the poly ADP-ribose polymerase (PARP) family.  They have been shown to directly bind to axin proteins, which neg. regulate the Wnt pathway by promoting β-catenin degrdn.  Inhibition of tankyrases may offer a novel approach to the treatment of APC-mutant colorectal cancer.  Hit compd. N-(2-methoxyphenyl)-4-(3-(4-oxo-3,4-dihydroquinazolin-2-yl)propanamido)benzamide was identified as an inhibitor of tankyrases through a combination of substructure searching of the Amgen compd. collection based on a minimal binding pharmacophore hypothesis and high-throughput screening.  Herein the authors report the structure- and property-based optimization of N-(2-methoxyphenyl)-4-(3-(4-oxo-3,4-dihydroquinazolin-2-yl)propanamido)benzamide leading to the identification of more potent and selective tankyrase inhibitors with improved pharmacokinetic properties in rodents, which are well suited as tool compds. for further in vivo validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-WJLflGn1C7Vg90H21EOLACvtfcHk0lidkm_HQLl9Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltrbN&md5=4620a3232873c1767b8289df2fac0c49</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm401317z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401317z%26sid%3Dliteratum%253Aachs%26aulast%3DHua%26aufirst%3DZ.%26aulast%3DBregman%26aufirst%3DH.%26aulast%3DBuchanan%26aufirst%3DJ.%2BL.%26aulast%3DChakka%26aufirst%3DN.%26aulast%3DGuzman-Perez%26aufirst%3DA.%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DBerry%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DEgge%26aufirst%3DB.%26aulast%3DEmkey%26aufirst%3DR.%26aulast%3DMullady%26aufirst%3DE.%2BL.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DAndrews%26aufirst%3DP.%2BS.%26aulast%3DAcquaviva%26aufirst%3DL.%26aulast%3DDovey%26aufirst%3DJ.%26aulast%3DMishra%26aufirst%3DA.%26aulast%3DNewcomb%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DD.%26aulast%3DSerafino%26aufirst%3DR.%26aulast%3DStrathdee%26aufirst%3DC.%2BA.%26aulast%3DTurci%26aufirst%3DS.%2BM.%26aulast%3DStanton%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DDiMauro%26aufirst%3DE.%2BF.%26atitle%3DDevelopment%2520of%2520novel%2520dual%2520Binders%2520as%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520tankyrase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D10003%26epage%3D10015%26doi%3D10.1021%2Fjm401317z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman-Perez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acquaviva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMauro, E. F.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of 2-aminopyridine oxazolidinones as potent and selective tankyrase inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1021/ml4003315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4003315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CktrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1218-1223&author=H.+Huangauthor=A.+Guzman-Perezauthor=L.+Acquavivaauthor=V.+Berryauthor=H.+Bregmanauthor=J.+Doveyauthor=H.+Gunaydinauthor=X.+Huangauthor=L.+Huangauthor=D.+Saffranauthor=R.+Serafinoauthor=S.+Schneiderauthor=C.+Wilsonauthor=E.+F.+DiMauro&title=Structure-based+design+of+2-aminopyridine+oxazolidinones+as+potent+and+selective+tankyrase+inhibitors&doi=10.1021%2Fml4003315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of 2-Aminopyridine Oxazolidinones as Potent and Selective Tankyrase Inhibitors</span></div><div class="casAuthors">Huang, Hongbing; Guzman-Perez, Angel; Acquaviva, Lisa; Berry, Virginia; Bregman, Howard; Dovey, Jennifer; Gunaydin, Hakan; Huang, Xin; Huang, Liyue; Saffran, Doug; Serafino, Randy; Schneider, Steve; Wilson, Cindy; DiMauro, Erin F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1218-1223</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant activation of the Wnt pathway has been implicated in the development and formation of many cancers.  TNKS inhibition has been shown to antagonize Wnt signaling via Axin stabilization in APC mutant colon cancer cell lines.  We employed structure-based design to identify a series of 2-aminopyridine oxazolidinones as potent and selective TNKS inhibitors.  These compds. exhibited good enzyme and cell potency as well as selectivity over other PARP isoforms.  Co-crystal structures of these 2-aminopyridine oxazolidinones complexed to TNKS reveal an induced-pocket binding mode that does not involve interactions with the nicotinamide binding pocket.  Oral dosing of lead compds. 3 and 4 resulted in significant effects on several Wnt-pathway biomarkers in a three day DLD-1 mouse tumor PD model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeQcoLA6UEfbVg90H21EOLACvtfcHk0lidkm_HQLl9Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CktrvP&md5=9aa2a27d3694d8ed6dded65ccf85d186</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fml4003315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4003315%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DGuzman-Perez%26aufirst%3DA.%26aulast%3DAcquaviva%26aufirst%3DL.%26aulast%3DBerry%26aufirst%3DV.%26aulast%3DBregman%26aufirst%3DH.%26aulast%3DDovey%26aufirst%3DJ.%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DSaffran%26aufirst%3DD.%26aulast%3DSerafino%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DDiMauro%26aufirst%3DE.%2BF.%26atitle%3DStructure-based%2520design%2520of%25202-aminopyridine%2520oxazolidinones%2520as%2520potent%2520and%2520selective%2520tankyrase%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D1218%26epage%3D1223%26doi%3D10.1021%2Fml4003315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="note"><p class="first last">Compound <b>1c</b> was reported as tankyrase inhibitor in</p></div><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wahlberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchiarulo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsell, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pol, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frostell, Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekblad, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Öncü, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kull, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigelt, J.</span></span> <span> </span><span class="NLM_article-title">Family-wide chemical profiling and structural analysis of PARP and tankyrase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">288</span>, <span class="refDoi"> DOI: 10.1038/nbt.2121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnbt.2121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22343925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVKrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=283-288&author=E.+Wahlbergauthor=T.+Karlbergauthor=E.+Kouznetsovaauthor=N.+Markovaauthor=A.+Macchiaruloauthor=A.+G.+Thorsellauthor=E.+Polauthor=%C3%85.+Frostellauthor=T.+Ekbladauthor=D.+%C3%96nc%C3%BCauthor=B.+Kullauthor=G.+M.+Robertsonauthor=R.+Pellicciariauthor=H.+Sch%C3%BClerauthor=J.+Weigelt&title=Family-wide+chemical+profiling+and+structural+analysis+of+PARP+and+tankyrase+inhibitors&doi=10.1038%2Fnbt.2121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Family-wide chemical profiling and structural analysis of PARP and tankyrase inhibitors</span></div><div class="casAuthors">Wahlberg, Elisabet; Karlberg, Tobias; Kouznetsova, Ekaterina; Markova, Natalia; Macchiarulo, Antonio; Thorsell, Ann-Gerd; Pol, Ewa; Frostell, Aasa; Ekblad, Torun; Oencue, Delal; Kull, Bjoern; Robertson, Graeme Michael; Pellicciari, Roberto; Schueler, Herwig; Weigelt, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-288</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibitors of poly-ADP-ribose polymerase (PARP) family proteins are currently in clin. trials as cancer therapeutics, yet the specificity of many of these compds. is unknown.  Here we evaluated a series of 185 small-mol. inhibitors, including research reagents and compds. being tested clin., for the ability to bind to the catalytic domains of 13 of the 17 human PARP family members including the tankyrases, TNKS1 and TNKS2.  Many of the best-known inhibitors, including TIQ-A, 6(5H)-phenanthridinone, olaparib, ABT-888 and rucaparib, bound to several PARP family members, suggesting that these mols. lack specificity and have promiscuous inhibitory activity.  We also detd. X-ray crystal structures for five TNKS2 ligand complexes and four PARP14 ligand complexes.  In addn. to showing that the majority of PARP inhibitors bind multiple targets, these results provide insight into the design of new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYSebsCVufibVg90H21EOLACvtfcHk0lgVlOt-eoSfdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVKrsb4%253D&md5=71f17fd5b6bb602dd70c8fcc17df95f9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2121%26sid%3Dliteratum%253Aachs%26aulast%3DWahlberg%26aufirst%3DE.%26aulast%3DKarlberg%26aufirst%3DT.%26aulast%3DKouznetsova%26aufirst%3DE.%26aulast%3DMarkova%26aufirst%3DN.%26aulast%3DMacchiarulo%26aufirst%3DA.%26aulast%3DThorsell%26aufirst%3DA.%2BG.%26aulast%3DPol%26aufirst%3DE.%26aulast%3DFrostell%26aufirst%3D%25C3%2585.%26aulast%3DEkblad%26aufirst%3DT.%26aulast%3D%25C3%2596nc%25C3%25BC%26aufirst%3DD.%26aulast%3DKull%26aufirst%3DB.%26aulast%3DRobertson%26aufirst%3DG.%2BM.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DSch%25C3%25BCler%26aufirst%3DH.%26aulast%3DWeigelt%26aufirst%3DJ.%26atitle%3DFamily-wide%2520chemical%2520profiling%2520and%2520structural%2520analysis%2520of%2520PARP%2520and%2520tankyrase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2012%26volume%3D30%26spage%3D283%26epage%3D288%26doi%3D10.1038%2Fnbt.2121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Motifs for molecular recognition exploiting hydrophobic enclosure in protein–ligand binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+molecular+recognition+exploiting+hydrophobic+enclosure+in+protein%E2%80%93ligand+binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0lgVlOt-eoSfdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520molecular%2520recognition%2520exploiting%2520hydrophobic%2520enclosure%2520in%2520protein%25E2%2580%2593ligand%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the Active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+Active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lh8TtWMWCB4eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520Active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulak, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holohan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garofalo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, L.</span></span> <span> </span><span class="NLM_article-title">Disruption of Wnt/β-catenin signaling and telomeric shortening are inextricable consequences of tankyrase inhibition in human cells</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2425</span>– <span class="NLM_lpage">2435</span>, <span class="refDoi"> DOI: 10.1128/MCB.00392-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1128%2FMCB.00392-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=25939383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaqtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=2425-2435&author=O.+Kulakauthor=H.+Chenauthor=B.+Holohanauthor=X.+Wuauthor=H.+Heauthor=D.+Borekauthor=Z.+Otwinowskiauthor=K.+Yamaguchiauthor=L.+A.+Garofaloauthor=Z.+Maauthor=W.+Wrightauthor=C.+Chenauthor=J.+W.+Shayauthor=X.+Zhangauthor=L.+Lum&title=Disruption+of+Wnt%2F%CE%B2-catenin+signaling+and+telomeric+shortening+are+inextricable+consequences+of+tankyrase+inhibition+in+human+cells&doi=10.1128%2FMCB.00392-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of Wnt/β-catenin signaling and telomeric shortening are inextricable consequences of tankyrase inhibition in human cells</span></div><div class="casAuthors">Kulak, Ozlem; Chen, Hua; Holohan, Brody; Wu, Xiaofeng; He, Huawei; Borek, Dominika; Otwinowski, Zbyszek; Yamaguchi, Kiyoshi; Garofalo, Lauren A.; Ma, Zhiqiang; Wright, Woodring; Chen, Chuo; Shay, Jerry W.; Zhang, Xuewu; Lum, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2425-2435</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Maintenance of chromosomal ends (telomeres) directly contributes to cancer cell immortalization.  The telomere protection enzymes belonging to the tankyrase (Tnks) subfamily of poly(ADP-ribose) polymerases (PARPs) have recently been shown to also control transcriptional response to secreted Wnt signaling mols.  Whereas Tnks inhibitors are currently being developed as therapeutic agents for targeting Wnt-related cancers and as modulators of Wnt signaling in tissue-engineering agendas, their impact on telomere length maintenance remains unclear.  Here, we leveraged a collection of Wnt pathway inhibitors with previously unassigned mechanisms of action to identify novel pharmacophores supporting Tnks inhibition.  A multifaceted exptl. approach that included structural, biochem., and cell biol. analyses revealed two distinct chemotypes with selectivity for Tnks enzymes.  Using these reagents, we revealed that Tnks inhibition rapidly induces DNA damage at telomeres and telomeric shortening upon long-term chem. exposure in cultured cells.  On the other hand, inhibitors of the Wnt acyltransferase Porcupine (Porcn) elicited neither effect.  Thus, Tnks inhibitors impact telomere length maintenance independently of their affects on Wnt/β-catenin signaling.  We discuss the implications of these findings for anticancer and regenerative medicine agendas dependent upon chem. inhibitors of Wnt/β-catenin signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsPK9Wr28kGbVg90H21EOLACvtfcHk0lh8TtWMWCB4eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaqtrrP&md5=c1b52764be92929c71d1d51cbcc1cac5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FMCB.00392-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00392-15%26sid%3Dliteratum%253Aachs%26aulast%3DKulak%26aufirst%3DO.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DHolohan%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DBorek%26aufirst%3DD.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DYamaguchi%26aufirst%3DK.%26aulast%3DGarofalo%26aufirst%3DL.%2BA.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DWright%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DShay%26aufirst%3DJ.%2BW.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLum%26aufirst%3DL.%26atitle%3DDisruption%2520of%2520Wnt%252F%25CE%25B2-catenin%2520signaling%2520and%2520telomeric%2520shortening%2520are%2520inextricable%2520consequences%2520of%2520tankyrase%2520inhibition%2520in%2520human%2520cells%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2015%26volume%3D35%26spage%3D2425%26epage%3D2435%26doi%3D10.1128%2FMCB.00392-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafiq, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Cabral, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munkonge, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinidade, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esdar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohdich, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansom, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wienke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span> <span> </span><span class="NLM_article-title">A novel tankyrase inhibitor, MSC2504877, enhances the effects of clinical CDK4/6 inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">201</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-36447-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fs41598-018-36447-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=30655555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A280%3ADC%252BB3cjis1Gjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=201&author=M.+Menonauthor=R.+Elliottauthor=L.+Bowersauthor=N.+Balanauthor=R.+Rafiqauthor=S.+Costa-Cabralauthor=F.+Munkongeauthor=I.+Trinidadeauthor=R.+Porterauthor=A.+D.+Campbellauthor=E.+R.+Johnsonauthor=C.+Esdarauthor=H.-P.+Buchstallerauthor=B.+Leuthnerauthor=F.+Rohdichauthor=R.+Schneiderauthor=O.+Sansomauthor=D.+Wienkeauthor=A.+Ashworthauthor=C.+J.+Lord&title=A+novel+tankyrase+inhibitor%2C+MSC2504877%2C+enhances+the+effects+of+clinical+CDK4%2F6+inhibitors&doi=10.1038%2Fs41598-018-36447-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A novel tankyrase inhibitor, MSC2504877, enhances the effects of clinical CDK4/6 inhibitors</span></div><div class="casAuthors">Menon Malini; Elliott Richard; Bowers Leandra; Balan Nicolae; Rafiq Rumana; Costa-Cabral Sara; Munkonge Felix; Trinidade Ines; Ashworth Alan; Lord Christopher J; Porter Roderick; Campbell Andrew D; Johnson Emma R; Sansom Owen; Esdar Christina; Buchstaller Hans-Peter; Leuthner Birgitta; Rohdich Felix; Schneider Richard; Wienke Dirk; Ashworth Alan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">201</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Inhibition of the PARP superfamily tankyrase enzymes suppresses Wnt/β-catenin signalling in tumour cells.  Here, we describe here a novel, drug-like small molecule inhibitor of tankyrase MSC2504877 that inhibits the growth of APC mutant colorectal tumour cells.  Parallel siRNA and drug sensitivity screens showed that the clinical CDK4/6 inhibitor palbociclib, causes enhanced sensitivity to MSC2504877.  This tankyrase inhibitor-CDK4/6 inhibitor combinatorial effect is not limited to palbociclib and MSC2504877 and is elicited with other CDK4/6 inhibitors and toolbox tankyrase inhibitors.  The addition of MSC2504877 to palbociclib enhances G1 cell cycle arrest and cellular senescence in tumour cells.  MSC2504877 exposure suppresses the upregulation of Cyclin D2 and Cyclin E2 caused by palbociclib and enhances the suppression of phospho-Rb, providing a mechanistic explanation for these effects.  The combination of MSC2504877 and palbociclib was also effective in suppressing the cellular hyperproliferative phenotype seen in Apc defective intestinal stem cells in vivo.  However, the presence of an oncogenic Kras p.G12D mutation in mice reversed the effects of the MSC2504877/palbociclib combination, suggesting one molecular route that could lead to drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRsCECdrvD2BkewzGPswiWOfW6udTcc2ebeD51Ugwlz-rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjis1Gjtg%253D%253D&md5=34ac002987c951a0ebd2f34f256ed21c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-36447-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-36447-4%26sid%3Dliteratum%253Aachs%26aulast%3DMenon%26aufirst%3DM.%26aulast%3DElliott%26aufirst%3DR.%26aulast%3DBowers%26aufirst%3DL.%26aulast%3DBalan%26aufirst%3DN.%26aulast%3DRafiq%26aufirst%3DR.%26aulast%3DCosta-Cabral%26aufirst%3DS.%26aulast%3DMunkonge%26aufirst%3DF.%26aulast%3DTrinidade%26aufirst%3DI.%26aulast%3DPorter%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DA.%2BD.%26aulast%3DJohnson%26aufirst%3DE.%2BR.%26aulast%3DEsdar%26aufirst%3DC.%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DRohdich%26aufirst%3DF.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DSansom%26aufirst%3DO.%26aulast%3DWienke%26aufirst%3DD.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26atitle%3DA%2520novel%2520tankyrase%2520inhibitor%252C%2520MSC2504877%252C%2520enhances%2520the%2520effects%2520of%2520clinical%2520CDK4%252F6%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D201%26doi%3D10.1038%2Fs41598-018-36447-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPietro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palermo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touré, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span> <span> </span><span class="NLM_article-title">Identification of NVP-TNKS656: The use of structure–efficiency relationships to generate a highly potent, selective, and orally active tankyrase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6495</span>– <span class="NLM_lpage">6511</span>, <span class="refDoi"> DOI: 10.1021/jm400807n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400807n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtV2msbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6495-6511&author=M.+D.+Shultzauthor=A.+K.+Cheungauthor=C.+A.+Kirbyauthor=B.+Firestoneauthor=J.+Fanauthor=C.+H.-T.+Chenauthor=Z.+Chenauthor=D.+N.+Chinauthor=L.+DiPietroauthor=A.+Fazalauthor=Y.+Fengauthor=P.+D.+Fortinauthor=T.+Gouldauthor=B.+Laguauthor=H.+Leiauthor=F.+Lenoirauthor=D.+Majumdarauthor=E.+Ochalaauthor=M.+G.+Palermoauthor=L.+Phamauthor=M.+Puauthor=T.+Smithauthor=T.+Stamsauthor=R.+C.+Tomlinsonauthor=B.+B.+Tour%C3%A9author=M.+Visserauthor=R.+M.+Wangauthor=N.+J.+Watersauthor=W.+Shao&title=Identification+of+NVP-TNKS656%3A+The+use+of+structure%E2%80%93efficiency+relationships+to+generate+a+highly+potent%2C+selective%2C+and+orally+active+tankyrase+inhibitor&doi=10.1021%2Fjm400807n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of NVP-TNKS656: The Use of Structure-Efficiency Relationships To Generate a Highly Potent, Selective, and Orally Active Tankyrase Inhibitor</span></div><div class="casAuthors">Shultz, Michael D.; Cheung, Atwood K.; Kirby, Christina A.; Firestone, Brant; Fan, Jianmei; Chen, Christine Hiu-Tung; Chen, Zhouliang; Chin, Donovan N.; DiPietro, Lucian; Fazal, Aleem; Feng, Yun; Fortin, Pascal D.; Gould, Ty; Lagu, Bharat; Lei, Huangshu; Lenoir, Francois; Majumdar, Dyuti; Ochala, Etienne; Palermo, M. G.; Pham, Ly; Pu, Minying; Smith, Troy; Stams, Travis; Tomlinson, Ronald C.; Toure, B. Barry; Visser, Michael; Wang, Run Ming; Waters, Nigel J.; Shao, Wenlin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6495-6511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrase 1 and 2 have been shown to be redundant, druggable nodes in the Wnt pathway.  As such, there has been intense interest in developing agents suitable for modulating the Wnt pathway in vivo by targeting this enzyme pair.  By utilizing a combination of structure-based design and LipE-based structure efficiency relationships, the core of XAV939 was optimized into a more stable, more efficient, but less potent dihydropyran motif 7.  This core was combined with elements of screening hits 2, 19, and 33 and resulted in highly potent, selective tankyrase inhibitors that are novel three pocket binders.  NVP-TNKS656 (43) was identified as an orally active antagonist of Wnt pathway activity in the MMTV-Wnt1 mouse xenograft model.  With an enthalpy-driven thermodn. signature of binding, highly favorable physicochem. properties, and high lipophilic efficiency, NVP-TNKS656 is a novel tankyrase inhibitor that is well suited for further in vivo validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDGWOimaJ0I7Vg90H21EOLACvtfcHk0liCkLb5gjBPIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2msbfE&md5=db3f1a471ca757e0219a5175c5017e9b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400807n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400807n%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BH.-T.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DDiPietro%26aufirst%3DL.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DFortin%26aufirst%3DP.%2BD.%26aulast%3DGould%26aufirst%3DT.%26aulast%3DLagu%26aufirst%3DB.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DOchala%26aufirst%3DE.%26aulast%3DPalermo%26aufirst%3DM.%2BG.%26aulast%3DPham%26aufirst%3DL.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DT.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DTomlinson%26aufirst%3DR.%2BC.%26aulast%3DTour%25C3%25A9%26aufirst%3DB.%2BB.%26aulast%3DVisser%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DShao%26aufirst%3DW.%26atitle%3DIdentification%2520of%2520NVP-TNKS656%253A%2520The%2520use%2520of%2520structure%25E2%2580%2593efficiency%2520relationships%2520to%2520generate%2520a%2520highly%2520potent%252C%2520selective%252C%2520and%2520orally%2520active%2520tankyrase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6495%26epage%3D6511%26doi%3D10.1021%2Fjm400807n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarravarapu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulak, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Regulation of tankyrase activity by a catalytic domain dimer interface</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">1780</span>– <span class="NLM_lpage">1785</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2018.07.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.bbrc.2018.07.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=30055800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVShtrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2018&pages=1780-1785&author=C.+Fanauthor=N.+Yarravarapuauthor=H.+Chenauthor=O.+Kulakauthor=P.+Dasariauthor=J.+Herbertauthor=K.+Yamaguchiauthor=L.+Lumauthor=X.+Zhang&title=Regulation+of+tankyrase+activity+by+a+catalytic+domain+dimer+interface&doi=10.1016%2Fj.bbrc.2018.07.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of tankyrase activity by a catalytic domain dimer interface</span></div><div class="casAuthors">Fan, Chen; Yarravarapu, Nageswari; Chen, Hua; Kulak, Ozlem; Dasari, Pranathi; Herbert, Jeremiah; Yamaguchi, Kiyoshi; Lum, Lawrence; Zhang, Xuewu</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1780-1785</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tankyrases (TNKS and TNKS2) are enzymes that catalyze poly-ADP-ribosylation (PARsylation) of their target proteins.  Tankyrase-mediated PARsylation plays crit. regulatory roles in cell signaling, particularly in the Wnt/β-catenin pathway.  The sterile alpha motif (SAM) domain in tankyrases mediates their oligomerization, which is essential for tankyrase function.  The oligomerization regulates the catalytic activity of tankyrases, but the underlying mechanism is unclear.  Our analyses of crystal structures of the tankyrase catalytic domain suggest that formation of a head-to-head dimer regulates the catalytic activity.  Our activity assays show that residues in the catalytic domain dimer interface are important for the PARsylation activity of tankyrases both in soln. and cells.  The dimer is weak and may only form in the context of the SAM domain-mediated oligomers of tankyrases, consistent with the dependence of the tankyrase activity on the SAM domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrAIbzOSgzWbVg90H21EOLACvtfcHk0lh8oT4AmKPbcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVShtrbN&md5=6dc34e2cb632da1f34dac69c4119b79f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2018.07.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2018.07.113%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DC.%26aulast%3DYarravarapu%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKulak%26aufirst%3DO.%26aulast%3DDasari%26aufirst%3DP.%26aulast%3DHerbert%26aufirst%3DJ.%26aulast%3DYamaguchi%26aufirst%3DK.%26aulast%3DLum%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DRegulation%2520of%2520tankyrase%2520activity%2520by%2520a%2520catalytic%2520domain%2520dimer%2520interface%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2018%26volume%3D503%26spage%3D1780%26epage%3D1785%26doi%3D10.1016%2Fj.bbrc.2018.07.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marala, R. B. J.</span></span> <span> </span><span class="NLM_article-title">Development of a high-throughput screening-amenable assay for human poly(ADP-ribose) polymerase inhibitors</span>. <i>J. Pharmacol. Toxicol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/S1056-8719(02)00223-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2FS1056-8719%2802%2900223-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=12628304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhs1ahs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2002&pages=137-141&author=J.+A.+Brownauthor=R.+B.+J.+Marala&title=Development+of+a+high-throughput+screening-amenable+assay+for+human+poly%28ADP-ribose%29+polymerase+inhibitors&doi=10.1016%2FS1056-8719%2802%2900223-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a high-throughput screening-amenable assay for human poly(ADP-ribose) polymerase inhibitors</span></div><div class="casAuthors">Brown, Janice A.; Marala, Ravi B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological and Toxicological Methods</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">137-141</span>CODEN:
                <span class="NLM_cas:coden">JPTMEZ</span>;
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase (PARP) plays a pivotal role in the repair of DNA strand breaks.  However, excessive activation of PARP causes a rapid depletion of intracellular energy, leading to cell death.  Inhibitors of PARP have been shown to reduce infarct size in animal models of myocardial ischemia.  PARP inhibitors may have potential therapeutic benefit in the treatment of myocardial ischemia, stroke, head trauma, and neurodegenerative disease, and as an adjunct therapy with chemotherapeutic agents/radiation in cancer therapy.  Assays reported in the literature and com. available PARP assay kits are labor-intensive, use radioactive reagents, use antibodies, and are not readily amenable to Hadamard transform spectroscopy (HTS).  Here we report the development and the validation of a nonradioactive PARP assay suitable for HTS.  This is a biotinylated NAD-based colorimetric assay in a 96-well plate format.  The assay is sensitive, reproducible, and easy to use.  The IC50 values generated for the known PARP inhibitors are in agreement with those generated using the com. radioactive kit and those reported in the literature.  The present study demonstrates a sensitive and reproducible methodol. capable of screening human PARP inhibitors in high-throughput format.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3Me-JvpXPsLVg90H21EOLACvtfcHk0lh8oT4AmKPbcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhs1ahs7o%253D&md5=d1eb6600deeaa8abbaac3573043a3138</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS1056-8719%2802%2900223-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1056-8719%252802%252900223-X%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DMarala%26aufirst%3DR.%2BB.%2BJ.%26atitle%3DDevelopment%2520of%2520a%2520high-throughput%2520screening-amenable%2520assay%2520for%2520human%2520poly%2528ADP-ribose%2529%2520polymerase%2520inhibitors%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2002%26volume%3D47%26spage%3D137%26epage%3D141%26doi%3D10.1016%2FS1056-8719%2802%2900223-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span> <span> </span><span class="NLM_article-title">Novel binding mode of a potent and selective tankyrase inhibitor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e33740</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0033740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1371%2Fjournal.pone.0033740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22438990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVaht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=H.+Gunaydinauthor=Y.+Guauthor=X.+Huang&title=Novel+binding+mode+of+a+potent+and+selective+tankyrase+inhibitor&doi=10.1371%2Fjournal.pone.0033740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel binding mode of a potent and selective tankyrase inhibitor</span></div><div class="casAuthors">Gunaydin, Hakan; Gu, Yan; Huang, Xin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e33740</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Tankyrases (TNKS1 and TNKS2) are key regulators of cellular processes such as telomere pathway and Wnt signaling.  IWRs (inhibitors of Wnt response) have recently been identified as potent and selective inhibitors of tankyrases.  However, it is not clear how these IWRs interact with tankyrases.  Here we report the crystal structure of the catalytic domain of human TNKS1 in complex with IWR2, which reveals a novel binding site for tankyrase inhibitors.  The TNKS1/IWR2 complex provides a mol. basis for their strong and specific interactions and suggests clues for further development of tankyrase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqidLqR8KN4obVg90H21EOLACvtfcHk0lh8oT4AmKPbcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVaht70%253D&md5=7ab7c96221b170534538c51605e76e85</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0033740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0033740%26sid%3Dliteratum%253Aachs%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DNovel%2520binding%2520mode%2520of%2520a%2520potent%2520and%2520selective%2520tankyrase%2520inhibitor%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0033740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casanas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warshamanage, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finke, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panepucci, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olieric, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nöll, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tampé, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Förster, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze-Briese, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunk, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span> <span> </span><span class="NLM_article-title">EIGER detector: Application in macromolecular crystallography</span>. <i>Acta Crystallogr. Sect. D Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1036</span>– <span class="NLM_lpage">1048</span>, <span class="refDoi"> DOI: 10.1107/S2059798316012304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS2059798316012304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=1036-1048&author=A.+Casanasauthor=R.+Warshamanageauthor=A.+D.+Finkeauthor=E.+Panepucciauthor=V.+Oliericauthor=A.+N%C3%B6llauthor=R.+Tamp%C3%A9author=S.+Brandstetterauthor=A.+F%C3%B6rsterauthor=M.+Muellerauthor=C.+Schulze-Brieseauthor=O.+Bunkauthor=M.+Wang&title=EIGER+detector%3A+Application+in+macromolecular+crystallography&doi=10.1107%2FS2059798316012304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">EIGER detector: application in macromolecular crystallography</span></div><div class="casAuthors">Casanas, Arnau; Warshamanage, Rangana; Finke, Aaron D.; Panepucci, Ezequiel; Olieric, Vincent; Noll, Anne; Tampe, Robert; Brandstetter, Stefan; Forster, Andreas; Mueller, Marcus; Schulze-Briese, Clemens; Bunk, Oliver; Wang, Meitian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1036-1048</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The development of single-photon-counting detectors, such as the PILATUS, has been a major recent breakthrough in macromol. crystallog., enabling noise-free detection and novel data-acquisition modes.  The new EIGER detector features a pixel size of 75 × 75 μm, frame rates of up to 3000 Hz and a dead time as low as 3.8 μs.  An EIGER 1M and EIGER 16M were tested on Swiss Light Source beamlines X10SA and X06SA for their application in macromol. crystallog.  The combination of fast frame rates and a very short dead time allows high-quality data acquisition in a shorter time.  The ultrafine φ-slicing data-collection method is introduced and validated and its application in finding the optimal rotation angle, a suitable rotation speed and a sufficient X-ray dose are presented.  An improvement of the data quality up to slicing at one tenth of the mosaicity has been obsd., which is much finer than expected based on previous findings.  The influence of key data-collection parameters on data quality is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2BJ29hEBUhrVg90H21EOLACvtfcHk0liQDT4XkI7uzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7I&md5=8230d3621ab53559b2f120d6a98c40de</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS2059798316012304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316012304%26sid%3Dliteratum%253Aachs%26aulast%3DCasanas%26aufirst%3DA.%26aulast%3DWarshamanage%26aufirst%3DR.%26aulast%3DFinke%26aufirst%3DA.%2BD.%26aulast%3DPanepucci%26aufirst%3DE.%26aulast%3DOlieric%26aufirst%3DV.%26aulast%3DN%25C3%25B6ll%26aufirst%3DA.%26aulast%3DTamp%25C3%25A9%26aufirst%3DR.%26aulast%3DBrandstetter%26aufirst%3DS.%26aulast%3DF%25C3%25B6rster%26aufirst%3DA.%26aulast%3DMueller%26aufirst%3DM.%26aulast%3DSchulze-Briese%26aufirst%3DC.%26aulast%3DBunk%26aufirst%3DO.%26aulast%3DWang%26aufirst%3DM.%26atitle%3DEIGER%2520detector%253A%2520Application%2520in%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D1036%26epage%3D1048%26doi%3D10.1107%2FS2059798316012304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kabsch%2C+W.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+2010%2C+66%2C+125%E2%80%93132%2C+10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0liQDT4XkI7uzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Collaborative
computational project, N. 4</span> <span> </span><span class="NLM_article-title">The CCP4 suite: Programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0Acomputational+project%2C+N.+4&title=The+CCP4+suite%3A+Programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520Programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version 2.11.7; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhYrscjGN9muA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luckert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götschel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joos, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pötz, O.</span></span> <span> </span><span class="NLM_article-title">Snapshots of protein dynamics and post-translational modifications in one experiment - beta-catenin and its functions</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">M110.007377</span>, <span class="refDoi"> DOI: 10.1074/mcp.M110.007377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1074%2Fmcp.M110.007377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=21378377" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M110.007377&author=K.+Luckertauthor=F.+G%C3%B6tschelauthor=P.+K.+Sorgerauthor=A.+Hechtauthor=T.+O.+Joosauthor=O.+P%C3%B6tz&title=Snapshots+of+protein+dynamics+and+post-translational+modifications+in+one+experiment+-+beta-catenin+and+its+functions&doi=10.1074%2Fmcp.M110.007377"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M110.007377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M110.007377%26sid%3Dliteratum%253Aachs%26aulast%3DLuckert%26aufirst%3DK.%26aulast%3DG%25C3%25B6tschel%26aufirst%3DF.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DHecht%26aufirst%3DA.%26aulast%3DJoos%26aufirst%3DT.%2BO.%26aulast%3DP%25C3%25B6tz%26aufirst%3DO.%26atitle%3DSnapshots%2520of%2520protein%2520dynamics%2520and%2520post-translational%2520modifications%2520in%2520one%2520experiment%2520-%2520beta-catenin%2520and%2520its%2520functions%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DM110.007377%26doi%3D10.1074%2Fmcp.M110.007377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luckert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gujral, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevecka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joos, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macbeath, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pötz, O.</span></span> <span> </span><span class="NLM_article-title">A dual array-based approach to assess the abundance and posttranslational modification state of signaling proteins</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">pl1</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2002372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1126%2Fscisignal.2002372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22234610" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=pl1&author=K.+Luckertauthor=T.+S.+Gujralauthor=M.+Chanauthor=M.+Seveckaauthor=T.+O.+Joosauthor=P.+K.+Sorgerauthor=G.+Macbeathauthor=O.+P%C3%B6tz&title=A+dual+array-based+approach+to+assess+the+abundance+and+posttranslational+modification+state+of+signaling+proteins&doi=10.1126%2Fscisignal.2002372"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2002372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2002372%26sid%3Dliteratum%253Aachs%26aulast%3DLuckert%26aufirst%3DK.%26aulast%3DGujral%26aufirst%3DT.%2BS.%26aulast%3DChan%26aufirst%3DM.%26aulast%3DSevecka%26aufirst%3DM.%26aulast%3DJoos%26aufirst%3DT.%2BO.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DMacbeath%26aufirst%3DG.%26aulast%3DP%25C3%25B6tz%26aufirst%3DO.%26atitle%3DA%2520dual%2520array-based%2520approach%2520to%2520assess%2520the%2520abundance%2520and%2520posttranslational%2520modification%2520state%2520of%2520signaling%2520proteins%26jtitle%3DSci.%2520Signaling%26date%3D2012%26volume%3D5%26spage%3Dpl1%26doi%3D10.1126%2Fscisignal.2002372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petereit, A. C.</span></span> <span> </span><span class="NLM_article-title">Optimizing solubility: kinetic versus thermodynamic solubility temptations and risks</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2012.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.ejps.2012.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22885099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=589-595&author=C.+Saalauthor=A.+C.+Petereit&title=Optimizing+solubility%3A+kinetic+versus+thermodynamic+solubility+temptations+and+risks&doi=10.1016%2Fj.ejps.2012.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing solubility: Kinetic versus thermodynamic solubility temptations and risks</span></div><div class="casAuthors">Saal, Christoph; Petereit, Anna Christine</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aim of this study was to assess the usefulness of kinetic and thermodn. soly. data in guiding medicinal chem. during lead optimization.  The soly. of 465 research compds. was measured using a kinetic and a thermodn. soly. assay.  In the thermodn. assay, polarized-light microscopy was used to investigate whether the result referred to the cryst. or to the amorphous compd.  From the comparison of kinetic and thermodn. soly. data it was noted that kinetic soly. measurements frequently yielded results which show considerably higher soly. compared to thermodn. soly.  This observation is ascribed to the fact that a kinetic soly. assay typically delivers results which refer to the amorphous compd.  In contrast, results from thermodn. soly. detns. more frequently refer to a cryst. phase.  Accordingly, thermodn. soly. data - esp. when used together with an assessment of the solid state form - are deemed to be more useful in guiding soly. optimization for research compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooxKTQHBpynbVg90H21EOLACvtfcHk0liKABK99YOGig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ&md5=ace8b962c04c3a62f028ec3e66522922</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2012.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2012.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DSaal%26aufirst%3DC.%26aulast%3DPetereit%26aufirst%3DA.%2BC.%26atitle%3DOptimizing%2520solubility%253A%2520kinetic%2520versus%2520thermodynamic%2520solubility%2520temptations%2520and%2520risks%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2012%26volume%3D47%26spage%3D589%26epage%3D595%26doi%3D10.1016%2Fj.ejps.2012.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Hans-Peter Buchstaller, Uwe Anlauf, Dieter Dorsch, Sarah Kögler, Daniel Kuhn, Martin Lehmann, Birgitta Leuthner, Sara Lodholz, Djordje Musil, Daniela Radtki, Corinna Rettig, Claudio Ritzert, Felix Rohdich, Richard Schneider, Ansgar Wegener, Stefan Weigt, Kai Wilkinson, <span class="NLM_string-name hlFld-ContribAuthor">Christina Esdar</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of a Screening Hit toward M2912, an Oral Tankyrase Inhibitor with Antitumor Activity in Colorectal Cancer Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (14)
                                     , 10371-10392. <a href="https://doi.org/10.1021/acs.jmedchem.1c00800" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00800</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00800%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOptimization%252Bof%252Ba%252BScreening%252BHit%252Btoward%252BM2912%25252C%252Ban%252BOral%252BTankyrase%252BInhibitor%252Bwith%252BAntitumor%252BActivity%252Bin%252BColorectal%252BCancer%252BModels%26aulast%3DBuchstaller%26aufirst%3DHans-Peter%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D03052021%26date%3D13072021%26volume%3D64%26issue%3D14%26spage%3D10371%26epage%3D10392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhiqing Liu, Pingyuan Wang, Eric A. Wold, Qiaoling Song, Chenyang Zhao, Changyun Wang, <span class="NLM_string-name hlFld-ContribAuthor">Jia Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">Small-Molecule Inhibitors Targeting the Canonical WNT Signaling Pathway for the Treatment of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (8)
                                     , 4257-4288. <a href="https://doi.org/10.1021/acs.jmedchem.0c01799" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01799</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01799%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall-Molecule%252BInhibitors%252BTargeting%252Bthe%252BCanonical%252BWNT%252BSignaling%252BPathway%252Bfor%252Bthe%252BTreatment%252Bof%252BCancer%26aulast%3DLiu%26aufirst%3DZhiqing%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14102020%26date%3D06042021%26volume%3D64%26issue%3D8%26spage%3D4257%26epage%3D4288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christina E. M. Schindler, Hannah Baumann, Andreas Blum, Dietrich Böse, Hans-Peter Buchstaller, Lars Burgdorf, Daniel Cappel, Eugene Chekler, Paul Czodrowski, Dieter Dorsch, Merveille K. I. Eguida, Bruce Follows, Thomas Fuchß, Ulrich Grädler, Jakub Gunera, Theresa Johnson, Catherine Jorand Lebrun, Srinivasa Karra, Markus Klein, Tim Knehans, Lisa Koetzner, Mireille Krier, Matthias Leiendecker, Birgitta Leuthner, Liwei Li, Igor Mochalkin, Djordje Musil, Constantin Neagu, Friedrich Rippmann, Kai Schiemann, Robert Schulz, Thomas Steinbrecher, Eva-Maria Tanzer, Andrea Unzue Lopez, Ariele Viacava Follis, Ansgar Wegener, <span class="NLM_string-name hlFld-ContribAuthor">Daniel Kuhn</span>. </span><span class="cited-content_cbyCitation_article-title">Large-Scale Assessment of Binding Free Energy Calculations in Active Drug Discovery Projects. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (11)
                                     , 5457-5474. <a href="https://doi.org/10.1021/acs.jcim.0c00900" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00900</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00900&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00900%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLarge-Scale%252BAssessment%252Bof%252BBinding%252BFree%252BEnergy%252BCalculations%252Bin%252BActive%252BDrug%252BDiscovery%252BProjects%26aulast%3DSchindler%26aufirst%3DChristina%2BE.%2BM.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D07082020%26date%3D03092020%26date%3D19082020%26volume%3D60%26issue%3D11%26spage%3D5457%26epage%3D5474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jo Waaler, Ruben G. G. Leenders, Sven T. Sowa, Shoshy Alam Brinch, Max Lycke, Piotr Nieczypor, Sjoerd Aertssen, Sudarshan Murthy, Albert Galera-Prat, Eddy Damen, Anita Wegert, Marc Nazaré, Lari Lehtiö, <span class="NLM_string-name hlFld-ContribAuthor">Stefan Krauss</span>. </span><span class="cited-content_cbyCitation_article-title">Preclinical Lead Optimization of a 1,2,4-Triazole Based Tankyrase Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 6834-6846. <a href="https://doi.org/10.1021/acs.jmedchem.0c00208" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00208</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00208%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPreclinical%252BLead%252BOptimization%252Bof%252Ba%252B1%25252C2%25252C4-Triazole%252BBased%252BTankyrase%252BInhibitor%26aulast%3DWaaler%26aufirst%3DJo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D05022020%26date%3D23062020%26date%3D08062020%26volume%3D63%26issue%3D13%26spage%3D6834%26epage%3D6846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ning Wang, Qin Yang, Zhihong Deng, Xuechun Mao, <span class="NLM_string-name hlFld-ContribAuthor">Yiyuan Peng</span>. </span><span class="cited-content_cbyCitation_article-title">Rhodium-Catalyzed Merging of 2-Arylquinazolinone and 2,2-Difluorovinyl Tosylate: Diverse Synthesis of Monofluoroolefin Quinazolinone Derivatives. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (24)
                                     , 14635-14644. <a href="https://doi.org/10.1021/acsomega.0c01344" title="DOI URL">https://doi.org/10.1021/acsomega.0c01344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c01344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c01344%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DRhodium-Catalyzed%252BMerging%252Bof%252B2-Arylquinazolinone%252Band%252B2%25252C2-Difluorovinyl%252BTosylate%25253A%252BDiverse%252BSynthesis%252Bof%252BMonofluoroolefin%252BQuinazolinone%252BDerivatives%26aulast%3DWang%26aufirst%3DNing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D25032020%26date%3D20052020%26date%3D15062020%26volume%3D5%26issue%3D24%26spage%3D14635%26epage%3D14644" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael P.  Sanderson</span>, <span class="hlFld-ContribAuthor ">Manja  Friese-Hamim</span>, <span class="hlFld-ContribAuthor ">Gina  Walter-Bausch</span>, <span class="hlFld-ContribAuthor ">Michael  Busch</span>, <span class="hlFld-ContribAuthor ">Stefanie  Gaus</span>, <span class="hlFld-ContribAuthor ">Djordje  Musil</span>, <span class="hlFld-ContribAuthor ">Felix  Rohdich</span>, <span class="hlFld-ContribAuthor ">Ugo  Zanelli</span>, <span class="hlFld-ContribAuthor ">Sondra L.  Downey-Kopyscinski</span>, <span class="hlFld-ContribAuthor ">Constantine S.  Mitsiades</span>, <span class="hlFld-ContribAuthor ">Oliver  Schadt</span>, <span class="hlFld-ContribAuthor ">Markus  Klein</span>, <span class="hlFld-ContribAuthor ">Christina  Esdar</span>. </span><span class="cited-content_cbyCitation_article-title">M3258 Is a Selective Inhibitor of the Immunoproteasome Subunit LMP7 (β5i) Delivering Efficacy in Multiple Myeloma Models. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer Therapeutics</span><span> <strong>2021,</strong> <em>20 </em>
                                    (8)
                                     , 1378-1387. <a href="https://doi.org/10.1158/1535-7163.MCT-21-0005" title="DOI URL">https://doi.org/10.1158/1535-7163.MCT-21-0005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/1535-7163.MCT-21-0005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F1535-7163.MCT-21-0005%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%2520Therapeutics%26atitle%3DM3258%252BIs%252Ba%252BSelective%252BInhibitor%252Bof%252Bthe%252BImmunoproteasome%252BSubunit%252BLMP7%252B%252528%2525CE%2525B25i%252529%252BDelivering%252BEfficacy%252Bin%252BMultiple%252BMyeloma%252BModels%26aulast%3DSanderson%26aufirst%3DMichael%2BP.%26date%3D2021%26date%3D2021%26volume%3D20%26issue%3D8%26spage%3D1378%26epage%3D1387" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hidemasa  Hikawa</span>, <span class="hlFld-ContribAuthor ">Taku  Nakayama</span>, <span class="hlFld-ContribAuthor ">Makiko  Takahashi</span>, <span class="hlFld-ContribAuthor ">Shoko  Kikkawa</span>, <span class="hlFld-ContribAuthor ">Isao  Azumaya</span>. </span><span class="cited-content_cbyCitation_article-title">Direct Use of Benzylic Alcohols for Multicomponent Synthesis of 2‐Aryl Quinazolinones Utilizing the π‐Benzylpalladium(II) System in Water. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2021,</strong> <em>23 </em><a href="https://doi.org/10.1002/adsc.202100535" title="DOI URL">https://doi.org/10.1002/adsc.202100535</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.202100535&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.202100535%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DDirect%252BUse%252Bof%252BBenzylic%252BAlcohols%252Bfor%252BMulticomponent%252BSynthesis%252Bof%252B2%2525E2%252580%252590Aryl%252BQuinazolinones%252BUtilizing%252Bthe%252B%2525CF%252580%2525E2%252580%252590Benzylpalladium%252528II%252529%252BSystem%252Bin%252BWater%26aulast%3DHikawa%26aufirst%3DHidemasa%26date%3D2021%26date%3D2021%26volume%3D23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chirag C.  Mehta</span>, <span class="hlFld-ContribAuthor ">Hardik G.  Bhatt</span>. </span><span class="cited-content_cbyCitation_article-title">Tankyrase inhibitors as antitumor agents: a patent update (2013 – 2020). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (7)
                                     , 645-661. <a href="https://doi.org/10.1080/13543776.2021.1888929" title="DOI URL">https://doi.org/10.1080/13543776.2021.1888929</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1888929&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1888929%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DTankyrase%252Binhibitors%252Bas%252Bantitumor%252Bagents%25253A%252Ba%252Bpatent%252Bupdate%252B%2525282013%252B%2525E2%252580%252593%252B2020%252529%26aulast%3DMehta%26aufirst%3DChirag%2BC.%26date%3D2021%26date%3D2021%26volume%3D31%26issue%3D7%26spage%3D645%26epage%3D661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhen  Wang</span>, <span class="hlFld-ContribAuthor ">Zilu  Li</span>, <span class="hlFld-ContribAuthor ">Haitao  Ji</span>. </span><span class="cited-content_cbyCitation_article-title">Direct targeting of
              β
              ‐catenin in the Wnt signaling pathway: Current progress and perspectives. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2021,</strong> <em>41 </em>
                                    (4)
                                     , 2109-2129. <a href="https://doi.org/10.1002/med.21787" title="DOI URL">https://doi.org/10.1002/med.21787</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21787%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DDirect%252Btargeting%252Bof%252B%2525CE%2525B2%252B%2525E2%252580%252590catenin%252Bin%252Bthe%252BWnt%252Bsignaling%252Bpathway%25253A%252BCurrent%252Bprogress%252Band%252Bperspectives%26aulast%3DWang%26aufirst%3DZhen%26date%3D2021%26date%3D2021%26volume%3D41%26issue%3D4%26spage%3D2109%26epage%3D2129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiewen  Chen</span>, <span class="hlFld-ContribAuthor ">En  Liang</span>, <span class="hlFld-ContribAuthor ">Jie  Shi</span>, <span class="hlFld-ContribAuthor ">Yinrong  Wu</span>, <span class="hlFld-ContribAuthor ">Kangmei  Wen</span>, <span class="hlFld-ContribAuthor ">Xingang  Yao</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Tang</span>. </span><span class="cited-content_cbyCitation_article-title">Metal-free synthesis of 1,4-benzodiazepines and quinazolinones from hexafluoroisopropyl 2-aminobenzoates at room temperature. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2021,</strong> <em>11 </em>
                                    (9)
                                     , 4966-4970. <a href="https://doi.org/10.1039/D1RA00324K" title="DOI URL">https://doi.org/10.1039/D1RA00324K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1RA00324K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1RA00324K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DMetal-free%252Bsynthesis%252Bof%252B1%25252C4-benzodiazepines%252Band%252Bquinazolinones%252Bfrom%252Bhexafluoroisopropyl%252B2-aminobenzoates%252Bat%252Broom%252Btemperature%26aulast%3DChen%26aufirst%3DJiewen%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D9%26spage%3D4966%26epage%3D4970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anu  Verma</span>, <span class="hlFld-ContribAuthor ">Ashwani  Kumar</span>, <span class="hlFld-ContribAuthor ">Arshiya  Chugh</span>, <span class="hlFld-ContribAuthor ">Sunil  Kumar</span>, <span class="hlFld-ContribAuthor ">Parvin  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Tankyrase inhibitors: emerging and promising therapeutics for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2021,</strong> <em>30 </em>
                                    (1)
                                     , 50-73. <a href="https://doi.org/10.1007/s00044-020-02657-7" title="DOI URL">https://doi.org/10.1007/s00044-020-02657-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-020-02657-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-020-02657-7%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DTankyrase%252Binhibitors%25253A%252Bemerging%252Band%252Bpromising%252Btherapeutics%252Bfor%252Bcancer%252Btreatment%26aulast%3DVerma%26aufirst%3DAnu%26date%3D2021%26date%3D2020%26volume%3D30%26issue%3D1%26spage%3D50%26epage%3D73" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiufang  Xu</span>, <span class="hlFld-ContribAuthor ">Miaofeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Faying  Xu</span>, <span class="hlFld-ContribAuthor ">Shaojie  Jiang</span>. </span><span class="cited-content_cbyCitation_article-title">Wnt signaling in breast cancer: biological mechanisms, challenges and opportunities. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer</span><span> <strong>2020,</strong> <em>19 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12943-020-01276-5" title="DOI URL">https://doi.org/10.1186/s12943-020-01276-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12943-020-01276-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12943-020-01276-5%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%26atitle%3DWnt%252Bsignaling%252Bin%252Bbreast%252Bcancer%25253A%252Bbiological%252Bmechanisms%25252C%252Bchallenges%252Band%252Bopportunities%26aulast%3DXu%26aufirst%3DXiufang%26date%3D2020%26date%3D2020%26volume%3D19%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0005.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-<i>t</i>-BuPhCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 3 h; (b) 4-COOMePhCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 2.5–24 h; (c) MeMgCl, CeCl<sub>3</sub>, THF; (d) HO(CH<sub>2</sub>)<sub>2</sub>OH, TsOH·H<sub>2</sub>O or HO(CH<sub>2</sub>)<sub>2</sub>OCH<sub>3</sub>, TsOH·H<sub>2</sub>O; (e) 4-COOMePyCHO, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 24 h.</p></p></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. WaterMap analysis of TNKS2 in complex with <b>3a</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AKU">5AKU</a>). Color scheme according to predicted Δ<i>G</i> gain when water site is displaced: red (unstable waters) with predicted Δ<i>G</i> gain >2 kcal/mol, stable waters in green with Δ<i>G</i> penalty <0 kcal/mol, yellow in between. Tyr1150, Gly1053, Ile1157, and Tyr1060 are omitted for clarity. Displayed are all water sites that have an overlap factor greater than 0.5 with <b>3a</b>. There are three water sites in close proximity to the <i>tert</i>-butyl group of <b>3a</b> (distance of 0.4, 0.6, and 1.2 Å of the respective methyl groups) that are replaced by the inhibitor. Two additional water sites from the second shell are shown interacting with one water site (shown by dashed lines). Design idea has been that this interaction can be mimicked by introduction of a hydroxy substituent instead of a methyl group.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) 4-(4-methyl-piperazin-1-yl)-benzaldehyde, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 150 °C, 2.5–24 h; (b) (i) 4-(4-formyl-phenyl)-piperidine-1-carboxylic acid <i>tert</i>-butyl ester, Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, DMA, 160 °C, 2 h; (ii) HCl (4 M in dioxane), CH<sub>2</sub>Cl<sub>2</sub>, rt, overnight; (c) CH<sub>2</sub>O, HCOOH, 80 °C, overnight.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0002.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode of <b>5k</b> in complex with TNKS1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a>). For clarity, amino acids 1201–1215 are not shown. The quinazolinone core makes three hydrogen bonds to Gly1185 and Ser1221, the terminal hydroxyl group exhibits two hydrogen bonds with water molecules (distances in Å).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0003.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cellular mechanistic study of <b>5k</b> in DLD1 model. Cells were treated for 24 h with increasing concentrations of <b>5k</b> and protein abundance for tankyrase (TNKS1/TNKS2) and axin2 was determined in cell lysates using an antibody-based ELISA-like read-out.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/medium/jm9b00656_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0004.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. <i>In vivo</i> PK/PD study with <b>5k</b> in DLD1 xenograft model. Single treatment of DLD1 tumor bearing mice with 10, 30, and 90 mg/kg of <b>5k</b> for 2, 4, 8, and 12 h, respectively, using 5 mice per group. (a) Plasma concentration time profiles of <b>5k</b> are expressed as free concentrations calculated based on free fraction in mouse serum (<i>f</i><sub>ub</sub> 8.6%). <i>Ex vivo</i> lysates were analyzed for protein level of (b) tankyrase (TNKS1/TNKS2), (c) axin2, and (d) active nuclear β-catenin and were compared to untreated control tumor lysates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00656/20190905/images/large/jm9b00656_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00656&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i71">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74244" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74244" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 35 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riffell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span> <span> </span><span class="NLM_article-title">Tankyrase-targeted therapeutics: expanding opportunities in the PARP family</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">923</span>– <span class="NLM_lpage">936</span>, <span class="refDoi"> DOI: 10.1038/nrd3868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnrd3868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=23197039" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCntbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=923-936&author=J.+L.+Riffellauthor=C.+J.+Lordauthor=A.+Ashworth&title=Tankyrase-targeted+therapeutics%3A+expanding+opportunities+in+the+PARP+family&doi=10.1038%2Fnrd3868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrase-targeted therapeutics: expanding opportunities in the PARP family</span></div><div class="casAuthors">Riffell, Jenna L.; Lord, Christopher J.; Ashworth, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">923-936</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The poly(ADP-ribose) polymerase (PARP) protein superfamily has wide-ranging roles in cellular processes such as DNA repair and WNT signalling.  Efforts to pharmacol. target PARP enzymes have largely focused on PARP1 and the closely related PARP2, but recent work highlighting the role of another family member, tankyrase 1 (TANK1; also known as PARP5A and ARTD5), in the control of WNT signalling has fuelled interest in the development of addnl. inhibitors to target this enzyme class.  Tankyrase function is also implicated in other processes such as the regulation of telomere length, lung fibrogenesis and myelination, suggesting that tankyrase inhibitors could have broad clin. utility.  Here, we discuss the biol. of tankyrases and the discovery of tankyrase-specific inhibitors.  We also consider the challenges that lie ahead for the clin. development of PARP family inhibitors in general.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6R0zBQ42UELVg90H21EOLACvtfcHk0ljoKASGLWFYrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCntbvI&md5=510c523982433e4c22f06f8d1679e6f6</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrd3868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3868%26sid%3Dliteratum%253Aachs%26aulast%3DRiffell%26aufirst%3DJ.%2BL.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26aulast%3DAshworth%26aufirst%3DA.%26atitle%3DTankyrase-targeted%2520therapeutics%253A%2520expanding%2520opportunities%2520in%2520the%2520PARP%2520family%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D923%26epage%3D936%26doi%3D10.1038%2Fnrd3868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauss, S.</span></span> <span> </span><span class="NLM_article-title">Tankyrases as drug targets</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>280</i></span>,  <span class="NLM_fpage">3576</span>– <span class="NLM_lpage">3593</span>, <span class="refDoi"> DOI: 10.1111/febs.12320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1111%2Ffebs.12320" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=23648170" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A280%3ADC%252BC3snhtFOqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2013&pages=3576-3593&author=L.+Lehti%C3%B6author=N.+Chiauthor=S.+Krauss&title=Tankyrases+as+drug+targets&doi=10.1111%2Ffebs.12320"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrases as drug targets</span></div><div class="casAuthors">Lehtio Lari; Chi Nai-Wen; Krauss Stefan</div><div class="citationInfo"><span class="NLM_cas:title">The FEBS journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">280</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3576-93</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Tankyrase 1 and tankyrase 2 are poly(ADP-ribosyl)ases that are distinguishable from other members of the enzyme family by the structural features of the catalytic domain, and the presence of a sterile α-motif multimerization domain and an ankyrin repeat protein-interaction domain.  Tankyrases are implicated in a multitude of cellular functions, including telomere homeostasis, mitotic spindle formation, vesicle transport linked to glucose metabolism, Wnt-β-catenin signaling, and viral replication.  In these processes, tankyrases interact with target proteins, catalyze poly(ADP-ribosyl)ation, and regulate protein interactions and stability.  The proposed roles of tankyrases in disease-relevant cellular processes have made them attractive drug targets.  Recently, several inhibitors have been identified.  The selectivity and potency of these small molecules can be rationalized by how they fit within the NAD(+)-binding groove of the catalytic domain.  Some molecules bind to the nicotinamide subsite, such as generic diphtheria toxin-like ADP-ribosyltransferase inhibitors, whereas others bind to a distinct adenosine subsite that diverges from other diphtheria toxin-like ADP-ribosyltransferases and confers specificity.  A highly potent dual-site inhibitor is also available.  Within the last few years, tankyrase inhibitors have proved to be useful chemical probes and potential lead compounds, especially for specific cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRxqKZ7T5loVg20zIbnCCI3fW6udTcc2ea8oWqoSJZ9c7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snhtFOqug%253D%253D&md5=9643c0f2d7ee877400bffcdc8ad44c85</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1111%2Ffebs.12320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.12320%26sid%3Dliteratum%253Aachs%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26aulast%3DChi%26aufirst%3DN.%26aulast%3DKrauss%26aufirst%3DS.%26atitle%3DTankyrases%2520as%2520drug%2520targets%26jtitle%3DFEBS%2520J.%26date%3D2013%26volume%3D280%26spage%3D3576%26epage%3D3593%26doi%3D10.1111%2Ffebs.12320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haikarainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krauss, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span> <span> </span><span class="NLM_article-title">Tankyrases: Structure, function and therapeutic implications in cancer</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">6472</span>– <span class="NLM_lpage">6488</span>, <span class="refDoi"> DOI: 10.2174/1381612820666140630101525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.2174%2F1381612820666140630101525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=24975604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgtLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=6472-6488&author=T.+Haikarainenauthor=S.+Kraussauthor=L.+Lehti%C3%B6&title=Tankyrases%3A+Structure%2C+function+and+therapeutic+implications+in+cancer&doi=10.2174%2F1381612820666140630101525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrases: Structure, Function and Therapeutic Implications in Cancer</span></div><div class="casAuthors">Haikarainen, Teemu; Krauss, Stefan; Lehtio, Lari</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">6472-6488</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Several cellular signaling pathways are regulated by ADP-ribosylation, a posttranslational modification catalyzed by members of the ARTD superfamily.  Tankyrases are distinguishable from the rest of this family by their unique domain organization, notably the sterile alpha motif responsible for oligomerization and ankyrin repeats mediating protein-protein interactions.  Tankyrases are involved in various cellular functions, such as telomere homeostasis, Wnt/β -catenin signaling, glucose metab., and cell cycle progression.  In these processes, Tankyrases regulate the interactions and stability of target proteins by poly (ADP-ribosyl)ation.  Modified proteins are subsequently recognized by the E3 ubiquitin ligase RNF146, poly-ubiquitinated and predominantly guided to 26S proteasomal degrdn.  Several small mol. inhibitors have been described for Tankyrases; they compete with the co-substrate NAD+ for binding to the ARTD catalytic domain.  The recent, highly potent and selective inhibitors possess several properties of lead compds. and can be used for proof-of-concept studies in cancer and other Tankyrase linked diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotYUovVf-x6LVg90H21EOLACvtfcHk0ljGWVuxnxLrtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgtLfN&md5=ced27d2889d245696043a8cfa891166c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F1381612820666140630101525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612820666140630101525%26sid%3Dliteratum%253Aachs%26aulast%3DHaikarainen%26aufirst%3DT.%26aulast%3DKrauss%26aufirst%3DS.%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26atitle%3DTankyrases%253A%2520Structure%252C%2520function%2520and%2520therapeutic%2520implications%2520in%2520cancer%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2014%26volume%3D20%26spage%3D6472%26epage%3D6488%26doi%3D10.2174%2F1381612820666140630101525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kamal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riyaz, Sd.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahim, A.</span></span> <span> </span><span class="NLM_article-title">Tankyrase inhibitors as therapeutic targets for cancer</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1967</span>– <span class="NLM_lpage">1976</span>, <span class="refDoi"> DOI: 10.2174/1568026614666140929115831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.2174%2F1568026614666140929115831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=25262803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=1967-1976&author=A.+Kamalauthor=Sd.+Riyazauthor=A.+K.+Srivastavaauthor=A.+Rahim&title=Tankyrase+inhibitors+as+therapeutic+targets+for+cancer&doi=10.2174%2F1568026614666140929115831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrase Inhibitors as Therapeutic Targets for Cancer</span></div><div class="casAuthors">Kamal, Ahmed; Riyaz, S.; Srivastava, Ajay Kumar; Rahim, Abdul</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">1967-1976</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Tankyrase 1 and 2 belonging to the family of poly(ADP-ribosyl)ases play an important role in PARsylation by utilizing NAD+ as a substrate in order to generate ADP-ribose polymers.  Tankyrases are involved in a no. of cellular functions, that includes telomere homeostasis, mitotic spindle formation, vesicle transport linked to glucose metab., Wnt/β-catenin signalling, and viral replication.  These roles of tankyrases in disease-relevant cellular processes have made them attractive drug targets.  Recently, several inhibitors have been identified as potential clin. leads.  The current review covers the progress, mechanism and binding modes of recently known Tankyrase inhibitors and discusses the rational approaches that were used to identify the tankyrase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruFM_1i4ZP1LVg90H21EOLACvtfcHk0ljGWVuxnxLrtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGgt7vP&md5=41cdf01744936fa538ef8e7307180feb</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.2174%2F1568026614666140929115831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026614666140929115831%26sid%3Dliteratum%253Aachs%26aulast%3DKamal%26aufirst%3DA.%26aulast%3DRiyaz%26aufirst%3DSd.%26aulast%3DSrivastava%26aufirst%3DA.%2BK.%26aulast%3DRahim%26aufirst%3DA.%26atitle%3DTankyrase%2520inhibitors%2520as%2520therapeutic%2520targets%2520for%2520cancer%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2014%26volume%3D14%26spage%3D1967%26epage%3D1976%26doi%3D10.2174%2F1568026614666140929115831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amé, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spenlehauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">The PARP superfamily</span>. <i>BioEssays</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">882</span>– <span class="NLM_lpage">893</span>, <span class="refDoi"> DOI: 10.1002/bies.20085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1002%2Fbies.20085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=15273990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2004&pages=882-893&author=J.-C.+Am%C3%A9author=C.+Spenlehauerauthor=G.+de+Murcia&title=The+PARP+superfamily&doi=10.1002%2Fbies.20085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The PARP superfamily</span></div><div class="casAuthors">Ame, Jean-Christophe; Spenlehauer, Catherine; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">BioEssays</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">882-893</span>CODEN:
                <span class="NLM_cas:coden">BIOEEJ</span>;
        ISSN:<span class="NLM_cas:issn">0265-9247</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribosyl)ation is an immediate DNA-damage-dependent post-translational modification of histones and other nuclear proteins that contributes to the survival of injured proliferating cells.  Poly(ADP-ribose) polymerases (PARPs) now constitute a large family of 18 proteins, encoded by different genes and displaying a conserved catalytic domain in which PARP-1 (113 kDa), the founding member, and PARP-2 (62 kDa) are so far the sole enzymes whose catalytic activity has been shown to be immediately stimulated by DNA strand breaks.  A large repertoire of sequences encoding novel PARPs now extends considerably the field of poly(ADP-ribosyl)ation reactions to various aspects of the cell biol. including cell proliferation and cell death.  Some of these new members interact with each other, share common partners and common subcellular localizations suggesting possible fine tuning in the regulation of this post-translational modification of proteins.  This review summarizes the present knowledge of this emerging superfamily, which might ultimately improve pharmacol. strategies to enhance both antitumor efficacy and the treatment of a no. of inflammatory and neurodegenerative disorders.  A provisional nomenclature is proposed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojkP1P6SYh67Vg90H21EOLACvtfcHk0ljGWVuxnxLrtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnt1CqtLk%253D&md5=892151993f54e3c1a03019001a365fbf</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fbies.20085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbies.20085%26sid%3Dliteratum%253Aachs%26aulast%3DAm%25C3%25A9%26aufirst%3DJ.-C.%26aulast%3DSpenlehauer%26aufirst%3DC.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DThe%2520PARP%2520superfamily%26jtitle%3DBioEssays%26date%3D2004%26volume%3D26%26spage%3D882%26epage%3D893%26doi%3D10.1002%2Fbies.20085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dantzer, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ame, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Murcia, G.</span></span> <span> </span><span class="NLM_article-title">Poly(ADP-ribose): Novel functions for an old molecule</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">7</span>),  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">528</span>, <span class="refDoi"> DOI: 10.1038/nrm1963</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnrm1963" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=16829982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2006&pages=517-528&issue=7&author=V.+Schreiberauthor=F.+Dantzerauthor=J.+C.+Ameauthor=G.+de+Murcia&title=Poly%28ADP-ribose%29%3A+Novel+functions+for+an+old+molecule&doi=10.1038%2Fnrm1963"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Poly(ADP-ribose): novel functions for an old molecule</span></div><div class="casAuthors">Schreiber, Valerie; Dantzer, Francoise; Ame, Jean-Christophe; de Murcia, Gilbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">517-528</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The addn. to proteins of the neg. charged polymer of ADP-ribose, which is synthesized by poly(ADP-ribose) polymerase (PARPs) from NAD, is a unique post-translational modification.  It regulates not only cell survival and cell-death programs, but also an increasing no. of other biol. functions with which novel members of the PARP family have been assocd.  These functions include transcriptional regulation, telomere cohesion and mitotic spindle formation during cell division, intracellular trafficking, and energy metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLsI4_BKfjdrVg90H21EOLACvtfcHk0lg1JkGpn3DuzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1Ohsb4%253D&md5=676e83755491500f3bf2137f348dfa1b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrm1963&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1963%26sid%3Dliteratum%253Aachs%26aulast%3DSchreiber%26aufirst%3DV.%26aulast%3DDantzer%26aufirst%3DF.%26aulast%3DAme%26aufirst%3DJ.%2BC.%26aulast%3Dde%2BMurcia%26aufirst%3DG.%26atitle%3DPoly%2528ADP-ribose%2529%253A%2520Novel%2520functions%2520for%2520an%2520old%2520molecule%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2006%26volume%3D7%26issue%3D7%26spage%3D517%26epage%3D528%26doi%3D10.1038%2Fnrm1963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liscio, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carotti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asciutti, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sardella, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchiarulo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camaioni, E.</span></span> <span> </span><span class="NLM_article-title">Targeting Wnt-driven cancers: Discovery of novel tankyrase inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">506</span>– <span class="NLM_lpage">522</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.09.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.ejmech.2017.09.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=29107427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslentbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2017&pages=506-522&author=M.+Ferriauthor=P.+Liscioauthor=A.+Carottiauthor=S.+Asciuttiauthor=R.+Sardellaauthor=A.+Macchiaruloauthor=E.+Camaioni&title=Targeting+Wnt-driven+cancers%3A+Discovery+of+novel+tankyrase+inhibitors&doi=10.1016%2Fj.ejmech.2017.09.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Wnt-driven cancers: Discovery of novel tankyrase inhibitors</span></div><div class="casAuthors">Ferri, Martina; Liscio, Paride; Carotti, Andrea; Asciutti, Stefania; Sardella, Roccaldo; Macchiarulo, Antonio; Camaioni, Emidio</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">506-522</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Recent years have seen substantially heightened interest in the discovery of tankyrase inhibitors (TNKSi) as new promising anticancer agents.  In this framework, the aim of this review article is focused on the description of potent TNKSi also endowed with disruptor activity toward the Wnt/β-catenin signaling pathway.  Beginning with an overview of the most characterized TNKSi deriving from several drug design approaches and classifying them on the basis of the mol. interactions with the target, the authors discuss only those ones acting against Wnt cancer cell lines.  In addn., comprehensive structure property relationships (SPR) emerging from the hit evolution processes and preclin. results are provided.  The authors then review the most promising TNKSi hitherto reported in literature, acting in vivo models of Wnt-driven cancers.  Some out-looks on current issues and future directions in this field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAs1ycCxVwl7Vg90H21EOLACvtfcHk0lg1JkGpn3DuzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslentbbJ&md5=9b4d843e6bb7ccdea56f9e9c8b70d18c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.09.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.09.030%26sid%3Dliteratum%253Aachs%26aulast%3DFerri%26aufirst%3DM.%26aulast%3DLiscio%26aufirst%3DP.%26aulast%3DCarotti%26aufirst%3DA.%26aulast%3DAsciutti%26aufirst%3DS.%26aulast%3DSardella%26aufirst%3DR.%26aulast%3DMacchiarulo%26aufirst%3DA.%26aulast%3DCamaioni%26aufirst%3DE.%26atitle%3DTargeting%2520Wnt-driven%2520cancers%253A%2520Discovery%2520of%2520novel%2520tankyrase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D142%26spage%3D506%26epage%3D522%26doi%3D10.1016%2Fj.ejmech.2017.09.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, F.</span></span> <span> </span><span class="NLM_article-title">RNF146 is a poly(ADP-ribose)-directed E3 ligase that regulates axin degradation and Wnt signaling</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">629</span>, <span class="refDoi"> DOI: 10.1038/ncb2222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fncb2222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=21478859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsFGmurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=623-629&author=Y.+Zhangauthor=S.+Liuauthor=C.+Mickaninauthor=Y.+Fengauthor=O.+Charlatauthor=G.+A.+Michaudauthor=M.+Schirleauthor=X.+Shiauthor=M.+Hildauthor=A.+Bauerauthor=V.+E.+Myerauthor=P.+M.+Finanauthor=J.+A.+Porterauthor=S.+M.+Huangauthor=F.+Cong&title=RNF146+is+a+poly%28ADP-ribose%29-directed+E3+ligase+that+regulates+axin+degradation+and+Wnt+signaling&doi=10.1038%2Fncb2222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">RNF146 is a poly(ADP-ribose)-directed E3 ligase that regulates axin degradation and Wnt signalling</span></div><div class="casAuthors">Zhang, Yue; Liu, Shanming; Mickanin, Craig; Feng, Yan; Charlat, Olga; Michaud, Gregory A.; Schirle, Markus; Shi, Xiaoying; Hild, Marc; Bauer, Andreas; Myer, Vic E.; Finan, Peter M.; Porter, Jeffery A.; Huang, Shih-Min A.; Cong, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">623-629</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Wnt/β-catenin signalling pathway plays essential roles in embryonic development and adult tissue homeostasis, and deregulation of this pathway has been linked to cancer.  Axin is a concn.-limiting component of the β-catenin destruction complex, and its stability is regulated by tankyrase.  However, the mol. mechanism by which tankyrase-dependent poly(ADP-ribosyl)ation (PARsylation) is coupled to ubiquitylation and degrdn. of axin remains undefined.  Here, we identify RNF146, a RING-domain E3 ubiquitin ligase, as a pos. regulator of Wnt signalling.  RNF146 promotes Wnt signalling by mediating tankyrase-dependent degrdn. of axin.  Mechanistically, RNF146 directly interacts with poly(ADP-ribose) through its WWE domain, and promotes degrdn. of PARsylated proteins.  Using proteomics approaches, we have identified BLZF1 and CASC3 as further substrates targeted by tankyrase and RNF146 for degrdn.  Thus, identification of RNF146 as a PARsylation-directed E3 ligase establishes a mol. paradigm that links tankyrase-dependent PARsylation to ubiquitylation.  RNF146-dependent protein degrdn. may emerge as a major mechanism by which tankyrase exerts its function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgvUCNwxGZLrVg90H21EOLACvtfcHk0ljC5K-BWC0k0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsFGmurk%253D&md5=0029be867466c74516f1a7fbab62369d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fncb2222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb2222%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DCharlat%26aufirst%3DO.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DFinan%26aufirst%3DP.%2BM.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DHuang%26aufirst%3DS.%2BM.%26aulast%3DCong%26aufirst%3DF.%26atitle%3DRNF146%2520is%2520a%2520poly%2528ADP-ribose%2529-directed%2520E3%2520ligase%2520that%2520regulates%2520axin%2520degradation%2520and%2520Wnt%2520signaling%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2011%26volume%3D13%26spage%3D623%26epage%3D629%26doi%3D10.1038%2Fncb2222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Callow, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sandoval, W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bheddah, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lill, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hongo, J.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkpatrick, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polakis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, M.</span></span> <span> </span><span class="NLM_article-title">Ubiquitin ligase RNF146 regulates tankyrase and axin to promote Wnt signaling</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>, <span class="NLM_elocation-id">e22595</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0022595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1371%2Fjournal.pone.0022595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=21799911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKgt7vO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=M.+G.+Callowauthor=H.+Tranauthor=L.+Phuauthor=T.+Lauauthor=J.+Leeauthor=W.+N.+Sandovalauthor=P.+S.+Liuauthor=S.+Bheddahauthor=J.+Taoauthor=J.+R.+Lillauthor=J.-A.+Hongoauthor=D.+Davisauthor=D.+S.+Kirkpatrickauthor=P.+Polakisauthor=M.+Costa&title=Ubiquitin+ligase+RNF146+regulates+tankyrase+and+axin+to+promote+Wnt+signaling&doi=10.1371%2Fjournal.pone.0022595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ubiquitin ligase RNF146 regulates tankyrase and Axin to promote Wnt signaling</span></div><div class="casAuthors">Callow, Marinella G.; Tran, Hoanh; Phu, Lilian; Lau, Ted; Lee, James; Sandoval, Wendy N.; Liu, Peter S.; Bheddah, Sheila; Tao, Janet; Lill, Jennie R.; Hongo, Jo-Anne; Davis, David; Kirkpatrick, Donald S.; Polakis, Paul; Costa, Mike</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e22595</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Canonical Wnt signaling is controlled intracellularly by the level of β-catenin protein, which is dependent on Axin scaffolding of a complex that phosphorylates β-catenin to target it for ubiquitylation and proteasomal degrdn.  This function of Axin is counteracted through relocalization of Axin protein to the Wnt receptor complex to allow for ligand-activated Wnt signaling.  AXIN1 and AXIN2 protein levels are regulated by tankyrase-mediated poly(ADP-ribosyl)ation (PARsylation), which destabilizes Axin and promotes signaling.  Mechanistically, how tankyrase limits Axin protein accumulation, and how tankyrase levels and activity are regulated for this function, are currently under investigation.  By RNAi screening, we identified the RNF146 RING-type ubiquitin E3 ligase as a pos. regulator of Wnt signaling that operates with tankyrase to maintain low steady-state levels of Axin proteins.  RNF146 also destabilizes tankyrases TNKS1 and TNKS2 proteins and, in a reciprocal relationship, tankyrase activity reduces RNF146 protein levels.  We show that RNF146, tankyrase, and Axin form a protein complex, and that RNF146 mediates ubiquitylation of all 3 proteins to target them for proteasomal degrdn.  RNF146 is a cytoplasmic protein that also prevents tankyrase protein aggregation at a centrosomal location.  Tankyrase auto-PARsylation and PARsylation of Axin is known to lead to proteasome-mediated degrdn. of these proteins, and we demonstrate that, through ubiquitylation, RNF146 mediates this process to regulate Wnt signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBK_DIvtiDN7Vg90H21EOLACvtfcHk0ljC5K-BWC0k0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKgt7vO&md5=df9dc5d0f0fc70e5d9c81783a34ee10c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0022595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0022595%26sid%3Dliteratum%253Aachs%26aulast%3DCallow%26aufirst%3DM.%2BG.%26aulast%3DTran%26aufirst%3DH.%26aulast%3DPhu%26aufirst%3DL.%26aulast%3DLau%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSandoval%26aufirst%3DW.%2BN.%26aulast%3DLiu%26aufirst%3DP.%2BS.%26aulast%3DBheddah%26aufirst%3DS.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DLill%26aufirst%3DJ.%2BR.%26aulast%3DHongo%26aufirst%3DJ.-A.%26aulast%3DDavis%26aufirst%3DD.%26aulast%3DKirkpatrick%26aufirst%3DD.%2BS.%26aulast%3DPolakis%26aufirst%3DP.%26aulast%3DCosta%26aufirst%3DM.%26atitle%3DUbiquitin%2520ligase%2520RNF146%2520regulates%2520tankyrase%2520and%2520axin%2520to%2520promote%2520Wnt%2520signaling%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26doi%3D10.1371%2Fjournal.pone.0022595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, S.-M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishina, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmeier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charlat, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiellette, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiessner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mickanin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghidelli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirschner, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lengauer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finan, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouwmeester, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, F.</span></span> <span> </span><span class="NLM_article-title">Tankyrase inhibition stabilizes axin and antagonizes Wnt signaling</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>461</i></span>,  <span class="NLM_fpage">614</span>– <span class="NLM_lpage">620</span>, <span class="refDoi"> DOI: 10.1038/nature08356</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnature08356" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=19759537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFentbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=614-620&author=S.-M.+A.+Huangauthor=Y.+M.+Mishinaauthor=S.+Liuauthor=A.+Cheungauthor=F.+Stegmeierauthor=G.+A.+Michaudauthor=O.+Charlatauthor=E.+Wielletteauthor=Y.+Zhangauthor=S.+Wiessnerauthor=M.+Hildauthor=X.+Shiauthor=C.+J.+Wilsonauthor=C.+Mickaninauthor=V.+Myerauthor=A.+Fazalauthor=R.+Tomlinsonauthor=F.+Serlucaauthor=W.+Shaoauthor=H.+Chengauthor=M.+Shultzauthor=C.+Rauauthor=M.+Schirleauthor=J.+Schleglauthor=S.+Ghidelliauthor=S.+Fawellauthor=C.+Luauthor=D.+Curtisauthor=M.+W.+Kirschnerauthor=C.+Lengauerauthor=P.+M.+Finanauthor=J.+A.+Tallaricoauthor=T.+Bouwmeesterauthor=J.+A.+Porterauthor=A.+Bauerauthor=F.+Cong&title=Tankyrase+inhibition+stabilizes+axin+and+antagonizes+Wnt+signaling&doi=10.1038%2Fnature08356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Tankyrase inhibition stabilizes axin and antagonizes Wnt signalling</span></div><div class="casAuthors">Huang, Shih-Min A.; Mishina, Yuji M.; Liu, Shanming; Cheung, Atwood; Stegmeier, Frank; Michaud, Gregory A.; Charlat, Olga; Wiellette, Elizabeth; Zhang, Yue; Wiessner, Stephanie; Hild, Marc; Shi, Xiaoying; Wilson, Christopher J.; Mickanin, Craig; Myer, Vic; Fazal, Aleem; Tomlinson, Ronald; Serluca, Fabrizio; Shao, Wenlin; Cheng, Hong; Shultz, Michael; Rau, Christina; Schirle, Markus; Schlegl, Judith; Ghidelli, Sonja; Fawell, Stephen; Lu, Chris; Curtis, Daniel; Kirschner, Marc W.; Lengauer, Christoph; Finan, Peter M.; Tallarico, John A.; Bouwmeester, Tewis; Porter, Jeffery A.; Bauer, Andreas; Cong, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7264</span>),
    <span class="NLM_cas:pages">614-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The stability of the Wnt pathway transcription factor β-catenin is tightly regulated by the multi-subunit destruction complex.  Deregulated Wnt pathway activity has been implicated in many cancers, making this pathway an attractive target for anticancer therapies.  However, the development of targeted Wnt pathway inhibitors has been hampered by the limited no. of pathway components that are amenable to small mol. inhibition.  Here, we used a chem. genetic screen to identify a small mol., XAV939, which selectively inhibits β-catenin-mediated transcription.  XAV939 stimulates β-catenin degrdn. by stabilizing axin, the concn.-limiting component of the destruction complex.  Using a quant. chem. proteomic approach, we discovered that XAV939 stabilizes axin by inhibiting the poly-ADP-ribosylating enzymes tankyrase 1 and tankyrase 2.  Both tankyrase isoforms interact with a highly conserved domain of axin and stimulate its degrdn. through the ubiquitin-proteasome pathway.  Thus, our study provides new mechanistic insights into the regulation of axin protein homeostasis and presents new avenues for targeted Wnt pathway therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom3c1M4OoOarVg90H21EOLACvtfcHk0ljC5K-BWC0k0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFentbzP&md5=6a76dfa7ed2a57b29e0c1ee367bd652c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature08356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08356%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DS.-M.%2BA.%26aulast%3DMishina%26aufirst%3DY.%2BM.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DCheung%26aufirst%3DA.%26aulast%3DStegmeier%26aufirst%3DF.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DCharlat%26aufirst%3DO.%26aulast%3DWiellette%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DWiessner%26aufirst%3DS.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DMickanin%26aufirst%3DC.%26aulast%3DMyer%26aufirst%3DV.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DTomlinson%26aufirst%3DR.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DShultz%26aufirst%3DM.%26aulast%3DRau%26aufirst%3DC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DGhidelli%26aufirst%3DS.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DKirschner%26aufirst%3DM.%2BW.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DFinan%26aufirst%3DP.%2BM.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DBouwmeester%26aufirst%3DT.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DBauer%26aufirst%3DA.%26aulast%3DCong%26aufirst%3DF.%26atitle%3DTankyrase%2520inhibition%2520stabilizes%2520axin%2520and%2520antagonizes%2520Wnt%2520signaling%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D614%26epage%3D620%26doi%3D10.1038%2Fnature08356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mariotti, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guettler, S.</span></span> <span> </span><span class="NLM_article-title">Regulation of Wnt/β-catenin signalling by tankyrase-dependent poly(ADP-ribosyl)ation and scaffolding</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>174</i></span>,  <span class="NLM_fpage">4611</span>– <span class="NLM_lpage">4636</span>, <span class="refDoi"> DOI: 10.1111/bph.14038</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1111%2Fbph.14038" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=28910490" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslGqs7bE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=174&publication_year=2017&pages=4611-4636&author=L.+Mariottiauthor=K.+Pollockauthor=S.+Guettler&title=Regulation+of+Wnt%2F%CE%B2-catenin+signalling+by+tankyrase-dependent+poly%28ADP-ribosyl%29ation+and+scaffolding&doi=10.1111%2Fbph.14038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of Wnt/β-catenin signalling by tankyrase-dependent poly(ADP-ribosyl)ation and scaffolding</span></div><div class="casAuthors">Mariotti, Laura; Pollock, Katie; Guettler, Sebastian</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">174</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">4611-4636</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The Wnt/β-catenin signalling pathway is pivotal for stem cell function and the control of cellular differentiation, both during embryonic development and tissue homeostasis in adults.  Its activity is carefully controlled through the concerted interactions of concn.-limited pathway components and a wide range of post-translational modifications, including phosphorylation, ubiquitylation, sumoylation, poly(ADP-ribosyl)ation (PARylation) and acetylation.  Regulation of Wnt/β-catenin signalling by PARylation was discovered relatively recently.  The PARP tankyrase PARylates AXIN1/2, an essential central scaffolding protein in the β-catenin destruction complex, and targets it for degrdn., thereby fine-tuning the responsiveness of cells to the Wnt signal.  The past few years have not only seen much progress in our understanding of the mol. mechanisms by which PARylation controls the pathway but also witnessed the successful development of tankyrase inhibitors as tool compds. and promising agents for the therapy of Wnt-dependent dysfunctions, including colorectal cancer.  Recent work has hinted at more complex roles of tankyrase in Wnt/β-catenin signalling as well as challenges and opportunities in the development of tankyrase inhibitors.  Here we review some of the latest advances in our understanding of tankyrase function in the pathway and efforts to modulate tankyrase activity to re-tune Wnt/β-catenin signalling in colorectal cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbg-X7McNWebVg90H21EOLACvtfcHk0lj3PNAVr0V79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslGqs7bE&md5=4ba71faa4fc2064e57390dae08b88c0f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1111%2Fbph.14038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.14038%26sid%3Dliteratum%253Aachs%26aulast%3DMariotti%26aufirst%3DL.%26aulast%3DPollock%26aufirst%3DK.%26aulast%3DGuettler%26aufirst%3DS.%26atitle%3DRegulation%2520of%2520Wnt%252F%25CE%25B2-catenin%2520signalling%2520by%2520tankyrase-dependent%2520poly%2528ADP-ribosyl%2529ation%2520and%2520scaffolding%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2017%26volume%3D174%26spage%3D4611%26epage%3D4636%26doi%3D10.1111%2Fbph.14038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">The
Cancer Genome Atlas Network.</span> <span> </span><span class="NLM_article-title">Comprehensive molecular characterization of human colon and rectal
cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>487</i></span>,  <span class="NLM_fpage">330</span>– <span class="NLM_lpage">337</span>, <span class="refDoi"> DOI: 10.1038/nature11252</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnature11252" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22810696" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=487&publication_year=2012&pages=330-337&author=The%0ACancer+Genome+Atlas+Network.&title=Comprehensive+molecular+characterization+of+human+colon+and+rectal%0Acancer&doi=10.1038%2Fnature11252"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature11252&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11252%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DComprehensive%2520molecular%2520characterization%2520of%2520human%2520colon%2520and%2520rectal%250Acancer%26jtitle%3DNature%26date%3D2012%26volume%3D487%26spage%3D330%26epage%3D337%26doi%3D10.1038%2Fnature11252" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vyas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, P.</span></span> <span> </span><span class="NLM_article-title">New PARP targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1038/nrc3748</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnrc3748" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=24898058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlKqsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2014&pages=502-509&author=S.+Vyasauthor=P.+Chang&title=New+PARP+targets+for+cancer+therapy&doi=10.1038%2Fnrc3748"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">New PARP targets for cancer therapy</span></div><div class="casAuthors">Vyas, Sejal; Chang, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">502-509</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Poly(ADP-ribose) polymerases (PARPs) modify target proteins post-translationally with poly(ADP-ribose) (PAR) or mono(ADP-ribose) (MAR) using NAD+ as substrate.  The best-studied PARPs generate PAR modifications and include PARP1 and the tankyrase PARP5A, both of which are targets for cancer therapy with inhibitors in either clin. trials or preclin. development.  There are 15 addnl. PARPs, most of which modify proteins with MAR, and their biol. is less well understood.  Recent data identify potentially cancer-relevant functions for these PARPs, which indicates that we need to understand more about these PARPs to effectively target them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouf6CtI5GhsrVg90H21EOLACvtfcHk0lj3PNAVr0V79Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlKqsb0%253D&md5=24a3b64d2d0518e13c9540ee012e29ca</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrc3748&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3748%26sid%3Dliteratum%253Aachs%26aulast%3DVyas%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DP.%26atitle%3DNew%2520PARP%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2014%26volume%3D14%26spage%3D502%26epage%3D509%26doi%3D10.1038%2Fnrc3748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nathubhai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Threadgill, M. D.</span></span> <span> </span><span class="NLM_article-title">Design and discovery of 2-arylquinazolin-4-ones as potent and selective inhibitors of tankyrases</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1173</span>– <span class="NLM_lpage">1177</span>, <span class="refDoi"> DOI: 10.1021/ml400260b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml400260b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aht7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1173-1177&author=A.+Nathubhaiauthor=P.+J.+Woodauthor=M.+D.+Lloydauthor=A.+S.+Thompsonauthor=M.+D.+Threadgill&title=Design+and+discovery+of+2-arylquinazolin-4-ones+as+potent+and+selective+inhibitors+of+tankyrases&doi=10.1021%2Fml400260b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Discovery of 2 Arylquinazolin-4-ones as Potent and Selective Inhibitors of Tankyrases</span></div><div class="casAuthors">Nathubhai, Amit; Wood, Pauline J.; Lloyd, Matthew D.; Thompson, Andrew S.; Threadgill, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1173-1177</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrases (TNKSs) are poly-(ADP-ribose)-polymerases (PARPs) that are overexpressed in several clin. cancers.  They regulate elongation of telomeres, regulate the Wnt system, and are essential for the function of the mitotic spindle.  A set of 2-arylquinazolin-4-ones has been designed and identified as potent and selective TNKS inhibitors, some being more potent and selective than the lead inhibitor XAV939, with IC50 = 3 nM vs.  TNKS-2.  Me was preferred at the 8-position and modest bulk at the 4-position of the 2-Ph group; electronic effects and H-bonding were irrelevant, but charge in the 4'-substituent must be avoided.  Mol. modeling facilitated initial design of the compds. and rationalization of the SAR of binding into the nicotinamide-binding site of the target enzymes.  These compds. have potential for further development into anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxfmX1o7LIsrVg90H21EOLACvtfcHk0ljB6YFKCkx1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aht7nM&md5=23eb9a910b7952c76392bb1ae3b1185b</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fml400260b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml400260b%26sid%3Dliteratum%253Aachs%26aulast%3DNathubhai%26aufirst%3DA.%26aulast%3DWood%26aufirst%3DP.%2BJ.%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26aulast%3DThompson%26aufirst%3DA.%2BS.%26aulast%3DThreadgill%26aufirst%3DM.%2BD.%26atitle%3DDesign%2520and%2520discovery%2520of%25202-arylquinazolin-4-ones%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520tankyrases%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D1173%26epage%3D1177%26doi%3D10.1021%2Fml400260b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nathubhai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haikarainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayward, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Descalzo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Threadgill, M. D.</span></span> <span> </span><span class="NLM_article-title">Structure-activity relationships of 2-arylquinazolin-4-ones as highly selective and potent inhibitors of the Tankyrases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.04.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.ejmech.2016.04.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=27163581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC28Xnslarsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2016&pages=316-327&author=A.+Nathubhaiauthor=T.+Haikarainenauthor=P.+C.+Haywardauthor=S.+Mu%C3%B1oz-Descalzoauthor=A.+S.+Thompsonauthor=M.+D.+Lloydauthor=L.+Lehti%C3%B6author=M.+D.+Threadgill&title=Structure-activity+relationships+of+2-arylquinazolin-4-ones+as+highly+selective+and+potent+inhibitors+of+the+Tankyrases&doi=10.1016%2Fj.ejmech.2016.04.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationships of 2-arylquinazolin-4-ones as highly selective and potent inhibitors of the tankyrases</span></div><div class="casAuthors">Nathubhai, Amit; Haikarainen, Teemu; Hayward, Penelope C.; Munoz-Descalzo, Silvia; Thompson, Andrew S.; Lloyd, Matthew D.; Lehtio, Lari; Threadgill, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-327</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Tankyrases (TNKSs), members of the PARP (Poly(ADP-ribose)polymerases) superfamily of enzymes, have gained interest as therapeutic drug targets, esp. as they are involved in the regulation of Wnt signalling.  A series of 2-arylquinazolin-4-ones with varying substituents at the 8-position was synthesized.  An 8-Me group (compared to 8-H, 8-OMe, 8-OH), together with a 4'-hydrophobic or electron-withdrawing group, provided the most potency and selectivity towards TNKSs.  Co-crystal structures of selected compds. with TNKS-2 revealed that the protein around the 8-position is more hydrophobic in TNKS-2 compared to PARP-1/2, rationalising the selectivity.  The NAD+-binding site contains a hydrophobic cavity which accommodates the 2-aryl group; in TNKS-2, this has a tunnel to the exterior but the cavity is closed in PARP-1. 8-Methyl-2-(4-trifluoromethylphenyl)quinazolin-4-one(I) was identified as a potent and selective inhibitor of TNKSs and Wnt signalling.  This compd. and analogs could serve as mol. probes to study proliferative signalling and for development of inhibitors of TNKSs as drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYiaLuEeSrjbVg90H21EOLACvtfcHk0ljB6YFKCkx1Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xnslarsbs%253D&md5=c4bb8bad634199c8027e1ddf64c56f5e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.04.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.04.041%26sid%3Dliteratum%253Aachs%26aulast%3DNathubhai%26aufirst%3DA.%26aulast%3DHaikarainen%26aufirst%3DT.%26aulast%3DHayward%26aufirst%3DP.%2BC.%26aulast%3DMu%25C3%25B1oz-Descalzo%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DA.%2BS.%26aulast%3DLloyd%26aufirst%3DM.%2BD.%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26aulast%3DThreadgill%26aufirst%3DM.%2BD.%26atitle%3DStructure-activity%2520relationships%2520of%25202-arylquinazolin-4-ones%2520as%2520highly%2520selective%2520and%2520potent%2520inhibitors%2520of%2520the%2520Tankyrases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2016%26volume%3D118%26spage%3D316%26epage%3D327%26doi%3D10.1016%2Fj.ejmech.2016.04.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Haikarainen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koivunen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narwal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkannagari, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obaji, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pihlajaniemi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehtiö, L.</span></span> <span> </span><span class="NLM_article-title">para-Substituted 2-phenyl-3,4-dihydroquinazolin-4-ones as potent and selective Tankyrase inhibitors</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1978</span>– <span class="NLM_lpage">1985</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1002%2Fcmdc.201300337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=24130191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1WmtbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1978-1985&author=T.+Haikarainenauthor=J.+Koivunenauthor=M.+Narwalauthor=H.+Venkannagariauthor=E.+Obajiauthor=P.+Joensuuauthor=T.+Pihlajaniemiauthor=L.+Lehti%C3%B6&title=para-Substituted+2-phenyl-3%2C4-dihydroquinazolin-4-ones+as+potent+and+selective+Tankyrase+inhibitors&doi=10.1002%2Fcmdc.201300337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">para-Substituted 2-Phenyl-3,4-dihydroquinazolin-4-ones As Potent and Selective Tankyrase Inhibitors</span></div><div class="casAuthors">Haikarainen, Teemu; Koivunen, Jarkko; Narwal, Mohit; Venkannagari, Harikanth; Obaji, Ezeogo; Joensuu, Paeivi; Pihlajaniemi, Taina; Lehtioe, Lari</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1978-1985</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Human tankyrases are attractive drug targets, esp. for the treatment of cancer.  We identified a set of highly potent tankyrase inhibitors based on a 2-phenyl-3,4-dihydroquinazolin-4-one scaffold.  Substitutions at the para position of the scaffold's Ph group were evaluated as a strategy to increase potency and improve selectivity.  The best compds. displayed single-digit nanomolar potencies, and profiling against several human diphtheria-toxin-like ADP-ribosyltransferases revealed that a subset of these compds. are highly selective tankyrase inhibitors.  The compds. also effectively inhibit Wnt signaling in HEK293 cells.  The binding mode of all inhibitors was studied by protein X-ray crystallog.  This allowed us to establish a structural basis for the development of highly potent and selective tankyrase inhibitors based on the 2-phenyl-3,4-dihydroquinazolin-4-one scaffold and outline a rational approach to the modification of other inhibitor scaffolds that bind to the nicotinamide site of the catalytic domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFPGLvCfYMc7Vg90H21EOLACvtfcHk0lj30ZfgTVp6hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1WmtbzK&md5=917b45b12c3170462210e8d260a1dd4c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300337%26sid%3Dliteratum%253Aachs%26aulast%3DHaikarainen%26aufirst%3DT.%26aulast%3DKoivunen%26aufirst%3DJ.%26aulast%3DNarwal%26aufirst%3DM.%26aulast%3DVenkannagari%26aufirst%3DH.%26aulast%3DObaji%26aufirst%3DE.%26aulast%3DJoensuu%26aufirst%3DP.%26aulast%3DPihlajaniemi%26aufirst%3DT.%26aulast%3DLehti%25C3%25B6%26aufirst%3DL.%26atitle%3Dpara-Substituted%25202-phenyl-3%252C4-dihydroquinazolin-4-ones%2520as%2520potent%2520and%2520selective%2520Tankyrase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1978%26epage%3D1985%26doi%3D10.1002%2Fcmdc.201300337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hua, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchanan, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman-Perez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teffera, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Egge, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emkey, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullady, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acquaviva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newcomb, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strathdee, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turci, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMauro, E. F.</span></span> <span> </span><span class="NLM_article-title">Development of novel dual Binders as potent, selective, and orally bioavailable tankyrase inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10003</span>– <span class="NLM_lpage">10015</span>, <span class="refDoi"> DOI: 10.1021/jm401317z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401317z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFWltrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=10003-10015&author=Z.+Huaauthor=H.+Bregmanauthor=J.+L.+Buchananauthor=N.+Chakkaauthor=A.+Guzman-Perezauthor=H.+Gunaydinauthor=X.+Huangauthor=Y.+Guauthor=V.+Berryauthor=J.+Liuauthor=Y.+Tefferaauthor=L.+Huangauthor=B.+Eggeauthor=R.+Emkeyauthor=E.+L.+Mulladyauthor=S.+Schneiderauthor=P.+S.+Andrewsauthor=L.+Acquavivaauthor=J.+Doveyauthor=A.+Mishraauthor=J.+Newcombauthor=D.+Saffranauthor=R.+Serafinoauthor=C.+A.+Strathdeeauthor=S.+M.+Turciauthor=M.+Stantonauthor=C.+Wilsonauthor=E.+F.+DiMauro&title=Development+of+novel+dual+Binders+as+potent%2C+selective%2C+and+orally+bioavailable+tankyrase+inhibitors&doi=10.1021%2Fjm401317z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Novel Dual Binders as Potent, Selective, and Orally Bioavailable Tankyrase Inhibitors</span></div><div class="casAuthors">Hua, Zihao; Bregman, Howard; Buchanan, John L.; Chakka, Nagasree; Guzman-Perez, Angel; Gunaydin, Hakan; Huang, Xin; Gu, Yan; Berry, Virginia; Liu, Jingzhou; Teffera, Yohannes; Huang, Liyue; Egge, Bryan; Emkey, Renee; Mullady, Erin L.; Schneider, Steve; Andrews, Paul S.; Acquaviva, Lisa; Dovey, Jennifer; Mishra, Ankita; Newcomb, John; Saffran, Douglas; Serafino, Randy; Strathdee, Craig A.; Turci, Susan M.; Stanton, Mary; Wilson, Cindy; DiMauro, Erin F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10003-10015</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrases (TNKS1 and TNKS2) are proteins in the poly ADP-ribose polymerase (PARP) family.  They have been shown to directly bind to axin proteins, which neg. regulate the Wnt pathway by promoting β-catenin degrdn.  Inhibition of tankyrases may offer a novel approach to the treatment of APC-mutant colorectal cancer.  Hit compd. N-(2-methoxyphenyl)-4-(3-(4-oxo-3,4-dihydroquinazolin-2-yl)propanamido)benzamide was identified as an inhibitor of tankyrases through a combination of substructure searching of the Amgen compd. collection based on a minimal binding pharmacophore hypothesis and high-throughput screening.  Herein the authors report the structure- and property-based optimization of N-(2-methoxyphenyl)-4-(3-(4-oxo-3,4-dihydroquinazolin-2-yl)propanamido)benzamide leading to the identification of more potent and selective tankyrase inhibitors with improved pharmacokinetic properties in rodents, which are well suited as tool compds. for further in vivo validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-WJLflGn1C7Vg90H21EOLACvtfcHk0lj30ZfgTVp6hA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFWltrbN&md5=4620a3232873c1767b8289df2fac0c49</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm401317z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401317z%26sid%3Dliteratum%253Aachs%26aulast%3DHua%26aufirst%3DZ.%26aulast%3DBregman%26aufirst%3DH.%26aulast%3DBuchanan%26aufirst%3DJ.%2BL.%26aulast%3DChakka%26aufirst%3DN.%26aulast%3DGuzman-Perez%26aufirst%3DA.%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DBerry%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DEgge%26aufirst%3DB.%26aulast%3DEmkey%26aufirst%3DR.%26aulast%3DMullady%26aufirst%3DE.%2BL.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DAndrews%26aufirst%3DP.%2BS.%26aulast%3DAcquaviva%26aufirst%3DL.%26aulast%3DDovey%26aufirst%3DJ.%26aulast%3DMishra%26aufirst%3DA.%26aulast%3DNewcomb%26aufirst%3DJ.%26aulast%3DSaffran%26aufirst%3DD.%26aulast%3DSerafino%26aufirst%3DR.%26aulast%3DStrathdee%26aufirst%3DC.%2BA.%26aulast%3DTurci%26aufirst%3DS.%2BM.%26aulast%3DStanton%26aufirst%3DM.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DDiMauro%26aufirst%3DE.%2BF.%26atitle%3DDevelopment%2520of%2520novel%2520dual%2520Binders%2520as%2520potent%252C%2520selective%252C%2520and%2520orally%2520bioavailable%2520tankyrase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D10003%26epage%3D10015%26doi%3D10.1021%2Fjm401317z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman-Perez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acquaviva, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bregman, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dovey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saffran, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serafino, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiMauro, E. F.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of 2-aminopyridine oxazolidinones as potent and selective tankyrase inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1021/ml4003315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml4003315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1CktrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1218-1223&author=H.+Huangauthor=A.+Guzman-Perezauthor=L.+Acquavivaauthor=V.+Berryauthor=H.+Bregmanauthor=J.+Doveyauthor=H.+Gunaydinauthor=X.+Huangauthor=L.+Huangauthor=D.+Saffranauthor=R.+Serafinoauthor=S.+Schneiderauthor=C.+Wilsonauthor=E.+F.+DiMauro&title=Structure-based+design+of+2-aminopyridine+oxazolidinones+as+potent+and+selective+tankyrase+inhibitors&doi=10.1021%2Fml4003315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of 2-Aminopyridine Oxazolidinones as Potent and Selective Tankyrase Inhibitors</span></div><div class="casAuthors">Huang, Hongbing; Guzman-Perez, Angel; Acquaviva, Lisa; Berry, Virginia; Bregman, Howard; Dovey, Jennifer; Gunaydin, Hakan; Huang, Xin; Huang, Liyue; Saffran, Doug; Serafino, Randy; Schneider, Steve; Wilson, Cindy; DiMauro, Erin F.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1218-1223</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aberrant activation of the Wnt pathway has been implicated in the development and formation of many cancers.  TNKS inhibition has been shown to antagonize Wnt signaling via Axin stabilization in APC mutant colon cancer cell lines.  We employed structure-based design to identify a series of 2-aminopyridine oxazolidinones as potent and selective TNKS inhibitors.  These compds. exhibited good enzyme and cell potency as well as selectivity over other PARP isoforms.  Co-crystal structures of these 2-aminopyridine oxazolidinones complexed to TNKS reveal an induced-pocket binding mode that does not involve interactions with the nicotinamide binding pocket.  Oral dosing of lead compds. 3 and 4 resulted in significant effects on several Wnt-pathway biomarkers in a three day DLD-1 mouse tumor PD model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeQcoLA6UEfbVg90H21EOLACvtfcHk0ljONxiLMtb3Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1CktrvP&md5=9aa2a27d3694d8ed6dded65ccf85d186</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fml4003315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml4003315%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DGuzman-Perez%26aufirst%3DA.%26aulast%3DAcquaviva%26aufirst%3DL.%26aulast%3DBerry%26aufirst%3DV.%26aulast%3DBregman%26aufirst%3DH.%26aulast%3DDovey%26aufirst%3DJ.%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DSaffran%26aufirst%3DD.%26aulast%3DSerafino%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DC.%26aulast%3DDiMauro%26aufirst%3DE.%2BF.%26atitle%3DStructure-based%2520design%2520of%25202-aminopyridine%2520oxazolidinones%2520as%2520potent%2520and%2520selective%2520tankyrase%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D1218%26epage%3D1223%26doi%3D10.1021%2Fml4003315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="note"><p class="first last">Compound <b>1c</b> was reported as tankyrase inhibitor in</p></div><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wahlberg, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouznetsova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macchiarulo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorsell, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pol, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frostell, Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekblad, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Öncü, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kull, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellicciari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schüler, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigelt, J.</span></span> <span> </span><span class="NLM_article-title">Family-wide chemical profiling and structural analysis of PARP and tankyrase inhibitors</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">283</span>– <span class="NLM_lpage">288</span>, <span class="refDoi"> DOI: 10.1038/nbt.2121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fnbt.2121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22343925" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVKrsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=283-288&author=E.+Wahlbergauthor=T.+Karlbergauthor=E.+Kouznetsovaauthor=N.+Markovaauthor=A.+Macchiaruloauthor=A.+G.+Thorsellauthor=E.+Polauthor=%C3%85.+Frostellauthor=T.+Ekbladauthor=D.+%C3%96nc%C3%BCauthor=B.+Kullauthor=G.+M.+Robertsonauthor=R.+Pellicciariauthor=H.+Sch%C3%BClerauthor=J.+Weigelt&title=Family-wide+chemical+profiling+and+structural+analysis+of+PARP+and+tankyrase+inhibitors&doi=10.1038%2Fnbt.2121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Family-wide chemical profiling and structural analysis of PARP and tankyrase inhibitors</span></div><div class="casAuthors">Wahlberg, Elisabet; Karlberg, Tobias; Kouznetsova, Ekaterina; Markova, Natalia; Macchiarulo, Antonio; Thorsell, Ann-Gerd; Pol, Ewa; Frostell, Aasa; Ekblad, Torun; Oencue, Delal; Kull, Bjoern; Robertson, Graeme Michael; Pellicciari, Roberto; Schueler, Herwig; Weigelt, Johan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">283-288</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibitors of poly-ADP-ribose polymerase (PARP) family proteins are currently in clin. trials as cancer therapeutics, yet the specificity of many of these compds. is unknown.  Here we evaluated a series of 185 small-mol. inhibitors, including research reagents and compds. being tested clin., for the ability to bind to the catalytic domains of 13 of the 17 human PARP family members including the tankyrases, TNKS1 and TNKS2.  Many of the best-known inhibitors, including TIQ-A, 6(5H)-phenanthridinone, olaparib, ABT-888 and rucaparib, bound to several PARP family members, suggesting that these mols. lack specificity and have promiscuous inhibitory activity.  We also detd. X-ray crystal structures for five TNKS2 ligand complexes and four PARP14 ligand complexes.  In addn. to showing that the majority of PARP inhibitors bind multiple targets, these results provide insight into the design of new inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYSebsCVufibVg90H21EOLACvtfcHk0ljONxiLMtb3Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVKrsb4%253D&md5=71f17fd5b6bb602dd70c8fcc17df95f9</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2121%26sid%3Dliteratum%253Aachs%26aulast%3DWahlberg%26aufirst%3DE.%26aulast%3DKarlberg%26aufirst%3DT.%26aulast%3DKouznetsova%26aufirst%3DE.%26aulast%3DMarkova%26aufirst%3DN.%26aulast%3DMacchiarulo%26aufirst%3DA.%26aulast%3DThorsell%26aufirst%3DA.%2BG.%26aulast%3DPol%26aufirst%3DE.%26aulast%3DFrostell%26aufirst%3D%25C3%2585.%26aulast%3DEkblad%26aufirst%3DT.%26aulast%3D%25C3%2596nc%25C3%25BC%26aufirst%3DD.%26aulast%3DKull%26aufirst%3DB.%26aulast%3DRobertson%26aufirst%3DG.%2BM.%26aulast%3DPellicciari%26aufirst%3DR.%26aulast%3DSch%25C3%25BCler%26aufirst%3DH.%26aulast%3DWeigelt%26aufirst%3DJ.%26atitle%3DFamily-wide%2520chemical%2520profiling%2520and%2520structural%2520analysis%2520of%2520PARP%2520and%2520tankyrase%2520inhibitors%26jtitle%3DNat.%2520Biotechnol.%26date%3D2012%26volume%3D30%26spage%3D283%26epage%3D288%26doi%3D10.1038%2Fnbt.2121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Motifs for molecular recognition exploiting hydrophobic enclosure in protein–ligand binding</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">813</span>, <span class="refDoi"> DOI: 10.1073/pnas.0610202104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1073%2Fpnas.0610202104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=17204562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=808-813&author=T.+Youngauthor=R.+Abelauthor=B.+Kimauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Motifs+for+molecular+recognition+exploiting+hydrophobic+enclosure+in+protein%E2%80%93ligand+binding&doi=10.1073%2Fpnas.0610202104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Motifs for molecular recognition exploiting hydrophobic enclosure in protein-ligand binding</span></div><div class="casAuthors">Young, Tom; Abel, Robert; Kim, Byungchan; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">808-813</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The thermodn. properties and phase behavior of water in confined regions can vary significantly from that obsd. in the bulk.  This is particularly true for systems in which the confinement is on the mol.-length scale.  In this study, we use mol. dynamics simulations and a powerful solvent anal. technique based on inhomogeneous solvation theory to investigate the properties of water mols. that solvate the confined regions of protein active sites.  Our simulations and anal. indicate that the solvation of protein active sites that are characterized by hydrophobic enclosure and correlated hydrogen bonds induce atypical entropic and enthalpic penalties of hydration.  These penalties apparently stabilize the protein-ligand complex with respect to the independently solvated ligand and protein, which leads to enhanced binding affinities.  Our anal. elucidates several challenging cases, including the super affinity of the streptavidin-biotin system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWumQ61YTqILVg90H21EOLACvtfcHk0lgJlyIRq8v9Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVegtr4%253D&md5=1109e3f48670cd910885f7fd527eb880</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0610202104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0610202104%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DKim%26aufirst%3DB.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DMotifs%2520for%2520molecular%2520recognition%2520exploiting%2520hydrophobic%2520enclosure%2520in%2520protein%25E2%2580%2593ligand%2520binding%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D808%26epage%3D813%26doi%3D10.1073%2Fpnas.0610202104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the Active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>– <span class="NLM_lpage">2831</span>, <span class="refDoi"> DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+Active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lgJlyIRq8v9Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520Active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulak, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holohan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borek, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otwinowski, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garofalo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, L.</span></span> <span> </span><span class="NLM_article-title">Disruption of Wnt/β-catenin signaling and telomeric shortening are inextricable consequences of tankyrase inhibition in human cells</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">2425</span>– <span class="NLM_lpage">2435</span>, <span class="refDoi"> DOI: 10.1128/MCB.00392-15</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1128%2FMCB.00392-15" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=25939383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFaqtrrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=2425-2435&author=O.+Kulakauthor=H.+Chenauthor=B.+Holohanauthor=X.+Wuauthor=H.+Heauthor=D.+Borekauthor=Z.+Otwinowskiauthor=K.+Yamaguchiauthor=L.+A.+Garofaloauthor=Z.+Maauthor=W.+Wrightauthor=C.+Chenauthor=J.+W.+Shayauthor=X.+Zhangauthor=L.+Lum&title=Disruption+of+Wnt%2F%CE%B2-catenin+signaling+and+telomeric+shortening+are+inextricable+consequences+of+tankyrase+inhibition+in+human+cells&doi=10.1128%2FMCB.00392-15"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Disruption of Wnt/β-catenin signaling and telomeric shortening are inextricable consequences of tankyrase inhibition in human cells</span></div><div class="casAuthors">Kulak, Ozlem; Chen, Hua; Holohan, Brody; Wu, Xiaofeng; He, Huawei; Borek, Dominika; Otwinowski, Zbyszek; Yamaguchi, Kiyoshi; Garofalo, Lauren A.; Ma, Zhiqiang; Wright, Woodring; Chen, Chuo; Shay, Jerry W.; Zhang, Xuewu; Lum, Lawrence</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2425-2435</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">1098-5549</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Maintenance of chromosomal ends (telomeres) directly contributes to cancer cell immortalization.  The telomere protection enzymes belonging to the tankyrase (Tnks) subfamily of poly(ADP-ribose) polymerases (PARPs) have recently been shown to also control transcriptional response to secreted Wnt signaling mols.  Whereas Tnks inhibitors are currently being developed as therapeutic agents for targeting Wnt-related cancers and as modulators of Wnt signaling in tissue-engineering agendas, their impact on telomere length maintenance remains unclear.  Here, we leveraged a collection of Wnt pathway inhibitors with previously unassigned mechanisms of action to identify novel pharmacophores supporting Tnks inhibition.  A multifaceted exptl. approach that included structural, biochem., and cell biol. analyses revealed two distinct chemotypes with selectivity for Tnks enzymes.  Using these reagents, we revealed that Tnks inhibition rapidly induces DNA damage at telomeres and telomeric shortening upon long-term chem. exposure in cultured cells.  On the other hand, inhibitors of the Wnt acyltransferase Porcupine (Porcn) elicited neither effect.  Thus, Tnks inhibitors impact telomere length maintenance independently of their affects on Wnt/β-catenin signaling.  We discuss the implications of these findings for anticancer and regenerative medicine agendas dependent upon chem. inhibitors of Wnt/β-catenin signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsPK9Wr28kGbVg90H21EOLACvtfcHk0lgJlyIRq8v9Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFaqtrrP&md5=c1b52764be92929c71d1d51cbcc1cac5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1128%2FMCB.00392-15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00392-15%26sid%3Dliteratum%253Aachs%26aulast%3DKulak%26aufirst%3DO.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DHolohan%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DBorek%26aufirst%3DD.%26aulast%3DOtwinowski%26aufirst%3DZ.%26aulast%3DYamaguchi%26aufirst%3DK.%26aulast%3DGarofalo%26aufirst%3DL.%2BA.%26aulast%3DMa%26aufirst%3DZ.%26aulast%3DWright%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DShay%26aufirst%3DJ.%2BW.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLum%26aufirst%3DL.%26atitle%3DDisruption%2520of%2520Wnt%252F%25CE%25B2-catenin%2520signaling%2520and%2520telomeric%2520shortening%2520are%2520inextricable%2520consequences%2520of%2520tankyrase%2520inhibition%2520in%2520human%2520cells%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2015%26volume%3D35%26spage%3D2425%26epage%3D2435%26doi%3D10.1128%2FMCB.00392-15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Menon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafiq, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Cabral, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munkonge, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinidade, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esdar, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchstaller, H.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leuthner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohdich, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sansom, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wienke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashworth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lord, C. J.</span></span> <span> </span><span class="NLM_article-title">A novel tankyrase inhibitor, MSC2504877, enhances the effects of clinical CDK4/6 inhibitors</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">201</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-36447-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1038%2Fs41598-018-36447-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=30655555" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A280%3ADC%252BB3cjis1Gjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=201&author=M.+Menonauthor=R.+Elliottauthor=L.+Bowersauthor=N.+Balanauthor=R.+Rafiqauthor=S.+Costa-Cabralauthor=F.+Munkongeauthor=I.+Trinidadeauthor=R.+Porterauthor=A.+D.+Campbellauthor=E.+R.+Johnsonauthor=C.+Esdarauthor=H.-P.+Buchstallerauthor=B.+Leuthnerauthor=F.+Rohdichauthor=R.+Schneiderauthor=O.+Sansomauthor=D.+Wienkeauthor=A.+Ashworthauthor=C.+J.+Lord&title=A+novel+tankyrase+inhibitor%2C+MSC2504877%2C+enhances+the+effects+of+clinical+CDK4%2F6+inhibitors&doi=10.1038%2Fs41598-018-36447-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">A novel tankyrase inhibitor, MSC2504877, enhances the effects of clinical CDK4/6 inhibitors</span></div><div class="casAuthors">Menon Malini; Elliott Richard; Bowers Leandra; Balan Nicolae; Rafiq Rumana; Costa-Cabral Sara; Munkonge Felix; Trinidade Ines; Ashworth Alan; Lord Christopher J; Porter Roderick; Campbell Andrew D; Johnson Emma R; Sansom Owen; Esdar Christina; Buchstaller Hans-Peter; Leuthner Birgitta; Rohdich Felix; Schneider Richard; Wienke Dirk; Ashworth Alan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">201</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Inhibition of the PARP superfamily tankyrase enzymes suppresses Wnt/β-catenin signalling in tumour cells.  Here, we describe here a novel, drug-like small molecule inhibitor of tankyrase MSC2504877 that inhibits the growth of APC mutant colorectal tumour cells.  Parallel siRNA and drug sensitivity screens showed that the clinical CDK4/6 inhibitor palbociclib, causes enhanced sensitivity to MSC2504877.  This tankyrase inhibitor-CDK4/6 inhibitor combinatorial effect is not limited to palbociclib and MSC2504877 and is elicited with other CDK4/6 inhibitors and toolbox tankyrase inhibitors.  The addition of MSC2504877 to palbociclib enhances G1 cell cycle arrest and cellular senescence in tumour cells.  MSC2504877 exposure suppresses the upregulation of Cyclin D2 and Cyclin E2 caused by palbociclib and enhances the suppression of phospho-Rb, providing a mechanistic explanation for these effects.  The combination of MSC2504877 and palbociclib was also effective in suppressing the cellular hyperproliferative phenotype seen in Apc defective intestinal stem cells in vivo.  However, the presence of an oncogenic Kras p.G12D mutation in mice reversed the effects of the MSC2504877/palbociclib combination, suggesting one molecular route that could lead to drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRsCECdrvD2BkewzGPswiWOfW6udTcc2ebKov_FXf9uMrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjis1Gjtg%253D%253D&md5=34ac002987c951a0ebd2f34f256ed21c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-36447-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-36447-4%26sid%3Dliteratum%253Aachs%26aulast%3DMenon%26aufirst%3DM.%26aulast%3DElliott%26aufirst%3DR.%26aulast%3DBowers%26aufirst%3DL.%26aulast%3DBalan%26aufirst%3DN.%26aulast%3DRafiq%26aufirst%3DR.%26aulast%3DCosta-Cabral%26aufirst%3DS.%26aulast%3DMunkonge%26aufirst%3DF.%26aulast%3DTrinidade%26aufirst%3DI.%26aulast%3DPorter%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DA.%2BD.%26aulast%3DJohnson%26aufirst%3DE.%2BR.%26aulast%3DEsdar%26aufirst%3DC.%26aulast%3DBuchstaller%26aufirst%3DH.-P.%26aulast%3DLeuthner%26aufirst%3DB.%26aulast%3DRohdich%26aufirst%3DF.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DSansom%26aufirst%3DO.%26aulast%3DWienke%26aufirst%3DD.%26aulast%3DAshworth%26aufirst%3DA.%26aulast%3DLord%26aufirst%3DC.%2BJ.%26atitle%3DA%2520novel%2520tankyrase%2520inhibitor%252C%2520MSC2504877%252C%2520enhances%2520the%2520effects%2520of%2520clinical%2520CDK4%252F6%2520inhibitors%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D201%26doi%3D10.1038%2Fs41598-018-36447-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirby, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Firestone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPietro, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fazal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fortin, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gould, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenoir, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumdar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochala, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palermo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stams, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomlinson, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Touré, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Visser, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, W.</span></span> <span> </span><span class="NLM_article-title">Identification of NVP-TNKS656: The use of structure–efficiency relationships to generate a highly potent, selective, and orally active tankyrase inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">6495</span>– <span class="NLM_lpage">6511</span>, <span class="refDoi"> DOI: 10.1021/jm400807n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400807n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtV2msbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=6495-6511&author=M.+D.+Shultzauthor=A.+K.+Cheungauthor=C.+A.+Kirbyauthor=B.+Firestoneauthor=J.+Fanauthor=C.+H.-T.+Chenauthor=Z.+Chenauthor=D.+N.+Chinauthor=L.+DiPietroauthor=A.+Fazalauthor=Y.+Fengauthor=P.+D.+Fortinauthor=T.+Gouldauthor=B.+Laguauthor=H.+Leiauthor=F.+Lenoirauthor=D.+Majumdarauthor=E.+Ochalaauthor=M.+G.+Palermoauthor=L.+Phamauthor=M.+Puauthor=T.+Smithauthor=T.+Stamsauthor=R.+C.+Tomlinsonauthor=B.+B.+Tour%C3%A9author=M.+Visserauthor=R.+M.+Wangauthor=N.+J.+Watersauthor=W.+Shao&title=Identification+of+NVP-TNKS656%3A+The+use+of+structure%E2%80%93efficiency+relationships+to+generate+a+highly+potent%2C+selective%2C+and+orally+active+tankyrase+inhibitor&doi=10.1021%2Fjm400807n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of NVP-TNKS656: The Use of Structure-Efficiency Relationships To Generate a Highly Potent, Selective, and Orally Active Tankyrase Inhibitor</span></div><div class="casAuthors">Shultz, Michael D.; Cheung, Atwood K.; Kirby, Christina A.; Firestone, Brant; Fan, Jianmei; Chen, Christine Hiu-Tung; Chen, Zhouliang; Chin, Donovan N.; DiPietro, Lucian; Fazal, Aleem; Feng, Yun; Fortin, Pascal D.; Gould, Ty; Lagu, Bharat; Lei, Huangshu; Lenoir, Francois; Majumdar, Dyuti; Ochala, Etienne; Palermo, M. G.; Pham, Ly; Pu, Minying; Smith, Troy; Stams, Travis; Tomlinson, Ronald C.; Toure, B. Barry; Visser, Michael; Wang, Run Ming; Waters, Nigel J.; Shao, Wenlin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6495-6511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tankyrase 1 and 2 have been shown to be redundant, druggable nodes in the Wnt pathway.  As such, there has been intense interest in developing agents suitable for modulating the Wnt pathway in vivo by targeting this enzyme pair.  By utilizing a combination of structure-based design and LipE-based structure efficiency relationships, the core of XAV939 was optimized into a more stable, more efficient, but less potent dihydropyran motif 7.  This core was combined with elements of screening hits 2, 19, and 33 and resulted in highly potent, selective tankyrase inhibitors that are novel three pocket binders.  NVP-TNKS656 (43) was identified as an orally active antagonist of Wnt pathway activity in the MMTV-Wnt1 mouse xenograft model.  With an enthalpy-driven thermodn. signature of binding, highly favorable physicochem. properties, and high lipophilic efficiency, NVP-TNKS656 is a novel tankyrase inhibitor that is well suited for further in vivo validation studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDGWOimaJ0I7Vg90H21EOLACvtfcHk0lhUZRIbitlEsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtV2msbfE&md5=db3f1a471ca757e0219a5175c5017e9b</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm400807n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400807n%26sid%3Dliteratum%253Aachs%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DKirby%26aufirst%3DC.%2BA.%26aulast%3DFirestone%26aufirst%3DB.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DC.%2BH.-T.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DDiPietro%26aufirst%3DL.%26aulast%3DFazal%26aufirst%3DA.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DFortin%26aufirst%3DP.%2BD.%26aulast%3DGould%26aufirst%3DT.%26aulast%3DLagu%26aufirst%3DB.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DLenoir%26aufirst%3DF.%26aulast%3DMajumdar%26aufirst%3DD.%26aulast%3DOchala%26aufirst%3DE.%26aulast%3DPalermo%26aufirst%3DM.%2BG.%26aulast%3DPham%26aufirst%3DL.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DT.%26aulast%3DStams%26aufirst%3DT.%26aulast%3DTomlinson%26aufirst%3DR.%2BC.%26aulast%3DTour%25C3%25A9%26aufirst%3DB.%2BB.%26aulast%3DVisser%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DR.%2BM.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DShao%26aufirst%3DW.%26atitle%3DIdentification%2520of%2520NVP-TNKS656%253A%2520The%2520use%2520of%2520structure%25E2%2580%2593efficiency%2520relationships%2520to%2520generate%2520a%2520highly%2520potent%252C%2520selective%252C%2520and%2520orally%2520active%2520tankyrase%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D6495%26epage%3D6511%26doi%3D10.1021%2Fjm400807n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yarravarapu, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulak, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dasari, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herbert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span> <span> </span><span class="NLM_article-title">Regulation of tankyrase activity by a catalytic domain dimer interface</span>. <i>Biochem. Biophys. Res. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">1780</span>– <span class="NLM_lpage">1785</span>, <span class="refDoi"> DOI: 10.1016/j.bbrc.2018.07.113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.bbrc.2018.07.113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=30055800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVShtrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2018&pages=1780-1785&author=C.+Fanauthor=N.+Yarravarapuauthor=H.+Chenauthor=O.+Kulakauthor=P.+Dasariauthor=J.+Herbertauthor=K.+Yamaguchiauthor=L.+Lumauthor=X.+Zhang&title=Regulation+of+tankyrase+activity+by+a+catalytic+domain+dimer+interface&doi=10.1016%2Fj.bbrc.2018.07.113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of tankyrase activity by a catalytic domain dimer interface</span></div><div class="casAuthors">Fan, Chen; Yarravarapu, Nageswari; Chen, Hua; Kulak, Ozlem; Dasari, Pranathi; Herbert, Jeremiah; Yamaguchi, Kiyoshi; Lum, Lawrence; Zhang, Xuewu</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1780-1785</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Tankyrases (TNKS and TNKS2) are enzymes that catalyze poly-ADP-ribosylation (PARsylation) of their target proteins.  Tankyrase-mediated PARsylation plays crit. regulatory roles in cell signaling, particularly in the Wnt/β-catenin pathway.  The sterile alpha motif (SAM) domain in tankyrases mediates their oligomerization, which is essential for tankyrase function.  The oligomerization regulates the catalytic activity of tankyrases, but the underlying mechanism is unclear.  Our analyses of crystal structures of the tankyrase catalytic domain suggest that formation of a head-to-head dimer regulates the catalytic activity.  Our activity assays show that residues in the catalytic domain dimer interface are important for the PARsylation activity of tankyrases both in soln. and cells.  The dimer is weak and may only form in the context of the SAM domain-mediated oligomers of tankyrases, consistent with the dependence of the tankyrase activity on the SAM domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqrAIbzOSgzWbVg90H21EOLACvtfcHk0lilRqHf8wgcOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVShtrbN&md5=6dc34e2cb632da1f34dac69c4119b79f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2018.07.113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2018.07.113%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DC.%26aulast%3DYarravarapu%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DKulak%26aufirst%3DO.%26aulast%3DDasari%26aufirst%3DP.%26aulast%3DHerbert%26aufirst%3DJ.%26aulast%3DYamaguchi%26aufirst%3DK.%26aulast%3DLum%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26atitle%3DRegulation%2520of%2520tankyrase%2520activity%2520by%2520a%2520catalytic%2520domain%2520dimer%2520interface%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2018%26volume%3D503%26spage%3D1780%26epage%3D1785%26doi%3D10.1016%2Fj.bbrc.2018.07.113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marala, R. B. J.</span></span> <span> </span><span class="NLM_article-title">Development of a high-throughput screening-amenable assay for human poly(ADP-ribose) polymerase inhibitors</span>. <i>J. Pharmacol. Toxicol. Methods</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">137</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1016/S1056-8719(02)00223-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2FS1056-8719%2802%2900223-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=12628304" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhs1ahs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2002&pages=137-141&author=J.+A.+Brownauthor=R.+B.+J.+Marala&title=Development+of+a+high-throughput+screening-amenable+assay+for+human+poly%28ADP-ribose%29+polymerase+inhibitors&doi=10.1016%2FS1056-8719%2802%2900223-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a high-throughput screening-amenable assay for human poly(ADP-ribose) polymerase inhibitors</span></div><div class="casAuthors">Brown, Janice A.; Marala, Ravi B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacological and Toxicological Methods</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">137-141</span>CODEN:
                <span class="NLM_cas:coden">JPTMEZ</span>;
        ISSN:<span class="NLM_cas:issn">1056-8719</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase (PARP) plays a pivotal role in the repair of DNA strand breaks.  However, excessive activation of PARP causes a rapid depletion of intracellular energy, leading to cell death.  Inhibitors of PARP have been shown to reduce infarct size in animal models of myocardial ischemia.  PARP inhibitors may have potential therapeutic benefit in the treatment of myocardial ischemia, stroke, head trauma, and neurodegenerative disease, and as an adjunct therapy with chemotherapeutic agents/radiation in cancer therapy.  Assays reported in the literature and com. available PARP assay kits are labor-intensive, use radioactive reagents, use antibodies, and are not readily amenable to Hadamard transform spectroscopy (HTS).  Here we report the development and the validation of a nonradioactive PARP assay suitable for HTS.  This is a biotinylated NAD-based colorimetric assay in a 96-well plate format.  The assay is sensitive, reproducible, and easy to use.  The IC50 values generated for the known PARP inhibitors are in agreement with those generated using the com. radioactive kit and those reported in the literature.  The present study demonstrates a sensitive and reproducible methodol. capable of screening human PARP inhibitors in high-throughput format.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3Me-JvpXPsLVg90H21EOLACvtfcHk0lilRqHf8wgcOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhs1ahs7o%253D&md5=d1eb6600deeaa8abbaac3573043a3138</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS1056-8719%2802%2900223-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1056-8719%252802%252900223-X%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DJ.%2BA.%26aulast%3DMarala%26aufirst%3DR.%2BB.%2BJ.%26atitle%3DDevelopment%2520of%2520a%2520high-throughput%2520screening-amenable%2520assay%2520for%2520human%2520poly%2528ADP-ribose%2529%2520polymerase%2520inhibitors%26jtitle%3DJ.%2520Pharmacol.%2520Toxicol.%2520Methods%26date%3D2002%26volume%3D47%26spage%3D137%26epage%3D141%26doi%3D10.1016%2FS1056-8719%2802%2900223-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gunaydin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span> <span> </span><span class="NLM_article-title">Novel binding mode of a potent and selective tankyrase inhibitor</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>, <span class="NLM_elocation-id">e33740</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0033740</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1371%2Fjournal.pone.0033740" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22438990" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVaht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&author=H.+Gunaydinauthor=Y.+Guauthor=X.+Huang&title=Novel+binding+mode+of+a+potent+and+selective+tankyrase+inhibitor&doi=10.1371%2Fjournal.pone.0033740"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel binding mode of a potent and selective tankyrase inhibitor</span></div><div class="casAuthors">Gunaydin, Hakan; Gu, Yan; Huang, Xin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e33740</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Tankyrases (TNKS1 and TNKS2) are key regulators of cellular processes such as telomere pathway and Wnt signaling.  IWRs (inhibitors of Wnt response) have recently been identified as potent and selective inhibitors of tankyrases.  However, it is not clear how these IWRs interact with tankyrases.  Here we report the crystal structure of the catalytic domain of human TNKS1 in complex with IWR2, which reveals a novel binding site for tankyrase inhibitors.  The TNKS1/IWR2 complex provides a mol. basis for their strong and specific interactions and suggests clues for further development of tankyrase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqidLqR8KN4obVg90H21EOLACvtfcHk0ljBhE-iFGtr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVaht70%253D&md5=7ab7c96221b170534538c51605e76e85</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0033740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0033740%26sid%3Dliteratum%253Aachs%26aulast%3DGunaydin%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DX.%26atitle%3DNovel%2520binding%2520mode%2520of%2520a%2520potent%2520and%2520selective%2520tankyrase%2520inhibitor%26jtitle%3DPLoS%2520One%26date%3D2012%26volume%3D7%26doi%3D10.1371%2Fjournal.pone.0033740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Casanas, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warshamanage, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finke, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panepucci, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olieric, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nöll, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tampé, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandstetter, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Förster, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulze-Briese, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunk, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span> <span> </span><span class="NLM_article-title">EIGER detector: Application in macromolecular crystallography</span>. <i>Acta Crystallogr. Sect. D Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>72</i></span>,  <span class="NLM_fpage">1036</span>– <span class="NLM_lpage">1048</span>, <span class="refDoi"> DOI: 10.1107/S2059798316012304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS2059798316012304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=72&publication_year=2016&pages=1036-1048&author=A.+Casanasauthor=R.+Warshamanageauthor=A.+D.+Finkeauthor=E.+Panepucciauthor=V.+Oliericauthor=A.+N%C3%B6llauthor=R.+Tamp%C3%A9author=S.+Brandstetterauthor=A.+F%C3%B6rsterauthor=M.+Muellerauthor=C.+Schulze-Brieseauthor=O.+Bunkauthor=M.+Wang&title=EIGER+detector%3A+Application+in+macromolecular+crystallography&doi=10.1107%2FS2059798316012304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">EIGER detector: application in macromolecular crystallography</span></div><div class="casAuthors">Casanas, Arnau; Warshamanage, Rangana; Finke, Aaron D.; Panepucci, Ezequiel; Olieric, Vincent; Noll, Anne; Tampe, Robert; Brandstetter, Stefan; Forster, Andreas; Mueller, Marcus; Schulze-Briese, Clemens; Bunk, Oliver; Wang, Meitian</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">72</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1036-1048</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The development of single-photon-counting detectors, such as the PILATUS, has been a major recent breakthrough in macromol. crystallog., enabling noise-free detection and novel data-acquisition modes.  The new EIGER detector features a pixel size of 75 × 75 μm, frame rates of up to 3000 Hz and a dead time as low as 3.8 μs.  An EIGER 1M and EIGER 16M were tested on Swiss Light Source beamlines X10SA and X06SA for their application in macromol. crystallog.  The combination of fast frame rates and a very short dead time allows high-quality data acquisition in a shorter time.  The ultrafine φ-slicing data-collection method is introduced and validated and its application in finding the optimal rotation angle, a suitable rotation speed and a sufficient X-ray dose are presented.  An improvement of the data quality up to slicing at one tenth of the mosaicity has been obsd., which is much finer than expected based on previous findings.  The influence of key data-collection parameters on data quality is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2BJ29hEBUhrVg90H21EOLACvtfcHk0ljBhE-iFGtr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOhtr7I&md5=8230d3621ab53559b2f120d6a98c40de</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS2059798316012304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798316012304%26sid%3Dliteratum%253Aachs%26aulast%3DCasanas%26aufirst%3DA.%26aulast%3DWarshamanage%26aufirst%3DR.%26aulast%3DFinke%26aufirst%3DA.%2BD.%26aulast%3DPanepucci%26aufirst%3DE.%26aulast%3DOlieric%26aufirst%3DV.%26aulast%3DN%25C3%25B6ll%26aufirst%3DA.%26aulast%3DTamp%25C3%25A9%26aufirst%3DR.%26aulast%3DBrandstetter%26aufirst%3DS.%26aulast%3DF%25C3%25B6rster%26aufirst%3DA.%26aulast%3DMueller%26aufirst%3DM.%26aulast%3DSchulze-Briese%26aufirst%3DC.%26aulast%3DBunk%26aufirst%3DO.%26aulast%3DWang%26aufirst%3DM.%26atitle%3DEIGER%2520detector%253A%2520Application%2520in%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%2520Sect.%2520D%2520Struct.%2520Biol.%26date%3D2016%26volume%3D72%26spage%3D1036%26epage%3D1048%26doi%3D10.1107%2FS2059798316012304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Kabsch%2C+W.+Acta+Crystallogr.%2C+Sect.+D%3A+Biol.+Crystallogr.+2010%2C+66%2C+125%E2%80%93132%2C+10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljBhE-iFGtr4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Collaborative
computational project, N. 4</span> <span> </span><span class="NLM_article-title">The CCP4 suite: Programs for protein crystallography</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative%0Acomputational+project%2C+N.+4&title=The+CCP4+suite%3A+Programs+for+protein+crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520suite%253A%2520Programs%2520for%2520protein%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version 2.11.7; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>– <span class="NLM_lpage">2132</span>, <span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgV3-UIzh-8ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luckert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Götschel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hecht, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joos, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pötz, O.</span></span> <span> </span><span class="NLM_article-title">Snapshots of protein dynamics and post-translational modifications in one experiment - beta-catenin and its functions</span>. <i>Mol. Cell. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">M110.007377</span>, <span class="refDoi"> DOI: 10.1074/mcp.M110.007377</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1074%2Fmcp.M110.007377" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=21378377" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=M110.007377&author=K.+Luckertauthor=F.+G%C3%B6tschelauthor=P.+K.+Sorgerauthor=A.+Hechtauthor=T.+O.+Joosauthor=O.+P%C3%B6tz&title=Snapshots+of+protein+dynamics+and+post-translational+modifications+in+one+experiment+-+beta-catenin+and+its+functions&doi=10.1074%2Fmcp.M110.007377"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M110.007377&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M110.007377%26sid%3Dliteratum%253Aachs%26aulast%3DLuckert%26aufirst%3DK.%26aulast%3DG%25C3%25B6tschel%26aufirst%3DF.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DHecht%26aufirst%3DA.%26aulast%3DJoos%26aufirst%3DT.%2BO.%26aulast%3DP%25C3%25B6tz%26aufirst%3DO.%26atitle%3DSnapshots%2520of%2520protein%2520dynamics%2520and%2520post-translational%2520modifications%2520in%2520one%2520experiment%2520-%2520beta-catenin%2520and%2520its%2520functions%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2011%26volume%3D10%26spage%3DM110.007377%26doi%3D10.1074%2Fmcp.M110.007377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luckert, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gujral, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sevecka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joos, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorger, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macbeath, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pötz, O.</span></span> <span> </span><span class="NLM_article-title">A dual array-based approach to assess the abundance and posttranslational modification state of signaling proteins</span>. <i>Sci. Signaling</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">pl1</span>, <span class="refDoi"> DOI: 10.1126/scisignal.2002372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1126%2Fscisignal.2002372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22234610" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2012&pages=pl1&author=K.+Luckertauthor=T.+S.+Gujralauthor=M.+Chanauthor=M.+Seveckaauthor=T.+O.+Joosauthor=P.+K.+Sorgerauthor=G.+Macbeathauthor=O.+P%C3%B6tz&title=A+dual+array-based+approach+to+assess+the+abundance+and+posttranslational+modification+state+of+signaling+proteins&doi=10.1126%2Fscisignal.2002372"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2002372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2002372%26sid%3Dliteratum%253Aachs%26aulast%3DLuckert%26aufirst%3DK.%26aulast%3DGujral%26aufirst%3DT.%2BS.%26aulast%3DChan%26aufirst%3DM.%26aulast%3DSevecka%26aufirst%3DM.%26aulast%3DJoos%26aufirst%3DT.%2BO.%26aulast%3DSorger%26aufirst%3DP.%2BK.%26aulast%3DMacbeath%26aufirst%3DG.%26aulast%3DP%25C3%25B6tz%26aufirst%3DO.%26atitle%3DA%2520dual%2520array-based%2520approach%2520to%2520assess%2520the%2520abundance%2520and%2520posttranslational%2520modification%2520state%2520of%2520signaling%2520proteins%26jtitle%3DSci.%2520Signaling%26date%3D2012%26volume%3D5%26spage%3Dpl1%26doi%3D10.1126%2Fscisignal.2002372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petereit, A. C.</span></span> <span> </span><span class="NLM_article-title">Optimizing solubility: kinetic versus thermodynamic solubility temptations and risks</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">589</span>– <span class="NLM_lpage">595</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2012.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=10.1016%2Fj.ejps.2012.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=22885099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2012&pages=589-595&author=C.+Saalauthor=A.+C.+Petereit&title=Optimizing+solubility%3A+kinetic+versus+thermodynamic+solubility+temptations+and+risks&doi=10.1016%2Fj.ejps.2012.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing solubility: Kinetic versus thermodynamic solubility temptations and risks</span></div><div class="casAuthors">Saal, Christoph; Petereit, Anna Christine</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">589-595</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aim of this study was to assess the usefulness of kinetic and thermodn. soly. data in guiding medicinal chem. during lead optimization.  The soly. of 465 research compds. was measured using a kinetic and a thermodn. soly. assay.  In the thermodn. assay, polarized-light microscopy was used to investigate whether the result referred to the cryst. or to the amorphous compd.  From the comparison of kinetic and thermodn. soly. data it was noted that kinetic soly. measurements frequently yielded results which show considerably higher soly. compared to thermodn. soly.  This observation is ascribed to the fact that a kinetic soly. assay typically delivers results which refer to the amorphous compd.  In contrast, results from thermodn. soly. detns. more frequently refer to a cryst. phase.  Accordingly, thermodn. soly. data - esp. when used together with an assessment of the solid state form - are deemed to be more useful in guiding soly. optimization for research compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooxKTQHBpynbVg90H21EOLACvtfcHk0lj4kGYYGarhhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2rs7bJ&md5=ace8b962c04c3a62f028ec3e66522922</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2012.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2012.07.019%26sid%3Dliteratum%253Aachs%26aulast%3DSaal%26aufirst%3DC.%26aulast%3DPetereit%26aufirst%3DA.%2BC.%26atitle%3DOptimizing%2520solubility%253A%2520kinetic%2520versus%2520thermodynamic%2520solubility%2520temptations%2520and%2520risks%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2012%26volume%3D47%26spage%3D589%26epage%3D595%26doi%3D10.1016%2Fj.ejps.2012.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UH4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3UH4','PDB','3UH4'); return false;">PDB: 3UH4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AKU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5AKU','PDB','5AKU'); return false;">PDB: 5AKU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU','PDB','6QXU'); return false;">PDB: 6QXU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RF5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2RF5','PDB','2RF5'); return false;">PDB: 2RF5</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i67"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00656">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_62262"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00656">10.1021/acs.jmedchem.9b00656</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Assay data of the best compounds with SD/SEM for the values, assay curve of biochemical assay IC<sub>50</sub> determination for <b>5k</b>, ligand <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map at 2.5σ cutoff, protocol for determination of microsomal stability (intrinsic clearance), protocol for the determination of Kv11.1 (hERG) ion channel activity, and procedure for Caco-2 <i>P</i><sub>app</sub> determination (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_001.pdf">jm9b00656_si_001.pdf (382.16 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00656/suppl_file/jm9b00656_si_002.csv">jm9b00656_si_002.csv (1.43 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Coordinates and structure factors are deposited at the Protein Data Bank with code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6QXU">6QXU</a> (<b>5k</b>). The authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00656&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00656%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-17%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00656" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799990b9e943d5a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
