
<html lang="en"     class="pb-page"  data-request-id="668d7cf4-e95b-4e4c-9fd7-666a49b5cdc5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2019.62.issue-17;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.9b00718;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis" /></meta><meta name="dc.Creator" content="Harald  Weinstabl" /></meta><meta name="dc.Creator" content="Matthias  Treu" /></meta><meta name="dc.Creator" content="Joerg  Rinnenthal" /></meta><meta name="dc.Creator" content="Stephan K.  Zahn" /></meta><meta name="dc.Creator" content="Peter  Ettmayer" /></meta><meta name="dc.Creator" content="Gerd  Bader" /></meta><meta name="dc.Creator" content="Georg  Dahmann" /></meta><meta name="dc.Creator" content="Dirk  Kessler" /></meta><meta name="dc.Creator" content="Klaus  Rumpel" /></meta><meta name="dc.Creator" content="Nikolai  Mischerikow" /></meta><meta name="dc.Creator" content="Fabio  Savarese" /></meta><meta name="dc.Creator" content="Thomas  Gerstberger" /></meta><meta name="dc.Creator" content="Moriz  Mayer" /></meta><meta name="dc.Creator" content="Andreas  Zoephel" /></meta><meta name="dc.Creator" content="Renate  Schnitzer" /></meta><meta name="dc.Creator" content="Wolfgang  Sommergruber" /></meta><meta name="dc.Creator" content="Paola  Martinelli" /></meta><meta name="dc.Creator" content="Heribert  Arnhof" /></meta><meta name="dc.Creator" content="Biljana  Peric-Simov" /></meta><meta name="dc.Creator" content="Karin S.  Hofbauer" /></meta><meta name="dc.Creator" content="Géraldine  Garavel" /></meta><meta name="dc.Creator" content="Yvonne  Scherbantin" /></meta><meta name="dc.Creator" content="Sophie  Mitzner" /></meta><meta name="dc.Creator" content="Thomas N.  Fett" /></meta><meta name="dc.Creator" content="Guido  Scholz" /></meta><meta name="dc.Creator" content="Jens  Bruchhaus" /></meta><meta name="dc.Creator" content="Michelle  Burkard" /></meta><meta name="dc.Creator" content="Roland  Kousek" /></meta><meta name="dc.Creator" content="Tuncay  Ciftci" /></meta><meta name="dc.Creator" content="Bernadette  Sharps" /></meta><meta name="dc.Creator" content="Andreas  Schrenk" /></meta><meta name="dc.Creator" content="Christoph  Harrer" /></meta><meta name="dc.Creator" content="Daniela  Haering" /></meta><meta name="dc.Creator" content="Bernhard  Wolkerstorfer" /></meta><meta name="dc.Creator" content="Xuechun  Zhang" /></meta><meta name="dc.Creator" content="Xiaobing  Lv" /></meta><meta name="dc.Creator" content="Alicia  Du" /></meta><meta name="dc.Creator" content="Dongyang  Li" /></meta><meta name="dc.Creator" content="Yali  Li" /></meta><meta name="dc.Creator" content="Jens  Quant" /></meta><meta name="dc.Creator" content="Mark  Pearson" /></meta><meta name="dc.Creator" content="Darryl B.  McConnell" /></meta><meta name="dc.Description" content="Phosphoglycerate dehydrogenase (PHGDH) is known to be the rate-limiting enzyme in the serine synthesis pathway in humans. It converts glycolysis-derived 3-phosphoglycerate to 3-phosphopyruvate in a..." /></meta><meta name="Description" content="Phosphoglycerate dehydrogenase (PHGDH) is known to be the rate-limiting enzyme in the serine synthesis pathway in humans. It converts glycolysis-derived 3-phosphoglycerate to 3-phosphopyruvate in a..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 31, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00718" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00718" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00718" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00718" /></link>
        
    
    

<title>Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00718" /></meta><meta property="og:title" content="Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0029.jpeg" /></meta><meta property="og:description" content="Phosphoglycerate dehydrogenase (PHGDH) is known to be the rate-limiting enzyme in the serine synthesis pathway in humans. It converts glycolysis-derived 3-phosphoglycerate to 3-phosphopyruvate in a co-factor-dependent oxidation reaction. Herein, we report the discovery of BI-4916, a prodrug of the co-factor nicotinamide adenine dinucleotide (NADH/NAD+)-competitive  PHGDH inhibitor BI-4924, which has shown high selectivity against the majority of other dehydrogenase targets. Starting with a fragment-based screening, a subsequent hit optimization using structure-based drug design was conducted to deliver a single-digit nanomolar lead series and to improve potency by 6 orders of magnitude. To this end, an intracellular ester cleavage mechanism of the ester prodrug was utilized to achieve intracellular enrichment of the actual carboxylic acid based drug and thus overcome high cytosolic levels of the competitive cofactors NADH/NAD+." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00718"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00718">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00718&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00718&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00718&amp;href=/doi/10.1021/acs.jmedchem.9b00718" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 7976-7997</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00708" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00720" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor <b>BI-4924</b> Disrupts Serine Biosynthesis</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Harald Weinstabl</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Harald Weinstabl</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#cea6afbcafa2aae0b9aba7a0bdbaafaca28eaca1aba6bca7a0a9abbce3a7a0a9aba2a6aba7a3e0ada1a3"><span class="__cf_email__" data-cfemail="71191003101d155f0614181f020510131d31131e141903181f1614035c181f16141d1914181c5f121e1c">[email protected]</span></a>. Tel: +43-1-80105-2803.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Harald++Weinstabl">Harald Weinstabl</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1308-2730" title="Orcid link">http://orcid.org/0000-0003-1308-2730</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthias Treu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthias Treu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthias++Treu">Matthias Treu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Joerg Rinnenthal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Joerg Rinnenthal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Joerg++Rinnenthal">Joerg Rinnenthal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Stephan K. Zahn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stephan K. Zahn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stephan+K.++Zahn">Stephan K. Zahn</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Peter Ettmayer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Peter Ettmayer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Ettmayer">Peter Ettmayer</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8422-2625" title="Orcid link">http://orcid.org/0000-0002-8422-2625</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gerd Bader</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gerd Bader</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gerd++Bader">Gerd Bader</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Georg Dahmann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Georg Dahmann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim Pharma GmbH & Co. KG, Birkendorfer Str. 65, 88400 Biberach an der Riß, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Georg++Dahmann">Georg Dahmann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dirk Kessler</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dirk Kessler</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dirk++Kessler">Dirk Kessler</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Klaus Rumpel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Klaus Rumpel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Klaus++Rumpel">Klaus Rumpel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nikolai Mischerikow</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nikolai Mischerikow</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nikolai++Mischerikow">Nikolai Mischerikow</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Fabio Savarese</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Fabio Savarese</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Fabio++Savarese">Fabio Savarese</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas Gerstberger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Gerstberger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Gerstberger">Thomas Gerstberger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Moriz Mayer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Moriz Mayer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Moriz++Mayer">Moriz Mayer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Zoephel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Zoephel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Zoephel">Andreas Zoephel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Renate Schnitzer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Renate Schnitzer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Renate++Schnitzer">Renate Schnitzer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wolfgang Sommergruber</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wolfgang Sommergruber</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wolfgang++Sommergruber">Wolfgang Sommergruber</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paola Martinelli</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paola Martinelli</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paola++Martinelli">Paola Martinelli</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heribert Arnhof</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heribert Arnhof</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heribert++Arnhof">Heribert Arnhof</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Biljana Peric-Simov</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Biljana Peric-Simov</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Biljana++Peric-Simov">Biljana Peric-Simov</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Karin S. Hofbauer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Karin S. Hofbauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Karin+S.++Hofbauer">Karin S. Hofbauer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Géraldine Garavel</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Géraldine Garavel</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=G%C3%A9raldine++Garavel">Géraldine Garavel</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yvonne Scherbantin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yvonne Scherbantin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yvonne++Scherbantin">Yvonne Scherbantin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sophie Mitzner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sophie Mitzner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sophie++Mitzner">Sophie Mitzner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Thomas N. Fett</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas N. Fett</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas+N.++Fett">Thomas N. Fett</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guido Scholz</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guido Scholz</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guido++Scholz">Guido Scholz</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jens Bruchhaus</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jens Bruchhaus</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jens++Bruchhaus">Jens Bruchhaus</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michelle Burkard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michelle Burkard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michelle++Burkard">Michelle Burkard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Roland Kousek</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roland Kousek</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roland++Kousek">Roland Kousek</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tuncay Ciftci</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tuncay Ciftci</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim Pharma GmbH & Co. KG, Birkendorfer Str. 65, 88400 Biberach an der Riß, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tuncay++Ciftci">Tuncay Ciftci</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bernadette Sharps</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bernadette Sharps</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bernadette++Sharps">Bernadette Sharps</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Andreas Schrenk</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas Schrenk</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Schrenk">Andreas Schrenk</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christoph Harrer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christoph Harrer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christoph++Harrer">Christoph Harrer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniela Haering</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniela Haering</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniela++Haering">Daniela Haering</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bernhard Wolkerstorfer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bernhard Wolkerstorfer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bernhard++Wolkerstorfer">Bernhard Wolkerstorfer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xuechun Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xuechun Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xuechun++Zhang">Xuechun Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaobing Lv</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaobing Lv</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaobing++Lv">Xiaobing Lv</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Alicia Du</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Alicia Du</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Alicia++Du">Alicia Du</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dongyang Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dongyang Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dongyang++Li">Dongyang Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yali Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yali Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech Park, Pudong New Area, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yali++Li">Yali Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jens Quant</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jens Quant</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jens++Quant">Jens Quant</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mark Pearson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mark Pearson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mark++Pearson">Mark Pearson</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Darryl B. McConnell</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Darryl B. McConnell</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Boehringer Ingelheim RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Darryl+B.++McConnell">Darryl B. McConnell</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00718&amp;href=/doi/10.1021%2Facs.jmedchem.9b00718" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 7976–7997</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 31, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>2 May 2019</li><li><span class="item_label"><b>Published</b> online</span>31 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 September 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00718" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00718</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D7976%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DHarald%2BWeinstabl%252C%2BMatthias%2BTreu%252C%2BJoerg%2BRinnenthal%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D17%26contentID%3Dacs.jmedchem.9b00718%26title%3DIntracellular%2BTrapping%2Bof%2Bthe%2BSelective%2BPhosphoglycerate%2BDehydrogenase%2B%2528PHGDH%2529%2BInhibitor%2BBI-4924%2BDisrupts%2BSerine%2BBiosynthesis%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D7997%26publicationDate%3DSeptember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00718"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2265</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">19</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00718" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Intracellular Trapping of the Selective Phosphoglycerate Dehydrogenase (PHGDH) Inhibitor BI-4924 Disrupts Serine Biosynthesis&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Harald&quot;,&quot;last_name&quot;:&quot;Weinstabl&quot;},{&quot;first_name&quot;:&quot;Matthias&quot;,&quot;last_name&quot;:&quot;Treu&quot;},{&quot;first_name&quot;:&quot;Joerg&quot;,&quot;last_name&quot;:&quot;Rinnenthal&quot;},{&quot;first_name&quot;:&quot;Stephan&quot;,&quot;last_name&quot;:&quot;K. Zahn&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Ettmayer&quot;},{&quot;first_name&quot;:&quot;Gerd&quot;,&quot;last_name&quot;:&quot;Bader&quot;},{&quot;first_name&quot;:&quot;Georg&quot;,&quot;last_name&quot;:&quot;Dahmann&quot;},{&quot;first_name&quot;:&quot;Dirk&quot;,&quot;last_name&quot;:&quot;Kessler&quot;},{&quot;first_name&quot;:&quot;Klaus&quot;,&quot;last_name&quot;:&quot;Rumpel&quot;},{&quot;first_name&quot;:&quot;Nikolai&quot;,&quot;last_name&quot;:&quot;Mischerikow&quot;},{&quot;first_name&quot;:&quot;Fabio&quot;,&quot;last_name&quot;:&quot;Savarese&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Gerstberger&quot;},{&quot;first_name&quot;:&quot;Moriz&quot;,&quot;last_name&quot;:&quot;Mayer&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Zoephel&quot;},{&quot;first_name&quot;:&quot;Renate&quot;,&quot;last_name&quot;:&quot;Schnitzer&quot;},{&quot;first_name&quot;:&quot;Wolfgang&quot;,&quot;last_name&quot;:&quot;Sommergruber&quot;},{&quot;first_name&quot;:&quot;Paola&quot;,&quot;last_name&quot;:&quot;Martinelli&quot;},{&quot;first_name&quot;:&quot;Heribert&quot;,&quot;last_name&quot;:&quot;Arnhof&quot;},{&quot;first_name&quot;:&quot;Biljana&quot;,&quot;last_name&quot;:&quot;Peric-Simov&quot;},{&quot;first_name&quot;:&quot;Karin&quot;,&quot;last_name&quot;:&quot;S. Hofbauer&quot;},{&quot;first_name&quot;:&quot;Géraldine&quot;,&quot;last_name&quot;:&quot;Garavel&quot;},{&quot;first_name&quot;:&quot;Yvonne&quot;,&quot;last_name&quot;:&quot;Scherbantin&quot;},{&quot;first_name&quot;:&quot;Sophie&quot;,&quot;last_name&quot;:&quot;Mitzner&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;N. Fett&quot;},{&quot;first_name&quot;:&quot;Guido&quot;,&quot;last_name&quot;:&quot;Scholz&quot;},{&quot;first_name&quot;:&quot;Jens&quot;,&quot;last_name&quot;:&quot;Bruchhaus&quot;},{&quot;first_name&quot;:&quot;Michelle&quot;,&quot;last_name&quot;:&quot;Burkard&quot;},{&quot;first_name&quot;:&quot;Roland&quot;,&quot;last_name&quot;:&quot;Kousek&quot;},{&quot;first_name&quot;:&quot;Tuncay&quot;,&quot;last_name&quot;:&quot;Ciftci&quot;},{&quot;first_name&quot;:&quot;Bernadette&quot;,&quot;last_name&quot;:&quot;Sharps&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Schrenk&quot;},{&quot;first_name&quot;:&quot;Christoph&quot;,&quot;last_name&quot;:&quot;Harrer&quot;},{&quot;first_name&quot;:&quot;Daniela&quot;,&quot;last_name&quot;:&quot;Haering&quot;},{&quot;first_name&quot;:&quot;Bernhard&quot;,&quot;last_name&quot;:&quot;Wolkerstorfer&quot;},{&quot;first_name&quot;:&quot;Xuechun&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Xiaobing&quot;,&quot;last_name&quot;:&quot;Lv&quot;},{&quot;first_name&quot;:&quot;Alicia&quot;,&quot;last_name&quot;:&quot;Du&quot;},{&quot;first_name&quot;:&quot;Dongyang&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Yali&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jens&quot;,&quot;last_name&quot;:&quot;Quant&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;Pearson&quot;},{&quot;first_name&quot;:&quot;Darryl&quot;,&quot;last_name&quot;:&quot;B. McConnell&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;7976-7997&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00718&quot;},&quot;abstract&quot;:&quot;Phosphoglycerate dehydrogenase (PHGDH) is known to be the rate-limiting enzyme in the serine synthesis pathway in humans. It converts glycolysis-derived 3-phosphoglycerate to 3-phosphopyruvate in a co-factor-dependent oxidation reaction. Herein, we report the discovery of BI-4916, a prodrug of the co-factor nicotinamide adenine dinucleotide (NADH/NAD+)-competitive PHGDH inhibitor BI-4924, which has shown high selectivity against the majority of other dehydrogenase targets. Starting with a fragment-based screening, a subsequent hit optimization using structure-based drug design was conducted to deliver a single-digit nanomolar lead series and to improve potency by 6 orders of magnitude. To this end, an intracellular ester cleavage mechanism of the ester prodrug was utilized to achieve intracellular enrichment of the actual carboxylic acid based drug and thus overcome high cytosolic levels of the competitive cofactors NADH/NAD+.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00718&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00718" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00718&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00718" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00718&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00718" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00718&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00718&amp;href=/doi/10.1021/acs.jmedchem.9b00718" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00718" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00718" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00718%26sid%3Dliteratum%253Aachs%26pmid%3D31365252%26genre%3Darticle%26aulast%3DWeinstabl%26date%3D2019%26atitle%3DIntracellular%2BTrapping%2Bof%2Bthe%2BSelective%2BPhosphoglycerate%2BDehydrogenase%2B%2528PHGDH%2529%2BInhibitor%2BBI-4924%2BDisrupts%2BSerine%2BBiosynthesis%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D17%26spage%3D7976%26epage%3D7997%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (3)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/jmcmar.2019.62.issue-17/20190912/jmcmar.2019.62.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0029.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Phosphoglycerate dehydrogenase (PHGDH) is known to be the rate-limiting enzyme in the serine synthesis pathway in humans. It converts glycolysis-derived 3-phosphoglycerate to 3-phosphopyruvate in a co-factor-dependent oxidation reaction. Herein, we report the discovery of <b>BI-4916</b>, a prodrug of the co-factor nicotinamide adenine dinucleotide (NADH/NAD<sup>+</sup>)-competitive  PHGDH inhibitor <b>BI-4924</b>, which has shown high selectivity against the majority of other dehydrogenase targets. Starting with a fragment-based screening, a subsequent hit optimization using structure-based drug design was conducted to deliver a single-digit nanomolar lead series and to improve potency by 6 orders of magnitude. To this end, an intracellular ester cleavage mechanism of the ester prodrug was utilized to achieve intracellular enrichment of the actual carboxylic acid based drug and thus overcome high cytosolic levels of the competitive cofactors NADH/NAD<sup>+</sup>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38245" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38245" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cancer cells can rewire their metabolism to support fast growth and proliferation, and indeed the “deregulation of cellular energetics” has been included among the hallmarks of cancer.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Metabolic enzymes have therefore raised interest as potential targets for therapies aimed at “fasting” cancer cells.</div><div class="NLM_p">Phosphoglycerate dehydrogenase (PHGDH), an enzyme in the serine synthesis pathway (SSP),<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> has been found to be involved in numerous cancers such as breast cancer,<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> melanoma,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> pancreatic cancer,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> kidney cancer,<a onclick="showRef(event, 'ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref7 ref8">(7,8)</a> and nonsmall cell lung cancer,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> as well as in pulmonary fibrosis.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a><i>PHGDH</i>, the gene coding for phosphoglycerate dehydrogenase, was found to be focally amplified or gained in melanoma and breast cancer,<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> where it is broadly overexpressed in about 70% of triple-negative cases even in the absence of amplification. Cells with <i>PHGDH</i> amplification and overexpression were shown to be highly sensitive to its depletion. A subgroup of PHGDH-overexpressing tumors was also observed among nonsmall cell lung cancer and lung adenocarcinoma cases, and also in this case, PHGDH knockdown selectively impaired the growth of overexpressing cells, suggesting an oncogenic function of PHGDH.<a onclick="showRef(event, 'ref9 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref11">(9,11)</a></div><div class="NLM_p">In more detail, a negative-selection RNA interference screening using an orthotopic human breast cancer xenograft model was developed by Possemato et al. in 2011 identifying PHGDH as a gene required for in vivo tumorigenesis.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In addition, PHGDH was shown to be localized in a genomic region of recurrent copy number gain in breast cancer, particularly in estrogen receptor-negative or triple-negative breast cancers (TNBC).<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Similarly, PHGDH is recurrently amplified in a genomic region of focal copy number gain most commonly found in melanoma. Knockdown of PHGDH selectively inhibited the growth of melanoma cells that exhibit PHGDH amplification versus those that lack this amplification.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> PHGDH protein expression was recently correlated with advanced TNM stage, larger tumor size, and poor outcomes in colorectal cancer patients.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Studies utilizing a large panel of NSCLC cell lines revealed that NRF2 controls the expression of the PHGDH gene and that PHGDH confers poor prognosis in human NSCLC.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Recently, a strong antiproliferative effect was shown in a pancreatic xenograft model by inhibition of PHGDH with IoxA, a natural withanolide.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">PHGDH is the rate-limiting enzyme in the serine synthesis pathway in humans, which diverts 3-phosphoglycerate from glycolytic serine biosynthesis by oxidizing it to 3-phosphopyruvate in an reduced nicotinamide (NA) adenine dinucleotide (NADH)/nicotinamide adenine dinucleotide (NAD<sup>+</sup>)-dependent manner (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Subsequently, 3-phosphopyruvate is converted to serine by phosphoserine transaminase and phosphoserine hydrolase. Serine is regarded as a nonessential amino acid (AA); however, it is indispensable and plays a critical role in several cellular processes such as the conversion to glycine supporting the backbone (BB) of central carbon metabolism (carbon unit for purine synthesis),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> the generation of sphingosine via reaction with palmitoyl-CoA,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> the regulation of redox potential (production of NADPH),<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and as a precursor of other amino acids (e.g., cysteine).<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> It is apparent that cells with PHGDH amplification and/or high expression are not sensitive to serine depletion, indicating that the loss of proliferation upon knockdown of PHGDH is not simply due to reduction in serine levels per se.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Although a plethora of published data support the contribution of the serine synthesis pathway to tumorigenesis, in particular the role of the key enzyme PHGDH, only recently the precise molecular mechanism of PHGDH in oncogenesis has been characterized in detail. Inhibition of PHGDH alters the nucleotide metabolism in a way that is independent of serine utilization and redox maintenance. Mechanistically, it rather results in disruption of the mass balance within the central carbon metabolism, leading to alterations in both the pentose phosphate pathway (PPP) and the tricarboxylic acid (TCA) cycle.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Indeed, restoration of the TCA cycle and PPP through addition of ribose and cell-permeable α-ketoglutarate to PHGDH-inhibitor-treated tumor cells was sufficient to rescue the antiproliferative effect generated by PHGDH inhibition. In their study, Reid and colleagues clearly showed that the mass balance and the requirement of maintaining flux balance are sufficient to regulate the flux into other neighboring anabolic pathways such as glycolysis and the TCA cycle.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0001.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known roles of PHGDH in tumor metabolism and tumor epigenetics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The lack of serine-derived glutathione has been reported to underlie the increased sensitivity toward reactive oxygen species of tumor cells lacking PHGDH.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> PHGDH has been further demonstrated to be able to use α-ketoglutarate as a substrate to produce the oncometabolite <span class="smallcaps smallerCapital">d</span>-2-hydroxyglutarate (<span class="smallcaps smallerCapital">d</span>-2-HG), perhaps linking PHGDH activity with changes in the epigenetic program of amplified tumor cells.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, as the PHGDH enzymatic activity toward 2-hydroxyglutarate production is extremely low compared with that of 3-phosphohydroxypyruvate, it remains to be determined if those levels of 2-hydroxyglutarate produced by PHGDH do play a similar role in the epigenetic reprogramming as it was shown for the mutated isocitrate-dehydrogenase 1 and 2 in glioma and acute myeloid leukemia<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>).</div><div class="NLM_p">In preclinical models, breast and pancreatic adenocarcinomas have been shown to be dependent on the SSP, suggesting that PHGDH may represent a therapeutic intervention point.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Developing highly potent and specific PHGDH inhibitors will not only promote understanding of the molecular mechanisms triggering the dependency of certain tumor (sub)types on this pathway but also open a novel therapeutic opportunity to treat those tumors. This emphasizes the need for compounds exhibiting low nanomolar cellular potency and exquisite specificity to enable better understanding of the target in a cellular context.</div><div class="NLM_p">The challenge when targeting metabolic enzymes in a competitive fashion is intrinsically tied to competing with millimolar intracellular levels of the cofactors NAD<sup>+</sup> and NADH accompanied by the fact that modulation of metabolic targets often needs high levels of target occupancy to achieve efficacy.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Preceding in-house target validation experiments and data published by Chen et al. strengthen this hypothesis. The authors showed that >90% PHGDH knockdown over 14 days was antiproliferative in PHGDH high breast cancer cell lines, and in line with this observation, <90% knockdown in vivo was not associated with tumor growth inhibition—indeed, defining the level and duration of inhibition of a given target required for efficacy is essential to design appropriate drugs.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Furthermore, PHGDH exhibits a polar shallow cofactor/substrate binding pocket, which makes the druggability of PHGDH extremely challenging.</div><div class="NLM_p">Several compounds have been reported in the literature to inhibit PHGDH, which can be divided into two groups. Allosteric inhibitors, such as CBR-5884 (<b>2</b>),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> NCT-503 (<b>3</b>),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> “compound 34” (<b>5</b>), and PKUMDL-WQ-2201 (<b>7</b>), display a micromolar potency and are not dependent on intracellular cofactor levels.<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> On the other hand, orthosteric inhibitors, such as Astra Zeneca’s indole-2-carboxamide <b>1</b> and fragment <b>6</b>, are validated adenosine pocket binders, thus underlying the competitive binding of the natural cofactor NADH/NAD<sup>+</sup>.<a onclick="showRef(event, 'ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref29 ref30">(29,30)</a> Compound <b>4</b> discovered by RAZE pharmaceuticals is likely to share the same binding mode due to structural similarity.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">Herein, we report on the discovery of a prodrug-based NADH/NAD<sup>+</sup>-competitive PHGDH inhibitor with high selectivity (based on proteomics analysis of variance (ANOVA) analysis) against the majority of other dehydrogenase targets. An intracellular ester cleavage mechanism of the permeable ester prodrug <b>BI-4916</b> was utilized to achieve intracellular enrichment of the actual carboxylic acid based poorly permeable drug <b>BI-4924</b> and thus to overcome high intracellular levels of the competitive cofactor NADH/NAD<sup>+</sup>. Compounds <b>1</b> and <b>BI-4924</b> are X-ray-validated NAD<sup>+</sup>-competitive binders. Compounds <b>3</b> (NCT-503; thiourea), <b>5</b> (“compound 34”; thioamide), and <b>7</b> (thiosemicarbazide) exhibit potentially reactive pan assay interference compounds (PAINS) motives (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33">(25−33)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0002.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overview of selected published PHGDH inhibitors and reported binding/inhibition data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_26164" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_26164" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Screening</h3><div class="NLM_p">Screening was performed in two different hit-finding approaches (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). First, a high-throughput screen (HTS) was performed based on a pool consisting of >1 000 000 molecular entities. Within this approach, no boundaries were set, and allosteric as well as orthosteric binders were evaluated with a biochemical NAD<sup>+</sup>-dependent screening cascade. Second, a fragment-based screen (FBS) was conducted in parallel.</div><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> High-Throughput Screen (HTS)</h4><div class="NLM_p">HTS yielded 27 000 primary hits (2.2% hit rate). The confirmed hits (15 200; 56% of primary hits) were profiled in a shortcut assay containing no PHGDH, and dual hitters were removed as false positives (11 250 compounds remaining; 74% of confirmed hits). These hits were triaged to 4750 (42%) hits by removal of unwanted substructures (e.g., PAINS)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and compounds with molecular weight ≥600 and clog <i>P</i> ≥ 5.5. For these triaged hits, dose–response curves were measured in PHGDH, lactate dehydrogenase (subtype A; LDHA), and shortcut assays. Thus, 2041 potent and selective PHGDH inhibitors were progressed to cluster analysis. A total of 323 representative hits from nine classes and singletons of interest were prioritized. The purity of 72 out of 104 class representatives was confirmed by liquid chromatography/mass spectrometry (LC/MS) using freshly prepared dimethyl sulfoxide (DMSO) stock solutions. Forty-three out of these 72 were confirmed as selective PHGDH inhibitors (>10-fold selectivity toward LDHA) with an IC<sub>50</sub> < 100 μM. Due to the fact that many of these hits (triazoles, aryl-pyrazoles) have been synthesized employing transition-metal catalysis and that dehydrogenases often show a high sensitivity toward transition metals,<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> a selection of metal ions was tested in the biochemical assays. It was discovered that transition-metal ions (e.g., Hg, Cu, Pt, Pd) are potent inhibitors of PHGDH, and indeed, for 5 out of 12 selected HTS hits, Hg and Cu impurities were quantified (68 elements screened by inductively coupled plasma (ICP)-MS) in sufficient concentrations to explain the observed PHGDH inhibition. In line with this finding, it was not surprising that 7 out of 10 resynthesized and purified HTS hits lost their PHGDH inhibition.</div><div class="NLM_p last">As a consequence, STD-NMR with and without NAD<sup>+</sup> and SPR was applied for orthogonal hit validation. The latter was found to tolerate metal impurities (only Pd<sup>2+</sup> shows a measurable <i>K</i><sub>d</sub> of significance), and the paramagnetic nature of copper leads to line broadening in STD-NMR.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> As a next step, we went back to the 4750 confirmed actives and performed a cluster analysis on the selected 3230 (68%) hits for rescreening at 50 μM for specific binding to PHGDH in the SPR assay. From the identified 323 (10%) PHGDH binders, 77 had a <i>K</i><sub>d</sub> < 300 μM, which were again clustered according to their chemical structure. This orthogonal HTS hit validation strategy translated in >80% SPR <i>K</i><sub>d</sub> confirmation for the resynthesized cluster representatives and resulted in three chemical series with X-ray-derived binding mode information among which compound <b>8</b> (indole-2-carboxamide) is a representative example.</div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Fragment-Based Screening (FBS)</h4><div class="NLM_p">A truncated version of human PHGDH according to a previously published structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G76">2G76</a>, <a href="https://www.thesgc.org/structures/2g76" class="extLink">https://www.thesgc.org/structures/2g76</a>) was designed (A4-V315; see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>), expressed, and purified in gram quantities. In contrast to the full-length protein (tetramer), it was found to be active as a dimer in solution and was also used for the biochemical assay as well as crystallographic analysis. FBS aimed at finding NAD-orthosteric binders; thus, an NAD stripping protocol was developed for the preparation of PHGDH apo-protein. An in-house library of 1536 fragments was complemented by a customized library of 324 soluble fragments (>100 μM aqueous solubility) selected based on learnings from an LDHA-fragment-based screening campaign.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a></div><div class="NLM_p">A total of 1860 fragments were screened in quadruplex mixtures by STD-NMR (250 μM fragment and 20 μM apo-protein) to reduce the time requirements of the screening campaign. STD-NMR spectra for the individual hits from 10 mixtures with >40% STD signal and 59 mixtures with >20% STD signal were recorded with and without NAD (2 mM). Sixty hits (3.2% hit rate) were identified that bind preferentially to the apo-protein and in a less pronounced manner to the NAD-bound protein. SPR <i>K</i><sub>d</sub> values for PHGDH, lactate dehydrogenase, and carbonic anhydrase II were measured for these 60 compounds, resulting in the identification of 19 specific PHGDH binders (32% hit rate).</div><div class="NLM_p">The development of a low-temperature cocrystallization protocol allowed the X-ray validation of an adenine pocket binder <b>9</b> out of the phenylpyrazole-5-carboxamide series (2.15 Å resolution). This fragment shows a strong interaction with D174 and thus exhibits suitable vectors along the NAD<sup>+</sup>-binding site (<named-content content-type="anchor" rid="fig3" type="simple"></named-content><a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Schematic representation of the hit-finding process. Saturation transfer difference NMR (STD-NMR)-guided-based FBS screening on the right (blue) and luciferase-based HTS screening on the left (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Validated hit from the STD-NMR FBS screen: phenylpyrazole-5-carboxamide <b>9</b> cocrystallized with PHGDH showing the main polar interaction to D174 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH">6RIH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">All hits identified bind orthosterically to the lipophilic adenine pocket and form a H-bond to D174.</div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Structural Analysis of PHGDH in Complex with Its Natural Cofactor and Substrate</h3><div class="NLM_p">Cocrystallization of PHGDH with its natural substrate <b>3-PG</b> was performed to gain more insight into the catalytic mechanism of the enzyme and to enable the structure-based design of transition-state analogues. The binding mode of the cofactor NAD<sup>+</sup> was found to be identical to previously published structures of human PHGDH (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). The sequence of murine PHGDH shows a high degree of similarity (98.5%) as well as identity (94.6%). Briefly, the nicotinamide (NA) interacts with the backbones of C233 and A285 and the side chain of D259. The ribose of the nicotinamide part as well as the linking phosphates form numerous direct and water-mediated hydrogen bonds with the protein. The main directed interaction of the adenosine moiety is the hydrogen bonding of the ribose hydroxyl groups with the side chain of D174. In a recent publication, <span class="smallcaps smallerCapital">l</span>-tartrate, derived from the crystallization buffer, bound in one of the two active sites and was discussed as an analogue of the natural substrate.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In our structure (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a>), <b>3-PG</b> indeed binds to the same position as the <span class="smallcaps smallerCapital">l</span>-tartrate. The phosphate group forms extensive interactions with the side chains of R53, S54, and R74 of monomer A as well as R134 of monomer B. Also, the rest of the <b>3-PG</b> moiety is part of an extensive hydrogen-bonding network with the active site of PHGDH.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Electron density map of the ternary complex of cofactor NAD<sup>+</sup> and substrate <b>3-PG</b> in PHGDH (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Structure-Based Compound Optimization</h3><div class="NLM_p">Both hit classes—phenylpyrazole-5-carboxamides from the FBS and indole-2-carboxamides from the HTS (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>)—form a H-bond to the side chain of D174 and the amide NH of the ligand. This acts as a polar anchor to place a lipophilic aromatic motif in the corresponding hydrophobic hotspot, referred to as the adenine pocket. The phenylpyrazole and the indole motif occupy the same part of the adenine pocket with similar orientation of the corresponding <i>N</i>-methyl groups and the carboxamide NH setting the stage for structure–activity relationship (SAR) transfer between both hit series.</div><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0013.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Overview of the Hit Optimization<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Double-tracked strategy: merging learnings from HTS and FBS leads to <b>BI-4924</b> and <b>BI-4916</b>.</p></p></figure><div class="NLM_p">The medicinal chemistry challenge during hit optimization was the kinked shape of the binding pocket harboring both apolar (adenosine binding region) and polar (phosphate binding region) sites. Early synthesized analogues had the tendency to point out of the binding pocket toward the bulk water, thus showing no gain in affinity toward PHGDH. Due to more favorable physicochemical properties (low clog <i>P</i>, high solubility in all relevant assays) as well as the commercial availability of suitable starting materials, our initial efforts were focused on the phenylpyrazole-5-carboxamides series.</div><div class="NLM_p">A detailed conformational analysis of the crystal structure of the screening hit <b>9</b> (<i>K</i><sub>d</sub> = 105 μM) confirmed that the observed binding confirmation is in alignment with the minimum energy confirmation induced by the 1,3-allylic strain.<a onclick="showRef(event, 'ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref37 ref38">(37,38)</a> The remaining proton of the cyclopropyl ring is in planar alignment with the carbonyl group of the corresponding phenylpyrazole-5-carboxamide, thus locking the conformation of the cyclopropyl moiety. Therefore, commercially available 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (<b>10</b>) (<i>K</i><sub>d</sub> not measurable) was chosen for amide library synthesis because a clear eutomer/distomer and vectors into the phosphate channel were expected based on the key learnings from the X-ray analysis. Phenethylamides from the library such as <b>11</b> showed a potency of 22 μM in the biochemical assay, whereas the distomer <b>12</b> and the methylene analogue <b>13</b> were not measurable within the detection range of up to 100 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Effect of Chiral Methyl Group on Binding Affinity</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0014.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char=".">biochemical PHGDH (low NAD<sup>+</sup>) IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(<i>R</i>)-11</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>(</b><i><b>S</b></i><b>)-12</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">>100</td></tr></tbody></table></div></div><div class="NLM_p">To gain insight into the structure–activity relationship (SAR) of the phosphate channel of PHGDH, a carboxylic acid was installed in the para-position of the phenyl core, thus offering diversification opportunities. Installation of an acidic motif resulted in a dramatic boost in potency (<b>17</b>). Subsequently, these findings were independently confirmed by Frédérick et al. as well as Fuller et al.<a onclick="showRef(event, 'ref29 ref39'); return false;" href="javascript:void(0);" class="ref ref29 ref39">(29,39)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>)</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Influence of Allylic Strain on Binding Affinities</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0015.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R1</th><th class="colsep0 rowsep0" align="center">R2</th><th class="colsep0 rowsep0" align="center" char=".">biochemical PHGDH (low NAD<sup>+</sup>) IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">1.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char=".">0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char=".">5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i><b>rac-</b></i><b>17</b></td><td class="colsep0 rowsep0" align="left">Me/H</td><td class="colsep0 rowsep0" align="left">H/Me</td><td class="colsep0 rowsep0" align="char" char=".">0.9</td></tr></tbody></table></div></div><div class="NLM_p">Having now single-digit micromolar compounds in hand, it was decided to re-evaluate the contribution of the allylic strain to the binding affinity. The significant loss in potency observed with the geminal dimethyl analogue <b>14</b> illustrated the contribution of the 1,3-allylic strain to the overall binding affinity and is in good alignment with quantum-mechanical geometry optimization for hydrogen atoms and the chiral methyl group at the B3LYP/6-31G* level based on ligand conformation extracted from X-ray of <b>15</b> ((<i>R</i>)-pyrazolo-derivative). The R-isomer is favored by −4.35 kcal/mol over the S-isomer in the given conformation (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray structure confirmation of compound <b>15</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3">6RJ3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As previously shown by Fuller et al.,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> the introduction of an acidic functionality led to a >50-fold increase in potency. Crystallographic analysis in conjunction with the biochemical assay data revealed that moving the acidic moiety away from the para-position led to a significant decrease in potency. However, the para-position was also judged to be the most suitable one for further growing into the NA site. Thus, it was envisioned to establish appropriate replacements with additional vectors enabling building the molecule toward the substrate binding site. A summary of these efforts is shown in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Disappointingly, neither neutral nor basic motifs resulted in the desired improvement in potency.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Evaluation of the Long-Range Electrostatic (LRE) Interactions in the Phosphate Channel</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0016.gif" alt="" id="fx3" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0017.gif" alt="" id="fx4" /></img><div></div></div><div class="NLM_p">As no direct interactions between the carboxylate and the protein could be identified, long-range electrostatic interactions were seen to be the most plausible explanation for the significant improvement in potency.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> This hypothesis is further supported by the presence of multiple basic residues within a ≤10 Å radius around the binding site, leading to a highly positively charged protein environment. Furthermore, the introduction of an acidic acyl sulfonamide <b>39</b> maintained binding affinity, again without showing direct interactions with the protein. This indicated that the exact placement of a negatively charged group is of minor importance, but its presence is crucial (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0007.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Long-range electrostatics in the PHGDH phosphate channel exemplarily shown on acyl sulfonamide <b>39</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5">6RJ5</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Inversion of the acyl sulfonamide <b>39</b> led to the constitutional isomer <b>40</b>. X-ray analysis confirmed an altered binding mode, now showing an additional polar interaction with the backbone NH of I155 (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). This combination of shape and charge led to further improvement of potency to an IC<sub>50</sub> of 70 nM in the biochemical assay.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0008.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Addressing a new polar interaction on I155 BB-NH with kinked acyl sulfonamide <b>40</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2">6RJ2</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The difference in potency of acyl sulfonamide isomers <b>39</b> and <b>40</b> indicates that shape and negative charge under physiological conditions are both crucial. The introduction of an additional kink in acyl sulfonamide <b>40</b> leads to a favorable positioning of a H-bond acceptor enabling I155–backbone NH interaction. Introduction of 2-(benzenesulfonyl)acetic acids (tosyl acetate) resulted in potent and highly acidic PHGDH inhibitor <b>43</b> (calcd p<i>K</i><sub>a</sub> ∼ 2.5). The efforts of optimizing the interactions inside the phosphate channel by tweaking the p<i>K</i><sub>a</sub> of the acidic motif are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, suggesting that the p<i>K</i><sub>a</sub> of the acid is playing a crucial role.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Optimization of the Interactions inside the Phosphate Channel of PHGDH</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0019.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0020.gif" alt="" id="fx7" /></img><div></div></div><div class="NLM_p">As previously mentioned in the screening section, an HTS was performed in parallel, which identified indole-2-carboxamides as potent PHGDH inhibitors. Aniline amide <b>8</b> inhibits PHGDH with an SPR <i>K</i><sub>d</sub> of 144 μM. Applying the learnings of the 1,3-allylic strain and the necessity of a negative charge of the phenylpyrazole-5-carboxamide series to the indole-2-carboxamide series resulted in compound <b>44</b> with an IC<sub>50</sub> of ∼2 μM in the biochemical driving assay. Subsequent optimization of the lipophilic interactions in the adenine pocket is illustrated in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Finally, the dichlorodimethyl substitution pattern on the indole moiety was identified as an additional potency trigger. To be prepared for the assessment of even more potent compounds, the NAD<sup>+</sup> concentrations were increased from 19 μM (low NAD<sup>+</sup>) to 250 μM (high NAD<sup>+</sup>), which shifts the IC<sub>50</sub> values to 10-fold higher values (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>).</div><figure id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0009.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Assay adaption for the profiling of potent PHGDH inhibitors. Red dots depict the reference assay @19 μM NAD<sup>+</sup>. Brown dots depict another run under the same conditions. Yellow dots indicate shifted IC<sub>50</sub> values using 250 μM NAD<sup>+</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Optimization of the Indole Moiety in the Adenine Pocket</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0021.gif" alt="" id="fx8" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0022.gif" alt="" id="fx9" /></img><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> SAR Transfer to the HTS Derived Indole-2-carboxamide Series</h3><div class="NLM_p">Following identification of an optimized binder in the adenine pocket, an SAR transfer from the optimization efforts in the phosphate channel was envisioned. Subsequently, tosyl acetate <b>58</b> showed an improved potency of IC<sub>50</sub> = 4 nM in the adapted NAD<sup>+</sup> high assay and 2 nM in the NAD<sup>+</sup> low assay (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>).</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Efforts of Merging of an Optimized Adenine Pocket Binder with an Optimized Phosphate Channel Binder</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0024.gif" alt="" id="fx11" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0025.gif" alt="" id="fx12" /></img><div></div></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Efforts To Optimize the Interaction with D174</h3><div class="NLM_p">To even further improve the potency of <b>58</b>, it was envisioned that an additional hydrogen-bond donor (HBD) could have a chelating effect on the D174 residue of PHGDH. As the crystal structure of NAD<sup>+</sup>/<b>3-PG</b> shows, two hydroxyl groups of the ribose moiety are chelating D174 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a>). Thus, it was attempted to transfer the learnings from the optimization of the indole series—summarized in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>—to improve the potency of fully decorated tosyl acetate <b>58</b>. Chelation as a potency driver is well described in the literature and was also used by Fuller et al. for compound <b>1</b>.<a onclick="showRef(event, 'ref29 ref41'); return false;" href="javascript:void(0);" class="ref ref29 ref41">(29,41)</a> In line with these findings, hydroxymethylene bearing compound <b>BI-4924</b> showed an even further improved IC<sub>50</sub> of ≤3 nM, which again is believed to hit the assay wall. Poor cell membrane permeability and a high efflux ratio (as judged by the industry standard assay CaCo-2) were identified as the main reasons for the high gap between the nanomolar potency in the biochemical assay and the moderate intracellular biomarker (BM) modulation in the micromolar range. An X-ray analysis of the binding mode is depicted in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>.</div><figure id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0010.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Binding mode of <b>BI-4924</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6">6RJ6</a>); (B) chemical structure of <b>BI-4924</b>; (C) additional in vitro data for <b>BI-4924</b>; m, murine; r, rat; h, human. CaCo-2 permeability: ab, apical–basolateral permeation/transport; ba, basolateral–apical permeation/transport; FCS, fetal calf serum.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As further elaborated in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>, neither the additional acidification of the hydroxyl group nor changing the nature of the HBD resulted in improved binding affinities.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. Optimization of the Bidentate D174 Interaction</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0026.gif" alt="" id="fx13" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0027.gif" alt="" id="fx14" /></img><div></div></div><div class="NLM_p last">The compounds were then evaluated for cellular activity. Effects of the compounds on the production of [<sup>13</sup>C]-serine, upon feeding of uniformly labeled [<sup>13</sup>C]-glucose, were measured by mass spectroscopy (for a detailed protocol, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>). Given the low p<i>K</i><sub>a</sub> of the acidic moiety, it was not surprising that the permeability in CaCo-2 cells of <b>BI-4924</b> is low (0.2 × 10<sup>–6</sup> cm/s). This leads to a significant offset between the biochemical IC<sub>50</sub> values and IC<sub>50</sub> values measured for the cellular biomarker modulation (3 nM biochemical IC<sub>50</sub> vs 2.2 μM IC<sub>50</sub> in serine modulation).</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Selectivity Profiling</h3><div class="NLM_p">To gain a better understanding of the binding selectivity of our PHGDH inhibitors, we synthesized two derivatives of <b>BI-4924</b>, compound <b>70</b> and compound <b>71</b>, each bearing a primary amine attached via a spacer to two distinct parts of the molecule. These PHGDH binders were immobilized on <i>N</i>-hydroxysuccinimide (NHS)-activated Sepharose via amide formation and incubated with MDA-MB-468 cell extracts in the presence or absence of 10 μM <b>BI-4924</b>. Identification and quantitative comparison of the Sepharose-bound protein fraction by trypsin digestion and LC–tandem mass spectrometry (MS/MS) analysis revealed a clean selectivity profile for <b>BI-4924</b>. Of the 31 dehydrogenases in the data set, only alcohol dehydrogenase class-3 and 3-hydroxyisobutyrate dehydrogenase were identified as offtargets (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>).</div><figure id="fig11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0011.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Selectivity profile of <b>BI-4924</b> obtained by quantitative chemical proteomics. Proteins (orange dots, dehydrogenases; blue dots, other proteins) from lysed MDA-MB-468 cell are displaced from affinity matrices functionalized with compounds <b>70</b> or <b>71</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>) by competition with 10 μM <b>BI-4924</b>. Data are plotted log 2 of the fold change vs DMSO control against −log 10 of the <i>P</i> value per protein (ANOVA, <i>N</i> = 3). All <i>t</i>-tests-tests performed were two-tailed. <i>t</i>-tests assuming equal variances.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Cellular Uptake</h3><div class="NLM_p">To overcome the inherent low permeability of the acid <b>BI-4924</b> and compensate for high intracellular NADH/NAD<sup>+</sup> levels (∼0.3 mM), we hypothesized that intracellular ester cleavage of a permeable ester containing the PHGDH prodrug would liberate the corresponding active carboxylic acid and, due to its poor permeability, lead to an accumulation of the acid inside cells. Thus, cellular uptake studies were performed in MDA-MB-468 cells to better understand these findings. After defined intervals, following treatment with ethyl ester <b>BI-4916</b>, MDA-MB-468 cells were harvested, and the concentrations of both the ester and the carboxylic acid were determined in the medium, as well as intracellularly. An analogous experiment was performed with the addition of <b>BI-4924</b> to the medium. Thus, it could be shown that <b>BI-4924</b> is up to 140-fold enriched inside the cells in a time-dependent manner after addition of <b>BI-4916</b> versus <b>BI-4924</b> to the medium (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>). This accumulation can be explained by differences of permeability between the prodrug and the acid and efficient conversion of prodrug into acid inside the cells. Simultaneously, the prodrug compound also gets cleaved in the medium, resulting again in an impermeable acid negligibly contributing to the intracellular drug levels (see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>). This medium’s instability is in good agreement with the ester’s stability in plasma protein containing media: 10% FCS <i>t</i><sub>(1/2)</sub> = 276 min; mouse serum <i>t</i><sub>(1/2)</sub> = 135 min. Finally, after complete conversion of the prodrug, the compound concentrations between the cell and the medium equilibrate, and the accumulation factor approaches unity after approximately 72 h. Using an area under the curve-based approach, an average accumulation factor of 23.4 over a time period of 72 h can be calculated (gray dashed line, <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>).</div><figure id="fig12" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0012.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Ratio of mean cellular concentrations of acid <b>BI-4924</b> (black diamonds) at various time points after the addition of 3 μM ester <b>BI-4916</b> vs 3 μM acid <b>BI-4924</b> into the medium. Error bars indicate the standard Gaussian error calculated from the standard deviations of the individual assays. The gray dashed line indicates the average accumulation factor for a time period of 72 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Biomarker Modulation</h3><div class="NLM_p last">As a next step, we envisioned to measure the cellular enrichment in a pathway-relevant manner to establish an in vitro pharmacokinetic/pharmacodynamic relationship. Therefore, a cellular target engagement study in disease-relevant MDA-MB-468 TNBC-cells was set up using uniformly labeled <sup>13</sup>C glucose as a substrate. Being metabolized according to <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, the MS-based quantification of <sup>13</sup>C serine served as readout. Indeed, treatment of MDA-MB-468 cells with ethyl ester <b>BI-4916</b> resulted in a ∼70-fold improved transient modulation of <sup>13</sup>C-derived serine (measured “IC<sub>50</sub>” = 29 <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="2846656819">[email protected]</a> h; IC<sub>50</sub> = 2000 <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="98f6d5d8afaa">[email protected]</a> h) compared with the application of <b>BI-4924</b>. This is in good agreement with the above-mentioned time-dependent enrichment factor and shows that CaCo-2 efflux does not seem to be predictive for MDA-MB-468 cells.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27088" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27088" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Herein, we report the discovery of the highly potent and selective PHGDH inhibitor <b>BI-4924</b> and its ester prodrug <b>BI-4916</b>. Based on the combined efforts of an HTS campaign as well as a fragment-based finding campaign, a ∼100 μM screening hit was optimized to single-digit nanomolar potency by means of a structure-based drug design. In a nutshell, compound <b>BI-4924</b> gains its binding affinity by a strong lipophilic interaction in the adenine pocket, accompanied by a bidentate polar anchor interaction of a secondary amide and a hydroxyl group with D174. Following the kink introduced by the 1,3-allylic strain, a tosyl acetate finally places the required negatively charged acidic moiety in the right position to interact with a backbone NH of I155 and the surrounding arginine as well as histidine side chains by means of LRE (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>A). Compound <b>BI-4924</b> is a lipophilic, highly plasma protein bound PHGDH inhibitor with excellent microsomal as well as hepatocytic stability (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>C). The low p<i>K</i><sub>a</sub> of the compound leads to excellent aqueous solubility over a broad pH range. Furthermore, <b>BI-4924</b> exhibits an excellent selectivity profile as shown by chemical proteomics (2/31 dehydrogenases) as well as in a CEREP counterscreen panel (4/40 offtargets >50% inhibition @ 10 μM; see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>).</div><div class="NLM_p last">The interconversion of <b>BI-4924</b> to its ester prodrug <b>BI-4916</b> increased cell penetration and thus led to a transient accumulation of <b>BI-4924</b> inside the cells, thereby generating an in vivo-like intracellular concentration–time profile for <b>BI-4924</b> in vitro. In agreement with a transient intracellular compound accumulation, a time-dependent double-digit nanomolar modulation of the <sup>13</sup>C serine flux could be observed. The intrinsic instability in the presence of esterases disqualified <b>BI-4916</b> as an in vivo tool. However, we are convinced that both <b>BI-4924</b> and <b>BI-4916</b> will serve as valuable in vitro tools to further explore yet unknown aspects of PHGDH inhibition.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47560" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47560" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Biochemical Assays</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> HTS PHGDH Inhibitor Assay</h4><div class="NLM_p last">To 100 nL of test compounds (10 mM in DMSO) in white Greiner low-volume plates is added 5 μL of enzyme working solution (200 mM Tris–HCl, pH 7.4/1 mM ethylenediaminetetraacetic acid (EDTA)/2 μM NAD/0.02% Tween20). The buffer is supplemented with 400 ng/mL PHGDH (construct corresponding to PBD id <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G76">2G76</a>; amino acids 4–315, N-terminal HIS-tag, TEV cleavage site; plasmid from Structural Genomics Consortium (SGC)) (prior to the assay). Each assay plate contains 16 wells with NAD alone as a high control. Next, 1 μL of AuCl<sub>3</sub> (50 mM in DMSO) is added to column 24 as a low control. Then, 5 μL of substrate solution (Promega bioluminescent NADH detection kit (#V865A)) supplemented with Promega substrate 1 (proluciferin), promega enzyme 1 (diaphorase), and 3-phosphoglycerate (280 mM stock) in the ratio (parts) 200:1:1:5 is added to each well, and the plates are incubated for 2 h at room temperature (RT). Measurement is done with EnVision (luminescence measurement, optimized for the shortest reading time per well without performance loss). Calculation: Signal for unknown compounds was normalized using high and low control wells as follows: %activity = ((RFU(sample) – RFU(low))/((RFU(high) – RFU(low))) × 100. An inhibitor of the PHGDH enzyme will give values lower than 100%CTL.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Diaphorase NADH Counterscreen (“Shortcut Assay”)</h4><div class="NLM_p last">To 100 nL of test compounds (10 mM in DMSO) in white Greiner low-volume plates is added 5 μL of assay buffer solution (200 mM Tris–HCl, pH 7.4/1 mM EDTA/1 μ M NADH/0.02% Tween20). Each assay plate contains 16 wells with the assay buffer solution alone as a high control. First, 1 μL of AuCl<sub>3</sub> (50 mM in DMSO) is added to column 24 as a low control. Next, 5 μL of substrate solution (Promega bioluminescent NADH detection kit (#V865A)) supplemented with Promega substrate 1 (proluciferin), promega enzyme 1 (diaphorase), and 3-phosphoglycerate (280 mM stock) in the ratio (parts) 200:1:1:5 is added to each well, and the plates are then incubated for 2 h at RT. Measurement is done with EnVision (luminescence measurement, optimized for the shortest reading time per well without performance loss). Calculation: Signal for unknown compounds was normalized using high and low control wells as follows: %activity = ((RFU(sample) – RFU(low))/((RFU(high) – RFU(low))) × 100. An inhibitor of the diaphorase/proluciferin/luciferase will give values lower than 100%CTL.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 3-Phosphoglycerate Dehydrogenase (PHGDH) Fluorescence Intensity Assay</h4><div class="NLM_p last">This assay is used to identify compounds that inhibit the enzymatic activity of PHGDH, which catalyzes the reaction of 3-phosphoglycerate (<b>3-PG</b>) and NAD to 3-phosphohydroxypyruvate and NADH. The produced NADH is used in a coupled reaction for diaphorase-mediated reduction of resazurin to resorufin, which can be measured in fluorescence intensity readout. The full-length version of the PHGDH enzyme was expressed in <i>Escherichia coli</i> with an N-terminal HIS-tag and a TEV cleavage site. The 3-phosphoglycerate substrate was purchased from Sigma. NAD, diaphorase, and resazurin were purchased from Sigma-Aldrich. Compounds are dispensed onto assay plates (black, low-volume, flat-bottom, 384-well, Corning) using an Access Labcyte Workstation with Labcyte Echo 55× from a DMSO compound stock solution. For the highest assay concentration of 100 μM, 150 nL of compound solution is transferred from a 10 mM DMSO compound stock solution. A series of 11 concentrations is transferred for each compound at which each concentration is 5-fold lower than the previous one. DMSO is added such that every well has a total of 150 nL compound solution. The assay has been performed at two different NAD/<b>3-PG</b> ratios (final assay concentrations): NAD low: 18.8 μM NAD/175 μM <b>3-PG</b>; NAD high: 250 μM NAD/500 μM <b>3-PG</b>. To the 150 nL of transferred compound stock solution is added 5 μL of PHGDH protein (final assay concentration 100 ng/mL) in the assay buffer (125 mM Tris–HCl, pH 7.5; 56.25 mM hydrazine sulfate pH 9.0; 2.5 mM EDTA; assay-specific NAD concentration; 0.0125% Tween20). Then, 10 μL of a mix containing assay-specific <b>3-PG</b> concentration, resazurin (25 μM final assay concentration), and diaphorase (35 μg/mL final assay concentration) is added. Plates are kept at room temperature. After 240 min incubation time, the fluorescence signal is measured in a PerkinElmer Envision HTS Multilabel Reader with an excitation wavelength at 530–560 nm and an emission wavelength at 590 nm. Each plate contains negative controls (diluted DMSO instead of the test compound; reaction as described with the PHGDH protein) and positive controls (diluted DMSO instead of the test compound; reaction as described with a buffer instead of the PHGDH protein). Negative and positive control values are used for normalization. An internal standard was used, and IC50 values are calculated and analyzed using a four-parametric logistic model.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Cellular Assays</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <sup>13</sup>C Serine Assay in MDA-MB-468 Cells</h4><div class="NLM_p last">MDA-MB-468 cells (5000/well) in medium (Dulbecco’s modified Eagle’s medium (DMEM) glucose free (Gibco # A14430-01) with 10% FCS (Gibco # 26140), Na-pyruvate (Gibco # 11360), and Glutamax I (Gibco # 35050), 90 μL/well) were seeded into a 96-well plate and incubated at 37 °C in a CO<sub>2</sub> incubator overnight. Test compounds were serially diluted (eight dilution steps (1:3)) in medium in duplicates, and 10 μL of this dilution was added to the cell culture (final DMSO concentration 1%). After an incubation at 37 °C in a CO<sub>2</sub> incubator for 1, 24, and 72 h, 20 mM <sup>13</sup>C glucose (Aldrich # 389374) in medium (50 μL) was added per well. After 180 min incubation at 37 °C in a CO<sub>2</sub> incubator, 150 μL of methanol (20% H<sub>2</sub>O, −80 °C) was added, and the plates were sealed immediately and frozen at −80 °C. On the day of measurement, plates were thawed and centrifuged and the supernatant was evaporated. The samples were resuspended in 100 μL of water for tandem mass spectrometry. <sup>13</sup>C serine levels were analyzed with LC–MS/MS using multiple reaction monitoring (MRM). Data analysis: detected peak areas with MRM transition 107.090/75.80 Da were integrated using Analyst software. IC<sub>50</sub> values were calculated from these values using a four-parametric logistic model.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Uptake Assay</h4><div class="NLM_p last">Before starting the incubation, 200 000 cells per well were seeded onto six-well plates and incubated overnight at 37 °C in a 3 mL incubation medium (DMEM (Lonza # 12-604F); 10% FCS (HyClone FBS Thermo Fisher #SH3007103)). Then, 9 μL of a 1 mM DMSO stock was added to the incubation, yielding a 3 μM concentration of the test compound in the supernatant at the beginning of the incubation. The experiment was stopped at various time points after the compound was added: 0 min, 10 min, 30 min, 60 min, 150 min, 7 h, 24 h, and 72 h. At each time point, the supernatant was taken and frozen to −80 °C. Cells were washed 3× with 1 mL of ice-cold phosphate-buffered saline buffer (Ca/Mg free Gibco # 14200-067) for 1 min each. While the first two washes were discarded, the third wash was taken and frozen to −80 °C to check the efficiency of the washing procedure. Cells were lysed with 500 μL of 80:20 acetonitrile/water and frozen to −80 °C. All samples were quantified via high-performance liquid chromatography (HPLC)–MS/MS using calibration curves for the ester and the corresponding acid for absolute quantification of time-dependent intracellular and extracellular concentrations. A 12-point calibration curve was generated with 50% acetonitrile/water. Samples were prepared accordingly. An internal standard was used for exact quantification. Samples were measured on a SCIEX QTRAP 4000 system and analyzed using Analyst software.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> (Bio)Physical Methods</h3><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> PHGDH SPR Assay</h4><div class="NLM_p last">Immobilization of the target protein: PHGDH (construct corresponding to PDB id <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G76">2G76</a>; amino acids 4–315, N-terminal HIS-tag, TEV cleavage site; plasmid from SGC) is immobilized on a CM5 chip (GE). The surface is activated with 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide (EDC)/NHS with a contact time of 540 s. The protein is diluted in NaAc (pH 5.0) + 0.5 mM tris(2-carboxyethyl)phosphine hydrochloride (TCEP). A protein immobilization level of 1000–6000 response units is achieved. For immobilization, we use 10 mM HBS-EP+ pH 7.4 (GE) + 0.5 mM TCEP as a running buffer. This process is performed at 20 °C. Interaction experiments: All interaction experiments are performed at 20 °C in tris(hydroxymethyl)aminomethane 50 mM, NaCl 150 mM, TCEP 0.5 mM, EDTA 1 mM, Tween20 0.02%, DMSO 1–5% (v/v), pH 7.4. The compounds are diluted in the running buffer and injected at a flow rate of 30 μL/min over the immobilized target protein. Data analysis: sensorgrams from reference surfaces and blank injections are subtracted from the raw data prior to data analysis, using Biacore T200 evaluation software. Sensorgrams recorded at different compound concentrations in multicycle experiments are fitted using a 1:1 interaction model, with a term for mass-transport included.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> LDH SPR Assay</h4><div class="NLM_p last">Immobilization of the target protein: lactate dehydrogenase chicken heart, recombinant from <i>E. coli</i> (Sigma-Aldrich, CAS No. 9001-60-9, order number 59747), is immobilized on a CM5 chip (GE). The surface is activated with EDC/NHS with a contact time of 540 s. The protein is diluted in NaAc (pH 5.5). A protein immobilization level of 1000–6000 response units is achieved. For immobilization, we use 10 mM HBS-EP+ pH 7.4 (GE) + 0.5 mM TCEP as a running buffer. This process is performed at 20 °C. Interaction experiments and data analysis are done as described for the PHGDH SPR assay.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> CAII SPR Assay</h4><div class="NLM_p last">Immobilization of the target protein: carbonic anhydrase II (Sigma-Aldrich) is immobilized on a CM5 chip (GE). The surface is activated with EDC/NHS with a contact time of 540 s. The protein is diluted in NaAc (pH 5.0) + 0.5 mM TCEP. A protein immobilization level of 1000–6000 response units is achieved. For immobilization, we use 10 mM HBS-EP+ pH 7.4 (GE) + 0.5 mM TCEP as a running buffer. This process is performed at 20 °C. Interaction experiments and data analysis are done as described for the PHGDH SPR assay.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Metal Salts: ICP-MS Method</h4><div class="NLM_p last">This method involves metal impurity determination from DMSO stock solutions. This method is designed to quantify metal impurities that are included in DMSO stock solutions as obtained from the respective dispensaries. No effort is made to bring insoluble metal components into solution, as we are only interested in soluble components that potentially interfere with biophysical or biochemical assays. The intended final compound concentration is approx. 0.2 μM, which is obtained as follows: 25 μL of 10 mM DMSO stock solution (or equivalent) is ordered and diluted with 235 μL of DMSO, 65 μL of internal standard solution (in most cases indium), and 975 μL of a freshly prepared 2% HNO<sub>3</sub> solution, yielding a total of 1300 μL that can be used for analysis. If the compound precipitates, the solution is filtered through a 0.45 μm cellulose filter. Data is evaluated with Mass Hunter software, and quantitation of the individual metals is obtained from calibration curves obtained on the same day with indium or gallium as the internal control standard. The threshold of quantitation for each element depends on how well it can be detected by ICP-MS and its natural occurrence as a ubiquitous contaminant in our setup. This value is represented in background equivalent concentrations and was determined for our specific setup for each metal reported below. The metal contaminants are reported in mol percent relative to the compound, if they exceed the thresholds shown below; otherwise, they are reported as <0.<i>XX</i> mol %. Mg < 0.40; Al < 0.10; Mn < 0.01; Fe < 0.01; Co < 0.01; Ni < 0.01; Cu < 0.02; Zn < 0.02; Ru < 0.01; Rh < 0.01; Pd < 0.01; Ag < 0.01; Cd < 0.01; Sn < 0.01; Cs < 0.01; Au < 0.01.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> NMR</h4><div class="NLM_p last">NMR experiments were recorded on Bruker Avance HD 400 and 500 MHz spectrometers equipped with BBO Prodigy and TCI CryoProbes at 298 K, respectively. Samples were dissolved in 600 μL of DMSO-<i>d</i><sub>6</sub>, and tetramethylsilane was added as an internal standard. One-dimensional (1D) <sup>1</sup>H spectra were acquired with 30° excitation pulses and an interpulse delay of 4.2 s with 64k data points and 20 ppm sweep width. 1D <sup>13</sup>C spectra were acquired with broadband composite pulse decoupling (WALTZ16) and an interpulse delay of 3.3 s with 64k data points and a sweep width of 240 ppm. Processing and analysis of 1D spectra were performed with Bruker TopSpin 3.2 software. No zero filling was performed, and spectra were manually integrated after automatic baseline correction. Chemical shifts are reported in ppm on the δ scale. Heteronuclear single-quantum coherence spectra were recorded on all samples to aid the interpretation of the data and to identify signals hidden underneath solvent peaks. Spectra were acquired with sweep widths obtained by automatic sweep width detection from 1D reference spectra in the direct dimension with 1k data points and with 210 ppm and 256 data points in the indirect dimension.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Protein Science</h3><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> PHGDH Expression</h4><div class="NLM_p last">A vector coding for PHGDH (A4-V315) with an N-terminal 6× His-tag and a TEV cleavage site was obtained from the Structural Genomics Consortium (SGC). Recombinant PHGDH was expressed in <i>E. coli</i> BL21(DE3) at 18 °C overnight in a 2× YT medium (Sigma).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> PHGDH Purification</h4><div class="NLM_p last">Cell pellets were resuspended in 50 mM NaH<sub>2</sub>PO<sub>4</sub>, 0.3 M NaCl, 10 mM imidazole, and 0.5 mM TCEP containing the protease inhibitor cocktail (COMPLETE without EDTA, Merck) according to the manufacturer’s specifications. Cells were broken by sonication at 4 °C. A 50% polyethyleneimine solution was added to a final concentration of 0.15% (v/v) and stirred for 30 min on ice followed by centrifugation at 26 000<i>g</i> for 45 min at 4 °C. The supernatant was applied to a NiNTA column (HisTrap, GE Healthcare) in buffer A (50 mM NaH<sub>2</sub>PO<sub>4</sub>, 0.3 M NaCl, 10 mM imidazole, 0.5 mM TCEP), washed with buffer A containing 50 mM imidazole, and eluted with a gradient to buffer A containing 0.25 M imidazole. Fractions containing PHGDH were pooled, and the His-tag was cleaved off by incubation with TEV protease at 4 °C overnight. PHGDH was further purified by gel filtration (HiLoad Superdex S75, GE Healthcare) equilibrated in 10 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (HEPES), 500 mM NaCl, 5% glycerol, 0.5 mM TCEP, pH 7.5.</div></div><div id="sec4_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Removal of NAD<sup>+</sup></h4><div class="NLM_p last">Activated charcoal powder (extrapure, Merck) was washed twice with water followed by ethanol and finally with 0.1 M HEPES buffer pH 7.5. The first two washing steps were performed in a small spin column (BioRad) to remove fines. A small amount (>2× the estimated mass of PHGDH in solution) of washed charcoal was added to the PHGDH solution. The suspension was put on a spinning wheel and rotated overnight at 4 °C. Charcoal was removed by centrifugation and/or filtration. Removal of NAD<sup>+</sup> was monitored at 322 and 260 nm. PHGDH was finally concentrated to ∼15 mg/mL in a centrifugal concentrator (BioMax) using a 10k cutoff membrane and stored at −80 °C.</div></div><div id="sec4_4_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Chemical Proteomics</h4><div class="NLM_p last">Cell pellets were lysed in 20 mM HEPES pH 7.5, 150 mM KCl, 1% Triton X-100 and cleared at 30 000<i>g</i>. The affinity matrices with compounds 70 and 71 were prepared by amide formation with magnetic NHS Sepharose (GE Healthcare) in DMSO/tetraethylammonium to a ligand density of 10%. Cell extract and 200 nL of affinity matrix were incubated in the presence of 10 μM <b>BI-4924</b> or DMSO. After 2 h incubation, the matrices were washed, and bound proteins were proteolytically eluted using trypsin (Pierce) in the presence of 5 mM TCEP and 10 mM 2-chloroacetamide. Peptides were purified by solid-phase extraction (Sep-Pak tC18, Waters) and analyzed on a nanoflow-LC system (Thermo Fisher) hyphenated to a Q-Exactive Plus mass spectrometer (Thermo Fisher). Spectra were assigned using Mascot (Matrix Science), and label-free quantification was performed using Progenesis QI (Waters).</div></div><div id="sec4_4_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> Protein Crystallization</h4><div class="NLM_p last">Protein crystallization was done with the hanging drop method by incubating PHDGH (12 mg/mL in 25 mM HEPES pH 7.5, 300 mM NaCl, 0.5 mM TCEP) with the ligand (100 mM DMSO stock solution, final concentration 1 mM) for 1 h. Next, 1 μL of the protein solution was mixed with 1 μL of the reservoir solution (25% poly(ethylene glycol) 8000, 100 mM 2-morpholinoethanesulfonate pH 6.6, 200 mM lithium sulfate, 4 mM dithiothreitol) at 4 °C. Crystals were frozen in liquid nitrogen, and data were collected at the SLS beam line X06SA (Swiss Light Source, Paul Scherrer Institute) at a wavelength of 1 Å using the PILATUS 6M detector. The crystals belonged to space group <i>P</i>2<sub>1</sub> and contained two monomers per asymmetric unit. Images were processed with autoPROC.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The resolution limits were set using default autoPROC settings. The structures were solved by molecular replacement using the PHGDH structure 2G76 as a search model. Subsequent model building and refinement were done using standard protocols using CCP4,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> COOT,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and autoBUSTER.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Statistics for data collection and refinement can be found in tables in the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>. Crystallographic work for the <b>3-PG</b> structure was done by Beryllium Discovery (Bedford, MA). The coordinates and structure factors have been deposited at the Protein Data Bank with the accession codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a> (<b>3-PG</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6">6RJ6</a> (<b>BI-4924</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3">6RJ3</a> (<b>15</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2">6RJ2</a> (<b>40</b>), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5">6RJ5</a> (<b>39</b>), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH">6RIH</a> (<b>9</b>).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Syntheses of Compounds</h3><div class="NLM_p last">General Section: All samples showed a purity of ≥95%. HPLC traces of selected compounds are available as <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>. Samples were analyzed on an Agilent 1200 series LC system coupled with an Agilent 6140 mass spectrometer. Purity was determined via UV detection with a bandwidth of 170 nm in the range from 230 to 400 nm. LC parameters were as follows: Waters Xbridge C18 column, 2.5 μm particle size, 2.1 × 20 mm<sup>2</sup> or 3.5 μm particle size, 2.1 × 30 mm<sup>2</sup>, run time 2.1 min, flow 1 mL/min, column temperature 60 °C, and 5 μL injections. Solvent A (20 mM NH<sub>4</sub>HCO<sub>3</sub>/NH<sub>3</sub>; pH 9), solvent B (MS grade acetonitrile). Start 10% B, gradient 10–95% B from 0.0 to 1.5 min, 95% B from 1.5 to 2.0 min, gradient 95–10% B from 2.0 to 2.1 min.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Synthetic Procedures</h3><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 3-[(1-Methyl-1<i>H</i>-indole-2-carbonyl)-amino]-benzoic Acid Methyl Ester (<b>Int-01</b>)</h4><div id="sec4_6_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> General Method A</h5><div class="NLM_p last">To 1-methylindole-2-carboxylic acid (50 mg, 0.28 mmol) and 1-[bis(dimethylamino)methylene]-1<i>H</i>-1,2,3-triazolo[4,5-b]pyridinium 3-oxid hexafluorophosphate (HATU) (163 mg, 0.42 mmol) in dry DMSO (300 μL), <i>i</i>Pr<sub>2</sub>NEt (135 μL, 0.84 mmol) was added and stirred for 5 min at room temperature. Methyl 3-aminobenzoate (51 mg, 0.34 mmol) was added and stirred for 1.5 h. The reaction mixture was partitioned between water and EtOAc, and the organic layer was washed with water and saturated aqueous NaCl solution. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo to give crude <b>Int-01</b> in 78 mg (90%) yield, which was used in the next step without further purification.</div></div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 3-[(1-Methyl-1<i>H</i>-indole-2-carbonyl)-amino]-benzoic Acid (<b>8</b>)</h4><div id="sec4_6_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> General Method B</h5><div class="NLM_p last">3-[(1-Methyl-1<i>H</i>-indole-2-carbonyl)-amino]-benzoic acid methyl ester (<b>Int-01</b>) (77 mg, 0.25 mmol) in MeOH (0.5 mL) was treated with 8 N aqueous NaOH (0.2 mL) and stirred for 1 h at 80 °C. The mixture was concentrated in vacuo, and the residue was dissolved in DMSO. The reaction mixture was filtered and purified by prep. HPLC (MeCN/water) to give <b>8</b> as a white solid in 54 mg (73%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.48 (s, 1H), 8.44 (s, 1H), 8.01 (br d, <i>J</i> = 8.20 Hz, 1H), 7.69 (s, 2H), 7.58 (d, <i>J</i> = 8.51 Hz, 1H), 7.47 (t, <i>J</i> = 7.88 Hz, 1H), 7.37 (s, 1H), 7.32 (t, <i>J</i> = 7.57 Hz, 1H), 7.14 (t, <i>J</i> = 7.41 Hz, 1H), 4.02 (s, 3H), COOH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.4, 160.7, 139.2, 138.8, 131.8, 131.8, 128.9, 125.5, 124.4, 124.1, 124.1, 121.9, 121.0, 120.4, 110.7, 105.9, 31.5. High-resolution mass spectrometry (HRMS) (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>2</sub>O<sub>3</sub>, 295.10772; found, 295.10797. HPLC <i>R</i><sub>t</sub> = 0.82.</div></div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid Cyclopropylamide (<b>9</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (600 mg, 2.97 mmol) in 630 mg yield (88%). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.52 (br s, 1H), 7.74 (br d, <i>J</i> = 7.10 Hz, 2H), 7.42 (t, <i>J</i> = 7.10 Hz, 2H), 7.32 (br t, <i>J</i> = 7.10 Hz, 1H), 7.20 (br s, 1H), 4.09 (br s, 3H), 2.82 (br s, 1H), 0.65–0.80 (m, 2H), 0.58 (br s, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.4, 148.1, 136.8, 132.6, 128.8, 127.8, 124.9, 104.1, 39.9, 22.6, 5.7. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>14</sub>H<sub>15</sub>N<sub>3</sub>O, 242.12879; found, 242.12900. HPLC <i>R</i><sub>t</sub> = 1.07.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid ((<i>R</i>)-1-Phenyl-ethyl)-amide (<b>11</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (25 mg, 0.12 mmol) in 32 mg (84%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.87 (br d, <i>J</i> = 7.88 Hz, 1H), 7.78 (br d, <i>J</i> = 7.25 Hz, 2H), 7.37–7.49 (m, 5H), 7.28–7.37 (m, 3H), 7.20–7.28 (m, 1H), 5.14 (quin, <i>J</i> = 7.01 Hz, 1H), 4.06 (s, 3H), 1.48 (br d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.5, 148.1, 144.4, 136.8, 132.6, 128.8, 128.3, 127.8, 126.8, 126.1, 124.9, 104.3, 48.1, 39.0, 22.2. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>N<sub>3</sub>O, 306.16009; found, 306.16084. HPLC <i>R</i><sub>t</sub> = 1.33.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid ((<i>S</i>)-1-Phenyl-ethyl)-amide (<b>12</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (25 mg, 0.12 mmol) in 31 mg (82%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.88 (br d, <i>J</i> = 7.88 Hz, 1H), 7.77 (dd, <i>J</i> = 1.26, 8.20 Hz, 2H), 7.44 (br s, 2H), 7.37–7.47 (m, 1H), 7.35–7.38 (m, 1H), 7.35–7.38 (m, 2H), 7.29–7.36 (m, 2H), 7.24 (t, <i>J</i> = 7.30 Hz, 1H), 5.13 (quin, <i>J</i> = 7.25 Hz, 1H), 4.06 (s, 3H), 1.48 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.5, 148.2, 144.5, 136.9, 132.6, 128.9, 128.4, 127.9, 126.8, 126.1, 125.0, 104.4, 48.2, 39.0, 22.2. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>N<sub>3</sub>O, 306.16009; found, 306.16077. HPLC <i>R</i><sub>t</sub> = 1.33.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> <i>N</i>-Benzyl-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>13</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (70 mg, 0.35 mmol) in 67 mg (66%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.09 (br t, <i>J</i> = 5.99 Hz, 1H), 7.75 (d, <i>J</i> = 7.25 Hz, 2H), 7.43 (t, <i>J</i> = 7.25 Hz, 2H), 7.29–7.37 (m, 6H), 7.25–7.28 (m, 1H), 4.47 (d, <i>J</i> = 5.99 Hz, 2H), 4.11 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 159.3, 148.2, 139.1, 136.8, 132.6, 128.9, 128.4, 127.8, 127.3, 126.9, 124.9, 104.2, 42.2, 39.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>N<sub>3</sub>O, 292.14444; found, 292.14438. HPLC <i>R</i><sub>t</sub> = 1.27.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 4-{2-[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]propan-2-yl}benzoic Acid (<b>14</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (66 mg). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.56–8.63 (m, 1H), 7.86 (d, <i>J</i> = 8.36 Hz, 2H), 7.80 (d, <i>J</i> = 7.35 Hz, 2H), 7.47 (d, <i>J</i> = 7.35 Hz, 2H), 7.40–7.45 (m, 3H), 7.33 (t, <i>J</i> = 7.35 Hz, 1H), 3.96 (s, 3H), 1.68 (s, 6H), acid OH not visible. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 168.0, 158.8, 151.3, 148.0, 137.3, 132.7, 131.6, 129.1, 128.8, 127.7, 124.9, 124.4, 104.6, 55.6, 38.9, 29.3. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>21</sub>N<sub>3</sub>O<sub>3</sub>, 364.16557; found, 364.16591. HPLC <i>R</i><sub>t</sub> = 0.91.</div></div><div id="sec4_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 4-[(1<i>R</i>)-1-[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl]benzoic Acid (<b>15</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (138 mg). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.65–13.07 (m, 1H), 8.98 (d, <i>J</i> = 7.86 Hz, 1H), 7.92 (d, <i>J</i> = 8.11 Hz, 2H), 7.78 (d, <i>J</i> = 7.35 Hz, 2H), 7.52 (d, <i>J</i> = 8.11 Hz, 2H), 7.39–7.48 (m, 3H), 7.33 (t, <i>J</i> = 7.35 Hz, 1H), 5.18 (quin, <i>J</i> = 7.22 Hz, 1H), 4.06 (s, 3H), 1.50 (d, <i>J</i> = 7.10 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.1, 158.6, 149.5, 148.1, 136.6, 132.6, 129.5, 129.3, 128.8, 127.8, 126.2, 124.9, 104.4, 48.1, 38.9, 21.9. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>, 350.14992; found, 350.15039. HPLC <i>R</i><sub>t</sub> = 0.85.</div></div><div id="sec4_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 4-{[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]methyl}benzoic Acid (<b>16</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (8 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.73 (s, 1H), 9.18 (t, <i>J</i> = 5.99 Hz, 1H), 7.92 (d, <i>J</i> = 8.51 Hz, 2H), 7.76 (dd, <i>J</i> = 1.10, 8.35 Hz, 2H), 7.38–7.48 (m, 4H), 7.28–7.36 (m, 2H), 4.53 (d, <i>J</i> = 5.99 Hz, 2H), 4.11 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 159.4, 148.3, 144.3, 136.6, 132.5, 129.5, 129.4, 128.9, 127.9, 127.3, 124.9, 104.2, 42.0, 39.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>3</sub>O<sub>3</sub>, 336.13427; found, 336.13482. HPLC <i>R</i><sub>t</sub> = 0.78.</div></div><div id="sec4_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 4-{1-[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl}benzoic Acid (<b>17</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (14 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.07–13.55 (m, 1H), 8.96 (d, <i>J</i> = 7.88 Hz, 1H), 7.92 (d, <i>J</i> = 8.20 Hz, 2H), 7.78 (d, <i>J</i> = 7.88 Hz, 2H), 7.51 (d, <i>J</i> = 8.20 Hz, 2H), 7.39–7.47 (m, 3H), 7.33 (t, <i>J</i> = 7.88 Hz, 1H), 5.10–5.30 (m, 1H), 4.05 (s, 3H), 1.49 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 158.6, 149.5, 148.2, 136.7, 132.6, 129.5, 129.3, 128.9, 127.8, 126.2, 124.9, 104.4, 48.2, 39.0, 21.9. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>, 350.14992; found, 350.15028. HPLC <i>R</i><sub>t</sub> = 0.86.</div></div><div id="sec4_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 1-Methyl-3-phenyl-<i>N</i>-[1-(4-sulfamoylphenyl)ethyl]-1<i>H</i>-pyrazole-5-carboxamide (<b>18</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (55 mg, 0.27 mmol) in 105 mg (82%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.00 (d, <i>J</i> = 7.57 Hz, 1H), 7.74–7.82 (m, 4H), 7.58 (d, <i>J</i> = 8.20 Hz, 2H), 7.41–7.47 (m, 3H), 7.28–7.36 (m, 3H), 5.16 (quin, <i>J</i> = 7.17 Hz, 1H), 4.05 (s, 3H), 1.50 (d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.6, 148.6, 148.1, 142.6, 136.6, 132.6, 128.9, 127.8, 126.5, 125.8, 124.9, 104.5, 48.2, 39.0, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O<sub>3</sub>S, 385.13289; found, 385.13405. HPLC <i>R</i><sub>t</sub> = 1.08.</div></div><div id="sec4_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid [(<i>R</i>)-1-(4-Nitro-phenyl)-ethyl]-amide (<b>19</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (1.00 g, 4.87 mmol) in 1.60 g (94%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.04 (d, <i>J</i> = 7.57 Hz, 1H), 8.23 (d, <i>J</i> = 8.70 Hz, 2H), 7.79 (d, <i>J</i> = 7.25 Hz, 2H), 7.68 (d, <i>J</i> = 8.70 Hz, 2H), 7.42–7.48 (m, 3H), 7.34 (t, <i>J</i> = 7.30 Hz, 1H), 5.23 (quin, <i>J</i> = 7.17 Hz, 1H), 4.06 (s, 3H), 1.52 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.7, 152.4, 148.1, 146.4, 136.5, 132.5, 128.8, 127.8, 127.3, 124.9, 123.6, 104.5, 48.2, 39.4, 21.8. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>, 351.14517; found, 351.14532. HPLC <i>R</i><sub>t</sub> = 1.41.</div></div><div id="sec4_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid [(<i>R</i>)-1-(4-Amino-phenyl)-ethyl]-amide (<b>20</b>)</h4><div class="NLM_p last">2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [(<i>R</i>)-1-(4-nitro-phenyl)-ethyl]-amide (<b>19</b>) (600 mg, 1.71 mmol), 10% Pd/C (91 mg, 0.09 mmol), and ammonium formate (756 mg, 12 mmol) in MeOH (6 mL) were stirred at room temperature for 2 h. The mixture was filtered and concentrated in vacuo. The residue was partitioned between water and CH<sub>2</sub>Cl<sub>2</sub>, and the organic layer was washed with water and saturated aqueous NaCl solution. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo to give <b>20</b> in 480 mg (87%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.65 (d, <i>J</i> = 8.51 Hz, 1H), 7.77 (d, <i>J</i> = 7.30 Hz, 2H), 7.43 (t, <i>J</i> = 7.30 Hz, 2H), 7.36 (s, 1H), 7.32 (t, <i>J</i> = 7.30 Hz, 1H), 7.05 (d, <i>J</i> = 8.50 Hz, 2H), 6.53 (d, <i>J</i> = 8.51 Hz, 2H), 4.89–5.26 (m, 3H), 4.08 (s, 3H), 1.42 (q, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.2, 148.0, 147.5, 137.0, 132.6, 131.1, 128.8, 127.7, 126.8, 124.9, 113.6, 104.1, 47.4, 39.4, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O, 321.17099; found, 321.17099. HPLC <i>R</i><sub>t</sub> = 1.21.</div></div><div id="sec4_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-[(1<i>R</i>)-1-(4-Iodophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-02</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (1.58 g, 7.70 mmol) in 2.66 g (80%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.90 (br d, <i>J</i> = 7.88 Hz, 1H), 7.78 (d, <i>J</i> = 7.88 Hz, 2H), 7.71 (d, <i>J</i> = 8.20 Hz, 2H), 7.44 (t, <i>J</i> = 7.80 Hz, 2H), 7.39 (s, 1H), 7.33 (t, <i>J</i> = 7.80 Hz, 1H), 7.22 (d, <i>J</i> = 8.20 Hz, 2H), 5.08 (quin, <i>J</i> = 7.17 Hz, 1H), 4.06 (s, 3H), 1.46 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.6, 148.2, 144.4, 137.1, 136.7, 132.6, 128.9, 128.6, 127.8, 124.9, 104.4, 92.5, 47.9, 21.9, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>IN<sub>3</sub>O, 432.05674; found, 432.05716. HPLC <i>R</i><sub>t</sub> = 1.48.</div></div><div id="sec4_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid {(<i>R</i>)-1-[4-(2-Oxo-pyrrolidin-1-yl)-phenyl]-ethyl}-amide (<b>21</b>)</h4><div id="sec4_6_15_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> General Method C</h5><div class="NLM_p last"><i>N</i>-[(1<i>R</i>)-1-(4-Iodophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-02</b>) (80 mg, 0.19 mmol), 2-pyrrolidone (12 μL, 0.15 mmol), K<sub>2</sub>CO<sub>3</sub> (52 mg, 0.37 mmol), <i>N</i>,<i>N</i>′-dimethylethylenediamine (20 μL, 0.19 mmol), and CuI (11 mg, 0.06 mmol) in dry DMF (500 μL) were stirred for 48 h at 90 °C. The mixture was filtered and concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give <b>21</b> in 32 mg (44%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.86 (d, <i>J</i> = 7.88 Hz, 1H), 7.77 (d, <i>J</i> = 7.50 Hz, 2H), 7.59 (d, <i>J</i> = 8.83 Hz, 2H), 7.43 (t, <i>J</i> = 7.50 Hz, 2H), 7.36–7.41 (m, 3H), 7.32 (t, <i>J</i> = 7.50 Hz, 1H), 5.11 (quin, <i>J</i> = 7.17 Hz, 1H), 4.06 (s, 3H), 3.81 (t, <i>J</i> = 7.50 Hz, 2H), 2.47 (t, <i>J</i> = 7.90 Hz, 2H), 2.05 (quin, <i>J</i> = 7.57 Hz, 2H), 1.47 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.7, 158.5, 148.1, 139.9, 138.3, 136.8, 132.6, 128.9, 127.8, 126.3, 124.9, 119.5, 104.3, 48.1, 47.8, 39.0, 32.3, 22.1, 17.5. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>24</sub>N<sub>4</sub>O<sub>2</sub>, 389.19720; found, 389.19693. HPLC <i>R</i><sub>t</sub> = 1.20.</div></div><div id="sec4_6_15_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> <i>N</i>-[1-(4-Methanesulfonylphenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>22</b>)</h5><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (50 mg, 0.24 mmol) in 24 mg (26%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.01 (d, <i>J</i> = 7.57 Hz, 1H), 7.90 (d, <i>J</i> = 8.30 Hz, 2H), 7.78 (d, <i>J</i> = 7.25 Hz, 2H), 7.66 (d, <i>J</i> = 8.30 Hz, 2H), 7.39–7.48 (m, 3H), 7.33 (t, <i>J</i> = 7.25 Hz, 1H), 5.20 (quin, <i>J</i> = 7.17 Hz, 1H), 4.05 (s, 3H), 3.20 (s, 3H), 1.51 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.7, 150.6, 148.2, 139.3, 136.6, 132.6, 128.9, 127.9, 127.2, 127.0, 124.9, 104.5, 48.3, 43.6, 39.0, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O<sub>3</sub>S, 384.13764; found, 384.13841. HPLC <i>R</i><sub>t</sub> = 1.18.</div></div><div id="sec4_6_15_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> <i>N</i>-[(1<i>R</i>)-1-(4-Bromophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-03</b>)</h5><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (400 mg, 1.95 mmol) in 658 mg (88%) yield and used in the next steps without further purification.</div></div></div><div id="sec4_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> <i>N</i>-[(1<i>R</i>)-1-{4-[(3<i>R</i>,4<i>S</i>)-3,4-Dihydroxypyrrolidin-1-yl]phenyl}ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>23</b>)</h4></div><div id="sec4_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> General Method D</h4><div class="NLM_p last">To <i>N</i>-[(1<i>R</i>)-1-(4-bromophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-03</b>) (50 mg, 0.13 mmol), sodium <i>tert</i>-butoxide (50 mg, 0.52 mmol), 2-(di-<i>tert</i>-butylphosphino)biphenyl (16 mg, 0.05 mmol), and tris(dibenzylideneacetone)dipalladium(0) (12 mg, 0.01 mmol) in dry dioxane (3 mL), <i>cis</i>-pyrrolidine-3,4-diol hydrochloride (27 mg, 0.20 mmol) was added under argon and stirred for 16 h at 50 °C. The mixture was filtered and concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give <b>23</b> in 12 mg (23%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69 (d, <i>J</i> = 8.20 Hz, 1H), 7.76 (d, <i>J</i> = 7.70 Hz, 2H), 7.43 (br t, <i>J</i> = 7.70 Hz, 2H), 7.35 (s, 1H), 7.32 (t, <i>J</i> = 7.70 Hz, 1H), 7.18 (d, <i>J</i> = 8.51 Hz, 2H), 6.43 (d, <i>J</i> = 8.83 Hz, 2H), 5.03 (quin, <i>J</i> = 7.25 Hz, 1H), 4.87 (d, <i>J</i> = 4.41 Hz, 2H), 4.08–4.15 (m, 2H), 4.06 (s, 3H), 3.00–3.11 (m, 2H), 1.43 (d, <i>J</i> = 6.94 Hz, 3H), OH protons not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.2, 148.1, 146.7, 137.0, 132.6, 130.7, 128.8, 127.8, 127.0, 124.9, 110.8, 104.2, 70.6, 52.5, 47.5, 39.0, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>26</sub>N<sub>4</sub>O<sub>3</sub>, 407.20777; found, 407.20825. HPLC <i>R</i><sub>t</sub> = 1.13.</div></div><div id="sec4_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> <i>N</i>-[(1<i>R</i>)-1-{4-[(3<i>S</i>)-3-Hydroxypyrrolidin-1-yl]phenyl}ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>24</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method D from <b>Int-03</b> (200 mg, 0.52 mmol) in 130 mg (64%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69 (d, <i>J</i> = 8.20 Hz, 1H), 7.76 (d, <i>J</i> = 7.25 Hz, 2H), 7.43 (t, <i>J</i> = 7.72 Hz, 2H), 7.35 (s, 1H), 7.32 (t, <i>J</i> = 7.70 Hz, 1H), 7.19 (d, <i>J</i> = 8.51 Hz, 2H), 6.47 (d, <i>J</i> = 8.51 Hz, 2H), 5.04 (quin, <i>J</i> = 7.25 Hz, 1H), 4.33–4.42 (m, 1H), 4.06 (s, 3H), 3.15–3.31 (m, 2H), 3.03 (dd, <i>J</i> = 1.58, 10.09 Hz, 1H), 1.96–2.07 (m, 1H), 1.82–1.91 (m, 1H), 1.44 (d, <i>J</i> = 7.25 Hz, 3H), 1H under H<sub>2</sub>O signal, COOH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.2, 148.1, 146.8, 137.0, 132.6, 130.7, 128.8, 127.8, 126.9, 124.9, 111.2, 104.2, 69.3, 56.1, 47.5, 45.6, 39.0, 33.8, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>26</sub>N<sub>4</sub>O<sub>2</sub>, 391.21285; found, 391.21347. HPLC <i>R</i><sub>t</sub> = 1.22.</div></div><div id="sec4_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 1-Methyl-3-phenyl-<i>N</i>-[(1<i>R</i>)-1-[4-(pyrrolidin-1-yl)phenyl]ethyl]-1<i>H</i>-pyrazole-5-carboxamide (<b>25</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method D from <b>Int-03</b> (200 mg, 0.52 mmol) in 120 mg (62%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69 (d, <i>J</i> = 8.20 Hz, 1H), 7.76 (d, <i>J</i> = 7.88 Hz, 2H), 7.43 (br t, <i>J</i> = 7.88 Hz, 2H), 7.36 (s, 1H), 7.32 (t, <i>J</i> = 7.30 Hz, 1H), 7.19 (d, <i>J</i> = 8.51 Hz, 2H), 6.50 (d, <i>J</i> = 8.51 Hz, 2H), 5.04 (quin, <i>J</i> = 7.25 Hz, 1H), 4.07 (s, 3H), 3.12–3.24 (m, 4H), 1.87–2.02 (m, 4H), 1.44 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.2, 148.0, 146.8, 137.0, 132.6, 130.7, 128.8, 127.7, 126.9, 124.9, 111.4, 104.1, 47.5, 47.4, 24.9, 22.1, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd C<sub>23</sub>H<sub>26</sub>N<sub>4</sub>O, 375.21794; found, 375.21823. HPLC <i>R</i><sub>t</sub> = 1.52.</div></div><div id="sec4_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> <i>N</i>-{1-[4-(1,1-Dioxo-1λ<sup>6</sup>,2-thiazolidin-2-yl)phenyl]ethyl}-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-04</b>)</h4><div class="NLM_p">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (350 mg, 1.64 mmol) in 643 mg (92%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.86 (br d, <i>J</i> = 7.86 Hz, 1H), 7.77 (br d, <i>J</i> = 7.35 Hz, 2H), 7.36–7.48 (m, 5H), 7.33 (br t, <i>J</i> = 7.60 Hz, 1H), 7.18 (br d, <i>J</i> = 7.35 Hz, 2H), 5.11 (br t, <i>J</i> = 6.84 Hz, 1H), 4.06 (br s, 3H), 3.72 (br s, 2H), 3.49 (br t, <i>J</i> = 7.22 Hz, 2H), 2.39 (br t, <i>J</i> = 6.59 Hz, 2H), 1.47 (br d, <i>J</i> = 6.34 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.5, 148.1, 139.7, 136.8, 132.6, 128.8, 127.8, 127.0, 124.9, 118.6, 104.3, 48.2, 47.7, 46.6, 38.9, 22.1, 18.5. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub>S, 425.16419; found, 425.16431. HPLC <i>R</i><sub>t</sub> = 1.21.</div><div class="NLM_p last">Chiral separation of this material yielded (<i>R</i>)-<i>N</i>-{1-[4-(1,1-dioxo-1λ<sup>6</sup>,2-thiazolidin-2-yl)phenyl]ethyl}-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>26</b>) in 30 mg (38%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.86 (br d, <i>J</i> = 8.20 Hz, 1H), 7.78 (d, <i>J</i> = 7.57 Hz, 2H), 7.38–7.47 (m, 5H), 7.33 (t, <i>J</i> = 7.20 Hz, 1H), 7.19 (d, <i>J</i> = 8.51 Hz, 2H), 5.12 (quin, <i>J</i> = 7.17 Hz, 1H), 4.07 (s, 3H), 3.73 (t, <i>J</i> = 6.46 Hz, 2H), 3.50 (t, <i>J</i> = 7.41 Hz, 2H), 2.40 (td, <i>J</i> = 6.98, 14.11 Hz, 2H), 1.48 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.4, 148.1, 139.6, 136.8, 132.6, 128.8, 127.8, 126.9, 124.9, 118.5, 104.3, 48.1, 47.6, 46.6, 39.4, 22.0, 18.5, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub>S, 425.16419; found, 425.16432. HPLC <i>R</i><sub>t</sub> = 1.29.</div></div><div id="sec4_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-[(1<i>R</i>)-1-[4-(2,5-Dioxoimidazolidin-1-yl)phenyl]ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>27</b>)</h4><div class="NLM_p last">To 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [(<i>R</i>)-1-(4-amino-phenyl)-ethyl]-amide (<b>20</b>) (50 mg, 0.16 mmol) and triethyl amine (32 μL, 0.23 mmol) in dry DMF (500 μL), ethyl isocyanate (30 mg, 0.23 mmol) was added and stirred for 16 h at room temperature. Then, sodium ethoxide (21% in EtOH, 29 μL, 0.078 mmol) was added and stirred for additional 2 h. The mixture was filtered and concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give <b>27</b> in 11 mg (17%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.93 (d, <i>J</i> = 7.88 Hz, 1H), 8.28 (s, 1H), 7.78 (d, <i>J</i> = 7.57 Hz, 2H), 7.49 (d, <i>J</i> = 8.20 Hz, 2H), 7.40–7.46 (m, 2H), 7.39 (br s, 1H), 7.27–7.35 (m, 3H), 5.16 (quin, <i>J</i> = 7.09 Hz, 1H), 4.00–4.13 (m, 5H), 1.51 (d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.1, 158.5, 156.5, 148.1, 144.0, 136.7, 132.6, 130.8, 128.8, 127.8, 126.7, 126.4, 124.9, 104.4, 47.9, 46.0, 39.0, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>21</sub>N<sub>5</sub>O<sub>3</sub>, 404.17172; found, 404.17183. HPLC <i>R</i><sub>t</sub> = 1.08.</div></div><div id="sec4_6_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> (2<i>S</i>)-1-{4-[(1<i>R</i>)-1-[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl]phenyl}-5-oxopyrrolidine-2-carboxylic Acid (<b>28</b>)</h4><div class="NLM_p last">(<i>S</i>)-1-(4-{(<i>R</i>)-1-[(2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carbonyl)-amino]-ethyl}-phenyl)-5-oxo-pyrrolidine-2-carboxylic acid ethyl ester was prepared from <i>N</i>-[(1<i>R</i>)-1-(4-iodophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-02</b>) (200 mg, 0.46 mmol) according to General Method C in 40 mg (19%) yield. The crude material (28 mg, 0.06 mmol) was saponified according to General Method B to give (2<i>S</i>)-1-{4-[(1<i>R</i>)-1-[(1-methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl]phenyl}-5-oxopyrrolidine-2-carboxylic acid (<b>28</b>) in 20 mg (76%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.01–13.31 (m, 1H), 8.86 (d, <i>J</i> = 7.86 Hz, 1H), 7.78 (d, <i>J</i> = 8.00 Hz, 2H), 7.37–7.50 (m, 7H), 7.33 (t, <i>J</i> = 7.00 Hz, 1H), 5.12 (quin, <i>J</i> = 7.22 Hz, 1H), 4.76–4.89 (m, 1H), 4.06 (s, 3H), 2.00–2.13 (m, 1H), 1.47 (d, <i>J</i> = 7.10 Hz, 3H), 3H under DMSO signal. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 173.3, 172.7, 157.8, 147.5, 139.9, 136.7, 136.2, 132.0, 128.2, 127.1, 125.8, 124.3, 120.1, 103.7, 60.0, 47.0, 30.0, 21.9, 21.4, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>24</sub>H<sub>24</sub>N<sub>4</sub>O<sub>4</sub>, 433.18703; found, 433.18706. HPLC <i>R</i><sub>t</sub> = 0.84.</div></div><div id="sec4_6_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid [(<i>R</i>)-1-(4-Guanidino-phenyl)-ethyl]-amide (<b>29</b>)</h4><div class="NLM_p last">2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [(<i>R</i>)-1-(4-amino-phenyl)-ethyl]-amide <b>(20</b>) (50 mg, 0.16 mmol) and cyanamide (8 mg, 0.19 mmol) in dioxane (500 μL) were treated with 4 N HCl in dioxane (58 μL) and stirred for 1 h at 120 °C in a microwave reactor. The mixture was concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give <b>29</b> in 12 mg (21%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (d, <i>J</i> = 7.88 Hz, 1H), 7.81–8.07 (m, 4H), 7.78 (d, <i>J</i> = 7.25 Hz, 2H), 7.37–7.48 (m, 5H), 7.33 (t, <i>J</i> = 7.30 Hz, 1H), 7.16 (d, <i>J</i> = 8.51 Hz, 2H), 5.14 (quin, <i>J</i> = 7.25 Hz, 1H), 4.06 (s, 3H), 1.48 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.7, 158.4, 156.1, 148.1, 141.8, 136.8, 132.6, 128.9, 127.8, 127.2, 124.9, 123.8, 104.3, 47.8, 39.0, 22.2. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>6</sub>O, 363.19279; found, 363.19270. HPLC <i>R</i><sub>t</sub> = 0.95.</div></div><div id="sec4_6_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 1-Methyl-<i>N</i>-{1-[4-(1-methylcarbamimidamido)phenyl]ethyl}-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>30</b>)</h4><div class="NLM_p last">1-Methyl-4-phenyl-1<i>H</i>-pyrazole-3-carboxylic acid (3.00 g, 14.9 mmol) and [4-(1-amino-ethyl)-phenyl]-methyl-carbamic acid <i>tert</i>-butyl ester (4.46 g, 17.8 mmol) were reacted according to General Method A to give methyl-(4-{1-[(2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carbonyl)-amino]-ethyl}-phenyl)-carbamic acid <i>tert</i>-butyl ester in 2.60 g (40%) yield. The crude material (2.00 g, 4.61 mmol) in dry dioxane (20 mL) was treated with HCl (4 N in dioxane, 20 mL) to give 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [1-(4-methylamino-phenyl)-ethyl]-amide hydrochloride in a quantitative yield upon evaporation. The crude material was reacted with cyanamide as described for the synthesis of <b>29</b> to give the target compound <b>30</b> in 76 mg (14%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.88 (s, 1H), 7.78 (d, <i>J</i> = 7.25 Hz, 2H), 7.38–7.48 (m, 5H), 7.33 (t, <i>J</i> = 7.30 Hz, 1H), 7.26 (d, <i>J</i> = 8.20 Hz, 2H), 5.17 (q, <i>J</i> = 6.94 Hz, 1H), 4.07 (s, 3H), 3.17 (s, 3H), 1.49 (d, <i>J</i> = 7.25 Hz, 3H), NH<sub>2</sub> and NH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 159.1, 158.4, 148.1, 142.7, 142.0, 136.7, 132.6, 128.9, 127.8, 127.3, 126.3, 124.9, 104.3, 47.7, 38.9, 22.2, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>24</sub>N<sub>6</sub>O, 377.20844; found, 377.20804. HPLC <i>R</i><sub>t</sub> = 1.12.</div></div><div id="sec4_6_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 1-Methyl-<i>N</i>-{1-[4-(<i>N</i>-methylmethanesulfonamido)phenyl]ethyl}-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>31</b>)</h4><div class="NLM_p last">Crude 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [1-(4-methylamino-phenyl)-ethyl]-amide hydrochloride (200 mg, 0.60 mmol) from the above synthesis in dry CH<sub>2</sub>Cl<sub>2</sub> (2 mL) was treated with methanesulfonyl chloride (200 μL, 1.21 mmol) and triethyl amine (120 μL, 0.91 mmol) and stirred for 12 h at room temperature. The mixture was concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give <b>31</b> in 165 mg (67%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.91 (d, <i>J</i> = 7.86 Hz, 1H), 7.78 (d, <i>J</i> = 7.20 Hz, 2H), 7.39–7.47 (m, 5H), 7.34–7.39 (m, 2H), 7.29–7.35 (m, 1H), 5.15 (quin, <i>J</i> = 7.22 Hz, 1H), 4.06 (s, 3H), 3.21 (s, 3H), 2.93 (s, 3H), 1.48 (d, <i>J</i> = 6.84 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.5, 148.2, 143.3, 140.3, 136.7, 132.6, 128.9, 127.8, 126.8, 126.4, 124.9, 104.4, 47.8, 38.9, 37.9, 35.0, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub>S, 413.16419; found, 413.16382. HPLC <i>R</i><sub>t</sub> = 1.29.</div></div><div id="sec4_6_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid [(<i>R</i>)-1-(4-Methanesulfonylamino-phenyl)-ethyl]-amide (<b>32</b>)</h4><div class="NLM_p last">2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [(<i>R</i>)-1-(4-amino-phenyl)-ethyl]-amide (<b>20</b>) (50 mg, 0.16 mmol) and Et<sub>3</sub>N (25 μL, 0.17 mmol) in dry CH<sub>2</sub>Cl<sub>2</sub> (0.5 mL) were treated with methanesulfonyl chloride (13 μL, 0.17 mmol) and stirred at room temperature overnight. The mixture was concentrated in vacuo. The crude product was purified by prep. HPLC (MeCN/water) to give <b>32</b> in 24 mg (39%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.68 (s, 1H), 8.84 (d, <i>J</i> = 7.88 Hz, 1H), 7.78 (d, <i>J</i> = 7.25 Hz, 2H), 7.44 (t, <i>J</i> = 7.25 Hz, 2H), 7.40 (s, 1H), 7.37 (d, <i>J</i> = 8.51 Hz, 2H), 7.33 (t, <i>J</i> = 7.25 Hz, 1H), 7.18 (d, <i>J</i> = 8.51 Hz, 2H), 5.11 (quin, <i>J</i> = 7.17 Hz, 1H), 4.07 (s, 3H), 2.97 (s, 3H), 1.47 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.4, 148.1, 139.9, 137.0, 136.8, 132.6, 128.8, 127.8, 127.0, 124.9, 120.0, 104.3, 47.6, 22.0, 2C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>4</sub>O<sub>3</sub>S, 399.14854; found, 399.14872. HPLC <i>R</i><sub>t</sub> = 1.22.</div></div><div id="sec4_6_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N</i>-[(1<i>R</i>)-1-(4-Methoxyphenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>33</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (50 mg, 0.24 mmol) in 13 mg (16%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.79 (br d, <i>J</i> = 8.20 Hz, 1H), 7.77 (d, <i>J</i> = 8.20 Hz, 2H), 7.43 (t, <i>J</i> = 7.70 Hz, 2H), 7.37 (s, 1H), 7.29–7.35 (m, 3H), 6.90 (d, <i>J</i> = 8.20 Hz, 2H), 5.10 (quin, <i>J</i> = 7.25 Hz, 1H), 4.06 (s, 3H), 3.73 (s, 3H), 1.46 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.3, 158.1, 148.1, 136.9, 136.3, 132.6, 128.8, 127.8, 127.3, 124.9, 113.7, 104.2, 55.1, 47.5, 39.0, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O<sub>2</sub>, 336.17065; found, 336.17096. HPLC <i>R</i><sub>t</sub> = 1.32.</div></div><div id="sec4_6_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>-[(1<i>R</i>)-1-(4-Carbamoylphenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>34</b>)</h4><div class="NLM_p last">4-[(1<i>R</i>)-1-[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl]benzoic acid (<b>15</b>) (86 mg, 0.24 mmol) in dry CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was treated with cat. DMF and oxalyl chloride (31 μL, 0.36 mmol). The mixture was stirred for 30 min at room temperature; then, ammonia (7 N in MeOH, 200 μL) was added and stirred for 1 h. The mixture was concentrated in vacuo. The crude product was purified by prep. HPLC (MeCN/water) to give <b>34</b> in 50 mg (59%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.93 (d, <i>J</i> = 7.88 Hz, 1H), 7.93 (br s, 1H), 7.83 (d, <i>J</i> = 8.20 Hz, 2H), 7.78 (d, <i>J</i> = 7.25 Hz, 2H), 7.36–7.49 (m, 5H), 7.33 (s, 2H), 5.16 (quin, <i>J</i> = 7.25 Hz, 1H), 4.05 (s, 3H), 1.49 (d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.8, 158.6, 148.2, 147.8, 136.8, 132.9, 132.6, 128.9, 127.9, 127.7, 125.9, 125.0, 104.4, 48.1, 39.0, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>4</sub>O<sub>2</sub>, 349.16590; found, 349.16635. HPLC <i>R</i><sub>t</sub> = 1.06.</div></div><div id="sec4_6_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 1-Methyl-3-phenyl-<i>N</i>-{1-[4-(pyrimidin-5-yl)phenyl]ethyl}-1<i>H</i>-pyrazole-5-carboxamide (<b>35</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (25 mg, 0.12 mmol) in 20 mg (74%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.18 (s, 1H), 9.13 (s, 2H), 8.96 (d, <i>J</i> = 7.86 Hz, 1H), 7.75–7.82 (m, 4H), 7.56 (d, <i>J</i> = 8.11 Hz, 2H), 7.39–7.48 (m, 3H), 7.33 (t, <i>J</i> = 7.30 Hz, 1H), 5.19 (quin, <i>J</i> = 7.22 Hz, 1H), 4.06 (s, 3H), 1.52 (d, <i>J</i> = 7.10 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.6, 157.2, 154.7, 148.2, 145.4, 136.8, 133.1, 132.6, 132.3, 128.9, 127.9, 127.0, 124.9, 104.4, 48.1, 39.6, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>21</sub>N<sub>5</sub>O, 384.18189; found, 384.18155. HPLC <i>R</i><sub>t</sub> = 1.20.</div></div><div id="sec4_6_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> 1-Methyl-<i>N</i>-[(1<i>R</i>)-1-(4-methylphenyl)ethyl]-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>36</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (30 mg, 0.15 mmol) in 82 mg (82%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.82 (d, <i>J</i> = 8.11 Hz, 1H), 7.77 (d, <i>J</i> = 7.80 Hz, 2H), 7.43 (t, <i>J</i> = 7.50 Hz, 2H), 7.39 (s, 1H), 7.32 (t, <i>J</i> = 7.50 Hz, 1H), 7.28 (d, <i>J</i> = 7.90 Hz, 2H), 7.14 (d, <i>J</i> = 7.90 Hz, 2H), 5.10 (quin, <i>J</i> = 7.22 Hz, 1H), 4.06 (s, 3H), 2.27 (s, 3H), 1.46 (d, <i>J</i> = 7.10 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.4, 148.1, 141.4, 136.9, 135.8, 128.8, 127.8, 126.0, 124.9, 104.3, 47.8, 38.9, 22.2, 20.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O, 320.17574; found, 320.17691. HPLC <i>R</i><sub>t</sub> = 1.40.</div></div><div id="sec4_6_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i>-[(1<i>R</i>)-1-[4-(Hydroxymethyl)phenyl]ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>37</b>)</h4><div class="NLM_p last">To methyl 4-[(1<i>R</i>)-1-[(1-methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl]benzoate (prepared according to General Method A, 300 mg, 0.83 mmol) in dry tetrahydrofuran (THF) (10 mL), lithium aluminum hydride (1 M in THF, 3 mL, 3 mmol) was added at 0 °C. The mixture was stirred for 16 h at room temperature, quenched with water, and filtered. The filtrate was concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give 37 in 140 mg (51%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.84 (br d, <i>J</i> = 8.20 Hz, 1H), 7.78 (br d, <i>J</i> = 7.25 Hz, 2H), 7.43 (t, <i>J</i> = 7.57 Hz, 2H), 7.40 (s, 1H), 7.30–7.37 (m, 3H), 7.28 (br d, <i>J</i> = 7.88 Hz, 2H), 5.07–5.19 (m, 1H), 4.46 (d, <i>J</i> = 5.67 Hz, 2H), 4.06 (s, 3H), 1.47 (d, <i>J</i> = 6.94 Hz, 3H) OH under signal at 5.12 ppm. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.4, 148.1, 142.8, 141.0, 136.8, 132.6, 128.8, 127.8, 126.5, 125.8, 124.9, 104.3, 62.7, 48.0, 22.2, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>21</sub>N<sub>3</sub>O<sub>2</sub>, 336.17065; found, 336.17119. HPLC <i>R</i><sub>t</sub> = 1.13.</div></div><div id="sec4_6_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>N</i>-[1-(4-Cyanophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>38</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (25 mg, 0.12 mmol) in 18 mg (75%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.98 (d, <i>J</i> = 7.57 Hz, 1H), 7.82 (d, <i>J</i> = 8.20 Hz, 2H), 7.78 (d, <i>J</i> = 6.94 Hz, 2H), 7.60 (d, <i>J</i> = 8.51 Hz, 2H), 7.42–7.47 (m, 2H), 7.42 (s, 1H), 7.33 (t, <i>J</i> = 7.60 Hz, 1H), 5.18 (quin, <i>J</i> = 7.25 Hz, 1H), 4.05 (s, 3H), 1.49 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.6, 150.3, 148.1, 136.5, 132.5, 132.4, 128.8, 127.8, 127.1, 124.9, 118.9, 109.6, 104.4, 48.2, 21.8, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>18</sub>N<sub>4</sub>O, 331.15533; found, 331.15497. HPLC <i>R</i><sub>t</sub> = 1.35.</div></div><div id="sec4_6_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>N</i>-[(1<i>R</i>)-1-[4-(Methanesulfonylcarbamoyl)phenyl]ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>39</b>)</h4><div class="NLM_p last">4-[(1<i>R</i>)-1-[(1-Methyl-3-phenyl-1<i>H</i>-pyrazol-5-yl)formamido]ethyl]benzoic acid (<b>15</b>) (200 mg, 0.57 mmol), 1-hydroxy-7-azabenzotriazole (93 mg, 0.69 mmol), and 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (132 mg, 0.69 mmol) were stirred for 10 min at room temperature. Methane sulfonamide (71 mg, 0.74 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (250 μL, 1.43 mmol) were added and stirred for 16 h at room temperature. The reaction was quenched with 1 N HCl, and the mixture was concentrated in vacuo. The crude product was purified by prep. HPLC (MeCN/water) to give <b>39</b> in 50 mg (20%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.86–12.31 (m, 1H), 8.97 (d, <i>J</i> = 7.88 Hz, 1H), 7.93 (d, <i>J</i> = 8.20 Hz, 2H), 7.79 (dd, <i>J</i> = 1.10, 8.04 Hz, 2H), 7.54 (d, <i>J</i> = 8.51 Hz, 2H), 7.39–7.48 (m, 3H), 7.30–7.37 (m, 1H), 5.19 (quin, <i>J</i> = 7.17 Hz, 1H), 4.06 (s, 3H), 1.50 (d, <i>J</i> = 7.25 Hz, 3H), 3H under DMSO signal. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.7, 159.1, 150.5, 148.6, 137.1, 133.0, 130.7, 129.3, 129.1, 128.3, 126.6, 125.4, 104.9, 48.6, 43.6, 41.8, 22.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub>S, 427.14345; found, 427.14377. HPLC <i>R</i><sub>t</sub> = 0.87.</div></div><div id="sec4_6_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> 2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic Acid [1-(4-Acetylsulfamoyl-phenyl)-ethyl]-amide (<b>40</b>)</h4><div class="NLM_p last">To 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [1-(4-sulfamoyl-phenyl)-ethyl]-amide (<b>18</b>) (50 mg, 0.13 mmol) and Et<sub>3</sub>N (75 μL, 0.52 mmol) in dry CH<sub>2</sub>Cl<sub>2</sub> (1 mL), acetyl chloride (19 μL, 0.26 mmol) was added at 0 °C and stirred for 33 h at room temperature. The mixture was concentrated in vacuo, and the crude product was purified by prep. HPLC (MeCN/water) to give <b>40</b> in 15 mg (27%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.07 (br s, 1H), 9.00 (d, <i>J</i> = 7.86 Hz, 1H), 7.88 (d, <i>J</i> = 8.50 Hz, 2H), 7.78 (d, <i>J</i> = 8.00 Hz, 2H), 7.62 (d, <i>J</i> = 8.50 Hz, 2H), 7.40–7.43 (m, 1H), 7.39–7.48 (m, 2H), 7.33 (t, <i>J</i> = 7.60 Hz, 1H), 5.19 (quin, <i>J</i> = 7.16 Hz, 1H), 4.06 (s, 3H), 1.90 (s, 3H), 1.50 (d, <i>J</i> = 7.10 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 169.1, 158.6, 150.2, 148.1, 138.1, 136.5, 132.5, 128.8, 127.8, 127.7, 126.6, 124.9, 104.5, 48.1, 38.9, 23.4, 21.8. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub>S, 427.14345; found, 427.14360. HPLC <i>R</i><sub>t</sub> = 0.86.</div></div><div id="sec4_6_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> [Methanesulfonyl-(4-{(<i>R</i>)-1-[(2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carbonyl)-amino]-ethyl}-phenyl)-amino]-acetic Acid (<b>41</b>)</h4><div class="NLM_p last">2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid [(<i>R</i>)-1-(4-methanesulfonylamino-phenyl)-ethyl]-amide (<b>32</b>) (240 mg, 0.60 mmol), methyl bromoacetate (63 μL, 0.66 mmol), and K<sub>2</sub>CO<sub>3</sub> (101 mg, 0.72 mmol) in dry MeCN (10 mL) were stirred at room temperature overnight. The mixture was partitioned between water and CH<sub>2</sub>Cl<sub>2</sub>, and the organic layer was washed with water and 1 N aqueous HCl. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. The crude intermediate (250 mg, 0.53 mmol) and 1 N aqueous LiOH solution (2.13 mL) in THF (10 mL) were stirred at room temperature overnight. The reaction mixture was treated with water, and the pH was adjusted to 4 using 1 N aqueous HCl. The aqueous layer was extracted with EtOAc, and the combined organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo to give <b>41</b> in 220 mg (80%) overall yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.90 (d, <i>J</i> = 8.20 Hz, 1H), 7.74–7.81 (m, 2H), 7.37–7.48 (m, 7H), 7.29–7.35 (m, 1H), 5.15 (quin, <i>J</i> = 7.25 Hz, 1H), 4.33 (s, 2H), 4.07 (s, 3H), 3.09 (s, 3H), 1.48 (d, <i>J</i> = 6.94 Hz, 3H), COOH not visible. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.6, 158.5, 148.1, 143.4, 139.0, 136.7, 132.6, 128.8, 127.8, 127.4, 126.8, 124.9, 104.3, 52.6, 47.7, 22.0, 2C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>24</sub>N<sub>4</sub>O<sub>5</sub>S, 457.15402; found, 457.15314. HPLC <i>R</i><sub>t</sub> = 1.01.</div></div><div id="sec4_6_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (4-{1-[(2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carbonyl)-amino]-ethyl}-phenyl)-acetic Acid (<b>42</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 2-methyl-5-phenyl-2<i>H</i>-pyrazole-3-carboxylic acid (20 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.85 (d, <i>J</i> = 7.88 Hz, 1H), 7.78 (d, <i>J</i> = 7.25 Hz, 2H), 7.37–7.46 (m, 2H), 7.37–7.40 (m, 1H), 7.29–7.34 (m, 1H), 7.28–7.36 (m, 2H), 7.21 (d, <i>J</i> = 7.88 Hz, 2H), 5.11 (quin, <i>J</i> = 7.17 Hz, 1H), 4.06 (s, 3H), 3.50 (s, 2H), 1.47 (q, <i>J</i> = 6.94 Hz, 3H), COOH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 172.8, 158.4, 149.7, 148.1, 142.5, 136.8, 132.6, 129.3, 128.8, 127.7, 125.9, 124.9, 104.3, 77.9, 47.9, 39.0, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>21</sub>N<sub>3</sub>O<sub>3</sub>, 364.16557; found, 364.16557. HPLC <i>R</i><sub>t</sub> = 0.86.</div></div><div id="sec4_6_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> 4-{(<i>R</i>)-1-[(2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carbonyl)-amino]-ethyl}-benzenesulfonyl)-acetic Acid Methyl Ester (<b>Int-10</b>)</h4><div class="NLM_p last"><i>N</i>-[(1<i>R</i>)-1-(4-Iodophenyl)ethyl]-1-methyl-3-phenyl-1<i>H</i>-pyrazole-5-carboxamide (<b>Int-02</b>) (420 mg, 0.97 mmol), 1,4-diazabicylco[2.2.2]octane bis(sulfur dioxide) adduct (271 mg, 1.07 mmol), Pd(OAc)2 (22 mg, 0.10 mmol), butyldi-1-adamantylphosphine (70 mg, 0.20 mmol), Et<sub>3</sub>N (331 μL, 2.92 mmol), and isopropanol (4 mL) were stirred for 1 h at 75 °C. Methyl bromoacetate (248 μL, 2.92 mmol) in DMF (1 mL) was added and stirred for 2 h at room temperature. The mixture was partitioned between water and EtOAc, and the organic layer was washed with water and saturated aqueous NH<sub>4</sub>Cl solution. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo. The crude product was purified by medium-pressure liquid chromatography (cyclohexane/EtOAc) to give <b>Int-10</b> in 330 mg (79%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.02 (d, <i>J</i> = 7.57 Hz, 1H), 7.89 (d, <i>J</i> = 8.51 Hz, 2H), 7.79 (d, <i>J</i> = 7.25 Hz, 2H), 7.68 (d, <i>J</i> = 8.51 Hz, 2H), 7.41–7.47 (m, 3H), 7.34 (t, <i>J</i> = 7.60 Hz, 1H), 5.22 (quin, <i>J</i> = 7.25 Hz, 1H), 4.64 (s, 2H), 4.06 (s, 3H), 3.59 (s, 3H), 1.51 (d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.2, 158.7, 151.1, 148.1, 137.4, 136.5, 132.5, 128.8, 128.4, 127.8, 126.8, 124.9, 104.5, 59.5, 52.5, 48.2, 21.9, 1C under DMSO signal. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>22</sub>H<sub>23</sub>N<sub>3</sub>O<sub>5</sub>S, 442.14312; found, 442.14279. HPLC <i>R</i><sub>t</sub> = 1.30.</div></div><div id="sec4_6_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (4-{(<i>R</i>)-1-[(2-Methyl-5-phenyl-2<i>H</i>-pyrazole-3-carbonyl)-amino]-ethyl}-benzenesulfonyl)-acetic Acid (<b>43</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>Int-10</b> (98 mg, 0.22 mmol) according to General Method B in 89 mg (93%) yield.</div></div><div id="sec4_6_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> 4-{(<i>R</i>)-1-[(1-Methyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>44</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 1-methylindole-2-carboxylic acid (50 mg, 0.29 mmol) in 55 mg (61%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.27–13.33 (m, 1H), 8.94 (d, <i>J</i> = 7.88 Hz, 1H), 7.91 (d, <i>J</i> = 8.51 Hz, 2H), 7.65 (d, <i>J</i> = 7.88 Hz, 1H), 7.45–7.56 (m, 3H), 7.22–7.30 (m, 2H), 7.10 (t, <i>J</i> = 7.30 Hz, 1H), 5.21 (quin, <i>J</i> = 7.25 Hz, 1H), 3.94 (s, 3H), 1.50 (d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 161.2, 150.0, 138.4, 131.9, 129.4, 129.2, 126.2, 125.5, 123.6, 121.5, 120.1, 110.5, 104.7, 48.1, 31.3, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>18</sub>N<sub>2</sub>O<sub>3</sub>, 323.13902; found, 323.13939. HPLC <i>R</i><sub>t</sub> = 0.85.</div></div><div id="sec4_6_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> 4-{(<i>R</i>)-1-[(5-Chloro-1-methyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>45</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (60 mg, 0.29 mmol) in 93 mg (91%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.86 (br s, 1H), 9.03 (br d, <i>J</i> = 7.88 Hz, 1H), 7.92 (br d, <i>J</i> = 8.20 Hz, 2H), 7.74 (s, 1H), 7.57 (br d, <i>J</i> = 8.83 Hz, 1H), 7.52 (br d, <i>J</i> = 7.88 Hz, 2H), 7.27 (dd, <i>J</i> = 1.42, 8.67 Hz, 1H), 7.22 (s, 1H), 5.20 (quin, <i>J</i> = 7.09 Hz, 1H), 3.93 (s, 3H), 1.50 (br d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 160.8, 149.9, 136.9, 133.3, 129.5, 129.2, 126.5, 126.2, 124.6, 123.6, 120.6, 112.3, 104.2, 48.2, 31.6, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>3</sub>, 357.10005; found, 357.10038. HPLC <i>R</i><sub>t</sub> = 0.90.</div></div><div id="sec4_6_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 4-{(<i>R</i>)-1-[(5-Chloro-7-methoxy-1-methyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>46</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 5-chloro-7-methoxy-1-methylindole-2-carboxylic acid (70 mg, 28 mmol) in 96 mg (88%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.01 (d, <i>J</i> = 7.88 Hz, 1H), 7.91 (d, <i>J</i> = 8.20 Hz, 2H), 7.51 (d, <i>J</i> = 8.20 Hz, 2H), 7.27 (d, <i>J</i> = 1.89 Hz, 1H), 7.07 (s, 1H), 6.76 (d, <i>J</i> = 1.58 Hz, 1H), 5.17 (quin, <i>J</i> = 7.17 Hz, 1H), 4.12 (s, 3H), 3.90 (s, 3H), 1.48 (d, <i>J</i> = 6.94 Hz, 3H), COOH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 160.8, 149.9, 148.3, 134.0, 129.5, 129.3, 128.0, 126.6, 126.2, 124.7, 113.2, 105.3, 105.0, 56.2, 48.3, 34.4, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>4</sub>, 387.11061; found, 387.11098. HPLC <i>R</i><sub>t</sub> = 0.98.</div></div><div id="sec4_6_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 4-{(<i>R</i>)-1-[(5-Chloro-1-methyl-7-trifluoromethoxy-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>47</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 5-chloro-1-methyl-7-trifluoromethoxy-1<i>H</i>-indole-2-carboxylic acid (40 mg, 0.13 mmol) in 56 mg yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.19 (d, <i>J</i> = 7.86 Hz, 1H), 7.92 (d, <i>J</i> = 8.36 Hz, 2H), 7.85 (d, <i>J</i> = 1.77 Hz, 1H), 7.52 (d, <i>J</i> = 8.36 Hz, 2H), 7.29–7.35 (m, 1H), 7.24 (s, 1H), 5.19 (quin, <i>J</i> = 7.16 Hz, 1H), 4.05 (s, 3H), 1.49 (d, <i>J</i> = 7.10 Hz, 3H), COOH not visible. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 160.2, 149.6, 135.9, 134.5, 130.0, 129.5, 129.3, 128.1, 126.2, 123.6, 120.6, 120.2 (q, <i>J</i>(CF) = 257.5 Hz), 116.4, 105.3, 48.3, 33.6, 21.9. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>4</sub>, 441.08235; found, 441.08145. HPLC <i>R</i><sub>t</sub> = 1.09.</div></div><div id="sec4_6_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 4-{(<i>R</i>)-1-[(6-Bromo-5-chloro-1-methyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>48</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 6-bromo-5-chloro-1-methylindole-2-carboxylic acid in 20 mg yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.09 (d, <i>J</i> = 7.86 Hz, 1H), 8.03 (s, 1H), 7.96 (s, 1H), 7.91 (d, <i>J</i> = 8.36 Hz, 2H), 7.51 (d, <i>J</i> = 8.36 Hz, 2H), 7.21 (s, 1H), 5.19 (quin, <i>J</i> = 7.22 Hz, 1H), 3.91 (s, 3H), 1.49 (d, <i>J</i> = 7.10 Hz, 3H) acid OH not visible. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.3, 160.6, 149.7, 137.6, 134.1, 129.5, 129.5, 126.2, 125.9, 124.4, 122.4, 116.0, 115.8, 104.4, 48.3, 31.9, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>BrClN<sub>2</sub>O<sub>3</sub>, 435.01056; found, 435.01065. HPLC <i>R</i><sub>t</sub> = 1.11.</div></div><div id="sec4_6_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 4-{(<i>R</i>)-1-[(3-Methyl-3<i>H</i>-benzo[<i>e</i>]indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>49</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 3-methyl-3<i>H</i>-benzo[<i>e</i>]indole-2-carboxylic acid (90 mg, 0.28 mmol) in 90 mg (86%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.96 (d, <i>J</i> = 7.88 Hz, 1H), 8.24 (d, <i>J</i> = 8.20 Hz, 1H), 7.90–8.00 (m, 3H), 7.88 (s, 1H), 7.74 (s, 2H), 7.51–7.64 (m, 3H), 7.45 (t, <i>J</i> = 7.25 Hz, 1H), 5.24 (quin, <i>J</i> = 7.17 Hz, 1H), 4.07 (s, 3H), 1.54 (d, <i>J</i> = 6.94 Hz, 3H), COOH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 161.1, 150.0, 135.4, 130.0, 129.4, 129.3, 128.9, 128.7, 128.0, 126.4, 126.2, 124.7, 123.8, 122.4, 120.5, 112.1, 104.3, 48.1, 31.8, 22.1. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>20</sub>N<sub>2</sub>O<sub>3</sub>, 373.15467; found, 373.15503. HPLC <i>R</i><sub>t</sub> = 0.96.</div></div><div id="sec4_6_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 4-{(<i>R</i>)-1-[(5-Chloro-1,6-dimethyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>50</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 5-chloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylic acid (218 mg, 0.57 mmol) in 193 mg (92%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.21–13.49 (m, 1H), 8.98 (d, <i>J</i> = 8.20 Hz, 1H), 7.91 (d, <i>J</i> = 8.20 Hz, 2H), 7.73 (s, 1H), 7.47–7.58 (m, 3H), 7.18 (s, 1H), 5.19 (quin, <i>J</i> = 7.25 Hz, 1H), 3.90 (s, 3H), 2.44 (s, 3H), 1.49 (d, <i>J</i> = 7.25 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 160.9, 149.9, 137.4, 132.6, 130.5, 129.4, 129.2, 126.2, 125.9, 124.8, 120.9, 112.3, 104.1, 48.1, 31.5, 22.0, 20.7. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>3</sub>, 371.11570; found, 371.11551. HPLC <i>R</i><sub>t</sub> = 1.00.</div></div><div id="sec4_6_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 4-{(<i>R</i>)-1-[(4,5-Dichloro-1-methyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>51</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 4,5-dichloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (50 mg, 0.21 mmol) in 35 mg (44%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.15 (d, <i>J</i> = 7.86 Hz, 1H), 7.92 (d, <i>J</i> = 8.36 Hz, 2H), 7.60 (d, <i>J</i> = 8.87 Hz, 1H), 7.53 (d, <i>J</i> = 8.36 Hz, 2H), 7.44 (d, <i>J</i> = 8.87 Hz, 1H), 7.36 (s, 1H), 5.20 (quin, <i>J</i> = 7.16 Hz, 1H), 3.96 (s, 3H), 1.51 (d, <i>J</i> = 7.10 Hz, 3H), COOH not visible. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 160.3, 149.8, 137.3, 133.7, 129.5, 129.3, 126.2, 125.4, 124.8, 123.0, 122.6, 111.4, 103.1, 48.3, 32.1, 22.0. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>, 391.06107; found, 391.06168. HPLC <i>R</i><sub>t</sub> = 1.01.</div></div><div id="sec4_6_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 4-{(<i>R</i>)-1-[(4-Chloro-1,6-dimethyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>52</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 4-chloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylic acid (229 mg, 1.02 mmol) in 287 mg (76%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.97–13.66 (m, 1H), 9.03 (br d, <i>J</i> = 7.60 Hz, 1H), 7.92 (br d, <i>J</i> = 7.60 Hz, 2H), 7.52 (br d, <i>J</i> = 7.60 Hz, 2H), 7.31 (br d, <i>J</i> = 13.69 Hz, 2H), 7.05 (br s, 1H), 5.19 (br t, <i>J</i> = 6.72 Hz, 1H), 3.92 (br s, 3H), 2.43 (br s, 3H), 1.50 (br d, <i>J</i> = 6.34 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 160.7, 150.0, 139.5, 134.4, 132.0, 129.5, 129.2, 126.2, 125.2, 122.1, 121.4, 109.6, 102.8, 48.2, 31.8, 22.0, 21.4. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>3</sub>, 371.11570; found, 371.11584. HPLC <i>R</i><sub>t</sub> = 1.04.</div></div><div id="sec4_6_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 4-{(<i>R</i>)-1-[(4-Chloro-5-fluoro-7-methoxy-1-methyl-1<i>H</i>-indole-2-carbonyl)-amino]-ethyl}-benzoic Acid (<b>55</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 4-chloro-5-fluoro-7-methoxy-1-methylindole-2-carboxylic acid (50 mg, 19 mmol) in 40 mg (53%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.50–13.30 (m, 1H), 9.09 (br d, <i>J</i> = 7.88 Hz, 1H), 7.92 (d, <i>J</i> = 8.51 Hz, 2H), 7.52 (d, <i>J</i> = 8.51 Hz, 2H), 7.23 (s, 1H), 6.90 (d, <i>J</i> = 11.98 Hz, 1H), 5.18 (quin, <i>J</i> = 7.17 Hz, 1H), 4.09–4.20 (m, 3H), 3.92 (s, 3H), 1.49 (d, <i>J</i> = 6.94 Hz, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 167.2 160.3, 152.2 (d, <i>J</i>(CF) = 237.1 Hz), 150.3, 149.7, 147.3 (d, <i>J</i>(CF) = 10.9 Hz), 134.4 (d, <i>J</i>(CF) = 4.7 Hz), 129.5, 129.4, 129.3, 126.4, 126.2, 125.3 (d, <i>J</i>(CF) = 3.6 Hz), 124.8, 103.5 (br d, <i>J</i>(CF) = 5.5 Hz), 101.0 (d, <i>J</i>(CF) = 20.0 Hz), 95.7 (br d, <i>J</i>(CF) = 30.0 Hz), 56.6, 48.3, 34.5, 21.9. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>18</sub>ClFN<sub>2</sub>O<sub>4</sub>, 405.10119; found, 405.10132. HPLC <i>R</i><sub>t</sub> = 1.00.</div></div><div id="sec4_6_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Methyl (2<i>Z</i>)-2-Azido-3-(2,3-dichloro-4-methylphenyl)prop-2-enoate (<b>Int-05</b>)</h4><div class="NLM_p last">Sodium (134 mg, 5.82 mmol) was slowly added to dry MeOH (10 mL). A mixture of 2,3-dichloro-4-methylbenzaldehyde (500 mg, 2.65 mmol) and ethyl-2-azido acetate (30% in CH<sub>2</sub>Cl<sub>2</sub>, 5.69 g, 13.23 mmol) is added at −78 °C and stirred for 1 h. The mixture is brought to room temperature and stirred overnight. The mixture was partitioned between water and EtOAc, and the organic layer was washed with water and saturated aqueous NaCl solution. The organic layer was dried (Na<sub>2</sub>SO<sub>4</sub>) and concentrated in vacuo to give <b>Int-05</b> as a yellow solid in 618 mg (82%) yield.</div></div><div id="sec4_6_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> Methyl 4,5-Dichloro-6-methyl-1<i>H</i>-indole-2-carboxylate (<b>Int-06</b>)</h4><div class="NLM_p last">Methyl (2<i>Z</i>)-2-azido-3-(2,3-dichloro-4-methylphenyl)prop-2-enoate (<b>Int-05</b>) (618 mg, 2.16 mmol) in <i>o</i>-xylene (10 mL) was stirred for 50 min at 130 °C. The reaction solution was concentrated in vacuo to provide the product <b>Int-06</b> in 595 mg (96%) yield.</div></div><div id="sec4_6_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> Methyl 4,5-Dichloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylate (<b>Int-07</b>)</h4><div class="NLM_p last">Methyl 4,5-dichloro-6-methyl-1<i>H</i>-indole-2-carboxylate (<b>Int-06</b>) (1.97 g, 7.62 mmol), K<sub>2</sub>CO<sub>3</sub> (2.21 g, 16.00 mmol), and iodomethane (1.64 g, 11.43 mmol) in dry DMF (20 mL) were stirred for 1 h at 60 °C. The solid was filtered off, and the filtrate was treated with water. The precipitate was filtered off, washed with water, and dried to provide the product <b>Int-07</b> as a white solid in 2.03 g (98%) yield.</div></div><div id="sec4_6_52" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> 4,5-Dichloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylic Acid (<b>Int-08</b>)</h4><div class="NLM_p">To methyl 4,5-dichloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylate (<b>Int-07</b>) (2.03 g, 7.46 mmol) in MeOH (50 mL), 8 M aqueous NaOH (2.0 mL, 16.0 mmol) was added and stirred for 1 h at 65 °C. The solvent was removed in vacuo, and the residue was treated with 2 N HCl. The precipitate was filtered off, washed with water, and dried to provide the product <b>Int-08</b> as a white solid in 1.91 g (99%) yield.</div><div class="NLM_p last">All other 1-methyl-1<i>H</i>-indole-2-carboxylic acids were prepared in an analogous way from commercially available starting materials.</div></div><div id="sec4_6_53" class="NLM_sec NLM_sec_level_3"><div id="ac_i102" class="anchor-spacer"></div><h4 class="article-section__title" id="_i102"> 4-[(1<i>R</i>)-1-[(4,5-Dichloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]ethyl]benzoic Acid (<b>57</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 4,5-dichloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylic acid (2 mg, 0.19 mmol) in 79 mg (quant.) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.10 (d, <i>J</i> = 7.57 Hz, 1H), 7.92 (d, <i>J</i> = 8.20 Hz, 2H), 7.59 (s, 1H), 7.52 (d, <i>J</i> = 8.20 Hz, 2H), 7.31 (s, 1H), 5.20 (quin, <i>J</i> = 7.17 Hz, 1H), 3.94 (s, 3H), 1.50 (d, <i>J</i> = 7.25 Hz, 3H), 3H under DMSO, acid OH not visible. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 160.4, 149.8, 136.9, 132.9, 132.2, 129.5, 129.3, 126.2, 123.7, 123.6, 123.3, 111.4, 103.1, 48.3, 31.9, 21.9, 21.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>20</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>3</sub>, 405.07672; found, 405.07687. HPLC <i>R</i><sub>t</sub> = 1.07.</div></div><div id="sec4_6_54" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> 2-{4-[(1<i>R</i>)-1-[(4,5-Dichloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]ethyl]benzenesulfonyl}acetic Acid (<b>58</b>)</h4><div class="NLM_p last">The title compound was prepared as described for the synthesis of 2-{4-[(1<i>S</i>)-1-[(4,5-dichloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]-2-hydroxyethyl]benzenesulfonyl}acetic acid (<b>BI-4924</b>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.04–13.52 (m, 1H), 9.14 (d, <i>J</i> = 7.60 Hz, 1H), 7.89 (d, <i>J</i> = 8.36 Hz, 2H), 7.67 (d, <i>J</i> = 8.36 Hz, 2H), 7.60 (s, 1H), 7.34 (s, 1H), 5.23 (quin, <i>J</i> = 7.16 Hz, 1H), 4.43 (s, 2H), 3.94 (s, 3H), 1.51 (d, <i>J</i> = 7.10 Hz, 3H), 3H under DMSO signal. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.9, 160.4, 151.0, 137.8, 136.9, 132.8, 132.2, 128.3, 126.7, 123.7, 123.6, 123.3, 111.4, 103.2, 60.2, 48.2, 31.9, 21.9, 21.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>5</sub>S, 439.06444; found, 439.06381 (in source fragmentation—CO<sub>2</sub>). HPLC <i>R</i><sub>t</sub> = 1.10.</div></div><div id="sec4_6_55" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> 2-{4-[(1<i>R</i>)-1-[(4-Chloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]ethyl]benzenesulfonyl}acetic Acid (<b>59</b>)</h4><div class="NLM_p last">The title compound was prepared as described for the synthesis of 2-{4-[(1<i>S</i>)-1-[(4,5-dichloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]-2-hydroxyethyl]benzenesulfonyl}acetic acid (<b>BI-4924</b>). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.91–13.62 (m, 1H), 9.07 (d, <i>J</i> = 7.60 Hz, 1H), 7.89 (d, <i>J</i> = 8.11 Hz, 2H), 7.67 (d, <i>J</i> = 8.36 Hz, 2H), 7.32 (d, <i>J</i> = 7.86 Hz, 2H), 7.05 (s, 1H), 5.23 (quin, <i>J</i> = 7.16 Hz, 1H), 4.46 (s, 2H), 3.93 (s, 3H), 2.43 (s, 3H), 1.51 (d, <i>J</i> = 6.84 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.0, 160.7, 151.2, 139.5, 137.7, 134.4, 131.8, 128.3, 126.7, 125.1, 122.1, 121.4, 109.5, 102.9, 60.0, 48.2, 31.8, 22.0, 21.3. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>21</sub>ClN<sub>2</sub>O<sub>5</sub>S, 449.09325; found, 449.09274. HPLC <i>R</i><sub>t</sub> = 1.13.</div></div><div id="sec4_6_56" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid (2-Hydroxy-1-phenyl-ethyl)-amide (<b>60</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (70 mg, 0.32 mmol) in 80 mg (76%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.84 (d, <i>J</i> = 8.20 Hz, 1H), 7.74 (d, <i>J</i> = 1.89 Hz, 1H), 7.56 (d, <i>J</i> = 8.83 Hz, 1H), 7.40 (d, <i>J</i> = 7.57 Hz, 2H), 7.33 (t, <i>J</i> = 7.57 Hz, 2H), 7.20–7.29 (m, 3H), 5.06 (dt, <i>J</i> = 5.83, 7.96 Hz, 1H), 4.97 (t, <i>J</i> = 5.83 Hz, 1H), 3.93 (s, 3H), 3.60–3.76 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.3, 141.2, 136.8, 133.6, 128.2, 127.0, 126.9, 126.6, 124.6, 123.5, 120.5, 112.3, 104.2, 64.5, 55.7, 31.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>17</sub>ClN<sub>2</sub>O<sub>2</sub>, 329.10513; found, 329.10558. HPLC <i>R</i><sub>t</sub> = 1.28.</div></div><div id="sec4_6_57" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> [(5-Chloro-1-methyl-1<i>H</i>-indole-2-carbonyl)-amino]-phenyl-acetic Acid (<b>61</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A and General Method B from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (70 mg, 0.32 mmol) in 59 mg (63%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.96 (br s, 1H), 9.16 (d, <i>J</i> = 7.57 Hz, 1H), 7.72 (d, <i>J</i> = 1.89 Hz, 1H), 7.58 (d, <i>J</i> = 9.14 Hz, 1H), 7.51 (d, <i>J</i> = 7.20 Hz, 2H), 7.39 (t, <i>J</i> = 7.20 Hz, 2H), 7.34 (t, <i>J</i> = 7.20 Hz, 1H), 7.23–7.30 (m, 2H), 5.60 (d, <i>J</i> = 7.57 Hz, 1H), 3.96 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.8, 161.3, 136.9, 136.9, 132.7, 128.4, 128.2, 128.0, 126.5, 124.6, 123.7, 120.7, 112.3, 105.0, 56.4, 31.7. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>15</sub>ClN<sub>2</sub>O<sub>3</sub>, 343.08440; found, 343.08392. HPLC <i>R</i><sub>t</sub> = 0.93.</div></div><div id="sec4_6_58" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> 5-Chloro-<i>N</i>-(3-hydroxy-1-phenylpropyl)-1-methyl-1<i>H</i>-indole-2-carboxamide (<b>62</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (70 mg, 0.32 mmol) in 60 mg (55%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.92 (d, <i>J</i> = 8.20 Hz, 1H), 7.74 (d, <i>J</i> = 1.89 Hz, 1H), 7.56 (d, <i>J</i> = 8.83 Hz, 1H), 7.40 (d, <i>J</i> = 7.70 Hz, 2H), 7.33 (t, <i>J</i> = 7.72 Hz, 2H), 7.26 (dd, <i>J</i> = 2.21, 8.83 Hz, 1H), 7.23 (t, <i>J</i> = 7.30 Hz, 1H), 7.17 (s, 1H), 5.15 (dt, <i>J</i> = 6.15, 8.43 Hz, 1H), 4.60 (t, <i>J</i> = 4.89 Hz, 1H), 3.92 (s, 3H), 3.38–3.54 (m, 2H), 2.03 (br dd, <i>J</i> = 5.52, 8.35 Hz, 1H), 1.76–1.95 (m, 1H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.9, 143.9, 136.8, 133.6, 128.3, 126.7, 126.6, 126.5, 124.6, 123.5, 120.5, 112.3, 104.0, 57.8, 50.0, 38.9, 31.5. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>2</sub>, 343.12078; found, 343.12134. HPLC <i>R</i><sub>t</sub> = 1.28.</div></div><div id="sec4_6_59" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid (2-Methoxy-1-phenyl-ethyl)-amide (<b>63</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (70 mg, 0.32 mmol) in 80 mg (73%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.98 (d, <i>J</i> = 8.20 Hz, 1H), 7.75 (d, <i>J</i> = 1.89 Hz, 1H), 7.57 (d, <i>J</i> = 8.83 Hz, 1H), 7.43 (d, <i>J</i> = 7.57 Hz, 2H), 7.35 (t, <i>J</i> = 7.57 Hz, 2H), 7.23–7.29 (m, 2H), 7.21 (s, 1H), 5.26 (dt, <i>J</i> = 5.36, 8.51 Hz, 1H), 3.94 (s, 3H), 3.69 (t, <i>J</i> = 1.00 Hz, 1H), 3.55 (dd, <i>J</i> = 5.36, 10.09 Hz, 1H), 3.31 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.1, 140.5, 136.8, 133.4, 128.3, 127.2, 127.0, 126.5, 124.6, 123.5, 120.6, 112.3, 104.2, 74.5, 58.0, 52.3, 31.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>2</sub>, 343.12078; found, 343.11996. HPLC <i>R</i><sub>t</sub> = 1.41.</div></div><div id="sec4_6_60" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid (2-Amino-1-phenyl-ethyl)-amide (<b>64</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (70 mg, 0.32 mmol) and (2-amino-2-phenyl-ethyl)-carbamic acid <i>tert</i>-butyl ester (124 mg, 0.50 mmol) with subsequent boc deprotection in 24 mg (23%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.06 (br d, <i>J</i> = 8.51 Hz, 1H), 7.82–8.16 (m, 2H), 7.77 (d, <i>J</i> = 1.89 Hz, 1H), 7.59 (d, <i>J</i> = 8.83 Hz, 1H), 7.45 (d, <i>J</i> = 7.30 Hz, 2H), 7.40 (t, <i>J</i> = 7.57 Hz, 2H), 7.24–7.35 (m, 3H), 5.26–5.37 (m, 1H), 3.95 (s, 3H), 3.17–3.30 (m, 2H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.6, 139.7, 136.9, 133.1, 128.7, 127.8, 126.7, 126.4, 124.8, 123.8, 120.6, 112.4, 104.7, 51.1, 43.1, 31.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>18</sub>ClN<sub>3</sub>O, 328.12112; found, 328.12151. HPLC <i>R</i><sub>t</sub> = 1.22.</div></div><div id="sec4_6_61" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid (Carbamoyl-phenyl-methyl)-amide (<b>65</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (70 mg, 0.32 mmol) in 91 mg (83%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.80 (d, <i>J</i> = 7.88 Hz, 1H), 7.66–7.74 (m, 2H), 7.57 (d, <i>J</i> = 9.14 Hz, 1H), 7.53 (d, <i>J</i> = 7.57 Hz, 2H), 7.37 (t, <i>J</i> = 7.60 Hz, 2H), 7.29–7.33 (m, 1H), 7.26 (s, 3H), 5.61 (d, <i>J</i> = 7.88 Hz, 1H), 3.94 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 171.5, 161.0, 138.6, 136.9, 133.0, 128.3, 127.6, 127.5, 126.6, 124.6, 123.6, 120.7, 112.3, 104.8, 56.5, 31.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>18</sub>H<sub>16</sub>ClN<sub>3</sub>O2, 342.10038; found, 342.10095. HPLC <i>R</i><sub>t</sub> = 1.21.</div></div><div id="sec4_6_62" class="NLM_sec NLM_sec_level_3"><div id="ac_i111" class="anchor-spacer"></div><h4 class="article-section__title" id="_i111"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid ((1<i>S</i>,2<i>R</i>)-2-Hydroxy-1-phenyl-propyl)-amide (<b>66</b>) and 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid ((1<i>S</i>,2<i>S</i>)-2-Hydroxy-1-phenyl-propyl)-amide (<b>67</b>)</h4><div class="NLM_p">The title compounds were prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (65 mg, 0.31 mmol) with subsequent supercritical fluid chromatography separation of the diastereomeric product mixture.</div><div class="NLM_p"><b>66</b>: <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.69 (br d, <i>J</i> = 8.11 Hz, 1H), 7.74 (br s, 1H), 7.56 (br d, <i>J</i> = 8.62 Hz, 1H), 7.42 (br d, <i>J</i> = 7.10 Hz, 2H), 7.17–7.36 (m, 5H), 4.84 (br d, <i>J</i> = 4.31 Hz, 2H), 3.94–4.05 (m, 1H), 3.91 (br s, 3H), 0.93–1.06 (m, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.2, 141.8, 136.8, 133.8, 128.1, 127.5, 126.8, 126.6, 124.6, 123.4, 120.5, 112.3, 104.1, 68.7, 59.9, 31.5, 20.9. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>2</sub>, 343.12078; found, 343.12103. HPLC <i>R</i><sub>t</sub> = 1.31.</div><div class="NLM_p last"><b>67</b>: <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.76 (d, <i>J</i> = 8.87 Hz, 1H), 7.74 (d, <i>J</i> = 2.03 Hz, 1H), 7.56 (d, <i>J</i> = 8.87 Hz, 1H), 7.42 (d, <i>J</i> = 7.35 Hz, 2H), 7.31 (t, <i>J</i> = 7.48 Hz, 2H), 7.19–7.28 (m, 2H), 7.16 (s, 1H), 4.85 (t, <i>J</i> = 7.98 Hz, 1H), 4.79 (d, <i>J</i> = 5.83 Hz, 1H), 4.00 (sxt, <i>J</i> = 6.24 Hz, 1H), 3.91 (s, 3H), 1.14 (d, <i>J</i> = 6.34 Hz, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.9, 141.3, 136.9, 133.7, 128.1, 127.8, 126.7, 126.6, 124.7, 123.6, 120.6, 112.3, 104.1, 68.4, 59.1, 31.6, 20.8. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>19</sub>ClN<sub>2</sub>O<sub>2</sub>, 343.12078; found, 343.12135. HPLC <i>R</i><sub>t</sub> = 1.29.</div></div><div id="sec4_6_63" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid ((<i>trans</i>)-3,3,3-Trifluoro-2-hydroxy-1-phenyl-propyl)-amide (<b>68</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (25 mg, 0.12 mmol) in 40 mg (85%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.07 (d, <i>J</i> = 9.46 Hz, 1H), 7.77 (d, <i>J</i> = 1.89 Hz, 1H), 7.56 (d, <i>J</i> = 8.83 Hz, 1H), 7.51 (d, <i>J</i> = 7.57 Hz, 2H), 7.36 (t, <i>J</i> = 7.60 Hz, 2H), 7.24–7.32 (m, 2H), 7.10 (s, 1H), 6.36–6.56 (m, 1H), 5.27 (t, <i>J</i> = 9.46 Hz, 1H), 4.42 (br s, 1H), 3.91 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.3, 139.7, 136.8, 133.1, 128.4, 128.1, 127.4, 126.4, 124.8, 123.7, 125.6 (q, <i>J</i>(CF) = 284.6 Hz), 120.6, 112.4, 104.2, 69.5 (q, <i>J</i>(CF) = 27.9 Hz), 52.4, 31.5. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>, 397.09252; found, 397.09347. HPLC <i>R</i><sub>t</sub> = 1.39.</div></div><div id="sec4_6_64" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> 5-Chloro-1-methyl-1<i>H</i>-indole-2-carboxylic Acid ((<i>cis</i>)-3,3,3-Trifluoro-2-hydroxy-1-phenyl-propyl)-amide (<b>69</b>)</h4><div class="NLM_p last">The title compound was prepared according to General Method A from 5-chloro-1-methyl-1<i>H</i>-indole-2-carboxylic acid (25 mg, 0.12 mmol) in 39 mg (82%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.86 (d, <i>J</i> = 9.38 Hz, 1H), 7.77 (d, <i>J</i> = 1.77 Hz, 1H), 7.57 (d, <i>J</i> = 8.87 Hz, 1H), 7.53 (d, <i>J</i> = 7.60 Hz, 2H), 7.37 (t, <i>J</i> = 7.60 Hz, 2H), 7.24–7.32 (m, 2H), 7.19 (s, 1H), 6.57–6.83 (m, 1H), 5.48 (dd, <i>J</i> = 4.18, 9.25 Hz, 1H), 4.40–4.46 (m, 1H), 3.92 (s, 3H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.9, 139.4, 136.9, 133.2, 128.3, 127.5, 126.5, 126.5, 124.7, 123.7, 120.7, 112.3, 104.5, 71.2(q, <i>J</i>(CF) = 28.5 Hz), 51.9, 31.5, CF<sub>3</sub> not visible. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>19</sub>H<sub>16</sub>ClF<sub>3</sub>N<sub>2</sub>O<sub>2</sub>, 397.09252; found, 397.08277. HPLC <i>R</i><sub>t</sub> = 1.42.</div></div><div id="sec4_6_65" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> 4,5-Dichloro-<i>N</i>-[(1<i>S</i>)-2-hydroxy-1-(4-iodophenyl)ethyl]-1,6-dimethyl-1<i>H</i>-indole-2-carboxamide (<b>Int-09</b>)</h4><div class="NLM_p last">To 4,5-dichloro-1,6-dimethyl-1<i>H</i>-indole-2-carboxylic acid (<b>Int-08</b>) (1.00 g, 3.87 mmol) and HATU (1.77 g, 4.65 mmol) in MeCN (15 mL), <i>i</i>Pr2NEt (1.68 mL, 9.69 mmol) was added and stirred for 10 min at room temperature. (2<i>S</i>)-2-Amino-2-(4-iodophenyl)ethan-1-ol hydrochloride (1.22 g, 6.07 mmol) was added and stirred for 1 h at room temperature. Water was added to the reaction mixture, and the precipitate was filtered off, washed with water, and dried to provide the product <b>Int-09</b> as a white solid in 2.00 g (97%) yield.</div></div><div id="sec4_6_66" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> Ethyl 2-{4-[(1<i>S</i>)-1-[(4,5-Dichloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]-2-hydroxyethyl]benzenesulfonyl}acetate (<b>BI-4916</b>)</h4><div class="NLM_p last">The title compound was prepared from 4,5-dichloro-<i>N</i>-[(1<i>S</i>)-2-hydroxy-1-(4-iodophenyl)ethyl]-1,6-dimethyl-1<i>H</i>-indole-2-carboxamide (<b>Int-09</b>) (2.00 g, 3.98 mmol) as described for <b>Int-10</b> in 1.32 g (63%) yield. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.04 (d, <i>J</i> = 7.88 Hz, 1H), 7.89 (d, <i>J</i> = 8.40 Hz, 2H), 7.69 (d, <i>J</i> = 8.40 Hz, 2H), 7.59 (s, 1H), 7.36 (s, 1H), 5.08–5.20 (m, 2H), 4.59 (s, 2H), 4.01 (q, <i>J</i> = 7.25 Hz, 2H), 3.93 (s, 3H), 3.65–3.81 (m, 2H), 1.02 (t, <i>J</i> = 7.09 Hz, 3H), 3H under DMSO signal. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.7, 161.0, 147.8, 137.6, 137.0, 132.9, 132.3, 128.2, 127.9, 123.8, 123.6, 123.4, 111.5, 103.3, 64.0, 61.5, 59.8, 55.6, 31.9, 21.6, 13.7. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>23</sub>H<sub>24</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>6</sub>S, 527.08049; found, 527.08060. HPLC <i>R</i><sub>t</sub> = 1.38. The purity of <b>BI-4916</b> is >95% according to HPLC analysis (for HPLC traces, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>).</div></div><div id="sec4_6_67" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> 2-{4-[(1<i>S</i>)-1-[(4,5-Dichloro-1,6-dimethyl-1<i>H</i>-indol-2-yl)formamido]-2-hydroxyethyl]benzenesulfonyl}acetic Acid (<b>BI-4924</b>)</h4><div class="NLM_p last">First, 8 M aqueous NaOH (0.40 mL, 3.20 mmol) was added to <b>BI-4916</b> (267 mg, 0.52 mmol) in MeCN (1 mL) and stirred for 30 min at 70 °C. The mixture was treated with 2 N HCl, and the precipitate was filtered off, washed with water, and dried to provide the product <b>BI-4924</b> as a white solid in 23 mg (36%) yield. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.06 (d, <i>J</i> = 7.86 Hz, 1H), 7.89 (d, <i>J</i> = 8.36 Hz, 2H), 7.68 (d, <i>J</i> = 8.36 Hz, 2H), 7.61 (s, 1H), 7.37 (s, 1H), 5.06–5.21 (m, 2H), 4.46 (s, 2H), 3.94 (s, 3H), 3.66–3.76 (m, 2H), acid OH not visible, 3H under DMSO signal. <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.0, 161.0, 147.5, 138.1, 137.0, 132.8, 132.3, 128.1, 127.8, 123.7, 123.6, 123.4, 111.5, 103.3, 64.0, 60.1, 55.5, 31.9, 21.6. HRMS (<i>m</i>/<i>z</i>): [M + H]<sup>+</sup> calcd for C<sub>21</sub>H<sub>20</sub>Cl<sub>2</sub>N<sub>2</sub>O<sub>6</sub>S, 455.05936; found, 455.05991 (in source fragmentation—CO<sub>2</sub>). HPLC <i>R</i><sub>t</sub> = 1.01. The purity of <b>BI-4924</b> is >95% according to HPLC analysis (for HPLC traces, see the <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i117"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00718" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00718" class="ext-link">10.1021/acs.jmedchem.9b00718</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Selectivity profile (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Biochemical assays (HTS PHGDH inhibitor assay, diaphorase NADH counterscreen, 3-phosphoglycerate dehydrogenase (PHGDH) fluorescence intensity assay); cellular assays (<sup>13</sup>C serine assay in MDA-MB-468 cells, cellular uptake assay); (bio)physical methods (ICP-MS, surface plasmon resonance); NMR (high-resolution mass spectroscopy); protein expression, chemoproteomics, protein crystallization; safety (CEREP panel); chromatography; synthetic procedures; HPLC traces of selected compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_002.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_003.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx">jm9b00718_si_001.xlsx (258.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_002.pdf">jm9b00718_si_002.pdf (3.49 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_003.csv">jm9b00718_si_003.csv (4.27 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes: compound <b>9</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH">6RIH</a>; compound <b>15</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3">6RJ3</a>; compound <b>39</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5">6RJ5</a>; compound <b>40</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2">6RJ2</a>; <b>BI-4924</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6">6RJ6</a>; <b>3-PG</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00718" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Harald Weinstabl</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1308-2730" title="Orcid link">http://orcid.org/0000-0003-1308-2730</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#96fef7e4f7faf2b8e1f3fff8e5e2f7f4fad6f4f9f3fee4fff8f1f3e4bbfff8f1f3fafef3fffbb8f5f9fb"><span class="__cf_email__" data-cfemail="a8c0c9dac9c4cc86dfcdc1c6dbdcc9cac4e8cac7cdc0dac1c6cfcdda85c1c6cfcdc4c0cdc1c586cbc7c5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthias Treu</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Joerg Rinnenthal</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephan K. Zahn</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Ettmayer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8422-2625" title="Orcid link">http://orcid.org/0000-0002-8422-2625</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gerd Bader</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Georg Dahmann</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
Pharma GmbH & Co. KG, Birkendorfer Str. 65, 88400 Biberach an der Riß, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dirk Kessler</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Klaus Rumpel</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nikolai Mischerikow</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fabio Savarese</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Gerstberger</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Moriz Mayer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Zoephel</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Renate Schnitzer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wolfgang Sommergruber</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paola Martinelli</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heribert Arnhof</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biljana Peric-Simov</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karin S. Hofbauer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Géraldine Garavel</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yvonne Scherbantin</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sophie Mitzner</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas N. Fett</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guido Scholz</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jens Bruchhaus</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michelle Burkard</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roland Kousek</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tuncay Ciftci</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
Pharma GmbH & Co. KG, Birkendorfer Str. 65, 88400 Biberach an der Riß, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bernadette Sharps</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Schrenk</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christoph Harrer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniela Haering</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bernhard Wolkerstorfer</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xuechun Zhang</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech
Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaobing Lv</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech
Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alicia Du</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech
Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongyang Li</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech
Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yali Li</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner Co., LTD., No. 5 Building, 998 Halei Road, Zhangjiang Hi-Tech
Park, Pudong New Area, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jens Quant</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark Pearson</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darryl B. McConnell</span> - <span class="hlFld-Affiliation affiliation">Boehringer Ingelheim
RCV GmbH & Co. KG, Dr.-Boehringer-Gasse 5-11, 1121 Vienna, Austria</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>H.W. and M.T. contributed equally to this work.</p><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">In addition, all crystallographic data can be downloaded from <a href="https://www.rcsb.org/" class="extLink">https://www.rcsb.org/</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><br /></br></div></li></ul></div><div class="ack" id="ACK-d7e7628-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i122">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36814" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36814" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors express their gratitude to Katja Hauer and Michael Galant for SPR measurements. Andreas Gollner is thanked for scientific discussions and proofreading. Julian Fuchs, Andreas Bergner, and Xiao-Ling Cockcroft are gratefully thanked for quantum-mechanical calculations. Susanne Mayer and Gabriele Glendinning are thanked for compound logistics. Helmut Bergner is thanked for ICP/MS measurements. Furthermore, the authors express their deep gratitude to Nathalie Harrer, Herwig Machat, Andreas Schrenk, Bernd Wellenzohn, Gorana Sijan, and Ida Dinold.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AA</td><td class="NLM_def"><p class="first last">amino acid</p></td></tr><tr><td class="NLM_term">BM</td><td class="NLM_def"><p class="first last">biomarker</p></td></tr><tr><td class="NLM_term">2-HG</td><td class="NLM_def"><p class="first last">2-hydroxyglutarate</p></td></tr><tr><td class="NLM_term">LRE</td><td class="NLM_def"><p class="first last">long-range electrostatics</p></td></tr><tr><td class="NLM_term">NAD+/H</td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">PHGDH</td><td class="NLM_def"><p class="first last">phosphoglycerate dehydrogenase</p></td></tr><tr><td class="NLM_term">SSP</td><td class="NLM_def"><p class="first last">serine synthesis pathway</p></td></tr><tr><td class="NLM_term">STD-NMR</td><td class="NLM_def"><p class="first last">saturation transfer difference NMR</p></td></tr><tr><td class="NLM_term">TCA</td><td class="NLM_def"><p class="first last">tricarboxylic acid</p></td></tr><tr><td class="NLM_term">TNBC</td><td class="NLM_def"><p class="first last">triple-negative breast cancer</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i125">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73686" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73686" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 45 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of Cancer: The Next Generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+Cancer%3A+The+Next+Generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0liDo0TtExFpWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520Cancer%253A%2520The%2520Next%2520Generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zogg, C. K.</span></span> <span> </span><span class="NLM_article-title">Phosphoglycerate Dehydrogenase: Potential Therapeutic Target and Putative Metabolic Oncogene</span>. <i>J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>, <span class="NLM_elocation-id">524101</span> <span class="refDoi"> DOI: 10.1155/2014/524101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1155%2F2014%2F524101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=25574168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC2MvkvFOitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&author=C.+K.+Zogg&title=Phosphoglycerate+Dehydrogenase%3A+Potential+Therapeutic+Target+and+Putative+Metabolic+Oncogene&doi=10.1155%2F2014%2F524101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoglycerate dehydrogenase: potential therapeutic target and putative metabolic oncogene</span></div><div class="casAuthors">Zogg Cheryl K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">524101</span>
        ISSN:<span class="NLM_cas:issn">1687-8450</span>.
    </div><div class="casAbstract">Exemplified by cancer cells' preference for glycolysis, for example, the Warburg effect, altered metabolism in tumorigenesis has emerged as an important aspect of cancer in the past 10-20 years.  Whether due to changes in regulatory tumor suppressors/oncogenes or by acting as metabolic oncogenes themselves, enzymes involved in the complex network of metabolic pathways are being studied to understand their role and assess their utility as therapeutic targets.  Conversion of glycolytic intermediate 3-phosphoglycerate into phosphohydroxypyruvate by the enzyme phosphoglycerate dehydrogenase (PHGDH)-a rate-limiting step in the conversion of 3-phosphoglycerate to serine-represents one such mechanism.  Forgotten since classic animal studies in the 1980s, the role of PHGDH as a potential therapeutic target and putative metabolic oncogene has recently reemerged following publication of two prominent papers near-simultaneously in 2011.  Since that time, numerous studies and a host of metabolic explanations have been put forward in an attempt to understand the results observed.  In this paper, I review the historic progression of our understanding of the role of PHGDH in cancer from the early work by Snell through its reemergence and rise to prominence, culminating in an assessment of subsequent work and what it means for the future of PHGDH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQQpDxpOqoFfO2qxNLjzXe8fW6udTcc2eaKZY7E3k3TLLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvkvFOitQ%253D%253D&md5=831a16d13000c1dec2fd435daf1b089b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1155%2F2014%2F524101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F524101%26sid%3Dliteratum%253Aachs%26aulast%3DZogg%26aufirst%3DC.%2BK.%26atitle%3DPhosphoglycerate%2520Dehydrogenase%253A%2520Potential%2520Therapeutic%2520Target%2520and%2520Putative%2520Metabolic%2520Oncogene%26jtitle%3DJ.%2520Oncol.%26date%3D2014%26volume%3D2014%26doi%3D10.1155%2F2014%2F524101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Possemato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaul, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birsoy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethumadhavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsun, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalaany, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mino-Kenudson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachtel, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">Functional Genomics Reveal that the Serine Synthesis Pathway is Essential in Breast Cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>476</i></span>,  <span class="NLM_fpage">346</span>, <span class="refDoi"> DOI: 10.1038/nature10350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnature10350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21760589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVOku7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2011&pages=346&author=R.+Possematoauthor=K.+M.+Marksauthor=Y.+D.+Shaulauthor=M.+E.+Pacoldauthor=D.+Kimauthor=K.+Birsoyauthor=S.+Sethumadhavanauthor=H.-K.+Wooauthor=H.+G.+Jangauthor=A.+K.+Jhaauthor=W.+W.+Chenauthor=F.+G.+Barrettauthor=N.+Stranskyauthor=Z.-Y.+Tsunauthor=G.+S.+Cowleyauthor=J.+Barretinaauthor=N.+Y.+Kalaanyauthor=P.+P.+Hsuauthor=K.+Ottinaauthor=A.+M.+Chanauthor=B.+Yuanauthor=L.+A.+Garrawayauthor=D.+E.+Rootauthor=M.+Mino-Kenudsonauthor=E.+F.+Brachtelauthor=E.+M.+Driggersauthor=D.+M.+Sabatini&title=Functional+Genomics+Reveal+that+the+Serine+Synthesis+Pathway+is+Essential+in+Breast+Cancer&doi=10.1038%2Fnature10350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Functional genomics reveal that the serine synthesis pathway is essential in breast cancer</span></div><div class="casAuthors">Possemato, Richard; Marks, Kevin M.; Shaul, Yoav D.; Pacold, Michael E.; Kim, Dohoon; Birsoy, Kivanc; Sethumadhavan, Shalini; Woo, Hin-Koon; Jang, Hyun G.; Jha, Abhishek K.; Chen, Walter W.; Barrett, Francesca G.; Stransky, Nicolas; Tsun, Zhi-Yang; Cowley, Glenn S.; Barretina, Jordi; Kalaany, Nada Y.; Hsu, Peggy P.; Ottina, Kathleen; Chan, Albert M.; Yuan, Bingbing; Garraway, Levi A.; Root, David E.; Mino-Kenudson, Mari; Brachtel, Elena F.; Driggers, Edward M.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">476</span>
        (<span class="NLM_cas:issue">7360</span>),
    <span class="NLM_cas:pages">346-350</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells adapt their metabolic processes to drive macromol. biosynthesis for rapid cell growth and proliferation.  RNA interference (RNAi)-based loss-of-function screening has proven powerful for the identification of new and interesting cancer targets, and recent studies have used this technol. in vivo to identify novel tumor suppressor genes.  Here we developed a method for identifying novel cancer targets via neg.-selection RNAi screening using a human breast cancer xenograft model at an orthotopic site in the mouse.  Using this method, we screened a set of metabolic genes assocd. with aggressive breast cancer and stemness to identify those required for in vivo tumorigenesis.  Among the genes identified, phosphoglycerate dehydrogenase (PHGDH) is in a genomic region of recurrent copy no. gain in breast cancer and PHGDH protein levels are elevated in 70% of estrogen receptor (ER)-neg. breast cancers.  PHGDH catalyzes the first step in the serine biosynthesis pathway, and breast cancer cells with high PHGDH expression have increased serine synthesis flux.  Suppression of PHGDH in cell lines with elevated PHGDH expression, but not in those without, causes a strong decrease in cell proliferation and a redn. in serine synthesis.  We find that PHGDH suppression does not affect intracellular serine levels, but causes a drop in the levels of α-ketoglutarate, another output of the pathway and a tricarboxylic acid (TCA) cycle intermediate.  In cells with high PHGDH expression, the serine synthesis pathway contributes approx. 50% of the total anaplerotic flux of glutamine into the TCA cycle.  These results reveal that certain breast cancers are dependent upon increased serine pathway flux caused by PHGDH overexpression and demonstrate the utility of in vivo neg.-selection RNAi screens for finding potential anticancer targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79QL1LpmDqLVg90H21EOLACvtfcHk0liDo0TtExFpWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOku7rL&md5=10d8df92c9fb450ee7edca54bf227558</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature10350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10350%26sid%3Dliteratum%253Aachs%26aulast%3DPossemato%26aufirst%3DR.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DShaul%26aufirst%3DY.%2BD.%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DBirsoy%26aufirst%3DK.%26aulast%3DSethumadhavan%26aufirst%3DS.%26aulast%3DWoo%26aufirst%3DH.-K.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DJha%26aufirst%3DA.%2BK.%26aulast%3DChen%26aufirst%3DW.%2BW.%26aulast%3DBarrett%26aufirst%3DF.%2BG.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DTsun%26aufirst%3DZ.-Y.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DKalaany%26aufirst%3DN.%2BY.%26aulast%3DHsu%26aufirst%3DP.%2BP.%26aulast%3DOttina%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DA.%2BM.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DMino-Kenudson%26aufirst%3DM.%26aulast%3DBrachtel%26aufirst%3DE.%2BF.%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DFunctional%2520Genomics%2520Reveal%2520that%2520the%2520Serine%2520Synthesis%2520Pathway%2520is%2520Essential%2520in%2520Breast%2520Cancer%26jtitle%3DNature%26date%3D2011%26volume%3D476%26spage%3D346%26doi%3D10.1038%2Fnature10350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J.
W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grassian, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metallo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muranen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharfi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastasiou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vokes, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroukhim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligon, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Phosphoglycerate Dehydrogenase Diverts Ggycolytic Flux and Contributes to Oncogenesis</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">869</span>, <span class="refDoi"> DOI: 10.1038/ng.890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fng.890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21804546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=869&author=J.%0AW.+Locasaleauthor=A.+R.+Grassianauthor=T.+Melmanauthor=C.+A.+Lyssiotisauthor=K.+R.+Mattainiauthor=A.+J.+Bassauthor=G.+Heffronauthor=C.+M.+Metalloauthor=T.+Muranenauthor=H.+Sharfiauthor=A.+T.+Sasakiauthor=D.+Anastasiouauthor=E.+Mullarkyauthor=N.+I.+Vokesauthor=M.+Sasakiauthor=R.+Beroukhimauthor=G.+Stephanopoulosauthor=A.+H.+Ligonauthor=M.+Meyersonauthor=A.+L.+Richardsonauthor=L.+Chinauthor=G.+Wagnerauthor=J.+M.+Asaraauthor=J.+S.+Bruggeauthor=L.+C.+Cantleyauthor=M.+G.+Vander+Heiden&title=Phosphoglycerate+Dehydrogenase+Diverts+Ggycolytic+Flux+and+Contributes+to+Oncogenesis&doi=10.1038%2Fng.890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoglycerate dehydrogenase diverts glycolytic flux and contributes to oncogenesis</span></div><div class="casAuthors">Locasale, Jason W.; Grassian, Alexandra R.; Melman, Tamar; Lyssiotis, Costas A.; Mattaini, Katherine R.; Bass, Adam J.; Heffron, Gregory; Metallo, Christian M.; Muranen, Taru; Sharfi, Hadar; Sasaki, Atsuo T.; Anastasiou, Dimitrios; Mullarky, Edouard; Vokes, Natalie I.; Sasaki, Mika; Beroukhim, Rameen; Stephanopoulos, Gregory; Ligon, Azra H.; Meyerson, Matthew; Richardson, Andrea L.; Chin, Lynda; Wagner, Gerhard; Asara, John M.; Brugge, Joan S.; Cantley, Lewis C.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">869-874</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most tumors exhibit increased glucose metab. to lactate, however, the extent to which glucose-derived metabolic fluxes are used for alternative processes is poorly understood.  Using a metabolomics approach with isotope labeling, we found that in some cancer cells a relatively large amt. of glycolytic carbon is diverted into serine and glycine metab. through phosphoglycerate dehydrogenase (PHGDH).  An anal. of human cancers showed that PHGDH is recurrently amplified in a genomic region of focal copy no. gain most commonly found in melanoma.  Decreasing PHGDH expression impaired proliferation in amplified cell lines.  Increased expression was also assocd. with breast cancer subtypes, and ectopic expression of PHGDH in mammary epithelial cells disrupted acinar morphogenesis and induced other phenotypic alterations that may predispose cells to transformation.  Our findings show that the diversion of glycolytic flux into a specific alternate pathway can be selected during tumor development and may contribute to the pathogenesis of human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIkj0NPfUByrVg90H21EOLACvtfcHk0lh4NLH1iecQHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLw%253D&md5=b903036172528f331b63ce84694ce851</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fng.890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.890%26sid%3Dliteratum%253Aachs%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DGrassian%26aufirst%3DA.%2BR.%26aulast%3DMelman%26aufirst%3DT.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DBass%26aufirst%3DA.%2BJ.%26aulast%3DHeffron%26aufirst%3DG.%26aulast%3DMetallo%26aufirst%3DC.%2BM.%26aulast%3DMuranen%26aufirst%3DT.%26aulast%3DSharfi%26aufirst%3DH.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DAnastasiou%26aufirst%3DD.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DVokes%26aufirst%3DN.%2BI.%26aulast%3DSasaki%26aufirst%3DM.%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DStephanopoulos%26aufirst%3DG.%26aulast%3DLigon%26aufirst%3DA.%2BH.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DRichardson%26aufirst%3DA.%2BL.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DBrugge%26aufirst%3DJ.%2BS.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DPhosphoglycerate%2520Dehydrogenase%2520Diverts%2520Ggycolytic%2520Flux%2520and%2520Contributes%2520to%2520Oncogenesis%26jtitle%3DNat.%2520Genet.%26date%3D2011%26volume%3D43%26spage%3D869%26doi%3D10.1038%2Fng.890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span> <span> </span><span class="NLM_article-title">PHGDH Amplification and Altered Glucose Metabolism in Human Melanoma: PHGDH Amplification and Altered Glucose Metabolism</span>. <i>Pigm. Cell Melanoma Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1115</span>, <span class="refDoi"> DOI: 10.1111/j.1755-148X.2011.00919.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1111%2Fj.1755-148X.2011.00919.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21981974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1yksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=1112-1115&author=E.+Mullarkyauthor=K.+R.+Mattainiauthor=M.+G.+Vander+Heidenauthor=L.+C.+Cantleyauthor=J.+W.+Locasale&title=PHGDH+Amplification+and+Altered+Glucose+Metabolism+in+Human+Melanoma%3A+PHGDH+Amplification+and+Altered+Glucose+Metabolism&doi=10.1111%2Fj.1755-148X.2011.00919.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH amplification and altered glucose metabolism in human melanoma</span></div><div class="casAuthors">Mullarky, Edouard; Mattaini, Katherine R.; Vander Heiden, Matthew G.; Cantley, Lewis C.; Locasale, Jason W.</div><div class="citationInfo"><span class="NLM_cas:title">Pigment Cell & Melanoma Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1112-1115</span>CODEN:
                <span class="NLM_cas:coden">PCMRCM</span>;
        ISSN:<span class="NLM_cas:issn">1755-1471</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The metabolic requirements of cancer cells differ from that of their normal counterparts.  To support their proliferation, cancer cells switch to a fermentative metab. that is thought to support biomass prodn.  Instances where metabolic enzymes promote tumorigenesis remain rare.  However, an enzyme involved in the de novo synthesis of serine, 3-phosphoglycerate dehydrogenase (PHGDH), was recently identified as a putative oncogene.  The potential mechanisms by which PHGDH promotes cancer are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsn22yOds6rVg90H21EOLACvtfcHk0lj5GZftl_5V2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1yksA%253D%253D&md5=9b30f3f50eb60fd2212be45a7e521130</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1755-148X.2011.00919.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1755-148X.2011.00919.x%26sid%3Dliteratum%253Aachs%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26atitle%3DPHGDH%2520Amplification%2520and%2520Altered%2520Glucose%2520Metabolism%2520in%2520Human%2520Melanoma%253A%2520PHGDH%2520Amplification%2520and%2520Altered%2520Glucose%2520Metabolism%26jtitle%3DPigm.%2520Cell%2520Melanoma%2520Res.%26date%3D2011%26volume%3D24%26spage%3D1112%26epage%3D1115%26doi%3D10.1111%2Fj.1755-148X.2011.00919.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span> <span> </span><span class="NLM_article-title">PHGDH is an Independent Prognosis Marker and Contributes Cell Proliferation, Migration and Invasion in Human Pancreatic Cancer</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>642</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.gene.2017.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.gene.2017.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29128633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2nu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=642&publication_year=2018&pages=43-50&author=Z.+Songauthor=C.+Fengauthor=Y.+Luauthor=Y.+Linauthor=C.+Dong&title=PHGDH+is+an+Independent+Prognosis+Marker+and+Contributes+Cell+Proliferation%2C+Migration+and+Invasion+in+Human+Pancreatic+Cancer&doi=10.1016%2Fj.gene.2017.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH is an independent prognosis marker and contributes cell proliferation, migration and invasion in human pancreatic cancer</span></div><div class="casAuthors">Song, Zhiwang; Feng, Chan; Lu, Yonglin; Lin, Yun; Dong, Chunyan</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">642</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-50</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">To investigate the expression, clin. significance, biol. function, and the potential mechanism of PHGDH in pancreatic cancer.  The expression of PHGDH in human pancreatic cancer tissues and corresponding adjacent normal tissues were analyzed through immunohistochem. staining.  Simultaneously, the assocn. between the PHGDH expression and the clinicopathol. parameters and OS and DFS was evaluated.  Human pancreatic cancer cell line BxPC-3 and SW1990 were selected to investigate the effect of PHGDH knockdown on cell proliferation, migration, and invasion.  In addn., we performed western blot to assess the expression of cyclin B1, and cyclin D1, MMP-2, and MMP-9 protein.  Our results suggested that the expression of PHGDH is increased in pancreatic cancer compared with adjacent normal tissues and the increased expression of PHGDH is assocd. with tumor size, lymph node metastasis, and TNM state of pancreatic cancer patients.  Moreover, the expression of PHGDH is an independent prognostic indicator for pancreatic cancer patients.  In addn., we found that knockdown of PHGDH in pancreatic cancer cells inhibits the cell proliferation, migration, and invasion abilities by down-regulating the expression of cyclin B1, and cyclin D1, MMP-2, and MMP-9.  Our data indicated that the expression of PHGDH is increased in pancreatic cancer and is an independent mol. prognostic factor for pancreatic cancer patients.  In addn., PHGDH controls cell proliferation, migration and invasion abilities.  Therefore, PHGDH could serve as an important prognostic indicator and therapeutic target for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhB-EuBx_5TbVg90H21EOLACvtfcHk0lj5GZftl_5V2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2nu77N&md5=2003892af6215c0e765faf48143db9b9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2017.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2017.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DFeng%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DPHGDH%2520is%2520an%2520Independent%2520Prognosis%2520Marker%2520and%2520Contributes%2520Cell%2520Proliferation%252C%2520Migration%2520and%2520Invasion%2520in%2520Human%2520Pancreatic%2520Cancer%26jtitle%3DGene%26date%3D2018%26volume%3D642%26spage%3D43%26epage%3D50%26doi%3D10.1016%2Fj.gene.2017.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nohata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itesako, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kofuji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahiya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enokida, H.</span></span> <span> </span><span class="NLM_article-title">PHGDH as a Key Enzyme for Serine Biosynthesis in HIF2α-targeting Therapy for Renal Cell Carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6329</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-1589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1158%2F0008-5472.CAN-17-1589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=28951458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=6321-6329&author=H.+Yoshinoauthor=N.+Nohataauthor=K.+Miyamotoauthor=M.+Yonemoriauthor=T.+Sakaguchiauthor=S.+Sugitaauthor=T.+Itesakoauthor=S.+Kofujiauthor=M.+Nakagawaauthor=R.+Dahiyaauthor=H.+Enokida&title=PHGDH+as+a+Key+Enzyme+for+Serine+Biosynthesis+in+HIF2%CE%B1-targeting+Therapy+for+Renal+Cell+Carcinoma&doi=10.1158%2F0008-5472.CAN-17-1589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH as a Key Enzyme for Serine Biosynthesis in HIF2α-Targeting Therapy for Renal Cell Carcinoma</span></div><div class="casAuthors">Yoshino, Hirofumi; Nohata, Nijiro; Miyamoto, Kazutaka; Yonemori, Masaya; Sakaguchi, Takashi; Sugita, Satoshi; Itesako, Toshihiko; Kofuji, Satoshi; Nakagawa, Masayuki; Dahiya, Rajvir; Enokida, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6321-6329</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Continuous activation of hypoxia-inducible factor (HIF) is important for progression of renal cell carcinoma (RCC) and acquired resistance to antiangiogenic multikinase and mTOR inhibitors.  Recently, HIF2α antagonists PT2385 and PT2399 were developed and are being evaluated in a phase I clin. trial for advanced or metastatic clear cell RCC (ccRCC).  However, resistance to HIF2α antagonists would be expected to develop.  In this study, we identified signals activated by HIF2α deficiency as candidate mediators of resistance to the HIF2α antagonists.  We established sunitinib-resistant tumor cells in vivo and created HIF2α-deficient variants of these cells using CRISPR/Cas9 technol.  Mechanistic investigations revealed that a regulator of the serine biosynthesis pathway, phosphoglycerate dehydrogenase (PHGDH), was upregulated commonly in HIF2α-deficient tumor cells along with the serine biosynthesis pathway itself.  Accordingly, treatment with a PHGDH inhibitor reduced the growth of HIF2α-deficient tumor cells in vivo and in vitro by inducing apoptosis.  Our findings identify the serine biosynthesis pathway as a source of candidate therapeutic targets to eradicate advanced or metastatic ccRCC resistant to HIF2α antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi_LDrzdakZLVg90H21EOLACvtfcHk0lj5GZftl_5V2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur7I&md5=adef1b67feb25a551c785471dce76645</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-1589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-1589%26sid%3Dliteratum%253Aachs%26aulast%3DYoshino%26aufirst%3DH.%26aulast%3DNohata%26aufirst%3DN.%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DYonemori%26aufirst%3DM.%26aulast%3DSakaguchi%26aufirst%3DT.%26aulast%3DSugita%26aufirst%3DS.%26aulast%3DItesako%26aufirst%3DT.%26aulast%3DKofuji%26aufirst%3DS.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DDahiya%26aufirst%3DR.%26aulast%3DEnokida%26aufirst%3DH.%26atitle%3DPHGDH%2520as%2520a%2520Key%2520Enzyme%2520for%2520Serine%2520Biosynthesis%2520in%2520HIF2%25CE%25B1-targeting%2520Therapy%2520for%2520Renal%2520Cell%2520Carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D6321%26epage%3D6329%26doi%3D10.1158%2F0008-5472.CAN-17-1589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenner, A.</span></span> <span> </span><span class="NLM_article-title">PHGDH is Key for Targeting HIF in RCC</span>. <i>Nat. Rev. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>, <span class="NLM_elocation-id">702</span> <span class="refDoi"> DOI: 10.1038/nrurol.2017.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnrurol.2017.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29116254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ptVOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&author=A.+Fenner&title=PHGDH+is+Key+for+Targeting+HIF+in+RCC&doi=10.1038%2Fnrurol.2017.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Kidney cancer: PHGDH is key for targeting HIF in RCC</span></div><div class="casAuthors">Fenner Annette</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Urology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">702</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTGMfqbtbJj6X6oPH0mYWL1fW6udTcc2eYO519Yw4qClbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ptVOgtw%253D%253D&md5=f82a7018e8220e42bf7989f0306a80e8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrurol.2017.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrurol.2017.176%26sid%3Dliteratum%253Aachs%26aulast%3DFenner%26aufirst%3DA.%26atitle%3DPHGDH%2520is%2520Key%2520for%2520Targeting%2520HIF%2520in%2520RCC%26jtitle%3DNat.%2520Rev.%2520Urol.%26date%3D2017%26volume%3D14%26doi%3D10.1038%2Fnrurol.2017.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeNicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudderth, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huffman, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBerardinis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">NRF2 Regulates Serine Biosynthesis in Non-small Cell Lung Cancer</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1481</span>, <span class="refDoi"> DOI: 10.1038/ng.3421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fng.3421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=26482881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslant7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2015&pages=1475-1481&author=G.+M.+DeNicolaauthor=P.-H.+Chenauthor=E.+Mullarkyauthor=J.+A.+Sudderthauthor=Z.+Huauthor=D.+Wuauthor=H.+Tangauthor=Y.+Xieauthor=J.+M.+Asaraauthor=K.+E.+Huffmanauthor=I.+I.+Wistubaauthor=J.+D.+Minnaauthor=R.+J.+DeBerardinisauthor=L.+C.+Cantley&title=NRF2+Regulates+Serine+Biosynthesis+in+Non-small+Cell+Lung+Cancer&doi=10.1038%2Fng.3421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">NRF2 regulates serine biosynthesis in non-small cell lung cancer</span></div><div class="casAuthors">DeNicola, Gina M.; Chen, Pei-Hsuan; Mullarky, Edouard; Sudderth, Jessica A.; Hu, Zeping; Wu, David; Tang, Hao; Xie, Yang; Asara, John M.; Huffman, Kenneth E.; Wistuba, Ignacio I.; Minna, John D.; DeBerardinis, Ralph J.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1475-1481</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumors have high energetic and anabolic needs for rapid cell growth and proliferation, and the serine biosynthetic pathway was recently identified as an important source of metabolic intermediates for these processes.  We integrated metabolic tracing and transcriptional profiling of a large panel of non-small cell lung cancer (NSCLC) cell lines to characterize the activity and regulation of the serine/glycine biosynthetic pathway in NSCLC.  Here we show that the activity of this pathway is highly heterogeneous and is regulated by NRF2, a transcription factor frequently deregulated in NSCLC.  We found that NRF2 controls the expression of the key serine/glycine biosynthesis enzyme genes PHGDH, PSAT1 and SHMT2 via ATF4 to support glutathione and nucleotide prodn.  Moreover, we show that expression of these genes confers poor prognosis in human NSCLC.  Thus, a substantial fraction of human NSCLCs activates an NRF2-dependent transcriptional program that regulates serine and glycine metab. and is linked to clin. aggressiveness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSELAUW2hlmbVg90H21EOLACvtfcHk0lhi8pJSLtNzTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslant7bO&md5=41d411b051f2e26da358c949585c0c5d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fng.3421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3421%26sid%3Dliteratum%253Aachs%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DChen%26aufirst%3DP.-H.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DSudderth%26aufirst%3DJ.%2BA.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DHuffman%26aufirst%3DK.%2BE.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DDeBerardinis%26aufirst%3DR.%2BJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DNRF2%2520Regulates%2520Serine%2520Biosynthesis%2520in%2520Non-small%2520Cell%2520Lung%2520Cancer%26jtitle%3DNat.%2520Genet.%26date%3D2015%26volume%3D47%26spage%3D1475%26epage%3D1481%26doi%3D10.1038%2Fng.3421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamanaka, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigdelioglu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meliton, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witt, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansbro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulin, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzy, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutlu, G. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Phosphoglycerate Dehydrogenase Attenuates Bleomycin-induced Pulmonary Fibrosis</span>. <i>Am. J. Respir. Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1165/rcmb.2017-0186OC</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1165%2Frcmb.2017-0186OC" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29019702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyks7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=585-593&author=R.+B.+Hamanakaauthor=R.+Nigdeliogluauthor=A.+Y.+Melitonauthor=Y.+Tianauthor=L.+J.+Wittauthor=E.+O%E2%80%99Learyauthor=K.+A.+Sunauthor=P.+S.+Woodsauthor=D.+Wuauthor=B.+Ansbroauthor=S.+Ardauthor=J.+M.+Rohdeauthor=N.+O.+Dulinauthor=R.+D.+Guzyauthor=G.+M.+Mutlu&title=Inhibition+of+Phosphoglycerate+Dehydrogenase+Attenuates+Bleomycin-induced+Pulmonary+Fibrosis&doi=10.1165%2Frcmb.2017-0186OC"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of phosphoglycerate dehydrogenase attenuates bleomycin-induced pulmonary fibrosis</span></div><div class="casAuthors">Hamanaka, Robert B.; Nigdelioglu, Recep; Meliton, Angelo Y.; Tian, Yufeng; Witt, Leah J.; O'Leary, Erin; Sun, Kaitlyn A.; Woods, Parker S.; Wu, David; Ansbro, Brandon; Ard, Shawn; Rohde, Jason M.; Dulin, Nickolai O.; Guzy, Robert D.; Mutlu, Gokhan M.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Respiratory Cell and Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">585-593</span>CODEN:
                <span class="NLM_cas:coden">AJRBEL</span>;
        ISSN:<span class="NLM_cas:issn">1044-1549</span>.
    
            (<span class="NLM_cas:orgname">American Thoracic Society</span>)
        </div><div class="casAbstract">Organ fibrosis, including idiopathic pulmonary fibrosis, is assocd. with significant morbidity and mortality.  Because currently available therapies have limited effect, there is a need to better understand the mechanisms by which organ fibrosis occurs.  We have recently reported that transforming growth factor (TGF)-β, a key cytokine that promotes fibrogenesis, induces the expression of the enzymes of the de novo serine and glycine synthesis pathway in human lung fibroblasts, and that phosphoglycerate dehydrogenase (PHGDH; the first and rate-limiting enzyme of the pathway) is required to promote collagen protein synthesis downstream of TGF-β.  In this study, we investigated whether inhibition of de novo serine and glycine synthesis attenuates lung fibrosis in vivo.  We found that TGF-β induces mRNA and protein expression of PHGDH in murine fibroblasts.  Similarly, intratracheal administration of bleomycin resulted in increased expression of PHGDH in mouse lungs, localized to fibrotic regions.  Using a newly developed small mol. inhibitor of PHGDH (NCT-503), we tested whether pharmacol. inhibition of PHGDH could inhibit fibrogenesis both in vitro and in vivo.  Treatment of murine and human lung fibroblasts with NCT-503 decreased TGF-β-induced collagen protein synthesis.  Mice treated with the PHGDH inhibitor beginning 7 days after intratracheal instillation of bleomycin had attenuation of lung fibrosis.  These results indicate that the de novo serine and glycine synthesis pathway is necessary for TGF-β-induced collagen synthesis and bleomycin-induced pulmonary fibrosis.  PHGDH and other enzymes in the de novo serine and glycine synthesis pathway may be a therapeutic target for treatment of fibrotic diseases, including idiopathic pulmonary fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrywNwhQatH9bVg90H21EOLACvtfcHk0liZh50UkDHOTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyks7jE&md5=acb85b0fe79cb75162af642eb4932442</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1165%2Frcmb.2017-0186OC&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1165%252Frcmb.2017-0186OC%26sid%3Dliteratum%253Aachs%26aulast%3DHamanaka%26aufirst%3DR.%2BB.%26aulast%3DNigdelioglu%26aufirst%3DR.%26aulast%3DMeliton%26aufirst%3DA.%2BY.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DWitt%26aufirst%3DL.%2BJ.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DK.%2BA.%26aulast%3DWoods%26aufirst%3DP.%2BS.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DAnsbro%26aufirst%3DB.%26aulast%3DArd%26aufirst%3DS.%26aulast%3DRohde%26aufirst%3DJ.%2BM.%26aulast%3DDulin%26aufirst%3DN.%2BO.%26aulast%3DGuzy%26aufirst%3DR.%2BD.%26aulast%3DMutlu%26aufirst%3DG.%2BM.%26atitle%3DInhibition%2520of%2520Phosphoglycerate%2520Dehydrogenase%2520Attenuates%2520Bleomycin-induced%2520Pulmonary%2520Fibrosis%26jtitle%3DAm.%2520J.%2520Respir.%2520Cell%2520Mol.%2520Biol.%26date%3D2018%26volume%3D58%26spage%3D585%26epage%3D593%26doi%3D10.1165%2Frcmb.2017-0186OC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hydbring, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambroise, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouchida, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goiny, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakifahmetoglu-Norberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norberg, E.</span></span> <span> </span><span class="NLM_article-title">PHGDH Defines a Metabolic Subtype in Lung Adenocarcinomas with Poor Prognosis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2289</span>– <span class="NLM_lpage">2303</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.05.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.celrep.2017.05.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=28614715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVaku7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=2289-2303&author=B.+Zhangauthor=A.+Zhengauthor=P.+Hydbringauthor=G.+Ambroiseauthor=A.+T.+Ouchidaauthor=M.+Goinyauthor=H.+Vakifahmetoglu-Norbergauthor=E.+Norberg&title=PHGDH+Defines+a+Metabolic+Subtype+in+Lung+Adenocarcinomas+with+Poor+Prognosis&doi=10.1016%2Fj.celrep.2017.05.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH Defines a Metabolic Subtype in Lung Adenocarcinomas with Poor Prognosis</span></div><div class="casAuthors">Zhang, Boxi; Zheng, Adi; Hydbring, Per; Ambroise, Gorbatchev; Ouchida, Amanda Tomie; Goiny, Michel; Vakifahmetoglu-Norberg, Helin; Norberg, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2289-2303</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mol. signatures are emerging determinants of choice of therapy for lung adenocarcinomas.  An evolving therapeutic approach includes targeting metabolic dependencies in cancers.  Here, using an integrative approach, we have dissected the metabolic fingerprints of lung adenocarcinomas, and we show that Phosphoglycerate dehydrogenase (PHGDH), the rate-limiting enzyme in serine biosynthesis, is highly expressed in a adenocarcinoma subset with poor prognosis.  This subset harbors a gene signature for DNA replication and proliferation.  Accordingly, models with high levels of PHGDH display rapid proliferation, migration, and selective channeling of serine-derived carbons to glutathione and pyrimidines, while depletion of PHGDH shows potent and selective toxicity to this subset.  Differential PHGDH protein levels were defined by its degrdn., and the deubiquitinating enzyme JOSD2 is a regulator of its protein stability.  Our study provides evidence that a unique metabolic program is activated in a lung adenocarcinoma subset, described by PHGDH, which confers growth and survival and may have therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuv8KYzWmPzrVg90H21EOLACvtfcHk0lhfCctBaTJohg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVaku7nP&md5=d6100f48feeda662452dc18732ac731a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.05.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.05.067%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DA.%26aulast%3DHydbring%26aufirst%3DP.%26aulast%3DAmbroise%26aufirst%3DG.%26aulast%3DOuchida%26aufirst%3DA.%2BT.%26aulast%3DGoiny%26aufirst%3DM.%26aulast%3DVakifahmetoglu-Norberg%26aufirst%3DH.%26aulast%3DNorberg%26aufirst%3DE.%26atitle%3DPHGDH%2520Defines%2520a%2520Metabolic%2520Subtype%2520in%2520Lung%2520Adenocarcinomas%2520with%2520Poor%2520Prognosis%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D2289%26epage%3D2303%26doi%3D10.1016%2Fj.celrep.2017.05.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, J. S.</span></span> <span> </span><span class="NLM_article-title">Expression Levels of Serine/Glycine Metabolism-related Proteins in Triple Negative Breast Cancer Tissues</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">4457</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1588-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1007%2Fs13277-013-1588-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=24390667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlOrtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=4457-4468&author=S.+Nohauthor=D.+H.+Kimauthor=W.+H.+Jungauthor=J.+S.+Koo&title=Expression+Levels+of+Serine%2FGlycine+Metabolism-related+Proteins+in+Triple+Negative+Breast+Cancer+Tissues&doi=10.1007%2Fs13277-013-1588-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Expression levels of serine/glycine metabolism-related proteins in triple negative breast cancer tissues</span></div><div class="casAuthors">Noh, Songmi; Kim, Do Hee; Jung, Woo Hee; Koo, Ja Seung</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4457-4468</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To evaluate the expression levels of serine/glycine metab.-related proteins (PHGDH, PSAT, PSPH, SHMT, and GLDC) in six different subtypes of triple neg. breast cancer (TNBC) patients and gain insight into their implications.  Formalin-fixed, paraffin-embedded tissues from 129 TNBC patients were assembled into tissue microarrays.  Immunohistochem. staining was performed for serine/glycine metab.-related proteins (PHGDH, PSAT, PSPH, SHMT, and GLDC) and surrogate immunohistochem. markers (CK5/6, EGFR, claudin 3, claudin 4, claudin 7, E-cadherin, STAT1, interleukin-8 [IL-8], AR, and GGT-1) for identifying the mol. subtype of TNBC.  TNBC subtype classifications included the following: basal-like (CK5/6-pos. and/or EGFR-pos.), mol. apocrine (AR-pos. and/or GGT-1-pos.), claudin-low (claudin 3-, claudin 4-, claudin 7-neg. and/or E-cadherin-neg.), immune-related (IL-8-neg. and stromal STAT1-pos.), mixed (features from two or more of the four subtypes), and null (no features from any of the four subtypes).  Tissues from basal marker-pos. patients showed increased expression levels of tumoral PHGDH compared with those from basal marker-neg. patients (p = 0.029); lack of stromal SHMT1 expression was significantly correlated with T stage (p = 0.016).  Multivariate Cox anal. revealed that a lack of stromal SHMT1 expression was an independent prognostic factor for predicting a shorter disease-free survival period (hazard ratio 4.002, 95 % confidence interval [CI] 1.077-14.83, p = 0.038); furthermore, a lack of tumoral PHGDH expression was predictive of a shorter overall survival rate (hazard ratio 3.053, 95 % CI 1.002-9.305, p = 0.050).  In conclusion, the most abundantly expressed serine/glycine metab.-related protein in basal-like TNBC tissues was tumoral PHGDH, and expression levels of stromal SHMT1 and tumoral PHGDH were inversely correlated with clin. prognostic factors.  Also, this study is the first to assess serine/glycine relationships at the protein level in regards to clin. outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbJy-OZWfJBLVg90H21EOLACvtfcHk0lhfCctBaTJohg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlOrtL0%253D&md5=fa67446343d9ebeefe392cca0b1777c3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1588-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1588-z%26sid%3Dliteratum%253Aachs%26aulast%3DNoh%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DD.%2BH.%26aulast%3DJung%26aufirst%3DW.%2BH.%26aulast%3DKoo%26aufirst%3DJ.%2BS.%26atitle%3DExpression%2520Levels%2520of%2520Serine%252FGlycine%2520Metabolism-related%2520Proteins%2520in%2520Triple%2520Negative%2520Breast%2520Cancer%2520Tissues%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D4457%26epage%3D4468%26doi%3D10.1007%2Fs13277-013-1588-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiang, J.-F.</span></span> <span> </span><span class="NLM_article-title">Increased Expression of PHGDH and Prognostic Significance in Colorectal Cancer</span>. <i>Transl. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1016/j.tranon.2016.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.tranon.2016.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27267836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FnvVertw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=191-196&author=X.-Q.+Jiaauthor=S.+Zhangauthor=H.-J.+Zhuauthor=W.+Wangauthor=J.-H.+Zhuauthor=X.-D.+Wangauthor=J.-F.+Qiang&title=Increased+Expression+of+PHGDH+and+Prognostic+Significance+in+Colorectal+Cancer&doi=10.1016%2Fj.tranon.2016.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Increased Expression of PHGDH and Prognostic Significance in Colorectal Cancer</span></div><div class="casAuthors">Jia Xiao-Qin; Zhang Shu; Zhu Hui-Jun; Wang Wei; Zhu Jin-Hong; Wang Xu-Dong; Qiang Jian-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Translational oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-6</span>
        ISSN:<span class="NLM_cas:issn">1936-5233</span>.
    </div><div class="casAbstract">Phosphoglycerate dehydrogenase (PHGDH) plays an essential role in cancer-specific metabolic reprogramming.  It has been reported as a putative metabolic oncogene in several types of human malignant tumors, such as breast cancer and melanoma.  To date, PHGDH expression in colorectal cancer (CRC) as well as its association with clinicopathological characteristics and prognostic implication remain undetermined.  In this study, we determined the PHGDH protein expression using tissue microarray immunohistochemistry (TMA-IHC) on 193 pairs of formalin-fixed, paraffin-embedded specimens of CRC and adjacent tissues, 25 chronic colitis, 41 low-, and 19 high-grade intraepithelial neoplasia specimens, and we also determined PHGDH mRNA level using quantitative reverse transcription PCR (qRT-PCR) on additional 23 pairs of fresh CRC tissues and adjacent tissues.  We found that both PHGDH mRNA and protein was highly expressed in tumor tissues in comparison with matched adjacent non-tumor tissues, and high PHGDH protein expression was correlated with advanced TNM stage (P = .038) and larger tumor (P = .001).  Multivariate Cox regression analysis showed that PHGDH protein expression (HR = 2.285, 95% CI = 1.18 to 4.41, P = .014), tumor differentiation (HR = .307, 95% CI = .154 to 0.609, P = .001), and TNM stage (HR = 1.791, 95% CI = 1.125 to 2.85, P = .014) were independent prognostic factors in CRC.  Kaplan-Meier survival curves and log rank test showed that high PHGDH protein expression contributed to poor outcome in CRC patients (P < .001).  In conclusion, these results suggest that assessment of PHGDH expression could be useful in identifying a high-risk subgroup of CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2rz7sre8coyEJCPKhJPuqfW6udTcc2eZdTBbNypf3Wrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FnvVertw%253D%253D&md5=242366977328c32161960259ee1d4fae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tranon.2016.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tranon.2016.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DX.-Q.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DH.-J.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DJ.-H.%26aulast%3DWang%26aufirst%3DX.-D.%26aulast%3DQiang%26aufirst%3DJ.-F.%26atitle%3DIncreased%2520Expression%2520of%2520PHGDH%2520and%2520Prognostic%2520Significance%2520in%2520Colorectal%2520Cancer%26jtitle%3DTransl.%2520Oncol.%26date%3D2016%26volume%3D9%26spage%3D191%26epage%3D196%26doi%3D10.1016%2Fj.tranon.2016.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Ixocarpalactone A from Dietary Tomatillo Inhibits Pancreatic Cancer Growth by Targeting PHGDH</span>. <i>Food Funct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">3386</span>, <span class="refDoi"> DOI: 10.1039/C9FO00394K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1039%2FC9FO00394K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=31112178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvF2jtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=3386&author=M.+Zhengauthor=J.+Guoauthor=J.+Xuauthor=K.+Yangauthor=R.+Tangauthor=X.+Guauthor=H.+Liauthor=L.+Chen&title=Ixocarpalactone+A+from+Dietary+Tomatillo+Inhibits+Pancreatic+Cancer+Growth+by+Targeting+PHGDH&doi=10.1039%2FC9FO00394K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Ixocarpalactone A from dietary tomatillo inhibits pancreatic cancer growth by targeting PHGDH</span></div><div class="casAuthors">Zheng, Mengzhu; Guo, Jing; Xu, Jiamin; Yang, Kaiyin; Tang, Ruotian; Gu, Xiaoxia; Li, Hua; Chen, Lixia</div><div class="citationInfo"><span class="NLM_cas:title">Food & Function</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3386-3395</span>CODEN:
                <span class="NLM_cas:coden">FFOUAI</span>;
        ISSN:<span class="NLM_cas:issn">2042-6496</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">3-Phosphoglycerate dehydrogenase (PHGDH) catalyzes the first rate-limiting step for the synthesis of glucose-derived serine by converting 3-phosphoglycerate (3-PG) to phosphohydroxypyruvate (p-Pyr), which has been reported to assoc. with tumorigenesis in many cancers.  Iox A, a natural withanolide obtained from dietary tomatillo (Physalis ixocarpa), showed significant PHGDH inhibitory activity with an IC50 value of 1.66 ± 0.28 μM, and it was further confirmed to bind directly to PHGDH by the MST assay.  Mol. docking demonstrated that Iox A coordinated at the allosteric site of PHGDH, which was consistent with the non-competitive kinetics.  Meanwhile, Iox A selectively inhibited the proliferation of high PHGDH-expressing cancer cell lines (SW1990, MCF-7 and HeLa) and showed no obvious cytotoxicities on normal human cells (LO2, L929 and HPDE6-C7).  In particular, Iox A showed a dose-dependent proapoptotic activity against SW1990 cells in a micromolar concn. as detected by flow cytometry and western blot anal.  Furthermore, Iox A significantly inhibited the tumor growth in a SW1990 xenograft mouse model with low toxicities, suggesting its potential therapeutic application in pancreatic cancer treatment.  Therefore, Iox A was identified as a novel natural PHGDH inhibitor with high targeting and low toxicities for the treatment of pancreatic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRpi-AH_ojUrVg90H21EOLACvtfcHk0li0IqdOoJhcQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvF2jtrc%253D&md5=edc1af7499bc0c6d72e60ab8f0d67142</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC9FO00394K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9FO00394K%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DTang%26aufirst%3DR.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DIxocarpalactone%2520A%2520from%2520Dietary%2520Tomatillo%2520Inhibits%2520Pancreatic%2520Cancer%2520Growth%2520by%2520Targeting%2520PHGDH%26jtitle%3DFood%2520Funct.%26date%3D2019%26spage%3D3386%26doi%3D10.1039%2FC9FO00394K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span> <span> </span><span class="NLM_article-title">Serine, Glycine and the One-Carbon Cycle: Cancer Metabolism in Full Circle</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1038/nrc3557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnrc3557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=23822983" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=572-583&author=J.+W.+Locasale&title=Serine%2C+Glycine+and+the+One-Carbon+Cycle%3A+Cancer+Metabolism+in+Full+Circle&doi=10.1038%2Fnrc3557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Serine, glycine and one-carbon units: cancer metabolism in full circle</span></div><div class="casAuthors">Locasale, Jason W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">572-583</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  One-carbon metab. involving the folate and methionine cycles integrates nutritional status from amino acids, glucose and vitamins, and generates diverse outputs, such as the biosynthesis of lipids, nucleotides and proteins, the maintenance of redox status and the substrates for methylation reactions.  Long considered a 'housekeeping' process, this pathway has recently been shown to have addnl. complexity.  Genetic and functional evidence suggests that hyperactivation of this pathway is a driver of oncogenesis and establishes a link to cellular epigenetic status.  Given the wealth of clin. available agents that target one-carbon metab., these new findings could present opportunities for translation into precision cancer medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro6rIkiwOb8LVg90H21EOLACvtfcHk0li0IqdOoJhcQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsLzE&md5=96e34f61a4dd32c09cc196493804b5fc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc3557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3557%26sid%3Dliteratum%253Aachs%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26atitle%3DSerine%252C%2520Glycine%2520and%2520the%2520One-Carbon%2520Cycle%253A%2520Cancer%2520Metabolism%2520in%2520Full%2520Circle%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D572%26epage%3D583%26doi%3D10.1038%2Fnrc3557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">The Importance of Serine Metabolism in Cancer</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>214</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1083/jcb.201604085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1083%2Fjcb.201604085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27458133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVykt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=214&publication_year=2016&pages=249-257&author=K.+R.+Mattainiauthor=M.+R.+Sullivanauthor=M.+G.+Vander%0AHeiden&title=The+Importance+of+Serine+Metabolism+in+Cancer&doi=10.1083%2Fjcb.201604085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The importance of serine metabolism in cancer</span></div><div class="casAuthors">Mattaini, Katherine R.; Sullivan, Mark R.; Heiden, Matthew G. Vander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">214</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-257</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">1540-8140</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Serine metab. is frequently dysregulated in cancers; however, the benefit that this confers to tumors remains controversial.  In many cases, extracellular serine alone is sufficient to support cancer cell proliferation, whereas some cancer cells increase serine synthesis from glucose and require de novo serine synthesis even in the presence of abundant extracellular serine.  Recent studies cast new light on the role of serine metab. in cancer, suggesting that active serine synthesis might be required to facilitate amino acid transport, nucleotide synthesis, folate metab., and redox homeostasis in a manner that impacts cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUVt9Uk-a5grVg90H21EOLACvtfcHk0ljzQE80iF7c0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVykt7nM&md5=3828ce3d50390d59aa82c5d7d0618175</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201604085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201604085%26sid%3Dliteratum%253Aachs%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DSullivan%26aufirst%3DM.%2BR.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DThe%2520Importance%2520of%2520Serine%2520Metabolism%2520in%2520Cancer%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2016%26volume%3D214%26spage%3D249%26epage%3D257%26doi%3D10.1083%2Fjcb.201604085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tedeschi, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markert, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gounder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvorzhinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolfi, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L. L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPaola, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirshfield, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boros, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertino, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oltvai, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, A.</span></span> <span> </span><span class="NLM_article-title">Contribution of Serine, Folate and Glycine Metabolism to the ATP, NADPH and Purine Requirements of Cancer Cells</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e877</span> <span class="refDoi"> DOI: 10.1038/cddis.2013.393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fcddis.2013.393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=24157871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OlurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&author=P.+M.+Tedeschiauthor=E.+K.+Markertauthor=M.+Gounderauthor=H.+Linauthor=D.+Dvorzhinskiauthor=S.+C.+Dolfiauthor=L.+L.-Y.+Chanauthor=J.+Qiuauthor=R.+S.+DiPaolaauthor=K.+M.+Hirshfieldauthor=L.+G.+Borosauthor=J.+R.+Bertinoauthor=Z.+N.+Oltvaiauthor=A.+Vazquez&title=Contribution+of+Serine%2C+Folate+and+Glycine+Metabolism+to+the+ATP%2C+NADPH+and+Purine+Requirements+of+Cancer+Cells&doi=10.1038%2Fcddis.2013.393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of serine, folate and glycine metabolism to the ATP, NADPH and purine requirements of cancer cells</span></div><div class="casAuthors">Tedeschi, P. M.; Markert, E. K.; Gounder, M.; Lin, H.; Dvorzhinski, D.; Dolfi, S. C.; Chan, L. L.-Y.; Qiu, J.; DiPaola, R. S.; Hirshfield, K. M.; Boros, L. G.; Bertino, J. R.; Oltvai, Z. N.; Vazquez, A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">e877</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent observations on cancer cell metab. indicate increased serine synthesis from glucose as a marker of poor prognosis.  We have predicted that a fraction of the synthesized serine is routed to a pathway for ATP prodn.  The pathway is composed by reactions from serine synthesis, one-carbon (folate) metab. and the glycine cleavage system (SOG pathway).  Here we show that the SOG pathway is upregulated at the level of gene expression in a subset of human tumors and that its level of expression correlates with gene signatures of cell proliferation and Myc target activation.  We have also estd. the SOG pathway metabolic flux in the NCI60 tumor-derived cell lines, using previously reported exchange fluxes and a personalized model of cell metab.  We find that the estd. rates of reactions in the SOG pathway are highly correlated with the proliferation rates of these cell lines.  We also observe that the SOG pathway contributes significantly to the energy requirements of biosynthesis, to the NADPH requirement for fatty acid synthesis and to the synthesis of purines.  Finally, when the PC-3 prostate cancer cell line is treated with the antifolate methotrexate, we observe a decrease in the ATP levels, AMP kinase activation and a decrease in ribonucleotides and fatty acids synthesized from [1,2-13C2]-D-glucose as the single tracer.  Taken together our results indicate that the SOG pathway activity increases with the rate of cell proliferation and it contributes to the biosynthetic requirements of purines, ATP and NADPH of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPNkAiO41cULVg90H21EOLACvtfcHk0ljzQE80iF7c0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OlurfI&md5=eebf6e0461df9e4415a78ca4dd51328c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.393%26sid%3Dliteratum%253Aachs%26aulast%3DTedeschi%26aufirst%3DP.%2BM.%26aulast%3DMarkert%26aufirst%3DE.%2BK.%26aulast%3DGounder%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DDvorzhinski%26aufirst%3DD.%26aulast%3DDolfi%26aufirst%3DS.%2BC.%26aulast%3DChan%26aufirst%3DL.%2BL.-Y.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DDiPaola%26aufirst%3DR.%2BS.%26aulast%3DHirshfield%26aufirst%3DK.%2BM.%26aulast%3DBoros%26aufirst%3DL.%2BG.%26aulast%3DBertino%26aufirst%3DJ.%2BR.%26aulast%3DOltvai%26aufirst%3DZ.%2BN.%26aulast%3DVazquez%26aufirst%3DA.%26atitle%3DContribution%2520of%2520Serine%252C%2520Folate%2520and%2520Glycine%2520Metabolism%2520to%2520the%2520ATP%252C%2520NADPH%2520and%2520Purine%2520Requirements%2520of%2520Cancer%2520Cells%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2013%26volume%3D4%26doi%3D10.1038%2Fcddis.2013.393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reid, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberti, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span> <span> </span><span class="NLM_article-title">Serine Synthesis Through PHGDH Coordinates Nucleotide Levels by Maintaining Central Carbon Metabolism</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">5442</span> <span class="refDoi"> DOI: 10.1038/s41467-018-07868-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fs41467-018-07868-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=30575741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BB3cnks1ejtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&author=M.+A.+Reidauthor=A.+E.+Allenauthor=S.+Liuauthor=M.+V.+Libertiauthor=P.+Liuauthor=X.+Liuauthor=Z.+Daiauthor=X.+Gaoauthor=Q.+Wangauthor=Y.+Liuauthor=L.+Laiauthor=J.+W.+Locasale&title=Serine+Synthesis+Through+PHGDH+Coordinates+Nucleotide+Levels+by+Maintaining+Central+Carbon+Metabolism&doi=10.1038%2Fs41467-018-07868-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Serine synthesis through PHGDH coordinates nucleotide levels by maintaining central carbon metabolism</span></div><div class="casAuthors">Reid Michael A; Allen Annamarie E; Liu Shiyu; Liberti Maria V; Liu Xiaojing; Dai Ziwei; Gao Xia; Locasale Jason W; Liu Pei; Wang Qian; Liu Ying; Lai Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5442</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phosphoglycerate dehydrogenase (PHGDH) catalyzes the committed step in de novo serine biosynthesis.  Paradoxically, PHGDH and serine synthesis are required in the presence of abundant environmental serine even when serine uptake exceeds the requirements for nucleotide synthesis.  Here, we establish a mechanism for how PHGDH maintains nucleotide metabolism.  We show that inhibition of PHGDH induces alterations in nucleotide metabolism independent of serine utilization.  These changes are not attributable to defects in serine-derived nucleotide synthesis and redox maintenance, another key aspect of serine metabolism, but result from disruption of mass balance within central carbon metabolism.  Mechanistically, this leads to simultaneous alterations in both the pentose phosphate pathway and the tri-carboxylic acid cycle, as we demonstrate based on a quantitative model.  These findings define a mechanism whereby disruption of one metabolic pathway induces toxicity by simultaneously affecting the activity of multiple related pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfmLTc2J-j04kFb3mqPUYDfW6udTcc2eZCZWU2tc1FCbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cnks1ejtQ%253D%253D&md5=e0d90406b6274becc65233dd4fab768b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07868-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07868-6%26sid%3Dliteratum%253Aachs%26aulast%3DReid%26aufirst%3DM.%2BA.%26aulast%3DAllen%26aufirst%3DA.%2BE.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiberti%26aufirst%3DM.%2BV.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26atitle%3DSerine%2520Synthesis%2520Through%2520PHGDH%2520Coordinates%2520Nucleotide%2520Levels%2520by%2520Maintaining%2520Central%2520Carbon%2520Metabolism%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26doi%3D10.1038%2Fs41467-018-07868-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, W.</span></span> <span> </span><span class="NLM_article-title">Repression of Phosphoglycerate Dehydrogenase Sensitizes Triple-negative Breast Cancer to Doxorubicin</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1007/s00280-016-3117-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1007%2Fs00280-016-3117-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27473325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12ls7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2016&pages=655-659&author=X.+Zhangauthor=W.+Bai&title=Repression+of+Phosphoglycerate+Dehydrogenase+Sensitizes+Triple-negative+Breast+Cancer+to+Doxorubicin&doi=10.1007%2Fs00280-016-3117-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Repression of phosphoglycerate dehydrogenase sensitizes triple-negative breast cancer to doxorubicin</span></div><div class="casAuthors">Zhang, Xia; Bai, Weijun</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Approx. 70 % of triple-neg. breast cancer (TNBC) cell lines are identified to upregulate phosphoglycerate dehydrogenase (PHGDH), which regulates the intracellular synthesis of serine and glycine, and promotes tumor growth.  In this work, the impact of this pathway on doxorubicin efficacy was evaluated.  Methods: MDA-MB-468, BT-20 and HCC70 cells were transfected with lentiviral vectors expressing short hairpin RNA (shRNA) against PHGDH.  In response to doxorubicin treatment, cellular proliferation was measured, ROS were evaluated and intracellular levels of serine, glycine and glutathione (GSH) were detd. using liq. chromatog.-mass spectrometry.  A TNBC orthotopic tumor model was used to examine the effect of PHGDH on doxorubicin efficacy in vivo.  Results: TNBC cells exposed to doxorubicin undergo metabolic remodeling, resulting in increased glucose flux for serine synthesis regulated by PHGDH.  Serine is then converted into GSH, which counters doxorubicin-induced formation of ROS.  Consequently, suppression of PHGDH by the use of the shRNA caused doxorubicin-induced oxidative stress and increased doxorubicin sensitivity.  The enhancement of doxorubicin efficacy through simultaneous suppression of PHGDH was validated in a mouse tumor model.  Conclusion: These results shed light on PHGDH that could be a promising target for increasing the effectiveness of chemotherapy in patients with TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5VjY1dAeQz7Vg90H21EOLACvtfcHk0lgGxVdDDLrfrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12ls7zL&md5=5ce16b5bdd652a1e6aaa3e4f427b431e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs00280-016-3117-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-016-3117-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DW.%26atitle%3DRepression%2520of%2520Phosphoglycerate%2520Dehydrogenase%2520Sensitizes%2520Triple-negative%2520Breast%2520Cancer%2520to%2520Doxorubicin%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2016%26volume%3D78%26spage%3D655%26epage%3D659%26doi%3D10.1007%2Fs00280-016-3117-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looper, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span> <span> </span><span class="NLM_article-title">Human Phosphoglycerate Dehydrogenase Produces the Oncometabolite d-2-Hydroxyglutarate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">510</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/cb500683c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500683c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGkurfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=510-516&author=J.+Fanauthor=X.+Tengauthor=L.+Liuauthor=K.+R.+Mattainiauthor=R.+E.+Looperauthor=M.+G.+Vander+Heidenauthor=J.+D.+Rabinowitz&title=Human+Phosphoglycerate+Dehydrogenase+Produces+the+Oncometabolite+d-2-Hydroxyglutarate&doi=10.1021%2Fcb500683c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Human phosphoglycerate dehydrogenase produces the oncometabolite D-2-Hydroxyglutarate</span></div><div class="casAuthors">Fan, Jing; Teng, Xin; Liu, Ling; Mattaini, Katherine R.; Looper, Ryan E.; Vander Heiden, Matthew G.; Rabinowitz, Joshua D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">510-516</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human D-3-phosphoglycerate dehydrogenase (PHGDH), the first enzyme in the serine biosynthetic pathway, is genomically amplified in tumors including breast cancer and melanoma.  In PHGDH-amplified cancer cells, knockdown of PHGDH is not fully rescued by exogenous serine, suggesting possible addnl. growth-promoting roles for the enzyme.  Here we show that, in addn. to catalyzing oxidn. of 3-phosphoglycerate, PHGDH catalyzes NADH-dependent redn. of α-ketoglutarate (AKG) to the oncometabolite D-2-hydroxyglutarate (D-2HG).  Knockdown of PHGDH decreased cellular 2HG by approx. 50% in the PHGDH-amplified breast cancer cell lines MDA-MB-468 (normal concn. 93 μM) and BT-20 (normal concn. 35 μM) and overexpression of PHGDH increased cellular 2HG by over 2-fold in non-PHGDH-amplified MDA-MB-231 breast cancer cells, which normally display very low PHGDH expression.  The reduced 2HG level in PHGDH knockdown cell lines can be rescued by PHGDH re-expression, but not by a catalytically inactive PHGDH mutant.  The initial connection between cancer and D-2HG involved prodn. of high levels of D-2HG by mutant isocitrate dehydrogenase.  More recently, however, elevated D-2HG has been obsd. in breast cancer tumors without isocitrate dehydrogenase mutation.  Our results suggest that PHGDH is one source of this D-2HG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWrcBz9QWsbVg90H21EOLACvtfcHk0lgGxVdDDLrfrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGkurfE&md5=20fc923706b5a481c8aa85670c626743</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb500683c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500683c%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DLooper%26aufirst%3DR.%2BE.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26atitle%3DHuman%2520Phosphoglycerate%2520Dehydrogenase%2520Produces%2520the%2520Oncometabolite%2520d-2-Hydroxyglutarate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D510%26epage%3D516%26doi%3D10.1021%2Fcb500683c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagwat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasanthakumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallman, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolniak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paietta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löwenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godley, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valk, P. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">Leukemic IDH1 and IDH2 Mutations Result in a Hypermethylation Phenotype, Disrupt TET2 Function, and Impair Hematopoietic Differentiation</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.ccr.2010.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21130701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGhsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=553-567&author=M.+E.+Figueroaauthor=O.+Abdel-Wahabauthor=C.+Luauthor=P.+S.+Wardauthor=J.+Patelauthor=A.+Shihauthor=Y.+Liauthor=N.+Bhagwatauthor=A.+Vasanthakumarauthor=H.+F.+Fernandezauthor=M.+S.+Tallmanauthor=Z.+Sunauthor=K.+Wolniakauthor=J.+K.+Peetersauthor=W.+Liuauthor=S.+E.+Choeauthor=V.+R.+Fantinauthor=E.+Paiettaauthor=B.+L%C3%B6wenbergauthor=J.+D.+Lichtauthor=L.+A.+Godleyauthor=R.+Delwelauthor=P.+J.+M.+Valkauthor=C.+B.+Thompsonauthor=R.+L.+Levineauthor=A.+Melnick&title=Leukemic+IDH1+and+IDH2+Mutations+Result+in+a+Hypermethylation+Phenotype%2C+Disrupt+TET2+Function%2C+and+Impair+Hematopoietic+Differentiation&doi=10.1016%2Fj.ccr.2010.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Leukemic IDH1 and IDH2 mutations result in a hypermethylation phenotype, disrupt TET2 function, and impair hematopoietic differentiation</span></div><div class="casAuthors">Figueroa, Maria E.; Abdel-Wahab, Omar; Lu, Chao; Ward, Patrick S.; Patel, Jay; Shih, Alan; Li, Yushan; Bhagwat, Neha; Vasanthakumar, Aparna; Fernandez, Hugo F.; Tallman, Martin S.; Sun, Zhuoxin; Wolniak, Kristy; Peeters, Justine K.; Liu, Wei; Choe, Sung E.; Fantin, Valeria R.; Paietta, Elisabeth; Loewenberg, Bob; Licht, Jonathan D.; Godley, Lucy A.; Delwel, Ruud; Valk, Peter J. M.; Thompson, Craig B.; Levine, Ross L.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">553-567</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Cancer-assocd. IDH mutations are characterized by neomorphic enzyme activity and resultant 2-hydroxyglutarate (2HG) prodn.  Mutational and epigenetic profiling of a large acute myeloid leukemia (AML) patient cohort revealed that IDH1/2-mutant AMLs display global DNA hypermethylation and a specific hypermethylation signature.  Furthermore, expression of 2HG-producing IDH alleles in cells induced global DNA hypermethylation.  In the AML cohort, IDH1/2 mutations were mutually exclusive with mutations in the α-ketoglutarate-dependent enzyme TET2, and TET2 loss-of-function mutations were assocd. with similar epigenetic defects as IDH1/2 mutants.  Consistent with these genetic and epigenetic data, expression of IDH mutants impaired TET2 catalytic function in cells.  Finally, either expression of mutant IDH1/2 or Tet2 depletion impaired hematopoietic differentiation and increased stem/progenitor cell marker expression, suggesting a shared proleukemogenic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mMhJR64Z4rVg90H21EOLACvtfcHk0lhouUH9TnEpVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGhsbvJ&md5=f058c7dbd2efd6a25f4044506b5a2ef3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DFigueroa%26aufirst%3DM.%2BE.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DWard%26aufirst%3DP.%2BS.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DShih%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBhagwat%26aufirst%3DN.%26aulast%3DVasanthakumar%26aufirst%3DA.%26aulast%3DFernandez%26aufirst%3DH.%2BF.%26aulast%3DTallman%26aufirst%3DM.%2BS.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DWolniak%26aufirst%3DK.%26aulast%3DPeeters%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChoe%26aufirst%3DS.%2BE.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DPaietta%26aufirst%3DE.%26aulast%3DL%25C3%25B6wenberg%26aufirst%3DB.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DGodley%26aufirst%3DL.%2BA.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DValk%26aufirst%3DP.%2BJ.%2BM.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DLeukemic%2520IDH1%2520and%2520IDH2%2520Mutations%2520Result%2520in%2520a%2520Hypermethylation%2520Phenotype%252C%2520Disrupt%2520TET2%2520Function%252C%2520and%2520Impair%2520Hematopoietic%2520Differentiation%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D18%26spage%3D553%26epage%3D567%26doi%3D10.1016%2Fj.ccr.2010.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borodovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seltzer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riggins, G. J.</span></span> <span> </span><span class="NLM_article-title">Altered Cancer Cell Metabolism in Gliomas with Mutant IDH1 or IDH2</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">83</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1097/CCO.0b013e32834d816a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1097%2FCCO.0b013e32834d816a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=22080945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Shu7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=83-89&author=A.+Borodovskyauthor=M.+J.+Seltzerauthor=G.+J.+Riggins&title=Altered+Cancer+Cell+Metabolism+in+Gliomas+with+Mutant+IDH1+or+IDH2&doi=10.1097%2FCCO.0b013e32834d816a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Altered cancer cell metabolism in gliomas with mutant IDH1 or IDH2</span></div><div class="casAuthors">Borodovsky, Alexandra; Seltzer, Meghan J.; Riggins, Gregory J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-89</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: IDH1/2 mutations occur in up to 70% of low-grade gliomas and secondary glioblastomas.  Mutation of these enzymes reduces the wildtype function of the enzyme (conversion of isocitrate to α-ketoglutarate) while conferring a new enzymic function, the prodn. of D-2-hydroxyglutarate (D-2-HG) from α-ketoglutarate (α-KG).  However, it is unclear how these enzymic changes contribute to tumorigenesis.  Here, we discuss the recent studies that demonstrate how IDH1/2 mutation may alter the metab. and epigenome of gliomas, how these changes may contribute to tumor formation, and opportunities they might provide for mol. targeting.  Recent findings: Metabolomic studies of IDH1/2 mutant cells have revealed alterations in glutamine, fatty acid, and citrate synthesis pathways.  Addnl., D-2-HG produced by IDH1/2 mutant cells can competitively inhibit α-KG-dependent enzymes, including histone demethylases and DNA hydroxylases, potentially leading to a distinct epigenetic phenotype.  Alterations in metab. and DNA methylation present possible mechanisms of tumorigenesis.  Summary: Recent attempts to improve outcomes for glioma patients have resulted in incremental gains.  Studies of IDH1/2 mutations have provided mechanistic insights into tumorigenesis and potential avenues for therapeutic intervention.  Further study of IDH1/2 mutations might allow for improved therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo59ZCcbKIXrbVg90H21EOLACvtfcHk0lhouUH9TnEpVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Shu7vP&md5=73509777089e369682a95e9738dd94b6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1097%2FCCO.0b013e32834d816a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0b013e32834d816a%26sid%3Dliteratum%253Aachs%26aulast%3DBorodovsky%26aufirst%3DA.%26aulast%3DSeltzer%26aufirst%3DM.%2BJ.%26aulast%3DRiggins%26aufirst%3DG.%2BJ.%26atitle%3DAltered%2520Cancer%2520Cell%2520Metabolism%2520in%2520Gliomas%2520with%2520Mutant%2520IDH1%2520or%2520IDH2%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2012%26volume%3D24%26spage%3D83%26epage%3D89%26doi%3D10.1097%2FCCO.0b013e32834d816a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, J.</span>; <span class="NLM_string-name">Imai, S.-i.</span></span> <span> </span><span class="NLM_article-title">Accurate Measurement of Nicotinamide Adenine Dinucleotide (NAD(+)) with High-Performance Liquid Chromatography</span>. In  <i>Sirtuins</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschey, M.</span></span>, Ed.; <span class="NLM_series">Methods in Molecular Biology (Methods and Protocols)</span>; <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">Totowa, NJ</span>, <span class="NLM_year">2013</span>; Vol.  <span class="NLM_volume">1077</span>, pp  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">215</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1007%2F978-1-62703-637-5_14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=203-215&author=J.+Yoshino&author=S.-i.+Imaiauthor=M.+Hirschey&title=Sirtuins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2F978-1-62703-637-5_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-62703-637-5_14%26sid%3Dliteratum%253Aachs%26aulast%3DYoshino%26aufirst%3DJ.%26atitle%3DAccurate%2520Measurement%2520of%2520Nicotinamide%2520Adenine%2520Dinucleotide%2520%2528NAD%2528%252B%2529%2529%2520with%2520High-Performance%2520Liquid%2520Chromatography%26btitle%3DSirtuins%26aulast%3DHirschey%26aufirst%3DM.%26pub%3DHumana%2520Press%26date%3D2013%26volume%3D1077%26spage%3D203%26epage%3D215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span> <span> </span><span class="NLM_article-title">Phosphoglycerate Dehydrogenase is Dispensable for Breast Tumor Maintenance and Growth</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2502</span>– <span class="NLM_lpage">2511</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.1540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.18632%2Foncotarget.1540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=24318446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC2c3ks1yksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2502-2511&author=J.+Chenauthor=F.+Chungauthor=G.+Yangauthor=M.+Puauthor=H.+Gaoauthor=W.+Jiangauthor=H.+Yinauthor=V.+Capkaauthor=S.+Kasibhatlaauthor=B.+Laffitteauthor=S.+Jaegerauthor=R.+Pagliariniauthor=Y.+Chenauthor=W.+Zhou&title=Phosphoglycerate+Dehydrogenase+is+Dispensable+for+Breast+Tumor+Maintenance+and+Growth&doi=10.18632%2Foncotarget.1540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoglycerate dehydrogenase is dispensable for breast tumor maintenance and growth</span></div><div class="casAuthors">Chen Jinyun; Chung Franklin; Yang Guizhi; Pu Minying; Gao Hui; Jiang Wei; Yin Hong; Capka Vladimir; Kasibhatla Shailaja; Laffitte Bryan; Jaeger Savina; Pagliarini Raymond; Chen Yaoyu; Zhou Wenlai</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2502-11</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells rely on aerobic glycolysis to maintain cell growth and proliferation via the Warburg effect.  Phosphoglycerate dehydrogenase (PHDGH) catalyzes the first step of the serine biosynthetic pathway downstream of glycolysis, which is a metabolic gatekeeper both for macromolecular biosynthesis and serine-dependent DNA synthesis.  Here, we report that PHDGH is overexpressed in many ER-negative human breast cancer cell lines.  PHGDH knockdown in these cells leads to a reduction of serine synthesis and impairment of cancer cell proliferation.  However, PHGDH knockdown does not affect tumor maintenance and growth in established breast cancer xenograft models, suggesting that PHGDH-dependent cancer cell growth may be context-dependent.  Our findings suggest that other mechanisms or pathways may bypass exclusive dependence on PHGDH in established human breast cancer xenografts, indicating that PHGDH is dispensable for the growth and maintenance and of tumors in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPyCo0Yw55VwyMywPm6HgQfW6udTcc2ebCpIHJfEqpILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3ks1yksw%253D%253D&md5=40631d3b54bdcaa27fbe92c262d83442</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1540%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DChung%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DCapka%26aufirst%3DV.%26aulast%3DKasibhatla%26aufirst%3DS.%26aulast%3DLaffitte%26aufirst%3DB.%26aulast%3DJaeger%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26atitle%3DPhosphoglycerate%2520Dehydrogenase%2520is%2520Dispensable%2520for%2520Breast%2520Tumor%2520Maintenance%2520and%2520Growth%26jtitle%3DOncotarget%26date%3D2013%26volume%3D4%26spage%3D2502%26epage%3D2511%26doi%3D10.18632%2Foncotarget.1540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucki, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beheshti
Zavareh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anglin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolay, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blenis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendt, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lairson, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Identification of a Small Molecule Inhibitor of 3-Phosphoglycerate Dehydrogenase to Target Serine Biosynthesis in Cancers</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">1778</span>– <span class="NLM_lpage">1783</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521548113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1073%2Fpnas.1521548113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=26831078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWgsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=1778-1783&author=E.+Mullarkyauthor=N.+C.+Luckiauthor=R.+Beheshti%0AZavarehauthor=J.+L.+Anglinauthor=A.+P.+Gomesauthor=B.+N.+Nicolayauthor=J.+C.+Y.+Wongauthor=S.+Christenauthor=H.+Takahashiauthor=P.+K.+Singhauthor=J.+Blenisauthor=J.+D.+Warrenauthor=S.-M.+Fendtauthor=J.+M.+Asaraauthor=G.+M.+DeNicolaauthor=C.+A.+Lyssiotisauthor=L.+L.+Lairsonauthor=L.+C.+Cantley&title=Identification+of+a+Small+Molecule+Inhibitor+of+3-Phosphoglycerate+Dehydrogenase+to+Target+Serine+Biosynthesis+in+Cancers&doi=10.1073%2Fpnas.1521548113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small molecule inhibitor of 3-phosphoglycerate dehydrogenase to target serine biosynthesis in cancers</span></div><div class="casAuthors">Mullarky, Edouard; Lucki, Natasha C.; Zavareh, Reza Beheshti; Anglin, Justin L.; Gomes, Ana P.; Nicolay, Brandon N.; Wong, Jenny C. Y.; Christen, Stefan; Takahashi, Hidenori; Singh, Pradeep K.; Blenis, John; Warren, J. David; Fendt, Sarah-Maria; Asara, John M.; DeNicola, Gina M.; Lyssiotis, Costas A.; Lairson, Luke L.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1778-1783</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cancer cells reprogram their metab. to promote growth and proliferation.  The genetic evidence pointing to the importance of the amino acid serine in tumorigenesis is striking.  The gene encoding the enzyme 3-phosphoglycerate dehydrogenase (PHGDH), which catalyzes the first committed step of serine biosynthesis, is overexpressed in tumors and cancer cell lines via focal amplification and nuclear factor erythroid-2-related factor 2 (NRF2)-mediated up-regulation.  PHGDH-overexpressing cells are exquisitely sensitive to genetic ablation of the pathway.  Here, we report the discovery of a selective small mol. inhibitor of PHGDH, CBR-5884, identified by screening a library of 800,000 drug-like compds.  CBR-5884 inhibited de novo serine synthesis in cancer cells and was selectively toxic to cancer cell lines with high serine biosynthetic activity.  Biochem. characterization of the inhibitor revealed that it was a noncompetitive inhibitor that showed a time-dependent onset of inhibition and disrupted the oligomerization state of PHGDH.  The identification of a small mol. inhibitor of PHGDH not only enables thorough preclin. evaluation of PHGDH as a target in cancers, but also provides a tool with which to study serine metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro8G0eAvxBCbVg90H21EOLACvtfcHk0lim_3TY-woyTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWgsr8%253D&md5=a0590d5cc4da060436d2df39d74455df</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521548113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521548113%26sid%3Dliteratum%253Aachs%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DLucki%26aufirst%3DN.%2BC.%26aulast%3DBeheshti%2BZavareh%26aufirst%3DR.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DGomes%26aufirst%3DA.%2BP.%26aulast%3DNicolay%26aufirst%3DB.%2BN.%26aulast%3DWong%26aufirst%3DJ.%2BC.%2BY.%26aulast%3DChristen%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DBlenis%26aufirst%3DJ.%26aulast%3DWarren%26aufirst%3DJ.%2BD.%26aulast%3DFendt%26aufirst%3DS.-M.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DLairson%26aufirst%3DL.%2BL.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DIdentification%2520of%2520a%2520Small%2520Molecule%2520Inhibitor%2520of%25203-Phosphoglycerate%2520Dehydrogenase%2520to%2520Target%2520Serine%2520Biosynthesis%2520in%2520Cancers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2016%26volume%3D113%26spage%3D1778%26epage%3D1783%26doi%3D10.1073%2Fpnas.1521548113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swier, L. J. Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possemato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freinkman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birsoy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu-Remaileh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaul, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">A PHGDH Inhibitor Reveals Coordination of Serine Synthesis and 1-Carbon Unit Fate</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">452</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnchembio.2070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27110680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVaktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=452-458&author=M.+E.+Pacoldauthor=K.+R.+Brimacombeauthor=S.+H.+Chanauthor=J.+M.+Rohdeauthor=C.+A.+Lewisauthor=L.+J.+Y.+M.+Swierauthor=R.+Possematoauthor=W.+W.+Chenauthor=L.+B.+Sullivanauthor=B.+P.+Fiskeauthor=S.+W.+Choauthor=E.+Freinkmanauthor=K.+Birsoyauthor=M.+Abu-Remailehauthor=Y.+D.+Shaulauthor=C.+M.+Liuauthor=M.+Zhouauthor=M.+J.+Kohauthor=H.+Chungauthor=S.+M.+Davidsonauthor=A.+Luengoauthor=A.+Q.+Wangauthor=X.+Xuauthor=A.+Yasgarauthor=L.+Liuauthor=G.+Raiauthor=K.+D.+Westoverauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=N.+S.+Grayauthor=M.+B.+Boxerauthor=D.+M.+Sabatini&title=A+PHGDH+Inhibitor+Reveals+Coordination+of+Serine+Synthesis+and+1-Carbon+Unit+Fate&doi=10.1038%2Fnchembio.2070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A PHGDH inhibitor reveals coordination of serine synthesis and one-carbon unit fate</span></div><div class="casAuthors">Pacold, Michael E.; Brimacombe, Kyle R.; Chan, Sze Ham; Rohde, Jason M.; Lewis, Caroline A.; Swier, Lotteke J. Y. M.; Possemato, Richard; Chen, Walter W.; Sullivan, Lucas B.; Fiske, Brian P.; Cho, Steve; Freinkman, Elizaveta; Birsoy, Kivanc; Abu-Remaileh, Monther; Shaul, Yoav D.; Liu, Chieh Min; Zhou, Minerva; Koh, Min Jung; Chung, Haeyoon; Davidson, Shawn M.; Luengo, Alba; Wang, Amy Q.; Xu, Xin; Yasgar, Adam; Liu, Li; Rai, Ganesha; Westover, Kenneth D.; Vander Heiden, Matthew G.; Shen, Min; Gray, Nathanael S.; Boxer, Matthew B.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">452-458</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Serine is both a proteinogenic amino acid and the source of one-carbon units essential for de novo purine and deoxythymidine synthesis.  In the canonical pathway of glucose-derived serine synthesis, Homo sapiens phosphoglycerate dehydrogenase (PHGDH) catalyzes the first, rate-limiting step.  Genetic loss of PHGDH is toxic toward PHGDH-overexpressing breast cancer cell lines even in the presence of exogenous serine.  Here, we used a quant. high-throughput screen to identify small-mol. PHGDH inhibitors.  These compds. reduce the prodn. of glucose-derived serine in cells and suppress the growth of PHGDH-dependent cancer cells in culture and in orthotopic xenograft tumors.  Surprisingly, PHGDH inhibition reduced the incorporation into nucleotides of one-carbon units from glucose-derived and exogenous serine.  We conclude that glycolytic serine synthesis coordinates the use of one-carbon units from endogenous and exogenous serine in nucleotide synthesis, and we suggest that one-carbon unit wasting thus may contribute to the efficacy of PHGDH inhibitors in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-N14m5iKDe7Vg90H21EOLACvtfcHk0lg_jQfnGsuMbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVaktLs%253D&md5=4917bfb6309946cd08b593ed26dbe73c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2070%26sid%3Dliteratum%253Aachs%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DChan%26aufirst%3DS.%2BH.%26aulast%3DRohde%26aufirst%3DJ.%2BM.%26aulast%3DLewis%26aufirst%3DC.%2BA.%26aulast%3DSwier%26aufirst%3DL.%2BJ.%2BY.%2BM.%26aulast%3DPossemato%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DW.%2BW.%26aulast%3DSullivan%26aufirst%3DL.%2BB.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DCho%26aufirst%3DS.%2BW.%26aulast%3DFreinkman%26aufirst%3DE.%26aulast%3DBirsoy%26aufirst%3DK.%26aulast%3DAbu-Remaileh%26aufirst%3DM.%26aulast%3DShaul%26aufirst%3DY.%2BD.%26aulast%3DLiu%26aufirst%3DC.%2BM.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DKoh%26aufirst%3DM.%2BJ.%26aulast%3DChung%26aufirst%3DH.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DLuengo%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DA%2520PHGDH%2520Inhibitor%2520Reveals%2520Coordination%2520of%2520Serine%2520Synthesis%2520and%25201-Carbon%2520Unit%2520Fate%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D452%26epage%3D458%26doi%3D10.1038%2Fnchembio.2070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ravez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillier, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frédérick, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketothioamide Derivatives: A Promising Tool to Interrogate Phosphoglycerate Dehydrogenase (PHGDH)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1591</span>– <span class="NLM_lpage">1597</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01166</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01166" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1591-1597&author=S.+Ravezauthor=C.+Corbetauthor=Q.+Spillierauthor=A.+Dutuauthor=A.+D.+Robinauthor=E.+Mullarkyauthor=L.+C.+Cantleyauthor=O.+Feronauthor=R.+Fr%C3%A9d%C3%A9rick&title=%CE%B1-Ketothioamide+Derivatives%3A+A+Promising+Tool+to+Interrogate+Phosphoglycerate+Dehydrogenase+%28PHGDH%29&doi=10.1021%2Facs.jmedchem.6b01166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketothioamide Derivatives: A Promising Tool to Interrogate Phosphoglycerate Dehydrogenase (PHGDH)</span></div><div class="casAuthors">Ravez, Severine; Corbet, Cyril; Spillier, Quentin; Dutu, Alice; Robin, Anita D.; Mullarky, Edouard; Cantley, Lewis C.; Feron, Olivier; Frederick, Raphael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1591-1597</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Given the putative role of PHGDH in cancer, development of inhibitors is required to explore its function.  In this context, we established and validated a straightforward enzymic assay suitable for high-throughput screening and we identified inhibitors with similar chem. scaffolds.  Through a convergent pharmacophore approach, we synthesized α-ketothioamides that exhibit interesting in vitro PHGDH inhibition and encouraging cellular results.  These novel probes may be used to understand the emerging biol. of this metabolic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMG07MHJ9Fx7Vg90H21EOLACvtfcHk0lg_jQfnGsuMbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKisrw%253D&md5=dae86884b63554c46fdf33d8e2cf42cd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01166%26sid%3Dliteratum%253Aachs%26aulast%3DRavez%26aufirst%3DS.%26aulast%3DCorbet%26aufirst%3DC.%26aulast%3DSpillier%26aufirst%3DQ.%26aulast%3DDutu%26aufirst%3DA.%26aulast%3DRobin%26aufirst%3DA.%2BD.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DFeron%26aufirst%3DO.%26aulast%3DFr%25C3%25A9d%25C3%25A9rick%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketothioamide%2520Derivatives%253A%2520A%2520Promising%2520Tool%2520to%2520Interrogate%2520Phosphoglycerate%2520Dehydrogenase%2520%2528PHGDH%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1591%26epage%3D1597%26doi%3D10.1021%2Facs.jmedchem.6b01166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberti, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Selective Allosteric Inhibitors of PHGDH and Serine Synthesis with Anti-tumor Activity</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.chembiol.2016.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=28042046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1yhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=55-65&author=Q.+Wangauthor=M.+V.+Libertiauthor=P.+Liuauthor=X.+Dengauthor=Y.+Liuauthor=J.+W.+Locasaleauthor=L.+Lai&title=Rational+Design+of+Selective+Allosteric+Inhibitors+of+PHGDH+and+Serine+Synthesis+with+Anti-tumor+Activity&doi=10.1016%2Fj.chembiol.2016.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Selective Allosteric Inhibitors of PHGDH and Serine Synthesis with Anti-tumor Activity</span></div><div class="casAuthors">Wang, Qian; Liberti, Maria V.; Liu, Pei; Deng, Xiaobing; Liu, Ying; Locasale, Jason W.; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-65</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metabolic reprogramming in cancer cells facilitates growth and proliferation.  Increased activity of the serine biosynthetic pathway through the enzyme phosphoglycerate dehydrogenase (PHGDH) contributes to tumorigenesis.  With a small substrate and a weak binding cofactor, (NAD+), inhibitor development for PHGDH remains challenging.  Instead of targeting the PHGDH active site, we computationally identified two potential allosteric sites and virtually screened compds. that can bind to these sites.  With subsequent characterization, we successfully identified PHGDH non-NAD+-competing allosteric inhibitors that attenuate its enzyme activity, selectively inhibit de novo serine synthesis in cancer cells, and reduce tumor growth in vivo.  Our study not only identifies novel allosteric inhibitors for PHGDH to probe its function and potential as a therapeutic target, but also provides a general strategy for the rational design of small-mol. modulators of metabolic enzyme function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5L2K1-sIbP7Vg90H21EOLACvtfcHk0liBauSV5NDx5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1yhsg%253D%253D&md5=bad62b07802dbf0d7a3bdbd1fd151b39</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLiberti%26aufirst%3DM.%2BV.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DRational%2520Design%2520of%2520Selective%2520Allosteric%2520Inhibitors%2520of%2520PHGDH%2520and%2520Serine%2520Synthesis%2520with%2520Anti-tumor%2520Activity%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D55%26epage%3D65%26doi%3D10.1016%2Fj.chembiol.2016.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spadola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edfeldt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R.</span></span> <span> </span><span class="NLM_article-title">An Improved Model for Fragment-based Lead Generation at AstraZeneca</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1272</span>– <span class="NLM_lpage">1283</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.drudis.2016.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27179986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC28bpvVyntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1272-1283&author=N.+Fullerauthor=L.+Spadolaauthor=S.+Cowenauthor=J.+Patelauthor=H.+Sch%C3%B6nherrauthor=Q.+Caoauthor=A.+McKenzieauthor=F.+Edfeldtauthor=A.+Rabowauthor=R.+Goodnow&title=An+Improved+Model+for+Fragment-based+Lead+Generation+at+AstraZeneca&doi=10.1016%2Fj.drudis.2016.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">An improved model for fragment-based lead generation at AstraZeneca</span></div><div class="casAuthors">Fuller Nathan; Spadola Loredana; Cowen Scott; Patel Joe; Schonherr Heike; Cao Qing; McKenzie Andrew; Goodnow Robert; Edfeldt Fredrik; Rabow Al</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1272-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Modest success rates in fragment-based lead generation (FBLG) projects at AstraZeneca (AZ) prompted operational changes to improve performance.  In this review, we summarize these changes, emphasizing the construction and composition of the AZ fragment library, screening practices and working model.  We describe the profiles of the screening method for specific fragment subsets and statistically assess our ability to follow up on fragment hits through near-neighbor selection.  Performance analysis of our second-generation fragment library (FL2) in screening campaigns illustrates the complementary nature of flat and 3D fragments in exploring protein-binding pockets and highlights our ability to deliver fragment hits using multiple screening techniques for various target classes.  The new model has had profound impact on the successful delivery of lead series to drug discovery projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhSnzpo3gNWkloaUqgkFHkfW6udTcc2eZjT9_F8mEVprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bpvVyntQ%253D%253D&md5=d8c40172502e32be3173afcd292be480</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DFuller%26aufirst%3DN.%26aulast%3DSpadola%26aufirst%3DL.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DH.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DEdfeldt%26aufirst%3DF.%26aulast%3DRabow%26aufirst%3DA.%26aulast%3DGoodnow%26aufirst%3DR.%26atitle%3DAn%2520Improved%2520Model%2520for%2520Fragment-based%2520Lead%2520Generation%2520at%2520AstraZeneca%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1272%26epage%3D1283%26doi%3D10.1016%2Fj.drudis.2016.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unterlass, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span> <span> </span><span class="NLM_article-title">Validating and Enabling Phosphoglycerate Dehydrogenase (PHGDH) as a Target for Fragment-based Drug Discovery in PHGDH-amplified Breast Cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">13139</span>– <span class="NLM_lpage">13153</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.18632%2Foncotarget.11487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29568346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC1MnktValsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=13139-13153&author=J.+E.+Unterlassauthor=A.+Basleauthor=T.+J.+Blackburnauthor=J.+Tuckerauthor=C.+Canoauthor=M.+E.+M.+Nobleauthor=N.+J.+Curtin&title=Validating+and+Enabling+Phosphoglycerate+Dehydrogenase+%28PHGDH%29+as+a+Target+for+Fragment-based+Drug+Discovery+in+PHGDH-amplified+Breast+Cancer&doi=10.18632%2Foncotarget.11487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Validating and enabling phosphoglycerate dehydrogenase (PHGDH) as a target for fragment-based drug discovery in PHGDH-amplified breast cancer</span></div><div class="casAuthors">Unterlass Judith E; Tucker Julie; Noble Martin E M; Curtin Nicola J; Basle Arnaud; Blackburn Timothy J; Cano Celine</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">13139-13153</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">3-Phosphoglycerate dehydrogenase (PHGDH) has recently been identified as an attractive target in cancer therapy as it links upregulated glycolytic flux to increased biomass production in cancer cells.  PHGDH catalyses the first step in the serine synthesis pathway and thus diverts glycolytic flux into serine synthesis.  We have used siRNA-mediated suppression of PHGDH expression to show that PHGDH is a potential therapeutic target in PHGDH-amplified breast cancer.  Knockdown caused reduced proliferation in the PHGDH-amplified cell line MDA-MB-468, whereas breast cancer cells with low PHGDH expression or with elevated PHGDH expression in the absence of genomic amplification were not affected.  As a first step towards design of a chemical probe for PHGDH, we report a fragment-based drug discovery approach for the identification of PHGDH inhibitors.  We designed a truncated PHGDH construct that gave crystals which diffracted to high resolution, and could be used for fragment soaking. 15 fragments stabilising PHGDH were identified using a thermal shift assay and validated by X-ray crystallography and ITC competition experiments to exhibit 1.5-26.2 mM affinity for PHGDH.  A structure-guided fragment growing approach was applied to the PHGDH binders from the initial screen, yielding greater understanding of the binding site and suggesting routes to achieve higher affinity NAD-competitive inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRatiF0vPAX33y-c-E1v-fmfW6udTcc2eZjT9_F8mEVprntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnktValsg%253D%253D&md5=4a2576e240366888437f5fae951fac7a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11487%26sid%3Dliteratum%253Aachs%26aulast%3DUnterlass%26aufirst%3DJ.%2BE.%26aulast%3DBasle%26aufirst%3DA.%26aulast%3DBlackburn%26aufirst%3DT.%2BJ.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DCano%26aufirst%3DC.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DValidating%2520and%2520Enabling%2520Phosphoglycerate%2520Dehydrogenase%2520%2528PHGDH%2529%2520as%2520a%2520Target%2520for%2520Fragment-based%2520Drug%2520Discovery%2520in%2520PHGDH-amplified%2520Breast%2520Cancer%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D13139%26epage%3D13153%26doi%3D10.18632%2Foncotarget.11487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raze Therapeutics, I.</span>; <span class="NLM_string-name">Mainolfi, N.</span>; <span class="NLM_string-name">Moyer, M. P.</span></span> <span> </span><span class="NLM_article-title">3-Phosphoglycerate Dehydrogenase Inhibitors and Uses Thereof</span>. <span class="NLM_patent">WO2017156177</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=I.+Raze+Therapeutics&author=N.+Mainolfi&author=M.+P.+Moyer&title=3-Phosphoglycerate+Dehydrogenase+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DRaze%2BTherapeutics%26aufirst%3DI.%26atitle%3D3-Phosphoglycerate%2520Dehydrogenase%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baljinnyam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Piperazine-1-Thiourea-based Human Phosphoglycerate Dehydrogenase Inhibitors</span>. <i>Biorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1739</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.bmc.2018.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29555419" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=1727-1739&author=J.+M.+Rohdeauthor=K.+R.+Brimacombeauthor=L.+Liuauthor=M.+E.+Pacoldauthor=A.+Yasgarauthor=D.+M.+Cheffauthor=T.+D.+Leeauthor=G.+Raiauthor=B.+Baljinnyamauthor=Z.+Liauthor=A.+Simeonovauthor=M.+D.+Hallauthor=M.+Shenauthor=D.+M.+Sabatiniauthor=M.+B.+Boxer&title=Discovery+and+Optimization+of+Piperazine-1-Thiourea-based+Human+Phosphoglycerate+Dehydrogenase+Inhibitors&doi=10.1016%2Fj.bmc.2018.02.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DRohde%26aufirst%3DJ.%2BM.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DCheff%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DT.%2BD.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DBaljinnyam%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Piperazine-1-Thiourea-based%2520Human%2520Phosphoglycerate%2520Dehydrogenase%2520Inhibitors%26jtitle%3DBiorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D1727%26epage%3D1739%26doi%3D10.1016%2Fj.bmc.2018.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgUE0qeDoklPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchlow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrosek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogtherr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3285</span>– <span class="NLM_lpage">3306</span>, <span class="refDoi"> DOI: 10.1021/jm201734r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201734r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3285-3306&author=R.+A.+Wardauthor=C.+Brassingtonauthor=A.+L.+Breezeauthor=A.+Caputoauthor=S.+Critchlowauthor=G.+Daviesauthor=L.+Goodwinauthor=G.+Hassallauthor=R.+Greenwoodauthor=G.+A.+Holdgateauthor=M.+Mrosekauthor=R.+A.+Normanauthor=S.+Pearsonauthor=J.+Tartauthor=J.+A.+Tuckerauthor=M.+Vogtherrauthor=D.+Whittakerauthor=J.+Wingfieldauthor=J.+Winterauthor=K.+Hudson&title=Design+and+Synthesis+of+Novel+Lactate+Dehydrogenase+A+Inhibitors+by+Fragment-Based+Lead+Generation&doi=10.1021%2Fjm201734r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation</span></div><div class="casAuthors">Ward, Richard A.; Brassington, Claire; Breeze, Alexander L.; Caputo, Alessandro; Critchlow, Susan; Davies, Gareth; Goodwin, Louise; Hassall, Giles; Greenwood, Ryan; Holdgate, Geoffrey A.; Mrosek, Michael; Norman, Richard A.; Pearson, Stuart; Tart, Jonathan; Tucker, Julie A.; Vogtherr, Martin; Whittaker, David; Wingfield, Jonathan; Winter, Jon; Hudson, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3285-3306</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDHA) catalyzes the conversion of pyruvate to lactate, utilizing NADH as a cofactor.  It has been identified as a potential therapeutic target in the area of cancer metab.  In this manuscript we report our progress using fragment-based lead generation (FBLG), assisted by x-ray crystallog. to develop small mol. LDHA inhibitors.  Fragment hits were identified through NMR and SPR screening and optimized into lead compds. with nanomolar binding affinities via fragment linking.  Also reported is their modification into cellular active compds. suitable for target validation work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH8QbdIUXNvbVg90H21EOLACvtfcHk0lgUE0qeDoklPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D&md5=d638e7a0db361c212ee084d5c05ee70c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm201734r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201734r%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26aulast%3DCaputo%26aufirst%3DA.%26aulast%3DCritchlow%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DHassall%26aufirst%3DG.%26aulast%3DGreenwood%26aufirst%3DR.%26aulast%3DHoldgate%26aufirst%3DG.%2BA.%26aulast%3DMrosek%26aufirst%3DM.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%26aulast%3DTart%26aufirst%3DJ.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DVogtherr%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWingfield%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DK.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Novel%2520Lactate%2520Dehydrogenase%2520A%2520Inhibitors%2520by%2520Fragment-Based%2520Lead%2520Generation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3285%26epage%3D3306%26doi%3D10.1021%2Fjm201734r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span> <span> </span><span class="NLM_article-title">Characterization of Ligand Binding by Saturation Transfer Difference NMR Spectroscopy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1784</span>– <span class="NLM_lpage">1788</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+Ligand+Binding+by+Saturation+Transfer+Difference+NMR+Spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0ligsr8xg6HMjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520Ligand%2520Binding%2520by%2520Saturation%2520Transfer%2520Difference%2520NMR%2520Spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unterlass, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into the Enzymatic Activity and Potential Substrate Promiscuity of Human 3-Phosphoglycerate Dehydrogenase (PHGDH)</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">104478</span> <span class="refDoi"> DOI: 10.18632/oncotarget.22327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.18632%2Foncotarget.22327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29262655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzjs1emsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=J.+E.+Unterlassauthor=R.+J.+Woodauthor=A.+Basleauthor=J.+Tuckerauthor=C.+Canoauthor=M.+M.+E.+Nobleauthor=N.+J.+Curtin&title=Structural+Insights+into+the+Enzymatic+Activity+and+Potential+Substrate+Promiscuity+of+Human+3-Phosphoglycerate+Dehydrogenase+%28PHGDH%29&doi=10.18632%2Foncotarget.22327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the enzymatic activity and potential substrate promiscuity of human 3-phosphoglycerate dehydrogenase (PHGDH)</span></div><div class="casAuthors">Unterlass Judith E; Tucker Julie; Noble Martin M E; Curtin Nicola J; Unterlass Judith E; Wood Robert J; Basle Arnaud; Cano Celine</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">61</span>),
    <span class="NLM_cas:pages">104478-104491</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells reprogram their metabolism and energy production to sustain increased growth, enable metastasis and overcome resistance to cancer treatments.  Although primary roles for many metabolic proteins have been identified, some are promiscuous in regards to the reaction they catalyze.  To efficiently target these enzymes, a good understanding of their enzymatic function and structure, as well as knowledge regarding any substrate or catalytic promiscuity is required.  Here we focus on the characterization of human 3-phosphoglycerate dehydrogenase (PHGDH).  PHGDH catalyzes the NAD(+)-dependent conversion of 3-phosphoglycerate to phosphohydroxypyruvate, which is the first step in the de novo synthesis pathway of serine, a critical amino acid for protein and nucleic acid biosynthesis.  We have investigated substrate analogues to assess whether PHGDH might possess other enzymatic roles that could explain its occasional over-expression in cancer, as well as to help with the design of specific inhibitors.  We also report the crystal structure of the catalytic subunit of human PHGDH, a dimer, solved with bound cofactor in one monomer and both cofactor and L-tartrate in the second monomer.  In vitro enzyme activity measurements show that the catalytic subunit of PHGDH is still active and that PHGDH activity could be significantly inhibited with adenosine 5'-diphosphoribose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJ7MGLt-1wSMERNNga7KD-fW6udTcc2eZuNlNrexEaabntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzjs1emsQ%253D%253D&md5=a2844b264542c8d34f4651817e18761a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.22327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.22327%26sid%3Dliteratum%253Aachs%26aulast%3DUnterlass%26aufirst%3DJ.%2BE.%26aulast%3DWood%26aufirst%3DR.%2BJ.%26aulast%3DBasle%26aufirst%3DA.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DCano%26aufirst%3DC.%26aulast%3DNoble%26aufirst%3DM.%2BM.%2BE.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DStructural%2520Insights%2520into%2520the%2520Enzymatic%2520Activity%2520and%2520Potential%2520Substrate%2520Promiscuity%2520of%2520Human%25203-Phosphoglycerate%2520Dehydrogenase%2520%2528PHGDH%2529%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26doi%3D10.18632%2Foncotarget.22327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, R. W.</span></span> <span> </span><span class="NLM_article-title">Allylic 1,3-Strain as a Controlling Factor in Stereoselective Transformations</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1841</span>– <span class="NLM_lpage">1860</span>, <span class="refDoi"> DOI: 10.1021/cr00098a009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00098a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADyaL1MXmtlWnurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1989&pages=1841-1860&author=R.+W.+Hoffmann&title=Allylic+1%2C3-Strain+as+a+Controlling+Factor+in+Stereoselective+Transformations&doi=10.1021%2Fcr00098a009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Allylic 1,3-strain as a controlling factor in stereoselective transformations</span></div><div class="casAuthors">Hoffmann, Reinhard W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1841-60</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 155 refs. on the manifestation of allylic strain in the stereochem. of a host of chem. reactions, including cyclizations, intermol. addn., hydrogenation, cyclopropanation, and others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDZG59Whd3WbVg90H21EOLACvtfcHk0ligsr8xg6HMjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtlWnurc%253D&md5=91540e91c0c31a112831dd5ac9d73f4d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcr00098a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00098a009%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DR.%2BW.%26atitle%3DAllylic%25201%252C3-Strain%2520as%2520a%2520Controlling%2520Factor%2520in%2520Stereoselective%2520Transformations%26jtitle%3DChem.%2520Rev.%26date%3D1989%26volume%3D89%26spage%3D1841%26epage%3D1860%26doi%3D10.1021%2Fcr00098a009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tietze, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, G.</span></span> <span> </span><span class="NLM_article-title">Ab Initio Molecular Orbital Calculations on Allylic 1,3-Strain of Electron-Donor- and Electron-Acceptor-Substituted Alkenes</span>. <i>Liebigs Ann.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>1996</i></span>,  <span class="NLM_fpage">1575</span>– <span class="NLM_lpage">1579</span>, <span class="refDoi"> DOI: 10.1002/jlac.199619961012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1002%2Fjlac.199619961012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1996&publication_year=1996&pages=1575-1579&author=L.+F.+Tietzeauthor=G.+Schulz&title=Ab+Initio+Molecular+Orbital+Calculations+on+Allylic+1%2C3-Strain+of+Electron-Donor-+and+Electron-Acceptor-Substituted+Alkenes&doi=10.1002%2Fjlac.199619961012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fjlac.199619961012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjlac.199619961012%26sid%3Dliteratum%253Aachs%26aulast%3DTietze%26aufirst%3DL.%2BF.%26aulast%3DSchulz%26aufirst%3DG.%26atitle%3DAb%2520Initio%2520Molecular%2520Orbital%2520Calculations%2520on%2520Allylic%25201%252C3-Strain%2520of%2520Electron-Donor-%2520and%2520Electron-Acceptor-Substituted%2520Alkenes%26jtitle%3DLiebigs%2520Ann.%26date%3D1996%26volume%3D1996%26spage%3D1575%26epage%3D1579%26doi%3D10.1002%2Fjlac.199619961012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ravez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillier, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marteau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frédérick, R.</span></span> <span> </span><span class="NLM_article-title">Challenges and Opportunities in the Development of Serine Synthetic Pathway Inhibitors for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01167</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01167" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOls7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1227-1237&author=S.+Ravezauthor=Q.+Spillierauthor=R.+Marteauauthor=O.+Feronauthor=R.+Fr%C3%A9d%C3%A9rick&title=Challenges+and+Opportunities+in+the+Development+of+Serine+Synthetic+Pathway+Inhibitors+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.6b01167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and Opportunities in the Development of Serine Synthetic Pathway Inhibitors for Cancer Therapy</span></div><div class="casAuthors">Ravez, Severine; Spillier, Quentin; Marteau, Romain; Feron, Olivier; Frederick, Raphael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1227-1237</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The recent advances in understanding the relation between cancer and metab. have highlighted the relevance of the serine synthetic pathway (SSP) that consists of three successive enzymic reactions.  Enzymes of the SSP, such as the phosphoglycerate dehydrogenase (PHGDH) and the phosphoserine aminotransferase 1 (PSAT-1), were recently highlighted because they are amplified in a significant subset of human tumors and their suppressions by RNAi caused a decrease in cancer cell survival and growth.  Currently, the field of drug discovery against these enzymes is still in its infancy and the identification of suitable inhibitors could serve to understand the emerging biol. of these metabolic enzymes.  In this review, the authors present the SSP as a significant and novel emerging area for medicinal chem. and the authors develop an overview of one of the key enzymes of the pathway, PHGDH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKa3KCVECsJ7Vg90H21EOLACvtfcHk0ljH_R3-lTaeMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOls7nJ&md5=79c2de60dd8cfad2bb480dc393ce23cf</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01167%26sid%3Dliteratum%253Aachs%26aulast%3DRavez%26aufirst%3DS.%26aulast%3DSpillier%26aufirst%3DQ.%26aulast%3DMarteau%26aufirst%3DR.%26aulast%3DFeron%26aufirst%3DO.%26aulast%3DFr%25C3%25A9d%25C3%25A9rick%26aufirst%3DR.%26atitle%3DChallenges%2520and%2520Opportunities%2520in%2520the%2520Development%2520of%2520Serine%2520Synthetic%2520Pathway%2520Inhibitors%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1227%26epage%3D1237%26doi%3D10.1021%2Facs.jmedchem.6b01167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caravella, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbeck, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitesides, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span> <span> </span><span class="NLM_article-title">Long-Range Electrostatic Contributions to Protein–Ligand Binding Estimated Using Protein Charge Ladders, Affinity Capillary Electrophoresis, and Continuum Electrostatic Theory</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">4340</span>– <span class="NLM_lpage">4347</span>, <span class="refDoi"> DOI: 10.1021/ja984195a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja984195a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADyaK1MXisFOmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=4340-4347&author=J.+A.+Caravellaauthor=J.+D.+Carbeckauthor=D.+C.+Duffyauthor=G.+M.+Whitesidesauthor=B.+Tidor&title=Long-Range+Electrostatic+Contributions+to+Protein%E2%80%93Ligand+Binding+Estimated+Using+Protein+Charge+Ladders%2C+Affinity+Capillary+Electrophoresis%2C+and+Continuum+Electrostatic+Theory&doi=10.1021%2Fja984195a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Long-Range Electrostatic Contributions to Protein-Ligand Binding Estimated Using Protein Charge Ladders, Affinity Capillary Electrophoresis, and Continuum Electrostatic Theory</span></div><div class="casAuthors">Caravella, Justin A.; Carbeck, Jeffrey D.; Duffy, David C.; Whitesides, George M.; Tidor, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4340-4347</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Affinity capillary electrophoresis and protein charge ladders are used together to measure the contributions of long-range electrostatic interactions to binding of substituted benzene sulfonamide inhibitors to derivs. of human carbonic anhydrase II.  The results are analyzed by continuum electrostatic calcns., which afford a detailed anal. of interactions of individual members of a population from a charge ladder.  A Monte Carlo simulation of the exptl. data using calcd. contributions of individual lysine side chains to inhibitor binding shows that a large no. of different patterns of acetylation are consistent with the exptl. results.  The calcns. predict significant differences in the contributions of some lysines to ΔG and simulations suggest that exptl. resoln. must be enhanced to be able to measure such differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxrl3LPkPRxrVg90H21EOLACvtfcHk0ljH_R3-lTaeMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisFOmsL8%253D&md5=87d77bea54553d1af9d7346178bdfacd</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja984195a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja984195a%26sid%3Dliteratum%253Aachs%26aulast%3DCaravella%26aufirst%3DJ.%2BA.%26aulast%3DCarbeck%26aufirst%3DJ.%2BD.%26aulast%3DDuffy%26aufirst%3DD.%2BC.%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26aulast%3DTidor%26aufirst%3DB.%26atitle%3DLong-Range%2520Electrostatic%2520Contributions%2520to%2520Protein%25E2%2580%2593Ligand%2520Binding%2520Estimated%2520Using%2520Protein%2520Charge%2520Ladders%252C%2520Affinity%2520Capillary%2520Electrophoresis%252C%2520and%2520Continuum%2520Electrostatic%2520Theory%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D4340%26epage%3D4347%26doi%3D10.1021%2Fja984195a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vallet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlgren, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grenthe, I.</span></span> <span> </span><span class="NLM_article-title">Chelate Effect and Thermodynamics of Metal Complex Formation in Solution: A Quantum Chemical Study</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">14941</span>– <span class="NLM_lpage">14950</span>, <span class="refDoi"> DOI: 10.1021/ja036646j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja036646j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVCltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=14941-14950&author=V.+Valletauthor=U.+Wahlgrenauthor=I.+Grenthe&title=Chelate+Effect+and+Thermodynamics+of+Metal+Complex+Formation+in+Solution%3A+A+Quantum+Chemical+Study&doi=10.1021%2Fja036646j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Chelate Effect and Thermodynamics of Metal Complex Formation in Solution: A Quantum Chemical Study</span></div><div class="casAuthors">Vallet, Valerie; Wahlgren, Ulf; Grenthe, Ingmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">14941-14950</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accuracy of quantum chem. predictions of structures and thermodn. data for metal complexes depends both on the quantum chem. methods and the chem. models used.  A thermodn. analog of the Eigen-Wilkins mechanism for ligand substitution reactions (Model A) turns out to be sufficiently simple to catch the essential chem. of complex formation reactions and allows quantum chem. calcns. at the ab initio level of thermodn. quantities both in gas phase and soln.; the latter by using the conductor-like polarizable continuum (CPCM) model.  Model A describes the complex formation as a two-step reaction: [M(H2O)x](aq) + L(aq) .dblharw. [M(H2O)x],L(aq); and [M(H2O)x],L(aq) .dblharw. [M(H2O)x-1L],(H2O)(aq).  The first step, the formation of an outer-sphere complex is described using the Fuoss equation and the second, the intramol. exchange between an entering ligand from the second and water in the first coordination shell, using quantum chem. methods.  The thermodn. quantities for this model were compared to those for the reaction: [M(H2O)x](aq) + L(aq) .dblharw. [M(H2O)x-1L](aq) + (H2O)(aq) (Model B), as calcd. for each reactant and product sep.  The models were tested using complex formation between Zn2+ and ammonia, methylamine, and ethylenediamine, and complex formation and chelate ring closure reactions in binary and ternary UO22+-oxalate systems.  The results show that the Gibbs energy of reaction for Model A are not strongly dependent on the no. of water ligands and the structure of the second coordination sphere; it provides a much more precise est. of the thermodn. of complex formation reactions in soln. than that obtained from Model B.  The agreement between the exptl. and calcd. data for the formation of Zn(NH3)2+(aq) and Zn(NH3)2+2(aq) is better than 8 kJ/mol for the former, as compared to 30 kJ/mol or larger, for the latter.  The Gibbs energy of reaction obtained for the UO2+2 oxalate systems using model B differs between 80 and 130 kJ/mol from the exptl. results, whereas the agreement with Model A is better.  The errors in the quantum chem. ests. of the entropy and enthalpy of reaction are somewhat larger than those for the Gibbs energy, but still in fair agreement with expts.; adding water mols. in the second coordination sphere improves the agreement significantly.  Reasons for the different performance of the two models are discussed.  The quantum chem. data were used to discuss the microscopic basis of exptl. enthalpy and entropy data, to det. the enthalpy and entropy contributions in chelate ring closure reactions and to discuss the origin of the so-called "chelate effect".  Contrary to many earlier suggestions, this is not even in the gas phase, a result of changes in translation entropy contributions.  There is no simple explanation of the high stability of chelate complexes; it is a result of both enthalpy and entropy contributions that vary from one system to the other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfoLO7IER0drVg90H21EOLACvtfcHk0ljH_R3-lTaeMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVCltrg%253D&md5=6d4bb99794e6d1aea670c814f46e07a8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja036646j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja036646j%26sid%3Dliteratum%253Aachs%26aulast%3DVallet%26aufirst%3DV.%26aulast%3DWahlgren%26aufirst%3DU.%26aulast%3DGrenthe%26aufirst%3DI.%26atitle%3DChelate%2520Effect%2520and%2520Thermodynamics%2520of%2520Metal%2520Complex%2520Formation%2520in%2520Solution%253A%2520A%2520Quantum%2520Chemical%2520Study%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D14941%26epage%3D14950%26doi%3D10.1021%2Fja036646j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data Processing and Analysis with the autoPROC toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+Processing+and+Analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lhO7bxEyujwJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520Processing%2520and%2520Analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Collaborative Computational Project, Number
4</span> <span> </span><span class="NLM_article-title">The CCP4 Suite: Programs
for Protein Crystallography</span>. <i>Acta Crystallogr.,
Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project%2C+Number%0A4&title=The+CCP4+Suite%3A+Programs%0Afor+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520Suite%253A%2520Programs%250Afor%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%250ASect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhO7bxEyujwJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T.</span></span> <i>BUSTER</i>, version 2.11.5; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+Smart&author=C.+Vonrhein&author=T.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 19 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Christine R. Cuthbertson, Zahra Arabzada, Armand Bankhead III, Armita Kyani, <span class="NLM_string-name hlFld-ContribAuthor">Nouri Neamati</span>. </span><span class="cited-content_cbyCitation_article-title">A Review of Small-Molecule Inhibitors of One-Carbon Enzymes: SHMT2 and MTHFD2 in the Spotlight. </span><span class="cited-content_cbyCitation_journal-name">ACS Pharmacology & Translational Science</span><span> <strong>2021,</strong> <em>4 </em>
                                    (2)
                                     , 624-646. <a href="https://doi.org/10.1021/acsptsci.0c00223" title="DOI URL">https://doi.org/10.1021/acsptsci.0c00223</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsptsci.0c00223&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsptsci.0c00223%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Pharmacology%2520%2526%2520Translational%2520Science%26atitle%3DA%252BReview%252Bof%252BSmall-Molecule%252BInhibitors%252Bof%252BOne-Carbon%252BEnzymes%25253A%252BSHMT2%252Band%252BMTHFD2%252Bin%252Bthe%252BSpotlight%26aulast%3DCuthbertson%26aufirst%3DChristine%2BR.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22122020%26date%3D01032021%26volume%3D4%26issue%3D2%26spage%3D624%26epage%3D646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wolfgang Jahnke, Daniel A. Erlanson, Iwan J. P. de Esch, Christopher N. Johnson, Paul N. Mortenson, Yuji Ochi, <span class="NLM_string-name hlFld-ContribAuthor">Tatsuya Urushima</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2019. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15494-15507. <a href="https://doi.org/10.1021/acs.jmedchem.0c01608" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01608%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2019%26aulast%3DJahnke%26aufirst%3DWolfgang%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D14092020%26date%3D23112020%26volume%3D63%26issue%3D24%26spage%3D15494%26epage%3D15507" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xia  Zhou</span>, <span class="hlFld-ContribAuthor ">Yuping  Tan</span>, <span class="hlFld-ContribAuthor ">Kun  Gou</span>, <span class="hlFld-ContribAuthor ">Lei  Tao</span>, <span class="hlFld-ContribAuthor ">Yuan  Luo</span>, <span class="hlFld-ContribAuthor ">Yue  Zhou</span>, <span class="hlFld-ContribAuthor ">Zeping  Zuo</span>, <span class="hlFld-ContribAuthor ">Qingxiang  Sun</span>, <span class="hlFld-ContribAuthor ">Youfu  Luo</span>, <span class="hlFld-ContribAuthor ">Yinglan  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel inhibitors of human phosphoglycerate dehydrogenase by activity-directed combinatorial chemical synthesis strategy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>115 </em>, 105159. <a href="https://doi.org/10.1016/j.bioorg.2021.105159" title="DOI URL">https://doi.org/10.1016/j.bioorg.2021.105159</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2021.105159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2021.105159%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Binhibitors%252Bof%252Bhuman%252Bphosphoglycerate%252Bdehydrogenase%252Bby%252Bactivity-directed%252Bcombinatorial%252Bchemical%252Bsynthesis%252Bstrategy%26aulast%3DZhou%26aufirst%3DXia%26date%3D2021%26volume%3D115%26spage%3D105159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Quentin  Spillier</span>, <span class="hlFld-ContribAuthor ">Raphaël  Frédérick</span>. </span><span class="cited-content_cbyCitation_article-title">Phosphoglycerate dehydrogenase (PHGDH) inhibitors: a comprehensive review 2015–2020. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (7)
                                     , 597-608. <a href="https://doi.org/10.1080/13543776.2021.1890028" title="DOI URL">https://doi.org/10.1080/13543776.2021.1890028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1890028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1890028%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DPhosphoglycerate%252Bdehydrogenase%252B%252528PHGDH%252529%252Binhibitors%25253A%252Ba%252Bcomprehensive%252Breview%252B2015%2525E2%252580%2525932020%26aulast%3DSpillier%26aufirst%3DQuentin%26date%3D2021%26date%3D2021%26volume%3D31%26issue%3D7%26spage%3D597%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingxue  Li</span>, <span class="hlFld-ContribAuthor ">Canrong  Wu</span>, <span class="hlFld-ContribAuthor ">Yueying  Yang</span>, <span class="hlFld-ContribAuthor ">Mengzhu  Zheng</span>, <span class="hlFld-ContribAuthor ">Silin  Yu</span>, <span class="hlFld-ContribAuthor ">Jinhui  Wang</span>, <span class="hlFld-ContribAuthor ">Lixia  Chen</span>, <span class="hlFld-ContribAuthor ">Hua  Li</span>. </span><span class="cited-content_cbyCitation_article-title">3-Phosphoglycerate dehydrogenase: a potential target for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Cellular Oncology</span><span> <strong>2021,</strong> <em>44 </em>
                                    (3)
                                     , 541-556. <a href="https://doi.org/10.1007/s13402-021-00599-9" title="DOI URL">https://doi.org/10.1007/s13402-021-00599-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13402-021-00599-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13402-021-00599-9%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520Oncology%26atitle%3D3-Phosphoglycerate%252Bdehydrogenase%25253A%252Ba%252Bpotential%252Btarget%252Bfor%252Bcancer%252Btreatment%26aulast%3DLi%26aufirst%3DMingxue%26date%3D2021%26date%3D2021%26volume%3D44%26issue%3D3%26spage%3D541%26epage%3D556" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefan  Bjelosevic</span>, <span class="hlFld-ContribAuthor ">Emily  Gruber</span>, <span class="hlFld-ContribAuthor ">Andrea  Newbold</span>, <span class="hlFld-ContribAuthor ">Carolyn  Shembrey</span>, <span class="hlFld-ContribAuthor ">Jennifer R.  Devlin</span>, <span class="hlFld-ContribAuthor ">Simon J.  Hogg</span>, <span class="hlFld-ContribAuthor ">Lev  Kats</span>, <span class="hlFld-ContribAuthor ">Izabela  Todorovski</span>, <span class="hlFld-ContribAuthor ">Zheng  Fan</span>, <span class="hlFld-ContribAuthor ">Thomas C.  Abrehart</span>, <span class="hlFld-ContribAuthor ">Giovanna  Pomilio</span>, <span class="hlFld-ContribAuthor ">Andrew  Wei</span>, <span class="hlFld-ContribAuthor ">Gareth P.  Gregory</span>, <span class="hlFld-ContribAuthor ">Stephin J.  Vervoort</span>, <span class="hlFld-ContribAuthor ">Kristin K.  Brown</span>, <span class="hlFld-ContribAuthor ">Ricky W.  Johnstone</span>. </span><span class="cited-content_cbyCitation_article-title">Serine Biosynthesis Is a Metabolic Vulnerability in FLT3-ITD–Driven Acute Myeloid Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Cancer Discovery</span><span> <strong>2021,</strong> <em>11 </em>
                                    (6)
                                     , 1582-1599. <a href="https://doi.org/10.1158/2159-8290.CD-20-0738" title="DOI URL">https://doi.org/10.1158/2159-8290.CD-20-0738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/2159-8290.CD-20-0738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F2159-8290.CD-20-0738%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Discovery%26atitle%3DSerine%252BBiosynthesis%252BIs%252Ba%252BMetabolic%252BVulnerability%252Bin%252BFLT3-ITD%2525E2%252580%252593Driven%252BAcute%252BMyeloid%252BLeukemia%26aulast%3DBjelosevic%26aufirst%3DStefan%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D6%26spage%3D1582%26epage%3D1599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Livingstone  Fultang</span>, <span class="hlFld-ContribAuthor ">Luciana  Gneo</span>, <span class="hlFld-ContribAuthor ">Carmela  De Santo</span>, <span class="hlFld-ContribAuthor ">Francis J.  Mussai</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Amino Acid Metabolic Vulnerabilities in Myeloid Malignancies. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2021,</strong> <em>11 </em><a href="https://doi.org/10.3389/fonc.2021.674720" title="DOI URL">https://doi.org/10.3389/fonc.2021.674720</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2021.674720&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2021.674720%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DTargeting%252BAmino%252BAcid%252BMetabolic%252BVulnerabilities%252Bin%252BMyeloid%252BMalignancies%26aulast%3DFultang%26aufirst%3DLivingstone%26date%3D2021%26date%3D2021%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia-Ying  Zhao</span>, <span class="hlFld-ContribAuthor ">Kai-Rui  Feng</span>, <span class="hlFld-ContribAuthor ">Feng  Wang</span>, <span class="hlFld-ContribAuthor ">Jian-Wei  Zhang</span>, <span class="hlFld-ContribAuthor ">Jay Fei  Cheng</span>, <span class="hlFld-ContribAuthor ">Guo-Qiang  Lin</span>, <span class="hlFld-ContribAuthor ">Dingding  Gao</span>, <span class="hlFld-ContribAuthor ">Ping  Tian</span>. </span><span class="cited-content_cbyCitation_article-title">A retrospective overview of PHGDH and its inhibitors for regulating cancer metabolism. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>217 </em>, 113379. <a href="https://doi.org/10.1016/j.ejmech.2021.113379" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113379</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113379%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252Bretrospective%252Boverview%252Bof%252BPHGDH%252Band%252Bits%252Binhibitors%252Bfor%252Bregulating%252Bcancer%252Bmetabolism%26aulast%3DZhao%26aufirst%3DJia-Ying%26date%3D2021%26volume%3D217%26spage%3D113379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kevin  Eade</span>, <span class="hlFld-ContribAuthor ">Marin L.  Gantner</span>, <span class="hlFld-ContribAuthor ">Joseph A.  Hostyk</span>, <span class="hlFld-ContribAuthor ">Takayuki  Nagasaki</span>, <span class="hlFld-ContribAuthor ">Sarah  Giles</span>, <span class="hlFld-ContribAuthor ">Regis  Fallon</span>, <span class="hlFld-ContribAuthor ">Sarah  Harkins-Perry</span>, <span class="hlFld-ContribAuthor ">Michelle  Baldini</span>, <span class="hlFld-ContribAuthor ">Esther W.  Lim</span>, <span class="hlFld-ContribAuthor ">Lea  Scheppke</span>, <span class="hlFld-ContribAuthor ">Michael I.  Dorrell</span>, <span class="hlFld-ContribAuthor ">Carolyn  Cai</span>, <span class="hlFld-ContribAuthor ">Evan H.  Baugh</span>, <span class="hlFld-ContribAuthor ">Charles J.  Wolock</span>, <span class="hlFld-ContribAuthor ">Martina  Wallace</span>, <span class="hlFld-ContribAuthor ">Rebecca B.  Berlow</span>, <span class="hlFld-ContribAuthor ">David B.  Goldstein</span>, <span class="hlFld-ContribAuthor ">Christian M.  Metallo</span>, <span class="hlFld-ContribAuthor ">Martin  Friedlander</span>, <span class="hlFld-ContribAuthor ">Rando  Allikmets</span>. </span><span class="cited-content_cbyCitation_article-title">Serine biosynthesis defect due to haploinsufficiency of PHGDH causes retinal disease. </span><span class="cited-content_cbyCitation_journal-name">Nature Metabolism</span><span> <strong>2021,</strong> <em>3 </em>
                                    (3)
                                     , 366-377. <a href="https://doi.org/10.1038/s42255-021-00361-3" title="DOI URL">https://doi.org/10.1038/s42255-021-00361-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42255-021-00361-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42255-021-00361-3%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Metabolism%26atitle%3DSerine%252Bbiosynthesis%252Bdefect%252Bdue%252Bto%252Bhaploinsufficiency%252Bof%252BPHGDH%252Bcauses%252Bretinal%252Bdisease%26aulast%3DEade%26aufirst%3DKevin%26date%3D2021%26date%3D2021%26volume%3D3%26issue%3D3%26spage%3D366%26epage%3D377" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shauni L.  Geeraerts</span>, <span class="hlFld-ContribAuthor ">Elien  Heylen</span>, <span class="hlFld-ContribAuthor ">Kim  De Keersmaecker</span>, <span class="hlFld-ContribAuthor ">Kim R.  Kampen</span>. </span><span class="cited-content_cbyCitation_article-title">The ins and outs of serine and glycine metabolism in cancer. </span><span class="cited-content_cbyCitation_journal-name">Nature Metabolism</span><span> <strong>2021,</strong> <em>3 </em>
                                    (2)
                                     , 131-141. <a href="https://doi.org/10.1038/s42255-020-00329-9" title="DOI URL">https://doi.org/10.1038/s42255-020-00329-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42255-020-00329-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42255-020-00329-9%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Metabolism%26atitle%3DThe%252Bins%252Band%252Bouts%252Bof%252Bserine%252Band%252Bglycine%252Bmetabolism%252Bin%252Bcancer%26aulast%3DGeeraerts%26aufirst%3DShauni%2BL.%26date%3D2021%26date%3D2021%26volume%3D3%26issue%3D2%26spage%3D131%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Keeley G.  Hewton</span>, <span class="hlFld-ContribAuthor ">Amritpal S.  Johal</span>, <span class="hlFld-ContribAuthor ">Seth J.  Parker</span>. </span><span class="cited-content_cbyCitation_article-title">Transporters at the Interface between Cytosolic and Mitochondrial Amino Acid Metabolism. </span><span class="cited-content_cbyCitation_journal-name">Metabolites</span><span> <strong>2021,</strong> <em>11 </em>
                                    (2)
                                     , 112. <a href="https://doi.org/10.3390/metabo11020112" title="DOI URL">https://doi.org/10.3390/metabo11020112</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/metabo11020112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmetabo11020112%26sid%3Dliteratum%253Aachs%26jtitle%3DMetabolites%26atitle%3DTransporters%252Bat%252Bthe%252BInterface%252Bbetween%252BCytosolic%252Band%252BMitochondrial%252BAmino%252BAcid%252BMetabolism%26aulast%3DHewton%26aufirst%3DKeeley%2BG.%26date%3D2021%26date%3D2021%26volume%3D11%26issue%3D2%26spage%3D112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hanyu  Xu</span>, <span class="hlFld-ContribAuthor ">Xiaoyu  Qing</span>, <span class="hlFld-ContribAuthor ">Qian  Wang</span>, <span class="hlFld-ContribAuthor ">Chunmei  Li</span>, <span class="hlFld-ContribAuthor ">Luhua  Lai</span>. </span><span class="cited-content_cbyCitation_article-title">Dimerization of PHGDH via the catalytic unit is essential for its enzymatic function. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2021,</strong> <em>296 </em>, 100572. <a href="https://doi.org/10.1016/j.jbc.2021.100572" title="DOI URL">https://doi.org/10.1016/j.jbc.2021.100572</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jbc.2021.100572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jbc.2021.100572%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DDimerization%252Bof%252BPHGDH%252Bvia%252Bthe%252Bcatalytic%252Bunit%252Bis%252Bessential%252Bfor%252Bits%252Benzymatic%252Bfunction%26aulast%3DXu%26aufirst%3DHanyu%26date%3D2021%26volume%3D296%26spage%3D100572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yun Pyo  Kang</span>, <span class="hlFld-ContribAuthor ">Aimee  Falzone</span>, <span class="hlFld-ContribAuthor ">Min  Liu</span>, <span class="hlFld-ContribAuthor ">Paloma  González-Sánchez</span>, <span class="hlFld-ContribAuthor ">Bo-Hyun  Choi</span>, <span class="hlFld-ContribAuthor ">Jonathan L.  Coloff</span>, <span class="hlFld-ContribAuthor ">James J.  Saller</span>, <span class="hlFld-ContribAuthor ">Florian A.  Karreth</span>, <span class="hlFld-ContribAuthor ">Gina M.  DeNicola</span>. </span><span class="cited-content_cbyCitation_article-title">PHGDH supports liver ceramide synthesis and sustains lipid homeostasis. </span><span class="cited-content_cbyCitation_journal-name">Cancer & Metabolism</span><span> <strong>2020,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s40170-020-00212-x" title="DOI URL">https://doi.org/10.1186/s40170-020-00212-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s40170-020-00212-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs40170-020-00212-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520%2526%2520Metabolism%26atitle%3DPHGDH%252Bsupports%252Bliver%252Bceramide%252Bsynthesis%252Band%252Bsustains%252Blipid%252Bhomeostasis%26aulast%3DKang%26aufirst%3DYun%2BPyo%26date%3D2020%26date%3D2020%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Murtas</span>, <span class="hlFld-ContribAuthor ">Giorgia Letizia  Marcone</span>, <span class="hlFld-ContribAuthor ">Silvia  Sacchi</span>, <span class="hlFld-ContribAuthor ">Loredano  Pollegioni</span>. </span><span class="cited-content_cbyCitation_article-title">L-serine synthesis via the phosphorylated pathway in humans. </span><span class="cited-content_cbyCitation_journal-name">Cellular and Molecular Life Sciences</span><span> <strong>2020,</strong> <em>77 </em>
                                    (24)
                                     , 5131-5148. <a href="https://doi.org/10.1007/s00018-020-03574-z" title="DOI URL">https://doi.org/10.1007/s00018-020-03574-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00018-020-03574-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00018-020-03574-z%26sid%3Dliteratum%253Aachs%26jtitle%3DCellular%2520and%2520Molecular%2520Life%2520Sciences%26atitle%3DL-serine%252Bsynthesis%252Bvia%252Bthe%252Bphosphorylated%252Bpathway%252Bin%252Bhumans%26aulast%3DMurtas%26aufirst%3DGiulia%26date%3D2020%26date%3D2020%26volume%3D77%26issue%3D24%26spage%3D5131%26epage%3D5148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Karsten  Krug</span>, <span class="hlFld-ContribAuthor ">Eric J.  Jaehnig</span>, <span class="hlFld-ContribAuthor ">Shankha  Satpathy</span>, <span class="hlFld-ContribAuthor ">Lili  Blumenberg</span>, <span class="hlFld-ContribAuthor ">Alla  Karpova</span>, <span class="hlFld-ContribAuthor ">Meenakshi  Anurag</span>, <span class="hlFld-ContribAuthor ">George  Miles</span>, <span class="hlFld-ContribAuthor ">Philipp  Mertins</span>, <span class="hlFld-ContribAuthor ">Yifat  Geffen</span>, <span class="hlFld-ContribAuthor ">Lauren C.  Tang</span>, <span class="hlFld-ContribAuthor ">David I.  Heiman</span>, <span class="hlFld-ContribAuthor ">Song  Cao</span>, <span class="hlFld-ContribAuthor ">Yosef E.  Maruvka</span>, <span class="hlFld-ContribAuthor ">Jonathan T.  Lei</span>, <span class="hlFld-ContribAuthor ">Chen  Huang</span>, <span class="hlFld-ContribAuthor ">Ramani B.  Kothadia</span>, <span class="hlFld-ContribAuthor ">Antonio  Colaprico</span>, <span class="hlFld-ContribAuthor ">Chet  Birger</span>, <span class="hlFld-ContribAuthor ">Jarey  Wang</span>, <span class="hlFld-ContribAuthor ">Yongchao  Dou</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Zhiao  Shi</span>, <span class="hlFld-ContribAuthor ">Yuxing  Liao</span>, <span class="hlFld-ContribAuthor ">Maciej  Wiznerowicz</span>, <span class="hlFld-ContribAuthor ">Matthew A.  Wyczalkowski</span>, <span class="hlFld-ContribAuthor ">Xi Steven  Chen</span>, <span class="hlFld-ContribAuthor ">Jacob J.  Kennedy</span>, <span class="hlFld-ContribAuthor ">Amanda G.  Paulovich</span>, <span class="hlFld-ContribAuthor ">Mathangi  Thiagarajan</span>, <span class="hlFld-ContribAuthor ">Christopher R.  Kinsinger</span>, <span class="hlFld-ContribAuthor ">Tara  Hiltke</span>, <span class="hlFld-ContribAuthor ">Emily S.  Boja</span>, <span class="hlFld-ContribAuthor ">Mehdi  Mesri</span>, <span class="hlFld-ContribAuthor ">Ana I.  Robles</span>, <span class="hlFld-ContribAuthor ">Henry  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Thomas F.  Westbrook</span>, <span class="hlFld-ContribAuthor ">Li  Ding</span>, <span class="hlFld-ContribAuthor ">Gad  Getz</span>, <span class="hlFld-ContribAuthor ">Karl R.  Clauser</span>, <span class="hlFld-ContribAuthor ">David  Fenyö</span>, <span class="hlFld-ContribAuthor ">Kelly V.  Ruggles</span>, <span class="hlFld-ContribAuthor ">Bing  Zhang</span>, <span class="hlFld-ContribAuthor ">D.R.  Mani</span>, <span class="hlFld-ContribAuthor ">Steven A.  Carr</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Ellis</span>, <span class="hlFld-ContribAuthor ">Michael A.  Gillette</span>, <span class="hlFld-ContribAuthor ">Shayan C.  Avanessian</span>, <span class="hlFld-ContribAuthor ">Shuang  Cai</span>, <span class="hlFld-ContribAuthor ">Daniel  Chan</span>, <span class="hlFld-ContribAuthor ">Xian  Chen</span>, <span class="hlFld-ContribAuthor ">Nathan J.  Edwards</span>, <span class="hlFld-ContribAuthor ">Andrew N.  Hoofnagle</span>, <span class="hlFld-ContribAuthor ">M. Harry  Kane</span>, <span class="hlFld-ContribAuthor ">Karen A.  Ketchum</span>, <span class="hlFld-ContribAuthor ">Eric  Kuhn</span>, <span class="hlFld-ContribAuthor ">Douglas A.  Levine</span>, <span class="hlFld-ContribAuthor ">Shunqiang  Li</span>, <span class="hlFld-ContribAuthor ">Daniel C.  Liebler</span>, <span class="hlFld-ContribAuthor ">Tao  Liu</span>, <span class="hlFld-ContribAuthor ">Jingqin  Luo</span>, <span class="hlFld-ContribAuthor ">Subha  Madhavan</span>, <span class="hlFld-ContribAuthor ">Chris  Maher</span>, <span class="hlFld-ContribAuthor ">Jason E.  McDermott</span>, <span class="hlFld-ContribAuthor ">Peter B.  McGarvey</span>, <span class="hlFld-ContribAuthor ">Mauricio  Oberti</span>, <span class="hlFld-ContribAuthor ">Akhilesh  Pandey</span>, <span class="hlFld-ContribAuthor ">Samuel H.  Payne</span>, <span class="hlFld-ContribAuthor ">David F.  Ransohoff</span>, <span class="hlFld-ContribAuthor ">Robert C.  Rivers</span>, <span class="hlFld-ContribAuthor ">Karin D.  Rodland</span>, <span class="hlFld-ContribAuthor ">Paul  Rudnick</span>, <span class="hlFld-ContribAuthor ">Melinda E.  Sanders</span>, <span class="hlFld-ContribAuthor ">Kenna M.  Shaw</span>, <span class="hlFld-ContribAuthor ">Ie-Ming  Shih</span>, <span class="hlFld-ContribAuthor ">Robbert J.C.  Slebos</span>, <span class="hlFld-ContribAuthor ">Richard D.  Smith</span>, <span class="hlFld-ContribAuthor ">Michael  Snyder</span>, <span class="hlFld-ContribAuthor ">Stephen E.  Stein</span>, <span class="hlFld-ContribAuthor ">David L.  Tabb</span>, <span class="hlFld-ContribAuthor ">Ratna R.  Thangudu</span>, <span class="hlFld-ContribAuthor ">Stefani  Thomas</span>, <span class="hlFld-ContribAuthor ">Yue  Wang</span>, <span class="hlFld-ContribAuthor ">Forest M.  White</span>, <span class="hlFld-ContribAuthor ">Jeffrey R.  Whiteaker</span>, <span class="hlFld-ContribAuthor ">Gordon A.  Whiteley</span>, <span class="hlFld-ContribAuthor ">Hui  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhen  Zhang</span>, <span class="hlFld-ContribAuthor ">Yingming  Zhao</span>, <span class="hlFld-ContribAuthor ">Heng  Zhu</span>, <span class="hlFld-ContribAuthor ">Lisa J.  Zimmerman</span>. </span><span class="cited-content_cbyCitation_article-title">Proteogenomic Landscape of Breast Cancer Tumorigenesis and Targeted Therapy. </span><span class="cited-content_cbyCitation_journal-name">Cell</span><span> <strong>2020,</strong> <em>183 </em>
                                    (5)
                                     , 1436-1456.e31. <a href="https://doi.org/10.1016/j.cell.2020.10.036" title="DOI URL">https://doi.org/10.1016/j.cell.2020.10.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cell.2020.10.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cell.2020.10.036%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%26atitle%3DProteogenomic%252BLandscape%252Bof%252BBreast%252BCancer%252BTumorigenesis%252Band%252BTargeted%252BTherapy%26aulast%3DKrug%26aufirst%3DKarsten%26date%3D2020%26volume%3D183%26issue%3D5%26spage%3D1436%26epage%3D1456.e31" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiawei  Xiang</span>, <span class="hlFld-ContribAuthor ">Lei  Tao</span>, <span class="hlFld-ContribAuthor ">Yue  Zhou</span>, <span class="hlFld-ContribAuthor ">Yuping  Tan</span>, <span class="hlFld-ContribAuthor ">Zicheng  Li</span>, <span class="hlFld-ContribAuthor ">Yinglan  Zhao</span>, <span class="hlFld-ContribAuthor ">Qingxiang  Sun</span>, <span class="hlFld-ContribAuthor ">Youfu  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of innovative thiourea derivatives as PHGDH inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Papers</span><span> <strong>2020,</strong> <em>74 </em>
                                    (11)
                                     , 3873-3886. <a href="https://doi.org/10.1007/s11696-020-01188-0" title="DOI URL">https://doi.org/10.1007/s11696-020-01188-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11696-020-01188-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11696-020-01188-0%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Papers%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Binnovative%252Bthiourea%252Bderivatives%252Bas%252BPHGDH%252Binhibitors%26aulast%3DXiang%26aufirst%3DJiawei%26date%3D2020%26date%3D2020%26volume%3D74%26issue%3D11%26spage%3D3873%26epage%3D3886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert M.  Li</span>, <span class="hlFld-ContribAuthor ">Jiangbin  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">The PHGDH enigma: Do cancer cells only need serine or also a redox modulator?. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2020,</strong> <em>476 </em>, 97-105. <a href="https://doi.org/10.1016/j.canlet.2020.01.036" title="DOI URL">https://doi.org/10.1016/j.canlet.2020.01.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2020.01.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2020.01.036%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DThe%252BPHGDH%252Benigma%25253A%252BDo%252Bcancer%252Bcells%252Bonly%252Bneed%252Bserine%252Bor%252Balso%252Ba%252Bredox%252Bmodulator%25253F%26aulast%3DLi%26aufirst%3DAlbert%2BM.%26date%3D2020%26volume%3D476%26spage%3D97%26epage%3D105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Quentin  Spillier</span>, <span class="hlFld-ContribAuthor ">Séverine  Ravez</span>, <span class="hlFld-ContribAuthor ">Judith  Unterlass</span>, <span class="hlFld-ContribAuthor ">Cyril  Corbet</span>, <span class="hlFld-ContribAuthor ">Charline  Degavre</span>, <span class="hlFld-ContribAuthor ">Olivier  Feron</span>, <span class="hlFld-ContribAuthor ">Raphaël  Frédérick</span>. </span><span class="cited-content_cbyCitation_article-title">Structure–Activity Relationships (SARs) of α-Ketothioamides as Inhibitors of Phosphoglycerate Dehydrogenase (PHGDH). </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2020,</strong> <em>13 </em>
                                    (2)
                                     , 20. <a href="https://doi.org/10.3390/ph13020020" title="DOI URL">https://doi.org/10.3390/ph13020020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph13020020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph13020020%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DStructure%2525E2%252580%252593Activity%252BRelationships%252B%252528SARs%252529%252Bof%252B%2525CE%2525B1-Ketothioamides%252Bas%252BInhibitors%252Bof%252BPhosphoglycerate%252BDehydrogenase%252B%252528PHGDH%252529%26aulast%3DSpillier%26aufirst%3DQuentin%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D2%26spage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0029.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0029.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0001.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Known roles of PHGDH in tumor metabolism and tumor epigenetics.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0002.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Overview of selected published PHGDH inhibitors and reported binding/inhibition data.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Schematic representation of the hit-finding process. Saturation transfer difference NMR (STD-NMR)-guided-based FBS screening on the right (blue) and luciferase-based HTS screening on the left (gray).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Validated hit from the STD-NMR FBS screen: phenylpyrazole-5-carboxamide <b>9</b> cocrystallized with PHGDH showing the main polar interaction to D174 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH">6RIH</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Electron density map of the ternary complex of cofactor NAD<sup>+</sup> and substrate <b>3-PG</b> in PHGDH (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0013.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. General Overview of the Hit Optimization<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Double-tracked strategy: merging learnings from HTS and FBS leads to <b>BI-4924</b> and <b>BI-4916</b>.</p></p></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray structure confirmation of compound <b>15</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3">6RJ3</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0007.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Long-range electrostatics in the PHGDH phosphate channel exemplarily shown on acyl sulfonamide <b>39</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5">6RJ5</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0008.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Addressing a new polar interaction on I155 BB-NH with kinked acyl sulfonamide <b>40</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2">6RJ2</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0009.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Assay adaption for the profiling of potent PHGDH inhibitors. Red dots depict the reference assay @19 μM NAD<sup>+</sup>. Brown dots depict another run under the same conditions. Yellow dots indicate shifted IC<sub>50</sub> values using 250 μM NAD<sup>+</sup>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0010.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. (A) Binding mode of <b>BI-4924</b> (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6">6RJ6</a>); (B) chemical structure of <b>BI-4924</b>; (C) additional in vitro data for <b>BI-4924</b>; m, murine; r, rat; h, human. CaCo-2 permeability: ab, apical–basolateral permeation/transport; ba, basolateral–apical permeation/transport; FCS, fetal calf serum.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0011.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Selectivity profile of <b>BI-4924</b> obtained by quantitative chemical proteomics. Proteins (orange dots, dehydrogenases; blue dots, other proteins) from lysed MDA-MB-468 cell are displaced from affinity matrices functionalized with compounds <b>70</b> or <b>71</b> (see <a href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx" class="ext-link">Supporting Information</a>) by competition with 10 μM <b>BI-4924</b>. Data are plotted log 2 of the fold change vs DMSO control against −log 10 of the <i>P</i> value per protein (ANOVA, <i>N</i> = 3). All <i>t</i>-tests-tests performed were two-tailed. <i>t</i>-tests assuming equal variances.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/medium/jm9b00718_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0012.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Ratio of mean cellular concentrations of acid <b>BI-4924</b> (black diamonds) at various time points after the addition of 3 μM ester <b>BI-4916</b> vs 3 μM acid <b>BI-4924</b> into the medium. Error bars indicate the standard Gaussian error calculated from the standard deviations of the individual assays. The gray dashed line indicates the average accumulation factor for a time period of 72 h.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-17/acs.jmedchem.9b00718/20190905/images/large/jm9b00718_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00718&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i125">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 45 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of Cancer: The Next Generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+Cancer%3A+The+Next+Generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lgHpdBR7o0B6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520Cancer%253A%2520The%2520Next%2520Generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zogg, C. K.</span></span> <span> </span><span class="NLM_article-title">Phosphoglycerate Dehydrogenase: Potential Therapeutic Target and Putative Metabolic Oncogene</span>. <i>J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>2014</i></span>, <span class="NLM_elocation-id">524101</span> <span class="refDoi"> DOI: 10.1155/2014/524101</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1155%2F2014%2F524101" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=25574168" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC2MvkvFOitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2014&publication_year=2014&author=C.+K.+Zogg&title=Phosphoglycerate+Dehydrogenase%3A+Potential+Therapeutic+Target+and+Putative+Metabolic+Oncogene&doi=10.1155%2F2014%2F524101"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoglycerate dehydrogenase: potential therapeutic target and putative metabolic oncogene</span></div><div class="casAuthors">Zogg Cheryl K</div><div class="citationInfo"><span class="NLM_cas:title">Journal of oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">2014</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">524101</span>
        ISSN:<span class="NLM_cas:issn">1687-8450</span>.
    </div><div class="casAbstract">Exemplified by cancer cells' preference for glycolysis, for example, the Warburg effect, altered metabolism in tumorigenesis has emerged as an important aspect of cancer in the past 10-20 years.  Whether due to changes in regulatory tumor suppressors/oncogenes or by acting as metabolic oncogenes themselves, enzymes involved in the complex network of metabolic pathways are being studied to understand their role and assess their utility as therapeutic targets.  Conversion of glycolytic intermediate 3-phosphoglycerate into phosphohydroxypyruvate by the enzyme phosphoglycerate dehydrogenase (PHGDH)-a rate-limiting step in the conversion of 3-phosphoglycerate to serine-represents one such mechanism.  Forgotten since classic animal studies in the 1980s, the role of PHGDH as a potential therapeutic target and putative metabolic oncogene has recently reemerged following publication of two prominent papers near-simultaneously in 2011.  Since that time, numerous studies and a host of metabolic explanations have been put forward in an attempt to understand the results observed.  In this paper, I review the historic progression of our understanding of the role of PHGDH in cancer from the early work by Snell through its reemergence and rise to prominence, culminating in an assessment of subsequent work and what it means for the future of PHGDH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQQpDxpOqoFfO2qxNLjzXe8fW6udTcc2eawOsm5YdELhLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MvkvFOitQ%253D%253D&md5=831a16d13000c1dec2fd435daf1b089b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1155%2F2014%2F524101&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1155%252F2014%252F524101%26sid%3Dliteratum%253Aachs%26aulast%3DZogg%26aufirst%3DC.%2BK.%26atitle%3DPhosphoglycerate%2520Dehydrogenase%253A%2520Potential%2520Therapeutic%2520Target%2520and%2520Putative%2520Metabolic%2520Oncogene%26jtitle%3DJ.%2520Oncol.%26date%3D2014%26volume%3D2014%26doi%3D10.1155%2F2014%2F524101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Possemato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marks, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaul, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birsoy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethumadhavan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woo, H.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jha, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, F. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stransky, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsun, Z.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowley, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barretina, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalaany, N. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Root, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mino-Kenudson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brachtel, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driggers, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">Functional Genomics Reveal that the Serine Synthesis Pathway is Essential in Breast Cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>476</i></span>,  <span class="NLM_fpage">346</span>, <span class="refDoi"> DOI: 10.1038/nature10350</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnature10350" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21760589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVOku7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=476&publication_year=2011&pages=346&author=R.+Possematoauthor=K.+M.+Marksauthor=Y.+D.+Shaulauthor=M.+E.+Pacoldauthor=D.+Kimauthor=K.+Birsoyauthor=S.+Sethumadhavanauthor=H.-K.+Wooauthor=H.+G.+Jangauthor=A.+K.+Jhaauthor=W.+W.+Chenauthor=F.+G.+Barrettauthor=N.+Stranskyauthor=Z.-Y.+Tsunauthor=G.+S.+Cowleyauthor=J.+Barretinaauthor=N.+Y.+Kalaanyauthor=P.+P.+Hsuauthor=K.+Ottinaauthor=A.+M.+Chanauthor=B.+Yuanauthor=L.+A.+Garrawayauthor=D.+E.+Rootauthor=M.+Mino-Kenudsonauthor=E.+F.+Brachtelauthor=E.+M.+Driggersauthor=D.+M.+Sabatini&title=Functional+Genomics+Reveal+that+the+Serine+Synthesis+Pathway+is+Essential+in+Breast+Cancer&doi=10.1038%2Fnature10350"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Functional genomics reveal that the serine synthesis pathway is essential in breast cancer</span></div><div class="casAuthors">Possemato, Richard; Marks, Kevin M.; Shaul, Yoav D.; Pacold, Michael E.; Kim, Dohoon; Birsoy, Kivanc; Sethumadhavan, Shalini; Woo, Hin-Koon; Jang, Hyun G.; Jha, Abhishek K.; Chen, Walter W.; Barrett, Francesca G.; Stransky, Nicolas; Tsun, Zhi-Yang; Cowley, Glenn S.; Barretina, Jordi; Kalaany, Nada Y.; Hsu, Peggy P.; Ottina, Kathleen; Chan, Albert M.; Yuan, Bingbing; Garraway, Levi A.; Root, David E.; Mino-Kenudson, Mari; Brachtel, Elena F.; Driggers, Edward M.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">476</span>
        (<span class="NLM_cas:issue">7360</span>),
    <span class="NLM_cas:pages">346-350</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer cells adapt their metabolic processes to drive macromol. biosynthesis for rapid cell growth and proliferation.  RNA interference (RNAi)-based loss-of-function screening has proven powerful for the identification of new and interesting cancer targets, and recent studies have used this technol. in vivo to identify novel tumor suppressor genes.  Here we developed a method for identifying novel cancer targets via neg.-selection RNAi screening using a human breast cancer xenograft model at an orthotopic site in the mouse.  Using this method, we screened a set of metabolic genes assocd. with aggressive breast cancer and stemness to identify those required for in vivo tumorigenesis.  Among the genes identified, phosphoglycerate dehydrogenase (PHGDH) is in a genomic region of recurrent copy no. gain in breast cancer and PHGDH protein levels are elevated in 70% of estrogen receptor (ER)-neg. breast cancers.  PHGDH catalyzes the first step in the serine biosynthesis pathway, and breast cancer cells with high PHGDH expression have increased serine synthesis flux.  Suppression of PHGDH in cell lines with elevated PHGDH expression, but not in those without, causes a strong decrease in cell proliferation and a redn. in serine synthesis.  We find that PHGDH suppression does not affect intracellular serine levels, but causes a drop in the levels of α-ketoglutarate, another output of the pathway and a tricarboxylic acid (TCA) cycle intermediate.  In cells with high PHGDH expression, the serine synthesis pathway contributes approx. 50% of the total anaplerotic flux of glutamine into the TCA cycle.  These results reveal that certain breast cancers are dependent upon increased serine pathway flux caused by PHGDH overexpression and demonstrate the utility of in vivo neg.-selection RNAi screens for finding potential anticancer targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq79QL1LpmDqLVg90H21EOLACvtfcHk0liLoA7PdUd3Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVOku7rL&md5=10d8df92c9fb450ee7edca54bf227558</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature10350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10350%26sid%3Dliteratum%253Aachs%26aulast%3DPossemato%26aufirst%3DR.%26aulast%3DMarks%26aufirst%3DK.%2BM.%26aulast%3DShaul%26aufirst%3DY.%2BD.%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DBirsoy%26aufirst%3DK.%26aulast%3DSethumadhavan%26aufirst%3DS.%26aulast%3DWoo%26aufirst%3DH.-K.%26aulast%3DJang%26aufirst%3DH.%2BG.%26aulast%3DJha%26aufirst%3DA.%2BK.%26aulast%3DChen%26aufirst%3DW.%2BW.%26aulast%3DBarrett%26aufirst%3DF.%2BG.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DTsun%26aufirst%3DZ.-Y.%26aulast%3DCowley%26aufirst%3DG.%2BS.%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DKalaany%26aufirst%3DN.%2BY.%26aulast%3DHsu%26aufirst%3DP.%2BP.%26aulast%3DOttina%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DA.%2BM.%26aulast%3DYuan%26aufirst%3DB.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DRoot%26aufirst%3DD.%2BE.%26aulast%3DMino-Kenudson%26aufirst%3DM.%26aulast%3DBrachtel%26aufirst%3DE.%2BF.%26aulast%3DDriggers%26aufirst%3DE.%2BM.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DFunctional%2520Genomics%2520Reveal%2520that%2520the%2520Serine%2520Synthesis%2520Pathway%2520is%2520Essential%2520in%2520Breast%2520Cancer%26jtitle%3DNature%26date%3D2011%26volume%3D476%26spage%3D346%26doi%3D10.1038%2Fnature10350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J.
W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grassian, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melman, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bass, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heffron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metallo, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muranen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharfi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anastasiou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vokes, N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beroukhim, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephanopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ligon, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">Phosphoglycerate Dehydrogenase Diverts Ggycolytic Flux and Contributes to Oncogenesis</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">869</span>, <span class="refDoi"> DOI: 10.1038/ng.890</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fng.890" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21804546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=869&author=J.%0AW.+Locasaleauthor=A.+R.+Grassianauthor=T.+Melmanauthor=C.+A.+Lyssiotisauthor=K.+R.+Mattainiauthor=A.+J.+Bassauthor=G.+Heffronauthor=C.+M.+Metalloauthor=T.+Muranenauthor=H.+Sharfiauthor=A.+T.+Sasakiauthor=D.+Anastasiouauthor=E.+Mullarkyauthor=N.+I.+Vokesauthor=M.+Sasakiauthor=R.+Beroukhimauthor=G.+Stephanopoulosauthor=A.+H.+Ligonauthor=M.+Meyersonauthor=A.+L.+Richardsonauthor=L.+Chinauthor=G.+Wagnerauthor=J.+M.+Asaraauthor=J.+S.+Bruggeauthor=L.+C.+Cantleyauthor=M.+G.+Vander+Heiden&title=Phosphoglycerate+Dehydrogenase+Diverts+Ggycolytic+Flux+and+Contributes+to+Oncogenesis&doi=10.1038%2Fng.890"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoglycerate dehydrogenase diverts glycolytic flux and contributes to oncogenesis</span></div><div class="casAuthors">Locasale, Jason W.; Grassian, Alexandra R.; Melman, Tamar; Lyssiotis, Costas A.; Mattaini, Katherine R.; Bass, Adam J.; Heffron, Gregory; Metallo, Christian M.; Muranen, Taru; Sharfi, Hadar; Sasaki, Atsuo T.; Anastasiou, Dimitrios; Mullarky, Edouard; Vokes, Natalie I.; Sasaki, Mika; Beroukhim, Rameen; Stephanopoulos, Gregory; Ligon, Azra H.; Meyerson, Matthew; Richardson, Andrea L.; Chin, Lynda; Wagner, Gerhard; Asara, John M.; Brugge, Joan S.; Cantley, Lewis C.; Vander Heiden, Matthew G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">869-874</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most tumors exhibit increased glucose metab. to lactate, however, the extent to which glucose-derived metabolic fluxes are used for alternative processes is poorly understood.  Using a metabolomics approach with isotope labeling, we found that in some cancer cells a relatively large amt. of glycolytic carbon is diverted into serine and glycine metab. through phosphoglycerate dehydrogenase (PHGDH).  An anal. of human cancers showed that PHGDH is recurrently amplified in a genomic region of focal copy no. gain most commonly found in melanoma.  Decreasing PHGDH expression impaired proliferation in amplified cell lines.  Increased expression was also assocd. with breast cancer subtypes, and ectopic expression of PHGDH in mammary epithelial cells disrupted acinar morphogenesis and induced other phenotypic alterations that may predispose cells to transformation.  Our findings show that the diversion of glycolytic flux into a specific alternate pathway can be selected during tumor development and may contribute to the pathogenesis of human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIkj0NPfUByrVg90H21EOLACvtfcHk0lhBm2viAbFuVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXpsFKqsLw%253D&md5=b903036172528f331b63ce84694ce851</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fng.890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.890%26sid%3Dliteratum%253Aachs%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DGrassian%26aufirst%3DA.%2BR.%26aulast%3DMelman%26aufirst%3DT.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DBass%26aufirst%3DA.%2BJ.%26aulast%3DHeffron%26aufirst%3DG.%26aulast%3DMetallo%26aufirst%3DC.%2BM.%26aulast%3DMuranen%26aufirst%3DT.%26aulast%3DSharfi%26aufirst%3DH.%26aulast%3DSasaki%26aufirst%3DA.%2BT.%26aulast%3DAnastasiou%26aufirst%3DD.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DVokes%26aufirst%3DN.%2BI.%26aulast%3DSasaki%26aufirst%3DM.%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DStephanopoulos%26aufirst%3DG.%26aulast%3DLigon%26aufirst%3DA.%2BH.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DRichardson%26aufirst%3DA.%2BL.%26aulast%3DChin%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DG.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DBrugge%26aufirst%3DJ.%2BS.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DPhosphoglycerate%2520Dehydrogenase%2520Diverts%2520Ggycolytic%2520Flux%2520and%2520Contributes%2520to%2520Oncogenesis%26jtitle%3DNat.%2520Genet.%26date%3D2011%26volume%3D43%26spage%3D869%26doi%3D10.1038%2Fng.890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span> <span> </span><span class="NLM_article-title">PHGDH Amplification and Altered Glucose Metabolism in Human Melanoma: PHGDH Amplification and Altered Glucose Metabolism</span>. <i>Pigm. Cell Melanoma Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1112</span>– <span class="NLM_lpage">1115</span>, <span class="refDoi"> DOI: 10.1111/j.1755-148X.2011.00919.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1111%2Fj.1755-148X.2011.00919.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21981974" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC38Xns1yksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2011&pages=1112-1115&author=E.+Mullarkyauthor=K.+R.+Mattainiauthor=M.+G.+Vander+Heidenauthor=L.+C.+Cantleyauthor=J.+W.+Locasale&title=PHGDH+Amplification+and+Altered+Glucose+Metabolism+in+Human+Melanoma%3A+PHGDH+Amplification+and+Altered+Glucose+Metabolism&doi=10.1111%2Fj.1755-148X.2011.00919.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH amplification and altered glucose metabolism in human melanoma</span></div><div class="casAuthors">Mullarky, Edouard; Mattaini, Katherine R.; Vander Heiden, Matthew G.; Cantley, Lewis C.; Locasale, Jason W.</div><div class="citationInfo"><span class="NLM_cas:title">Pigment Cell & Melanoma Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1112-1115</span>CODEN:
                <span class="NLM_cas:coden">PCMRCM</span>;
        ISSN:<span class="NLM_cas:issn">1755-1471</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The metabolic requirements of cancer cells differ from that of their normal counterparts.  To support their proliferation, cancer cells switch to a fermentative metab. that is thought to support biomass prodn.  Instances where metabolic enzymes promote tumorigenesis remain rare.  However, an enzyme involved in the de novo synthesis of serine, 3-phosphoglycerate dehydrogenase (PHGDH), was recently identified as a putative oncogene.  The potential mechanisms by which PHGDH promotes cancer are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbsn22yOds6rVg90H21EOLACvtfcHk0lhBm2viAbFuVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xns1yksA%253D%253D&md5=9b30f3f50eb60fd2212be45a7e521130</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1111%2Fj.1755-148X.2011.00919.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1755-148X.2011.00919.x%26sid%3Dliteratum%253Aachs%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26atitle%3DPHGDH%2520Amplification%2520and%2520Altered%2520Glucose%2520Metabolism%2520in%2520Human%2520Melanoma%253A%2520PHGDH%2520Amplification%2520and%2520Altered%2520Glucose%2520Metabolism%26jtitle%3DPigm.%2520Cell%2520Melanoma%2520Res.%26date%3D2011%26volume%3D24%26spage%3D1112%26epage%3D1115%26doi%3D10.1111%2Fj.1755-148X.2011.00919.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span> <span> </span><span class="NLM_article-title">PHGDH is an Independent Prognosis Marker and Contributes Cell Proliferation, Migration and Invasion in Human Pancreatic Cancer</span>. <i>Gene</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>642</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1016/j.gene.2017.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.gene.2017.11.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29128633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2nu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=642&publication_year=2018&pages=43-50&author=Z.+Songauthor=C.+Fengauthor=Y.+Luauthor=Y.+Linauthor=C.+Dong&title=PHGDH+is+an+Independent+Prognosis+Marker+and+Contributes+Cell+Proliferation%2C+Migration+and+Invasion+in+Human+Pancreatic+Cancer&doi=10.1016%2Fj.gene.2017.11.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH is an independent prognosis marker and contributes cell proliferation, migration and invasion in human pancreatic cancer</span></div><div class="casAuthors">Song, Zhiwang; Feng, Chan; Lu, Yonglin; Lin, Yun; Dong, Chunyan</div><div class="citationInfo"><span class="NLM_cas:title">Gene</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">642</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-50</span>CODEN:
                <span class="NLM_cas:coden">GENED6</span>;
        ISSN:<span class="NLM_cas:issn">0378-1119</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">To investigate the expression, clin. significance, biol. function, and the potential mechanism of PHGDH in pancreatic cancer.  The expression of PHGDH in human pancreatic cancer tissues and corresponding adjacent normal tissues were analyzed through immunohistochem. staining.  Simultaneously, the assocn. between the PHGDH expression and the clinicopathol. parameters and OS and DFS was evaluated.  Human pancreatic cancer cell line BxPC-3 and SW1990 were selected to investigate the effect of PHGDH knockdown on cell proliferation, migration, and invasion.  In addn., we performed western blot to assess the expression of cyclin B1, and cyclin D1, MMP-2, and MMP-9 protein.  Our results suggested that the expression of PHGDH is increased in pancreatic cancer compared with adjacent normal tissues and the increased expression of PHGDH is assocd. with tumor size, lymph node metastasis, and TNM state of pancreatic cancer patients.  Moreover, the expression of PHGDH is an independent prognostic indicator for pancreatic cancer patients.  In addn., we found that knockdown of PHGDH in pancreatic cancer cells inhibits the cell proliferation, migration, and invasion abilities by down-regulating the expression of cyclin B1, and cyclin D1, MMP-2, and MMP-9.  Our data indicated that the expression of PHGDH is increased in pancreatic cancer and is an independent mol. prognostic factor for pancreatic cancer patients.  In addn., PHGDH controls cell proliferation, migration and invasion abilities.  Therefore, PHGDH could serve as an important prognostic indicator and therapeutic target for pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhB-EuBx_5TbVg90H21EOLACvtfcHk0liWcWmvQxdbhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2nu77N&md5=2003892af6215c0e765faf48143db9b9</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.gene.2017.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gene.2017.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DZ.%26aulast%3DFeng%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DDong%26aufirst%3DC.%26atitle%3DPHGDH%2520is%2520an%2520Independent%2520Prognosis%2520Marker%2520and%2520Contributes%2520Cell%2520Proliferation%252C%2520Migration%2520and%2520Invasion%2520in%2520Human%2520Pancreatic%2520Cancer%26jtitle%3DGene%26date%3D2018%26volume%3D642%26spage%3D43%26epage%3D50%26doi%3D10.1016%2Fj.gene.2017.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nohata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemori, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itesako, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kofuji, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakagawa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahiya, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enokida, H.</span></span> <span> </span><span class="NLM_article-title">PHGDH as a Key Enzyme for Serine Biosynthesis in HIF2α-targeting Therapy for Renal Cell Carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6329</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-17-1589</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1158%2F0008-5472.CAN-17-1589" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=28951458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=6321-6329&author=H.+Yoshinoauthor=N.+Nohataauthor=K.+Miyamotoauthor=M.+Yonemoriauthor=T.+Sakaguchiauthor=S.+Sugitaauthor=T.+Itesakoauthor=S.+Kofujiauthor=M.+Nakagawaauthor=R.+Dahiyaauthor=H.+Enokida&title=PHGDH+as+a+Key+Enzyme+for+Serine+Biosynthesis+in+HIF2%CE%B1-targeting+Therapy+for+Renal+Cell+Carcinoma&doi=10.1158%2F0008-5472.CAN-17-1589"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH as a Key Enzyme for Serine Biosynthesis in HIF2α-Targeting Therapy for Renal Cell Carcinoma</span></div><div class="casAuthors">Yoshino, Hirofumi; Nohata, Nijiro; Miyamoto, Kazutaka; Yonemori, Masaya; Sakaguchi, Takashi; Sugita, Satoshi; Itesako, Toshihiko; Kofuji, Satoshi; Nakagawa, Masayuki; Dahiya, Rajvir; Enokida, Hideki</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">6321-6329</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Continuous activation of hypoxia-inducible factor (HIF) is important for progression of renal cell carcinoma (RCC) and acquired resistance to antiangiogenic multikinase and mTOR inhibitors.  Recently, HIF2α antagonists PT2385 and PT2399 were developed and are being evaluated in a phase I clin. trial for advanced or metastatic clear cell RCC (ccRCC).  However, resistance to HIF2α antagonists would be expected to develop.  In this study, we identified signals activated by HIF2α deficiency as candidate mediators of resistance to the HIF2α antagonists.  We established sunitinib-resistant tumor cells in vivo and created HIF2α-deficient variants of these cells using CRISPR/Cas9 technol.  Mechanistic investigations revealed that a regulator of the serine biosynthesis pathway, phosphoglycerate dehydrogenase (PHGDH), was upregulated commonly in HIF2α-deficient tumor cells along with the serine biosynthesis pathway itself.  Accordingly, treatment with a PHGDH inhibitor reduced the growth of HIF2α-deficient tumor cells in vivo and in vitro by inducing apoptosis.  Our findings identify the serine biosynthesis pathway as a source of candidate therapeutic targets to eradicate advanced or metastatic ccRCC resistant to HIF2α antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqi_LDrzdakZLVg90H21EOLACvtfcHk0liWcWmvQxdbhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVSnur7I&md5=adef1b67feb25a551c785471dce76645</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-17-1589&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-17-1589%26sid%3Dliteratum%253Aachs%26aulast%3DYoshino%26aufirst%3DH.%26aulast%3DNohata%26aufirst%3DN.%26aulast%3DMiyamoto%26aufirst%3DK.%26aulast%3DYonemori%26aufirst%3DM.%26aulast%3DSakaguchi%26aufirst%3DT.%26aulast%3DSugita%26aufirst%3DS.%26aulast%3DItesako%26aufirst%3DT.%26aulast%3DKofuji%26aufirst%3DS.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DDahiya%26aufirst%3DR.%26aulast%3DEnokida%26aufirst%3DH.%26atitle%3DPHGDH%2520as%2520a%2520Key%2520Enzyme%2520for%2520Serine%2520Biosynthesis%2520in%2520HIF2%25CE%25B1-targeting%2520Therapy%2520for%2520Renal%2520Cell%2520Carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D6321%26epage%3D6329%26doi%3D10.1158%2F0008-5472.CAN-17-1589" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenner, A.</span></span> <span> </span><span class="NLM_article-title">PHGDH is Key for Targeting HIF in RCC</span>. <i>Nat. Rev. Urol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>, <span class="NLM_elocation-id">702</span> <span class="refDoi"> DOI: 10.1038/nrurol.2017.176</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnrurol.2017.176" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29116254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC1M7ptVOgtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&author=A.+Fenner&title=PHGDH+is+Key+for+Targeting+HIF+in+RCC&doi=10.1038%2Fnrurol.2017.176"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Kidney cancer: PHGDH is key for targeting HIF in RCC</span></div><div class="casAuthors">Fenner Annette</div><div class="citationInfo"><span class="NLM_cas:title">Nature reviews. Urology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">702</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTGMfqbtbJj6X6oPH0mYWL1fW6udTcc2ea7Vsy9zlQnVrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7ptVOgtw%253D%253D&md5=f82a7018e8220e42bf7989f0306a80e8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnrurol.2017.176&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrurol.2017.176%26sid%3Dliteratum%253Aachs%26aulast%3DFenner%26aufirst%3DA.%26atitle%3DPHGDH%2520is%2520Key%2520for%2520Targeting%2520HIF%2520in%2520RCC%26jtitle%3DNat.%2520Rev.%2520Urol.%26date%3D2017%26volume%3D14%26doi%3D10.1038%2Fnrurol.2017.176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">DeNicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudderth, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huffman, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeBerardinis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">NRF2 Regulates Serine Biosynthesis in Non-small Cell Lung Cancer</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1475</span>– <span class="NLM_lpage">1481</span>, <span class="refDoi"> DOI: 10.1038/ng.3421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fng.3421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=26482881" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslant7bO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2015&pages=1475-1481&author=G.+M.+DeNicolaauthor=P.-H.+Chenauthor=E.+Mullarkyauthor=J.+A.+Sudderthauthor=Z.+Huauthor=D.+Wuauthor=H.+Tangauthor=Y.+Xieauthor=J.+M.+Asaraauthor=K.+E.+Huffmanauthor=I.+I.+Wistubaauthor=J.+D.+Minnaauthor=R.+J.+DeBerardinisauthor=L.+C.+Cantley&title=NRF2+Regulates+Serine+Biosynthesis+in+Non-small+Cell+Lung+Cancer&doi=10.1038%2Fng.3421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">NRF2 regulates serine biosynthesis in non-small cell lung cancer</span></div><div class="casAuthors">DeNicola, Gina M.; Chen, Pei-Hsuan; Mullarky, Edouard; Sudderth, Jessica A.; Hu, Zeping; Wu, David; Tang, Hao; Xie, Yang; Asara, John M.; Huffman, Kenneth E.; Wistuba, Ignacio I.; Minna, John D.; DeBerardinis, Ralph J.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1475-1481</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumors have high energetic and anabolic needs for rapid cell growth and proliferation, and the serine biosynthetic pathway was recently identified as an important source of metabolic intermediates for these processes.  We integrated metabolic tracing and transcriptional profiling of a large panel of non-small cell lung cancer (NSCLC) cell lines to characterize the activity and regulation of the serine/glycine biosynthetic pathway in NSCLC.  Here we show that the activity of this pathway is highly heterogeneous and is regulated by NRF2, a transcription factor frequently deregulated in NSCLC.  We found that NRF2 controls the expression of the key serine/glycine biosynthesis enzyme genes PHGDH, PSAT1 and SHMT2 via ATF4 to support glutathione and nucleotide prodn.  Moreover, we show that expression of these genes confers poor prognosis in human NSCLC.  Thus, a substantial fraction of human NSCLCs activates an NRF2-dependent transcriptional program that regulates serine and glycine metab. and is linked to clin. aggressiveness.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSELAUW2hlmbVg90H21EOLACvtfcHk0lhTBHrlTKVI0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslant7bO&md5=41d411b051f2e26da358c949585c0c5d</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fng.3421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.3421%26sid%3Dliteratum%253Aachs%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DChen%26aufirst%3DP.-H.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DSudderth%26aufirst%3DJ.%2BA.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DHuffman%26aufirst%3DK.%2BE.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DDeBerardinis%26aufirst%3DR.%2BJ.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DNRF2%2520Regulates%2520Serine%2520Biosynthesis%2520in%2520Non-small%2520Cell%2520Lung%2520Cancer%26jtitle%3DNat.%2520Genet.%26date%3D2015%26volume%3D47%26spage%3D1475%26epage%3D1481%26doi%3D10.1038%2Fng.3421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamanaka, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigdelioglu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meliton, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witt, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woods, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansbro, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dulin, N. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzy, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutlu, G. M.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Phosphoglycerate Dehydrogenase Attenuates Bleomycin-induced Pulmonary Fibrosis</span>. <i>Am. J. Respir. Cell Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">593</span>, <span class="refDoi"> DOI: 10.1165/rcmb.2017-0186OC</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1165%2Frcmb.2017-0186OC" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29019702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslyks7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=585-593&author=R.+B.+Hamanakaauthor=R.+Nigdeliogluauthor=A.+Y.+Melitonauthor=Y.+Tianauthor=L.+J.+Wittauthor=E.+O%E2%80%99Learyauthor=K.+A.+Sunauthor=P.+S.+Woodsauthor=D.+Wuauthor=B.+Ansbroauthor=S.+Ardauthor=J.+M.+Rohdeauthor=N.+O.+Dulinauthor=R.+D.+Guzyauthor=G.+M.+Mutlu&title=Inhibition+of+Phosphoglycerate+Dehydrogenase+Attenuates+Bleomycin-induced+Pulmonary+Fibrosis&doi=10.1165%2Frcmb.2017-0186OC"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of phosphoglycerate dehydrogenase attenuates bleomycin-induced pulmonary fibrosis</span></div><div class="casAuthors">Hamanaka, Robert B.; Nigdelioglu, Recep; Meliton, Angelo Y.; Tian, Yufeng; Witt, Leah J.; O'Leary, Erin; Sun, Kaitlyn A.; Woods, Parker S.; Wu, David; Ansbro, Brandon; Ard, Shawn; Rohde, Jason M.; Dulin, Nickolai O.; Guzy, Robert D.; Mutlu, Gokhan M.</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Respiratory Cell and Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">585-593</span>CODEN:
                <span class="NLM_cas:coden">AJRBEL</span>;
        ISSN:<span class="NLM_cas:issn">1044-1549</span>.
    
            (<span class="NLM_cas:orgname">American Thoracic Society</span>)
        </div><div class="casAbstract">Organ fibrosis, including idiopathic pulmonary fibrosis, is assocd. with significant morbidity and mortality.  Because currently available therapies have limited effect, there is a need to better understand the mechanisms by which organ fibrosis occurs.  We have recently reported that transforming growth factor (TGF)-β, a key cytokine that promotes fibrogenesis, induces the expression of the enzymes of the de novo serine and glycine synthesis pathway in human lung fibroblasts, and that phosphoglycerate dehydrogenase (PHGDH; the first and rate-limiting enzyme of the pathway) is required to promote collagen protein synthesis downstream of TGF-β.  In this study, we investigated whether inhibition of de novo serine and glycine synthesis attenuates lung fibrosis in vivo.  We found that TGF-β induces mRNA and protein expression of PHGDH in murine fibroblasts.  Similarly, intratracheal administration of bleomycin resulted in increased expression of PHGDH in mouse lungs, localized to fibrotic regions.  Using a newly developed small mol. inhibitor of PHGDH (NCT-503), we tested whether pharmacol. inhibition of PHGDH could inhibit fibrogenesis both in vitro and in vivo.  Treatment of murine and human lung fibroblasts with NCT-503 decreased TGF-β-induced collagen protein synthesis.  Mice treated with the PHGDH inhibitor beginning 7 days after intratracheal instillation of bleomycin had attenuation of lung fibrosis.  These results indicate that the de novo serine and glycine synthesis pathway is necessary for TGF-β-induced collagen synthesis and bleomycin-induced pulmonary fibrosis.  PHGDH and other enzymes in the de novo serine and glycine synthesis pathway may be a therapeutic target for treatment of fibrotic diseases, including idiopathic pulmonary fibrosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrywNwhQatH9bVg90H21EOLACvtfcHk0lhUc_-0_cCzmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslyks7jE&md5=acb85b0fe79cb75162af642eb4932442</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1165%2Frcmb.2017-0186OC&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1165%252Frcmb.2017-0186OC%26sid%3Dliteratum%253Aachs%26aulast%3DHamanaka%26aufirst%3DR.%2BB.%26aulast%3DNigdelioglu%26aufirst%3DR.%26aulast%3DMeliton%26aufirst%3DA.%2BY.%26aulast%3DTian%26aufirst%3DY.%26aulast%3DWitt%26aufirst%3DL.%2BJ.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DK.%2BA.%26aulast%3DWoods%26aufirst%3DP.%2BS.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DAnsbro%26aufirst%3DB.%26aulast%3DArd%26aufirst%3DS.%26aulast%3DRohde%26aufirst%3DJ.%2BM.%26aulast%3DDulin%26aufirst%3DN.%2BO.%26aulast%3DGuzy%26aufirst%3DR.%2BD.%26aulast%3DMutlu%26aufirst%3DG.%2BM.%26atitle%3DInhibition%2520of%2520Phosphoglycerate%2520Dehydrogenase%2520Attenuates%2520Bleomycin-induced%2520Pulmonary%2520Fibrosis%26jtitle%3DAm.%2520J.%2520Respir.%2520Cell%2520Mol.%2520Biol.%26date%3D2018%26volume%3D58%26spage%3D585%26epage%3D593%26doi%3D10.1165%2Frcmb.2017-0186OC" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hydbring, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambroise, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouchida, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goiny, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakifahmetoglu-Norberg, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norberg, E.</span></span> <span> </span><span class="NLM_article-title">PHGDH Defines a Metabolic Subtype in Lung Adenocarcinomas with Poor Prognosis</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2289</span>– <span class="NLM_lpage">2303</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.05.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.celrep.2017.05.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=28614715" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVaku7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2017&pages=2289-2303&author=B.+Zhangauthor=A.+Zhengauthor=P.+Hydbringauthor=G.+Ambroiseauthor=A.+T.+Ouchidaauthor=M.+Goinyauthor=H.+Vakifahmetoglu-Norbergauthor=E.+Norberg&title=PHGDH+Defines+a+Metabolic+Subtype+in+Lung+Adenocarcinomas+with+Poor+Prognosis&doi=10.1016%2Fj.celrep.2017.05.067"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">PHGDH Defines a Metabolic Subtype in Lung Adenocarcinomas with Poor Prognosis</span></div><div class="casAuthors">Zhang, Boxi; Zheng, Adi; Hydbring, Per; Ambroise, Gorbatchev; Ouchida, Amanda Tomie; Goiny, Michel; Vakifahmetoglu-Norberg, Helin; Norberg, Erik</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2289-2303</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mol. signatures are emerging determinants of choice of therapy for lung adenocarcinomas.  An evolving therapeutic approach includes targeting metabolic dependencies in cancers.  Here, using an integrative approach, we have dissected the metabolic fingerprints of lung adenocarcinomas, and we show that Phosphoglycerate dehydrogenase (PHGDH), the rate-limiting enzyme in serine biosynthesis, is highly expressed in a adenocarcinoma subset with poor prognosis.  This subset harbors a gene signature for DNA replication and proliferation.  Accordingly, models with high levels of PHGDH display rapid proliferation, migration, and selective channeling of serine-derived carbons to glutathione and pyrimidines, while depletion of PHGDH shows potent and selective toxicity to this subset.  Differential PHGDH protein levels were defined by its degrdn., and the deubiquitinating enzyme JOSD2 is a regulator of its protein stability.  Our study provides evidence that a unique metabolic program is activated in a lung adenocarcinoma subset, described by PHGDH, which confers growth and survival and may have therapeutic implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuv8KYzWmPzrVg90H21EOLACvtfcHk0lgAsP5a9ZpILw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVaku7nP&md5=d6100f48feeda662452dc18732ac731a</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.05.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.05.067%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DA.%26aulast%3DHydbring%26aufirst%3DP.%26aulast%3DAmbroise%26aufirst%3DG.%26aulast%3DOuchida%26aufirst%3DA.%2BT.%26aulast%3DGoiny%26aufirst%3DM.%26aulast%3DVakifahmetoglu-Norberg%26aufirst%3DH.%26aulast%3DNorberg%26aufirst%3DE.%26atitle%3DPHGDH%2520Defines%2520a%2520Metabolic%2520Subtype%2520in%2520Lung%2520Adenocarcinomas%2520with%2520Poor%2520Prognosis%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D19%26spage%3D2289%26epage%3D2303%26doi%3D10.1016%2Fj.celrep.2017.05.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, J. S.</span></span> <span> </span><span class="NLM_article-title">Expression Levels of Serine/Glycine Metabolism-related Proteins in Triple Negative Breast Cancer Tissues</span>. <i>Tumor Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">4457</span>– <span class="NLM_lpage">4468</span>, <span class="refDoi"> DOI: 10.1007/s13277-013-1588-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1007%2Fs13277-013-1588-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=24390667" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntlOrtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=4457-4468&author=S.+Nohauthor=D.+H.+Kimauthor=W.+H.+Jungauthor=J.+S.+Koo&title=Expression+Levels+of+Serine%2FGlycine+Metabolism-related+Proteins+in+Triple+Negative+Breast+Cancer+Tissues&doi=10.1007%2Fs13277-013-1588-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Expression levels of serine/glycine metabolism-related proteins in triple negative breast cancer tissues</span></div><div class="casAuthors">Noh, Songmi; Kim, Do Hee; Jung, Woo Hee; Koo, Ja Seung</div><div class="citationInfo"><span class="NLM_cas:title">Tumor Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">4457-4468</span>CODEN:
                <span class="NLM_cas:coden">TUMBEA</span>;
        ISSN:<span class="NLM_cas:issn">1010-4283</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">To evaluate the expression levels of serine/glycine metab.-related proteins (PHGDH, PSAT, PSPH, SHMT, and GLDC) in six different subtypes of triple neg. breast cancer (TNBC) patients and gain insight into their implications.  Formalin-fixed, paraffin-embedded tissues from 129 TNBC patients were assembled into tissue microarrays.  Immunohistochem. staining was performed for serine/glycine metab.-related proteins (PHGDH, PSAT, PSPH, SHMT, and GLDC) and surrogate immunohistochem. markers (CK5/6, EGFR, claudin 3, claudin 4, claudin 7, E-cadherin, STAT1, interleukin-8 [IL-8], AR, and GGT-1) for identifying the mol. subtype of TNBC.  TNBC subtype classifications included the following: basal-like (CK5/6-pos. and/or EGFR-pos.), mol. apocrine (AR-pos. and/or GGT-1-pos.), claudin-low (claudin 3-, claudin 4-, claudin 7-neg. and/or E-cadherin-neg.), immune-related (IL-8-neg. and stromal STAT1-pos.), mixed (features from two or more of the four subtypes), and null (no features from any of the four subtypes).  Tissues from basal marker-pos. patients showed increased expression levels of tumoral PHGDH compared with those from basal marker-neg. patients (p = 0.029); lack of stromal SHMT1 expression was significantly correlated with T stage (p = 0.016).  Multivariate Cox anal. revealed that a lack of stromal SHMT1 expression was an independent prognostic factor for predicting a shorter disease-free survival period (hazard ratio 4.002, 95 % confidence interval [CI] 1.077-14.83, p = 0.038); furthermore, a lack of tumoral PHGDH expression was predictive of a shorter overall survival rate (hazard ratio 3.053, 95 % CI 1.002-9.305, p = 0.050).  In conclusion, the most abundantly expressed serine/glycine metab.-related protein in basal-like TNBC tissues was tumoral PHGDH, and expression levels of stromal SHMT1 and tumoral PHGDH were inversely correlated with clin. prognostic factors.  Also, this study is the first to assess serine/glycine relationships at the protein level in regards to clin. outcomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbJy-OZWfJBLVg90H21EOLACvtfcHk0lgAsP5a9ZpILw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntlOrtL0%253D&md5=fa67446343d9ebeefe392cca0b1777c3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs13277-013-1588-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13277-013-1588-z%26sid%3Dliteratum%253Aachs%26aulast%3DNoh%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DD.%2BH.%26aulast%3DJung%26aufirst%3DW.%2BH.%26aulast%3DKoo%26aufirst%3DJ.%2BS.%26atitle%3DExpression%2520Levels%2520of%2520Serine%252FGlycine%2520Metabolism-related%2520Proteins%2520in%2520Triple%2520Negative%2520Breast%2520Cancer%2520Tissues%26jtitle%3DTumor%2520Biol.%26date%3D2014%26volume%3D35%26spage%3D4457%26epage%3D4468%26doi%3D10.1007%2Fs13277-013-1588-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, X.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiang, J.-F.</span></span> <span> </span><span class="NLM_article-title">Increased Expression of PHGDH and Prognostic Significance in Colorectal Cancer</span>. <i>Transl. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1016/j.tranon.2016.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.tranon.2016.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27267836" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FnvVertw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2016&pages=191-196&author=X.-Q.+Jiaauthor=S.+Zhangauthor=H.-J.+Zhuauthor=W.+Wangauthor=J.-H.+Zhuauthor=X.-D.+Wangauthor=J.-F.+Qiang&title=Increased+Expression+of+PHGDH+and+Prognostic+Significance+in+Colorectal+Cancer&doi=10.1016%2Fj.tranon.2016.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Increased Expression of PHGDH and Prognostic Significance in Colorectal Cancer</span></div><div class="casAuthors">Jia Xiao-Qin; Zhang Shu; Zhu Hui-Jun; Wang Wei; Zhu Jin-Hong; Wang Xu-Dong; Qiang Jian-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Translational oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">191-6</span>
        ISSN:<span class="NLM_cas:issn">1936-5233</span>.
    </div><div class="casAbstract">Phosphoglycerate dehydrogenase (PHGDH) plays an essential role in cancer-specific metabolic reprogramming.  It has been reported as a putative metabolic oncogene in several types of human malignant tumors, such as breast cancer and melanoma.  To date, PHGDH expression in colorectal cancer (CRC) as well as its association with clinicopathological characteristics and prognostic implication remain undetermined.  In this study, we determined the PHGDH protein expression using tissue microarray immunohistochemistry (TMA-IHC) on 193 pairs of formalin-fixed, paraffin-embedded specimens of CRC and adjacent tissues, 25 chronic colitis, 41 low-, and 19 high-grade intraepithelial neoplasia specimens, and we also determined PHGDH mRNA level using quantitative reverse transcription PCR (qRT-PCR) on additional 23 pairs of fresh CRC tissues and adjacent tissues.  We found that both PHGDH mRNA and protein was highly expressed in tumor tissues in comparison with matched adjacent non-tumor tissues, and high PHGDH protein expression was correlated with advanced TNM stage (P = .038) and larger tumor (P = .001).  Multivariate Cox regression analysis showed that PHGDH protein expression (HR = 2.285, 95% CI = 1.18 to 4.41, P = .014), tumor differentiation (HR = .307, 95% CI = .154 to 0.609, P = .001), and TNM stage (HR = 1.791, 95% CI = 1.125 to 2.85, P = .014) were independent prognostic factors in CRC.  Kaplan-Meier survival curves and log rank test showed that high PHGDH protein expression contributed to poor outcome in CRC patients (P < .001).  In conclusion, these results suggest that assessment of PHGDH expression could be useful in identifying a high-risk subgroup of CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR2rz7sre8coyEJCPKhJPuqfW6udTcc2eZJWN3Wghyvyrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FnvVertw%253D%253D&md5=242366977328c32161960259ee1d4fae</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.tranon.2016.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tranon.2016.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DX.-Q.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DH.-J.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DJ.-H.%26aulast%3DWang%26aufirst%3DX.-D.%26aulast%3DQiang%26aufirst%3DJ.-F.%26atitle%3DIncreased%2520Expression%2520of%2520PHGDH%2520and%2520Prognostic%2520Significance%2520in%2520Colorectal%2520Cancer%26jtitle%3DTransl.%2520Oncol.%26date%3D2016%26volume%3D9%26spage%3D191%26epage%3D196%26doi%3D10.1016%2Fj.tranon.2016.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Ixocarpalactone A from Dietary Tomatillo Inhibits Pancreatic Cancer Growth by Targeting PHGDH</span>. <i>Food Funct.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_fpage">3386</span>, <span class="refDoi"> DOI: 10.1039/C9FO00394K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1039%2FC9FO00394K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=31112178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvF2jtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2019&pages=3386&author=M.+Zhengauthor=J.+Guoauthor=J.+Xuauthor=K.+Yangauthor=R.+Tangauthor=X.+Guauthor=H.+Liauthor=L.+Chen&title=Ixocarpalactone+A+from+Dietary+Tomatillo+Inhibits+Pancreatic+Cancer+Growth+by+Targeting+PHGDH&doi=10.1039%2FC9FO00394K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Ixocarpalactone A from dietary tomatillo inhibits pancreatic cancer growth by targeting PHGDH</span></div><div class="casAuthors">Zheng, Mengzhu; Guo, Jing; Xu, Jiamin; Yang, Kaiyin; Tang, Ruotian; Gu, Xiaoxia; Li, Hua; Chen, Lixia</div><div class="citationInfo"><span class="NLM_cas:title">Food & Function</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">3386-3395</span>CODEN:
                <span class="NLM_cas:coden">FFOUAI</span>;
        ISSN:<span class="NLM_cas:issn">2042-6496</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">3-Phosphoglycerate dehydrogenase (PHGDH) catalyzes the first rate-limiting step for the synthesis of glucose-derived serine by converting 3-phosphoglycerate (3-PG) to phosphohydroxypyruvate (p-Pyr), which has been reported to assoc. with tumorigenesis in many cancers.  Iox A, a natural withanolide obtained from dietary tomatillo (Physalis ixocarpa), showed significant PHGDH inhibitory activity with an IC50 value of 1.66 ± 0.28 μM, and it was further confirmed to bind directly to PHGDH by the MST assay.  Mol. docking demonstrated that Iox A coordinated at the allosteric site of PHGDH, which was consistent with the non-competitive kinetics.  Meanwhile, Iox A selectively inhibited the proliferation of high PHGDH-expressing cancer cell lines (SW1990, MCF-7 and HeLa) and showed no obvious cytotoxicities on normal human cells (LO2, L929 and HPDE6-C7).  In particular, Iox A showed a dose-dependent proapoptotic activity against SW1990 cells in a micromolar concn. as detected by flow cytometry and western blot anal.  Furthermore, Iox A significantly inhibited the tumor growth in a SW1990 xenograft mouse model with low toxicities, suggesting its potential therapeutic application in pancreatic cancer treatment.  Therefore, Iox A was identified as a novel natural PHGDH inhibitor with high targeting and low toxicities for the treatment of pancreatic cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRpi-AH_ojUrVg90H21EOLACvtfcHk0li7ntnU9c9OeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvF2jtrc%253D&md5=edc1af7499bc0c6d72e60ab8f0d67142</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1039%2FC9FO00394K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9FO00394K%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DK.%26aulast%3DTang%26aufirst%3DR.%26aulast%3DGu%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DIxocarpalactone%2520A%2520from%2520Dietary%2520Tomatillo%2520Inhibits%2520Pancreatic%2520Cancer%2520Growth%2520by%2520Targeting%2520PHGDH%26jtitle%3DFood%2520Funct.%26date%3D2019%26spage%3D3386%26doi%3D10.1039%2FC9FO00394K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span> <span> </span><span class="NLM_article-title">Serine, Glycine and the One-Carbon Cycle: Cancer Metabolism in Full Circle</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">572</span>– <span class="NLM_lpage">583</span>, <span class="refDoi"> DOI: 10.1038/nrc3557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnrc3557" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=23822983" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsLzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2013&pages=572-583&author=J.+W.+Locasale&title=Serine%2C+Glycine+and+the+One-Carbon+Cycle%3A+Cancer+Metabolism+in+Full+Circle&doi=10.1038%2Fnrc3557"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Serine, glycine and one-carbon units: cancer metabolism in full circle</span></div><div class="casAuthors">Locasale, Jason W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">572-583</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  One-carbon metab. involving the folate and methionine cycles integrates nutritional status from amino acids, glucose and vitamins, and generates diverse outputs, such as the biosynthesis of lipids, nucleotides and proteins, the maintenance of redox status and the substrates for methylation reactions.  Long considered a 'housekeeping' process, this pathway has recently been shown to have addnl. complexity.  Genetic and functional evidence suggests that hyperactivation of this pathway is a driver of oncogenesis and establishes a link to cellular epigenetic status.  Given the wealth of clin. available agents that target one-carbon metab., these new findings could present opportunities for translation into precision cancer medicine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro6rIkiwOb8LVg90H21EOLACvtfcHk0li7ntnU9c9OeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGrsLzE&md5=96e34f61a4dd32c09cc196493804b5fc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrc3557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc3557%26sid%3Dliteratum%253Aachs%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26atitle%3DSerine%252C%2520Glycine%2520and%2520the%2520One-Carbon%2520Cycle%253A%2520Cancer%2520Metabolism%2520in%2520Full%2520Circle%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2013%26volume%3D13%26spage%3D572%26epage%3D583%26doi%3D10.1038%2Fnrc3557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander
Heiden, M. G.</span></span> <span> </span><span class="NLM_article-title">The Importance of Serine Metabolism in Cancer</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>214</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1083/jcb.201604085</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1083%2Fjcb.201604085" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27458133" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVykt7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=214&publication_year=2016&pages=249-257&author=K.+R.+Mattainiauthor=M.+R.+Sullivanauthor=M.+G.+Vander%0AHeiden&title=The+Importance+of+Serine+Metabolism+in+Cancer&doi=10.1083%2Fjcb.201604085"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The importance of serine metabolism in cancer</span></div><div class="casAuthors">Mattaini, Katherine R.; Sullivan, Mark R.; Heiden, Matthew G. Vander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">214</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-257</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">1540-8140</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Serine metab. is frequently dysregulated in cancers; however, the benefit that this confers to tumors remains controversial.  In many cases, extracellular serine alone is sufficient to support cancer cell proliferation, whereas some cancer cells increase serine synthesis from glucose and require de novo serine synthesis even in the presence of abundant extracellular serine.  Recent studies cast new light on the role of serine metab. in cancer, suggesting that active serine synthesis might be required to facilitate amino acid transport, nucleotide synthesis, folate metab., and redox homeostasis in a manner that impacts cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUVt9Uk-a5grVg90H21EOLACvtfcHk0lic3qqm0AMwLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVykt7nM&md5=3828ce3d50390d59aa82c5d7d0618175</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201604085&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201604085%26sid%3Dliteratum%253Aachs%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DSullivan%26aufirst%3DM.%2BR.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26atitle%3DThe%2520Importance%2520of%2520Serine%2520Metabolism%2520in%2520Cancer%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2016%26volume%3D214%26spage%3D249%26epage%3D257%26doi%3D10.1083%2Fjcb.201604085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tedeschi, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markert, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gounder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dvorzhinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolfi, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, L. L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiPaola, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirshfield, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boros, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertino, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oltvai, Z. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazquez, A.</span></span> <span> </span><span class="NLM_article-title">Contribution of Serine, Folate and Glycine Metabolism to the ATP, NADPH and Purine Requirements of Cancer Cells</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>, <span class="NLM_elocation-id">e877</span> <span class="refDoi"> DOI: 10.1038/cddis.2013.393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fcddis.2013.393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=24157871" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1OlurfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&author=P.+M.+Tedeschiauthor=E.+K.+Markertauthor=M.+Gounderauthor=H.+Linauthor=D.+Dvorzhinskiauthor=S.+C.+Dolfiauthor=L.+L.-Y.+Chanauthor=J.+Qiuauthor=R.+S.+DiPaolaauthor=K.+M.+Hirshfieldauthor=L.+G.+Borosauthor=J.+R.+Bertinoauthor=Z.+N.+Oltvaiauthor=A.+Vazquez&title=Contribution+of+Serine%2C+Folate+and+Glycine+Metabolism+to+the+ATP%2C+NADPH+and+Purine+Requirements+of+Cancer+Cells&doi=10.1038%2Fcddis.2013.393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Contribution of serine, folate and glycine metabolism to the ATP, NADPH and purine requirements of cancer cells</span></div><div class="casAuthors">Tedeschi, P. M.; Markert, E. K.; Gounder, M.; Lin, H.; Dvorzhinski, D.; Dolfi, S. C.; Chan, L. L.-Y.; Qiu, J.; DiPaola, R. S.; Hirshfield, K. M.; Boros, L. G.; Bertino, J. R.; Oltvai, Z. N.; Vazquez, A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">Oct.</span>),
    <span class="NLM_cas:pages">e877</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent observations on cancer cell metab. indicate increased serine synthesis from glucose as a marker of poor prognosis.  We have predicted that a fraction of the synthesized serine is routed to a pathway for ATP prodn.  The pathway is composed by reactions from serine synthesis, one-carbon (folate) metab. and the glycine cleavage system (SOG pathway).  Here we show that the SOG pathway is upregulated at the level of gene expression in a subset of human tumors and that its level of expression correlates with gene signatures of cell proliferation and Myc target activation.  We have also estd. the SOG pathway metabolic flux in the NCI60 tumor-derived cell lines, using previously reported exchange fluxes and a personalized model of cell metab.  We find that the estd. rates of reactions in the SOG pathway are highly correlated with the proliferation rates of these cell lines.  We also observe that the SOG pathway contributes significantly to the energy requirements of biosynthesis, to the NADPH requirement for fatty acid synthesis and to the synthesis of purines.  Finally, when the PC-3 prostate cancer cell line is treated with the antifolate methotrexate, we observe a decrease in the ATP levels, AMP kinase activation and a decrease in ribonucleotides and fatty acids synthesized from [1,2-13C2]-D-glucose as the single tracer.  Taken together our results indicate that the SOG pathway activity increases with the rate of cell proliferation and it contributes to the biosynthetic requirements of purines, ATP and NADPH of cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPNkAiO41cULVg90H21EOLACvtfcHk0lic3qqm0AMwLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1OlurfI&md5=eebf6e0461df9e4415a78ca4dd51328c</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fcddis.2013.393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fcddis.2013.393%26sid%3Dliteratum%253Aachs%26aulast%3DTedeschi%26aufirst%3DP.%2BM.%26aulast%3DMarkert%26aufirst%3DE.%2BK.%26aulast%3DGounder%26aufirst%3DM.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DDvorzhinski%26aufirst%3DD.%26aulast%3DDolfi%26aufirst%3DS.%2BC.%26aulast%3DChan%26aufirst%3DL.%2BL.-Y.%26aulast%3DQiu%26aufirst%3DJ.%26aulast%3DDiPaola%26aufirst%3DR.%2BS.%26aulast%3DHirshfield%26aufirst%3DK.%2BM.%26aulast%3DBoros%26aufirst%3DL.%2BG.%26aulast%3DBertino%26aufirst%3DJ.%2BR.%26aulast%3DOltvai%26aufirst%3DZ.%2BN.%26aulast%3DVazquez%26aufirst%3DA.%26atitle%3DContribution%2520of%2520Serine%252C%2520Folate%2520and%2520Glycine%2520Metabolism%2520to%2520the%2520ATP%252C%2520NADPH%2520and%2520Purine%2520Requirements%2520of%2520Cancer%2520Cells%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2013%26volume%3D4%26doi%3D10.1038%2Fcddis.2013.393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reid, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberti, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span> <span> </span><span class="NLM_article-title">Serine Synthesis Through PHGDH Coordinates Nucleotide Levels by Maintaining Central Carbon Metabolism</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>, <span class="NLM_elocation-id">5442</span> <span class="refDoi"> DOI: 10.1038/s41467-018-07868-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fs41467-018-07868-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=30575741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BB3cnks1ejtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&author=M.+A.+Reidauthor=A.+E.+Allenauthor=S.+Liuauthor=M.+V.+Libertiauthor=P.+Liuauthor=X.+Liuauthor=Z.+Daiauthor=X.+Gaoauthor=Q.+Wangauthor=Y.+Liuauthor=L.+Laiauthor=J.+W.+Locasale&title=Serine+Synthesis+Through+PHGDH+Coordinates+Nucleotide+Levels+by+Maintaining+Central+Carbon+Metabolism&doi=10.1038%2Fs41467-018-07868-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Serine synthesis through PHGDH coordinates nucleotide levels by maintaining central carbon metabolism</span></div><div class="casAuthors">Reid Michael A; Allen Annamarie E; Liu Shiyu; Liberti Maria V; Liu Xiaojing; Dai Ziwei; Gao Xia; Locasale Jason W; Liu Pei; Wang Qian; Liu Ying; Lai Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5442</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Phosphoglycerate dehydrogenase (PHGDH) catalyzes the committed step in de novo serine biosynthesis.  Paradoxically, PHGDH and serine synthesis are required in the presence of abundant environmental serine even when serine uptake exceeds the requirements for nucleotide synthesis.  Here, we establish a mechanism for how PHGDH maintains nucleotide metabolism.  We show that inhibition of PHGDH induces alterations in nucleotide metabolism independent of serine utilization.  These changes are not attributable to defects in serine-derived nucleotide synthesis and redox maintenance, another key aspect of serine metabolism, but result from disruption of mass balance within central carbon metabolism.  Mechanistically, this leads to simultaneous alterations in both the pentose phosphate pathway and the tri-carboxylic acid cycle, as we demonstrate based on a quantitative model.  These findings define a mechanism whereby disruption of one metabolic pathway induces toxicity by simultaneously affecting the activity of multiple related pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSfmLTc2J-j04kFb3mqPUYDfW6udTcc2eYcGi78MDtcZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cnks1ejtQ%253D%253D&md5=e0d90406b6274becc65233dd4fab768b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fs41467-018-07868-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-018-07868-6%26sid%3Dliteratum%253Aachs%26aulast%3DReid%26aufirst%3DM.%2BA.%26aulast%3DAllen%26aufirst%3DA.%2BE.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiberti%26aufirst%3DM.%2BV.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26atitle%3DSerine%2520Synthesis%2520Through%2520PHGDH%2520Coordinates%2520Nucleotide%2520Levels%2520by%2520Maintaining%2520Central%2520Carbon%2520Metabolism%26jtitle%3DNat.%2520Commun.%26date%3D2018%26volume%3D9%26doi%3D10.1038%2Fs41467-018-07868-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, W.</span></span> <span> </span><span class="NLM_article-title">Repression of Phosphoglycerate Dehydrogenase Sensitizes Triple-negative Breast Cancer to Doxorubicin</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">655</span>– <span class="NLM_lpage">659</span>, <span class="refDoi"> DOI: 10.1007/s00280-016-3117-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1007%2Fs00280-016-3117-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27473325" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12ls7zL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2016&pages=655-659&author=X.+Zhangauthor=W.+Bai&title=Repression+of+Phosphoglycerate+Dehydrogenase+Sensitizes+Triple-negative+Breast+Cancer+to+Doxorubicin&doi=10.1007%2Fs00280-016-3117-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Repression of phosphoglycerate dehydrogenase sensitizes triple-negative breast cancer to doxorubicin</span></div><div class="casAuthors">Zhang, Xia; Bai, Weijun</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">655-659</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: Approx. 70 % of triple-neg. breast cancer (TNBC) cell lines are identified to upregulate phosphoglycerate dehydrogenase (PHGDH), which regulates the intracellular synthesis of serine and glycine, and promotes tumor growth.  In this work, the impact of this pathway on doxorubicin efficacy was evaluated.  Methods: MDA-MB-468, BT-20 and HCC70 cells were transfected with lentiviral vectors expressing short hairpin RNA (shRNA) against PHGDH.  In response to doxorubicin treatment, cellular proliferation was measured, ROS were evaluated and intracellular levels of serine, glycine and glutathione (GSH) were detd. using liq. chromatog.-mass spectrometry.  A TNBC orthotopic tumor model was used to examine the effect of PHGDH on doxorubicin efficacy in vivo.  Results: TNBC cells exposed to doxorubicin undergo metabolic remodeling, resulting in increased glucose flux for serine synthesis regulated by PHGDH.  Serine is then converted into GSH, which counters doxorubicin-induced formation of ROS.  Consequently, suppression of PHGDH by the use of the shRNA caused doxorubicin-induced oxidative stress and increased doxorubicin sensitivity.  The enhancement of doxorubicin efficacy through simultaneous suppression of PHGDH was validated in a mouse tumor model.  Conclusion: These results shed light on PHGDH that could be a promising target for increasing the effectiveness of chemotherapy in patients with TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5VjY1dAeQz7Vg90H21EOLACvtfcHk0ljWdLf3i66oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12ls7zL&md5=5ce16b5bdd652a1e6aaa3e4f427b431e</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1007%2Fs00280-016-3117-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-016-3117-4%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DBai%26aufirst%3DW.%26atitle%3DRepression%2520of%2520Phosphoglycerate%2520Dehydrogenase%2520Sensitizes%2520Triple-negative%2520Breast%2520Cancer%2520to%2520Doxorubicin%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2016%26volume%3D78%26spage%3D655%26epage%3D659%26doi%3D10.1007%2Fs00280-016-3117-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattaini, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Looper, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabinowitz, J. D.</span></span> <span> </span><span class="NLM_article-title">Human Phosphoglycerate Dehydrogenase Produces the Oncometabolite d-2-Hydroxyglutarate</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">510</span>– <span class="NLM_lpage">516</span>, <span class="refDoi"> DOI: 10.1021/cb500683c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500683c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFGkurfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=510-516&author=J.+Fanauthor=X.+Tengauthor=L.+Liuauthor=K.+R.+Mattainiauthor=R.+E.+Looperauthor=M.+G.+Vander+Heidenauthor=J.+D.+Rabinowitz&title=Human+Phosphoglycerate+Dehydrogenase+Produces+the+Oncometabolite+d-2-Hydroxyglutarate&doi=10.1021%2Fcb500683c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Human phosphoglycerate dehydrogenase produces the oncometabolite D-2-Hydroxyglutarate</span></div><div class="casAuthors">Fan, Jing; Teng, Xin; Liu, Ling; Mattaini, Katherine R.; Looper, Ryan E.; Vander Heiden, Matthew G.; Rabinowitz, Joshua D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">510-516</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human D-3-phosphoglycerate dehydrogenase (PHGDH), the first enzyme in the serine biosynthetic pathway, is genomically amplified in tumors including breast cancer and melanoma.  In PHGDH-amplified cancer cells, knockdown of PHGDH is not fully rescued by exogenous serine, suggesting possible addnl. growth-promoting roles for the enzyme.  Here we show that, in addn. to catalyzing oxidn. of 3-phosphoglycerate, PHGDH catalyzes NADH-dependent redn. of α-ketoglutarate (AKG) to the oncometabolite D-2-hydroxyglutarate (D-2HG).  Knockdown of PHGDH decreased cellular 2HG by approx. 50% in the PHGDH-amplified breast cancer cell lines MDA-MB-468 (normal concn. 93 μM) and BT-20 (normal concn. 35 μM) and overexpression of PHGDH increased cellular 2HG by over 2-fold in non-PHGDH-amplified MDA-MB-231 breast cancer cells, which normally display very low PHGDH expression.  The reduced 2HG level in PHGDH knockdown cell lines can be rescued by PHGDH re-expression, but not by a catalytically inactive PHGDH mutant.  The initial connection between cancer and D-2HG involved prodn. of high levels of D-2HG by mutant isocitrate dehydrogenase.  More recently, however, elevated D-2HG has been obsd. in breast cancer tumors without isocitrate dehydrogenase mutation.  Our results suggest that PHGDH is one source of this D-2HG.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWrcBz9QWsbVg90H21EOLACvtfcHk0ljWdLf3i66oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFGkurfE&md5=20fc923706b5a481c8aa85670c626743</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcb500683c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500683c%26sid%3Dliteratum%253Aachs%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DTeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMattaini%26aufirst%3DK.%2BR.%26aulast%3DLooper%26aufirst%3DR.%2BE.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DRabinowitz%26aufirst%3DJ.%2BD.%26atitle%3DHuman%2520Phosphoglycerate%2520Dehydrogenase%2520Produces%2520the%2520Oncometabolite%2520d-2-Hydroxyglutarate%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D510%26epage%3D516%26doi%3D10.1021%2Fcb500683c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Figueroa, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shih, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagwat, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasanthakumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallman, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolniak, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choe, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fantin, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paietta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löwenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godley, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delwel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valk, P. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnick, A.</span></span> <span> </span><span class="NLM_article-title">Leukemic IDH1 and IDH2 Mutations Result in a Hypermethylation Phenotype, Disrupt TET2 Function, and Impair Hematopoietic Differentiation</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2010.11.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.ccr.2010.11.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21130701" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFGhsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=553-567&author=M.+E.+Figueroaauthor=O.+Abdel-Wahabauthor=C.+Luauthor=P.+S.+Wardauthor=J.+Patelauthor=A.+Shihauthor=Y.+Liauthor=N.+Bhagwatauthor=A.+Vasanthakumarauthor=H.+F.+Fernandezauthor=M.+S.+Tallmanauthor=Z.+Sunauthor=K.+Wolniakauthor=J.+K.+Peetersauthor=W.+Liuauthor=S.+E.+Choeauthor=V.+R.+Fantinauthor=E.+Paiettaauthor=B.+L%C3%B6wenbergauthor=J.+D.+Lichtauthor=L.+A.+Godleyauthor=R.+Delwelauthor=P.+J.+M.+Valkauthor=C.+B.+Thompsonauthor=R.+L.+Levineauthor=A.+Melnick&title=Leukemic+IDH1+and+IDH2+Mutations+Result+in+a+Hypermethylation+Phenotype%2C+Disrupt+TET2+Function%2C+and+Impair+Hematopoietic+Differentiation&doi=10.1016%2Fj.ccr.2010.11.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Leukemic IDH1 and IDH2 mutations result in a hypermethylation phenotype, disrupt TET2 function, and impair hematopoietic differentiation</span></div><div class="casAuthors">Figueroa, Maria E.; Abdel-Wahab, Omar; Lu, Chao; Ward, Patrick S.; Patel, Jay; Shih, Alan; Li, Yushan; Bhagwat, Neha; Vasanthakumar, Aparna; Fernandez, Hugo F.; Tallman, Martin S.; Sun, Zhuoxin; Wolniak, Kristy; Peeters, Justine K.; Liu, Wei; Choe, Sung E.; Fantin, Valeria R.; Paietta, Elisabeth; Loewenberg, Bob; Licht, Jonathan D.; Godley, Lucy A.; Delwel, Ruud; Valk, Peter J. M.; Thompson, Craig B.; Levine, Ross L.; Melnick, Ari</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">553-567</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Cancer-assocd. IDH mutations are characterized by neomorphic enzyme activity and resultant 2-hydroxyglutarate (2HG) prodn.  Mutational and epigenetic profiling of a large acute myeloid leukemia (AML) patient cohort revealed that IDH1/2-mutant AMLs display global DNA hypermethylation and a specific hypermethylation signature.  Furthermore, expression of 2HG-producing IDH alleles in cells induced global DNA hypermethylation.  In the AML cohort, IDH1/2 mutations were mutually exclusive with mutations in the α-ketoglutarate-dependent enzyme TET2, and TET2 loss-of-function mutations were assocd. with similar epigenetic defects as IDH1/2 mutants.  Consistent with these genetic and epigenetic data, expression of IDH mutants impaired TET2 catalytic function in cells.  Finally, either expression of mutant IDH1/2 or Tet2 depletion impaired hematopoietic differentiation and increased stem/progenitor cell marker expression, suggesting a shared proleukemogenic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5mMhJR64Z4rVg90H21EOLACvtfcHk0ljWdLf3i66oFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFGhsbvJ&md5=f058c7dbd2efd6a25f4044506b5a2ef3</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.11.015%26sid%3Dliteratum%253Aachs%26aulast%3DFigueroa%26aufirst%3DM.%2BE.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DWard%26aufirst%3DP.%2BS.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DShih%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DBhagwat%26aufirst%3DN.%26aulast%3DVasanthakumar%26aufirst%3DA.%26aulast%3DFernandez%26aufirst%3DH.%2BF.%26aulast%3DTallman%26aufirst%3DM.%2BS.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DWolniak%26aufirst%3DK.%26aulast%3DPeeters%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DChoe%26aufirst%3DS.%2BE.%26aulast%3DFantin%26aufirst%3DV.%2BR.%26aulast%3DPaietta%26aufirst%3DE.%26aulast%3DL%25C3%25B6wenberg%26aufirst%3DB.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DGodley%26aufirst%3DL.%2BA.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DValk%26aufirst%3DP.%2BJ.%2BM.%26aulast%3DThompson%26aufirst%3DC.%2BB.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DMelnick%26aufirst%3DA.%26atitle%3DLeukemic%2520IDH1%2520and%2520IDH2%2520Mutations%2520Result%2520in%2520a%2520Hypermethylation%2520Phenotype%252C%2520Disrupt%2520TET2%2520Function%252C%2520and%2520Impair%2520Hematopoietic%2520Differentiation%26jtitle%3DCancer%2520Cell%26date%3D2010%26volume%3D18%26spage%3D553%26epage%3D567%26doi%3D10.1016%2Fj.ccr.2010.11.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borodovsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seltzer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riggins, G. J.</span></span> <span> </span><span class="NLM_article-title">Altered Cancer Cell Metabolism in Gliomas with Mutant IDH1 or IDH2</span>. <i>Curr. Opin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">83</span>– <span class="NLM_lpage">89</span>, <span class="refDoi"> DOI: 10.1097/CCO.0b013e32834d816a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1097%2FCCO.0b013e32834d816a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=22080945" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Shu7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2012&pages=83-89&author=A.+Borodovskyauthor=M.+J.+Seltzerauthor=G.+J.+Riggins&title=Altered+Cancer+Cell+Metabolism+in+Gliomas+with+Mutant+IDH1+or+IDH2&doi=10.1097%2FCCO.0b013e32834d816a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Altered cancer cell metabolism in gliomas with mutant IDH1 or IDH2</span></div><div class="casAuthors">Borodovsky, Alexandra; Seltzer, Meghan J.; Riggins, Gregory J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">83-89</span>CODEN:
                <span class="NLM_cas:coden">CUOOE8</span>;
        ISSN:<span class="NLM_cas:issn">1040-8746</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: IDH1/2 mutations occur in up to 70% of low-grade gliomas and secondary glioblastomas.  Mutation of these enzymes reduces the wildtype function of the enzyme (conversion of isocitrate to α-ketoglutarate) while conferring a new enzymic function, the prodn. of D-2-hydroxyglutarate (D-2-HG) from α-ketoglutarate (α-KG).  However, it is unclear how these enzymic changes contribute to tumorigenesis.  Here, we discuss the recent studies that demonstrate how IDH1/2 mutation may alter the metab. and epigenome of gliomas, how these changes may contribute to tumor formation, and opportunities they might provide for mol. targeting.  Recent findings: Metabolomic studies of IDH1/2 mutant cells have revealed alterations in glutamine, fatty acid, and citrate synthesis pathways.  Addnl., D-2-HG produced by IDH1/2 mutant cells can competitively inhibit α-KG-dependent enzymes, including histone demethylases and DNA hydroxylases, potentially leading to a distinct epigenetic phenotype.  Alterations in metab. and DNA methylation present possible mechanisms of tumorigenesis.  Summary: Recent attempts to improve outcomes for glioma patients have resulted in incremental gains.  Studies of IDH1/2 mutations have provided mechanistic insights into tumorigenesis and potential avenues for therapeutic intervention.  Further study of IDH1/2 mutations might allow for improved therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo59ZCcbKIXrbVg90H21EOLACvtfcHk0liuqNJG5p3_Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Shu7vP&md5=73509777089e369682a95e9738dd94b6</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1097%2FCCO.0b013e32834d816a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCCO.0b013e32834d816a%26sid%3Dliteratum%253Aachs%26aulast%3DBorodovsky%26aufirst%3DA.%26aulast%3DSeltzer%26aufirst%3DM.%2BJ.%26aulast%3DRiggins%26aufirst%3DG.%2BJ.%26atitle%3DAltered%2520Cancer%2520Cell%2520Metabolism%2520in%2520Gliomas%2520with%2520Mutant%2520IDH1%2520or%2520IDH2%26jtitle%3DCurr.%2520Opin.%2520Oncol.%26date%3D2012%26volume%3D24%26spage%3D83%26epage%3D89%26doi%3D10.1097%2FCCO.0b013e32834d816a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, J.</span>; <span class="NLM_string-name">Imai, S.-i.</span></span> <span> </span><span class="NLM_article-title">Accurate Measurement of Nicotinamide Adenine Dinucleotide (NAD(+)) with High-Performance Liquid Chromatography</span>. In  <i>Sirtuins</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirschey, M.</span></span>, Ed.; <span class="NLM_series">Methods in Molecular Biology (Methods and Protocols)</span>; <span class="NLM_publisher-name">Humana Press</span>: <span class="NLM_publisher-loc">Totowa, NJ</span>, <span class="NLM_year">2013</span>; Vol.  <span class="NLM_volume">1077</span>, pp  <span class="NLM_fpage">203</span>– <span class="NLM_lpage">215</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1007%2F978-1-62703-637-5_14" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=203-215&author=J.+Yoshino&author=S.-i.+Imaiauthor=M.+Hirschey&title=Sirtuins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1007%2F978-1-62703-637-5_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-62703-637-5_14%26sid%3Dliteratum%253Aachs%26aulast%3DYoshino%26aufirst%3DJ.%26atitle%3DAccurate%2520Measurement%2520of%2520Nicotinamide%2520Adenine%2520Dinucleotide%2520%2528NAD%2528%252B%2529%2529%2520with%2520High-Performance%2520Liquid%2520Chromatography%26btitle%3DSirtuins%26aulast%3DHirschey%26aufirst%3DM.%26pub%3DHumana%2520Press%26date%3D2013%26volume%3D1077%26spage%3D203%26epage%3D215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capka, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasibhatla, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laffitte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagliarini, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span> <span> </span><span class="NLM_article-title">Phosphoglycerate Dehydrogenase is Dispensable for Breast Tumor Maintenance and Growth</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2502</span>– <span class="NLM_lpage">2511</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.1540</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.18632%2Foncotarget.1540" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=24318446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC2c3ks1yksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=2502-2511&author=J.+Chenauthor=F.+Chungauthor=G.+Yangauthor=M.+Puauthor=H.+Gaoauthor=W.+Jiangauthor=H.+Yinauthor=V.+Capkaauthor=S.+Kasibhatlaauthor=B.+Laffitteauthor=S.+Jaegerauthor=R.+Pagliariniauthor=Y.+Chenauthor=W.+Zhou&title=Phosphoglycerate+Dehydrogenase+is+Dispensable+for+Breast+Tumor+Maintenance+and+Growth&doi=10.18632%2Foncotarget.1540"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphoglycerate dehydrogenase is dispensable for breast tumor maintenance and growth</span></div><div class="casAuthors">Chen Jinyun; Chung Franklin; Yang Guizhi; Pu Minying; Gao Hui; Jiang Wei; Yin Hong; Capka Vladimir; Kasibhatla Shailaja; Laffitte Bryan; Jaeger Savina; Pagliarini Raymond; Chen Yaoyu; Zhou Wenlai</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2502-11</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells rely on aerobic glycolysis to maintain cell growth and proliferation via the Warburg effect.  Phosphoglycerate dehydrogenase (PHDGH) catalyzes the first step of the serine biosynthetic pathway downstream of glycolysis, which is a metabolic gatekeeper both for macromolecular biosynthesis and serine-dependent DNA synthesis.  Here, we report that PHDGH is overexpressed in many ER-negative human breast cancer cell lines.  PHGDH knockdown in these cells leads to a reduction of serine synthesis and impairment of cancer cell proliferation.  However, PHGDH knockdown does not affect tumor maintenance and growth in established breast cancer xenograft models, suggesting that PHGDH-dependent cancer cell growth may be context-dependent.  Our findings suggest that other mechanisms or pathways may bypass exclusive dependence on PHGDH in established human breast cancer xenografts, indicating that PHGDH is dispensable for the growth and maintenance and of tumors in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRPyCo0Yw55VwyMywPm6HgQfW6udTcc2eat5AjJb_nKzbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c3ks1yksw%253D%253D&md5=40631d3b54bdcaa27fbe92c262d83442</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1540&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1540%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DChung%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DPu%26aufirst%3DM.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DW.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DCapka%26aufirst%3DV.%26aulast%3DKasibhatla%26aufirst%3DS.%26aulast%3DLaffitte%26aufirst%3DB.%26aulast%3DJaeger%26aufirst%3DS.%26aulast%3DPagliarini%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26atitle%3DPhosphoglycerate%2520Dehydrogenase%2520is%2520Dispensable%2520for%2520Breast%2520Tumor%2520Maintenance%2520and%2520Growth%26jtitle%3DOncotarget%26date%3D2013%26volume%3D4%26spage%3D2502%26epage%3D2511%26doi%3D10.18632%2Foncotarget.1540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucki, N. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beheshti
Zavareh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anglin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomes, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolay, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blenis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendt, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asara, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeNicola, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssiotis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lairson, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span> <span> </span><span class="NLM_article-title">Identification of a Small Molecule Inhibitor of 3-Phosphoglycerate Dehydrogenase to Target Serine Biosynthesis in Cancers</span>. <i>Proc. Natl. Acad. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">1778</span>– <span class="NLM_lpage">1783</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521548113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1073%2Fpnas.1521548113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=26831078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWgsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=1778-1783&author=E.+Mullarkyauthor=N.+C.+Luckiauthor=R.+Beheshti%0AZavarehauthor=J.+L.+Anglinauthor=A.+P.+Gomesauthor=B.+N.+Nicolayauthor=J.+C.+Y.+Wongauthor=S.+Christenauthor=H.+Takahashiauthor=P.+K.+Singhauthor=J.+Blenisauthor=J.+D.+Warrenauthor=S.-M.+Fendtauthor=J.+M.+Asaraauthor=G.+M.+DeNicolaauthor=C.+A.+Lyssiotisauthor=L.+L.+Lairsonauthor=L.+C.+Cantley&title=Identification+of+a+Small+Molecule+Inhibitor+of+3-Phosphoglycerate+Dehydrogenase+to+Target+Serine+Biosynthesis+in+Cancers&doi=10.1073%2Fpnas.1521548113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a small molecule inhibitor of 3-phosphoglycerate dehydrogenase to target serine biosynthesis in cancers</span></div><div class="casAuthors">Mullarky, Edouard; Lucki, Natasha C.; Zavareh, Reza Beheshti; Anglin, Justin L.; Gomes, Ana P.; Nicolay, Brandon N.; Wong, Jenny C. Y.; Christen, Stefan; Takahashi, Hidenori; Singh, Pradeep K.; Blenis, John; Warren, J. David; Fendt, Sarah-Maria; Asara, John M.; DeNicola, Gina M.; Lyssiotis, Costas A.; Lairson, Luke L.; Cantley, Lewis C.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1778-1783</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Cancer cells reprogram their metab. to promote growth and proliferation.  The genetic evidence pointing to the importance of the amino acid serine in tumorigenesis is striking.  The gene encoding the enzyme 3-phosphoglycerate dehydrogenase (PHGDH), which catalyzes the first committed step of serine biosynthesis, is overexpressed in tumors and cancer cell lines via focal amplification and nuclear factor erythroid-2-related factor 2 (NRF2)-mediated up-regulation.  PHGDH-overexpressing cells are exquisitely sensitive to genetic ablation of the pathway.  Here, we report the discovery of a selective small mol. inhibitor of PHGDH, CBR-5884, identified by screening a library of 800,000 drug-like compds.  CBR-5884 inhibited de novo serine synthesis in cancer cells and was selectively toxic to cancer cell lines with high serine biosynthetic activity.  Biochem. characterization of the inhibitor revealed that it was a noncompetitive inhibitor that showed a time-dependent onset of inhibition and disrupted the oligomerization state of PHGDH.  The identification of a small mol. inhibitor of PHGDH not only enables thorough preclin. evaluation of PHGDH as a target in cancers, but also provides a tool with which to study serine metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro8G0eAvxBCbVg90H21EOLACvtfcHk0liuqNJG5p3_Wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWgsr8%253D&md5=a0590d5cc4da060436d2df39d74455df</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521548113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521548113%26sid%3Dliteratum%253Aachs%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DLucki%26aufirst%3DN.%2BC.%26aulast%3DBeheshti%2BZavareh%26aufirst%3DR.%26aulast%3DAnglin%26aufirst%3DJ.%2BL.%26aulast%3DGomes%26aufirst%3DA.%2BP.%26aulast%3DNicolay%26aufirst%3DB.%2BN.%26aulast%3DWong%26aufirst%3DJ.%2BC.%2BY.%26aulast%3DChristen%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DH.%26aulast%3DSingh%26aufirst%3DP.%2BK.%26aulast%3DBlenis%26aufirst%3DJ.%26aulast%3DWarren%26aufirst%3DJ.%2BD.%26aulast%3DFendt%26aufirst%3DS.-M.%26aulast%3DAsara%26aufirst%3DJ.%2BM.%26aulast%3DDeNicola%26aufirst%3DG.%2BM.%26aulast%3DLyssiotis%26aufirst%3DC.%2BA.%26aulast%3DLairson%26aufirst%3DL.%2BL.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26atitle%3DIdentification%2520of%2520a%2520Small%2520Molecule%2520Inhibitor%2520of%25203-Phosphoglycerate%2520Dehydrogenase%2520to%2520Target%2520Serine%2520Biosynthesis%2520in%2520Cancers%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%26date%3D2016%26volume%3D113%26spage%3D1778%26epage%3D1783%26doi%3D10.1073%2Fpnas.1521548113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swier, L. J. Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Possemato, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiske, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freinkman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Birsoy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abu-Remaileh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaul, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luengo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westover, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vander Heiden, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span> <span> </span><span class="NLM_article-title">A PHGDH Inhibitor Reveals Coordination of Serine Synthesis and 1-Carbon Unit Fate</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">452</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1038%2Fnchembio.2070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27110680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVaktLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=452-458&author=M.+E.+Pacoldauthor=K.+R.+Brimacombeauthor=S.+H.+Chanauthor=J.+M.+Rohdeauthor=C.+A.+Lewisauthor=L.+J.+Y.+M.+Swierauthor=R.+Possematoauthor=W.+W.+Chenauthor=L.+B.+Sullivanauthor=B.+P.+Fiskeauthor=S.+W.+Choauthor=E.+Freinkmanauthor=K.+Birsoyauthor=M.+Abu-Remailehauthor=Y.+D.+Shaulauthor=C.+M.+Liuauthor=M.+Zhouauthor=M.+J.+Kohauthor=H.+Chungauthor=S.+M.+Davidsonauthor=A.+Luengoauthor=A.+Q.+Wangauthor=X.+Xuauthor=A.+Yasgarauthor=L.+Liuauthor=G.+Raiauthor=K.+D.+Westoverauthor=M.+G.+Vander+Heidenauthor=M.+Shenauthor=N.+S.+Grayauthor=M.+B.+Boxerauthor=D.+M.+Sabatini&title=A+PHGDH+Inhibitor+Reveals+Coordination+of+Serine+Synthesis+and+1-Carbon+Unit+Fate&doi=10.1038%2Fnchembio.2070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A PHGDH inhibitor reveals coordination of serine synthesis and one-carbon unit fate</span></div><div class="casAuthors">Pacold, Michael E.; Brimacombe, Kyle R.; Chan, Sze Ham; Rohde, Jason M.; Lewis, Caroline A.; Swier, Lotteke J. Y. M.; Possemato, Richard; Chen, Walter W.; Sullivan, Lucas B.; Fiske, Brian P.; Cho, Steve; Freinkman, Elizaveta; Birsoy, Kivanc; Abu-Remaileh, Monther; Shaul, Yoav D.; Liu, Chieh Min; Zhou, Minerva; Koh, Min Jung; Chung, Haeyoon; Davidson, Shawn M.; Luengo, Alba; Wang, Amy Q.; Xu, Xin; Yasgar, Adam; Liu, Li; Rai, Ganesha; Westover, Kenneth D.; Vander Heiden, Matthew G.; Shen, Min; Gray, Nathanael S.; Boxer, Matthew B.; Sabatini, David M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">452-458</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Serine is both a proteinogenic amino acid and the source of one-carbon units essential for de novo purine and deoxythymidine synthesis.  In the canonical pathway of glucose-derived serine synthesis, Homo sapiens phosphoglycerate dehydrogenase (PHGDH) catalyzes the first, rate-limiting step.  Genetic loss of PHGDH is toxic toward PHGDH-overexpressing breast cancer cell lines even in the presence of exogenous serine.  Here, we used a quant. high-throughput screen to identify small-mol. PHGDH inhibitors.  These compds. reduce the prodn. of glucose-derived serine in cells and suppress the growth of PHGDH-dependent cancer cells in culture and in orthotopic xenograft tumors.  Surprisingly, PHGDH inhibition reduced the incorporation into nucleotides of one-carbon units from glucose-derived and exogenous serine.  We conclude that glycolytic serine synthesis coordinates the use of one-carbon units from endogenous and exogenous serine in nucleotide synthesis, and we suggest that one-carbon unit wasting thus may contribute to the efficacy of PHGDH inhibitors in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-N14m5iKDe7Vg90H21EOLACvtfcHk0lgGw7K4TOY7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVaktLs%253D&md5=4917bfb6309946cd08b593ed26dbe73c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2070%26sid%3Dliteratum%253Aachs%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DChan%26aufirst%3DS.%2BH.%26aulast%3DRohde%26aufirst%3DJ.%2BM.%26aulast%3DLewis%26aufirst%3DC.%2BA.%26aulast%3DSwier%26aufirst%3DL.%2BJ.%2BY.%2BM.%26aulast%3DPossemato%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DW.%2BW.%26aulast%3DSullivan%26aufirst%3DL.%2BB.%26aulast%3DFiske%26aufirst%3DB.%2BP.%26aulast%3DCho%26aufirst%3DS.%2BW.%26aulast%3DFreinkman%26aufirst%3DE.%26aulast%3DBirsoy%26aufirst%3DK.%26aulast%3DAbu-Remaileh%26aufirst%3DM.%26aulast%3DShaul%26aufirst%3DY.%2BD.%26aulast%3DLiu%26aufirst%3DC.%2BM.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DKoh%26aufirst%3DM.%2BJ.%26aulast%3DChung%26aufirst%3DH.%26aulast%3DDavidson%26aufirst%3DS.%2BM.%26aulast%3DLuengo%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DA.%2BQ.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DVander%2BHeiden%26aufirst%3DM.%2BG.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26atitle%3DA%2520PHGDH%2520Inhibitor%2520Reveals%2520Coordination%2520of%2520Serine%2520Synthesis%2520and%25201-Carbon%2520Unit%2520Fate%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D452%26epage%3D458%26doi%3D10.1038%2Fnchembio.2070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ravez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillier, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robin, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullarky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantley, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frédérick, R.</span></span> <span> </span><span class="NLM_article-title">α-Ketothioamide Derivatives: A Promising Tool to Interrogate Phosphoglycerate Dehydrogenase (PHGDH)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1591</span>– <span class="NLM_lpage">1597</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01166</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01166" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKisrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1591-1597&author=S.+Ravezauthor=C.+Corbetauthor=Q.+Spillierauthor=A.+Dutuauthor=A.+D.+Robinauthor=E.+Mullarkyauthor=L.+C.+Cantleyauthor=O.+Feronauthor=R.+Fr%C3%A9d%C3%A9rick&title=%CE%B1-Ketothioamide+Derivatives%3A+A+Promising+Tool+to+Interrogate+Phosphoglycerate+Dehydrogenase+%28PHGDH%29&doi=10.1021%2Facs.jmedchem.6b01166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">α-Ketothioamide Derivatives: A Promising Tool to Interrogate Phosphoglycerate Dehydrogenase (PHGDH)</span></div><div class="casAuthors">Ravez, Severine; Corbet, Cyril; Spillier, Quentin; Dutu, Alice; Robin, Anita D.; Mullarky, Edouard; Cantley, Lewis C.; Feron, Olivier; Frederick, Raphael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1591-1597</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Given the putative role of PHGDH in cancer, development of inhibitors is required to explore its function.  In this context, we established and validated a straightforward enzymic assay suitable for high-throughput screening and we identified inhibitors with similar chem. scaffolds.  Through a convergent pharmacophore approach, we synthesized α-ketothioamides that exhibit interesting in vitro PHGDH inhibition and encouraging cellular results.  These novel probes may be used to understand the emerging biol. of this metabolic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMG07MHJ9Fx7Vg90H21EOLACvtfcHk0lgGw7K4TOY7jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKisrw%253D&md5=dae86884b63554c46fdf33d8e2cf42cd</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01166%26sid%3Dliteratum%253Aachs%26aulast%3DRavez%26aufirst%3DS.%26aulast%3DCorbet%26aufirst%3DC.%26aulast%3DSpillier%26aufirst%3DQ.%26aulast%3DDutu%26aufirst%3DA.%26aulast%3DRobin%26aufirst%3DA.%2BD.%26aulast%3DMullarky%26aufirst%3DE.%26aulast%3DCantley%26aufirst%3DL.%2BC.%26aulast%3DFeron%26aufirst%3DO.%26aulast%3DFr%25C3%25A9d%25C3%25A9rick%26aufirst%3DR.%26atitle%3D%25CE%25B1-Ketothioamide%2520Derivatives%253A%2520A%2520Promising%2520Tool%2520to%2520Interrogate%2520Phosphoglycerate%2520Dehydrogenase%2520%2528PHGDH%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1591%26epage%3D1597%26doi%3D10.1021%2Facs.jmedchem.6b01166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liberti, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locasale, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Selective Allosteric Inhibitors of PHGDH and Serine Synthesis with Anti-tumor Activity</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">55</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.chembiol.2016.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=28042046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1yhsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=55-65&author=Q.+Wangauthor=M.+V.+Libertiauthor=P.+Liuauthor=X.+Dengauthor=Y.+Liuauthor=J.+W.+Locasaleauthor=L.+Lai&title=Rational+Design+of+Selective+Allosteric+Inhibitors+of+PHGDH+and+Serine+Synthesis+with+Anti-tumor+Activity&doi=10.1016%2Fj.chembiol.2016.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Selective Allosteric Inhibitors of PHGDH and Serine Synthesis with Anti-tumor Activity</span></div><div class="casAuthors">Wang, Qian; Liberti, Maria V.; Liu, Pei; Deng, Xiaobing; Liu, Ying; Locasale, Jason W.; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">55-65</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Metabolic reprogramming in cancer cells facilitates growth and proliferation.  Increased activity of the serine biosynthetic pathway through the enzyme phosphoglycerate dehydrogenase (PHGDH) contributes to tumorigenesis.  With a small substrate and a weak binding cofactor, (NAD+), inhibitor development for PHGDH remains challenging.  Instead of targeting the PHGDH active site, we computationally identified two potential allosteric sites and virtually screened compds. that can bind to these sites.  With subsequent characterization, we successfully identified PHGDH non-NAD+-competing allosteric inhibitors that attenuate its enzyme activity, selectively inhibit de novo serine synthesis in cancer cells, and reduce tumor growth in vivo.  Our study not only identifies novel allosteric inhibitors for PHGDH to probe its function and potential as a therapeutic target, but also provides a general strategy for the rational design of small-mol. modulators of metabolic enzyme function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5L2K1-sIbP7Vg90H21EOLACvtfcHk0ljHGRzKb_-5hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1yhsg%253D%253D&md5=bad62b07802dbf0d7a3bdbd1fd151b39</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLiberti%26aufirst%3DM.%2BV.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DLocasale%26aufirst%3DJ.%2BW.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DRational%2520Design%2520of%2520Selective%2520Allosteric%2520Inhibitors%2520of%2520PHGDH%2520and%2520Serine%2520Synthesis%2520with%2520Anti-tumor%2520Activity%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D55%26epage%3D65%26doi%3D10.1016%2Fj.chembiol.2016.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spadola, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönherr, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edfeldt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabow, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R.</span></span> <span> </span><span class="NLM_article-title">An Improved Model for Fragment-based Lead Generation at AstraZeneca</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1272</span>– <span class="NLM_lpage">1283</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.04.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.drudis.2016.04.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=27179986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC28bpvVyntQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2016&pages=1272-1283&author=N.+Fullerauthor=L.+Spadolaauthor=S.+Cowenauthor=J.+Patelauthor=H.+Sch%C3%B6nherrauthor=Q.+Caoauthor=A.+McKenzieauthor=F.+Edfeldtauthor=A.+Rabowauthor=R.+Goodnow&title=An+Improved+Model+for+Fragment-based+Lead+Generation+at+AstraZeneca&doi=10.1016%2Fj.drudis.2016.04.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">An improved model for fragment-based lead generation at AstraZeneca</span></div><div class="casAuthors">Fuller Nathan; Spadola Loredana; Cowen Scott; Patel Joe; Schonherr Heike; Cao Qing; McKenzie Andrew; Goodnow Robert; Edfeldt Fredrik; Rabow Al</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1272-83</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Modest success rates in fragment-based lead generation (FBLG) projects at AstraZeneca (AZ) prompted operational changes to improve performance.  In this review, we summarize these changes, emphasizing the construction and composition of the AZ fragment library, screening practices and working model.  We describe the profiles of the screening method for specific fragment subsets and statistically assess our ability to follow up on fragment hits through near-neighbor selection.  Performance analysis of our second-generation fragment library (FL2) in screening campaigns illustrates the complementary nature of flat and 3D fragments in exploring protein-binding pockets and highlights our ability to deliver fragment hits using multiple screening techniques for various target classes.  The new model has had profound impact on the successful delivery of lead series to drug discovery projects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhSnzpo3gNWkloaUqgkFHkfW6udTcc2ebGGtZKWySF9Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28bpvVyntQ%253D%253D&md5=d8c40172502e32be3173afcd292be480</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.04.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.04.023%26sid%3Dliteratum%253Aachs%26aulast%3DFuller%26aufirst%3DN.%26aulast%3DSpadola%26aufirst%3DL.%26aulast%3DCowen%26aufirst%3DS.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DSch%25C3%25B6nherr%26aufirst%3DH.%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DMcKenzie%26aufirst%3DA.%26aulast%3DEdfeldt%26aufirst%3DF.%26aulast%3DRabow%26aufirst%3DA.%26aulast%3DGoodnow%26aufirst%3DR.%26atitle%3DAn%2520Improved%2520Model%2520for%2520Fragment-based%2520Lead%2520Generation%2520at%2520AstraZeneca%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2016%26volume%3D21%26spage%3D1272%26epage%3D1283%26doi%3D10.1016%2Fj.drudis.2016.04.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unterlass, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackburn, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span> <span> </span><span class="NLM_article-title">Validating and Enabling Phosphoglycerate Dehydrogenase (PHGDH) as a Target for Fragment-based Drug Discovery in PHGDH-amplified Breast Cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">13139</span>– <span class="NLM_lpage">13153</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.11487</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.18632%2Foncotarget.11487" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29568346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC1MnktValsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=13139-13153&author=J.+E.+Unterlassauthor=A.+Basleauthor=T.+J.+Blackburnauthor=J.+Tuckerauthor=C.+Canoauthor=M.+E.+M.+Nobleauthor=N.+J.+Curtin&title=Validating+and+Enabling+Phosphoglycerate+Dehydrogenase+%28PHGDH%29+as+a+Target+for+Fragment-based+Drug+Discovery+in+PHGDH-amplified+Breast+Cancer&doi=10.18632%2Foncotarget.11487"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Validating and enabling phosphoglycerate dehydrogenase (PHGDH) as a target for fragment-based drug discovery in PHGDH-amplified breast cancer</span></div><div class="casAuthors">Unterlass Judith E; Tucker Julie; Noble Martin E M; Curtin Nicola J; Basle Arnaud; Blackburn Timothy J; Cano Celine</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">13139-13153</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">3-Phosphoglycerate dehydrogenase (PHGDH) has recently been identified as an attractive target in cancer therapy as it links upregulated glycolytic flux to increased biomass production in cancer cells.  PHGDH catalyses the first step in the serine synthesis pathway and thus diverts glycolytic flux into serine synthesis.  We have used siRNA-mediated suppression of PHGDH expression to show that PHGDH is a potential therapeutic target in PHGDH-amplified breast cancer.  Knockdown caused reduced proliferation in the PHGDH-amplified cell line MDA-MB-468, whereas breast cancer cells with low PHGDH expression or with elevated PHGDH expression in the absence of genomic amplification were not affected.  As a first step towards design of a chemical probe for PHGDH, we report a fragment-based drug discovery approach for the identification of PHGDH inhibitors.  We designed a truncated PHGDH construct that gave crystals which diffracted to high resolution, and could be used for fragment soaking. 15 fragments stabilising PHGDH were identified using a thermal shift assay and validated by X-ray crystallography and ITC competition experiments to exhibit 1.5-26.2 mM affinity for PHGDH.  A structure-guided fragment growing approach was applied to the PHGDH binders from the initial screen, yielding greater understanding of the binding site and suggesting routes to achieve higher affinity NAD-competitive inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRatiF0vPAX33y-c-E1v-fmfW6udTcc2ebGGtZKWySF9Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MnktValsg%253D%253D&md5=4a2576e240366888437f5fae951fac7a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.11487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.11487%26sid%3Dliteratum%253Aachs%26aulast%3DUnterlass%26aufirst%3DJ.%2BE.%26aulast%3DBasle%26aufirst%3DA.%26aulast%3DBlackburn%26aufirst%3DT.%2BJ.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DCano%26aufirst%3DC.%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DValidating%2520and%2520Enabling%2520Phosphoglycerate%2520Dehydrogenase%2520%2528PHGDH%2529%2520as%2520a%2520Target%2520for%2520Fragment-based%2520Drug%2520Discovery%2520in%2520PHGDH-amplified%2520Breast%2520Cancer%26jtitle%3DOncotarget%26date%3D2018%26volume%3D9%26spage%3D13139%26epage%3D13153%26doi%3D10.18632%2Foncotarget.11487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raze Therapeutics, I.</span>; <span class="NLM_string-name">Mainolfi, N.</span>; <span class="NLM_string-name">Moyer, M. P.</span></span> <span> </span><span class="NLM_article-title">3-Phosphoglycerate Dehydrogenase Inhibitors and Uses Thereof</span>. <span class="NLM_patent">WO2017156177</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=I.+Raze+Therapeutics&author=N.+Mainolfi&author=M.+P.+Moyer&title=3-Phosphoglycerate+Dehydrogenase+Inhibitors+and+Uses+Thereof"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DRaze%2BTherapeutics%26aufirst%3DI.%26atitle%3D3-Phosphoglycerate%2520Dehydrogenase%2520Inhibitors%2520and%2520Uses%2520Thereof%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rohde, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brimacombe, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pacold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasgar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheff, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baljinnyam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simeonov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatini, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span> <span> </span><span class="NLM_article-title">Discovery and Optimization of Piperazine-1-Thiourea-based Human Phosphoglycerate Dehydrogenase Inhibitors</span>. <i>Biorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1739</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2018.02.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1016%2Fj.bmc.2018.02.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29555419" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=1727-1739&author=J.+M.+Rohdeauthor=K.+R.+Brimacombeauthor=L.+Liuauthor=M.+E.+Pacoldauthor=A.+Yasgarauthor=D.+M.+Cheffauthor=T.+D.+Leeauthor=G.+Raiauthor=B.+Baljinnyamauthor=Z.+Liauthor=A.+Simeonovauthor=M.+D.+Hallauthor=M.+Shenauthor=D.+M.+Sabatiniauthor=M.+B.+Boxer&title=Discovery+and+Optimization+of+Piperazine-1-Thiourea-based+Human+Phosphoglycerate+Dehydrogenase+Inhibitors&doi=10.1016%2Fj.bmc.2018.02.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2018.02.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2018.02.016%26sid%3Dliteratum%253Aachs%26aulast%3DRohde%26aufirst%3DJ.%2BM.%26aulast%3DBrimacombe%26aufirst%3DK.%2BR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DPacold%26aufirst%3DM.%2BE.%26aulast%3DYasgar%26aufirst%3DA.%26aulast%3DCheff%26aufirst%3DD.%2BM.%26aulast%3DLee%26aufirst%3DT.%2BD.%26aulast%3DRai%26aufirst%3DG.%26aulast%3DBaljinnyam%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DSimeonov%26aufirst%3DA.%26aulast%3DHall%26aufirst%3DM.%2BD.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DSabatini%26aufirst%3DD.%2BM.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26atitle%3DDiscovery%2520and%2520Optimization%2520of%2520Piperazine-1-Thiourea-based%2520Human%2520Phosphoglycerate%2520Dehydrogenase%2520Inhibitors%26jtitle%3DBiorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D1727%26epage%3D1739%26doi%3D10.1016%2Fj.bmc.2018.02.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+Substructure+Filters+for+Removal+of+Pan+Assay+Interference+Compounds+%28PAINS%29+from+Screening+Libraries+and+for+Their+Exclusion+in+Bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhBIwReUfATKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520Substructure%2520Filters%2520for%2520Removal%2520of%2520Pan%2520Assay%2520Interference%2520Compounds%2520%2528PAINS%2529%2520from%2520Screening%2520Libraries%2520and%2520for%2520Their%2520Exclusion%2520in%2520Bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ward, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brassington, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breeze, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caputo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchlow, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodwin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassall, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holdgate, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrosek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norman, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tart, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vogtherr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, K.</span></span> <span> </span><span class="NLM_article-title">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">3285</span>– <span class="NLM_lpage">3306</span>, <span class="refDoi"> DOI: 10.1021/jm201734r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201734r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=3285-3306&author=R.+A.+Wardauthor=C.+Brassingtonauthor=A.+L.+Breezeauthor=A.+Caputoauthor=S.+Critchlowauthor=G.+Daviesauthor=L.+Goodwinauthor=G.+Hassallauthor=R.+Greenwoodauthor=G.+A.+Holdgateauthor=M.+Mrosekauthor=R.+A.+Normanauthor=S.+Pearsonauthor=J.+Tartauthor=J.+A.+Tuckerauthor=M.+Vogtherrauthor=D.+Whittakerauthor=J.+Wingfieldauthor=J.+Winterauthor=K.+Hudson&title=Design+and+Synthesis+of+Novel+Lactate+Dehydrogenase+A+Inhibitors+by+Fragment-Based+Lead+Generation&doi=10.1021%2Fjm201734r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Lactate Dehydrogenase A Inhibitors by Fragment-Based Lead Generation</span></div><div class="casAuthors">Ward, Richard A.; Brassington, Claire; Breeze, Alexander L.; Caputo, Alessandro; Critchlow, Susan; Davies, Gareth; Goodwin, Louise; Hassall, Giles; Greenwood, Ryan; Holdgate, Geoffrey A.; Mrosek, Michael; Norman, Richard A.; Pearson, Stuart; Tart, Jonathan; Tucker, Julie A.; Vogtherr, Martin; Whittaker, David; Wingfield, Jonathan; Winter, Jon; Hudson, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3285-3306</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lactate dehydrogenase A (LDHA) catalyzes the conversion of pyruvate to lactate, utilizing NADH as a cofactor.  It has been identified as a potential therapeutic target in the area of cancer metab.  In this manuscript we report our progress using fragment-based lead generation (FBLG), assisted by x-ray crystallog. to develop small mol. LDHA inhibitors.  Fragment hits were identified through NMR and SPR screening and optimized into lead compds. with nanomolar binding affinities via fragment linking.  Also reported is their modification into cellular active compds. suitable for target validation work.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH8QbdIUXNvbVg90H21EOLACvtfcHk0lhBIwReUfATKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVSgtLs%253D&md5=d638e7a0db361c212ee084d5c05ee70c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm201734r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201734r%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DBrassington%26aufirst%3DC.%26aulast%3DBreeze%26aufirst%3DA.%2BL.%26aulast%3DCaputo%26aufirst%3DA.%26aulast%3DCritchlow%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DG.%26aulast%3DGoodwin%26aufirst%3DL.%26aulast%3DHassall%26aufirst%3DG.%26aulast%3DGreenwood%26aufirst%3DR.%26aulast%3DHoldgate%26aufirst%3DG.%2BA.%26aulast%3DMrosek%26aufirst%3DM.%26aulast%3DNorman%26aufirst%3DR.%2BA.%26aulast%3DPearson%26aufirst%3DS.%26aulast%3DTart%26aufirst%3DJ.%26aulast%3DTucker%26aufirst%3DJ.%2BA.%26aulast%3DVogtherr%26aufirst%3DM.%26aulast%3DWhittaker%26aufirst%3DD.%26aulast%3DWingfield%26aufirst%3DJ.%26aulast%3DWinter%26aufirst%3DJ.%26aulast%3DHudson%26aufirst%3DK.%26atitle%3DDesign%2520and%2520Synthesis%2520of%2520Novel%2520Lactate%2520Dehydrogenase%2520A%2520Inhibitors%2520by%2520Fragment-Based%2520Lead%2520Generation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D3285%26epage%3D3306%26doi%3D10.1021%2Fjm201734r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, B.</span></span> <span> </span><span class="NLM_article-title">Characterization of Ligand Binding by Saturation Transfer Difference NMR Spectroscopy</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">1784</span>– <span class="NLM_lpage">1788</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1521-3773(19990614)38:12<1784::AID-ANIE1784>3.0.CO;2-Q</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=1784-1788&author=M.+Mayerauthor=B.+Meyer&title=Characterization+of+Ligand+Binding+by+Saturation+Transfer+Difference+NMR+Spectroscopy&doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of ligand binding by saturation transfer difference NMR spectroscopy</span></div><div class="casAuthors">Mayer, Moriz; Meyer, Bernd</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1784-1788</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH</span>)
        </div><div class="casAbstract">The difference between a satn. transfer spectrum and a normal NMR spectrum provides a new and fast method (satn. transfer difference (STD) NMR spectroscopy) to screen compd. libraries for binding activity to proteins.  STD NMR spectroscopy of mixts. of potential ligands with as little as 1 nmol of protein yields 1D and 2D NMR spectra that exclusively show signals from mols. with binding affinity.  In addn., the ligand's binding epitope is easily identified because ligand residues in direct contact to the protein show much stronger STD signals.  For example, the binding specificity of Lewisb-hexasaccharide to Aleulria aurantia agglutinin (AAA) can be mapped to the two fucosyl residues.  The efficiency of the STD NMR technique is shown by the binding of N-acetylglucosamine (GlcNAc) to wheat germ agglutinin (WGA).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf8j8HiAqDArVg90H21EOLACvtfcHk0lh3eIHQVTYvdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXktVCntb0%253D&md5=03f825af40d22911062112dc6074cf46</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291521-3773%252819990614%252938%253A12%253C1784%253A%253AAID-ANIE1784%253E3.0.CO%253B2-Q%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%26aulast%3DMeyer%26aufirst%3DB.%26atitle%3DCharacterization%2520of%2520Ligand%2520Binding%2520by%2520Saturation%2520Transfer%2520Difference%2520NMR%2520Spectroscopy%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D1999%26volume%3D38%26spage%3D1784%26epage%3D1788%26doi%3D10.1002%2F%28SICI%291521-3773%2819990614%2938%3A12%3C1784%3A%3AAID-ANIE1784%3E3.0.CO%3B2-Q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Unterlass, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cano, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noble, M. M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtin, N. J.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into the Enzymatic Activity and Potential Substrate Promiscuity of Human 3-Phosphoglycerate Dehydrogenase (PHGDH)</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">104478</span> <span class="refDoi"> DOI: 10.18632/oncotarget.22327</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.18632%2Foncotarget.22327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=29262655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A280%3ADC%252BC1Mzjs1emsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&author=J.+E.+Unterlassauthor=R.+J.+Woodauthor=A.+Basleauthor=J.+Tuckerauthor=C.+Canoauthor=M.+M.+E.+Nobleauthor=N.+J.+Curtin&title=Structural+Insights+into+the+Enzymatic+Activity+and+Potential+Substrate+Promiscuity+of+Human+3-Phosphoglycerate+Dehydrogenase+%28PHGDH%29&doi=10.18632%2Foncotarget.22327"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into the enzymatic activity and potential substrate promiscuity of human 3-phosphoglycerate dehydrogenase (PHGDH)</span></div><div class="casAuthors">Unterlass Judith E; Tucker Julie; Noble Martin M E; Curtin Nicola J; Unterlass Judith E; Wood Robert J; Basle Arnaud; Cano Celine</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">61</span>),
    <span class="NLM_cas:pages">104478-104491</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cancer cells reprogram their metabolism and energy production to sustain increased growth, enable metastasis and overcome resistance to cancer treatments.  Although primary roles for many metabolic proteins have been identified, some are promiscuous in regards to the reaction they catalyze.  To efficiently target these enzymes, a good understanding of their enzymatic function and structure, as well as knowledge regarding any substrate or catalytic promiscuity is required.  Here we focus on the characterization of human 3-phosphoglycerate dehydrogenase (PHGDH).  PHGDH catalyzes the NAD(+)-dependent conversion of 3-phosphoglycerate to phosphohydroxypyruvate, which is the first step in the de novo synthesis pathway of serine, a critical amino acid for protein and nucleic acid biosynthesis.  We have investigated substrate analogues to assess whether PHGDH might possess other enzymatic roles that could explain its occasional over-expression in cancer, as well as to help with the design of specific inhibitors.  We also report the crystal structure of the catalytic subunit of human PHGDH, a dimer, solved with bound cofactor in one monomer and both cofactor and L-tartrate in the second monomer.  In vitro enzyme activity measurements show that the catalytic subunit of PHGDH is still active and that PHGDH activity could be significantly inhibited with adenosine 5'-diphosphoribose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRJ7MGLt-1wSMERNNga7KD-fW6udTcc2eY2uQMNLFcpErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mzjs1emsQ%253D%253D&md5=a2844b264542c8d34f4651817e18761a</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.22327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.22327%26sid%3Dliteratum%253Aachs%26aulast%3DUnterlass%26aufirst%3DJ.%2BE.%26aulast%3DWood%26aufirst%3DR.%2BJ.%26aulast%3DBasle%26aufirst%3DA.%26aulast%3DTucker%26aufirst%3DJ.%26aulast%3DCano%26aufirst%3DC.%26aulast%3DNoble%26aufirst%3DM.%2BM.%2BE.%26aulast%3DCurtin%26aufirst%3DN.%2BJ.%26atitle%3DStructural%2520Insights%2520into%2520the%2520Enzymatic%2520Activity%2520and%2520Potential%2520Substrate%2520Promiscuity%2520of%2520Human%25203-Phosphoglycerate%2520Dehydrogenase%2520%2528PHGDH%2529%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26doi%3D10.18632%2Foncotarget.22327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, R. W.</span></span> <span> </span><span class="NLM_article-title">Allylic 1,3-Strain as a Controlling Factor in Stereoselective Transformations</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">1841</span>– <span class="NLM_lpage">1860</span>, <span class="refDoi"> DOI: 10.1021/cr00098a009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr00098a009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADyaL1MXmtlWnurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1989&pages=1841-1860&author=R.+W.+Hoffmann&title=Allylic+1%2C3-Strain+as+a+Controlling+Factor+in+Stereoselective+Transformations&doi=10.1021%2Fcr00098a009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Allylic 1,3-strain as a controlling factor in stereoselective transformations</span></div><div class="casAuthors">Hoffmann, Reinhard W.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1841-60</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    </div><div class="casAbstract">A review with 155 refs. on the manifestation of allylic strain in the stereochem. of a host of chem. reactions, including cyclizations, intermol. addn., hydrogenation, cyclopropanation, and others.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDZG59Whd3WbVg90H21EOLACvtfcHk0lh3eIHQVTYvdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtlWnurc%253D&md5=91540e91c0c31a112831dd5ac9d73f4d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fcr00098a009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr00098a009%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DR.%2BW.%26atitle%3DAllylic%25201%252C3-Strain%2520as%2520a%2520Controlling%2520Factor%2520in%2520Stereoselective%2520Transformations%26jtitle%3DChem.%2520Rev.%26date%3D1989%26volume%3D89%26spage%3D1841%26epage%3D1860%26doi%3D10.1021%2Fcr00098a009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tietze, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, G.</span></span> <span> </span><span class="NLM_article-title">Ab Initio Molecular Orbital Calculations on Allylic 1,3-Strain of Electron-Donor- and Electron-Acceptor-Substituted Alkenes</span>. <i>Liebigs Ann.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>1996</i></span>,  <span class="NLM_fpage">1575</span>– <span class="NLM_lpage">1579</span>, <span class="refDoi"> DOI: 10.1002/jlac.199619961012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1002%2Fjlac.199619961012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1996&publication_year=1996&pages=1575-1579&author=L.+F.+Tietzeauthor=G.+Schulz&title=Ab+Initio+Molecular+Orbital+Calculations+on+Allylic+1%2C3-Strain+of+Electron-Donor-+and+Electron-Acceptor-Substituted+Alkenes&doi=10.1002%2Fjlac.199619961012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fjlac.199619961012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjlac.199619961012%26sid%3Dliteratum%253Aachs%26aulast%3DTietze%26aufirst%3DL.%2BF.%26aulast%3DSchulz%26aufirst%3DG.%26atitle%3DAb%2520Initio%2520Molecular%2520Orbital%2520Calculations%2520on%2520Allylic%25201%252C3-Strain%2520of%2520Electron-Donor-%2520and%2520Electron-Acceptor-Substituted%2520Alkenes%26jtitle%3DLiebigs%2520Ann.%26date%3D1996%26volume%3D1996%26spage%3D1575%26epage%3D1579%26doi%3D10.1002%2Fjlac.199619961012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ravez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spillier, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marteau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feron, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frédérick, R.</span></span> <span> </span><span class="NLM_article-title">Challenges and Opportunities in the Development of Serine Synthetic Pathway Inhibitors for Cancer Therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1227</span>– <span class="NLM_lpage">1237</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01167</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01167" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOls7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1227-1237&author=S.+Ravezauthor=Q.+Spillierauthor=R.+Marteauauthor=O.+Feronauthor=R.+Fr%C3%A9d%C3%A9rick&title=Challenges+and+Opportunities+in+the+Development+of+Serine+Synthetic+Pathway+Inhibitors+for+Cancer+Therapy&doi=10.1021%2Facs.jmedchem.6b01167"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Challenges and Opportunities in the Development of Serine Synthetic Pathway Inhibitors for Cancer Therapy</span></div><div class="casAuthors">Ravez, Severine; Spillier, Quentin; Marteau, Romain; Feron, Olivier; Frederick, Raphael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1227-1237</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The recent advances in understanding the relation between cancer and metab. have highlighted the relevance of the serine synthetic pathway (SSP) that consists of three successive enzymic reactions.  Enzymes of the SSP, such as the phosphoglycerate dehydrogenase (PHGDH) and the phosphoserine aminotransferase 1 (PSAT-1), were recently highlighted because they are amplified in a significant subset of human tumors and their suppressions by RNAi caused a decrease in cancer cell survival and growth.  Currently, the field of drug discovery against these enzymes is still in its infancy and the identification of suitable inhibitors could serve to understand the emerging biol. of these metabolic enzymes.  In this review, the authors present the SSP as a significant and novel emerging area for medicinal chem. and the authors develop an overview of one of the key enzymes of the pathway, PHGDH.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKa3KCVECsJ7Vg90H21EOLACvtfcHk0liLWSecaw7Fvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOls7nJ&md5=79c2de60dd8cfad2bb480dc393ce23cf</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01167%26sid%3Dliteratum%253Aachs%26aulast%3DRavez%26aufirst%3DS.%26aulast%3DSpillier%26aufirst%3DQ.%26aulast%3DMarteau%26aufirst%3DR.%26aulast%3DFeron%26aufirst%3DO.%26aulast%3DFr%25C3%25A9d%25C3%25A9rick%26aufirst%3DR.%26atitle%3DChallenges%2520and%2520Opportunities%2520in%2520the%2520Development%2520of%2520Serine%2520Synthetic%2520Pathway%2520Inhibitors%2520for%2520Cancer%2520Therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1227%26epage%3D1237%26doi%3D10.1021%2Facs.jmedchem.6b01167" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caravella, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbeck, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitesides, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tidor, B.</span></span> <span> </span><span class="NLM_article-title">Long-Range Electrostatic Contributions to Protein–Ligand Binding Estimated Using Protein Charge Ladders, Affinity Capillary Electrophoresis, and Continuum Electrostatic Theory</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">4340</span>– <span class="NLM_lpage">4347</span>, <span class="refDoi"> DOI: 10.1021/ja984195a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja984195a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADyaK1MXisFOmsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=1999&pages=4340-4347&author=J.+A.+Caravellaauthor=J.+D.+Carbeckauthor=D.+C.+Duffyauthor=G.+M.+Whitesidesauthor=B.+Tidor&title=Long-Range+Electrostatic+Contributions+to+Protein%E2%80%93Ligand+Binding+Estimated+Using+Protein+Charge+Ladders%2C+Affinity+Capillary+Electrophoresis%2C+and+Continuum+Electrostatic+Theory&doi=10.1021%2Fja984195a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Long-Range Electrostatic Contributions to Protein-Ligand Binding Estimated Using Protein Charge Ladders, Affinity Capillary Electrophoresis, and Continuum Electrostatic Theory</span></div><div class="casAuthors">Caravella, Justin A.; Carbeck, Jeffrey D.; Duffy, David C.; Whitesides, George M.; Tidor, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4340-4347</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Affinity capillary electrophoresis and protein charge ladders are used together to measure the contributions of long-range electrostatic interactions to binding of substituted benzene sulfonamide inhibitors to derivs. of human carbonic anhydrase II.  The results are analyzed by continuum electrostatic calcns., which afford a detailed anal. of interactions of individual members of a population from a charge ladder.  A Monte Carlo simulation of the exptl. data using calcd. contributions of individual lysine side chains to inhibitor binding shows that a large no. of different patterns of acetylation are consistent with the exptl. results.  The calcns. predict significant differences in the contributions of some lysines to ΔG and simulations suggest that exptl. resoln. must be enhanced to be able to measure such differences.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxrl3LPkPRxrVg90H21EOLACvtfcHk0liLWSecaw7Fvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXisFOmsL8%253D&md5=87d77bea54553d1af9d7346178bdfacd</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja984195a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja984195a%26sid%3Dliteratum%253Aachs%26aulast%3DCaravella%26aufirst%3DJ.%2BA.%26aulast%3DCarbeck%26aufirst%3DJ.%2BD.%26aulast%3DDuffy%26aufirst%3DD.%2BC.%26aulast%3DWhitesides%26aufirst%3DG.%2BM.%26aulast%3DTidor%26aufirst%3DB.%26atitle%3DLong-Range%2520Electrostatic%2520Contributions%2520to%2520Protein%25E2%2580%2593Ligand%2520Binding%2520Estimated%2520Using%2520Protein%2520Charge%2520Ladders%252C%2520Affinity%2520Capillary%2520Electrophoresis%252C%2520and%2520Continuum%2520Electrostatic%2520Theory%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1999%26volume%3D121%26spage%3D4340%26epage%3D4347%26doi%3D10.1021%2Fja984195a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vallet, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wahlgren, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grenthe, I.</span></span> <span> </span><span class="NLM_article-title">Chelate Effect and Thermodynamics of Metal Complex Formation in Solution: A Quantum Chemical Study</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>125</i></span>,  <span class="NLM_fpage">14941</span>– <span class="NLM_lpage">14950</span>, <span class="refDoi"> DOI: 10.1021/ja036646j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja036646j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovVCltrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=14941-14950&author=V.+Valletauthor=U.+Wahlgrenauthor=I.+Grenthe&title=Chelate+Effect+and+Thermodynamics+of+Metal+Complex+Formation+in+Solution%3A+A+Quantum+Chemical+Study&doi=10.1021%2Fja036646j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Chelate Effect and Thermodynamics of Metal Complex Formation in Solution: A Quantum Chemical Study</span></div><div class="casAuthors">Vallet, Valerie; Wahlgren, Ulf; Grenthe, Ingmar</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">14941-14950</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The accuracy of quantum chem. predictions of structures and thermodn. data for metal complexes depends both on the quantum chem. methods and the chem. models used.  A thermodn. analog of the Eigen-Wilkins mechanism for ligand substitution reactions (Model A) turns out to be sufficiently simple to catch the essential chem. of complex formation reactions and allows quantum chem. calcns. at the ab initio level of thermodn. quantities both in gas phase and soln.; the latter by using the conductor-like polarizable continuum (CPCM) model.  Model A describes the complex formation as a two-step reaction: [M(H2O)x](aq) + L(aq) .dblharw. [M(H2O)x],L(aq); and [M(H2O)x],L(aq) .dblharw. [M(H2O)x-1L],(H2O)(aq).  The first step, the formation of an outer-sphere complex is described using the Fuoss equation and the second, the intramol. exchange between an entering ligand from the second and water in the first coordination shell, using quantum chem. methods.  The thermodn. quantities for this model were compared to those for the reaction: [M(H2O)x](aq) + L(aq) .dblharw. [M(H2O)x-1L](aq) + (H2O)(aq) (Model B), as calcd. for each reactant and product sep.  The models were tested using complex formation between Zn2+ and ammonia, methylamine, and ethylenediamine, and complex formation and chelate ring closure reactions in binary and ternary UO22+-oxalate systems.  The results show that the Gibbs energy of reaction for Model A are not strongly dependent on the no. of water ligands and the structure of the second coordination sphere; it provides a much more precise est. of the thermodn. of complex formation reactions in soln. than that obtained from Model B.  The agreement between the exptl. and calcd. data for the formation of Zn(NH3)2+(aq) and Zn(NH3)2+2(aq) is better than 8 kJ/mol for the former, as compared to 30 kJ/mol or larger, for the latter.  The Gibbs energy of reaction obtained for the UO2+2 oxalate systems using model B differs between 80 and 130 kJ/mol from the exptl. results, whereas the agreement with Model A is better.  The errors in the quantum chem. ests. of the entropy and enthalpy of reaction are somewhat larger than those for the Gibbs energy, but still in fair agreement with expts.; adding water mols. in the second coordination sphere improves the agreement significantly.  Reasons for the different performance of the two models are discussed.  The quantum chem. data were used to discuss the microscopic basis of exptl. enthalpy and entropy data, to det. the enthalpy and entropy contributions in chelate ring closure reactions and to discuss the origin of the so-called "chelate effect".  Contrary to many earlier suggestions, this is not even in the gas phase, a result of changes in translation entropy contributions.  There is no simple explanation of the high stability of chelate complexes; it is a result of both enthalpy and entropy contributions that vary from one system to the other.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfoLO7IER0drVg90H21EOLACvtfcHk0liLWSecaw7Fvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovVCltrg%253D&md5=6d4bb99794e6d1aea670c814f46e07a8</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fja036646j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja036646j%26sid%3Dliteratum%253Aachs%26aulast%3DVallet%26aufirst%3DV.%26aulast%3DWahlgren%26aufirst%3DU.%26aulast%3DGrenthe%26aufirst%3DI.%26atitle%3DChelate%2520Effect%2520and%2520Thermodynamics%2520of%2520Metal%2520Complex%2520Formation%2520in%2520Solution%253A%2520A%2520Quantum%2520Chemical%2520Study%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D14941%26epage%3D14950%26doi%3D10.1021%2Fja036646j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data Processing and Analysis with the autoPROC toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+Processing+and+Analysis+with+the+autoPROC+toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0liz-LwrXuBT-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520Processing%2520and%2520Analysis%2520with%2520the%2520autoPROC%2520toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Collaborative Computational Project, Number
4</span> <span> </span><span class="NLM_article-title">The CCP4 Suite: Programs
for Protein Crystallography</span>. <i>Acta Crystallogr.,
Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">760</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1107/S0907444994003112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1107%2FS0907444994003112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=15299374" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=1994&pages=760-763&author=Collaborative+Computational+Project%2C+Number%0A4&title=The+CCP4+Suite%3A+Programs%0Afor+Protein+Crystallography&doi=10.1107%2FS0907444994003112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444994003112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444994003112%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DThe%2520CCP4%2520Suite%253A%2520Programs%250Afor%2520Protein%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%250ASect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1994%26volume%3D50%26spage%3D760%26epage%3D763%26doi%3D10.1107%2FS0907444994003112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liz-LwrXuBT-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T.</span></span> <i>BUSTER</i>, version 2.11.5; <span class="NLM_publisher-name">Global Phasing Ltd.</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.+Smart&author=C.+Vonrhein&author=T.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G76" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2G76','PDB','2G76'); return false;">PDB: 2G76</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH','PDB','6RIH'); return false;">PDB: 6RIH</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA','PDB','6CWA'); return false;">PDB: 6CWA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3','PDB','6RJ3'); return false;">PDB: 6RJ3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5','PDB','6RJ5'); return false;">PDB: 6RJ5</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2','PDB','6RJ2'); return false;">PDB: 6RJ2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6','PDB','6RJ6'); return false;">PDB: 6RJ6</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i117"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00718">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_27553"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00718">10.1021/acs.jmedchem.9b00718</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Selectivity profile (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">Biochemical assays (HTS PHGDH inhibitor assay, diaphorase NADH counterscreen, 3-phosphoglycerate dehydrogenase (PHGDH) fluorescence intensity assay); cellular assays (<sup>13</sup>C serine assay in MDA-MB-468 cells, cellular uptake assay); (bio)physical methods (ICP-MS, surface plasmon resonance); NMR (high-resolution mass spectroscopy); protein expression, chemoproteomics, protein crystallization; safety (CEREP panel); chromatography; synthetic procedures; HPLC traces of selected compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_002.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_003.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_001.xlsx">jm9b00718_si_001.xlsx (258.24 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_002.pdf">jm9b00718_si_002.pdf (3.49 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00718/suppl_file/jm9b00718_si_003.csv">jm9b00718_si_003.csv (4.27 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes: compound <b>9</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RIH">6RIH</a>; compound <b>15</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ3">6RJ3</a>; compound <b>39</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ5">6RJ5</a>; compound <b>40</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ2">6RJ2</a>; <b>BI-4924</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6RJ6">6RJ6</a>; <b>3-PG</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6CWA">6CWA</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00718&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-17%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00718%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00718" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679919feda0c3dc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
