
<html lang="en"     class="pb-page"  data-request-id="6062f48e-6bc7-4101-9b24-2c12128906d6"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.6b01516;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2017.60.issue-1;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Design and Synthesis of a Series of l-trans-4-Substituted Prolines as Selective Antagonists for the Ionotropic Glutamate Receptors Including Functional and X-ray Crystallographic Studies of New Subtype Selective Kainic Acid Receptor Subtype 1 (GluK1) Antagonist (2S,4R)-4-(2-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid" /></meta><meta name="dc.Creator" content="Niels  Krogsgaard-Larsen" /></meta><meta name="dc.Creator" content="Claudia G.  Delgar" /></meta><meta name="dc.Creator" content="Karina  Koch" /></meta><meta name="dc.Creator" content="Patricia M. G. E.  Brown" /></meta><meta name="dc.Creator" content="Charlotte  Møller" /></meta><meta name="dc.Creator" content="Liwei  Han" /></meta><meta name="dc.Creator" content="Tri H. V.  Huynh" /></meta><meta name="dc.Creator" content="Stinne W.  Hansen" /></meta><meta name="dc.Creator" content="Birgitte  Nielsen" /></meta><meta name="dc.Creator" content="Derek  Bowie" /></meta><meta name="dc.Creator" content="Darryl S.  Pickering" /></meta><meta name="dc.Creator" content="Jette Sandholm  Kastrup" /></meta><meta name="dc.Creator" content="Karla  Frydenvang" /></meta><meta name="dc.Creator" content="Lennart  Bunch" /></meta><meta name="dc.Description" content="Ionotropic glutamate receptor antagonists are valuable tool compounds for studies of neurological pathways in the central nervous system. On the basis of rational ligand design, a new class of sele..." /></meta><meta name="Description" content="Ionotropic glutamate receptor antagonists are valuable tool compounds for studies of neurological pathways in the central nervous system. On the basis of rational ligand design, a new class of sele..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 22, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01516" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01516" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01516" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01516" /></link>
        
    
    

<title>Design and Synthesis of a Series of l-trans-4-Substituted Prolines as Selective Antagonists for the Ionotropic Glutamate Receptors Including Functional and X-ray Crystallographic Studies of New Subtype Selective Kainic Acid Receptor Subtype 1 (GluK1) Antagonist (2S,4R)-4-(2-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01516" /></meta><meta property="og:title" content="Design and Synthesis of a Series of l-trans-4-Substituted Prolines as Selective Antagonists for the Ionotropic Glutamate Receptors Including Functional and X-ray Crystallographic Studies of New Subtype Selective Kainic Acid Receptor Subtype 1 (GluK1) Antagonist (2S,4R)-4-(2-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0013.jpeg" /></meta><meta property="og:description" content="Ionotropic glutamate receptor antagonists are valuable tool compounds for studies of neurological pathways in the central nervous system. On the basis of rational ligand design, a new class of selective antagonists, represented by (2S,4R)-4-(2-carboxyphenoxy)pyrrolidine-2-carboxylic acid (1b), for cloned homomeric kainic acid receptors subtype 1 (GluK1) was attained (Ki = 4 μM). In a functional assay, 1b displayed full antagonist activity with IC50 = 6 ± 2 μM. A crystal structure was obtained of 1b when bound in the ligand binding domain of GluK1. A domain opening of 13–14° was seen compared to the structure with glutamate, consistent with 1b being an antagonist. A structure–activity relationship study showed that the chemical nature of the tethering atom (C, O, or S) linking the pyrrolidine ring and the phenyl ring plays a key role in the receptor selectivity profile and that substituents on the phenyl ring are well accommodated by the GluK1 receptor." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01516"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01516">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01516&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01516&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01516&amp;href=/doi/10.1021/acs.jmedchem.6b01516" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 441-457</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01482" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01539" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design and Synthesis of a Series of <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-Substituted Prolines as Selective Antagonists for the Ionotropic Glutamate Receptors Including Functional and X-ray Crystallographic Studies of New Subtype Selective Kainic Acid Receptor Subtype 1 (GluK1) Antagonist (2<i>S</i>,4<i>R</i>)-4-(2-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Niels++Krogsgaard-Larsen">Niels Krogsgaard-Larsen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Claudia+G.++Delgar">Claudia G. Delgar</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Karina++Koch">Karina Koch</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Patricia+M.+G.+E.++Brown">Patricia M. G. E. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Charlotte++M%C3%B8ller">Charlotte Møller</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Liwei++Han">Liwei Han</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tri+H.+V.++Huynh">Tri H. V. Huynh</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stinne+W.++Hansen">Stinne W. Hansen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Birgitte++Nielsen">Birgitte Nielsen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Derek++Bowie">Derek Bowie</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Darryl+S.++Pickering">Darryl S. Pickering</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jette+Sandholm++Kastrup">Jette Sandholm Kastrup</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Karla++Frydenvang">Karla Frydenvang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lennart++Bunch">Lennart Bunch</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-0180-4639" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Bunch Research Group, <sup>‡</sup>Biostructural Research Group, <sup>§</sup>Medicinal Chemistry Group, <sup>∥</sup>Molecular and Cellular Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">#</span> <span class="aff-text">Bowie Lab, Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div><div class="corresp-info"><strong>*</strong>For L.B.: phone, +45 35336244; E-mail, <a href="/cdn-cgi/l/email-protection#09656c6b7c497a7c676d27627c276d62"><span class="__cf_email__" data-cfemail="bdd1d8dfc8fdcec8d3d993d6c893d9d6">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>For K.F., correspondence regarding the biostructural work: phone, +45 35336207; E-mail, <a href="/cdn-cgi/l/email-protection#3e555f4c525f10584c475a5b50485f50597e4d4b505a10554b105a55"><span class="__cf_email__" data-cfemail="9ff4feedf3feb1f9ede6fbfaf1e9fef1f8dfeceaf1fbb1f4eab1fbf4">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01516&amp;href=/doi/10.1021%2Facs.jmedchem.6b01516" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 441–457</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 22, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 October 2016</li><li><span class="item_label"><b>Published</b> online</span>22 December 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 January 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01516" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01516</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D441%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DNiels%2BKrogsgaard-Larsen%252C%2BClaudia%2BG.%2BDelgar%252C%2BKarina%2BKoch%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D1%26contentID%3Dacs.jmedchem.6b01516%26title%3DDesign%2Band%2BSynthesis%2Bof%2Ba%2BSeries%2Bof%2Bl-trans-4-Substituted%2BProlines%2Bas%2BSelective%2BAntagonists%2Bfor%2Bthe%2BIonotropic%2BGlutamate%2BReceptors%2BIncluding%2BFunctional%2Band%2BX-ray%2BCrystallographic%2BStudies%2Bof%2BNew%2BSubtype%2BSelective%2BKainic%2BAcid%2BReceptor%2BSubtype%2B1%2B%2528GluK1%2529%2BAntagonist%2B%25282S%252C4R%2529-4-%25282-Carboxyphenoxy%2529pyrrolidine-2-carboxylic%2BAcid%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D457%26publicationDate%3DJanuary%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01516"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1103</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01516" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design and Synthesis of a Series of l-trans-4-Substituted Prolines as Selective Antagonists for the Ionotropic Glutamate Receptors Including Functional and X-ray Crystallographic Studies of New Subtype Selective Kainic Acid Receptor Subtype 1 (GluK1) Antagonist (2S,4R)-4-(2-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Niels&quot;,&quot;last_name&quot;:&quot;Krogsgaard-Larsen&quot;},{&quot;first_name&quot;:&quot;Claudia&quot;,&quot;last_name&quot;:&quot;G. Delgar&quot;},{&quot;first_name&quot;:&quot;Karina&quot;,&quot;last_name&quot;:&quot;Koch&quot;},{&quot;first_name&quot;:&quot;Patricia&quot;,&quot;last_name&quot;:&quot;M. G. E. Brown&quot;},{&quot;first_name&quot;:&quot;Charlotte&quot;,&quot;last_name&quot;:&quot;Møller&quot;},{&quot;first_name&quot;:&quot;Liwei&quot;,&quot;last_name&quot;:&quot;Han&quot;},{&quot;first_name&quot;:&quot;Tri&quot;,&quot;last_name&quot;:&quot;H. V. Huynh&quot;},{&quot;first_name&quot;:&quot;Stinne&quot;,&quot;last_name&quot;:&quot;W. Hansen&quot;},{&quot;first_name&quot;:&quot;Birgitte&quot;,&quot;last_name&quot;:&quot;Nielsen&quot;},{&quot;first_name&quot;:&quot;Derek&quot;,&quot;last_name&quot;:&quot;Bowie&quot;},{&quot;first_name&quot;:&quot;Darryl&quot;,&quot;last_name&quot;:&quot;S. Pickering&quot;},{&quot;first_name&quot;:&quot;Jette&quot;,&quot;last_name&quot;:&quot;Sandholm Kastrup&quot;},{&quot;first_name&quot;:&quot;Karla&quot;,&quot;last_name&quot;:&quot;Frydenvang&quot;},{&quot;first_name&quot;:&quot;Lennart&quot;,&quot;last_name&quot;:&quot;Bunch&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;441-457&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01516&quot;},&quot;abstract&quot;:&quot;Ionotropic glutamate receptor antagonists are valuable tool compounds for studies of neurological pathways in the central nervous system. On the basis of rational ligand design, a new class of selective antagonists, represented by (2S,4R)-4-(2-carboxyphenoxy)pyrrolidine-2-carboxylic acid (1b), for cloned homomeric kainic acid receptors subtype 1 (GluK1) was attained (Ki = 4 μM). In a functional assay, 1b displayed full antagonist activity with IC50 = 6 ± 2 μM. A crystal structure was obtained of 1b when bound in the ligand binding domain of GluK1. A domain opening of 13–14° was seen compared to the structure with glutamate, consistent with 1b being an antagonist. A structure–activity relationship study showed that the chemical nature of the tethering atom (C, O, or S) linking the pyrrolidine ring and the phenyl ring plays a key role in the receptor selectivity profile and that substituents on the phenyl ring are well accommodated by the GluK1 receptor.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01516&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01516" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01516&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01516" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01516&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01516" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01516&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01516&amp;href=/doi/10.1021/acs.jmedchem.6b01516" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01516" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01516" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01516%26sid%3Dliteratum%253Aachs%26pmid%3D28005385%26genre%3Darticle%26aulast%3DKrogsgaard-Larsen%26date%3D2017%26atitle%3DDesign%2Band%2BSynthesis%2Bof%2Ba%2BSeries%2Bof%2Bl-trans-4-Substituted%2BProlines%2Bas%2BSelective%2BAntagonists%2Bfor%2Bthe%2BIonotropic%2BGlutamate%2BReceptors%2BIncluding%2BFunctional%2Band%2BX-ray%2BCrystallographic%2BStudies%2Bof%2BNew%2BSubtype%2BSelective%2BKainic%2BAcid%2BReceptor%2BSubtype%2B1%2B%2528GluK1%2529%2BAntagonist%2B%25282S%252C4R%2529-4-%25282-Carboxyphenoxy%2529pyrrolidine-2-carboxylic%2BAcid%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D1%26spage%3D441%26epage%3D457%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/jmcmar.2017.60.issue-1/20170112/jmcmar.2017.60.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Ionotropic glutamate receptor antagonists are valuable tool compounds for studies of neurological pathways in the central nervous system. On the basis of rational ligand design, a new class of selective antagonists, represented by (2<i>S</i>,4<i>R</i>)-4-(2-carboxyphenoxy)pyrrolidine-2-carboxylic acid (<b>1b</b>), for cloned homomeric kainic acid receptors subtype 1 (GluK1) was attained (<i>K</i><sub>i</sub> = 4 μM). In a functional assay, <b>1b</b> displayed full antagonist activity with IC<sub>50</sub> = 6 ± 2 μM. A crystal structure was obtained of <b>1b</b> when bound in the ligand binding domain of GluK1. A domain opening of 13–14° was seen compared to the structure with glutamate, consistent with <b>1b</b> being an antagonist. A structure–activity relationship study showed that the chemical nature of the tethering atom (C, O, or S) linking the pyrrolidine ring and the phenyl ring plays a key role in the receptor selectivity profile and that substituents on the phenyl ring are well accommodated by the GluK1 receptor.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47111" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47111" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">(<i>S</i>)-Glutamic acid (Glu, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) functions as the major excitatory neurotransmitter in the mammalian central nervous system (CNS). The Glu receptors are divided into two major classes based on of their structural nature: the ionotropic Glu receptors (iGluRs)<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and the G-protein coupled metabotropic Glu receptors (mGluRs).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Lastly, uptake of Glu from the synapse is mediated by the excitatory amino acid transporters (EAATs).<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0001.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of Glu and iGluR group-defining agonists AMPA, KA, and NMDA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The Glu receptors are involved in basic neurophysiological processes such as motor function, cognition, and plasticity. This, however, is a delicate balance as excessive Glu signaling has been shown to be neurotoxic,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a> ultimately leading to neuronal death. Several psychiatric diseases and neurological disorders have been proposed to have an origin in a malfunctioning Glu neurotransmitter system(s) which are therefore considered as promising therapeutic targets. Diseases span over depression,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> anxiety,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and schizophrenia,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> while neurological disorders comprise neurodegenerative diseases such as Alzheimer’s,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Parkinson’s,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and epilepsy.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The iGluRs are divided into three groups (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>): the α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptors (subunits GluA1–4), the kainic acid (KA) receptors (subunits GluK1–5), and the <i>N</i>-methyl-<span class="smallcaps smallerCapital">d</span>-aspartic acid (NMDA) receptors (subunits GluN1, GluN2A-D, and GluN3A,B).</div><div class="NLM_p">Upon extending the distance between the α-amino acid moiety and the distal γ-carboxylic acid group of the endogenous agonist Glu, the ligand binding domain (LBD) is prevented from closing, leading to antagonist function.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> This design strategy has led to a number of chemically distinct parental scaffolds (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>): 3′-functionalized 2-amino-3-(4′-isoxazolyl)propionic acids (e.g., 2-amino-3-[5-<i>tert</i>-butyl-3-(phosphonomethoxy)-1,2-oxazol-4-yl]propanoic acid (ATPO)), amino acid bioisosteric quinoxalinediones (e.g., 6,7-dinitroquinoxaline-2,3-dione (DNQX) and 7-nitro-2,3-dioxo-1,4-dihydroquinoxaline-6-carbonitrile (CNQX)),<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> 6-functionalized decahydroisoquinoline-3-carboxylic acids (<b>I</b>),<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16, 17)</a> substituted phenylalanines (<b>II</b>),<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a><i>N</i>-functionalized analogues of the natural product willardiine (e.g., α-amino-3-[(4-carboxyphenyl)methyl]-3,4-dihydro-2,4-dioxo-1(2<i>H</i>)-pyrimidinepropanoic acid (<b>III</b>)),<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and recently 3-(3-carboxyphenyl)pyrrolidine-2-carboxylic acids (<b>IV</b>)<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a><a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). By SAR studies of these parental scaffolds, group-selective iGluR antagonists have been disclosed, but only subtype selective antagonists for the KA receptor subtype GluK1 has been reported: (3<i>S</i>,4a<i>R</i>,6<i>S</i>,8a<i>R</i>)-6-(((<i>S</i>)-2-carboxy-4,4-difluoropyrrolidin-1-yl)methyl)decahydroisoquinoline-3-carboxylic acid (LY466195)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and (<i>S</i>)-3-((3-(2-amino-2-carboxyethyl)-5-methyl-2,6-dioxo-3,6-dihydropyrimidin-1(2<i>H</i>)-yl)methyl)thiophene-2-carboxylic acid (UBP310)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>). In our search for novel parental scaffolds for subsequent discovery of subtype selective iGluR antagonists, we here present our work on a successful scaffold-jump of <b>IV</b>.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0002.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of iGluR antagonists and subtype selective GluK1 antagonists.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10833" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10833" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Inspired by our previous work on the <i>trans</i>-3-(3-carboxyphenyl)proline scaffold <b>IV</b>,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20, 21)</a> we were intrigued by the thought of repositioning the 3-phenyl ring to the <i>trans</i>-4-position. This affords compound <b>V</b> (purple), which is linear in shape with respect to the α-amino acid and the distal carboxylic acid functionalities (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). In silico energy minimization of <b>V</b> followed by superimposition onto the binding conformation of <b>IVa</b> in GluA2 (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yma">4yma</a>)<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> by aligning the two α-amino acid functionalities shows that the two distal carboxylic acid functionalities align poorly and are perpendicular to each other (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B). For compound <b>V</b> to attain a more convex shape as <b>IVa</b>, a sp<sup>3</sup>-tethering carbon atom between the pyrrolidine ring and the phenyl group was introduced (compound <b>1a</b>). Superimposition of a low-energy conformation of <b>1a</b> with the X-ray diffraction determined binding conformation of <b>IVa</b> (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yma">4yma</a>) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B) confirms that <b>1a</b> has a convex shape similar to <b>IVa</b>, placing its phenyl ring in parallel to the phenyl ring of <b>IVa</b>.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0003.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Rational design of 4-substituted proline <b>1a</b> starting from 3-substituted proline <b>IV</b> (e.g., the 5-hydroxy analogue <b>IVa</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>). (B) Superimposition of binding mode of <b>IVa</b> in GluA2-LBD (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yma">4yma</a>) (green), <b>V</b> (purple), and <b>1a</b> (gray) by alignment of the α-amino acid moieties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The synthesis of <b>1a</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) commenced with benzylation of readily available protected (<i>S</i>)-pyroglutaminol <b>2</b> to give <b>3</b> in acceptable yield as the single <i>trans</i>-diastereomer. Reduction of the <i>endo</i>-cyclic carbonyl group with borane provided the expected pyrrolidine ring but also the benzylic alcohol from reduction of the arylic ester (compound <b>4</b>). Cleavage of the silyl ether with TBAF gave diol <b>5</b>, which was oxidized to the diacid <b>6</b> using TEMPO. Removal of the BOC group was done by HCl(g)/AcOH in diethyl ether to provide target compound <b>1a</b> as the HCl salt.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0006.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Stereo-Controlled Synthesis of <b>1a</b> Starting from Readily Available Protected (<i>S</i>)-Pyroglutaminol <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) LHMDS, THF, −78 °C to rt (64%); (b) BH<sub>3</sub>, THF, reflux (56%); (c) TBAF, THF, rt (52%); (d) TEMPO, NaClO<sub>2</sub>, NaOCl, MeCN, H<sub>2</sub>O, 0 °C (62%); (e) HCl(g)/AcOH in Et<sub>2</sub>O rt (99%).</p></p></figure><div class="NLM_p">The binding affinity of <b>1a</b> toward the plethora of ionotropic Glu receptors was evaluated at native AMPA, KA, and NMDA receptors (rat synaptosomes) and cloned rat homomeric subtypes GluK1–3 (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). <b>1a</b> displayed negligible affinity for AMPA and KA receptors but midrange micromolar affinity for the NMDA receptors (<i>K</i><sub>i</sub> = 69 μM). At cloned homomeric GluK1–3 receptors, <b>1a</b> was also without appreciable affinity. The observed selectivity for the NMDA receptors prompted us to explore the influence of the chemical nature of the tethering carbon atom on iGluRs group and subtype binding affinity.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Furthermore, we decided to investigate the position of the carboxylate group on the phenyl ring and challenge the <i>trans</i>-stereochemistry by synthesizing selected <i>cis</i>-stereoisomers. In all, syntheses of three <i>trans</i>-ether analogues <b>1b</b>–<b>d</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), two <i>trans</i>-thioether analogues <b>1e</b>,<b>f</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>), three <i>cis</i>-ethers <b>1g</b>–<b>i</b> (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>), and two <i>cis</i>-thioethers <b>1j</b>,<b>k</b> (<a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>) were carried out.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0007.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Stereo-Controlled Synthesis of <b>1b</b>–<b>d</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>cis</i>-4-Hydroxy-proline <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) appropriately substituted phenol, DIAD, PPh<sub>3</sub>, THF, rt (53–91%); (b) LiOH, THF, 0 °C to rt (53–91%, two steps; (c) <b>1b</b>,<b>d</b>:TFA in DCM, then 1 M HCl, <b>1c</b>:HCl/AcOH in Et<sub>2</sub>O (89–96%).</p></p></figure><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0008.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Stereo-Controlled Synthesis of <b>1e</b>,<b>f</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>cis</i>-4-Hydroxy-proline <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MsCl, TEA, THF, 0 °C (98%); (b) appropriately substituted thiophenol, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (c) LiOH, THF, 0 °C to rt (78% for <b>12</b>); (d) TFA in DCM (94% and 40%, <b>1e</b> and <b>1f</b>, respectively).</p></p></figure><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0009.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Stereo-Controlled Synthesis of <b>1g</b>–<b>i</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-Hydroxyproline <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Appropriately substituted phenol, DIAD, PPh<sub>3</sub>, THF, rt; (b) LiOH, THF, 0 °C to rt (53–91%, two steps); (c) TFA in DCM, then 1 M HCl or HCl/AcOH in Et<sub>2</sub>O (85–93%).</p></p></figure><figure id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0010.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Stereo-Controlled Synthesis of <b>1j</b>,<b>k</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-Hydroxy-proline <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MsCl, TEA, THF, 0 °C (90%); (b) Appropriately substituted thiophenol, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (c) LiOH, THF, 0 °C to rt (67%, two steps for <b>19</b>); (d) TFA in DCM (74% for <b>1j</b> and 38% for <b>1k</b>, three steps).</p></p></figure><div class="NLM_p">All 10 analogues were synthesized by an efficient strategy starting from commercially available suitably protected (<i>S</i>)-<i>cis</i>-4-hydroxy-proline (<b>7</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref internalNav" href="#sch3" aria-label="3">3</a>) or (<i>S</i>)-<i>trans</i>-4-hydroxy-proline (<b>14</b>) (<a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>). Generally, the <i>cis</i>/<i>trans</i>-ether analogues were obtained directly by a Mitsunobu protocol (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> and <a class="ref internalNav" href="#sch4" aria-label="4">4</a>), while the <i>cis</i>/<i>trans</i>-thioethers were obtained by a two-step procedure with mesylates <b>11</b> and <b>18</b> as intermediates (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a> and <a class="ref internalNav" href="#sch5" aria-label="5">5</a>).</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Binding Affinity of <b>1b</b>–<b>k</b></h3><div class="NLM_p">Pharmacological characterization of <b>1b</b>–<b>k</b> at native ionotropic Glu receptors (rat synaptosomes) and rat homomeric cloned GluK1–3 receptors revealed both <i>expected</i> as well as <i>unexpected</i> binding affinity profiles (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). None of the compounds showed appreciable affinity for AMPA receptors. By substituting the carbon tether for an oxygen atom (compound <b>1c</b>), an unexpected shift in pharmacological profile from selective midrange micromolar affinity for the NMDA receptors to selective midrange micromolar affinity for GluK1 was observed. However, even more interestingly, repositioning the arylic carboxylic acid group to the <i>ortho</i>-position increased the affinity by 9-fold (compound <b>1b</b>). In contrast, when the carboxylic acid group was placed in the <i>para</i>-position, compound <b>1d</b>, the pharmacological profile was shifted back to selectivity for the NMDA receptors.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Affinities of 4-Substituted Prolines at Native iGluRs (Rat Synaptosomes) and Homomeric Cloned Rat GluK1–3 Receptors<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0011.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">2,4 stereochemistry</th><th class="colsep0 rowsep0" align="center">-X-</th><th class="colsep0 rowsep0" align="center">COOH</th><th class="colsep0 rowsep0" align="center" char=".">AMPA <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">KA <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">NMDA <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">GluK1 <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">GluK2 <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">GluK3 <i>K</i><sub>i</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1a</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i></td><td class="colsep0 rowsep0" align="left">CH<sub>2</sub></td><td class="colsep0 rowsep0" align="left"><i>meta</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">69 [4.18 ± 0.05]</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1b</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left"><i>ortho</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">4.0 [5.40 ± 0.03]</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1c</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left"><i>meta</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">36 [4.48 ± 0.12]</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>1000</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1d</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left"><i>para</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">81 [4.11 ± 0.09]</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1e</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i></td><td class="colsep0 rowsep0" align="left">S</td><td class="colsep0 rowsep0" align="left"><i>ortho</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">19 [4.73 ± 0.05]</td><td class="colsep0 rowsep0" align="center">8.4 [5.10 ± 0.11]</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">22 [4.66 ± 0.02]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1f</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i></td><td class="colsep0 rowsep0" align="left">S</td><td class="colsep0 rowsep0" align="left"><i>meta</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">78 [4.11 ± 0.03]</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1g</b></td><td class="colsep0 rowsep0" align="left"><i>cis</i></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left"><i>meta</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1h</b></td><td class="colsep0 rowsep0" align="left"><i>cis</i></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left"><i>ortho</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1i</b></td><td class="colsep0 rowsep0" align="left"><i>cis</i></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left"><i>para</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1j</b></td><td class="colsep0 rowsep0" align="left"><i>cis</i></td><td class="colsep0 rowsep0" align="left">S</td><td class="colsep0 rowsep0" align="left"><i>ortho</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1k</b></td><td class="colsep0 rowsep0" align="left"><i>cis</i></td><td class="colsep0 rowsep0" align="left">S</td><td class="colsep0 rowsep0" align="left"><i>meta</i></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LY466195<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">0.05</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="center">8.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">UBP310<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="center">0.022</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">0.93</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Radioligands: AMPA, [<sup>3</sup>H]AMPA; KA, [<sup>3</sup>H]KA; NMDA, [<sup>3</sup>H]CGP-39653; GluK1, [<sup>3</sup>H]NF608; GluK2 and GluK3, [<sup>3</sup>H]KA. For all, p<i>K</i><sub>i</sub> ± SEM (M) in square brackets.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">From ref <a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">22</a>.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">From ref <a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">21</a>.</p></div></div></div><div class="NLM_p last">The thioether analogue of <b>1b</b>, compound <b>1e</b>, was no longer selective for GluK1 but displayed midrange micromolar affinity for native NMDA receptors, GluK1, and GluK3 receptors (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). This was unexpected and underlines how subtle chemical changes to a ligand may modify its pharmacological profile. Thioether <b>1f</b> displays midrange micromolar affinity for native NMDA receptors which is in contrast to its oxygen analogue <b>1c</b>, showing highest binding affinity for homomeric GluK1 receptors. Finally, all the synthesized <i>cis</i>-stereoisomers <b>1g</b>–<b>k</b> displayed negligible affinity for any of the iGluRs. The four analogues <b>1b</b>–<b>d</b>,<b>g</b> were characterized in a conventional [<sup>3</sup>H]-<span class="smallcaps smallerCapital">d</span>-Asp uptake assay and displayed no inhibitory activity at EAAT1–3 (IC<sub>50</sub> > 1000 μM).<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Functional Characterization of <b>1b</b> at Recombinant Homomeric GluK1 and GluK2 Receptors</h3><div class="NLM_p">Given the GluK1-selective binding affinity profile of <b>1b</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), we next characterized its function at recombinantly expressed KA receptors. Outside-out patches containing either GluK1 or GluK2 homomers were subjected to fast Glu applications (250 ms, 1 mM). An inhibition curve was constructed by subjecting the patches to various concentrations of <b>1b</b> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). As expected from our previous results, GluK2 responses were unaffected by concentrations of <b>1b</b> up to 100 μM, while GluK1 responses were almost completely blocked at this concentration (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). The IC<sub>50</sub> was determined to 5.7 ± 2.0 μM for GluK1 receptors and >100 μM for GluK2 receptors (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C).</div><figure id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0004.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>1b</b> inhibits recombinant GluK1 but not GluK2 in HEK293T cells. (A) Example traces of GluK1 responses evoked by 1 mM Glu (250 ms) in outside-out patches at a holding potential of −100 mV, in the absence (left) or presence (right, gray) of 100 μM <b>1b</b>. (B) Example traces of GluK2 responses evoked by 1 mM Glu (250 ms) in outside-out patches at a holding potential of −100 mV, in the absence (left) or presence (right, light gray) of 100 μM <b>1b</b>. (C) Inhibition plot for <b>1b</b> for 1 mM Glu. Data are represented as a percentage of the control response in the absence of <b>1b</b> (mean ± SEM); 3–5 responses were obtained for each concentration of <b>1b</b> tested.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> X-ray Structure of <b>1b</b> in GluK1 LBD</h3><div class="NLM_p">As <b>1b</b> turned out to be a selective GluK1 antagonist, we crystallized <b>1b</b> with a soluble construct of the GluK1 LBD. The binding affinity (<i>K</i><sub>i</sub> = 2.27 ± 0.12 μM) at GluK1 LBD was found to be similar to that of full-length GluK1 (4.0 μM; <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). The complex crystallizes with two molecules in the asymmetric unit of the crystal, forming a dimer (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A, <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Despite the low resolution of the structure (3.2 Å), the electron density was well-defined for most of the amino acid residues, and <b>1b</b> could be modeled into the density in both binding sites (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). After molecular replacement (only protein atoms included), the electron density revealed the position of <b>1b</b>. However, the conformation of the pyrrolidine ring could not be determined unambiguously at this resolution. <b>1b</b> was inserted in a low energy conformation, allowing a hydrogen bond to be formed between the pyrrolidine nitrogen of <b>1b</b> to the side chain carboxylate of Glu738 (GluK1–2 numbering including signal peptide). Unfortunately, it has not been possible to improve the resolution beyond 3 Å.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Crystal Data, Data Collection, And Refinement Statistics of GluK1 LBD in Complex with <b>1b</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">Crystal Data</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">beamline</td><td class="colsep0 rowsep0" align="left">I911–3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>4<sub>1</sub>2<sub>1</sub>2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unit cell dimensions (Å)</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>a</i> = <i>b</i> = 71.99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>c</i> = 233.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">molecules in a.u.<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">Data collection and Processing</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution</td><td class="colsep0 rowsep0" align="left">29.75–3.18 (3.29–3.18)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">unique reflections</td><td class="colsep0 rowsep0" align="left">10993</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average multiplicity</td><td class="colsep0 rowsep0" align="left">6.5 (6.7)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.8 (100)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wilson <i>B</i>-factor (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">56</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub> (%)</td><td class="colsep0 rowsep0" align="left">14.0 (40.6)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ<i>I</i></td><td class="colsep0 rowsep0" align="left">4.4 (1.9)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="2" align="center">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">number of:</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">amino acid residues</td><td class="colsep0 rowsep0" align="left">504</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1b</b> molecules</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfate ions</td><td class="colsep0 rowsep0" align="left">6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>work</sub><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a> (%)/<i>R</i><sub>free</sub><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a> (%)</td><td class="colsep0 rowsep0" align="left">21.9/27.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>B</i>-values (Å<sup>2</sup>) for:</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">amino acid residues/<b>1b</b></td><td class="colsep0 rowsep0" align="left">50/42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">sulfate ions</td><td class="colsep0 rowsep0" align="left">106</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">RMS deviation:</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond length (Å)/angles (deg)</td><td class="colsep0 rowsep0" align="left">0.002/0.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">residues in allowed regions of Ramachandran plot (%)<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">100.0</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">a.u.: Asymmetric unit of the crystal.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Values in parentheses correspond to the outermost resolution shell.</p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last"><i>R</i><sub>work</sub> = ∑<sub>hkl</sub>(∥<i>F</i><sub>o,hkl</sub>| – |<i>F</i><sub>c,hkl</sub>∥)/|<i>F</i><sub>o,hkl</sub>|, where |<i>F</i><sub>o,hkl</sub>| and |<i>F</i><sub>c,hkl</sub>| are the observed and calcdulated structure factor amplitudes, respectively.</p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last"><i>R</i><sub>free</sub> is equivalent to <i>R</i><sub>work</sub>, but calcdulated with 5% of the reflections omitted from the refinement process.</p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">The Ramachandran plot was calcdulated according to MolProbity.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></p></div></div></div><figure id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0005.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of GluK1 LBD with the antagonist <b>1b</b> bound. (A) GluK1 LBD (molA, cyan; molB, pale cyan) in complex with <b>1b</b> (orange) crystallizes as a dimer. (B) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density for <b>1b</b> after molecular replacement (black), at which point the ligand was not introduced into the model, as well as the final electron density (white). Both maps are contoured at 1 σ. (C) Binding site interactions with <b>1b</b>. Hydrogen bonds are shown as stippled black lines (up to 3.4 Å). Residues forming the pocket (within 4 Å) for the aromatic ring of <b>1b</b> are shown. (D) Comparison of the binding mode of <b>1b</b> and kainate (black; pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e0x">4e0x</a>, gray) in GluK1 LBD. GluK1 structures were superimposed on residues of D1. Interactions of the ligands with Thr690 are shown as well as the D1–D2 interlobe hydrogen bond. The tube illustrates the different domain openings through movement of D2 residues. Hydrogen bonds are colored cyan for <b>1b</b> and black for kainate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The amino acid part of <b>1b</b> interacts with binding site residues in the same way as previously observed for agonists and antagonists.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> The pyrrolidine carboxylate group of <b>1b</b> makes salt bridges with the side chain of the essential Arg523 and furthermore hydrogen bonds to the main chain nitrogen of Thr518 and the side chain hydroxyl group of Ser689 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C). The pyrrolidine nitrogen forms salt bridges with the side chain carboxylate of Glu738 as well as a hydrogen bond to the backbone oxygen of Pro516 and to the side chain hydroxyl group of Thr518. The distal carboxylate group of <b>1b</b> interacts with Thr690 through a hydrogen bond to the side chain hydroxyl group and to the backbone nitrogen of Glu738. The aromatic ring is placed in a binding pocket surrounded within 4 Å by Ser721, Met737, Glu738, and Ser741 from D2 as well as Glu441 from D1 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>C).</div><div class="NLM_p">When an antagonist binds to the LBD, it stabilizes the LBD in an open conformation<a onclick="showRef(event, 'ref14 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref27">(14, 27)</a> whereas an agonist stabilizes a closed conformation of the LBD. We have calculated the lobes D1–D2 opening of <b>1b</b> relative to the structure of GluK1 LBD with glutamate (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f36">2f36</a>, molA), in which the LBD is in a fully closed conformation. The domain opening was found to be 14.2° for molA and 12.8° for molB, which are both in the lower end of observed domain openings induced by antagonists when crystallized with GluK1 LBD (10–49°).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> This suggested that it might be possible to introduce even larger substituents into the <b>1b</b> scaffold. Interestingly, the domain opening observed for the partial agonist kainate is 8.4° (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e0x">4e0x</a>, molA), which leaves the question open as to where the exact cutoff is for domain opening of GluK1 antagonism vs partial agonism.</div><div class="NLM_p">Comparing the binding modes of <b>1b</b> and kainate in GluK1 LBD reveals that the aromatic ring of <b>1b</b> is positioned in the binding site between the isopropenyl group and the distal carboxylate of kainate (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). Because of the larger aromatic moiety in <b>1b</b>, the LBD is stabilized in the more open conformation, illustrated by difference in the positions of residues in lobe D2 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>D). This change in domain opening makes it possible for the distal carboxylate of both kainate and <b>1b</b> to interact with the same D2 residue, Thr690. Furthermore, the isopropenyl group in kainate and the aromatic ring in <b>1b</b> both interfere with the important interlobe hydrogen bond (between Glu441 in D1 and Ser721 in D2). The interaction is longer than observed for full agonists, e.g., 3.9 Å for <b>1b</b> versus 3.0 Å in the structure with glutamate bound (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f36">2f36</a>, molA).</div><div class="NLM_p">Compound <b>1b</b> possesses a 9-fold higher affinity for GluK1 compared with compound <b>1c</b> (carboxylate group in <i>ortho</i> vs <i>meta</i>-position, respectively) and complete loss of binding affinity (compound <b>1d</b>, <i>K</i><sub>i</sub> > 100 μM) when placed in the <i>para</i>-position. A structural analysis thereof suggests that for the <i>meta</i>-position, the carboxylic acid would be located at a hydrophobic site (Leu735, Met737, and Val685). A conformational change in the side chain of Met737 would be required to accommodate the carboxylic acid, which might contribute to the lower affinity of <b>1c</b>. The carboxylic acid moiety in the <i>para</i>-position in <b>1d</b> would lead to clash with Ser721 and Met737, consistent with the observed loss of affinity.</div><div class="NLM_p last">To address the selectivity of <b>1b</b> toward GluK1 over GluK2,3, we compared the amino residues of difference among the orthosteric binding sites. It seems that the most likely explanation for the loss of affinity at GluK2 and GluK3 is the presence of the larger Asn residue in GluK2 and GluK3 compared to Ser721 in GluK1, which may result in unfavorable steric contacts to the benzene moiety of <b>1b</b>.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Design, Synthesis, and Pharmacological Evaluation of <b>1l</b>–<b>t</b></h3><div class="NLM_p">With this new class of selective GluK1 antagonists in hand, we set to explore the SAR of <b>1b</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). We were curious to explore the insertion of a methylene group between the aryl ring and the carboxylic acid functionality (<b>1l</b>) for enhancement of ligand flexibility and thereby possibility for hydrogen bonding interactions with the receptor and to reposition this substituent to the <i>meta</i>-position (<b>1m</b>), as an analogue of <b>1c</b>. In analogues <b>1n</b>–<b>p</b>, the phenyl ring was replaced with a naphthyl group to increase lipophilicity and thereby possibly enhance binding affinity. Alternatively, incorporation of a 4-carboxylic acid group (<b>1q</b>), a 4-methyl group (<b>1r</b>), a 4-acetyl group (<b>1s</b>), and a 6-methyl group (<b>1t</b>) were planned.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Affinities of <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-Substituted Prolines <b>1l</b>–<b>t</b> at Native iGluRs and Homomeric Cloned GluK1–3 Receptors<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0012.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">AMPA <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">KA <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">NMDA <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">GluA2 <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">GluK1 <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center" char=".">GluK2 <i>K</i><sub>i</sub> (μM)</th><th class="colsep0 rowsep0" align="center">GluK3 <i>K</i><sub>i</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1l</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">24 [4.62 ± 0.01]</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1m</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1n</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">3.5 [5.455 ± 0.005]</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1o</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1p</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">4.5 [5.35 ± 0.08]</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">137 [3.88 ± 0.07]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1q</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">≈100</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1r</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">24 [4.63 ± 0.06]</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1s</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">4.7 [5.33 ± 0.03]</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">30 [4.53 ± 0.06]</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1t</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="center">5.4 [5.27 ± 0.03]</td><td class="colsep0 rowsep0" align="char" char=".">>1000</td><td class="colsep0 rowsep0" align="center">>100</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Radioligands: AMPA, [<sup>3</sup>H]AMPA; KA, [<sup>3</sup>H]KA; NMDA, [<sup>3</sup>H]CGP-39653; GluK1, [<sup>3</sup>H]NF608; GluK2 and GluK3, [<sup>3</sup>H]KA. For all, p<i>K</i><sub>i</sub> ± SEM (M) in square brackets.</p></div></div></div><div class="NLM_p">The synthesis of analogues <b>1l</b>-<b>t</b> followed the strategy outlined for <b>1b</b>–<b>d</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>). Starting from <i>cis</i>-alcohol <b>7</b>, the appropriate phenolic esters <b>21</b>–<b>28</b> were reacted to give <i>trans</i>-ethers <b>29</b>–<b>37</b> in good to high yields and stereochemically pure. Subsequent global deprotection by reflux in 4 M HCl gave target structures <b>1l</b>–<b>t</b>. All analogues were characterized pharmacologically in binding assays at native iGluRs and homomeric GluA2 and GluK1–3 receptor subtypes (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Insertion of a methylene group between the aryl ring and the carboxylic acid <b>1l</b> results in a 5-fold decrease in GluK1 affinity but maintaining the subtype selectivity profile. Reposition of the very same group to the <i>meta</i>-position, compound <b>1m</b>, leads to complete loss of binding affinity for GluK1 and any of the iGluRs. Of the three naphthyl analogues <b>1n</b>–<b>p</b>, both <b>1n</b> and <b>1p</b> maintain low micromolar affinity for the GluK1 receptor. The flexible nature of the side chains of Ser741 and Met737, respectively, allow for the additional bulkiness of the naphthyl group. On the other hand, <b>1o</b> is without affinity for the GluK1 receptor, which can be ascribed to a clear steric clash with residues Tyr489 and Glu441.</div><div class="NLM_p last">The 4-position was probed with chemically diverse substituents (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>): A carboxylic acid (<b>1q</b>) led to complete loss of binding affinity analogous to <b>1d</b>, while a 4-methyl group (<b>1r</b>) led to a 6-fold reduction in binding affinity. Only a 4-acetyl group (<b>1s</b>) showed the same affinity as the lead structure (<b>1b</b>), consistent with the acetyl group being positioned in a partly hydrophobic pocket (Val685, Leu735, and Met737). Finally, a 6-methyl group (<b>1t</b>) was well accommodated by the GluK1 receptor and showed similar binding affinity compared with <b>1b</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40158" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40158" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In conclusion, the <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-substituted proline scaffold was disclosed as a new scaffold acting as iGluR antagonists. It was shown that the chemical nature of the tethering atom linking together the proline functionality and the substituted aryl ring was of key importance for the observed iGluR profile. The position of the aryl carboxylate functionality together with the carbon, oxygen, and sulfur tethers resulted in differentiated pharmacological profiles. Most notably, analogues <b>1b</b>,<b>n</b>,<b>p</b>,<b>t</b> were shown to be selective GluK1 antagonists in the low micromolar range. An X-ray structure of <b>1b</b> in complex with GluK1 LBD revealed that <b>1b</b> interacts with binding site amino acid residues in a similar way to KA but induces a ∼5° larger domain opening. A SAR study of <b>1b</b> showed that introduction of substituents were allowed for on the aryl ring, compounds <b>1n</b>,<b>p</b>,<b>s</b>,<b>t</b>, but limitations also existed (loss of binding affinity) when large bulkiness was oriented in the direction of residues Glu441 and Tyr489, compounds <b>1o</b>,<b>q</b>.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33042" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33042" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General</h3><div class="NLM_p">All reagents were obtained from commercial suppliers and used without further purification. Dry solvents were obtained differently. THF was distilled over sodium/benzophenone. Et<sub>2</sub>O was dried over neatly cut sodium. All solvents were tested for water content using a Carl Fisher apparatus. Water- or air-sensitive reactions were conducted in flame-dried glassware under nitrogen with syringe–septum cap technique. Purification was performed by dry column vacuum chromatography (DCVC) with silica gel size 25–40 μm (Merck, Silica gel 60). For TLC, Merck TLC silica gel F<sub>254</sub> plates were used with appropriate spray reagents: KMnO<sub>4</sub> or molybdenum blue. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were obtained on a Varian Mercury Plus (300 MHz) and a Varian Gemini 2000 instrument (75 MHz), respectively. Preparative HPLC was either done using an Agilent HPLC system with a 1100 series pump (20 mL/min), a 1200 series multiple wavelength detector, and a Zorbax, 300 SB-C18 column (21.2 mm × 250 mm, 7 μm), or using a TSP UV100 detector, a JASCO 880-PU pump (10 mL/min), a Merck Hitachi D-2000 Chromato-Integrator, and an XTerraPrep MS C18, 10 mm × 300 mm, 10 μm) column. LC-MS was performed using an Agilent 1200 series solvent delivery system equipped with an autoinjector coupled to an Agilent 6400 series triple quadrupole mass spectrometer and an electrospray ionization source. The XBridge column (4.6 mm × 100 mm, 3.5 μm) was eluted at 1.0 mL/min with a linear gradient of H<sub>2</sub>O/CH<sub>3</sub>CN/HCOOH (A, 90:10:0.05; and B, 10:90:0.046). Optical rotation was measured using a PerkinElmer 241 spectrometer, with Na lamp at 589 nm. Melting points were measured using an automated melting point apparatus, MPA100 OptiMelt (SRS) and are stated uncorrected. Compounds were dry either under high vacuum or freeze-dried using a Holm & Halby, Heto LyoPro 6000 freeze-drier.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> For the Experimental Work Leading to <b>1l</b>–<b>t</b>, the Following Apparatuses Were Used</h4><div class="NLM_p last"><sup>1</sup>H NMR spectra were recorded on a 400 MHz Avance Bruker and <sup>13</sup>C NMR spectra on a 100 MHz Avance Bruker, respectively, using CDCl<sub>3</sub>, D<sub>2</sub>O, or CD<sub>3</sub>OD as solvent with CDCl<sub>3</sub> (δ 7.26/77.0), D<sub>2</sub>O (δ 4.79), or CD<sub>3</sub>OD (δ 3.31/49.0) as internal standards, respectively. Analytical HPLC was performed using a LaChrom Merck Hitachi L-7100 pump, autosampler L-7200, and UV detector L-7400 (254 nm) installed with a ChromolithSpeedROD RP-18 column (4.6 mm × 50 mm) using a 5 → 95% CH<sub>3</sub>CN gradient in H<sub>2</sub>O containing 0.1% TFA with a flow-rate of 4 mL/min. Preparative HPLC was done using Dionex Ultimate 3000 pump, Dionex Ultimate 3000 multiple wavelength detector, and Dionex Ultimate 3000 fraction collector and Gemini-NX C18 column (10 mm × 250 mm, 5 μm). The loop size was 10 mL, the mobile phase was 0.1% TFA in different mixtures of H<sub>2</sub>O and CH<sub>3</sub>CN, and the flow was 5 mL/min. LC-MS was performed using a Agilent 1100 HPLC systems with a XBridge 3.5 μm C-18 column (100 mm × 4.60 mm) using gradient elution from buffer A (H<sub>2</sub>O:CH<sub>3</sub>CN:HCOOH, 95:5:0.1) to buffer B (H<sub>2</sub>O:CH<sub>3</sub>CN: HCOOH, 5:95:0.05) over 10 min flow rate: 1.0 mL/min, coupled to an Hewlett-Packard 1100 series mass spectrometer with an electrospray ionization source. Melting points were measured using a Stanford Research Systems Optimelt automatic melting point system and are stated uncorrected. Optical rotation was measured using a Bellingham-Stanley ADP410 polarimeter with Na lamp (589 nm). Compounds were dried under high vacuum. The purity of all tested compounds <b>1a</b>–<b>t</b> was determined by HPLC to be >95% (254 nm).</div></div><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> (2<i>S</i>,4<i>R</i>)-4-(3-Carboxybenzyl)-proline Hydrochloride (<b>1a</b>)</h4><div class="NLM_p last">Diacid <b>6</b> (121 mg, 0.35 mmol, 1.00 equiv) was dissolved in dry Et<sub>2</sub>O (2 mL), cooled to 0 °C, and 1 M HCl in Et<sub>2</sub>O (4.18 mL, 4.18 mmol, 11.9 equiv) added under an atmosphere of nitrogen. The mixture was stirred at rt for 18 h and concentrated in vacuo to yield the deprotected amino acid <b>1a</b> (98 mg, 99%) as a yellow foam. The mixture was triturated with CH<sub>3</sub>CN twice to yield a white, colloid precipitate that was isolated by decantation and drying of the semisolid in vacuo. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 2.05–2.18 (1H, m), 2.18–2.34 (1H, m), 2.57–2.75 (1H, m), 2.75–2.88 (2H, m), 2.98–3.10 (1H, m), 3.52 (1H, dd, <i>J</i> = 11, 7 Hz), 4.40 (1H, dd, <i>J</i> = 9, 4 Hz), 7.38–7.53 (2H, m), 7.78–7.88 (2H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 34.5, 37.6, 39.1, 50.8, 60.5, 128.4, 129.5, 130.3, 130.4, 134.6, 140.4, 171.0, 173.3. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 250.1, found 250.1. OR: [α]<sup>22</sup><sub>D</sub> −17 (<i>c</i> = 0.34; 2 M NaOH). Melting point: decomposition.</div></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> (2<i>S</i>,4<i>R</i>)-4-(2-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid Trifluoroacetic Acid (<b>1b</b>)</h4><div class="NLM_p last">Diacid <b>8</b> (70 mg, 0.20 mmol, 1.00 equiv) was suspended in DCM (1.5 mL) and dissolved by addition of TFA (2 mL). The mixture was left to stir at rt under an atmosphere of nitrogen for 18 h. The mixture was concentrated in vacuo, suspended in H<sub>2</sub>O, and freeze-dried to yield amino acid <b>1b</b> (65 mg, 89%) as a white solid. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 2.33 (1H, ddd, <i>J</i> = 15, 11, 5 Hz), 2.62 (1H, dd, <i>J</i> = 15, 8 Hz), 3.63 (2H, m), 4.53 (1H, dd, <i>J</i> = 11, 8 Hz), 5.20 (1H, m), 6.97–7.05 (2H, m), 7.46 (1H, ddd, <i>J</i> = 8, 8, 2 Hz), 7.64 (1H, dd, <i>J</i> = 8, 2 Hz). <sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 35.2, 51.4, 59.7, 77.6, 115.7, 121.3, 122.6, 132.1, 134.8, 155.2, 170.4, 172.4. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.2. OR: [α]<sup>22</sup><sub>D</sub> −6.7 (<i>c</i> = 0.37; H<sub>2</sub>O). Melting point: decomposition.</div></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> (2<i>S</i>,4<i>R</i>)-4-(3-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1c</b>)</h4><div class="NLM_p last">Diacid <b>9</b> (120 mg, 0.34 mmol, 1.00 equiv) was dissolved in AcOH (3 mL) and 1 M HCl in Et<sub>2</sub>O (2 mL) added. The mixture was left to stir at rt under an atmosphere of nitrogen for 18 h. The white precipitate formed was isolated by decantation of the solvents, washing with Et<sub>2</sub>O, suspension in Et<sub>2</sub>O, and filtration. The solid was dried in vacuo to yield amino acid <b>1c</b> (94 mg, 96%) as a white solid. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 2.43 (1H, ddd, <i>J</i> = 14, 10, 5 Hz), 2.74 (1H, dd, <i>J</i> = 14, 8 Hz), 3.59–3.80 (2H, m), 4.61 (1H, dd, <i>J</i> = 11, 8 Hz), 5.31 (1H, bs), 7.22 (1H, dd, <i>J</i> = 8, 2 Hz), 7.44 (1H, t, <i>J</i> = 8 Hz), 7.51 (1H, bs), 7.64 (1H, d, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 35.1, 51.6, 59.4, 76.6, 116.9, 121.9, 124.0, 130.7, 131.8, 156.2, 170.4, 172.4. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.0. HPLC: >99% (254 nm). OR: [α]<sup>22</sup><sub>D</sub> −9.5 (<i>c</i> = 0.57; 99.9% EtOH). Melting point: decomposition.</div></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> (2<i>S</i>,4<i>R</i>)-4-(4-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1d</b>)</h4><div class="NLM_p last">Diacid <b>10</b> (89 mg, 0.25 mmol, 1.00 equiv) was suspended in DCM (4.5 mL) and TFA (4.5 mL) added. The mixture was left to stir at rt under an atmosphere of nitrogen for 18 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O, and freeze-dried. The salt was dissolved in 2 M HCl (5 mL), concentrated in vacuo, dissolved in H<sub>2</sub>O (3 × 5 mL), and concentrated in vacuo after each addition. The salt was then dissolved in H<sub>2</sub>O and freeze-dried to yield amino acid <b>1d</b> (65 mg, 90%) as a reddish foam. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 1.36 (1H, m), 1.63 (1H, dd, <i>J</i> = 14, 8 Hz), 2.24 (1H, d, <i>J</i> = 13 Hz), 2.66 (1H, dd, <i>J</i> = 14, 5 Hz), 3.07 (1H, dd, <i>J</i> = 9, 8 Hz), 4.18 (1H, m), 6.23 (2H, dm, <i>J</i> = 9 Hz), 7.19 (2H, dm, <i>J</i> = 9 Hz). <sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 34.4, 48.6, 57.7, 75.5, 111.9, 125.7, 128.0, 156.0, 171.6, 178.3. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.1. OR: [α]<sup>22</sup><sub>D</sub> −81.18 (<i>c</i> = 0.11; 2 M NaOH). Melting point: decomposition.</div></div><div id="sec4_1_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> (2<i>S</i>,4<i>R</i>)-4-((2-Carboxyphenyl)thio)pyrrolidine-2-carboxylic Acid Trifluoroacetic Acid (<b>1e</b>)</h4><div class="NLM_p last">Protected thioether <b>12</b> (196 mg, 0.53 mmol) was dissolved in DCM (5 mL) and TFA (4 mL) added. The mixture was stirred at rt for 18 h under an atmosphere of nitrogen and subsequently concentrated in vacuo. The product was dissolved in H<sub>2</sub>O (15 mL) and freeze-dried to yield amino acid <b>1e</b> (191 mg, 94%) as a pale-brown solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 2.41–2.48 (1H, m), 2.60–2.68 (1H, m), 3.37 (1H, dd, <i>J</i> = 12, 4 Hz), 3.87 (1H, dd, <i>J</i> = 12, 4 Hz), 4.25 (1H, s), 4.63 (1H, dd, <i>J</i> = 12, 8 Hz), 7.34 (1H, t, <i>J</i> = 8 Hz), 7.45 (1H, d, <i>J</i> = 8 Hz), 7.54 (1H, t, <i>J</i> = 8, Hz), 7.88 (1H, d, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (400 MHz, D<sub>2</sub>O): δ 34.3, 41.9, 50.8, 59.0, 126.7, 129.3, 130.4, 131.1, 133.2, 135.3, 163.0, 171.2. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 268.1, found 268.1. OR: [α]<sup>22</sup><sub>D</sub> −11.78 (<i>c</i> = 0.44; H<sub>2</sub>O and 2% TFA). Melting point: decomposition.</div></div><div id="sec4_1_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> (2<i>S</i>,4<i>R</i>)-4-((3-Carboxyphenyl)thio)pyrrolidine-2-carboxylic Acid (<b>1f</b>)</h4><div class="NLM_p last">Methyl 3-mercaptobenzoate (286 mg, 1.70 mmol, 1.44 equiv) was dissolved in DMF (12 mL) and K<sub>2</sub>CO<sub>3</sub> (243 mg, 1.76 mmol, 1.49 equiv) added. Mesylate <b>11</b> (365 mg, 1.18 mmol, 1.00 equiv) was added to the mixture, which was subsequently left to stir at rt under an atmosphere of nitrogen for 18 h. To the mixture was added 10% brine (100 mL) and extracted with Et<sub>2</sub>O (3 × 60 mL). The combined organic phases was washed with 0.5 M NaOH (2 × 80 mL) and brine (60 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude oil was dissolved in THF (10 mL), cooled to 0 °C, and 2 M NaOH (5 mL) added dropwise over the course of 5 min. After additional 15 min at 0 °C, the mixture was left to stir at rt for 19 h. The mixture was added H<sub>2</sub>O (20 mL) and washed with Et<sub>2</sub>O (2 × 20 mL). The aqueous phase was acidified (pH ≈ 1) using conc HCl, extracted with a combination of Et<sub>2</sub>O and THF (3:4) (20 mL), and concentrated in vacuo to give crude, protected diacid <b>13</b>. The crude diacid was suspended in DCM (2 mL) and dissolved by addition of TFA (2 mL). The mixture was left to stir at rt under an atmosphere of nitrogen for 22 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O (30 mL), and washed with DCM (3 × 30 mL). The water phase was freeze-dried to yield amino acid <b>1f</b> (121 mg, 40%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 2.33–2.40 (1H, m), 2.49–2.57 (1H, m), 3.32 (1H, dd, <i>J</i> = 12, 4 Hz), 3.77 (1H, dd, <i>J</i> = 12, 4 Hz), 4.07 (1H, pentet, <i>J</i> = 4 Hz), 4.60 (1H, t, <i>J</i> = 8 Hz), 7.37 (1H, t, <i>J</i> = 8 Hz), 7.56 (1H, d, <i>J</i> = 8 Hz), 7.78 (1H, d, <i>J</i> = 8 Hz), 7.85 (1H, s). <sup>13</sup>C NMR (400 MHz, D<sub>2</sub>O): δ 34.4, 43.5, 50.7, 58.9, 129.1, 129.66, 130.7, 132.4, 133.0, 136.4, 169.2, 171.4. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 268.1, found 268.1. OR: [α]<sup>22</sup><sub>D</sub> −6.6 (<i>c</i> = 0.29; H<sub>2</sub>O). Melting point: decomposition.</div></div><div id="sec4_1_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> (2<i>S</i>,4<i>S</i>)-4-(1-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid Trifluoroacetic Acid (<b>1g</b>)</h4><div class="NLM_p last">Protected proline <b>15</b> (288 mg, 0.82 mmol, 1.00 equiv) was suspended in DCM (5 mL) and dissolved by addition of TFA (5 mL). The mixture was left to stir at rt under an atmosphere of nitrogen for 18 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O (5 mL), and freeze-dried to yield proline salt <b>1g</b> (260 mg, 87%) as a brownish foam. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 2.46–2.62 (2H, m), 3.47 (1H, dd, <i>J</i> = 13, 4 Hz), 3.69 (1H, d, <i>J</i> = 13 Hz), 4.29 (1H, dd, <i>J</i> = 10, 4 Hz), 5.14 (1H, m), 6.95–7.03 (2H, m), 7.40–7.46 (1H, m), 7.61 (1H, dd, <i>J</i> = 8, 2 Hz). <sup>13</sup>C NMR (400 MHz, D<sub>2</sub>O): δ 34.3, 51.1, 59.5, 76.6, 115.2, 121.3, 122.2, 131.5, 134.2, 154.5, 170.4, 172.7. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.1. OR: [α]<sup>22</sup><sub>D</sub> −7.5 (<i>c</i> = 0.31; H<sub>2</sub>O with 1% TFA). Melting point: decomposition.</div></div><div id="sec4_1_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> (2<i>S</i>,4<i>S</i>)-4-(3-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1h</b>)</h4><div class="NLM_p last">Protected proline <b>16</b> (204 mg, 0.58 mmol, 1.00 equiv) was suspended in DCM (5 mL) and dissolved by addition of TFA (5 mL). The mixture was left to stir at rt under an atmosphere of nitrogen for 18 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O (5 mL), and freeze-dried. The salt was dissolved in 2 M HCl (3 mL), concentrated in vacuo, redissolved in H<sub>2</sub>O (3 × 5 mL), and concentrated in vacuo to give <b>1h</b> (156 mg, 93%) as a reddish foam. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 2.50–2.67 (2H, m), 3.59 (1H, dd, <i>J</i> = 13, 4 Hz), 3.70 (1H, d, <i>J</i> = 13 Hz), 4.59 (1H, t, <i>J</i> = 6 Hz), 5.14 (1H, m), 7.03 (1H, ddm, <i>J</i> = 8, 3 Hz), 7.25–7.34 (2H, m), 7.47 (1H, dm, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (75 MHz, D<sub>2</sub>O): δ 34.3, 51.4, 58.8, 75.2, 116.3, 121.3, 123.4, 130.1, 131.0, 155.1, 169.6, 171.7. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.1. OR: [α]<sup>22</sup><sub>D</sub> + 8.0 (c = 0.56; H<sub>2</sub>O). Melting point: decomposition.</div></div><div id="sec4_1_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> (2<i>S</i>,4<i>S</i>)-4-(1-Carboxyphenoxy)pyrrolidine-2-carboxylic Acid Trifluoroacetic Acid (<b>1i</b>)</h4><div class="NLM_p last">Protected proline <b>17</b> (262 mg, 0.75 mmol, 1.00 equiv) was suspended in DCM (5 mL) and TFA (5 mL) added. The mixture was left to stir at rt under an atmosphere of nitrogen for 18 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O (5 mL), and freeze-dried to yield proline salt <b>1i</b> (231 mg, 85%) as a brownish foam. <sup>1</sup>H NMR (300 MHz, D<sub>2</sub>O): δ 2.51–2.66 (2H, m), 3.57 (1H, dd, <i>J</i> = 13, 4 Hz), 3.70 (1H, d, <i>J</i> = 13 Hz), 4.53 (1H, dd, <i>J</i> = 9, 4 Hz), 5.25 (1H, m), 6.87–6.92 (2H, m), 7.84–7.90 (2H, m). <sup>13</sup>C NMR (400 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 34.3, 50.7, 57.8, 75.0, 113.7, 115.2, 123.9, 131.4, 158.5, 166.8, 170.2. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.0. OR: [α]<sup>22</sup><sub>D</sub> + 28 (<i>c</i> = 0.25; H<sub>2</sub>O and 1% TFA). Melting point: decomposition.</div></div><div id="sec4_1_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> (2<i>S</i>,4<i>S</i>)-4-((2-Carboxyphenyl)thio)pyrrolidine-2-carboxylic Acid (<b>1j</b>)</h4><div class="NLM_p last">Protected proline <b>19</b> (100 mg, 0.27 mmol) was suspended in DCM (1 mL) and dissolved by addition of TFA (1 mL). The mixture was left to stir at rt under nitrogen for 1 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O (30 mL), and washed with DCM (3 × 20 mL). The aqueous phase was freeze-dried to yield <b>1j</b> as its zwitterion (54 mg, 74%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 2.18–2.25 (1H, m), 2.77–2.85 (1H, m), 3.34 (0.5H, d, <i>J</i> = 4 Hz), 3.39 (0.5H, d, <i>J</i> = 4 Hz), 3.76 (1H, dd, <i>J</i> = 12, 8 Hz), 4.15 (1H, m), 4.50 (1H, dd, <i>J</i> = 8, 4 Hz), 7.26 (1H, t, <i>J</i> = 8 Hz), 7.35 (1H, t, <i>J</i> = 8 Hz), 7.47 (2H, t, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (400 MHz, D<sub>2</sub>O): δ 34.0, 41.6, 50.8, 58.9, 126.6, 129.5, 130.4, 131.0, 133.1, 135.3, 162.8, 170.9. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 268.1, found 268.1. OR: [α]<sup>22</sup><sub>D</sub> −4.7 (<i>c</i> = 0.19; H<sub>2</sub>O and 5% TFA). Melting point: decomposition.</div></div><div id="sec4_1_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> (2<i>S</i>,4<i>S</i>)-4-((3-Carboxyphenyl)thio)pyrrolidine-2-carboxylic Acid (<b>1k</b>)</h4><div class="NLM_p last">Methyl 3-mercaptobenzoate (473 mg, 2.81 mmol, 1.51 equiv) was dissolved in DMF (22 mL) and K<sub>2</sub>CO<sub>3</sub> (397 mg, 2.87 mmol, 1.54 equiv) added. Mesylate <b>18</b> (604 mg, 1.87 mmol, 1.00 equiv) was dropwise added to the mixture, which was subsequently left to stir at rt under nitrogen for 18 h. The mixture was added 1% brine (400 mL) and extracted with Et<sub>2</sub>O (3 × 100 mL). The combined organic phases was washed with 0.5 M NaOH (2 × 100 mL) and brine (80 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The oil was dissolved in THF (10 mL), cooled to 0 °C, and 2 M NaOH (5 mL) added dropwise over the course of 5 min. After an additional 15 min at 0 °C, the mixture was left to stir at rt for 20 h. The mixture was added H<sub>2</sub>O (30 mL) and washed with Et<sub>2</sub>O (2 × 30 mL). The aqueous phase was acidified (pH ≈ 1) using conc HCl, extracted with a combination of Et<sub>2</sub>O and THF (3:4) (40 mL), and concentrated in vacuo to give crude diacid <b>20</b>. The crude diacid <b>20</b> was suspended in DCM (4 mL) and dissolved by addition of TFA (4 mL). The mixture was left to stir at rt under nitrogen for 21 h. The mixture was concentrated in vacuo, dissolved in H<sub>2</sub>O (30 mL), and washed with DCM (3 × 30 mL). The aqueous phase was freeze-dried to yield <b>1k</b> as its zwitterion (188 mg, 38%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O): δ 2.25–2.32 (1H, m), 2.79–2.87 (1H, m), 3.48 (1H, dd, <i>J</i> = 12, 4 Hz), 3.76 (1H, dd, <i>J</i> = 12, 4 Hz), 4.20 (1H, p, <i>J</i> = 4 Hz), 4.44 (1H, dd, <i>J</i> = 12, 8 Hz), 7.55 (1H, t, <i>J</i> = 8 Hz), 7.76 (1H, d, <i>J</i> = 8 Hz), 7.78 (1H, dd, <i>J</i> = 8, 4 Hz), 8.01 (1H, s). <sup>13</sup>C NMR (400 MHz, D<sub>2</sub>O): δ 34.3, 43.7, 50.7, 59.8, 129.3, 129.7, 130.9, 132.9, 133.2, 137.0, 169.7, 171.8. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 268.1, found 268.1. OR: [α]<sup>22</sup><sub>D</sub> −18 (<i>c</i> = 0.24; H<sub>2</sub>O). Melting point: decomposition.</div></div><div id="sec4_1_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> (2<i>S</i>,4<i>R</i>)-4-(2-(Carboxymethyl)phenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1l</b>)</h4><div class="NLM_p last">To a flask containing protected proline <b>29</b> (97 mg, 0.25 mmol, 1.00 equiv) was added 4 M HCl (5 mL), and the mixture was heated to reflux for 19 h. The mixture was concentrated in vacuo (water aspirator). Water (2 × 20 mL) was added, and the mixture was concentrated in vacuo after each addition. The mixture was purified using prep HPLC. The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL), yielding <b>1l</b> (39 mg, 53%) as a beige semisolid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 2.30–2.45 (m, 1H), 2.59–2.69 (m, 1H), 3.53–3.66 (m, 3H), 3.67–3.78 (m, 1H), 4.48–4.60 (m, 1H), 5.16–5.26 (m, 1H), 6.87–7.03 (m, 2H), 7.16–7.24 (m, 1H), 7.24–7.34 (m, 1H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) δ 34.6, 35.9, 51.1, 58.8, 75.4, 112.3, 121.9, 123.6, 129.2, 131.8, 153.9, 171.1, 176.9. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 266.1, found 266.0. OR: [α]<sup>25</sup><sub>D</sub> + 8.0 (<i>c</i> = 0.5; H<sub>2</sub>O). Melting point: decomposition.</div></div><div id="sec4_1_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> (2<i>S</i>,4<i>R</i>)-4-(3-(Carboxymethyl)phenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1m</b>)</h4><div class="NLM_p last">Carried out as described for <b>1l</b> with <b>30</b> (142 mg, 0.361 mmol, 1.00 equiv), 4 M HCl (5 mL), and heating to reflux for 2 h. The mixture was purified using prep HPLC to yield <b>1m</b> as the TFA salt. The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL) to yield <b>1m</b> (62 mg, 57%) as a white solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 2.45 (ddd, <i>J</i> = 4, 10, 14 Hz, 1H), 2.72–2.85 (m, 1H), 3.69–3.78 (m, 4H), 4.67 (dd, <i>J</i> = 8, 10 Hz, 1H), 5.27–5.34 (m, 1H), 6.93–7.06 (m, 3H), 7.39 (dd, <i>J</i> = 8 Hz, 1H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) δ 34.4, 40.3, 51.0, 58.6, 75.8, 114.8, 117.1, 123.4, 130.2, 136.0, 155.9, 171.7, 176.4. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 266.1, found 266.0. OR: [α]<sup>25</sup><sub>D</sub> + 80 (<i>c</i> = 0.25; H<sub>2</sub>O). TLC: <i>R</i><sub>f</sub> 0.46 (heptane:EtOAc, 1:1). HPLC: >99% (254 nm). Melting point: 203.2–211.0 °C.</div></div><div id="sec4_1_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (2<i>S</i>,4<i>R</i>)-4-((3-Carboxynaphthalen-2-yl)oxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1n</b>)</h4><div class="NLM_p last">Carried out as described for <b>1l</b> with <b>31</b> (97 mg, 0.25 mmol), 4 M HCl (5 mL), and heating to reflux for 19 h. The mixture was purified using prep HPLC to yield <b>1n</b> as the TFA salt. The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL) to yield <b>1n</b> (40 mg, 63%) as a beige semisolid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 2.44 (ddd, <i>J</i> = 4, 10, 15 Hz, 1H), 2.78 (dd, <i>J</i> = 8, 14 Hz, 1H), 3.67 (s, 2H), 4.67 (t, <i>J</i> = 9 Hz, 1H), 5.39 (br s, 1H), 7.35–7.46 (m, 2H), 7.52 (t, <i>J</i> = 8 Hz, 1H), 7.77 (d, <i>J</i> = 8 Hz, 1H), 7.85 (d, <i>J</i> = 8 Hz, 1H), 8.35 (s, 1H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) δ 36.2, 52.2, 60.2, 78.6, 112.2, 124.0, 126.5, 127.8, 129.8, 129.8, 129.9, 129.9, 134.4, 137.3, 153.4, 169.7, 171.4. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 302.1, found 302.0. OR: [α]<sup>25</sup><sub>D</sub> + 1.6 (<i>c</i> = 1.25; abs EtOH). Melting point: decomposition.</div></div><div id="sec4_1_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (2<i>S</i>,4<i>R</i>)-4-((2-Carboxynaphthalen-1-yl)oxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1o</b>)</h4><div class="NLM_p last">Carried out as described for <b>1l</b> with <b>32</b> (276 mg, 0.643 mmol, 1.00 equiv), 4 M HCl (7.5 mL), and heated to reflux for 39 h. The mixture was purified using prep HPLC to yield the TFA salt of <b>1o</b>. The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL) to yield <b>1o</b> (88 mg, 41%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 2.22–2.29 (m, 1H), 2.76 (dd, <i>J</i> = 8, 14 Hz, 1H), 3.69–3.74 (m, 1H), 3.90 (d, <i>J</i> = 13 Hz, 1H), 5.18 (br s, 1H), 7.69–7.76 (m, 2H), 7.83 (d, <i>J</i> = 7 Hz, 1H), 7.86–7.90 (m, 1H), 8.02 (d, <i>J</i> = 6 Hz, 1H), 8.22 (s, 1H). <sup>13</sup>C NMR (100 MHz, D<sub>2</sub>O) δ 35.0, 51.4, 59.8, 84.1, 119.9, 122.8, 124.9, 126.2, 127.4, 127.8, 128.2, 129.0, 136.7, 153.5, 170.2, 172.4. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 302.1, found 302.0. Melting point: 218.5–229.8 °C.</div></div><div id="sec4_1_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> (2<i>S</i>,4<i>R</i>)-4-((1-Carboxynaphthalen-2-yl)oxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1p</b>)</h4><div class="NLM_p last">To a flask containing protected proline <b>33</b> (168 mg, 0.391 mmol, 1.00 equiv), 4 M HCl (5 mL) was added, and the mixture was heated to reflux for 23 h. The mixture was concentrated in vacuo, H<sub>2</sub>O (10 mL) added, and concentrated in vacuo again. Then 2 M NaOH (5 mL) was added and the mixture heated to reflux for 2.5 h. H<sub>2</sub>O (10 mL) was added and the mixture washed with DCM (3 × 10 mL). The aqueous phase was acidified using 6 M HCl, the precipitate retained on a filter, and dried in vacuo to yield the HCl salt of <b>1p</b> (67 mg, 51%) as a beige/brown solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 2.42 (ddd, <i>J</i> = 4, 11, 14 Hz, 1H), 2.72–2.83 (m, 1H), 3.63–3.73 (m, 1H), 3.76 (dd, <i>J</i> = 4, 13 Hz, 1H), 4.72 (dd, <i>J</i> = 7, 11 Hz, 1H), 5.48 (t, <i>J</i> = 4 Hz, 1H), 7.42–7.52 (m, 2H), 7.56 (ddd, <i>J</i> = 1, 7, 8 Hz, 1H), 7.84–7.93 (m, 2H), 8.02 (d, <i>J</i> = 9 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 36.2, 52.5, 60.0, 79.6, 117.2, 122.5, 125.2, 126.3, 128.9, 129.4, 131.0, 131.8, 133.1, 151.9, 170.7, 171.2. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 302.1, found 302.0. Melting point: decomposition.</div></div><div id="sec4_1_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-(((3<i>R</i>,5<i>S</i>)-5-Carboxypyrrolidin-3-yl)oxy)isophthalic Acid (<b>1q</b>)</h4><div class="NLM_p last">To a flask with protected proline <b>34</b> (189 mg, 0.432 mmol, 1.00 equiv) was added 4 M HCl (4 mL), and the mixture was heated to reflux for 21 h. The mixture was concentrated in vacuo, H<sub>2</sub>O (10 mL) added, and the mixture was concentrated in vacuo again to yield the HCl salt of <b>1q</b> (136 mg, 95%) as an off-white solid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 2.51 (ddd, <i>J</i> = 4, 10, 15 Hz, 1H), 2.73–2.82 (m, 1H), 3.65–3.78 (m, 2H), 4.77 (dd, <i>J</i> = 8, 10 Hz, 1H), 5.43–5.48 (br s, 1H), 7.29 (d, <i>J</i> = 9 Hz, 1H), 8.21 (dd, <i>J</i> = 2, 9 Hz, 1H), 8.51 (d, <i>J</i> = 2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 34.7, 50.9, 58.4, 77.4, 114.9, 121.7, 124.2, 133.6, 134.9, 159.2, 166.9, 167.2, 169.3. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 296.1, found 296.0. OR: [α]<sup>25</sup><sub>D</sub> −109 (<i>c</i> = 0.055; 99.9% EtOH). Melting point: 184.2–198.0 °C.</div></div><div id="sec4_1_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> (2<i>S</i>,4<i>R</i>)-4-(2-Carboxy-4-methylphenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1r</b>)</h4><div class="NLM_p last">Carried out as described for <b>1l</b> with <b>35</b> (90 mg, 0.237 mmol, 1.00 equiv), 4 M HCl (5 mL), and heated to reflux for 24 h. The mixture was purified using prep HPLC to yield the TFA salt of <b>1r.</b> The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL) to yield the HCl salt of <b>1r</b> (50 mg, 70%) as a beige semisolid. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ 0.99 (s, 3H), 2.41–2.48 (m, 2H), 3.40 (dd, <i>J</i> = 8, 11 Hz, 1H), 3.50–3.56 (m, 2H), 3.97–4.04 (m, 1H), 5.74 (d, <i>J</i> = 8 Hz, 1H), 6.07–6.15 (m, 1H), 6.31 (d, <i>J</i> = 2 Hz, 1H). <sup>13</sup>C NMR (101 MHz, CD<sub>3</sub>OD) δ 18.0, 33.1, 49.6, 57.5, 76.0, 114.4, 119.3, 130.4, 131.0, 133.5, 151.4, 168.7, 170.1. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 266.1, found 266.0. OR: [α]<sup>25</sup><sub>D</sub> −2 (<i>c</i> = 1; H<sub>2</sub>O). Melting point: 173.3–207.9 °C.</div></div><div id="sec4_1_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> (2<i>S</i>,4<i>R</i>)-4-(4-Acetyl-2-carboxyphenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1s</b>)</h4><div class="NLM_p last">Carried out as described for <b>1l</b> with <b>36</b> (152 mg, 0.373 mmol, 1.00 equiv), 4 M HCl (5 mL), and heated to reflux for 19 h. The mixture was purified using prep HPLC to yield the TFA salt of <b>1s.</b> The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL) to yield the HCl salt of <b>1s</b> (76 mg, 62%) as a beige semisolid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 2.50–2.59 (m, 1H), 2.60 (s, 3H), 2.81 (ddt, <i>J</i> = 2, 8, 15 Hz, 1H), 3.74–3.88 (m, 2H), 4.71 (dd, <i>J</i> = 8, 10 Hz, 1H), 5.47 (t, <i>J</i> = 4 Hz, 1H), 7.18 (d, <i>J</i> = 8.85 Hz, 1H), 8.10 (dd, <i>J</i> = 2.39, 8.85 Hz, 1H), 8.27 (d, <i>J</i> = 2.33 Hz, 1H). <sup>13</sup>C NMR (101 MHz, D<sub>2</sub>O) δ 25.9, 34.6, 50.9, 58.9, 77.1, 114.3, 114.8, 117.7, 120.7, 129.8, 132.6, 134.6, 159.1, 162.7, 163.0, 168.9, 171.4, 201.0. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 294.1, found 294.0. OR: [α]<sup>25</sup><sub>D</sub> −36 (<i>c</i> = 0.055; H<sub>2</sub>O). Melting point: 140.1–212.1 °C.</div></div><div id="sec4_1_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> (2<i>S</i>,4<i>R</i>)-4-(2-Carboxy-6-methylphenoxy)pyrrolidine-2-carboxylic Acid Hydrochloride (<b>1t</b>)</h4><div class="NLM_p last">Carried out as described for <b>1l</b> with <b>37</b> (59 mg, 0.16 mmol, 1.00 equiv), 4 M HCl (2 mL), and heated to reflux for 19 h. The mixture was purified using prep HPLC to yield the TFA salt of <b>1t.</b> The HCl salt was obtained by concentrating the product in vacuo from 1 M HCl (3 × 10 mL) to yield the HCl salt of <b>1t</b> (40 mg, 86%) as a reddish semisolid. <sup>1</sup>H NMR (400 MHz, D<sub>2</sub>O) δ 2.09–2.23 (m, 1H), 2.10 (s, 3H), 2.47–2.61 (m, 1H), 3.52–3.71 (m, 2H), 4.67 (d, <i>J</i> = 8 Hz, 1H), 4.77–4.84 (m, 1H), 6.99–7.12 (m, 1H), 7.39 (dd, <i>J</i> = 2, 8 Hz, 1H), 7.55 (dd, <i>J</i> = 2, 8 Hz, 1H). <sup>13</sup>C NMR (101 MHz, D<sub>2</sub>O) δ 16.1, 34.6, 51.5, 58.9, 81.9, 124.1, 124.7, 129.3, 132.5, 136.5, 153.7, 170.1, 171.0. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 266.1, found 266.0; Melting point: decomposition.</div></div><div id="sec4_1_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> (3<i>R</i>,5<i>S</i>)-(<i>tert</i>-Butoxycarbonyl)-5-(((<i>tert</i>-butyl)dimethylsilyloxy)methyl)-3-(3-(methoxycarbo-nyl)benzyl)-pyrrolidin-2-one (<b>3</b>)</h4><div class="NLM_p last">Protected pyroglutaminol <b>2</b> (500 mg, 1.52 mmol, 1.00 equiv) was weighed out in a dry flask that was afterward purged with and placed under an atmosphere of nitrogen. Dry THF (2 mL) was added and the solution cooled to −78 °C. Then 1 M LHMDS (1.8 mL, 1.8 mmol, 1.2 equiv) was dropwise added over the course of 10 min and stirred for 2 h prior to use. Methyl 3-bromomethylbenzoate (454 mg, 1.98 mmol, 1.31 equiv) was dissolved in dry THF (2 mL) in a dry flask and dropwise added to the mixture of compound <b>2</b> over the course of 15 min. The mixture was stirred for 3 h before being removed from the cooling bath and allowed to warm up to rt. The mixture was quenched by addition of saturated NH<sub>4</sub>Cl (4 mL) and transferred to a separatory funnel containing EtOAc (5 mL). The aqueous phase was reextracted with EtOAc (2 × 5 mL), and the combined organic phases was washed with brine, dried over MgSO<sub>4</sub>, and concentrated in vacuo. The product was purified using DCVC (0–28% EtOAc in heptanes) to afford phenyl ester <b>3</b> as a dark-orange oil (467 mg, 64%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 0.00 (6H, s), 0.85 (9H, s), 1.55 (9H, s), 1.75–1.93 (2H, m), 1.95–2.07 (1H, m), 3.05–3.20 (1H, m), 3.30–3.40 (1H, m), 3.56–3.64 (1H, m), 3.83–3.93 (4H, m), 4.00–4.09 (1H, m), 7.33–7.40 (2H, m), 7.81–7.85 (1H, m), 7.85–7.90 (1H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ −5.39, −5.35, 18.3, 26.0, 28.3, 28.5, 37.1, 44.5, 52.2, 57.0, 64.2, 82.9, 127.7, 128.6, 129.8, 130.3, 133.4, 139.6, 150.0, 166.9, 175.4. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 378.2, found 378.1. TLC: <i>R</i><sub>f</sub> 0.37 (EtOAc:heptanes, 1:4). OR: [α]<sup>20.5</sup><sub>D</sub> −31.55 (<i>c</i> = 0.37 g/100 mL; Abs EtOH).</div></div><div id="sec4_1_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> (2<i>S</i>,4<i>R</i>)-(<i>tert</i>-Butoxycarbonyl)-2-(((<i>tert</i>-butyl)dimethylsilyloxy)methyl)-4-(3-(hydroxymethyl)benzyl)-pyrrolidine (<b>4</b>)</h4><div class="NLM_p last">Compound <b>3</b> (1.31 g, 2.75 mmol, 1.00 equiv) was weighed out in a dry flask and dissolved in dry THF (10 mL) under an atmosphere of nitrogen. Then 1 M BH<sub>3</sub>·THF complex (14.6 mL, 14.6 mmol, 5.31 equiv) was added and the mixture heated to reflux for 21 h. The mixture was cooled to 0 °C and extra THF (35 mL) added. H<sub>2</sub>O (4.2 mL) was carefully added dropwise over the course of 30 min. Then 2 M NaOH (22.3 mL) was carefully added dropwise over the course of 30 min followed by addition of 30% H<sub>2</sub>O<sub>2</sub> (7 mL) over the course of 15 min. After stirring for 5 min at 0 °C, the mixture was removed from the ice-bath and left to stir for 1.5 h at rt. The mixture was transferred to a separation funnel containing sat. NaHCO<sub>3</sub> (75 mL) and EtOAc (75 mL). The aqueous phase was extracted with EtOAc (2 × 75 mL). The pooled organic phases were washed with brine (100 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DVCV (0–100% EtOAc in heptanes) to yield benzyl alcohol <b>4</b> (0.68 g, 56%) as a thick, colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 0.03 (6H, s), 0.88 (9H, s), 1.46 (9H, s), 1.55–1.85 (2H, m), 1.95–2.15 (1H, m), 2.50–2.75 (3H, m), 2.95–3.15 (1H, m), 3.35–3.55 (1.5H, m), 3.55–3.85 (2H, m), 3.85–3.95 (0.5H, m), 4.68 (2H, s), 7.08 (1H, d, <i>J</i> = 7 Hz), 7.12–7.25 (3H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ −5.1, 18.5, 26.1, 28.8, 34.1, 34.7, 37.9, 39.0, 39.9, 52.1, 52.5, 58.6, 58.8, 63.6, 64.0, 65.5, 79.2, 79.5, 124.8, 127.3, 128.0, 128.6, 140.7, 141.1, 154.5. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 380.2, found 380.2 (-<i>t</i>-Bu). TLC: <i>R</i><sub>f</sub> 0.46 (EtOAc:heptanes, 1:1). OR: [α]<sup>20.5</sup><sub>D</sub> −28.41 (<i>c</i> = 0.35 g/100 mL; Abs EtOH).</div></div><div id="sec4_1_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> (2<i>S</i>,4<i>R</i>)-<i>tert</i>-butyl 2-(hydroxymethyl)-4-(3-(hydroxymethyl)benzyl)pyrrolidine-1-carboxylate (<b>5</b>)</h4><div class="NLM_p last">Alcohol <b>4</b> (0.67 g, 1.55 mmol, 1.00 equiv) was dissolved in dry THF (11 mL) and 1 M TBAF in THF (4.66 mL, 4.66 mmol, 3.01 equiv) added under an atmosphere of nitrogen. The mixture was left to stir at rt for 1 h before adding satd NaHCO<sub>3</sub> (40 mL). The mixture was transferred to a separatory funnel containing EtOAc (30 mL). The aqueous phase was extracted with EtOAc (2 × 50 mL), and the pooled organic phases were washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DVCV (0–100% EtOAc in toluene) to yield diol <b>5</b> (0.26 g, 52%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 1.48 (9H, s), 1.60–2.05 (3H, m), 2.35–2.55 (1H, m), 2.55–2.70 (2H, m), 3.10–3.20 (1H, m), 3.42 (1H, dd, <i>J</i> = 10, 7 Hz), 3.52–3.65 (2H, m), 4.08 (1H, br s), 4.46 (1H, br s), 4.69 (2H, s), 7.07 (1H, d, J = 7 Hz), 7.16 (1H, s), 7.18–7.33 (2H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 28.7, 34.3, 39.1, 39.4, 52.6, 59.6, 65.4, 68.0, 80.6, 125.0, 127.3, 128.0, 128.7, 140.4, 141.2. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 266.1, found 266.1 (-<i>t</i>-Bu). TLC: <i>R</i><sub>f</sub> 0.10 (EtOAc:heptanes, 1:1). OR: [α]<sup>20.5</sup><sub>D</sub> −14.11 (<i>c</i> = 0.25 g/100 mL; Abs EtOH).</div></div><div id="sec4_1_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(3-carboxybenzyl)pyrrolidine-2-carboxylic Acid (<b>6</b>)</h4><div class="NLM_p last">Diol <b>5</b> (180 mg, 0.56 mmol, 1.00 equiv) was dissolved in CH<sub>3</sub>CN (2.4 mL) and a sodium phosphate buffer (1.8 mL, pH = 6.7 (670 mM)) and TEMPO (12 mg, 0.08 mmol, 0.14 equiv) added. The reaction mixture was heated to 35 °C and a solution of NaClO<sub>2</sub> (200 mg, 2.24 mmol, 4.00 equiv) in H<sub>2</sub>O (0.48 mL) added dropwise and a solution of NaOCl (8 μL 10–13% aqueous solution, approximately 1 mg, approximately 0.013 mmol, 0.023 equiv) in H<sub>2</sub>O (0.24 mL) simultaneously over the course of 2 h. The lack of characteristic intermediate coloring of mixture facilitated the addition of additional TEMPO (10 mg, 0.06 mmol, 0.11 equiv) and NaOCl (2 drops). This resulted in a dark coloring of the mixture that disappeared shortly after. The mixture was stirred for 18 h at 35 °C. The mixture was cooled to 0 °C, added H<sub>2</sub>O (2 mL) followed by addition of 2 M NaOH (dropwise until pH = 10). The mixture was dropwise added a solution of Na<sub>2</sub>SO<sub>3</sub> (300 mg, 2.38 mmol, 4.25 equiv) in H<sub>2</sub>O (2 mL). The mixture was stirred at rt for 30 min. The mixture was transferred to a separatory funnel and washed with Et<sub>2</sub>O (2 × 20 mL). The aqueous mixture was made acidic using 4 M HCl (a few drops, pH = 2) and extracted with Et<sub>2</sub>O (3 × 20 mL). The pooled organic phases was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to yield diacid <b>6</b> (122 mg, 62%) as a white solid. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 1.44 (4H, s), 1.50 (5H, s), 1.75–2.10 (1.46H, m), 2.25–2.40 (0.57H, m), 2.55–2.95 (3H, m), 3.00–3.20 (1H, m), 3.58 (0.57H, m), 3.80 (0.46H, m), 4.32 (0.46H, m), 4.42 (0.57H, d, <i>J</i> = 8 Hz), 7.33–7.44 (2H, m), 7.87–7.99 (2H, m). <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 28.4, 28.5, 35.0, 36.2, 38.4, 38.8, 39.2, 51.5, 51.8, 59.0, 80.7, 81.2, 128.3, 128.8, 129.7, 130.2, 134.0, 140.1, 153.8, 155.4, 171.6, 176.6, 178.7. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 350.2, found 350.3. TLC: <i>R</i><sub>f</sub> 0.46 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>20</sup><sub>D</sub> −3.4 (<i>c</i> = 0.38; Abs EtOH).</div></div><div id="sec4_1_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(2-carboxyphenoxy)pyrrolidine-2-carboxylic Acid (<b>8</b>)</h4><div class="NLM_p last">Commercially available <b>7</b> (140 mg, 0.57 mmol, 1.00 equiv), methyl 2-hydroxybenzoate (108 mg, 0.71 mmol, 1.24 equiv), PPh<sub>3</sub> (140 mg, 0.69 mmol, 1.21 equiv), and DIAD (181 mg, 0.69 mmol, 1.21 equiv) were weighed out in a dry flask and dissolved in dry THF (4 mL) under an atmosphere of nitrogen. The mixture was left to stir at rt for 18 h. The mixture was added Et<sub>2</sub>O/THF (1:1, 20 mL) and washed with 2 M NaOH (2 × 10 mL), brine (15 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–24% EtOAc in toluene). The mixture was dissolved in THF (6 mL), cooled to 0 °C, and 1 M LiOH (6 mL) added dropwise over the course of 2 min followed by addition of 4 M NaOH (1 mL) over the course of 2 min. The mixture was left to stir at 0 °C for 15 min and at rt for 20 h. The mixture was added H<sub>2</sub>O (20 mL) and the aqueous phase washed with Et<sub>2</sub>O (2 × 20 mL). To the aqueous phase was added conc HCl (pH = 1–2) and extracted with a mixture of Et<sub>2</sub>O and THF (1:1, 3 × 20 mL). The pooled organic phases was washed with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–100% EtOAc in heptanes containing 2% AcOH) to yield diacid <b>8</b> (123 mg, 61%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, (CDCl<sub>3</sub>): δ 1.42/1.43 (9H, 2<i>s</i>), 2.30–2.50 (1H, m), 2.55–2.75 (1H, m), 3.67–3.95 (2H, m), 4.49 (0.5H, t, <i>J</i> = 8 Hz), 4.59 (0.5H, t, <i>J</i> = 8 Hz), 5.09 (1H, bs), 6.96 (1H, dd, <i>J</i> = 8, 5 Hz), 7.07 (1H, td, <i>J</i> = 8, 4 Hz), 7.50 (1H, m), 7.98 (1H, dm, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (75 MHz, (CDCl<sub>3</sub>): δ 28.3, 28.4, 35.3, 36.6, 51.7, 52.1, 57.8, 57.9, 76.8, 77.2, 81.5, 81.7, 114.8, 115.1, 119.7, 120.1, 122.1, 133.2, 133.4, 134.6, 134.7, 153.9, 155.3, 156.3, 168.6, 168.7, 177.1, 177.5. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.2. HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.13 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>25</sup><sub>D</sub> −42 (<i>c</i> = 0.52; Abs EtOH).</div></div><div id="sec4_1_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(3-carboxyphenoxy)pyrrolidine-2-carboxylic Acid (<b>9</b>)</h4><div class="NLM_p last">Commercially available <b>7</b> (300 mg, 1.22 mmol, 1.00 equiv), methyl 3-hydroxybenzoate (230 mg, 1.51 mmol, 1.24 equiv), and PPh<sub>3</sub> (374 mg, 1.43 mmol, 1.17 equiv) were weighed out in a dry flask, dissolved in dry THF (8 mL), and cooled to 0 °C. DIAD (385 mg, 1.47 mmol, 1.20 equiv) was dropwise added over 15 min under an atmosphere of nitrogen, and the mixture was left to stir at rt for 18 h. The mixture was added Et<sub>2</sub>O (20 mL) and washed with 2 M NaOH (2 × 10 mL), brine (15 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC ((1) column. 0–25% EtOAc in toluene; (2) column. 0–35% in heptanes). The mixture was dissolved in THF (10 mL), cooled to 0 °C, and 2 M NaOH (5 mL) added dropwise over the course of 2 min. The mixture was left to stir at 0 °C for 15 min and at rt for 20 h. To the mixture was added H<sub>2</sub>O (20 mL) and the aqueous phase washed with Et<sub>2</sub>O (2 × 20 mL). The aqueous phase was added conc HCl (pH = 1–2) and extracted with a mixture of Et<sub>2</sub>O and THF (1:1, 3 × 20 mL). The pooled organic phases was washed with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to yield diacid <b>9</b> (336 mg, 91%) as a white foam. <sup>1</sup>H NMR (300 MHz, (CD<sub>3</sub>)<sub>2</sub>SO): δ 1.36 (9H, s), 2.10–2.30 (1H, m), 2.35–2.52 (1H, m), 3.50–3.70 (2H, m), 4.21 (1H, q, <i>J</i> = 8 Hz), 5.08 (1H, br s), 7.20 (1H, dd, <i>J</i> = 8, 2 Hz), 7.36–7.46 (2H, m), 7.54 (1H, d, <i>J</i> = 7 Hz). <sup>13</sup>C NMR (75 MHz, (CD<sub>3</sub>)<sub>2</sub>SO): δ 27.9, 28.1, 35.0, 35.8, 51.6, 51.8, 57.4, 57.6, 74.8, 75.7, 79.3, 115.5, 115.8, 120.2, 120.3, 122.0, 122.1, 129.9, 132.2, 153.0, 153.4, 156.4, 166.8, 173.2, 173.6. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.0. TLC: <i>R</i><sub>f</sub> 0.18 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>25</sup><sub>D</sub> −23 (<i>c</i> = 0.45; Abs EtOH).</div></div><div id="sec4_1_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(4-carboxyphenoxy)pyrrolidine-2-carboxylic Acid (<b>10</b>)</h4><div class="NLM_p last">Commercially available <b>7</b> (140 mg, 0.57 mmol, 1.00 equiv), methyl 2-hydroxybenzoate (108 mg, 0.71 mmol, 1.24 equiv), PPh<sub>3</sub> (140 mg, 0.69 mmol, 1.21 equiv), and DIAD (181 mg, 0.69 mmol, 1.21 equiv) were weighed out in a dry flask and dissolved in dry THF (4 mL) under an atmosphere of nitrogen. The mixture was left to stir at rt for 18 h. To the mixture was added Et<sub>2</sub>O/THF (1:1, 20 mL) and washed with 2 M NaOH (2 × 10 mL) and brine (15 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–24% EtOAc in toluene). The mixture was dissolved in THF (6 mL), cooled to 0 °C, and 1 M LiOH (6 mL) added dropwise over the course of 2 min followed by addition of 4 M NaOH (1 mL) over the course of 2 min. The mixture was left to stir at 0 °C for 15 min and at rt for 20 h. The mixture was added H<sub>2</sub>O (20 mL) and the aqueous phase washed with Et<sub>2</sub>O (2 × 20 mL). To the aqueous phase was added conc HCl (pH = 1–2) and extracted with a mixture of Et<sub>2</sub>O and THF (1:1, 3 × 20 mL). The pooled organic phases was washed with brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–100% EtOAc in heptanes containing 2% AcOH) to yield diacid <b>10</b> (106 mg, 53%) as a colorless oil. <sup>1</sup>H NMR (300 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 1.34/1.35 (9H, 2 × s), 2.15–2.30 (1H, m), 2.35–2.54 (1H, m), 3.54 (1H, d, <i>J</i> = 13 Hz), 3.64 (1H, td, <i>J</i> = 11, 3 Hz), 4.19 (1H, q, <i>J</i> = 8 Hz), 5.08 (1H, br s), 6.99 (2H, d, <i>J</i> = 9 Hz), 7.86 (2H, d, <i>J</i> = 9 Hz). <sup>13</sup>C NMR (75 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 27.9, 28.1, 35.1, 35.9, 51.7, 52.0, 57.4, 57.6, 74.9, 75.7, 79.3, 115.0, 123.3, 131.4, 153.0, 153.4, 160.1, 166.8, 173.2, 173.7. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 252.1, found 252.1. HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.19 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>25</sup><sub>D</sub> −22 (<i>c</i> = 0.45; Abs EtOH).</div></div><div id="sec4_1_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (2<i>S</i>,4<i>S</i>)-1-<i>tert</i>-Butyl 2-methyl 4-((methylsulfonyl)oxy)pyrrolidine-1,2-dicarboxylate (<b>11</b>)</h4><div class="NLM_p last">Commercially available alcohol <b>7</b> (550 mg, 2.24 mmol, 1.00 equiv) was dissolved in dry THF (5 mL) and added TEA (1.70 mL, 3.95 mmol, 1.76 equiv) under an atmosphere of nitrogen. The mixture was cooled to 0 °C, dropwise added mesyl chloride (0.30 mL, 3.88 mmol, 1.73 equiv) over the course of 10 min, and the mixture was stirred at 0 °C for an additional 5 h. The mixture was added H<sub>2</sub>O (10 mL) and extracted with EtOAc (3 × 10 mL). The combined organic phases were washed with 1 M HCl (10 mL), sat. NaHCO<sub>3</sub> (15 mL), brine (10 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to yield mesylate <b>11</b> (714 mg, 98%) as a slightly, white solid. <sup>1</sup>H NMR (300 MHz, (CDCl<sub>3</sub>): δ 1.44, 1.49 (9H, 2<i>s</i>), 2.46–2.58 (2H, m), 3.02 (3H, s), 3.75–3.84 (2H, m), 3.76 (3H, s), 4.42 (0.53H, dd, <i>J</i> = 8, 3 Hz), 4.53 (0.46H, dd, <i>J</i> = 7, 5 Hz), 5.24 (1H, m).</div></div><div id="sec4_1_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> (2<i>S</i>,4<i>R</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-((2-carboxyphenyl)thio)pyrrolidine-2-carboxylic Acid (<b>12</b>)</h4><div class="NLM_p last">Methyl 2-mercaptobenzoate (307 mg, 183 mmol, 1.63 equiv) was dissolved in DMF (7 mL) and K<sub>2</sub>CO<sub>3</sub> (265 mg, 1.92 mmol, 1.71 equiv) added. Mesylate <b>11</b> (345 mg, 1.12 mmol, 1.00 equiv) was dropwise added to the mixture over the course of 5 min, which was subsequently heated to 50 °C and stirred under an atmosphere of nitrogen for 18 h. The mixture was added 10% brine (75 mL) and extracted with Et<sub>2</sub>O (2 × 75 mL). The combined organic phases were washed with 0.5 M NaOH (2 × 50 mL) and brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude oil was dissolved in THF (7.5 mL), cooled to 0 °C, and 2 M LiOH (20 mL) added dropwise over the course of 5 min. After an additional 15 min at 0 °C, the mixture was left to stir for 18 h at rt. To the mixture was added H<sub>2</sub>O (75 mL) and was washed with Et<sub>2</sub>O (2 × 50 mL). The aqueous phase was acidified (pH ≈ 1) using conc HCl and extracted with a combination of Et<sub>2</sub>O and THF (1:1) (3 × 70 mL). The combined organic phases was washed with brine, dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified using DCVC (0–75% EtOAc in toluene and 2% AcOH), yielding thioether <b>12</b> (304 mg, 78%) as a clear, colorless oil. <sup>1</sup>H NMR (300 MHz, (CDCl<sub>3</sub>): δ 1.43/1.47 (9H, 2<i>s</i>), 2.26–2.68 (2H, m), 3.40–3.56 (1H, m), 3.96–4.06 (1.5H, m), 4.17 (0.5H, dd, <i>J</i> = 11, 7 Hz), 4.46 (0.5H, dd, <i>J</i> = 8, 5 Hz), 4.57 (0.5H, dd, <i>J</i> = 8, 3 Hz), 7.13–7.25 (1H, m), 7.33 (1H, t, <i>J</i> = 10 Hz), 7.48 (1H, q, <i>J</i> = 8 Hz), 8.07 (1H, dd, <i>J</i> = 8, 3 Hz), 12.0 (2H, bs). <sup>13</sup>C NMR (75 MHz, (CDCl<sub>3</sub>): δ 28.4, 28.5, 35.5, 36.5, 40.5, 41.0, 52.4, 52.5, 58.4, 58.5, 81.3, 81.7, 124.7, 124.8, 126.6, 126.8, 126.9, 132.5, 133.5, 140.7, 140.8, 153.5, 155.0, 171.3, 176.5, 178.4. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 368.1, found 268.1 (-Boc). HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.19 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>22</sup><sub>D</sub> −23 (<i>c</i> = 0.49; abs EtOH).</div></div><div id="sec4_1_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> (2<i>S</i>,4<i>S</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(2-carboxyphenoxy)pyrrolidine-2-carboxylic Acid (<b>15</b>)</h4><div class="NLM_p last">Methyl ester <b>14</b> (303 mg, 1.24 mmol, 1.00 equiv), methyl 2-hydroxybenzoate (236 mg, 1.55 mmol, 1.26 equiv), PPh<sub>3</sub> (395 mg, 1.51 mmol, 1.22 equiv), and DIAD (325 mg, 1.61 mmol, 1.30 equiv) were weighed out in a dry flask and dissolved in dry THF (6 mL) under an atmosphere of nitrogen. The mixture was left to stir at rt for 18 h. To the mixture was added a combination of Et<sub>2</sub>O and THF (1:1, 40 mL) and washed with 2 M NaOH (2 × 20 mL) and brine (20 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–30% EtOAc in toluene). The diester was dissolved in THF (6 mL), cooled to 0 °C, and 2 M LiOH (12 mL) added dropwise over the course of 5 min. The mixture was left to stir at 0 °C for 15 min and at rt for 20 h. The mixture was added H<sub>2</sub>O (40 mL) and the aqueous phase washed with Et<sub>2</sub>O (2 × 40 mL). The aqueous phase was added conc HCl (pH ≈ 0) and extracted with a mixture of Et<sub>2</sub>O and THF (1:1, 3 × 40 mL). The combined organic phases was washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to yield diacid <b>15</b> (333 mg, 77%) as a clear, colorless oil. <sup>1</sup>H NMR (300 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 1.36/1.41 (9H, 2<i>s</i>), 2.10–2.18 (1H, m), 2.59–2.74 (1H, m), 3.38 (1H, dd, <i>J</i> = 11, 4 Hz), 3.79–3.87 (1H, m), 4.19–4.27 (1H, m), 4.93–5.01 (1H, m), 7.00–7.09 (2H, m), 7.44–7.50 (1H, m), 7.62 (1H, dd, <i>J</i> = 8, 2 Hz). <sup>13</sup>C NMR (75 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 28.0, 28.2, 34.7, 35.5, 51.3, 51.7, 57.2, 57.4, 75.8, 76.7, 78.9, 79.1, 115.5, 121.0, 122.5, 130.9, 132.8, 152.7, 153.2, 155.7, 166.8, 172.7, 173.0. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 352.1, found 252.1 (-Boc). HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.20 (EtOAc:heptanes, 2:1 + 2% AcOH).</div></div><div id="sec4_1_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> (2<i>S</i>,4<i>S</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(3-carboxyphenoxy)pyrrolidine-2-carboxylic Acid (<b>16</b>)</h4><div class="NLM_p last">Methyl ester <b>14</b> (350 mg, 1.43 mmol, 1.00 equiv), methyl 3-hydroxybenzoate (270 mg, 1.77 mmol, 1.24 equiv), DIAD (351 mg, 1.74 mmol, 1.22 equiv), and PPh<sub>3</sub> (452 mg, 1.72 mmol, 1.20 equiv) were weight out in a dry flask and dissolved in dry THF (8 mL) under an atmosphere of nitrogen. The mixture was left to stir at rt for 18 h. To the mixture was added a combination of Et<sub>2</sub>O and THF (1:1, 40 mL) and washed with 2 M NaOH (2 × 20 mL) and brine (25 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–24%, EtOAc in toluene). The diester was dissolved in THF (12 mL), cooled to 0 °C, and dropwise added 1 M LiOH (12 mL) over the course of 2 min and subsequently 4 M NaOH (2 mL) over the course of 4 min. The mixture was left to stir at 0 °C for 15 min and at rt for 20 h. The mixture was added H<sub>2</sub>O (40 mL) and the aqueous phase washed with Et<sub>2</sub>O (2 × 40 mL). The aqueous phase was added conc HCl (pH ≈ 1) and extracted with a mixture of Et<sub>2</sub>O and THF (1:1, 3 × 40 mL). The pooled organic phases were washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–100% EtOAc in heptanes containing 2% AcOH) to yield diacid <b>16</b> (343 mg, 68%) as a clear, colorless oil. <sup>1</sup>H NMR (300 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 1.36/1.40 (9H, 2<i>s</i>), 2.15–2.26 (1H, m), 2.50–2.65 (1H, m), 3.41 (1H, dd, <i>J</i> = 12, 6 Hz), 3.65–3.77 (1H, m), 4.28 (1H, m), 5.05–5.15 (1H, m), 7.07–7.14 (1H, m), 7.32–7.35 (1H, m), 7.40 (1H, t, <i>J</i> = 8 Hz), 7.52 (1H, dm, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (75 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 28.0, 28.2, 34.7, 35.5, 51.4, 51.8, 57.2, 57.4, 74.5, 75.6, 78.8, 79.0, 116.0, 119.8, 121.8, 129.8, 132.2, 153.1, 153.2, 156.3, 166.8, 172.7, 172.9. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 352.1, found 252.1 (-Boc). HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.27 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>25</sup><sub>D</sub> −17 (<i>c</i> = 0.49; abs EtOH).</div></div><div id="sec4_1_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> (2<i>S</i>,4<i>S</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-(4-carboxyphenoxy)pyrrolidine-2-carboxylic Acid (<b>17</b>)</h4><div class="NLM_p last">Methyl ester <b>14</b> (305 mg, 1.24 mmol, 1.00 equiv), methyl 4-hydroxybenzoate (246 mg, 1.62 mmol, 1.30 equiv), PPh<sub>3</sub> (395 mg, 1.51 mmol, 1.21 equiv), and DIAD (317 mg, 1.57 mmol, 1.26 equiv) were weighed out in a dry flask and dissolved in dry THF (6 mL) under an atmosphere of nitrogen. The mixture was left to stir at rt for 18 h. To the mixture was added a combination of Et<sub>2</sub>O and THF (1:1, 40 mL) and washed with 2 M NaOH (2 × 20 mL) and brine (25 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using DCVC (0–30% EtOAc in toluene). The diester was dissolved in THF (6 mL), cooled to 0 °C, and 2 M LiOH (12 mL) added dropwise over the course of 5 min. The mixture was left to stir at 0 °C for 15 min and at rt for 20 h. To the mixture was added H<sub>2</sub>O (40 mL) and the aqueous phase washed with Et<sub>2</sub>O (2 × 40 mL). The aqueous phase was added conc HCl (pH = 0) and extracted with a mixture of Et<sub>2</sub>O and THF (1:1, 3 × 40 mL). The combined organic phases was washed with brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to yield diacid <b>17</b> (311 mg, 71%) as a clear, colorless oil. <sup>1</sup>H NMR (300 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 1.36/1.40 (9H, 2<i>s</i>), 2.17–2.22 (2H, m), 2.51–2.66 (1H, m), 3.41 (1H, dd, <i>J</i> = 12, 6 Hz), 3.69–3.78 (1H, m), 4.24–4.32 (1H, m), 5.08–5.11 (1H, m), 6.92 (2H, dd, <i>J</i> = 9, 3 Hz), 7.86 (2H, d, <i>J</i> = 9 Hz). <sup>13</sup>C NMR (75 MHz, ((CD<sub>3</sub>)<sub>2</sub>SO): δ 28.0, 28.2, 30.5, 34.7, 35.5, 51.5, 51.8, 57.2, 57.4, 74.6, 75.7, 78.9, 79.0, 115.0, 123.2, 131.3, 153.0, 153.1, 166.8, 172.6, 172.9. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 352.1, found 252.1 (-Boc). HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.23 (EtOAc:heptanes, 2:1 + 2% AcOH).</div></div><div id="sec4_1_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-((methylsulfonyl)oxy)pyrrolidine-1,2-dicarboxylate (<b>18</b>)</h4><div class="NLM_p last">Alcohol <b>14</b> (2.11 g, 8.60 mmol, 1.00 equiv) was dissolved in dry THF (15 mL) and TEA (1.70 mL, 12.2 mmol, 1.42 equiv) added under nitrogen. The mixture was cooled to 0 °C and mesyl chloride (1.00 mL, 12.9 mmol, 1.50 equiv) added dropwise over the course of 15 min, and the mixture was stirred for additional 5 h. To the mixture was added H<sub>2</sub>O (30 mL) and extracted with EtOAc (3 × 25 mL). The combined organic phases was washed with 1 M HCl (30 mL), satd NaHCO<sub>3</sub> (40 mL), and brine (30 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo to yield mesylate <b>18</b> (2.50 g, 90%) as a white solid. <sup>1</sup>H NMR (300 MHz, (CDCl<sub>3</sub>): 1.43, 1.47 (9H, 2<i>s</i>), 2.25 (0.40H, dd, <i>J</i> = 14, 5 Hz), 2.28 (0.60, dd, <i>J</i> = 14, 5 Hz), 2.58 (0.40, ddd, <i>J</i> = 14, 8, 3 Hz), 2.67 (0.60, tdd, <i>J</i> = 14, 8, 2 Hz) 3.06 (3H, s), 3.72–3.89 (5H, m), 4.40 (0.60H, t, <i>J</i> = 8 Hz), 4.46 (0.40H, t, <i>J</i> = 8 Hz), 5.26 (1H, m).</div></div><div id="sec4_1_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (2<i>S</i>,4<i>S</i>)-1-(<i>tert</i>-Butoxycarbonyl)-4-((2-carboxyphenyl)thio)pyrrolidine-2-carboxylic Acid (<b>19</b>)</h4><div class="NLM_p last">Methyl 2-mercaptobenzoate (180 mg, 1.07 mmol, 1.66 equiv) was dissolved in DMF (4 mL) and K<sub>2</sub>CO<sub>3</sub> (149 mg, 1.08 mmol, 1.67 equiv) added. Mesylate <b>18</b> (200 mg, 0.65 mmol, 1.00 equiv) was dropwise added to the mixture, which was subsequently heated to 50 °C and stirred under nitrogen for 18 h. To the mixture was added 10% brine (40 mL) and extracted with Et<sub>2</sub>O (2 × 50 mL). The combined organic phases was washed with 0.5 M NaOH (2 × 30 mL) and brine (50 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The oil was purified using DCVC (0–22% EtOAc in toluene). The oil was dissolved in THF (5 mL), cooled to 0 °C, and 2 M LiOH (10 mL) added dropwise over the course of 5 min. After an additional 15 min at 0 °C, the mixture was left to stir for 18 h at rt. To the mixture was added H<sub>2</sub>O (50 mL) and was washed with Et<sub>2</sub>O (2 × 25 mL). The aqueous phase was acidified (pH ≈ 1) using conc HCl and extracted with a combination of Et<sub>2</sub>O and THF (1:1) (3 × 25 mL). The combined organic phases was washed with brine (25 mL), dried over MgSO<sub>4</sub>, filtered and concentrated in vacuo. The crude product was purified using DCVC (0–75% EtOAc in toluene added 2% AcOH), yielding thioether <b>19</b> (153 mg, 67%) as a white solid. <sup>1</sup>H NMR (300 MHz, (CDCl<sub>3</sub>): δ 1.47/1.54 (9H, 2<i>s</i>), 2.44–2.66 (2H, m), 3.60–3.78 (1H, m), 3.89–4.03 (1H, m), 4.09–4.16 (1H, m), 4.44 (0.5H, d, <i>J</i> = 8 Hz), 4.55 (0.5H, d, <i>J</i> = 8 Hz), 7.20 (1H, d, <i>J</i> = 8 Hz), 7.33 (1H, d, <i>J</i> = 8 Hz), 7.48 (2H, t, <i>J</i> = 8 Hz). <sup>13</sup>C NMR (400 MHz, (CDCl<sub>3</sub>): δ 28.3, 28.5, 34.9, 36.0, 42.0, 50.9, 57.8, 58.0, 80.7, 81.0, 124.7, 124.8, 126.9, 127.1, 128.3, 128.5, 132.5, 132.6, 132.9, 133.1, 135.8, 140.3, 140.8, 153.9, 154.7, 171.7, 179.1, 179.7. LCMS: <i>m</i>/<i>z</i> [M + H]<sup>+</sup>: calcd 368.1, found 268.1 (-Boc). HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.18 (EtOAc:heptanes, 2:1 + 2% AcOH). OR: [α]<sup>22</sup><sub>D</sub> −28 (<i>c</i> = 0.29; abs EtOH).</div></div><div id="sec4_1_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Methyl 2-(2-Hydroxyphenyl) Acetate (<b>21</b>)</h4><div class="NLM_p last">2-Hydroxyphenylacetic acid (1.00 g, 6.57 mmol, 1.00 equiv) was dissolved in MeOH (20 mL). H<sub>2</sub>SO<sub>4</sub> (0.55 mL, 10.3 mmol, 1.57 equiv) was added dropwise, and the reaction mixture was heated to reflux (3 h) under an atmosphere of nitrogen. The mixture was concentrated in vacuo and the residue dissolved in water (68 mL) and extracted with EtOAc (3 × 10 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified using flash chromatography (heptane:EtOAc, 1:1) to give ester <b>21</b> as a white solid (745 mg, 4.48 mmol, 68%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.69 (s, 2H), 3.75 (s, 3H), 6.88 (ddd, <i>J</i> = 2, 7 Hz, 1H), 6.94 (dd, <i>J</i> = 1, 8 Hz, 1H), 7.10 (dd, <i>J</i> = 2, 8 Hz, 1H), 7.20 (ddd, <i>J</i> = 2, 8 Hz, 1H), 7.30 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 37.7, 52.7, 117.7, 120.5, 120.9, 129.3, 131.0, 155.2, 174.3. LC-MS: <i>m</i>/<i>z</i> (M + H<sup>+</sup>) calcd 167.1, found 167.1.TLC: <i>R</i><sub>f</sub> 0.65 (heptane/EtOAc, 1:1). HPLC: >99% (254 nm). Melting point: 72.4–73.7 °C. Characterization is consistent with the literature.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_1_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Methyl 2-(3-Hydroxyphenyl) Acetate (<b>22</b>)</h4><div class="NLM_p last">3-Hydroxyphenyl acetic acid (3.00 g, 19.7 mmol, 1.00 equiv) was dissolved in MeOH (66 mL). H<sub>2</sub>SO<sub>4</sub> (1.6 mL, 30.02 mmol, 1.52 equiv) was added slowly, and the reaction mixture was heated to reflux overnight (18 h). The mixture was concentrated in vacuo and the residue dissolved in water (68 mL) and extracted with EtOAc (4 × 33 mL). The combined organic fractions were dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The crude product was purified using flash chromatography (heptane:EtOAc, 1:1) to yield ester <b>22</b> as a brown/orange oil (2.7 g, 82%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.58 (s, 2H), 3.70 (s, 3H), 6.70–6.80 (m, 2H), 6.82 (ddd, <i>J</i> = 1, 2, 7 Hz, 1H), 7.18 (dd, <i>J</i> = 8 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) 41.1, 52.2, 114.3, 116.2, 121.5, 129.8, 135.4, 155.9, 172.4. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 167.1, found 167.1. TLC: <i>R</i><sub>f</sub> 0.46 (heptane:EtOAc, 1:1). Characterization is consistent with the literature.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a></div></div><div id="sec4_1_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Methyl 3-Hydroxy-2-naphthoate (<b>23</b>)</h4><div class="NLM_p last">Carried out as described for <b>22</b>, starting from 3-hydroxy-2-naphthoic acid (1.00 g, 5.33 mmol, 1.00 equiv), MeOH (20 mL), conc H<sub>2</sub>SO<sub>4</sub> (0.40 mL, 7.50 mmol, 1.41 equiv), and reflux for 44 h. The mixture was purified using flash chromatography (heptane:EtOAc, 4:1) to yield ester <b>12</b> as a yellow solid (821 mg, 77%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.03 (s, 3H), 7.32 (s, 1H), 7.32 (ddd, <i>J</i> = 1.20, 6.76, 8.13 Hz, 1H), 7.50 (ddd, <i>J</i> = 1, 7, 8 Hz, 1H), 7.69 (dt, <i>J</i> = 1, 8 Hz, 1H), 7.80 (dd, <i>J</i> = 1, 8 Hz, 1H), 8.50 (d, <i>J</i> = 1 Hz, 1H), 10.42 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 52.7, 78.3, 111.9, 124.1, 126.5, 127.2, 129.3, 129.4, 132.6, 138.1, 156.5, 170.4. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 203.1, found 203.0. HPLC: 99% (254 nm). Melting point: 71.4–73.4 °C.</div></div><div id="sec4_1_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Methyl 1-Hydroxy-2-naphthoate (<b>24</b>)</h4><div class="NLM_p last">Carried out as described for <b>22</b>, starting from 1-hydroxy-2-naphthoic acid (1.01 g, 5.31 mmol, 1.00 equiv), MeOH (20 mL), conc H<sub>2</sub>SO<sub>4</sub> (0.40 mL, 7.50 mmol, 1.41 equiv), and heated to reflux for 44 h. Mixture was purified using flash chromatography (heptane:EtOAc, 99:1) to give the ester <b>24</b> as a white solid (342 mg, 32%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.00 (s, 3H), 7.29 (dd, <i>J</i> = 1, 9 Hz, 1H), 7.53 (ddd, <i>J</i> = 1, 7, 8 Hz, 1H), 7.61 (ddd, <i>J</i> = 1, 7, 8 Hz, 1H), 7.77 (dd, <i>J</i> = 2, 8 Hz, 1H), 7.78 (dd, <i>J</i> = 2, 8 Hz, 1H), 8.42 (ddt, <i>J</i> = 1, 2, 8 Hz, 1H), 11.98 (s, 1H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 52.4, 105.7, 118.8, 124.0, 124.4, 124.9, 125.9, 127.6, 129.5, 137.4, 161.1, 171.6. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 203.1, found 203.0. HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.37 (heptane:EtOAc, 1:1). Melting point: 78.3–79.4 °C.</div></div><div id="sec4_1_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Methyl 2-Hydroxy-1-naphthoate (<b>25</b>)</h4><div class="NLM_p last">2-Hydroxy-1-naphthoic acid (1.01 g, 5.39 mmol, 1.00 equiv) and KHCO<sub>3</sub> (640 mg, 6.39 mmol, 1.19 equiv) were weighed out and dissolved in anhydrous DMF (3.5 mL). CH<sub>3</sub>I (0.50 mL, 1.13 g, 7.96 mmol, 1.48 equiv) was added dropwise, while the reaction mixture was stirred under an atmosphere of nitrogen. The reaction mixture was left for 21 h at rt, after which it was quenched using H<sub>2</sub>O (10 mL). The mixture was extracted using EtOAc (2 × 10 mL), and the combined organic phases were washed with satd NaHCO<sub>3</sub> (0.5 mL), H<sub>2</sub>O (10 mL), and brine (0.5 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1 + 2% AcOH) to yield ester <b>25</b> as a white solid (444 mg, 41%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 4.12 (s, 3H), 7.17 (d, <i>J</i> = 9 Hz, 1H), 7.37 (ddd, <i>J</i> = 1, 7, 8 Hz, 1H), 7.56 (ddd, <i>J</i> = 1, 7, 9 Hz, 1H), 7.72–7.79 (m, 1H), 7.90 (d, <i>J</i> = 9 Hz, 1H), 8.74 (d, <i>J</i> = 8 Hz, 1H), 12.25 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 52.4, 104.7, 119.3, 123.7, 125.3, 128.5, 128.7, 129.1, 131.8, 136.9, 164.4, 172.8. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 203.1, found 203.0. HPLC: 87% (254 nm). TLC: <i>R</i><sub>f</sub> 0.76 (heptane:EtOAc, 1:1 + 2% AcOH). Melting point: 79.4–80.5 °C.</div></div><div id="sec4_1_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> Dimethyl 4-Hydroxyisophthalate (<b>26</b>)</h4><div class="NLM_p last">Carried out as described for <b>27</b> starting from 4-hydroxyisophthalic acid (1.01 mg, 5.52 mmol, 1.00 equiv), MeOH (15 mL), and conc H<sub>2</sub>SO<sub>4</sub> (0.2 mL, 3.75 mmol, 0.68 equiv) and heated to reflux for 18 h. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1 + 2% AcOH) to yield ester <b>26</b> as a white semisolid (504 mg, 44%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 3.91 (s, 3H), 3.99 (s, 3H), 7.02 (d, <i>J</i> = 9 Hz, 1H), 8.12 (dd, <i>J</i> = 2, 9 Hz, 1H), 8.57 (d, <i>J</i> = 2 Hz, 1H), 11.19 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 52.3, 52.8, 112.3, 118.0, 121.6, 132.7, 136.8, 150.1, 165.3, 166.2. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 211.1, found 211.0. HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.70 (heptane:EtOAc, 1:1 + 2% AcOH).</div></div><div id="sec4_1_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> Methyl 5-Methylsalicylate (<b>27</b>)</h4><div class="NLM_p last">5-Methylsalicylic acid (1.00 g, 6.59 mmol, 1.00 equiv) was dissolved in MeOH (15 mL). H<sub>2</sub>SO<sub>4</sub> (0.2 mL, 3.75 mmol, 0.57 equiv) was added dropwise, and the reaction mixture was heated to reflux for 24 h. The mixture was concentrated in vacuo. The residue was dissolved in EtOAc (10 mL) and washed with satd NaHCO<sub>3</sub> (2 × 10 mL) and brine (10 mL), dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1 + 2.5% AcOH) to yield ester <b>27</b> as a clear yellow/orange oil (621 mg, 57%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 2.28 (s, 3H), 3.94 (s, 3H), 6.88 (d, <i>J</i> = 8 Hz, 1H), 7.23–7.30 (m, 1H), 7.63 (d, <i>J</i> = 2 Hz, 1H), 10.54 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 20.4, 52.2, 111.9, 117.3, 128.3, 129.6, 136.7, 159.5, 170.6. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 167.1, found 167.1. HPLC: >99% (254 nm). TLC: <i>R</i><sub>f</sub> 0.76 (heptane:EtOAc, 1:1 + 2% AcOH).</div></div><div id="sec4_1_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> Methyl 3-Methylsalicylate (<b>28</b>)</h4><div class="NLM_p last">Carried out as described for <b>27</b> with 3-methylsalicylic acid (<b>8</b>) (1.03 g, 6.74 mmol, 1.00 equiv), MeOH (20 mL), and conc H<sub>2</sub>SO<sub>4</sub> (0.4 mL, 7.50 mmol, 1.11 equiv) and heated to reflux for 4 days. The mixture was purified by flash chromatography (heptane:EtOAc, 4:1) to yield ester <b>28</b> as a clear oil (534 mg, 49%). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 2.27 (s, 3H), 3.94 (s, 3H), 6.78 (d, <i>J</i> = 8 Hz, 1H), 7.32 (dd, <i>J</i> = 2, 7 Hz, 1H), 7.69 (dd, <i>J</i> = 2, 8 Hz, 1H), 11.01 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 15.8, 52.4, 111.8, 118.6, 126.8, 127.5, 136.6, 160.2, 171.2. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 167.1, found 167.1. TLC: <i>R</i><sub>f</sub> 0.46 (heptane:EtOAc, 1:1).</div></div><div id="sec4_1_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-(2-(2-Methoxy-2-oxoethyl)phenoxy)pyrrolidine-1,2-dicarboxylate (<b>29</b>)</h4><div class="NLM_p last">In a dry flask <i>N</i>-Boc-<i>cis</i>-4-hydroxy-<span class="smallcaps smallerCapital">l</span>-proline methyl ester (201 mg, 0.82 mmol, 1.00 equiv), methyl 2- hydroxy-phenylacetate (<b>21</b>) (280 mg, 1.69 mmol, 1.00 equiv), and Ph<sub>3</sub>P (590 mg, 2.25 mmol, 1.33 eqiuv) were dissolved in anhydrous THF (2.7 mL) under an atmosphere of argon and cooled to 0 °C. DIAD (0.32 mL, 1.63 mmol, 0.96 equiv) was added dropwise, and the mixture was left to stir at rt over the course of 20 h. The reaction mixture was diluted with Et<sub>2</sub>O:THF (1:1, 40 mL) and quenched/washed with satd Na<sub>2</sub>CO<sub>3</sub>:H<sub>2</sub>O (1:1, 2 × 20 mL) and brine (25 mL). The organic phase was dried over MgSO<sub>4</sub>, filtered, and concentrated in vacuo. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1 + 2% AcOH) to yield proline <b>29</b> (97 mg, 30%) as a clear, off-white semisolid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.31–1.44 (br s, 9H), 2.14 (ddd, <i>J</i> = 5, 8, 13 Hz, 1H), 2.39–2.53 (m, 1H), 3.42–3.58 (m, 1H), 3.61 (s, 3H), 3.64–3.75 (m, 3H), 3.68 (s, 3H), 4.29–4.44 (m, 1H), 4.84–4.98 (m, 1H), 6.29 (s, 1H), 6.72 (d, <i>J</i> = 8 Hz, 1H), 6.83–6.93 (m, 1H), 7.08–7.21 (m, 3H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) (rotamers) δ 22.1, 28.4, 28.5, 35.9, 36.3, 36.8, 37.8, 52.0, 52.3, 57.8, 58.2, 70.3, 74.8, 75.4, 77.3, 80.6, 112.1, 112.3, 117.7, 121.5, 124.3, 128.7, 129.3, 131.2, 131.6, 154.9, 156.5, 172.2, 173.4, 173.6. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 294.1, found 294.0. TLC: <i>R</i><sub>f</sub> 0.48 (heptane:EtOAc, 1:1 + 2% AcOH).</div></div><div id="sec4_1_1_44" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-(3-(2-Methoxy-2-oxoethyl)phenoxy)pyrrolidine-1,2-dicarboxylate (<b>30</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (500 mg, 2.04 mmol, 1.00 equiv), Ph<sub>3</sub>P (644 mg, 2.45 mmol, 1.20 equiv), methyl-3-hydroxy-phenylacetate (<b>22</b>) (407 mg, 2.45 mmol, 1.20 equiv), DIAD (507 mg, 2.51 mmol, 1.23 equiv), and anhydous THF (14.5 mL) and stirring for 43 h under an atmosphere of argon. The mixture was concentrated on Celite and purified using flash chromatography (0–100% EtOAc in heptanes) to yield proline <b>30</b> as a white semisolid (142 mg, 18% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.44 (s, 9H), 2.14–2.28 (m, 1H), 2.48–2.53 (m, 1H), 3.59 (br s, 2H), 3.70 (s, 3H), 3.75 (s, 3H), 3.79 (d, <i>J</i> = 3 Hz, 2H), 4.42 (t, <i>J</i> = 7.83 Hz, 0.5H), 4.48 (t, <i>J</i> = 7.83 Hz, 0.5H), 4.90 (br s, 1H), 6.75 (d, <i>J</i> = 8 Hz, 1H), 6.83 (s, 1H), 6.89 (d, <i>J</i> = 7 Hz, 1H), 7.21–7.25 (dd, <i>J</i> = 7 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) δ 21.8, 28.1, 28.2, 35.5, 36.4, 41.0, 51.8, 51.9, 52.2, 57.5, 57.9, 74.5, 80.3, 113.9, 116.5, 122.2, 129.6, 135.5, 156.9, 171.6, 173.2. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 294.1, found 294.0 (-Boc). TLC: <i>R</i><sub>f</sub> 0.35 (heptane:EtOAc, 1:1 + 2% AcOH).</div></div><div id="sec4_1_1_45" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-((3-(Methoxycarbonyl) naphthalen-2-yl)oxy) pyrrolidine-1,2-dicarboxylate (<b>31</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (201 mg, 0.82 mmol, 1.00 equiv), methyl 3-hydroxy-2-naphthoate (<b>23</b>) (198 mg, 0.98 mmol, 1,20 equiv), Ph<sub>3</sub>P (257 mg, 0.98 mmol, 1.20 equiv), anhydrous THF (7 mL), and DIAD (0.2 mL, 205 mg, 1.02 mmol, 1.24 equiv) and stirring for 26 h under an atmosphere of argon at rt. The mixture was concentrated on Celite and purified using flash chromatography (heptane:EtOAc, 2:1) to yield proline <b>31</b> (81 mg, 23% yield) as a semisolid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.43 (s, 9H), 2.25–2.32 (m, 1H), 2.60–2.72 (m, 1H), 3.76 (br s, 3H), 3.78–3.87 (m, 1H), 3.92 (s, 3H), 4.51 (t, <i>J</i> = 8 Hz, 0.5H), 4.55 (t, <i>J</i> = 8 Hz, 0.5H), 5.09 (br d, <i>J</i> = 14 Hz, 1H), 7.15 (s, 1H), 7.34–7.44 (m, 1H), 7.52 (m, 1H), 7.71 (t, <i>J</i> = 9 Hz, 1H), 7.82 (dd, <i>J</i> = 5, 8 Hz, 1H), 8.30 (s, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) (rotamers) δ 14.2, 14.3, 21.1, 22.1, 22.8, 28.4, 28.5, 29.1, 32.0, 35.7, 36.8, 51.8, 52.2, 52.2, 52.3, 52.4, 57.9, 58.3, 60.5, 70.1, 76.1, 80.5, 110.3, 110.4, 123.0, 125.0, 126.6, 128.2, 128.7, 128.8, 133.3, 135.8, 152.9, 153.9, 156.5, 166.8, 171.3, 173.6. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 330.1, found 330.1. TLC: <i>R</i><sub>f</sub> 0.26 (heptane:EtOAc, 2:1).</div></div><div id="sec4_1_1_46" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-((2-(Methoxycarbonyl)naphthalen-1-yl)oxy)pyrrolidine-1,2-dicarboxylate (<b>32</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (203 mg, 0.83 mmol, 1.00 equiv), methyl 1-hydroxy-2-naphthoate (<b>24</b>) (199 mg, 0.98 mmol, 1.18 equiv), Ph<sub>3</sub>P (257 mg, 0.98 mmol, 1.18 equiv), and anhydrous THF (6 mL), DIAD (0.2 mL, 205 mg, 1.02 mmol, 1.23 equiv) and stirring for 26 h under an atmosphere of argon at rt. The mixture was purified using flash chromatography (heptane/EtOAc, 1:1) to yield a sticky oil (276 mg, 79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.45 (s, 9H), 2.08–2.23 (m, 1H), 2.68–2.82 (m, 1H), 3.54 (ddd, <i>J</i> = 4, 7, 12 Hz, 1H), 3.71 (s, 3H), 3.73–3.86 (m, 1H), 3.96 (br s, 3H), 4.64 (t, <i>J</i> = 8 Hz, 0.5H), 4.72 (t, <i>J</i> = 8 Hz, 0.5H), 4.95–4.99 (m, 1H), 7.48–7.67 (m, 3H), 7.80–7.89 (m, 2H), 8.15 (d, <i>J</i> = 8 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) (rotamers) δ 14.1, 21.9, 22.7, 28.3, 28.4, 31.9, 36.5, 37.4, 51.7, 52.1, 52.2, 52.3, 58.0, 58.4, 70.1, 77.2, 80.3, 82.9, 83.9, 123.4, 123.6, 124.0, 126.7, 126.8, 127.7, 128.6, 136.6, 166.5, 173.3, 173.4. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 330.1, found 330.1. TLC: <i>R</i><sub>f</sub> 0.62 (heptane:EtOAc, 1:1).</div></div><div id="sec4_1_1_47" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> (2<i>S,</i>4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-(3-(2-Methoxy-2-oxoethyl)phenoxy)pyrrolidine-1,2-dicarboxylate (<b>33</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (208 mg, 0.85 mmol, 1.00 equiv), methyl 2-hydroxy-1-naphthoate (<b>25</b>) (203 mg, 1.00 mmol, 1.18 equiv), Ph<sub>3</sub>P (258 mg, 0.98 mmol, 1.15 equiv), anhydrous THF (6 mL), and DIAD (0.20 mL, 205 mg, 1.02 mmol, 1.20 equiv) and stirring for 18 h under an atmosphere of argon at rt. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1 + 2% AcOH) to yield proline <b>33</b> (168 mg, 48% yield) as a sticky solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.44 (s, 9H), 2.19 (m, 1H), 2.53–2.63 (m, 1H), 3.72 (s, 3H), 3.75–3.80 (m, 1H), 4.00 (s, 3H), 4.45 (t, <i>J</i> = 8 Hz, 0.5H), 4.50 (t, <i>J</i> = 8 Hz, 0.5H), 5.06–5.13 (m, 1H), 7.20 (dd, <i>J</i> = 2, 9 Hz, 1H), 7.41 (ddt, <i>J</i> = 2, 7, 8 Hz, 1H), 7.51 (ddt, <i>J</i> = 2, 7, 8 Hz, 1H), 7.78 (m, 2H), 7.89 (dd, <i>J</i> = 4, 9 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) (rotamers) δ 14.1, 20.5, 21.9, 22.7, 28.3, 28.3, 29.0, 31.9, 35.8, 36.7, 52.0, 52.1, 52.3, 52.3, 52.5, 57.6, 58.0, 80.5, 115.5, 115.5, 120.1, 124.0, 124.9, 127.8, 127.9, 128.1, 129.2, 131.0, 131.7, 151.7, 153.6, 154.3, 168.1, 173.4, 175.5. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 330.1, found 330.1 (- Boc). TLC: <i>R</i><sub>f</sub> 0.49 (heptane:EtOAc, 1:1 + 2% AcOH).</div></div><div id="sec4_1_1_48" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-(2,4-Bis(methoxycarbonyl)phenoxy)pyrrolidine-1,2-dicarboxylate (<b>34</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (200 mg, 0.82 mmol, 1.00 equiv), dimethyl 4-hydroxy-isophthalate (<b>26</b>) (195 mg, 0.93 mmol, 1.22 equiv), Ph<sub>3</sub>P (262 mg, 1.00 mmol, 1.22 equiv), anhydrous THF (6 mL), and DIAD (0.20 mL, 1.02 mmol, 1.24 equiv) and stirring for 4 h under an atmosphere of argon at rt. The mixture was purified using flash chromatography (heptane/EtOAc, 1:1) to yield proline <b>34</b> (189 mg, 53% yield) as a solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.38–1.45 (s, 9H), 2.26 (ddd, <i>J</i> = 5, 8, 14 Hz, 1H), 2.51–2.67 (m, 1H), 3.66 (s, 3H), 3.75 (br s, 2H), 3.92 (s, 3H), 3.94 (s, 3H), 4.48 (t, <i>J</i> = 8 Hz, 0.5H), 4.52 (t, <i>J</i> = 8 Hz, 0.5H), 5.00–5.10 (m, 1H), 6.93 (dd, <i>J</i> = 3, 9 Hz, 1H), 8.06–8.15 (m, 1H), 8.45 (d, <i>J</i> = 3 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) (rotamers) δ 28.2, 28.3, 35.6, 36.7, 51.7, 51.9, 52.2, 52.3, 52.4, 57.6, 58.0, 76.1, 76.8, 80.6, 113.9, 121.6, 123.2, 133.9, 134.7, 153.6, 159.7, 165.6, 165.8, 173.1, 173.2. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 338.1, found 338.0. TLC: <i>R</i><sub>f</sub> 0.40 (heptane:EtOAc, 1:1).</div></div><div id="sec4_1_1_49" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl-4 (2-(Methoxycarbonyl)-4-methylphenoxy)-pyrrolidine-1,2-dicarboxylate (<b>35</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (201 mg, 0.82 mmol, 1.00 equiv), <b>27</b> (165 mg, 0.99 mmol, 1.21 equiv), Ph<sub>3</sub>P (260 mg, 0.99 mmol, 1.21 equiv), anhydrous THF (6 mL), and DIAD (0.2 mL, 205 mg, 1.02 mmol, 1.24 equiv) and stirring for 4 h under an atmosphere of argon at rt. The mixture was purified using flash chromatography (heptane/EtOAc, 1:1) to yield proline <b>35</b> (91 mg, 29%) as a sticky solid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.43 (s, 9H), 2.17–2.22 (m, 1H), 2.31 (d, <i>J</i> = 3 Hz, 3H), 2.50–2.66 (m, 1H), 3.70–3.75 (m, 1H), 3.75 (s, 3H), 3.86 (s, 3H), 4.45 (t, <i>J</i> = 8 Hz, 0.5H), 4.53 (t, <i>J</i> = 8 Hz, 0.5H), 4.98 (m, 1H), 6.82 (d, <i>J</i> = 8 Hz, 1H), 7.23 (d, <i>J</i> = 7 Hz, 1H), 7.60 (s, 1H). LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 294.1, found 294.1. TLC: <i>R</i><sub>f</sub> 0.49 (heptane:EtOAc 1:1).</div></div><div id="sec4_1_1_50" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-(4-Acetyl-2-(methoxycarbonyl)phenoxy)pyrrolidine-1,2-dicarboxylate (<b>36</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (204 mg, 0.83 mmol, 1.00 equiv), methyl 5-acetylsalicylate (<b>27</b>) (199 mg, 1.03 mmol, 1.24 equiv), Ph<sub>3</sub>P (258 mg, 0.98 mmol, 1.18 equiv), anhydrous THF (6 mL), and DIAD (0.20 mL, 1.02 mmol, 1.23 equiv) and stirring for 15 h under an atmosphere of argon at rt. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1) to yield proline <b>36</b> (152 mg, 46% yield) as a semisolid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.43 (s, 9H), 2.26 (m, 1H), 2.50 (s, 3H), 2.51–2.66 (m, 1H), 3.66 (s, 3H), 3.70–3.80 (m, 1H), 3.90 (s, 3H), 4.23–4.38 (m, 1H), 4.41–4.55 (m, 1H), 5.06 (d, <i>J</i> = 9 Hz, 1H), 6.94 (d, <i>J</i> = 8 Hz, 1H), 8.01–8.10 (m, 1H), 8.36 (s, 1H). LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 322.1 found 322.1. TLC: <i>R</i><sub>f</sub> 0.21 (heptane:EtOAc, 1:1).</div></div><div id="sec4_1_1_51" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (2<i>S</i>,4<i>R</i>)-1-<i>tert</i>-Butyl 2-Methyl 4-(2-(methoxycarbonyl)-6-methylphenoxy) pyrrolidine-1,2-dicarboxylate (<b>37</b>)</h4><div class="NLM_p last">Carried out as described for <b>29</b> with <i>N</i>-Boc-<i>cis</i>-4-hydroxy-(<i>S</i>)-proline methyl ester (202 mg, 0.82 mmol, 1.00 equiv), methyl 3-methylsalicylate (<b>28</b>) (221 mg, 1.33 mmol, 1.62 equiv), Ph<sub>3</sub>P (267 mg, 1.02 mmol, 1.24 equiv), anhydrous THF (6 mL), and DIAD (0.20 mL, 1.02 mmol, 1.24 equiv) and stirring 29 h under an atmosphere of argon at rt. The mixture was purified using flash chromatography (heptane:EtOAc, 1:1) to yield proline <b>37</b> (59 mg, 19% yield) as a semisolid. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 1.44 (s, 9H), 2.05–2.15 (m, 1H), 2.25 (s, 3H), 3.48–3.53 (m, 1H), 3.63–3.72 (m, 1H), 3.74 (s, 3H), 3.89 (s, 3H), 4.59 (t, <i>J</i> = 8 Hz, 0.5H), 4.58 (t, <i>J</i> = 8 Hz, 0.5H), 4.66 (br s, 1H), 7.04–7.08 (m, 1H), 7.35 (t, <i>J</i> = 7 Hz, 1H), 7.66 (d, <i>J</i> = 8 Hz, 1H). <sup>13</sup>C NMR (100 MHz, CDCl<sub>3</sub>) (rotamers) δ 16.5, 16.7, 22.1, 28.4, 28.5, 36.5, 37.5, 51.6, 52.2, 52.4, 58.0, 58.5, 70.2, 80.5, 81.4, 82.4, 123.9, 124.5, 129.8, 133.1, 135.5, 135.6, 153.8, 154.4, 155.1, 156.5, 166.9, 172.4, 173.4, 173.6. LCMS: <i>m</i>/<i>z</i> (M + H)<sup>+</sup>: calcd 294.1, found 294.1. TLC: <i>R</i><sub>f</sub> 0.21 (heptane:EtOAc, 1:1).</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i70" class="anchor-spacer"></div><h3 class="article-section__title" id="_i70"> Cell Culture and Transfection</h3><div class="NLM_p last">HEK293<i>T</i>/17 cells (ATCC) were maintained in minimal essential medium (MEM) containing GlutaMAX and supplemented with 10% fetal bovine serum (Invitrogen). Cells were plated at a low density on poly-<span class="smallcaps smallerCapital">d</span>-lysine-coated 35 mm plastic dishes and were transiently transfected 24 h later using the calcdium phosphate method. Rat cDNA encoding for GluK1–2a and GluK2a subunits was cotransfected with cDNA encoding for eGFP to identify transfected cells.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i71" class="anchor-spacer"></div><h3 class="article-section__title" id="_i71"> Radioligand Binding Pharmacology</h3><div class="NLM_p last">Binding at native iGluR in rat brain synaptosomal membranes was carried out as previously detailed.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Ligand affinities at recombinant rat GluA2 and GluK1–3 were determined as previously described,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> applying [<sup>3</sup>H]KA or [<sup>3</sup>H]NF608<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> in GluK1 binding experiments. Data were analyzed using GraphPad Prism 6 (GraphPad Software, San Diego, CA) to determine ligand <i>K</i><sub>i</sub> values.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> Electrophysiological Recordings</h3><div class="NLM_p last">Experiments were performed 36–48 h after transfection. Agonist solutions were rapidly applied to outside-out patches excised from transfected cells using a piezoelectric stack (Physik Instrumente). Solution exchange (10–90% rise time of 250–350 μs) was determined in a separate experiment by measuring the liquid junction current. All recordings were performed using an Axopatch 200B (Molecular Devices) using thick-walled borosilicate glass pipettes (3–6 MΩ) coated with dental wax to reduce electrical noise. Current records were filtered at 5 kHz, digitized at 25 kHz, and series resistance (3–12 MΩ) compensated by 95%. Recordings were performed at a holding potential of −100 mV. All experiments were performed at rt. All chemicals used for electrophysiology were purchased from Sigma-Aldrich unless otherwise indicated. The external solution contained (in mM) 150 NaCl, 5 HEPES, 0.1 MgCl<sub>2</sub>, and 0.1 CaCl<sub>2</sub>, pH 7.3–7.4. The internal solution contained (in mM) 115 NaCl, 10 NaF, 5 HEPES, 5 Na<sub>4</sub>BAPTA (Life Technologies), 1 MgCl<sub>2</sub>, 0.5 CaCl<sub>2</sub>, and 10 Na<sub>2</sub>ATP, pH 7.3–7.4. The osmotic pressure of all solutions was adjusted to 295–300 mOsm with sucrose. Concentrated (100×) Glu stocks were prepared and stored at −20 °C; stocks were thawed and diluted on the day of the experiment. Concentrated (100 mM in 10 mM NaOH) stocks of <b>1b</b> were prepared and stored at −20 °C; stocks were thawed on the day of the experiment and added to the control and agonist external solutions for the following final concentrations (μM): 0.001, 0.01, 0.1, 1, 10, 30, 100, and 300. As a control, an equivalent amount of NaOH was added to the external solutions not containing <b>1b</b>. Data acquisition was performed using pClamp10 software (Molecular Devices), tabulated using Excel (Microsoft Corp) and illustrated using Adobe Illustrator. Inhibition curves were fit using Origin 7.0 (OriginLab) using the following equation:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/jm-2016-015162_m001.gif" alt="" /></img></span>Where <i>I</i><sub>max</sub> is the peak current in the absence of antagonist, <i>I</i> is the peak current at ([antagonist]), IC<sub>50</sub> is the concentration required for 50% inhibition, and <i>n</i> is the slope.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> X-ray Structure Determination</h3><div class="NLM_p">Rat GluK1 LBD (segment S1 residues 430–544 and segment S2 residues 667–805, numbering according to UNP P22756) was expressed and purified in the presence of glutamate as previously described.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Crystallization of GluK1 LBD in complex with <b>1b</b> was performed using the hanging drop vapor diffusion method at 6 °C. A protein solution consisting of 5.1 mg/mL GluK1 LBD and 5.16 mM <b>1b</b> in 10 mM HEPES, 20 mM NaCl, and 1 mM EDTA, pH 7.0, was prepared and used for crystallization experiments. Crystals for diffraction studies were obtained with the following reservoir conditions: 15.2% PEG4000, 0.3 M lithium sulfate, and 0.1 M cacodylate buffer pH 6.5 (1 μL of protein–ligand complex and 1 μL of reservoir). The crystals grew within a few days, and they were flash-cooled with liquid nitrogen after immersion in cryo-buffer (1 μL of reservoir solution added to 1 μL of 8% glycerol, 8% ethylene glycol, 9% sucrose, and 2% glucose).</div><div class="NLM_p last">Diffraction data were collected on beamline I911-3<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> at MAX-lab, Lund, Sweden, to a final resolution of 3.18 Å. The data were processed with XDS<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and scaled using SCALA<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> in CCP4i.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> The structure was solved by molecular replacement using PHASER<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> within CCP4i and with the structure of GluK1 LBD in complex with the antagonist (<i>S</i>)-ATPO as a search model (pdb code, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1vso">1vso</a>; molA, protein atoms only).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The search model was defined into two lobes: D1 and D2. Two molecules were found in the asymmetric unit of the crystal. Visual inspection of the structure in COOT<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> revealed well-defined electron density for the residues in most areas and density corresponding to the ligand <b>1b</b>. The ligand coordinates were created in Maestro [Maestro version 9.3.5, Schrödinger, LLC, New York, NY, 2012] and fitted into the electron density. Topology and parameter files for <b>1b</b> were obtained using eLBOW,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> keeping ligand geometry from Maestro. The Phaser solution was used for model building manually in COOT. The structure was refined in PHENIX<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> with individual isotropic <i>B</i> factors, NCS, secondary structural restraints as well as optimizing weights (X-ray/stereochemistry and X-ray/ADP). Finally, the structure was validated using tools in COOT and PHENIX as well as the PDB ADIT validation server. Statistics of data collection and refinement can be found in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>. D1–D2 domain openings were calcdulated using the DynDom server.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="NOTES-d246e7013-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i74"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01516">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47162" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47162" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01516" class="ext-link">10.1021/acs.jmedchem.6b01516</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01516/suppl_file/jm6b01516_si_001.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01516/suppl_file/jm6b01516_si_001.csv">jm6b01516_si_001.csv (1.74 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure coordinates and corresponding structure factor file of GluK1 LBD with <b>1b</b> has been deposited in the Protein Data Bank under the accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5m2v">5m2v</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01516" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22730" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22730" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karla Frydenvang</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#bdd6dccfd1dc93dbcfc4d9d8d3cbdcd3dafdcec8d3d993d6c893d9d6"><span class="__cf_email__" data-cfemail="85eee4f7e9e4abe3f7fce1e0ebf3e4ebe2c5f6f0ebe1abeef0abe1ee">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lennart Bunch</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0180-4639" title="Orcid link">http://orcid.org/0000-0002-0180-4639</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fe929b9c8bbe8d8b909ad0958bd09a95"><span class="__cf_email__" data-cfemail="214d4443546152544f450f4a540f454a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Niels Krogsgaard-Larsen</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Claudia G. Delgar</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Karina Koch</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patricia M. G. E. Brown</span> - <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charlotte Møller</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liwei Han</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tri H. V. Huynh</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stinne W. Hansen</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Birgitte Nielsen</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Derek Bowie</span> - <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Darryl S. Pickering</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jette Sandholm Kastrup</span> - <span class="hlFld-Affiliation affiliation">†Bunch
Research Group, ‡Biostructural Research Group, §Medicinal Chemistry Group, ∥Molecular and Cellular
Pharmacology Group, Faculty of Health and Medical Sciences, University of Copenhagen, Universitetsparken 2, Copenhagen 2100, Denmark</span>; 
    <span class="hlFld-Affiliation affiliation">Bowie Lab,
Department of Pharmacology & Therapeutics, Faculty of Medicine, McGill University, Montreal, Quebec H3G 0B1, Canada</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d246e7041-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i76">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50362" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50362" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the Lundbeck Foundation, Danish Research Council (FSS), University of Copenhagen Programme of Excellence (GluTarget), European Community’s Seventh Framework Programme BioStruct-X, and Danscatt for financial support. PMGEB was supported by a Doctoral Fellowship Award from the Fonds de Recherche du Québe—Santé (FRQS). Heidi Peterson is thanked for technical assistance with expression and purification of GluK1 LBD. The MAX-lab synchrotron facility in Lund, Sweden, is thanked for providing beamtime.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i77" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i77"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i78" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i78"> Abbreviations Used</h2><tr><td class="NLM_term">AMPA</td><td class="NLM_def"><p class="first last">α-amino-3-hydroxy-5-methyl-4-isoxazolepropionic acid</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">EAATs</td><td class="NLM_def"><p class="first last">excitatory amino acid transporters</p></td></tr><tr><td class="NLM_term">Glu</td><td class="NLM_def"><p class="first last">(<i>S</i>)-glutamic acid</p></td></tr><tr><td class="NLM_term">mGluRs</td><td class="NLM_def"><p class="first last">G-protein coupled metabotropic Glu receptors</p></td></tr><tr><td class="NLM_term">iGluRs</td><td class="NLM_def"><p class="first last">ionotropic Glu receptors</p></td></tr><tr><td class="NLM_term">KA</td><td class="NLM_def"><p class="first last">kainic acid</p></td></tr><tr><td class="NLM_term">LBD</td><td class="NLM_def"><p class="first last">ligand binding domain</p></td></tr><tr><td class="NLM_term">LHMDS</td><td class="NLM_def"><p class="first last">lithium hexamethyldisilazide</p></td></tr><tr><td class="NLM_term">NMDA</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl-<span class="smallcaps smallerCapital">d</span>-aspartic acid</p></td></tr><tr><td class="NLM_term">rt</td><td class="NLM_def"><p class="first last">room temperature</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54568" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54568" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 41 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Traynelis, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wollmuth, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menniti, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vance, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dingledine, R.</span><span> </span><span class="NLM_article-title">Glutamate Receptor Ion Channels: Structure, Regulation, and Function</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span><span class="refDoi"> DOI: 10.1124/pr.109.002451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1124%2Fpr.109.002451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20716669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=405-496&author=S.+F.+Traynelisauthor=L.+P.+Wollmuthauthor=C.+J.+McBainauthor=F.+S.+Mennitiauthor=K.+M.+Vanceauthor=K.+K.+Ogdenauthor=K.+B.+Hansenauthor=H.+Yuanauthor=S.+J.+Myersauthor=R.+Dingledine&title=Glutamate+Receptor+Ion+Channels%3A+Structure%2C+Regulation%2C+and+Function&doi=10.1124%2Fpr.109.002451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate receptor ion channels: structure, regulation, and function</span></div><div class="casAuthors">Traynelis, Stephen F.; Wollmuth, Lonnie P.; McBain, Chris J.; Menniti, Frank S.; Vance, Katie M.; Ogden, Kevin K.; Hansen, Kasper B.; Yuan, Hongjie; Myers, Scott J.; Dingledine, Ray</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-496</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  The mammalian ionotropic glutamate receptor family encodes 18 gene products that coassemble to form ligand-gated ion channels contg. an agonist recognition site, a transmembrane ion permeation pathway, and gating elements that couple agonist-induced conformational changes to the opening or closing of the permeation pore.  Glutamate receptors mediate fast excitatory synaptic transmission in the central nervous system and are localized on neuronal and non-neuronal cells.  These receptors regulate a broad spectrum of processes in the brain, spinal cord, retina, and peripheral nervous system.  Glutamate receptors are postulated to play important roles in numerous neurol. diseases and have attracted intense scrutiny.  The description of glutamate receptor structure, including its transmembrane elements, reveals a complex assembly of multiple semiautonomous extracellular domains linked to a pore-forming element with striking resemblance to an inverted potassium channel.  In this review we discuss International Union of Basic and Clin. Pharmacol. glutamate receptor nomenclature, structure, assembly, accessory subunits, interacting proteins, gene expression and translation, post-translational modifications, agonist and antagonist pharmacol., allosteric modulation, mechanisms of gating and permeation, roles in normal physiol. function, as well as the potential therapeutic use of pharmacol. agents acting at glutamate receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov0rj4nMr5ObVg90H21EOLACvtfcHk0lhBiuW4ZZzFXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSltbbJ&md5=5d0a7945b5ad99113adf9f6312bafe13</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fpr.109.002451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.109.002451%26sid%3Dliteratum%253Aachs%26aulast%3DTraynelis%26aufirst%3DS.%2BF.%26aulast%3DWollmuth%26aufirst%3DL.%2BP.%26aulast%3DMcBain%26aufirst%3DC.%2BJ.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DVance%26aufirst%3DK.%2BM.%26aulast%3DOgden%26aufirst%3DK.%2BK.%26aulast%3DHansen%26aufirst%3DK.%2BB.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DMyers%26aufirst%3DS.%2BJ.%26aulast%3DDingledine%26aufirst%3DR.%26atitle%3DGlutamate%2520Receptor%2520Ion%2520Channels%253A%2520Structure%252C%2520Regulation%252C%2520and%2520Function%26jtitle%3DPharmacol.%2520Rev.%26date%3D2010%26volume%3D62%26spage%3D405%26epage%3D496%26doi%3D10.1124%2Fpr.109.002451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Ferraguti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shigemoto, R.</span><span> </span><span class="NLM_article-title">Metabotropic Glutamate Receptors</span> <span class="citation_source-journal">Cell Tissue Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">483</span><span class="NLM_x">–</span> <span class="NLM_lpage">504</span><span class="refDoi"> DOI: 10.1007/s00441-006-0266-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1007%2Fs00441-006-0266-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=16847639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvFalt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2006&pages=483-504&author=F.+Ferragutiauthor=R.+Shigemoto&title=Metabotropic+Glutamate+Receptors&doi=10.1007%2Fs00441-006-0266-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Metabotropic glutamate receptors</span></div><div class="casAuthors">Ferraguti, Francesco; Shigemoto, Ryuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cell & Tissue Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">483-504</span>CODEN:
                <span class="NLM_cas:coden">CTSRCS</span>;
        ISSN:<span class="NLM_cas:issn">0302-766X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Metabotropic glutamate receptors (mGlus) are a family of G-protein-coupled receptors activated by the neurotransmitter glutamate.  Mol. cloning has revealed eight different subtypes (mGlu1-8) with distinct mol. and pharmacol. properties.  Multiplicity in this receptor family is further generated through alternative splicing. mGlus activate a multitude of signalling pathways important for modulating neuronal excitability, synaptic plasticity and feedback regulation of neurotransmitter release.  In this review, we summarize anatomical findings (from our work and that of other labs.) describing their distribution in the central nervous system.  Recent evidence regarding the localization of these receptors in peripheral tissues will also be examd.  The distinct regional, cellular and subcellular distribution of mGlus in the brain will be discussed in view of their relationship to neurotransmitter release sites and of possible functional implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPxClJvmJpzLVg90H21EOLACvtfcHk0lhBiuW4ZZzFXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvFalt70%253D&md5=7c4ded2874fd22764db90e87bfb0c0a6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00441-006-0266-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00441-006-0266-5%26sid%3Dliteratum%253Aachs%26aulast%3DFerraguti%26aufirst%3DF.%26aulast%3DShigemoto%26aufirst%3DR.%26atitle%3DMetabotropic%2520Glutamate%2520Receptors%26jtitle%3DCell%2520Tissue%2520Res.%26date%3D2006%26volume%3D326%26spage%3D483%26epage%3D504%26doi%3D10.1007%2Fs00441-006-0266-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Danbolt, N. C.</span><span> </span><span class="NLM_article-title">Glutamate Uptake</span> <span class="citation_source-journal">Prog. Neurobiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1016/S0301-0082(00)00067-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2FS0301-0082%2800%2900067-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=11369436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsFWns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2001&pages=1-105&author=N.+C.+Danbolt&title=Glutamate+Uptake&doi=10.1016%2FS0301-0082%2800%2900067-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate uptake</span></div><div class="casAuthors">Danbolt, N. C.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Neurobiology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-105</span>CODEN:
                <span class="NLM_cas:coden">PGNBA5</span>;
        ISSN:<span class="NLM_cas:issn">0301-0082</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with >1200 refs.  Brain tissue has a remarkable ability to accumulate glutamate.  This ability is due to glutamate transporter proteins present in the plasma membranes of both glial cells and neurons.  The transporter proteins represent the only (significant) mechanism for removal of glutamate from the extracellular fluid and their importance for the long-term maintenance of low and non-toxic concns. of glutamate is now well documented.  In addn. to this simple, but essential glutamate removal role, the glutamate transporters appear to have more sophisticated functions in the modulation of neurotransmission.  They may modify the time course of synaptic events, the extent and pattern of activation and desensitization of receptors outside the synaptic cleft and at neighboring synapses (intersynaptic cross-talk).  Further, the glutamate transporters provide glutamate for synthesis of e.g.  GABA, glutathione, and protein, and for energy prodn.  They also play roles in peripheral organs and tissues (e.g. bone, heart, intestine, kidneys, pancreas, and placenta).  Glutamate uptake appears to be modulated on virtually all possible levels, i.e. DNA transcription, mRNA splicing and degrdn., protein synthesis and targeting, and actual amino acid transport activity and assocd. ion channel activities.  A variety of sol. compds. (e.g. glutamate, cytokines, and growth factors) influence glutamate transporter expression and activities.  Neither the normal functioning of glutamatergic synapses nor the pathogenesis of major neurol. diseases (e.g. cerebral ischemia, hypoglycemia, amyotrophic lateral sclerosis, Alzheimer's disease, traumatic brain injury, epilepsy, and schizophrenia) as well as non-neurol. diseases (e.g. osteoporosis) can be properly understood unless more is learned about these transporter proteins.  Like glutamate itself, glutamate transporters are somehow involved in almost all aspects of normal and abnormal brain activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbDph2PBuBbVg90H21EOLACvtfcHk0lhaLYvb-9zHFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsFWns74%253D&md5=a7118c1df858a0994af6207f852f82b6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0301-0082%2800%2900067-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0301-0082%252800%252900067-8%26sid%3Dliteratum%253Aachs%26aulast%3DDanbolt%26aufirst%3DN.%2BC.%26atitle%3DGlutamate%2520Uptake%26jtitle%3DProg.%2520Neurobiol.%26date%3D2001%26volume%3D65%26spage%3D1%26epage%3D105%26doi%3D10.1016%2FS0301-0082%2800%2900067-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Jensen, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahlke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjørn-Yoshimoto, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Excitatory Amino Acid Transporters: Recent Insights into Molecular Mechanisms, Novel Modes of Modulation and New Therapeutic Possibilities</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">116</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span><span class="refDoi"> DOI: 10.1016/j.coph.2014.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.coph.2014.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=25466154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=116-123&author=A.+A.+Jensenauthor=C.+Fahlkeauthor=W.+E.+Bj%C3%B8rn-Yoshimotoauthor=L.+Bunch&title=Excitatory+Amino+Acid+Transporters%3A+Recent+Insights+into+Molecular+Mechanisms%2C+Novel+Modes+of+Modulation+and+New+Therapeutic+Possibilities&doi=10.1016%2Fj.coph.2014.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Excitatory amino acid transporters: recent insights into molecular mechanisms, novel modes of modulation and new therapeutic possibilities</span></div><div class="casAuthors">Jensen, Anders A.; Fahlke, Christoph; Bjoern-Yoshimoto, Walden E.; Bunch, Lennart</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">116-123</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The five excitatory amino acid transporters (EAAT1-5) mediating the synaptic uptake of the major excitatory neurotransmitter glutamate are differently expressed throughout the CNS and at the synaptic level.  Although EAATs are crucial for normal excitatory neurotransmission, explorations into the physiol. functions mediated by the different transporter subtypes and their resp. therapeutic potential have so far been sparse, in no small part due to the limited selection of pharmacol. tools available.  In the present update, we outline important new insights into the mol. compns. of EAATs and their intricate transport process, the novel approaches to pharmacol. modulation of the transporters that have emerged, and interesting new perspectives in EAAT as drug targets proposed in recent years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHCPIMplW6KLVg90H21EOLACvtfcHk0lhaLYvb-9zHFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvF&md5=4619d6aebdc64cfd77a44bae4379eb53</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2014.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2014.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DA.%2BA.%26aulast%3DFahlke%26aufirst%3DC.%26aulast%3DBj%25C3%25B8rn-Yoshimoto%26aufirst%3DW.%2BE.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DExcitatory%2520Amino%2520Acid%2520Transporters%253A%2520Recent%2520Insights%2520into%2520Molecular%2520Mechanisms%252C%2520Novel%2520Modes%2520of%2520Modulation%2520and%2520New%2520Therapeutic%2520Possibilities%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D20%26spage%3D116%26epage%3D123%26doi%3D10.1016%2Fj.coph.2014.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Bjørn-Yoshimoto, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underhill, S. M.</span><span> </span><span class="NLM_article-title">The Importance of the Excitatory Amino Acid Transporter 3 (EAAT3)</span> <span class="citation_source-journal">Neurochem. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span><span class="refDoi"> DOI: 10.1016/j.neuint.2016.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.neuint.2016.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=27233497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252Fkt1KisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2016&pages=4-18&author=W.+E.+Bj%C3%B8rn-Yoshimotoauthor=S.+M.+Underhill&title=The+Importance+of+the+Excitatory+Amino+Acid+Transporter+3+%28EAAT3%29&doi=10.1016%2Fj.neuint.2016.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The importance of the excitatory amino acid transporter 3 (EAAT3)</span></div><div class="casAuthors">Bjorn-Yoshimoto Walden E; Underhill Suzanne M</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry international</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4-18</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The neuronal excitatory amino acid transporter 3 (EAAT3) is fairly ubiquitously expressed in the brain, though it does not necessarily maintain the same function everywhere.  It is important in maintaining low local concentrations of glutamate, where its predominant post-synaptic localization can buffer nearby glutamate receptors and modulate excitatory neurotransmission and synaptic plasticity.  It is also the main neuronal cysteine uptake system acting as the rate-limiting factor for the synthesis of glutathione, a potent antioxidant, in EAAT3 expressing neurons, while on GABAergic neurons, it is important in supplying glutamate as a precursor for GABA synthesis.  Several diseases implicate EAAT3, and modulation of this transporter could prove a useful therapeutic approach.  Regulation of EAAT3 could be targeted at several points for functional modulation, including the level of transcription, trafficking and direct pharmacological modulation, and indeed, compounds and experimental treatments have been identified that regulate EAAT3 function at different stages, which together with observations of EAAT3 regulation in patients is giving us insight into the endogenous function of this transporter, as well as the consequences of altered function.  This review summarizes work done on elucidating the role and regulation of EAAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTRow0Q3ClcxdUSIfHIn4FfW6udTcc2eY89DJS7R5Eb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252Fkt1KisA%253D%253D&md5=27ff5a73f8de4c0550ebe7ee83660fbd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2016.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2016.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DBj%25C3%25B8rn-Yoshimoto%26aufirst%3DW.%2BE.%26aulast%3DUnderhill%26aufirst%3DS.%2BM.%26atitle%3DThe%2520Importance%2520of%2520the%2520Excitatory%2520Amino%2520Acid%2520Transporter%25203%2520%2528EAAT3%2529%26jtitle%3DNeurochem.%2520Int.%26date%3D2016%26volume%3D98%26spage%3D4%26epage%3D18%26doi%3D10.1016%2Fj.neuint.2016.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Choi, D. W.</span><span> </span><span class="NLM_article-title">Glutamate Neurotoxicity and Diseases of the Nervous System</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">634</span><span class="refDoi"> DOI: 10.1016/0896-6273(88)90162-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2F0896-6273%2888%2990162-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=2908446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A280%3ADyaK3c7ms1GnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1988&pages=623-634&author=D.+W.+Choi&title=Glutamate+Neurotoxicity+and+Diseases+of+the+Nervous+System&doi=10.1016%2F0896-6273%2888%2990162-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate neurotoxicity and diseases of the nervous system</span></div><div class="casAuthors">Choi D W</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">623-34</span>
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSgYvXebUy8NTW1zwg2yUK5fW6udTcc2eY89DJS7R5Eb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3c7ms1GnsQ%253D%253D&md5=1e33ba0bd4c1d830f6b7c7d76e29dbd3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0896-6273%2888%2990162-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0896-6273%252888%252990162-6%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DD.%2BW.%26atitle%3DGlutamate%2520Neurotoxicity%2520and%2520Diseases%2520of%2520the%2520Nervous%2520System%26jtitle%3DNeuron%26date%3D1988%26volume%3D1%26spage%3D623%26epage%3D634%26doi%3D10.1016%2F0896-6273%2888%2990162-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Lau, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tymianski, M.</span><span> </span><span class="NLM_article-title">Glutamate Receptors, Neurotoxicity and Neurodegeneration</span> <span class="citation_source-journal">Pfluegers Arch.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">460</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1007/s00424-010-0809-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1007%2Fs00424-010-0809-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20229265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlGns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=460&publication_year=2010&pages=525-542&author=A.+Lauauthor=M.+Tymianski&title=Glutamate+Receptors%2C+Neurotoxicity+and+Neurodegeneration&doi=10.1007%2Fs00424-010-0809-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate receptors, neurotoxicity and neurodegeneration</span></div><div class="casAuthors">Lau, Anthony; Tymianski, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Pfluegers Archiv</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">460</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-542</span>CODEN:
                <span class="NLM_cas:coden">PFLABK</span>;
        ISSN:<span class="NLM_cas:issn">0031-6768</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Glutamate excitotoxicity is a hypothesis that states excessive glutamate causes neuronal dysfunction and degeneration.  As glutamate is a major excitatory neurotransmitter in the central nervous system (CNS), the implications of glutamate excitotoxicity are many and far-reaching.  Acute CNS insults such as ischemia and traumatic brain injury have traditionally been the focus of excitotoxicity research.  However, glutamate excitotoxicity has also been linked to chronic neurodegenerative disorders such as amyotrophic lateral sclerosis, multiple sclerosis, Parkinson's disease and others.  Despite the continued research into the mechanisms of excitotoxicity, there are currently no pharmacol. interventions capable of providing significant neuroprotection in the clin. setting of brain ischemia or injury.  This review addresses the current state of excitotoxic research, focusing on the structure and physiol. of glutamate receptors; mol. mechanisms underlying excitotoxic cell death pathways and their interactions with each other; the evidence for glutamate excitotoxicity in acute neurol. diseases; lab. and clin. attempts at modulating excitotoxicity; and emerging targets for excitotoxicity research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKnG8BQuqVZLVg90H21EOLACvtfcHk0lhIokm4BiYU_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlGns7o%253D&md5=9429b771c07ca3f65c509c8bc6566cac</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00424-010-0809-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00424-010-0809-1%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DA.%26aulast%3DTymianski%26aufirst%3DM.%26atitle%3DGlutamate%2520Receptors%252C%2520Neurotoxicity%2520and%2520Neurodegeneration%26jtitle%3DPfluegers%2520Arch.%26date%3D2010%26volume%3D460%26spage%3D525%26epage%3D542%26doi%3D10.1007%2Fs00424-010-0809-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Machado-Vieira, R.; Henter, I. D.; Zarate, C. A.,  Jr.</span><span> </span><span class="NLM_article-title">New Targets for Rapid Antidepressant Action</span>.  <span class="citation_source-journal">Prog. Neurobiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1016/j.pneurobio.2015.12.001</span> ..</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.pneurobio.2015.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=26724279" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=R.+Machado-Vieira&author=I.+D.+Henter&author=C.+A.+Zarate&title=New+Targets+for+Rapid+Antidepressant+Action&doi=10.1016%2Fj.pneurobio.2015.12.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.pneurobio.2015.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pneurobio.2015.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DMachado-Vieira%26aufirst%3DR.%26atitle%3DNew%2520Targets%2520for%2520Rapid%2520Antidepressant%2520Action%26jtitle%3DProg.%2520Neurobiol.%26date%3D2015%26doi%3D10.1016%2Fj.pneurobio.2015.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Riaza Bermudo-Soriano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Rodriguez, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaquero-Lorenzo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baca-Garcia, E.</span><span> </span><span class="NLM_article-title">New Perspectives in Glutamate and Anxiety</span> <span class="citation_source-journal">Pharmacol., Biochem. Behav.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">752</span><span class="NLM_x">–</span> <span class="NLM_lpage">774</span><span class="refDoi"> DOI: 10.1016/j.pbb.2011.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.pbb.2011.04.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2012&pages=752-774&author=C.+Riaza+Bermudo-Sorianoauthor=M.+M.+Perez-Rodriguezauthor=C.+Vaquero-Lorenzoauthor=E.+Baca-Garcia&title=New+Perspectives+in+Glutamate+and+Anxiety&doi=10.1016%2Fj.pbb.2011.04.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pbb.2011.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbb.2011.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DRiaza%2BBermudo-Soriano%26aufirst%3DC.%26aulast%3DPerez-Rodriguez%26aufirst%3DM.%2BM.%26aulast%3DVaquero-Lorenzo%26aufirst%3DC.%26aulast%3DBaca-Garcia%26aufirst%3DE.%26atitle%3DNew%2520Perspectives%2520in%2520Glutamate%2520and%2520Anxiety%26jtitle%3DPharmacol.%252C%2520Biochem.%2520Behav.%26date%3D2012%26volume%3D100%26spage%3D752%26epage%3D774%26doi%3D10.1016%2Fj.pbb.2011.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Wierońska, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorn, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doller, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilc, A.</span><span> </span><span class="NLM_article-title">Metabotropic Glutamate Receptors as Targets for New Antipsychotic Drugs: Historical Perspective and Critical Comparative Assessment</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">10</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.pharmthera.2015.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=26549541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVSks7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2016&pages=10-27&author=J.+M.+Wiero%C5%84skaauthor=S.+H.+Zornauthor=D.+Dollerauthor=A.+Pilc&title=Metabotropic+Glutamate+Receptors+as+Targets+for+New+Antipsychotic+Drugs%3A+Historical+Perspective+and+Critical+Comparative+Assessment&doi=10.1016%2Fj.pharmthera.2015.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Metabotropic glutamate receptors as targets for new antipsychotic drugs: Historical perspective and critical comparative assessment</span></div><div class="casAuthors">Wieronska, Joanna M.; Zorn, Stevin H.; Doller, Dario; Pilc, Andrzej</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10-27</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">In this review, we aim to present, discuss and clarify our current understanding regarding the prediction of possible antipsychotic effects of metabotropic glutamate (mGlu) receptor ligands.  The no. of preclin. trials clearly indicates, that this group of compds. constitutes an excellent alternative to presently used antipsychotic therapy, being effective not only to pos., but also neg. and cognitive symptoms of schizophrenia.  Although the results of clin. trials that were performed for the group of mGlu2/3 agonists were not so enthusiastic as in animal studies, they still showed that mGlu ligands do not induced variety of side effects typical for presently used antipsychotics, and were generally well tolerated.  The lack of satisfactory effectiveness towards schizophrenia symptoms of mGlu2/3 activators in humans could be a result of variety of uncontrolled factors and unidentified biomarkers different for each schizophrenia patient, that should be taken into consideration in the future set of clin. trials.  The subject is still open for further research, and the novel classes of mGlu5 or mGlu2/3 agonists/PAMs were recently introduced, including the large group of compds. from the third group of mGlu receptors, esp. of mGlu4 subtype.  Finally, more precise treatment based on simultaneous administration of minimal doses of the ligands for two or more receptors, seems to be promising in the context of symptoms-specific schizophrenia treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor64l-ntwmK7Vg90H21EOLACvtfcHk0lhIokm4BiYU_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVSks7vM&md5=23b8ec790518a3ad5c92cb6c3895e618</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DWiero%25C5%2584ska%26aufirst%3DJ.%2BM.%26aulast%3DZorn%26aufirst%3DS.%2BH.%26aulast%3DDoller%26aufirst%3DD.%26aulast%3DPilc%26aufirst%3DA.%26atitle%3DMetabotropic%2520Glutamate%2520Receptors%2520as%2520Targets%2520for%2520New%2520Antipsychotic%2520Drugs%253A%2520Historical%2520Perspective%2520and%2520Critical%2520Comparative%2520Assessment%26jtitle%3DPharmacol.%2520Ther.%26date%3D2016%26volume%3D157%26spage%3D10%26epage%3D27%26doi%3D10.1016%2Fj.pharmthera.2015.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Gonzalez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurado-Coronel, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ávila, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabogal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capani, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barreto, G. E.</span><span> </span><span class="NLM_article-title">NMDARs in Neurological Diseases: A Potential Therapeutic Target</span> <span class="citation_source-journal">Int. J. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span><span class="refDoi"> DOI: 10.3109/00207454.2014.940941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.3109%2F00207454.2014.940941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=25051426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVaju7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=315-327&author=J.+Gonzalezauthor=J.+C.+Jurado-Coronelauthor=M.+F.+%C3%81vilaauthor=A.+Sabogalauthor=F.+Capaniauthor=G.+E.+Barreto&title=NMDARs+in+Neurological+Diseases%3A+A+Potential+Therapeutic+Target&doi=10.3109%2F00207454.2014.940941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">NMDARs in neurological diseases: a potential therapeutic target</span></div><div class="casAuthors">Gonzalez, Janneth; Jurado-Coronel, Juan Camilo; Avila, Marcos Fidel; Sabogal, Angelica; Capani, Francisco; Barreto, George E.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">315-327</span>CODEN:
                <span class="NLM_cas:coden">IJNUB7</span>;
        ISSN:<span class="NLM_cas:issn">0020-7454</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">N-methyl-D-aspartate ionotropic glutamate receptor (NMDARs) is a ligand-gated ion channel that plays a crit. role in excitatory neurotransmission, brain development, synaptic plasticity assocd. with memory formation, central sensitization during persistent pain, excitotoxicity and neurodegenerative diseases in the central nervous system (CNS).  Within iGluRs, NMDA receptors have been the most actively investigated for their role in neurol. diseases, esp. neurodegenerative pathologies such as Alzheimer's and Parkinson's diseases.  It has been demonstrated that excessive activation of NMDA receptors (NMDARs) plays a key role in mediating some aspects of synaptic dysfunction in several CNS disorders, so extensive research has been directed on the discovery of compds. that are able to reduce NMDARs activity.  This review discusses the role of NMDARs on neurol. pathologies and the possible therapeutic use of agents that target this receptor.  Addnl., we delve into the role of NMDARs in Alzheimer's and Parkinson's diseases and the receptor antagonists that have been tested on in vivo models of these pathologies.  Finally, we put into consideration the importance of antioxidants to counteract oxidative capacity of the signaling cascade in which NMDARs are involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpab8NK1TOnqrVg90H21EOLACvtfcHk0lhZfSCoUcJlLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVaju7fJ&md5=50f77e6ec1187bba03ed53eb61e721a4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3109%2F00207454.2014.940941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00207454.2014.940941%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DJ.%26aulast%3DJurado-Coronel%26aufirst%3DJ.%2BC.%26aulast%3D%25C3%2581vila%26aufirst%3DM.%2BF.%26aulast%3DSabogal%26aufirst%3DA.%26aulast%3DCapani%26aufirst%3DF.%26aulast%3DBarreto%26aufirst%3DG.%2BE.%26atitle%3DNMDARs%2520in%2520Neurological%2520Diseases%253A%2520A%2520Potential%2520Therapeutic%2520Target%26jtitle%3DInt.%2520J.%2520Neurosci.%26date%3D2015%26volume%3D125%26spage%3D315%26epage%3D327%26doi%3D10.3109%2F00207454.2014.940941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Finlay, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duty, S.</span><span> </span><span class="NLM_article-title">Therapeutic Potential of Targeting Glutamate Receptors in Parkinson’s Disease</span> <span class="citation_source-journal">J. Neural Transm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">880</span><span class="refDoi"> DOI: 10.1007/s00702-014-1176-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1007%2Fs00702-014-1176-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2014&pages=861-880&author=C.+Finlayauthor=S.+Duty&title=Therapeutic+Potential+of+Targeting+Glutamate+Receptors+in+Parkinson%E2%80%99s+Disease&doi=10.1007%2Fs00702-014-1176-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00702-014-1176-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00702-014-1176-4%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DC.%26aulast%3DDuty%26aufirst%3DS.%26atitle%3DTherapeutic%2520Potential%2520of%2520Targeting%2520Glutamate%2520Receptors%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Neural%2520Transm.%26date%3D2014%26volume%3D121%26spage%3D861%26epage%3D880%26doi%3D10.1007%2Fs00702-014-1176-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, J. M.</span><span> </span><span class="NLM_article-title">AMPA Receptors as Therapeutic Targets for Neurological Disorders</span> <span class="citation_source-journal">Adv. Protein Chem. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span><span class="refDoi"> DOI: 10.1016/bs.apcsb.2015.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fbs.apcsb.2015.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=26920691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOisb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=203-261&author=K.+Leeauthor=L.+Goodmanauthor=C.+Fourieauthor=S.+Schenkauthor=B.+Leitchauthor=J.+M.+Montgomery&title=AMPA+Receptors+as+Therapeutic+Targets+for+Neurological+Disorders&doi=10.1016%2Fbs.apcsb.2015.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">AMPA receptors as therapeutic targets for neurological disorders</span></div><div class="casAuthors">Lee, Kevin; Goodman, Lucy; Fourie, Chantelle; Schenk, Susan; Leitch, Beulah; Montgomery, Johanna M.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Protein Chemistry and Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">Ion Channels as Therapeutic Targets, Part A</span>),
    <span class="NLM_cas:pages">203-261</span>CODEN:
                <span class="NLM_cas:coden">APCSG7</span>;
        ISSN:<span class="NLM_cas:issn">1876-1623</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Almost every neurol. disease directly or indirectly affects synapse function in the brain.  However, these diseases alter synapses through different mechanisms, ultimately resulting in altered synaptic transmission and/or plasticity.  Glutamate is the major neurotransmitter that mediates excitatory synaptic transmission in the brain through activation of alpha-amino-3-hydroxy-5-methyl-4-isoxazole-propionate (AMPA) receptors.  These receptors have therefore been identified as a target for the development of therapeutic treatments for neurol. disorders including epilepsy, neurodegenerative diseases, autism, and drug addiction.  The fact that AMPA receptors play a dominant role throughout the brain raises the significant challenge of selectively targeting only those regions affected by disease, and clin. trials have raised doubt regarding the feasibility of specifically targeting AMPA receptors for new therapeutic options.  Benzamide compds. that act as pos. allosteric AMPA receptor modulators, known as AMPAkines, can act on specific brain regions and were initially proposed to revolutionize the treatment of cognitive deficits assocd. with neurol. disorders.  Their therapeutic potential has since declined due to inconsistent results in clin. trials.  However, recent advances in basic biomedical research are significantly increasing our knowledge of AMPA receptor structure, binding sites, and interactions with auxiliary proteins.  In particular, the large complex of postsynaptic proteins that interact with AMPA receptor subunits have been shown to control AMPA receptor insertion, location, pharmacol., synaptic transmission, and plasticity.  These proteins are now being considered as alternative therapeutic target sites for modulating AMPA receptors in neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuZyN45J6nyLVg90H21EOLACvtfcHk0lhZfSCoUcJlLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOisb7E&md5=cac1a247d6e5e58ea9181fd14e3fcf04</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fbs.apcsb.2015.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.apcsb.2015.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%26aulast%3DGoodman%26aufirst%3DL.%26aulast%3DFourie%26aufirst%3DC.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DLeitch%26aufirst%3DB.%26aulast%3DMontgomery%26aufirst%3DJ.%2BM.%26atitle%3DAMPA%2520Receptors%2520as%2520Therapeutic%2520Targets%2520for%2520Neurological%2520Disorders%26jtitle%3DAdv.%2520Protein%2520Chem.%2520Struct.%2520Biol.%26date%3D2016%26volume%3D103%26spage%3D203%26epage%3D261%26doi%3D10.1016%2Fbs.apcsb.2015.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Armstrong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gouaux, E.</span><span> </span><span class="NLM_article-title">Mechanisms for Activation and Antagonism of an AMPA-Sensitive Glutamate Receptor: Crystal Structures of the GluR2 Ligand Binding Core</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">165</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span><span class="refDoi"> DOI: 10.1016/S0896-6273(00)00094-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2FS0896-6273%2800%2900094-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=11086992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=165-181&author=N.+Armstrongauthor=E.+Gouaux&title=Mechanisms+for+Activation+and+Antagonism+of+an+AMPA-Sensitive+Glutamate+Receptor%3A+Crystal+Structures+of+the+GluR2+Ligand+Binding+Core&doi=10.1016%2FS0896-6273%2800%2900094-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms for activation and antagonism of an AMPA-sensitive glutamate receptor: crystal structures of the GluR2 ligand binding core</span></div><div class="casAuthors">Armstrong, Neali; Gouaux, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">165-181</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Crystal structures of the GluR2 ligand binding core (S1S2) have been detd. in the apo state and in the presence of the antagonist DNQX, the partial agonist kainate, and the full agonists AMPA and glutamate.  The domains of the S1S2 ligand binding core are expanded in the apo state and contract upon ligand binding with the extent of domain sepn. decreasing in the order of apo > DNQX > kainate > glutamate ≃ AMPA.  These results suggest that agonist-induced domain closure gates the transmembrane channel and the extent of receptor activation depends upon the degree of domain closure.  AMPA and glutamate also promote a 180° flip of a trans peptide bond in the ligand binding site.  The crystal packing of the ligand binding cores suggests modes for subunit-subunit contact in the intact receptor and mechanisms by which allosteric effectors modulate receptor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNnbcNBzyXN7Vg90H21EOLACvtfcHk0lhZfSCoUcJlLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGltLw%253D&md5=6be3ca25ce094c142ba3977a404afb3d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0896-6273%2800%2900094-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0896-6273%252800%252900094-5%26sid%3Dliteratum%253Aachs%26aulast%3DArmstrong%26aufirst%3DN.%26aulast%3DGouaux%26aufirst%3DE.%26atitle%3DMechanisms%2520for%2520Activation%2520and%2520Antagonism%2520of%2520an%2520AMPA-Sensitive%2520Glutamate%2520Receptor%253A%2520Crystal%2520Structures%2520of%2520the%2520GluR2%2520Ligand%2520Binding%2520Core%26jtitle%3DNeuron%26date%3D2000%26volume%3D28%26spage%3D165%26epage%3D181%26doi%3D10.1016%2FS0896-6273%2800%2900094-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Honoré, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drejer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodge, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, F. E.</span><span> </span><span class="NLM_article-title">Quinoxalinediones: Potent Competitive Non-NMDA Glutamate Receptor Antagonists</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">241</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span><span class="refDoi"> DOI: 10.1126/science.2899909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1126%2Fscience.2899909" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=241&publication_year=1988&pages=701-703&author=T.+Honor%C3%A9author=S.+N.+Daviesauthor=J.+Drejerauthor=E.+J.+Fletcherauthor=P.+Jacobsenauthor=D.+Lodgeauthor=F.+E.+Nielsen&title=Quinoxalinediones%3A+Potent+Competitive+Non-NMDA+Glutamate+Receptor+Antagonists&doi=10.1126%2Fscience.2899909"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.2899909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2899909%26sid%3Dliteratum%253Aachs%26aulast%3DHonor%25C3%25A9%26aufirst%3DT.%26aulast%3DDavies%26aufirst%3DS.%2BN.%26aulast%3DDrejer%26aufirst%3DJ.%26aulast%3DFletcher%26aufirst%3DE.%2BJ.%26aulast%3DJacobsen%26aufirst%3DP.%26aulast%3DLodge%26aufirst%3DD.%26aulast%3DNielsen%26aufirst%3DF.%2BE.%26atitle%3DQuinoxalinediones%253A%2520Potent%2520Competitive%2520Non-NMDA%2520Glutamate%2520Receptor%2520Antagonists%26jtitle%3DScience%26date%3D1988%26volume%3D241%26spage%3D701%26epage%3D703%26doi%3D10.1126%2Fscience.2899909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Ornstein, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleisch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borromeo, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugar, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leander, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodge, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepp, D. D.</span><span> </span><span class="NLM_article-title">Structure-Activity Studies of 6-Substituted Decahydroisoquinoline-3- Carboxylic Acid AMPA Receptor Antagonists. 2. Effects of Distal Acid Bioisosteric Substitution, Absolute Stereochemical Preferences, and in Vivo Activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2232</span><span class="NLM_x">–</span> <span class="NLM_lpage">2244</span><span class="refDoi"> DOI: 10.1021/jm950913h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm950913h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2232-2244&author=P.+L.+Ornsteinauthor=M.+B.+Arnoldauthor=N.+K.+Allenauthor=T.+Bleischauthor=P.+S.+Borromeoauthor=C.+W.+Lugarauthor=J.+D.+Leanderauthor=D.+Lodgeauthor=D.+D.+Schoepp&title=Structure-Activity+Studies+of+6-Substituted+Decahydroisoquinoline-3-+Carboxylic+Acid+AMPA+Receptor+Antagonists.+2.+Effects+of+Distal+Acid+Bioisosteric+Substitution%2C+Absolute+Stereochemical+Preferences%2C+and+in+Vivo+Activity&doi=10.1021%2Fjm950913h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm950913h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm950913h%26sid%3Dliteratum%253Aachs%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26aulast%3DArnold%26aufirst%3DM.%2BB.%26aulast%3DAllen%26aufirst%3DN.%2BK.%26aulast%3DBleisch%26aufirst%3DT.%26aulast%3DBorromeo%26aufirst%3DP.%2BS.%26aulast%3DLugar%26aufirst%3DC.%2BW.%26aulast%3DLeander%26aufirst%3DJ.%2BD.%26aulast%3DLodge%26aufirst%3DD.%26aulast%3DSchoepp%26aufirst%3DD.%2BD.%26atitle%3DStructure-Activity%2520Studies%2520of%25206-Substituted%2520Decahydroisoquinoline-3-%2520Carboxylic%2520Acid%2520AMPA%2520Receptor%2520Antagonists.%25202.%2520Effects%2520of%2520Distal%2520Acid%2520Bioisosteric%2520Substitution%252C%2520Absolute%2520Stereochemical%2520Preferences%252C%2520and%2520in%2520Vivo%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2232%26epage%3D2244%26doi%3D10.1021%2Fjm950913h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Ornstein, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleisch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borromeo, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugar, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leander, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodge, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepp, D. D.</span><span> </span><span class="NLM_article-title">Structure-Activity Studies of 6-(Tetrazolylalkyl)-Substituted Decahydroisoquinoline-3-Carboxylic Acid AMPA Receptor Antagonists. 1. Effects of Stereochemistry, Chain Length, and Chain Substitution</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2219</span><span class="NLM_x">–</span> <span class="NLM_lpage">2231</span><span class="refDoi"> DOI: 10.1021/jm950912p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm950912p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2219-2231&author=P.+L.+Ornsteinauthor=M.+B.+Arnoldauthor=N.+K.+Allenauthor=T.+Bleischauthor=P.+S.+Borromeoauthor=C.+W.+Lugarauthor=J.+D.+Leanderauthor=D.+Lodgeauthor=D.+D.+Schoepp&title=Structure-Activity+Studies+of+6-%28Tetrazolylalkyl%29-Substituted+Decahydroisoquinoline-3-Carboxylic+Acid+AMPA+Receptor+Antagonists.+1.+Effects+of+Stereochemistry%2C+Chain+Length%2C+and+Chain+Substitution&doi=10.1021%2Fjm950912p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm950912p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm950912p%26sid%3Dliteratum%253Aachs%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26aulast%3DArnold%26aufirst%3DM.%2BB.%26aulast%3DAllen%26aufirst%3DN.%2BK.%26aulast%3DBleisch%26aufirst%3DT.%26aulast%3DBorromeo%26aufirst%3DP.%2BS.%26aulast%3DLugar%26aufirst%3DC.%2BW.%26aulast%3DLeander%26aufirst%3DJ.%2BD.%26aulast%3DLodge%26aufirst%3DD.%26aulast%3DSchoepp%26aufirst%3DD.%2BD.%26atitle%3DStructure-Activity%2520Studies%2520of%25206-%2528Tetrazolylalkyl%2529-Substituted%2520Decahydroisoquinoline-3-Carboxylic%2520Acid%2520AMPA%2520Receptor%2520Antagonists.%25201.%2520Effects%2520of%2520Stereochemistry%252C%2520Chain%2520Length%252C%2520and%2520Chain%2520Substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2219%26epage%3D2231%26doi%3D10.1021%2Fjm950912p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Szymańska, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contreras-Sanz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serafimoska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansen, T. N.</span><span> </span><span class="NLM_article-title">A New Phenylalanine Derivative Acts as an Antagonist at the AMPA Receptor GluA2 and Introduces Partial Domain Closure: Synthesis, Resolution, Pharmacology, and Crystal Structure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">7289</span><span class="NLM_x">–</span> <span class="NLM_lpage">7298</span><span class="refDoi"> DOI: 10.1021/jm200862h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200862h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7289-7298&author=E.+Szyma%C5%84skaauthor=K.+Frydenvangauthor=A.+Contreras-Sanzauthor=D.+S.+Pickeringauthor=E.+Frolaauthor=Z.+Serafimoskaauthor=B.+Nielsenauthor=J.+S.+Kastrupauthor=T.+N.+Johansen&title=A+New+Phenylalanine+Derivative+Acts+as+an+Antagonist+at+the+AMPA+Receptor+GluA2+and+Introduces+Partial+Domain+Closure%3A+Synthesis%2C+Resolution%2C+Pharmacology%2C+and+Crystal+Structure&doi=10.1021%2Fjm200862h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm200862h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200862h%26sid%3Dliteratum%253Aachs%26aulast%3DSzyma%25C5%2584ska%26aufirst%3DE.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DContreras-Sanz%26aufirst%3DA.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DFrola%26aufirst%3DE.%26aulast%3DSerafimoska%26aufirst%3DZ.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26aulast%3DJohansen%26aufirst%3DT.%2BN.%26atitle%3DA%2520New%2520Phenylalanine%2520Derivative%2520Acts%2520as%2520an%2520Antagonist%2520at%2520the%2520AMPA%2520Receptor%2520GluA2%2520and%2520Introduces%2520Partial%2520Domain%2520Closure%253A%2520Synthesis%252C%2520Resolution%252C%2520Pharmacology%252C%2520and%2520Crystal%2520Structure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7289%26epage%3D7298%26doi%3D10.1021%2Fjm200862h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">More, J. C. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troop, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolman, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jane, D. E.</span><span> </span><span class="NLM_article-title">Structural Requirements for Novel Willardiine Derivatives Acting as AMPA and Kainate Receptor Antagonists</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1100</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0705148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1038%2Fsj.bjp.0705148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=12684265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtVahsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2003&pages=1093-1100&author=J.+C.+a+Moreauthor=H.+M.+Troopauthor=N.+P.+Dolmanauthor=D.+E.+Jane&title=Structural+Requirements+for+Novel+Willardiine+Derivatives+Acting+as+AMPA+and+Kainate+Receptor+Antagonists&doi=10.1038%2Fsj.bjp.0705148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structural requirements for novel willardiine derivatives acting as AMPA and kainate receptor antagonists</span></div><div class="casAuthors">More, Julia C. A.; Troop, Helen M.; Dolman, Nigel P.; Jane, David E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1093-1100</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The natural product willardiine is an AMPA receptor agonist.  We have examd. the structural changes required to convert willardiine into an antagonist at AMPA and kainate receptors.  Structure-activity anal. has been carried out to discover the structural features required to increase the potency and/or selectivity of the antagonists at AMPA or kainate receptors.  Redn. of the fast component of the dorsal root-evoked ventral root potential (fDR-VRP) has been used to investigate AMPA receptor antagonist activity.  To examine antagonist activity at kainate receptors, the ability of compds. to depress kainate-induced depolarizations of dorsal root fibers was assessed.  Blocking ionization of the uracil ring by adding a Me group to the N3 position was not sufficient to convert willardiine into an antagonist.  However, willardiine derivs. with a side-chain bearing a carboxylic acid group at the N3-position of the uracil ring could antagonize AMPA and kainate receptors.  S stereochem. was optimal for antagonism.  When compds. with differing interacidic group chain lengths were compared, a group chain length of two methylene groups was preferable for AMPA receptor antagonism in the series of compds. bearing a carboxyalkyl side chain (UBP275, UBP277 and UBP279 reduced the fDR-VRP with IC50 values of 287 ± 41, 23.8 ± 3.9 and 136 ± 17 μM, resp.).  For kainate receptor antagonism, two or three methylene groups were almost equally acceptable (UBP277 and UBP279 reduced dorsal root kainate responses with apparent KD values of 73.1 ± 4.5 and 60.5 ± 4.1 μM, resp.).  Adding an iodo group to the 5-position of UBP277 and UBP282 enhanced activity at kainate receptors (UBP291 and UBP301 antagonized kainate responses on the dorsal root with apparent KD values of 9.83 ± 1.62 and 5.94 ± 0.63 μM, resp.).  The most useful antagonist identified in this study was UBP301, which was a potent and ∼30-fold selective kainate receptor antagonist.  UBP282 may also be of use in isolating a non-GluR5-mediated kainate response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5Qgxlye0jorVg90H21EOLACvtfcHk0liqUyb4NYVf-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtVahsbk%253D&md5=1005bbe6195892c0bef935b0ebe75f1f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705148%26sid%3Dliteratum%253Aachs%26aulast%3DMore%26aufirst%3DJ.%2BC.%2Ba%26aulast%3DTroop%26aufirst%3DH.%2BM.%26aulast%3DDolman%26aufirst%3DN.%2BP.%26aulast%3DJane%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520Requirements%2520for%2520Novel%2520Willardiine%2520Derivatives%2520Acting%2520as%2520AMPA%2520and%2520Kainate%2520Receptor%2520Antagonists%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D138%26spage%3D1093%26epage%3D1100%26doi%3D10.1038%2Fsj.bjp.0705148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Larsen, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venskutonytė, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valadés, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Discovery of a New Class of Ionotropic Glutamate Receptor Antagonists by the Rational Design of (2S,3R)-3-(3-Carboxyphenyl)-Pyrrolidine-2-Carboxylic Acid</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span><span class="refDoi"> DOI: 10.1021/cn100093f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100093f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=107-114&author=A.+M.+Larsenauthor=R.+Venskutonyt%C4%97author=E.+A.+Valad%C3%A9sauthor=B.+Nielsenauthor=D.+S.+Pickeringauthor=L.+Bunch&title=Discovery+of+a+New+Class+of+Ionotropic+Glutamate+Receptor+Antagonists+by+the+Rational+Design+of+%282S%2C3R%29-3-%283-Carboxyphenyl%29-Pyrrolidine-2-Carboxylic+Acid&doi=10.1021%2Fcn100093f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a New Class of Ionotropic Glutamate Receptor Antagonists by the Rational Design of (2S,3R)-3-(3-Carboxyphenyl)-pyrrolidine-2-carboxylic Acid</span></div><div class="casAuthors">Larsen, Ann M.; Venskutonyte, Raminta; Valades, Elena Anton; Nielsen, Birgitte; Pickering, Darryl S.; Bunch, Lennart</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">107-114</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The kainic acid (KA) receptors belong to the class of glutamate (Glu) receptors in the brain and constitute a promising target for the treatment of neurol. and/or psychiatric diseases such as schizophrenia, major depression, and epilepsy.  Five KA subtypes have been identified and named GluK1-5.  In this article, we present the discovery of (2S,3R)-3-(3-carboxyphenyl)-pyrrolidine-2-carboxylic acid (1) based on a rational design process.  Target compd. 1 was synthesized by a stereoselective strategy in 10 steps from com. available starting materials.  Binding affinities of 1 at native ionotropic Glu receptors were detd. to be in the micromolar range (AMPA, 51 μM; KA, 22 μM; NMDA 6 μM), with the highest affinity for cloned homomeric KA receptor subtypes GluK1,3 (3.0 and 8.1 μM, resp.).  Functional characterization of 1 by two electrode voltage clamp (TEVC) electrophysiol. at a nondesensitizing mutant of GluK1 showed full competitive antagonistic behavior with a Kb of 11.4 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe72VGdj51HrVg90H21EOLACvtfcHk0liqUyb4NYVf-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbvE&md5=dfe53691cb11fa493b0aaaec96a60909</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcn100093f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100093f%26sid%3Dliteratum%253Aachs%26aulast%3DLarsen%26aufirst%3DA.%2BM.%26aulast%3DVenskutonyt%25C4%2597%26aufirst%3DR.%26aulast%3DValad%25C3%25A9s%26aufirst%3DE.%2BA.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520a%2520New%2520Class%2520of%2520Ionotropic%2520Glutamate%2520Receptor%2520Antagonists%2520by%2520the%2520Rational%2520Design%2520of%2520%25282S%252C3R%2529-3-%25283-Carboxyphenyl%2529-Pyrrolidine-2-Carboxylic%2520Acid%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2011%26volume%3D2%26spage%3D107%26epage%3D114%26doi%3D10.1021%2Fcn100093f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Krogsgaard-Larsen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storgaard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Møller, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demmer, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monrad, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Structure-Activity Relationship Study of Ionotropic Glutamate Receptor Antagonist (2S,3R)-3-(3-Carboxyphenyl)pyrrolidine-2-Carboxylic Acid</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">6131</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00750</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00750" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6131-6150&author=N.+Krogsgaard-Larsenauthor=M.+Storgaardauthor=C.+M%C3%B8llerauthor=C.+S.+Demmerauthor=J.+Hansenauthor=L.+Hanauthor=R.+N.+Monradauthor=B.+Nielsenauthor=D.+Tapkenauthor=D.+S.+Pickeringauthor=J.+S.+Kastrupauthor=K.+Frydenvangauthor=L.+Bunch&title=Structure-Activity+Relationship+Study+of+Ionotropic+Glutamate+Receptor+Antagonist+%282S%2C3R%29-3-%283-Carboxyphenyl%29pyrrolidine-2-Carboxylic+Acid&doi=10.1021%2Facs.jmedchem.5b00750"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00750%26sid%3Dliteratum%253Aachs%26aulast%3DKrogsgaard-Larsen%26aufirst%3DN.%26aulast%3DStorgaard%26aufirst%3DM.%26aulast%3DM%25C3%25B8ller%26aufirst%3DC.%26aulast%3DDemmer%26aufirst%3DC.%2BS.%26aulast%3DHansen%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DMonrad%26aufirst%3DR.%2BN.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DTapken%26aufirst%3DD.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DStructure-Activity%2520Relationship%2520Study%2520of%2520Ionotropic%2520Glutamate%2520Receptor%2520Antagonist%2520%25282S%252C3R%2529-3-%25283-Carboxyphenyl%2529pyrrolidine-2-Carboxylic%2520Acid%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6131%26epage%3D6150%26doi%3D10.1021%2Facs.jmedchem.5b00750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Weiss, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gates, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dieckman, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemens-Smith, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizkalla, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calligaro, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepp, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattiuz, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salhoff, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katofiasc, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phebus, L. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenck, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filla, S. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ornstein, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleakman, D.</span><span> </span><span class="NLM_article-title">Pharmacological Characterization of the Competitive GLU K5 Receptor Antagonist Decahydroisoquinoline LY466195 in Vitro and in Vivo</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">781</span><span class="refDoi"> DOI: 10.1124/jpet.106.101428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1124%2Fjpet.106.101428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=16690725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD28XnslOltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2006&pages=772-781&author=B.+Weissauthor=A.+Altauthor=A.+M.+Ogdenauthor=M.+Gatesauthor=D.+K.+Dieckmanauthor=A.+Clemens-Smithauthor=K.+H.+Hoauthor=K.+Jarvieauthor=G.+Rizkallaauthor=R.+A.+Wrightauthor=D.+O.+Calligaroauthor=D.+Schoeppauthor=E.+L.+Mattiuzauthor=R.+E.+Stratfordauthor=B.+Johnsonauthor=C.+Salhoffauthor=M.+Katofiascauthor=L.+a+Phebusauthor=K.+Schenckauthor=M.+Cohenauthor=S.+a+Fillaauthor=P.+L.+Ornsteinauthor=K.+W.+Johnsonauthor=D.+Bleakman&title=Pharmacological+Characterization+of+the+Competitive+GLU+K5+Receptor+Antagonist+Decahydroisoquinoline+LY466195+in+Vitro+and+in+Vivo&doi=10.1124%2Fjpet.106.101428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of the competitive GLUK5 receptor antagonist decahydroisoquinoline LY466195 in vitro and in vivo</span></div><div class="casAuthors">Weiss, Brianne; Alt, Andrew; Ogden, Ann Marie; Gates, Mary; Dieckman, Donna K.; Clemens-Smith, Amy; Ho, Ken H.; Jarvie, Keith; Rizkalla, Geihan; Wright, Rebecca A.; Calligaro, David O.; Schoepp, Darryle; Mattiuz, Edward L.; Stratford, Robert E.; Johnson, Bryan; Salhoff, Craig; Katofiasc, Mary; Phebus, Lee A.; Schenck, Kathryn; Cohen, Marlene; Filla, Sandra A.; Ornstein, Paul L.; Johnson, Kirk W.; Bleakman, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The excitatory neurotransmitter glutamate has been implicated in both migraine and persistent pain.  The identification of the kainate receptor GLUK5 in dorsal root ganglia, the dorsal horn, and trigeminal ganglia makes it a target of interest for these indications.  We examd. the in vitro and in vivo pharmacol. of the competitive GLUK5-selective kainate receptor antagonist LY466195, the most potent GLUK5 antagonist described to date.  Comparisons were made to the competitive GLUK5/α-amino-3-hydroxy-5-methyl-4-isoxazole propionic acid (AMPA) receptor antagonist LY293558 [(3S,4aR,6R,8aR)-6-[2-(1(2)H-tetrazole-5-yl)ethyl]decahydroisoquinoline-3-carboxylic acid], other decahydroisoquinoline GLUK5 receptor antagonists, and the noncompetitive AMPA receptor antagonist LY300168 [1-(4-aminophenyl)-4-methyl-7,8-methylenedioxy-5H-2,3-benzodi-azepine].  When characterized electrophysiol. in rat dorsal root ganglion neurons, LY466195 antagonized kainate (30 μM)-induced currents with an IC50 value of 0.045 ± 0.011 μM.  In HEK293 cells transfected with GLUK5, GLUK2/GLUK5, or GLUK5/GLUK6 receptors, LY466195 produced IC50 values of 0.08 ± 0.02, 0.34 ± 0.17, and 0.07 ± 0.02 μM, resp.  LY466195 was efficacious in a dural plasma protein extravasation (PPE) model of migraine with an ID100 value of 100 μg/kg i.v.  LY466195 was also efficacious in the c-fos migraine model, with a dose of 1 μg/kg i.v. significantly reducing the no. of Fos-pos. cells in the rat nucleus caudalis after elec. stimulation of the trigeminal ganglion.  Furthermore, LY466195 showed no contractile activity in the rabbit saphenous vein in vitro.  The di-Et ester prodrug of LY466195 was also efficacious in the same PPE and c-fos models after oral administration at doses of 10 and 100 μg/kg, resp. while having no N-methyl-D-aspartate antagonist-like behavioral effects at oral doses up to 100 mg/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwe6dtAoo61bVg90H21EOLACvtfcHk0liBhaUWFtVDmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnslOltro%253D&md5=decb91d0148d7a4049f15e1c4622925a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.101428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.101428%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DB.%26aulast%3DAlt%26aufirst%3DA.%26aulast%3DOgden%26aufirst%3DA.%2BM.%26aulast%3DGates%26aufirst%3DM.%26aulast%3DDieckman%26aufirst%3DD.%2BK.%26aulast%3DClemens-Smith%26aufirst%3DA.%26aulast%3DHo%26aufirst%3DK.%2BH.%26aulast%3DJarvie%26aufirst%3DK.%26aulast%3DRizkalla%26aufirst%3DG.%26aulast%3DWright%26aufirst%3DR.%2BA.%26aulast%3DCalligaro%26aufirst%3DD.%2BO.%26aulast%3DSchoepp%26aufirst%3DD.%26aulast%3DMattiuz%26aufirst%3DE.%2BL.%26aulast%3DStratford%26aufirst%3DR.%2BE.%26aulast%3DJohnson%26aufirst%3DB.%26aulast%3DSalhoff%26aufirst%3DC.%26aulast%3DKatofiasc%26aufirst%3DM.%26aulast%3DPhebus%26aufirst%3DL.%2Ba%26aulast%3DSchenck%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DM.%26aulast%3DFilla%26aufirst%3DS.%2Ba%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26aulast%3DJohnson%26aufirst%3DK.%2BW.%26aulast%3DBleakman%26aufirst%3DD.%26atitle%3DPharmacological%2520Characterization%2520of%2520the%2520Competitive%2520GLU%2520K5%2520Receptor%2520Antagonist%2520Decahydroisoquinoline%2520LY466195%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2006%26volume%3D318%26spage%3D772%26epage%3D781%26doi%3D10.1124%2Fjpet.106.101428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Mayer, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolman, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jane, D. E.</span><span> </span><span class="NLM_article-title">Crystal Structures of the Kainate Receptor GluR5 Ligand Binding Core Dimer with Novel GluR5-Selective Antagonists</span> <span class="citation_source-journal">J. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">2852</span><span class="NLM_x">–</span> <span class="NLM_lpage">2861</span><span class="refDoi"> DOI: 10.1523/JNEUROSCI.0123-06.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1523%2FJNEUROSCI.0123-06.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=16540562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD28XivFKmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2852-2861&author=M.+L.+Mayerauthor=A.+Ghosalauthor=N.+P.+Dolmanauthor=D.+E.+Jane&title=Crystal+Structures+of+the+Kainate+Receptor+GluR5+Ligand+Binding+Core+Dimer+with+Novel+GluR5-Selective+Antagonists&doi=10.1523%2FJNEUROSCI.0123-06.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the kainate receptor GluR5 ligand binding core dimer with novel GluR5-selective antagonists</span></div><div class="casAuthors">Mayer, Mark L.; Ghosal, Alokesh; Dolman, Nigel P.; Jane, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2852-2861</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Glutamate receptor (GluR) ion channels mediate fast synaptic transmission in the mammalian CNS.  Numerous crystallog. studies, the majority on the GluR2-subtype AMPA receptor, have revealed the structural basis for binding of subtype-specific agonists.  In contrast, because there are far fewer antagonist-bound structures, the mechanisms for antagonist binding are much less well understood, particularly for kainate receptors that exist as multiple subtypes with a distinct biol. encoded by the GluR5-7, KA1, and KA2 genes.  We describe here high-resoln. crystal structures for the GluR5 ligand-binding core complex with UBP302 and UBP310, novel GluR5-selective antagonists.  The crystal structures reveal the structural basis for the high selectivity for GluR5 obsd. in radiolabel displacement assays for the isolated ligand binding cores of the GluR2, GluR5, and GluR6 subunits and during inhibition of glutamate-activated currents in studies on full-length ion channels.  The antagonists bind via a novel mechanism and do not form direct contacts with the E723 side chain, as occurs in all previously solved AMPA and kainate receptor agonist and antagonist complexes.  This results from a hyperextension of the ligand binding core compared with previously solved structures.  As a result, in dimer assemblies, there is a 22 Å extension of the ion channel linkers in the transition from antagonist- to glutamate-bound forms.  This large conformational change is substantially different from that described for AMPA receptors, was not possible to predict from previous work, and suggests that glutamate receptors are capable of much larger movements than previously thought.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFkS4TUumki7Vg90H21EOLACvtfcHk0liBhaUWFtVDmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivFKmtLs%253D&md5=5a880d1af759237096a2919d856d5e2a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.0123-06.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.0123-06.2005%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%2BL.%26aulast%3DGhosal%26aufirst%3DA.%26aulast%3DDolman%26aufirst%3DN.%2BP.%26aulast%3DJane%26aufirst%3DD.%2BE.%26atitle%3DCrystal%2520Structures%2520of%2520the%2520Kainate%2520Receptor%2520GluR5%2520Ligand%2520Binding%2520Core%2520Dimer%2520with%2520Novel%2520GluR5-Selective%2520Antagonists%26jtitle%3DJ.%2520Neurosci.%26date%3D2006%26volume%3D26%26spage%3D2852%26epage%3D2861%26doi%3D10.1523%2FJNEUROSCI.0123-06.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Martinez-Perez, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyengar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shannon, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleakman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleisch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castaño, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Prado, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominguez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escribano, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filla, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudziak, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mateo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathes, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattiuz, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, A. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmons, R. M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stack, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ornstein, P. L.</span><span> </span><span class="NLM_article-title">GluK1 Antagonists from 6-(Carboxy)phenyl Decahydroisoquinoline Derivatives. SAR and Evaluation of a Prodrug Strategy for Oral Efficacy in Pain Models</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">6459</span><span class="NLM_x">–</span> <span class="NLM_lpage">6462</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.bmcl.2013.09.046" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6459-6462&author=J.+A.+Martinez-Perezauthor=S.+Iyengarauthor=H.+E.+Shannonauthor=D.+Bleakmanauthor=A.+Altauthor=B.+M.+Arnoldauthor=M.+G.+Bellauthor=T.+J.+Bleischauthor=A.+M.+Casta%C3%B1oauthor=M.+Del+Pradoauthor=E.+Dominguezauthor=A.+M.+Escribanoauthor=S.+A.+Fillaauthor=K.+H.+Hoauthor=K.+J.+Hudziakauthor=C.+K.+Jonesauthor=A.+Mateoauthor=B.+M.+Mathesauthor=E.+L.+Mattiuzauthor=A.+M.+L.+Ogdenauthor=R.+M.+A.+Simmonsauthor=D.+R.+Stackauthor=R.+E.+Stratfordauthor=M.+A.+Winterauthor=Z.+Wuauthor=P.+L.+Ornstein&title=GluK1+Antagonists+from+6-%28Carboxy%29phenyl+Decahydroisoquinoline+Derivatives.+SAR+and+Evaluation+of+a+Prodrug+Strategy+for+Oral+Efficacy+in+Pain+Models&doi=10.1016%2Fj.bmcl.2013.09.046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.046%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez-Perez%26aufirst%3DJ.%2BA.%26aulast%3DIyengar%26aufirst%3DS.%26aulast%3DShannon%26aufirst%3DH.%2BE.%26aulast%3DBleakman%26aufirst%3DD.%26aulast%3DAlt%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DB.%2BM.%26aulast%3DBell%26aufirst%3DM.%2BG.%26aulast%3DBleisch%26aufirst%3DT.%2BJ.%26aulast%3DCasta%25C3%25B1o%26aufirst%3DA.%2BM.%26aulast%3DDel%2BPrado%26aufirst%3DM.%26aulast%3DDominguez%26aufirst%3DE.%26aulast%3DEscribano%26aufirst%3DA.%2BM.%26aulast%3DFilla%26aufirst%3DS.%2BA.%26aulast%3DHo%26aufirst%3DK.%2BH.%26aulast%3DHudziak%26aufirst%3DK.%2BJ.%26aulast%3DJones%26aufirst%3DC.%2BK.%26aulast%3DMateo%26aufirst%3DA.%26aulast%3DMathes%26aufirst%3DB.%2BM.%26aulast%3DMattiuz%26aufirst%3DE.%2BL.%26aulast%3DOgden%26aufirst%3DA.%2BM.%2BL.%26aulast%3DSimmons%26aufirst%3DR.%2BM.%2BA.%26aulast%3DStack%26aufirst%3DD.%2BR.%26aulast%3DStratford%26aufirst%3DR.%2BE.%26aulast%3DWinter%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26atitle%3DGluK1%2520Antagonists%2520from%25206-%2528Carboxy%2529phenyl%2520Decahydroisoquinoline%2520Derivatives.%2520SAR%2520and%2520Evaluation%2520of%2520a%2520Prodrug%2520Strategy%2520for%2520Oral%2520Efficacy%2520in%2520Pain%2520Models%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6459%26epage%3D6462%26doi%3D10.1016%2Fj.bmcl.2013.09.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Jensen, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bräuner-Osborne, H.</span><span> </span><span class="NLM_article-title">Pharmacological Characterization of Human Excitatory Amino Acid Transporters EAAT1, EAAT2 and EAAT3 in a Fluorescence-Based Membrane Potential Assay</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">2115</span><span class="NLM_x">–</span> <span class="NLM_lpage">2127</span><span class="refDoi"> DOI: 10.1016/j.bcp.2004.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.bcp.2004.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=15135308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvVCis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2004&pages=2115-2127&author=A.+A.+Jensenauthor=H.+Br%C3%A4uner-Osborne&title=Pharmacological+Characterization+of+Human+Excitatory+Amino+Acid+Transporters+EAAT1%2C+EAAT2+and+EAAT3+in+a+Fluorescence-Based+Membrane+Potential+Assay&doi=10.1016%2Fj.bcp.2004.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of human excitatory amino acid transporters EAAT1, EAAT2 and EAAT3 in a fluorescence-based membrane potential assay</span></div><div class="casAuthors">Jensen, Anders A.; Brauner-Osborne, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2115-2127</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We have expressed the human excitatory amino acid transporters EAAT1, EAAT2 and EAAT3 stably in HEK293 cells and characterized the transporters pharmacol. in a conventional [3H]-d-aspartate uptake assay and in a fluorescence-based membrane potential assay, the FLIPR Membrane Potential (FMP) assay.  The Km and Ki values obtained for 12 std. EAAT ligands at EAAT1, EAAT2 and EAAT3 in the FMP assay correlated well with the Ki values obtained in the [3H]-d-aspartate assay (r2 values of 0.92, 0.92, and 0.95, resp.).  Furthermore, the pharmacol. characteristics of the cell lines in the FMP assay were in good agreement with previous findings in electrophysiol. studies of the transporters.  The FMP assay was capable of distinguishing between substrates and non-substrate inhibitors and to discriminate between "full" and "partial" substrates at the transporters.  Taking advantage of the prolific nature of the FMP assay, interactions of the EAATs with substrates and inhibitors were studied in some detail.  This is the first report of a high throughput screening assay for EAATs.  We propose that the assay will be of great use in future studies of the transporters.  Although conventional electrophysiol. set-ups might be superior in terms of studying sophisticated kinetic aspects of the uptake process, the FMP assay enables the collection of considerable amts. of highly reproducible data with relatively little labor.  Furthermore, considering that the no. of EAAT ligands presently available is limited, and that almost all of these are characterized by low potency and a low degree of subtype selectivity, future screening of compd. libraries at the EAAT-cell lines in the FMP assay could help identify structurally and pharmacol. novel ligands for the transporters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS6Um79wVYz7Vg90H21EOLACvtfcHk0ljZ6ndopQTzpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvVCis78%253D&md5=06e3c536844acca52fdba733175a93c6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2004.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2004.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DA.%2BA.%26aulast%3DBr%25C3%25A4uner-Osborne%26aufirst%3DH.%26atitle%3DPharmacological%2520Characterization%2520of%2520Human%2520Excitatory%2520Amino%2520Acid%2520Transporters%2520EAAT1%252C%2520EAAT2%2520and%2520EAAT3%2520in%2520a%2520Fluorescence-Based%2520Membrane%2520Potential%2520Assay%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2004%26volume%3D67%26spage%3D2115%26epage%3D2127%26doi%3D10.1016%2Fj.bcp.2004.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Møllerud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span> </span><span class="NLM_article-title">Lessons from Crystal Structures of Kainate Receptors</span> <span class="citation_source-journal">Neuropharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">16</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span><span class="refDoi"> DOI: 10.1016/j.neuropharm.2016.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.neuropharm.2016.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=27236079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FksVWhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2017&pages=16-28&author=S.+M%C3%B8llerudauthor=K.+Frydenvangauthor=D.+S.+Pickeringauthor=J.+S.+Kastrup&title=Lessons+from+Crystal+Structures+of+Kainate+Receptors&doi=10.1016%2Fj.neuropharm.2016.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from crystal structures of kainate receptors</span></div><div class="casAuthors">Mollerud Stine; Frydenvang Karla; Pickering Darryl S; Kastrup Jette Sandholm</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">Pt A</span>),
    <span class="NLM_cas:pages">16-28</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kainate receptors belong to the family of ionotropic glutamate receptors.  These receptors assemble from five subunits (GluK1-5) into tetrameric ion channels.  Kainate receptors are located at both pre- and postsynaptic membranes in the central nervous system where they contribute to excitatory synaptic transmission and modulate network excitability by regulating neurotransmitter release.  Dysfunction of kainate receptors has been implicated in several neurological disorders such as epilepsy, schizophrenia and depression.  Here we provide a review on the current understanding of kainate receptor structure and how they bind agonists, antagonists and ions.  The first structure of the ligand-binding domain of the GluK1 subunit was reported in 2005, seven years after publication of the crystal structure of a soluble construct of the ligand-binding domain of the AMPA-type subunit GluA2.  Today, a full-length structure has been determined of GluK2 by cryo electron microscopy to 7.6 ÅA resolution as well as 84 high-resolution crystal structures of N-terminal domains and ligand-binding domains, including agonist and antagonist bound structures, modulatory ions and mutations.  However, there are still many unanswered questions and challenges in front of us.  This article is part of the Special Issue entitled 'Ionotropic glutamate receptors'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSorFhi6dO1iNhSDZx9hyjKfW6udTcc2eav45ANODJNdLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FksVWhsw%253D%253D&md5=7b98312e2ef8a69ecac23e2d2a045bb7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2016.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2016.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25B8llerud%26aufirst%3DS.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26atitle%3DLessons%2520from%2520Crystal%2520Structures%2520of%2520Kainate%2520Receptors%26jtitle%3DNeuropharmacology%26date%3D2017%26volume%3D112%26spage%3D16%26epage%3D28%26doi%3D10.1016%2Fj.neuropharm.2016.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Hald, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprogøe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madsen, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmermann, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahring, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liljefors, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schousboe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egebjerg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajhede, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span> </span><span class="NLM_article-title">Partial Agonism and Antagonism of the Ionotropic Glutamate Receptor iGluR5: Structures of the Ligand-Binding Core in Complex with Domoic Acid and 2-Amino-3-[5-Tert-Butyl-3-(Phosphonomethoxy)-4-Isoxazolyl]propionic Acid</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">25726</span><span class="NLM_x">–</span> <span class="NLM_lpage">25736</span><span class="refDoi"> DOI: 10.1074/jbc.M700137200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1074%2Fjbc.M700137200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=25726-25736&author=H.+Haldauthor=P.+Naurauthor=D.+S.+Pickeringauthor=D.+Sprog%C3%B8eauthor=U.+Madsenauthor=D.+B.+Timmermannauthor=P.+K.+Ahringauthor=T.+Liljeforsauthor=A.+Schousboeauthor=J.+Egebjergauthor=M.+Gajhedeauthor=J.+S.+Kastrup&title=Partial+Agonism+and+Antagonism+of+the+Ionotropic+Glutamate+Receptor+iGluR5%3A+Structures+of+the+Ligand-Binding+Core+in+Complex+with+Domoic+Acid+and+2-Amino-3-%5B5-Tert-Butyl-3-%28Phosphonomethoxy%29-4-Isoxazolyl%5Dpropionic+Acid&doi=10.1074%2Fjbc.M700137200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M700137200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M700137200%26sid%3Dliteratum%253Aachs%26aulast%3DHald%26aufirst%3DH.%26aulast%3DNaur%26aufirst%3DP.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DSprog%25C3%25B8e%26aufirst%3DD.%26aulast%3DMadsen%26aufirst%3DU.%26aulast%3DTimmermann%26aufirst%3DD.%2BB.%26aulast%3DAhring%26aufirst%3DP.%2BK.%26aulast%3DLiljefors%26aufirst%3DT.%26aulast%3DSchousboe%26aufirst%3DA.%26aulast%3DEgebjerg%26aufirst%3DJ.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26atitle%3DPartial%2520Agonism%2520and%2520Antagonism%2520of%2520the%2520Ionotropic%2520Glutamate%2520Receptor%2520iGluR5%253A%2520Structures%2520of%2520the%2520Ligand-Binding%2520Core%2520in%2520Complex%2520with%2520Domoic%2520Acid%2520and%25202-Amino-3-%255B5-Tert-Butyl-3-%2528Phosphonomethoxy%2529-4-Isoxazolyl%255Dpropionic%2520Acid%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D25726%26epage%3D25736%26doi%3D10.1074%2Fjbc.M700137200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lghbErzuPwVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Epple, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karanewsky, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bursulaya, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kreusch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuntland, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerken, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iskandar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin Seidel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, S. S.</span><span> </span><span class="NLM_article-title">3,4,5-Trisubstituted Isoxazoles as Novel PPARδ Agonists. Part 2</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5488</span><span class="NLM_x">–</span> <span class="NLM_lpage">5492</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2006.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.bmcl.2006.08.052" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=5488-5492&author=R.+Eppleauthor=M.+Azimioaraauthor=R.+Russoauthor=Y.+Xieauthor=X.+Wangauthor=C.+Cowauthor=J.+Wityakauthor=D.+Karanewskyauthor=B.+Bursulayaauthor=A.+Kreuschauthor=T.+Tuntlandauthor=A.+Gerkenauthor=M.+Iskandarauthor=E.+Saezauthor=H.+Martin+Seidelauthor=S.+S.+Tian&title=3%2C4%2C5-Trisubstituted+Isoxazoles+as+Novel+PPAR%CE%B4+Agonists.+Part+2&doi=10.1016%2Fj.bmcl.2006.08.052"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DAzimioara%26aufirst%3DM.%26aulast%3DRusso%26aufirst%3DR.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCow%26aufirst%3DC.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DKaranewsky%26aufirst%3DD.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DKreusch%26aufirst%3DA.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DGerken%26aufirst%3DA.%26aulast%3DIskandar%26aufirst%3DM.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DMartin%2BSeidel%26aufirst%3DH.%26aulast%3DTian%26aufirst%3DS.%2BS.%26atitle%3D3%252C4%252C5-Trisubstituted%2520Isoxazoles%2520as%2520Novel%2520PPAR%25CE%25B4%2520Agonists.%2520Part%25202%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D5488%26epage%3D5492%26doi%3D10.1016%2Fj.bmcl.2006.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Assaf, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venskutonytė, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abrahamsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajhede, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, A. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gefflaut, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Chemoenzymatic Synthesis of New 2,4-Syn-Functionalized (S)-Glutamate Analogues and Structure-Activity Relationship Studies at Ionotropic Glutamate Receptors and Excitatory Amino Acid Transporters</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1614</span><span class="NLM_x">–</span> <span class="NLM_lpage">1628</span><span class="refDoi"> DOI: 10.1021/jm301433m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301433m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1614-1628&author=Z.+Assafauthor=A.+P.+Larsenauthor=R.+Venskutonyt%C4%97author=L.+Hanauthor=B.+Abrahamsenauthor=B.+Nielsenauthor=M.+Gajhedeauthor=J.+S.+Kastrupauthor=A.+a+Jensenauthor=D.+S.+Pickeringauthor=K.+Frydenvangauthor=T.+Gefflautauthor=L.+Bunch&title=Chemoenzymatic+Synthesis+of+New+2%2C4-Syn-Functionalized+%28S%29-Glutamate+Analogues+and+Structure-Activity+Relationship+Studies+at+Ionotropic+Glutamate+Receptors+and+Excitatory+Amino+Acid+Transporters&doi=10.1021%2Fjm301433m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoenzymatic synthesis of new 2,4-syn-functionalized (S)-glutamate analogues and structure-activity relationship studies at ionotropic glutamate receptors and excitatory amino acid transporters</span></div><div class="casAuthors">Assaf, Zeinab; Larsen, Anja P.; Venskutonyte, Raminta; Han, Liwei; Abrahamsen, Bjarke; Nielsen, Birgitte; Gajhede, Michael; Kastrup, Jette S.; Jensen, Anders A.; Pickering, Darryl S.; Frydenvang, Karla; Gefflaut, Thierry; Bunch, Lennart</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1614-1628</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the mammalian central nervous system, (S)-glutamate (Glu) is released from the presynaptic neuron where it activates a plethora of pre- and postsynaptic Glu receptors.  The fast acting ionotropic Glu receptors (iGluRs) are ligand gated ion channels and are believed to be involved in a vast no. of neurol. functions such as memory and learning, synaptic plasticity, and motor function.  The synthesis of 14 enantiopure 2,4-syn-Glu analogs, e.g. (I), (II), (III), is accessed by a short and efficient chemoenzymic approach starting from readily available cyclohexanone (IV).  Pharmacol. characterization at the iGluRs and EAAT1-3 subtypes revealed analog II as a selective GluK1 ligand with low nanomolar affinity.  Two X-ray crystal structures of the key analog II in the ligand-binding domain (LBD) of GluA2 and GluK3 were detd.  Partial domain closure was seen in the GluA2-LBD complex with II comparable to that induced by kainate.  In contrast, full domain closure was obsd. in the GluK3-LBD complex with II, similar to that of GluK3-LBD with glutamate bound.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNDU3_wI6ErbVg90H21EOLACvtfcHk0lghbErzuPwVSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbs%253D&md5=820d359e25bfc4d9505c20bc303cd67d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm301433m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301433m%26sid%3Dliteratum%253Aachs%26aulast%3DAssaf%26aufirst%3DZ.%26aulast%3DLarsen%26aufirst%3DA.%2BP.%26aulast%3DVenskutonyt%25C4%2597%26aufirst%3DR.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DAbrahamsen%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26aulast%3DJensen%26aufirst%3DA.%2Ba%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DGefflaut%26aufirst%3DT.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DChemoenzymatic%2520Synthesis%2520of%2520New%25202%252C4-Syn-Functionalized%2520%2528S%2529-Glutamate%2520Analogues%2520and%2520Structure-Activity%2520Relationship%2520Studies%2520at%2520Ionotropic%2520Glutamate%2520Receptors%2520and%2520Excitatory%2520Amino%2520Acid%2520Transporters%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1614%26epage%3D1628%26doi%3D10.1021%2Fjm301433m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Alcaide, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marconi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haym, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Møllerud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Synthesis and Pharmacological Characterization of the Selective GluK1 Radioligand (S)-2-Amino-3-(6-[ 3 H]-2,4-Dioxo-3,4-dihydrothieno[3,2-D]pyrimidin-1(2H)-Yl)propanoic Acid ([3H]-NF608)</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2136</span><span class="NLM_x">–</span> <span class="NLM_lpage">2144</span><span class="refDoi"> DOI: 10.1039/C6MD00339G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1039%2FC6MD00339G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2136-2144&author=A.+Alcaideauthor=L.+Marconiauthor=A.+Marekauthor=I.+Haymauthor=B.+Nielsenauthor=S.+M%C3%B8llerudauthor=M.+Jensenauthor=P.+Contiauthor=D.+S.+Pickeringauthor=L.+Bunch&title=Synthesis+and+Pharmacological+Characterization+of+the+Selective+GluK1+Radioligand+%28S%29-2-Amino-3-%286-%5B+3+H%5D-2%2C4-Dioxo-3%2C4-dihydrothieno%5B3%2C2-D%5Dpyrimidin-1%282H%29-Yl%29propanoic+Acid+%28%5B3H%5D-NF608%29&doi=10.1039%2FC6MD00339G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC6MD00339G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00339G%26sid%3Dliteratum%253Aachs%26aulast%3DAlcaide%26aufirst%3DA.%26aulast%3DMarconi%26aufirst%3DL.%26aulast%3DMarek%26aufirst%3DA.%26aulast%3DHaym%26aufirst%3DI.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DM%25C3%25B8llerud%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DM.%26aulast%3DConti%26aufirst%3DP.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520Characterization%2520of%2520the%2520Selective%2520GluK1%2520Radioligand%2520%2528S%2529-2-Amino-3-%25286-%255B%25203%2520H%255D-2%252C4-Dioxo-3%252C4-dihydrothieno%255B3%252C2-D%255Dpyrimidin-1%25282H%2529-Yl%2529propanoic%2520Acid%2520%2528%255B3H%255D-NF608%2529%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D2136%26epage%3D2144%26doi%3D10.1039%2FC6MD00339G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Naur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vestergaard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skov, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egebjerg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajhede, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span> </span><span class="NLM_article-title">Crystal Structure of the Kainate Receptor GluR5 Ligand-Binding Core in Complex with (S)-Glutamate</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">579</span><span class="NLM_x">, </span> <span class="NLM_fpage">1154</span><span class="NLM_x">–</span> <span class="NLM_lpage">1160</span><span class="refDoi"> DOI: 10.1016/j.febslet.2005.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.febslet.2005.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=15710405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlOnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2005&pages=1154-1160&author=P.+Naurauthor=B.+Vestergaardauthor=L.+K.+Skovauthor=J.+Egebjergauthor=M.+Gajhedeauthor=J.+S.+Kastrup&title=Crystal+Structure+of+the+Kainate+Receptor+GluR5+Ligand-Binding+Core+in+Complex+with+%28S%29-Glutamate&doi=10.1016%2Fj.febslet.2005.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the kainate receptor GluR5 ligand-binding core in complex with (S)-glutamate</span></div><div class="casAuthors">Naur, Peter; Vestergaard, Bente; Skov, Lars K.; Egebjerg, Jan; Gajhede, Michael; Kastrup, Jette Sandholm</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1154-1160</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The x-ray structure of the ligand-binding core of the kainate receptor GluR5 (GluR5-S1S2) in complex with (S)-glutamate was detd. to 1.95 Å resoln.  The overall GluR5-S1S2 structure comprises two domains and is similar to the related AMPA receptor GluR2-S1S2J. (S)-glutamate binds as in GluR2-S1S2J.  Distinct features are obsd. for Ser741, which stabilizes a highly coordinated network of water mols. and forms an interdomain bridge.  The GluR5 complex exhibits a high degree of domain closure (26°) relative to apo GluR2-S1S2J.  In addn., GluR5-S1S2 forms a novel dimer interface with a different arrangement of the two protomers compared to GluR2-S1S2J.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbtbs9jrVfsbVg90H21EOLACvtfcHk0ljdH6nvGoY5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlOnsLg%253D&md5=53693a84dad10af87aa0cff370788d8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2005.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2005.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DNaur%26aufirst%3DP.%26aulast%3DVestergaard%26aufirst%3DB.%26aulast%3DSkov%26aufirst%3DL.%2BK.%26aulast%3DEgebjerg%26aufirst%3DJ.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Kainate%2520Receptor%2520GluR5%2520Ligand-Binding%2520Core%2520in%2520Complex%2520with%2520%2528S%2529-Glutamate%26jtitle%3DFEBS%2520Lett.%26date%3D2005%26volume%3D579%26spage%3D1154%26epage%3D1160%26doi%3D10.1016%2Fj.febslet.2005.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Ursby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fredslund, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrador, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thunnissen, M. M. G. M.</span><span> </span><span class="NLM_article-title">The Macromolecular Crystallography Beamline I911–3 at the MAX IV Laboratory</span> <span class="citation_source-journal">J. Synchrotron Radiat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">653</span><span class="refDoi"> DOI: 10.1107/S0909049513011734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0909049513011734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=23765310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=648-653&author=T.+Ursbyauthor=J.+Ungeauthor=R.+Appioauthor=D.+T.+Loganauthor=F.+Fredslundauthor=C.+Svenssonauthor=K.+Larssonauthor=A.+Labradorauthor=M.+M.+G.+M.+Thunnissen&title=The+Macromolecular+Crystallography+Beamline+I911%E2%80%933+at+the+MAX+IV+Laboratory&doi=10.1107%2FS0909049513011734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The macromolecular crystallography beamline I911-3 at the MAX IV laboratory</span></div><div class="casAuthors">Ursby, Thomas; Unge, Johan; Appio, Roberto; Logan, Derek T.; Fredslund, Folmer; Svensson, Christer; Larsson, Krister; Labrador, Ana; Thunnissen, Marjolein M. G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">648-653</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. beamline I911-3, part of the Cassiopeia/I911 suite of beamlines, is based on a superconducting wiggler at the MAX II ring of the MAX IV Lab. in Lund, Sweden.  The beamline is energy-tunable within a range between 6 and 18 keV.  I911-3 opened for users in 2005.  In 2010-2011 the exptl. station was completely rebuilt and refurbished such that it has become a state-of-the-art exptl. station with better possibilities for rapid throughput, crystal screening and work with smaller samples.  This paper describes the complete I911-3 beamline and how it is embedded in the Cassiopeia suite of beamlines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHrhyNC6OdLLVg90H21EOLACvtfcHk0ljdH6nvGoY5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D&md5=4f204e3c3a4f2b8afe560f3e1b0f30c8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0909049513011734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049513011734%26sid%3Dliteratum%253Aachs%26aulast%3DUrsby%26aufirst%3DT.%26aulast%3DUnge%26aufirst%3DJ.%26aulast%3DAppio%26aufirst%3DR.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DFredslund%26aufirst%3DF.%26aulast%3DSvensson%26aufirst%3DC.%26aulast%3DLarsson%26aufirst%3DK.%26aulast%3DLabrador%26aufirst%3DA.%26aulast%3DThunnissen%26aufirst%3DM.%2BM.%2BG.%2BM.%26atitle%3DThe%2520Macromolecular%2520Crystallography%2520Beamline%2520I911%25E2%2580%25933%2520at%2520the%2520MAX%2520IV%2520Laboratory%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2013%26volume%3D20%26spage%3D648%26epage%3D653%26doi%3D10.1107%2FS0909049513011734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljdH6nvGoY5YA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Evans, P. R.</span><span> </span><span class="NLM_article-title">An Introduction to Data Reduction: Space-Group Determination, Scaling and Intensity Statistics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">282</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1107/S090744491003982X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS090744491003982X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+Introduction+to+Data+Reduction%3A+Space-Group+Determination%2C+Scaling+and+Intensity+Statistics&doi=10.1107%2FS090744491003982X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0ljcwroQsoFAMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS090744491003982X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744491003982X%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520Introduction%2520to%2520Data%2520Reduction%253A%2520Space-Group%2520Determination%252C%2520Scaling%2520and%2520Intensity%2520Statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2FS090744491003982X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP 4 Suite and Current Developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP+4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljcwroQsoFAMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP%25204%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0ljcwroQsoFAMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liehnPyJ35IRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (eLBOW): A Tool for Ligand Coordinate and Restraint Generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+A+Tool+for+Ligand+Coordinate+and+Restraint+Generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0liehnPyJ35IRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520A%2520Tool%2520for%2520Ligand%2520Coordinate%2520and%2520Restraint%2520Generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liehnPyJ35IRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Hayward, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Systematic Analysis of Domain Motions in Proteins from Conformational Change: New Results on Citrate Synthase and T4 Lysozyme</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">144</span><span class="NLM_x">–</span> <span class="NLM_lpage">154</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19980201)30:2<144::AID-PROT4>3.0.CO;2-N</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=9489922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADyaK1cXnvVaqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1998&pages=144-154&author=S.+Haywardauthor=H.+J.+C.+Berendsen&title=Systematic+Analysis+of+Domain+Motions+in+Proteins+from+Conformational+Change%3A+New+Results+on+Citrate+Synthase+and+T4+Lysozyme&doi=10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic analysis of domain motions in proteins from conformational change: new results on citrate synthase and T4 lysozyme</span></div><div class="casAuthors">Hayward, Steven; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">144-154</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Methods developed originally to analyze domain motions from simulation were adapted and extended for the anal. of x-ray conformers and for proteins with >2 domains.  The method can be applied as an automatic procedure to any case where >1 conformation is available.  The basis of the methodol. is that domains can be recognized from the difference in the parameters governing their quasi-rigid body motion, and in particular their rotation vectors.  A clustering algorithm is used to det. clusters of rotation vectors corresponding to main-chain segments that form possible dynamic domains.  Domains are accepted for further anal. on the basis of a ratio of interdomain to intradomain fluctuation, and Chasles' theorem is used to det. interdomain screw axes.  Finally residues involved in the interdomain motion are identified.  The methodol. was tested on citrate synthase and the M6I mutant of phage T4 lysozyme.  In both cases, new aspects of their conformational changes were revealed, as were individual residues intimately involved in their dynamics.  For citrate synthase, the β-sheet was identified to be part of the hinging mechanism.  In the case of phage T4 lysozyme, 1 of the 4 transitions in the pathway from the closed to the open conformation, furnished 4 dynamic domains rather than the expected 2.  This result indicated that the no. of dynamic domains a protein possesses may not be a const. of the motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWIAHnw5p1LVg90H21EOLACvtfcHk0liVbY6JpIy8Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnvVaqtg%253D%253D&md5=da16e74e98b9d35ca2c6a3fc82848ae2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819980201%252930%253A2%253C144%253A%253AAID-PROT4%253E3.0.CO%253B2-N%26sid%3Dliteratum%253Aachs%26aulast%3DHayward%26aufirst%3DS.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DSystematic%2520Analysis%2520of%2520Domain%2520Motions%2520in%2520Proteins%2520from%2520Conformational%2520Change%253A%2520New%2520Results%2520on%2520Citrate%2520Synthase%2520and%2520T4%2520Lysozyme%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1998%26volume%3D30%26spage%3D144%26epage%3D154%26doi%3D10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Silke Kayser, Jacob C. Hansen, Markus Staudt, Aleksandra Moroz, Younes Larsen, Piero Temperini, Feng Yi, Jed T. Syrenne, Niels Krogsgaard-Larsen, Stylianos Iliadis, Birgitte Nielsen, Kasper B. Hansen, Darryl S. Pickering, <span class="NLM_string-name hlFld-ContribAuthor">Lennart Bunch</span>. </span><span class="cited-content_cbyCitation_article-title">Stereoselective Synthesis of New (2S,3R)-3-Carboxyphenyl)pyrrolidine-2-carboxylic Acid Analogues Utilizing a C(sp3)–H Activation Strategy and Structure–Activity Relationship Studies at the Ionotropic Glutamate Receptors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Neuroscience</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 674-701. <a href="https://doi.org/10.1021/acschemneuro.0c00003" title="DOI URL">https://doi.org/10.1021/acschemneuro.0c00003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschemneuro.0c00003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschemneuro.0c00003%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Neuroscience%26atitle%3DStereoselective%252BSynthesis%252Bof%252BNew%252B%2525282S%25252C3R%252529-3-Carboxyphenyl%252529pyrrolidine-2-carboxylic%252BAcid%252BAnalogues%252BUtilizing%252Ba%252BC%252528sp3%252529%2525E2%252580%252593H%252BActivation%252BStrategy%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%252BStudies%252Bat%252Bthe%252BIonotropic%252BGlutamate%252BReceptors%26aulast%3DKayser%26aufirst%3DSilke%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02012020%26date%3D27012020%26date%3D17022020%26volume%3D11%26issue%3D5%26spage%3D674%26epage%3D701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ramakotaiah Mulamreddy, N. D. Prasad Atmuri, <span class="NLM_string-name hlFld-ContribAuthor">William D. Lubell</span>. </span><span class="cited-content_cbyCitation_article-title">4-Vinylproline. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2018,</strong> <em>83 </em>
                                    (21)
                                     , 13580-13586. <a href="https://doi.org/10.1021/acs.joc.8b02177" title="DOI URL">https://doi.org/10.1021/acs.joc.8b02177</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.8b02177&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.8b02177%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3D4-Vinylproline%26aulast%3DMulamreddy%26aufirst%3DRamakotaiah%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D22082018%26date%3D11102018%26date%3D24092018%26volume%3D83%26issue%3D21%26spage%3D13580%26epage%3D13586" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Yao  Chu</span>, <span class="hlFld-ContribAuthor ">Na  Li</span>, <span class="hlFld-ContribAuthor ">Dan  Liang</span>, <span class="hlFld-ContribAuthor ">Zheng-Hui  Li</span>, <span class="hlFld-ContribAuthor ">Yong-Sheng  Zheng</span>, <span class="hlFld-ContribAuthor ">Ji-Kai  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Accessing substituted pyrrolidines via formal [3+2] cycloaddition of 1,3,5-triazinanes and donor-acceptor cyclopropanes. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2018,</strong> <em>59 </em>
                                    (8)
                                     , 715-718. <a href="https://doi.org/10.1016/j.tetlet.2018.01.016" title="DOI URL">https://doi.org/10.1016/j.tetlet.2018.01.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2018.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2018.01.016%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DAccessing%252Bsubstituted%252Bpyrrolidines%252Bvia%252Bformal%252B%25255B3%25252B2%25255D%252Bcycloaddition%252Bof%252B1%25252C3%25252C5-triazinanes%252Band%252Bdonor-acceptor%252Bcyclopropanes%26aulast%3DChu%26aufirst%3DZhi-Yao%26date%3D2018%26volume%3D59%26issue%3D8%26spage%3D715%26epage%3D718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0001.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of Glu and iGluR group-defining agonists AMPA, KA, and NMDA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0002.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures of iGluR antagonists and subtype selective GluK1 antagonists.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0003.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Rational design of 4-substituted proline <b>1a</b> starting from 3-substituted proline <b>IV</b> (e.g., the 5-hydroxy analogue <b>IVa</b><a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a>). (B) Superimposition of binding mode of <b>IVa</b> in GluA2-LBD (pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yma">4yma</a>) (green), <b>V</b> (purple), and <b>1a</b> (gray) by alignment of the α-amino acid moieties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0006.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Stereo-Controlled Synthesis of <b>1a</b> Starting from Readily Available Protected (<i>S</i>)-Pyroglutaminol <b>2</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) LHMDS, THF, −78 °C to rt (64%); (b) BH<sub>3</sub>, THF, reflux (56%); (c) TBAF, THF, rt (52%); (d) TEMPO, NaClO<sub>2</sub>, NaOCl, MeCN, H<sub>2</sub>O, 0 °C (62%); (e) HCl(g)/AcOH in Et<sub>2</sub>O rt (99%).</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0007.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Stereo-Controlled Synthesis of <b>1b</b>–<b>d</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>cis</i>-4-Hydroxy-proline <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) appropriately substituted phenol, DIAD, PPh<sub>3</sub>, THF, rt (53–91%); (b) LiOH, THF, 0 °C to rt (53–91%, two steps; (c) <b>1b</b>,<b>d</b>:TFA in DCM, then 1 M HCl, <b>1c</b>:HCl/AcOH in Et<sub>2</sub>O (89–96%).</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0008.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Stereo-Controlled Synthesis of <b>1e</b>,<b>f</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>cis</i>-4-Hydroxy-proline <b>7</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MsCl, TEA, THF, 0 °C (98%); (b) appropriately substituted thiophenol, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (c) LiOH, THF, 0 °C to rt (78% for <b>12</b>); (d) TFA in DCM (94% and 40%, <b>1e</b> and <b>1f</b>, respectively).</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0009.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Stereo-Controlled Synthesis of <b>1g</b>–<b>i</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-Hydroxyproline <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) Appropriately substituted phenol, DIAD, PPh<sub>3</sub>, THF, rt; (b) LiOH, THF, 0 °C to rt (53–91%, two steps); (c) TFA in DCM, then 1 M HCl or HCl/AcOH in Et<sub>2</sub>O (85–93%).</p></p></figure><figure data-id="sch5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0010.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Stereo-Controlled Synthesis of <b>1j</b>,<b>k</b> Starting from Readily Available Protected <span class="smallcaps smallerCapital">l</span>-<i>trans</i>-4-Hydroxy-proline <b>14</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) MsCl, TEA, THF, 0 °C (90%); (b) Appropriately substituted thiophenol, K<sub>2</sub>CO<sub>3</sub>, DMF, 50 °C; (c) LiOH, THF, 0 °C to rt (67%, two steps for <b>19</b>); (d) TFA in DCM (74% for <b>1j</b> and 38% for <b>1k</b>, three steps).</p></p></figure><figure data-id="fig4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0004.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compound <b>1b</b> inhibits recombinant GluK1 but not GluK2 in HEK293T cells. (A) Example traces of GluK1 responses evoked by 1 mM Glu (250 ms) in outside-out patches at a holding potential of −100 mV, in the absence (left) or presence (right, gray) of 100 μM <b>1b</b>. (B) Example traces of GluK2 responses evoked by 1 mM Glu (250 ms) in outside-out patches at a holding potential of −100 mV, in the absence (left) or presence (right, light gray) of 100 μM <b>1b</b>. (C) Inhibition plot for <b>1b</b> for 1 mM Glu. Data are represented as a percentage of the control response in the absence of <b>1b</b> (mean ± SEM); 3–5 responses were obtained for each concentration of <b>1b</b> tested.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/medium/jm-2016-015162_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0005.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Structure of GluK1 LBD with the antagonist <b>1b</b> bound. (A) GluK1 LBD (molA, cyan; molB, pale cyan) in complex with <b>1b</b> (orange) crystallizes as a dimer. (B) 2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> electron density for <b>1b</b> after molecular replacement (black), at which point the ligand was not introduced into the model, as well as the final electron density (white). Both maps are contoured at 1 σ. (C) Binding site interactions with <b>1b</b>. Hydrogen bonds are shown as stippled black lines (up to 3.4 Å). Residues forming the pocket (within 4 Å) for the aromatic ring of <b>1b</b> are shown. (D) Comparison of the binding mode of <b>1b</b> and kainate (black; pdb code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e0x">4e0x</a>, gray) in GluK1 LBD. GluK1 structures were superimposed on residues of D1. Interactions of the ligands with Thr690 are shown as well as the D1–D2 interlobe hydrogen bond. The tube illustrates the different domain openings through movement of D2 residues. Hydrogen bonds are colored cyan for <b>1b</b> and black for kainate.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-1/acs.jmedchem.6b01516/20170106/images/large/jm-2016-015162_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01516&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i79">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71435" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71435" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 41 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Traynelis, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wollmuth, L. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McBain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Menniti, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vance, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, K. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dingledine, R.</span><span> </span><span class="NLM_article-title">Glutamate Receptor Ion Channels: Structure, Regulation, and Function</span> <span class="citation_source-journal">Pharmacol. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">62</span><span class="NLM_x">, </span> <span class="NLM_fpage">405</span><span class="NLM_x">–</span> <span class="NLM_lpage">496</span><span class="refDoi"> DOI: 10.1124/pr.109.002451</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1124%2Fpr.109.002451" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20716669" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVSltbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=405-496&author=S.+F.+Traynelisauthor=L.+P.+Wollmuthauthor=C.+J.+McBainauthor=F.+S.+Mennitiauthor=K.+M.+Vanceauthor=K.+K.+Ogdenauthor=K.+B.+Hansenauthor=H.+Yuanauthor=S.+J.+Myersauthor=R.+Dingledine&title=Glutamate+Receptor+Ion+Channels%3A+Structure%2C+Regulation%2C+and+Function&doi=10.1124%2Fpr.109.002451"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate receptor ion channels: structure, regulation, and function</span></div><div class="casAuthors">Traynelis, Stephen F.; Wollmuth, Lonnie P.; McBain, Chris J.; Menniti, Frank S.; Vance, Katie M.; Ogden, Kevin K.; Hansen, Kasper B.; Yuan, Hongjie; Myers, Scott J.; Dingledine, Ray</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-496</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  The mammalian ionotropic glutamate receptor family encodes 18 gene products that coassemble to form ligand-gated ion channels contg. an agonist recognition site, a transmembrane ion permeation pathway, and gating elements that couple agonist-induced conformational changes to the opening or closing of the permeation pore.  Glutamate receptors mediate fast excitatory synaptic transmission in the central nervous system and are localized on neuronal and non-neuronal cells.  These receptors regulate a broad spectrum of processes in the brain, spinal cord, retina, and peripheral nervous system.  Glutamate receptors are postulated to play important roles in numerous neurol. diseases and have attracted intense scrutiny.  The description of glutamate receptor structure, including its transmembrane elements, reveals a complex assembly of multiple semiautonomous extracellular domains linked to a pore-forming element with striking resemblance to an inverted potassium channel.  In this review we discuss International Union of Basic and Clin. Pharmacol. glutamate receptor nomenclature, structure, assembly, accessory subunits, interacting proteins, gene expression and translation, post-translational modifications, agonist and antagonist pharmacol., allosteric modulation, mechanisms of gating and permeation, roles in normal physiol. function, as well as the potential therapeutic use of pharmacol. agents acting at glutamate receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov0rj4nMr5ObVg90H21EOLACvtfcHk0lgMcOTaWVDeVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVSltbbJ&md5=5d0a7945b5ad99113adf9f6312bafe13</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1124%2Fpr.109.002451&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.109.002451%26sid%3Dliteratum%253Aachs%26aulast%3DTraynelis%26aufirst%3DS.%2BF.%26aulast%3DWollmuth%26aufirst%3DL.%2BP.%26aulast%3DMcBain%26aufirst%3DC.%2BJ.%26aulast%3DMenniti%26aufirst%3DF.%2BS.%26aulast%3DVance%26aufirst%3DK.%2BM.%26aulast%3DOgden%26aufirst%3DK.%2BK.%26aulast%3DHansen%26aufirst%3DK.%2BB.%26aulast%3DYuan%26aufirst%3DH.%26aulast%3DMyers%26aufirst%3DS.%2BJ.%26aulast%3DDingledine%26aufirst%3DR.%26atitle%3DGlutamate%2520Receptor%2520Ion%2520Channels%253A%2520Structure%252C%2520Regulation%252C%2520and%2520Function%26jtitle%3DPharmacol.%2520Rev.%26date%3D2010%26volume%3D62%26spage%3D405%26epage%3D496%26doi%3D10.1124%2Fpr.109.002451" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Ferraguti, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shigemoto, R.</span><span> </span><span class="NLM_article-title">Metabotropic Glutamate Receptors</span> <span class="citation_source-journal">Cell Tissue Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">326</span><span class="NLM_x">, </span> <span class="NLM_fpage">483</span><span class="NLM_x">–</span> <span class="NLM_lpage">504</span><span class="refDoi"> DOI: 10.1007/s00441-006-0266-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1007%2Fs00441-006-0266-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=16847639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvFalt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=326&publication_year=2006&pages=483-504&author=F.+Ferragutiauthor=R.+Shigemoto&title=Metabotropic+Glutamate+Receptors&doi=10.1007%2Fs00441-006-0266-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Metabotropic glutamate receptors</span></div><div class="casAuthors">Ferraguti, Francesco; Shigemoto, Ryuichi</div><div class="citationInfo"><span class="NLM_cas:title">Cell & Tissue Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">326</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">483-504</span>CODEN:
                <span class="NLM_cas:coden">CTSRCS</span>;
        ISSN:<span class="NLM_cas:issn">0302-766X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Metabotropic glutamate receptors (mGlus) are a family of G-protein-coupled receptors activated by the neurotransmitter glutamate.  Mol. cloning has revealed eight different subtypes (mGlu1-8) with distinct mol. and pharmacol. properties.  Multiplicity in this receptor family is further generated through alternative splicing. mGlus activate a multitude of signalling pathways important for modulating neuronal excitability, synaptic plasticity and feedback regulation of neurotransmitter release.  In this review, we summarize anatomical findings (from our work and that of other labs.) describing their distribution in the central nervous system.  Recent evidence regarding the localization of these receptors in peripheral tissues will also be examd.  The distinct regional, cellular and subcellular distribution of mGlus in the brain will be discussed in view of their relationship to neurotransmitter release sites and of possible functional implications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPxClJvmJpzLVg90H21EOLACvtfcHk0lgMcOTaWVDeVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvFalt70%253D&md5=7c4ded2874fd22764db90e87bfb0c0a6</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1007%2Fs00441-006-0266-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00441-006-0266-5%26sid%3Dliteratum%253Aachs%26aulast%3DFerraguti%26aufirst%3DF.%26aulast%3DShigemoto%26aufirst%3DR.%26atitle%3DMetabotropic%2520Glutamate%2520Receptors%26jtitle%3DCell%2520Tissue%2520Res.%26date%3D2006%26volume%3D326%26spage%3D483%26epage%3D504%26doi%3D10.1007%2Fs00441-006-0266-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Danbolt, N. C.</span><span> </span><span class="NLM_article-title">Glutamate Uptake</span> <span class="citation_source-journal">Prog. Neurobiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">105</span><span class="refDoi"> DOI: 10.1016/S0301-0082(00)00067-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2FS0301-0082%2800%2900067-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=11369436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjsFWns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2001&pages=1-105&author=N.+C.+Danbolt&title=Glutamate+Uptake&doi=10.1016%2FS0301-0082%2800%2900067-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate uptake</span></div><div class="casAuthors">Danbolt, N. C.</div><div class="citationInfo"><span class="NLM_cas:title">Progress in Neurobiology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-105</span>CODEN:
                <span class="NLM_cas:coden">PGNBA5</span>;
        ISSN:<span class="NLM_cas:issn">0301-0082</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">A review, with >1200 refs.  Brain tissue has a remarkable ability to accumulate glutamate.  This ability is due to glutamate transporter proteins present in the plasma membranes of both glial cells and neurons.  The transporter proteins represent the only (significant) mechanism for removal of glutamate from the extracellular fluid and their importance for the long-term maintenance of low and non-toxic concns. of glutamate is now well documented.  In addn. to this simple, but essential glutamate removal role, the glutamate transporters appear to have more sophisticated functions in the modulation of neurotransmission.  They may modify the time course of synaptic events, the extent and pattern of activation and desensitization of receptors outside the synaptic cleft and at neighboring synapses (intersynaptic cross-talk).  Further, the glutamate transporters provide glutamate for synthesis of e.g.  GABA, glutathione, and protein, and for energy prodn.  They also play roles in peripheral organs and tissues (e.g. bone, heart, intestine, kidneys, pancreas, and placenta).  Glutamate uptake appears to be modulated on virtually all possible levels, i.e. DNA transcription, mRNA splicing and degrdn., protein synthesis and targeting, and actual amino acid transport activity and assocd. ion channel activities.  A variety of sol. compds. (e.g. glutamate, cytokines, and growth factors) influence glutamate transporter expression and activities.  Neither the normal functioning of glutamatergic synapses nor the pathogenesis of major neurol. diseases (e.g. cerebral ischemia, hypoglycemia, amyotrophic lateral sclerosis, Alzheimer's disease, traumatic brain injury, epilepsy, and schizophrenia) as well as non-neurol. diseases (e.g. osteoporosis) can be properly understood unless more is learned about these transporter proteins.  Like glutamate itself, glutamate transporters are somehow involved in almost all aspects of normal and abnormal brain activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrbDph2PBuBbVg90H21EOLACvtfcHk0lgMcOTaWVDeVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXjsFWns74%253D&md5=a7118c1df858a0994af6207f852f82b6</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2FS0301-0082%2800%2900067-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0301-0082%252800%252900067-8%26sid%3Dliteratum%253Aachs%26aulast%3DDanbolt%26aufirst%3DN.%2BC.%26atitle%3DGlutamate%2520Uptake%26jtitle%3DProg.%2520Neurobiol.%26date%3D2001%26volume%3D65%26spage%3D1%26epage%3D105%26doi%3D10.1016%2FS0301-0082%2800%2900067-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Jensen, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fahlke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bjørn-Yoshimoto, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Excitatory Amino Acid Transporters: Recent Insights into Molecular Mechanisms, Novel Modes of Modulation and New Therapeutic Possibilities</span> <span class="citation_source-journal">Curr. Opin. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">116</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span><span class="refDoi"> DOI: 10.1016/j.coph.2014.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.coph.2014.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=25466154" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=116-123&author=A.+A.+Jensenauthor=C.+Fahlkeauthor=W.+E.+Bj%C3%B8rn-Yoshimotoauthor=L.+Bunch&title=Excitatory+Amino+Acid+Transporters%3A+Recent+Insights+into+Molecular+Mechanisms%2C+Novel+Modes+of+Modulation+and+New+Therapeutic+Possibilities&doi=10.1016%2Fj.coph.2014.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Excitatory amino acid transporters: recent insights into molecular mechanisms, novel modes of modulation and new therapeutic possibilities</span></div><div class="casAuthors">Jensen, Anders A.; Fahlke, Christoph; Bjoern-Yoshimoto, Walden E.; Bunch, Lennart</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">116-123</span>CODEN:
                <span class="NLM_cas:coden">COPUBK</span>;
        ISSN:<span class="NLM_cas:issn">1471-4892</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The five excitatory amino acid transporters (EAAT1-5) mediating the synaptic uptake of the major excitatory neurotransmitter glutamate are differently expressed throughout the CNS and at the synaptic level.  Although EAATs are crucial for normal excitatory neurotransmission, explorations into the physiol. functions mediated by the different transporter subtypes and their resp. therapeutic potential have so far been sparse, in no small part due to the limited selection of pharmacol. tools available.  In the present update, we outline important new insights into the mol. compns. of EAATs and their intricate transport process, the novel approaches to pharmacol. modulation of the transporters that have emerged, and interesting new perspectives in EAAT as drug targets proposed in recent years.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHCPIMplW6KLVg90H21EOLACvtfcHk0lguHLdVjAUiRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVCktbvF&md5=4619d6aebdc64cfd77a44bae4379eb53</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.coph.2014.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coph.2014.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DA.%2BA.%26aulast%3DFahlke%26aufirst%3DC.%26aulast%3DBj%25C3%25B8rn-Yoshimoto%26aufirst%3DW.%2BE.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DExcitatory%2520Amino%2520Acid%2520Transporters%253A%2520Recent%2520Insights%2520into%2520Molecular%2520Mechanisms%252C%2520Novel%2520Modes%2520of%2520Modulation%2520and%2520New%2520Therapeutic%2520Possibilities%26jtitle%3DCurr.%2520Opin.%2520Pharmacol.%26date%3D2015%26volume%3D20%26spage%3D116%26epage%3D123%26doi%3D10.1016%2Fj.coph.2014.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Bjørn-Yoshimoto, W. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underhill, S. M.</span><span> </span><span class="NLM_article-title">The Importance of the Excitatory Amino Acid Transporter 3 (EAAT3)</span> <span class="citation_source-journal">Neurochem. Int.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span><span class="refDoi"> DOI: 10.1016/j.neuint.2016.05.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.neuint.2016.05.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=27233497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252Fkt1KisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2016&pages=4-18&author=W.+E.+Bj%C3%B8rn-Yoshimotoauthor=S.+M.+Underhill&title=The+Importance+of+the+Excitatory+Amino+Acid+Transporter+3+%28EAAT3%29&doi=10.1016%2Fj.neuint.2016.05.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">The importance of the excitatory amino acid transporter 3 (EAAT3)</span></div><div class="casAuthors">Bjorn-Yoshimoto Walden E; Underhill Suzanne M</div><div class="citationInfo"><span class="NLM_cas:title">Neurochemistry international</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4-18</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The neuronal excitatory amino acid transporter 3 (EAAT3) is fairly ubiquitously expressed in the brain, though it does not necessarily maintain the same function everywhere.  It is important in maintaining low local concentrations of glutamate, where its predominant post-synaptic localization can buffer nearby glutamate receptors and modulate excitatory neurotransmission and synaptic plasticity.  It is also the main neuronal cysteine uptake system acting as the rate-limiting factor for the synthesis of glutathione, a potent antioxidant, in EAAT3 expressing neurons, while on GABAergic neurons, it is important in supplying glutamate as a precursor for GABA synthesis.  Several diseases implicate EAAT3, and modulation of this transporter could prove a useful therapeutic approach.  Regulation of EAAT3 could be targeted at several points for functional modulation, including the level of transcription, trafficking and direct pharmacological modulation, and indeed, compounds and experimental treatments have been identified that regulate EAAT3 function at different stages, which together with observations of EAAT3 regulation in patients is giving us insight into the endogenous function of this transporter, as well as the consequences of altered function.  This review summarizes work done on elucidating the role and regulation of EAAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTRow0Q3ClcxdUSIfHIn4FfW6udTcc2eazifpoGHYOXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252Fkt1KisA%253D%253D&md5=27ff5a73f8de4c0550ebe7ee83660fbd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.neuint.2016.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuint.2016.05.007%26sid%3Dliteratum%253Aachs%26aulast%3DBj%25C3%25B8rn-Yoshimoto%26aufirst%3DW.%2BE.%26aulast%3DUnderhill%26aufirst%3DS.%2BM.%26atitle%3DThe%2520Importance%2520of%2520the%2520Excitatory%2520Amino%2520Acid%2520Transporter%25203%2520%2528EAAT3%2529%26jtitle%3DNeurochem.%2520Int.%26date%3D2016%26volume%3D98%26spage%3D4%26epage%3D18%26doi%3D10.1016%2Fj.neuint.2016.05.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Choi, D. W.</span><span> </span><span class="NLM_article-title">Glutamate Neurotoxicity and Diseases of the Nervous System</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">–</span> <span class="NLM_lpage">634</span><span class="refDoi"> DOI: 10.1016/0896-6273(88)90162-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2F0896-6273%2888%2990162-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=2908446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A280%3ADyaK3c7ms1GnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=1988&pages=623-634&author=D.+W.+Choi&title=Glutamate+Neurotoxicity+and+Diseases+of+the+Nervous+System&doi=10.1016%2F0896-6273%2888%2990162-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate neurotoxicity and diseases of the nervous system</span></div><div class="casAuthors">Choi D W</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">623-34</span>
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSgYvXebUy8NTW1zwg2yUK5fW6udTcc2eazifpoGHYOXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3c7ms1GnsQ%253D%253D&md5=1e33ba0bd4c1d830f6b7c7d76e29dbd3</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0896-6273%2888%2990162-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0896-6273%252888%252990162-6%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DD.%2BW.%26atitle%3DGlutamate%2520Neurotoxicity%2520and%2520Diseases%2520of%2520the%2520Nervous%2520System%26jtitle%3DNeuron%26date%3D1988%26volume%3D1%26spage%3D623%26epage%3D634%26doi%3D10.1016%2F0896-6273%2888%2990162-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Lau, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tymianski, M.</span><span> </span><span class="NLM_article-title">Glutamate Receptors, Neurotoxicity and Neurodegeneration</span> <span class="citation_source-journal">Pfluegers Arch.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">460</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1007/s00424-010-0809-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1007%2Fs00424-010-0809-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20229265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntlGns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=460&publication_year=2010&pages=525-542&author=A.+Lauauthor=M.+Tymianski&title=Glutamate+Receptors%2C+Neurotoxicity+and+Neurodegeneration&doi=10.1007%2Fs00424-010-0809-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Glutamate receptors, neurotoxicity and neurodegeneration</span></div><div class="casAuthors">Lau, Anthony; Tymianski, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Pfluegers Archiv</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">460</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">525-542</span>CODEN:
                <span class="NLM_cas:coden">PFLABK</span>;
        ISSN:<span class="NLM_cas:issn">0031-6768</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Glutamate excitotoxicity is a hypothesis that states excessive glutamate causes neuronal dysfunction and degeneration.  As glutamate is a major excitatory neurotransmitter in the central nervous system (CNS), the implications of glutamate excitotoxicity are many and far-reaching.  Acute CNS insults such as ischemia and traumatic brain injury have traditionally been the focus of excitotoxicity research.  However, glutamate excitotoxicity has also been linked to chronic neurodegenerative disorders such as amyotrophic lateral sclerosis, multiple sclerosis, Parkinson's disease and others.  Despite the continued research into the mechanisms of excitotoxicity, there are currently no pharmacol. interventions capable of providing significant neuroprotection in the clin. setting of brain ischemia or injury.  This review addresses the current state of excitotoxic research, focusing on the structure and physiol. of glutamate receptors; mol. mechanisms underlying excitotoxic cell death pathways and their interactions with each other; the evidence for glutamate excitotoxicity in acute neurol. diseases; lab. and clin. attempts at modulating excitotoxicity; and emerging targets for excitotoxicity research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKnG8BQuqVZLVg90H21EOLACvtfcHk0lh8XKx5DkVsVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntlGns7o%253D&md5=9429b771c07ca3f65c509c8bc6566cac</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1007%2Fs00424-010-0809-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00424-010-0809-1%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DA.%26aulast%3DTymianski%26aufirst%3DM.%26atitle%3DGlutamate%2520Receptors%252C%2520Neurotoxicity%2520and%2520Neurodegeneration%26jtitle%3DPfluegers%2520Arch.%26date%3D2010%26volume%3D460%26spage%3D525%26epage%3D542%26doi%3D10.1007%2Fs00424-010-0809-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Machado-Vieira, R.; Henter, I. D.; Zarate, C. A.,  Jr.</span><span> </span><span class="NLM_article-title">New Targets for Rapid Antidepressant Action</span>.  <span class="citation_source-journal">Prog. Neurobiol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span>DOI: <span class="refDoi"> DOI: 10.1016/j.pneurobio.2015.12.001</span> ..</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.pneurobio.2015.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=26724279" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=R.+Machado-Vieira&author=I.+D.+Henter&author=C.+A.+Zarate&title=New+Targets+for+Rapid+Antidepressant+Action&doi=10.1016%2Fj.pneurobio.2015.12.001"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.pneurobio.2015.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pneurobio.2015.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DMachado-Vieira%26aufirst%3DR.%26atitle%3DNew%2520Targets%2520for%2520Rapid%2520Antidepressant%2520Action%26jtitle%3DProg.%2520Neurobiol.%26date%3D2015%26doi%3D10.1016%2Fj.pneurobio.2015.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Riaza Bermudo-Soriano, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perez-Rodriguez, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vaquero-Lorenzo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baca-Garcia, E.</span><span> </span><span class="NLM_article-title">New Perspectives in Glutamate and Anxiety</span> <span class="citation_source-journal">Pharmacol., Biochem. Behav.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">752</span><span class="NLM_x">–</span> <span class="NLM_lpage">774</span><span class="refDoi"> DOI: 10.1016/j.pbb.2011.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.pbb.2011.04.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2012&pages=752-774&author=C.+Riaza+Bermudo-Sorianoauthor=M.+M.+Perez-Rodriguezauthor=C.+Vaquero-Lorenzoauthor=E.+Baca-Garcia&title=New+Perspectives+in+Glutamate+and+Anxiety&doi=10.1016%2Fj.pbb.2011.04.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.pbb.2011.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pbb.2011.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DRiaza%2BBermudo-Soriano%26aufirst%3DC.%26aulast%3DPerez-Rodriguez%26aufirst%3DM.%2BM.%26aulast%3DVaquero-Lorenzo%26aufirst%3DC.%26aulast%3DBaca-Garcia%26aufirst%3DE.%26atitle%3DNew%2520Perspectives%2520in%2520Glutamate%2520and%2520Anxiety%26jtitle%3DPharmacol.%252C%2520Biochem.%2520Behav.%26date%3D2012%26volume%3D100%26spage%3D752%26epage%3D774%26doi%3D10.1016%2Fj.pbb.2011.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Wierońska, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zorn, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doller, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilc, A.</span><span> </span><span class="NLM_article-title">Metabotropic Glutamate Receptors as Targets for New Antipsychotic Drugs: Historical Perspective and Critical Comparative Assessment</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">157</span><span class="NLM_x">, </span> <span class="NLM_fpage">10</span><span class="NLM_x">–</span> <span class="NLM_lpage">27</span><span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.pharmthera.2015.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=26549541" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVSks7vM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2016&pages=10-27&author=J.+M.+Wiero%C5%84skaauthor=S.+H.+Zornauthor=D.+Dollerauthor=A.+Pilc&title=Metabotropic+Glutamate+Receptors+as+Targets+for+New+Antipsychotic+Drugs%3A+Historical+Perspective+and+Critical+Comparative+Assessment&doi=10.1016%2Fj.pharmthera.2015.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Metabotropic glutamate receptors as targets for new antipsychotic drugs: Historical perspective and critical comparative assessment</span></div><div class="casAuthors">Wieronska, Joanna M.; Zorn, Stevin H.; Doller, Dario; Pilc, Andrzej</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10-27</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">In this review, we aim to present, discuss and clarify our current understanding regarding the prediction of possible antipsychotic effects of metabotropic glutamate (mGlu) receptor ligands.  The no. of preclin. trials clearly indicates, that this group of compds. constitutes an excellent alternative to presently used antipsychotic therapy, being effective not only to pos., but also neg. and cognitive symptoms of schizophrenia.  Although the results of clin. trials that were performed for the group of mGlu2/3 agonists were not so enthusiastic as in animal studies, they still showed that mGlu ligands do not induced variety of side effects typical for presently used antipsychotics, and were generally well tolerated.  The lack of satisfactory effectiveness towards schizophrenia symptoms of mGlu2/3 activators in humans could be a result of variety of uncontrolled factors and unidentified biomarkers different for each schizophrenia patient, that should be taken into consideration in the future set of clin. trials.  The subject is still open for further research, and the novel classes of mGlu5 or mGlu2/3 agonists/PAMs were recently introduced, including the large group of compds. from the third group of mGlu receptors, esp. of mGlu4 subtype.  Finally, more precise treatment based on simultaneous administration of minimal doses of the ligands for two or more receptors, seems to be promising in the context of symptoms-specific schizophrenia treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGor64l-ntwmK7Vg90H21EOLACvtfcHk0lh8XKx5DkVsVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVSks7vM&md5=23b8ec790518a3ad5c92cb6c3895e618</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DWiero%25C5%2584ska%26aufirst%3DJ.%2BM.%26aulast%3DZorn%26aufirst%3DS.%2BH.%26aulast%3DDoller%26aufirst%3DD.%26aulast%3DPilc%26aufirst%3DA.%26atitle%3DMetabotropic%2520Glutamate%2520Receptors%2520as%2520Targets%2520for%2520New%2520Antipsychotic%2520Drugs%253A%2520Historical%2520Perspective%2520and%2520Critical%2520Comparative%2520Assessment%26jtitle%3DPharmacol.%2520Ther.%26date%3D2016%26volume%3D157%26spage%3D10%26epage%3D27%26doi%3D10.1016%2Fj.pharmthera.2015.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Gonzalez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jurado-Coronel, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ávila, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabogal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capani, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barreto, G. E.</span><span> </span><span class="NLM_article-title">NMDARs in Neurological Diseases: A Potential Therapeutic Target</span> <span class="citation_source-journal">Int. J. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span><span class="refDoi"> DOI: 10.3109/00207454.2014.940941</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.3109%2F00207454.2014.940941" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=25051426" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVaju7fJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=315-327&author=J.+Gonzalezauthor=J.+C.+Jurado-Coronelauthor=M.+F.+%C3%81vilaauthor=A.+Sabogalauthor=F.+Capaniauthor=G.+E.+Barreto&title=NMDARs+in+Neurological+Diseases%3A+A+Potential+Therapeutic+Target&doi=10.3109%2F00207454.2014.940941"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">NMDARs in neurological diseases: a potential therapeutic target</span></div><div class="casAuthors">Gonzalez, Janneth; Jurado-Coronel, Juan Camilo; Avila, Marcos Fidel; Sabogal, Angelica; Capani, Francisco; Barreto, George E.</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">125</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">315-327</span>CODEN:
                <span class="NLM_cas:coden">IJNUB7</span>;
        ISSN:<span class="NLM_cas:issn">0020-7454</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">N-methyl-D-aspartate ionotropic glutamate receptor (NMDARs) is a ligand-gated ion channel that plays a crit. role in excitatory neurotransmission, brain development, synaptic plasticity assocd. with memory formation, central sensitization during persistent pain, excitotoxicity and neurodegenerative diseases in the central nervous system (CNS).  Within iGluRs, NMDA receptors have been the most actively investigated for their role in neurol. diseases, esp. neurodegenerative pathologies such as Alzheimer's and Parkinson's diseases.  It has been demonstrated that excessive activation of NMDA receptors (NMDARs) plays a key role in mediating some aspects of synaptic dysfunction in several CNS disorders, so extensive research has been directed on the discovery of compds. that are able to reduce NMDARs activity.  This review discusses the role of NMDARs on neurol. pathologies and the possible therapeutic use of agents that target this receptor.  Addnl., we delve into the role of NMDARs in Alzheimer's and Parkinson's diseases and the receptor antagonists that have been tested on in vivo models of these pathologies.  Finally, we put into consideration the importance of antioxidants to counteract oxidative capacity of the signaling cascade in which NMDARs are involved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpab8NK1TOnqrVg90H21EOLACvtfcHk0ljSKGxzB5h-QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVaju7fJ&md5=50f77e6ec1187bba03ed53eb61e721a4</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3109%2F00207454.2014.940941&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F00207454.2014.940941%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DJ.%26aulast%3DJurado-Coronel%26aufirst%3DJ.%2BC.%26aulast%3D%25C3%2581vila%26aufirst%3DM.%2BF.%26aulast%3DSabogal%26aufirst%3DA.%26aulast%3DCapani%26aufirst%3DF.%26aulast%3DBarreto%26aufirst%3DG.%2BE.%26atitle%3DNMDARs%2520in%2520Neurological%2520Diseases%253A%2520A%2520Potential%2520Therapeutic%2520Target%26jtitle%3DInt.%2520J.%2520Neurosci.%26date%3D2015%26volume%3D125%26spage%3D315%26epage%3D327%26doi%3D10.3109%2F00207454.2014.940941" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Finlay, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duty, S.</span><span> </span><span class="NLM_article-title">Therapeutic Potential of Targeting Glutamate Receptors in Parkinson’s Disease</span> <span class="citation_source-journal">J. Neural Transm.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">121</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">880</span><span class="refDoi"> DOI: 10.1007/s00702-014-1176-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1007%2Fs00702-014-1176-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2014&pages=861-880&author=C.+Finlayauthor=S.+Duty&title=Therapeutic+Potential+of+Targeting+Glutamate+Receptors+in+Parkinson%E2%80%99s+Disease&doi=10.1007%2Fs00702-014-1176-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1007%2Fs00702-014-1176-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00702-014-1176-4%26sid%3Dliteratum%253Aachs%26aulast%3DFinlay%26aufirst%3DC.%26aulast%3DDuty%26aufirst%3DS.%26atitle%3DTherapeutic%2520Potential%2520of%2520Targeting%2520Glutamate%2520Receptors%2520in%2520Parkinson%25E2%2580%2599s%2520Disease%26jtitle%3DJ.%2520Neural%2520Transm.%26date%3D2014%26volume%3D121%26spage%3D861%26epage%3D880%26doi%3D10.1007%2Fs00702-014-1176-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Lee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goodman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fourie, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leitch, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Montgomery, J. M.</span><span> </span><span class="NLM_article-title">AMPA Receptors as Therapeutic Targets for Neurological Disorders</span> <span class="citation_source-journal">Adv. Protein Chem. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">203</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span><span class="refDoi"> DOI: 10.1016/bs.apcsb.2015.10.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fbs.apcsb.2015.10.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=26920691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVOisb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=203-261&author=K.+Leeauthor=L.+Goodmanauthor=C.+Fourieauthor=S.+Schenkauthor=B.+Leitchauthor=J.+M.+Montgomery&title=AMPA+Receptors+as+Therapeutic+Targets+for+Neurological+Disorders&doi=10.1016%2Fbs.apcsb.2015.10.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">AMPA receptors as therapeutic targets for neurological disorders</span></div><div class="casAuthors">Lee, Kevin; Goodman, Lucy; Fourie, Chantelle; Schenk, Susan; Leitch, Beulah; Montgomery, Johanna M.</div><div class="citationInfo"><span class="NLM_cas:title">Advances in Protein Chemistry and Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">Ion Channels as Therapeutic Targets, Part A</span>),
    <span class="NLM_cas:pages">203-261</span>CODEN:
                <span class="NLM_cas:coden">APCSG7</span>;
        ISSN:<span class="NLM_cas:issn">1876-1623</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Almost every neurol. disease directly or indirectly affects synapse function in the brain.  However, these diseases alter synapses through different mechanisms, ultimately resulting in altered synaptic transmission and/or plasticity.  Glutamate is the major neurotransmitter that mediates excitatory synaptic transmission in the brain through activation of alpha-amino-3-hydroxy-5-methyl-4-isoxazole-propionate (AMPA) receptors.  These receptors have therefore been identified as a target for the development of therapeutic treatments for neurol. disorders including epilepsy, neurodegenerative diseases, autism, and drug addiction.  The fact that AMPA receptors play a dominant role throughout the brain raises the significant challenge of selectively targeting only those regions affected by disease, and clin. trials have raised doubt regarding the feasibility of specifically targeting AMPA receptors for new therapeutic options.  Benzamide compds. that act as pos. allosteric AMPA receptor modulators, known as AMPAkines, can act on specific brain regions and were initially proposed to revolutionize the treatment of cognitive deficits assocd. with neurol. disorders.  Their therapeutic potential has since declined due to inconsistent results in clin. trials.  However, recent advances in basic biomedical research are significantly increasing our knowledge of AMPA receptor structure, binding sites, and interactions with auxiliary proteins.  In particular, the large complex of postsynaptic proteins that interact with AMPA receptor subunits have been shown to control AMPA receptor insertion, location, pharmacol., synaptic transmission, and plasticity.  These proteins are now being considered as alternative therapeutic target sites for modulating AMPA receptors in neurol. disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuZyN45J6nyLVg90H21EOLACvtfcHk0ljSKGxzB5h-QQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVOisb7E&md5=cac1a247d6e5e58ea9181fd14e3fcf04</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fbs.apcsb.2015.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.apcsb.2015.10.004%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DK.%26aulast%3DGoodman%26aufirst%3DL.%26aulast%3DFourie%26aufirst%3DC.%26aulast%3DSchenk%26aufirst%3DS.%26aulast%3DLeitch%26aufirst%3DB.%26aulast%3DMontgomery%26aufirst%3DJ.%2BM.%26atitle%3DAMPA%2520Receptors%2520as%2520Therapeutic%2520Targets%2520for%2520Neurological%2520Disorders%26jtitle%3DAdv.%2520Protein%2520Chem.%2520Struct.%2520Biol.%26date%3D2016%26volume%3D103%26spage%3D203%26epage%3D261%26doi%3D10.1016%2Fbs.apcsb.2015.10.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Armstrong, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gouaux, E.</span><span> </span><span class="NLM_article-title">Mechanisms for Activation and Antagonism of an AMPA-Sensitive Glutamate Receptor: Crystal Structures of the GluR2 Ligand Binding Core</span> <span class="citation_source-journal">Neuron</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">165</span><span class="NLM_x">–</span> <span class="NLM_lpage">181</span><span class="refDoi"> DOI: 10.1016/S0896-6273(00)00094-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2FS0896-6273%2800%2900094-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=11086992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvVGltLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=165-181&author=N.+Armstrongauthor=E.+Gouaux&title=Mechanisms+for+Activation+and+Antagonism+of+an+AMPA-Sensitive+Glutamate+Receptor%3A+Crystal+Structures+of+the+GluR2+Ligand+Binding+Core&doi=10.1016%2FS0896-6273%2800%2900094-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms for activation and antagonism of an AMPA-sensitive glutamate receptor: crystal structures of the GluR2 ligand binding core</span></div><div class="casAuthors">Armstrong, Neali; Gouaux, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">165-181</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Crystal structures of the GluR2 ligand binding core (S1S2) have been detd. in the apo state and in the presence of the antagonist DNQX, the partial agonist kainate, and the full agonists AMPA and glutamate.  The domains of the S1S2 ligand binding core are expanded in the apo state and contract upon ligand binding with the extent of domain sepn. decreasing in the order of apo > DNQX > kainate > glutamate ≃ AMPA.  These results suggest that agonist-induced domain closure gates the transmembrane channel and the extent of receptor activation depends upon the degree of domain closure.  AMPA and glutamate also promote a 180° flip of a trans peptide bond in the ligand binding site.  The crystal packing of the ligand binding cores suggests modes for subunit-subunit contact in the intact receptor and mechanisms by which allosteric effectors modulate receptor activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNnbcNBzyXN7Vg90H21EOLACvtfcHk0ljDTYaCKyyUiQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnvVGltLw%253D&md5=6be3ca25ce094c142ba3977a404afb3d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2FS0896-6273%2800%2900094-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0896-6273%252800%252900094-5%26sid%3Dliteratum%253Aachs%26aulast%3DArmstrong%26aufirst%3DN.%26aulast%3DGouaux%26aufirst%3DE.%26atitle%3DMechanisms%2520for%2520Activation%2520and%2520Antagonism%2520of%2520an%2520AMPA-Sensitive%2520Glutamate%2520Receptor%253A%2520Crystal%2520Structures%2520of%2520the%2520GluR2%2520Ligand%2520Binding%2520Core%26jtitle%3DNeuron%26date%3D2000%26volume%3D28%26spage%3D165%26epage%3D181%26doi%3D10.1016%2FS0896-6273%2800%2900094-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Honoré, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drejer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodge, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, F. E.</span><span> </span><span class="NLM_article-title">Quinoxalinediones: Potent Competitive Non-NMDA Glutamate Receptor Antagonists</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">241</span><span class="NLM_x">, </span> <span class="NLM_fpage">701</span><span class="NLM_x">–</span> <span class="NLM_lpage">703</span><span class="refDoi"> DOI: 10.1126/science.2899909</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1126%2Fscience.2899909" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=241&publication_year=1988&pages=701-703&author=T.+Honor%C3%A9author=S.+N.+Daviesauthor=J.+Drejerauthor=E.+J.+Fletcherauthor=P.+Jacobsenauthor=D.+Lodgeauthor=F.+E.+Nielsen&title=Quinoxalinediones%3A+Potent+Competitive+Non-NMDA+Glutamate+Receptor+Antagonists&doi=10.1126%2Fscience.2899909"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1126%2Fscience.2899909&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.2899909%26sid%3Dliteratum%253Aachs%26aulast%3DHonor%25C3%25A9%26aufirst%3DT.%26aulast%3DDavies%26aufirst%3DS.%2BN.%26aulast%3DDrejer%26aufirst%3DJ.%26aulast%3DFletcher%26aufirst%3DE.%2BJ.%26aulast%3DJacobsen%26aufirst%3DP.%26aulast%3DLodge%26aufirst%3DD.%26aulast%3DNielsen%26aufirst%3DF.%2BE.%26atitle%3DQuinoxalinediones%253A%2520Potent%2520Competitive%2520Non-NMDA%2520Glutamate%2520Receptor%2520Antagonists%26jtitle%3DScience%26date%3D1988%26volume%3D241%26spage%3D701%26epage%3D703%26doi%3D10.1126%2Fscience.2899909" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Ornstein, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleisch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borromeo, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugar, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leander, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodge, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepp, D. D.</span><span> </span><span class="NLM_article-title">Structure-Activity Studies of 6-Substituted Decahydroisoquinoline-3- Carboxylic Acid AMPA Receptor Antagonists. 2. Effects of Distal Acid Bioisosteric Substitution, Absolute Stereochemical Preferences, and in Vivo Activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2232</span><span class="NLM_x">–</span> <span class="NLM_lpage">2244</span><span class="refDoi"> DOI: 10.1021/jm950913h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm950913h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2232-2244&author=P.+L.+Ornsteinauthor=M.+B.+Arnoldauthor=N.+K.+Allenauthor=T.+Bleischauthor=P.+S.+Borromeoauthor=C.+W.+Lugarauthor=J.+D.+Leanderauthor=D.+Lodgeauthor=D.+D.+Schoepp&title=Structure-Activity+Studies+of+6-Substituted+Decahydroisoquinoline-3-+Carboxylic+Acid+AMPA+Receptor+Antagonists.+2.+Effects+of+Distal+Acid+Bioisosteric+Substitution%2C+Absolute+Stereochemical+Preferences%2C+and+in+Vivo+Activity&doi=10.1021%2Fjm950913h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm950913h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm950913h%26sid%3Dliteratum%253Aachs%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26aulast%3DArnold%26aufirst%3DM.%2BB.%26aulast%3DAllen%26aufirst%3DN.%2BK.%26aulast%3DBleisch%26aufirst%3DT.%26aulast%3DBorromeo%26aufirst%3DP.%2BS.%26aulast%3DLugar%26aufirst%3DC.%2BW.%26aulast%3DLeander%26aufirst%3DJ.%2BD.%26aulast%3DLodge%26aufirst%3DD.%26aulast%3DSchoepp%26aufirst%3DD.%2BD.%26atitle%3DStructure-Activity%2520Studies%2520of%25206-Substituted%2520Decahydroisoquinoline-3-%2520Carboxylic%2520Acid%2520AMPA%2520Receptor%2520Antagonists.%25202.%2520Effects%2520of%2520Distal%2520Acid%2520Bioisosteric%2520Substitution%252C%2520Absolute%2520Stereochemical%2520Preferences%252C%2520and%2520in%2520Vivo%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2232%26epage%3D2244%26doi%3D10.1021%2Fjm950913h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Ornstein, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleisch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borromeo, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lugar, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leander, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lodge, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepp, D. D.</span><span> </span><span class="NLM_article-title">Structure-Activity Studies of 6-(Tetrazolylalkyl)-Substituted Decahydroisoquinoline-3-Carboxylic Acid AMPA Receptor Antagonists. 1. Effects of Stereochemistry, Chain Length, and Chain Substitution</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">2219</span><span class="NLM_x">–</span> <span class="NLM_lpage">2231</span><span class="refDoi"> DOI: 10.1021/jm950912p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm950912p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=2219-2231&author=P.+L.+Ornsteinauthor=M.+B.+Arnoldauthor=N.+K.+Allenauthor=T.+Bleischauthor=P.+S.+Borromeoauthor=C.+W.+Lugarauthor=J.+D.+Leanderauthor=D.+Lodgeauthor=D.+D.+Schoepp&title=Structure-Activity+Studies+of+6-%28Tetrazolylalkyl%29-Substituted+Decahydroisoquinoline-3-Carboxylic+Acid+AMPA+Receptor+Antagonists.+1.+Effects+of+Stereochemistry%2C+Chain+Length%2C+and+Chain+Substitution&doi=10.1021%2Fjm950912p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm950912p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm950912p%26sid%3Dliteratum%253Aachs%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26aulast%3DArnold%26aufirst%3DM.%2BB.%26aulast%3DAllen%26aufirst%3DN.%2BK.%26aulast%3DBleisch%26aufirst%3DT.%26aulast%3DBorromeo%26aufirst%3DP.%2BS.%26aulast%3DLugar%26aufirst%3DC.%2BW.%26aulast%3DLeander%26aufirst%3DJ.%2BD.%26aulast%3DLodge%26aufirst%3DD.%26aulast%3DSchoepp%26aufirst%3DD.%2BD.%26atitle%3DStructure-Activity%2520Studies%2520of%25206-%2528Tetrazolylalkyl%2529-Substituted%2520Decahydroisoquinoline-3-Carboxylic%2520Acid%2520AMPA%2520Receptor%2520Antagonists.%25201.%2520Effects%2520of%2520Stereochemistry%252C%2520Chain%2520Length%252C%2520and%2520Chain%2520Substitution%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D2219%26epage%3D2231%26doi%3D10.1021%2Fjm950912p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Szymańska, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Contreras-Sanz, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frola, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serafimoska, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johansen, T. N.</span><span> </span><span class="NLM_article-title">A New Phenylalanine Derivative Acts as an Antagonist at the AMPA Receptor GluA2 and Introduces Partial Domain Closure: Synthesis, Resolution, Pharmacology, and Crystal Structure</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">7289</span><span class="NLM_x">–</span> <span class="NLM_lpage">7298</span><span class="refDoi"> DOI: 10.1021/jm200862h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200862h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=7289-7298&author=E.+Szyma%C5%84skaauthor=K.+Frydenvangauthor=A.+Contreras-Sanzauthor=D.+S.+Pickeringauthor=E.+Frolaauthor=Z.+Serafimoskaauthor=B.+Nielsenauthor=J.+S.+Kastrupauthor=T.+N.+Johansen&title=A+New+Phenylalanine+Derivative+Acts+as+an+Antagonist+at+the+AMPA+Receptor+GluA2+and+Introduces+Partial+Domain+Closure%3A+Synthesis%2C+Resolution%2C+Pharmacology%2C+and+Crystal+Structure&doi=10.1021%2Fjm200862h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm200862h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200862h%26sid%3Dliteratum%253Aachs%26aulast%3DSzyma%25C5%2584ska%26aufirst%3DE.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DContreras-Sanz%26aufirst%3DA.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DFrola%26aufirst%3DE.%26aulast%3DSerafimoska%26aufirst%3DZ.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26aulast%3DJohansen%26aufirst%3DT.%2BN.%26atitle%3DA%2520New%2520Phenylalanine%2520Derivative%2520Acts%2520as%2520an%2520Antagonist%2520at%2520the%2520AMPA%2520Receptor%2520GluA2%2520and%2520Introduces%2520Partial%2520Domain%2520Closure%253A%2520Synthesis%252C%2520Resolution%252C%2520Pharmacology%252C%2520and%2520Crystal%2520Structure%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D7289%26epage%3D7298%26doi%3D10.1021%2Fjm200862h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">More, J. C. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Troop, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolman, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jane, D. E.</span><span> </span><span class="NLM_article-title">Structural Requirements for Novel Willardiine Derivatives Acting as AMPA and Kainate Receptor Antagonists</span> <span class="citation_source-journal">Br. J. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">1093</span><span class="NLM_x">–</span> <span class="NLM_lpage">1100</span><span class="refDoi"> DOI: 10.1038/sj.bjp.0705148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1038%2Fsj.bjp.0705148" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=12684265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjtVahsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2003&pages=1093-1100&author=J.+C.+a+Moreauthor=H.+M.+Troopauthor=N.+P.+Dolmanauthor=D.+E.+Jane&title=Structural+Requirements+for+Novel+Willardiine+Derivatives+Acting+as+AMPA+and+Kainate+Receptor+Antagonists&doi=10.1038%2Fsj.bjp.0705148"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Structural requirements for novel willardiine derivatives acting as AMPA and kainate receptor antagonists</span></div><div class="casAuthors">More, Julia C. A.; Troop, Helen M.; Dolman, Nigel P.; Jane, David E.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1093-1100</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">0007-1188</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The natural product willardiine is an AMPA receptor agonist.  We have examd. the structural changes required to convert willardiine into an antagonist at AMPA and kainate receptors.  Structure-activity anal. has been carried out to discover the structural features required to increase the potency and/or selectivity of the antagonists at AMPA or kainate receptors.  Redn. of the fast component of the dorsal root-evoked ventral root potential (fDR-VRP) has been used to investigate AMPA receptor antagonist activity.  To examine antagonist activity at kainate receptors, the ability of compds. to depress kainate-induced depolarizations of dorsal root fibers was assessed.  Blocking ionization of the uracil ring by adding a Me group to the N3 position was not sufficient to convert willardiine into an antagonist.  However, willardiine derivs. with a side-chain bearing a carboxylic acid group at the N3-position of the uracil ring could antagonize AMPA and kainate receptors.  S stereochem. was optimal for antagonism.  When compds. with differing interacidic group chain lengths were compared, a group chain length of two methylene groups was preferable for AMPA receptor antagonism in the series of compds. bearing a carboxyalkyl side chain (UBP275, UBP277 and UBP279 reduced the fDR-VRP with IC50 values of 287 ± 41, 23.8 ± 3.9 and 136 ± 17 μM, resp.).  For kainate receptor antagonism, two or three methylene groups were almost equally acceptable (UBP277 and UBP279 reduced dorsal root kainate responses with apparent KD values of 73.1 ± 4.5 and 60.5 ± 4.1 μM, resp.).  Adding an iodo group to the 5-position of UBP277 and UBP282 enhanced activity at kainate receptors (UBP291 and UBP301 antagonized kainate responses on the dorsal root with apparent KD values of 9.83 ± 1.62 and 5.94 ± 0.63 μM, resp.).  The most useful antagonist identified in this study was UBP301, which was a potent and ∼30-fold selective kainate receptor antagonist.  UBP282 may also be of use in isolating a non-GluR5-mediated kainate response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5Qgxlye0jorVg90H21EOLACvtfcHk0lgnCcd4ZP8-fA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjtVahsbk%253D&md5=1005bbe6195892c0bef935b0ebe75f1f</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fsj.bjp.0705148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.bjp.0705148%26sid%3Dliteratum%253Aachs%26aulast%3DMore%26aufirst%3DJ.%2BC.%2Ba%26aulast%3DTroop%26aufirst%3DH.%2BM.%26aulast%3DDolman%26aufirst%3DN.%2BP.%26aulast%3DJane%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520Requirements%2520for%2520Novel%2520Willardiine%2520Derivatives%2520Acting%2520as%2520AMPA%2520and%2520Kainate%2520Receptor%2520Antagonists%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2003%26volume%3D138%26spage%3D1093%26epage%3D1100%26doi%3D10.1038%2Fsj.bjp.0705148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Larsen, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venskutonytė, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valadés, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Discovery of a New Class of Ionotropic Glutamate Receptor Antagonists by the Rational Design of (2S,3R)-3-(3-Carboxyphenyl)-Pyrrolidine-2-Carboxylic Acid</span> <span class="citation_source-journal">ACS Chem. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">107</span><span class="NLM_x">–</span> <span class="NLM_lpage">114</span><span class="refDoi"> DOI: 10.1021/cn100093f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cn100093f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=107-114&author=A.+M.+Larsenauthor=R.+Venskutonyt%C4%97author=E.+A.+Valad%C3%A9sauthor=B.+Nielsenauthor=D.+S.+Pickeringauthor=L.+Bunch&title=Discovery+of+a+New+Class+of+Ionotropic+Glutamate+Receptor+Antagonists+by+the+Rational+Design+of+%282S%2C3R%29-3-%283-Carboxyphenyl%29-Pyrrolidine-2-Carboxylic+Acid&doi=10.1021%2Fcn100093f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a New Class of Ionotropic Glutamate Receptor Antagonists by the Rational Design of (2S,3R)-3-(3-Carboxyphenyl)-pyrrolidine-2-carboxylic Acid</span></div><div class="casAuthors">Larsen, Ann M.; Venskutonyte, Raminta; Valades, Elena Anton; Nielsen, Birgitte; Pickering, Darryl S.; Bunch, Lennart</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Neuroscience</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">107-114</span>CODEN:
                <span class="NLM_cas:coden">ACNCDM</span>;
        ISSN:<span class="NLM_cas:issn">1948-7193</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The kainic acid (KA) receptors belong to the class of glutamate (Glu) receptors in the brain and constitute a promising target for the treatment of neurol. and/or psychiatric diseases such as schizophrenia, major depression, and epilepsy.  Five KA subtypes have been identified and named GluK1-5.  In this article, we present the discovery of (2S,3R)-3-(3-carboxyphenyl)-pyrrolidine-2-carboxylic acid (1) based on a rational design process.  Target compd. 1 was synthesized by a stereoselective strategy in 10 steps from com. available starting materials.  Binding affinities of 1 at native ionotropic Glu receptors were detd. to be in the micromolar range (AMPA, 51 μM; KA, 22 μM; NMDA 6 μM), with the highest affinity for cloned homomeric KA receptor subtypes GluK1,3 (3.0 and 8.1 μM, resp.).  Functional characterization of 1 by two electrode voltage clamp (TEVC) electrophysiol. at a nondesensitizing mutant of GluK1 showed full competitive antagonistic behavior with a Kb of 11.4 μM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe72VGdj51HrVg90H21EOLACvtfcHk0lgG4Qik5bSVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVWhsbvE&md5=dfe53691cb11fa493b0aaaec96a60909</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fcn100093f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcn100093f%26sid%3Dliteratum%253Aachs%26aulast%3DLarsen%26aufirst%3DA.%2BM.%26aulast%3DVenskutonyt%25C4%2597%26aufirst%3DR.%26aulast%3DValad%25C3%25A9s%26aufirst%3DE.%2BA.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520a%2520New%2520Class%2520of%2520Ionotropic%2520Glutamate%2520Receptor%2520Antagonists%2520by%2520the%2520Rational%2520Design%2520of%2520%25282S%252C3R%2529-3-%25283-Carboxyphenyl%2529-Pyrrolidine-2-Carboxylic%2520Acid%26jtitle%3DACS%2520Chem.%2520Neurosci.%26date%3D2011%26volume%3D2%26spage%3D107%26epage%3D114%26doi%3D10.1021%2Fcn100093f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Krogsgaard-Larsen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storgaard, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Møller, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Demmer, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monrad, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tapken, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Structure-Activity Relationship Study of Ionotropic Glutamate Receptor Antagonist (2S,3R)-3-(3-Carboxyphenyl)pyrrolidine-2-Carboxylic Acid</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">6131</span><span class="NLM_x">–</span> <span class="NLM_lpage">6150</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00750</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00750" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=6131-6150&author=N.+Krogsgaard-Larsenauthor=M.+Storgaardauthor=C.+M%C3%B8llerauthor=C.+S.+Demmerauthor=J.+Hansenauthor=L.+Hanauthor=R.+N.+Monradauthor=B.+Nielsenauthor=D.+Tapkenauthor=D.+S.+Pickeringauthor=J.+S.+Kastrupauthor=K.+Frydenvangauthor=L.+Bunch&title=Structure-Activity+Relationship+Study+of+Ionotropic+Glutamate+Receptor+Antagonist+%282S%2C3R%29-3-%283-Carboxyphenyl%29pyrrolidine-2-Carboxylic+Acid&doi=10.1021%2Facs.jmedchem.5b00750"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00750%26sid%3Dliteratum%253Aachs%26aulast%3DKrogsgaard-Larsen%26aufirst%3DN.%26aulast%3DStorgaard%26aufirst%3DM.%26aulast%3DM%25C3%25B8ller%26aufirst%3DC.%26aulast%3DDemmer%26aufirst%3DC.%2BS.%26aulast%3DHansen%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DMonrad%26aufirst%3DR.%2BN.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DTapken%26aufirst%3DD.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DStructure-Activity%2520Relationship%2520Study%2520of%2520Ionotropic%2520Glutamate%2520Receptor%2520Antagonist%2520%25282S%252C3R%2529-3-%25283-Carboxyphenyl%2529pyrrolidine-2-Carboxylic%2520Acid%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D6131%26epage%3D6150%26doi%3D10.1021%2Facs.jmedchem.5b00750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Weiss, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gates, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dieckman, D. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clemens-Smith, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizkalla, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calligaro, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schoepp, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattiuz, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salhoff, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katofiasc, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phebus, L. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schenck, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filla, S. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ornstein, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleakman, D.</span><span> </span><span class="NLM_article-title">Pharmacological Characterization of the Competitive GLU K5 Receptor Antagonist Decahydroisoquinoline LY466195 in Vitro and in Vivo</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">318</span><span class="NLM_x">, </span> <span class="NLM_fpage">772</span><span class="NLM_x">–</span> <span class="NLM_lpage">781</span><span class="refDoi"> DOI: 10.1124/jpet.106.101428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1124%2Fjpet.106.101428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=16690725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD28XnslOltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=318&publication_year=2006&pages=772-781&author=B.+Weissauthor=A.+Altauthor=A.+M.+Ogdenauthor=M.+Gatesauthor=D.+K.+Dieckmanauthor=A.+Clemens-Smithauthor=K.+H.+Hoauthor=K.+Jarvieauthor=G.+Rizkallaauthor=R.+A.+Wrightauthor=D.+O.+Calligaroauthor=D.+Schoeppauthor=E.+L.+Mattiuzauthor=R.+E.+Stratfordauthor=B.+Johnsonauthor=C.+Salhoffauthor=M.+Katofiascauthor=L.+a+Phebusauthor=K.+Schenckauthor=M.+Cohenauthor=S.+a+Fillaauthor=P.+L.+Ornsteinauthor=K.+W.+Johnsonauthor=D.+Bleakman&title=Pharmacological+Characterization+of+the+Competitive+GLU+K5+Receptor+Antagonist+Decahydroisoquinoline+LY466195+in+Vitro+and+in+Vivo&doi=10.1124%2Fjpet.106.101428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of the competitive GLUK5 receptor antagonist decahydroisoquinoline LY466195 in vitro and in vivo</span></div><div class="casAuthors">Weiss, Brianne; Alt, Andrew; Ogden, Ann Marie; Gates, Mary; Dieckman, Donna K.; Clemens-Smith, Amy; Ho, Ken H.; Jarvie, Keith; Rizkalla, Geihan; Wright, Rebecca A.; Calligaro, David O.; Schoepp, Darryle; Mattiuz, Edward L.; Stratford, Robert E.; Johnson, Bryan; Salhoff, Craig; Katofiasc, Mary; Phebus, Lee A.; Schenck, Kathryn; Cohen, Marlene; Filla, Sandra A.; Ornstein, Paul L.; Johnson, Kirk W.; Bleakman, David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">318</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">772-781</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">The excitatory neurotransmitter glutamate has been implicated in both migraine and persistent pain.  The identification of the kainate receptor GLUK5 in dorsal root ganglia, the dorsal horn, and trigeminal ganglia makes it a target of interest for these indications.  We examd. the in vitro and in vivo pharmacol. of the competitive GLUK5-selective kainate receptor antagonist LY466195, the most potent GLUK5 antagonist described to date.  Comparisons were made to the competitive GLUK5/α-amino-3-hydroxy-5-methyl-4-isoxazole propionic acid (AMPA) receptor antagonist LY293558 [(3S,4aR,6R,8aR)-6-[2-(1(2)H-tetrazole-5-yl)ethyl]decahydroisoquinoline-3-carboxylic acid], other decahydroisoquinoline GLUK5 receptor antagonists, and the noncompetitive AMPA receptor antagonist LY300168 [1-(4-aminophenyl)-4-methyl-7,8-methylenedioxy-5H-2,3-benzodi-azepine].  When characterized electrophysiol. in rat dorsal root ganglion neurons, LY466195 antagonized kainate (30 μM)-induced currents with an IC50 value of 0.045 ± 0.011 μM.  In HEK293 cells transfected with GLUK5, GLUK2/GLUK5, or GLUK5/GLUK6 receptors, LY466195 produced IC50 values of 0.08 ± 0.02, 0.34 ± 0.17, and 0.07 ± 0.02 μM, resp.  LY466195 was efficacious in a dural plasma protein extravasation (PPE) model of migraine with an ID100 value of 100 μg/kg i.v.  LY466195 was also efficacious in the c-fos migraine model, with a dose of 1 μg/kg i.v. significantly reducing the no. of Fos-pos. cells in the rat nucleus caudalis after elec. stimulation of the trigeminal ganglion.  Furthermore, LY466195 showed no contractile activity in the rabbit saphenous vein in vitro.  The di-Et ester prodrug of LY466195 was also efficacious in the same PPE and c-fos models after oral administration at doses of 10 and 100 μg/kg, resp. while having no N-methyl-D-aspartate antagonist-like behavioral effects at oral doses up to 100 mg/kg.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwe6dtAoo61bVg90H21EOLACvtfcHk0lgG4Qik5bSVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnslOltro%253D&md5=decb91d0148d7a4049f15e1c4622925a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1124%2Fjpet.106.101428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.106.101428%26sid%3Dliteratum%253Aachs%26aulast%3DWeiss%26aufirst%3DB.%26aulast%3DAlt%26aufirst%3DA.%26aulast%3DOgden%26aufirst%3DA.%2BM.%26aulast%3DGates%26aufirst%3DM.%26aulast%3DDieckman%26aufirst%3DD.%2BK.%26aulast%3DClemens-Smith%26aufirst%3DA.%26aulast%3DHo%26aufirst%3DK.%2BH.%26aulast%3DJarvie%26aufirst%3DK.%26aulast%3DRizkalla%26aufirst%3DG.%26aulast%3DWright%26aufirst%3DR.%2BA.%26aulast%3DCalligaro%26aufirst%3DD.%2BO.%26aulast%3DSchoepp%26aufirst%3DD.%26aulast%3DMattiuz%26aufirst%3DE.%2BL.%26aulast%3DStratford%26aufirst%3DR.%2BE.%26aulast%3DJohnson%26aufirst%3DB.%26aulast%3DSalhoff%26aufirst%3DC.%26aulast%3DKatofiasc%26aufirst%3DM.%26aulast%3DPhebus%26aufirst%3DL.%2Ba%26aulast%3DSchenck%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DM.%26aulast%3DFilla%26aufirst%3DS.%2Ba%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26aulast%3DJohnson%26aufirst%3DK.%2BW.%26aulast%3DBleakman%26aufirst%3DD.%26atitle%3DPharmacological%2520Characterization%2520of%2520the%2520Competitive%2520GLU%2520K5%2520Receptor%2520Antagonist%2520Decahydroisoquinoline%2520LY466195%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2006%26volume%3D318%26spage%3D772%26epage%3D781%26doi%3D10.1124%2Fjpet.106.101428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Mayer, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolman, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jane, D. E.</span><span> </span><span class="NLM_article-title">Crystal Structures of the Kainate Receptor GluR5 Ligand Binding Core Dimer with Novel GluR5-Selective Antagonists</span> <span class="citation_source-journal">J. Neurosci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">2852</span><span class="NLM_x">–</span> <span class="NLM_lpage">2861</span><span class="refDoi"> DOI: 10.1523/JNEUROSCI.0123-06.2005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1523%2FJNEUROSCI.0123-06.2005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=16540562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD28XivFKmtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2006&pages=2852-2861&author=M.+L.+Mayerauthor=A.+Ghosalauthor=N.+P.+Dolmanauthor=D.+E.+Jane&title=Crystal+Structures+of+the+Kainate+Receptor+GluR5+Ligand+Binding+Core+Dimer+with+Novel+GluR5-Selective+Antagonists&doi=10.1523%2FJNEUROSCI.0123-06.2005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the kainate receptor GluR5 ligand binding core dimer with novel GluR5-selective antagonists</span></div><div class="casAuthors">Mayer, Mark L.; Ghosal, Alokesh; Dolman, Nigel P.; Jane, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2852-2861</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    
            (<span class="NLM_cas:orgname">Society for Neuroscience</span>)
        </div><div class="casAbstract">Glutamate receptor (GluR) ion channels mediate fast synaptic transmission in the mammalian CNS.  Numerous crystallog. studies, the majority on the GluR2-subtype AMPA receptor, have revealed the structural basis for binding of subtype-specific agonists.  In contrast, because there are far fewer antagonist-bound structures, the mechanisms for antagonist binding are much less well understood, particularly for kainate receptors that exist as multiple subtypes with a distinct biol. encoded by the GluR5-7, KA1, and KA2 genes.  We describe here high-resoln. crystal structures for the GluR5 ligand-binding core complex with UBP302 and UBP310, novel GluR5-selective antagonists.  The crystal structures reveal the structural basis for the high selectivity for GluR5 obsd. in radiolabel displacement assays for the isolated ligand binding cores of the GluR2, GluR5, and GluR6 subunits and during inhibition of glutamate-activated currents in studies on full-length ion channels.  The antagonists bind via a novel mechanism and do not form direct contacts with the E723 side chain, as occurs in all previously solved AMPA and kainate receptor agonist and antagonist complexes.  This results from a hyperextension of the ligand binding core compared with previously solved structures.  As a result, in dimer assemblies, there is a 22 Å extension of the ion channel linkers in the transition from antagonist- to glutamate-bound forms.  This large conformational change is substantially different from that described for AMPA receptors, was not possible to predict from previous work, and suggests that glutamate receptors are capable of much larger movements than previously thought.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFkS4TUumki7Vg90H21EOLACvtfcHk0lg0AVykeO_oWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XivFKmtLs%253D&md5=5a880d1af759237096a2919d856d5e2a</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.0123-06.2005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.0123-06.2005%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DM.%2BL.%26aulast%3DGhosal%26aufirst%3DA.%26aulast%3DDolman%26aufirst%3DN.%2BP.%26aulast%3DJane%26aufirst%3DD.%2BE.%26atitle%3DCrystal%2520Structures%2520of%2520the%2520Kainate%2520Receptor%2520GluR5%2520Ligand%2520Binding%2520Core%2520Dimer%2520with%2520Novel%2520GluR5-Selective%2520Antagonists%26jtitle%3DJ.%2520Neurosci.%26date%3D2006%26volume%3D26%26spage%3D2852%26epage%3D2861%26doi%3D10.1523%2FJNEUROSCI.0123-06.2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Martinez-Perez, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyengar, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shannon, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleakman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alt, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleisch, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castaño, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Del Prado, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominguez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Escribano, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filla, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hudziak, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mateo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathes, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattiuz, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogden, A. M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simmons, R. M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stack, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratford, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ornstein, P. L.</span><span> </span><span class="NLM_article-title">GluK1 Antagonists from 6-(Carboxy)phenyl Decahydroisoquinoline Derivatives. SAR and Evaluation of a Prodrug Strategy for Oral Efficacy in Pain Models</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">6459</span><span class="NLM_x">–</span> <span class="NLM_lpage">6462</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2013.09.046</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.bmcl.2013.09.046" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=6459-6462&author=J.+A.+Martinez-Perezauthor=S.+Iyengarauthor=H.+E.+Shannonauthor=D.+Bleakmanauthor=A.+Altauthor=B.+M.+Arnoldauthor=M.+G.+Bellauthor=T.+J.+Bleischauthor=A.+M.+Casta%C3%B1oauthor=M.+Del+Pradoauthor=E.+Dominguezauthor=A.+M.+Escribanoauthor=S.+A.+Fillaauthor=K.+H.+Hoauthor=K.+J.+Hudziakauthor=C.+K.+Jonesauthor=A.+Mateoauthor=B.+M.+Mathesauthor=E.+L.+Mattiuzauthor=A.+M.+L.+Ogdenauthor=R.+M.+A.+Simmonsauthor=D.+R.+Stackauthor=R.+E.+Stratfordauthor=M.+A.+Winterauthor=Z.+Wuauthor=P.+L.+Ornstein&title=GluK1+Antagonists+from+6-%28Carboxy%29phenyl+Decahydroisoquinoline+Derivatives.+SAR+and+Evaluation+of+a+Prodrug+Strategy+for+Oral+Efficacy+in+Pain+Models&doi=10.1016%2Fj.bmcl.2013.09.046"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.09.046&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.09.046%26sid%3Dliteratum%253Aachs%26aulast%3DMartinez-Perez%26aufirst%3DJ.%2BA.%26aulast%3DIyengar%26aufirst%3DS.%26aulast%3DShannon%26aufirst%3DH.%2BE.%26aulast%3DBleakman%26aufirst%3DD.%26aulast%3DAlt%26aufirst%3DA.%26aulast%3DArnold%26aufirst%3DB.%2BM.%26aulast%3DBell%26aufirst%3DM.%2BG.%26aulast%3DBleisch%26aufirst%3DT.%2BJ.%26aulast%3DCasta%25C3%25B1o%26aufirst%3DA.%2BM.%26aulast%3DDel%2BPrado%26aufirst%3DM.%26aulast%3DDominguez%26aufirst%3DE.%26aulast%3DEscribano%26aufirst%3DA.%2BM.%26aulast%3DFilla%26aufirst%3DS.%2BA.%26aulast%3DHo%26aufirst%3DK.%2BH.%26aulast%3DHudziak%26aufirst%3DK.%2BJ.%26aulast%3DJones%26aufirst%3DC.%2BK.%26aulast%3DMateo%26aufirst%3DA.%26aulast%3DMathes%26aufirst%3DB.%2BM.%26aulast%3DMattiuz%26aufirst%3DE.%2BL.%26aulast%3DOgden%26aufirst%3DA.%2BM.%2BL.%26aulast%3DSimmons%26aufirst%3DR.%2BM.%2BA.%26aulast%3DStack%26aufirst%3DD.%2BR.%26aulast%3DStratford%26aufirst%3DR.%2BE.%26aulast%3DWinter%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DOrnstein%26aufirst%3DP.%2BL.%26atitle%3DGluK1%2520Antagonists%2520from%25206-%2528Carboxy%2529phenyl%2520Decahydroisoquinoline%2520Derivatives.%2520SAR%2520and%2520Evaluation%2520of%2520a%2520Prodrug%2520Strategy%2520for%2520Oral%2520Efficacy%2520in%2520Pain%2520Models%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D6459%26epage%3D6462%26doi%3D10.1016%2Fj.bmcl.2013.09.046" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Jensen, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bräuner-Osborne, H.</span><span> </span><span class="NLM_article-title">Pharmacological Characterization of Human Excitatory Amino Acid Transporters EAAT1, EAAT2 and EAAT3 in a Fluorescence-Based Membrane Potential Assay</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">2115</span><span class="NLM_x">–</span> <span class="NLM_lpage">2127</span><span class="refDoi"> DOI: 10.1016/j.bcp.2004.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.bcp.2004.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=15135308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjvVCis78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2004&pages=2115-2127&author=A.+A.+Jensenauthor=H.+Br%C3%A4uner-Osborne&title=Pharmacological+Characterization+of+Human+Excitatory+Amino+Acid+Transporters+EAAT1%2C+EAAT2+and+EAAT3+in+a+Fluorescence-Based+Membrane+Potential+Assay&doi=10.1016%2Fj.bcp.2004.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological characterization of human excitatory amino acid transporters EAAT1, EAAT2 and EAAT3 in a fluorescence-based membrane potential assay</span></div><div class="casAuthors">Jensen, Anders A.; Brauner-Osborne, Hans</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Pharmacology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2115-2127</span>CODEN:
                <span class="NLM_cas:coden">BCPCA6</span>;
        ISSN:<span class="NLM_cas:issn">0006-2952</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science B.V.</span>)
        </div><div class="casAbstract">We have expressed the human excitatory amino acid transporters EAAT1, EAAT2 and EAAT3 stably in HEK293 cells and characterized the transporters pharmacol. in a conventional [3H]-d-aspartate uptake assay and in a fluorescence-based membrane potential assay, the FLIPR Membrane Potential (FMP) assay.  The Km and Ki values obtained for 12 std. EAAT ligands at EAAT1, EAAT2 and EAAT3 in the FMP assay correlated well with the Ki values obtained in the [3H]-d-aspartate assay (r2 values of 0.92, 0.92, and 0.95, resp.).  Furthermore, the pharmacol. characteristics of the cell lines in the FMP assay were in good agreement with previous findings in electrophysiol. studies of the transporters.  The FMP assay was capable of distinguishing between substrates and non-substrate inhibitors and to discriminate between "full" and "partial" substrates at the transporters.  Taking advantage of the prolific nature of the FMP assay, interactions of the EAATs with substrates and inhibitors were studied in some detail.  This is the first report of a high throughput screening assay for EAATs.  We propose that the assay will be of great use in future studies of the transporters.  Although conventional electrophysiol. set-ups might be superior in terms of studying sophisticated kinetic aspects of the uptake process, the FMP assay enables the collection of considerable amts. of highly reproducible data with relatively little labor.  Furthermore, considering that the no. of EAAT ligands presently available is limited, and that almost all of these are characterized by low potency and a low degree of subtype selectivity, future screening of compd. libraries at the EAAT-cell lines in the FMP assay could help identify structurally and pharmacol. novel ligands for the transporters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpS6Um79wVYz7Vg90H21EOLACvtfcHk0lguBRfOwWrgvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjvVCis78%253D&md5=06e3c536844acca52fdba733175a93c6</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2004.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2004.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DJensen%26aufirst%3DA.%2BA.%26aulast%3DBr%25C3%25A4uner-Osborne%26aufirst%3DH.%26atitle%3DPharmacological%2520Characterization%2520of%2520Human%2520Excitatory%2520Amino%2520Acid%2520Transporters%2520EAAT1%252C%2520EAAT2%2520and%2520EAAT3%2520in%2520a%2520Fluorescence-Based%2520Membrane%2520Potential%2520Assay%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2004%26volume%3D67%26spage%3D2115%26epage%3D2127%26doi%3D10.1016%2Fj.bcp.2004.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Møllerud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span> </span><span class="NLM_article-title">Lessons from Crystal Structures of Kainate Receptors</span> <span class="citation_source-journal">Neuropharmacology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">16</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span><span class="refDoi"> DOI: 10.1016/j.neuropharm.2016.05.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.neuropharm.2016.05.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=27236079" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FksVWhsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2017&pages=16-28&author=S.+M%C3%B8llerudauthor=K.+Frydenvangauthor=D.+S.+Pickeringauthor=J.+S.+Kastrup&title=Lessons+from+Crystal+Structures+of+Kainate+Receptors&doi=10.1016%2Fj.neuropharm.2016.05.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons from crystal structures of kainate receptors</span></div><div class="casAuthors">Mollerud Stine; Frydenvang Karla; Pickering Darryl S; Kastrup Jette Sandholm</div><div class="citationInfo"><span class="NLM_cas:title">Neuropharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">Pt A</span>),
    <span class="NLM_cas:pages">16-28</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Kainate receptors belong to the family of ionotropic glutamate receptors.  These receptors assemble from five subunits (GluK1-5) into tetrameric ion channels.  Kainate receptors are located at both pre- and postsynaptic membranes in the central nervous system where they contribute to excitatory synaptic transmission and modulate network excitability by regulating neurotransmitter release.  Dysfunction of kainate receptors has been implicated in several neurological disorders such as epilepsy, schizophrenia and depression.  Here we provide a review on the current understanding of kainate receptor structure and how they bind agonists, antagonists and ions.  The first structure of the ligand-binding domain of the GluK1 subunit was reported in 2005, seven years after publication of the crystal structure of a soluble construct of the ligand-binding domain of the AMPA-type subunit GluA2.  Today, a full-length structure has been determined of GluK2 by cryo electron microscopy to 7.6 ÅA resolution as well as 84 high-resolution crystal structures of N-terminal domains and ligand-binding domains, including agonist and antagonist bound structures, modulatory ions and mutations.  However, there are still many unanswered questions and challenges in front of us.  This article is part of the Special Issue entitled 'Ionotropic glutamate receptors'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSorFhi6dO1iNhSDZx9hyjKfW6udTcc2eZY9V_X8xuFW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FksVWhsw%253D%253D&md5=7b98312e2ef8a69ecac23e2d2a045bb7</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2016.05.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2016.05.014%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25B8llerud%26aufirst%3DS.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26atitle%3DLessons%2520from%2520Crystal%2520Structures%2520of%2520Kainate%2520Receptors%26jtitle%3DNeuropharmacology%26date%3D2017%26volume%3D112%26spage%3D16%26epage%3D28%26doi%3D10.1016%2Fj.neuropharm.2016.05.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Hald, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprogøe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madsen, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmermann, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahring, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liljefors, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schousboe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egebjerg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajhede, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span> </span><span class="NLM_article-title">Partial Agonism and Antagonism of the Ionotropic Glutamate Receptor iGluR5: Structures of the Ligand-Binding Core in Complex with Domoic Acid and 2-Amino-3-[5-Tert-Butyl-3-(Phosphonomethoxy)-4-Isoxazolyl]propionic Acid</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">25726</span><span class="NLM_x">–</span> <span class="NLM_lpage">25736</span><span class="refDoi"> DOI: 10.1074/jbc.M700137200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1074%2Fjbc.M700137200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=25726-25736&author=H.+Haldauthor=P.+Naurauthor=D.+S.+Pickeringauthor=D.+Sprog%C3%B8eauthor=U.+Madsenauthor=D.+B.+Timmermannauthor=P.+K.+Ahringauthor=T.+Liljeforsauthor=A.+Schousboeauthor=J.+Egebjergauthor=M.+Gajhedeauthor=J.+S.+Kastrup&title=Partial+Agonism+and+Antagonism+of+the+Ionotropic+Glutamate+Receptor+iGluR5%3A+Structures+of+the+Ligand-Binding+Core+in+Complex+with+Domoic+Acid+and+2-Amino-3-%5B5-Tert-Butyl-3-%28Phosphonomethoxy%29-4-Isoxazolyl%5Dpropionic+Acid&doi=10.1074%2Fjbc.M700137200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M700137200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M700137200%26sid%3Dliteratum%253Aachs%26aulast%3DHald%26aufirst%3DH.%26aulast%3DNaur%26aufirst%3DP.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DSprog%25C3%25B8e%26aufirst%3DD.%26aulast%3DMadsen%26aufirst%3DU.%26aulast%3DTimmermann%26aufirst%3DD.%2BB.%26aulast%3DAhring%26aufirst%3DP.%2BK.%26aulast%3DLiljefors%26aufirst%3DT.%26aulast%3DSchousboe%26aufirst%3DA.%26aulast%3DEgebjerg%26aufirst%3DJ.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26atitle%3DPartial%2520Agonism%2520and%2520Antagonism%2520of%2520the%2520Ionotropic%2520Glutamate%2520Receptor%2520iGluR5%253A%2520Structures%2520of%2520the%2520Ligand-Binding%2520Core%2520in%2520Complex%2520with%2520Domoic%2520Acid%2520and%25202-Amino-3-%255B5-Tert-Butyl-3-%2528Phosphonomethoxy%2529-4-Isoxazolyl%255Dpropionic%2520Acid%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D25726%26epage%3D25736%26doi%3D10.1074%2Fjbc.M700137200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: All-Atom Structure Validation for Macromolecular Crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+All-Atom+Structure+Validation+for+Macromolecular+Crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0ljj1aIScO7kLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520All-Atom%2520Structure%2520Validation%2520for%2520Macromolecular%2520Crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Epple, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Azimioara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wityak, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karanewsky, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bursulaya, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kreusch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tuntland, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerken, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iskandar, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saez, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin Seidel, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, S. S.</span><span> </span><span class="NLM_article-title">3,4,5-Trisubstituted Isoxazoles as Novel PPARδ Agonists. Part 2</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">5488</span><span class="NLM_x">–</span> <span class="NLM_lpage">5492</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2006.08.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.bmcl.2006.08.052" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=5488-5492&author=R.+Eppleauthor=M.+Azimioaraauthor=R.+Russoauthor=Y.+Xieauthor=X.+Wangauthor=C.+Cowauthor=J.+Wityakauthor=D.+Karanewskyauthor=B.+Bursulayaauthor=A.+Kreuschauthor=T.+Tuntlandauthor=A.+Gerkenauthor=M.+Iskandarauthor=E.+Saezauthor=H.+Martin+Seidelauthor=S.+S.+Tian&title=3%2C4%2C5-Trisubstituted+Isoxazoles+as+Novel+PPAR%CE%B4+Agonists.+Part+2&doi=10.1016%2Fj.bmcl.2006.08.052"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.08.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.08.052%26sid%3Dliteratum%253Aachs%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DAzimioara%26aufirst%3DM.%26aulast%3DRusso%26aufirst%3DR.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCow%26aufirst%3DC.%26aulast%3DWityak%26aufirst%3DJ.%26aulast%3DKaranewsky%26aufirst%3DD.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DKreusch%26aufirst%3DA.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DGerken%26aufirst%3DA.%26aulast%3DIskandar%26aufirst%3DM.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DMartin%2BSeidel%26aufirst%3DH.%26aulast%3DTian%26aufirst%3DS.%2BS.%26atitle%3D3%252C4%252C5-Trisubstituted%2520Isoxazoles%2520as%2520Novel%2520PPAR%25CE%25B4%2520Agonists.%2520Part%25202%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D5488%26epage%3D5492%26doi%3D10.1016%2Fj.bmcl.2006.08.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Assaf, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsen, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venskutonytė, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abrahamsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajhede, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, A. a</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frydenvang, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gefflaut, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Chemoenzymatic Synthesis of New 2,4-Syn-Functionalized (S)-Glutamate Analogues and Structure-Activity Relationship Studies at Ionotropic Glutamate Receptors and Excitatory Amino Acid Transporters</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1614</span><span class="NLM_x">–</span> <span class="NLM_lpage">1628</span><span class="refDoi"> DOI: 10.1021/jm301433m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301433m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1614-1628&author=Z.+Assafauthor=A.+P.+Larsenauthor=R.+Venskutonyt%C4%97author=L.+Hanauthor=B.+Abrahamsenauthor=B.+Nielsenauthor=M.+Gajhedeauthor=J.+S.+Kastrupauthor=A.+a+Jensenauthor=D.+S.+Pickeringauthor=K.+Frydenvangauthor=T.+Gefflautauthor=L.+Bunch&title=Chemoenzymatic+Synthesis+of+New+2%2C4-Syn-Functionalized+%28S%29-Glutamate+Analogues+and+Structure-Activity+Relationship+Studies+at+Ionotropic+Glutamate+Receptors+and+Excitatory+Amino+Acid+Transporters&doi=10.1021%2Fjm301433m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Chemoenzymatic synthesis of new 2,4-syn-functionalized (S)-glutamate analogues and structure-activity relationship studies at ionotropic glutamate receptors and excitatory amino acid transporters</span></div><div class="casAuthors">Assaf, Zeinab; Larsen, Anja P.; Venskutonyte, Raminta; Han, Liwei; Abrahamsen, Bjarke; Nielsen, Birgitte; Gajhede, Michael; Kastrup, Jette S.; Jensen, Anders A.; Pickering, Darryl S.; Frydenvang, Karla; Gefflaut, Thierry; Bunch, Lennart</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1614-1628</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the mammalian central nervous system, (S)-glutamate (Glu) is released from the presynaptic neuron where it activates a plethora of pre- and postsynaptic Glu receptors.  The fast acting ionotropic Glu receptors (iGluRs) are ligand gated ion channels and are believed to be involved in a vast no. of neurol. functions such as memory and learning, synaptic plasticity, and motor function.  The synthesis of 14 enantiopure 2,4-syn-Glu analogs, e.g. (I), (II), (III), is accessed by a short and efficient chemoenzymic approach starting from readily available cyclohexanone (IV).  Pharmacol. characterization at the iGluRs and EAAT1-3 subtypes revealed analog II as a selective GluK1 ligand with low nanomolar affinity.  Two X-ray crystal structures of the key analog II in the ligand-binding domain (LBD) of GluA2 and GluK3 were detd.  Partial domain closure was seen in the GluA2-LBD complex with II comparable to that induced by kainate.  In contrast, full domain closure was obsd. in the GluK3-LBD complex with II, similar to that of GluK3-LBD with glutamate bound.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNDU3_wI6ErbVg90H21EOLACvtfcHk0ljj1aIScO7kLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Sktbs%253D&md5=820d359e25bfc4d9505c20bc303cd67d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fjm301433m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301433m%26sid%3Dliteratum%253Aachs%26aulast%3DAssaf%26aufirst%3DZ.%26aulast%3DLarsen%26aufirst%3DA.%2BP.%26aulast%3DVenskutonyt%25C4%2597%26aufirst%3DR.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DAbrahamsen%26aufirst%3DB.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26aulast%3DJensen%26aufirst%3DA.%2Ba%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DFrydenvang%26aufirst%3DK.%26aulast%3DGefflaut%26aufirst%3DT.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DChemoenzymatic%2520Synthesis%2520of%2520New%25202%252C4-Syn-Functionalized%2520%2528S%2529-Glutamate%2520Analogues%2520and%2520Structure-Activity%2520Relationship%2520Studies%2520at%2520Ionotropic%2520Glutamate%2520Receptors%2520and%2520Excitatory%2520Amino%2520Acid%2520Transporters%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1614%26epage%3D1628%26doi%3D10.1021%2Fjm301433m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Alcaide, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marconi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haym, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nielsen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Møllerud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pickering, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunch, L.</span><span> </span><span class="NLM_article-title">Synthesis and Pharmacological Characterization of the Selective GluK1 Radioligand (S)-2-Amino-3-(6-[ 3 H]-2,4-Dioxo-3,4-dihydrothieno[3,2-D]pyrimidin-1(2H)-Yl)propanoic Acid ([3H]-NF608)</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">2136</span><span class="NLM_x">–</span> <span class="NLM_lpage">2144</span><span class="refDoi"> DOI: 10.1039/C6MD00339G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1039%2FC6MD00339G" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2136-2144&author=A.+Alcaideauthor=L.+Marconiauthor=A.+Marekauthor=I.+Haymauthor=B.+Nielsenauthor=S.+M%C3%B8llerudauthor=M.+Jensenauthor=P.+Contiauthor=D.+S.+Pickeringauthor=L.+Bunch&title=Synthesis+and+Pharmacological+Characterization+of+the+Selective+GluK1+Radioligand+%28S%29-2-Amino-3-%286-%5B+3+H%5D-2%2C4-Dioxo-3%2C4-dihydrothieno%5B3%2C2-D%5Dpyrimidin-1%282H%29-Yl%29propanoic+Acid+%28%5B3H%5D-NF608%29&doi=10.1039%2FC6MD00339G"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1039%2FC6MD00339G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00339G%26sid%3Dliteratum%253Aachs%26aulast%3DAlcaide%26aufirst%3DA.%26aulast%3DMarconi%26aufirst%3DL.%26aulast%3DMarek%26aufirst%3DA.%26aulast%3DHaym%26aufirst%3DI.%26aulast%3DNielsen%26aufirst%3DB.%26aulast%3DM%25C3%25B8llerud%26aufirst%3DS.%26aulast%3DJensen%26aufirst%3DM.%26aulast%3DConti%26aufirst%3DP.%26aulast%3DPickering%26aufirst%3DD.%2BS.%26aulast%3DBunch%26aufirst%3DL.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520Characterization%2520of%2520the%2520Selective%2520GluK1%2520Radioligand%2520%2528S%2529-2-Amino-3-%25286-%255B%25203%2520H%255D-2%252C4-Dioxo-3%252C4-dihydrothieno%255B3%252C2-D%255Dpyrimidin-1%25282H%2529-Yl%2529propanoic%2520Acid%2520%2528%255B3H%255D-NF608%2529%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D2136%26epage%3D2144%26doi%3D10.1039%2FC6MD00339G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Naur, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vestergaard, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skov, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egebjerg, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gajhede, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kastrup, J. S.</span><span> </span><span class="NLM_article-title">Crystal Structure of the Kainate Receptor GluR5 Ligand-Binding Core in Complex with (S)-Glutamate</span> <span class="citation_source-journal">FEBS Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">579</span><span class="NLM_x">, </span> <span class="NLM_fpage">1154</span><span class="NLM_x">–</span> <span class="NLM_lpage">1160</span><span class="refDoi"> DOI: 10.1016/j.febslet.2005.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1016%2Fj.febslet.2005.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=15710405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlOnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=579&publication_year=2005&pages=1154-1160&author=P.+Naurauthor=B.+Vestergaardauthor=L.+K.+Skovauthor=J.+Egebjergauthor=M.+Gajhedeauthor=J.+S.+Kastrup&title=Crystal+Structure+of+the+Kainate+Receptor+GluR5+Ligand-Binding+Core+in+Complex+with+%28S%29-Glutamate&doi=10.1016%2Fj.febslet.2005.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the kainate receptor GluR5 ligand-binding core in complex with (S)-glutamate</span></div><div class="casAuthors">Naur, Peter; Vestergaard, Bente; Skov, Lars K.; Egebjerg, Jan; Gajhede, Michael; Kastrup, Jette Sandholm</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">579</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1154-1160</span>CODEN:
                <span class="NLM_cas:coden">FEBLAL</span>;
        ISSN:<span class="NLM_cas:issn">0014-5793</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The x-ray structure of the ligand-binding core of the kainate receptor GluR5 (GluR5-S1S2) in complex with (S)-glutamate was detd. to 1.95 Å resoln.  The overall GluR5-S1S2 structure comprises two domains and is similar to the related AMPA receptor GluR2-S1S2J. (S)-glutamate binds as in GluR2-S1S2J.  Distinct features are obsd. for Ser741, which stabilizes a highly coordinated network of water mols. and forms an interdomain bridge.  The GluR5 complex exhibits a high degree of domain closure (26°) relative to apo GluR2-S1S2J.  In addn., GluR5-S1S2 forms a novel dimer interface with a different arrangement of the two protomers compared to GluR2-S1S2J.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbtbs9jrVfsbVg90H21EOLACvtfcHk0lj2ejuoChm-vg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlOnsLg%253D&md5=53693a84dad10af87aa0cff370788d8b</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2005.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2005.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DNaur%26aufirst%3DP.%26aulast%3DVestergaard%26aufirst%3DB.%26aulast%3DSkov%26aufirst%3DL.%2BK.%26aulast%3DEgebjerg%26aufirst%3DJ.%26aulast%3DGajhede%26aufirst%3DM.%26aulast%3DKastrup%26aufirst%3DJ.%2BS.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520Kainate%2520Receptor%2520GluR5%2520Ligand-Binding%2520Core%2520in%2520Complex%2520with%2520%2528S%2529-Glutamate%26jtitle%3DFEBS%2520Lett.%26date%3D2005%26volume%3D579%26spage%3D1154%26epage%3D1160%26doi%3D10.1016%2Fj.febslet.2005.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Ursby, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unge, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Appio, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Logan, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fredslund, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Svensson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larsson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labrador, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thunnissen, M. M. G. M.</span><span> </span><span class="NLM_article-title">The Macromolecular Crystallography Beamline I911–3 at the MAX IV Laboratory</span> <span class="citation_source-journal">J. Synchrotron Radiat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">648</span><span class="NLM_x">–</span> <span class="NLM_lpage">653</span><span class="refDoi"> DOI: 10.1107/S0909049513011734</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0909049513011734" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=23765310" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=648-653&author=T.+Ursbyauthor=J.+Ungeauthor=R.+Appioauthor=D.+T.+Loganauthor=F.+Fredslundauthor=C.+Svenssonauthor=K.+Larssonauthor=A.+Labradorauthor=M.+M.+G.+M.+Thunnissen&title=The+Macromolecular+Crystallography+Beamline+I911%E2%80%933+at+the+MAX+IV+Laboratory&doi=10.1107%2FS0909049513011734"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">The macromolecular crystallography beamline I911-3 at the MAX IV laboratory</span></div><div class="casAuthors">Ursby, Thomas; Unge, Johan; Appio, Roberto; Logan, Derek T.; Fredslund, Folmer; Svensson, Christer; Larsson, Krister; Labrador, Ana; Thunnissen, Marjolein M. G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Synchrotron Radiation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">648-653</span>CODEN:
                <span class="NLM_cas:coden">JSYRES</span>;
        ISSN:<span class="NLM_cas:issn">0909-0495</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The macromol. crystallog. beamline I911-3, part of the Cassiopeia/I911 suite of beamlines, is based on a superconducting wiggler at the MAX II ring of the MAX IV Lab. in Lund, Sweden.  The beamline is energy-tunable within a range between 6 and 18 keV.  I911-3 opened for users in 2005.  In 2010-2011 the exptl. station was completely rebuilt and refurbished such that it has become a state-of-the-art exptl. station with better possibilities for rapid throughput, crystal screening and work with smaller samples.  This paper describes the complete I911-3 beamline and how it is embedded in the Cassiopeia suite of beamlines.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHrhyNC6OdLLVg90H21EOLACvtfcHk0lg5hD2tFZrmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsVenu7Y%253D&md5=4f204e3c3a4f2b8afe560f3e1b0f30c8</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1107%2FS0909049513011734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0909049513011734%26sid%3Dliteratum%253Aachs%26aulast%3DUrsby%26aufirst%3DT.%26aulast%3DUnge%26aufirst%3DJ.%26aulast%3DAppio%26aufirst%3DR.%26aulast%3DLogan%26aufirst%3DD.%2BT.%26aulast%3DFredslund%26aufirst%3DF.%26aulast%3DSvensson%26aufirst%3DC.%26aulast%3DLarsson%26aufirst%3DK.%26aulast%3DLabrador%26aufirst%3DA.%26aulast%3DThunnissen%26aufirst%3DM.%2BM.%2BG.%2BM.%26atitle%3DThe%2520Macromolecular%2520Crystallography%2520Beamline%2520I911%25E2%2580%25933%2520at%2520the%2520MAX%2520IV%2520Laboratory%26jtitle%3DJ.%2520Synchrotron%2520Radiat.%26date%3D2013%26volume%3D20%26spage%3D648%26epage%3D653%26doi%3D10.1107%2FS0909049513011734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lg5hD2tFZrmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Evans, P. R.</span><span> </span><span class="NLM_article-title">An Introduction to Data Reduction: Space-Group Determination, Scaling and Intensity Statistics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">282</span><span class="NLM_x">–</span> <span class="NLM_lpage">292</span><span class="refDoi"> DOI: 10.1107/S090744491003982X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS090744491003982X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=21460446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=282-292&author=P.+R.+Evans&title=An+Introduction+to+Data+Reduction%3A+Space-Group+Determination%2C+Scaling+and+Intensity+Statistics&doi=10.1107%2FS090744491003982X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">An introduction to data reduction: Space-group determination, scaling and intensity statistics</span></div><div class="casAuthors">Evans, Philip R.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">282-292</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  This paper presents an overview of how to run the CCP4 programs for data redn. (SCALA, POINTLESS and CTRUNCATE) through the CCP4 graphical interface ccp4i and points out some issues that need to be considered, together with a few examples.  It covers detn. of the point-group symmetry of the diffraction data (the Laue group), which is required for the subsequent scaling step, examn. of systematic absences, which in many cases will allow inference of the space group, putting multiple data sets on a common indexing system when there are alternatives, the scaling step itself, which produces a large set of data-quality indicators, estn. of |F| from intensity and finally examn. of intensity statistics to detect crystal pathologies such as twinning.  An appendix outlines the scoring schemes used by the program POINTLESS to assign probabilities to possible Laue and space groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEq8RhfQOW8LVg90H21EOLACvtfcHk0lg5hD2tFZrmhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLo%253D&md5=f425388a87744d721d324118d50e4f9a</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS090744491003982X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS090744491003982X%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26atitle%3DAn%2520Introduction%2520to%2520Data%2520Reduction%253A%2520Space-Group%2520Determination%252C%2520Scaling%2520and%2520Intensity%2520Statistics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D282%26epage%3D292%26doi%3D10.1107%2FS090744491003982X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keegan, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krissinel, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leslie, A. G. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNicholas, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pannu, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potterton, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vagin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. S.</span><span> </span><span class="NLM_article-title">Overview of the CCP 4 Suite and Current Developments</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">–</span> <span class="NLM_lpage">242</span><span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP+4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liA4H7AIzQS3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP%25204%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storoni, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span> <span class="citation_source-journal">J. Appl. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">658</span><span class="NLM_x">–</span> <span class="NLM_lpage">674</span><span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lhQoxjZX1cOBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohkamp, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Features and Development of Coot</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">486</span><span class="NLM_x">–</span> <span class="NLM_lpage">501</span><span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lhQoxjZX1cOBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, P. D.</span><span> </span><span class="NLM_article-title">Electronic Ligand Builder and Optimization Workbench (eLBOW): A Tool for Ligand Coordinate and Restraint Generation</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">65</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1080</span><span class="refDoi"> DOI: 10.1107/S0907444909029436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909029436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=19770504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2009&pages=1074-1080&author=N.+W.+Moriartyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adams&title=Electronic+Ligand+Builder+and+Optimization+Workbench+%28eLBOW%29%3A+A+Tool+for+Ligand+Coordinate+and+Restraint+Generation&doi=10.1107%2FS0907444909029436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">electronic Ligand Builder and Optimization Workbench (eLBOW): a tool for ligand coordinate and restraint generation</span></div><div class="casAuthors">Moriarty, Nigel W.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1074-1080</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The electronic Ligand Builder and Optimization Workbench (eLBOW) is a program module of the PHENIX suite of computational crystallog. software.  It is designed to be a flexible procedure that uses simple and fast quantum-chem. techniques to provide chem. accurate information for novel and known ligands alike.  A variety of input formats and options allow the attainment of a no. of diverse goals including geometry optimization and generation of restraints.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojnN4AVY_DwrVg90H21EOLACvtfcHk0ljkSvpYggGhGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFKnsLnN&md5=3f9a9c6c31c9a4b014a9f151fb6aa8d9</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0907444909029436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909029436%26sid%3Dliteratum%253Aachs%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DElectronic%2520Ligand%2520Builder%2520and%2520Optimization%2520Workbench%2520%2528eLBOW%2529%253A%2520A%2520Tool%2520for%2520Ligand%2520Coordinate%2520and%2520Restraint%2520Generation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2009%26volume%3D65%26spage%3D1074%26epage%3D1080%26doi%3D10.1107%2FS0907444909029436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, I. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse-Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">PHENIX: A Comprehensive Python-Based System for Macromolecular Structure Solution</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">213</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+A+Comprehensive+Python-Based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0ljkSvpYggGhGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520A%2520Comprehensive%2520Python-Based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Hayward, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berendsen, H. J. C.</span><span> </span><span class="NLM_article-title">Systematic Analysis of Domain Motions in Proteins from Conformational Change: New Results on Citrate Synthase and T4 Lysozyme</span> <span class="citation_source-journal">Proteins: Struct., Funct., Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">144</span><span class="NLM_x">–</span> <span class="NLM_lpage">154</span><span class="refDoi"> DOI: 10.1002/(SICI)1097-0134(19980201)30:2<144::AID-PROT4>3.0.CO;2-N</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=9489922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;key=1%3ACAS%3A528%3ADyaK1cXnvVaqtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1998&pages=144-154&author=S.+Haywardauthor=H.+J.+C.+Berendsen&title=Systematic+Analysis+of+Domain+Motions+in+Proteins+from+Conformational+Change%3A+New+Results+on+Citrate+Synthase+and+T4+Lysozyme&doi=10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic analysis of domain motions in proteins from conformational change: new results on citrate synthase and T4 lysozyme</span></div><div class="casAuthors">Hayward, Steven; Berendsen, Herman J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">144-154</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Methods developed originally to analyze domain motions from simulation were adapted and extended for the anal. of x-ray conformers and for proteins with >2 domains.  The method can be applied as an automatic procedure to any case where >1 conformation is available.  The basis of the methodol. is that domains can be recognized from the difference in the parameters governing their quasi-rigid body motion, and in particular their rotation vectors.  A clustering algorithm is used to det. clusters of rotation vectors corresponding to main-chain segments that form possible dynamic domains.  Domains are accepted for further anal. on the basis of a ratio of interdomain to intradomain fluctuation, and Chasles' theorem is used to det. interdomain screw axes.  Finally residues involved in the interdomain motion are identified.  The methodol. was tested on citrate synthase and the M6I mutant of phage T4 lysozyme.  In both cases, new aspects of their conformational changes were revealed, as were individual residues intimately involved in their dynamics.  For citrate synthase, the β-sheet was identified to be part of the hinging mechanism.  In the case of phage T4 lysozyme, 1 of the 4 transitions in the pathway from the closed to the open conformation, furnished 4 dynamic domains rather than the expected 2.  This result indicated that the no. of dynamic domains a protein possesses may not be a const. of the motion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWIAHnw5p1LVg90H21EOLACvtfcHk0lioQCa8XgLWCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnvVaqtg%253D%253D&md5=da16e74e98b9d35ca2c6a3fc82848ae2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291097-0134%252819980201%252930%253A2%253C144%253A%253AAID-PROT4%253E3.0.CO%253B2-N%26sid%3Dliteratum%253Aachs%26aulast%3DHayward%26aufirst%3DS.%26aulast%3DBerendsen%26aufirst%3DH.%2BJ.%2BC.%26atitle%3DSystematic%2520Analysis%2520of%2520Domain%2520Motions%2520in%2520Proteins%2520from%2520Conformational%2520Change%253A%2520New%2520Results%2520on%2520Citrate%2520Synthase%2520and%2520T4%2520Lysozyme%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D1998%26volume%3D30%26spage%3D144%26epage%3D154%26doi%3D10.1002%2F%28SICI%291097-0134%2819980201%2930%3A2%3C144%3A%3AAID-PROT4%3E3.0.CO%3B2-N" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yma" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4yma','PDB','4yma'); return false;">PDB: 4yma</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e0x" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e0x','PDB','4e0x'); return false;">PDB: 4e0x</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f36" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2f36','PDB','2f36'); return false;">PDB: 2f36</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1vso" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1vso','PDB','1vso'); return false;">PDB: 1vso</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5m2v" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5m2v','PDB','5m2v'); return false;">PDB: 5m2v</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-NOTES-d246e7013-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i74"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01516">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_22222"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01516">10.1021/acs.jmedchem.6b01516</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01516/suppl_file/jm6b01516_si_001.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01516/suppl_file/jm6b01516_si_001.csv">jm6b01516_si_001.csv (1.74 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The structure coordinates and corresponding structure factor file of GluK1 LBD with <b>1b</b> has been deposited in the Protein Data Bank under the accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5m2v">5m2v</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01516&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01516%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-1%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01516" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679921171f1bd187","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
