
<html lang="en"     class="pb-page"  data-request-id="279bca83-3d93-4d58-b125-7ba793a1ece5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.7b00596;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2017.60.issue-17;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Structure-Guided Optimization of HIV Integrase Strand Transfer Inhibitors" /></meta><meta name="dc.Creator" content="Xue Zhi  Zhao" /></meta><meta name="dc.Creator" content="Steven J.  Smith" /></meta><meta name="dc.Creator" content="Daniel P.  Maskell" /></meta><meta name="dc.Creator" content="Mathieu  Métifiot" /></meta><meta name="dc.Creator" content="Valerie E.  Pye" /></meta><meta name="dc.Creator" content="Katherine  Fesen" /></meta><meta name="dc.Creator" content="Christophe  Marchand" /></meta><meta name="dc.Creator" content="Yves  Pommier" /></meta><meta name="dc.Creator" content="Peter  Cherepanov" /></meta><meta name="dc.Creator" content="Stephen H.  Hughes" /></meta><meta name="dc.Creator" content="Terrence R.  Burke, Jr." /></meta><meta name="dc.Description" content="Integrase mutations can reduce the effectiveness of the first-generation FDA-approved integrase strand transfer inhibitors (INSTIs), raltegravir (RAL) and elvitegravir (EVG). The second-generation ..." /></meta><meta name="Description" content="Integrase mutations can reduce the effectiveness of the first-generation FDA-approved integrase strand transfer inhibitors (INSTIs), raltegravir (RAL) and elvitegravir (EVG). The second-generation ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="August 10, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00596" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta><meta name="citation_fulltext_world_readable" content=""/></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00596" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00596" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00596" /></link>
        
    
    

<title>Structure-Guided Optimization of HIV Integrase Strand Transfer Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00596" /></meta><meta property="og:title" content="Structure-Guided Optimization of HIV Integrase Strand Transfer Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0006.jpeg" /></meta><meta property="og:description" content="Integrase mutations can reduce the effectiveness of the first-generation FDA-approved integrase strand transfer inhibitors (INSTIs), raltegravir (RAL) and elvitegravir (EVG). The second-generation agent, dolutegravir (DTG), has enjoyed considerable clinical success; however, resistance-causing mutations that diminish the efficacy of DTG have appeared. Our current findings support and extend the substrate envelope concept that broadly effective INSTIs can be designed by filling the envelope defined by the DNA substrates. Previously, we explored 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamides as an INSTI scaffold, making a limited set of derivatives, and concluded that broadly effective INSTIs can be developed using this scaffold. Herein, we report an extended investigation of 6-substituents as well the first examples of 7-substituted analogues of this scaffold. While 7-substituents are not well-tolerated, we have identified novel substituents at the 6-position that are highly effective, with the best compound (6p) retaining better efficacy against a broad panel of known INSTI resistant mutants than any analogues we have previously described." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00596"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00596">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00596&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00596&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00596&amp;href=/doi/10.1021/acs.jmedchem.7b00596" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 17</span><span class="cit-fg-pageRange">, 7315-7332</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/17" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00480" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00606" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Optimization of HIV Integrase Strand Transfer Inhibitors</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xue+Zhi++Zhao">Xue Zhi Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Steven+J.++Smith">Steven J. Smith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+P.++Maskell">Daniel P. Maskell</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mathieu++M%C3%A9tifiot">Mathieu Métifiot</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Valerie+E.++Pye">Valerie E. Pye</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Katherine++Fesen">Katherine Fesen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christophe++Marchand">Christophe Marchand</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yves++Pommier">Yves Pommier</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Cherepanov">Peter Cherepanov</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stephen+H.++Hughes">Stephen H. Hughes</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Terrence+R.++Burke%2C++Jr.">Terrence R. Burke, Jr.</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-9925-8586" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Chemical Biology Laboratory and <sup>‡</sup>HIV Dynamics and Replication Program, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Frederick, Maryland 21702, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">§</span> <span class="aff-text">Developmental Therapeutics Branch and Laboratory of Molecular Pharmacology, Center for Cancer Research, National Cancer Institute, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">∥</span> <span class="aff-text">Chromatin Structure and Mobile DNA, The Francis Crick Institute, London NW1 1AT, United Kingdom</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">⊥</span> <span class="aff-text">Imperial College London, St-Mary’s Campus, Norfolk Place, London W2 1PG, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>Phone: 301-846-5906. Fax: 301-846-6033. E-mail: <a href="/cdn-cgi/l/email-protection#147661667f716071547c71787d6c3a7a7d7c3a737b62"><span class="__cf_email__" data-cfemail="d1b3a4a3bab4a5b491b9b4bdb8a9ffbfb8b9ffb6bea7">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00596&amp;href=/doi/10.1021%2Facs.jmedchem.7b00596" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 17</span><span class="cit-pageRange">, 7315–7332</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 24, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 April 2017</li><li><span class="item_label"><b>Published</b> online</span>10 August 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 September 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00596" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00596</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/page/rightslinkno.jsp">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"><a href="/page/policy/authorchoice_termsofuse.html"><img src="/pb-assets/images/programsLogos/OA_Icons_Ac-1612971054130.svg" alt="ACS AuthorChoice" /></img><span>ACS AuthorChoice</span></a></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00596"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2888</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">21</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00596" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Optimization of HIV Integrase Strand Transfer Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xue&quot;,&quot;last_name&quot;:&quot;Zhi Zhao&quot;},{&quot;first_name&quot;:&quot;Steven&quot;,&quot;last_name&quot;:&quot;J. Smith&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;P. Maskell&quot;},{&quot;first_name&quot;:&quot;Mathieu&quot;,&quot;last_name&quot;:&quot;Métifiot&quot;},{&quot;first_name&quot;:&quot;Valerie&quot;,&quot;last_name&quot;:&quot;E. Pye&quot;},{&quot;first_name&quot;:&quot;Katherine&quot;,&quot;last_name&quot;:&quot;Fesen&quot;},{&quot;first_name&quot;:&quot;Christophe&quot;,&quot;last_name&quot;:&quot;Marchand&quot;},{&quot;first_name&quot;:&quot;Yves&quot;,&quot;last_name&quot;:&quot;Pommier&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Cherepanov&quot;},{&quot;first_name&quot;:&quot;Stephen&quot;,&quot;last_name&quot;:&quot;H. Hughes&quot;},{&quot;first_name&quot;:&quot;Terrence&quot;,&quot;last_name&quot;:&quot;R. Burke&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;08&quot;,&quot;day&quot;:&quot;10&quot;,&quot;issue&quot;:&quot;17&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;7315-7332&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00596&quot;},&quot;abstract&quot;:&quot;Integrase mutations can reduce the effectiveness of the first-generation FDA-approved integrase strand transfer inhibitors (INSTIs), raltegravir (RAL) and elvitegravir (EVG). The second-generation agent, dolutegravir (DTG), has enjoyed considerable clinical success; however, resistance-causing mutations that diminish the efficacy of DTG have appeared. Our current findings support and extend the substrate envelope concept that broadly effective INSTIs can be designed by filling the envelope defined by the DNA substrates. Previously, we explored 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamides as an INSTI scaffold, making a limited set of derivatives, and concluded that broadly effective INSTIs can be developed using this scaffold. Herein, we report an extended investigation of 6-substituents as well the first examples of 7-substituted analogues of this scaffold. While 7-substituents are not well-tolerated, we have identified novel substituents at the 6-position that are highly effective, with the b&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00596&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00596" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00596&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00596" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00596&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00596" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00596&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00596&amp;href=/doi/10.1021/acs.jmedchem.7b00596" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00596" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00596" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00596%26sid%3Dliteratum%253Aachs%26pmid%3D28737946%26genre%3Darticle%26aulast%3DZhao%26date%3D2017%26atitle%3DStructure-Guided%2BOptimization%2Bof%2BHIV%2BIntegrase%2BStrand%2BTransfer%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D17%26spage%3D7315%26epage%3D7332%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291137" title="Lipids">Lipids</a>,</li><li><a href="/action/doSearch?ConceptID=291362" title="Antimicrobial agents">Antimicrobial agents</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292349" title="Alkyls">Alkyls</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/17" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/jmcmar.2017.60.issue-17/20170914/jmcmar.2017.60.issue-17.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Integrase mutations can reduce the effectiveness of the first-generation FDA-approved integrase strand transfer inhibitors (INSTIs), raltegravir (RAL) and elvitegravir (EVG). The second-generation agent, dolutegravir (DTG), has enjoyed considerable clinical success; however, resistance-causing mutations that diminish the efficacy of DTG have appeared. Our current findings support and extend the substrate envelope concept that broadly effective INSTIs can be designed by filling the envelope defined by the DNA substrates. Previously, we explored 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamides as an INSTI scaffold, making a limited set of derivatives, and concluded that broadly effective INSTIs can be developed using this scaffold. Herein, we report an extended investigation of 6-substituents as well the first examples of 7-substituted analogues of this scaffold. While 7-substituents are not well-tolerated, we have identified novel substituents at the 6-position that are highly effective, with the best compound (<b>6p</b>) retaining better efficacy against a broad panel of known INSTI resistant mutants than any analogues we have previously described.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27630" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27630" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">HIV-1 integrase (IN) plays a key role in the viral life cycle, inserting the double-stranded DNA that is generated by reverse transcription of the viral RNA genome into the genome of the host cell.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Integration is essential for viral replication, and for this reason, IN is a therapeutic target for the treatment of HIV infections. To date, three HIV IN antagonists have been approved for clinical use: raltegravir (RAL, <b>1</b>), elvitegravir (EVG, <b>2</b>), and dolutegravir (DTG, <b>3</b>) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2-4)</a> These drugs belong to a class of compounds called integrase strand transfer inhibitors (INSTIs) because they inhibit DNA strand transfer (ST), the second step of integration catalyzed by IN, rather than the first step, the 3′-processing reaction (<b>3′</b>-P).<a onclick="showRef(event, 'ref5 ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref5 ref6 ref7 ref8">(5-8)</a> Development of drug resistance mutations is a common problem in antiviral therapy and, not surprisingly, mutations affecting the susceptibility of the virus to RAL and EVG have rapidly emerged.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9-11)</a> However, the second-generation inhibitor, DTG, retains potency against some but not all RAL/EVG resistant HIV variants.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15 ref16">(12-16)</a> Therefore, the development of new small molecules that have minimal toxicity and improved efficacy against the existing resistant mutants remains an important research objective.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HIV-1 integrase inhibitors. Colored areas indicate regions of intended correspondence.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Retroviral integration is mediated by IN multimers that are assembled on the viral DNA ends, forming a stable synaptic complex, also referred to as the intasome.<a onclick="showRef(event, 'ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20 ref21">(18-21)</a> The INSTIs only bind to the active site of IN when the processed viral DNA ends are appropriately bound to the intasome.<a onclick="showRef(event, 'ref8 ref22'); return false;" href="javascript:void(0);" class="ref ref8 ref22">(8, 22)</a> The way in which INSTIs bind to the intasome was elucidated by solving crystal structures of the orthologous retroviral IN from the prototype foamy virus (PFV).<a onclick="showRef(event, 'ref19 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref19 ref23 ref24">(19, 23, 24)</a> The INSTIs are “interfacial” inhibitors; they bind to the active site of IN and interact with the bound viral DNA following the 3′-processing step.<a onclick="showRef(event, 'ref8 ref19 ref25'); return false;" href="javascript:void(0);" class="ref ref8 ref19 ref25">(8, 19, 25)</a> Essential structural features that contribute to the binding of INSTIs include an array of three heteroatoms (highlighted in red, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) that chelate the two catalytic Mg<sup>2+</sup> ions in the IN active site and a halobenzyl side chain (halophenyl portion highlighted in blue, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) that stacks with the penultimate nucleotide (a deoxycytidine) at the 3′ end of the viral DNA.<a onclick="showRef(event, 'ref8 ref19'); return false;" href="javascript:void(0);" class="ref ref8 ref19">(8, 19)</a> We have recently shown that the 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide motif (<b>4</b>) can serve a useful platform for developing HIV-1 IN inhibitors that retain high efficacy against the RAL/EVG-resistant mutants.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> We initially examined the properties of a series of analogues related to structure <b>4</b> by varying the substituents at the 4-position. Our objective was to identify compounds that retain efficacy against the mutations Y143R, N155H, and Q148H/G140S, which have been associated with clinical resistance to RAL,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and some of these mutations also play a role in the development of resistance against DTG.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> This approach yielded compounds including <b>4a</b>–<b>d</b>, which are approximately equivalent to RAL in their potency against recombinant wild-type (WT) HIV-1 IN in biochemical assays. However, the small molecules also showed improved antiviral efficacies against the Y143R and N155H mutants in cell-based assays.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> Although antiviral efficacies against the Q148H/G140S double mutant were also improved relative to RAL, the new compounds were inferior to DTG, prompting us to continue our developmental efforts.</div><div class="NLM_p last">Structural studies using the PFV intasome have revealed that the tricyclic system of DTG is sufficiently extended to make contacts with G187 in the β4−α2 loop of PFV IN (G118 in IN).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> It has been argued that the interactions with this region may contribute to the improved properties of DTG and other second-generation INSTIs.<a onclick="showRef(event, 'ref4 ref23 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref4 ref23 ref29 ref30">(4, 23, 29, 30)</a> Therefore, we considered that adding functionality to either the 6- or 7-positions of <b>4</b> could interact with the same region of the catalytic site (highlighted in green and cyan, respectively, in the structures of DTG and <b>4</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). In a preliminary work, we modified the 6-position of <b>4</b> and showed that adding linear side chains bearing terminal hydroxyl groups can improve antiviral efficacies against the Q148H/G140S double mutant to levels approaching that of DTG.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Furthermore, depending on the 6-substituent, compounds could retain essentially all of their antiviral potency against a more extensive panel of HIV-1-based vectors that carry the major DTG-resistant IN mutants, including the G118R, T66I, E92Q, R263K, and H51Y single mutants and the H51Y/R263K double mutant.<a onclick="showRef(event, 'ref17 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref17 ref32 ref33 ref34">(17, 32-34)</a> These data have two important implications: First, 6-substituents can have an important role in maintaining antiviral efficacy against resistant mutant forms of IN. Second, compounds that are broadly effective against mutant forms of IN bind in ways that involve substrate mimicry and, when bound, fit within the “substrate envelope”.<a onclick="showRef(event, 'ref31 ref35'); return false;" href="javascript:void(0);" class="ref ref31 ref35">(31, 35)</a> However, the data were confined to a small number of 6-modificaitons that had limited chemical diversity. Given the potential promise of 6-substituents, we felt that it was important to more thoroughly examine this position and, accordingly, we describe here an extensive set of 6-modifications (designated as the <b>6</b> series analogues, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). We were also interested in examining substituents at the 7-position of <b>4</b> because these could potentially afford an alternate way to engage G118 in the β4−α2 loop of IN. We present the first examination of compounds that have modifications at the 7-position (designated as the <b>5</b> series analogues). To provide a more complete SAR study, we have also introduced various functionalities at the 4-position in combination with various 6- and 7-substitutents. These efforts have resulted in one of the most potent and effective compounds that we have developed to date. This work sheds additional light on enzyme–inhibitor interactions that can contribute to retention of efficacy against strains of virus that contain mutant forms of IN resistant to the approved INSTIs.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Chemistry</h3><div class="NLM_p">To prepare our 7-substituted analogues <b>5a</b>–<b>i</b>, we employed 2,6-dichloronicotinic acid (<b>7</b>) as the starting material. The 6-chloro group of <b>7</b> was converted into the final 7-substituent in two ways. In one route, the 6-chloro moiety was displaced with methoxyl (KO<sup><i>t</i></sup>Bu in MeOH),<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and this was followed by esterification of the 3-carboxy group to yield <b>8</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). In the second route, the 3-carboxyl group of <b>7</b> was first converted directly to the 3-methyl ester. In both cases, the 2-chloro group was then displaced with BnONH<sub>2</sub> to give the corresponding 2-benzyloxyamine-containing derivatives (<b>10a</b> and <b>9</b>, respectively). The 6-chloro-substituents of the second series <b>9</b> were transformed into the corresponding 6-piperidinyl and 6-morpholino derivatives (<b>10b</b> and <b>10c</b>, respectively) by heating with DMF solutions of piperidine or morpholine. Using a procedure similar to those we previously reported (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>),<a onclick="showRef(event, 'ref26 ref27 ref31'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref31">(26, 27, 31)</a> intermediates <b>10a</b>–<b>c</b> were transformed to the 4-hydroxy (<b>5a</b>–<b>c</b>), 4-amino (<b>5d</b>–<b>f</b>), and 4-[2-(methyl glycinate)] (<b>5g</b>–<b>i</b>) final products.</div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0004.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4,7-Bis-Substituted Analogues <b>5a–5i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) KO<sup><i>t</i></sup>Bu, MeOH, 65 °C; (ii) H<sub>2</sub>SO<sub>4</sub>, MeOH; (iii) BnONH<sub>2</sub>, DIEA, dioxane, 110 °C; (iv) (b) piperidine or (c) morpholine, DMF, 80 °C; (v) ClCOCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub>, TEA, DCM; (vi) NaOMe, MeOH; (vii) 2,4-diF-BnNH<sub>2</sub>, DMF, 140 °C; (viii) TsCl, DIEA, MeCN; (ix) RNH<sub>2</sub>, DIEA, DMF, 50 °C, R = 2,4-diMeOBn (a) or CH<sub>2</sub>CO<sub>2</sub>Me (b); (x) TFA, DCM; (xi) H<sub>2</sub>, 10% Pd/C, MeOH.</p></p></figure><div class="NLM_p">We transformed the 6-bromo analogues (<b>17a</b>–<b>c</b>, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) into a variety of 6-substituted INSTIs by coupling alkynes and alkenes using Sonogashira and Heck coupling chemistries.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> In addition to the 6-substituted products with a 4-hydroxyl (<b>6a</b>), we prepared analogues having a free amine at the 4-position (<b>6b</b>–<b>p</b>) as well as analogues with 4-[2-(methyl glycinate)]amine (<b>6q</b>) and 4-(2-hydroxyethyl)amine (<b>6r</b>–<b>t</b>) (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0005.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4,6-Bis-Substituted Analogues <b>6a</b>–<b>6t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) TsCl, Et<sub>3</sub>N, CH<sub>3</sub>CN; (ii) RNH<sub>2</sub>, DIEA, DMF, 50 °C, R = 2,4-diMeOBn (DMP) (a), CH<sub>2</sub>CH<sub>2</sub>OH (b), or CH<sub>2</sub>CO<sub>2</sub>Me (c); (iii) alkyne, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, DIEA, CuI, DMF, 70 °C or alkene, Pd(OAc)<sub>2</sub>, TEA, PPh<sub>3</sub>, DMF, 100 °C; (iv) TFA, DCM; (v) H<sub>2</sub>, 10% Pd/C, MeOH.</p></p></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Evaluation of New Derivatives Using in Vitro IN Catalytic Assays and Single-Round Virus Replication Assays</h3><div class="NLM_p">We designed compounds <b>5a</b>–<b>i</b> to examine the effects of substituents at the 4-position (hydroxyl, amine, or methyl glycinate) in combination with modifications of the 7-position (methoxyl, piperidinyl, or morpholino). We evaluated the compounds by in vitro IN activity assays that utilize <sup>32</sup>P-radiolabeled oligonucleotides (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> We also tested the potency of the compounds in single-round replication assays that employ HIV-1 vectors carrying WT IN. We evaluated the antiviral potencies of a subset of the compounds against HIV-1 vectors carrying the canonical Y143R, N155H, and Q148H/G140S mutations that are associated with high-level HIV-1 resistance to RAL and virological failure in patients (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).<a onclick="showRef(event, 'ref17 ref32 ref33 ref34 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref17 ref32 ref33 ref34 ref38 ref39">(17, 32-34, 38, 39)</a> As we recently reported, compounds that lack substituents at the 6- and 7-positions and have a hydroxyl group at the 4-position (<b>5′a</b>) showed lower potencies in the ST assay and poorer antiviral potencies than the corresponding 4-amino-containing compound (<b>5′d</b>) (<a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>).<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> In our current work, we found that a compound with a 4-[2-(methyl glycinate)] modification of the 4-amino group (<b>5′g</b>) retained its enzymatic potency in these assays relative to <b>5′d</b>. Bulky substituents at the 7-position (<b>5b</b>, <b>5c</b>, <b>5e</b>, <b>5f</b>, <b>5h</b>. and <b>5i</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) diminished potency by three orders of magnitude relative to the corresponding unsubsituted analogues (<b>5′a</b>, <b>5′d</b>, and <b>5′g</b>, respectively). Although a 7-methoxyl group was better tolerated (compare with <b>5a</b>, <b>5d</b>, and <b>5g</b>), in all cases there was a reduction in the antiviral potency (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibitory Potencies of Compounds Using an in Vitro IN Assay<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0010.gif" alt="" id="GRAPHIC-d239e834-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">in vitro (IC<sub>50</sub> μM)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">R<sup>4</sup></th><th class="colsep0 rowsep0" align="center">R<sup>6</sup></th><th class="colsep0 rowsep0" align="center">R<sup>7</sup></th><th class="colsep0 rowsep0" align="center">3′-P</th><th class="colsep0 rowsep0" align="center">ST</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5′a</b><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">1.2 ± 0.2</td><td class="colsep0 rowsep0" align="center">0.055 ± 0.008</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="center">32 ± 4</td><td class="colsep0 rowsep0" align="center">0.033 ± 0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">piperidinyl</td><td class="colsep0 rowsep0" align="center">>333</td><td class="colsep0 rowsep0" align="center">70 ± 16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5c</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">morpholino</td><td class="colsep0 rowsep0" align="center">>333</td><td class="colsep0 rowsep0" align="center">1.9 ± 0.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5′d</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">2.5 ± 0.3</td><td class="colsep0 rowsep0" align="center">0.019 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5d</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="center">18 ± 2</td><td class="colsep0 rowsep0" align="center">0.011 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5e</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">piperidinyl</td><td class="colsep0 rowsep0" align="center">>333</td><td class="colsep0 rowsep0" align="center">54 ± 18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5f</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">morpholino</td><td class="colsep0 rowsep0" align="center">>333</td><td class="colsep0 rowsep0" align="center">54 ± 10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5′g</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">NHCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">0.71 ± 0.10</td><td class="colsep0 rowsep0" align="center">0.021 ± 0.011</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5g</b></td><td class="colsep0 rowsep0" align="left">NHCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="center">20 ± 2</td><td class="colsep0 rowsep0" align="center">0.042 ± 0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5h</b></td><td class="colsep0 rowsep0" align="left">NHCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">piperidinyl</td><td class="colsep0 rowsep0" align="center">>333</td><td class="colsep0 rowsep0" align="center">96 ± 40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5i</b></td><td class="colsep0 rowsep0" align="left">NHCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">morpholino</td><td class="colsep0 rowsep0" align="center">>333</td><td class="colsep0 rowsep0" align="center">6.2 ± 0.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>5</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">14.3 ± 1.1</td><td class="colsep0 rowsep0" align="center">0.012 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6′b</b><a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>5</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">8.2 ± 1.2</td><td class="colsep0 rowsep0" align="center">0.0027 ± 0.0004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>3</sub>OAc</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">14.3 ± 2.1</td><td class="colsep0 rowsep0" align="center">0.010 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>6</sub>OBz</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">143 ± 14</td><td class="colsep0 rowsep0" align="center">3.3 ± 0.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>3</sub><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a>Hex</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">289 ± 49</td><td class="colsep0 rowsep0" align="center">3.3 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>4</sub>Ph</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">42 ± 4</td><td class="colsep0 rowsep0" align="center">2.0 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6f</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>Ph</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">48 ± 4</td><td class="colsep0 rowsep0" align="center">0.014 ± 0.003</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>3</sub>N(CH<sub>3</sub>)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">3.0 ± 0.4</td><td class="colsep0 rowsep0" align="center">0.014 ± 0.002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6h</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>3</sub>O(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">1.5 ± 0.4</td><td class="colsep0 rowsep0" align="center">0.0020 ± 0.0004</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6i</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CO<sub>2</sub>H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND<a class="ref internalNav" href="#t1fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="center">0.004 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6j</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CON(CH<sub>3</sub>)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">0.008 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6k</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH)<sub>2</sub>CON(CH<sub>3</sub>)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">0.005 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CONH<sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">0.002 ± 0.007</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6m</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH)<sub>2</sub>CONH<sup><i>i</i></sup>Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">0.006 ± 0.005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6n</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CONH(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">0.002 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6o</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH)<sub>2</sub>CONH(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">0.007 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6p</b></td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">2.9 ± 0.6</td><td class="colsep0 rowsep0" align="center">0.0031 ± 0.0005</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6q</b></td><td class="colsep0 rowsep0" align="left">NHCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>5</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">2.1 ± 0.5</td><td class="colsep0 rowsep0" align="center">0.004 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6r</b></td><td class="colsep0 rowsep0" align="left">NH(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>5</sub>OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">4.0 ± 0.5</td><td class="colsep0 rowsep0" align="center">0.0025 ± 0.0002</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6s</b></td><td class="colsep0 rowsep0" align="left">NH(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>5</sub>OAc</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6t</b></td><td class="colsep0 rowsep0" align="left">NH(CH<sub>2</sub>)<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">(CH<sub>2</sub>)<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="center">3.2 ± 0.3</td><td class="colsep0 rowsep0" align="center">0.0031 ± 0.0003</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">Assays were performed using a gel-based protocol with a Mg<sup>2+</sup> cofactor as described.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">Data have been reported previously.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Data have been reported previously.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">Data have been reported previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></p></div><div class="footnote" id="t1fn5"><sup>e</sup><p class="last">Not determined.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Antiviral Potencies in Cells Infected with HIV-1 Vectors That Carry WT or Resistant IN Mutants<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char="±">EC<sub>50</sub> (nM/FC<a class="ref internalNav" href="#t2fn5" aria-label="e">e</a>)</th><th class="colsep0 rowsep0" align="center" char="±"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center" char="±">WT</th><th class="colsep0 rowsep0" align="center">Y143R</th><th class="colsep0 rowsep0" align="center">N155H</th><th class="colsep0 rowsep0" align="center">G140S/Q148H</th><th class="colsep0 rowsep0" align="center" char="±">CC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>RAL</b> (<b>1</b>)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">4 ± 2</td><td class="colsep0 rowsep0" align="center">162 ± 16 (41×)</td><td class="colsep0 rowsep0" align="center">154 ± 33 (39×)</td><td class="colsep0 rowsep0" align="center">1900 ± 300 (475×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>EVG</b> (<b>2</b>)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">6.4 ± 0.8</td><td class="colsep0 rowsep0" align="center">7.9 ± 2.3 (1×)</td><td class="colsep0 rowsep0" align="center">90 ± 18 (14×)</td><td class="colsep0 rowsep0" align="center">5700 ± 1100 (891×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DTG</b> (<b>3</b>)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">1.6 ± 0.9</td><td class="colsep0 rowsep0" align="center">4.3 ± 1.2 (3×)</td><td class="colsep0 rowsep0" align="center">3.6 ± 1.3 (2×)</td><td class="colsep0 rowsep0" align="center">5.8 ± 0.5 (4×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5′a</b><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">6.2 ± 2.9</td><td class="colsep0 rowsep0" align="center">11 ± 2 (2×)</td><td class="colsep0 rowsep0" align="center">31 ± 8 (5×)</td><td class="colsep0 rowsep0" align="center">308 ± 125 (50×)</td><td class="colsep0 rowsep0" align="char" char="±">137 ± 20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b></td><td class="colsep0 rowsep0" align="char" char="±">215 ± 31</td><td class="colsep0 rowsep0" align="center">ND<a class="ref internalNav" href="#t2fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char="±">12 ± 5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5′d</b><a class="ref internalNav" href="#t2fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="char" char="±">1.1 ± 0.7</td><td class="colsep0 rowsep0" align="center">2.5 ± 0.6 (2×)</td><td class="colsep0 rowsep0" align="center">5.3 ± 2.3 (5×)</td><td class="colsep0 rowsep0" align="center">35 ± 9 (32×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5d</b></td><td class="colsep0 rowsep0" align="char" char="±">30 ± 9</td><td class="colsep0 rowsep0" align="center">62 ± 26 (2×)</td><td class="colsep0 rowsep0" align="center">400 ± 101 (13×)</td><td class="colsep0 rowsep0" align="center">3600 ± 1600 (120×)</td><td class="colsep0 rowsep0" align="char" char="±">2.2 ± 0.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5′g</b></td><td class="colsep0 rowsep0" align="char" char="±">3.8 ± 1.2</td><td class="colsep0 rowsep0" align="center">4.6 ± 2.2 (1×)</td><td class="colsep0 rowsep0" align="center">19 ± 7 (5×)</td><td class="colsep0 rowsep0" align="center">36 ± 16 (9×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5g</b></td><td class="colsep0 rowsep0" align="char" char="±">32 ± 13</td><td class="colsep0 rowsep0" align="center">52 ± 13 (2×)</td><td class="colsep0 rowsep0" align="center">1260 ± 320 (39×)</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char="±">12 ± 3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6a</b></td><td class="colsep0 rowsep0" align="char" char="±">24 ± 4</td><td class="colsep0 rowsep0" align="center">8.3 ± 1.7 (0.3×)</td><td class="colsep0 rowsep0" align="center">32 ± 3 (1×)</td><td class="colsep0 rowsep0" align="center">29 ± 10 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6′b</b><a class="ref internalNav" href="#t2fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="±">1.3 ± 0.2</td><td class="colsep0 rowsep0" align="center">3.0 ± 0.5 (2×)</td><td class="colsep0 rowsep0" align="center">2.4 ± 0.8 (2×)</td><td class="colsep0 rowsep0" align="center">9.4 ± 3.6 (7×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.4</td><td class="colsep0 rowsep0" align="center">2.9 ± 0.9 (2×)</td><td class="colsep0 rowsep0" align="center">5.4 ± 2.5 (4×)</td><td class="colsep0 rowsep0" align="center">13 ± 8 (9×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6c</b></td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 1.6</td><td class="colsep0 rowsep0" align="center">3.5 ± 1.2 (0.7×)</td><td class="colsep0 rowsep0" align="center">5.4 ± 1.9 (1×)</td><td class="colsep0 rowsep0" align="center">21 ± 7.8 (4×)</td><td class="colsep0 rowsep0" align="char" char="±">14.4 ± 4.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6d</b></td><td class="colsep0 rowsep0" align="char" char="±">53 ± 14</td><td class="colsep0 rowsep0" align="center">112 ± 14 (2×)</td><td class="colsep0 rowsep0" align="center">146 ± 37 (3×)</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char="±">27.4 ± 3.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6e</b></td><td class="colsep0 rowsep0" align="char" char="±">11 ± 2</td><td class="colsep0 rowsep0" align="center">4.7 ± 0 (0.4×)</td><td class="colsep0 rowsep0" align="center">39 ± 10 (4×)</td><td class="colsep0 rowsep0" align="center">128 ± 44 (12×)</td><td class="colsep0 rowsep0" align="char" char="±">13 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6f</b></td><td class="colsep0 rowsep0" align="char" char="±">5.6 ± 1.9</td><td class="colsep0 rowsep0" align="center">5.5 ± 2.4 (1×)</td><td class="colsep0 rowsep0" align="center">21 ± 5 (4×)</td><td class="colsep0 rowsep0" align="center">140 ± 34 (25×)</td><td class="colsep0 rowsep0" align="char" char="±">18 ± 2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6g</b></td><td class="colsep0 rowsep0" align="char" char="±">6.1 ± 2.1</td><td class="colsep0 rowsep0" align="center">7.0 ± 2.3 (1×)</td><td class="colsep0 rowsep0" align="center">29 ± 9 (5×)</td><td class="colsep0 rowsep0" align="center">32 ± 5 (5×)</td><td class="colsep0 rowsep0" align="char" char="±">5.0 ± 1.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6h</b></td><td class="colsep0 rowsep0" align="char" char="±">1.6 ± 0.2</td><td class="colsep0 rowsep0" align="center">1.2 ± 0.6 (0.8×)</td><td class="colsep0 rowsep0" align="center">4.2 ± 1.5 (3×)</td><td class="colsep0 rowsep0" align="center">35 ± 3.7 (22×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6i</b></td><td class="colsep0 rowsep0" align="char" char="±">1.9 ± 0.5</td><td class="colsep0 rowsep0" align="center">4.9 ± 1.5 (3×)</td><td class="colsep0 rowsep0" align="center">10 ± 3 (5×)</td><td class="colsep0 rowsep0" align="center">195 ± 25 (103×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6j</b></td><td class="colsep0 rowsep0" align="char" char="±">1.3 ± 0.5</td><td class="colsep0 rowsep0" align="center">6 ± 1 (5×)</td><td class="colsep0 rowsep0" align="center">6.8 ± 1.6 (5×)</td><td class="colsep0 rowsep0" align="center">16 ± 5 (12×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6k</b></td><td class="colsep0 rowsep0" align="char" char="±">1.6 ± 0.9</td><td class="colsep0 rowsep0" align="center">7.2 ± 0.3 (5×)</td><td class="colsep0 rowsep0" align="center">15 ± 2 (9×)</td><td class="colsep0 rowsep0" align="center">58 ± 19 (36×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6l</b></td><td class="colsep0 rowsep0" align="char" char="±">5 ± 1.3</td><td class="colsep0 rowsep0" align="center">8.7 ± 2.2 (2×)</td><td class="colsep0 rowsep0" align="center">11 ± 4 (2×)</td><td class="colsep0 rowsep0" align="center">13 ± 1 (3×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6m</b></td><td class="colsep0 rowsep0" align="char" char="±">2.1 ± 0.7</td><td class="colsep0 rowsep0" align="center">11 ± 1 (5×)</td><td class="colsep0 rowsep0" align="center">12 ± 3 (6×)</td><td class="colsep0 rowsep0" align="center">286 ± 8 (136×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6n</b></td><td class="colsep0 rowsep0" align="char" char="±">263 ± 52</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6o</b></td><td class="colsep0 rowsep0" align="char" char="±">196 ± 32</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="center">ND</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6p</b></td><td class="colsep0 rowsep0" align="char" char="±">0.67 ± 0.15</td><td class="colsep0 rowsep0" align="center">0.67 ± 0.23 (1×)</td><td class="colsep0 rowsep0" align="center">2.3 ± 0.2 (3×)</td><td class="colsep0 rowsep0" align="center">5.3 ± 1.8 (8×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6q</b></td><td class="colsep0 rowsep0" align="char" char="±">3.3 ± 1.7</td><td class="colsep0 rowsep0" align="center">3.5 ± 1.6 (1×)</td><td class="colsep0 rowsep0" align="center">11 ± 2 (3×)</td><td class="colsep0 rowsep0" align="center">42 ± 2 (13×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6r</b></td><td class="colsep0 rowsep0" align="char" char="±">13 ± 4.2</td><td class="colsep0 rowsep0" align="center">12 ± 2 (0.9×)</td><td class="colsep0 rowsep0" align="center">15 ± 4 (1×)</td><td class="colsep0 rowsep0" align="center">16 ± 7 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6s</b></td><td class="colsep0 rowsep0" align="char" char="±">5.3 ± 1.2</td><td class="colsep0 rowsep0" align="center">4.1 ± 1 (0.8×)</td><td class="colsep0 rowsep0" align="center">10 ± 2 (2×)</td><td class="colsep0 rowsep0" align="center">5.3 ± 2.1 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6t</b></td><td class="colsep0 rowsep0" align="char" char="±">4.1 ± 1</td><td class="colsep0 rowsep0" align="center">3.8 ± 1.9 (0.9×)</td><td class="colsep0 rowsep0" align="center">3.9 ± 2.1 (1×)</td><td class="colsep0 rowsep0" align="center">16 ± 4 (4×)</td><td class="colsep0 rowsep0" align="char" char="±">>250</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Cytotoxic concentration resulting in 50% reduction in the level of ATP in human osteosarcoma (HOS) cells; EC<sub>50</sub> values obtained from cells infected with lentiviral vector harboring WT or indicated IN mutants as previously reported.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Data has been reported previously.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a></p></div><div class="footnote" id="t2fn3"><sup>c</sup><p class="last">Data have been reported previously.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></p></div><div class="footnote" id="t2fn4"><sup>d</sup><p class="last">Data have been reported previously.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></p></div><div class="footnote" id="t2fn5"><sup>e</sup><p class="last">FC, fold change relative to WT = EC<sub>50</sub> of mutants/EC<sub>50</sub> of WT.</p></div><div class="footnote" id="t2fn6"><sup>f</sup><p class="last">Not determined.</p></div></div></div><div class="NLM_p">We recently described a series of structurally related 4-amino-containing INSTIs, which possess 6-substituents consisting of primary hydroxyl or sulfone groups tethered by alkyl chains of various lengths.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The chemical diversity of these substituents was limited. To extend the diversity of the modifications at the 6-position, our current study examines analogues containing tethered acetoxy (<b>6b</b>), benzyloxy (<b>6c</b>), cyclohexyl (<b>6d</b>), phenyl (<b>6e</b> and <b>6f</b>), dimethylamino (<b>6g</b>), carboxyl (<b>6i</b>), methyl ester (<b>6p</b>), and carboxamido having a variety of alkylamide groups (<b>6j</b>–<b>6o</b>) groups (<a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). Several of the resulting compounds retained low-nanomolar IC<sub>50</sub> values in ST reactions. <i>N</i>,<i>N</i>-Dimethylpropylamine (<b>6g</b>, ST IC<sub>50</sub> = 14 nM), propanoic acid (<b>6i</b>), propanoic amides (<b>6j–o</b>), and propanoic acid methyl ester (<b>6p</b>) as well as the reverse ester <b>6b</b> displayed IC<sub>50</sub> values of 10 nM or less in the ST assay. Tethering a phenyl ring with an ethylene chain maintained good inhibitory potency (<b>6f</b>, ST IC<sub>50</sub> = 14 nM). However, appending phenyl or cyclohexyl groups by increasing spacer lengths reduced potency by as much as three orders of magnitude: three methylenes (cyclohexyl-containing <b>6d</b>, ST IC<sub>50</sub> = 3.3 μM), four methylenes (phenyl-containing <b>6e</b>, ST IC<sub>50</sub> = 2.0 μM), or longer (<b>6c</b>, ST IC<sub>50</sub> = 3.3 μM) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). This result contrasts with our previous finding that appending a phenyl ring using a three-unit chain that contains a sulfone moiety can result in retention of high potency. In the latter case, a cocrystal structure with the PFV intasome revealed that the sulfone group causes the phenyl ring to adopt an unusual π–π stacking orientation, in which it is folded up under the naphthyridine ring system and fills the catalytic space more completely.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a></div><div class="NLM_p">Most 6-substituted analogues in our current study showed low nanomolar potencies against the WT enzyme in the in vitro ST assay and against a WT HIV-1 vector in the single-round infectivity assay (<a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>, respectively). However, for the 6-tethered 2-hydroxyethylamides (<b>6n</b> and <b>6o</b>), in vitro ST inhibitory potencies and antiviral potencies against a vector that replicates using WT IN differed considerably (IC<sub>50</sub> < 10 nM as compared to EC<sub>50</sub> ≥ 200 nM). Additionally, while analogues <b>6c</b>, <b>6d</b>, and <b>6e</b> showed micromolar ST-inhibitory potencies in the in vitro assay, they were significantly more potent in the antiviral assay. The reasons for these discrepancies are not clear. Among the remaining compounds, <b>6p</b>, with a 6-(CH<sub>2</sub>)<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub> substituent, exhibited the best antiviral profile against the panel of resistant mutants (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). This derivative displayed a modest improvement in antiviral potency against a vector carrying a WT IN (EC<sub>50</sub> = 0.67 nM) as compared to <b>6′b</b>, which had been the most promising compound in work we recently reported.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Inhibitor <b>6′b</b> contains a 6-(CH<sub>2</sub>)<sub>5</sub>OH group and a primary amine at the 4-position. Adding 4-[2-(methyl glycinate)] or 2-hydroxyethyl substituents to the 4-amino group of <b>6′b</b> did not improve the inhibitory profiles (compounds <b>6q</b> and <b>6r</b>, respectively), nor were the antiviral potencies improved by the addition of a 2-hydroxyethyl substituent to the 4-amino group of <b>6p</b> to yield <b>6t</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Most 6-subsituted analogues were not cytotoxic within the range tested (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Exceptions were found with <b>6c</b>, <b>6d</b>, and <b>6e</b>, which also showed anomalous discrepancies between their in vitro and antiviral potencies, and with <b>6f</b> and <b>6g</b>. The reasons for the greater cytotoxicities of these analogues are not clear.</div><div class="NLM_p last">An important component of our current study is to test the effects of substituents at the 7-position. Given proximity to the metal-chelating 8-aryl nitrogen, we were uncertain of what the effects such modifications would have. Bulky groups at the 7-position (<b>5b</b>, <b>5c</b>, <b>5e</b>, <b>5f</b>, <b>5h</b>, and <b>5i</b>) resulted in from two to three orders of magnitude loss of inhibitory potency against the WT enzyme in vitro. These latter data were consistent with the bulky substituents causing a disruption of essential interactions with the catalytic metal ions. Of greater interest was our finding that 7-OCH<sub>3</sub> substituents were well tolerated against WT enzyme in in vitro assays (see <b>5a</b>, <b>5d</b>, and <b>5g</b>, <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Potencies of the New Compounds Against an Extended Set of Drug Resistant IN Mutants</h3><div class="NLM_p">We selected a subset of the analogues (<b>6b</b>, <b>6p</b>, <b>6r</b>, <b>6s</b>, and <b>6t</b>) for additional evaluation in single-round viral replication assays using HIV-1 vectors harboring single G118R, T66I, E92Q, R263K, and H51Y, or the double H51Y/R263K substitutions associated with DTG resistance (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>).<a onclick="showRef(event, 'ref17 ref32 ref33 ref34 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref17 ref32 ref33 ref34 ref38 ref39">(17, 32-34, 38, 39)</a> For comparison, we also included <b>6′b</b>, which was one of the most broadly effective of our previously reported inhibitors.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The new compounds showed good antiviral potencies against the entire panel of mutants. Although several compounds appear to be promising, the most potent analogue (by a small margin) was the 6-methyl 3-propanoate-containing <b>6p</b>, which showed a slightly better inhibitory profile against the first panel of mutants (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) while also retaining greater inhibitory potency than DTG against the second, more extended panel of mutants (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Antiviral Potencies in Cells Infected with HIV-1 Vectors That Carry DTG-Resistant IN Mutants<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="7" align="center" char="±">EC<sub>50</sub> (nM/FC<a class="ref internalNav" href="#t3fn3" aria-label="c">c</a>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center" char="±">WT</th><th class="colsep0 rowsep0" align="center" char="±">G118R</th><th class="colsep0 rowsep0" align="center" char="±">T66I</th><th class="colsep0 rowsep0" align="center" char="±">E92Q</th><th class="colsep0 rowsep0" align="center" char="±">R263K</th><th class="colsep0 rowsep0" align="center" char="±">H51Y</th><th class="colsep0 rowsep0" align="center" char="±">H51Y/R263K</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>RAL</b> (<b>1</b>)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">4 ± 2</td><td class="colsep0 rowsep0" align="char" char="±">36 ± 5 (9×)</td><td class="colsep0 rowsep0" align="char" char="±">2.8 ± 0.4 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">30 ± 10 (8×)</td><td class="colsep0 rowsep0" align="char" char="±">5.7 ± 2.3 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">3.4 ± 0.2 (0.9×)</td><td class="colsep0 rowsep0" align="char" char="±">6 ± 2.3 (2×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>EVG</b> (<b>2</b>)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">6.4 ± 0.8</td><td class="colsep0 rowsep0" align="char" char="±">21 ± 10 (3×)</td><td class="colsep0 rowsep0" align="char" char="±">66 ± 1 (10×)</td><td class="colsep0 rowsep0" align="char" char="±">154 ± 34 (24×)</td><td class="colsep0 rowsep0" align="char" char="±">10 ± 6 (2×)</td><td class="colsep0 rowsep0" align="char" char="±">4.5 ± 2.1 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">53 ± 18 (8×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DTG</b> (<b>3</b>)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">1.6 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="±">13 ± 5 (8×)</td><td class="colsep0 rowsep0" align="char" char="±">0.9 ± 0.8 (0.6×)</td><td class="colsep0 rowsep0" align="char" char="±">2.3 ± 0.4 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">11 ± 3 (7×)</td><td class="colsep0 rowsep0" align="char" char="±">3.2 ± 0.2 (2×)</td><td class="colsep0 rowsep0" align="char" char="±">16 ± 2 (10×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6′b</b><a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">1.3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">5.3 ± 1.6 (4×)</td><td class="colsep0 rowsep0" align="char" char="±">0.93 ± 0.24 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">3.8 ± 2.3 (3×)</td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 0.1 (2×)</td><td class="colsep0 rowsep0" align="char" char="±">3.8 ± 0.6 (3×)</td><td class="colsep0 rowsep0" align="char" char="±">2.6 ± 1.4 (2×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6b</b></td><td class="colsep0 rowsep0" align="char" char="±">1.4 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">5.9 ± 1.4 (4×)</td><td class="colsep0 rowsep0" align="char" char="±">0.75 ± 0.07 (0.5×)</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.1 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">1.8 ± 0.3 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">0.8 ± 0.2 (0.6×)</td><td class="colsep0 rowsep0" align="char" char="±">3.9 ± 1.9 (3×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6p</b></td><td class="colsep0 rowsep0" align="char" char="±">0.67 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 1.5 (7×)</td><td class="colsep0 rowsep0" align="char" char="±">0.53 ± 0.06 (0.8×)</td><td class="colsep0 rowsep0" align="char" char="±">2.0 ± 1.1 (3×)</td><td class="colsep0 rowsep0" align="char" char="±">0.5 ± 0.0 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">0.63 ± 0.30 (0.9×)</td><td class="colsep0 rowsep0" align="char" char="±">2.4 ± 0.8 (3×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6r</b></td><td class="colsep0 rowsep0" align="char" char="±">13 ± 4.2</td><td class="colsep0 rowsep0" align="char" char="±">24 ± 8 (2×)</td><td class="colsep0 rowsep0" align="char" char="±">1.9 ± 0.07 (0.1×)</td><td class="colsep0 rowsep0" align="char" char="±">6.5 ± 0.8 (0.5×)</td><td class="colsep0 rowsep0" align="char" char="±">18 ± 2 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">11 ± 1.4 (0.8×)</td><td class="colsep0 rowsep0" align="char" char="±">31 ± 10 (2×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6s</b></td><td class="colsep0 rowsep0" align="char" char="±">5.3 ± 1.2</td><td class="colsep0 rowsep0" align="char" char="±">15 ± 2 (3×)</td><td class="colsep0 rowsep0" align="char" char="±">1.2 ± 0.5 (0.2×)</td><td class="colsep0 rowsep0" align="char" char="±">3.8 ± 1.3 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">3.6 ± 0.9 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">7.4 ± 0.9 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">15 ± 1.8 (3×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6t</b></td><td class="colsep0 rowsep0" align="char" char="±">4.1 ± 1</td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 0.6 (1×)</td><td class="colsep0 rowsep0" align="char" char="±">0.62 ± 0.15 (0.2×)</td><td class="colsep0 rowsep0" align="char" char="±">2.7 ± 1.2 (0.7×)</td><td class="colsep0 rowsep0" align="char" char="±">6.5 ± 0.8 (2×)</td><td class="colsep0 rowsep0" align="char" char="±">6.9 ± 2.9 (2×)</td><td class="colsep0 rowsep0" align="char" char="±">2.5 ± 0.1 (0.9×)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">EC<sub>50</sub> values obtained from cells infected with a lentiviral vector harboring WT or the indicated IN mutants as previously reported.</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">Data for these compounds have been reported previously.<a onclick="showRef(event, 'ref27 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref31">(27, 31)</a></p></div><div class="footnote" id="t3fn3"><sup>c</sup><p class="last">FC, fold change relative to WT = EC<sub>50</sub> of mutants/EC<sub>50</sub> of WT.</p></div></div></div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Comparison of Antiviral Potencies and Fold Improvement of Compounds <b>6′b</b> and <b>6p</b> Relative to DTG</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center">EC<sub>50</sub> (nM)<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a> (fold improvement<a class="ref internalNav" href="#t4fn2" aria-label="b">b</a>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">integrase mutants</th><th class="colsep0 rowsep0" align="center">DTG (<b>3</b>)<a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><b>6′b</b><a class="ref internalNav" href="#t4fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center"><b>6p</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">WT</td><td class="colsep0 rowsep0" align="center">1.6 ± 0.9</td><td class="colsep0 rowsep0" align="center">1.3 ± 0.2 (0.81×)</td><td class="colsep0 rowsep0" align="center">0.67 ± 0.15 (0.42×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Y143R</td><td class="colsep0 rowsep0" align="center">4.3 ± 1.2</td><td class="colsep0 rowsep0" align="center">3.0 ± 0.5 (0.71×)</td><td class="colsep0 rowsep0" align="center">0.67 ± 0.23 (0.16×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">N155H</td><td class="colsep0 rowsep0" align="center">3.6 ± 1.3</td><td class="colsep0 rowsep0" align="center">2.4 ± 0.8 (0.67×)</td><td class="colsep0 rowsep0" align="center">2.3 ± 0.2 (0.64×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G140S/Q148H</td><td class="colsep0 rowsep0" align="center">5.8 ± 0.5</td><td class="colsep0 rowsep0" align="center">9.4 ± 3.6 (1.62×)</td><td class="colsep0 rowsep0" align="center">5.3 ± 1.8 (0.91×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G118R</td><td class="colsep0 rowsep0" align="center">13 ± 5</td><td class="colsep0 rowsep0" align="center">5.3 ± 1.6 (0.41×)</td><td class="colsep0 rowsep0" align="center">4.8 ± 1.5 (0.37×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">T66I</td><td class="colsep0 rowsep0" align="center">0.9 ± 0.8</td><td class="colsep0 rowsep0" align="center">0.93 ± 0.24 (1.03×)</td><td class="colsep0 rowsep0" align="center">0.53 ± 0.06 (0.59×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">E92Q</td><td class="colsep0 rowsep0" align="center">2.3 ± 0.4</td><td class="colsep0 rowsep0" align="center">3.8 ± 2.3 (1.65×)</td><td class="colsep0 rowsep0" align="center">2.0 ± 1.1 (0.87×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">R263K</td><td class="colsep0 rowsep0" align="center">11 ± 3</td><td class="colsep0 rowsep0" align="center">2.6 ± 0.1 (0.24×)</td><td class="colsep0 rowsep0" align="center">0.5 ± 0.0 (0.05×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H51Y</td><td class="colsep0 rowsep0" align="center">3.2 ± 0.2</td><td class="colsep0 rowsep0" align="center">3.8 ± 0.6 (1.19×)</td><td class="colsep0 rowsep0" align="center">0.63 ± 0.30 (0.20×)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">H51Y/R263K</td><td class="colsep0 rowsep0" align="center">16 ± 2</td><td class="colsep0 rowsep0" align="center">2.6 ± 1.4 (0.16×)</td><td class="colsep0 rowsep0" align="center">2.4 ± 0.8 (0.15×)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>a</sup><p class="last">EC<sub>50</sub> values obtained from cells infected with a lentiviral vector harboring WT or the indicated IN mutants as previously reported.</p></div><div class="footnote" id="t4fn2"><sup>b</sup><p class="last">Fold improvement relative to DTG = EC<sub>50</sub> of compounds/EC<sub>50</sub> of DTG.</p></div><div class="footnote" id="t4fn3"><sup>c</sup><p class="last">Data for these compounds have been reported previously.<a onclick="showRef(event, 'ref27 ref31'); return false;" href="javascript:void(0);" class="ref ref27 ref31">(27, 31)</a></p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Crystal Structures of the PFV Intasome in Complex with <b>5′g</b>, <b>5g</b>, and <b>6p</b></h3><div class="NLM_p">To understand how some of the current analogues interact with the active site of IN, we soaked PFV intasome crystals in the presence of <b>5′g</b>, <b>5g</b>, or <b>6p</b> and refined the resulting structures (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). We recently described a crystal structure of <b>5′d</b> bound to the active site of PFV intasome.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The inhibitor <b>5′g</b> bound to PFV IN differs from the previously reported <b>5′d</b> structure only in having a <i>N</i>-(methyl 2-glycinate)amine moiety at its 4-position and, predictably, the two compounds bind to the active site in a very similar fashion. Superposition with the DTG-bound structure confirmed that 8-naphthyridine nitrogen and 1-<i>N</i>-hydroxyl of <b>5′g</b> take the place of the 6-oxo and 7-hydroxyl heteroatoms of DTG. The naphthyridine 2-oxo carbonyl of <b>5′g</b> corresponds to the ring 8-oxo carbonyl of DTG (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> However, the 4-<i>N</i>-(methyl 2-glycinate)amine group of <b>5′g</b> extends into the region which is occupied by the first base of the scissile dinucleotide in viral DNA prior to 3′-P (designated A18 in the crystal structure, <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A), which may be related to the ability of the compound to inhibit the 3′-P reaction.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Therefore, an extended form of substrate mimicry may arise from the modification at the 4-position of <b>5′g</b>, which was not seen in the cocrystal structure of <b>5′d</b>.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The idea that inhibitors that remain within the “substrate envelope” are particularly effective against resistant forms of the target enzyme was proposed by Schiffer and colleagues and is based on the idea that to be able to support viral replication, any mutant form of a viral enzyme must be capable of binding its normal substrate(s).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Thus, if the bound inhibitor remains within an envelope defined by the substrate(s), the inhibitor is expected to retain some efficacy against mutant forms of the enzyme.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Even though the original hypothesis was based on work with protease inhibitors, it seems likely that this principle can be applied broadly to antiviral compounds that bind at the active sites of essential viral enzymes. The 4-<i>N</i>-(methyl 2-glycinate) amine group of <b>5′g</b> is situated near (within 5 Å) to the side chains of Y212 and P214 (residues corresponding to Y143 and P145 in HIV-1 IN) and to the adenosine base of the 3′ terminal viral DNA residue (A17), which adopts two alternative conformations in the crystal structure (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). The interactions with the side chains are much less extensive in the cases of DTG (<b>3</b>) and <b>5′d</b>. Extension into this region of the catalytic site is also seen with several highly potent INSTIs having diverse structures (see <a href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_001.pdf" class="ext-link">Figure S1</a> in Supporting Information).</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. X-Ray Data Collection and Refinement Statistics<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"><b>5′g</b></th><th class="colsep0 rowsep0" align="center"><b>5g</b></th><th class="colsep0 rowsep0" align="center"><b>6p</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left">Data Collection</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">space group</td><td class="colsep0 rowsep0" align="left"><i>P</i>4<sub>1</sub>2<sub>1</sub>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>4<sub>1</sub>2<sub>1</sub>2</td><td class="colsep0 rowsep0" align="left"><i>P</i>4<sub>1</sub>2<sub>1</sub>2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">cell dimensions <i>a</i>, <i>b</i>, <i>c</i> (Å)</td><td class="colsep0 rowsep0" align="left">159.6, 159.6, 124.1</td><td class="colsep0 rowsep0" align="left">158.6, 158.6, 123.1</td><td class="colsep0 rowsep0" align="left">160.5, 160.5, 123.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">resolution range (Å)</td><td class="colsep0 rowsep0" align="left">71.36–2.55 (2.62–2.55)</td><td class="colsep0 rowsep0" align="left">79.29–2.60(2.67–2.60)</td><td class="colsep0 rowsep0" align="left">71.77–2.77 (2.84–2.77)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i><sub>merge</sub></td><td class="colsep0 rowsep0" align="left">0.062 (1.49)</td><td class="colsep0 rowsep0" align="left">0.025 (0.55)</td><td class="colsep0 rowsep0" align="left">0.072 (1.075)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>I</i>/σ(<i>I</i>)</td><td class="colsep0 rowsep0" align="left">26.6 (2.0)</td><td class="colsep0 rowsep0" align="left">19.5 (1.4)</td><td class="colsep0 rowsep0" align="left">18.9 (2.0)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">completeness (%)</td><td class="colsep0 rowsep0" align="left">99.8 (98.8)</td><td class="colsep0 rowsep0" align="left">99.9 (99.7)</td><td class="colsep0 rowsep0" align="left">98.8 (97.4)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">redundancy</td><td class="colsep0 rowsep0" align="left">11.5 (11.0)</td><td class="colsep0 rowsep0" align="left">14.4 (14.2)</td><td class="colsep0 rowsep0" align="left">7.2 (7.3)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left">Refinement</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reflections (total/free)</td><td class="colsep0 rowsep0" align="left">52565/2601</td><td class="colsep0 rowsep0" align="left">48670/2438</td><td class="colsep0 rowsep0" align="left">40988/2057</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>R</i>/<i>R</i><sub>free</sub></td><td class="colsep0 rowsep0" align="left">0.179/0.207</td><td class="colsep0 rowsep0" align="left">0.176/0.204</td><td class="colsep0 rowsep0" align="left">0.177/0.205</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">no. atoms</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein, DNA</td><td class="colsep0 rowsep0" align="left">5150</td><td class="colsep0 rowsep0" align="left">5129</td><td class="colsep0 rowsep0" align="left">5127</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligand</td><td class="colsep0 rowsep0" align="left">159</td><td class="colsep0 rowsep0" align="left">132</td><td class="colsep0 rowsep0" align="left">146</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">229</td><td class="colsep0 rowsep0" align="left">109</td><td class="colsep0 rowsep0" align="left">87</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">average <i>B</i>-factors (Å<sup>2</sup>)</td><td class="colsep0 rowsep0" align="left">71.7</td><td class="colsep0 rowsep0" align="left">84.3</td><td class="colsep0 rowsep0" align="left">75.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">protein, DNA</td><td class="colsep0 rowsep0" align="left">71.3</td><td class="colsep0 rowsep0" align="left">83.9</td><td class="colsep0 rowsep0" align="left">74.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ligands</td><td class="colsep0 rowsep0" align="left">91.8</td><td class="colsep0 rowsep0" align="left">117.23</td><td class="colsep0 rowsep0" align="left">107.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">water</td><td class="colsep0 rowsep0" align="left">67.2</td><td class="colsep0 rowsep0" align="left">78.9</td><td class="colsep0 rowsep0" align="left">68.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">rmsd</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond lengths (Å)</td><td class="colsep0 rowsep0" align="left">0.038</td><td class="colsep0 rowsep0" align="left">0.039</td><td class="colsep0 rowsep0" align="left">0.012</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bond angles (deg)</td><td class="colsep0 rowsep0" align="left">0.56</td><td class="colsep0 rowsep0" align="left">0.76</td><td class="colsep0 rowsep0" align="left">1.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Ramachandran plot (%)</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">favored</td><td class="colsep0 rowsep0" align="left">99</td><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left">98</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">outliers</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0</td><td class="colsep0 rowsep0" align="left">0.2</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">Each structure was determined from a single crystal. Data for the highest resolution shells are given in parentheses.</p></div></div></div><figure id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0001.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structures of PFV intasome-bound inhibitors. (A) Bound <b>5′g</b> (cream) with overlays showing the relative positions of pre-3′-P viral DNA (violet; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I">4E7I</a>) and DTG (<b>3</b>) (green; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S3M">3S3M</a>). The 4-<i>N</i>-(methyl 2-glycinate) moiety of <b>5′g</b> is circled in red, highlighting its correspondence with the T<sub>+2</sub> adenosine (A18) base of pre-3′-P viral DNA. (B) Bound <b>5′g</b> with its 4-<i>N</i>-(methyl 2-glycinate) moiety shown as transparent spheres. Atoms within 5 Å (Y212 and P214 and A17 of 3′-P DNA) are shown in green as a stick diagram. For both A and B, metal ions are shown as solid blue spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Prior structural studies using the PFV intasome showed that the tricyclic system of DTG makes contacts with G187 in the β4−α2 loop of PFV IN (G118 in IN), and it has been argued that the interactions with this region may contribute to the improved properties of DTG and other second-generation INSTIs.<a onclick="showRef(event, 'ref4 ref23 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref4 ref23 ref29 ref30">(4, 23, 29, 30)</a> As has already been discussed, we thought that it might be possible to make useful contacts with the same region of the catalytic site by adding functionality to the 6- and 7-positions of <b>4</b>. To this end, we introduced substituents at the 6- and 7-positions of <b>4</b> that could potentially recapitulate some of the desirable features of DTG (highlighted in green and cyan, respectively, in the structures of DTG and <b>4</b>, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). We obtained the crystal structure of <b>6p</b> bound to the PFV intasome because it represents the most potent 6-substituted analogue in our current study. The structure revealed that the 6-methyl 3-propanoate side chain of the compound is situated within the region defined by an envelope defined by both the pre-3′-P viral DNA and the host target DNA substrate (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>). This is consistent with our recent report that 6-substituents of the compounds exhibiting the best antiviral profiles against the panel of resistant mutants bind within an envelope defined as the confluence of viral DNA substrates.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The structure also shows that the methyl carboxylate group of <b>6p</b> makes van der Waals interactions with the β4−α2 loop of IN, becoming sandwiched between main chain atoms of Q186 and G187 (residues corresponding to HIV-1 N117 and G118, respectively) and the side chain of Y212 (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0002.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structures of <b>6p</b> bound to the PFV intasome. Structures of the pre-3′-P viral DNA (violet; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I">4E7I</a>) and target complex host DNA substrate (gray; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7K">4E7K</a>) are overlaid. The 6-(methyl 3-propanoate) group of <b>6p</b> is circled in red, showing its correspondence with aspects of both pre-3′-P viral DNA and the host target DNA. Cofactor Mg<sup>2+</sup> ions are shown as blue spheres; the structures of bound viral DNAs have been omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0003.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structures of PFV intasome-bound inhibitors. (A) Bound <b>5g</b> (orange) with superimposed <b>6p</b> (cyan) and DTG (<b>3</b>) (green; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S3M">3S3M</a>) showing closest distance to the Gly187 α-methylene (side chains of Asp186 and Gln187 shown in semitransparent), bound 3′-P DNAs have been omitted for clarity. (B) Superimposed bound 3′-P DNAs associated with A, circled in cyan are the oxygens in the C16–A17 phosphoryl linkages, whose engagement in electrostatic interactions with the side chain imidazole nitrogen of the N224H mutant (corresponding to IN mutant N155H) would be broken on binding of the INSTIs. (C) Bound <b>5g</b> with superimposed pre-3′-P viral DNA (violet; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I">4E7I</a>) and target complex host DNA substrate (gray; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7K">4E7K</a>), circled in red is the 7-OCH<sub>3</sub>, which protrudes outside the substrate envelope. Metal ions are shown as solid blue spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To understand the effects of introducing a 7-OCH<sub>3</sub> substituent, we solved the structure of <b>5g</b> bound to the PFV intasome and overlaid the corresponding structure of DTG bound to the PFV intasome (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). We observed that the 7-OCH<sub>3</sub> group of <b>5g</b> is approximately 3.6 Å from the G187 residue in the β4−α2 loop, which compares favorably with DTG and <b>6p</b> (approximately 3.9 and 3.7 Å, respectively, <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>A). The 7-OCH<sub>3</sub> group of <b>5g</b> did not appear to significantly alter the position of the metal chelating heteroatoms of <b>5g</b> relative to <b>6p</b>. However, it is not immediately obvious why <b>5g</b> is more susceptible to the N155H mutation compared to <b>6p</b> or to the more closely related <b>5′g</b>. Because the N155 residue of HIV-1 IN corresponds to N224 in PFV IN, the X-ray crystal structure with the mutation N224H in PFV IN did show that the side chain of His224 interacts with a phosphoryl oxygen atom in viral DNA and that INSTI binding is accompanied by a loss of this contact.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> The associated energetic penalty may explain the loss potency of some compounds against N155H HIV-1 IN. In the cocrystal structure of PFV intasome with <b>5g,</b> the location of this phosphoryl oxygen is nearly identical to that seen for <b>6p</b>, <b>5′g</b>, and DTG (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>C). The abilities of some INSTIs to readjust in the active site of IN may help explain variations among in their susceptibilities of the different INSTIs to the N155H mutation.<a onclick="showRef(event, 'ref23 ref24 ref41'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref41">(23, 24, 41)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30832" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30832" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Earlier, we reported that the 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide scaffold represents a potentially attractive platform for the development of INSTIs that retain good antiviral efficacy against a panel of viral vectors carrying resistant mutant forms of IN.<a onclick="showRef(event, 'ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref26 ref27">(26, 27)</a> More recently, we began an exploration of the effects of modifying the 6-position of our earlier compounds.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> We found that 6-substituents may play an important role in enhancing the maintenance of antiviral efficacy against drug-resistant mutant forms of IN. Furthermore, the most broadly effective compounds are substrate mimics that fit within the “substrate envelope.″ Our original purpose in synthesizing 6-substituents was to create additional interactions within areas of the catalytic region accessed by the third ring of DTG. However, the range of functionality reported in the prior paper was quite limited, consisting of primary hydroxyl groups tethered by linear polymethylene chains of varying lengths and a single tethered phenyl sulfone. We now report a much more extensive exploration of functionalities at the 6-position as well as the first examination of derivatives with modifications at the 7-position. We also tested whether the compounds could be improved by introduced by varying functionalities at the 4-position.</div><div class="NLM_p last">We observed that −OCH<sub>3</sub> groups are somewhat better tolerated than bulky subsituents at the 7-position when examined in vitro against WT enzyme. However, compounds with 7-OMe groups showed significantly reduced antiviral potency when tested against viral vectors that carry the N155H mutation. We found that a variety of substituents can be added at the 6-position, which enhance the antiviral potencies of the compounds against the major RAL-, EVG-, and DTG-resistant IN mutants. Many of these compounds are not cytotoxic at the tested concentrations, and the best analogue, <b>6p</b>, is the most effective in terms of its ability to inhibit the entire spectrum of IN mutants of any of the compounds we have described. The crystal structure of the PFV intasome with <b>6p</b> bound supports the idea that interactions with the β4−α2 loop are important and that compounds that bind within the envelope defined by both the viral and host DNA substrates helps the compounds retain broad efficacy. We found that the 4-amine substituent in <b>5′g</b> accesses a region of the active site of IN that would normally be occupied by the base of the T<sub>+2</sub> adenosine of the viral DNA prior to the 3′-P cleavage, providing further support that binding within an envelope defined by the DNA substrates helps the compounds retain broad efficacy. Thus, our results will inform the design and development of next-generation INSTIs with broader efficacy against the known drug resistant mutants.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73784" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73784" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Procedures</h3><div class="NLM_p last">Proton (<sup>1</sup>H) and carbon (<sup>13</sup>C) NMR spectra were recorded on a Varian 400 MHz spectrometer or a Varian 500 MHz spectrometer and are reported in ppm relative to TMS and referenced to the solvent in which the spectra were collected. Solvent was removed by rotary evaporation under reduced pressure, and anhydrous solvents were obtained commercially and used without further drying. Purification by silica gel chromatography was performed using Combiflash with EtOAc–hexanes solvent systems. Preparative high pressure liquid chromatography (HPLC) was conducted using a Waters Prep LC4000 system having photodiode array detection and Phenomenex C<sub>18</sub> columns (catalogue no. 00G-4436-P0-AX, 250 mm × 21.2 mm 10 μm particle size, 110 Å pore) at a flow rate of 10 mL/min. Binary solvent systems consisting of A = 0.1% aqueous TFA and B = 0.1% TFA in acetonitrile were employed with gradients as indicated. Products were obtained as amorphous solids following lyophilization. Electrospray ionization-mass spectrometric (ESI-MS) were acquired with an Agilent LC/MSD system equipped with a multimode ion source. Purities of samples subjected to biological testing were assessed using this system and shown to be ≥95%. High resolution mass spectrometric (HRMS) were acquired by LC/MS-ESI using LTQ-Orbitrap-XL at 30K resolution.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Procedure I for the Synthesis of Carboxamides (<b>5a</b>–<b>i</b> and <b>6a</b>–<b>t</b>)</h3><div class="NLM_p">Benzyl protected compounds (<b>13</b>, <b>15</b>, <b>16</b>, and <b>19</b>) (0.1 mmol) were suspended in methanol (10 mL) and ethyl acetate (3 mL). One equivalent of Pd/C (10%) was added. The mixture was stirred at room temperature under hydrogen. When the starting material was disappeared (TLC), the crude mixture was filtered and washed by methanol. The filtrate was concentrated and purified by HPLC to provide final carboxamides (<b>5a</b>–<b>i</b> and <b>6a</b>–<b>t</b>).</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>N</i>-(2,4-Difluorobenzyl)-1,4-dihydroxy-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>5a</b>)</h4><div class="NLM_p last">Treatment of <b>13a</b> as outlined in general procedure I and purification by preparative HPLC (with a linear gradient of 30% B to 90% B over 30 min; retention time = 25.5 min) provided <b>5a</b> as a white fluffy solid (22% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.00 (s, 1H), 10.42 (d, <i>J</i> = 5.8 Hz, 1H), 8.29 (dd, <i>J</i> = 8.7, 0.9 Hz, 1H), 7.48 (dd, <i>J</i> = 15.3, 8.6 Hz, 1H), 7.30–7.25 (m, 1H), 7.10 (dd, <i>J</i> = 9.8, 7.3 Hz, 1H), 6.83–6.81 (m, 1H), 4.62 (d, <i>J</i> = 6.0 Hz, 2H), 4.02 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 378.1 (MH<sup>+</sup>).</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>N</i>-(2,4-Difluorobenzyl)-1,4-dihydroxy-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>5b</b>)</h4><div class="NLM_p last">Treatment of <b>13b</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 50% B to 90% B over 30 min; retention time = 24.0 min) provided <b>5b</b> as a white fluffy solid (13% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.52 (bs, 1H), 10.36 (t, <i>J</i> = 5.9 Hz, 1H), 7.93 (d, <i>J</i> = 9.1 Hz, 1H), 7.41 (dd, <i>J</i> = 15.3, 8.7 Hz, 1H), 7.25–7.19 (m, 1H), 7.04 (dd, <i>J</i> = 9.3, 6.7 Hz, 1H), 6.82 (d, <i>J</i> = 9.2 Hz, 1H), 4.54 (d, <i>J</i> = 5.8 Hz, 2H), 3.74 (s, 4H), 1.61 (bs, 2H), 1.52 (bs, 4H). ESI-MS <i>m</i>/<i>z</i>: 431.2 (MH<sup>+</sup>).</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(2,4-Difluorobenzyl)-1,4-dihydroxy-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>5c</b>)</h4><div class="NLM_p last">Treatment of <b>13c</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 30% B to 90% B over 30 min; retention time = 24.3 min) provided <b>5c</b> as a white fluffy solid (30% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.60 (s, 1H), 10.36 (t, <i>J</i> = 6.0 Hz, 1H), 8.01 (d, <i>J</i> = 9.1 Hz, 1H), 7.41 (dd, <i>J</i> = 15.3, 8.6 Hz, 1H), 7.25–7.19 (m, 1H), 7.06–7.02 (m, 1H), 6.83 (d, <i>J</i> = 9.1 Hz, 1H), 4.54 (d, <i>J</i> = 6.0 Hz, 2H), 3.73 (s, 4H), 3.65 (t, <i>J</i> = 4.5 Hz, 4H). ESI-MS <i>m</i>/<i>z</i>: 433.1 (MH<sup>+</sup>).</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>5d</b>)</h4><div class="NLM_p last">Treatment of <b>16a</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 30% B to 75% B over 30 min; retention time = 22.4 min) provided <b>5d</b> as a white fluffy solid (37% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.60 (t, <i>J</i> = 5.6 Hz, 1H), 10.51 (bs, 1H), 8.48 (d, <i>J</i> = 8.8 Hz, 1H), 7.38 (dd, <i>J</i> = 15.4, 8.7 Hz, 1H), 7.22–7.17 (m, 1H), 7.02 (td, <i>J</i> = 8.5, 2.5 Hz, 1H), 6.72 (d, <i>J</i> = 8.8 Hz, 1H), 4.46 (d, <i>J</i> = 5.7 Hz, 2H), 3.94 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 377.1 (MH<sup>+</sup>).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>5e</b>)</h4><div class="NLM_p last">Treatment of <b>16b</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 50% B to 70% B over 30 min; retention time = 20.6 min) provided <b>5e</b> as a white fluffy solid (49% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (t, <i>J</i> = 5.8 Hz, 2H), 8.19 (d, <i>J</i> = 9.2 Hz, 1H), 7.36 (dd, <i>J</i> = 15.4, 8.7 Hz, 1H), 7.21–7.16 (m, 1H), 7.02 (t, <i>J</i> = 9.0 Hz, 1H), 6.75 (d, <i>J</i> = 9.2 Hz, 1H), 4.44 (d, <i>J</i> = 5.7 Hz, 2H), 3.69 (bs, 4H), 1.59 (bs, 2H), 1.50 (bs, 4H). ESI-MS <i>m</i>/<i>z</i>: 430.2 (MH<sup>+</sup>).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>5f</b>)</h4><div class="NLM_p last">Treatment of <b>16c</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 30% B to 65% B over 30 min; retention time = 24.0 min) provided <b>5f</b> as a white fluffy solid (28% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (t, <i>J</i> = 5.7 Hz, 1H), 10.27 (bs, 1H), 10.19 (bs, 1H), 8.26 (d, <i>J</i> = 9.1 Hz, 1H), 7.36 (dd, <i>J</i> = 15.4, 8.7 Hz, 1H), 7.21–7.16 (m, 1H), 7.02 (td, <i>J</i> = 8.2, 2.0 Hz, 1H), 6.76 (d, <i>J</i> = 9.1 Hz, 1H), 4.44 (d, <i>J</i> = 5.7 Hz, 2H), 3.66 (d, <i>J</i> = 3.3 Hz, 8H). ESI-MS <i>m</i>/<i>z</i>: 432.1 (MH<sup>+</sup>).</div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Methyl 2-((3-((2,4-Difluorobenzyl)carbamoyl)-1-hydroxy-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>5g</b>)</h4><div class="NLM_p last">Treatment of <b>15a</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 30% B to 80% B over 30 min; retention time = 23.7 min) provided <b>5g</b> as a white fluffy solid (20% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.17 (bs, 1H), 10.61 (bs, 1H), 10.45 (t, <i>J</i> = 5.7 Hz, 1H), 8.31 (d, <i>J</i> = 9.0 Hz, 1H), 7.45 (dd, <i>J</i> = 15.3, 8.7 Hz, 1H), 7.22–7.17 (m, 1H), 7.02 (td, <i>J</i> = 8.5, 1.6 Hz, 1H), 6.65 (d, <i>J</i> = 8.9 Hz, 1H), 4.48 (d, <i>J</i> = 5.4 Hz, 2H), 4.45 (d, <i>J</i> = 5.9 Hz, 2H), 3.94 (s, 3H), 3.63 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 449.1 (MH<sup>+</sup>).</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Methyl 2-((3-((2,4-Difluorobenzyl)carbamoyl)-1-hydroxy-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>5h</b>)</h4><div class="NLM_p last">Treatment of <b>15b</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 50% B to 70% B over 30 min; retention time = 22.9 min) provided <b>5h</b> as a white fluffy solid (29% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.45 (bs, 1H), 10.70 (t, <i>J</i> = 5.7 Hz, 1H), 7.98 (d, <i>J</i> = 9.4 Hz, 1H), 7.39 (dd, <i>J</i> = 15.3, 8.7 Hz, 1H), 7.22–7.16 (m, 1H), 7.02 (td, <i>J</i> = 8.6, 2.1 Hz, 1H), 6.65 (d, <i>J</i> = 9.4 Hz, 1H), 4.45 (bs, 2H), 4.44 (s, 2H), 3.70 (bs, 4H), 3.64 (s, 3H), 1.60 (bs, 3H), 1.51 (bs, 3H). ESI-MS <i>m</i>/<i>z</i>: 502.2 (MH<sup>+</sup>).</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Methyl 2-((3-((2,4-Difluorobenzyl)carbamoyl)-1-hydroxy-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>5i</b>)</h4><div class="NLM_p last">Treatment of <b>15c</b> as outlined in general procedure I, and purification by preparative HPLC (with a linear gradient of 30% B to 70% B over 30 min; retention time = 26.0 min) provided <b>5i</b> as a white fluffy solid (18% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.43 (bs, 1H), 10.66 (t, <i>J</i> = 5.8 Hz, 1H), 10.31 (bs, 1H), 8.06 (d, <i>J</i> = 9.3 Hz, 1H), 7.40 (dd, <i>J</i> = 15.4, 8.7 Hz, 1H), 7.21–7.16 (m, 1H), 7.02 (dd, <i>J</i> = 9.8, 7.4 Hz, 1H), 6.66 (d, <i>J</i> = 9.3 Hz, 1H), 4.47 (d, <i>J</i> = 5.4 Hz, 2H), 4.45 (d, <i>J</i> = 5.7 Hz, 2H), 3.66 (d, <i>J</i> = 5.2 Hz, 8H), 3.63 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 504.2 (MH<sup>+</sup>).</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> <i>N</i>-(2,4-Difluorobenzyl)-1,4-dihydroxy-6-(5-hydroxypentyl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6a</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17a</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available pent-4-yn-1-ol as outlined in general procedures J and I and purification by preparative HPLC (linear gradient of 30% B to 70% B over 30 min; retention time = 23.8 min) provided <b>6a</b> as a white fluffy solid (46% yield, two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.96 (bs, 1H), 10.52 (t, <i>J</i> = 6.1 Hz, 1H), 8.70 (d, <i>J</i> = 2.0 Hz, 1H), 8.26 (d, <i>J</i> = 2.0 Hz, 1H), 7.48 (dd, <i>J</i> = 15.3, 8.6 Hz, 1H), 7.30–7.25 (m, 1H), 7.09 (td, <i>J</i> = 8.5, 2.5 Hz, 1H), 4.63 (d, <i>J</i> = 5.9 Hz, 2H), 3.37 (t, <i>J</i> = 6.5 Hz, 2H), 2.73 (t, <i>J</i> = 7.6 Hz, 2H), 1.65–1.59 (m, 2H), 1.46–1.42 (m, 2H), 1.35–1.28 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 434.1 (MH<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>22</sub>F<sub>2</sub>N<sub>3</sub>O<sub>5</sub> [MH<sub>+</sub>], 434.1522; found, 434.1507.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 3-(5-Amino-6-((2,4-difluorobenzyl)carbamoyl)-8-hydroxy-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)propyl Acetate (<b>6b</b>)</h4><div class="NLM_p last">Treatment of <b>19a</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 30% B to 70% B over 30 min; retention time = 20.6 min) provided <b>6b</b> as a white fluffy solid (71% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (t, <i>J</i> = 5.7 Hz, 1H), 8.60 (s, 1H), 8.57 (s, 1H), 7.42 (dd, <i>J</i> = 15.6, 7.5 Hz, 1H), 7.24 (t, <i>J</i> = 9.9 Hz, 1H), 7.07 (t, <i>J</i> = 8.5 Hz, 1H), 4.52 (d, <i>J</i> = 5.5 Hz, 2H), 4.05 (t, <i>J</i> = 6.3 Hz, 2H), 2.75 (t, <i>J</i> = 7.9 Hz, 2H), 2.00 (s, 3H), 1.97 (d, <i>J</i> = 7.7 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 447.1 (MH<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>21</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 447.1475; found, 447.1477.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 6-(5-Amino-6-((2,4-difluorobenzyl)carbamoyl)-8-hydroxy-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)hexyl Benzoate (<b>6c</b>)</h4><div class="NLM_p last">Protection of 1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-6-(6-hydroxyhex-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (50 mg, 0.073 mmol) using benzoic anhydride (25 mg, 0.11 mmol) in pyridine (0.5 mL) (room temperature, 2 h) provided 6-(8-(benzyloxy)-6-((2,4-difluorobenzyl)carbamoyl)-5-((2,4-dimethoxybenzyl)amino)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)hex-5-yn-1-yl benzoate as a white solid (56 mg, 96% yield). [<sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 11.99 (t, <i>J</i> = 5.8 Hz, 1H), 10.69 (t, <i>J</i> = 5.8 Hz, 1H), 8.67 (d, <i>J</i> = 1.4 Hz, 1H), 8.34 (d, <i>J</i> = 1.7 Hz, 1H), 8.07–8.05 (m, 2H), 7.39–7.68 (m, 2H), 7.57 (t, <i>J</i> = 7.4 Hz, 1H), 7.45 (t, <i>J</i> = 7.8 Hz, 2H), 7.40–7.36 (m, 4H), 7.23 (d, <i>J</i> = 8.9 Hz, 1H), 6.86–6.79 (m, 2H), 6.48–6.46 (m, 2H), 5.26 (s, 2H), 4.75 (d, <i>J</i> = 5.9 Hz, 2H), 4.60 (d, <i>J</i> = 5.6 Hz, 2H), 4.41 (t, <i>J</i> = 6.4 Hz, 2H), 3.81 (s, 3H), 3.78 (s, 3H), 2.54 (t, <i>J</i> = 7.0 Hz, 2H), 1.99–1.93 (m, 2H), 1.84–1.78 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 787.3 (MH<sup>+</sup>).] After treatment as outlined in general procedure H and I and purification by preparative HPLC (linear gradient of 55% B to 75% B over 30 min; retention time = 25.4 min), <b>6c</b> was provided as a white fluffy solid (45% yield, two steps). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (t, <i>J</i> = 5.7 Hz, 1H), 8.58 (d, <i>J</i> = 1.9 Hz, 1H), 8.54 (s, 1H), 7.94 (dd, <i>J</i> = 8.4, 1.3 Hz, 2H), 7.64 (dt, <i>J</i> = 8.7, 1.3 Hz, 1H), 7.53–7.50 (m, 2H), 7.42 (dd, <i>J</i> = 15.3, 8.5 Hz, 1H), 7.26–7.21 (m, 1H), 7.06 (dd, <i>J</i> = 9.8, 7.3 Hz, 1H), 4.51 (d, <i>J</i> = 5.6 Hz, 2H), 4.26 (t, <i>J</i> = 6.5 Hz, 2H), 2.70–2.67 (m, 2H), 1.73–1.66 (m, 4H), 1.45–1.42 (m, 2H), 1.39–1.38 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 551.2 (MH<sup>+</sup>). HRMS calcd C<sub>29</sub>H<sub>29</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 551.2089; found, 551.2089.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-Amino-6-(3-cyclohexylpropyl)-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6d</b>)</h4><div class="NLM_p last">Treatment of <b>19b</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 50% B to 80% B over 30 min; retention time = 27.5 min) provided <b>6d</b> as a white fluffy solid (40% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (t, <i>J</i> = 5.8 Hz, 1H), 8.51 (s, 1H), 8.47 (s, 1H), 7.36 (dd, <i>J</i> = 15.5, 8.6 Hz, 1H), 7.20–7.15 (m, 1H), 7.00 (t, <i>J</i> = 8.5 Hz, 1H), 4.45 (d, <i>J</i> = 5.7 Hz, 2H), 2.59 (t, <i>J</i> = 7.7 Hz, 2H), 1.63–1.52 (m, 7H), 1.16–1.00 (m, 6H), 0.82–0.75 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 471.2 (MH<sup>+</sup>). HRMS calcd C<sub>25</sub>H<sub>29</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub> [MH<sup>+</sup>], 471.2202; found, 471.2204.</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-2-oxo-6-(4-phenylbutyl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6e</b>)</h4><div class="NLM_p last">Treatment of <b>19c</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 40% B to 80% B over 30 min; retention time = 26.0 min) provided <b>6e</b> as a white fluffy solid (27% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (t, <i>J</i> = 5.8 Hz, 1H), 8.58 (s, 1H), 8.54 (s, 1H), 7.42 (dd, <i>J</i> = 15.5, 8.5 Hz, 1H), 7.26 (t, <i>J</i> = 7.6 Hz, 2H), 7.24–7.21 (m, 1H), 7.19–7.14 (m, 3H), 7.07 (t, <i>J</i> = 8.5 Hz, 1H), 4.51 (d, <i>J</i> = 5.7 Hz, 2H), 2.72 (t, <i>J</i> = 7.3 Hz, 2H), 2.62 (t, <i>J</i> = 7.5 Hz, 2H), 1.69 (dd, <i>J</i> = 14.8, 7.1 Hz, 2H), 1.65–1.53 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 479.2 (MH<sup>+</sup>). HRMS calcd C<sub>26</sub>H<sub>25</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub> [MH<sup>+</sup>], 479.1889; found, 479.1880.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-2-oxo-6-phenethyl-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6f</b>)</h4><div class="NLM_p last">Treatment of <b>19d</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 40% B to 80% B over 30 min; retention time = 22.5 min) provided <b>6f</b> as a white fluffy solid (65% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (bs, 1H), 10.51 (bs, 1H), 8.61 (s, 1H), 8.53 (s, 1H), 7.45–7.42 (m, 1H), 7.35–7.32 (m, 1H), 7.27 (d, <i>J</i> = 7.0 Hz, 1H), 7.23 (d, <i>J</i> = 6.7 Hz, 2H), 7.20–7.17 (m, 1H), 7.08–7.05 (m, 1H), 4.52 (d, <i>J</i> = 5.5 Hz, 2H), 2.99 (bs, 4H). ESI-MS <i>m</i>/<i>z</i>: 451.1 (MH<sup>+</sup>). HRMS calcd C<sub>24</sub>H<sub>21</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub> [MH<sup>+</sup>], 451.1576; found, 451.1576.</div></div><div id="sec4_2_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-6-(3-(dimethylamino)propyl)-1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6g</b>)</h4><div class="NLM_p last">Treatment of <b>19e</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 30% B to 55% B over 30 min; retention time = 17.2 min) provided <b>6g</b> as a white fluffy solid (52% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.68 (t, <i>J</i> = 5.8 Hz, 1H), 10.57 (s, 1H), 8.62 (s, 1H), 8.57 (s, 1H), 7.42 (dd, <i>J</i> = 15.5, 8.5 Hz, 1H), 7.27–7.22 (m, 1H), 7.07 (dd, <i>J</i> = 9.8, 7.3 Hz, 1H), 4.52 (d, <i>J</i> = 5.8 Hz, 2H), 3.06 (d, <i>J</i> = 7.7 Hz, 2H), 2.78 (s, 6H), 2.74 (t, <i>J</i> = 7.5 Hz, 2H), 2.01 (dt, <i>J</i> = 15.7, 7.8 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 432.2 (MH<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>24</sub>F<sub>2</sub>N<sub>5</sub>O<sub>3</sub> [MH<sup>+</sup>], 432.1842; found, 432.1824.</div></div><div id="sec4_2_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-6-(3-(2-hydroxyethoxy)propyl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6h</b>)</h4><div class="NLM_p last">Treatment of <b>19f</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 30% B to 60% B over 30 min; retention time = 20.5 min) provided <b>6h</b> as a white fluffy solid (86% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.69 (t, <i>J</i> = 5.8 Hz, 1H), 8.59 (d, <i>J</i> = 1.9 Hz, 1H), 8.56 (d, <i>J</i> = 1.9 Hz, 1H), 7.52–7.35 (m, 1H), 7.24 (ddd, <i>J</i> = 10.5, 9.4, 2.6 Hz, 1H), 7.07 (ddd, <i>J</i> = 10.4, 8.1, 2.2 Hz, 1H), 4.52 (d, <i>J</i> = 5.7 Hz, 2H), 3.51 (t, <i>J</i> = 5.3 Hz, 2H), 3.41 (dt, <i>J</i> = 7.5, 5.7 Hz, 4H), 2.76–2.73 (m, 2H), 1.92–1.86 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 449.1 (MH<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>23</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 449.1631; found, 449.1631.</div></div><div id="sec4_2_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 3-(5-Amino-6-((2,4-difluorobenzyl)carbamoyl)-8-hydroxy-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)propanoic Acid (<b>6i</b>)</h4><div class="NLM_p last">Treatment of <b>19g</b> with aqueous NaOH and then as outlined in general procedure I and purification by preparative HPLC (with a linear gradient of 30% B to 50% B over 30 min; retention time = 19.5 min) provided <b>6i</b> as a white fluffy solid (80% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (t, <i>J</i> = 5.4 Hz, 1H), 8.68–8.66 (m, 1H), 8.63–8.60 (m, 1H), 8.57 (d, <i>J</i> = 1.9 Hz, 1H), 8.53 (d, <i>J</i> = 1.9 Hz, 1H), 7.40–7.34 (m, 1H), 7.21–7.16 (m, 1H), 7.04–6.99 (m, 1H), 4.47 (d, <i>J</i> = 5.5 Hz, 2H), 2.87 (t, <i>J</i> = 7.5 Hz, 2H), 2.62 (t, <i>J</i> = 7.6 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 419.1 (MH<sup>+</sup>). HRMS calcd C<sub>19</sub>H<sub>17</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 419.1162; found, 419.1147.</div></div><div id="sec4_2_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-6-(3-(dimethylamino)-3-oxopropyl)-1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6j</b>) and (<i>E</i>)-4-Amino-<i>N</i>-(2,4-difluorobenzyl)-6-(3-(dimethylamino)-3-oxoprop-1-en-1-yl)-1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6k</b>)</h4><div class="NLM_p last">Treatment of <b>18e</b> as outlined in general procedures I and H and purification by preparative HPLC (with a linear gradient of 30% B to 45% B over 30 min) provided <b>6j</b> and <b>6k</b> as white fluffy solids. For <b>6j</b>: retention time = 20.7 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (t, <i>J</i> = 5.8 Hz, 1H), 10.46 (brs, 1H), 8.56 (d, <i>J</i> = 1.8 Hz, 1H), 8.50 (d, <i>J</i> = 1.8 Hz, 1H), 7.36 (dd, <i>J</i> = 15.4, 8.6 Hz, 1H), 7.20–7.14 (m, 1H), 7.00 (td, <i>J</i> = 8.6, 1.9 Hz, 1H), 4.45 (d, <i>J</i> = 5.7 Hz, 2H), 2.90 (s, 3H), 2.84 (t, <i>J</i> = 7.4 Hz, 2H), 2.74 (s, 3H), 2.65 (t, <i>J</i> = 7.5 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 446.2 (MH<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>22</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub> [MH<sup>+</sup>], 446.1634; found, 446.1617. For <b>6k</b>: retention time = 24.1 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (s, 1H), 10.53 (t, <i>J</i> = 5.8 Hz, 1H), 8.94 (s, 2H), 7.49 (d, <i>J</i> = 15.4 Hz, 1H), 7.37 (dd, <i>J</i> = 12.2, 5.3 Hz, 1H), 7.31 (d, <i>J</i> = 15.5 Hz, 1H), 7.21–7.15 (m, 1H), 7.01 (td, <i>J</i> = 8.9, 2.3 Hz, 1H), 4.46 (d, <i>J</i> = 5.7 Hz, 2H), 3.14 (s, 3H), 2.89 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 444.1 (MH<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>20</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub> [MH<sup>+</sup>), 444.1478; found, 444.1475.</div></div><div id="sec4_2_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-6-(3-(isopropylamino)-3-oxopropyl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6l</b>) and (<i>E</i>)-4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-6-(3-(isopropylamino)-3-oxoprop-1-en-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6m</b>)</h4><div class="NLM_p last">Treatment of <b>18f</b> as outlined in general procedures I and H and purification by preparative HPLC (with a linear gradient of 30% B to 45% B over 30 min) provided <b>6l</b> and <b>6m</b> as white fluffy solids. For <b>6l</b>: retention time = 23.0 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.62 (t, <i>J</i> = 5.7 Hz, 1H), 10.46 (brs, 1H), 8.50 (d, <i>J</i> = 1.8 Hz, 1H), 8.47 (s, 1H), 7.61 (d, <i>J</i> = 7.5 Hz, 1H), 7.36 (dd, <i>J</i> = 15.4, 8.7 Hz, 1H), 7.20–7.14 (m, 1H), 7.02–6.98 (m, 1H), 4.45 (d, <i>J</i> = 5.7 Hz, 2H), 3.76–3.68 (m, 1H), 2.84 (t, <i>J</i> = 7.5 Hz, 2H), 2.36 (t, <i>J</i> = 7.6 Hz, 2H), 0.90 (d, <i>J</i> = 6.6 Hz, 6H). ESI-MS <i>m</i>/<i>z</i>: 460.2 (MH<sup>+</sup>). HRMS calcd C<sub>22</sub>H<sub>24</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub> [MH<sup>+</sup>], 460.1791; found, 460.1772. For <b>6m</b>: retention time = 27.9 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (s, 1H), 10.53 (t, <i>J</i> = 5.8 Hz, 1H), 8.84 (s, 1H), 8.82 (s, 1H), 8.01 (d, <i>J</i> = 7.7 Hz, 1H), 7.41 (d, <i>J</i> = 15.8 Hz, 1H), 7.37–7.33 (m, 1H), 7.20–7.15 (m, 1H), 7.00 (td, <i>J</i> = 8.6, 2.3 Hz, 1H), 6.66 (d, <i>J</i> = 15.9 Hz, 1H), 4.46 (d, <i>J</i> = 5.7 Hz, 2H), 3.91 (td, <i>J</i> = 13.3, 6.6 Hz, 1H), 1.05 (d, <i>J</i> = 6.6 Hz, 6H). ESI-MS <i>m</i>/<i>z</i>: 458.2 (MH<sup>+</sup>). HRMS calcd C<sub>22</sub>H<sub>22</sub>F<sub>2</sub>N<sub>5</sub>O<sub>4</sub> [MH<sup>+</sup>], 458.1634; found, 458.1628.</div></div><div id="sec4_2_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-6-(3-((2-hydroxyethyl)amino)-3-oxopropyl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6n</b>) and (<i>E</i>)-4-Amino-<i>N</i>-(2,4-difluorobenzyl)-1-hydroxy-6-(3-((2-hydroxyethyl)amino)-3-oxoprop-1-en-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6o</b>)</h4><div class="NLM_p last">Treatment of <b>18g</b> as outlined in general procedures I and H and purification by preparative HPLC (with a linear gradient of 20% B to 50% B over 30 min) provided <b>6n</b> and <b>6o</b> as white fluffy solids. For <b>6n</b>: retention time = 22.1 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.56 (d, <i>J</i> = 1.9 Hz, 1H), 8.49 (d, <i>J</i> = 1.8 Hz, 1H), 7.42 (dd, <i>J</i> = 15.4, 8.6 Hz, 1H), 7.25–7.19 (m, 1H), 7.06 (dd, <i>J</i> = 9.4, 7.7 Hz, 1H), 4.50 (s, 2H), 3.32 (t, <i>J</i> = 6.1 Hz, 2H), 3.08 (t, <i>J</i> = 6.1 Hz, 2H), 2.92 (t, <i>J</i> = 7.5 Hz, 2H), 2.48 (t, <i>J</i> = 7.5 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 462.1 (MH<sup>+</sup>), 484.1 (MNa<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>22</sub>F<sub>2</sub>N<sub>5</sub>O<sub>5</sub> [MH<sup>+</sup>], 462.1584; found, 462.1562. For <b>6o</b>: retention time = 23.9 min. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.63 (s, 1H), 10.53 (t, <i>J</i> = 5.8 Hz, 1H), 8.85 (s, 1H), 8.83 (s, 1H), 8.09 (t, <i>J</i> = 5.8 Hz, 1H), 7.43 (d, <i>J</i> = 15.8 Hz, 1H), 7.36 (dd, <i>J</i> = 15.4, 8.6 Hz, 1H), 7.20–7.15 (m, 1H), 7.03–6.98 (m, 1H), 6.73 (d, <i>J</i> = 15.9 Hz, 1H), 4.65 (brs, 1H), 4.46 (d, <i>J</i> = 5.6 Hz, 2H), 3.40 (brs, 2H), 3.20 (dd, <i>J</i> = 11.9, 6.0 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 460.1 (MH<sup>+</sup>), 482.1 (MNa<sup>+</sup>). HRMS calcd C<sub>21</sub>H<sub>20</sub>F<sub>2</sub>N<sub>5</sub>O<sub>5</sub> [MH<sup>+</sup>], 460.1427; found, 460.1421.</div></div><div id="sec4_2_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> Methyl 3-(5-Amino-6-((2,4-difluorobenzyl)carbamoyl)-8-hydroxy-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)propanoate (<b>6p</b>)</h4><div class="NLM_p last">Treatment of <b>19g</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 30% B to 65% B over 30 min; retention time = 23.4 min) provided <b>6p</b> as a white fluffy solid (72% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.68 (t, <i>J</i> = 5.8 Hz, 1H), 8.62 (d, <i>J</i> = 1.4 Hz, 1H), 8.58 (s, 1H), 7.42 (dd, <i>J</i> = 15.3, 8.6 Hz, 1H), 7.26–7.22 (m, 1H), 7.07 (t, <i>J</i> = 8.5 Hz, 1H), 4.51 (d, <i>J</i> = 5.8 Hz, 2H), 3.58 (s, 3H), 2.95 (t, <i>J</i> = 7.5 Hz, 2H), 2.77 (t, <i>J</i> = 7.6 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 433.1 (MH<sup>+</sup>). HRMS calcd C<sub>20</sub>H<sub>19</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 433.1318; found, 433.1311.</div></div><div id="sec4_2_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> Methyl 2-((3-((2,4-Difluorobenzyl)carbamoyl)-1-hydroxy-6-(5-hydroxypentyl)-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>6q</b>)</h4><div class="NLM_p last">Treatment of <b>18i</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 30% B to 60% B over 30 min; retention time = 20.5 min) provided <b>6q</b> as a white fluffy solid (82% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.22 (bs, 1H), 10.05 (bs, 1H), 8.48 (s, 1H), 8.22 (s, 1H), 7.52 (dd, <i>J</i> = 15.6, 8.6 Hz, 1H), 7.24–7.11 (m, 1H), 7.00 (t, <i>J</i> = 8.5 Hz, 1H), 4.41 (t, <i>J</i> = 5.5 Hz, 4H), 3.59 (s, 3H), 3.32 (t, <i>J</i> = 6.5 Hz, 4H), 2.62 (t, <i>J</i> = 7.6 Hz, 2H), 1.61–1.47 (m, 2H), 1.45–1.33 (m, 2H), 1.32–1.18 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 505.2 (MH<sup>+</sup>). HRMS calcd C<sub>24</sub>H<sub>27</sub>F<sub>2</sub>N<sub>4</sub>O<sub>6</sub> [MH<sup>+</sup>], 505.1893; found, 505.1889.</div></div><div id="sec4_2_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(2,4-Difluorobenzyl)-1-hydroxy-4-((2-hydroxyethyl)amino)-6-(5-hydroxypentyl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>6r</b>)</h4><div class="NLM_p last">Treatment of <b>18j</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 25% B to 40% B over 30 min; retention time = 24.5 min) provided <b>6r</b> as a white fluffy solid (22% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.98 (s, 1H), 10.52 (s, 1H), 8.54 (s, 1H), 8.37 (s, 1H), 7.50–7.47 (m, 1H), 7.24 (t, <i>J</i> = 10.0 Hz, 1H), 7.07 (t, <i>J</i> = 8.4 Hz, 1H), 4.50 (d, <i>J</i> = 5.6 Hz, 2H), 3.70 (bs, 2H), 3.61 (bs, 2H), 3.38 (t, <i>J</i> = 6.5 Hz, 2H), 2.69 (t, <i>J</i> = 7.4 Hz, 2H), 1.63–1.60 (m, 2H), 1.46–1.44 (m, 2H), 1.33–1.24 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 477.2 (MH<sup>+</sup>). HRMS calcd C<sub>23</sub>H<sub>26</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 477.1944; found, 477.1944.</div></div><div id="sec4_2_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 5-(6-((2,4-Difluorobenzyl)carbamoyl)-8-hydroxy-5-((2-hydroxyethyl)amino)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)pentyl Acetate (<b>6s</b>)</h4><div class="NLM_p last">The mixture of <b>18j</b> (74 mg, 0.13 mmol), triethylamine (109 μL, 0.79 mmol), and acetic anhydride (37 μL, 0.40 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was stirred at room temperature (3 h). The crude mixture was purified by silica gel chromatography to provide 5-(8-(benzyloxy)-6-((2,4-difluorobenzyl)carbamoyl)-5-((2-hydroxyethyl)amino)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)pent-4-yn-1-yl acetate as a colorless oil (31 mg, 39% yield). [<sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.00 (brs, 1H), 10.69 (t, <i>J</i> = 5.6 Hz, 1H), 8.31 (s, 1H), 7.67 (d, <i>J</i> = 6.9 Hz, 2H), 7.41–7.36 (m, 4H), 6.86–6.80 (m, 2H), 5.24 (s, 2H), 4.62 (d, <i>J</i> = 5.6 Hz, 2H), 4.35 (t, <i>J</i> = 5.4 Hz, 2H), 3.94 (dd, <i>J</i> = 10.6, 5.3 Hz, 2H), 3.83 (t, <i>J</i> = 5.9 Hz, 2H), 2.60 (t, <i>J</i> = 7.0 Hz, 2H), 2.11 (s, 3H), 1.90 (p, <i>J</i> = 6.5 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 605.2 (MH<sup>+</sup>).] This compound was treated as outlined in general procedure I and purified by preparative HPLC (linear gradient of 30% B to 55% B over 30 min; retention time = 24.1 min) to provide <b>6s</b> as a white fluffy solid (12% yield, two steps from <b>18j</b>). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.56 (brs, 1H), 10.29 (brs, 1H), 8.53 (s, 1H), 8.30 (s, 1H), 7.53–7.48 (m, 1H), 7.22 (t, <i>J</i> = 9.9 Hz, 1H), 7.07–7.03 (m, 1H), 4.48 (d, <i>J</i> = 5.8 Hz, 2H), 4.17–4.15 (m, 2H), 3.81 (d, <i>J</i> = 4.5 Hz, 2H), 3.38–3.36 (m, 2H), 2.68 (t, <i>J</i> = 7.4 Hz, 2H), 1.97 (s, 3H), 1.64–1.57 (m, 2H), 1.47–1.41 (m, 2H), 1.33–1.29 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 519.2 (MH<sup>+</sup>), 541.1 (MNa<sup>+</sup>).</div></div><div id="sec4_2_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Methyl 3-(6-((2,4-Difluorobenzyl)carbamoyl)-8-hydroxy-5-((2-hydroxyethyl)amino)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)propanoate (<b>6t</b>)</h4><div class="NLM_p last">Treatment of <b>18k</b> as outlined in general procedure I and purification by preparative HPLC (linear gradient of 30% B to 50% B over 30 min; retention time = 21.0 min) provided <b>6t</b> as a white fluffy solid (20% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.09 (bs, 1H), 10.54 (t, <i>J</i> = 5.7 Hz, 1H), 8.57 (d, <i>J</i> = 1.8 Hz, 1H), 8.43 (d, <i>J</i> = 1.7 Hz, 1H), 7.46 (dd, <i>J</i> = 15.4, 8.6 Hz, 1H), 7.26–7.21 (m, 1H), 7.09–7.05 (m, 1H), 4.50 (d, <i>J</i> = 5.7 Hz, 2H), 3.71 (s, 2H), 3.61 (t, <i>J</i> = 5.1 Hz, 2H), 3.58 (s, 3H), 2.96 (t, <i>J</i> = 7.4 Hz, 2H), 2.73 (t, <i>J</i> = 7.4 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 477.1 (MH<sup>+</sup>). HRMS calcd C<sub>23</sub>H<sub>26</sub>F<sub>2</sub>N<sub>4</sub>O<sub>5</sub> [MH<sup>+</sup>], 477.1580; found, 477.1581.</div></div></div><div id="sec4_2_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Methyl 2-Chloro-6-methoxynicotinate (<b>8</b>)</h3><div class="NLM_p last">Commercially available 2,6-dichloronicotinic acid <b>7</b> (3.1 g, 16 mmol) was added to the mixture of potassium <i>t</i>-butoxide (5.7 g, 48 mmol) in methanol (75 mL). The reaction mixture was stirred (65 °C, 24 h). The reaction mixture was concentrated and acidified using concentrated aqueous HCl. The crude mixture was filtered. The formed solid was collected to provide 2-chloro-6-methoxynicotinic acid as a white solid (2.8 g, 94% yield).<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> [<sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.33 (bs, 1H), 8.19 (d, <i>J</i> = 8.5 Hz, 1H), 6.92 (d, <i>J</i> = 8.5 Hz, 1H), 3.92 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.52, 164.54, 147.69, 143.80, 120.02, 109.82, 54.86. ESI-MS <i>m</i>/<i>z</i>: 188.0 (MH<sup>+</sup>).] 2-Chloro-6-methoxynicotinic acid (2.8 g, 15 mmol) was suspended in thionyl chloride (20 mL). The suspension was stirred and refluxed (3 h). The reaction mixture was concentrated. The residue was mixed with methanol (20 mL). The mixture was refluxed (3 h) and concentrated. The crude residue was purified by silica gel chromatography to provide <b>8</b> as a colorless oil (2.5 g, 81% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.89 (d, <i>J</i> = 8.5 Hz, 1H), 6.48 (d, <i>J</i> = 8.5 Hz, 1H), 3.77 (s, 3H), 3.70 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 164.56, 164.25, 148.96, 142.60, 118.11, 108.96, 54.30, 52.09.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> General Procedure A for the Synthesis of Methyl 2-((Benzyloxy)amino)-nicotinates (<b>9</b> and <b>10a</b>)</h3><div class="NLM_p">2-Chloronicotinates (<b>7</b> and <b>8</b>) (12 mmol) were mixed with <i>O</i>-benzylhydroxylamine (48 mmol) and DIEA (36 mmol). The reaction mixture was stirred (110 °C, 18 h). The crude mixture was purified by silica gel chromatography to provide methyl 2-((benzyloxy)amino)-nicotinates (<b>9</b> and <b>10a</b>).</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Methyl 2-((Benzyloxy)amino)-6-chloronicotinate (<b>9</b>)</h4><div class="NLM_p last">Treatment of methyl 2,6-dichloronicotinate, which was prepared from commercially available 2,6-dichloronicotinic acid <b>7</b> in 91% yield based on known method.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a><sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.10 (d, <i>J</i> = 8.1 Hz, 1H), 7.31 (d, <i>J</i> = 8.1 Hz, 1H), 3.90 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 163.95, 152.88, 149.74, 142.46, 125.12, 122.79, 52.94.], as outlined in general procedure A provided <b>9</b> as a colorless oil (55% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.08 (s, 1H), 8.07–7.97 (m, 1H), 7.52–7.46 (m, 2H), 7.41–7.25 (m, 3H), 6.77–6.68 (m, 1H), 5.03 (s, 2H), 3.80 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.13, 159.29, 155.57, 142.02, 135.92, 129.17(2C), 128.45(3C), 114.10, 104.88, 78.29, 52.26. ESI-MS <i>m</i>/<i>z</i>: 293.1 (MH<sup>+</sup>).</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Methyl 2-((Benzyloxy)amino)-6-methoxynicotinate (<b>10a</b>)</h4><div class="NLM_p last">Treatment of <b>8</b> as outlined in general procedure A provided <b>10a</b> as a colorless oil (51% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.22 (s, 1H), 7.99 (d, <i>J</i> = 8.6 Hz, 1H), 7.47–7.45 (m, 2H), 7.38–7.29 (m, 3H), 6.16 (d, <i>J</i> = 8.6 Hz, 1H), 5.09 (s, 2H), 3.99 (s, 3H), 3.78 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.82, 166.70, 160.47, 142.09, 136.36, 128.81 (2C), 128.42 (2C), 128.26, 101.26, 98.90, 78.39, 53.74, 51.67.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> General Procedure B for the Synthesis of 6-Substituted Methyl 2-((Benzyloxy)amino)-nicotinates (<b>10b</b> and <b>10c</b>)</h3><div class="NLM_p">Methyl 2-((benzyloxy)amino)-6-chloronicotinate (<b>9</b>) (1 mmol) was dissolved in DMF (1 mL). Piperidine or morphiline (4 mmol) was added. The mixture was stirred and heated (80 °C, 1 h). The crude mixture was purified by silica gel chromatography to provide 6-substituted methyl 2-((benzyloxy)amino)-nicotinates (<b>10b</b> and <b>10c</b>).</div><div id="sec4_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Methyl 2-((Benzyloxy)amino)-6-(piperidin-1-yl)nicotinate (<b>10b</b>)</h4><div class="NLM_p last">Treatment of <b>9</b> with piperidine as outlined in general procedure B provided <b>10b</b> as a colorless oil (88% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.12 (s, 1H), 7.85 (dd, <i>J</i> = 9.0, 0.7 Hz, 1H), 7.46–7.44 (m, 2H), 7.36–7.30 (m, 4H), 6.05 (d, <i>J</i> = 9.0 Hz, 1H), 5.07 (s, 2H), 3.74 (d, <i>J</i> = 0.7 Hz, 3H), 3.39–3.66 (m, 4H), 1.67–1.65 (m, 2H), 1.63–1.61 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.99, 160.55, 159.92, 140.74, 136.79, 128.74 (2C), 128.60, 128.42, 128.33 (2C), 128.05, 97.10, 94.46, 78.11, 51.22, 45.70, 25.66, 24.74. ESI-MS <i>m</i>/<i>z</i>: 342.2 (MH<sup>+</sup>).</div></div><div id="sec4_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Methyl 2-((Benzyloxy)amino)-6-morpholinonicotinate (<b>10c</b>)</h4><div class="NLM_p last">Treatment of <b>9</b> with morpholine as outlined in general procedure B provided <b>10c</b> as a colorless oil (99% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.13 (s, 1H), 7.88 (d, <i>J</i> = 8.8 Hz, 1H), 7.41 (dd, <i>J</i> = 8.0, 1.5 Hz, 2H), 7.34–7.23 (m, 3H), 5.99 (d, <i>J</i> = 8.9 Hz, 1H), 5.03 (s, 2H), 3.73 (s, 3H), 3.73–3.71 (m, 4H), 3.63–3.60 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 166.86, 160.19 (2C), 141.05, 136.70, 128.63 (2C), 128.36 (2C), 128.11, 96.84, 95.76, 78.11, 66.57 (2C), 51.36, 44.73 (2C). ESI-MS <i>m</i>/<i>z</i>: 344.2 (MH<sup>+</sup>).</div></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> General Procedure C for the Synthesis of Methyl Nicotinates (<b>11a</b>–<b>c</b>)</h3><div class="NLM_p">Methyl 3-chloro-3-oxopropanoate (12 mmol) was added dropwise to a solution of methyl 2-((benzyloxy)amino)-nicotinates (<b>10a</b>–<b>c</b>) (6 mmol) and triethylamine (12 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (40 mL). The mixture was stirred at room temperature (1 h). The crude mixture was filtered and the filtrate was concentrated. The crude residue was purified by silica gel chromatography to provide methyl nicotinates (<b>11a</b>–<b>c</b>).</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Methyl 2-(<i>N</i>-(Benzyloxy)-3-methoxy-3-oxopropanamido)-6-(piperidin-1-yl)nicotinate (<b>11b</b>)</h4><div class="NLM_p last">Treatment of <b>10b</b> as outlined in general procedure C provided <b>11b</b> as a yellow oil (76% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 7.93 (d, <i>J</i> = 8.9 Hz, 1H), 7.30 (d, <i>J</i> = 3.8 Hz, 2H), 7.24–7.23 (m, 3H), 6.49 (d, <i>J</i> = 9.0 Hz, 1H), 4.99 (s, 2H), 3.90 (s, 1H), 3.80 (s, 1H), 3.72 (s, 3H), 3.62 (s, 3H), 3.55–3.56 (m, 4H), 1.61–1.58 (m, 2H), 1.55–1.54 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 442.2 (MH<sup>+</sup>).</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Methyl 2-(<i>N</i>-(Benzyloxy)-3-methoxy-3-oxopropanamido)-6-morpholinonicotinate (<b>11c</b>)</h4><div class="NLM_p last">Treatment of <b>10c</b> as outlined in general procedure C provided <b>11c</b> as a yellow oil (79% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.04 (d, <i>J</i> = 8.8 Hz, 1H), 7.32–7.28 (m, 5H), 6.52 (dd, <i>J</i> = 8.9, 0.9 Hz, 1H), 5.01 (s, 2H), 3.85–3.83 (m, 1H), 3.78 (s, 3H), 3.77–3.75 (m, 4H), 3.72–3.71 (m, 1H), 3.68 (s, 3H), 3.60–3.57 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 444.2 (MH<sup>+</sup>).</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> General Procedure D for the Synthesis of Methyl Carboxylates (<b>12a</b>–<b>c</b>)</h3><div class="NLM_p">A solution of sodium methanolate (16 mmol, 25% in methanol) was added to a solution of methyl nicotinates (<b>11a</b>–<b>c</b>) (6 mmol) in methanol (4 mL). The mixture was stirred at room temperature (18 h). The reaction mixture was brought to pH 4 by the addition of aqueous HCl (2N) and stirred (0 °C, 15 min). The crude suspension was filtered, and the solid was collected to provide methyl carboxylates (<b>12a</b>–<b>c</b>).</div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Methyl 1-(Benzyloxy)-4-hydroxy-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxylate (<b>12a</b>)</h4><div class="NLM_p last">Treatment of <b>10a</b> as outlined in general procedure C and D provided <b>12a</b> as a white solid (54% yield, two steps). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.20 (d, <i>J</i> = 8.7 Hz, 1H), 7.62–7.59 (m, 2H), 7.35–7.32 (m, 3H), 6.61 (d, <i>J</i> = 8.7 Hz, 1H), 5.21 (s, 2H), 4.05 (s, 3H), 4.00 (s, 3H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 172.60, 170.32, 167.69, 156.93, 149.75, 136.69, 134.30, 129.68 (2C), 128.94, 128.42 (2C), 107.37, 102.64, 96.48, 77.90, 54.52, 53.01.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> Methyl 1-(Benzyloxy)-4-hydroxy-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxylate (<b>12b</b>)</h4><div class="NLM_p last">Treatment of <b>11b</b> as outlined in general procedure D provided <b>12b</b> as a white solid in 62% yield. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 13.91 (s, 1H), 8.00 (d, <i>J</i> = 9.1 Hz, 1H), 7.61 (dd, <i>J</i> = 7.4, 1.7 Hz, 2H), 7.34 (d, <i>J</i> = 7.0 Hz, 3H), 6.49 (d, <i>J</i> = 9.1 Hz, 1H), 5.18 (s, 2H), 3.98 (s, 3H), 3.75–3.73 (m, 4H), 1.71–1.70 (m, 2H), 1.65–1.63 (m, 4H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>) δ 172.95, 170.00, 160.00, 157.59, 150.44, 135.06, 134.75, 129.66(2C), 128.71, 128.33(2C), 102.62, 98.44, 94.11, 77.47 (2C), 52.65, 46.08, 25.76 (2C), 24.56. ESI-MS <i>m</i>/<i>z</i>: 410.2 (MH<sup>+</sup>).</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> Methyl 1-(Benzyloxy)-4-hydroxy-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxylate (<b>12c</b>)</h4><div class="NLM_p last">Treatment of <b>11c</b> as outlined in general procedure D provided <b>12c</b> as a white solid (84% yield). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.97 (d, <i>J</i> = 9.3 Hz, 1H), 7.53–7.51 (m, 2H), 7.39–7.33 (m, 4H), 6.60 (d, <i>J</i> = 9.3 Hz, 1H), 5.00 (s, 2H), 3.72–3.54 (m, 11H). ESI-MS <i>m</i>/<i>z</i>: 412.2 (MH<sup>+</sup>).</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> General Procedure E for the Synthesis of Carboxyamides (<b>13a</b>–<b>c</b>)</h3><div class="NLM_p">A solution of methyl carboxylates (<b>12a</b>–<b>c</b>) (0.3 mmol) and (2,4-difluorophenyl)methanamine (1.5 mmol) in DMF (1 mL) was heated (140 °C, 2 h) in microwave reactor. The crude mixture was purified by silica gel chromatography and recrystallized from methanol to provide carboxamides (<b>13a</b>–<b>c</b>).</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-hydroxy-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>13a</b>)</h4><div class="NLM_p last">Treatment of <b>12a</b> as outlined in general procedure E provided <b>13a</b> as a white solid (45% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.24 (t, <i>J</i> = 5.6 Hz, 1H), 8.22 (dd, <i>J</i> = 8.7, 0.4 Hz, 1H), 7.56–7.53 (m, 2H), 7.35–7.29 (m, 4H), 6.81–6.73 (m, 2H), 6.62 (dd, <i>J</i> = 8.7, 0.4 Hz, 1H), 5.20 (d, <i>J</i> = 0.6 Hz, 2H), 4.58 (d, <i>J</i> = 6.0 Hz, 2H), 4.02 (d, <i>J</i> = 0.5 Hz, 3H). ESI-MS <i>m</i>/<i>z</i>: 468.2 (MH<sup>+</sup>).</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-hydroxy-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>13b</b>)</h4><div class="NLM_p last">Treatment of <b>12b</b> as outlined in general procedure E provided <b>13b</b> as a white solid (90% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.30 (t, <i>J</i> = 6.0 Hz, 1H), 10.03 (s, 1H), 8.05 (dd, <i>J</i> = 9.1, 0.7 Hz, 1H), 7.58–7.56 (m, 2H), 7.38–7.33 (m, 4H), 6.85–6.76 (m, 2H), 6.54 (dd, <i>J</i> = 9.1, 0.7 Hz, 1H), 5.20 (s, 2H), 4.60 (d, <i>J</i> = 6.0 Hz, 2H), 3.76–3.74 (m, 4H), 1.71–1.69 (m, 2H), 1.67–1.61 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 521.2 (MH<sup>+</sup>).</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-hydroxy-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>13c</b>)</h4><div class="NLM_p last">Treatment of <b>12c</b> as outlined in general procedure E provided <b>13c</b> as a white solid (89% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.22 (bs, 1H), 8.08 (d, <i>J</i> = 9.1 Hz, 1H), 7.50–7.48 (m, 2H), 7.33–7.24 (m, 4H), 6.79–6.69 (m, 2H), 6.48 (d, <i>J</i> = 9.1 Hz, 1H), 5.15 (s, 2H), 4.56 (d, <i>J</i> = 6.2 Hz, 2H), 3.73–3.66 (m, 8H). ESI-MS <i>m</i>/<i>z</i>: 523.2 (MH<sup>+</sup>).</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> General Procedure F for the Synthesis of <i>p</i>-Methylbenzenesulfonates (<b>14a</b>–<b>c</b>)</h3><div class="NLM_p">Compound 4-hydroxyl analogues (<b>13a</b>–<b>c</b>) (0.5 mmol) were dissolved in CH<sub>3</sub>CN (3 mL). DIEA (2.8 mmol), CH<sub>2</sub>Cl<sub>2</sub> (2 mL), and <i>p</i>-methylbenzene-1-sulfonyl chloride (1.4 mmol) were added. The reaction mixture was stirred at room temperature (18 h). The mixture crude was purified by silica gel chromatography to provide <i>p</i>-methylbenzenesulfonates (<b>14a</b>–<b>c</b>).</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl 4-Methylbenzenesulfonate (<b>14a</b>)</h4><div class="NLM_p last">Treatment of <b>13a</b> as outlined in general procedure F provided <b>14a</b> as a colorless oil (68% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.53 (t, <i>J</i> = 5.9 Hz, 1H), 8.01 (d, <i>J</i> = 8.8 Hz, 1H), 7.87 (d, <i>J</i> = 8.3 Hz, 2H), 7.58–7.56 (m, 2H), 7.46–7.44 (m, 1H), 7.37–7.35 (m, 3H), 7.32 (dd, <i>J</i> = 8.6, 0.7 Hz, 2H), 6.83–6.74 (m, 2H), 6.64 (d, <i>J</i> = 8.8 Hz, 1H), 5.25 (s, 2H), 4.48–4.46 (m, 2H), 4.06 (s, 3H), 2.43 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 622.1(MH<sup>+</sup>).</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridin-4-yl 4-Methylbenzenesulfonate (<b>14b</b>)</h4><div class="NLM_p last">Treatment of <b>13b</b> as outlined in general procedure F provided <b>14b</b> as a colorless oil (53% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.83 (t, <i>J</i> = 5.9 Hz, 1H), 7.88 (dd, <i>J</i> = 8.8, 3.2 Hz, 3H), 7.58–7.55 (m, 2H), 7.44–7.35 (m, 4H), 7.31–7.29 (m, 2H), 6.82–6.73 (m, 2H), 6.57 (d, <i>J</i> = 9.3 Hz, 1H), 5.22 (s, 2H), 4.44 (s, 2H), 3.76 (bs, 4H), 2.42 (s, 3H), 1.72–1.69 (m, 2H), 1.66–1.64 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 675.2 (MH<sup>+</sup>).</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl 4-Methylbenzenesulfonate (<b>14c</b>)</h4><div class="NLM_p last">Treatment of <b>13c</b> as outlined in general procedure F provided <b>14c</b> as a colorless oil (27% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 8.71 (t, <i>J</i> = 5.9 Hz, 1H), 8.19–8.18 (m, 2H), 7.94 (d, <i>J</i> = 9.2 Hz, 1H), 7.86 (d, <i>J</i> = 8.4 Hz, 2H), 7.36–7.50 (m, 2H), 7.36–7.28 (m, 3H), 6.82–6.74 (m, 3H), 6.55 (dd, <i>J</i> = 9.2, 0.7 Hz, 1H), 5.21 (s, 2H), 4.43–4.41 (m, 2H), 3.78–3.73 (m, 8H), 2.42 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 677.2 (MH<sup>+</sup>).</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i66" class="anchor-spacer"></div><h3 class="article-section__title" id="_i66"> General Procedure G for the Synthesis of 4-Substituted Amines (<b>15a</b>–<b>f</b>, <b>17d</b>, and <b>17e</b>)</h3><div class="NLM_p">A solution of <i>p</i>-methylbenzenesulfonates (<b>14a</b>–<b>c</b> and <b>17b</b>) (0.2 mmol), DIPEA (2 mmol), and methyl 2-aminoacetate hydrochloride or 2,4-dimethoxyphenyl)methanamine (1 mmol) in DMF (2 mL) was stirred and heated (50 °C, 1 h). The crude mixture was purified by silica gel chromatography to provide 4-substituted amines (<b>15a</b>–<b>f</b>, <b>17d</b>, and <b>17e</b>).</div><div id="sec4_9_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> Methyl 2-((1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>15a</b>)</h4><div class="NLM_p last">Treatment of <b>14a</b> with methyl 2-aminoacetate hydrochloride as outlined in general procedure G provided <b>15a</b> as a white solid (91% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.20 (t, <i>J</i> = 5.5 Hz, 1H), 10.70 (t, <i>J</i> = 5.7 Hz, 1H), 8.02 (d, <i>J</i> = 8.9 Hz, 1H), 7.59–7.56 (m, 2H), 7.41–7.32 (m, 4H), 6.82–6.74 (m, 2H), 6.55 (d, <i>J</i> = 8.9 Hz, 1H), 5.20 (s, 2H), 4.61 (d, <i>J</i> = 5.8 Hz, 2H), 4.35 (d, <i>J</i> = 5.5 Hz, 2H), 4.02 (s, 3H), 3.76 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 539.2 (MH<sup>+</sup>).</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> Methyl 2-((1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>15b</b>)</h4><div class="NLM_p last">Treatment of <b>14b</b> with methyl 2-aminoacetate hydrochloride as outlined in general procedure G provided <b>15b</b> as a white solid (80% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.99 (t, <i>J</i> = 5.7 Hz, 1H), 10.78 (t, <i>J</i> = 5.8 Hz, 1H), 7.81 (d, <i>J</i> = 9.3 Hz, 1H), 7.58–7.55 (m, 2H), 7.41–7.31 (m, 4H), 6.81–6.72 (m, 2H), 6.40 (d, <i>J</i> = 9.3 Hz, 1H), 5.18 (s, 2H), 4.60 (d, <i>J</i> = 5.7 Hz, 2H), 4.32 (d, <i>J</i> = 5.6 Hz, 2H), 3.76 (s, 3H), 3.71–3.69 (m, 4H), 1.71–1.67 (m, 2H), 1.64–1.59 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 592.2 (MH<sup>+</sup>).</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> Methyl 2-((1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>15c</b>)</h4><div class="NLM_p last">Treatment of <b>14c</b> with methyl 2-aminoacetate hydrochloride as outlined in general procedure G provided <b>15c</b> as a white solid (63% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.06 (t, <i>J</i> = 5.6 Hz, 1H), 10.73 (t, <i>J</i> = 5.8 Hz, 1H), 7.88 (dd, <i>J</i> = 9.2, 0.8 Hz, 1H), 7.53–7.50 (m, 2H), 7.41–7.27 (m, 3H), 6.81–6.73 (m, 3H), 6.38 (dd, <i>J</i> = 9.3, 0.8 Hz, 1H), 5.16 (s, 2H), 4.60 (d, <i>J</i> = 5.8 Hz, 2H), 4.32 (d, <i>J</i> = 5.5 Hz, 2H), 3.76 (s, 3H), 3.75–3.73 (m, 4H), 3.68–3.65 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 594.2 (MH<sup>+</sup>).</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>15d</b>)</h4><div class="NLM_p last">Treatment of <b>14a</b> with (2,4-dimethoxyphenyl)methanamine as outlined in general procedure G provided <b>15d</b> as a white solid which was directly used in next step. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.88 (t, <i>J</i> = 6.2 Hz, 1H), 10.77 (t, <i>J</i> = 5.8 Hz, 1H), 8.10 (dd, <i>J</i> = 8.9, 0.7 Hz, 1H), 7.61–7.59 (m, 2H), 7.38–7.31 (m, 4H), 7.24–7.21 (m, 1H), 6.81–6.73 (m, 2H), 6.47–6.42 (m, 3H), 5.24 (s, 2H), 4.67 (d, <i>J</i> = 6.1 Hz, 2H), 4.56 (d, <i>J</i> = 5.7 Hz, 2H), 4.02 (s, 3H), 3.77 (s, 3H), 3.75 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 617.2 (MH<sup>+</sup>).</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>15e</b>)</h4><div class="NLM_p last">Treatment of <b>14b</b> with (2,4-dimethoxyphenyl)methanamine as outlined in general procedure G provided <b>15e</b> as a white solid (65% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.71 (t, <i>J</i> = 6.2 Hz, 1H), 10.85 (t, <i>J</i> = 5.8 Hz, 1H), 7.88 (d, <i>J</i> = 9.3 Hz, 1H), 7.60–7.57 (m, 2H), 7.37–7.32 (m, 4H), 7.27–7.24 (m, 1H), 6.81–6.72 (m, 2H), 6.44–6.42 (m, 2H), 6.30 (d, <i>J</i> = 9.3 Hz, 1H), 5.20 (s, 2H), 4.64 (d, <i>J</i> = 6.2 Hz, 2H), 4.55 (d, <i>J</i> = 6.1 Hz, 2H), 3.77 (s, 3H), 3.76 (s, 3H), 3.70–3.67 (m, 4H), 1.70–1.66 (m, 2H), 1.63–1.58 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 670.3 (MH<sup>+</sup>).</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>15f</b>)</h4><div class="NLM_p last">Treatment of <b>14c</b> with (2,4-dimethoxyphenyl)methanamine as outlined in general procedure G provided <b>15f</b> as a white solid which was directly used in next step. <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 11.73 (t, <i>J</i> = 6.1 Hz, 1H), 10.75 (t, <i>J</i> = 5.8 Hz, 1H), 7.95 (d, <i>J</i> = 9.2 Hz, 1H), 7.52–7.50 (m, 2H), 7.33–7.21 (m, 5H), 6.80–6.71 (m, 4H), 6.30 (d, <i>J</i> = 9.2 Hz, 1H), 5.16 (s, 2H), 4.63 (d, <i>J</i> = 6.1 Hz, 2H), 4.51 (d, <i>J</i> = 5.8 Hz, 2H), 3.76 (s, 3H), 3.74 (s, 3H), 3.73–3.71 (m, 4H), 3.65–3.63 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 672.3 (MH<sup>+</sup>).</div></div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> General Procedure H for the Synthesis of Amine Analogues (<b>16a</b>–<b>c</b>, <b>19b</b>–<b>g</b>)</h3><div class="NLM_p">The 2,4-dimethoxybenzyl protected compounds (<b>15d</b>–<b>f</b> and <b>18a</b>–<b>h</b>) (0.25 mmol) were dissolved in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). TFA (2 mL) was added at room temperature. The reaction mixture was concentrated. The crude residue was purified by silica gel chromatography to provide amine analogues (<b>16a</b>–<b>c</b> and <b>19b</b>–<b>g</b>).</div><div id="sec4_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> 4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-7-methoxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>16a</b>)</h4><div class="NLM_p last">Treatment of <b>15d</b> as outlined in general procedure H provided <b>16a</b> as a colorless oil (72% yield, two steps (general procedures G and H). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.60 (t, <i>J</i> = 5.9 Hz, 1H), 7.85 (dd, <i>J</i> = 8.7, 0.7 Hz, 1H), 7.58 (dd, <i>J</i> = 7.3, 2.2 Hz, 2H), 7.36–7.32 (m, 4H), 6.81–6.74 (m, 2H), 6.58 (dd, <i>J</i> = 8.8, 0.7 Hz, 1H), 5.21 (s, 2H), 4.58 (d, <i>J</i> = 5.8 Hz, 2H), 4.03 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 467.2 (MH<sup>+</sup>).</div></div><div id="sec4_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-2-oxo-7-(piperidin-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>16b</b>)</h4><div class="NLM_p last">Treatment of <b>15e</b> as outlined in general procedure H provided <b>16b</b> as a colorless oil (46% yield, two steps (general procedures G and H). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.62 (t, <i>J</i> = 5.9 Hz, 1H), 7.58 (d, <i>J</i> = 9.1 Hz, 1H), 7.54–7.52 (m, 2H), 7.35–7.26 (m, 4H), 6.78–6.69 (m, 2H), 6.44 (d, <i>J</i> = 9.2 Hz, 1H), 5.15 (s, 2H), 4.54 (d, <i>J</i> = 5.8 Hz, 2H), 3.70–3.65 (m, 4H), 1.66–1.63 (m, 2H), 1.61–1.57 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 520.2 (MH<sup>+</sup>).</div></div><div id="sec4_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> 4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-7-morpholino-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>16c</b>)</h4><div class="NLM_p last">Treatment of <b>15f</b> as outlined in general procedure H provided <b>16c</b> as a colorless oil (32% yield, two steps (general procedures G and H). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 10.61 (t, <i>J</i> = 5.7 Hz, 1H), 7.70 (d, <i>J</i> = 9.0 Hz, 1H), 7.57–7.49 (m, 2H), 7.41–7.28 (m, 3H), 6.83–6.71 (m, 3H), 6.48 (dd, <i>J</i> = 9.1, 0.7 Hz, 1H),5.18 (s, 2H), 4.58 (d, <i>J</i> = 5.8 Hz, 2H), 3.81–3.75 (m, 4H), 3.69–3.67 (m, 4H). ESI-MS <i>m</i>/<i>z</i>: 522.2 (MH<sup>+</sup>).</div></div><div id="sec4_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> Methyl 2-((1-(Benzyloxy)-6-bromo-3-((2,4-difluorobenzyl)carbamoyl)-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>17d</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-3-((2,4-difluorobenzyl)carbamoyl)-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl 4-methylbenzenesulfonate <b>17b</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with methyl glycinate as outlined in general procedure G provided <b>17d</b> as a white solid (74% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.05 (t, <i>J</i> = 6.2 Hz, 1H), 10.61 (t, <i>J</i> = 5.7 Hz, 1H), 8.58 (d, <i>J</i> = 2.1 Hz, 1H), 8.34 (d, <i>J</i> = 2.2 Hz, 1H), 7.57 (dd, <i>J</i> = 7.3, 1.9 Hz, 2H), 7.39–7.22 (m, 4H), 7.17 (dd, <i>J</i> = 8.9, 4.3 Hz, 1H), 6.85–6.63 (m, 2H), 6.47–6.34 (m, 2H), 5.15 (s, 2H), 4.64 (d, <i>J</i> = 6.3 Hz, 2H), 4.52 (d, <i>J</i> = 5.7 Hz, 2H), 3.74 (s, 3H), 3.73 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 587.1, 589.1 (MH<sup>+</sup>).</div></div><div id="sec4_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> 1-(Benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2-hydroxyethyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>17e</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-3-((2,4-difluorobenzyl)carbamoyl)-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl 4-methylbenzenesulfonate <b>17b</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with 2-aminoethan-1-ol as outlined in general procedure G provided <b>17e</b> as a white solid (93% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 11.65 (bs, 1H), 10.65 (bs, 1H), 8.64 (d, <i>J</i> = 1.9 Hz, 1H), 8.47 (d, <i>J</i> = 1.9 Hz, 1H), 7.58 (d, <i>J</i> = 6.1 Hz, 2H), 7.31 (t, <i>J</i> = 7.2 Hz, 4H), 6.79–6.73 (m, 2H), 5.15 (s, 2H), 4.55 (d, <i>J</i> = 5.6 Hz, 2H), 3.88–3.86 (m, 2H), 3.78–3.75 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 559.1, 561.1 (MH<sup>+</sup>).</div></div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> General Procedure J for the Synthesis of 6-Alkylated Analogues (<b>18a</b>, <b>18b</b>, <b>18d</b>, <b>18i</b>, and <b>18j</b>) Using Sonogashira Reaction</h3><div class="NLM_p last">Bromides (<b>17a</b> and <b>17c</b>–<b>e</b>) (0.3 mmol) were mixed with bis(triphenylphosphine)palladium(II) dichloride (9 μmol), DIEA (0.3 mmol), and copper(I) iodide (0.03 mmol) in DMF (1.5 mL) in a reaction tube. The related terminal alkynes (0.45 mmol) were added. The reaction mixture was flushed with argon and sealed with a cap and heated (70 °C, 4 h). The crude mixture was purified by silica gel chromatography to provide 6-alkylated analogue (<b>18a</b>, <b>18b</b>, <b>18d</b>, <b>18i</b>, and <b>18j</b>).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i80" class="anchor-spacer"></div><h3 class="article-section__title" id="_i80"> General Procedure K for the Synthesis of 6-Alkylated Analogues (<b>18c</b>, <b>18e</b>–<b>h</b>, and <b>18k</b>) Using Heck Reaction</h3><div class="NLM_p">The mixture of bromides (<b>17c</b> and <b>17e</b>) (0.2 mmol), alkenes (0.4 mmol), triethylamine (0.6 mmol), diacetoxypalladium (0.04 mmol), and triphenylphosphine (0.08 mmol) in DMF (0.5 mL) was flushed with argon and sealed in a reaction tube. The reaction mixture was microwave-heated (100 °C, 5 h). The crude mixture was purified by silica gel chromatography to provide 6-alkylated analogues (<b>18c</b>, <b>18e</b>–<b>h</b>, and <b>18k</b>).</div><div id="sec4_12_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> 1-(Benzyloxy)-6-(3-cyclohexylprop-1-yn-1-yl)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18a</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available prop-2-yn-1-ylcyclohexane as outlined in general procedure J provided <b>18a</b> as a colorless oil (79% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.02 (t, <i>J</i> = 5.8 Hz, 1H), 10.70 (t, <i>J</i> = 5.7 Hz, 1H), 8.66 (t, <i>J</i> = 1.6 Hz, 1H), 8.32 (s, 1H), 7.68 (d, <i>J</i> = 7.7 Hz, 2H), 7.40–7.36 (m, 4H), 7.25–7.24 (m, 1H), 6.85–6.78 (m, 2H), 6.48–6.45 (m, 2H), 5.25 (s, 2H), 4.75 (d, <i>J</i> = 6.0 Hz, 2H), 4.60 (d, <i>J</i> = 5.6 Hz, 2H), 3.82 (s, 3H), 3.80 (s, 3H), 2.32 (d, <i>J</i> = 6.6 Hz, 2H), 1.85 (d, <i>J</i> = 12.3 Hz, 2H), 1.78–1.68 (m, 3H), 1.57 (ddd, <i>J</i> = 10.9, 7.0, 3.6 Hz, 1H), 1.31–1.16 (m, 3H), 1.05 (qd, <i>J</i> = 12.4, 3.3 Hz, 2H).</div></div><div id="sec4_12_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-6-(4-phenylbut-1-yn-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18b</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available but-3-yn-1-ylbenzene as outlined in general procedure J provided <b>18b</b> as a colorless oil (98% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 11.99 (t, <i>J</i> = 5.6 Hz, 1H), 10.68 (t, <i>J</i> = 5.7 Hz, 1H), 8.62 (s, 1H), 8.31 (s, 1H), 7.68–7.66 (m, 2H), 7.39–7.31 (m, 6H), 7.27–7.21 (m, 4H), 6.85–6.77 (m, 2H), 6.47–6.46 (m, 2H), 5.24 (s, 2H), 4.73 (d, <i>J</i> = 5.8 Hz, 2H), 4.59 (d, <i>J</i> = 5.7 Hz, 2H), 3.81 (s, 3H), 3.76 (s, 3H), 2.92 (d, <i>J</i> = 7.4 Hz, 2H), 2.73 (t, <i>J</i> = 7.4 Hz, 2H).</div></div><div id="sec4_12_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> (<i>E</i>)-1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-6-styryl-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18c</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available styrene as outlined in general procedure K provided <b>18c</b> as a colorless oil (27% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.14 (t, <i>J</i> = 6.7 Hz, 1H), 10.81 (t, <i>J</i> = 5.9 Hz, 1H), 8.71 (s, 1H), 8.44 (s, 1H), 7.71 (d, <i>J</i> = 6.7 Hz, 2H), 7.44–7.38 (m, 10H), 6.99 (d, <i>J</i> = 16.4 Hz, 1H), 6.88–6.81 (m, 2H), 6.58–6.52 (m, 3H), 5.29 (s, 2H), 4.80 (d, <i>J</i> = 6.7 Hz, 2H), 4.65 (d, <i>J</i> = 5.6 Hz, 2H), 3.86 (s, 3H), 3.77 (s, 3H).</div></div><div id="sec4_12_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-6-(3-(2-hydroxyethoxy)prop-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18d</b>)</h4><div class="NLM_p last">Treatment of 1-(Benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with 2-(prop-2-yn-1-yloxy)ethanol as outlined in general procedure J provided <b>18d</b> as a colorless oil (64% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.06 (t, <i>J</i> = 5.9 Hz, 1H), 10.66 (t, <i>J</i> = 5.7 Hz, 1H), 8.70 (d, <i>J</i> = 1.2 Hz, 1H), 8.38 (d, <i>J</i> = 1.4 Hz, 1H), 7.67 (d, <i>J</i> = 6.6 Hz, 2H), 7.41–7.34 (m, 4H), 7.24 (d, <i>J</i> = 7.9 Hz, 1H), 6.85–6.79 (m, 2H), 6.49–6.43 (m, 2H), 5.25 (s, 2H), 4.75 (d, <i>J</i> = 6.0 Hz, 2H), 4.60 (d, <i>J</i> = 5.7 Hz, 2H), 4.45 (s, 2H), 3.82 (s, 3H), 3.84–3.81 (m, 2H), 3.80 (s, 3H), 3.71–3.69 (m, 2H).</div></div><div id="sec4_12_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> (<i>E</i>)-1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-6-(3-(dimethylamino)-3-oxoprop-1-en-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18e</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available <i>N</i>,<i>N</i>-dimethylacrylamide as outlined in general procedure K provided <b>18e</b> as a colorless oil (41% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.17 (t, <i>J</i> = 6.5 Hz, 1H), 10.74 (t, <i>J</i> = 5.7 Hz, 1H), 8.79 (d, <i>J</i> = 1.8 Hz, 1H), 8.39 (d, <i>J</i> = 1.8 Hz, 1H), 7.72–7.67 (m, 3H), 7.60–7.55 (m, 1H), 7.51–7.46 (m, 1H), 7.44–7.38 (m, 4H), 6.88–6.81 (m, 2H), 6.51–6.48 (m, 2H), 5.28 (s, 2H), 4.77 (d, <i>J</i> = 6.6 Hz, 2H), 4.64 (d, <i>J</i> = 5.7 Hz, 2H), 3.83 (s, 3H), 3.83 (s, 3H), 3.07 (s, 3H), 3.00 (s, 3H).</div></div><div id="sec4_12_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> (<i>E</i>)-1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-6-(3-(isopropylamino)-3-oxoprop-1-en-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18f</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available <i>N</i>-isopropylacrylamide as outlined in general procedure K provided <b>18f</b> as a colorless oil (93% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 12.14 (t, <i>J</i> = 6.8 Hz, 1H), 10.75 (t, <i>J</i> = 5.7 Hz, 1H), 8.65 (d, <i>J</i> = 1.8 Hz, 1H), 8.35 (d, <i>J</i> = 1.8 Hz, 1H), 7.69–7.64 (m, 2H), 7.51–7.51 (m, 1H), 7.48–7.42 (m, 3H), 7.40–7.33 (m, 3H), 6.87–6.79 (m, 2H), 6.58 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 6.54 (d, <i>J</i> = 2.2 Hz, 1H), 5.44 (d, J = 15.6 Hz, 1H), 5.25 (s, 2H), 4.71 (d, <i>J</i> = 6.8 Hz, 2H), 4.61 (t, <i>J</i> = 9.6 Hz, 2H), 4.24–4.16 (m, 1H), 3.86 (s, 3H), 3.79 (s, 3H), 1.24 (s, 3H), 1.23 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 698.2 (MH<sup>+</sup>), 720.2 (MNa<sup>+</sup>).</div></div><div id="sec4_12_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> (<i>E</i>)-1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-6-(3-((2-hydroxyethyl)amino)-3-oxoprop-1-en-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18g</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available <i>N</i>-(2-hydroxyethyl)acrylamide as outlined in general procedure K provided <b>18g</b> (84% yield). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>) δ 6.54 (s, 1H), 6.31 (d, <i>J</i> = 17.0 Hz, 1H), 6.16 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 5.69 (d, <i>J</i> = 10.2 Hz, 1H), 3.77 (t, <i>J</i> = 5.0 Hz, 2H), 3.51 (dd, <i>J</i> = 9.8, 4.9 Hz, 2H), 3.31 (d, <i>J</i> = 25.1 Hz, 1H).</div></div><div id="sec4_12_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i88" class="anchor-spacer"></div><h4 class="article-section__title" id="_i88"> (<i>E</i>)-Methyl 3-(8-(Benzyloxy)-6-((2,4-difluorobenzyl)carbamoyl)-5-((2,4-dimethoxybenzyl)amino)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)acrylate (<b>18h</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available methyl acrylate as outlined in general procedure K provided <b>18h</b> as a colorless oil (38% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.35 (t, <i>J</i> = 6.6 Hz, 1H), 10.73 (t, <i>J</i> = 5.6 Hz, 1H), 8.72 (s, 1H), 8.40 (s, 1H), 7.69–7.66 (m, 2H), 7.59 (d, <i>J</i> = 16.1 Hz, 1H), 7.42–7.38 (m, 4H), 7.23 (d, <i>J</i> = 8.0 Hz, 1H), 6.88–6.82 (m, 2H), 6.57 (d, <i>J</i> = 1.9 Hz, 1H), 6.54–6.50 (m, 1H), 6.34–6.32 (d, <i>J</i> = 8.3 Hz, 1H), 6.02 (d, <i>J</i> = 16.0 Hz, 1H), 5.27 (s, 2H), 4.78 (d, <i>J</i> = 6.7 Hz, 2H), 4.64 (d, <i>J</i> = 5.7 Hz, 2H), 3.85 (s, 3H), 3.85 (s, 3H), 3.83 (s, 3H).</div></div><div id="sec4_12_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i89" class="anchor-spacer"></div><h4 class="article-section__title" id="_i89"> Methyl 2-((1-(Benzyloxy)-3-((2,4-difluorobenzyl)carbamoyl)-6-(5-hydroxypent-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridin-4-yl)amino)acetate (<b>18i</b>)</h4><div class="NLM_p last">Treatment of <b>17d</b> with commercially available pent-4-yn-1-ol as outlined in general procedure J provided <b>18i</b> as a colorless oil (45% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 12.33 (s, 1H), 10.65 (t, <i>J</i> = 5.7 Hz, 1H), 8.69 (s, 1H), 8.19 (s, 1H), 7.67–7.65 (m, 2H), 7.43–7.37 (m, 4H), 6.87–6.80 (m, 2H), 5.24 (s, 2H), 4.67 (d, <i>J</i> = 5.6 Hz, 2H), 4.46 (d, <i>J</i> = 4.9 Hz, 2H), 3.84 (s, 3H), 3.85–3.83 (m, 2H), 2.61 (t, <i>J</i> = 7.0 Hz, 2H), 1.91 (t, <i>J</i> = 6.5 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 591.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i90" class="anchor-spacer"></div><h4 class="article-section__title" id="_i90"> 1-(Benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-4-((2-hydroxyethyl)amino)-6-(5-hydroxypent-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>18j</b>)</h4><div class="NLM_p last">Treatment of <b>17e</b> with commercially available pent-4-yn-1-ol as outlined in general procedure J provided <b>18j</b> as a colorless oil (57% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 11.68 (s, 1H), 10.69 (t, <i>J</i> = 5.5 Hz, 1H), 8.61 (d, <i>J</i> = 1.8 Hz, 1H), 8.32 (d, <i>J</i> = 1.8 Hz, 1H), 7.61–7.57 (m, 2H), 7.32–7.29 (m, 4H), 6.79–6.72 (m, 2H), 5.16 (s, 2H), 4.55 (d, <i>J</i> = 5.5 Hz, 2H), 3.86–3.84 (m, 2H), 3.80–3.75 (m, 2H), 2.52 (t, <i>J</i> = 7.0 Hz, 2H), 1.84–1.82 (m, 2H), 1.53–1.49 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 563.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> (<i>E</i>)-Methyl 3-(8-(Benzyloxy)-6-((2,4-difluorobenzyl)carbamoyl)-5-((2-hydroxyethyl)amino)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)acrylate (<b>18k</b>)</h4><div class="NLM_p last">Treatment of <b>17e</b> with commercially available methyl acrylate as outlined in general procedure K provided <b>18k</b> as a colorless oil (52% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 11.82 (s, 1H), 10.71 (s, 1H), 8.84 (s, 1H), 8.67 (s, 1H), 7.74 (d, <i>J</i> = 16.2 Hz, 1H), 7.58–7.55 (m, 2H), 7.41–7.38 (m, 4H), 6.87–6.81 (m, 2H), 6.55 (d, <i>J</i> = 16.1 Hz, 1H), 5.26 (s, 2H), 4.64 (d, <i>J</i> = 5.6 Hz, 2H), 3.85 (s, 3H), 3.60 (t, <i>J</i> = 4.9 Hz, 2H), 2.62–2.60 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 565.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 3-(5-Amino-8-(benzyloxy)-6-((2,4-difluorobenzyl)carbamoyl)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)prop-2-yn-1-yl Acetate (<b>19a</b>)</h4><div class="NLM_p last">Compound 4-amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-6-(3-hydroxyprop-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> (29 mg, 0.06 mmol) was mixed with acetic anhydride (8.4 μL, 0.09 mmol) in pyridine (0.5 mL). The solution was stirred at room temperature (2 h). The crude mixture was purified by silica gel chromatography to provide <b>19a</b> as a colorless oil (24 mg, 76% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.54 (t, <i>J</i> = 5.7 Hz, 1H), 8.77 (d, <i>J</i> = 1.7 Hz, 1H), 8.16 (d, <i>J</i> = 1.7 Hz, 1H), 7.65 (dd, <i>J</i> = 7.5, 1.6 Hz, 2H), 7.42–7.33 (m, 4H), 6.87–6.80 (m, 2H), 5.25 (s, 2H), 4.93 (s, 2H), 4.63 (d, <i>J</i> = 5.7 Hz, 2H), 2.16 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 533.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 4-Amino-1-(benzyloxy)-6-(3-cyclohexylprop-1-yn-1-yl)-<i>N</i>-(2,4-difluorobenzyl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>19b</b>)</h4><div class="NLM_p last">Treatment of <b>18a</b> as outlined in general procedure H provided <b>19b</b> as a colorless oil (91% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.60 (t, <i>J</i> = 5.8 Hz, 1H), 8.74 (d, <i>J</i> = 1.8 Hz, 1H), 8.03 (d, <i>J</i> = 1.9 Hz, 1H), 7.66 (dd, <i>J</i> = 7.5, 1.8 Hz, 2H), 7.42–7.35 (m, 4H), 6.87–6.80 (m, 2H), 5.25 (s, 2H), 4.63 (d, <i>J</i> = 5.8 Hz, 2H), 2.34 (d, <i>J</i> = 6.7 Hz, 2H), 1.88 (dd, <i>J</i> = 13.5, 1.9 Hz, 2H), 1.78–1.74 (m, 2H), 1.70 (ddd, <i>J</i> = 12.6, 5.0, 2.6 Hz, 1H), 1.62–1.55 (m, 1H), 1.34–1.25 (m, 2H), 1.22–1.16 (m, 1H), 1.07 (ddd, <i>J</i> = 24.4, 12.4, 3.3 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 557.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i94" class="anchor-spacer"></div><h4 class="article-section__title" id="_i94"> 4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-2-oxo-6-(4-phenylbut-1-yn-1-yl)-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>19c</b>)</h4><div class="NLM_p last">Treatment of <b>18b</b> as outlined in general procedure H provided <b>19c</b> as a colorless oil (94% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.61 (t, <i>J</i> = 5.8 Hz, 1H), 8.69 (s, 1H), 8.00 (s, 1H), 7.65 (dd, <i>J</i> = 7.1, 1.7 Hz, 2H), 7.39–7.32 (m, 6H), 7.28–7.26 (m, 3H), 6.87–6.80 (m, 2H), 5.24 (s, 2H), 4.63 (d, <i>J</i> = 5.8 Hz, 2H), 2.94 (t, <i>J</i> = 7.4 Hz, 2H), 2.74 (t, <i>J</i> = 7.5 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 565.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> (<i>E</i>)-4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-2-oxo-6-styryl-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>19d</b>)</h4><div class="NLM_p last">Treatment of <b>18c</b> as outlined in general procedure H provided <b>19d</b> as a colorless oil (68% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.65 (t, <i>J</i> = 5.4 Hz, 1H), 8.89 (s, 1H), 8.12 (s, 1H), 7.81 (d, <i>J</i> = 8.3 Hz, 1H), 7.68 (d, <i>J</i> = 7.1 Hz, 2H), 7.63 (d, <i>J</i> = 7.0 Hz, 1H), 7.53 (d, <i>J</i> = 8.1 Hz, 2H), 7.40–7.38 (m,4H), 7.33–7.30 (m, 1H), 7.17 (dd, <i>J</i> = 47.6, 17.5 Hz, 2H), 6.89–6.79 (m, 2H), 5.28 (s, 2H), 4.64 (d, <i>J</i> = 5.2 Hz, 2H). ESI-MS <i>m</i>/<i>z</i>: 539.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-6-(3-(dimethylamino)prop-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>19e</b>)</h4><div class="NLM_p last">Treatment of 1-(benzyloxy)-6-bromo-<i>N</i>-(2,4-difluorobenzyl)-4-((2,4-dimethoxybenzyl)amino)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide <b>17c</b><a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> with commercially available <i>N</i>,<i>N</i>-dimethylprop-2-yn-1-amine as outlined in general procedures J and H provided <b>19e</b> as a colorless oil (33% yield, two steps). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.57 (t, <i>J</i> = 5.7 Hz, 1H), 8.77 (d, <i>J</i> = 1.6 Hz, 1H), 8.11 (d, <i>J</i> = 1.6 Hz, 1H), 7.67–7.65 (m, 2H), 7.42–7.36 (m, 4H), 6.87–6.80 (m, 2H), 5.25 (s, 2H), 4.63 (d, <i>J</i> = 5.7 Hz, 2H), 3.51 (s, 2H), 2.42 (s, 6H). ESI-MS <i>m</i>/<i>z</i>: 518.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 4-Amino-1-(benzyloxy)-<i>N</i>-(2,4-difluorobenzyl)-6-(3-(2-hydroxyethoxy)prop-1-yn-1-yl)-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamide (<b>19f</b>)</h4><div class="NLM_p last">Treatment of <b>18d</b> as outlined in general procedure H provided <b>19f</b> as a colorless oil (87% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.55 (t, <i>J</i> = 5.7 Hz, 1H), 8.79 (d, <i>J</i> = 1.7 Hz, 1H), 8.08 (d, <i>J</i> = 1.8 Hz, 1H), 7.68–7.67 (m, 2H), 7.41–7.38 (m, 4H), 6.88–6.82 (m, 2H), 5.27 (s, 2H), 4.64 (d, <i>J</i> = 5.8 Hz, 2H), 4.49 (s, 2H), 3.86–3.84 (m, 2H), 3.77–3.75 (m, 2H). ESI-MS <i>m</i>/<i>z</i>: 535.2 (MH<sup>+</sup>).</div></div><div id="sec4_12_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> (<i>E</i>)-Methyl 3-(5-Amino-8-(benzyloxy)-6-((2,4-difluorobenzyl)carbamoyl)-7-oxo-7,8-dihydro-1,8-naphthyridin-3-yl)acrylate (<b>19g</b>)</h4><div class="NLM_p last">Treatment of <b>18h</b> as outlined in general procedure H provided <b>19g</b> as a colorless oil (38% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 10.54 (d, <i>J</i> = 5.6 Hz, 1H), 8.89 (d, <i>J</i> = 1.5 Hz, 1H), 8.13 (s, 1H), 7.76 (d, <i>J</i> = 16.1 Hz, 1H), 7.67 (d, <i>J</i> = 6.1 Hz, 2H), 7.41–7.37 (m, 4H), 6.87–6.81 (m, 2H), 6.59 (d, <i>J</i> = 16.1 Hz, 1H), 5.27 (s, 2H), 4.64 (d, <i>J</i> = 5.7 Hz, 2H), 3.86 (s, 3H). ESI-MS <i>m</i>/<i>z</i>: 521.2 (MH<sup>+</sup>).</div></div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> HIV-1 IN Biochemical Assays</h3><div class="NLM_p last">As previously described,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> inhibitors or an equivalent volume of DMSO were added to a reaction mixture containing 20 nM DNA substrate 5′ <sup>32</sup>P-labeled on the transferred strand and 400 nM IN in 50 mM MOPS pH 7.2, 7.5 mM MgCl<sub>2</sub>, and 14 mM 2-mercaptoethanol. Reactions were allowed to proceed for 2 h at 37 °C and were stopped by the addition of an equal volume of loading buffer [1% sodium dodecyl sulfate, 0.025% bromophenol blue, and 0.025% xylene cyanol made in formamide]. Reaction products, separated in 16% polyacrylamide denaturing sequencing gels, were visualized using by phosphorimaging using a Typhoon 8600 instrument (GE Healthcare) and quantified using the ImageQuant 5.1 software (GE Healthcare). Data analyses (linear regression, IC<sub>50</sub> determination, and standard deviation) were performed using Prism 5.0 software from GraphPad.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> HIV-1 Vector Constructs</h3><div class="NLM_p last">pNLNgoMIVR<sup>–</sup>Δ<i>E</i>nv.LUC has been described previously.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The IN reading frame was removed from pNLNgoMIVR<sup>–</sup>Δ<i>E</i>nv.LUC by digestion with <i>Kpn</i>I and <i>Sal</i>I and inserted between the <i>Kpn</i>I and <i>Sal</i>I sites of pBluescript II KS+. With this construct as the WT template, the following HIV-1 IN mutants were prepared using the QuikChange II XL (Stratagene, La Jolla, CA) site-directed mutagenesis protocol: H51Y, T66I, E92Q, G118R, Y143R, Q148H, N155H, R263K, G140S + Q148H, and H51Y + R263K. The following sense and cognate antisense (not shown) oligonucleotides (Integrated DNA Technologies, Coralville, IA) were used in the mutagenesis: H51Y, 5′-CTAAAAGGGGAAGCCATGTATGGACAAGTAGACTGTA-3′; T66I, 5′-CCAGGAATATGGCAGCTAGATTGTATACATTTAGAAGGAAAAGTT-3′; E92Q, 5′-GCAGAAGTAATTCCAGCACAGACAGGGCAAGAAA-3′; G118R, 5′-GTACATACAGACAATCGCAGCAATTTCACCAGTAC-3′; G140S, 5′-GGGGATCAAGCAGGAATTTAGCATTCCCTACAATC-3′; Y143C, 5′-GCAGGAATTTGGCATTCCCCGCAATCCCCAAAGTCAAGGA-3′; Q148H, 5′-CATTCCCTACAATCCCCAAAGTCATGGAGTAATAGAATCTA-3′; N155H, 5′-CCAAAGTCAAGGAGTAATAGAATCTATGCATAAAGAATTAAAGAAAATTATAGGACA-3′; R263K 5′-AAAGTAGTGCCAAGAAAAAAAGCAAAGATCATC-3′. The double mutation G140S + Q148H was constructed using the previously generated Q148H mutant and the appropriate oligonucleotides to introduce the second mutation, G140S. The double mutation H51Y + R263K was constructed by using the previously generated H51Y mutant and the appropriate oligonucleotides for the second mutation, R263K. The DNA sequence of each construct was verified by DNA sequencing. The mutant IN coding sequences from pBluescript II KS+ were subcloned into pNLNgoMIVR<sup>–</sup>Δ<i>E</i>nv.LUC (between the <i>Kpn</i>I and <i>Sal</i>I sites) to produce the full-length mutant HIV-1 IN constructs. The mutant forms of the vector were validated by DNA sequence determination.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> Single-Round HIV-1 Infectivity Assay</h3><div class="NLM_p last">Assays were performed using the human embryonic kidney cell culture cell line 293 acquired from the American Type Culture Collection (ATCC) and the human osteosarcoma cell line, HOS, obtained from Dr. Richard Schwartz (Michigan State University, East Lansing, MI) using methods that have been previously reported.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44, 45)</a> The 293 and HOS cells were grown in Dulbecco’s Modified Eagle’s Medium (Invitrogen, Carlsbad, CA) supplemented with 5% (v/v) fetal bovine serum, 5% newborn calf serum, and penicillin (50 units/mL) plus streptomycin (50 μg/mL; Quality Biological, Gaithersburg, MD). The transfection vector, pNLNgoMIVR<sup>–</sup>ΔLUC was made from pNLNgoMIVR<sup>–</sup>Δ<i>E</i>nv.HSA by removing the HSA reporter gene and replacing it with a luciferase reporter gene, which was inserted between the NotI and XhoI restriction sites.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> VSV-<i>g</i>-pseudotyped HIV was produced by transfection of 293 cells.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> On the day prior to transfection, 293 cells were plated on 100 mm diameter dishes at a density of 1.5 × 10<sup>6</sup> cells per plate and transfected with 16 μg of pNLNgoMIVR<sup>–</sup>ΔLUC and 4 μg of pHCMV-g (obtained from Dr. Jane Burns, University of California, San Diego) using the calcium phosphate method. At approximately 6 h after the calcium phosphate precipitate was added, 293 cells were washed twice with phosphate-buffered saline (PBS) and incubated with fresh media for 48 h. The virus-containing supernatants were harvested, clarified by low-speed centrifugation, filtered, and diluted for use in antiviral infection assays. On the day prior to the screen, HOS cells were seeded in 96-well luminescence cell culture plates at a density of 4000 cells in 100 μL per well. On the day of the assay for cellular cytotoxicity, cells were treated with compounds using a concentration range from 250 to 0.05 μM and then incubated at 37 °C (48 h). On the day of the assay for antiviral activity, cells were treated with compounds using a concentration range from 5 to 0.0001 μM using 11 serial dilutions and then incubated at 37 °C (3 h). After 3 h, 100 μL of virus-stock [diluted to achieve a luciferase signal between 0.2 and 1.5 relative luciferase units (RLUs)] was added to each well and incubation was continued (37 °C, 48 h). Cellular cytotoxicity was measured by using the ATP Lite luminescence detection system and monitored by adding 50 μL of cell lysis buffer from the luminescence ATP detection assay to each well followed by mixing at 700 rpm at room temperature using a compact thermomixer (5 min). After addition of 50 μL of reconstituted luminescence ATP detection assay reagent to all wells except for the negative control/background wells, the plates were mixed at 700 rpm at room temperature using a compact thermomixer (5 min) and incubated at room temperature to allow time for signal development (20 min), and cytotoxicity was determined using the microplate reader. Infectivity was measured using the Steady-lite Plus luminescence reporter gene assay system (PerkinElmer, Waltham, MA). Luciferase activity was measured by adding 100 μL of Steady-lite Plus buffer (PerkinElmer) to the cells, incubating at room temperature (20 min), and measuring luminescence using a microplate reader. Both cytotoxicity and antiviral activity were normalized to the cellular cytotoxicity and infectivity using cells incubated in absence of the respective target compounds. KaleidaGraph (Synergy Software, Reading, PA) was used to perform nonlinear regression analysis on the data. Final IC<sub>50</sub> values were determined from the fit model.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> X-ray Crystallography</h3><div class="NLM_p last">PFV intasome crystals were soaked in the presence of 0.5–1 mM INSTIs in cryoprotection solution prior to snap freezing in liquid nitrogen as described.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> X-ray diffraction data collected on beamlines I04 and I03 of the Diamond Light Source (Oxfordshire, UK) were processed using XDS<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> and Aimless.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Structures were determined via rigid-body refinement of model generated from PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BDZ">4BDZ</a> by removing small molecule atoms, and the compounds were fitted into resulting <i>F</i><sub>o</sub>–<i>F</i><sub>c</sub> difference maps. The models were built in Coot,<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> refined in Phenix,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> and validated using MolProbity.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Relevant data collection and refinement statistics are given in <a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i103"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00596">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52207" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52207" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00596" class="ext-link">10.1021/acs.jmedchem.7b00596</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Overlays of the PFV crystal structures of several bound INSTIs highlighting the correspondence of features corresponding to the 4-position of the current compounds; predicted molPAINS values for final products (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_001.pdf">jm7b00596_si_001.pdf (454.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_002.csv">jm7b00596_si_002.csv (5.12 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for PFV intasome-bound <b>5′g</b>, <b>5g</b>, and <b>6p</b> have been deposited in the RCSB Protein Data Bank (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMA">5MMA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NO1">5NO1</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMB">5MMB</a>, respectively).</p><div class="testing" data-doi="10.1021/acs.jmedchem.7b00596" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12646" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12646" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Terrence R. Burke</span> - <span class="hlFld-Affiliation affiliation">†Chemical
Biology Laboratory and ‡HIV Dynamics and Replication Program, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Frederick, Maryland 21702, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Chromatin
Structure and Mobile DNA, The Francis Crick
Institute, London NW1 1AT, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Imperial
College London, St-Mary’s
Campus, Norfolk Place, London W2 1PG, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9925-8586" title="Orcid link">http://orcid.org/0000-0001-9925-8586</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#89ebfcfbe2ecfdecc9e1ece5e0f1a7e7e0e1a7eee6ff"><span class="__cf_email__" data-cfemail="a1c3d4d3cac4d5c4e1c9c4cdc8d98fcfc8c98fc6ced7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xue Zhi Zhao</span> - <span class="hlFld-Affiliation affiliation">†Chemical
Biology Laboratory and ‡HIV Dynamics and Replication Program, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Frederick, Maryland 21702, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Chromatin
Structure and Mobile DNA, The Francis Crick
Institute, London NW1 1AT, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Imperial
College London, St-Mary’s
Campus, Norfolk Place, London W2 1PG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Steven J. Smith</span> - <span class="hlFld-Affiliation affiliation">†Chemical
Biology Laboratory and ‡HIV Dynamics and Replication Program, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Frederick, Maryland 21702, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Chromatin
Structure and Mobile DNA, The Francis Crick
Institute, London NW1 1AT, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Imperial
College London, St-Mary’s
Campus, Norfolk Place, London W2 1PG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel P. Maskell</span> - <span class="hlFld-Affiliation affiliation">Chromatin
Structure and Mobile DNA, The Francis Crick
Institute, London NW1 1AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mathieu Métifiot</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Valerie E. Pye</span> - <span class="hlFld-Affiliation affiliation">Chromatin
Structure and Mobile DNA, The Francis Crick
Institute, London NW1 1AT, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Katherine Fesen</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christophe Marchand</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yves Pommier</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Cherepanov</span> - <span class="hlFld-Affiliation affiliation">Chromatin
Structure and Mobile DNA, The Francis Crick
Institute, London NW1 1AT, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">Imperial
College London, St-Mary’s
Campus, Norfolk Place, London W2 1PG, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stephen H. Hughes</span> - <span class="hlFld-Affiliation affiliation">Developmental
Therapeutics Branch and Laboratory of Molecular Pharmacology, Center
for Cancer Research, National Cancer Institute,
National Institutes of Health, Bethesda, Maryland 20892, United States</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Certain of the reported compounds are included in patent applications.<br /></br></div></li></ul></div><div class="ack" id="ACK-d239e8873-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i105">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32706" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32706" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank the staff of the Diamond Light Source beamlines I04 and I03 for assistance with X-ray data collection. Our studies are supported by the NIH Intramural Program, Center for Cancer Research (ZIA BC 007363, Z01 BC 007333), National Cancer Institute, and by grants from the NIH AIDS Intramural Targeted Program (IATAP).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i106" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i106"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i107" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i107"> Abbreviations Used</h2><tr><td class="NLM_term">HIV-1</td><td class="NLM_def"><p class="first last">human immunodeficiency virus type 1</p></td></tr><tr><td class="NLM_term">PFV</td><td class="NLM_def"><p class="first last">prototype foamy virus</p></td></tr><tr><td class="NLM_term">AIDS</td><td class="NLM_def"><p class="first last">acquired immune deficiency syndrome</p></td></tr><tr><td class="NLM_term">FDA</td><td class="NLM_def"><p class="first last">Food and Drug Administration</p></td></tr><tr><td class="NLM_term">IN</td><td class="NLM_def"><p class="first last">integrase</p></td></tr><tr><td class="NLM_term">RAL</td><td class="NLM_def"><p class="first last">raltegravir</p></td></tr><tr><td class="NLM_term">EVG</td><td class="NLM_def"><p class="first last">elvitegravir</p></td></tr><tr><td class="NLM_term">DTG</td><td class="NLM_def"><p class="first last">dolutegravir</p></td></tr><tr><td class="NLM_term">3-P</td><td class="NLM_def"><p class="first last">3-processing</p></td></tr><tr><td class="NLM_term">ST</td><td class="NLM_def"><p class="first last">strand transfer</p></td></tr><tr><td class="NLM_term">INSTIs</td><td class="NLM_def"><p class="first last">integrase strand transfer inhibitors</p></td></tr><tr><td class="NLM_term">DNA</td><td class="NLM_def"><p class="first last">DNA</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximum inhibitory concentration</p></td></tr><tr><td class="NLM_term">EC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximal effective concentration</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last">dimethylformamide</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high-pressure liquid chromatography</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate-buffered saline</p></td></tr><tr><td class="NLM_term">RLUs</td><td class="NLM_def"><p class="first last">relative luciferase units</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i108">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43311" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43311" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 51 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Lesbats, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Retroviral DNA integration</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">12730</span><span class="NLM_x">–</span> <span class="NLM_lpage">12757</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00125</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00125" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC28Xot1Sksr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=12730-12757&author=P.+Lesbatsauthor=A.+N.+Engelmanauthor=P.+Cherepanov&title=Retroviral+DNA+integration&doi=10.1021%2Facs.chemrev.6b00125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Retroviral DNA Integration</span></div><div class="casAuthors">Lesbats, Paul; Engelman, Alan N.; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">12730-12757</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The integration of a DNA copy of the viral RNA genome into host chromatin is the defining step of retroviral replication.  This enzymic process is catalyzed by the virus-encoded integrase protein, which is conserved among retroviruses and LTR-retrotransposons.  Retroviral integration proceeds via two integrase activities: 3'-processing of the viral DNA ends, followed by the strand transfer of the processed ends into host cell chromosomal DNA.  Herein we review the mol. mechanism of retroviral DNA integration, with an emphasis on reaction chemistries and architectures of the nucleoprotein complexes involved.  We addnl. discuss the latest advances on anti-integrase drug development for the treatment of AIDS and the utility of integrating retroviral vectors in gene therapy applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG1CjjtylBirVg90H21EOLACvtfcHk0lgMNgyLr2u4sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xot1Sksr8%253D&md5=f3a6b640085583b294cdd82e0a384234</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00125%26sid%3Dliteratum%253Aachs%26aulast%3DLesbats%26aufirst%3DP.%26aulast%3DEngelman%26aufirst%3DA.%2BN.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DRetroviral%2520DNA%2520integration%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D12730%26epage%3D12757%26doi%3D10.1021%2Facs.chemrev.6b00125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Summa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrocchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crescenzi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiore, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez Paz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laufer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteagudo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muraglia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nizi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orvieto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pescatore, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarpelli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stillmock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, M.</span><span> </span><span class="NLM_article-title">Discovery of raltegravir, a potent, selective orally bioavailable HIV-integrase inhibitor for the treatment of HIV-AIDS infection</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5843</span><span class="NLM_x">–</span> <span class="NLM_lpage">5855</span><span class="refDoi"> DOI: 10.1021/jm800245z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800245z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5843-5855&author=V.+Summaauthor=A.+Petrocchiauthor=F.+Bonelliauthor=B.+Crescenziauthor=M.+Donghiauthor=M.+Ferraraauthor=F.+Fioreauthor=C.+Gardelliauthor=O.+Gonzalez+Pazauthor=D.+J.+Hazudaauthor=P.+Jonesauthor=O.+Kinzelauthor=R.+Lauferauthor=E.+Monteagudoauthor=E.+Muragliaauthor=E.+Niziauthor=F.+Orvietoauthor=P.+Paceauthor=G.+Pescatoreauthor=R.+Scarpelliauthor=K.+Stillmockauthor=M.+V.+Witmerauthor=M.+Rowley&title=Discovery+of+raltegravir%2C+a+potent%2C+selective+orally+bioavailable+HIV-integrase+inhibitor+for+the+treatment+of+HIV-AIDS+infection&doi=10.1021%2Fjm800245z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Raltegravir, a Potent, Selective Orally Bioavailable HIV-Integrase Inhibitor for the Treatment of HIV-AIDS Infection</span></div><div class="casAuthors">Summa, Vincenzo; Petrocchi, Alessia; Bonelli, Fabio; Crescenzi, Benedetta; Donghi, Monica; Ferrara, Marco; Fiore, Fabrizio; Gardelli, Cristina; Gonzalez Paz, Odalys; Hazuda, Daria J.; Jones, Philip; Kinzel, Olaf; Laufer, Ralph; Monteagudo, Edith; Muraglia, Ester; Nizi, Emanuela; Orvieto, Federica; Pace, Paola; Pescatore, Giovanna; Scarpelli, Rita; Stillmock, Kara; Witmer, Marc V.; Rowley, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5843-5855</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human immunodeficiency virus type-1 (HIV-1) integrase is one of the three virally encoded enzymes required for replication and therefore a rational target for chemotherapeutic intervention in the treatment of HIV-1 infection.  We report here the discovery of Raltegravir, the first HIV-integrase inhibitor approved by FDA for the treatment of HIV infection.  It derives from the evolution of 5,6-dihydroxypyrimidine-4-carboxamides and N-methyl-4-hydroxypyrimidinone-carboxamides, which exhibited potent inhibition of the HIV-integrase catalyzed strand transfer process.  Structural modifications on these mols. were made in order to maximize potency as HIV-integrase inhibitors against the wild type virus, a selection of mutants, and optimize the selectivity, pharmacokinetic, and metabolic profiles in preclin. species.  The good profile of Raltegravir has enabled its progression toward the end of phase III clin. trials for the treatment of HIV-1 infection and culminated with the FDA approval as the first HIV-integrase inhibitor for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYo6KKNjJRCrVg90H21EOLACvtfcHk0lgMNgyLr2u4sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM&md5=f90f994c7cd031a4063cbb3e91b4c3f0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm800245z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800245z%26sid%3Dliteratum%253Aachs%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DBonelli%26aufirst%3DF.%26aulast%3DCrescenzi%26aufirst%3DB.%26aulast%3DDonghi%26aufirst%3DM.%26aulast%3DFerrara%26aufirst%3DM.%26aulast%3DFiore%26aufirst%3DF.%26aulast%3DGardelli%26aufirst%3DC.%26aulast%3DGonzalez%2BPaz%26aufirst%3DO.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DLaufer%26aufirst%3DR.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DMuraglia%26aufirst%3DE.%26aulast%3DNizi%26aufirst%3DE.%26aulast%3DOrvieto%26aufirst%3DF.%26aulast%3DPace%26aufirst%3DP.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DScarpelli%26aufirst%3DR.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DWitmer%26aufirst%3DM.%2BV.%26aulast%3DRowley%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520raltegravir%252C%2520a%2520potent%252C%2520selective%2520orally%2520bioavailable%2520HIV-integrase%2520inhibitor%2520for%2520the%2520treatment%2520of%2520HIV-AIDS%2520infection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5843%26epage%3D5855%26doi%3D10.1021%2Fjm800245z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Sato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motomura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aramaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawakami, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuzaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamataka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodama, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuoka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, H.</span><span> </span><span class="NLM_article-title">Novel HIV-1 integrase inhibitors derived from quinolone antibiotics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1506</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span><span class="refDoi"> DOI: 10.1021/jm0600139</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0600139" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=1506-1508&author=M.+Satoauthor=T.+Motomuraauthor=H.+Aramakiauthor=T.+Matsudaauthor=M.+Yamashitaauthor=Y.+Itoauthor=H.+Kawakamiauthor=Y.+Matsuzakiauthor=W.+Watanabeauthor=K.+Yamatakaauthor=S.+Ikedaauthor=E.+Kodamaauthor=M.+Matsuokaauthor=H.+Shinkai&title=Novel+HIV-1+integrase+inhibitors+derived+from+quinolone+antibiotics&doi=10.1021%2Fjm0600139"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm0600139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0600139%26sid%3Dliteratum%253Aachs%26aulast%3DSato%26aufirst%3DM.%26aulast%3DMotomura%26aufirst%3DT.%26aulast%3DAramaki%26aufirst%3DH.%26aulast%3DMatsuda%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DM.%26aulast%3DIto%26aufirst%3DY.%26aulast%3DKawakami%26aufirst%3DH.%26aulast%3DMatsuzaki%26aufirst%3DY.%26aulast%3DWatanabe%26aufirst%3DW.%26aulast%3DYamataka%26aufirst%3DK.%26aulast%3DIkeda%26aufirst%3DS.%26aulast%3DKodama%26aufirst%3DE.%26aulast%3DMatsuoka%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DH.%26atitle%3DNovel%2520HIV-1%2520integrase%2520inhibitors%2520derived%2520from%2520quinolone%2520antibiotics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D1506%26epage%3D1508%26doi%3D10.1021%2Fjm0600139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Johns, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawasuji, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weatherhead, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Temelkoff, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taoda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiyama, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuji, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshinaga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garvey, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, T.</span><span> </span><span class="NLM_article-title">Carbamoyl pyridone HIV-1 integrase Inhibitors 3. A diastereomeric approach to chiral nonracemic tricyclic ring systems and the discovery of dolutegravir (S/GSKi349572) and (S/GSK1265744)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">5901</span><span class="NLM_x">–</span> <span class="NLM_lpage">5916</span><span class="refDoi"> DOI: 10.1021/jm400645w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400645w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5901-5916&author=B.+A.+Johnsauthor=T.+Kawasujiauthor=J.+G.+Weatherheadauthor=T.+Taishiauthor=D.+P.+Temelkoffauthor=H.+Yoshidaauthor=T.+Akiyamaauthor=Y.+Taodaauthor=H.+Muraiauthor=R.+Kiyamaauthor=M.+Fujiauthor=N.+Tanimotoauthor=J.+Jeffreyauthor=S.+A.+Fosterauthor=T.+Yoshinagaauthor=T.+Sekiauthor=M.+Kobayashiauthor=A.+Satoauthor=M.+N.+Johnsonauthor=E.+P.+Garveyauthor=T.+Fujiwara&title=Carbamoyl+pyridone+HIV-1+integrase+Inhibitors+3.+A+diastereomeric+approach+to+chiral+nonracemic+tricyclic+ring+systems+and+the+discovery+of+dolutegravir+%28S%2FGSKi349572%29+and+%28S%2FGSK1265744%29&doi=10.1021%2Fjm400645w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm400645w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400645w%26sid%3Dliteratum%253Aachs%26aulast%3DJohns%26aufirst%3DB.%2BA.%26aulast%3DKawasuji%26aufirst%3DT.%26aulast%3DWeatherhead%26aufirst%3DJ.%2BG.%26aulast%3DTaishi%26aufirst%3DT.%26aulast%3DTemelkoff%26aufirst%3DD.%2BP.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DAkiyama%26aufirst%3DT.%26aulast%3DTaoda%26aufirst%3DY.%26aulast%3DMurai%26aufirst%3DH.%26aulast%3DKiyama%26aufirst%3DR.%26aulast%3DFuji%26aufirst%3DM.%26aulast%3DTanimoto%26aufirst%3DN.%26aulast%3DJeffrey%26aufirst%3DJ.%26aulast%3DFoster%26aufirst%3DS.%2BA.%26aulast%3DYoshinaga%26aufirst%3DT.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DM.%2BN.%26aulast%3DGarvey%26aufirst%3DE.%2BP.%26aulast%3DFujiwara%26aufirst%3DT.%26atitle%3DCarbamoyl%2520pyridone%2520HIV-1%2520integrase%2520Inhibitors%25203.%2520A%2520diastereomeric%2520approach%2520to%2520chiral%2520nonracemic%2520tricyclic%2520ring%2520systems%2520and%2520the%2520discovery%2520of%2520dolutegravir%2520%2528S%252FGSKi349572%2529%2520and%2520%2528S%252FGSK1265744%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5901%26epage%3D5916%26doi%3D10.1021%2Fjm400645w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felock, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stillmock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grobler, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espeseth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabryelski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schleif, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span> </span><span class="NLM_article-title">Inhibitors of strand transfer that prevent integration and inhibit HIV-1 replication in cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1126/science.287.5453.646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1126%2Fscience.287.5453.646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10649997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVyjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=646-650&author=D.+J.+Hazudaauthor=P.+Felockauthor=M.+Witmerauthor=A.+Wolfeauthor=K.+Stillmockauthor=J.+A.+Groblerauthor=A.+Espesethauthor=L.+Gabryelskiauthor=W.+Schleifauthor=C.+Blauauthor=M.+D.+Miller&title=Inhibitors+of+strand+transfer+that+prevent+integration+and+inhibit+HIV-1+replication+in+cells&doi=10.1126%2Fscience.287.5453.646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of strand transfer that prevent integration and inhibit HIV-1 replication in cells</span></div><div class="casAuthors">Hazuda, Daria J.; Felock, Peter; Witmer, Marc; Wolfe, Abigail; Stillmock, Kara; Grobler, Jay A.; Espeseth, Amy; Gabryelski, Lori; Schleif, William; Blau, Carol; Miller, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">5453</span>),
    <span class="NLM_cas:pages">646-650</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Integrase is essential for human immunodeficiency virus-type 1 (HIV-1) replication; however, potent inhibition of the isolated enzyme in biochem. assays has not readily translated into antiviral activity in a manner consistent with inhibition of integration.  In this report, we describe diketo acid inhibitors of HIV-1 integrase that manifest antiviral activity as a consequence of their effect on integration.  The antiviral activity of these compds. is due exclusively to inhibition of one of the two catalytic functions of integrase, strand transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLfQNYMkTCVrVg90H21EOLACvtfcHk0lhUjqqqq_ReDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVyjtQ%253D%253D&md5=d160c8fa31a9ca0a3530dceab66a892c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.287.5453.646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.287.5453.646%26sid%3Dliteratum%253Aachs%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DFelock%26aufirst%3DP.%26aulast%3DWitmer%26aufirst%3DM.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DGrobler%26aufirst%3DJ.%2BA.%26aulast%3DEspeseth%26aufirst%3DA.%26aulast%3DGabryelski%26aufirst%3DL.%26aulast%3DSchleif%26aufirst%3DW.%26aulast%3DBlau%26aufirst%3DC.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26atitle%3DInhibitors%2520of%2520strand%2520transfer%2520that%2520prevent%2520integration%2520and%2520inhibit%2520HIV-1%2520replication%2520in%2520cells%26jtitle%3DScience%26date%3D2000%26volume%3D287%26spage%3D646%26epage%3D650%26doi%3D10.1126%2Fscience.287.5453.646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Métifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiselev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Selectivity for strand-transfer over 3′-processing and susceptibility to clinical resistance of HIV-1 integrase inhibitors are driven by key enzyme–DNA interactions in the active site</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">6896</span><span class="NLM_x">–</span> <span class="NLM_lpage">6906</span><span class="refDoi"> DOI: 10.1093/nar/gkw592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1093%2Fnar%2Fgkw592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=6896-6906&author=M.+M%C3%A9tifiotauthor=B.+C.+Johnsonauthor=E.+Kiselevauthor=L.+Marlerauthor=X.+Z.+Zhaoauthor=T.+R.+Burkeauthor=C.+Marchandauthor=S.+H.+Hughesauthor=Y.+Pommier&title=Selectivity+for+strand-transfer+over+3%E2%80%B2-processing+and+susceptibility+to+clinical+resistance+of+HIV-1+integrase+inhibitors+are+driven+by+key+enzyme%E2%80%93DNA+interactions+in+the+active+site&doi=10.1093%2Fnar%2Fgkw592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw592%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A9tifiot%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BC.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DMarler%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DSelectivity%2520for%2520strand-transfer%2520over%25203%25E2%2580%25B2-processing%2520and%2520susceptibility%2520to%2520clinical%2520resistance%2520of%2520HIV-1%2520integrase%2520inhibitors%2520are%2520driven%2520by%2520key%2520enzyme%25E2%2580%2593DNA%2520interactions%2520in%2520the%2520active%2520site%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D6896%26epage%3D6906%26doi%3D10.1093%2Fnar%2Fgkw592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Bar-Magen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sloan, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faltenbacher, V. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donahue, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span> </span><span class="NLM_article-title">Comparative biochemical analysis of HIV-1 subtype B and C integrase enzymes</span> <span class="citation_source-journal">Retrovirology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="refDoi"> DOI: 10.1186/1742-4690-6-103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1186%2F1742-4690-6-103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=103&author=T.+Bar-Magenauthor=R.+D.+Sloanauthor=V.+H.+Faltenbacherauthor=D.+A.+Donahueauthor=B.+D.+Kuhlauthor=M.+Oliveiraauthor=H.+Xuauthor=M.+A.+Wainberg&title=Comparative+biochemical+analysis+of+HIV-1+subtype+B+and+C+integrase+enzymes&doi=10.1186%2F1742-4690-6-103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1742-4690-6-103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1742-4690-6-103%26sid%3Dliteratum%253Aachs%26aulast%3DBar-Magen%26aufirst%3DT.%26aulast%3DSloan%26aufirst%3DR.%2BD.%26aulast%3DFaltenbacher%26aufirst%3DV.%2BH.%26aulast%3DDonahue%26aufirst%3DD.%2BA.%26aulast%3DKuhl%26aufirst%3DB.%2BD.%26aulast%3DOliveira%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26atitle%3DComparative%2520biochemical%2520analysis%2520of%2520HIV-1%2520subtype%2520B%2520and%2520C%2520integrase%2520enzymes%26jtitle%3DRetrovirology%26date%3D2009%26volume%3D6%26spage%3D103%26doi%3D10.1186%2F1742-4690-6-103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maertens, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">3′-Processing and strand transfer catalysed by retroviral integrase in crystallo</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">3020</span><span class="NLM_x">–</span> <span class="NLM_lpage">3028</span><span class="refDoi"> DOI: 10.1038/emboj.2012.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Femboj.2012.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=22580823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVertbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=3020-3028&author=S.+Hareauthor=G.+N.+Maertensauthor=P.+Cherepanov&title=3%E2%80%B2-Processing+and+strand+transfer+catalysed+by+retroviral+integrase+in+crystallo&doi=10.1038%2Femboj.2012.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">3'-Processing and strand transfer catalysed by retroviral integrase in crystallo</span></div><div class="casAuthors">Hare, Stephen; Maertens, Goedele N.; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3020-3028</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Retroviral integrase (IN) is responsible for two consecutive reactions, which lead to insertion of a viral DNA copy into a host cell chromosome.  Initially, the enzyme removes di- or trinucleotides from viral DNA ends to expose 3'-hydroxyls attached to the invariant CA dinucleotides (3'-processing reaction).  Second, it inserts the processed 3'-viral DNA ends into host chromosomal DNA (strand transfer).  Herein, we report a crystal structure of prototype foamy virus IN bound to viral DNA prior to 3'-processing.  Furthermore, taking advantage of its dependence on divalent metal ion cofactors, we were able to freeze trap the viral enzyme in its ground states contg. all the components necessary for 3'-processing or strand transfer.  Our results shed light on the mechanics of retroviral DNA integration and explain why HIV IN strand transfer inhibitors are ineffective against the 3'-processing step of integration.  The ground state structures moreover highlight a striking substrate mimicry utilized by the inhibitors in their binding to the IN active site and suggest ways to improve upon this clin. relevant class of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmcESnfHFZn7Vg90H21EOLACvtfcHk0ljArc3JTfCsCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVertbk%253D&md5=02545febfca7173b767d203e90210a74</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.118%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DMaertens%26aufirst%3DG.%2BN.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3D3%25E2%2580%25B2-Processing%2520and%2520strand%2520transfer%2520catalysed%2520by%2520retroviral%2520integrase%2520in%2520crystallo%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D3020%26epage%3D3028%26doi%3D10.1038%2Femboj.2012.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Abram, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferris, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvord, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span> </span><span class="NLM_article-title">Nature, position, and frequency of mutations made in a single cycle of HIV-1 replication</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">9864</span><span class="NLM_x">–</span> <span class="NLM_lpage">9878</span><span class="refDoi"> DOI: 10.1128/JVI.00915-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1128%2FJVI.00915-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20660205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlarsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=9864-9878&author=M.+E.+Abramauthor=A.+L.+Ferrisauthor=W.+Shaoauthor=W.+G.+Alvordauthor=S.+H.+Hughes&title=Nature%2C+position%2C+and+frequency+of+mutations+made+in+a+single+cycle+of+HIV-1+replication&doi=10.1128%2FJVI.00915-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nature, position, and frequency of mutations made in a single cycle of HIV-1 replication</span></div><div class="casAuthors">Abram, Michael E.; Ferris, Andrea L.; Shao, Wei; Alvord, W. Gregory; Hughes, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9864-9878</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">There is considerable HIV-1 variation in patients.  The extent of the variation is due to the high rate of viral replication, the high viral load, and the errors made during viral replication.  Mutations can arise from errors made either by host DNA-dependent RNA polymerase II or by HIV-1 reverse transcriptase (RT), but the relative contributions of these two enzymes to the mutation rate are unknown.  In addn., mutations in RT can affect its fidelity, but the effect of mutations in RT on the nature of the mutations that arise in vivo is poorly understood.  We have developed an efficient system, based on existing technol., to analyze the mutations that arise in an HIV-1 vector in a single cycle of replication.  A lacZα reporter gene is used to identify viral DNAs that contain mutations which are analyzed by DNA sequencing.  The forward mutation rate in this system is 1.4 × 10-5 mutations/bp/cycle, equiv. to the retroviral av.  This rate is about 3-fold lower than previously reported for HIV-1 in vivo and is much lower than what has been reported for purified HIV-1 RT in vitro.  Although the mutation rate was not affected by the orientation of lacZα, the sites favored for mutations (hot spots) in lacZα depended on which strand of lacZα was present in the viral RNA.  The pattern of hot spots seen in lacZα in vivo did not match any of the published data obtained when purified RT was used to copy lacZα in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtHBhCUY9xnLVg90H21EOLACvtfcHk0ljArc3JTfCsCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlarsrnL&md5=c9adb40a987b765cab188bb9505b0a9f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FJVI.00915-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00915-10%26sid%3Dliteratum%253Aachs%26aulast%3DAbram%26aufirst%3DM.%2BE.%26aulast%3DFerris%26aufirst%3DA.%2BL.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DAlvord%26aufirst%3DW.%2BG.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26atitle%3DNature%252C%2520position%252C%2520and%2520frequency%2520of%2520mutations%2520made%2520in%2520a%2520single%2520cycle%2520of%2520HIV-1%2520replication%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D9864%26epage%3D9878%26doi%3D10.1128%2FJVI.00915-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Wainberg, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaharatos, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, B. G.</span><span> </span><span class="NLM_article-title">Development of antiretroviral drug resistance</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">365</span><span class="NLM_x">, </span> <span class="NLM_fpage">637</span><span class="NLM_x">–</span> <span class="NLM_lpage">646</span><span class="refDoi"> DOI: 10.1056/NEJMra1004180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1056%2FNEJMra1004180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=21848464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKnu7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=637-646&author=M.+A.+Wainbergauthor=G.+J.+Zaharatosauthor=B.+G.+Brenner&title=Development+of+antiretroviral+drug+resistance&doi=10.1056%2FNEJMra1004180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Antiretroviral drug resistance</span></div><div class="casAuthors">Wainberg, Mark A.; Zaharatos, Gerasimos J.; Brenner, Bluma G.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">637-646</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on numerous efforts in development and optimization of antiretroviral treatment strategies against a broad spectrum of HIV infection subtypes.  Suboptimal therapies that include such drugs as stavudine and didanosine can facilitate the selection of the K65R mutation, which, in turn, may limit many secondary treatment options.  In addn., suboptimal regimens put patients at risk for the development of nonnucleoside reverse-transcriptase inhibitor-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7uylwyOaxNbVg90H21EOLACvtfcHk0ljArc3JTfCsCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKnu7bF&md5=294d823167d52dca3bf4b5f769a0270c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1004180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1004180%26sid%3Dliteratum%253Aachs%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DZaharatos%26aufirst%3DG.%2BJ.%26aulast%3DBrenner%26aufirst%3DB.%2BG.%26atitle%3DDevelopment%2520of%2520antiretroviral%2520drug%2520resistance%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D637%26epage%3D646%26doi%3D10.1056%2FNEJMra1004180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Garrido, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villacian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahonero, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattery, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutierrez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caballero, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Houtte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soriano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Mendoza, C.</span><span> </span><span class="NLM_article-title">Broad phenotypic cross-resistance to elvitegravir in HIV-infected patients failing on raltegravir-containing regimens</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2873</span><span class="NLM_x">–</span> <span class="NLM_lpage">2878</span><span class="refDoi"> DOI: 10.1128/AAC.06170-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1128%2FAAC.06170-11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=2873-2878&author=C.+Garridoauthor=J.+Villacianauthor=N.+Zahoneroauthor=T.+Patteryauthor=F.+Garciaauthor=F.+Gutierrezauthor=E.+Caballeroauthor=M.+Van+Houtteauthor=V.+Sorianoauthor=C.+de+Mendoza&title=Broad+phenotypic+cross-resistance+to+elvitegravir+in+HIV-infected+patients+failing+on+raltegravir-containing+regimens&doi=10.1128%2FAAC.06170-11"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FAAC.06170-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.06170-11%26sid%3Dliteratum%253Aachs%26aulast%3DGarrido%26aufirst%3DC.%26aulast%3DVillacian%26aufirst%3DJ.%26aulast%3DZahonero%26aufirst%3DN.%26aulast%3DPattery%26aufirst%3DT.%26aulast%3DGarcia%26aufirst%3DF.%26aulast%3DGutierrez%26aufirst%3DF.%26aulast%3DCaballero%26aufirst%3DE.%26aulast%3DVan%2BHoutte%26aufirst%3DM.%26aulast%3DSoriano%26aufirst%3DV.%26aulast%3Dde%2BMendoza%26aufirst%3DC.%26atitle%3DBroad%2520phenotypic%2520cross-resistance%2520to%2520elvitegravir%2520in%2520HIV-infected%2520patients%2520failing%2520on%2520raltegravir-containing%2520regimens%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D2873%26epage%3D2878%26doi%3D10.1128%2FAAC.06170-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Geretti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armenia, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccherini-Silberstein, F.</span><span> </span><span class="NLM_article-title">Emerging patterns and implications of HIV-1 integrase inhibitor resistance</span> <span class="citation_source-journal">Curr. Opin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">686</span><span class="refDoi"> DOI: 10.1097/QCO.0b013e32835a1de7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1097%2FQCO.0b013e32835a1de7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=23086187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1OltbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=677-686&author=A.+M.+Gerettiauthor=D.+Armeniaauthor=F.+Ceccherini-Silberstein&title=Emerging+patterns+and+implications+of+HIV-1+integrase+inhibitor+resistance&doi=10.1097%2FQCO.0b013e32835a1de7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging patterns and implications of HIV-1 integrase inhibitor resistance</span></div><div class="casAuthors">Geretti, Anna Maria; Armenia, Daniele; Ceccherini-Silberstein, Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Infectious Diseases</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">677-686</span>CODEN:
                <span class="NLM_cas:coden">COIDE5</span>;
        ISSN:<span class="NLM_cas:issn">0951-7375</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: This review highlights recent data on the pathways of resistance that impact the clin. activity of first-generation and second-generation integrase inhibitors.  Recent findings: Raltegravir (RAL) and elvitegravir (EVG) are highly efficacious in first-line antiretroviral therapy, with small nos. of virol. failures obsd. in clin. trials.  Durable activity in treatment-experienced patients requires a fully supportive background regimen.  RAL and EVG show a low-to-moderate genetic barrier to resistance and extensive cross-resistance, which preclude their sequential use.  Resistance to dolutegravir (DTG) is not selected as readily in vitro and has not emerged in studies of treatment-naive patients to date.  Both in vitro and in vivo, DTG retains activity against several RAL and EVG resistant strains, but susceptibility is variably impaired by multiple mutations within the G148 pathway, which are common after RAL or EVG failure.  Cross-resistance can be partially overcome by doubling DTG dosing to twice daily, but durability of responses remains dependent on a supportive background regimen.  There is variability in the integrase gene of circulating HIV strains, which does not appear to reduce drug activity, although it may influence the emergence and evolution of integrase resistance.  Transmission of integrase resistance remains rare but surveillance is required.  Summary: Integrase inhibitors provide a potent option for the treatment of HIV infection.  Drug resistance remains a challenge, which may be partially overcome by the introduction of second-generation compds.  Prompt management of RAL and EVG failure is required to prevent the accumulation of multiple resistance mutations that reduce DTG susceptibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJNLh3j8EBULVg90H21EOLACvtfcHk0lipkpnuKqEGpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1OltbfE&md5=ad104b6dff368c146f9cef84a553506f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FQCO.0b013e32835a1de7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQCO.0b013e32835a1de7%26sid%3Dliteratum%253Aachs%26aulast%3DGeretti%26aufirst%3DA.%2BM.%26aulast%3DArmenia%26aufirst%3DD.%26aulast%3DCeccherini-Silberstein%26aufirst%3DF.%26atitle%3DEmerging%2520patterns%2520and%2520implications%2520of%2520HIV-1%2520integrase%2520inhibitor%2520resistance%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2012%26volume%3D25%26spage%3D677%26epage%3D686%26doi%3D10.1097%2FQCO.0b013e32835a1de7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Quashie, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesplede, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span> </span><span class="NLM_article-title">Evolution of HIV integrase resistance mutations</span> <span class="citation_source-journal">Curr. Opin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1097/QCO.0b013e32835ba81c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1097%2FQCO.0b013e32835ba81c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=23242340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCru7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=43-49&author=P.+K.+Quashieauthor=T.+Mespledeauthor=M.+A.+Wainberg&title=Evolution+of+HIV+integrase+resistance+mutations&doi=10.1097%2FQCO.0b013e32835ba81c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of HIV integrase resistance mutations</span></div><div class="casAuthors">Quashie, Peter K.; Mesplede, Thibault; Wainberg, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Infectious Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-49</span>CODEN:
                <span class="NLM_cas:coden">COIDE5</span>;
        ISSN:<span class="NLM_cas:issn">0951-7375</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review discussing integrase strand transfer inhibitors (INSTIs) have become a key component of antiretroviral therapy since the approval of twice-daily raltegravir in 2007 and the more recent approval of elvitegravir in 2012.  At the same time, a third compd., dolutegravir, is in late-phase clin. trials, being tested as part of a multidrug once-daily formulation comprising this INSTI and two other antiretroviral (ARV) drugs.  This review focuses on the factors leading to the development of drug resistance mutations (DRMs) against INSTIs, evidence of cross-resistance among them, and the results of regimen simplification in regard to this topic.  Recent findings: Sequencing data show that DRMs are highly dynamic in patients failing INSTI therapy.  Considerations of viral fitness and drug resistance can together det. the evolution of drug resistance mutations, and in this regard the Y143 and Q148 pathways are superior to the N155 pathway in the promotion of resistance.  Preventing the emergence of DRMs requires that effective reverse transcriptase or other inhibitors be used together with INSTIs and that high-level adherence to treatment be maintained.  Summary: Because of the susceptibility to drug resistance, INSTIs should always be used together with other effective ARV drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwQyw6THfTb7Vg90H21EOLACvtfcHk0lipkpnuKqEGpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCru7vK&md5=e0fe7fb17f03a29ce13e8579b9baeac4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1097%2FQCO.0b013e32835ba81c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQCO.0b013e32835ba81c%26sid%3Dliteratum%253Aachs%26aulast%3DQuashie%26aufirst%3DP.%2BK.%26aulast%3DMesplede%26aufirst%3DT.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26atitle%3DEvolution%2520of%2520HIV%2520integrase%2520resistance%2520mutations%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2013%26volume%3D26%26spage%3D43%26epage%3D49%26doi%3D10.1097%2FQCO.0b013e32835ba81c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Ballantyne, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, C. M.</span><span> </span><span class="NLM_article-title">Dolutegravir: First global approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">1627</span><span class="NLM_x">–</span> <span class="NLM_lpage">1637</span><span class="refDoi"> DOI: 10.1007/s40265-013-0121-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1007%2Fs40265-013-0121-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1627-1637&author=A.+D.+Ballantyneauthor=C.+M.+Perry&title=Dolutegravir%3A+First+global+approval&doi=10.1007%2Fs40265-013-0121-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0121-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0121-4%26sid%3Dliteratum%253Aachs%26aulast%3DBallantyne%26aufirst%3DA.%2BD.%26aulast%3DPerry%26aufirst%3DC.%2BM.%26atitle%3DDolutegravir%253A%2520First%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1627%26epage%3D1637%26doi%3D10.1007%2Fs40265-013-0121-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Hurt, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebastian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hicks, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eron, J. J.</span><span> </span><span class="NLM_article-title">Resistance to HIV integrase strand transfer inhibitors among clinical specimens in the united states, 2009–2012</span> <span class="citation_source-journal">Clin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1093/cid/cit697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1093%2Fcid%2Fcit697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24145878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=423-431&author=C.+B.+Hurtauthor=J.+Sebastianauthor=C.+B.+Hicksauthor=J.+J.+Eron&title=Resistance+to+HIV+integrase+strand+transfer+inhibitors+among+clinical+specimens+in+the+united+states%2C+2009%E2%80%932012&doi=10.1093%2Fcid%2Fcit697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to HIV Integrase Strand Transfer Inhibitors Among Clinical Specimens in the United States, 2009-2012</span></div><div class="casAuthors">Hurt, Christopher B.; Sebastian, Joseph; Hicks, Charles B.; Eron, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Infectious Diseases</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">423-431</span>CODEN:
                <span class="NLM_cas:coden">CIDIEL</span>;
        ISSN:<span class="NLM_cas:issn">1058-4838</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Among 3012 patients with clin. specimens submitted for integrase resistance genotyping from Jan. 2009 to Dec. 2012, 15.6% harbored ≥1 "major" integrase inhibitor mutation.  Intermediate or high-level dolutegravir resistance was predicted for 6.4% (41.2% of those with ≥1 major mutation).  Background. Data on integrase inhibitor resistance come primarily from clin. trials and in vitro studies.  We examd. results of all clin. indicated integrase genotypic resistance tests (GRTs) performed at a US national referral lab from 2009 through 2012.  Methods. Integrase sequences and demog. data were compiled with paired protease-reverse transcriptase (PR-RT) GRT results, when available.  Analyses utilized the Stanford HIV Drug Resistance Database.  "Major" integrase mutations included T66AIK, E92QV, F121Y, Y143CHR, S147G, Q148HKR, and N155H; multiple accessory mutations were also assessed.  Results. Among 3294 sequences from 3012 patients, 471 patients had viruses with ≥1 raltegravir or elvitegravir resistance mutation (15.6%).  Q148 and N155 pathways were equally represented (both n = 197); 84 had Y143 mutations.  Q148 rarely occurred without accessory mutations (n = 3).  Among 224 patients with serial integrase GRTs, 22 with baseline wild-type acquired a major mutation, after a median 224 days between tests (interquartile range, 148-335 days).  Major mutations were obsd. to persist up to 462 days.  Most (62%) had paired PR-RT results.  Patients with integrase-resistant viruses were older and more likely to have PR-RT mutations (both P < .001).  Among those with PR-RT data, 42 patients had 4-class resistance (2.3%).  Sex, geog. region, and test year were not assocd. with integrase resistance.  High-level dolutegravir resistance was predicted in 12% of patients with raltegravir- or elvitegravir-resistant viruses (2% of all patients).  Conclusions. Approx. 1 in 6 US patients undergoing integrase GRT for clin. decision making harbors significant resistance, with Q148 and N155 pathways equally common.  Dolutegravir is likely to have full or partial activity against most variants obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL2okgFAZ2nLVg90H21EOLACvtfcHk0ljKiF8HWxRpYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOmtrY%253D&md5=570b2787981051ef85c9b3230761429d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fcid%2Fcit697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcid%252Fcit697%26sid%3Dliteratum%253Aachs%26aulast%3DHurt%26aufirst%3DC.%2BB.%26aulast%3DSebastian%26aufirst%3DJ.%26aulast%3DHicks%26aufirst%3DC.%2BB.%26aulast%3DEron%26aufirst%3DJ.%2BJ.%26atitle%3DResistance%2520to%2520HIV%2520integrase%2520strand%2520transfer%2520inhibitors%2520among%2520clinical%2520specimens%2520in%2520the%2520united%2520states%252C%25202009%25E2%2580%25932012%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2014%26volume%3D58%26spage%3D423%26epage%3D431%26doi%3D10.1093%2Fcid%2Fcit697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">McCormack, P. L.</span><span> </span><span class="NLM_article-title">Dolutegravir: A review of its use in the management of HIV-1 infection in adolescents and adults</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241</span><span class="NLM_x">–</span> <span class="NLM_lpage">1254</span><span class="refDoi"> DOI: 10.1007/s40265-014-0256-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1007%2Fs40265-014-0256-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1241-1254&author=P.+L.+McCormack&title=Dolutegravir%3A+A+review+of+its+use+in+the+management+of+HIV-1+infection+in+adolescents+and+adults&doi=10.1007%2Fs40265-014-0256-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs40265-014-0256-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-014-0256-y%26sid%3Dliteratum%253Aachs%26aulast%3DMcCormack%26aufirst%3DP.%2BL.%26atitle%3DDolutegravir%253A%2520A%2520review%2520of%2520its%2520use%2520in%2520the%2520management%2520of%2520HIV-1%2520infection%2520in%2520adolescents%2520and%2520adults%26jtitle%3DDrugs%26date%3D2014%26volume%3D74%26spage%3D1241%26epage%3D1254%26doi%3D10.1007%2Fs40265-014-0256-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Mesplede, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quashie, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span> </span><span class="NLM_article-title">Resistance to HIV integrase inhibitors</span> <span class="citation_source-journal">Curr. Opin. HIV AIDS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">408</span><span class="refDoi"> DOI: 10.1097/COH.0b013e328356db89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1097%2FCOH.0b013e328356db89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=22789986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2htLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=401-408&author=T.+Mespledeauthor=P.+K.+Quashieauthor=M.+A.+Wainberg&title=Resistance+to+HIV+integrase+inhibitors&doi=10.1097%2FCOH.0b013e328356db89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to HIV integrase inhibitors</span></div><div class="casAuthors">Mesplede, Thibault; Quashie, Peter K.; Wainberg, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in HIV & AIDS</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">401-408</span>CODEN:
                <span class="NLM_cas:coden">COHABM</span>;
        ISSN:<span class="NLM_cas:issn">1746-630X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: HIV integrase inhibitors are potent antiretroviral drugs that efficiently decrease viral load in patients.  Emergence of resistance mutations against this new class of drugs represents a threat to their long-term efficacy.  The purpose of this review is to provide new information about the most recent mutations identified and other mutations that confer resistance to several integrase inhibitors.  Recent findings: New resistance mutations, such as G118R, R263K and S153Y, have been recently identified through in-vitro selection studies with second-generation integrase strand-transfer inhibitors (INSTIs).  These add to the three main resistance pathways involving mutations at positions Y143, N155 and Q148.  Structural modeling, biochem. analyses and deep sequencing are methods that currently help in the understanding of the mechanisms of resistance conferred by these mutations.  Although these new resistance mutations appear to confer only low levels of cross-resistance to second-generation drugs, the Q148 pathway with numerous secondary mutations has the potential to significantly decrease susceptibility to all drugs of the INSTI family.  Summary: Recent mutations selected in vitro with second-generation INSTIs suggest the existence of low levels of cross-resistance between these drugs and first-generation compds.  In clin. practice, the emergence of mutations at position Q148 should be monitored whenever possible.  More datasets are needed to assess the long-term efficacy of second-generation INSTIs in patients failing older INSTIs such as raltegravir and elvitegravir.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsULhoKhjr9bVg90H21EOLACvtfcHk0ljKiF8HWxRpYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2htLrO&md5=aea851c3b23668be783f7584cb9e9507</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1097%2FCOH.0b013e328356db89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCOH.0b013e328356db89%26sid%3Dliteratum%253Aachs%26aulast%3DMesplede%26aufirst%3DT.%26aulast%3DQuashie%26aufirst%3DP.%2BK.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26atitle%3DResistance%2520to%2520HIV%2520integrase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520HIV%2520AIDS%26date%3D2012%26volume%3D7%26spage%3D401%26epage%3D408%26doi%3D10.1097%2FCOH.0b013e328356db89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuuchi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craigie, R.</span><span> </span><span class="NLM_article-title">Retroviral DNA integration: reaction pathway and critical intermediates</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1295</span><span class="NLM_x">–</span> <span class="NLM_lpage">1304</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7601005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Fsj.emboj.7601005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=1295-1304&author=M.+Liauthor=M.+Mizuuchiauthor=T.+R.+Burkeauthor=R.+Craigie&title=Retroviral+DNA+integration%3A+reaction+pathway+and+critical+intermediates&doi=10.1038%2Fsj.emboj.7601005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601005%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DM.%26aulast%3DMizuuchi%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DCraigie%26aufirst%3DR.%26atitle%3DRetroviral%2520DNA%2520integration%253A%2520reaction%2520pathway%2520and%2520critical%2520intermediates%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D1295%26epage%3D1304%26doi%3D10.1038%2Fsj.emboj.7601005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valkov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Retroviral intasome assembly and inhibition of DNA strand transfer</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x">, </span> <span class="NLM_fpage">232</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1038/nature08784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Fnature08784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20118915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=232-236&author=S.+Hareauthor=S.+Guptaauthor=E.+Valkovauthor=A.+Engelmanauthor=P.+Cherepanov&title=Retroviral+intasome+assembly+and+inhibition+of+DNA+strand+transfer&doi=10.1038%2Fnature08784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Retroviral intasome assembly and inhibition of DNA strand transfer</span></div><div class="casAuthors">Hare, Stephen; Gupta, Saumya Shree; Valkov, Eugene; Engelman, Alan; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7286</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Integrase is an essential retroviral enzyme that binds both termini of linear viral DNA and inserts them into a host cell chromosome.  The structure of full-length retroviral integrase, either sep. or in complex with DNA, has been lacking.  Furthermore, although clin. useful inhibitors of HIV integrase have been developed, their mechanism of action remains speculative.  Here the authors present a crystal structure of full-length integrase from the prototype foamy virus in complex with its cognate DNA.  The structure shows the organization of the retroviral intasome comprising an integrase tetramer tightly assocd. with a pair of viral DNA ends.  All three canonical integrase structural domains are involved in extensive protein-DNA and protein-protein interactions.  The binding of strand-transfer inhibitors displaces the reactive viral DNA end from the active site, disarming the viral nucleoprotein complex.  These findings define the structural basis of retroviral DNA integration, and will allow modeling of the HIV-1 intasome to aid in the development of antiretroviral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqdEDTFdvZMbVg90H21EOLACvtfcHk0lhxgEfUZaUhwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D&md5=9551e74b6bc33494855651c3b157d753</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature08784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08784%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DValkov%26aufirst%3DE.%26aulast%3DEngelman%26aufirst%3DA.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DRetroviral%2520intasome%2520assembly%2520and%2520inhibition%2520of%2520DNA%2520strand%2520transfer%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D232%26epage%3D236%26doi%3D10.1038%2Fnature08784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Passos, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebensburg, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghirlando, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeon, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkriabai, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kvaratskhelia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craigie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyumkis, D.</span><span> </span><span class="NLM_article-title">Cryo-EM structures and atomic model of the HIV-1 strand transfer complex intasome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">89</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1126/science.aah5163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1126%2Fscience.aah5163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=28059769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1WjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=89-92&author=D.+O.+Passosauthor=M.+Liauthor=R.+Yangauthor=S.+V.+Rebensburgauthor=R.+Ghirlandoauthor=Y.+Jeonauthor=N.+Shkriabaiauthor=M.+Kvaratskheliaauthor=R.+Craigieauthor=D.+Lyumkis&title=Cryo-EM+structures+and+atomic+model+of+the+HIV-1+strand+transfer+complex+intasome&doi=10.1126%2Fscience.aah5163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structures and atomic model of the HIV-1 strand transfer complex intasome</span></div><div class="casAuthors">Passos, Dario Oliveira; Li, Min; Yang, Renbin; Rebensburg, Stephanie V.; Ghirlando, Rodolfo; Jeon, Youngmin; Shkriabai, Nikoloz; Kvaratskhelia, Mamuka; Craigie, Robert; Lyumkis, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6320</span>),
    <span class="NLM_cas:pages">89-92</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Like all retroviruses, HIV-1 irreversibly inserts a viral DNA (vDNA) copy of its RNA genome into host target DNA (tDNA).  The intasome, a higher-order nucleoprotein complex composed of viral integrase (IN) and the ends of linear vDNA, mediates integration.  Productive integration into host chromatin results in the formation of the strand transfer complex (STC) contg. catalytically joined vDNA and tDNA.  HIV-1 intasomes have been refractory to high-resoln. structural studies.  Here, the authors used a sol. IN fusion protein to facilitate structural studies, through which they present a high-resoln. cryo-electron microscopy (cryo-EM) structure of the core tetrameric HIV-1 STC and a higher-order form that adopts C-terminal domain rearrangements.  The distinct STC structures highlighted how HIV-1 could use the common retroviral intasome core architecture to accommodate different IN domain modules for assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSEOCZd1EWr7Vg90H21EOLACvtfcHk0lhxgEfUZaUhwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1WjtQ%253D%253D&md5=f69a83cd205c453c055ff9ba6ac10723</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1126%2Fscience.aah5163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aah5163%26sid%3Dliteratum%253Aachs%26aulast%3DPassos%26aufirst%3DD.%2BO.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DRebensburg%26aufirst%3DS.%2BV.%26aulast%3DGhirlando%26aufirst%3DR.%26aulast%3DJeon%26aufirst%3DY.%26aulast%3DShkriabai%26aufirst%3DN.%26aulast%3DKvaratskhelia%26aufirst%3DM.%26aulast%3DCraigie%26aufirst%3DR.%26aulast%3DLyumkis%26aufirst%3DD.%26atitle%3DCryo-EM%2520structures%2520and%2520atomic%2520model%2520of%2520the%2520HIV-1%2520strand%2520transfer%2520complex%2520intasome%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D89%26epage%3D92%26doi%3D10.1126%2Fscience.aah5163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Ballandras-Colas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maskell, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrao, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swuec, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jónsson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotecha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pye, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrésdóttir, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">A supramolecular assembly mediates lentiviral DNA integration</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1126/science.aah7002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1126%2Fscience.aah7002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=28059770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Wjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=93-95&author=A.+Ballandras-Colasauthor=D.+P.+Maskellauthor=E.+Serraoauthor=J.+Lockeauthor=P.+Swuecauthor=S.+R.+J%C3%B3nssonauthor=A.+Kotechaauthor=N.+J.+Cookauthor=V.+E.+Pyeauthor=I.+A.+Taylorauthor=V.+Andr%C3%A9sd%C3%B3ttirauthor=A.+N.+Engelmanauthor=A.+Costaauthor=P.+Cherepanov&title=A+supramolecular+assembly+mediates+lentiviral+DNA+integration&doi=10.1126%2Fscience.aah7002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A supramolecular assembly mediates lentiviral DNA integration</span></div><div class="casAuthors">Ballandras-Colas, Allison; Maskell, Daniel P.; Serrao, Erik; Locke, Julia; Swuec, Paolo; Jonsson, Stefan R.; Kotecha, Abhay; Cook, Nicola J.; Pye, Valerie E.; Taylor, Ian A.; Andresdottir, Valgerdur; Engelman, Alan N.; Costa, Alessandro; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6320</span>),
    <span class="NLM_cas:pages">93-95</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Retroviral integrase (IN) functions within the intasome nucleoprotein complex to catalyze insertion of viral DNA into cellular chromatin.  Using cryo-electron microscopy, we now visualize the functional maedi-visna lentivirus intasome at 4.9 angstrom resoln.  The intasome comprises a homo-hexadecamer of IN with a tetramer-of-tetramers architecture featuring eight structurally distinct types of IN protomers supporting two catalytically competent subunits.  The conserved intasomal core, previously obsd. in simpler retroviral systems, is formed between two IN tetramers, with a pair of C-terminal domains from flanking tetramers completing the synaptic interface.  Our results explain how HIV-1 IN, which self-assocs. into higher-order multimers, can form a functional intasome, reconcile the bulk of early HIV-1 IN biochem. and structural data, and provide a lentiviral platform for design of HIV-1 IN inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvoOxgcU9YVbVg90H21EOLACvtfcHk0lhxgEfUZaUhwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Wjug%253D%253D&md5=769d0fcbeaa6454e0feee221e344db9e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.aah7002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aah7002%26sid%3Dliteratum%253Aachs%26aulast%3DBallandras-Colas%26aufirst%3DA.%26aulast%3DMaskell%26aufirst%3DD.%2BP.%26aulast%3DSerrao%26aufirst%3DE.%26aulast%3DLocke%26aufirst%3DJ.%26aulast%3DSwuec%26aufirst%3DP.%26aulast%3DJ%25C3%25B3nsson%26aufirst%3DS.%2BR.%26aulast%3DKotecha%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DN.%2BJ.%26aulast%3DPye%26aufirst%3DV.%2BE.%26aulast%3DTaylor%26aufirst%3DI.%2BA.%26aulast%3DAndr%25C3%25A9sd%25C3%25B3ttir%26aufirst%3DV.%26aulast%3DEngelman%26aufirst%3DA.%2BN.%26aulast%3DCosta%26aufirst%3DA.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DA%2520supramolecular%2520assembly%2520mediates%2520lentiviral%2520DNA%2520integration%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D93%26epage%3D95%26doi%3D10.1126%2Fscience.aah7002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Espeseth, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felock, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grobler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anthony, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egbertson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melamed, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamill, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span> </span><span class="NLM_article-title">HIV-1 integrase inhibitors that compete with the target DNA substrate define a unique strand transfer conformation for integrase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">11244</span><span class="NLM_x">–</span> <span class="NLM_lpage">11249</span><span class="refDoi"> DOI: 10.1073/pnas.200139397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1073%2Fpnas.200139397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=11016953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnsF2rtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=11244-11249&author=A.+S.+Espesethauthor=P.+Felockauthor=A.+Wolfeauthor=M.+Witmerauthor=J.+Groblerauthor=N.+Anthonyauthor=M.+Egbertsonauthor=J.+Y.+Melamedauthor=S.+Youngauthor=T.+Hamillauthor=J.+L.+Coleauthor=D.+J.+Hazuda&title=HIV-1+integrase+inhibitors+that+compete+with+the+target+DNA+substrate+define+a+unique+strand+transfer+conformation+for+integrase&doi=10.1073%2Fpnas.200139397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 integrase inhibitors that compete with the target DNA substrate define a unique strand transfer conformation for integrase</span></div><div class="casAuthors">Espeseth, Amy S.; Felock, Peter; Wolfe, Abigail; Witmer, Marc; Grobler, Jay; Anthony, Neville; Egbertson, Melissa; Melamed, Jeffrey Y.; Young, Steve; Hamill, Terence; Cole, James L.; Hazuda, Daria J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11244-11249</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Diketo acids such as L-731,988 are potent inhibitors of HIV-1 integrase that inhibit integration and viral replication in cells.  These compds. exhibit the unique ability to inhibit the strand transfer activity of integrase in the absence of an effect on 3' end processing.  To understand the reasons for this distinct inhibitory profile, we developed a scintillation proximity assay that permits anal. of radiolabeled inhibitor binding and integrase function.  High-affinity binding of L-731,988 is shown to require the assembly of a specific complex on the HIV-1 long terminal repeat.  The interaction of L-731,988 with the complex and the efficacy of L-731,988 in strand transfer can be abrogated by the interaction with target substrates, suggesting competition between the inhibitor and the target DNA.  The L-731,988 binding site and that of the target substrate are thus distinct from that of the donor substrate and are defined by a conformation of integrase that is only adopted after assembly with the viral end.  These results elucidate the basis for diketo acid inhibition of strand transfer and have implications for integrase-directed HIV-1 drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa7Er-rFxWyLVg90H21EOLACvtfcHk0liq21TjfcQWVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnsF2rtb0%253D&md5=d3e049d527ec4b0349e0c902f61742dd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.200139397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.200139397%26sid%3Dliteratum%253Aachs%26aulast%3DEspeseth%26aufirst%3DA.%2BS.%26aulast%3DFelock%26aufirst%3DP.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DWitmer%26aufirst%3DM.%26aulast%3DGrobler%26aufirst%3DJ.%26aulast%3DAnthony%26aufirst%3DN.%26aulast%3DEgbertson%26aufirst%3DM.%26aulast%3DMelamed%26aufirst%3DJ.%2BY.%26aulast%3DYoung%26aufirst%3DS.%26aulast%3DHamill%26aufirst%3DT.%26aulast%3DCole%26aufirst%3DJ.%2BL.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26atitle%3DHIV-1%2520integrase%2520inhibitors%2520that%2520compete%2520with%2520the%2520target%2520DNA%2520substrate%2520define%2520a%2520unique%2520strand%2520transfer%2520conformation%2520for%2520integrase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D11244%26epage%3D11249%26doi%3D10.1073%2Fpnas.200139397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaxa-Chamiec, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Structural and functional analyses of the second-generation integrase strand transfer inhibitor dolutegravir (S/GSK1349572)</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">565</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span><span class="refDoi"> DOI: 10.1124/mol.111.073189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1124%2Fmol.111.073189" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=565-572&author=S.+Hareauthor=S.+J.+Smithauthor=M.+Metifiotauthor=A.+Jaxa-Chamiecauthor=Y.+Pommierauthor=S.+H.+Hughesauthor=P.+Cherepanov&title=Structural+and+functional+analyses+of+the+second-generation+integrase+strand+transfer+inhibitor+dolutegravir+%28S%2FGSK1349572%29&doi=10.1124%2Fmol.111.073189"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fmol.111.073189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.111.073189%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DJaxa-Chamiec%26aufirst%3DA.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DStructural%2520and%2520functional%2520analyses%2520of%2520the%2520second-generation%2520integrase%2520strand%2520transfer%2520inhibitor%2520dolutegravir%2520%2528S%252FGSK1349572%2529%26jtitle%3DMol.%2520Pharmacol.%26date%3D2011%26volume%3D80%26spage%3D565%26epage%3D572%26doi%3D10.1124%2Fmol.111.073189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vos, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thuring, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Molecular mechanisms of retroviral integrase inhibition and the evolution of viral resistance</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">20057</span><span class="NLM_x">–</span> <span class="NLM_lpage">20062</span><span class="refDoi"> DOI: 10.1073/pnas.1010246107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1073%2Fpnas.1010246107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=21030679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyntrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=20057-20062&author=S.+Hareauthor=A.+M.+Vosauthor=R.+F.+Claytonauthor=J.+W.+Thuringauthor=M.+D.+Cummingsauthor=P.+Cherepanov&title=Molecular+mechanisms+of+retroviral+integrase+inhibition+and+the+evolution+of+viral+resistance&doi=10.1073%2Fpnas.1010246107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms of retroviral integrase inhibition and the evolution of viral resistance</span></div><div class="casAuthors">Hare, Stephen; Vos, Ann M.; Clayton, Reginald F.; Thuring, Jan W.; Cummings, Maxwell D.; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">20057-20062, S20057/1-S20057/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The development of HIV integrase (IN) strand transfer inhibitors (INSTIs) and understanding of viral resistance to these mols. have been hampered by a paucity of available structural data.  Co-crystal structures of the prototype foamy virus (PFV) intasome with raltegravir and elvitegravir were recently reported, establishing the general INSTI binding mode.  Now an expanded set of co-crystal structures contg. PFV intasomes complexed with first- and second-generation INSTIs at resolns. of up to 2.5 Å is presented.  Importantly, the improved resoln. allowed the authors to refine the complete coordination spheres of the catalytic metal cations within the INSTI-bound intasome active site.  It is shown that like the Q148H/G140S and N155H HIV-1 IN variants, the analogous S217H and N224H PFV INs display reduced sensitivity to raltegravir in vitro.  Crystal structures of the mutant PFV intasomes in INSTI-free and -bound forms revealed that the amino acid substitutions necessitate considerable conformational rearrangements within the IN active site to accommodate an INSTI, thus explaining their adverse effects on raltegravir antiviral activity.  Furthermore, the structures predict phys. proximity and an interaction between HIV-1 IN mutant residues His148 and Ser/Ala140, rationalizing the co-evolution of Q148H and G140S/A mutations in drug-resistant viral strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovxU36XC1DQrVg90H21EOLACvtfcHk0liq21TjfcQWVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyntrnF&md5=5d66aa03a9478dad36e878d43c408099</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1010246107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1010246107%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DVos%26aufirst%3DA.%2BM.%26aulast%3DClayton%26aufirst%3DR.%2BF.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DMolecular%2520mechanisms%2520of%2520retroviral%2520integrase%2520inhibition%2520and%2520the%2520evolution%2520of%2520viral%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D20057%26epage%3D20062%26doi%3D10.1073%2Fpnas.1010246107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span> </span><span class="NLM_article-title">Interfacial inhibitors: targeting macromolecular complexes</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.1038/nrd3404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Fnrd3404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1aitr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=25-36&author=Y.+Pommierauthor=C.+Marchand&title=Interfacial+inhibitors%3A+targeting+macromolecular+complexes&doi=10.1038%2Fnrd3404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Interfacial inhibitors: targeting macromolecular complexes</span></div><div class="casAuthors">Pommier, Yves; Marchand, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-36</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Interfacial inhibitors belong to a broad class of natural products and synthetic drugs that are commonly used to treat cancers as well as bacterial and HIV infections.  They bind selectively to interfaces as macromol. machines assemble and are set in motion.  The bound drugs transiently arrest the targeted mol. machines, which can initiate allosteric effects, or desynchronize macromol. machines that normally function in concert.  Here, we review five archetypical examples of interfacial inhibitors: the camptothecins, etoposide, the quinolone antibiotics, the vinca alkaloids and the novel anti-HIV inhibitor raltegravir.  We discuss the common and diverging elements between interfacial and allosteric inhibitors and give a perspective for the rationale and methods used to discover novel interfacial inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjeQ4_1kAh-bVg90H21EOLACvtfcHk0liq21TjfcQWVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1aitr3F&md5=e69c1b97286dc65855ee8784203d4d85</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd3404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3404%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DMarchand%26aufirst%3DC.%26atitle%3DInterfacial%2520inhibitors%253A%2520targeting%2520macromolecular%2520complexes%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D25%26epage%3D36%26doi%3D10.1038%2Fnrd3404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span> </span><span class="NLM_article-title">Bicyclic 1-hydroxy-2-oxo-1,2-dihydropyridine-3-carboxamide-containing HIV-1 integrase inhibitors having high antiviral potency against cells harboring raltegravir-resistant integrase mutants</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1573</span><span class="NLM_x">–</span> <span class="NLM_lpage">1582</span><span class="refDoi"> DOI: 10.1021/jm401902n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401902n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1573-1582&author=X.+Z.+Zhaoauthor=S.+J.+Smithauthor=M.+Metifiotauthor=B.+C.+Johnsonauthor=C.+Marchandauthor=Y.+Pommierauthor=S.+H.+Hughesauthor=T.+R.+Burke&title=Bicyclic+1-hydroxy-2-oxo-1%2C2-dihydropyridine-3-carboxamide-containing+HIV-1+integrase+inhibitors+having+high+antiviral+potency+against+cells+harboring+raltegravir-resistant+integrase+mutants&doi=10.1021%2Fjm401902n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm401902n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401902n%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BC.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3DBicyclic%25201-hydroxy-2-oxo-1%252C2-dihydropyridine-3-carboxamide-containing%2520HIV-1%2520integrase%2520inhibitors%2520having%2520high%2520antiviral%2520potency%2520against%2520cells%2520harboring%2520raltegravir-resistant%2520integrase%2520mutants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1573%26epage%3D1582%26doi%3D10.1021%2Fjm401902n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span> </span><span class="NLM_article-title">4-Amino-1-hydroxy-2-oxo-1,8-naphthyridine-containing compounds having high potency against raltegravir-resistant integrase mutants of HIV-1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">5190</span><span class="NLM_x">–</span> <span class="NLM_lpage">5202</span><span class="refDoi"> DOI: 10.1021/jm5001908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5001908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlyqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5190-5202&author=X.+Z.+Zhaoauthor=S.+J.+Smithauthor=M.+Metifiotauthor=C.+Marchandauthor=P.+L.+Boyerauthor=Y.+Pommierauthor=S.+H.+Hughesauthor=T.+R.+Burke&title=4-Amino-1-hydroxy-2-oxo-1%2C8-naphthyridine-containing+compounds+having+high+potency+against+raltegravir-resistant+integrase+mutants+of+HIV-1&doi=10.1021%2Fjm5001908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">4-Amino-1-hydroxy-2-oxo-1,8-naphthyridine-Containing Compounds Having High Potency against Raltegravir-Resistant Integrase Mutants of HIV-1</span></div><div class="casAuthors">Zhao, Xue Zhi; Smith, Steven J.; Metifiot, Mathieu; Marchand, Christophe; Boyer, Paul L.; Pommier, Yves; Hughes, Stephen H.; Burke, Terrence R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5190-5202</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There are currently three HIV-1 integrase (IN) strand transfer inhibitors (INSTIs) approved by the FDA for the treatment of AIDS.  However, the emergence of drug-resistant mutants emphasizes the need to develop addnl. agents that have improved efficacies against the existent resistant mutants.  As reported herein, we modified our recently disclosed 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamides IN inhibitors to develop compds. that have improved efficacies against recombinant IN in biochem. assays.  These new compds. show single-digit nanomolar antiviral potencies against HIV vectors that carry wild-type (WT) IN in a single round replication assay and have improved potency against vectors harboring the major forms of drug resistant IN mutants.  These compds. also have low toxicity for cultured cells, which in several cases, results in selectivity indexes (CC50/EC50) of greater than 10000.  The compds. have the potential, with addnl. structural modifications, to yield clin. agents that are effective against the known strains of resistant viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCcPytjSVBlLVg90H21EOLACvtfcHk0lgpaUSb2EY_ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlyqtbY%253D&md5=047465ac43cc10dbd31eeeaf99166b8c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm5001908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5001908%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3D4-Amino-1-hydroxy-2-oxo-1%252C8-naphthyridine-containing%2520compounds%2520having%2520high%2520potency%2520against%2520raltegravir-resistant%2520integrase%2520mutants%2520of%2520HIV-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5190%26epage%3D5202%26doi%3D10.1021%2Fjm5001908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Wensing, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günthard, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paredes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillay, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shafer, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richman, D. D.</span><span> </span><span class="NLM_article-title">2017 Update of the drug resistance mutations in HIV-1</span> <span class="citation_source-journal">Top. Anitivir. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=132-141&author=A.+M.+Wensingauthor=V.+Calvezauthor=H.+F.+G%C3%BCnthardauthor=V.+A.+Johnsonauthor=R.+Paredesauthor=D.+Pillayauthor=R.+W.+Shaferauthor=D.+D.+Richman&title=2017+Update+of+the+drug+resistance+mutations+in+HIV-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWensing%26aufirst%3DA.%2BM.%26aulast%3DCalvez%26aufirst%3DV.%26aulast%3DG%25C3%25BCnthard%26aufirst%3DH.%2BF.%26aulast%3DJohnson%26aufirst%3DV.%2BA.%26aulast%3DParedes%26aufirst%3DR.%26aulast%3DPillay%26aufirst%3DD.%26aulast%3DShafer%26aufirst%3DR.%2BW.%26aulast%3DRichman%26aufirst%3DD.%2BD.%26atitle%3D2017%2520Update%2520of%2520the%2520drug%2520resistance%2520mutations%2520in%2520HIV-1%26jtitle%3DTop.%2520Anitivir.%2520Med.%26date%3D2017%26volume%3D14%26spage%3D132%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Kawasuji, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weatherhead, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taoda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikamiyama-Iwata, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiyama, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuji, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshinaga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garvey, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, T.</span><span> </span><span class="NLM_article-title">Carbamoyl pyridone HIV-1 integrase inhibitors. 2. Bi- and tricyclic derivatives result in superior antiviral and pharmacokinetic profiles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1135</span><span class="refDoi"> DOI: 10.1021/jm301550c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301550c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVeltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1124-1135&author=T.+Kawasujiauthor=B.+A.+Johnsauthor=H.+Yoshidaauthor=J.+G.+Weatherheadauthor=T.+Akiyamaauthor=T.+Taishiauthor=Y.+Taodaauthor=M.+Mikamiyama-Iwataauthor=H.+Muraiauthor=R.+Kiyamaauthor=M.+Fujiauthor=N.+Tanimotoauthor=T.+Yoshinagaauthor=T.+Sekiauthor=M.+Kobayashiauthor=A.+Satoauthor=E.+P.+Garveyauthor=T.+Fujiwara&title=Carbamoyl+pyridone+HIV-1+integrase+inhibitors.+2.+Bi-+and+tricyclic+derivatives+result+in+superior+antiviral+and+pharmacokinetic+profiles&doi=10.1021%2Fjm301550c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Carbamoyl Pyridone HIV-1 Integrase Inhibitors. 2. Bi- and Tricyclic Derivatives Result in Superior Antiviral and Pharmacokinetic Profiles</span></div><div class="casAuthors">Kawasuji, Takashi; Johns, Brian A.; Yoshida, Hiroshi; Weatherhead, Jason G.; Akiyama, Toshiyuki; Taishi, Teruhiko; Taoda, Yoshiyuki; Mikamiyama-Iwata, Minako; Murai, Hitoshi; Kiyama, Ryuichi; Fuji, Masahiro; Tanimoto, Norihiko; Yoshinaga, Tomokazu; Seki, Takahiro; Kobayashi, Masanori; Sato, Akihiko; Garvey, Edward P.; Fujiwara, Tamio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1124-1135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This work is a continuation of our initial discovery of a potent monocyclic carbamoyl pyridone human immunodeficiency virus type-1 (HIV-1) integrase inhibitor that displayed favorable antiviral and pharmacokinetic properties.  We report herein a series of bicyclic carbamoylpyridone (I; satd. and unsatd. bridge; R = e.g., alkoxyethyl) analogs to address conformational issues from our initial SAR studies.  This modification of the core unit succeeded to deliver low nanomolar potency in std. antiviral assays.  An addnl. hydroxyl substituent on the bicyclic scaffold provides remarkable improvement of antiviral efficacies against clin. relevant resistant viruses.  These findings led to addnl. cyclic tethering of the naked hydroxyl group resulting in tricyclic carbamoyl pyridone inhibitors (II; Y = O, N; n = 0-2; R' = e.g., alkyl, Ac, methanesulfonyl) to address remaining issues and deliver potential clin. candidates.  The tricyclic carbamoyl pyridone derivs. described herein served as the immediate leads in mols. to the next generation integrase inhibitor dolutegravir which is currently in late stage clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQLpPj5UB1TbVg90H21EOLACvtfcHk0lgpaUSb2EY_ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVeltA%253D%253D&md5=4bfe919f8fdbbc84e346e3f01e084b70</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm301550c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301550c%26sid%3Dliteratum%253Aachs%26aulast%3DKawasuji%26aufirst%3DT.%26aulast%3DJohns%26aufirst%3DB.%2BA.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DWeatherhead%26aufirst%3DJ.%2BG.%26aulast%3DAkiyama%26aufirst%3DT.%26aulast%3DTaishi%26aufirst%3DT.%26aulast%3DTaoda%26aufirst%3DY.%26aulast%3DMikamiyama-Iwata%26aufirst%3DM.%26aulast%3DMurai%26aufirst%3DH.%26aulast%3DKiyama%26aufirst%3DR.%26aulast%3DFuji%26aufirst%3DM.%26aulast%3DTanimoto%26aufirst%3DN.%26aulast%3DYoshinaga%26aufirst%3DT.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DGarvey%26aufirst%3DE.%2BP.%26aulast%3DFujiwara%26aufirst%3DT.%26atitle%3DCarbamoyl%2520pyridone%2520HIV-1%2520integrase%2520inhibitors.%25202.%2520Bi-%2520and%2520tricyclic%2520derivatives%2520result%2520in%2520superior%2520antiviral%2520and%2520pharmacokinetic%2520profiles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1124%26epage%3D1135%26doi%3D10.1021%2Fjm301550c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Raheem, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walji, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abeywickrema, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clas, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubost, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grobler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hafey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartingh, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajkovic, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rada, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rearden, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreier, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sisko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steele, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truchon, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, P. J.</span><span> </span><span class="NLM_article-title">Discovery of 2-pyridinone aminals: A prodrug strategy to advance a second generation of HIV-1 integrase strand transfer inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8154</span><span class="NLM_x">–</span> <span class="NLM_lpage">8165</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8154-8165&author=I.+T.+Raheemauthor=A.+M.+Waljiauthor=D.+Kleinauthor=J.+M.+Sandersauthor=D.+A.+Powellauthor=P.+Abeywickremaauthor=G.+Barbeauthor=A.+Bennetauthor=S.+D.+Clasauthor=D.+Dubostauthor=M.+Embreyauthor=J.+Groblerauthor=M.+J.+Hafeyauthor=T.+J.+Hartinghauthor=D.+J.+Hazudaauthor=M.+D.+Millerauthor=K.+P.+Mooreauthor=N.+Pajkovicauthor=S.+Patelauthor=V.+Radaauthor=P.+Reardenauthor=J.+D.+Schreierauthor=J.+Siskoauthor=T.+G.+Steeleauthor=J.+F.+Truchonauthor=J.+Waiauthor=M.+Xuauthor=P.+J.+Coleman&title=Discovery+of+2-pyridinone+aminals%3A+A+prodrug+strategy+to+advance+a+second+generation+of+HIV-1+integrase+strand+transfer+inhibitors&doi=10.1021%2Facs.jmedchem.5b01037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-Pyridinone Aminals: A Prodrug Strategy to Advance a Second Generation of HIV-1 Integrase Strand Transfer Inhibitors</span></div><div class="casAuthors">Raheem, Izzat T.; Walji, Abbas M.; Klein, Daniel; Sanders, John M.; Powell, David A.; Abeywickrema, Pravien; Barbe, Guillaume; Bennet, Amrith; Childers, Karla; Christensen, Melodie; Clas, Sophie-Dorothee; Dubost, David; Embrey, Mark; Grobler, Jay; Hafey, Michael J.; Hartingh, Timothy J.; Hazuda, Daria J.; Kuethe, Jeffrey T.; Dunn, Jamie McCabe; Miller, Michael D.; Moore, Keith P.; Nolting, Andrew; Pajkovic, Natasa; Patel, Sangita; Peng, Zuihui; Rada, Vanessa; Rearden, Paul; Schreier, John D.; Sisko, John; Steele, Thomas G.; Truchon, Jean-Francois; Wai, John; Xu, Min; Coleman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8154-8165</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The search for new mol. constructs that resemble the crit. two-metal binding pharmacophore required for HIV integrase strand transfer inhibition represents a vibrant area of research within drug discovery.  Here the authors present the discovery of a new class of HIV integrase strand transfer inhibitors based on the 2-pyridinone core of MK-0536.  These efforts led to the identification of two lead compds. I and II with excellent antiviral activity and preclin. pharmacokinetic profiles to support a once-daily human dose prediction.  Dose escalating PK studies in dog revealed significant issues with limited oral absorption and required an innovative prodrug strategy to enhance the high-dose plasma exposures of the parent mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGEsuT4C8WELVg90H21EOLACvtfcHk0ljVrhHqCyTbuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersrvJ&md5=bec0c2f7710a1e4e8cc3a502067a2236</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01037%26sid%3Dliteratum%253Aachs%26aulast%3DRaheem%26aufirst%3DI.%2BT.%26aulast%3DWalji%26aufirst%3DA.%2BM.%26aulast%3DKlein%26aufirst%3DD.%26aulast%3DSanders%26aufirst%3DJ.%2BM.%26aulast%3DPowell%26aufirst%3DD.%2BA.%26aulast%3DAbeywickrema%26aufirst%3DP.%26aulast%3DBarbe%26aufirst%3DG.%26aulast%3DBennet%26aufirst%3DA.%26aulast%3DClas%26aufirst%3DS.%2BD.%26aulast%3DDubost%26aufirst%3DD.%26aulast%3DEmbrey%26aufirst%3DM.%26aulast%3DGrobler%26aufirst%3DJ.%26aulast%3DHafey%26aufirst%3DM.%2BJ.%26aulast%3DHartingh%26aufirst%3DT.%2BJ.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DMoore%26aufirst%3DK.%2BP.%26aulast%3DPajkovic%26aufirst%3DN.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DRada%26aufirst%3DV.%26aulast%3DRearden%26aufirst%3DP.%26aulast%3DSchreier%26aufirst%3DJ.%2BD.%26aulast%3DSisko%26aufirst%3DJ.%26aulast%3DSteele%26aufirst%3DT.%2BG.%26aulast%3DTruchon%26aufirst%3DJ.%2BF.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520of%25202-pyridinone%2520aminals%253A%2520A%2520prodrug%2520strategy%2520to%2520advance%2520a%2520second%2520generation%2520of%2520HIV-1%2520integrase%2520strand%2520transfer%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8154%26epage%3D8165%26doi%3D10.1021%2Facs.jmedchem.5b01037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maskell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pye, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.</span><span> </span><span class="NLM_article-title">HIV-1 integrase strand transfer inhibitors with reduced susceptibility to drug resistant mutant integrases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1081</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00948</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00948" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1074-1081&author=X.+Z.+Zhaoauthor=S.+Smithauthor=D.+Maskellauthor=M.+Metifiotauthor=V.+Pyeauthor=K.+Fesenauthor=C.+Marchandauthor=Y.+Pommierauthor=P.+Cherepanovauthor=S.+Hughesauthor=T.+R.+Burke&title=HIV-1+integrase+strand+transfer+inhibitors+with+reduced+susceptibility+to+drug+resistant+mutant+integrases&doi=10.1021%2Facschembio.5b00948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00948%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DMaskell%26aufirst%3DD.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DPye%26aufirst%3DV.%26aulast%3DFesen%26aufirst%3DK.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DCherepanov%26aufirst%3DP.%26aulast%3DHughes%26aufirst%3DS.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3DHIV-1%2520integrase%2520strand%2520transfer%2520inhibitors%2520with%2520reduced%2520susceptibility%2520to%2520drug%2520resistant%2520mutant%2520integrases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1074%26epage%3D1081%26doi%3D10.1021%2Facschembio.5b00948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Wainberg, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesplede, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raffi, F.</span><span> </span><span class="NLM_article-title">What if HIV were unable to develop resistance against a new therapeutic agent?</span> <span class="citation_source-journal">BMC Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="refDoi"> DOI: 10.1186/1741-7015-11-249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1186%2F1741-7015-11-249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24267867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntVais7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=249&author=M.+A.+Wainbergauthor=T.+Mespledeauthor=F.+Raffi&title=What+if+HIV+were+unable+to+develop+resistance+against+a+new+therapeutic+agent%3F&doi=10.1186%2F1741-7015-11-249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">What if HIV were unable to develop resistance against a new therapeutic agent?</span></div><div class="casAuthors">Wainberg, Mark A.; Mesplede, Thibault; Raffi, Francois</div><div class="citationInfo"><span class="NLM_cas:title">BMC Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249/1-249/6</span>CODEN:
                <span class="NLM_cas:coden">BMMECZ</span>;
        ISSN:<span class="NLM_cas:issn">1741-7015</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Background: The HIV integrase inhibitor, Dolutegravir (DTG), was recently approved by the Food and Drug Administration in the United States and is the only HIV drug that has not selected for resistance mutations in the clinic when used as part of first-line therapy.  This has led to speculation that DTG might have a higher genetic barrier for the development of drug resistance than the other compds. that are used in therapy.  Discussion: In this Opinion article, we speculate that this is due to greatly diminished replication capacity on the part of viruses that might become resistant to DTG when the drug is used in initial therapy and that DTG might be able to be used in HIV prevention and eradication strategies.  We also note that no compensatory mutation that might restore viral replication fitness to HIV in the aftermath of the appearance of a single drug resistance mutation has yet to be obsd.  Summary: DTG is a valuable addn. to the anti-HIV armamentarium of drugs and its long-term utility may potentially exceed its obvious use in treatment of HIV disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqiJYPc1BhMrVg90H21EOLACvtfcHk0ljVrhHqCyTbuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntVais7w%253D&md5=7a72d67fa56360e1b82f567e44245582</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1741-7015-11-249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1741-7015-11-249%26sid%3Dliteratum%253Aachs%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DMesplede%26aufirst%3DT.%26aulast%3DRaffi%26aufirst%3DF.%26atitle%3DWhat%2520if%2520HIV%2520were%2520unable%2520to%2520develop%2520resistance%2520against%2520a%2520new%2520therapeutic%2520agent%253F%26jtitle%3DBMC%2520Med.%26date%3D2013%26volume%3D11%26spage%3D249%26doi%3D10.1186%2F1741-7015-11-249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Shah, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeSimone, J. A.,  Jr.</span><span> </span><span class="NLM_article-title">Dolutegravir: A new integrase strand transfer inhibitor for the treatment of HIV</span> <span class="citation_source-journal">Pharmacotherapy</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">506</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1002/phar.1386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1002%2Fphar.1386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24347095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvF2ktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=506-520&author=B.+M.+Shahauthor=J.+J.+Schaferauthor=J.+A.+DeSimone&title=Dolutegravir%3A+A+new+integrase+strand+transfer+inhibitor+for+the+treatment+of+HIV&doi=10.1002%2Fphar.1386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dolutegravir: A New Integrase Strand Transfer Inhibitor for the Treatment of HIV</span></div><div class="casAuthors">Shah, Bhavik M.; Schafer, Jason J.; De Simone, Joseph A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">506-520</span>CODEN:
                <span class="NLM_cas:coden">PHPYDQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-9114</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The first two integrase strand transfer inhibitors (INSTIs) approved for treatment of patients infected with human immunodeficiency virus (HIV) were raltegravir and elvitegravir.  Both raltegravir and elvitegravir are now guideline-preferred agents as part of an antiretroviral regimen for treatment-naive patients.  However, raltegravir is dosed twice/day.  Elvitegravir is available in a single-tablet regimen and dosed once/day because it is administered with the pharmacokinetic booster cobicistat, a potent CYP3A4 inhibitor that can lead to clin. significant drug-drug interactions.  In addn., raltegravir and elvitegravir have a low genetic barrier to resistance and are assocd. with cross-resistance.  Dolutegravir is a new-generation INSTI administered once/day without a pharmacokinetic booster and can be coformulated in a single-tablet regimen.  Phase III studies have demonstrated the efficacy and safety of dolutegravir for treatment-naive and treatment-experienced patients.  Compared with other INSTIs, dolutegravir has a higher genetic barrier to resistance.  Dolutegravir was approved by the U.S. Food and Drug Administration in August 2013 and joins raltegravir and elvitegravir as guideline-preferred agents for the management for HIV-infected treatment-naive patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5J0O3fs7R87Vg90H21EOLACvtfcHk0lhZ6QqZwEQy1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvF2ktLg%253D&md5=41c2865b578fae8326d39c8c0f94cbf7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fphar.1386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fphar.1386%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DB.%2BM.%26aulast%3DSchafer%26aufirst%3DJ.%2BJ.%26aulast%3DDeSimone%26aufirst%3DJ.%2BA.%26atitle%3DDolutegravir%253A%2520A%2520new%2520integrase%2520strand%2520transfer%2520inhibitor%2520for%2520the%2520treatment%2520of%2520HIV%26jtitle%3DPharmacotherapy%26date%3D2014%26volume%3D34%26spage%3D506%26epage%3D520%26doi%3D10.1002%2Fphar.1386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Malet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gimferrer Arriaga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artese, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrotta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcaro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delelis, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tmeizeh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katlama, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valantin, M.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccherini-Silberstein, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcelin, A.-G.</span><span> </span><span class="NLM_article-title">New raltegravir resistance pathways induce broad cross-resistance to all currently used integrase inhibitors</span> <span class="citation_source-journal">J. Antimicrob. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2118</span><span class="NLM_x">–</span> <span class="NLM_lpage">2122</span><span class="refDoi"> DOI: 10.1093/jac/dku095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1093%2Fjac%2Fdku095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24710029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1SqsbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=2118-2122&author=I.+Maletauthor=L.+Gimferrer+Arriagaauthor=A.+Arteseauthor=G.+Costaauthor=L.+Parrottaauthor=S.+Alcaroauthor=O.+Delelisauthor=A.+Tmeizehauthor=C.+Katlamaauthor=M.-A.+Valantinauthor=F.+Ceccherini-Silbersteinauthor=V.+Calvezauthor=A.-G.+Marcelin&title=New+raltegravir+resistance+pathways+induce+broad+cross-resistance+to+all+currently+used+integrase+inhibitors&doi=10.1093%2Fjac%2Fdku095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">New raltegravir resistance pathways induce broad cross-resistance to all currently used integrase inhibitors</span></div><div class="casAuthors">Malet, Isabelle; Gimferrer Arriaga, Laura; Artese, Anna; Costa, Giosue; Parrotta, Lucia; Alcaro, Stefano; Delelis, Olivier; Tmeizeh, Ahmed; Katlama, Christine; Valantin, Marc-Antoine; Ceccherini-Silberstein, Francesca; Calvez, Vincent; Marcelin, Anne-Genevieve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2118-2122</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The possibility of replacing raltegravir or elvitegravir with dolutegravir in heavily treatment-experienced patients failing on raltegravir/elvitegravir has been evaluated in VIKING trials.  All studied patients failed by the most common pathways, Y143, Q148 and N155, and dolutegravir demonstrated efficacy except for Q148 viruses.  The aim of this study was to explore, in the same way, the behavior of dolutegravir in comparison with raltegravir and elvitegravir against the atypical resistance integrase profiles, G118R and F121Y, described in HIV-1 patients failing on raltegravir therapy.  The behavior of integrases with mutations G118R and F121Y towards raltegravir, elvitegravir and dolutegravir was analyzed by evaluating phenotypic susceptibility and by means of in silico techniques (investigating binding affinities and the stabilization of the inhibitors in terms of their hydrogen bond network).  Results The phenotypic anal. of G118R and F121Y showed high resistance to raltegravir, elvitegravir and dolutegravir with a fold change >100 when the clin. derived integrase was used, and resistance was also seen when mutations were tested alone in an NL43 backbone, but more often with a lower fold change.  In silico, results showed that G118R and F121Y enzymes were assocd. with reduced binding affinities to each of the inhibitors and with a decreased no. of hydrogen bonds compared with the wild-type complexes.  This study showed that G118R and F121Y mutations, rarely described in patients failing on raltegravir, induced broad cross-resistance to all currently used integrase inhibitors.  These results are in accordance with our thermodn. and geometric anal. indicating decreased stability compared with the wild-type complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5AjEYyjsbn7Vg90H21EOLACvtfcHk0lhZ6QqZwEQy1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1SqsbfM&md5=376a841a2a93f8dea573f89874c12c9b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdku095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdku095%26sid%3Dliteratum%253Aachs%26aulast%3DMalet%26aufirst%3DI.%26aulast%3DGimferrer%2BArriaga%26aufirst%3DL.%26aulast%3DArtese%26aufirst%3DA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DParrotta%26aufirst%3DL.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DDelelis%26aufirst%3DO.%26aulast%3DTmeizeh%26aufirst%3DA.%26aulast%3DKatlama%26aufirst%3DC.%26aulast%3DValantin%26aufirst%3DM.-A.%26aulast%3DCeccherini-Silberstein%26aufirst%3DF.%26aulast%3DCalvez%26aufirst%3DV.%26aulast%3DMarcelin%26aufirst%3DA.-G.%26atitle%3DNew%2520raltegravir%2520resistance%2520pathways%2520induce%2520broad%2520cross-resistance%2520to%2520all%2520currently%2520used%2520integrase%2520inhibitors%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D2118%26epage%3D2122%26doi%3D10.1093%2Fjac%2Fdku095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">King, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prabu-Jeyabalan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nalivaika, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiffer, C. A.</span><span> </span><span class="NLM_article-title">Combating susceptibility to drug resistance: Lessons from HIV-1 protease</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1333</span><span class="NLM_x">–</span> <span class="NLM_lpage">1338</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2004.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1016%2Fj.chembiol.2004.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=15489160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1333-1338&author=N.+M.+Kingauthor=M.+Prabu-Jeyabalanauthor=E.+A.+Nalivaikaauthor=C.+A.+Schiffer&title=Combating+susceptibility+to+drug+resistance%3A+Lessons+from+HIV-1+protease&doi=10.1016%2Fj.chembiol.2004.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Susceptibility to Drug Resistance Lessons from HIV-1 Protease</span></div><div class="casAuthors">King, Nancy M.; Prabu-Jeyabalan, Moses; Nalivaika, Ellen A.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1333-1338</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Drug resistance is a major obstacle in modern medicine.  However, resistance is rarely considered in drug development and may inadvertently be facilitated, as many designed inhibitors contact residues that can mutate to confer resistance, without significantly impairing function.  Contemporary drug design often ignores the detailed at. basis for function and primarily focuses on disrupting the target's activity, which is necessary but not sufficient for developing a robust drug.  In this study, we examine the impact of drug-resistant mutations in HIV-1 protease on substrate recognition and demonstrate that most primary active site mutations do not extensively contact substrates, but are crit. to inhibitor binding.  We propose a general, structure-based strategy to reduce the probability of drug resistance by designing inhibitors that interact only with those residues that are essential for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbsk2AyTcCmbVg90H21EOLACvtfcHk0lhZ6QqZwEQy1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D&md5=701599b15cf9ecc2c026ad4eac650a90</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DCombating%2520susceptibility%2520to%2520drug%2520resistance%253A%2520Lessons%2520from%2520HIV-1%2520protease%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1333%26epage%3D1338%26doi%3D10.1016%2Fj.chembiol.2004.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Hirokawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, S.</span><span> </span><span class="NLM_article-title">Synthesis and structure–affinity relationships of novel N-(1-ethyl-4-methylhexahydro-1,4-diazepin-6-yl)pyridine-3-carboxamides with potent serotonin 5-HT3 and dopamine D2 receptor antagonistic activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">702</span><span class="NLM_x">–</span> <span class="NLM_lpage">715</span><span class="refDoi"> DOI: 10.1021/jm020270n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020270n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=702-715&author=Y.+Hirokawaauthor=I.+Fujiwaraauthor=K.+Suzukiauthor=H.+Haradaauthor=T.+Yoshikawaauthor=N.+Yoshidaauthor=S.+Kato&title=Synthesis+and+structure%E2%80%93affinity+relationships+of+novel+N-%281-ethyl-4-methylhexahydro-1%2C4-diazepin-6-yl%29pyridine-3-carboxamides+with+potent+serotonin+5-HT3+and+dopamine+D2+receptor+antagonistic+activity&doi=10.1021%2Fjm020270n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm020270n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020270n%26sid%3Dliteratum%253Aachs%26aulast%3DHirokawa%26aufirst%3DY.%26aulast%3DFujiwara%26aufirst%3DI.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DHarada%26aufirst%3DH.%26aulast%3DYoshikawa%26aufirst%3DT.%26aulast%3DYoshida%26aufirst%3DN.%26aulast%3DKato%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520structure%25E2%2580%2593affinity%2520relationships%2520of%2520novel%2520N-%25281-ethyl-4-methylhexahydro-1%252C4-diazepin-6-yl%2529pyridine-3-carboxamides%2520with%2520potent%2520serotonin%25205-HT3%2520and%2520dopamine%2520D2%2520receptor%2520antagonistic%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D702%26epage%3D715%26doi%3D10.1021%2Fjm020270n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">LaFemina, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeGrow, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hastings, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emini, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span> </span><span class="NLM_article-title">Inhibition of human immunodeficiency virus integrase by bis-catechols</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">320</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1128/AAC.39.2.320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1128%2FAAC.39.2.320" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=320-324&author=R.+L.+LaFeminaauthor=P.+L.+Grahamauthor=K.+LeGrowauthor=J.+C.+Hastingsauthor=A.+Wolfeauthor=S.+D.+Youngauthor=E.+A.+Eminiauthor=D.+J.+Hazuda&title=Inhibition+of+human+immunodeficiency+virus+integrase+by+bis-catechols&doi=10.1128%2FAAC.39.2.320"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1128%2FAAC.39.2.320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.39.2.320%26sid%3Dliteratum%253Aachs%26aulast%3DLaFemina%26aufirst%3DR.%2BL.%26aulast%3DGraham%26aufirst%3DP.%2BL.%26aulast%3DLeGrow%26aufirst%3DK.%26aulast%3DHastings%26aufirst%3DJ.%2BC.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DS.%2BD.%26aulast%3DEmini%26aufirst%3DE.%2BA.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26atitle%3DInhibition%2520of%2520human%2520immunodeficiency%2520virus%2520integrase%2520by%2520bis-catechols%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1995%26volume%3D39%26spage%3D320%26epage%3D324%26doi%3D10.1128%2FAAC.39.2.320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Charpentier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karmochkine, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laureillard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisserand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belec, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Si-Mohamed, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piketty, C.</span><span> </span><span class="NLM_article-title">Drug resistance profiles for the HIV integrase gene in patients failing raltegravir salvage therapy</span> <span class="citation_source-journal">HIV Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">765</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1111/j.1468-1293.2008.00628.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1111%2Fj.1468-1293.2008.00628.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=18651855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVemtr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=765-70&author=C.+Charpentierauthor=M.+Karmochkineauthor=D.+Laureillardauthor=P.+Tisserandauthor=L.+Belecauthor=L.+Weissauthor=A.+Si-Mohamedauthor=C.+Piketty&title=Drug+resistance+profiles+for+the+HIV+integrase+gene+in+patients+failing+raltegravir+salvage+therapy&doi=10.1111%2Fj.1468-1293.2008.00628.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Drug resistance profiles for the HIV integrase gene in patients failing raltegravir salvage therapy</span></div><div class="casAuthors">Charpentier, C.; Karmochkine, M.; Laureillard, D.; Tisserand, P.; Belec, L.; Weiss, L.; Si-Mohamed, A.; Piketty, C.</div><div class="citationInfo"><span class="NLM_cas:title">HIV Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">765-770</span>CODEN:
                <span class="NLM_cas:coden">HMIEAB</span>;
        ISSN:<span class="NLM_cas:issn">1464-2662</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Objectives: Recent data showed the selection of mutations in the integrase gene, mainly involving position 148 or 155, in patients displaying virol. failure (VF) on raltegravir (RAL) therapy.  Here, we describe the development of RAL resistance, in both plasmatic and cellular compartments, in three heavily pretreated HIV-infected patients failing RAL-contg. regimens.  Methods: Three of 17 patients receiving RAL displayed VF.  The entire integrase gene, isolated from plasma and peripheral blood mononuclear cells (PBMC), was sequenced.  A clonal anal. was performed in one patient at the time of VF.  Results: At the time of VF, RAL-resistance mutations were selected: (i) Q148R in patients 1 and 3; (ii) T66A and E92Q in patient 2.  A gradual accumulation of new mutations was obsd. in all patients, including G140S, Q148H and N155H in patient 1, L74I in patient 2, and G140S in patient 3.  Clonal anal. showed the coexistence, in patient 1, of the two common resistance pathways (mutations Q148R/H and N155H) found in distinct quasi-species.  In addn., RAL-resistance mutations were detected in HIV DNA in all patients.  Conclusions: Having rapidly established, resistance to RAL evolves and diversifies, and is likely to impact the efficacy of subsequently used second-generation integrase inhibitors.  Moreover, RAL-resistance mutations can be archived early in PBMC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAzqme5X5_G7Vg90H21EOLACvtfcHk0lhY_IKZ99ZhDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVemtr%252FJ&md5=ae94b8c464b80ebb8dd2795d87dc225f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1111%2Fj.1468-1293.2008.00628.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1468-1293.2008.00628.x%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DC.%26aulast%3DKarmochkine%26aufirst%3DM.%26aulast%3DLaureillard%26aufirst%3DD.%26aulast%3DTisserand%26aufirst%3DP.%26aulast%3DBelec%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DL.%26aulast%3DSi-Mohamed%26aufirst%3DA.%26aulast%3DPiketty%26aufirst%3DC.%26atitle%3DDrug%2520resistance%2520profiles%2520for%2520the%2520HIV%2520integrase%2520gene%2520in%2520patients%2520failing%2520raltegravir%2520salvage%2520therapy%26jtitle%3DHIV%2520Med.%26date%3D2008%26volume%3D9%26spage%3D765%26epage%3D70%26doi%3D10.1111%2Fj.1468-1293.2008.00628.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Cooper, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steigbigel, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rockstroh, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katlama, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazzarin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clotet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eron, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schechter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markowitz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loutfy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennox, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Killar, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilde, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strohmaier, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meibohm, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nessly, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNubile, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaacs, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teppler, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, B.-Y.</span><span> </span><span class="NLM_article-title">Subgroup and resistance analyses of raltegravir for resistant HIV-1 infection</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span><span class="refDoi"> DOI: 10.1056/NEJMoa0708978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1056%2FNEJMoa0708978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=18650513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2008&pages=355-365&author=D.+A.+Cooperauthor=R.+T.+Steigbigelauthor=J.+M.+Gatellauthor=J.+K.+Rockstrohauthor=C.+Katlamaauthor=P.+Yeniauthor=A.+Lazzarinauthor=B.+Clotetauthor=P.+N.+Kumarauthor=J.+E.+Eronauthor=M.+Schechterauthor=M.+Markowitzauthor=M.+R.+Loutfyauthor=J.+L.+Lennoxauthor=J.+Zhaoauthor=J.+Chenauthor=D.+M.+Ryanauthor=R.+R.+Rhodesauthor=J.+A.+Killarauthor=L.+R.+Gildeauthor=K.+M.+Strohmaierauthor=A.+R.+Meibohmauthor=M.+D.+Millerauthor=D.+J.+Hazudaauthor=M.+L.+Nesslyauthor=M.+J.+DiNubileauthor=R.+D.+Isaacsauthor=H.+Tepplerauthor=B.-Y.+Nguyen&title=Subgroup+and+resistance+analyses+of+raltegravir+for+resistant+HIV-1+infection&doi=10.1056%2FNEJMoa0708978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Subgroup and resistance analyses of raltegravir for resistant HIV-1 infection</span></div><div class="casAuthors">Cooper, David A.; Steigbigel, Roy T.; Gatell, Jose M.; Rockstroh, Jurgen K.; Katlama, Christine; Yeni, Patrick; Lazzarin, Adriano; Clotet, Bonaventura; Kumar, Princy N.; Eron, Joseph E.; Schechter, Mauro; Markowitz, Martin; Loutfy, Mona R.; Lennox, Jeffrey L.; Zhao, Jing; Chen, Joshua; Ryan, Desmond M.; Rhodes, Rand R.; Killar, John A.; Gilde, Lucinda R.; Strohmaier, Kim M.; Meibohm, Anne R.; Miller, Michael D.; Hazuda, Daria J.; Nessly, Michael L.; DiNubile, Mark J.; Isaacs, Robin D.; Teppler, Hedy; Nguyen, Bach-Yen</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">We evaluated the efficacy of raltegravir and the development of viral resistance in two identical trials involving patients who were infected with human immunodeficiency virus type 1 (HIV-1) with triple-class drug resistance and in whom antiretroviral therapy had failed.  We conducted subgroup analyses of the data from week 48 in both studies according to baseline prognostic factors.  Genotyping of the integrase gene was performed in raltegravir recipients who had virol. failure.  Virol. responses to raltegravir were consistently superior to responses to placebo, regardless of the baseline values of HIV-1 RNA level; CD4 cell count; genotypic or phenotypic sensitivity score; use or nonuse of darunavir, enfuvirtide, or both in optimized background therapy; or demog. characteristics.  Among patients in the two studies combined who were using both enfuvirtide and darunavir for the first time, HIV-1 RNA levels of less than 50 copies per mL were achieved in 89% of raltegravir recipients and 68% of placebo recipients.  HIV-1 RNA levels of less than 50 copies per mL were achieved in 69% and 80% of the raltegravir recipients and in 47% and 57% of the placebo recipients using either darunavir or enfuvirtide for the first time, resp.  At 48 wk, 105 of the 462 raltegravir recipients (23%) had virol. failure.  Genotyping was performed in 94 raltegravir recipients with virol. failure.  Integrase mutations known to be assocd. with phenotypic resistance to raltegravir arose during treatment in 64 patients (68%).  Forty-eight of these 64 patients (75%) had two or more resistance-assocd. mutations.  When combined with an optimized background regimen in both studies, a consistently favorable treatment effect of raltegravir over placebo was shown in clin. relevant subgroups of patients, including those with baseline characteristics that typically predict a poor response to antiretroviral therapy: a high HIV-1 RNA level, low CD4 cell count, and low genotypic or phenotypic sensitivity score.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdQbxOYiV4b7Vg90H21EOLACvtfcHk0lhY_IKZ99ZhDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFSqtb8%253D&md5=cd4bc42edc03c66485d399ba620477cc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0708978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0708978%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DD.%2BA.%26aulast%3DSteigbigel%26aufirst%3DR.%2BT.%26aulast%3DGatell%26aufirst%3DJ.%2BM.%26aulast%3DRockstroh%26aufirst%3DJ.%2BK.%26aulast%3DKatlama%26aufirst%3DC.%26aulast%3DYeni%26aufirst%3DP.%26aulast%3DLazzarin%26aufirst%3DA.%26aulast%3DClotet%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%2BN.%26aulast%3DEron%26aufirst%3DJ.%2BE.%26aulast%3DSchechter%26aufirst%3DM.%26aulast%3DMarkowitz%26aufirst%3DM.%26aulast%3DLoutfy%26aufirst%3DM.%2BR.%26aulast%3DLennox%26aufirst%3DJ.%2BL.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DRyan%26aufirst%3DD.%2BM.%26aulast%3DRhodes%26aufirst%3DR.%2BR.%26aulast%3DKillar%26aufirst%3DJ.%2BA.%26aulast%3DGilde%26aufirst%3DL.%2BR.%26aulast%3DStrohmaier%26aufirst%3DK.%2BM.%26aulast%3DMeibohm%26aufirst%3DA.%2BR.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DNessly%26aufirst%3DM.%2BL.%26aulast%3DDiNubile%26aufirst%3DM.%2BJ.%26aulast%3DIsaacs%26aufirst%3DR.%2BD.%26aulast%3DTeppler%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DB.-Y.%26atitle%3DSubgroup%2520and%2520resistance%2520analyses%2520of%2520raltegravir%2520for%2520resistant%2520HIV-1%2520infection%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2008%26volume%3D359%26spage%3D355%26epage%3D365%26doi%3D10.1056%2FNEJMoa0708978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nalam, M. N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rana, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiffer, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tidor, B.</span><span> </span><span class="NLM_article-title">Testing the substrate-envelope hypothesis with designed pairs of compounds</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2433</span><span class="NLM_x">–</span> <span class="NLM_lpage">2441</span><span class="refDoi"> DOI: 10.1021/cb400468c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400468c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1yiur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2433-2441&author=Y.+Shenauthor=M.+D.+Altmanauthor=A.+Aliauthor=M.+N.+L.+Nalamauthor=H.+Caoauthor=T.+M.+Ranaauthor=C.+A.+Schifferauthor=B.+Tidor&title=Testing+the+substrate-envelope+hypothesis+with+designed+pairs+of+compounds&doi=10.1021%2Fcb400468c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Testing the substrate-envelope hypothesis with designed pairs of compounds</span></div><div class="casAuthors">Shen, Yang; Altman, Michael D.; Ali, Akbar; Nalam, Madhavi N. L.; Cao, Hong; Rana, Tariq M.; Schiffer, Celia A.; Tidor, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2433-2441</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acquired resistance to therapeutic agents is a significant barrier to the development of clin. effective treatments for diseases in which evolution occurs on clin. time scales, frequently arising from target mutations.  We previously reported a general strategy to design effective inhibitors for rapidly mutating enzyme targets, which we demonstrated for HIV-1 protease inhibition.  Specifically, we developed a computational inverse design procedure with the added constraint that designed inhibitors bind entirely inside the substrate envelope, a consensus vol. occupied by natural substrates.  The rationale for the substrate-envelope constraint is that it prevents designed inhibitors from making interactions beyond those required by substrates and thus limits the availability of mutations tolerated by substrates but not by designed inhibitors.  The strategy resulted in subnanomolar inhibitors that bind robustly across a clin. derived panel of drug-resistant variants.  To further test the substrate-envelope hypothesis, here we have designed, synthesized, and assayed derivs. of our original compds. that are larger and extend outside the substrate envelope.  Our designs resulted in pairs of compds. that are very similar to one another, but one respects and one violates the substrate envelope.  The envelope-respecting inhibitor demonstrates robust binding across a panel of drug-resistant protease variants, whereas the envelope-violating one binds tightly to wild type but loses affinity to at least one variant.  This study provides strong support for the substrate-envelope hypothesis as a design strategy for inhibitors that reduce susceptibility to resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxEZ752OIVobVg90H21EOLACvtfcHk0liFMfCkSkfDgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1yiur%252FE&md5=8febc495df52db0dd9463e25a2f5284d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fcb400468c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400468c%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DNalam%26aufirst%3DM.%2BN.%2BL.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DRana%26aufirst%3DT.%2BM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DTidor%26aufirst%3DB.%26atitle%3DTesting%2520the%2520substrate-envelope%2520hypothesis%2520with%2520designed%2520pairs%2520of%2520compounds%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2433%26epage%3D2441%26doi%3D10.1021%2Fcb400468c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Das, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauman, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frenkel, Y. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewi, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatkin, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, E.</span><span> </span><span class="NLM_article-title">High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: Strategic flexibility explains potency against resistance mutations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">1471</span><span class="refDoi"> DOI: 10.1073/pnas.0711209105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1073%2Fpnas.0711209105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=18230722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1466-1471&author=K.+Dasauthor=J.+D.+Baumanauthor=A.+D.+Clarkauthor=Y.+V.+Frenkelauthor=P.+J.+Lewiauthor=A.+J.+Shatkinauthor=S.+H.+Hughesauthor=E.+Arnold&title=High-resolution+structures+of+HIV-1+reverse+transcriptase%2FTMC278+complexes%3A+Strategic+flexibility+explains+potency+against+resistance+mutations&doi=10.1073%2Fpnas.0711209105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: strategic flexibility explains potency against resistance mutations</span></div><div class="casAuthors">Das, Kalyan; Bauman, Joseph D.; Clark, Arthur D.; Frenkel, Yulia V.; Lewi, Paul J.; Shatkin, Aaron J.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1466-1471</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">TMC278 is a diarylpyrimidine (DAPY) nonnucleoside reverse transcriptase inhibitor (NNRTI) that is highly effective in treating wild-type and drug-resistant HIV-1 infections in clin. trials at relatively low doses (∼ 25-75 mg/day).  We have detd. the structure of wild-type HIV-1 RT complexed with TMC278 at 1.8 Å resoln., using an RT crystal form engineered by systematic RT mutagenesis.  This high-resoln. structure reveals that the cyanovinyl group of TMC278 is positioned in a hydrophobic tunnel connecting the NNRTI-binding pocket to the nucleic acid-binding cleft.  The crystal structures of TMC278 in complexes with the double mutant K103N/Y181C (2.1 Å) and L100I/K103N HIV-1 RTs (2.9 Å) demonstrated that TMC278 adapts to bind mutant RTs.  In the K103N/Y181C RT/TMC278 structure, loss of the arom. ring interaction caused by the Y181C mutation is counterbalanced by interactions between the cyanovinyl group of TMC278 and the arom. side chain of Y183, which is facilitated by an ∼ 1.5 Å shift of the conserved Y183MDD motif.  In the L1001/K103N RT/ TMC278 structure, the binding mode of TMC278 is significantly altered so that the drug conforms to changes in the binding pocket primarily caused by the L1001 mutation.  The flexible binding pocket acts as a mol. "shrink wrap" that makes a shape complementary to the optimized TMC278 in wild-type and drug-resistant forms of HIV-1 RT.  The crystal structures provide a better understanding of how the flexibility of an inhibitor can compensate for drug-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXRbleFGfFk7Vg90H21EOLACvtfcHk0liFMfCkSkfDgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D&md5=5f2ea5e4d88e9822ceecee636e1e4bd1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711209105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711209105%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DFrenkel%26aufirst%3DY.%2BV.%26aulast%3DLewi%26aufirst%3DP.%2BJ.%26aulast%3DShatkin%26aufirst%3DA.%2BJ.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHigh-resolution%2520structures%2520of%2520HIV-1%2520reverse%2520transcriptase%252FTMC278%2520complexes%253A%2520Strategic%2520flexibility%2520explains%2520potency%2520against%2520resistance%2520mutations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D1466%26epage%3D1471%26doi%3D10.1073%2Fpnas.0711209105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Hirokawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, S.</span><span> </span><span class="NLM_article-title">An efficient synthesis of 5-bromo-2-methoxy-6methylaminopyridine-3-carboxylic acid</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1847</span><span class="NLM_x">–</span> <span class="NLM_lpage">1853</span><span class="refDoi"> DOI: 10.1248/cpb.48.1847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1248%2Fcpb.48.1847" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2000&pages=1847-1853&author=Y.+Hirokawaauthor=T.+Horikawaauthor=S.+Kato&title=An+efficient+synthesis+of+5-bromo-2-methoxy-6methylaminopyridine-3-carboxylic+acid&doi=10.1248%2Fcpb.48.1847"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1248%2Fcpb.48.1847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.48.1847%26sid%3Dliteratum%253Aachs%26aulast%3DHirokawa%26aufirst%3DY.%26aulast%3DHorikawa%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DS.%26atitle%3DAn%2520efficient%2520synthesis%2520of%25205-bromo-2-methoxy-6methylaminopyridine-3-carboxylic%2520acid%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2000%26volume%3D48%26spage%3D1847%26epage%3D1853%26doi%3D10.1248%2Fcpb.48.1847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddali, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Activities, crystal structures and molecular dynamics of dihydro-1H-isoindole derivatives, inhibitors of HIV-1 integrase</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span><span class="refDoi"> DOI: 10.1021/cb300471n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300471n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=209-217&author=M.+Metifiotauthor=K.+Maddaliauthor=B.+C.+Johnsonauthor=S.+Hareauthor=S.+J.+Smithauthor=X.+Z.+Zhaoauthor=C.+Marchandauthor=T.+R.+Burkeauthor=S.+H.+Hughesauthor=P.+Cherepanovauthor=Y.+Pommier&title=Activities%2C+crystal+structures+and+molecular+dynamics+of+dihydro-1H-isoindole+derivatives%2C+inhibitors+of+HIV-1+integrase&doi=10.1021%2Fcb300471n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fcb300471n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300471n%26sid%3Dliteratum%253Aachs%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DMaddali%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DB.%2BC.%26aulast%3DHare%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DCherepanov%26aufirst%3DP.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DActivities%252C%2520crystal%2520structures%2520and%2520molecular%2520dynamics%2520of%2520dihydro-1H-isoindole%2520derivatives%252C%2520inhibitors%2520of%2520HIV-1%2520integrase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D209%26epage%3D217%26doi%3D10.1021%2Fcb300471n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddali, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span> </span><span class="NLM_article-title">Bicyclic hydroxy-1H-pyrrolopyridine-trione containing HIV-1 integrase inhibitors</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">165</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01270.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1111%2Fj.1747-0285.2011.01270.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=22107736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2012&pages=157-165&author=X.+Z.+Zhaoauthor=K.+Maddaliauthor=M.+Metifiotauthor=S.+J.+Smithauthor=B.+C.+Vuauthor=C.+Marchandauthor=S.+H.+Hughesauthor=Y.+Pommierauthor=T.+R.+Burke&title=Bicyclic+hydroxy-1H-pyrrolopyridine-trione+containing+HIV-1+integrase+inhibitors&doi=10.1111%2Fj.1747-0285.2011.01270.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bicyclic hydroxy-1H-pyrrolopyridine-trione containing HIV-1 integrase inhibitors</span></div><div class="casAuthors">Zhao, Xue Zhi; Maddali, Kasthuraiah; Metifiot, Mathieu; Smith, Steven J.; Vu, B. Christie; Marchand, Christophe; Hughes, Stephen H.; Pommier, Yves; Burke, Terrence R., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-165</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">HIV-1 integrase (IN) is a validated therapeutic target for the treatment of AIDS.  However, the emergence of resistance to raltegravir, the sole marketed FDA-approved IN inhibitor, emphasizes the need to develop second-generation inhibitors that retain efficacy against clin. relevant IN mutants.  We report herein bicyclic hydroxy-1H-pyrrolopyridinetriones as a new family of HIV-1 integrase inhibitors that were efficiently prepd. using a key "Pummerer cyclization deprotonation cycloaddn." cascade of imidosulfoxides.  In in vitro HIV-1 integrase assays, the analogs showed low micromolar inhibitory potencies with selectivity for strand transfer reactions as compared with 3'-processing inhibition.  A representative chlorofluorobenzyl hydroxypyrrolopyridinetrione retained most of its inhibitory potency against the three major raltegravir-resistant IN mutant enzymes, G140S/Q148H, Y143R, and N155H.  In antiviral assays employing viral vectors coding these IN mutants, the chlorofluorobenzyl hydroxypyrrolopyridinetrione was approx. 200- and 20-fold less affected than raltegravir against the G140S/Q148H and Y143R mutations, resp.  The representative chlorofluorobenzyl hydroxypyrrolopyridinetrione was affected approx. 10-fold less by the N155H mutation than raltegravir.  Hydroxypyrrolopyridinetriones are a novel structural class that may be further optimized to obtain HIV-1 integrase inhibitors capable of overcoming raltegravir resistance in HIV-1 strains possessing G140S/Q148H mutations in HIV-1 integrase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_-V1xazfn1LVg90H21EOLACvtfcHk0ljmvrvkV7wGgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWrtLY%253D&md5=467e940ff1b737e55486e8f578db6da0</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01270.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01270.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DMaddali%26aufirst%3DK.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DVu%26aufirst%3DB.%2BC.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3DBicyclic%2520hydroxy-1H-pyrrolopyridine-trione%2520containing%2520HIV-1%2520integrase%2520inhibitors%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2012%26volume%3D79%26spage%3D157%26epage%3D165%26doi%3D10.1111%2Fj.1747-0285.2011.01270.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Brachmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koenig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Julius, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldbruegge, M.</span><span> </span><span class="NLM_article-title">A reverse genetic approach for generating gene replacement mutants in Ustilago maydis</span> <span class="citation_source-journal">Mol. Genet. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">226</span><span class="refDoi"> DOI: 10.1007/s00438-004-1047-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1007%2Fs00438-004-1047-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=15316769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD2cXns1eqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=2004&pages=216-226&author=A.+Brachmannauthor=J.+Koenigauthor=C.+Juliusauthor=M.+Feldbruegge&title=A+reverse+genetic+approach+for+generating+gene+replacement+mutants+in+Ustilago+maydis&doi=10.1007%2Fs00438-004-1047-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A reverse genetic approach for generating gene replacement mutants in Ustilago maydis</span></div><div class="casAuthors">Brachmann, A.; Koenig, J.; Julius, C.; Feldbruegge, M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics and Genomics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">216-226</span>CODEN:
                <span class="NLM_cas:coden">MGGOAA</span>;
        ISSN:<span class="NLM_cas:issn">1617-4615</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">We describe a versatile strategy for generating gene replacement mutants in the phytopathogenic fungus Ustilago maydis.  The system includes the choice of 32 different insertion cassettes for genetic engineering purposes, such as gene disruption and more sophisticated insertions of reporter genes, heterologous promoters or combinations of the two.  PCR-amplified flanking sequences needed for homologous recombination are ligated to the resp. insertion cassettes via Sfi I sites.  As proof of principle we generated two replacement mutants in which the endogenous promoter of the pheromone gene mfa1 drives expression of the Green Fluorescent Protein gene (gfp).  Simultaneously, expression of the mfa1 ORF is controlled either by the carbon source-regulated crg1 promoter or the nitrogen source-regulated nar1 promoter.  In both cases gfp expression was pheromone-inducible and pheromone expression was only detected when the heterologous promoters were active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqckY_KQd6m37Vg90H21EOLACvtfcHk0ljmvrvkV7wGgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXns1eqsLg%253D&md5=ae76faf1e45e14ef570b2cf60e8d67a3</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs00438-004-1047-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00438-004-1047-z%26sid%3Dliteratum%253Aachs%26aulast%3DBrachmann%26aufirst%3DA.%26aulast%3DKoenig%26aufirst%3DJ.%26aulast%3DJulius%26aufirst%3DC.%26aulast%3DFeldbruegge%26aufirst%3DM.%26atitle%3DA%2520reverse%2520genetic%2520approach%2520for%2520generating%2520gene%2520replacement%2520mutants%2520in%2520Ustilago%2520maydis%26jtitle%3DMol.%2520Genet.%2520Genomics%26date%3D2004%26volume%3D272%26spage%3D216%26epage%3D226%26doi%3D10.1007%2Fs00438-004-1047-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljmvrvkV7wGgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">–</span> <span class="NLM_lpage">1214</span><span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0ljmvrvkV7wGgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lj0vNYI4Qi9iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">The Phenix software for automated determination of macromolecular structures</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">94</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2011.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1016%2Fj.ymeth.2011.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=21821126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=94-106&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=S.+Jainauthor=G.+J.+Kapralauthor=R.+W.+Grosse+Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=The+Phenix+software+for+automated+determination+of+macromolecular+structures&doi=10.1016%2Fj.ymeth.2011.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Phenix software for automated determination of macromolecular structures</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li-Wei; Jain, Swati; Kapral, Gary J.; Grosse Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-106</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  X-ray crystallog. is a crit. tool in the study of biol. systems.  It is able to provide information that has been a prerequisite to understanding the fundamentals of life.  It is also a method that is central to the development of new therapeutics for human disease.  Significant time and effort are required to det. and optimize many macromol. structures because of the need for manual interpretation of complex numerical data, often using many different software packages, and the repeated use of interactive three-dimensional graphics.  The Phenix software package has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on automation.  This has required the development of new algorithms that minimize or eliminate subjective input in favor of built-in expert-systems knowledge, the automation of procedures that are traditionally performed by hand, and the development of a computational framework that allows a tight integration between the algorithms.  The application of automated methods is particularly appropriate in the field of structural proteomics, where high throughput is desired.  Features in Phenix for the automation of exptl. phasing with subsequent model building, mol. replacement, structure refinement and validation are described and examples given of running Phenix from both the command line and graphical user interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgjdX_7M-NcLVg90H21EOLACvtfcHk0lj0vNYI4Qi9iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F&md5=ba8f207cab9f3597dc0c6db885667a3f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2011.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2011.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse%2BKunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DThe%2520Phenix%2520software%2520for%2520automated%2520determination%2520of%2520macromolecular%2520structures%26jtitle%3DMethods%26date%3D2011%26volume%3D55%26spage%3D94%26epage%3D106%26doi%3D10.1016%2Fj.ymeth.2011.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lj0vNYI4Qi9iw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenova, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddali, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.</span><span> </span><span class="NLM_article-title">2,3-Dihydro-6,7-dihydroxy-1H-isoindol-1-one-based HIV-1 integrase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">251</span><span class="NLM_x">–</span> <span class="NLM_lpage">259</span><span class="refDoi"> DOI: 10.1021/jm070715d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070715d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=251-259&author=X.+Z.+Zhaoauthor=E.+A.+Semenovaauthor=B.+C.+Vuauthor=K.+Maddaliauthor=C.+Marchandauthor=S.+H.+Hughesauthor=Y.+Pommierauthor=T.+R.+Burke&title=2%2C3-Dihydro-6%2C7-dihydroxy-1H-isoindol-1-one-based+HIV-1+integrase+inhibitors&doi=10.1021%2Fjm070715d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm070715d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070715d%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSemenova%26aufirst%3DE.%2BA.%26aulast%3DVu%26aufirst%3DB.%2BC.%26aulast%3DMaddali%26aufirst%3DK.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3D2%252C3-Dihydro-6%252C7-dihydroxy-1H-isoindol-1-one-based%2520HIV-1%2520integrase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D251%26epage%3D259%26doi%3D10.1021%2Fjm070715d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 21 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Steven J. Smith, Xue Zhi Zhao, Dario Oliveira Passos, Valerie E. Pye, Peter Cherepanov, Dmitry Lyumkis, Terrence R. Burke, Jr., <span class="NLM_string-name hlFld-ContribAuthor">Stephen H. Hughes</span>. </span><span class="cited-content_cbyCitation_article-title">HIV-1 Integrase Inhibitors with Modifications That Affect Their Potencies against Drug Resistant Integrase Mutants. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2021,</strong> <em>7 </em>
                                    (6)
                                     , 1469-1482. <a href="https://doi.org/10.1021/acsinfecdis.0c00819" title="DOI URL">https://doi.org/10.1021/acsinfecdis.0c00819</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00819&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00819%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DHIV-1%252BIntegrase%252BInhibitors%252Bwith%252BModifications%252BThat%252BAffect%252BTheir%252BPotencies%252Bagainst%252BDrug%252BResistant%252BIntegrase%252BMutants%26aulast%3DSmith%26aufirst%3DSteven%2BJ.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D23112020%26date%3D09032021%26volume%3D7%26issue%3D6%26spage%3D1469%26epage%3D1482" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ashley N. Matthew, Florian Leidner, Gordon J. Lockbaum, Mina Henes, Jacqueto Zephyr, Shurong Hou, Desaboini Nageswara Rao, Jennifer Timm, Linah N. Rusere, Debra A. Ragland, Janet L. Paulsen, Kristina Prachanronarong, Djade I. Soumana, Ellen A. Nalivaika, Nese Kurt Yilmaz, Akbar Ali, <span class="NLM_string-name hlFld-ContribAuthor">Celia A. Schiffer</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Design Strategies to Avoid Resistance in Direct-Acting Antivirals and Beyond. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2021,</strong> <em>121 </em>
                                    (6)
                                     , 3238-3270. <a href="https://doi.org/10.1021/acs.chemrev.0c00648" title="DOI URL">https://doi.org/10.1021/acs.chemrev.0c00648</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.0c00648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.0c00648%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DDrug%252BDesign%252BStrategies%252Bto%252BAvoid%252BResistance%252Bin%252BDirect-Acting%252BAntivirals%252Band%252BBeyond%26aulast%3DMatthew%26aufirst%3DAshley%2BN.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D22062020%26date%3D07012021%26volume%3D121%26issue%3D6%26spage%3D3238%26epage%3D3270" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Lei Wang, Stefan G. Sarafianos, <span class="NLM_string-name hlFld-ContribAuthor">Zhengqiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Cutting into the Substrate Dominance: Pharmacophore and Structure-Based Approaches toward Inhibiting Human Immunodeficiency Virus Reverse Transcriptase-Associated Ribonuclease H. </span><span class="cited-content_cbyCitation_journal-name">Accounts of Chemical Research</span><span> <strong>2020,</strong> <em>53 </em>
                                    (1)
                                     , 218-230. <a href="https://doi.org/10.1021/acs.accounts.9b00450" title="DOI URL">https://doi.org/10.1021/acs.accounts.9b00450</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.accounts.9b00450&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.accounts.9b00450%26sid%3Dliteratum%253Aachs%26jtitle%3DAccounts%2520of%2520Chemical%2520Research%26atitle%3DCutting%252Binto%252Bthe%252BSubstrate%252BDominance%25253A%252BPharmacophore%252Band%252BStructure-Based%252BApproaches%252Btoward%252BInhibiting%252BHuman%252BImmunodeficiency%252BVirus%252BReverse%252BTranscriptase-Associated%252BRibonuclease%252BH%26aulast%3DWang%26aufirst%3DLei%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D23082019%26date%3D27122019%26volume%3D53%26issue%3D1%26spage%3D218%26epage%3D230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven J.  Smith</span>, <span class="hlFld-ContribAuthor ">Xue Zhi  Zhao</span>, <span class="hlFld-ContribAuthor ">Dario Oliveira  Passos</span>, <span class="hlFld-ContribAuthor ">Dmitry  Lyumkis</span>, <span class="hlFld-ContribAuthor ">Terrence R.  Burke</span>, <span class="hlFld-ContribAuthor ">Stephen H.  Hughes</span>. </span><span class="cited-content_cbyCitation_article-title">Integrase Strand Transfer Inhibitors Are Effective Anti-HIV Drugs. </span><span class="cited-content_cbyCitation_journal-name">Viruses</span><span> <strong>2021,</strong> <em>13 </em>
                                    (2)
                                     , 205. <a href="https://doi.org/10.3390/v13020205" title="DOI URL">https://doi.org/10.3390/v13020205</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/v13020205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fv13020205%26sid%3Dliteratum%253Aachs%26jtitle%3DViruses%26atitle%3DIntegrase%252BStrand%252BTransfer%252BInhibitors%252BAre%252BEffective%252BAnti-HIV%252BDrugs%26aulast%3DSmith%26aufirst%3DSteven%2BJ.%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D2%26spage%3D205" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan N.  Engelman</span>, <span class="hlFld-ContribAuthor ">Peter  Cherepanov</span>. </span><span class="cited-content_cbyCitation_article-title">Close‐up: HIV/SIV intasome structures shed new light on integrase inhibitor binding and viral escape mechanisms. </span><span class="cited-content_cbyCitation_journal-name">The FEBS Journal</span><span> <strong>2021,</strong> <em>288 </em>
                                    (2)
                                     , 427-433. <a href="https://doi.org/10.1111/febs.15438" title="DOI URL">https://doi.org/10.1111/febs.15438</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/febs.15438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Ffebs.15438%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520FEBS%2520Journal%26atitle%3DClose%2525E2%252580%252590up%25253A%252BHIV%25252FSIV%252Bintasome%252Bstructures%252Bshed%252Bnew%252Blight%252Bon%252Bintegrase%252Binhibitor%252Bbinding%252Band%252Bviral%252Bescape%252Bmechanisms%26aulast%3DEngelman%26aufirst%3DAlan%2BN.%26date%3D2021%26date%3D2020%26volume%3D288%26issue%3D2%26spage%3D427%26epage%3D433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Murugesan  Vanangamudi</span>, <span class="hlFld-ContribAuthor ">Pramod C.  Nair</span>, <span class="hlFld-ContribAuthor ">S. E. Maida  Engels</span>, <span class="hlFld-ContribAuthor ">Senthilkumar  Palaniappan</span>, <span class="hlFld-ContribAuthor ">Vigneshwaran  Namasivayam</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights to Human Immunodeficiency Virus (HIV-1) Targets and Their Inhibition. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 63-95. <a href="https://doi.org/10.1007/978-981-16-0267-2_3" title="DOI URL">https://doi.org/10.1007/978-981-16-0267-2_3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-16-0267-2_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-16-0267-2_3%26sid%3Dliteratum%253Aachs%26atitle%3DStructural%252BInsights%252Bto%252BHuman%252BImmunodeficiency%252BVirus%252B%252528HIV-1%252529%252BTargets%252Band%252BTheir%252BInhibition%26aulast%3DVanangamudi%26aufirst%3DMurugesan%26date%3D2021%26date%3D2021%26spage%3D63%26epage%3D95%26pub%3DSpringer%2520Singapore%26atitle%3DAntiviral%252BDrug%252BDiscovery%252Band%252BDevelopment%26aulast%3DLiu%26aufirst%3DXinyong%26date%3D2021%26volume%3D1322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ilona K.  Jóźwik</span>, <span class="hlFld-ContribAuthor ">Dario O.  Passos</span>, <span class="hlFld-ContribAuthor ">Dmitry  Lyumkis</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Biology of HIV Integrase Strand Transfer Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (9)
                                     , 611-626. <a href="https://doi.org/10.1016/j.tips.2020.06.003" title="DOI URL">https://doi.org/10.1016/j.tips.2020.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DStructural%252BBiology%252Bof%252BHIV%252BIntegrase%252BStrand%252BTransfer%252BInhibitors%26aulast%3DJ%25C3%25B3%25C5%25BAwik%26aufirst%3DIlona%2BK.%26date%3D2020%26volume%3D41%26issue%3D9%26spage%3D611%26epage%3D626" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven J.  Smith</span>, <span class="hlFld-ContribAuthor ">Xue Zhi  Zhao</span>, <span class="hlFld-ContribAuthor ">Dario Oliveira  Passos</span>, <span class="hlFld-ContribAuthor ">Dmitry  Lyumkis</span>, <span class="hlFld-ContribAuthor ">Terrence R.  Burke</span>, <span class="hlFld-ContribAuthor ">Stephen H.  Hughes</span>. </span><span class="cited-content_cbyCitation_article-title">HIV-1 Integrase Inhibitors That Are Active against Drug-Resistant Integrase Mutants. </span><span class="cited-content_cbyCitation_journal-name">Antimicrobial Agents and Chemotherapy</span><span> <strong>2020,</strong> <em>64 </em>
                                    (9)
                                     <a href="https://doi.org/10.1128/AAC.00611-20" title="DOI URL">https://doi.org/10.1128/AAC.00611-20</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AAC.00611-20&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAAC.00611-20%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%2520Agents%2520and%2520Chemotherapy%26atitle%3DHIV-1%252BIntegrase%252BInhibitors%252BThat%252BAre%252BActive%252Bagainst%252BDrug-Resistant%252BIntegrase%252BMutants%26aulast%3DSmith%26aufirst%3DSteven%2BJ.%26date%3D2020%26volume%3D64%26issue%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jintong  Du</span>, <span class="hlFld-ContribAuthor ">Jing  Guo</span>, <span class="hlFld-ContribAuthor ">Dongwei  Kang</span>, <span class="hlFld-ContribAuthor ">Zhihong  Li</span>, <span class="hlFld-ContribAuthor ">Guan  Wang</span>, <span class="hlFld-ContribAuthor ">Jianbing  Wu</span>, <span class="hlFld-ContribAuthor ">Zhen  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Fang</span>, <span class="hlFld-ContribAuthor ">Xuben  Hou</span>, <span class="hlFld-ContribAuthor ">Zhangjian  Huang</span>, <span class="hlFld-ContribAuthor ">Guobo  Li</span>, <span class="hlFld-ContribAuthor ">Xiaoyun  Lu</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>, <span class="hlFld-ContribAuthor ">Li  Rao</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>, <span class="hlFld-ContribAuthor ">Xiaojin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yihua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">New techniques and strategies in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Chinese Chemical Letters</span><span> <strong>2020,</strong> <em>31 </em>
                                    (7)
                                     , 1695-1708. <a href="https://doi.org/10.1016/j.cclet.2020.03.028" title="DOI URL">https://doi.org/10.1016/j.cclet.2020.03.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cclet.2020.03.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cclet.2020.03.028%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Chemical%2520Letters%26atitle%3DNew%252Btechniques%252Band%252Bstrategies%252Bin%252Bdrug%252Bdiscovery%26aulast%3DDu%26aufirst%3DJintong%26date%3D2020%26volume%3D31%26issue%3D7%26spage%3D1695%26epage%3D1708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrey A.  Ivashchenko</span>, <span class="hlFld-ContribAuthor ">Yan A.  Ivanenkov</span>, <span class="hlFld-ContribAuthor ">Angela G.  Koryakova</span>, <span class="hlFld-ContribAuthor ">Ruben N.  Karapetian</span>, <span class="hlFld-ContribAuthor ">Oleg D.  Mitkin</span>, <span class="hlFld-ContribAuthor ">Vladimir A.  Aladinskiy</span>, <span class="hlFld-ContribAuthor ">Dmitry V.  Kravchenko</span>, <span class="hlFld-ContribAuthor ">Nikolai P.  Savchuk</span>, <span class="hlFld-ContribAuthor ">Alexander V.  Ivashchenko</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, biological evaluation and in silico modeling of novel integrase strand transfer inhibitors (INSTIs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>189 </em>, 112064. <a href="https://doi.org/10.1016/j.ejmech.2020.112064" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112064%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%25252C%252Bbiological%252Bevaluation%252Band%252Bin%252Bsilico%252Bmodeling%252Bof%252Bnovel%252Bintegrase%252Bstrand%252Btransfer%252Binhibitors%252B%252528INSTIs%252529%26aulast%3DIvashchenko%26aufirst%3DAndrey%2BA.%26date%3D2020%26volume%3D189%26spage%3D112064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Min  Li</span>, <span class="hlFld-ContribAuthor ">Xuemin  Chen</span>, <span class="hlFld-ContribAuthor ">Huaibin  Wang</span>, <span class="hlFld-ContribAuthor ">Kellie A.  Jurado</span>, <span class="hlFld-ContribAuthor ">Alan N.  Engelman</span>, <span class="hlFld-ContribAuthor ">Robert  Craigie</span>. </span><span class="cited-content_cbyCitation_article-title">A Peptide Derived from Lens Epithelium–Derived Growth Factor Stimulates HIV-1 DNA Integration and Facilitates Intasome Structural Studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Biology</span><span> <strong>2020,</strong> <em>432 </em>
                                    (7)
                                     , 2055-2066. <a href="https://doi.org/10.1016/j.jmb.2020.01.040" title="DOI URL">https://doi.org/10.1016/j.jmb.2020.01.040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmb.2020.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmb.2020.01.040%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Biology%26atitle%3DA%252BPeptide%252BDerived%252Bfrom%252BLens%252BEpithelium%2525E2%252580%252593Derived%252BGrowth%252BFactor%252BStimulates%252BHIV-1%252BDNA%252BIntegration%252Band%252BFacilitates%252BIntasome%252BStructural%252BStudies%26aulast%3DLi%26aufirst%3DMin%26date%3D2020%26volume%3D432%26issue%3D7%26spage%3D2055%26epage%3D2066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dario Oliveira  Passos</span>, <span class="hlFld-ContribAuthor ">Min  Li</span>, <span class="hlFld-ContribAuthor ">Ilona K.  Jóźwik</span>, <span class="hlFld-ContribAuthor ">Xue Zhi  Zhao</span>, <span class="hlFld-ContribAuthor ">Diogo  Santos-Martins</span>, <span class="hlFld-ContribAuthor ">Renbin  Yang</span>, <span class="hlFld-ContribAuthor ">Steven J.  Smith</span>, <span class="hlFld-ContribAuthor ">Youngmin  Jeon</span>, <span class="hlFld-ContribAuthor ">Stefano  Forli</span>, <span class="hlFld-ContribAuthor ">Stephen H.  Hughes</span>, <span class="hlFld-ContribAuthor ">Terrence R.  Burke</span>, <span class="hlFld-ContribAuthor ">Robert  Craigie</span>, <span class="hlFld-ContribAuthor ">Dmitry  Lyumkis</span>. </span><span class="cited-content_cbyCitation_article-title">Structural basis for strand-transfer inhibitor binding to HIV intasomes. </span><span class="cited-content_cbyCitation_journal-name">Science</span><span> <strong>2020,</strong> <em>367 </em>
                                    (6479)
                                     , 810-814. <a href="https://doi.org/10.1126/science.aay8015" title="DOI URL">https://doi.org/10.1126/science.aay8015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/science.aay8015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscience.aay8015%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%26atitle%3DStructural%252Bbasis%252Bfor%252Bstrand-transfer%252Binhibitor%252Bbinding%252Bto%252BHIV%252Bintasomes%26aulast%3DPassos%26aufirst%3DDario%2BOliveira%26date%3D2020%26date%3D2020%26volume%3D367%26issue%3D6479%26spage%3D810%26epage%3D814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan N.  Engelman</span>. </span><span class="cited-content_cbyCitation_article-title">Multifaceted HIV integrase functionalities and therapeutic strategies for their inhibition. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2019,</strong> <em>294 </em>
                                    (41)
                                     , 15137-15157. <a href="https://doi.org/10.1074/jbc.REV119.006901" title="DOI URL">https://doi.org/10.1074/jbc.REV119.006901</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.REV119.006901&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.REV119.006901%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DMultifaceted%252BHIV%252Bintegrase%252Bfunctionalities%252Band%252Btherapeutic%252Bstrategies%252Bfor%252Btheir%252Binhibition%26aulast%3DEngelman%26aufirst%3DAlan%2BN.%26date%3D2019%26volume%3D294%26issue%3D41%26spage%3D15137%26epage%3D15157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xianheng  Wang</span>, <span class="hlFld-ContribAuthor ">Song  Chen</span>, <span class="hlFld-ContribAuthor ">Changkuo  Zhao</span>, <span class="hlFld-ContribAuthor ">Liangye  Long</span>, <span class="hlFld-ContribAuthor ">Yuhe  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Preparation of Dolutegravir Intermediate Diastereomer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2019,</strong> <em>56 </em>
                                    (7)
                                     , 2063-2067. <a href="https://doi.org/10.1002/jhet.3588" title="DOI URL">https://doi.org/10.1002/jhet.3588</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.3588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.3588%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DPreparation%252Bof%252BDolutegravir%252BIntermediate%252BDiastereomer%26aulast%3DWang%26aufirst%3DXianheng%26date%3D2019%26date%3D2019%26volume%3D56%26issue%3D7%26spage%3D2063%26epage%3D2067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Tang</span>, <span class="hlFld-ContribAuthor ">Ha T.  Do</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Huber</span>, <span class="hlFld-ContribAuthor ">Mary C.  Casey</span>, <span class="hlFld-ContribAuthor ">Karen A.  Kirby</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Wilson</span>, <span class="hlFld-ContribAuthor ">Jayakanth  Kankanala</span>, <span class="hlFld-ContribAuthor ">Michael A.  Parniak</span>, <span class="hlFld-ContribAuthor ">Stefan G.  Sarafianos</span>, <span class="hlFld-ContribAuthor ">Zhengqiang  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacophore-based design of novel 3-hydroxypyrimidine-2,4-dione subtypes as inhibitors of HIV reverse transcriptase-associated RNase H: Tolerance of a nonflexible linker. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>166 </em>, 390-399. <a href="https://doi.org/10.1016/j.ejmech.2019.01.081" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.081%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPharmacophore-based%252Bdesign%252Bof%252Bnovel%252B3-hydroxypyrimidine-2%25252C4-dione%252Bsubtypes%252Bas%252Binhibitors%252Bof%252BHIV%252Breverse%252Btranscriptase-associated%252BRNase%252BH%25253A%252BTolerance%252Bof%252Ba%252Bnonflexible%252Blinker%26aulast%3DTang%26aufirst%3DJing%26date%3D2019%26volume%3D166%26spage%3D390%26epage%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Léa  El Khoury</span>, <span class="hlFld-ContribAuthor ">Krystel  El Hage</span>, <span class="hlFld-ContribAuthor ">Jean-Philip  Piquemal</span>, <span class="hlFld-ContribAuthor ">Serge  Fermandjian</span>, <span class="hlFld-ContribAuthor ">Richard G.  Maroun</span>, <span class="hlFld-ContribAuthor ">Nohad  Gresh</span>, <span class="hlFld-ContribAuthor ">Zeina  Hobaika</span>. </span><span class="cited-content_cbyCitation_article-title">Spectrometric and computational studies of the binding of HIV-1 integrase inhibitors to viral DNA extremities. </span><span class="cited-content_cbyCitation_journal-name">PeerJ Physical Chemistry</span><span> <strong>2019,</strong> <em>1 </em>, e6. <a href="https://doi.org/10.7717/peerj-pchem.6" title="DOI URL">https://doi.org/10.7717/peerj-pchem.6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.7717/peerj-pchem.6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.7717%2Fpeerj-pchem.6%26sid%3Dliteratum%253Aachs%26jtitle%3DPeerJ%2520Physical%2520Chemistry%26atitle%3DSpectrometric%252Band%252Bcomputational%252Bstudies%252Bof%252Bthe%252Bbinding%252Bof%252BHIV-1%252Bintegrase%252Binhibitors%252Bto%252Bviral%252BDNA%252Bextremities%26aulast%3DEl%2BKhoury%26aufirst%3DL%25C3%25A9a%26date%3D2019%26date%3D2019%26volume%3D1%26spage%3De6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven J.  Smith</span>, <span class="hlFld-ContribAuthor ">Xue Zhi  Zhao</span>, <span class="hlFld-ContribAuthor ">Terrence R.  Burke</span>, <span class="hlFld-ContribAuthor ">Stephen H.  Hughes</span>. </span><span class="cited-content_cbyCitation_article-title">Efficacies of Cabotegravir and Bictegravir against drug-resistant HIV-1 integrase mutants. </span><span class="cited-content_cbyCitation_journal-name">Retrovirology</span><span> <strong>2018,</strong> <em>15 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12977-018-0420-7" title="DOI URL">https://doi.org/10.1186/s12977-018-0420-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12977-018-0420-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12977-018-0420-7%26sid%3Dliteratum%253Aachs%26jtitle%3DRetrovirology%26atitle%3DEfficacies%252Bof%252BCabotegravir%252Band%252BBictegravir%252Bagainst%252Bdrug-resistant%252BHIV-1%252Bintegrase%252Bmutants%26aulast%3DSmith%26aufirst%3DSteven%2BJ.%26date%3D2018%26date%3D2018%26volume%3D15%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jomy M  George</span>, <span class="hlFld-ContribAuthor ">Safia S  Kuriakose</span>, <span class="hlFld-ContribAuthor ">Nicola  Dee</span>, <span class="hlFld-ContribAuthor ">Pam  Stoll</span>, <span class="hlFld-ContribAuthor ">Tahaniyat  Lalani</span>, <span class="hlFld-ContribAuthor ">Robin  Dewar</span>, <span class="hlFld-ContribAuthor ">Muhammad A  Khan</span>, <span class="hlFld-ContribAuthor ">Muhammad T  Rehman</span>, <span class="hlFld-ContribAuthor ">Zehava  Grossman</span>, <span class="hlFld-ContribAuthor ">Frank  Maldarelli</span>, <span class="hlFld-ContribAuthor ">Alice K  Pau</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid Development of High-Level Resistance to Dolutegravir With Emergence of T97A Mutation in 2 Treatment-Experienced Individuals With Baseline Partial Sensitivity to Dolutegravir. </span><span class="cited-content_cbyCitation_journal-name">Open Forum Infectious Diseases</span><span> <strong>2018,</strong> <em>5 </em>
                                    (10)
                                     <a href="https://doi.org/10.1093/ofid/ofy221" title="DOI URL">https://doi.org/10.1093/ofid/ofy221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/ofid/ofy221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fofid%2Fofy221%26sid%3Dliteratum%253Aachs%26jtitle%3DOpen%2520Forum%2520Infectious%2520Diseases%26atitle%3DRapid%252BDevelopment%252Bof%252BHigh-Level%252BResistance%252Bto%252BDolutegravir%252BWith%252BEmergence%252Bof%252BT97A%252BMutation%252Bin%252B2%252BTreatment-Experienced%252BIndividuals%252BWith%252BBaseline%252BPartial%252BSensitivity%252Bto%252BDolutegravir%26aulast%3DGeorge%26aufirst%3DJomy%2BM%26date%3D2018%26date%3D2018%26volume%3D5%26issue%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Steven J.  Smith</span>, <span class="hlFld-ContribAuthor ">Xue Zhi  Zhao</span>, <span class="hlFld-ContribAuthor ">Terrence R.  Burke</span>, <span class="hlFld-ContribAuthor ">Stephen H.  Hughes</span>. </span><span class="cited-content_cbyCitation_article-title">HIV-1 Integrase Inhibitors That Are Broadly Effective against Drug-Resistant Mutants. </span><span class="cited-content_cbyCitation_journal-name">Antimicrobial Agents and Chemotherapy</span><span> <strong>2018,</strong> <em>62 </em>
                                    (9)
                                     <a href="https://doi.org/10.1128/AAC.01035-18" title="DOI URL">https://doi.org/10.1128/AAC.01035-18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1128/AAC.01035-18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1128%2FAAC.01035-18%26sid%3Dliteratum%253Aachs%26jtitle%3DAntimicrobial%2520Agents%2520and%2520Chemotherapy%26atitle%3DHIV-1%252BIntegrase%252BInhibitors%252BThat%252BAre%252BBroadly%252BEffective%252Bagainst%252BDrug-Resistant%252BMutants%26aulast%3DSmith%26aufirst%3DSteven%2BJ.%26date%3D2018%26volume%3D62%26issue%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher P.  Gordon</span>, <span class="hlFld-ContribAuthor ">Neal  Dalton</span>, <span class="hlFld-ContribAuthor ">Nicholas  Vandegraaff</span>, <span class="hlFld-ContribAuthor ">John  Deadman</span>, <span class="hlFld-ContribAuthor ">David I.  Rhodes</span>, <span class="hlFld-ContribAuthor ">Jonathan A.  Coates</span>, <span class="hlFld-ContribAuthor ">Stephen G.  Pyne</span>, <span class="hlFld-ContribAuthor ">Renate  Griffith</span>, <span class="hlFld-ContribAuthor ">John B.  Bremner</span>, <span class="hlFld-ContribAuthor ">Paul A.  Keller</span>. </span><span class="cited-content_cbyCitation_article-title">A structure-based design approach to advance the allyltyrosine-based series of HIV integrase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron</span><span> <strong>2018,</strong> <em>74 </em>
                                    (12)
                                     , 1253-1268. <a href="https://doi.org/10.1016/j.tet.2017.11.052" title="DOI URL">https://doi.org/10.1016/j.tet.2017.11.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tet.2017.11.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tet.2017.11.052%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%26atitle%3DA%252Bstructure-based%252Bdesign%252Bapproach%252Bto%252Badvance%252Bthe%252Ballyltyrosine-based%252Bseries%252Bof%252BHIV%252Bintegrase%252Binhibitors%26aulast%3DGordon%26aufirst%3DChristopher%2BP.%26date%3D2018%26volume%3D74%26issue%3D12%26spage%3D1253%26epage%3D1268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fengyuan  Yang</span>, <span class="hlFld-ContribAuthor ">Guoxun  Zheng</span>, <span class="hlFld-ContribAuthor ">Tingting  Fu</span>, <span class="hlFld-ContribAuthor ">Xiaofeng  Li</span>, <span class="hlFld-ContribAuthor ">Gao  Tu</span>, <span class="hlFld-ContribAuthor ">Ying Hong  Li</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>, <span class="hlFld-ContribAuthor ">Weiwei  Xue</span>, <span class="hlFld-ContribAuthor ">Feng  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of the binding mode and resistance profile for a dual-target pyrrolyl diketo acid scaffold against HIV-1 integrase and reverse-transcriptase-associated ribonuclease H. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (37)
                                     , 23873-23884. <a href="https://doi.org/10.1039/C8CP01843J" title="DOI URL">https://doi.org/10.1039/C8CP01843J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP01843J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP01843J%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DPrediction%252Bof%252Bthe%252Bbinding%252Bmode%252Band%252Bresistance%252Bprofile%252Bfor%252Ba%252Bdual-target%252Bpyrrolyl%252Bdiketo%252Bacid%252Bscaffold%252Bagainst%252BHIV-1%252Bintegrase%252Band%252Breverse-transcriptase-associated%252Bribonuclease%252BH%26aulast%3DYang%26aufirst%3DFengyuan%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D37%26spage%3D23873%26epage%3D23884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0006.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. HIV-1 integrase inhibitors. Colored areas indicate regions of intended correspondence.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0004.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 4,7-Bis-Substituted Analogues <b>5a–5i</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) KO<sup><i>t</i></sup>Bu, MeOH, 65 °C; (ii) H<sub>2</sub>SO<sub>4</sub>, MeOH; (iii) BnONH<sub>2</sub>, DIEA, dioxane, 110 °C; (iv) (b) piperidine or (c) morpholine, DMF, 80 °C; (v) ClCOCH<sub>2</sub>CO<sub>2</sub>CH<sub>3</sub>, TEA, DCM; (vi) NaOMe, MeOH; (vii) 2,4-diF-BnNH<sub>2</sub>, DMF, 140 °C; (viii) TsCl, DIEA, MeCN; (ix) RNH<sub>2</sub>, DIEA, DMF, 50 °C, R = 2,4-diMeOBn (a) or CH<sub>2</sub>CO<sub>2</sub>Me (b); (x) TFA, DCM; (xi) H<sub>2</sub>, 10% Pd/C, MeOH.</p></p></figure><figure data-id="sch2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0005.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of 4,6-Bis-Substituted Analogues <b>6a</b>–<b>6t</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (i) TsCl, Et<sub>3</sub>N, CH<sub>3</sub>CN; (ii) RNH<sub>2</sub>, DIEA, DMF, 50 °C, R = 2,4-diMeOBn (DMP) (a), CH<sub>2</sub>CH<sub>2</sub>OH (b), or CH<sub>2</sub>CO<sub>2</sub>Me (c); (iii) alkyne, Pd(PPh<sub>3</sub>)<sub>2</sub>Cl<sub>2</sub>, DIEA, CuI, DMF, 70 °C or alkene, Pd(OAc)<sub>2</sub>, TEA, PPh<sub>3</sub>, DMF, 100 °C; (iv) TFA, DCM; (v) H<sub>2</sub>, 10% Pd/C, MeOH.</p></p></figure><figure data-id="fig2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0001.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structures of PFV intasome-bound inhibitors. (A) Bound <b>5′g</b> (cream) with overlays showing the relative positions of pre-3′-P viral DNA (violet; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I">4E7I</a>) and DTG (<b>3</b>) (green; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S3M">3S3M</a>). The 4-<i>N</i>-(methyl 2-glycinate) moiety of <b>5′g</b> is circled in red, highlighting its correspondence with the T<sub>+2</sub> adenosine (A18) base of pre-3′-P viral DNA. (B) Bound <b>5′g</b> with its 4-<i>N</i>-(methyl 2-glycinate) moiety shown as transparent spheres. Atoms within 5 Å (Y212 and P214 and A17 of 3′-P DNA) are shown in green as a stick diagram. For both A and B, metal ions are shown as solid blue spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0002.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Crystal structures of <b>6p</b> bound to the PFV intasome. Structures of the pre-3′-P viral DNA (violet; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I">4E7I</a>) and target complex host DNA substrate (gray; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7K">4E7K</a>) are overlaid. The 6-(methyl 3-propanoate) group of <b>6p</b> is circled in red, showing its correspondence with aspects of both pre-3′-P viral DNA and the host target DNA. Cofactor Mg<sup>2+</sup> ions are shown as blue spheres; the structures of bound viral DNAs have been omitted for clarity.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/medium/jm-2017-00596a_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0003.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Crystal structures of PFV intasome-bound inhibitors. (A) Bound <b>5g</b> (orange) with superimposed <b>6p</b> (cyan) and DTG (<b>3</b>) (green; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S3M">3S3M</a>) showing closest distance to the Gly187 α-methylene (side chains of Asp186 and Gln187 shown in semitransparent), bound 3′-P DNAs have been omitted for clarity. (B) Superimposed bound 3′-P DNAs associated with A, circled in cyan are the oxygens in the C16–A17 phosphoryl linkages, whose engagement in electrostatic interactions with the side chain imidazole nitrogen of the N224H mutant (corresponding to IN mutant N155H) would be broken on binding of the INSTIs. (C) Bound <b>5g</b> with superimposed pre-3′-P viral DNA (violet; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I">4E7I</a>) and target complex host DNA substrate (gray; PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7K">4E7K</a>), circled in red is the 7-OCH<sub>3</sub>, which protrudes outside the substrate envelope. Metal ions are shown as solid blue spheres.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-17/acs.jmedchem.7b00596/20170908/images/large/jm-2017-00596a_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00596&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i108">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50361" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50361" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 51 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Lesbats, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Retroviral DNA integration</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">12730</span><span class="NLM_x">–</span> <span class="NLM_lpage">12757</span><span class="refDoi"> DOI: 10.1021/acs.chemrev.6b00125</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.6b00125" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC28Xot1Sksr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=12730-12757&author=P.+Lesbatsauthor=A.+N.+Engelmanauthor=P.+Cherepanov&title=Retroviral+DNA+integration&doi=10.1021%2Facs.chemrev.6b00125"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Retroviral DNA Integration</span></div><div class="casAuthors">Lesbats, Paul; Engelman, Alan N.; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">12730-12757</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The integration of a DNA copy of the viral RNA genome into host chromatin is the defining step of retroviral replication.  This enzymic process is catalyzed by the virus-encoded integrase protein, which is conserved among retroviruses and LTR-retrotransposons.  Retroviral integration proceeds via two integrase activities: 3'-processing of the viral DNA ends, followed by the strand transfer of the processed ends into host cell chromosomal DNA.  Herein we review the mol. mechanism of retroviral DNA integration, with an emphasis on reaction chemistries and architectures of the nucleoprotein complexes involved.  We addnl. discuss the latest advances on anti-integrase drug development for the treatment of AIDS and the utility of integrating retroviral vectors in gene therapy applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG1CjjtylBirVg90H21EOLACvtfcHk0lh4oxnJJSFOHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xot1Sksr8%253D&md5=f3a6b640085583b294cdd82e0a384234</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.6b00125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.6b00125%26sid%3Dliteratum%253Aachs%26aulast%3DLesbats%26aufirst%3DP.%26aulast%3DEngelman%26aufirst%3DA.%2BN.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DRetroviral%2520DNA%2520integration%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D12730%26epage%3D12757%26doi%3D10.1021%2Facs.chemrev.6b00125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Summa, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrocchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crescenzi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fiore, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez Paz, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzel, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laufer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monteagudo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muraglia, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nizi, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orvieto, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pace, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pescatore, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scarpelli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stillmock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowley, M.</span><span> </span><span class="NLM_article-title">Discovery of raltegravir, a potent, selective orally bioavailable HIV-integrase inhibitor for the treatment of HIV-AIDS infection</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">5843</span><span class="NLM_x">–</span> <span class="NLM_lpage">5855</span><span class="refDoi"> DOI: 10.1021/jm800245z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm800245z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=5843-5855&author=V.+Summaauthor=A.+Petrocchiauthor=F.+Bonelliauthor=B.+Crescenziauthor=M.+Donghiauthor=M.+Ferraraauthor=F.+Fioreauthor=C.+Gardelliauthor=O.+Gonzalez+Pazauthor=D.+J.+Hazudaauthor=P.+Jonesauthor=O.+Kinzelauthor=R.+Lauferauthor=E.+Monteagudoauthor=E.+Muragliaauthor=E.+Niziauthor=F.+Orvietoauthor=P.+Paceauthor=G.+Pescatoreauthor=R.+Scarpelliauthor=K.+Stillmockauthor=M.+V.+Witmerauthor=M.+Rowley&title=Discovery+of+raltegravir%2C+a+potent%2C+selective+orally+bioavailable+HIV-integrase+inhibitor+for+the+treatment+of+HIV-AIDS+infection&doi=10.1021%2Fjm800245z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Raltegravir, a Potent, Selective Orally Bioavailable HIV-Integrase Inhibitor for the Treatment of HIV-AIDS Infection</span></div><div class="casAuthors">Summa, Vincenzo; Petrocchi, Alessia; Bonelli, Fabio; Crescenzi, Benedetta; Donghi, Monica; Ferrara, Marco; Fiore, Fabrizio; Gardelli, Cristina; Gonzalez Paz, Odalys; Hazuda, Daria J.; Jones, Philip; Kinzel, Olaf; Laufer, Ralph; Monteagudo, Edith; Muraglia, Ester; Nizi, Emanuela; Orvieto, Federica; Pace, Paola; Pescatore, Giovanna; Scarpelli, Rita; Stillmock, Kara; Witmer, Marc V.; Rowley, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5843-5855</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Human immunodeficiency virus type-1 (HIV-1) integrase is one of the three virally encoded enzymes required for replication and therefore a rational target for chemotherapeutic intervention in the treatment of HIV-1 infection.  We report here the discovery of Raltegravir, the first HIV-integrase inhibitor approved by FDA for the treatment of HIV infection.  It derives from the evolution of 5,6-dihydroxypyrimidine-4-carboxamides and N-methyl-4-hydroxypyrimidinone-carboxamides, which exhibited potent inhibition of the HIV-integrase catalyzed strand transfer process.  Structural modifications on these mols. were made in order to maximize potency as HIV-integrase inhibitors against the wild type virus, a selection of mutants, and optimize the selectivity, pharmacokinetic, and metabolic profiles in preclin. species.  The good profile of Raltegravir has enabled its progression toward the end of phase III clin. trials for the treatment of HIV-1 infection and culminated with the FDA approval as the first HIV-integrase inhibitor for the treatment of HIV-1 infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYo6KKNjJRCrVg90H21EOLACvtfcHk0lh55E8YrUW3DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVymtb%252FM&md5=f90f994c7cd031a4063cbb3e91b4c3f0</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1021%2Fjm800245z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800245z%26sid%3Dliteratum%253Aachs%26aulast%3DSumma%26aufirst%3DV.%26aulast%3DPetrocchi%26aufirst%3DA.%26aulast%3DBonelli%26aufirst%3DF.%26aulast%3DCrescenzi%26aufirst%3DB.%26aulast%3DDonghi%26aufirst%3DM.%26aulast%3DFerrara%26aufirst%3DM.%26aulast%3DFiore%26aufirst%3DF.%26aulast%3DGardelli%26aufirst%3DC.%26aulast%3DGonzalez%2BPaz%26aufirst%3DO.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DJones%26aufirst%3DP.%26aulast%3DKinzel%26aufirst%3DO.%26aulast%3DLaufer%26aufirst%3DR.%26aulast%3DMonteagudo%26aufirst%3DE.%26aulast%3DMuraglia%26aufirst%3DE.%26aulast%3DNizi%26aufirst%3DE.%26aulast%3DOrvieto%26aufirst%3DF.%26aulast%3DPace%26aufirst%3DP.%26aulast%3DPescatore%26aufirst%3DG.%26aulast%3DScarpelli%26aufirst%3DR.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DWitmer%26aufirst%3DM.%2BV.%26aulast%3DRowley%26aufirst%3DM.%26atitle%3DDiscovery%2520of%2520raltegravir%252C%2520a%2520potent%252C%2520selective%2520orally%2520bioavailable%2520HIV-integrase%2520inhibitor%2520for%2520the%2520treatment%2520of%2520HIV-AIDS%2520infection%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D5843%26epage%3D5855%26doi%3D10.1021%2Fjm800245z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Sato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motomura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aramaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamashita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ito, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawakami, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuzaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watanabe, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamataka, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ikeda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodama, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuoka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shinkai, H.</span><span> </span><span class="NLM_article-title">Novel HIV-1 integrase inhibitors derived from quinolone antibiotics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">1506</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span><span class="refDoi"> DOI: 10.1021/jm0600139</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0600139" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=1506-1508&author=M.+Satoauthor=T.+Motomuraauthor=H.+Aramakiauthor=T.+Matsudaauthor=M.+Yamashitaauthor=Y.+Itoauthor=H.+Kawakamiauthor=Y.+Matsuzakiauthor=W.+Watanabeauthor=K.+Yamatakaauthor=S.+Ikedaauthor=E.+Kodamaauthor=M.+Matsuokaauthor=H.+Shinkai&title=Novel+HIV-1+integrase+inhibitors+derived+from+quinolone+antibiotics&doi=10.1021%2Fjm0600139"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm0600139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0600139%26sid%3Dliteratum%253Aachs%26aulast%3DSato%26aufirst%3DM.%26aulast%3DMotomura%26aufirst%3DT.%26aulast%3DAramaki%26aufirst%3DH.%26aulast%3DMatsuda%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DM.%26aulast%3DIto%26aufirst%3DY.%26aulast%3DKawakami%26aufirst%3DH.%26aulast%3DMatsuzaki%26aufirst%3DY.%26aulast%3DWatanabe%26aufirst%3DW.%26aulast%3DYamataka%26aufirst%3DK.%26aulast%3DIkeda%26aufirst%3DS.%26aulast%3DKodama%26aufirst%3DE.%26aulast%3DMatsuoka%26aufirst%3DM.%26aulast%3DShinkai%26aufirst%3DH.%26atitle%3DNovel%2520HIV-1%2520integrase%2520inhibitors%2520derived%2520from%2520quinolone%2520antibiotics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D1506%26epage%3D1508%26doi%3D10.1021%2Fjm0600139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Johns, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawasuji, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weatherhead, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Temelkoff, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taoda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiyama, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuji, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffrey, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foster, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshinaga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garvey, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, T.</span><span> </span><span class="NLM_article-title">Carbamoyl pyridone HIV-1 integrase Inhibitors 3. A diastereomeric approach to chiral nonracemic tricyclic ring systems and the discovery of dolutegravir (S/GSKi349572) and (S/GSK1265744)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">5901</span><span class="NLM_x">–</span> <span class="NLM_lpage">5916</span><span class="refDoi"> DOI: 10.1021/jm400645w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400645w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=5901-5916&author=B.+A.+Johnsauthor=T.+Kawasujiauthor=J.+G.+Weatherheadauthor=T.+Taishiauthor=D.+P.+Temelkoffauthor=H.+Yoshidaauthor=T.+Akiyamaauthor=Y.+Taodaauthor=H.+Muraiauthor=R.+Kiyamaauthor=M.+Fujiauthor=N.+Tanimotoauthor=J.+Jeffreyauthor=S.+A.+Fosterauthor=T.+Yoshinagaauthor=T.+Sekiauthor=M.+Kobayashiauthor=A.+Satoauthor=M.+N.+Johnsonauthor=E.+P.+Garveyauthor=T.+Fujiwara&title=Carbamoyl+pyridone+HIV-1+integrase+Inhibitors+3.+A+diastereomeric+approach+to+chiral+nonracemic+tricyclic+ring+systems+and+the+discovery+of+dolutegravir+%28S%2FGSKi349572%29+and+%28S%2FGSK1265744%29&doi=10.1021%2Fjm400645w"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Fjm400645w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400645w%26sid%3Dliteratum%253Aachs%26aulast%3DJohns%26aufirst%3DB.%2BA.%26aulast%3DKawasuji%26aufirst%3DT.%26aulast%3DWeatherhead%26aufirst%3DJ.%2BG.%26aulast%3DTaishi%26aufirst%3DT.%26aulast%3DTemelkoff%26aufirst%3DD.%2BP.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DAkiyama%26aufirst%3DT.%26aulast%3DTaoda%26aufirst%3DY.%26aulast%3DMurai%26aufirst%3DH.%26aulast%3DKiyama%26aufirst%3DR.%26aulast%3DFuji%26aufirst%3DM.%26aulast%3DTanimoto%26aufirst%3DN.%26aulast%3DJeffrey%26aufirst%3DJ.%26aulast%3DFoster%26aufirst%3DS.%2BA.%26aulast%3DYoshinaga%26aufirst%3DT.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DM.%2BN.%26aulast%3DGarvey%26aufirst%3DE.%2BP.%26aulast%3DFujiwara%26aufirst%3DT.%26atitle%3DCarbamoyl%2520pyridone%2520HIV-1%2520integrase%2520Inhibitors%25203.%2520A%2520diastereomeric%2520approach%2520to%2520chiral%2520nonracemic%2520tricyclic%2520ring%2520systems%2520and%2520the%2520discovery%2520of%2520dolutegravir%2520%2528S%252FGSKi349572%2529%2520and%2520%2528S%252FGSK1265744%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D5901%26epage%3D5916%26doi%3D10.1021%2Fjm400645w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felock, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stillmock, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grobler, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espeseth, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabryelski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schleif, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blau, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span> </span><span class="NLM_article-title">Inhibitors of strand transfer that prevent integration and inhibit HIV-1 replication in cells</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">646</span><span class="NLM_x">–</span> <span class="NLM_lpage">650</span><span class="refDoi"> DOI: 10.1126/science.287.5453.646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1126%2Fscience.287.5453.646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10649997" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD3cXovVyjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2000&pages=646-650&author=D.+J.+Hazudaauthor=P.+Felockauthor=M.+Witmerauthor=A.+Wolfeauthor=K.+Stillmockauthor=J.+A.+Groblerauthor=A.+Espesethauthor=L.+Gabryelskiauthor=W.+Schleifauthor=C.+Blauauthor=M.+D.+Miller&title=Inhibitors+of+strand+transfer+that+prevent+integration+and+inhibit+HIV-1+replication+in+cells&doi=10.1126%2Fscience.287.5453.646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of strand transfer that prevent integration and inhibit HIV-1 replication in cells</span></div><div class="casAuthors">Hazuda, Daria J.; Felock, Peter; Witmer, Marc; Wolfe, Abigail; Stillmock, Kara; Grobler, Jay A.; Espeseth, Amy; Gabryelski, Lori; Schleif, William; Blau, Carol; Miller, Michael D.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">5453</span>),
    <span class="NLM_cas:pages">646-650</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Integrase is essential for human immunodeficiency virus-type 1 (HIV-1) replication; however, potent inhibition of the isolated enzyme in biochem. assays has not readily translated into antiviral activity in a manner consistent with inhibition of integration.  In this report, we describe diketo acid inhibitors of HIV-1 integrase that manifest antiviral activity as a consequence of their effect on integration.  The antiviral activity of these compds. is due exclusively to inhibition of one of the two catalytic functions of integrase, strand transfer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLfQNYMkTCVrVg90H21EOLACvtfcHk0lgm0S1jEmDX_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXovVyjtQ%253D%253D&md5=d160c8fa31a9ca0a3530dceab66a892c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.287.5453.646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.287.5453.646%26sid%3Dliteratum%253Aachs%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DFelock%26aufirst%3DP.%26aulast%3DWitmer%26aufirst%3DM.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DStillmock%26aufirst%3DK.%26aulast%3DGrobler%26aufirst%3DJ.%2BA.%26aulast%3DEspeseth%26aufirst%3DA.%26aulast%3DGabryelski%26aufirst%3DL.%26aulast%3DSchleif%26aufirst%3DW.%26aulast%3DBlau%26aufirst%3DC.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26atitle%3DInhibitors%2520of%2520strand%2520transfer%2520that%2520prevent%2520integration%2520and%2520inhibit%2520HIV-1%2520replication%2520in%2520cells%26jtitle%3DScience%26date%3D2000%26volume%3D287%26spage%3D646%26epage%3D650%26doi%3D10.1126%2Fscience.287.5453.646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Métifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiselev, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marler, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Selectivity for strand-transfer over 3′-processing and susceptibility to clinical resistance of HIV-1 integrase inhibitors are driven by key enzyme–DNA interactions in the active site</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">6896</span><span class="NLM_x">–</span> <span class="NLM_lpage">6906</span><span class="refDoi"> DOI: 10.1093/nar/gkw592</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1093%2Fnar%2Fgkw592" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=6896-6906&author=M.+M%C3%A9tifiotauthor=B.+C.+Johnsonauthor=E.+Kiselevauthor=L.+Marlerauthor=X.+Z.+Zhaoauthor=T.+R.+Burkeauthor=C.+Marchandauthor=S.+H.+Hughesauthor=Y.+Pommier&title=Selectivity+for+strand-transfer+over+3%E2%80%B2-processing+and+susceptibility+to+clinical+resistance+of+HIV-1+integrase+inhibitors+are+driven+by+key+enzyme%E2%80%93DNA+interactions+in+the+active+site&doi=10.1093%2Fnar%2Fgkw592"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw592%26sid%3Dliteratum%253Aachs%26aulast%3DM%25C3%25A9tifiot%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BC.%26aulast%3DKiselev%26aufirst%3DE.%26aulast%3DMarler%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DSelectivity%2520for%2520strand-transfer%2520over%25203%25E2%2580%25B2-processing%2520and%2520susceptibility%2520to%2520clinical%2520resistance%2520of%2520HIV-1%2520integrase%2520inhibitors%2520are%2520driven%2520by%2520key%2520enzyme%25E2%2580%2593DNA%2520interactions%2520in%2520the%2520active%2520site%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3D6896%26epage%3D6906%26doi%3D10.1093%2Fnar%2Fgkw592" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Bar-Magen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sloan, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faltenbacher, V. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donahue, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhl, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oliveira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span> </span><span class="NLM_article-title">Comparative biochemical analysis of HIV-1 subtype B and C integrase enzymes</span> <span class="citation_source-journal">Retrovirology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">103</span><span class="refDoi"> DOI: 10.1186/1742-4690-6-103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1186%2F1742-4690-6-103" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2009&pages=103&author=T.+Bar-Magenauthor=R.+D.+Sloanauthor=V.+H.+Faltenbacherauthor=D.+A.+Donahueauthor=B.+D.+Kuhlauthor=M.+Oliveiraauthor=H.+Xuauthor=M.+A.+Wainberg&title=Comparative+biochemical+analysis+of+HIV-1+subtype+B+and+C+integrase+enzymes&doi=10.1186%2F1742-4690-6-103"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2F1742-4690-6-103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1742-4690-6-103%26sid%3Dliteratum%253Aachs%26aulast%3DBar-Magen%26aufirst%3DT.%26aulast%3DSloan%26aufirst%3DR.%2BD.%26aulast%3DFaltenbacher%26aufirst%3DV.%2BH.%26aulast%3DDonahue%26aufirst%3DD.%2BA.%26aulast%3DKuhl%26aufirst%3DB.%2BD.%26aulast%3DOliveira%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26atitle%3DComparative%2520biochemical%2520analysis%2520of%2520HIV-1%2520subtype%2520B%2520and%2520C%2520integrase%2520enzymes%26jtitle%3DRetrovirology%26date%3D2009%26volume%3D6%26spage%3D103%26doi%3D10.1186%2F1742-4690-6-103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maertens, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">3′-Processing and strand transfer catalysed by retroviral integrase in crystallo</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">3020</span><span class="NLM_x">–</span> <span class="NLM_lpage">3028</span><span class="refDoi"> DOI: 10.1038/emboj.2012.118</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Femboj.2012.118" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=22580823" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvVertbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=3020-3028&author=S.+Hareauthor=G.+N.+Maertensauthor=P.+Cherepanov&title=3%E2%80%B2-Processing+and+strand+transfer+catalysed+by+retroviral+integrase+in+crystallo&doi=10.1038%2Femboj.2012.118"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">3'-Processing and strand transfer catalysed by retroviral integrase in crystallo</span></div><div class="casAuthors">Hare, Stephen; Maertens, Goedele N.; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3020-3028</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Retroviral integrase (IN) is responsible for two consecutive reactions, which lead to insertion of a viral DNA copy into a host cell chromosome.  Initially, the enzyme removes di- or trinucleotides from viral DNA ends to expose 3'-hydroxyls attached to the invariant CA dinucleotides (3'-processing reaction).  Second, it inserts the processed 3'-viral DNA ends into host chromosomal DNA (strand transfer).  Herein, we report a crystal structure of prototype foamy virus IN bound to viral DNA prior to 3'-processing.  Furthermore, taking advantage of its dependence on divalent metal ion cofactors, we were able to freeze trap the viral enzyme in its ground states contg. all the components necessary for 3'-processing or strand transfer.  Our results shed light on the mechanics of retroviral DNA integration and explain why HIV IN strand transfer inhibitors are ineffective against the 3'-processing step of integration.  The ground state structures moreover highlight a striking substrate mimicry utilized by the inhibitors in their binding to the IN active site and suggest ways to improve upon this clin. relevant class of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmcESnfHFZn7Vg90H21EOLACvtfcHk0lgIfyJoN1JcOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvVertbk%253D&md5=02545febfca7173b767d203e90210a74</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.118&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.118%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DMaertens%26aufirst%3DG.%2BN.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3D3%25E2%2580%25B2-Processing%2520and%2520strand%2520transfer%2520catalysed%2520by%2520retroviral%2520integrase%2520in%2520crystallo%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D3020%26epage%3D3028%26doi%3D10.1038%2Femboj.2012.118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Abram, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferris, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alvord, W. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span> </span><span class="NLM_article-title">Nature, position, and frequency of mutations made in a single cycle of HIV-1 replication</span> <span class="citation_source-journal">J. Virol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">84</span><span class="NLM_x">, </span> <span class="NLM_fpage">9864</span><span class="NLM_x">–</span> <span class="NLM_lpage">9878</span><span class="refDoi"> DOI: 10.1128/JVI.00915-10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1128%2FJVI.00915-10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20660205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlarsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2010&pages=9864-9878&author=M.+E.+Abramauthor=A.+L.+Ferrisauthor=W.+Shaoauthor=W.+G.+Alvordauthor=S.+H.+Hughes&title=Nature%2C+position%2C+and+frequency+of+mutations+made+in+a+single+cycle+of+HIV-1+replication&doi=10.1128%2FJVI.00915-10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Nature, position, and frequency of mutations made in a single cycle of HIV-1 replication</span></div><div class="casAuthors">Abram, Michael E.; Ferris, Andrea L.; Shao, Wei; Alvord, W. Gregory; Hughes, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Virology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">9864-9878</span>CODEN:
                <span class="NLM_cas:coden">JOVIAM</span>;
        ISSN:<span class="NLM_cas:issn">0022-538X</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">There is considerable HIV-1 variation in patients.  The extent of the variation is due to the high rate of viral replication, the high viral load, and the errors made during viral replication.  Mutations can arise from errors made either by host DNA-dependent RNA polymerase II or by HIV-1 reverse transcriptase (RT), but the relative contributions of these two enzymes to the mutation rate are unknown.  In addn., mutations in RT can affect its fidelity, but the effect of mutations in RT on the nature of the mutations that arise in vivo is poorly understood.  We have developed an efficient system, based on existing technol., to analyze the mutations that arise in an HIV-1 vector in a single cycle of replication.  A lacZα reporter gene is used to identify viral DNAs that contain mutations which are analyzed by DNA sequencing.  The forward mutation rate in this system is 1.4 × 10-5 mutations/bp/cycle, equiv. to the retroviral av.  This rate is about 3-fold lower than previously reported for HIV-1 in vivo and is much lower than what has been reported for purified HIV-1 RT in vitro.  Although the mutation rate was not affected by the orientation of lacZα, the sites favored for mutations (hot spots) in lacZα depended on which strand of lacZα was present in the viral RNA.  The pattern of hot spots seen in lacZα in vivo did not match any of the published data obtained when purified RT was used to copy lacZα in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtHBhCUY9xnLVg90H21EOLACvtfcHk0lgIfyJoN1JcOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlarsrnL&md5=c9adb40a987b765cab188bb9505b0a9f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1128%2FJVI.00915-10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FJVI.00915-10%26sid%3Dliteratum%253Aachs%26aulast%3DAbram%26aufirst%3DM.%2BE.%26aulast%3DFerris%26aufirst%3DA.%2BL.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DAlvord%26aufirst%3DW.%2BG.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26atitle%3DNature%252C%2520position%252C%2520and%2520frequency%2520of%2520mutations%2520made%2520in%2520a%2520single%2520cycle%2520of%2520HIV-1%2520replication%26jtitle%3DJ.%2520Virol.%26date%3D2010%26volume%3D84%26spage%3D9864%26epage%3D9878%26doi%3D10.1128%2FJVI.00915-10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Wainberg, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaharatos, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brenner, B. G.</span><span> </span><span class="NLM_article-title">Development of antiretroviral drug resistance</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">365</span><span class="NLM_x">, </span> <span class="NLM_fpage">637</span><span class="NLM_x">–</span> <span class="NLM_lpage">646</span><span class="refDoi"> DOI: 10.1056/NEJMra1004180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1056%2FNEJMra1004180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=21848464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVKnu7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=365&publication_year=2011&pages=637-646&author=M.+A.+Wainbergauthor=G.+J.+Zaharatosauthor=B.+G.+Brenner&title=Development+of+antiretroviral+drug+resistance&doi=10.1056%2FNEJMra1004180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Antiretroviral drug resistance</span></div><div class="casAuthors">Wainberg, Mark A.; Zaharatos, Gerasimos J.; Brenner, Bluma G.</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">365</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">637-646</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">A review on numerous efforts in development and optimization of antiretroviral treatment strategies against a broad spectrum of HIV infection subtypes.  Suboptimal therapies that include such drugs as stavudine and didanosine can facilitate the selection of the K65R mutation, which, in turn, may limit many secondary treatment options.  In addn., suboptimal regimens put patients at risk for the development of nonnucleoside reverse-transcriptase inhibitor-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7uylwyOaxNbVg90H21EOLACvtfcHk0lgIfyJoN1JcOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVKnu7bF&md5=294d823167d52dca3bf4b5f769a0270c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1056%2FNEJMra1004180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMra1004180%26sid%3Dliteratum%253Aachs%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DZaharatos%26aufirst%3DG.%2BJ.%26aulast%3DBrenner%26aufirst%3DB.%2BG.%26atitle%3DDevelopment%2520of%2520antiretroviral%2520drug%2520resistance%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2011%26volume%3D365%26spage%3D637%26epage%3D646%26doi%3D10.1056%2FNEJMra1004180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Garrido, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villacian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahonero, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattery, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gutierrez, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caballero, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Houtte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soriano, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Mendoza, C.</span><span> </span><span class="NLM_article-title">Broad phenotypic cross-resistance to elvitegravir in HIV-infected patients failing on raltegravir-containing regimens</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2873</span><span class="NLM_x">–</span> <span class="NLM_lpage">2878</span><span class="refDoi"> DOI: 10.1128/AAC.06170-11</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1128%2FAAC.06170-11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2012&pages=2873-2878&author=C.+Garridoauthor=J.+Villacianauthor=N.+Zahoneroauthor=T.+Patteryauthor=F.+Garciaauthor=F.+Gutierrezauthor=E.+Caballeroauthor=M.+Van+Houtteauthor=V.+Sorianoauthor=C.+de+Mendoza&title=Broad+phenotypic+cross-resistance+to+elvitegravir+in+HIV-infected+patients+failing+on+raltegravir-containing+regimens&doi=10.1128%2FAAC.06170-11"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1128%2FAAC.06170-11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.06170-11%26sid%3Dliteratum%253Aachs%26aulast%3DGarrido%26aufirst%3DC.%26aulast%3DVillacian%26aufirst%3DJ.%26aulast%3DZahonero%26aufirst%3DN.%26aulast%3DPattery%26aufirst%3DT.%26aulast%3DGarcia%26aufirst%3DF.%26aulast%3DGutierrez%26aufirst%3DF.%26aulast%3DCaballero%26aufirst%3DE.%26aulast%3DVan%2BHoutte%26aufirst%3DM.%26aulast%3DSoriano%26aufirst%3DV.%26aulast%3Dde%2BMendoza%26aufirst%3DC.%26atitle%3DBroad%2520phenotypic%2520cross-resistance%2520to%2520elvitegravir%2520in%2520HIV-infected%2520patients%2520failing%2520on%2520raltegravir-containing%2520regimens%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2012%26volume%3D56%26spage%3D2873%26epage%3D2878%26doi%3D10.1128%2FAAC.06170-11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Geretti, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Armenia, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccherini-Silberstein, F.</span><span> </span><span class="NLM_article-title">Emerging patterns and implications of HIV-1 integrase inhibitor resistance</span> <span class="citation_source-journal">Curr. Opin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">686</span><span class="refDoi"> DOI: 10.1097/QCO.0b013e32835a1de7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1097%2FQCO.0b013e32835a1de7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=23086187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1OltbfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2012&pages=677-686&author=A.+M.+Gerettiauthor=D.+Armeniaauthor=F.+Ceccherini-Silberstein&title=Emerging+patterns+and+implications+of+HIV-1+integrase+inhibitor+resistance&doi=10.1097%2FQCO.0b013e32835a1de7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging patterns and implications of HIV-1 integrase inhibitor resistance</span></div><div class="casAuthors">Geretti, Anna Maria; Armenia, Daniele; Ceccherini-Silberstein, Francesca</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Infectious Diseases</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">677-686</span>CODEN:
                <span class="NLM_cas:coden">COIDE5</span>;
        ISSN:<span class="NLM_cas:issn">0951-7375</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: This review highlights recent data on the pathways of resistance that impact the clin. activity of first-generation and second-generation integrase inhibitors.  Recent findings: Raltegravir (RAL) and elvitegravir (EVG) are highly efficacious in first-line antiretroviral therapy, with small nos. of virol. failures obsd. in clin. trials.  Durable activity in treatment-experienced patients requires a fully supportive background regimen.  RAL and EVG show a low-to-moderate genetic barrier to resistance and extensive cross-resistance, which preclude their sequential use.  Resistance to dolutegravir (DTG) is not selected as readily in vitro and has not emerged in studies of treatment-naive patients to date.  Both in vitro and in vivo, DTG retains activity against several RAL and EVG resistant strains, but susceptibility is variably impaired by multiple mutations within the G148 pathway, which are common after RAL or EVG failure.  Cross-resistance can be partially overcome by doubling DTG dosing to twice daily, but durability of responses remains dependent on a supportive background regimen.  There is variability in the integrase gene of circulating HIV strains, which does not appear to reduce drug activity, although it may influence the emergence and evolution of integrase resistance.  Transmission of integrase resistance remains rare but surveillance is required.  Summary: Integrase inhibitors provide a potent option for the treatment of HIV infection.  Drug resistance remains a challenge, which may be partially overcome by the introduction of second-generation compds.  Prompt management of RAL and EVG failure is required to prevent the accumulation of multiple resistance mutations that reduce DTG susceptibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJNLh3j8EBULVg90H21EOLACvtfcHk0lgpBjXRTxf0QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1OltbfE&md5=ad104b6dff368c146f9cef84a553506f</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1097%2FQCO.0b013e32835a1de7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQCO.0b013e32835a1de7%26sid%3Dliteratum%253Aachs%26aulast%3DGeretti%26aufirst%3DA.%2BM.%26aulast%3DArmenia%26aufirst%3DD.%26aulast%3DCeccherini-Silberstein%26aufirst%3DF.%26atitle%3DEmerging%2520patterns%2520and%2520implications%2520of%2520HIV-1%2520integrase%2520inhibitor%2520resistance%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2012%26volume%3D25%26spage%3D677%26epage%3D686%26doi%3D10.1097%2FQCO.0b013e32835a1de7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Quashie, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesplede, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span> </span><span class="NLM_article-title">Evolution of HIV integrase resistance mutations</span> <span class="citation_source-journal">Curr. Opin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">43</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1097/QCO.0b013e32835ba81c</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1097%2FQCO.0b013e32835ba81c" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=23242340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVCru7vK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=43-49&author=P.+K.+Quashieauthor=T.+Mespledeauthor=M.+A.+Wainberg&title=Evolution+of+HIV+integrase+resistance+mutations&doi=10.1097%2FQCO.0b013e32835ba81c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of HIV integrase resistance mutations</span></div><div class="casAuthors">Quashie, Peter K.; Mesplede, Thibault; Wainberg, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Infectious Diseases</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">43-49</span>CODEN:
                <span class="NLM_cas:coden">COIDE5</span>;
        ISSN:<span class="NLM_cas:issn">0951-7375</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A review discussing integrase strand transfer inhibitors (INSTIs) have become a key component of antiretroviral therapy since the approval of twice-daily raltegravir in 2007 and the more recent approval of elvitegravir in 2012.  At the same time, a third compd., dolutegravir, is in late-phase clin. trials, being tested as part of a multidrug once-daily formulation comprising this INSTI and two other antiretroviral (ARV) drugs.  This review focuses on the factors leading to the development of drug resistance mutations (DRMs) against INSTIs, evidence of cross-resistance among them, and the results of regimen simplification in regard to this topic.  Recent findings: Sequencing data show that DRMs are highly dynamic in patients failing INSTI therapy.  Considerations of viral fitness and drug resistance can together det. the evolution of drug resistance mutations, and in this regard the Y143 and Q148 pathways are superior to the N155 pathway in the promotion of resistance.  Preventing the emergence of DRMs requires that effective reverse transcriptase or other inhibitors be used together with INSTIs and that high-level adherence to treatment be maintained.  Summary: Because of the susceptibility to drug resistance, INSTIs should always be used together with other effective ARV drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqwQyw6THfTb7Vg90H21EOLACvtfcHk0lgpBjXRTxf0QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVCru7vK&md5=e0fe7fb17f03a29ce13e8579b9baeac4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1097%2FQCO.0b013e32835ba81c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FQCO.0b013e32835ba81c%26sid%3Dliteratum%253Aachs%26aulast%3DQuashie%26aufirst%3DP.%2BK.%26aulast%3DMesplede%26aufirst%3DT.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26atitle%3DEvolution%2520of%2520HIV%2520integrase%2520resistance%2520mutations%26jtitle%3DCurr.%2520Opin.%2520Infect.%2520Dis.%26date%3D2013%26volume%3D26%26spage%3D43%26epage%3D49%26doi%3D10.1097%2FQCO.0b013e32835ba81c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Ballantyne, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, C. M.</span><span> </span><span class="NLM_article-title">Dolutegravir: First global approval</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">73</span><span class="NLM_x">, </span> <span class="NLM_fpage">1627</span><span class="NLM_x">–</span> <span class="NLM_lpage">1637</span><span class="refDoi"> DOI: 10.1007/s40265-013-0121-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1007%2Fs40265-013-0121-4" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1627-1637&author=A.+D.+Ballantyneauthor=C.+M.+Perry&title=Dolutegravir%3A+First+global+approval&doi=10.1007%2Fs40265-013-0121-4"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0121-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0121-4%26sid%3Dliteratum%253Aachs%26aulast%3DBallantyne%26aufirst%3DA.%2BD.%26aulast%3DPerry%26aufirst%3DC.%2BM.%26atitle%3DDolutegravir%253A%2520First%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1627%26epage%3D1637%26doi%3D10.1007%2Fs40265-013-0121-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Hurt, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sebastian, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hicks, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eron, J. J.</span><span> </span><span class="NLM_article-title">Resistance to HIV integrase strand transfer inhibitors among clinical specimens in the united states, 2009–2012</span> <span class="citation_source-journal">Clin. Infect. Dis.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">431</span><span class="refDoi"> DOI: 10.1093/cid/cit697</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1093%2Fcid%2Fcit697" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24145878" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVOmtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2014&pages=423-431&author=C.+B.+Hurtauthor=J.+Sebastianauthor=C.+B.+Hicksauthor=J.+J.+Eron&title=Resistance+to+HIV+integrase+strand+transfer+inhibitors+among+clinical+specimens+in+the+united+states%2C+2009%E2%80%932012&doi=10.1093%2Fcid%2Fcit697"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to HIV Integrase Strand Transfer Inhibitors Among Clinical Specimens in the United States, 2009-2012</span></div><div class="casAuthors">Hurt, Christopher B.; Sebastian, Joseph; Hicks, Charles B.; Eron, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Infectious Diseases</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">423-431</span>CODEN:
                <span class="NLM_cas:coden">CIDIEL</span>;
        ISSN:<span class="NLM_cas:issn">1058-4838</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Among 3012 patients with clin. specimens submitted for integrase resistance genotyping from Jan. 2009 to Dec. 2012, 15.6% harbored ≥1 "major" integrase inhibitor mutation.  Intermediate or high-level dolutegravir resistance was predicted for 6.4% (41.2% of those with ≥1 major mutation).  Background. Data on integrase inhibitor resistance come primarily from clin. trials and in vitro studies.  We examd. results of all clin. indicated integrase genotypic resistance tests (GRTs) performed at a US national referral lab from 2009 through 2012.  Methods. Integrase sequences and demog. data were compiled with paired protease-reverse transcriptase (PR-RT) GRT results, when available.  Analyses utilized the Stanford HIV Drug Resistance Database.  "Major" integrase mutations included T66AIK, E92QV, F121Y, Y143CHR, S147G, Q148HKR, and N155H; multiple accessory mutations were also assessed.  Results. Among 3294 sequences from 3012 patients, 471 patients had viruses with ≥1 raltegravir or elvitegravir resistance mutation (15.6%).  Q148 and N155 pathways were equally represented (both n = 197); 84 had Y143 mutations.  Q148 rarely occurred without accessory mutations (n = 3).  Among 224 patients with serial integrase GRTs, 22 with baseline wild-type acquired a major mutation, after a median 224 days between tests (interquartile range, 148-335 days).  Major mutations were obsd. to persist up to 462 days.  Most (62%) had paired PR-RT results.  Patients with integrase-resistant viruses were older and more likely to have PR-RT mutations (both P < .001).  Among those with PR-RT data, 42 patients had 4-class resistance (2.3%).  Sex, geog. region, and test year were not assocd. with integrase resistance.  High-level dolutegravir resistance was predicted in 12% of patients with raltegravir- or elvitegravir-resistant viruses (2% of all patients).  Conclusions. Approx. 1 in 6 US patients undergoing integrase GRT for clin. decision making harbors significant resistance, with Q148 and N155 pathways equally common.  Dolutegravir is likely to have full or partial activity against most variants obsd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqL2okgFAZ2nLVg90H21EOLACvtfcHk0lgpBjXRTxf0QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVOmtrY%253D&md5=570b2787981051ef85c9b3230761429d</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1093%2Fcid%2Fcit697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fcid%252Fcit697%26sid%3Dliteratum%253Aachs%26aulast%3DHurt%26aufirst%3DC.%2BB.%26aulast%3DSebastian%26aufirst%3DJ.%26aulast%3DHicks%26aufirst%3DC.%2BB.%26aulast%3DEron%26aufirst%3DJ.%2BJ.%26atitle%3DResistance%2520to%2520HIV%2520integrase%2520strand%2520transfer%2520inhibitors%2520among%2520clinical%2520specimens%2520in%2520the%2520united%2520states%252C%25202009%25E2%2580%25932012%26jtitle%3DClin.%2520Infect.%2520Dis.%26date%3D2014%26volume%3D58%26spage%3D423%26epage%3D431%26doi%3D10.1093%2Fcid%2Fcit697" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">McCormack, P. L.</span><span> </span><span class="NLM_article-title">Dolutegravir: A review of its use in the management of HIV-1 infection in adolescents and adults</span> <span class="citation_source-journal">Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241</span><span class="NLM_x">–</span> <span class="NLM_lpage">1254</span><span class="refDoi"> DOI: 10.1007/s40265-014-0256-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1007%2Fs40265-014-0256-y" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1241-1254&author=P.+L.+McCormack&title=Dolutegravir%3A+A+review+of+its+use+in+the+management+of+HIV-1+infection+in+adolescents+and+adults&doi=10.1007%2Fs40265-014-0256-y"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs40265-014-0256-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-014-0256-y%26sid%3Dliteratum%253Aachs%26aulast%3DMcCormack%26aufirst%3DP.%2BL.%26atitle%3DDolutegravir%253A%2520A%2520review%2520of%2520its%2520use%2520in%2520the%2520management%2520of%2520HIV-1%2520infection%2520in%2520adolescents%2520and%2520adults%26jtitle%3DDrugs%26date%3D2014%26volume%3D74%26spage%3D1241%26epage%3D1254%26doi%3D10.1007%2Fs40265-014-0256-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Mesplede, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quashie, P. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wainberg, M. A.</span><span> </span><span class="NLM_article-title">Resistance to HIV integrase inhibitors</span> <span class="citation_source-journal">Curr. Opin. HIV AIDS</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">401</span><span class="NLM_x">–</span> <span class="NLM_lpage">408</span><span class="refDoi"> DOI: 10.1097/COH.0b013e328356db89</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1097%2FCOH.0b013e328356db89" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=22789986" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2htLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=401-408&author=T.+Mespledeauthor=P.+K.+Quashieauthor=M.+A.+Wainberg&title=Resistance+to+HIV+integrase+inhibitors&doi=10.1097%2FCOH.0b013e328356db89"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to HIV integrase inhibitors</span></div><div class="casAuthors">Mesplede, Thibault; Quashie, Peter K.; Wainberg, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in HIV & AIDS</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">401-408</span>CODEN:
                <span class="NLM_cas:coden">COHABM</span>;
        ISSN:<span class="NLM_cas:issn">1746-630X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: HIV integrase inhibitors are potent antiretroviral drugs that efficiently decrease viral load in patients.  Emergence of resistance mutations against this new class of drugs represents a threat to their long-term efficacy.  The purpose of this review is to provide new information about the most recent mutations identified and other mutations that confer resistance to several integrase inhibitors.  Recent findings: New resistance mutations, such as G118R, R263K and S153Y, have been recently identified through in-vitro selection studies with second-generation integrase strand-transfer inhibitors (INSTIs).  These add to the three main resistance pathways involving mutations at positions Y143, N155 and Q148.  Structural modeling, biochem. analyses and deep sequencing are methods that currently help in the understanding of the mechanisms of resistance conferred by these mutations.  Although these new resistance mutations appear to confer only low levels of cross-resistance to second-generation drugs, the Q148 pathway with numerous secondary mutations has the potential to significantly decrease susceptibility to all drugs of the INSTI family.  Summary: Recent mutations selected in vitro with second-generation INSTIs suggest the existence of low levels of cross-resistance between these drugs and first-generation compds.  In clin. practice, the emergence of mutations at position Q148 should be monitored whenever possible.  More datasets are needed to assess the long-term efficacy of second-generation INSTIs in patients failing older INSTIs such as raltegravir and elvitegravir.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsULhoKhjr9bVg90H21EOLACvtfcHk0lgooT9SzGzxAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2htLrO&md5=aea851c3b23668be783f7584cb9e9507</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1097%2FCOH.0b013e328356db89&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FCOH.0b013e328356db89%26sid%3Dliteratum%253Aachs%26aulast%3DMesplede%26aufirst%3DT.%26aulast%3DQuashie%26aufirst%3DP.%2BK.%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26atitle%3DResistance%2520to%2520HIV%2520integrase%2520inhibitors%26jtitle%3DCurr.%2520Opin.%2520HIV%2520AIDS%26date%3D2012%26volume%3D7%26spage%3D401%26epage%3D408%26doi%3D10.1097%2FCOH.0b013e328356db89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizuuchi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craigie, R.</span><span> </span><span class="NLM_article-title">Retroviral DNA integration: reaction pathway and critical intermediates</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1295</span><span class="NLM_x">–</span> <span class="NLM_lpage">1304</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7601005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Fsj.emboj.7601005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=1295-1304&author=M.+Liauthor=M.+Mizuuchiauthor=T.+R.+Burkeauthor=R.+Craigie&title=Retroviral+DNA+integration%3A+reaction+pathway+and+critical+intermediates&doi=10.1038%2Fsj.emboj.7601005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601005%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DM.%26aulast%3DMizuuchi%26aufirst%3DM.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DCraigie%26aufirst%3DR.%26atitle%3DRetroviral%2520DNA%2520integration%253A%2520reaction%2520pathway%2520and%2520critical%2520intermediates%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D1295%26epage%3D1304%26doi%3D10.1038%2Fsj.emboj.7601005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valkov, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Retroviral intasome assembly and inhibition of DNA strand transfer</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">464</span><span class="NLM_x">, </span> <span class="NLM_fpage">232</span><span class="NLM_x">–</span> <span class="NLM_lpage">236</span><span class="refDoi"> DOI: 10.1038/nature08784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Fnature08784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20118915" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=464&publication_year=2010&pages=232-236&author=S.+Hareauthor=S.+Guptaauthor=E.+Valkovauthor=A.+Engelmanauthor=P.+Cherepanov&title=Retroviral+intasome+assembly+and+inhibition+of+DNA+strand+transfer&doi=10.1038%2Fnature08784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Retroviral intasome assembly and inhibition of DNA strand transfer</span></div><div class="casAuthors">Hare, Stephen; Gupta, Saumya Shree; Valkov, Eugene; Engelman, Alan; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">464</span>
        (<span class="NLM_cas:issue">7286</span>),
    <span class="NLM_cas:pages">232-236</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Integrase is an essential retroviral enzyme that binds both termini of linear viral DNA and inserts them into a host cell chromosome.  The structure of full-length retroviral integrase, either sep. or in complex with DNA, has been lacking.  Furthermore, although clin. useful inhibitors of HIV integrase have been developed, their mechanism of action remains speculative.  Here the authors present a crystal structure of full-length integrase from the prototype foamy virus in complex with its cognate DNA.  The structure shows the organization of the retroviral intasome comprising an integrase tetramer tightly assocd. with a pair of viral DNA ends.  All three canonical integrase structural domains are involved in extensive protein-DNA and protein-protein interactions.  The binding of strand-transfer inhibitors displaces the reactive viral DNA end from the active site, disarming the viral nucleoprotein complex.  These findings define the structural basis of retroviral DNA integration, and will allow modeling of the HIV-1 intasome to aid in the development of antiretroviral drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqdEDTFdvZMbVg90H21EOLACvtfcHk0lgooT9SzGzxAw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yntLg%253D&md5=9551e74b6bc33494855651c3b157d753</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fnature08784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08784%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DValkov%26aufirst%3DE.%26aulast%3DEngelman%26aufirst%3DA.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DRetroviral%2520intasome%2520assembly%2520and%2520inhibition%2520of%2520DNA%2520strand%2520transfer%26jtitle%3DNature%26date%3D2010%26volume%3D464%26spage%3D232%26epage%3D236%26doi%3D10.1038%2Fnature08784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Passos, D. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rebensburg, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghirlando, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeon, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shkriabai, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kvaratskhelia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Craigie, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lyumkis, D.</span><span> </span><span class="NLM_article-title">Cryo-EM structures and atomic model of the HIV-1 strand transfer complex intasome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">89</span><span class="NLM_x">–</span> <span class="NLM_lpage">92</span><span class="refDoi"> DOI: 10.1126/science.aah5163</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1126%2Fscience.aah5163" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=28059769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1WjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=89-92&author=D.+O.+Passosauthor=M.+Liauthor=R.+Yangauthor=S.+V.+Rebensburgauthor=R.+Ghirlandoauthor=Y.+Jeonauthor=N.+Shkriabaiauthor=M.+Kvaratskheliaauthor=R.+Craigieauthor=D.+Lyumkis&title=Cryo-EM+structures+and+atomic+model+of+the+HIV-1+strand+transfer+complex+intasome&doi=10.1126%2Fscience.aah5163"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM structures and atomic model of the HIV-1 strand transfer complex intasome</span></div><div class="casAuthors">Passos, Dario Oliveira; Li, Min; Yang, Renbin; Rebensburg, Stephanie V.; Ghirlando, Rodolfo; Jeon, Youngmin; Shkriabai, Nikoloz; Kvaratskhelia, Mamuka; Craigie, Robert; Lyumkis, Dmitry</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6320</span>),
    <span class="NLM_cas:pages">89-92</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Like all retroviruses, HIV-1 irreversibly inserts a viral DNA (vDNA) copy of its RNA genome into host target DNA (tDNA).  The intasome, a higher-order nucleoprotein complex composed of viral integrase (IN) and the ends of linear vDNA, mediates integration.  Productive integration into host chromatin results in the formation of the strand transfer complex (STC) contg. catalytically joined vDNA and tDNA.  HIV-1 intasomes have been refractory to high-resoln. structural studies.  Here, the authors used a sol. IN fusion protein to facilitate structural studies, through which they present a high-resoln. cryo-electron microscopy (cryo-EM) structure of the core tetrameric HIV-1 STC and a higher-order form that adopts C-terminal domain rearrangements.  The distinct STC structures highlighted how HIV-1 could use the common retroviral intasome core architecture to accommodate different IN domain modules for assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSEOCZd1EWr7Vg90H21EOLACvtfcHk0lhfBXVoDJPSJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1WjtQ%253D%253D&md5=f69a83cd205c453c055ff9ba6ac10723</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1126%2Fscience.aah5163&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aah5163%26sid%3Dliteratum%253Aachs%26aulast%3DPassos%26aufirst%3DD.%2BO.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DRebensburg%26aufirst%3DS.%2BV.%26aulast%3DGhirlando%26aufirst%3DR.%26aulast%3DJeon%26aufirst%3DY.%26aulast%3DShkriabai%26aufirst%3DN.%26aulast%3DKvaratskhelia%26aufirst%3DM.%26aulast%3DCraigie%26aufirst%3DR.%26aulast%3DLyumkis%26aufirst%3DD.%26atitle%3DCryo-EM%2520structures%2520and%2520atomic%2520model%2520of%2520the%2520HIV-1%2520strand%2520transfer%2520complex%2520intasome%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D89%26epage%3D92%26doi%3D10.1126%2Fscience.aah5163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Ballandras-Colas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maskell, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serrao, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Locke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swuec, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jónsson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotecha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pye, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrésdóttir, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">A supramolecular assembly mediates lentiviral DNA integration</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">355</span><span class="NLM_x">, </span> <span class="NLM_fpage">93</span><span class="NLM_x">–</span> <span class="NLM_lpage">95</span><span class="refDoi"> DOI: 10.1126/science.aah7002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1126%2Fscience.aah7002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=28059770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Wjug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=93-95&author=A.+Ballandras-Colasauthor=D.+P.+Maskellauthor=E.+Serraoauthor=J.+Lockeauthor=P.+Swuecauthor=S.+R.+J%C3%B3nssonauthor=A.+Kotechaauthor=N.+J.+Cookauthor=V.+E.+Pyeauthor=I.+A.+Taylorauthor=V.+Andr%C3%A9sd%C3%B3ttirauthor=A.+N.+Engelmanauthor=A.+Costaauthor=P.+Cherepanov&title=A+supramolecular+assembly+mediates+lentiviral+DNA+integration&doi=10.1126%2Fscience.aah7002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">A supramolecular assembly mediates lentiviral DNA integration</span></div><div class="casAuthors">Ballandras-Colas, Allison; Maskell, Daniel P.; Serrao, Erik; Locke, Julia; Swuec, Paolo; Jonsson, Stefan R.; Kotecha, Abhay; Cook, Nicola J.; Pye, Valerie E.; Taylor, Ian A.; Andresdottir, Valgerdur; Engelman, Alan N.; Costa, Alessandro; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6320</span>),
    <span class="NLM_cas:pages">93-95</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Retroviral integrase (IN) functions within the intasome nucleoprotein complex to catalyze insertion of viral DNA into cellular chromatin.  Using cryo-electron microscopy, we now visualize the functional maedi-visna lentivirus intasome at 4.9 angstrom resoln.  The intasome comprises a homo-hexadecamer of IN with a tetramer-of-tetramers architecture featuring eight structurally distinct types of IN protomers supporting two catalytically competent subunits.  The conserved intasomal core, previously obsd. in simpler retroviral systems, is formed between two IN tetramers, with a pair of C-terminal domains from flanking tetramers completing the synaptic interface.  Our results explain how HIV-1 IN, which self-assocs. into higher-order multimers, can form a functional intasome, reconcile the bulk of early HIV-1 IN biochem. and structural data, and provide a lentiviral platform for design of HIV-1 IN inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvoOxgcU9YVbVg90H21EOLACvtfcHk0lhfBXVoDJPSJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Wjug%253D%253D&md5=769d0fcbeaa6454e0feee221e344db9e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1126%2Fscience.aah7002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aah7002%26sid%3Dliteratum%253Aachs%26aulast%3DBallandras-Colas%26aufirst%3DA.%26aulast%3DMaskell%26aufirst%3DD.%2BP.%26aulast%3DSerrao%26aufirst%3DE.%26aulast%3DLocke%26aufirst%3DJ.%26aulast%3DSwuec%26aufirst%3DP.%26aulast%3DJ%25C3%25B3nsson%26aufirst%3DS.%2BR.%26aulast%3DKotecha%26aufirst%3DA.%26aulast%3DCook%26aufirst%3DN.%2BJ.%26aulast%3DPye%26aufirst%3DV.%2BE.%26aulast%3DTaylor%26aufirst%3DI.%2BA.%26aulast%3DAndr%25C3%25A9sd%25C3%25B3ttir%26aufirst%3DV.%26aulast%3DEngelman%26aufirst%3DA.%2BN.%26aulast%3DCosta%26aufirst%3DA.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DA%2520supramolecular%2520assembly%2520mediates%2520lentiviral%2520DNA%2520integration%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D93%26epage%3D95%26doi%3D10.1126%2Fscience.aah7002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Espeseth, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Felock, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witmer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grobler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anthony, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Egbertson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melamed, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamill, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cole, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span> </span><span class="NLM_article-title">HIV-1 integrase inhibitors that compete with the target DNA substrate define a unique strand transfer conformation for integrase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">11244</span><span class="NLM_x">–</span> <span class="NLM_lpage">11249</span><span class="refDoi"> DOI: 10.1073/pnas.200139397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1073%2Fpnas.200139397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=11016953" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnsF2rtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=11244-11249&author=A.+S.+Espesethauthor=P.+Felockauthor=A.+Wolfeauthor=M.+Witmerauthor=J.+Groblerauthor=N.+Anthonyauthor=M.+Egbertsonauthor=J.+Y.+Melamedauthor=S.+Youngauthor=T.+Hamillauthor=J.+L.+Coleauthor=D.+J.+Hazuda&title=HIV-1+integrase+inhibitors+that+compete+with+the+target+DNA+substrate+define+a+unique+strand+transfer+conformation+for+integrase&doi=10.1073%2Fpnas.200139397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">HIV-1 integrase inhibitors that compete with the target DNA substrate define a unique strand transfer conformation for integrase</span></div><div class="casAuthors">Espeseth, Amy S.; Felock, Peter; Wolfe, Abigail; Witmer, Marc; Grobler, Jay; Anthony, Neville; Egbertson, Melissa; Melamed, Jeffrey Y.; Young, Steve; Hamill, Terence; Cole, James L.; Hazuda, Daria J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">11244-11249</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Diketo acids such as L-731,988 are potent inhibitors of HIV-1 integrase that inhibit integration and viral replication in cells.  These compds. exhibit the unique ability to inhibit the strand transfer activity of integrase in the absence of an effect on 3' end processing.  To understand the reasons for this distinct inhibitory profile, we developed a scintillation proximity assay that permits anal. of radiolabeled inhibitor binding and integrase function.  High-affinity binding of L-731,988 is shown to require the assembly of a specific complex on the HIV-1 long terminal repeat.  The interaction of L-731,988 with the complex and the efficacy of L-731,988 in strand transfer can be abrogated by the interaction with target substrates, suggesting competition between the inhibitor and the target DNA.  The L-731,988 binding site and that of the target substrate are thus distinct from that of the donor substrate and are defined by a conformation of integrase that is only adopted after assembly with the viral end.  These results elucidate the basis for diketo acid inhibition of strand transfer and have implications for integrase-directed HIV-1 drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa7Er-rFxWyLVg90H21EOLACvtfcHk0lhfBXVoDJPSJQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXnsF2rtb0%253D&md5=d3e049d527ec4b0349e0c902f61742dd</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1073%2Fpnas.200139397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.200139397%26sid%3Dliteratum%253Aachs%26aulast%3DEspeseth%26aufirst%3DA.%2BS.%26aulast%3DFelock%26aufirst%3DP.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DWitmer%26aufirst%3DM.%26aulast%3DGrobler%26aufirst%3DJ.%26aulast%3DAnthony%26aufirst%3DN.%26aulast%3DEgbertson%26aufirst%3DM.%26aulast%3DMelamed%26aufirst%3DJ.%2BY.%26aulast%3DYoung%26aufirst%3DS.%26aulast%3DHamill%26aufirst%3DT.%26aulast%3DCole%26aufirst%3DJ.%2BL.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26atitle%3DHIV-1%2520integrase%2520inhibitors%2520that%2520compete%2520with%2520the%2520target%2520DNA%2520substrate%2520define%2520a%2520unique%2520strand%2520transfer%2520conformation%2520for%2520integrase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D11244%26epage%3D11249%26doi%3D10.1073%2Fpnas.200139397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jaxa-Chamiec, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Structural and functional analyses of the second-generation integrase strand transfer inhibitor dolutegravir (S/GSK1349572)</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">565</span><span class="NLM_x">–</span> <span class="NLM_lpage">572</span><span class="refDoi"> DOI: 10.1124/mol.111.073189</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1124%2Fmol.111.073189" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2011&pages=565-572&author=S.+Hareauthor=S.+J.+Smithauthor=M.+Metifiotauthor=A.+Jaxa-Chamiecauthor=Y.+Pommierauthor=S.+H.+Hughesauthor=P.+Cherepanov&title=Structural+and+functional+analyses+of+the+second-generation+integrase+strand+transfer+inhibitor+dolutegravir+%28S%2FGSK1349572%29&doi=10.1124%2Fmol.111.073189"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1124%2Fmol.111.073189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fmol.111.073189%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DJaxa-Chamiec%26aufirst%3DA.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DStructural%2520and%2520functional%2520analyses%2520of%2520the%2520second-generation%2520integrase%2520strand%2520transfer%2520inhibitor%2520dolutegravir%2520%2528S%252FGSK1349572%2529%26jtitle%3DMol.%2520Pharmacol.%26date%3D2011%26volume%3D80%26spage%3D565%26epage%3D572%26doi%3D10.1124%2Fmol.111.073189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vos, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clayton, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thuring, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span> </span><span class="NLM_article-title">Molecular mechanisms of retroviral integrase inhibition and the evolution of viral resistance</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">20057</span><span class="NLM_x">–</span> <span class="NLM_lpage">20062</span><span class="refDoi"> DOI: 10.1073/pnas.1010246107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1073%2Fpnas.1010246107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=21030679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyntrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=20057-20062&author=S.+Hareauthor=A.+M.+Vosauthor=R.+F.+Claytonauthor=J.+W.+Thuringauthor=M.+D.+Cummingsauthor=P.+Cherepanov&title=Molecular+mechanisms+of+retroviral+integrase+inhibition+and+the+evolution+of+viral+resistance&doi=10.1073%2Fpnas.1010246107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms of retroviral integrase inhibition and the evolution of viral resistance</span></div><div class="casAuthors">Hare, Stephen; Vos, Ann M.; Clayton, Reginald F.; Thuring, Jan W.; Cummings, Maxwell D.; Cherepanov, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">20057-20062, S20057/1-S20057/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The development of HIV integrase (IN) strand transfer inhibitors (INSTIs) and understanding of viral resistance to these mols. have been hampered by a paucity of available structural data.  Co-crystal structures of the prototype foamy virus (PFV) intasome with raltegravir and elvitegravir were recently reported, establishing the general INSTI binding mode.  Now an expanded set of co-crystal structures contg. PFV intasomes complexed with first- and second-generation INSTIs at resolns. of up to 2.5 Å is presented.  Importantly, the improved resoln. allowed the authors to refine the complete coordination spheres of the catalytic metal cations within the INSTI-bound intasome active site.  It is shown that like the Q148H/G140S and N155H HIV-1 IN variants, the analogous S217H and N224H PFV INs display reduced sensitivity to raltegravir in vitro.  Crystal structures of the mutant PFV intasomes in INSTI-free and -bound forms revealed that the amino acid substitutions necessitate considerable conformational rearrangements within the IN active site to accommodate an INSTI, thus explaining their adverse effects on raltegravir antiviral activity.  Furthermore, the structures predict phys. proximity and an interaction between HIV-1 IN mutant residues His148 and Ser/Ala140, rationalizing the co-evolution of Q148H and G140S/A mutations in drug-resistant viral strains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovxU36XC1DQrVg90H21EOLACvtfcHk0lhAwxvG8R_R4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyntrnF&md5=5d66aa03a9478dad36e878d43c408099</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1010246107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1010246107%26sid%3Dliteratum%253Aachs%26aulast%3DHare%26aufirst%3DS.%26aulast%3DVos%26aufirst%3DA.%2BM.%26aulast%3DClayton%26aufirst%3DR.%2BF.%26aulast%3DThuring%26aufirst%3DJ.%2BW.%26aulast%3DCummings%26aufirst%3DM.%2BD.%26aulast%3DCherepanov%26aufirst%3DP.%26atitle%3DMolecular%2520mechanisms%2520of%2520retroviral%2520integrase%2520inhibition%2520and%2520the%2520evolution%2520of%2520viral%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D20057%26epage%3D20062%26doi%3D10.1073%2Fpnas.1010246107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span> </span><span class="NLM_article-title">Interfacial inhibitors: targeting macromolecular complexes</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">36</span><span class="refDoi"> DOI: 10.1038/nrd3404</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1038%2Fnrd3404" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1aitr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=25-36&author=Y.+Pommierauthor=C.+Marchand&title=Interfacial+inhibitors%3A+targeting+macromolecular+complexes&doi=10.1038%2Fnrd3404"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Interfacial inhibitors: targeting macromolecular complexes</span></div><div class="casAuthors">Pommier, Yves; Marchand, Christophe</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-36</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Interfacial inhibitors belong to a broad class of natural products and synthetic drugs that are commonly used to treat cancers as well as bacterial and HIV infections.  They bind selectively to interfaces as macromol. machines assemble and are set in motion.  The bound drugs transiently arrest the targeted mol. machines, which can initiate allosteric effects, or desynchronize macromol. machines that normally function in concert.  Here, we review five archetypical examples of interfacial inhibitors: the camptothecins, etoposide, the quinolone antibiotics, the vinca alkaloids and the novel anti-HIV inhibitor raltegravir.  We discuss the common and diverging elements between interfacial and allosteric inhibitors and give a perspective for the rationale and methods used to discover novel interfacial inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjeQ4_1kAh-bVg90H21EOLACvtfcHk0lhAwxvG8R_R4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1aitr3F&md5=e69c1b97286dc65855ee8784203d4d85</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrd3404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3404%26sid%3Dliteratum%253Aachs%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DMarchand%26aufirst%3DC.%26atitle%3DInterfacial%2520inhibitors%253A%2520targeting%2520macromolecular%2520complexes%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D25%26epage%3D36%26doi%3D10.1038%2Fnrd3404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span> </span><span class="NLM_article-title">Bicyclic 1-hydroxy-2-oxo-1,2-dihydropyridine-3-carboxamide-containing HIV-1 integrase inhibitors having high antiviral potency against cells harboring raltegravir-resistant integrase mutants</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">1573</span><span class="NLM_x">–</span> <span class="NLM_lpage">1582</span><span class="refDoi"> DOI: 10.1021/jm401902n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401902n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=1573-1582&author=X.+Z.+Zhaoauthor=S.+J.+Smithauthor=M.+Metifiotauthor=B.+C.+Johnsonauthor=C.+Marchandauthor=Y.+Pommierauthor=S.+H.+Hughesauthor=T.+R.+Burke&title=Bicyclic+1-hydroxy-2-oxo-1%2C2-dihydropyridine-3-carboxamide-containing+HIV-1+integrase+inhibitors+having+high+antiviral+potency+against+cells+harboring+raltegravir-resistant+integrase+mutants&doi=10.1021%2Fjm401902n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjm401902n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401902n%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DJohnson%26aufirst%3DB.%2BC.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3DBicyclic%25201-hydroxy-2-oxo-1%252C2-dihydropyridine-3-carboxamide-containing%2520HIV-1%2520integrase%2520inhibitors%2520having%2520high%2520antiviral%2520potency%2520against%2520cells%2520harboring%2520raltegravir-resistant%2520integrase%2520mutants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D1573%26epage%3D1582%26doi%3D10.1021%2Fjm401902n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyer, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span> </span><span class="NLM_article-title">4-Amino-1-hydroxy-2-oxo-1,8-naphthyridine-containing compounds having high potency against raltegravir-resistant integrase mutants of HIV-1</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">5190</span><span class="NLM_x">–</span> <span class="NLM_lpage">5202</span><span class="refDoi"> DOI: 10.1021/jm5001908</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5001908" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptlyqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=5190-5202&author=X.+Z.+Zhaoauthor=S.+J.+Smithauthor=M.+Metifiotauthor=C.+Marchandauthor=P.+L.+Boyerauthor=Y.+Pommierauthor=S.+H.+Hughesauthor=T.+R.+Burke&title=4-Amino-1-hydroxy-2-oxo-1%2C8-naphthyridine-containing+compounds+having+high+potency+against+raltegravir-resistant+integrase+mutants+of+HIV-1&doi=10.1021%2Fjm5001908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">4-Amino-1-hydroxy-2-oxo-1,8-naphthyridine-Containing Compounds Having High Potency against Raltegravir-Resistant Integrase Mutants of HIV-1</span></div><div class="casAuthors">Zhao, Xue Zhi; Smith, Steven J.; Metifiot, Mathieu; Marchand, Christophe; Boyer, Paul L.; Pommier, Yves; Hughes, Stephen H.; Burke, Terrence R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5190-5202</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">There are currently three HIV-1 integrase (IN) strand transfer inhibitors (INSTIs) approved by the FDA for the treatment of AIDS.  However, the emergence of drug-resistant mutants emphasizes the need to develop addnl. agents that have improved efficacies against the existent resistant mutants.  As reported herein, we modified our recently disclosed 1-hydroxy-2-oxo-1,2-dihydro-1,8-naphthyridine-3-carboxamides IN inhibitors to develop compds. that have improved efficacies against recombinant IN in biochem. assays.  These new compds. show single-digit nanomolar antiviral potencies against HIV vectors that carry wild-type (WT) IN in a single round replication assay and have improved potency against vectors harboring the major forms of drug resistant IN mutants.  These compds. also have low toxicity for cultured cells, which in several cases, results in selectivity indexes (CC50/EC50) of greater than 10000.  The compds. have the potential, with addnl. structural modifications, to yield clin. agents that are effective against the known strains of resistant viruses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCcPytjSVBlLVg90H21EOLACvtfcHk0ljCfFKH8HOl_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptlyqtbY%253D&md5=047465ac43cc10dbd31eeeaf99166b8c</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm5001908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5001908%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DBoyer%26aufirst%3DP.%2BL.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3D4-Amino-1-hydroxy-2-oxo-1%252C8-naphthyridine-containing%2520compounds%2520having%2520high%2520potency%2520against%2520raltegravir-resistant%2520integrase%2520mutants%2520of%2520HIV-1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D5190%26epage%3D5202%26doi%3D10.1021%2Fjm5001908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Wensing, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günthard, H. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paredes, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pillay, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shafer, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richman, D. D.</span><span> </span><span class="NLM_article-title">2017 Update of the drug resistance mutations in HIV-1</span> <span class="citation_source-journal">Top. Anitivir. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">132</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=132-141&author=A.+M.+Wensingauthor=V.+Calvezauthor=H.+F.+G%C3%BCnthardauthor=V.+A.+Johnsonauthor=R.+Paredesauthor=D.+Pillayauthor=R.+W.+Shaferauthor=D.+D.+Richman&title=2017+Update+of+the+drug+resistance+mutations+in+HIV-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWensing%26aufirst%3DA.%2BM.%26aulast%3DCalvez%26aufirst%3DV.%26aulast%3DG%25C3%25BCnthard%26aufirst%3DH.%2BF.%26aulast%3DJohnson%26aufirst%3DV.%2BA.%26aulast%3DParedes%26aufirst%3DR.%26aulast%3DPillay%26aufirst%3DD.%26aulast%3DShafer%26aufirst%3DR.%2BW.%26aulast%3DRichman%26aufirst%3DD.%2BD.%26atitle%3D2017%2520Update%2520of%2520the%2520drug%2520resistance%2520mutations%2520in%2520HIV-1%26jtitle%3DTop.%2520Anitivir.%2520Med.%26date%3D2017%26volume%3D14%26spage%3D132%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Kawasuji, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johns, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weatherhead, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akiyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taishi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taoda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikamiyama-Iwata, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiyama, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fuji, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanimoto, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshinaga, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garvey, E. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, T.</span><span> </span><span class="NLM_article-title">Carbamoyl pyridone HIV-1 integrase inhibitors. 2. Bi- and tricyclic derivatives result in superior antiviral and pharmacokinetic profiles</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1124</span><span class="NLM_x">–</span> <span class="NLM_lpage">1135</span><span class="refDoi"> DOI: 10.1021/jm301550c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301550c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3sXntVeltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1124-1135&author=T.+Kawasujiauthor=B.+A.+Johnsauthor=H.+Yoshidaauthor=J.+G.+Weatherheadauthor=T.+Akiyamaauthor=T.+Taishiauthor=Y.+Taodaauthor=M.+Mikamiyama-Iwataauthor=H.+Muraiauthor=R.+Kiyamaauthor=M.+Fujiauthor=N.+Tanimotoauthor=T.+Yoshinagaauthor=T.+Sekiauthor=M.+Kobayashiauthor=A.+Satoauthor=E.+P.+Garveyauthor=T.+Fujiwara&title=Carbamoyl+pyridone+HIV-1+integrase+inhibitors.+2.+Bi-+and+tricyclic+derivatives+result+in+superior+antiviral+and+pharmacokinetic+profiles&doi=10.1021%2Fjm301550c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Carbamoyl Pyridone HIV-1 Integrase Inhibitors. 2. Bi- and Tricyclic Derivatives Result in Superior Antiviral and Pharmacokinetic Profiles</span></div><div class="casAuthors">Kawasuji, Takashi; Johns, Brian A.; Yoshida, Hiroshi; Weatherhead, Jason G.; Akiyama, Toshiyuki; Taishi, Teruhiko; Taoda, Yoshiyuki; Mikamiyama-Iwata, Minako; Murai, Hitoshi; Kiyama, Ryuichi; Fuji, Masahiro; Tanimoto, Norihiko; Yoshinaga, Tomokazu; Seki, Takahiro; Kobayashi, Masanori; Sato, Akihiko; Garvey, Edward P.; Fujiwara, Tamio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1124-1135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This work is a continuation of our initial discovery of a potent monocyclic carbamoyl pyridone human immunodeficiency virus type-1 (HIV-1) integrase inhibitor that displayed favorable antiviral and pharmacokinetic properties.  We report herein a series of bicyclic carbamoylpyridone (I; satd. and unsatd. bridge; R = e.g., alkoxyethyl) analogs to address conformational issues from our initial SAR studies.  This modification of the core unit succeeded to deliver low nanomolar potency in std. antiviral assays.  An addnl. hydroxyl substituent on the bicyclic scaffold provides remarkable improvement of antiviral efficacies against clin. relevant resistant viruses.  These findings led to addnl. cyclic tethering of the naked hydroxyl group resulting in tricyclic carbamoyl pyridone inhibitors (II; Y = O, N; n = 0-2; R' = e.g., alkyl, Ac, methanesulfonyl) to address remaining issues and deliver potential clin. candidates.  The tricyclic carbamoyl pyridone derivs. described herein served as the immediate leads in mols. to the next generation integrase inhibitor dolutegravir which is currently in late stage clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQLpPj5UB1TbVg90H21EOLACvtfcHk0ljCfFKH8HOl_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXntVeltA%253D%253D&md5=4bfe919f8fdbbc84e346e3f01e084b70</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm301550c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301550c%26sid%3Dliteratum%253Aachs%26aulast%3DKawasuji%26aufirst%3DT.%26aulast%3DJohns%26aufirst%3DB.%2BA.%26aulast%3DYoshida%26aufirst%3DH.%26aulast%3DWeatherhead%26aufirst%3DJ.%2BG.%26aulast%3DAkiyama%26aufirst%3DT.%26aulast%3DTaishi%26aufirst%3DT.%26aulast%3DTaoda%26aufirst%3DY.%26aulast%3DMikamiyama-Iwata%26aufirst%3DM.%26aulast%3DMurai%26aufirst%3DH.%26aulast%3DKiyama%26aufirst%3DR.%26aulast%3DFuji%26aufirst%3DM.%26aulast%3DTanimoto%26aufirst%3DN.%26aulast%3DYoshinaga%26aufirst%3DT.%26aulast%3DSeki%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DM.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DGarvey%26aufirst%3DE.%2BP.%26aulast%3DFujiwara%26aufirst%3DT.%26atitle%3DCarbamoyl%2520pyridone%2520HIV-1%2520integrase%2520inhibitors.%25202.%2520Bi-%2520and%2520tricyclic%2520derivatives%2520result%2520in%2520superior%2520antiviral%2520and%2520pharmacokinetic%2520profiles%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1124%26epage%3D1135%26doi%3D10.1021%2Fjm301550c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Raheem, I. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walji, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abeywickrema, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbe, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennet, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clas, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dubost, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Embrey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grobler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hafey, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartingh, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pajkovic, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rada, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rearden, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreier, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sisko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steele, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Truchon, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, P. J.</span><span> </span><span class="NLM_article-title">Discovery of 2-pyridinone aminals: A prodrug strategy to advance a second generation of HIV-1 integrase strand transfer inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8154</span><span class="NLM_x">–</span> <span class="NLM_lpage">8165</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01037</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01037" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8154-8165&author=I.+T.+Raheemauthor=A.+M.+Waljiauthor=D.+Kleinauthor=J.+M.+Sandersauthor=D.+A.+Powellauthor=P.+Abeywickremaauthor=G.+Barbeauthor=A.+Bennetauthor=S.+D.+Clasauthor=D.+Dubostauthor=M.+Embreyauthor=J.+Groblerauthor=M.+J.+Hafeyauthor=T.+J.+Hartinghauthor=D.+J.+Hazudaauthor=M.+D.+Millerauthor=K.+P.+Mooreauthor=N.+Pajkovicauthor=S.+Patelauthor=V.+Radaauthor=P.+Reardenauthor=J.+D.+Schreierauthor=J.+Siskoauthor=T.+G.+Steeleauthor=J.+F.+Truchonauthor=J.+Waiauthor=M.+Xuauthor=P.+J.+Coleman&title=Discovery+of+2-pyridinone+aminals%3A+A+prodrug+strategy+to+advance+a+second+generation+of+HIV-1+integrase+strand+transfer+inhibitors&doi=10.1021%2Facs.jmedchem.5b01037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 2-Pyridinone Aminals: A Prodrug Strategy to Advance a Second Generation of HIV-1 Integrase Strand Transfer Inhibitors</span></div><div class="casAuthors">Raheem, Izzat T.; Walji, Abbas M.; Klein, Daniel; Sanders, John M.; Powell, David A.; Abeywickrema, Pravien; Barbe, Guillaume; Bennet, Amrith; Childers, Karla; Christensen, Melodie; Clas, Sophie-Dorothee; Dubost, David; Embrey, Mark; Grobler, Jay; Hafey, Michael J.; Hartingh, Timothy J.; Hazuda, Daria J.; Kuethe, Jeffrey T.; Dunn, Jamie McCabe; Miller, Michael D.; Moore, Keith P.; Nolting, Andrew; Pajkovic, Natasa; Patel, Sangita; Peng, Zuihui; Rada, Vanessa; Rearden, Paul; Schreier, John D.; Sisko, John; Steele, Thomas G.; Truchon, Jean-Francois; Wai, John; Xu, Min; Coleman, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8154-8165</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The search for new mol. constructs that resemble the crit. two-metal binding pharmacophore required for HIV integrase strand transfer inhibition represents a vibrant area of research within drug discovery.  Here the authors present the discovery of a new class of HIV integrase strand transfer inhibitors based on the 2-pyridinone core of MK-0536.  These efforts led to the identification of two lead compds. I and II with excellent antiviral activity and preclin. pharmacokinetic profiles to support a once-daily human dose prediction.  Dose escalating PK studies in dog revealed significant issues with limited oral absorption and required an innovative prodrug strategy to enhance the high-dose plasma exposures of the parent mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGEsuT4C8WELVg90H21EOLACvtfcHk0ljKNFLj1legiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersrvJ&md5=bec0c2f7710a1e4e8cc3a502067a2236</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01037%26sid%3Dliteratum%253Aachs%26aulast%3DRaheem%26aufirst%3DI.%2BT.%26aulast%3DWalji%26aufirst%3DA.%2BM.%26aulast%3DKlein%26aufirst%3DD.%26aulast%3DSanders%26aufirst%3DJ.%2BM.%26aulast%3DPowell%26aufirst%3DD.%2BA.%26aulast%3DAbeywickrema%26aufirst%3DP.%26aulast%3DBarbe%26aufirst%3DG.%26aulast%3DBennet%26aufirst%3DA.%26aulast%3DClas%26aufirst%3DS.%2BD.%26aulast%3DDubost%26aufirst%3DD.%26aulast%3DEmbrey%26aufirst%3DM.%26aulast%3DGrobler%26aufirst%3DJ.%26aulast%3DHafey%26aufirst%3DM.%2BJ.%26aulast%3DHartingh%26aufirst%3DT.%2BJ.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DMoore%26aufirst%3DK.%2BP.%26aulast%3DPajkovic%26aufirst%3DN.%26aulast%3DPatel%26aufirst%3DS.%26aulast%3DRada%26aufirst%3DV.%26aulast%3DRearden%26aufirst%3DP.%26aulast%3DSchreier%26aufirst%3DJ.%2BD.%26aulast%3DSisko%26aufirst%3DJ.%26aulast%3DSteele%26aufirst%3DT.%2BG.%26aulast%3DTruchon%26aufirst%3DJ.%2BF.%26aulast%3DWai%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520of%25202-pyridinone%2520aminals%253A%2520A%2520prodrug%2520strategy%2520to%2520advance%2520a%2520second%2520generation%2520of%2520HIV-1%2520integrase%2520strand%2520transfer%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8154%26epage%3D8165%26doi%3D10.1021%2Facs.jmedchem.5b01037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maskell, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pye, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fesen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.</span><span> </span><span class="NLM_article-title">HIV-1 integrase strand transfer inhibitors with reduced susceptibility to drug resistant mutant integrases</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1074</span><span class="NLM_x">–</span> <span class="NLM_lpage">1081</span><span class="refDoi"> DOI: 10.1021/acschembio.5b00948</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00948" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1074-1081&author=X.+Z.+Zhaoauthor=S.+Smithauthor=D.+Maskellauthor=M.+Metifiotauthor=V.+Pyeauthor=K.+Fesenauthor=C.+Marchandauthor=Y.+Pommierauthor=P.+Cherepanovauthor=S.+Hughesauthor=T.+R.+Burke&title=HIV-1+integrase+strand+transfer+inhibitors+with+reduced+susceptibility+to+drug+resistant+mutant+integrases&doi=10.1021%2Facschembio.5b00948"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00948&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00948%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSmith%26aufirst%3DS.%26aulast%3DMaskell%26aufirst%3DD.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DPye%26aufirst%3DV.%26aulast%3DFesen%26aufirst%3DK.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DCherepanov%26aufirst%3DP.%26aulast%3DHughes%26aufirst%3DS.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3DHIV-1%2520integrase%2520strand%2520transfer%2520inhibitors%2520with%2520reduced%2520susceptibility%2520to%2520drug%2520resistant%2520mutant%2520integrases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1074%26epage%3D1081%26doi%3D10.1021%2Facschembio.5b00948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Wainberg, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesplede, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raffi, F.</span><span> </span><span class="NLM_article-title">What if HIV were unable to develop resistance against a new therapeutic agent?</span> <span class="citation_source-journal">BMC Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="refDoi"> DOI: 10.1186/1741-7015-11-249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1186%2F1741-7015-11-249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24267867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXntVais7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2013&pages=249&author=M.+A.+Wainbergauthor=T.+Mespledeauthor=F.+Raffi&title=What+if+HIV+were+unable+to+develop+resistance+against+a+new+therapeutic+agent%3F&doi=10.1186%2F1741-7015-11-249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">What if HIV were unable to develop resistance against a new therapeutic agent?</span></div><div class="casAuthors">Wainberg, Mark A.; Mesplede, Thibault; Raffi, Francois</div><div class="citationInfo"><span class="NLM_cas:title">BMC Medicine</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">249/1-249/6</span>CODEN:
                <span class="NLM_cas:coden">BMMECZ</span>;
        ISSN:<span class="NLM_cas:issn">1741-7015</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Background: The HIV integrase inhibitor, Dolutegravir (DTG), was recently approved by the Food and Drug Administration in the United States and is the only HIV drug that has not selected for resistance mutations in the clinic when used as part of first-line therapy.  This has led to speculation that DTG might have a higher genetic barrier for the development of drug resistance than the other compds. that are used in therapy.  Discussion: In this Opinion article, we speculate that this is due to greatly diminished replication capacity on the part of viruses that might become resistant to DTG when the drug is used in initial therapy and that DTG might be able to be used in HIV prevention and eradication strategies.  We also note that no compensatory mutation that might restore viral replication fitness to HIV in the aftermath of the appearance of a single drug resistance mutation has yet to be obsd.  Summary: DTG is a valuable addn. to the anti-HIV armamentarium of drugs and its long-term utility may potentially exceed its obvious use in treatment of HIV disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqiJYPc1BhMrVg90H21EOLACvtfcHk0ljKNFLj1legiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXntVais7w%253D&md5=7a72d67fa56360e1b82f567e44245582</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1186%2F1741-7015-11-249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1741-7015-11-249%26sid%3Dliteratum%253Aachs%26aulast%3DWainberg%26aufirst%3DM.%2BA.%26aulast%3DMesplede%26aufirst%3DT.%26aulast%3DRaffi%26aufirst%3DF.%26atitle%3DWhat%2520if%2520HIV%2520were%2520unable%2520to%2520develop%2520resistance%2520against%2520a%2520new%2520therapeutic%2520agent%253F%26jtitle%3DBMC%2520Med.%26date%3D2013%26volume%3D11%26spage%3D249%26doi%3D10.1186%2F1741-7015-11-249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Shah, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schafer, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DeSimone, J. A.,  Jr.</span><span> </span><span class="NLM_article-title">Dolutegravir: A new integrase strand transfer inhibitor for the treatment of HIV</span> <span class="citation_source-journal">Pharmacotherapy</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">506</span><span class="NLM_x">–</span> <span class="NLM_lpage">520</span><span class="refDoi"> DOI: 10.1002/phar.1386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1002%2Fphar.1386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24347095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvF2ktLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=506-520&author=B.+M.+Shahauthor=J.+J.+Schaferauthor=J.+A.+DeSimone&title=Dolutegravir%3A+A+new+integrase+strand+transfer+inhibitor+for+the+treatment+of+HIV&doi=10.1002%2Fphar.1386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dolutegravir: A New Integrase Strand Transfer Inhibitor for the Treatment of HIV</span></div><div class="casAuthors">Shah, Bhavik M.; Schafer, Jason J.; De Simone, Joseph A., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">506-520</span>CODEN:
                <span class="NLM_cas:coden">PHPYDQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-9114</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  The first two integrase strand transfer inhibitors (INSTIs) approved for treatment of patients infected with human immunodeficiency virus (HIV) were raltegravir and elvitegravir.  Both raltegravir and elvitegravir are now guideline-preferred agents as part of an antiretroviral regimen for treatment-naive patients.  However, raltegravir is dosed twice/day.  Elvitegravir is available in a single-tablet regimen and dosed once/day because it is administered with the pharmacokinetic booster cobicistat, a potent CYP3A4 inhibitor that can lead to clin. significant drug-drug interactions.  In addn., raltegravir and elvitegravir have a low genetic barrier to resistance and are assocd. with cross-resistance.  Dolutegravir is a new-generation INSTI administered once/day without a pharmacokinetic booster and can be coformulated in a single-tablet regimen.  Phase III studies have demonstrated the efficacy and safety of dolutegravir for treatment-naive and treatment-experienced patients.  Compared with other INSTIs, dolutegravir has a higher genetic barrier to resistance.  Dolutegravir was approved by the U.S. Food and Drug Administration in August 2013 and joins raltegravir and elvitegravir as guideline-preferred agents for the management for HIV-infected treatment-naive patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5J0O3fs7R87Vg90H21EOLACvtfcHk0ljKNFLj1legiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvF2ktLg%253D&md5=41c2865b578fae8326d39c8c0f94cbf7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fphar.1386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fphar.1386%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DB.%2BM.%26aulast%3DSchafer%26aufirst%3DJ.%2BJ.%26aulast%3DDeSimone%26aufirst%3DJ.%2BA.%26atitle%3DDolutegravir%253A%2520A%2520new%2520integrase%2520strand%2520transfer%2520inhibitor%2520for%2520the%2520treatment%2520of%2520HIV%26jtitle%3DPharmacotherapy%26date%3D2014%26volume%3D34%26spage%3D506%26epage%3D520%26doi%3D10.1002%2Fphar.1386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Malet, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gimferrer Arriaga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Artese, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parrotta, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alcaro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delelis, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tmeizeh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katlama, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valantin, M.-A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceccherini-Silberstein, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Calvez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcelin, A.-G.</span><span> </span><span class="NLM_article-title">New raltegravir resistance pathways induce broad cross-resistance to all currently used integrase inhibitors</span> <span class="citation_source-journal">J. Antimicrob. Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">2118</span><span class="NLM_x">–</span> <span class="NLM_lpage">2122</span><span class="refDoi"> DOI: 10.1093/jac/dku095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1093%2Fjac%2Fdku095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=24710029" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1SqsbfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2014&pages=2118-2122&author=I.+Maletauthor=L.+Gimferrer+Arriagaauthor=A.+Arteseauthor=G.+Costaauthor=L.+Parrottaauthor=S.+Alcaroauthor=O.+Delelisauthor=A.+Tmeizehauthor=C.+Katlamaauthor=M.-A.+Valantinauthor=F.+Ceccherini-Silbersteinauthor=V.+Calvezauthor=A.-G.+Marcelin&title=New+raltegravir+resistance+pathways+induce+broad+cross-resistance+to+all+currently+used+integrase+inhibitors&doi=10.1093%2Fjac%2Fdku095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">New raltegravir resistance pathways induce broad cross-resistance to all currently used integrase inhibitors</span></div><div class="casAuthors">Malet, Isabelle; Gimferrer Arriaga, Laura; Artese, Anna; Costa, Giosue; Parrotta, Lucia; Alcaro, Stefano; Delelis, Olivier; Tmeizeh, Ahmed; Katlama, Christine; Valantin, Marc-Antoine; Ceccherini-Silberstein, Francesca; Calvez, Vincent; Marcelin, Anne-Genevieve</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2118-2122</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The possibility of replacing raltegravir or elvitegravir with dolutegravir in heavily treatment-experienced patients failing on raltegravir/elvitegravir has been evaluated in VIKING trials.  All studied patients failed by the most common pathways, Y143, Q148 and N155, and dolutegravir demonstrated efficacy except for Q148 viruses.  The aim of this study was to explore, in the same way, the behavior of dolutegravir in comparison with raltegravir and elvitegravir against the atypical resistance integrase profiles, G118R and F121Y, described in HIV-1 patients failing on raltegravir therapy.  The behavior of integrases with mutations G118R and F121Y towards raltegravir, elvitegravir and dolutegravir was analyzed by evaluating phenotypic susceptibility and by means of in silico techniques (investigating binding affinities and the stabilization of the inhibitors in terms of their hydrogen bond network).  Results The phenotypic anal. of G118R and F121Y showed high resistance to raltegravir, elvitegravir and dolutegravir with a fold change >100 when the clin. derived integrase was used, and resistance was also seen when mutations were tested alone in an NL43 backbone, but more often with a lower fold change.  In silico, results showed that G118R and F121Y enzymes were assocd. with reduced binding affinities to each of the inhibitors and with a decreased no. of hydrogen bonds compared with the wild-type complexes.  This study showed that G118R and F121Y mutations, rarely described in patients failing on raltegravir, induced broad cross-resistance to all currently used integrase inhibitors.  These results are in accordance with our thermodn. and geometric anal. indicating decreased stability compared with the wild-type complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5AjEYyjsbn7Vg90H21EOLACvtfcHk0lhwFocZCa2m0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1SqsbfM&md5=376a841a2a93f8dea573f89874c12c9b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdku095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdku095%26sid%3Dliteratum%253Aachs%26aulast%3DMalet%26aufirst%3DI.%26aulast%3DGimferrer%2BArriaga%26aufirst%3DL.%26aulast%3DArtese%26aufirst%3DA.%26aulast%3DCosta%26aufirst%3DG.%26aulast%3DParrotta%26aufirst%3DL.%26aulast%3DAlcaro%26aufirst%3DS.%26aulast%3DDelelis%26aufirst%3DO.%26aulast%3DTmeizeh%26aufirst%3DA.%26aulast%3DKatlama%26aufirst%3DC.%26aulast%3DValantin%26aufirst%3DM.-A.%26aulast%3DCeccherini-Silberstein%26aufirst%3DF.%26aulast%3DCalvez%26aufirst%3DV.%26aulast%3DMarcelin%26aufirst%3DA.-G.%26atitle%3DNew%2520raltegravir%2520resistance%2520pathways%2520induce%2520broad%2520cross-resistance%2520to%2520all%2520currently%2520used%2520integrase%2520inhibitors%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2014%26volume%3D69%26spage%3D2118%26epage%3D2122%26doi%3D10.1093%2Fjac%2Fdku095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">King, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prabu-Jeyabalan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nalivaika, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiffer, C. A.</span><span> </span><span class="NLM_article-title">Combating susceptibility to drug resistance: Lessons from HIV-1 protease</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1333</span><span class="NLM_x">–</span> <span class="NLM_lpage">1338</span><span class="refDoi"> DOI: 10.1016/j.chembiol.2004.08.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1016%2Fj.chembiol.2004.08.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=15489160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2004&pages=1333-1338&author=N.+M.+Kingauthor=M.+Prabu-Jeyabalanauthor=E.+A.+Nalivaikaauthor=C.+A.+Schiffer&title=Combating+susceptibility+to+drug+resistance%3A+Lessons+from+HIV-1+protease&doi=10.1016%2Fj.chembiol.2004.08.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Combating Susceptibility to Drug Resistance Lessons from HIV-1 Protease</span></div><div class="casAuthors">King, Nancy M.; Prabu-Jeyabalan, Moses; Nalivaika, Ellen A.; Schiffer, Celia A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1333-1338</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Drug resistance is a major obstacle in modern medicine.  However, resistance is rarely considered in drug development and may inadvertently be facilitated, as many designed inhibitors contact residues that can mutate to confer resistance, without significantly impairing function.  Contemporary drug design often ignores the detailed at. basis for function and primarily focuses on disrupting the target's activity, which is necessary but not sufficient for developing a robust drug.  In this study, we examine the impact of drug-resistant mutations in HIV-1 protease on substrate recognition and demonstrate that most primary active site mutations do not extensively contact substrates, but are crit. to inhibitor binding.  We propose a general, structure-based strategy to reduce the probability of drug resistance by designing inhibitors that interact only with those residues that are essential for function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbsk2AyTcCmbVg90H21EOLACvtfcHk0lhwFocZCa2m0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXos1Ciu78%253D&md5=701599b15cf9ecc2c026ad4eac650a90</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2004.08.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2004.08.010%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DN.%2BM.%26aulast%3DPrabu-Jeyabalan%26aufirst%3DM.%26aulast%3DNalivaika%26aufirst%3DE.%2BA.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26atitle%3DCombating%2520susceptibility%2520to%2520drug%2520resistance%253A%2520Lessons%2520from%2520HIV-1%2520protease%26jtitle%3DChem.%2520Biol.%26date%3D2004%26volume%3D11%26spage%3D1333%26epage%3D1338%26doi%3D10.1016%2Fj.chembiol.2004.08.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Hirokawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harada, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshida, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, S.</span><span> </span><span class="NLM_article-title">Synthesis and structure–affinity relationships of novel N-(1-ethyl-4-methylhexahydro-1,4-diazepin-6-yl)pyridine-3-carboxamides with potent serotonin 5-HT3 and dopamine D2 receptor antagonistic activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">702</span><span class="NLM_x">–</span> <span class="NLM_lpage">715</span><span class="refDoi"> DOI: 10.1021/jm020270n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020270n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=702-715&author=Y.+Hirokawaauthor=I.+Fujiwaraauthor=K.+Suzukiauthor=H.+Haradaauthor=T.+Yoshikawaauthor=N.+Yoshidaauthor=S.+Kato&title=Synthesis+and+structure%E2%80%93affinity+relationships+of+novel+N-%281-ethyl-4-methylhexahydro-1%2C4-diazepin-6-yl%29pyridine-3-carboxamides+with+potent+serotonin+5-HT3+and+dopamine+D2+receptor+antagonistic+activity&doi=10.1021%2Fjm020270n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm020270n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020270n%26sid%3Dliteratum%253Aachs%26aulast%3DHirokawa%26aufirst%3DY.%26aulast%3DFujiwara%26aufirst%3DI.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DHarada%26aufirst%3DH.%26aulast%3DYoshikawa%26aufirst%3DT.%26aulast%3DYoshida%26aufirst%3DN.%26aulast%3DKato%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520structure%25E2%2580%2593affinity%2520relationships%2520of%2520novel%2520N-%25281-ethyl-4-methylhexahydro-1%252C4-diazepin-6-yl%2529pyridine-3-carboxamides%2520with%2520potent%2520serotonin%25205-HT3%2520and%2520dopamine%2520D2%2520receptor%2520antagonistic%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D702%26epage%3D715%26doi%3D10.1021%2Fjm020270n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">LaFemina, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeGrow, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hastings, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emini, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span> </span><span class="NLM_article-title">Inhibition of human immunodeficiency virus integrase by bis-catechols</span> <span class="citation_source-journal">Antimicrob. Agents Chemother.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">320</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1128/AAC.39.2.320</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1128%2FAAC.39.2.320" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=320-324&author=R.+L.+LaFeminaauthor=P.+L.+Grahamauthor=K.+LeGrowauthor=J.+C.+Hastingsauthor=A.+Wolfeauthor=S.+D.+Youngauthor=E.+A.+Eminiauthor=D.+J.+Hazuda&title=Inhibition+of+human+immunodeficiency+virus+integrase+by+bis-catechols&doi=10.1128%2FAAC.39.2.320"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1128%2FAAC.39.2.320&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.39.2.320%26sid%3Dliteratum%253Aachs%26aulast%3DLaFemina%26aufirst%3DR.%2BL.%26aulast%3DGraham%26aufirst%3DP.%2BL.%26aulast%3DLeGrow%26aufirst%3DK.%26aulast%3DHastings%26aufirst%3DJ.%2BC.%26aulast%3DWolfe%26aufirst%3DA.%26aulast%3DYoung%26aufirst%3DS.%2BD.%26aulast%3DEmini%26aufirst%3DE.%2BA.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26atitle%3DInhibition%2520of%2520human%2520immunodeficiency%2520virus%2520integrase%2520by%2520bis-catechols%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1995%26volume%3D39%26spage%3D320%26epage%3D324%26doi%3D10.1128%2FAAC.39.2.320" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Charpentier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karmochkine, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laureillard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tisserand, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belec, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weiss, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Si-Mohamed, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piketty, C.</span><span> </span><span class="NLM_article-title">Drug resistance profiles for the HIV integrase gene in patients failing raltegravir salvage therapy</span> <span class="citation_source-journal">HIV Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">765</span><span class="NLM_x">–</span> <span class="NLM_lpage">70</span><span class="refDoi"> DOI: 10.1111/j.1468-1293.2008.00628.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1111%2Fj.1468-1293.2008.00628.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=18651855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVemtr%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=765-70&author=C.+Charpentierauthor=M.+Karmochkineauthor=D.+Laureillardauthor=P.+Tisserandauthor=L.+Belecauthor=L.+Weissauthor=A.+Si-Mohamedauthor=C.+Piketty&title=Drug+resistance+profiles+for+the+HIV+integrase+gene+in+patients+failing+raltegravir+salvage+therapy&doi=10.1111%2Fj.1468-1293.2008.00628.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Drug resistance profiles for the HIV integrase gene in patients failing raltegravir salvage therapy</span></div><div class="casAuthors">Charpentier, C.; Karmochkine, M.; Laureillard, D.; Tisserand, P.; Belec, L.; Weiss, L.; Si-Mohamed, A.; Piketty, C.</div><div class="citationInfo"><span class="NLM_cas:title">HIV Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">765-770</span>CODEN:
                <span class="NLM_cas:coden">HMIEAB</span>;
        ISSN:<span class="NLM_cas:issn">1464-2662</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Objectives: Recent data showed the selection of mutations in the integrase gene, mainly involving position 148 or 155, in patients displaying virol. failure (VF) on raltegravir (RAL) therapy.  Here, we describe the development of RAL resistance, in both plasmatic and cellular compartments, in three heavily pretreated HIV-infected patients failing RAL-contg. regimens.  Methods: Three of 17 patients receiving RAL displayed VF.  The entire integrase gene, isolated from plasma and peripheral blood mononuclear cells (PBMC), was sequenced.  A clonal anal. was performed in one patient at the time of VF.  Results: At the time of VF, RAL-resistance mutations were selected: (i) Q148R in patients 1 and 3; (ii) T66A and E92Q in patient 2.  A gradual accumulation of new mutations was obsd. in all patients, including G140S, Q148H and N155H in patient 1, L74I in patient 2, and G140S in patient 3.  Clonal anal. showed the coexistence, in patient 1, of the two common resistance pathways (mutations Q148R/H and N155H) found in distinct quasi-species.  In addn., RAL-resistance mutations were detected in HIV DNA in all patients.  Conclusions: Having rapidly established, resistance to RAL evolves and diversifies, and is likely to impact the efficacy of subsequently used second-generation integrase inhibitors.  Moreover, RAL-resistance mutations can be archived early in PBMC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAzqme5X5_G7Vg90H21EOLACvtfcHk0lhj2jomtfiVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVemtr%252FJ&md5=ae94b8c464b80ebb8dd2795d87dc225f</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1111%2Fj.1468-1293.2008.00628.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1468-1293.2008.00628.x%26sid%3Dliteratum%253Aachs%26aulast%3DCharpentier%26aufirst%3DC.%26aulast%3DKarmochkine%26aufirst%3DM.%26aulast%3DLaureillard%26aufirst%3DD.%26aulast%3DTisserand%26aufirst%3DP.%26aulast%3DBelec%26aufirst%3DL.%26aulast%3DWeiss%26aufirst%3DL.%26aulast%3DSi-Mohamed%26aufirst%3DA.%26aulast%3DPiketty%26aufirst%3DC.%26atitle%3DDrug%2520resistance%2520profiles%2520for%2520the%2520HIV%2520integrase%2520gene%2520in%2520patients%2520failing%2520raltegravir%2520salvage%2520therapy%26jtitle%3DHIV%2520Med.%26date%3D2008%26volume%3D9%26spage%3D765%26epage%3D70%26doi%3D10.1111%2Fj.1468-1293.2008.00628.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Cooper, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steigbigel, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gatell, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rockstroh, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katlama, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeni, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazzarin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clotet, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eron, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schechter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Markowitz, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loutfy, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lennox, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Killar, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilde, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strohmaier, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meibohm, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazuda, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nessly, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiNubile, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Isaacs, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teppler, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, B.-Y.</span><span> </span><span class="NLM_article-title">Subgroup and resistance analyses of raltegravir for resistant HIV-1 infection</span> <span class="citation_source-journal">N. Engl. J. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">359</span><span class="NLM_x">, </span> <span class="NLM_fpage">355</span><span class="NLM_x">–</span> <span class="NLM_lpage">365</span><span class="refDoi"> DOI: 10.1056/NEJMoa0708978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1056%2FNEJMoa0708978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=18650513" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovFSqtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2008&pages=355-365&author=D.+A.+Cooperauthor=R.+T.+Steigbigelauthor=J.+M.+Gatellauthor=J.+K.+Rockstrohauthor=C.+Katlamaauthor=P.+Yeniauthor=A.+Lazzarinauthor=B.+Clotetauthor=P.+N.+Kumarauthor=J.+E.+Eronauthor=M.+Schechterauthor=M.+Markowitzauthor=M.+R.+Loutfyauthor=J.+L.+Lennoxauthor=J.+Zhaoauthor=J.+Chenauthor=D.+M.+Ryanauthor=R.+R.+Rhodesauthor=J.+A.+Killarauthor=L.+R.+Gildeauthor=K.+M.+Strohmaierauthor=A.+R.+Meibohmauthor=M.+D.+Millerauthor=D.+J.+Hazudaauthor=M.+L.+Nesslyauthor=M.+J.+DiNubileauthor=R.+D.+Isaacsauthor=H.+Tepplerauthor=B.-Y.+Nguyen&title=Subgroup+and+resistance+analyses+of+raltegravir+for+resistant+HIV-1+infection&doi=10.1056%2FNEJMoa0708978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Subgroup and resistance analyses of raltegravir for resistant HIV-1 infection</span></div><div class="casAuthors">Cooper, David A.; Steigbigel, Roy T.; Gatell, Jose M.; Rockstroh, Jurgen K.; Katlama, Christine; Yeni, Patrick; Lazzarin, Adriano; Clotet, Bonaventura; Kumar, Princy N.; Eron, Joseph E.; Schechter, Mauro; Markowitz, Martin; Loutfy, Mona R.; Lennox, Jeffrey L.; Zhao, Jing; Chen, Joshua; Ryan, Desmond M.; Rhodes, Rand R.; Killar, John A.; Gilde, Lucinda R.; Strohmaier, Kim M.; Meibohm, Anne R.; Miller, Michael D.; Hazuda, Daria J.; Nessly, Michael L.; DiNubile, Mark J.; Isaacs, Robin D.; Teppler, Hedy; Nguyen, Bach-Yen</div><div class="citationInfo"><span class="NLM_cas:title">New England Journal of Medicine</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">355-365</span>CODEN:
                <span class="NLM_cas:coden">NEJMAG</span>;
        ISSN:<span class="NLM_cas:issn">0028-4793</span>.
    
            (<span class="NLM_cas:orgname">Massachusetts Medical Society</span>)
        </div><div class="casAbstract">We evaluated the efficacy of raltegravir and the development of viral resistance in two identical trials involving patients who were infected with human immunodeficiency virus type 1 (HIV-1) with triple-class drug resistance and in whom antiretroviral therapy had failed.  We conducted subgroup analyses of the data from week 48 in both studies according to baseline prognostic factors.  Genotyping of the integrase gene was performed in raltegravir recipients who had virol. failure.  Virol. responses to raltegravir were consistently superior to responses to placebo, regardless of the baseline values of HIV-1 RNA level; CD4 cell count; genotypic or phenotypic sensitivity score; use or nonuse of darunavir, enfuvirtide, or both in optimized background therapy; or demog. characteristics.  Among patients in the two studies combined who were using both enfuvirtide and darunavir for the first time, HIV-1 RNA levels of less than 50 copies per mL were achieved in 89% of raltegravir recipients and 68% of placebo recipients.  HIV-1 RNA levels of less than 50 copies per mL were achieved in 69% and 80% of the raltegravir recipients and in 47% and 57% of the placebo recipients using either darunavir or enfuvirtide for the first time, resp.  At 48 wk, 105 of the 462 raltegravir recipients (23%) had virol. failure.  Genotyping was performed in 94 raltegravir recipients with virol. failure.  Integrase mutations known to be assocd. with phenotypic resistance to raltegravir arose during treatment in 64 patients (68%).  Forty-eight of these 64 patients (75%) had two or more resistance-assocd. mutations.  When combined with an optimized background regimen in both studies, a consistently favorable treatment effect of raltegravir over placebo was shown in clin. relevant subgroups of patients, including those with baseline characteristics that typically predict a poor response to antiretroviral therapy: a high HIV-1 RNA level, low CD4 cell count, and low genotypic or phenotypic sensitivity score.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdQbxOYiV4b7Vg90H21EOLACvtfcHk0lhj2jomtfiVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovFSqtb8%253D&md5=cd4bc42edc03c66485d399ba620477cc</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1056%2FNEJMoa0708978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1056%252FNEJMoa0708978%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DD.%2BA.%26aulast%3DSteigbigel%26aufirst%3DR.%2BT.%26aulast%3DGatell%26aufirst%3DJ.%2BM.%26aulast%3DRockstroh%26aufirst%3DJ.%2BK.%26aulast%3DKatlama%26aufirst%3DC.%26aulast%3DYeni%26aufirst%3DP.%26aulast%3DLazzarin%26aufirst%3DA.%26aulast%3DClotet%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DP.%2BN.%26aulast%3DEron%26aufirst%3DJ.%2BE.%26aulast%3DSchechter%26aufirst%3DM.%26aulast%3DMarkowitz%26aufirst%3DM.%26aulast%3DLoutfy%26aufirst%3DM.%2BR.%26aulast%3DLennox%26aufirst%3DJ.%2BL.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DRyan%26aufirst%3DD.%2BM.%26aulast%3DRhodes%26aufirst%3DR.%2BR.%26aulast%3DKillar%26aufirst%3DJ.%2BA.%26aulast%3DGilde%26aufirst%3DL.%2BR.%26aulast%3DStrohmaier%26aufirst%3DK.%2BM.%26aulast%3DMeibohm%26aufirst%3DA.%2BR.%26aulast%3DMiller%26aufirst%3DM.%2BD.%26aulast%3DHazuda%26aufirst%3DD.%2BJ.%26aulast%3DNessly%26aufirst%3DM.%2BL.%26aulast%3DDiNubile%26aufirst%3DM.%2BJ.%26aulast%3DIsaacs%26aufirst%3DR.%2BD.%26aulast%3DTeppler%26aufirst%3DH.%26aulast%3DNguyen%26aufirst%3DB.-Y.%26atitle%3DSubgroup%2520and%2520resistance%2520analyses%2520of%2520raltegravir%2520for%2520resistant%2520HIV-1%2520infection%26jtitle%3DN.%2520Engl.%2520J.%2520Med.%26date%3D2008%26volume%3D359%26spage%3D355%26epage%3D365%26doi%3D10.1056%2FNEJMoa0708978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Shen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altman, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nalam, M. N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rana, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schiffer, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tidor, B.</span><span> </span><span class="NLM_article-title">Testing the substrate-envelope hypothesis with designed pairs of compounds</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">2433</span><span class="NLM_x">–</span> <span class="NLM_lpage">2441</span><span class="refDoi"> DOI: 10.1021/cb400468c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400468c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1yiur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2433-2441&author=Y.+Shenauthor=M.+D.+Altmanauthor=A.+Aliauthor=M.+N.+L.+Nalamauthor=H.+Caoauthor=T.+M.+Ranaauthor=C.+A.+Schifferauthor=B.+Tidor&title=Testing+the+substrate-envelope+hypothesis+with+designed+pairs+of+compounds&doi=10.1021%2Fcb400468c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Testing the substrate-envelope hypothesis with designed pairs of compounds</span></div><div class="casAuthors">Shen, Yang; Altman, Michael D.; Ali, Akbar; Nalam, Madhavi N. L.; Cao, Hong; Rana, Tariq M.; Schiffer, Celia A.; Tidor, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2433-2441</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acquired resistance to therapeutic agents is a significant barrier to the development of clin. effective treatments for diseases in which evolution occurs on clin. time scales, frequently arising from target mutations.  We previously reported a general strategy to design effective inhibitors for rapidly mutating enzyme targets, which we demonstrated for HIV-1 protease inhibition.  Specifically, we developed a computational inverse design procedure with the added constraint that designed inhibitors bind entirely inside the substrate envelope, a consensus vol. occupied by natural substrates.  The rationale for the substrate-envelope constraint is that it prevents designed inhibitors from making interactions beyond those required by substrates and thus limits the availability of mutations tolerated by substrates but not by designed inhibitors.  The strategy resulted in subnanomolar inhibitors that bind robustly across a clin. derived panel of drug-resistant variants.  To further test the substrate-envelope hypothesis, here we have designed, synthesized, and assayed derivs. of our original compds. that are larger and extend outside the substrate envelope.  Our designs resulted in pairs of compds. that are very similar to one another, but one respects and one violates the substrate envelope.  The envelope-respecting inhibitor demonstrates robust binding across a panel of drug-resistant protease variants, whereas the envelope-violating one binds tightly to wild type but loses affinity to at least one variant.  This study provides strong support for the substrate-envelope hypothesis as a design strategy for inhibitors that reduce susceptibility to resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxEZ752OIVobVg90H21EOLACvtfcHk0lhj2jomtfiVsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1yiur%252FE&md5=8febc495df52db0dd9463e25a2f5284d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fcb400468c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400468c%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAltman%26aufirst%3DM.%2BD.%26aulast%3DAli%26aufirst%3DA.%26aulast%3DNalam%26aufirst%3DM.%2BN.%2BL.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DRana%26aufirst%3DT.%2BM.%26aulast%3DSchiffer%26aufirst%3DC.%2BA.%26aulast%3DTidor%26aufirst%3DB.%26atitle%3DTesting%2520the%2520substrate-envelope%2520hypothesis%2520with%2520designed%2520pairs%2520of%2520compounds%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2433%26epage%3D2441%26doi%3D10.1021%2Fcb400468c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Das, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauman, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frenkel, Y. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewi, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shatkin, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arnold, E.</span><span> </span><span class="NLM_article-title">High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: Strategic flexibility explains potency against resistance mutations</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">1466</span><span class="NLM_x">–</span> <span class="NLM_lpage">1471</span><span class="refDoi"> DOI: 10.1073/pnas.0711209105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1073%2Fpnas.0711209105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=18230722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=1466-1471&author=K.+Dasauthor=J.+D.+Baumanauthor=A.+D.+Clarkauthor=Y.+V.+Frenkelauthor=P.+J.+Lewiauthor=A.+J.+Shatkinauthor=S.+H.+Hughesauthor=E.+Arnold&title=High-resolution+structures+of+HIV-1+reverse+transcriptase%2FTMC278+complexes%3A+Strategic+flexibility+explains+potency+against+resistance+mutations&doi=10.1073%2Fpnas.0711209105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: strategic flexibility explains potency against resistance mutations</span></div><div class="casAuthors">Das, Kalyan; Bauman, Joseph D.; Clark, Arthur D.; Frenkel, Yulia V.; Lewi, Paul J.; Shatkin, Aaron J.; Hughes, Stephen H.; Arnold, Eddy</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1466-1471</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">TMC278 is a diarylpyrimidine (DAPY) nonnucleoside reverse transcriptase inhibitor (NNRTI) that is highly effective in treating wild-type and drug-resistant HIV-1 infections in clin. trials at relatively low doses (∼ 25-75 mg/day).  We have detd. the structure of wild-type HIV-1 RT complexed with TMC278 at 1.8 Å resoln., using an RT crystal form engineered by systematic RT mutagenesis.  This high-resoln. structure reveals that the cyanovinyl group of TMC278 is positioned in a hydrophobic tunnel connecting the NNRTI-binding pocket to the nucleic acid-binding cleft.  The crystal structures of TMC278 in complexes with the double mutant K103N/Y181C (2.1 Å) and L100I/K103N HIV-1 RTs (2.9 Å) demonstrated that TMC278 adapts to bind mutant RTs.  In the K103N/Y181C RT/TMC278 structure, loss of the arom. ring interaction caused by the Y181C mutation is counterbalanced by interactions between the cyanovinyl group of TMC278 and the arom. side chain of Y183, which is facilitated by an ∼ 1.5 Å shift of the conserved Y183MDD motif.  In the L1001/K103N RT/ TMC278 structure, the binding mode of TMC278 is significantly altered so that the drug conforms to changes in the binding pocket primarily caused by the L1001 mutation.  The flexible binding pocket acts as a mol. "shrink wrap" that makes a shape complementary to the optimized TMC278 in wild-type and drug-resistant forms of HIV-1 RT.  The crystal structures provide a better understanding of how the flexibility of an inhibitor can compensate for drug-resistance mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXRbleFGfFk7Vg90H21EOLACvtfcHk0lgzixay5-_jmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFCit74%253D&md5=5f2ea5e4d88e9822ceecee636e1e4bd1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711209105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711209105%26sid%3Dliteratum%253Aachs%26aulast%3DDas%26aufirst%3DK.%26aulast%3DBauman%26aufirst%3DJ.%2BD.%26aulast%3DClark%26aufirst%3DA.%2BD.%26aulast%3DFrenkel%26aufirst%3DY.%2BV.%26aulast%3DLewi%26aufirst%3DP.%2BJ.%26aulast%3DShatkin%26aufirst%3DA.%2BJ.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DArnold%26aufirst%3DE.%26atitle%3DHigh-resolution%2520structures%2520of%2520HIV-1%2520reverse%2520transcriptase%252FTMC278%2520complexes%253A%2520Strategic%2520flexibility%2520explains%2520potency%2520against%2520resistance%2520mutations%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D1466%26epage%3D1471%26doi%3D10.1073%2Fpnas.0711209105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Hirokawa, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horikawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, S.</span><span> </span><span class="NLM_article-title">An efficient synthesis of 5-bromo-2-methoxy-6methylaminopyridine-3-carboxylic acid</span> <span class="citation_source-journal">Chem. Pharm. Bull.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">1847</span><span class="NLM_x">–</span> <span class="NLM_lpage">1853</span><span class="refDoi"> DOI: 10.1248/cpb.48.1847</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1248%2Fcpb.48.1847" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2000&pages=1847-1853&author=Y.+Hirokawaauthor=T.+Horikawaauthor=S.+Kato&title=An+efficient+synthesis+of+5-bromo-2-methoxy-6methylaminopyridine-3-carboxylic+acid&doi=10.1248%2Fcpb.48.1847"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1248%2Fcpb.48.1847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1248%252Fcpb.48.1847%26sid%3Dliteratum%253Aachs%26aulast%3DHirokawa%26aufirst%3DY.%26aulast%3DHorikawa%26aufirst%3DT.%26aulast%3DKato%26aufirst%3DS.%26atitle%3DAn%2520efficient%2520synthesis%2520of%25205-bromo-2-methoxy-6methylaminopyridine-3-carboxylic%2520acid%26jtitle%3DChem.%2520Pharm.%2520Bull.%26date%3D2000%26volume%3D48%26spage%3D1847%26epage%3D1853%26doi%3D10.1248%2Fcpb.48.1847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddali, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hare, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherepanov, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span> </span><span class="NLM_article-title">Activities, crystal structures and molecular dynamics of dihydro-1H-isoindole derivatives, inhibitors of HIV-1 integrase</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">217</span><span class="refDoi"> DOI: 10.1021/cb300471n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb300471n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=209-217&author=M.+Metifiotauthor=K.+Maddaliauthor=B.+C.+Johnsonauthor=S.+Hareauthor=S.+J.+Smithauthor=X.+Z.+Zhaoauthor=C.+Marchandauthor=T.+R.+Burkeauthor=S.+H.+Hughesauthor=P.+Cherepanovauthor=Y.+Pommier&title=Activities%2C+crystal+structures+and+molecular+dynamics+of+dihydro-1H-isoindole+derivatives%2C+inhibitors+of+HIV-1+integrase&doi=10.1021%2Fcb300471n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fcb300471n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb300471n%26sid%3Dliteratum%253Aachs%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DMaddali%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DB.%2BC.%26aulast%3DHare%26aufirst%3DS.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DCherepanov%26aufirst%3DP.%26aulast%3DPommier%26aufirst%3DY.%26atitle%3DActivities%252C%2520crystal%2520structures%2520and%2520molecular%2520dynamics%2520of%2520dihydro-1H-isoindole%2520derivatives%252C%2520inhibitors%2520of%2520HIV-1%2520integrase%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D209%26epage%3D217%26doi%3D10.1021%2Fcb300471n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddali, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metifiot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.,  Jr.</span><span> </span><span class="NLM_article-title">Bicyclic hydroxy-1H-pyrrolopyridine-trione containing HIV-1 integrase inhibitors</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">79</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">165</span><span class="refDoi"> DOI: 10.1111/j.1747-0285.2011.01270.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1111%2Fj.1747-0285.2011.01270.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=22107736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2012&pages=157-165&author=X.+Z.+Zhaoauthor=K.+Maddaliauthor=M.+Metifiotauthor=S.+J.+Smithauthor=B.+C.+Vuauthor=C.+Marchandauthor=S.+H.+Hughesauthor=Y.+Pommierauthor=T.+R.+Burke&title=Bicyclic+hydroxy-1H-pyrrolopyridine-trione+containing+HIV-1+integrase+inhibitors&doi=10.1111%2Fj.1747-0285.2011.01270.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Bicyclic hydroxy-1H-pyrrolopyridine-trione containing HIV-1 integrase inhibitors</span></div><div class="casAuthors">Zhao, Xue Zhi; Maddali, Kasthuraiah; Metifiot, Mathieu; Smith, Steven J.; Vu, B. Christie; Marchand, Christophe; Hughes, Stephen H.; Pommier, Yves; Burke, Terrence R., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">157-165</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">HIV-1 integrase (IN) is a validated therapeutic target for the treatment of AIDS.  However, the emergence of resistance to raltegravir, the sole marketed FDA-approved IN inhibitor, emphasizes the need to develop second-generation inhibitors that retain efficacy against clin. relevant IN mutants.  We report herein bicyclic hydroxy-1H-pyrrolopyridinetriones as a new family of HIV-1 integrase inhibitors that were efficiently prepd. using a key "Pummerer cyclization deprotonation cycloaddn." cascade of imidosulfoxides.  In in vitro HIV-1 integrase assays, the analogs showed low micromolar inhibitory potencies with selectivity for strand transfer reactions as compared with 3'-processing inhibition.  A representative chlorofluorobenzyl hydroxypyrrolopyridinetrione retained most of its inhibitory potency against the three major raltegravir-resistant IN mutant enzymes, G140S/Q148H, Y143R, and N155H.  In antiviral assays employing viral vectors coding these IN mutants, the chlorofluorobenzyl hydroxypyrrolopyridinetrione was approx. 200- and 20-fold less affected than raltegravir against the G140S/Q148H and Y143R mutations, resp.  The representative chlorofluorobenzyl hydroxypyrrolopyridinetrione was affected approx. 10-fold less by the N155H mutation than raltegravir.  Hydroxypyrrolopyridinetriones are a novel structural class that may be further optimized to obtain HIV-1 integrase inhibitors capable of overcoming raltegravir resistance in HIV-1 strains possessing G140S/Q148H mutations in HIV-1 integrase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_-V1xazfn1LVg90H21EOLACvtfcHk0lgzixay5-_jmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWrtLY%253D&md5=467e940ff1b737e55486e8f578db6da0</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2011.01270.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2011.01270.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DMaddali%26aufirst%3DK.%26aulast%3DMetifiot%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DS.%2BJ.%26aulast%3DVu%26aufirst%3DB.%2BC.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3DBicyclic%2520hydroxy-1H-pyrrolopyridine-trione%2520containing%2520HIV-1%2520integrase%2520inhibitors%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2012%26volume%3D79%26spage%3D157%26epage%3D165%26doi%3D10.1111%2Fj.1747-0285.2011.01270.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Brachmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koenig, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Julius, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feldbruegge, M.</span><span> </span><span class="NLM_article-title">A reverse genetic approach for generating gene replacement mutants in Ustilago maydis</span> <span class="citation_source-journal">Mol. Genet. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">216</span><span class="NLM_x">–</span> <span class="NLM_lpage">226</span><span class="refDoi"> DOI: 10.1007/s00438-004-1047-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1007%2Fs00438-004-1047-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=15316769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD2cXns1eqsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=2004&pages=216-226&author=A.+Brachmannauthor=J.+Koenigauthor=C.+Juliusauthor=M.+Feldbruegge&title=A+reverse+genetic+approach+for+generating+gene+replacement+mutants+in+Ustilago+maydis&doi=10.1007%2Fs00438-004-1047-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">A reverse genetic approach for generating gene replacement mutants in Ustilago maydis</span></div><div class="casAuthors">Brachmann, A.; Koenig, J.; Julius, C.; Feldbruegge, M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Genetics and Genomics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">272</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">216-226</span>CODEN:
                <span class="NLM_cas:coden">MGGOAA</span>;
        ISSN:<span class="NLM_cas:issn">1617-4615</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">We describe a versatile strategy for generating gene replacement mutants in the phytopathogenic fungus Ustilago maydis.  The system includes the choice of 32 different insertion cassettes for genetic engineering purposes, such as gene disruption and more sophisticated insertions of reporter genes, heterologous promoters or combinations of the two.  PCR-amplified flanking sequences needed for homologous recombination are ligated to the resp. insertion cassettes via Sfi I sites.  As proof of principle we generated two replacement mutants in which the endogenous promoter of the pheromone gene mfa1 drives expression of the Green Fluorescent Protein gene (gfp).  Simultaneously, expression of the mfa1 ORF is controlled either by the carbon source-regulated crg1 promoter or the nitrogen source-regulated nar1 promoter.  In both cases gfp expression was pheromone-inducible and pheromone expression was only detected when the heterologous promoters were active.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqckY_KQd6m37Vg90H21EOLACvtfcHk0lgdf1jqgh4ssg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXns1eqsLg%253D&md5=ae76faf1e45e14ef570b2cf60e8d67a3</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1007%2Fs00438-004-1047-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00438-004-1047-z%26sid%3Dliteratum%253Aachs%26aulast%3DBrachmann%26aufirst%3DA.%26aulast%3DKoenig%26aufirst%3DJ.%26aulast%3DJulius%26aufirst%3DC.%26aulast%3DFeldbruegge%26aufirst%3DM.%26atitle%3DA%2520reverse%2520genetic%2520approach%2520for%2520generating%2520gene%2520replacement%2520mutants%2520in%2520Ustilago%2520maydis%26jtitle%3DMol.%2520Genet.%2520Genomics%26date%3D2004%26volume%3D272%26spage%3D216%26epage%3D226%26doi%3D10.1007%2Fs00438-004-1047-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Kabsch, W.</span><span> </span><span class="NLM_article-title">XDS</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">132</span><span class="refDoi"> DOI: 10.1107/S0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2FS0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgdf1jqgh4ssg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2FS0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Evans, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murshudov, G. N.</span><span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">1204</span><span class="NLM_x">–</span> <span class="NLM_lpage">1214</span><span class="refDoi"> DOI: 10.1107/S0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2FS0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgdf1jqgh4ssg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2FS0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Emsley, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cowtan, K.</span><span> </span><span class="NLM_article-title">Coot: Model-building tools for molecular graphics</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">60</span><span class="NLM_x">, </span> <span class="NLM_fpage">2126</span><span class="NLM_x">–</span> <span class="NLM_lpage">2132</span><span class="refDoi"> DOI: 10.1107/S0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+Model-building+tools+for+molecular+graphics&doi=10.1107%2FS0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lgdf1jqgh4ssg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520Model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2FS0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Adams, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Afonine, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunkóczi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Echols, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hung, L.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grosse Kunstleve, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCoy, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriarty, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oeffner, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Read, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terwilliger, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zwart, P. H.</span><span> </span><span class="NLM_article-title">The Phenix software for automated determination of macromolecular structures</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">94</span><span class="NLM_x">–</span> <span class="NLM_lpage">106</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2011.07.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1016%2Fj.ymeth.2011.07.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=21821126" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2011&pages=94-106&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.-W.+Hungauthor=S.+Jainauthor=G.+J.+Kapralauthor=R.+W.+Grosse+Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=The+Phenix+software+for+automated+determination+of+macromolecular+structures&doi=10.1016%2Fj.ymeth.2011.07.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Phenix software for automated determination of macromolecular structures</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li-Wei; Jain, Swati; Kapral, Gary J.; Grosse Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">94-106</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  X-ray crystallog. is a crit. tool in the study of biol. systems.  It is able to provide information that has been a prerequisite to understanding the fundamentals of life.  It is also a method that is central to the development of new therapeutics for human disease.  Significant time and effort are required to det. and optimize many macromol. structures because of the need for manual interpretation of complex numerical data, often using many different software packages, and the repeated use of interactive three-dimensional graphics.  The Phenix software package has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on automation.  This has required the development of new algorithms that minimize or eliminate subjective input in favor of built-in expert-systems knowledge, the automation of procedures that are traditionally performed by hand, and the development of a computational framework that allows a tight integration between the algorithms.  The application of automated methods is particularly appropriate in the field of structural proteomics, where high throughput is desired.  Features in Phenix for the automation of exptl. phasing with subsequent model building, mol. replacement, structure refinement and validation are described and examples given of running Phenix from both the command line and graphical user interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgjdX_7M-NcLVg90H21EOLACvtfcHk0lg3O_avUx6Hqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlaqt73F&md5=ba8f207cab9f3597dc0c6db885667a3f</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2011.07.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2011.07.005%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.-W.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse%2BKunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DThe%2520Phenix%2520software%2520for%2520automated%2520determination%2520of%2520macromolecular%2520structures%26jtitle%3DMethods%26date%3D2011%26volume%3D55%26spage%3D94%26epage%3D106%26doi%3D10.1016%2Fj.ymeth.2011.07.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Chen, V. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arendall, W. B.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Headd, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keedy, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Immormino, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapral, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, D. C.</span><span> </span><span class="NLM_article-title">MolProbity: all-atom structure validation for macromolecular crystallography</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">12</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1107/S0907444909042073</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=10.1107%2FS0907444909042073" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=20057044" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;key=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=12-21&author=V.+B.+Chenauthor=W.+B.+Arendallauthor=J.+J.+Headdauthor=D.+A.+Keedyauthor=R.+M.+Immorminoauthor=G.+J.+Kapralauthor=L.+W.+Murrayauthor=J.+S.+Richardsonauthor=D.+C.+Richardson&title=MolProbity%3A+all-atom+structure+validation+for+macromolecular+crystallography&doi=10.1107%2FS0907444909042073"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">MolProbity: all-atom structure validation for macromolecular crystallography</span></div><div class="casAuthors">Chen, Vincent B.; Arendall, W. Bryan, III; Headd, Jeffrey J.; Keedy, Daniel A.; Immormino, Robert M.; Kapral, Gary J.; Murray, Laura W.; Richardson, Jane S.; Richardson, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12-21</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">MolProbity is a structure-validation web service that provides broad-spectrum solidly based evaluation of model quality at both the global and local levels for both proteins and nucleic acids.  It relies heavily on the power and sensitivity provided by optimized hydrogen placement and all-atom contact anal., complemented by updated versions of covalent-geometry and torsion-angle criteria.  Some of the local corrections can be performed automatically in MolProbity and all of the diagnostics are presented in chart and graphical forms that help guide manual rebuilding.  X-ray crystallog. provides a wealth of biol. important mol. data in the form of at. three-dimensional structures of proteins, nucleic acids and increasingly large complexes in multiple forms and states.  Advances in automation, in everything from crystn. to data collection to phasing to model building to refinement, have made solving a structure using crystallog. easier than ever.  However, despite these improvements, local errors that can affect biol. interpretation are widespread at low resoln. and even high-resoln. structures nearly all contain at least a few local errors such as Ramachandran outliers, flipped branched protein side chains and incorrect sugar puckers.  It is crit. both for the crystallographer and for the end user that there are easy and reliable methods to diagnose and correct these sorts of errors in structures.  MolProbity is the authors' contribution to helping solve this problem and this article reviews its general capabilities, reports on recent enhancements and usage, and presents evidence that the resulting improvements are now beneficially affecting the global database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIjgsh59dO-7Vg90H21EOLACvtfcHk0lg3O_avUx6Hqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXit1Kktg%253D%253D&md5=b5fc7574f43f01dd6e43c3663ca4f779</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1107%2FS0907444909042073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909042073%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DArendall%26aufirst%3DW.%2BB.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DKeedy%26aufirst%3DD.%2BA.%26aulast%3DImmormino%26aufirst%3DR.%2BM.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DMurray%26aufirst%3DL.%2BW.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26atitle%3DMolProbity%253A%2520all-atom%2520structure%2520validation%2520for%2520macromolecular%2520crystallography%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D12%26epage%3D21%26doi%3D10.1107%2FS0907444909042073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Zhao, X. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenova, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vu, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maddali, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marchand, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pommier, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burke, T. R.</span><span> </span><span class="NLM_article-title">2,3-Dihydro-6,7-dihydroxy-1H-isoindol-1-one-based HIV-1 integrase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">251</span><span class="NLM_x">–</span> <span class="NLM_lpage">259</span><span class="refDoi"> DOI: 10.1021/jm070715d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm070715d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=251-259&author=X.+Z.+Zhaoauthor=E.+A.+Semenovaauthor=B.+C.+Vuauthor=K.+Maddaliauthor=C.+Marchandauthor=S.+H.+Hughesauthor=Y.+Pommierauthor=T.+R.+Burke&title=2%2C3-Dihydro-6%2C7-dihydroxy-1H-isoindol-1-one-based+HIV-1+integrase+inhibitors&doi=10.1021%2Fjm070715d"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm070715d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm070715d%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DX.%2BZ.%26aulast%3DSemenova%26aufirst%3DE.%2BA.%26aulast%3DVu%26aufirst%3DB.%2BC.%26aulast%3DMaddali%26aufirst%3DK.%26aulast%3DMarchand%26aufirst%3DC.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DPommier%26aufirst%3DY.%26aulast%3DBurke%26aufirst%3DT.%2BR.%26atitle%3D2%252C3-Dihydro-6%252C7-dihydroxy-1H-isoindol-1-one-based%2520HIV-1%2520integrase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D251%26epage%3D259%26doi%3D10.1021%2Fjm070715d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7I','PDB','4E7I'); return false;">PDB: 4E7I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S3M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3S3M','PDB','3S3M'); return false;">PDB: 3S3M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4E7K','PDB','4E7K'); return false;">PDB: 4E7K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BDZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4BDZ','PDB','4BDZ'); return false;">PDB: 4BDZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMA','PDB','5MMA'); return false;">PDB: 5MMA</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NO1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NO1','PDB','5NO1'); return false;">PDB: 5NO1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMB','PDB','5MMB'); return false;">PDB: 5MMB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i103"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00596">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_53460"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00596">10.1021/acs.jmedchem.7b00596</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Overlays of the PFV crystal structures of several bound INSTIs highlighting the correspondence of features corresponding to the 4-position of the current compounds; predicted molPAINS values for final products (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_001.pdf">jm7b00596_si_001.pdf (454.31 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00596/suppl_file/jm7b00596_si_002.csv">jm7b00596_si_002.csv (5.12 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Atomic coordinates and structure factors for PFV intasome-bound <b>5′g</b>, <b>5g</b>, and <b>6p</b> have been deposited in the RCSB Protein Data Bank (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMA">5MMA</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5NO1">5NO1</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MMB">5MMB</a>, respectively).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00596&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00596%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-17%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00596" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799bce1881424cb","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
